<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/8/1429" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/8/1429.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/8/1429.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/8/1429.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/8/1429/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/8/1429.atom" />
<meta name="HW.pisa" content="ggg;3/8/1429" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure" />
<meta name="DC.Identifier" content="10.1534/g3.113.006197" />
<meta name="DC.Date" content="2013-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Chen and Slatkin. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="It is a challenging task to infer selection intensity and allele age from population genetic data. Here we present a method that can efficiently estimate selection intensity and allele age from the multilocus haplotype structure in the vicinity of a segregating mutant under positive selection. We use a structured-coalescent approach to model the effect of directional selection on the gene genealogies of neutral markers linked to the selected mutant. The frequency trajectory of the selected allele follows the Wright-Fisher model. Given the position of the selected mutant, we propose a simplified multilocus haplotype model that can efficiently model the dynamics of the ancestral haplotypes under the joint influence of selection and recombination. This model approximates the ancestral genealogies of the sample, which reduces the number of states from an exponential function of the number of single-nucleotide polymorphism loci to a quadratic function. That allows parameter inference from data covering DNA regions as large as several hundred kilo-bases. Importance sampling algorithms are adopted to evaluate the probability of a sample by exploring the space of both allele frequency trajectories of the selected mutation and gene genealogies of the linked sites. We demonstrate by simulation that the method can accurately estimate selection intensity for moderate and strong positive selection. We apply the method to a data set of the G6PD gene in an African population and obtain an estimate of 0.0456 (95% confidence interval 0.0144−0.0769) for the selection intensity. The proposed method is novel in jointly modeling the multilocus haplotype pattern caused by recombination and mutation, allowing the analysis of haplotype data in recombining regions. Moreover, the method is applicable to data from populations under exponential growth and a variety of other demographic histories." />
<meta name="DC.Contributor" content="Hua Chen" />
<meta name="DC.Contributor" content="Montgomery Slatkin" />
<meta name="article:published_time" content="2013-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;It is a challenging task to infer selection intensity and allele age from population genetic data. Here we present a method that can efficiently estimate selection intensity and allele age from the multilocus haplotype structure in the vicinity of a segregating mutant under positive selection. We use a structured-coalescent approach to model the effect of directional selection on the gene genealogies of neutral markers linked to the selected mutant. The frequency trajectory of the selected allele follows the Wright-Fisher model. Given the position of the selected mutant, we propose a simplified multilocus haplotype model that can efficiently model the dynamics of the ancestral haplotypes under the joint influence of selection and recombination. This model approximates the ancestral genealogies of the sample, which reduces the number of states from an exponential function of the number of single-nucleotide polymorphism loci to a quadratic function. That allows parameter inference from data covering DNA regions as large as several hundred kilo-bases. Importance sampling algorithms are adopted to evaluate the probability of a sample by exploring the space of both allele frequency trajectories of the selected mutation and gene genealogies of the linked sites. We demonstrate by simulation that the method can accurately estimate selection intensity for moderate and strong positive selection. We apply the method to a data set of the &lt;i&gt;G6PD&lt;/i&gt; gene in an African population and obtain an estimate of 0.0456 (95% confidence interval 0.0144−0.0769) for the selection intensity. The proposed method is novel in jointly modeling the multilocus haplotype pattern caused by recombination and mutation, allowing the analysis of haplotype data in recombining regions. Moreover, the method is applicable to data from populations under exponential growth and a variety of other demographic histories.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/08/01" />
<meta name="citation_mjid" content="ggg;3/8/1429" />
<meta name="citation_id" content="3/8/1429" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/8/1429" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/8/1429.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/8/1429.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/8/1429.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006197" />
<meta name="citation_pmid" content="23797107" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1429" />
<meta name="citation_lastpage" content="1442" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hua Chen" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of California, Berkeley, California 94720" />
<meta name="citation_author_email" content="hchen007@gmail.com" />
<meta name="citation_author" content="Montgomery Slatkin" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of California, Berkeley, California 94720" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=N. H. Barton;citation_title=The effect of hitch-hiking on neutral genealogies.;citation_pages=123-133;citation_volume=72;citation_year=1998;citation_doi=10.1017/S0016672398003462" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Bersaglieri;citation_author=P. C. Sabeti;citation_author=N. Patterson;citation_author=T. Vanderploeg;citation_author=S. F. Schaffner;citation_title=Genetic signatures of strong recent positive selection at the lactase gene.;citation_pages=1111-1120;citation_volume=74;citation_year=2004;citation_issue=6;citation_pmid=15114531;citation_doi=10.1086/421051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Braverman;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_author=C. H. Langley;citation_author=W. Stephan;citation_title=The hitchhiking effect on the site frequency spectrum of DNA polymorphisms.;citation_pages=783-796;citation_volume=140;citation_year=1995;citation_issue=2;citation_pmid=7498754" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=H. Chen;citation_title=The joint allele frequency spectrum of multiple populations: a coalescent theory approach.;citation_pages=179-195;citation_volume=81;citation_year=2012;citation_issue=2;citation_pmid=22155588;citation_doi=10.1016/j.tpb.2011.11.004" />
<meta name="citation_reference" content="citation_title=Statistical methods for inference of positive selection from genetic polymorphism. Ph.D. thesis, University of California, Berkeley.;citation_author=H. Chen;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Chen;citation_author=N. Patterson;citation_author=D. Reich;citation_title=Population differentiation as a test for selective sweeps.;citation_volume=20;citation_year=2010;citation_pmid=20086244;citation_doi=10.1101/gr.100545.109" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=G. Coop;citation_author=R. C. Griffiths;citation_title=Ancestral inference on gene trees under selection.;citation_pages=241-251;citation_volume=64;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Adv. Appl. Probab.;citation_author=M. De Lorio;citation_author=R. C. Griffiths;citation_title=Importance sampling in coalescent histories I.;citation_pages=417-433;citation_volume=36;citation_year=2004;citation_doi=10.1239/aap/1086957579" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. Durrett;citation_author=J. Schweinsberg;citation_title=Approximating selective sweeps.;citation_pages=129-138;citation_volume=66;citation_year=2004;citation_issue=2;citation_pmid=15302222;citation_doi=10.1016/j.tpb.2004.04.002" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Probab.;citation_author=A. Etheridge;citation_author=P. Pfaffelhuber;citation_author=A. Wakolbinger;citation_title=An approximate sampling formula under genetic hitchhiking.;citation_pages=685-729;citation_volume=16;citation_year=2006;citation_doi=10.1214/105051606000000114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=Hitchhiking Under Positive Darwinian Selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Fearnhead;citation_author=P. Donnelly;citation_title=Estimating Recombination Rates From Population Genetic Data;citation_pages=1299-1318;citation_volume=159;citation_year=2001;citation_issue=3;citation_pmid=11729171" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. Felsenstein;citation_title=Phylogenies From Molecular Sequences: Inference And Reliability;citation_pages=521-565;citation_volume=22;citation_year=1988;citation_issue=1;citation_pmid=3071258;citation_doi=10.1146/annurev.ge.22.120188.002513" />
<meta name="citation_reference" content="citation_title=Statistics in Molecular Biology and Genetics;citation_title=Likelihoods on coalescents: A Monte Carlo sampling approach to inferring parameters from population samples of molecular data;citation_author=J. Felsenstein;citation_author=M. Kuhner;citation_author=J. Yamato;citation_author=P. Beerli;citation_pages=163-185;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. X. Fu;citation_author=W. H. Li;citation_title=Statistical tests of neutrality of mutations.;citation_pages=693-709;citation_volume=133;citation_year=1993;citation_issue=3;citation_pmid=8454210" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=R. C. Griffiths;citation_author=P. Marjoram;citation_title=Ancestral inference from samples of DNA sequences with recombination.;citation_pages=479-502;citation_volume=3;citation_year=1996;citation_issue=4;citation_pmid=9018600;citation_doi=10.1089/cmb.1996.3.479" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=R. C. Griffiths;citation_author=S. Tavaré;citation_title=Sampling Theory for Neutral Alleles in a Varying Environment;citation_pages=403-410;citation_volume=344;citation_year=1994;citation_issue=1310;citation_pmid=7800710;citation_doi=10.1098/rstb.1994.0079" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=R. C. Griffiths;citation_author=S. Tavaré;citation_title=Simulating probability distributions in the coalescent.;citation_pages=131-159;citation_volume=46;citation_year=1994;citation_doi=10.1006/tpbi.1994.1023" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=R. C. Griffiths;citation_author=S. Tavaré;citation_title=Unrooted genealogical tree probabilities in the infinitely-many-sites model.;citation_pages=77-98;citation_volume=127;citation_year=1995;citation_issue=1;citation_pmid=7734858;citation_doi=10.1016/0025-5564(94)00044-Z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_title=The coalescent process in models with selection and recombination.;citation_pages=831-840;citation_volume=819;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Y. G. Kamberov;citation_author=S. Wang;citation_author=J. Tan;citation_author=P. Gerbault;citation_author=A. Wark;citation_title=Modeling Recent Human Evolution in Mice by Expression of a Selected EDAR Variant.;citation_pages=691-702;citation_volume=152;citation_year=2013;citation_issue=4;citation_pmid=23415220;citation_doi=10.1016/j.cell.2013.01.016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. L. Kaplan;citation_author=T. Darden;citation_author=R. R. Hudson;citation_title=The coalescent process in models with selection.;citation_pages=819-829;citation_volume=120;citation_year=1988;citation_issue=3;citation_pmid=3066685" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. L. Kaplan;citation_author=R. Hudson;citation_author=C. Langley;citation_title=The &quot;hitchhiking effect&quot; revisited.;citation_pages=887-899;citation_volume=123;citation_year=1989;citation_issue=4;citation_pmid=2612899" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Kim;citation_author=W. Stephan;citation_title=Detecting a Local Signature of Genetic Hitchhiking Along a Recombining Chromosome;citation_pages=765-777;citation_volume=160;citation_year=2002;citation_issue=2;citation_pmid=11861577" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=S. M. Krone;citation_author=C. Neuhauser;citation_title=Ancestral Processes with Selection;citation_pages=210-237;citation_volume=51;citation_year=1997;citation_issue=3;citation_pmid=9245777;citation_doi=10.1006/tpbi.1997.1299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. K. Kuhner;citation_author=J. Yamato;citation_author=J. Felsenstein;citation_title=Estimating effective population size and mutation rate from sequence data using Metropolis-Hastings sampling.;citation_pages=1421-1430;citation_volume=140;citation_year=1995;citation_issue=4;citation_pmid=7498781" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Myers;citation_author=L. Bottolo;citation_author=C. Freeman;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A Fine-Scale Map of Recombination Rates and Hotspots Across the Human Genome;citation_pages=321-324;citation_volume=310;citation_year=2005;citation_issue=5746;citation_pmid=16224025;citation_doi=10.1126/science.1117196" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Nielsen;citation_author=S. Williamson;citation_author=Y. Kim;citation_author=M. J. Hubisz;citation_author=A. G. Clark;citation_title=Genomic scans for selective sweeps using SNP data;citation_pages=1566-1575;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251466;citation_doi=10.1101/gr.4252305" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=T. Ohta;citation_author=M. Kimura;citation_title=The effect of selected linked locus on heterozygosity of neutral alleles (the hitch-hiking effect).;citation_pages=313-326;citation_volume=25;citation_year=1975;citation_issue=3;citation_pmid=1183812;citation_doi=10.1017/S0016672300015731" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Paul;citation_author=M. SteinrÃijcken;citation_author=Y. Song;citation_title=An accurate sequentially Markov conditional sampling distribution for the coalescent with recombination.;citation_volume=187;citation_year=2011;citation_pmid=21270390;citation_doi=10.1534/genetics.110.125534" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Peng;citation_author=Z. Yang;citation_author=H. Zhang;citation_author=C. Cui;citation_author=X. Qi;citation_title=Genetic variations in Tibetan populations and high-altitude adaptation at the Himalayas.;citation_volume=28;citation_year=2011;citation_pmid=21030426;citation_doi=10.1093/molbev/msq290" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. Rannala;citation_author=J. Reeve;citation_title=High-resolution multipoint linkage-disequilibrium mapping in the context of a human genome sequence.;citation_pages=159-178;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11410841;citation_doi=10.1086/321279" />
<meta name="citation_reference" content="citation_journal_title=Pac Symp BioComput.;citation_title=Joint Bayesian estimation of mutation location and age using linkage disequilibrium.;citation_author=B. Rannala;citation_author=J. Reeve;citation_pages=526-534;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Ruwende;citation_author=S. C. Khoo;citation_author=R. W. Snow;citation_author=S. N. Yates;citation_author=D. Kwiatkowski;citation_title=Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria.;citation_pages=246-249;citation_volume=376;citation_year=1995;citation_issue=6537;citation_pmid=7617034;citation_doi=10.1038/376246a0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=D. E. Reich;citation_author=J. M. Higgins;citation_author=H. Z. Levine;citation_author=D. J. Richter;citation_title=Detecting recent positive selection in the human genome from haplotype structure.;citation_pages=832-837;citation_volume=419;citation_year=2002;citation_issue=6909;citation_pmid=12397357;citation_doi=10.1038/nature01140" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=P. Varilly;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=E. Hostetter;citation_title=Genome-wide detection and characterization of positive selection in human populations.;citation_pages=913-918;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943131;citation_doi=10.1038/nature06250" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Saunders;citation_author=M. Slatkin;citation_author=C. Garner;citation_author=M. Hammer;citation_author=M. Nachman;citation_title=The extent of linkage disequilibrium caused by selection on G6PD in humans.;citation_volume=171;citation_year=2005;citation_pmid=16020776;citation_doi=10.1534/genetics.105.048140" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=T. Simonson;citation_author=Y. Yang;citation_author=C. Huff;citation_author=H. Yun;citation_author=G. Qin;citation_title=Genetic evidence for high-altitude adaptation in Tibet.;citation_volume=329;citation_year=2010;citation_pmid=20466884;citation_doi=10.1126/science.1189406" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=P. Slade;citation_title=Most recent common ancestor probability distributions in gene genealogies under selection.;citation_pages=291-305;citation_volume=58;citation_year=2000;citation_issue=4;citation_pmid=11162788;citation_doi=10.1006/tpbi.2000.1488" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. Slatkin;citation_title=Simulating genealogies of selected alleles in a population of variable size.;citation_pages=49-57;citation_volume=78;citation_year=2001;citation_issue=1;citation_pmid=11556137;citation_doi=10.1017/S0016672301005183" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=M. Slatkin;citation_title=A vectorized method of importance sampling with applications to models of mutation and migration.;citation_pages=339-348;citation_volume=62;citation_year=2002;citation_issue=4;citation_pmid=12427457;citation_doi=10.1016/S0040-5809(02)00007-2" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. Slatkin;citation_title=A Bayesian method for jointly estimating allele age and selection intensity.;citation_pages=129-137;citation_volume=90;citation_year=2008;citation_issue=1;citation_pmid=18289407" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. A. Spencer;citation_author=G. Coop;citation_title=Selsim: a program to simulate population genetic data with natural selection and recombination.;citation_volume=20;citation_year=2004;citation_issue=18;citation_pmid=15271777;citation_doi=10.1093/bioinformatics/bth417" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=W. Stephan;citation_author=T. Wiehe;citation_author=M. Lenz;citation_title=The effect of strongly selected substitutions on neutral polymorphism: analytical results based on diffusion theory.;citation_pages=237-254;citation_volume=41;citation_year=1992;citation_doi=10.1016/0040-5809(92)90045-U" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=N. Smith;citation_author=P. Donnelly;citation_title=A new statistical method for haplotype reconstruction from population data.;citation_pages=978-989;citation_volume=68;citation_year=2001;citation_issue=4;citation_pmid=11254454;citation_doi=10.1086/319501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=K. Tang;citation_author=K. R. Thornton;citation_author=M. Stoneking;citation_title=A new approach for using genome scans to detect recent positive selection in the human genome.;citation_pages=e171-e171;citation_volume=5;citation_year=2007;citation_issue=7;citation_pmid=17579516;citation_doi=10.1371/journal.pbio.0050171" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Tishkoff;citation_author=R. Varkonyi;citation_author=N. Cahinhinan;citation_author=S. Abbes;citation_author=G. Argyropoulos;citation_title=Haplotype diversity and linkage disequilibrium at human G6PD: recent origin of alleles that confer malarial resistance.;citation_volume=293;citation_year=2001;citation_pmid=11423617;citation_doi=10.1126/science.1061573" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. Tishkoff;citation_author=F. A. Reed;citation_author=A. Ranciaro;citation_author=B. F. Voight;citation_author=C. C. Babbitt;citation_title=Convergent adaptation of human lactase persistence in Africa and Europe.;citation_pages=31-40;citation_volume=39;citation_year=2007;citation_issue=1;citation_pmid=17159977;citation_doi=10.1038/ng1946" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. C. Verrelli;citation_author=J. H. McDonald;citation_author=G. Argyropoulos;citation_author=G. Destro-Bisol;citation_author=A. Froment;citation_title=Evidence for balancing selection from nucleotide sequence analyses of human G6PD.;citation_pages=1112-1128;citation_volume=71;citation_year=2002;citation_issue=5;citation_pmid=12378426;citation_doi=10.1086/344345" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=B. F. Voight;citation_author=S. Kudaravalli;citation_author=X. Wen;citation_author=J. K. Pritchard;citation_title=A map of recent positive selection in the human genome.;citation_pages=446-458;citation_volume=4;citation_year=2006;citation_doi=10.1371/journal.pbio.0040446" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. Wood;citation_author=D. Stover;citation_author=M. Slatkin;citation_author=M. Nachman;citation_author=M. Hammer;citation_title=The beta -globin recombinational hotspot reduces the effects of strong selection around HbC, a recently arisen mutation providing resistance to malaria.;citation_pages=637-642;citation_volume=77;citation_year=2005;citation_issue=4;citation_pmid=16175509;citation_doi=10.1086/491748" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Xu;citation_author=S. Li;citation_author=Y. Yang;citation_author=J. Tan;citation_author=H. Lou;citation_title=A genome-wide search for signals of high-altitude adaptation in Tibetans.;citation_volume=28;citation_year=2011;citation_pmid=20961960;citation_doi=10.1093/molbev/msq277" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=X. Yi;citation_author=Y. Liang;citation_author=E. Huerta-Sanchez;citation_author=X. Jin;citation_author=Z. X. P. Cuo;citation_title=Sequencing of 50 Human Exomes Reveals Adaptation to High Altitude;citation_pages=75-78;citation_volume=329;citation_year=2010;citation_issue=5987;citation_pmid=20595611;citation_doi=10.1126/science.1190371" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
<meta name="twitter:description" content="It is a challenging task to infer selection intensity and allele age from population genetic data. Here we present a method that can efficiently estimate selection intensity and allele age from the multilocus haplotype structure in the vicinity of a segregating mutant under positive selection. We use a structured-coalescent approach to model the effect of directional selection on the gene genealogies of neutral markers linked to the selected mutant. The frequency trajectory of the selected allele follows the Wright-Fisher model. Given the position of the selected mutant, we propose a simplified multilocus haplotype model that can efficiently model the dynamics of the ancestral haplotypes under the joint influence of selection and recombination. This model approximates the ancestral genealogies of the sample, which reduces the number of states from an exponential function of the number of single-nucleotide polymorphism loci to a quadratic function. That allows parameter inference from data covering DNA regions as large as several hundred kilo-bases. Importance sampling algorithms are adopted to evaluate the probability of a sample by exploring the space of both allele frequency trajectories of the selected mutation and gene genealogies of the linked sites. We demonstrate by simulation that the method can accurately estimate selection intensity for moderate and strong positive selection. We apply the method to a data set of the G6PD gene in an African population and obtain an estimate of 0.0456 (95% confidence interval 0.0144−0.0769) for the selection intensity. The proposed method is novel in jointly modeling the multilocus haplotype pattern caused by recombination and mutation, allowing the analysis of haplotype data in recombining regions. Moreover, the method is applicable to data from populations under exponential growth and a variety of other demographic histories." />
<meta name="og-title" property="og:title" content="Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/8/1429" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="It is a challenging task to infer selection intensity and allele age from population genetic data. Here we present a method that can efficiently estimate selection intensity and allele age from the multilocus haplotype structure in the vicinity of a segregating mutant under positive selection. We use a structured-coalescent approach to model the effect of directional selection on the gene genealogies of neutral markers linked to the selected mutant. The frequency trajectory of the selected allele follows the Wright-Fisher model. Given the position of the selected mutant, we propose a simplified multilocus haplotype model that can efficiently model the dynamics of the ancestral haplotypes under the joint influence of selection and recombination. This model approximates the ancestral genealogies of the sample, which reduces the number of states from an exponential function of the number of single-nucleotide polymorphism loci to a quadratic function. That allows parameter inference from data covering DNA regions as large as several hundred kilo-bases. Importance sampling algorithms are adopted to evaluate the probability of a sample by exploring the space of both allele frequency trajectories of the selected mutation and gene genealogies of the linked sites. We demonstrate by simulation that the method can accurately estimate selection intensity for moderate and strong positive selection. We apply the method to a data set of the G6PD gene in an African population and obtain an estimate of 0.0456 (95% confidence interval 0.0144−0.0769) for the selection intensity. The proposed method is novel in jointly modeling the multilocus haplotype pattern caused by recombination and mutation, allowing the analysis of haplotype data in recombining regions. Moreover, the method is applicable to data from populations under exponential growth and a variety of other demographic histories." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
    <title>Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uvEks3yez_psmTcprGczN9gKnpWbLpXZRUYy_XQc3Yw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459515","apath":"\/ggg\/3\/8\/1429.atom","pisa":"ggg;3\/8\/1429","ac":{"\/ggg\/3\/8\/1429.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/8\/1429.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/8\/1429"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/8\/1429"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/8\/1429":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226JYvdD\\\\\\\/iKZT8D6E\\\\\\\/pcvhSiJ7VE5eJlOJqL07AREv3TbRV14O8Weoj5i1Y6jNfnILimLEmyPwxGVQhDVASi7pGMKVl9b998OpOEkzij09ez2v\\\\\\\/XoiHHQMENwxc\\\\\\\/WRo0OYVDquuqbppf0hh958XCjy0RcuOKXhUbYyg5hSApy4gCYyCd5V6XdaNoP4ruzNJvyWX\\\\\\\/XlYhO6GQ2B8sQk\\\\\\\/CfKDNGzgUCoUg5i0tBbkEk1LB2I18TtzlO+QjO75RrUHqkA6paWkC7g2LkXQvsqIuaeVIHYDmQ7SyYfV5R4n88o615HjSnBNxAAME2I+s76xOHxVTHyVYc9TVnOHv7mMX8DSquIXM2euedaa9B\\\\\\\/SOZKIJ3WgY1x+WLXXl66ajuXPQm2ay28BO1XCMHYEoShAv6W3khlff1HW+kZ8gbcsJ3WTaiPsNQNSc7DhFiUrnONgGC0Zw5J7HzsCFfta579vd74x6N1yPL19wf9FL+hu\\\\\\\/\\\\\\\/z3iMbOz3IhyQHil3XZViXzohDEknGwhh1bLKUafoyCjdnxQiAg0Eopxff1PAgyQncfoL\\\\\\\/s7Ru62sZRh0G6Rvfnb3fsXuGDq+Eye56oVSLp2m\\\\\\\/jSXbWlPI898yadxoV\\\\\\\/HgWPSuOcEXuLiBQpAirjU0YuqXNYk9GUQd2kOGYWZZatwzx1rlu\\\\\\\/iJsgY829OugMU2kSQ\\\\\\\/3Kjmzs6V+lYMLBVjOJHp5lBXI\\\\\\\/FeKZZ2U7Wj\\\\\\\/EHeGEYlp9nZCgG9oc+WD9kyyWH1n+RR69MlfExgEbdCrwv5ky+x73178d7YhF+c3A8Nb8AUD3Lei2Af6qTPyXLUIwol5VIsRQISb7N6YDluF0DXaa7mMqOZJPhff5uqq5HiCgAvcnckj+dJx0VqFtdqiKr2FP\\\\\\\/zVtIE8T6h2BEkHZ0qg\\\\\\\/Ut4xpMBqU7LsQwfAaH0aTrv0aPIBsmPdtxgMFQ2n8ZOMz+rY5WOtamftdWr78tyLdEzXq6nyfwnvPwubtRWpuz81Jjrunn6II9N3wxHewQ2ywlt+Mfn7\\\\\\\/nDFBEubX\\\\\\\/4hiaVraLJR0b3Hk7TmWr+oZnVg88yryblXQZxiT4aufHpxWaSBIc9xKcQMRvHj0GapnWA+J+dci1nykhfXyNH\\\\\\\/zcufuz4\\\\\\\/b1a2rQyoMoJvGm56XLQWTIW8MYocodyk2ZpQTglFEDaEH9rDNRWvsdRbotbdqCzZ6YQepDbjN6lDIetvw77NJevhy4Ox0FGwGsjib3MXpHaJxETnEj0BBbOak9Ifa\\\\\\\/oGTD+vdlXFxklGMJJmU82LUiYJ0sujXby9KS\\\u0022,\\\u0022iv\\\u0022:\\\u0022MSgJIa2fxaNgzl8APc0uWA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022360d50a66ae8c040c485b355c2438654a4a3b8c3b5f2ea005f424387c4585f8c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pQFq5giNT9Qxiq+bmEjR30mlY\\\\\\\/y\\\\\\\/gsQ30P3\\\\\\\/PU\\\\\\\/\\\\\\\/fU5CNayFP5rLiVP63t56AKDqWIy2PM77eLe\\\\\\\/vtjx\\\\\\\/RSWCtdCc0pi5mIppqhbKvB6WaLrl4ztsKtU2imUAnbkVLZgRPmInbyvfK1XmhbMNtaUiShMNCmnOKw31YiLQ8L3dttfuYacMaOQk5Vji7X7Fp0H14BUheQEtOwcmdMnfq7qKMi9yIMJCMFW6a9Rdc1VfGOYjk9+3Ujx\\\\\\\/BtQpK3AE2eJHegdpdBHafm5byIiusCjoTgCQ7hH+6\\\\\\\/ckuXPcqDV4kgbABOeC1sPMH1yfmVP+KNznPLs0MxMiJhk3YpeZhUioRTTQ8ivV+6onzjszoyb5UqEPAqWqi94FTZSoDEYZsymrXGWXpd0txOaXTeTUMF2Kprd9\\\\\\\/AWJLl08CbOYZPCju7i+znziQRdL0\\\\\\\/am6MeI0DOWJFj05W2N1CnvI9n4OwW+vjV7k+5wOd9P1vWhi6zEJzBq2TDoFtRNIpV5U2asbA059ca3Fg75CjO0Hp\\\\\\\/SZFKQqMXFK16U8O3SmaofY4UabJWJUqS+5g1cpobIdV7e9DczlYVvz7kHBpz1pJ0Z+z6WoZYBQiL4JnOdstcp\\\\\\\/U7f6OnknbNVJrmE0Cv3YFtQl9Uur6+4FZlaK5M8eKCWr4Xy17I3G0oO3sW5lOOCMr8CX\\\\\\\/9X\\\\\\\/E81chAZUaHKB6moW7lxziTeF\\\\\\\/0\\\\\\\/UWTLOgQ7C05m5WuiE+zaP6JgxKXXxR8ru2gxYwT4hP086OkCASDAu6iQyu56ekzAHHpfw4NyZ9VcNs+9JRIPDFshIO6BedUMz5DawN5D\\\\\\\/8ixqyD4fs+6UGXm7jByWYAoZOHjSgMg9FcDlnNszuFs2xJfU1sbxS2fqWRghcPtTv4sDFxSz\\\\\\\/Ah1V6s1DCEG\\\\\\\/s+e0exs8vGVGzVsoS\\\\\\\/vvaVctDJOtGJTs0PTetr2RQcEHV4jZYg3VVLRvD7pkTtG3HDbvsEkooLK8o9G8WZv\\\\\\\/rmxiniYaVsR2UDtSh\\\\\\\/90i\\\\\\\/AV0S4ZM8HobnisTWMNBHc\\\\\\\/6NhK4O1Oj+U3hkrmK\\\\\\\/f3De9EteuMn0SG5uzplC7E6mGhlG6cil26D6Ey\\\\\\\/Qjki4\\\\\\\/shSmVDS2RIE61IUMDFcrUw0\\\\\\\/SbJ35ko4\\\\\\\/WZU7dgb20B0vZ5znzg6K\\\\\\\/qjKb8kCBCleAO5JiT0MTN\\\\\\\/e3lwlhrXxylNWEuEpBY28eAaIt\\\\\\\/5j7MOei8Iqz8RVZbm78NiniBp5oZRF+5hcXmsvjGNO\\\\\\\/PVkAmFApkya27c9d0PRwUSPo4uIefBCXEGhet9YJJEhCuvvs97oEdKVV1Qx2o+MDl4ET+BBol\\\\\\\/vYgY+x1SDVPBaTEZjh+8m\\\\\\\/YAD0ZZerwUwI9DXUewBkTyEe58v1XlythQIVHmv3r+lOPE58zNwii93RQD9wjWmshdeuWHItl3FwexmW\\\\\\\/XqieNM+xWd8wbMj617brXJpE8YOb7Y2BWZCfFC6ECdbrT8EA1m8L+oOtwd2XZp538o4Olvgy4CUeWKwiO5SPR53vRaJHi3XTAjcTsRBd8gio260RZxvtwS0R90zPZS9PzF3vcPhO9oLjmWXNlIq7XmlN50\\\\\\\/VVIafoGj3Ss2NAPHGSoFWYIRK5uvrrKHDQ\\\\\\\/0uIlDvNJbzCJnJm8nOGziixPrAMkBBoXgjmP8X3wMQtYa13PxVOyxIfsgguj5Lp893T69ODsUPqORRKwbIThhCcETAEX3VIFNLIOqAj+ARgE9qrBjCLxs0kEQtjJqxVznMhif0rApimCeXMQyeeDDU1E4JRqUWsoTKFBGXvi+y8pwDJnMYyQesswlBs7KqEhUs6GmSiBHaZzn3+yL46mZzaLYqv2GNkLyemV4bKHsA3gJTBhL864WE1Pt2U1lvqfT0mHnMPSgcaIGOKPFIjLcpf7vAnIqXqNfDMCwgWZiTzGHgW7JzcjArurs1gFXhpXwiPMj4jZGYorfnTTBsvcFLBe+2PRo2iPss3aQ0BZ5C2rb7Z3+hTyeAPpyFzemZVhUfRklDVSpXI7ocZl97UYXsAUXpqypcfwnpDoe8nySbD5UjLLE6nk+BGvmmVIdPrq+VH2Tf5dfXLm5ilLYDbF9jOi44ZPSUXhr+Nn7WaUjkKd7xDTnSGZYj\\\\\\\/vBwzaF22BeOu9r1gqiz7pNpMqzuDuM30wWYN\\\\\\\/pejOE4D3gnYVLrrMhiWV6K95nVIE6D3eqJ2Y5LVJ4pS6Kr5y62AXOrbY793anCuJwujFLYQ3mjDGvD7GeF7oWio2bO4Dq4OaQTK45u8554bQbSi4N4YByY+FeQK7Hw5r2uGptUMycRTcQabASnT+4Wwl6Akyr4aqQepkEq1gMZ9URA7zqxsQ\\\\\\\/MMrTcCDGXYbOXI8oZcTLbz1Rx7JK3GvL3Cenaywm751hnpnYQh2\\\\\\\/eK16xZ9wuNtVdISxaMCAmUmH\\\\\\\/MJToATEHmWJeGWutlC8sEXs4tzE7sHaoeKtxv1E+OorpU3v\\\\\\\/z+l0KqbODH3oF9xsk2QwsH1SmTHcjdJlhw1MyD+TMb7PSGRNHc5mu50lAmj1a3Bt2jdfWIDJa3zCSm7HE+ZhVPpQqCWy3UOIuPr9zhbTsIpQq0pWQDdrX2N0OsJHttZvOxWlcvExS6mNVOOhRuhHsIIlUWekOD6tjrraR5fS\\\\\\\/Yi6OvSlXO0De7E4yi5GYQgeFEWcUwhAqr200Jw5CYOfluN2LfLLpsBcDoFEaa\\\\\\\/YRnThDZGmMI\\\\\\\/FZbTxlXSKt4biw7SOSTqCjB21UCO\\\\\\\/gzJJCtNIw1hcdFkJn\\\\\\\/zYN6k14nS941ddca\\\\\\\/bfkbl+nCACLkg8NB5gzXVejLfam442PeXWb7ijnIgTFySgpvISEyYY64mJRyq+JU00D4pro+12iJLscK912sDtpJalcl64Xd5zvq5MGgSpYfmuclLDeBfC4OYeeLw3Ve6pblcwYGsiRh\\\\\\\/EgENl9jJhgII+9DO+pj5Ayxt9uzxkLw8AQ\\\\\\\/iaWhzuR73roJFj\\\\\\\/FSv7+41OJBwmSjoaOE8\\\\\\\/RM1bqvT2IUfcPmhsissuY73Ud7iK6vrfrfOQcqkPGIFWDqSquH7\\\\\\\/BzTyMsKiBjmUrEABuWEtdT5nc4yj02AAqje033ARBi0Riv4jb4VX79VAq4fji7UK4x9Bo+w40G6DlQurq9\\\\\\\/mMy1Y4MPYVp8FMdeSl59kE3ArsW96YaBTqKTc1a3kTk\\\\\\\/nm+J8LPmJwF79BEGu0OyBBAyJXkEp\\\\\\\/yNL1mHet3OjFjpcVxOsJarHIVISiCoY\\\\\\\/RIWD+Yl7fZxI\\\\\\\/QiF6j3o64RSfsztRzOXWI5Fu2gMN+KjlfeP6K6SsQJuUk68ijXSTMUu8im6sTsXXkFVG3Nm\\\\\\\/CN6AanUE+xf0WHSBXW0a2+VVSqFFrPlsYnQEYHhLGAlLRP+REqIFhp4hBgvyaCgAs\\\\\\\/v\\\\\\\/qICklGON6r+d8g5w5smf\\\\\\\/V1WJ6tFAnfETlduUzW4VvFabRdNVT\\\\\\\/2WvEHyF4BB7wuLYWN0JTzPrOgGPOe8\\\\\\\/Q9DskS6y21u6B\\\\\\\/6wApdZVAMJTsF\\\\\\\/psr7KDoHZe19CqNj1AK\\\\\\\/UgNNyacqhB97DzQEF6mylDxhyx6FWb+sEdi7DfEXR6VCkHRwnhnhZ77BvtW1zPi8okz0eas\\\\\\\/X0EsxtZe56ZrFnuAMiB+g89BFD0g2YNCDG9IZNj3p\\\\\\\/wp2AUSdenE3tNKQnMQn3Mc1IMcBParCRHckcb5Y0zg17T3BxgXeC94kD1ydamVbaPylUwTKvqAfDZZ\\\\\\\/1LnpebFlIZAkYB0HIJOKGSqqtgUZeKoCgqFjmhtMSf6TVprF2muqsyfiYTEopskiGb8M1D\\\\\\\/omy8DzozNL9ai44PBi1\\\\\\\/KRuj8q0Xv3VA1o4l1eJp\\\\\\\/cPn9HDrzAOVQjUzwk\\\\\\\/JvKjCbOcU5Gt\\\\\\\/6sJbEb7Glcqw\\\\\\\/eF80Z7KYwd+eUCTk9rUOZ4939wUhw23qJWX75odCW3q5nUMJo0FI+b+glOGtcd6\\\\\\\/aAKxgnuz8P0D4IizEMLDju3Krhol47PM3qqNh+2bWwWczQz\\\\\\\/8crdZRxyQYOcflBo4vsU5hP0kdHay864lCms8lUkahSQFDDVCYi+OjWgIMpSfLuA6YeIO4mzS+bay9W37Yg0pFGC7lGwmHMhbJbKw4YACZNj3l+a1XJh7lqshaEwwQBOglcAhcCSCAgvD4mpWg6rT3Zryz56at+J5pvAxshvAWKd3Ok5TSmhchU3wRCrUc0i\\\\\\\/iXWkZKlbflgegX5EuObZ9iRNmvdhFeTtLafR+Y9H6vt56sA38GshCoocIbxbrk5PToB\\\\\\\/71yBD+3lJx+m49xYR6eXqIrhh6+WkOf7UQMgWKQNXQD3B4g\\\\\\\/3oo+R5t\\\\\\\/fJDCEMuQXgNgOehejJ7TJAYiSh0hbPLcN5Y0scKXpwvLPAoHSvF7S6KXK0EvchdSZsMGcxAH\\\\\\\/hJ3aSxRk10yuuW2eOWszOZ\\\\\\\/JdsMFQmQD1y+WOS81R+YdzONkkvEivrpljzcQPFRjDSB1k9BiTnJLQCEGhL8SkiWNO0uo11fQLMPOhviSogfGt8Xb5yP8xwoxnWNbxoxvmyzop7qTKMv\\\\\\\/StUISr3EhR3FrVqWpzgv8tWBIRWksn+2hf5hNa2Am6L1NDR8+7ZoMuEPFlRVIdGvd80VYyLCut5T7j4R+vuPoY\\\\\\\/XAltm2NGsPWRAF6mj9dzC8wokteQE8JnWQSeiszmBjhKByFrut9BjPBkw2DIDHn2Eikr5K8UL1vR6uZlOu8F5gM9P1AfYlwVCV21Qf1LtX1E+SGP5uPgMxlDPWLPy1mkNMMyZ0O5wyQDGUo7zqG7RL6SAMzrX5EYz2pQlHBYkKQFeM1k3oQDVmgKjlujgE\\\\\\\/xEym715l7pShuI8JPSDGSBVu8\\\\\\\/uP+WvQ6SkWBfP+xgOYqmrabOgZH2LGKLJ5V9j9ZVN8dfYWKLG042UTUyBZTC3wnmLo0apuPJCr+Ho7XG8y0Pgbx27zoEC7tTL5UUSKJGnrgkmfZjTq42U7pL7s+8LW7kSCtMO6mmhkQ8OGF+5fNzE7Ov0VLEK+S+Ibdkqz24v6EUJkeFX9H1Sb1jh1Uieevmld7l1eZCbitVxmmcOnPqcjjeLbmNMMnn\\\\\\\/d9y\\\\\\\/6qdUaGP4Wnfphz+jHz+p+f3JMHnAuAoZomKw1WmHJjBAG18gGjONm\\\\\\\/Upw8gY8H92wRGEgPLG7ncUqS\\\\\\\/h4Mx2WrjyiEXzJvJoqlh6e7SXgNGAiC3k8kXTGrfiaaJPSD0MTYYlC+uHZWVu1OOtjHXhG6FkW4Iekk553XkLHrTaWk25V0oezZ8I6smUP5TfVum66WjW9fd77Ohce8HEDOxphkEe+wbqQJ9KTvrB0P4ftjE8Llt9HRSh50+FxuZitYbqZQRiYpNp+\\\\\\\/1wuo6pxzlqVdtyhcaFzM81R3onJ59ut4CvJcqRh7l83TfusUWn1HPk5ODC12aG5OiesvtgtkABTMmAm8Dw5jRcQQERa+QCGBOs43ubtjMLZPO2hTqNM+2LlGz9zpNf2Jb0tXo57LfxML0RI721TsLo3SGhgHotWDAIcYfbHSilu9WLianQH0PKBgtqoniR2JLVcmmMTL+Lix7v6dsuA3e9kw8udXKKy86ES9oNyc0jcoUzaGBzmhKDrybqqIoZlnwO3QMO3MK1UgiqJ4D6U2XWXLnZB6IoPj+5l2p\\\\\\\/LJhNk7dPjzLcyqF5CT427oCb6hfqQ7HCK1DOe7ozikrF4eZFequIwwOS52AJ4ElSuY3KpiJ+kKnRaDFwbO5plgocM45DeFSorzm26CnNJsdbHJFVPM8oa1zhbmO6SRIznQKzP0fWwYbRL\\\\\\\/gQoLKAmgB24dqStvnJvZCP7rPsSqo7ans2W3XJaGV0JOtCyiFssYTEyYmuKZUkAE2jECruAvzhM9L7ue4yUPO9Rse5oy7kigBt8\\\\\\\/lMRjXHuHDBiA5IfccpSQnVwT930k0XvpnXiNXbWl5KGeSdkm3CmSl5nzob5ckv0+L\\\\\\\/tXY5uQpyggf9YhXJRzSsNZEZU4mjHoW5RWtT6pSNq0EJhVU3i\\\\\\\/s\\\\\\\/QOzyl\\\\\\\/qcyOxRnJrCnCPPVRcFt\\\\\\\/K\\\\\\\/EbVgvikheNRjHh7mZJamIlEz0ya+rTkCqUGbvX35fmJCpBdUeXrCfYEQAaCA5upJhwqB9A5kYCZLBRp1GDg+lYpG72jUZsqZ9cM\\\\\\\/5HOrMm77k1aLOJZgBxmTZ1Hd3tL2krJeYI1BP26adxrRZwe\\\\\\\/MfxPjo8AyHwwP79mRQmaGgQWUEg2eNQ+M8dvPRymIoAI1Tr1gNTHJL2AahC8oRcmcxKmu2TRS3XuWTnNlL9DzpKZ9RrPLuNxTkMZU+K+VlZbWOzI7BZL8Wu3m8Tw0bEbiY59c+NmeIKPf3flxyPYqdzkn8lLJUCycNlctzCerNe6EszaSPdTJtU542jGgb5PMb6tfdEXmknW04AX21ZvmbVvDFmpu7FTsrgcooK3a1DzhcBERepdNuNUZC\\\\\\\/q5RxxApC74AVoEkH5W9ojClmY3CDxr\\\\\\\/IH8DMHIRYqGA1zQGuYPrzl+ZKUUCu9sIo4QMOKW5FU2UwcjzGapVL\\\\\\\/p5x2losAXDyQOByh\\\\\\\/g+PY2lVHtuXbCroogRYVNromUcaiIsh+RC+fVWeHqQV6dyaIofOShUKWhJZrNczawcarHLM2mefZ\\\\\\\/q5zlVnDmOqhQIjJVIpbNHH\\\\\\\/POKrcmDiGKv+bHcqWRgIeHl8o4qQkNbpvynARIhiuVg6TJU4qwLn25vQiJkCxthbmjadz6tAGfocIsOsrZlwsrSKfMi13t3MtAI0yEtNOOMsThH8fsi5zEqIgOYPe2qfRjy3MphyT3tiwYaEdrSDnVUr1I+43IAMpoklQ4MxYqGqM3IINlag4b5xkIBH4JRGhgBkRQmYy2Ly\\\\\\\/Hgkj70u9DzKWZLhNv\\\\\\\/OEqjkxzoa4Ptejbebj990JRXtUKx4PWH1RwB3gV5IZ\\\\\\\/z8woDika3Lws5qzXlHKWBDtVszy\\\\\\\/FVQuEhDTt\\\\\\\/3EbHLgi1fclhJ9yeT0oX6c2mksAXd6kSR1cWYYBwQUVIug61vb\\\\\\\/G8iWlzqlsEItaU+v87HxcAuRfAvtM1yr6CoPg34UqLLNHTjqpc5EoyPnJ7aqqUDzbWLVZdhQMQSQ8bCrnyFNY8d0quOC6HnAeJAHUs9wteREOigfBa14BK9djkm6L90UPhDeirZ+a4OIWCCjWYzaOwQG2TA702iqobCeHwmvpfY967aqHrDmJgncVAHA\\\\\\\/G9eDygO0rOdCWxgGZj58xyer0zEfkCojnhycH9dfpzHgzJlgafOhMC\\\\\\\/wZrWoPNGLd5pGycCqWheEbd\\\\\\\/e\\\\\\\/dKumqHasjNkCbdGx98bWrNDzDG37ODZ0Z9R5QlHzldhfTWzrjYdLCOFOCS2weFgVIezZjVDNrLdmooM2r2Mt9cJ42C8maRnuRrOxuWIqH9fcx7evXmnlXjLJVC5uMxGLuvEIqqolrTMY\\\\\\\/hATP4+LPXvcDX8gGFyENFpxtF2LC54PpWKdde1A2ti0KaRLmnOrDeKAZQd2iCmhP7ArS7U6uYwjUA1ZeOpOGjycg7b70FXl1uw2cQrdHS0XjjVLGHqPdmB\\\\\\\/\\\\\\\/\\\\\\\/yXEMiTAKvP2Y6ysoeVcK1s9YrDcbjeW3NKVO86f1ePFpctEcecvQ+w2yAyHK0liwbxks+xqZbAN0fW0gXbT133rQhXJvD0Ymea3Qqkf5DR+OxmvDx2KBRYJ942jRIEBuT618yqHoBvttJWLxKKU9ahfVU2ixEUn2zWR0Z1dfL7\\\\\\\/o9RrlAxbNrdX6c7YhcHNsOihByHo11zPKJUn4EzBfxTO1cDJDj+IsklNhA3m8m5I5giIHUI66ehmx1VDqre2m8M6iHQCt\\\\\\\/9e4LMNGtcu\\\\\\\/qCvgLfNy82ON4XC\\\\\\\/zjntqPA8Q1ogMmAMy+7vV0VhwEBNcoVhK6SpQHYKb8kOP7qyb3Q2N3E8J+0IeAIxgOASEgAD3Zfe5pYvzy8dGeBPUoJwHMmW51PQfV3NV6JxMgLSIs3gwrpRT7jiqdyNqgRc56WXq0S1XeL3a2ttA25OUlOdx3ymTa4AP9WDtnrrlX1velbYhzs82Qw3y89O5+n0Ye59InUkxj4GBHAXGIuCgLFWIVm+kEGI0iBSqWkaKla023fee2ADEEjSkK+Q+mUjZJGTFKZ9xAFVvOuGK3HRD816JEPjvDmJVkGf+xj7anTvRLZZCFYrGOkPq42TBVUNmqManThwS6eStSqjozDm0VGF4Jm3BQX4jtFAoMGkakWIpkM1nTf1IGNJAqazos+NEKQKMuDcogeT5i1mQvLDl5LLjBAxS+HbkLy\\\\\\\/QT1Z4en9ouDzUNonMC4zEMYC7nrzyEa1h8iMwErgnMtl1Kgl7JhbzxefrJ6jRzsvYtw\\\\\\\/0BYhHkvKgwjB69NRa5bAdWTbnlaiQFTM9RmVLNY1R1fYMgSWLz5m3KWwp87tIAYSHtBVM+bZb0K3hWcHFh9nbkyd3AdBAIuQGKcdNjlwiBd67kAh0z8wlptMHLGLH1+majIM53QT9DdshSG2Tt9wB5VWKhbaQJxOon9yefOTwGmaGUYK+iD5zjQ5xQlus42UdzGjrhIKoRDtxXmTmTjeAdkW10CLdLp5AJneDRcvvN8GsxnkFmpfa69YhYTp4fNms\\\\\\\/4ejYxVv5\\\\\\\/XKkRerN3KpvOPZxh9c3g9LVT8jfEJUL11lnLoyVN70LbPqiJ247TiCmt5BvbEfg3xE1rE\\\\\\\/Ekco69Jl3\\\\\\\/tKSWW7M2fqE48NFAxCDge28oXCeDSi2wjxktiwzF2ZX\\\\\\\/r6N+csL7qlZIr6Hc5OXghPKDTE0eyb8Lirdurypm2k3buttBO8A6VMRX52spTo6k7d8KlfitpY8CEq+uJ0fuco74sd6sX58UzGDjR5Hq4\\\\\\\/kuKlc9H996R\\\\\\\/9Rl6X4rXoPG+HkEdINKFd50LNMPsqi9oObgBpuK7+X0vV9AE3kMyGHe54u+mnhPzL2fJ8qbzeWCvfT+cVMUcjazEJowUsbbCVfTTnvBktn9dtjVGRySNEHdd9ruDFNX9BOp1LTvfOgTQIDgjiiYeICP1+9VeuO+gzpLE7ypBXd8GLR764wVpuJQmr\\\\\\\/3znVvqUB\\\\\\\/lBt0rUVsU2AVsfOu5uZsbFbg7toG5BlKav1SXWTxIr\\\\\\\/xWavEBgikmjdBlCFWYdwzhLEJuHuMqAPH0Hqvb\\\\\\\/gXhUJxYhO1kfLge6NNujA3FETKLJaHhd1t6ehdJFBh1ZSDEVXZMczl2ogN8BEUl+1EClH6ctHYhUzJ7U12\\\\\\\/N+P8B4252Vk3bO61RcCMd4Gur3g8nmSIkZfN3Z5mK+QqCnf8vPhmREJQkQyQFoKfiNPSCTP61bwKTBDTGd7EY+z5Er7vF2D4GRBV4W1ricnPJ7eZZTqGYQ28l2DU1xq21OTuQbAYCfElOSlF9eIrCX2PUzVO4xhw9JKETvipcBNLc\\\\\\\/zN8wVvulCOZwA30aaCAKUNA7sgFfJSGZXqmYjaFua10MfJoR8+GsucYP5Dx\\\\\\\/ogI9QsBLlgVTGOwWAG6XPr1EFTqtotWOEOElgxwXz3Ns8RQ1Cp+n+Qh9aeJD8kzcB29SsLBgZf29OVPos4KXS+Cn0k5Y+OeCZHHoKIoNkrG4WanWPHllw1\\\\\\\/SSnBZ2tNsCK5BCs+oEtFYA+M+X7xuOdEePgBaVuJXdWEIj2LYQrR6VccTO1l+Lz2BfMwraqU8UCznIEWXWsi7nHgR2GXu3seGKgTR51INc\\\\\\\/oy3OvZ7E9kEHd99v0ihFznHTn7+u+5sNeBrDap2eNQTmURDVhYg8o0oT6+ddYyysVVojPbt4VBdi1axzy627MlilyfvJHnV\\\\\\\/uHSqvavqYsrf80ZQxzKOaiMU1N0bOauyMYhHQz2ne6nHJJfsiIs26ijRiD1QoguZ91WVKzXydMTXqJ3Tsgpzpl8mr59HdSwnb8Yx3H4wzBNtBRhcCH9GA9aN+JA1++kYLw3jLRxnbvQEGqtcrPBnsSzOQju+DuAdG5XMT5suyHhHzWY2CQWEjV4Knw+mH2gC2pwqdvREq7vwSn4GSzfVxPSlUBS8f\\\\\\\/VtG2wcHQYX2vbjrc5m\\\\\\\/l27eqbnfBXLQRloYN9\\\\\\\/0niHhDFlou8BCTWHQlyOCknUCJ\\\\\\\/FAbFkattg8gqIOnp2JPR+7a+A+s7BShXvm4KWJTg0UR4l7oNVANKDXe+DbLAwkZ7OPa\\\\\\\/S22dNeefrEvt5Lzh5HubrDy1qsOfyFWQyb7H78xjDi6\\\\\\\/46UODsRpD49x9AH56fDysvPhtSv1kbxfMe1qZSIvnoojbUYr\\\\\\\/SgJTGE2pZe2tJdNjj0l\\\\\\\/\\\\\\\/Hm\\\\\\\/qQTmR5m1fon5en2+XUOuHH5R\\\\\\\/jyQcYPcXabP0QBgtm1ImzVVlci3KnDEjvDufjhLM0h8hNBu7DPcaI9YlSbJs\\\\\\\/S1FFyGm6QaLJlItUl08ZsGXJdmaikEKzVObfGp9JDyJMYPTCY1TxCcQbC7Nd9+BpRF9Z8IeDq4Q9Dsi\\\\\\\/sSbOKs3+B5O6a+P4hoJUmmj79X5ZCOo2WZEMkh0enB\\\\\\\/OzqfO2u2xERY12kxS1N\\\\\\\/8LTMesVdH5tByFWdZL7rJzL5swxPgt7ZT8rCgOBZfLbhOsY55LeER7ty1FVoECnjB9Kb\\\\\\\/mHBwaDht36aISiEXLPQ3rdzGCsZxp2nv2F244hRPRsxKwhIZUdN8NQnA27hw1GYpHc8lgINveogaCsv06A87FnEFSDrb572JL+\\\\\\\/A5pfeQHF3awFWuj5wQxXPoCB1IPKQ3VaWT0sIo5zlinmKQ8R6CZRXwmLcXHp4YuRYYFhS4hPyOEvchaLnbjmacE6OWqvY4lTkZhukbLjkyZu44RqTbLMpTbWCbBWyHF4u1182\\\\\\\/9cAEALaBEyagI1eJ1ru7NS9fUcXrR1W8Olu2piKDrHskiY6Lu1kaO2NFyK\\\\\\\/b39L2wrdMq5zN29zFCBa0vBYojeraTJYjwnnamT2m0jTZA\\\\\\\/95kRrLs2H79pVnah0vZA8rQB78TbeQumRaGvW1at8kQfmHsKfyI14Zf2Teo\\\\\\\/7CZ8q8Wtgk\\\\\\\/mYCrgornA36epDZVB05M+odMyuLD9KlRhlteGnSA2qu3J8Q3Zu9DCc77+QoDO87VRvhajCKkCO037fCGO9Ak8PP\\\\\\\/VOjALZ2sXwXZ8dsRVKX\\\\\\\/aVYANPwO0kvtYkTlTZNOFCqnVlQUn6C153XLjunivKyvyCVXr9MvcjrXb3PLmkZqqdNmz\\\\\\\/7qTHc1vBJ+eSH38IcwZTWwDVd3MWC7oicQ6jkQ3XIVUM\\\\\\\/OWfEPC+onkIDit4q72xINWXXbJz1wu5zm8GLy\\\\\\\/sxvYWek\\\\\\\/uudZ30PqQH1Lgygh2E03Wq6rbqFEJeKnvSzJTJOQYoWMHPnNFEn8gpPNTL2f5VeE5z7TRRTFf7xIDUAd1dHjLvMSN+Z6kz9dLKlVVL9oc1TIeq3wa4OVI37VJOEAi4FkFq7rNwYfiYNtMdCkZYS6tGvMOwlFoXlGXwM5CX+YN54Raj5cwzMiod1mtmrnypysy1SbX671TnP78udqlmhGKJUeIRGVaBgT9mvn9Qwcs0x397agWjWWfV8gWeJH+0wKJAKWkL3rqCn6tN8+HdcM7fai+QeQjhWDfQ2QuHjWpfPfEZOddBktOB168CmiMX7QMVDtXxjAbg9FqAU4O9jOY9T8EUhUAYftpT3NZLbm\\\\\\\/vfX52+IjH5n8SPD3hy0ZDEP8HlIYhpeTnFo8CbQZUmdRnYkbQ3mSG\\\\\\\/odeRGt49RJR2s8HYWK+kxeYKndtIeR1JxRt+DOZyI4qznqk0qZUNXFLnhN6uPC22kgjLjEWr46U+4UJVuLQ1A5OD\\\\\\\/fTxeTORLSWz++NYCP6kzsK8A2PC4BFdk0Td8GdfGlf0qSOko8UUhsBqovhKdYIkHTTJF1W\\\\\\\/Fz1E9WarH\\\\\\\/qvXColn\\\\\\\/60cpaTDzB0x+tQ901WtTYbbyXYOyNZt7FkvwG7wcaoT3SS0jBgfbdGbYKJg\\\\\\\/mAz0PHvJrwClSC1aeSkPIt79BXl4tQjAVhYJDqSr8sLVZHjfOdFXJZxikCgNAlspWf8V0iY\\\\\\\/G7X5E11s0RXZLFd6xliPHOiwJJ5MdyetcjNhKhODydKmvEybFgp0MmLvYmzLH8V0EgAAv8S0MO3peOlfTh2TKaXAhHhPXptbz6YPgj6Pb\\\\\\\/ElX\\\\\\\/pfsGoLq6m55PxNdUeBTRTWL4GeqV4b2ZEtKe\\\\\\\/7lG76gkzW4t3fLvl96vSwrqMY5pleCjB3xoYPJIJs\\\\\\\/sDtkLxL4ZBmmfV1ljcs8tHvOolLkJ7TRNufJo2S1bagUBULqUcBOthYkrKoodeP9JetMs6zGISWyo7Y82y4C\\\\\\\/DW689T7w2mWf2z19ahq6\\\\\\\/bAnkJCP7OgEjJZUjmb9+fYovzFHfJ95g6gO\\\\\\\/nKMeYs9q7frAYJYgpY0Jv4il\\\\\\\/+PK8l0jys2+YtfnFgopz5x\\\\\\\/8md9gJWKlnsGOSu2CvP2OQGu\\\\\\\/4IPqWw8n02\\\\\\\/BnEQm+0G\\\\\\\/KrWBhzHZKyMnsZ5Pq\\\\\\\/9pJflGNLpw7xznDdYkKSDxUd\\\\\\\/y55D05Zew0rf6k5jH\\\\\\\/JKheTlVusW7j2irQtBlHK+3OLDivpwAxOdkjQ3M3Z3NfHlNCwjP7Djx8SS85Dd+RMuJKJeN0n6bVH+j114nmpg6vUS8gYqtcdtP8XRTg2gvmNHszAhEohDPdwKbTBLb+WWdlp3fWE97K20BDQvXmkpJiHuA9ivgOj97hIGR9nIwn0UslepW9pByOeGWGS21K\\\\\\\/VLakzGhNYL0eE2bOjyQooS4OPiqbqALr4oCIT\\\\\\\/heV5TY5wAvnCF2R9VeIUrPw98Q3roozPxvc1+zslexbgbYaxQcMKlUDtrHGvd7Iyr2VRf3+NBjxYlNAwTBQbT9BRxzlncbKzLMdcjjnapEaZtamFtBBUMN2wnMhe2jw3XidmOJtxU3XLF7OCX\\\\\\\/b+N5v6IzJ85BWnfLuAETwQ4qeGMKv+jGB1fYAlpl2WT+ermYSf\\\\\\\/wtMkl2IJvU1AxLB3Jq0sQjDgrAlJbZtCeE0Ro4fkhCTZPfxAAbz2v98FCTZASHix7riVVJ2rdIBg9WaaHU7z12hHV55Tt1zXDYOI4KHFwKTzcqItKaMBh9NrbajTHX61YPO+WX4Cp8KFe6pPhOiIKRztfgPLu58HL2kD84xEQ\\\\\\\/OKyM7zRz432Vyc+1er8t3qNSKKYNcF2ajaAfriatVTHVQBgC+rEKDxdxjdcYANzrydgEIjhVjN5Bs7n22FkzHP25A8ntSkLFe+1jDpU\\\\\\\/VaI3pSESyf8r0naWMCQJ6yAK99ZiPjJuIwpT7p5vuvyBeHA3RYREaadyrxNwx3DRk+\\\\\\\/wfjUrPVuxdU621acq+bY\\\\\\\/1aAZwyBIQ8SCFkyyQCH44KeQxna2JmD7i0Dx83ROkUMuzKvIwAfxqjRGaUf3Lc4DATt2kHzOUE16eY0tePPVb4oPcKiPPHmW9zptQl\\\\\\\/dxX1p1qUVyhtcrl2gulzgM4opxMxL0rY6DaXMVuHITibbkdSO4adv3fmMoKGtB\\\\\\\/NjGa14O\\\\\\\/pi6vdu4k0JUMs9u3petk0XMKJLU58Gr+d0hnPNqgYfLVgSGD+2zVd64qdTIOAnHbJc72JWkeSAIj2lWuzRRxJtNhsHBeZwLC+BzB+TYWzHL2HEndp9N6tQkTkGQ1Mr8RysTI6x6YemEzF+lZJI5Qca07LcLxWQYX8CDXhvhbUsTOCJZTXeLBTHFaPSxTL9HfzoG7TDUoAGXcSWkxtkyFdVNPS3PcefGUDq82WDvyJkRUTXLDNm\\\\\\\/EPxe2r4vDCDBsveDbFDGthAmjcO8lY1rIPFE+\\\\\\\/y6Rg32H1E9PT0K2IsttImUo3gwagqgi4JFGX0X\\\\\\\/v\\\\\\\/WrpawpedVcajvdREVxurqU6ixeRFG\\\\\\\/OFLpLXVKUsK+CbcL3zRvkFD2LyeryO7FKNxYaGi6HWh0Pexes+W6OF8sYDvNnXv3UMqVdhYs4iNCR1q8dB86JksNl3oq1X3FVQtMTLXnrzI4hGc5COUYhtmrqB7u+Jf3DHzaBKByel3W8OKs31LkTdF4vEJkZDaIBapp0ekFGkJa3oFtYua+8ORYE+AJYIbKaZflIVybVRw1DWaMPyB4hXSm9od4kFKu5jRwRi7grHxJKBMnNF\\\\\\\/ePN8a1BY\\\\\\\/rwfvOK10VrwfgL6SC4lCYqY4y8Eq87ZW+NvIBAtzLaxvOmg3rw0kPUNioJyWgVU8lUiC3+DMBdlhwr5\\\\\\\/a8REfp516cxs5E0oisgigDiNTD1\\\\\\\/HCbey8Kkc5eiSPyh71zM3AXWKFQ5pVKxYJRs8AaOjLGbN1VHzpIuwDZPERKldUM3S8ZJLQE6PwsmJdwyxOlMwlwIaV7P6pSPyFTqJzc6D4+5pxgSikjKNuTWfXRKt0mFsOegU4aLM7nt6ohQKaGCnBI8oFijfdSC\\\\\\\/q02uPMCnBMTT7bAGwwAmEN5sa+0eoZvbVxnGHR3JGZxwYf40qc3tT4ZF0Ul7fKw8v1XrzDUdsRc03dJXq5U0+3no3f7k7GEtz2VVnaLjSAy9bIq853hOvO430evy\\\\\\\/rW1SkfYXop7Rmx8geEys67po9WD70FuZwifF90k0Ai8dIcGWQciy6kEDD7doEKHFN2Jm7QfGfqg\\\\\\\/6+sKtD6Wow8nCBdhZaFIudyICusTGAkgtGVagwOmQ2w1\\\\\\\/GBrOUYmRFYEuyVpMhE\\\\\\\/R4HxqUBU4NI0AyUOdGvYZBkPBRVwiJ2mXtGWfxhGtsNraS8OI0Cxs5BAzZkiF7FGQst\\\\\\\/SxB1YRiwfvuiU5+CMuD6YinZPIau4gmAcBaROxGCjdCsiYazDchHBsb\\\\\\\/Cpt1OaogMffoWRAahqPcDyl3EQB7YdM26EQBV3KPHx5wiUVfionyR+hthlgqxcuylBs+kDY8YhsOp0mVAKW+aM7ntrpLnPPMJYUpYThmTYnCMutZCrDO3uxkZjV\\\\\\\/hoJdbjTt7xJ9JhETTZRQYgFprlb8hLZJyaNb2mIhyFvA+NH6hTk6ziKGo\\\\\\\/vbY3E42kHWpM1F5AyYRnz77pZXuCYM7Tl5PLLJK9\\\\\\\/sscyC97k0vOMJXYaPT2qd0JvEa\\\\\\\/E9EbZwKF3urX4WzhGWg4Mi5L6aZNZ\\\\\\\/ptKIMAFly4gd3rCbdWr2suukkHVcbQKIo6PmzIZ5FD1W+azlQJNn3\\\\\\\/eZTP+wqp7MIfWAkxKZ8WDDpbstpoPetGTuHxlbjRcnpS60Rc0o3AidP96jDZi4Qr0ItTGJdILPjubP4I7ka80UHs09AbT9vb9ko3aoNZV7QhvltzxGHnoNJ5\\\\\\\/T9+aNVo7IuLiYXQsgX6FwblP6cFoM1nMf73vxmbPmhvmiMl\\\\\\\/NhQQDLZRW6InglWFti16kQtxfidRwc0nUDUJykb7WgXHZ5\\\\\\\/G1GHjnTlpc8I7Vh7zXvi7gz0W\\\\\\\/tFGWESGkhgI5lPAT2G5gemWZI1pQTr45kuoZ0huoAJBwPiBduZkltwLWN9cWpx2QNLxAOeCTibWRXrhBWBldclBFin4ceqMgSs7Zu5DKn8FeEeoh6T3f02Np4uyob0RvUV+Bait+D88TYFG\\\\\\\/ZFC1\\\\\\\/SerB9VgdXMLeOcWKt8GV87XRNw0rmN7CD7RpnRV4nAUSb7Y4G9HaUkK8IT35cVMuDBL0OOlk6\\\\\\\/EL6AUzrhFI8OyUv65tvaqbQBzhSEHnWpxPIHqOv5GCvh5Jm71\\\\\\\/Qy+eUv7+VJmowXasa\\\\\\\/gxlry8i9usMO6+A8T5h6w2NXBxsiC1ep90XUpWmudN0EG0qHZmaR\\\\\\\/ksK8rMzN5fBXe5LMLkTjKTDwnKpr8YUqpIgEkTqtMZ20ZaEjnASfrBHH9hq\\\\\\\/27Trob\\\\\\\/sD5F8\\\\\\\/LUhCTiaAzPsZAzCkkEIV7brcilOP454u9GI41vja4PJyYCb76+dcaSFJZYUh\\\\\\\/qZhBYaQSwjhYMudTO\\\\\\\/6uM9wLbhSEBEpCkKamPz9QAN8FLa078zve\\\\\\\/3wokVP8DiEG82gAJSx1AbibObbZFhbWVCfP0\\\\\\\/Fa9ybgvzFgJVsUaxvvTI8JZc2VUCyu8X0Un9zPe64fx+m9U7aPwuulsTgi1HJXWeYuq68wdDAezvm1qfiCKjE1gG9d2k00wJmKwup4+XNLCBts1gsXLcxbKdRaFwkRE1E\\\\\\\/NC7J6\\\\\\\/ivvkkQzMno\\\\\\\/eqD+GPRUxPbO55D3wGyNuR4g4NTejM43ETvXxsLmvTQh0XFu4apNOyHBaLQ6V1IYAGz\\\\\\\/shP4BwlwWrQrQZXXljyDiYP8XFA\\\\\\\/a3PMmqJ0QvaYo5TYdY25bSGGteTkXqSn5S7UP6CPo12aSYS+4xjNArUfml4Dn+PVtcGBwNZs9FmA\\\\\\\/DQ8F3XLT9qvxtNKl272+QeAiEB1Y89h66WIFIXfOUw9TKhpJxAWs9+NTfX6\\\\\\\/Uh19nlze4ByLaUk9faPFz1v+btd2yjrvRZB\\\\\\\/fKSdZWfFV7xi8ai0FXpio5iQCBf+ZLuilKn+dlZgORRYXOs5OQDpNPiGo3sbsgnjaDKMjkQzb76iZiWQHH2A\\\\\\\/yFIuHKP3X9\\\\\\\/yGA\\\\\\\/Zqb04Ze7ps60Gj8rf60dMDkx\\\\\\\/a21DepQBENS2Gf5Jru9nUhCmv1S8cG6+n8hoB\\\\\\\/R+cjkz8a6\\\\\\\/r5nl0fGPioDrVxuKza\\\\\\\/YrfQTPUO\\\\\\\/VRnIeM8qmbLVr1uzmH2\\\\\\\/824oS0Bnl8OAdg78tOee8N7oJmSszaayaMA\\\\\\\/pzjwEmLYcoqsKYAF2lhy1uoHygstHIOjTbjzihI39lHGbiug+S7K9xGEb8W+gRC6S2keNCNm9O3GzRv0G\\\\\\\/tv+gObDh5WH8iV0QkRDARD\\\\\\\/yfuRDKFn7I0apoiQBF6jOt4rfTqBgYonV0retLqAh+\\\\\\\/SN4lgVGWqkjdklSj7MHDZO3nDq9I8iFjBaYheJ1AHagkY3frvlRYDcfCrOJBRWP4XAIG71ndXXTQBFbqQr56UEgPB36nqRSLsxHu5pqgKdzF14ocpZIfh8Wu5Zm6PaW++7sindODVsnzaLgNL+Ai+VJEm42G6vzT9k39zG1OZlDgfaXw43TEF3OjOcu\\\\\\\/1PW5CsOroMpji\\\\\\\/MWFEVppSDM\\\\\\\/63VorATkXam9c7qopYd57dzA2qlnNW9c2k4qDJ8CBiiun64CcZjIdIESM\\\\\\\/MqRZiVGajsodMjtxxdrDuVf7BiG5DKI6DIkqwh02C7iDpjLthAXxoxyyRvOAc3ICa9hQWuDAAvfIuZHa2HCpqsYzpgJy8PVcf8f\\\\\\\/JYYltX9w9sxI8j9fsX2wL+pdMJ36ZXVgBB7jCRh+XuPjZupnvfuNvZSg2yVMqC9XFAywzdRwRsPMriMbDUZnceZe4iL2UcnxEOjGeC3BlKlWu77HFoubthSVdjXC3wtyFdGTK8\\\\\\\/8V\\\\\\\/H3zqxEB1JR0qiNMkBUYlL2E\\\\\\\/4agNOQfm73p68nfSIMBg8KzGXCccHk0bexc\\\\\\\/JLayvLpia47IIV\\\\\\\/6Q5Ui0blYqRrLL1XokZ0h7WJlIyfyvKwSo0IRfSTGE9Zu31TqrivHqJeQxO\\\\\\\/SfsOqRskJIAZlXEG4Y035vCFdpPU+F24Z7LYnajc0A3YZMMINq\\\\\\\/rm6a1GeFV3SR3HYID4t5CRBvBuGIe7l32j\\\\\\\/hLrfGxD7AH57FS+pV6iyLPdbxea8OcFr+kYrniHkQpOtxh3VSjKN\\\\\\\/8tgiUWwXaGawhOEcFFPmKB5jGrvUSb+X\\\\\\\/uVXVVp\\\\\\\/raEx\\\\\\\/57Wp5FzIIHw5e2w0GfbZl2KL6uR8rkIUZueI1CAUO7Mj+6mw8R81RaFLkxmUNcLDPB5dXhRYUOIVLq2xqSidsAE\\\\\\\/GkxjP2PBjCKi6DuYjVp0wJUFdHTgYmDCwfZtI4cd4REtKuZNvkm\\\\\\\/dUAJSX8UgmHRl70MyLuJkmrPm4wQB5slruB4gRweqo8boSOq27ikyGxGe1fjywa6+GsE7vZpd+9VT2dExEl3Cu5RtcdrlTPlNtsMIaVCFzbkXIafO\\\\\\\/oAxpbLfDSf4L2+AHeNzCd6ux61M6rDxZXqjvPOCeCS145LP1gLk64uECE9tX0pRb0HSOcK4GgN8COVzJlRSvUZLkYjaKualdRIMyHrFARWv8FROP6A9uFWGsA38vdbek305UcK\\\\\\\/WZKzJENx0WYRaeWj7VB1hmAa9tBbwLbdai41sR+wh+O+1Zg\\\\\\\/Bf9g+QIVydfk4t9xIEkYhCOlnYfZbIPlKAkbo+yifPQ4jKICCLCvq5Ico5BX1mxZvfCzYWZ3vwKCkmekVZx6leWiA3DFVyRvedGDK84nN2g+66mliT77OaYl4B\\\\\\\/K7SwCVOsgfGOSSA2EdRJtX2wXaq3GhFrsGZYe+A5PFs5G7Jz6AWNCwGi3ZnlMGZNla1suP\\\\\\\/Q+ZwbFgDdke0EIZP1fJBx+K3zd89LHHwbX5kX1Nsp0fbq+eoUmuNLzl6SLH5EhIo+qTg66s+wasrmgkFKlO1kPxRvyZYhrI05V56hbOA9deIRUaeELyejdPPfsTkx+sfzystWR1xwg0x8xdfWzpItml58S8r2StuTQtiBJxRrPmEMnKJjEPdtqrXy\\\\\\\/akXiJekyKuOWBU7X7ZYfYn9EwwvmJjB2l10qZaHHbx0Zs+QKCLVfNQvSZQXWn14S0VwPrnHL0RcrMKaiZYVa0AxOQSBf4igERmXGRlEbQBJYyYT30R3SMp79sdtLeNQZ7UhHyf9Rq2E70Vodgu8Bd6G3qu51OCvCMyBpKKNhWGQpw0OlFaAYDSp9ZSIs783\\\\\\\/iCozyxCouZ00wsvJykGzjac7FllrI1PVKL8JNuwTiMUFCeKqQN2dX2HN9g97rIvMUQlMwxPsKGV5c5QCGrdkWMgfEkv+MkechN7Fo88j4U+IrvB6FqusldV82fkg9dvvSDrGpcQAG9av+XEFP\\\\\\\/vHkRD2fP7OxyvMBOAltFoxT19jTecUmtum6K3JTEsAXHQyNR9csEp8cGiSlwNEqMdBs0jwcTRKc+vfRFk6i0IFkWB36vxPrzmLwXGchjR6cRzTOqfaVsfi2r\\\\\\\/ef3oYr9SsiLbdEkEBFVVt66bF3\\\\\\\/MYuSTUfyaPRtVzCs448sCG5nmjXF4Xv\\\\\\\/GsiPs3GYTcg7u2w1s\\\\\\\/zO7zytAp6dj9i6bGUx3yVxUR70r84CLOIyTk6tXCGC7ptSJU5Zi8kzvto1\\\\\\\/+w6fMV7z+68jLlcc5QZAlaGwQyqvOD8DvkBk0cz9DN4xSM40vD52jRw46eUb07Q92yb9al8w2yDtuz0M314EPUjQWicOCf3Oq6uLJQBfXWyVWsmUgRFT4pYrhwJYCx7a6mxq4gJVfF8V++IUiRhS6ynUyNgC6n5tnPqgDhgMOPDiDPZ5si7up2ORUYqDms3kqPcuvnVvDLOMwVj5h8Yuk3vpTHJ9XKn8LOMsfKLQOks3Br4bymHHbisrEZ1akBkBNbI+PItgxunGFdfwKV56dJ5QqjmA3JCXMMA5Lk2\\\\\\\/woxuBk5KWOs6\\\\\\\/cy5cr12so2sjNJKi\\\\\\\/IY5BRM+NgjqHfOXpPZ4Q4ArS2cMtuLP8DUS3nON9Br0Jfi+Z81t1GSO2Kq\\\\\\\/PVpJ+tunUYrj+j9Z2YkXCxXdVszdj6b50FaFhyEyLpwhEafCqb2\\\\\\\/8QkvNf0tSvx\\\\\\\/sihHlfZPFHU4WlzAQvTJ2FckbfXsKnD1JWo1NLYfzjmcEPlMvs+phTuE9EU\\\\\\\/MvxKJGYFwDVc\\\\\\\/UWrKCb5jj4T\\\\\\\/uIxMyt\\\\\\\/W1dUjyCCRsPyU87m22f7uHN4nWA1P+2X7qY1k81kgkvL85DhL\\\\\\\/wAVlckTxkCNcQ\\\\\\\/YY6WFyYPVktJaRK9S2F5m4rkhnftWR50BFsADdtdBGY0+E\\\\\\\/ZBCGgUP\\\\\\\/IAVd8J9hox+Zr2SOwPKl6YuUaUkpRe8YkixFCYLtBdNub0qxPxa75X+B15y1l9NF\\\\\\\/FZ5DAdsnbZ2YLrUcZiQ3ooErubwftOWwWDkaBo4daeLa9nZ0vkcUWUC6JGWgJ4qxkC3BWivH+RPNMcq4ig8RXphvCxuR2ht9Fw\\\\\\\/lz5VvifxFS3LiLBpf1aXdDt\\\\\\\/KfWWcuBRevPvCMZX5FqZxGPW9gLPSYiAD\\\\\\\/PK+OJDKObEETfbpVE7HbFrhy8P7naQYYkIl9c8YkdwMHDWo5ZiPtXxkaEVHLuIKs19PeUsF3D5DW2CjJM06pBtqlfY92WEVpEUMxwTNy9l7KLkEZO3O1WxTVkqXVaNLAUlq1AoDzXkc5foNNrq3vmJ8CQJcUGovF9y5EzKfxyoVcFNb8cCvT78X6SuL3Ey\\\\\\\/OscNL+q769yzc6NAmn3+rdzbACQm6MTVhGqwQgLeGwrlVoUkKH+QW6Ww4XvOZhy0WBY2ULGoiZU0DG4F0z8Qj1bfr0ESdVH4QUwOB7n\\\\\\\/Zzevy6xtRtSwZdfgxAODOf6EEOuEo\\\\\\\/B6lR2x30OVVdre944h5EEBxiKtW+LP4D29OfKJWHxYS8dTs4ltgIGtTI\\\\\\\/LieP00bXJcJlL+XHddtxuUnCRCD8vRi0aIsclmepSqAEdIWkYWL0HScvoH3Yohd76ltuouDdig7VWCc7gPLcPbwX\\\\\\\/LCraWfMU550mhXNxIllE9qPYyiKQ9wqPci7nG7qOQ3y7aEoPiGwQgMtFl5wBjww+YLm8iGUnv1Qd\\\\\\\/4nFCYY+3+EUuMk9NpLBDIvG5v3hlvSUSJdpGh2vaQAR6Ki\\\\\\\/hdSDcC53s+YUAGQKrSKQXngU7IEwyeVNWfnm8ipQMmr2Xziq2SvPGzanohCl7vI+u51w0GoRo3iWjO7Ewx3Nm\\\\\\\/YFwbfl06M2YM\\\\\\\/tV8o\\\\\\\/EzbFpLW6ivP5O+WoNj7DHwVOGsamhdG4oIz+J\\\\\\\/03B77CjsKWH4HPEQVsDhCEa+Xn3zijNiWBkUUTLfl5JNBny5+gWoHhFIS3OUNjG+is\\\\\\\/K9DKtzY4BWxBVZgGV97SImYvijchjcFnyaRiNMwk\\\\\\\/e1sDYH5xKL\\\\\\\/lcFdLYqyTBohMzWx4bMDYpedlkRveUeI35bkfQUqM\\\\\\\/IZrf15wDGX6w89\\\\\\\/VCBvrv9VsORVt0F2ySQ1zGdBOrrGvdVJq7UpAisGSww6URlEZ853el0Sv0tWy0r3EEklx5X1ZCvoHWaRNbK91Okhj5Zxb2eY1U74WsZMobgHQ4JVojmdcwEx6uy7ND0XuO3PEagIH+TyZoOxN3Gn9AvTIZya5jQaaYiLCI0oBsaivfkY0mEQA6eFNXxhbf2jHbzWpjvStbT5+gZqgEtXcgi8\\\\\\\/lAHghkM\\\\\\\/rCrgcPBo8qeIIQem\\\\\\\/WBbF9IoL4wtggad7xU\\\\\\\/sQboKI7rZ16gPeP2Bz+KbXDZ5T9IDk4y+zBD8bbd0BSQb3zLdBqxpFQoxlVNOJ4havAKVibFkKtcNestanjraknfwsnqOk8Pe4Ay\\\\\\\/4JmlXg7R0HwmBHtwlCAYgxjpALltXKzaltpwYW0129S0\\\\\\\/Yvr4\\\\\\\/WntkW4qTcxwnvg\\\\\\\/64SQwDcF37Qg1rSyKhuAWbW172dT5ESCyLYnl3PnHTqy4+T\\\\\\\/MEgXRPunuXj5QFKUPU0jB8pgxQPbOH\\\\\\\/e4Hh7HINX0fxI1pgXNKMNfW\\\\\\\/Y9KXeZezFtwF\\\\\\\/mtm\\\\\\\/rQCsFjVi09Bgd509KBIXml3yR6GWW5MgTZX4qe\\\\\\\/QU5b\\\\\\\/W2w+BUpgQYebKZc2fTJ7PwZzz4rNZWLOjuccxVikagwFdnrbfFA33uJibKHkZJNmQG+6VS\\\\\\\/chh0r00PA5QM1uhJAq19BPQfZ773oKkdHPlj9jOLbzilo\\\\\\\/2rA1NIyFIIEVTMiU1jN\\\\\\\/ErRoPcuRmcOASnQxcbBQex4a\\\\\\\/87PYkek+CffeAWDIDQEvATJ8JITUBzIcJW\\\\\\\/DIcFlE4xBUwv3o6fnV3fePJDJALc2kf1PF8xo8NI7Mz7tr7Bzk24JZ3pNjBENXUePvfPp6BVpSKUSW0\\\\\\\/DfnacglwRKTb8Ee+uiIMb1cZbRcSOAZLAdOyfrHq1Vweh6OqadTrAQQ5MABY7J\\\\\\\/w4tBTGch3kazK2zTyXNSjI9KfVpMORI+1ojYpLcXjmN4VLK90DYjKcCOf7fM\\\\\\\/UWcakktM7zBU5UOA4W\\\\\\\/eMqzsvUSB82U9V\\\\\\\/ezq584rHe0ObImzLZBIzB+HuJZnanqVS1W+lliLBd\\\\\\\/WHo4H1ezyZ\\\\\\\/Fz8gOJe7MUp0fQLeBhv0pv20mhAagD8YHBdHZpW4ALd9YNAPDHRyB7wh2Dw+FpAbEaD4ifsB9YxMAdRrt4vhgy65u66lQ8OHIHj0keSlh6WhVSBz33Be2SzuRwHnlxtCgi1dUavvwhd726Z1Qt8rCr8Jsi0FmULX86CaH6TBteLkVkVTCkEaqmv2wU7AucaN30avUel74SD5jkH4AALmZsEJCawAMXpps0C86nxxb1AgbBL0if86V+TY0Jy5nNXgtXUPd5vn9C9B6R5nrjbOC9EyTuKPSl+xztVwt2C67VKiQ7+mQDReYsyuqfnKYoR4\\\\\\\/kwP2QYnPYdPwih9lnLI\\\\\\\/NpkadnUfLW9nKNkqBNIi+w\\\\\\\/qbQ+mUc98APo060tGxJHfVwRvgtZgfuU\\\\\\\/SK5RWHZPvXON61DNmafq7LqL1UAo220BM4x+0LN5Zbhfbb+4SWnpFiuJCPT\\\\\\\/Efd\\\\\\\/2YL4El98KpKTrYq7waOLTy6tczZRWOjaovPXl2CYshA\\\\\\\/hoIxLFtK9xVgqXhTecEF+HsG4\\\\\\\/ABruOZvuapszc9LUY6DvL9je3Zitwnr\\\\\\\/s5eU0tucM46AwcBSyqn2Wuoz0\\\\\\\/lR\\\\\\\/iz9TY6nx1eGwP5sV1hLvqpnauQgwmLzfk\\\\\\\/YME4InWNpaKRwim90ySB8i088hxy3wS+aSlulvxSe421FdUVJOwzYqPZyyYvg1iSq+6rY7bXaSx36wAEEPGbF+kxKjv0GVJSZ+XyJ3s9FoDj7gCQTtHzYQ0k+1Rw5C8\\\\\\\/DaRAhTQdYP0ZM9Ye9y3F5fzqWYTEpzyoAW6Sqz7MZVuYHZAxHj+arGPv4pbUKBPqy46\\\\\\\/HfZDtTCmhrms7kvePDeWTEfSwyrIy+gI9uOHRISXz5jTz6e4gK5fKwIxgyJ4WURDxWW\\\\\\\/79wt\\\\\\\/U1gGiwEIHe8MGc56ZGN1Ghko5L3vXQMJ89ecu1flE0pRmC2gGq6IMyj\\\\\\\/FOnHJSdp+H6I+8OsvGYUPzyJUZFC3JbkYHLSMikTg5RqZpwlhiRsPmUXMo6M\\\\\\\/jAi\\\\\\\/thAB\\\\\\\/MFA6pJ53D77h81qsUOYoVj0J1BaENf4Ops8jSOatUahsZ+kJmr9GM9JdyKo5SMkiBSx6iax99uvEv2vKO4Cr6wD5Aq3j6OupyPudiMoWGAbHntqIxRUNE2TUriUonHL2JJfvrZY4VNHgMx+JEbdBPdne6ivS\\\\\\\/EzsTdAjjHvG7KqF+kuSlErItLCC\\\\\\\/PCK6AFnyk0ex0Iv6m1+XbN5dccV5YnLzDal0pS5MzxkmzYTu4Lh5c6FX4goOw1t5ZQl6yW8gUMbhs4p4EC348AZC\\\\\\\/rL7qwMO+N9pCXxI4bqu10lZQT8ayi1kkVVWBlJKf84Wg7MAGRkGyosiY+smXy\\\\\\\/czZ6lvthNTQCiaXcJtYyIgKd9hiMoxUYN8ZugcCUEmr+guSY2qhm+H\\\\\\\/C1VGJZAFQTiZnRS+zp3A\\\\\\\/ExT0LTAAGxS6If2xlmRJf\\\\\\\/kjpvTRkhcYIfxb\\\\\\\/dq2PhJWn\\\\\\\/uCtRSUPXJ9kvcWmmO\\\\\\\/xbBcNV2K2nJs0b9kjXq7PMroo68sRsIlk4abxWb5xHSy4Loxy1quCRuA5p9e6pwJnN1fDf\\\\\\\/y+qFpUIC4uKPBCuDQlwCozVkG7+62pkw7DjDQs\\\\\\\/JJiVRdBJMslLKFH+mJMPgCOyxHE0rhU52N\\\\\\\/\\\\\\\/v3Nbe4m+Lq8SZiJos6a2OarPovjKnJ\\\\\\\/IYYwieNVN01qpnQGfHK\\\\\\\/nIUgarWDczKSV2AQXFmZhsNAw5p+CwMaXgtHeWiugYVxd4DVhikkDf0MVHvfIzgA9aMWYJxd4hm0Kg4iey1N+xveLppPg6lYV89o+\\\\\\\/12YREzHaT3ISxitSxL1dbVMYlc75RX9VaeUNNwB5WwOb6Vve5M+zEApjoBAeIJcnbxtyd+cHtQ+ogcEmrY2oD42olmw5zQWDHlOeePQ2iyU+D5iGLqHI6lCz57Y\\\\\\\/99LAYnjB5NW4\\\\\\\/HZhjCMj+U9FBWxN4WFIAUBGh0FZ1ntPnBvmBTgqmvw0RJm+rrUjLWh7DNjimw4mUKPkVu5Xd\\\\\\\/OThmm1rufivbF7Z00zO1h0P4FG1wOlI6\\\\\\\/Id0s\\\\\\\/Pt8F1tRhgiiA8kEiYU\\\\\\\/O1XA6wNsKs7G5PtrOs\\\\\\\/USKuCwgOQhB4YhmJmlGcRlm1khDzKXJlfFhqZE5bHqK\\\\\\\/qhcmO2jsITBjx1d2CCilTaXpnz2JSKQ\\\\\\\/AXF9RhwI5ghMrgznsfQgIJBPF8ClmLEGZsYJpoMMIITfwguK+g0GR3UbeTDfzfztkZjyPQC9Yd0bIX9ttj\\\\\\\/AaZFCfiXjiPIJ38QqUAmyqR4sfuXwSuAEYAA2D70sDifGvZbyyc7MvozrdrwZ+en0kkBEmVYjqMV7hoyrPUKNnn0pQCZ0uiiQSUmKZgVo9yDx68nYvUuzFugTMI1YfI5tTWFnGpJl4LCQglryH77I9P\\\\\\\/YsCDp6pYPvGAgXXcXVrql+uRp1V3IaurmPX7W7pphVyx0FwyhurOm33ImHlyV88d0Y0rN5IcjPcKX260iEItIgajufnHNu\\\\\\\/96j8oKXyO\\\\\\\/YnI\\\\\\\/LjFB\\\\\\\/J32\\\\\\\/0INjP\\\\\\\/pKFYdoCGQf8VmIiitF+jdVwHwKyQPV5tKUJNKD96kRly6qGrQA9f4d20+llBC8SMYVczc8MnZJr9KSPXXExwr319n3u4+aJvgnFDy6ywE1C5E4nuop1URO3pRqrz7\\\\\\\/aL5DeQRLOf90ozyXnu4OKFhFoi5ATFP3Uoq4Sjtfvh+n24BJg\\\\\\\/3wnflRNEomXu96sBWEqa00H8Kr\\\\\\\/dulIGqGovwByQ9BQMA\\\\\\\/jhNxA84EUaAQGeYzy06LpVPfh1Y6JqblZxmom7aWaYsDXqsJou63SFeMbVT29i5hJiIZS5yZnE1ZVHqAueFWHFWwqrqC9lwRrweqCZptmn\\\\\\\/Fbghe4g9NCyht6zC9+gg6TclMQtn2OSTOW32NR2J9D9S7AYnQ35gUV8Majk+YS8J+ft4KFiY+4OhKIovngciMkgf3ISdyCLO5NXo2iRziBbJy71JqeaN5OHSyKEGR2MhJ5ggbfAoZz0kabYPInpZ1iAcnavajxiT2VXzFLaityvGppGgeEQwZ1DW4SApHGwhvEjy6lH8Sj+Dtp+4oRlwvTgEesOYJ\\\\\\\/5b1EXwnNDHYsNNCsmz5YriSWsRRb0fyvkh4q2DGI4fB7DSSkG+nT+2fuY5xk+6BrtwvtXesV2sYNhYJogpyOKaIewooQCKfuRJPaIJrfrzDAxgHnmVm\\\\\\\/7fkkm+ZgZ6tvLlSHKqXBMxD9ikve1KvrtEeDxB9sGqugxss4hJc4IUp9ZOyrHD2XfmVB365ux6uwjz1xMjCaQ5Del+1umfnhEsOBVqtqn5SjBWppldTtIVuVxgM\\\\\\\/txsTHF0E29XXOwX9OS2f6tMeXdSSi0juql1xtN+Wpoz\\\\\\\/Re+oa3rxbIubI5kFj2AXHiCNxzW8\\\\\\\/GpkYEu9hxo61MLi3BmkgSwzz8AabykHg+Wso0stT5+kI4KnNtKhUAzxY\\\\\\\/\\\\\\\/8bw5XxUlPLaFIHS8d8t4SciCx85V0GFGHy9rnLNOBAIEadzRRU4GSy7HeL\\\\\\\/cmEBPkC22wU44sSOzECnGbI71iyfuFSeZVvARmtKFKeojKaktpOx8zZ5M+dK9npDb8t+czcjT9twLDM7xDqDpRFinx\\\\\\\/3r08MPIK4gaws35FR48wmyHvnE4fX34EckvSa1YBoxWFF+3R2jOQXIKvzQ3\\\\\\\/wC6irQVX4dQ8tQOwAbno0lVuNOcj7CAyoXrLIUwpsY2Fvf\\\\\\\/z2PY2fbGsz91dqFnZnr0lXKF3mZNsg\\\\\\\/nqEvlwCZ4CTNhlYfIJXyybwjP8Zgr+hOLAv\\\\\\\/UW3AWKHsYFXzU4TSYLbwx\\\\\\\/jPFIfeWS7wW6J9QaQim26g68AwGOPGywuX7TnfUJp\\\\\\\/DF1m8fm5Sd9wroyhVKPbP29CjDS+MKZePxhTXURz1BtBb0kSSKg71f5FzwKtg9yv29DFPvu1sSL4Ijp1qtyZqhtJsakNVmfDnHKCP99mWLSnjfK\\\\\\\/9d1OoKTs2163xGOJA8FxaLI+\\\\\\\/Pvz3NCoyTFBm5sKMSuaVM53Ik26A5T7Yr94j2t7AGot3WMIYZVfQYcrZiDucjO2WghA3A8108SSYwCTf7gIKjTEiNFwnohYBSxhHZXUhnL\\\\\\\/Y4BebI+t4rTJ\\\\\\\/CY+s8ccA1mVucCSLSOZbqbVBZ3TST8DPxC3Q645Deuyr+AA4YmvZaE+krKEtdx4yKZEjTLKTG+tiVlfm2V\\\\\\\/whedMIqnSaJ8NvbYoiJr7TRu+haz5Fz0Iqc0+WYOGKZLQTGifrPSHC4olSL6lay1454uWt\\\\\\\/0KkcdfUVKzRFVZ8XlNA3qhxcl0qskIgrQ\\\\\\\/gJcRE5P0HsBmQt5Bpcd82r4CPkt\\\\\\\/m5XXJuGQWlvqbZ09jrU3H6aczmCZNKsV6btvOD5ze\\\\\\\/kBWassl9vLmFQ5clbeOOPhE9wdtZICoUWy1XFk45RPl1RbPAjZn\\\\\\\/qJz556\\\\\\\/XtSMaqSUlaCok0NlIeR2VuwdlsfN2hbY6Y8gAXmpH\\\\\\\/nG8YATFhpSw+sAbS\\\\\\\/fGmyTdXi6VBmtx5KQ4i0S0+a7z+GqnUjygTRsjsB1lXNarTfaY5lRiEBWguhz43LbVCuzP0c6TjbKQnoPpD9dqS\\\\\\\/9wxo\\\\\\\/xBVMg0Bt3GUkfwKQwX3I772CZDnBSpX25wHf\\\\\\\/kZoI22lCn+Xm7LTvPgg6nraPx+7dvdGYjkUz7uxf0AK8ClqrKnDlRf7c1S3kVFjGQbRZYk7l5d2gOOkqHwmLDfXwJWTJO7MtFMMXIe9NPClj0FRsARezOBJvoT929sF7P+ccckt2m0VZlz+V7rMbd\\\\\\\/95OUUF4P9XozFQaphd4B+ndwqGalzhZNVj92WeC8dSglGDPILpqA8l\\\\\\\/XgmiczGG5TWsxG\\\\\\\/YJe3+9ny4FMt2CvrSAmousrSVFG\\\\\\\/+yjuiyUFpr3udsxfZ2KjFXAzvKwg6Qlzj9ktncZALg1F2dRPV01mb1qRQ0WTW6udvXNCJtMnAwAU\\\\\\\/oDiMZ9QNIrYKAZ2cPSDpjZI+itg0hoPEFFu0kbpNq58JwkyUUxiHJXGw+grgtPPkBGkDJH1AcdmNNRkr1hO\\\\\\\/jvN6tgEYu5ZsSgSB19hPdgWZYc7q8wvUY1Sjm8hyLN9f1ixzPM6txGIdtx\\\\\\\/QQqV06EXoUluUhQRO\\\\\\\/gEEzcbQ+lydbJvFgnnRdexaF8fzdLdpStbYvKBlEx\\\\\\\/kL0FIO4HW3d5XbDI+\\\\\\\/Nm9\\\\\\\/2DXNWYDqsMh2pNq7TBKz15WKeSqF+YvzVm0EvBvntFmDrGjb1JB7Hm4rXTJ+xtw77ts2T6XvBZgeN13SnGMRzsPVSiGJF+taF4N\\\\\\\/CbZ8trbVX++P2fuaWhw52UUjn20AaWu2NWaz+6THgyjC6fLU14s6aLT9hCR1SPrtbIyj2dHQJ7Pxpazo2+CoYbLcWEqgLeW\\\\\\\/0myk8yHqZFcwlUbWVJ7rjtw4jMgddX93uip0kAUfWBcsJC8y1qSsTQvNch1i4brWpEZAiYPOCgSkLA4PvDVvzNEUg1JhEsHjQRXznDcUntVNBdNBg122zb1m4YcLlBLe9LcwebQUoE6ihf6fCD6ZXZrBUjGjfy3lZPUjotoL3PxIyNvAI4hXwH4OmCNvM86wkmGzXKxTfjDw7tCFxcOBvSRH6xvzePJAVNBDnQLLuwCpK+oFQUrhMBMKTXAFO5LV9gohon1NoGsON6ij9H3o4yLUpRRF4pxEEL8VKy86eAGMMzq2ydvjERVM2HCr2flt8Y1dk9loESocxTVHWM9ZDpHGEuID3E4h02NWMRcfme8cJiLur67vwh9TRGhdJxbCu9VF\\\\\\\/LpJJaGfo5\\\\\\\/QC9VW87fqWnBhkuOonMCyXiCJe3x73cMvW02o0zEukSXQii+dB0fK7Hd3LXR5yJI9zeJIkS9BaoE\\\\\\\/48ABecSIBegpgVOzj7PkzatVCcs1vEA+tQtDLlIjOW9SB6omuQnwjWfj4pAQnsBxvyEjkcXg1KVvsFUREcz+5hpfRt5T\\\\\\\/Gv+5fQd\\\\\\\/yS1kRhczflToWDVuS+M63Hizax65bRFvEk+oCMwCSq5cyRpittZIEQHTPnnFrQM2NRXCqk4BQM+OpbOmzHL+X5QlI\\\\\\\/szSZwbfVrm3iHTwGvfiVazDUfMBswsTVwC68gdUMZlFzr0QBqOiiWvRgqVKFckD\\\\\\\/i4rDX2V9LR+qzqPaHmBtv0qnoAkTQWvPXQdSEFlym4E6bL5VuehLfFIpxKelecc4EwTD3K5yC0HxxhCSteFpQsLjfUgxCFCo2dxe3DS9uYgjrGCMbv8ooVyGD++Zqw7\\\\\\\/ZRiS6wJ3m9nLykFrdH3VwdVZSfp6CW5rOAureTQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wlsEWb8awn6uLCSJf8LVBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221c2e4addcff9af0c0b440a93e1da28892d7788d3533c890a1ffe1caf492a4e2d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nLM6jNNG0P9WcpBIcn+p7zBq0ycANzffsFGvB1jOUdWGamd29LQHLsm96pIaiPbCO2AYdVFkt1XudW8WIZ1NowE8mBJgYozYHATsFyXcLVFIF8I9d16lRDZt2RXQkhnUjhfJ9xCJnfA0ZfbnL\\\\\\\/EJKXkZopFQbfxmiJ3YBJ5KM3U\\\\\\\/irDnx61jVh4Pr2IpOvRD32nNq825Zef41IGrMB23bPiL9QJeEnjgDWHpHxTLzPqk+5AfpoMDCoBcxCS36xUIPr+biok5pDJ9SIcoHcXDyQ\\\\\\\/0JZX24g4+7uhalhCZqEzC70XpTrZguuvC5sBRtSLohVjKyHEHZLnm11F6DEVHRXpu8U6lfgJ4v\\\\\\\/GAMoYnN0BLAJyYdQIWLj0G4gp+NLfTGSrhRgV6UsAwABLFaNkKnbsqqRUV2z18v8WWiOyG3cqce9D+yDt3OdZDxh4tLOdFWRLN+o3abs4\\\\\\\/2yhDVBBf93mPJiEYs89EhmXx0nmIDuuFJ98986HpK7jnR\\\\\\\/oVaq9HqHsrAMkevCuqlrl0pa1QhNYa2KBTyGOcgZHllr6fvWFEAoxgZvXI3UhaaZHOd1NfGG5TArtxY4JntsY7fIWWwhjYY2XjbrlyvTbPEGBFVAch5j+ri1p29WyBT95tiRo1VW5EYsqdAfkP1MeFXewPMz8Uhua+lO0NZeSN\\\\\\\/L7pw5ANj6pGAGpi6NK17t0Id0RWXVWD6GLh4KPvJ4pSIlaXlrOnbAXHwQBvX3CnZjpiVLgjD3dFOgiV+EiGhTgeGSWhommhakw+69Gode5Bql6KeYARYOzWR+LNLRxN7lS2JUPaKV1588JB6jeTw1d6TFN1mE+T0X2vW1y5jqKPnm3xtfXBRu6zd9dSxujLzD5nKdrjPKB36SDglFm0UvpekkONTg9f6UV42sbpDg3ER08CV9AJZmIRqFkNxCz8uiApB3Rgn1TqqqOblKAbpQzu7NEaEEf\\\\\\\/z2lFxbXJNR6AnsZqWswcfd6fRBrD6Mil4eU+84IQaBTxYJMeBPlkL1We1QVT3jB8XcIur\\\\\\\/QA8j4tVQFa\\\\\\\/WQxa6BmADIbXm5LPn\\\\\\\/z81gCMfzfJyHYSLAkptfpcjEeigIhagHQncbbyL83Sedf6mbtBGv\\\\\\\/PFzFtZOopWZGhshIQR2maVQWah1f9\\\\\\\/dHQOQE5ckrN7bodUR\\\\\\\/1eKwoDsKuXh17C3a6SqiwjKbPlH9R6c2X7JopxpSi3zoL2OoAbjfoxhM4jETEM7j+KwY0iL6Snzok4o2kkITcoKlN2wpWYgpa9UXm9YfvNG6ZXrrxswFlO8b0faAsxMOoS+nGoG3Ddez\\\\\\\/X\\\\\\\/KbuiAMxOc\\\\\\\/Ik9a64Ong58uc06Tsw1upm\\\\\\\/kAVPHfMWlPmVf0Lbh8hiGPtwbiKcSUjkhBOMIZsfdWdb2q8CP3RGYhX3KiQ4T+heUllQQV8F+xPX\\\\\\\/WS4zevw1hULfDigeJPUDjjyzPQqETxxLgBEWdWCVBIdBOj1Hjae7NrHSoG6ApYXXiyKOi4TZ1Exug0wKJNLHgcdssxrjVX1JTbYIgjrmZmw4VtDwtXfcsVbNx2GoyXVROeLvNycNuV5WXWrJGNnFo8MlelEZyVkodnWtiNfwtr7xNgrwoxAf4PQgqVfZPiI8Jz1\\\\\\\/M80zKD4GycvhJDUy+IYwYr9o8bL05Eof\\\\\\\/asoPTVGMESMAYPs7Q7W1ytHsgSzVbrekNtDv4vtarIalojFtLNhLtgR1dhRojD3bGvRLQnPIbRUurafPR6kdMSSFnFyLG215Pmy6tIDY7YRAh70IARwXS5Tsk3fYef+SVVewfSg4ZRQw4rZbyo\\\\\\\/oR5Y\\\\\\\/Dy29lueqG7zhZ7QeyEQ+dqd1yI9XY+pdy552y9oK28VFNyJ2jMUlMMlw5vuoRaHmwR6TC5nysyGA0CWjTjpo3WOS2JxGbtEe3QVHc3yY\\\\\\\/0uK\\\\\\\/mn+KZTFGkeb53hSEunJrfaCpa92pTSGiOp6VXeMxtnVmsDz1ODrfUNsBZzDTzkhJAWrLBgYEdgGyUD1F\\\\\\\/Ipu+2UNeL14w1Wr9rg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wew\\\\\\\/HxylVSrmmxqi\\\\\\\/yQBog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220a29889c82aea1f52dcb8398ffe2e7734b04f4c7bdf76413a7277f9bec5e289d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459515 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1429" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nTyZtxSld3wAWGsRHIvZ8558wxzui2LJBys33KJZi08" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YYJO2B6iTrnQ5SINwWLDrlzieMWvHnfNC3SYogsnJBM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/8/1419" title="Variation in Meiotic Recombination Frequencies Between Allelic Transgenes Inserted at Different Sites in the Drosophila melanogaster Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/8/1443" title="FijiWings: An Open Source Toolkit for Semiautomated Morphometric Analysis of Insect Wings" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459515" id="node459515" data-pisa="ggg;3/8/1429" data-pisa-master="ggg;g3.113.006197" data-apath="/ggg/3/8/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hua Chen</span> and <span class="highwire-citation-author" data-delta="1">Montgomery Slatkin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1442;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459515" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hchen007@gmail.com" class="" data-icon-position="" data-hide-link-title="0">hchen007@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Montgomery Slatkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Montgomery%2BSlatkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Slatkin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMontgomery%2BSlatkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/8/1429" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459515" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459515/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1429.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459515" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459515/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1429.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459515" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459515/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/8/1429.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459515" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459515/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">It is a challenging task to infer selection intensity and allele age from population genetic data. Here we present a method that can efficiently estimate selection intensity and allele age from the multilocus haplotype structure in the vicinity of a segregating mutant under positive selection. We use a structured-coalescent approach to model the effect of directional selection on the gene genealogies of neutral markers linked to the selected mutant. The frequency trajectory of the selected allele follows the Wright-Fisher model. Given the position of the selected mutant, we propose a simplified multilocus haplotype model that can efficiently model the dynamics of the ancestral haplotypes under the joint influence of selection and recombination. This model approximates the ancestral genealogies of the sample, which reduces the number of states from an exponential function of the number of single-nucleotide polymorphism loci to a quadratic function. That allows parameter inference from data covering DNA regions as large as several hundred kilo-bases. Importance sampling algorithms are adopted to evaluate the probability of a sample by exploring the space of both allele frequency trajectories of the selected mutation and gene genealogies of the linked sites. We demonstrate by simulation that the method can accurately estimate selection intensity for moderate and strong positive selection. We apply the method to a data set of the <em>G6PD</em> gene in an African population and obtain an estimate of 0.0456 (95% confidence interval 0.0144−0.0769) for the selection intensity. The proposed method is novel in jointly modeling the multilocus haplotype pattern caused by recombination and mutation, allowing the analysis of haplotype data in recombining regions. Moreover, the method is applicable to data from populations under exponential growth and a variety of other demographic histories.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/selection-coefficient" class="hw-term hw-article-keyword hw-article-keyword-selection-coefficient" rel="nofollow">selection coefficient</a></li><li class="kwd"><a href="/keyword/allele-age" class="hw-term hw-article-keyword hw-article-keyword-allele-age" rel="nofollow">allele age</a></li><li class="kwd"><a href="/keyword/haplotype-structure" class="hw-term hw-article-keyword hw-article-keyword-haplotype-structure" rel="nofollow">haplotype structure</a></li><li class="kwd"><a href="/keyword/structured-coalescent" class="hw-term hw-article-keyword hw-article-keyword-structured-coalescent" rel="nofollow">structured coalescent</a></li><li class="kwd"><a href="/keyword/importance-sampling" class="hw-term hw-article-keyword hw-article-keyword-importance-sampling" rel="nofollow">importance sampling</a></li><li class="kwd"><a href="/keyword/time-varying-population-size" class="hw-term hw-article-keyword hw-article-keyword-time-varying-population-size" rel="nofollow">time-varying population size</a></li></ul><div class="section intro" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">There is an increased interest in elucidating the role of natural selection in the evolution of human and other species using population genetic data. Evidence shows that selection has been actively shaping the genetic diversity of human populations during the process of adaptation to new environments and infectious diseases (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sabeti <em>et al.</em> 2002</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bersaglieri <em>et al.</em> 2004</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tishkoff <em>et al.</em> 2007</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Simonson <em>et al.</em> 2010</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yi <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Peng <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Xu <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kamberov <em>et al.</em> 2013</a>). Selection in human populations can leave “footprints” in patterns of single-nucleotide polymorphisms (SNPs) in the vicinity of the selected mutant. Numerous methods have been developed to detect natural selection based on such polymorphism patterns (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tajima 1989</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fu and Li 1993</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fay and Wu 2000</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kim and Stephan 2002</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Sabeti <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nielsen <em>et al.</em> 2005</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Voight <em>et al.</em> 2006</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sabeti <em>et al.</em> 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tang <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2010</a>). However, only a few methods are available for inferring quantities of the selective process, such as selection intensity and allele age. Among the existing methods, some consider single markers linked to the selected locus (<em>e.g.</em>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Slatkin 2001</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kim and Stephan 2002</a>), whereas more sophisticated methods gain information by exploiting the haplotype structure of multiple marker loci. For example, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Coop and Griffiths (2004)</a> inferred selection intensity and allele age by analyzing mutations among different haplotypes along their genealogical history. <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Coop and Griffiths (2004)</a> used an importance sampling algorithm to explore possible gene genealogies. Recombination is not allowed by their method, and thus it works only for nonrecombining regions. Rannala and Reeve (2004) extended their former likelihood approach for disease mapping (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rannala and Reeve 2001</a>) to estimate allele age of a mutant under neutrality using multiple linked markers, and employed Markov Chain Monte Carlo to generate the posterior distribution of allele age of the mutant. <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Slatkin (2008)</a> presented a Bayesian method for jointly inferring selection intensity and allele age of the mutant using linkage disequilibria of multiple marker loci and generated the probability of data with an importance sampling algorithm.</p><p id="p-4">The aforementioned multilocus methods all require modeling the effect of selection on the genealogical structure of neutral markers under a coalescent framework, which can be done in two ways. The first approach is to use the Krone-Neuhauser ancestral selection graph (ASG; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Krone and Neuhauser 1997</a>). In the ASG, the genealogy of the selected allele is embedded in a branching-coalescing graph so that both selection and mutation can be incorporated in the graph. The ASG approach is useful for simulating genealogies under weak selection. For moderate or strong selection, the ASG becomes so large that the computation becomes intractable. The ASG method’s performance was dramatically improved by truncating the ASG (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Slade 2000</a>) to avoid generating very large ASGs. This approach, however, has not been extended to the analysis of multiple linked neutral mutations. The second approach is the structured coalescent (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kaplan <em>et al.</em> 1988</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hudson and Kaplan 1988</a>), which generates historical frequency trajectories of the selected allele and then treats chromosomes carrying the mutant allele and nonmutant allele as subpopulations between which there exists “gene flow” caused by recombination. For alleles under balancing selection, the allele frequencies were assumed to be constant. For positively selected alleles, the allele frequency trajectories can either be generated by stochastic simulation or be approximated using deterministic equations.</p><p id="p-5">The aforementioned multilocus methods all adopted the structured-coalescent model of selection to estimate the selection parameters (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Coop and Griffiths 2004</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Slatkin 2008</a>), which is also the model used in our proposed method. But the approach to sampling the allele frequency trajectory from its probability distribution in our method differs from these others. <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Coop and Griffiths (2004)</a> generated random trajectories of selected mutations under the Moran model, which has the property of time reversibility under mutation and additive selection (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Watterson 1975</a>) but works only for populations of constant size. Rannala and Reeve (2004) made an assumption that the historical allele frequencies of neutral markers are constant during the whole process, which may not hold for real populations, especially for markers under the hitch-hiking effect. <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Slatkin (2008)</a> used a linear birth-and-death process to approximate the genealogical trees of haplotypes carrying selected mutants, which is an adequate approximation for mutants in low frequency, but may not be suitable for common mutants. We use the Wright-Fisher model instead of the Moran model to generate allele frequency trajectories, and apply the importance sampling scheme in <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Slatkin (2001)</a> to weight the trajectories when estimating selection parameters. This allows us to model the selective sweeps in a population with time-varying size, and allows us to analyze both high- and low-frequency alleles under selection.</p><p id="p-6">One advantage of the proposed method over the existing methods is that we model the dynamics of ancestral haplotypes under the joint effects of selection, mutation and recombination during a selective sweep process. In contrast, <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Coop and Griffiths (2004)</a> assumed no recombination in their model, whereas both <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Slatkin (2008)</a> and Rannala and Reeve (2004) simplified the transitions among different multi-loci haplotypes induced by recombination. In particular, some of the recombination events between different types of haplotypes within the selected haplotype group were ignored. This restriction can cause significant bias when the selected allele is in medium or high frequency in the population. Our method explicitly describes the frequencies of different selected haplotypes over time during the selective process. Therefore, our proposed method applies to both high- and low-frequency mutants. Our method also requires some approximations to improve computational efficiency.</p><p id="p-7">As we will demonstrate in the section <em>A simplified multi-locus model for haplotype structure</em>, the model efficiently reduces the state space of ancestral haplotypes from an exponential function of the number of SNP loci to a quadratic function, and thus allows the inference of allele age and selection intensity from multi-SNP haplotypes spanning several hundred kilo-bases or even mega-bases affected by strong selections. We then modify the importance sampling method of <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Griffiths and Tavaré (1994b)</a> to obtain the probability of a sample configuration and to estimate the selection parameters by averaging over genealogies of the linked sites. Note that an alternative choice is to adopt the existing importance sampling algorithms developed for multilocus ancestral recombination graph (ARG) under neutrality (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Griffiths and Marjoram 1996</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fearnhead and Donnelly 2001</a>) and incorporate the ARG into the structured-coalescent model. However, because of the large state space of genealogies that has to be explored by the importance sampling algorithms for a multilocus ARG, this approach is intractable on a genomic scale of hundreds of kilobases.</p></div><div class="section methods" id="sec-2"><h2 class="">Overview of the Method</h2><p id="p-8">Suppose the data consist of <em>n</em><sub>sample</sub> haplotypes with known phase collected from the current population. If genotype data are collected, the phase can be estimated by available algorithms (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Scheet and Stephens 2006</a>). The haplotypes are divided into two groups: the selected haplotypes, which are the chromosomes carrying the selected allele, and the background haplotypes, which do not carry the selected allele. In this method, we view the coalescent process in the <em>n</em> selected haplotypes as in a structured subpopulation (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The <em>n</em> selected haplotypes are represented by a <em>n</em> by <em>m</em> matrix with “1” or “0” for each entry, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-1.gif"/></noscript></span></span><em><sub>i</sub></em><sub>,</sub><em><sub>j</sub></em>, corresponding to the allele type of the <em>i</em>th haplotype at the <em>j</em>th SNP position. The position of the selected mutant is assumed to be known, as is the genetic distance between the <em>j</em><sub>1</sub>th and <em>j</em><sub>2</sub>th SNP, {<em>r<sub>j</sub></em><sub>1</sub><em><sub>j</sub></em><sub>2</sub>, 1 ≤ <em>j</em><sub>1</sub>, <em>j</em><sub>2</sub> ≤ <em>m</em>}.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An illustration of the structured-coalescent approach for modeling positive selection. The historical population sizes are indicated by the distance between the two dashed lines; and the allele frequency trajectory of the selected allele is indicated with a thin solid curve. The coalescent history of the selected locus with five derived lineages (solid bold lines) and five ancestral lineages (dotted bold lines) is superimposed on the trajectory and population size curves. The present time, t = 0, is at the bottom. And the time at which the trajectory of the selected mutant merged to the population-size curve denotes the time when the selected mutant arose in the population, i.e., the allele age T. In the model presented in the main text, only the sub-genealogies in the selected allele groups (bold solid lines) are considered." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An illustration of the structured-coalescent approach for modeling positive selection. The historical population sizes are indicated by the distance between the two dashed lines; and the allele frequency trajectory of the selected allele is indicated with a thin solid curve. The coalescent history of the selected locus with five derived lineages (solid bold lines) and five ancestral lineages (dotted bold lines) is superimposed on the trajectory and population size curves. The present time, &lt;em&gt;t&lt;/em&gt; = 0, is at the bottom. And the time at which the trajectory of the selected mutant merged to the population-size curve denotes the time when the selected mutant arose in the population, &lt;em&gt;i.e.&lt;/em&gt;, the allele age &lt;em&gt;T&lt;/em&gt;. In the model presented in the main text, only the sub-genealogies in the selected allele groups (bold solid lines) are considered.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F1.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/8/1429/F1.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463101" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">An illustration of the structured-coalescent approach for modeling positive selection. The historical population sizes are indicated by the distance between the two dashed lines; and the allele frequency trajectory of the selected allele is indicated with a thin solid curve. The coalescent history of the selected locus with five derived lineages (solid bold lines) and five ancestral lineages (dotted bold lines) is superimposed on the trajectory and population size curves. The present time, <em>t</em> = 0, is at the bottom. And the time at which the trajectory of the selected mutant merged to the population-size curve denotes the time when the selected mutant arose in the population, <em>i.e.</em>, the allele age <em>T</em>. In the model presented in the main text, only the sub-genealogies in the selected allele groups (bold solid lines) are considered.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">We model the effect of selection with the structured-coalescent scheme (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The likelihood function of the observed selected haplotype data, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-2.gif"/></noscript></span></span>, can be computed from<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-4.gif"/></noscript></span></span> denotes the genealogy and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-5.gif"/></noscript></span></span> the frequency trajectory of the selected mutant. Neither <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-6.gif"/></noscript></span></span> nor <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-7.gif"/></noscript></span></span> is observed directly. When constructing the likelihood function, they are often integrated out (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Felsenstein 1988</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Griffiths and Tavaré, 1994b</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kuhner <em>et al.</em> 1995</a>).</p><p id="p-11">The frequency trajectory of the selected mutant, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-8.gif"/></noscript></span></span>, is a random process that follows the Wright-Fisher model and has a probability distribution, <em>ℙ(<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-9.gif"/></noscript></span></span>|s</em>, <em>Γ</em>), that depends on the selection intensity <em>s</em> and the nuisance parameter set Γ, which includes all the other parameters related to the population history. Conditional on any given frequency trajectory, the sampling probability of the data is constructed by summing over all possible genealogical events: <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-10.gif"/></noscript></span></span>.</p><p id="p-12">For computationally efficient evaluation of the sampling probability, we propose a novel simplified multilocus model for the transition of different types of selected haplotypes (see the section <em>A simplified multilocus model for haplotype structure</em>). We compute the extent of the ancestral haplotypes in the vicinity of the selected mutant under the combined effects of recombination and selection. Together with the infinitely-many-sites model for mutations, the simplified multilocus haplotype model is used to approximate the evolutionary dynamics of the data.</p><p id="p-13">Because the spaces of both gene genealogies and allele frequency trajectories are too large to explore, ℙ(<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-11.gif"/></noscript></span></span>) and ℙ(<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-12.gif"/></noscript></span></span>|<em>s</em>, Γ) cannot be expressed in closed forms. We use the importance sampling algorithms to sample genealogies and trajectories that are compatible with the data from the proposal distributions. Then the likelihood is estimated as the weighted average probability for the samples. The importance weights are obtained by taking the ratio of the probabilities of true distribution and the proposal distribution. The procedure of evaluating the likelihood is illustrated by the flowchart in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Two main steps of the flowchart correspond to sampling <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-14.gif"/></noscript></span></span> using the importance sampling algorithms. The details of the algorithms and calculation of the importance ratios are presented in the section <em>Importance sampling and proposal distributions</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flowchart of importance sampling procedures of the method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flowchart of importance sampling procedures of the method.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F2.medium.gif" width="371" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/8/1429/F2.medium.gif" width="371" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463086" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Flowchart of importance sampling procedures of the method.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section other1" id="sec-3"><h2 class="">A simplified multilocus model for haplotype structure</h2><p id="p-15">We model the transition of the selected haplotypes (haplotypes carrying the selected mutant) under the influence of evolutionary events including recombination and mutation. We start with a sample of selected haplotypes collected from the current generation. When looking backward in time, we can eventually trace these selected haplotypes to one common ancestor (the ancestral haplotype) because all copies of the selected allele are descended from a single mutation. During the selection process, recombination breaks up and mixes the fragments with the background haplotypes. Recombination combined with mutation generates the different selected haplotypes that contain some segments of the ancestral haplotype. The number of distinct selected haplotypes at different times in the history is called the ancestral process. This ancestral process, conditional on the frequency trajectory of the selected mutant, can be viewed as a structured-coalescent process, because the selected haplotypes evolve as a subpopulation of the entire haplotype pool, with the size of the subpopulation determined by the mutant allele frequency, and the transitions among different haplotypes following the simplified multilocus model.</p><p id="p-16">To illustrate the state space of the ancestral process and the joint effect of selection and recombination on the transitions between the ancestral states, we will start with a simple case of only two loci, the selected locus and a partially linked SNP locus. We then extend the two-locus model to a simplified multi-locus haplotype model, after making several approximations for computational efficiency. Then in the section <em>Sampling probability of a multi-locus haplotype configuration</em>, the simplified multilocus haplotype model and the infinitely-many-sites model for mutation are used to derive the sampling probability for haplotype configuration of a sample by summing over possible ancestral states of the genealogical history, that is, the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-15.gif"/></noscript></span></span> component of likelihood function conditional on a simulated allele frequency trajectory.</p><div id="sec-4" class="subsection"><h3>A two-locus model</h3><p id="p-17">The two-locus haplotype model involves only the selected locus and one neutral marker, the positions of which are assumed known. The selected locus has the mutant allele A and the other neutral allele a. The neutral marker locus has two alleles B and b. Let <em>Q</em>(<em>t</em>) = (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>) denote the number of haplotypes AB, Ab, aB, and ab in the sample at time <em>t</em>. Conditional on the ancestral allele frequency trajectory, {<em>X<sub>t</sub></em>, <em>t</em> &gt; 0}, the “ancestral process” <em>Q</em>(<em>t</em>), which is defined as the numbers of each haplotype, can be approximated by the inhomogeneous Markov process (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hudson and Kaplan 1988</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Durrett and Schweinsberg 2004</a>). The states that the process <em>Q</em>(<em>t</em>) can jump from state (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>) to include (<em>q</em><sub>1</sub> − 1, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub> − 1, <em>q</em><sub>2</sub> + 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub> − 1, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub> + 1, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub> − 1, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub> + 1), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub> − 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub> + 1, <em>q</em><sub>2</sub> − 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub> − 1; <em>q</em><sub>3</sub> + 1, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub> − 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub> + 1), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub> − 1, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub> + 1, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub> − 1, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub> + 1; <em>q</em><sub>3</sub> − 1, <em>q</em><sub>4</sub>), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub> − 1, <em>q</em><sub>4</sub> + 1), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub> − 1), (<em>q</em><sub>1</sub> + 1, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub> − 1), (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub> + 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub> − 1) and (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub> + 1, <em>q</em><sub>4</sub> − 1). The transition probabilities from (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>) to the first four states are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, and the other transition probabilities can be constructed similarly. We assume an infinitely-many-sites model for mutations, so there are no new or recurrent mutations between the two alleles of either the selected locus or the neutral marker locus. Let <em>N<sub>AB</sub></em>(<em>t</em>), <em>N<sub>Ab</sub></em>(<em>t</em>), <em>N<sub>aB</sub></em>(<em>t</em>), and <em>N<sub>ab</sub></em>(<em>t</em>) be the population counts of the four corresponding haplotypes <em>AB</em>, <em>Ab</em>, <em>aB</em> and <em>ab</em> at time <em>t</em> respectively. For the transition from (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>) to (<em>q</em><sub>1</sub> − 1, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), no recombination has occurred and two lineages of haplotype AB are chosen to coalesce. The coalescence rate is <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-16.gif"/></noscript></span></span>; for the transition from (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>) to (<em>q</em><sub>1</sub> − 1, <em>q</em><sub>2</sub> + 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), one of the <em>N<sub>Ab</sub></em>(<em>t</em>) + <em>N<sub>ab</sub></em>(<em>t</em>) lineages, which carry the <em>b</em> allele, must be chosen to recombine, and the rate is <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-17.gif"/></noscript></span></span>. And the other transition rates can be obtained by similar rationale. Note that the selected allele first entered into the population at time <em>T</em>, so the states of the embedded Markov chain should satisfy <em>Q</em>(<em>T</em>) ∈ {(1, 0; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), (0, 1; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>)} and <em>Q</em>(<em>t</em>) = (0, 0; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>), <em>t</em> &gt; <em>T</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463088/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463088/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Definitions of notations used in this article</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463107/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463107/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Possible transitions from (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>; <em>q</em><sub>3</sub>, <em>q</em><sub>4</sub>) and the rates for the two-locus haplotype model</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>A simplified multilocus model for haplotype structure</h3><p id="p-18">The model for two-locus haplotypes can be naturally extended to multilocus haplotypes. However, the extent of haplotype structure that is used to infer selection intensity and allele age usually spans a large region which covers several hundred kilobases, or even more than a mega-base. In such a large region, there may be hundreds of polymorphic sites. When the number of SNP loci increases, the number of possible states of the ancestral process increases exponentially, and the transition matrix becomes so large that numerical evaluation becomes impossible. It is thus necessary to develop a parsimonious model for multilocus haplotypes that is both computationally fast and statistically efficient.</p><p id="p-19">The novel multilocus model we present here exploits the extents of the “ancestral haplotypes” retained during the selection process. We use the term “ancestral haplotype” to refer to the alleles at each SNP position on the ancestral chromosome, and the term “background haplotypes” to refer to the other haplotypes. In this model, we consider the interplay between selection and recombination acting upon the ancestral haplotype. As the selected allele increases its frequency, recombination breaks up the ancestral haplotypes and mixes them with the background haplotypes, resulting in the sample we observe at present. We make several simplifications or assumptions to expedite the computation in the sections to follow.</p><div id="sec-6" class="subsection"><h4>The ancestral state of each position along the haplotypes is assumed known:</h4><p id="p-20">For each position of a chromosome, it is assumed to be known whether the allele at that position is descended from an ancestral haplotype or one of the background haplotypes. In reality, the ancestral haplotype information cannot be observed directly from the data. The ancestral states and the break points of the ancestral haplotypes have to be inferred for each chromosome from patterns of SNP variation by other means (for example, the hidden Markov model for detecting recent positive selection, see <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen (2007)</a>).</p></div><div id="sec-7" class="subsection"><h4>The haplotype structure of background haplotypes is ignored:</h4><p id="p-21">It is reasonable to believe that the primary information for the inference of allele age and selection intensity comes from the extent of the ancestral haplotypes retained during selection. For example, states of the jump process for a two-locus haplotype model are reduced to <em>Q</em>(<em>t</em>) = (<em>q</em><sub>1</sub>, <em>q</em><sub>2</sub>), and the absorbing states are now (1,0) and (0,1).</p></div><div id="sec-8" class="subsection"><h4>The population frequencies of ancestral haplotypes at time t are approximated using the expectations:</h4><p id="p-22">To obtain the transition rates in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> for a two-locus model, the population counts of haplotypes <em>AB</em>, <em>Ab</em>, <em>aB</em>, and <em>ab</em> at time <em>t</em>, <em>N<sub>AB</sub></em>(<em>t</em>), <em>N<sub>Ab</sub></em>(<em>t</em>), <em>N<sub>aB</sub></em>(<em>t</em>), and <em>N<sub>ab</sub></em>(<em>t</em>), are needed. For a multilocus haplotype model, these values correspond to the population frequencies of haplotypes at time <em>t</em>. In studies of fine-scale disease mapping, these allele frequencies were assumed to be constant over time and identical to the observed frequencies in the current population (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Rannala and Reeve 2001</a>). When there is selection, the allele frequencies of nonrecombined haplotypes change over time, and their expectations have to be derived using a deterministic model. These expectations will be used as an estimate of the true haplotype frequencies at any time <em>t</em>. The details of the equations for the haplotype frequencies at time <em>t</em> can be seen in Appendix A.</p></div><div id="sec-9" class="subsection"><h4>Multiple “migrations” of ancestral haplotype fragments between selected and neutral haplotypes are ignored:</h4><p id="p-23">In the ancestral process of a two-locus haplotype model, there is a probability that a lineage of the neutral marker experiences two recombination events during the sweep process. In other words, the ancestral haplotype crosses over twice with a background haplotype at the marker position and the segment of the ancestral haplotype “migrates” to and then back from the group of neutral haplotypes. The probability for such events are small during a selective sweep, and thus are ignored (on order <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-44.gif"/></noscript></span></span>, where <em>α</em> = 2<em>N s</em>, see <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Etheridge <em>et al.</em> 2006</a>).</p><p id="p-24">Because of the assumptions (1)−(4), the state space of the ancestral process can be reduced by considering the unique pattern of ancestral haplotype lengths in combination with the occurrence of mutation since selection began. Because we have assumed that the ancestral states of SNPs along the chromosomes are known, for every haplotype, we can determine the break points of the ancestral haplotype caused by the recombination events nearest to the mutant, in addition to the locations where mutations occurred within each ancestral haplotype region. With this information known, the ancestral haplotype on each side of the mutant can be coded as follows: for every selected haplotype, we record the SNPs to the left and to the right sides that delimit the ancestral haplotype; if there are mutations within the ancestral haplotype regions, the positions of the mutants are also recorded and listed as “mutation coordinates” behind the two “recombination coordinates.” In <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, we give an example of 10 selected haplotypes consisting of 25 SNPs, among which the ancestral haplotype regions are highlighted. The selected mutant is located at position 18 (shown in boldface type). The left end of the ancestral haplotype for the first haplotype is 7 to the left of the mutant, and the right end is 6 to the right, such that the first haplotype is recorded as (7, 6). For haplotype 3, the full code is (12, 7, 21) with a mutation occurring in position 21. By this coding rule, the configuration of the sample listed in Column 3 of <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> is summarized in Column 3.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463082/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463082/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">An example of haplotype configuration to demonstrate the coding rules used to denote the haplotype structure</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">For a recoded haplotype type <em>h</em> = (<em>R</em><sub>1</sub>, <em>R</em><sub>2</sub>, <em>M</em><sub>1</sub>, …, <em>M<sub>k</sub></em>), the first two entries, corresponding to the left and right break points of the ancestral haplotypes, are the “recombination coordinates” and the other entries are the “mutation coordinates.” In this model, the transition among different haplotypes is caused by recombination and mutation. For the coded haplotypes consisting of only recombination coordinates, the transition among different haplotype types can occur only through recombination. If there are <em>m<sub>L</sub></em> loci to the left of the mutant and <em>m<sub>R</sub></em> loci to the right of the mutant, the total number of possible allele types is (<em>m<sub>L</sub></em> + 1) × (<em>m<sub>R</sub></em> + 1). The number of possible states is greatly reduced compared to a direct extension of the two-locus model, whose state space grows exponentially with number of SNPs.</p><p id="p-27">In <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, we present a partial list of transition rates caused by recombination for a 4-locus haplotype model. Assume that the haplotype has 4 SNP loci, with the alleles on the ancestral haplotype being A, B, C, and D. A is the selected mutant, and the order of the four loci along the chromosome is the same as their alphabetical order. We use the notation [<em>ABCD</em>] to denote the intact segment of ancestral haplotype. And similarly [<em>AB</em> − <em>d</em>] indicates that the first two loci have the inherited ancestral haplotype of A and B, the allele of the third locus is arbitrary, and the fourth locus is a background haplotype. Examples in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> show some of the one-step transition rates starting from state [<em>ABcd</em>]. For example, for haplotype [<em>ABcd</em>] to jump to [<em>ABCD</em>], one of the [<em>ABcd</em>] haplotypes should be chosen and recombination has to occur between allele B and c, in such a way that the chosen haplotype crosses over with haplotypes [<em>ABCD</em>], [<em>AbCD</em>] and [<em>a</em> − <em>CD</em>]. The one-step transition probability is: <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-45.gif"/></noscript></span></span>, with <em>X<sub>t</sub></em> being the population allele frequency of allele <em>A</em> at time <em>t</em>, and <em>P</em><sub>[.]</sub>(<em>t</em>) being the frequency of the haplotype in square parenthesis among either the selected haplotype or the background haplotype group, depending on the allele type carried by the particular haplotype at the selected mutant locus. Because of assumption (4) we made previously (also made by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Durrett and Schweinsberg (2004)</a>), the second and the third terms are small and can be ignored. With this simplification, the transition rate in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> becomes <em>r<sub>BC</sub></em> ⋅ <em>P</em><sub>[</sub><em><sub>ABCD</sub></em><sub>]</sub>(<em>t</em>) ⋅ <em>X<sub>t</sub></em>. Similarly, we can simplify the other transition rates shown in <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463094/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463094/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">The transition probabilities for some states of the multilocus haplotype model</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">A mutation coordinate records the SNP position at which the haplotype has an allele mutated from the ancestral haplotype. We assume an infinitely-many-sites model for mutations on the haplotypes. According to <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Griffiths and Tavaré (1995)</a>, the set of nonrecombining haplotypes carrying these mutations is identical to a rooted gene tree, and the sequence of mutations corresponds to the path from the haplotype to the common ancestor [the root of the gene tree, see <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Griffiths and Tavaré (1995)</a> for a detailed discussion]. We use the same notation scheme for mutations as <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Griffiths and Tavaré (1995)</a>. Because the haplotypes we investigate are from recombining regions, an additional constraint is added to reflect the effect of recombinations: the sequence of mutations we recorded as mutation coordinates includes only those located between the two recombination coordinates, that is, the subset of mutations on the retained ancestral haplotype region. For haplotype data from nonrecombining regions, the recombination coordinates are identical for all haplotypes (the left and right ends of the whole haplotype), and the mutation coordinates define a gene tree with the rooted genealogy, meaning that the state of the common ancestor of the sample is known (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Griffiths and Tavaré, 1994b</a>). This is the type of data analyzed by the approach of <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Coop and Griffiths (2004)</a>. Thus their method can be viewed as a special case of our method with no recombination.</p></div></div><div id="sec-10" class="subsection"><h3>Sampling probability of a multilocus haplotype configuration</h3><p id="p-29">In the section <em>A simplified multilocus model for haplotype structure</em>, we described a novel simplified multilocus model that can dramatically reduce the state space of the haplotype ancestral process, and illustrated how to obtain the transition probabilities between different states. We now consider the computation of the probability of a sample of multilocus haplotypes.</p><p id="p-30">A sample of selected haplotypes can be coded and summarized by the rules introduced in the section <em>A simplified multi-locus model for haplotype structure</em> and grouped into <em>d</em> distinct groups <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-49.gif"/></noscript></span></span> = {<em>h<sub>1</sub></em>, …, <em>h<sub>d</sub></em>} with the corresponding multiplicities <strong>n</strong> = {<em>n</em><sub>1</sub>, …, <em>n<sub>d</sub></em>}. We define the sampling probability, <em>q</em>((<span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-50.gif"/></noscript></span></span>, <strong>n</strong>)<em><sub>t</sub></em>), to be the probability of observing the sample configuration (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-51.gif"/></noscript></span></span>, <strong>n)</strong> at <em>t</em> generations before the current generation. The entire history of the sample configuration {(<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-52.gif"/></noscript></span></span>, <strong>n</strong>)<em><sub>t</sub></em>, <em>t</em> &gt; 0} can be described by a Markov process that starts at time <em>t</em> = 0 and continues until reaching the absorbing state (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-53.gif"/></noscript></span></span>, <strong>e<sub>1</sub></strong>) at a random time <strong><em>τ</em></strong>, where <strong>e<sub>j</sub></strong> denotes the unit vector <strong>e<sub>j</sub></strong> = (0,0, …, 1, …, 0) with only the <em>j</em>th entry being 1. When <em>t</em> = 0, <em>q</em>((<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-54.gif"/></noscript></span></span>, <strong>n</strong>)<sub>0</sub>) is the partial likelihood of the data which is sufficient for the inference of selection intensity and allele age. The sampling probability at time <em>t</em> can be obtained by recursively summing over all possible state paths in the backward Markov process. The recursive formula can be written as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-55.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-56.gif"/></noscript></span></span> is the transition probability of jumping from state (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-57.gif"/></noscript></span></span>′, <strong>n</strong>′) at time <em>v</em> to state (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-58.gif"/></noscript></span></span>, <strong>n</strong>) at time <em>t</em>, and <em>g</em>(<em>v</em>|<em>n</em>, <em>t</em>) is the density function of the inter-arrival time to the next event given an event at time <em>t</em>.</p><p id="p-31">As the Markovian ancestral process is restricted to the selected haplotypes, the process behaves as if in a population with temporally varying size {<em>I<sub>t</sub></em>, <em>t</em> = 0.1, …, <em>T</em>}. If time is measured in a scale of 2<em>N</em><sub>0</sub> generations, the coalescent rate is <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-59.gif"/></noscript></span></span>, with <em>X<sub>t</sub></em> = <em>I<sub>t</sub></em>/2<em>N<sub>t</sub></em> and λ<em><sub>t</sub></em> = <em>N<sub>t</sub></em>/<em>N</em><sub>0</sub> being the population size ratio. With the same scaling, the mutation rate for the <em>i</em>th haplotype is <em>θ<sub>i</sub></em> = 4<em>N</em><sub>0</sub><em>μβ<sub>i</sub></em> = <em>θ β<sub>i</sub></em>, and the recombination rate is <em>ρ<sub>i</sub></em> = 4<em>N</em><sub>0</sub><em>cβ<sub>i</sub></em> = <em>ρβ<sub>i</sub></em>. <em>θ</em> = 4<em>N</em><sub>0</sub><em>μ</em> and <em>ρ</em> = 4<em>N</em><sub>0</sub><em>c</em> are the scaled mutation rate and recombination rate for the whole haplotype, and <em>β<sub>i</sub></em> denotes the proportion of the retained ancestral haplotype region, or the inter-region between two recombination coordinates, out of the entire length of the <em>i</em>th haplotype. Note that <em>β<sub>i</sub></em> changes over time with the change of recombination coordinates of the <em>i</em>th haplotype. The inter-arrival time to the next event, <em>v</em>, given that the last event happened at time <em>t</em> has a non-homogeneous exponential distribution, with the density function in the form of<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-60.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span><em>t</em> &lt; <em>v</em> &lt; ∞, where at time <em>v</em>, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-61.gif"/></noscript></span></span> is the rate for the any events. As the allele frequency trajectory {<em>X<sub>t</sub></em>, 0 ≤ <em>t</em> ≤ <em>T</em>} is a discrete-time random process following the Wright-Fisher model, we adopt the geometric distribution for discrete time instead of using the continuous approximation in Equation 3:</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-62.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span><p id="p-32">Conditional on an event happening at time <em>v</em>, the probabilities for the event being a mutation, recombination or coalescent are respectively<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-63.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>If a mutation occurs, one of the lineages in the sample is chosen to mutate into other types according to the mutation model, and the mutation coordinate of that haplotype is modified correspondingly; if a coalescence event occurs within the <em>j</em>th haplotype group, two of the existing lineages with haplotype <em>h<sub>j</sub></em> are chosen at random to coalesce, and the number of lineages in the <em>j</em>th haplotype group, <em>n<sub>j</sub></em>, is decreased by 1; otherwise, a position along the haplotype is chosen for the recombination event to occur with the consequence that one of the recombination coordinates is changed to record the recombination at that position (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> for a realization of the genealogical history for a sample of four haplotypes).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A realization of the genealogies for a sample of four haplotypes (lineages 1&#x2212;4) to illustrate possible events in the genealogies. Black denotes the ancestral haplotype region (see the main text for the definition of &#x201C;ancestral haplotypes&#x201D;), and white denotes background haplotypes. A star denotes a neutral mutant arising on the ancestral haplotype. The present time, t = 0, is on the bottom. When going back in time, the events are coalescent (lineages 2 and 3 coalesce to the ancestral lineage 5), recombination (lineage 6 &#x2192; lineage 4), coalescent (lineages 5 and 6 coalesce to the ancestral lineage 7), recombination (lineage 8 &#x2192; lineage 7), mutation (on lineage 9), coalescent (lineages 8 and 9 coalesce to lineage 10) in sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A realization of the genealogies for a sample of four haplotypes (lineages 1&#x2212;4) to illustrate possible events in the genealogies. Black denotes the ancestral haplotype region (see the main text for the definition of &#x201C;ancestral haplotypes&#x201D;), and white denotes background haplotypes. A star denotes a neutral mutant arising on the ancestral haplotype. The present time, &lt;em&gt;t&lt;/em&gt; = 0, is on the bottom. When going back in time, the events are coalescent (lineages 2 and 3 coalesce to the ancestral lineage 5), recombination (lineage 6 &#x2192; lineage 4), coalescent (lineages 5 and 6 coalesce to the ancestral lineage 7), recombination (lineage 8 &#x2192; lineage 7), mutation (on lineage 9), coalescent (lineages 8 and 9 coalesce to lineage 10) in sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F3.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/8/1429/F3.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463103" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">A realization of the genealogies for a sample of four haplotypes (lineages 1−4) to illustrate possible events in the genealogies. Black denotes the ancestral haplotype region (see the main text for the definition of “ancestral haplotypes”), and white denotes background haplotypes. A star denotes a neutral mutant arising on the ancestral haplotype. The present time, <em>t</em> = 0, is on the bottom. When going back in time, the events are coalescent (lineages 2 and 3 coalesce to the ancestral lineage 5), recombination (lineage 6 → lineage 4), coalescent (lineages 5 and 6 coalesce to the ancestral lineage 7), recombination (lineage 8 → lineage 7), mutation (on lineage 9), coalescent (lineages 8 and 9 coalesce to lineage 10) in sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">We now present the detailed recursion equation (Equation 6), expressed as a sum over the above three types of events in a way corresponding to Equation 2 for our model. Under the infinitely-many-sites mutation model (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Watterson 1975</a>) and the proposed multilocus haplotype model for the extent of ancestral haplotypes, summing over possible one-step configuration changes at time <em>v</em> leads to the following equation:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-64.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>The notation in Equation 6 has the following meaning: <strong>e<sub>k</sub></strong> is the <em>k</em>th unit vector, representing a multiplicity of the <em>k</em>th distinct haplotype; <em>β<sub>l</sub></em> is the length of ancestral haplotype region divided by length of the <em>l</em>th haplotype. We follow the notation of <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Griffiths and Tavaré (1995)</a> and use several operators to denote the changes of sample configuration: <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-65.gif"/></noscript></span></span> is the shift operator that can be operated on a specific haplotype <em>h<sub>k</sub></em> or the entire haplotype set <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-66.gif"/></noscript></span></span>. Specifically, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-67.gif"/></noscript></span></span><em>h<sub>k</sub></em> represents the haplotype obtained by deleting the first mutation coordinate of haplotype <em>h<sub>k</sub></em>. Similarly, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-68.gif"/></noscript></span></span><em><sub>k</sub></em><span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-69.gif"/></noscript></span></span> represents the new set of distinct haplotypes obtained by deleting the first mutation coordinate of the <em>k</em>th distinct haplotype, <em>h<sub>k</sub></em>, in <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-70.gif"/></noscript></span></span>. Another operator that operates on the entire haplotype is <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-71.gif"/></noscript></span></span><em><sub>k</sub></em> which removes the <em>k</em>th distinct haplotype <em>h<sub>k</sub></em> from the set <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-72.gif"/></noscript></span></span>. The third coordinate change operator <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-73.gif"/></noscript></span></span> is defined in this manuscript to denote the coordinate changes caused by recombinations: <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-74.gif"/></noscript></span></span><em>h<sub>i</sub></em> changes one of the two recombination coordinates of haplotype <em>i</em>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-75.gif"/></noscript></span></span>, and eliminates all mutation coordinates outside the regions delimited by <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-77.gif"/></noscript></span></span>.</p><p id="p-35">Next we explain how we derive the recursive formula in Equation 6. Starting from time <em>v</em> back from the present, there are four possible paths to arrive at the sample configuration (<span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-78.gif"/></noscript></span></span>, <strong>n</strong>) at time <em>v</em>: (1) a coalescent event occurred at time <em>v</em>, and a possible sample configuration prior to time <em>v</em> was (<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-79.gif"/></noscript></span></span>, <strong>n</strong> − <strong>e<sub>k</sub></strong>); (2) a mutation occurred on a haplotype <em>h<sub>k</sub></em> that had only single multiplicity, or <em>n<sub>k</sub></em> = 1, at time <em>v</em>, and a new mutation coordinate was added to <em>h<sub>k</sub></em>; (3) a mutation occurred on a haplotype <em>h<sub>j</sub></em> that had multiplicity greater than 1, or <em>n<sub>j</sub></em> &gt; 1, at time <em>v</em>, and a new haplotype <em>h<sub>k</sub></em> with <em>n<sub>k</sub></em> = 1 was generated by adding the new mutation coordinate to <em>h<sub>j</sub></em>; (4) a recombination event occurred, and altered one of the recombination coordinates of <em>h<sub>j</sub></em> by that of <em>h<sub>i</sub></em>. Note that a recombination event not only changes the recombination coordinate, it also changes the mutation coordinates: after the recombination coordinates are changed by a recombination event, all the mutation coordinates of that haplotype are checked, and only those located within the interregion between the two new recombination coordinates are kept. The four terms on the RHS of Equation 6 correspond to the above four paths respectively, and the derivation of the first three terms follows <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Griffiths and Tavaré (1994a</a>, <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">1995)</a>. In the first path, the sample configuration at time <em>v</em> compatible with the occurrence of coalescence is (<span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-80.gif"/></noscript></span></span>, <strong>n</strong> − <strong>n<sub>k</sub></strong>), the probability that the event occurred at time <em>v</em> is a coalescent event is <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-81.gif"/></noscript></span></span>. And whenstarting from the configuration (<span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-82.gif"/></noscript></span></span>, <strong>n</strong> − <strong>n<sub>k</sub></strong>) and going forward in time, the probability that one of the <em>n<sub>k</sub></em> − 1 haplotype <strong>e</strong><em><sub>k</sub></em> is chosen to duplicate is <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-83.gif"/></noscript></span></span> (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Griffiths and Tavaré, 1994a</a>). The one-step transition probability of <em>p</em>((<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-84.gif"/></noscript></span></span>, <strong>n</strong>)<sub>v</sub>|(<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-85.gif"/></noscript></span></span>, <strong>n</strong> − <strong>e<sub>k</sub></strong>)<sub>v</sub>) is then <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-86.gif"/></noscript></span></span>. Note that a restriction for haplotype group <em>h<sub>k</sub></em> is that there must be more than one lineage in group <em>h<sub>k</sub></em> at time <em>v</em>. Summing over all possible haplotype groups that have multiplicity <em>n<sub>k</sub></em> ≥ 2 at time <em>v</em>, and are compatible for coalescent events to occur, we obtain the first term of Equation 6. The second and the third path correspond to the cases when the event occurring at time <em>v</em> is a mutation. Under the assumption of the infinitely many-sites model, if a mutation event occurs, it can result only in one of the single-multiplicity haplotype groups at time <em>v</em> (<em>n<sub>k</sub></em> = 1) and the mutation coordinate must be a singleton in the sample configuration at time <em>v</em>. In both the second and the third paths, the chance that a mutation occurred at time <em>v</em> is <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-87.gif"/></noscript></span></span>. The configuration at time <em>v</em> compatible with the occurrence of second path is (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-88.gif"/></noscript></span></span><em><sub>k</sub></em><span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-89.gif"/></noscript></span></span>, <strong>n</strong>), and the probability for the mutation to happen to haplotype <em>h<sub>k</sub></em> is <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-90.gif"/></noscript></span></span>. Summing over all haplotypes satisfying <em>n<sub>k</sub></em> = 1 and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-91.gif"/></noscript></span></span><em>h</em> ≠ <em>h<sub>j</sub></em> for all <em>j</em> yields the second term in Equation 6. In the third path, the probability for the mutation to happen to haplotype <em>h<sub>k</sub></em> is <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-92.gif"/></noscript></span></span> with the sample configuration prior to the event being <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-93.gif"/></noscript></span></span> for all k with <em>n<sub>k</sub></em> = 1 at time <em>v</em>. In the fourth path, recombination occurs with a probability of <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-94.gif"/></noscript></span></span>. If recombination causes a haplotype <em>h<sub>j</sub></em> to become <em>h<sub>i</sub></em>, the haplotype configuration prior to the event is (<span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-95.gif"/></noscript></span></span><em>h<sub>i</sub></em><span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-96.gif"/></noscript></span></span>, (<strong>n</strong> − <strong>e<sub>i</sub></strong> + <strong>e<sub>j</sub></strong>)) and the probability for a haplotype <em>h<sub>j</sub></em> changing into <em>h<sub>i</sub></em> is <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-97.gif"/></noscript></span></span>, with <em>n<sub>j</sub></em> ≥ 0. The transition probability <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-98.gif"/></noscript></span></span> between different haplotypes follows the multilocus haplotype model presented in the section <em>A simplified multi-locus model for haplotype structure</em>, where <em>h<sub>i</sub></em> and <em>h<sub>j</sub></em> correspond to one of the distinct haplotypes defined by the “recombination coordinates”. Combining these possibilities and averaging over the time to the first event more ancient than <em>t</em>, the sampling distribution of haplotype configuration, (<span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-99.gif"/></noscript></span></span>, <strong>n</strong>), is analogous to Equation 2:</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-100.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span><p id="p-36">Although we have reduced the state numbers defined by recombinations from <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-101.gif"/></noscript></span></span> to (<em>m<sub>L</sub></em> + 1) × (<em>m<sub>R</sub></em> + 1) for the simplified multilocus haplotype model, it is still difficult to numerically solve the distribution function induced by the Markov process. Therefore ,we still need to use the importance sampling algorithms to estimate the sampling probability, as will be shown in following sections.</p></div></div><div class="section other2" id="sec-11"><h2 class="">Importance sampling and proposal distributions</h2><div id="sec-12" class="subsection"><h3>Likelihood and importance sampling</h3><p id="p-37">Importance sampling algorithms are used to efficiently sample from the probability spaces of frequency trajectories and intra-allelic genealogies in order to approximate the integral in the likelihood function (see Equation 1). The likelihood function in Equation 1 can be expressed as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-102.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>where ℚ(<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-103.gif"/></noscript></span></span>) and ℚ(<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-104.gif"/></noscript></span></span>|<em>s</em>, Γ) are the proposal distributions that have non-zero weight only on genealogies and trajectories compatible with the data <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-105.gif"/></noscript></span></span> (that is, ℙ(<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-106.gif"/></noscript></span></span>| <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-107.gif"/></noscript></span></span>) = 1). Suppose that <em>M</em> random frequency trajectories and <em>L</em> genealogical histories for each of the trajectories are sampled, then the approximation to Equation 8 becomes</p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-108.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span><p id="p-38">where <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-109.gif"/></noscript></span></span><sup>(</sup><em><sup>m</sup></em><sup>)</sup> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-110.gif"/></noscript></span></span><sup>(</sup><em><sup>k</sup></em><sup>)</sup> are the <em>m</em>th and <em>k</em>th independent samples from the proposal distributions. The ratio <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-111.gif"/></noscript></span></span> is called the importance weight. The importance sampling algorithm for the genealogies will be presented in the section <em>A proposal distribution for sampling genealogical histories conditional on a trajectory</em> and the algorithm for the allele frequency trajectories in the section <em>The proposal distribution for sampling allele frequency trajectories of the selected allele in a population of time-varying size</em>. We illustrate the proposal distributions and the calculation of importance weights in those two sections.</p><p id="p-39">Allele age, <em>T</em>, is not explicitly expressed as a variable in the likelihood function. It is the end point of the frequency trajectory, and thus depends on <em>s</em> through ℙ(<span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-112.gif"/></noscript></span></span>|<em>s</em>). Once the maximum likelihood estimate <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-113.gif"/></noscript></span></span> is found, the posterior distribution of <em>T</em> can be obtained from the repeated samples of <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-114.gif"/></noscript></span></span> given <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-115.gif"/></noscript></span></span>. This method for estimating allele age has been used by <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Coop and Griffiths (2004)</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Saunders <em>et al.</em> (2005)</a> and <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wood <em>et al.</em> (2005)</a>, while it is different from the Bayesian approach of <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Slatkin (2008)</a>, who assumed a prior for allele age and jointly inferred both selection intensity and allele age.</p></div><div id="sec-13" class="subsection"><h3>A proposal distribution for sampling genealogical histories conditional on a trajectory</h3><p id="p-40">The recursion of the genealogical histories given in Equation 6 for the likelihood of the data cannot be computed exactly for large data sets since there are too many compatible sets of ancestral states. We adopt an importance sampling algorithm to approximate the likelihood by Monte Carlo methods. There are many ways of constructing the proposal distributions for the importance sampling algorithm (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Griffiths and Tavaré, 1994b</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stephens <em>et al.</em> 2001</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Paul <em>et al.</em> 2011</a>). Here we follow the scheme developed by <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Griffiths and Tavaré (1994b)</a>. As described in previous sections, the infinitely-many-sites model for mutations in conjunction with the simplified multi-locus haplotype model is assumed.</p><p>In the algorithm, a Markov process starting from the configuration in the current generation (<span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-116.gif"/></noscript></span></span>, <strong>n</strong>)<sub>0</sub>, conditional on a randomly sampled historical frequency trajectory {<em>X<sub>t</sub></em>, <em>t</em> = 0, …, <em>T</em>}, is constructed and simulated backward in time until reaching the absorbing state (<span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-117.gif"/></noscript></span></span>, <strong>e<sub>1</sub></strong>)<em><sub>τ</sub></em> at time <strong><em>τ</em></strong>. The algorithm is summarized as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-42">Generate time to the next event, <em>v</em>, by the density function given in Equation 4;</p></li><li id="list-item-2"><p id="p-43">Choose one of the three possible events (recombination, mutation or coalescence) from the proposal distribution. We first define the total rate that any event occurs at time v as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-118.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span></p><p id="p-44">where<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-119.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span></p><p id="p-45">The <strong>proposal distribution</strong> is designed in such a way that a possible event at time <em>v</em> is chosen with probability in proportion to the size of each term in <em>ℏ</em>((<span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-120.gif"/></noscript></span></span>, <strong>n</strong>)<em><sub>v</sub></em>):<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-121.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span></p></li><li id="list-item-3"><p id="p-46">Update the configuration to reflect the chosen event. Let <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-122.gif"/></noscript></span></span><em><sub>j</sub></em>, <em>j</em> ≥ 0 denote the <em>j</em>th event during the genealogical history, and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-123.gif"/></noscript></span></span><sub>0</sub> = (<span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-124.gif"/></noscript></span></span>, <strong>n</strong>)<sub>0</sub>. ℚ(<span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-125.gif"/></noscript></span></span><sub>j</sub> |<span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-126.gif"/></noscript></span></span><em><sub>j</sub></em><sub>−1</sub>) is the transition probability of the backward Markov process determined by the proposal distribution (Equation 12). Similarly, ℙ(<span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-127.gif"/></noscript></span></span><em><sub>j</sub></em><sub>−1</sub> |<span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-128.gif"/></noscript></span></span><em><sub>j</sub></em>) is the transition probability of the forward Markov process. The sequential importance weight for the <em>j</em>th step change <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-129.gif"/></noscript></span></span> is estimated. Here we illustrate the calculation of the importance weight for the case in which the chosen event is <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-130.gif"/></noscript></span></span><em><sub>k</sub></em> = (<span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-131.gif"/></noscript></span></span>, <strong>n</strong> − <strong>e<em><sub>k</sub></em></strong>). As shown in the section <em>Sampling probability of a multilocus haplotype configuration</em>, <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-132.gif"/></noscript></span></span>, and from Equation 12 we have <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-133.gif"/></noscript></span></span>. Taking the ratio of the two terms, we obtain the importance weight for the <em>j</em>th step: <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-134.gif"/></noscript></span></span>. <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> provides more details of importance weights for other events.</p></li><li id="list-item-4"><p id="p-47">Repeat steps 1−3 to continue generating the historical events in the genealogy backward in time.</p></li><li id="list-item-5"><p id="p-48">Stop when the absorbing state is reached, that is, a single lineage remains in the sample configuration, (<span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-135.gif"/></noscript></span></span>, <strong>e</strong><sub>1</sub>), or if the proposed time for next event is beyond the end of the frequency trajectories;</p></li><li id="list-item-6"><p id="p-49">Assume that there are <em>I</em> steps until the Markov chain reaches the absorbing states, the ratio of the forward/backward paths is the product of sequential importance weights:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-136.gif"/></noscript></span> <span class="disp-formula-label">(13)</span></span></p><p id="p-50">and is used in the likelihood function. For those paths with time beyond the end of the given frequency trajectories, {<em>X<sub>t</sub></em>}, the ratio is set to zero, which means the sample is rejected.</p></li></ol><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463079/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463079/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">The proposal distribution and importance weights for the importance sampling algorithm presented in the section <em>A proposal distribution for sampling genealogical histories conditional on a trajectory</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>The proposal distribution for sampling allele frequency trajectories of the selected allele in a population of time-varying size</h3><p>We use the backward Wright-Fisher model under selection to sample the allele frequency trajectories. The importance sampling algorithm for sampling frequency trajectories of the selected allele is described as follows. A detailed explanation can be found in the original paper (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Slatkin 2001</a>).</p><ol class="list-ord " id="list-2"><li id="list-item-7"><p id="p-53">Given a selection intensity s and parameter set Γ, a sample path is simulated from <em>t</em> = 0 (current generation) with <em>I</em><sub>0</sub> copies of A, and then proceeds backward from generation to generation assuming the following binomial distribution:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-168.gif"/></noscript></span> <span class="disp-formula-label">(14)</span></span></p><p id="p-54">where <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-169.gif"/></noscript></span></span> satisfies: <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-170.gif"/></noscript></span></span>, and <em>Y<sub>t</sub></em><sub>−1</sub> = <em>I<sub>t</sub></em><sub>−1</sub>/(2<em>N<sub>t</sub></em><sub>−1</sub>). The backward process is stopped at time <em>T</em> when the allele is lost. The probability of the backward process is calculated as: <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-171.gif"/></noscript></span></span></p></li><li id="list-item-8"><p id="p-55">For a frequency trajectory <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-172.gif"/></noscript></span></span><sup>(</sup><em><sup>m</sup></em><sup>)</sup> simulated in Step 1, the probability it is generated by the forward process is computed. In the Wright-Fisher model with selection, the number of allele A from generation <em>t</em> to generation <em>t</em> − 1 follows a binomial distribution:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-173.gif"/></noscript></span> <span class="disp-formula-label">(15)</span></span></p><p id="p-56">with<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-174.gif"/></noscript></span> <span class="disp-formula-label">(16)</span></span></p><p id="p-57">which is the allele frequency of A after selection in generation <em>t</em>. In Equation 16, <em>X<sub>t</sub></em> = <em>I<sub>t</sub></em>/(2<em>N<sub>t</sub></em>) is the frequency of allele A before selection in generation <em>t</em>. The probability of the sample path <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-175.gif"/></noscript></span></span><sup>(</sup><em><sup>m</sup></em><sup>)</sup> is<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-176.gif"/></noscript></span> <span class="disp-formula-label">(17)</span></span>where ℙ(<em>I<sub>T</sub></em><sub>−1</sub>|<em>I<sub>T</sub></em>) = 1 if <em>I<sub>T</sub></em><sub>−1</sub> = 1 and 0 otherwise. And the subscript <em>F</em> indicates that the process is “forward” in time.</p></li><li id="list-item-9"><p id="p-58">The importance weight is calculated as (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Slatkin 2001</a>):<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-177.gif"/></noscript></span> <span class="disp-formula-label">(18)</span></span></p><p id="p-59">where <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-178.gif"/></noscript></span></span> is the population size at the first generation after the allele is lost in the backward process. The multiplication of <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-179.gif"/></noscript></span></span> is needed in Equation 18, since the rate of influx of new mutations is proportional to the population size of that generation.</p></li></ol></div></div><div class="section other3" id="sec-15"><h2 class="">Applications</h2><div id="sec-16" class="subsection"><h3>Simulation</h3><p id="p-60">Using the coalescent simulator SelSim (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Spencer and Coop 2004</a>), data are generated for two sets of parameters corresponding to medium and strong selection respectively: <em>θ</em> = 4<em>Nμ</em> = 500, <em>ρ</em> = 4<em>Nr</em> = 500, <em>Ns</em> = 50, and <em>θ</em> = 4<em>Nμ</em> = 500, <em>ρ</em> = 4<em>Nr</em> = 500, <em>Ns</em> = 500, where <em>θ</em>, <em>ρ</em>, and <em>Ns</em> represent the mutation rate, recombination rate and selection coefficient scaled by the effective population size. The frequencies of the selected alleles at the present are chosen to be 0.60. Since the Moran model is used in SelSim, whereas the Wright-Fisher model is used in our method, the effective population size in the simulations is scaled to match that of a Wright-Fisher model by multiplying by a factor of 2 (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Watterson 1975</a>). We estimate the log-likelihood of <em>s</em> for a range of selection coefficients with the other parameters in Γ known, assuming that the population has a constant size of 10,000. The curves of the log-likelihood over the grid of <em>s</em> values are smoothed by a local polynomial smoother. This smoother fits a linear function to a subset of data points within a local window of the target point where the log-likelihood is to be estimated. The fitting is carried out by the weighted least square regression, which gives more weight to points close to the target point and less weight to distant points. The log-likelihood is thus estimated as the fitted value at the target point. The size of the local window or the bandwidth is chosen by eye for each curve. The log-likelihood curves are plotted in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relative likelihood curve for the simulated data with the selection coefficient s = 0.05 and a constant population size N = 10,000. The comparison of eight estimates of the likelihood curves is presented. Each estimate is an independent run of our method on different simulated data. The results are from 1 million iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The relative likelihood curve for the simulated data with the selection coefficient &lt;em&gt;s&lt;/em&gt; = 0.05 and a constant population size &lt;em&gt;N&lt;/em&gt; = 10,000. The comparison of eight estimates of the likelihood curves is presented. Each estimate is an independent run of our method on different simulated data. The results are from 1 million iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F4.medium.gif" width="428" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/8/1429/F4.medium.gif" width="428" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463075" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-61" class="first-child">The relative likelihood curve for the simulated data with the selection coefficient <em>s</em> = 0.05 and a constant population size <em>N</em> = 10,000. The comparison of eight estimates of the likelihood curves is presented. Each estimate is an independent run of our method on different simulated data. The results are from 1 million iterations.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relative likelihood curve for the simulated data with the selection coefficient s = 0.005 and a constant population size N = 10,000. The comparison of eight estimates of the likelihood curves is presented. Each estimate is an independent run of our method on different simulated data. The results are from 1 million iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The relative likelihood curve for the simulated data with the selection coefficient &lt;em&gt;s&lt;/em&gt; = 0.005 and a constant population size &lt;em&gt;N&lt;/em&gt; = 10,000. The comparison of eight estimates of the likelihood curves is presented. Each estimate is an independent run of our method on different simulated data. The results are from 1 million iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F5.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/8/1429/F5.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463084" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-62" class="first-child">The relative likelihood curve for the simulated data with the selection coefficient <em>s</em> = 0.005 and a constant population size <em>N</em> = 10,000. The comparison of eight estimates of the likelihood curves is presented. Each estimate is an independent run of our method on different simulated data. The results are from 1 million iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-63">To evaluate the performance of the importance sampling approximation, we perform eight independent simulations for every parameter combination. One million iterations in the importance sampling algorithm are required to ensure good estimates, and the likelihood curves are presented together in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>. For the data set simulated with <em>s</em> = 0.005, the MLE ranges from 0.0041 to 0.0073. For the data set with <em>s</em> = 0.05, the MLE ranges from 0.032 to 0.0543.</p></div><div id="sec-17" class="subsection"><h3>Glucose-6-phosphate dehydrogenase (<em>G6PD)</em></h3><p id="p-64">The <em>G6PD</em> gene is located on the X-chromosome. Some alleles are known to confer the resistance to malaria (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ruwende <em>et al.</em> 1995</a>). Case-control studies have demonstrated that a common variant, <em>G6PD-202A</em>, reduces the risk of malaria by approximately 50% (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Ruwende <em>et al.</em> 1995</a>). This allele is at low frequency in most populations but has an intermediate frequency in sub-Saharan Africa. Several population genetic studies have investigated the effect of a recent selective sweep in this region (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tishkoff <em>et al.</em> 2001</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Sabeti <em>et al.</em> 2002</a>). Here we use the data in <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Sabeti <em>et al.</em> (2002)</a>, which consists of 252 males from there African populations in a 440-Kb region covering the <em>G6PD</em> gene. We analyze only the 60 haplotypes from the Beni population. There are 10 haplotypes containing the 202-A allele in the sample. We assume that the frequency of the selected allele in the Beni population is the same as that estimated from the sample which is 0.1667. The recombination fractions among SNPs are obtained by interpolation with the Oxford fine-scale recombination map (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Myers <em>et al.</em> 2005</a>). The recombination rate in the <em>G6PD</em> gene region is heterogeneous with two recombination hot-spots, and the overall averaged recombination rate for the region is 1.4410 cM/Mb. We determine the end points of ancestral haplotypes and mutations by running the hidden Markov model (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen 2007</a>). The data configuration is coded by the rules presented in the section <em>A simplified multilocus model for haplotype structure</em> as shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463095/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463095/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">The sample configuration of the <em>G6PD</em> data according to coding rules in the section <em>A simplified multilocus model for haplotype structure</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-65">Because the hidden Markov model analysis indicates there are no mutations in the ancestral haplotype regions, we set <em>θ</em> to 0.0. We assume an effective population size of <em>N</em> = 10,000, which is constant over time. Because <em>G6PD</em> is X-linked, <em>N<sub>e</sub></em> is 3/4 of the autosomal size. We assume an additive model for selection, which means the fitness’s of the three genotypes aa, Aa, and AA are 1, 1 + 1/2<em>s</em>, and 1 + <em>s</em>, respectively. The likelihood of the selection coefficient is estimated by our method from 1 million iterations of the importance sampling algorithm. The log-likelihood curve is plotted in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The selection coefficient is estimated to be 0.0456 (95% confidence interval of 0.0144−0.0769). From the estimated selection coefficient, the age of the 202-A allele can be estimated. As shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, given the selection coefficients estimated, the corresponding posterior distribution of allele age is plotted.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Likelihood curve for the G6PD data as a function of the selection coefficient with a constant population size of 10,000. The likelihood curve is smoothed by a local polynomial smoother. The point estimate of the selection coefficients is 0.0456 with the 95% confidence interval of (0.0144, 0.0769)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Likelihood curve for the &lt;em&gt;G6PD&lt;/em&gt; data as a function of the selection coefficient with a constant population size of 10,000. The likelihood curve is smoothed by a local polynomial smoother. The point estimate of the selection coefficients is 0.0456 with the 95% confidence interval of (0.0144, 0.0769).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F6.medium.gif" width="411" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/8/1429/F6.medium.gif" width="411" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463097" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-66" class="first-child">Likelihood curve for the <em>G6PD</em> data as a function of the selection coefficient with a constant population size of 10,000. The likelihood curve is smoothed by a local polynomial smoother. The point estimate of the selection coefficients is 0.0456 with the 95% confidence interval of (0.0144, 0.0769).</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The posterior distribution of the allele age in generations when the selection coefficient is set to the value estimated from Figure 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-173285134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The posterior distribution of the allele age in generations when the selection coefficient is set to the value estimated from Figure 6.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1429/F7.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/3/8/1429/F7.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1429/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463077" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-67" class="first-child">The posterior distribution of the allele age in generations when the selection coefficient is set to the value estimated from <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-68">We have developed a likelihood method for estimating selection intensity and allele age from haplotype structure of multilocus SNPs closely linked to a selected mutant. The likelihood is based on the proposed simplified multilocus haplotype model, which describes the ancestral process of haplotype extent under the joint effects of selection, recombination and mutation. In this model, the state space of the ancestral process is determined by the extent of intact ancestral haplotypes in the vicinity of the selected mutant and the new mutations arising on the ancestral haplotypes during the selective process. Our method adopts importance sampling algorithms to efficiently explore the genealogical history of the sample for evaluation of the sampling probability. By applying the method to both simulated and real data, we demonstrate that the extent of the haplotype structure is informative for the inference of selection intensity of a recent positive selection.</p><p id="p-69">Our method has two merits. First, by exploiting the extent of haplotype structure and focusing only on subprocesses of the ARG related to the retained ancestral segments on the selected haplotypes, we dramatically reduce the computational burden such that data sets from genomic regions of mega-base magnitude can be analyzed. Second, our method can allow for changes in population size. This is especially important for samples from human populations outside Africa, because population growth can affect the pattern of linkage disequilibrium and haplotype structure, and thus lead to an incorrect estimation of the selection intensity if the effect of demographic history is not explicitly modeled. In our analysis of simulated data, we found that the estimated selection coefficient is accurate but is sensitive to the recombination rates assumed. Since the variability of recombination rates is high over human genome (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Myers <em>et al.</em> 2005</a>), good estimates of local rates are necessary to obtain accurate estimates of selection coefficients.</p><p id="p-70">Another factor that may affect the estimates of selection intensity and allele age is the SNP marker density in the data. Because mutants that have experienced positive selections are typically young, new mutations at nearby loci accumulate at a relatively slow rate compared to the rate of recombination that breaks down linkage disequilibrium. Therefore, we expect fewer segregating sites observed in regions under recent positive selection. In the low-density SNP data, these segregating sites are likely not typed. In the two data sets of the <em>G6PD</em> region we analyzed, <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Sabeti <em>et al.</em> (2002)</a>’s and <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Verrelli <em>et al.</em> (2002)</a>’s data, no new mutations at closely linked loci were detected. Because their <em>G6PD</em> data were not generated by resequencing, a proportion of mutations may not have been identified or included in the data. We expect that resequencing data from the target gene regions will be more informative for identifying the occurrence of recombinations and mutations during the selective process. The method developed in this paper is for identifying ongoing positive selection. If the selected allele has been fixed in the population, mutations accumulated since its fixation become informative and important for inferring the fixation time, for which the allele frequency spectrum after a selective sweep is a better choice (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen 2012</a>).</p><p id="p-71">The importance sampling algorithm for the genealogies adopted in this paper was developed by <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Griffiths and Tavaré (1994b)</a>. In their proposal distribution, at each step any possible event that could lead to the current configuration is considered and sampled in proportion to their rate of occurrence (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Felsenstein <em>et al.</em> 1999</a>). More efficient proposal distributions have been developed (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Stephens <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Slatkin 2002</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Lorio and Griffiths 2004</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Paul <em>et al.</em> 2011</a>) and can be adopted to improve the computational efficiency of our method.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgment</h2><p id="p-78">We are grateful to Drs. Kun Chen, Graham Coop, and Steve Evans for helpful discussions; to Dr. Pardis Sabeti for kindly providing us the <em>G6PD</em> data; and to Dr. Jeff Wall and the anonymous reviewers for their helpful comments. The work was supported by a National Institutes of Health grant GM-40282 to M.S.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-19"><h3>The probability distribution for the extent of an intact ancestral haplotype under selection</h3><p id="p-72">We show how to derive the distribution for the extent of an intact ancestral haplotype at time <em>t</em> during a selective process. We investigate a selective sweep that starts from a single copy of the selected allele. Here we denote the time when the new selected mutant arises by 0, and look forward in time. Let <em>X<sub>t</sub></em> be frequency of the selected allele at time <em>t</em>. If we ignore the initial randomness of allele frequency trajectories, which is usually modeled using a supercritical branching process, <em>X<sub>t</sub></em> can be well approximated using the deterministic logistic equation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ohta and Kimura, 1975</a>):<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-180.gif"/></noscript></span> <span class="disp-formula-label">(19)</span></span>where <em>X</em><sub>0</sub> can be set to 1/2<em>N</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kaplan <em>et al.</em> 1989</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stephan <em>et al.</em> 1992</a>). Assume the selected locus has two alleles A and a, with A being the advantageous allele. We are modeling a continuous segment between the selected mutant and the neutral marker, which has two alleles B and b. We use upper case letters to denote that the position is descended from the ancestral haplotype, and lower case letters to denote the position descended from the background haplotypes. Note here the two “alleles” are defined according to whether they are descended from the ancestral haplotype or not, instead of the true observed nucleotide types at that locus. We use [<em>AB</em>] to denote a segment of ancestral haplotype with loci A and B being the end points. Let <em>P</em><sub>[</sub><em><sub>AB</sub></em><sub>]</sub>(<em>t</em>) be the relative population frequency of such fragments among all haplotypes carrying the selected allele A at time <em>t</em>. Furthermore, we use [<em>A</em>−] to denote an ancestral haplotype with <em>A</em> being from the ancestral haplotype, while the state of the other end point of the fragment is not determined.</p><p id="p-73">For a random ancestral haplotype [<em>AB</em>] or [<em>Ab</em>], if it recombines with any [<em>A</em>−] haplotype during the interval [0, <em>t</em>], it does not change <em>P</em><sub>[</sub><em><sub>AB</sub></em><sub>]</sub>(<em>t</em>). The only possible change is caused by “effective” recombinations, that is, recombination with an [<em>ab</em>], or [<em>aB</em>] haplotype from the neutral haplotype “sub-population”. The expected number of effective recombination events for a [<em>AB</em>] recombining with any [<em>a</em>−] haplotype during the interval [0, <em>t</em>] is</p><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-181.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span><p id="p-74">It is not hard to see that the number of effective events on an [<em>AB</em>] ancestral haplotype during the time interval [0, <em>t</em>] follows a Poisson distribution. <em>P</em><sub>[</sub><em><sub>AB</sub></em><sub>]</sub>(<em>t</em>) is then identical to the probability of no effective recombination between [<em>AB</em>] for an ancestral haplotype:</p><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1429/embed/mml-math-182.gif"/></noscript></span> <span class="disp-formula-label">(21)</span></span><p id="p-75">Similarly, for an ancestral haplotype [<em>ABC</em>], the probability of being intact during the interval (0, <em>t</em>) follows the Equation 21, except that the recombination fraction is replaced by <em>r</em><sub>[</sub><em><sub>AC</sub></em><sub>]</sub>.</p><p id="p-76">Note that in Equation 21, when <em>st</em> is small, which means either the selective process is at an early stage or the selection is weak, the term (1 − (1 − <em>e<sup>st</sup></em>) <em>X</em><sub>0</sub>)<em><sup>r</sup></em><sup>/</sup><em><sup>s</sup></em> ≈ 1, and thus similar to the neutral case. However, if <em>st</em> gets larger, the term cannot be ignored. For example, if <em>r</em> = 0.001, <em>s</em> = 0.01, <em>X</em><sub>0</sub> = 0.001 and <em>t</em> = 1000, the relative bias can be as large as ~27%.</p><p id="p-77">Also note that in the aforementioned derivation for the distribution of ancestral haplotypes, we ignore the randomness of the frequency trajectory of the selected allele at the very early stage of the selective process, and approximate the trajectory with a deterministic equation. Ignoring the randomness of the allele frequency trajectory at the early stage can bias the inference of parameters related to the sweep process, but as pointed out in previous studies (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kaplan <em>et al.</em> 1989</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Braverman <em>et al.</em> 1995</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barton, 1998</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Durrett and Schweinsberg, 2004</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Etheridge <em>et al.</em> 2006</a>), when the selection intensity is sufficiently strong, the bias is small. For this reason, our method is more suitable to analyze genes under strong selections.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-79"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-02-12"><span class="received-label">Received </span>February 12, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-06-14"><span class="accepted-label">Accepted </span>June 14, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Chen and Slatkin</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.1" data-doi="10.1017/S0016672398003462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The effect of hitch-hiking on neutral genealogies.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D72%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672398003462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672398003462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077703600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.2" data-doi="10.1086/421051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bersaglieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanderploeg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic signatures of strong recent positive selection at the lactase gene.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBersaglieri%26rft.auinit1%253DT.%26rft.volume%253D74%26rft.issue%253D6%26rft.spage%253D1111%26rft.epage%253D1120%26rft.atitle%253DGenetic%2Bsignatures%2Bof%2Bstrong%2Brecent%2Bpositive%2Bselection%2Bat%2Bthe%2Blactase%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F421051%26rft_id%253Dinfo%253Apmid%252F15114531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/421051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15114531&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221651900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braverman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The hitchhiking effect on the site frequency spectrum of DNA polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBraverman%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D140%26rft.issue%253D2%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bhitchhiking%2Beffect%2Bon%2Bthe%2Bsite%2Bfrequency%2Bspectrum%2Bof%2BDNA%2Bpolymorphisms.%26rft_id%253Dinfo%253Apmid%252F7498754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzIvNzgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.4" data-doi="10.1016/j.tpb.2011.11.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The joint allele frequency spectrum of multiple populations: a coalescent theory approach.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol</abbr>. <span class="cit-vol">81</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DChen%26rft.auinit1%253DH.%26rft.volume%253D81%26rft.issue%253D2%26rft.spage%253D179%26rft.epage%253D195%26rft.atitle%253DThe%2Bjoint%2Ballele%2Bfrequency%2Bspectrum%2Bof%2Bmultiple%2Bpopulations%253A%2Ba%2Bcoalescent%2Btheory%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2011.11.004%26rft_id%253Dinfo%253Apmid%252F22155588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2011.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22155588&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-3.8.1429.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Statistical methods for inference of positive selection from genetic polymorphism. Ph.D. thesis, University of California, Berkeley.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.6" data-doi="10.1101/gr.100545.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population differentiation as a test for selective sweeps.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.100545.109%26rft_id%253Dinfo%253Apmid%252F20086244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMzkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Ancestral inference on gene trees under selection.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D64%26rft.spage%253D241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.8" data-doi="10.1239/aap/1086957579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Lorio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Importance sampling in coalescent histories I.</span> <abbr class="cit-jnl-abbrev">Adv. Appl. Probab.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAppl.%2BProbab.%26rft.volume%253D36%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1239%252Faap%252F1086957579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1239/aap/1086957579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000221918900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.9" data-doi="10.1016/j.tpb.2004.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrett</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweinsberg</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Approximating selective sweeps.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DDurrett%26rft.auinit1%253DR.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D138%26rft.atitle%253DApproximating%2Bselective%2Bsweeps.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2004.04.002%26rft_id%253Dinfo%253Apmid%252F15302222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2004.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15302222&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223509900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.10" data-doi="10.1214/105051606000000114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etheridge</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaffelhuber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakolbinger</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An approximate sampling formula under genetic hitchhiking.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Probab.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BProbab.%26rft.volume%253D16%26rft.spage%253D685%26rft_id%253Dinfo%253Adoi%252F10.1214%252F105051606000000114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/105051606000000114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking under positive Darwinian selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1413%26rft.atitle%253DHitchhiking%2BUnder%2BPositive%2BDarwinian%2BSelection%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fearnhead</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Estimating recombination rates from population genetic data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFearnhead%26rft.auinit1%253DP.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D1299%26rft.epage%253D1318%26rft.atitle%253DEstimating%2BRecombination%2BRates%2BFrom%2BPopulation%2BGenetic%2BData%26rft_id%253Dinfo%253Apmid%252F11729171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS8zLzEyOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.13" data-doi="10.1146/annurev.ge.22.120188.002513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Phylogenies from molecular sequences: inference and reliability.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D521%26rft.epage%253D565%26rft.atitle%253DPhylogenies%2BFrom%2BMolecular%2BSequences%253A%2BInference%2BAnd%2BReliability%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.22.120188.002513%26rft_id%253Dinfo%253Apmid%252F3071258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.22.120188.002513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3071258&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R470400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-3.8.1429.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamato</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beerli</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Likelihoods on coalescents: A Monte Carlo sampling approach to inferring parameters from population samples of molecular data</span>, pp. <span class="cit-fpage">163</span>–<span class="cit-lpage">185</span> in <span class="cit-source">Statistics in Molecular Biology and Genetics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Seillier-Moiseiwitsch</span>  <span class="cit-name-given-names">F.</span></span>. <span class="cit-publ-name">Institute of Mathematical Statistics</span>, <span class="cit-publ-loc">Beachwood, OH</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Statistical tests of neutrality of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D693%26rft.epage%253D709%26rft.atitle%253DStatistical%2Btests%2Bof%2Bneutrality%2Bof%2Bmutations.%26rft_id%253Dinfo%253Apmid%252F8454210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzMvNjkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.16" data-doi="10.1089/cmb.1996.3.479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Ancestral inference from samples of DNA sequences with recombination.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DGriffiths%26rft.auinit1%253DR.%2BC.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D479%26rft.epage%253D502%26rft.atitle%253DAncestral%2Binference%2Bfrom%2Bsamples%2Bof%2BDNA%2Bsequences%2Bwith%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.1996.3.479%26rft_id%253Dinfo%253Apmid%252F9018600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.1996.3.479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9018600&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996WE51500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.17" data-doi="10.1098/rstb.1994.0079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span><span class="cit-comment">a</span> <span class="cit-article-title">Sampling theory for neutral alleles in a varying environment.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond., B</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DGriffiths%26rft.auinit1%253DR.%2BC.%26rft.volume%253D344%26rft.issue%253D1310%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DSampling%2BTheory%2Bfor%2BNeutral%2BAlleles%2Bin%2Ba%2BVarying%2BEnvironment%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1994.0079%26rft_id%253Dinfo%253Apmid%252F7800710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNDQvMTMxMC80MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.18" data-doi="10.1006/tpbi.1994.1023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span><span class="cit-comment">b</span> <span class="cit-article-title">Simulating probability distributions in the coalescent.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D46%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1994.1023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1994.1023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PL43900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.19" data-doi="10.1016/0025-5564(94)00044-Z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Unrooted genealogical tree probabilities in the infinitely-many-sites model.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DGriffiths%26rft.auinit1%253DR.%2BC.%26rft.volume%253D127%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D98%26rft.atitle%253DUnrooted%2Bgenealogical%2Btree%2Bprobabilities%2Bin%2Bthe%2Binfinitely-many-sites%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252894%252900044-Z%26rft_id%253Dinfo%253Apmid%252F7734858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(94)00044-Z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7734858&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QP73700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The coalescent process in models with selection and recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">819</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D819%26rft.spage%253D831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.21" data-doi="10.1016/j.cell.2013.01.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamberov</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerbault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling recent human evolution in mice by expression of a selected EDAR variant.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKamberov%26rft.auinit1%253DY.%2BG.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D691%26rft.epage%253D702%26rft.atitle%253DModeling%2BRecent%2BHuman%2BEvolution%2Bin%2BMice%2Bby%2BExpression%2Bof%2Ba%2BSelected%2BEDAR%2BVariant.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.01.016%26rft_id%253Dinfo%253Apmid%252F23415220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23415220&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314945600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darden</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The coalescent process in models with selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKaplan%26rft.auinit1%253DN.%2BL.%26rft.volume%253D120%26rft.issue%253D3%26rft.spage%253D819%26rft.epage%253D829%26rft.atitle%253DThe%2Bcoalescent%2Bprocess%2Bin%2Bmodels%2Bwith%2Bselection.%26rft_id%253Dinfo%253Apmid%252F3066685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIwLzMvODE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The” hitchhiking effect” revisited.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKaplan%26rft.auinit1%253DN.%2BL.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D887%26rft.epage%253D899%26rft.atitle%253DThe%2B%2522hitchhiking%2Beffect%2522%2Brevisited.%26rft_id%253Dinfo%253Apmid%252F2612899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzQvODg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting a local signature of genetic hitchhiking along a recombining chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D160%26rft.issue%253D2%26rft.spage%253D765%26rft.epage%253D777%26rft.atitle%253DDetecting%2Ba%2BLocal%2BSignature%2Bof%2BGenetic%2BHitchhiking%2BAlong%2Ba%2BRecombining%2BChromosome%26rft_id%253Dinfo%253Apmid%252F11861577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzIvNzY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.25" data-doi="10.1006/tpbi.1997.1299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuhauser</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Ancestral processes with selection.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DKrone%26rft.auinit1%253DS.%2BM.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D210%26rft.epage%253D237%26rft.atitle%253DAncestral%2BProcesses%2Bwith%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1997.1299%26rft_id%253Dinfo%253Apmid%252F9245777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1997.1299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9245777&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XN64600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamato</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Estimating effective population size and mutation rate from sequence data using Metropolis-Hastings sampling.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKuhner%26rft.auinit1%253DM.%2BK.%26rft.volume%253D140%26rft.issue%253D4%26rft.spage%253D1421%26rft.epage%253D1430%26rft.atitle%253DEstimating%2Beffective%2Bpopulation%2Bsize%2Band%2Bmutation%2Brate%2Bfrom%2Bsequence%2Bdata%2Busing%2BMetropolis-Hastings%2Bsampling.%26rft_id%253Dinfo%253Apmid%252F7498781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MC80LzE0MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.27" data-doi="10.1126/science.1117196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A fine-scale map of recombination rates and hotspots across the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DS.%26rft.volume%253D310%26rft.issue%253D5746%26rft.spage%253D321%26rft.epage%253D324%26rft.atitle%253DA%2BFine-Scale%2BMap%2Bof%2BRecombination%2BRates%2Band%2BHotspots%2BAcross%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1117196%26rft_id%253Dinfo%253Apmid%252F16224025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0Ni8zMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.28" data-doi="10.1101/gr.4252305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic scans for selective sweeps using SNP data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1566</span>–<span class="cit-lpage">1575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1566%26rft.epage%253D1575%26rft.atitle%253DGenomic%2Bscans%2Bfor%2Bselective%2Bsweeps%2Busing%2BSNP%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4252305%26rft_id%253Dinfo%253Apmid%252F16251466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.29" data-doi="10.1017/S0016672300015731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">The effect of selected linked locus on heterozygosity of neutral alleles (the hitch-hiking effect).</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DOta%26rft.auinit1%253DT.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D313%26rft.epage%253D326%26rft.atitle%253DThe%2Beffect%2Bof%2Bselected%2Blinked%2Blocus%2Bon%2Bheterozygosity%2Bof%2Bneutral%2Balleles%2B%2528the%2Bhitch-hiking%2Beffect%2529.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300015731%26rft_id%253Dinfo%253Apmid%252F1183812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300015731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1183812&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AT49000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.30" data-doi="10.1534/genetics.110.125534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SteinrÃijcken</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An accurate sequentially Markov conditional sampling distribution for the coalescent with recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125534%26rft_id%253Dinfo%253Apmid%252F21270390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ny80LzExMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.31" data-doi="10.1093/molbev/msq290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic variations in Tibetan populations and high-altitude adaptation at the Himalayas.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq290%26rft_id%253Dinfo%253Apmid%252F21030426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzIvMTA3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.32" data-doi="10.1086/321279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rannala</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeve</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High-resolution multipoint linkage-disequilibrium mapping in the context of a human genome sequence.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRannala%26rft.auinit1%253DB.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D159%26rft.epage%253D178%26rft.atitle%253DHigh-resolution%2Bmultipoint%2Blinkage-disequilibrium%2Bmapping%2Bin%2Bthe%2Bcontext%2Bof%2Ba%2Bhuman%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321279%26rft_id%253Dinfo%253Apmid%252F11410841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11410841&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.8.1429.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rannala</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeve</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Joint Bayesian estimation of mutation location and age using linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Pac Symp BioComput.</abbr> <span class="cit-fpage">526</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.34" data-doi="10.1038/376246a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruwende</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khoo</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snow</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">376</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRuwende%26rft.auinit1%253DC.%26rft.volume%253D376%26rft.issue%253D6537%26rft.spage%253D246%26rft.epage%253D249%26rft.atitle%253DNatural%2Bselection%2Bof%2Bhemi-%2Band%2Bheterozygotes%2Bfor%2BG6PD%2Bdeficiency%2Bin%2BAfrica%2Bby%2Bresistance%2Bto%2Bsevere%2Bmalaria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F376246a0%26rft_id%253Dinfo%253Apmid%252F7617034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/376246a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7617034&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RK33100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.35" data-doi="10.1038/nature01140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">H. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting recent positive selection in the human genome from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D419%26rft.issue%253D6909%26rft.spage%253D832%26rft.epage%253D837%26rft.atitle%253DDetecting%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome%2Bfrom%2Bhaplotype%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01140%26rft_id%253Dinfo%253Apmid%252F12397357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12397357&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178769800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.36" data-doi="10.1038/nature06250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varilly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hostetter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide detection and characterization of positive selection in human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D913%26rft.epage%253D918%26rft.atitle%253DGenome-wide%2Bdetection%2Band%2Bcharacterization%2Bof%2Bpositive%2Bselection%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06250%26rft_id%253Dinfo%253Apmid%252F17943131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943131&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.37" data-doi="10.1534/genetics.105.048140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The extent of linkage disequilibrium caused by selection on G6PD in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048140%26rft_id%253Dinfo%253Apmid%252F16020776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS8zLzEyMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.38" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: Application to inferring missing genotype and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.39" data-doi="10.1126/science.1189406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic evidence for high-altitude adaptation in Tibet.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189406%26rft_id%253Dinfo%253Apmid%252F20466884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjkvNTk4Ny83MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.40" data-doi="10.1006/tpbi.2000.1488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Most recent common ancestor probability distributions in gene genealogies under selection.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSlade%26rft.auinit1%253DP.%2BF.%26rft.volume%253D58%26rft.issue%253D4%26rft.spage%253D291%26rft.epage%253D305%26rft.atitle%253DMost%2Brecent%2Bcommon%2Bancestor%2Bprobability%2Bdistributions%2Bin%2Bgene%2Bgenealogies%2Bunder%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2000.1488%26rft_id%253Dinfo%253Apmid%252F11162788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2000.1488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162788&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166772300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.41" data-doi="10.1017/S0016672301005183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Simulating genealogies of selected alleles in a population of variable size.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D57%26rft.atitle%253DSimulating%2Bgenealogies%2Bof%2Bselected%2Balleles%2Bin%2Ba%2Bpopulation%2Bof%2Bvariable%2Bsize.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672301005183%26rft_id%253Dinfo%253Apmid%252F11556137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672301005183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11556137&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170741300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.42" data-doi="10.1016/S0040-5809(02)00007-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A vectorized method of importance sampling with application to models of mutation and migration.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D62%26rft.issue%253D4%26rft.spage%253D339%26rft.epage%253D348%26rft.atitle%253DA%2Bvectorized%2Bmethod%2Bof%2Bimportance%2Bsampling%2Bwith%2Bapplications%2Bto%2Bmodels%2Bof%2Bmutation%2Band%2Bmigration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0040-5809%252802%252900007-2%26rft_id%253Dinfo%253Apmid%252F12427457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0040-5809(02)00007-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12427457&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Bayesian method for jointly estimating allele age and selection intensity.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">129</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D137%26rft.atitle%253DA%2BBayesian%2Bmethod%2Bfor%2Bjointly%2Bestimating%2Ballele%2Bage%2Band%2Bselection%2Bintensity.%26rft_id%253Dinfo%253Apmid%252F18289407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18289407&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254308300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.44" data-doi="10.1093/bioinformatics/bth417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selsim: a program to simulate population genetic data with natural selection and recombination.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">18</span>): <span class="cit-fpage">3673</span>–<span class="cit-lpage">3675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth417%26rft_id%253Dinfo%253Apmid%252F15271777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTgvMzY3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.45" data-doi="10.1016/0040-5809(92)90045-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiehe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenz</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The effect of strongly selected substitutions on neutral polymorphism: analytical results based on diffusion theory.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D41%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252892%252990045-U%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(92)90045-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992HM04300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.46" data-doi="10.1086/319501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new statistical method for haplotype reconstruction from population data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D978%26rft.epage%253D989%26rft.atitle%253DA%2Bnew%2Bstatistical%2Bmethod%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319501%26rft_id%253Dinfo%253Apmid%252F11254454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/319501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11254454&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167666000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical methods for testing the neutral mutations hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.48" data-doi="10.1371/journal.pbio.0050171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new approach for using genome scans to detect recent positive selection in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTang%26rft.auinit1%253DK.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De171%26rft.epage%253De171%26rft.atitle%253DA%2Bnew%2Bapproach%2Bfor%2Busing%2Bgenome%2Bscans%2Bto%2Bdetect%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050171%26rft_id%253Dinfo%253Apmid%252F17579516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17579516&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.49" data-doi="10.1126/science.1061573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varkonyi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cahinhinan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argyropoulos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Haplotype diversity and linkage disequilibrium at human G6PD: recent origin of alleles that confer malarial resistance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1061573%26rft_id%253Dinfo%253Apmid%252F11423617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTMvNTUyOS80NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.50" data-doi="10.1038/ng1946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranciaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbitt</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Convergent adaptation of human lactase persistence in Africa and Europe.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTishkoff%26rft.auinit1%253DS.%2BA.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D40%26rft.atitle%253DConvergent%2Badaptation%2Bof%2Bhuman%2Blactase%2Bpersistence%2Bin%2BAfrica%2Band%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1946%26rft_id%253Dinfo%253Apmid%252F17159977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17159977&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243136500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.51" data-doi="10.1086/344345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verrelli</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argyropoulos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Destro-Bisol</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froment</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence for balancing selection from nucleotide sequence analyses of human G6PD.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVerrelli%26rft.auinit1%253DB.%2BC.%26rft.volume%253D71%26rft.issue%253D5%26rft.spage%253D1112%26rft.epage%253D1128%26rft.atitle%253DEvidence%2Bfor%2Bbalancing%2Bselection%2Bfrom%2Bnucleotide%2Bsequence%2Banalyses%2Bof%2Bhuman%2BG6PD.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F344345%26rft_id%253Dinfo%253Apmid%252F12378426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/344345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12378426&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178884300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.52" data-doi="10.1371/journal.pbio.0040446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A map of recent positive selection in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D4%26rft.spage%253D446%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235991300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.53" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.54" data-doi="10.1086/491748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stover</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The <em>β</em>-globin recombinational hotspot reduces the effects of strong selection around HbC, a recently arisen mutation providing resistance to malaria.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWood%26rft.auinit1%253DE.%2BT.%26rft.volume%253D77%26rft.issue%253D4%26rft.spage%253D637%26rft.epage%253D642%26rft.atitle%253DThe%2Bbeta%2B-globin%2Brecombinational%2Bhotspot%2Breduces%2Bthe%2Beffects%2Bof%2Bstrong%2Bselection%2Baround%2BHbC%252C%2Ba%2Brecently%2Barisen%2Bmutation%2Bproviding%2Bresistance%2Bto%2Bmalaria.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F491748%26rft_id%253Dinfo%253Apmid%252F16175509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/491748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16175509&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.55" data-doi="10.1093/molbev/msq277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A genome-wide search for signals of high-altitude adaptation in Tibetans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq277%26rft_id%253Dinfo%253Apmid%252F20961960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzIvMTAwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1429.56" data-doi="10.1126/science.1190371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sanchez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuo</span>  <span class="cit-name-given-names">Z. X. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing of 50 human exomes reveals adaptation to high altitude.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYi%26rft.auinit1%253DX.%26rft.volume%253D329%26rft.issue%253D5987%26rft.spage%253D75%26rft.epage%253D78%26rft.atitle%253DSequencing%2Bof%2B50%2BHuman%2BExomes%2BReveals%2BAdaptation%2Bto%2BHigh%2BAltitude%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190371%26rft_id%253Dinfo%253Apmid%252F20595611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjkvNTk4Ny83NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/8/1429.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/8/1419" title="Variation in Meiotic Recombination Frequencies Between Allelic Transgenes Inserted at Different Sites in the Drosophila melanogaster Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/8/1443" title="FijiWings: An Open Source Toolkit for Semiautomated Morphometric Analysis of Insect Wings" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/8"><span>Volume 3 Issue 8, August 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/8.cover-source.jpg?itok=Dmhq6EsG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1CZDqKdjPCJbLGF705kc1qawiWPa7cciRCDa37fyOEU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/8/1429" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/8/1429" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/8/1429" class="active" data-icon-position="" data-hide-link-title="0">Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459515" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ee7h5uRzGpEWANlLYRWBhn7gZ4_4t9isSMOnvaQPX-E" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/8/1429.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/8/1429" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-5hb9j7ugHBzTfZj-3h5F4VA82RXwf0dKeiZrYvMd-LY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/8/1429" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/8/1429.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459515" id="node459515--2" data-pisa="ggg;3/8/1429" data-pisa-master="ggg;g3.113.006197" data-apath="/ggg/3/8/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hua Chen</span> and <span class="highwire-citation-author" data-delta="1">Montgomery Slatkin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1442;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459515--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hchen007@gmail.com" class="" data-icon-position="" data-hide-link-title="0">hchen007@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Montgomery Slatkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Montgomery%2BSlatkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Slatkin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMontgomery%2BSlatkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/8/1429" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/8/1429&amp;title=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/8/1429&amp;title=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/8/1429&amp;title=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/8/1429&amp;text=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/8/1429&amp;title=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/8/1429&amp;t=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/8/1429&amp;title=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/8/1429&amp;title=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459515/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459515" id="node459515--3" data-pisa="ggg;3/8/1429" data-pisa-master="ggg;g3.113.006197" data-apath="/ggg/3/8/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inferring Selection Intensity and Allele Age from Multilocus Haplotype Structure</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hua Chen</span> and <span class="highwire-citation-author" data-delta="1">Montgomery Slatkin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1442;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459515--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hua Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHua%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hchen007@gmail.com" class="" data-icon-position="" data-hide-link-title="0">hchen007@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Montgomery Slatkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Montgomery%2BSlatkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Slatkin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMontgomery%2BSlatkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1429&amp;rft.epage=1442&amp;rft.atitle=Inferring%20Selection%20Intensity%20and%20Allele%20Age%20from%20Multilocus%20Haplotype%20Structure&amp;rft.volume=3&amp;rft.issue=8&amp;rft.date=2013-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chen%2C+Hua&amp;rft.au=Slatkin%2C+Montgomery"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459515/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459515/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459515/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459515/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459515/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459515/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459515/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459515/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459515/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459515/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459515/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459515/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/8/1429" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Introduction</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Overview of the Method</a></li><li class="even"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">A simplified multilocus model for haplotype structure</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Importance sampling and proposal distributions</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Applications</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgment</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/8/1429.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/8/1429.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/3/8/1429.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
