<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/8/1253" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/8/1253.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/8/1253.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/8/1253.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/8/1253/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/8/1253.atom" />
<meta name="HW.pisa" content="ggg;3/8/1253" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens" />
<meta name="DC.Identifier" content="10.1534/g3.113.006643" />
<meta name="DC.Date" content="2013-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Crooks et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We analyzed genotypes from ~10K single-nucleotide polymorphisms (SNPs) in two families of an F2 intercross between Red Junglefowl and White Leghorn chickens. Possible null alleles were found by patterns of incompatible and missing genotypes. We estimated that 2.6% of SNPs had null alleles compared with 2.3% with genotyping errors and that 40% of SNPs in which a parent and offspring were genotyped as different homozygotes had null alleles. Putative deletions were identified by null alleles at adjacent markers. We found two candidate deletions that were supported by fluorescence intensity data from a 60K SNP chip. One of the candidate deletions was from the Red Junglefowl, and one was present in both the Red Junglefowl and White Leghorn. Both candidate deletions spanned protein-coding regions and were close to a previously detected quantitative trait locus affecting body weight in this population. This study demonstrates that the ~50K SNP genotyping arrays now available for several agricultural species can be used to identify null alleles and deletions in data from large families. We suggest that our approach could be a useful complement to linkage analysis in experimental crosses." />
<meta name="DC.Contributor" content="Lucy Crooks" />
<meta name="DC.Contributor" content="Örjan Carlborg" />
<meta name="DC.Contributor" content="Stefan Marklund" />
<meta name="DC.Contributor" content="Anna M. Johansson" />
<meta name="article:published_time" content="2013-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We analyzed genotypes from ~10K single-nucleotide polymorphisms (SNPs) in two families of an F&lt;sub&gt;2&lt;/sub&gt; intercross between Red Junglefowl and White Leghorn chickens. Possible null alleles were found by patterns of incompatible and missing genotypes. We estimated that 2.6% of SNPs had null alleles compared with 2.3% with genotyping errors and that 40% of SNPs in which a parent and offspring were genotyped as different homozygotes had null alleles. Putative deletions were identified by null alleles at adjacent markers. We found two candidate deletions that were supported by fluorescence intensity data from a 60K SNP chip. One of the candidate deletions was from the Red Junglefowl, and one was present in both the Red Junglefowl and White Leghorn. Both candidate deletions spanned protein-coding regions and were close to a previously detected quantitative trait locus affecting body weight in this population. This study demonstrates that the ~50K SNP genotyping arrays now available for several agricultural species can be used to identify null alleles and deletions in data from large families. We suggest that our approach could be a useful complement to linkage analysis in experimental crosses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/08/01" />
<meta name="citation_mjid" content="ggg;3/8/1253" />
<meta name="citation_id" content="3/8/1253" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/8/1253" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/8/1253.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/8/1253.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/8/1253.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006643" />
<meta name="citation_pmid" content="23708300" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1253" />
<meta name="citation_lastpage" content="1260" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lucy Crooks" />
<meta name="citation_author_institution" content="Department of Cell and Molecular Biology, Uppsala University, Uppsala, SE-75124, Sweden" />
<meta name="citation_author" content="Örjan Carlborg" />
<meta name="citation_author_institution" content="Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden" />
<meta name="citation_author" content="Stefan Marklund" />
<meta name="citation_author_institution" content="Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden" />
<meta name="citation_author" content="Anna M. Johansson" />
<meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Uppsala, SE-75007, Sweden" />
<meta name="citation_author_email" content="anna.johansson@slu.se" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=Ö. Carlborg;citation_author=S. Kerje;citation_author=K. Schütz;citation_author=L. Jacobsson;citation_author=P. Jensen;citation_title=A Global Search Reveals Epistatic Interaction Between QTL for Early Growth in the Chicken;citation_pages=413-421;citation_volume=13;citation_year=2003;citation_issue=3;citation_pmid=12618372;citation_doi=10.1101/gr.528003" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Conrad;citation_author=T. D. Andrews;citation_author=N. P. Carter;citation_author=M. E. Hurles;citation_author=J. K. Prichard;citation_title=A high-resolution survey of deletion polymorphism in the human genome.;citation_pages=75-81;citation_volume=38;citation_year=2006;citation_issue=1;citation_pmid=16327808;citation_doi=10.1038/nm0106-75" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. F. Conrad;citation_author=D. Pinto;citation_author=R. Redon;citation_author=L. Feuk;citation_author=O. Gokcumen;citation_title=Origins and functional impact of copy number variation in the human genome.;citation_pages=704-712;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=19812545;citation_doi=10.1038/nature08516" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Craddock;citation_author=M. E. Hurles;citation_author=N. Cardin;citation_author=R. D. Pearson;citation_author=V. Plagnol;citation_title=Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.;citation_pages=713-720;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20360734;citation_doi=10.1038/nature08979" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Dorshorst;citation_author=A.-M. Molin;citation_author=C.-J. Rubin;citation_author=A. M. Johansson;citation_author=L. Strömstedt;citation_title=A complex genomic rearrangement involving the endothelin 3 locus causes dermal hyperpigmentation in the chicken.;citation_pages=e1002412-e1002412;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22216010;citation_doi=10.1371/journal.pgen.1002412" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. Girirajan;citation_author=C. D. Campbell;citation_author=E. E. Eichler;citation_title=Human copy number variation and complex genetic disease.;citation_pages=203-226;citation_volume=45;citation_year=2011;citation_pmid=21854229;citation_doi=10.1146/annurev-genet-102209-163544" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. Groenen;citation_author=P. Wahlberg;citation_author=M. Folglio;citation_author=H. H. Cheng;citation_author=H.-J. Megens;citation_title=A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate.;citation_volume=19;citation_year=2009;citation_pmid=19088305;citation_doi=10.1101/gr.086538.108" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Groenen;citation_author=H. J. Megens;citation_author=Y. Zare;citation_author=W. C. Warren;citation_author=L. W. Hillier;citation_title=The development and characterization of a 60K SNP chip for chicken.;citation_pages=274-274;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21627800;citation_doi=10.1186/1471-2164-12-274" />
<meta name="citation_reference" content="citation_journal_abbrev=Pigment Cell Melanoma Res;citation_author=U. Gunnarsson;citation_author=S. Kerje;citation_author=B. Bed’hom;citation_author=A. S. Sahlqvist;citation_author=O. Ekwall;citation_title=The Dark brown plumage color in chickens is caused by an 8.3-kb deletion upstream of SOX10.;citation_pages=268-274;citation_volume=24;citation_year=2011;citation_issue=2;citation_pmid=21210960;citation_doi=10.1111/j.1755-148X.2011.00825.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=F. Imsland;citation_author=C. Feng;citation_author=H. Boije;citation_author=B. Bed’hom;citation_author=V. Fillon;citation_title=The Rose-comb mutation in chickens constitutes a structural rearrangement causing both altered comb morphology and defective sperm motility.;citation_pages=e1002775-e1002775;citation_volume=8;citation_year=2012;citation_issue=6;citation_pmid=22761584;citation_doi=10.1371/journal.pgen.1002775" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=S. Kerje;citation_author=Ö. Carlborg;citation_author=L. Jacobsson;citation_author=K. Schütz;citation_author=C. Hartmann;citation_title=The twofold difference in adult size between the red junglefowl and White Leghorn chickens is largely explained by a limited number of QTLs.;citation_pages=264-274;citation_volume=34;citation_year=2003;citation_issue=4;citation_pmid=12873214;citation_doi=10.1046/j.1365-2052.2003.01000.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. H. Kerstens;citation_author=R. P. Crooijmans;citation_author=B. W. Dibbits;citation_author=A. Vereijken;citation_author=R. Okimoto;citation_title=Structural variation in the chicken genome identified by paired-end next-generation DNA sequencing of reduced representation libraries.;citation_pages=94-94;citation_volume=12;citation_year=2011;citation_pmid=21291514;citation_doi=10.1186/1471-2164-12-94" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. R. Kohler;citation_author=D. J. Cutler;citation_title=Simultaneous discovery and testing of deletions for disease association in SNP genotyping studies.;citation_pages=684-699;citation_volume=81;citation_year=2007;citation_issue=4;citation_pmid=17846995;citation_doi=10.1086/520823" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Korn;citation_author=F. G. Kuruvilla;citation_author=S. A. McCarroll;citation_author=A. Wysoker;citation_author=J. Nemesh;citation_title=Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs.;citation_pages=1253-1260;citation_volume=40;citation_year=2008;citation_issue=10;citation_pmid=18776909;citation_doi=10.1038/ng.237" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. McCarroll;citation_author=T. N. Hadnott;citation_author=G. H. Perry;citation_author=P. C. Sabeti;citation_author=M. C. Zody;citation_title=Common deletion polymorphisms in the human genome.;citation_pages=86-92;citation_volume=38;citation_year=2006;citation_issue=1;citation_pmid=16468122;citation_doi=10.1038/ng1696" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. McCarroll;citation_author=A. Huett;citation_author=P. Kuballa;citation_author=S. D. Chilewski;citation_author=A. Landry;citation_title=Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn's disease.;citation_pages=1107-1112;citation_volume=40;citation_year=2008;citation_issue=9;citation_pmid=19165925;citation_doi=10.1038/ng.215" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. McCarroll;citation_author=F. G. Kuruvilla;citation_author=J. M. Korn;citation_author=S. Cawley;citation_author=J. Nemesh;citation_title=Integrated detection and population-genetic analysis of SNPs and copy number variation.;citation_pages=1166-1174;citation_volume=40;citation_year=2008;citation_issue=10;citation_pmid=18776908;citation_doi=10.1038/ng.238" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. S. Ng;citation_author=P. Wu;citation_author=J. Foley;citation_author=A. Foley;citation_author=M. L. McDonald;citation_title=The chicken frizzle feather is due to an Î±-keratin (KRT75) mutation that causes a defective rachis.;citation_pages=e1002748-e1002748;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22829773;citation_doi=10.1371/journal.pgen.1002748" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C.-J. Rubin;citation_author=M.-C. Zody;citation_author=J. Eriksson;citation_author=J. R. Meadows;citation_author=E. Sherwood;citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication.;citation_pages=587-591;citation_volume=464;citation_year=2010;citation_issue=7288;citation_pmid=20220755;citation_doi=10.1038/nature08832" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=K. E. Schutz;citation_author=S. Kerje;citation_author=L. Jacobsson;citation_author=B. Forkman;citation_author=Ö. Carlborg;citation_title=Major growth QTLs in fowl are related to fearful behavior: possible genetic links between fear responses and production traits in a red junglefowl x white leghorn intercross.;citation_pages=121-130;citation_volume=34;citation_year=2004;citation_issue=1;citation_pmid=14739702;citation_doi=10.1023/B:BEGE.0000009481.98336.fc" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=X. Wang;citation_author=S. Nahashon;citation_author=T. K. Feaster;citation_author=A. Bohannon-Stewart;citation_author=N. Adafope;citation_title=An initial map of chromosomal segmental copy number variations in the chicken.;citation_pages=351-351;citation_volume=11;citation_year=2010;citation_pmid=20525236;citation_doi=10.1186/1471-2164-11-351" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Wright;citation_author=H. Boije;citation_author=J. R. Meadows;citation_author=B. Bed’hom;citation_author=D. Gourichon;citation_title=Copy number variation in intron 1 of SOX5 causes the Pea-comb phenotype in chickens.;citation_pages=e1000512-e1000512;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19521496;citation_doi=10.1371/journal.pgen.1000512" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
<meta name="twitter:description" content="We analyzed genotypes from ~10K single-nucleotide polymorphisms (SNPs) in two families of an F2 intercross between Red Junglefowl and White Leghorn chickens. Possible null alleles were found by patterns of incompatible and missing genotypes. We estimated that 2.6% of SNPs had null alleles compared with 2.3% with genotyping errors and that 40% of SNPs in which a parent and offspring were genotyped as different homozygotes had null alleles. Putative deletions were identified by null alleles at adjacent markers. We found two candidate deletions that were supported by fluorescence intensity data from a 60K SNP chip. One of the candidate deletions was from the Red Junglefowl, and one was present in both the Red Junglefowl and White Leghorn. Both candidate deletions spanned protein-coding regions and were close to a previously detected quantitative trait locus affecting body weight in this population. This study demonstrates that the ~50K SNP genotyping arrays now available for several agricultural species can be used to identify null alleles and deletions in data from large families. We suggest that our approach could be a useful complement to linkage analysis in experimental crosses." />
<meta name="og-title" property="og:title" content="Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/8/1253" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We analyzed genotypes from ~10K single-nucleotide polymorphisms (SNPs) in two families of an F2 intercross between Red Junglefowl and White Leghorn chickens. Possible null alleles were found by patterns of incompatible and missing genotypes. We estimated that 2.6% of SNPs had null alleles compared with 2.3% with genotyping errors and that 40% of SNPs in which a parent and offspring were genotyped as different homozygotes had null alleles. Putative deletions were identified by null alleles at adjacent markers. We found two candidate deletions that were supported by fluorescence intensity data from a 60K SNP chip. One of the candidate deletions was from the Red Junglefowl, and one was present in both the Red Junglefowl and White Leghorn. Both candidate deletions spanned protein-coding regions and were close to a previously detected quantitative trait locus affecting body weight in this population. This study demonstrates that the ~50K SNP genotyping arrays now available for several agricultural species can be used to identify null alleles and deletions in data from large families. We suggest that our approach could be a useful complement to linkage analysis in experimental crosses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8.cover-source.jpg" />
    <title>Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8smBI9eORqU0cqcRiamMlyeFFBg24z3yBHbTtMlioIE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459512","apath":"\/ggg\/3\/8\/1253.atom","pisa":"ggg;3\/8\/1253","ac":{"\/ggg\/3\/8\/1253.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/8\/1253.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/8\/1253"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/8\/1253"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/8\/1253.figures-only":true,"\/content\/3\/8\/1253":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OCEYTaIVEYFgGS6RV+V6dwJDOYOX7b4\\\\\\\/GljESwzMcyZ6MEdxwZwkEy03ooRN9Rpn2pFuYjoSaV3S1MmR4CydLa2pbDep3nH5aGM9LEWJQ6HkKipPzb1mpjZ2QYgoWtzhrifdgtIh7qddu8eiZJUQ1n2AywIGqGiMX05gQRjzSp8ozrmxTuL2cCT5cBBfxmCtJtG5tAR3ENEQSYKu\\\\\\\/TqJRQ6Nl\\\\\\\/5dEBJ15UIs2HGoBVtxtvJ3op3Y3l3pVMF78V0uzYObsgPCWsHAI5\\\\\\\/+K1gHoSDF2FXavUXE\\\\\\\/7aa0CzFCpZwC3irTQV1yxQQNUh3KBDzElJEsJacIemw36oVO\\\\\\\/RQkMqgH38O0L\\\\\\\/rUR5vycGtR\\\\\\\/yQ3QeZmzYZhCpCPNZGFuT7UwmONFWZ9cAxIO7IXVyKI2fsZjWEy6HwCgBVn8YHl7qtPIm190Xtl+Ank38u1cBYTRkHcmwRyqBBixH4+fVtgR6XzvR55hR+bbIDs\\\\\\\/2+tDw08MwHBrTP+eJiLl56Xtb33dhVwv1kYxRyoBN1YHE\\\\\\\/9fMRytjUkQ07TuC2RMWD6c08qfESYoocTd\\\\\\\/xNb0DcjyDKqXY60ul7qb6M+Y9IhIckn\\\\\\\/H+7J4h307X5MoYvrbf3k\\\\\\\/irVruu+WZjPZYYmfimSvXXIrS25mQBkqNShPlKN0izQgt1HzczsEmzrCbMrtREiJMGLPnpF2Ujst5DHTOmOzsotzl61SXWOuJuCIqdDsfhrIP0Nnuql+4d6SwVQEi6yybKtfxMCHUSVWVFgyiXOlDKxcUqIXiNa8Qro2UPTNaFEfyXWJQN3walIOR2HeJBJp4vEjEYNTsTmVkr7GSyet6K6ibt2VQIHnXxyHlZW1S1sQwosyUWMOFAGSsrQYnteZHBtuuQImu4pJrFGOWf6Z8ztdqhMGVbUS4zjhs0VKFx+w53hK\\\\\\\/yQZIOx7eRF572K\\\\\\\/4526zkcEU4+ngBFnSqEW3tMFNuZy\\\\\\\/Mi\\\\\\\/jKQplJuvZBN9+w2aAyLQUt4cv4ph4C83gH0EpFFqjpAcgPHp1LhyzErcjJ3KKFjqr4AGb4lM1Du1v+8BzmPE9Stl+PyxMMfzHsgVv255k5nb98I5znJrEkaeioGXE74P8tSFXdVluSgZwC7rFy++je6eGxhfHJYApyqioD46Wt4jydWJKxFoT5+QOUlk9d2TbV8GjvgR+4ipzQ5eI1aUSsRJBqdmjLJ9FHPQzadgIG1O+FyVxkUvJ+u2sWKrdZDNLteEBkJyVJ+RzhiXovm3Kg93a7Yb+DB\\\\\\\/UT\\\\\\\/k8pPr2z\\\\\\\/JMvqK5xmS\\\u0022,\\\u0022iv\\\u0022:\\\u00228dLHaimrwCVd4gq0jQpndw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c60516644bf90ba0524973203fdd5d352962fb95ff13aa01e09d8a380fb7f8a6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vLtIo8XpoQmwLsdz87vZly7cCUizS15gFd1V1IpB84Hz4aEFvKMl\\\\\\\/wvhY4WYbwdteKp415YIAxGoYx1muXgMWPUV8LKhEQt6h4PUodjcyv9PChG4MjMhh86zRuGXImF+yjvkGjeSb9\\\\\\\/7GMNrwbZErGzgNVHNwrN\\\\\\\/qmQpISf3dMLPdxgFxHnhZJPCcBsdfP1RF6Ro4tMtuEacKq7sLiorzGrW+OxrJgXTLlo6CYo1Izocnh+sPAJt6JyQSzsoIYZjr5x0waOnLuhFh3Sw\\\\\\\/1KV\\\\\\\/OUxxLanjzpRQDyu7csA8PoFuYPZpnEC1\\\\\\\/99qfq6Sm2tv1APW7MPE4O6K5t0gfSKatpWCwTK9evDwZQvuFFqYoxQnjnUs5G\\\\\\\/cMzsw3op0EePBZLkc5nw0mIpaTLDxhWybGY9cTilaNj5t\\\\\\\/ZR\\\\\\\/bFkV\\\\\\\/b+4uXSoNQbfhWn0qJBlJtnHuPNKNwinSyTquY4ZpLSFHBMH9+d8ud1uo\\\\\\\/on0DkC7taCiJYWBChiLqCl1DxwH8TWcl\\\\\\\/eyRNZBAtinaI+PzishxlLQpx2H3xVT7S4TuMnRZRfgRvlNPQzFPFLIs2QEdPJ28fnqTfMNShMocs2YbZuYPH\\\\\\\/FRVyLdI\\\\\\\/HZ7VH7SG7GoxX4yKIpDGScFylwSKZBwxwT4jGTEa0Pix7NWh5ffXNQ59YZjmXv5RF8R5SL2aHuKYQ+bgSXyOhPSJgVp3zv4Y9XkCKbxafoj4O4T1BQY2etYnBRfuBYo9vc\\\\\\\/KNcdDsfZ30EwClyr7K\\\\\\\/NC2CUDKVqH1S4miChOKjZkwdsIUY+5IKr9\\\\\\\/s3lOAIhzkKJt76eA6rk+D8XaCcdfgs4IYuSefTRoIL65QgTFkDXXPDpNRBcBe2OmBUIPV+GIUxAh+UGKHwg7Vf69cDJJjdBayed2why58of7gjBx0Tm+z9GHOvvOJjeQO65iQjp7MxwpFLX6bWyiLj2q2pFnzVXtpCQKZ\\\\\\\/6CsgFUvYsDT4cxUt3iDzrTeHFOPUqn\\\\\\\/7PiJoQIbwYJ63LXikxNSIpyFcFhs7MO0wjbJnzBWi8hKOVtSHfyDw141JSjYeagZf3syVsEqcdCgJ15Iu2p2AELkzmuxR2HxTj\\\\\\\/1Qi2xMPQ5EJ\\\\\\\/0+ycYcNGi920NdexQUnGZzM680NFDvHwDiLL4BlqwR4IXPBJRT2dYL+cu9lk8hKbaIvHWsW7aSJs8xqq6Yf2OgN3LKA5GPJ5jSYWE10YDiTkgx29qipAQNy4pwiz3cVP0MKwFelgXQzIMtBLmB\\\\\\\/BZWPdtpME\\\\\\\/x3OkSv0Lqt0PMT782bEtxPxvVoA3Dxs+xcJhM0hcoagTeiaC6IY72P0Hp2GGYCErFF6omCUcBQ1X6tTUMrrgEPJFgFxoHQaI3XvInChglMW\\\\\\\/dgRoyy\\\\\\\/wCQGVv3M4Rjf8z13qP0JzXMkwJR4k7qQxu3ZdLfqDZHf3O0SuCeO0iUhSW99\\\\\\\/XVQngCtaTAxbkXNXmT0Tydy2Vv6suj59nGD9Q\\\\\\\/5NwM03lWjdrK8IfWNzXc68wRTNIADA6wbRnUaepB6Q4mN0j+a5zX46lYuORT6SD1q4sjyS4D2ezOMIqz0bPJMllCVZsJHtoPj\\\\\\\/PUU2oGlS30DxCFMpuynsQi9ZvrVfi3lcEm6hiGIe8CghDzLMbAeqCfN02lYegnHKa3LHMtGxtV0wybNEh4QFbBqXlBq8S6d+NvtcJXtmO+YDM8sxCQqyx\\\\\\\/hlekt8N7DDWWU9iWU\\\\\\\/rmTL42TFlrhlpovLFKPmRX8wUXbMPTjT+66blHrefMQjKoITigLAxAKo1EmHAv2VlkPXdwo7lR+xQ7Bwx6p97pwT6YpJhrklHF4TLM3RqyrWsdp17sPBwirWOcvVT2R4yGreSgvzqKEyrKgb2OZ6ZBG5rtLQKnkRv+zpW67PBunBhWXGQnA0hD6TeDXc8y5LFhhamKXNzxyg7gNW4aPPoaGzpUMq5ozdyANckiwV79Hghnm7gMf3AGcrwmM7TG8vcthhfCOPYsFDWgFk\\\\\\\/bMFrmpHicYUyNwFwF6V6igb\\\\\\\/Ujlzlbgbq6L5cxr36tp3nWU6BGrjxuFx\\\\\\\/hrxPXC81J6Qjkh30hD\\\\\\\/JjmbqgSSoGOxBWn0Ef2bHh3BEYuelpSmg2GgK0mD1l5O\\\\\\\/7qcHeNCAvOlzOKQrtkDnG7d98MOeGZQlYkm3lr2Ddv0leMmE5d6Bh2IYilMzcwebjaA4mi4U+8N+mXbzsIpz+pTFB41g\\\\\\\/yjQd7WGu6xz0A38FcKpW9m+pZOyGmuwh1SwvWQ8pumEcHM7csRrmQhc0ZBHWZOPeCib3V5EuZ3Y7XZ05+ALEEPudmpGR4dt6c2mK98+SJCoVyzvbadM3z\\\\\\\/lwhtKpD7IkFsv1bObj1qB448EAe\\\\\\\/pI6nQ1b+Is6XKmbS8XqjKjI6ea\\\\\\\/OQoWE7es\\\\\\\/WV1ap18TP5\\\\\\\/jll1dbmH8Ul5dNH621P2B2h72cTUowyfHaX+VCEW+B5+mNYQJEZuh+T7rXuTDhhrK11xgwXpmc826t2rG0AQVtn\\\\\\\/+5h+51UzKTpTjCjWdMBkJeiI18\\\\\\\/GM37R8qa5XSUl36Bt71PKo\\\\\\\/9UMz817\\\\\\\/GkZ+CQLZ2ZaIBPIfsgiH0RZAkSv11Uo\\\\\\\/siK5Wv26LsUlBTP\\\\\\\/0qe4XIgL1z\\\\\\\/z4ufNk+\\\\\\\/4Uxbbc9x1Cy7sXv\\\\\\\/p3pakNhNDJDfIhgd3nBxGMozrfrVn5ISJbV28V\\\\\\\/FA\\\\\\\/Zb1FjUIQFSzNKuGsxcEIjKi+ZqUtSoKoKCfk\\\\\\\/x4BPWT7d3kD2qn+b\\\\\\\/dqmJ2l4PIOj9h7I6rJsL7L1WSoAI7NCVke\\\\\\\/wgQNkoxR4\\\\\\\/tZfEQRxad4FekKynz0IwlebbHDHuF3oLk+cKauoEMW7+CvIK4Tf5gxWR07NeOt6T64XnoBVbB\\\\\\\/pS8CArpWbmF59MLl0upuwJ5sxCkzRTBqW\\\\\\\/MkKE6TWxZ9Xd13d+8G4Bevv3Gl4U4jR+96R8qI1\\\\\\\/f3WnvieVJsAxIPnAi0+58\\\\\\\/d2T9gA7zX2Uq\\\\\\\/akle8g4apWNw8NXM2fwglEbg0XH98juSLr0rxeUaVA+578V2VkOuIPZYynNMsU5DLslZn0qRMPd64TkFVD4vwkfJXn5X9yzq0qfuKb+Bhe+qVxOVgzgLPLeVVihsjM2X5z5NUp1unI7hx0MfHZi1MzZGJB310e0I8rNJsjOTYmRPGZgwLsZ1\\\\\\\/rsoeIaQrXMKUhT98qux9fU7wE065THbANpbbYQDeaEgyM7Ketz1j667KFcm2I86Hxp9Hmbhvb1yBbZ4zX7drsTFowU70s4Umo277Dsx2pba134PyBIIJsQuwleCWNdpmpi\\\\\\\/MEWLP912EKdKDPYG8wolqW1Szso9Mbds48EH5iHCd4TAH6yyMGBmkO8V1oprfP8HNpomnH7Z6xGPWz3oqH6u8WhD6GBr9z5GrIQmTw4O\\\\\\\/bbUSM+dzAKRG31nsXzBM76tCMw50nA7eJm698KdIy10r7uQPgFx6R6CxmyXKQ0NLmmwzTTpeJ6hNN4xFXIOQLnrfT1zRKrhSNCDpMxZkaffCW6fTXGztaxE\\\\\\\/3bQhmojQv8dR\\\\\\\/WJAxct2risnecAGdn\\\\\\\/okllLxLnFoHoN9ySbon4EwKfonWJDImlePovqxUD\\\\\\\/QVS53cQdCnrqWCgmJBtPVA9PHbgzJ8OZ1nUJc6B+mwMZdiUtKWK0U+BJFGcV9lPXenGLJ6Z5EuwK+c1LBYQUhhEOKmLqiWQ1adCJNG0qx6ZiHg+ZLB1YWNtMqMVho4muYsmgvQr5MGYlh4A+BkKgFjUgwX2\\\\\\\/fGUakHFI083rwYbp2JQXktwczKxAWN1PDfhoFUKmmiHrlQUycl+Q5rOWpIgV06\\\\\\\/f6opm63LTkcU26sl0xknhryg8y1totMVXVL\\\\\\\/Nj2315x2Bha4b9VFhjzVGD+nmbv2HO\\\\\\\/JHvzdpvbl3UWo4Sk5PY2k\\\\\\\/PTxZifgej1QPZLHgbzCIyV3W38Q2\\\\\\\/C3NW\\\\\\\/k9vBxcB5V93vQhmnXwkIScHpqG3UX0\\\\\\\/QS+2PZVv2WFQQQyGxQ+aGwP9E3u0oikdGE1KeSBLO\\\\\\\/JGAd4NhUj50mR8HXp9OoUyJzNvgYDE32FMk01uZhOgMNNuIqkFYksbo03cjDV2EmNYooMWpJDx3JqwdIZRadiG6I7Yt5JDBrwDE9bvtr6IHC514G9nnqHpPeioodzdhbgCwz6UDRDqc3ujZgt73t7Tn3yJogYZLJEnTVILmAJPc93LW2ptZVRKDoFiRB1rrlhg5Fxx3xUVQc47B3VgyqT97yeJiACRf+tN\\\\\\\/\\\\\\\/\\\\\\\/zXcRXGTGJK\\\\\\\/3naGwDz\\\\\\\/lWACMDSFTyvC1qXrkMSyYzj52UJSCtIrDaPNT\\\\\\\/zzEoDsNN3rbGq97fkM5qPMpeFAx6tLyn83JttO9DspnPVt+4NiRMSFPda6i2JivcH5zhSOlFSI7S\\\\\\\/ygidGLSzHBO5hwc\\\\\\\/9Zq5xrrqarCg9+71CmumE2LweAn38owAagttHCRomWC75iPkAUWa7bVRVvX5jL4+c4dnJ\\\\\\\/wmO\\\\\\\/oPOFM6gNRDCmYsezhHXUJrolO37gogmffRkI4EhO3p3v7\\\\\\\/wauW9pDltW+pVKiDWmqMM+yFrNMUkznjoZMKncP7CWHMNZs2Q81PImNqxl6mDw9hRHlervpqhlXQhMkdQ8\\\\\\\/RWiKWjOpVLDAD8ayTIwnFFUeELeRiVdW8xEes8L385\\\\\\\/A0cal2TngcYiTX9iICNyWIQ6hLeBiStQXQ2n7MeeDqnPm5PDgYbBTo8\\\\\\\/iAUGtY2L3xmP\\\\\\\/RjF8RjcKA7lY7g7I+s9h9+oYn5UGW\\\\\\\/VJwln64fEJJr0xIOfZjZtMl6HseY0LlkwJeoFAkNNFKtP6lqoDrRkU9FYu9y7iPSOrupLKE0hnc9i5YQbkFXV2m\\\\\\\/4ng08HUel+O3Wb9iEjss9+ioq3TjUjsK\\\\\\\/dgb3lsygzwQ0vRWhC\\\\\\\/qJ7UaFE7zJWRuRopOEmbqgWnvHXllz3iivB6d068OvLXvgB6MwTAnZPUtkzpL5RdbJlaNe+liVtS+W8fTohpj5w+A9JsXcEvJ\\\\\\\/aUx1hWZuMPcMNaJukyZiwrikfuDgaqML+bfZHvAzXic9a5WMFonjuQnWQNmmpWJlW9OC8U3rsgq8tO+hKSM0Y\\\\\\\/4ur0XS2SPYxrJzUWZ8R7fcScIuGpb4OcfwAyLwVIPU9lQaG\\\\\\\/dXJlg2MAdzl4jlEJ0OHuxBe6F91imFusEkI63BmnZjgRcYCta3SryF6AfcBztKjtR149rBFUyhd+Uk27+s4u4vkGsTjlnlJAk\\\\\\\/JoAoKZNpI0dXnJSSTS771Gqk6VQCoOvC6quAVlu2uaM+Xo4wP5oUd4QPqfpWe2Hi923CtvAleNzyjo3yVjGtwY8kxywQk\\\\\\\/0fNTuA0+nQLtSC\\\\\\\/Z5n05FXzCK0C9c8nWvW1dvnyoSqE2rQVUdb\\\\\\\/HuhRqkh9Kjy7H3b0MUvYlxjaNzKpLIDe4C5WLk3gh49K7n1ToShKc6M0ihQ6t2H8ACEPJ4xVxMjw81JxlHzMERX52aCxcBAL+\\\\\\\/U09uYbzT4oAmn1rs51dtvRrddh9fxoiYHyMGPvISStB03zCn1+Pl5KjryHyijsuPAp7xu2BCcQJ7Yqbe0S1ZUZAqdpYZsWVFSXupEw4lF4BnwAWUF8iRei6g07CR62ZLiB5l\\\\\\\/srdZ9o1DGf8mIkeVJ2it4IxUtceJnwhnIKjPP8\\\\\\\/Wv3jGbIpVXw6Z59Y6h7mcYDdoMcyhEkY6+TqJHppxn\\\\\\\/MlL6fyrCCu5cRoJYrJtaHOcnQva9jgX60nPGtC2lceSXdAtGh83GkjfcqDqAV4O8gXzW\\\\\\\/2l8u2fD41nYwu5pcuOgMXpwD5D90+cRszASU7EHqWEXhCJ423vyEOwAwJxgHF5Ps0+sc335VtoU9rmQ07DaXNmw7eePE0R6fuVqEx3pZxMZEsrHgSlVGvQ1lTeVD7IybWJ86hsC0ag4Oc8dabht\\\\\\\/0SY7+FIrbTlfh3OdVNprzCAH\\\\\\\/5lNw32LO2myA2D1NtttaqzVLeO14aBiopv8tuEMo2aRJvZJbUUMfJSxIQwLcKybYNUwA2PPe3Kp1CbSL2hSndeoYrlpHWUOFbdg6PU84TkO00VtgYGgU+KXNRveXYmO3iUnNZoSkHmZtFQEP1aEgxXtD8DAjPq00UbvlmwvYrnbQG+a2X6fewoWc+vz9KoAj5D1bEE0HLWdiMQFiSO++Z6XrxWvuj4MF\\\\\\\/aWCh3rPRdqn6xffrJw2t8dHauJmwdrHmQZEShyGYSlKesy72i13JMPYKnU51+wg49PFzar3MGUq\\\\\\\/T0tqua2ipdbCLfVj3JqjN8fInGqq7ZZgVPUppaWiqlHzZ\\\\\\\/+8JmpVgTscXDAqHxP52qXX+rW3zv\\\\\\\/9UEej9iuMpVDyN61CpMWBCFi30xkQdOPgqXCPInSRgceeuXIlePcnROifmsir9b0Ir4WrwhlBtZ6tHQegA36nTVw6tZJrdjvbYzc+rS2hrhe7DOfpGzFiu+9kYREj1aa+yzVRLuiZ05XnmGDhinp1cZNKBUfNBxoHdkCWW9ELHgOrgm211VoASOR5jML1mwEktRMEZPT3n3IPnfe3IhB0PIpdA6J4Mm\\\\\\\/7jpk9Fu7YCBdK7a8I9UibuE8TszR\\\\\\\/uCJahdLzlUhpwKgFHXiAFZpvCWN29G96YlV+wksO5asOLcGGKnlnczugdQfss0L5qh0KVR+9aiszsbu85LOP60BYXQU3F\\\\\\\/6nk1J815Q0Dmh5VHm0iqwH3rVQa7peW4ubxcFshe3iFZ0EKJ7Sr7t0B3XCiIm8PXReHJMD8jEhnQQs8AM3iYNnqh9bEUlCPLT\\\\\\\/+wHXYOfMcnys6\\\\\\\/OLt70m3jpJuExpNgztgmxLFlTY+KCqdLmrmBtXJKGGhh0\\\\\\\/P7rvr\\\\\\\/XHj3jcNS1r9nwKa3dI1xUwNjUYbsa1Fv4YIBSRSkOAAewt+\\\\\\\/\\\\\\\/HlHaa2\\\\\\\/TzwCSW\\\\\\\/9TgV2doTVbF9ZqX3d0MMKS4CwBl50fUiIbhiJgWRFZvPjFiiy8WkzN0HSmdeEug6RXIA7prXVsg+1DrJqVcPW2dLi5xDA6Mydayy48Y4w9U8bk+225k8mzMU6+8Fcu8Z3Gk0iKXnDjNc3wrJreNgXzTU4X0bGL5v9iV\\\\\\\/mR+WxYC6A1cyd0g\\\\\\\/1NzNZtQ03R1qmDhaLqn0yvMlpXpXZeoHFP3+IvB2Ycd38QYGKKo9dLyBiKn0IakN\\\\\\\/t8IBT55Lejt0qekNU9Omgv5PKTVDE4Z6HE+XTZgrVWJS65W7nw7hXTkh4Se9RkTioBSBdfdb\\\\\\\/Obd6ccFxXIIJorv6mmVmX\\\\\\\/wbZpteWtFqlTHF7YOjrc5Q66jB0UkkhPzxmGo1Ph9535Nj3Xh7QqEp5jVlRVcFt6Omcz6o4qzmMzlcdinqoJwRCxc69xVzANcgqv8Olw3TAHiYAFyyKzlpszDI7vcfhQhQcMBuFuI1BcLszJewKNfFbgAcwDjNhXYTkeb3JF8Zalq7ytbSBXh6rvnI7Aw7WVdZR5LnB4a2lyy4REA8UAbZUIXbl5g8hRmXrjm6d8Q1kmPPLBn\\\\\\\/Ekl2BhzGX1R6LpmzHLutVGD2w++cd2BBOCbrvglDVPInWU5Dun8FIehjJPVVaYuimJCezzl9aht8ldlUAqfdEmbjn\\\\\\\/Z+\\\\\\\/9XYwhybvBwqWVspztjzWrc+SP3XZgPhpnFeYzKgBBErtmXMQoFlwu0OtNMxdGue66cmLvytAPDiVq\\\\\\\/FtHh4JDUkaGABtMH7wv3YyCgykKDD73EAB9beRTdTsJbavh0USpllrm9dEhDLdp1prDmj7h6\\\\\\\/e4WRsoU7U8QAXzPbsPxh0JCEUWOnlwLlbvsF+Fhe5ijj2wvYsjMlHpl3oObJdGMkmwMlcgQLbxyGnKcoZsIN7c9RlcEzY1Uk+xY5A\\\\\\\/ANFDNm9qLUlq+zrW\\\\\\\/jSW811hCLDmCzRZAItXDWx0TEaoxpp0wd8zJhPfFUjCKnxIGzgXrR4fZlWwoJlq5PUWaUS8+Hx14Spt0W2S0a1Kgx9u8pAsNoNd5jT+saQJeTOzbPlMzKytjiSwQ6591slDQR0IIKEpMjUO9xozzrvKrCWazfc78hLh+JBHqpcCqC+wHicGJsffQaEm4DN358RY+4k6PkLl0WWU7kBd654ynDxFrviVXWFKcvf4HDzozQicyqYGbNjk5Nqi\\\\\\\/bN75wcf7rvLwn\\\\\\\/zBnnRX38AQhOzYHJYCMEtyZbdhil2AD\\\\\\\/8NeY\\\\\\\/0wbkH9J+RBTTKCsPBLy3mFToiYzNcgtjoVfa\\\\\\\/RPaHCR2wG2hG3eCZM+DgabZfcW9ZfI54x5aYfwTBqgNRfY0dP17DMaZopu9aYSjFMzK3XsaK5jmiNJ60Y78iRZupn7xrw8e1NqMCAn\\\\\\\/4vNn3O8pFBjvLgCU5JMH8ZUlAEPpg3edWEd0dNiCWHGYA8u90rnJgb\\\\\\\/rE8MIwTZ+c8tANqif\\\\\\\/5VXhw2hrLOzLCeU6UIwhk8ZNU0ZGHR8hhNdSA4N3MvYZY9wnF0+YmbbZGrMFeitTebdPFA\\\\\\\/Tvsn4X4ai57YWlLmsWWWkn02QRii1LxVHbGQM6vgZ9RszHzdstkChofIfaSgE6MDd7MQ1jSKwbXNSxJhGCl97sm+KNmIjpBQtI1nDzPQQo1gedg99r8ipyW1BNWphNaeuRMyQiSt3ZELlpHBl5nbD1s1DpQYdGXyQCPs7jmCvD9q47UeQC2JUzhNOSvl3g\\\\\\\/lV0nhCIclWhnFGD4JJzM1r7Ya5oETudqS05a17fJzhCm3RA7G0BtNeI2M+Fv\\\\\\\/bXnWDReX6Z2dnKgWhiwhzjh6DP4XUOBXoiNvVqKWk22twk8dmdup5OKQfkoXefs5LcAIhFBJ2O4i6K1byxVhpwkJ7BF2Yo0cJivx5p8kBLQYFmcy\\\\\\\/zxkyteiizv45Z1Cq\\\\\\\/+x0qAHhQCev5Sz6\\\\\\\/14iKrxCnrSzyfeFB4Qw2cXBJ8F6xYxO\\\\\\\/jqODhS698sWPB3tWQEkm9ue9fJ9VPqYayLhEexxUSwTbgn6wN\\\\\\\/Ih+EnBgab76s4dteQehRkw711F9+49dbzIMoFEaS\\\\\\\/xEllgUkoA840+W4WoRiFBV1wImz6Jj0AJ62LzoFhbVtvk2QVPAh+GYFChMTBtn8P+4pk38LkDDFhdAcfLmCpuP+M7pIQLnJSBhfsYgQuN+KGiNMXTmuNgbZmUpA4i5pOBhK0HHiazFcdrJtoUFtqYQd8rq7jNr+04FFFphrAmruyDjFirYxNSItDXHlk\\\\\\\/AynClsTZCkKXH2hwc0QarVHs2c3QOXQ7rwhcaFPF+5MvxrLkzY9VuDQVh2goFto5cbg8nhhogpzBQiY2n7CCqkkx3isdzL0gavxFV+qyrkmvqZafsr83nuA8l9y+H0hosGWMywOeE5a1t8pKWlzrCR\\\\\\\/yl\\\\\\\/A5DfHkMQ83vQOL34bN01hgyDMbT8F0WLH\\\\\\\/\\\\\\\/nwCGOCHeDCxrSo7dADbNaZy5\\\\\\\/hza6prETbK87QHE+acwRdznTz2lE4dzGRnnjJGXY9CJvdMR6OFxTwv49FTUKcZ22731c8Qof4tyEt0Fi1UG20cRjB8FruQqLse1+MU9UNduFR15eVMNQPE5nQi0pfgVNMdQRnw0bDHYE5xHdf\\\\\\\/CMr74k8Vni263bNh7GWdqntopHeVJWww8R\\\\\\\/siS7uTUke2W9Gu7Gi7eOWG0OEz\\\\\\\/602cUUYo3iOOnKDgGVj9HMIV\\\\\\\/gemBxkzyvfY8NpseIe8oM3+\\\\\\\/vuKXcpCbl6hsVCOr54zjQnA2Xc865VBJ8LBSvMpMpZ9ZUsfozDGsoo5vqj8Dz9nMqw90QyllDVxHM2YKvs1DWZD7PVzqBb4gWuGPAPEgRRT9ZtgPU7plBq6hXEa7TU8FStCCs5LqUCg1pttQ+6PcV2N22jg2IRhWyvsCMjFLAnSZyY6EwMZ5O+EoyFjr4o9ZUauyq5s8RnWKP+FVwhCl\\\\\\\/E98pesGBXWtfEWHzDMj6xJ1KBlZCMnCW3hve2BJsWLzvfeNRj1tHijXSt\\\\\\\/645zqtquFND4qVI5G9+J\\\\\\\/suWbM1sSKqSq07OUSqpfhxjat1DzLG7\\\\\\\/JkRSLwcLDWiqbI\\\\\\\/Na4Kqw\\\\\\\/o8m1ZCVuF5ijOaz\\\\\\\/56xl7f06YqNnp7\\\\\\\/zbifj6V\\\\\\\/K3z22o+fSyn7P3cWMQvcF45M+u6eLoUC\\\\\\\/v4ClRCik\\\\\\\/G2blk4xzZyZt1NysH5HbS\\\\\\\/hDuXw+iSzrDnXQFciv4YAJQfS6sF\\\\\\\/vBMn9yq7y26cfaeyTD+sG62XWmSUhLlPSH\\\\\\\/BgATH5IC3rfR5NYVNNQi\\\\\\\/9MH6ufNVoMsln6KnHfjEtCAzkJvwhdpH6TkCXuCsu1kEiNizbF\\\\\\\/u0rUbLMCZksBxvDFM+mXasY0watE9aWrdwzf7nSBoSoPnyfY4s+\\\\\\\/qVqJTxncE1opOtvdpR2qS1V6LInQ3iUjniDYi5ypzNSrzkWXjgyf9RYcpyUr6VT\\\\\\\/KEAzbio05ddcpwglTVYlRuXCcq8W0HU2anuSQBjfcOkxhfHyfmjzBpe4OTcNJDtYoOJuqFJOyWpgQqTiOylmYwv82QlFtrAJCYCOsXiN\\\\\\\/V410t+vBby+3csiBbZNzuJeq2uvX5wFEqs5kXJqyV9ptmxGm1KIzUQcbGKsAg2QUNdBDsiY1WD5V\\\\\\\/nI\\\\\\\/2\\\\\\\/\\\\\\\/kBQLJQcxmc+EgkiuJXkQPzvqXuEfMMExKrRzI9bgCMr56qrc545SURBr3n52J+dB3YI1UR4p0AOhsU2DTd6u2UzMaD5VSkusUx2sAQH\\\\\\\/0dswQ84ydSPIYEg6GPQn4bbUZYM42YEGMVbxjtz1SG0x8C1\\\\\\\/a\\\\\\\/fG0tGDia6qyMH3YUR0xCqY6lUHA9oxB5JMIn3eR77GNdmflCotriYNpeRRfXJjs5GcB7gqaCQ0413aTsaz51Eglpld55aIJKjVd1\\\\\\\/1JiQnMoKhPUmFR6IGNO54rrtFlHg3P9s+3jACVBTxaz7rB0BfHVBGaaspS5k+RKl90d+D9LLq2OVw3+1gomc3dkv8PTFWflERfwe+dFjyn1TMTk1ZU04Sptu\\\\\\\/uqHluFD7tkiQdEPXK0QqtD3y\\\\\\\/WMeQ2ahVqeYO2UTt22nA13jrtOJf\\\\\\\/y7h5Andomc5WYnjNuuQuj\\\\\\\/Gd0EPdKFMbVEmoErLxePF2FyBvDTt+1eMIM64Tk+SpHMGwBCbbS5vqntnMSwvhTZ8Dka\\\\\\\/JmFD0O+ig\\\\\\\/6uHB5yPaThDSPVccjU\\\\\\\/p3miSY84Jt1OhinJcnIxlMrh4nLn1rP\\\\\\\/3AKmwnj6V563le0orw68j3yKpt6T2Q7wvUoUXtp2AXn4YH8nQYecFVDZghwX+YWMbYeKQ5UaJ2Zk3O1N1WUDbZL7VWoPIEMRsjppSh8cTioO14fDfSDOwZaRLlw2viDDbnWiWrCeo+Sw660yRdxCjtLtzLjwGj0QH687Tf9vS7Agjjt4aQHe7TgWs8vslpu7DMiDMtBcPz5gr4++6qyYKhmhEXWCVK2IHZzEYJeV7FpjrxcKoO4COmeO+N\\\\\\\/kh3iMcO3\\\\\\\/omMfQ\\\\\\\/2uKcm2TM1m8YAP2P2PH0j57rYV16haXFb3RrDNNKeaCo6GZa3scNgoiY\\\\\\\/9RLvcZdIq5pfrcV7sM83CorI209HtuZj8WRwskpIt2ysxh3QyWBmRredicPMBLDe\\\\\\\/RX+2aA4iQf+lWfDJ7ErWeTzOlx7YE\\\\\\\/5niFYPKnA3xXniUIlkqqLqdAYh8VmSVl2pZZQMVva5GlxsQZqczQd7GwNHCwJDiA6S0jo83TpyFQbETPhBus\\\\\\\/pJCzVhv94cuRk1FFO+En06WMtfsyrv7DDhxWcqf8aK0AH1g5bazfw8IMv4nXN1NRpx9ASknRRear13BRwS25E3qmuSezzPR+Kpwwfej7cBkn8oDWZkrm6gsXcmQ4\\\\\\\/PudqI85drV\\\\\\\/gHeiYIie9TvYOB8hW\\\\\\\/TEDhg57JRaw+oZUcD4julMAfxXUcAq0db1ZGfFG40idwMxIrkYGViOc7n8pwSaMmEFKXwIjvNot1NCO0qA7VaSsHs2VJDSvzrkeyrEiaK+pAglXRY846vmde2sYtPKQAnVMNxlKwh9DLwXRRg3QdVXjLwUNwW0c0sL50d2p6VfKHGuX0ahfuXX87D7GITHJCM5ndjCPDmsKVyUWKMuXUPWgi41bdDjR8EQADadRTGnbo1oHc97G6kANrxkmC81QkryGDew8NSQNhVZ5f4qBmzzkCZ2iVLqHHgen08NzZfAbuyJrB1HJ9wZFBhbB85wbrXhLVw1at85UlOYVEN0TjV1UnxiKp4GUldyYEg9rkM+IVvfrf5VpulDJf\\\\\\\/7p7kDdeI5jCbdAbZHjvRoJKbmg3BfLcV\\\\\\\/3yhSw+j798AS5zkw+VlEdICfjaWFGLMmVbAL+1U9njkmQ\\\\\\\/IW0dExZO6+H62acYzDTsMp3TqEyP6GIuynn6oRsOjhtgRkCb9p6GNvyIqLdn67oaReUcQGnOH2zcRqWcg2bAPG1gpO977jQr5cVrAHIAttVQAJH6uJkwUFAGeNy\\\\\\\/vDnekLwbVYlmayRK+X3AYnGPrAQ3hR7VqQzqIfZYA+m9vADJCegwh+XoO7ewtNWWN7153jH6IbKCTuHKc3QrvCVs9ZRSNxsCmthYEic+0An5CbR5+Cr53NiutRCpKyVIRV1XQtHqhZVRRWcjQiPAELcD\\\\\\\/9zf+BGuKrsCPEzUaxFy\\\\\\\/ANhl5I2XMgxeTd4bIMOFlud5C8\\\\\\\/uE8mqtK5d8LsanHFfkb63BpnQitCFG689ACTPoRbHUwzCW2cWO4p6mCGseP8xyfioYwoiiWHPyShQ1SgI7\\\\\\\/5D0+qLche1UB+Qyr+1\\\\\\\/BSQmPT78cNnyHeh6Ub+AH7XwRYUBn3osMVrnNGrqFWFKUa0IAwBXCfsrfjEOPlHoF8IX1Ipf7LHD7nmY1ti7nl1NTGWJ06VaYgvDZ3VXDhBPMHnY9EMBq\\\\\\\/\\\\\\\/IchJWKlABqPVlePuzvhaUUEk2twXnP+Lh0ANltGirnOPSMBe7UfMj5kMGkkD3q362kP+eS7C9E1Zul8WnzWId0naZ3Pk1jr4o+0dg9YZB4QtMdci13fKCYUHrkitRdABXUqA8yAZTV+XfR+4hUOZ8n1am1Oh5zeyk8QSUgtJNYcCZ8M1uD6ISAX4HwQeVfa5zwprmr29zdU3Emp\\\\\\\/I1lXnsX8K8FHYBha1hKH6w0H8JFho5mbPhtLOk70yf5NrIURwEQB0HmaENBKRtuUGLTSDlA4lO6\\\\\\\/m3R\\\\\\\/UUUI7xbbTSJfbgU7HL6lT8I05gmirbsXJw+2Ff4yPzj6tv13XP3cmd5VwI0eBNkjD1qc6UqwotFf7vvaCir42EVJ2VaohmKBqi9pwU0a9w2BPQI52F6jrtGegdohEyEU2PHvxqJguGpijTWU3UeU4+Z5zjYtTCTlwx+Zi6uE+bnGy8L0W80LeA1jpLhifTe8JxPetANWA1WfO9qe4TBpHn2nHwS8h105eQxYfJpu2yO8dyKiA484VWvG0s6E5WFI0ZnQVi04mIcCnZcirlcPfaWnTvsOUPlqXEHCJtzGBlKNMGKBu47v1BMHhHJue38P0Iq216Y2QIEv2TROvRV8Q+Cm2NtzsnAHC4SKqm50uJriWBCivmLZBs3qovTc3cXMzHat17K6RwvBiYicNYCuoEWhItKzGAO2vf\\\\\\\/x6G5i884vYIGzpx3VyLXSHX0D+BQ1UREMNBu9fdNT2PAw9LhfgVcRlJYezP7VhNhslSvzsMAOsAzRK30lwDCDIn3gAl3DJN1Ee5M5PcRHOv9hxdOc+IqDQ0fm5\\\\\\\/2K7pL3SUNttCw4nloPq\\\\\\\/BezDSWUVFctReRFiUD1LU7npptLaknDyhlZ\\\\\\\/u4U\\\\\\\/0U6TgB4\\\\\\\/+ezUxgMQhFO23zu0s1YRgnkM7Djgm\\\\\\\/qECdNykNoMOXPapqhvrlKN8qkkfMu90Hxs239MeUpdyKxie7B4tH+Ju966Cd+PyD4ueSktXZlibey+c\\\\\\\/7IWfB1M2PNSMtA9rGZASaU7M7ybZ+cp5ObQnhfMDePKlRBEMNuPXDmuHhW0kv2IaufRcSi7Ov1ilXzRcee3WL4jyyQ9FYPRyULf96\\\\\\\/NhS0F7eWp34tBB13aOgGM7nTL\\\\\\\/FJDm9r58ix3Axot1RfR6jdSG8OSTzFhLGaMKyvCz9LEpgQaECokcdV72DxpuIVqs2bvo+5l1dtokRjuIg88duxXYJU76C9f2PkJ78zkkm0Sf120KrnDZqrfxeDpuliir8f27qrOyQTIXk8DOlQPyoWQGlMMq0jvqfQJA2MOHNqyhIM6lJ9MaEI4qhcrX\\\\\\\/TwddYSDzZF1orU1bpxK3j+kI7i6xempwMLlmaiV8kYZVHa0+j\\\\\\\/BktAZ6GRPn3S8w5KAIBu8YiS6JRctkwx2472zR6L6OC0maVZytAk69K0XFeeqZ1gdSSM9RWh3CDbyENLeovwpEepBTORMq+HgG5f+QSaG1rAxzT0kCUioPGJqKxM\\\\\\\/lPIpF0z4ewPg\\\\\\\/rbBKSX2IMXzWY81H2a4lI\\\\\\\/9VX6GaMC2lmPWBLvmr4u7SBvKBD4TyxvzXDAHV8O9uc1iaqdipZmpuZdcLbqy8J9Wvm5e91FdFrv9N3PTygU0yiil\\\\\\\/Qds2edYk51Ba14Au5XtSZd5BiI8Q6nfuZKGVWJygRGmzmRWzKS\\\\\\\/dudxKqnmF6mxEKBG9B7+rFqfAdOGOtuutU5dNVVvUnAZyXHLhrFpzdIh8R7BehcD57LUt4hU8F1z2gNSQT2GQxcdmOxnfGGRi7O0JChPqVV41euFpWIX8lMwh+D1EU949mq+K+dgilcTWEu7wikfJOaJXS\\\\\\\/uhNyKfAYVL2p7AUSuN4K8REcioMLgPes2hnXg8McQNSYs1ofqMpjZCe5\\\\\\\/LgXdHncpME5s7irF2IzFs0a\\\\\\\/wSsDZv0+3ss\\\\\\\/SBz\\\\\\\/Dsm+B3NAMFnk5u8VqjUmACrH1FhkTc0DBOiN6ig3m6K8oO91WBMqf3r49E\\\\\\\/hAMi56rVCEUZ+sA4eARZTR+oCByAuM9hY4zKMQfhFtWekQ8XHRp3rPKlN84N\\\\\\\/auFWUdjjZTxm7woX+4uF44SQdrawrgcgclAsHuwcyVrjDS25xVFWQJ1D9agk\\\\\\\/V+5Q3qk21SsRjMe0nZ8m3W6rrSmIruxmYIXOJY0V56X4Y1lCXrqPhc1VNTDSmiAvL89oVBdB+6IihrWv7WE5QckCXihEM9hHP\\\\\\\/ap\\\\\\\/q8uBzTCO8c73h2bBxHfX3C7a8qzEBNTQgsjnsoGMmzLtBPLnTbyyB3lfox7qKRLoe0L46K+LufS8P3ZptiMAeNCJVDq52NXk9fauqoHS9zi5mzU1ctBumeiZ4WZpbBIwnP6jyRQvXV77D6bWPrTl3pM2nhz9rY6yRYyR0UXh7XK5s7YLoaHcl2I\\\\\\\/h6i9jtUB7TlEiRcwr1frSyGJsvfScqPmuMXsx1V9c0Dk4B\\\\\\\/eqH87XH4d+zY6\\\\\\\/hQPl3DSQUPMjkdtqDrSt7j4LRuATIxpVb+z306iu0L2w\\\\\\\/A1i9n\\\\\\\/ddjEzdGdfEXx\\\\\\\/vzZgD\\\\\\\/FFUVgyklWgEdvtuPkZ7BHP2WccpH8Z\\\\\\\/Tg\\\\\\\/GseTY32szHjcblPqrowH\\\\\\\/dFKGoWudLcFS\\\\\\\/3gFANXHS9fTNhBSFEWazOjMVXPBvJwSwTcoCACm9wMOJZwEk30EvmGt+F5fQ6tCstNGuchH1BIUa17\\\\\\\/Tz1CNr2ljLW69q11SXdTgdlY\\\\\\\/sbDWUTGRg4TJCb\\\\\\\/76Lr75mdsyp61ZZ0Is+upgpG7Pl6w8US0RfF\\\\\\\/ZF9ROGNwUE7RdZVdlcZcf7mG142aHNhK\\\\\\\/cS3kv4zRKNJpTXqToDzzxESucdWZ9jQ7HHZ4ux6psvvNj05V36r4+ZfbqZWbJD8dAfmiGnLHBU2Ks9wglkKdTkL4AAI04SgnESqwxmgcy05\\\\\\\/p8lKSpS\\\\\\\/DIwZ\\\\\\\/uwC5BmpZxWyHHEucsC6J4F4tIjk85r3ksyyTB8CwbonVDxC5u18eNCJcqEP62fXjmZ4PPfC0IxAyTDRhk3lc22gBdpUVLZQiZldmyqiZF0Gwy4PT5I7Eq0trQaYo6QXAbeY64oQqKNQyCYKrWIqlVv8zcxP7F70rcWtGxIJFRdPN72HR2J7FBrdFJwppkiduygagSMD9pYqqNcYnf6aRfN2U0WJ5O1su4qi1T\\\\\\\/dl1JIV2zcUGWoDyN5Yv5D33FdmyzXyjmjKDFOlUG9uVGPWVbrbZwN+WNFPzwQMwYBSO4bsvh08U8QXnbvznHNIO96rx5ua0lrztEGt8Z41eqST7yvN9uwQDUtiUq2WKJ14RzFdpe5I89KmF\\\\\\\/K5q6eO0wmG5r\\\\\\\/eln4sz9fIfley1QknNBsncnhNBB7kK0Avb1ndYsxCkH\\\\\\\/gXPqhM8LC9\\\\\\\/9pPGS+mqdx1plUOQvRlkBqQMd4zWKot6mnZReO0ctVTyA3JXDkk6EP+D439ww8i7POY7kE8gXdQzpYk4SWWvnCxJ8a2hbxiZq1bPJ6E9hgjFoVbiyfa6dZsJAj1aH2VfY4iWxrvzn6L3PijNKGJ0L9phGpJXNSHo0fyMDD+I3O7xbyOGF+2iOkV45FE4xOAC0Ek+H3EckFqae7jTxSlaOhG0ic5MZdxuFbXhtVWy9j1f\\\\\\\/8ybSn2uEH6YlDWYx8UdL\\\\\\\/olo9BNh3Ya0Z2rvNg4AsJCFHbt6UbKfYWodNQ1VYO14BqdrMSW0ddrK\\\\\\\/QvYnij\\\\\\\/uuEyX0uwyFuJmCSozyMPIFAdDB4sZqiaCFbxllcoo+2bS8V3IYxcoe713szgabNmJoIMgiEMB2VifhUxQQkPPZoAemux9+1lYCHrjo9+zXN3VMrnaqPpxA0oqOKPpQuyzBUdSjbJtDYcENwnKEaFXzSOpV9NgAdSoYBMo5VqmgdQUObwZowTLUiyb7VxiOJ7Fv2n\\\\\\\/JgI77l6tZifUeXdrStYKpFUN6FyJFD+ItmWdbNO6WxjXZuzRWVnFcVvCWUUHgY4HDwlf2w6WJXMOvVPDN2QSN5b56L6V379UGcY5znaCuMbn1p7ay\\\\\\\/2xtd3xcAjcVIDHsIbLUZ2dtt\\\\\\\/YQykMqHbo6x1Y7X9ZxAjjtYANGCCCCSxGyRBY56JD61jhthw2RlkyvYVX6fjQER6asqDTcRPnR2EVTtofzLuz93mzug3jNgQs9nHXoL13ZkOyR0wtPTGTMB7QzIDT0WJxT187z2w9lId1\\\\\\\/uj7QK8lk+fw06DGtRztv9oDU90dNBVjFj6e2xiGMllrZ\\\\\\\/Mbdc3\\\\\\\/9wC\\\\\\\/\\\\\\\/XAHhUOLbs0xM519nlsq5uBGmVwozfLI5van6M7AVqLZCc1iDqG9B2oagkcFrTZFdyHba0QC0oQ3tSpy8BLhCxRshkcFdCIDOXNwDYKldtZyEWwKJHEL9pJa59GgRKfuZu3nLKcwdFnv1BYhmocAnXwyYrR+eSl72Kn5l1+tKRToDyVPD7SvUhSJB5jMvBaB0hZK94N9wy1o7mowWO+NUlAOLjk0nV0owh0zFLwA7xe6qCMu7PUkJtAxDBiVodT1FFMGjFNhCW7goLzt9K6431fMMAcKgh\\\\\\\/LW6qKt1zIREhiRViIt4NoDu42aApE7xH\\\\\\\/QjG59LKqw4o7IRTvvlgCT+3i0louT5TCSEOGQwCw7kXmogqbRtJYjmxJSTV5x0VgsBecijWhxodOhKU+Sy9yDS94BshMSF+SZsLtpCjeBYazekHoWcmPn6FdNoWGGN5\\\\\\\/yNHDASvRTZm8RaABd9NQIbWFzD4rs6\\\\\\\/EV7y5EUz7fk+8R77tZXoomgZqDI11ozPGPo81ewn+TvwAJYY\\\\\\\/jbsXPr7LUGqJg\\\\\\\/mj66UJlVV+mhDjpBzaLpqa4VjfDWhtqtqSIxJlcvI4u1mcLdh+fmSAHG86meHCjjXPQRUHrIJullwS96ILwLrzBYYzLUu7oPZcFhnNoJ8bIq+cjev8XK30nvsZsCMyWBs5DcMACqtSaToRfzeXvlVjsqPJadkYTHJf7MiemXmqG+bbT1LR1OlwKo6HljagK+quOqaeQ12a7fUEy+br+AIdoKAfyrqLkewuerw87mh9DjEOTEb17AZr0yn6eqOr6GN4f8Q4U0OX3os1k4x5zXLMvWZaKOHAzCbrzAY9XW14RLFeivRvkfxfo42uHKrlf\\\\\\\/mMDXWoP8Q4kVqcypsTKof9XW\\\\\\\/dztTWa3w1Qj8m49wOBkhUoqSvXKT3vvrxvsPosEQHq3ULqTZhCiAlLE3u19AoFBEmVArNOIuwzLcT0KepaHwvzBi3geu3FkJgYhu0ikDkdBHHVaAQJg+wGRz6wcjKjLcU5dbjaHBLwKpOttczvV\\\\\\\/MmIf4fiEKXwYP5C14ClESZ9lODeRvs6zW3WdJ+e3KEo8Wq0JFBhRgwrubaS8Ss1tf14Xe3Y6gGmynXTOKBWw4ofLN3XARbYRE7kkWU2jmLgNcvsQaYauCnV9mm\\\\\\\/gDO4z4uv1Wf2dggEHbgzSVC8kg+nnzzi\\\\\\\/xIRnd9z0ung49yrqZD8mt4HXMDll8+XeI5DzMXAP3qz0t1Dfj6x4VS9xqnvHCvBDgdhXRjmDSQMzvqkuGP1CsQZmEGWjp2Y+8iBnSW\\\\\\\/BjT4lpOuh4u04G6Su9qUU01uvHzCd0NcfbRuABQZGpcPzsZ4Fb7XhtbQUOi0rgRgWgJgtwbCTKoB1B5DNqBI7q\\\\\\\/shZ+qilc1XiFf\\\\\\\/vqgjn3qjgRaDoguGmyemmFaOMBDWKhavpn+ORnumVRPKUbP2B15y6bfP9xI6ji650+sl9AWTu\\\\\\\/f1XmNBvdfPpzFCwbCNtChtRUBf8XeG7+0JMEUV1YVXGUzcERf9q95wa0VfISABhRKVWSGwD2jmcZ0B+HzoLlmCz4m+KUuae9FSfDnDM9T2\\\\\\\/OJ1hNH3cFlJQSDPB851Si3lpjeb3yGJ9JxDXGTccfzfrf\\\\\\\/SWYkeQL9qVfsUKquMKddhRBPy3CbFPesoPp6NqN7nwP9QSUF3s8r2X64XYK45mOf6ckcsEcme6KLlFvqUzy4G+N8tiR+1+vzQVSo3OCgGPrR++jpBd4FV+uHG5hG\\\\\\\/i8PtVhK0cduSlQUyTwzEGk816TreeQJi\\\\\\\/zR1wniReK8Mmzp2uVuQBhp23Uakhp6MIXp+HziLE3exdXdoBnY\\\\\\\/LzqPt5jMLwryMwlH9XgDr+kWdbRRgTpYsPJf9a1S3VFvisWRnZKf9I7QpzH4DaYaWwN9UNZ3x1OIzj1axTu\\\\\\\/fCcWeliSOKDSALSuF\\\\\\\/J+i74yRLD82LLi1Y2ORKi1oYcIgsDyZ6HF7IXJrDfXY9E5zQLJmG9g0uOefY1WmFU8NWs5spp9czvrrgLTAs4MqG6PFHatxjz53+4U9sQPBtEMZ6Nf0eWi\\\\\\\/Syz511nHdl2+kRljUsI6oWffGE7eS+XyFDgn9ORJm9Q4tHRmIR3UdDuZZkHCXOyBvEhzDyqzGZ9JS8tgTHHOqrjHf3mPMHAq9SVPiPBSRNKD9ax5TttGoVPB0qhVO4lQJmdxQr2PN3tzdLboboQHAMN78abqHjukxTwzZXmH6d0E\\\\\\\/VWp831lH+h2UzBVMSa08LMafUjkmGQ1pGhuQ5wFzZKGKw1fZkfrMk\\\\\\\/\\\\\\\/JYVtpoAfIVo1Q9iQ5aoQFuiCdwpW844rv2tJvFFTrHHQ9ZkbRf7ZvaCrkJqHN3jMK5OoUbmgM+G23hrAlYIAMk2JFjygRbg9r2SdgdbFkUoYNb\\\\\\\/SNFqdYpWBfyyJQxagqaF8DTjQj8jHxu95dzGk7niItOWkeS6GJ6R3K2TfyaaIUtNqSu8IFcHcvifQWTmCTILMRIRN\\\\\\\/X+EZkaYV36TIT1BgLJAPhQfCM6jZCq8ZlgvE3ZRL7rJMj9pMtcHbvPHe+b74qfW2Y1oobTY8ibPRHXosxRAYHkgHxZBDUaijhIiO334V0iLGAUAblvEFNxAg903A56BaFcBgG9OzjyFHW7BtleK5ZWrEdD3GT+RiYZCKyoXUXjLJfw+jzPHpkGm2QzGS6l61jv2kWYn+NtxVHbyh8WsUeZY1IlGsv2rG38RLmTPBiupR++h2AnAc3DGjX9BDNorYCXPkad1y+rdh\\\\\\\/qY0Kl2pRcYvoIHeUUfajJplUyWkiyzUZ04mz+MsEA9+7MIUMFL4xBIzAcTdsfqgC1ArZ8OY+frf+KeJrAC+i\\\\\\\/bDIrCnWNvJJsbjhwp\\\\\\\/rBpW\\\\\\\/USumc75FCZmb2XpoKHdUgTpJcAESH+tE7d5hoiHKOBnuu8WhrN0EH7tvukNxlHSdxsNepKRTK2Bcq0bsHqv5DnRm6YTq8p85Z1TRR25jfyhceSzMUlUovJk7YWTzarsFb5uNHfHv2lz\\\\\\\/y92tz0500f7HY8e3HSfKPYVykw55\\\\\\\/K0RoAoFvgRctTwlSl7WgZS\\\\\\\/2\\\\\\\/p0qHy6GgqyzRm6Bd3RdtbXIcddkgR0SBObH3IucsfqqKWARo4szDfz\\\\\\\/r+0iWza5+e\\\\\\\/WpRrXgwHV4U1hc015X9Q9d05R0dawrHGVG2G8x6UIdd2ZU9HCoUg5rdTDV\\\\\\\/3ZyvwP1q+f5HfuEV78jWJJld9IpExKhtNOOszhy3lRDQGxa+t1E0tWu3GCOcCskZ6HOIf+sMWDMVxAANAalYpxbHArj+RFIEk+GcnUao91uzp0NZFCK2JO98BsjqEZtlJ5YTZ2eWCr9STS8lOJRwDrpnK4hZmjaFr6u4GJPJKx4rgg2RQTpvLHSqvd0yOY4drM6bpBCchtAN1SHU9k\\\\\\\/N5CeDP1wuWji2U6yDJ2ryCTO8vj\\\\\\\/RWn8hGovOMeRSxfGecA7oFrSXkfBfoK2UVMjeWdl9EU3xS+a4AGKZmliaZ3EG9+q4Lfb9QCKa+aUToS8RExiRJ\\\\\\\/m2QG+78p9tidhj9OrlTkPWz0yw721iMYLJAEzEy2DoXuXoiTqRdrh0WtWOFo4BOpoYsxZcefx3Wp+xPxYZVbF65B\\\\\\\/ujv8TXYzDKrh+Gj3eih+ydrlIfK9ELdqGpGLFP0d3TSWPAdDWok59gkv2BgqpW3tnonEuGvkoSqGJ6esnCpvkWbCzd8NuXdcZQ+RbdBMPhrCFOdzm6EWIGRuTzxd6v2gIGZ8N0QXLfKP3NOWXr55SiwvNriv+i6RPOZUAGIMUABi2hk8IyjFSj\\\\\\\/oPTDgzQK2zZbO0Uhmix+mxzrYOXiGPJi8wwbdi+qtqTUOzkv63XNZhUKnTZxMGkeNNeXiPj6HIGabJUMbeRu6wLlquz4shDSlfWGTarJmD43D9UKi4DmHxfCG03uwtFIvjOj2Buby9vuR55IsxxWUjUZhpCRjPqi3nJ7EKwadFqePjNFMHbvJudfRpoCEDh7dN75\\\\\\\/8IZ7CpVODAS8xiwutRgFJ2fuXYuxNugDUuBGlPL0bAjFmmGVhUPUlr\\\\\\\/r8gHaRJkjpJYlhmJ\\\\\\\/kHBybvHTH+Ia6+TMG\\\\\\\/O19AcuW3rtvwNSqES2wQidm3JnOzwcB2KI+bXnPbIacZAAhWTu+Js8ZkgQOkv8aa9jSPx0ZaXVBRG+XxDfSIfQWCo8s9r9dXgYwWZXqlxR71l+qQf6SIuhSrcVXpBMpofqRjsOs6gnhYfLvtUgEHaVEdGFXsnvpLnFWY3pmKZVk2kpkcADt5M+YuINuQksOlArQ2839TYsBYit6dvlwh4ThOKjH8OPzBJFFeh+NZa7FC83D0gLLK+ZqQ4pIEaNLNDqporjWqpI8xkFu068901uXJqT50kBlMr\\\\\\\/e\\\\\\\/WDVfjxocYX3s\\\\\\\/3aQEDCFFcajsJaSWFI5zRJntfqDa9cVNqW09RWoGf4M5g+yHiJVaOxarD8TXLmoRI0ch9FF6xOqlt3oZXzpumS++QgdaktwpxkknZJ+kVPiowZv+IWYUFTFOFhFqzPYgjluuobOH3nYrumkFJg+JLKKgh9dOe2G0pfgIJAB+IDQR\\\\\\\/HTa70olfnCL02f91NCJGK9lAHPesf4Xhgp5+ZpkXZENAzOav8IfTQ8evjxae1fox4hfDGDu8XUo3pDiYuortEXeuqM4O1AXGOiglZ4wW\\\\\\\/XunlemUzJm8U5u32RKJ95svbe12yWOPesTM1AvBrxNZIjOrZiLZI3OuANLGMP0RARu8DmJI1+YtNj9fYNKrCWuTtkH30j0gcR\\\\\\\/axqb3\\\\\\\/QhWJv7DWtdlPJNvqi9Slxw8n5HulslmJwt4ekur9UUxxWBOJ+SWFifKsZrhca6fyY3+BKNF6TvCiSvsLwZSLpDf6bW1yAoXCv18FAH2xaI8xuvG1GwaZYsidPNoCAvBSJHW5LRqItCESQIF\\\\\\\/VJmq5fK7Gd9UK\\\\\\\/1TYZs4EHsvXg3\\\\\\\/vGHJRfqELeVbARFX3ZE2V6fFMQNguAXoM8djVYQ4ndkNCEpC5Mr2Xyh+lWZSdPu+ZVXFmarqPQX8QOF2mFn3PEAhkbaC5xjSzKXfV+2KqEW3JAQfPruP77mEntmwFORoE\\\\\\\/ff5gotO3mzlso\\\\\\\/oq9FaL\\\\\\\/+TU1gSkcjStyNVX7NiFr9461SDAIFw6bjc0TQ8uCFAmphT39pCWEiLuMIfUHEyh\\\\\\\/dljyC59OyaSGq\\\\\\\/m3gf28ykdXU8giSOW5CIoF4wahr5njPUCzBuIYodhUhgTv9chuAz9NY8qSy6kXNxrbLhiu61hN4KpsW4jOoIJZle18LTVGM\\\\\\\/0+Os4c0MebxyH1H2Txwv9yjnIbEPuvPI8t1n2RE2fxzBuCLdQFtcNUmFpCC7Wm6F4zUqaErHCKsBAuFqj3FWTGx71EyAdafiz7\\\\\\\/qxT38hlvgRKF31bDo6i0ANuCVPGhYFaoxFILYjTrwjHxfPfRdXm5tY1Digh+tDNI7Vw02meD8EjbivgKT+tkQFylk+Y2XTaPkzU1KDHC2GlA\\\\\\\/UsV1F8H\\\\\\\/cuUQA+4W5lhFZSthnVObfT58GQe+NTDvbtqcHQ4hz\\\\\\\/hWVojXdqjSRlZHTtDjcUdwpugmQ6ZMPR4ENAjMajNQm\\\\\\\/4z2ZicBbQ2QXcyDK3xcbvGK8BjHJ8nOeccQ3t6ttQtKk2hukw6h4RfffybTTdVq8hr15Pj15NI\\\\\\\/wl4NNfhajk2PMyukBMN2DYzIJ3Y5Qv88QibjDgYQZVorEgmqXYG9n1yfv6e1ArVRnShmrh2OJgQB2mbRem69wClvBKYMeKlsOSxLa23\\\\\\\/i5q2QQVrwVAobJoI1y\\\\\\\/S8QpYdgT4qmr7YsTIk0J3BjJP\\\\\\\/HXNxlYqGN2L77sfYl91dO19sa66Zew\\\\\\\/VB4zdnNHkfJELmGDU2ZOiSv87ZNUm7twtosRbzV9+5KJEhk4\\\\\\\/2sH44GdOqKkNTsrif5JAilaCS7d04kl4P\\\\\\\/OWJIDqy5R+DOJAC+GmFjDnDC0KLjVmq5qwgWzVTYSyXIqb0APGKQbhaNTNVBbjaW+EmDU6wyORs64O33TkB+MxW2CAuZlESIXof3ypUz3zFqdmU0l6vxXlynxZT\\\\\\\/M5Oo4THXpoD6qilURSNnq2VAZaa1SVyh23gAnpEqsIN6ntb5KcMNsvyIUlLoQ8BfJEXMo6Kxq9Axg4KhD0aVV18W5BwAH2Bf\\\\\\\/WEw5HawUN6x+sZrdp2gMsRQl16PstfyBpz1wh0iQ72PFEH7awW6mCngG7J2OgO5nCKCR7Qeib1jjdXGxcwl6shZDe6xmbCjA9Lx+BilYthCknd0AIDCNg6iE46Qd1CWYswY\\\\\\\/YWe8M1kJIS5W\\\\\\\/W4218Bc8QtejkfvNVtIQ+Bqo5x4v5++iZtA93zABx+H95hWjGsb\\\\\\\/mzAxfbxMI2Ka7jwtxaxpYCxWshjlBaE05PWn\\\\\\\/3a6ppuZMVRy\\\\\\\/SKt5VrXC5nAaXcUFGU6mEl5oyhaDPA0XAsM+nLf2cSVwXwY3SsgLg+nnNB0ttdD3epjpEn7zkcwpTR\\\\\\\/zSqO5pbKeJoDSThl20SejHfBBDTbF2BPiCMgLTp0errx5ig6CQIoPy4hU\\\\\\\/jRJcvuOOjtb4fq+DLO+uRfsW7pQvU7LjKjUdaczUNiS6YwbmLwCjooV5RQ5KNIrpDZuRotdV0qxFL6gcPO1Q4J+xxw4zkC\\\\\\\/Gdaga2wwvq1FSB7ZjWshog5OzjG62bcvUkfnUSQJEBq3dxmuHdupy0Bhc1MMqu1vjig6MsSj5jQhWKx+iHoFJ3RldDBM8ZGRSTBSwxFBDA5Nh9o51NfUaVvUVgw96xWqGHMHLbR3hllFAc+tkybPbhHCnmXqRa3b7UtPnuChZl4\\\\\\\/122K\\\\\\\/jzScOd8loCvMeA72zaiP4yqIZL1Fr8mzC5MJfmVAYfGf5PwELXF6qs8dP2CvXtcIbqlLBUT3F6Xmo+XoqvW\\\\\\\/Qv62lDjdMBT6sr8l49BI3kzkpVZaTinNOe5f3oNGTOwGG5dciGBw8iJH9YOJXkxKYhOF6mQrilaTzD4M25MyKuDUWmRV2b\\\\\\\/UYdk9BACxcBC\\\\\\\/IKgOlwiJ\\\\\\\/oJBHTBbF2+LplgnJdpB+07GC+CAMVUhogETQuxdF\\\\\\\/BM16Vbhdcobkq386clRI6a\\\\\\\/roWjbkOAQDDusIDT0Ot7ncxMxB5rNBNavhEhLh3liO77GKODoY8r6HIcL1pADUsXldSaSEl6Lt\\\\\\\/tn64e2fszEx30IwJ9DmDSG40NeSnno34dpGoNblcyoUh5HIkU9ZvhNjNsqWkzhegbUyLhejk8DVvlWTIpL79CgS2UHXBzOJnlnrPxUgggGEDfoAbQpk239ABTtwRc\\\\\\\/w6SILcC4QNWwNYkzjCtdcFnwwzgxnqarOIDZcUx+j\\\\\\\/BRcsDWKrFEv7OSaMMknUAPZhxQczvDrl6qgBdlU31dgExDqov+6ctq7dremzwPX6xk4qTAaAm7nYvWM1wvd8ciUomCJOhapP67deYZVGJDGsu0wPZMHMwRBFUdyxNawrXrw6oMJFZ41coj1TYHPpU8c6Sn1L+wLgSJKImrDGzk24tJ\\\\\\\/JpG8jbq0UPs+emz3IfJLliKjNp1GD8t5SMuMGmvWoRDqUzJET3eJGa\\\\\\\/DFGo5fXBAqZlYimqh+EMt6eDNqBq+EVeLSMFmVbwJ1Pta2IPucnMG689fdg1ukqo0BOYICo9855bxM7R9iAF+V0niiYlFiIBDjMIvnH7dFX8UHfeHfOEufncBgKOiDyVLgdoMlp8RFHYylq9361aCKU0a78jO3y3wfAM90SUm2AURADU5FVi1XnmglWnMoXFD8wmHdAynNXYnwQcwNF\\\\\\\/Y8Dr0yFwRDghUqVye7q8c7CoijgawkGWGz4wdUa4jcdrOiNXoi6mmuDvHkqY5cZ65\\\\\\\/ZVLK6FS+tnj1orw14bEZu3cRTf0mAXFebGJXK6fTxdIlbhcYZE71OSVqyzRc5YIMEsCLUdXF0WqbMTxsTQPlhyI3nAX1R2ZnxTZVIKv+yTVXiZHGwyKgWl\\\\\\\/Dm\\\\\\\/G9PsF9zJAMlBemXBVmXAqzzbLMDYG3rOsqun75Q1mcdMb4bcGTz7PM+8BBLhxl6QqxwHHjS3NM4q3+f2SFCP1UXk2T9AIUuFGmP9F3phgezLVQx0unL+VB56e+4G5uAwBMxCnBOd5vXs49j1j1Nyc27fJ9xmZlnA8U+VZ+ueOKsnB\\\\\\\/jMvjIPxLyCgKesQ6jigVmYW4fE\\\\\\\/OmOlh0heLT+MIbpgMiEaHppCzvwRDChj34EY82M0CXgpCU3I\\\\\\\/3u6gU1oglD64O0og7RQILWGWgOdGSk\\\\\\\/wYP0pdofok8iCck0\\\\\\\/M08OXopOp92OskTI+Mi9RvV57Qq2zYt13IPtqe7vKEIBziBS8W42PFTUrZLSRwovYwc8Is5LYJavU8XgDgR8OksPfMi96lNx33fIY9XtpngynY8WyYadAMfNwOgUweaad9prO1Kw3YBGlHrmWiISNBWvT9r+oGYFw1gcXvAa13TgjbKbwHQm2uVvPtlCsNjhTDNjeRr8xRyjyYVS457cZwS8PZwpb19FnWNH3Bfss8IKAP1PQ0xGndJqvVk6KK9ssA55zN7P7Afy\\\\\\\/48qcq7xj6vwH61T+p9xGlpU0R+LL4ubJILRWIz5mdzemPZ02o9fKDC08yTtqZILfwEBIoyzEIMceqlfh9X1PJ\\\\\\\/XRr\\\\\\\/QCUVQ7A3Kb19bxZb1iTKRT49CrnFAeedHPP1lCDaUnZQIvaGTueEbNm0kJR3hnFQzYShBpwwWcqPV9qHi2h0VtiQDgpobQ9OkKCwrUrMtuZ45a2pzJ2xmYbmG0fWiWgV3ifgXtJO7ZOUrZ3YPLNBQEyWKcf3QmxTBLvwIj5ZC90a1dvTFACVBYVIlqvhltFPPigTlX7ycWKwVz1ZpHhQVy8CfsuUin2dh4TO9BoTlVJxfkF2G\\\\\\\/NRa23d1jK4OeF\\\\\\\/EksBFyArKwsNIAihA99t5qyCpHsFBHWayEp5EGFgRZrg3b6UYJ99ZaVBsB20MW+oQ3CCErMdakMeXJ9aa3XKVTcV\\\\\\\/+ndpkCqbuBsQ37CkGPjltQz3AG5qj6MBqOz330P\\\\\\\/HA9xuoNS1RWgqFUyLB13oXd4sq6EqfVTqDGB2TtThOSS2zYYmj10kaNtQv1\\\\\\\/\\\\\\\/BwnZZC9UnaW75+j9JKvhusSszaxIg2wcRHSSs1UFJj8qj1Bb0FPMmemjC5xsGuyPTc4ZjQthEJgaXvKhJ8ncLefQrhqc9irYNQHCRP2OiZKuOrN9vK6Y+Z4HHb0u7oSnDmuxKNlIY6vb7ZukEYuFcCL7nVLSqDK6SIXJ04AjWTSbtldkF0B0k0AdHweTYIBJBbvPQl6ikvNn4ED2oULZxU8dpRVo204H23F6lFxjzD6FmWiGbLzMDLwB6YFm3FkOekh0xqI8M+Y1XuPccv\\\\\\\/oXCG9Ak05HBHthABohN4D7nQRrpy1Vpzc8RjVzeLDDf9l59oafKgDPI1AMOriHvRmTA35\\\\\\\/Y4xeIqgw9sIjdqY0V3Tb7MPKnkGzSVPIXnw3mgGi1xX6qfWLEgLR8OtM02\\\\\\\/wMa\\\\\\\/Ql0xphr3yu6PngNZWyU2hPfaWVE2nIuh5a+WCb5YdYWqrmrO+wJBYCdp99GVSofgZa6D7rrk6L5mjIgqCzS9VXG7DPlR0\\\\\\\/9lMEd5qkcE7uxguASKU6VEXW3TnDGRLs3bHUsDrGRxjylh+t6q3Tov9PdPfxeOxieukjQBtzM+HSCIwM0aCBaoQgNuJwuEgkQRfn9an\\\\\\\/SbbYvT5AMUA18uEoTVX+f+pTros1LHk2Velwdq8fntfcxAVRBIEnTu1NOe6lfJJptvQP2IIaT66Z0vvvYksTb\\\\\\\/cgTwp3mZVTDypU+Owf+3b+V7jwtiqVwdjk9lFrvp7JKBDLb9qilY70NrteVMxG0Hb33Dnn4deyOFU1ofpMAx2hqDMBRd1ODt1pR8lS5MNAtUZJgf61QmNh3pQDm7Fu6YsWjRQDsAQ6CH0Y8U\\\\\\\/bgLsMueRTVuyB7Rt5rFLVTW3\\\\\\\/0VVOz8cYMuTbmdFXyFE8dmiFGhPgQMeRezeFWhFP89ePB8\\\\\\\/pO9lEtGIhFkiyDl8G3W6eN1KmA+B1omB7DEX89C5jFHQGxrBFfFVOW4e01jmfga7Ib9SAeh9t7LC66LobzC9Gm6gQvGmQvGbjnWahUT+SG7DxR4TWbM3V5bAh46IxB6oa2BYhj6bIWmQpHqxdYEBeMV0q3D3uxIXABVPcv4D5KvOfF5W0\\\\\\\/iQpCm1zb80qQrm0tEEX4c009uAj+i8tDVrISdbOUvqc\\\\\\\/lFt6mOXCC90iJ+Quci2UFu\\\\\\\/xrHRmFFY1tJ8dxC0mMlgRfO0V0f3g9W85gLm9OsLE2tct\\\\\\\/pKtPLbCSPQuOokidCuY\\\\\\\/7UgrypxVMzG5vrspjUZQBGY67965TWj7RApX4B3tEHsD3ow3m5\\\\\\\/sG7ZT6SsAx74rxqTKxo7DkCFNqlPZyUX0Tf5Tb+1y5QdCXCH6\\\\\\\/1IT5Yn16Z9eTLwcYCrLtc3RYSZnZ1NFA0O\\\\\\\/Qsiv8EgXMLHk4n\\\\\\\/XJ4NYrBCP0HhFrETqahme0Y18V4lXuLNvqC47i8aUuWPDClsZT+uS20JcxDsWFcAm7PeRVTdv4a7JNhZyFQEUYG5djZPmxrJHvgpHzwN8P3JLEYZ+6Ntidf6cr\\\\\\\/FNeybdcX8fKz5+c9vsWYmFdU9OEa306Meiip6izA0VjH0zvLaFqGphjglXrIHHTBoiPoDcivxqgRCkmiAIqAd61eTWyLdH\\\\\\\/LVUf2nwAlDgcMS+9uXv0ElQTbZROfVijcoit97PHzb4F9AqvtHGtsU2OuUYIr08Z\\\\\\\/wikfEk9vx2zTSnNgNnKGI7mAEv5KcDrylqYrGM3479uPVIyDfFKyq9nxhI4rQirxPSVew2ExXC0p+MdQMx0vGg1aY\\\\\\\/BE6OfrPjrhesUTbMSmPOQigYMZXq\\\\\\\/W5PH7M0yhh13OOx+xPxA0K0gnLXfUn8ISWebaLkVY8MgTA1mHdLjTPDDVt5HTXdksqx\\\\\\\/bQF\\\\\\\/T0ycnXvIfPq3jXrLKockto3NZ8pYJCqMR+DrmsyKpuv1IN8PqpcB9z18swV\\\\\\\/KJG+V5M\\\\\\\/Gn4MjzHjGd8mrX8li8VlIJDNhK5Uafe36LbTnjrUQj4wrzB0AvF7E8PQQrv3IskzlLUwhLOgn8HCszk2f5rR2\\\\\\\/3B13e0omC13g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jR6kpEfYEQhBgVc6Zxf4gw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b6e35c5df1f6b175efcba185ce08232678f6c859a335343c9bf49c92e49d81c9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222Y7arV5RdC44FlJUxgJL3cLd1Ini\\\\\\\/DCj1pv6X9I\\\\\\\/cj\\\\\\\/fa0IqUJ7nx8ScMSBV+W7CUaueNTyrj1nkxrkwhTiZ+cYh91wlXv5IhXl5g\\\\\\\/ldyqMdTPR066V2HsgL0s6BL7L5jzcdkU+diCymNw4cbd0y4T9ALK8QxPVjFUrD8H4E2+PSmsWjVIC0HkGDA6lq\\\\\\\/WQB8D81LMRO+lCEDjfvm8N93HkMKkqk2+2TN1qW9XCECDzLyW0C1Bq+bETrbN\\\\\\\/m+TtXD9psVdcg\\\\\\\/hYOSs7Gt1Ad9rCmcOD8Uv8vRmvPXrIuDIVpB2IE\\\\\\\/n60BDHN3x88o2qiA45rHqsj1R1q0xcyONt5tCV8JzYHa2lwXg2q+i+X+DDwLGwRaPgZ2gYdKIFqq0a0kJbx8u6dQGkvxaN7EiyT1d6MWW+eqIxwFCtX3hwcZtj9TTMSfePJ31uh6SC\\\\\\\/jbWPG35Ebr8+W+QBUHY5vZ88kXXqDjUGknbqDAYSYyYeI4GCbGZ\\\\\\\/iexrgSTd87TxrOO9Nltgm966HEkllq8N33fiCuwXZU+b7RaYzELx1FsBcZ8CbpkpRSZp3epyhvkYmBkHFiXYsvX+HSCPkL1KPk5ItppQ2Y5+eRnIhuXz+PFZ3uafcj7wCQL5gB+uyxrCmdbKknfNotirAscfOO\\\\\\\/UMx2SADt1diQzKw4gnDo\\\\\\\/opxSQ+rDFitK5FEj9ro3pMKdKgVqLNF\\\\\\\/f08iE2xVCU+rhUHg2rUr+suDFWN\\\\\\\/7WD\\\\\\\/N4x8zbkYlel83hLZM67MV5uJiMLGD3kp2ZgY29+t9Js4RL0+pYZztEReC13LTB\\\\\\\/rbqiA6GJRZbmHBmH\\\\\\\/CCxaukBzIDkasGHQlyRzjYO6VpZdChW6f+N3+Dn5BhTuGfQL6aH\\\\\\\/Gqh4aLDatnfMhB6BX5qNtvtbrvd6igfHr+ZlAf0KgOeqqTgoR9CWwkRsL6Z37sjmEXJqEeGHMzMqPku75lUnlTfr3C+NLpI3MzgJmJKbsMl4abvSunTLLVGVKZV6tsvltMWS3+\\\\\\\/VMn5l5GOPSTDudK\\\\\\\/Faj5BH5Mmnz69Bj4VQPxhKbV\\\\\\\/ogrG2SLsax6aJnj0VBK4ykAe7HHayFiYgrrJVxWa8eCppyy2WZA0Dcof2qCUsfewbvoKhPy6tp0eGMmD\\\\\\\/jYzVLVJCYE7L3AWFAr0ZNpPQ8iStJxoYOsfQfssj5UKZx+7UB3q73MzHi7T+9FEI0SdEPXCYEVAuSxHmRnBCZkbQAYnUitEBGS6\\\\\\\/RjtFy6GcnZuY93\\\\\\\/SmmNk69BnOoxk24dauAdvoN\\\\\\\/\\\\\\\/s1qqUhElTKJ5qrlZYa8LbOM9bNP0NyNBo9bWCwvFH2Ya0HSioX3smhR4YGmB9hwHsYt1whDMEJlPlHpraDaCltdrePtbOEWOAD1IG8kNRWQT406nI\\\\\\\/ark2MHaTDvG7SglQWHX6SE4j5ackUmr1nF5b2foauQTgFgdv\\\\\\\/BnXv7wNHXtkX0OqxsKb9ekmMbA+xXDwTuWR66ogplgRpaTZwbj6rNICiH4Lj+ccEhEegvhAnTMx6xZxLWPO4reRwTWIGtlwA1ijcLAVSeCyzVEgvD0Gdd37G9hFE0t3XQl\\\\\\\/6tw4PysfwIsxBVQ1UEOyQIncPn++xeMi1aHWlbycVBfDTcVaKvyvu0MTfgGVtr3vvtfwn8FIy\\\\\\\/RqlQi3AWJBVpKTk5imueO0h8fkHqB7F14mhyxjP1+1fZnuvhzsd1gbHErkrCgvxpVwhYNqss\\\\\\\/dR6XJ1haLFBNoKUDajvqlqOS6Zs4LoHiG4LtIqaed\\\\\\\/yTAb8FmDXls2uFryEhzxJR0it52TBZUzRNsPQr63\\\\\\\/ENsZKWvGTL++wwXwFq5elTja5J7gDrlGDr7UTqLAOz2cxHzE\\\\\\\/1rLPxfq\\\\\\\/7lu7mMSQcAtJDUjwj8bw2RVwy4Xb29b1NOkJQ328xhJzo8VfUV3PMPI9E45wk61e93UlQc94sYjNUSB2IGpyba5RvIyea\\\\\\\/15XWYCSB\\\\\\\/F1sVNh\\\\\\\/WpEute5DARFkfoNhlDxV72ZowM7UFPT73WiTqHxAdTeqIg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UVYbTU7YKcs+eL22KzZhPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002293eadc471c70f3cd4c78d0bee4312d5ba20557da46ee4d33a8eedb424141eccc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459512 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1253" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MBZeLTanGEA7a9_vCN6bCQIjFYPM6HTng8DYI-jUwsA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1253" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BcHw74csMwo4VPIg1g3M3uGo7gquqd0CRFnIbRQlYlg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/8/1241" title="Animal Models and Integrated Nested Laplace Approximations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/8/1261" title="Recombination-Induced Tag Exchange (RITE) Cassette Series to Monitor Protein Dynamics in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459512" id="node459512" data-pisa="ggg;3/8/1253" data-pisa-master="ggg;g3.113.006643" data-apath="/ggg/3/8/1253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lucy Crooks</span>, <span class="highwire-citation-author" data-delta="1">Örjan Carlborg</span>, <span class="highwire-citation-author" data-delta="2">Stefan Marklund</span> and <span class="highwire-citation-author" data-delta="3">Anna M. Johansson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1253-1260;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006643 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459512" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lucy Crooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, Uppsala, SE-75124, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BCrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crooks%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucy%2BCrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stefan Marklund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BMarklund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marklund%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BMarklund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anna M. Johansson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Uppsala, SE-75007, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BM.%2BJohansson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johansson%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BM.%2BJohansson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.johansson@slu.se" class="" data-icon-position="" data-hide-link-title="0">anna.johansson@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/8/1253" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459512" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459512/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1253.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459512" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459512/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1253.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459512" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459512/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/8/1253.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459512" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459512/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/8/1253.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459512" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459512/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">We analyzed genotypes from ~10K single-nucleotide polymorphisms (SNPs) in two families of an F<sub>2</sub> intercross between Red Junglefowl and White Leghorn chickens. Possible null alleles were found by patterns of incompatible and missing genotypes. We estimated that 2.6% of SNPs had null alleles compared with 2.3% with genotyping errors and that 40% of SNPs in which a parent and offspring were genotyped as different homozygotes had null alleles. Putative deletions were identified by null alleles at adjacent markers. We found two candidate deletions that were supported by fluorescence intensity data from a 60K SNP chip. One of the candidate deletions was from the Red Junglefowl, and one was present in both the Red Junglefowl and White Leghorn. Both candidate deletions spanned protein-coding regions and were close to a previously detected quantitative trait locus affecting body weight in this population. This study demonstrates that the ~50K SNP genotyping arrays now available for several agricultural species can be used to identify null alleles and deletions in data from large families. We suggest that our approach could be a useful complement to linkage analysis in experimental crosses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/null-allele" class="hw-term hw-article-keyword hw-article-keyword-null-allele" rel="nofollow">null allele</a></li><li class="kwd"><a href="/keyword/deletion" class="hw-term hw-article-keyword hw-article-keyword-deletion" rel="nofollow">deletion</a></li><li class="kwd"><a href="/keyword/chicken" class="hw-term hw-article-keyword hw-article-keyword-chicken" rel="nofollow">chicken</a></li></ul><p id="p-5">Null alleles at genetic markers are alleles that do not amplify during genotyping. They can be due to a deletion encompassing the marker, polymorphisms in the sequence where the primer should anneal or a single-nucleotide polymorphism (SNP) site being triallelic. Individuals that are heterozygous for a null allele often are genotyped as homozygous for their other allele. Consequently, null alleles can create incompatibilities between the genotypes of parents and offspring. Genotype incompatibilities can also occur through genotyping error. Because incorrect genotypes will affect the results of genetic analyses, markers showing incompatible genotypes often are excluded or some genotypes changed to missing values. However, identifying null alleles can be valuable for discovering functional variation. In particular, null alleles are an indication of a possible deletion. In humans, incompatibilities and missing values in genotype data from high-density SNP arrays (~1 million SNPs) have been used to map hundreds of deletions in the range of 1−1000 kb (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Conrad <em>et al.</em> 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">McCarroll <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kohler and Cutler 2007</a>).</p><p id="p-6">Copy number variation, which includes deletions, has been recognized as an important component of human genetic diversity in the last few years. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">McCarroll <em>et al.</em> (2008b)</a> detected 3048 regions with copy number variants (CNVs) from 270 people with a hybrid SNP-CNV array (1.8 million sites). Most of the CNVs that were in more than one individual were 1–500 kb long and they totaled 1.1% of the genome. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Conrad <em>et al.</em> (2010)</a> found and validated 8,599 CNVs from 41 people by array comparative genome hybridization (42 million probes). The majority of these CNVs were also 1−500 kb long and they covered 3.7% of the genome. CNVs have been associated with several complex human diseases (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Girirajan <em>et al.</em> 2011</a>). For example, a common 20-kb deletion upstream of the <em>IRGM</em> gene is a possible causal factor in Crohn’s disease (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">McCarroll <em>et al.</em> 2008a</a>). In the future, genome-wide association studies may be extended to CNVs (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Craddock <em>et al.</em> 2010</a>), and a challenge is to integrate CNVs with SNPs and smaller insertions and deletions (indels) into combined genetic analyses (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Korn <em>et al.</em> 2008</a>).</p><p id="p-7">Less is known about the prevalence of CNVs in most other species. In chickens, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Wang <em>et al.</em> (2010)</a> measured two to four individuals from each of three domestic breeds by array comparative genome hybridization (~400K probes). They found 96 CNVs with lengths from 10 kb to 2 Mb, which totaled 1.3% of the genome. Two studies have applied next-generation sequencing to CNV detection in chickens. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Rubin <em>et al.</em> (2010)</a> sequenced pooled samples for eight domestic chicken lines and Red Junglefowl at low depth and identified deletions by regions with no read coverage. They reported 1284 deletions, nearly all of which were shorter than 6 kb. One of the few longer deletions coincided with a QTL found in a cross between two of the lines and was associated with increased growth in the intercross offspring. <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kerstens <em>et al.</em> (2011)</a> performed paired-end sequencing on reduced representation libraries from pooled samples for four commercial chicken lines. They looked for paired reads mapping a significantly different distance apart than the expected insert size and discovered 188 CNVs. Of the deletions, most were smaller than 3 kb, and 40% were line specific. These studies indicate that deletions can contribute to genetic and phenotypic differences between chicken populations. However, the size range of deletions found by resequencing compared with the findings described previously for humans suggests that these sequencing approaches may be limited for detecting large deletions. Sequencing is also considerably more expensive than genotyping with microarrays. Although pooling samples reduces the cost, it provides no information on the CNVs in each individual.</p><p id="p-8">Several traits in chicken have recently been shown to be caused by structural variations in the genome, demonstrating that it is necessary to look beyond point mutations as causative genetic factors. An 8-kb deletion upstream of <em>SOX10</em> is the causative mutation for the Dark brown plumage color (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gunnarsson <em>et al.</em> 2011</a>). The frizzle feather trait is caused by a smaller deletion of 69 bp in a conserved region of <em>KRT75</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ng <em>et al.</em> 2012</a>). A complex rearrangement involving the <em>EDN3</em> locus produces the black pigmentation in Silkie chickens and three other breeds (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dorshorst <em>et al.</em> 2011</a>). Two mutations affecting the shape of the comb have also been shown to be structural rearrangements. The Rose-comb is a result of an inversion on chromosome 7 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Imsland <em>et al.</em> 2012</a>) and the Pea-comb is caused by a large number of copies of a duplicated sequence in intron 1 of <em>SOX5</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Wright <em>et al.</em> 2009</a>).</p><p id="p-9">Recently ~50K SNP arrays have become available for several agricultural species. Here, we explore the incidence of null alleles in genotype data of this density and their potential for finding deletions. An advantage of many animals and plants compared with humans for family-based null allele studies is the large number of offspring. There is stronger evidence for a null allele if genotype incompatibilities and missing values are seen for multiple individuals. The deletion studies in humans used only one offspring. We analyzed genotypes for just more than 10K SNPs in two chicken families from an experimental intercross. We distinguished five classes of possible null alleles and statistically assessed how well each class identified null alleles rather than genotyping error. Putative deletions were inferred by null alleles at adjacent markers. These were examined further by genotyping a subset of individuals with a 60K SNP chip. We discuss how applying this null allele method to intercross data could complement linkage analysis for identifying genetic factors underlying complex traits.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Studied population</h3><p id="p-10">Individuals were from two three-generation families of a well-studied F<sub>2</sub> intercross between White Leghorn and Red Junglefowl chickens (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kerje <em>et al.</em> 2003</a>). The White Leghorn is a domestic layer breed with a much greater growth rate and body size than the Red Junglefowl, which is the major wild ancestor. The founders for the population were a single Red Junglefowl male and three White Leghorn females. There were four F<sub>1</sub> individuals in the two studied families; they all had the same father and two also had the same mother. From these four F<sub>1</sub> parents, two families were generated comprising 23 and 25 F<sub>2</sub> individuals, respectively. Genotypes from 10,150 SNPs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Groenen <em>et al.</em> 2009</a>) on autosomes 1−28 were examined (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006643/-/DC1/FileS1.zip">File S1</a>). Physical positions for the 10K SNPs were taken from the Single-Nucleotide Polymorphism Database build 131 and for those not found in Single-Nucleotide Polymorphism Database, from the chicken genome assembly, build 2.1.</p></div><div id="sec-3" class="subsection"><h3>Null allele detection</h3><p id="p-11">We wrote software to report SNPs with genotype incompatibilities or many missing values in the three-generation families. When a parent and offspring are genotyped as homozygous for different alleles and the other parent has the allele in the offspring, the offspring could have inherited a null allele from the first parent. When several offspring have missing values and both parents are genotyped as homozygous or missing, the missing genotypes could be the result of inheriting two null alleles. If there are different homozygotes in the offspring but no heterozygotes, one parent has a missing genotype and the other parent is heterozygous, the first parent could have two null alleles with all the offspring inheriting one null allele. The genotypes for all three generations had to be consistent with a null allele. For an F<sub>1</sub> parent to have a null allele, one of its parents (grandparents) had to be homozygous or missing. The software is available from the authors on request.</p></div><div id="sec-4" class="subsection"><h3>Classes of null alleles</h3><p id="p-12">Cases of genotype incompatibilities that could not be explained by null alleles alone were treated as genotyping errors. These were split into instances where a parent and offspring were genotyped as different homozygotes and other errors. The remaining cases of possible null alleles were categorized into five classes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) based on the number of genotype incompatibilities and how many genotyping errors would be needed to explain them. We define one genotyping incompatibility as an incompatibility between one parent and one of its offspring. The classes are (1) more than one incompatibility that cannot be explained by one genotyping error (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>), (2) more than one incompatibility that can be explained by one genotyping error (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>), (3) one incompatibility where no more incompatibilities are expected (<em>i.e.</em>, the incompatibility is between a parent and grandparent and the parents appear to have the same allele; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>), (4) one incompatibility where more incompatibilities would be expected (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>), and (5) more than five offspring with missing values (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1E</a>) or occurrence of both homozygotes and no heterozygotes in the offspring (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1F</a>). For class (5), there had to be more than one offspring with a nonmissing genotype, and when the classification was based on the absence of heterozygotes, there had to be more than one offspring with each homozygote genotype.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of the five null allele classes. Examples are taken from the chicken data analyzed in this article (File S1) with the different alleles labeled 1 and 2. Missing genotypes are written as &#x201C;--.&#x201D; Null allele genotypes that could explain the observed genotypes are shown in square brackets with a null allele represented by 0, i.e., the genotype 00 is homozygous for a null allele and the genotype 20 has one copy of allele 2 and one null allele. The number of F2 offspring with each genotype is given in round brackets. (A) A class (1) null allele. Here, the F2 offspring genotyped 22 are a different homozygote from the F1 parent genotyped as 11. They could have inherited a null allele from the first parent (transmitted from the first of its parents) and a 2 allele from the second parent. There are 22 instances in which a parent and offspring are genotyped as different homozygotes and three genotyping errors would be needed to explain these inconsistencies (each parent actually having genotype 12 and one of the parents of the second parent having genotype 11 or 12). (B) A class (2) null allele. There are 14 instances in which a parent and offspring are genotyped as different homozygotes. However, a single genotyping error could explain the inconsistencies (first F1 parent actually having genotype 12). (C) A class (3) null allele. The second F1 parent is a different homozygote from one of its parents. Because both F1 parents have the same homozygous genotype, no incompatibilities are expected in the F2 offspring. (D) A class (4) null allele. One F2 offspring is genotyped as a different homozygote from its parent. If there were a null allele, more offspring would be expected to show the same incompatibility. (E) A class (5) null allele. Multiple F2 offspring with missing values indicate the inheritance of a null allele from each parent. (F) Another class (5) null allele. F2 offspring are genotyped as different homozygotes and none are genotyped as heterozygotes. The first F1 parent could have two null alleles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886772152" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of the five null allele classes. Examples are taken from the chicken data analyzed in this article (File S1) with the different alleles labeled 1 and 2. Missing genotypes are written as &#x201C;--.&#x201D; Null allele genotypes that could explain the observed genotypes are shown in square brackets with a null allele represented by 0, &lt;em&gt;i.e.&lt;/em&gt;, the genotype 00 is homozygous for a null allele and the genotype 20 has one copy of allele 2 and one null allele. The number of F&lt;sub&gt;2&lt;/sub&gt; offspring with each genotype is given in round brackets. (A) A class (1) null allele. Here, the F&lt;sub&gt;2&lt;/sub&gt; offspring genotyped 22 are a different homozygote from the F&lt;sub&gt;1&lt;/sub&gt; parent genotyped as 11. They could have inherited a null allele from the first parent (transmitted from the first of its parents) and a 2 allele from the second parent. There are 22 instances in which a parent and offspring are genotyped as different homozygotes and three genotyping errors would be needed to explain these inconsistencies (each parent actually having genotype 12 and one of the parents of the second parent having genotype 11 or 12). (B) A class (2) null allele. There are 14 instances in which a parent and offspring are genotyped as different homozygotes. However, a single genotyping error could explain the inconsistencies (first F&lt;sub&gt;1&lt;/sub&gt; parent actually having genotype 12). (C) A class (3) null allele. The second F&lt;sub&gt;1&lt;/sub&gt; parent is a different homozygote from one of its parents. Because both F&lt;sub&gt;1&lt;/sub&gt; parents have the same homozygous genotype, no incompatibilities are expected in the F&lt;sub&gt;2&lt;/sub&gt; offspring. (D) A class (4) null allele. One F&lt;sub&gt;2&lt;/sub&gt; offspring is genotyped as a different homozygote from its parent. If there were a null allele, more offspring would be expected to show the same incompatibility. (E) A class (5) null allele. Multiple F&lt;sub&gt;2&lt;/sub&gt; offspring with missing values indicate the inheritance of a null allele from each parent. (F) Another class (5) null allele. F&lt;sub&gt;2&lt;/sub&gt; offspring are genotyped as different homozygotes and none are genotyped as heterozygotes. The first F&lt;sub&gt;1&lt;/sub&gt; parent could have two null alleles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1253/F1.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/8/1253/F1.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463213" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Examples of the five null allele classes. Examples are taken from the chicken data analyzed in this article (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006643/-/DC1/FileS1.zip">File S1</a>) with the different alleles labeled 1 and 2. Missing genotypes are written as “--.” Null allele genotypes that could explain the observed genotypes are shown in square brackets with a null allele represented by 0, <em>i.e.</em>, the genotype 00 is homozygous for a null allele and the genotype 20 has one copy of allele 2 and one null allele. The number of F<sub>2</sub> offspring with each genotype is given in round brackets. (A) A class (1) null allele. Here, the F<sub>2</sub> offspring genotyped 22 are a different homozygote from the F<sub>1</sub> parent genotyped as 11. They could have inherited a null allele from the first parent (transmitted from the first of its parents) and a 2 allele from the second parent. There are 22 instances in which a parent and offspring are genotyped as different homozygotes and three genotyping errors would be needed to explain these inconsistencies (each parent actually having genotype 12 and one of the parents of the second parent having genotype 11 or 12). (B) A class (2) null allele. There are 14 instances in which a parent and offspring are genotyped as different homozygotes. However, a single genotyping error could explain the inconsistencies (first F<sub>1</sub> parent actually having genotype 12). (C) A class (3) null allele. The second F<sub>1</sub> parent is a different homozygote from one of its parents. Because both F<sub>1</sub> parents have the same homozygous genotype, no incompatibilities are expected in the F<sub>2</sub> offspring. (D) A class (4) null allele. One F<sub>2</sub> offspring is genotyped as a different homozygote from its parent. If there were a null allele, more offspring would be expected to show the same incompatibility. (E) A class (5) null allele. Multiple F<sub>2</sub> offspring with missing values indicate the inheritance of a null allele from each parent. (F) Another class (5) null allele. F<sub>2</sub> offspring are genotyped as different homozygotes and none are genotyped as heterozygotes. The first F<sub>1</sub> parent could have two null alleles.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Evidence from the two families</h3><p id="p-14">SNPs were given an overall classification from the results in the two families. When a SNP was in a different null allele class for the two families, if these were classes (1)−(4), the SNP was put in the numerically lower class. Because we subsequently found that class (4) SNPs were the least likely to have true null alleles, SNPs that were class (4) in one family and class (5) in the other were reported as class (5). SNPs with evidence for a null allele in one family and a genotyping error in the other family were placed in the null allele class. When a SNP had a genotyping error in both families and a parent and offspring were different homozygotes in one family but not the other, the SNP was recorded in the different homozygotes error category.</p></div><div id="sec-6" class="subsection"><h3>Evaluating null allele classification</h3><p id="p-15">The different null allele classes identify SNPs with varied evidence for null alleles but they could have genotyping errors rather than null alleles. We found that very few SNPs classed as having genotyping errors showed errors in both families. The number of SNPs with detected errors was also considerably higher in family 1 than family 2. In contrast, in these data, null alleles should be fairly often seen in both families and be found at similar frequencies in the two families, because they share a grandfather and grandmother and the other grandmothers are from the same line. We therefore used a statistical approach to assess whether the null allele classes were likely to contain a high proportion of SNPs with genotyping errors. First, we compared the percentage of SNPs with possible null alleles in both families for each null allele class with the percentage of SNPs in the different homozygotes error class that showed genotyping errors in both families. Second, we compared the ratio of SNPs identified in family 1 to family 2 for each null allele class with the different homozygotes error class. A much greater percentage found in both families than for genotyping error and a ratio in family 1 to family 2 close to one is evidence that most of the SNPs in the null allele class do not have genotyping errors.</p><p id="p-16">For null allele classes in which the ratio of SNPs in family 1 to family 2 was greater than one, we estimated the percentage of SNPs that had genotyping errors as follows. Treating the null allele class as a mixture of SNPs with null alleles, which occur at equal rates in the two families, and a fraction <em>x</em> of SNPs with different homozygotes genotyping errors, which occur at a ratio <em>r<sub>e</sub></em> in family 1 to family 2, the ratio of SNPs from family 1 to family 2 in the null allele class is given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1253/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1253/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>Hence, <em>x</em> can be estimated by</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1253/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/8/1253/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span></div><div id="sec-7" class="subsection"><h3>Putative deletions</h3><p id="p-17">We looked for possible null alleles at adjacent SNPs in the same family. A region was considered a putative deletion if a F<sub>1</sub> parent and at least three F<sub>2</sub> individuals showed evidence of linked null alleles at both SNPs. The size of the putative deletion was estimated as the distance between the SNPs.</p></div><div id="sec-8" class="subsection"><h3>Denser genotyping and intensity data</h3><p id="p-18">On the basis of the segregation patterns at the SNPs showing putative deletions, we chose 22 F<sub>2</sub> individuals for denser genotyping with the Illumina 60K chicken SNP chip described by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Groenen <em>et al.</em> (2011)</a>. Eight F<sub>2</sub> individuals were chosen from each family, such that half were expected to have each deletion. The shared grandfather, the shared grandmother, and the F<sub>1</sub> parents of both families also were genotyped. The fluorescence intensity data for SNPs in and near the putative deletion regions were plotted along with results from 80 other individuals genotyped at the same time, to help identify genotype clusters. Plots should show three clusters for biallelic SNPs: individuals homozygous for one allele should fall close to one axis; individuals homozygous for the other allele, close to the other axis; and individuals heterozygous for the two alleles, on the <em>x = y</em> line in the center of the plot. We inspected the plots by eye for additional clusters. A second cluster near an axis but at lower values indicates individuals with one copy of an allele and one null allele and a cluster near the origin indicates individuals with two null alleles. If SNPs within and outside the original 10K interval showed null alleles, the extent of the candidate deletion was increased to the span of these SNPs.</p></div><div id="sec-9" class="subsection"><h3>Genes in candidate deletion regions</h3><p id="p-19">The Ensembl (<a href="http://www.ensembl.org/">http://www.ensembl.org/</a>) and UniProt (<a href="http://www.uniprot.org/">http://www.uniprot.org/</a>) databases were accessed on 2/3/2012 to determine any genes within the candidate deletion regions and the possible functions of any protein products.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-20">In total, we detected possible null alleles in 330 of the 10150 SNP markers (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Class (5) was largest and nearly all of these were cases in which many F<sub>2</sub> individuals had missing values that could be due to two null alleles. It is unsurprising that this was the largest null allele class as missing genotypes can be observed regardless of the other genotypes in a family but whether there can be genotype incompatibilities will depend on which genotypes are present. Only two of the cases in class (5) showed an absence of heterozygote individuals and presence of both homozygotes in the offspring. Classes (2) and (4) were also large. Most of the SNPs in class (2) had a potential null allele in one F<sub>1</sub> parent. Nearly all the cases in class (4) were due to one F<sub>2</sub> offspring having a genotype that was incompatible with one of its parents. Class (1) contained 28 SNPs and for about half of these cases there appeared to be a null allele in both parents. In the others, the parent with the possible null allele and both its parents were homozygous for the same allele and therefore a single genotyping error cannot explain the observed genotype incompatibilities. Class (3) was the smallest class. These were SNPs with a single incompatibility between an F<sub>1</sub> parent and one of its parents and both F<sub>1</sub> parents appeared to have the same allele so a null allele could not cause incompatibilities between the parents and the F<sub>2</sub> offspring.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463187/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463187/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of SNPs with possible null alleles and genotyping errors</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">There were 164 SNPs that we categorized as having genotyping errors and not null alleles (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Just less than half of these had a parent and offspring genotyped as different homozygotes. In null allele classes (1), (2), and (5), approximately 40% of the SNPs had evidence of null alleles in both families (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). For class (3) this was 20%. None of the SNPs in class (4) were identified in both families. In the genotyping error categories, 4% and 7% of SNPs were recorded with errors in both families. More than 15 times more SNPs from family 1 than family 2 had apparent genotyping errors involving different homozygotes (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). There were also nearly 10 times more SNPs from family 1 than family 2 in null allele class (4). The ratio of SNPs from family 1 to family 2 was slightly above one for null allele classes (2) and (3) and below one for classes (1) and (5). We estimated that 60% of the SNPs in class (4) had genotyping errors rather than null alleles (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The estimates for class (2) and (3) were only 3% and 1%, respectively.</p><p id="p-27">Four putative deletions were found from the 10K genotypes, which were on chromosomes 1 and 3 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Two of these were identified by class (5) null alleles, one by class (2) null alleles, and one by a class (1) and a class (2) null allele. The estimated sizes of the putative deletions ranged from 5 to 157 kb. Two of the regions were detected in both families, one only in family 1, and one only in family 2. The genotypes for the regions are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, with the suspected null alleles indicated. For region 2 the putative deletion is from the Red Junglefowl and for region 4, from the White Leghorn. For regions 1 and 3 the origin of the potential deletion is not conclusive from the genotypes. For region 1, the grandmothers had missing values suggesting that the deletion is from the White Leghorn. For region 3, the subsequent intensity data indicated that the deletion was homozygous in the Red Junglefowl individual but also occurred in the White Leghorn.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463073/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463073/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Putative deletion regions identified from 10K SNP genotypes</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotypes at adjacent SNPs indicating putative deletions. Genotypes for the two SNPs are shown one above the other, with the first SNP on top. The number of F2 individuals with each genotype is shown in round brackets. Missing genotypes are written as &#x201C;--.&#x201D; Null allele genotypes that could explain the observed genotypes are shown in square brackets with a null allele represented by 0. Within the brackets &#x201C;&#x2013;&#x201D; denotes an unknown allele. In region 1, the null allele was assumed to be in the F0 individuals that had missing genotypes at both markers. In region 3, the null allele genotypes were inferred using both the genotype calls and the intensity data. (A) Region 1 at 35.7 Mb on chromosome 1 observed in family 2. (B) Region 2 from 125.9&#x2212;126.1 Mb on chromosome 1 observed in both families. (C) Region 3 at 166 Mb on chromosome 1 observed in both families. (D) Region 4 at 18.3 Mb on chromosome 3 observed in family 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886772152" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotypes at adjacent SNPs indicating putative deletions. Genotypes for the two SNPs are shown one above the other, with the first SNP on top. The number of F&lt;sub&gt;2&lt;/sub&gt; individuals with each genotype is shown in round brackets. Missing genotypes are written as &#x201C;--.&#x201D; Null allele genotypes that could explain the observed genotypes are shown in square brackets with a null allele represented by 0. Within the brackets &#x201C;&#x2013;&#x201D; denotes an unknown allele. In region 1, the null allele was assumed to be in the F&lt;sub&gt;0&lt;/sub&gt; individuals that had missing genotypes at both markers. In region 3, the null allele genotypes were inferred using both the genotype calls and the intensity data. (A) Region 1 at 35.7 Mb on chromosome 1 observed in family 2. (B) Region 2 from 125.9&#x2212;126.1 Mb on chromosome 1 observed in both families. (C) Region 3 at 166 Mb on chromosome 1 observed in both families. (D) Region 4 at 18.3 Mb on chromosome 3 observed in family 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1253/F2.medium.gif" width="225" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/8/1253/F2.medium.gif" width="225" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463180" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Genotypes at adjacent SNPs indicating putative deletions. Genotypes for the two SNPs are shown one above the other, with the first SNP on top. The number of F<sub>2</sub> individuals with each genotype is shown in round brackets. Missing genotypes are written as “--.” Null allele genotypes that could explain the observed genotypes are shown in square brackets with a null allele represented by 0. Within the brackets “–” denotes an unknown allele. In region 1, the null allele was assumed to be in the F<sub>0</sub> individuals that had missing genotypes at both markers. In region 3, the null allele genotypes were inferred using both the genotype calls and the intensity data. (A) Region 1 at 35.7 Mb on chromosome 1 observed in family 2. (B) Region 2 from 125.9−126.1 Mb on chromosome 1 observed in both families. (C) Region 3 at 166 Mb on chromosome 1 observed in both families. (D) Region 4 at 18.3 Mb on chromosome 3 observed in family 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">There was support for two of the putative deletions from the 60K SNP chip data. They were the regions that were detected in both families. For region 2, there were five new SNPs with positions inside the 10K interval. In the intensity plot for the second of these SNPs, the grandfather and F<sub>2</sub> individuals inferred to have the deletion clustered along the <em>y</em>-axis and the F<sub>1</sub> parents thought to have the deletion were close to the <em>x</em>-axis (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Therefore, in this subset of individuals, only one allele was detected and it was a different allele in the grandfather and F<sub>2</sub> individuals inferred to have the deletion than in the parents. These data show a null allele consistent with the results from the 10K SNPs and support a deletion. However, there were some conflicting findings. The intensity data from the first and third SNPs showed that the individuals inferred to have the deletion were heterozygous and therefore did not have a null allele at these SNPs. Because of these conflicting findings, we did not change our estimate of the extent of this candidate deletion.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intensity plot for a SNP in region 2. Intensity values are colored as follows: red, F2 individuals believed to have the null allele; black, other F2 individuals; dark purple, grandfather; light purple, grandmother; dark blue, parents thought to have the null allele; light blue, other parents; gray, additional individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886772152" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intensity plot for a SNP in region 2. Intensity values are colored as follows: red, F&lt;sub&gt;2&lt;/sub&gt; individuals believed to have the null allele; black, other F&lt;sub&gt;2&lt;/sub&gt; individuals; dark purple, grandfather; light purple, grandmother; dark blue, parents thought to have the null allele; light blue, other parents; gray, additional individuals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1253/F3.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/8/1253/F3.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463162" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Intensity plot for a SNP in region 2. Intensity values are colored as follows: red, F<sub>2</sub> individuals believed to have the null allele; black, other F<sub>2</sub> individuals; dark purple, grandfather; light purple, grandmother; dark blue, parents thought to have the null allele; light blue, other parents; gray, additional individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The 60K SNP chip contained one new marker located between the 10K markers in region 3. The intensity plot for this SNP, and the closest SNP on either side had a distinct cluster of individuals near the origin and the remaining individuals grouped together at greater x-values (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The cluster close to the origin contained the grandfather (who had missing genotypes at both SNPs in the 10K data), the F<sub>2</sub> individuals with missing genotypes at both SNPs in the 10K data and the parents from each family that had missing genotypes at both SNPs in the 10K data. Hence, the intensity data show that these individuals have two null alleles. It follows, and is supported by their similar intensity values, that the grandmother, other parents and remaining F<sub>2</sub> individuals all have one null allele. The intensity data for these three additional markers support a deletion in this region and show that the deletion is in both the Red Junglefowl and the White Leghorn, with the Red Junglefowl being homozygous for the deletion and one White Leghorn female being heterozygous for the deletion. Based on the 10K data, at least one other White Leghorn female is also heterozygous for the deletion. We increased the estimated extent of this candidate deletion to 165.93−165.98 Mb with an estimated size of 55.6 kb.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intensity plot for three SNPs spanning region 3. Intensity values are colored as follows: red, F2 individuals that had missing genotypes at both SNPs in the 10K data; black, other F2 individuals; dark purple, grandfather; light purple, grandmother; dark blue, parents that had missing genotypes at both SNPs in the 10K data; light blue, other parents; gray, additional individuals (A) SNP preceding region 3. (B) SNP within region 3. (C) SNP succeeding region 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-886772152" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Intensity plot for three SNPs spanning region 3. Intensity values are colored as follows: red, F&lt;sub&gt;2&lt;/sub&gt; individuals that had missing genotypes at both SNPs in the 10K data; black, other F&lt;sub&gt;2&lt;/sub&gt; individuals; dark purple, grandfather; light purple, grandmother; dark blue, parents that had missing genotypes at both SNPs in the 10K data; light blue, other parents; gray, additional individuals (A) SNP preceding region 3. (B) SNP within region 3. (C) SNP succeeding region 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/8/1253/F4.medium.gif" width="440" height="149"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/8/1253/F4.medium.gif" width="440" height="149"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/8/1253/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463071" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Intensity plot for three SNPs spanning region 3. Intensity values are colored as follows: red, F<sub>2</sub> individuals that had missing genotypes at both SNPs in the 10K data; black, other F<sub>2</sub> individuals; dark purple, grandfather; light purple, grandmother; dark blue, parents that had missing genotypes at both SNPs in the 10K data; light blue, other parents; gray, additional individuals (A) SNP preceding region 3. (B) SNP within region 3. (C) SNP succeeding region 3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">There were no new markers within region 1 on the 60K SNP chip. The intensity plots of the closest marker on either side of region 1 gave no indication of null alleles. Additional data would therefore have to be collected to evaluate this region. For region 4, there was one new marker on the 60K SNP chip located between the 10K markers. The intensity plots for this marker and the nearest marker on either side showed that the F<sub>2</sub> individuals thought to carry the deletion were heterozygous and do not support a deletion in this region.</p><p id="p-35">Region 2 covers the first seven out of nine exons of gene <em>GLRA2</em> (Ensembl). However, the span of the potential deletion is uncertain because of the conflicting results from the 60K SNP data. The gene has a single transcript <em>ENSGALT00000026747</em>. The protein is uncharacterized but clusters with 90% identity to glycine receptor subunit alpha-2 proteins in human, mouse and rat, which are involved in inhibitory neurotransmission (UniProt). Region 3 covers the complete length of gene <em>ENSGALG00000014545</em>, which has a single transcript, <em>ENSGALT00000023466</em> (Ensembl). The protein is uncharacterized but clusters with 90% identity to zinc finger RNA-binding proteins in human, orangutan, mouse and rat (UniProt).</p></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-36">We detected a large number of possible null alleles by examining ~10K SNP genotypes for genotype incompatibilities and missing values in two large chicken families. The suspected null alleles were grouped into different classes based on the type of evidence for null alleles. We used the findings across families to assess whether the classes detected true null alleles. From the null allele results, we identified four potential deletions segregating in the population. Two of these were supported by new genotyping results from a 60K SNP chip, one was not supported by the new data and in one case the denser chip provided no additional information. The two candidate deletions were on chromosome 1. Both were present in the Red Junglefowl line and one was also in the White Leghorn Line. Each candidate spans a protein-coding region.</p><p id="p-37">We evaluated the null allele classes by measuring the percentage of SNPs that were identified with possible null alleles in both families and the ratio of SNPs found from family 1 to family 2. Both approaches indicate that a high proportion of SNPs in classes (1)-(3) and class (5) have true null alleles but suggest that many class (4) SNPs have genotyping errors. Class (4) consists of cases with one genotype incompatibility, predominately between an F<sub>1</sub> parent and one of the F<sub>2</sub> offspring, where more incompatibilities would be expected if the parent had a null allele. Class (3) also consists of SNPs with one incompatibility but the incompatibility is between an F<sub>1</sub> parent and F<sub>0</sub> grandparent and because of the genotypes in the family a null allele could not cause incompatibilities in the F<sub>2</sub> offspring. SNPs in both these classes could have a single genotyping error rather than a null allele. However, the distribution of SNPs between families in these two classes was very different. For class (3), 22% of SNPs had evidence of null alleles in both families compared with 0 in class (4). The ratio of SNPs found in family 1 to family 2 was just greater than 1 for class (3) and nearly 10 for class (4). By comparing this ratio to the value for different homozygote errors, we estimated that only 1% of the SNPs in class (3) had genotyping errors. We were able to distinguish classes (1)−(4) because we had genotypes for multiple offspring. In studies with a single offspring, such as those in humans, null alleles can only generate one genotype incompatibility. Our results suggest that null allele detection is more accurate when multiple offspring are genotyped and cases with a genotype incompatibility between a parent and only one of the offspring are excluded.</p><p id="p-38">Null allele class (5) predominantly contained SNPs with missing genotypes and we had expected this class to have a high false-positive rate because there was no direct evidence for a null allele. Instead, our statistical assessment indicated that most class (5) SNPs had true null alleles. It was also class (5) SNPs that identified one of the candidate deletions. Class (5) probably had a high rate of true null alleles because we had data for multiple offspring and only included SNPs based on missing genotypes when at least five individuals had missing genotypes. Random genotyping error is unlikely to cause several missing genotypes. However, there could be multiple missing genotypes if there were technical problems with the genotyping assay. This type of genotyping failure would not be detected by the measures we used to assess the null allele classes if it produced missing values in both families. To try to reduce the possibility that class (5) SNPs were due to problems with the genotyping assay, we excluded instances where all, or all but one, F<sub>2</sub> individuals had missing genotypes. The average number of individuals that had genotype calls for SNPs in class (5) was seventeen per family, which suggests that assay problems are not a general explanation for these cases.</p><p id="p-39">The estimated percentage of SNPs with null alleles after excluding class (4) was 2.6%. Adding class (4) to the error categories, the estimated percentage of SNPs with genotyping errors was 2.3%. The actual values of both may be higher because not all genotyping errors and probably not all null alleles would have been detectable given the genotypes in a family. Some null alleles in the F<sub>0</sub> individuals will not have been transmitted to the F<sub>1</sub> parents. Null alleles in the grandfather had a higher chance of being transmitted than null alleles in one of the grandmothers because he fathered all the F<sub>1</sub> parents. Although we have estimated the error rate within the null allele classes to be low, some of the possible null alleles will be genotyping errors. Including more related families should increase the number of null alleles found because it increases the chance of the alleles being transmitted and the alleles may occur with a different set of genotypes. However there will be diminishing returns from adding families as the number of families is increased. The number of SNPs with genotyping errors will increase if more families are added and this increase should be independent of the number of families already included. Our results suggest that a relatively high proportion of genotype inconsistencies in chicken SNP data could be due to null alleles and therefore there may be useful information in these data. We estimated that nearly 40% of SNPs in which a parent and offspring were genotyped as different homozygotes had null alleles. When only a single marker shows evidence of a null allele, it is not possible to know whether this is due to a deletion or an additional SNP in the primer-annealing site that interferes with the amplification. In this article, we only consider null alleles as a sign of a potential deletion when two adjacent SNPs show evidence of null alleles in the same individuals.</p><p id="p-40">To gain support for the putative deletions we identified from our null allele results, we genotyped selected individuals with a 60K SNP chip. The intensity data from the 60K chip provided support for two of the potential deletions. However, for one of these, region 2, there were conflicting results from the SNPs located within the 10K interval. One SNP showed null alleles in the individuals thought to have the deletion, but for two of the other SNPs these individuals were heterozygous and therefore did not have null alleles. Overall, we believe that there is sufficient evidence to consider this region as a candidate deletion. Possible explanations for the discrepancies in results between SNPs are that the relative positions of the 60K and 10K SNPs are incorrect in the consensus sequence, that the probes for those SNPs not showing null alleles also target another position in the genome, or that there is a more complex rearrangement in this region. Quantitative PCR with a series of primers flanking and within region 2 would confirm if there is a deletion and help in determining its size and understanding the different findings across SNPs.</p><p id="p-41">The data used in this study are from two families of an F<sub>2</sub> intercross between a domesticated White Leghorn chicken line and its wild ancestor, the Red Junglefowl. The founder lines differ considerably in a number of traits including growth, egg production and behavior (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Kerje <em>et al.</em> 2003</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Schutz <em>et al.</em> 2004</a>). One of our candidate deletions were detected only in the Red Junglefowl and the other was homozygous in the Red Junglefowl but segregating in the White Leghorn founders, and therefore they may contribute to the observed phenotypic differences between these lines. A genome-wide scan for QTL affecting growth in this intercross detected 22 loci (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Carlborg <em>et al.</em> 2003</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Kerje <em>et al.</em> 2003</a>). Our candidate deletions are each located within 9 cM of one of these QTL, based on the consensus linkage map (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Groenen <em>et al.</em> 2009</a>). Region 2 was near an epistatic QTL that interacted with a locus on chromosome 14 to affect hatch weight. Region 3 was located near a QTL that affected post-hatch body weight at four different ages and growth between these ages. To investigate these candidates further, the deletions need to be confirmed and their breakpoints determined. After validation, the next step would be to genotype the whole intercross population for the deletion and directly test for an association with body weight.</p><p id="p-42">We suggest that our method for detecting potential deletions could be a useful complement to linkage analysis of intercross data. Linkage analysis generally identifies fairly broad QTL regions that contain numerous genes. Deletions that are close to QTL are possible candidates for the variants underlying QTL effects. Usually in CNV studies, many variants are detected but it is difficult to evaluate which could be functionally important. When linkage regions have been associated with a trait, there is incidental evidence that putative deletions in these regions are functional. Intercross data have some advantages for identifying null alleles and inferring deletions from them. The sets of related families can increase confidence in possible null alleles that are detected in several families. For both our candidate deletions that were supported by the 60K data, null alleles were implicated in both families. There is also data available from grandparents. Checking that grandparental genotypes are consistent with null alleles is an additional filter to improve accuracy.</p><p id="p-43">High-density SNP arrays have been used for CNV discovery in humans for several years. These studies are usually carried out in populations of unrelated individuals and require consistent signals at multiple SNPs to call a CNV. Lower density SNP arrays have recently become available for a number of domesticated plants and animals. The density in these arrays is probably too low to detect moderate-sized deletions using the population-level approaches applied to human data. Here, we have shown that with large families, where the pattern of null alleles in the pedigree can be used, data from these lower density arrays can identify null alleles at adjacent markers that could be deletions. We considered linked null alleles at two adjacent SNPs sufficient to infer a candidate deletion and two of our four potential deletions were supported by additional genotyping data. Some of the single SNPs with possible null alleles, particularly those in class (1) and (2) that were identified in both families, may also be due to deletions and it would be interesting to investigate these regions with denser SNP data. Despite only requiring two SNPs to show evidence of null alleles, both our candidate deletions were estimated to be larger than 50 kb. More, and shorter, deletions will be detectable as higher density arrays are produced, reflecting the increase in reported CNV numbers with array density in human genetics.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Per Jensen and Leif Andersson for their work with developing the intercross population and sharing this data. We also thank Leif Andersson for allowing us to use unpublished genetic data. This study was funded by a Future Research Leader grant to Ö.C. from the Swedish Foundation for Strategic research and a EURYI Award to Ö.C.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006643/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006643/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-45"><em>Communicating editor: D.W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-12-11"><span class="received-label">Received </span>December 11, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-05-14"><span class="accepted-label">Accepted </span>May 14, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Crooks <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.1" data-doi="10.1101/gr.528003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerje</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schütz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A global search reveals epistatic interaction between QTL for early growth in the chicken.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D413%26rft.epage%253D421%26rft.atitle%253DA%2BGlobal%2BSearch%2BReveals%2BEpistatic%2BInteraction%2BBetween%2BQTL%2Bfor%2BEarly%2BGrowth%2Bin%2Bthe%2BChicken%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.528003%26rft_id%253Dinfo%253Apmid%252F12618372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNDEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.2" data-doi="10.1038/nm0106-75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prichard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution survey of deletion polymorphism in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D81%26rft.atitle%253DA%2Bhigh-resolution%2Bsurvey%2Bof%2Bdeletion%2Bpolymorphism%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm0106-75%26rft_id%253Dinfo%253Apmid%252F16327808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm0106-75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16327808&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234227200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.3" data-doi="10.1038/nature08516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Origins and functional impact of copy number variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D704%26rft.epage%253D712%26rft.atitle%253DOrigins%2Band%2Bfunctional%2Bimpact%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08516%26rft_id%253Dinfo%253Apmid%252F19812545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.4" data-doi="10.1038/nature08979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Craddock</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D713%26rft.epage%253D720%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2BCNVs%2Bin%2B16%252C000%2Bcases%2Bof%2Beight%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08979%26rft_id%253Dinfo%253Apmid%252F20360734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20360734&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.5" data-doi="10.1371/journal.pgen.1002412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorshorst</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molin</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strömstedt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Complex genomic rearrangement involving the <em>Endothelin 3</em> locus causes dermal hyperpigmentation in the chicken.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDorshorst%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002412%26rft.epage%253De1002412%26rft.atitle%253DA%2Bcomplex%2Bgenomic%2Brearrangement%2Binvolving%2Bthe%2Bendothelin%2B3%2Blocus%2Bcauses%2Bdermal%2Bhyperpigmentation%2Bin%2Bthe%2Bchicken.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002412%26rft_id%253Dinfo%253Apmid%252F22216010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22216010&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.6" data-doi="10.1146/annurev-genet-102209-163544"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Girirajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Human copy number variation and complex disease genetics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DGirirajan%26rft.auinit1%253DS.%26rft.volume%253D45%26rft.spage%253D203%26rft.epage%253D226%26rft.atitle%253DHuman%2Bcopy%2Bnumber%2Bvariation%2Band%2Bcomplex%2Bgenetic%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-102209-163544%26rft_id%253Dinfo%253Apmid%252F21854229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-102209-163544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21854229&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299299600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.7" data-doi="10.1101/gr.086538.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folglio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086538.108%26rft_id%253Dinfo%253Apmid%252F19088305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy84LzEyNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.8" data-doi="10.1186/1471-2164-12-274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zare</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The development and characterization of a 60K SNP chip for chicken.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGroenen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D274%26rft.epage%253D274%26rft.atitle%253DThe%2Bdevelopment%2Band%2Bcharacterization%2Bof%2Ba%2B60K%2BSNP%2Bchip%2Bfor%2Bchicken.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-274%26rft_id%253Dinfo%253Apmid%252F21627800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21627800&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.9" data-doi="10.1111/j.1755-148X.2011.00825.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunnarsson</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerje</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bed’hom</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahlqvist</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ekwall</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Dark brown plumage color in chickens is caused by an 8.3-kb deletion upstream of SOX10.</span> <abbr class="cit-jnl-abbrev">Pigment Cell Melanoma Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPigment%2BCell%2BMelanoma%2BRes%26rft.aulast%253DGunnarsson%26rft.auinit1%253DU.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D268%26rft.epage%253D274%26rft.atitle%253DThe%2BDark%2Bbrown%2Bplumage%2Bcolor%2Bin%2Bchickens%2Bis%2Bcaused%2Bby%2Ban%2B8.3-kb%2Bdeletion%2Bupstream%2Bof%2BSOX10.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-148X.2011.00825.x%26rft_id%253Dinfo%253Apmid%252F21210960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-148X.2011.00825.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21210960&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288219200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.10" data-doi="10.1371/journal.pgen.1002775"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imsland</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boije</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bed’hom</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fillon</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Rose-comb mutation in chickens constitutes a structural rearrangement causing both altered comb morphology and defective sperm motility.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DImsland%26rft.auinit1%253DF.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002775%26rft.epage%253De1002775%26rft.atitle%253DThe%2BRose-comb%2Bmutation%2Bin%2Bchickens%2Bconstitutes%2Ba%2Bstructural%2Brearrangement%2Bcausing%2Bboth%2Baltered%2Bcomb%2Bmorphology%2Band%2Bdefective%2Bsperm%2Bmotility.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002775%26rft_id%253Dinfo%253Apmid%252F22761584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002775&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761584&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.11" data-doi="10.1046/j.1365-2052.2003.01000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerje</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schütz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The twofold difference in adult size between the red junglefowl and White Leghorn chickens is largely explained by a limited number of QTLs.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DKerje%26rft.auinit1%253DS.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D264%26rft.epage%253D274%26rft.atitle%253DThe%2Btwofold%2Bdifference%2Bin%2Badult%2Bsize%2Bbetween%2Bthe%2Bred%2Bjunglefowl%2Band%2BWhite%2BLeghorn%2Bchickens%2Bis%2Blargely%2Bexplained%2Bby%2Ba%2Blimited%2Bnumber%2Bof%2BQTLs.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2052.2003.01000.x%26rft_id%253Dinfo%253Apmid%252F12873214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2052.2003.01000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12873214&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184331200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.12" data-doi="10.1186/1471-2164-12-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerstens</span>  <span class="cit-name-given-names">H. H</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dibbits</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vereijken</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okimoto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Structural variation in the chicken genome identified by paired-end next-generation DNA sequencing of reduced representation libraries.</span> <abbr class="cit-jnl-abbrev">BMC Genomics.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKerstens%26rft.auinit1%253DH.%2BH.%26rft.volume%253D12%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DStructural%2Bvariation%2Bin%2Bthe%2Bchicken%2Bgenome%2Bidentified%2Bby%2Bpaired-end%2Bnext-generation%2BDNA%2Bsequencing%2Bof%2Breduced%2Brepresentation%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-94%26rft_id%253Dinfo%253Apmid%252F21291514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21291514&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.13" data-doi="10.1086/520823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Simultaneous discovery and testing of deletions for disease association in SNP genotyping studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">684</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DKohler%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D684%26rft.epage%253D699%26rft.atitle%253DSimultaneous%2Bdiscovery%2Band%2Btesting%2Bof%2Bdeletions%2Bfor%2Bdisease%2Bassociation%2Bin%2BSNP%2Bgenotyping%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F520823%26rft_id%253Dinfo%253Apmid%252F17846995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/520823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846995&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249307600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.14" data-doi="10.1038/ng.237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuruvilla</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1253</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorn%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D40%26rft.issue%253D10%26rft.spage%253D1253%26rft.epage%253D1260%26rft.atitle%253DIntegrated%2Bgenotype%2Bcalling%2Band%2Bassociation%2Banalysis%2Bof%2BSNPs%252C%2Bcommon%2Bcopy%2Bnumber%2Bpolymorphisms%2Band%2Brare%2BCNVs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.237%26rft_id%253Dinfo%253Apmid%252F18776909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18776909&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259651000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.15" data-doi="10.1038/ng1696"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadnott</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Common deletion polymorphisms in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMcCarroll%26rft.auinit1%253DS.%2BA.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D86%26rft.epage%253D92%26rft.atitle%253DCommon%2Bdeletion%2Bpolymorphisms%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1696%26rft_id%253Dinfo%253Apmid%252F16468122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1696&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16468122&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234227200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.16" data-doi="10.1038/ng.215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huett</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuballa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chilewski</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">Deletion polymorphism upstream of <em>IRGM</em> associated with altered <em>IRGM</em> expression and Crohn’s disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMcCarroll%26rft.auinit1%253DS.%2BA.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D1107%26rft.epage%253D1112%26rft.atitle%253DDeletion%2Bpolymorphism%2Bupstream%2Bof%2BIRGM%2Bassociated%2Bwith%2Baltered%2BIRGM%2Bexpression%2Band%2BCrohn%2527s%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.215%26rft_id%253Dinfo%253Apmid%252F19165925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165925&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258761200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.17" data-doi="10.1038/ng.238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuruvilla</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cawley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">Integrated detection and population-genetic analysis of SNPs and copy number variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMcCarroll%26rft.auinit1%253DS.%2BA.%26rft.volume%253D40%26rft.issue%253D10%26rft.spage%253D1166%26rft.epage%253D1174%26rft.atitle%253DIntegrated%2Bdetection%2Band%2Bpopulation-genetic%2Banalysis%2Bof%2BSNPs%2Band%2Bcopy%2Bnumber%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.238%26rft_id%253Dinfo%253Apmid%252F18776908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18776908&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259651000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.18" data-doi="10.1371/journal.pgen.1002748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The chicken frizzle feather is due to an α-keratin (KRT75) mutation that causes a defective rachis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNg%26rft.auinit1%253DC.%2BS.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002748%26rft.epage%253De1002748%26rft.atitle%253DThe%2Bchicken%2Bfrizzle%2Bfeather%2Bis%2Bdue%2Bto%2Ban%2B%25CE%25B1-keratin%2B%2528KRT75%2529%2Bmutation%2Bthat%2Bcauses%2Ba%2Bdefective%2Brachis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002748%26rft_id%253Dinfo%253Apmid%252F22829773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22829773&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.19" data-doi="10.1038/nature08832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwood</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome resequencing reveals loci under selection during chicken domestication.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRubin%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D464%26rft.issue%253D7288%26rft.spage%253D587%26rft.epage%253D591%26rft.atitle%253DWhole-genome%2Bresequencing%2Breveals%2Bloci%2Bunder%2Bselection%2Bduring%2Bchicken%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08832%26rft_id%253Dinfo%253Apmid%252F20220755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220755&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275974200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.20" data-doi="10.1023/B:BEGE.0000009481.98336.fc"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schutz</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerje</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forkman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Major growth QTLs in fowl are related to fearful behavior: possible genetic links between fear responses and production traits in a red junglefowl x White Leghorn intercross.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DSch%25C3%25BCtz%26rft.auinit1%253DK.%2BE.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D121%26rft.epage%253D130%26rft.atitle%253DMajor%2Bgrowth%2BQTLs%2Bin%2Bfowl%2Bare%2Brelated%2Bto%2Bfearful%2Bbehavior%253A%2Bpossible%2Bgenetic%2Blinks%2Bbetween%2Bfear%2Bresponses%2Band%2Bproduction%2Btraits%2Bin%2Ba%2Bred%2Bjunglefowl%2Bx%2Bwhite%2Bleghorn%2Bintercross.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253ABEGE.0000009481.98336.fc%26rft_id%253Dinfo%253Apmid%252F14739702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:BEGE.0000009481.98336.fc&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14739702&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187427100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.21" data-doi="10.1186/1471-2164-11-351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nahashon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feaster</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohannon-Stewart</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adafope</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An initial map of chromosomal segmental copy number variations in the chicken.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D11%26rft.spage%253D351%26rft.epage%253D351%26rft.atitle%253DAn%2Binitial%2Bmap%2Bof%2Bchromosomal%2Bsegmental%2Bcopy%2Bnumber%2Bvariations%2Bin%2Bthe%2Bchicken.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-351%26rft_id%253Dinfo%253Apmid%252F20525236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525236&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.8.1253.22" data-doi="10.1371/journal.pgen.1000512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boije</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bed’hom</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourichon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variation in intron 1 of SOX5 causes the Pea-comb phenotype in chickens.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWright%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000512%26rft.epage%253De1000512%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bintron%2B1%2Bof%2BSOX5%2Bcauses%2Bthe%2BPea-comb%2Bphenotype%2Bin%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000512%26rft_id%253Dinfo%253Apmid%252F19521496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19521496&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F8%2F1253.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/8/1253.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/8/1241" title="Animal Models and Integrated Nested Laplace Approximations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/8/1261" title="Recombination-Induced Tag Exchange (RITE) Cassette Series to Monitor Protein Dynamics in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/8"><span>Volume 3 Issue 8, August 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/8.cover-source.jpg?itok=Dmhq6EsG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/8/1253.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mZ0ZZszeXf_t55pGKu1bNPF0MdFK3EaLE5g3BFxoqtQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/8/1253" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/8/1253" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/8/1253" class="active" data-icon-position="" data-hide-link-title="0">Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459512" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2JRbipc0Xe8IbduB3sHLT_jXR6x3HQJ84e4RaIFc3fI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/8/1253.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/8/1253" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-PEATyEcomB0TrO3RbTWkTWhaVWDPIh1OsrnmEtnnDiw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/8/1253" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/8/1253.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459512" id="node459512--2" data-pisa="ggg;3/8/1253" data-pisa-master="ggg;g3.113.006643" data-apath="/ggg/3/8/1253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lucy Crooks</span>, <span class="highwire-citation-author" data-delta="1">Örjan Carlborg</span>, <span class="highwire-citation-author" data-delta="2">Stefan Marklund</span> and <span class="highwire-citation-author" data-delta="3">Anna M. Johansson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1253-1260;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006643 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459512--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lucy Crooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, Uppsala, SE-75124, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BCrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crooks%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucy%2BCrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stefan Marklund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BMarklund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marklund%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BMarklund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anna M. Johansson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Uppsala, SE-75007, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BM.%2BJohansson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johansson%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BM.%2BJohansson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.johansson@slu.se" class="" data-icon-position="" data-hide-link-title="0">anna.johansson@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/8/1253" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/8/1253&amp;title=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/8/1253&amp;title=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/8/1253&amp;title=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/8/1253&amp;text=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/8/1253&amp;title=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/8/1253&amp;t=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/8/1253&amp;title=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/8/1253&amp;title=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459512/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459512" id="node459512--3" data-pisa="ggg;3/8/1253" data-pisa-master="ggg;g3.113.006643" data-apath="/ggg/3/8/1253.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Null Alleles and Deletions from SNP Genotypes for an Intercross Between Domestic and Wild Chickens</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lucy Crooks</span>, <span class="highwire-citation-author" data-delta="1">Örjan Carlborg</span>, <span class="highwire-citation-author" data-delta="2">Stefan Marklund</span> and <span class="highwire-citation-author" data-delta="3">Anna M. Johansson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1253-1260;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006643 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459512--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lucy Crooks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Cell and Molecular Biology, Uppsala University, Uppsala, SE-75124, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BCrooks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crooks%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucy%2BCrooks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Örjan Carlborg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C3%96rjan%2BCarlborg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlborg%20%C3%96&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3A%25C3%2596rjan%2BCarlborg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stefan Marklund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Clinical Sciences, Swedish University of Agricultural Sciences, Uppsala, SE-756 51, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BMarklund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marklund%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BMarklund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anna M. Johansson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Uppsala, SE-75007, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BM.%2BJohansson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johansson%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BM.%2BJohansson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anna.johansson@slu.se" class="" data-icon-position="" data-hide-link-title="0">anna.johansson@slu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1253&amp;rft.epage=1260&amp;rft.atitle=Identification%20of%20Null%20Alleles%20and%20Deletions%20from%20SNP%20Genotypes%20for%20an%20Intercross%20Between%20Domestic%20and%20Wild%20Chickens&amp;rft.volume=3&amp;rft.issue=8&amp;rft.date=2013-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Crooks%2C+Lucy&amp;rft.au=Carlborg%2C+%C3%96rjan&amp;rft.au=Marklund%2C+Stefan&amp;rft.au=Johansson%2C+Anna+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459512/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459512/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459512/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459512/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459512/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459512/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459512/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459512/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459512/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459512/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459512/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459512/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/8/1253" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/8/1253.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/8/1253.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/8/1253.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/8/1253.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
