<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/4/1183" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/4/1183.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/4/1183.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/4/1183.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/4/1183/embed/mml-math-3.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/4/1183.atom" />
<meta name="HW.pisa" content="ggg;8/4/1183" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.118.200053" />
<meta name="DC.Date" content="2018-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Barria et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Piscirickettsia salmonis is one of the main infectious diseases affecting coho salmon ( Oncorhynchus kisutch ) farming, and current treatments have been ineffective for the control of this disease. Genetic improvement for P. salmonis resistance has been proposed as a feasible alternative for the control of this infectious disease in farmed fish. Genotyping by sequencing (GBS) strategies allow genotyping of hundreds of individuals with thousands of single nucleotide polymorphisms (SNPs), which can be used to perform genome wide association studies (GWAS) and predict genetic values using genome-wide information. We used double-digest restriction-site associated DNA (ddRAD) sequencing to dissect the genetic architecture of resistance against P. salmonis in a farmed coho salmon population and to identify molecular markers associated with the trait. We also evaluated genomic selection (GS) models in order to determine the potential to accelerate the genetic improvement of this trait by means of using genome-wide molecular information. A total of 764 individuals from 33 full-sib families (17 highly resistant and 16 highly susceptible) were experimentally challenged against P. salmonis and their genotypes were assayed using ddRAD sequencing. A total of 9,389 SNPs markers were identified in the population. These markers were used to test genomic selection models and compare different GWAS methodologies for resistance measured as day of death (DD) and binary survival (BIN). Genomic selection models showed higher accuracies than the traditional pedigree-based best linear unbiased prediction (PBLUP) method, for both DD and BIN. The models showed an improvement of up to 95% and 155% respectively over PBLUP. One SNP related with B-cell development was identified as a potential functional candidate associated with resistance to P. salmonis defined as DD." />
<meta name="DC.Contributor" content="Agustín Barría" />
<meta name="DC.Contributor" content="Kris A. Christensen" />
<meta name="DC.Contributor" content="Grazyella M. Yoshida" />
<meta name="DC.Contributor" content="Katharina Correa" />
<meta name="DC.Contributor" content="Ana Jedlicki" />
<meta name="DC.Contributor" content="Jean P. Lhorente" />
<meta name="DC.Contributor" content="William S. Davidson" />
<meta name="DC.Contributor" content="José M. Yáñez" />
<meta name="article:published_time" content="2018-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Piscirickettsia salmonis&lt;/i&gt; is one of the main infectious diseases affecting coho salmon (&lt;i&gt;Oncorhynchus kisutch&lt;/i&gt;) farming, and current treatments have been ineffective for the control of this disease. Genetic improvement for &lt;i&gt;P. salmonis&lt;/i&gt; resistance has been proposed as a feasible alternative for the control of this infectious disease in farmed fish. Genotyping by sequencing (GBS) strategies allow genotyping of hundreds of individuals with thousands of single nucleotide polymorphisms (SNPs), which can be used to perform genome wide association studies (GWAS) and predict genetic values using genome-wide information. We used double-digest restriction-site associated DNA (ddRAD) sequencing to dissect the genetic architecture of resistance against &lt;i&gt;P. salmonis&lt;/i&gt; in a farmed coho salmon population and to identify molecular markers associated with the trait. We also evaluated genomic selection (GS) models in order to determine the potential to accelerate the genetic improvement of this trait by means of using genome-wide molecular information. A total of 764 individuals from 33 full-sib families (17 highly resistant and 16 highly susceptible) were experimentally challenged against &lt;i&gt;P. salmonis&lt;/i&gt; and their genotypes were assayed using ddRAD sequencing. A total of 9,389 SNPs markers were identified in the population. These markers were used to test genomic selection models and compare different GWAS methodologies for resistance measured as day of death (DD) and binary survival (BIN). Genomic selection models showed higher accuracies than the traditional pedigree-based best linear unbiased prediction (PBLUP) method, for both DD and BIN. The models showed an improvement of up to 95% and 155% respectively over PBLUP. One SNP related with B-cell development was identified as a potential functional candidate associated with resistance to &lt;i&gt;P. salmonis&lt;/i&gt; defined as DD.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/04/01" />
<meta name="citation_mjid" content="ggg;8/4/1183" />
<meta name="citation_id" content="8/4/1183" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/4/1183" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/4/1183.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/4/1183.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/4/1183.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200053" />
<meta name="citation_pmid" content="29440129" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1183" />
<meta name="citation_lastpage" content="1194" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Agustín Barría" />
<meta name="citation_author_institution" content="Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile" />
<meta name="citation_author_institution" content="Doctorado en Acuicultura, Programa Cooperativo Universidad de Chile, Universidad Católica del Norte, Pontificia Universidad Católica de Valparaíso, 8820808 Chile" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2813-4559" />
<meta name="citation_author" content="Kris A. Christensen" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0021-0326" />
<meta name="citation_author" content="Grazyella M. Yoshida" />
<meta name="citation_author_institution" content="Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile" />
<meta name="citation_author_institution" content="Animal Science Department, Universidade Estadual Paulista “Júlio de Mesquita Filho”, Faculdade de Ciências Agrárias e Veterinárias, Campus Jaboticabal, Jaboticabal 14884-900, Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6788-7369" />
<meta name="citation_author" content="Katharina Correa" />
<meta name="citation_author_institution" content="Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile" />
<meta name="citation_author_institution" content="Aquainnovo S.A., Puerto Montt 5503032, Chile" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1386-8522" />
<meta name="citation_author" content="Ana Jedlicki" />
<meta name="citation_author_institution" content="Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0626-1247" />
<meta name="citation_author" content="Jean P. Lhorente" />
<meta name="citation_author_institution" content="Aquainnovo S.A., Puerto Montt 5503032, Chile" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9157-4231" />
<meta name="citation_author" content="William S. Davidson" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3002-3225" />
<meta name="citation_author" content="José M. Yáñez" />
<meta name="citation_author_institution" content="Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile" />
<meta name="citation_author_institution" content="Aquainnovo S.A., Puerto Montt 5503032, Chile" />
<meta name="citation_author_institution" content="Núcleo Milenio INVASAL, Concepción 4070386, Chile" />
<meta name="citation_author_email" content="jmayanez@uchile.cl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6612-4087" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=Y. Aiba;citation_author=M. Kameyama;citation_author=T. Yamazaki;citation_author=T. F. Tedder;citation_author=T. Kurosaki;citation_title=Regulation of B-cell development by BCAP and CD19 through their binding to phosphoinositide 3-kinase.;citation_volume=111;citation_year=2008;citation_issue=3;citation_pmid=18025150;citation_doi=10.1182/blood-2007-08-109769" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Bangera;citation_author=K. Correa;citation_author=J. P. Lhorente;citation_author=R. Figueroa;citation_author=J. M. Yáñez;citation_title=Genomic predictions can accelerate selection for resistance against Piscirickettsia salmonis in Atlantic salmon (Salmo salar).;citation_pages=121;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s12864-017-3487-y" />
<meta name="citation_reference" content="citation_author=M. S. O. Brieuc;citation_author=C. D. Waters;citation_author=J. E. Seeb;citation_year=2014" />
<meta name="citation_reference" content="citation_author=N. R. Campbell;citation_author=S. E. LaPatra;citation_author=K. Overturf;citation_author=R. Towner;citation_author=S. R. Narum;citation_year=2014" />
<meta name="citation_reference" content="citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_year=2011;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: An analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_issue=11;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. Correa;citation_author=R. Bangera;citation_author=R. Figueroa;citation_author=J. P. Lhorente;citation_author=J. M. Yáñez;citation_title=The use of genomic information increases the accuracy of breeding value predictions for sea louse (Caligus rogercresseyi) resistance in Atlantic salmon (Salmo salar).;citation_pages=15;citation_volume=49;citation_year=2017;citation_issue=1;citation_doi=10.1186/s12711-017-0291-8" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Res.;citation_author=K. Correa;citation_author=M. Filp;citation_author=D. Cisterna;citation_author=M. E. Cabrejos;citation_author=C. Gallardo-Escárate;citation_title=Effect of triploidy in the expression of immune-related genes in coho salmon Oncorhynchus kisutch (Walbaum) infected with Piscirickettsia salmonis.;citation_pages=59-63;citation_volume=46;citation_year=2015;citation_doi=10.1111/are.12584" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. Correa;citation_author=J. Lhorente;citation_author=M. Lopez;citation_author=L. Bassini;citation_author=S. Naswa;citation_title=Genome-wide association analysis reveals loci associated with resistance against Piscirickettsia salmonis in two Atlantic salmon (Salmo salar L.) chromosomes.;citation_pages=854;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26499328;citation_doi=10.1186/s12864-015-2038-7" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Dis.;citation_author=J. Cvitanich;citation_author=O. Garate;citation_author=C. E. Smith;citation_title=The isolation of a rickettsia‐like organism causing disease and mortality in Chilean salmonids and its confirmation by Koch’s postulate.;citation_pages=121-145;citation_volume=14;citation_year=1991;citation_issue=2;citation_doi=10.1111/j.1365-2761.1991.tb00584.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_issue=3;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. W. Davey;citation_author=T. Cezard;citation_author=P. Fuentes-Utrilla;citation_author=C. Eland;citation_author=K. Gharbi;citation_title=Special features of RAD Sequencing data: Implications for genotyping.;citation_pages=3151-3164;citation_volume=22;citation_year=2013;citation_issue=11;citation_doi=10.1111/mec.12084" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Dou;citation_author=X. Li;citation_author=Q. Fu;citation_author=W. Jiao;citation_author=Y. Li;citation_title=Evaluation of the 2b-RAD method for genomic selection in scallop breeding.;citation_pages=19244;citation_volume=6;citation_year=2016;citation_issue=1;citation_pmid=26754638;citation_doi=10.1038/srep19244" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=P. Dufflocq;citation_author=J. P. Lhorente;citation_author=R. Bangera;citation_author=R. Neira;citation_author=S. Newman;citation_title=Correlated response of flesh color to selection for harvest weight in coho salmon (Oncorhynchus kisutch).;citation_pages=6-11;citation_volume=472;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics.;citation_author=D.S. Falconer;citation_author=Mackay. T.F.C.;citation_year=1996" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_title=Bayesian Methods Applied to GWAS.;citation_author=R.L. Fernando;citation_author=D. Garrick;citation_volume=1019;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Acta Agric. Scand. Sect. Anim. Sci.;citation_author=R. L. Fernando;citation_author=D. Habier;citation_author=C. Stricker;citation_author=J. C. M. Dekkers;citation_author=L. R. Totir;citation_title=Genomic selection.;citation_pages=192-195;citation_volume=57;citation_year=2007;citation_doi=10.1080/09064700801959395" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=H. M. Gjøen;citation_author=T. Refstie;citation_author=O. Ulla;citation_author=B. Gjerde;citation_title=Genetic correlations between survival of Atlantic salmon in challenge and field tests.;citation_pages=277-288;citation_volume=158;citation_year=1997;citation_issue=3-4;citation_doi=10.1016/S0044-8486(97)00203-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_author=T. Meuwissen;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Genome-wide association and genomic selection in animal breeding.;citation_pages=876-883;citation_volume=53;citation_year=2010;citation_issue=11;citation_pmid=21076503;citation_doi=10.1139/G10-076" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Immunology;citation_journal_abbrev=Nat Rev Immunol;citation_author=S. Herzog;citation_author=M. Reth;citation_author=H. Jumaa;citation_title=Regulation of B-cell proliferation and differentiation by pre-B-cell receptor signalling.;citation_pages=195-205;citation_volume=9;citation_year=2009;citation_issue=3;citation_pmid=19240758;citation_doi=10.1038/nri2491" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. D. Houston;citation_author=J. W. Davey;citation_author=S. C. Bishop;citation_author=N. R. Lowe;citation_author=J. C. Mota-Velasco;citation_title=Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.;citation_pages=244-244;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22702806;citation_doi=10.1186/1471-2164-13-244" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. D. Houston;citation_author=J. B. Taggart;citation_author=T. Cézard;citation_author=M. Bekaert;citation_author=N. R. Lowe;citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).;citation_pages=90;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=24524230;citation_doi=10.1186/1471-2164-15-90" />
<meta name="citation_reference" content="citation_title=Immunobiology 5, The Immune System in Health and Disease;citation_title=B-cell activation by armed helper T cells.;citation_author=C. Janeway;citation_author=P. Travers;citation_author=M. Walport;citation_author=M. Shlomchik;citation_pages=343-360;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=A. Legarra;citation_author=O. F. Christensen;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Single Step, a general approach for genomic selection.;citation_pages=54-65;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.04.029" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Nature immunology;citation_journal_abbrev=Nat Immunol;citation_author=J. Li;citation_author=D. R. Barreda;citation_author=Y.-A. Zhang;citation_author=H. Boshra;citation_author=A. E. Gelman;citation_title=B lymphocytes from early vertebrates have potent phagocytic and microbicidal abilities.;citation_pages=1116-1124;citation_volume=7;citation_year=2006;citation_issue=10;citation_pmid=16980980;citation_doi=10.1038/ni1389" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Lillehammer;citation_author=T. H. E. Meuwissen;citation_author=A. K. Sonesson;citation_title=A low-marker density implementation of genomic selection in aquaculture using within-family genomic breeding values.;citation_pages=39;citation_volume=45;citation_year=2013;citation_issue=1;citation_doi=10.1186/1297-9686-45-39" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=S. Liu;citation_author=R. L. Vallejo;citation_author=Y. Palti;citation_author=G. Gao;citation_author=D. P. Marancik;citation_title=Identification of single nucleotide polymorphism markers associated with bacterial cold water disease resistance and spleen size in rainbow trout.;citation_pages=1-10;citation_volume=6;citation_year=2015;citation_pmid=25674101;citation_doi=10.3389/fgene.2015.00001" />
<meta name="citation_reference" content="citation_title=Genetics and analysis of quantiative traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. J. Macqueen;citation_author=C. R. Primmer;citation_author=R. D. Houston;citation_author=B. F. Nowak;citation_author=L. Bernatchez;citation_title=Functional Annotation of All Salmonid Genomes (FAASG): an international initiative supporting future salmonid research, conservation and aquaculture.;citation_pages=484;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3862-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="Misztal, I., S. Tsuruta, D. Lourenco, Y. Masuda, I. Aguilar et al., 2016 Manual for BLUPF90 Family of Programs. University of Georgia, Athens, GA." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Moen;citation_author=J. Torgersen;citation_author=N. Santi;citation_author=W. S. Davidson;citation_author=M. Baranski;citation_title=Epithelial cadherin determines resistance to infectious pancreatic necrosis virus in Atlantic salmon.;citation_volume=200;citation_year=2015;citation_pmid=26041276;citation_doi=10.1534/genetics.115.175406" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=H. M. Nielsen;citation_author=A. K. Sonesson;citation_author=H. Yazdi;citation_author=T. H. E. Meuwissen;citation_title=Comparison of accuracy of genome-wide and BLUP breeding value estimates in sib based aquaculture breeding schemes.;citation_pages=259-264;citation_volume=289;citation_year=2009;citation_issue=3-4;citation_doi=10.1016/j.aquaculture.2009.01.027" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Res.;citation_author=J. Ødegård;citation_author=M. Baranski;citation_author=B. Gjerde;citation_author=T. Gjedrem;citation_title=Methodology for genetic evaluation of disease resistance in aquaculture species: Challenges and future prospects.;citation_pages=103-114;citation_volume=42;citation_year=2011;citation_doi=10.1111/j.1365-2109.2010.02669.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. Ødegård;citation_author=T. Moen;citation_author=N. Santi;citation_author=S. A. Korsvoll;citation_author=S. Kjøglum;citation_title=Genomic prediction in an admixed population of Atlantic salmon (Salmo salar).;citation_pages=1-8;citation_volume=5;citation_year=2014;citation_pmid=24567736;citation_doi=10.3389/fgene.2014.00001" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. Ødegård;citation_author=I. Olesen;citation_author=B. Gjerde;citation_author=G. Klemetsdal;citation_title=Evaluation of statistical models for genetic analysis of challenge test data on furunculosis resistance in Atlantic salmon (Salmo salar): Prediction of field survival.;citation_pages=116-123;citation_volume=259;citation_year=2006;citation_issue=1-4;citation_doi=10.1016/j.aquaculture.2006.05.034" />
<meta name="citation_reference" content="citation_journal_title=Fish &amp; shellfish immunology;citation_journal_abbrev=Fish Shellfish Immunol;citation_author=H. S. Øverland;citation_author=E. F. Pettersen;citation_author=A. Rønneseth;citation_author=H. I. Wergeland;citation_title=Phagocytosis by B-cells and neutrophils in Atlantic salmon (Salmo salar L.) and Atlantic cod (Gadus morhua L.).;citation_pages=193-204;citation_volume=28;citation_year=2010;citation_issue=1;citation_pmid=19874896;citation_doi=10.1016/j.fsi.2009.10.021" />
<meta name="citation_reference" content="citation_author=C. Palaiokostas;citation_author=S. Ferarreso;citation_author=R. Franch;citation_author=R. D. Houston;citation_author=L. Bargelloni;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Y. Palti;citation_author=G. Gao;citation_author=S. Liu;citation_author=M. P. Kent;citation_author=S. Lien;citation_title=The development and characterization of a 57K single nucleotide polymorphism array for rainbow trout.;citation_pages=662-672;citation_volume=15;citation_year=2015;citation_issue=3;citation_pmid=25294387;citation_doi=10.1111/1755-0998.12337" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Palti;citation_author=R. L. Vallejo;citation_author=G. Gao;citation_author=S. Liu;citation_author=A. G. Hernandez;citation_title=Detection and validation of QTL affecting bacterial cold water disease resistance in rainbow trout using restriction-site associated dna sequencing.;citation_pages=e0138435;citation_volume=10;citation_year=2015;citation_issue=9;citation_doi=10.1371/journal.pone.0138435" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Pulgar;citation_author=C. Hödar;citation_author=D. Travisany;citation_author=A. Zuñiga;citation_author=C. Domínguez;citation_title=Transcriptional response of Atlantic salmon families to Piscirickettsia salmonis infection highlights the relevance of the iron-deprivation defence system.;citation_pages=495;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26141111;citation_doi=10.1186/s12864-015-1716-9" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=M. L. Rise;citation_author=S. R. M. Jones;citation_author=G. D. Brown;citation_author=K. R. von Schalburg;citation_author=W. S. Davidson;citation_title=Microarray analyses identify molecular biomarkers of Atlantic salmon macrophage and hematopoietic kidney response to Piscirickettsia salmonis infection.;citation_volume=20;citation_year=2004;citation_issue=1;citation_pmid=15454580;citation_doi=10.1152/physiolgenomics.00036.2004" />
<meta name="citation_reference" content="citation_author=D. Robledo;citation_author=C. Palaiokostas;citation_author=L. Bargelloni;citation_author=P. Martínez;citation_author=R. Houston;citation_year=2017;citation_doi=10.1111/raq.12193" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_issue=4;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Dis.;citation_author=M. Rozas;citation_author=R. Enríquez;citation_title=Piscirickettsiosis and Piscirickettsia salmonis in fish: a review.;citation_pages=163-188;citation_volume=37;citation_year=2014;citation_issue=3;citation_pmid=24279295;citation_doi=10.1111/jfd.12211" />
<meta name="citation_reference" content="Sernapesca, 2016 Informe Sanitario de Salmonicultura en Centros Marinos 2016. ([Online]. Valparaiso, Chile. Updated 14 August 2017. http://www.sernapesca.cl/index.php?option=com_remository&amp;Itemid=246&amp;func=fileinfo&amp;id=27560)." />
<meta name="citation_reference" content="citation_journal_title=Research in veterinary science;citation_journal_abbrev=Res Vet Sci;citation_author=M. J. Stear;citation_author=S. C. Bishop;citation_author=B. A. Mallard;citation_author=H. Raadsma;citation_title=The sustainability, feasibility and desirability of breeding livestock for disease resistance.;citation_pages=1-7;citation_volume=71;citation_year=2001;citation_issue=1;citation_pmid=11666141;citation_doi=10.1053/rvsc.2001.0496" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=J. E. Bron;citation_title=Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.;citation_pages=47;citation_volume=48;citation_year=2016;citation_issue=1;citation_doi=10.1186/s12711-016-0226-9" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=R. L. Vallejo;citation_author=T. D. Leeds;citation_author=B. O. Fragomeni;citation_author=G. Gao;citation_author=A. G. Hernandez;citation_title=Evaluation of genome-enabled selection for bacterial cold water disease resistance using progeny performance data in rainbow trout: insights on genotyping methods and genomic prediction models.;citation_pages=1-13;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00096" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. L. Vallejo;citation_author=T. D. Leeds;citation_author=G. Gao;citation_author=J. E. Parsons;citation_author=K. E. Martin;citation_title=Genomic selection models double the accuracy of predicted breeding values for bacterial cold water disease resistance compared to a traditional pedigree-based model in rainbow trout aquaculture.;citation_pages=17;citation_volume=49;citation_year=2017;citation_issue=1;citation_doi=10.1186/s12711-017-0293-6" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=R. L. Vallejo;citation_author=S. Liu;citation_author=G. Gao;citation_author=B. O. Fragomeni;citation_author=A. G. Hernandez;citation_title=Similar genetic architecture with shared and unique quantitative trait loci for bacterial cold water disease resistance in two rainbow trout breeding populations.;citation_pages=156;citation_volume=8;citation_year=2017;citation_doi=10.3389/fgene.2017.00156" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=H. Wang;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=A. Legarra;citation_author=R. L. Fernando;citation_title=Genome-wide association mapping including phenotypes from relatives without genotypes in a single-step (ssGWAS) for 6-week body weight in broiler chickens.;citation_pages=1-10;citation_volume=5;citation_year=2014;citation_pmid=24567736;citation_doi=10.3389/fgene.2014.00001" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=H. Wang;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=A. Legarra;citation_author=W. M. Muir;citation_title=Genome-wide association mapping including phenotypes from relatives without genotypes.;citation_pages=73-83;citation_volume=94;citation_year=2012;citation_issue=02;citation_pmid=22624567;citation_doi=10.1017/S0016672312000274" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. M. Yáñez;citation_author=R. Bangera;citation_author=J. P. Lhorente;citation_author=A. Barria;citation_author=M. Oyarzun;citation_title=Negative genetic correlation between resistance against Piscirickettsia salmonis and harvest weight in coho salmon (Oncorhynchus kisutch).;citation_pages=8-13;citation_volume=459;citation_year=2016;citation_doi=10.1016/j.aquaculture.2016.03.020" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. M. Yáñez;citation_author=S. Naswa;citation_author=M. E. Lopez;citation_author=L. Bassini;citation_author=K. Correa;citation_title=Genomewide single nucleotide polymorphism discovery in Atlantic salmon (Salmo salar): validation in wild and farmed American and European populations.;citation_pages=1002-1011;citation_volume=16;citation_year=2016;citation_issue=4;citation_doi=10.1111/1755-0998.12503" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. M. Yáñez;citation_author=R. Bangera;citation_author=J. P. Lhorente;citation_author=M. Oyarzún;citation_author=R. Neira;citation_title=Quantitative genetic variation of resistance against Piscirickettsia salmonis in Atlantic salmon (Salmo salar).;citation_pages=155-159;citation_volume=414–415;citation_year=2013;citation_doi=10.1016/j.aquaculture.2013.08.009" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. M. Yáñez;citation_author=L. N. Bassini;citation_author=M. Filp;citation_author=J. P. Lhorente;citation_author=R. W. Ponzoni;citation_title=Inbreeding and effective population size in a coho salmon (Oncorhynchus kisutch) breeding nucleus in Chile.;citation_pages=S15-S19;citation_volume=420–421;citation_year=2014;citation_doi=10.1016/j.aquaculture.2013.05.028" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. M. Yáñez;citation_author=R. D. Houston;citation_author=S. Newman;citation_title=Genetics and genomics of disease resistance in salmonid species.;citation_pages=1-13;citation_volume=5;citation_year=2014;citation_pmid=24567736;citation_doi=10.3389/fgene.2014.00001" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. M. Yáñez;citation_author=J. P. Lhorente;citation_author=L. N. Bassini;citation_author=M. Oyarzún;citation_author=R. Neira;citation_title=Genetic co-variation between resistance against both Caligus rogercresseyi and Piscirickettsia salmonis, and body weight in Atlantic salmon (Salmo salar).;citation_pages=295-298;citation_volume=433;citation_year=2014;citation_doi=10.1016/j.aquaculture.2014.06.026" />
<meta name="citation_reference" content="citation_journal_title=G3 Genes Genomes Genet;citation_author=G.M. Yoshida;citation_author=R. Bangera;citation_author=R. Carvalheiro;citation_author=K. Correa;citation_author=R. Figueroa;citation_author=J. P. Lhorente;citation_author=J. M. Yáñez;citation_title=Genomic prediction accuracy for resistance against Piscirickettsia salmonis in farmed rainbow trout.;citation_pages=719-726;citation_volume=8;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="twitter:description" content="Piscirickettsia salmonis is one of the main infectious diseases affecting coho salmon ( Oncorhynchus kisutch ) farming, and current treatments have been ineffective for the control of this disease. Genetic improvement for P. salmonis resistance has been proposed as a feasible alternative for the control of this infectious disease in farmed fish. Genotyping by sequencing (GBS) strategies allow genotyping of hundreds of individuals with thousands of single nucleotide polymorphisms (SNPs), which can be used to perform genome wide association studies (GWAS) and predict genetic values using genome-wide information. We used double-digest restriction-site associated DNA (ddRAD) sequencing to dissect the genetic architecture of resistance against P. salmonis in a farmed coho salmon population and to identify molecular markers associated with the trait. We also evaluated genomic selection (GS) models in order to determine the potential to accelerate the genetic improvement of this trait by means of using genome-wide molecular information. A total of 764 individuals from 33 full-sib families (17 highly resistant and 16 highly susceptible) were experimentally challenged against P. salmonis and their genotypes were assayed using ddRAD sequencing. A total of 9,389 SNPs markers were identified in the population. These markers were used to test genomic selection models and compare different GWAS methodologies for resistance measured as day of death (DD) and binary survival (BIN). Genomic selection models showed higher accuracies than the traditional pedigree-based best linear unbiased prediction (PBLUP) method, for both DD and BIN. The models showed an improvement of up to 95% and 155% respectively over PBLUP. One SNP related with B-cell development was identified as a potential functional candidate associated with resistance to P. salmonis defined as DD." />
<meta name="og-title" property="og:title" content="Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/4/1183" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Piscirickettsia salmonis is one of the main infectious diseases affecting coho salmon ( Oncorhynchus kisutch ) farming, and current treatments have been ineffective for the control of this disease. Genetic improvement for P. salmonis resistance has been proposed as a feasible alternative for the control of this infectious disease in farmed fish. Genotyping by sequencing (GBS) strategies allow genotyping of hundreds of individuals with thousands of single nucleotide polymorphisms (SNPs), which can be used to perform genome wide association studies (GWAS) and predict genetic values using genome-wide information. We used double-digest restriction-site associated DNA (ddRAD) sequencing to dissect the genetic architecture of resistance against P. salmonis in a farmed coho salmon population and to identify molecular markers associated with the trait. We also evaluated genomic selection (GS) models in order to determine the potential to accelerate the genetic improvement of this trait by means of using genome-wide molecular information. A total of 764 individuals from 33 full-sib families (17 highly resistant and 16 highly susceptible) were experimentally challenged against P. salmonis and their genotypes were assayed using ddRAD sequencing. A total of 9,389 SNPs markers were identified in the population. These markers were used to test genomic selection models and compare different GWAS methodologies for resistance measured as day of death (DD) and binary survival (BIN). Genomic selection models showed higher accuracies than the traditional pedigree-based best linear unbiased prediction (PBLUP) method, for both DD and BIN. The models showed an improvement of up to 95% and 155% respectively over PBLUP. One SNP related with B-cell development was identified as a potential functional candidate associated with resistance to P. salmonis defined as DD." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
    <title>Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pfWowgjDVdUl_aeM-9CIcHKWx5OGycDHq4yi54bD9Ps","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489136","apath":"\/ggg\/8\/4\/1183.atom","pisa":"ggg;8\/4\/1183","ac":{"\/ggg\/8\/4\/1183.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/4\/1183.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1183"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1183"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/4\/1183":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022X89JvOdI0aEc3gDGwqYT\\\\\\\/ICTpAmzWhyIHom3h96S0N2juNQyibpqs9AT149o7vD5x+JwJ9b85l7kGmthW5HbAwZkicIpuSsyFKOhs8pQkN+rRbkRn1C+HwKSLtUqYpvKc0NOHDNT5CSjSJmrTHFIvHXGs5jlFcQRqeT7W3J99YXCrbZHfkzIxGo+VNbr9LpJCVn1oAsoFumlmVKtTfQ4P+mtvk670vXQr9X2cGEzk5F3yXhr607S4qISdyIFMVT8qQHLQssdgu\\\\\\\/QSMylr0NGHBE9t8MxxIdbZtCze5UWUyA9vFYwW7keC6pLhuV4j7cId\\\\\\\/wZrnGPlSJiAqLsDTY\\\\\\\/R1BpzAzHQVOJK1AgXUzec\\\\\\\/aUONshqBuHYhH+dMbgdX8lrNPv5l9dj3Jq3w8QOWivnFPZ8uo5gGl7wSD3kdC8xd2mxHryKVR2C60xvXJ4AS391jgAkyWU6GB9BH92W6xwVASj4GA3QJCGOuDeLV04cz1rJxxN7FcCFcNP31axX8zmDQ5jNMwDmqR7kGj1MtHiPi6SOrGfLExS\\\\\\\/\\\\\\\/UYIo5kELbBWgUeXgGu9NUuM2m1jE7X21qkQz2zdge4qgLUyd0s0KfbBymkY6G6r4sLrGT0eN8aWS0U8jJwFalcCzA1dUR28xLnywqNk0ceP6o5vjE7Y4yLdnpAvhso4Dc8kZ58eSfQ4rAbQFb3K+XEEqUoa7CmVBohsxgu7\\\\\\\/Quluhm2YljCTKGJsmTL+f1hTdd7fFaFg\\\\\\\/Sz+9dOUV1XkFvDPe2C1BaKDKkjaHLwzRM7tTEjivSQ4ZoV\\\\\\\/Pi89CUUmLsEX\\\\\\\/YMNLbP+G5ZbX0l\\\\\\\/390b11sOqleMZoj+HpEbWNstmS\\\\\\\/r3k8\\\\\\\/cybu9Wnu6G5ZkZNK1qlha6QuGloT\\\\\\\/IbFve6ahYy4292sLPUSv52qqj2RyDh63DGlSsnXTHQzvyL4sqJdahApFQndGqqsBBlpFkH37Kkwt4BTJ3+xzlIcp7y3bMEDblPxpSS8vg1FEW\\\\\\\/OjMXS\\\\\\\/Ono4aRNM2lGaYiA0yO+nTEUKBoBq+uvENimHITfJqohLs3BnTNdwhY5nCENc6W8C6GHTt2zUx573\\\\\\\/KacdBccgy27LupYMy9ZlNRWlzERwAEfx+92krwljuHRpQHdjm3+11jzQtrY\\\\\\\/xUuq+8U2ZKMjeqRuSJuxgZ3ofmwVc8JlchgSzvAbWXV\\\\\\\/UjxXlwi62WMl\\\\\\\/kHK45P3JBr5tf8pkuPxChesushcNVB\\\\\\\/HuODu5qbJqsLdyRMG\\\\\\\/toBXHnZAhVpBYh64GvdUzW60qQd7B9oZuIpFUF\\\u0022,\\\u0022iv\\\u0022:\\\u0022uJ0JauVNI\\\\\\\/YtgP79xrSYTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022094fd0212f4e44fa621a8ea1f8d772176aecd939616d0c3125089e462cd359fb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bvR9uUX7teIc58pVUDEYVsbb5jYziZz2ub5gfeOIG0OHdvYhLzlzs7nOqXK5qyY78gYM9fMmwrtp+RuogLxUXfDTlGV\\\\\\\/zgSYYiMMA59\\\\\\\/gdaL12Rk+eNrnk9j03kqCte\\\\\\\/eu4mPzcT3oo3K2rnLjJfEuF0pm2belwvTYkf8DlxQC7dh++YnbuSVeW+6VaeXqlu+C9MS\\\\\\\/7da+6TzFIUXii+MLkdFpIW7gX9HnUNrMuHQ4HHdI9F46iVdRjsfZFhtp86E\\\\\\\/LCM8t9aesxKP69kGkvvVF+3az4NRFLaeXLM9sjToT7k+ZFvv6j0uhE0ILhXWLjWnxs\\\\\\\/cbpFm+GHAXtxiEF8c4I7Np+QsKOrt2LdqhOQfnSlOymS\\\\\\\/kCqsjVaMeZvejtgURoqF4aBu3O331\\\\\\\/fEDJ6D3JgBO+GDhl\\\\\\\/7jD4y4rCt2tQI5TRDFYZbT4mFqHgt9vlPej7vRu++kfoYk1P+SFD8OuRtD0V2dNemn\\\\\\\/mpM52RfR24ArIrk4jsDlFFTkn0BQnSHvlyzWwxcXj8unAYoI9HyODyTvCQ9ucuSmy72bYM96qHYIv4sMQq\\\\\\\/zIRNwSjIlnkSDww52gmQtosILPLN0paFSFKJBHfTNkJVmomqX4yvjAkysc7JF9Ytf1t\\\\\\\/yjx4a0btckInYbXX4HkCThmK5q0qcvxPEiT7doWJzc8AjlHQWdUQphIVk8J31\\\\\\\/SaGXL+LXvS+d58xzHht7HRqyqqZeS+v64lcPui9ZF1HMa7T0e+jIu9ACk7JibujKAfp02WKxksVchHMn+m62s5VRdQFYNzdhzLnApwHpxR9Zs9WmyFNKwLr223bELGidOgoy7VFS9VTyiIObROyvpDDNRnz1fUN1N+oyC+Fm6U6qS+\\\\\\\/1mB7G2DuFY1erpIlkDCvusDcGD2YSYCL\\\\\\\/dA8yLevsANlj8x5pVpVvyrWDScFdp99RnmbFV9IgedpRoIq0d\\\\\\\/TlF\\\\\\\/JOuOyyUihI3ZFrotgqVJKt9zX\\\\\\\/\\\\\\\/iw7JXIYjcGHCNyfWEy3rBiucc66E2gE8NRJ3iAnL562Y3O8ACL94nj6Fi7z7MFjyop2eiirjzJAXXhCY+eRp+6THSS4yvFc7HqKXpaLTRvOVggul67H0VLGTwYqi9MI7PgAFGVL3DKP5CumJBiXtHsuPNZ0lVf7BJn6eohz6B8i577B7h\\\\\\\/ha4iYKKQI9HTLEwVGFHLTYVgoNnSN4WMB2RjLXMNGkn7TDp1fqlRlPx+u8QN9LZqexp9vy+1+nb30mngoYyuf8NWeUexpPJA\\\\\\\/lA4SCSQSQeGNOzVnoogIQyPUPbs1vduFS\\\\\\\/ymXL6Pv4RJHtoXh88z1VAhSpBQ\\\\\\\/iu39bYYCXAIIghXhFnI1pDt6AlHT4q1cEIgLnhnxDZw4GQQ87U9eWccJ0XQ898b9HuZwG\\\\\\\/UL6Q1MqHrBOy3uUC5CeHzvf4WTPGgoADvD1UBINYnn65TqUbBY1+l+aYz7FqCBFZpLsbrO2TACQkDzo3e7gkVyj8sD0UkxtSW9+QIIKcCAYwbZvzXU0JrYmt+Zdejc18Iq26f\\\\\\\/t4ipzy1yBviIVBe8oqmdgrjzyv7J+tDwIC9U3ra8ReXN4frwxFC6yEkkDgfd2g8pnaNSrDib6y8DlBWT63z6B4WdUKD6+enf2MGk8YS6DMBiTIVR0G+VaYF999oTpsZOWWiK2t97qG2Ian4WzHUOY0uc3Trbv2LoczPS6Wzo9pgfJ2eLdC5PhyynBJAgeczdEaGOiVXWVDxlU6xQO0XjVroTg+Z8WuVRyGySOqna1ylj5sSrT4r\\\\\\\/1D6N+Iz4dzWrXW9c+pvmDXS8wj9+rLgTXWqHoRF8f+V\\\\\\\/svWaS64tTOWuCEMw6I8\\\\\\\/8panSvPWK5v\\\\\\\/erdRNr2Y3LtIQro9dihlxO+gQK1h26lXUVboc\\\\\\\/zjiAbFIkDOvOamoBe0JrUETCBVgZGhybxgQuPYnYCcp7Vp\\\\\\\/wcrgCv372aEl19g+TRtz7ai8UNYJCsvAU7bi7aWrl+b2zhQR0fcGw7ez2aLfZ7IulKfRsr81y4WdIk09+qXiJWpUB2EXWNkBwMjrWBbk5mmhBwQK1fVlNdWhtCbJjlOyyJ44p6MdHUhnIsGx6VusWrjIBdIuxqxMcgzgCUwzyuVXwpSsmX8D3mbT7xDN4uqsVICTfwBw6Ii082fghhq99lVSj\\\\\\\/3jLEuEbsbyrSUkWL+q\\\\\\\/70Pt99cxsvtxlo73bZtmn6qtREMN6+7wxG5rZvZyX71ZY0Fq3a\\\\\\\/zkh8n8XlJFtgldwC1NzFf2fShh1SW22bNSYJQoFEUpN4u\\\\\\\/vMfGMlwdgNrJHdHJSGNC3lXuuVQeldrCNsDC+f26CZgonvRfgpqtwFOG8lkpDYTE+rnIscvZ5hxwuGo2ltbSg6wm\\\\\\\/btNC\\\\\\\/4ygATC52swZaUNILIcwd5tBpOsaUjQ\\\\\\\/MO+0aedWsb5busQwEq\\\\\\\/32BT4QKg7oVNFvatJAbxv1G98vZP4EcRQh3Q6B+TiQ9RrHVMBccY1emPSJhyCgL05Sl5TCenWi\\\\\\\/YwvtbMvZAkmE1TbSbfdKhE2x4zWvGP+JTTvaq0tfrCb0EUfAAtx4NC2xf8J\\\\\\\/N52uhIiFNRNUuD7SgBEsaRljQabRLIVeyzF7osBGM8KxHbIlNJBbeTC6qsyl0gHmwWP\\\\\\\/U9vdixeUJgI8\\\\\\\/tGl1w9RF92EP6P9+0x3RTOUQ+BDIdPNk5IJEalAU0L0\\\\\\\/R7Q6IYhdZAUUEGzLrfgR\\\\\\\/Ksh3rte1\\\\\\\/hlQ\\\\\\\/relZzyJuPn1DpdBktim\\\\\\\/+o0\\\\\\\/YbKIrlNUG+2ZM\\\\\\\/\\\\\\\/90zLSAc5YFW1dV4wV93RPknNhdClFEvVlLOMPj2\\\\\\\/9BLWLWgd75JPjFouRTfc7Y7lJAeqZY43DUXVbQjCCoOGHO7RiJroVYU91fP7uPwqUTltiIA\\\\\\\/s4BnOGL82OIu7LuVyVCDCHBFE6JbUsZH8rjTUNYGKjBzxBjO0QH0FIEfRnEzypm1UH4rDG7mlJsgpGmw\\\\\\\/H8MVautX1Atzknxsve67PWCOC\\\\\\\/xq4MpKYVe36w7+3gzQqG9jaDdE\\\\\\\/St3IEJ5IMfXu5Qbb4DwVEvD1LIeTnks+cFdYUQSu4hj1SAnYuqZqXyaTRxZLgmniEAWqf8z+mkGHsELf83whC2WtQQKBwcfet+kmGx3EgcYY05ggyNmS7khNutmzg3DefpyrS6AVvxHaLC+30CIvA7E8oXv4cHSBTRIJaSeMhfxZlVC+MQXKDpTFqf2S2v3UIQQuI3FPrQilmLIJEhrWKCzK7XYimcuID8iqDrKBJXl9Fuobbtg7r7e9\\\\\\\/jissS4sGt3eaguPC7jrSGFqv+bsse+Z\\\\\\\/CMJSz3bXKh+7W1inI82fwkhy0DYrQfR1xZK5RNck+XFD9PYUFLWoQIIDDx3Mgq5jvJAu9mywlsKgFBTZwTc9rNJ5KaTZ0XUK0GJMif5cmNznLnG1DNRD\\\\\\\/rIQu6oR9KjJrl41EaYAAaagxk8sagyIa1AK\\\\\\\/+ARomu4SDlhTVWLutG8u8KFDriOb6IICllgnleAqSEGd94qYhQl+g4alM4Ef+0VCt+98iIZQb3LTEuNqFB\\\\\\\/awrcKFMU7iZ4W4Xi+Ebl5P0gpWVcQSIIbic3HLyJ2OsPZOIGb0gwO7zGybtydhxhLqntXBPp3Nhg9o+fxdBnGGaDkNjEE52KoaTVlhxBKBmeltX6tlzORVA8o78g2PCMAyuRI4Omq3PApkPlbJrQi44sk76ulWcdhGuZMs1k5NbFf2n3FOHuLmtH8lhngAbUtKd7tIA0JGhevHjxNQjDH9lbMig9opap\\\\\\\/5VxnOKyWfJAd2d33cEOnic\\\\\\\/JxAxb5MOyEyF1sb9ufEdhyQ+uhBxCe35BFgHro7Jomu5BmCQDpSXPhl18wtRgFwUkktvOo9MgLXIOzvoHozKpSDCuuCVn2Zzp0vT\\\\\\\/tzdNTdoX9DGMD8Ky8Y3+pQGLs0GzbYOuuL\\\\\\\/\\\\\\\/M6Lv7oP8ME8WKLkW3wmRC22JE2Ma+h54EMyolrDPC7rLcOi0zPrgqT38uDwK6k4GRe1el8UhF36vK8SSdlEwK\\\\\\\/pCQur6bNuaevXfnVbtrzC09YMvvtik3bwjOUpEKVVO1HRgoePrc10TZfm6+YhnfWI0ks3tDfAdMObPJ7ya+mu7d8Yn2vQWms3I6fx2l6p0igfZhGOK36GCx7XHkMBLeLgNgLheAu5J8CX17HEcdTtI+anOQQDQkqXVf4M1CsPK9QI5b5Oh1I5DWVSAAZQT5gExNQxg2ORDbC2ZXBMABMfhDP1xd8XBpe5zlcO00Zo5IUgk\\\\\\\/GFv02HXlk2h4pK9MAd\\\\\\\/EqgmPJF6EOfqDuJHTEKUMgv6RYJHu\\\\\\\/gwNntBNnU1UayDs1gWlPxDgdCde8eabbXC0\\\\\\\/syEnM3o2lFz5auS\\\\\\\/B2pDTp0YKlhGsppcXNcRSL1j6fDBSFSN\\\\\\\/yJmZWh3tm7YOktwUFtUA5jHGp+Q7qM74SojIYcSJ4jQBSUxqLqQB\\\\\\\/G9IpYYbYu6oJkDwzGPNPJf8547tUF5RSAlv7o6xgwPRcl+CquRsv97oLlx6DUXUncgFxdQl6luFJ5yxFGBFn1XDqo0kpTRoIyqns5RuYWOouWrhphzTQ30v\\\\\\\/HV0d7dv9AVsl8GjdX203hQHl+r9YLbe2hCr3ifNgVI+1L5Y+SLatKKHFJQYfYG9YyQBOjvfmMSCdqGHDSN9g8GL46w6Kx51AlFu7xU\\\\\\\/MSk1HZCQFxK\\\\\\\/OOTWQGvj0Zgx4mtYGJvnLWvFAji7WEu9dHj8pmES\\\\\\\/fDOWsPI7keV5RcKZzOsGPZeI2hEzKGH7r08fDQ8AyKDNRTN23GCrrK2E6sNl4F11Jv9Fu6rKUJNgt8L907LnphPvy\\\\\\\/bBfRHXrUQTy9OgVSsxS1sQzNBpldxEgwp0zk4E8HgCk4fc6vKIn+vlqmXEmk03fUx\\\\\\\/mizFnly07Bft91x8GnaFZsym5f\\\\\\\/S\\\\\\\/TsfFy3JcSI41AUZJU61AEAzrmkvjSAM2SA4dK7H0Ds+FmKaYUpQIxOt0SZuOKiEbL9j6zgUMiKra5Nbmj2PxfsuHsvhQhB0vwjGQffNj2NuN3naEhaYG\\\\\\\/YprJ\\\\\\\/sGENUnT0wyAWXQ4szqjD8RQOhLUaJaw6oSx0PXI1\\\\\\\/+8zY5HYHzrs7lphDmC1Itu\\\\\\\/afT2NaP7pbHMNkPsJFPsFbyYqc1At9lVUWgh2sY5ntfTo6UGK3eJXzZJfMzdKxjLT3c0UQWAkndR7l\\\\\\\/gND\\\\\\\/T5xhTCmYvlXToefl\\\\\\\/jLvRDA6kgHrOm6dVCNK7wSk0NQ7ZejD\\\\\\\/onfY4BOqGEFK0bZniNu1StzHLchzKQ9M7LM3krx0gx4FrcOtWbGSMVX9gm8\\\\\\\/hQLwNX0MNJfQrjDKs9fX2kqFfPCRjo9f3GzMougUdGFFiHLAIa68cK8R\\\\\\\/To8QJinFy12MQF2r+ezqJNsaol6+m7ILug4ZWyuhCzapA5totVH7vyCmhqjbxVnDkyl4xbaS\\\\\\\/JJThuNGuTnkH4\\\\\\\/ff4D\\\\\\\/Y3gflzrFTHN69VZj\\\\\\\/2hIMkeeK9KGBr3pXRD+0OjSSYZOefDCJ\\\\\\\/oI9FPKGhGtIaKDQOkAzcRrFdN359rs3wvKxF+m0eUOnvHJKMh9SKPHJjC0HFb5wyk8cIN5SUj4okhSxfk6aVVvStuazCQbyVjA+8D47Q8RKuU9mwBUMfuofRfF593LwCgalWvIqOsxoUQlkQxXdqMWVJI1DUOPS0Ipum\\\\\\\/+T65Q27H33wBnZtKY3vIJYezlv2XJjJQTaB3HIAin4nvPVL80feF\\\\\\\/NqIPlwkmUBfHlWdTEs8AVL5JoKsR+uBUYMPQ6NWbmLWR0qftlSebcJ+qOqd6lQiNj6+iHz\\\\\\\/rCDgvckpjUm77o5D+GF3xTOjcYPoJguLe+4p16sq1\\\\\\\/wNQnO+xhsM6xKXPqENV2j\\\\\\\/KD+wQd6+++rWIEhUdGAZNdlv7+U6ikIHIW5ATPhqw03Xo9t5F3PC\\\\\\\/2qkEKF+qB+DJYhJ0ryPj6TX49ITT9wgimiJcXc85RRjKAkPcogTGB\\\\\\\/O2YKPzeCP3hB\\\\\\\/oPlSZLIjb7YG1O0lpAHooeGFNiGteq4oRFgHu8xGQPYFqNgiVq44bltSLCFZ5rlGU6qn4lvJgQD4OO793KacvrGP0xI4TeHqduDG\\\\\\\/Dy++Egc8LW2QW9aRIQ0IzM+2A8jeT26vRwq8CO8GzxEGtCcQ7Gd2zJslLBkgcrWXV5wZ+to2XsR8HxHMOUtxt3B4A3Eu3JwgAzlOR3lLkexQFrzlINkcjoelXs0xYPBK4ST6OGocNMj7pP0B9QJVyuiL9DEUhId+DZP0UdZJdCAtKR+X2WPRkvMbj\\\\\\\/mJTDnKUhG3yvgRmPxZRGa+TFE4TZLcoZJ8y0QYJhqpWTENV15Kqax8SESenpt\\\\\\\/LMYWm\\\\\\\/\\\\\\\/ruarITNaeYc\\\\\\\/SGvwBpJgVePFcIEu9EbsvOBWkqSm083bX4e4L6K3YvbeaeLGVFvADIRgs0CGVVFx1kC4PMYW7+5fA28ICGwXjTDW3Ave64P0\\\\\\\/3kSzAE1wxWQkWgp1LpWgevXBrIHYfph6cqXahKbk1p8TkviW2PMS384eCjOWIFv18Mk1OrnALbdwL1Ql87ShppjcPnAj9pwuJow6DYZUoPbMcbS4oT0U0ZSu485QySTyPWc5WENVfsUvvc0TEuuoXP5H4AZj5TI6YMozE1JO7kFTh7UnIfNeZnf6vPiIghMI\\\\\\\/CeMxQ8U+tJzLVs+AXhRkfncusDFFwDboQYKzBEVSKWm5JxaKOKN8sIC\\\\\\\/MYtTYIE5RO\\\\\\\/RnznYIZWMc1pZb2JDfCrqD\\\\\\\/9AfimDLXd5CGGY9CWLJ7cEBJcdfetjrAiyZsHulmesjsWxL8JnSTCnolAro+pbuVU\\\\\\\/+2uvGwkuCr1kTXV8vhfH4OOkZTcCOteklvGqSwAtQaARQy89XqNgISUvPqx\\\\\\\/\\\\\\\/u5pOTGmphiWvfvU11hpWoAJZo\\\\\\\/+i6RTCof7c1nQH5zDihrAual2yQFQCeKNti0Stjpvhq4kpt9IkqpefwJl1UqxzImDyyzarnJYAAXG8psdCKJHV8lnDXlO0NRA6ZBIHDCi0xIPWLLi\\\\\\\/HfqKqoc8itwLGtQHnA9raSA7yReLiefkXWoxkzKsvIpZxHP+NU\\\\\\\/kDE0Xr6d3X7nKtiBDuXlEJTtjCXXES4PqqKZG6Gl5UpLZqHczo5vLH2ofbGTw9k5zoKv\\\\\\\/aLrNGaJ0ZXrMKCJGsAT91nsAVK3SHy59fcSgD3XqbCtLB3bk1GmB\\\\\\\/eg3PHcfh57laNvKMiOnv8q0qpuedr9v37aA+4fZWsflRLovp5pFJ3MyrnfBNoH9lKi4DtGdIbu25AG81uBAe+Wn3O4+w\\\\\\\/qYpufweO0Ej77efG8dvXs2LGYBRDQywA+qv\\\\\\\/Dhz\\\\\\\/\\\\\\\/XNQKbkHWg0OIyxrFw3U8XcaDEf1GwsAsFetM7Jm09\\\\\\\/3zaivp0Kqi7nB3cXH8xXy0phcfDWjdRd4S2nyQ6XdguqybbZtQQYJ6jFH2sEVLnV8rhoBUY4z0qXuZFY0jo1fkDHc0UDVUWdz9xT3i5YqGL4K3+5QPqIfpF\\\\\\\/vhd7muT0JEgXBTKgs2xZM+JmRqFbj\\\\\\\/k4d8QSVebb\\\\\\\/Ly4msXMDkuL4My98H96s8cSdExeH9V3gpIs6h4cSdlrCrnBlLSxzP70FukEkE+EB9e7ep8jsH5JE6Rk\\\\\\\/Iu2osFbGv+pE\\\\\\\/ldkzr4ACLXZ7Il6z2qaUNjBDbSUS6Mjlo08II0Ub15Q1DDIyaWB1ROm+bXSt2KGPKH4Bp6d2P4v4KewXco3ro1ev0dysjcrqE11IxqU35aYjnURso9wtyNZp+It+ZSOHaYIn\\\\\\\/LnptxQIjbFgV4ojkOLFxi\\\\\\\/PEsS\\\\\\\/F2O1FncqYWue+qHZCABiTQmgvgwG05yuALG\\\\\\\/OvubX4Bc3rUtmq4ANihSXmKPAnN5xqW6vB8qyOuP9EU3aMmEibUbDVh+V4NKaaczEMMxciC8TFSAZVbzT6oCvziQlDv86Zy4CDGgIJUxqo0ksF7wNz0vEGxpYUrBvesXd3hCENWv34OOaQ\\\\\\\/v8pyqWfqXZNrDiCT93lPOQpOwCoYPcF8ow3liv+Gb09guNgBc3QV\\\\\\\/0o+rta9va\\\\\\\/bJlrx4r8LqqdlbOed15pMPN37WHNeO0\\\\\\\/GtklVdRThBZy7I0wss9JJPc1Wg3Plc35AwMGPmjwK6U37nP9ehCaaOo6zUNDI1hrCXpDKhL4cWYmMVQxuW7VeeGfGDAQr387wTW8tgSd09VTZYPPYfmhnw+Qjs8xSwicxQNl2naT2Ci9hFhVB0ghJ8+sJzHxwN0Ay0dBOKovqQ3xPL1oPow2c\\\\\\\/lK8iVNSpMut3Lq7H9oChUb0EvdPX4oOkZAF7GZuGu3lTJJ9Jz\\\\\\\/0bqzIPRI\\\\\\\/Lp1rSiRctFHhq3Cp1QPniesQFmRUoa7UUPGwnhe5uHEKqc0T2YdVan7Wfu1U9EHSJz8\\\\\\\/DxFyj45WDqBTqHZnArSeDg3q0jKEB9f7LJG3gyyqG0HusJ+f45cBurX+DjUlqz+4TzEYu6oN7BjeJc7iA087UNkJ8iq7TIifnfkaNHEcTDnQ034E7VAirlhQ8ievZdfEpDLy0VIFfTBf8CGQzn8Zrg0rYnlS6xdo+yK\\\\\\\/T1Y+nZgHJFrUIL6InV+Kea+2BKwZ5cNRiQg0dK87Xf3yDyVbyYBe+ig9OB+JHBfmytwXEDWb8QxWHSpv03taOSvIFHlgDyDcCRM188bimFAVywc\\\\\\\/zzU5+H+pxHMlXPuT5MfJ4hndZGQGG9OrMs3qQGTdb55WMuMGEiuda1XEwe1kYxrlgP3awT\\\\\\\/5QJrUGtjWskOIHGcke3w7D35oc36katYYqcZZzupzQRIFUHsI8C3xAvE4O0nKkGOXmLevAnOGpjsb6BDQzSBkp+d06wQ3UudAo2iE7wzXgo7AmQdmNzhJicpubG8BDR1NWG7uWwK0kc38eBtvDDw5MDiD+UZPpfWswBm\\\\\\\/rxFvzi5TrJfoWDPbz9fZ54+Gu57ZQpu+Pr9EQ2BsbkYLddTFjDh8YUfzsFYltzbSoqD39tQkB17AdBQzUp0mx4l7o9x0Ofr8MQlNNCEWek8467sbQxyp2ovMBd2nSXPUolFQB3XlowYo0FS34KLUBXm4YeZ1gMUTlUNNho\\\\\\\/u6ugK2V509lyKw4OYioeuBBEKAi0hcZ8wLIIhtLFhAAzWuYdtXJQOJ0Vq5eqwLxpNjxEW4zSsBiU0CjBEAYyfqsgGRGdicFJIoVD0RAmyNzIAZTA5lZdBtVGZNr7WMJufMqqeAQibxb50HSmaTZNY+fmel5je96KCa8H3hEbeHppI\\\\\\\/zPIbfHd4QToC9pwdJ8+QjyePpgNXJFBhTF\\\\\\\/FqyJTrtGvVAmyTs410b7lwVvX22tM0ilNsfP+e+nVM5sUDCowEPKiT+o0TZWwg\\\\\\\/UqEW1bgymFkEdXpctO\\\\\\\/5lOU5ESHWx8Tw1Gbb6ElIJe1PNoB5q0mhNWgEYMUFYHzrcw1N4EIuJquw4xBq2NAQnspAqfEHI9Aj61ViWukWDoVzacZw666t82NEW2kAIHeJ7DsdXZYILcwuBpV8x+bqtTvs4u02frxNekJo5aDb01vrYZL65myxsEliLe7L0G1\\\\\\\/EQURRbb4LeQg\\\\\\\/Nh5MJUwjNoxBnsRfBc2UcBfKiwJlP04nkAhZXx9FVWf93oPyrPDnkULjBH4G8VrXb\\\\\\\/l6V0Ce22ihD2dU20gWDQ\\\\\\\/UE+B\\\\\\\/tOwC2osDREOVcBm2+S2Q1\\\\\\\/k+LWasLL9uE+cntlw1mp+TorrzVihI\\\\\\\/0YUTEWG7MkhpFQJbXQSXbpvHoPkzUgWylzgLE9mmPha8OSFYkpc7IM6Gu3yDicxqLx96BHiKsGoTyZqupRtqycXMQ4\\\\\\\/oSKCl9DJSmI5NYeoYksPxVAuhymFVvvX0WYsQc\\\\\\\/foW2q99fiPCJcFaJU6aDqDMClsfprlq6vtgxT6KmL5DQy+7XNXVu7y89YDArTBVgYbJQSbc5pIspmasCVav8OaGGnFcQLjUG6jHxziq0FVANUZajMvmY6rluQ0vHQSAN8bQRqOrtddsiOBjOHKCwwZkyN0lxcXtZiZ5FZeRxcd7pIYjNuN40TEKImToLB9I8d8fWQqWUOEpaI4PC+o\\\\\\\/mQEIzxLr9bfIAgWQmsOvXlcvtl8Bt40wbSseBr4fQcTN0g5UMp41gXyCRMULoIGfg1VIWQLdDTTAPUZq6NTPANuUp+7GV7xsYQ7V+OBND1zmUWJtMoz6Pb6CZNN+MO\\\\\\\/ngJddv5I6LFfadKnYf5FIdkAntHOsts0ytePfofre2u3SIH6pI1nJO8uiqgLdm+a+zBJuObXdS36TADrAOTNVyZS4jxl7Hsty3zNFh77MkBeWhSn2oo1MUsr2rmqFboxnTHCdoCm+qELgc\\\\\\\/gJDMTkKLcZ6lRalqgjGNudyBgBydgtUL0+iVKN\\\\\\\/Y2Wwy1FOaEaB9MOt6c7TOlz0H\\\\\\\/mn1SBfRSq7+TI6\\\\\\\/EhEorNboamtU+rknDezCMChsKU4cI1OJP35Wetykp6ReI9QOden2UXa8Q1mDa\\\\\\\/p+KST+IFd070Z5RMJrj4ag2F2GbLqdHNYC497pnaWn4NAS6cAS5Zm0TBtp5SCIb8lj1Nd2hFGGzMvgDU+uHcDelJGFn4\\\\\\\/skOHbCb1ElcavnIwhbKdR+ARrM4tsKGrC8+cbEfps4QSDw3WMnoSW1uDFjxJVKZVsJP9eEyeb52Wx\\\\\\\/ZeCtvTRX8glkxVKAofxSm+joIvS+WtoJFXZV63bWZF+JYSlGNhZ6HYYXMh\\\\\\\/6n4H9npgdMqyFZmAJpziYeXaSm6BZUY8hb9aQeoH0zsg\\\\\\\/FCj4jlm+c\\\\\\\/XSrraE5sEHMkF2xDMacrE80aXH8GqhwH+OUGPOg6UNGZuDcgDuISxHTRohORbrIZrweSqW1kq63NWFT6Gqr0SBdhRtHHnTD7jP7l8g9YCsiKej7QTCMF4+Gq3oJy3YSHzKhtJh+L78Iihf+bc0vyKKZyLIb\\\\\\\/dKvveaJl3BuvMhsR6WUFZphYdo1ZGlvK5q0vb5SzLniT6z5XoJYPz\\\\\\\/OrU2X6t8MwIsEgYLzoIsq+E6Lf7EDkuU+xcqT5q+5ofxjX3Aao+BcmwipSm7B\\\\\\\/KFokmzBEE9l4tAFeMII7fP83Hz3xqIZ6VVUK46uMe\\\\\\\/WZ3M2Y2ySTMXG1vAdrarLe3hj9hh1d7n82mkA2BuDgPlvcw+FmqIb\\\\\\\/mEqB2qCnGHhNtjw8gzwmdRPIkGgtkENDL8OX01w46zAG6NZM5g\\\\\\\/L9NAa8sjLTEU\\\\\\\/SkKsf3LdFGEvSTB9M5X8zEA6aUtlb2DrC+1twKCeIm038RcRDxhTmcXRT3K5oqmXENyU2cSMOruZ6yAHz6FA5spJ2Tl8KOu6kG60EpDJzuHHNG6oabIfO2YAHgit3ym5DYlZW2ZRc+\\\\\\\/HDpnvfrrbHymRaVAqrCsu88Zzo4sSrWjCTQAt6RnLcVeDM16ag1ciUuIUhx1YBDt2j48zQgKiM3XGYlVtukgkHrmBuZ4l7QbMHxgNrspxtAJQ5zURMi2JoYCgx0ElIx50Ly0X8dvMccdk+y62aqpJIVPAp0v0HbGFxxbSE2vRcJSmAO5mTH6bMCoyg9XQUS9cr4\\\\\\\/hvyKpVSV2AwPfhvmNRy+1bQYu0Crz0KNuKmrmz0l\\\\\\\/dH7Vqn0wT6lo\\\\\\\/r714uXupgGjSM4mA4CsC8TYnGrqazBMltrrZNL9WYR0NPvlZCB\\\\\\\/WNDsNwTgOnlF5RVKsQWjttebH9JaoUze4IO\\\\\\\/lp0FABDGMuTnBzjqF7kq11mLTH4TWA0afMB3VdNXJUAB99oYR9UjQmsrspmyHApQep12jHNH8wIFaQgmibL8ogx0vZma+gyDReIvGIJ3Yppbc\\\\\\\/OmBaCW75cwIHnb0\\\\\\\/DFPxWZqHr7obJJPRNJWygAhnxlwaz+wxiCHhQtP635oF4hLFyqyfU4YL\\\\\\\/gnJZxolv\\\\\\\/yeTYTfNSgUgH6ozW2Z8PF+FQAV+ZUqqBjRyWkCtCETJtq7EBFs26+bWMA1tzd6U3OPdrbeBvkmTmmcOUTDKdLJxz9jRDLPSb8yAHDH4ch\\\\\\\/nQzZG2JkUDxXlX6uFPJVIyzsN2BEOw8vFzjg8JC5q4M6wA0ypr35mfiaJMd6l87ZHMYavzL6WZ05QIekREPWj6B7EfZqpafI5sA1YjEoxJGooL9DB1mtcec+ory3rPk1P+lMybqBjNA5PU1gKBcY2IY5+8Vx2X3PNwyEmKWQTtRkBSfLnUH1mHHOF\\\\\\\/jowuXs7cEvCqP1vdSSxOXtmesi+iCaHve1DYPQAyMv1xnOop4LkHN+SblHf1941dp6uoZjzf\\\\\\\/ZfrG56AgRzg9tr6Hguvp0Uq6flmrz22HrNT1dAfNSQXIzhNlOXOvIz2y9wZQOh0czDqxb7q75ISTSuw84HyW+hbTwXUq6EgkOcukq2nBJRDXN50LgYJV4El34nAAp4mGIC\\\\\\\/XM8kIecMVudFzq6K4NJZH8WwUiaYmqoQ5lLmzfzZikOBJX3JaxzGkCfuvqh\\\\\\\/hrpBsGdoc6fiNR+PLNmHJcXudT4aDXdR7L8509GlcLqTq\\\\\\\/4DccCbOb763t4W9eWP0AKc40BxZOcPtNMgsUT4YCOBxrPeo+yp40Omz3pi2CptCDLBEmueIE0YLFtq7Z3jPS9dc\\\\\\\/fXGLYo0WQ8pUAovm\\\\\\\/GgzCzI5hNepN3Y2tPQ6xOQ\\\\\\\/DNRY0utPa8EvBsogaOFqlMTOdIvwBT\\\\\\\/Y\\\\\\\/hxqasFoe+nrlDTbUyNVjaZVVxHhc48dg0y8YMYRy95OqeMZVVQub3gR24gfkdw3SxmWvtBXILPAsu33dEhm8AlHgU9s7qBVA+bc3xeZMmFNmrgwQaOw5hYjAcuK0nrPYO+S62SfGYAVMPs9Eb0D1HfCo0qGeoVuV3HlJ9Ig9NBw0IVPtMshuMY0bARiGTi6qwBiI4QK7y2XammmWmEWNyhQvqIVNiIe+z9\\\\\\\/6PX9x1CMaQ7MQ\\\\\\\/Wq+GnQjyf+NJtUA5dxpgaSB6+Im+CVkQyDm+atDA5f38O5sCk+0\\\\\\\/RnsvxZyvQ\\\\\\\/FNSwQjHICSo+1TGm4wlPrEM9JrVLEnYvRmQjjAZ8bxuCyaU1GuNMqrG0O5+nYhtOHAWVOoJHdkjFoD+BY7zhM7jwxvRvGNWMGZp+m3YmFryAsqzGXMMcJ2hbZSJfWveTM1OvQcuz1tMEB73wsK1o8bOFxGCU3RiG2KgSsoJUY2\\\\\\\/w8KzAI\\\\\\\/pS34RRPjtl\\\\\\\/qjfft9m+U2tjEIqfIl612FDlnAbwpSj4NvK\\\\\\\/Ma7rku3GZNB4Dq9dxmx65w9Mq7sEiooaWJh3sCTUxG5MOQGbZvUQ84\\\\\\\/47I58kJPOvm172IuHlcyqOPtymmUtdzq5CWN3G8zCq5n9LpO452dAdkbWiGZVqReVewoW\\\\\\\/x4H\\\\\\\/V1bOutS7lfRnQnwlEcH+Gl1qcsD35pWnbaY6PN6djRK09L0zuklmLYdUwJAgThkm5V2P5yyPARTXgEUIgfq68nuxSlISUKQ5vVBMH+8XSLJgs3J0805PvgWKbGRXY3FNRqbFYG7HutFhzZtfwZbZIPFqyREZ8GQMuru4P6k9rnOEPfHsfKJIKXTXGxU5M07mujOUzy8nCUuuVPVF5X0sxZ70pWTdDQ0FIdbyarR52n2y7mdoxR0TdsczT5BgLAjsQoFktxtBzsmiJNvAucDie2bJbRHLfv8rB+AbPCvfKKxwR2ONZO39d1Ii\\\\\\\/HpHffvop5zpjBbjG4Kzx65vPWrHPfuR1DGlfBxnYD2NLEiudWgM6mQAxYSLJWdY5GWCKUVDLO3VQCg2nh7vVZvr\\\\\\\/2370NnRX5KZ7cTAc9OSEfgX66soqr7hLaLYK6lkxHHcx\\\\\\\/\\\\\\\/X63UWwBh7elVdnbpMMHw3i9tdm3BdnZi2DOSNvpSLbi8uoTyWCvPxDO0oXp5amrINnfO1voWft8aANT6ul4WfLHIKEG7wLaG1QPte8DzpRpGJLmVTdGHsSeY\\\\\\\/qYlVkLKE9T33Tyjc5bEy5z3Ts5Al5BoSzV+5OLgxc\\\\\\\/RGJqFjo0vXdd8vkfiBaSJCmHSdxbMsWWqZw2d5vRINRcwWhf5P0tr7PKdOKPwmzl4F2ha3bq+d76xZ6Dec\\\\\\\/9YDlJoGsk6uqahWjRrExAPyO1l2+V0VUysYraUQTMxUGWuuReKmiYV\\\\\\\/in46\\\\\\\/7uvPpndw\\\\\\\/K1QWWNfMre4s0hTIM93X8ACops2Z31T+fiDebRysAN47tHDgO\\\\\\\/H8P7aE9FOFiy+sAQ6Dr+kp5mqQjKxJk8yMEx8rhCgxjsOzl3y8p7g93qnJEFCd64n1GKC7bmIOyW\\\\\\\/IOlZUEPd9dfUvoD\\\\\\\/FOWa8\\\\\\\/Q7vs2OMEBG10LTF5FujEL6BAXP6oMaNnQts46URUdDq1v3V4CKJuvDuEyfiYn89HvYhp0OPOWNk7gvZ13NGuLuGfqxxsc9m6nc5Xl7soq69ANdtWSMSBm4G0awVoeB2t0nmm8wNk\\\\\\\/ezGUkjsnT8jeK\\\\\\\/XkcIgvfVI8CZ5EMEgnfc6xiwAkUG81ceckRgd64I6xizfPL3PRSqtwbqMWI6pyIfDr0aaLM06kYSsVQyOAdCTJ8MK387XdBCkbXjvBm4L\\\\\\\/duzdms21DdhxwsWazr0Uzw4e7T9rs0j9W9+bityxYGji3RidJi7UJzdcyIOj\\\\\\\/KK7uuyUlzhxAdFEAG9HlIFBhuncSacyU7cBEOk2cpLIvcnySp1kUbGNm++Wq2iJIqOJFH1RjYVrimoOfbEwTnn8Mz75A6XFgfsxMRTM\\\\\\\/oYi\\\\\\\/y4Ah68bwrzYgpk02\\\\\\\/qhVyI\\\\\\\/g1zuU3MWIiFy0Z+e09pr1IKbl90w8QEwKz0I7kMqMMCBuW+BeEeERSrqesujIXAwx8XY44nabDKumDyhaXGy5RmuAjTnHnI84NSVr+rf\\\\\\\/4VDA6unzkQoU5cQmEM\\\\\\\/3+984bavAcyCJgxBjaz0a3XiTEs7ZNFiJdXPjsMe1a+qr8BokE571Patj9yFngQ4WL8c0PiKLao4zodRZ5+usa7gIeeCEph0iVisQ\\\\\\\/n\\\\\\\/ntnxm5Te1B\\\\\\\/3bTPUjPlEAtG9XL4CZI6A2Z7FxNIuuwwZnNPfl2iIKl6j\\\\\\\/KJC+jDXr1nX\\\\\\\/0nSIkqVH\\\\\\\/dk28lZtrEoOyiZI99W3nIeUTJI+00GLEKbqUOrWp0O+BAGiVGLcR9tPGg43ibuToUzuH2W3H+1jMETtLAPSxSVqpR5rDfXA8E30Eet0rtbNWt3Q7RVTKimrDtBBox7ewTGUa9MZUBMQ1NuekQdNCtnxUSHYAPQBSqHsRUbEW\\\\\\\/9yj5VrgNyqQuTeFgXAky+k2\\\\\\\/fpvQk4gpK5q9uxkxrFhAmKkdykWGMxkVizOQvw64xw\\\\\\\/Y\\\\\\\/5r33w6ut4K3KKdoRqxE2BmuBhEYazgo0kpaUJ2A\\\\\\\/7\\\\\\\/mOGCXZnNV9k8tNjYxAa05\\\\\\\/4mlcab5RjChu2J5P3tb8aqFNoAa\\\\\\\/v3E68VqoWcfeQYKSk2jWJWlUfNTYukm+irTGUhK69BCAnmVcw68QKY85hF+sfeMvRgRXIwTmuXr9nuJMSBX1CwjF3BKtUBur52mh7UE\\\\\\\/pQs\\\\\\\/iK0o9PxilRTlg1azFeczMTItphFVU1B9QWX2fkFzGkC8Yzir7kiBMcqOTuew9pwYDiTAIME6MoQ4rqGMwDfSIcsEtTkVmFFaUhW7naaAhapIZ\\\\\\\/h1UV92chZJfCu3vDWhk2lSePMwQbWZj+aQgoyCxeJ2cWmIy9\\\\\\\/szFnQRoT3EZjJdfiWAoZ9ZK5G37+9lv3F+kGtI83NBeyvpO6gBmjbtm0\\\\\\\/iH2ZX0lAC\\\\\\\/9CipNTCbcT8W\\\\\\\/dvELSfinF5jiFHN3ligqky8Tr6W8up28DN3khfSwGTANsj7xW3Djm4J2cX6mTdYYoO117tpDWq8ewcstGpsttx+P0VD7b7luZF4VmIju0Dl7fOq4PZiC0i8CRe6xhNGijItZ\\\\\\\/Clv1LkDyO3emDJPMZ+UK4\\\\\\\/HHKtrjoPyIEVxTxfZhmvpTEK4nK+i5bXapAiu3V29R3\\\\\\\/9+xpSuxLLeaCe6d+1K7e0ufl+Vo0sXc4Ii\\\\\\\/sgm2FeVuAEOckWwyxxwb8h0DEytzeg0d2j3F\\\\\\\/zlsj12rR7u0ZuMrXxuJwftAV6VwVH7UUV7EaHsY\\\\\\\/9ubsGRMD6xEe\\\\\\\/CyKKAGe17\\\\\\\/oVEUlUTBwHucYTw47rKnKCtBlQHQIrTvgaaeSrVTnYRR3QS6K3XjHmME2EgBNnNZ6gqQWDoSJF5cyBwg+6olj4yvrCofudrmNYVztKUCI0H\\\\\\\/ZhcZjPMuXeCn4OpBsI3YLnRD+Jv\\\\\\\/eWw2Ei9zlWlZ4fxyYnARtMlRiRlpEP+zTrsBMGW1E9\\\\\\\/\\\\\\\/V7uQYgDHudiv+F\\\\\\\/gdAHFHsYbHkf+JzeQLha1OWC40bcfDHeMuOwmQfLhfauH2fw+KOz35TajQXsOo\\\\\\\/Ek0YV6AwqOnpdNqZQlPQ6Afw0UrP+NvnyKqhg5dR3pfGRqoIu1Ra4dZBvz16HhgY7zBnTsecytQwNdw3KO8OauY6l3bebvr8ynKteyglt3Pxx51rFBt81mhykUPYHMwtokiTPe+QmZwZ\\\\\\\/M3Qsnd9WG4dh01rFNVqFNqkhlAlk2tUhpeuBkcM0vFFMIIASQqIB\\\\\\\/1q3NnT+6toPX72dtuAjgsauO2d97fdD\\\\\\\/nNMa0i4uJrYcjl3SYdTrG0ggsj5glhmYu4ELb8M0Fd0FwtK0vkhnpb+0962Vf060dNFIGxZQA+V40IVYRYjZSCgwkPHg4OjSC0wPzO0CBTXly0OGc5rfVw3bExxzq098eixFhIk5cb3x4GEur18c9DGtxHzqEa9B0TAaHrY0FtF8\\\\\\\/oDA48pwOQZPRUTonUWrm4NocaWqiFqxZYX8S1nF+CyC1PJPtY3B7Wh7ITLTkK1Y3iovJ20DblF1cbbkvPs8xrTLqC7XcFRjFywzvI\\\\\\\/Y4SKS4iMMRHfAvBKIrrElMQmACUNjeM5D48X2w1f4P12bCNthOSFqL21hnMOZt6ZASyaNqhHDRj7US7Wq1jopBS172aexshPcXMDUelmZTV+ZqGgQQ+3dXCa0+g0AAVNdbxts0ftAIrMWE+VAqBv+GG6velVrm0UcGlAxbC0tElftk4cUi4DYZFxVgYZa8RAH1Wjl8wlxaxpoqyLdanPCVNQ9JSaTFbVm7JChAXGKzoh63m6myDK+UjLiAk51DGZZe5Hypm1gB7WRFOFbdT4EVMI4zl3A8qdnZ+kK3O75v4aDh2kmhnDQtp2zy36I1mg6AR21kL55g+loei+OItltw9Eu048vO9x6OhKlCRcU6i624bqGWzWXKyvU1XmlJbhfl\\\\\\\/XZR6Cm+lyYDgQuL5dQXckLC6YDdUkVKWuR217BgBJNb01T7HGHRKynuZt81sLCiPZ1U7pOLKtTwcxxRoHAeH7bie6zC1sOpqMpf2BMwJPQ8GVrAVvt6VdktUEJLrlejKXHPE2Sjb6bxnwHfqWmPcU84SybwEB2ds9gHwDO5XeZ+qo8keE4v+ubPGc2yTJIF1h\\\\\\\/me6+XaQCeZP9GWz+G0fbaIE7c2xRlHxO51Dwagim5GXY\\\\\\\/n7Wd8Jb2XQETvsY8bRQMUq8xnCRnsRsBBovx\\\\\\\/GbR\\\\\\\/PBqnU\\\\\\\/f2r1Zv1StD32BDEsb2RXHeMnCUEWYXyzWPEGOloGRo9mR6B7FVdQOcQsmE7N7BxYG7+Jp4UA+9Jbkj6ilSqZHs7jfoW3nefYlKF4\\\\\\\/SwL4eQ3JtdqJOlpROKDfSb9rLjPdO8Q5NsWfynKxgFrriCoiD0cBn9fmSnxs2L5z0w3ZCZz0yEoKhfKp9Es73gDn0XCg+7XWkbxJFmPBZdNRDlcti6vrxev+3WSIWU2TTVKQKGmnyJQdnaJ1YKZPRKFlFTDtu6qNKZjfA8z6F2idwD0nY+vWZh\\\\\\\/ixjCqE1u4eMS\\\\\\\/U\\\\\\\/LkHMXNuNt+Qp3Yqw0u5KPBhXeowvurS8NuGbxCkt\\\\\\\/QLkfDxAHCcYL1xbAxURMsOr25X90zSthfGinBrlp3PNC\\\\\\\/MnzoNBfmUbiDuH4NPj4qaVKBinjsSqdJMRW8D6Xhwnhte+UnIwqGWvFNB6DaD1b4iCJmmWvkfOKqyH0wV1JkoJBlc9rWBUasU9KI\\\\\\\/VGb4FedKbfF7CWfgArnIzie+YAyS6JGi\\\\\\\/O5FYCNIjIlkO\\\\\\\/eGZogzz5c6hYOCy6Im3dSIeiqlE9yfe5fQgmjTeVVqXS6lyCsuwtPewRl9FWjMWiGLCKulx1CKMdJ9d08UGz1fWgFXsIaW93DUIjVw7bPsAfn7q+5opZcCsraAeDHMqIOi66IccWucrf\\\\\\\/N8hyvpTA0XHh3Th3ai8Xz+Gl7GwyUasVq2j\\\\\\\/T0813QQ8Vw1vbCSWQxbfi+OdjSV3GVQzqpl+HWpQNQxfwca5o7NyrUV\\\\\\\/9SJF+Y9pbxMdRqiQP14KPl9w7mQZd\\\\\\\/76BpWhTYGL3gd3RJNg0qi0MO+zmWbvHUOUZd\\\\\\\/mcbSu7gVWidXDipeWi6Cna3YkXEXbeUXQpfCDBPUQVPmUAAZrBxRRyVHDEfa0K6h84JL8hSbjUuwNTfMCovx9ZFu+f+KJlqQ6Hyj8+kxSUcdJUMi50hm5KUjki324gzp7BHrhJmQ5FnzSGpuGplFT2kvowYrmEC\\\\\\\/Kwe8ZVnZdX7uidUvrq6NHukBFjElpKA3Q9U6fWKh3DaRD8D+jxoAkhmreLS+\\\\\\\/oeCGmOA8qWChPgFN1TzhsJX46DJCKUlvOiIlS4rCuoXTEVb3Fs0wTFuB2AnHeU2\\\\\\\/SKTRiEVlnIYaAMcdevW+8Rswc306IdmJYSkMMiyOBhPksdtZPP3p414wANtp4kyHHveycxOAWgsqjQ+i0av5UgZ1sM2MLn\\\\\\\/psk91dlZtfRvEOwAb+Lb1ix2hdYduiAd+Hme4a6cRUyi749BadZV75tAT3eFx58O6nSMkYJw5MmDhpignVRN\\\\\\\/r\\\\\\\/SNc5g3dnAoty8EiDYW+ZytDz2\\\\\\\/h40k1p3EQyL4ZE1EHExtIVVsX\\\\\\\/tD7Ykazi00aPa0OmCVpmBDZXNxoY1la6\\\\\\\/dSd+IpmdY71st9cmSmPujUUX46F+UVcmuP8UYc62PVKc5SI8kTiRrlzfBfkh66SS5JKoB\\\\\\\/XbJRdQx0XrOl4TRRgPx\\\\\\\/SUKb8+BLO\\\\\\\/5ohxh4KtSoTKSv1SuSRyovkEagUe3PK4HKIXthISLQudYhg7ZsZa6Mv3U1jMf35ImgrXUJJ5Lok2sfytT1QdPjNKliuB3mn6g3v3bs+ihtV3pGMRy1G57zlTbJwJq8fkvmYiK+hfLYEVTdqjD1V1r8iOd743BxlXcuzb7DkS2c6pljUjNPfQx7PCAvsL8+xcoGX7tTgBvAwQ3wRG\\\\\\\/9av4V7byZwndq\\\\\\\/Fbi3eN+dVGqPlLCQwzUUDd3Y9RL2KsYwDyIXpQv+MMBUbffRutJFBmqRXqxsVZoUefciPt5vqjQ5Ho\\\\\\\/44Zqu3hv0ZjuOpOmo2214W0nha6OXyJTY4n9IGXnuPd9WMG2gKKOeUbvA8YefirMlb6wTS\\\\\\\/OptlcVbr5xXs0Q35M9QpWf3MWb6KXV6s2fZJVDHe7hWpAcLkkCFiITXq6rL4lV2BirvD4A9uMepzmtyULCWETX9vJctpkdICCDX+jI9Bd4IIr3OFaLdiOynJ3+IK69rJ\\\\\\\/3ZXNicdMyTNt22lRYeBCdXRJ\\\\\\\/hHLqAYEs+KBSqkd1y9oyuJbwnjam2qIubDJ4QZu3CGXSMLJ04ZqLWyT\\\\\\\/42spr95icXuwLhnWFSQRaZFH2FqkAZLzZg5deESWtETZ6MueyiCpfgyGKOZn4bA1aekIZblW\\\\\\\/NeiwP5tUAtRCbxSLqQoIeHgM0pwfT0WtWYUusxWEgOvDftxjKjshI8XX++LD8MQPSG9qJQ2froFsbtysVfn7X36UsEOZiOUWW7q2G2EXLC\\\\\\\/MO3ctZgi0\\\\\\\/KhYhi6Y3l3wXGH2zNHyDbGvohg\\\\\\\/A\\\\\\\/CIP1uC7LsXseXfDi265TqC00IZxV+qPIRRNcMl89+u0SB2umGk6DNC\\\\\\\/CYWNU1rLbpXqRVtPGuRyZjeoW+LZ9+8v02Ub6irQY\\\\\\\/jdnurmqJ2E4r+u8o4uEsDYR1mOgEADQMBZ53IMorucJMOxyPWTueZdkfLNeEPnQJ\\\\\\\/BBkhulDm0OUD9yePLYVvp8N5fAylGGZ\\\\\\\/x3TdCJgPSqDL+eWGpVXSep2xrQdULKxeki3OUWmvsWX4QsmO4SHbclstvTkr8cvDdIn2BTrzjxPfHGDZWeOTXqu+qM8t54ecoYMO8Ym1Mh7wR6q7NZq1mw\\\\\\\/wzrxVwf5LN2bzNdFEWfBV6uevjf+3B+7OQZHoRiy4qw\\\\\\\/PV6r5Vc27eqtiEW10TFiNuSVIWc2v21Vx\\\\\\\/Vw6DNXYbMC1HV8KmKHuBnu3nxCmB3\\\\\\\/z2HUOEEoGKgWToE8z+rldfO8F\\\\\\\/J1cRf8HXgmx\\\\\\\/XnFa0J\\\\\\\/9GGQ2rVeBP4XeV7agmEro64qunDtqoe9QFLEGwHyPW2oxyBR3K23zkQ4GqKFICx5LOJIG+v+idLlQoIfuY16Fg035H1tZ3CekV5K0h8hyuvCKpWRr9XTYp3GKSXUV35PPMxmvZOo8vBjPd8HQy7hjbnezXPLBoOcQopEDdXShe62dbbplqw6+316QbNynwSN06wkr7wkDgLpgQVI3O98xf3UZWE0M8vWYpOoYEhzhOw\\\\\\\/Y+5JyO386ee\\\\\\\/QW011W6sxxPwkLZZMUzSDCwMlHVsYTJno+SOfkraSeey0UyGD+1yrC38gyoGmnqQntHq0\\\\\\\/Om5r7F1Lb6LbKmHJ9oqVDR5PQqhnVG163ChcT6WNUVVkE2ceQg5eI6EhC5gVLbeOxDFHpv9YtvvtDlFT2l7GarTHOUTsP3QRslB5AUYXOWDaMGHIVw7NgazKtgWVH9z5fmTCkJ9mImMUxDIwW5qnmZErFhnQA5yE+eF3wUQg8bZDbRsXLP9iSFIGe8CcreUxYEonuLgTyBVG24\\\\\\\/YzBgOz5feAXUSVlRqCLX1tWc96PiC2hIi+D7pplx1UdTgsPIIXy6u1k7n1E1tcTDxGI4UoEYOZeHR5RpS+XlFZHjj7RlvcCLyYGaP5wwKSf+Mljk40qX\\\\\\\/zJ6upp8XK3Sy1AxAXpMzJQK6\\\\\\\/qfV30MntrgDjKCiu4I2QUijfdqIWlD5G3mq3xFRAn3w37NcGW8HT5x5DRE9iYav8ZUzJFeIGTh2LLvkGfCNbKXt3nB8ALcYZdbd2ed7\\\\\\\/Ts7fR3eF2IqqXfCX4V+lQHFKv9dEdeo3onDMskN7ZtM7iWb0RwxgE6ULS4HGnSToVuXO\\\\\\\/KQ8bDQqQGfgAxOvqSL3ueTGJR1\\\\\\\/WzLMPniiI1g9pE6o+vJh3mSnZZgA4dcW35gJE7t2PkHwA3OrAhE9UIJytg17rx59Jj\\\\\\\/KKpVd37r9sS7sCkP5Y6P3TBTL7AVWZ+Ws4vjZ2lo8SQTL04Kv84tZQvVMP85uSMciSFA26RQOJZ5jA7enncR67NzzQul+eFaGbUDzDsYIc9ZPSNHwZ3yYuT+98KAEI7Z1q42Jm+CB3FsDy+2W64glzF2qyHG+0hSf+s5B6e81vU\\\\\\\/tKAQVJgjs2A29E0GAtyxVS09w06dkoNfwsDUtImguJEwwRni+5HeBC3eTXzLdYj1ItwqNJkKTwOJwijlBKdTgQ8qioAdjMeEaPGszYLy+NmNH5RsGtOPInnkqIw26YejBn+AQ3DNWNcBLzSbLREKGSqjhHur\\\\\\\/2fOu4zoBXyPm7NKfuvv3eWWxevJ7Ny2RW59yTe7t4xgud\\\\\\\/k+R8f16gdU90zI40SiC8E7Z4tBID2CRoC7J\\\\\\\/gI6eiary5yuhw5erSklZY\\\\\\\/FC9mXqLHaFxuU0SOAatb9wThend5OkeTS1xyV\\\\\\\/zLri59N6KqkYvmabItHf9kgLS7cuNqpBjCNXGfFLGNwEdpmfoiAl0fN8vbWi\\\\\\\/Kqbgyk4Q3yvphnXkzsiOu1LgMaCPaEO\\\\\\\/G+QQ0addZOXvJ6ld9lGF5RFUEL0PnafUEPJGm1XmvXNz\\\\\\\/bkJ4d8wrhHjHzkEYmyAyPVZx9seEIduZgYoiFtreeStBIJ1qGqkaxxcCNiKPtsabVki4Ply7M6cn5gknD5DOuS806IBgJqi4vCCiCZlnWH3F+kSliDh9ZUFBt9fvrWyrCUVlSvvNtvrRgvIKMJS3Vckvr4Wh+7gxFS5MZ0LhwyJrv6unmo28B8FSIpadvLOJ8w2QfIaOGcwJ3b6HkcCtkQGweJMzdFBJQ+qNdDMp5Wo\\\\\\\/5EmhC5zbtqpW9s1PhaiNlQU\\\\\\\/l4\\\\\\\/1fuWzXEspJ5nFEkP7cCZ6igkCWJfL7Z8sGCk2gqW\\\\\\\/r2rnftemzUEydQxIr\\\\\\\/W7CjdObuXuCMIHHFB79\\\\\\\/QBtIN1oWx0BBjw5Z2U0pERMt2GRlOTS+NtWJbOpyOgpcnWYTdpK9R0N\\\\\\\/CGi7jXDFW+6jqr8+VxFlFzJltA3+7YcxphNO3gHyc5lhZaBqHesXov1dymoiYjyDBL8cVY\\\\\\\/E56rvwN6Wjk6341yOJ8ikHPYlpr9VDemKs+qiGnGaE1xcl8LoIs1\\\\\\\/TTbso85\\\\\\\/I6Lbgmxxl0FH6B+59khAcwPYPdodK1xVI1A7et3u9Yp4MDHmHXlIKwyKSooKcn6rPKV3MhGHTh3oDGE4a9h6u4MuYBOY62uOzPFYwVGVST9acdhhsMdfDDEBMwDo46iCO\\\\\\\/iAOnyqpbJB0elhgVMrcjXTD1V\\\\\\\/r27TQV3D8rT5qMACaQMqfSKy83p19X7kwyqtL6R8WIscHkRg695xMHxrXZK7J7TDNO\\\\\\\/NPdpo7UsUL16Th+ad2Mr7PvNOrFNdnaaIU6k3IszcuslQG2kmnPOsl1LkrPuNjvjNxEiENhJRTQ9jSWaCn7e1nsmgpuB4EGnSMFHDWBiSqtsfbQxr0JTlzbc4ZuNRAUekRbwDnA09fC1IrmbPSldWePQUxMe3u2reOgkx+DE+JJa4oDW4+HcBCMwiH5G38hZKiLwd3i0bNAfynhwbkesSeF79miuZwvDDzFNtBh30ZG4Q8\\\\\\\/mi5DLeqsbjAq1HuWHRKucaKYhhcLuIHfoFgu\\\\\\\/cMl4s\\\\\\\/e4IqnqATwKobn7MCZQr9klOmbzdNOAr28mTUNZyR7X9yBkVjX0ZhoxY6F\\\\\\\/ZIqJC8WcgsdxuCgdHZAA9PCg4A7PgLxT200kkXAFf2wTLSiQuarjdkoWEdrohNn37W+jn3ikwtvepycYZQEe1KbfC19vR6QCwRyF9x1FfXHL8qzupvhnRfnqxngpWhI0\\\\\\\/fVHmxOCMyG9LJUa8YJM3O6RC2n4JKkoFjnv2rM+q7QxH37KS2MUwqlyQatCEp55tCKy3hWpigNc\\\\\\\/\\\\\\\/N25O\\\\\\\/2wQFByIs+4X+EeZQaHnGyP09i8X45kEolWRyvgWbDG+jJU1MvoniCQZc3ViLNuF96sohrlFwPp4hb9XAww37cqlgm2AnmXodDos0Gqt0Zid0pQNFXVo51uLSKpqjCbdBAFXo5aWeMGLX38igc0LsCPmrBmcdJgGy6mdPaR\\\\\\\/2Sv+rjcqtCEju1nYiFnQVbfNNVC6inHiEqC5dJgdfMiRu3+MHliKeDuo0U47ioT6\\\\\\\/JpqlpQLspbjB3ozUgjlzq04rtYlyYzshTiX\\\\\\\/BFaX8SHEQBo8fok8VH02MOIdlUsnanZLxNZSTiTWd5oUiq2njav2ChR3kBJq8Ll3gl3CQ5u8dRuVxgJ3TUQ2yMQLCRA8v\\\\\\\/G3uSicfEFFa+8PsmQ+Gg0O2p1bdgbgai2NTNRFIQZbn0\\\\\\\/KsCbgRErh2oAecoE08F216rxt4Psq3JxMK+twJ63SjcgNf6D0uayO7RHTnbqoRiU8eRskwWpVm7CN3VKh2C\\\\\\\/qx2Myymq2Z12dtKZonZ8b7ukIZgM7tyJgWbGi9UWZPrTqrpwpsWo5\\\\\\\/usOTR1aKPBDwVFeEjBTi7PR+e2qW7G83Hm6i6aoYZSTkZIPBaJ6+xnURTfSl1siO\\\\\\\/jtdYn1r9VBaG2kZ7kgC203s2D6AL9sjnGUxGSh6eKODHRjt049CSFgRFD+oq1ykmToNbR2mI8v1FHVVSzrjj7yuoei7JnRHZFSf6N8NiHeT9CgQUc1xF9MOhnigjY6MdTPLAQdI+T5ViqERE5+7H1WTqrIp2X9W3o9BRh+UGMeRhH72GgJtIPTrrqez4DNitvUdFiTQRI4hvqRn2gMW6HQWhEV6Z8psyoTIwndDx3kC0OYrStwjc6PBM0PE3+rwTlO6nYMzEtNebmdlWJjszU8sTc\\\\\\\/Tf\\\\\\\/vI1CH+s4EkkFPY+wlLEplfMh7sTXTJBcPe7qwGlv1lb4A9ZJy0CDClHPz5DheoX8\\\\\\\/cTQz2cTehRnlQniCjVpodtLfOr82gC8uHJEw4QcccIqACAf3YiHrVgQKUAH1KtXrn0YtRRNN9873vr\\\\\\\/fbuqZwB6sZzWhNvKGHl7YVeMrJPsbMXjONIz6y5zLTkCWTiokuRyj64BC\\\\\\\/ezK6IkBeI1sjkuqISe\\\\\\\/Qs+7l5nNis1OAcR8xghv2tlQPKFCclGtvgFz4bFHjqZsuFvvGRCpqJi+lnXw0o+gCvcWjwtfuCLJCXrGgWXSyM\\\\\\\/67lXDCbVwp81K00Lvu75r4s8g\\\\\\\/hPVLpTN1hzOH4PlGiHmN614XNy64CK5kgOBq7tAjFlP7ZzuHSbd2L3FdKs2OsOKFiMfxbqg0ewKey4e9fr2ZG+8DFC3Py\\\\\\\/Qn0ascQqMjhFA62zNweJtPiMP9IGt9iYX6yVTvxCVHglW4295JdMJSJhwX2+E4Sw8eFuNJCWLXLNqIfasjob+s6A8xJbKdk+q6fgFaCdPW7hzJHkahh0EWHg21jJ7Cnvq+\\\\\\\/CPgR505kKavmFpQy3MbjWjTH+5EY5dOgqWNsX0HYiRZPkzhXa7er41\\\\\\\/Ycs4T4fD+UTr59rEyn5wD9x4xgDMvayPvr2yz8GfdhLt0DSfKs8v61W6mc8zY4LU\\\\\\\/Z\\\\\\\/TJjY3ztqfr0IIMpalIfvSw7actM2btXsA0Y4SiixGtRoGMYbpPDRpq5wQcwHsadsbuWbtWN0dx\\\\\\\/18z6J0k1P0DPpPNYyT8c9rXzSH8pMkTBXM4wBa9B4vmbN3qF75bdefdsaDOUP3OmkHo3z4z\\\\\\\/dj\\\\\\\/1\\\\\\\/dB54WTP6pybfyv5DxWdtgeI+8n14dZlt5OC3rxdH5CdQz0goh+fxKtcsB0I9oQ3p7AHgga73+UIndKU0kQQbfZB8OL7XnNw3lLZ3aofubuuIuxTNd8xkLrQe7zAjdxMd8tzI6KzXVTpaG5cUPha97yV3V+F3oBSYBWLL+mCGxwHdQdMZSkw9NvCXoKXFQUtB7WqUwrj3Oh+ZmwmSaeM8J5niELmVSRuc+of0lJcL5GmLpiV\\\\\\\/QnSr0FC4Umz0Ktsa8IeoA19AF6BBFslG1XXldS6Wf8WUrrRrnxt3RVUxcKmRUHerX9UBb0BFZt1yZJbFOSs4k5Rwlt+lLV4SB5T6AEzUkAWm+2lAhrf5JgCU2ReAsQ\\\\\\\/yttfLdB5+XjYPyZ4ebrRVjzvXATuUgNp1o5\\\\\\\/96QzTOxJAFSrn5JHqjA4E9ddNizLJv0FwVnsQbiZWp4uYR0moT1T1S8olFsIL8Vismr3\\\\\\\/cZwVX0MKuE72NO9WLZAFViDWe4IQaqgkJ9Bb6IbXaUutlba\\\\\\\/WohVduH9haGQqD5ZJL+QKPwH3RBi9ixKZ1LnRF\\\\\\\/aP5Oa9S5xq5vNvGp67FgtjwjHmkcIgdjfJOKQ9yxMFfEgzj7hAdy2851cG3Hrj22OQrP8K8hAWqBqc2VuaWvAyvZ80UbWUdYfr8Tm2pPcrTfAtOIhC7nxbvjqrgtvBjqn098KCOd8Z1\\\\\\\/NussNJ\\\\\\\/\\\\\\\/LmIBoD7gH6FRhhcPEgbN3iitSO1gRmPMWdcxNitiRa3rUg7JMN+2SoRI44jMwnyFmMOOL9dYekxuyLBpZaCFdAmeJKMeT1wDPQBpYYhOg2SgY84v5ZYLEViGV0u2rjl\\\\\\\/qCgYwvqIUXuZvjZJSU\\\\\\\/i6Z6ecJEfaX9VV5ZGlEFkmmFAHSS8anZRivcSciNObKCmrgy58ZfSNWc0xuUN5l9qCyca5+Fw1Z9il1tnVUaYk4Vd36FtknqD9gvN7oH6fWenUkHaoLh0s6tNbbUa+9Gf2wEoWkl3BpSQbhcFN\\\\\\\/UqsoaWk4rF4ZK6Pe+73usHGGZWx0zml1YApfLFlwQlRkspAHjEQPIp76O0uyjjYSqmvq6YjKYZQMtZ6dMokdgXVcWu\\\\\\\/DIJdC\\\\\\\/V9rtKPMeszRtk8B3JZF07OcMyyMKkOcU2WtJor32hB2c9aGvEm79aoLUbjlmyV17G1Rm0DUg875FA757QG0r4tvTU4p++hjVYHkAFc9z7r2WG2saLpbExro\\\\\\\/kdUZy7r61alqPKDQcrbb\\\\\\\/v7s521vBRaywd4XidVdHzDmSPC1OeCFPzK6UJWrLV7omQZBMlq5mGT08t20SNjMPfWW1ttT\\\\\\\/LuEGgKRnRglme\\\\\\\/r\\\\\\\/OaxuhoBmvBsbkx+pd4kJQIjpkBTcvMwpt+y9sV\\\\\\\/2yIUfm1tUaGemjc0\\\\\\\/PzAcQxzgiVdlPooFFACzGHtgTsm0d30VHbMaYRX0Eq\\\\\\\/ePlY5jGBTd7+wmrrnsergrPd+5NlPj42j9kjJ6mhiQdiHMuX4RtpAG0PqGPQsx078FnrmrpIk56Kmqwh74QdiBRLDtVCAE1JMVZDJwynCJOllGOG9sAROXjYzZPPTTYhDmn118krI6ovH+FGjUu9\\\\\\\/peGeKOJenHNiGwzZ71pTiQeeS8ETe7DXuJOCO8gZnLbu9usSEJLbFeNRLGhP7tzkeE3YjZvYLWBVm2xtIAxZi2e\\\\\\\/QQs0kDksv7npVE0RqIZ9Fr5yr3iX12p9w4HsCFvzgJzROqBlDeV9nRaHEC4dhodJX7yoZL+xMgt+1JtMNGXWD7rJUaXcL3jhGVxfJ1l3y3AERiceyMFbADj3Ary2PzazJKRbI6dprFWgk\\\\\\\/CcF56adzEKFdxPOVTtI+8\\\\\\\/jsXamv5oecjeDVKEsRTb68Yc1twfR0YSTNlBycrvtW1sJJQI6IDlQdEmgSvDhz\\\\\\\/77UTXtxIMWWgidjI4NBMoEZRZWWNmXAXSLQnX\\\\\\\/U3jgD5FF3OOEaE9ytb0iGu63zHl\\\\\\\/7ZQ5OTqhTQMcWS\\\\\\\/ohNsf3Nd\\\\\\\/ZO69blQvbJZ+wIwJWLyj5ZnOPAq4jbARIyrek8EFdMlEEZAtTGlxTfwIprWRJ3XmHyQMPAX3LN4uZ0u9oKLC79dupywOpN7qmMf\\\\\\\/LMM5P8lINKQxKvGh7kNKepvOPeitTo4Ap3Y5PgogUHutHFxvzHtbIWhB+HeOvtRXqT9ecke2TDSM2C4vfkpmueWmwZFLZFO3k3rbAbi2IXaBucQQEht9AUYIPj6qWKbzFYADmPuTDozhZmtz9BFoRXXh9MZvLvwqNATKXTthrWdLkA3DAUMAGjuTw+PO9szIO6\\\\\\\/QvluZltC\\\\\\\/3XOwO5oQLIQip7K6J7RbL7c2SYVVOQX1oHqQu8PfCGdYuNUzht7c2q3oPHpPX6vm2wPWb6LCA4bAK54Ns6vKgYp\\\\\\\/WK9\\\\\\\/g\\\\\\\/p+1IvE2EP6hHqL6Mmf09z2hVu7jpWVlCqeqjIYT3YL3wdfRWEqHhhPz10QeeBi6pAxIQ8grw+BJlrgkGB3Mni5zTogPSC9yVebDOq8dGCHvulc+VkOSZkX3hBjJbw\\\\\\\/eEJ1AxWkmultI9LCRcdv6IF5L3MB5Uhn+m\\\\\\\/mfj8kIQt4hWIPaiexK5Zdw1ZmFWvG4aT9I5h+slHnp\\\\\\\/Uqlkl0oDyGrVSTBi7OhikEoBI1OvPQHGiPmHOM+I0hB1cOWTtpZ1pfEgwM9nU0oq4rLAJ056FuhrLcK0RblGewY1LEdrf1m1tOXWR7PGn\\\\\\\/84\\\\\\\/5RFLmQMSn29KgZST8h6QLi6Fmv0xpPr6BgKOMGOMJQFf6dM1WDp0wjHLqab6RpjMrLQUrRKbjwWTi29yDDiWK+vq3gz2PR5E+0zqtxz6Svibt8ET6kJPOjUflN7tmP9eg\\\\\\\/x7ic\\\\\\\/6noJ9dvkEE44pkB8JXOaYBhF9qn9ztffB6M4F59EFFCZ\\\\\\\/rKTrwdr1wB8ZoReKWFh+6w9GSvt8kuYGAyV8Pj4herOIU1cXkQ6A1xm6DrA3h89Mo2wpfpZAECeomjQr+m2uRu9hN+dUiBAHQONKWwcWcYkaQMR\\\\\\\/mWjLx7RxjbD\\\\\\\/53cdUXAQRybMHJSK4gaFDVaF2JylA53p7o6FJJ3uU\\\\\\\/kkd20BIJtYvT2K8Em5E\\\\\\\/KQC+729Zc9WJfV7koWafhdNhWg3OPjVzX4Un5Fnnbvf5QKMS1JduSmZlAMXTD3maa51Q4RyLwMqioqUqX9DcyF51fuzTX7wcJ8xmiXi4NlM83UQ4naQfewZQ6ES\\\\\\\/bLkSQG8khXNuYBc2BKLHkX4zjZIbQjInYTCXMa16dUUnUPAkRDIw+2tMtsX\\\\\\\/\\\\\\\/G4MPXQieXcZbhL6qG5Pqn9WEzW1Gnofyf2o8wj4qJ39Yklfra7w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/2hQAn6db\\\\\\\/ErX9uYSbuDcg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a6ba6499e1ec78ce536b6590a493ecf9771611a80ac25872367469df1f25ddbc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UtsLO7Hywp1vKwLgRrIZ1Q7vevPAFLq2PNqaOEkryC5SoG3OSj1bHj6OkDySD\\\\\\\/Y6Lm+6ZL7KF9OsyBQV3gx9QudEwvxQ0QqWYDxY+Tv+As06empw5FOutxtIoEEecYr6bHbvzGHrbZ+nFmNfRZeS347kZhJb4s7rqOqNz44oxtWvomowlGStiJkBZuIzwJ4UAd\\\\\\\/NgpLAcdrTqoe7JVeXQUY\\\\\\\/fQD8HlJdrMxrV4823Z0NCZ31Sh39tvQR8RivyF7u64nZu3XmzgrRDw6UMwGiN03KjkdQ22sYeoGjgedU6Od1VG07JgJgav3sKX3Q4fw2r+eeVGwiGjo1VemEk73UTrjgu5CrY28PJDophontGjI+tFv58uIPitY0mgqHc7gkE7yZpYSpAMgWupZgnNKt8VW4hhZctDb3WnGhWhYz5dD0cYoK8l1Z\\\\\\\/YZt0hsVQsO7UJlKOMYzkg6dNhtenA34hDbZmJjPJDQQwQRF04PHGaRETb2LMO8cj+zb6b2vNfo7\\\\\\\/3iGElq45mg\\\\\\\/1pXIKnzEHiL3iBy1H7IAMIC6MIUCVnZnOukNTB86kO9pBjuMcqQpP1YDmkjkfiQlCabGqIH83+G8feu\\\\\\\/bZPaL\\\\\\\/Fsb0+TcnWCeIZt5\\\\\\\/sz7vYTOGZYVjXlOqMClZ60PhY2WBODD7tN7NUChz+VCUpKjqYyAqs0V1I8Y8JGVTfvVwrdftpCJMlQNubAZYyDtYtu+yfOudXJNcENVremxkfLns2X4FoTFAfTXSESzdBzlLaq0szcCM2T32IhoPWKkiz2ycMxpL7M2YGdTsXiPQccuizFjhjlidXq9Mwjj3aqBznEhYUCnUarpI2uf189cv6YB1b1x6J\\\\\\\/MBTjh\\\\\\\/GMrVcrFqsTsfGZ1sY+66Ee4bekHeMhgsqw4i5qQ7wmOSFSg\\\\\\\/dyszCXmth30Swz6+ReghxiBnDdC5hJAdQ95xDcbdtSNJDx9eLCUW6Wp5iz6iiNNU3km2xuEH5KnYI12V1hRF\\\\\\\/vE2c6Jp\\\\\\\/VfRpPlf4Aqdku+x+tDTMZTC39pEJ5lMT5jZA2BNSu1S9kPKpg\\\\\\\/hGa\\\\\\\/MY0Rzw9bVvMuTFVBPozASyjwfpexiGvlscR\\\\\\\/0BN9t9uULST3EuW0r0AgrvMpoRUOYK87WAT+SPbGmQQjDHE+5IegLI8oBk84eQjFsAIElXDJITEN0BMt3vsCuxUC+cYeNFw4owVhtsTeMASLD70OPcO7oIoQsrtCloOHhCquQeI+c4FrAuqTG7KEjDagayAf3N3BzMbqNjoyWDsrgFCpXEW5eNnLvgBz9n1ZbiKZBEqjfOHDnj2ckk2XJKd4W\\\\\\\/lNlwTGanDG74mQ9Z0ULSVqxN8GoNKlG9be\\\\\\\/ogXry5oZdzKCC9xuwySBIdmq0cwXTHTR3f2EueR5BUIUfCHLZL0NXzvT4h+Kz3Ry5CjCtvxlzWN5CrtfN6yqal5ndkcO5r6Z\\\\\\\/ZRzR8DZSz6e8Ag3ExkkVohH0CULJKzyVWUmZc2t2ROlGPazVGLeTmlVDiTLG0PKpsn\\\\\\\/WwQji26+V22eAKzEBQmjELvK6DuObyU0cCZCOFWJEw8CbMTExvcjxDYTS8yFp0IemmA6MmQ9C57tmYC0j4DxBxRK\\\\\\\/Osphdn6DHNPdNhrZjcTpJPbggZnKwZcSQEt0SpgsF4fLITIFUHR73Dta+mQGD64EzCDh1nOWJxyoCAdYSg5IeiHH8PJNIvIJMLE79DIY1yNwG3M9BsqIDfWlvWNjMq899OR6hwy37k5ltb8GVekvWDuZ3\\\\\\\/Xzp3Vw2ubwkUDbqnpDOi\\\\\\\/o4QF0tPdtKofqjOtHhsg85fVDDJh6Wo4uUo6ZbJHnOsOUYJUxdFDSu2ScgcA9kOCW0EGd0FARPq6VmZNSPB7\\\\\\\/UDqTLOqnbOBLneU7ABRL46f3n6W+SnyBm6xiG5al92dNmROip3hX0PliEqWitqWfCKvWH4ug\\\\\\\/ht6suXSLyi2nK66aOuDTi1xG3Ko0LZERfmfUzZRLvvEL7jAiM04FBN9gqIafZROsPoUoDkQmJhy9ZQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00226Jy24xfjx+nMSz5lSP5Uig==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e5ef696cbecfc353af8ea954d75c5ac43e5afbac08cf19db0c72cd9982e8fc90\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489136 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1183" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6CSU18vQdkhkg3MNsp7WPbVmNOZ567ijKK0Jg_6WsOk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1JAYLG6F7HWciDZbKR1-TUQvS0NePIoTQ5K_AFVe0C4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1173" title="Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1195" title="Genome-Wide Association and Genomic Selection for Resistance to Amoebic Gill Disease in Atlantic Salmon" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489136" id="node489136" data-pisa="ggg;8/4/1183" data-pisa-master="ggg;g3.118.200053" data-apath="/ggg/8/4/1183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Predictions and Genome-Wide Association Study of Resistance Against <em>Piscirickettsia salmonis</em> in Coho Salmon (<em>Oncorhynchus kisutch</em>) Using ddRAD Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2813-4559" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Agustín Barría</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0021-0326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kris A. Christensen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6788-7369" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Grazyella M. Yoshida</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1386-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katharina Correa</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0626-1247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ana Jedlicki</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9157-4231" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean P. Lhorente</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3002-3225" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William S. Davidson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6612-4087" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José M. Yáñez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1183-1194;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200053 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489136" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Agustín Barría </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Doctorado en Acuicultura, Programa Cooperativo Universidad de Chile, Universidad Católica del Norte, Pontificia Universidad Católica de Valparaíso, 8820808 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agust%C3%ADn%2BBarr%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%C3%ADa%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAgust%25C3%25ADn%2BBarr%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2813-4559" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Agustín Barría</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kris A. Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kris%2BA.%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKris%2BA.%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0021-0326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kris A. Christensen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Grazyella M. Yoshida </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Animal Science Department, Universidade Estadual Paulista “Júlio de Mesquita Filho”, Faculdade de Ciências Agrárias e Veterinárias, Campus Jaboticabal, Jaboticabal 14884-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grazyella%2BM.%2BYoshida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yoshida%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGrazyella%2BM.%2BYoshida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6788-7369" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Grazyella M. Yoshida</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katharina Correa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BCorrea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Correa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BCorrea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1386-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina Correa</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ana Jedlicki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BJedlicki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jedlicki%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BJedlicki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0626-1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ana Jedlicki</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean P. Lhorente </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BP.%2BLhorente%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lhorente%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BP.%2BLhorente%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9157-4231" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean P. Lhorente</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">William S. Davidson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BS.%2BDavidson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davidson%20WS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BS.%2BDavidson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3002-3225" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William S. Davidson</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">José M. Yáñez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Núcleo Milenio INVASAL, Concepción 4070386, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BY%C3%A1%C3%B1ez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Y%C3%A1%C3%B1ez%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BY%25C3%25A1%25C3%25B1ez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6612-4087" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José M. Yáñez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmayanez@uchile.cl" class="" data-icon-position="" data-hide-link-title="0">jmayanez@uchile.cl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/4/1183" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489136" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489136/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1183.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489136" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489136/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1183.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489136" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489136/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1183.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489136" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489136/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/4/1183.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489136" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489136/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Piscirickettsia salmonis</em> is one of the main infectious diseases affecting coho salmon (<em>Oncorhynchus kisutch</em>) farming, and current treatments have been ineffective for the control of this disease. Genetic improvement for <em>P. salmonis</em> resistance has been proposed as a feasible alternative for the control of this infectious disease in farmed fish. Genotyping by sequencing (GBS) strategies allow genotyping of hundreds of individuals with thousands of single nucleotide polymorphisms (SNPs), which can be used to perform genome wide association studies (GWAS) and predict genetic values using genome-wide information. We used double-digest restriction-site associated DNA (ddRAD) sequencing to dissect the genetic architecture of resistance against <em>P. salmonis</em> in a farmed coho salmon population and to identify molecular markers associated with the trait. We also evaluated genomic selection (GS) models in order to determine the potential to accelerate the genetic improvement of this trait by means of using genome-wide molecular information. A total of 764 individuals from 33 full-sib families (17 highly resistant and 16 highly susceptible) were experimentally challenged against <em>P. salmonis</em> and their genotypes were assayed using ddRAD sequencing. A total of 9,389 SNPs markers were identified in the population. These markers were used to test genomic selection models and compare different GWAS methodologies for resistance measured as day of death (DD) and binary survival (BIN). Genomic selection models showed higher accuracies than the traditional pedigree-based best linear unbiased prediction (PBLUP) method, for both DD and BIN. The models showed an improvement of up to 95% and 155% respectively over PBLUP. One SNP related with B-cell development was identified as a potential functional candidate associated with resistance to <em>P. salmonis</em> defined as DD.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/selective-breeding" class="hw-term hw-article-keyword hw-article-keyword-selective-breeding" rel="nofollow">selective breeding</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping by sequencing</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/oncorhynchus-kisutch" class="hw-term hw-article-keyword hw-article-keyword-oncorhynchus-kisutch" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Oncorhynchus kisutch</em></a></li><li class="kwd"><a href="/keyword/disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-disease-resistance" rel="nofollow">disease resistance</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-4">Chile is the largest producer of coho salmon (<em>Oncorhynchus kisutch</em>) globally, reaching about 160,000 tons in 2014, representing more than 90% of total production (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">FAO 2016</a>). However, the success and sustainability of this industry is constantly threatened by infectious diseases, including Salmon Rickettsial Syndrome (SRS). This disease is caused by <em>Piscirickettsia salmonis</em>, a gram-negative and facultative intracellular bacteria, which was isolated for the first time in Chile in coho salmon (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cvitanich <em>et al.</em> 1991</a>). Data from the Chilean National Fisheries and Aquaculture Service (Sernapesca) indicates that during the first half of 2016, 53% of the moralities ascribed to infectious diseases in coho salmon were associated with SRS (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sernapesca 2016</a>). To date, control measures and treatments for SRS are based on antibiotics and vaccines. However, both strategies have not had the expected effectiveness under field conditions (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rozas and Enríquez 2014</a>).</p><p id="p-5">Because of this, it is necessary to develop alternative strategies for the control of this disease (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yáñez <em>et al.</em> 2014a</a>). In this regard, breeding for enhanced disease resistance is a feasible and sustainable option to improve animal health, welfare and productivity (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stear <em>et al.</em> 2001</a>). A primary requisite for including disease resistance into a breeding program is the presence of significant additive genetic variation for the trait (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Falconer and Mackay 1996</a>). Commonly, data to evaluate resistance comes from experimental challenges carried out using siblings of the selection candidates (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ødegård <em>et al.</em> 2011</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Yáñez <em>et al.</em> 2014a</a>). Quantitative studies have estimated significant genetic variation for resistance against different pathogens in salmonid species (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Ødegård <em>et al.</em> 2011</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Yáñez <em>et al.</em> 2014a</a>). For instance, low to moderate heritabilities for resistance against <em>P. salmonis</em> in Atlantic salmon (<em>Salmo salar</em>) (h<sup>2</sup> = 0.11 to 0.41) (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yáñez <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yáñez <em>et al.</em> 2014b</a>) and coho salmon (h<sup>2</sup> = 0.16) (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yáñez <em>et al.</em> 2016a</a>) have been estimated.</p><p id="p-6">Marker assisted selection (MAS) can improve production traits in cases where the phenotypes are difficult to measure in the selected candidates (<em>e.g.</em>, disease resistance traits) and the total additive genetic variance explained by genetic markers is high (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hayes and Goddard 2010</a>). This methodology has been successfully applied for the improvement of resistance against the Infectious Pancreatic Necrosis Virus (IPNV) in Atlantic salmon, which is controlled by a major quantitative trait locus (QTL) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Houston <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Moen <em>et al.</em> 2015</a>). In the case of polygenic traits, genomic selection (GS) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meuwissen <em>et al.</em> 2001</a>) can significantly improve selection accuracy of breeding values compared to traditional selection, and therefore enhance the response of selection for disease resistance in salmonid species (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tsai <em>et al.</em> 2016</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vallejo <em>et al.</em> 2016</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">2017a</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bangera <em>et al.</em> 2017</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Correa <em>et al.</em> 2017</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yoshida <em>et al.</em> 2017</a>).</p><p id="p-7">Genotyping by sequencing (GBS) is an alternative for genotyping in cases when SNP panels are not available. This approach reduces the complexity of the genome, and can be used to identify thousands of markers without prior marker discovery efforts or a reference genome. Currently, several approaches of GBS have been developed, significantly reducing the cost and labor (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Peterson <em>et al.</em> 2012</a>). These methodologies have been widely used in salmonid species, to generate dense linkage maps (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gonen <em>et al.</em> 2014</a>), perform association studies to identify genomic regions involved in the resistance against pathogens (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Campbell <em>et al.</em> 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Palti <em>et al.</em> 2015b</a>) and generate SNPs resources (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Houston <em>et al.</em> 2012</a>).</p><p id="p-8">Double-digest restriction-site associated DNA (ddRAD) reduces DNA complexity by digesting DNA with two restriction enzymes (REs) simultaneously, without random shearing (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Peterson <em>et al.</em> 2012</a>). This approach has been widely used in genetic studies in aquaculture species (reviewed in <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Robledo <em>et al.</em> 2017</a>).</p><p id="p-9">In the present study, we used ddRAD sequencing to dissect the genetic architecture of resistance against <em>P. salmonis</em> in a farmed coho salmon population and identify molecular markers associated with the trait. Furthermore, GS models were used to evaluate the potential to accelerate the genetic improvement of resistance against <em>P. salmonis</em> in this coho salmon population by means of using genome-wide molecular information.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Coho salmon breeding population</h3><p id="p-10">The coho salmon population used in the present study came from a unique 2012 year-class population. This population belongs to a genetic improvement program that was established in 1997 and is owned by Pesquera Antares and managed by Aquainnovo (Puerto Montt, Chile). Further details about this breeding population, in terms of reproductive management, rearing conditions, fish tagging and breeding objectives are described by <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Yáñez <em>et al.</em> (2014c</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">2016a</a>) and <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dufflocq <em>et al.</em> (2016)</a>.</p></div><div id="sec-3" class="subsection"><h3>Experimental challenge</h3><p id="p-11">The experimental challenge against <em>P. salmonis</em> was performed as it is described in detail by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Correa <em>et al.</em> (2015a)</a> and <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Yáñez <em>et al.</em> (2016a)</a>. Briefly, 2,606 individuals, belonging to 107 maternal full-sib families and 60 paternal half-sib families, were challenged against <em>P. salmonis</em>. Prior to the experiment, each fish was tagged with a passive integrated transponder (PIT-tag), placed in the abdominal cavity for genealogy traceability during the challenge test. The <em>P. salmonis</em> challenge was performed at Aquainnovo´s research station, located in Lenca River, X Region, Chile.</p><p id="p-12">For the lethal dose 50 (LD<sub>50</sub>) calculation, a random sample of 80 fish were selected from the population. Four different dilutions from the <em>P. salmonis</em> inoculum were evaluated (1/10, 1/100, 1/1000 and 1/10000). Twenty fish were challenged at each dilution. The dilutions were intraperitoneally (IP) injected with a volume of 0.2ml/fish. Daily mortality was recorded. This preliminary test spanned 26 days and a dilution of 1:680 was estimated as the LD<sub>50</sub>.</p><p id="p-13">For the main challenge, fish were distributed into three tanks (7m<sup>3</sup>) with a salt water concentration of 31 ppt. An average of eight individuals (ranging from 1 to 18) from each of the 107 families were distributed into each tank. The experimental challenge was performed through an intraperitoneal (IP) injection with 0.2ml/fish of the LD<sub>50</sub> inoculum. The average weight of the fish at the inoculation was 279g (SD = 138g). To ensure that these fish were free from other pathogens, qRT-PCR was previously performed in order to control for the presence of Infectious Salmon Anemia Virus (ISAV), IPNV and <em>Flavobacterium spp</em>.</p><p id="p-14">The <em>P. salmonis</em> challenge continued for up to 50 days post IP injection. Throughout the challenge, environmental parameters (pH, temperature, salinity and oxygen) were measured and controlled. Fish were removed from the tanks after death, and a sample of the anterior kidney was taken and stored at -80° in RNALater. A necropsy assay was performed in conjunction with qRT-PCR to confirm the cause of death and the presence of <em>P. salmonis</em>. This was also done to control for the presence of other pathogens, such as <em>Vibrio ordalii</em>, <em>Renibacterium salmoninarum</em> and IPNV.</p></div><div id="sec-4" class="subsection"><h3>ddRAD library preparation and sequencing</h3><p id="p-15">Ten ddRAD libraries were produced by multiplexing 828 individuals following the protocol described by <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Peterson <em>et al.</em> (2012)</a>. For this, 64 parents (males and females) and 764 offspring representing the 17 most resistant and 16 most susceptible families were selected. An average of 23 (ranging from 11 to 43 individuals) offspring per family were chosen. Briefly, total DNA was extracted using the commercial kit Wizard SV Genomic DNA purification System (Promega) according to the manufacturer’s protocol. Between 80 and 200 ng of DNA, from each individual was digested with two restriction enzymes (New England Biolabs, UK; NEB); 10 U of <em>Sbf</em>I (specific for the CCTGCA|GG motif)) and <em>Mse</em>I (specific for the T|TAA motif) in a 12 µl reaction volume, including 1 µl of <em>Sbf</em>I and <em>Mse</em>I adapter (8.3 pM), for 90 min at 37°. The ligation reaction was carried out by adding 1 µl of T4 ligase (NEB) diluted 1:100 in T4 buffer and incubating for 150 min at 37° and subsequently at 16° overnight.</p><p id="p-16">Each ligation mix was diluted with 189 µl of dilute TE buffer (1:10). Kodak DNA Polymerase (ABM), a high-fidelity polymerase, was used to amplify DNA fragments with the correct adapters. PCR reactions (20 µl) were prepared containing 10 µl of PCR mix 2x, 1 µl of primer mix (10 µM each), 6 µl of diluted ligation mix and 3 µl of nuclease-free water. Each sample was PCR amplified using the following conditions: 95° for 2 min, followed by 17 cycles of 95° for 20s, 66° for 30s and 68° for 40s. After PCR, amplicon quality was checked by loading 5 µl on a 2% agarose gel. Subsequently, samples were pooled, so that the final concentration was similar among them within each library. Each library was concentrated through an evaporation step for 80 min in a Centrivap Mobile Console Centrifugal Evaporator (Labconco). This step was conducted until 300 µl of the generated library was obtained. Final volume of each library was loaded on a 1% agarose gel. Size of the bands selected for sequencing ranged from 750 and 1,500 bp and between 1,800 and 2,500 bp. DNA was purified through the QIAquick gel extraction kit (Qiagen) following manufacturer’s instructions. Finally, libraries were sequenced on an Illumina Hiseq2500 platform, using 150 base single-end.</p></div><div id="sec-5" class="subsection"><h3>SNP identification</h3><p id="p-17">Raw sequence reads obtained from Illumina sequencing were analyzed using STACKS v. 1.41 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2013</a>). This software was specifically developed to analyze short-read data generated through next generation sequencing (NGS) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Davey <em>et al.</em> 2013</a>).</p><p id="p-18">Sample reads were trimmed to 134 bp for all subsequent analyses, demultiplexed and filtered using process_radtags. Rad-tags which passed the quality filter were aligned to the Oncorhynchus kisutch reference genome (GenBank: MPKV00000000.1) using BWA v. 0.7.12 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li and Durbin 2009</a>). The reference genome was indexed (using the index function) and alignments were performed using the mem algorithm; all parameters were set as default. Loci were then built using pstacks with a minimum depth of coverage of three to build a locus (-m 3).</p><p id="p-19">A catalog of loci was constructed using the cstacks program using only the parents’ loci from pstacks. To build the catalog, the maximum number of mismatches allowed between sample tags was set to three (-n 3), and the matching was based on genomic location (g). After catalog construction, the sstacks program was used in order to match rad-tags against the catalog based again on genomic location (g), followed by the populations software, using defaults parameters. Loci were considered as valid if they were present in at least 75% of the individuals of the population. As a quality control step, the following parameters were used to filter low-confidence SNPs: Minor Allele Frequency (MAF) ≤ 0.05, Hardy-Weinberg Equilibrium (HWE) <em>P</em> &lt; 1x10-6 and genotyping call rate &lt; 0.80. Individuals were removed from the analyses if their genotyping call rates were below 0.70.</p></div><div id="sec-6" class="subsection"><h3>Trait definitions</h3><p id="p-20">Resistance against <em>P. salmonis</em> was defined as the day of death (DD) with values ranging from 1 to 50 depending on the time of death. Additionally, resistance was also evaluated as a binary (BIN) trait, either dead or alive at the end of the challenge. Values for this trait were 1 in cases where the fish died during the challenge, or 0 if the fish survived until the end of the challenge. Initial Body Weight (IW) for each fish, was measured prior to the IP injection.</p></div><div id="sec-7" class="subsection"><h3>Pedigree-based BLUP</h3><p id="p-21">All challenged individuals (n = 2,606) were used for the pedigree-based approach, PBLUP, as a control for the performance evaluation of genomic predictions. A linear univariate animal model was used to estimate variance components and predict Estimated Breeding Values (EBVs) for DD, while for BIN a univariate threshold animal model was fitted (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The model used was as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-3.gif"/></noscript></span> </span>In the previous equation, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-4.gif"/></noscript></span></span> is a vector of phenotypes (BIN or DD), <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-5.gif"/></noscript></span></span> is a vector of fixed effects (sex and tank as factors, and initial weight as covariate), <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-6.gif"/></noscript></span></span> is a vector of random additive polygenic genetic effects that follows a normal distribution ∼N(0,A<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-7.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-9.gif"/></noscript></span></span> are incidence matrices, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-10.gif"/></noscript></span></span> is the additive relationship matrix, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-11.gif"/></noscript></span></span> is the random residual (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lynch and Walsh 1998</a>). Both models were fitted using the BLUPF90 set of programs (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Misztal <em>et al.</em> 2016</a>) by means of the AIREMLF90 and THRGIBBS1F90 modules to analyze DD and BIN, respectively. The MCMC Gibbs sampling scheme set for running THRGIBBS1F90, included a total of 200,000 iterations. The first 20,000 were discarded as burn-in iterations, and then from the remaining 180,000 samples, one from every 50 samples was saved for analysis. This Gibbs sampling scheme collected 3,600 independent samples for analysis.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489151/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489151/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Estimated genetic parameters and accuracy of breeding values (EBV) estimation for resistance against <em>P. salmonis</em> using a pedigree-based model</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Heritabilities for PBLUP models were computed as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-12.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-14.gif"/></noscript></span></span> are the additive genetic and residual variances for each trait. In the case of BIN, the residual variance was set to 1.</p></div><div id="sec-8" class="subsection"><h3>Genomic BLUP</h3><p id="p-24">The SNP based variance components and GEBVs were estimated using genomic BLUP (GBLUP), similar to the PBLUP model described above, and implemented in the BLUPF90 software. The GBLUP is a modification of the PBLUP method, where <em>g</em> is a vector of random additive genetic polygenic effects with a distribution ∼N(0, G<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-15.gif"/></noscript></span></span>) and the numerator relationship matrix <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-16.gif"/></noscript></span></span> is replaced by a genomic relationship matrix <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-17.gif"/></noscript></span></span>, as described by (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">VanRaden 2008</a>). Only genotyped animals, which passed all quality controls (n = 580) were analyzed.</p></div><div id="sec-9" class="subsection"><h3>Single step genomic GBLUP</h3><p id="p-25">The single-step GBLUP (ssGBLUP) and weighted single-step GBLUP (wssGBLUP) models were similar to the PBLUP model except for a combined genomic and pedigree relationship. The kinship matrix used was H (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>), in which genotype and pedigree data are combined. The inverse of the matrix H is:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-18.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-19.gif"/></noscript></span></span> is the inverse numerator relationship matrix for all animals, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-20.gif"/></noscript></span></span> is the inverse of a pedigree-based relationship matrix for genotyped animals only, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-21.gif"/></noscript></span></span> is the inverse genomic relationship matrix. SNPs were equally weighted and given an initial value of one in the ssGBLUP method. In the wssGBLUP method, the marker variances were used as weights. The marker variances were estimated from allele frequencies, and from marker effects that were calculated in the ssGBLUP method (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wang <em>et al.</em> 2014</a>). The DD trait was analyzed as a linear trait using AIREMLF90 and BLUPF90, whereas, BIN was analyzed as a threshold trait with THRGIBBS1F90 in the BLUPF90 family of programs (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Misztal <em>et al.</em> 2016</a>). The MCMC Gibbs scheme for the estimation of the genetic parameters for BIN for ssGBLUP and wssGBLUP, were estimated identically as described above. The ssGBLUP and wssGBLUP models included all the genotyped animals which passed quality control (n = 580), and all the phenotyped fish (n = 2,606) from 107 families.</p></div><div id="sec-10" class="subsection"><h3>Bayes C</h3><p id="p-26">The Bayes C (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Habier <em>et al.</em> 2011</a>) analyses were performed using GS3 software. A total of 200,000 iterations were used in the Gibbs sampling, with a burn-in period of 20,000 cycles. The results were saved every 50 cycles. The number of samples in this analysis totaled 4,000. Convergence and autocorrelation were assessed by visual inspection of trace plots of the posterior variance components. The adjusted model can be described, in matrix notation, as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-22.gif"/></noscript></span></span>where y is the vector of phenotypic records (DD or BIN), X is an incidence matrix of fixed effects (sex and tank as factors and IW as covariate), β is the vector of fixed effects, T is an incidence matrix of polygenic effects, p is a random vector of polygenic effects of all individuals in the pedigree, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-23.gif"/></noscript></span></span> is the vector of the genotypes for the <em>i<sup>th</sup></em> SNP for each animal, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-24.gif"/></noscript></span></span> is the random allele substitution effect of the <em>i<sup>th</sup></em> SNP, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-25.gif"/></noscript></span></span> is an indicator variable (0, 1) sampled from a binomial distribution with parameters determined such that 1% of the markers were included in the model, and e is a vector of residual effects. The following prior distributions were assumed for the genetic random effects: Independent and identical mixture distributions for the SNP effects; each SNP has a point mass at zero having a probability π and a univariate normal distribution with a probability of 1 – π with null mean and variance <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-26.gif"/></noscript></span></span>; which in turn has a scaled inverse chi-squared prior with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-27.gif"/></noscript></span></span> degrees of freedom and scale parameter <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-28.gif"/></noscript></span></span> (or <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-29.gif"/></noscript></span></span>) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fernando and Garrick 2013</a>). The scale parameter was estimated as a function of the genetic variance population, based on the mean SNP allele frequency and number of markers assumed with nonzero effects (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fernando <em>et al.</em> 2007</a>). Only genotyped animals, which passed quality control (n = 580) were used.</p></div><div id="sec-11" class="subsection"><h3>Genomic prediction accuracy</h3><p id="p-27">The different models were compared using a fivefold cross validation scheme. To reduce stochastic effects of sampling, the cross-validation analysis was replicated ten times. Briefly, all challenged individuals (genotyped, phenotyped, or both), were randomly separated into five validations sets. For each set, predictions were made by masking the animals’ phenotypes and using the remaining fish as a training set to estimate the marker effects. Thus, for each cross-validation run, the dataset was split into a training set (80%) and a validation set (20%). Accuracy was used to assess the performance of each model and was estimated as follows:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-30.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-31.gif"/></noscript></span></span> is the correlation between the EBV of a given model (predicted for the validation set using information from the training set) and the actual phenotype, while <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1183/embed/mml-math-32.gif"/></noscript></span></span> is the square root of the pedigree-based estimate of heritability (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Correa <em>et al.</em> 2017</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Legarra 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Palaiokostas <em>et al.</em> 2016</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Tsai <em>et al.</em> 2016</a>). Finally, accuracies were calculated for each model and compared to those obtained with the PBLUP model.</p></div><div id="sec-12" class="subsection"><h3>Genome-wide association study</h3><p id="p-28">In order to identify associations between genetic markers and <em>P. salmonis</em> resistance, as DD or BIN, four genome-wide association methodologies were performed using the BLUPF90 set of programs (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Misztal <em>et al.</em> 2016</a>). GBLUP, ssGBLUP, wssGBLUP and w3ssGBLUP models were used to analyze the DD and BIN traits, using a linear and a threshold model respectively (as described above in model 1). For the GBLUP model, the pedigree-based relationship matrix (A) was replaced by a genomic matrix (G). For the ssGBLUP, wssGBLUP and w3ssGBLUP GWAS models the H matrix was used as described above. SNPs were weighted equally (and given a weight of one), for the ssGBLUP model. For the wssGBLUP and w3ssGBLUP methods, weights were determined based on individual marker variances that were estimated using both the marker effects, which were calculated previously for the ssGBLUP model, and marker allele frequencies (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Wang <em>et al.</em> 2014</a>). For the GBLUP GWAS model, only genotyped animals, which passed quality control (n = 580), were analyzed. For the ssGBLUP, wssGBLUP and w3ssGBLUP GWAS models, all the genotyped animals passing quality control (n = 580), and all the phenotyped fish (n = 2,606) were used. Due to practical reasons, the molecular markers anchored to the different scaffolds not placed in chromosomes, were assigned as chromosome 31. The parents of the challenged individuals were not included in the GWAS analysis because they did not have associated phenotype information, as they were not submitted to the challenge experiment.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-29">Table S1 contains genotypic data (available at the public dryad digital repository <a href="https://doi.org/10.5061/dryad.b273q6p">https://doi.org/10.5061/dryad.b273q6p</a>), <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1/TableS2.txt">Table S2</a> contains phenotypic data, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1/TableS3.txt">Table S3</a> contains the pedigree information. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1/TableS4.docx">Table S4</a> contains the full list of genes located within the top ten 1-Mbp windows proximate to each SNP associated with P. salmonis resistance for DD and BIN identified through wssGBLUP. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1/TableS5.xlsx">Table S5</a> contains information of the top ten markers which explain the highest percentage of the genetic variance for each method and trait. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1/TableS6.xlsx">Table S6</a> contains results from the 10 replicated CV.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Challenge test</h3><p id="p-30">Mortality began on the 10<sup>th</sup> day after the <em>P. salmonis</em> challenge, with evident symptoms of SRS and pathological lesions typical of SRS. These signs include swollen kidney, splenomegaly and yellowish liver tone and coloration (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rozas and Enríquez 2014</a>). Challenged families showed considerable phenotypic variation for <em>P. salmonis</em> resistance. Average mortality of all the 107 families reached 38.53% during the 50-day challenge. The average cumulative mortality rate among the 17 best and 16 worst families, selected for genotyping, reached 19% and 63%, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Kaplan-Meier curves for Piscirickettsia salmonis experimental challenge in coho salmon. Average mortality curves for the 107 full-sib families, and the 17 best and 16 worst families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-895484190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Kaplan-Meier curves for &lt;em&gt;Piscirickettsia salmonis&lt;/em&gt; experimental challenge in coho salmon. Average mortality curves for the 107 full-sib families, and the 17 best and 16 worst families.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1183/F1.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/4/1183/F1.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489211" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Kaplan-Meier curves for <em>Piscirickettsia salmonis</em> experimental challenge in coho salmon. Average mortality curves for the 107 full-sib families, and the 17 best and 16 worst families.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>ddRAD sequencing</h3><p id="p-32">Prior to quality control (QC), per base quality (Phred score) was evaluated. The average quality score ranged from 36 to 38 among libraries, indicating high quality of data. Illumina sequencing, including parents, yielded an average of 156,058,078 (± 16 million) of raw sequences per library. After initial QC, which included the removal of low quality sequences and reads with either missing or ambiguous barcodes, an average of 31,660,024 of the reads were removed. In parallel with QC, reads were trimmed to 134 bp. Thus, 79% of the raw reads were retained for subsequent analysis. To create a set of all possible alleles in the population, data sets of the parental samples were used to create a STACKS catalog. This catalog consisted of 106,309 unique ddRAD loci from which 20,068 markers from 757 individuals were identified. Quality filtering reduced this to 9,389 putative bi allelic SNPs (see Table S1) with an average sequencing depth of 38x ranging from 11x to 501x. These markers were identified segregating along the genome of 580 individuals (see Table S1).</p></div><div id="sec-17" class="subsection"><h3>Genome-wide association analysis</h3><p id="p-33">Four genome-wide association methodologies were performed either for DD and BIN. These approximations include GBLUP, ssGBLUP, wssGBLUP and w3ssGBLUP. For both traits, all the models showed a similar association pattern. In the case of the ssGBLUP methodology, the GWAS plots become less noisy as the iterations progress, and the peaks associated with the traits become more distinct (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For DD, a marker potentially associated with <em>P. salmonis</em> resistance was located on chromosome 11 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). This marker was identified in all of the four models and was within the top ten markers explaining most of the percentage of the genetic variance (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic association analyses for resistance against Piscirickettsia salmonis in a coho salmon population (defined as day of death) for four different models; GBLUP (A), ssGBLUP (B), wssGBLUP (C) and w3ssGBLUP (D). The gray area highlights the SNPs (in green), which are among the top ten markers explaining a high percentage of the genetic variance in the four models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-895484190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic association analyses for resistance against &lt;em&gt;Piscirickettsia salmonis&lt;/em&gt; in a coho salmon population (defined as day of death) for four different models; GBLUP (A), ssGBLUP (B), wssGBLUP (C) and w3ssGBLUP (D). The gray area highlights the SNPs (in green), which are among the top ten markers explaining a high percentage of the genetic variance in the four models.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1183/F2.medium.gif" width="175" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/4/1183/F2.medium.gif" width="175" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489212" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Genomic association analyses for resistance against <em>Piscirickettsia salmonis</em> in a coho salmon population (defined as day of death) for four different models; GBLUP (A), ssGBLUP (B), wssGBLUP (C) and w3ssGBLUP (D). The gray area highlights the SNPs (in green), which are among the top ten markers explaining a high percentage of the genetic variance in the four models.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic association analyses for resistance against Piscirickettsia salmonis in a coho salmon population (for survival as a binary trait) for four different models; GBLUP (A), ssGBLUP (B), wssGBLUP (C) and w3ssGBLUP (D). The gray area highlights SNPs (in green) that were among the top ten markers explaining a high percentage of the genetic variance in the four models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-895484190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic association analyses for resistance against &lt;em&gt;Piscirickettsia salmonis&lt;/em&gt; in a coho salmon population (for survival as a binary trait) for four different models; GBLUP (A), ssGBLUP (B), wssGBLUP (C) and w3ssGBLUP (D). The gray area highlights SNPs (in green) that were among the top ten markers explaining a high percentage of the genetic variance in the four models.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1183/F3.medium.gif" width="180" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/4/1183/F3.medium.gif" width="180" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489244" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Genomic association analyses for resistance against <em>Piscirickettsia salmonis</em> in a coho salmon population (for survival as a binary trait) for four different models; GBLUP (A), ssGBLUP (B), wssGBLUP (C) and w3ssGBLUP (D). The gray area highlights SNPs (in green) that were among the top ten markers explaining a high percentage of the genetic variance in the four models.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489271/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489271/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Top ten markers associated with <em>Piscirickettsia salmonis</em> resistance defined as DD and BIN in coho salmon, using wssGBLUP method</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">The availability of a high quality coho salmon reference genome (GenBank accession number MPVK00000000.1), made it possible to identify genes near this marker. Within ∼55 Kbp of this marker is the <em>phosphoinositide-3-kinase adaptor protein 1</em> (pik3ap1) gene; a gene related with innate host defense through B-cells development (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aiba <em>et al.</em> 2008</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Herzog <em>et al.</em> 2009</a>).</p><p id="p-43">In the case of BIN, two molecular markers, explaining most of the genetic variance, were identified in all of the four models (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). One of these markers is located on chromosome 29, while the second marker was identified only at a scaffold level (Scaffold04124) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). For BIN, host immune response related genes were not found proximate to any of the suggestive molecular markers.</p><p id="p-44">However, some genes within a 1-Mb window of these markers have been suggested to be involved with <em>P. salmonis</em> infection. The <em>14 KDa Phosphohistidine phosphatase-like</em> (PHPT1), <em>gelsolin-like</em> (GSN), and <em>glutamine synthase-like</em> (GS) genes are located on chromosome 29 and near associated markers. <em>Claudin-10</em> was found on scaffold04124. These genes have been previously identified as being up-regulated in <em>Salmo salar</em> individuals with low susceptibility to <em>P. salmonis (</em><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52"><em>Pulgar et al. 2015</em></a><em>)</em>. Moreover, the <em>retinoic acid receptor RXR-alpha-A-like</em> (RXRA) gene, located on chromosome 29, has previously been identified as a molecular biomarker for <em>P. salmonis</em> infections, and has been found to be down-regulated in macrophages during infection (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rise <em>et al.</em> 2004</a>).</p><p id="p-45">A full list of genes that are located within a 1-Mbp window proximate to the suggestive markers associated with <em>P. salmonis</em> resistance for Okis11, Okis29 and at a scaffold level, identified through wssGBLUP model, is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1/TableS4.docx">Table S4</a>. In the case of the marker located on the scaffold, the surrounding sequence was blasted against the <em>Salmo salar</em> reference genome (NC_027300.1).</p></div><div id="sec-18" class="subsection"><h3>Genetic parameters and predictions</h3><p id="p-46">Significant additive genetic variation was estimated for both DD and BIN when using all the data from challenged individuals from the 107 maternal, full-sib families (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Using the pedigree-based model without genomic data, estimates of the narrow sense heritability for DD and BIN were equal to 0.14 (± 0.034) and 0.27 (± 0.043), respectively.</p><p id="p-47">Based on a fivefold cross validation, the accuracy of the PBLUP model was slightly lower for DD (0.271) than for BIN (0.316) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). When genomic data were included, accuracies for DD and BIN were higher than those achieved using only phenotypic data. However, there is considerable variation between models and trait definitions. The accuracies for the different models ranged from 0.299 (ssGBLUP) to 0.529 (GBLUP) for the DD trait, and from 0.314 (ssGBLUP) to 0.807 (GBLUP) for the BIN. For DD, all the models with genomic data outperformed the pedigree-based model. The relative increase in accuracy ranged from 10 (ssGBLUP) to 95% (GBLUP) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). In the case of BIN, the relative increase in accuracy ranged from 20 (wssGBLUP) to 155% (GBLUP). However, one of the genomic models (ssGBLUP) had a similar accuracy to the PBLUP model, with a relative accuracy 1% lower than the pedigree-based model.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of predicted accuracies (R) for Piscirickettsia salmonis resistance in a coho salmon population comparing between PBLUP and models with genomic data for DD (red bars) and BIN (blue bars)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-895484190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of predicted accuracies (R) for &lt;em&gt;Piscirickettsia salmonis&lt;/em&gt; resistance in a coho salmon population comparing between PBLUP and models with genomic data for DD (red bars) and BIN (blue bars).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1183/F4.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/4/1183/F4.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1183/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489293" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Comparison of predicted accuracies (R) for <em>Piscirickettsia salmonis</em> resistance in a coho salmon population comparing between PBLUP and models with genomic data for DD (red bars) and BIN (blue bars).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Interestingly, the accuracies obtained with GBLUP were higher than ssGBLUP and wssGBLUP for both traits. These accuracy values reached 0.529, 0.299 and 0.417 for DD and 0.807, 0.314 and 0.3797 for BIN. For both traits, the models with better performance were, GBLUP &gt; Bayes C &gt; wssGBLUP &gt; ssGBLUP.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-50">Significant genetic variation for <em>P. salmonis</em> resistance was detected in the present study. Moderate heritabilities were estimated using different trait definitions, either for DD or BIN. Estimated heritabilities were higher for resistance as a binary trait when compared to DD.</p><p id="p-51">Previously, <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Yáñez <em>et al.</em> (2016a)</a>, estimated a heritability of 0.16 for resistance against <em>P. salmonis</em>, defined as day of death (from the same coho salmon population) through a bivariate linear model. Our study estimated a similar heritability value (0.14) in the same population. Differences in the estimations are likely due to the univariate model we used instead of a bivariate model. We also estimated heritability for <em>P. salmonis</em> resistance using a threshold model for the binary trait, which was higher (0.27) than the value reported for DD. These results are consistent with previous findings for resistance against <em>P. salmonis</em> in Atlantic salmon using pedigree information (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Yáñez <em>et al.</em> 2013</a>).</p><p id="p-52">When resistance, defined as day of death, was analyzed using a linear model, heritability was estimated as 0.18 (0.03). When using a threshold model to analyze resistance as a binary trait, a heritability of 0.24 (0.04) was calculated (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Yáñez <em>et al.</em> 2013</a>, <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">2014b</a>). The genetic variation and heritability values for <em>P. salmonis</em> resistance are in accord with different studies that have also found significant genetic resistance to other bacterial diseases in salmonid species (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gjøen <em>et al.</em> 1997</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ødegård <em>et al.</em> 2006</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Vallejo <em>et al.</em> 2016</a>).</p><p id="p-53">Bacterial disease resistance has been suggested to be a polygenic trait in aquaculture species. For example, <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Palaiokostas <em>et al.</em> (2016)</a> suggested that resistance against <em>Photobacterium damselae subsp</em>. has a polygenic architecture in Gilthead Sea Bream (<em>Sparus aurata</em>). Using a 50K SNP genotyping array, it was possible to elucidate a moderately polygenic architecture of <em>P. salmonis</em> resistance in Atlantic salmon (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Correa <em>et al.</em> 2015b</a>). In the current study, and using 9K SNPs, a similar genetic architecture for resistance against <em>P. salmonis</em> in coho salmon population was found. However, it is likely that the moderate number of individuals could limit the power to detect QTL of larger effect controlling <em>P. salmonis</em> resistance in coho salmon.</p><p id="p-54">Among the top ten genetic markers for DD and BIN, one and two markers were identified among all four models, respectively. The availability of an annotated, coho salmon genome made it possible to identify the <em>phosphoinositide-3-kinase adaptor protein 1</em> (pik3ap1) (also known as the <em>adaptor protein B-cell PI3K adaptor</em> (BCAP)), a gene that is related with B-cell development (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Herzog <em>et al.</em> 2009</a>), proximate to the genetic marker found to be associated with resistance. The role of B-cells, through the humoral response, has been widely investigated and elucidated (reviewed in <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Janeway <em>et al.</em> 2001</a>). B-cells may also have phagocytic activity in both rainbow trout (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2006</a>) and Atlantic salmon (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Øverland <em>et al.</em> 2010</a>). These cells are capable of ingesting large particles and bacteria; killing them through phagolysosome fusion (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2006</a>). Studies <em>in vitro</em>, showed that from the total phagocytic leukocytes isolated from Atlantic salmon head kidney, 37% were B-cells. Additionally, 77% were B-cells when leukocytes were isolated from peripheral blood (PB). The phagocytic ability of B-cells was three times higher than those observed in neutrophils in head kidney, while in PB no differences were observed (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Øverland <em>et al.</em> 2010</a>).</p><p id="p-55">We hypothesize that B-cell development could help in the immune response against <em>P. salmonis</em> in coho salmon population through its phagocytic activity. Further studies are needed in order to have a better understanding of the role of these cells in the resistance against <em>P. salmonis</em>, and its ability to digest and kill bacteria. For resistance defined as the BIN trait, no genes related with immune response were identified near genetic markers associated with this trait. A possible explanation of this observation, is that the region near the genetic marker acts as a regulatory sequence. Also, it could be due to the relative small sample size for the GWAS, which ideally should be over 1000 animals (limiting the resolution of the GWAS).</p><p id="p-56">The underlying genetic basis for this trait may have an important impact on the accuracy of genomic predictions. Thus, performance comparisons of different algorithms is required for either GS and GWAS when a complex trait is studied for the first time within a population (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vallejo <em>et al.</em> 2017b</a>). The current study is the first to evaluate the genetic architecture of resistance against <em>Piscirickettsia salmonis</em> in coho salmon through different algorithms (to our knowledge). We used a genomic model that assumes that genetic variances are controlled by an infinite number of markers with minimum effects on the trait (<em>i.e.</em>, GBLUP). The GBLUP method calculates genomic relationship (G matrix) using all the genotyped markers for this reason. An extension of this method, which combines genomic (G) and pedigree-based (A) relationship information into the H relationship matrix (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Legarra <em>et al.</em> 2014</a>) was also evaluated. This method, called single-step GBLUP (ssGBLUP), mimics a Bayesian selection model in that it only fits SNPs that explain moderate to large genetic variance of the trait (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wang <em>et al.</em> 2012</a>). Results of these different genomic-wide association analyses suggest that resistance against <em>Piscirickettsia</em> salmonis has a polygenic architecture, with no major QTL (<em>i.e.</em>, explaining &gt;= 10% of the genetic variation) segregating in the current population (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-57">When resistance was defined as DD, the accuracy for the PBLUP model was slightly lower than resistance measured as BIN (0.271 and 0.316, respectively). These values are consistent with the results obtained for resistance against the bacterial disease pasteurellosis in <em>S. aurata</em> defined as day of death, authors reached an accuracy up to 0.30 through a pedigree-based model (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Palaiokostas <em>et al.</em> 2016</a>). However, both accuracy values are slightly lower when compared with values obtained for resistance against sea lice in Atlantic salmon. In this regard, <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Correa <em>et al.</em> (2017)</a> reached an accuracy of 0.41 for resistance against <em>Caligus rogercresseyi</em>, while <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Tsai <em>et al.</em> (2016)</a> reached a prediction accuracy ∼0.5 for resistance against <em>Lepeophtheirus salmonis</em> using PBLUP.</p><p id="p-58">The results from our genomic predictions are in agreement with previous studies that showed higher estimated accuracies using GS than with PBLUP in the same half/full-sib family structure in salmon breeding programs (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nielsen <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lillehammer <em>et al.</em> 2013</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bangera <em>et al.</em> 2017</a>).</p><p id="p-59">The GS model that showed the best performance in terms of accuracy of predictions for DD was GBLUP. This was followed by Bayes C, wssGBLUP and ssGBLUP. The addition of genomic information allowed an improvement up to 95% in accuracy for this trait. The current study showed a relative accuracy prediction improvement of 54% when comparing PBLUP to wssGBLUP for this trait. This improvement is different to that obtained by <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Vallejo <em>et al.</em> (2016)</a> for resistance against Bacterial Cold Water Disease (BCWD), defined as day of death, in rainbow trout (<em>Oncorhynchus mykiss</em>). The authors reported a reduction in the predictive ability (PA) of 20 and 26% using wssGBLUP, either with a chip array (40K) or through RAD sequencing (10K) respectively. The authors attributed this pronounced reduction to stochastic fluctuations due to a small training group of their study (n = 583). However, when the number of individuals was increased, this model outperformed the pedigree-based model, reaching a relative increase in accuracy up to 108%, for the same trait definition (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Vallejo <em>et al.</em> 2017a</a>).</p><p id="p-60">In the case of BIN, the use of the ssGBLUP model, did not show an improvement in the accuracy prediction (with a reduction of 1% in comparison to the accuracy obtained though the pedigree-based model). However, there was an estimated relative increase in accuracy ranging from 20 to 155% when comparing PBLUP to the others genomic models. In this regard, Bayes C and GBLUP showed an increase in accuracy of 140% and 155%, respectively. These high values are similar to the improvements seen with resistance against BCWD defined as a binary trait, which reached a relative increase up to 97% (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Vallejo <em>et al.</em> 2017a</a>).</p><p id="p-61">The relative improvements ranged in accuracy from -1 to 155% for BIN and from 10 to 95% for DD are greater to those obtained for other diseases resistance studies in Atlantic salmon; even with lower marker numbers. Using an identical random selection design as in the current study, sea lice resistance showed a relative improvement in accuracy of 22% relative to PBLUP when using 37K SNPs (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Correa <em>et al.</em> 2017</a>). For reliability, an improvement up to 52% with 220K SNPs was reached (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ødegård <em>et al.</em> 2014</a>). <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Tsai <em>et al.</em> (2016)</a> reported that when a non-full siblings design was used, an improvement in accuracy of 250% and 500% was reached in two different populations compared to PBLUP. However, when the methodology was changed to a random selection scheme this improvement only reached up to 27% using 33K SNPs.</p><p id="p-62">In case of <em>Piscirickettsia salmonis</em> resistance, and using the same cross-validation scheme in our study, the relative reliability was increased by 25% and 30% for resistance, defined as day of death or as a binary trait, respectively with 50K SNPs (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bangera <em>et al.</em> 2017</a>).</p><p id="p-63">In the current study we evaluated a wide range of different models of GS for their potential implementation in aquaculture. The performance of each implemented model varied according to the underlying genetic architecture of the trait (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daetwyler <em>et al.</em> 2010</a>). Thus, it is valuable to perform these comparisons to identify the best performing method using real data. Similar accuracies among PBLUP and ssGLBUP models could be due to the predicted GEBV as both models are overrepresented by polygenic EBV (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bangera <em>et al.</em> 2017</a>). GBLUP estimated genetic relationships using genotype and pedigree data rather than just average relationship as PBLUP (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Habier <em>et al.</em> 2007</a>). This allows a more accurate genetic relationship matrix and provides an increase in performance, as seen in the current data, due to the close family relationship. Moreover, GBLUP had significantly better performance when resistance was defined as a linear or binary trait compared to the other evaluated genomic models. We suggest that this could be an effect of only genotyping families from the opposite sides of the mortality distribution (<em>i.e.</em>, most resistant and most susceptible), and not all the challenged individuals. <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bangera <em>et al.</em> (2017)</a>, reported similar accuracies among GBLUP and Bayesian methods between GBLUP and Bayes C using 10K SNPs in Atlantic salmon. Similar results that have been seen in dairy cattle for most traits (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">de Roos <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goddard 2009</a>)</p><p id="p-64">The greater improvement in accuracy for the binary trait BIN, compared with the linear trait DD, could be due a better fit of the threshold model for BIN than the fit of the linear model for DD. It could also be due to the higher estimated heritability.</p><p id="p-65">We hypothesize that the large improvement values seen in the current study are likely due to an increased level of linkage disequilibrium (LD) found within this farmed coho salmon population. Additional studies are needed to elucidate the minimum number of markers necessary for GS.</p><p id="p-66">Our results, using ddRAD sequencing, are in agreement with other genomic studies, which utilize GBS techniques with aquaculture species. Using RAD sequencing, some authors have previously performed genomic-wide association studies in rainbow trout looking for associations with disease resistance. From a total of 4K identified SNPs, 31 markers were significantly associated with either BCWD or Infectious Hematopoietic Necrosis Virus (IHNV) resistance as a binary trait (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Campbell <em>et al.</em> 2014</a>). These authors also showed the potential of these RAD markers to predict an animal’s phenotype. In the case of BCWD resistance, defined as a linear and binary trait, <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Palti <em>et al.</em> (2015b)</a> identified suggestive and significant SNPs in two different families and candidate genes associated with this trait using ∼5K markers per family. Similar numbers of SNPs were used by <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> (2015)</a> to significantly associate 18 SNPs.</p><p id="p-67">Genomic selection predictions are in accordance with studies aimed to evaluate GS using others GBS techniques, in both, relative increase in accuracy and number of discovered SNPs. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dou <em>et al.</em> (2016)</a> predicted higher accuracies for shell height and shell width using GBLUP and Bayes methods in Yesso scallop (Patinopecten yessoensis) using 2K SNPs identified by 2b-RAD. Identical methodology allowed <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Palaiokostas <em>et al.</em> (2016)</a> to reach a relative increase in accuracy up to 53% with 12K SNPs using Bayesian methods compared to PBLUP. A study in rainbow trout using RAD sequencing identified 10K SNPs. Even then, the accuracies were similar with GS models compared to PBLUP, and the authors predicted that increasing the number of individuals could lead to a relative increase in accuracy up to 69% (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Vallejo <em>et al.</em> 2016</a>).</p><p id="p-68">The genotyping strategy was aimed at the i) evaluation of genomic selection methods; and ii) allowing the identification of molecular markers associated with the trait by means of GWAS. The aim was maximizing the phenotypic variance within the sample while keeping a balanced representation of fish per family. Thus, genotyping strategy was not totally random, but specifically focused on the most extreme families; 17 resistant and 16 susceptible families. We aimed at genotyping all the fish belonging to each selected family. Thus, each family was represented within the sample with an average of 23 (ranging from 11 to 43) fish/family.</p><p id="p-69">The availability of dense SNP arrays for coho salmon, as it is already the case for Atlantic salmon (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Houston <em>et al.</em> 2014</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yáñez <em>et al.</em> 2016b</a>) and rainbow trout (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Palti <em>et al.</em> 2015a</a>), may increase the accuracy for predicting genomic breeding values and the power for the determination of the genetic factors involved in economically-important traits. It is also expected that in the near future, further functional studies for a better understanding of <em>P. salmonis</em> resistance and other complex traits in salmonids will be facilitated by the international initiative on the Functional Annotation of All Salmonid Genomes, FAASG (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Macqueen <em>et al.</em> 2017</a>).</p><p id="p-70">We have evaluated different GS models, and demonstrated that the use of genomic prediction is a feasible strategy for the improvement of breeding value prediction. This information could be used for the implementation of genomic information in genetic programs for <em>Piscirickettsia salmonis</em> resistance in farmed coho salmon populations.</p><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-71">Moderate significant genetic variation was estimated for resistance against <em>Piscirickettsia salmonis</em> in coho salmon, using either pedigree or genomic information. These results highlight the feasibility of including this trait into genetic improvement programs. Our study shows that genomic prediction methods, using ddRAD genotypes (including 9K SNPs), has a substantial advantage in terms of accuracy when compared to pedigree-based model for either DD or BIN. The improvement was up to 95 and 155% respectively in the current population. The association analyses were used to identify a gene related with B-cell development, which could also be involved in resistance against <em>P. salmonis</em>. To our knowledge, this is the first study aimed at dissecting the genetic architecture of <em>P. salmonis</em> resistance in a coho salmon population.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">AB and KC acknowledge the National Commission of Scientific and Technologic Research (CONICYT) for the funding through the National PhD funding program. AB acknowledges the Government of Canada for the funding through the Canada-Chile Leadership Exchange Scholarship. This project was funded by the U-Inicia grant, from the Vicerrectoria de Investigación y Desarrollo, Universidad de Chile. This work has been conceived on the frame of the grant FONDEF NEWTON-PICARTE (IT14I10100), funded by CONICYT (Government of Chile) and the Newton Fund - The British Council (Government of United Kingdom). This work has been partially supported by Núcleo Milenio INVASAL from Iniciativa Científica Milenio (Ministerio de Economía, Fomento y Turismo, Gobierno de Chile). This research was carried out in conjunction with EPIC4 (Enhanced Production in Coho: Culture, Community, Catch), a project supported by the government of Canada through Genome Canada, Genome British Columbia, and Genome Quebec.</p><p id="p-73">Authors’ contributions: AB performed DNA extraction, library construction, ddRAD analysis, GWAS analysis, and wrote the initial version of the manuscript. KrC performed library construction and contributed to the data analysis. AB, GY and KC performed genomic prediction analysis. AJ performed DNA extraction. JPL contributed with study design. WD contributed with analysis and discussion. JMY conceived and designed the study, supervised work of AB and contributed to the analysis, discussion and writing. All authors have reviewed and approved the manuscript.</p><p id="p-74">Animal ethics approval: Challenge and sampling procedures were approved by the Comité de Bioética Animal from the Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile (Certificate N08-2015).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-75"><em>Communicating editor: R. Houston</em></p></li><li class="fn" id="fn-10"><p id="p-76">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200053/-/DC1</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-26"><span class="received-label">Received </span>September 26, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-02-01"><span class="accepted-label">Accepted </span>February 1, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Barria <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hot topic: A unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.2" data-doi="10.1182/blood-2007-08-109769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aiba</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kameyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamazaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedder</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurosaki</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Regulation of B-cell development by BCAP and CD19 through their binding to phosphoinositide 3-kinase.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">111</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1497</span>–<span class="cit-lpage">1503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1182/blood-2007-08-109769</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2007-08-109769%26rft_id%253Dinfo%253Apmid%252F18025150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTExLzMvMTQ5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.4" data-doi="10.1186/s12864-017-3487-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bangera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic predictions can accelerate selection for resistance against Piscirickettsia salmonis in Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3487-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3487-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3487-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, and K. A. Naish, <span class="cit-pub-date">2014</span> A dense linkage map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) reveals variable chromosomal divergence after an ancestral whole genome duplication event. G3 Genes Genomes Genet. 4: 447–460. <a href="https://doi.org/10.1534/g3.113.009316">https://doi.org/10.1534/g3.113.009316</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaPatra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overturf</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Towner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Association mapping of disease resistance traits in rainbow trout using restriction site associated DNA sequencing. G3 Genes Genomes Genet. 4: 2473–81. <a href="https://doi.org/10.1534/g3.114.014621">https://doi.org/10.1534/g3.114.014621</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.7" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <em>Stacks</em>: building and genotyping loci <em>de novo</em> from short-read sequences. G3 Genes Genomes Genet. 1: 171–182.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.000240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.8" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: An analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.9" data-doi="10.1186/s12711-017-0291-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bangera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The use of genomic information increases the accuracy of breeding value predictions for sea louse (Caligus rogercresseyi) resistance in Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0291-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0291-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0291-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.10" data-doi="10.1111/are.12584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cisterna</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrejos</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallardo-Escárate</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Effect of triploidy in the expression of immune-related genes in coho salmon Oncorhynchus kisutch (Walbaum) infected with Piscirickettsia salmonis.</span> <abbr class="cit-jnl-abbrev">Aquacult. Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/are.12584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BRes.%26rft.volume%253D46%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fare.12584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/are.12584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.11" data-doi="10.1186/s12864-015-2038-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naswa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome-wide association analysis reveals loci associated with resistance against Piscirickettsia salmonis in two Atlantic salmon (Salmo salar L.) chromosomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">854</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2038-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D854%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2038-7%26rft_id%253Dinfo%253Apmid%252F26499328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2038-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26499328&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.12" data-doi="10.1111/j.1365-2761.1991.tb00584.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cvitanich</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garate</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The isolation of a rickettsia‐like organism causing disease and mortality in Chilean salmonids and its confirmation by Koch’s postulate.</span> <abbr class="cit-jnl-abbrev">J. Fish Dis.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">121</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2761.1991.tb00584.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BDis.%26rft.volume%253D14%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2761.1991.tb00584.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2761.1991.tb00584.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FU36800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.13" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.14" data-doi="10.1111/mec.12084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes-Utrilla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Special features of RAD Sequencing data: Implications for genotyping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3151</span>–<span class="cit-lpage">3164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3151%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.15" data-doi="10.1038/srep19244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of the 2b-RAD method for genomic selection in scallop breeding.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">19244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep19244</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19244%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19244%26rft_id%253Dinfo%253Apmid%252F26754638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26754638&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dufflocq</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bangera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Correlated response of flesh color to selection for harvest weight in coho salmon (Oncorhynchus kisutch).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">472</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D472%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.17" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1183.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D.S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">T.F.C.</span>  <span class="cit-name-given-names">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>. <span class="cit-article-title">Introduction to Quantitative Genetics.</span> fourth ed. <span class="cit-publ-name">Longman Group Limited,</span> <span class="cit-publ-loc">Harlow, Essex, U.K.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1183.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FAO</span></li></ol><cite>, <span class="cit-pub-date">2016</span>. Fisheries and Aquaculture Department [Online]. (Rome. Updated 31 January 2016. <a href="http://www.fao.org/fishery/statistics/global-aquaculture-production/query/en">http://www.fao.org/fishery/statistics/global-aquaculture-production/query/en</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1183.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bayesian Methods Applied to GWAS.</span> In: Gondro C., van der Werf J., Hayes B. (eds) Genome-Wide Association Studies and Genomic Prediction. Methods in Molecular Biology (Methods and Protocols), vol <span class="cit-vol">1019</span>. <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">Totowa, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.21" data-doi="10.1080/09064700801959395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stricker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totir</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">Acta Agric. Scand. Sect. Anim. Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/09064700801959395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgric.%2BScand.%2BSect.%2BAnim.%2BSci.%26rft.volume%253D57%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1080%252F09064700801959395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/09064700801959395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.22" data-doi="10.1016/S0044-8486(97)00203-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gjøen</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Refstie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulla</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic correlations between survival of Atlantic salmon in challenge and field tests.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">158</span>(<span class="cit-issue">3-4</span>): <span class="cit-fpage">277</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0044-8486(97)00203-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D158%26rft.spage%253D277%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0044-8486%252897%252900203-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0044-8486(97)00203-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000071468200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.23" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: Prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9308-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.24" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.25" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.26" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.27" data-doi="10.1139/G10-076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association and genomic selection in animal breeding.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">876</span>–<span class="cit-lpage">883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/G10-076</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DHayes%26rft.auinit1%253DB.%26rft.volume%253D53%26rft.issue%253D11%26rft.spage%253D876%26rft.epage%253D883%26rft.atitle%253DGenome-wide%2Bassociation%2Band%2Bgenomic%2Bselection%2Bin%2Banimal%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG10-076%26rft_id%253Dinfo%253Apmid%252F21076503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G10-076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076503&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.28" data-doi="10.1038/nri2491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jumaa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulation of B-cell proliferation and differentiation by pre-B-cell receptor signalling.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nri2491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DHerzog%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DRegulation%2Bof%2BB-cell%2Bproliferation%2Band%2Bdifferentiation%2Bby%2Bpre-B-cell%2Breceptor%2Bsignalling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri2491%26rft_id%253Dinfo%253Apmid%252F19240758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri2491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19240758&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263677400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.29" data-doi="10.1186/1471-2164-13-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mota-Velasco</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-244</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D13%26rft.spage%253D244%26rft.epage%253D244%26rft.atitle%253DCharacterisation%2Bof%2BQTL-linked%2Band%2Bgenome-wide%2Brestriction%2Bsite-associated%2BDNA%2B%2528RAD%2529%2Bmarkers%2Bin%2Bfarmed%2BAtlantic%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-244%26rft_id%253Dinfo%253Apmid%252F22702806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22702806&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.30" data-doi="10.1186/1471-2164-15-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-90</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-90%26rft_id%253Dinfo%253Apmid%252F24524230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24524230&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1183.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janeway</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walport</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shlomchik</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">B-cell activation by armed helper T cells.</span>, pp. <span class="cit-fpage">343</span>–<span class="cit-lpage">360</span> in <span class="cit-source"><em>Immunobiology 5</em>, <em>The Immune System in Health and Disease</em></span>, <span class="cit-edition">Ed. 5</span>. <span class="cit-publ-name">Churchill Livingstone</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.32" data-doi="10.1016/j.livsci.2014.04.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Single Step, a general approach for genomic selection.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2014.04.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.04.029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.04.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.33" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>. <a href="https://doi:10.1534/genetics.108.088575">https://doi:10.1534/genetics.108.088575</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.34" data-doi="10.1038/ni1389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barreda</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boshra</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">B lymphocytes from early vertebrates have potent phagocytic and microbicidal abilities.</span> <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1116</span>–<span class="cit-lpage">1124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ni1389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.stitle%253DNat%2BImmunol%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D1116%26rft.epage%253D1124%26rft.atitle%253DB%2Blymphocytes%2Bfrom%2Bearly%2Bvertebrates%2Bhave%2Bpotent%2Bphagocytic%2Band%2Bmicrobicidal%2Babilities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni1389%26rft_id%253Dinfo%253Apmid%252F16980980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni1389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16980980&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241437100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.35" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.36" data-doi="10.1186/1297-9686-45-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lillehammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A low-marker density implementation of genomic selection in aquaculture using within-family genomic breeding values.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-45-39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.37" data-doi="10.3389/fgene.2015.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marancik</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of single nucleotide polymorphism markers associated with bacterial cold water disease resistance and spleen size in rainbow trout.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2015.00298</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00001%26rft_id%253Dinfo%253Apmid%252F25674101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25674101&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1183.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and analysis of quantiative traits</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.39" data-doi="10.1186/s12864-017-3862-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macqueen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Primmer</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Functional Annotation of All Salmonid Genomes (FAASG): an international initiative supporting future salmonid research, conservation and aquaculture.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3862-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D484%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3862-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3862-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1183.41"><div class="cit-metadata"><cite>Misztal, I., S. Tsuruta, D. Lourenco, Y. Masuda, I. Aguilar <em>et al.,</em> <span class="cit-pub-date">2016</span> <span class="cit-article-title">Manual for BLUPF90 Family of Programs.</span> <span class="cit-publ-name">University of Georgia</span>, <span class="cit-publ-loc">Athens, GA</span>. </cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.42" data-doi="10.1534/genetics.115.175406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgersen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epithelial cadherin determines resistance to infectious pancreatic necrosis virus in Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1313</span>–<span class="cit-lpage">1326</span>. <a href="https://doi:%2010.1534/genetics.115.175406">https://doi: 10.1534/genetics.115.175406</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175406%26rft_id%253Dinfo%253Apmid%252F26041276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEzMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.43" data-doi="10.1016/j.aquaculture.2009.01.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yazdi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparison of accuracy of genome-wide and BLUP breeding value estimates in sib based aquaculture breeding schemes.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">289</span>(<span class="cit-issue">3-4</span>): <span class="cit-fpage">259</span>–<span class="cit-lpage">264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2009.01.027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D289%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2009.01.027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2009.01.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.44" data-doi="10.1111/j.1365-2109.2010.02669.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjedrem</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Methodology for genetic evaluation of disease resistance in aquaculture species: Challenges and future prospects.</span> <abbr class="cit-jnl-abbrev">Aquacult. Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2109.2010.02669.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BRes.%26rft.volume%253D42%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2109.2010.02669.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2109.2010.02669.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.45" data-doi="10.3389/fgene.2014.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsvoll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjøglum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in an admixed population of Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>. <a href="https://doi:%2010.3389/fgene.2014.00402">https://doi: 10.3389/fgene.2014.00402</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567736&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.46" data-doi="10.1016/j.aquaculture.2006.05.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olesen</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemetsdal</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluation of statistical models for genetic analysis of challenge test data on furunculosis resistance in Atlantic salmon (Salmo salar): Prediction of field survival.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">259</span>(<span class="cit-issue">1-4</span>): <span class="cit-fpage">116</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2006.05.034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D259%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2006.05.034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2006.05.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.47" data-doi="10.1016/j.fsi.2009.10.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Øverland</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rønneseth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wergeland</span>  <span class="cit-name-given-names">H. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phagocytosis by B-cells and neutrophils in Atlantic salmon (Salmo salar L.) and Atlantic cod (Gadus morhua L.).</span> <abbr class="cit-jnl-abbrev">Fish Shellfish Immunol.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">193</span>–<span class="cit-lpage">204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fsi.2009.10.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish%2B%2526%2Bshellfish%2Bimmunology%26rft.stitle%253DFish%2BShellfish%2BImmunol%26rft.aulast%253DOverland%26rft.auinit1%253DH.%2BS.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D193%26rft.epage%253D204%26rft.atitle%253DPhagocytosis%2Bby%2BB-cells%2Band%2Bneutrophils%2Bin%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2BL.%2529%2Band%2BAtlantic%2Bcod%2B%2528Gadus%2Bmorhua%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsi.2009.10.021%26rft_id%253Dinfo%253Apmid%252F19874896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsi.2009.10.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19874896&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferarreso</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Genomic prediction of resistance to pasteurellosis in gilthead sea bream (<em>Sparus aurata</em>) using 2b-RAD sequencing. G3 Genes Genomes Genet. Available at: <a href="https://doi:10.1534/g3.116.035220">https://doi:10.1534/g3.116.035220</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.49" data-doi="10.1111/1755-0998.12337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">The development and characterization of a 57K single nucleotide polymorphism array for rainbow trout.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">662</span>–<span class="cit-lpage">672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12337%26rft_id%253Dinfo%253Apmid%252F25294387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25294387&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.50" data-doi="10.1371/journal.pone.0138435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Detection and validation of QTL affecting bacterial cold water disease resistance in rainbow trout using restriction-site associated dna sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">e0138435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0138435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0138435%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0138435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0138435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.51" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: An inexpensive method for <em>de novo</em> SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>. <a href="https://doi:10.1371/journal.pone.0037135">https://doi:10.1371/journal.pone.0037135</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.52" data-doi="10.1186/s12864-015-1716-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pulgar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hödar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travisany</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuñiga</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domínguez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptional response of Atlantic salmon families to Piscirickettsia salmonis infection highlights the relevance of the iron-deprivation defence system.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1716-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D495%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1716-9%26rft_id%253Dinfo%253Apmid%252F26141111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1716-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26141111&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.53" data-doi="10.1152/physiolgenomics.00036.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rise</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Schalburg</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microarray analyses identify molecular biomarkers of Atlantic salmon macrophage and hematopoietic kidney response to Piscirickettsia salmonis infection.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">21</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/physiolgenomics.00036.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00036.2004%26rft_id%253Dinfo%253Apmid%252F15454580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00036.2004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15454580&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225840900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.54" data-doi="10.1111/raq.12193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Applications of genotyping by sequencing in aquaculture breeding and genetics. Rev. Aquac. 1–13.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/raq.12193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1111%252Fraq.12193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/raq.12193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.55" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.104935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.56" data-doi="10.1111/jfd.12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enríquez</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Piscirickettsiosis and Piscirickettsia salmonis in fish: a review.</span> <abbr class="cit-jnl-abbrev">J. Fish Dis.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">163</span>–<span class="cit-lpage">188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jfd.12211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BDis.%26rft.volume%253D37%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjfd.12211%26rft_id%253Dinfo%253Apmid%252F24279295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jfd.12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24279295&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-other" id="cit-8.4.1183.57"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Sernapesca</span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Informe Sanitario de Salmonicultura en Centros Marinos 2016. ([Online]</span>. Valparaiso, Chile. Updated 14 August 2017. <a href="http://www.sernapesca.cl/index.php?option=com_remository&amp;Itemid=246&amp;func=fileinfo&amp;id=27560">http://www.sernapesca.cl/index.php?option=com_remository&amp;Itemid=246&amp;func=fileinfo&amp;id=27560</a>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.58" data-doi="10.1053/rvsc.2001.0496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stear</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallard</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The sustainability, feasibility and desirability of breeding livestock for disease resistance.</span> <abbr class="cit-jnl-abbrev">Res. Vet. Sci.</abbr> <span class="cit-vol">71</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1053/rvsc.2001.0496</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2Bveterinary%2Bscience%26rft.stitle%253DRes%2BVet%2BSci%26rft.aulast%253DStear%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D71%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DThe%2Bsustainability%252C%2Bfeasibility%2Band%2Bdesirability%2Bof%2Bbreeding%2Blivestock%2Bfor%2Bdisease%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1053%252Frvsc.2001.0496%26rft_id%253Dinfo%253Apmid%252F11666141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1053/rvsc.2001.0496&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11666141&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172365600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.59" data-doi="10.1186/s12711-016-0226-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bron</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0226-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0226-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0226-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.60" data-doi="10.3389/fgene.2016.00096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leeds</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragomeni</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of genome-enabled selection for bacterial cold water disease resistance using progeny performance data in rainbow trout: insights on genotyping methods and genomic prediction models.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.61" data-doi="10.1186/s12711-017-0293-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leeds</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection models double the accuracy of predicted breeding values for bacterial cold water disease resistance compared to a traditional pedigree-based model in rainbow trout aquaculture.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0293-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0293-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0293-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.62" data-doi="10.3389/fgene.2017.00156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragomeni</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Similar genetic architecture with shared and unique quantitative trait loci for bacterial cold water disease resistance in two rainbow trout breeding populations.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2017.00156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D8%26rft.spage%253D156%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2017.00156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2017.00156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.63" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.64" data-doi="10.3389/fgene.2014.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association mapping including phenotypes from relatives without genotypes in a single-step (ssGWAS) for 6-week body weight in broiler chickens.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567736&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.65" data-doi="10.1017/S0016672312000274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping including phenotypes from relatives without genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">02</span>): <span class="cit-fpage">73</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672312000274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D94%26rft.issue%253D2%26rft.spage%253D73%26rft.epage%253D83%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bincluding%2Bphenotypes%2Bfrom%2Brelatives%2Bwithout%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672312000274%26rft_id%253Dinfo%253Apmid%252F22624567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672312000274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22624567&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.66" data-doi="10.1016/j.aquaculture.2016.03.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bangera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barria</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oyarzun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Negative genetic correlation between resistance against Piscirickettsia salmonis and harvest weight in coho salmon (Oncorhynchus kisutch).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2016.03.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D459%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2016.03.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2016.03.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.67" data-doi="10.1111/1755-0998.12503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naswa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomewide single nucleotide polymorphism discovery in Atlantic salmon (Salmo salar): validation in wild and farmed American and European populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1002</span>–<span class="cit-lpage">1011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D1002%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.68" data-doi="10.1016/j.aquaculture.2013.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bangera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oyarzún</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neira</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantitative genetic variation of resistance against Piscirickettsia salmonis in Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">414–415</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2013.08.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D414415%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2013.08.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2013.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.69" data-doi="10.1016/j.aquaculture.2013.05.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponzoni</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Inbreeding and effective population size in a coho salmon (Oncorhynchus kisutch) breeding nucleus in Chile.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">420–421</span>: <span class="cit-fpage">S15</span>–<span class="cit-lpage">S19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2013.05.028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D420421%26rft.spage%253DS15%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2013.05.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2013.05.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.70" data-doi="10.3389/fgene.2014.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetics and genomics of disease resistance in salmonid species.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>. <a href="https://doi:10.3389/fgene.2014.00415">https://doi:10.3389/fgene.2014.00415</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567736&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.71" data-doi="10.1016/j.aquaculture.2014.06.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassini</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oyarzún</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">c</span> <span class="cit-article-title">Genetic co-variation between resistance against both Caligus rogercresseyi and Piscirickettsia salmonis, and body weight in Atlantic salmon (Salmo salar).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">433</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2014.06.026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D433%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2014.06.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2014.06.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1183.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">G.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bangera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalheiro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction accuracy for resistance against <em>Piscirickettsia salmonis</em> in farmed rainbow trout.</span> <abbr class="cit-jnl-abbrev">G3 Genes Genomes Genet</abbr>. <span class="cit-vol">8</span>:<span class="cit-fpage">719</span>–<span class="cit-lpage">726</span>. <a href="https://doi.org/10.1534/g3.117.300499">https://doi.org/10.1534/g3.117.300499</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%2BGenomes%2BGenet%26rft.volume%253D8%26rft.spage%253D719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/4/1183.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1173" title="Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1195" title="Genome-Wide Association and Genomic Selection for Resistance to Amoebic Gill Disease in Atlantic Salmon" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/4"><span>Volume 8 Issue 4, April 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/4.cover-source.jpg?itok=wC37V0ox" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Tq1ccPc-L20dgilQY82rfPBMVAFUjSvuMsKpcMa62fc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/4/1183" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/4/1183" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/4/1183" class="active" data-icon-position="" data-hide-link-title="0">Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489136" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ATrHe7mNcQAhhogiCY5A0deAj_OV4NmDLZ3ewJ6bRE0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1183.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/4/1183" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wC-WcPm8OYkKBnQG0tl2F6i8eixuFXUIfNFMmGrT_iE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/4/1183" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1183.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489136" id="node489136--2" data-pisa="ggg;8/4/1183" data-pisa-master="ggg;g3.118.200053" data-apath="/ggg/8/4/1183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Predictions and Genome-Wide Association Study of Resistance Against <em>Piscirickettsia salmonis</em> in Coho Salmon (<em>Oncorhynchus kisutch</em>) Using ddRAD Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2813-4559" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Agustín Barría</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0021-0326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kris A. Christensen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6788-7369" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Grazyella M. Yoshida</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1386-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katharina Correa</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0626-1247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ana Jedlicki</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9157-4231" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean P. Lhorente</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3002-3225" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William S. Davidson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6612-4087" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José M. Yáñez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1183-1194;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200053 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489136--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Agustín Barría </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Doctorado en Acuicultura, Programa Cooperativo Universidad de Chile, Universidad Católica del Norte, Pontificia Universidad Católica de Valparaíso, 8820808 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agust%C3%ADn%2BBarr%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%C3%ADa%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAgust%25C3%25ADn%2BBarr%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2813-4559" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Agustín Barría</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kris A. Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kris%2BA.%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKris%2BA.%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0021-0326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kris A. Christensen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Grazyella M. Yoshida </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Animal Science Department, Universidade Estadual Paulista “Júlio de Mesquita Filho”, Faculdade de Ciências Agrárias e Veterinárias, Campus Jaboticabal, Jaboticabal 14884-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grazyella%2BM.%2BYoshida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yoshida%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGrazyella%2BM.%2BYoshida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6788-7369" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Grazyella M. Yoshida</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katharina Correa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BCorrea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Correa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BCorrea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1386-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina Correa</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ana Jedlicki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BJedlicki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jedlicki%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BJedlicki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0626-1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ana Jedlicki</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean P. Lhorente </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BP.%2BLhorente%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lhorente%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BP.%2BLhorente%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9157-4231" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean P. Lhorente</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">William S. Davidson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BS.%2BDavidson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davidson%20WS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BS.%2BDavidson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3002-3225" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William S. Davidson</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">José M. Yáñez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Núcleo Milenio INVASAL, Concepción 4070386, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BY%C3%A1%C3%B1ez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Y%C3%A1%C3%B1ez%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BY%25C3%25A1%25C3%25B1ez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6612-4087" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José M. Yáñez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmayanez@uchile.cl" class="" data-icon-position="" data-hide-link-title="0">jmayanez@uchile.cl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/4/1183" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/4/1183&amp;title=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/4/1183&amp;title=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/4/1183&amp;title=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/4/1183&amp;text=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/4/1183&amp;title=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/4/1183&amp;t=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/4/1183&amp;title=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/4/1183&amp;title=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489136/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489136" id="node489136--3" data-pisa="ggg;8/4/1183" data-pisa-master="ggg;g3.118.200053" data-apath="/ggg/8/4/1183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Predictions and Genome-Wide Association Study of Resistance Against <em>Piscirickettsia salmonis</em> in Coho Salmon (<em>Oncorhynchus kisutch</em>) Using ddRAD Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2813-4559" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Agustín Barría</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0021-0326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kris A. Christensen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6788-7369" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Grazyella M. Yoshida</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1386-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katharina Correa</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0626-1247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ana Jedlicki</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9157-4231" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean P. Lhorente</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-3002-3225" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William S. Davidson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6612-4087" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José M. Yáñez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1183-1194;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200053 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489136--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Agustín Barría </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Doctorado en Acuicultura, Programa Cooperativo Universidad de Chile, Universidad Católica del Norte, Pontificia Universidad Católica de Valparaíso, 8820808 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agust%C3%ADn%2BBarr%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barr%C3%ADa%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAgust%25C3%25ADn%2BBarr%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2813-4559" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Agustín Barría</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kris A. Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kris%2BA.%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKris%2BA.%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0021-0326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kris A. Christensen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Grazyella M. Yoshida </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Animal Science Department, Universidade Estadual Paulista “Júlio de Mesquita Filho”, Faculdade de Ciências Agrárias e Veterinárias, Campus Jaboticabal, Jaboticabal 14884-900, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grazyella%2BM.%2BYoshida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yoshida%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGrazyella%2BM.%2BYoshida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6788-7369" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Grazyella M. Yoshida</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Katharina Correa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BCorrea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Correa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BCorrea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1386-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina Correa</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ana Jedlicki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BJedlicki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jedlicki%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BJedlicki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0626-1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ana Jedlicki</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean P. Lhorente </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BP.%2BLhorente%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lhorente%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BP.%2BLhorente%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9157-4231" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean P. Lhorente</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">William S. Davidson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, BC V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BS.%2BDavidson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davidson%20WS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BS.%2BDavidson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3002-3225" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William S. Davidson</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">José M. Yáñez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Ciencias Veterinarias y Pecuarias, Universidad de Chile, La Pintana, Santiago 8820808, Chile</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Aquainnovo S.A., Puerto Montt 5503032, Chile</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Núcleo Milenio INVASAL, Concepción 4070386, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BY%C3%A1%C3%B1ez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Y%C3%A1%C3%B1ez%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BY%25C3%25A1%25C3%25B1ez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6612-4087" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José M. Yáñez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmayanez@uchile.cl" class="" data-icon-position="" data-hide-link-title="0">jmayanez@uchile.cl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1183&amp;rft.epage=1194&amp;rft.atitle=Genomic%20Predictions%20and%20Genome-Wide%20Association%20Study%20of%20Resistance%20Against%20Piscirickettsia%20salmonis%20in%20Coho%20Salmon%20%28Oncorhynchus%20kisutch%29%20Using%20ddRAD%20Sequencing&amp;rft.volume=8&amp;rft.issue=4&amp;rft.date=2018-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Barr%C3%ADa%2C+Agust%C3%ADn&amp;rft.au=Christensen%2C+Kris+A.&amp;rft.au=Yoshida%2C+Grazyella+M.&amp;rft.au=Correa%2C+Katharina&amp;rft.au=Jedlicki%2C+Ana&amp;rft.au=Lhorente%2C+Jean+P.&amp;rft.au=Davidson%2C+William+S.&amp;rft.au=Y%C3%A1%C3%B1ez%2C+Jos%C3%A9+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489136/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489136/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489136/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489136/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489136/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489136/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489136/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489136/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489136/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489136/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489136/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489136/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/4/1183" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/4/1183.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/4/1183.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/4/1183.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/4/1183.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
