<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/4/1173" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/4/1173.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/4/1173.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/4/1173.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/4/1173/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/4/1173.atom" />
<meta name="HW.pisa" content="ggg;8/4/1173" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection" />
<meta name="DC.Identifier" content="10.1534/g3.118.200091" />
<meta name="DC.Date" content="2018-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Muller et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) offers the possibility to estimate the effects of genome-wide molecular markers, which can be used to calculate genomic estimated breeding values (GEBVs) for individuals without phenotypes. GEBVs can serve as a selection criterion in recurrent GS, maximizing single-cycle but not necessarily long-term genetic gain. As simple genome-wide sums, GEBVs do not take into account other genomic information, such as the map positions of loci and linkage phases of alleles. Therefore, we herein propose a novel selection criterion called expected maximum haploid breeding value (EMBV). EMBV predicts the expected performance of the best among a limited number of gametes that a candidate contributes to the next generation, if selected. We used simulations to examine the performance of EMBV in comparison with GEBV as well as the recently proposed criterion optimal haploid value (OHV) and weighted GS. We considered different population sizes, numbers of selected candidates, chromosome numbers and levels of dominant gene action. Criterion EMBV outperformed GEBV after about 5 selection cycles, achieved higher long-term genetic gain and maintained higher diversity in the population. The other selection criteria showed the potential to surpass both GEBV and EMBV in advanced cycles of the breeding program, but yielded substantially lower genetic gain in early to intermediate cycles, which makes them unattractive for practical breeding. Moreover, they were largely inferior in scenarios with dominant gene action. Overall, EMBV shows high potential to be a promising alternative selection criterion to GEBV for recurrent genomic selection." />
<meta name="DC.Contributor" content="Dominik Müller" />
<meta name="DC.Contributor" content="Pascal Schopp" />
<meta name="DC.Contributor" content="Albrecht E. Melchinger" />
<meta name="article:published_time" content="2018-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) offers the possibility to estimate the effects of genome-wide molecular markers, which can be used to calculate genomic estimated breeding values (GEBVs) for individuals without phenotypes. GEBVs can serve as a selection criterion in recurrent GS, maximizing single-cycle but not necessarily long-term genetic gain. As simple genome-wide sums, GEBVs do not take into account other genomic information, such as the map positions of loci and linkage phases of alleles. Therefore, we herein propose a novel selection criterion called expected maximum haploid breeding value (EMBV). EMBV predicts the expected performance of the best among a limited number of gametes that a candidate contributes to the next generation, if selected. We used simulations to examine the performance of EMBV in comparison with GEBV as well as the recently proposed criterion optimal haploid value (OHV) and weighted GS. We considered different population sizes, numbers of selected candidates, chromosome numbers and levels of dominant gene action. Criterion EMBV outperformed GEBV after about 5 selection cycles, achieved higher long-term genetic gain and maintained higher diversity in the population. The other selection criteria showed the potential to surpass both GEBV and EMBV in advanced cycles of the breeding program, but yielded substantially lower genetic gain in early to intermediate cycles, which makes them unattractive for practical breeding. Moreover, they were largely inferior in scenarios with dominant gene action. Overall, EMBV shows high potential to be a promising alternative selection criterion to GEBV for recurrent genomic selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/04/01" />
<meta name="citation_mjid" content="ggg;8/4/1173" />
<meta name="citation_id" content="8/4/1173" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/4/1173" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/4/1173.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/4/1173.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/4/1173.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200091" />
<meta name="citation_pmid" content="29434032" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1173" />
<meta name="citation_lastpage" content="1181" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dominik Müller" />
<meta name="citation_author_institution" content="University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and" />
<meta name="citation_author" content="Pascal Schopp" />
<meta name="citation_author_institution" content="KWS SAAT SE, Grimsehlstraβe 31, 37574 Einbeck" />
<meta name="citation_author" content="Albrecht E. Melchinger" />
<meta name="citation_author_institution" content="University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and" />
<meta name="citation_author_email" content="melchinger@uni-hohenheim.de" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=L.-Y. Chang;citation_author=S. Toghiani;citation_author=A. Ling;citation_author=S. E. Aggrey;citation_author=R. Rekaya;citation_title=High density marker panels, SNPs prioritizing and accuracy of genomic selection;citation_pages=4;citation_volume=19;citation_year=2018;citation_issue=1;citation_doi=10.1186/s12863-017-0595-2" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=R. E. Comstock;citation_author=H. F. Robinson;citation_author=P. H. Harvey;citation_title=A Breeding Procedure Designed To Make Maximum Use of Both General and Specific Combining Ability;citation_pages=360-367;citation_volume=41;citation_year=1949;citation_issue=8;citation_doi=10.2134/agronj1949.00021962004100080006x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=P. Crnokrak;citation_author=D. A. Roff;citation_title=Dominance variance: associations with selection and fitness;citation_pages=530-540;citation_volume=75;citation_year=1995;citation_issue=5;citation_doi=10.1038/hdy.1995.169" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. J. Hayden;citation_author=G. C. Spangenberg;citation_author=B. J. Hayes;citation_title=Selection on optimal haploid value increases genetic gain and preserves more genetic diversity relative to genomic selection;citation_volume=200;citation_year=2015;citation_issue=4;citation_pmid=26092719;citation_doi=10.1534/genetics.115.178038" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_issue=2;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. E. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Goiffon;citation_author=A. Kusmec;citation_author=L. Wang;citation_author=G. Hu;citation_author=P. Schnable;citation_title=Improving response in genomic selection with a population-based selection strategy: optimal population value selection;citation_volume=206;citation_year=2017;citation_issue=3;citation_pmid=28526698;citation_doi=10.1534/genetics.116.197103" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Gorjanc;citation_author=J. Jenko;citation_author=S. J. Hearne;citation_author=J. M. Hickey;citation_title=Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations;citation_pages=30;citation_volume=17;citation_year=2016;citation_issue=1;citation_pmid=26732811;citation_doi=10.1186/s12864-015-2345-z" />
<meta name="citation_reference" content="citation_journal_title=Maydica;citation_author=A. R. Hallauer;citation_author=M. J. Carena;citation_title=Recurrent selection methods to improve germplasm in maize;citation_pages=266-283;citation_volume=57;citation_year=2012;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Han;citation_author=J. N. Cameron;citation_author=L. Wang;citation_author=W. D. Beavis;citation_title=The predicted cross value for genetic introgression of multiple alleles.;citation_volume=205;citation_year=2017;citation_issue=4;citation_pmid=28122824;citation_doi=10.1534/genetics.116.197095" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_issue=6;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Agron.;citation_author=F. H. Hull;citation_title=Recurrent selection for specific combining ability in corn.;citation_pages=134-145.;citation_volume=37;citation_year=1945;citation_issue=2;citation_doi=10.2134/agronj1945.00021962003700020006x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J.-L. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Agron.;citation_author=M. T. Jenkins;citation_title=The segregation of genes affecting yield of grain in maize;citation_pages=55-63;citation_volume=32;citation_year=1940;citation_issue=1;citation_doi=10.2134/agronj1940.00021962003200010008x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H. Liu;citation_author=T. H. E. Meuwissen;citation_author=A. C. Sørensen;citation_author=P. Berg;citation_title=Upweighting rare favourable alleles increases long-term genetic gain in genomic selection programs;citation_pages=19;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0101-0" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_pages=980;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. E. Melchinger;citation_author=R. K. Gumber;citation_author=R. B. Leipert;citation_author=M. Vuylsteke;citation_author=M. Kuiper;citation_title=Prediction of testcross means and variances among F3 progenies of F1 crosses from testcross means and genetic distances of their parents in maize.;citation_pages=503-512;citation_volume=96;citation_year=1998;citation_issue=3–4;citation_pmid=24710890;citation_doi=10.1007/s001220050767" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. A. Mikel;citation_author=J. W. Dudley;citation_title=Evolution of North American Dent Corn from Public to Proprietary Germplasm;citation_pages=1193-1205;citation_volume=46;citation_year=2006;citation_issue=3;citation_doi=10.2135/cropsci2005.10-0371" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. M. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="Müller, D. 2017 embvr: Computation of expected maximum haploid breeding values. doi: 10.5281/ zenodo.556392." />
<meta name="citation_reference" content="citation_author=D. Müller;citation_author=K. W. Broman;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=D. Müller;citation_author=P. Schopp;citation_author=A. E. Melchinger;citation_title=Persistency of prediction accuracy and genetic gain in synthetic populations under recurrent genomic selection.;citation_volume=7;citation_year=2017;citation_issue=3;citation_pmid=28064189;citation_doi=10.1534/g3.116.036582" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=N. J. Smith;citation_author=P. Donnelly;citation_title=A new statistical method for haplotype reconstruction from population data.;citation_pages=978-989;citation_volume=68;citation_year=2001;citation_issue=4;citation_pmid=11254454;citation_doi=10.1086/319501" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Van Inghelandt;citation_author=J. C. Reif;citation_author=B. S. Dhillon;citation_author=P. Flament;citation_author=A. E. Melchinger;citation_title=Extent and genome-wide distribution of linkage disequilibrium in commercial maize germplasm.;citation_pages=11-20;citation_volume=123;citation_year=2011;citation_issue=1;citation_pmid=21404061;citation_doi=10.1007/s00122-011-1562-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=A. Legarra;citation_title=On the additive and dominant variance and covariance of individuals within the genomic selection scope.;citation_volume=195;citation_year=2013;citation_issue=4;citation_pmid=24121775;citation_doi=10.1534/genetics.113.155176" />
<meta name="citation_reference" content="citation_journal_title=G3.;citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=“Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments”.;citation_pages=1427-1436;citation_volume=2;citation_year=2012;citation_issue=11" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=N. R. Wray;citation_author=M. E. Goddard;citation_title=Increasing long-term response to selection;citation_pages=431-451;citation_volume=26;citation_year=1994;citation_issue=5;citation_doi=10.1051/gse:19940504" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Zeng;citation_author=A. Toosi;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_author=D. J. Garrick;citation_title=Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action.;citation_pages=11;citation_volume=45;citation_year=2013;citation_pmid=23621868;citation_doi=10.1186/1297-9686-45-11" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_author=M. Olsen;citation_author=E. Buckler;citation_title=Rapid cycling genomic selection in a multi-parental tropical maize population.;citation_volume=7;citation_year=2017;citation_issue=7;citation_pmid=28533335;citation_doi=10.1534/g3.117.043141" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) offers the possibility to estimate the effects of genome-wide molecular markers, which can be used to calculate genomic estimated breeding values (GEBVs) for individuals without phenotypes. GEBVs can serve as a selection criterion in recurrent GS, maximizing single-cycle but not necessarily long-term genetic gain. As simple genome-wide sums, GEBVs do not take into account other genomic information, such as the map positions of loci and linkage phases of alleles. Therefore, we herein propose a novel selection criterion called expected maximum haploid breeding value (EMBV). EMBV predicts the expected performance of the best among a limited number of gametes that a candidate contributes to the next generation, if selected. We used simulations to examine the performance of EMBV in comparison with GEBV as well as the recently proposed criterion optimal haploid value (OHV) and weighted GS. We considered different population sizes, numbers of selected candidates, chromosome numbers and levels of dominant gene action. Criterion EMBV outperformed GEBV after about 5 selection cycles, achieved higher long-term genetic gain and maintained higher diversity in the population. The other selection criteria showed the potential to surpass both GEBV and EMBV in advanced cycles of the breeding program, but yielded substantially lower genetic gain in early to intermediate cycles, which makes them unattractive for practical breeding. Moreover, they were largely inferior in scenarios with dominant gene action. Overall, EMBV shows high potential to be a promising alternative selection criterion to GEBV for recurrent genomic selection." />
<meta name="og-title" property="og:title" content="Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/4/1173" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) offers the possibility to estimate the effects of genome-wide molecular markers, which can be used to calculate genomic estimated breeding values (GEBVs) for individuals without phenotypes. GEBVs can serve as a selection criterion in recurrent GS, maximizing single-cycle but not necessarily long-term genetic gain. As simple genome-wide sums, GEBVs do not take into account other genomic information, such as the map positions of loci and linkage phases of alleles. Therefore, we herein propose a novel selection criterion called expected maximum haploid breeding value (EMBV). EMBV predicts the expected performance of the best among a limited number of gametes that a candidate contributes to the next generation, if selected. We used simulations to examine the performance of EMBV in comparison with GEBV as well as the recently proposed criterion optimal haploid value (OHV) and weighted GS. We considered different population sizes, numbers of selected candidates, chromosome numbers and levels of dominant gene action. Criterion EMBV outperformed GEBV after about 5 selection cycles, achieved higher long-term genetic gain and maintained higher diversity in the population. The other selection criteria showed the potential to surpass both GEBV and EMBV in advanced cycles of the breeding program, but yielded substantially lower genetic gain in early to intermediate cycles, which makes them unattractive for practical breeding. Moreover, they were largely inferior in scenarios with dominant gene action. Overall, EMBV shows high potential to be a promising alternative selection criterion to GEBV for recurrent genomic selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
    <title>Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TCAzW6VWFAQr0aOyi46Pb7FPq-NXkHp4kNijf6W4mv0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489127","apath":"\/ggg\/8\/4\/1173.atom","pisa":"ggg;8\/4\/1173","ac":{"\/ggg\/8\/4\/1173.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/4\/1173.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1173"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1173"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/4\/1173":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UdVvD+8YkU8kx+an2GP1ZZDPSkmEKJPZxQLxsrsv7pyU+ik27PS3ttAnOBEMQ7jfJnESQQPpOGCmOsC\\\\\\\/0HRdwUJxnLwO4DNwGtJtirLuco+i5zcNhDfZpwWdAQG8PbIeIxCTci+aRC2hy+uaXkpXKZRGnFmzBc3g\\\\\\\/UHnVrF5+4TG6yGMq1IkqJZMfq2fZOA+Jx+g7WjWqogfSiuDW1YhP343mO9ngTYcUfSRxtRZaMesp\\\\\\\/MInVd4oo7RiCBHRpis8ln0VeABFiEr5GbYXLvdTSCP6S58iaPAfBP3x0fngop5cES4FB7Bl\\\\\\\/cJCfKR4EiFRbABgi5cM6rAZGXQD1eSPFUBwf5g3JpfYZUWGEQgXOClc2t7I7we08D4k3aDRiC1XgXtC12NwuEDVy5kqtLMYRPjRalybtzzBHBihPWad1mV+mSFE2QdwO5F16Z+WTXZTOpiGBpupnxNB7KzFvIrAEJsKyuM5vB0DwT9ZTXZVly0xVYVbND\\\\\\\/Oc+i\\\\\\\/Dtd7Rj52tVn51QiJvyi5G6kDmlaKVwKylN4fWn+G47HQyzpe+fVWkB2gZwMY7VHy\\\\\\\/WEHUBx5d9XuVKZez2qVSuMV5BepdJqemjbm3ocxbuPsr2idJPC4mnxmjuIn2eyAWongKFMZvw\\\\\\\/hG6DBP9ooZe7Gl8EzMZ4DMldCUC8A70fEFMNhgTl1k5+uc7f5rSiKR+VUIJmItubDRTnTvZdmiSjjFQ7MrCkFewbb4wFloPO6PSyB6qOnkH8S5Hde1FLVKYYNr1IUDbMKcnXmijYYvx+cXcmykX9GLdUskfI8xR3i3m3IoEB9ztV6iBylIZylLntj1vJGm7WG6xvpF5TCGnmyWrr8zidIUdGUPvhLwtUM6YXc0do6zthsz7KGH0W1IAqNI024igPUQipywqVBKJq2O5zjRmYS0IJ2orO\\\\\\\/kZSqqT6RjyOkM5AHJxNbWpvZBKQt7PoJthBow1jIVTGeIzddqYvUIlzdOexYKskmzxJJso0pYZ30S7ffcB9xdhoUADIaLI2DJE7fS1rY2BGM9wW4JECoMZ9KDkSV5VD5o7XfT0UFROKAls4UIfKHVkZawHW+ZllzWGeKb8ODGMgYzGtnPHrN74015s66XOzXLnowLKsxuxWs0KsS1e7ZpjpuxkMSEBepJIcth2d9\\\\\\\/HAbRO3EPNd2nD5vwONXH7joBTcQLwFhitmDOlde1VqhORBksWkSedRdYo8kfbNjkEXY+C+7bYenr7Eq4cmSFlGleSoZFuZHulMN+pB8LAaYz2YreBkVogP\\\u0022,\\\u0022iv\\\u0022:\\\u0022dtDzOjz00coE61QwrIYC0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022967d5b0e917184e4f1917f760cc03141177dbf160daebbe3de0f7b9efeeb9331\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222JTIF9GvD6zdDNuT8oeLINJhLYp8EBlyYdFTKGszmJgbd+Of1FpXdHdMj1uWOsC2rUQwONrsfcS79+teVGTtUU1WOMCsfUDFc+NbAHNRxtfus6YutYbpoIrAUHclcrtLMxnsMlBYsEKImBB3GPRlrvhr5r4LlMQ3UR3wcIgu+NIyqK7cHApZk5FCDQ3eCjN0\\\\\\\/\\\\\\\/qWvbTnjRkjhuLfxoh3AnRusu\\\\\\\/gVSeuuzfqGynIEXqV1Z1fYtV5gDFavxcmcP0R5N+h6usR8GE3FSn+eRzMbDlW1O0efYiNmJz6g8K5NcX3CTTXdXv+K8elG9jremCRbiniJAir6g7jyt\\\\\\\/AqDaZl6a8UKn5Wk4dh4lQYkne45dwT2LlvqtIXXS3yh6pYNfq2YsqIc0qbgT7xASWFetqTSBSD+eFgPcPSja7ILHT73wqn2Z9ztUS\\\\\\\/HMiLnQ\\\\\\\/GHqmTEXFzqXQY3t8jmbc79bymJbBiK+2s5sUYjUEK\\\\\\\/+XCFM9DJQDs+1\\\\\\\/8ODcu0Jjld9z2PKfYhPLf1KQGkCU7E\\\\\\\/Mcj9ZQRENxFenBZeptT6skbZFX3x5bcifeH8y4draljrycu4UivIitc5bzW+L7KIDftXy66YglWqbP722iOV7ucTI5429jh5ELWItjyRMw3RmBF2bcusLovBg2eM6XJ4C0ZMhDwfLiNKROhjvUAbbjNPDLFkwOWOvqugfkdwX8PsDxdrghrYJsmvjG8eMBAqU6ozQ5k9xfrmOWOZFFoUv59E+Nyu\\\\\\\/n999mYdxtYWKjjAyB5WsZHRKESWJFmjmyfgFdfrAErThvmLUilCwyAZ2JFqkGfiFLI2TWm8C0DcG9sB7AcmfkwKM7LpRMz7rAlMDrIpPhG+IDepU1QAPWkkHWsGpra0ZbV1gwGvhiO11vlmZrNbJoh7ojRDhoMeNDzbK9BA+ueVqgJ4SRSgtwe1sh9iSIPmT5qnZvTdfjAFk9vh7sS4yg0HpmBQ88eL5WL+7TeEL21rvrT1ZNaMw5JhlWDPw8UgpxIEdP3VcHpjV03p9GqoSGyyCrtgxGEkJfamD5ilZXMKEHEnjXUO\\\\\\\/bbxLfH+ZxBMIFHDZ\\\\\\\/lhYWvKFGRUETH+F0hr6Y8RklhaY3mWxhDPC2bdKsRqS3aF48T5zvqghrLrpCGk7w6cDDWciRr567sI\\\\\\\/fSupcdD2OA0wup\\\\\\\/b\\\\\\\/fve9UHJgpg+SJ3lRlLzETfwHrz80EaeCxHCzfXlMXT4iuMq48WqNK4AwAkJg8TAeHWb2liz3WsUldOE5zGfq6OiDEhm7KXPWX91SXKmKEGn561s3+bBEBVXDqVZdmXTS6X2p0LZBzR88xrfD1YrFp17DFBA8fuRM7JvkkBlCqXi7vA09hy2Zk1Qsohoxgvqw1EUig2JBEn4kiP2OMMwJAhqe9i3V1MUJ37ErAXVaT1uXME0g8TN4m8BzPVBG2SzuLFBTazuE7kanKaBYSQQqzIUrzzqEbx7\\\\\\\/9SBIjkdZ9tPNZDOnipAYL56P+GoZ7WElvNw3MvRNN4b5W3Wyue\\\\\\\/QmQ1dZ9TGcZ39TEm1w7X+oCbjrraXsB3\\\\\\\/5XX\\\\\\\/tpV218oX7zyFEL1AzOKQDpVltUsa8s+aOpvjmdjJam2YAN0UrfNK68e0Nhu20dx707RtQmxZnBrqa4tzhYmFLMV0rLKL7KmsTWUui7bXRd677alRKniIVFGn3NCbkpjTh9vDJ5PrDO58pKt+HQNcAXyeKNwQ\\\\\\\/GU0E\\\\\\\/Pl0zNUHMrAXdrf6TYl7cZyEVcJJi6ZKlLVw9cJc4nhdGF7QG\\\\\\\/rFxXNwjNYiyenSO1qA+PJvCUMNQNJeMvSP4oWzHhXVWksPUkssnmVOQfLJFS1pQPwfwuZf2\\\\\\\/14cMCUOX8sTcDtEAcbXcl8KjRy+H0UlrIYk\\\\\\\/mvyQJj3z4QQLGnFtNbJj6GoaP54R1sKSFwNEr0swR1FuiAv7cSeHyPB4NFw9ovGGQGoo90WGR0R+8gvArzjKZRnCRAOyGkNn3MoQsvlmrbkU10WFpzzybyUk0smaxQxBxwzBaxmOoR9wzjJLssQmb\\\\\\\/+QFHEcOML9Tld8Ynk3GU0Y0R6pTmq7YDM0PR0+LZzE42vG8QlyOcNPtu9fe5tcLsyqQthNlY8FmMgZfc49i1dIfhwWjqfRvCZp5003GuBVeCVZD5nPlqXu\\\\\\\/HtcYt54fk4CSVtUuRksBjz0WBVJvFbtk\\\\\\\/1ouWfDb\\\\\\\/Wl\\\\\\\/RhMAvC5hiMMQ+6pI\\\\\\\/+pjbjev6K8lzc7QVNCMNd+Zg8XwT9Do325TZd+7kJhcbGKD0iUeTkD\\\\\\\/TnUSkgYpOZ6gDOk2ZDTaQIhkc+uk4akUVlPvgnlp8McctSy1r7IbS2MoMv35ArB2mQ+WsqLpV7S7ljGaH52P1uSThXBl2nNQzOoW9Kekrb2HRrv75pntKhpnJ8rgohMqN+oxeNaBk\\\\\\\/gFtqAm6SHwcdouotp3bmyz2I9XZdoMH+xeFVOqW3ZfU87G8In4ylYDv\\\\\\\/lNkMAekGtvxfm10x3XizaMfxaCaGlHt2OIXwG080wi\\\\\\\/suxcGqfh8iBkFHJ6iLj+aY22ypsi\\\\\\\/vw5hba\\\\\\\/c9VqYGGohwdFVd4VMnzkkQxSSIz1Eyy\\\\\\\/kkNRdhDTVh1Z0WZFlnBj4KENTBpsw1KkeyeFtw4BOR2ugvEkT5uX03EluKUGMb24sBaMEBZxDXeLOWvW1VE7\\\\\\\/CMnDg9UlLzht6iUGolscD28C5OkTPL2xwrSWWE0m7\\\\\\\/Bsn1o\\\\\\\/IeVyDJAkE\\\\\\\/iPrMDL5xVBm7xF9a\\\\\\\/IMPLzIjhrGtUQHvM68A8BjrM8ReKAUj78rj333lXvLfZ\\\\\\\/vtS41d7DCc9\\\\\\\/XFRtaiaeaPnkjvKD\\\\\\\/QiL+ZDI9v9Ur+p1u8yCv7uoag6ruh8vM9evHP2i3BIa1j07nKeexPI\\\\\\\/umK5hdzRtonfeI1DA55FI5jPao2Q8aXY5uiL\\\\\\\/ZYPRbded3wxmOTA2lMf+aTPyVNiSMNeCSVnT97CcZHBH90sL1aOLFV+4p\\\\\\\/kiD4Rw2Mck\\\\\\\/DThY\\\\\\\/hrUGukLNJoltWvTj701b2yg6oM9zwy0nz2QF+rFUMV3bmZhmR4xW7b7azTkOGhwVCeQvwWm5uN4lYMS3dj80Lit9iBC5xiZoRueFkmJQeUdmXyiq7XGPAxsNw2UVn6Br0LutqNv8Zj9LnP6eQdFfEtvVMb9MO149uUmXENme8R78phVaRv4lnQMgipf8qLwAiMeBEIacks2laa5KmbVhjyZ8fAPvlCh84sJP2OaFYNoesoXBoRdfuCnFAQdQSyEPCZwzwgy0mMJM3xZS+YIFVJDsJSl2w3O+A4flQRHCmUktfG2cNKIrV2cfTOyU4eDNcTjYANNYPv0072zAjyyu4fvwvsl184SRq1kkW1vjUEQep6K107ALJaJu\\\\\\\/62fwO+wMHKAODS4OOR+uicsu9\\\\\\\/yvb43q2y4YcK1ybxLDFrph0RBU4KLMvmUTppvqivAoPqNZwNhpMGERvAhf+eBxR716LhPTOncJmxPvtt0VahVu5B\\\\\\\/5Ynpd9WnVOpoOZbwyW6LhevcpcY+JiHJR5EXHke6XS3S5Y3ilD\\\\\\\/gbYv27cdl0pKlyqumMB2D6ICTcGikSxT4db0EiCboAXOoYyMhMucQVcljnf1EFTgJh\\\\\\\/VGL3K2QTCurdmE83\\\\\\\/JuxXUikk+wA8rLKqzYdbU+EejFlOnOEjMJ0V\\\\\\\/gEca0NrWq\\\\\\\/K2FRs8x\\\\\\\/Amkev+szQFp4X+pSpR3yByO0jFAJYkkn0QEFqDtzBuMSCrREjoUomI2y\\\\\\\/NG7QxOzn11q2JlAEEWMG\\\\\\\/+q\\\\\\\/sfnYf1qxcqRduo1r+AQYdoq2I549GhjpsdF+b+68hpYZGZddCxXS86kkKMSVTENmVH45RkWf6OFEkhm8t3eTmhJjVDRAMiA2iPpdyL48GisFTqlWzdX\\\\\\\/Nwet03lSxB+J6rSCURao1JPujfIiTvJhm7CRXO3A68fN9hYJ7T6bAft4uYr2KSrsUXO3oSkC7NRykfxqmUx0w3tH53V+r7HHswiM3H5DPu1dAMaxFtPaPg0zFCjbJbJnMnhK80qsS5x+cx\\\\\\\/xMBNCo+\\\\\\\/4MgzNmo7KA\\\\\\\/Bbg41J2Ezq3MBpkqjepGjMyYbPujMWqo46Ny2bvv5EdEzWl4YPQkNuCgfNdEDQaS\\\\\\\/Cg8m6FFu3i\\\\\\\/oRDuaY999G8\\\\\\\/N8PJ9cYbQJEkvH2tGAP5ffbtPGsLuzgsM8hj7JbqT2Z57+jsepiHlm7HKffcWIDBm+FITBcOsfXNUgYScr9wGQNyOACw4nQru2OQTsWJ1OROdgU9uztLusyBqsG1e7AtOZ\\\\\\\/Lqy+ZE+tgLMUPSj70ZZH2Tpx+lyhJy+KEEQEqHdV78AUKhO3eIM3jn2QThjyPb29Y4cu+6rlb30uwqg8XIOFhC4iNU4klYNbbZ7IP81UCc5GFT\\\\\\\/zXBgDFk4Xt1\\\\\\\/5OenYhaucZcuboWF5hD6U555JVwfD0JiRM4rMUNITBfu0yHVCJ6s51uJMlN3z00OGxJGUc1tGMeHrPCTjxuUNjVYI9XqUd275zjDSQC6L9+HcFDWscU24J\\\\\\\/mgirnrkbjGTh426R5owGZqvyB12dma18o9LQBce8IC8LcirjDl7fo1gFSf7DmrWGnsdEB7HAFciqn9XWvLxrH6QDIGwzHfZKEbmGuv8ej60FPPYcRXbDgoK+HcINC6gbzp6gubzD7QpRfU1mi3z2CNBJucyzE2oIS7U2sdTU3oFan2+Edhb\\\\\\\/oVol3f1KPBjaCKZo47XxjtbaMdi6BVJszsPFlkU\\\\\\\/h14BaWH1eSG2vOXPYHI0iRbqtonpe9ffLfkFIULhcvcvMOOyDviXdp8GxwiGF7ETR0U5wSLMEu9xzu4YWMFjawM12v374\\\\\\\/EGibl3yDlnyVcmKxyQBXiOsNGEYep8LUAPJEQ8J0qcNAPBffK+a8To07mVuPlfsfSHuwZ0fNba9gbp+F0fc7P7nNR+5XiTKUsbA0481GdSXdu+j27uKo9jZFP2rE40sDl2BQL6NR+bn\\\\\\\/rU5s6QEECZertop1+NKWEf3Jh2LsZw8OSskch7H5yI4xx0PJK3HpsOIxuqHWbHpr0h4ghJZX5E0dU6+uv8dzNKLuVwvB\\\\\\\/8iU\\\\\\\/Aq0vtwWiDbq+2xrZ45AdRpXb28+ntoW9jAuB\\\\\\\/guuKTYxQuh+ittfK+TpduoUcWzp2qyhcsJ3APoPs\\\\\\\/GL+VcW8wmcp9GKOgkn5zmrPP4nZxTpOSOoqzJsz+Nzebi7CWbnFcJjvX4rFYrp1DCHElgCq5MQG7J9VBFjrsr5\\\\\\\/hVzguW8VMEKGZ\\\\\\\/h4FHXXU62NQ4v8BkTUCmYv81VM6yTYwvkrUXBsyM+oOkKBhQMCP84fH0alyO9T8FFC+w1UmbNdEjMcKuME8+Yef4yxUnPLSAMqEzPUFZ4WErgQeiQ67pecxfeksI+WSFZ5xOK\\\\\\\/ZoJvKk7Z3miEpkGhgAY6kNGmhxJ3PL33Kq38OfqMvg2h3iPY7nRhH2R6kzTPNL6lJhdIvk7abDchBkszzUPiLxhM9QfKyEWfCpyj0HRtTlEJmDezQWmWIkTR75RPA3yIwTn0XwqI73dz5rbdD3d2qNhU9v9atew4FbUgBgOLS64JNqfBKgH+7pKo+Few7mW9Ktyvxhg5m08gkGh9zXcESWKjb5qoPcEmgmzCwtRbFiPvuZu+Qh6NS5HMELWZ4+bKl8YWroAHDzBynAmk5mS016vWmbs5TXfRpk2PUK59x4GKboukddVSJ8tXQCgl1iwDZfhp+G6+ikc5BkB3sf37xa1ailvEhCqa0z8yTotIy9BHnlDxe23fYxp6Z1MiXufabYe+LHAJ2mHjLgU5G\\\\\\\/Jm6QWN85lkDUfWIlqqCB0H\\\\\\\/S\\\\\\\/89vm4YajtgrVBEWLDAq4r059fYmeS9HdDs2hA0195elzi\\\\\\\/5ZR6Th4yRmcVGisvS64nApMtHpjcYBMr0wGkoYLxC1niSNxf4HOq2OQR1DNZNrXzVkLw\\\\\\\/l7JweWOdeq6Xi0+QZH\\\\\\\/tc1yAx4b+Fjw\\\\\\\/dKLiMMk4YVgAKOwgXfasfV8dMylnBpurUCb+uFJtclpXwdF\\\\\\\/Tc8sB+odrRXATMiJqxZ9KiS7MeFSYc22gXbUlzRsvVOxjiReKxcWuTmYLXQBvql0kNyhakee\\\\\\\/Ju3+KrYJOOIGPlsv\\\\\\\/kQYNMACgaEakhqKMsCxWsSc55dd\\\\\\\/CXd36mACQlQwAPjQo9nrlnwdb2XyQ8AbjEuwo9OxER4AJgSqBnBArCPMuqCHdSFPIXCFU5HB9f2bwwsYce+wxopRTwwRfR96hUc8iX9ZuW9tIhi+6rYd90p1lE2EQ5aKIsO0DK4WneZwRNWvv7d0Xye0qfot70UPJ3SwguyiLbzccbPYIyOyomvk2tw\\\\\\\/cLcz3atvo9tdsqrMwC6utxdPv1Nyw49\\\\\\\/PYvrIe5eAI1piH3xkZVZbGNiKvARoO6AF3qS4gO5Q0Q9m5jR6qMkmIsfgpUgpwP4lqbWqd7N9vL+2HbR1i4atr3CVrQk8HKTBRyFnGjrIl09QRbTjfRvkPYCjrtZkRZXRMINmfD3bCrfHiOntB854lopuIWF2v3dyRE0UZLrN9bBMpW9J0cHmwgwoQHouNDNuTdh46zq+D+GfrWbmZMZYpB2IpicN2Vk\\\\\\\/mZ5C6636nDUU01mbgR0hA2dAT5ebskH2YgZZjaDhT1y\\\\\\\/Lxxqz4EmKMh35BzspalQ7XWcgjz1EL8c6irYd9ceCB+Q2SC4T\\\\\\\/QTRMz21oefHr19w+ASG6njTTzymZ0ABZ+5B8KiRN3j\\\\\\\/SH5IEnHtF5DTtFju1FM6x8wnWJdw1054zQykQAzHdOr+LOlOqf+HZYpXLrAL+gWqr45DKcY8HYxHz2vRwmzrt71KJYTxuvy9L+rJK6uAP3asl10\\\\\\\/gyAzunX+IRAsx637jzWOnUltxTQZG+3IVmvzPVTAWkssWN3xftNNPxrKDtShFQ3qCu+WcfjcQTUFHmaTEAdpPEnldGXfAWM8HNuOxHPUVsz31YzGEdJINiwY2eCKvs1z46UtzPfFV2YwWgMfGEE44thpeaV2d\\\\\\\/WuQtrzwOfcNgb+0stxJM8Q+q9cJFdvaTpy8wHseZMlzFPUt53L3rtfXQWRXKgjbE\\\\\\\/rFM8b+wkQJt+V2VdsOrgZwrKNX\\\\\\\/ZUaMdBjO6klgAI2G0QtyHgF8DB6xyKoNNdBCVPXDJUBfEJs96DZrrFe4\\\\\\\/x8AxajPIbvNG41aJEhyUzmZAIJc\\\\\\\/jlsKEiFZKqjeMt6lNBW3WkvyQ3S\\\\\\\/4676z2CXiOTuBrUc9ZvrTOstaEy0q2QN6B7pZWfKe+B4\\\\\\\/YmANcbA4WBX+rfLWa+hWGU7IfawUbP0qixRCXmlCunq\\\\\\\/t4Zz6dhUa5HMFlTpdGhXXI6MpXma0xyfWMiLJjaV7ODJawEeRqKymIEF7k6tdK6c19\\\\\\\/YXegxR8q+W7zC8S05WpMKiJBX1k6djuFUm5kjjlWdqU3TvjZnG486KEBFcuviZBB5R8x5drZ7jJ+6p5NCEKWtebD8NKuv0s1FgQgpatTccOUw6BQZU+GEZiLwDauy\\\\\\\/MK7TBJTxtGF4itn4H\\\\\\\/o1bDAnrr4ZEr4deMn7xHcUroEqmUll6TDTlrVeWX\\\\\\\/TnDcFzJvtI2k3MbAvbF2bO2VPGuqe2H0uCDxkDwUgg2Ks4qe\\\\\\\/+9o9JhMU9EPhBcqbauWnilTmDie+Chz3Fe7S1Fe2a8dn3CoGeriONkAFEQaXGOaPKzyFRhQVyhTe\\\\\\\/vSiKZAyD3ocPZ4U5gz3pHvAw+Y88TzvjkScriclYbi0y9E3cmjLwelofNAqaB4ZxnlkqCkVTUF\\\\\\\/b0Zy+Ay1N\\\\\\\/uwtqFBspDsVUYEYaIIA8t2XnZbVm2H2GRZ4N5P81\\\\\\\/9VVzp\\\\\\\/VL9OZnH0VmKsGQmqJ7vdt1QZVWslsrwESaFPGT3UgHiR7ZdvbJTYNyKixOiF8+caRIctn1OzZ9\\\\\\\/SQ2JK3pWoJ3\\\\\\\/rG3mecc\\\\\\\/dwdrkMJX3+41vpBPZSdevtzIeS2CYoEyMfxAvcE6c1ypHLkwxwaxmtm7CDIxBird9RwgCpJ4UG3JlNmrHHCntl1672DYBxQ9pWz1qaAM1mKKeiF2DfDrn+ThOV8dhVoBFjUJd3oso6FilSflU3BsN8JY2zZtT\\\\\\\/958L2+PGIIt5kXVhoDGA4\\\\\\\/K9JC9qHi5wYnK9+16vqMUaBriP2hlh2zDvkuAxYPpUBuWRoq25wyf3phye\\\\\\\/DADSsJ6m0AM1qBQ\\\\\\\/TpcMvHU3JgJOYGwI2YeVUA0mddxG6hnwKTDKnVfzWZNWFysb6IWurbDVEuiVGqdFZgCspDgMgPAuAzfzozUz9GxhxyWz3XX3FCUKM6go9FxGNnpHExr7oKDYu\\\\\\\/1AhQtq7eqoYm3UTImkagoBPu\\\\\\\/uekjfFqydXaWuYlXAwb3snMPSk40CR4OBfXN50ZumgPjSQ5SVsStgbOZ6ItuTnPl5fOBgUHwXzn4HeFzY3p9OhA+N9NW9G\\\\\\\/z+qVuQ6f576WGog2rMOIz9\\\\\\\/cz\\\\\\\/tkX4HCTdXe5PuRLGye0qXsLRBfrKCC5xuObL\\\\\\\/LrwcwjjP+XQGn0VuHpe1rbEvlfmGD5K8t92\\\\\\\/lcEhFqfDIKBUdELeBBWlI4NgbBgwJegCcEq9CLOML3Rpa7ERFxIqCSXjeYJpAe9sCRFsNw87QQbkgVqrunbQNo6HdSWilab7epWv6NLnDXyqZnTXJ4LAXqWdeH3Y2I2mrO9Idj2DnJwKL8CrjT7jEyK+YviAEnbAxC9NMDRwd6IzIBD+6a4M\\\\\\\/wJaWNNK1e1WvLxVH+yLpWGgj06+6iksKKm3by0L3SwN3dkAWxDVAh+T89fGRXZeCknQTgBHo8P4eQ\\\\\\\/PchEkP1N7K8PNkFXkFLjCcPdWNjpvUcOE9Qxi6g0TZkaGgL58pgM5Mp\\\\\\\/UJ3R5UqviP8hnILgUSR2RHIuYoP2yejGS2CRU8etaIIPuD3BRLpy23Phtfti78+23OjkoUrGL4CPZrJOirJ3BAxlj1iHINJUpHvrrj1X3NS1tfH4wNIO4pFfyDLsgDT\\\\\\\/S+l8G5WmfW0WOu4XG\\\\\\\/jzq3fE+VSE1AThYhp8\\\\\\\/VP4441lIWFoAdb9NH\\\\\\\/TUZZawuzUE0lOIUyWl\\\\\\\/Bp0Ot9V4IZGvXcY0wRkXVsLslM1bl9sIRQalF0\\\\\\\/CD0\\\\\\\/HVSwEg9sb2REZRX1uxuaYUdkAAp84EXKbsVpAyFViaY+0\\\\\\\/txtaWrsGuOWMbamHEVmjmywvbgHi3zGpCBO8u3WGIqJH6FLGRwPv7h8hUN52KiPdOWoG4Z0N+Uh6J5soPtc9114bx5gIGseE79+NCUpBsrWjTBVeI9VyAA7w6V2ug3edSzFbT63RMb5T+kEool5lE1p7j9e8Z0I2kaA4n+bwpXOO1B2HWfL9qS2m\\\\\\\/fo8Rshe6LXjgNKs\\\\\\\/30qxeEN0CiKBREOF8sa7paZ2bu7dzlGcGPATKou92HMuwpt+OYvlSo25R+ZGRBdXVdnIDEzgQi7NpyhYccxf4CGwx+zk9opbHbKkd6QxvOkGg1SYU++yEFIa2\\\\\\\/x78nA28dkJ1q7shvQzxGK24LDRyj+wFgPO9\\\\\\\/TWHr3ogqb6Qn9Zfd0UaZH6qAgrRBKycYvytHV3jIGx23NnBGFzPSTBnKL4poRZLcpj3NlSaICUWckri8z6Oxd9Ja9OOqurXUwP7jiAipvLzcxNyguicO6F4CCCBp\\\\\\\/RlJzQEPzpz+MVja0TfPFuJoQDidMGnxgG+K1BewOYeXE\\\\\\\/DO222Hl7qJaKuH3ParyITY3iDr6qZO0672WUBi3yk5vhL8XAb+6S8lqpUovbarYnzucMfvUNIBvKFC3T\\\\\\\/sHZpS9KHNt+f0QYrHrCPj4m2LnhWYFJ6W8xkgnyFBbn2whhEHIdimIOU+ZyfGUyqyLBQBEMLNeGWwMHUnbX428udd9K75FIzdvJ\\\\\\\/Zfu9oevOzTYTcX1qxMRBbNoyZdOjwbEf09psuxQxKYYyqm+qqMXBGYy0EdnLNnvWc09v\\\\\\\/nLsXyOycDi4\\\\\\\/9OcK1BAByiI2LnKKXMxT1qLddR63NvzuMiPDpH\\\\\\\/8y0WntGLutpNDLd3lWVbA\\\\\\\/3FE65Rt8ynp6GHMgTFh5NO\\\\\\\/0iDP8jQkXfjVah7vcpP5+bNU7iuf76c0nzhcrmHKuoQL0w5OjnOOzD2ngE7GwzcbOwDOK39gDxL8JgVGmBJnaHveo5ucoVHLvsLoibPY9VQIa4Fq0JTWypJk6VmVgncbMHGvf6douJU30VIA3QYi4FYkE+djLtnYWYjrmOxx929lZk2vBsZ2NTWZ7a1rIVhM7kXnmz4UUOD1N7YDO86CcnpkMzuU9D+y6dcUJt\\\\\\\/J5IfpLLPbof1erJvJKcjqGBpoL5ugcE++PDSjl9Ii9jiwNDTWyBCUidjy5pxZdD8DgspttpdgxdENsLMHVdGaSBmDsNqFQDhP6kqGHnz8S6esBh4DklZV\\\\\\\/DBDoZ7FVitq\\\\\\\/CeQnONWUNN09XMx4v7RvdRoLvqwiiioENRYSREub0KCoyCAxFq8w1sG7JahwvnGq4q0hngqw52p0Xn1PAoUZwRNIQm9IK5m4L6jVzJ7kmxDbJGnRXoPruvqUtrdvsN2I0BnFcT04U+Tsbt6Q9sqfgD2G\\\\\\\/U8oULa42BSwejAK9xeXPdoyyCfccjip500h213ZRgtFCdGUNTnNQAa+FG1fWGarnQEKY+vw0UEShVMzUkyE0xFGQRdRDDBqIYEBpRpBDude5CU4p1HqD4EsMNe\\\\\\\/Qt5GTZeb8QAbRYQw1eUQ+Hr\\\\\\\/62V5K2kYHtEi65wSIbJbPrHW988MjmhdskO1tqbdUe9OwMloCI1YrnChHq9vl2i975HqVIz4TvK\\\\\\\/SiHPlNOJhXWvqD13QCbLUgKvBqm91rBURzuWTFryp4ZI97z86ygLbXWaY\\\\\\\/8abjlA9KXyvDzuAqOP0hlGUu5eTvPmjDrEJIghFcD0OesbYmrlORIjrWhlZMHQCgc3SNykWWMG7K\\\\\\\/TVI7VE1tc\\\\\\\/7bNZtQgrDXJsW59Xa0mUE5ILmGeUcrQIQSc63RygtktDf5V2wRoWptIY0GbhFSjRcOAzQTQChnKtpcySIAEgFM1UYe+rqYUv\\\\\\\/xGP5ZCYQOSs+50BKsPcHCmU+NT46NTC1UUee+aff0YBv03J8JWuaay7T7FpRy21LrCdMyEp2vswvP3tEf8Z4rHBs8fu5uM9p\\\\\\\/YVV2oJP2HPpzLtTCS1Ap+FufYqPNUYq6a1gqQARN62rx5FrIZmEji+z94XLDWLdkoSVBwuxiLT6Mk5VpoSBwdYK1B6MxXh+seMQfJOQp\\\\\\\/0E3GwaJSoZd2g74qkjOXhK6NlItWjAn\\\\\\\/aRACPRzB3Kj6DQ8ADTxJCNeY6+4UOh+vEDrMWod0+nhlY6AnGmv\\\\\\\/kLVpwEJ8wirBTWW9CbKjEO0UrByeZNfO4IhQFLe0hwFAwzUBz\\\\\\\/kNeAwcUfKWRHaMXmQ7CKciHkoK32\\\\\\\/MFPXgodwxBsOxvdLO3ala+kUvT3CWrsQ7SRB7ghlfbkY0xRb3GNdVFt9a3mWjzncfFgL7uLYHos4G4W1Em4ZTWxk1e5Uv2y0ZpTkOZnp1K+0+8NBaqqh4DMLI9Q\\\\\\\/EFsEH\\\\\\\/qAmwebm147KAS7bPQbXmEMGgIxz3CTPNBFMP+wnKaKx4Dpjm0eoKeIBuZZeRgMVfKQYM2mpdrOLHdxkTMvlXg8KlIGrGgj+cmWpL1hyO7\\\\\\\/BKlOkm4m7Nox7wfUl99bbuuWP5Lc7hRwwHujNu3aiCZm1kRREhX1YF86trYvF9O6PTfVDj1aaDWgbK4U4mEdxxdTzQ07vCTe4rXJB4j0T4tCix2IdNZmgGKOWNEAbbf4JP6dc0WsNRJ8nGQOO651qwdYKMVbyXcvrCy\\\\\\\/7sUA1dK+aKANMHXpL\\\\\\\/TK+2kWRfx+TOXW0UbnIqbk6G1fg6D4BwN7lqWxzuQPhy6iq95iQRT8v9KwAHSI11IZsUKcc6FGtmiRFJb8LcPpY3BZLIVLlA4skvi8iMJJGRjkmN5bBw2DQhkJh9ZLgXi3YCFnVUhiVNqneZqnw+bt9Bw9XrX4HFBvzL+tdUk6POQnkiwOTuVwbkjBQdvrpAqJ0lvzAFSEgituA0TAgpLPx0+MCol4JwHRKL5sboa8RkrDrwRZEmsZU6S+++46XPI23AOE+It7XhmOf3H5GOIc1LNwoucljkRpETONdp8qwxIulP0m\\\\\\\/z4Bda5Ncbx4upe0ijgDRXJ4S7qbne+nU7B8M8uMXkfDSXnnW14R94AE857C1AF71sXtuDpZ\\\\\\\/nZinNZPjvxn6XF3f9LeQMGeKd6ew5444zVQTTYeICAebA3scIX6DMYxegNMxWiP2Am80hiTCLQZoeX6AKUmttyu6h3xiPVpN7TqQ7gMIAJIrOBh33PnsvXU63OFukpGNihRJGEd8Jyh1vxBoAcHSLYRK4Arw2qbXL8zxfee\\\\\\\/AmLbLyvqnmBvI4Va\\\\\\\/w0s\\\\\\\/kS5g4muAYFHL7TtA9Fysud2rJl+TX0g9UUtX6Lzgn0buEezh8p+MJJ5iwjPCdTYVI1GRXB4PApMZmniDIG0hBL3BuD926OVBSADrX+8Ftjn4wYIzuPbCh2\\\\\\\/W623Ak5UOTJ3Az7mDBDHKHPKABrZ2roMgtUp4dk39\\\\\\\/oRHE+7lg3aTAWtCgnJYoaRif48iCHNlUW5NHC+VQDS5rN5EZRszo4TH2qDYGwFX5+lqzGxSlc\\\\\\\/AKYJzxwUyi6U7HsSXBV9DhGI4gkNJSGu3ibOSkQCEVYD+TNYWX7P7yqFK5i9l0bcX2jjNtCB0JIexCwF29XIWfzbOgQ0knkZZWRRWOZiqHvG+rE0ghcbvwZk2lE6QhiLHcBUNZOOaHjh\\\\\\\/kGUv2lZ8fazZrKzZuVcSvxnWJFS+4mstLSkBObljeoKcNtNKjnUB0P9+\\\\\\\/R7aEI0AilDHNGqNND33vAfqHJLczVc1cMsU4MiVGBW5RB5L+aTH5tOCURfErmVyRrInGd3KM62GkwTjtQvokSODJmjTe8Imdv4ihgkayZhGkfwrk1AFLHCQG+aHVTk0rddWvokGm83ZiqD9Zv79qzw7M\\\\\\\/u1yEEoJeXpiBjQmQSKKWBQHC+E96LyuryrVy0bcdBh2wTZ71HlXAC74naqENvSFvav2YOZ69jPTxHid2nDzHtBEvoRk0\\\\\\\/NM2NtPgBagYpOgk8epxaBzrBE\\\\\\\/+pomULo1EAoHtqNydxnc0HAtUZgSo6HNyZEGVhF+SqtsyluWP1bLBYH1EDFdB8cPwzrtBexfFRRE5gSg+AiC9wyCnG6AK6kP09uInYojtmU7g6jQt8N+VsXZaQdawcjKctzpD08OAkw3t0iayXrNG38Tuajxgw91fTxHc9FyK4WDFpAMSFKxYXvQCAy6lay2267doaGg2HPOcxkIw\\\\\\\/\\\\\\\/0iZX7mqh4vyO8oPalLIT3bLotoXvRpGZfcGQrjRkYz8oK2Cvm+rsKbqZgW2xQdU42IsIraFCN+lIzEuO2fa4kn2oFaaowmUust5MF3250\\\\\\\/XLFbUNWT69YxvoSDnDGgOyfjvuEffn03JvCRL+sc0oWna5qmmWm5fs6lwLnks74ZCjZdxzegktEKc92wMl0Hm3sTJfaaZYFP0Hz2ipBVVLqbuoO+LzTMsDxGwiTgFCiO8Wstk\\\\\\\/0thO0II94Z9wtF0gtBzb9m3vHid9ZhNUVaZzuzatMnOdBhsPvixKWdJKinm3ORHbq8mqPNGeM3KkVSftHo46HYrj3\\\\\\\/cvw5fbG71FhnYuh9MSZTsW4wRqR9UIntJ7RreRKfR8gGS+7vCSC5XgS9uc2AB7XcEO1dJs8Cn+VCn8fhnym4wmYSIGZdDSmRsp3DTxAx7ZfB9cEW9wnxvit8aMtccMz\\\\\\\/kSBg3jmGCSu0dnGpdLOabRmjOe9fYjBQNibl6OVO43CbEDSjXxVuRH3eMbTtjLMcS47H75Y9UTzHSuEyp5UkDIwJNHQIoPlSDgwGDnzoWNEdo4aZ8qFy39WUOztQ3KaNjiMAspFopBYE18spx695C1jA31\\\\\\\/TrOirRt1tAQ9fNj9969+CeirK76k4odBrG5lB915v4sF77QnVPrV0u3eYsWdSf73qK0ObclsqYpFXXwE9sHq2Emhc4gDPwcVhjnbgqXFP4e2E7u61Y1ImKFpolpkjmrO4DciV1mFsy72ulMVYDbE4l3AkidAqslPVWAA4IBDCQ4LIDLKdMHqfaFyXdWu6kDUrPvVaF1jnKxoosjYRbf+5FFQ2Eco+Mp0cP80dis9oR1MIKQf9wuBy5oEc4bvA4Ejrb4FvKzQ1rw9HY0rykA2Qu4fMHrRa4aPPyIiyR3d\\\\\\\/016XyKukRjP1J5zn01ov1bE5mlh0jb596NRT+rQTLicrkAh03EVvnfAuboSvKu871BhE7VQB8pWYOfeHA7m+\\\\\\\/vHY8NmwaBWvhPttLzpOfPrvR6oCLOCm9drjP13LQ6suA\\\\\\\/8hielXdwjWGN802Cl+InjAaz9QKadmy82+bTpUU4BgXivHCpavaKuKnVbfR+SVGoY6W4mUNDz4FwmH3puVnkGENJq2r3YkzDvd33Qjwev82he0e3m\\\\\\\/fbRbJgGdmlaNecTpAzTT0nbJuBug3uu\\\\\\\/biPauReXcbiewKtXo4Fz4+bysb5IgdYK2BEF1o7MBNQbYyv3LC23c6Leqs4k0Rdv6gcxRyLOypsIutxLEPrvvzSuOo92mGiJroJa29Yu2GHgp3kt4XENiFHzMOu7ryyAdV6xyT8Snq8sBVcP1qZ8dFhH7STwAyCwsE6Plrh349khuYZ5D02g5rbwf78wkHyt+FUhuwz9+zYud0HYGFV4T0gAYv2cFRDj0w+ow\\\\\\\/DWpnSLWuirH4e3BSIi+OgfAaROfh4MvH3ivJGot6SB4cgCNG67ERlJF2wn+cL3X3L8bBiH61fFyBzDmvRAD\\\\\\\/Q5f2CdSB6pa8\\\\\\\/IqjDZ17kTC1hyImF6iCzgivWzukJqefgddRM+67Qy5npOzI5OoRs2OMUBB6rndzcR\\\\\\\/6mYBsdboUkS3RaEyRLcVIDMdx4e6RoDMc+jBF0Ek301s3ccUS5OGz5AdiZF+P1sYrRNz03Vwg72+mYKbkyYR8SeADncy7Nbd4TQsqE0sCbFZICtWHov1CtLOMmQGzot5n7HFUDK+02DgI0qAJa0zARqqFsKiLK8kcPbhW+\\\\\\\/AZVABIO40CxTnNiTGc87Aoc9fMh2V2dDaxvsWLNAk7qBNEEGwghWJNntX3EB5p922U21UuOISNAoCA+9cJkLDFUl+9\\\\\\\/f0f9mdh6yyUeOLiru57s4pnD1HAb7eK87yGs9OGyRKIKUqUBscEeapxE9GhAe8ETXU7bd0zGxPtqsyoYlegXcOs9Qu8Y2Z3I1ecxJ35r\\\\\\\/znY0whiLe\\\\\\\/MFAycP7LgLOd65WBZnlp7rJFb+t2avC83Iyr3vO4pq8STMNCbSYeNRWJq8Gb6Z8CT74lLcUs1LpoICrVL1wrq43qb04xK2jXJdVVbBonDRoUI9NEzlSRM8CaAmviG8VMe\\\\\\\/WAIRV7uvsIHM5iwsDvl1y5qNE1qmBRap28iVs7owHd0k3agjkbzfqiZi7eILhWmQ6rayNaWmMDnzaE+XjyLfDaqaF6wCi8oXabYCIh3+zsFJ0NuCeZB0Yup7iWLPZhtvSYypTfhuZ4j8\\\\\\\/KXy8Yw4HJoQRzO5fpKWUco\\\\\\\/LeAFEjpVSsJtSOW7OH65oBsgJlVL4nSYdpZ9qXHzABI+u7H2IBhX48leHgPZFBudQYWTFHDhg+Af7+cvetlyBTABShu0ly7KKD9pbfi6GBmOXItEOTf0lSq3N9AkZX8Nv\\\\\\\/3ealRntniK+m4RT312B25RP\\\\\\\/UvdRYtXt8UTrYiKo0dhOMPgaBOy9KGpSl87Di5QSVimLRgbTUqKscspSib2D0K8smYDpq1qnDhJkEYdfkq\\\\\\\/jNJGOyZSY2Yg6Xnxiyf4kelPZHLrXN0Vrsrge8HPjGalTcXd+Ld1gw0JQbIue21S63NfBw30E2hC0cEI+rmeyabEzjRT194GUalA0DY\\\\\\\/zGspmSC4leW4+mYH7cocnRx\\\\\\\/ExQyFw93w51rQKE0EhMqo7KSa9Udeo2OVsFjavWRFnHxCQLr8JhdylI4UENp9NPsMH9SOA21kUoQIi3RLDxt8lzB6ubnaMrDwIS1ALCUETlTNl5KLqICzztoTngjSACuG+i0vzS19oHRD9agDEXfrXJzgq8XgFQBmSimQUvxXRf5+rFyTACyW+lIBI5AzRmlo4ov5q3YRSMrqWHQK4QBRip4oaC+KEdcLSuzxcQct9dYzLh6hVvG4vXVfwp4dDKmSneOfyZKs8WKdutV5WVeS7u1Ayg2xfYDETDkQjP4OHEIbVO1tSESCe7c2fLY2odS3sc8N4a\\\\\\\/8vdS82k1IdIXQXSAzn9fgzJvncRjF0GTdA4tDRmyxRch8CNFad6IKK4Y5k44MN1q0AocXW3yvUZP\\\\\\\/1AUz5flZoKFOuLQbn0\\\\\\\/WstKkLhk9J2SX6TTFPoRlSkbWIUTZCsEchPMgsj4UjlJKo0\\\\\\\/qB3a63qeuT6mEr6SJAVgiw9Jlq2lprxsEm1XGxIZu5OEOsVXOr6yztQdY4a\\\\\\\/2trpDYUG\\\\\\\/U1sOVzjrH\\\\\\\/BfKrfrHgZmRT1efyyZmmCnShNpZvBl7roS0cYMdoU6hYae8+ULr6+I7Iijy3dNOHcatWsaU8PP4Y4SLHb7d2jMqbkR4Y+5fCK6hTj4kqXWduaxBZJlsp3UzcFjPJOW8zUqf4PeWf248w7OxTeGBfs38Hv5ppZxRCs0tB2OfUisKt2iZfw6VY4DIrJL2wsdoLrFM4BDFpE7taLbzEcJWufApBGzdx3iV8cXhVPz3bcu3RRZdKyf2GVT+cOZ4WdBRhyJSLP5eWgUbK4D6fZBSIyheFZ87FoDMhtCODnko0DslVUDluIjaCQjKpjpOszFGAFEnPPrsDShNspf8e1\\\\\\\/Xxvm\\\\\\\/sAbe+dqVlFT83oOLooXoEqOTwnI3X0pIpaJIiTRczavTAAK3AEofL4ovBw7m7QKXVsb551sJLkV3qUthov3FjbsQ0\\\\\\\/zy7h+Tmoouq91D993zqrL3r5PeZDQafT3ma+YKQuAWD0sKoKByq43NeViIfvl6MKsCzcM6kHIs5YSZqtYp7EBUZkqKJQ+NXiCl82hhTwpviTz26Yya3HU6tethWKA\\\\\\\/btzeABoDoWPvOf5G9cgKypEYVR7JeyK0xlILNiCZMvBYqNnZxYjvzTkWoJBjiZ+zIWpECDa8tcHWeMx+zz9Z0UtHjncFo2vorQB8K39JxuBwpiV0CmvzIn0ovFxVKdHv6otmDDWXVNUuPDZKVYUnV7n24+kZOd4y5WS8RZn1NLZmGv8uZATz50WFd6WTA\\\\\\\/n6zFtqyS+r8481jvlBGobgunzf0wMWrpOW4meagY8md+\\\\\\\/ACpQ1Z9dVIL\\\\\\\/VEilsZ9ZXfYsMcAuYlcqlsaRcTP7liNobNyWvoprXzIs6zyUUFsrrxJ4k5djlUGUntbyU25drWVee8AHeIa9msRZE9blX0SHJ7PVJxui8xLzvCVyqpth5BzcJ9y0Yci3eM\\\\\\\/tLnxkpsMtcCtSrwMNiiV0RW5JxX3EOgJlAihUy7b4VzsshQOpQhxii2OjhZc1b4UxlwHvmGOddcKwQfKHrbvrj0FMpYj3di2nAYigsb8kjhL40PaluptB1q2J264B4V84pV60e9sFqYVbp12f4jNyQl5S3poGyOfDlTWgCwk2tiTgqeeXDx8quZiLWPOZkLzd\\\\\\\/b1hoUYKHhkkfUBXTyiiuP2u5ygDHTVVi49y4EcErFfQOvMsuUMzzyh0+A75ks\\\\\\\/Q3EzrmANhvjfDPX77iJImPwlxnk1m9lrNHcONwvi1mPNhzWni7o1iJTTqNsAOo8hK2QVGnkkepLi9jXn3dYuhXF7TxitoAVsinpShggNuMuH0LAbZIGCFeiWHDQLCE5N9PWXbij3GGwCFgwXLyb6igs7ee4cXFREySaO7ueHE3QFL\\\\\\\/Uz3WrdljZ8gCjhK9gFtG4OedPnEMrcO49lTvSXf9qvEA19dZrMkeMezU6kPIBUUQEC3ilTMj+Qavc5GB+zeitMuH+S6loLNSGCqCCSHTzcIda\\\\\\\/gYhkILnJyTOCU6Tog+Q5pyuOM6QNkTJ88N0M0kp5w\\\\\\\/sG+tyJJj8Ymy4Cu9IPT0v4xl5PvRhTFZU6OtymuEwLcCXS1+c6QEzqlWSroOd5Uq4Fim3\\\\\\\/PqN5CW9\\\\\\\/+UrWA6zDkgJKKOFQGWSfQ\\\\\\\/\\\\\\\/h7jn9k0CpxXqMmrBWXqDowbekyH3iaQqJUWvTrfnTyaqCqSr6KbygfTbqG5EaP6R2LxwyFMEx4SgXQKioKwpkbAyadqB7zcK4fejXw+oNJwRXgHw7jMnJo1CggOD4CHbPTVe\\\\\\\/qwoc8evwAp7LtP0HAOlzxX5+\\\\\\\/\\\\\\\/uaP2ji366frdQ56IFd3+jU62AfwtZjeyh\\\\\\\/dNwF0yRmkVK\\\\\\\/hIySVlElYPzvPrHt7DutudfOYV7MM\\\\\\\/wm8jSSLWbe\\\\\\\/N8Wzuxn5LqR0NT9df64EoyRKkxJiJJFA7fUbttnfFxUqV0LRXGmVgBn6CyUgnSLNSM8HEyaWnuLbzSmlrMPXKGPZ+CrigrqonQOazx5tfFsKIgaOiJ4mREqwTg83avDS0joCQM1MWc9s0Gkogt4RPw0zJ\\\\\\\/4c5e0aqUeZnbiz0Qz89Vt8SfDKRVXgiQuugoruoSCYHWC6cjgUUCuFUL54ISsrOullJNqQ1nmm37FgmjrvJi2\\\\\\\/PzI62y0lVRdazvioKpA0zkEuDVmWdoiGHEXSRHxeUgHNrYlF8YWk6uHsKvIvkBANwvk+gTnKn8Jo8TNKjsrVslf9+McUHhBUDipNPZogkv07A9Lg78Ou+8jBKeNzOTbTP4wJx8SXrJtyvskjF9A3owMv4EAv1V16zywAJ8+D67C8eH3DKREBsfoXelTbe1q+dyD+LDRca40nCS9jXyqfikWrcepaNvc5mU2n+47mt9raMQhHvsjlPsPbLInn78o6rVBIIokcriLS0f6wAXuqN8t3Aivesx6HnEyYjarDL5viXCEwXXExs9Y+0Tg00f1YvkTjOV+5NM7\\\\\\\/pESpIjIBGLuM5O8pGNGtbpIHASMd+JcARRjAFoRS469cHYoW+IWhqavQkbTPtQSV2ZrfiDVmZKBxOw9sSwtxtGvw4RWqDde4s7WAZKP+k\\\\\\\/GWPcSnQRI9oWupjabPKUu11\\\\\\\/CcLSC3RJAtO4N8b8v9IqwwFx3HtWjgzgo4Lho6\\\\\\\/x7sp5n1SdmhOuPARHBKyI74+wWAlwxH+TmqeXwHfGMFaLxKWs2U6azrqsr3RKL2Bn8L4tFi+xVlzFPytQUb1KCgjCbEB6\\\\\\\/7tbRZayKiGVBaSIuX5rcD5i6ThJcGvO0GHs4JeNawAL5\\\\\\\/xLGP3YHeE887bI+p3JnjI5F78OEAh549\\\\\\\/JG4tRL4C\\\\\\\/vvPe+mqQ9WPLD\\\\\\\/PrlsGNGzoAJInk34bYPvZytqQAFvRO+Y8N9uuczua7lA6MaJPEUd9LSDgw\\\\\\\/tS3c\\\\\\\/QFbONfK9xszh\\\\\\\/cymtuthzfyUe5A5P+vbcXQmqApN+lAJPJulER10NQAGhcO+0mp9YMn10dCQRUtgLP\\\\\\\/rrfnrniZhu99JelWqhwDUKiGOqDHidC6y45CtWS4sG8Ftgx6beYSFBMwfHEdtBp9b0c5GUrUUfSClBPQuSQBOIRxNTinPBAUlvPrJo22xD73jDDnA3L8e4OtdqNOgGgatyrlbV9e5V+E0ByPnhIjQns8L5nfRsMvW52lAZ2yUC8a3VvhAlM+pd\\\\\\\/rhTzv9Vg3h8UBPnvdg4ZIYcEUiw\\\\\\\/yFgRrQHIS2LIA2DLVnTNpF\\\\\\\/K\\\\\\\/9d6dJz2DP\\\\\\\/kkkdXkShSSvVmPMzubvx4B5MN1AUyK77RP2UKGOB\\\\\\\/I0EsSq1erMKFsnHQFeg9XmC2nRBhFI2rytOzbrTUI1HX4NnPMLcYioelflSmCDr5IUqmqi9bMznTjdLZp\\\\\\\/n3erQBILac8M80+OMAF0muivXgAqeNh76JBPD07Q9hQK2jvQqGpdQQAKiWXFDYhxN6skCBQOB5S\\\\\\\/kJvGJuDIAv0b2bBqve39wTzCU0ubvisQ4NOjFtF\\\\\\\/OFh2XtW6\\\\\\\/z1fAR80epvnhWIVQCStH+Ailrpr8OZQIhTxG5KjqJkuLDUavg8N2TIiMsa9cqJVt8C5EEtx+ZjdW5fkjufdbzpBZsVoXBkTK9mvSOgTGOZDnzfKihqcejMiiX06Xelvg797wpw01VD5O2d3EExm2NAw0faSbq7gf2EkYAP3mVoyzOAR6Dr7VP1sUxhpD9gileK2jQeySzpQMiEmMNpHxKLuVQ3hObNKBZSXRw6ZchZUAi0ywYPvh5iiiqjAHgDiSJIz6S9YoB1MucIHwVavTCtllAOUVuuk0h8Nt25NS05CgMG7tP6AtiZMflDwx+2ypnokBn0SJHvQLLNyIVOrqLGtqVAmgEARtISS\\\\\\\/rmO8HnBufRgZDrkWIL2iGaZu80R47olPUqXER4mOjm86LLMFOCvKRAQCWq9HgJFZWNLKerwEM+y\\\\\\\/5EhrTduinqzWGaS2Fpcap65UdEymRFxFC3rmqngqzOVCVlxZQImEiTIU78Ns\\\\\\\/CEpgB2c4tGLNj7\\\\\\\/ziNzEg+ZQ2BbdzJqZYllC1uMn9YkR9Lg+QCJcEf\\\\\\\/91yhfkyEnxh1A8jhZ0EvAmDLdNkhSYHGeRTLFrUdV1IZSAh90StUFkR8SXodEgZAPrWD+qA9bzFi7ymdJPi2ddSeAuvPYvuAl5XoblX\\\\\\\/pjX7m86CHnc3o5E72suVQySGQvZGVwh6kJzZOse1B39VR3T53ENaytg9kAQurfUm+M4TkQds5USpjTdLk1GX+GZ7Nz2p7tOQUjb3qCBP8vexsSHulYdQGZMBwh1Cm6mvwHCs09L6fdWNZuUivR3Nx33gwaeN+WnAJ1jRFATkU1fwTQaKhEBQhBYyNtN8\\\\\\\/tJg85lghksw7hrwmXY4s4CGIiWdoedTCRBPddQX+DoT5t0aGKaWQZ7sBQwELe5nCiOCsPOVSBW\\\\\\\/5MEt41F2kO8NRuSfBStFDrGNGtLo6e1RbqXpm501qbf9sf0kLBL+6UEuXCXhyMnibDboRhkOTOtZ4LKE2pY4YgRlYtaPqk3o35nWf\\\\\\\/GuRimiDsJOa1EoJRXeaEqPFOy3J3NDZrS\\\\\\\/2qgTqtxtNlakIHYaswf7NcrpJtZZYupuirvmaM85bdRNo7WMn6h41zCJBnagZ3w0\\\\\\\/zJmBXNuldfAAZtSwKKPf2ZXAfw\\\\\\\/aaJyWvz6odpl+s3POG9TIvKG67M\\\\\\\/tvguQ5Y4\\\\\\\/LUBX2nck\\\\\\\/oL7KNVCVAOG\\\\\\\/Bfi91kOXnFRFLRYwx7coKkrgNacHnjtvFfYhsRXSZVTlXLdDCEoS622oXBJXgYqPbLsMshTof3YNkWLaAAB434SzwUnxkGWIaG5ijm00Jdig3GxMjQjIkyehwSHS6xsWdcQOZRi0dlJi41dlAremwKfV6BTI0KBh8m7oFk+VKKW4LuTjSFUx5oyKNtXkWLQdi423UNWsfk1dGXiFJsPdjgaqrRCaLGluLQeTmTxXV1hxOS2LMoawONB5Co715pNjN2mxgSuONqiChpooJ8SUJYDSdWTY6r0IqUY7O2we12gy1G1PTHVnwiibADj596r1CzRWDmta9B5UftmI80KTZXteKX4zg5ZFEvcWH7OhnJj6zfeZsN\\\\\\\/4hZmtR5quVkuZYPVkWeWOMu8N1cICfL2kYy1RVUdcuLzUjq+Nercqpi2Ppo4+bRSMNIktbuNIJwfPCgLl6txehoRpqHvilrIfnPDvtDIHrcFTcTR1sWwMtkqSyffIkREoZ5EGdU8RS2FZlGxdVdZlUxaWlpS6IMDXa4vCGZL55D\\\\\\\/HQp1q4a65XrgvdSphcBHysMKkL0LbE12i0Tthu\\\\\\\/nocOonepHDejib1q6qeeA1TpjLRQa2snFw13VH4cBG9o2bOgnFe0Avf46hfrUXWyX527nvP7nsB+kQEWB034Q+9kq28xMn5mJSXdpGJmbWdAX4NSdOO1Hz0e03JmZO7sHzlQaIfQUwKS7fKwFv16ds9dm\\\\\\\/5rybyjnRW6WJAhuCsMbKhJG4VPrfDQ1R4Lk30H+5wyRGc0Hfy9ORnRp0zxw4eyL5yQc5iJFEqMVpnt5H8InqSR80vShWmZHG4ZkEyc\\\\\\\/5cshzh8CENooDrO+eF5r7w0JJK+PqMgjfsug6J0Sj\\\\\\\/auUmteD1zjlTTWdTzEvB\\\\\\\/zuhzRIBlNYJEiEdcQ35GnrCcZLgj968xTAtKz6VWzv7JlcwkzFV7wfPXmBdZgZH9xfXXjDerhEvA9ItKqGYJjtX2mLAHuu3Pa+bZi\\\\\\\/rd6VDX9+UqjPb0oAw1Zo\\\\\\\/\\\\\\\/eDLIG+LcF3IW8CGkQIwFOHNItWra1dzH3C9hhUwBm+Xpy+NV5vxt4tkmjccJe0RJqwm3VJzkD1t8ZJsABY7fAbGGZrK9bEbjFWGiGKNjKr2BRbxCAX1FzB7XXD26oHfCbTmh77ellBuQilM6EynoxuYyaFQN\\\\\\\/YIRhicT8LXRgAq9LRJ\\\\\\\/bRTkdO1JnvycLi1aYA8gb\\\\\\\/ObFvPnioYByWWNYW+u1fX7WVwl+en9Ee9WuQ6qFtavgl9IC9fWxlgwL9OEDDq\\\\\\\/z\\\\\\\/mT+KVJIbwzI+gPARZW0ZlEq6j15d2bf4cKggtdBwlGlbzXXr0U20f\\\\\\\/NMqVmBducHT9arMSbZFoe1LeBXKEKDlAUkqvqjEOStFijS9FpaQmcOC7wfw9Yksdewh2C26twhN9+yGqrR5Imdi5\\\\\\\/DnkuW5g8SSNZa8OfpB3Qg+KjLFxgxBm2CEBxyxUKth170E4QcbuIxT4G8fO4DaRJI8LHqf\\\\\\\/emXtIWp+A05wf\\\\\\\/LsySoLd2FhiozHuMLFsOtSnql2jAxiV9AP7wNAV+3SMPIG50Fn2CYiebAACeu3cqIs9VHcold0DHMtnoTQaY\\\\\\\/m4zWiEiADS9QP6u5CEliXJZT5\\\\\\\/WM3XQSXM\\\\\\\/zNp0b5mSTJR+x4GLg1Pm9vehPaWGfoGQqdHAfF7P0MPtRDK4ptEkwmJMesXU\\\\\\\/LD2en1BoawyL4G2aOmyVHOkRmaOzwB+69\\\\\\\/7V3Qshi744WZ6mH6DkbHOJg\\\\\\\/\\\\\\\/sjZK1mvo42W3XwZK4Tih2r8bH4N65+zhtJdxg5CmGLU9PlCLiZm0wMC8qFFa\\\\\\\/hYqkKszE94YzFGsFrQ6o5G0+HxpSwZiO6KgrE6pOCRswKHR+q7qidhiIUHW4tmidmBN4bjSLkGEs30fIB\\\\\\\/MRtvXCb9R7dJR0y3Agvb\\\\\\\/SqinjNKbxojmAS2ZZt8J\\\\\\\/Tgj\\\\\\\/0Shj0zk5ncVWWgCvAzDGQoOvQ3HxdyuGldbH903nXi8+3ChXkj1gpSr7bQq16i8MU0miRspzq0VyX3f5JCIIbZ6haN1Eq1en4O8jXxAscfgnlAbBXRS4raxytveqsCAajabHeKNBJViUCacvqECs+Qfc5HLGVnsQvewGhHv+dahGaVEPxF56krbpSIRJoOlc\\\\\\\/OJiKYBxgtN+yzRCa5JIMPtKjRnJNTsNMITBIjCJyFbuGi1pnmT4dNqBiy1Luj\\\\\\\/XlNz6RFJVl5vXc4He6gZTctLNBbc6s4fQk57hdzuIb6bBOqttLUa61rrvWJCoPVietymw7PkeEz+gICcJMYp542OnfO5esvUiRCyY9POeVyWiiUSx\\\\\\\/aW3S7z6efGtcgM7KK8w6s74oLNl7Kl8lpISK\\\\\\\/YR61\\\\\\\/3sEK+unkJo+6HWqvnBAp65kGhAk029\\\\\\\/meP8uPE8g\\\\\\\/9mviJ\\\\\\\/UUjNIgQwAGrSmCbDGZDOX+tfh9DO8zVz1pVxjzYhQhGUKu2mlld1dps46BER4\\\\\\\/mVE6jxxhtx2iP9UlRBjVuskmsDoHyNEumKhU8HtLZKvgHLzXhLrwjJUBea+GpG1iH\\\\\\\/slIG5nGe2lyU3\\\\\\\/xhPOPP7e9v+Y9iEKTPOur9GK739oJC7THnpmrDq8kEfZPSqQG8f3ok0fErStvpxOywS61tIgeHecAuPYXVCUKWw1Rl5llPyRR0zRzSw0SGYVaRWw8169iRyg\\\\\\\/VOqnuC0YZtHjPjB4zDXa\\\\\\\/yc9ZnLr2QTDfNWWI9E2dsZMgI2\\\\\\\/9VWIzYC0w8A7GVHhSFO7DDKow9w7Udtip9qoezM7qaakrB5ufM+SlUVhI\\\\\\\/CDArHevKSkDyJqGIu2qw01T1220MXM\\\\\\\/8JL2Rds+FPmnv1t\\\\\\\/5JoO2SLTcNa6E+L6GDty4Re7hGATJmEIQj7CVHHxN5GFGskihEtDTVuWndGa0dHo0Zz75xYaHqYSTmPdejCOKjaARu9I7ru2si9YGRMtntkkyePw+wegV6NrLmpfhb7tkayfjIBq4udJiiGba2i1ui\\\\\\\/g0sf\\\\\\\/B5JVmwzQmO0Nss5cqPfDkW3mKB6HvNpnRS19c+eO9+tB0nIwgS3gaF\\\\\\\/pqbcTReX3zluyp2cAf8o+Jqvrqn8rbgNs+dPf\\\\\\\/BNTPkb6FC5c8uRRS69zLlqYqbSrKjyXcfXNd6glnGhbf24RZ5XYny1qGqZ26Vc7Y\\\\\\\/4XbY5hBKG7a6iwuDX96+SVX4Htysgwdh3YAnp5aEKQ98FFAcdzZImiJsDqkOG3RK1yMHKSOEQsrsy4VFI38jFV+p3lfWLD4yYpffWiEqC\\\\\\\/rcDSFTXS4jkzjI9Sghh5xINfK7ukMmyDJeSaumugGl94dx3Hyf3YBVOlyzc4YYSU1Uuto0zPu9bK7YWjqAlVNeKgZUyRJBmnH7AlTVoVGaNvEwBv9spIM7kQ\\\\\\\/IhWfY25k7MzmJU9364oiWHsdcwCrJ+xQETS16\\\\\\\/dKhWmk7XxeXteAPKBMgrYAnTJylz\\\\\\\/DKAhnB2QDGj5OIi6tJeKRFGDsexZuJguquKI\\\\\\\/VJZKBdA\\\\\\\/7+1mYH8+3\\\\\\\/51TbFx68Nf7rLTFbZ5naTzz3ei\\\\\\\/G86vNWIQ5z0QzHQ1LccbdW4pjNSeFug8JXR0WO7M8ySlRLnTHbb9sGj8zTinLPt7+s+9QoUWmvS8JZPHf+oc+qDlxXO\\\\\\\/IvcRqY+SD2ZUTPaIymbn+LQuS3wxmLtmnU\\\\\\\/POvPUGwiyqv8f8sgPg4MOOYROkVdel9N6uWLBN21xotfkqYG8lzXW1Ao4fDCNU4rqitPz576iqXrczdJ+UyhT3sJN\\\\\\\/vbZjEru2N9l3bzAxn2qXKoj02VF0wXhW\\\\\\\/iiJuiv8lkcqYpY2CLMRnzLU0OlgoplQ8RcaIhhcNAbPMPF9xIK4dxpSXqETNF2d4B1jkYD\\\\\\\/LrB8qDGcSDH+gWAChPqGmIcag42yo9IFILhoQuhTDoMWzfFDUuY2bpn0wB1b3qIdXNyi66JkHwMTnrjszJaOkbRMU9NaZwwdK5OcYbdOAGEoPLXuEY9sWeJ7lPTAil1DzTXss6vg85K4Oank3zXHvnNYxiFd9sW6or+Yai0GcveXwn0Dm3tgNMt0gk1Ed3eA7sbqeDT4720Fdcj4Hghywv3EQCWLwIqWPq8T3b5uQtWs0DjB8Y6yPBV+v\\\\\\\/qWieuVo0LOqhDAohGlc08qF5WpqhSAspygvnD5mO5P3ZslYu8My5hY0ywKJbKwsqcFTVD+yxJjuHjHKNSQn0wtKs6FT\\\\\\\/Y3nxibDMSsHHAP\\\\\\\/CTxyNhmHvMau02MBYIBZ\\\\\\\/\\\\\\\/2fFn9ekzHsIvqK0vW1sRTYtQv0CwFwXwh7274b60Z\\\\\\\/OaA7KmivmiGoHUmvQXDGz0oph7fYxZKhNe6EbuJcUCN7Pk46Rvd4HFUP5eaxfN5dp\\\\\\\/kHzGTQtmvMuB4LO5bjfpEQhO1KUR838OJEwGsQc+wNdOTTDE+bJC3Cv0bXO9uUcH89mQjI1S7waAtaIq\\\\\\\/opEYOHIrNQ\\\\\\\/f5fNwX6lFVyn1jNQMWC6dMNugmCdISb4I1HdE5vVX8dlggIKMYDZGqj58Sq60lTrWlm6isRl0nFiHArR8l3NHkCTCxXQTsLZEAPQ5IB8Ap2qudvxDX58Lr2Qo8KFBO0SStUWhdomUwG5Gyw14ZTXMgQXsFAKknsUMV+6D4hKZ56fC1GgtGLM6IimEUbkm19RrYwppOn\\\\\\\/R8\\\\\\\/tbmCZihMG7JM9ssuJJVAdEoUc7XGboNRaQOfLkaHLugCIaGFLlzIh98WDFh0WA9x1bGLwEC2z0BGJMPEIbzg8zUP2d+sCJyHX65bEd1Sbm+xJL1CUdFrKOo7orQ3jcJKiSRqXKdAge4\\\\\\\/LI7RNU4d98eZy4GTp4iqwjr30akfvktxd6W5wIFjSkb7xX37Jrgs3ELTs7n5ivyD5EtPSWQsrZxF3uBtom\\\\\\\/0lgBpRJyIK2PQNTjb\\\\\\\/VA1jbwt6KsgOKfDf0s1qcUWhXo6ao5aR5R1xXj8tRjRh2FpRKWsXLYVMcV+0Iqd7UG9+6OOV\\\\\\\/npxTZ6qUunVg8qV7iNs6Auf96VRncYb4edJQ3WGDn738jqD\\\\\\\/LFSosxR4HPfXhiIZ6T1AlwkEdtU25bYF5FMfYcoensDJMdXApUUhYMkJUZI6lmrU\\\\\\\/d10\\\\\\\/1lK08JFN+2++9mDZB5nxOYyUHshtaWYqnpNUR+J7y8a1lcq6I23fi1l9AgF8ueZPiMzPvqUmcCciYQ+ClhHV6HM0ppf8tgqfoHhdQdMe5Lh7F9XVJJW9+bLgwyFT83PRMpFOA8T7YcFOpOwmvuG6wWZLhnB7P7YobVhzpwIdgjWcP1ylwsl0O0uZ7gofkDyOUzf\\\\\\\/dVf9Bmq0JOdSl8Kx1DhqlDWLKxajP\\\\\\\/lcts2FDaHrv+sBl5Ru7fdpCksndA0aTL77lO9m1vl0d9qXHiklBqbC3dVDy0uP\\\\\\\/cw1F0PtiyEhLxBUpCeq3Fhzf\\\\\\\/sAQFruZ0MxlQWTbtmRzwgt8sS7avc4\\\\\\\/zl0fWk787a4T9pSzC9X5RbWW07PE9XsXS\\\\\\\/ADdi7e86QZHKIU\\\\\\\/q8n7y4LECR92D5eM05fBJz\\\\\\\/bJJsP55BOeC7TV4mw\\\\\\\/LA6TGfIJzedEwc+EKltxAaS6p\\\\\\\/EYZCri7jY6lZUwpj7MChSx7zWNOkOdj3dZuB3nANkzl6HnadTpWdgLX2VXJesMKqxF91Z1HKtkT\\\\\\\/wh4dyomCzgfgkPHoBIXwGDUCchGKS68yMl1KkSeDz0IP0bWeECO4LsFbv4VlPw6SN3Ns5zXNWPRyJnXcc8OqXTiDYEDmfgto2zJ62Vs7MGY\\\\\\\/b5U06nVBjIV8MOfr37vlL5sKNb7ZdbIBwuWMF6MVeXHvUrkaBVXefR37+dZy9\\\\\\\/9nYhGUk8REd\\\\\\\/cSViAcMfWaR6jc7NWJ5+ljYnBwSG6NMx1kL0wb+Sxxn5qvQoMcvLCjXQwJuFUO3cV1vyVzQW3JgACImFKP9obO\\\\\\\/fRPNfZ3wsK6vZCAbT0Wah7bfxtc5I4pRpo2GijDiwPiR5XtIaVk7UUULwGFe2eA0ueseZp2ymkse3wYVD70qgxHXfqryr\\\\\\\/PGU\\\\\\\/I7bKXpOEa3Sa+EVjdpYqHoi15Xb9Nlen3ZrI\\\\\\\/j01Fl51vpu6Z5S9PnT0E7m8Yklj7GNR5b6t0i+K8LXUIjQMVZijUkvxv2eB4yGvNR+sPM76R0WG+ZU0HC\\\\\\\/rXF45dBQ+siJJfoqhV\\\\\\\/CDe\\\\\\\/gZ5jFVVyAfnhmlYeBBW5sP1h1lODPXEr\\\\\\\/6oHEcOwUtCkuRT+Gbg6AirvhBWzznMkbs0KjL6z2qiVGVeOczOa\\\\\\\/EDIakqBsgYTRbgthar4jrknIA++DCsuJjJfOHRS8AlpTvpmcCHomA2\\\\\\\/nWQskkg+yWJ0hOuK1Htel3MfGeF0Ivc3fk0Ll\\\\\\\/urDdEcE8dZhsUOLlHzFjArhC34KonIJeI8odn\\\\\\\/I6YvOmDht0uwwcWYl2iZ8GThl3cp6Z3NYj5bOFTO6wo8wERJDqCJEAAdAt17I\\\\\\\/Jfs4GlwUUZLrAHBQdzjHiVVZMPT6fzJbX9nMzfqGvlr+Hfd05t1nkthnojTIBOarvTRpTvLQNfYLvUzW7JCxeTOhk9mGsWJnzzURRlnMg+ZCMhwUZQ95k12fVm55ERgv89sns0BQbkzOHKCr1Uq88fQ6DgB6bXXdvzgMY88iRRRxOwBLZ4895JHtN9Fx1l9trnkZ7DWGV9rz8KMPBtILEQw5NbhoNAb+vt2\\\\\\\/zpbmBpWdpA1XFXYkFeNWZjviznO+HvtBDF2\\\\\\\/\\\\\\\/Yd8i1bNUqXPaQnkpQCq4a3ch0mYK5qS8weG8\\\\\\\/2rzNB9+H9oK13TFO0PF6i9toiz1myVpvs6aAlMNWXDMzPUGgOmcBEiWJvq40aoLD1fsba4VzHN7Lvf3R0kZltkiUAyW0\\\\\\\/fzOuroFw6jnb81GUSUVA1BHnxmjJ\\\\\\\/VwwcSCPW3RuFD\\\\\\\/ewFyGgfFuBqXUvTNgjpjSfGvaPf6ieAHQC684OuO8ze+uZWZxzJzTDxp3H5p\\\\\\\/B4T1GJd9QSndi8tum+\\\\\\\/NuWhh0NFf4NEtLxDShgH26Yh+lYfRjEnsYTg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sceupKAgYRbcnrM94sU4\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002261004d3471311d31f406ffb4e963bc53edcdb889867d932bf9ab48ba2cbce278\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IbMobSLDZExZr\\\\\\\/7TJk+1K0crJ9eLMn3bjIAJcz50iFPViRDlpZzlF3YJk6H+qpwvzi7EBx3pwXET7whwXJ8fmB7Zc9H9WnGCMtMruYbjOcDsXjgcMVIk2pkepeuEC6CK9rdXkQZwxcKKAvduZUqaT5RbnF9W6lTFAAzNpIFz6QZhEbGGhAMsU5D+LB3HCfxHTKoNFUE4HYHYKr7+zFpMO8tH9MP4hb+fo0LF\\\\\\\/+YOLdzA\\\\\\\/xCrTr7BCReN5s9YBk1wO+fMJrV25fFhhyITGHPeLMB2cl49loJnTHbVvf+laIm\\\\\\\/6DkK1+Yr3jA7arNaB0NpWFwKy\\\\\\\/5ZmyNSlMaD6yL5BxI13eNu5AMJvfFz0q6FQoJajs9meL32a5wscUVLWtqmz5IJhKrf72lSkET\\\\\\\/Tbx7XILf5prDe+uRKEdaOQ3Kz09Qq+YIhqGFAYWd7v1AA7pfW\\\\\\\/VCyFjY89GsWAuuzqh+gZOfU1O3J3U+3hgISJJlYL7msr5YpeZzC6XITiiQuPuNk0zqChEidQd3bpPShmW+5PChcvhLBA\\\\\\\/Bm9xpwcFrvNfotEjleCsXnICCNj3NC7hrdZXzrY4Z9lLcmJNPQvEwfPgM0zFnCskWxJEfTZCsL75dZKtA7oTqjo4O7g7OKRYR0Q\\\\\\\/BgyqUGO0bRVbbLEX6ykThYbumC7vQroE6SMYOfkZC7Do2gSrWboArRU\\\\\\\/UP\\\\\\\/fL61qng7G0Yu2e0lhQUzDjw9cjVjaNvAjczEanuMM913FgsyeTGl6j1b+1nHEXlWTx3CkDSoiM25Ccc2yczoV+sONgyFychvP1oe4qhJhdU02hbIM7WwciKYqCasOfxLUoMi+GdFE9GaxnKBn3hsEDUgE+kZm7tDTJBKtpxo53o8XQY\\\\\\\/d50v20o04YaxFlPeUuDvWEnfAUhVCtjFDNvWRpppRyLNAm9Va8veQ91bj\\\\\\\/S7O400zob7\\\\\\\/1irtLfb9iDW3myF5GFHU7nAfQbuoWJIlW6OwTPjfAOPSoED21\\\\\\\/5RRDnwKDrmoPW0EKO6mXyGAF75lXWPuS1yqhql3qxQvugM0JGdvqymBHHX3uecUQ9mFUrgaTAe23MsN9lOUvIS7E6\\\\\\\/sRrNYqPB2evTKrkXBnIM6251O6cce44KdoH3LeevHip+jlDpjUEzFek+sPsmJZC1GGI0GM8XajRKS5rk7aoEVZ4hNSPRLf2pbtEwo\\\\\\\/simFBwVxMbMx3wtWAQ0Us4fblNQk6enl9AuunvaA0P+1xqCK1Marvir5wQDfVY2h2oFtGtaKBTSiG7lYOwVFglT1vhDxrZxGSaenJY1xFh71e37Ka2aFE7+b7MZgzrvoCIMGsjPIHtTFor0+V6g2BEzQOyI+ZCgSRhuYTBigmYrlI0LmfNmgUuR0YOqV03CmrkaG\\\\\\\/bjOFKTYWfgx5sSmSukbDrs6ghMO24uXEtQgb\\\\\\\/bxivNYP3Rl23C0kvUeDWeKgB\\\\\\\/V0oAfWvZ+fzCdAhxcmFKLrmXMX0684E9O5aIag6w0Po+xfDzx36uzeeD6ddxHsUkBcoJPZFloXsJ4JFIdZhKaUKmQXnrT+dM0BRKVw\\\\\\\/KJU42fH+aV0LyTDyRSsVjY3m8mSvyBExxA8jiHFK4tP9aeMKVifEKdCq8X1pBpzCQe6AAV9sdQBV25cr4bv\\\\\\\/0YtpIMFXMHKLr34NsqgYK8CVcg8Qz8gDthhvCVBOR9O\\\\\\\/nqmhokYnj7+WiY4gNljMLcqmPTq7VZfrHArG7U1E1pIN4ooisBfpdPYeHzzDetqND5Wcawolob1l2Iuzsn21kLWQ+OHdMFOPYKuDocIeXWZzcX5JFqbOE1p6cz3s+bi\\\\\\\/nLPhzig61vmQ8x+eX\\\\\\\/8+4fL7kX4Pp6NF\\\\\\\/9fS0FZ8TfSrxw7IoM76ZvMdaY2vXKZbg5YsTXuMal3nN\\\\\\\/eXMFZX47Bs+rtDO7WjQacdcX7LRqmFqHjWPcJFUiczffu4orqNG28irWGaF5VE7Ev6cjYIYVO7+NDj+unh1mWEddp2JFS2p8R30zGHv6\\\\\\\/LBbmiN8jRI7H8aoQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wbHmo5FT3zWaAsilRkinCg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b39504f49ef2cc4cf84c85b6c6658ea39ae8b836cf65889bf6f264e8256879e8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489127 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1173" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wI2BvilWQ8DzXiyDPPCl1R6u9OPqraV4doW3GvEssAw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1173" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-szExRJiUHOZD9zCe-fwvlaYK-NaH4PnVszpEF-si2nY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1161" title="The Molecular and Genetic Characterization of Second Chromosome Balancers in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1183" title="Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489127" id="node489127" data-pisa="ggg;8/4/1173" data-pisa-master="ggg;g3.118.200091" data-apath="/ggg/8/4/1173.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dominik Müller</span>, <span class="highwire-citation-author" data-delta="1">Pascal Schopp</span> and <span class="highwire-citation-author" data-delta="2">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1173-1181;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200091 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489127" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominik Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominik%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pascal Schopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>KWS SAAT SE, Grimsehlstraβe 31, 37574 Einbeck</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BSchopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schopp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BSchopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/4/1173" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489127" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1173.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489127" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1173.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489127" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1173.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489127" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/4/1173.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489127" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic selection (GS) offers the possibility to estimate the effects of genome-wide molecular markers, which can be used to calculate genomic estimated breeding values (GEBVs) for individuals without phenotypes. GEBVs can serve as a selection criterion in recurrent GS, maximizing single-cycle but not necessarily long-term genetic gain. As simple genome-wide sums, GEBVs do not take into account other genomic information, such as the map positions of loci and linkage phases of alleles. Therefore, we herein propose a novel selection criterion called expected maximum haploid breeding value (EMBV). EMBV predicts the expected performance of the best among a limited number of gametes that a candidate contributes to the next generation, if selected. We used simulations to examine the performance of EMBV in comparison with GEBV as well as the recently proposed criterion optimal haploid value (OHV) and weighted GS. We considered different population sizes, numbers of selected candidates, chromosome numbers and levels of dominant gene action. Criterion EMBV outperformed GEBV after about 5 selection cycles, achieved higher long-term genetic gain and maintained higher diversity in the population. The other selection criteria showed the potential to surpass both GEBV and EMBV in advanced cycles of the breeding program, but yielded substantially lower genetic gain in early to intermediate cycles, which makes them unattractive for practical breeding. Moreover, they were largely inferior in scenarios with dominant gene action. Overall, EMBV shows high potential to be a promising alternative selection criterion to GEBV for recurrent genomic selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-gain" class="hw-term hw-article-keyword hw-article-keyword-genetic-gain" rel="nofollow">genetic gain</a></li><li class="kwd"><a href="/keyword/doubled-haploid" class="hw-term hw-article-keyword hw-article-keyword-doubled-haploid" rel="nofollow">doubled haploid</a></li><li class="kwd"><a href="/keyword/optimal-haploid-value" class="hw-term hw-article-keyword hw-article-keyword-optimal-haploid-value" rel="nofollow">optimal haploid value</a></li><li class="kwd"><a href="/keyword/expected-maximum-haploid-breeding-value" class="hw-term hw-article-keyword hw-article-keyword-expected-maximum-haploid-breeding-value" rel="nofollow">expected maximum haploid breeding value</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-4">The identification, selection and propagation of superior individuals builds the foundation of all breeding efforts. The breeding potential of a candidate is classically determined by its breeding value (BV), the sum of all additive effects at quantitative trait loci (QTL) affecting a complex trait (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lynch and Walsh 1998</a>). While BVs have been estimated in progeny tests, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Meuwissen <em>et al.</em> (2001)</a> proposed genomic selection (GS) to predict BVs prior to phenotypic evaluation. The principle is to use genome-wide marker data and phenotypes of training individuals to calculate locus-specific allele substitution effects. Genomic estimated breeding values (GEBVs) are then calculated as predictors of BVs. Selecting individuals ranked according to their BVs maximizes the population mean of the next cycle when they are recombined, but a repeated application of this selection strategy does not necessarily maximize long-term genetic gain over several generations (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wray and Goddard 1994</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2015</a>). GEBVs, as predictors of BVs, are subject to the same constraints. This suboptimal behavior can be explained by the fact that GEBVs are simple genome-wide sums of estimates of allele substitution effects, which can conceal the contribution of favorable alleles with small effects. The later are less relevant for short-term gain and can be easily lost, especially if their frequency is low, but can play an important role for long-term gain by maintaining useful genetic variance (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jannink 2010</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2015</a>).</p><p id="p-5">To prevent the loss of rare favorable alleles, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Goddard (2009)</a> proposed a modified GEBV that weights estimated allele substitution effects using the frequencies of favorable alleles, such that rare alleles receive a higher weight. This criterion does not take into account the magnitude of the effects based on the premise that, for optimal long-term genetic gain, all favorable alleles should ultimately be fixed. Later, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Jannink (2010)</a> suggested a modification called weighted GS, herein referred to as weighted GEBV (wGEBV). This considers also the magnitude of the effects, because especially for QTL with small effects determining which allele is the favorable one is problematic. wGEBV proved to be superior to GEBVs in terms of long-term genetic gain (19 cycles) in spring barley (<em>Hordeum vulgare</em> L., <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Jannink 2010</a>).</p><p id="p-6">Differential weighting of substitution effects does not take into account other important information often available at no extra cost, such as genetic map positions of loci and linkage phases of alleles at different loci. New selection criteria utilizing this information could be defined based on the prospects that candidates produce superior gametes with favorable combinations of haplotypes. While the average performance of progeny of such candidates might be inferior to that of individuals selected on GEBVs alone, the top-performing individuals in the progeny are expected to be superior, which boosts the genetic gain achievable in future generations.</p><p id="p-7">In this vein, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> (2015)</a> recently proposed a criterion called optimal haploid value (OHV), which aims at predicting the theoretically optimal combination of haplotypes in a gamete produced from a heterozygous candidate. The criterion was tested in simulations of a bread wheat breeding program and showed increased genetic gain compared to selection on GEBVs. In their study, genetic progress was measured as the performance of the best doubled haploid (DH) line (generated by chromosome doubling of a gamete) produced from selected individuals. By definition, OHV does not take into account the finite size of the breeding population; hence, it merely considers the possibility of a superior gamete, disregarding its probability (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2017</a>). Moreover, OHV requires the genome to be partitioned into haplotypes, and it is yet an unsolved problem how this should be optimally accomplished.</p><p id="p-8">In view of these limitations, we herein propose a novel selection criterion called expected maximum haploid breeding value (EMBV). It characterizes the breeding potential of a candidate in terms of the performance of the top gametes it is able to produce. If a candidate is selected for recombination, it will contribute a certain number of gametes to the next generation. This number can be directly ascertained under controlled matings or easily estimated under random mating conditions. The EMBV is then defined as the expected GEBV of the best among all DH lines derived from these gametes. Hence, EMBV takes the finite population size into account and it is not necessary to partition the genome into haplotypes.</p><p id="p-9">The objectives of our study were (i) to evaluate <em>in silico</em> the potential of EMBV as an alternative selection criterion in a generic recurrent selection (RS) program and (ii) compare it to the criteria OHV, wGEBV and GEBV with respect to genetic gain and genetic diversity across 50 selection cycles. The performance of OHV was assessed under optimal conditions with respect to the partitioning of the genome into haplotypes. In order to evaluate the effect of gene action on the comparison of the selection criteria, we compared purely additive gene action with completely dominant gene action at all loci. Furthermore, we considered the effect of population size, the number of selected individuals and the number of chromosomes on the relative performance of the different selection criteria.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Genetic model</h3><p id="p-10">We considered a quantitative trait with additive and dominant gene action at all <em>L</em> loci. Each locus was bi-allelic with alleles <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-1.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-2.gif"/></noscript></span></span> and possible unordered genotypes <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-3.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-5.gif"/></noscript></span></span> We assumed that the locations of loci and of alleles on homologous chromosomes are known (phased genotypic data). For each locus of a diploid, heterozygous individual <em>i</em>, let <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-6.gif"/></noscript></span></span> be an indicator variable indicating absence or presence of the <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-7.gif"/></noscript></span></span> allele at the <em>l</em>-th locus on the <em>j</em>-th haploid genome (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-8.gif"/></noscript></span></span> referring to the maternal and paternal genome). Then <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-9.gif"/></noscript></span></span> is a genotypic score counting the number of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-10.gif"/></noscript></span></span> alleles. Following the genetic model of <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lynch and Walsh (1998)</a>, we assumed that the genotypic value of an individual <em>i</em> with genotype <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-11.gif"/></noscript></span></span> at locus <em>l</em> is given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-13.gif"/></noscript></span></span> is the homozygous effect (half the difference between the two homozygous genotypes) and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-14.gif"/></noscript></span></span> the dominance coefficient (deviation of the heterozygote from the mean of the homozygotes in units of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-15.gif"/></noscript></span></span>). Effects <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-16.gif"/></noscript></span></span> were independently drawn from a gamma distribution <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-17.gif"/></noscript></span></span> following <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Meuwissen <em>et al.</em> (2001)</a>, such that <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-18.gif"/></noscript></span></span> was implicitly defined as the favorable allele. We assumed two extreme scenarios where gene action at all QTL was either purely additive, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-19.gif"/></noscript></span></span> or completely dominant/recessive, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-20.gif"/></noscript></span></span> was either 1 or <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-21.gif"/></noscript></span></span> with equal probability. Dominance coefficients <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-22.gif"/></noscript></span></span> were assumed to be stochastically independent of additive effects, following <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Zeng <em>et al.</em> (2013)</a>. The genome-wide genotypic value of an individual <em>i</em> was computed as <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-23.gif"/></noscript></span></span> The average effect of an allelic substitution <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-24.gif"/></noscript></span></span> at a single locus <em>l</em> was computed as <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-25.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-26.gif"/></noscript></span></span> is the frequencies of allele <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-27.gif"/></noscript></span></span> at the respective locus. The BV of individual <em>i</em> was computed as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-28.gif"/></noscript></span></span> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Vitezica <em>et al.</em> 2013</a>). Following <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> (2015)</a>, we assumed that QTL genotypes and effects are known without error, <em>i.e.</em>, marker loci are identical to QTL and their associated allele substitution effects are identical to the simulated substitution effects for QTL. This was done in order to assess the performance of the investigated selection criteria under optimal conditions. Consequences in the practical case where the trait genetic architecture is unknown and (marker) allele substitution effects can be only estimated with some degree of precision are addressed in the discussion.</p></div><div id="sec-3" class="subsection"><h3>Selection criteria</h3><div id="sec-4" class="subsection"><h4>GEBV:</h4><p id="p-11">The GEBV of individual <em>i</em> is canonically computed as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>which is the genome-wide sum of all substitution effects for the respective alleles.</p></div><div id="sec-5" class="subsection"><h4>EMBV:</h4><p id="p-12">The EMBV measures the breeding potential of a candidate in terms of the expected GEBV of the best out of <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-30.gif"/></noscript></span></span> DH lines produced by it (visualized in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-31.gif"/></noscript></span></span> denotes the number of gametes the candidate is expected to contribute to the next generation, if it is selected. If <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-32.gif"/></noscript></span></span> denotes the GEBV of a random DH line produced by candidate <em>i</em>, the EMBV is formally defined as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-35.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-36.gif"/></noscript></span></span> is the largest order statistic (maximum) of a random sample of size <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-37.gif"/></noscript></span></span> An alternative formulation of EMBV using a normal approximation for the distribution of GEBVs of DH lines produced by <em>i</em> is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS2.pdf">File S2</a> and discussed below.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of the computation of EMBV for a heterozygous selection candidate. A (conceptually) infinite population of gametes is generated in silico from the candidate by simulating meiosis events. The corresponding doubled haploid (DH) lines are evaluated for their GEBVs, yielding a distribution of GEBVs (blue curve). The candidate&#x2019;s GEBV corresponds to the mean GEBV of the DH lines. The EMBV is defined as the expected value of the maximum GEBV of a random sample of DH lines of size  where  is the expected number of gametes the candidate will contribute to the next generation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-982444675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Illustration of the computation of EMBV for a heterozygous selection candidate. A (conceptually) infinite population of gametes is generated &lt;em&gt;in silico&lt;/em&gt; from the candidate by simulating meiosis events. The corresponding doubled haploid (DH) lines are evaluated for their GEBVs, yielding a distribution of GEBVs (blue curve). The candidate&#x2019;s GEBV corresponds to the mean GEBV of the DH lines. The EMBV is defined as the expected value of the maximum GEBV of a random sample of DH lines of size &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the expected number of gametes the candidate will contribute to the next generation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1173/F1.medium.gif" width="440" height="150"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/4/1173/F1.medium.gif" width="440" height="150"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489302" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Illustration of the computation of EMBV for a heterozygous selection candidate. A (conceptually) infinite population of gametes is generated <em>in silico</em> from the candidate by simulating meiosis events. The corresponding doubled haploid (DH) lines are evaluated for their GEBVs, yielding a distribution of GEBVs (blue curve). The candidate’s GEBV corresponds to the mean GEBV of the DH lines. The EMBV is defined as the expected value of the maximum GEBV of a random sample of DH lines of size <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F1/embed/mml-math-33.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F1/embed/mml-math-34.gif"/></noscript></span></span> is the expected number of gametes the candidate will contribute to the next generation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>OHV:</h4><p id="p-14">For the computation of OHV, the entire set of loci <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-38.gif"/></noscript></span></span> ordered along the genome, is partitioned into <em>N</em> disjoint non-empty subsets <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-39.gif"/></noscript></span></span> (corresponding to haplotypes), such that <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-41.gif"/></noscript></span></span> for all <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-42.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-43.gif"/></noscript></span></span> and all <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-44.gif"/></noscript></span></span> According to <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> (2015)</a>, the OHV of a selection candidate <em>i</em> is computed as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><em>i.e.</em>, for each haplotype, the maximum breeding value over all haploid genomes is determined and twice the sum of these values is taken as OHV.</p></div><div id="sec-7" class="subsection"><h4>wGEBV:</h4><p id="p-15">In selection criterion wGEBV marker effects are weighted by a coefficient that depends on the frequency <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-46.gif"/></noscript></span></span> of the favorable allele <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-47.gif"/></noscript></span></span> The associated locus weights were computed according to <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Goddard (2009)</a> as <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-48.gif"/></noscript></span></span> and wGEBVs were calculated with the modification proposed by <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Jannink (2010)</a> as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>For all criteria, allele frequency <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-50.gif"/></noscript></span></span> was freshly computed as the sample frequencies of allele <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-51.gif"/></noscript></span></span> in each cycle of the breeding program; accordingly, allele substitution effects <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-52.gif"/></noscript></span></span> (and locus weights <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-53.gif"/></noscript></span></span> for wGEBV) varied between cycles. It is important to note that the EMBV and OHV of a completely homozygous individual are identical to its GEBV; hence, these selection criteria only differ for heterozygous genotypes. The computation of GEBV and wGEBV only requires genome-wide co-dominant bi-allelic markers with effect estimates, whereas both EMBV and OHV additionally require a genetic map and phased marker genotypes of the candidates. Criterion EMBV further requires software for simulating meiosis events (<em>e.g.</em>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Müller and Broman 2017</a>).</p></div></div><div id="sec-8" class="subsection"><h3>Simulation of the base population and genome structure</h3><p id="p-16">We considered a diploid species with a constant genome length of <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-54.gif"/></noscript></span></span> cM. The genome was subdivided into <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-55.gif"/></noscript></span></span> segregating chromosomes with equal length (<em>i.e.</em>, 400, 100 and 50 cM, respectively). Bi-allelic QTL were uniformly distributed along the genome with a density of 2 QTL per cM, corresponding to a total of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-56.gif"/></noscript></span></span> QTL. The simulation of the base population was conducted as in <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Müller <em>et al.</em> (2017)</a>. Briefly, a historical population of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-57.gif"/></noscript></span></span> diploid individuals was subject to random mating for <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-58.gif"/></noscript></span></span> generations. A population bottleneck was simulated by arbitrarily selecting 40 individuals that were further randomly mated for 15 generations to build up extensive linkage disequilibrium, as often observed in elite germplasm in plant breeding (<em>e.g.</em>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Van Inghelandt <em>et al.</em> 2011</a>). The population was then expanded to 5,000 individuals and randomly mated for three more generations to remove close family relationships and establish the base population. Finally, all monomorphic loci were removed from the genotypic data. The base population was simulated only once for each value of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-59.gif"/></noscript></span></span> The distribution of allele frequencies (data not shown) and linkage disequilibrium (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS3.pdf">File S3</a>) in terms of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-60.gif"/></noscript></span></span> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hill and Robertson 1968</a>) was similar for different <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-61.gif"/></noscript></span></span></p></div><div id="sec-9" class="subsection"><h3>Breeding program</h3><p id="p-17">From the base population, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-62.gif"/></noscript></span></span> individuals were randomly sampled without replacement and constituted the candidates in cycle <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-63.gif"/></noscript></span></span> We considered four distinct breeding programs, starting from the same set of individuals in <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-64.gif"/></noscript></span></span> that only differed in the selection criterion, namely the use of GEBV, wGEBV, OHV or EMBV to select <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-65.gif"/></noscript></span></span> candidates for establishing the next generation. For criterion EMBV, the number of gametes <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-66.gif"/></noscript></span></span> contributed, on average, by one selected individual to the next generation of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-67.gif"/></noscript></span></span> new candidates was estimated as <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-68.gif"/></noscript></span></span> rounded to the nearest integer. In a given cycle <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-69.gif"/></noscript></span></span> all candidates were evaluated and ranked for the applied selection criterion and the best <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-70.gif"/></noscript></span></span> candidates were selected. For creating cycle <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-71.gif"/></noscript></span></span> the selected individuals were randomly mated, <em>i.e.</em>, both parents of each future individual were randomly drawn with replacement from the selected candidates, allowing for self-fertilization (father = mother). One gamete per parent was produced and both gametes united to form the new progeny. In cycle <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-72.gif"/></noscript></span></span> the population mean (average of all genotypic values) and the standard deviation of BVs (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-73.gif"/></noscript></span></span>) were calculated. In each later cycle <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-74.gif"/></noscript></span></span> all individuals were genotyped and the difference between the population mean in <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-76.gif"/></noscript></span></span> was computed. This difference was then scaled by <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-77.gif"/></noscript></span></span> and the result was recorded as the genetic gain (<em>R</em>), analogous to <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Jannink (2010)</a>. Hence, <em>R</em> is measured as the progress of the population mean in units of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-78.gif"/></noscript></span></span> relative to <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-79.gif"/></noscript></span></span> Note that <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-80.gif"/></noscript></span></span> can vary between the different scenarios and among samples of founder individuals from the base population within scenarios. Scaling by <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-81.gif"/></noscript></span></span> aims to correct for this difference in the initially available additive variance, but does not affect comparisons between the four selection criteria. In each selection cycle <em>t</em>, genetic diversity was calculated as the variance of the BVs of all candidates (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-82.gif"/></noscript></span></span>), divided by <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-83.gif"/></noscript></span></span> The breeding program was continued for a total of 50 selection cycles. The factors investigated in our simulations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were (i) the number of candidates in each cycle, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-84.gif"/></noscript></span></span> (ii) the number of selected individuals as parents for the next generation, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-85.gif"/></noscript></span></span> (iii) the number of chromosomes, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-86.gif"/></noscript></span></span> and (iv) the level of dominance, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-87.gif"/></noscript></span></span> (no dominance) or <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-88.gif"/></noscript></span></span> (complete dominance). The breeding program was replicated at least 600 times for each scenario, starting with sampling the initial candidates in <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-89.gif"/></noscript></span></span> from the base population and the simulation of homozygous effects and sampling of the signs of dominance coefficients. The homozygous effects were always scaled to achieve unit additive genetic variance in the base population. Summary statistics are generally reported as arithmetic means across all replicates.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489173/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489173/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Factors investigated, with symbols and list of levels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Computation of OHV and estimation of EMBV</h3><p id="p-19">The estimation of OHVs requires the specification of haplotypes. The most straightforward way, which we pursued, is to agree on a number of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-98.gif"/></noscript></span></span> equidistant breakpoints that partition each chromosome into <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-99.gif"/></noscript></span></span> haplotypes of equal length (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> 2015</a>). We explored different values for <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-100.gif"/></noscript></span></span> starting from 1 (<em>i.e.</em>, entire chromosomes) and following the geometric sequence <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-101.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-102.gif"/></noscript></span></span> as long as the haplotypes had a length <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-103.gif"/></noscript></span></span> cM. An overview of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-104.gif"/></noscript></span></span> and segment lengths for different <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-105.gif"/></noscript></span></span> is shown in Table S1-1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS1.pdf">File S1</a> and results for <em>R</em> obtained with criterion OHV are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS1.pdf">File S1</a>. In the following, we only show those results for criterion OHV where <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-106.gif"/></noscript></span></span> was found to yield maximum <em>R</em> after 50 cycles of selection.</p><p id="p-20">While GEBVs, OHVs and wGEBVs can be directly computed from genotypic data and allele substitution effects, the estimation of EMBVs is computationally demanding, because an overall large number of DHs has to be generated per individual. We estimated EMBVs by repeatedly producing <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-107.gif"/></noscript></span></span> gametes, determining the maximum GEBV among them as described above, and taking the arithmetic mean of the maxima over all replicates. The number of replicates was dynamically adapted such that the empirical standard error was smaller than 0.01 (but at least 10 replicates were taken). This strategy was chosen to balance estimation accuracy and computation time, but in practical applications, computation time is not a bottleneck. We developed a C++ routine for the fast estimation of EMBVs, which is publicly available via a wrapper R package <em>embvr</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Müller 2017</a>). A possible alternative approach for rapid analytical computation of EMBVs is described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS2.pdf">File S2</a>.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-21">Datasets and source code used in our simulations are publicly available from <a href="https://doi.org/10.5281/zenodo.1161723withtheDOI10.5281/zenodo.1161723">https://doi.org/10.5281/zenodo.1161723</a>. File supplental_figures contains supplementary figures. File supplement_1 contains results on the optimal number of haplotypes for selection criterion OHV. File supplement_2 presents an approximation of EMBV using the normal distribution.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><p id="p-22">The genetic gain <em>R</em> generally approached a plateau (selection limit) for all selection criteria as the breeding program proceeded (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>). During selection, an increasing number of causal polymorphisms became fixed, such that in late stages of the breeding program, individuals were nearly homozygous and the genetic variance was depleted (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2c</a>). An exception was selection criterion wGEBV, where still considerable genetic progress was achieved after 50 cycles of selection. The rate of genetic progress and the selection limit depended on the selection pressure via the number of selected individuals <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-108.gif"/></noscript></span></span> If only a single candidate was selected (<span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-109.gif"/></noscript></span></span>), which corresponds to recurrent selfing, genetic progress was initially very fast, but <em>R</em> quickly reached a low selection limit after about 10 cycles. Conversely, under mild selection pressure with <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-110.gif"/></noscript></span></span> genetic progress was slow at the beginning, but endured over the entire breeding program and <em>R</em> generally did not fully reach the selection limit, even after 50 cycles.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Genetic gain (R), (B) relative genetic gain and (C) additive variance () for selection criteria genomic-estimated breeding value (GEBV), expected maximum haploid breeding value (EMBV), optimal haploid value (OHV) and weighted GEBV (wGEBV) under recurrent selection. Results refer to  and   number of chromosomes;  number of selection candidates;  number of selected individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-982444675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;(A) Genetic gain (&lt;em&gt;R&lt;/em&gt;), (B) relative genetic gain and (C) additive variance (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-111&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) for selection criteria genomic-estimated breeding value (GEBV), expected maximum haploid breeding value (EMBV), optimal haploid value (OHV) and weighted GEBV (wGEBV) under recurrent selection. Results refer to &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-112&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-113&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-114&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; number of chromosomes; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-115&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; number of selection candidates; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-116&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; number of selected individuals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1173/F2.medium.gif" width="305" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/4/1173/F2.medium.gif" width="305" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489308" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">(A) Genetic gain (<em>R</em>), (B) relative genetic gain and (C) additive variance (<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-111.gif"/></noscript></span></span>) for selection criteria genomic-estimated breeding value (GEBV), expected maximum haploid breeding value (EMBV), optimal haploid value (OHV) and weighted GEBV (wGEBV) under recurrent selection. Results refer to <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-113.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-114.gif"/></noscript></span></span> number of chromosomes; <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-115.gif"/></noscript></span></span> number of selection candidates; <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F2/embed/mml-math-116.gif"/></noscript></span></span> number of selected individuals.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-13" class="subsection"><h3>Genetic gain</h3><div id="sec-14" class="subsection"><h4>Additive gene action:</h4><p id="p-24">Selection criterion EMBV was, in advanced selection cycles, clearly superior to GEBV in terms of genetic gain (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2a</a>), but minimally weaker in the first cycles (until about cycle 5). After this point, <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-117.gif"/></noscript></span></span> surpassed and strictly increased relative to <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-118.gif"/></noscript></span></span> during selection. After 50 cycles, <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-119.gif"/></noscript></span></span> reached a genetic gain of <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-120.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-121.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-122.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-123.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-124.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-125.gif"/></noscript></span></span>) larger than <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-126.gif"/></noscript></span></span> With selection criterion OHV, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-127.gif"/></noscript></span></span> increased at a lower rate than <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-128.gif"/></noscript></span></span> in early cycles. However, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-129.gif"/></noscript></span></span> generally caught up to <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-130.gif"/></noscript></span></span> and eventually surpassed it. The larger <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-131.gif"/></noscript></span></span> the more cycles it took for <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-132.gif"/></noscript></span></span> to surpass <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-133.gif"/></noscript></span></span> (9 cycles for <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-134.gif"/></noscript></span></span> compared to 38 cycles for <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-135.gif"/></noscript></span></span>). After 50 cycles, <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-136.gif"/></noscript></span></span> was <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-137.gif"/></noscript></span></span> higher than <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-138.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-139.gif"/></noscript></span></span> but <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-140.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-141.gif"/></noscript></span></span> higher for <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-142.gif"/></noscript></span></span> and 10, respectively, exceeding the performance of EMBV. Criterion wGEBV showed a unique behavior. In general, <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-143.gif"/></noscript></span></span> increased slower than <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-144.gif"/></noscript></span></span> in the first few cycles, similar to OHV, and plateaued for <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-145.gif"/></noscript></span></span> and 3 at a level <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-146.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-147.gif"/></noscript></span></span> respectively, below <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-148.gif"/></noscript></span></span> However, for <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-149.gif"/></noscript></span></span> although <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-150.gif"/></noscript></span></span> also initially slowly increased, it surpassed <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-151.gif"/></noscript></span></span> after 25 cycles and eventually reached a value <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-152.gif"/></noscript></span></span> larger than <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-153.gif"/></noscript></span></span> after 50 cycles, also surmounting all other criteria.</p></div><div id="sec-15" class="subsection"><h4>Dominant gene action:</h4><p id="p-25">If gene action at all loci was completely dominant, both the overall level of <em>R</em> (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2a</a>), as well as the advantage of the alternative selection criteria over GEBV (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2b</a>) were reduced, but the extent depended on the criterion. While EMBV appeared to be robust to dominant gene action for different values of <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-154.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-155.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-156.gif"/></noscript></span></span> were severely reduced for <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-157.gif"/></noscript></span></span> reaching only <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-158.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-159.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-160.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-161.gif"/></noscript></span></span>) more than <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-162.gif"/></noscript></span></span> after 50 cycles.</p></div><div id="sec-16" class="subsection"><h4>Number of candidates and chromosomes:</h4><p id="p-26">Reducing the number of selection candidates <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-163.gif"/></noscript></span></span> from 50 (standard scenario) to 30 lead to a reduction in the overall level of <em>R</em> for all selection criteria (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The larger population size with <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-164.gif"/></noscript></span></span> caused a slightly higher allelic diversity in <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-165.gif"/></noscript></span></span> calculated as the average number of alleles per QTL, of 1.97 compared to 1.94 for <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-166.gif"/></noscript></span></span> This increases the probability that rare favorable alleles in the base population are also present in the breeding population, and hence benefits long-term genetic gain. The ranking between different selection criteria for <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-167.gif"/></noscript></span></span> was similar to <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-168.gif"/></noscript></span></span> Comparing OHV with EMBV, <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-169.gif"/></noscript></span></span> tended to decrease relative to <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-170.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-171.gif"/></noscript></span></span> was lowered from 50 to 30 individuals.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic gain (R) in cycle  for selection criteria genomic-estimated breeding value (GEBV), expected maximum haploid breeding value (EMBV), optimal haploid value (OHV) and weighted GEBV (wGEBV) under recurrent selection with purely additive gene action. Boxes and whiskers indicate standard errors and standard deviations across replicates, respectively.  number of chromosomes;  number of selection candidates;  number of selected individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-982444675" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Genetic gain (&lt;em&gt;R&lt;/em&gt;) in cycle &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-172&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for selection criteria genomic-estimated breeding value (GEBV), expected maximum haploid breeding value (EMBV), optimal haploid value (OHV) and weighted GEBV (wGEBV) under recurrent selection with purely additive gene action. Boxes and whiskers indicate standard errors and standard deviations across replicates, respectively. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-173&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; number of chromosomes; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-174&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; number of selection candidates; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-175&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; number of selected individuals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1173/F3.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/4/1173/F3.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1173/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489294" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Genetic gain (<em>R</em>) in cycle <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-172.gif"/></noscript></span></span> for selection criteria genomic-estimated breeding value (GEBV), expected maximum haploid breeding value (EMBV), optimal haploid value (OHV) and weighted GEBV (wGEBV) under recurrent selection with purely additive gene action. Boxes and whiskers indicate standard errors and standard deviations across replicates, respectively. <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-173.gif"/></noscript></span></span> number of chromosomes; <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-174.gif"/></noscript></span></span> number of selection candidates; <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/F3/embed/mml-math-175.gif"/></noscript></span></span> number of selected individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Larger <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-176.gif"/></noscript></span></span> slightly elevated the overall level of <em>R</em> for all selection criteria (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS3.pdf">File S3</a>). With a constant genome size of <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-177.gif"/></noscript></span></span> cM assumed in our study, increasing <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-178.gif"/></noscript></span></span> increased the overall number of recombinations between loci, which benefited long-term genetic gain. The relative differences in <em>R</em> gain between the selection criteria was hardly influenced by <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-179.gif"/></noscript></span></span> However, it must be taken into account that for OHV, we considered only the optimal number of haplotypes <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-180.gif"/></noscript></span></span> For instance, choosing <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-181.gif"/></noscript></span></span> per chromosome yielded optimal <em>R</em> only for <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-182.gif"/></noscript></span></span> but not for <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-183.gif"/></noscript></span></span> (Figure S1-3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS1.pdf">File S1</a>).</p></div></div><div id="sec-17" class="subsection"><h3>Genetic diversity</h3><p id="p-29">The criteria EMBV and OHV generally showed the ability to maintain higher genetic diversity in terms of <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-184.gif"/></noscript></span></span> in the population than GEBV, while criterion wGEBV only showed larger <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-185.gif"/></noscript></span></span> than GEBV for <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-186.gif"/></noscript></span></span> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2c</a>). The rate of decline of <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-187.gif"/></noscript></span></span> became more pronounced when <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-188.gif"/></noscript></span></span> was reduced from 10 to 1. Across all cycles, <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-189.gif"/></noscript></span></span> was always larger for criterion OHV compared to GEBV. After 50 cycles, <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-190.gif"/></noscript></span></span> was entirely depleted with EMBV and GEBV, but not with OHV and wGEBV for <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-191.gif"/></noscript></span></span> Here, <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-192.gif"/></noscript></span></span> (OHV) and <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-193.gif"/></noscript></span></span> (wGEBV) of <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-194.gif"/></noscript></span></span> was left. For <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-195.gif"/></noscript></span></span> wGEBV showed a higher <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-196.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-197.gif"/></noscript></span></span> in cycle 50 that for <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-198.gif"/></noscript></span></span> (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS3.pdf">File S3</a>). Remnant <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-199.gif"/></noscript></span></span> explains why the selection limit was not fully reached in the case of OHV and wGEBV (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2a</a>). This indicates that the final genetic gain of OHV and wGEBV would have been higher if selection was continued for more than 50 cycles. Generally, <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-200.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-201.gif"/></noscript></span></span> had only small effects on <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-202.gif"/></noscript></span></span> for the different selection criteria (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS3.pdf">File S3</a>). Trends were similar under completely dominant gene action (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2c</a>, Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS3.pdf">File S3</a>).</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-30">Genomic selection allows for predicting GEBVs of unphenotyped individuals and has been proposed for RS to increase genetic gain per unit time (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Windhausen <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gorjanc <em>et al.</em> 2016</a>). A first empirical study on GS in a multi-parental population produced from 18 tropical maize lines showed promising results, reporting 2% genetic gain in grain yield per year (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Zhang <em>et al.</em> 2017</a>). However, selection on GEBVs is expected to maximize single-cycle genetic gain, but not genetic gain over several cycles. In this study, we propose a novel selection criterion called expected maximum haploid breeding value (EMBV) as an alternative to the use of GEBVs for RS. EMBV takes into account information about genetic map positions of loci, linkage phases between alleles and the population size to improved long-term genetic gain. We used extensive computer simulations to compare EMBV to two other alternative selection criteria, wGEBV and OHV (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Goddard 2009</a>; <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Jannink 2010</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> 2015</a>) in a generic RS program.</p><p id="p-31">RS was pioneered in maize (<em>Zea mays</em> L.) breeding (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jenkins 1940</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hull 1945</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Comstock <em>et al.</em> 1949</a>) and two basic types of selection strategies have been developed, intra- and inter-population improvement, where the latter is also called reciprocal RS. RS had only a limited but yet significant impact on the development of improved inbred lines in commercial hybrid breeding. Most notably, the Iowa Stiff Stalk Synthetic produced many successful inbred lines and its traces are present in a large proportion of today’s elite germplasm (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mikel and Dudley 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hallauer and Carena 2012</a>). Because of the historically limited success of RS, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Hallauer and Carena (2012)</a> recommend to tightly integrate the development of elite inbreed lines with germplasm enhancement programs driven by RS. This is particularly facilitated by the DH technology, which allows for rapid development of fully homozygous lines ready for testcross evaluation. While RS (either intra- or inter-population) can be used to steadily improve the germplasm, DH lines can be simultaneously created and tested as spin-offs from top parents. We expect EMBV to be also highly suitable for the selection of such DH parents, because by its very definition, it enables the identification of parents that most likely produce top performing DH lines. Genetic progress is then not measured in terms of population mean performance, but in terms of the performance of the best DH that can be achieved for line development, similar to <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> (2015)</a>. If EMBV is deployed for both RS and spin-off DH production, the parents used for DH line development do not need to be recruited from the individuals selected for intercrossing, but can constitute a separate set. This is because the ranking of the candidates in both applications will likely differ due to (i) differences in <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-203.gif"/></noscript></span></span> and (ii) differences in allele substitution effect estimates, which occur if different testers are used and gene action is not purely additive. For intra-population RS, the tester is the (current) population (<em>e.g.</em>, evaluation of half-sibs), whereas for inter-population RS, the tester stems from the opposite heterotic group. In both cases, the selection of DH parents requires substitution effects being estimated from testcrosses. EMBV might also be successfully applied independently of RS in advanced hybrid breeding programs, where new lines are commonly developed from bi-parental crosses between recycled elite lines. However, these extensions require further investigation.</p><div id="sec-19" class="subsection"><h3>EMBV</h3><p id="p-32">The EMBV is an independent property of each selection candidate and is derived from the distribution of their virtual DH progeny. By this approach, the ultimate goal of using EMBVs is not to maximize genetic gain in the subsequent generation, but to improve gain in later stages of the breeding program. This is underlined by our result that selection on EMBVs needed around 5 cycles to outperform GEBV (Figures S4 and S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS3.pdf">File S3</a>), even though the initial penalty of using EMBVs was minimal. By selection on EMBVs, only individuals that are expected to produce the best gametes in the next generation are advanced. If such top gametes eventually unite, a superior individual is created, which, if selected for further breeding, can increase the population mean of future selection cycles. Due to the linearity of expectations, the EMBV can also be expressed as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-204.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-205.gif"/></noscript></span></span> is the GEBV of candidate <em>i</em>, <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-206.gif"/></noscript></span></span> is the standard deviation of the GEBVs of the DH lines derived from <em>i</em>, and <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-207.gif"/></noscript></span></span> the expected value of the largest order statistic of <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-208.gif"/></noscript></span></span> random variables from <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-209.gif"/></noscript></span></span> assuming the GEBVs of the virtual DH progeny are normally distributed. This is described in greater detail in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS2.pdf">File S2</a>. When expressed in this way, the EMBV can be immediately interpreted as a compromise between the candidate’s GEBV (current breeding potential) and its segregation variance (indicative of future breeding potential). Increasing the number of contributed gametes <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-210.gif"/></noscript></span></span> increases <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-211.gif"/></noscript></span></span> (Figure S2-1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS2.pdf">File S2</a>and hence the importance of <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-212.gif"/></noscript></span></span> Hence, the ranking of candidates can vary, depending on <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-213.gif"/></noscript></span></span> (see Figure S2-3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS2.pdf">File S2</a> for an example). Candidates with intermediate GEBVs showed larger variation in <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-214.gif"/></noscript></span></span> compared to candidates with low or high GEBVs. Therefore, selection on EMBV often times chooses candidates with suboptimal GEBV, but in return larger <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-215.gif"/></noscript></span></span></p></div><div id="sec-20" class="subsection"><h3>OHV</h3><p id="p-33">Application of criterion OHV requires the definition of haplotypes, from which the optimal combination of haplotype values is calculated. OHV conceptually fits into the framework of EMBV in that <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-216.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-217.gif"/></noscript></span></span> given complete linkage among loci within a haplotype but free recombination between haplotypes. The need for an explicit specification of haplotypes could be considered as a disadvantage of OHV. Our results demonstrate that OHV has a large potential to boost long-term genetic gain. However, these results might be overly optimistic, because we only used optimal values of <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-218.gif"/></noscript></span></span> As <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> (2015)</a> pointed out, decreasing <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-219.gif"/></noscript></span></span> (increasing haplotype lengths) shifts the breeding goal of maximizing genetic gain into the future, which is underlined by our results (Figure S1-1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1/FileS1.pdf">File S1</a>). The reason is that a gamete exhibiting the OHV (or being at least close to it) can only be produced through the accumulation of favorable recombination events close to the haplotype borders. By definition, OHV only considers the possibility of the optimal gamete combining only the best haplotypes, not taking into account its probability of occurrence (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Han <em>et al.</em> 2017</a>). If <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-220.gif"/></noscript></span></span> is chosen such that genetic gain is maximized at an earlier stage of the breeding program, gain in cycle 50 is compromised (results not shown). As a consequence, OHV needs to be tuned according to the length of the breeding program. We observed substantial losses of genetic gain for OHV relative to GEBV in early selection cycles, in accordance with a simulation study by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Goiffon <em>et al.</em> (2017)</a>. This was not found by <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> (2015)</a>, likely because they evaluated genetic progress in terms of the performance of only the best DH produced from all selected individuals. Hence, if a (nearly) optimal gamete is eventually produced, it will directly and exclusively enter into the measurement of genetic gain. Conversely, we measured genetic progress as the average genotypic value of the entire breeding population.</p></div><div id="sec-21" class="subsection"><h3>wGEBV</h3><p id="p-34">Criterion wGEBV was unique because it performed poorly for small <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-221.gif"/></noscript></span></span> but clearly outperformed all other criteria for <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-222.gif"/></noscript></span></span> in terms of long-term genetic gain. In the latter case, remnant <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-223.gif"/></noscript></span></span> suggests that if selection was continued further, the difference would have been even larger. We suspect that wGEBV is not competitive for small <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-224.gif"/></noscript></span></span> because of strong genetic drift. This will rapidly result in a loss of many highly favorable but low-frequency alleles from the population. Only a very limited number of recombination events occur before individuals ultimately become homozygous; hence, there is not enough opportunity for combinations of favorable alleles to appear. Below, we explain why we expect that the superiority of wGEBV for <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-225.gif"/></noscript></span></span> is likely overestimated.</p><p id="p-35">Recently, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> (2015)</a> proposed a further modification of the original approach to wGEBV. In their study, the effect weights are not only determined by the favorable allele frequency and change due to shifts in the latter, but also by a parameter regulating the initial weight at the beginning of selection and by the number of remaining generations until the end of the breeding program (“time horizon”). The closer the breeding program comes to its end, the lower the weight on effects with low favorable allele frequencies. They showed that their modified approach can improve on wGEBV in terms of long-term genetic gain. However, similar to wGEBV in our study, their method showed a clear performance penalty during the first cycles.</p></div><div id="sec-22" class="subsection"><h3>Genetic diversity</h3><p id="p-36">The genetic diversity maintained in the breeding population was substantially higher for EMBV than GEBV. Selection based on GEBVs puts rare favorable alleles at a high risk of becoming lost. This is because such alleles will occur only in a small number of candidates. If they coincide with many unfavorable alleles, their positive effect is concealed. In other words, if rare favorable alleles are only present in candidates with an otherwise low GEBV, they will likely be lost. On the other hand, criterion EMBV allows rare favorable alleles to recombine and be joined with other favorable alleles into a high-performing gamete, reducing negative selection pressure on them. Moreover, the interpretation of the EMBV as a compromise between the GEBV and the segregation variance makes it evident that EMBV positively weights and maintains diversity. Criterion OHV maintained the highest diversity. Because it is computed as the sum of only the favorable haplotypes, it allows rare favorable alleles, similar to EMBV, to be separated from unfavorable alleles on other haplotypes and joined with favorable ones. Similar to OHV, criterion wGEBV was able to maintain relatively high genetic diversity, but only for <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-226.gif"/></noscript></span></span> This is because the differential weighting leads to a strong selection of rare favorable alleles (<a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Jannink 2010</a>). This effect was canceled by genetic drift if <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-227.gif"/></noscript></span></span> was small.</p></div><div id="sec-23" class="subsection"><h3>Effect of dominant gene action</h3><p id="p-37">In inter-population RS, breeders usually apply the same or closely related testers from the opposite heteroric group for several selection cycles. In this case, the genetic model for testcross performance behaves (in the absence of epistasis) like a model with only additive gene action (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Melchinger <em>et al.</em> 1998</a>), so that our simulations assuming additive gene action closely reflect this situation. However, for intra-population RS, the current cycle usually serves as a tester, and therefore allele frequencies are variable. Thus, in the presence of dominant gene action (<span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1173/embed/mml-math-228.gif"/></noscript></span></span>) the allele substitution effects will change with changes in the allele frequencies across selection cycles. Moreover, in reality dominant gene action appears to be the rule rather than the exception (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Crnokrak and Roff 1995</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hill <em>et al.</em> 2008</a>). For these reasons, we investigated the extreme case of completely dominant gene action at all loci to assess the potential impact on the comparison of the selection criteria. Our results showed that EMBV and, to a lesser extent OHV, are robust with respect to dominance. On the other hand, the performance of wGEBV was severely affected under complete dominance. An explanation for the good performance of EMBV and OHV could be that these criteria are based on the assessment of homozygous individuals (DH lines), which removes the masking effect on recessive alleles present in heterozygotes, which affects criteria GEBV and wGEBV. Moreover, wGEBV was specifically proposed as a criterion for long-term population improvement, so it is implicitly assumed that substitution effects have a long-lasting significance, which does not hold under dominant gene action if allele frequencies change.</p></div><div id="sec-24" class="subsection"><h3>Further research</h3><div id="sec-25" class="subsection"><h4>Estimation of allele substitution effects:</h4><p id="p-38">In our study, we assumed that both the loci as well as the effect sizes of QTL are perfectly known. In practice, QTL are generally unknown and markers are used as proxies and their allele substitution effects have to be estimated from a training set, using one of several available analytical methods (<em>cf</em>. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">de los Campos <em>et al.</em> 2013</a>). However, a high degree of colinearity among markers, especially in high density marker panels, entails that the effect of a QTL is distributed among surrounding markers in a complex manner, reducing the statistical power to accurately estimate their effects (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chang <em>et al.</em> 2018</a>) Obviously, the accuracy of marker effects is further impaired for traits with low heritability or if insufficient phenotypic data are available.</p><p id="p-39">We expect that estimation of allele substitution effects of markers will differently affect the selection criteria. In wGEBV, individual effects are weighted by the frequency of the favorable allele. However, the more inaccurate marker effects are estimated, the lower the significance of their effects and the higher the chance that the wrong allele is considered being favorable, potentially causing selection in the wrong direction. We therefore surmise that the potential of wGEBV is largely overestimated by the assumption of known substitution effects. Conversely, the criteria OHV and EMBV do not rely on individual marker effects, but consider entire haplotypes. This is done explicitly in OHV and implicitly in EMBV, because the virtual DH progeny of a selection candidate reflect colinearity among markers due to cosegregation. Hence, we expect that these criteria are less susceptible to inaccuracies in effect estimates, but this warrants further research.</p><p id="p-40">Furthermore, as the predictive information of effect estimates erodes over multiple selection cycles due to changes in linkage disequilibrium between QTL and markers (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Muir 2007</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Müller <em>et al.</em> 2017</a>), periodic re-training of the prediction model is required, for instance every third cycle (<a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Jannink 2010</a>). Substitution effects should also be recalculated in every generation based on the allele frequencies of the respective tester, which is especially relevant if only a small fraction of the candidates is advanced.</p></div><div id="sec-26" class="subsection"><h4>Phasing:</h4><p id="p-41">The application of selection criteria EMBV and OHV requires the availability of phased genotypic data. If selection candidates are F1 crosses from homozygous inbred lines, all linkage phases are known. Otherwise, genotypes have to be phased before the candidates can be evaluated. In the past years, numerous software tools have been developed to achieve this task, <em>e.g.</em>, PHASE (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Stephens <em>et al.</em> 2001</a>), its successor fastPHASE (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Scheet and Stephens 2006</a>) or BEAGLE (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Browning and Browning 2007</a>). However, as phasing is still associated with a certain error rate, additional investigations are required to assess the influence of phasing error on the performance of EMBV and OHV. While selection criteria GEBV and wGEBV will stay unaffected by phasing errors, we expect that EMBV and OHV could both show a slightly reduced performance.</p></div></div><div id="sec-27" class="subsection"><h3>Conclusions</h3><p id="p-42">We showed in a proof-of-concept that our novel selection criterion EMBV has the potential to yield higher long-term genetic gain as compared to using GEBVs while not jeopardizing short-term gain. Although criterion OHV performed well in the long run, it was not competitive with GEBV in early cycles, which makes it unattractive for practical breeding programs. Criterion wGEBV also showed promising long-term results for quantitative traits with purely additive gene action, but was also accompanied by a performance penalty in early cycles and was, moreover, sensitive to deviation from additive gene action. EMBV could also be a promising approach for the selection of parents for producing DH lines in hybrid breeding programs, which is a subject of future research.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We cordially thank Pedro Correa Brauner, Willem Molenaar, Tobias Schrag, and Matthias Westhues for reviewing the manuscript and providing valuable suggestions for its improvement. We declare no conflict of interest associated with this study. We declare that ethical standards were met, and all the experiments comply with the current laws of the country in which they were performed.</p><p id="p-44">DM developed selection criterion EMBV, conceptualized the simulation study, conducted the simulations, analyzed the data and wrote the manuscript. PS supported in conceptualizing the simulation study and reviewed the manuscript. AEM reviewed the manuscript. All authors read and approved the final version of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: D. J. de Koning</em></p></li><li class="fn" id="fn-4"><p id="p-46">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200091/-/DC1</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-07"><span class="received-label">Received </span>July 7, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-01-31"><span class="accepted-label">Accepted </span>January 31, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Muller <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.1" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>. <span class="cit-comment">10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.2" data-doi="10.1186/s12863-017-0595-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toghiani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aggrey</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rekaya</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">High density marker panels, SNPs prioritizing and accuracy of genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-017-0595-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D19%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-017-0595-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-017-0595-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.3" data-doi="10.2134/agronj1949.00021962004100080006x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comstock</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">A breeding procedure designed to make maximum use of both general and specific combining ability</span>. <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">360</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj1949.00021962004100080006x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DComstock%26rft.auinit1%253DR.%2BE.%26rft.volume%253D41%26rft.issue%253D8%26rft.spage%253D360%26rft.epage%253D367%26rft.atitle%253DA%2BBreeding%2BProcedure%2BDesigned%2BTo%2BMake%2BMaximum%2BUse%2Bof%2BBoth%2BGeneral%2Band%2BSpecific%2BCombining%2BAbility%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1949.00021962004100080006x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1949.00021962004100080006x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1949UC61700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.4" data-doi="10.1038/hdy.1995.169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crnokrak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Dominance variance: associations with selection and fitness</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">75</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">530</span>–<span class="cit-lpage">540</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1995.169</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D75%26rft.spage%253D530%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1995.169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1995.169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TD22100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.5" data-doi="10.1534/genetics.115.178038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spangenberg</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection on optimal haploid value increases genetic gain and preserves more genetic diversity relative to genomic selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1341</span>–<span class="cit-lpage">1348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178038%26rft_id%253Dinfo%253Apmid%252F26092719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEzNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.6" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.7" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9308-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.8" data-doi="10.1534/genetics.116.197103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goiffon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusmec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improving response in genomic selection with a population-based selection strategy: optimal population value selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1675</span>–<span class="cit-lpage">1682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197103%26rft_id%253Dinfo%253Apmid%252F28526698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzE2NzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.9" data-doi="10.1186/s12864-015-2345-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenko</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2345-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2345-z%26rft_id%253Dinfo%253Apmid%252F26732811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2345-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26732811&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carena</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recurrent selection methods to improve germplasm in maize</span>. <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">266</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D57%26rft.spage%253D266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.11" data-doi="10.1534/genetics.116.197095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The predicted cross value for genetic introgression of multiple alleles.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1409</span>–<span class="cit-lpage">1423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197095%26rft_id%253Dinfo%253Apmid%252F28122824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS80LzE0MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.12" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.13" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF01245622</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.14" data-doi="10.2134/agronj1945.00021962003700020006x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hull</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1945</span> <span class="cit-article-title">Recurrent selection for specific combining ability in corn.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Agron.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">134</span>–<span class="cit-lpage">145.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BAgron.%26rft.volume%253D37%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1945.00021962003700020006x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1945.00021962003700020006x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.15" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-35</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.16" data-doi="10.2134/agronj1940.00021962003200010008x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1940</span> <span class="cit-article-title">The segregation of genes affecting yield of grain in maize</span>. <abbr class="cit-jnl-abbrev">J. Am. Soc. Agron.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">55</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj1940.00021962003200010008x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BAgron.%26rft.volume%253D32%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1940.00021962003200010008x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1940.00021962003200010008x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.17" data-doi="10.1186/s12711-015-0101-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Upweighting rare favourable alleles increases long-term genetic gain in genomic selection programs</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0101-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0101-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0101-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1173.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> p. <span class="cit-fpage">980</span> in <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>, <span class="cit-edition">Ed. 1st</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.19" data-doi="10.1007/s001220050767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gumber</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leipert</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuylsteke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuiper</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Prediction of testcross means and variances among F3 progenies of F1 crosses from testcross means and genetic distances of their parents in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">96</span>(<span class="cit-issue">3–4</span>): <span class="cit-fpage">503</span>–<span class="cit-lpage">512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050767</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D96%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050767%26rft_id%253Dinfo%253Apmid%252F24710890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24710890&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.21" data-doi="10.2135/cropsci2005.10-0371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mikel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution of North American dent corn from public to proprietary germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1193</span>–<span class="cit-lpage">1205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2005.10-0371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMikel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D1193%26rft.epage%253D1205%26rft.atitle%253DEvolution%2Bof%2BNorth%2BAmerican%2BDent%2BCorn%2Bfrom%2BPublic%2Bto%2BProprietary%2BGermplasm%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.10-0371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.10-0371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237375900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.22" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00700.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-other" id="cit-8.4.1173.23"><div class="cit-metadata unstructured">Müller, D. 2017 <em>embvr: Computation of expected maximum haploid breeding values</em>. doi: 10.5281/ zenodo.556392.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1173.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite> (<span class="cit-pub-date">2017</span>) <em>Meiosis: Simulation of Meiosis in Plant Breeding Research</em>. doi: .<span class="cit-pub-id cit-pub-id-doi">10.5281/zenodo.581386</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.25" data-doi="10.1534/g3.116.036582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Persistency of prediction accuracy and genetic gain in synthetic populations under recurrent genomic selection.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">801</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036582</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036582%26rft_id%253Dinfo%253Apmid%252F28064189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMy84MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.26" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/502802</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.27" data-doi="10.1086/319501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new statistical method for haplotype reconstruction from population data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">978</span>–<span class="cit-lpage">989</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/319501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D978%26rft.epage%253D989%26rft.atitle%253DA%2Bnew%2Bstatistical%2Bmethod%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319501%26rft_id%253Dinfo%253Apmid%252F11254454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/319501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11254454&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167666000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.28" data-doi="10.1007/s00122-011-1562-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Inghelandt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flament</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extent and genome-wide distribution of linkage disequilibrium in commercial maize germplasm</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1562-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVan%2BInghelandt%26rft.auinit1%253DD.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D20%26rft.atitle%253DExtent%2Band%2Bgenome-wide%2Bdistribution%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bcommercial%2Bmaize%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1562-3%26rft_id%253Dinfo%253Apmid%252F21404061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1562-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21404061&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.29" data-doi="10.1534/genetics.113.155176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the additive and dominant variance and covariance of individuals within the genomic selection scope.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1223</span>–<span class="cit-lpage">1230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155176%26rft_id%253Dinfo%253Apmid%252F24121775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">“Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments”.</span> <abbr class="cit-jnl-abbrev">G3.</abbr> <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3.%26rft.volume%253D2%26rft.spage%253D1427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.31" data-doi="10.1051/gse:19940504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Increasing long-term response to selection</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">431</span>–<span class="cit-lpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-26-5-431</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19940504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19940504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PL67000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.32" data-doi="10.1186/1297-9686-45-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toosi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-45-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-11%26rft_id%253Dinfo%253Apmid%252F23621868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23621868&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1173.33" data-doi="10.1534/g3.117.043141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Rapid cycling genomic selection in a multi-parental tropical maize population.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">2315</span>–<span class="cit-lpage">2326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.043141%26rft_id%253Dinfo%253Apmid%252F28533335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNy8yMzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/4/1173.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1161" title="The Molecular and Genetic Characterization of Second Chromosome Balancers in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1183" title="Genomic Predictions and Genome-Wide Association Study of Resistance Against Piscirickettsia salmonis in Coho Salmon (Oncorhynchus kisutch) Using ddRAD Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/4"><span>Volume 8 Issue 4, April 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/4.cover-source.jpg?itok=wC37V0ox" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1173" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IrXbMUInJ2lgY9qfkq_VIBxsOQRLbYdKnM0A04TzzvQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/4/1173" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/4/1173" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/4/1173" class="active" data-icon-position="" data-hide-link-title="0">Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489127" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_Gdpnz6WjXdoBCZTXwOIciSH-Zpyx7fXczZbrk27x_U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1173.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/4/1173" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eP2dhNx56q7MJ5vPSu7wPhEcz7Lf7si7y6jjmDpkc30" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/4/1173" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1173.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489127" id="node489127--2" data-pisa="ggg;8/4/1173" data-pisa-master="ggg;g3.118.200091" data-apath="/ggg/8/4/1173.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dominik Müller</span>, <span class="highwire-citation-author" data-delta="1">Pascal Schopp</span> and <span class="highwire-citation-author" data-delta="2">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1173-1181;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200091 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489127--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominik Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominik%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pascal Schopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>KWS SAAT SE, Grimsehlstraβe 31, 37574 Einbeck</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BSchopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schopp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BSchopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/4/1173" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/4/1173&amp;title=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/4/1173&amp;title=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/4/1173&amp;title=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/4/1173&amp;text=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/4/1173&amp;title=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/4/1173&amp;t=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/4/1173&amp;title=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/4/1173&amp;title=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489127/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489127" id="node489127--3" data-pisa="ggg;8/4/1173" data-pisa-master="ggg;g3.118.200091" data-apath="/ggg/8/4/1173.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection on Expected Maximum Haploid Breeding Values Can Increase Genetic Gain in Recurrent Genomic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dominik Müller</span>, <span class="highwire-citation-author" data-delta="1">Pascal Schopp</span> and <span class="highwire-citation-author" data-delta="2">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1173-1181;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200091 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489127--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominik Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominik%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominik%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pascal Schopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>KWS SAAT SE, Grimsehlstraβe 31, 37574 Einbeck</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BSchopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schopp%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BSchopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Hohenheim, Institute of Plant Breeding, Seed Science and Population Genetics, Fruhwirthstr. 21, 70599 Stuttgart and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1173&amp;rft.epage=1181&amp;rft.atitle=Selection%20on%20Expected%20Maximum%20Haploid%20Breeding%20Values%20Can%20Increase%20Genetic%20Gain%20in%20Recurrent%20Genomic%20Selection&amp;rft.volume=8&amp;rft.issue=4&amp;rft.date=2018-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=M%C3%BCller%2C+Dominik&amp;rft.au=Schopp%2C+Pascal&amp;rft.au=Melchinger%2C+Albrecht+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489127/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489127/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489127/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489127/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489127/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489127/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489127/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489127/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489127/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489127/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489127/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489127/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/4/1173" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/4/1173.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/4/1173.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/4/1173.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/4/1173.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
