<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/4/1347" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/4/1347.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/4/1347.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/4/1347.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/4/1347/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/4/1347.atom" />
<meta name="HW.pisa" content="ggg;8/4/1347" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials" />
<meta name="DC.Identifier" content="10.1534/g3.117.300454" />
<meta name="DC.Date" content="2018-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Cuevas et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In this study, we compared the prediction accuracy of the main genotypic effect model (MM) without G×E interactions, the multi-environment single variance G×E deviation model (MDs), and the multi-environment environment-specific variance G×E deviation model (MDe) where the random genetic effects of the lines are modeled with the markers (or pedigree). With the objective of further modeling the genetic residual of the lines, we incorporated the random intercepts of the lines (l) and generated another three models. Each of these 6 models were fitted with a linear kernel method (Genomic Best Linear Unbiased Predictor, GB) and a Gaussian Kernel (GK) method. We compared these 12 model-method combinations with another two multi-environment G×E interactions models with unstructured variance-covariances (MUC) using GB and GK kernels (4 model-method). Thus, we compared the genomic-enabled prediction accuracy of a total of 16 model-method combinations on two maize data sets with positive phenotypic correlations among environments, and on two wheat data sets with complex G×E that includes some negative and close to zero phenotypic correlations among environments. The two models (MDs and MDE with the random intercept of the lines and the GK method) were computationally efficient and gave high prediction accuracy in the two maize data sets. Regarding the more complex G×E wheat data sets, the prediction accuracy of the model-method combination with G×E, MDs and MDe, including the random intercepts of the lines with GK method had important savings in computing time as compared with the G×E interaction multi-environment models with unstructured variance-covariances but with lower genomic prediction accuracy." />
<meta name="DC.Contributor" content="Jaime Cuevas" />
<meta name="DC.Contributor" content="Italo Granato" />
<meta name="DC.Contributor" content="Roberto Fritsche-Neto" />
<meta name="DC.Contributor" content="Osval A. Montesinos-Lopez" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Massaine Bandeira e Sousa" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="article:published_time" content="2018-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In this study, we compared the prediction accuracy of the main genotypic effect model (MM) without G×E interactions, the multi-environment single variance G×E deviation model (MDs), and the multi-environment environment-specific variance G×E deviation model (MDe) where the random genetic effects of the lines are modeled with the markers (or pedigree). With the objective of further modeling the genetic residual of the lines, we incorporated the random intercepts of the lines (l) and generated another three models. Each of these 6 models were fitted with a linear kernel method (Genomic Best Linear Unbiased Predictor, GB) and a Gaussian Kernel (GK) method. We compared these 12 model-method combinations with another two multi-environment G×E interactions models with unstructured variance-covariances (MUC) using GB and GK kernels (4 model-method). Thus, we compared the genomic-enabled prediction accuracy of a total of 16 model-method combinations on two maize data sets with positive phenotypic correlations among environments, and on two wheat data sets with complex G×E that includes some negative and close to zero phenotypic correlations among environments. The two models (MDs and MDE with the random intercept of the lines and the GK method) were computationally efficient and gave high prediction accuracy in the two maize data sets. Regarding the more complex G×E wheat data sets, the prediction accuracy of the model-method combination with G×E, MDs and MDe, including the random intercepts of the lines with GK method had important savings in computing time as compared with the G×E interaction multi-environment models with unstructured variance-covariances but with lower genomic prediction accuracy.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/04/01" />
<meta name="citation_mjid" content="ggg;8/4/1347" />
<meta name="citation_id" content="8/4/1347" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/4/1347" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/4/1347.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/4/1347.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/4/1347.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300454" />
<meta name="citation_pmid" content="29476023" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1347" />
<meta name="citation_lastpage" content="1365" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jaime Cuevas" />
<meta name="citation_author_institution" content="Universidad de Quintana Roo, Chetumal, Quintana Roo, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0685-2867" />
<meta name="citation_author" content="Italo Granato" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2093-6810" />
<meta name="citation_author" content="Roberto Fritsche-Neto" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4310-0047" />
<meta name="citation_author" content="Osval A. Montesinos-Lopez" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, CP 28040 Colima, Edo. de Colima, México" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México" />
<meta name="citation_author" content="Massaine Bandeira e Sousa" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=P. L. Cornelius;citation_author=R. Trethowan;citation_author=G. McLaren;citation_title=Modeling Additive x Environment and Additive x Additive x Environment Using Genetic Covariances of Relatives of Wheat Genotypes;citation_pages=311-320;citation_volume=47;citation_year=2007;citation_issue=1;citation_doi=10.2135/cropsci2006.09.0564" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Crossa;citation_author=J. Burgueño;citation_author=P. L. Cornelius;citation_author=R. Trethowan;citation_author=A. Krishnamachari;citation_title=Modeling Genotype x Environment Interaction Using Additive Genetic Covariances of Relatives for Predicting Breeding Values of Wheat Genotypes;citation_pages=1722-1733;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2005.11-0427" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez-Rodríguez;citation_author=J. M. Hickey;citation_author=C. Chen;citation_author=G. de los Campos;citation_author=J. Burgueño;citation_author=V. S. Windhausen;citation_author=E. Bucker;citation_author=J-L. Jannink;citation_author=M. A. López-Cruz;citation_author=R. Babu;citation_year=2013;citation_pmid=24022750;citation_doi=10.1534/g3.113.008227" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=M. Maccaferri;citation_author=R. Tuberosa;citation_author=J. Burgueño;citation_title=Extending the marker × environment interaction model for genomic-enabled prediction and genome-wide association analysis in durum wheat.;citation_pages=2193-2209;citation_volume=56;citation_year=2016;citation_issue=5" />
<meta name="citation_reference" content="citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=O. Montesinos-Lopez;citation_author=J. Burgueno;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_year=2017;citation_pmid=27793970;citation_doi=10.1534/g3.116.035584" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic Prediction of Genotype × Environment Interaction Kernel Regression Models.;citation_pages=1-20;citation_volume=9;citation_year=2016;citation_issue=3;citation_doi=10.3835/plantgenome2015.04.0025" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="de los Campos, G., and A. Grüneberg, 2016. MTM (Multiple-Trait Model) package. http://quantgen.github.io/MTM/vignette.html." />
<meta name="citation_reference" content="de los Campos, G., and P. Pérez-Rodríguez, 2016. BGLR: Bayesian generalized linear regression. R package version 1.0.5: https://CRAN.R." />
<meta name="citation_reference" content="Granato, I., J. Cuevas, and F. Luna, 2017. BGGE (Bayesian Genomics G×E). https://github.com/italo-granato/BGGE/tree/master/R." />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. Fritz;citation_title=Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_author=M. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=R. P. Singh;citation_author=E. Autrique;citation_author=G. de los Campos;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. Mota;citation_author=R. Tempelman;citation_author=P. Lopes;citation_author=I. Aguilar;citation_author=F. Silva;citation_title=Genotype by environment interaction for tick resistence of Hereford and Braford beef cattle using reaction norm models.;citation_volume=48;citation_year=2016;citation_issue=3;citation_doi=10.1186/s12711&amp;ndash;015&amp;ndash;0178&amp;ndash;5" />
<meta name="citation_reference" content="citation_author=H. Oakey;citation_author=B. Cullis;citation_author=R. Thompson;citation_author=J. Comadran;citation_author=C. Halpin;citation_author=R. Waugh;citation_year=2016;citation_pmid=26976443;citation_doi=10.1534/g3.116.027524" />
<meta name="citation_reference" content="citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manès;citation_author=S. Dreisigacker;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_author=K. Bondalapati;citation_author=G. De Meyer;citation_author=F. Pita;citation_title=A Pedigree-Based Reaction Norm Model for Prediction of Cotton Yield in Multienvironment Trials.;citation_pages=1143-1151;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.08.0577" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=S. Pérez-Elizalde;citation_author=J. Cuevas;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_title=Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction.;citation_pages=512-532;citation_volume=20;citation_year=2015;citation_issue=4;citation_doi=10.1007/s13253-015-0229-y" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=H. P. Piepho;citation_title=Analyzing genotype-environment data by mixed models with multiplicative effects.;citation_pages=761-766;citation_volume=53;citation_year=1997;citation_doi=10.2307/2533976" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. P. Piepho;citation_title=Empirical best linear unbiased prediction in cultivar trials using factor analytic variance covariance structure.;citation_pages=195-201;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050885" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. B. Smith;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_title=The analysis of crop cultivar breeding and evaluation trials: an overview of current mixed model approaches.;citation_pages=449-462;citation_volume=143;citation_year=2005;citation_doi=10.1017/S0021859605005587" />
<meta name="citation_reference" content="citation_title=G3;citation_author=M. B. Sousa;citation_author=J. Cuevas;citation_author=E. G. O. Couto;citation_author=P. Pérez-Rodríguez;citation_author=D. Jarquín;citation_title=Genomic-enabled prediction in maize using kernel models with genotype × environment interaction.;citation_volume=7;citation_year=2017;citation_pmid=28455415;citation_doi=10.1534/g3.117.042341" />
<meta name="citation_reference" content="citation_author=S. Sukumaran;citation_author=J. Crossa;citation_author=D. Jarquin;citation_author=M. Lopes;citation_author=M. P. Reynolds;citation_year=2017;citation_pmid=27903632;citation_doi=10.1534/g3.116.036251" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull.;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="twitter:description" content="In this study, we compared the prediction accuracy of the main genotypic effect model (MM) without G×E interactions, the multi-environment single variance G×E deviation model (MDs), and the multi-environment environment-specific variance G×E deviation model (MDe) where the random genetic effects of the lines are modeled with the markers (or pedigree). With the objective of further modeling the genetic residual of the lines, we incorporated the random intercepts of the lines (l) and generated another three models. Each of these 6 models were fitted with a linear kernel method (Genomic Best Linear Unbiased Predictor, GB) and a Gaussian Kernel (GK) method. We compared these 12 model-method combinations with another two multi-environment G×E interactions models with unstructured variance-covariances (MUC) using GB and GK kernels (4 model-method). Thus, we compared the genomic-enabled prediction accuracy of a total of 16 model-method combinations on two maize data sets with positive phenotypic correlations among environments, and on two wheat data sets with complex G×E that includes some negative and close to zero phenotypic correlations among environments. The two models (MDs and MDE with the random intercept of the lines and the GK method) were computationally efficient and gave high prediction accuracy in the two maize data sets. Regarding the more complex G×E wheat data sets, the prediction accuracy of the model-method combination with G×E, MDs and MDe, including the random intercepts of the lines with GK method had important savings in computing time as compared with the G×E interaction multi-environment models with unstructured variance-covariances but with lower genomic prediction accuracy." />
<meta name="og-title" property="og:title" content="Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/4/1347" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In this study, we compared the prediction accuracy of the main genotypic effect model (MM) without G×E interactions, the multi-environment single variance G×E deviation model (MDs), and the multi-environment environment-specific variance G×E deviation model (MDe) where the random genetic effects of the lines are modeled with the markers (or pedigree). With the objective of further modeling the genetic residual of the lines, we incorporated the random intercepts of the lines (l) and generated another three models. Each of these 6 models were fitted with a linear kernel method (Genomic Best Linear Unbiased Predictor, GB) and a Gaussian Kernel (GK) method. We compared these 12 model-method combinations with another two multi-environment G×E interactions models with unstructured variance-covariances (MUC) using GB and GK kernels (4 model-method). Thus, we compared the genomic-enabled prediction accuracy of a total of 16 model-method combinations on two maize data sets with positive phenotypic correlations among environments, and on two wheat data sets with complex G×E that includes some negative and close to zero phenotypic correlations among environments. The two models (MDs and MDE with the random intercept of the lines and the GK method) were computationally efficient and gave high prediction accuracy in the two maize data sets. Regarding the more complex G×E wheat data sets, the prediction accuracy of the model-method combination with G×E, MDs and MDe, including the random intercepts of the lines with GK method had important savings in computing time as compared with the G×E interaction multi-environment models with unstructured variance-covariances but with lower genomic prediction accuracy." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
    <title>Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wVyWjjKq3TQYpUv_Iqdk5F6PRDKPoQHVRI8KJ0N-7WY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489137","apath":"\/ggg\/8\/4\/1347.atom","pisa":"ggg;8\/4\/1347","ac":{"\/ggg\/8\/4\/1347.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/4\/1347.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1347"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1347"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/4\/1347":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ARK0OYSFAFCRgLq0foVyncJE0bNXhOdjhCVWYGEPxJu+JQWIcyEuk+NU28WeQ5dsufQb4X1az9Q2576tonPh\\\\\\\/czF2w9lDZ1GRh4\\\\\\\/875oiVEXjAhqCY8XaJw6wc6p3tZmGPO43OR3rhZ741pmAZWkB4PvVcMU6\\\\\\\/KXV5zLZRW12DsOb07jHhvmShyP7dnLpSKiEnWqRfECmcA9DgO8hOCGFDlJyymCMp5XdvuAn+KKV0t8m4em6NSC8\\\\\\\/Pd5rd1ewEE3KtSQnNqa1xjowb77j8AfLktXYa\\\\\\\/5TFXdvtPRzhyl3iZEj0jBiqU78hmQlCb6S0HRyN4FB9LL3T4DjWkQ060Q8j+SyfocPHFFLSaQpmVxGXArc2yFaah9OpEqAPRZJETlI4xw2DFPSa\\\\\\\/iKStzOTCkozCynEQpMibOSffonS9GNEzrOfw812+NJ5\\\\\\\/ABdXXRWQXxqVhQXgxVA0ELVWsRBJv19qOby12WLLXIFbTU221+wHsWm9cpJeR2hB3ZtKEHUoZec2Q7bfsgqe2y1+UzhBrihMYq9MI+jsOot3XNTOM+doXQF0Dz6rQszxcz6Ow6yiYpmeDAOigQe4IvkjoS6EDGhLv6oEg0KhrMrLOmXcYYNpfyUPoDCw69AbTgza860QmAjeUAQJdh8V61ru3MkuXd8svIa6sUls02W8O0XqFrNkyZQZ+bD5IyvRNHmqYf481jAQ4JNwKR8XBbI2+ns4lcQcSaLAVFysseXr6VV3aGl2Rp\\\\\\\/LdvpsnYsjGxNdDK4U7wiCHgkIXx+hBEycVWjC4EsscWVedVqDaRag6J4s8hfEunYxIYt8F6+shgFEKKVuxYG+uzfQn3XtTZhKD8UnUsZP1pgpipIi+4gT\\\\\\\/7GiOWu96HV5D7gs7rjG5RK1epiCQcgSjALO4ItD0ltwBWQyW7oZawRaOiv0HRIiAZwWbBFbwjj5cbU9PxfT5Ch63ceAKlFHbL\\\\\\\/Xo+k4fBlozjW6PhFrlWr\\\\\\\/DSZRWc0jYPoc5D\\\\\\\/r+mEjJLvzZLUgyg5SmMHpVLUDKHd8GetIjDKnJsHuBr4f9kDE7WkgsEqyahcRuXAp15tyfEsxxG7GgpbuiDjFEHHLiiECpZE927di0BSEbjmjtiFsQHXZuLlqmr7Th5yydjHcc4Ia6\\\\\\\/5YtvWlcewnURT2uA+mq5K3zbxjVeGdH1ud9FC2fanp6Yc69hcUhtGe4tbxh3hMh\\\\\\\/ZRzMsOnUteYk3wi\\\\\\\/3\\\\\\\/FO7XxJ4RyX+l3zNRZGGXk1jX9HtvMRJafi4ZkTBUbPAZJ2KHOF+DOP22\\\u0022,\\\u0022iv\\\u0022:\\\u0022G3pAaV9HWFCcMyXYUJIptA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e7c48ce5be591d97041787a108262e92af49e927a17cea8c0ab1d2aee83f774c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+B5iZUyfvXWYnO2v4TJuKQyAl1HlAQD4Q96EIsLEE8IVNKmAUsoPYA7AKfqzO8JvXsBEitMMDg0\\\\\\\/BuCdJB7ZI\\\\\\\/lNheAFYpLYSWRvAnPQK9vchP8NVyIS29Efo+wdJGuaCgd8PnWp+yJGcfnBk3xARLdbZYEIDtsy2BOhiNkQayhFR5zdQQzRpod97TbTO7fgY3qwDjK0NBLn5ZdqZY\\\\\\\/mFltKFuqwqsXqW3cqo0KYhCdWsI29+6Bu8S8eGZvcL44U3hkXcpTAH0hPk83AgPgoacl6XtcnI8s8tEHgE2OmHxV4Q5nbldx58qG3Zbxs4mwq4iFS5wdzhho7tDBnVLlEADklgjQXmkKbz12W4Qk6+56C5St1MnMuhJMkCWdAOuijKk4Tv\\\\\\\/rPRTLjmIoYmvGULCRMHKGqq3v7oLHoQeJgGcDbonEiaJ0eoWBauG815e\\\\\\\/3GtOSUP7v5qnwEmO+qttDT1i3k2sR\\\\\\\/k4nL5H6LHUVKoproVA8pyw5eAfH4G91pPZ1s+0ZHvkmoRgC4wiv1HEyOvhuNPR9kIxDO9Nyj65DCiSYWse+cjdcOtegniGnZhufiR8Lpl50Bhu1Oo+aGhppO+IlyJPk+8xLvlkhazEBhYsQlG9Tq1InOktcBDCL8ZPw+C+e0NHp21kjHtZrtsku9byFlT8QgyfIQoKXsm52AkDFaT6RzfzogUdoxR39G5w3fXWqIFBS57qbPDF30g\\\\\\\/4xf1I6hUjYr1vgFwU3SuhOtaj9M0XR0Bk918jL+LZ2Cka3P3A0Y0JYlAWu4LNbAMRYUm9a7PZdS2flWpeLGdKaT9P+sK3D+xrlnQiaXcEtGoM9Y+q\\\\\\\/Y\\\\\\\/wzCZ5XzmHktF49iq3oiZAhAQzLjaZe2z3b2QFR8SnCFArqO92UW9RhyN\\\\\\\/MBLF3pZFAMf6\\\\\\\/7kzESYfrGpxDwn6qcgJOz7A\\\\\\\/WYQoQg5j+LPfPbKjkQbWZ2Dv1sDCP3rbjeyePqQZgrGPntAkQYYQB9QAOjPNhYvSrtmuSifJJaJlqgHnAPdGX6BQKpm4J+yb8vjWa+t+SAZTrpEh\\\\\\\/\\\\\\\/UA10+ztffMgvPc83ATT07jTFMZK2xbxxIx29T7yV7Jdxgc0COdpQQI4ElVni814SYMqGPme4dxzb3Jj8Vu31emf+WvW\\\\\\\/th42XJzpLAMtQq2RH3REryY\\\\\\\/6l14QAWrkG8qbM41Waelw85h5uIHWOUTqC6+v+cpaoLIzmVM9F\\\\\\\/acau6QNsLP5bjgnFyVLZTZVuKHJtuahbcaM\\\\\\\/C3bSBHH\\\\\\\/zMT+jmfVNGHcMtnyyB2zZN21gQal70A676qZG0EUp0G8NsytszGvUO+zVYVA6VqoO349oRlWg0c6CxJEqElD+I7+7qv8vJzdeeG7btg3yJvModZCa68IOm3de7EqZdO4eDxktGjUxUkDzTqMB6RV\\\\\\\/3c22yBk7z8MhUrtETxSuUwXKPQepsN53AUlNB0oCWjNw1Ffu8l\\\\\\\/Fr9URRcSzwbA6ojll4gJpoFYDXd24palrJGA3OXTp3SPF6PfTTeju8U86kwQbwBqwKctqnX7MRaUhJj9zJPi+np4SADlna\\\\\\\/jsXl\\\\\\\/gEzRwNdjcaxtRxDb68W35Cp5J919wqIAfhoKpbcBpZw2dMT63pNLoaElcQm7o+yF+g2wpuMnWqC3SELkdbYlZVwl6eJbw7OPRB\\\\\\\/Kpefcy10MNLSayB6xy+TmoexBV5qezE3b3zkUQzRvhcCbKPKN30uuRdRSEsX6Ni\\\\\\\/PGWrrs10rfb531l8xsdAK6oMNVFXiMhqKMcutmhFDLanyzuTOP\\\\\\\/n3tbkdMjWTtB3fj3buwk8UlMA4J1khBBsbs7JYwixH4GCGKvdsEiPSoKO3wUrqeDHXv79+3xv2rDR2GmsYNAsES79mUUMBYmo2KTi\\\\\\\/Fc98yjpN\\\\\\\/RtVVJNGnsdtQowjwWjUbiVXr81ST2+S8JLzDOJr\\\\\\\/A1y3Cs\\\\\\\/x2+\\\\\\\/rBm9eQoGYf30UqT0Voi2eOJfrFx43g98M1LUX2v1nBaRhb9zE83jEqVrrLwa8z+HihtHos\\\\\\\/DsQIvveLVPyAqqEywBmmHK7ol2pELKd6B\\\\\\\/hONgT5UiYkYMYqYmSlvRqDmI\\\\\\\/zjZazyaaPYccvm4fJm75ik5gTptxTY9ay+\\\\\\\/D0MU2Yx+6o1HKn7qCmQpa5I9SWYLX+ZDTj5Fs\\\\\\\/HtvWrxhpJBiVKYPir+8m66VEbVyxaG2S4Xd\\\\\\\/XDsvtpDcm033Ra4KQKAG6L9hUKwSU+ANCbyFjJBgs+zjK0kHimMi5zr+kI1yHxQj1cneUxOBGd+bTdyAyCNmSW\\\\\\\/t1wYaCnQ9MuBE06tn1qqNmPWjTaRRWX5+G230iGB6W1NhmL3Y1duhhdVU\\\\\\\/J9vKN9I3Vd28CKgXb75mDhft4wt\\\\\\\/EW1dSTcES1hcjesomXTJnW66A9u4QdktwznoMgDDm9Ym3xcJDfdTk\\\\\\\/ixgtXOmM4JjrOQt5ZKn6DF0jkIAvd5ZLLKBAgZwiAojUJECHH3H6Tf4tC2SVWa8\\\\\\\/xYTz0mPnpkVcooJxaau8OPB7gN\\\\\\\/ytDG\\\\\\\/JeOfQDE+Gw8ahuUXdPDR3wpb4FWB3ksC939y1HuZtP8aNdqUlakDj\\\\\\\/vn2oaiUVBuHOVJvEwoWV7skEo\\\\\\\/7ZawSB7clibzxnlPYF5IKhog7qrzt6CAPqVNJ55vvrf8F6c9S2qlzdeK77cP5zrB2TkoOI1gLGossdDLl4BxMpNvfLB+rAicmlKrO\\\\\\\/5qWjMVE5kgtmGdrW9OLey1RERaU1UzIbeloLgB3KMBTw1qc9aXtTmyDR7cwsZuTbGHKZnmnoU\\\\\\\/RqYHk6CQixwJdH6Vn5nXL+zXvvLHs7sgY4324PPVx2TL\\\\\\\/ABO3EHpjVcuC5iEp2CwGgxBrLbtD4YGBDeh1q29uXziVROrsfsTBSV9AF4KjTO8kbboQqxTqdG0BVlr+jt54zU0YxL\\\\\\\/3eAK5Zyy3my18ZHiceyVB62bg9dW+zqIS2aQc\\\\\\\/0vha13Ia\\\\\\\/g8p0w0iLKgWzzyAV5eo163WJTq1c4ZioeigMgVqcV7U8o2d1HhQAgFTbhEl+DcLkNm3PvUTs7TDxC9Q2aF75ioxqK7l9QH3bcbxLB0U2l8z9bvVieYKeE5oyaMl+cMuKVr5J\\\\\\\/Z6CvIbIH0F\\\\\\\/k6MTK5A77kwreD2nWmjFnzIwh5RYpZ+yzBk6lSn1RSzFfHK77Wsd70w4PEc3ub4iD1sPnDz\\\\\\\/2ULGdUVlSaziA8NMxaCAqVTPSFgvzOG5EdPGfMbNYHDMrGyni\\\\\\\/BGXlgYodZaBHKgqAUPdV5hGF0TbxLvvxGjpP11y7m0jnBBJhOsx7LsW4o9EBhB\\\\\\\/rNFHnVhExgL\\\\\\\/T8EeRGFm4U9e00pXWHMFNEH87yM3EyYp7apB8B2VtSzrcDlHsNj6t5nxKVt4flR3QoL3eA6CywJCTOSfTZsqZDOc0vzs+7lNb2vlD4rztzqYPq8RgqZP2huGXfNAS3LHDPAXtMkh2QL+GfKLcg28CXw+9Zp2th17GMuT1l63pKdjnEKXIq6pj\\\\\\\/Xm77W8A+P5dMqCP4UrggYQBnhPngGgvrspIK67cYROM1J9FiqL7ybj8wc55oR3ecftqX5+cO4SXqRNnsSiYdW5J53+Bnnx5WKP7F1fKDySeAp9ySwayH5N3xt+UKCZCgyfxa3mr5ti15o9Y0lmOmZFWoii3KR4xbaOJOi2SBGMARjqORCopUFrCEJ\\\\\\\/jm8mqK\\\\\\\/fYTWRQD08cjypykZ0r7Wdp2ALI9qNBzmiEe7duoSYL2vQAO6giCSJqZe9zMMTS+7mOLZsP88w+r1DyJXZVKNclNyFeKsiRFMc3dAU5yjNrLo1t+yHjr89VuFXmBHaV6a0lhHPH5P2g20bMc6A5Xcr19lV2X+J4xip+Q8UdxkIJC\\\\\\\/\\\\\\\/b2kzumA3c1HQ53y9W7EDLvucHNfNPSrCzRyirn8kA7qVsx6GmEyMdaZ8ck4pR60C7bKQLX+MI6gvRc8sicRi\\\\\\\/LZ8EkKEQeCbdgPPWoH4smE\\\\\\\/obROE5y+zBL8oiXJJvCssnqfkB24fYcC2zIyZJCZgM4F3bQQwGiJY4+bXyA+sGBUj6aUAcKK\\\\\\\/42cSTQuGq159O2fBcLMSR8yRgmG1FutgDR3w\\\\\\\/rcdYe6+Fvrg\\\\\\\/qOMy27aFO2ECfDPdnqQ9kgRtdpnp9EuPJ\\\\\\\/th7pLvpC5SrJh+7oBmAYAyhP1Vk+y\\\\\\\/8SnoR+qxQhLmqRUz+ANMG24D6E5u1B4VQR2TkKPkgZzuQLbg60p9ocwxbdrY0t1PukvUkvHMBedckFsffH1tP5p\\\\\\\/qrcCJ2ZGITHa\\\\\\\/IVvcaB\\\\\\\/W8IOu0t2kSZS8U1qu6eVo57cd4JAb2qOjPTL6p1ZN0h8PivkHPl2xabTbo\\\\\\\/eqk3DSqbM5pSMM6cLsaYRdsceQnaCuc\\\\\\\/dXd2V7rqN0CFoH+CdmqHdxqEDvLVbiWB5RAh9NKvAz\\\\\\\/MYLhUNSBBPWUbDVI1BTEYBRu3PT+pTiQ1BqsB1S9gwPgN6yLiJ8xXoYhkafkt9fgUp\\\\\\\/W83mr0iuTdIiu1dtf\\\\\\\/ugiZaEvP7uSPkcKlvLVncZ29ex1zrXujr5gNuhGjGBqn1L4gCBk6uScq5Q2zn003cXAJj1r\\\\\\\/oBT7jbfk1FWoZJUkaIMs7hcf+7i9VSyvLwiEH34Gi9cHpFqmmY9SR98oLsr8Qt9ImUL8cL56uZ\\\\\\\/Z8qsyvEnp76D2a75+MeJosDvr+XnH3h+g\\\\\\\/9RMIlmzZVH3p0VLAggiusiFIgX6oeKfJ379ipa7Fy0tbL2vVTTwCcwEBf0nwSxfKA5Pq1ylor3tcFkj+4vFGeYgkzAGNa7NwHUssSBEwfP1X4ApGqEs2PMaVPC+zk1UrzjN0CU2KWF5AtOxN65oybrC9Rsn6Y7Vew6SCMKMStB1g21M\\\\\\\/Cq2FeavR5DZro8y2FHZHtjWly0vI6wWub4jkB5oNpCm\\\\\\\/YMZV\\\\\\\/xeV4RACuAhbUftnMO0xPsJjg4PcDGGH7WRyk\\\\\\\/vKdqbe3Tb1rXa8yEYt20Up5cgTPd\\\\\\\/HhFQ\\\\\\\/BlMNAJ7H47tEU6+v5l4EvfiI8A6U8OhaGKUjaUinxP4uCxnR7Lu89jUvNFO6\\\\\\\/YeZcOrsZqBmdd2gyHFN30ab1dig2aZK\\\\\\\/XoPj5woYCfC9XEN9UyMcKgP+LAf6aGLJEEzqHT5ze7o81q7Jc+So8gqmQr06O3dEPb+lgEZ9baZV\\\\\\\/53WYpcVh8Ps4RiwiR8cdA6E9GfrwRtJxvN5Hmn4kjfA4JbLkuxZ3U\\\\\\\/pl3CPVPd+1c8IsfprqhZYPjvli+A+a4HOckS2uksvbuy2KuEUSVd4+\\\\\\\/OWFEEAFStO8O6JAM6j6NvIIpPuIw5nxK4JbAtTSL8GtSI5BjWQiI9aAciBIJeBEKeVRlRmhXzYI7IHLF\\\\\\\/Y9RJyIz\\\\\\\/o\\\\\\\/GDAjz1NM5pNsn+e2lT7pCtZ4nn+nq2cKg4vjzB9QyJMlF3s7jXE2SB7LO6nfSWw7LprQ7JmsB0n9\\\\\\\/JDZichzW\\\\\\\/UiQdxThZa24tNLpH54ddu4HTiW+R2\\\\\\\/YiGDZwE3449ukmViEQa6AIzfYFL2Hy7PSKM3cVgAUvJkkXyI7HwcFR8k5IGj7B49zjcny3w\\\\\\\/nPdZTqwG54UQae+mTQWssFSZo4JXaoXaRDnGdYnKil5kzaSSL0A3YnPSw+Q05vFTPxsoZi3i\\\\\\\/38vqjLOgZOW7eawp2qx+wRm08FJxcO+9bhASu4VIwItUEC8eo5AuniAl2sKEJEBNA03zPWcAPJ969Amjm6D4eNPqxTq9mPewKjCoXXxl0KEpICh\\\\\\\/pG8mRp6Vw7iTX8M+D2gqa1WogtKN2SEhygl5X+1GB\\\\\\\/fsgrvRALP+26gN19n0LjpR9VJ117JdLFUO6xdfrjYXnQkKdzPG9oUhPQqbBZMziakiBnJDbnVHIgcBI\\\\\\\/TkcE5Upxg3mENY5oLu3NHKAErHgR4Z0CMjiV4fXyrRKY2SLEsPDjcN3wCmJ7uMWPa2CFXPH7I0xHqp6hLkr+ptrB\\\\\\\/Sa3rmT7zJ6szqQGo4kDc+2lZcglMfB2uIxkxC+6c9XDaJePCIMYlH5Lj7TeHHC4oi7nmy0FZhvCntKYvhHlGE9KOc3WkUyD1s2uYIaArbUhhGH\\\\\\\/JU8A+jx7JmWWnC++kJNSew++3a0SWE1sKpAV4t60LZ5YiM1Vy2385+AbmP99bN3FmGxNaGELSwXtPUaqrCoqI\\\\\\\/HsccFptV\\\\\\\/q06Ig+UO0nido3OUjrzdPQx+G46hNe1Rk6q7CWyTq2eg7+N4PUPay+xXicYPlsMQ+A\\\\\\\/o0OdpbEm0RtDf5yq7p2bXyVI\\\\\\\/0CgytsuJR1p\\\\\\\/YCf00iH\\\\\\\/yGevpHBmjMp42nMUekRbX1zqJ4QmJhGWroK80rvUCkpWcC4UuTxQnNZXrLpjIsuKQnzjx0Vg9T+rs\\\\\\\/06jdsgNrh3ikRx6z0\\\\\\\/M0dTjttX6yLWTMDIFgBpqKgTVsdJ8zoB4CGgf7ujWuPxdD\\\\\\\/OriDvOx5FXg1xGrvY\\\\\\\/NQgQJPJQVmwd1sCzTN0ptCWar7MTeyZtFczAEI33nX1OONtHQT65tdyFWQIvqH2VvDQg6ZigsWGErjl55m1Xaw80hhdAARu2cQp6qIa+CgKVpTazoFVHTTjqXuO6hD4I13OBwN8DjuuqtDFTWb68H7uZgyMNe+uGIgcxbf3adinqmu\\\\\\\/nF8Rtj\\\\\\\/H+OT9pcFQKZA5aqKuJl8KfrVFZas9OgwZPy2v3URpSrrKXx6sGS9p+NWtf7usxeP5wV2u0XDYKQOXZDqrGBWrqKnvSYMMynxOmi3S9N15T9aka7+ee6fEYMAegJjRCL5zgU4qyb6bAQfW91EnklnveJnODH\\\\\\\/XbiQZvjTlCpuoaDaT47FHrxg2mevdY6D1m8yjJTB9MCWCSq3X7I3txvmjD808XPziVnlVN59AG2ldEyB1\\\\\\\/mmeaA7mT\\\\\\\/zPS9kFfYzhpOhk79oJrJbT+8OJFDJD3GfpBV8dn\\\\\\\/T0guFwRjwCcKhexpy2HS5aMlCxB\\\\\\\/HnvJlYzokSFlzv2xODH83OuoHoPHoLuQcAyP6Or2GVYebNbRgsOviY2sMM9XLWvDkbRqAc+T1VP51XvfdbSUlGVSjdBMRfgfkBwBdA6FP8XTrmZki97vOg2rmd9VBxFgTQXYvK+f1pFrl83BDR1onr5C0TnS4Emf7zu4LbG1zEbb0M7tL5PF0s3sNpem9tJOyAP6NsfL\\\\\\\/vAmRPdNBlbmGOx66ISKIvhzLQuVtnXfqZwU69hvDCOF\\\\\\\/HaBSr0y0DiR+0bjbeJH+80kPAHhbz7QFZ0wnrGMBRr27+8L6Bw1gCDF70OKrbatwm1gHviI89i\\\\\\\/\\\\\\\/WnnYSsKgLDpL8DFdi6Ay31gre8E2OWjYm4xX8oIzguzMrsCnuyPVAmGiTcI42zJkzWm7KCpkIu87DhmnO854n4GXONJUmI3uX7anzqLAbuKpXq8RDSfdbLr+FNxbts\\\\\\\/iwPnbCk\\\\\\\/Ty7n0wR1CQWJFPkCfInVAquoHj9G3tkKLC8cz\\\\\\\/OiM1Mw1UUw86fPtst8+lBRAStzXmBfpfunJJp4dF1g5R9jarnoFVdDscRZEHtiV2VrC631u28TxxIkU2MdSol7y6+r7JBPY+ChaCGhYC9Zee39TloI\\\\\\\/6qOvgYXu9d2WAWXbfyz7mfw629eI\\\\\\\/4AFbOnt9G+8PvS6NXuNrTnXhqh\\\\\\\/XdvG3pXwWDV5phvmC\\\\\\\/Vi2+3RlUGbXZJ2CHaga51YQyGLe+6k6Ugg63fhZ45TjVPBwmnX\\\\\\\/fpkwSVf6Qi94gmNQtMo7XVJ0GZC5JslV+2WX3fi7NQ8hGb31nkqp75lkm9sNTylh5s1z+IMxcrwujvt0cnpSg67jvprr454QkWelDalOzww76cMnoMGu\\\\\\\/cWzA4xkR50ECIlVF0JwAhCPrusMXsHTTgdxVwi4sNWSSPxOnpmhmFihiOGgrxgUbNbMD+E\\\\\\\/jjRQpI9FSyHNS+pV0mmzlGOIilTY2Po4MufKI2EJmiR4PZ8EcwNQKloYZ6gFXDM6swH96EWtDgmpXeNkK4WfEXXQER2v3ShDTQC4kbekTxUzmsrTLI6SDzRZnaztOzZ09Yr+d3qpOf8tzgAYdNQWPDAtK6F9MNHafc6ZjAVQMXozjwU+dD9yP2S\\\\\\\/LhhFx9B2WsJgz9h5H9u0sPH3Kxss7hs5S\\\\\\\/Qga8RZQaE2R0OMcxR3sPXntVucoVD2EBOjRCy8uWTxDotHqGZ5MzOkvEzDMGXdnI\\\\\\\/sm3eYS38oJCaSRa0x1LcTE0ynM3ZOia8hgms7H3V3Xj2NCeSsTUKNBV3tMttLnlhOXYjHYvze+KxQjneSZviQb8t4XdL3JCP\\\\\\\/sF86yLCFfpGXZNneRcoVemvXsdBzhMRv1qJevJyD7StxNtVjzLJvV4L2+gKeWrmlV78N6kigVcEkaNWgBif+Db2emOuR63Gpv2G5ScTQqGpmrPr7B5n2hbJQ5GrxlVbrjQPVQVdHTfMBYXfcUKzzRVxzARo9waEukK+hyLoe+uPoFpsE77NL5SxGx\\\\\\\/KCO49Ujd34uXm8Ul4xxJWskS8y6PEJA7wYPlijhJtjba3aHe88CqcbL0J5iWkA8iJT0gYWIzXVyoxq3dFQ9lqOp5SZSrOq9gi9gmyk9ex2SC62gCqLKItupeuqucITuQWeX4VXj2cAhdkA5e3XDKIGy23KJDfhkQ1MvNut9wpMCCuFlWYRZ90DO8oVd3jrVh\\\\\\\/45kfD2KQdDwUXPyzKeeD3U81R1OKxgQtW4rTpwNnQlCBd0K+YkslZLR+66je4ArVhHw7wvoJwG3OsXvsTpjhDWlqTjEb8PnquXb2DdIs473r3prwcWNL5u8XP3LhN1v\\\\\\\/b0Num1VXTpJpIkTaCScz3n+KkRreeM0BMPvLR2Gcejjoy9zd6X6taKJzEWlGiZNT+HOG6xsXnj\\\\\\\/jHWqSR1B6qTz\\\\\\\/5Zmex+sl4syN\\\\\\\/fw21YPQ9K32IPU5yweGhFp03wD67yXslnUv6T+BqN9PHynbjVEDOSWlfTqY1q9VAi3TZDXyLqngfEiGzREKo+ru0ejwP98SrBa5cbPTkkGBL5ETuipHDWtNqcA35esBHlPv6Himo9M9AeCVhRLSm+BS2x3II59ehZP7W63y4300oKZve6pj8tcR0e57cEmaGuAjkWDepBcWnO0w2oA5bWeEpNKNFRrWPUAykjxoFQPFHNBF8sEpvSEm+7c4PscZIxX1LLh7qQebJ5iccJbvTefC0Ht78GDXO28CaE2Nn3mBBz5hOcIktDgPnJxs5KWVHRRTiysfuY9ux72lg1J9w\\\\\\\/bh8nTgOzaEOkzcgxC2yPybLr6Km0mK03atVLLS2RPhD2IJyFUwluIMrot7mKdMC3j9w7j\\\\\\\/+vXldR6axuAhpYljhI6oAwSFeG825Yk759E5IzxrBM\\\\\\\/r2PiXQn\\\\\\\/bBe2xi6ABvUW9XvcdV2HlZrJ9SCtnd23QMkHySS\\\\\\\/HyQNq0rHVAnzJ8V3xJFG0yr9z\\\\\\\/IUQGdauNTe7QwHWK29jze0ivUdWAGftR81ZgsBcHqL9lKO1Cl+6y6EllyJQBJjwq160T3ulb6KeMXTgQCL8OPs4VCVT0u53BBWqLLboNBog5Dki9Fb8fg87IP\\\\\\\/Avg9T8qxqLJDui61FMVlQ8WVWwsbL1cWWs2sm6B2QKUZuew8V7bFzM5lRTkJENOGM9GybFW0TsAhpC8RryVvvtrcysT0sTl2cK0pH2bMqncjCvp8\\\\\\\/DgQVlfcPAErpChPlgyNqjzGTxgWV2Rwqt0jrmN0RiZE9xxsNY9mRpfzcRx3kaj99XWrp9iOU0kv5Oc4XNqrBJQWcdkoQU76A4\\\\\\\/LOFxIXuw\\\\\\\/bW72ad2OfUxI24FTBGkjj7Ayr5hy2ntkwlN+h9lDwA\\\\\\\/sIl8YS6b68OZJ3yjNj1y79MctaSNCHCVF7SzAL5MlNuPU2npYOTABQ\\\\\\\/QJgvQiNwzYtm4G6ihH\\\\\\\/ew1DdBwc4pIv83Y21rCW8ko0IU2G1tQUl6zYGvzPmTwCyYoXquwOpJE0yh9zpocobMWPbSjXStEdH26kYKcEqK80\\\\\\\/QzuQDtIvKiB6rX9WkqiCbQkZOc03iKrUvBORc1t\\\\\\\/bRzvSqoUSgiAOJuNVCyB2KoVzOECV\\\\\\\/aP2Rr\\\\\\\/VtajBdMai8W4tb3DZnxQH5vF78I5xs2HsdfAkFxXzRxwVpXWO\\\\\\\/dWBqsv6OSUnfye8jdfrKvh7RDPzXr9Oy4BQoUGRuiFNc6VS1MjNo7ShVt0eYV0bhBDesM013fs1PDpfvwZ98TRWtdbgv7sijwZClra\\\\\\\/uH2obDO2rhVystqylFr5DrLCAxVEx2kvRHB9NMbJXjQn6BfuBaD2IgBklHPw25pmWGV+iX09oNPdrBUjX2odYyUiuGZHAOLaSplLszqZzcQ1QqxAxqZC5jQSY9V5EWzQy\\\\\\\/Ag4UFoU\\\\\\\/lGp9gbhQu14qMhpDb54u+tqFsxduEz9+z0KE9CNHoteRnGCzH1ZWY8TB\\\\\\\/lPO+uKjPZv9WsxnixeivpRu0cr6raIdQilR2y6lYavUixvWtwl7kQplj05PtCIF1umRWB9EozBKqdF3DlhJE+tYZXINidS17pS99kBt6wdmE\\\\\\\/WvIjEjD63b6mMsfe1FMOkxv0OZdCQ8sQ03Jz69DRFL2TUQfzFcNJHNnNErXZqkXrIlicGhNJNr+zrx4t4eeIniWlYKiu2QCegxOLRE6SpE21zHJkntQ0gHY1VMsrhDTq7sQ9eaxL6lHtFIWgGCgKbPYSBDYnOuaTFUkG4Hc0Ar4pR9YL7V9yeFGuc5WB9E1btv7hm+U\\\\\\\/nJGA3aSRYp6eiDzLJ4UnCGGN3KsZ8pAH0fmlrWTgI8RSGZ9esQehan7JOV4FmAWGYxPyupW7HMgjl3R7zu4pzf58S+amZqd5VeQ4J9oRZThrbiNhaXQqZMSfysAXxVNYh3NQvTr6Pg2BgbxTMM3QpJBLmYGsZDWrefq8FuEyucItH4Be9BWYbe5\\\\\\\/iZ6KSquPpBgmgq5CxspdNl3pUxF9tPBTPAqJAymbu+\\\\\\\/5Wa5GwA487AZgrvMCki\\\\\\\/84ovALnZniP4NVcHohtpBLMSYLFf6VME99HgcWKT3cGLpcW+hCAmCpIWVXz0vJVgqTmY45TFyIF8ogVI8jDBlTV6tGQkGK7\\\\\\\/t4Rt230U4i1zvC0PddTZLLN1XZaX8W6Zt36bbWDyZw+zOKZY5ovAuosImTlBfs9U4nxyf5V\\\\\\\/42DZSbuZDhG8EIJP1VcpWpiTglGbVAwQiCjZ27QosTx0GgsD3MYE1bTfWs+6zE7NtTuH0EG704mS6nSVRkeyX04dq+5hEU9hhJgAZlMJ4tUk\\\\\\\/l7yEpD4AD\\\\\\\/frLZvCEhVeu3VXFuvndzoE8F2UOG60GrkXbAdoojAZ7yXkulBXUMO0L5At757tTUj8k7aMyvexo\\\\\\\/F0Hv6v3uYcHgtBhHp3HnVJI+VxPCaOXtT1g2S61ks6gwnSOZv+HafX4DGO55XWCK3FVRUndim1qUPPFxo3fZ2uuhtfY0m52RWHpXEgh8flL0L2CEQfgqjuepNsjgZiO7HWC90HtftFIgaB0AOvUIyro5ke1WTcy37XIo3g76rmKQ0FrLvsqUcj7I4w7Ic8IVgfh8PYlQH3KW\\\\\\\/e6VXHe2Bpuo5J+jyRqHoet2fqry0LHhtLvU\\\\\\\/C+2vq3L7NzW5olsZIF6VIxm3uCc983gtuGqBEd35kB2dzGfOi2iRlWoO\\\\\\\/3hvlr8V7mPLWwrFWel3Gw2QGIelIS3df5gV+7ugUHfL9F4GUrCGCDpSWFNEHSP5hPo0htBVCGeTsORkbBo0oHqoUcadmx0uirrFf+ICSZj60l+2aZBDTyPjXlHRJclDiH70XDpivNGTVQZXBNMuHbRA9AXW+WmydmSJqWO4f3A8DcIQUWvwKNJieTLuC0mr46OewWYPpedBA42EUPCVvy2l4n9Bv+NbbcpNEoCgK7xhoX8WDizFwuXGO1P+\\\\\\\/\\\\\\\/ks104kYakVKgRzwSmq0\\\\\\\/vL88ktRN3k3yoGYFgOSscS3mZBo70XpHJMqeJmQ0NnchJYOyeRTiOdEnnmIJct327GIweeu6yQxImlDO\\\\\\\/fi\\\\\\\/umjhL05DtNkG7ahJcwiEqPehtRwC0DoaycKLNTS+xvc7HpywisZXM8xohFLr2ihfIWpBkVemmY1wpBc0puGA9X8hvvWn8AEd8li7X1e7q+WVWcA80i2yiUemDjuI4XfP0ABmCgBl4wxOFsTDqTV70Y1qsKgbt5q8rWucGfdw8q+QD5DuSbbdW6bOvph3XbQfCS\\\\\\\/9hYaPHZI1g\\\\\\\/4eT6O85Am4b95BxE5JxVrD5H8PCPTkj9it+j1C8lLuckW1sVol8BzPbnWn0dacSrD523eBCp1PxiQ\\\\\\\/ydc832rCxC2TYLoReqgfiFCDdW8vKDk\\\\\\\/vxdfDvl\\\\\\\/Ln\\\\\\\/qJfYXBSsL4RK1Z8XjnO6q48NOWTZ9xXahXpCYYC7QiofrtAPBdOxxOQLvNMRm4EQnxILWJn6Tijr3QMNfngB6FHwCIKlxm\\\\\\\/OhyMd6hqCq4Z+CyAu1zuou3T+wmYI4je8B2cK1al+gzmZuxBGSI2Cpl8ivgwinZJKgt1vsHcHF1OmH6\\\\\\\/W3Lqqmoqfj54z24c3J5DECtKLtAdNBOIsGF\\\\\\\/obXAHtNABhacSeS6PCH721ETGIuXLh0mGGrHpFREO2fTtDB8Jqzqub3YubAuquW1xTe4KZyaTby93Fk3P2s9Ihp0V8UM\\\\\\\/TA0QUACP5eu3VFpWDZA42PjVSyy8m4\\\\\\\/bZfGxlaX0vZP2dZF8W6i6VpODsTQPHOvBugKVeMFrDWfRJ\\\\\\\/kMCVOtRfqTG+6Y1FMMReoWksRVK6qgJXiSA63Uqyelq4\\\\\\\/fF1emkuWY2bQymhDDOnT9ZmLPL6fezva9yxdEiUz7uK8IY0Ad8OTXELwzdhr5951k6cEXA+9z2+IGMdyvq\\\\\\\/r03L9P1rNfyPWfITV0eRs8w0Pl3RHVCyHgZPzQPmWWiRh+GzG9G7uintNzUvi7rOOe+nuurNKDRRryf2dQOk6NI5IEUxtWuLyiGqV+Nzp+iU3T0Nu9dVtecZxCwMFYpe0H5yH3LNWWyihDlI\\\\\\\/O4i9xDiPnfgL6LYffb0GvzsS+I614HQ9gUC5HHSucKV4eowcB7czvU0eaJfGbdP+ZCL1wnqx55VU+iDujvgsKdBion59RuV3dd6v8nFr8MXb4ImTzRU1wRGUe3eHKi8S+d2KT4VYUjVYGcR0P2Btx8MzeyaGQYrPMMnjp8IcB2KIbIRde1415vNA8Ic+I3P7afot+l6Q9y3LpCvCnP++yNwQNYqbuKFBtGv1rfm2FcEAnp9UQlVaEmYp\\\\\\\/ybeupUnwacbLMZyP1Q5OVuPSgA3iFh5ZepiimOUkphZTKDz2XJuEAuCS4+5sZylBN0L\\\\\\\/wVBMgZWy2dXpO5Utu8DznvFUeMK\\\\\\\/Tvlc6sQgGMd8YLAJyejYbBqiTFqhjqhr46CGGA+SzvQoR2aQAWCV3pgA\\\\\\\/CGxFwQvcfOKYVYpNR2v5jgDfY5DJXzkoRx6hk+GsWa7p17FYmiRbm8WY1ZfRBB0vzx261rUhL4pHvQqEi3I8HQRIiO4Srrigj2miSE2uVi2yq2IbUA40+MusLA5zV4bG8Xr4GKoGzvbWWz5\\\\\\\/IlHCrJoEMnlzv1GARc4x6sJPPgryuoSIPKHKR+I4Qx2ZMyqkBlU4EQBT9JFt02qJ8FQuV0fc6Y+Rz2MB4mWmR+yw4fr3CEz\\\\\\\/0jba1NOQy6o1PWxdO9HfO72NXjGmc6v0izBTRXzdnstPJ7WUJKUuwG5KX7YCV99aDA0OqqsyyEk3nzoK\\\\\\\/4v5ftTDoYfBYENWUzCct+IiXCToEy3v+mkfEjXy3FFl4c211NFmXjWyHXl+gD6xdz7PUTSnZci2VA+TsvhGRbAcoWz8u+2NMSCPr+maLNnf6BB7qxgaP8mP+sroHnZ5Izy5tpQ+EuOLtbjsdvpoNExrcKL2EN2iGLRKFbQrby0GTkfoI\\\\\\\/EXFjjci68yVf5dxdUF7BAdnu1kGsTUYEiXF4IR4DMwZGr0KGwGptM9xe8wp+ja4N\\\\\\\/wlC06ZBGLt+pUthXg50Zl8qyJlvU3133MGVSLxXuKBwVHvykb7+xUoRlB21S3G+Zf5+\\\\\\\/\\\\\\\/xK80F8gs9xS9hQZV8MadBOu89yOM8wQK8DuBdYfTqbLwyAc4bPt6S1fZ4PBoJ\\\\\\\/X\\\\\\\/RF9a\\\\\\\/UEHZbC4MtXRDNSK1nrSiAAhFQO8pN0vNPcFz\\\\\\\/oIt\\\\\\\/\\\\\\\/E5\\\\\\\/j2lSsobwg6MuOf+VsqFqlX2J6FA8e5zjBatKM9gwjnIwbU\\\\\\\/VCY2xydwaAAHul0XMMrYk0OeL51xZ8EyqihlJIZQ4CS8a7aEEQpXIWCKxsEtu\\\\\\\/aJGT\\\\\\\/1P+meM+DJHJc8bKXuElY2SuPe0huMuS3J28pKqbozbBuFYA5sutPRq7KLUeTeJVyqd90jzcfljAoISj6cuBIkHXIXk5bYE1Gt64Pi+u6IwV5N2zpSORB3hHSn5NP4v5sjfiNDDgJx68\\\\\\\/PWqTMgSSra8AyKEEkTemzq4UAd40p5urioiGdjeIHYkoKL2Uax32lFeZ2emzVCJ2qm+IW5yqPP\\\\\\\/aZ9jbtXM8YJwse9BFWR3+MkCL4cDfKd7v4P1mmrA891T9HhjF6oPEK8zbZQ\\\\\\\/aOwdlnDjRub2yuU7kQidoqV1hnMz5k3alWWDwIu7Z8aqKLLHntsNxuamEOOtWS1ZstEs3Mbav0XYwU2Ldg7v08cg0tyhJnmHS8zpi+ePbRxrNYb7jS+uGu2DJ4r8WwSTTdFGky1U6dLd6DgN7lL35lXG3AeqVQBTyDh3KAC\\\\\\\/DyMmJmnCV8zZ171Lx\\\\\\\/ZoXh0jl7G0ZcEwAYdOTxZyQ\\\\\\\/CzxF8j1UIFAinThiACLwsMUY0q\\\\\\\/gnI3yeTWMdlLbC\\\\\\\/h7Eeevv2j0NpE5Y54MVdlX+42dLDhXBQdi8c2YMDwGTw7gB83jvQk8a4Ck6vWhxe9KZgCP+NRNQjl72Ll9Xc0x3HxiedGrhMNhyg1fB36K4PBfQ\\\\\\\/EL5qs1GcFNplCV3VdG0atfAHDCEisKT\\\\\\\/neD4AxSVFMigXblHdgfZi+\\\\\\\/V7JZ7aUzsRaDsHuF68w+XamfQaKw5WocLBtC+kxF0XfSsnJ7kTs2ImBI0w9qsoo5udFynQdn+CYYsp9NgtWs6aBcL5dBJW1ATeJZllCAApqA8tL1n9EWi6OoGChIaUBukmP2w7w+QiPisKt1Y8r5Gx0P8KoHr3350gC9Wo3FG2dMz86hiCNBxcTRfi9WQjXZBiBKtcFDFGRFC4Ic2sexNHLKMpmrxR+4fxTPWKDoWmDQO\\\\\\\/s2m4GFX9cEX5nHIbgeL7XSAlpI03xnTb0HSYFociblOyVnzs4M9rK\\\\\\\/g8og+8r5Pa1LS5gt8HvoxLyQlnUYBS3Z8eE53jPanO22yFGRF6X2MnhbyKY\\\\\\\/Oxm1fidMJOFjt0NkVKH85IX3E69ONOzQf3HVNejbJR+hriUoPn1aaq0W3ZTOUIy5dgdpgeegGgmyrAD8Dn1mvlWdKir6hHrCd5SGCZuhnT7vXYyxBYITTcnnzR157oYxD4UzRS\\\\\\\/c7x3V60n\\\\\\\/Mm\\\\\\\/uULXa\\\\\\\/6sdLrxtPiQSgjjYH2iuPrpr+JxUNPgEwdZcOr7+ccHKE33OGmxRmUJ7AYlvOS+XOQGORNC3HppzULaYSXOkvjrdeEpDH0iYBr0cBGxmM1cXnv9oIj2FBCtXBBeoIaaAozIdPSWsYYVmhXUMjvmNljrsOO4479cU4\\\\\\\/FBj9MQch0d8VGaTZtjFVwXHfdvdqY5GmeP2uYDjFudcUnRqlKbPppl38mjVCnqHBXcbMGtSIDTb9+YC5WPNtNG\\\\\\\/b9r7d7gPKU71GaiQulsiAfxGQIyqGU\\\\\\\/NPff\\\\\\\/7NdDmYiYZtXdWb4Kw8j7m\\\\\\\/NDVkjJOi9Dl3AkUNwfn4WntXEUppwD+HJz6vsWf+eyOyCMIi59I8otpriAO+cVwXE4CrzQ9HdvZLvxWDIaRu2hxRqWhH2ptAyCvE0d7uqYDQNv\\\\\\\/CBbKMuUobH2hAdLSBTYsvYr7PxFMLVy58yw\\\\\\\/CeP642w4kHgkbultRhpR\\\\\\\/ab51mSsgcETls5ts4cYapscSGKCGnmETzr6048FUo90STAvCcbkNPBduow8gd7YfZPECBwQ4LHhBtLWLAJ6qRAtPUXqgsJaoILHSGtjDeRLXiKKIKUPj\\\\\\\/91aNL9R5rOxNO0rR3XeZ6eyy9uWfunS91PMjlfPozOnPkY4OX3A8O+18Tnl80AudX6nWngb7GdJ4RocMUFUEo5m44jnYafhsXaZJzlnhuba7hk794d6SojG3Om9dtaALb+pmGdhaZqq3wiioOLX87QfgpvxMnMmall8Gn2qkBO4BkUUP04ssblBS9t7IRh1ion6Azo8nD\\\\\\\/1bbqv0wVCV5pgtAUKXPUNVxUQEsZ5w5GzPVFgW7FwBylwo7zVqLt9cSH\\\\\\\/IBQ8feHi69GiaeoprRNhq+rIGvthG3nJlwZ0cBvMt2mW7g2roTsuYvcFnktTPjcHuPg4g7DFN9\\\\\\\/IsIvv1HcfWMhJnRu+lza7X7aBdKkMJ\\\\\\\/4KKyG1Hc4HQHf76IMvUZOTScxQ\\\\\\\/mxlP7x7bylSouFCONZE77UHfL7cgSC6O++2mgIcmF9MrTnmWwIRXGq2qIg8xBEELWIuseDEPvifjopf8XW83xHkaFRs7JMfuKCY8kfy5d42I1n5PAgoYevUcg8CRblDn28rp\\\\\\\/ddbzwx2v2r07vORMeiztes\\\\\\\/l+6gBZuITfezflRytzzksVuUAFxDyS5sVpLb7bKf6S32JRgEOYhIxDV7noQi+rTO6Y2IHJ0mFuFRhW\\\\\\\/UxBAMhBDp633MkYZxNmGZ818TPwSlW6KY3MuA10zLORtWp5j3sVKseouxjzJUyJ9hKt2koYkIPshvqfRpmqmqiqKG1BT2culyhC7Z4FwNZWiYO99+mrJEAtTNuYb15ZIuH19Wy7qHNaRUI+4asB2UE5bHkey2ngrwen7CTD7kU0MP1b7bFX16E6veUYKYmvN2\\\\\\\/h5T8LrzX5V7Yfiz\\\\\\\/T0AjBjPEemr5acJYbzdiOSMRx7+rF2bO9GfR5\\\\\\\/98knsVmkzkQsEDGox3bVBkVJcbaFxSs0HG2WO9kxD7FKVA1N8YoitSt0VWH8umxSC6gGCqc7x905GsU9H242uqqFob1mf4JDKDJEMSebm5ax6WmRX+lk7KTW4HBLGfeIy+GzF6PI0ls1HYAIb2VvyHDpp2YyRXybAU\\\\\\\/LpLvoE2sYN1EYU7KnzOALU8g+1EK\\\\\\\/S5tyUZTKwzvO5zR7ZH+drvolOtystNoZxcoMbsRW\\\\\\\/yj1gNOg2QqnGqsB5LVDDOmBE16a95t8XXWFF0JC\\\\\\\/ZHKwNoVrhpgSN+GjAvlSi14tX8iqCYi0Ufd2PKOcWdsHaJnVHicT+GCCo5r+6iljHvcJloATu1kuCgGjy9jAh4ZsXmEKJMdn9VslEHbSca6FFdLWxWutbYv4GY4UKd6257Q9YO94WIvSMowDEODkJ6l7s12+AImryEe+8ctLIdrONi051AEhf1jvkoTz9Wsfr0rjvORpW4GswvjRWbldOHuMGsjiJ2RI5kGO1\\\\\\\/fohmVjFuepHBg6MQBc1s48MXhx2vSskDQEBwJaz2Obf0+KC9EL6bdrP7TTSmQs5mYPDOSJT7SkM0ByUXOPAPHRodIR6KUeJtWp51Y1Emi+TAiN3F9votoaxNVMI792u33bbQDFR9Ld5akxEbXKg5589GXFlsii4p3amuCWCKsyVZ6BiOyNoaC5CYa\\\\\\\/sehX8AkSH\\\\\\\/ywcwjbPPWl1Qllonnvf8nANljbc+nZoFw4xI5u6KSUckD2rU5dfyGEwNwY8kiOByFcmvfAXx5HrlZRx163cxlL7KlutLRFBRH1WytFGLV5wg6lHMenp7iH8ZG4xDM\\\\\\\/xkrTsk0mpSzCvqo+HTUAqGxLjxdtW\\\\\\\/YiYCFk9OE0+DUP6SJljJBOthNLNPUGILJF6IhDnTQa3VbXd0Nqd1BZ39yEwWjh4v33Vl1Bvbx4u+sg9k7ah9HF\\\\\\\/OHnK3ypWYCqEXkL8kOqkyoPtm2iAkSF+4PMkTjIQGqV\\\\\\\/jQjLR19roFzt9DB606PP3oL\\\\\\\/JZYbaNAHJkoWZNsSe9iuVGzXWnP1XVcSAZD2u2X42iZMaWjUDU2CHArt1xKPXE\\\\\\\/Rfos5ihpV2PELZwwjR\\\\\\\/4dm5gtcpJzWh1KUZ\\\\\\\/uxFbfCcv\\\\\\\/8JhUYAeRj\\\\\\\/d2Gn1b399cw2PRV1xJwP0nLZ1ZbWb5oQhRxTvXDS\\\\\\\/j\\\\\\\/5u467NXuyaTdYEFKPALW8\\\\\\\/eyZMhwbdpQcn1r\\\\\\\/LZIi\\\\\\\/45MOkVRGUm33BxMBjLEL\\\\\\\/RS1xcEfxTCK2wnlYxS\\\\\\\/uBBcmEj9i\\\\\\\/Z4yGNVoYUW4TXs9I9d32ZMzKll0bgCFnOBHrxEkBrhehItdRB7BpwkucpE9xZZ+f0NmsNNQmvlH0P+\\\\\\\/xy4ukdzt1\\\\\\\/7D2Sy3nhzFj+xTujn+pFLBqvz5xVTeKYhqnQDJwch15BSv\\\\\\\/FWdeCVnvVTUGh0iHw8QE2FgG0xVpLtoK0uVXRJWwcICEzIRPfSTMeSK05Fm0CO8NfhZ+WFEzHqsQze8esamTT\\\\\\\/l2N4PIduqnv1XyGGVRsRlxtTJ+KZ59\\\\\\\/SltK5Z19K3Q4qV4TX3AMOwiT6JigvvHKEeTEZDwcmea1QOWGtCE05FHS9ar7r\\\\\\\/mFH+NqLF9Hnjy80+PkdPWtsY07feq5\\\\\\\/mrs4ZSoAd+GCv7AUKivJKwq8BWxFWWQpKxCp8s4GZIsMGzFA5h+5TQtwf25tkoE16Pu16Dk8+whJ3buwYXaOsnZo9Q4f+ohGmN2WzVLuPoxQOuvfdJMFO+AIuh+gCrW6XaE4CmKEFnBFyclQ1xTYPMWgncMakLIvKzCfFWSf6vRVMPclslM8h6n\\\\\\\/p0USO++vZkX7hlAfbUCqI+riBzj9elp8wyd9LPUjBGld659CJHfyrllhN2E1puS4WoQE+fmfuqUW4sHh0nDByz8Y+YTBFfwG+qZ0ox6nFnUAKVLwmR\\\\\\\/3ni0VhJAvWTTolBIw7Gs1+e+WBlJ73v2Tx13cn7aABiFsK72V5d3DY4j3ZUzQT8uNmUvJnxz+WtEOg43+WNsM9jrPSqLxrVi+UF6dNCq16St+Uqq9WqoWcK65YPWZxadMvezc22p+dL8W\\\\\\\/0BE8xuLGUBW82CwWoiIY+4FtQwepTdlZyxDDTiCnuN6TfeyVqBZKT6O9\\\\\\\/tqZ80haWPKGNtBnkAOGdr4+I8r310g8rCFYygPEALH4IQvPDPttYXHGa7jfUcHei0oS94oyKsULRVCoHkL02PyMd24d0Oqoz4f\\\\\\\/LpGxTN4NVc2ChVh0lbRHObQJmZCTplPMK+3dL0nLyZQbmerar81H+eDA3rh6E4bXDz0DyYvQ8BF0TjClIEZQlBUsQDlkA2PyHKbo8fDdABRtGqA30XKMI\\\\\\\/ZeqhIubBf\\\\\\\/aHO7bgx8sD00lwweqHs9wJQvDlew8dozOS4DXj9Invqs12ga4ES7U\\\\\\\/MXhrDtjvnU78iucw3mUy\\\\\\\/YfC2i69BO1GT2SSej8zQ2X7ka0GvJeYHNrH\\\\\\\/bMG0ZYRRZVQAEeYuZn917clXq388P1eeo016riwCz4fGvIkb\\\\\\\/KNGQILrVkNuOT+lrtUj4iKGqi+6tRIPRngHE3en27u\\\\\\\/lf1FqSeuGTtrgiS4Xhinu\\\\\\\/MeTj0GhksL+PVCy176LQyf7PwEBeyRAGOFpI70g5QvKQ8u6Gjn1F4k2qC6x8sH3wmuqbvTLTHEmE\\\\\\\/FK2HjxK\\\\\\\/cZswx+jgxdAvB+6SfgiCDGm1IAJwFc5Ai4EbVq1NAc0GczM1B9nE5nk+mV3p7RRegZGnd3FB\\\\\\\/9LbPK5o9s4C9Uen7f0Gl0Yfwplf2iY7C7i1Ittnt8rZK2HBhp760l9aX3oijP0F558qySlFpZoVTCvzdSacwqgFHI2fhK6A6ecD84sdj3qV4YvntHS+ETqUCmDYwj4FJ4w0wY2xk3Qc01W5uLTAXbFVKZm6eFby199bZDxtoqEmsyWRzjJyVMJoW4Esmq8p+id1XrhOicw0fn0bK8LNX7LCP48qfADsVdFm8omP5ChCuadjG6usOHCaff9Qa1U7JrqIRl3z51uB9XfR9wnQTT\\\\\\\/PJxD\\\\\\\/D7QW5KrpTlKFfJ3iHKEY4TlH6gKpCkSbjV8vcKF5Sa5m5WIuKF9B8DMl9ah4849GnAIPRWe1uNqE+xUZhyR41diP6Ip5\\\\\\\/OnNNa2WI\\\\\\\/ghJX87AjEndELm0WDL4kDMzY5mTgqB3Vd0G3khMJYFwADLJOR+ZRcbJ14QKGU2NXdhL4LdI1yeJM7zMr21WPLIQc9ZP90MMfh6\\\\\\\/rI0sLDcc5kJPR16Tb7rf9wPUTSc+futhEXapDhYGQ\\\\\\\/l3+9r6Qi1ZiMVbxPI\\\\\\\/U8E07E5Qfby1sxbP\\\\\\\/unPnIFpkBeHeJo4+miyiO1XOyaupQRH0faO0HMFruJCi\\\\\\\/HPrjVmmEizSE5Zown0UBfFIRyzHbTntk\\\\\\\/bm\\\\\\\/qEz8+n7Tp51Y7Q\\\\\\\/zQ\\\\\\\/3yZGZnGu+8RG62\\\\\\\/TP0t3APhn5hV+q6Z1WCcMMukdt8ecCN+0SVfhALodjWQF\\\\\\\/iPcioxFz0VUyXbCn+c8Phed6zlEmQop6pq5GQ5xK9Wdg+\\\\\\\/wZWTmabfQDFZss3P4lR26xIRKAtPZUIUnENPLI5t2hSDYJ6IW8NF7fRZ+U+1Zf4qdqMlm9DREAO9TTGv3tufl6x6tCLAdabsYiy4gkGs3QZ\\\\\\\/Y6ID8pjZ4ZA6wtn9MNAdFkywrrTkQc4rzZoEbyKufnnlajVeDUiasKhSZtKmyiE38MFNwilrOwxbFBcaetrxEV\\\\\\\/5vlwpbb5gSZQNAJInai\\\\\\\/PT2EdU9S+nNJPU9T32okK6UK6m+xtszTVjzAZ\\\\\\\/kzIUJXU2bSrLQ93YnGo8MqzOxVEyYLD2izGoXmLZxFXL3PZaZLQ\\\\\\\/ZKRAPhc3+fFkfTTRJ\\\\\\\/np7TsiK1TBzDxFrTvf8Q\\\\\\\/FhVplfW8WwFoGNJBEIxj7lYMdcNq3leYyb8sWeoTP6epfbSRvagEEWKQ0IQH3LLEU6k2I119zyGQlhJ76mtTM6U2qAERbnGW4ft7GJAIln173aAHnfnhhA0E\\\\\\\/DkW33dqm3Ur5UPsQJEP1vhe5R9uVFVm3RkxuTw6Gbp6RCc7mA8c+8rCESSQxWscKUK9hilGmudr+FUwmnipsDzll\\\\\\\/kuvWBbbpZet3+Lca\\\\\\\/wXiHpdvqtP9AXW+pXPH8dc+RznLt41735+S0F50wU8w2QtfraCthzsvF5xVZSojjm38IH+gUJENP1B\\\\\\\/Nd4+PyJP07v0rtTEIkecD5asBw3mQHu5e6ylz28QifaBmKQFGXn4uN2GBb\\\\\\\/P0+ueuGcR3oUcUrkowuzKsphJpKJvcT0K+VTlxIy7jJHFl1Ln2WKcD6xIPgmgaX3wO+0tly9KFsaZw0yFJQjYqnh6d1UX+SgEJozduKWeNyUjXS1xTGDssXhdZW9hQtvatdsyEKnapq7hR9rWcxBDXX6neCCmQhtwTn2UV3ypkMOoq9PhCx\\\\\\\/atMM7yCmdWqcLmy1zWCW9M7f7akeTeZW7WvObbBt8feiuJpXDz6mC2+OsSm0Z8plYjepa2Jlm0Rdyw9j1QTSfM8A7EKJDi4J++ZE+ootiaZT8kDzs9r\\\\\\\/xhVPZEhZv+4hkiwyfIlfgEXtPUXSHPJ0QQESGAlUwTdofX6P8RxP4nKHBur4XYBW350UgQT1I0ANMNrl3h4EwrSMAzfRLES03nqRunqx8hoqC0ZL68U1e+gKIoQx6x5UIQ55OIZoqTO19WuCMxYX1q4Qh0ApG0bU\\\\\\\/wf5v63WcB1wIf39gJpJTxGDbttiAZzHJ2XkmeYA7KsMeImimb0FJsC9zpENOAC\\\\\\\/GyACjRli9f9VTzyqotJkexko3GhMGQofi7McUR09Gv86q9VXlsB4Q\\\\\\\/36geyO97pqKLA658GwzXNGY\\\\\\\/E3GJpxNUp+6JbgIicYBS16VZgkbpN\\\\\\\/0NQdqnnS+cOVJ5M4g0r+hlAF3mByn1YTGa1Y3W6UGXRoBY6LPxAUyHe1TTM3o\\\\\\\/BTY8CRJUVcPLcdX\\\\\\\/y7gbeFNIjSc3KwTBSaeF0DCu+e0l9t04KD07SXUtkLZjFP+HMl0vF\\\\\\\/PasDdMqANGRmiK5g0dxtZCV7Lris\\\\\\\/rYdjhoM16vfUqjcy3aevHJ7aMPan2VEuyGUhqjF65VrqhN2+zmV8UBWbTmGWNBd4dyndlBqcG6qUfINKCRk6UcJWSr6Ylq8i4QQ0W5RyNFJGmqni4t4twkJAuRbju2vhNQCyAoy0+FN7bmHAGaNo7Kbxty1pNGVaggm8Ol+8\\\\\\\/5P9W\\\\\\\/Zk3WsGBMMOhNrgJlMwNDUPQygSGqOzqyOZ85tGvS4BszdgXWlf04oCNpMAmG1JsXcJERhBftyj7z0lz6g3BlICPCcaXR37qw80KioaGoyUTE2W3c3+fEZziH9aFUgP4u7AieEoUOQmEc6hNnfiQjnvTDTInaQUdoIT6BaiEh\\\\\\\/\\\\\\\/gcejww3N8Gjjp6\\\\\\\/q0EZhRMiRDcrKg+3dTaBWXIqI0goPoZs\\\\\\\/aSrQh0x2A75xO8Z+xeUVQv6eQ8PgqfbNwUtFOW56PSvnVe\\\\\\\/gw18TNvSIWR7PmNj+I1OWqN5cq+9usDBPipiWjQt9kk2N6E9zr+3gtHVI2yDkocFKKMXTHyWY3x2T7B+8XasD1IhjAWZH6F2wtvcV4nf028jdqbhs7n1IkLlJDsS0nmogh1APgh2rvj\\\\\\\/ovj7TXpHNax73l03DlXnweelAWae2YUQyvPwmcdngunnXXaAMmv7aDmzqbokzS7XTzjZTsSyYzv1R5xI\\\\\\\/8K8tOZMXwRnyC7zfk\\\\\\\/Mqg7X9FZK3Tq\\\\\\\/6h6BSXcnrNni86l1\\\\\\\/qqyCxrBA6NqLPGVMaCT8VAR7QmxDczR56Ef\\\\\\\/mMtefTzAJPYvBgAOpRCwI+ss4Qc7hDi4rpLkvD1gCnKpgcDFDs466P2IhVuu3jpDwvhl0BeFxk4Q7Ce1TZ\\\\\\\/hlIr9vepmGVZNDyYYMWdJHvCIub\\\\\\\/V\\\\\\\/FfukcvyWS5ysvhu6IcRJXr6+EqM+UTLSgxTKNAiQQ\\\\\\\/UPMTAKd4ot32JhEv9Cuss0\\\\\\\/Rxv8\\\\\\\/KtbJ7TrTyqaJQuIX0tw0VW8kaMRA\\\\\\\/YRQ09BK7GEjIUftDQTCdY9z8Knqau5A7OEBj6F8q4r6q\\\\\\\/MZhuGi1BwjWC9YIB2afqfhjQY07JEjjsEjeNnCaI4+AfWZx9C6cgMuNGU10HVR2BML6gFCcYKN\\\\\\\/RUFACeXrxCIJs52\\\\\\\/6+dOGgwpzl9\\\\\\\/7WGxTJNIxewxt1UdeT29BtRgvkix04G6WiejgGmAS4PmIk17S+ygYsakfrHVEDMF8kJEZWlFqQIWW8Piao+96x2HsNL\\\\\\\/XXtqy7hi4zJGZQdN3Pi8w8nyfjyeczxrGoPTRgIRJlkssVSLP9Tx2xm0YajmGiYEcMrm3I7xKS0h2SGhmsY9p18DctKdHqSNRvnDYhAdHu\\\\\\\/JwE0ZI9S5uWM69lANy8jY5LXIEycHUFHXOCwdVU05wJrA1YG66\\\\\\\/4lZ7k7t\\\\\\\/Cn\\\\\\\/GQS6pEq5wxhG\\\\\\\/8HcXPe6GPtBOjC\\\\\\\/+MrYWrNXvYAusdS4UHpd3TNvaqf+VxK8mmys6mz\\\\\\\/W4FjRlNLmP\\\\\\\/Bj5+MdlzpLXRmBRqry66b5QP9gVagSKXL7yP8ZuhK4bB\\\\\\\/CRvoEu5c+5TDE4qfZPGvDE2WhaYZtoszr6T8X\\\\\\\/tkwJuHGK1Q6rqEfz9IAwBZ3TV2TifoOPachGXis6RBHRk7mVjDoOfWnTZr8VRizZP2HnKqkIFDyiK5omkLJwx4CPiwoYEkkqeVgAYiNW91M02KTN0gfQkhLwgkAxrYEaFz4DsVRJZSKnZIXWhyrKoqboe3r3xs+YaEAxv9qhAQXj9WaA5RRDsOjgbnHGXm0JcStDGQ553jngMjefP6wgRucJ\\\\\\\/bd\\\\\\\/HvxZ8wgPXdjrReiqWDc6IVqmqyBpjrGpRlFwwbTHDXJSZ2\\\\\\\/QGexysm3dsS9vl+pV2iT1WcQqzNzTgGh5empHAnCOTCIzTXRL3rDPtpLFSpVAlhsLGV\\\\\\\/HgczZhEqXN6RSs64t\\\\\\\/XTx\\\\\\\/SGJALQyDK1BWzmJfKaV5sqLuiPHTWFhbt++Fuj0HDNndTKNZbMUUAxh+r+0tdvMaT3ZPBgXFIrqD8vtX59s9Z05SLrmWYhpgNdybc+0lJkznms\\\\\\\/AAHQHw3bfDhIgDT9SyZCNKKrXEd1y3KlRlZC6O6BmiAMhjezfWm0LipMWc6HQjj\\\\\\\/2IfiOPuWlxN+nSuDEWEIG6eEYyMXoGNA0sbnTFHGF96+hesqYuYci\\\\\\\/udZB4tUoXEqhzXYZJ6mOR65jSlMZLrpxXYME+BA6ZPFzqnvIPdaNyP9cqWTpLaPxFB5kDWSb3wAqxuVyWiw73PCa9sjFY7543DcBgDeYUFX\\\\\\\/qkwVnEbiCSR0cfujcQr5aOOVsyi4Z3A5tE1ee7650ikanugLB0F2Zah2VKIe96gdDjQlmytpZwVnwZkVwJL3P\\\\\\\/B3+Nd9uY\\\\\\\/m5+4+HnxsjomybFOi5Qvg1Dh6z2J9ZaIeB1QiFBHMAR+Y2T7A4PeMmw1PTNsC+m9qaG4LAxwJ7YOlkVzPRa3l9VXspEd\\\\\\\/BbWYm+um+AqlP3Ta8V3XccNh6h4E9bBStDqZHEE5N4WBsxP5Lg1oa3PPLsthKg22ACI8hNXjMLvyniU97H2yeSbDrffDAo5t1IMaDGoQA4dmakbLBWw+IOoNeUJOHsK8xRlkwvjV\\\\\\\/+DoyzXQm\\\\\\\/B93ogOiDsEK4z5WeATPpRWQfrxMatpONqVnc\\\\\\\/oOrWXKv0o9ySq5hIIlfEHN4fX+mQbLCPoHMCIoAAKrWqEzLjH5g95g1\\\\\\\/z9+twszOTI75MW46PlkU\\\\\\\/NzMb0KrL8kBiGg\\\\\\\/pkWZMgeO7ilRbPMwNWceUzm9zS45mGRhImSaJVHDrJ1hBlZtQstTynWB\\\\\\\/0tWQx8vBXllQlUAxwA6n43XOcML5pGWg8VKWG29pJ7oz8sTjHh4WuV8SoCwCy\\\\\\\/sNenn2pwtRPwg1Le8dYpMCEjJ6oTapBoHgrD0RtAsJZ3j6meUiVeGC7DYE5x2d7LO8rHchuYbWPy+30kSmU67hvbgGDC1+zOHqWvvARD3p74RQ0PevA8DbQObG1HuKpxaJpm\\\\\\\/jefgxC1+5IsqVExRmUx\\\\\\\/EKmh3\\\\\\\/SigWAhhvG42MG+M+\\\\\\\/XU94J7bvcufJg0JwlwqYgZ0eNO4u8W8Tb+A1gBK0DvStHGCfeJVC8z7fZCbVonR3n1uzUO88d4rG+bpYFDp95dnBG4+zNf\\\\\\\/VdPkJA3KNIVX1Juuihgz95KhA2Pv\\\\\\\/YciIGfizmp3OXiQmt9oQhzGXCvkRvs6uO2fF8G8n7Az3x6oXFnlCsUsyyjIDqdkRGwgOUlN0FJ5UEF1XVVgCniMG0DsklYexSq0t1YeyLnd9xx5OK3UTbfBRzo104Ma1kTt8KaYNaYeIA0pgzpktZqXJfHFVQFEE9cRePmfnPRtWErIpOV+YUsWN8ATBr6ivW47ZfBYLIOKJo+yuR9MZOW+5eZlS6RRbYcZkmvMQFaRag5TsiOaZjDufYAkUtYewUdokw9WaZoRpx6dqd7XOIrXU42mVSGP7N1e8jhE09TrA3eC7KpWIYBxSqGI0Uz2GV4jPMNBveXL\\\\\\\/8b6XY0pE3m6JSV9jvrRq\\\\\\\/5ibF1VU7vGAfFL9SNg1+HQy2zmkLOvq\\\\\\\/1k79DQERVciMxNKaofmviSbq+HFg111d05w8A4iROtRBTBrizBuw5XjvtA15nhJN2O0rHftNqPsKwd0ZJNLk4WCi3eXipgmiQ7XuNWuoVuXrSkYu0z9seSN6saV1a3cND9WcWk0Oca\\\\\\\/U9aUZq\\\\\\\/TWqfauOtyXnaDWCW74OCiodex6efSoIIicBkyRyIJjF2fhoaMwDleBOpnqR1CA0Qt+kdqePGv+Cy5kE3bZvb7qXbCrFI+1YZXIcEoY4zG3xPbAAbJrDeoibhYCb0iIVamyhKmpY5LMn7e\\\\\\\/\\\\\\\/NwhFAMA0\\\\\\\/Wq3rdNL3L1wEy0RUiANosI+qXbkyGnhEjHzXQFtzRBhE\\\\\\\/n7VBoc0TRs1CumneJsk2rHt1bhvDutZx3sdqa\\\\\\\/tTo87SpmLThul+OuvTwrfJQJZMLvhKS2msSjJyCWN307I5OdPZ3okKJKxHbnwwkzcSFHGbArYObAap+9RVkUmbmOLoLrrRIsxP3+6SdxSx+aWb6o2lnzfC3zLWKz6ktUazSJGGenMf2KxsYwtconaocTT5DnDHMYBPy00y85lUW1C5BDduBpqp3l7ac7r2fR3DITnG4wLW9\\\\\\\/Yd6GSyIrQfNIKXgNC4Tt3IZ1JNe\\\\\\\/l6OYKbfEEi+2izIVWBnzPtRNuVgCK68ViHsIbbqly8RvHIJ4HqWajmI1sm5Pf4jYvfX0\\\\\\\/9dOACWhOb8SzosQka47Hfp8+9sXWNbrYhb0Nz+0iSrj22Cj7fYN2RfqA5RN2zYNOoxNMG9RxEwlgA9Ip9jNqGkxuyHOKbJb3ZRkiEnNuHMXC02jrSj8+jd5PGZYjXqW\\\\\\\/rtjpV5XVB7S4etRq99FjwkYjfxPqCzbGFSX9RvXnAkqZG7XWUg5BllVAVCe886obs9bN9BnY9xppmFgVADfqGKvl0+oI\\\\\\\/\\\\\\\/TuJNOTpTsaMAiuJfTfowoql33v8rbj7olTjGc1DP2PM1gS1c+WWvP+QFZVIecOAD3EDIgaxEnegCBqOsj3ReWbvFGoZynFt0tajDN2qY9ejtfoggBN\\\\\\\/Rx7tYGcJ5UfCyfyPEavocILLgvucV+dnsHg668HR9QEOM+5h1+hhgSIH7Fe5d+NTh9R28digWnf74fTVQaC1aUWCk1uWK1P3nOmm5lbjl8MF+hAVu9qAeNajYoKu9wRloR+2L7e08fqCYO+ZVwVdWWJdgZLlzR9JI+c7X8wA+WNABiNlwxap8Y1315Ucp0Jh1+vdEGECEzbdHBQ5NqVf4LXXhHZdxLKoOUVLwVRblFaSvCCBEh3KGFo+EfMAJDE2GyDSOqBsrE640DnbYiejEL55Ii+9ZwqvsADdq05YCaJft179BR1u05ICMNoboMfHhTJ7cbyN8LEoz+qkO8E1\\\\\\\/oUmU93Iq6NWCYJEReOEMh1Eg9VjcNx\\\\\\\/Z\\\\\\\/q6WpW+Y3ogcmj\\\\\\\/rKn9bshQruEjK6cz\\\\\\\/GWArfiGbNlKV5c8QOnKsuB\\\\\\\/NMCXILY0VMnJtEEDJBUsGPmKAfaDgupJvtu6Izcl11efiF5aEvheU+nmo9\\\\\\\/BvXwn3Sc2tm\\\\\\\/7xHTRBbUvL3fo+5uZr\\\\\\\/nhvuTOb5Q50+vZOEWZR8d2iUVl8bJU1jaouaBZW+chTxYyypQjkK+zBqQ43CatdvcFv4X1cf3S7+Y46ZVtFjFnlS7LglJYCjRndC0YtjCJAyq3N2x2V8Pi6xEp1bMwOJWLbI0tdANTxCY0HCzkDUmCvx\\\\\\\/ys70\\\\\\\/TmKV6ABvrKKUQ9HymWIsD4FIxqyrvlOSwH7EDplbtcWaoQonBl8rLknurm2Kx4doxtPUK5QmhVCHFLfN9crH5AK+f\\\\\\\/jjxZSQpLTOd3Zzt5O+pFKWjWjelN5xjtdCKsQKExTP11E2O8lnQbt+WH7c9t4Uh2JZJv+mfTcR641uKjNLvZ6D+J\\\\\\\/iPK9z3E9zcldSbA20QJq26KW+hs\\\\\\\/1qmBe\\\\\\\/I9SrzH6pS6Ai4PFoZx+YCbKMxg0GUVFxlgSD0ZgBhFQ8NIXlfxjXU4IL8IxQF18x0sV0x9B4kA\\\\\\\/IvQpckOR5dmzMtuwg\\\\\\\/5opnzYImugQ0T6HUepE7nf1k77pwkSF9Q45ZiTGrIXRkMZ5HT\\\\\\\/X3ge3e3jG640hzhAkxKP6I7DK9zX8knlSAJPT2bn6Hog4kYlGALGlBDNXCHzCuAiNNC1rorsbgfX0SgIILLDenRrvGkFrlbgVTFcWyijeSI7ury7Z3DOU9p7xpu50YHwzAAYE9LA3r7T0RghhY+E0SrSpbCGE5qjTfdgSw\\\\\\\/WCES5qkqyPSKUGMYRXqu1qiSKj85WaVcinB\\\\\\\/alNsoPDKKs7nfLhIumYdCV8DRgLD2YMOHl6WyL19DYx5bYrSC6iMigcZpnZUmaPFqRgRUysM5bAPobAGXoaTzHbdpVkXzMOihBTluTKk17mCVuNi56OYXyepyP5hG6BiAzoZxIRzhfahC4a1YEP\\\\\\\/Ah95Ps2yGkLLLOm1vVGpGI0GfRw5a\\\\\\\/zsWGWV89yfSYMkSZdZa+VL97GN3vLm\\\\\\\/7ZLjJOQrLuWLr\\\\\\\/YTJArh0PGtv\\\\\\\/zRSTV6iFf08N46ToAj+\\\\\\\/6N6dtaDJGq\\\\\\\/tQydgdOFKogL\\\\\\\/BGSvaI3hYOKdJ6m38itk3R78Or3LiGdOG\\\\\\\/fFYQQNXwHMaEFINhTrpvvrx7zzj9SIe\\\\\\\/ALnsJLOEniiys2Za3p\\\\\\\/G\\\\\\\/fMei8tZOH0xiq1x\\\\\\\/8z1zZNOUJog==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fNJRtg+4sS+i\\\\\\\/4koCY+FDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022868135997fc1da20741f39aa769dcae0bcbceb6be713f2d983ea089286355b51\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KjXxpV6aKYk8JntQV+jKsJ+C3Iu6vHCvVMA1irUsT9vduLX9p\\\\\\\/yAZlhbZJsgHmeBgpg7gADn292DbZNIz\\\\\\\/xzOreEdNj0PoGb4DFjm553EwXIrpklZrCtXxYwNhH9gLP9AYnWOhyWc\\\\\\\/FhiKKza\\\\\\\/ocK8hn1DW8JEMcP429GVNgdF4OxWYOogGPHFrUAMmvhL7LPUqMgitb4OO8h\\\\\\\/VUfhbvkpWRQ3vtGhQOEqOkWS24aNPn3R6AT5ZNRFz7fss2YCstNu4gr3xeEE2D2qrS\\\\\\\/H1dr5slhP7emqgxYETuzk1OYnBi+KT2JH9vIexGjb4n5d2gIBGLUQarx\\\\\\\/PLBwhjZl3\\\\\\\/L2RaIUzWL9OW2b9Em7a+2GFua1ETFKxC34RwjD+8+BQbC1AXbO7kq9+\\\\\\\/TTDWkq+6XuMxVqmmvZqW8E3gRsjI46jZxAuRYhsIuu9kQrB3wmONFtcRbUmJLB6QqgVHOhuxSV6IpJICJCHuLeK+B6L+DVHRXjlkx5ylwFyKacn7ouP8+WYdx4rwNWemQqQXRbBTYAp3WAib7AcWfAxfzASizUTeZxMNZ01Ao2CpUTLrpIzpOC6XC49Uk\\\\\\\/RcvsCrXOzuYHfkEI+DnnjjZXUEZyUrL6iXKQXiZUEaUF2zSkWPlRhP7eKsTfzBU\\\\\\\/KFs\\\\\\\/0clM6Yur9OV4qKzhbWbjfVkFOA9PsSdKlTY9R\\\\\\\/1yRU3RWVvJUW2Ou7CjnX6T5CVHCo71ppRfOUKN9KjNoDhks56W+2r5qWMfdZrRz9ne0mlY\\\\\\\/WKfw8Q2ushREC3RrqedcrnIXcA4VlzZXjvID8xpcfcEqvVRuuHYsQOuBGgZ+YHAXSmPdr0GAwuv++XAcy9YwD4l3Cln+gAcWHsdWgj8Uo7tW\\\\\\\/F7GB6RmrmiLLNfs7XS7TdyJOqAZCXXJhh0gR8E1te9uWZ5A+8h+FCP2qlABNdZR3v8qqIkiUwH3fKoC0KUBQxECZNb1vilsli87QApDEgtFqHhUzSkaRQBjvsNtCn6wTmaUTnmLm776zElh5dW4b\\\\\\\/OWEG8rR6nCnhoLM12GKUqkOi8X3LPSQ2LBTQ+tLWlwpK2Q8UgEMT7F4+PbaOn0rPLnqdqTY+DawA83aQPvHpAsJBQ2Jm7y0Kb4bNQOjfAkeQeR4IUuHNdUThTUqRb1OjPAFesk7p+Hz3x+ZqboiobXMlNIw1x+Z4+J\\\\\\\/waneUpQmOT94EiIrSlB\\\\\\\/P\\\\\\\/xMw1frvEE7NJdHXlHF9KxW1mYshBsGdUumy\\\\\\\/vxHMVyzY6m\\\\\\\/0uh+Ud7Q7dOkS+r1WlQ\\\\\\\/2FtQTYnZNz6nP6LlpZFrPxVFeHwggLQ9mROjTqW4vO5p3wlkpVmqXMajCrpsjCbQN0kYIcIs4uGWDamZxWrMsdZY+7OW7gEhAaKeGECLPoEk9F14yljLycU6YKry9Jw8Ysi+2vQCFskydD1ltySnPRneI96Y\\\\\\\/nrqYenkQTl3se5FMvFPWl\\\\\\\/zNSVwS\\\\\\\/XiQfJ\\\\\\\/VrL7wzCS73kJ9QJjEBY9QdyTkN7Wy6+gpZSdpZ4M0JOU5SkpdBhQsfzXmpTNT3KxtggyEWc+6xLrL0d5YV8Wc+C3W190MHmZNDpR1W4l4FeHORjc\\\\\\\/ozk9qVOUim6qK5A5mV+NReEX2u60wHVumhCQzlM02rcaiT9CG+XoiWcXSHVhINcEHEh4EU\\\\\\\/4gw8eOKM7s8auIzXwU9KOW4HTYjHfmgGYZtO0NrR6ChjczM6v2S3jnJFRhOJjzwoSAXRr7bHV074kZ7GpwKIgon5M8VF5nmqheN6U94jd1zXQ5pBmLEIRouZTFtbTR0psfKNE30Pz+3SWReknQmi9GlalgMD\\\\\\\/nM2MC\\\\\\\/X8hLMlfheavyf3VjC6TB8+rbfAM\\\\\\\/rloJKl6Z5bcGWAX10tcGJq6ugCLLsW\\\\\\\/92wW47WLx5xwpnTDo2UD8KbZrtu+OW9pkgnLRvecqlGQz\\\\\\\/NOLKXrFbuyn+yPGT6y2\\\\\\\/aLxO4V9mPijjIpkkxwKXBqAT2YK051ply4BOkh51otKcf6re1EQSkqOziXjyjJRhD8rbQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qizTaV6gcJ+J6CMz1dAvPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221022a48853ef58703c0584827e4062aceab3734d2515b17ab5ddd5139280c25f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489137 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1347" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FLCRyjidO9vZoMkcgeTkIUbdRRe67RhxAgZCnD3SOms" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1347" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iD5jgyaBw9IsCc5nIrSPo5bcvAnR8T84JsFn0z70vl8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1335" title="A Genomic Reference Panel for Drosophila serrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1367" title="A Genetic Screen Identifies PRP18a, a Putative Second Step Splicing Factor Important for Alternative Splicing and a Normal Phenotype in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489137" id="node489137" data-pisa="ggg;8/4/1347" data-pisa-master="ggg;g3.117.300454" data-apath="/ggg/8/4/1347.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Italo Granato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Fritsche-Neto</span>, <span class="highwire-citation-author" data-delta="3">Osval A. Montesinos-Lopez</span>, <span class="highwire-citation-author" data-delta="4">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="5">Massaine Bandeira e Sousa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1347-1365;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300454 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489137" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Italo Granato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Italo%2BGranato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Granato%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItalo%2BGranato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Italo Granato</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Fritsche-Neto</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Osval A. Montesinos-Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span>Facultad de Telemática, Universidad de Colima, CP 28040 Colima, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-Lopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-Lopez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-Lopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Massaine Bandeira e Sousa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massaine%2BBandeira%2Be%2BSousa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandeira%20e%20Sousa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMassaine%2BBandeira%2Be%2BSousa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/4/1347" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489137" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489137/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1347.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489137" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489137/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1347.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489137" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489137/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/4/1347.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489137" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489137/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In this study, we compared the prediction accuracy of the main genotypic effect model (MM) without G×E interactions, the multi-environment single variance G×E deviation model (MDs), and the multi-environment environment-specific variance G×E deviation model (MDe) where the random genetic effects of the lines are modeled with the markers (or pedigree). With the objective of further modeling the genetic residual of the lines, we incorporated the random intercepts of the lines (l) and generated another three models. Each of these 6 models were fitted with a linear kernel method (Genomic Best Linear Unbiased Predictor, GB) and a Gaussian Kernel (GK) method. We compared these 12 model-method combinations with another two multi-environment G×E interactions models with unstructured variance-covariances (MUC) using GB and GK kernels (4 model-method). Thus, we compared the genomic-enabled prediction accuracy of a total of 16 model-method combinations on two maize data sets with positive phenotypic correlations among environments, and on two wheat data sets with complex G×E that includes some negative and close to zero phenotypic correlations among environments. The two models (MDs and MDE with the random intercept of the lines and the GK method) were computationally efficient and gave high prediction accuracy in the two maize data sets. Regarding the more complex G×E wheat data sets, the prediction accuracy of the model-method combination with G×E, MDs and MDe, including the random intercepts of the lines with GK method had important savings in computing time as compared with the G×E interaction multi-environment models with unstructured variance-covariances but with lower genomic prediction accuracy.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-enabled-prediction-accuracy" class="hw-term hw-article-keyword hw-article-keyword-genomic-enabled-prediction-accuracy" rel="nofollow">Genomic-enabled prediction accuracy</a></li><li class="kwd"><a href="/keyword/genotype-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genotype-&#xD7;-environment-interaction" rel="nofollow">genotype × environment interaction</a></li><li class="kwd"><a href="/keyword/main-genetic-effects" class="hw-term hw-article-keyword hw-article-keyword-main-genetic-effects" rel="nofollow">main genetic effects</a></li><li class="kwd"><a href="/keyword/deviations-main-genetic-effects" class="hw-term hw-article-keyword hw-article-keyword-deviations-from-main-genetic-effects" rel="nofollow">deviations from main genetic effects</a></li><li class="kwd"><a href="/keyword/random-intercepts" class="hw-term hw-article-keyword hw-article-keyword-random-intercepts" rel="nofollow">random intercepts</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li></ul><p id="p-3">Genomic selection (GS) predicts breeding values of complex traits based on dense marker information (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Meuwissen <em>et al.</em> 2001</a>) and has shown good prediction accuracy achieved by random cross-validation partitions of plant breeding data (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pérez-Rodríguez <em>et al.</em> 2012</a>). As molecular markers become cheaper and more abundant, GS-assisted breeding has become commonly used in plant and animal improvement. When performing genomic prediction of breeding values of unobserved individuals, the relationship between individuals in the training and testing sets is computed through the genomic relationship matrix, and the prediction model is referred to as the Genomic Best Linear Unbiased Predictor (GBLUP) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">VanRaden, 2007</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2008</a>).</p><p id="p-4">Multi-environment trials are routinely conducted in plant breeding to estimate and take advantage of genotype × environment interaction (G×E) for selecting stable and high performing lines across environments and within environments. Therefore, implementation of GS strategies in plant breeding should be useful for estimating the parameters of the model and predicting G×E, as is commonly done in conventional plant breeding. Modern statistical analyses of multi-environment trials assess G×E by using pedigree information with linear mixed models (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Piepho<span class="sc">,</span> 1997</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">1998</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Smith <em>et al.</em> 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Crossa <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Burgueño <em>et al.</em> 2007</a>); however, these models do not incorporate marker information.</p><p id="p-5">A Bayesian GBLUP regression model for assessing genomic-enabled prediction combining G×E introduces the main effects of environments and lines and the interaction effects of markers and environmental co-variables via random variance-covariance structures (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> 2014</a>). The Bayesian regression model of <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> is similar to that of <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> with one difference: that genomic values are partitioned into components that are stable across environments (main genomic effects) and others that are environment-specific (genomic G×E) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2016</a>). Although both models assume positive sample correlations among environments and can be fitted using the BGLR package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos and Pérez-Rodríguez 2016</a>), the advantage of the model of <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> over the model of <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> is that it can be implemented using both shrinkage methods and variable selection methods and is efficient when applied to sets of environments that have positive correlations because the genetic covariance between any pair of environments is the variance of the main effect, which makes the covariance between pairs of environments positive (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> 2015</a>).</p><p id="p-6"><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> (2016)</a> used the Bayesian model of <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> to compare methods that apply GS models with G×E using a linear kernel (GBLUP) (GB) and a non-linear Gaussian kernel (GK) for single-environment and multi-environment breeding data sets. The authors found the GK models had higher prediction accuracy than the GB models and explained that the GK models captured major and complex marker effects in addition to their interaction effects. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> compared the prediction accuracy of the multi-environment, single variance G×E deviation model (MDs) of <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> with GK (MDs-GK) and the prediction accuracy of the multi-environment environment-specific variance G×E deviation model (MDe) of <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> with the GK method (MDe-GK). Then, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> compared the models including the GK method with the prediction accuracy of their counterpart models using the GB methods (MDs-GB and MDe-GB). In addition, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> also compared the accuracy of the four previous models with the accuracy of the multi-environment, main genotypic effect (MM) of <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> using the GB and GK methods (MM-GB, and MM-GK). Results show that for grain yield, a notable increase in prediction accuracy of GK over the GB methods ranged from 9 to 49% in one data set and from 34 to 70% in another data set.</p><p id="p-7">In general, the previous linear mixed multi-environment models assumed the environments as fixed or random effects, and lines as random effects incorporating into the model the random slope of the genetic effect of the lines distributed as a normal random variable with zero mean and variance-covariance structure constructed from markers or pedigree; also, the genetic effect (intercept) of the lines can be considered as having a normal distribution with zero mean and constant variance (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mota <em>et al.</em> 2016</a>). The random intercept of the lines is often not included in the model when no exchange of information occurs, assuming the intercepts are independent (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Pérez-Rodríguez <em>et al.</em> 2015</a>). However, recent studies have incorporated random intercepts (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Mota <em>et al.</em> 2016</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2017</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sukumaran <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> 2017</a>) in order to achieve higher genomic-prediction accuracy in cases where lines were observed in some environments but not in others (random cross-validation 2, CV2 of <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>); this is because the posterior distribution of the intercept generates a variance-covariance structure that allows exchanging information between the lines of the training and testing sets. When newly developed lines have never been observed (untested) (random cross-validation CV1, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>), models do not improve the prediction accuracy with or without random intercept when compared with the single-environment model. One limitation of these multi-environment genomic G×E models for achieving relatively high genomic-enabled predictions is that correlations among environments should be positive. Also, none of the applications of the models of <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Sukumaran <em>et al.</em> (2017)</a>, and <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Jarquín <em>et al.</em> (2017)</a> compared genomic-enabled prediction accuracy with GB kernel <em>vs.</em> GK kernel.</p><p id="p-8">The previous Bayesian regression models of <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> use the Hadamard product for modeling G×E and show that the exchange of information between environments is achieved by means of the variance-covariance matrix of the main effects. Thus, the variance component of the main effects measures the stability across environments and the variance component of the specific effects measures the deviations from the main effects due to specific combinations of lines in environments (G×E). This approach has the advantage that it can be used when the number of lines in each environment is the same, but also when there is an unbalanced number of lines in environments, as shown by <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a>.</p><p id="p-9">On the other hand, GBLUP methodology (together with pedigree) can incorporate and model G×E effects, by means of the Kronecker product of the variance-covariance matrices of the genetic relationship between environments and the genomic or pedigree relationship between the lines (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Oakey <em>et al.</em> 2016</a>) where the structure of the models allows estimating negative genetic correlations between environments. Based on this, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> recently compared a Bayesian regression model for the genetic effects described by the Kronecker product of unstructured variance-covariance matrices of genetic correlations between environments and genomic kernels under the GB and GK methods. An extension includes an extra genetic residual component with random intercepts. Results of the analyses of five data sets indicated that including the random intercepts is still beneficial for increasing genomic prediction accuracy when lines have been tested in some environments. However, one drawback of the Bayesian regression models of <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> is the computing time for the iteration required for the Monte Carlo Markov Chain (MCMC) method to achieve the convergence of the posterior and predictive distributions.</p><p id="p-10">Recently <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> (2017)</a> proposed an R package called Bayesian Genomic G×E (BGGE) to obtain a rapid fit of Bayesian mixed linear models with homogeneous error variances for the models of <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a>, <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> and also for the models used by <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> (MM, MDs, and MDE). The approach of <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> (2017)</a> uses an R library that saves time by using the structure of the block diagonal matrices with additional parameterizations to shorten the iteration time without losing precision.</p><p id="p-11">Based on the above, the main objective of this study was to compute the prediction accuracy of 16 model-method combinations and compare their prediction accuracy for four different data sets (two maize and two wheat multi-environment trials) with an unbalanced number of lines in environments, and different complexity of the G×E interaction. The 16 model-methods comprise the multi-environment, main genotypic effect (MM), the multi-environment, single variance G×E deviation model (MDs) and the multi-environment environment-specific variance G×E deviation model (MDe) with the GB and GK kernel methods and with and without including random intercepts (12 model-methods) plus 4 Bayesian regression models for the genetic effects described by the Kronecker product of unstructured variance-covariance (MUC) matrices of genetic correlations between environments and genomic kernels under the GB and GK methods and their extensions, including an extra genetic residual component with random intercepts. We discuss the advantages and disadvantages of the different model-methods for sets of environments with different G×E characteristics and different degrees of unbalance among lines.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-12">This study uses four multi-environment plant breeding data sets with different characteristics. Two maize data sets used by <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> (HEL and USP) had different numbers of maize hybrids in each environment and positive correlations between environments, whereas the two wheat data sets used by <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> (WHE1 and WHE5) had environments with negative or zero correlations but with the same number of wheat lines in each location.</p><p id="p-13">We used the same models of <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> (MM, MDs, and MDe) with linear (GB) and non-linear kernels (GK) (MM-GB, MM-GK, MDs-GB, MDs-GK, MDe-GB, MDe-GK) plus the addition of one random intercept component (l) that captures the variation of genetic residuals (MMl-GB, MMl<strong>-</strong>GK, MDsl-GB, MDsl-GK, MDel-GB, MDel-GK). These 12 model-methods were fitted with the BGGE package (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> 2017</a>).</p><p id="p-14">In this study models 2 and 3 of <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> are renamed as Multi-environment Unstructured Covariance (MUC) and Multi-environment Unstructured Covariance with random intercept vector <strong><em>f</em></strong> (MUC<strong><em>f</em></strong>), respectively, each fitted with the GB and GK kernel methods. Therefore, 4 additional models are included, MUC-GB, MUC-GK, MUC<strong><em>f</em></strong>-GB, and MUC<strong><em>f</em></strong>-GK. These models were fitted with the MTM package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos and Grüneberg 2016</a>) and their prediction accuracy was compared with the other 12 model-method combinations.</p><p id="p-15">In the first step, the phenotypic data were fitted according to the experimental design employed for each experiment, and the Best Linear Unbiased Estimates (BLUE) of the lines or hybrids for each location or environments were computed. In the second step, the various genomic models were fitted to perform random cross-validation and compute the prediction accuracy of the 16 model-method combinations.</p><div id="sec-2" class="subsection"><h3>Experimental data</h3><div id="sec-3" class="subsection"><h4>Maize data set HEL:</h4><p id="p-16">This maize data set comprises 452 maize hybrids evaluated in 2015 at five sites in Brazil: Nova Mutum (NM) and Sorriso (SO) in the state of Mato Grosso; Pato de Minas (PM) and Ipiaçú (IP) in the state of Minas Gerais; and Sertanópolis (SE) in the state of Paraná. The experimental design was a randomized block with two replicates per genotype and environment. Different numbers of hybrids were planted in each environment. The HEL parent lines were genotyped with an Affymetrix Axiom Maize Genotyping Array of 616 K SNPs with standard quality controls removing markers with a <em>Call Rate ≥</em> 0.95.</p></div><div id="sec-4" class="subsection"><h4>Maize data set USP:</h4><p id="p-17">This data set comprises 740 maize hybrids evaluated at Piracicaba and Anhumas, each with two levels of nitrogen (N) fertilization: Ideal N (IN) and Low N (LN) for a total of four artificial environments (P-IN, P-LN, A-IN, and A-IN). The hybrids were evaluated using an augmented block design including two replicated commercial hybrids as checks. There was an imbalance because not all hybrids were evaluated in all locations. Similar to the maize data set HEL, the USP parent lines were genotyped with an Affymetrix Axiom Maize Genotyping Array of 616 K SNPs with standard quality controls removing markers with a <em>Call Rate ≥</em> 0.95.</p></div><div id="sec-5" class="subsection"><h4>Wheat data set WHE1:</h4><p id="p-18">A historical set of 599 wheat lines from CIMMYT’s Global Wheat Program was evaluated in four mega-environments (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2016</a>) and genotyped using 1447 Diversity Array Technology (DArT) markers generated by Triticarte Pty. Ltd. (Canberra, Australia; <a href="http://www.triticarte.com.au">http://www.triticarte.com.au</a>). Markers with a minor allele frequency lower than 0.05 were not included.</p></div><div id="sec-6" class="subsection"><h4>Wheat data set WHE5:</h4><p id="p-19">This data set is described by <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> and includes 807 wheat lines evaluated in five environments using an alpha-lattice design with three replicates in each environment at CIMMYT’s wheat breeding station at Cd. Obregon, Mexico. The environments were three irrigation regimes (0i = zero irrigation, 2i = two irrigations, and 5i = five irrigations), two planting systems (B = bed planting and F = flat planting) and two different planting dates (N = normal and L = late).</p><p id="p-20">Genotypic data consisted of genotyping-by-sequencing (GBS) data, and markers with a minor allele frequency (MAF) lower than 0.05 were removed. After editing the missing markers, a total of 14,217 GBS markers were available for analyzing this data set.</p></div><div id="sec-7" class="subsection"><h4>Availability of the phenotypic and genotypic experimental data:</h4><p id="p-21"><a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> describe the two maize data sets and <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> give details of the two wheat data sets. The two maize data sets, HEL and USP, can be downloaded from the link <a href="http://hdl.handle.net/11529/10887">http://hdl.handle.net/11529/10887</a>, whereas the two wheat data sets can be found at the link <a href="http://hdl.handle.net/11529/10710">http://hdl.handle.net/11529/10710</a>, from where DATASET1.Wheat_GY.Rdata (Wheat data set WHE1) and DATASET5.Wheat_GY.Rdata (Wheat data set WHE5) were obtained.</p></div></div><div id="sec-8" class="subsection"><h3>Statistical models</h3><p id="p-22">The components of the 8 basic models are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and their full descriptions are given below and in Appendix 1. They include an overall mean (μ) and the fixed effects of the environments (other effects can be incorporated) modeled with the incident matrix <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-1.gif"/></noscript></span></span> and one vector of fixed effects <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-2.gif"/></noscript></span></span> for each environment. For the first group of six models (MM, MMl MDs, MDsl, MDe, and MDel), it is assumed that their genetic random components <strong><em>g</em></strong> have a normal distribution with mean zero and a variance-covariance structure comprising a known matrix K generated from markers (and computed using the GB or GK methods) multiplied by an unknown scaled parameter (variance component). Also 4 models in this group had different forms for modeling the G×E, MDs and MDe, with a variance-covariance structure constructed by the Hadamard product of the corresponding matrices and incorporating (or not) the random intercepts (l).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489303/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489303/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Components of the 8 models included in this study. Each of these models is fitted with the linear kernel (GB) and the Gaussian kernel (GK)</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">A second group of models (MUC) considers that their random components have a normal distribution with zero mean and a variance-covariance structure modeled by the Kronecker product of a matrix with unknown covariances among environments multiplied by a known K (computed using the GB or GK methods) and incorporating (or not) the random intercepts (<strong><em>f</em></strong>).</p><div id="sec-9" class="subsection"><h4>The multi-environment main genotypic effect model (MM):</h4><p id="p-24">Model MM (1) (Appendix 1) is equivalent to the across-environment model of <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> and when in the distribution of the random genetic effects g is used in model MM, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-48.gif"/></noscript></span></span> is used in the covariance (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">VanRaden 2007</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">2008</a>); the model is the GBLUP across environments (MM-GB), where X is the standardized matrix of molecular markers for the individuals of order <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-49.gif"/></noscript></span></span>, where p is the number of markers.</p><p id="p-25">However, markers can have a more complex function than the linear GBLUP. For example, the Gaussian kernel (GK) function (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2016</a>) is computed as <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-50.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-51.gif"/></noscript></span></span> is the Euclidean distance between the i<sup>th</sup> and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-52.gif"/></noscript></span></span> individuals given by the markers; <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-53.gif"/></noscript></span></span> is the bandwidth parameter that controls the rate of decay of K values (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Pérez-Rodríguez <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pérez-Elizalde <em>et al.</em> 2015</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2016</a>). In this work, GK is <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-54.gif"/></noscript></span></span>), where <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-55.gif"/></noscript></span></span> and the median of the distances is used as a scaling factor (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Crossa <em>et al.</em> 2010</a>). When in the distribution of the random genetic effects g of the MM model (1) is used with <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-56.gif"/></noscript></span></span>), in the covariance the model is the Gaussian kernel across environments (MM-GK) (<a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> 2017</a>).</p><p id="p-26">The genetic variation between lines that is not explained by g in (1) (Appendix 1) can be captured by the random vector l that is considered a random intercept for each line; thus when random effects l are added, model MM becomes model MMl<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-57.gif"/></noscript></span></span>where the random intercepts <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-58.gif"/></noscript></span></span> with I being the identity matrix of size <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-59.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-60.gif"/></noscript></span></span> the variance component that indicates the influence of l; the incidence matrix <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-61.gif"/></noscript></span></span> connects the genotypes to the phenotypes. As in MM, the kernel matrix K of the random effect <strong><em>g</em></strong> of model MMl can be fitted with GBLUP (MMl-GB) or with Gaussian kernel (MMl-GK).</p></div><div id="sec-10" class="subsection"><h4>The multi-environment single variance genotype × environment interaction deviation model (MDs):</h4><p id="p-27">Model (2) (Appendix 1) (MDs) adds to model (1) the random interaction effect of the environments with the genetic information of the lines (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-62.gif"/></noscript></span></span>). When the random component l is added to model (2), the MDs model becomes MDsl:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-63.gif"/></noscript></span></span>Each environment matrix <strong><em>K</em></strong> (Appendix 1) of models MDs and MDsl can be fitted with a linear kernel (MDs-GB, MDsl-GB) or a Gaussian kernel (MDs-GK, MDsl-GK).</p></div><div id="sec-11" class="subsection"><h4>Multi-environment environment-specific variance genotype × environment deviation model (MDe):</h4><p id="p-28">The environment-specific variance genotype × environment deviation model (MDe) (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> 2015</a>) differs from MDs on how the interaction component is considered; <strong><em>g</em></strong> is the main genetic effect across environments and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-64.gif"/></noscript></span></span> is the specific genetic effect in each environment. When the random component l is added to (3) (Appendix 1), the MDe model becomes MDel:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-65.gif"/></noscript></span></span>where matrices <strong><em>K</em></strong> for <strong>g</strong> and <strong><em>K<sub>E</sub></em></strong> for <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-66.gif"/></noscript></span></span> of models MDe and MDel can be fitted with a linear kernel (MDe-GB, MDel-GB) or with a Gaussian kernel (MDe-GK, MDel-GK).</p></div><div id="sec-12" class="subsection"><h4>Multi-environment With unstructured variance-covariance (MUC):</h4><p id="p-29">This model considers that there is a genetic correlation between environments that can be modeled with matrices of order <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-67.gif"/></noscript></span></span> (where <em>m</em> denotes the environment) (<a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2017</a>). The MUC is expressed as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-68.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-69.gif"/></noscript></span></span>is a vector with the observation <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-70.gif"/></noscript></span></span> belonging to the <em>j</em><sup>th</sup> environment <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-71.gif"/></noscript></span></span>each of the same size (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-72.gif"/></noscript></span></span>; the random vector <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-73.gif"/></noscript></span></span> is the vector of genetic values, and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-74.gif"/></noscript></span></span>the vector of random errors both assumed normally distributed with u <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-75.gif"/></noscript></span></span> and <strong><span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-76.gif"/></noscript></span></span></strong> where <strong>⊗</strong> is the Kronecker product.</p><p id="p-30">The variance-covariance matrix of u is the Kronecker product of one unstructured matrix with information between environments <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-77.gif"/></noscript></span></span> that needs to be estimated and another known matrix with information between the lines based on <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-78.gif"/></noscript></span></span>. Then the <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-79.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-80.gif"/></noscript></span></span>is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-81.gif"/></noscript></span></span>where the <em>j<sup>th</sup></em> diagonal element is the genetic variance <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-82.gif"/></noscript></span></span>within the <em>j<sup>th</sup></em> environment, and the off-diagonal elements are the genetic covariances <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-83.gif"/></noscript></span></span>between environments <em><span class="underline">j</span></em> and <em>j</em>’. For a large number of environments, a factor analytical model usually performs better than the unstructured model (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Oakey <em>et al.</em> 2016</a>). Furthermore, matrix Σ is an error diagonal matrix of order <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-84.gif"/></noscript></span></span>, <em>i.e.</em>, Σ<strong>=</strong><em>diag</em>(<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-85.gif"/></noscript></span></span>).</p></div><div id="sec-13" class="subsection"><h4>Multi-environment With un-structured variance-covariance and random intercepts (MUCf):</h4><p id="p-31">The MUC model can be extended by adding an extra variability to account for genetic variance among individuals across environments, that is, by adding the random vector <strong><em>f</em></strong> (<a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2017</a>). Therefore, the extension of the previous random linear model is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-86.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-87.gif"/></noscript></span></span> with the random vectors <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-88.gif"/></noscript></span></span>being independent of <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-89.gif"/></noscript></span></span> and normally distributed <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-90.gif"/></noscript></span></span>. Matrix <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-91.gif"/></noscript></span></span>is unstructured and captures genetic variance-covariance effects between the individuals across environments that were not captured by the <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-92.gif"/></noscript></span></span> matrix; matrix <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-93.gif"/></noscript></span></span> can be expressed as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-94.gif"/></noscript></span></span>where the <em>j<sup>th</sup></em> diagonal element of the<span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-95.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-96.gif"/></noscript></span></span> is the genetic environmental variance <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-97.gif"/></noscript></span></span> within the <em>j<sup>th</sup></em> environment, and the off-diagonal element is the genetic covariance <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-98.gif"/></noscript></span></span> between environments <em><span class="underline">j</span></em> and <em>j</em>’. Similar to the previous cases, models MUC and MUC<strong><em>f</em></strong> can be fitted using GB or GK kernels to generate the four model-method MUC-GB,MUC-GK, MUC<strong><em>f</em></strong>-GB, MUC<strong><em>f</em></strong>-GK.</p></div><div id="sec-14" class="subsection"><h4>Model implementation and random cross-validation for assessing prediction accuracy in the four data sets:</h4><p id="p-32">For the two maize data sets, models MM-GB, MM-GK, MDs-GB, MDs-GK, MDe-GB, and MDe-GK were fitted with the new software BGGE (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> 2017</a>). Models MMl-GB, MMl-GK, MDsl-GB, MDsl-GK, MDel-GB, and MDel-GK were also fitted with BGGE with the same random partitions used by <a id="xref-ref-27-10" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> to make results comparable for random-cross-validation 1 (CV1) and random cross-validation 2 (CV2) (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>). Models MUC<strong><em>f</em></strong> and MUC of <a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> were fitted using the software MTM (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de los Campos and Grüneberg 2016</a>) with the GB and GK kernel methods and with the same random partitions used for the 12 model-method combinations previously defined for random cross-validations CV1 and CV2. A fivefold random cross-validation was used assigning 80% of the observations to the training sets and 20% to the testing (validation) set. However, most of the results and discussion focus on cross-validation CV2. The two wheat data sets were fitted with the 12 model-method combinations (models MMl-GB, MMl-GK, MDsl-GB, MDsl-GK, MDel-GB, MDel-GK, MM-GB, MM-GK, MDs-GB, MDs-GK, MDe-GB, MDe-GK) using the BGGE software of <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> (2017)</a>.</p><p id="p-33">Two random cross-validations (CV1 and CV2) were generated; CV1 attempts to mimic a situation where a set of lines were never evaluated in a set of environments, whereas CV2 mimics a sparse testing scheme where some lines were evaluated in some environments but not in others. Results based on CV2 are shown in the main text, tables and figures. Results of random cross-validation CV1 are given in Tables S1-S4 of Appendix 2. To implement the proposed 12 model-method combinations, 50 random partitions were performed with 80% of the lines used for training and the remaining 20% of the lines used for testing. The metric for measuring the performance of prediction accuracy was the Pearson correlation calculated between the observed and predicted values of the testing sets.</p></div></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><p id="p-34">The results are given in four sections, one for each data set. In each section, we provide the results of the variance component estimates and the prediction accuracy for each of the 12 model-method combinations.</p><div id="sec-16" class="subsection"><h3>Maize data set HEL</h3><p id="p-35">This maize data set has a total of 452 maize hybrids with a different number in each of the five sites (<span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-99.gif"/></noscript></span></span> 247, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-100.gif"/></noscript></span></span> 330, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-101.gif"/></noscript></span></span> 452, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-102.gif"/></noscript></span></span> 367, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-103.gif"/></noscript></span></span> 330). The sample phenotypic correlations among locations are positive with intermediate-to-low values, where location SE has low correlations with all the other locations, and locations NM, IP, and PM show relatively high correlations with the other locations (Table A1, Appendix 3).</p><p id="p-36">Models without the random component l always show a lower residual variance component in the GK models than in the GB models; for example, for model MDs-GK, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-104.gif"/></noscript></span></span> = 0.278 and for MDs-GB, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-105.gif"/></noscript></span></span> = 0.591 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). However, when the models include l, these differences become smaller; for example, for MDsl-GK, <strong><span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-106.gif"/></noscript></span></span></strong> = 0.277 and for MDsl-GB, <strong><span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-107.gif"/></noscript></span></span></strong> = 0.368, indicating that for method GB, the random component l explains the variation of the observations better, whereas for GK, including l does not have much influence on the residual. This is also reflected in the small value of <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-108.gif"/></noscript></span></span> = 0.013 for MDsl-GK as compared with <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-109.gif"/></noscript></span></span> = 0.243 for MDsl-GB.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489237/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489237/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">MAIZE HEL data set. Estimated variance components for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs) and environment-specific variance G×E deviation model (MDe) with two kernels, GBLUP (GB) and Gaussian (GK), with l and without l, for grain yield (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">The size of the genetic component, <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-119.gif"/></noscript></span></span>, is always much higher for MM-GK, MDs-GK, and MDe-GK than for models with the GB method. For models MMl, MDsl, and MDel, the sum of <strong><span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-120.gif"/></noscript></span></span></strong> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-121.gif"/></noscript></span></span> is higher than the component <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-122.gif"/></noscript></span></span> for models MM, MDs, and MDe. For example, for model MMl-GB, <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-123.gif"/></noscript></span></span> is 0.429, whereas for model MM-GB, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-124.gif"/></noscript></span></span> is 0.356; MDsl-GB summation <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-125.gif"/></noscript></span></span> is 0.415 <em>vs.</em> MDs-GB with <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-126.gif"/></noscript></span></span> = 0.370, and for model MDel-GB <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-127.gif"/></noscript></span></span>= 0.430 <em>vs.</em> MDs-GB with <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-128.gif"/></noscript></span></span> = 0.370. The variance explained by the G×E of MDs, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-129.gif"/></noscript></span></span>, is higher for GK than for GB and slightly higher for models with the random component l than for models without <strong>l</strong>. The variance components for the specific environments show increases in MDel-GK compared to MDel-GB, and in MDe-GK compared to MDe-GK (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-39">Models including the random component l with GK did not improve the prediction accuracy of the locations as compared with the prediction accuracy of models without l with GK (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); however, models with l had consistently higher prediction accuracies than models with GB. In all cases, MMl showed lower prediction accuracy than models with G×E (MDsl and MDel). Similarly, model MM had lower prediction accuracies than models that incorporate G×E (MDs and MDe). These differences are smaller for locations that had higher sample phenotypic correlations with other locations than for locations with low phenotypic correlations. For example, location NM had prediction accuracies of 0.569, 0.589, and 0.588 for models MMl-GB, MDsl-GB, and MDel-GB, respectively, whereas location SE with low sample phenotypic correlations among locations had prediction accuracies of 0.372, 0.544, and 0.548 for models MMl-GB, MDsl-GB, and MDel-GB, respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489216/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489216/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Maize HEL data set. Mean Pearson’s correlation (50 partitions) of each location for random cross-validation CV2, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUC<em>f</em>) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 5 locations (SO, SE, PM, NM, and IP) of maize data set HEL for the proposed models MDel-GK, MDel-GB, MUCf-GK, MUCf-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1775730164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 5 locations (SO, SE, PM, NM, and IP) of maize data set HEL for the proposed models MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GK, MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GB, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GK, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1347/F1.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/4/1347/F1.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489204" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-41" class="first-child">Plot of the prediction accuracy using Pearson’s correlation for each of the 5 locations (SO, SE, PM, NM, and IP) of maize data set HEL for the proposed models MDe<strong><em>l</em></strong>-GK, MDe<strong><em>l</em></strong>-GB, MUC<strong><em>f</em></strong>-GK, MUC<strong><em>f</em></strong>-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">All models with kernel GK had higher prediction accuracies (with and without the random component l) than models with kernel GB (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). However, these differences are lower for models that include the random component l (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). For example, for location SO, the prediction accuracies for models MDsl-GK and MDsl-GB were 0.673 and 0.639, respectively, whereas for MDs-GK and MDs-GB, the mean prediction accuracies were 0.666 and 0.466, respectively. Comparing models with kernel GB, with and without l, the predictions are always higher when the model includes <strong>l</strong> than when the model excludes l; for example, for location IP, the mean prediction accuracies were 0.778 and 0.683 for MDsl-GB and MDs-GB, respectively (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Note that the variance component of the random effect l <strong><span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-130.gif"/></noscript></span></span></strong> was 0.243 for model MDsl-GB (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Furthermore, model 3 from <a id="xref-ref-8-10" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> with the unstructured variance-covariance component <strong><em>f</em></strong> or model 2 without <strong><em>f</em></strong> did not show any clear superiority, in terms of mean prediction accuracy, over models MDsl and MDel and MDs and MDe with GK and GB (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-43">Random cross-validation CV1 decreased the prediction accuracy as compared with results achieved for CV2 (Table S1, Appendix 2); the trends and patterns of the prediction accuracy of the locations between models and methods are similar to those found for CV2, including those found for models MUC and MUC<strong><em>f</em></strong>.</p><p id="p-44">In summary, results from maize data HEL indicated that models with the random component l with GK including G×E (MDsl-GK and MDel-GK) show similar mean prediction accuracy as models excluding the random component l. However, this did not occur with GB models where including the random component l increased the prediction accuracy for all 5 locations. Prediction accuracy using GK was always higher than using GB with or without the random component l. Also, the differences between the models with and without <strong>l</strong> and between GK and GB were smaller for locations that had higher sample phenotypic correlations with other locations. Finally, the differences in prediction accuracy were negligible between the proposed models including G×E with GK and GB and with and without the random effect l and models MUC<strong><em>f</em></strong> and MUC for all locations.</p></div><div id="sec-17" class="subsection"><h3>Maize data set USP</h3><p id="p-45">This maize data set is comprised of 739 maize hybrids with different numbers of lines in each of the four sites (<span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-131.gif"/></noscript></span></span> 731, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-132.gif"/></noscript></span></span> 732, <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-133.gif"/></noscript></span></span> 731, <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-134.gif"/></noscript></span></span> 737). Locations P-IN and A-IN had relatively high correlations with the other locations, whereas A-LN had low ones (Table A1, Appendix 3). The residual variance components for GK are smaller than those for GB for models MM, MDs and MDe; for instance, MM-GK had <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-135.gif"/></noscript></span></span> = 0.589 while MM-GB had <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-136.gif"/></noscript></span></span> = 0.854. Similarly, the residual variance components for MDsl and MDel with GK are lower than for MDsl and MDel with GB. The variance components of the random intercept (<span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-137.gif"/></noscript></span></span>) of GK methods are not negligible (as in data set HEL) and are always lower than for the corresponding GB methods (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489148/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489148/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Maize USP data set. Estimated variance components for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs) and environment-specific variance G×E deviation model (MDe) with two kernels, GBLUP (GB) and Gaussian kernel (GK), with l and without l for grain yield (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">The estimated genetic variance components <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-146.gif"/></noscript></span></span> for GB in models MM, MDs and MDe were 0.214, 0.209, and 0.206, respectively (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), increasing the genetic environmental stability (<span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-147.gif"/></noscript></span></span>of models MMl-GB, MDsl-GB, and MDel-GB to 0.511, 0.513, and 0.514, respectively. The specific components for each environment of models MDe-GB and MDel-GB were negligible. The variance component (<span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-148.gif"/></noscript></span></span>) of the G×E models MDs and MDsl for GB and GK was also negligible.</p><p id="p-48">In general, models with l-GB had similar prediction accuracy as models with l-GK, whereas the increase in prediction accuracy of models without l-GK over models with GB is clear. For example, for P-LN, models MDsl-GK and MDsl-GB had prediction accuracies of 0.545 and 0.546, respectively, whereas for MDs-GK and MDs-GB, the prediction accuracies were 0.524 and 0.325 (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Models with l-GB showed significant improvement in prediction accuracy compared to models GB without l; for example, for location P-IN, the mean prediction accuracies of MDsl-GB and MDs-GB were 0.591 and 0.368, respectively (due to the influence of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-149.gif"/></noscript></span></span> = 0.349 for model MDsl-GB; see <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). All models with GK with the random intercept l and with high values of <strong><span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-150.gif"/></noscript></span></span></strong>gave higher prediction accuracies than GK models without l. There are no clear differences between model MUC<strong><em>f</em></strong> and the proposed model with the random component l  with GK and GB in all the locations. Similar results were found for model MUC when compared to models without l. For this data set, results from CV1 (Table S2, Appendix 2) were all similar and lower than those obtained for CV2.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489265/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489265/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Maize USP data set. Mean Pearson’s correlation (50 partitions) of each environment for random cross-validation CV2, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 4 environments (P-LN, P-IN, A-LN, A-IN) of maize data set USP for the proposed models MDel-GK, MDel-GB, MUCf-GK, MUCf-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1775730164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 4 environments (P-LN, P-IN, A-LN, A-IN) of maize data set USP for the proposed models MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GK, MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GB, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GK, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1347/F2.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/4/1347/F2.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489323" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-50" class="first-child">Plot of the prediction accuracy using Pearson’s correlation for each of the 4 environments (P-LN, P-IN, A-LN, A-IN) of maize data set USP for the proposed models MDe<strong><em>l</em></strong>-GK, MDe<strong><em>l</em></strong>-GB, MUC<strong><em>f</em></strong>-GK, MUC<strong><em>f</em></strong>-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">In summary, results from maize data USP indicate that models with the random component l (MDsl-GK and MDel-GK) show higher mean prediction accuracy than models without l and using the linear kernel GB. The G×E variance component of models MDs and MDsl with GK and GB had negligible <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-151.gif"/></noscript></span></span>, indicating less complex G×E than that found for maize data set HEL. The differences in the mean prediction accuracy between the proposed models with or without the random effect l and models MUC<strong><em>f</em></strong> and MUC are small for models with GK and not clearly superior to the proposed models with GB.</p></div><div id="sec-18" class="subsection"><h3>Wheat data set WHE1</h3><p id="p-52">For this data set, environment E1 had negative correlations with the other environments (E2-E4), whereas environments E2-E4 had high correlations among themselves (Table A1, Appendix 3). Models with GK fitted the WHE1 data better than models with kernel GB (low residual variances of GK models as compared to GB models). Also, models with random component l had lower residual variance components than models without l. As opposed to the previous two maize data sets, where the magnitude of the variance components determines the prediction ability, the presence of environments with negative correlations with other environments makes interpreting the variance components in relation to their predictive ability not as straightforward as in the previous two data sets (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). For example, models MMl and MM with GK and GB had estimates of the random error variance that were much higher (∼0.8) than those of the other models; thus the prediction accuracy of these models is expected to be low for at least the environments with negative correlations.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489305/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489305/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Wheat WHE1 data set. Estimated variance components for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs) and environment-specific variance G×E deviation model (MDe) with two kernels, GBLUP (GB) and Gaussian (GK), with l and without l for grain yield (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><p id="p-54">The genetic variance component <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-160.gif"/></noscript></span></span> varied for models MM-GB, MDs-GB, and MDe-GB (0.192, 0.219, and 0.414, respectively) as well as for the GK models (0.599, 0.752, and 1.404, respectively). The contribution of l measured in <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-161.gif"/></noscript></span></span> was small for MDsl-GK and MDsl-GB (0.101 and 0.107) (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>) and negligible for the other models with l. On the other hand, the G×E interaction variance components <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-162.gif"/></noscript></span></span> for GK and GB are important (MDsl-GK <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-163.gif"/></noscript></span></span> = 1.637, MDsl-GB <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-164.gif"/></noscript></span></span> = 0.42; MDs-GK <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-165.gif"/></noscript></span></span> = 1.349, MDs-GB <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-166.gif"/></noscript></span></span> = 0.349) and much higher than in the two maize data sets. Models MDel-GK and MDel-GB showed high specific variance components for E1 (3.356 and 1.058, respectively) and for E4 (1.147 and 0.3) causing most of the interaction in this data set (these are the environments with the lowest sample correlations with the other environments) and contributed the least to genetic environmental stability.</p><p id="p-55">Models with G×E (MDs and MDe) had mean prediction accuracies higher than MM models with lower mean prediction accuracy in E1 and E4 as compared with E2 and E3 (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The exceptions are models MM-GB and MM-GK, which had higher prediction accuracy than models MDs-GB and MDS-GB in E3. Models MDel-GK and MDe-GK had higher prediction accuracy than models MM, MDs and MDe with and without l for GB and GK in all locations, except MDe-GK in E1. However, in all cases and environments, models MUC<strong><em>f</em></strong> and MUC had better prediction accuracies than all 12 genomic model-method combinations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Lower prediction accuracies were found for CV1 (Table S3, Appendix 2) than for CV2; however, the decrease in prediction accuracy of CV1 was lower than for the two wheat data sets.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489252/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489252/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">WHEAT WHE1 data set. Mean Pearson’s correlation (50 partitions) of each environment for random cross-validation CV2, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 4 environments (E1-E4) of wheat data set WHE1 for the proposed models, MDel-GK, MDel-GB, MUCf-GK, MUCf-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1775730164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 4 environments (E1-E4) of wheat data set WHE1 for the proposed models, MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GK, MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GB, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GK, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1347/F3.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/4/1347/F3.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489242" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-57" class="first-child">Plot of the prediction accuracy using Pearson’s correlation for each of the 4 environments (E1-E4) of wheat data set WHE1 for the proposed models, MDe<strong><em>l</em></strong>-GK, MDe<strong><em>l</em></strong>-GB, MUC<strong><em>f</em></strong>-GK, MUC<strong><em>f</em></strong>-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">In summary, G×E for this data set is more complex than for the two previous maize data sets. This is expressed by higher values of <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-167.gif"/></noscript></span></span> (given by models MDsl and MDs) compared to those computed for the maize data sets, as well as the higher values of the variance components specific to environments (<span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-168.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-169.gif"/></noscript></span></span>) compared to those computed for other environments in this data set, as well as in the maize data sets. For the 12 model-method combinations, the models with the highest prediction accuracy for the environments were MDel and MDe. However, models MU<strong><em>f</em></strong> and MUC had the highest prediction accuracy for each environment and for both methods, GK and GB.</p></div><div id="sec-19" class="subsection"><h3>Wheat data set WHE5</h3><p id="p-59">This data set has sample phenotypic correlations between environments that are close to zero or negative (Table A1, Appendix 3). Only one high phenotypic correlation was observed between environments 5iBN and 5iFN (0.546). <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> shows the high residual variance components of models MMl-GK, MMl-GB, MM-GK and MM-GB, whereas for models incorporating G×E (MDs and MDe with GK and GB and with and without l), the residual variance components were much smaller.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489172/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489172/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">WHEAT WHE5 data set. Estimated variance components for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs) and environment-specific variance G×E deviation model (MDe) with two kernels, GBLUP (GB) and Gaussian kernel (GK), with l and without l (<a id="xref-ref-27-11" class="xref-bibr" href="#ref-27"><strong>Sousa <em>et al.</em> 2017</strong></a>), for grain yield (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><p id="p-61">The variance components of the genetic main effects with GB and l were low (0.064 and 0.061 for MDsl-GB and MDel-GB, respectively), indicating low exchange of information between environments. The most influential variance components were related to the G×E, <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-179.gif"/></noscript></span></span>. For example, for models MDs-GB, the variance component <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-180.gif"/></noscript></span></span> is 0.618 and 0.636 for MDsl-GB, whereas it increases to <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-181.gif"/></noscript></span></span> = 1.482 for MDs-GK and to <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-182.gif"/></noscript></span></span> = 1.49 for MDsl-GK (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>); this result indicates the importance of G×E interaction. The influence of the random component l in this data set is negligible. The variance components related to specific environments are similar for the five environments and for MDe models with and without random component l.</p><p id="p-62">Among the 12 model-method combinations, the best predictive models were MDel-GK and MDe-GK in all locations (<a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 9</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). However, models MDsl-GK and MDs-GK also had relatively high prediction accuracies that were very similar to those of models MDel-GK and MDe-GK. Similar results were found for models with linear kernel GB (<a id="xref-table-wrap-9-2" class="xref-table" href="#T9">Table 9</a>). Models with the random intercept l showed no increase in prediction accuracy (values of <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-183.gif"/></noscript></span></span> close to zero) as compared to models without l.</p><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489169/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489169/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9</span> <span class="caption-title">WHEAT WHE5 data set. Mean Pearson’s correlation (50 partitions) of each environment for random cross-validation CV2, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 5 environments (0iFN, 2iBH, 5iBH, 5iBN, 5iFN) of wheat data set WHE5 for the proposed models MDel-GK, MDel-GB, MUCf-GK, MUCf-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1775730164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the prediction accuracy using Pearson&#x2019;s correlation for each of the 5 environments (0iFN, 2iBH, 5iBH, 5iBN, 5iFN) of wheat data set WHE5 for the proposed models MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GK, MDe&lt;strong&gt;&lt;em&gt;l&lt;/em&gt;&lt;/strong&gt;-GB, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GK, MUC&lt;strong&gt;&lt;em&gt;f&lt;/em&gt;&lt;/strong&gt;-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1347/F4.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/4/1347/F4.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1347/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489222" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-64" class="first-child">Plot of the prediction accuracy using Pearson’s correlation for each of the 5 environments (0iFN, 2iBH, 5iBH, 5iBN, 5iFN) of wheat data set WHE5 for the proposed models MDe<strong><em>l</em></strong>-GK, MDe<strong><em>l</em></strong>-GB, MUC<strong><em>f</em></strong>-GK, MUC<strong><em>f</em></strong>-GB, MDe-GK, MDe-GB, MUC-GK, and MUC-GB.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">The comparison of the prediction accuracy of these 12 model-method combinations with the mean prediction accuracy of models MUC<strong><em>f</em></strong> and MUC (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) indicated the higher mean prediction accuracy of MUC<strong><em>f</em></strong> and MUC over the mean prediction accuracy of the proposed models with (or without) the random effect l. For this data set, the prediction accuracies of CV1 were similar to those found under CV2 (Table S4, Appendix 2).</p><p id="p-66">In summary, the complex G×E interaction in this data set is expressed by the large variance component <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-184.gif"/></noscript></span></span>. Models with random component l did not increase the prediction accuracy of the corresponding models without l (reflected in their values of <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-185.gif"/></noscript></span></span> close to zero). Of the 12 model-method combinations, models MDel-GK and MDe-GK gave the highest prediction accuracies. However, the best predictive models overall and for each environment were MU<strong><em>f</em></strong> and MUC.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><div id="sec-21" class="subsection"><h3>Effect of random component l</h3><p id="p-67">From a statistical perspective, the mixed models can better explain the variation among lines in environments (G×E) by considering two factors: environments and lines. The environmental effects (<span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-186.gif"/></noscript></span></span>) are considered as fixed effects with the relationship<span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-187.gif"/></noscript></span></span>; however, the effects of the lines are considered random in <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-188.gif"/></noscript></span></span> for model MMl. The g is the common random effect of each line derived from the markers and l is considered the random intercept for each line. If we make the transformation <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-189.gif"/></noscript></span></span>, as in <a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a>, then <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-190.gif"/></noscript></span></span><strong>0</strong>, <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-191.gif"/></noscript></span></span> where matrix <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-192.gif"/></noscript></span></span> comprises submatrices (or blocks) where the submatrices off the block diagonal generated the exchange of information between environments with positive correlations. As discussed by <a id="xref-ref-17-11" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a>, this exchange of information is not effective when there are negative correlations between sites (or environments) due to the fact that they are based on <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-193.gif"/></noscript></span></span>. Similarly, if <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-194.gif"/></noscript></span></span>, then <strong><span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-195.gif"/></noscript></span></span></strong>0, <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-196.gif"/></noscript></span></span> and the exchange of information occurred in the submatrices off the block diagonal between the environments with positive correlations and when <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-197.gif"/></noscript></span></span> is not zero. On the other hand, in models MDsl and MDel, the component l has influence only when there is exchange of information across environments and the G×E is simple; otherwise, as in the WHE5 data set, the contribution of l is negligible when the G×E is complex.</p><p id="p-68">The random effects <strong><em>l</em></strong> are independent and identically distributed (iid) thus do not have the possibility of exchanging of information from tested lines to untested lines and therefore do not have any estimate of these values if no evaluation data on a line exists (CV1). Then, when trying to predict values of untested lines, only available information between lines come from the <strong><em>g</em></strong> part of the model. In a number of cases, substantial variation for the <strong><em>l</em></strong> effects were found suggesting that the additive part of the model (<strong><em>g</em></strong>) is not capturing the total genetic value very well. In these cases, since usually the GK method did as well as the GB with <strong><em>l</em></strong> model, there is a major advantage to the GK method in that it can better predict untested genotypes since the marker information is being used in a way that captures more of the genetic variation. On the other hand, if the breeder is concerned about gain from selection following intermating and generating a new population, the breeder should only be selecting based on the additive breeding values and realizing that the breeding values are not the complete genotypic value (commercial value), such that response to selection after intermating will be less than expected based on total genetic variance.</p></div><div id="sec-22" class="subsection"><h3>Effects of including G×E interaction</h3><p id="p-69">In general, results show that when GBLUP is used for prediction under random cross-validation CV2, models MDsl-GB and MDel-GB that incorporate G×E had higher prediction accuracy than models MDs-GB and MDe-GB also with G×E. This improvement depends on <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-198.gif"/></noscript></span></span> and the magnitude of the correlations between environments. For maize data sets (HEL and USP) with positive sample correlations between environments, models MDsl<strong>-</strong>GB and MDel<strong>-</strong>GB had higher prediction accuracy than models MDs-GB and MDe-GB, whereas in wheat data set WHE1, models MDsl and MDel had better prediction accuracy than models MDs and MDe only in environments with positive correlations. Finally, for environments in wheat data set WHE5 with negligible <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-199.gif"/></noscript></span></span>, the accuracy of models MDsl-GB and MDel-GB did not improve much over that of models MDs-GB and MDe-GB without l.</p></div><div id="sec-23" class="subsection"><h3>Effects of including the Gaussian kernel</h3><p id="p-70">In general, models MDs and MDe with the Gaussian kernel (GK) had higher prediction accuracy than models with GB, although these differences were smaller for models MDsl and MDel. When GK models were better than GB models, results show that <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-200.gif"/></noscript></span></span> was negligible for GK models and when the prediction accuracy of MDsl and MDel was only slightly superior to that of models MDs and MDe (as in maize data set HEL). On the contrary, when using GK, the prediction accuracy was not better than when using GB, as in the case of maize data set USP; then the contribution of <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-201.gif"/></noscript></span></span> was important and the prediction accuracy of MDsl and MDel was superior to that of their counterparts MDs and MDe. These results indicate that models with random intercepts are useful when used with the linear kernel (GB) but not when used with the Gaussian kernel (GK). This is because the GK method without l explains most of the genetic variance (additive and epistasis effects) between lines with negligible genetic residuals that are not picked up by the <strong><em>l</em></strong>.</p></div><div id="sec-24" class="subsection"><h3>The effect of the sample covariance among environments</h3><p id="p-71">The behavior of the covariance between observations of the <em>i</em>th line in the <em>j</em>th and <em>j</em>’th environments explains some of the results obtained in the four data sets. The covariance between <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-202.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-203.gif"/></noscript></span></span> of models MM, MDs and MDe is the same; it is determined by the genetic variance component <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-204.gif"/></noscript></span></span>. It would be expected that the estimate of <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-205.gif"/></noscript></span></span> would be proportional to the sample covariance of the observations. This only occurred when the sample covariances were positive because <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-206.gif"/></noscript></span></span> can take only positive values; when the sample covariances between some environments are negative, this distorts the estimations of the genetic variance component (<span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-207.gif"/></noscript></span></span>) and therefore affects the prediction accuracy of the unobserved phenotypes of the lines in the testing set.</p><p id="p-72">On the other hand, when the sample covariance between <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-208.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-209.gif"/></noscript></span></span> of models MMl, MDsl and MDel is determined by the summation <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-210.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-211.gif"/></noscript></span></span>, then the higher <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-212.gif"/></noscript></span></span>, the higher the estimated sample covariance (association) of the lines in environments and, therefore, the higher the prediction accuracy compared with those achieved by models MM, MDs and MDe (without the random effect <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-213.gif"/></noscript></span></span>. Again, the presence of negative sample covariances distorts the behavior of the estimated genetic variance components and this negatively affects the prediction accuracy of these models.</p></div><div id="sec-25" class="subsection"><h3>Models With G×E With the Kronecker product <em>vs.</em> models With G×E With the Hadamard product</h3><p id="p-73">Less restrictive G×E genomic-enabled prediction models that allow any covariance value between environments had better prediction accuracy than models with more restrictive assumptions at the level of association between lines in environments affecting the estimation of the genetic variance components. Less restrictive models consider variance-covariance matrices represented by the Kronecker product of the variances and covariances of the environmental and genetic values (with the linear or non-linear kernels constructed with the markers) (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-8-11" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2017</a>). When a random intercept (<span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-214.gif"/></noscript></span></span> is added to these models based on the Kronecker product (<a id="xref-ref-8-12" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2017</a>), the genomic-enabled prediction accuracy increased for random cross-validation CV2 and for environments with negative sample covariance. These advantages of the G×E genomic-enabled prediction models using the Kronecker product for defining variance-covariance environmental matrices with negative or zero environmental relationship over the Hadamard product defined by models MDsl and MDel are less when sample covariances between environments are all positive. The disadvantages of models with Kronecker products are that defining and measuring environmental stability is not clear, plus they demand higher computing resources compared to G×E genomic-enabled prediction models using the Hadamard product.</p></div><div id="sec-26" class="subsection"><h3>Required computing time for fitting the models</h3><p id="p-74">We performed all the analyses in an Ubuntu Linux server with 256 GB of RAM and 32 CPUs core. To compare the computing time, we counted the mean computing time in seconds for fitting one random partition for random cross-validation CV1 for the maize data set HELIX with the same number of 50 partitions and the same number of iterations in the model. For the models with G×E without l or <strong><em>f</em></strong>, the mean computing time for one random partition was 290, 319, and 3110 for models MDs, MDe, and MUC, respectively. For models with G×E with random intercept l or <strong><em>f</em></strong>, the mean computing time for one random partition was 489, 541, and 4938 for models MDsl, MDel, and MUC<strong><em>f</em></strong>, respectively. The differences in computing time between models MDs and MDe are low, but for model MUC, the required mean computing time needed to fit the model increased 10 times for one random partition.</p></div><div id="sec-27" class="subsection"><h3>Advantages and disadvantages of the proposed models</h3><p id="p-75">In general, G×E genomic-enabled prediction models MDsl and MDel had similar prediction accuracy and, in both cases, environmental stability and G×E can be assessed and measured. Furthermore, in models MDsl and MDel, when the sample correlation among environments is positive, their prediction accuracy is similar or slightly higher than the accuracy achieved with the more flexible Kronecker product models (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-8-13" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2017</a>) for the variance-covariance matrices. The advantage of models MDsl and MDel with the Hadamard product for the variance-covariance is that they can perform highly dimensional matrix operations very fast and, therefore, save time when fitting these models. The BGGE software developed by <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> (2017)</a> is indeed an example of this efficiency for fitting models MDsl and MDel by means of the Hadamard product.</p><p id="p-76">When the main objective is prediction accuracy, we recommend checking for sample covariance (or correlations) between environments before using MDsl and MDel G×E genomic-enabled prediction models. Models MDsl, MDel, MDs and MDe are recommended when the sample correlations are positive and not close to zero. We also recommend fitting models MDsl, and MDel to the training set and estimating the variance component of the random intercept <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-215.gif"/></noscript></span></span>; if it is negligible, only models MDs and MDe should be used. When the number of lines in each environment is not the same, models MDsl, MDel, MDs, and MDe can be efficiently fitted with the BGGE software, whereas models MUC<strong><em>f</em></strong> and MUC of <a id="xref-ref-8-14" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> with an unbalanced number of lines in each environments require intensive computational resources.</p></div><div id="sec-28" class="subsection"><h3>CONCLUSIONS</h3><p id="p-77">Results indicate that when the sample phenotypic correlations between environments were intermediate to moderate (HEL, USP), models with G×E with random intercept l (MDsl, MDe<span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-216.gif"/></noscript></span></span> and Gaussian kernel (GK) had the advantages of other models without their disadvantages. These models allow: (i) finding regions of the chromosomes with environmental stability (<a id="xref-ref-15-10" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-17-12" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> 2015</a>), (ii) the fitted computing time is fast (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Granato <em>et al.</em> 2017</a>), and (iii) increasing the prediction accuracy in the CV2 to a level of the Gaussian kernels of <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> (2016)</a> and <a id="xref-ref-27-12" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> (2017)</a> or other more flexible models such as those used by <a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Burgueño <em>et al.</em> (2012)</a> and <a id="xref-ref-8-15" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a>. For sample low or negative phenotypic correlations like in data sets WHE1, WHE5, the prediction accuracy of model MUC<strong><em>f</em></strong> with GK of <a id="xref-ref-8-16" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2017)</a> is the one that should be used.</p><p id="p-78">Including the random intercept l for each line made it possible to capture some extra genetic variability. Models MDs and MDe assessed the complexity of the genomic G×E present in the two maize data sets (with all environments with positive correlations) by means of the Hadamard product between markers and environments as in models from <a id="xref-ref-15-11" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em> (2014)</a> (MM, and MDs) and <a id="xref-ref-17-13" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em> (2015)</a> (MDe). For the two maize data sets with positive sample correlations among environments, the Hadamard models MM, MDs and MDe with l had similar prediction accuracies as models MUC<strong><em>f</em></strong> and MUC that use a Kronecker product for assessing G×E. The advantage of models MMl, MDsl, and MDel over models MUC<strong><em>f</em></strong> and MUC is shorter computing time when the number of lines in different environments is very unbalanced, as in the case of the two maize data sets.</p><p id="p-79">For the two wheat data sets, the number of lines in each environment is the same. However, in view of the fact that the sample correlation among environments is not positive for all pair-wise environment combinations, using models MM, MDs and MDe with or without l is less favorable than using models MUC<strong><em>f</em></strong> and MUCwith a Kronecker product for modeling G×E. The reduced prediction accuracy of the Hadamard product models <em>vs.</em> the Kronecker product models indicated the flexibility of models MUC<strong><em>f</em></strong> and MUC for assessing complex G×E multi-environment data sets. Regardless of: (i) whether l is included or not, and (ii) the type of data set at hand (with more or less complex G×E) and the balanced or unbalanced data structure, the prediction accuracy of the Gaussian kernel was better than the prediction accuracy of the linear kernel GBLUP for all four data sets.</p></div></div><div class="section app" id="app-1"><h2 class="">APPENDIX 1</h2><div class="section" id="sec-29"><h3>The multi-environment main genotypic effect model (MM)</h3><p id="p-80">The multi-environment model (MM) considers the fixed effects of environments (<span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-217.gif"/></noscript></span></span>), as well as the random genetic effects across environments (g)<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-218.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-219.gif"/></noscript></span></span>is a vector with the observations <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-220.gif"/></noscript></span></span> of the <em>j</em><sup>th</sup> environment <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-221.gif"/></noscript></span></span>each of size <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-222.gif"/></noscript></span></span>, such that one line in one environment represents the <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-223.gif"/></noscript></span></span> observation of the i<sup>th</sup> line <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-224.gif"/></noscript></span></span> in the <em>j</em><sup>th</sup> environment. The scalar <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-225.gif"/></noscript></span></span>is a general mean and the vector 1 is of size <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-226.gif"/></noscript></span></span> The fixed effects of the environment for the data used in this study are modeled with the incidence matrix of the environments <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-227.gif"/></noscript></span></span> of order <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-228.gif"/></noscript></span></span>, where the parameters to be estimated are the intercept for each environment (<span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-229.gif"/></noscript></span></span>) with the vector <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-230.gif"/></noscript></span></span> of order <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-231.gif"/></noscript></span></span>. Incorporating other fixed effects into the model is straightforward.</p><p id="p-81">The random vector of genetic effects <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-232.gif"/></noscript></span></span>follows a multivariate normal distribution with mean zero and a covariance matrix<span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-233.gif"/></noscript></span></span>, that is, <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-234.gif"/></noscript></span></span>, where the vector g  of order <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-235.gif"/></noscript></span></span> represents the genetic random effects across all environments for each line, and the kernel matrix K is a symmetric semidefinite positive matrix constructed with molecular markers of order <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-236.gif"/></noscript></span></span>. If the number of lines is the same in each environment, then <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-237.gif"/></noscript></span></span>; otherwise, when there are different numbers of lines in each environment, n represents the number of unique lines included in the model in some environments. The incidence matrix <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-238.gif"/></noscript></span></span> connects genotypes with phenotypes for each environment, with order <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-239.gif"/></noscript></span></span>. Variance component <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-240.gif"/></noscript></span></span> is the genetic variance of the lines across all environments and represents the sensitivity or environmental stability. Finally, the random errors are assumed to be homoscedastic and independent, <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-241.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-242.gif"/></noscript></span></span> is the error variance.</p></div><div class="section" id="sec-30"><h3>The multi-environment single variance genotype × environment interaction deviation model (MDs)</h3><p id="p-82">This model adds to the MM model the random interaction effects of the environments with the genetic information of the lines (<span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-243.gif"/></noscript></span></span>) (<a id="xref-ref-27-13" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em>, 2017</a>; <a id="xref-ref-15-12" class="xref-bibr" href="#ref-15">Jarquín <em>et al.</em>, 2014</a>):<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-244.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>The vector of random effects G×E interaction, ge, has a multivariate normal distribution, <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-245.gif"/></noscript></span></span>), where (<span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-246.gif"/></noscript></span></span> is the Hadamard product operator, and <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-247.gif"/></noscript></span></span> is the variance component of the G×E interaction. Matrix <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-248.gif"/></noscript></span></span>is a block diagonal constructed with the matrices K <strong>(<span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-249.gif"/></noscript></span></span></strong>…<span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-250.gif"/></noscript></span></span>for each environment; therefore, there is no exchange (borrowing) of information between environments:</p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-251.gif"/></noscript></span></span></div><div class="section" id="sec-31"><h3>Multi-environment environment-specific variance genotype × environment deviation model (MDe)</h3><p id="p-83">The multi-environment, environment-specific variance genotype × environment deviation model (MDe) (<a id="xref-ref-17-14" class="xref-bibr" href="#ref-17">López-Cruz <em>et al.</em>, 2015</a>) differs from MDs in how the random interaction component is modeled:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-252.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where g is the main genetic effect across all the environments and <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-253.gif"/></noscript></span></span> represents the specific genetic effects in each environment such that <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-254.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-255.gif"/></noscript></span></span> is a matrix block diagonal generated with individuals included in each environment:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-256.gif"/></noscript></span></span>with a variance component specific for each environment <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4/1347/embed/mml-math-257.gif"/></noscript></span></span>(<a id="xref-ref-27-14" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em>, 2017</a>).</p></div></div><div class="section app" id="app-2"><h2 class="">APPENDIX 2</h2><div id="T10" class="table pos-anchor"><div class="table-inline"><table frame="void" id="table-10"><col width="12.16%" span="1" id="column-100"/><col width="10.54%" span="1" id="column-101"/><col width="10.55%" span="1" id="column-102"/><col width="10.56%" span="1" id="column-103"/><col width="10.56%" span="1" id="column-104"/><col width="11.72%" span="1" id="column-105"/><col width="10.71%" span="1" id="column-106"/><col width="9.25%" span="1" id="column-107"/><col width="13.95%" span="1" id="column-108"/><thead id="thead-10"><tr id="tr-100"><th valign="bottom" align="left" scope="col" rowspan="1" colspan="1" id="th-105" class="table-left table-vbottom"></th><th valign="bottom" colspan="8" align="center" scope="colgroup" rowspan="1" id="th-106" class="table-center table-vbottom">Proposed models with random effects l and <strong><em>f</em></strong></th></tr><tr id="tr-101"><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-107" class="table-center table-vbottom">Location<a id="xref-fn-9-1" class="xref-table-fn" href="#fn-9"><sup>*</sup></a></th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-108" class="table-center table-vbottom">MMl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-109" class="table-center table-vbottom">MMl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-110" class="table-center table-vbottom">MDsl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-111" class="table-center table-vbottom">MDsl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-112" class="table-center table-vbottom">MDel<strong>-</strong>GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-113" class="table-center table-vbottom">MDel<strong>-</strong>GB</th><th valign="bottom" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-114" class="table-center table-vbottom">MUC<strong><em>f</em></strong>-GK MUC<strong><em>f</em></strong>-GB</th></tr></thead><tbody id="tbody-10"><tr id="tr-102"><td rowspan="1" colspan="1" id="td-897">IP</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-898" class="table-center table-vmiddle">0.571 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-899" class="table-center table-vmiddle">0.439 (0.12)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-900" class="table-center table-vmiddle">0.745 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-901" class="table-center table-vmiddle">0.644 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-902" class="table-center table-vmiddle">0.749 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-903" class="table-center table-vmiddle">0.634 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-904" class="table-center table-vmiddle">0.756 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-905" class="table-center table-vmiddle">0.659 (0.06)</td></tr><tr id="tr-103"><td rowspan="1" colspan="1" id="td-906">NM</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-907" class="table-center table-vmiddle">0.503 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-908" class="table-center table-vmiddle">0.354 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-909" class="table-center table-vmiddle">0.532 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-910" class="table-center table-vmiddle">0.385 (0.11)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-911" class="table-center table-vmiddle">0.525 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-912" class="table-center table-vmiddle">0.365 (0.11)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-913" class="table-center table-vmiddle">0.537 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-914" class="table-center table-vmiddle">0.381 (0.11)</td></tr><tr id="tr-104"><td rowspan="1" colspan="1" id="td-915">PM</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-916" class="table-center table-vmiddle">0.661 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-917" class="table-center table-vmiddle">0.574 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-918" class="table-center table-vmiddle">0.753 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-919" class="table-center table-vmiddle">0.682 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-920" class="table-center table-vmiddle">0.753 (0.04)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-921" class="table-center table-vmiddle">0.685 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-922" class="table-center table-vmiddle">0.751 (0.04)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-923" class="table-center table-vmiddle">0.685 (0.05)</td></tr><tr id="tr-105"><td rowspan="1" colspan="1" id="td-924">SE</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-925" class="table-center table-vmiddle">0.347 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-926" class="table-center table-vmiddle">0.202 (0.11)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-927" class="table-center table-vmiddle">0.505 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-928" class="table-center table-vmiddle">0.370 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-929" class="table-center table-vmiddle">0.513 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-930" class="table-center table-vmiddle">0.366 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-931" class="table-center table-vmiddle">0.489 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-932" class="table-center table-vmiddle">0.349 (0.09)</td></tr><tr id="tr-106"><td rowspan="1" colspan="1" id="td-933">SO</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-934" class="table-center table-vmiddle">0.442 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-935" class="table-center table-vmiddle">0.287 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-936" class="table-center table-vmiddle">0.552 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-937" class="table-center table-vmiddle">0.402 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-938" class="table-center table-vmiddle">0.552 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-939" class="table-center table-vmiddle">0.395 (0.12)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-940" class="table-center table-vmiddle">0.551 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-941" class="table-center table-vmiddle">0.39 (0.09)</td></tr><tr id="tr-107"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-942" class="table-left table-vmiddle"></td><td valign="bottom" colspan="8" align="center" rowspan="1" id="td-943" class="table-center table-vbottom">Proposed models without random effects l and <strong><em>f</em></strong></td></tr><tr id="tr-108"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-944" class="table-center table-vbottom">Location<a id="xref-fn-9-2" class="xref-table-fn" href="#fn-9"><sup>*</sup></a></td><td valign="bottom" align="left" rowspan="1" colspan="1" id="td-945" class="table-left table-vbottom">MMl-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-946" class="table-center table-vbottom">MM-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-947" class="table-center table-vbottom">MDs-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-948" class="table-center table-vbottom">MDs-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-949" class="table-center table-vbottom">MDe-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-950" class="table-center table-vbottom">MDe<strong>-</strong>GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-951" class="table-center table-vbottom">MUC-GK MUC-GB</td></tr><tr id="tr-109"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-952" class="table-center table-vmiddle">IP</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-953" class="table-center table-vmiddle">0.575 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-954" class="table-center table-vmiddle">0.426 (0.11)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-955" class="table-center table-vmiddle">0.752 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-956" class="table-center table-vmiddle">0.607 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-957" class="table-center table-vmiddle">0.755 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-958" class="table-center table-vmiddle">0.618 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-959" class="table-center table-vmiddle">0.758 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-960" class="table-center table-vmiddle">0.641 (0.08)</td></tr><tr id="tr-110"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-961" class="table-center table-vmiddle">NM</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-962" class="table-center table-vmiddle">0.506 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-963" class="table-center table-vmiddle">0.361 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-964" class="table-center table-vmiddle">0.54 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-965" class="table-center table-vmiddle">0.394 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-966" class="table-center table-vmiddle">0.538 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-967" class="table-center table-vmiddle">0.394 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-968" class="table-center table-vmiddle">0.545 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-969" class="table-center table-vmiddle">0.391 (0.1)</td></tr><tr id="tr-111"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-970" class="table-center table-vmiddle">PM</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-971" class="table-center table-vmiddle">0.662 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-972" class="table-center table-vmiddle">0.533 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-973" class="table-center table-vmiddle">0.758 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-974" class="table-center table-vmiddle">0.662 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-975" class="table-center table-vmiddle">0.754 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-976" class="table-center table-vmiddle">0.671 (0.04)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-977" class="table-center table-vmiddle">0.754 (0.04)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-978" class="table-center table-vmiddle">0.669 (0.05)</td></tr><tr id="tr-112"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-979" class="table-center table-vmiddle">SE</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-980" class="table-center table-vmiddle">0.346 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-981" class="table-center table-vmiddle">0.219 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-982" class="table-center table-vmiddle">0.527 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-983" class="table-center table-vmiddle">0.321 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-984" class="table-center table-vmiddle">0.524 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-985" class="table-center table-vmiddle">0.339 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-986" class="table-center table-vmiddle">0.505 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-987" class="table-center table-vmiddle">0.319 (0.11)</td></tr><tr id="tr-113"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-988" class="table-center table-vmiddle">SO</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-989" class="table-center table-vmiddle">0.455 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-990" class="table-center table-vmiddle">0.293 (0.11)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-991" class="table-center table-vmiddle">0.576 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-992" class="table-center table-vmiddle">0.376 (0.1)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-993" class="table-center table-vmiddle">0.555 (0.09)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-994" class="table-center table-vmiddle">0.383 (0.11)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-995" class="table-center table-vmiddle">0.56 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-996" class="table-center table-vmiddle">0.377 (0.1)</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-9"><p id="p-84"><a class="rev-xref" href="#xref-fn-9-1">↵</a><span class="fn-label">*</span> Locations are: IP: Ipiaçú-MG, NM: Nova Mutum-MT, PM: Pato de Minas-MG, SE: Sertanópolis-PR, and SO: Sorriso-MT.</p></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.</span> <span class="caption-title">Maize HEL data set. Mean Pearson’s correlation (50 partitions) of each location for random cross-validation CV1, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="T11" class="table pos-anchor"><div class="table-inline"><table frame="void" id="table-11"><col width="13.7%" span="1" id="column-109"/><col width="10.5%" span="1" id="column-110"/><col width="10.5%" span="1" id="column-111"/><col width="10.5%" span="1" id="column-112"/><col width="10.5%" span="1" id="column-113"/><col width="11.82%" span="1" id="column-114"/><col width="10.53%" span="1" id="column-115"/><col width="9.16%" span="1" id="column-116"/><col width="12.79%" span="1" id="column-117"/><thead id="thead-11"><tr id="tr-114"><th valign="bottom" align="left" scope="col" rowspan="1" colspan="1" id="th-115" class="table-left table-vbottom"></th><th valign="bottom" colspan="8" align="center" scope="colgroup" rowspan="1" id="th-116" class="table-center table-vbottom">Proposed models with random effects l and <strong><em>f</em></strong></th></tr><tr id="tr-115"><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-117" class="table-center table-vbottom">Environment<a id="xref-fn-10-1" class="xref-table-fn" href="#fn-10"><sup>*</sup></a></th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-118" class="table-center table-vbottom">MMl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-119" class="table-center table-vbottom">MMl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-120" class="table-center table-vbottom">MDsl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-121" class="table-center table-vbottom">MDsl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-122" class="table-center table-vbottom">MDel<strong>-</strong>GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-123" class="table-center table-vbottom">MDel<strong>-</strong>GB</th><th valign="bottom" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-124" class="table-center table-vbottom">MUC<strong><em>f</em></strong>-GK MUC<strong><em>f</em></strong>-GB</th></tr></thead><tbody id="tbody-11"><tr id="tr-116"><td rowspan="1" colspan="1" id="td-997">P-LN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-998" class="table-center table-vbottom">0.28 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-999" class="table-center table-vbottom">0.272 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1000" class="table-center table-vbottom">0.307 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1001" class="table-center table-vbottom">0.293 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1002" class="table-center table-vbottom">0.303 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1003" class="table-center table-vbottom">0.286 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1004" class="table-center table-vmiddle">0.294 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1005" class="table-center table-vmiddle">0.286 (0.06)</td></tr><tr id="tr-117"><td rowspan="1" colspan="1" id="td-1006">P-IN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1007" class="table-center table-vbottom">0.304 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1008" class="table-center table-vbottom">0.298 (0.08)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1009" class="table-center table-vbottom">0.335 (0.08)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1010" class="table-center table-vbottom">0.329 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1011" class="table-center table-vbottom">0.335 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1012" class="table-center table-vbottom">0.332 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1013" class="table-center table-vmiddle">0.331 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1014" class="table-center table-vmiddle">0.327 (0.06)</td></tr><tr id="tr-118"><td rowspan="1" colspan="1" id="td-1015">A-LN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1016" class="table-center table-vbottom">0.287 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1017" class="table-center table-vbottom">0.283 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1018" class="table-center table-vbottom">0.305 (0.08)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1019" class="table-center table-vbottom">0.31 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1020" class="table-center table-vbottom">0.303 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1021" class="table-center table-vbottom">0.309 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1022" class="table-center table-vmiddle">0.321 (0.08)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1023" class="table-center table-vmiddle">0.309 (0.07)</td></tr><tr id="tr-119"><td rowspan="1" colspan="1" id="td-1024">A-IN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1025" class="table-center table-vbottom">0.389 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1026" class="table-center table-vbottom">0.386 (0.08)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1027" class="table-center table-vbottom">0.42 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1028" class="table-center table-vbottom">0.413 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1029" class="table-center table-vbottom">0.425 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1030" class="table-center table-vbottom">0.422 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1031" class="table-center table-vmiddle">0.418 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1032" class="table-center table-vmiddle">0.417 (0.07)</td></tr><tr id="tr-120"><td valign="bottom" align="left" scope="row" rowspan="1" colspan="1" id="td-1033" class="table-left table-vbottom"></td><td valign="bottom" colspan="8" align="center" rowspan="1" id="td-1034" class="table-center table-vbottom">Proposed models without random effects l and <strong><em>f</em></strong></td></tr><tr id="tr-121"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-1035" class="table-center table-vbottom">Environment<a id="xref-fn-10-2" class="xref-table-fn" href="#fn-10"><sup>*</sup></a></td><td valign="bottom" align="left" rowspan="1" colspan="1" id="td-1036" class="table-left table-vbottom">MMl-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1037" class="table-center table-vbottom">MM-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1038" class="table-center table-vbottom">MDs-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1039" class="table-center table-vbottom">MDs-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1040" class="table-center table-vbottom">MDe-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1041" class="table-center table-vbottom">MDe<strong>-</strong>GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1042" class="table-center table-vbottom">MUC-GK MUC-GB</td></tr><tr id="tr-122"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1043" class="table-center table-vmiddle">P-LN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1044" class="table-center table-vbottom">0.286 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1045" class="table-center table-vbottom">0.278 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1046" class="table-center table-vbottom">0.305 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1047" class="table-center table-vbottom">0.289 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1048" class="table-center table-vbottom">0.313 (0.08)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1049" class="table-center table-vbottom">0.295 (0.07)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1050" class="table-center table-vmiddle">0.311 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1051" class="table-center table-vmiddle">0.30 (0.06)</td></tr><tr id="tr-123"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1052" class="table-center table-vmiddle">P-IN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1053" class="table-center table-vbottom">0.285 (0.08)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1054" class="table-center table-vbottom">0.313 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1055" class="table-center table-vbottom">0.324 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1056" class="table-center table-vbottom">0.332 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1057" class="table-center table-vbottom">0.324 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1058" class="table-center table-vbottom">0.33 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1059" class="table-center table-vmiddle">0.318 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1060" class="table-center table-vmiddle">0.341 (0.06)</td></tr><tr id="tr-124"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1061" class="table-center table-vmiddle">A-LN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1062" class="table-center table-vbottom">0.262 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1063" class="table-center table-vbottom">0.292 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1064" class="table-center table-vbottom">0.278 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1065" class="table-center table-vbottom">0.313 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1066" class="table-center table-vbottom">0.285 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1067" class="table-center table-vbottom">0.308 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1068" class="table-center table-vmiddle">0.300 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1069" class="table-center table-vmiddle">0.318 (0.07)</td></tr><tr id="tr-125"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1070" class="table-center table-vmiddle">A-IN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1071" class="table-center table-vbottom">0.365 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1072" class="table-center table-vbottom">0.391 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1073" class="table-center table-vbottom">0.395 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1074" class="table-center table-vbottom">0.415 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1075" class="table-center table-vbottom">0.403 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1076" class="table-center table-vbottom">0.417 (0.06)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1077" class="table-center table-vmiddle">0.406 (0.05)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1078" class="table-center table-vmiddle">0.424 (0.07)</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-10"><p id="p-85"><a class="rev-xref" href="#xref-fn-10-1">↵</a><span class="fn-label">*</span> Environments are: Anhumas ideal N (A-IN), Anhumas low N (A-LN), Piracicaba ideal N (P-IN) and Piracicaba low N (P-LN)</p></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2.</span> <span class="caption-title">Maize USP data set. Mean Pearson’s correlation (50 partitions) of each location for random cross-validation CV1, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="T12" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-12"><col width="13.59%" span="1" id="column-118"/><col width="10.41%" span="1" id="column-119"/><col width="10.41%" span="1" id="column-120"/><col width="10.41%" span="1" id="column-121"/><col width="10.41%" span="1" id="column-122"/><col width="11.72%" span="1" id="column-123"/><col width="11.31%" span="1" id="column-124"/><col width="8.2%" span="1" id="column-125"/><col width="13.54%" span="1" id="column-126"/><thead id="thead-12"><tr id="tr-126"><th valign="bottom" align="left" scope="col" rowspan="1" colspan="1" id="th-125" class="table-left table-vbottom"></th><th valign="bottom" colspan="8" align="center" scope="colgroup" rowspan="1" id="th-126" class="table-center table-vbottom">Proposed models with random effects l and <strong><em>f</em></strong></th></tr><tr id="tr-127"><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-127" class="table-center table-vbottom">Environment</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-128" class="table-center table-vbottom">MMl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-129" class="table-center table-vbottom">MMl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-130" class="table-center table-vbottom">MDsl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-131" class="table-center table-vbottom">MDsl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-132" class="table-center table-vbottom">MDel<strong>-</strong>GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-133" class="table-center table-vbottom">MDel<strong>-</strong>GB</th><th valign="bottom" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-134" class="table-center table-vbottom">MUC<strong><em>f</em></strong>-GK MUC<strong><em>f</em></strong>-GB</th></tr></thead><tbody id="tbody-12"><tr id="tr-128"><td rowspan="1" colspan="1" id="td-1079">E1</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1080" class="table-center table-vbottom">0.048 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1081" class="table-center table-vbottom">0.054 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1082" class="table-center table-vbottom">0.545 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1083" class="table-center table-vbottom">0.512 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1084" class="table-center table-vbottom">0.558 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1085" class="table-center table-vbottom">0.510 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1086" class="table-center table-vbottom">0.560 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1087" class="table-center table-vbottom">0.515 (0.06)</td></tr><tr id="tr-129"><td rowspan="1" colspan="1" id="td-1088">E2</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1089" class="table-center table-vbottom">0.397 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1090" class="table-center table-vbottom">0.405 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1091" class="table-center table-vbottom">0.49 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1092" class="table-center table-vbottom">0.476 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1093" class="table-center table-vbottom">0.48 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1094" class="table-center table-vbottom">0.474 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1095" class="table-center table-vbottom">0.472 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1096" class="table-center table-vbottom">0.478 (0.05)</td></tr><tr id="tr-130"><td rowspan="1" colspan="1" id="td-1097">E3</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1098" class="table-center table-vbottom">0.368 (0.07)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1099" class="table-center table-vbottom">0.373 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1100" class="table-center table-vbottom">0.405 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1101" class="table-center table-vbottom">0.366 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1102" class="table-center table-vbottom">0.416 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1103" class="table-center table-vbottom">0.399 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1104" class="table-center table-vbottom">0.413 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1105" class="table-center table-vbottom">0.386 (0.06)</td></tr><tr id="tr-131"><td rowspan="1" colspan="1" id="td-1106">E4</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1107" class="table-center table-vbottom">0.341 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1108" class="table-center table-vbottom">0.329 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1109" class="table-center table-vbottom">0.472 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1110" class="table-center table-vbottom">0.439 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1111" class="table-center table-vbottom">0.467 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1112" class="table-center table-vbottom">0.441 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1113" class="table-center table-vbottom">0.464 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1114" class="table-center table-vbottom">0.450 (0.04)</td></tr><tr id="tr-132"><td valign="bottom" align="left" scope="row" rowspan="1" colspan="1" id="td-1115" class="table-left table-vbottom"></td><td valign="bottom" colspan="8" align="center" rowspan="1" id="td-1116" class="table-center table-vbottom">Proposed models without random effects l and <strong><em>f</em></strong></td></tr><tr id="tr-133"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-1117" class="table-center table-vbottom">Environment</td><td valign="bottom" align="left" rowspan="1" colspan="1" id="td-1118" class="table-left table-vbottom">MMl-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1119" class="table-center table-vbottom">MM-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1120" class="table-center table-vbottom">MDs-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1121" class="table-center table-vbottom">MDs-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1122" class="table-center table-vbottom">MDe-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1123" class="table-center table-vbottom">MDe<strong>-</strong>GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1124" class="table-center table-vbottom">MUC-GK MUC-GB</td></tr><tr id="tr-134"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-1125" class="table-center table-vbottom">E1</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1126" class="table-center table-vbottom">0.066 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1127" class="table-center table-vbottom">0.049 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1128" class="table-center table-vbottom">0.544 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1129" class="table-center table-vbottom">0.472 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1130" class="table-center table-vbottom">0.539 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1131" class="table-center table-vbottom">0.495 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1132" class="table-center table-vbottom">0.571 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1133" class="table-center table-vbottom">0.513 (0.04)</td></tr><tr id="tr-135"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-1134" class="table-center table-vbottom">E2</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1135" class="table-center table-vbottom">0.416 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1136" class="table-center table-vbottom">0.414 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1137" class="table-center table-vbottom">0.476 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1138" class="table-center table-vbottom">0.475 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1139" class="table-center table-vbottom">0.472 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1140" class="table-center table-vbottom">0.464 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1141" class="table-center table-vbottom">0.465 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1142" class="table-center table-vbottom">0.454 (0.05)</td></tr><tr id="tr-136"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-1143" class="table-center table-vbottom">E3</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1144" class="table-center table-vbottom">0.377 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1145" class="table-center table-vbottom">0.384 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1146" class="table-center table-vbottom">0.397 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1147" class="table-center table-vbottom">0.388 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1148" class="table-center table-vbottom">0.423 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1149" class="table-center table-vbottom">0.392 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1150" class="table-center table-vbottom">0.405 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1151" class="table-center table-vbottom">0.381 (0.05)</td></tr><tr id="tr-137"><td valign="bottom" align="center" scope="row" rowspan="1" colspan="1" id="td-1152" class="table-center table-vbottom">E4</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1153" class="table-center table-vbottom">0.339 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1154" class="table-center table-vbottom">0.339 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1155" class="table-center table-vbottom">0.469 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1156" class="table-center table-vbottom">0.437 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1157" class="table-center table-vbottom">0.46 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1158" class="table-center table-vbottom">0.416 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1159" class="table-center table-vbottom">0.456 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1160" class="table-center table-vbottom">0.418 (0.05)</td></tr></tbody></table><div class="table-foot"></div></div><div class="table-caption"><span class="table-label">Table S3.</span> <span class="caption-title">Wheat data set WHE1. Mean Pearson’s correlation (50 partitions) of each location for random cross-validation CV1, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div><div id="T13" class="table pos-anchor"><div class="table-inline"><table frame="void" id="table-13"><col width="13.71%" span="1" id="column-127"/><col width="10.5%" span="1" id="column-128"/><col width="10.5%" span="1" id="column-129"/><col width="10.5%" span="1" id="column-130"/><col width="10.5%" span="1" id="column-131"/><col width="11.82%" span="1" id="column-132"/><col width="10.53%" span="1" id="column-133"/><col width="9.16%" span="1" id="column-134"/><col width="12.78%" span="1" id="column-135"/><thead id="thead-13"><tr id="tr-138"><th valign="bottom" align="left" scope="col" rowspan="1" colspan="1" id="th-135" class="table-left table-vbottom"></th><th valign="bottom" colspan="8" align="center" scope="colgroup" rowspan="1" id="th-136" class="table-center table-vbottom">Proposed models with random effects l and <strong><em>f</em></strong></th></tr><tr id="tr-139"><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-137" class="table-center table-vbottom">Environment<a id="xref-fn-11-1" class="xref-table-fn" href="#fn-11"><sup>*</sup></a></th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-138" class="table-center table-vbottom">MMl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-139" class="table-center table-vbottom">MMl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-140" class="table-center table-vbottom">MDsl-GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-141" class="table-center table-vbottom">MDsl-GB</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-142" class="table-center table-vbottom">MDel<strong>-</strong>GK</th><th valign="bottom" align="center" scope="col" rowspan="1" colspan="1" id="th-143" class="table-center table-vbottom">MDel<strong>-</strong>GB</th><th valign="bottom" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-144" class="table-center table-vbottom">MUC<strong><em>f</em></strong>-GK MUC<strong><em>f</em></strong>-GB</th></tr></thead><tbody id="tbody-13"><tr id="tr-140"><td rowspan="1" colspan="1" id="td-1161">0iFN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1162" class="table-center table-vbottom">0.348 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1163" class="table-center table-vbottom">0.301 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1164" class="table-center table-vbottom">0.601 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1165" class="table-center table-vbottom">0.553 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1166" class="table-center table-vbottom">0.611 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1167" class="table-center table-vbottom">0.555 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1168" class="table-center table-vbottom">0.614 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1169" class="table-center table-vbottom">0.554 (0.03)</td></tr><tr id="tr-141"><td rowspan="1" colspan="1" id="td-1170">2iBN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1171" class="table-center table-vbottom">0.217 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1172" class="table-center table-vbottom">0.201 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1173" class="table-center table-vbottom">0.474 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1174" class="table-center table-vbottom">0.431 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1175" class="table-center table-vbottom">0.47 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1176" class="table-center table-vbottom">0.439 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1177" class="table-center table-vbottom">0.475 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1178" class="table-center table-vbottom">0.448 (0.04)</td></tr><tr id="tr-142"><td rowspan="1" colspan="1" id="td-1179">5iBH</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1180" class="table-center table-vbottom">0.321 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1181" class="table-center table-vbottom">0.35 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1182" class="table-center table-vbottom">0.67 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1183" class="table-center table-vbottom">0.635 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1184" class="table-center table-vbottom">0.668 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1185" class="table-center table-vbottom">0.634 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1186" class="table-center table-vbottom">0.679 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1187" class="table-center table-vbottom">0.633 (0.03)</td></tr><tr id="tr-143"><td rowspan="1" colspan="1" id="td-1188">5iBN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1189" class="table-center table-vbottom">0.163 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1190" class="table-center table-vbottom">0.136 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1191" class="table-center table-vbottom">0.399 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1192" class="table-center table-vbottom">0.353 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1193" class="table-center table-vbottom">0.395 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1194" class="table-center table-vbottom">0.345 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1195" class="table-center table-vbottom">0.401 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1196" class="table-center table-vbottom">0.358 (0.05)</td></tr><tr id="tr-144"><td rowspan="1" colspan="1" id="td-1197">5iFN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1198" class="table-center table-vbottom">0.084 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1199" class="table-center table-vbottom">0.082 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1200" class="table-center table-vbottom">0.334 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1201" class="table-center table-vbottom">0.309 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1202" class="table-center table-vbottom">0.328 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1203" class="table-center table-vbottom">0.306 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1204" class="table-center table-vbottom">0.336 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1205" class="table-center table-vbottom">0.315 (0.04)</td></tr><tr id="tr-145"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1206" class="table-left table-vmiddle"></td><td valign="bottom" colspan="8" align="center" rowspan="1" id="td-1207" class="table-center table-vbottom">Proposed models without random effects l and <strong><em>f</em></strong></td></tr><tr id="tr-146"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1208" class="table-center table-vmiddle">Environment<a id="xref-fn-11-2" class="xref-table-fn" href="#fn-11"><sup>*</sup></a></td><td valign="bottom" align="left" rowspan="1" colspan="1" id="td-1209" class="table-left table-vbottom">MMl-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1210" class="table-center table-vbottom">MM-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1211" class="table-center table-vbottom">MDs-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1212" class="table-center table-vbottom">MDs-GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1213" class="table-center table-vbottom">MDe-GK</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1214" class="table-center table-vbottom">MDe<strong>-</strong>GB</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1215" class="table-center table-vbottom">MUC-GK MUC-GB</td></tr><tr id="tr-147"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1216" class="table-center table-vmiddle">0iFN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1217" class="table-center table-vbottom">0.341 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1218" class="table-center table-vbottom">0.288 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1219" class="table-center table-vbottom">0.61 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1220" class="table-center table-vbottom">0.562 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1221" class="table-center table-vbottom">0.612 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1222" class="table-center table-vbottom">0.557 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1223" class="table-center table-vbottom">0.625 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1224" class="table-center table-vbottom">0.558 (0.04)</td></tr><tr id="tr-148"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1225" class="table-center table-vmiddle">2iBN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1226" class="table-center table-vbottom">0.205 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1227" class="table-center table-vbottom">0.216 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1228" class="table-center table-vbottom">0.478 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1229" class="table-center table-vbottom">0.439 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1230" class="table-center table-vbottom">0.473 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1231" class="table-center table-vbottom">0.436 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1232" class="table-center table-vbottom">0.476 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1233" class="table-center table-vbottom">0.429 (0.06)</td></tr><tr id="tr-149"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1234" class="table-center table-vmiddle">5iBH</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1235" class="table-center table-vbottom">0.323 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1236" class="table-center table-vbottom">0.333 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1237" class="table-center table-vbottom">0.67 (0.02)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1238" class="table-center table-vbottom">0.624 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1239" class="table-center table-vbottom">0.662 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1240" class="table-center table-vbottom">0.627 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1241" class="table-center table-vbottom">0.680 (0.03)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1242" class="table-center table-vbottom">0.638 (0.03)</td></tr><tr id="tr-150"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1243" class="table-center table-vmiddle">5iBN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1244" class="table-center table-vbottom">0.171 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1245" class="table-center table-vbottom">0.163 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1246" class="table-center table-vbottom">0.397 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1247" class="table-center table-vbottom">0.357 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1248" class="table-center table-vbottom">0.405 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1249" class="table-center table-vbottom">0.356 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1250" class="table-center table-vbottom">0.407 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1251" class="table-center table-vbottom">0.354 (0.05)</td></tr><tr id="tr-151"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1252" class="table-center table-vmiddle">5iFN</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1253" class="table-center table-vbottom">0.107 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1254" class="table-center table-vbottom">0.114 (0.06)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1255" class="table-center table-vbottom">0.33 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1256" class="table-center table-vbottom">0.311 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1257" class="table-center table-vbottom">0.329 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1258" class="table-center table-vbottom">0.307 (0.05)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1259" class="table-center table-vbottom">0.337 (0.04)</td><td valign="bottom" align="center" rowspan="1" colspan="1" id="td-1260" class="table-center table-vbottom">0.303 (0.04)</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-11"><p id="p-86"><a class="rev-xref" href="#xref-fn-11-1">↵</a><span class="fn-label">*</span> Environments are described by a sequence of codes: 0i, 2i and 5i denote the number of irrigation; B/F denotes whether the planting system was ‘bed’ (B) or ‘flat’ (F); N/H denotes whether planting date was normal (N) or late (H, simulating heat).</p></li></ul></div></div><div class="table-caption"><span class="table-label">Table S4.</span> <span class="caption-title">Wheat data set WHE5. Mean Pearson’s correlation (50 partitions) of each location for random cross-validation CV1, for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs), environment-specific variance G×E deviation model (MDe), multi-environment unstructured covariance models (MUC and MUCf) with two kernels, GBLUP (GB) and Gaussian kernel (GK) for grain yield with the proposed random effect l and without the random effect l (standard deviation in parentheses)</span><div class="sb-div caption-clear"></div></div></div></div><div class="section app" id="app-3"><h2 class="">APPENDIX 3</h2><div id="T14" class="table pos-anchor"><div class="table-inline"><table frame="void" id="table-14"><col width="16.61%" span="1" id="column-136"/><col width="10.28%" span="1" id="column-137"/><col width="12.1%" span="1" id="column-138"/><col width="7.29%" span="1" id="column-139"/><col width="12.26%" span="1" id="column-140"/><col width="10.26%" span="1" id="column-141"/><col width="10.12%" span="1" id="column-142"/><col width="7.17%" span="1" id="column-143"/><col width="13.91%" span="1" id="column-144"/><thead id="thead-14"><tr id="tr-152"><th valign="middle" align="left" scope="col" rowspan="1" colspan="1" id="th-145" class="table-left table-vmiddle"></th><th valign="middle" colspan="8" align="center" scope="colgroup" rowspan="1" id="th-146" class="table-center table-vmiddle">HEL (452 maize lines) (<a id="xref-ref-27-15" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> 2017</a>)</th></tr><tr id="tr-153"><th valign="middle" align="center" scope="col" rowspan="1" colspan="1" id="th-147" class="table-center table-vmiddle">Location<a id="xref-fn-12-1" class="xref-table-fn" href="#fn-12"><sup>*</sup></a></th><th valign="middle" align="center" scope="col" rowspan="1" colspan="1" id="th-148" class="table-center table-vmiddle">Ipiaçú (IP) (247)</th><th valign="middle" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-149" class="table-center table-vmiddle">Nova Mutum (NM) (330)</th><th valign="middle" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-150" class="table-center table-vmiddle">Pato de Minas (PM) (452)</th><th valign="middle" colspan="2" align="center" scope="colgroup" rowspan="1" id="th-151" class="table-center table-vmiddle">Sertanópolis (SE) (367)</th><th valign="middle" align="center" scope="col" rowspan="1" colspan="1" id="th-152" class="table-center table-vmiddle">Sorriso (SO) (330)</th></tr></thead><tbody id="tbody-14"><tr id="tr-154"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1261" class="table-left table-vmiddle">Nova Mutum (NM)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1262" class="table-center table-vmiddle">0.46</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1263" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1264" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1265" class="table-center table-vmiddle">—</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1266" class="table-center table-vmiddle">—</td></tr><tr id="tr-155"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1267" class="table-left table-vmiddle">Pato de Minas (PM)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1268" class="table-center table-vmiddle">0.51</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1269" class="table-center table-vmiddle">0.44</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1270" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1271" class="table-center table-vmiddle">—</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1272" class="table-center table-vmiddle">—</td></tr><tr id="tr-156"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1273" class="table-left table-vmiddle">Sertanópolis (SE)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1274" class="table-center table-vmiddle">0.29</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1275" class="table-center table-vmiddle">0.36</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1276" class="table-center table-vmiddle">0.30</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1277" class="table-center table-vmiddle">—</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1278" class="table-center table-vmiddle">—</td></tr><tr id="tr-157"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1279" class="table-left table-vmiddle">Sorriso (SO)</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1280" class="table-center table-vmiddle">0.43</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1281" class="table-center table-vmiddle">0.48</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1282" class="table-center table-vmiddle">0.39</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1283" class="table-center table-vmiddle">0.38</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1284" class="table-center table-vmiddle">—</td></tr><tr id="tr-158"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1285" class="table-left table-vmiddle"></td><td valign="middle" colspan="8" align="center" rowspan="1" id="td-1286" class="table-center table-vmiddle">USP (739 maize lines) (<a id="xref-ref-27-16" class="xref-bibr" href="#ref-27">Sousa <em>et al.</em> 2017</a>)</td></tr><tr id="tr-159"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1287" class="table-center table-vmiddle">Environment</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1288" class="table-center table-vmiddle">Piracicaba-LN (P-LN) (731)</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1289" class="table-center table-vmiddle">Piracicaba-IN (P-IN) (732)</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1290" class="table-center table-vmiddle">Anhumas-LN (A-LN) (731)</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1291" class="table-center table-vmiddle">Anhumas-IN (L-IN) (737)</td></tr><tr id="tr-160"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1292" class="table-left table-vmiddle">Piracicaba-IN (P-LN)</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1293" class="table-center table-vmiddle">0.54</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1294" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1295" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1296" class="table-center table-vmiddle">—</td></tr><tr id="tr-161"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1297" class="table-left table-vmiddle">Anhumas-LN (P-IN)</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1298" class="table-center table-vmiddle">0.31</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1299" class="table-center table-vmiddle">0.35</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1300" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1301" class="table-center table-vmiddle">—</td></tr><tr id="tr-162"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1302" class="table-left table-vmiddle">Anhumas-IN (A-IN)</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1303" class="table-center table-vmiddle">0.43</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1304" class="table-center table-vmiddle">0.47</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1305" class="table-center table-vmiddle">0.47</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1306" class="table-center table-vmiddle">—</td></tr><tr id="tr-163"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1307" class="table-left table-vmiddle"></td><td valign="middle" colspan="8" align="center" rowspan="1" id="td-1308" class="table-center table-vmiddle">WHE1 (599 wheat lines)</td></tr><tr id="tr-164"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1309" class="table-center table-vmiddle">Location<a id="xref-fn-12-2" class="xref-table-fn" href="#fn-12"><sup>*</sup></a></td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1310" class="table-center table-vmiddle">E1</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1311" class="table-center table-vmiddle">E2</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1312" class="table-center table-vmiddle">E3</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1313" class="table-center table-vmiddle">E4</td><td valign="middle" align="left" rowspan="1" colspan="1" id="td-1314" class="table-left table-vmiddle"></td></tr><tr id="tr-165"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1315" class="table-left table-vmiddle">E2</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1316" class="table-center table-vmiddle">−0.19</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1317" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1318" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1319" class="table-center table-vmiddle">—</td><td valign="middle" align="left" rowspan="1" colspan="1" id="td-1320" class="table-left table-vmiddle"></td></tr><tr id="tr-166"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1321" class="table-left table-vmiddle">E3</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1322" class="table-center table-vmiddle">−0.19</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1323" class="table-center table-vmiddle">0.661</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1324" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1325" class="table-center table-vmiddle">—</td><td valign="middle" align="left" rowspan="1" colspan="1" id="td-1326" class="table-left table-vmiddle"></td></tr><tr id="tr-167"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1327" class="table-left table-vmiddle">E4</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1328" class="table-center table-vmiddle">−0.12</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1329" class="table-center table-vmiddle">0.411</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1330" class="table-center table-vmiddle">0.388</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1331" class="table-center table-vmiddle">—</td><td valign="middle" align="left" rowspan="1" colspan="1" id="td-1332" class="table-left table-vmiddle"></td></tr><tr id="tr-168"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1333" class="table-left table-vmiddle"></td><td valign="middle" colspan="8" align="center" rowspan="1" id="td-1334" class="table-center table-vmiddle">WHE5 (807 wheat lines)</td></tr><tr id="tr-169"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1335" class="table-center table-vmiddle">Location<a id="xref-fn-12-3" class="xref-table-fn" href="#fn-12"><sup>*</sup></a></td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1336" class="table-center table-vmiddle">0iFN</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1337" class="table-center table-vmiddle">2iBN</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1338" class="table-center table-vmiddle">5iBH</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1339" class="table-center table-vmiddle">5iBN</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1340" class="table-center table-vmiddle">5iFN</td></tr><tr id="tr-170"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1341" class="table-left table-vmiddle">2iBN</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1342" class="table-center table-vmiddle">0.166</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1343" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1344" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1345" class="table-center table-vmiddle">—</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1346" class="table-center table-vmiddle">—</td></tr><tr id="tr-171"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1347" class="table-left table-vmiddle">5iBH</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1348" class="table-center table-vmiddle">0.30</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1349" class="table-center table-vmiddle">−0.033</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1350" class="table-center table-vmiddle">—</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1351" class="table-center table-vmiddle">—</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1352" class="table-center table-vmiddle">—</td></tr><tr id="tr-172"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1353" class="table-left table-vmiddle">5iBN</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1354" class="table-center table-vmiddle">−0.10</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1355" class="table-center table-vmiddle">0.122</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1356" class="table-center table-vmiddle">−0.091</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1357" class="table-center table-vmiddle">—</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1358" class="table-center table-vmiddle">—</td></tr><tr id="tr-173"><td valign="middle" align="left" scope="row" rowspan="1" colspan="1" id="td-1359" class="table-left table-vmiddle">5iFN</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1360" class="table-center table-vmiddle">−0.01</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1361" class="table-center table-vmiddle">0.035</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1362" class="table-center table-vmiddle">0.023</td><td valign="middle" colspan="2" align="center" rowspan="1" id="td-1363" class="table-center table-vmiddle">0.546</td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-1364" class="table-center table-vmiddle">—</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-12"><p id="p-87"><a class="rev-xref" href="#xref-fn-12-1">↵</a><span class="fn-label">*</span> Locations in HEL data set are: IP: Ipiaçú-MG, NM: Nova Mutum-MT, PM: Pato de Minas-MG, SE: Sertanópolis-PR, and SO: Sorriso-MT. Locations in USP data set are: IN = ideal Nitrogen; LN = low nitrogen. In WHE5 data set, environments are described by a sequence of codes: 0i, 2i and 5i denote the number of irrigations; B/F denotes whether the planting system was ‘bed’ (B) or ‘flat’ (F); N/H denotes whether planting date was normal (N) or late (H, simulating heat).</p></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1.</span> <span class="caption-title">Table A1. Phenotypic Pearson’s correlations among locations for grain yield for the four data sets HEL (maize), USP (maize), WHE1 (wheat), WHE2 (wheat). For HEL and USP maize data sets, the number in parentheses below each location’s name indicates the number of lines sown. For the two data sets in the wheat experiments (WHE1 and WHE2), the number of wheat lines is given in parentheses</span><div class="sb-div caption-clear"></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-13"><p id="p-88">Communicating Editor: J. Holland</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-11-19"><span class="received-label">Received </span>November 19, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-02-21"><span class="accepted-label">Accepted </span>February 21, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Cuevas <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.1" data-doi="10.2135/cropsci2006.09.0564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling additive x environment and additive x additive x environment using genetic covariance of relatives of wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBurgueno%26rft.auinit1%253DJ.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D311%26rft.epage%253D320%26rft.atitle%253DModeling%2BAdditive%2Bx%2BEnvironment%2Band%2BAdditive%2Bx%2BAdditive%2Bx%2BEnvironment%2BUsing%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.09.0564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.09.0564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244430100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.2" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.3" data-doi="10.2135/cropsci2005.11-0427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnamachari</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modeling genotype × environment interaction using additive genetic covariances of relatives for predicting breeding values of wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1722</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCrossa%26rft.auinit1%253DJ.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1722%26rft.epage%253D1733%26rft.atitle%253DModeling%2BGenotype%2Bx%2BEnvironment%2BInteraction%2BUsing%2BAdditive%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bfor%2BPredicting%2BBreeding%2BValues%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.11-0427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.11-0427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.4" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.5" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.6" data-doi="10.1534/g3.113.008227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Genomic prediction in maize breeding populations with genotyping-by-sequencing. <em>G3: Genes|Genomes|Genetics</em><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1534/g3.113.008227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008227%26rft_id%253Dinfo%253Apmid%252F24022750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTkwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMzQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuberosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extending the marker × environment interaction model for genomic-enabled prediction and genome-wide association analysis in durum wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">2193</span>–<span class="cit-lpage">2209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.8" data-doi="10.1534/g3.116.035584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models. <em>G3: Genes|Genomes|Genetics 7:41–53</em>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1534/g3.116.035584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035584%26rft_id%253Dinfo%253Apmid%252F27793970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjcvMS80MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMzQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.9" data-doi="10.3835/plantgenome2015.04.0025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic Prediction of Genotype × Environment Interaction Kernel Regression Models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.03.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.04.0025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.04.0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.11" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-8.4.1347.12"><div class="cit-metadata unstructured">de los Campos, G., and A. Grüneberg, 2016. MTM (Multiple-Trait Model) package. <a href="http://quantgen.github.io/MTM/vignette.html">http://quantgen.github.io/MTM/vignette.html</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-8.4.1347.13"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2016. BGLR: Bayesian generalized linear regression. R package version 1.0.5: https://CRAN.R.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-8.4.1347.14"><div class="cit-metadata unstructured">Granato, I., J. Cuevas, and F. Luna, 2017. BGGE (Bayesian Genomics G×E). <a href="https://github.com/italo-granato/BGGE/tree/master/R">https://github.com/italo-granato/BGGE/tree/master/R</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.15" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1347.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.16" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemes da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>. Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. <em>G3: Genes|Genomes|Genetics 5(4): 569–582</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.18" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTM0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.19" data-doi="10.1186/s12711&amp;ndash;015&amp;ndash;0178&amp;ndash;5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mota</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype by environment interaction for tick resistence of Hereford and Braford beef cattle using reaction norm models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">3</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711–015–0178–5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711%2526ndash%253B015%2526ndash%253B0178%2526ndash%253B5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711&amp;ndash;015&amp;ndash;0178&amp;ndash;5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.20" data-doi="10.1534/g3.116.027524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oakey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>. <span class="cit-pub-date">2016</span>. Genomic Selection in Multi-environment Crop Trials. G3:Genes|Genomes|Genetics 6: 1313–1326<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1534/g3.116.027524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.027524%26rft_id%253Dinfo%253Apmid%252F26976443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNS8xMzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manès</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>. Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat. <em>G3: Genes|Genomes|Genetics 2(12): 1595–605</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.22" data-doi="10.2135/cropsci2014.08.0577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondalapati</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Meyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pita</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Pedigree-Based Reaction Norm Model for Prediction of Cotton Yield in Multienvironment Trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.08.0577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.08.0577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.08.0577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.23" data-doi="10.1007/s13253-015-0229-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">512</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D20%26rft.spage%253D512%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13253-015-0229-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13253-015-0229-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.24" data-doi="10.2307/2533976"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Analyzing genotype-environment data by mixed models with multiplicative effects.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D53%26rft.spage%253D761%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533976&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XE96500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.25" data-doi="10.1007/s001220050885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Empirical best linear unbiased prediction in cultivar trials using factor analytic variance covariance structure.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075377300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.26" data-doi="10.1017/S0021859605005587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The analysis of crop cultivar breeding and evaluation trials: an overview of current mixed model approaches.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0021859605005587</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D143%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859605005587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859605005587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1347.27" data-doi="10.1534/g3.117.042341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couto</span>  <span class="cit-name-given-names">E. G. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic-enabled prediction in maize using kernel models with genotype × environment interaction.</span> <span class="cit-source">G3</span> <span class="cit-vol">7</span>: <span class="cit-fpage">1995</span>–<span class="cit-lpage">2014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.042341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.042341%26rft_id%253Dinfo%253Apmid%252F28455415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xOTk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzNDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.28" data-doi="10.1534/g3.116.036251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>., <span class="cit-pub-date">2017</span>. Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico. <em>G3: Genes|Genomes|Genetics</em>, <em>7</em>(2), 481–495. //doi.org/<span class="cit-pub-id cit-pub-id-doi">10.1534/g3.116.036251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036251%26rft_id%253Dinfo%253Apmid%252F27903632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi80ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTM0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull.%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1347.30" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1347.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/4/1347.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1335" title="A Genomic Reference Panel for Drosophila serrata" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1367" title="A Genetic Screen Identifies PRP18a, a Putative Second Step Splicing Factor Important for Alternative Splicing and a Normal Phenotype in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/4"><span>Volume 8 Issue 4, April 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/4.cover-source.jpg?itok=wC37V0ox" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1347" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jq5dh_sSnK9TeZTm6B6HOmpZ-gcgXL9_UDucEjNh3zA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/4/1347" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/4/1347" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/4/1347" class="active" data-icon-position="" data-hide-link-title="0">Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489137" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-acIoEon64B4Q0bXdnj3jIiVejOLufX1n0oMn7_TjD9g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1347.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/4/1347" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BLOPkVCg-oAg92mO0Y7CUyDCLQfU4XCK9Vl1cjFFwTg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/4/1347" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1347.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489137" id="node489137--2" data-pisa="ggg;8/4/1347" data-pisa-master="ggg;g3.117.300454" data-apath="/ggg/8/4/1347.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Italo Granato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Fritsche-Neto</span>, <span class="highwire-citation-author" data-delta="3">Osval A. Montesinos-Lopez</span>, <span class="highwire-citation-author" data-delta="4">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="5">Massaine Bandeira e Sousa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1347-1365;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300454 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489137--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Italo Granato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Italo%2BGranato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Granato%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItalo%2BGranato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Italo Granato</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Fritsche-Neto</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Osval A. Montesinos-Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span>Facultad de Telemática, Universidad de Colima, CP 28040 Colima, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-Lopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-Lopez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-Lopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Massaine Bandeira e Sousa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massaine%2BBandeira%2Be%2BSousa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandeira%20e%20Sousa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMassaine%2BBandeira%2Be%2BSousa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/4/1347" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/4/1347&amp;title=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/4/1347&amp;title=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/4/1347&amp;title=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/4/1347&amp;text=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/4/1347&amp;title=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/4/1347&amp;t=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/4/1347&amp;title=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/4/1347&amp;title=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489137/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489137" id="node489137--3" data-pisa="ggg;8/4/1347" data-pisa-master="ggg;g3.117.300454" data-apath="/ggg/8/4/1347.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction Kernel Models with Random Intercepts for Multi-environment Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Italo Granato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Fritsche-Neto</span>, <span class="highwire-citation-author" data-delta="3">Osval A. Montesinos-Lopez</span>, <span class="highwire-citation-author" data-delta="4">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="5">Massaine Bandeira e Sousa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1347-1365;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300454 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489137--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Italo Granato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Italo%2BGranato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Granato%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItalo%2BGranato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Italo Granato</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Fritsche-Neto</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Osval A. Montesinos-Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span>Facultad de Telemática, Universidad de Colima, CP 28040 Colima, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-Lopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-Lopez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-Lopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Massaine Bandeira e Sousa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massaine%2BBandeira%2Be%2BSousa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandeira%20e%20Sousa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMassaine%2BBandeira%2Be%2BSousa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT). Apdo. Postal 6-641, 06600 México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1347&amp;rft.epage=1365&amp;rft.atitle=Genomic-Enabled%20Prediction%20Kernel%20Models%20with%20Random%20Intercepts%20for%20Multi-environment%20Trials&amp;rft.volume=8&amp;rft.issue=4&amp;rft.date=2018-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cuevas%2C+Jaime&amp;rft.au=Granato%2C+Italo&amp;rft.au=Fritsche-Neto%2C+Roberto&amp;rft.au=Montesinos-Lopez%2C+Osval+A.&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Bandeira+e+Sousa%2C+Massaine&amp;rft.au=Crossa%2C+Jos%C3%A9"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489137/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489137/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489137/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489137/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489137/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489137/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489137/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489137/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489137/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489137/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489137/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489137/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/4/1347" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX 1</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX 2</a></li><li class="odd"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX 3</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/4/1347.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/4/1347.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/4/1347.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
