<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/4/1299" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/4/1299.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/4/1299.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/4/1299.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/4/1299.atom" />
<meta name="HW.pisa" content="ggg;8/4/1299" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Systematic Complex Haploinsufficiency-Based Genetic Analysis of Candida albicans Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes" />
<meta name="DC.Identifier" content="10.1534/g3.117.300515" />
<meta name="DC.Date" content="2018-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Glazier et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic interaction analysis is a powerful approach to the study of complex biological processes that are dependent on multiple genes. Because of the largely diploid nature of the human fungal pathogen Candida albicans , genetic interaction analysis has been limited to a small number of large-scale screens and a handful for gene-by-gene studies. Complex haploinsufficiency, which occurs when a strain containing two heterozygous mutations at distinct loci shows a phenotype that is distinct from either of the corresponding single heterozygous mutants, is an expedient approach to genetic interactions analysis in diploid organisms. Here, we describe the construction of a barcoded-library of 133 heterozygous TF deletion mutants and deletion cassettes for designed to facilitate complex haploinsufficiency-based genetic interaction studies of the TF networks in C. albicans . We have characterized the phenotypes of these heterozygous mutants under a broad range of in vitro conditions using both agar-plate and pooled signature tag-based assays. Consistent with previous studies, haploinsufficiency is relative uncommon. In contrast, a set of 12 TFs enriched in mutants with a role in adhesion were found to have altered competitive fitness at early time points in a murine model of disseminated candidiasis. Finally, we characterized the genetic interactions of a set of biofilm related TFs in the first two steps of biofilm formation, adherence and filamentation of adherent cells. The genetic interaction networks at each stage of biofilm formation are significantly different indicating that the network is not static but dynamic." />
<meta name="DC.Contributor" content="Virginia E. Glazier" />
<meta name="DC.Contributor" content="Thomas Murante" />
<meta name="DC.Contributor" content="Kristy Koselny" />
<meta name="DC.Contributor" content="Daniel Murante" />
<meta name="DC.Contributor" content="Marisol Esqueda" />
<meta name="DC.Contributor" content="Gina A. Wall" />
<meta name="DC.Contributor" content="Melanie Wellington" />
<meta name="DC.Contributor" content="Chiung-Yu Hung" />
<meta name="DC.Contributor" content="Anuj Kumar" />
<meta name="DC.Contributor" content="Damian J. Krysan" />
<meta name="article:published_time" content="2018-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Systematic Complex Haploinsufficiency-Based Genetic Analysis of Candida albicans Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic interaction analysis is a powerful approach to the study of complex biological processes that are dependent on multiple genes. Because of the largely diploid nature of the human fungal pathogen &lt;i&gt;Candida albicans&lt;/i&gt;, genetic interaction analysis has been limited to a small number of large-scale screens and a handful for gene-by-gene studies. Complex haploinsufficiency, which occurs when a strain containing two heterozygous mutations at distinct loci shows a phenotype that is distinct from either of the corresponding single heterozygous mutants, is an expedient approach to genetic interactions analysis in diploid organisms. Here, we describe the construction of a barcoded-library of 133 heterozygous TF deletion mutants and deletion cassettes for designed to facilitate complex haploinsufficiency-based genetic interaction studies of the TF networks in &lt;i&gt;C. albicans&lt;/i&gt;. We have characterized the phenotypes of these heterozygous mutants under a broad range of &lt;i&gt;in vitro&lt;/i&gt; conditions using both agar-plate and pooled signature tag-based assays. Consistent with previous studies, haploinsufficiency is relative uncommon. In contrast, a set of 12 TFs enriched in mutants with a role in adhesion were found to have altered competitive fitness at early time points in a murine model of disseminated candidiasis. Finally, we characterized the genetic interactions of a set of biofilm related TFs in the first two steps of biofilm formation, adherence and filamentation of adherent cells. The genetic interaction networks at each stage of biofilm formation are significantly different indicating that the network is not static but dynamic.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/04/01" />
<meta name="citation_mjid" content="ggg;8/4/1299" />
<meta name="citation_id" content="8/4/1299" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/4/1299" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/4/1299.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/4/1299.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/4/1299.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300515" />
<meta name="citation_pmid" content="29472308" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1299" />
<meta name="citation_lastpage" content="1314" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Virginia E. Glazier" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642" />
<meta name="citation_author" content="Thomas Murante" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642" />
<meta name="citation_author" content="Kristy Koselny" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642" />
<meta name="citation_author" content="Daniel Murante" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642" />
<meta name="citation_author" content="Marisol Esqueda" />
<meta name="citation_author_institution" content="Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA" />
<meta name="citation_author" content="Gina A. Wall" />
<meta name="citation_author_institution" content="Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA" />
<meta name="citation_author" content="Melanie Wellington" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642" />
<meta name="citation_author" content="Chiung-Yu Hung" />
<meta name="citation_author_institution" content="Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA" />
<meta name="citation_author" content="Anuj Kumar" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Michigan 48109" />
<meta name="citation_author" content="Damian J. Krysan" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6330-3365" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=S. Amorim-Vaz;citation_author=E. Delarze;citation_author=F. Ischer;citation_author=D. Sanglard;citation_author=A. T. Coste;citation_title=Examining the virulence of Candida albicans transcription factor mutants using Galleria mellonella and mouse infection models.;citation_pages=367;citation_volume=6;citation_year=2015;citation_pmid=25999923;citation_doi=10.3389/fmicb.2015.00367" />
<meta name="citation_reference" content="citation_author=J. Azadmanesh;citation_author=A.M. Gowen;citation_author=P.E. Creger;citation_author=N.D. Schafer;citation_author=J.R. Blankenship;citation_year=2017;citation_pmid=28951491;citation_doi=10.1534/g3.117.300224" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Bharucha;citation_author=Y. Chabrier-Rosello;citation_author=T. Xu;citation_author=C. Johnson;citation_author=S. Sobczynski;citation_title=A large-scale complex haploinsufficiency-based genetic interaction screen in Candida albicans: analysis of the RAM network during morphogenesis.;citation_pages=e1002058-e1002058;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=22103005;citation_doi=10.1371/journal.pgen.1002058" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Chaillot;citation_author=M. A. Cook;citation_author=J. Corbeil;citation_author=A. Sellam;citation_title=Genome-wide screen for haploinsufficient cell size genes in the opportunistic yeast Candida albicans.;citation_volume=7;citation_year=2017;citation_issue=2;citation_pmid=28040776;citation_doi=10.1534/g3.116.037986" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=I. A. Cleary;citation_author=A. L. Lazzell;citation_author=C. Monteagudo;citation_author=D. P. Thomas;citation_author=S. P. Saville;citation_title=BRG1 and NRG1 form a novel feedback circuit regulating Candida albicans hypha formation and virulence.;citation_pages=557-573;citation_volume=85;citation_year=2012;citation_issue=3;citation_pmid=22757963;citation_doi=10.1111/j.1365-2958.2012.08127.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=B. VanderSluis;citation_author=E. N. Koch;citation_author=A. Baryshnikova;citation_author=C. Pons;citation_title=A global genetic interaction network maps a wiring diagram of cellular function;citation_pages=aaf1420-aaf1420;citation_volume=353;citation_year=2016;citation_issue=6306;citation_pmid=27708008;citation_doi=10.1126/science.aaf1420" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=N. Dabas;citation_author=J. Morschhauser;citation_title=Control of ammonium permease expression and filamentous growth by the GATA transcription factors GLN3 and GAT1 in Candida albicans.;citation_volume=6;citation_year=2007;citation_issue=5;citation_pmid=17369441;citation_doi=10.1128/EC.00307-06" />
<meta name="citation_reference" content="citation_journal_title=Microbiol. Spectr.;citation_author=J. V. Desai;citation_author=A. P. Mitchell;citation_title=Candida albicans biofilm development and Igenetic control.;citation_pages=3;citation_volume=3;citation_year=2015;citation_doi=10.1128/microbiolspec.MB-0005-2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Deutschbauer;citation_author=D. F. Jaramillo;citation_author=M. Proctor;citation_author=J. Kumm;citation_author=M. E. Hillenmeyer;citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.;citation_volume=169;citation_year=2005;citation_issue=4;citation_pmid=15716499;citation_doi=10.1534/genetics.104.036871" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=J. S. Finkel;citation_author=W. Xu;citation_author=D. Huang;citation_author=E. M. Hill;citation_author=J. V. Desai;citation_title=Portrait of Candida albicans adherence regulators.;citation_pages=e1002525-e1002525;citation_volume=8;citation_year=2012;citation_issue=2;citation_pmid=22359502;citation_doi=10.1371/journal.ppat.1002525" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=E. P. Fox;citation_author=C. K. Bui;citation_author=J. E. Nett;citation_author=N. Hartooni;citation_author=M. C. Mui;citation_title=An expanded regulatory network temporally controls Candida albicans biofilm formation.;citation_pages=1226-1239;citation_volume=96;citation_year=2015;citation_issue=6;citation_pmid=25784162;citation_doi=10.1111/mmi.13002" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=M. Gerami-Nejad;citation_author=L. F. Zacchi;citation_author=M. McClellan;citation_author=K. Matter;citation_author=J. Berman;citation_title=Shuttle vectors for facile gap repair cloning and integration into a neutral locus in Candida albicans.;citation_volume=159;citation_year=2013;citation_issue=3;citation_pmid=23306673;citation_doi=10.1099/mic.0.064097-0" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=V. E. Glazier;citation_author=T. Murante;citation_author=D. Murante;citation_author=K. Koselny;citation_author=Y. Liu;citation_title=Genetic analysis of the Candida albicans biofilm transcription factor network using simple and complex haploinsufficiency.;citation_pages=e1006948;citation_volume=13;citation_year=2017;citation_issue=8;citation_doi=10.1371/journal.pgen.1006948" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=B. Haarer;citation_author=S. Viggiano;citation_author=M. A. Hibbs;citation_author=O. G. Troyanskaya;citation_author=D. C. Amberg;citation_title=Modeling complex genetic interactions in a simple eukaryotic genome: actin displays a rich spectrum of complex haploinsufficiencies.;citation_volume=21;citation_year=2007;citation_issue=2;citation_pmid=17167106;citation_doi=10.1101/gad.1477507" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=A. D. Hernday;citation_author=S. M. Noble;citation_author=Q. M. Mitrovich;citation_author=A. D. Johnson;citation_title=Genetics and molecular biology in Candida albicans.;citation_pages=737-758;citation_volume=470;citation_year=2010;citation_pmid=20946834;citation_doi=10.1016/S0076-6879(10)70031-8" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. A. Hickman;citation_author=G. Zeng;citation_author=A. Forche;citation_author=M. P. Hirakawa;citation_author=D. Abbey;citation_title=The ‘obligate diploid’ Candida albicans forms mating-competent haploids.;citation_pages=55-59;citation_volume=494;citation_year=2013;citation_issue=7435;citation_pmid=23364695;citation_doi=10.1038/nature11865" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=O. R. Homann;citation_author=J. Dea;citation_author=S. M. Noble;citation_author=A. D. Johnson;citation_title=A phenotypic profile of the Candida albicans regulatory network.;citation_pages=e1000783-e1000783;citation_volume=5;citation_year=2009;citation_issue=12;citation_pmid=20041210;citation_doi=10.1371/journal.pgen.1000783" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. T. Kelly;citation_author=D. M. MacCallum;citation_author=S. D. Clancy;citation_author=F. C. Odds;citation_author=A. J. Brown;citation_title=The Candida albicans CaACE2 gene affects morphogenesis, adherence and virulence.;citation_pages=969-983;citation_volume=53;citation_year=2004;citation_issue=3;citation_pmid=15255906;citation_doi=10.1111/j.1365-2958.2004.04185.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. Kelly;citation_author=K. J. Kwon-Chung;citation_title=A zinc finger protein from Candida albicans is involved in sucrose utilization.;citation_pages=222-232;citation_volume=174;citation_year=1992;citation_issue=1;citation_pmid=1729210;citation_doi=10.1128/jb.174.1.222-232.1992" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Med.;citation_author=J. R. Köhler;citation_author=A. Casadevall;citation_author=J. Perfect;citation_title=The spectrum of fungi that infects humans.;citation_volume=5;citation_year=2014;citation_issue=1;citation_pmid=25367975;citation_doi=10.1101/cshperspect.a019273" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Li;citation_author=S. Cornelius;citation_author=Y.-Y. Liu;citation_author=L. Wang;citation_author=A.-I. Barabasi;citation_title=The fundamental advantages of temporal networks;citation_pages=1042-1046;citation_volume=358;citation_year=2017;citation_issue=6366;citation_pmid=29170233;citation_doi=10.1126/science.aai7488" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=Y. Li;citation_author=C. Su;citation_author=X. Mao;citation_author=F. Cao;citation_author=J. Chen;citation_title=Roles of Candida albicans Sfl1 in hyphal development.;citation_volume=6;citation_year=2007;citation_issue=11;citation_pmid=17715361;citation_doi=10.1128/EC.00199-07" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=M. B. Lohse;citation_author=M. Gulati;citation_author=A. Valle Arevalo;citation_author=A. Fishburn;citation_author=A. D. Johnson;citation_title=Assessment and optimizations of Candida albicans in vitro biofilm assays.;citation_volume=61;citation_year=2017;citation_issue=5;citation_pmid=28289028;citation_doi=10.1128/AAC.02749-16" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. M. Luscombe;citation_author=M. M. Babu;citation_author=H. Yu;citation_author=M. Snyder;citation_author=S. A. Teichmann;citation_title=Genomic analysis of regulatory network dynamics reveals large topological changes.;citation_pages=308-312;citation_volume=431;citation_year=2004;citation_issue=7006;citation_pmid=15372033;citation_doi=10.1038/nature02782" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Mangan;citation_author=U. Alon;citation_title=Structure and function of the feed-forward loop network motif.;citation_volume=100;citation_year=2003;citation_issue=21;citation_pmid=14530388;citation_doi=10.1073/pnas.2133841100" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mani;citation_author=R. P. St Onge;citation_author=J. L. Hartman;citation_author=G. Giaever;citation_author=F. P. Roth;citation_title=Defining genetic interaction.;citation_volume=105;citation_year=2008;citation_pmid=18305163;citation_doi=10.1073/pnas.0712255105" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. Martinez;citation_author=P. O. Ljungdahl;citation_title=Divergence of Stp1 and Stp2 Transcription Factors in Candida albicans Places Virulence Factors Required for Proper Nutrient Acquisition under Amino Acid Control;citation_pages=9435-9446;citation_volume=25;citation_year=2005;citation_issue=21;citation_pmid=16227594;citation_doi=10.1128/MCB.25.21.9435-9446.2005" />
<meta name="citation_reference" content="citation_journal_title=mSphere;citation_journal_abbrev=mSphere;citation_author=N. Nguyen;citation_author=M. M. F. Quail;citation_author=A. D. Hernday;citation_title=An Efficient, Rapid, and Recyclable System for CRISPR-Mediated Genome Editing in Candida albicans;citation_pages=e00149-17-e00149-17;citation_volume=2;citation_year=2017;citation_issue=2;citation_pmid=28497115;citation_doi=10.1128/mSphereDirect.00149-17" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. J. Nobile;citation_author=E. P. Fox;citation_author=J. E. Nett;citation_author=T. R. Sorrells;citation_author=Q. M. Mitrovich;citation_title=A recently evolved transcriptional network controls biofilm development in Candida albicans.;citation_pages=126-138;citation_volume=148;citation_year=2012;citation_issue=1-2;citation_pmid=22265407;citation_doi=10.1016/j.cell.2011.10.048" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=S. M. Noble;citation_author=A. D. Johnson;citation_title=Strains and Strategies for Large-Scale Gene Deletion Studies of the Diploid Human Fungal Pathogen Candida albicans;citation_pages=298-309;citation_volume=4;citation_year=2005;citation_issue=2;citation_pmid=15701792;citation_doi=10.1128/EC.4.2.298-309.2005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. M. Noble;citation_author=S. French;citation_author=L. A. Kohn;citation_author=V. Chen;citation_author=A. D. Johnson;citation_title=Systematic screens of a Candida albicans homozygous deletion library decouple morphogenetic switching and pathogenicity.;citation_pages=590-598;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20543849;citation_doi=10.1038/ng.605" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=J. Oh;citation_author=E. Fung;citation_author=U. Schlecht;citation_author=R. W. Davis;citation_author=G. Giaever;citation_title=Gene annotation and drug target discovery in Candida albicans with a tagged transposon mutant collection.;citation_pages=e1001140-e1001140;citation_volume=6;citation_year=2010;citation_issue=10;citation_pmid=20949076;citation_doi=10.1371/journal.ppat.1001140" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=C. Onyewu;citation_author=F. L. Wormley;citation_author=J. R. Perfect;citation_author=J. Heitman;citation_title=The Calcineurin Target, Crz1, Functions in Azole Tolerance but Is Not Required for Virulence of Candida albicans;citation_pages=7330-7333;citation_volume=72;citation_year=2004;citation_issue=12;citation_pmid=15557662;citation_doi=10.1128/IAI.72.12.7330-7333.2004" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=J. C. Pérez;citation_author=C. A. Kumamoto;citation_author=A. D. Johnson;citation_title=Candida albicans commensalism and pathogenicity are intertwined traits directed by a tightly knit transcriptional regulatory circuit.;citation_pages=e1001510;citation_volume=11;citation_year=2013;citation_issue=3;citation_pmid=23526879;citation_doi=10.1371/journal.pbio.1001510" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. Reinke;citation_author=J. C. Chen;citation_author=S. Aronova;citation_author=T. Powers;citation_title=Caffeine targets TOR complex I and provides evidence for a regulatory link between the FRB and kinase domains of Tor1p.;citation_volume=281;citation_year=2006;citation_issue=42;citation_pmid=16923813;citation_doi=10.1074/jbc.M603107200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Saputo;citation_author=K. L. Norman;citation_author=T. Murante;citation_author=B. N. Horton;citation_author=J. D. L. C. Diaz;citation_title=Complex haploinsufficiency-based genetic analysis of the NDR/Lats Kinase Cbk1 provides insight into its multiple functions in Candida albicans.;citation_volume=203;citation_year=2016;citation_issue=3;citation_pmid=27206715;citation_doi=10.1534/genetics.116.188029" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=S. Saputo;citation_author=Y. Chabrier-Rosello;citation_author=F. C. Luca;citation_author=A. Kumar;citation_author=D. J. Krysan;citation_title=The RAM network in pathogenic fungi.;citation_volume=11;citation_year=2012;citation_issue=6;citation_pmid=22544903;citation_doi=10.1128/EC.00044-12" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=A. Schweizer;citation_author=S. Rupp;citation_author=B. N. Taylor;citation_author=M. Röllinghoff;citation_author=K. Schröppel;citation_title=The TEA/ATTS transcription factor CaTec1p regulates hyphal development and virulence in Candida albicans.;citation_pages=435-445;citation_volume=38;citation_year=2000;citation_issue=3;citation_pmid=11069668;citation_doi=10.1046/j.1365-2958.2000.02132.x" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=T. Shekhar-Guturja;citation_author=W. A. Tebung;citation_author=H. Mount;citation_author=N. Liu;citation_author=J. R. Köhler;citation_title=Beauvericin potentiates azole activity via inhibition of multidrug efflux, blocks Candida albicans morphogenesis, and Is effluxed via Yor1 and circuitry controlled by Zcf29.;citation_volume=60;citation_year=2016;citation_pmid=27736764;citation_doi=10.1128/AAC.01959-16" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=M. J. Spiering;citation_author=G. P. Moran;citation_author=M. Chauvel;citation_author=D. M. Maccallum;citation_author=J. Higgins;citation_title=Comparative transcript profiling of Candida albicans and Candida dubliniensis identifies SFL2, a C. albicans gene required for virulence in a reconstituted epithelial infection model.;citation_volume=9;citation_year=2010;citation_issue=2;citation_pmid=20023067;citation_doi=10.1128/EC.00291-09" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Stearns;citation_author=D. Botstein;citation_title=Unlinked noncomplementation: isolation of new conditional-lethal mutations in each of the tubulin genes of Saccharomyces cerevisiae.;citation_pages=249-260;citation_volume=119;citation_year=1988;citation_issue=2;citation_pmid=3294100" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=G. Lesage;citation_author=G. D. Bader;citation_author=H. Ding;citation_author=H. Xu;citation_title=Global Mapping of the Yeast Genetic Interaction Network;citation_pages=808-813;citation_volume=303;citation_year=2004;citation_issue=5659;citation_pmid=14764870;citation_doi=10.1126/science.1091317" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=M. A. Uhl;citation_author=M. Biery;citation_author=N. Craig;citation_author=A. D. Johnson;citation_title=Haploinsufficiency-based large-scale forward genetic analysis of filamentous growth in the diploid human fungal pathogen C.albicans;citation_pages=2668-2678;citation_volume=22;citation_year=2003;citation_issue=11;citation_pmid=12773383;citation_doi=10.1093/emboj/cdg256" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Vandeputte;citation_author=F. Ischer;citation_author=D. Sanglard;citation_author=A. T. Coste;citation_title=In vivo systematic analysis of Candida albicans Zn2-Cys6 transcription factors mutants for mice organ colonization.;citation_pages=e26962-e26962;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=22073120;citation_doi=10.1371/journal.pone.0026962" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=E. M. Vasicek;citation_author=E. L. Berkow;citation_author=S. A. Flowers;citation_author=K. S. Barker;citation_author=P. D. Rogers;citation_title=UPC2 is universally essential for azole antifungal resistance in Candida albicans.;citation_volume=13;citation_year=2014;citation_issue=7;citation_pmid=24659578;citation_doi=10.1128/EC.00221-13" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=V. K. Vyas;citation_author=M. I. Barrasa;citation_author=G. R. Fink;citation_title=A Candida albicans CRISPR system permits genetic engineering of essential genes and gene families;citation_pages=e1500248-e1500248;citation_volume=1;citation_year=2015;citation_issue=3;citation_pmid=25977940;citation_doi=10.1126/sciadv.1500248" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=D. Xu;citation_author=B. Jiang;citation_author=T. Ketela;citation_author=S. Lemieux;citation_author=K. Veillette;citation_title=Genome-wide fitness test and mechanism-of-action studies of inhibitory compounds in Candida albicans.;citation_pages=e92-e92;citation_volume=3;citation_year=2007;citation_issue=6;citation_pmid=17604452;citation_doi=10.1371/journal.ppat.0030092" />
<meta name="citation_reference" content="citation_journal_title=Virulence;citation_author=Q. R. Xu;citation_author=L. Yan;citation_author=Q. Z. Lv;citation_author=M. Zhou;citation_author=X. Sui;citation_title=Molecular genetic techniques for gene manipulation in Candida albicans.;citation_pages=507-520;citation_volume=5;citation_year=2014;citation_issue=4;citation_doi=10.4161/viru.28893" />
<meta name="citation_reference" content="citation_journal_title=Journal of biology (Online);citation_journal_abbrev=J Biol;citation_author=L. V. Zhang;citation_author=O. D. King;citation_author=S. L. Wong;citation_author=D. S. Goldberg;citation_author=A. H. Tong;citation_title=Motifs, themes and thematic maps of an integrated Saccharomyces cerevisiae interaction network.;citation_pages=6-6;citation_volume=4;citation_year=2005;citation_issue=2;citation_pmid=15982408;citation_doi=10.1186/jbiol23" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=S. Znaidi;citation_author=A. Nesseir;citation_author=M. Chauvel;citation_author=T. Rossignol;citation_author=C. d’Enfert;citation_title=A comprehensive functional portrait of two heat shock factor-type transcriptional regulators involved in Candida albicans morphogenesis and virulence.;citation_pages=e1003519;citation_volume=9;citation_year=2013;citation_issue=8;citation_pmid=23966855;citation_doi=10.1371/journal.ppat.1003519" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Systematic Complex Haploinsufficiency-Based Genetic Analysis of Candida albicans Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="twitter:description" content="Genetic interaction analysis is a powerful approach to the study of complex biological processes that are dependent on multiple genes. Because of the largely diploid nature of the human fungal pathogen Candida albicans , genetic interaction analysis has been limited to a small number of large-scale screens and a handful for gene-by-gene studies. Complex haploinsufficiency, which occurs when a strain containing two heterozygous mutations at distinct loci shows a phenotype that is distinct from either of the corresponding single heterozygous mutants, is an expedient approach to genetic interactions analysis in diploid organisms. Here, we describe the construction of a barcoded-library of 133 heterozygous TF deletion mutants and deletion cassettes for designed to facilitate complex haploinsufficiency-based genetic interaction studies of the TF networks in C. albicans . We have characterized the phenotypes of these heterozygous mutants under a broad range of in vitro conditions using both agar-plate and pooled signature tag-based assays. Consistent with previous studies, haploinsufficiency is relative uncommon. In contrast, a set of 12 TFs enriched in mutants with a role in adhesion were found to have altered competitive fitness at early time points in a murine model of disseminated candidiasis. Finally, we characterized the genetic interactions of a set of biofilm related TFs in the first two steps of biofilm formation, adherence and filamentation of adherent cells. The genetic interaction networks at each stage of biofilm formation are significantly different indicating that the network is not static but dynamic." />
<meta name="og-title" property="og:title" content="Systematic Complex Haploinsufficiency-Based Genetic Analysis of Candida albicans Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/4/1299" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic interaction analysis is a powerful approach to the study of complex biological processes that are dependent on multiple genes. Because of the largely diploid nature of the human fungal pathogen Candida albicans , genetic interaction analysis has been limited to a small number of large-scale screens and a handful for gene-by-gene studies. Complex haploinsufficiency, which occurs when a strain containing two heterozygous mutations at distinct loci shows a phenotype that is distinct from either of the corresponding single heterozygous mutants, is an expedient approach to genetic interactions analysis in diploid organisms. Here, we describe the construction of a barcoded-library of 133 heterozygous TF deletion mutants and deletion cassettes for designed to facilitate complex haploinsufficiency-based genetic interaction studies of the TF networks in C. albicans . We have characterized the phenotypes of these heterozygous mutants under a broad range of in vitro conditions using both agar-plate and pooled signature tag-based assays. Consistent with previous studies, haploinsufficiency is relative uncommon. In contrast, a set of 12 TFs enriched in mutants with a role in adhesion were found to have altered competitive fitness at early time points in a murine model of disseminated candidiasis. Finally, we characterized the genetic interactions of a set of biofilm related TFs in the first two steps of biofilm formation, adherence and filamentation of adherent cells. The genetic interaction networks at each stage of biofilm formation are significantly different indicating that the network is not static but dynamic." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
    <title>Systematic Complex Haploinsufficiency-Based Genetic Analysis of Candida albicans Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"MtOYYpAYTiyHAKAy7AsVbokOnc72Ea_qpztFcGm4VLg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489123","apath":"\/ggg\/8\/4\/1299.atom","pisa":"ggg;8\/4\/1299","ac":{"\/ggg\/8\/4\/1299.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/4\/1299.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1299"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1299"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/4\/1299.abstract":true,"\/content\/8\/4\/1299":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MgOSHL8xG+GL2hzJv9WprjCZ4jwh7QGT1fPV0y2NmQYkQpwrug+qOhyAM8RyN8SADLDsE7owRr+xG+Fdj\\\\\\\/00bTNg2Zi+Fue4dyMdMWLIIJKEb2ZBzSfek6L9C1J\\\\\\\/4Qr8yQvWGClNNOahdRgQexkd\\\\\\\/2SJ2RtT83bziCvqfnxp4ma+gzX7i2CArMd9J1kF\\\\\\\/Yg0BpYYFSqh2MJCHt0+BVMTtk6gv3CFS2ZWRnvmYU66sXIATqa0sn2rxBpNrg7rZOy4ysGrdylXEsjb2xhx69YdOOy1mN8DNuGNEmgGfog5hHtHqa6o0SZGTO5nqyJQAeKDY3KN78kfr3JEm7B4VXoDTRdNeq2btBPXKpCPR8hUvpQgV3jtg5vOVb9K7RhRATmU1Re7QFlA6EherOKKjHinD3Yp9FQSSCLmBTZ\\\\\\\/57O5UQ++2\\\\\\\/lqMjjcyDU+ydTCJEPKyGci5SN2heDgsmxC7dYK70x8QJUVtyAyjwo2WJr1oFzLBCUFwO7+RURamRbw0slTA6Vss4twnpEVjSw9056bX+owm3e9MU1oLwt+Z+TWlt7YuDeeTxwvTIfEKgQRLeJymWS3s5b1MDMeGwomSfz90gVkcpJ3lWVR4qtogux\\\\\\\/fdzwrzttN5z9L3X+CMw9NovjdmJyD4e\\\\\\\/+DnQnw+E05OBmuqqxyDVr2NoAMvzTMby8OHMOh6y1KhmcWT34TLb+htcq+CQQEbyqc99rwHkCTHZ4HBL00XoKAKPTtHR\\\\\\\/ZtF0vPr4cpCmn2M3BCCa107NL26XcBnSLuJkpSTeZFpknqemh9o56RiF4TGWQASrVM5rGxLwXERGAIGDBnlgE0OiKc2nlEU0pUEt5JaEz4jFycifVxvPevQeg55DXtqY3VQ7x5qUqFXkQv0swZH5q9SOaY53D\\\\\\\/zFuXBjWxJ1HzYpBVqtWqHb+oN12xVTiAUTG9hX\\\\\\\/tOP\\\\\\\/n40Qj4T8KDTi8\\\\\\\/RLrTslxuZY8dBL0PluROVx7Iz8CnHfzsoPP\\\\\\\/y5y1k6ENHyTB1v\\\\\\\/OJbqOrgcdtttshziH52QRl4agFQWBsI9RHGdDFvMYNUr+qGv5tGfFIVjei932Bgo7pLngPuPpQ+DQfBfAFjXMpLQZmktg7Awhj9IpE89PrjWLb\\\\\\\/IiNEGFR+5WJvQiLHXgpRjbmJDDmJ+ocSC1nOXb+VMEIAMXAgKGY9OWBBFAIQ+Jq04ATmjZX2GEQHvALPbtHfJaADCYiXr3vXwSwjH3nwhqH6gxlFGfFB5L0qMXkBnk1MrClpCYePaaNYprHHPZODKbGMhNOyv78wXX\\\u0022,\\\u0022iv\\\u0022:\\\u0022AwShyojVCZSwKAxNdMXCaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225c4706ee03335f98912b62903946db12b9053d760a1f23ffb960e795dbc888b4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kZIv7d\\\\\\\/+igJUmSTnH0mdZ4V4wtpYkeLO1AFUzpSYePyjDtThFAUN0kgF3g7lWxnX4pPQKKgb+W+E5mPSqFxBUufBCAbal+uIN1bGRKMcr9z90+jX9E0rtDf8QLz6LK8\\\\\\\/YmWz4TsH9ncP5Q76ZGIretWgxF0BKSOJaWf1tFxkrU7zvmRo81HMR6XtCADWymBa+gS4M1iwZ1FkDhWPMPJ5Ov0XU4AdkAFSGMXS7y\\\\\\\/nyjKNXQcoTkK3mJLii5FshtUxgRosLMtz5kB43Rl1OGZHyM\\\\\\\/yJcHsKHXtYtq2N3XeVn9LRwvBqM95u9zm0xxGfay1XtXSxXKkboK3sNuXRoBxNWcEdXvO9OCkMo9Vu6HlpB0K8AUZW5DPD2AjGgoVZnVmMvrKOaDYyw\\\\\\\/kLlnNEZRgdIEcRcFQz+xE9cRsnhdIac2Xuh1xz4BXmvpfrl4g0eXW1cJHvO\\\\\\\/iV4CeJVxBz3NHrswn5OazyE673d\\\\\\\/EPoqbgtiFdl3GQUYv7+SCqrNiuN3xoOTsiEJBVJrO1EHj+69Z30O2R\\\\\\\/omsoB7Da0YiT1yvp\\\\\\\/KjnAxYVYxySg1Wg+r6QrPEwAlb4Tp++zTY0fkY9zegcFSkUQn\\\\\\\/aNRcqQOvGuyGVrTQaSmEAdeN61W8dM4C+4k\\\\\\\/6mkZsUcS\\\\\\\/gBIKFRC1q\\\\\\\/wGQyRJUCRTOMC3iu5785MStnGlV6S5G1o32MS29dY3DcJ1dGZYvQP2IIxK\\\\\\\/vAKXg1FXPNOvhppH1g1EdHcN4pJGiVQ7Tse83CyMu7TdV3WoqmYbwFJfR3GclpXajDtEBwbqXemGPOh9Xv0k+ski1Kt9rwRjyjJFUm63oBV8PXgbb+rZTCx3IVLLwaE0mBK5M6L2F3jmv4gHphvTbMCfdERJn4HDwS8ZIj1085TlJppA97SMXpdwZF8YFn\\\\\\\/PIBFiOAzdeiLV6OOzmKzjxlewDamwzpcb5ak5VV9EZcXbtnZmLkfI3+EzUevv4RGOzNaXXFLjjd17LgtPcOTe7A5iwaF5S+5x\\\\\\\/QN6pYfOVSjpqEXPm\\\\\\\/0mqkFwhdUPEKtu6bEffU3j\\\\\\\/YNWqWViEPLoGTEexuUhR5NGbKs\\\\\\\/w95fQqbVpEGYCOSYxsMbp\\\\\\\/c8IxVF10HC78UQ1wiidD5jS7fVIb7SM0+nymZa2rBtDwwhcaGgeOklshXmmeZMj1RZvlI9elrrN0nqFsMm9jDvHjoPnyJY4HCFcfeClpvolzRcoDZIqCkEgQjm1zolvsX\\\\\\\/\\\\\\\/bUToyhtQdQY7WgJxmEdd+Zc+\\\\\\\/gm7AvmfMH7nU5MW3cwLIdSJqmfLoQwU6PCtIyBWm6VAhF6nWK1gLKt4xpzRhVsBHPCVP6yRcByjmGH6oATPsE3PlSZ5XtKz3oVqCosMNaMSTbi3GvXhWEsAXd2HHKjFuSRYfInMMhJY7fLConvbOTUxiNPoCtFWfHblCqE3z9pSAbpFV5Ef9Nv1hHSSLSnXoPVawXXpda14nyjpuDAf\\\\\\\/HbXruXjb00hPhHI6XtlzG3jZvH\\\\\\\/7H8IDknq9bG9r5qPH6SyIJ\\\\\\\/+Rhso8Yrqbz85C8yxEPKECcgYSkm1ICA4HSeTQyEFVBwEp4ErPyRVnL8MoCnb6WK98xzWTRaLfjuQyNpYE4f5LU7LZrI\\\\\\\/nsMM2OHXRNPOYk5ytwxT+mlLU5w5totGFpKHjGez0L11Qjbx5emgRnwM\\\\\\\/k+hRv2nRSixGtF6L399cTQKgQN5UAfWVdpGAhlGAIM0DBncnbmcoti9+GzcMptKnDjon80CvzuuHKaS\\\\\\\/9aMAGfL6JDrS5jUcrVX+3BxrzPMQ7wBtduBrWPSsO4mJBBK39xkuARKa2byTMvZd2aJKF0mdvnv9vpmwG1wt0GFempYzrrd45cUCPyrFztGYetdpMBQf++oiox3k9OOv1xgkhw0m82LZ\\\\\\\/FEnqTEcG0ATGfBPWcoDARmYUMurqzoni8+2qtMs3Dk6V3TQzFXdrm2kZ66q7s8WCe9s0vgAfcL3+nssDt5xUQdQppWLmQXOZ0OyINC7Eh9WsZKNdNq8xhJDX8NUiebWJXdRYcgQMAT1AT1ZO8xVU8jqxwBk5MHefkDFlDta5Re\\\\\\\/UYZYw3D5VO8HnSJ4XUUaQ+ehwqp6hn8sErEk\\\\\\\/NEIqU1dLnZT7\\\\\\\/AeFM+n2iCOwbVgPMNimlsX7N2x7Pf0XN91dCVmXSevuzGXa1QX1kCANzaEt1r760ZmjHPXZ1P7D3s4YKQ3fEhlLHhO1rXwZmpyNM\\\\\\\/ZOj97mgLKqauxvX5pYgB2jtwFob14YLktzpWkSLN6bjBJK8lS2eIOjV7zWL+ZqWSoBbEUkAhK5daqnM7dt77HmbtpJr\\\\\\\/lqh\\\\\\\/\\\\\\\/nj1KjFOGDhIZrLWPzKdHzGXkGCyIbsbFJD9zMH3jgjAJnz5HEBFVbfclyTyKeuQNI+SiTqJxmy4I0h8OMSa807gzVFF0pxfxreJyXTJbwrPMaLf2k04QW+kc2f1xSPnbO4q8Bun7ut+U1JFxBixD32JMJAbQccW19xgwS0e+jfZPakZDiE\\\\\\\/WJS4Q8PElwhOHwLQSG\\\\\\\/g6rh2K22A94vZfiROSRIkml56TE5EKmqJgb70txzj2fXgPa1dzcSh1xEM91J8RFeAh+rlRqr0jlBdrmLZFYTk8OSTmC9DDT2vtbvMlEBrMM6Htyirwz7vNMliGj5WqziX2Kv102wS2HTPNPMQd8B8Lqppg4zxYPF6fG2Pe8HfzAVtivD6BMJ7i7jrxQnRUtlnJKZadYeKflcQ+crqpAv4EcgiTlBMtC6OYjd7tYGebNUVzEB3fu7dnYSOUMhzFITkSqdIUWvlnXnGnEj2Zusd0bNTPoQ6nvY5tVBJ5j3+Wxu1U\\\\\\\/dMZahEm1Sgl+U3+6K5Jdkw5l6mHpiA0iT5Ls0C0iBB3WqQtaFeCCqwhdKtSkEVIdAA7T+WKxOn3n72rCqwhe29a1vtaGOnw68TFIjVATQON4WZTO6nVSHWH3mf1TUPspquIal31Wr\\\\\\\/X6epN1lYkRquo4eXWa9NP5+i3cM5rbj\\\\\\\/b5EWHZXQB+hILIwtRIvLwrDmAT26KFjiGN1cEVEDzTTgag9ZOYHEYMcGv4W\\\\\\\/lI6sPyx9QTMAUwfof1ZbG\\\\\\\/zd+Hc6x\\\\\\\/xb+CBe+91HzcHG+uT0EjmmPr1K+3Ze2yIgh8AjBQpkrlF4Y5E25o+z4Hc4zqRAQQ9XCqUN1CxCtjsSCtwnRneJywW8ZWYFyZvQYofaa0t53IxE9rEPq3fy0yXxnz+nEccXF6YyiSl76wn6VwzI3oELiL\\\\\\\/Uvd\\\\\\\/9NG7TKe6dVCm6erxjKNoGF4GFpQGw4SwCCwTBmLcKzxfFGiR1GiqmFnQzQTkIpQRZ0mYyiqbI4BzrJsLMhsAIP8\\\\\\\/twViXYje8LuXcDm4AhNkkbZX5YR21MTjj6KYlSHS62xfmBzMMrg1FRl7tCpJrH2D5Nt13gIoEQj4OARZuBnfRGg934AyhJLuLSkh1LfqKCzjvKPdtX+BlhJzPAcvkQuAgBjR6aRyqPNo+PEn+yKxP2u5Fb0g+bfDp1rLRJdfbb5n+1jbRzdF1P\\\\\\\/DQq2cZR\\\\\\\/CWq\\\\\\\/7CyszdaDlqO8FtSXoddno0c5jRLtGLCKgxo8Q\\\\\\\/WWRtj9\\\\\\\/7KWl90Gs87sOWYqrvGBkbZ0K1oy6EuB2FF\\\\\\\/LGsHDbCT8dzNwEP25\\\\\\\/6BsmP77lMXxRmTBRXsnNXwVEfbUE1NnmGqZkGeRpFwq756n2GTqP0aBQUmPs9yAbHFB5ltv6ip1TC3jLQ5SgtLUhGExxAxRB2mFljTE8J0n1XScf45k9iLxlmb6jLEpgc\\\\\\\/SrchxM3oeyabvte\\\\\\\/\\\\\\\/qkWsxfOfRmjn\\\\\\\/ptyTKb7eDP\\\\\\\/BKHoLufcPW50QsSvEsV1XEro1B1HeR6UhFTYDPXp46trsigOCHZDOiFbo\\\\\\\/m0dGyo7Ofi\\\\\\\/qKaBHvrbuFlbAspbJLvTUoTst+Bb7Q+mbKHr8P0tPyqI\\\\\\\/zIGmOjUoS4uUhJk+UoyW7WppOKBFEsg5u1X\\\\\\\/q5ABPYxaXYd11CQbRVLrvyWmldj+M7QbMzfFgeG\\\\\\\/Pvmu2PCL4EKU48EEWhMHi2wLSvvpcno2cxdJj7n5232vaUq5imwdrJseQ7R7+kdZL8X+4Zhr8NxBCwZa+SeRZmzXxMzB62eFfn3XGfleu9sRQzOBlfRzOe14McwA1gYOtSiyhjzSuKhU9g4YY77Hlu1jBmmNxhq1ZU0SKZPKb5WxVJLCLPZhOVu+N68QISLoq37umlpAPFBKnymiFjxHsMU0QodddaiWuJg2gJwDiS0bOZI6pjSNkpyY8wghBqf4ne+GKxeiE4USZerZPzj8EWR6MqOHKPZ31S9FJljEaZpub82ExJf2GCpnXcnX63\\\\\\\/MwmHz\\\\\\\/Lmmfjhc7ubzA4yWfl8HkMID\\\\\\\/l6M433KmW5qx0+wqBa24Nuq7W7aQA0kF\\\\\\\/C14qqyq9QGfzaKaYuyMHbjl3rerZ48MuHx2sabXzgaQqc+uV3gq9dIC9x9YSLyuBIYFpniK753ZrdHMs9xAh\\\\\\\/DSy4hcKNvjezEVZ\\\\\\\/bM7Gkn16DVcvnkEMg3Th2qTNF1lNH23GeL\\\\\\\/ebAUP24fUb1uvr8xxYFA4SwqvCKoAslnnr6U\\\\\\\/9fwJ4QTJPE7IYu7KZ3tCuZwGPIv1\\\\\\\/G+rWMPNUMycy\\\\\\\/AWBTsubf73uEo2Ud+bCj7SM2pwlFgnH8gikgde3PuTtSjm995GRUl\\\\\\\/MYY9sa9QB62Af1iJzB78cs\\\\\\\/tQVoduhZ\\\\\\\/xnjGIIuK\\\\\\\/jxFL5R1JWPyoSQsn0ac7s55YpdqRgzDuYig0+NXlXszY3DLB9vfLjB5AkybKeQrmABc5unyGCxMj8Zqv9cesti8lqUqEZqmMqcqrgHYvrkrdvaO52\\\\\\\/SBbigBw+vm\\\\\\\/z5Ujb9Akne+lai9H85CGeWk7\\\\\\\/bJL0yHM3Ofpx9SOpNDYOX0cU5QcPPk4HmT7FmJB5BptMyqL+gCRVz0jxEgrCIpFJJn3bh1OwIWEUgbwLlql+urr9\\\\\\\/Wfd1w6tgi8woNjTpPWXYecA1Yy3j4YadLpCF0wgmCXWiYjav1MeVA5xKYIXG8gESgtQYBtbGZRzuzO7pZi463dRxX4k+saX3U8hdUhtIxRAuPgbqxXa4ojVWZ0O9RMsdEA8CXhIthp4pIqA8APke\\\\\\\/KDOgppBz162yUcAVcZcp6Itl1Lm6P8amWcIyHuk29CvanME0r8EJat3MTweGkRYqx+VKYL2emuK5Sof7p9ZCDGiQ\\\\\\\/OZksSgBZL2M2+LU0igbqWPZ8\\\\\\\/8jJ6cIoVQk7ocZl1ftYRIIldnMvUFQJs+6Sxsxll48RomoXiN8W9HmmxIGxwjHM9c6Szec8KuVrOB9FUambcg7TV+s6RRWO+VcdJyLyWwAcSnI7cDJfokbQTivNRfZF9GU1+mbl2iBGdB\\\\\\\/uykNTOxJ\\\\\\\/9Mt1tUZoXL0fkiDcX6DA3isz6e+9O21mwVqZjP0eDRMjer\\\\\\\/tgHrvbPOjmjTTB67FRnoDIFY1DskltVffNOUo3Em7vv53LG3U2pZvowuZihfFc\\\\\\\/EmhwdKK\\\\\\\/lW8k10rsIYZAJt73Y2PbB4S8sUKelA+SilV63LjM6Gi6S\\\\\\\/ezZtmajMmVvPCoaG4LIcSpFcUnx5l34bANWbTBhHUSB7ZpZBQnw1KHIACsR6W8J3r5eOSgGvNcTUjX+aXM+8+7XfmrW4j4jreNYBbx25qbUALbwWnq2TTl8GHWnjri3UAsNxPlz0P2iY2HaZClA9IrLh3TFVX7lSJY\\\\\\\/Ds4xKAtcPIPE33cFDUn4EWU6BMHe9AjguzvdMY6vZ6WXLfOcCuxT0OcVVnp0UWr56njpeZn9bxBjLrK\\\\\\\/tL1ObZdzMd95H+YTg8\\\\\\\/Jx7sDtZPKJPWMdpKR+kxQaflkyvnqaKoifOyAZeOh\\\\\\\/m0JdZoh6U3jOimgueMUoGcdMken36InVJHAE3CS1dUahNOu+yM2G+gDngyDMQKzfYnlM5OGkFHht6bAkD\\\\\\\/HUmI0IXTcw0RjX2hxel7s+rgWUUGamxqVlCIWFvWN9HLkzQBJru8bBytIgctYirhH7vt1dD1JrVrdC3IpEfMc7wYf8g6NTok\\\\\\\/QIeFTCPaw\\\\\\\/yjRLQnSUcrfhI2mrjYnqJnXXfUUaLLuh11MmKG2edIdcftJyHlRUfidtEkOL5nDtnnM03hGkEtzO7IGTcSq6yEDgzeIPbpeJKbv6qBtX1LPFxFXvd7v0FH7ylDSp8H0z1sm+\\\\\\\/EpJQLyaIkuNg0Zztr1hUZztU15FkeyGf5pQ0ZwdjDT1yQ7zR7OqK7gUZeUhb3DFZ3HY+pVGUMp4tkQN\\\\\\\/av+6zqU+5kkA0n\\\\\\\/Bzcymp83WOb9NYqa6HwGrrpugmWmcc0hD4tDluwgfZcc15\\\\\\\/vdt7+k++EajBFwdRA5vo0\\\\\\\/YQUNHESYPm\\\\\\\/vmvq8XPSTo1gwW9OxF4XNKwmCKTWFwULq55gTxEi1xMbCZm4NxN+mTr0hlp1iWTVZfUZ+oRAlGuYVkMcaFKaLb1h8nXKzPwvHTHtUqzXq8A9Yzzl6V0cpfcszOlj4BvOqm2ViP69d\\\\\\\/WgICjUHbUdCL2U1e9nr1AO3\\\\\\\/nWF2Gx8BqXzNpLHyeWGgfM9HoFYmjMJrxUFfEI0l+wX8wcLgmJboy64dgtm8oj\\\\\\\/I40QeIylkw+QoSx9U+QzYdRSdqZzGXc9aLevbybYXccDAt5chCXn9mqI4KUo\\\\\\\/cCCfZ7H9rPQ+tjuwUXKLbMwMO1mg8aaG8+1i1o4lbkJMIO93ymSz+ONQyDKVbkhW6kG4HUYMVjKyeIMbV1A8OJbTSMsLLhiYnyoanfjSPy0mCBpJHGziI6pPquinmEKc7zNBO0ENMMR0f3qtQ9BLYyPefL6rQPRoD8drZS+h1WHvZdQbHb\\\\\\\/T7+uF7AopeGJzEHHVXK5C2iG+XmT1RcpXs7bMo4r5+9AyC8Ydbl65EdxRJh1WwlMyokoqMa7qVQJsR+jcUaPAk93mXjEXPyiFLhp3Pakcq5rsNLWWaLd3s8hJvMyyvdQF5J\\\\\\\/vdiErQ3PGCiuqcIgHDs8\\\\\\\/HZJ2rmKqoTLYTgzieJ8iZYE3KzqBiuCQ74rc60zhYnX2JEUUGwznksxbcc7fLAZ++YxKV7lheL1FtL0cutkr4m\\\\\\\/c4yyP62xy8x\\\\\\\/2MvEAJHvQxnzhGx1bIqeBBXf5S3oEDq\\\\\\\/UwvO0RlOVqGxwphMKyi8eWATmTGBcoHI2lM\\\\\\\/+qvR11wFB+Q5KSayAOEZ18hjNK\\\\\\\/7oF\\\\\\\/UKbLhWbaWx7TOqeY242lk2fdiebRFYoBwFR2A3FOLwBz68QkT3zkV8BC6Qv75OZmc45JQCMjNDQsC3Gk8xUsTcQKh9\\\\\\\/y5PdmkVflRiYpjHQNC39YtXAQWWp5BUSNLUWitjisci4meFlVbV6O3cBZsBpHH8pnsdUWdrm3Hr1u\\\\\\\/jrly1HI+7nB58XOfYaH9a+aCJVB6XXjCOGHj+Q0hUuixaCxZB5+4gFMIf66jFdZ49latMwlyFhBDqotn4npqaZZVMpuEkv0gPYWy4CLHmSPhLMjCjqqV7O0O7jk2a4SZ8DRyCFmWlkqGw6up2+Am6sxtQEXWYHRrOeOA\\\\\\\/4NaWCuANEjWt0Ddr8zaAfMnSm+oAeOKAlC6UIiFnMbj\\\\\\\/Ypu+Ewg976CsAFwksTbHuIh62wUQeXX8ZOFv\\\\\\\/UDQpry\\\\\\\/JghfYtA4xv6p5DodSdw4eaLckPd0IxVbZxngGmT8lPLWp6duYVWvGBEuq4+llwa\\\\\\\/v3Wt7WoE76CxxH8g7d3Btb8kiP1sVkxLXM0dYorlPF3nrpYnPkWorog0jbxgzC20caQDgcU9eZ2BHdIexTATcGkTKzDA2MSjjS2sXZHGcgNAOM+uGNxGriW6\\\\\\\/HodtuzKURJ0LBXjCFpHWiO74GsF2d+8inaIvAhugGRSmIGwaP0EXvyNnwQfAb68\\\\\\\/70r3R\\\\\\\/U8\\\\\\\/Az4NzwSI97qLBMCZ+8PPbavZu\\\\\\\/060+fqVnaqtlBMf\\\\\\\/FGh\\\\\\\/lrfw1cfXBRHpQN0vF9rJQAJ5j8OU93dSyixP24FISlTI5l4PM4XTs\\\\\\\/LunBhptYjG2Mr4UNBaflYO2kBQjFkTITPTInOLHm9DGIJs0HcGkWYmuuvaljhJyE8KnzXSFXdOi1BGTBxMHCKEDUEY8zcT0NwC5pa8KMedPVSQVEePyzgcyzjdwcte717pHXQeqRCgcCYn+nyETv8Vy+T4LFtRwQUzY7GwhgqPfPNf8UC3dibGj6d2sQY0f40OtaZo7LrOaCCydG3lDYkCI\\\\\\\/gYQakwq3tT2yaW8T1AZS3Fy7gv6SefrnZueRpIHzIP9ROye3KlluvHa2Os1mEuP1Fz2dlPANoy+Q00kNeUr1sYjRHCmbCdwYctqR775iGpkg2Qx8b+3zP5yM7sIazVriGXGLEOjLb89vbSq87Euttnq5ymK8JHKx00CE\\\\\\\/AjabchSLnUYO8AXihcfnlCLi+ogVBIC7jvHXz7b841qzRS4zoZH5ihqc+ENgN+Q6sRXJPLzm9Fol18CqUGdooktPBH7gpmr9l2dfbMDDIMSJW+3iK4qpjKJ54+7huLG5XhiSifwCeR5pk1Iy2bR74fOCPPM4yXVwyJK4347AnI314gG3uMH10My7eXaPbKWBHK6F449LcxGvahO4pilVUgfq7M1exxmNu248ZxkLWFNCodDQEds1AnH88qiFkNRhDhRmbZST8owH2xPMBI1ZCvbFoKpcJzFDG\\\\\\\/C8ovzS6LJpJf7khedQ0jUnO4BI9YBzsLEe0Xqth1zi3AIaQXxFqGgYMm2ksNW96\\\\\\\/gw2x9ymTxQcG+LRdg1zmTUQ\\\\\\\/9rFySVEu+tTKdkHkxxQZuaGKgFBPdvG22OPQdGsJwxDMl\\\\\\\/5SCQLYMAgRVgUYROvw3OCH+CqHcA04lYiroB\\\\\\\/SD4gn\\\\\\\/Dz596zQ59MR9skpH1jZQIT8ftL\\\\\\\/eB\\\\\\\/LwM4n\\\\\\\/nYNEFfYRPd9st\\\\\\\/mCboxQD\\\\\\\/a9OcVC+YiRISkGfcMtdXWgMivn6FkrmjJwlVTO74aCcwzg4TjCHh3Xx6k8wxpl7lmaGFoJ1Up03uSunD4u3WSe9bltA1UELYJPd0cNOamLj\\\\\\\/nbuGVBDVry5LB\\\\\\\/KM2HCE+MFRBLgYUimpVWIaKhsTnwbFztbwKSmRYZ3wH+3Ml8pNEP7jh2gUn9jXIL4JYTo5Vimgg53e8vjkb54Ekku8vHLXbDOiJMh\\\\\\\/KowpipvDDJk8wNFJ0NAuJ5N15bC6j04ka1LhtpTJcHgoSWL\\\\\\\/fVIgXTdymzHWyPPB6Hu\\\\\\\/Y++CEZhLI0ybthsiroiIpY5E0nrRLHLmWCUHfd1LvtfuBTP5qa5HZjhDZeZ5FHEEtNPiB343z5owAbkX49Gfbgqw9vi5dAsyH6uYljlJqMbYnm3jF8JDKMMpH90PGolagwL1QS4KDnBWMuC09+05VSrQA0IYH+1qO3Rnz9zGe5qcOwvrINSAUeJ9a7zPzlJyanapO3BQl9hzgwxRfqCBhMMr90crUgMsVnIB2xr3JyS1dnqGdh40Yf1N+iGooa\\\\\\\/fDvoG4n52TxxKTm5qDnISyjf1iP+dFDX9iRV8sDbltKrrK0elgU1DVxiG+tNuE6xLf1xXoHqcZOQuDFhC3RX8JczCUB3VgQfRhe7j+lX5d1kv9szxjvuAH1v9XdWxCAl+LD1uL4k1e1ukvZZy4o1qK2ht1WfGiGPQCR8VXw7MN3bDVqoQLBJzr4zBT2tOJ5MbbPt8ciTZneXzhD4GY0aLzzAXqlmexpIG8EKf2i9KXhWSavvx5IbV3VFVpUzWLbYqE1dLyTYBUj\\\\\\\/cunnQd2SrWq0VSpLy292Q+CYQJUvEYnDP51h158LUPvcLPwUHxFrHO\\\\\\\/YCB4Dm8jD36rMBrtqmA93VHK4dF\\\\\\\/IIgrNFxUtf0JHJ4BCvzZ5E26nOc2g9mTN08JLoVTyU\\\\\\\/XhEbwLPZQex+OKW2g0MrDv4kM+pnkE45YT6rnT9qyA4Pg6ENWPIHHjZTD7YStqpEDt6Mq3H9HJI8hSEljo4aCFIGQdBAI0fubKKCAFmdYcyFOsZdsrNtzuJyNEtjxDrBPuczVUZgKsAHPv3etPxjecks5Es1maIyQsy4JjOr+jgrzyuUAVHBizR\\\\\\\/duzI9oB8T\\\\\\\/KftUid2rK\\\\\\\/xRzjylT6COIpiP3TUz4e5pJbXy1m01eHmzqnRDDjxQ78cvwL1DcGEAr21TkQybUb2IusZQA1Vd\\\\\\\/5SRz+C\\\\\\\/Ox7PfABnEZmxYc4oik9bDvizOwG52qyzTNxOgGSr96cwwPE7oPI1ESbcJNH\\\\\\\/XUyRauFLPmc0r59na9ncijvc9sObZEZ4y+Axlm8dwpJ0NhNg2z8QKiBeFW0WId0uAzxFmgLiYVGLbt2qE2n9T5Mi7uHcmIOJSL5xHRCJrv2\\\\\\\/ESFUkR\\\\\\\/ov\\\\\\\/qs2W7b8gyVnree7tSxv\\\\\\\/hahjXY4NJoazwPvswgcs3zdIk7c5K8hjyQjAKIeYNVqVqZl4Q+Hcl4XIpm3Rt+B5\\\\\\\/1oimsF8rk9zSQelY34rHIJXFzP0xUnwKDRFxxRmev8WIdiLNvJ2NjoPWv6UoxO+ARv45k+XObf\\\\\\\/i4CEHOOHn9dfM9l9tI91sspbsktNld3O1fXHzUpGsDOYRaqVIEGH+zDytGgmlStnGr1yLULKfPn+jUldU5BjOownnsOlR7AjcYwP07R1JY2TmvX88B54YtRodoZrZjehnxhXGa9ctvwk1B5YHnBTBiEmE+rRHWjnTmmvQyYbB2fLj7+lkEcCSc1LUNOvYrRyO27NvzCDNplL2QTKMtjYD3f3pGNkOhYxIkkdPB4kpaYUSzYh0O3Y1MK8hykWGu1YxNqK66bggt8eEX03IVzBtcuRSRsbEa3GFpOfRrbLcrhkSsP2ZBWQLyhL7R70ABmiA1b\\\\\\\/bwYubaA7iOigwoJ5sHFjF2zkJ7AzPeJ2veLtZf95bMcFTryGkZAqtQRUvqPASMmpygqteXBCZZEJfoGjCaicIzhML0ub8Plqgq8c\\\\\\\/xGFhvk9SqNv2lMITDiyxo4VdBwkoXIQQv6vox\\\\\\\/Nrumg2lV71P3FvwIDGXE6BrDt\\\\\\\/Ha0TPhPLU94ZBH3OtgfFtTpty4m5Qw\\\\\\\/hNWPsYhSvkqV228CKEsx2rC+c7V2vWfIiRQ9d7FUlOqSdf87VVTwyIZUk7ZIwyc+3fVW2QPehHaxJ6\\\\\\\/qdjUTawTBxmHTFACS4eICZ86Vbm4DY078GHPe0KDjwYeEMOk0Dxgr0Yq5xQfuM91eYEab0u88x0aSJdjPIDl43x96keevBLhE9Y1WlmyDpUdj8OwmZgAQ9LpG0f7MTjbk+zblzUqlYfO1KxwAfmP2XBSuc+f\\\\\\\/Mdb3KLGuLksjoG4aJ6egLx2J6hnploD\\\\\\\/\\\\\\\/sfpUAZGCNz0eD81\\\\\\\/pYkoFa+irgOUdEFQ0FkdMpfDKMyvLPhAkZxXynNkx0085BRx4iy1A1+W9tYwxEpYh9sJtd4KEYX6DsEfb9G27OEkDHCjNcgA7RRk\\\\\\\/6PMYp\\\\\\\/RUBXDzZzQVpDy3Q0bnh5mgbbydo1cpZSIZfzjd4h0v5NTp+TGxIbQFuyWfGayenumz4UbpjSj4k6KVjo9dtQqDZGTVk3P098HA9rtHsg3WEY6dgoSwjvlLdGW2tlIssJ56WdvAj\\\\\\\/3bzeEYQzumC409L8eRUp7zZEK8Xz2YdDQbpPKAU9qm2f8A+55M9mBfW0Qd33Tnrg9PnYyeN49OdCvCuavfyrbAVarsAdtBk55CC1MJxEfeBFK3YoK3juhMhIZKvNq+ClIjgh59qJFqr3dn02zusZgru1PYcoK3BJRLHWev83623pYHCKUoeychfLtx0KqGV6Pnu5lUdfMgAO\\\\\\\/P2ct8VmosX2lFqn71IKOhEwf\\\\\\\/rKrTDsIYR9L7OW4JplaQyKMcKfkXiOD7ifDvqm8jGIC+a3W2UUVG8Hm3nFyJjVpepYNWT4gzRc7QQUk0JWNaY8s73v1Hm\\\\\\\/JYYgaBrbUVpbwCNVp+NO6M4UoC8LMZvVfTLfiITPbogtMFxe08O7q2ZOvKWbNO5yKT3zavOesT9ci1G+ZVTCIE9U9wXVsFKSU6i3vJQBs5yJSUgGNOz4oKo4higeDfgxOkUvQrsIKY49FKFf0USnAfh0GTz2\\\\\\\/DedRT6Ki5OM2JssyFWs1vWQ85IXxIKS6Ymu4VXcAgXEELn2NXMHy21n84o5fR\\\\\\\/PjRKGpW5oWeFdGpZ3BlVJmVjby6II345BCBntoJNpKZIbM5ls5hzN5kdWoOoJj0a7CIiTNkN7iRV7+vG3l4fQ6eT2h2T9UJ8VyDn5Si5taZvHfYpPIwYiByXVWPucNaLd+BkqZQTvl3tx2q\\\\\\\/A8HeAwldkHHkFm\\\\\\\/\\\\\\\/\\\\\\\/j8Ic+4Q8hxWVw+75vLntV0lKylH5qAWc1gdlJ4dG2bx4nF9QEQMUqZ0mSje28r5hQu8zVqAAC7KE5ozEZfCFsJumAMqqjHxK8wP7bAAuRF8IDTnVRK86E9qgw36S1J4hx9xmknB8NfhmTwi36aaOmsmSpR8j9PgGm3kLUZ5jJXWDINwahVVWGyehyjn3ArnrxtXsQCI\\\\\\\/qpErhmGIsEM+74Hb1+HoRG2LZANjl04V3J7VfnKoMJJwUNPSOqo6QABNVo8k9GDQucQ2D2gbFC48R8WyD7LM+rEbbsHTpnR1Tk\\\\\\\/gTmUrxRRjHD+MsFkjzWIQzv88bWQ1zkWsLDi7deJ5iLQh8Eonm9fogWaT+Vyyyu+OMyyPC6OKcf3sDKnQzpvj2zrSN+83c2lv1o3EfCsoEHxjWT\\\\\\\/KszKPwdnw9c1rSBzM2xxy7ed93aK1UII2n2Ho4YHdR\\\\\\\/jCbB7ZfKEBqqNbn4rF\\\\\\\/o4OK4zC0RVHuTnmHb7HqSjlVZEPJuCSkqLLzJXoksGsRj6iLQvaehsNeY7mw5DTHBQH6hs6PqJ4N\\\\\\\/Chae1IDnffbwR+jwVnd5uBdv+gEqGsyCn5eZSviAAe55HICS\\\\\\\/8FXoTIosHYwabpj0dNV\\\\\\\/fRR5igm6mpNWLHcN0prGu10Y3ot4DmTwrlDfMrQPCcK2u7HvVxBceT3m4Gb2XYd8PNtiv\\\\\\\/Bo4SNfxHyrCXqLLm42oqsvHZ0M8JN+dapdQpzdyPwJLQkTfy3QLSUSthYV7fM\\\\\\\/R0vHwsqLuIxDPT+sbEMvbP44EqEJSP6q0as2tVMH1bkeHM8iW5GOWB700enkpWNQ2+hIWAUJCL0kC26sWcHv4gC7\\\\\\\/cjj3TorpYsYn97JKj0kTFXJ06O\\\\\\\/ioTdAAMUQXGhKHYeaxzMPPPvuGDxFbq0K\\\\\\\/Hbd2TjZIZvJkMe61oq1CgtlXbPzMrMB\\\\\\\/aAewb2xJj3EH04CQSGl8uplOX0KMtrgzpVbOyuuQfU+4xX9aUfBrtdlZd+eVZG9\\\\\\\/4zfnpz52wnVDjXdLmTCCO6uaYJHawm+P2vJop\\\\\\\/p\\\\\\\/L0jNNYYQS52Xt6oq1\\\\\\\/3jWo1ef8PK2uxWSZLFgctsO2iz4Y6RXZzEKTyoQ33qcaAmuypR\\\\\\\/ULzU3QnfgEYjZt2wZ+Uje55UaaoqeSKg5p4l70smcU0a2IQ4xvklfyYbGnDlFoUGYmqY6qh4kPfiNkxZJHDRp\\\\\\\/F85SSpMXCBzvpl3jwOeinY0V2Vgk1kT4T5ftEnQhxdg0rhO8gLbAuZTagTCVEYbfvdPIU5rjZBKZIrRvZHU+GbpkcXTnsvUrMc+TmW\\\\\\\/nDvyzpdrpFOgRqQyf6sZxzE1raQpVjlTHifhaUHycK2ZehlTbuxZCyeAYXDh7O+xhK4h5YCFIkhbzNI4mMZkbmUHuQbfPE2LK8nSpJkFz3AwzPsDBdI1GikVpHPmYXynSFzWDnprG6eUUhrlTMaVVmrJXnFCW9+T2Japqla7IBNC1RMqk6XbfKh6iJUBeHDa\\\\\\\/NMahCtoXEKJR1mv+IjUnOhxaca+\\\\\\\/I\\\\\\\/wuQLzlIAMsrZ2GrnCHPT10JND6hQqWbISlku6aot\\\\\\\/tylFTFeI7okfZZul0Omv9KmBIFDkKpUMdtstZyFNnfdCgvjbx+KOYt7gvuSpzjvptzQUebGpCrZ4lAgHiAExbUeK5le4J+ERRwi0sgWurMhfK5WA3RzUbbP+8RFn9yWW4yncK3bgDbnQhyQscPBU2n1Vwdqlr8mtFiRldlJaWV3O7uMHl\\\\\\\/5W0hKBWb9uMQwG6fPMZ\\\\\\\/uLDHHJzLFplJpinStPOwwM5rnhqM4Pn0y0u6crvPjvdIkSeR8ieDMfvt4VtTEhHFGwWaQHsxpix0m7ZDuX1jqaVnWxit72GaYvAIbtDPP2\\\\\\\/wk4ikBI7C7WzXJqdqBYcNBBWB0rOYJD61JMUQX\\\\\\\/PyQvq6JdL\\\\\\\/TOv6NgXJIS\\\\\\\/6P+cTT5\\\\\\\/FVzsY2Q\\\\\\\/Y3cCoa8\\\\\\\/wAmYqnzPYEtRR0hPTD7A+nD0I6h3efZ1SRMIwHHy0hHAcxoguVbfT86\\\\\\\/r0AA46+\\\\\\\/6+C3pdIfJe+QUmsJ2QekHuNfErbdtp9Gc+Pdil+S+iDh2bgavSy5wKHS\\\\\\\/AmjSOb3Rr37\\\\\\\/S\\\\\\\/vccxHlgEUXuWMLz4KBSbD9+aJf8TneMYfXvpvE7FDnAbKFNJvpzSR5qe2FfQSHfgBbc6X3etrIWXFaEvVTBvkRERK46\\\\\\\/DxRPvlWPwamvHAaZI5vMmfyqopk4lssQ7G0gY\\\\\\\/FXY+RXKGGRLXRpQkigwTcMGAjlhaipf9lSSKhmBe+g\\\\\\\/oy06RGMNyV5sBZf1iQQgQdLRZgTK0\\\\\\\/UIruvWBK+k3bNYk31plHHRCMXzSPu3iuyTbpEbDXqB0w8mUK+akJdAlHaxKjwVB9vEv4K2euwpdnH3HJltAJeoB+cFi42iBO0I2U7ziJiRYa6jo58uX7+MpTdBiYx\\\\\\\/9gqGXj8zRhkuLsy\\\\\\\/VbZthVS2xtyNbecrv+iXO8zZEaP44HIfLhM+kr\\\\\\\/Sa2FmqUdlOyU5Z7fkOJhy8sd9VJqat3ETZ2sS+JjBO2eQy6jwchBB4F\\\\\\\/uFE\\\\\\\/H46e7yWv3CZruT0bZY8FRivra+dQVgIWHRj4Uu2Z9Pbz42YsdzBJd\\\\\\\/1raRZLhPxp3lU2kq7Qx3BqavTAVMSJeqLqu9DxacXCqFJZaoZle9PFLMb6FjbqGmVJNe9brHQ7VyGiHegIsF8lSpc86j3EcRU5O6Oy3VvMB3FyXWd85A9XBLPA8I3Ds8L2c9Abo4FqPLvreIyuZocrRNWRBYiuv+PeFGkASGsbvuXMOwJzgtp\\\\\\\/VlYEpDXecmxtYErODXvCTHGMlbJN7hpcaUjiSNzUgKnUa5R11GAceoNKneupczZRHF6jAKfmqq+ssAXxTQ7wCTxsuRY9nKpnG\\\\\\\/ba8Pdbw4KDX9ywo6abeeDaCJLqpgvFe2kI6XSmhmkBxbD++975s63IWLxXKbNxtZT4KS8tlfxyqWSBsvfRgKQu7rHl4v495x73Gh0bbP5vknSApKMuYO3wPxJrjXgjJqA3\\\\\\\/1j2Ev\\\\\\\/9xjxNl+MyRQYeOu9cjvekI2HvnykQjdVEUb2T8VJEw4RlAnrEKR0t5TAJAdc\\\\\\\/+eFEHY\\\\\\\/JUq4ZB6A0nrEZo4NWyXrJ0xJsqPZ2\\\\\\\/gjvQtyyJHYM+w8pmN7j39eHeYaU46EYI5c+DgfTC2RPPeu81wQhRHYrmRpUr4FcoFhn2KR1\\\\\\\/wzu501D\\\\\\\/H2XtMpnWRO4Xo29ZbSkqrqxUPBPBiOmI3zmA5l3gZulHQaOI31gvK5N4+SiRNo\\\\\\\/WJ+rrAmeqL0XSukwmHauAdVy1eN7ldisigaa11FjLh0kgR9v3OBlfNWJ5BpI2iQy4nbTaeqN0melqHRKI35WDmZ4KpKvBM4iddVVQ7HXYHTRnmtEBn3ve160poKobT7jhd8McJkMcVKHxou4tvTCHURwk62ZtTBWbQca6VeVPseaJJpMlMOwNHjcJgq06hcGl6YkFG\\\\\\\/qMrq02QmTY7XWABUSI1xVsum5vL0b0PS1oaajPoisuFZtpFjsC0rkivGIgypCPJP6+PU\\\\\\\/uMMBrCVhm3BrQIFOdk9D0Q8qVJ06IgoDjAjqqx\\\\\\\/jQaVRuqnqmzx0OmfjXdTcO6W2yOCn3yciDjZDRKyT9rXwGdmNS4HL0TFMiyC+8PcKMpTiu8tgSt0Q1Bddg7v0uWG6FXdK4l3bB75ZOmzkrev4E4O1yPTRj7WAK9J2erNVVcCsSQH\\\\\\\/4ANeqMPxKGt5rX2RrFNIYRBphC3hnTOItc\\\\\\\/Hmmu+sPu6N790mOHPjY6jKvJmkwj+icfV7fbKLd97dvIO+4t2g\\\\\\\/z9Yo+pFMOv99cLabbzT38jiXeFHLvSyQbQlJRuiMCl0vd2S9JhfEprOXHPUYSG3EtRhPwsxDrSlIQqV1OyVL6XHPvX6jTEL0oiI2Bw\\\\\\\/nydr6JCNaKIs0PuFomeFkPywH\\\\\\\/sVuKL6dZavrSYo9h4WBlHl6K+PxgIei5h+yxtxvCvu3G1X70VIgJceSUnkRhMPp+i\\\\\\\/FvIrxGDEDRlqzALHXV6PnwqLuV4yukM1H1iJ1cNJqazCb381JSb2lNIDJ\\\\\\\/+X2AKElFJFBaTZJ+068EfX++a3LQxkrNVgu\\\\\\\/KdH9ylNoi3wOTxTIzbamIWhydpsqepmpO+1j6MpAtgw5cSf9IlfXQWp\\\\\\\/lBRDh+qCwsbA6sRo9XA+W9SKh2Mh2UbnA\\\\\\\/O9xFnVR6HF36PLYsLPKAluUw7xMiiA0wXXeAauiChBGUVyJIXepT+uOaktBQfQPHtnAuB6aj5285Y6z7TNOvopG3uXBi4MKMi+h7BVvo9UEpbAg+JtSCEc74m2O4nbFIukRlwPDp+sh\\\\\\\/0v1soQPiwUqpsX4QLo6aa29eot8UZGfarRwzgK2FRltWkOwJlqn+Z6WlzAaq0SRaJsBmt3PC8zGEp62F3tSc8ICAoPxXMel1fqzA8qI7jrpGEzizttWHIMJexvnMxrfm\\\\\\\/6d3ji+9MrNLfY1+dceBWphjpDWifEHByF9ECyeVTCyIdSLBndX3vmtQw7mx00qzq2e6VZXD9Pyizw\\\\\\\/FqzGceMx4Yc9fIgNnlhPcMagrxnmGnQanqy97YnrnMud3++QMzlmCGeP26zWmr2+oV7EqramvjMD\\\\\\\/SBwVS9d+0dU05yXHcDYG6ndpcxEyE0hi9xB7wx4RoaoZzpvNOAtDqXNL8oIIO5FXuyHe0P6af0mvcZcZLhM427DBHIRahUw0HiJIveUVlzcfrV8\\\\\\\/mtBEvzsJZxN2\\\\\\\/djYEgcYXZxL7t6gMosmBHyXxf9OEwRLn5oGoe8TdSuMV5oNFJvaeiKsnpyFBRXdzOVq70qUz3GRyx8QFXpH7+sqBdfT8T9ewtQwEDw0z8LtKkpmCGmU2eieZhitLfcth2tw7uqsq3xTWyv7XgOD1Vid9sE2MzD0P+V9ID9x\\\\\\\/4HIa2HzcVFGESoEPEQXzPnLZluWuqArEKEUHxNm1XrNjT+MOdzgHH0PdyzLtUlnTJ9FoZ6kk5\\\\\\\/2ujaJ0ywme4qM5gpKpfY9xWj4nnjiwz4LShnH7uKVGbZMd2CccHlBYvtMKPTUgEq1b2E2SeAs1lsB+PcTtpJjRR1+QKB9HoXh3uiUvs+nkRr+D2wAENPP3D\\\\\\\/ZTq\\\\\\\/YojXAJ6oq13OT5fTer269UN8X3w89R0n\\\\\\\/97h6h\\\\\\\/K3fEf+AZUR8RarI2CVUzx5TR5J2Xe3DcHRSKbTePRyKOqUPWHPIMqDiSCwyBgakq4K\\\\\\\/gvMG2c3jnEBcNy3ufSvgsqSqy4wgI+xMIOgl2IwgBZDa8rcM39kiJpvp2N52cRH2CgPMp733AJqWnaudYgWi1VGZ2xHz2mwtkW37vbQk5YtDB2uXVAaWd07aBvYR4SyVkbLV+UMBWZ0chbmnCewpyHoSGgURSC7YmDAlIzyt1w6kVttwzQJJZZokMTzhY48tzze4OOMg4DN6OzKlGEU\\\\\\\/IsyrZblaAJt2yoV08Y\\\\\\\/gb8Xct67nZ5EAr2gwt6jjkYDlnuasZOFVP5RYBkJigmuvw0hQ6TmGeDPQZPVDsLkeYuBKitT03vtJPQGAOH\\\\\\\/OCcmVVROEsXPTQfLne2\\\\\\\/+y1gvhrUzwOnA35W3Wmw8ePeJyEO\\\\\\\/q0IqAlNKYGfcWw4JJGM5bHPxCkemRrnih9XiSkfdz4k3F5OWOlYcqZ0XO422skl5tgp4U0Ut6riyYLoH+DTUy3gx7iIZTxeM2xDhk\\\\\\\/CZ57KV3aVALIBHDmsd5w9BbtkAWl9xOt9o8HTNeuhT0AjawvH9Q5qy4T2Blx2s4KiAFGpc9k\\\\\\\/P\\\\\\\/Ma5huuvsHkMoeaASh\\\\\\\/+Lk3HClgVI404c0BIlWicao2MBn4dkxRXkCEPPlVB5cbdbTnULDjqVvf1L5GlIpLWXufXudhHxW8V+6eKVoICC5frwdHe7c+a5m3GZ366o9KKjD7bvW\\\\\\\/AsSt\\\\\\\/hi\\\\\\\/0vYG5ZWFhunCjoj3e\\\\\\\/cNmKClasm63w8aVoe9HBNiI8y16\\\\\\\/GSIIJFftHqNfGbKbwSdLt3SnydPPLmjQVW2PIHtGbVXBv3BtMFxw3uMwEkyqci44sobowE7jF2c\\\\\\\/hTp+rfzMuelkmkHGvco3nwilvZkA9X3mGQRl\\\\\\\/G0UOv\\\\\\\/w6vl7XNbH0k+IZN7hzrACJVAbU0rx71RT5eQITdY7WsZMBK1UPTVPVoQ22+kA6OVwDlwXHczMnJZJzY5ffUkxOwTmdAMDsgmX3Vf9t4+mAA4cmaMNJTPCr8orjbityGfjyRVXeZZQ+4tPTLQeltxFtQuFKTdIBNbQw9wRbXVK6x0GKQwV9G6H1FRSKJMDi\\\\\\\/4IfH374+xqbpf0WGJgrIEzGCuqJOt2RQ97ss2WW1Tkw5+E6YaPK8rYj9VN7BIEDHjBxWabGI6S1otZTTSAQVGXoan1lJCRS6VL9rp+smreup0CwbGgsXOq+elBiKPFCLC3eNwCYGwlFQDng3hs5Xkn39vF\\\\\\\/bCSe6fxRddgGHbIi\\\\\\\/43\\\\\\\/ZEFKeNSp5nTXS0yjw1NxFxQR\\\\\\\/7VMLr3qc5U+DjK7QlX1PjtR83zrkxohC8MOjQ\\\\\\\/RW+aCQVVfJOyCOpp8lzQrS9Dbm2mUVfj3BTxfW2814++jfq6+It82qa95IhhRAD8Fw0t7YA1UbM\\\\\\\/3nOiCuzYHGUgwtpj845wEizAEgJdwAGmZnjG5crxzsNfXwL5Uo8yVBcpgyiegIj6ERCmnjRiVwZ+zew1sQ39kWbVfFyU32rY0fYA4NAXJOyxKcBThuureez3zFc2vuEh97IAZV2bZC0MpHn3pc0F1AVUz5f+pBpp6Ys7gvLWxFj3r6BeX8W4YzRKOStYNgOOEj252FtAGdLrNB+HzaYlh0EKmkcFqXjYOUhulDbO7888+0DJ3cz31\\\\\\\/1BFyWiszQIPT2\\\\\\\/r0mNvK1mgRoqwZhTuwemnFEUnlpUGQJ2s8ChfkTbWNVz4ekcVMVmdUuuMc\\\\\\\/EbWpO\\\\\\\/dRt18WY5yTKKP5+lEYJiGqvuFpp\\\\\\\/cd6CmNm1d60mhDeMdPZl\\\\\\\/vJffIvUcLgliQTuadtkYlRwWCr+WYMhwEVRE08gRmFZLifmnMgDBssPDr3mfrTu90hqIvkwkgexJGddvqKNF5cqIvtrloA4mtVdOXiOxyVsbBdAuKKuUGOKfqPTqugyCUSJOgvmoXK4KSNqr54jl2eBnejkD2K48O73KDSFrBfYBwnRD1BrwwC2GU\\\\\\\/8+BP+KGMGf228naMM\\\\\\\/ZGKSBskqsxgFO3OdSvsBG860bE5epipjicOU1qlhF3CnaPn5HQg9SHYkpi6QXT\\\\\\\/JiEroRTl\\\\\\\/JMLhJhIdmwcTi6\\\\\\\/NNFcgHlbbnnz5Ve03wkC0FmyP77O30XlLi4CBIiy+Y3i1mv2nUDbliGeicuGq7l4v8Sby+xZFmMuY1OOuo2nbS8tkoVuIx5lOtZ8cDZZadcWTxYbkTQ0xMwAtAUpyTSs6g5wa4Wwi4wHVyYeiGsyeWZRYnOIyBL+gXnAXLkai\\\\\\\/g05ezFQxiN5qNe5sFBTv0FK9PV5j8ColxtaM0eHverWq+mMMLZEvGtsren5Aq\\\\\\\/7f0jldBEArAJysGJ+SoCmxYd4FofD07EibRyroj093ireTBnbx8jYa0Sh6ORHX496P\\\\\\\/YzLMMCtgkmaqWjWVnox0yeKyh0j94SnFDNv6zR9ePNNtAa3\\\\\\\/nhIr1o1AS9PU5YAhZKDj9ntTfh\\\\\\\/pzyEK6RfoMgMdITAtXmeGPlwEbHfDj6OrV59N5JxuUPqy9QIavZk1qghY7mLmKT\\\\\\\/48K56QRdTPEy5K3B\\\\\\\/5AYK75I\\\\\\\/+r51LpDwiFrkycbF5bRMCXWCrjwjlxOT282aDabVHpT6bcgYXnL4oWp1n0VJ\\\\\\\/C\\\\\\\/1Kq+2LZ28VhKVkwTf2Tnlpnli\\\\\\\/7LDFAHf65nD9R2\\\\\\\/0NmdxVdWub4yHA5Re4qJQX6pbPtLCEh9Zdgxdv\\\\\\\/mP4ng+lg9zT99tpmvBMc3xLW\\\\\\\/tC3vyEJ2lephhWrYA6AoKjnn1l+GCM52cVz3VR9fUCH9liPBZe3HgF2Ni2j\\\\\\\/TqOvA93i8OVf0mKngaM2H\\\\\\\/+B+OT9xDUA9Ys4Z6q71zHDEPz2\\\\\\\/UuC4tFX7qi8AE1DJT\\\\\\\/6luP3IX+arnneJ7+W4vDHRh36NIBE6Z1uE32QIL3923wA93RWhCzd5sPUj333hg0HarNb3teYXaBcz4MxJvIne9mOpTn12rM7qA9EAPSJhnMC49AcSxYbbm4CBE+5dqMpIDfzfAKNJ+Hoq1HOu2DxgJZe4l55eWX7gfeZZgIyeJBVsQoVpoEsS+QakRBcC38CGEhgXuS688VZKvhus71ImqjyxiDU9bVG+I6DlM\\\\\\\/tXoA1K4CMSqhHa0zB9QzyYc1cQ7Y2FesgM2R3HS9JmG3ff8N5a1lGUpXUpfbFkmr0x53ibZ6+nrdUBtwiG28nZshDAOI0hkIv1bfpDSxfwiKpC5Bl2+SWmnzq5kzttxrhTANlXnCK9jlv+05AfyVYDEEZ3LLV7YjLhr06ZgBpSHX9S5StP3lzVM3auuZc0HvRgVcYPgWP9MQQTameCVK5GYNOnE3BmIE7i0VyWAxJ4MbBAFJO2XJM34AF8REh4NRFGVgoCInqf14vyT3OFOKRBns1PtKj3a3J5Dfd57SHlMoF7BzDStfDzqiU8tWynqdDHNEvW5OsIaxX\\\\\\\/n0bJWeg51g+bRMPidASICYP8m+hkjmJe1yYy27kocwJwu7iIsIPSkCt3X7UfsFPEpE+rmgF9jelf+AmNrg\\\\\\\/1E6iqtbK0cM9WxVRs1rVphDS3tflOHhC3BLQC0QyW1fofvzvrLM9Tjh5IeXXJUiUkK3LrONl29Dh4TCkShlkIByQXEKOyr97QgHAQPDV1ydBEw3k+tV0x5QCaPUj8Pa1RNIu7KeIbSCrEllXAPDQdygpZO\\\\\\\/YBXYOasSj7+I27vox6qfjEArAb11cuO66lknBcZpSuQU5IvVQvmsQXAPsYe5pMhFhMpC9YIE8qf2P5eemb1zl5\\\\\\\/AN8CNtHrAtlXzNeC0fqIb+z2KbtgyGjny9Jvd\\\\\\\/v0pQN4osEaldtphj418aP3xG6px+xfbZDJ6KjA5GhdMf5Uoznelr+TTslqvunyczfDA7AvEICl\\\\\\\/U9M\\\\\\\/7kD5yU9w7Y7NKRS9eGFSTKxfbLI9FiPWOkzUfOQXXGW2UdzIY9VbPdLC2S+pyoGe8ujshXzc7f2szEu6mIfsAp9U+R\\\\\\\/URrCqXkZzOXeOjKmZQn9tY7PmDwLGcIy7NBRijx+n0HuqUMh30QIuLi\\\\\\\/vU3DuyVIR\\\\\\\/NV18ixTi6rL8u6ALreklZPk2JVZbmP4o8zFu1S+g820slDyQFOOJS3lqoWHioQjdZ6NORlzWayT3K1x2wE5ZOiu2SHU3VJnA+ql6u6x1U5bm6rCUbOIQ0TM0b0PH36Qh9w9uCKJJnhBnTjyvg9\\\\\\\/zg2B+EgfaP2uDps6u0S03hyzuKoXIdQcxNOp9AKxTOUR9opm0d5ZERhsyyxk\\\\\\\/27ZMbe0EJifExuzE4+DwgxFP\\\\\\\/LEY8fHgWkFilLw+uLTgvOjiAOp64wYYw\\\\\\\/HymAFLIXFKtwi3TtxrTPXKz1iOfEHmfUH3q5tAO+WpE1DZ78lRaYQFMmazW2rxl\\\\\\\/\\\\\\\/+zWfD7jT7aErcJmWp18n7044L05nPEpJOE34c4aPZUiey6wOeNIdMR4qfed9AVy16GIs91Hut\\\\\\\/ZT8xw\\\\\\\/zQva0SnIsNP+eTzSfLaXABI12Y0CFoSKOadCKLc3tS5VPWWqGXC+U0eRr96KRUGYsUfDGrUwFkrvUuB+qkFcw1Asq5YOAs6sf7dAuzMmHILFewPw\\\\\\\/4zHqjGcTdrivXMz9al800YC6SDPjwNdO8l2G9AoJ2d5HllUaLJqbuE2ymz8FKwQiBu3x1o82jFS2etQSURXXhuSm2qQ7B8OgjK4pw5tnfVKpUZ+UnhH50B+6E9rhRI4jF7WfgyCurm74m5Di0Q22b0rx4isopouc8uFc\\\\\\\/OuiEkx\\\\\\\/17a2dM9C+biPFEGRQo2m9cL12W8faJ81ts9xSRR8tyH4RB2\\\\\\\/gzSEkxSrRQSo7MwAPb1QE3g\\\\\\\/lWArvwq6i0vInTqvQ8XgddmIexz7NKW5FHyaWpqisRvdK0OcIZLfoedUmVSBqpa9sitXJYIf51sTtycFZ9n\\\\\\\/ExCon0pEjzGgnCLY3ANH+E6ZMHcQbbVgSMlUQhNENOYzzjqtYav4MjNzOwsr9ZcO60E4vBeZL39aLNmfPeElv8ccMlbjLsq561p2\\\\\\\/jzfPj8WDJ+W+tI4Lh6119IEVDZHOYvlbZZunn7oROd\\\\\\\/tTqftQ7czeeu1YxjJFNPfwzHh9gF2HtJhPtqQrMwd6LV51WSygC2XcYrFTE1wOPoL+EpwX5SuOmdSflY9\\\\\\\/PNGjJEYaxp6+q+pNj29JQl18YdY4XS6G+6LunXfg3\\\\\\\/HNPtxpg6CfRpvmg9lV6ZNAiSvXv+ftm+VCNNONtTpmd7vJMzBe1tL4aKc+hnbweC4y0O0bysxryKXOcfz5wFxgpxcjZreoybasGWdK+r88npJ4QdKBWHpmC1VdnocZ8KGTvxUrmYZvattmT47NLtk30TP\\\\\\\/AbtTRyK1ACRuECp7jOUEUX7j3UGNsY+hYClj6+Z0qxynbAIEXvxd1lRuPgIxmqoGHWLFyxYWVGFPTkvdbT\\\\\\\/LgIcGf0\\\\\\\/gCBdv4JO8B81qKSvGDgcfGQ9deFGlu5ULUhZchUyNQe7Hms7GejTETfIqxp+XLzS13sX2qNQlFRvrkQPGNGzXrq+WM3f\\\\\\\/1q6r3kesivm2G3mscg2B+HSOXg\\\\\\\/Ti\\\\\\\/vGwePn+3ljU188dIQN5XVCzp+gokioM+xr+jjGlSgkK+FghuYfqyRCrJq3UlJmE9MERs2vEgD4x\\\\\\\/lVb2LYa2loczX0SzTTqK7+f+LpuwnC6zCZC\\\\\\\/rDKDIWj5YERFxQJy5njqRQyvmeTJNQSvuvE6qkBgbnwprb5bbEaLE2bGQraOzKrCZhc8rRiCUzbQg2wQXb8FDcGHAlkEm4izjbT0i4iGgzRW4iXvefBKlCJfTDdMoYrejPLrcAh8j74Kz1ThCK4lqRocLC4onsdPBZJOWEGisBbMbKawEGND+U0dpgeKWxF1HlXrDWsNYuE8lI5Ke7\\\\\\\/Fu+6RXl9ccoZ72asQOuKNAZBAiO1UGB8MgCr90iaYB6KCMij8gxN6TfDuBP5JHicbHRO0TG8oMvJPFI1Y5D+CgFsd+oVxjfBJuIhI9WZlSgx7Ym0uraYZ1GUv47pBH61IniFwjS1Erqr+1WExiB+FNq+mieMFN9XVPqWVrrd3nbHRyEptUnDzvARaBwQQKMWwbu1mGQEFj8sGCu\\\\\\\/IUQ5YbJx2QKMkIOX50ijgOxWtnS4NoAiTzgXf0tXdidSb+sne0Hu4gP85oSLL5SkTiU+vOAA1DsYJ4oOT0lVnrXyMKtWPbz397DKE4N6AsR2JgAablhWs4qSFm1O3Cn4zxKTkMMXXmAfCnild12Y+LQhjJIlnOIL9LUlFv73zu9s\\\\\\\/Oq8eqTHHqr2yLhLTc+7e5LpvPD701mzSnyT2UPqM\\\\\\\/NJ1hga0IJKKnOVUUes9yfkWGu18JP9XjlaN50QXc5HptI55R9GKes2l4C25QlFJRyhSfF9BL44AzHpz7pRFSZH5MZh94KkqEIpKLs\\\\\\\/Q7vw182U9oRCpvGDtIceIrGC\\\\\\\/IwGM8Ug3x2O2+J7Ij1MjGQ7z8bnie\\\\\\\/sM22+v0ISxUV6gEeswa+TFXTl00ng3naxKQ6lS1pUTXfCuvt91DnjdI9gDAgoP+3sU9HynDCIR9MRoeIWGYYzYClssus\\\\\\\/tZswiW3Fp8QrNvqcr0\\\\\\\/L5sYPclB935nuZ\\\\\\\/7ovRAtFpKE+nJKW1GvP7O5f1n9Mwren56e4b+hgBdlwCiw44iurMiLkyxyVLYtvbpJyAB832RZni5HbSAOgGtSYok12Dlz4mrkRl9Bn6SO6Qi\\\\\\\/jN4XmvAzd+gJ5eFOg6cb22jnasPVSZ6IGjk8+OedD4+U86+KowJNCdosNeng8befNnyI3GbCywrLFio7IwORHbYM2g6ayKUcS361jHEbeQuVFNHh15VKOAhcsZsY8h75p5EXTs5u+xlIB3TMmG\\\\\\\/2E2vttaH47RQ1owdjjE8G+Xx66hOIkjPEskcIRdJtCbu9jwkp6\\\\\\\/Bx2QxhG+o\\\\\\\/joYR1f3KvPBNV\\\\\\\/SZkdGxzfGtwS5EVu3a0hho6vqbE5MoK5vDSM+05kSQWOSY27KhE0iYcwOw3KwCblvkR8KdBusUJe\\\\\\\/gFSM5EPw97BFZc78Bj55fGMDw6TLDy39GHWQcj1RwPMe3zrD8NW3wAn2qDOadOaM0tJ5TFmT+llqNbiaLUu6XT7JDU5OmwL+V7cF6TT7\\\\\\\/KP6kHSWwbZMQB2DQF2KJxGPytxP4JGYgBexoBRE+tsE51FydzZei6dB45Wp15IYHxU6Y2firzw6r3+2KZ6uUfUAdRrn+\\\\\\\/\\\\\\\/Q2c1xoDXVTq86VKvV1A49Vq5jLsM\\\\\\\/UrvUJ3AED3Ho8iAqJy2pBOq+eOC4sI6DPD3eoPZPHjyIY0bch29j+0JVwgGg602F8592W7PlRFybQxZDrJtTxWPJW1yxmh939jumoi4oeit0B+0aDnLXHlyTSMKWsZuqeAWRDL+4Tfl2A5HqrBdWoc+m4eIp\\\\\\\/zOity+i7VP\\\\\\\/v87xzVTdYy0vVYFN2XoYWAWsG5dZuy6Gro9vcVUGrzJB0XWnDl\\\\\\\/iotV1W8wDcnjDe2UAy7sMRJKZT8V+9Bv8TsBBhpNJt+UqhfaYzX\\\\\\\/0arOa6z1BHsIKTjo4QvuK6\\\\\\\/+p3zh9eZGMkow\\\\\\\/has3Ck3sKfvgW\\\\\\\/Q2aVBKGe7YSD4c9PG0T24ZJYfsMFUJjQNJxO4NEjzQJO55rg7Jg4DDDaV0r0xu9RtONCh6ZiH3NQO5IXlK7jbB9a8CE1pVrfroGaUjfEUYVpHdskLBYGNwt6yDDM6b+2qAQJ2oYibaCcEjL4aACJEvuDZdoMRxi0wxT8\\\\\\\/vvGALyRJhv2Yao0gz9ES+Ywipp1IB\\\\\\\/9C7xRY\\\\\\\/lSt375xT7DiK2cCTdeE2BRDLwTPVCvTgxIF8tmmcdOcTc0TezdWUTVB1OQ0mBAnEHKaYWi8GaD4hIWOBCDiqxRzx5F4B9ZUZt4kz2Bpj0b4n\\\\\\\/0mp7aVGB3Ra44WTeK2Pj1l3MImLyB4FgEmkPL7Ne\\\\\\\/kyNURJqbnqiGfPvHPfDOpBb8+PesAUfEUzU81tMMWMVdVVVrgHT9cnD0nrg49Qa43KlVhE1LhcaKdlW22\\\\\\\/\\\\\\\/f4Ofvfb6mScTIjzv87wLk5uEYBGJ525+RmTlMNQ5yRI1F\\\\\\\/UECiKfdbX1n9120DAKYlsJtIkJVkFVzhc\\\\\\\/Mnws+pFhHkKqznv\\\\\\\/SRQ3drUjXsLnLGyF+cXYsLsIUBKoFJgCo1thxnxpQArOKkLE0syc6GVU8S2SYXQw0GL875GYezphP0Y2fqsKq11NjJDmqO1r4l3ddXSDFjbq8UHCUleopsqPBmZfXMm0s5chmCE11mjnYt+f4qDe3GlGa0i62TZ+ycM1\\\\\\\/+86VhvE5L7Xa3j1M618qsi1wl8U4GgA6xIrhff\\\\\\\/LhHT2Ptjm3S\\\\\\\/QaEIs3YGwa2OMRryKiB6TqRAnBx7eHl5W1GNGgsoOR5dcRUlD8r6X9bscwgotFfmEMN\\\\\\\/qWO5LwKUeRIvZsuODsaLdp714SICaq1y4zsUFJ45FWT1DG\\\\\\\/X9CuVkhCgS0oewp\\\\\\\/yBPTzVQA3z\\\\\\\/onzn9IU\\\\\\\/x+8zjjIQVdPBza9Nh8F96MDZfwG2aqxgVeXhOHA1R9PuNCb71F+4IA80UIeQUIPotUrvoa3XVFAvj+iMNd0iM4\\\\\\\/mNRzB5ye3zq1Gl5EknPerCNA0MkPMOlDucsRebzZ4w7iAZFZrUFY20jwqBj247V4Xy1ojnuD0e0VbJJNXLSZg2lOa09B7t5fqa3cqNCPudos2XFi0X1LucajIqQ9vm0024ZO\\\\\\\/b\\\\\\\/J2eQducEsFh2N8rZH8P0wS5KqMduu+izZ1qWSf6\\\\\\\/fYJucUdj2O3CP4qnVOZ0NrvO0syYNrS\\\\\\\/vKqb4qq3Esa2s+KFX7\\\\\\\/LBEVALJKG8lxF+XcRVDTQ8VZv+palik50bRCBKt8mO8As31RfsuhgVXAIdT4cGQepWRZXYyfCHdo1tVvH6NAqDi9krCHCjn+LjALtuN+x17ZiNQ8xojG7aQWY8SW23dlggiWEnmEd6+0ELWCd\\\\\\\/tBY4S1EUxLKW3gycAmzKJGiTRIbI3rHDvxfcISRMWTbq+FOspyNi6a9p3DceT8StGR6okKZoU+DYvDGr+0OfZiSvcX7PBL\\\\\\\/hyruOcoBbmcj2aR+RcAXAoxXjXsZ7x+jDSw40E+reCC4KTbpSn16XOd89EGmTV51q9ioMdmrH0O\\\\\\\/JsQJdckvER+MhNl0UtTOHZB3rQSJtUvNoZhh0qex1KMePYsG\\\\\\\/vrvdoKkCwKU1uslTMbkKT5rD\\\\\\\/QGhhVdFaEiSFL9Fr0w6NbOncUYG5E5DGYn4V8tvwz0Ykm1fWc8PvChYTD+XT4DAE60WDWBO5OBj0gN+JZifW8AA9ybZNvhRW0yqfQx70RBeQxWtYfbUDJN8OJjBTaNpAs3kjmNWWFXvpZe9JWMGUV4qu44R5FsEiejbUghTGkhKw0WfBWLL9kiwrTxUZ+YtUP+gh8D9UFem\\\\\\\/kHQ9A5yNQvXS+QJeEuSOD3gknJaC\\\\\\\/rlTdMRSAoHC909fToSHHZdDxgfRGPeUORFKliW59M6pEThN+5QaGdnbk3gGBjCBvOPTv6e9HglesHKvNtzw3d2SPV9fXL8LpLrnV\\\\\\\/io5Psji+G7J4bzdhSXn8Rx\\\\\\\/+AUR\\\\\\\/ibYvSU604scCv4dS9Gtmd+XEuEZ52ENV5LG9iUQzAsBcix+AsiqHBU\\\\\\\/0Oipqbt0LBRN8P2mqZQOKYL\\\\\\\/NIgNiLEBVWyK2sh2J5\\\\\\\/g5FFNKS17YXGvbF\\\\\\\/xxHKBPKMJJ72Ne2L58LkQ5nsTHtV8hQvJ+iP4EflmS96yA7XyM5f38EZHJuonWZfN2Iob8EBKPlVJVS394TeyXFHbc9S0I2MU5BoNUDeBh44QT33Y\\\\\\\/RcCVHorX8yAD9mRzx8IsWLCZgAu07wn4tB+RexP4HaabqxtigIVLqvh5Zp58uMuk6wX1l5z2sFeH46FvFvCWuuhdAO\\\\\\\/BzTcPqVcZ1+at6\\\\\\\/bJnI+vYC8aLT72OEgKEs7lCJn22lK16ogHG9qrvGsrDCY49ljRx+qxHmMegvD8hSrO8hC0gQKQFGhPARPdP5D7dZaryH8W4cPyLHmnuHcgXz7RtMmo4Tq5r5HB3Rhtp0d+dXDrreySdqFwdCqkuSRljodIoLZSZfbrJhaMAid4uafQpNbjBJdAYzsjRRj4182+N803IXrI99+jIPNHR1sgYtsW4ArUIiwFAlaAHDrF1rrIdCZNdC+9XHZ+Vs1gnuwH4ZCnAH95Z03k9AKPLRU7zQ8q6RJl8b25ZdvFB12pi8+rQbbzZYfCiqFa03GVFKgUkzUrr4u3H9b1ubiR7b3OGluBKxabNVkTuvcmOqGt6x\\\\\\\/U+p\\\\\\\/KoBHmoblhmcm0VXuE1anZN+P9\\\\\\\/05jUp3dTkpvsOM6KI2WIVc2jMxWsbSNxCYQaC5NjwVrq1CLNrMaS+5R4\\\\\\\/Z0p7XJaMmpYyQt1j5LgXBzi4iRio5bvB84YE3Oekdubw+fr\\\\\\\/G+zFjjqUN\\\\\\\/FvNNftIETp6msY4ki+KzOilR0fMQMUbJ6m\\\\\\\/wfsSqPWHR6qzp8Wtt74mLWuf+6FzJ8RFzyLNRvuYXT6Ew1\\\\\\\/0dNvtCfavEG0WoA7xTZDqTSpos\\\\\\\/Ffi3SNJVPR0carUWEMieWQtSisiVLLTJBa4yhNaTc8TQCgsyyXnrhkTVFFpBnSp62Jbrm0fLOEckTuHphnN4K3wx9qav9ouirQ+9KlBuDb0rcVIRMcUCbX69Asw6GDefiAgSbnTNdaWXe6u+WtXHVSXdAQRPCSAwtZYTNlgM3\\\\\\\/YEUM4xJ+XoRlbgRGpD51Lqzg\\\\\\\/0LXpUzawdylWdYnCSwguTGljzpg43xjMCslpHIksAULFljUnmAFNVgj0iJwAY82Rc0Zb4+sjRNHxUInLRV9XnQBQp75gHz\\\\\\\/0u7OnoeVyklbiRfUbi\\\u0022,\\\u0022iv\\\u0022:\\\u0022LzomJKSg7FicGU+SiXVjRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002210b36f8edc2f8fc7df1fc4e04731a8ac27d42f8941b69260a36876d4ba6f6fa3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221KB7\\\\\\\/Ty2hJNyYyvl3WNImtaS8WOwq45LXSQoWOAoS0iIGEQUiXFBp7xr1u1Rkk8m2lEWyKwlXBm4TFS6Hph0CRsu\\\\\\\/qWtfgLePTCpR4NV6ujRImdx0\\\\\\\/Tn+PTgih3egmBbgdSUbCk9o5UpaEHITrxzJO\\\\\\\/2wv6eiq79Mr\\\\\\\/K4\\\\\\\/dJSHQY4L0Zr+o68EFethRndpZInaTxt10b2K8WGVtJUQBxxlJm40CKwY83oZSR3qgvCE3ePrQgzxISOhnfAT4iY5LotKseA+1bPTkJtEVnThHXYvC+IrpvViM1NDZa8UvOgmZygRI+OBGk747PdR\\\\\\\/IQFb5ow22cTWqpEEGxWYPufm4vJ+iCbembe6ZWHNguEPMpJK\\\\\\\/da4KhpiEYaVGph+FmF+PqekkOdDdoopVOZxYEuN8yI\\\\\\\/k6leSBqwINox6ebi4bu3LlYVCW8bHGEQUI0GbpcMWaRHUS68DURjiHoaVz+ccEuyDyEdixne4JCzojHwNDx0wK7BvbX5dJldFuXlNLs8Bqmx1HS85qAaXzUWJcTL9+pEP7oBNaqAw\\\\\\\/XsD4JghXbGyzYFXgtM7xjnnbK1PeDS9FWZCyoBVpX1r02en9W8LdrJvGXS1r+U1\\\\\\\/URD45kFbiG522\\\\\\\/6ab+Tr6BA1qsxP\\\\\\\/P3xi05rj1uWRvrthHn2X898BiHWPOabNIf1pjl5Y5hxhs1Lh01s+tq8H9bfVNt5ssFKx9BJmMZdL9oMEmzMEAIotpdYu+7EXJMevQ89a21z4h5oOOAvRSRkDFoLAbglN5HvbsFP5XuQXmpLHmGrawMw8bcWeLonmZUXMgF2ezSZV3EmShy4ruoBiO2BzjVXInQaBolsbzFycAAgQPn6qGrSyrdEiX9iGHkx8Bs3Q7r3qd1P4kb5hzYtdBhRkGEANyrlV0Vz7sGqyX+YPo34GZHpXcYC4ixpU9GsCvEkftqaKU+CYXZIsWM2ir7dvnxX+bK2mCIDn6QXXJUUSuAsXxU+9EVDfdo8xHaSp2mVlXrJZL5XAmdzEzezx3WqaMMkyS9MPePNnAQleCKLkUqu1I70DzizZFDEwu8e3zJGe32AoGE9Ofr8F6CzujwQXZFN6v3RsR8DPHPwWgrb3tcE6VXWpCE7H9phZjtv7IN8TQ+iA1wd9KkbjXPhQE7bX8RfxHLm0DFm1rIW4knv+iwG8RElX4LUK+FyxWzR\\\\\\\/\\\\\\\/GPuCbqaSnvehUjXkU5HX2drs+AptOlrhL+II+lUQRuS4ONG01p9GqDugKZ60vUehwdBD3Q9Bo+JMAHg7K5fPD28SeGHy66TqibPQjI6dTuW72CDsiI6hNocuXms9ZxbxQmIRVQzsmPHnkTqO1SMfnmcdGdjk8e7I4QrgLKRFvLSmgpTwCltOvdPJ3ELLad+rXrufS7IO3LG8DHKO24dx66fZWFPCBmV\\\\\\\/2tuQsRNjBDbTcheRpDLKE+HHZrS6LA39Lx+oWJKY3hvnyg\\\\\\\/8cXl6mx4Bdm635QaQM9v\\\\\\\/JwNBvZToziETp9gOveES\\\\\\\/sfx8EBC287XXcvdwX0vy+ebcqL9K5QKnSdVcc5Z\\\\\\\/lkFQHjq1A7dx+CBQQdDJghTDKgU7CXbFioNCJivIorIQDRVmJdbu7wlaE8a\\\\\\\/DCpT++5d\\\\\\\/Js\\\\\\\/jvhIdhQipUDqQlVk2CJd8fJGVI4sCDwGHlO18+p08iifJCSvnb7pKn4kmJOjoRy4nGLYyC2zJyZ8PbAtB2IHyeZzkUI1j7hPZXyu0nJVK5XruHjC0RSdR0pnaSWPxsR6gSbaTX4AQ4VizDM3P\\\\\\\/2XEXAAinONeYVsjCQLshG5foZr3CZnMkqqCF6q2nCzJriglhfwFyrTlhb\\\\\\\/UotfSfq7z0LJZVBzf\\\\\\\/NQtrpE9duxg3QMMlknkbSgB3ezawUZOTqupEvXxBHUBGFlzFKkxvUOXj7v11zYIX+w5NP5XgwR8835+Iu69F2b\\\\\\\/tXqFwiRmWLzSJIzJ7lOafeQsLwUah6Yu8hjpdW6tS9OBZz59ekhIE0w1qRh2MBpydqFtvlIkdrMYpJvb41p864YmVLYdRe7iak\\\\\\\/fTsdTOc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022sbsluvxodTqNO4ZtFqasig==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022120633103d3727d47dc2fa6dfb7af0964208f1ba7fe1701af693c2db06d0a851\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489123 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1299" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7CJ-wQsFNgg4e_m7lGhpUtudTNw1083wD3x8eDngcmQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1299" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Eazo-F6lsdv1NG-tgNjOrs-PI5MwFXFC7rKLcuoIUj8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1291" title="Photosensitive Alternative Splicing of the Circadian Clock Gene timeless Is Population Specific in a Cold-Adapted Fly, Drosophila montana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1315" title="Examination of Signatures of Recent Positive Selection on Genes Involved in Human Sialic Acid Biology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489123" id="node489123" data-pisa="ggg;8/4/1299" data-pisa-master="ggg;g3.117.300515" data-apath="/ggg/8/4/1299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systematic Complex Haploinsufficiency-Based Genetic Analysis of <em>Candida albicans</em> Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Virginia E. Glazier</span>, <span class="highwire-citation-author" data-delta="1">Thomas Murante</span>, <span class="highwire-citation-author" data-delta="2">Kristy Koselny</span>, <span class="highwire-citation-author" data-delta="3">Daniel Murante</span>, <span class="highwire-citation-author" data-delta="4">Marisol Esqueda</span>, <span class="highwire-citation-author" data-delta="5">Gina A. Wall</span>, <span class="highwire-citation-author" data-delta="6">Melanie Wellington</span>, <span class="highwire-citation-author" data-delta="7">Chiung-Yu Hung</span>, <span class="highwire-citation-author" data-delta="8">Anuj Kumar</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-6330-3365" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Damian J. Krysan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1299-1314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300515 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489123" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Virginia E. Glazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BE.%2BGlazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazier%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BE.%2BGlazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Murante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BMurante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murante%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BMurante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristy Koselny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristy%2BKoselny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koselny%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristy%2BKoselny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Murante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BMurante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murante%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BMurante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marisol Esqueda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marisol%2BEsqueda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esqueda%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarisol%2BEsqueda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gina A. Wall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gina%2BA.%2BWall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wall%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGina%2BA.%2BWall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Melanie Wellington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Melanie%2BWellington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellington%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMelanie%2BWellington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Chiung-Yu Hung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chiung-Yu%2BHung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hung%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChiung-Yu%2BHung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Anuj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular, Cellular and Developmental Biology, University of Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnuj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Damian J. Krysan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damian%2BJ.%2BKrysan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krysan%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADamian%2BJ.%2BKrysan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6330-3365" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Damian J. Krysan</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/4/1299" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489123" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1299.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489123" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1299.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489123" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1299.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489123" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/4/1299.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489123" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489123/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genetic interaction analysis is a powerful approach to the study of complex biological processes that are dependent on multiple genes. Because of the largely diploid nature of the human fungal pathogen <em>Candida albicans</em>, genetic interaction analysis has been limited to a small number of large-scale screens and a handful for gene-by-gene studies. Complex haploinsufficiency, which occurs when a strain containing two heterozygous mutations at distinct loci shows a phenotype that is distinct from either of the corresponding single heterozygous mutants, is an expedient approach to genetic interactions analysis in diploid organisms. Here, we describe the construction of a barcoded-library of 133 heterozygous TF deletion mutants and deletion cassettes for designed to facilitate complex haploinsufficiency-based genetic interaction studies of the TF networks in <em>C. albicans</em>. We have characterized the phenotypes of these heterozygous mutants under a broad range of <em>in vitro</em> conditions using both agar-plate and pooled signature tag-based assays. Consistent with previous studies, haploinsufficiency is relative uncommon. In contrast, a set of 12 TFs enriched in mutants with a role in adhesion were found to have altered competitive fitness at early time points in a murine model of disseminated candidiasis. Finally, we characterized the genetic interactions of a set of biofilm related TFs in the first two steps of biofilm formation, adherence and filamentation of adherent cells. The genetic interaction networks at each stage of biofilm formation are significantly different indicating that the network is not static but dynamic.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/candida-albicans" class="hw-term hw-article-keyword hw-article-keyword-candida-albicans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Candida albicans</em></a></li><li class="kwd"><a href="/keyword/haploinsufficiency" class="hw-term hw-article-keyword hw-article-keyword-haploinsufficiency" rel="nofollow">haploinsufficiency</a></li><li class="kwd"><a href="/keyword/complex-haploinsufficiency" class="hw-term hw-article-keyword hw-article-keyword-complex-haploinsufficiency" rel="nofollow">complex haploinsufficiency</a></li><li class="kwd"><a href="/keyword/hyphal-morphogenesis" class="hw-term hw-article-keyword hw-article-keyword-hyphal-morphogenesis" rel="nofollow">hyphal morphogenesis</a></li><li class="kwd"><a href="/keyword/biofilm-formation" class="hw-term hw-article-keyword hw-article-keyword-biofilm-formation" rel="nofollow">biofilm formation</a></li><li class="kwd"><a href="/keyword/disseminated-candidiasis" class="hw-term hw-article-keyword hw-article-keyword-disseminated-candidiasis" rel="nofollow">disseminated candidiasis</a></li></ul><p id="p-4"><em>Candida albicans</em> is one of the most important fungal pathogens of humans and causes disease in both immunocompetent and immunocompromised individuals (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Köhler <em>et al.</em> 2014</a>). The application of genetic analysis to <em>C. albicans</em> has been, and will remain, crucial to understanding pathogenesis, host-fungus interactions, and mechanisms of drug action (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Xu <em>et al.</em> 2014</a>). Over the past twenty-five years, most of the modern tools and techniques of molecular genetics developed in other systems have been applied to the study of <em>C. albicans</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hernday <em>et al.</em> 2010</a>). These include the development of recyclable genetic markers, PCR-based cassette generation, large-scale collections of mutants and inducible alleles, and, most recently, CRISPR/Cas9-based genetic editing (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Vyas <em>et al.</em> 2015</a>).</p><p id="p-5">One area of <em>C. albicans</em> genetic analysis that has not been extensively developed compared to other systems is genetic interaction analysis. Genetic interaction analysis allows the identification and characterization of networks of genes whose products work together to regulate and/or mediate a common cellular process. Genetic interaction analysis is currently most highly developed in the model yeast <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tong <em>et al.</em> 2004</a>). Through clever application of genetics and high throughput biology, the facile mating cycle of <em>S. cerevisiae</em> has been harnessed to generate genetic interaction networks encompassing ∼90% of its genes and include ∼23 million double mutants (Costanzo <em>et al.</em> 2016). Although <em>C. albicans</em> is arguably the most mature genetic system for pathogenic fungi, genetic interaction analysis has remained significantly under-developed.</p><p id="p-6">One of the barriers to genetic interaction analysis in <em>C. albicans</em> is that it is primarily a diploid organism without a traditional sexual cycle (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hickman <em>et al.</em> 2013</a>). Typically, genetic interactions are studied by generating strains that contain null mutants of the two genes of interest. The phenotypes of the resulting double mutants are then compared with the single mutants. For <em>S. cerevisiae</em> and other haploid organisms, the construction of the requisite double null mutant involves two gene deletion events. In the diploid <em>C. albicans</em>, however, four discrete gene deletions are required to generate the mutants at two loci. Not surprisingly, multiple null mutants have been constructed infrequently in <em>C. albicans</em> and, until recently, no systematic approaches to studying large numbers of double mutants have been reported. Of course, the recent applications of CRISPR/Cas9 to <em>C. albicans</em> are likely to increase the number of multiple homozygous gene deletions that are studied (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nguyen <em>et al.</em> 2017</a>).</p><p id="p-7">Double homozygous mutants, however, will not be applicable to, or appropriate for, all systems. An alternative strategy for genetic interaction analysis in diploids is based on the phenomenon of complex haploinsufficiency (CHI). Simple haploinsufficiency occurs when a heterozygous mutation in a diploid organism leads to an observable phenotype (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Uhl <em>et al.</em> 2003</a>). CHI, correspondingly, refers to the case when a strain with heterozygous mutations at two different loci has a phenotype that differs from both of the corresponding single heterozygotes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Haarer <em>et al.</em> 2007</a>). CHI was initially referred to as unlinked non-complementation in the context of isolating <em>S. cerevisiae</em> mutants that interacted with essential genes such as tubulin (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stearns and Botstein 1988</a>). With the advent of large-scale genetic interaction analysis methods in <em>S. cerevisiae</em>, the concept of CHI was applied to the identification of genes that interact with actin using a high throughput approach (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Haarer <em>et al.</em> 2007</a>).</p><p id="p-8">Our group has previously applied CHI to <em>C. albicans</em> using transposon-based methods to generate double heterozygous mutants from a common parental or “query” strain (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bharucha <em>et al.</em> 2011</a>). Specifically, we used transposon-mediated CHI to identify genes and pathways that interact with Cbk1 (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bharucha <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Saputo <em>et al.</em> 2016</a>), the key kinase of the <span class="underline">R</span>egulation of <span class="underline">A</span>ce2 and <span class="underline">M</span>orphogenesis pathway (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Saputo <em>et al.</em> 2012</a>). Recently, we reported a CHI-based genetic interaction analysis of a set of biofilm-related transcription factors (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). Here, we describe the construction of a library of strains and plasmids for the systematic analysis of haploinsufficiency and CHI associated with <em>C. albicans</em> transcription factors (TF). Although CRISPR/Cas9-based approaches are likely to be very useful for genetic interaction studies (Vyas <em>et al.</em> 2015), CHI will offer advantages in a number of settings and, thus, should represent be a complementary approach to the analysis of multi-loci homozygous deletion mutants.</p><p id="p-9">For example, there are at least two settings in which CHI will likely be the preferred approach. First, essential genes are not amenable to homozygous deletion analysis. Second, and more generally, genetic interactions can only be observed if the phenotypes of the single mutants are sufficiently mild to allow the detection of additional changes from that baseline. If a homozygous deletion mutant of interest completely abolishes a given cellular function, then creating a double homozygous deletion mutant with a candidate interacting gene will be completely uninformative because no additional changes in phenotype is possible. This is a reasonably frequent situation for some of the most intensely studied processes in <em>C. albicans</em> pathobiology. Homozygous TF deletion mutants that affect biofilm formation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nobile <em>et al.</em> 2012</a>) and hyphal morphogenesis (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>) frequently show almost complete loss of function. For example, strains lacking Brg1 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cleary <em>et al.</em> 2012</a>) or Tec1 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schweizer <em>et al.</em> 2000</a>) are almost exclusively yeast-form under most filament inducing conditions. As such, it would be difficult to determine whether these two genes interact functionally by studying the double homozygous deletion mutants. As described herein, we have used CHI to show that these two genes, in fact, do genetically interact. A similar situation is encountered in the initial adhesion step of biofilm formation where strains lacking <em>BCR1</em> adhere at levels barely above background (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>). Finally, <em>in vivo</em> studies of <em>C. albicans</em> homozygous deletion mutants have identified numerous “avirulent” strains; double homozygous mutants, again, would not be informative in this setting because there is little dynamic range available for measuring additional virulence variation in the double mutants.</p><p id="p-10">We have constructed and characterized the phenotypes of a set of heterozygous TF deletion mutants using <em>in vitro</em> assays and an <em>in vivo</em> model of systemic candidiasis in mice. We also used a set of double heterozygous biofilm-related TF deletion strains (Rob1, Efg1, Ndt80, Bcr1, Brg1, and Tec1) derived from this library to characterize the initial adhesion and filamentation steps in biofilm formation. These genetic interaction studies indicate that Tec1 plays a previously unrecognized role in adhesion to abiotic surfaces. In addition, we have found that the set of TF genetic interactions vary significantly depending on whether the cells are adhered to abiotic surfaces or are in planktonic conditions. Under the latter conditions, we provide evidence that Rob1 plays a role in the initiation of hyphae formation based on its genetic interaction with other TFs known to function during this stage.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and plasmids</h3><p id="p-11">All heterozygous and double heterozygous deletion strains of <em>Candida albicans</em> were constructed from SN152 using auxotrophic markers of <em>LEU2</em> or <em>HIS1</em> which were flanked with homologous regions of 100-500bp for targeted integration (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Noble and Johnson 2005</a>). Deletion cassettes were amplified by PCR using template DNA isolated from <em>C. albicans</em> strains in Homann’s TF deletion collection (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). The primers contain SbfI restriction sites, to allow release of the cassette from the plasmids. Sequences of primers used to generate deletion cassettes and to check integration are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS1.xlsx;">Table S1</a>. The primers amplified both the <em>LEU2</em> and <em>HIS1</em> deletion cassettes. Amplicons were cloned into pCR4TOPO (Invitrogen). To differentiate between the two auxotrophic genes, DNA was isolated from <em>E. coli</em>-transformants and digested with <em>Bgl</em>II, a restriction enzyme that cuts in the <em>HIS1</em> gene but not the <em>LEU2</em> gene. <em>LEU2</em>-containing plasmids were then sequenced to confirm the identity of the inserts.</p><p id="p-12">To generate heterozygous <em>LEU2</em>-marked deletion mutants, <em>LEU2</em>-marked cassettes were excised from the plasmid by digestion with SbfI and transformed into <em>C. albicans</em> SN152 using standard lithium-acetate protocols (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hernday <em>et al.</em> 2010</a>). Leu2+ transformants were screened for integration at the correct genomic locus by PCR using primers internal to the deletion cassette and upstream of the integration site (See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS1.xlsx;">Table S1</a> for check primers). The heterozygous nature of six strains was confirmed by quantitative PCR comparing the deletion mutant to the parental strain and the expected reduction in gene copy number was observed for each strain (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>); we have not performed this for each strain. Two independent isolates of each heterozygous deletion stain were archived in the collection.</p><p id="p-13">The deletion cassettes contain the barcodes originally created by Homann <em>et al.</em> (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>) and the barcode for each strain is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS1.xlsx;">Table S1</a>. The amplification efficiency of each barcode was verified and, in some instances, altered for poor performing barcodes. A reference strain containing a <em>NAT1</em>-marked cassette with a unique barcode was constructed by cloning the barcode sequence into pDUP3 by gap-repair recombinational cloning in <em>S. cerevisiae</em>. The resulting plasmid was integrated to an empty region of the genome as described (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gerami-Nejad <em>et al.</em> 2013</a>).</p><p id="p-14">The strains containing the <em>pTDH1-TEC1</em> allele have been described previously (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). The libraries will be deposited at the Fungal Genetics Stock Center. Prior to deposit, all strains and plasmids are available upon request from the corresponding author.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic analysis of heterozygous deletion strains</h3><p id="p-15"><em>Candida albicans</em> strains were grown overnight in liquid YPD at 30° with shaking, adjusted to OD<sub>600</sub> of 1.0 in PBS, and five 10-fold serial dilutions generated. The strains were then spotted onto agar plates with the appropriate media. Under a given condition, the phenotype of each heterozygous deletion strain was compared to the other heterozygous deletion strains in the pool. Strains with altered phenotypes on the initial screening plates were confirmed by repeat testing with a second independent isolate and compared to both wild type and the homozygous deletion mutant. Conditions and media for phenotyping were based on those described (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>): YPD 30°; Spider medium; YPD hypoxia (BD GasPak EZ) 30°; 14mM CuSO<sub>4</sub> in YPD; 100µM fluconazole in YPD; 15mM caffeine in YPD; YP-glycerol (30%); YNB-glycerol (30%); 1.5M NaCl in YPD and 1.5M sorbitol in YPD.</p></div><div id="sec-4" class="subsection"><h3>Pooled competitive <em>in vitro</em> and <em>in vivo</em> competitive fitness assays</h3><p id="p-16">The Insitutional Animal Care and Use Committee at the University of Rochester approved all protocols used in this study. Mice were obtained from Charles River Laboratories. Three pools of 42-44 strains were prepared containing stains with unique barcodes by combining individual strain cultures at equal cell density. The initial inoculum pool was immediately flash frozen and DNA isolated. The pooled cultures were used to inoculate YPD cultures which were grown at 37° to mid-log phase (OD<sub>600</sub> ∼1) over a time period that corresponded to ∼20 generation. The cultures were harvested and DNA was isolated. For <em>in vivo</em> experiments, the pooled cultures were used to infect 5 CD1 (ICR) mice for each pool at 5 × 10<sup>5</sup> CFU/mL by tail vein injection. The mice were sacraficed 24 hr after inoculation and the kidneys were harvested. Homogenates of the kidneys were plated on YPD+gentamicin+vancomycin plates and incubated for 24 hr at 30°. Plates with ∼500 colonies were scraped of cells and DNA was isolateed. The DNA from the inoculum and from samples recovered after <em>in vitro</em> or <em>in vivo</em> growth was analyzed by quantitative PCR using primers specific to each barcode. The abundance of each strain in the recovered (R) samples was normalized to the inoculum (I) using the procedure described (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Noble <em>et al.</em> 2010</a>). The experiments were performed twice and the mean abundances for each strain were calculated. The statistical significance of deviations in R/I from the pool mean for each strain were evaluated by Student’s <em>t</em>-test using the Benjimani-Hochberg correction for multiple comparisons; statistical significance defined as <em>P</em> &lt; 0.05.</p></div><div id="sec-5" class="subsection"><h3>Two-strain competitive fitness assays</h3><p id="p-17">The relative, competive fitness between the reference strain and single or double heterozygotes was determined by preparing an inoculum with equal amounts of the reference strain SN250 containing a barcode integrated in a region of the genome without any ORFs and the mutant strain as determined by hemacytometry. A portion of the inoculum was flash frozen and DNA was isolated from the cells. The inoculum was used to infect mice (N= 5 for each competition) as described above for the pooled assays. The ratio of mutant/reference was determined by quantiative PCR with primers to the barcodes of the deletion mutant and reference strain, respectively.</p></div><div id="sec-6" class="subsection"><h3>Adherence Under biofilm forming conditions</h3><p id="p-18"><em>C. albicans</em> strains were grown overnight at 30° with shaking in YPD liquid cultures. Cells were washed three times in PBS and re-suspended to an optical density (OD<sub>600</sub>) of 0.5 in biofilm-inducing media YET-1% sucrose (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). The suspension of cells (200 µL) of cells was incubated for 90 min at 37° in 96-well, tissue culture treated, polystyrene plates. After 90 min, the media was removed and non-adherent cells were removed by careful washing with PBS. The density of adhered cells was measured by reading the optical density at 600 nM (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lohse <em>et al.</em> 2017</a>) using a SpectraMax plate reader. At least 3 replicate wells were analyzed for each strain. The density for each mutant strain was normalized to the reference strain and referred to as normalized adherence fitness (AF). Epistasis between two mutants (X and Y) was defined as <em>Ɛ</em> = AF<sub>XY</sub>—AF<sub>X</sub>AF<sub>Y</sub> where AF<sub>XY</sub> is the AF of the double mutant and AF<sub>X</sub> and AF<sub>Y</sub> are the normalized biofilm densities of the two heterozygotes. AF<sub>X</sub>AF<sub>Y</sub> represents the expected AF for the double mutant if there was no genetic interaction. Therefore, <em>Ɛ</em> = 0 indicate that the two mutations function independently. Double mutants whose SEM range was completely outside of the standard relative error of the expected values were defined as having an epistatic interaction (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). <em>Ɛ</em> &gt; 1 indicates a buffering interaction and <em>Ɛ</em> &lt; 1 indicates a cooperative interaction.</p></div><div id="sec-7" class="subsection"><h3>Measurement of adherent cell filament length</h3><p id="p-19"><em>C. albicans</em> strains were grown overnight at 30° with shaking in YPD liquid cultures. Cells were washed three times in PBS then re-suspended to an optical density (OD<sub>600</sub>) of 0.5 in biofilm-inducing media YET-1% sucrose (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). The cell suspensions were used to inoculate chamber slides and placed at 37° for 145 min. The medium and suspended cells were removed and the slides washed with PBS to remove non-adhered cells. The slides were heated at 100° to fix cells. The adhered cells were imaged using bright-field settings on a Nikon i80 microscope with a Photometrics CoolSnap HQ<sub>2</sub> camera. The length of filamented cells was determined by measuring the path-length of a line drawn along the long axis of the cell beginning at the bud neck using the NIS-Elements software package (Nikon). At least 100 cells were measure for each strain and two independent experiments were performed. The mean length was calculated and the statistical significance of the differences between strains were evaluated using the Mann-Whitney <em>U</em>-test (<em>P</em> &lt; 0.05 was defined as statistical significance). Genetic interactions were calculated as described above after the length of each mutant was normalized to the reference strain length.</p></div><div id="sec-8" class="subsection"><h3>Quantitative assays of fungal yeast/hyphae morphology using ImageStream<sup>X</sup> MKII</h3><p id="p-20">Frozen fungal stocks were steaked onto yeast extract-peptone-dextrose (YPD) agar plates and cells were allowed to grow overnight at 30°. Yeasts from the plates were then inoculated into flasks containing 25 ml of YPD broth and incubated for 16 h at 30° with shaking at 180 rpm. Under these conditions, all <em>C. albicans</em> strains grew as budding yeasts. Cultured yeasts were washed twice with phosphate-buffered saline (PBS) and counted using a hemocytometer. Cell density was adjusted to a final concentration of 2 × 10<sup>7</sup>/ml with RPMI 1640 medium containing <span class="sc">l</span>-glutamine, but without phenol red (Gibco). Cells were seeded at 2 × 10<sup>6</sup> (100 µl) per well in a 96-well round bottom plate and incubated for 6 hr at 37°. Cells were washed twice with PBS followed by incubation with a solution containing 2.5 µg/ml FM 4-64FX membrane staining dye (Thermo-Fisher Scientific Cat.# F34653) and 0.4 µg/ml calcofluor white M2R (CFW; Sigma Cat# 6258) in a modified phosphate buffer saline, pH 8.0 for 30 min at 37°. Following staining, wells were washed and resuspended in 30 µl 2% paraformaldehyde solution. Samples were analyzed with an ImageStream<sup>X</sup> MKII (Millipore) image flow cytometer with a 7‐µm core at low flow rate and high sensitivity using INSPIRE software. Cells were analyzed along with 7% Speed Beads (1 μm polystyrene beads) for calibration of the flow and focus of the flow cytometer. Fluorochromes were excited with 405, 561 and 785 nm lasers and light emitted by the fluorescently‐labeled cells was collected through a ×60 objective, providing a pixel size of 0.3 μm<sup>2</sup>. Of the twelve channels available, channels 1 and 9 were used for bright-field images, channel 6 for side scatter light intensity and channels 5 and 7 for fluorochrome intensities of FM 4-64FX and CFW, respectively. Single‐color reference samples for each fluorochrome were generated by inclusion of cells that had been incubated with each fluorochromes separately. Images were analyzed using IDEAS software version 6.2 (Millipore). A compensation matrix was built with the data from single‐color reference samples to allow removal of spectral overlap to adjacent channels from each detection channel. Single- cell images were first gated on the focused cells with a size (area) range between 50-200 pixels (15-60 μm<sup>2</sup>). A dot plot of bright field area <em>vs.</em> aspect ratio (minor axes/major axes) was used to distinguish yeast from elongated hyphae population that had an aspect ratio less than 0.7 and area greater than 21 μm<sup>2</sup>. The percentages of yeast and hyphae populations were measured and presented as means ± standard errors.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-21">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Construction of a library of <em>C. albicans</em> heterozygous transcription factor deletion strains and a collection of plasmid-borne transcription factor deletion cassettes</h3><p id="p-22">To characterize the function of TFs regulating <em>C. albicans</em> gene expression, Homann <em>et al.</em> created a deletion library of transcription factor (TF) null mutants (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). In order to facilitate the application of CHI to the analysis of <em>C. albicans</em> TF networks, we generated an analogous library of 133 heterozygous deletion mutants as well as a collection of the respective deletion cassettes cloned into <em>E. coli</em> plasmids. To do so, DNA was isolated from the strains in the Homann collection (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>) and used as templates for the PCR amplification of each TF deletion cassette. The cassettes containing the <em>LEU2</em> auxotrophic marker (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) were cloned into a TOPO-based <em>E. coli</em> vector by TA cloning techniques. The primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS1.xlsx;">Table S1</a>) contain SbfI restriction sites to facilitate release of the cassettes. The deletion cassettes have 100-500 bp sequences homologous to the 5′ and 3′ UTR regions flanking the TF loci. The cassettes also contain barcoded signature tags adjacent to the auxotrophic markers. Each plasmid was sequenced to verify the barcodes and regions of TF-specific homology.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic for construction of heterozygous transcription factor deletion library and LEU2-marked deletion cassettes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic for construction of heterozygous transcription factor deletion library and &lt;em&gt;LEU2&lt;/em&gt;-marked deletion cassettes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F1.medium.gif" width="329" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/4/1299/F1.medium.gif" width="329" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489223" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Schematic for construction of heterozygous transcription factor deletion library and <em>LEU2</em>-marked deletion cassettes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To construct heterozygous deletion strains (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), plasmids were digested with SbfI to release the cassettes which, in turn, were used to transform <em>C. albicans</em> strain SN152 (Leu<sup>-</sup>/His<sup>-</sup>/Arg<sup>-</sup>). Two independently derived strains were isolated for each TF heterozygote and the integration sites were confirmed by standard PCR analysis. Thus, we have constructed two libraries: one composed of TF <em>LEU2</em>-marked deletion cassettes in <em>E. coli</em>, and one composed of single heterozygous TF deletion mutants in <em>C. albicans</em>. The deletion collection can serve as a resource from which one can quickly generate systematic sets of double heterozygotes in a convenient <em>C. albicans</em> strain background that has been validated for <em>in vivo</em> studies (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Noble and Johnson 2005</a>). Alternatively, one could start from a query heterozygous deletion strain and generate double mutants using the set of deletion cassettes. Both of these resources will be deposited at the Fungal Genetics Stock Center.</p></div><div id="sec-12" class="subsection"><h3>Phenotypic characterization of the heterozygous deletion collection</h3><p id="p-25">Four large-scale haploinsufficiency-based screens have been performed in <em>C. albicans</em>, previously (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Uhl <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Xu <em>et al.</em> 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Oh <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chaillot <em>et al.</em> 2017</a>). Two were based on transposon-generated collections (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Uhl <em>et al.</em> 2003</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Oh <em>et al.</em> 2010</a>) while the other two were based on a collection of heterozygous deletion mutants. Our heterozygous transcription factor deletion library corresponds to the previously characterized homozygous transcription factor deletion library and the strains have two remaining auxotrophic markers to facilitate generation of double heterozygotes as well as complementation (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). To characterize the collection of TF heterozygotes, we first performed a set of phenotyping experiments similar to those previously reported by Homann <em>et al.</em> for the homozygous TF deletion collection (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). This data provides a baseline for future CHI analyses of similar phenotypes and represents a systematic examination of the frequency of haploinsufficiency within <em>C. albicans</em> transcription factors.</p><p id="p-26">We examined the growth of 133 of the 137 strains under the following conditions: YPD 30°, Spider agar, YPD hypoxia, YPD 14mM copper, YPD 100µM fluconazole, YPD 15mM caffeine, YP-Glycerol, YNB-Glycerol, YPD 1.5M NaCl and YPD 1.5M sorbitol. These conditions were not as extensive as that performed for the homozygous deletion set but covered a range of biologically distinct physiological states and stresses (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). The results of this phenotyping are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. For the majority of conditions, we observed no evidence of haploinsufficiency or haploproficiency. Filamentation was the phenotype for which we observed the highest rate of haploinsufficiency/haploproficiency: 13/133 (10% of mutants). Representative examples of haploproficient (<em>TUP1</em>, orf19.6109) and haploinsufficient (<em>ROB1</em>, orf19.4998; <em>ZFU3</em>, orf19.6888) strains are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>. For all but two of the mutants (<em>ZFU3</em>, orf19.6888) and <em>NDT80</em>, orf19.2119), the filamentation phenotypes were concordant with the corresponding homozygous deletion mutant. Vandeputte <em>et al.</em> have reported that the homozygous deletion of <em>ZFU3</em> (orf19.6888) is hyperfilamentous (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Vandeputte <em>et al.</em> 2011</a>). The homozygous deletion of <em>NDT80</em> (orf19.2119), on the other hand, is deficient in filamentation. As we have recently reported, the increased filamentation shown by <em>ndt80</em>Δ (orf19.2119) is dependent on Rob1 (orf19.4998) and Tec1 (orf19.5908) (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). Although we have not investigated the mechanism of this observation, it seems that loss of one allele may trigger a dysregulated compensatory process that is dependent on the remaining allele. When both alleles are missing, the compensatory process is not able to support filamentation.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489270/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489270/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Heterozygous TF mutants with <em>in vitro</em> phenotypes</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative haploinsufficent and haploproficient strains. Ten-fold dilution series of the indicated strains were spotted on agar plates and incubated at either 37&#xB0;C (A) or 30&#xB0;C for 1-3 days and photographed. Two independent isolates of each heterozygous strain were plated as noted by -1 and -2. A. The indicated strains were plated on YPD (yeast peptone dextrose) and SM (Spider medium). B. The strains were plated on YPD and YPD+15 mM caffeine. C. The indicated strains were plated on YPD+1%DMSO and YPD+1%DMSO+fluconazole." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Representative haploinsufficent and haploproficient strains. Ten-fold dilution series of the indicated strains were spotted on agar plates and incubated at either 37&#xB0;C (A) or 30&#xB0;C for 1-3 days and photographed. Two independent isolates of each heterozygous strain were plated as noted by -1 and -2. A. The indicated strains were plated on YPD (yeast peptone dextrose) and SM (Spider medium). B. The strains were plated on YPD and YPD+15 mM caffeine. C. The indicated strains were plated on YPD+1%DMSO and YPD+1%DMSO+fluconazole.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F2.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/4/1299/F2.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489155" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Representative haploinsufficent and haploproficient strains. Ten-fold dilution series of the indicated strains were spotted on agar plates and incubated at either 37°C (A) or 30°C for 1-3 days and photographed. Two independent isolates of each heterozygous strain were plated as noted by -1 and -2. A. The indicated strains were plated on YPD (yeast peptone dextrose) and SM (Spider medium). B. The strains were plated on YPD and YPD+15 mM caffeine. C. The indicated strains were plated on YPD+1%DMSO and YPD+1%DMSO+fluconazole.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Two heterozygotes, <em>upc2</em>Δ and <em>crz1</em>Δ, were hypersensitive to fluconazole, which is consistent with their well-characterized roles in the cellular response to this drug (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Vasicek <em>et al.</em> 2014</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Onyewu <em>et al.</em> 2004</a>). Upc2 (orf19.391) is a key regulator of ergosterol biosynthesis genes while Crz1 (orf19.7359) is an important transcriptional regulator under control of the calmodulin-calcineurin axis. Two heterozygotes (<em>GLN3</em>, orf19.3912 and <em>STP3</em>, orf19.5915) were resistant to caffeine (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>), a drug that interacts with the TOR signaling pathway (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Reinke <em>et al.</em> 2006</a>), while a third (<em>ZCF29</em>, orf19.5133) was more sensitive. The caffeine phenotypes for all three heterozygotes are concordant with their corresponding homozygous deletion mutants (<a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). The TOR pathway is critically involved in the regulation of amino acid and nitrogen metabolism and, consistent with that function, <em>GLN3</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dabas and Morschhauser 2007</a>) and <em>STP3</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Martinez and Ljungdahl 2005</a>) are also involved in nitrogen metabolism. <em>ZCF29</em> (orf19.5133) has recently been shown to be important in drug resistance (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shekhar-Guturja <em>et al.</em> 2016</a>). Homozygous deletions of <em>GLN3</em> (orf19.3912) <em>STP3</em> (orf19.5915) and <em>ZCF29</em> (orf19.5133) also show altered susceptibility to rapamycin (<a id="xref-ref-17-11" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>).</p></div><div id="sec-13" class="subsection"><h3>Low rate of haploinsufficiency in rich media During pooled competitive growth</h3><p id="p-30">The heterozygous TF mutants each contain a barcode and, therefore, are amenable to signature tag methods to quantitate phenotypic differences in pooled samples (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Noble <em>et al.</em> 2010</a>). We, therefore, asked if any of the TF heterozygotes showed growth defects in rich medium (YPD) at 30°. Pools containing 44-48 mutants were prepared by combining equal numbers of cells. The DNA from a sample of the resulting pool was isolated as the input and the ratio of strains was determined by quantitative PCR using barcode specific primers. The pool was then used to inoculate an YPD culture which was incubated for 20 generations to a final cell density between 0.5-1.0 OD<sub>600</sub>. The cells were harvested, the DNA was isolated, and the abundance of each strain was determined by quantitative PCR. The ratio of each strain in the recovered sample relative to the input sample was determined using the method described by Noble <em>et al.</em> and is described in more detail in the Materials and Methods (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Noble <em>et al.</em> 2010</a>).</p><p id="p-31">We defined a significant change in the abundance of a heterozygote strain as a statistically significant (<em>P</em> &lt; 0.0001, Student’s <em>t</em>-test with the Benjamani-Hochberg correction for multiple comparisons) twofold increase or decrease (± 1 log<sub>2</sub>) in abundance relative to the inoculum at 24 hr. Although <em>uga33</em>Δ showed increased fitness, none of the heterozygous deletion mutants showed a competitive disadvantage under these conditions (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS2.xlsx;">Table S2</a> for data). These observations are consistent with our plate-based assays in which we saw no apparent growth defect in rich media. The rate of haploinsufficiency in the <em>S. cerevisiae</em> genome has been estimated to be approximately 3% (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Deutschbauer <em>et al.</em> 2005</a>). Thus, if these were a random set of mutants, then we would expect 3-4 haploinsufficient strains at most. As such, it is not surprising that we did not observe growth defects. This set of data does, however, serve to validate our primers and barcodes and provides a benchmark for future studies using double heterozygous mutants or heterozygous mutants under other conditions.</p></div><div id="sec-14" class="subsection"><h3>Pooled competitive growth assay in a murine model of disseminated candidiasis identifies haploinsufficient transcription factor mutants</h3><p id="p-32">Previous studies of TF homozygous deletion mutants using signature-tagged approaches in murine models of disseminated candidasis have identified a sub-set that is required for infection (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amorim-Vaz <em>et al.</em> 2015</a>). We, therefore, carried out a similar experiment using pools of heterozygous TF mutants and the signature-tagged approach described above. The pools of mutants were used to infect mice by tail vein injection. The mice were killed at 24hr and 48hr, the kidneys were harvested, and the homogenates plated on YPD plates. After 24hr at 30°, the plated cells were collected and the DNA was isolated. The relative proportions of the pooled strains were determined by quantitative PCR as described for the <em>in vitro</em> experiment. We identified heterozygotes that a showed statistically significant (<em>P</em> &lt; 0.0001, Student’s <em>t</em>-test with the Benjamani-Hochberg correction for multiple comparisons), twofold or more increase or decrease (± 1 log<sub>2</sub>) in abundance relative to the inoculum. Twelve heterozygous mutants met these criteria; eight showed a competitive fitness disadvantage while four were more fit. The genes with altered competitive fitness and their annotations are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>; the full set of data for the pools are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS3.xlsx;">Table S3</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489174/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489174/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">HeterozygousTF mutants with altered infectivity in murine model of disseminated candidiasis</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Five of the twelve mutants (<em>ace2Δ</em> orf19.6124, <em>suc1Δ</em> orf19.7319, <em>tyr6Δ</em> orf19.6824, <em>crz2Δ</em> orf19.2356, and <em>uga33Δ</em> orf19.7317) were identified by Finkel <em>et al.</em> as being required for adherence <em>in vitro</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Finkel <em>et al.</em> 2012</a>). The experiment examined two time points early in the infection process (24 hr and 48 hr), times when differences in endothelial attachment may be quite prominent. Therefore, it seems reasonable that strains with altered adherence would be identified in this experiment. Of these five mutants, <em>uga33</em>Δ (orf19.7317) and <em>tyr6</em>Δ (orf19.6824) showed increased fitness while the other three strains showed reduced fitness. The <em>uga33</em>Δ (orf19.7317) also showed increased fitness during growth in rich medium. It is, therefore, possible that this mutant has a competitive growth advantage <em>in vivo</em> as well. Although we cannot exclude a proliferation defect or delay for the other mutants, the lack of haploinsufficiency for these strains during growth in rich media reduces the likelihood that general growth defects contribute to the decreased infection fitness. We selected mutants with roles in <em>in vitro</em> adherence (<em>ace2Δ</em>, orf19.6124 and <em>suc1Δ</em>, orf19.7319) and morphogenesis (<em>sfl1Δ</em>, orf19.454 and <em>sfl2Δ</em>, orf19.3969) to confirm the pooled results with competitive experiments with wild type strains.</p><p id="p-37">Ace2 (orf19.6124), the TF regulated by the RAM signaling pathway (Saptuo <em>et al.</em> 2015), and Suc1 (orf19.7319), a zinc finger TF originally identified as a regulator of sucrose utilization (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kelly and Kwon-Chung 1992</a>), both were identified by Finkel <em>et al.</em> as regulators of adherence (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Finkel <em>et al.</em> 2012</a>). The homozygous deletion mutant of <em>ACE2</em> (orf19.6124) also has been shown to have a virulence defect in disseminated candidiasis (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kelly <em>et al.</em> 2004</a>). A previous screen of pooled homozygous TF deletion mutants without morphology defects found that <em>suc1</em>ΔΔ (orf19.7319) has a competitive fitness defect when animals are harvested late in infection (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pérez <em>et al.</em> 2013</a>). Finally, we also tested the haploinsufficiency of <em>sfl1</em>Δ and <em>sfl2</em>Δ, two heat shock-type TFs with opposite effects on filamentaton (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Znaidi <em>et al.</em> 2013</a>). The homozygous deletion mutant of <em>SFL1</em> (orf19.454) shows increased filamentation while <em>sfl2Δ</em> (orf19.3969) has a filamentation defect. As noted above, <em>slf1</em>Δ is hyperfilamentous while <em>sfl2</em>Δ was similar to wild type. In addition, <em>SFL1</em> (orf19.454) is required for virulence in a disseminated candidiasis model (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2007</a>) while <em>SFL2</em> (orf19.3969) was dispensable (Spiering <em>et al.</em> 2010).</p><p id="p-38">As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3B</a>, three (<em>ACE2</em> orf19.6124, <em>SUC1</em> orf19.7319, and <em>SFL1</em> orf19.454) of the four heterozygous strains showed significantly reduced fitness relative to the wild type reference strain in head to head competitions at 24 hr. The <em>sfl2</em>Δ (orf19.3969) showed slightly increased fitness but this is not likely to be biologically significant. These data are consistent with the findings of Spiering <em>et al.</em> which showed that <em>SFL2</em> (orf19.3969) was not required for infectivity at 3 days (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Spiering <em>et al.</em> 2010</a>). Therefore, our pooled screening with 48 strains had a true positive rate of 75%; it is likely that the specificity of the screening experiment could be increased with the use of smaller pools.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="TF heterozygous strains have altered competitive fitness at early stages of dissemination in a murine model of candidiasis. A. CDR1 mice were infected with pools of orthogonally barcoded heterozygous deletion mutants and harvested at 24 or 48 hr post infection (N = 5 for each time point). The competitive fitness was calculated as described in Materials and Methods. Competitive fitness data from representative pool of 48 heterozygous strains is shown. Mutants with statistically significant (P &lt; 0.05, Student&#x2019;s t-test with Benjamani-Hochberg correction for multiple comparisions) decreased (red points) or increased (green points) competitive fitness relative to the median pool R/I. B and C. Competitive fitness of the indicated strain relative to a wild type reference strain. Bars indicate means of 3-5 independent mice with error bars indicate standard error of the means." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;TF heterozygous strains have altered competitive fitness at early stages of dissemination in a murine model of candidiasis. A. CDR1 mice were infected with pools of orthogonally barcoded heterozygous deletion mutants and harvested at 24 or 48 hr post infection (N = 5 for each time point). The competitive fitness was calculated as described in Materials and Methods. Competitive fitness data from representative pool of 48 heterozygous strains is shown. Mutants with statistically significant (&lt;em&gt;P&lt;/em&gt; &lt; 0.05, Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test with Benjamani-Hochberg correction for multiple comparisions) decreased (red points) or increased (green points) competitive fitness relative to the median pool R/I. B and C. Competitive fitness of the indicated strain relative to a wild type reference strain. Bars indicate means of 3-5 independent mice with error bars indicate standard error of the means.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F3.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/4/1299/F3.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489297" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">TF heterozygous strains have altered competitive fitness at early stages of dissemination in a murine model of candidiasis. A. CDR1 mice were infected with pools of orthogonally barcoded heterozygous deletion mutants and harvested at 24 or 48 hr post infection (N = 5 for each time point). The competitive fitness was calculated as described in Materials and Methods. Competitive fitness data from representative pool of 48 heterozygous strains is shown. Mutants with statistically significant (<em>P</em> &lt; 0.05, Student’s <em>t</em>-test with Benjamani-Hochberg correction for multiple comparisions) decreased (red points) or increased (green points) competitive fitness relative to the median pool R/I. B and C. Competitive fitness of the indicated strain relative to a wild type reference strain. Bars indicate means of 3-5 independent mice with error bars indicate standard error of the means.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Two of the genes that we chose to validate, <em>ACE2</em> (orf19.6124) and <em>SFL1</em> (orf19.454) are constitutively filamentous as homozygous deletions (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kelly <em>et al.</em> 2004</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Saputo <em>et al.</em> 2012</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Znaidi <em>et al.</em> 2013</a>). This morphological phenotype complicates infectivity and virulence studies using tail vein infections. The <em>ace2</em>Δ (orf19.6124) mutant is not hyperfilamentous <em>in vitro</em>, nor did it have a growth phenotype. Therefore, our data indicate that Ace2 (orf19.6124) may have effects on infectivity that are independent of its role in the regulation of morphology. Since the <em>sfl1</em>Δ (orf19.454) mutant is hyperfilamentous, it remains possible and, indeed likely, that its apparent infectivity defect is due to this morphological feature.</p><p id="p-41">Ace2 (orf19.6124) is regulated by the RAM network through phosphorylation by Cbk1 (orf19.4909) (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Saputo <em>et al.</em> 2012</a>). To our knowledge, the virulence of Cbk1 (orf19.4909) has not been assessed previously. This may be due to the fact that <em>cbk1</em>ΔΔ (orf19.4909), like <em>ace2</em>ΔΔ (orf19.6124), has a profound cell separation defect that complicates interpretation of <em>in vivo</em> studies (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Saputo <em>et al.</em> 2012</a>). Based on the phenotype showed by the <em>ACE2</em> (orf19.6124) heterozygote, we hypothesized that the <em>CBK1</em> (orf19.4909) heterozygote may also show haploinsufficiency with respect to initial infection. Therefore, we asked if <em>cbk1</em>Δ (orf19.4909) showed a competitive disadvantage relative to WT in the early fungal burden experiment. Consistent with the behavior of the <em>ACE2</em> heterozygote (orf19.6124), <em>cbk1</em>Δ/<em>CBK1</em> (orf19.4909) is out-competed by WT at 24hr (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>).</p><p id="p-42">Finally, we generated the double heterozygote of <em>cbk1</em>Δ <em>ace2</em>Δ and examined its infectivity relative to each of the single heterozygotes. Surprisingly, the double mutant was restored to WT infectivity (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). One possible explanation for this observation is that we had previously observed that deletion of one <em>ACE2</em> (orf19.6124) allele in the <em>cbk1</em>Δ (orf19.4909) background restored filamentation to the haploinsufficient parental <em>cbk1</em>Δ (orf19.4909) strain (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Saputo <em>et al.</em> 2016</a>). The homozygous <em>cbk1</em>ΔΔ (orf19.4909) strain is completely unable to form filaments while the <em>ace2</em>ΔΔ (orf19.6124) is hyper-filamentous. It is, therefore, possible that the increased filamentation displayed by the double <em>cbk1</em>Δ <em>ace2</em>Δ correlates with its improved infectivity relative to the two single heterozygotes.</p></div><div id="sec-15" class="subsection"><h3>Complex haploinsufficiency-based analysis reveals role for Tec1 During initial adherence Under biofilm forming conditions</h3><p id="p-43">As part of a separate study of the genetic interaction network for mature biofilm formation (<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>), we used our library to construct a complete set of double heterozygous TF deletion mutants corresponding to the six member biofilm TF network (Efg1 (orf19.610), Bcr1 (orf19.723), Tec1 (orf19.5908), Ndt80 (orf19.2119), Rob1 (orf19.4998), and Brg1 (orf19.4056)) elucidated by Nobile <em>et al.</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Nobile <em>et al.</em> 2012</a>). Recently, the Johnson group reported the effect of homozygous deletion mutants of these TFs on the initial adherence step of biofilm formation (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>). Accordingly, we used the corresponding set of double heterozygotes to identify functionally important interactions between the TFs during initial adhesion to abiotic surfaces. We used the same optical density-based assay to evaluate initial adhesion as Fox <em>et al.</em> (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>); however, we used YETS medium instead of Spider medium because it gave less sample-to-sample variation (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>). As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, three homozygous deletion stains showed reduced initial adhesion (<em>EFG1</em> (orf19.610), <em>ROB1</em> (orf19.4998), and <em>BCR1</em> (orf19.723)) while only <em>ROB1</em> (orf19.4998) showed haploinsufficiency.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haploinsufficiency and genetic interactions of biofilm TFs during the initial adherence stage of biofilm formation. A. The indicated strains were incubated in microtiter plates in YETS medium at 37&#xB0;C for 90 min before non-adherent cells were removed by careful washing with PBS. The bars indicate the mean OD600 for each strain (triplicate) with error bars indicating standard deviation. Strains with a statistically significant (Student&#x2019;s t-test P &lt; 0.05) change from the reference strain are indicated by red bars. B. The expected adherence fitness (AF = Mutant OD600/Reference OD600) for a given double mutant (AFmutA X AFmutB) is plotted on x-axis for each biofilm TF double heterozgote and the observed AF for the double mutant is plotted on the y-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. Network of biofilm TF interactions during the adhesion step. Green edges indicate alleviating interactions; red edges indicate aggravating interactions. D. Aggravating interaction network centered on Tec1. E. Alleviating interaction network." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haploinsufficiency and genetic interactions of biofilm TFs during the initial adherence stage of biofilm formation. A. The indicated strains were incubated in microtiter plates in YETS medium at 37&#xB0;C for 90 min before non-adherent cells were removed by careful washing with PBS. The bars indicate the mean OD&lt;sub&gt;600&lt;/sub&gt; for each strain (triplicate) with error bars indicating standard deviation. Strains with a statistically significant (Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test &lt;em&gt;P&lt;/em&gt; &lt; 0.05) change from the reference strain are indicated by red bars. B. The expected adherence fitness (&lt;em&gt;AF&lt;/em&gt; = Mutant OD&lt;sub&gt;600&lt;/sub&gt;/Reference OD&lt;sub&gt;600&lt;/sub&gt;) for a given double mutant (&lt;em&gt;AF&lt;/em&gt;&lt;sub&gt;mutA&lt;/sub&gt; X &lt;em&gt;AF&lt;/em&gt;&lt;sub&gt;mutB&lt;/sub&gt;) is plotted on x-axis for each biofilm TF double heterozgote and the observed AF for the double mutant is plotted on the y-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. Network of biofilm TF interactions during the adhesion step. Green edges indicate alleviating interactions; red edges indicate aggravating interactions. D. Aggravating interaction network centered on Tec1. E. Alleviating interaction network.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F4.medium.gif" width="398" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/4/1299/F4.medium.gif" width="398" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489285" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Haploinsufficiency and genetic interactions of biofilm TFs during the initial adherence stage of biofilm formation. A. The indicated strains were incubated in microtiter plates in YETS medium at 37°C for 90 min before non-adherent cells were removed by careful washing with PBS. The bars indicate the mean OD<sub>600</sub> for each strain (triplicate) with error bars indicating standard deviation. Strains with a statistically significant (Student’s <em>t</em>-test <em>P</em> &lt; 0.05) change from the reference strain are indicated by red bars. B. The expected adherence fitness (<em>AF</em> = Mutant OD<sub>600</sub>/Reference OD<sub>600</sub>) for a given double mutant (<em>AF</em><sub>mutA</sub> X <em>AF</em><sub>mutB</sub>) is plotted on x-axis for each biofilm TF double heterozgote and the observed AF for the double mutant is plotted on the y-axis. Mutants that fall within the zone defined by the diagonal (ε = 0 ± estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. Network of biofilm TF interactions during the adhesion step. Green edges indicate alleviating interactions; red edges indicate aggravating interactions. D. Aggravating interaction network centered on Tec1. E. Alleviating interaction network.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">To characterize genetic interactions between mutants, we normalized the optical density for each mutant to wild type and will refer to this ratio as the adhesion fitness (<em>AF</em>). The multiplicative model was used to define genetic interactions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mani <em>et al.</em> 2008</a>); this model is the same as that used by Boone and co-workers for the genome-wide <em>S. cerevisiae</em> experiments (Constanzo <em>et al.</em> 2016). In the multiplicative model, the expected fitness of a double mutant derived from a pair of non-interacting genes is equal to the product of the fitness values for each individual mutant: <em>AF<sub>12</sub></em> = <em>AF</em><sub>1</sub> x <em>AF</em><sub>2</sub> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mani <em>et al.</em> 2008</a>). This is also called genetic neutrality and indicates that the two genes affect the phenotype independently. If <em>AF</em><sub>12</sub> &lt; <em>AF</em><sub>1</sub> x <em>AF</em><sub>2</sub>, then the phenotype of the double mutant is more severe than predicted by the model and these genes show an aggravating interaction. If, on the other hand, <em>AF</em><sub>12</sub> &gt; <em>AF</em><sub>1</sub> x <em>AF</em><sub>2</sub>, then the two genes show an alleviating or suppressing interaction.</p><p id="p-46">Traditionally, genetic interactions are expressed as Ɛ = <em>AF</em><sub>12</sub> – (<em>AF</em><sub>1</sub> x <em>AF</em><sub>2</sub>) where Ɛ &gt; 0 is an aggravating interaction, Ɛ &lt; 0 is an alleviating interaction and Ɛ = 0 is no interaction. We have provided Ɛ values for each double mutant pair in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS2.xlsx;">Table S2</a>. However, as a more easily evaluated graphical representation of the data, we plotted the expected AF <em>vs.</em> the observed AF for the all double mutants with the estimated error. Mutants that plotted above the diagonal show aggravating interactions while those below the diagonal display alleviating interactions. In addition, we required that a double mutant show a statistically significant difference from the expected value to be classified as an interacting pair of genes (Student’s <em>t</em>-test, <em>P</em> &lt; 0.05).</p><p id="p-47">An extensive set of genetic interactions was identified of the adhesion phenotype (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>) with only two gene-pairs out of the total 15 gene-pairs showing no interaction (See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS4.xlsx;">Table S4</a> for complete set of adhesion data and ε calculations). Three TFs [Tec1 (orf19.5908), Rob1 (orf19.4998), and Efg1 (orf19.610)] interacted with all five of the other TFs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). Interestingly, Tec1 (orf19.5908) showed an aggravating interaction with all other TFs. Thus, the aggravating interaction network for adhesion is centered on Tec1 (orf19.5908) as a central node (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). This relationship would not have been identified using single gene analysis because the <em>tec1</em>Δ/Δ (orf19.5908) mutant does not have a defect in adhesion (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>). Consequently, it appears that Tec1 (orf19.5908) functions in an ancillary or redundant manner to regulate the initial adhesion step of biofilm formation. In addition, our data indicate that Ndt80 (orf19.2119) and Brg1 (orf19.4056) also contribute to this step but likewise do so in a manner that overlaps with other TFs because their homozygous deletion mutants do not show a phenotype. Furthermore, our data are consistent with the notion that multiple TFs have overlapping functions during this step of biofilm formation.</p><p id="p-48">If <em>TEC1</em> (orf19.5908) plays a significant but ancillary role in adhesion, then its over-expression could be expected to suppress the adhesion defects displayed by other mutants. Therefore, we placed <em>TEC1</em> (orf19.5908) under control of the strong, constitutive promoter <em>TDH1</em> using the methods previously described by Finkel <em>et al.</em> (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Finkel <em>et al.</em> 2012</a>). As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, over-expression of <em>TEC1</em> (orf19.5908) increases adhesion in WT and <em>tec1</em>Δ (orf19.5908) strains. Additionally, increased expression of <em>TEC1</em> (orf19.5908) in all of the adhesion-deficient strains that we tested led to increased adhesion. Taken together, these data suggest that <em>TEC1</em> (orf19.5908) plays an important but ancillary role in the initial adhesion step of biofilm formation. In this role, it would function cooperatively with other TFs but be dispensable when a full complement of its functional partners is present.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overexpression of TEC1 increases adhesion of WT and suppresses adhesion defects of other biofilm TF deletion mutants. The mean adhesion of the indicated strains with and without an allele of TEC1 under the control of the strong, constitutive TDH1 promoter is shown by the bars with error bars indicating the standard deviation of duplicates or triplicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overexpression of &lt;em&gt;TEC1&lt;/em&gt; increases adhesion of WT and suppresses adhesion defects of other biofilm TF deletion mutants. The mean adhesion of the indicated strains with and without an allele of &lt;em&gt;TEC1&lt;/em&gt; under the control of the strong, constitutive TDH1 promoter is shown by the bars with error bars indicating the standard deviation of duplicates or triplicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F5.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/4/1299/F5.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489291" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-49" class="first-child">Overexpression of <em>TEC1</em> increases adhesion of WT and suppresses adhesion defects of other biofilm TF deletion mutants. The mean adhesion of the indicated strains with and without an allele of <em>TEC1</em> under the control of the strong, constitutive TDH1 promoter is shown by the bars with error bars indicating the standard deviation of duplicates or triplicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">A second interaction network consisting of only alleviating or suppressive interactions also emerges from adhesion data. In this case, a four-node structure connects Efg1 (orf19.610), Brg1 (orf19.4056), Bcr1 (orf19.723), and Rob1 (orf19.4998). Ndt80 (orf19.2119) has an alleviating interaction only with Efg1 (orf19.610). Alleviating interactions between two genes can indicate that the two genes function in a linear pathway and a “diminishing returns” effect limits the severity of two mutations that affect genes in the same pathway (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Mani <em>et al.</em> 2008</a>). Based on this interpretation, each of the nodes in the alleviating network would function in parallel pathways where reductions in gene dosage along one pathway are buffered. However, reduction in two genes/nodes that are not linearly-related disrupts both parallel pathways and leads to an aggravating interaction.</p></div><div id="sec-16" class="subsection"><h3>The biofilm TF network buffers adherent <em>C. albicans</em> cells From genetic perturbations that affect filament length</h3><p id="p-51">The second step of biofilm formation is considered to be filamentation of the adherent cells (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Desai and Mitchell 2015</a>). To our knowledge, this step of biofilm formation has not been probed genetically. To do so, we developed an assay to directly measure the length of hyphae adhered to glass chamber slides. We inoculated each chamber slide with a given strain in YETS medium. After 2 hr, the chambers were washed to remove non-adherent cells. The adherent cells were heat fixed and then examined by light microscopy. The length of the filament was measured using commercially available software. The filament was defined as the region between the angle of the mother cell-filament junction and the distal tip of the filamentous portion of the cell. We did not distinguish between pseudohyphae or hyphae. Mutants that formed no discernible hyphae were excluded from analysis. A representative set of images are shown for the reference strain, <em>tec1</em>Δ (orf19.5908), <em>bcr1</em>Δ (orf19.723), and <em>tec1</em>Δ <em>bcr1</em>Δ are shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genetic interactions of biofilm TFs during filamentation of adherent cells are distinct from adhesion and mature biofilm formation. A. Representative micrographs used to measure the filament length of the indicated strains. The lengths were determined as described in the Materials and Methods with the mean and standard deviation for the indicated strains derived from the measurement of at least one hunderd cells. B. The normalized filament length (mutant length/WT length) expected for a given double mutant was calculated as the product of the normalized mutant lengths of the two single mutants and is plotted on the y-axis. The observed filament length for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. The genetic interaction network of adherent cell filamentation centered on BCR1. Green indicates alleviating interactions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The genetic interactions of biofilm TFs during filamentation of adherent cells are distinct from adhesion and mature biofilm formation. A. Representative micrographs used to measure the filament length of the indicated strains. The lengths were determined as described in the Materials and Methods with the mean and standard deviation for the indicated strains derived from the measurement of at least one hunderd cells. B. The normalized filament length (mutant length/WT length) expected for a given double mutant was calculated as the product of the normalized mutant lengths of the two single mutants and is plotted on the y-axis. The observed filament length for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. The genetic interaction network of adherent cell filamentation centered on &lt;em&gt;BCR1&lt;/em&gt;. Green indicates alleviating interactions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F6.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/4/1299/F6.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489186" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-52" class="first-child">The genetic interactions of biofilm TFs during filamentation of adherent cells are distinct from adhesion and mature biofilm formation. A. Representative micrographs used to measure the filament length of the indicated strains. The lengths were determined as described in the Materials and Methods with the mean and standard deviation for the indicated strains derived from the measurement of at least one hunderd cells. B. The normalized filament length (mutant length/WT length) expected for a given double mutant was calculated as the product of the normalized mutant lengths of the two single mutants and is plotted on the y-axis. The observed filament length for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (ε = 0 ± estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. The genetic interaction network of adherent cell filamentation centered on <em>BCR1</em>. Green indicates alleviating interactions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">We first examined adherent cell filament formation using the homozygous biofilm TF deletion mutants. Consistent with previously reported filamentation phenotypes under other conditions (<a id="xref-ref-17-12" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>), none homozygous deletion biofilm TF mutants were able to filament with the exception of <em>NDT80</em> (orf19.2119) which formed shorter filaments than the reference strain. The average reference strain filament length was 92 ± 30 µm (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). Next, we determined whether the heterozygous TF deletion mutants were haploinsufficient or haploproficient. The filaments formed by five of the six heterozygous TF deletion mutants were between 11 and 20 µm shorter than the reference strain in a statistically significant manner (Mann-Whitney <em>U</em>-test, <em>P</em> &lt; 0.05). The complete set of filament length data and calculations for ε values are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS5.xlsx;">Table S5</a>. Although the filaments formed by <em>ndt80Δ</em> (orf19.2119) were longer than those formed by the reference strain, the difference was not statistically different that wild type.</p><p id="p-54">Next, the filament length of the double heterozygous biofilm TF mutants was measured; the genetic interactions within the double mutants were determined using calculations described earlier. We defined an interaction as a filament length that differed from the expected value by ≥ 30% based on the standard distribution of the lengths. As shown in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>, 7 out of 15 double heterozygote mutants showed genetic interactions. A number of double mutants had statistically significantly shorter filaments than either single mutants but ε = 0 for these strains, indicating that the genes did not interact and, hence, function independently. For example, the filaments of the <em>tec1 bcr1</em> double heterozygote are shorter than either single heterozygote but are equal to the length expected if each single mutant contributed independently to the observed phenotype (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A/B</a>). All of the double mutants that show genetic interactions have filaments that are longer than predicted by the multiplicative model, indicating that only alleviating or suppressive interactions were observed.</p><p id="p-55">As shown in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>, <em>BCR1</em> (orf19.723) interacts with all five of the network TFs. Consequently, <em>BCR1</em> (orf19.723) forms the central hub of a pair of interlocking three-node motifs involving <em>ROB1-BRG1</em> and <em>EFG1-NDT80</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2005</a>). Because this network is made entirely of alleviating interactions, it is quite distinct from either network observed during initial adhesion. It appears that the interacting TFs buffer the network against genetic perturbation with <em>BCR1</em> (orf19.723) playing a key role in this effect. These observations, taken together with the adherence analysis, suggest that Tec1 (orf19.5908) and Bcr1 (orf19.723) play important but distinct roles during the initial stages of <em>C. albicans</em> interaction with abiotic surfaces under biofilm inducing conditions.</p></div><div id="sec-17" class="subsection"><h3>Distinct TF interactions regulate filament length Under planktonic conditions relative to biofilm conditions</h3><p id="p-56">The extensive literature on <em>C. albicans</em> filamentation suggests that the genetic requirements for this process are dependent on specific stimuli (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Saputo <em>et al.</em> 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Azadmanesh <em>et al.</em> 2017</a>). We, therefore, were interested in comparing the genetic interactions of the biofilm regulators during planktonic filamentation to those for cells adhered to an abiotic surface. To do so, we took advantage of a flow cytometry-based imaging assay that both distinguishes yeast from filamentous cells and allows the measurement of filament length (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Among a number of advantages to this approach, it allows the rapid analysis of very large numbers of cells and, thereby, enables the detection of subtle changes in morphologic distribution with high statistical power. This is particularly advantageous for the analysis of simple and complex haploinsufficiency where phenotypes for individual mutants can be modest but informative.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genetic interaction network of biofilm TFs during planktonic filament formation is distinct from adherent cell filamentation. A. Plot of the distribution of filament lengths for WT and rob1&#x394; tec1&#x394; derived ImageStream-based analysis along with mean length (&#xB5;m) &#xB1; SD for both strains. B. The normalized filament length (mutant length/WT length) expected for a given double mutant was calculated as the product of the normalized mutant lengths of the two single mutants and is plotted on the y-axis. The observed filament length for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. Genetic interaction network for alleviating interactions centered on BRG1. D. Genetic interaction network for aggravating interactions centered on ROB1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The genetic interaction network of biofilm TFs during planktonic filament formation is distinct from adherent cell filamentation. A. Plot of the distribution of filament lengths for WT and &lt;em&gt;rob1&lt;/em&gt;&#x394; &lt;em&gt;tec1&lt;/em&gt;&#x394; derived ImageStream-based analysis along with mean length (&#xB5;m) &#xB1; SD for both strains. B. The normalized filament length (mutant length/WT length) expected for a given double mutant was calculated as the product of the normalized mutant lengths of the two single mutants and is plotted on the y-axis. The observed filament length for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. Genetic interaction network for alleviating interactions centered on &lt;em&gt;BRG1&lt;/em&gt;. D. Genetic interaction network for aggravating interactions centered on &lt;em&gt;ROB1&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F7.medium.gif" width="331" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/4/1299/F7.medium.gif" width="331" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489286" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-57" class="first-child">The genetic interaction network of biofilm TFs during planktonic filament formation is distinct from adherent cell filamentation. A. Plot of the distribution of filament lengths for WT and <em>rob1</em>Δ <em>tec1</em>Δ derived ImageStream-based analysis along with mean length (µm) ± SD for both strains. B. The normalized filament length (mutant length/WT length) expected for a given double mutant was calculated as the product of the normalized mutant lengths of the two single mutants and is plotted on the y-axis. The observed filament length for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (ε = 0 ± estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. Genetic interaction network for alleviating interactions centered on <em>BRG1</em>. D. Genetic interaction network for aggravating interactions centered on <em>ROB1</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">The biofilm-inducing medium used in the experiments described above (YETS) is not an efficient inducer of hyphae under planktonic conditions and, therefore, we used RPMI tissue culture medium at 37° for six hours to induce hyphae. Under these conditions, 90% of the reference strain SN250 formed hyphal cells (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>) while the homozygous deletions formed no filaments. All of the heterozygous and double heterozygous deletion mutants formed filaments with measurable lengths. The data for the full set single and double heterozygotes as well as the wild type control are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS6.xlsx">Table S6</a>. Two heterozygous mutants, <em>brg1</em>Δ (orf19.4056) and <em>rob1</em>Δ (orf19.4998), showed haploinsufficiency with statistically significant reductions in mean filament length of 27% and 20%, respectively.</p><p id="p-59">Using the definitions for genetic interactions described above for the analysis of cell length of adhered cells, eight aggravating and five alleviating interactions were identified. The <em>brg1</em>Δ (orf19.4056) mutant only participated in alleviating interactions and is a hub connecting a network of alleviating interactions as shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8B</a>. Indeed, deletion of any other TF in the <em>brg1</em>Δ background suppressed its defect in filament length. In addition, the <em>BRG1</em> (orf19.4056)-centered network contains two three-TF modules within the alleviating interaction network. This suggests that reduced copy number in other TFs triggers a compensatory response that overcomes the defects associated with reduced <em>BRG1</em> (orf19.4056) copy number.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genetic interaction network of biofilm TFs affecting the proportions of yeast and filamentous cells under planktonic conditions is similar to that for length under planktonic conditions. A. ImageStream gating diagrams showing WT and rob1&#x394; tec1&#x394; yeast and filamentous cells. B. The expected normalized proportion of filamentous cells (mutant/WT) for the double mutants was calculated as the product of the normalized filament proportion of the two single mutants and is plotted on the y-axis. The observed normalized filament proportion for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. The genetic interaction network for alleviating interactions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The genetic interaction network of biofilm TFs affecting the proportions of yeast and filamentous cells under planktonic conditions is similar to that for length under planktonic conditions. A. ImageStream gating diagrams showing WT and &lt;em&gt;rob1&lt;/em&gt;&#x394; &lt;em&gt;tec1&lt;/em&gt;&#x394; yeast and filamentous cells. B. The expected normalized proportion of filamentous cells (mutant/WT) for the double mutants was calculated as the product of the normalized filament proportion of the two single mutants and is plotted on the y-axis. The observed normalized filament proportion for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (&#x3B5; = 0 &#xB1; estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. The genetic interaction network for alleviating interactions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F8.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/4/1299/F8.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489272" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-60" class="first-child">The genetic interaction network of biofilm TFs affecting the proportions of yeast and filamentous cells under planktonic conditions is similar to that for length under planktonic conditions. A. ImageStream gating diagrams showing WT and <em>rob1</em>Δ <em>tec1</em>Δ yeast and filamentous cells. B. The expected normalized proportion of filamentous cells (mutant/WT) for the double mutants was calculated as the product of the normalized filament proportion of the two single mutants and is plotted on the y-axis. The observed normalized filament proportion for a given mutant is plotted on the x-axis. Mutants that fall within the zone defined by the diagonal (ε = 0 ± estimated error) have no genetic interaction while those above the diagonal (green) display alleviating interactions while those below (red) show aggravating interactions. C. The genetic interaction network for alleviating interactions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">In contrast to the filamention of the adherent cells, five aggravating interactions were observed under planktonic conditions. Specifically, the set of interactions organizes into a highly connected network centered on <em>ROB1</em> (orf19.4998). <em>ROB1</em> (orf19.4998) interacts with all of the other TFs with the exception of <em>BRG1</em> (orf19.4056) in a cooperative manner. In addition, each of the <em>ROB1</em> (orf19.4998) interacting genes also interacts cooperatively with two other TFs in this network. Thus, Rob1 (orf19.4998) and Brg1 (orf19.4056) have very distinct functional interactions during planktonic filamentation formation. Rob1 (orf19.4998) functions cooperatively with multiple TFs during planktonic filamentation. Brg1 (orf19.4056), on the other hand, does not appear to cooperate with any of the other TFs. However, reduced gene dosage of the TFs that function in the Rob1 (orf19.4998) network appears to lead to a compensatory response that suppresses the effect of decreased Brg1 (orf19.4056) function.</p><p id="p-62">Our results indicate that the network of genetic interactions controlling filament length varies dramatically depending on whether the cells are adherent to a glass surface or are suspended as planktonic cells. We cannot rule out the possibility that differences in culture medium (YETS <em>vs.</em> RMPI) contribute in some way to the reorganization of the TF network interactions. Because YETS medium does not induce planktonic cells to form true hyphae, we feel that signals specific to adherence are likely to make a significant contribution to the transcriptional program initiated during filamentation. Even if these changes are completely due to the medium, our data provide a dramatic example of how external cues reorganzie the specific interactions between a common set of TFs to yield a common morphological outcome.</p></div><div id="sec-18" class="subsection"><h3>The network of TF interactions regulating the ratio of yeast to filaments Under planktonic conditions is similar to that for planktonic filament length</h3><p id="p-63">Although the relative proportion of yeast and filament forms could not be used to assess filamentation under adherent conditions for technical reasons, the Image Stream technology allowed us to examine this phenotype under planktonic conditions (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>). As mentioned above, 90% of the reference strain SN250 formed filaments. Of the heterozygous mutants, <em>brg1</em>Δ (orf19.4056) was the only strain that showed a statistically significant reduction in the proportion of filamentous cells (78% hyphae, <em>P</em> &lt; 0.0001, Fisher’s exact test, N = 2720). Similar to the interactions observed for filament length, deletion of one allele of <em>ROB1</em> (orf19.4998), <em>BCR1</em> (orf19.723), and <em>EFG1</em> (orf19.610) in the <em>brg1</em>Δ (orf19.4056) background suppressed its haploinsufficiency and restored the <em>brg1</em>Δ double mutants to reference strain levels of filament formation (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>). The full set of data are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1/TableS6.xlsx">Table S6</a>.</p><p id="p-64">Consistent with the filament length data, <em>ROB1</em> (orf19.4998) showed aggravating interactions with <em>TEC1</em> (orf19.5908), <em>EFG1</em> (orf19.610), <em>BCR1</em> (orf19.723), and <em>NDT80</em> (orf19.2119) (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8B&amp;C</a>). <em>TEC1</em> (orf19.5908) and <em>EFG1</em> (orf19.610) also show an aggravating interaction. The most severely affected double mutant is the <em>rob1</em>Δ <em>tec1Δ</em> strain (60% hyphae, <em>P</em> &lt; 0.0001, Fisher’s exact test, N = 2468). The interaction network for aggravating interactions for this set of TFs is shown in <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8C</a>. <em>ROB1</em> (orf19.4998) appears to function as a local node and as a component of a three TF module with <em>EFG1</em> (orf19.610) and <em>TEC1</em> (orf19.5908). The most common function of three-component TF modules is as feed-forward loops (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mangan and Alon 2003</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2005</a>). Consistent with that formulation, Efg1 (orf19.610) is known to directly bind and regulate the expression of <em>TEC1</em> (orf19.5908). Nobile <em>et al.</em> has shown that Tec1 (orf19.5908) and Efg1 (orf19.610) bind to the promoter of <em>ROB1</em> (orf19.4998) (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Nobile <em>et al.</em> 2012</a>) and, in other work, our laboratory found that Tec1 (orf19.5908) and Efg1 (orf19.610) cooperate with Rob1 (orf19.4998) to regulate <em>ROB1</em> (orf19.4998) expression (<a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). Since all three of these TFs regulate genes that are involved in hyphae morphogenesis, it is likely that cooperative effects on target gene expression also contribute to the observed genetic interactions as well. The similarity in genetic interactions observed for filament length and proportion of filamentous cells during planktonic conditions indicates that these two phenotypes are controlled by very similar transcriptional responses.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-65">The genetic analysis of simple and complex heterozygotes can provide information that is complementary to that obtained by the analysis of homozygous deletion strains. In addition, genetic interaction analysis can be problematic in diploid organisms such as <em>C. albicans</em>, particularly if the gene of interest is essential or if the homozygous deletion mutant has a very strong phenotype. Here, we described a set of strains and plasmids designed to facilitate genetic interaction analysis in <em>C.albicans</em> using CHI. We have characterized phenotypes for this set of TF heterozygous deletion strains both <em>in vitro</em> and <em>in vivo</em> as a benchmark for subsequent studies. Consistent with previous studies of the prevalence of haploinsufficiency in other yeast, the phenotype is relatively uncommon and we identified only a handful of heterozygous strains that displayed altered function.</p><p id="p-66">The <em>in vitro</em> phenotype with the most haploinsufficient/proficient mutants was filamentation. Homann <em>et al.</em> found that eight homozygous TF deletion mutants were deficient for filamentation on Spider medium at 37° while an additional eight strains were hyperfilamentous (<a id="xref-ref-17-13" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). Of the eight homozygous mutants that showed decreased filamentation, three were also haploinsufficient. In contrast, all of the heterozygous counterparts of hyperfilamentous homozygous TF deletions were also hyperfilamentous. In addition, a ninth mutant, <em>ZFU3</em> (orf19.6888) showed increased filamentation. Although this strain was not hyperfilamentous in the data set reported by Homann <em>et al.</em> Vandeputte <em>et al.</em> reported that <em>zfu3</em>ΔΔ (orf19.6888) was hyperfilamentous (<a id="xref-ref-17-14" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Vandeputte <em>et al.</em> 2011</a>). The rate of haploinsufficiency among the negative transcriptional regulators of filamentation (100%) is strikingly above the rate for the overall genome and for other phenotypes that we tested with this set of mutants. Based on these data, it seems that <em>C. albicans</em> may be more sensitive to alterations in the gene dosage of negative regulators of hyphal morphogenesis than of positive regulators.</p><p id="p-67">We did not observe haploinsufficient growth phenotypes for the TF heterozygous deletion strains on solid agar plates with rich media and only one heterozygous deletion mutant, <em>UGA33</em> (orf19.7317), showed increased growth in competitive pooled assays in the same media. Homan <em>et al.</em> found that nine homozygous mutants had growth defects under the same conditions and, therefore, it is not surprising that no haploinsufficiency was observed given the low rate of its occurrence overall (<a id="xref-ref-17-15" class="xref-bibr" href="#ref-17">Homann <em>et al.</em> 2009</a>). In contrast, haploinsufficient strains were identified at either 24 or 48 hr during disseminated candidiasis in a mouse model. Although there have been previous competitive infectivity screens using homozygous deletion mutants, none had used the early time points that we employed. Nearly one-half of the TF heterozygotes with altered infectivity at early time points were mutants that had been previously shown to be involved in abiotic adhesion under flow conditions (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Finkel <em>et al.</em> 2012</a>). Although we have not determined whether these strains are also haploinsufficient for adherence to abiotic surfaces under flow conditions, it is reasonable that our experiment would identify such mutants since endothelial cell attachment under conditions of flow is a crucial and early step in the establishment of a disseminated infection.</p><p id="p-68">We used a set of double heterozygous mutants derived from TFs previously shown to regulate biofilm formation to genetically characterize two early steps in the establishment <em>C. albicans</em> biofilm (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Nobile <em>et al.</em> 2012</a>; <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>): adherence to an abiotic surface and filamentation. Previously, we had used the same set of double mutants to study the genetic interactions at the point when a relatively mature biofilm had formed, <em>i.e.</em>, 48 hr (<a id="xref-ref-13-10" class="xref-bibr" href="#ref-13">Glazier <em>et al.</em> 2017</a>). At the stage of initial adherence, we found that the set of six TFs displayed a very different set of intra-network genetic interactions relative those observed for mature biofilms. Thus, the roles of the TFs vary depending on the specific stage of biofilm formation. This is consistent with the recently reported homozygous deletion study of the same set of mutants (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>). Combined with those results, our data support a model in which different stages of biofilm formation specify different combinations of TFs for optimal function.</p><p id="p-69">We also found that <em>TEC1</em> (orf19.5908) plays an important role that was not evident from analysis of the homozygous deletion mutants. Deletion of one allele of <em>TEC1</em> (orf19.5908) in the context of a heterozgous deletion mutation in any of the other TFs led to an aggravating interaction, indicating that <em>TEC1</em> (orf19.5908) functions cooperatively with the other TFs. Consequently, Tec1 appears to be required to buffer the cell against reduced gene dosage in other biofilm-related TFs during the initial adhesion step. Further supporting this role for Tec1 (orf19.5908) is the fact that overexpression of <em>TEC1</em> (orf19.5908) increases adhesion of wild type, single heterozygotes, and double heterozygotes with decreased biofilm adhesion. Tec1 (orf19.5908) is bound by and regulated by all other network TFs during biofilm formation (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Nobile <em>et al.</em> 2012</a>) as well as by itself. The genetics of its interaction with other TFs, taken together with its regulation pattern, suggest that <em>TEC1</em> (orf19.5908) is an important output of the TF network. Furthermore, it interacts with all of the other five TFs as part of three, three-module motifs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>). Within genetic interaction networks, three module TF motifs commonly represent feed-forward loops (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Mangan and Alon 2003</a>). Thus, the structure of the adhesion network strongly supports the notion that all of the other TFs contribute to <em>TEC1</em> (orf19.5908) expression through a feed-forward mechanism. Finally, this overall model is supported by the transcriptional profiling data reported by Fox <em>et al.</em> in that <em>TEC1</em> expression is elevated in adherent cells as compared to planktonic cells grown at the same temperature (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Fox <em>et al.</em> 2015</a>).</p><p id="p-70">One of the most potentially informative features of genetic interaction studies is its ability to provide insights into how sets of genes with common functions work together to regulate a biological process (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Costanzo <em>et al.</em> 2016</a>). Taking the genetic interactions for the biofilm-related TFs for the early stages of biofilm formation together with our previous data for the genetic interactions at a much later stage of the biofilm process (48 hr), it is clear that the network is not a static set of interactions but is instead more malleable in nature. Taking the set of Tec1 (orf19.5908) and Bcr1 (orf19.723) interactions as illustrative examples (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>), the variable nature of their interactions with other network TF over the course of biofilm formation becomes quite clear. At first glance, it may seem that the interactions of <em>BCR1</em> (orf19.723) are relatively stable because it interacts with most of the TFs at each step of the process. However, only one of its thirteen interactions, an alleviating interaction with <em>BCR1</em> (orf19.723), is consistently maintained across each stage of biofilm formation. The interactions displayed <em>TEC1</em> (orf19.5908) are drastically different at each stage and none are consistently manifest at each stage. These dynamic reorganizations are characteristic of a temporal network, as opposed to a static network (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2017</a>). One of the advantages of temporal networks is that they are able to control a wide range of functions with a relatively small number of nodes (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2017</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genetic interactions of the biofilm TFs are dynamic and are temporally reorganized during biofilm formation. The genetic interaction networks for BCR1 and TEC1 are shown for adhesion, adherent cell filamentation, and mature biofilm stages of biofilm formation. The interaction networks for the mature biofilm stage are derived from Glazier et al. 2017." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-516162839" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The genetic interactions of the biofilm TFs are dynamic and are temporally reorganized during biofilm formation. The genetic interaction networks for &lt;em&gt;BCR1&lt;/em&gt; and &lt;em&gt;TEC1&lt;/em&gt; are shown for adhesion, adherent cell filamentation, and mature biofilm stages of biofilm formation. The interaction networks for the mature biofilm stage are derived from Glazier &lt;em&gt;et al.&lt;/em&gt; 2017.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1299/F9.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/4/1299/F9.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1299/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489202" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-71" class="first-child">The genetic interactions of the biofilm TFs are dynamic and are temporally reorganized during biofilm formation. The genetic interaction networks for <em>BCR1</em> and <em>TEC1</em> are shown for adhesion, adherent cell filamentation, and mature biofilm stages of biofilm formation. The interaction networks for the mature biofilm stage are derived from Glazier <em>et al.</em> 2017.</p><div class="sb-div caption-clear"></div></div></div><p id="p-72">The significant changes in biofilm TF network topology and function also fit well with Luscombe <em>et al.</em> who showed that, in <em>S. cerevisiae</em>, TFs do not necessarily serve as permanent hubs but instead frequently had transient roles leading to significant changes in network topology in response, in particular, to environmental cues (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Luscombe <em>et al.</em> 2004</a>). During biofilm formation, the cell’s microenvironment alters dramatically as the the process progresses and likely provides the external cues leading to the reorganization of the network topology and the specific alterations in the roles of the specific TFs within the network. Consequently, the genetic interactions of the <em>C. albicans</em> biofilm TFs provide insights into how a relatively small set of TFs is able to orchestrate the multistep process of constructing a biofilm. What remains to be explored are the mechanisms by which the cell is able to reorganize the specific contributions of the individual TFs to execute each step of this complex process.</p><p id="p-73">In summary, we have constructed a set of strains and plasmids for the systematic genetic interaction analysis of <em>C. albicans</em> TFs based on simple and complex haploinsufficiency. In addition to providing these tools and baseline phenotypic characterization for the strains, we have used the library to probe genetics regulation of the early stages of biofilm formation and filamentation. The latter analyses provide genetic support for the notion that the TF interaction networks are quite plastic with specific combinations of TFs working together under specific conditions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-74">We thank Jose Lopez-Ribot (UTSA) for assistance with flow cytometry-based assays; Scott Filler (UCLA) and Aaron Mitchell (Carnegie-Mellon) for discussion and advice with manuscript preparation. This work was supported by NIH grants F32AI26634 (VEG) and 1R01AI098450 (DJK).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Current Address: Department of Pediatrics, Carver College of Medicine, Iowa City, Iowa 52242.</p></li><li class="fn" id="fn-6"><p id="p-75"><em>Communicating editor: J. Berman</em></p></li><li class="fn" id="fn-7"><p id="p-76">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300515/-/DC1</a>.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-12-11"><span class="received-label">Received </span>December 11, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-02-18"><span class="accepted-label">Accepted </span>February 18, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Glazier <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.1" data-doi="10.3389/fmicb.2015.00367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amorim-Vaz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delarze</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ischer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coste</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Examining the virulence of <em>Candida albicans</em> transcription factor mutants using <em>Galleria mellonella</em> and mouse infection models.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2015.00367</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D6%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2015.00367%26rft_id%253Dinfo%253Apmid%252F25999923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2015.00367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25999923&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.2" data-doi="10.1534/g3.117.300224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azadmanesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowen</span>  <span class="cit-name-given-names">A.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Creger</span>  <span class="cit-name-given-names">P.E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">N.D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenship</span>  <span class="cit-name-given-names">J.R.</span></span></li></ol><cite>, 2017 Filamentation involves two overlapping, but distinct, programs of filamentation in the pathogenic fungus <em>Candida albicans</em>. G3 (Bethesda). pii: g3.300224.<span class="cit-pub-date">2017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300224</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300224%26rft_id%253Dinfo%253Apmid%252F28951491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjcvMTEvMzc5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.3" data-doi="10.1371/journal.pgen.1002058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bharucha</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabrier-Rosello</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobczynski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large-scale complex haploinsufficiency-based genetic interaction screen in <em>Candida albicans</em>: analysis of the RAM network during morphogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e1002058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBharucha%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002058%26rft.epage%253De1002058%26rft.atitle%253DA%2Blarge-scale%2Bcomplex%2Bhaploinsufficiency-based%2Bgenetic%2Binteraction%2Bscreen%2Bin%2BCandida%2Balbicans%253A%2Banalysis%2Bof%2Bthe%2BRAM%2Bnetwork%2Bduring%2Bmorphogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002058%26rft_id%253Dinfo%253Apmid%252F22103005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22103005&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.4" data-doi="10.1534/g3.116.037986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaillot</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbeil</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sellam</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide screen for haploinsufficient cell size genes in the opportunistic yeast <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.037986</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.037986%26rft_id%253Dinfo%253Apmid%252F28040776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi8zNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.5" data-doi="10.1111/j.1365-2958.2012.08127.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleary</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazzell</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteagudo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saville</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>BRG1</em> and <em>NRG1</em> form a novel feedback circuit regulating <em>Candida albicans</em> hypha formation and virulence.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">557</span>–<span class="cit-lpage">573</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2012.08127.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DCleary%26rft.auinit1%253DI.%2BA.%26rft.volume%253D85%26rft.issue%253D3%26rft.spage%253D557%26rft.epage%253D573%26rft.atitle%253DBRG1%2Band%2BNRG1%2Bform%2Ba%2Bnovel%2Bfeedback%2Bcircuit%2Bregulating%2BCandida%2Balbicans%2Bhypha%2Bformation%2Band%2Bvirulence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08127.x%26rft_id%253Dinfo%253Apmid%252F22757963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08127.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22757963&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.6" data-doi="10.1126/science.aaf1420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A global genetic interaction network maps a wiring diagram of cellular function.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>(<span class="cit-issue">6306</span>): <span class="cit-fpage">aaf1420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aaf1420</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D353%26rft.issue%253D6306%26rft.spage%253Daaf1420%26rft.epage%253Daaf1420%26rft.atitle%253DA%2Bglobal%2Bgenetic%2Binteraction%2Bnetwork%2Bmaps%2Ba%2Bwiring%2Bdiagram%2Bof%2Bcellular%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf1420%26rft_id%253Dinfo%253Apmid%252F27708008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTMvNjMwNi9hYWYxNDIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.7" data-doi="10.1128/EC.00307-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dabas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Control of ammonium permease expression and filamentous growth by the GATA transcription factors <em>GLN3</em> and <em>GAT1</em> in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">875</span>–<span class="cit-lpage">888</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00307-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00307-06%26rft_id%253Dinfo%253Apmid%252F17369441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI2LzUvODc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.8" data-doi="10.1128/microbiolspec.MB-0005-2014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Candida albicans</em> biofilm development and Igenetic control.</span> <abbr class="cit-jnl-abbrev">Microbiol. Spectr.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/microbiolspec.MB-0005-2014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BSpectr.%26rft.volume%253D3%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fmicrobiolspec.MB-0005-2014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/microbiolspec.MB-0005-2014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.9" data-doi="10.1534/genetics.104.036871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.036871</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036871%26rft_id%253Dinfo%253Apmid%252F15716499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.10" data-doi="10.1371/journal.ppat.1002525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finkel</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Portrait of <em>Candida albicans</em> adherence regulators.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1002525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1002525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DFinkel%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1002525%26rft.epage%253De1002525%26rft.atitle%253DPortrait%2Bof%2BCandida%2Balbicans%2Badherence%2Bregulators.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002525%26rft_id%253Dinfo%253Apmid%252F22359502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22359502&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.11" data-doi="10.1111/mmi.13002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bui</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nett</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartooni</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mui</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An expanded regulatory network temporally controls <em>Candida albicans</em> biofilm formation.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">96</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1226</span>–<span class="cit-lpage">1239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mmi.13002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D96%26rft.spage%253D1226%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.13002%26rft_id%253Dinfo%253Apmid%252F25784162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.13002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25784162&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.12" data-doi="10.1099/mic.0.064097-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerami-Nejad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zacchi</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Shuttle vectors for facile gap repair cloning and integration into a neutral locus in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">159</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">565</span>–<span class="cit-lpage">579</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mic.0.064097-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.064097-0%26rft_id%253Dinfo%253Apmid%252F23306673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.064097-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23306673&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.13" data-doi="10.1371/journal.pgen.1006948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glazier</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murante</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murante</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koselny</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic analysis of the <em>Candida albicans</em> biofilm transcription factor network using simple and complex haploinsufficiency.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1006948</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006948</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006948%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.14" data-doi="10.1101/gad.1477507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haarer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viggiano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hibbs</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amberg</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling complex genetic interactions in a simple eukaryotic genome: actin displays a rich spectrum of complex haploinsufficiencies.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">148</span>–<span class="cit-lpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1477507</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1477507%26rft_id%253Dinfo%253Apmid%252F17167106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjEvMi8xNDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.15" data-doi="10.1016/S0076-6879(10)70031-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernday</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitrovich</span>  <span class="cit-name-given-names">Q. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetics and molecular biology in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">758</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0076-6879(10)70031-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DHernday%26rft.auinit1%253DA.%2BD.%26rft.volume%253D470%26rft.spage%253D737%26rft.epage%253D758%26rft.atitle%253DGenetics%2Band%2Bmolecular%2Bbiology%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252810%252970031-8%26rft_id%253Dinfo%253Apmid%252F20946834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(10)70031-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275827900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.16" data-doi="10.1038/nature11865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickman</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The ‘obligate diploid’ <em>Candida albicans</em> forms mating-competent haploids.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>(<span class="cit-issue">7435</span>): <span class="cit-fpage">55</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11865</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11865%26rft_id%253Dinfo%253Apmid%252F23364695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23364695&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314741200032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.17" data-doi="10.1371/journal.pgen.1000783"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Homann</span>  <span class="cit-name-given-names">O. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dea</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A phenotypic profile of the <em>Candida albicans</em> regulatory network.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1000783</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000783</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHomann%26rft.auinit1%253DO.%2BR.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De1000783%26rft.epage%253De1000783%26rft.atitle%253DA%2Bphenotypic%2Bprofile%2Bof%2Bthe%2BCandida%2Balbicans%2Bregulatory%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000783%26rft_id%253Dinfo%253Apmid%252F20041210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20041210&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.18" data-doi="10.1111/j.1365-2958.2004.04185.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacCallum</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clancy</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odds</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The <em>Candida albicans</em> Ca<em>ACE2</em> gene affects morphogenesis, adherence and virulence.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">969</span>–<span class="cit-lpage">983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2004.04185.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKelly%26rft.auinit1%253DM.%2BT.%26rft.volume%253D53%26rft.issue%253D3%26rft.spage%253D969%26rft.epage%253D983%26rft.atitle%253DThe%2BCandida%2Balbicans%2BCaACE2%2Bgene%2Baffects%2Bmorphogenesis%252C%2Badherence%2Band%2Bvirulence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2004.04185.x%26rft_id%253Dinfo%253Apmid%252F15255906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2004.04185.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15255906&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.19" data-doi="10.1128/jb.174.1.222-232.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon-Chung</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A zinc finger protein from <em>Candida albicans</em> is involved in sucrose utilization.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">222</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/jb.174.1.222-232.1992</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKelly%26rft.auinit1%253DR.%26rft.volume%253D174%26rft.issue%253D1%26rft.spage%253D222%26rft.epage%253D232%26rft.atitle%253DA%2Bzinc%2Bfinger%2Bprotein%2Bfrom%2BCandida%2Balbicans%2Bis%2Binvolved%2Bin%2Bsucrose%2Butilization.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.1.222-232.1992%26rft_id%253Dinfo%253Apmid%252F1729210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc0LzEvMjIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.20" data-doi="10.1101/cshperspect.a019273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Köhler</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perfect</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The spectrum of fungi that infects humans.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Med.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">a019273</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a019273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a019273%26rft_id%253Dinfo%253Apmid%252F25367975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNzaHBlcnNwZWN0bWVkIjtzOjU6InJlc2lkIjtzOjExOiI1LzEvYTAxOTI3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.21" data-doi="10.1126/science.aai7488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barabasi</span>  <span class="cit-name-given-names">A.-I.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The fundamental advantages of temporal networks.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">358</span>(<span class="cit-issue">6366</span>): <span class="cit-fpage">1042</span>–<span class="cit-lpage">1046</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aai7488</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DA.%26rft.volume%253D358%26rft.issue%253D6366%26rft.spage%253D1042%26rft.epage%253D1046%26rft.atitle%253DThe%2Bfundamental%2Badvantages%2Bof%2Btemporal%2Bnetworks%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aai7488%26rft_id%253Dinfo%253Apmid%252F29170233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTgvNjM2Ni8xMDQyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.22" data-doi="10.1128/EC.00199-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Roles of <em>Candida albicans</em> Sfl1 in hyphal development.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2112</span>–<span class="cit-lpage">2121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00199-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00199-07%26rft_id%253Dinfo%253Apmid%252F17715361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI2LzExLzIxMTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.23" data-doi="10.1128/AAC.02749-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gulati</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valle Arevalo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishburn</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Assessment and optimizations of <em>Candida albicans</em> in vitro biofilm assays.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">61</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e02749-16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AAC.02749-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.02749-16%26rft_id%253Dinfo%253Apmid%252F28289028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjE0OiI2MS81L2UwMjc0OS0xNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.24" data-doi="10.1038/nature02782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luscombe</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teichmann</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genomic analysis of regulatory network dynamics reveals large topological changes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>(<span class="cit-issue">7006</span>): <span class="cit-fpage">308</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02782</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLuscombe%26rft.auinit1%253DN.%2BM.%26rft.volume%253D431%26rft.issue%253D7006%26rft.spage%253D308%26rft.epage%253D312%26rft.atitle%253DGenomic%2Banalysis%2Bof%2Bregulatory%2Bnetwork%2Bdynamics%2Breveals%2Blarge%2Btopological%2Bchanges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02782%26rft_id%253Dinfo%253Apmid%252F15372033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15372033&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223864000041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.25" data-doi="10.1073/pnas.2133841100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alon</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Structure and function of the feed-forward loop network motif.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">11980</span>–<span class="cit-lpage">11985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.2133841100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2133841100%26rft_id%253Dinfo%253Apmid%252F14530388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIxLzExOTgwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.26" data-doi="10.1073/pnas.0712255105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. L.</span>  <span class="cit-name-suffix">4th</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">F. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Defining genetic interaction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3461</span>–<span class="cit-lpage">3466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0712255105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712255105%26rft_id%253Dinfo%253Apmid%252F18305163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzkvMzQ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.27" data-doi="10.1128/MCB.25.21.9435-9446.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Divergence of Stp1 and Stp2 transcription factors in <em>Candida albicans</em> places virulence factors required for proper nutrient acquisition under amino acid control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">9435</span>–<span class="cit-lpage">9446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.25.21.9435-9446.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMartinez%26rft.auinit1%253DP.%26rft.volume%253D25%26rft.issue%253D21%26rft.spage%253D9435%26rft.epage%253D9446%26rft.atitle%253DDivergence%2Bof%2BStp1%2Band%2BStp2%2BTranscription%2BFactors%2Bin%2BCandida%2Balbicans%2BPlaces%2BVirulence%2BFactors%2BRequired%2Bfor%2BProper%2BNutrient%2BAcquisition%2Bunder%2BAmino%2BAcid%2BControl%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.21.9435-9446.2005%26rft_id%253Dinfo%253Apmid%252F16227594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8yMS85NDM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.28" data-doi="10.1128/mSphereDirect.00149-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernday</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An efficient, rapid, and recyclable system for CRISPR-mediated genome editing in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">MSphere</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e00149-17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mSphereDirect.00149-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.stitle%253DmSphere%26rft.aulast%253DNguyen%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253De00149-17%26rft.epage%253De00149-17%26rft.atitle%253DAn%2BEfficient%252C%2BRapid%252C%2Band%2BRecyclable%2BSystem%2Bfor%2BCRISPR-Mediated%2BGenome%2BEditing%2Bin%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmSphereDirect.00149-17%26rft_id%253Dinfo%253Apmid%252F28497115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibXNwaCI7czo1OiJyZXNpZCI7czoxMzoiMi8yL2UwMDE0OS0xNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.29" data-doi="10.1016/j.cell.2011.10.048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nobile</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nett</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitrovich</span>  <span class="cit-name-given-names">Q. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A recently evolved transcriptional network controls biofilm development in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>(<span class="cit-issue">1-2</span>): <span class="cit-fpage">126</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2011.10.048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNobile%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D148%26rft.issue%253D1-2%26rft.spage%253D126%26rft.epage%253D138%26rft.atitle%253DA%2Brecently%2Bevolved%2Btranscriptional%2Bnetwork%2Bcontrols%2Bbiofilm%2Bdevelopment%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.10.048%26rft_id%253Dinfo%253Apmid%252F22265407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.10.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22265407&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299540700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.30" data-doi="10.1128/EC.4.2.298-309.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Strains and strategies for large-scale gene deletion studies of the diploid human fungal pathogen <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">298</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.4.2.298-309.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D298%26rft.epage%253D309%26rft.atitle%253DStrains%2Band%2BStrategies%2Bfor%2BLarge-Scale%2BGene%2BDeletion%2BStudies%2Bof%2Bthe%2BDiploid%2BHuman%2BFungal%2BPathogen%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.2.298-309.2005%26rft_id%253Dinfo%253Apmid%252F15701792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI0LzIvMjk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.31" data-doi="10.1038/ng.605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohn</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Systematic screens of a <em>Candida albicans</em> homozygous deletion library decouple morphogenetic switching and pathogenicity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">590</span>–<span class="cit-lpage">598</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.605</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D590%26rft.epage%253D598%26rft.atitle%253DSystematic%2Bscreens%2Bof%2Ba%2BCandida%2Balbicans%2Bhomozygous%2Bdeletion%2Blibrary%2Bdecouple%2Bmorphogenetic%2Bswitching%2Band%2Bpathogenicity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.605%26rft_id%253Dinfo%253Apmid%252F20543849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20543849&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.32" data-doi="10.1371/journal.ppat.1001140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlecht</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene annotation and drug target discovery in <em>Candida albicans</em> with a tagged transposon mutant collection.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1001140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DOh%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De1001140%26rft.epage%253De1001140%26rft.atitle%253DGene%2Bannotation%2Band%2Bdrug%2Btarget%2Bdiscovery%2Bin%2BCandida%2Balbicans%2Bwith%2Ba%2Btagged%2Btransposon%2Bmutant%2Bcollection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1001140%26rft_id%253Dinfo%253Apmid%252F20949076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1001140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20949076&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.33" data-doi="10.1128/IAI.72.12.7330-7333.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onyewu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wormley</span>  <span class="cit-name-given-names">F. L.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perfect</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The calcineurin target, Crz1, functions in azole tolerance but is not required for virulence of <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">72</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">7330</span>–<span class="cit-lpage">7333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/IAI.72.12.7330-7333.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DOnyewu%26rft.auinit1%253DC.%26rft.volume%253D72%26rft.issue%253D12%26rft.spage%253D7330%26rft.epage%253D7333%26rft.atitle%253DThe%2BCalcineurin%2BTarget%252C%2BCrz1%252C%2BFunctions%2Bin%2BAzole%2BTolerance%2Bbut%2BIs%2BNot%2BRequired%2Bfor%2BVirulence%2Bof%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.72.12.7330-7333.2004%26rft_id%253Dinfo%253Apmid%252F15557662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjEwOiI3Mi8xMi83MzMwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.34" data-doi="10.1371/journal.pbio.1001510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumamoto</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Candida albicans</em> commensalism and pathogenicity are intertwined traits directed by a tightly knit transcriptional regulatory circuit.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1001510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001510%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001510%26rft_id%253Dinfo%253Apmid%252F23526879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23526879&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.35" data-doi="10.1074/jbc.M603107200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aronova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Caffeine targets TOR complex I and provides evidence for a regulatory link between the FRB and kinase domains of Tor1p.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>(<span class="cit-issue">42</span>): <span class="cit-fpage">31616</span>–<span class="cit-lpage">31626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M603107200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603107200%26rft_id%253Dinfo%253Apmid%252F16923813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDIvMzE2MTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.36" data-doi="10.1534/genetics.116.188029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saputo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murante</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">J. D. L. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Complex haploinsufficiency-based genetic analysis of the NDR/Lats Kinase Cbk1 provides insight into its multiple functions in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1217</span>–<span class="cit-lpage">1233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.188029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.188029%26rft_id%253Dinfo%253Apmid%252F27206715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy8zLzEyMTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.37" data-doi="10.1128/EC.00044-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saputo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabrier-Rosello</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luca</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krysan</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The RAM network in pathogenic fungi.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">708</span>–<span class="cit-lpage">717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00044-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00044-12%26rft_id%253Dinfo%253Apmid%252F22544903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMS82LzcwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.38" data-doi="10.1046/j.1365-2958.2000.02132.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rupp</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röllinghoff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schröppel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The TEA/ATTS transcription factor CaTec1p regulates hyphal development and virulence in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">435</span>–<span class="cit-lpage">445</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2958.2000.02132.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSchweizer%26rft.auinit1%253DA.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D435%26rft.epage%253D445%26rft.atitle%253DThe%2BTEA%252FATTS%2Btranscription%2Bfactor%2BCaTec1p%2Bregulates%2Bhyphal%2Bdevelopment%2Band%2Bvirulence%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2000.02132.x%26rft_id%253Dinfo%253Apmid%252F11069668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2000.02132.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11069668&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165404000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.39" data-doi="10.1128/AAC.01959-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shekhar-Guturja</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tebung</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Köhler</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Beauvericin potentiates azole activity via inhibition of multidrug efflux, blocks <em>Candida albicans</em> morphogenesis, and Is effluxed via Yor1 and circuitry controlled by Zcf29.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">7468</span>–<span class="cit-lpage">7480</span>. <a href="https://doi.org/10.1128/AAC.01959-16">https://doi.org/10.1128/AAC.01959-16</a>doi:</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.01959-16%26rft_id%253Dinfo%253Apmid%252F27736764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjEwOiI2MC8xMi83NDY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.40" data-doi="10.1128/EC.00291-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spiering</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauvel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccallum</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative transcript profiling of <em>Candida albicans</em> and <em>Candida dubliniensis</em> identifies <em>SFL2</em>, a <em>C. albicans</em> gene required for virulence in a reconstituted epithelial infection model.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">251</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00291-09</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00291-09%26rft_id%253Dinfo%253Apmid%252F20023067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMjUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Unlinked noncomplementation: isolation of new conditional-lethal mutations in each of the tubulin genes of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStearns%26rft.auinit1%253DT.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D260%26rft.atitle%253DUnlinked%2Bnoncomplementation%253A%2Bisolation%2Bof%2Bnew%2Bconditional-lethal%2Bmutations%2Bin%2Beach%2Bof%2Bthe%2Btubulin%2Bgenes%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F3294100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE5LzIvMjQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.42" data-doi="10.1126/science.1091317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Global mapping of the yeast genetic interaction network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>(<span class="cit-issue">5659</span>): <span class="cit-fpage">808</span>–<span class="cit-lpage">813</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1091317</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D303%26rft.issue%253D5659%26rft.spage%253D808%26rft.epage%253D813%26rft.atitle%253DGlobal%2BMapping%2Bof%2Bthe%2BYeast%2BGenetic%2BInteraction%2BNetwork%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1091317%26rft_id%253Dinfo%253Apmid%252F14764870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1OS84MDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.43" data-doi="10.1093/emboj/cdg256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uhl</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biery</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craig</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Haploinsufficiency-based large-scale forward genetic analysis of filamentous growth in the diploid human fungal pathogen <em>C.albicans</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2668</span>–<span class="cit-lpage">2678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/emboj/cdg256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DUhl%26rft.auinit1%253DM.A.%26rft.volume%253D22%26rft.issue%253D11%26rft.spage%253D2668%26rft.epage%253D2678%26rft.atitle%253DHaploinsufficiency-based%2Blarge-scale%2Bforward%2Bgenetic%2Banalysis%2Bof%2Bfilamentous%2Bgrowth%2Bin%2Bthe%2Bdiploid%2Bhuman%2Bfungal%2Bpathogen%2BC.albicans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdg256%26rft_id%253Dinfo%253Apmid%252F12773383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTEvMjY2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.44" data-doi="10.1371/journal.pone.0026962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vandeputte</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ischer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coste</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">In vivo systematic analysis of <em>Candida albicans</em> Zn2-Cys6 transcription factors mutants for mice organ colonization.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e26962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0026962</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVandeputte%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De26962%26rft.epage%253De26962%26rft.atitle%253DIn%2Bvivo%2Bsystematic%2Banalysis%2Bof%2BCandida%2Balbicans%2BZn2-Cys6%2Btranscription%2Bfactors%2Bmutants%2Bfor%2Bmice%2Borgan%2Bcolonization.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0026962%26rft_id%253Dinfo%253Apmid%252F22073120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0026962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22073120&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.45" data-doi="10.1128/EC.00221-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vasicek</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berkow</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flowers</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>UPC2</em> is universally essential for azole antifungal resistance in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">933</span>–<span class="cit-lpage">946</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00221-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00221-13%26rft_id%253Dinfo%253Apmid%252F24659578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMy83LzkzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.46" data-doi="10.1126/sciadv.1500248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrasa</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A <em>Candida albicans</em> CRISPR system permits genetic engineering of essential genes and gene families.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1500248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/sciadv.1500248</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DVyas%26rft.auinit1%253DV.%2BK.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253De1500248%26rft.epage%253De1500248%26rft.atitle%253DA%2BCandida%2Balbicans%2BCRISPR%2Bsystem%2Bpermits%2Bgenetic%2Bengineering%2Bof%2Bessential%2Bgenes%2Band%2Bgene%2Bfamilies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1500248%26rft_id%253Dinfo%253Apmid%252F25977940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS8zL2UxNTAwMjQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.47" data-doi="10.1371/journal.ppat.0030092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ketela</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemieux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veillette</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide fitness test and mechanism-of-action studies of inhibitory compounds in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.0030092</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DXu%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De92%26rft.epage%253De92%26rft.atitle%253DGenome-wide%2Bfitness%2Btest%2Band%2Bmechanism-of-action%2Bstudies%2Bof%2Binhibitory%2Bcompounds%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0030092%26rft_id%253Dinfo%253Apmid%252F17604452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0030092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17604452&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.48" data-doi="10.4161/viru.28893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sui</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular genetic techniques for gene manipulation in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Virulence</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">507</span>–<span class="cit-lpage">520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/viru.28893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirulence%26rft.volume%253D5%26rft.spage%253D507%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fviru.28893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/viru.28893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.49" data-doi="10.1186/jbiol23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Motifs, themes and thematic maps of an integrated <em>Saccharomyces cerevisiae</em> interaction network.</span> <abbr class="cit-jnl-abbrev">J. Biol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/jbiol23</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiology%2B%2528Online%2529%26rft.stitle%253DJ%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DL.%2BV.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D6%26rft.epage%253D6%26rft.atitle%253DMotifs%252C%2Bthemes%2Band%2Bthematic%2Bmaps%2Bof%2Ban%2Bintegrated%2BSaccharomyces%2Bcerevisiae%2Binteraction%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fjbiol23%26rft_id%253Dinfo%253Apmid%252F15982408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/jbiol23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15982408&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1299.50" data-doi="10.1371/journal.ppat.1003519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Znaidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesseir</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauvel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d’Enfert</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A comprehensive functional portrait of two heat shock factor-type transcriptional regulators involved in <em>Candida albicans</em> morphogenesis and virulence.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1003519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1003519</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003519%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003519%26rft_id%253Dinfo%253Apmid%252F23966855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23966855&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/4/1299.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1291" title="Photosensitive Alternative Splicing of the Circadian Clock Gene timeless Is Population Specific in a Cold-Adapted Fly, Drosophila montana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1315" title="Examination of Signatures of Recent Positive Selection on Genes Involved in Human Sialic Acid Biology" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/4"><span>Volume 8 Issue 4, April 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/4.cover-source.jpg?itok=wC37V0ox" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1299.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-APAd41mBkcOw67nnbxLD4M1XYqfT0ODG82yF_Fgu82w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/4/1299" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/4/1299" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/4/1299" class="active" data-icon-position="" data-hide-link-title="0">Systematic Complex Haploinsufficiency-Based Genetic Analysis of Candida albicans Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489123" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5d5ra8odp0Lm5HORajUVWVqw2lT_Ds2XSF4yhl8rXt8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1299.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/4/1299" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1ErPjeuOsUjr1ZF9NVcxelqEPxgpl9Sk5Qn78Hih4Lo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/4/1299" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1299.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489123" id="node489123--2" data-pisa="ggg;8/4/1299" data-pisa-master="ggg;g3.117.300515" data-apath="/ggg/8/4/1299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systematic Complex Haploinsufficiency-Based Genetic Analysis of <em>Candida albicans</em> Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Virginia E. Glazier</span>, <span class="highwire-citation-author" data-delta="1">Thomas Murante</span>, <span class="highwire-citation-author" data-delta="2">Kristy Koselny</span>, <span class="highwire-citation-author" data-delta="3">Daniel Murante</span>, <span class="highwire-citation-author" data-delta="4">Marisol Esqueda</span>, <span class="highwire-citation-author" data-delta="5">Gina A. Wall</span>, <span class="highwire-citation-author" data-delta="6">Melanie Wellington</span>, <span class="highwire-citation-author" data-delta="7">Chiung-Yu Hung</span>, <span class="highwire-citation-author" data-delta="8">Anuj Kumar</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-6330-3365" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Damian J. Krysan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1299-1314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300515 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489123--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Virginia E. Glazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BE.%2BGlazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazier%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BE.%2BGlazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Murante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BMurante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murante%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BMurante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristy Koselny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristy%2BKoselny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koselny%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristy%2BKoselny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Murante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BMurante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murante%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BMurante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marisol Esqueda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marisol%2BEsqueda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esqueda%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarisol%2BEsqueda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gina A. Wall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gina%2BA.%2BWall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wall%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGina%2BA.%2BWall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Melanie Wellington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Melanie%2BWellington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellington%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMelanie%2BWellington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Chiung-Yu Hung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chiung-Yu%2BHung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hung%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChiung-Yu%2BHung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Anuj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular, Cellular and Developmental Biology, University of Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnuj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Damian J. Krysan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damian%2BJ.%2BKrysan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krysan%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADamian%2BJ.%2BKrysan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6330-3365" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Damian J. Krysan</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/4/1299" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/4/1299&amp;title=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/4/1299&amp;title=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/4/1299&amp;title=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/4/1299&amp;text=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/4/1299&amp;title=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/4/1299&amp;t=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/4/1299&amp;title=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/4/1299&amp;title=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489123/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489123" id="node489123--3" data-pisa="ggg;8/4/1299" data-pisa-master="ggg;g3.117.300515" data-apath="/ggg/8/4/1299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systematic Complex Haploinsufficiency-Based Genetic Analysis of <em>Candida albicans</em> Transcription Factors: Tools and Applications to Virulence-Associated Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Virginia E. Glazier</span>, <span class="highwire-citation-author" data-delta="1">Thomas Murante</span>, <span class="highwire-citation-author" data-delta="2">Kristy Koselny</span>, <span class="highwire-citation-author" data-delta="3">Daniel Murante</span>, <span class="highwire-citation-author" data-delta="4">Marisol Esqueda</span>, <span class="highwire-citation-author" data-delta="5">Gina A. Wall</span>, <span class="highwire-citation-author" data-delta="6">Melanie Wellington</span>, <span class="highwire-citation-author" data-delta="7">Chiung-Yu Hung</span>, <span class="highwire-citation-author" data-delta="8">Anuj Kumar</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0001-6330-3365" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Damian J. Krysan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1299-1314;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300515 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489123--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Virginia E. Glazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BE.%2BGlazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazier%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BE.%2BGlazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Murante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BMurante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murante%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BMurante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristy Koselny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristy%2BKoselny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koselny%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristy%2BKoselny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Murante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BMurante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murante%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BMurante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marisol Esqueda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marisol%2BEsqueda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esqueda%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarisol%2BEsqueda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gina A. Wall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gina%2BA.%2BWall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wall%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGina%2BA.%2BWall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Melanie Wellington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Melanie%2BWellington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellington%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMelanie%2BWellington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Chiung-Yu Hung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology and South Texas Center for Emerging Infectious Diseases, University of Texas, San Antonio, TX 78249, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chiung-Yu%2BHung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hung%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChiung-Yu%2BHung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Anuj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular, Cellular and Developmental Biology, University of Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnuj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Damian J. Krysan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pediatrics, University of Rochester Medical Center, Rochester, New York, 14642</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damian%2BJ.%2BKrysan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krysan%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADamian%2BJ.%2BKrysan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6330-3365" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Damian J. Krysan</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1299&amp;rft.epage=1314&amp;rft.atitle=Systematic%20Complex%20Haploinsufficiency-Based%20Genetic%20Analysis%20of%20Candida%20albicans%20Transcription%20Factors%3A%20Tools%20and%20Applications%20to%20Virulence-Associated%20Phenotypes&amp;rft.volume=8&amp;rft.issue=4&amp;rft.date=2018-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Glazier%2C+Virginia+E.&amp;rft.au=Murante%2C+Thomas&amp;rft.au=Koselny%2C+Kristy&amp;rft.au=Murante%2C+Daniel&amp;rft.au=Esqueda%2C+Marisol&amp;rft.au=Wall%2C+Gina+A.&amp;rft.au=Wellington%2C+Melanie&amp;rft.au=Hung%2C+Chiung-Yu&amp;rft.au=Kumar%2C+Anuj&amp;rft.au=Krysan%2C+Damian+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489123/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489123/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489123/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489123/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489123/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489123/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489123/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489123/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489123/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489123/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489123/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489123/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/4/1299" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/4/1299.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/4/1299.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/4/1299.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/4/1299.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
