<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/4/1129" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/4/1129.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/4/1129.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/4/1129.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/4/1129.atom" />
<meta name="HW.pisa" content="ggg;8/4/1129" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation" />
<meta name="DC.Identifier" content="10.1534/g3.117.300468" />
<meta name="DC.Date" content="2018-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Peccoud et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Chimeric reads can be generated by in vitro recombination during the preparation of high-throughput sequencing libraries. Our attempt to detect biological recombination between the genomes of dengue virus (DENV; +ssRNA genome) and its mosquito host using the Illumina Nextera sequencing library preparation kit revealed that most, if not all, detected host–virus chimeras were artificial. Indeed, these chimeras were not more frequent than with control RNA from another species (a pillbug), which was never in contact with DENV RNA prior to the library preparation. The proportion of chimera types merely reflected those of the three species among sequencing reads. Chimeras were frequently characterized by the presence of 1-20 bp microhomology between recombining fragments. Within-species chimeras mostly involved fragments in opposite orientations and located less than 100 bp from each other in the parental genome. We found similar features in published datasets using two other viruses: Ebola virus (EBOV; -ssRNA genome) and a herpesvirus (dsDNA genome), both produced with the Illumina Nextera protocol. These canonical features suggest that artificial chimeras are generated by intra-molecular template switching of the DNA polymerase during the PCR step of the Nextera protocol. Finally, a published Illumina dataset using the Flock House virus (FHV; +ssRNA genome) generated with a protocol preventing artificial recombination revealed the presence of 1-10 bp microhomology motifs in FHV–FHV chimeras, but very few recombining fragments were in opposite orientations. Our analysis uncovered sequence features characterizing recombination breakpoints in short-read sequencing datasets, which can be helpful to evaluate the presence and extent of artificial recombination." />
<meta name="DC.Contributor" content="Jean Peccoud" />
<meta name="DC.Contributor" content="Sébastian Lequime" />
<meta name="DC.Contributor" content="Isabelle Moltini-Conclois" />
<meta name="DC.Contributor" content="Isabelle Giraud" />
<meta name="DC.Contributor" content="Louis Lambrechts" />
<meta name="DC.Contributor" content="Clément Gilbert" />
<meta name="article:published_time" content="2018-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Chimeric reads can be generated by &lt;i&gt;in vitro&lt;/i&gt; recombination during the preparation of high-throughput sequencing libraries. Our attempt to detect biological recombination between the genomes of dengue virus (DENV; +ssRNA genome) and its mosquito host using the Illumina Nextera sequencing library preparation kit revealed that most, if not all, detected host–virus chimeras were artificial. Indeed, these chimeras were not more frequent than with control RNA from another species (a pillbug), which was never in contact with DENV RNA prior to the library preparation. The proportion of chimera types merely reflected those of the three species among sequencing reads. Chimeras were frequently characterized by the presence of 1-20 bp microhomology between recombining fragments. Within-species chimeras mostly involved fragments in opposite orientations and located less than 100 bp from each other in the parental genome. We found similar features in published datasets using two other viruses: Ebola virus (EBOV; -ssRNA genome) and a herpesvirus (dsDNA genome), both produced with the Illumina Nextera protocol. These canonical features suggest that artificial chimeras are generated by intra-molecular template switching of the DNA polymerase during the PCR step of the Nextera protocol. Finally, a published Illumina dataset using the Flock House virus (FHV; +ssRNA genome) generated with a protocol preventing artificial recombination revealed the presence of 1-10 bp microhomology motifs in FHV–FHV chimeras, but very few recombining fragments were in opposite orientations. Our analysis uncovered sequence features characterizing recombination breakpoints in short-read sequencing datasets, which can be helpful to evaluate the presence and extent of artificial recombination.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/04/01" />
<meta name="citation_mjid" content="ggg;8/4/1129" />
<meta name="citation_id" content="8/4/1129" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/4/1129" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/4/1129.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/4/1129.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/4/1129.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300468" />
<meta name="citation_pmid" content="29434031" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1129" />
<meta name="citation_lastpage" content="1138" />
<meta name="citation_author" content="Jean Peccoud" />
<meta name="citation_author_institution" content="Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3356-7869" />
<meta name="citation_author" content="Sébastian Lequime" />
<meta name="citation_author_institution" content="Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France" />
<meta name="citation_author_institution" content="CNRS, UMR 2000, Paris, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3140-0651" />
<meta name="citation_author" content="Isabelle Moltini-Conclois" />
<meta name="citation_author_institution" content="Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France" />
<meta name="citation_author_institution" content="CNRS, UMR 2000, Paris, France" />
<meta name="citation_author" content="Isabelle Giraud" />
<meta name="citation_author_institution" content="Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France" />
<meta name="citation_author" content="Louis Lambrechts" />
<meta name="citation_author_institution" content="Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France" />
<meta name="citation_author_institution" content="CNRS, UMR 2000, Paris, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5958-2138" />
<meta name="citation_author" content="Clément Gilbert" />
<meta name="citation_author_institution" content="Laboratoire Evolution, Génomes, Comportement, Écologie, UMR 9191 CNRS, UMR 247 IRD, Université Paris-Sud, 91198 Gif-sur-Yvette, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2131-7467" />
<meta name="citation_reference" content="citation_journal_abbrev=Future Microbiol;citation_author=R. Bartenschlager;citation_author=S. Miller;citation_title=Molecular aspects of Dengue virus replication.;citation_pages=155-165;citation_volume=3;citation_year=2008;citation_issue=2;citation_pmid=18366336;citation_doi=10.2217/17460913.3.2.155" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=P. J. Cascone;citation_author=C. D. Carpenter;citation_author=X. H. Li;citation_author=A. E. Simon;citation_title=Recombination between satellite RNAs of turnip crinkle virus.;citation_pages=1709-1715;citation_volume=9;citation_year=1990;citation_pmid=1693330" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. M. Clark;citation_title=Novel non-templated nucleotide addition reactions catalyzed by procaryotic and eucaryotic DNA polymerases;citation_pages=9677-9686;citation_volume=16;citation_year=1988;citation_issue=20;citation_pmid=2460825;citation_doi=10.1093/nar/16.20.9677" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=J. Cocquet;citation_author=A. Chong;citation_author=G. Zhang;citation_author=R. A. Veitia;citation_title=Reverse transcriptase template switching and false alternative transcripts.;citation_pages=127-131;citation_volume=88;citation_year=2006;citation_issue=1;citation_pmid=16457984;citation_doi=10.1016/j.ygeno.2005.12.013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=F. Di Giallonardo;citation_author=O. Zagordi;citation_author=Y. Duport;citation_author=C. Leemann;citation_author=B. Joos;citation_title=Next-Generation Sequencing of HIV-1 RNA Genomes: Determination of Error Rates and Minimizing Artificial Recombination.;citation_pages=e74249;citation_volume=8;citation_year=2013;citation_issue=9;citation_pmid=24058534;citation_doi=10.1371/journal.pone.0074249" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=M. J. Eckwahl;citation_author=H. Arnion;citation_author=S. Kharytonchyk;citation_author=T. Zang;citation_author=P. D. Bieniasz;citation_title=Analysis of the human immunodeficiency virus-1 RNA packageome.;citation_volume=22;citation_year=2016;citation_issue=8;citation_pmid=27247436;citation_doi=10.1261/rna.057299.116" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. C. Edgar;citation_author=B. J. Haas;citation_author=J. C. Clemente;citation_author=C. Quince;citation_author=R. Knight;citation_title=UCHIME improves sensitivity and speed of chimera detection.;citation_volume=27;citation_year=2011;citation_issue=16;citation_pmid=21700674;citation_doi=10.1093/bioinformatics/btr381" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. Fansiri;citation_author=A. Fontaine;citation_author=L. Diancourt;citation_author=V. Caro;citation_author=B. Thaisomboonsuk;citation_title=Genetic Mapping of Specific Interactions between Aedes aegypti Mosquitoes and Dengue Viruses.;citation_pages=e1003621;citation_volume=9;citation_year=2013;citation_issue=8;citation_pmid=23935524;citation_doi=10.1371/journal.pgen.1003621" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Feschotte;citation_author=C. Gilbert;citation_title=Endogenous viruses: insights into viral evolution and impact on host biology.;citation_pages=283-296;citation_volume=13;citation_year=2012;citation_issue=4;citation_pmid=22421730;citation_doi=10.1038/nrg3199" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. Fontaine;citation_author=D. Jiolle;citation_author=I. Moltini-Conclois;citation_author=S. Lequime;citation_author=L. Lambrechts;citation_title=Excretion of dengue virus RNA by Aedes aegypti allows non-destructive monitoring of viral dissemination in individual mosquitoes.;citation_pages=24885;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep24885" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=M. J. Fraser;citation_author=G. E. Smith;citation_author=M. D. Summers;citation_title=Acquisition of Host Cell DNA Sequences by Baculoviruses: Relationship Between Host DNA Insertions and FP Mutants of Autographa californica and Galleria mellonella Nuclear Polyhedrosis Viruses;citation_pages=287-300;citation_volume=47;citation_year=1983;citation_issue=2;citation_pmid=16789244" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry (Mosc.);citation_author=P. B. García;citation_author=N. L. Robledo;citation_author=A. L. Islas;citation_title=Analysis of non-template-directed nucleotide addition and template switching by DNA polymerase.;citation_pages=16515-16524;citation_volume=43;citation_year=2004;citation_issue=51;citation_doi=10.1021/bi0491853" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=K. Ghoshal;citation_author=J. Theilmann;citation_author=R. Reade;citation_author=A. Maghodia;citation_author=D. Rochon;citation_title=Encapsidation of Host RNAs by Cucumber Necrosis Virus Coat Protein during both Agroinfiltration and Infection.;citation_volume=89;citation_year=2015;citation_issue=21;citation_pmid=26269190;citation_doi=10.1128/JVI.01466-15" />
<meta name="citation_reference" content="citation_title=Viruses as vectors of horizontal transfer of genetic material in eukaryotes. Curr Opin Virol.;citation_author=C. Gilbert;citation_author=R. Cordaux;citation_pages=16-22;citation_volume=25;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Gilbert;citation_author=J. Peccoud;citation_author=A. Chateigner;citation_author=B. Moumen;citation_author=R. Cordaux;citation_title=Continuous Influx of Genetic Material from Host to Virus Populations.;citation_pages=e1005838;citation_volume=12;citation_year=2016;citation_issue=2;citation_doi=10.1371/journal.pgen.1005838" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. K. Gire;citation_author=A. Goba;citation_author=K. G. Andersen;citation_author=R. S. G. Sealfon;citation_author=D. J. Park;citation_title=Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak.;citation_volume=345;citation_year=2014;citation_issue=6202;citation_pmid=25214632;citation_doi=10.1126/science.1259657" />
<meta name="citation_reference" content="citation_journal_title=Nature immunology;citation_journal_abbrev=Nat Immunol;citation_author=B. Goic;citation_author=N. Vodovar;citation_author=J. A. Mondotte;citation_author=C. Monot;citation_author=L. Frangeul;citation_title=RNA-mediated interference and reverse transcription control the persistence of RNA viruses in the insect model Drosophila.;citation_pages=396-403;citation_volume=14;citation_year=2013;citation_issue=4;citation_pmid=23435119;citation_doi=10.1038/ni.2542" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=B. Goic;citation_author=K. A. Stapleford;citation_author=L. Frangeul;citation_author=A. J. Doucet;citation_author=V. Gausson;citation_title=Virus-derived DNA drives mosquito vector tolerance to arboviral infection.;citation_pages=12410;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms12410" />
<meta name="citation_reference" content="citation_journal_title=Journal of virological methods;citation_journal_abbrev=J Virol Methods;citation_author=I. Görzer;citation_author=C. Guelly;citation_author=S. Trajanoski;citation_author=E. Puchhammer-Stöckl;citation_title=The impact of PCR-generated recombination on diversity estimation of mixed viral populations by deep sequencing.;citation_pages=248-252;citation_volume=169;citation_year=2010;citation_issue=1;citation_pmid=20691210;citation_doi=10.1016/j.jviromet.2010.07.040" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry (Mosc.);citation_author=A. L. Guieysse;citation_author=D. Praseuth;citation_author=M. Grigoriev;citation_author=A. Harel-Bellan;citation_author=C. Hélène;citation_title=Oligonucleotide-directed switching of DNA polymerases to a dead-end track.;citation_pages=9193-9199;citation_volume=34;citation_year=1995;citation_issue=28;citation_pmid=7619820;citation_doi=10.1021/bi00028a032" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_author=M. G. Guzman;citation_author=E. Harris;citation_title=Dengue.;citation_pages=453-465;citation_volume=385;citation_year=2015;citation_issue=9966;citation_pmid=25230594;citation_doi=10.1016/S0140-6736(14)60572-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Haas;citation_author=D. Gevers;citation_author=A. M. Earl;citation_author=M. Feldgarden;citation_author=D. V. Ward;citation_title=Chimeric 16S rRNA sequence formation and detection in Sanger and 454-pyrosequenced PCR amplicons.;citation_volume=21;citation_year=2011;citation_issue=3;citation_pmid=21212162;citation_doi=10.1101/gr.112730.110" />
<meta name="citation_reference" content="citation_title=ClickSeq: replacing fragmentation and enzymatic ligation with click-chemistry to prevent sequence chimeras. Methods Mol Biol.;citation_author=E. Jaworski;citation_author=A. Routh;citation_pages=71-85;citation_volume=1712;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biosci Bioeng;citation_author=T. Kanagawa;citation_title=Bias and artifacts in multitemplate polymerase chain reactions (PCR).;citation_pages=317-323;citation_volume=96;citation_year=2003;citation_issue=4;citation_pmid=16233530;citation_doi=10.1016/S1389-1723(03)90130-7" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Katzourakis;citation_author=R. J. Gifford;citation_title=Endogenous viral elements in animal genomes.;citation_pages=e1001191-e1001191;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21124940;citation_doi=10.1371/journal.pgen.1001191" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Krupovic;citation_author=N. Zhi;citation_author=J. Li;citation_author=G. Hu;citation_author=E. V. Koonin;citation_title=Multiple layers of chimerism in a single-stranded DNA virus discovered by deep sequencing.;citation_volume=7;citation_year=2015;citation_issue=4;citation_pmid=25840414;citation_doi=10.1093/gbe/evv034" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=BMC biotechnology [electronic resource];citation_journal_abbrev=BMC Biotechnol;citation_author=R. S. Lasken;citation_author=T. B. Stockwell;citation_title=Mechanism of chimera formation during the Multiple Displacement Amplification reaction.;citation_pages=19-19;citation_volume=7;citation_year=2007;citation_issue=1;citation_pmid=17430586;citation_doi=10.1186/1472-6750-7-19" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=Y. Li;citation_author=L. A. Ball;citation_title=Nonhomologous RNA recombination during negative-strand synthesis of flock house virus RNA.;citation_pages=3854-3860;citation_volume=67;citation_year=1993;citation_issue=7;citation_pmid=7685402" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Li;citation_author=W. B. Lott;citation_author=K. Lowry;citation_author=A. Jones;citation_author=H. M. Thu;citation_title=Defective interfering viral particles in acute dengue infections.;citation_pages=e19447-e19447;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21559384;citation_doi=10.1371/journal.pone.0019447" />
<meta name="citation_reference" content="citation_journal_abbrev=Viruses;citation_author=D. P. Martin;citation_author=P. Biagini;citation_author=P. Lefeuvre;citation_author=M. Golden;citation_author=P. Roumagnac;citation_title=Recombination in eukaryotic single stranded DNA viruses.;citation_pages=1699-1738;citation_volume=3;citation_year=2011;citation_issue=12;citation_pmid=21994803;citation_doi=10.3390/v3091699" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Meyerhans;citation_author=J. P. Vartanian;citation_author=S. Wain-Hobson;citation_title=DNA recombination during PCR;citation_pages=1687-1691;citation_volume=18;citation_year=1990;citation_issue=7;citation_pmid=2186361;citation_doi=10.1093/nar/18.7.1687" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. W. Miller;citation_author=L. K. Miller;citation_title=A virus mutant with an insertion of a copia-like transposable element.;citation_pages=562-564;citation_volume=299;citation_year=1982;citation_issue=5883;citation_pmid=6289125;citation_doi=10.1038/299562a0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=P. D. Nagy;citation_author=J. J. Bujarski;citation_title=Homologous RNA recombination in brome mosaic virus: AU-rich sequences decrease the accuracy of crossovers;citation_pages=415-426;citation_volume=70;citation_year=1996;citation_issue=1;citation_pmid=8523555" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=P. D. Nagy;citation_author=A. E. Simon;citation_title=New insights into the mechanisms of RNA recombination.;citation_pages=1-9;citation_volume=235;citation_year=1997;citation_issue=1;citation_pmid=9300032;citation_doi=10.1006/viro.1997.8681" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. J. Odelberg;citation_author=R. B. Weiss;citation_author=A. Hata;citation_author=R. White;citation_title=Template-switching during DNA synthesis by Thermus aquaticus DNA polymerase I.;citation_volume=23;citation_year=1995;citation_issue=11;citation_pmid=7596836;citation_doi=10.1093/nar/23.11.2049" />
<meta name="citation_reference" content="citation_title=Current Protocols in Molecular Biology;citation_title=Ribosomal RNA depletion for efficient use of RNA-seq capacity;citation_author=D. O’Neil;citation_author=H. Glowatz;citation_author=M. Schlumpberger;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=S. Pääbo;citation_author=D. M. Irwin;citation_author=A. C. Wilson;citation_title=DNA damage promotes jumping between templates during enzymatic amplification;citation_pages=4718-4721;citation_volume=265;citation_year=1990;citation_issue=8;citation_pmid=2307682" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Patel;citation_author=M. Lin;citation_author=M. Laney;citation_author=N. Kurn;citation_author=S. Rose;citation_title=Formation of chimeric DNA primer extension products by template switching onto an annealed downstream oligonucleotide.;citation_pages=2969-2974;citation_volume=93;citation_year=1996;citation_issue=7;citation_pmid=8610152;citation_doi=10.1073/pnas.93.7.2969" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Peccoud;citation_author=V. Loiseau;citation_author=R. Cordaux;citation_author=C. Gilbert;citation_title=Massive horizontal transfer of transposable elements in insects.;citation_volume=114;citation_year=2017;citation_issue=18;citation_pmid=28416702;citation_doi=10.1073/pnas.1621178114" />
<meta name="citation_reference" content="citation_journal_title=Infect. Genet. Evol.;citation_author=M. Pérez-Losada;citation_author=M. Arenas;citation_author=J. C. Galán;citation_author=F. Palero;citation_author=F. González-Candelas;citation_title=Recombination in viruses: Mechanisms, methods of study, and evolutionary consequences.;citation_pages=296-307;citation_volume=30;citation_year=2015;citation_doi=10.1016/j.meegid.2014.12.022" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=J. Pogany;citation_author=J. Romero;citation_author=J. J. Bujarski;citation_title=Effect of 5' and 3' terminal sequences, overall length, and coding capacity on the accumulation of defective RNAs associated with broad bean mottle bromovirus in planta.;citation_pages=236-243;citation_volume=228;citation_year=1997;citation_issue=2;citation_pmid=9123830;citation_doi=10.1006/viro.1996.8377" />
<meta name="citation_reference" content="R Core Team, (2017). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL https://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=A. J. Raffo;citation_author=W. O. Dawson;citation_title=Construction of tobacco mosaic virus subgenomic replicons that are replicated and spread systemically in tobacco plants.;citation_pages=277-289;citation_volume=184;citation_year=1991;citation_issue=1;citation_pmid=1871972;citation_doi=10.1016/0042-6822(91)90844-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Routh;citation_author=J. E. Johnson;citation_title=Discovery of functional genomic motifs in viruses with ViReMa-a Virus Recombination Mapper-for analysis of next-generation sequencing data.;citation_volume=42;citation_year=2014;citation_issue=2;citation_pmid=24137010;citation_doi=10.1093/nar/gkt916" />
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U A;citation_author=A. Routh;citation_author=T. Domitrovic;citation_author=J. E. Johnson;citation_title=Host RNAs, including transposons, are encapsidated by a eukaryotic single-stranded RNA virus.;citation_volume=109;citation_year=2012;citation_issue=6;citation_pmid=22308402;citation_doi=10.1073/pnas.1116168109" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=A. Routh;citation_author=S. R. Head;citation_author=P. Ordoukhanian;citation_author=J. E. Johnson;citation_title=ClickSeq: Fragmentation-Free Next-Generation Sequencing via Click Ligation of Adaptors to Stochastically Terminated 3′-Azido cDNAs.;citation_pages=2610-2616;citation_volume=427;citation_year=2015;citation_issue=16;citation_doi=10.1016/j.jmb.2015.06.011" />
<meta name="citation_reference" content="citation_journal_title=Clinica Chimica Acta;citation_journal_abbrev=Clinica Chimica Acta;citation_author=F. V. Shammas;citation_author=R. Heikkilä;citation_author=A. Osland;citation_title=Fluorescence-based method for measuring and determining the mechanisms of recombination in quantitative PCR.;citation_pages=19-28;citation_volume=304;citation_year=2001;citation_issue=1-2;citation_pmid=11165195;citation_doi=10.1016/S0009-8981(00)00374-0" />
<meta name="citation_reference" content="citation_journal_title=Retrovirology;citation_author=W. Shao;citation_author=V. F. Boltz;citation_author=J. E. Spindler;citation_author=M. F. Kearney;citation_author=F. Maldarelli;citation_title=Analysis of 454 sequencing error rate, error sources, and artifact recombination for detection of Low-frequency drug resistance mutations in HIV-1 DNA.;citation_pages=18;citation_volume=10;citation_year=2013;citation_issue=1;citation_pmid=23402264;citation_doi=10.1186/1742-4690-10-18" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=E. Simon-Loriere;citation_author=E. C. Holmes;citation_title=Why do RNA viruses recombine?;citation_pages=617-626;citation_volume=9;citation_year=2011;citation_issue=8;citation_pmid=21725337;citation_doi=10.1038/nrmicro2614" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. K. Vaz;citation_author=T. J. Mahony;citation_author=C. A. Hartley;citation_author=E. V. Fowler;citation_author=N. Ficorilli;citation_title=The first genome sequence of a metatherian herpesvirus: Macropodid herpesvirus 1.;citation_pages=70;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Viruses, Genes, and Cancer;citation_author=R. A. Weiss;citation_title=Exchange of Genetic Sequences Between Viruses and Hosts;citation_year=2017;citation_doi=10.1007/82_2017_21" />
<meta name="citation_reference" content="citation_journal_title=RNA N. Y. N;citation_author=K. A. White;citation_author=T. J. Morris;citation_title=RNA determinants of junction site selection in RNA virus recombinants and defective interfering RNAs.;citation_pages=1029-1040;citation_volume=1;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Virus Res.;citation_author=F. Zanini;citation_author=J. Brodin;citation_author=J. Albert;citation_author=R. A. Neher;citation_title=Error rates, PCR recombination, and sampling depth in HIV-1 whole genome deep sequencing.;citation_pages=106-114;citation_volume=239;citation_year=2017;citation_doi=10.1016/j.virusres.2016.12.009" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=X.-C. Zeng;citation_author=S.-X. Wang;citation_title=Evidence that BmTXK beta-BmKCT cDNA from Chinese scorpion Buthus martensii Karsch is an artifact generated in the reverse transcription process.;citation_pages=183-184;citation_volume=520;citation_year=2002;citation_issue=1-3;citation_pmid=12044895;citation_doi=10.1016/S0014-5793(02)02812-0" />
<meta name="twitter:title" content="A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="twitter:description" content="Chimeric reads can be generated by in vitro recombination during the preparation of high-throughput sequencing libraries. Our attempt to detect biological recombination between the genomes of dengue virus (DENV; +ssRNA genome) and its mosquito host using the Illumina Nextera sequencing library preparation kit revealed that most, if not all, detected host–virus chimeras were artificial. Indeed, these chimeras were not more frequent than with control RNA from another species (a pillbug), which was never in contact with DENV RNA prior to the library preparation. The proportion of chimera types merely reflected those of the three species among sequencing reads. Chimeras were frequently characterized by the presence of 1-20 bp microhomology between recombining fragments. Within-species chimeras mostly involved fragments in opposite orientations and located less than 100 bp from each other in the parental genome. We found similar features in published datasets using two other viruses: Ebola virus (EBOV; -ssRNA genome) and a herpesvirus (dsDNA genome), both produced with the Illumina Nextera protocol. These canonical features suggest that artificial chimeras are generated by intra-molecular template switching of the DNA polymerase during the PCR step of the Nextera protocol. Finally, a published Illumina dataset using the Flock House virus (FHV; +ssRNA genome) generated with a protocol preventing artificial recombination revealed the presence of 1-10 bp microhomology motifs in FHV–FHV chimeras, but very few recombining fragments were in opposite orientations. Our analysis uncovered sequence features characterizing recombination breakpoints in short-read sequencing datasets, which can be helpful to evaluate the presence and extent of artificial recombination." />
<meta name="og-title" property="og:title" content="A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/4/1129" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Chimeric reads can be generated by in vitro recombination during the preparation of high-throughput sequencing libraries. Our attempt to detect biological recombination between the genomes of dengue virus (DENV; +ssRNA genome) and its mosquito host using the Illumina Nextera sequencing library preparation kit revealed that most, if not all, detected host–virus chimeras were artificial. Indeed, these chimeras were not more frequent than with control RNA from another species (a pillbug), which was never in contact with DENV RNA prior to the library preparation. The proportion of chimera types merely reflected those of the three species among sequencing reads. Chimeras were frequently characterized by the presence of 1-20 bp microhomology between recombining fragments. Within-species chimeras mostly involved fragments in opposite orientations and located less than 100 bp from each other in the parental genome. We found similar features in published datasets using two other viruses: Ebola virus (EBOV; -ssRNA genome) and a herpesvirus (dsDNA genome), both produced with the Illumina Nextera protocol. These canonical features suggest that artificial chimeras are generated by intra-molecular template switching of the DNA polymerase during the PCR step of the Nextera protocol. Finally, a published Illumina dataset using the Flock House virus (FHV; +ssRNA genome) generated with a protocol preventing artificial recombination revealed the presence of 1-10 bp microhomology motifs in FHV–FHV chimeras, but very few recombining fragments were in opposite orientations. Our analysis uncovered sequence features characterizing recombination breakpoints in short-read sequencing datasets, which can be helpful to evaluate the presence and extent of artificial recombination." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
    <title>A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qI_DjTbdTv08ajdUFjbYfBggibxpCcXAArhkt3UWYy8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489121","apath":"\/ggg\/8\/4\/1129.atom","pisa":"ggg;8\/4\/1129","ac":{"\/ggg\/8\/4\/1129.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/4\/1129.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1129"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1129"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/4\/1129":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IJpbxmMlXx+hUJMN5BHATqJ3F33SDHqtP6xcZUZwb8Jp5jWnKo5+b2I0IfZ\\\\\\\/GKN1D\\\\\\\/AkIBurgVLYYT2Jc5OagdW3TX+YoXtHV7tKa\\\\\\\/Nd8v5XToIU4rcWOLgzHvRsyBu90g\\\\\\\/Imt7yaEl1J1TBaR+M88eagnrqEroeGq2E6eYEJTdegKBGM2cuCJAVk1WA38rzmW+G0NC8mbx3L8QZVNpRT5WpaifcV3FFIjVBCxmA82k1UCMXDFsxo6WkT705K3MhIigyQM7HcxkWUtMnHc80l0r7Icp8ynvPWePro7lpjSM4BogYNnGugGkgE9eUSGgmLQ0fw43B3l2Qg5aaN9HqJZOfcI3YLEu97VXzEQogUE6jhfw4gYzHdTYfVrVjYFpkKO\\\\\\\/UDp8iumSIX8ZFlgbWdgPBFhQqu70vIFRo9Rkxv0sWSry513jN0f6CKnxi4eP4+mWSTBiDYdaP0AAod+JU+bEDkeq4ciCK0DEJd2EA0i6e9oclRHaZvU92vD4IMx5ie32X+s7HdJcFiCBJscyaZXb1ph1JmHaZj4GfJA6eD8xK909gDd8pdZsXYQmpdCgT4rJ5qpq54xk5FmcNHN1On3CKQa0SCddJylUFGs1AInaO+DCobExBG9UeBXZxfc40UtinCwLTtIIrKtBtyveecWq4WzdUp166Zgfiou8XkDdmlJCL4yO6ULcupBu6PQYerEy5o3hh8Rg1gB0Nq3Qfmq9TDPv7sC0YM+u\\\\\\\/lukwrqYcKF\\\\\\\/g0IHeE2Zcm\\\\\\\/Kch2fnRnGdRE\\\\\\\/5AuIWmL7pHhbB3eJ58fl495VeEJL1ZoNFmHZX7hsqd2zQ4gVwCDNq6zoMHts8bV\\\\\\\/ugWghtid7tZRMVcmqcNFrF3PBiBKZgzjEYPHS31EGYgXWZSPv1gg\\\\\\\/gxFbEPehVmSUi5XFWfrMYLsfHnxNfH2\\\\\\\/QJYZZ0y4Dj4ht1RjkrwJLzh7rWnnvWpi6IRcPh6fm2EBraXrehjaPGEM9SmxGmKBWR1JkpFACfp9SMu1ZeCsRgJu+3RXuCgdZu4phGw34iWB0wGnP8aXtt8ccUZDJWn449mgvlgoKeCHc9RWwgkWtsxDsNLlu6HhT0YhaoqtRJ3OibmGuQ43utmq5WAqsuHv6t3rxfvQpfxleA0nKdxRj4cCdB+\\\\\\\/pTBKlfOwZfOpE5KHwe+B48wY8Wk3XRDuWubwd4Vn6TVMGfwv9oV0RSk3j\\\\\\\/8enDVPR+qJ8Y\\\\\\\/gjCqqxnai34FaKn4imCF\\\\\\\/w2hIQZkKKe5cHr\\\\\\\/UlEO5Mu0g2OxRT\\\\\\\/OZEy4YkI+MSvRU\\\u0022,\\\u0022iv\\\u0022:\\\u0022xnfLiLebCh9hPFif3hVr0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221c1beb02d902172834fde1104cd4639d285e7c4d8d7ac3b7469cf346ef83acc7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x71kcLlX1NWGFC6RS60JeUUuCd5o2iw7xDQOZ2xlmAr68PgPir\\\\\\\/Vc\\\\\\\/FDDLkWsjPNP1IyRImIHLiKORS7U\\\\\\\/PEPdbPZlLtkAEuPH7aUsHZ1Vzx6MYZFF7fHEP0oEf1IL\\\\\\\/PXIsNNICEfTe8lJQqCCLjFMwziUFuYl6XGOj9jXfXmMRJoRvZcKQfmDdlRJqvQ06f2BrOUiZkFN2n3mOHiLWOLGnHl9eX0ViHJvYk8luMj7cfuaPSV\\\\\\\/9gcudU68DtT5xbTXL\\\\\\\/ggLBMXTzsjh8sH7hXvjctsAf\\\\\\\/KQXiNyyXvDoNcn7X\\\\\\\/I+ljGd+zVFv7vctPaC2tFq\\\\\\\/XB7fx8PwiPWmPU9fVTf93Ilg1x8jywMYGFdZGzqTmaDBfff7siMPg5GhfSAk3xmsgMPNaBPOIucIcrJXeeATtpffwnj2L5ysBGfUcWFiBHw7J6OeccqcymgYeM2nW2CXM8vHjTh0lruTY7aKUVvvNGcvv9dm3vIohe741BHGMQwF3HjMiUiFR7opo1zwLh8HGXTaZT9cX\\\\\\\/LAZSfGdMc++QFOy6l6NwOzc6CXO2NXhKSkGZ7MOr1pUFztQSqZMUL+SbZlFM7wSqZ1kXobR3qM3ZVwROUqJwzCpaY0mlg2HGWrQB8xShDttsFzt17pGN\\\\\\\/I7+UvjjNPkx6mP+YduBCdn2h7fngwp9mbCxh6uorWp4UEun6G9lFPTEsQwnDmXiW4u7WpjmGx\\\\\\\/WmzVbEhQqdJco7rLNYjpVlbYS+x78Dw171od0sQTT75j9uAVk3yXivURLOfE+Hq2J1hlUWzAeo0XAW53UrWbu8TkU\\\\\\\/6yS0Z0dsDt8ajwf26myCmIUvt3zmab7rmxtuRWHPwLJW0F6si8vl4u6OH0rf79BsQpUB33dtiFQecsFq2C2TRfziz1tiNWaDQmYiW7OwAPLeIErB16RyAb7laW2vYsHHJqK0ydMDDuzLU4lPH9JoZ\\\\\\\/46SZCeCRynE9tp228Rs6D8sjEp8gmJR1YF80QsPRfMU2lSp9WgwjHsuK9UU4Sewyb\\\\\\\/cXOKNi4Lw1QvzbyxRtyuy\\\\\\\/GutXbt8g6fifqYWB8hRHfgHcDygWFGkEaC6BXTLiAn0AHGejfyOdXedgImkyGuSgUuJZWfmrAvpFvBdkQGyXZIombdoCc0tovrWuO21yP7ekEWN2DA7kmGOX\\\\\\\/Rzlixz1CPTDMFsm9pHNH3I08bWfvzxA+M+H60J31D5y43MLUhqKhqZL44SQS1FjfxOX5Sd9wJLsoDFq\\\\\\\/wvrQQSh2mFZs3WpguvVpiX9Cx5zFVL8ph9f62fnO0vuLdkqEPiLoDXXJaWYEvXOuMNnTc\\\\\\\/CxkxTc3ca1lJV7kq8FLIuBs0Fv+D+vfhcaef7ynn0elI4SefBoSAo8ArV+qvr0SGtd\\\\\\\/HR7yj5zrjA+7AlIJu9kGlOWcZssuDxSxL0x4aeZ6rMeFiz2w2JLoh55zW\\\\\\\/3fC3jZVQFEyoD6XC7Xe6SbZJaxa0kaGcJYVxCGY6S1\\\\\\\/Vg8a4v1Pf1xJmvC8T6oKAmz\\\\\\\/EWa1VTJflB++a3\\\\\\\/P\\\\\\\/TYkyX2m0VmMfArIVYg5nuntj2VYM7sZIuTgmmiMUFyi7+VPHWd83RqrrJzvKKR1LlT0oYLlV\\\\\\\/Kxv640x42EwWKJsaz5Bp323bmLwVA6bt817WqWWVklaBkm615L+LjMHq3qswkRIIaa9vEvcLJC7p\\\\\\\/2GUxHzgLQH6R7\\\\\\\/fbeMjGeCN7Jl5Cw1g0kTsllv3xPcBjdJQ9KBcZmOiPo4CdS72WVRtwDb+vaNXY65Du0zzAIAVgmaaec4ohQ5m8EsZys1gL+lTkEHUO\\\\\\\/YNEzSDHhkCStGdXi1Awpc9tywTyIrdXl4Ze+bCMPLrxwhp9IrUDRpIj7i9CNSqxTE3NDdvNJz5WPgWU1Uwa5dkm1\\\\\\\/p\\\\\\\/jXwrPTkqReMqtUuAQkv4c6vuLPCQ6rU9krq0wjtlo853ArRw7hGh99MSeiHUBoY\\\\\\\/nz55QIXNvzY7JF1MJhn40wI+nmoSaDvxWDelRMYHp8Ooe6pqV4REyhGwqJ+vZDEUDREQh+sjjJ9t+ghK2mE4g3VvZIgSzVnq4cRrVb5tUzjRA3zhJPHaHV19BSNyKUmLt4JoqQy8b0+lUfryX3Hc5JWGxYQbujSTJGHPf3EYX8PCsZFSDPe4a3gpOAHuFINpcpVw0du0FYOqIXVgDc+0kWT6r1uqMrvQZh\\\\\\\/Jrv8X0kFNjHcDNQp7Ggrorz6U0lUIy5In114iEclf6CTN1hAVsfGcXETV\\\\\\\/8IEqS2Fnr7NMYdNI6lKFPSwdkZgnk1cJdw3CzaW33jFdOjb4yW\\\\\\\/Q8ya78JvNBjPveoBfjuDX3CBJd+gLg7JrKBZFDeZzvRa+MdikNHMehL7nFNjOacITkgdXDr\\\\\\\/pEq8rxlOBas+9lM3qsMzGMRXbeR5ZM\\\\\\\/eqCzd\\\\\\\/OYNOzq80FrmQRIu0mq\\\\\\\/ypOtOFS0tjRHVB8BwDT0rOdjE28gHxSWNdqtVbp78ZOPWKYVxp+1ABV74YyNczxljh+WNRCM9LI3Vf5s5jj0uelAURuG6cZPFWcGEYMRj9SudFuADaavdGGJQDA2shCcGBi8TARG1H2areUw1i\\\\\\\/MCXz3JH6enXN92j6kYXR50g9uRVVgqdZFA3XfXazQCaITay7gJSkeA7wHFt22AK9DPehXv8IH0Pg+H17GRW29KY9UPgPY7A03jhOyDEMDwkjvEhyElsHtKtV8agHZ8LtUF8iQBEWU+uTLa71UHyIDyTdzHXFxJlzzpmmemot1YXWDbDIqhUI\\\\\\\/dh45bj6iq+vf7DUUh3O\\\\\\\/A9m1+7CPWOefVygMkJ10Tt4to7NsIKFfc3jwOoG0SsfLSENDQ11SUBhA5DN5iO2GBpWRy9loInWZgIJvsbBj66NodIbT3Il86s7aaPfs\\\\\\\/A9z1AnIgsbjVeHvsfMzGVvfJxWtRy8awdV8pdnOteZiEDkNn5s3SCUDYw5mQ1\\\\\\\/cwZAV8+SPuMLKoSX9qVyPiMBQGFVNrWstHKPc7CeNFt3QLbM8ZC4fuYKH59krBydk2UbWqJdAEbPPivY1RG3K0bOLPSsDS16dtzi2ANeOnpLwxpw3LvYQB\\\\\\\/V4y6qGFZ4D6pO+oOal+p6ZQRvMIteFOL657jhjKrmpN5nuJlVicnHbQqnZXo3XBawlEsmUbs8cd8eZL1jPqzUyRFQO4H58SWLcVp1G2t157Hcygr5ojbclTx9lIxRJUeCZAgj1nFJe5XnaqQwb9H46t5KNy5wrCrYJXew8dPXDqNRNjr5JmZds2EAqXZXahuXMgIu0E3Yl+LJpboZgTD8xMQ9XIZPJrox\\\\\\\/bEDvefNGPWidOYEwsVsLl49sB2Q1m766Y2eUQ80qLTZ7gOEs3Jh3nHsKjZfaTfLaYZImCuqp0yPrk+klZdk94+FKD0mo4uTgQ+7uS0H7Qq4f8qmB\\\\\\\/icLShz1On\\\\\\\/4Cn\\\\\\\/ph8jdGnlC4eFMQCb1feMubdNDfs6Abpn8Hd7ln2iKVHyjuyiRouKGThIcuwomACjwhITvq05M9kjY2Boj8cgEfHmIJWUXBFZeUtdZ2X3CNTXrZLdQRgwXT5au7wTDQn5h7pYqUj7B2T7VikLsY6wJpxtmfi7RoJfVP\\\\\\\/Cu\\\\\\\/06bmRmzaPzT0HcZuUg7xF+cs5DlkLbOEXgzL0C3PVz2rO3yvbiFhvL1VKV696N67NSjs1OCjbM1DIGUHqVJRvAS2qI0cTcRF9NZB0bJDHHPKmCmRRn0k2rDdrE0do3K4Nh12LCztaJrVp6p8fsxS09tXEOW++lzi+ROlztcXgTgCZseU+U0juSMYngJ3ElFQaHXHS\\\\\\\/6FJkLURPUObm6uEE8pwMq82Xxi3KTzrcWZLDjQ7MEFQLtDsh8KG4Zgy2L6N2fwzULRj1jXrO\\\\\\\/XXvIIEZNQTpwVwIBt\\\\\\\/+PXUZwcICFE04jheqaT9CjIRskOaN5f3eJX2m\\\\\\\/UlQrAbe6M5mw\\\\\\\/ziJ5tSUri54iDT4gbit9vxo5DfwI3Szs\\\\\\\/+8PcgcSvkEv87\\\\\\\/PtYrLFVwrOFcl864mVd+nK1T8xodOLEXgDatbMMX50ZpBM8U\\\\\\\/Wp6pFX8R55WzbMojJG1DnCli\\\\\\\/3N2ONbNBpnhu8Ur6lP9YObfdb7\\\\\\\/QNqzDL7kTYad3+L581bWTUbD27sT3FYnAP6Ij\\\\\\\/r8WSb6qeTNkMuAifWHiDZYSyTGxIZeZjWkmeOFl2QGEPIq440uD2I91fEcn70cTPis6aDFxI1us4BvFlsxsAkyTc4tc6JgV4u69fiS2DQjZZzBWm2mu1IY9AUHH+Z\\\\\\\/auuoRqnWa9FYzgNuUCyxpTI\\\\\\\/9y5ZkAEu0Ilo\\\\\\\/c6D2VGof2aAFbx80AtMOI3wBq4McPW3NRT33nFQYHXuhucc5+tzX7j+UfJEhoOkr7xWrSrGft8KEzX\\\\\\\/mSFy7dIxORBrryUR0VlpfkvBtSmLJNvigNA+NaigW+I5ya9WAZOXMMhhUg7jn7o8p7yqVAgrDRy1ZTTw\\\\\\\/H\\\\\\\/2\\\\\\\/8s8ZSAfXC6Lwn9l\\\\\\\/p\\\\\\\/yrdJC4Fg5WMJgSvv2XaZ74DMJ0+ScTu0D2W1kdGimBIyavI3nCQmnFRiKgKsvrc3iXmYPuggtuIBvkJIFP+\\\\\\\/8PZhsyH6aE6oeE7I7CtgpnGa8Luhri4uCL6kCicvONoQMek+Fmb5hm2q+cY\\\\\\\/LEo8yXGBYn\\\\\\\/mz9qbM2hl33tsiHTv5XyojqZeLJw0a6sJ7vk+yzLOBUq7wG6C3fJaMQxnov4V9etnSiUVNSPAcI2Fhtc5sslQoNiNgkGyCSWMLZHhgDan0ZVWRxm7\\\\\\\/bbZymnqbmfRfoiBgLNpfjhWTe1uAk016FX7cQxXLUHDyJFOoMQuqdGrr75jt\\\\\\\/993YoFN3rss7EsG\\\\\\\/2rXwTRnLivLajAhAEjeahF2Nh6mEczbTI6jqYVxoCG1NFMn1tj+UlisszWDt7w+ZAepiMPdVcvuRU4OxwcbZ5xJEPf7vaHlHS9ATPqfCGszFwEWG06gxAGnar84oXHdokdtVPuAWBOEuU6CuyDkjAMJgDp3V6NC3Doj\\\\\\\/t6BLSoua8hAUUiS0qol5pbiQ9beqQ6LUyP0\\\\\\\/e6J3Ay1ZCEiBlcGtyfTO9QqydEt5VnTaz3IdQBIU8GuIzHOTFSIE+v9aajJ\\\\\\\/RgO+IgsW8muyJCV3gq+xt6LgyyGRdXpEa\\\\\\\/FHtzKvXpqlO\\\\\\\/ny3T4Td+Tg2YhHyARgjEzofofShkxIr\\\\\\\/A8jneEAzxAO8gCEuaHiC02AhR9P2IoyZRKSaG64KIyZMYm5oo\\\\\\\/KXy0l8Q0ryqx1EhgQLWdI0mNVEeMMQhR\\\\\\\/Ig4\\\\\\\/viRWhCm3MBaqJRaPdKVRSkTpBJbLa6\\\\\\\/fZY2isdRRCohzb7VI2FOIjnpVi9xaZpKWRsLIfkaan2uCE98vKTGgLpSJQ+kr6YjWSNOL57kMan51\\\\\\\/t3M\\\\\\\/QlW5HscFrqBFoVm28PigvFyu1NHMxFL0CsLuVgOJQA6RHYq4OpOfSOWW5GOpcH\\\\\\\/IXz5K75LgNI08uBm8Dq6+I42kCU8CqanFw4CUlU3z\\\\\\\/FqdBT9+CAQUuu2xM+9n5duV7PcOrv5DdUpk3fSo\\\\\\\/d9sVEn4i+5wZ10tfKnwxlG0tMGt7hUdicYtZW8CSe25+U62L7F2p4h5IkYbCYizxpnBazmVtZ2XaxwJzjCAm5QZhIfgF5MIyUn1ftmNuaEAOImcs\\\\\\\/dyaGQ33BCw7WDc6HSAD8b6FoL7nZxgUauLQ\\\\\\\/f9A0RVGzLuzcOm27SptAybtm9\\\\\\\/Mtc9wdVyB47\\\\\\\/M6a8Qy9r5rwBquS4Od7RYrMlt6klg6\\\\\\\/P\\\\\\\/9zPE6R2U9q\\\\\\\/YPjJb1WMq73gnGxM+uijiK+xlaZv\\\\\\\/TTTPUmPigXaan6+MPBaqKpBeC11W9V1fal5ToQyLlADL81ugB5jeHuHGHV3LLQlibT5ix0E83LMUsfZC7xBZ8JYTtP5Mr9B8Gv3YgucNFH1LFp0XtxRzNj65MEYP0LoQSW8k3EWCg8B04fJAeCpprYei7CE7mDBf1l73NA66TLKWEXu3zLEyC7OtPar5ZBjz8A3EqnBeJFmQnM4q9hfHPCzS4SJrvMJw3TdpIDTj0tQ6F4Wl2m5iTbjROA+NP\\\\\\\/ZVPOhxX4UYPnZkOPqWPtDkNKa9XqmXOBzdKjiw0Dq6dkw9q9AQrMHStghIunCQLEZBIKhG28MwJVHUAkXcMC4puidu\\\\\\\/CzXhvxvNxe1wUk4fz0uyTPr1eAJUTPFTqJzVOOLYuSPLtF2SNv9795AJeX65deWTTioK+0Oq3hod625m01+CophDQSZg8BVp+X97YAKaSuqCf86z3WDKaDpVLiu6uTObEbnSg3DaezVoD8ikM5KKZ+Mn4syROzbekkP5PGwZDMcN3WxB7GP7+o6d6mKkwQwV5QdBKKZf3mtg0GW\\\\\\\/CTCz7QQBVQrXOVIXnTJsv1URQQWNp8mYd99ZHXb4YwMmWoMur32+Y85rD5erwYhK83bAVEPbgQdEk+7VYV5lNr2+RopKRq7Kg0f8ZGVEe11WQ9R0mFAPzfuM0zzNRbfNb2RdSjT\\\\\\\/lzPjn9dY7ABa5Iywn0ibHk53zgwBEfmZuuHXPJ0qKDfR9DQ3Oyv2DjKghJF\\\\\\\/fr1xAX7ncVilWsZiVBqyyLafF9m82CfZzdyj38qnQge3iWYafGVs7NpX6dunKGNmscEZCDhzV20BafrSG6BjrDC3mCXvZRNAuFTkVNhyPkERr\\\\\\\/KMjevwOYlF5nvyjaWF\\\\\\\/Qiq4rJ+l9zyFM5yPEi+AP+L0RXnwo5jBjQ63wxJQ6aH6nZQEkwYkOhIPVw1JgGmAtJMfW5NMh+FpyQygMyizuT6kTvvwT3TFb+gjlK1c7aFFRx4n+PF20jlQB2E5gPpqBOZwh3aj\\\\\\\/F7sSPtzpS37aSTsnEyhQPXiVWPSCKc6vkoG0tDJkdjEG5loZbBsE+\\\\\\\/PUEVjhnn4ReC6t1Ts0FFQs1Pq0qpSZo6ppfvMDTgWnMdx4RqZNILOkhCOvlKfNwhyHB1kRFxjcs3OW0IYkbj2RMJ8o256clgFj8WSh56fRcAqjpmKdSEU9hJxRNm6OYBKFe3U374hRp\\\\\\\/gd5eP2yKAb0XtoX40mX0VCXgiq0KnBcdnEP2VpfyioLtRE69o+nb3SqQyKgoUrO04EGTS+PAbJEtnCsA8e2nfAB6FSPHjFmdmQTf48LXYj9n7MmYnPXM8s8mrYQbzuVwky3B789t7RuH+jtZhc\\\\\\\/1pEtOYpJ\\\\\\\/TdW2UutdDk+nv8zzsmcgPgR9d+KR1a9MRq+40I1hBATsFLmo6HeSwn80OpqX9CKcJ42h5yPO6wh1DFSb\\\\\\\/zitjUKxZwbvIGbJ\\\\\\\/XLFWZCsZhNu0xfQTGjxk5YQuHD4JhaFdZuSTwDENnAoDF67Z77BCTS+a65YOPl5R0mA8mExhbwVjYZ787pdRmfBq5Tp0l90P05DWVNJ9ufGctpEAwG8YN7QIX4fd8wLBlSG8AA4qF6ZymJMw4RVod0OCeVuj4pcrnz\\\\\\\/bZJtIb5p7LVMNoq4fyTeewl2F7\\\\\\\/Ogx1IkMOH3K9ngCNRxxpWcXmD1ODBodwjwhtuRZyweGhT\\\\\\\/yYMIRwpGR1CcjP18wyfzVIwyV2918EDLlGeazIC0SIUu3EXlOmwDtH9O6pVNthMSFj9bMYmLR22wd8+OFHxWvUpzub12v\\\\\\\/Yn5XwzGR7l8i+jgjzUhv3kFfHiM0FbVwaJL53Bg1\\\\\\\/OimdIP5SB4T7+VuTxU64aIiQbTq4J9Zr9bZlo1OJl4jf5KFUHf54cwREXrt4VRsiZ9hfm1I7y6yVCiG5tlBdDOUCIB0UXmgWAySq8E5VGCLLd9k9+5GYMoItxqLnyKTc2fr37PAsXwiAri2n+TT1a6p7+EXZyh2gMS+sDbBDgQscd8DTtuvSJqxbGg0rlIvM6EFnoMPzAy5hlQBMugKGNUlfyo7E5CEp5HPAuqg9QXNLKt5F08eEjmWde6cybZHQSWBpwnSz0MOkHo0M6BKcUX8XPBNSW6t\\\\\\\/4aM3fkYZkXvYYUcnSRHbSNLA98i+rBy1\\\\\\\/xiEbyTrqX0fys38gXxtf6vexryF9WJI6JkvflBZHfFD65JgneN13hyKFNnrWkAm\\\\\\\/mlovnAbl\\\\\\\/Eh\\\\\\\/\\\\\\\/5psryGxHdc2Qz3fFGcJEICvPb1FAO1Ap1ilHeDA7j6zmdJ3C8xzXtKEyAPVOf3e5VRj5ozNMpr+GslCcQtOXLYGnX29FAcC77+9V0\\\\\\\/7GOo6rSjdboBf1S1Wf\\\\\\\/92ui4obMs2QjKnENCM6lmQEVjDLcUbyJDdRf9YA7SfXuWmlOZYlt2DTVc+qyfVz151n8lvaqKrx1wzANBi2mClOPGVgjtafKtRJXIAd9g\\\\\\\/5tP6a6vA9GbHIOXfm3rm+TSnKge4ubbhh\\\\\\\/i7pN5\\\\\\\/sAv9CM0Fm5fyqi\\\\\\\/JU8C78MDoAnXnW0wJ5E2WENb1M3R0W8SxDQJt2NdrWRRjkmn29bQGHSiHxY3W\\\\\\\/MjY7FtCjoNXSNYDCBToUso9Pp\\\\\\\/3k\\\\\\\/qaRP2FF7aHBdviYEVj2whpUDres4zB2doAcbwQJL0K2ooAdYx8oLXB2jSSU7gQ32c4kfINXJ2KYn4b12c6qUAgO\\\\\\\/z9mfuBdee8zVIPXZmwCP07Kx9Vf8O4iojSMi4iTCRk5DyN7emO8wa\\\\\\\/JXqhyFc8BmsJeQsXiyj7BjDEwgJXMVVC1qadph2zNz9GpQKjExetqG76HRCFq2R0w6bEAY9eMk8PDOxiW8l2p6a6xDrx9ZIOXBt1l86STwwiWwPiQVu7HXbmGT+A7vrJCJBwTjHVN9J\\\\\\\/1jtgPBOTgO\\\\\\\/WA33SnaIdODVY3oWPZBZzIpvRUirWRaBj1pK4KEcIRb26rsQJ8N1lW1Do\\\\\\\/96w61E11CC5j57k1YfXz7tb4dLsZ5x4rIef6JwtL5Kvrc0MfeiPMLzWStcRTMuGC3vXcqj6ohycfaIl1JemZ9RwVgyzGv0kgvMD+LVyfRxRZY8DwgUc1rbh7gzltGQ9m\\\\\\\/ZH2vRWPpNfpxDtlM2BJvV7tQhbHCMDaHdiVhsh75WNwBVgBKmYuwDyTA7cCPJcgHG3ThnIUTYtj5hTtgBnhD10T7foT93n1PXbsN2J6JiT4p9jx3yJMy1V80s03FEJXWoX0o4JV5aQf8oUbS38VxQVhjRb+ARsZBfb6eDXx85UITGP5Pq81DPf135d3mtpBcQqPP8DemSXu+zpbYRvH7k6ohX8\\\\\\\/hKcC2thJr35bIXzCLJZJPaUwxrc\\\\\\\/ktvUKUyVeDEx5WFYEUZrBp6IpT9jM6eIdwtjON46Fi+Wihuvnmu82RuEq+KJ5If8vA+SXXNRUdRXGNYKEJOE632XXG4X1douOwz64DbBu6DJzMTBlTRaeqEyn4dpWRF9MPcoUkHko05fzAg4Df5q374yZqIl+n6o9gVtZGgS+7W3jzcfq4RZwEx8ZumWFpBmAGfml6FJZoPTPYa1kx1DkOOw\\\\\\\/ttWj5Z8gyB0wIJXr1K0WZSl\\\\\\\/7QJbllFm1Wu2LDQITU2ayGOu8NZrs3yQEY3yXLnC2kf+zSPe\\\\\\\/etTXa6TWrJDpDTNY4j00psBk\\\\\\\/tqxETL99lWy+JVZb5IIesHEs5MIewM+SOTun9\\\\\\\/ZtcR3B3cxhJiCCiWnIWA3n7jZjOkNEFh6WnjU0vGpyyhhH+9u+xyjPeFzaPzXhqDez07sJSoaSfXW+rLPWuYosqLknbIeP+MCQERlLshhgWqasqQrhzw0ovtyakgSMkLiOrM9Ms0U1wiZYxx2ajV\\\\\\\/aXUy94O3OstyJ7IOEBl\\\\\\\/vEByaM\\\\\\\/T7\\\\\\\/NXoafmoNRGZj\\\\\\\/5910hJyssqRpw5aQDCQyqhJveItinoh5LXemfApD91jq4GkxMTYxMxYoqUqKF7d4sdONvjCsr+FXMntKlqaGNpAduRv1BId\\\\\\\/rgSFLEhL+o0vwPQw3aph+2O8g7t926xydjuYH5sHKsFVfbuaxJRx+n1xuoLErlu\\\\\\\/iNq4YT3\\\\\\\/sU4SweAgBXH2+\\\\\\\/ykPl+B7htGKqKWcatXVcxFuM5EULHEHeiOLu11zpRZYjCLXSuHTaJRNwDj6cTVRqUaifkEeRzNNDRJizYmfn5K+ndbNF\\\\\\\/Y4Pcl7\\\\\\\/rBgrUnBs8aBrKCogb1e21E5N9mukF37CXP8odv2Vb5eKmXtWQVCrSZPjl2kXvK2QDRrRz5oiGu8uG0D3KHqMuPNWmffP1RCRZ+kKEheFmjXC716VH8av7DIMNRU4veXJLsYletN3vN6wQXyiZGAAnXL8pWu2EuNcQ7W2LW0HyPyy4tem9P3MSfE2Y0BzA9129UHH5WqyU9g1Bvv5dqTavoyFnEBizyV\\\\\\\/Tpk\\\\\\\/WhgcXgVQHJkHkRZ8WrgwZas9unDsvdx+HRORiJnjM3ANQhH\\\\\\\/\\\\\\\/kpzQK4bcDVcSoJeiAJWUYZV2G2hK7Dmjjl2QWafWnCQ5m6Xa+3LxYD\\\\\\\/\\\\\\\/Yf5TrQY2MCZkZKfE5SobouC6ytV3bI9E9MbGQYZsMoCd5tTqr1DZs+QrP9QDzMYfB5Sy\\\\\\\/WavIdHJBU8YPS7HcBB+59nhGz\\\\\\\/pJfA1wJTPIF\\\\\\\/m0wRd6o12dfAwvuAjriGr0\\\\\\\/cvTT5qRpV6tF4cVCqkzgFfxoMVGLCRpc\\\\\\\/yTw4x6RANpKNdztfoX\\\\\\\/YSjh6awCwjc20ssBP86Wyt8Q6l2OaEg7QRpIVSZtD3hMZgYT6sS4Z9kDVYoUfY7YAGs3hrwKpfAQlM9gTTJ1aOP8zOXNCzipkTN8l7bEP5Xt55tFdnlFJPORjGYMwUzlWJKu1TulsHXw4Y1ek8O+dJ5+KO5xi3JiA7eb85KibD0qlKl8f4ZUIJlMbnMaE7dK6eLvZojLXOm4lV5116tOjHazA+zvCO37rqB9\\\\\\\/2cVDkTwicpGgGJ0DcZx2NZe3ERoIPhmZcbx+SXHZguP32ky8o8BQo1ycIJpPhwjtCDbC47nX09z+HxgeI7gqJGos76nfmEu6+YagkbDVTCxAh54+P40Bw1hp5afor0P6vfHj0rSmQFeEradlxX6\\\\\\\/ztgX5G4AtQ8YZ+h5MBGVqeQdpwKfaQEl6rxE1sRuBrOr+IAizns\\\\\\\/MvcqI60p8sF\\\\\\\/iSbkq7iTen6yjc1KXuPurzvJB\\\\\\\/vPSuVNczqMKUrOtnGFyN\\\\\\\/xKP4MOYx1fzYfgSgpYgUQoBJ9Rl4lc4Q1W42hfbxzv8I822r7yXY3zhx6kLnl6sMOiJrVE7UaQf+kicpIvUw20fjncFyqD6wJ9jJ3mCbgJFPrEz50KdHkBjlXnUGwrndgWbA5THzv4wX1QTMeXxT0Ud8TJG8ZG+GyAVJtj+5mKDGgDO+ebjyWPDHLLnzPaT8HoXXDTmW7szlOlWoIYXFSyYqHUXux2md43VaYPyp7BV5S2BlGMQc0q4AmOKZGJ5NnFCuLJXE7eKIeaFHIbEA0urWTmo5KlWz0lkyxCUTNVtfWL4V2OWW5Zhsch8cJUsYA9OjFEIUnXhD+SMwYt697hHrH4F8NpwuWwQDh27wwSiODZE9dSM\\\\\\\/C93X61YVQiLAz54vfRGkn8eitN1kBanx74dW82NQRGZMh1E\\\\\\\/7cie5fi14jRhh9\\\\\\\/355IWZsf0eues8sTEWSowukANEb1qVI0U1dNND0VPMrUUjdglYmnX\\\\\\\/ZQ6tViAYwWtp\\\\\\\/1LunjOMhFCj5Sjr3c8ckG7qDadU6fvHw1DOsWnToPpN7fSlXumG2Grsg9esaPqWcQvbwVWh9zO6QNrSzhTk5kHF2RUhaeEpBJSCNOQalc0abNcmzia8S+L7LqlVrvDzfdqN4JisojWU6bj2gUrsCvh6RpyqRqj2Cu89a45bS7qHzh9GqaJ\\\\\\\/akLHztGXiZu0ia0VOTfgSeSJZukjb+uxPLnQNeOdgMGQCxLKE18GS3UPMjTGxM1tKqZHB19nkrOLwIJmzzq3jCuOlHXt7mQnHbnXTo0pvtBm+BPvqEOoAPQvaRqM2u+4mQK2noj\\\\\\\/QFJIVpWJ6V5sX92mOPVcDAENrL6tXSc19Nvy9Xrvp4pS0w0h+yYguH+cqvrbO5mAmE9KEs3tAKnbYe81VK8j5zeqqz5sF1ga77K6jDNJaPz5DcEhgbke6H5YNnBZU+DLiPeUZpHEfR5IGjcE6rfH+hA6vWGCjSjeG6nHAahwjSHBv\\\\\\\/lcbqWUX9\\\\\\\/HB9g+\\\\\\\/NdHOYwwOWU7MP2Wzt7jqgTNFBFcioab8Y4bsq3CyJQwwBXQtborNdixb3auDzM8mn3iwpxlYOw7xLbOFAOM2tnqvb4sC8wMGwY8Mwh\\\\\\\/+ap8wXmDKMRtrbn4wl6h6TBOKvv80TKaGa7lYNRI3+SMmAdg6q9zG0ipYv7Zp7HL21u8vXDEklZv\\\\\\\/\\\\\\\/I9Z8CtCdHyEyolkoql4m\\\\\\\/UjZ7ac1onKJMtwR37oxMrbsv\\\\\\\/an1UwuWj01fVhd6qcRaVRG0Kg2HhtIskYoIMFYwskak+qinlUwWHGP2PRGP38kvg4CDGu3a8fxsEOmcJkxZRBUqfO8P1JKDZpc68Ihc8D1HO836uPSiB2b+l193qjrtLdIs5s86CZFoTHC4TL3AXxCTqrfCcu0ddHBlXbqcCWYOHqYTou8ZGEWuQZ99KFOh+wXFuSadIdtp7gXQT3uBgDM1pEtw74crS7dbr408B2pyMdxIFA2u\\\\\\\/TAjPseqA1BQkBrp0Z\\\\\\\/bnj7J9xujfvhVx0B1FEGJAyZDnVqAlepWe2PwuDLTxmnhGmoDeRW1TpPNF3haaPMwodEa9Y9pltTfslbkX7vPCRtCChfEsgleqEZCzdYSyKONGKb4Uq\\\\\\\/aZfTuMR6JCod9dZF9pdznJRSmL9AAPOySNS0m6y6XdEP238WkOGQcDetP+mJhbSEz5Wj3yyI2bItz\\\\\\\/11w3Vtjr+9bp2lxBEh+pSKPL5FEiaUnFdadODf+Qa8Q+eN12J6Bf2ovLm5zywCe1eDefFr3M+KtBx1shJ5T8z6H7TuxbGgOQ2oIJV78hapfnnCR33h16JxwrUdrYf2K2xxu2RClms3i3lM7fZmOlJ6regHi2KzEsy5IuU7Bw15zV1VWo42bBCFkNRUXhrX8pzhHom0vS5599C01mE747z4QxPHGxTvjMWPjVXgb9za0E6+fHh\\\\\\\/6SByJeT\\\\\\\/Kat8E2vvf6tTwWRvxeGO+4LiENNkXMErfPD6sYdQO0MA88SfmVu7CucCNqD9PhixwErsFEpxTgS+Ds8OLd1ENGXPyQCnFplNKSWm7fhimCQGD68wPcg0l0Z3ETiSMd1WSGNCHCzFppUD3iYN8Q+\\\\\\\/sHoMOBlv7Cex8ibl5C1N0UBw3W5QMWw2EZz6yLvGCSE5KILfKI9hnHfJhStAvo+dmMKRdTGkcoAr7WNgmvmzpAdO\\\\\\\/O6+qfyAWfaKBOuJY9o8FBRkxnYIOhHuWA38MRgt1MNCSoG6rIIqJ6CiIHkujrd5Q9YMG1Y1wdYzguf1oFdHfCPotNJZLvj6+ttUDDuyOfiYxFBDOKqBjYsBTgvD2W1vMuVPA6vrqfG57+hfTwmFS+LQjGdIo1DZEvnvKRRgIpj9jN0Vw35YQZwdeLkR5l+xtqRI3fzRTxNn1\\\\\\\/52ur2kMax+0t9do3pBvdKnumYNsTH9AnEUqN58OnHHqNnOss6aiXXjf9TqnAa6xy1GfgZJafIUD3zMsgPmHUfNrPzfaEmbFXoWZVKfOSmZAYKNIsBnlgNu9OdjSF+e\\\\\\\/KMb+tcW6MbqMr+w1cwJ7vpxqaxO5aMXs9FA+s297HCWBxsnRHS3HJ2kX9eweReCQQvvYzIENBUjC7rTWCcrE\\\\\\\/ng8mUVFcW6MzZOlb1+iadeDRf6LqyYGBBD0jL9l4LLm\\\\\\\/zkA8XFL1nA2DZAqC9GevPsiE2b66YJbP+dB1siVYJuoQdfOpqX3xzAByTuvy6Soe4\\\\\\\/Iw9uQJ\\\\\\\/INKoCnwI6L5FWC9pEMxKTZfcvtWuvBH\\\\\\\/SDVnvqGNkWgVbnPwyoQX5uOH4na1npi+5rdS3nMQtdwDXjtzid5H\\\\\\\/8GMFOmY6boWQzoD215b7YoHjvIsqztd0ZOktz9XSQXkeajlsbTnh9j\\\\\\\/smP7kNrJ7E21y98DY\\\\\\\/TKLNrF3m4ltyM39U\\\\\\\/DNd4k5BP+xcM7seh7WCYlcov10lKrKQAH5lv2pnJKZr7n7M2kb6\\\\\\\/8EuhridGDfQ\\\\\\\/nw5cYR3o0YDNitBLP\\\\\\\/gGVS\\\\\\\/kkUR8XKFSOObqmqRcalgxQiQHgcIh9tiA+Gnc+cTjG46p7NkJgzzqO0gZxlTNkJQIMIHnaN0p2ldswcv1yfM8KPWRh5aCofXpM4udLWSLYLcV043bw9o10zlCKJv5eSFqRkf3SgSFy4OWJMGQAtLoOzIMtdPo6Zc3V\\\\\\\/jN1zg6UwSCb9l9F\\\\\\\/mYezlPK532pQkLxmwZJ0ca4p\\\\\\\/5gBd7NKmKtgEMWzqieNEt5yEPVwqKx5nngmx5IDijzaRBJih5Ye1ogCo8f325Yc1O2dAhgU5fVW9ywkcxt1Nwuk24a6BygyCVxGTdly49HZHS02LA9JOtCDvO\\\\\\\/lOBd91YuGm2bFows+0LNDOuQibEmsIYMxa6cQoRTsfBqlNsLyS9cCwfdrNkhtxs+Dy9hrHJlLlUPz4gXfpq3seYKx2AtMSWi4ZqRrW59aAiLisd1X3y08Zf0dc6y09kr8XG23zQtslwwdHGdMaxTgCUfsywom5LJ4fLIJyGPPzkIj8IMxwxKalEv4FYaBw5K2ehtpFh\\\\\\\/bb5hf3pYho0kSkXSiJrAJazR2c1SPyX4AbbFW6h48JsSEIezoEJOpGYiLce4Hgb5\\\\\\\/xUfOGNRn3gwQewHqRud9iX0rICsP8Kb63DnXsL0t2Rg52qJDFqSiqo+S2jlJ9N\\\\\\\/WwhB\\\\\\\/ibz6Fq4oahTtgag6vMUIK5MTT43MAg6KpzRfdTbfE7MtuEOlyl4oDrFdIQskTZBMUCMgwKf0P+WyOsDF48Q51xWbbUhSxTa9\\\\\\\/19bjmzsazcr4eOhxequ1R+0YUS7\\\\\\\/qXnqoF872fcMQ2ORReTz5EYM3\\\\\\\/AEkUghGRvDKO1Lcl5CgI+0\\\\\\\/1h9CRCzdS4\\\\\\\/82L9tWehMyQLVg9hDhWSa7\\\\\\\/gqlSQCuE7AkqzfZ6tOPxTg7xc3u3IG0FvyQ3wlpL5ffRz3zFgIggzQNnBoa8z9uIN2ukdScgsy\\\\\\\/cnJ\\\\\\\/ysJLW6y9\\\\\\\/RCwbalpzvVsIDbWQ49RrEUF8tDs49LofjsjsAFLKH6yHfpYhlSoFpjGeCTjNINyMMU9Ry6169KRfOpDhRnfevOBX3IoQJIoj93S3+VZEpErCK42\\\\\\\/GFTDaiBF2FmXIQ89QxpuScK5aW1FcyARJwDNJ7N3noYqCH2orUocngvJ1PZONUgUnxzB8zXe0J2cNXqRRtP7WKpHgBeETYp8m4QR7VtypcoqKc2AQQrU\\\\\\\/1kyGrpXoY9gvpYSxHBjUoG2UXoYLMAp6ybAHXCxGk25W9lM5LNSGGoCVzjXNlJPHMPR+BbxQqvewe5EZRDN2dFE3AXUt\\\\\\\/MIe30U+MuTPLZCnYLPYLRdgAt30oxbwILD1UP2itQf1QF6Qoa9jvJtLOBclBA1bIvXiR\\\\\\\/SB5cskIhGcXiCXOTK3woNlckkEzbXpMb9saP6bbke25aelyZq98RUPyKnHS+vI5YQvSdEoI2VgAy1umws2DojzvBXnMdG5zBTDuaOkuHz+nEGotgU0WC0ZVWCTaZyqaBaXAXRbW+gzqPnLq4RpUNlCgWOMep52K\\\\\\\/Xs75Lgenvs\\\\\\\/rADWRbSHic5ZKAttkDvqRjQQ0qxH2VGUz69QZRX72NU64Gm1wV\\\\\\\/791FemNc1cNbrHWsrrFUL7bCx3xj6zFxdT80hTr5qOMne+E4BaSb0JOzlSY3wI5gg6P\\\\\\\/BhBXLqmR3zvOZl0CBUcBwr6syq7gyuRm8Q3AMC5DB0gNqVBfYt\\\\\\\/jmvJibvIJGahw7BvyxekwrSrRuj9ht9cWEQwWz5CL5oHur7TShzVYwEUrz2VYYnxeAin9pHpasxvvFhIPA1F28S8mIRX4wXsUoNtAS1A3vz2Tw04rVb0+fTIxudMQYnZS4W8DZbAFUXdB9MwOHTH667TJ\\\\\\\/jjRPIPBtT0tSOSDkQxpmmXRysehBhCxsOfv3tjtfyxbSNYG7Q8tbxCdqOIAL1K3kkuNCzzNfNKgVWa1BNw62OBalAynFgrfZHYSwOL9cS\\\\\\\/XrsubqvkaEBi\\\\\\\/jfisPD1rKoQDAg1mNkPUMNM+f2LBgbRe1+V3nmDKUdIGdD4rwVvUKxpHqZosVHIJUiT9Q7OGxM0+Vn4rhs48ddSCWMediuq1PMQfOwgq8JkMx+\\\\\\\/6fgNV2iyneUVI3Yuyh7a5W8xFkV2imooHvcIrfDqCddtIR9O3BQjDrw1tHJTJpYqOnD+bLuDS6POkUPbaMYtl0dIDHngemU67pzaT2N0bKv8MEfhaRyN8qlRUWkuB2V5gciI7zzXf217y5rNVWZZYrtxKqOtDzFiq9y5zgm3RANwCMpHrO2RmfcfJAaJSo+VlGB3I66WpY8hDu65+N7xrcNTtVG4tihxGs2CvIi1A\\\\\\\/CAPznUSv3NsC3sOWkK87NvSnfG3IRc3vrUwmdSScLW9ME\\\\\\\/CEOiKOqfH6R+AjEocjBy6xwJuX+qpsma9ilibS1gWu8+lHQKuVyCrXvcrYcPcz8BKidjJktcu+34+1PVgZDCgLgemlFep2ZoaFQNHmeZLTIh\\\\\\\/\\\\\\\/9onCDlMyt\\\\\\\/VXr1Gt11t9\\\\\\\/l44E9O2WODWXOwSSraluPBkw8d8gEcBq2VLC3Mz1QNphbVReEqzVRLkHnE3lxqVImIyzmPXl1CJUFdknv9JPyRpnx8TkQDy\\\\\\\/D4ZJ70kJLXC2XGxacvUB7b1ENOZSyk5v808z+CzvoOYrWNDlhWdIhSWd7GO2reKxDjwhkvK8C6HXV\\\\\\\/5FczqXwzutjiDlb8Igjpj6boCvt66XcplTJ3T+qOApQSlddSKiE9dyfHQwMNjeW5UXTqJdcCTyZKFSbAo8ekz3D+5qdwyoL8bRsMQjzBsV18HnZlwsPuQ3qLrwT3H6jNE4+P2+u93DupgV14Lmnf\\\\\\\/UYTMEl2fuCkF0kpG+KX2KsY3x6EcHEnGnyApVmI3HxukrLkkdfVus1rSICgEWA+VmwkURTZ68pK9e9iu3B7rxZQ8wEMEeRvWM9iWcro8ZSmzbiAnhs74Jee0yRTPL\\\\\\\/KwnsfBfWAZvHn0z6BCZ1XxvgiLeFbjCb0yierM\\\\\\\/XPXi6IGCnsdh8U2kebkPVc+bEG\\\\\\\/hJyMFYmk0C9\\\\\\\/fuGYHhmb2Uq8kZl2kk1Uyo7Orv2wLaM0pGSopKzp7GYjBSvxqD1gBdeth9PLTvmnCkg+U52NcUk1GyMMAql0k\\\\\\\/IR3XdhYxA02WyEsnUvwqaZGIAXFI+no9hNDeRuQSAWKuj6wbvBmG1\\\\\\\/0081gXU7cdFEvTTzCqBkNYAa67RsGDrXVQMfrJduVhwo6JG7MJk+ZyWJjP4tQKn5L9dhC95K2+W3vpxuHhRxhzkAi8stZOWdWCPcC8PlzA0X8oc27Bj\\\\\\\/SNf04l9EOmg1HoXdcDF2hvrMiiDy7Lzd6eETok1ygMUOOuIzbUUclJyUiAiAA7vDVQKORrNy5Y9WpPAq7H+SW4I1+vjOcNchcMxZTtAGtZz7InZn4XssfkIoO\\\\\\\/8+Nlz4Sgn5ELgMTBKbcVU\\\\\\\/PNehK3J3D2uiqbW4Bu7rpbXxpm2pukDksGkZEkVOKvRQETUh5Z\\\\\\\/b8vnlrNYh26XrvWfQ2517oE\\\\\\\/fzjepzu8yKVbgqxH\\\\\\\/SGCg84\\\\\\\/CzF0fmQfy\\\\\\\/yvAbKT2zCDbceIRcpWV2u4Fd3tOGXs2zwHJQSduoA7LqQ8V4b1jwK5k6eDEeaS4l2Ya5bMm4OwNKLSweWPZdHftBlVHlWyCA505kZ1VMO2SATVn94Q+6nqpw16yhF8FhS4TxJitGvhf2OqK\\\\\\\/1f7w7OLgqsKbZnHVBn6e3QHT43xiYy09yYNCWydereBtF8xex6K\\\\\\\/utFEdS5\\\\\\\/yJh6j6lPOkrFFJRgGfHqLvzSLgTHN7z5rQVTxvhNe2X6IAcTwNEgD5ymuSxb8wPGPkbJH6rSdwUZVaKvUc0l9AKDmxt1O4h1oBn\\\\\\\/pur3pJpRUzEvnLh0UHwtNfDrP+QqqtwPKHVdLTzMpSpi5sWCVZXyhVOJAiVIfKlsWARMKri1cSAzaCduq2HsO1sdRwQWTIdf8\\\\\\\/scV35Bb5WfwJ4bAqmDGp5S7IWG0Nr9B2sVbTc5RDFlC0xxmXKVvzTBDCkTzIwnC2QLPNsat+fQm0JlcU6pUhx2cHRxSaWGRlF1PHiYmwvcnuybsyQgj\\\\\\\/wy3TqVYysEYVr\\\\\\\/mJs5aOAQ3bQD39sBQyYO2YXsviFbH6Cf4rTFr4NB5eS5bPDYQH68UdsGhMtwyUcQHy\\\\\\\/4Cl4nk8O2EilD6VE6S8\\\\\\\/60P9w8aYPknq7Ki3Wnw2iSzccA8e24hOLLZaTGaSYmlSsD+EhWpsITrROJf0VwYfdnt\\\\\\\/EogaI0mtitkKf8KLlWWS\\\\\\\/Ov+D761MeZCVPcVikHh718ZB9gaMTcQrwwhuXG8ZhP51SDKsx2IdYdyaWvU9KLRrRdbb3zXkzZl79phLnUPSUly6mV2R7XUwL\\\\\\\/rwfSZtiyunBUTBrjzEfOmtvTvDaja5Skqs\\\\\\\/Mpw75rA8rogOUyzbE9gPq5WdSBo9c59I8ULA5P1WEjZWiJ+JSBo8IicuMe4Y2G6PbQEjVoSH+8QWbieYvF94FDptmuv\\\\\\\/VoTm1sVZ\\\\\\\/ZTUxj8NpD\\\\\\\/atfhL7YcPY25UsX6ldmuUVFqE0MHKzPgqEqbXwfY+XvZyqj6kqiN3KGCBZIczqzzj1okMSZA5P1+I4AtlVo2PzVW1\\\\\\\/0l7zghi2VfKM4WabTGIAHV5XyLDxCz9Ef9Kj85r1iVsqg2a9ncahEn3vORwH4BZm2Un5YL0goWDjcxgKuUpyxTYXCifmOF85m\\\\\\\/P5eA4d4A\\\\\\\/YaSp+gqgyt5bpc18A6jhH6fpP29FgGnQyc9yxzPcYSyU42eXX6werf7p8L0mf7l3aQJCBlV0kTVzaSv6cYWt9acYN5IOzpqHyKdxuhpxDaYJ91CmD7Vx6kW6Sg0rndnLRFerZJoyiX4H2auOKOZ42zc07HKTGEEOgL4LZwF1YZvnM9B2\\\\\\\/mkTie6BGdZaRs5M0V+zV7hoOy\\\\\\\/Txf2L4exAUQWYJ8lOIVCErBZgacnirLHgSGH3ILmwj2QV9ylmDtz5GcT3ONx1fphw8MlK0T+i400+SjFh+wUDL9sfVnV6ZOgY3acMhfdvpYaSrRuXHnrUtkF8KDeyamAnBWsxZXXr6nDwQGmmj4rP7i4A9ICMp+wwpFlR\\\\\\\/\\\\\\\/DX2w0v6x4lcHYyiZedKWl38aZubdjpjkcFRux8UOk6NJ7eX8FonNRZFkyKOPISpQ5oO8zoGjHVz+l1Fig781HMi1lSv1mi+mysRM1dOCod6dw8BaZ3UWMyZeHuMVlM1JlscUVpnXVKsohOjrwWy4HKN8MM02Yaut88kX2dQ7WzlOEW3OcZrEOqWuoywIDbLsWyiXIF5wXbw7\\\\\\\/Q4BI8Z3KdDZpIGMnOqUaEu\\\\\\\/0D2j858YdROUcwn8hxXemwdA1RCU0vV59GRizZ1VR9fi79Hss9+b9dAKKaqQgEBu\\\\\\\/\\\\\\\/Pat1Cml0OUMfog8W2bQzvaYObJ0e0obEFVOTdfc9pOFVNRODqyVCvbSvsF+siaZuVsh28OaRD\\\\\\\/VI6xLdDQa9FFDikM725g3PZqdGCAX3XoIjMa0ZTXH1Ks4C6+nQooPiKH3O7x+CIGHswcUaddPTu+Lt7931myLyowZK81xEmJL59YCh43Ea\\\\\\\/gN8a7v89cuihWsaCoK7XL4qllg8Rh+p9nLQKOHepbc7aTDv2B\\\\\\\/Qma4ecw04lDZ54tCZh+MW8bul3rYK2dLLD2YftVU1uSoLdFR9kohMQ6cShaFmGsQiD5\\\\\\\/OJ8j42aXEJMS1qrm0XZ\\\\\\\/ks1R2Dts2AVem8ac9d5Klt6pTv3LgY5uYz1MzR76AnqGUR48VVyirDD+6KzVKBgmLXq2R+\\\\\\\/1sOCsYCOfNwpyRLxGTenDzYan+lZYVOS0Sm8+pZzhXx2fhMxVcnUj1bpO0wlQ+9nNA1H2VaJwtWUUICfedz\\\\\\\/TGtMsY6KJpkS6c7G0VorQLHpl3ggwPOVlDZC4zklxS4BIxf9+AoRgC4d+Ep0bWxxrMats5bPLp3lZNb6ToGRBRugH02aayINXqHiiCVFy5Da9E8hTqA0fcWWFCellEVYCscymM2H36g4PBiAot7aha61aVnqNoEFCCiR+xt+0H\\\\\\\/WsaLSwaUakUvIEWp7PFdVRlwpozUukihBvu6+FD5A22LhTpU4oDVewNQ4yYLKTkAQXvHX+PU0S\\\\\\\/eMUoZT\\\\\\\/DIGThMo4vMp5sjlv72InhYGoFE5auw94oKvOg6dY3UXTb9fA+ufwbgQVNSzh5H4ERdXQCq8zPBbdYhVyDTGA8N0LfRc\\\\\\\/0Z50cOew6L0BblsiXYENLkRQdquAOCDmDKMIO8tek2xNqcGtBoQTgRn9N7YH\\\\\\\/busEDKS12L3vLHvEdZdRzaIasnTDEiFQhoqAoy\\\\\\\/2qE+pyuyR1VRvHNfi1Zv3OqqNmwO249r75un9zPApVCDNYCA6+OLXbielOh913n4in6rjJYtPt+o\\\\\\\/+jkRN8\\\\\\\/JOdhmSqw4XyGlQSVvkVmkVUnMXCx3g\\\\\\\/IbdeCYTNGiwqEFGLwwtI+je3prTTnqkye3M\\\\\\\/BigMSSacYCeQ+ut8pxQpdbNExTzx3ySHknAVBZTXm+0B\\\\\\\/lqtXViarSxJcxlaP7Ql+dcbWHYOZd2pc0ftAySGnu4zdgIRQT9hw3ApDcJWuRED9Xmm7012moGxbpCUWchNAPm8Hbkb9pQugbU22oPuFZ3DCfghBZByB6in0NDux86cVdoIMubp7NU3EJlBG4sLCAAyPUL8oRYc2SIFwSAD0YZ7UOHUQ9u42ALzNsrBuLZj7e79CvKML2KqGN3YrhLVeWdEMEtDYmpwlG7rUcwKLPZEGYr3jrTVHbSr3DZsfYLAa586oI9FGGnE58nUnoVlyqCoTq4tW5imMaKXLjfmNeV3WiXpIN4AjrY9jjAowwMplwQ1iu0nlDsmzSOHbqtPXKQm45XXISZhPorkwjHdEfxEjQ8YfjKBQ2peKgAu8uxcdDg\\\\\\\/Gz5OV7Uhb8BXeJXwPEm3+LmXfp7vzjnEEZSyz4D8umOGeCvXWkdmIqjJPX7i+cugJverSy9qz8pxyPIEhYPYPUC4FyROQkUe+aLmGd7f4MM+aL3SOpN6sVTNPH7fGjFYtP6ngF\\\\\\\/+JLTYQwE1Rd1EuDQcP2XOgS6xuWLwREfilWVwbbiLyFZmA5uRdbWNRzSBWOBm4\\\\\\\/RmqzTQplzpLl6+NM54dGXdSMA5qqm5fZyY2dEQjNazRtzF7\\\\\\\/TKlv+7B8C0grPjjZG32DEW0s0GtX1qITEceyfj3e2\\\\\\\/DTjbaSmURIPshvTzNObpSgUDpbZnqzzwf3YryYd4YK2wf6U+kv1JambnwQmh6OApqILetTImZ0aF\\\\\\\/eSpoQ2rV3ymyU7s\\\\\\\/GwdW+9sV+2tVbwJ6tIMuCYBff6hnRlkq043AbJy7wYCpR5bLrlhjLbcbiFUfK2V0S1VvzKpszQlWYQWhfxjI3Eb+eoK73whajaLm7P8Wi8IzSYlhqOaSAmc160MvjTvIIGPfUIQWm9jtxQ6IgL5zVt+9Lbymp4eiouFA6Eij4kyC2yWuT2\\\\\\\/+YglJEH0uIndK+fOxIcrMxmu855qp1g4LEkQ8sX0SPKu6RqX+1MzuAv2Ybs1zISvlrnjsznnkm1AYE1NpgKh6vxtEHhAceOSL7E\\\\\\\/Ttw2N\\\\\\\/RByh8D0kk376awrPXb1uDmELbNsh+vk5AZ5SDkYXB0eU8li9V7aJ88IprQYfCIU6\\\\\\\/ro7Rx0IfWw98z31HSOLA0E6Ex8FzZrHGtZMpNIVOh9OGZpss3cm6JDPkoVs6UyuLOSrkBvhqgmtSdAt\\\\\\\/0zBtRDeza0FStcHk1yVRV2hwhYypnclNrCbu6umUXTJl+Bj+6bBhhUnK0YNyujerJ00h8lD2kb\\\\\\\/ozSLVQURY9Di4N70MFQfPP7e+kY1FkKudcPurkNJ8mc36hvZ+qhfqDKCCTytna6Po6hF56ibh3ai25d1q2Xhk+249rFwVglj\\\\\\\/iCE2h2gs2jhEkNscKGTWtYRD6QyjDX6iEwU9Q2WMUceGfMiWbQcemGRXnxiH8zn9yVOasCItnsfG2GWRBYxLLEG7Ub\\\\\\\/edzojugB9bH6D6DHoBFD9dqD7KWNlyLfDFTRfHNH3Ek\\\\\\\/PTZ17RXXUoCjR\\\\\\\/z\\\\\\\/F+ynD2GJ1nPhEnfdWWDILnDWHblSfKOx9+YVLB9NTMO0EEmFSphr+8tTLW49nwTk48HGiiu7ekOfAIJqWOiGtfMgrT83AHnMRQsoOHifBRV1uiB+4hN4ShLaBADpycXvR1IpNb9bbYLKDQl6jk5ubp5wVeFyi7QsVvWFQSsR3tdfGlssHpNH3gkImoqGazwDpeE62uh9h7+4arWUAcoRc7iQG3aO4juIliYWqKkJBcYbOiXhjZ7eGH3QPnW\\\\\\\/8Kaw0Paho2isMAXwEkq+OEOpH+7\\\\\\\/9JyqTvqhfosY4vip19xl+QA5WgHF77jtwVIxmjJrpYFx3RDHlmfvueNnrr1\\\\\\\/HiHVODSt682bE9jU5RnLa2+hdxQvDBE4iss0m3LrFgWYXYwDD4xuV5TcCN9CNIfdnOTFJaWYP0COmWCi6UP\\\\\\\/3EZEYACNLf1C4H2Sw7iclocm5gMtlSKGFI5u9JeNQccODBn2FuUPuTs+TuaOTMeQ7DZFBurf82G1n7eDgjqIOO3osDKo4QdVsHs\\\\\\\/qu05EtECYiQ06jbZlQh3dIAQ01Gyr0QBT9rHv5+WZ1fXxhekA4r1FTBuRIATifHTPn7eoaJVssi7rnDDVsIkYO3GWUkf4w89RuD5TERP\\\\\\\/FUXXIqbQG1ERaXt5PbNDkBz\\\\\\\/QpPCDiWEdjZ+Dyuu\\\\\\\/NO4IxhxxoenKkf+PDIi3R9OipC+R8Krp2LGy11KfRVtxCvBQth1hCwL1BImFr7ivwrsGMuGoavcTN5lWDOieHe7mc5e7rDvprfoW3QxRuuB7FfhMRx9Zk2AnK0\\\\\\\/Cbz3Bj2FHipbn9rj9tER7K83HUb+CcQhl3EzBQfgoO82vuSfNiKorPvXOpptMJ1dzRxAtW7nbKvNfb3hNqyqFr8L\\\\\\\/iwukiZWHvrlws1vX2PpouLK86+b6doNvOUTa899v55qYj9ZB2qXXNIJ2HyhA8p2Zet69PPIBh\\\\\\\/xjsUC7a6KyhVDmPzHPuiZVTCYJ8pX5RF+TkLFps3xl+Nx9DDJb1fFI7aXtQC09yT4sgcvd0CTxKN+pt3aO7fv8nQgOueHuj7RccrB1Knddj+YMu0cx2wdhCNu4ZPUfy4GOrANNu0MOpXAinjSWLkrYIWcGph0nNieznVWbeNMkp1w5AzYevCWXSkGLt6J3um4uMgcd03LgY5bHYYZB6rJsKqTIXfTMmn98mlTwu2\\\\\\\/TpcI9vqZySM9kPbczBoTZIlUXu\\\\\\\/18SnmmxlYUgKsifLrtgEy4\\\\\\\/VMMZwRuGLKEhrIfH7qTB5Nvs2aZjbRUdVTAX2kJFQis54U6EkbC7l50qWC+WwISw2fWRqHOcazQcw8phVSsNWQG37gCtMfRv+0FPfGAJvKMLB2FKXUScsrd5TvyBzhpA0LUh0YTNd1IqvdiCb6BXkgQzx41hyS8Sduglgld+P5beJnXW5pCBiufvk+MRD8VO\\\\\\\/VOvdhnb40EFlLq9G5YNnrRgLqzAPETHUZ+8uAJH23kcI+B1pSZtYR++ZHCEvLlefRXZzUjCw3d5eny9NYM2XEgZDAzXWD5eCUSeFnY5oUAFUYR2qD7\\\\\\\/r\\\\\\\/fK6mC3hFArhmoIuNztU9yfF158c76y0KAEN8HdQ+Z\\\\\\\/OxbgNKiGFgVlpCYEZagDB6RIG16wQN1LcTYfB6eN7DoBYU7UztWJ4KJX2Pt37RN46x4yZlcV9V2w1qsB3S9MZJIh6l6BhTNzmHTiDB5JmlBScy6RHr02Avi7eaEc85WWjVPCO4jlvnfs2Ps9U8PJSnNVbGwkh872ND8miYtJAm4DaGSh4yuwDqgo53od2F894X2ZN6Pm895oCYvADAdrpBas5xH4kmspWBURGJeaSXfFZ5JHSLlnhmiB+WihnRbwPjPIlf0Jp\\\\\\\/SsDzPOaoRMNu+qsw\\\\\\\/XCQRYQhDCAH34lRs2ImuLkYMJn3wkQMFwtn1WKpQpPivOnA4m0SOFmPgq1Y9inxra+r+pur7U8mQK3w5DZkUusaYP81iQg2uQK1Gccp2L94F+hmBQJYIy2mvNIy4NEUOgHGnwd4C6os2cjb46XRrw3kBo56ZRCiduixutHev8pkh66lyEBEe2z9+MJxER+3X851gBrv\\\\\\\/GaRjvBfRRdSZIF7m3+VLeRkyB\\\\\\\/5ckkX2jdeaVLBBcqC\\\\\\\/pE1ixdJWcmHASi7VkzZ5\\\\\\\/snu9tTNauNtQgZis7fbB4Gb+DhyEK9PiRP58fMKwq\\\\\\\/lEx+dPtcTsh\\\\\\\/9LknvoUMhIynauk\\\\\\\/3VjIrglJbCm1teO4DMxvqEaWRrBZO\\\\\\\/qbZvwYKPl0yns9dQrskCjLsdIBV\\\\\\\/2wRt5LLXPSWeB494vIv0fC+5m9\\\\\\\/Em28bJLIt6uIbw8d3v9uy19ZY1gZ5GdSsPSfY9HraFAc92U2o52T\\\\\\\/++7ApkT4m0EzY9dT+KVlt36CTe8n2fGlCyYCSpxjUQUTWKd\\\\\\\/thXXeFsZAV5ItKrGktRi\\\\\\\/57IMrGxYvV20pq1F9dXa+hZEomxsEr9KNpb70yDTUO+7IOB2s1gaLwHfCkWCTnABQbwma+za6Dx\\\\\\\/zx02OkraJ5Qc5Fk2lUCtPGchFvkudns47U+e8P4NGkviuAs4YX7o7KhK2gt\\\\\\\/rHd1kRWdrqJW2BEFNCTNbJf+2Vr7vNH6AiwTisIX5iUVMO9\\\\\\\/AEzlTDXW8fxN32gLu0YNhOYC9lMEp3JNRDZppVg36oJLFPhPSLQZbOOgczmeovz87GhwOaiEY7erGLyZSHcl\\\\\\\/tEXh3EAewuINSfLWwhQhA2jl70oLx4y78UBRcuBejW8zwl4tcutXXlBoWmaO6dNoAg9KSoxEuS0qk8jLO5i3\\\\\\\/lK6Z3ZF75T1Jg7m7zCfv4WeV6oY4zh7av9Ugw9udfrDe1ZYYfcypD3exAAvaEY4G3ByCxNpVNJ2ulnNY8rpplJDzn+1wxiiMfOwy1cMXCF96fGdrs4DM+xeoYroPjhtEUwm3H2tusg4fIvarY3QwSsdtwXBcVYMY9kRt5mjrLT4x\\\\\\\/YG3REnP+CPStWtZURbj4DvOabYaZ2Sw2PctB0DdBQBZHLYGFfH35RDtNNdHX3dt0iNEZtCUdjufawTPaTUk5ATVEkiBZjQrwc+kDyHhPZwx0fhwhvViFXlCcnGMpRWdPxsN3XYQwkoF8C4htdhN2S1DYerQ1H5J+QSWAxrxvR\\\\\\\/94uC6AUQc3p0wVVi9KNKgFTkMXFTUwKnGQ9Cqxp3ueR6teXAggmFtvVmzgOkpS\\\\\\\/HLsrHo3q3LYBCDYmdZZnOAxUHRaUzwjRcQ+2lNLJv4Upav1ZWs\\\\\\\/H\\\\\\\/8cJYb9Jg4N6tI4pNSrxfrYrJW+w8uCjjxRvWm+ghpko9xJiFQb+QiJYmDORyjnN3M9J0UxFw7N08rpp+OiMXIMXKch3UjV\\\\\\\/pihv\\\\\\\/Gfez\\\\\\\/ZUJofdNiHwU68VIBpS50gB5aRNwawhhop1CcN3+8l3ZRlPgq\\\\\\\/EnKQXf9NhOVaWsFq2ojXrINaqerLvYD7XR1TVkObWUVW62+7y36EQc6Y4p7srTGCG7E\\\\\\\/41TnQRg6ydM7KyboNYH6tCtfboFY18tWRvMV3yyDZQboWMEMIyIn4KUBxJ8kUjeeN4Y+38H8kNk60C8fWmq1PdXQ45iIMqBxx2RoDrZKlRiG4OTGFQta+5JH6HMGGJO5btYfKkz2XWBr0mfXmk6u1l9ijlBwkVA7noWIqSBvAHwtVfYfoNkvSGA3AIeFH86fhXO97LFxILiP\\\\\\\/y234Y2pw58PQL+bmGLlpa9sK\\\\\\\/OG4eu\\\\\\\/4TW+TgRauaGqgTpgCBbW7nfVHCLK6q7YsQEHZ9LwymbYvk9QpH7T1FZnHdqbDbNurJ+P2E3rtWB7bkwvVXUyKmNBUKltxYLM2uCXE4quiTJ05lijW39jpE2aMfFGCIIvhjnfMWxQhh1gTdFyfsID4koq2tBEGDqMhuHDUiXxyo6nxLw848Iy87hMtv55ZQWz6+CRAD\\\\\\\/ktKI9ezFdSQc8b3lsj35OhNvV7slI\\\\\\\/MP94yjEMMTXnJuxh2tjRny7Br8XWXbhxePGz9gzfGRejuqqU\\\\\\\/W3r+EAbkLKNDsRgQFgbLdNfJFvHz\\\\\\\/kglsq4e4hOfaSfSCVzyAsMGb4St6TMdH2jfn+RbYmEQePfml1BrFpD68Oio60qOriSARoPm+yGoyglibFT0u1p4HeBqiBbQ+bLQNijWA0NNVkAfouqAhIfuena6qipUwWByYzESndfFFAnrALxBFQ2P\\\\\\\/EDWRquKc2wle1vX0M+Sh5GWGY81gkcdkfekJ+erT1yWZLtsBFF1c5tbz1GqntT6qvtuLCHqC8ZpkQBqYU0aFDhg9T8pNobnLR3QpGJL1R8BTc7+uQHF1Nv1AaskYhkuMAnqM1u+jkYSaD7Kd6hOsVxwtt77sV02fsLL40ktnBsVyoZd+aYft626EQ0Kda\\\\\\\/GHEVQvhIbaG1V2nNm45C5yQCQx9Yytmla5MGIq98U+HDnhG4+MWQMXmY5LImdMmpcDLOACltABXMZoc7\\\\\\\/mNorsY9XM0eCY2SpQydkOn4GyQnYnOSmTt4Bii233nDHScamAA188RQsWnMc\\\\\\\/Q0ZB2YCOut\\\\\\\/fgboX4Hi7g+3KFeGMvGKDxXhuMTKnrSeXAuE9ApJUKCFtWoX5iRHjwMtlgIwE\\\\\\\/LI3\\\\\\\/cxh8mPJt9SPXMMX7kTewKQoO63nlRHuSY88Ue2I9QJkks2Ymc6Kp94jST1dqD2mySN6pcnEDjaDzfdtknno78lAYv41YXshaAEbyqsCpVaO\\\\\\\/EXckZ\\\\\\\/IU71oouEwh02ZSF\\\\\\\/Gc+hPzrSl4MHjf8S0rLAO0odpZm5yxE\\\\\\\/PQIQJ+R0yL5i+jCiQ21EEVJZV4LXHDsjee+IBz37+QyGzI6MpE4zxTCuKcwDqEC5wMN3RJTsWwXBd43CF\\\\\\\/NbOv26vx8rVwg8wqUFTEUJDwbQ74hCDrdqQhM0c421EXnRD2thvZkDsK6M+YdjbRcYx20JQVOf+eb5K9WY6eCUSBl3hYtGOpJmsneGpH\\\\\\\/Egzl5afgSKKpSyuEXgn8Fcg8lUw5TrkJKnbHkntpbwoQl1oX5IGFlCaDtRpoBqzCsPShC8BZr3PyJwDnv53Y2sezv+d84EvDXpx1XQT\\\\\\\/K5o4eyVUMbbqKcPuAaY5cvKD6dDsfjIoQOVVUR3bjneMSSq1hBbYfyLH8VX2xkYbujit6ZuL0\\\\\\\/SPejoA1pf4P3P0Acj9nThzXfr7lpsc2HT7\\\\\\\/dG6mqiNGQow1mvAhdGaDU90nwyJSoyOydlZ8hVYgvj2fUTCJSM50Va\\\\\\\/fHy1d7Kql9NPhl5ewPHvVtz4IdIOXooV7qYMsyK6PFYE5r3Yp0HseVPeJWSVcXLHFQpEun8fDn5QVB8tlXI4yeKXD1OlzeHx7LcyMtSeMwJ61T0mp7HfYvufW\\\\\\\/lpZKb1ExOGFHbjebiqts79fwqyfwsr5nsIjzClkEnK4wGwOTKc2rmex6dHfNjYHK1Th4Nat8tJ7fRTCuE1I2hGxAr\\\\\\\/0TESJoehNMgPu0r12gOLvm3ubBOErCXTLr0On6UL2QZ+ASU9t3QEoJyZthswZfMGSJjMeMLKfkAUj52EsXuMCuogsynhLJTgA8IZgn9LCeWul6Azrd3spN8oEYfDCZ33X92ItJVFmJyIIsNKU6DZooaNd3iqqtjvOcU4bddtKE47an\\\\\\\/l9DPNrhe+FNmtWeP8PQN05NkGJtKqk2mJd9fvdajMxF3OIoUdp68kQ2BwLb2slv6sQqrsCyuaT1wBK4HwXyztSzmrs2pj9OhzLw0Gdp0iEq7H\\\\\\\/RpOtF018iL26laED9dEPh2Pusc\\\\\\\/oyJjNsYRgI5iDuSmerhoxgq6mFLphWjPY212CSpFSteB6rg9g5DVJ6YxtjySCCYX\\\\\\\/vX2Ci\\\\\\\/wBXUUKpENMoJg05cFfbvUPsGK94Uqe0HPFQokpln1JXXbLDrFtC1fC1D5FQNHHrV0aKe3wWN5q7pOVz\\\\\\\/\\\\\\\/rkyJs0+7ZGNWuwAKgK1WJCt\\\\\\\/Tk7cEDwJvogBNMmEZORgcxV2Py\\\\\\\/ydyFlhTG4fypDaX2EfmUUegb8o63WzlpYa31XY1RQGQABQQx3W5ZJopVLwjsj1VK4hcSgq20etiNvzNgmycsIixD2RTK6XLJRRVL1\\\\\\\/0P50fJIETTq1bLsWeoCAkSqFS4wUI1AYE16A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ss\\\\\\\/SSZoMESdKq9YB9u3jjQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d5d42db4c96473d241d29426d2296c96e4f7d40c402e01f1006ae350b0638322\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bx7jMkqqDMq+rwQ1chAJsdf4B39OGZPnNTWt8uXDt8Erp\\\\\\\/O4v8QLDJwskAwQxiAd+9noweYHEl2rJ8ScgDNEniYWe7ofJiY6fahvI3OeVzSy+SYliKGkNiBZmLIuzzHn8CqJZ\\\\\\\/qUwAwqa4+CGweQeR\\\\\\\/w+Dv7NTOJ\\\\\\\/cuYOpxIZgKkRBDGNHsGPvzJnDgzGV3jet\\\\\\\/vhleJ6iPuvjRf6zuTXBWspAlsVFiC2jErpsnpKXmUQA5yrcNDeJ+j5r4gEpPrNC4GR75p4gAZ\\\\\\\/euVzbMGkYTGeGXPJCGIuJG\\\\\\\/n4Fl9igkFOQ8frdr2i2hqcBHUlyFnRqyIWwKEeFlWmC0ntFjYZzq8t1TlmYSCU7diC8b8vPqYavSaYLvvYCR2TqVForRvaYL4z1YOqXjSMdGgbLrEIKTTUR0qUnTM8J5hhwlau6C7v2VeoJdLH2uX7u7z04+cPyj87t0kjeh8jlmeSKQD52+UjXa3Spsk+I0JnTw10rbRJlMIl+uj5LPYSq56aCl\\\\\\\/\\\\\\\/4LULIjwLYGEiXiDSo5Ee3kBHgd\\\\\\\/yEA0rUUJFCNoCAVNJ5ldQx+7ZdXn47uyACydPq2hUU1MhHrwKoJ91z7TI89UkPymA4Yskppb6lut4SeGi8m1h7Xi0uimPsr4Y8x3AwwRyhTKW5rbyMwYLV63JeX4ce0oSlgIq6msUBW+BSGnkwfqLG5H5uJl5HSGvCUVdNGElL3JtDsUpKSSWP4KKiYEFkIfHTQBTebEDFtvD\\\\\\\/HFfN0n0KmDO1LVG5LAyq73aGvfnzhOTnHkv7l7B81LaZpFtNgSrdvEtxrGZbZxgcttv25fZDtRl4U3f1zm8YkeOiSnDez+LNALEqjN9VIco45zOjT6PDycO66mt+zvLCKNC4QSByn7fME8t8dX+48jRLSwkgUbjIHTc7raMHYU6ZdL+HA0lp+xlfEmyywkuAYFKQMzXRcnawRjNb2ti+vIvCBX5Mefi5COO3dLzRgQUVDM67raoim1ifOFpP382MoFr1XsJJCq+b+7VCREe68aK9Iweqy9Vx4X9w\\\\\\\/I85cgwQUkEKNVQZd433RcpMRtAd7ovdfGweiWhpobGsLkrSmCjNtH1XAmojtDPW9mSfwY3Toi+KxXtfkUH1KYs0YZcQROY6H77WJy14TzWirUxfxXloiwzF+Wah+NLrcgTM12xJLciikszn8hqjMlicbIIknmbIWaV2P81vDRFQnBgnO6M4VMGw1JvPledd0Qn5dnFu+0OTukgQe\\\\\\\/rTgbmtu+7I6HxzZolJ\\\\\\\/IJ7XHL1klcTXNxAeexzysvS6pIEgwuXP9HfwIxoN0CnyyuZzxMkxiaTctx2GT1PAiLPn9kL7CuaRPA27tDNOQxRagajA24H+fH3koMn6csqIfjxk03nFq4T5Z\\\\\\\/8zhPUVo5BFlqmVYgNG1f9QiRh4YAHowEOh3zS\\\\\\\/q3j8bsfNtFYclJbkWBiu9f\\\\\\\/yBiybNAVyWgRA5sHGaOIM32X1U\\\\\\\/Bfekd2WCO3Th4kPhetVyH+NPz4B\\\\\\\/Xfh6E8hpthV4gXyknBStS\\\\\\\/YdECLz3lqfy2lzkdOW9o7FUwiEJgs6EDYF4ke\\\\\\\/gTswsceLtWydM0y3Y9Tck4NillGIecMa1baqQERljboc3ZqIzQloEitsQu7elqs5hY3lkaaz1KLeyY\\\\\\\/T2hXART9sPU9X31lHzhM6OTIZ6Nn+\\\\\\\/kxZhIDGGGevFi5IgS0iVeK\\\\\\\/Nr6UaPU9S1nPBm+EaJRI66B0Dw8O4tSNFbcs4A9JAK8Zr7FCQwuyh4+YVr6ArqovkWlwRgxLL4Fnt+EuSdnVhLPaU\\\\\\\/0FBHuv\\\\\\\/I4fe0WuOXdpKMkdGo+0bwABvkxtY6AmZrQlzoL0DeMcZyMzOR7WxWnxWHM0r5ITZyfKN67eYbykVTu5wN+d1igJ4+vANu0iIeacX+1uLBeyEbZG1tsUhDj7VAXCW87BxOYnr8Rrz37GH1ZiX95KzHeR4NHTvR7ymLVmaVDkRjkcSYtjXP0pp06qakvHoX5jzNynzxHsRVFRdFmb1mVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FOtRQDcn8iSuJ0l4lK\\\\\\\/zbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223e50ea7df8f48cd8688707f53d9351e2154abed4c7cee08dc474823a3c3089b4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489121 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1129" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2ZI06j6NDjzt0wHBT3-ftc8WafchKRw-1bthsoRpKf4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1129" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PpAKwlav6MdxrRyQhFDPjHvTE7hzysOHbPcm_t9ktiU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1119" title="A Metagenome-Wide Association Study and Arrayed Mutant Library Confirm Acetobacter Lipopolysaccharide Genes Are Necessary for Association with Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1139" title="Maize Transposable Elements Ac/Ds as Insertion Mutagenesis Tools in Candida albicans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489121" id="node489121" data-pisa="ggg;8/4/1129" data-pisa-master="ggg;g3.117.300468" data-apath="/ggg/8/4/1129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3356-7869" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean Peccoud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3140-0651" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sébastian Lequime</span>, <span class="highwire-citation-author" data-delta="2">Isabelle Moltini-Conclois</span>, <span class="highwire-citation-author" data-delta="3">Isabelle Giraud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5958-2138" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Louis Lambrechts</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2131-7467" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clément Gilbert</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1129-1138;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300468 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.300468" data-atom-uri="/ggg/8/4/1129.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489121" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jean Peccoud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BPeccoud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peccoud%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BPeccoud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3356-7869" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean Peccoud</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sébastian Lequime </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastian%2BLequime%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lequime%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS%25C3%25A9bastian%2BLequime%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3140-0651" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sébastian Lequime</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Isabelle Moltini-Conclois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BMoltini-Conclois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moltini-Conclois%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BMoltini-Conclois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Isabelle Giraud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BGiraud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giraud%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BGiraud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Louis Lambrechts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BLambrechts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lambrechts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouis%2BLambrechts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5958-2138" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louis Lambrechts</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Clément Gilbert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Laboratoire Evolution, Génomes, Comportement, Écologie, UMR 9191 CNRS, UMR 247 IRD, Université Paris-Sud, 91198 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cl%C3%A9ment%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACl%25C3%25A9ment%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2131-7467" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clément Gilbert</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/4/1129" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489121" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489121/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1129.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489121" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489121/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1129.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489121" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489121/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1129.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489121" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489121/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/4/1129.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489121" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489121/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Chimeric reads can be generated by <em>in vitro</em> recombination during the preparation of high-throughput sequencing libraries. Our attempt to detect biological recombination between the genomes of dengue virus (DENV; +ssRNA genome) and its mosquito host using the Illumina Nextera sequencing library preparation kit revealed that most, if not all, detected host–virus chimeras were artificial. Indeed, these chimeras were not more frequent than with control RNA from another species (a pillbug), which was never in contact with DENV RNA prior to the library preparation. The proportion of chimera types merely reflected those of the three species among sequencing reads. Chimeras were frequently characterized by the presence of 1-20 bp microhomology between recombining fragments. Within-species chimeras mostly involved fragments in opposite orientations and located less than 100 bp from each other in the parental genome. We found similar features in published datasets using two other viruses: Ebola virus (EBOV; -ssRNA genome) and a herpesvirus (dsDNA genome), both produced with the Illumina Nextera protocol. These canonical features suggest that artificial chimeras are generated by intra-molecular template switching of the DNA polymerase during the PCR step of the Nextera protocol. Finally, a published Illumina dataset using the Flock House virus (FHV; +ssRNA genome) generated with a protocol preventing artificial recombination revealed the presence of 1-10 bp microhomology motifs in FHV–FHV chimeras, but very few recombining fragments were in opposite orientations. Our analysis uncovered sequence features characterizing recombination breakpoints in short-read sequencing datasets, which can be helpful to evaluate the presence and extent of artificial recombination.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/artificial-chimeras" class="hw-term hw-article-keyword hw-article-keyword-artificial-chimeras" rel="nofollow">artificial chimeras</a></li><li class="kwd"><a href="/keyword/high-throughput-sequencing" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-sequencing" rel="nofollow">high-throughput sequencing</a></li><li class="kwd"><a href="/keyword/illumina" class="hw-term hw-article-keyword hw-article-keyword-illumina" rel="nofollow">Illumina</a></li><li class="kwd"><a href="/keyword/virus" class="hw-term hw-article-keyword hw-article-keyword-virus" rel="nofollow">virus</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li></ul><p id="p-3">Recombination is an important process shaping virus biology and evolution, as it may trigger changes in viral host ranges, virulence and tissue tropism, as well as generate entirely new viruses (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Krupovic <em>et al.</em>, 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Martin <em>et al.</em>, 2011</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pérez-Losada <em>et al.</em>, 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Simon-Loriere and Holmes 2011</a>). In addition to recombining with other viral genomes, viruses may also recombine with the genome of their host, either as an obligate step of their replication cycle, or accidentally (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Weiss 2017</a>). Such recombination events may lead to the integration of viral genomes into host genomes, forming endogenous viral elements (EVEs) with varying consequences on host–virus interactions and host biology in general (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Feschotte and Gilbert 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katzourakis and Gifford 2010</a>). Reciprocally, virus–host recombination may also result in the integration of a piece of the host genome into a viral genome, supporting the idea that viruses may be efficient vectors of horizontal transfer of genetic material between species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gilbert and Cordaux 2017</a>).</p><p id="p-4">The present study was initially designed to survey virus–host recombination events between dengue virus (DENV) and <em>Aedes albopictus</em> mosquito cells by high-throughput (Illumina) sequencing. Dengue viruses are positive, single-stranded RNA (+ssRNA) viruses belonging to the <em>Flaviviridae</em> family (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Guzman and Harris 2015</a>). These arthropod-borne viruses are transmitted among humans by mosquitoes of the <em>Aedes</em> genus. DENV replication takes place in the cytoplasm of both human and mosquito cells and involves the production of a double-stranded RNA intermediate generated by the virus-encoded RNA-dependent RNA-polymerase (RdRp) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bartenschlager and Miller 2008</a>). The frequent finding of defective DENV genomes resulting from virus–virus non-homologous recombination in DENV-infected patients (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em>, 2011</a>) led us to hypothesize that DENV may also recombine at an appreciable frequency with RNAs of their host (human or mosquito).</p><p id="p-5">Rather than biological chimeras, our analysis detected a large number of artificial chimeras. Two of the enzymes involved in protocols of sequencing library preparation are known to generate such chimeras. Reverse transcriptases used to generate cDNA libraries are known to be prone to both inter- and intra-molecular template switching (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cocquet <em>et al.</em>, 2006</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zeng and Wang 2002</a>). DNA polymerases used in some library preparation protocols can also generate <em>in vitro</em> recombination, via pausing on an incompletely elongated strand or premature termination, and reannealing of the strand to another DNA fragment in the following PCR cycle (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Meyerhans <em>et al.</em>, 1990</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pääbo <em>et al.</em>, 1990</a>). Such biases, as well as their associated problems in many areas of research, have long been recognized and various protocol optimizations and controls have been proposed to take them into account (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Di Giallonardo <em>et al.</em>, 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Edgar <em>et al.</em>, 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Görzer <em>et al.</em>, 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Haas <em>et al.</em>, 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Routh <em>et al.</em>, 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Shao <em>et al.</em>, 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zanini <em>et al.</em>, 2017</a>). In this study, we were not able to observe any convincing evidence of biological recombination between DENV and mosquito RNA, suggesting that it is unlikely to be frequent. However, using appropriate controls, we investigated the nature of artificial chimeras detected in our sequencing libraries. To generalize our findings, we also analyzed multiple published sequence datasets generated with similar sequencing protocols. Together, our results uncover sequence features typical of artificial chimeras, which may help detect and characterize such artifacts in future studies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and methods</h2><div id="sec-2" class="subsection"><h3>Virus and cell lines</h3><p id="p-6">C6/36 (<em>Aedes albopictus</em>) mosquito cells were maintained at 28° in Leibovitz’s L-15 medium (Life Technologies) supplemented with 10% fetal bovine serum (Life Technologies), 1% non-essential amino acids (Life Technologies) and 0.1% Penicillin-Streptomycin (Life Technologies) as previously described (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fontaine <em>et al.</em>, 2016</a>). A sub-confluent culture of C6/36 cells grown in a 75-cm<sup>2</sup> flask was inoculated with DENV type 1 isolate KDH0026A (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fansiri <em>et al.</em>, 2013</a>) at a multiplicity of infection of 0.01 as previously described (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fontaine <em>et al.</em>, 2016</a>). The cell culture supernatant was harvested after 5 days of incubation at 28°.</p></div><div id="sec-3" class="subsection"><h3>Library preparation</h3><p id="p-7">Total RNA was extracted from the supernatant of the DENV-infected C6/36 cell culture using TRIzol (Life Technologies). Total RNA was treated with Turbo DNAse (Life Technologies) and purified with magnetic beads (Beckman Coulter). Several studies have shown that artificial recombination events can occur at elevated rates during the RT-PCR and/or PCR steps involved in the preparation of sequencing libraries. To subsequently estimate the amount of such technical chimeras, mosquito + DENV RNA was mixed with purified RNA from the common pillbug <em>Armadillidium vulgare</em>. Given that the analyzed pillbug and DENV have never been in contact before the mix, any junction identified between their RNAs can only be artificial. Pillbug RNA was used because we had easy access to this material and because pillbugs and mosquitoes diverged ∼500 million years ago, ensuring good confidence in the assignment of chimeric sequencing reads to one or the other species. The mix was performed to target a 1:1 ratio in the concentrations of the two RNA samples, following RNA quantification by fluorometry (Qubit RNA HS, Life Technologies).</p><p id="p-8">The RNA mix was reverse transcribed in duplicates using Transcriptor High Fidelity cDNA Synthesis Kit (Roche Applied Science) and random hexamers. The second strand was synthetized in a single reaction with <em>E. coli</em> DNA ligase (New England Biolabs [NEB]), <em>E. coli</em> DNA polymerase I (NEB), <em>E. coli</em> RNAse H (NEB) in second-strand synthesis buffer (NEB). The newly synthesized dsDNA was purified with magnetic beads (Agencourt AMPure XP, Beckman Coulter) and its concentration was measured by fluorometric quantification (Quant-iT PicoGreen dsDNA, Invitrogen).</p><p id="p-9">A sequencing library was prepared in duplicate using Nextera XT DNA Library Preparation Kit (Illumina) following the manufacturer’s instructions. Duplicate libraries were multiplexed with other libraries and sequenced in single end on an Illumina NextSequation 500 platform using a mid-output 150 cycles v2 kit (Illumina). Sequencing reads were demultiplexed using bcl2fastq v2.15.0 (Illumina).</p></div><div id="sec-4" class="subsection"><h3>Detection of chimeric reads</h3><p id="p-10">Detection of chimeric reads resulting from either host–virus or virus–virus recombination used an approach previously developed to search for moth sequences integrated into genomes of the AcMNPV baculovirus (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gilbert <em>et al.</em>, 2016</a>). Briefly, this approach involves blastn (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Camacho <em>et al.</em>, 2009</a>) homology searches of the sequencing reads against reference genomes and/or transcriptomes. Searches were performed separately with the DENV genome (Genbank accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HG316481&amp;atom=%2Fggg%2F8%2F4%2F1129.atom">HG316481</a>), with the <em>Ae</em>. <em>albopictus</em> genome (MNAF02000000) and with the <em>Ar. vulgare</em> genome (LYUU01000000), allowing for the recovery of two possible hits per read in each blast (option –max_target_seqs 2).</p><p id="p-11">The following steps were performed with custom R (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">R Core Team 2017</a>) scripts. All reads containing a region of at least 20 bp that aligns multiple times to one or more reference genomes were identified. Among those alignments, only the one with the best blast score was kept. A read was then identified as chimeric when two portions align to two genomes or two non-adjacent regions of the same genome (in the following only the former case is considered for simplicity, but the principles apply to both). Each region of such read must align over at least 28 bp, and over at least 16 bp to one genome <em>only</em>. The read must be aligned to both genomes over at least 90% of its full length. Aligned read regions are allowed to overlap by up to 20 bp or to be separated by at most 5 bp. The overlap corresponds to a microhomology between both genomes at the recombination point, and the separation should correspond to non-templated addition of nucleotides at this point. These criteria were based on the distribution of real data in <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gilbert <em>et al.</em> (2016)</a>. Reanalysis of the datasets published by <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015)</a> (see below) yielded chimeric read counts similar to those obtained by these authors despite the use of different methods. Importantly, the approach designed by <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Routh and Johnson (2014)</a> and <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015)</a> does not allow the identification of overlapping alignments in the chimeric reads. It first uses a read mapper [Bowtie2 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langmead and Salzberg 2012</a>)] to find the best alignment between a chimeric read and a reference genome, cuts the remaining unaligned portion of the chimeric read and searches for the best alignment between this second read portion and the same or another reference genome. Hence, the second alignment cannot include a portion of the read involved in the alignment of the first read portion, thereby precluding recovery of possible overlaps between alignments of the two read portions.</p><p id="p-12">To check whether microhomology lengths at recombination points were consistent with those expected by chance, considering the sequences of the analyzed genomes, the distribution of homology length was compared to that of random chimeric reads generated <em>in silico</em>. Each <em>in silico</em> read was made of two regions extracted from random locations of a pair of genomes (DENV, EBOV, herpesvirus, FHV or pillbug RNA genes). The lengths of the two regions were chosen at random, with the conditions that both were at least 28 bp and their sum was the size of a read (150 bp). These reads were then blasted against the sequences from which they were generated and the blast outputs were submitted to the same analysis as that performed on real data.</p><p id="p-13">Chimeric reads potentially resulting from PCR duplicates generated during the preparation of the Illumina sequencing library were identified as reads having the same alignment coordinates in the two genome sequences they involve. Only one read among duplicates was retained. This selection leads to conservative estimates of the number of chimeras, at it is not excluded that apparent PCR duplicates result from sequencing the exact same portion (same start and end position) of distinct RNA molecules generated by non-PCR (viral) replication of a host–virus or virus–virus recombination event. The probability of sequencing the same original biological chimera at the same position increases with sequencing depth.</p></div><div id="sec-5" class="subsection"><h3>Annotation of transposable elements and ribosomal RNA genes</h3><p id="p-14">Many transposable elements (TEs) from a host (moth) genome have been shown to transpose into the genome of a DNA virus (the AcMNPV) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fraser <em>et al.</em>, 1983</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gilbert <em>et al.</em>, 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Miller and Miller 1982</a>). TEs indeed represented virtually all the moth sequences that were linked to AcMNPV sequences and they were inserted at their preferred target duplication sites in the virus genome (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Gilbert <em>et al.</em>, 2016</a>). Here, it was not expected to find signs of <em>bona fide</em> transposition of mosquito TEs into DENV genomes because TEs transpose into DNA, not into RNA, and the DENV replication cycle does not normally involve reverse transcription of the DENV genome into DNA. Yet, it has been shown that upon infection of flies and mosquitoes, a number of RNA viruses including DENV produce viral DNA (vDNA) that is transcribed and involved in small-RNA-mediated antiviral immunity (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Goic <em>et al.</em>, 2013</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2016</a>). Importantly, these vDNAs are generated by template switching of reverse transcriptases encoded by host retroelements. As a result, they are frequently flanked by retrotransposon sequences. It is possible that transcription of vDNAs could initiate or terminate in their upstream or downstream TE flanking sequence, which could produce TE-virus chimeric transcripts. In this context, the production of vDNAs could increase the likelihood of TEs, compared to non-TE sequences, to be joined to viral genome fragments and to be encapsidated into virions. To assess whether mosquito–DENV chimeric reads involve mosquito TEs, the alignment positions of chimeric reads were located with respect to TE copies that were annotated by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Peccoud <em>et al.</em> (2017)</a>.</p><p id="p-15">Ribosomal RNA genes were also annotated in the mosquito and pillbug genomes. For this, we used a full-length copy of <em>Ae. albopictus</em> ribosomal RNA genes (accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=L22060&amp;atom=%2Fggg%2F8%2F4%2F1129.atom">L22060</a>) and a set of Isopoda partial RNA genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300468/-/DC1/FigureS1.pdf">Supplementary Table 1</a>) as queries to perform blastn searches with default parameters on the genome of the two species. No annotated full-length copy of all ribosomal RNA genes is available for <em>Ar. vulgare</em>.</p></div><div id="sec-6" class="subsection"><h3>Analysis of other types of chimeras</h3><p id="p-16">To better understand the process of chimera formation, other types of chimeric reads were analyzed. Because DENV–DENV chimeric reads involve the same, very short, genome, the following properties could be assessed: (i) the distance between recombining sequences in the source genome, relative to the distance distribution between two randomly drawn positions in this genome, (ii) the number of breakpoints detected in 100-bp windows sliding along the DENV genome, and (iii) the orientation of the two recombining sequences (same or different orientations) in each chimera. Pillbug–pillbug chimeras were investigated in the same fashion. This analysis focused on chimeras involving two sequences of 18S ribosomal pillbug RNA, which composed a large number of pillbug–pillbug chimeras. Ribosomal 18S RNA chimeras were identified by blasting all reads on an 18S rRNA sequence of <em>Ar. vulgare</em> (Genbank accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AJ287061.1&amp;atom=%2Fggg%2F8%2F4%2F1129.atom">AJ287061.1</a>).</p><p id="p-17">The presence and properties of chimeras found in other datasets was examined in a published high-throughput sequencing dataset of EBOV produced with the same library preparation protocol as in the present study (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gire <em>et al.</em>, 2014</a>). Unlike DENV, EBOV (<em>Mononegavirales</em>, <em>Filoviridae</em>) is a -ssRNA virus. To detect EBOV–EBOV chimeras, all reads were aligned to the Makona-G3686 EBOV genome (accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KM034562&amp;atom=%2Fggg%2F8%2F4%2F1129.atom">KM034562</a>) with blastn, followed by the same analysis of blast results described above. Whether chimeras could be formed during the synthesis of cDNA or during the PCR step of the Nextera library-preparation protocol was assessed by analyzing in the same fashion a published Illumina dataset produced to sequence the genome of a DNA virus, the Macropodid herpesvirus 1 (MaHV-1) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Vaz <em>et al.</em>, 2016</a>).</p><p id="p-18">Finally, a high-throughput sequencing dataset of FHV published by <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015)</a>, which is devoid of artificial chimeras, was reanalyzed. This dataset allowed the characterization of properties of biological recombination events in an RNA virus, in particular the microhomology length between recombining sequences, using the same protocol applied to the other datasets. In their study, <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015)</a> used click chemistry rather than enzymatic reactions for the ligation of Illumina adaptors. The method generates unbiased libraries, with an extremely low artificial recombination rate. FHV is a +ssRNA virus belonging to the <em>Alphanodavirus</em> genus in the <em>Nodaviridae</em> family. It encapsidates a bipartite genome, composed of RNA1 (3,107 bp), which encodes the viral RdRp, and RNA2 (1,400 bp), which encodes the viral capsid protein.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-19">Raw sequence reads data from DENV, <em>Ae. albopictus</em> and <em>Ar. vulgare</em> are available in fastq format at the NCBI short read archive under accession number SRP129541. The R codes used for the data analysis are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300468/-/DC1/FileS1.docx">File S1</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Biological recombination Between DENV and host RNA is likely infrequent</h3><p id="p-20">Alignments of the reads to the DENV genome and to the <em>Ae. albopictus</em> mosquito and <em>Ar. vulgare</em> pillbug genomes yielded similar numbers of aligned reads for the two technical replicates that we performed (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The ratio of DENV + mosquito reads over pillbug reads were 2.6 (replicate 1) and 3.4 (replicate 2). The average sequencing depth of the DENV genome was 26,423X (replicate 1) and 28,594X (replicate 2). Hereafter, we refer to total numbers of reads obtained in the two replicates.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489209/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489209/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of chimeric reads detected in all datasets analyzed in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">We identified 3,717 chimeric reads containing a junction between pillbug and DENV RNA. These reads represent 3,639 breakpoints that all differ from each other by their locations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and which must have been created <em>in vitro</em> since pillbug and DENV RNAs have not been in contact <em>in vivo</em>. The observed proportion of pillbug–DENV chimeric reads involving pillbug ribosomal RNA (73%) was not significantly different from the proportion of all pillbug reads assigned to rRNA (Chi<sup>2</sup> <em>p-value</em> = 0.4; 1 df). This indicates that the probability for pillbug sequences to recombine <em>in vitro</em> with DENV sequences mainly depends on their relative proportions in the RNA extracts. The proportion of pillbug and mosquito reads aligning to rRNA genes (74% and 77%, respectively) is consistent with previous RNAseq datasets produced without rRNA depletion (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">O’Neil <em>et al.</em>, 2013</a>).</p><p id="p-23">We detected 566 mosquito–DENV chimeric reads, representing 555 unique chimeras. None of the recombination breakpoints we sequenced were found to involve a mosquito TE. Instead, most of the chimeras (74%) contain mosquito rRNA, a proportion that is not significantly different from that observed among all mosquito reads (Chi<sup>2</sup> <em>p-value</em> = 0.4; 1 df). It is noteworthy that the number of mosquito–DENV chimeras is lower than expected given the number of pillbug–DENV chimeras and the proportion of reads from each species in the whole dataset (Chi<sup>2</sup> <em>p-value</em> &lt; 0.001; 1 df). The reason underlying this discrepancy is unclear and we could not find any evidence suggesting it may be biological. On the opposite, the lower proportion and similar distribution of microhomology length of mosquito-DENV chimeras compared to artificial pillbug-DENV chimeras (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), indicate that most, if not all, mosquito-DENV chimeras were artificially produced.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of lengths of microhomology motifs found in mosquito&#x2013;DENV (A) and pillbug&#x2013;DENV (B) breakpoints. The black/gray and red histograms show the observed and expected distributions, respectively. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. This was needed for comparison to the expected distribution, which does not account for negative microhomology." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-320229543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of lengths of microhomology motifs found in mosquito&#x2013;DENV (A) and pillbug&#x2013;DENV (B) breakpoints. The black/gray and red histograms show the observed and expected distributions, respectively. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. This was needed for comparison to the expected distribution, which does not account for negative microhomology.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1129/F1.medium.gif" width="293" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/4/1129/F1.medium.gif" width="293" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489175" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Distribution of lengths of microhomology motifs found in mosquito–DENV (A) and pillbug–DENV (B) breakpoints. The black/gray and red histograms show the observed and expected distributions, respectively. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. This was needed for comparison to the expected distribution, which does not account for negative microhomology.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Interestingly, the majority of the pillbug–DENV and mosquito–DENV junctions (56% and 57%, respectively) are characterized by the presence of 1-20 bp microhomologies (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The distributions of microhomology lengths for both types of chimeras are very similar to each other and at the same time clearly higher than those expected by chance. This shows that linked sequences tend to be similar at their recombination point. Still, a substantial proportion of pillbug–DENV (26.7%) and mosquito–DENV (28%) chimeras devoid of microhomologies are characterized by the presence of 1-5 nucleotides that did not derive from any of the two recombining sequences. The presence of these additional nucleotides is likely due to the known ability of DNA polymerases to synthesize DNA in a non-template directed manner (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Clark 1988</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">García <em>et al.</em>, 2004</a>).</p></div><div id="sec-10" class="subsection"><h3>Canonical features of chimeras found in various Illumina Nextera sequencing datasets</h3><p id="p-26">DENV–DENV chimeric reads were far more numerous than arthropod–DENV ones (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), which can be explained by the high proportion (70%) of DENV reads in the datasets. DENV–DENV breakpoints are also characterized by the presence of 1–20 bp microhomologies and the microhomology length distribution clearly differs from that expected by chance (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Most DENV–DENV recombination events involved sequences separated by distances that are much shorter than expected if recombination took place randomly along the DENV genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Plotting the density of recombination breakpoints along the DENV genome (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) did not reveal any particular hotspots, indicating that recombination takes place along the entire length of the DENV genome at similar frequencies. Interestingly, the vast majority of DENV–DENV chimeras (60,781 out of 81,516, ∼74.5%) involve sequences in opposite orientation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characteristics of DENV&#x2013;DENV breakpoints. (A) Distribution of lengths of microhomology motifs, as in Figure 1. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. (B) Density of observed (black/gray) and expected (red) distances separating recombining fragments found in DENV&#x2013;DENV himeras. (C) Number of DENV&#x2013;DENV breakpoints per 100-bp, non-overlapping windows along the DENV genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-320229543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characteristics of DENV&#x2013;DENV breakpoints. (A) Distribution of lengths of microhomology motifs, as in Figure 1. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. (B) Density of observed (black/gray) and expected (red) distances separating recombining fragments found in DENV&#x2013;DENV himeras. (C) Number of DENV&#x2013;DENV breakpoints per 100-bp, non-overlapping windows along the DENV genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1129/F2.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/4/1129/F2.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489264" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Characteristics of DENV–DENV breakpoints. (A) Distribution of lengths of microhomology motifs, as in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. (B) Density of observed (black/gray) and expected (red) distances separating recombining fragments found in DENV–DENV himeras. (C) Number of DENV–DENV breakpoints per 100-bp, non-overlapping windows along the DENV genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We detected 8,170 unique intra-pillbug ribosomal RNA breakpoints that are characterized by the same features as DENV–DENV breakpoints, <em>i.e.</em>, most of them (81%) show 1–20 bp microhomology as well as evidence of non-templated nucleotide addition and breakpoints devoid of microhomology (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300468/-/DC1/FigureS1.pdf">Supplementary Figure 1</a>), recombination between fragments in opposite orientation (84.4% of the chimeras) that are typically distant by a few dozen nucleotides.</p><p id="p-29">By analyzing the EBOV dataset of (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gire <em>et al.</em>, 2014</a>), we found 144,556 chimeric reads representing 16,504 different EBOV–EBOV breakpoints, which were characterized by the same features as DENV–DENV or pillbug–pillbug chimeras (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300468/-/DC1/FigureS1.pdf">Supplementary Figure 2</a>). The length of microhomology between recombining sequences was longer than expected by chance, most recombining sequences were distant from less than 100bp and were in opposite orientations (65% and 92%, respectively).</p><p id="p-30">Finally, our analysis of the herpesvirus dataset of <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Vaz <em>et al.</em> (2016)</a> detected 2,507 MaHV-1–MaHV-1 chimeric reads representing 1,139 different breakpoints. These were characterized by features similar to those of RNA breakpoints (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300468/-/DC1/FigureS1.pdf">Supplementary Figure 3</a>): 95% show 1–20 bp microhomology and recombination occurs predominantly (at 93%) between nearby fragments in opposite orientations.</p></div><div id="sec-11" class="subsection"><h3>Features of natural recombination Between viral sequences</h3><p id="p-31">In the FHV dataset of <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015)</a>, we found a total of 744,201 FHV-RNA1–FHV-RNA1 and 45,155 FHV-RNA2–FHV-RNA2 chimeric reads representing respectively 3,675 and 1,313 unique recombination breakpoints. We note that in spite of the differences between our approach and that of <a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015)</a> in terms of alignment method (mapping <em>vs.</em> blast) and criteria to detect chimeric reads, our numbers are similar to those obtained in the original study (766,143 FHV-RNA1–FHV-RNA1 and 39,856 FHV-RNA2–FHV-RNA2; <a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> 2015</a>). Among these, very few breakpoints involved genome regions in opposite orientation (73 out of 4,988, ∼1.5%), indicating that recombination occurred via a mechanism different from that underlying DENV–DENV, EBOV–EBOV and MaHV-1–MaHV-1 chimeras. Interestingly, we found that the majority (64%) of FHV–FHV breakpoints were characterized by the presence of 1-20 bp microhomologies, with the bulk (89%) of these microhomologies being 1-7 bp (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The distribution of distances between sequences involved in chimeras was wider than those previously described, but still narrower than that expected by chance (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characteristics of FHV&#x2013;FHV breakpoints. (A) Distribution of lengths of microhomology motifs as in Figure 1. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. (B) Density of observed (black/gray) and expected (red) distances separating recombining fragments found in FHV&#x2013;FHV chimeras. The number of FHV&#x2013;FHV breakpoints per 100-bp, non-overlapping windows along the FHV genome is shown in (C) for the FHV RNA1 and (D) for the FHV RNA2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-320229543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characteristics of FHV&#x2013;FHV breakpoints. (A) Distribution of lengths of microhomology motifs as in Figure 1. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. (B) Density of observed (black/gray) and expected (red) distances separating recombining fragments found in FHV&#x2013;FHV chimeras. The number of FHV&#x2013;FHV breakpoints per 100-bp, non-overlapping windows along the FHV genome is shown in (C) for the FHV RNA1 and (D) for the FHV RNA2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1129/F3.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/4/1129/F3.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489318" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Characteristics of FHV–FHV breakpoints. (A) Distribution of lengths of microhomology motifs as in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>. Negative lengths (gray bars) represent insertion of non-templated nucleotides at junction points. Frequencies of observed microhomologies were rescaled so that the heights of black bars sum to 1. (B) Density of observed (black/gray) and expected (red) distances separating recombining fragments found in FHV–FHV chimeras. The number of FHV–FHV breakpoints per 100-bp, non-overlapping windows along the FHV genome is shown in (C) for the FHV RNA1 and (D) for the FHV RNA2.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Formation of artificial chimeras During PCR</h3><p id="p-33">We evaluated the biological origin of host-virus (mosquito-DENV) recombination by comparing its rate to that of recombination between genomes (pillbug-DENV) that never were in contact in a cellular context. This control indicated that the DENV genome and the genome of its mosquito host did not recombine <em>in vivo</em> to a detectable extent, hence, that most, if not all, virus-host chimeras were artificially produced.</p><p id="p-34">We also uncovered within-genome chimeras (<em>e.g.</em>, DENV-DENV chimeras) in greater numbers. To assess their biological origin, chimeras involving the genomes of species that were not in contact prior to DNA processing (<em>e.g.</em>, pillbug-DENV chimeras) cannot be used as a baseline expectation because they necessarily occurred between different molecules, contrary to within genome chimeras, which most frequently occurred within the same molecule (see below). Thus, they are not informative. However, it is notable that the various within-genome chimeras we detected in datasets obtained with the Illumina Nextera library preparation kit have similar features, whether they involve RNA or DNA templates from viruses or other source material (<em>e.g.</em>, pillbug): (i) recombining fragments present comparable microhomology length distributions at their junction points, (ii) junction sites appear evenly distributed along molecules, (iii) distances between recombining regions of are mostly shorter than 100 pb, and (iv) chimeric reads mostly link sequences in opposite orientations. These features were not shared by FHV chimeras sequenced by the ClickSeq technology (<a id="xref-ref-48-9" class="xref-bibr" href="#ref-48">Routh <em>et al.</em>, 2015</a>), whose biological origin is established. Hence, the sequencing technology, rather than differences between analyzed organisms, seems to explain features of chimeric reads. We therefore conservatively interpret Nextera-sequenced chimeras as produced during the processing of genetic material for sequencing.</p><p id="p-35">Because sequencing of the DNA virus MaHV-1 did not require cDNA synthesis by reverse transcription, recombination events detected in this dataset must have taken place during the PCR step of the Nextera library preparation protocol. Short recombination distances indicate that most breakpoints are generated by intra-molecular, rather than inter-molecular, recombination between close positions. Indeed, there is no reason for nearby sequences to preferentially recombine if these belonged to different molecules, considering the absence of recombination hotspots along genomes. This observation also excludes chimera production during the sequencing reaction. Such error may have arisen if the fluorescence signals generated by sequencing two different DNA fragments were somehow swapped in the course of the run. Such a reading error has no reason to preferentially involve fragments coming from nearby locations in the source genome and in opposite orientation. As for chimeras formed during the bridge PCR generating a DNA cluster on the flow cell, these are expected to generate garbled reads, not chimeric reads.</p><p id="p-36">Various models, most of which are experimentally verified, have been proposed to explain the formation of chimeras during PCR (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kanagawa 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Shammas <em>et al.</em>, 2001</a>). These include several types of template-switching mechanisms (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Guieysse <em>et al.</em>, 1995</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Odelberg <em>et al.</em>, 1995</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Patel <em>et al.</em>, 1996</a>). The majority of the chimeric reads detected in our study show the very features expected to be produced by template switching during extension of a primer along a double-stranded (or partially double-stranded) template (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Guieysse <em>et al.</em>, 1995</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Patel <em>et al.</em>, 1996</a>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In this configuration, strand displacement is catalyzed by the polymerase. Primer extension may proceed poorly as the displaced strand tends to reanneal to the template, inducing dissociation of the extending strand. The dissociated extending strand may then reanneal to the displaced strand at a position downstream of the primer binding site (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Patel <em>et al.</em>, 1996</a>). Reannealing with the displaced strand can explain why the two fragments composing a chimera tend to have opposite orientations. Our data also show that chimeras involve an excess of microhomology of a few nucleotides. This suggests that the 3′ end of the extending strand often reanneals over a few complementary bases on the displaced stand. Consistent with this deduction, template switching was shown to be most efficient when annealing occurred over 7-9 bp, in some PCR conditions (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Guieysse <em>et al.</em>, 1995</a>). As few as three adjacent complementary base pairs appear sufficient to favor template switching during elongation, since microhomologies of 3 pb and longer are more frequent than expected (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figures 1</a>-<a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>), but we cannot evaluate whether shorter lengths (1-2 pb) promote template switching.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model for the formation of artificial chimeras during the PCR step of the Illumina Nextera library preparation kit. Artificial chimeras most likely take place by intra-molecular template switching (TS) during extension of a primer along a double-stranded (or partially double-stranded) template. The dissociated extending strand reanneals to the displaced strand at a position upstream in respect to the direction of extension. Such reannealing is facilitated by base pairing over 1-20 bp (here, 3 bp), i.e., the microhomology motifs we detect in our analyses of recombination breakpoints. Recombining sequences are preferentially located at short distances from each other (" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-320229543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Model for the formation of artificial chimeras during the PCR step of the Illumina Nextera library preparation kit. Artificial chimeras most likely take place by intra-molecular template switching (TS) during extension of a primer along a double-stranded (or partially double-stranded) template. The dissociated extending strand reanneals to the displaced strand at a position upstream in respect to the direction of extension. Such reannealing is facilitated by base pairing over 1-20 bp (here, 3 bp), &lt;em&gt;i.e.&lt;/em&gt;, the microhomology motifs we detect in our analyses of recombination breakpoints. Recombining sequences are preferentially located at short distances from each other (&lt;100 bp). The bottom schematic shows how a chimeric read sequenced from a recombining fragment (top) would align on the source genome sequence (bottom). Dashed lines connect start and end alignment coordinates on the read to their counterparts on the genome sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1129/F4.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/4/1129/F4.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1129/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489164" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Model for the formation of artificial chimeras during the PCR step of the Illumina Nextera library preparation kit. Artificial chimeras most likely take place by intra-molecular template switching (TS) during extension of a primer along a double-stranded (or partially double-stranded) template. The dissociated extending strand reanneals to the displaced strand at a position upstream in respect to the direction of extension. Such reannealing is facilitated by base pairing over 1-20 bp (here, 3 bp), <em>i.e.</em>, the microhomology motifs we detect in our analyses of recombination breakpoints. Recombining sequences are preferentially located at short distances from each other (&lt;100 bp). The bottom schematic shows how a chimeric read sequenced from a recombining fragment (top) would align on the source genome sequence (bottom). Dashed lines connect start and end alignment coordinates on the read to their counterparts on the genome sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">We could not find any published experimental work that would help explain why recombining sequences are preferentially located at short distances from each other (&lt;100 bp). We suspect this pattern of preferred distance might be due to steric constraints inherent to the polymerase and/or to other PCR conditions, including the size of the template. Finally, many of the chimeras do not show the features expected to result from the intra-molecular template-switching mechanism characterized by <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Patel <em>et al.</em> (1996)</a>, including those showing no microhomology, non-templated addition of nucleotides or involving different source genomes, hence different molecules (like arthropod–DENV chimeras). While many of these chimeras may have been generated through other, unknown, <em>in vitro</em> recombination mechanisms, the possibility remains that some of them are biological chimeras. Nevertheless, the robust detection and analysis of biological recombination in RNA viruses clearly requires new library preparation protocols that suppress or drastically reduce the rate of artificial recombination, such as the one developed by <a id="xref-ref-48-10" class="xref-bibr" href="#ref-48">Routh <em>et al.</em> (2015</a>) and <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jaworski and Routh (2018)</a>.</p><p id="p-39">Remarkably, the features characterizing artificial chimeras produced with the PCR-based Illumina Nextera library preparation protocol are very similar to those generated by the non-PCR-based, Multiple Displacement Amplification (MDA) protocol routinely used for whole-genome amplification (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lasken and Stockwell 2007</a>). An analysis of 475 artificial chimeras generated by MDA followed by 454 (Life Sciences) sequencing revealed a majority of fragments in opposite orientation (85%) and the presence of 2-21 bp microhomologies in 70% of all MDA chimeras (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lasken and Stockwell 2007</a>). The MDA differs from the PCR in several aspects, such as the use of random hexamer primers, the absence of cycles of varying temperatures and the production of branched, long DNA molecules. Yet, the mechanism thought to be responsible for the formation of MDA chimeras is virtually identical to the one explaining PCR-based chimeras, and may perhaps even be reinforced by the strong displacement activity of the phi29 DNA polymerase used in MDA (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lasken and Stockwell 2007</a>).</p></div><div id="sec-14" class="subsection"><h3>Biological recombination of RNA viruses may be facilitated by microhomology</h3><p id="p-40">Most FHV sequences generated using the ClickSeq library preparation protocol (<a id="xref-ref-48-11" class="xref-bibr" href="#ref-48">Routh <em>et al.</em>, 2015</a>) and involved in chimeras presented 1-7 bp microhomologies, which is longer than expected by chance (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). This is fully consistent with an earlier low-throughput study that found that 24 out of 40 FHV–FHV breakpoints were characterized by the presence of 1-7 bp microhomology, supporting an occasional role for base pairing between the nascent and template strand in facilitating recombination (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Ball 1993</a>). Much like <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li &amp; Ball (1993)</a>, we found a substantial number of FHV–FHV breakpoints devoid of microhomology, suggesting that base pairing is not necessary for recombination to occur.</p><p id="p-41">The mechanisms inducing RNA virus recombination can broadly be classified into two categories, replicase-driven template switching and RNA-breakage induced recombination (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nagy and Simon 1997</a>). To our knowledge, FHV recombination has never been studied experimentally. However, the experimental study of defective interfering RNAs from other RNA viruses such as Tombusvirus (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">White and Morris 1995</a>) and Bromovirus (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pogany <em>et al.</em>, 1997</a>) suggests that the presence of microhomology is in agreement with recombination taking place via intramolecular template switching of the viral RdRp (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Nagy and Simon 1997</a>). A fairly large number of FHV–FHV chimeras were characterized by the presence of non-templated addition of nucleotides (554 out of 4,988). Such non-templated nucleotides have previously been found at recombination breakpoints of the turnip crinkle virus (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cascone <em>et al.</em>, 1990</a>), tobamovirus (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Raffo and Dawson 1991</a>) and brome mosaic virus (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nagy and Bujarski 1996</a>). Interestingly, the addition of such nucleotides by the RdRp at the 3′ end of the nascent strand has been proposed as one of the factors triggering partial dissociation of the nascent strand from the first template, pausing of the RdRp and template switching (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Nagy and Simon 1997</a>). In terms of distance between recombining regions, we found a pattern partially mirroring that generated by artificial recombination, with an enrichment of template switches over short distances (&lt; 100 bp) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). However, the enrichment toward short distances was clearly less pronounced than for artificial recombination, and we observed two other marked peaks of longer preferred distances (&gt; 500 bp). Another characteristic differentiating artificial chimeras from FHV-FHV chimeras is the presence of hotspots of recombination along the viral genome (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C,D</a>), which were previously detected by <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Routh and Jonhson (2014)</a>.</p></div><div id="sec-15" class="subsection"><h3>Conclusion</h3><p id="p-42">This study was initiated to search for evidence of <em>in vivo</em> host-RNA virus recombination in the DENV–mosquito cell system. The overrepresentation of technical chimeras resulting from <em>in vitro</em> recombination prevented us from identifying any such event, be it between host and DENV or intra-DENV. We found no evidence in support of frequent <em>in vivo</em> recombination, a prerequisite for the integration of host sequences into the DENV genome and their horizontal transfer to another host. This possibility will have to be further investigated using appropriate library preparation protocols. Importantly, a substantial amount of host RNAs can be co-encapsidated, together with the viral genome, by RNA viruses (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Eckwahl <em>et al.</em>, 2016</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ghoshal <em>et al.</em>, 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Routh <em>et al.</em>, 2012</a>). Whether these host RNAs, not integrated into the viral genome, can then be transferred between hosts by viruses remains an interesting question to address.</p><p id="p-43">The rates of artificial recombination we detected (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, column 3) are consistent with those previously reported (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Di Giallonardo <em>et al.</em>, 2013</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Haas <em>et al.</em>, 2011</a>; <a id="xref-ref-48-12" class="xref-bibr" href="#ref-48">Routh <em>et al.</em>, 2015</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Shao <em>et al.</em>, 2013</a>). They further outline the necessity to identify and remove technical chimeras from high-throughput sequencing datasets obtained by techniques using PCR, before downstream analyses (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Edgar <em>et al.</em>, 2011</a>; <a id="xref-ref-48-13" class="xref-bibr" href="#ref-48">Routh <em>et al.</em>, 2015</a>), if these analyses can be affected by their presence. Fortunately, the low frequency of artificial chimeric reads (&lt;1% of all reads) and the absence of clear recombination hotspots greatly mitigate the impact of chimeras on most analyses, such as genome sequencing and assembly, variant detection and DNA conformation analyses, as these rely on the consistency between several reads obtained from a given locus or variant. The number of non-chimeric reads should largely exceed that of chimeric reads at any locus.</p><p id="p-44">However, analyses that specifically search for rare recombination events will be affected by artificially-produced recombination, and we do not see a reliable criterion to determine any individual chimera as biological, unless the mechanism that created it is readily apparent (<em>e.g.</em>, transposition), or evidence indicate <em>in vivo</em> replication of the chimera (<em>e.g.</em>, <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Gilbert <em>et al.</em> 2016</a>). Indeed, artificial chimeras share features with biological chimeras (<a id="xref-ref-48-14" class="xref-bibr" href="#ref-48">Routh <em>et al.</em>, 2015</a>): both often involve microhomology and short distances between recombining fragments. However, artificial recombination mostly joins fragments in opposite orientations (&gt;75% of breakpoints), in stark contrast to biological recombination in FHV (∼1.5%). Coupled to a bias-free library preparation protocol and an internal control to test the occurrence of <em>in vitro</em> recombination (<em>e.g.</em>, mixing DNA/RNA of species that were never in contact), this distinctive feature may be useful to globally evaluate the presence of artificial recombination in datasets obtained from short-read sequencing technology.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank Richard Cordaux for valuable comments during the design of the study and on earlier versions of the manuscript. This work was supported by Agence Nationale de la Recherche Grant ANR-15-CE32-0011-01 TransVir (to C.G.), Agence Nationale de la Recherche Grant ANR-17-ERC2-0016-01 (to L.L.), the French Government’s Investissement d’Avenir program Laboratoire d’Excellence Integrative Biology of Emerging Infectious Diseases Grant ANR-10-LABX-62-IBEID (to L.L.), the City of Paris Emergence(s) program in Biomedical Research (to L.L.), the 2015–2020 State-Region Planning Contract and European Regional Development Fund, and intramural funds from the Centre National de la Recherche Scientifique and the University of Poitiers (to J.P. and C.G.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-46"><em>Communicating Editor: B. Andrews</em></p></li><li class="fn" id="fn-4"><p id="p-47">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.10.1534/g3.117.300468/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.10.1534/g3.117.300468/-/DC1</a>.</p></li></ul></div><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Peccoud <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.1" data-doi="10.2217/17460913.3.2.155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartenschlager</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular aspects of Dengue virus replication.</span> <abbr class="cit-jnl-abbrev">Future Microbiol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">155</span>–<span class="cit-lpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2217/17460913.3.2.155</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFuture%2BMicrobiol%26rft.aulast%253DBartenschlager%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D165%26rft.atitle%253DMolecular%2Baspects%2Bof%2BDengue%2Bvirus%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.2217%252F17460913.3.2.155%26rft_id%253Dinfo%253Apmid%252F18366336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/17460913.3.2.155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18366336&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258835900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.2" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cascone</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Recombination between satellite RNAs of turnip crinkle virus.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D9%26rft.spage%253D1709%26rft_id%253Dinfo%253Apmid%252F1693330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1693330&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.4" data-doi="10.1093/nar/16.20.9677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Novel non-templated nucleotide addition reactions catalyzed by procaryotic and eucaryotic DNA polymerases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">9677</span>–<span class="cit-lpage">9686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/16.20.9677</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DClark%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D16%26rft.issue%253D20%26rft.spage%253D9677%26rft.epage%253D9686%26rft.atitle%253DNovel%2Bnon-templated%2Bnucleotide%2Baddition%2Breactions%2Bcatalyzed%2Bby%2Bprocaryotic%2Band%2Beucaryotic%2BDNA%2Bpolymerases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F16.20.9677%26rft_id%253Dinfo%253Apmid%252F2460825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/16.20.9677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2460825&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q656200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.5" data-doi="10.1016/j.ygeno.2005.12.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cocquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reverse transcriptase template switching and false alternative transcripts.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">88</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">127</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ygeno.2005.12.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DCocquet%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D127%26rft.epage%253D131%26rft.atitle%253DReverse%2Btranscriptase%2Btemplate%2Bswitching%2Band%2Bfalse%2Balternative%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2005.12.013%26rft_id%253Dinfo%253Apmid%252F16457984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2005.12.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16457984&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238552300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.6" data-doi="10.1371/journal.pone.0074249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Giallonardo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zagordi</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duport</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leemann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joos</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Next-Generation Sequencing of HIV-1 RNA Genomes: Determination of Error Rates and Minimizing Artificial Recombination.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">e74249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0074249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De74249%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0074249%26rft_id%253Dinfo%253Apmid%252F24058534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0074249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24058534&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.7" data-doi="10.1261/rna.057299.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckwahl</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnion</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kharytonchyk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of the human immunodeficiency virus-1 RNA packageome.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1228</span>–<span class="cit-lpage">1238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.057299.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.057299.116%26rft_id%253Dinfo%253Apmid%252F27247436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTIyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.8" data-doi="10.1093/bioinformatics/btr381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clemente</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quince</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">UCHIME improves sensitivity and speed of chimera detection.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2194</span>–<span class="cit-lpage">2200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr381</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr381%26rft_id%253Dinfo%253Apmid%252F21700674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21700674&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293620800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.9" data-doi="10.1371/journal.pgen.1003621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fansiri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontaine</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diancourt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caro</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaisomboonsuk</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic Mapping of Specific Interactions between Aedes aegypti Mosquitoes and Dengue Viruses.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1003621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003621%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003621%26rft_id%253Dinfo%253Apmid%252F23935524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935524&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.10" data-doi="10.1038/nrg3199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feschotte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Endogenous viruses: insights into viral evolution and impact on host biology.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">283</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeschotte%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D283%26rft.epage%253D296%26rft.atitle%253DEndogenous%2Bviruses%253A%2Binsights%2Binto%2Bviral%2Bevolution%2Band%2Bimpact%2Bon%2Bhost%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3199%26rft_id%253Dinfo%253Apmid%252F22421730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22421730&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.11" data-doi="10.1038/srep24885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fontaine</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiolle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moltini-Conclois</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lequime</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambrechts</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Excretion of dengue virus RNA by Aedes aegypti allows non-destructive monitoring of viral dissemination in individual mosquitoes.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">24885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D24885%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep24885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep24885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Acquisition of Host Cell DNA Sequences by Baculoviruses: Relationship Between Host DNA Insertions and FP Mutants of Autographa californica and Galleria mellonella Nuclear Polyhedrosis Viruses.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFraser%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D287%26rft.epage%253D300%26rft.atitle%253DAcquisition%2Bof%2BHost%2BCell%2BDNA%2BSequences%2Bby%2BBaculoviruses%253A%2BRelationship%2BBetween%2BHost%2BDNA%2BInsertions%2Band%2BFP%2BMutants%2Bof%2BAutographa%2Bcalifornica%2Band%2BGalleria%2Bmellonella%2BNuclear%2BPolyhedrosis%2BViruses%26rft_id%253Dinfo%253Apmid%252F16789244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjQ3LzIvMjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.13" data-doi="10.1021/bi0491853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">García</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Islas</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analysis of non-template-directed nucleotide addition and template switching by DNA polymerase.</span> <abbr class="cit-jnl-abbrev">Biochemistry (Mosc.)</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">51</span>): <span class="cit-fpage">16515</span>–<span class="cit-lpage">16524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/bi0491853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Mosc.%2529%26rft.volume%253D43%26rft.spage%253D16515%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi0491853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi0491853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.14" data-doi="10.1128/JVI.01466-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghoshal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theilmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reade</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maghodia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rochon</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Encapsidation of Host RNAs by Cucumber Necrosis Virus Coat Protein during both Agroinfiltration and Infection.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">10748</span>–<span class="cit-lpage">10761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JVI.01466-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01466-15%26rft_id%253Dinfo%253Apmid%252F26269190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMS8xMDc0OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordaux</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Viruses as vectors of horizontal transfer of genetic material in eukaryotes. Curr Opin Virol.</span> <span class="cit-vol">25</span>:<span class="cit-fpage">16</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.16" data-doi="10.1371/journal.pgen.1005838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peccoud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chateigner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moumen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordaux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Continuous Influx of Genetic Material from Host to Virus Populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1005838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005838%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.17" data-doi="10.1126/science.1259657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gire</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goba</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sealfon</span>  <span class="cit-name-given-names">R. S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>(<span class="cit-issue">6202</span>): <span class="cit-fpage">1369</span>–<span class="cit-lpage">1372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1259657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.18" data-doi="10.1038/ni.2542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goic</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vodovar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondotte</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frangeul</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-mediated interference and reverse transcription control the persistence of RNA viruses in the insect model Drosophila.</span> <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">396</span>–<span class="cit-lpage">403</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ni.2542</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.stitle%253DNat%2BImmunol%26rft.aulast%253DGoic%26rft.volume%253D14%26rft.spage%253D396%26rft.atitle%253DRNA-mediated%2Binterference%2Band%2Breverse%2Btranscription%2Bcontrol%2Bthe%2Bpersistence%2Bof%2BRNA%2Bviruses%2Bin%2Bthe%2Binsect%2Bmodel%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.2542%26rft_id%253Dinfo%253Apmid%252F23435119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.2542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23435119&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.19" data-doi="10.1038/ncomms12410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goic</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapleford</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frangeul</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doucet</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gausson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Virus-derived DNA drives mosquito vector tolerance to arboviral infection.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms12410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D12410%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms12410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms12410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.20" data-doi="10.1016/j.jviromet.2010.07.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Görzer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trajanoski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchhammer-Stöckl</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of PCR-generated recombination on diversity estimation of mixed viral populations by deep sequencing.</span> <abbr class="cit-jnl-abbrev">J. Virol. Methods</abbr> <span class="cit-vol">169</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">248</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jviromet.2010.07.040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DGorzer%26rft.auinit1%253DI.%26rft.volume%253D169%26rft.issue%253D1%26rft.spage%253D248%26rft.epage%253D252%26rft.atitle%253DThe%2Bimpact%2Bof%2BPCR-generated%2Brecombination%2Bon%2Bdiversity%2Bestimation%2Bof%2Bmixed%2Bviral%2Bpopulations%2Bby%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2010.07.040%26rft_id%253Dinfo%253Apmid%252F20691210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2010.07.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20691210&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282398600040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.21" data-doi="10.1021/bi00028a032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guieysse</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Praseuth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harel-Bellan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hélène</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Oligonucleotide-directed switching of DNA polymerases to a dead-end track.</span> <abbr class="cit-jnl-abbrev">Biochemistry (Mosc.)</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">28</span>): <span class="cit-fpage">9193</span>–<span class="cit-lpage">9199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/bi00028a032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Mosc.%2529%26rft.volume%253D34%26rft.spage%253D9193%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00028a032%26rft_id%253Dinfo%253Apmid%252F7619820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00028a032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7619820&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RK55400032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.22" data-doi="10.1016/S0140-6736(14)60572-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dengue.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">385</span>(<span class="cit-issue">9966</span>): <span class="cit-fpage">453</span>–<span class="cit-lpage">465</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0140-6736(14)60572-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D385%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252814%252960572-9%26rft_id%253Dinfo%253Apmid%252F25230594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(14)60572-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25230594&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.23" data-doi="10.1101/gr.112730.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gevers</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldgarden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chimeric 16S rRNA sequence formation and detection in Sanger and 454-pyrosequenced PCR amplicons.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">494</span>–<span class="cit-lpage">504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.112730.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.112730.110%26rft_id%253Dinfo%253Apmid%252F21212162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvNDk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaworski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Routh</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">ClickSeq: replacing fragmentation and enzymatic ligation with click-chemistry to prevent sequence chimeras. Methods Mol Biol.</span> <span class="cit-vol">1712</span>:<span class="cit-fpage">71</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.25" data-doi="10.1016/S1389-1723(03)90130-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanagawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Bias and artifacts in multitemplate polymerase chain reactions (PCR).</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">96</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">317</span>–<span class="cit-lpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1389-1723(03)90130-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiosci%2BBioeng%26rft.aulast%253DKanagawa%26rft.auinit1%253DT.%26rft.volume%253D96%26rft.issue%253D4%26rft.spage%253D317%26rft.epage%253D323%26rft.atitle%253DBias%2Band%2Bartifacts%2Bin%2Bmultitemplate%2Bpolymerase%2Bchain%2Breactions%2B%2528PCR%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1389-1723%252803%252990130-7%26rft_id%253Dinfo%253Apmid%252F16233530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1389-1723(03)90130-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16233530&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187530500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.26" data-doi="10.1371/journal.pgen.1001191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katzourakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Endogenous viral elements in animal genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1001191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1001191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKatzourakis%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001191%26rft.epage%253De1001191%26rft.atitle%253DEndogenous%2Bviral%2Belements%2Bin%2Banimal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001191%26rft_id%253Dinfo%253Apmid%252F21124940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124940&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.27" data-doi="10.1093/gbe/evv034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multiple layers of chimerism in a single-stranded DNA virus discovered by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">993</span>–<span class="cit-lpage">1001</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv034%26rft_id%253Dinfo%253Apmid%252F25840414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25840414&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.28" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.29" data-doi="10.1186/1472-6750-7-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lasken</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockwell</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mechanism of chimera formation during the Multiple Displacement Amplification reaction.</span> <abbr class="cit-jnl-abbrev">BMC Biotechnol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1472-6750-7-19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbiotechnology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBiotechnol%26rft.aulast%253DLasken%26rft.auinit1%253DR.%2BS.%26rft.volume%253D7%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DMechanism%2Bof%2Bchimera%2Bformation%2Bduring%2Bthe%2BMultiple%2BDisplacement%2BAmplification%2Breaction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6750-7-19%26rft_id%253Dinfo%253Apmid%252F17430586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6750-7-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17430586&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Nonhomologous RNA recombination during negative-strand synthesis of flock house virus RNA.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">3854</span>–<span class="cit-lpage">3860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLi%26rft.auinit1%253DY%26rft.volume%253D67%26rft.issue%253D7%26rft.spage%253D3854%26rft.epage%253D3860%26rft.atitle%253DNonhomologous%2BRNA%2Brecombination%2Bduring%2Bnegative-strand%2Bsynthesis%2Bof%2Bflock%2Bhouse%2Bvirus%2BRNA.%26rft_id%253Dinfo%253Apmid%252F7685402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY3LzcvMzg1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.31" data-doi="10.1371/journal.pone.0019447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lott</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thu</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Defective Interfering Viral Particles in Acute Dengue Infections.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e19447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019447</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLi%26rft.auinit1%253DD.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De19447%26rft.epage%253De19447%26rft.atitle%253DDefective%2Binterfering%2Bviral%2Bparticles%2Bin%2Bacute%2Bdengue%2Binfections.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019447%26rft_id%253Dinfo%253Apmid%252F21559384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21559384&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.32" data-doi="10.3390/v3091699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biagini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefeuvre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roumagnac</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination in Eukaryotic Single Stranded DNA Viruses.</span> <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1699</span>–<span class="cit-lpage">1738</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/v3091699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DMartin%26rft.auinit1%253DD.%2BP.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D1699%26rft.epage%253D1738%26rft.atitle%253DRecombination%2Bin%2Beukaryotic%2Bsingle%2Bstranded%2BDNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv3091699%26rft_id%253Dinfo%253Apmid%252F21994803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v3091699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994803&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295211400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.33" data-doi="10.1093/nar/18.7.1687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyerhans</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vartanian</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wain-Hobson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">DNA recombination during PCR.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1687</span>–<span class="cit-lpage">1691</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/18.7.1687</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMeyerhans%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D1687%26rft.epage%253D1691%26rft.atitle%253DDNA%2Brecombination%2Bduring%2BPCR%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.7.1687%26rft_id%253Dinfo%253Apmid%252F2186361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/18.7.1687&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2186361&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CZ68400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.34" data-doi="10.1038/299562a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">A virus mutant with an insertion of a copia-like transposable element.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">299</span>(<span class="cit-issue">5883</span>): <span class="cit-fpage">562</span>–<span class="cit-lpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/299562a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMiller%26rft.auinit1%253DD.%2BW.%26rft.volume%253D299%26rft.issue%253D5883%26rft.spage%253D562%26rft.epage%253D564%26rft.atitle%253DA%2Bvirus%2Bmutant%2Bwith%2Ban%2Binsertion%2Bof%2Ba%2Bcopia-like%2Btransposable%2Belement.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F299562a0%26rft_id%253Dinfo%253Apmid%252F6289125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/299562a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6289125&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bujarski</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Homologous RNA recombination in brome mosaic virus: AU-rich sequences decrease the accuracy of crossovers.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D415%26rft.epage%253D426%26rft.atitle%253DHomologous%2BRNA%2Brecombination%2Bin%2Bbrome%2Bmosaic%2Bvirus%253A%2BAU-rich%2Bsequences%2Bdecrease%2Bthe%2Baccuracy%2Bof%2Bcrossovers%26rft_id%253Dinfo%253Apmid%252F8523555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjcwLzEvNDE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.36" data-doi="10.1006/viro.1997.8681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">New Insights into the Mechanisms of RNA Recombination.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">235</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/viro.1997.8681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D235%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253DNew%2Binsights%2Binto%2Bthe%2Bmechanisms%2Bof%2BRNA%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1997.8681%26rft_id%253Dinfo%253Apmid%252F9300032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1997.8681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9300032&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XT89500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.37" data-doi="10.1093/nar/23.11.2049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odelberg</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hata</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Template-switching during DNA synthesis by Thermus aquaticus DNA polymerase I.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2049</span>–<span class="cit-lpage">2057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/23.11.2049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.11.2049%26rft_id%253Dinfo%253Apmid%252F7596836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/23.11.2049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7596836&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RE99600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1129.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glowatz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlumpberger</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ribosomal RNA depletion for efficient use of RNA-seq capacity</span> in <span class="cit-source">Current Protocols in Molecular Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kingston</span>  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Seidman</span>  <span class="cit-name-given-names">J. G.</span></span> <em>et al</em>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pääbo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">DNA damage promotes jumping between templates during enzymatic amplification.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">4718</span>–<span class="cit-lpage">4721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DPaabo%26rft.auinit1%253DS.%26rft.volume%253D265%26rft.issue%253D8%26rft.spage%253D4718%26rft.epage%253D4721%26rft.atitle%253DDNA%2Bdamage%2Bpromotes%2Bjumping%2Bbetween%2Btemplates%2Bduring%2Benzymatic%2Bamplification%26rft_id%253Dinfo%253Apmid%252F2307682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNjUvOC80NzE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzExMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.40" data-doi="10.1073/pnas.93.7.2969"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurn</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Formation of chimeric DNA primer extension products by template switching onto an annealed downstream oligonucleotide.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">2969</span>–<span class="cit-lpage">2974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.93.7.2969</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPatel%26rft.auinit1%253DR.%26rft.volume%253D93%26rft.issue%253D7%26rft.spage%253D2969%26rft.epage%253D2974%26rft.atitle%253DFormation%2Bof%2Bchimeric%2BDNA%2Bprimer%2Bextension%2Bproducts%2Bby%2Btemplate%2Bswitching%2Bonto%2Ban%2Bannealed%2Bdownstream%2Boligonucleotide.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.7.2969%26rft_id%253Dinfo%253Apmid%252F8610152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5My83LzI5NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.41" data-doi="10.1073/pnas.1621178114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peccoud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loiseau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordaux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Massive horizontal transfer of transposable elements in insects.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>(<span class="cit-issue">18</span>): <span class="cit-fpage">4721</span>–<span class="cit-lpage">4726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1621178114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1621178114%26rft_id%253Dinfo%253Apmid%252F28416702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzE4LzQ3MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.42" data-doi="10.1016/j.meegid.2014.12.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Losada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galán</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palero</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Candelas</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination in viruses: Mechanisms, methods of study, and evolutionary consequences.</span> <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">296</span>–<span class="cit-lpage">307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.meegid.2014.12.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol.%26rft.volume%253D30%26rft.spage%253D296%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2014.12.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2014.12.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.43" data-doi="10.1006/viro.1996.8377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pogany</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bujarski</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Effect of 5′ and 3′ terminal sequences, overall length, and coding capacity on the accumulation of defective RNAs associated with broad bean mottle bromovirus in planta.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">228</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">236</span>–<span class="cit-lpage">243</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/viro.1996.8377</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DPogany%26rft.auinit1%253DJ.%26rft.volume%253D228%26rft.issue%253D2%26rft.spage%253D236%26rft.epage%253D243%26rft.atitle%253DEffect%2Bof%2B5%2527%2Band%2B3%2527%2Bterminal%2Bsequences%252C%2Boverall%2Blength%252C%2Band%2Bcoding%2Bcapacity%2Bon%2Bthe%2Baccumulation%2Bof%2Bdefective%2BRNAs%2Bassociated%2Bwith%2Bbroad%2Bbean%2Bmottle%2Bbromovirus%2Bin%2Bplanta.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1996.8377%26rft_id%253Dinfo%253Apmid%252F9123830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1996.8377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9123830&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-web" id="cit-8.4.1129.44"><div class="cit-metadata unstructured">R Core Team, (2017). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.45" data-doi="10.1016/0042-6822(91)90844-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raffo</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">W. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Construction of tobacco mosaic virus subgenomic replicons that are replicated and spread systemically in tobacco plants.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">184</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">277</span>–<span class="cit-lpage">289</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0042-6822(91)90844-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DRaffo%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D184%26rft.issue%253D1%26rft.spage%253D277%26rft.epage%253D289%26rft.atitle%253DConstruction%2Bof%2Btobacco%2Bmosaic%2Bvirus%2Bsubgenomic%2Breplicons%2Bthat%2Bare%2Breplicated%2Band%2Bspread%2Bsystemically%2Bin%2Btobacco%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252891%252990844-2%26rft_id%253Dinfo%253Apmid%252F1871972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(91)90844-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1871972&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.46" data-doi="10.1093/nar/gkt916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Routh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Discovery of functional genomic motifs in viruses with ViReMa-a Virus Recombination Mapper-for analysis of next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt916</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt916%26rft_id%253Dinfo%253Apmid%252F24137010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24137010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.47" data-doi="10.1073/pnas.1116168109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Routh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domitrovic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Host RNAs, including transposons, are encapsidated by a eukaryotic single-stranded RNA virus.</span> <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U A</abbr> <span class="cit-vol">109</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1907</span>–<span class="cit-lpage">1912</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1116168109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1116168109%26rft_id%253Dinfo%253Apmid%252F22308402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzYvMTkwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.48" data-doi="10.1016/j.jmb.2015.06.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Routh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Head</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ordoukhanian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ClickSeq: Fragmentation-Free Next-Generation Sequencing via Click Ligation of Adaptors to Stochastically Terminated 3′-Azido cDNAs.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">427</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2610</span>–<span class="cit-lpage">2616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2015.06.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D427%26rft.spage%253D2610%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.06.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.49" data-doi="10.1016/S0009-8981(00)00374-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shammas</span>  <span class="cit-name-given-names">F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heikkilä</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osland</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Fluorescence-based method for measuring and determining the mechanisms of recombination in quantitative PCR.</span> <abbr class="cit-jnl-abbrev">Clin. Chim. Acta Int. J. Clin. Chem.</abbr> <span class="cit-vol">304</span>(<span class="cit-issue">1-2</span>): <span class="cit-fpage">19</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0009-8981(00)00374-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinica%2BChimica%2BActa%26rft.stitle%253DClinica%2BChimica%2BActa%26rft.aulast%253DShammas%26rft.auinit1%253DF.%2BV.%26rft.volume%253D304%26rft.issue%253D1-2%26rft.spage%253D19%26rft.epage%253D28%26rft.atitle%253DFluorescence-based%2Bmethod%2Bfor%2Bmeasuring%2Band%2Bdetermining%2Bthe%2Bmechanisms%2Bof%2Brecombination%2Bin%2Bquantitative%2BPCR.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0009-8981%252800%252900374-0%26rft_id%253Dinfo%253Apmid%252F11165195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0009-8981(00)00374-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11165195&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167221700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.50" data-doi="10.1186/1742-4690-10-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boltz</span>  <span class="cit-name-given-names">V. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spindler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maldarelli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of 454 sequencing error rate, error sources, and artifact recombination for detection of Low-frequency drug resistance mutations in HIV-1 DNA.</span> <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1742-4690-10-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRetrovirology%26rft.volume%253D10%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-10-18%26rft_id%253Dinfo%253Apmid%252F23402264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-10-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23402264&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.51" data-doi="10.1038/nrmicro2614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon-Loriere</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Why do RNA viruses recombine?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">617</span>–<span class="cit-lpage">626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro2614</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DSimon-Loriere%26rft.auinit1%253DE.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D617%26rft.epage%253D626%26rft.atitle%253DWhy%2Bdo%2BRNA%2Bviruses%2Brecombine%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2614%26rft_id%253Dinfo%253Apmid%252F21725337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21725337&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaz</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahony</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartley</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fowler</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficorilli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The first genome sequence of a metatherian herpesvirus: Macropodid herpesvirus 1.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1129.53" data-doi="10.1007/82_2017_21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Exchange of Genetic Sequences Between Viruses and Hosts</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span> in <span class="cit-source"><em>Viruses</em>, <em>Genes</em>, <em>and Cancer</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bister</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Springer International Publishing</span>, <span class="cit-publ-loc">Cham</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/82_2017_21</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%252C%2BGenes%252C%2Band%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1007%252F82_2017_21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/82_2017_21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">RNA determinants of junction site selection in RNA virus recombinants and defective interfering RNAs.</span> <abbr class="cit-jnl-abbrev">RNA N. Y. N</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BN.%2BY.%2BN%26rft.volume%253D1%26rft.spage%253D1029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.55" data-doi="10.1016/j.virusres.2016.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Error rates, PCR recombination, and sampling depth in HIV-1 whole genome deep sequencing.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">239</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.virusres.2016.12.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D239%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2016.12.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2016.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1129.56" data-doi="10.1016/S0014-5793(02)02812-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">X.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.-X.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence that BmTXK beta-BmKCT cDNA from Chinese scorpion Buthus martensii Karsch is an artifact generated in the reverse transcription process.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">520</span>(<span class="cit-issue">1-3</span>): <span class="cit-fpage">183</span>–<span class="cit-lpage">184, author reply 185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0014-5793(02)02812-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DZeng%26rft.auinit1%253DX.%2BC.%26rft.volume%253D520%26rft.issue%253D1-3%26rft.spage%253D183%26rft.epage%253D184%26rft.atitle%253DEvidence%2Bthat%2BBmTXK%2Bbeta-BmKCT%2BcDNA%2Bfrom%2BChinese%2Bscorpion%2BButhus%2Bmartensii%2BKarsch%2Bis%2Ban%2Bartifact%2Bgenerated%2Bin%2Bthe%2Breverse%2Btranscription%2Bprocess.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252802%252902812-0%26rft_id%253Dinfo%253Apmid%252F12044895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(02)02812-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12044895&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/4/1129.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1119" title="A Metagenome-Wide Association Study and Arrayed Mutant Library Confirm Acetobacter Lipopolysaccharide Genes Are Necessary for Association with Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1139" title="Maize Transposable Elements Ac/Ds as Insertion Mutagenesis Tools in Candida albicans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/4"><span>Volume 8 Issue 4, April 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/4.cover-source.jpg?itok=wC37V0ox" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1129" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VmZSMjOHDXNmZAF2r0RX13BfqLvk7y2VTJ-V_-7oBsI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/4/1129" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/4/1129" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/4/1129" class="active" data-icon-position="" data-hide-link-title="0">A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489121" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Eompn2P89a9Yavm1PVReQm4_hnayfnapkZBAPTcsJgM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1129.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/4/1129" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JMbFMFhn4PBzgDE7FfXvWoTPkvwsoUR4bTPBrlj1jIo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/4/1129" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1129.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489121" id="node489121--2" data-pisa="ggg;8/4/1129" data-pisa-master="ggg;g3.117.300468" data-apath="/ggg/8/4/1129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3356-7869" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean Peccoud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3140-0651" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sébastian Lequime</span>, <span class="highwire-citation-author" data-delta="2">Isabelle Moltini-Conclois</span>, <span class="highwire-citation-author" data-delta="3">Isabelle Giraud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5958-2138" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Louis Lambrechts</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2131-7467" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clément Gilbert</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1129-1138;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300468 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.300468" data-atom-uri="/ggg/8/4/1129.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489121--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jean Peccoud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BPeccoud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peccoud%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BPeccoud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3356-7869" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean Peccoud</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sébastian Lequime </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastian%2BLequime%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lequime%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS%25C3%25A9bastian%2BLequime%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3140-0651" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sébastian Lequime</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Isabelle Moltini-Conclois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BMoltini-Conclois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moltini-Conclois%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BMoltini-Conclois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Isabelle Giraud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BGiraud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giraud%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BGiraud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Louis Lambrechts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BLambrechts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lambrechts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouis%2BLambrechts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5958-2138" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louis Lambrechts</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Clément Gilbert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Laboratoire Evolution, Génomes, Comportement, Écologie, UMR 9191 CNRS, UMR 247 IRD, Université Paris-Sud, 91198 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cl%C3%A9ment%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACl%25C3%25A9ment%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2131-7467" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clément Gilbert</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/4/1129" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/4/1129&amp;title=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/4/1129&amp;title=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/4/1129&amp;title=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/4/1129&amp;text=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/4/1129&amp;title=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/4/1129&amp;t=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/4/1129&amp;title=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/4/1129&amp;title=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489121/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489121" id="node489121--3" data-pisa="ggg;8/4/1129" data-pisa-master="ggg;g3.117.300468" data-apath="/ggg/8/4/1129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Survey of Virus Recombination Uncovers Canonical Features of Artificial Chimeras Generated During Deep Sequencing Library Preparation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3356-7869" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean Peccoud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3140-0651" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sébastian Lequime</span>, <span class="highwire-citation-author" data-delta="2">Isabelle Moltini-Conclois</span>, <span class="highwire-citation-author" data-delta="3">Isabelle Giraud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5958-2138" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Louis Lambrechts</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-2131-7467" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clément Gilbert</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1129-1138;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300468 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.117.300468" data-atom-uri="/ggg/8/4/1129.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489121--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jean Peccoud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean%2BPeccoud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peccoud%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean%2BPeccoud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3356-7869" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean Peccoud</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sébastian Lequime </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastian%2BLequime%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lequime%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS%25C3%25A9bastian%2BLequime%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3140-0651" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sébastian Lequime</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Isabelle Moltini-Conclois </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BMoltini-Conclois%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moltini-Conclois%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BMoltini-Conclois%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Isabelle Giraud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span>Laboratoire Ecologie et Biologie des Interactions Unité Mixte de Recherche (UMR) Centre National de la Recherche Scientifique (CNRS) 7267, Université de Poitiers, 86000 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BGiraud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giraud%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BGiraud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Louis Lambrechts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span>Insect-Virus Interactions Group, Department of Genomes and Genetics, Institut Pasteur, Paris, France</div><div class='author-affiliation'><span class='nlm-sup'>3</span>CNRS, UMR 2000, Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BLambrechts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lambrechts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouis%2BLambrechts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5958-2138" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louis Lambrechts</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Clément Gilbert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span>Laboratoire Evolution, Génomes, Comportement, Écologie, UMR 9191 CNRS, UMR 247 IRD, Université Paris-Sud, 91198 Gif-sur-Yvette, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cl%C3%A9ment%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACl%25C3%25A9ment%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2131-7467" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clément Gilbert</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1129&amp;rft.epage=1138&amp;rft.atitle=A%20Survey%20of%20Virus%20Recombination%20Uncovers%20Canonical%20Features%20of%20Artificial%20Chimeras%20Generated%20During%20Deep%20Sequencing%20Library%20Preparation&amp;rft.volume=8&amp;rft.issue=4&amp;rft.date=2018-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Peccoud%2C+Jean&amp;rft.au=Lequime%2C+S%C3%A9bastian&amp;rft.au=Moltini-Conclois%2C+Isabelle&amp;rft.au=Giraud%2C+Isabelle&amp;rft.au=Lambrechts%2C+Louis&amp;rft.au=Gilbert%2C+Cl%C3%A9ment"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489121/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489121/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489121/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489121/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489121/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489121/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489121/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489121/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489121/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489121/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489121/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489121/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/4/1129" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/4/1129.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/4/1129.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/4/1129.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/4/1129.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
