<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/4/1327" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/4/1327.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/4/1327.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/4/1327.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/4/1327.atom" />
<meta name="HW.pisa" content="ggg;8/4/1327" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in Capsella" />
<meta name="DC.Identifier" content="10.1534/g3.117.300467" />
<meta name="DC.Date" content="2018-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Bachmann et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Rapid advances in short-read DNA sequencing technologies have revolutionized population genomic studies, but there are genomic regions where this technology reaches its limits. Limitations mostly arise due to the difficulties in assembly or alignment to genomic regions of high sequence divergence and high repeat content, which are typical characteristics for loci under strong long-term balancing selection. Studying genetic diversity at such loci therefore remains challenging. Here, we investigate the feasibility and error rates associated with targeted long-read sequencing of a locus under balancing selection. For this purpose, we generated bacterial artificial chromosomes (BACs) containing the Brassicaceae S -locus, a region under strong negative frequency-dependent selection which has previously proven difficult to assemble in its entirety using short reads. We sequence S -locus BACs with single-molecule long-read sequencing technology and conduct de novo assembly of these S -locus haplotypes. By comparing repeated assemblies resulting from independent long-read sequencing runs on the same BAC clone we do not detect any structural errors, suggesting that reliable assemblies are generated, but we estimate an indel error rate of 5.7×10−5. A similar error rate was estimated based on comparison of Illumina short-read sequences and BAC assemblies. Our results show that, until de novo assembly of multiple individuals using long-read sequencing becomes feasible, targeted long-read sequencing of loci under balancing selection is a viable option with low error rates for single nucleotide polymorphisms or structural variation. We further find that short-read sequencing is a valuable complement, allowing correction of the relatively high rate of indel errors that result from this approach." />
<meta name="DC.Contributor" content="Jörg A. Bachmann" />
<meta name="DC.Contributor" content="Andrew Tedder" />
<meta name="DC.Contributor" content="Benjamin Laenen" />
<meta name="DC.Contributor" content="Kim A. Steige" />
<meta name="DC.Contributor" content="Tanja Slotte" />
<meta name="article:published_time" content="2018-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in Capsella" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Rapid advances in short-read DNA sequencing technologies have revolutionized population genomic studies, but there are genomic regions where this technology reaches its limits. Limitations mostly arise due to the difficulties in assembly or alignment to genomic regions of high sequence divergence and high repeat content, which are typical characteristics for loci under strong long-term balancing selection. Studying genetic diversity at such loci therefore remains challenging. Here, we investigate the feasibility and error rates associated with targeted long-read sequencing of a locus under balancing selection. For this purpose, we generated bacterial artificial chromosomes (BACs) containing the Brassicaceae &lt;i&gt;S&lt;/i&gt;-locus, a region under strong negative frequency-dependent selection which has previously proven difficult to assemble in its entirety using short reads. We sequence &lt;i&gt;S&lt;/i&gt;-locus BACs with single-molecule long-read sequencing technology and conduct &lt;i&gt;de novo&lt;/i&gt; assembly of these &lt;i&gt;S&lt;/i&gt;-locus haplotypes. By comparing repeated assemblies resulting from independent long-read sequencing runs on the same BAC clone we do not detect any structural errors, suggesting that reliable assemblies are generated, but we estimate an indel error rate of 5.7×10&lt;sup&gt;−5&lt;/sup&gt;. A similar error rate was estimated based on comparison of Illumina short-read sequences and BAC assemblies. Our results show that, until &lt;i&gt;de novo&lt;/i&gt; assembly of multiple individuals using long-read sequencing becomes feasible, targeted long-read sequencing of loci under balancing selection is a viable option with low error rates for single nucleotide polymorphisms or structural variation. We further find that short-read sequencing is a valuable complement, allowing correction of the relatively high rate of indel errors that result from this approach.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/04/01" />
<meta name="citation_mjid" content="ggg;8/4/1327" />
<meta name="citation_id" content="8/4/1327" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/4/1327" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/4/1327.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/4/1327.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/4/1327.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300467" />
<meta name="citation_pmid" content="29476024" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1327" />
<meta name="citation_lastpage" content="1333" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jörg A. Bachmann" />
<meta name="citation_author_institution" content="Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden" />
<meta name="citation_author_email" content="jorg.bachmann@su.se" />
<meta name="citation_author" content="Andrew Tedder" />
<meta name="citation_author_institution" content="Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden" />
<meta name="citation_author" content="Benjamin Laenen" />
<meta name="citation_author_institution" content="Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden" />
<meta name="citation_author" content="Kim A. Steige" />
<meta name="citation_author_institution" content="Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden" />
<meta name="citation_author" content="Tanja Slotte" />
<meta name="citation_author_institution" content="Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. F. Au;citation_author=J. G. Underwood;citation_author=L. Lee;citation_author=W. H. Wong;citation_title=Improving PacBio Long Read Accuracy by Short Read Alignment.;citation_pages=e46679-e46679;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056399;citation_doi=10.1371/journal.pone.0046679" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. S. Bechsgaard;citation_author=V. Castric;citation_author=D. Charlesworth;citation_author=X. Vekemans;citation_author=M. H. Schierup;citation_title=The transition to self-compatibility in Arabidopsis thaliana and evolution within S-haplotypes over 10 Myr.;citation_volume=23;citation_year=2006;citation_issue=9;citation_pmid=16782760;citation_doi=10.1093/molbev/msl042" />
<meta name="citation_reference" content="citation_title=Long Read Sequencing Technology to Solve Complex Genomic Regions Assembly in Plants.;citation_author=A. Bellec;citation_author=A. Courtial;citation_author=S. Cauet;citation_author=N. Rodde;citation_author=S. Vautrin;citation_pages=128;citation_volume=3;citation_year=2016;citation_doi=10.4172/2469-9853.1000128" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina Sequence Data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=V. Castric;citation_author=J. S. Bechsgaard;citation_author=S. Grenier;citation_author=R. Noureddine;citation_author=M. H. Schierup;citation_title=Molecular Evolution within and between Self-Incompatibility Specificities.;citation_volume=27;citation_year=2010;citation_issue=1;citation_pmid=19773365;citation_doi=10.1093/molbev/msp224" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=V. Castric;citation_author=X. Vekemans;citation_title=Evolution under strong balancing selection: how many codons determine specificity at the female self-incompatibility gene SRK in Brassicaceae?;citation_pages=132-132;citation_volume=7;citation_year=2007;citation_issue=1;citation_pmid=17683611;citation_doi=10.1186/1471-2148-7-132" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=D. Charlesworth;citation_title=The status of supergenes in the 21st century: Recombination suppression in Batesian mimicry and sex chromosomes and other complex adaptations.;citation_pages=74-90;citation_volume=9;citation_year=2016;citation_issue=1;citation_pmid=27087840;citation_doi=10.1111/eva.12291" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=D. Charlesworth;citation_author=P. Awadalla;citation_author=B. K. Mable;citation_author=M. H. Schierup;citation_title=Population-level Studies of Multiallelic Self-incompatibility Loci, with Particular Reference to Brassicaceae;citation_pages=227-239;citation_volume=85;citation_year=2000;citation_issue=suppl_1;citation_doi=10.1006/anbo.1999.1015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Charlesworth;citation_author=C. Bartolomé;citation_author=M. H. Schierup;citation_author=B. K. Mable;citation_title=Haplotype Structure of the Stigmatic Self-Incompatibility Gene in Natural Populations of Arabidopsis lyrata.;citation_volume=20;citation_year=2003;citation_issue=11;citation_pmid=12832651;citation_doi=10.1093/molbev/msg170" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Charlesworth;citation_author=B. K. Mable;citation_author=M. H. Schierup;citation_author=C. Bartolomé;citation_author=P. Awadalla;citation_title=Diversity and Linkage of Genes in the Self-Incompatibility Gene Family in Arabidopsis lyrata;citation_pages=1519-1535;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930757" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C.-S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_issue=6;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Du;citation_author=Y. Sun;citation_title=Improve homology search sensitivity of PacBio data by correcting frameshifts.;citation_volume=32;citation_year=2016;citation_issue=17;citation_pmid=27587671;citation_doi=10.1093/bioinformatics/btw458" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Durand;citation_author=R. Meheust;citation_author=M. Soucaze;citation_author=P. M. Goubet;citation_author=S. Gallina;citation_title=Dominance hierarchy arising from the evolution of a complex small RNA regulatory network;citation_pages=1200-1205;citation_volume=346;citation_year=2014;citation_issue=6214;citation_pmid=25477454;citation_doi=10.1126/science.1259442" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Eid;citation_author=A. Fehr;citation_author=J. Gray;citation_author=K. Luong;citation_author=J. Lyle;citation_title=Real-time DNA sequencing from single polymerase molecules.;citation_volume=323;citation_year=2009;citation_issue=5910;citation_pmid=19023044;citation_doi=10.1126/science.1162986" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. Fijarczyk;citation_author=W. Babik;citation_title=Detecting balancing selection in genomes: Limits and prospects.;citation_pages=3529-3545;citation_volume=24;citation_year=2015;citation_issue=14;citation_doi=10.1111/mec.13226" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=S. Goodwin;citation_author=J. D. McPherson;citation_author=W. R. McCombie;citation_title=Coming of age: Ten years of next-generation sequencing technologies.;citation_pages=333-351;citation_volume=17;citation_year=2016;citation_issue=6;citation_pmid=27184599;citation_doi=10.1038/nrg.2016.49" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. M. Goubet;citation_author=H. Bergès;citation_author=A. Bellec;citation_author=E. Prat;citation_author=N. Helmstetter;citation_title=Contrasted patterns of molecular evolution in dominant and recessive self-incompatibility haplotypes in Arabidopsis.;citation_pages=e1002495-e1002495;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22457631;citation_doi=10.1371/journal.pgen.1002495" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Gouy;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building.;citation_volume=27;citation_year=2010;citation_issue=2;citation_pmid=19854763;citation_doi=10.1093/molbev/msp259" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y.-L. Guo;citation_author=J. S. Bechsgaard;citation_author=T. Slotte;citation_author=B. Neuffer;citation_author=M. Lascoux;citation_title=Recent speciation of Capsella rubella from Capsella grandiflora, associated with loss of self-incompatibility and an extreme bottleneck.;citation_volume=106;citation_year=2009;citation_issue=13;citation_pmid=19307580;citation_doi=10.1073/pnas.0808012106" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y.-L. Guo;citation_author=X. Zhao;citation_author=C. Lanz;citation_author=D. Weigel;citation_title=Evolution of the S-Locus Region in Arabidopsis Relatives.;citation_volume=157;citation_year=2011;citation_issue=2;citation_pmid=21810962;citation_doi=10.1104/pp.111.174912" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Hackl;citation_author=R. Hedrich;citation_author=J. Schultz;citation_author=F. Förster;citation_title=Proovread: Large-scale high-accuracy PacBio correction through iterative short read consensus.;citation_volume=30;citation_year=2014;citation_issue=21;citation_pmid=25015988;citation_doi=10.1093/bioinformatics/btu392" />
<meta name="citation_reference" content="Harris, R. S., 2007 Improved Pairwise Alignment of Genomic DNA. PhD thesis, Penn. State Univ." />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=J. M. Heather;citation_author=B. Chain;citation_title=The sequence of sequencers : The history of sequencing DNA.;citation_pages=1-8;citation_volume=107;citation_year=2016;citation_issue=1;citation_pmid=26554401;citation_doi=10.1016/j.ygeno.2015.11.003" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=P. W. Hedrick;citation_title=Balancing selection and MHC.;citation_pages=207-214;citation_volume=104;citation_year=1999;citation_issue=3;citation_doi=10.1023/A:1026494212540" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Plant Ecol. Divers.;citation_author=M. H. Jørgensen;citation_author=K. Lagesen;citation_author=B. K. Mable;citation_author=A. K. Brysting;citation_title=Using high-throughput sequencing to investigate the evolution of self-incompatibility genes in the Brassicaceae: strategies and challenges.;citation_pages=473-484;citation_volume=5;citation_year=2012;citation_issue=4;citation_doi=10.1080/17550874.2012.748098" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Kamau;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=Linkage disequilibrium and recombination rate estimates in the self-incompatibility region of Arabidopsis lyrata.;citation_volume=176;citation_year=2007;citation_issue=4;citation_pmid=17565949;citation_doi=10.1534/genetics.107.072231" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Katoh;citation_author=K. Misawa;citation_author=K. Kuma;citation_author=T. Miyata;citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;citation_pages=3059-3066;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136088;citation_doi=10.1093/nar/gkf436" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=S. Koren;citation_author=M. C. Schatz;citation_author=B. P. Walenz;citation_author=J. Martin;citation_author=J. T. Howard;citation_title=Hybrid error correction and de novo assembly of single-molecule sequencing reads.;citation_pages=693-700;citation_volume=30;citation_year=2012;citation_issue=7;citation_pmid=22750884;citation_doi=10.1038/nbt.2280" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=M. Kusaba;citation_author=K. Dwyer;citation_author=J. Hendershot;citation_author=J. Vrebalov;citation_author=J. B. Nasrallah;citation_title=Self-Incompatibility in the Genus Arabidopsis: Characterization of the S Locus in the Outcrossing A. lyrata and Its Autogamous Relative A. thaliana;citation_pages=627-643;citation_volume=13;citation_year=2001;citation_issue=3;citation_pmid=11251101;citation_doi=10.1105/tpc.13.3.627" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=J.-B. Leducq;citation_author=C. C. Gosset;citation_author=R. Gries;citation_author=K. Calin;citation_author=É. Schmitt;citation_title=Self-Incompatibility in Brassicaceae: Identification and Characterization of SRK -Like Sequences Linked to the S -Locus in the Tribe Biscutelleae. G3 Genes, Genomes;citation_pages=983-992;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.010843" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=V. Llaurens;citation_author=A. Whibley;citation_author=M. Joron;citation_title=Genetic architecture and balancing selection: the life and death of differentiated variants.;citation_pages=2430-2448;citation_volume=26;citation_year=2017;citation_issue=9;citation_doi=10.1111/mec.14051" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=E. R. Mardis;citation_title=DNA sequencing technologies: 2006–2016.;citation_pages=213-218;citation_volume=12;citation_year=2017;citation_issue=2;citation_doi=10.1038/nprot.2016.182" />
<meta name="citation_reference" content="citation_journal_title=CUTADAPT REMOVES ADAPTER SEQUENCES FROM HIGHTHROUGHPUT SEQUENCING READS;citation_author=M. Martin;citation_pages=10;citation_volume=17;citation_year=2011;citation_doi=10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Miege;citation_author=V. Ruffio-Châble;citation_author=M. H. Schierup;citation_author=D. Cabrillac;citation_author=C. Dumas;citation_title=Intrahaplotype Polymorphism at the Brassica S Locus;citation_pages=811-822;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606555" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. W. Myers;citation_author=G. G. Sutton;citation_author=A. L. Delcher;citation_author=I. M. Dew;citation_author=D. P. Fasulo;citation_title=A Whole-Genome Assembly of Drosophila;citation_pages=2196-2204;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731133;citation_doi=10.1126/science.287.5461.2196" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. E. Nasrallah;citation_author=P. Liu;citation_author=S. Sherman-Broyles;citation_author=N. A. Boggs;citation_author=J. B. Nasrallah;citation_title=Natural variation in expression of self-incompatibility in Arabidopsis thaliana: implications for the evolution of selfing.;citation_volume=101;citation_year=2004;citation_issue=45;citation_pmid=15505209;citation_doi=10.1073/pnas.0406970101" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Y. Novikova;citation_author=T. Tsuchimatsu;citation_author=S. Simon;citation_author=V. Nizhynska;citation_author=V. Voronin;citation_title=Genome sequencing reveals the origin of the allotetraploid Arabidopsis suecica.;citation_pages=957-968;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msw299" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Quail;citation_author=M. E. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="R Development Core Team, 2008 R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria. ISBN 3–900051–07–0, URL http://www.R-project.org." />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=J. A. Reuter;citation_author=D. V. Spacek;citation_author=M. P. Snyder;citation_title=High-Throughput Sequencing Technologies.;citation_pages=586-597;citation_volume=58;citation_year=2015;citation_issue=4;citation_pmid=26000844;citation_doi=10.1016/j.molcel.2015.05.004" />
<meta name="citation_reference" content="citation_journal_title=Genomics Proteomics Bioinformatics;citation_author=A. Rhoads;citation_author=K. F. Au;citation_title=PacBio Sequencing and Its Applications.;citation_pages=278-289;citation_volume=13;citation_year=2015;citation_issue=5;citation_pmid=26542840;citation_doi=10.1016/j.gpb.2015.08.002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Salmela;citation_author=E. Rivals;citation_title=LoRDEC: Accurate and efficient long read error correction.;citation_volume=30;citation_year=2014;citation_issue=24;citation_pmid=25165095;citation_doi=10.1093/bioinformatics/btu538" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=F. Sanger;citation_author=S. Nicklen;citation_title=DNA sequencing with chain-terminating inhibitors;citation_pages=5463-5467;citation_volume=74;citation_year=1977;citation_issue=12;citation_pmid=271968;citation_doi=10.1073/pnas.74.12.5463" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. H. Schierup;citation_author=B. K. Mable;citation_author=P. Awadalla;citation_author=D. Charlesworth;citation_title=Identification and Characterization of a Polymorphic Receptor Kinase Gene Linked to the Self-Incompatibility Locus of Arabidopsis lyrata;citation_pages=387-399;citation_volume=158;citation_year=2001;citation_issue=1;citation_pmid=11333247" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=H. Shiba;citation_title=A Pollen Coat Protein, SP11/SCR, Determines the Pollen S-Specificity in the Self-Incompatibility of Brassica Species;citation_pages=2095-2103;citation_volume=125;citation_year=2001;citation_issue=4;citation_pmid=11299389;citation_doi=10.1104/pp.125.4.2095" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_issue=9;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=R. Steinkamp;citation_author=S. Waack;citation_author=B. Morgenstern;citation_title=AUGUSTUS: a web server for gene finding in eukaryotes;citation_pages=W309-W312;citation_volume=32;citation_year=2004;citation_issue=suppl_2;citation_pmid=15215400;citation_doi=10.1093/nar/gkh379" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. J. Travers;citation_author=C. S. Chin;citation_author=D. R. Rank;citation_author=J. S. Eid;citation_author=S. W. Turner;citation_title=A flexible and efficient template format for circular consensus sequencing and SNP detection.;citation_volume=38;citation_year=2010;citation_issue=15;citation_pmid=20571086;citation_doi=10.1093/nar/gkq543" />
<meta name="citation_reference" content="citation_title=Next Generation Sequence Assembly with AMOS;citation_author=T. J. Treangen;citation_author=D. D. Sommer;citation_author=F. E. Angly;citation_author=S. Koren;citation_author=M. Pop;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Tsuchimatsu;citation_author=P. M. Goubet;citation_author=S. Gallina;citation_author=A. C. Holl;citation_author=I. Fobis-Loisy;citation_title=Patterns of polymorphism at the self-incompatibility locus in 1,083 Arabidopsis thaliana genomes.;citation_volume=34;citation_year=2017;citation_issue=8;citation_doi=10.1093/molbev/msx122" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. Tsuchimatsu;citation_author=P. Kaiser;citation_author=C.-L. Yew;citation_author=J. B. Bachelier;citation_author=K. K. Shimizu;citation_title=Recent loss of self-incompatibility by degradation of the male component in allotetraploid Arabidopsis kamchatica.;citation_pages=e1002838-e1002838;citation_volume=8;citation_year=2012;citation_issue=7;citation_pmid=22844253;citation_doi=10.1371/journal.pgen.1002838" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=X. Vekemans;citation_author=M. Slatkin;citation_title=Gene and allelic genealogies at a gametophytic self-incompatibility locus.;citation_pages=1157-1165;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982569" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=THE DISTRIBUTION OF SELF-STERILITY ALLELES IN POPULATIONS;citation_pages=538-552;citation_volume=24;citation_year=1939;citation_issue=4;citation_pmid=17246937" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in Capsella" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
<meta name="twitter:description" content="Rapid advances in short-read DNA sequencing technologies have revolutionized population genomic studies, but there are genomic regions where this technology reaches its limits. Limitations mostly arise due to the difficulties in assembly or alignment to genomic regions of high sequence divergence and high repeat content, which are typical characteristics for loci under strong long-term balancing selection. Studying genetic diversity at such loci therefore remains challenging. Here, we investigate the feasibility and error rates associated with targeted long-read sequencing of a locus under balancing selection. For this purpose, we generated bacterial artificial chromosomes (BACs) containing the Brassicaceae S -locus, a region under strong negative frequency-dependent selection which has previously proven difficult to assemble in its entirety using short reads. We sequence S -locus BACs with single-molecule long-read sequencing technology and conduct de novo assembly of these S -locus haplotypes. By comparing repeated assemblies resulting from independent long-read sequencing runs on the same BAC clone we do not detect any structural errors, suggesting that reliable assemblies are generated, but we estimate an indel error rate of 5.7×10−5. A similar error rate was estimated based on comparison of Illumina short-read sequences and BAC assemblies. Our results show that, until de novo assembly of multiple individuals using long-read sequencing becomes feasible, targeted long-read sequencing of loci under balancing selection is a viable option with low error rates for single nucleotide polymorphisms or structural variation. We further find that short-read sequencing is a valuable complement, allowing correction of the relatively high rate of indel errors that result from this approach." />
<meta name="og-title" property="og:title" content="Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in Capsella" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/4/1327" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Rapid advances in short-read DNA sequencing technologies have revolutionized population genomic studies, but there are genomic regions where this technology reaches its limits. Limitations mostly arise due to the difficulties in assembly or alignment to genomic regions of high sequence divergence and high repeat content, which are typical characteristics for loci under strong long-term balancing selection. Studying genetic diversity at such loci therefore remains challenging. Here, we investigate the feasibility and error rates associated with targeted long-read sequencing of a locus under balancing selection. For this purpose, we generated bacterial artificial chromosomes (BACs) containing the Brassicaceae S -locus, a region under strong negative frequency-dependent selection which has previously proven difficult to assemble in its entirety using short reads. We sequence S -locus BACs with single-molecule long-read sequencing technology and conduct de novo assembly of these S -locus haplotypes. By comparing repeated assemblies resulting from independent long-read sequencing runs on the same BAC clone we do not detect any structural errors, suggesting that reliable assemblies are generated, but we estimate an indel error rate of 5.7×10−5. A similar error rate was estimated based on comparison of Illumina short-read sequences and BAC assemblies. Our results show that, until de novo assembly of multiple individuals using long-read sequencing becomes feasible, targeted long-read sequencing of loci under balancing selection is a viable option with low error rates for single nucleotide polymorphisms or structural variation. We further find that short-read sequencing is a valuable complement, allowing correction of the relatively high rate of indel errors that result from this approach." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/4.cover-source.jpg" />
    <title>Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in Capsella | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CJ2d451I_xzmpbC30KXNPIi1EzUBv1lrSnVy_Iw1RLo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489124","apath":"\/ggg\/8\/4\/1327.atom","pisa":"ggg;8\/4\/1327","ac":{"\/ggg\/8\/4\/1327.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/4\/1327.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1327"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/4\/1327"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/4\/1327.abstract":true,"\/content\/8\/4\/1327":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228Etyy4ezVrkdIDBT2OYXdsIEeIIPv5NrB2wbT8Cc9AQh6mmk6VcRjH20nV8nKeu\\\\\\\/AIcuiW4WP6gyZbfqXVRjSXoOV2f3klaRuCgoFFCuqa3m2EJBWwFgpLR+nl894kTPUaAw8PaollEu1STE0krb59oztpY7\\\\\\\/29l9DbRHir3pcCBKKQr8ZdBhRv7FkWPvLC7C7W5makb26t5arnZXzJZWv08oSTkFvLIklX8DPY++Sx6KqtHtNqx5cCFmlsMDZZeLCrsBYLa+F3k2jQ1Sv2DmtIuroIU7caLUxEGRDgoIbJWL+rfM8ptW08Tu3H0b+VYnxOjbcG+r733GMDSRupoKG9QFx1n7UebeBwKJdiMvMsf2e8IsSHwvtWoRZoI4NrH9vXsRruIqCA6lz5MDj9F\\\\\\\/s7Ia8qJ86cTJGE\\\\\\\/SUDwo9x76gHb2\\\\\\\/2BhtDbattMfAJw695ls1JzM4eyPrB1eXn2+12o7NY0VHoV9dpHNQSBrbUKqRg3dAlX6rRJ8BIYlwbORYDr5r84ykJN7Li68I7EBo2ImNsNPKtH9BzHGO6PuoZTLrtOoWmz3GAql7YIh+h+RZqrh9GvD9m0cOI\\\\\\\/XPOg\\\\\\\/Z\\\\\\\/9mRmIIDGeRILbz4I5krIBYr\\\\\\\/pTbXFl353ZU9Uwnz2TVW0J6zKZBvuoWoXCvhhZzefL5KAhYIcBzKOatKuFHDop5Gemmttmme7dX3\\\\\\\/K291\\\\\\\/s+bb8zed5f5Kji7Z7r+y6udFXScN1\\\\\\\/4w86pPlom+wkFn8zXhOguFkJHdH5Xtp\\\\\\\/ilHu\\\\\\\/\\\\\\\/Ztnknslg0uEOMERgdRmn5\\\\\\\/9rvpsvcKk196x8xqz\\\\\\\/D6h0JdZ6DLWwQrToWAfW73hsUhlugDPOR7oVmuYMH9ROrsGMVrHhJJ8EhJiBBvA3qQMuIfoDj2GwV635ibm\\\\\\\/IKVcWkkE1zcxOkUQLcQ+vgcRBT17SwT+BXLtXaS5Bf79WyNFvb5eiL1QD0D9rRbbcISrXz+ZeyR1eT\\\\\\\/z1KDsouL6M3TitWooBuUi8uARBtNH4pmAvcsWqX6XarOKYkt+nmJCI0Jo0Hu\\\\\\\/ct8qeaYsOIPwxKyw\\\\\\\/OnZnInLxeby12lKAkBt3N5dsuGAUXvzFZCluygdLWkyHHMEVJyjq5SkzCNS53pTSfO6qUM0KS72qd4Z+gm44mvDtFakpq0t8rwYK2gL892Lu0hMWWb7Jw\\\\\\\/GVFDJZ9YhjQZFiGfzqGmgp25EZpbLIYmNzXhoTZmbnMhUXREPp2i57a1DbN3tGGlDT8ZgfQ79B05\\\\\\\/rd8c\\\\\\\/87xcwmavfTMV7SmYwY9uKMMKq6\\\u0022,\\\u0022iv\\\u0022:\\\u0022nZ63ZBhtOdO\\\\\\\/7w8urh5Bng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002267f824e9388433b1b77c1a1655906792dd140adec72d2d78e7e2780dcd7a32f7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gSv0uSWvYURse3AXysAJ6oJS3qylqvFJHqzUQVonPSQ6d+lkbafim7Y0lmwCAsV2dbvGEuS3vSByw40L3ds0FKOX75BKpu\\\\\\\/r9B8xi3cfjU9L3U2WcpwGY1HQFkz+kFEQqS5VNNxlZFKAgjNSsz7viBhutPRCInzwUHjBDZNlYtz01or20PpaRjwX6cLH6JCaOCI6texX75A4sPJR0EhfErzyYJq8OmNWwh3mQAuu3On4JdvzfLCCZuqnW931r1IQk17LhNJdu6qtu5wfcQsx4jr3qKh+C5w5rulwG7oZ7obNUHeSYWH80+gL7sTmtXYzsd3mqNxP9Z+bB1yRdpnAwnHlCUhRdfdAASkXyht0i4pTqxomCd81Qm7aEKIufDwJ6h3U1O9Dd+1BiSFUeTkNlu9uZrcU8PeGX+W47ONiNOOL5zsYy8pNPi0pLil+Oitnhc968TCyobfcmIxLz2yka\\\\\\\/84Q8JmRcrvS1sTIHVmw39hbhrts5jRkX3Z7tQizjdhZghxmlraAKiaNdokIBIRB2sCqnkUDQS4FziF+NGZwwAydMp9wPxDUEr142n+SgcHF\\\\\\\/nhfG+klVAB8TH4IR\\\\\\\/tvEZCfL8aav6PPml\\\\\\\/Flci11DfYb8uRtdd9\\\\\\\/x9TT+V\\\\\\\/9tKakoiCQx046n\\\\\\\/tyAzA1rI8i+5fGwH\\\\\\\/w1LGqkr7IXhAHJ\\\\\\\/Vj0ABRDP+ZgzIIMXaiGTIE7hF5yR9MxR1qcWNu\\\\\\\/o88ZD9bQPfbQh1085Yk3W+1G4JplTGfWsKwG0008g2sO531C2k4BxUFhLV5Z6dnqwEeDbj+ayKyjOuzIf57yjPIZksgvvDFOtpRN\\\\\\\/Zz2dBwoISXU\\\\\\\/KimoqroyQo550q6jdn\\\\\\\/AsDpkVQIW4H0FA8d3UMltN3jLtcxklpiEzGZ8kDV2HoRqW2yZ6p8MkAkyC78z7yjgoPDzlxPj+DiRJiU6BH7D1T0pPd7kDbWoZBbazveX0oVu3ZkoNpsnVtIQssXMAIg2bbOUkInNXBaa3M8KbQk9fcvwplbCNmlLAQVcjVgAhzws6rUe6AvwZZf0U3YMlersAImrYWqJ4pQnDZh3Inf3D81Aaulu\\\\\\\/yKmlCUEHAUGeMrn39e8erMq6Lr4zjJ2Qt4CLnOvx1CJwwF9qA545HjMMfRmMCmdC\\\\\\\/NTZth8Jt2jX3skbyBTo0asrLBhOSXlpCOCrKe0AYwbC13EVepnPuZP00mvkGShhBivUflxQ\\\\\\\/7tB9oFTBPve4dsew1y9Ho781OyYeKeWFAHsIPeg40ioXQflfGuByuVBL8dioC5mdaG98NZi1+tyW8LXVE+Hz1nLRuhZ3FPsn3DNbmajzR2XyH2w0dhz\\\\\\\/BfQRSJVOmv4+57dbnXkrzkObjl1oGFOu6WI6gA4Tm03nH8kvt4cN4kxk75CCkXmOFJGlkoSdC1va8vcOzkW\\\\\\\/A4dlp5x3nQEH9ShvwyLpQkjccv\\\\\\\/6uLaagiw6AJOjtOE8UlgOwF7sU4x3ETGVi3iRlZJ7BvSjHeuo5kB1Nu8sVDMjlfkhgEFUf1mRK6VE4ylkybPQoyx3urGJbB+MREfhEn8TxbJPwAuteDAq1Ot7odg587R2KPJCRALxpORCBmW8u1c\\\\\\\/bAFbrNd5JADpawfTEmDROO4KVsfnS8W13XEApAnVg6MX5mwt4qyaoJSmknAxi9\\\\\\\/2P7\\\\\\\/FI\\\\\\\/H4OfalufP5c9VvK7+wKZ\\\\\\\/b5pHyQCnnRuXCxr0LuWP3UHccuI2ICo9yGAnJY0tacI3Si4RPvlm6NgNBGTTJFVj5kA1dEAxaJVSlIr5+NeQkW5ZCpm9I3QG1Ki6JCNlTWUZAo6R0wJZ625TP582a5vF2xlHxJLEP5rv59Iu1SEUC4m+cfHgzKrZGRIlFUy7TTjAg\\\\\\\/5Q+6fr9rm+b+ylr9LyAZs5HZpZfMwos5P+p0JLb6QvGR8uB65284SJXdJYX7XHseyJvHoRmSxI0VFIOS9wMvm9K8Jd+xqEmdw4pMEEEhS9NXg3lzRo5zM\\\\\\\/0lKJU15cnF7LNKvZfRb8SkqDYAIxxRt8Z5P8LtfsK+MnBVpfFiwzhElM0Wy+bE7MTcNFC11NZ1+XzGXI\\\\\\\/vjjG9tM4kKMAkJlUOGwtLd6rFPXTnXScI4AaToZVk0KVIkz9kjILZK6ug96agvcb6s6UQwLRlRq6Hh46+WoCv4ecVtY5xHD9Qu2ya53H1OQZ6uS\\\\\\\/0mVozQ7407T5lW61vsMbBWLbuzi2+Gi\\\\\\\/M2VlbT+dlZ\\\\\\\/BRIYBvnqs9cVTpJPjsArrHVYuprtxA07z+aAprrZ6Qe2mveGLocQU1iONvmRovaHX\\\\\\\/9fE0O5PxoFCIBJWVP7LXPPBgS6OpUy7cZ6LQfwCsEEKLTpILiiBfmiy9XgBkSxm+FEILu9qN42inSFMLs4widQtoTy40ETt\\\\\\\/tKdH5LlzPG4AYGjwAGoHMTamXGUkx6zZGu7gyBuPdGB67UVVJABoky1NnOzuW9MAhaY1BsQx9GpCD8Zqa+QNvpd7e+WRmJFL3CbMi5arFI2fkhY710nivbSOnneuY3d3cB8zEoPohA4pqoa6AZbq\\\\\\\/3G5WY4AjNeRUMSfo8Bdem1HcJrCstFNtcGo43ai5HokHaLH7rMVz+RxrYM2wHzwqpHNxrmhDGwKmndrEYgpkPkJSlbQhBgMmr4p1UrZm2JfZGSRkgdLBEPRKvHDqxDj2OVkANaEUpYbSbGhchTPOdeMhEDsQbNB6NckTvYpqZMXCk4mf6eAHGo0KrImIXN1yVroMiYh\\\\\\\/fHeMWoz9sFqbz4w5LY3V5gVR3JnffibSbVmD73bPCpfIF1wHjQZa7onHKVo24dyR3t4ZiJIc9koQRcGVGBIH4LFjuaM3oTaSsXh28A3IjwErQl5HhQ4JXE4x16hdZHPhluJjmC3hgoqBkPVVGUAjGzZm339aXbe6u6Zu8nT\\\\\\\/dr4mSHelD7UnA+Zo1aEqMjy6dsXoBNpm3JiUsA\\\\\\\/1cczGokpG4TJbyU0xyLlmI59ViMrCSAWIIVlGn6bgE55ptnNv4+L9VdOvsrPtLWrUn7Ai5X0JPqm8+0VkLavAj841WOp41dUIzUwcOJNOJ9Qso3OhepiZvhOVjeqEH7kpbV1qEKH4Kk7sm5eReXLIpyLQNSExQ1ZBkuvO6EHxEG6Q3xCrk8RVjfeuNYvBQ66Ps5qAVapY1V5VQx9WgN2UDilCBqfrSQja\\\\\\\/6OmYEgZJOm2t2fwNUsWQrGoki30tQ1hv\\\\\\\/c5rotsCNVL\\\\\\\/mDAHqBH9Twe3eW7qkX\\\\\\\/kd0BkoORWXn6s7BRkniQw0FGeu2Y0DG4Wnouc2keRsAJMdmMbHn7sIe9k6r2BcdIyO6HKolGPPZdhcOqoySy550quIhMjrgoxDEdGMYj2D3TVq\\\\\\\/9kn3L2ezl9hKRoZm9tymeo6kklWvNAa2MO4AOws0UrA96TtDTmLJsUhtGHMGjdaFFUbVIQUh+NQ1OokQ+VGUSLbrWlv\\\\\\\/P0fl0zrEmROVe4dAm7KbNb3R5JV2eTuZa87cQn+v9zaDbUpTrkRLE9u\\\\\\\/DtJBonbtzGpu37r1sw2sCEQlKxxk43TNjFTfPWocGDGBNJGGtWVuYMVP5TDnEO3VTWdzjriJTtWC+FZpBWScJaLZB5DALrd8xYGuxRTpXMC58FhwRxggjPG0OwLO9uIjAJkHA497ifxZI3G+nKfzNRL5xLTesZEqxVYmC3AF6pamZaoHzfFnAYEkdyBbsvwKH4n0DkYZd8tU1w1WY6tad1pkje4OQF6sFliKTfrHla067oNExG6yv69Wkc+JFI7+MW5tx56WaS3xanzM1wybHRsc3uTYrvfyVfwvKZF5NJcAlEdhC8+lpA2Q2HSKcDwmuJ4mUV94UTZq6oDOIyZtsrF8f2wauw5cG\\\\\\\/fh616hXiM\\\\\\\/+WFrQS5EF5hSAb0DX0yj3LWyhVOhAWwkrS9K6RW4CVm0FmTJMqV3lwdHlELNDeD94EtDEZeubGVyWXex3sfyd1HLGh4\\\\\\\/Nqfk4ycadl7pg\\\\\\\/20YfBS8UeaDbMi5DKhizNG9DhrH\\\\\\\/1Wqt5FMIJQvUmUyAHisfFjx2WTqPtqrRpn4IwEb8wQj1m5VdoyGn1KCGNKxb1twmK3pnFgSTEa253DaIQd7soYvFcDY2g+kgT+QOQhXGxr+ciM6Nrbkc89HaCN3Ka\\\\\\\/bWUUtqo+D2qpcOTHXVErGYshuAVpWO5k8hX46eMpEN7lo9R8vgfc\\\\\\\/JZd1tTQCNzBF\\\\\\\/tcPR6+bmXHLi0lQzCm\\\\\\\/8ZW5SOHA8huqVNB6gabi3p8Jm6LVXLJyHCLLL7xctTriKtJ4FecPy9v1V5i4ZSKwMrYmnWyzm2WJ3l1Dlb0Ms1nXfomcD\\\\\\\/zhk9JFerWPMG35rc6pOQumH3yLhKOlb+29MHT8WWwuy89JoBv10+\\\\\\\/XFnL\\\\\\\/yG7Vp6s0nugYawpJYLuD6z+FHR\\\\\\\/Us+oLWeQgIY7WSVJOKlTvJo7euLPslRrArNhZKJ0GCkg\\\\\\\/jxAdh9JUStpxipq2Hurbm0j1hPmntl5XfVEfZAbiGjlWB27N+\\\\\\\/sepJJXyTamKEzHLv05LCh9PGCJhjwr6I\\\\\\\/PKu+jl4hDwQREoU4eByIrdOIx97wWfdu1cAA3ZYaNjLQ5H73yTczNZdbQ9ma+DgisscEFmRkrS112r4wmcPKz8WCwCI4PvELUwHbFXbpqBy0Ek7int8Zvm1srbCZkTYbJiRwTO2D9ey0VPJDBn\\\\\\\/1eIhG9mG3SP7z\\\\\\\/i0IDcTiSQTajp11MQSh7oFvgG6hX4E\\\\\\\/ezTh6uIkblVNle7Koror1hi5hPpbJYTRULfppkDCLk1NNwCglsp29Djljw6jzWiPmrIjLeMv+kU1fRbN0OWAzMJv81BkPY4\\\\\\\/VybyClJE15GzR2VPRuGN0bMNzwla8qxza4VJ08ifTSdbJvpdkWsR2LuEucFNyvx80DUrRZbmbfulkULtdG+y0uyR0f5FahXUfNQcQD7X+0BDQZc3ddxe5MJ4Mm3tB3UsjzhGAjdJVfwUyuthpppu9pHS\\\\\\\/rUua\\\\\\\/zRxH48wWHfCqMdkUTggGFoW2BZpItY8re\\\\\\\/BjbjFRfpH7oxZzrWSAU+YWe42od+8F828EvLL18iVAkZzGaWrVyrVKHbxW\\\\\\\/bK6IBRzzEfvrv8AVdgsgmyhKiN4DrHS59G4QECoRXXnnN2ntCs1mqxL8t+qI4EnuUKuocbxBBRYrwZYykjp1akmblau4eOJRV9DYqsFTOVxuBtFVGfSvo2qTIk6Onxpmnw4WE07qpYSLlBbv8CrMwkYpcWqqKnyGR0s567AtfdSJWSt9fMUJdeRI44A38iSTHhJqZHKP+GIR9HdHslYhnGfRkma7PN0tM8b5LAicMXcW9ycRp0YZ0LmmmFL+w+WJ5hUi9P7+0on0bxhMJIeWCAyE1k8B1BjXTq5gdlbrlkEfgrUhGT8ZM9XsbiBlUc4G5NTqGXbSAGt+lNZkD5Mvl7JYfoHY80+G0aPvCzCObfB2Ipfs+KGfNVQQZqnkOls9T\\\\\\\/hqUJQPkO4N59+emgkkVvdLHfKoWVSkRM6QlXhis2jLIKJQu47e3s4hRZJFoJrGYo34XzgSX\\\\\\\/kDsEGn\\\\\\\/FE1cdYjFUnTZgkgTSTzOpfb\\\\\\\/PD0\\\\\\\/PGxloozYpA9F5b+qx0LsNAN9+gfJJlTleG2CoAkxJm7KpQ2arMkSolqIZrGC\\\\\\\/p6dpy3p8IxdDBZGo1cVh6htnW7VuzfSHB\\\\\\\/uBoD9mXvtq5pXOW\\\\\\\/0swHYTxjJX8k5Ow9VFo0qxeWDKmn4JUvkXiTahmLuA187b+jVxc+XXmNpCOSh9XA8g1FXtpwjPU9Qq7NAhsuHO7M2oqYbDL1nfgd5z\\\\\\\/xxyEngXO1IkOIhnB4e5DMi4G+IBVGrxDTdg7JV1m1QWnOOHsPQP5zDrbLTYVVPsWm8vB5plInp3+LCW0GQ0bAcLmAmn1eQsc3BTY3HVmEs0dy3uqKHvuvox+72JKYipEZByiWXhk0IpSBHgTQA4GXo8ag4HzAqX6DAMZWc\\\\\\\/IWlmD6OmBGL+4uQBhdZDiDq68F1eTsjYkTs0Pm\\\\\\\/DDAJm4Mj0xa2aXyLpAHqRp5stdwvYGrMc+6ikk5aKfPG0RX6ctqnr\\\\\\\/3JxPW0lwfKvzLAKWHHBIrfmib3SMNtH+2aLPDJWsehc4opjz3mMKT9pfk4xSOGXLHaCr6S90ti6ZdHcAf7XGmdOZFAVgNW\\\\\\\/RndQbE5UwNoC\\\\\\\/BHDQTOmJaNUt4f3OQlyvWWLqiA2MAE+lNlLP3kb39K\\\\\\\/auXSZy2XacUvNiFBQIqhrgZ2lS56+w3gs1OnJe1tWg8f7aELlJhQ+Q8q5OGNqqPAdVZyJZxylzD247GkNBGj5IcH97eGv8FHH12Ef12R2s\\\\\\\/15yE7WKWSL\\\\\\\/QD3wlj\\\\\\\/31EK7cZejsUVF0CxmrC3ry9HLWMkmgxObA81LHp83Fx09SZE8M7HxRDrO6MC4aZ2zD+2W7YkvWoaiVsWMdeagNuUi3ztJ+FJmLB9SkQ6KK14bgvaBHBaBCZj0x9BKxrm4oTmZmYMBytLjQcvPyet3+yIoXCj\\\\\\\/VbClxGtD2Wr8LD3zQaP6V12O5h65Mybtng3NSepmimRzicqbCWF26bTXJoRJlvKHZApwJRn3nvPXaM6wZKYWqoMRueAuBA2BUHmxw2j50MTcS6YaHaFin\\\\\\\/5C7zKtTlO41Ng+oDenbBv+UeK7lCWRCgYVk4xPIdVQ7cg8EKUdz74wdbH4FjOU7XFFzSA6InwzdDCDUXMZhrbtLukck+EkkV+8FhZFjxE6kKdEQ3FZpM\\\\\\\/rsH33fW0XILo+cffi2GI2T6eM99bAel\\\\\\\/QGTcR0aWMJBYW2yhWRyqgLpoZb1wr48+zF+pCsTfWj0c1sqTX7Gf2r6wrNf52ELGlKYxCOvPpCqbtN5LKWiWti+Xs\\\\\\\/tISs1sfeRYj0tAb2tmstRUN0z9q2dGpuK6K+6lFJrM3Ca5vFpVAhxbGLxugDeYMCmBv\\\\\\\/O9FKsM\\\\\\\/xzNAK4YIJcBLZHTkjXbt+9uIspmLjEjKLjc31hRweAe8MrrZAI1bFdgYDI6p7QLKNy66ry1lWjAo\\\\\\\/08gLWBKlHOyNeHaUqZNwwDkbPkuPT5c5XyWo+0J5zqK8ci1ghZGlywgVsDPA1ysB7U2hcF9Ya0H+VZofQ12ilM65MbnWpNTOST03hdty5vYjR8FJ4MqsRdwecg4qkcJxM+B38CMJQ+773ZSX5WM9X24uva8hmNX7p5UUGqE4JP\\\\\\\/kHXBngC\\\\\\\/wxJwpl0WRv+OkBEcY386dxGay\\\\\\\/a82DdCDxSegEft2XdELlT2O9p4C7yDJZxLLGqzX8FOj1mZzOnJ114Qfu\\\\\\\/G+qgcPphBhqB4G3vafyc4ZZZT8s1lQPx8hcy2D2rvIFKhC9+s5i29\\\\\\\/vqk1kCZ\\\\\\\/Oz4rynqAsMXWkHVJt\\\\\\\/GIXco1eJ+maSvOC3BoFK\\\\\\\/0kN1g\\\\\\\/NGokA\\\\\\\/jQdZySHXKhJ\\\\\\\/gLQI2gURXkwPXIosEtYEX7T3LOgK+UB7hk0CQoWQGpTErJtPu7ZszoEs9i7O7jUZCVIGFiYHJQwuNo\\\\\\\/ZsHgKcyXODHsVE8INeKXKMbdTL+f+w+MES0BkGyq55aZ0CVRGnXVW6vyXqK7Ajjq6Ilk\\\\\\\/bBBZsB0Im8CwjPm1idX3jNt1s9RFwHwcLhqc1QulGvQ8Stkc6M8yMjXa31kyZS5gFconDyzBTyrtkchvkAqNC9BHONxDfvtNh7LvdTisKeCx\\\\\\\/X4qQwpqBZPgHa\\\\\\\/GP4LJSjod7rlUh3LhzVLuxSWJ5hdLQbxovrPP3cUkzchC3J2c\\\\\\\/C3cfLyFIbMU5e+T2FbDoIe7vuhrOe9E\\\\\\\/TGHGxK9EST1Bvg2YLZQHXCukaDtS2igxUnAVJ4ImlKs+sShK8Zi+m3Yr8wsTmbTCC4a5JhHZyQy5sBenBnP+vq1u9Fg7dkZEphlfECbcCxYdVjSekunh8BdMckPlDTYMS6TaKhAjC8iATxdWkIGwnEt6ShPHpzimojhiB9z0XuC8iuSEt7z4Pm\\\\\\\/JF+ANzmfFyHyIGyXfRQv2oZZcZKhL7TpIPI2zPBUsIcwqfDaUShnbLVDkQ0iJKDL6bVh\\\\\\\/Ft1OA6ArO5Pxqdfv9wqTlb4Gx+DEmO2JdpeUIDtf6bip2t+SbLNrOr0X4UiQBxvQkwpePIhKnhO4PoXx\\\\\\\/cp\\\\\\\/FMp9V+7uL4O\\\\\\\/J5SfIU\\\\\\\/65dcDQACDnj7I4dM15v9HL7tQmYETLP6VDvVyvfSHVbO5ztIGcCtDVln4oUJhh+7p6HkmIcU2fPRjMVlv39kRoaSYEf2U4+cRrX6adLlyOvnV+ZnnooZg1lWEpLW3pGKlSClEOGpBQIOGrHvM3pRZOfycFTD+TMIpdaagrzVp6D38Kqptb8le90HLaGJJKzkSYVYaaGp+pscXaGuLCySEtt26hq\\\\\\\/i7RmBGIi3d\\\\\\\/2zSeaFpUBEEfnVb0+uIjBLRUHg1\\\\\\\/47Q6RifZv4hgKVwRWX5KNz0ZIj18nbOYN\\\\\\\/\\\\\\\/nGXAyKe+WlW\\\\\\\/yE9MD90\\\\\\\/Gyux5Bj2agBHsiNGCQ+pJOkSq1sLAgXiFH380uLjUEc2hEaWoP7uB6UhifBnYoSQek6ZB7Q6dkhUaMDdPTZdamiWoFaRWODWdBIGMZxYPFFm2SAkvEBMKq010Y\\\\\\\/swNaol6YcvyYbXu5Xp7QzX137ZEWVtV1YVp4B2o1D3tjfWF8baEPB7u7KjctR9InylW75pjmTNd5GxQzevEMrajssk3Cl0f6hDpixTWhhVxLxF733w6+4Fmu22Wwe5a5mmzkmvwVePUTZCtIg6PewheUag4bXhwbd6je0+bRJn7OMlZqJQZmfdInKqVcu6pweEQB2NLno3ymd0g\\\\\\\/RsYEYxXxmF4rjIzDpoRJrBLUHN2YvxCt\\\\\\\/JUM8\\\\\\\/gE2SGP3xR\\\\\\\/\\\\\\\/j7j+84HbzKn0a6Prh9B7w3RcPnP\\\\\\\/IAKzogxn3AsZIKN1nWnHFMfX9VVH5n3M3nMzviNbDNHKwSqVq95MrjFmC+w6DxXtB4xqgmtl\\\\\\\/1wHjImIh7ttrby3LyfqUYAuVGpSNSc+U6kzn39Q\\\\\\\/szjU642Cw+3nV57u\\\\\\\/v5Fulz2l+khV9Vrvx+nO135jOQfUi6ru3erXWFAH15GC4krKWaVYhhzTLlPBc7\\\\\\\/s0gY+24HVXb+2bNzPyLV7RSZqeHPglStBCHiwBZQgnrE5KQ2jhiseTmhMPSR6ad2oF2w3BJo3UPNmHQQ1rIuUcYmQjXuVCI4eUVuroIDpgh2XjmjKNY7lOKGB0sTDbKM4ZZyQe7+HgY8rQyplQnLm\\\\\\\/6DtHOpbzSJjXrjxQsXfnxdghG8YKNhlHerocuyCyHHIaB1V\\\\\\\/pzuT8Z+tonYUByCZLP\\\\\\\/WJ8Xk2stXGZXeij1AmR1vJCCdhIsS2FVrADKy3Zbs2mn1HM3EZWmaXdtmzkbjQzUvSL8zBga+cEAdzNv827g75jxm1RsSQBuIXLG5DkXt6fbXklIG9nSy1Yq+vwUmDeqFYlGM3VoczGOErzUsp\\\\\\\/v+xw1+OHvXlEiv9JKIE7uRYtxH1OnyrmZ5akRONY9moBpkUbIHKK4ddx2cHgK1KOPpxsb8WkuarkkRxshZfYDZ9P4m3QwrYjVfCiPuTBBWj9jAckM0SAuYLQ8jwjwHs9Aj94CT8Im9e9z5iHk\\\\\\\/EgR8Bo\\\\\\\/33nFEcCPB82O+fsT409nQbsdVAU1GjnopntfWe32YH71kvCqTsuH2hmO\\\\\\\/xrsmPyDZPpTvfSIMTRn50MkihxpIjQqo0tpf8pqaNlTDYAQI8cK2WMCdK9AlUtXKOlkqMOxC24yExX64WYF2E28nY+PVsuriQMtXrQTOLkeVgK6oUFvm81WsfUP+XDRNRV5wmHhRT2WQE9lcHHAfBDMZesQST54ZnaUypG3gVynhUFs5YcqTGUwgnM12GCK04BtQV8hY5O5u7Tskz\\\\\\\/Yamjcuz6y9T6zUJJcdf8iULiRJsjt665LcH3D5GzGWMQBozHp2B2TGpJrhP82xPmqerLfaAUWd6WnK6kDo4xnFou3YMzrQcaBMzLRMwQY3dSZlkmDHI4WaBcLDWanXgcTmx3jDKntOSEal52Zj\\\\\\\/r+1GD5IY+TLocJKD2Wx7\\\\\\\/GpyZW2mhZ\\\\\\\/BTVACtulWLDoEvFtDim5SgL7QnFGkywP3\\\\\\\/l6FOhbnj1qxJcQoVd4nUJeCNy9pcaY9LUj491DFvk3p2TaN7MgKCmhAQdMabHVEJk3ArBAZDHHL2YlcIETJuRZCZJxRk\\\\\\\/4k8OdVeGG127BTIKVMZokE3je3D7wYav\\\\\\\/dlvrX4iHj2WNbueCpv8pvx0xMT7tQgwO1QF9PmRixl5lHfnTf3X0NBc9E\\\\\\\/\\\\\\\/biG7g5xxaYo4GPimqpQynbZEsI17u0J69NCgZ\\\\\\\/kmPS7K9y2N1sna4klG4MiBuccenVW+FmZ1NGEzrJ5GddNZPAbpEsu6TgYVLtP9SW\\\\\\\/\\\\\\\/x8maTtqXDl3UjB0poO8HDaflmCD4bSUtdtveNatZSCdLUIunSuEckpDvvjYwU2d3s+9eJp8NOJqTM8xY\\\\\\\/ySm3HvxDTm\\\\\\\/vv7Cr1NaVedBQnkv3OPEYN3lp+93QTQ9bTQkv\\\\\\\/p4w7w1A3IPbEF8tuPrYa1WSj\\\\\\\/jqLlRegcyP9kE3KsjuMHdrFB0z03\\\\\\\/UaE6VYbx8+WwuSKRMllMfyi\\\\\\\/xAMM7J3eEsFmpzHhXF44PYfdjtUD5wzgpdKgmz3PS+H\\\\\\\/4of3RRHWA5fXVSJdvAdvwIWzHgkJBPY\\\\\\\/jknYRClXAXl8EtJXCFHBOoxIAYNfcmf3SaerVlJAnGqa7J9xlwv8he1raL7YCp0bCRRaKyEZHeb1itEVyUtzAw+vCs8AVLBt8qd8o9Y3NP6\\\\\\\/8m5hLVz4zQBeWjJeYfsB3oPIkI0oeiBRiXZxO0mog1Wq5tpe5xOOqODveRiCkasA1wDvJpwKo425siXLrTqoMccRy5ZvngrzIIBT\\\\\\\/NY+dFP6Kh95J+yV8bqO9Ij41mKJ66iESQGAfSiLIMDs5x8ho2MiAAu9pcDXUN39A3hhc3UPGkdf9R3+n6ikBTHW3ZTTzWdQWQVv4IeRhNe\\\\\\\/RZY1dVE8aPhlTp4f9g4HHL09aplwCDo0q9EpsaRvs+o9DisaSoifKnl5JMNXNy4o7jCCyjeVGjU88GpU5F2TTJqFmOE3UqWsCyMrlMTZicvtRnsMBv6mQ0EVEDiCsryQQf\\\\\\\/I\\\\\\\/gr6NVQNaRwKAJUdWJSCsMyW6GSIDQf3vgMS5R8JYH1nIMy5kDWANna9QvhA+S8r2WoKjeWcf2PPwyey91jbLeNVHX3SI7HUIqpwB9K\\\\\\\/+Q8Q9czUeU0aZI6B6DSkwQgZldTTvMU2VGhQmLD+OaW1tEFNRNKabA05M3tI3wm+LRCDkMDU1Lc+suvq7eGTaFes3LMJtpoDg19r8ALcYq7HNus\\\\\\\/1p7Ymwsx6ddQVW1Z1cbrjc3MiaUBdZSKIT38vNnVeLLITeFfAjKB1MCZzco0OIOHDT2SCNs+K+qTUkRUBHWtIdiSB6x6\\\\\\\/mFEB6AdY3WAxhiHuMiG6F\\\\\\\/sv8Mxrl3gKV6bCyDpJhIMHAm+bfwpS0R7ThjAouxABHnJE4395e7CatXzV6okv8oxnevDfuT+UbX3\\\\\\\/GJsj0u3L1ZppTDY2jZTtTFpE\\\\\\\/J0YtbnfRjFFwW1k2c5w12TaKlUsqKw71k8fERh45Zma9oaahPejAs9jSSak78TtSdipib6in2eLa36jEzD3H9DWduJnyRwH5gOECnpg4I8J9ZcaCySusBlWtOYxu+EzSCwBxMASKIp7gFSql1WAUOJ8zjZOiwgEAzLW6b6+jehkijgleUODWX4ADeKc4kPqp+UVnvb4GTzmgJ15f\\\\\\\/Vw5uYRp0urElgmuZr01uQWIbCm2siaLVwxIX505lRI3oxaKArHxADD63ibI3m25PufJhNNAljZKt+7xawinkEEkaRdjoPmaPtA7fyzWGrweq039KWmTql0D5KpAA8wiSKiRi5wmwT6gfNLFNPftG8Q1leZAF5KA7Ff4tLzvdPBRRFPBaxP4F0xorMCRKSW3Eg3aM7smh5ldhd9wusXRgAWiRePf\\\\\\\/pzSl88W8ohUDtn8Q4x1mN1+cSqgLo7sYAWjPII0LAHcHz\\\\\\\/XfBWc5P1OrLBGFM5bTPSwGvw4tF8W\\\\\\\/MU7IQIULfZrAAWsPdo2F6WPFIF++JAkN7+R6UsfeERZOLmEzGwELnShA\\\\\\\/GFq+cnpfMNWgQbITkpoCuObP5CcjCna8ndlEcxGe5OwrgaZCqtfLP4menKd8fbBztsNOHQDjEGyalswjBM6kb8t+iMA+JbLtZKCyy16kLHO4zGFiHODSrDptzD\\\\\\\/Fo6NMz76oEPSIhWVQlCOTnKlNRB9vpwyFYoA2Jb\\\\\\\/oPNBLtsbyntE7hVlnEZQUkusfhhYDg0PG2adMh+ol3bQgOwtM\\\\\\\/GpbTzDAn\\\\\\\/tqL0w0rjMzxLff+ixD+\\\\\\\/rHVhok9PAB0EKBi8YOKnlmc\\\\\\\/JuWesufuQ88vCPlS62QuAQfe7+sW7ehZlPJqrETmzaICijFjZoQWbp4SxwpVQEefjk+lVpJ9tgI10Ct4Ia+HP\\\\\\\/wdtZWhnGVXnL6SlE1k0GNOiymzSfpX8x9HK4im7F\\\\\\\/mRzSZ+YUatqSl46ZfGW1NHEcYAEI5EfcTW7ApOgE3WUV5WPXpnMQu3ul3nepuawO+0oZZN3Et0R7Cs1A7Bm2+HB8odOcqhJ0dgDtYuIVIJywH5r93BoAfnv39AAASOTBthG793a7DQQmy0P8bttUIyGqdcjL9KROrMvDfFmBDzakDixyYjmeEs+iOLeLdzOA1qlv2BkpD3\\\\\\\/qzCvwOO2cAJ7YoMf5v5dMJ12R6S95\\\\\\\/TNPCXyyDS7TcpvVJYVBFZ7SqPVVUTziXqYEb3hYsjH1CaJdwekGSM7AcOpUGBwFAzKS2bOOeissPEiOliAGl\\\\\\\/pcEOlPOaNj6oc57gTuUsxBx9vsIQ9JOCL5se5y+yLrxDcA81VBubdEhY3gd\\\\\\\/7xMmuzA0hVZw6mBGYFNBpTPki+lM81iUT8XwlsqScFdjuHooYinPcMfT3klf3Ad5+CttNu8KWPcOWz9uSjM2i8WnBiaL05iAA\\\\\\\/GN9aAAkeanUgCdslb6kG6tPGiSuAIDu8kx\\\\\\\/9oC9iKDdWVuUOJBawdKzkrGupMgYJW25+oy0TJj1KXgsVEhRnSAtgyYWlOBGv+bYeBkU4YTmpBulrYZ2E1Akcat2IrTWSjgKCcxmFAubf6AV\\\\\\\/m9eH\\\\\\\/Yx8tEVNecODfhNs\\\\\\\/pEzlNgxsy0SlarL94LvvbqjtuMf61a3yJItGIxPN5Sh1IJ+TeYNFpP1TIDCgrTBZqMMf3Y1Hbqxbss4mnCHZtz47aJQ4TdwOMKaN2p9GTEZEoFo1V7V89TUZ63Gt4GWr4pnHutoICgGVQENhL\\\\\\\/c86yn+QButbjnbKwN\\\\\\\/FD04hHaBGPZgpGr0UbpGcc5F\\\\\\\/TWR7Jk6AjtRo\\\\\\\/9uuHhYbQW4HRyLu5vdYjKOtqBJAZ6OA1\\\\\\\/1PU3k8eq5NTAD38aFaURQlbmSjAZZ\\\\\\\/k2kFdJdHea2N2SOm+Iji40PWTRZcXuBAHTAEwC8Mt2IpcZaUIQnjEPjEvHgJRbNqqoaP5rbefXp8OR6iLEAA2auniRCn5on7Xca3YQ0Yk\\\\\\\/Hvl3C8IhrCxWJ0wjWnoeKv\\\\\\\/RFI5\\\\\\\/enTaiBt+nCLf55aAqSkTH1K9+0KjlicNAOvTFkXmAiJjl3NFYXva1CAVyni2lJVOKiZmc0+UXo\\\\\\\/vrbCQSYOFmV+NmmzLezW0+jB9Pl+8YgOTsfV110s1IileZ3\\\\\\\/Vz1ID6FMAKrsW0\\\\\\\/CHV80tgVd8GeJ6ikU7WVuqggt13kQxcIvd57CDDDj8kqP9+GXbkXW6WGTS3nM4EgYK5jIrtn+3rAJ+KQCTwAH86z+pixB3sgHPAfTKxGXcwNap0b3Fen46ndZWCJRroVCUAYk9pw9JYspT+h0gc3MIze8bgjVHXbiXRU2U1OwmO5BiYEdV09zRekd69TiARSEup9NCXKGjl5VAXTDGaLoPgARc5xSBj17J15oc2J9E+++crd0bQUTU5YGBs3lAZNHZm5tVawduhNWM2BYdBadajuXZMULtnt1IHyo0REs8wBuVXOnfm9\\\\\\\/LAGX6vGpOYNDtL+at2N37AH\\\\\\\/ZH\\\\\\\/Cyp9jFIIcLq+kKMvIGWPUDF2YhXF5lfsyQ\\\\\\\/eNxJ4OuFDIBODupv8MejoERCB64i35w3fV81qqBo2c2BpCRWMpiUUA+N4LjnTDU8EwVPEPDR+KD6tShYEPEuhYjCWouUV4vMV1S5yQU03hPGjWWo7gWKf12ykXI74Ni3omWSdG2VjKF+q0adoXBDi6dxvODHhn1ByVNwWNURvSHX4zx4jRT9cmbQagRXDXQavQpuLLYE9hIe5pdp6nFLBFqXX74\\\\\\\/f8Fue1DqykYuXYjgoqWWJghAfN5NWVNDLqF3kU44daZI3Dy7l7RVNmZzaXKXgj57yiNFto9xGZOe2K+mXXtcD7Btwjvfea+It8txYwj7H5p4K5daUkfpcc++nuTTbYvz53oFsOJrWI100+0Xf1E\\\\\\\/D3Nx19thUxrTjL6pZdtr5hhsNEWr3eLdny1jl2IIrcnmA9KZfesy8+ap2yumIpbm8JKClnidf2FIQQ5Tjlmu4UygYmh2vjQjFL7oVueWvy4STgu1UrsAnBwCxlaS8EDFQ92M\\\\\\\/e5A2Znp\\\\\\\/+uIcDJcDmf0EmEW9TRiTYVk0xm8ms7R9aUjqHcPeHSv\\\\\\\/SCrqCrWsWTuikmkZveeGnpovgiCZ1XbFK+KoL9FLVb3I\\\\\\\/0bRq3r8ifJT9f+CU47xSi\\\\\\\/xRBVAtglq3OyXaQCG+ET0lQ6o1nXp2zmljQAswo+VIUHkRa6WRzj2EaPfL\\\\\\\/3NMenHPR5zJzO9FeenqfoJIS3huG9FPx63c96BZQL3mwgw\\\\\\\/G3aG9FJ5PYLb4dzSTre6hkffOAuF+ycmCRcKHh5GWY3V17Bvb7IXf49MG6\\\\\\\/zXE+Ctc83arKewrcYY1paeaZwJ9h4rbDBoZo9+tLGLHeMoySsRy3xNzpvNTFwyC\\\\\\\/Qm8k1mHPT5IVigbIyEKU4wOWGj05a666m8hwt6Y6oNVPv2wOzy6EgDDycvzoIbLnEo4z2bOU3DLSvc2Zeuk4edZD9DXX40\\\\\\\/a15\\\\\\\/\\\\\\\/\\\\\\\/suQXIUW6Fm5UlV2JOIRXu\\\\\\\/eL22KgFFVp\\\\\\\/yj1RV9vJo+NB8gWxNGPniCrj\\\\\\\/YMe91bIND1ZRoCrGSmu9nIyZE50WDJ7uhxU2+Nk3YXgBMEZOFLHJIC56SnpH3sHzLJSOF8i4QsobgswyeSbcCrnZvngkO52mKbXfCaKifOE1CttAUQVojIV0FamGId94Fji+RW0NfjcK2dBYvWWix74CCBRTnfxYHhffhYLApy8j1HQt7R4NrRMO3BksL7sWd\\\\\\\/beP+udQU1NMV7BAAk+x5nqOIlftv9reA4X8xlHOqQANNFWJzA0Q0\\\\\\\/m49b5e457vHMl2nWIV7\\\\\\\/mZ6+nJdzc4\\\\\\\/eIpB\\\\\\\/fTInI4mQhP0aE3dZd8QvJ+E4FwrvTS4cVb9z8jJZvLXvEPfm7ZnTJVp60OAVJ5EWQHr4NEKDKqrHQs4xGzZL2sXonuCdaQBvoTuL+UrNIcVUg13nz474vYVTTrQKsYdSUye0eocXp1YDiC0MPnIy8x\\\\\\\/2tXE\\\\\\\/NAUsr9i2zU5iR8mxsurMiaIQm+2ulYw3M3OY7gDlfZH96TK8xvKBi\\\\\\\/quCwJsaJ170tjTu5icDjMz77\\\\\\\/v2lLrZ+a2TGvzNFRZ9YHm4pPReYKDZp8VvfLdIyQ3iIOPnJMC5OeGYlU5PZ9ajBJG+4qdfIHNm48OG8jLsOMVKtUIXWYQ+HTgRG7joQEG\\\\\\\/a1K94rTz\\\\\\\/IIE8nt92GamfIjwB1tdq1A07hdjRze+RT25+DNp0TsEuf6Ievl\\\\\\\/LzFKx8nCATOR24fKukUILmj7AsUBv+lxSOTn14\\\\\\\/TW\\\\\\\/ivJflRQmLVhXfNfDGwmJt2u5BWjWkE96oB4OmwD3z2hNEt77HOx2UX1r\\\\\\\/TvIxZgyPjdzjPEFLvvzBSUP70O2P8cmeVePDdgcpMWbm0P1DjJ1ipQLuZz4o1Tz3nFMsJVUbDthKsMadrHa3NEdmSmcHqtb8sD6Eycb4KdmqRoeoW9z58L4gg1in64tbjhxKq9pPKHaUK7HWYtgITGgeFTjCR5Wp7ThLK3Ih0scBvpxdTc7+2+LPOkEjirxUpcJGnCE0S5C64y3szA96e4BpUw5jmocp\\\\\\\/RZaUwvve2M1gXKWIVKu\\\\\\\/5OKNT0sRCWtrH\\\\\\\/7OyniCIGm\\\\\\\/6lZHD3RcRMZtRUhWbZmVAEtlv1PkHCO2oyoSKkwzykuFQUqHvqH1By53Iw0sfUaL08MpomJsvR6\\\\\\\/aPUrhLzeGUif0r6g8eG5TYxZ2Y24lF0oxwi\\\\\\\/k+Ue7+oXrCCoh4AsJDvRUMbYoRSOTRO3+TJKbH4bwDfJMwjd\\\\\\\/PfU12GCzKO7BfxDwOgZ3Jh\\\\\\\/WDVKmbx055OTGS1URJ7gYIdknEeoE\\\\\\\/fSLy\\\\\\\/Iz6+DUUaf8OFafPNLDk2XvAqOV1HaxL86lNPLGGxcCrqYBWGIhBzD+ys7JdkZsqgGRJ5de2j\\\\\\\/s1AqA21kqbHWcLOZV\\\\\\\/y\\\\\\\/6q6gVi\\\\\\\/uC+msd6ogV9AVYjtK8b40XS2wVytlqG9uf31VNh\\\\\\\/EGxai5w9GDCfWpOBqYB9oJEtCngXWEo82x8+xZ02CPNULhIV\\\\\\\/dxCZ0qqA+6wZrG2sCjoRKdWZQ4A4y+komzb722o5AzqKKsOfziM74gESQuFZ804dMvTm0vavxTp0YhVTXzHCU7sFETiDPHCAibgVbEiB9f\\\\\\\/S5cuwePcECMEy8dX7bXmtXPYMkVd44yWQeq4+PD5uZb+iWPNJCxezJoI+Q5jmLWaSH58J+JF7UobtpYfjdK8S65kRl2a3BoS+bqTS\\\\\\\/S\\\\\\\/X5HVN9TJgSW1SM31Zhq6rcZS\\\\\\\/m\\\\\\\/ACBBCR8FoFID5TTwTbiKOvSPufsnRePxbqqPkar+KXfUWu0OcM+zyR8RUOWlkjTli+T1HFbm4OyKsipILw4ayHDU\\\\\\\/\\\\\\\/GZ4Ks4IG4uVaFi87MIOE9POFNVotZhk1gD6+3Y1+r1OX\\\\\\\/EdtXhkzvW2TezmitSq1LxAiWWxShbnOEJLtRUrHyZGB39iups8ZYeyviCqMpqhCS9JDYYMzNNoRVTGxBHlQ5fZ+Ce\\\\\\\/QbyKg0MQUfWsGye6CQlwjmoAJTURoBVNUwoO15ucteQz+2Hv7Fjd\\\\\\\/JuC4DND7XZG1uZ1tn5ACGyM29MJhUG+dr\\\\\\\/yIOlTNqDyn+YvduABrPWIdkrvLqb6J8aB\\\\\\\/pWK0MLZFV0Yz4imouv8ikETRNO+IeoYGCOpVDpb5EAMmCfIhHg8q\\\\\\\/c3xayG1JJcGZ9KmO7CoErXcfWYHBffrRSQfvTYSV9JYnM619+wrNWHb5qoaAZEmlXov56BR63BHw2z5vKxrxeWyjEqJUYtQR38ymUdWrWdLBN8IzHYgp0WIxnoKYKfoVBM9t9EtRxw+NbqBV7zuDzEDqytP497wT0EkN\\\\\\\/OCt33a4UYPdOgw19kovtwO24Gh\\\\\\\/+80ENlrn1BUNa\\\\\\\/GGWGkES7N6L5rDb4IjPsU5e8oa6QsYo6r58hzvXrkSQ9XxLoH\\\\\\\/AY+o3J9XzAUoUrY3EEfAeSgnphB5e9QQScff3wNiN6j4dQw2oJpKBerDbw3LyxF8JuVnI4dvfBtc9Wmv62K1yNFXLC8mSgMU0B9G9hNwJ+qNKVB0mxjIj8qetdMwhk19x2RmJiAlP+5VzMpJv5y4ZgrlV8nWDFOZI215pLyIpq1sTxpPAM2dLu3ZpwdO8rNAOU3U5dnGI8MBLsi813ek5V+jar8aEnpVXvQ0DKkItTSPlPaP1CC0JpIG6f3mWUQHJIpQP1\\\\\\\/jjNRupYJxAyTk40YFlD1x3mXwTjCyAS9q1jhlswj5B25WxHqPc5ekj6L6cmeX9bdcXWlqQYyx8Q4emBproxUfs801L6m8Uwjd7xJ1NN6MT5\\\\\\\/OSYp5bueNf3WyNDiGP87w98xJWXDZE58dUSaGXr4ozSRUsY1s2Vr9LQmEtLqV2DqesIkcD9WjAbhDf006zhNMRC9jD1jPdq1UL7mDHrVQgGBsLXz7V0x6l9PEXi7CpvqP7RYhUYcdKI8vJ0xMInOkgbehJhltTYnnprhTCCZsWj6U6hc375fM5fDypPQRj6zBClX5375bqugB1uiauKsZLrKDUAlepR583Twu1VQ8YIg\\\\\\\/XzJjcCi\\\\\\\/ggwhhL1fnzxElOqjjzeH+X95cpvb2RKgbb7ihUcVvKRXujf+5tpytapen3KrNdJ43moCwDJgEmVGhUNShMlBZrZ1yPqLPFNGrpCJK9TEkhkYNEGmRQdUP+P\\\\\\\/SeezW5ol1Bd5hW1sk0Jkwu\\\\\\\/eCIj3zEsFjHfA8cRTVq9oJ8dlMyiRSr\\\\\\\/bH+2+1M7DZR2RNe2yv09XCXueF6IgIxTAdAixpaTb5G67QT2xRJJP5EQzhCANkU97A\\\\\\\/U82p5ZylGr3bbQ3LXsjTEItvqVC8V+soghjltg6OUWTHL2j\\\\\\\/rm\\\\\\\/k3lZKRhBBgDgQJSkI8edmdSbVm0PcQKLv6Z2RvcWPEYAbOE+zvdV8RZAuu4yCBFvXFNwyxqK3aEBf0dDbWF4D2sZVLQm7D5ffV6cObFt4WnJJB\\\\\\\/vpPyAKmD40mEAp0DXbfi2WQTF86nfX2Y0D4jml7fS8fhDX\\\\\\\/0Oosqe6ahyEuQGNXvtrDQbOzRXoSQrLubEIW3aNdXXKTDelwabPLusnVtaRqSghqalwCE0WkX4uhoNKIFzQhvrc5GOzZfTuJIuMmIU7lv3bMPj0j9kn\\\\\\\/uniCKqn4e1uW0RfjlTQTbyREspZrkBnyz7xP2QWC4d1sKLDHXrqrDqipKbMp7hWS+D\\\\\\\/ZHc4m8wE2zBSov1YkauiXBGn8JnUUXJUzNn3hdf7cFYwsRIxSAIzoQSs8+BtLI9y367XTbBYHF73Go98KWDg+4MG5EKlkB7Q2DV4M4uvn6rW3j+HcZZ6H8IaDbZUh1Fv+82XTuzTeUyhI5ilz9shFCpjb8MpzygjY1E7I42JPtjTQFxK8DFyYugNccby2rMR9w+4Rb9DgXhjqeSv9qrs6ZGdmvmA\\\\\\\/LQ3thM1d4EVNdhJonqT0o6nrcPLiFdUsDa3zOmm00ylQfXWv4Bjp67myyR4qXNB1Lz\\\\\\\/g4MIlLOcvge5Kkmklbj+2VweY3St9EDNo5+XwaWKU0cCtqFD3pwn3exgk11kcR13kACLSqynYpAbR\\\\\\\/bRX\\\\\\\/UAzKRKTuvdJNVBCj+A8qN\\\\\\\/tBFbg+8LZ90pJPnH7P6Rn4XSJDbkzHWIKvT1cy\\\\\\\/9tVpzdPe2C3HM8L+hr0T8i1YAou+SuC6BkZ\\\\\\\/gOilqUq3DNw2M4Xqnw7mv6OcfSnhnI1UnBcolK87KOPWbVDh3ifYd1mKP2FFVjbuquVcUU3rlyneiQcROdkr2bW97zU7GQuk+rBTyll4QEELdsKqQaDgCg9ZduNWLK3vAQ846zPJ5CxDhnswUOfMs00bg5p0NxgnVsORO8nXS7bjQLdymwpyZvzbaqr1Jc\\\\\\\/vm9nmhNo\\\\\\\/WGXPESu+xxphXwBBem6X50huGgTon1\\\\\\\/rYaMOMXEK9wVIo7jPPlCEzDJiCu\\\\\\\/c8hyBTip8yHsq5pEmsFGKEFiC7v2FHQtV2lnUfGfHwj+OorFZUYlZfji612rsSchBPjhMmLQuJCEYEgsREXnvD23NRBYSpruOHHpcnZ+mbWjtwYSY1j0L8utOrOw1+ORhHOTwIAM3uWlGAsjLqu4kgMHdNqlkEVC9dp1KVrz\\\\\\\/b7smreseY006uLtfhCPYg1tPvIHc51DKyVBhAiU9bzMLEx8eZlWkx9A1irWNTQrLPSZiZ9kDWHMTAfB\\\\\\\/5H3i14iAea48rkGheFSEc\\\\\\\/zYIfd2A9GWoMViCwnHWAA1HQsPt6N7Jl09kYNeKaYSBEIuJcfKIRixGcB11pzbsUEIXpbdbi5xd1X6II5BMhztQ1SA++ZvbmWb+cUSpB\\\\\\\/A+xNHx7SZr7mZ+aU7SV1kvW3MsM\\\\\\\/kP0dROYE0H98ey4wKug8+DolTvKyd2ZQQs5DVxFJh7Sq9kESCL0r23S2JmY8XXIZbzz2Ik6HdSB26r0m5M\\\\\\\/aMsVuEfN\\\\\\\/sSwwVexgglJ29OoE\\\\\\\/hftBhqenaSYsEY2sDbxwDKFfP5u0xh8mlrD\\\\\\\/VvEuQKD6K+P4DMloQlQDljnWmN4ys7o3JJrF+ckU+DR0g12KKbC1HA49cmjbM51W1vEgFWZG5EAdwoLJUEmcmtHmBpSJj8NAjNecRZSODap616elyMhXlD29GlPQn7PaG8VcZCJy2Mhog0mO7vnQHWVPUIJFjQCIc13bunB1Q0bWnRlGnF6zx\\\\\\\/x754bdM1GiHGjpwwR36QCHziJ2CoQ1Td5\\\\\\\/0cDALEsFiVTv6EQPXd4eM1cKoKcyamKWPzmDGBTC3NiPHBL2vKNOcz28qZuykEAogGeww7LinUUOl9wiE8cQErQRyYjdev8zA2DFJar2pYbpOvCLqo3PSsm1d6DRUXGnchOimSQK9mswR\\\\\\\/NM9irZ2hvMxaJu6inq5SylHqtu\\\\\\\/AcCy1YiYhWov\\\\\\\/9vokWty4NRBcyFLoweHQw3sSl+o+voEs3tFPBlb0zZeW0gU3CihtpAwVE+Mj0rLSTFps2QK3gGp99A7qVWSZvqCdztuAbCMJBjS\\\\\\\/c+eVLaTFxqdcjw8J1TO\\\\\\\/1MIXu378PnjCW+TyFODpP2FmepfylnxmYggCdsSbpPI63BAmlErCwC\\\\\\\/EQ0DUOvYi5A0aj0shVGTcuk792G9\\\\\\\/9y2cKGK3DO2I2HXGs2+RHaETnKGSHYWdtTzWb8h\\\\\\\/YW3ifdRrG93dK7yiTcaDSlzyWWYkIu+9N4i1nDmzG0vnoRK5zDBSfEW933kqIDlWVfNRX1IouUM4B849Zi5l2wZA\\\\\\\/Kkxn2v5fhgFgZ0LhYWY8KZaiJN52TBroVjTENBJBOZnENbTgxvASZzlkTgap16NykNY7BhBuqOMku4+8Z+B3m0+FJF1brD+6vsHY7qcJKt\\\\\\\/7K0WqM8eRrvY+xvLnQDhFfRhzSK5mDNG871Sh4aAGKhMVS37UiYC4mZ10KyiLzC1pQQ+qXkGOxZA3IrgvcfNCnwPSjH7vSCGZI7r00dWj\\\\\\\/AfeLUu9IkDM3ad\\\\\\\/FPOh0Eh\\\\\\\/D\\\\\\\/WIOwT9ORrSpHg56Y8lVFpbUnBLUdEfT+WGHMEAvd9Os4U\\\\\\\/8k96D1I87qEvDqFmn1pcZ2lxCIL8Z1F+nZ4UU\\\\\\\/88MWqUXYd77AFZePwM2Dkwo3lLEkvXub9wDcwL6rqWsnryOCe7YhxGWN6ZND1\\\\\\\/nJXXEMYDij1EwOuJ+bETYpL2rOAfL0n0IzN+gzIzCsyeAx7pD5kxZBz8kGlTpVX7h5GpxCXVifWb+AtLx6niA9LGFh6tXUyUt5nKhm890VIqZjKddfUw45fleE+\\\\\\\/UuF+X0Yx1een5q40xS+T5BufVehdFLWuJoXbqwuzZ1CDVdxJB8XZ2g6wKlzVPqVdJLAWT49EFRd1eJx0C3PI89bNaEJ7mV4hjaFspbSxvNtBNIGfrHq7ljlKMqbB1LeoPG0X7WGQOec6dDR+36X4U2E\\\\\\\/BwL17OnsCIPhPqiGTYXT+KQLFPzCOdcv+Guwijyh\\\\\\\/hL8nmaZNNLHbHKld0fkXd+nUcrGelABEZ0WbK72fnaNumzwnEDvRwsL\\\\\\\/8QUJlZe1YTEUO7SXrcsOrHtMK+3QT\\\\\\\/550fwNGhj76LRei4qeLypbK0pTFO7ODpTcMIVVcWxxzqsRvooFgAGBNRZuVsFcN5BUFlZWmyFuj2XvihV2WzOY9qUqVb8RirIVhkUiR7x3UczH9ukhgDIIxeZ8OL+mrpekx0gR2bENa9E\\\\\\\/4wqfPqVRQKO9zTECtZONoFqwGOWvBNBfhM18Xg5PzssSXlVq16q+r\\\\\\\/dDufMEohaSYzcPi\\\\\\\/C0eInScuuIyS2IBm\\\\\\\/8LFH7yzU\\\\\\\/9woCBHDd8dcrYDZjP1Nfr++K5rjZH2kLhUjETV\\\\\\\/FFYRCfGVgExgXCtj75O1h0+gqb65eAQ2L1FbpyOsXHxcmV9ZLTobZ59XY\\\\\\\/ys8N+2d81YoCv2KPlXXFj9g3qcFKipHviKzOY6OrO+vMzbXJ8YdVg+WQ9cUVEsOUkSPdOr2fpZXqopai1L+Rzl6rx2s8tHJ1iTacOgHybfRGshxxVWEiyyxUq9lqU5OkVtPwilV1Na5oByi7th25krjYpjVmoxi\\\\\\\/gxZcVkwZN6mWQVrqVnPh4aJE7AHrQN+x1+W2HSUg7Jkhz6AruoBPzhon8hs4rZwr4L3jz64b4iIOHIqFXXMKsyDokOevnlU+lPau5vvz2FPY8XU9QgfsYYtsg+fPz0JlLnZMDXzHg0k1+ZVSuGVTZS5NaachCsbLCtiL8RNrr2evM\\\\\\\/b82+xcVVp9vxkHdaPXyLPtis4lwb04AYk\\\\\\\/VT\\\\\\\/AC+TCESnFNi9IUz0o1CDo9GTTXwrRS6+WeyE8v5F+eYp63epNSB9HnojiEj7JvZwJfZdetK5FAqTTEbPwV1tB6mSbIGYlxS7ySllYSuioBxpgTZJKbnZCK+eb8Ub1ZBriJuoZvI3ncNe\\\\\\\/YCgoO7AnUOfzmeAmP2AfcutkranNGWN+kPzS7ntNV2o0XXkpTDlf8z\\\\\\\/4rm2DhATJKbKfWhHWzjJRFXgFIQ4TXO\\\\\\\/Qal6LXJAMlQeMkOnAxDpbujLgOGEccvhorvZcQaIcFnE60GBqliODeCeKYyP00+g2zHzHnqsEuD4FmxzwuT4uYywbSj5mqTjC5OL4qGgvOC6HSvQu0x3dmgAbdyOM2NDVFXNLS9Xso02o4Z\\\\\\\/DbSqwWAjG8XO6vhTnLcUS2uc9DK7gNSwA2MZDcU22kWLJYy0RP4Bw+BO0uXSqu90hTBJeMXoG2qy+Ls080Q1MJyxeCvgRWLLJ14GxirOBACTs9T6uSS8tGRhaAVgGstK6uV19DyKcPteeOIFATA\\\\\\\/5lxIQFTCjzELI\\\\\\\/rU7wE7h1VJlUmxY0AhOTEZZiQxc7jxY8LbhA9tcGk3xvRmSvl71d4OddNFOux4YFIw5ZNz2suLlc0yKoy3\\\\\\\/TNkrP+a6eC\\\\\\\/vfHhGOs5ecZMcOSfY3uYpsYTsV8mhy9BjMasezP8Hx9azuS4Mt+O+MybJeF2JHvF441r7bJ6\\\\\\\/tpNOrE4D9xSul1jX92wLWdlabcZkJwKScLjmjDVYPRP1F0q4HFN3bVfLV8OuoL3gi8AGrKjWSA035FSL3xfMwhBQrBM+md4IS5WSN91YJjHWdOZs4tLMpeB0zP\\\\\\\/GjKefh9pomWgX83hZoNKzqTvkEJIsM\\\\\\\/5i7\\\\\\\/8KEkn2y+jLFnAEl9KZLq9WnZn8jKU67bobcQVBssyahiKKnYuEihnXQiengtP+ggWHW0IHA4gsrXOcS6kDWl7Yfb0Pu0lJOw4ks8JGOIOvvgDIZf4383u\\\\\\\/MdbVe6mcSJLhYWUEOEvOMQTnrWbK5sOjVdvbf\\\\\\\/e0eezChV0pguf\\\\\\\/xtPLEgZ1EciIwCUZkMrJZt88vqN0uh0oGBIk6j+Mi7nS\\\\\\\/z1hMczy1qVt3uJpPH3PBvj6Re+Se\\\\\\\/tjvrvH8c0R213m6DXyeTnvGGOMjp6SuTiSt0BezLl4y0HowdLLb5o8TjzJI1twH7x1NHwrSrBZb+yf7R7xrgU8YJjm4mT3bKfDazIdaIPzN3DXGtOXDX8MKkUfwqt+IOIcGO9UmiOVbbYbpvxzTo2qFeFmXHSZqq1wvOCRiYfbQckC+hirQlA+QPUB3ukf2A16AczmX6LPO8Zely6Mrccg86oiPIN8mfXl4AwYENvsKEsvVDyfADJDgtpQJ890cGa6G5Xt6CotPiliLUUWGiVJbVh6ITmei+AzJJ7F8P\\\\\\\/Nx6T8ZynZowD+SeHxZuJ1O+SW+huAyZUE5p9AR1OtVsUm1I0hZvXrbvKmjVcTuAp2vfNGUz7kRQrrYh1yJFa2XsFYMaOQ3QcqZCSOELfNE6bZhZgbXLkJNXSmGXMYB0Ig7RVf2QVJjSe5eybierDffj+t4XHPflk6tRqbjxW+CVczg5i2KXYBL6nTnh7ghZOPOG\\\\\\\/sTl4RgBSMmLWYI4hjl5U8INhZjQEJfZCBV1meqUF+Qac0QSRK\\\\\\\/kZouZjSMEG32792rIQa7\\\\\\\/YSJfq+YUjShXcLPZ2HeMpz4mYzvXnbIueanMraQmqlVHu88BX5jbydONhAZenwaZO62DHb0q64VBouFMhKREnES19zI98E4BCcjOJK2LBlVu716Oj2LxGVHFPZn6klEuIGXoTWr70QitXuMljoqBWDikL\\\\\\\/zvRA8MusBdDdE8xZqDiDpH0D36OtskC2QTZ3sLwkaYz5y8h+Or2kWvAQTA5KGBEQU2D4\\\\\\\/gSIQ22Yt7uPac63rbLu+V8\\\\\\\/bOJ9ySS5rjr\\\\\\\/zowiw\\\\\\\/pFaG0DyvyxRKnuczGcnsbq42R2UkeDVv8wnNbYN7mSjEL+iOSQkMB9oKGuJJiiffSZiiUhXzvyPq17WPIA8r6jfneO\\\\\\\/Y4V4Q42Xv8jAYlajj9NNXc2quHJgnb1h4Nq3amBHzU721xdOFpHhHcRWlzcbrVp2Iy3mtkcxOwAqUw2txLYOO9AG7S47ODMvqG8a1enHQf\\\\\\\/1IlPKO335yVmmemi1Czy9usjl5B9l6fProMrxztJTYwJL8qzJrL8VIqwy3pc44I5+LFjHaXjCczpLfhcPJfCGD1jeBtzhwkymg3kQiAa1bHg4EYcmKh9nAqo9EreyTAYkeKoXp4nnOuVsct5OnwEXM\\\\\\\/HXTyguc9fe77jp0kLWmyOueS51O1YeiXpyhvsiOLEg7Wm7Qo8fYuF2N28CmekKm5X2aXFg44v5AXMxC04dSi8+2d6xWTG\\\\\\\/xOTcYDx6QQdi\\\\\\\/s97v\\\\\\\/4LRkxx0VBJC8eyhAev6s\\\\\\\/TN0Wb0DNMOoy6VBSU1zPd1oZ\\\\\\\/tQT5kiTuHgXxxHQspw\\\\\\\/NMJn2JV\\\\\\\/qE9v0yIAcMnMv939M9TMBd2v2+i\\\\\\\/4BPB4OvgHHbgwIizOlPM20uodakJriJhgrR49r2quedUzRb8EkcDLB+wrd3QgGAQiZsrXvcHeljD1Od72EOHZnzESevkJJBdNv7g8P5NREqjKQhdI9J1EoUAGOz0HGyvKj4ugHKXfhN+xyduQyJKloWnHvKSOtEk9AH17Z7Q0qJCkdYT4Hq6y59tz36H70PXz+GeHVYrCfs99tK\\\\\\\/GPhS2xqAWkZLF1YfWJK3csbP\\\\\\\/D4U\\\\\\\/wRPIOzuZe\\\\\\\/C1N7mz++2LqVzWVh9GKrwzfBDCUHEPx0FCS2aN2DRTvLbdg+51Cc95wQmnKX0wwED59UWN8wVusg1vIIyhSvTS0rbRspAhR3hCwVqTxuiA8Nzp4u5yHVkZ1sRgEOeFQYofLT4fYoURsFb9NaN\\\\\\\/IeDM\\\\\\\/hai3rVz1B5NDP\\\\\\\/XkEW1hZQsuIUQU39+p4FRzI1h5htZUqJvLXiW\\\\\\\/eCYEWxumJvOx7AHMeWLyHXmOlpeLIHvjeZaz6MINe0lao1QyY9TDQKlprXiTTE2a8+cP1ghcrvuDSSkbbQgJx2N0+mdQs3zakVDUyjFkt2towo+rjRkupdXcFrxw8ri\\\\\\\/iBJTjosRdXBGCzSL96uwuhk59paT7fUMSeK8wblEw07jXFiGdY3pUTp4Zdnxi6xYmn2EAlagDe59IVDrvi2nLOAHGi9b\\\\\\\/aMjG+6gqlggqQ8lGh7CrwW2DrV1f+7B5kxP3jaXBGinBQ9WLHmqqHJccuap4g1z9rzHbE2LRgM5xlzhLnJUjjTtP83usxte13p\\\\\\\/HCqMsI4VfzVnoiiYlUkmV6idMsQajvs42i+iQViWSaCIW8TTJv476DL6EjVDtPqC7anOLynVuv0WDalpIgxLJxLxK1paxmmkh7zuSO35JcnzhkhB6msC3jpIH+ug\\\\\\\/fdlZMtrxJXUBkOjPvsLK54txndLV7sLTBE8xSwPo0dzT6kjfTFfmkrzRFJnbgjk89+xhoC2UV1aZD538xnnSNJx8w5N7NtY9Q80c2qOxOZYFHEJSE45l2AiikCPBAQ4fkH3GftOcpvTXbVAzPjmQjrhy6N1LVMOvg7bp3SbHjTmAeVDz7vxZUuSZ\\\\\\\/4YCzG5PBzQBTpmpewEpkhJUZtW6oJSSe4o4iZ85uns67QDwoDlw9Rk2Vy7uTMP9NSAHfaf81l5zRqU1SLWBivNdNCcIAN71ieiCLknfr\\\\\\\/uP36vxlQh3OWpGd4xnXRG9wVVx5UvCNOkJgMjbyc+GnrC0uHgjpN1AOXizHabDMfdqMxaFBA+FXD565qPy0b9\\\\\\\/JHmkrh8sWg+Foyd\\\\\\\/V1FrzLaDuEHkMrJWjMMe1uwqBUVBsbMAeJrW+qULoef5qrtrVSWlQ2jEJ2sJEn2AlMsBoViR2D76WHaFFYX9gldDFoxSteudq5NBhq+tJs9GmXuk6YFTdnkp0NjASOwe59vEpLfFUNBRHG\\\\\\\/u+sbMERsCwY3nxOjJlbFTc1T+85iqyJaA2pjR3COA\\\\\\\/GNetNjVmvyF10flNB5YycTFUo3ejHs12qB8+PeHlelrbYlCbu0Ehs1bdenLc+uZ0gznJ3Wzx2WJncI3fZY2hysq2Uf6+tFioGfpbvZj842XNM2AIUd2UKf5NSOIDskWw9foFAET9foGfAZLrobEyWvOtDSWFMpkInIBeXqXFcB4nTi5aZRP8KYr5yGvhqLdCNiMovrMM+ZgfJ9fvw9LM2WpZT+n\\\\\\\/File9QEitXZaNRCXtueemwtk68XDVPOSxwj6AiRbplp9UjI9JikZl4KE3e1BRSzl3wtLXB7+UeGLaERDLYpMZ2EoYKGL64VsSnJoZrNkhNGKIAepItHje\\\\\\\/F+g3twfY5P7QvVZv+MLvySvs0LALOlWT26gCEdBOj2PIl5cxzoq6o0UKZPoHFIWKc2bznI28+\\\\\\\/f2\\\\\\\/qbXJs2y2l\\\\\\\/Cme05EK3YNKXsqsfG3aw1r7hlUjJOw1oQR\\\\\\\/vZQuCmoT+X4h2yTwwww\\\\\\\/cSiqgHE1oBsSDEnbPNn\\\\\\\/K7fxrHYEqXjZE5Oi8iPZMkdFUmQb77dkEUkSw8wbV7WmOzJcMi8ckIgCrb8mvJH9p5mPEzMmOt5DOtQMwppdGP60intqnwpI58T\\\\\\\/LIB6nzPvNLF0KdRY+svRVItbqYyiAIOjxtO8NjgmOImmRca6hZPEdW2rejX7QpsQzfxIJn+lAD7ctNZ8rG\\\\\\\/Tt6oC6fEgH86qxOBB4R8MXSLJEn30zXQ1abwyaZQ6pHgCb13EuioOOe15ZPvN6uRmL85PDbhIo6jkSKlcX40Lkvqs21sqtmlqOtUp75iiAKM27n5JoNsSKtiqhwbA0\\\\\\\/XKR00fDbizFK1WIlhLSFDHNBmHHfHGtQWRE1wvMfYLgUol0z8g8X\\\\\\\/Hlj3tWnxTTsUQ\\\\\\\/TBHHItKskneHBTt6WTSYOKlB+NZ+2wKKvL1isWshkImDqeB5m4+3acLIE7qERSLBXgq0DSfddZavYyAaNwgbEvFXdfHeWmWjFFDGVTCDGwKt3Z7AWgNC1qvUhGl+UjDG+5ah5j3zTK32AS\\\\\\\/7ZMthkZjh12yZabuTN8g+oIGKhE7a54hlzGs4GhBydUtPr93xDrD6arMPHYaFEc8xQAg7\\\\\\\/HKT9BOE1YkjapFOo3Y+6PrlXOYBa0sbESfbkstzLwYViZHUtPzQHxCZ8rVRHiP0osH3PL4KXuFBxlt7ogNYwj3CAJT8kXS1FfvbFTGbihB1t5gX46ymv2fMxN5sCv8JVMcOfMkKAcyNYdc5dLuiJGccE45rYubZUV7IxJhQb6ApaPmABVdcp6k+ghHXq96LC\\\\\\\/iUPIKXdo6UuuohMCQDqeL+TKOnem5ryFNTQfKTK9yaDrkW6WHQ6mYyyHxNLkQ6xSLxW5l9YTejxd2pJxeioD8YStV7URfKKh2r0bcZmpdBo687QL7QhdhknQnGqxT6kY9Gj+cUf05HDGLtYvZq2PhPnGKwQBOGamPnMhmASV+0HRXDgjVZJlySaZNOUHHQYcoRSaXkrmb7aQuDqo5X21cHJFQDM9OYL19NKpsE4JjJPzl4i86iOu1\\\\\\\/wswWfApig7yac1sJmu\\\\\\\/ktAKcuRPHqDAArgAGhSwzinsMqSh9dMUH1DV+IjTnJicl+kHu49vbCmpZAnd408KjqQGnvc4+ADHTi4zZjP\\\\\\\/w26R71k7hKTfqT21hXqffWe5V7KperFNhPSvmeYK6UJc1Q5GxPCJhoF1jpMokBkxhjUlZEx7gWSNFFnqGHWhxPdcH0zSwC3d0yklRiqXNhDamgEkd0zrIY7pJAQH1XXJzPmduV2A09i7EoMRtHNAOgrqTIVSPJRba6F5cQBn7aWcydmOyUo6btg0ZhWg0jem\\\\\\\/cHAd963zzGcXczQjUcyIXrVVNTNpWVr\\\\\\\/UfRvFUwRSFswuXeOlmealQ4OMN59\\\\\\\/5uqai0\\\\\\\/fHls6PfRoRq9lAQN7sPQKRAj76po+\\\\\\\/oq\\\u0022,\\\u0022iv\\\u0022:\\\u0022MT9dk5rTFRJ7bOdpfMzSNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ffd9c182ae7e98cae8764b6a72b37ab9eb6e5138ebe9a4314e5e8eb7300df60f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r8h53j\\\\\\\/cSYKJQxB1N9UUOzIrWG755o5QW\\\\\\\/68c8Q5sX4Uke\\\\\\\/TUFcGQ586\\\\\\\/puRpTsQyTMu3IHIBJUdwvBIO9A77QWzEf95tsUHv7JEDFCGJqD3xrd5kGKmAJhYrZXnZpJAcc+HblJzJPSOXKX9CYW7dHcltx+UuuggCkRT5vsyr9uNtNl+RWLyzg3eAa92niMLcMyr9T8G7GUrAJev24b\\\\\\\/y22I2m8PoIu\\\\\\\/t96BP8lyvPA2dbJOuXVcnnvvZQvG9IDAVzv3wct1e8CXT9rBcQME2Q9rfg0JN6qDxrkWlPtbMx2Vn1t\\\\\\\/RRBR9Rtpnpqf9AAG8XIxawyGcCMjnIXC8Dk0IHFiriQ\\\\\\\/QeY3xfh0ieoqabT8lYNOKq9KIUhXrhRI6YsVcM5NNp8\\\\\\\/\\\\\\\/4iorSlnzK61vxhBASSnYOrKsWGWghLwXEhAiUDHWIgnW+y\\\\\\\/RzelVjVnT8jsmq4Oj0MzoCH6uKdLmEbBtt7cVKcKPrST3Ht\\\\\\\/VVseGcY9gPJrQwEdI6E2rbyVvJ98VqiH+6gEjKuouXVxfXAi\\\\\\\/fsykOHEJ3jB8UawATCh2GKa5rFbcnMZy0PHs6MwxdKHZ5VchiygHSsmOMCxwLseKYjulEDWfTJGMLh8yO9I0QoTW2iFhOvie+r2VyB6GZCeKpiv7wug4Ng8xCmMZnux1sg99Mz4q2E7wl4vndzSLXs25TZ8rzOTw8ZLVlty8GxkaGCAQIxvkhaIjkNeN30sTdzbntqkiAay7WcyW0+E0BPxuj5mv9zhp24fjhdJDqPRHZv56yfq2d\\\\\\\/+NaTrQBRAf\\\\\\\/cf6OSdP1vZyKikFk4tbyUJZ9RMWO5gbQZgu9l6MWjRtkOf94Ot6jpbGoyfv4Szq8TVF5Wkw7P6PNXgNd04PLeMGcsrM\\\\\\\/Ie8sS+rNXlpAtT150GbjOzkxqFYcOEk214ltKC\\\\\\\/xutDFzB4Dq229bUwGiSmr1T79Mq2aPKHihJQPAB0CdPTaO+Cj8LkdOykXuA30VL74TkjfB5rjWIliPAJY5u4K27Q8F\\\\\\\/SN3phaFbEcCqDFc\\\\\\\/+mAQ\\\\\\\/FbxvgbdwLt9apJgYciZiArf0qqlqbvzqpi5Xu441B3T\\\\\\\/coOaOPa6D\\\\\\\/PI2+UpzEABruvfglDroyLl88KISWrfd21ofnrES6PPMX3t26rJPwZG07s2AKb0tHIXsd00PwGuaV1AMeK7n0INNvZge3sHDJHoz2L+3h3VAezP\\\\\\\/4eajarJPCtrb7h4V2uiYPfSfqSJrJHciVWIMr\\\\\\\/7MMkO\\\\\\\/BwNQtfLy5YjILvldP5\\\\\\\/xSKJJVak7BMlzFhZHgsNvAHHCwZE4V1GbMT8b+m3DHNoojM3fsxmmkAig34sOi9sUypDjp701atF+uRGhiWhSmt1qbdd4RuKNMfgS0Y0SFdT4uZuM29HVkbfVCsMORVHfWQCWDA6+uPEk2XU426lfZfZF7XQp5Ic7QEdhYXWPK1h7dx\\\\\\\/pc+RaloqmBYRoCwL\\\\\\\/wpgkkBaP9ZRPG0nMs3+djrEHPbjccL3AN3cafEl+wEXIFxxWaYWG3gkuvp8du+Nro8YBt6BDazb+GSOP11WGPQTRbDVe34rYFn5ks9N\\\\\\\/3YFHq2lAcn\\\\\\\/KffmSZVSr6BEEwYXrSt8Pg3VfQoy4AgPhEXyU2c34Y9GWEYvyFVRK0Aocep8TMmvUKGzYu2Gybo\\\\\\\/MXp2fmkJp8Bd2+oCNUWCd3ZTengpWEHyM5GS5Q6fZ+EEIstuL9lwwSWMoyTakVKlmf3+hwBuCM9+emKMwOTnYY4mVscClf\\\\\\\/yL9ONuRA6TezIQOnggKamcMx+FqjfH8abQpAu545+VmUL9D0Gfzeb6jUR8CMVogOBFumGVmbbkiEip2UpJXdUAzj2bxMCjRRVcyQ8VfQsN9yY4P\\\\\\\/u2QrjhU4CZYBRl+TNLZpNW4TGqZI9289sNbmcRlmncbD43u6BN\\\\\\\/ritIZLBgtk\\\\\\\/a2pJfOogrCqhZX192FRyqwXZSasMf\\\\\\\/8XMlNjOLHx\\\\\\\/T4BgohJKLhiTt8OGrNYvaIDCE9M6xes2fDWhcos\\\\\\\/iW3HiXTzguY+OOqE9G2XueQV2MP4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022tD4vrAUml7LM748qgIoWNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220edbd029389be15f6da5084ba7844834d8aec41794f595125d199e9a73900eb4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489124 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1327" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aR69eba2myA4h4fxfJIbYiiDMFnJjq4OYIxmcw4XEkQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1327" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CnJA1iWH33RIq5vHceo8co99lWpx6bET4rdGL04douE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1315" title="Examination of Signatures of Recent Positive Selection on Genes Involved in Human Sialic Acid Biology" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1335" title="A Genomic Reference Panel for Drosophila serrata" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489124" id="node489124" data-pisa="ggg;8/4/1327" data-pisa-master="ggg;g3.117.300467" data-apath="/ggg/8/4/1327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in <em>Capsella</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jörg A. Bachmann</span>, <span class="highwire-citation-author" data-delta="1">Andrew Tedder</span>, <span class="highwire-citation-author" data-delta="2">Benjamin Laenen</span>, <span class="highwire-citation-author" data-delta="3">Kim A. Steige</span> and <span class="highwire-citation-author" data-delta="4">Tanja Slotte</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1327-1333;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300467 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489124" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jörg A. Bachmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rg%2BA.%2BBachmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachmann%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rg%2BA.%2BBachmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jorg.bachmann@su.se" class="" data-icon-position="" data-hide-link-title="0">jorg.bachmann@su.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Tedder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BTedder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tedder%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BTedder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Benjamin Laenen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BLaenen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laenen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BLaenen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kim A. Steige </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kim%2BA.%2BSteige%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steige%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKim%2BA.%2BSteige%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tanja Slotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BSlotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Slotte%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanja%2BSlotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/4/1327" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489124" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1327.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489124" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1327.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489124" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/4/1327.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489124" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/4/1327.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489124" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Rapid advances in short-read DNA sequencing technologies have revolutionized population genomic studies, but there are genomic regions where this technology reaches its limits. Limitations mostly arise due to the difficulties in assembly or alignment to genomic regions of high sequence divergence and high repeat content, which are typical characteristics for loci under strong long-term balancing selection. Studying genetic diversity at such loci therefore remains challenging. Here, we investigate the feasibility and error rates associated with targeted long-read sequencing of a locus under balancing selection. For this purpose, we generated bacterial artificial chromosomes (BACs) containing the Brassicaceae <em>S</em>-locus, a region under strong negative frequency-dependent selection which has previously proven difficult to assemble in its entirety using short reads. We sequence <em>S</em>-locus BACs with single-molecule long-read sequencing technology and conduct <em>de novo</em> assembly of these <em>S</em>-locus haplotypes. By comparing repeated assemblies resulting from independent long-read sequencing runs on the same BAC clone we do not detect any structural errors, suggesting that reliable assemblies are generated, but we estimate an indel error rate of 5.7×10<sup>−5</sup>. A similar error rate was estimated based on comparison of Illumina short-read sequences and BAC assemblies. Our results show that, until <em>de novo</em> assembly of multiple individuals using long-read sequencing becomes feasible, targeted long-read sequencing of loci under balancing selection is a viable option with low error rates for single nucleotide polymorphisms or structural variation. We further find that short-read sequencing is a valuable complement, allowing correction of the relatively high rate of indel errors that result from this approach.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/single-molecule-real-time-sequencing" class="hw-term hw-article-keyword hw-article-keyword-single-molecule-real-time-sequencing" rel="nofollow">single-molecule real-time sequencing</a></li><li class="kwd"><a href="/keyword/bacterial-artificial-chromosomes" class="hw-term hw-article-keyword hw-article-keyword-bacterial-artificial-chromosomes" rel="nofollow">bacterial artificial chromosomes</a></li><li class="kwd"><a href="/keyword/sequencing-errors" class="hw-term hw-article-keyword hw-article-keyword-sequencing-errors" rel="nofollow">sequencing errors</a></li><li class="kwd"><a href="/keyword/assembly" class="hw-term hw-article-keyword hw-article-keyword-assembly" rel="nofollow">assembly</a></li><li class="kwd"><a href="/keyword/self-incompatibility-locus" class="hw-term hw-article-keyword hw-article-keyword-self-incompatibility-locus" rel="nofollow">self-incompatibility locus</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/capsella" class="hw-term hw-article-keyword hw-article-keyword-capsella" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Capsella</em></a></li><li class="kwd"><a href="/keyword/brassicaceae" class="hw-term hw-article-keyword hw-article-keyword-brassicaceae" rel="nofollow">Brassicaceae</a></li></ul><p id="p-6">DNA sequencing has come a long way since Sanger’s “chain termination” technique first improved our ability to sequence DNA (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sanger and Nicklen 1977</a>). In the last decade, major breakthroughs in massively parallel sequencing have unfolded whole new generations of technologies, currently allowing us to reliably uncover vast amounts of genomic information (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heather and Chain 2016</a>). However, despite recent advances in high-throughput (short-read) sequencing technologies (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Reuter <em>et al.</em> 2015</a>), some genetic regions remain difficult to study using short read data due to their size and complex architecture (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mardis 2017</a>). Notably, loci under long-term balancing selection, such as the Major-Histocompatibility (<em>MHC</em>) locus (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hedrick 1999</a>) or plant self-incompatibility (<em>S</em>) loci as in the Brassicaceae <em>S</em>-locus, have a large number of highly divergent alleles maintained by negative frequency-dependent selection over long periods of time (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wright 1939</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Vekemans and Slatkin 1994</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Charlesworth <em>et al.</em> 2000</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Castric and Vekemans 2007</a>). Their genetic architecture is affected by long-term balancing selection that promotes the emergence and co-existence of numerous differentiated alleles (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Llaurens <em>et al.</em> 2017</a>). In such regions, high repeat content, diversity, and rearrangements makes assembly and especially re-sequencing approaches based on mapping short reads to a reference genome, difficult (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mardis 2017</a>).</p><p id="p-7">Previously, studies on genetic diversity at the <em>S</em>-locus would rely on polymerase chain-reaction (PCR) amplification of specific regions of interest (often the <em>S</em>-locus receptor kinase gene <em>SRK</em>) in combination with either Sanger sequencing (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Shiba 2001</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kusaba <em>et al.</em> 2001</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Miege <em>et al.</em> 2001</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schierup <em>et al.</em> 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Charlesworth, <em>et al.</em> 2003a</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2003b</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nasrallah <em>et al.</em> 2004</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bechsgaard <em>et al.</em> 2006</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kamau <em>et al.</em> 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Castric <em>et al.</em> 2010</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tsuchimatsu <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Leducq <em>et al.</em> 2014</a>) or short-read sequencing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2009</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jørgensen <em>et al.</em> 2012</a>). The two major caveats of this approach are; it is not always applicable to use (general) PCR primers for very divergent alleles, and regulatory regions in intergenic regions of high complexity are not resolved. For this reason, researchers have resorted to targeted sequencing of the entire <em>S</em>-locus region using massively parallel sequencing of bacterial artificial chromosomes (BACs) containing the <em>S</em>-locus (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Durand <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Novikova <em>et al.</em> 2017</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tsuchimatsu <em>et al.</em> 2017</a>). However, due to the high repeat content of the <em>S</em>-locus, next-generation sequencing of and assembly of <em>S</em>-locus BACs with short- or medium-length reads resulted in several contigs, thus requiring additional PCR-based testing to elucidate gene order and orientation (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> 2012</a>).</p><p id="p-8">In contrast to short-read sequencing technologies, SMRT sequencing should provide a better basis for reliably assembling repetitive regions, due to mean read lengths &gt;20 kb and maximum reads &gt;60 kb (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rhoads and Au 2015</a>). Eventually, <em>de novo</em> assembly of whole genomes for each individual would be the goal for the study of evolution of loci under long-term balancing selection. However, the high costs per base of SMRT sequencing currently limit the feasibility of this approach, especially for studying population genetic variation at loci under balancing selection. Here, we therefore investigate the utility of targeted sequencing of two different Brassicaceae <em>S</em>-locus sequences in two BACs using SMRT sequencing, with a focus on quantifying assembly errors, single nucleotide polymorphism (SNP) errors and indel errors. By comparison of two assemblies of independent SMRT sequencing one of the two <em>S</em>-locus BACs, we find that this approach is efficient and highly accurate with regard to structural errors and single nucleotide polymorphisms. Mapping short-read data to the an SMRT assembly of the second <em>S</em>-locus BAC for error correction, we find that correction for indel errors is necessary, especially for studies aiming to identify functional polymorphisms. This method can thus be valuable for a wide range of genomic studies of complex genomic regions, where reference-based approaches for studying genetic variation are not feasible.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-9">We surface-sterilized seeds of four accessions of the self-incompatible crucifer <em>Capsella grandiflora</em> (from Epiros, Zagori, Greece) with 10% bleach and 70% ethanol. Seeds were stratified at 2-4° in the dark on plates with 0.8% agar and half-strength MS medium (Murashige and Skoog basal salt mixture, Sigma-Aldrich Co. MI, USA). After two weeks, we moved the plates to climate controlled growth chambers (16 h light at 20° / 8 h dark at 18°, 70% max. humidity, 122 uE light intensity) to allow the seeds to germinate. After approximately 1 week, we transplanted seedlings to pots with soil in the climate-controlled chambers. We kept the plants under dark conditions for 4 days prior to sampling young leaves for BAC library construction.</p></div><div id="sec-3" class="subsection"><h3>BAC library construction and screening</h3><p id="p-10">To sequence full-length <em>S</em>-locus haplotypes, we followed a strategy similar to <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> (2012)</a> based on BAC libraries. High molecular weight DNA was extracted from 10 g of young leaves per library, and we pooled leaves from two individuals per library. The DNA was digested with <em>Hin</em>dIII and ligated to pCC1BAC cloning vector (Epicentre, an Illumina company, WI, USA), after several size selection steps. BAC libraries were screened for flanking regions of the <em>S</em>-locus by hybridization with DNA probes and PCR amplification with specific primers and further selected based on mean insert size. Clones of colonies that tested positive for <em>U-box</em> and <em>ARK3</em> flanking genes of the <em>S</em>-locus were selected for sequencing. All BAC library production and screening was performed by the French Plant Genomic Resource Centre (CNRGV) at INRA.</p></div><div id="sec-4" class="subsection"><h3>Sequencing</h3><p id="p-11">We conducted SMRT sequencing (Pacific Biosciences of California, CA, USA) of two different Brassicaceae <em>S</em>-locus sequences in two BAC clones at the Uppsala Genome Center, National Genomics Infrastructure Sweden. DNA fragments over 10 kbp were selected using BluePippin Size selection (Sage Science, MA, USA) and the SMRTbell Template Prep Kit 1.0 (Pacific Biosciences of California, CA, USA) was used for library preparation, with an insert size of 500 bp to 20 kb. SMRT sequencing was done on the RSII system, using P5-C3 chemistry.</p><p id="p-12">To assess sequencing and assembly errors, we generated two independent libraries of one BAC clone (CgrS-BAC1), which was then subjected to independent SMRT sequencing and assembly, whereas the second BAC, (CgrS-BAC2), was sequenced once with SMRT sequencing. To assess indel errors, we also generated short-read sequencing (MiSeq, Illumina, Inc., San Diego, USA) data for the second <em>S</em>-locus BAC (CgrS-BAC2). The sequencing library (TruSeq PCRfree DNA sample preparation kit, Illumina, Inc., CA, USA) was prepared from 1 μg of DNA, following the manufacturers’ guidelines. We generated 1.1 million paired-end 250 bp reads on the MiSeq using v2 sequencing chemistry (Illumina, Inc., CA, USA).</p></div><div id="sec-5" class="subsection"><h3>Bioinformatic data processing and assembly</h3><p id="p-13">We assembled raw SMRT reads from each BAC clone using the Hierarchical Genome Assembly Process (HGAP.3) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2013</a>) with default settings. The pipeline generates a <em>de novo</em> assembly with Celera Assembler 8.3rc2 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Myers <em>et al.</em> 2000</a>) and includes a consensus polishing step using the Quiver algorithm (Pacific Biosciences of California, Inc., CA, USA). Per sequenced BAC clone, this process yielded a large assembled fragment (contig) containing the region of interest (<em>S</em>-locus), as well as several contigs containing <em>E. coli</em> sequences. As HGAP.3 does not split reads, assembling a circular molecule results in overlapping ends of reduced coverage, and we therefore conducted circularisation and removed overlapping ends using minimus2 v3.1.0 of the AMOS suite (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Treangen <em>et al.</em> 2011</a>) This was followed by another Quiver polishing step (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2013</a>) to improve the quality in the region that was formerly split between the two ends of the sequence, and finally trimming of the vector sequence.</p><p id="p-14">We quality filtered and trimmed raw reads from Illumina MiSeq sequencing to remove adapters using cutadapt v1.3 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Martin 2011</a>) which identified the most likely used adapters. Subsequently, we trimmed all adapters as well as low-quality reads with Trimmomatic v0.36 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>).</p></div><div id="sec-6" class="subsection"><h3>Error estimation and correction</h3><p id="p-15">To estimate assembly and sequencing error rates, we compared the <em>S</em>-locus contigs from independent sequencing and assembly of CgrS-BAC1. We generated a pairwise alignment of the two <em>S</em>-locus assemblies using Mafft v7.310 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Katoh <em>et al.</em> 2002</a>) and assessed the total number of assembly errors (<em>i.e.</em>, structural differences between the assemblies), and the numbers and base-pair locations of indels and SNPs, that represent sequencing errors.</p><p id="p-16">To generate an additional estimate of sequencing error rates, we used Illumina MiSeq data for CgrS-BAC2. We mapped short reads to the polished assembly of the BAC clone using bwa-mem v 0.7.8 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Durbin 2009</a>). Finally, we estimated indel error rates and corrected these indel errors using pacbio-util, based on the consensus of the mapped Illumina reads (<a href="https://github.com/douglasgscofield/PacBio-utilities">https://github.com/douglasgscofield/PacBio-utilities</a>). Indel error rate was calculated as number of insertions and deletions, divided by assembly length to get a per base-pair error rate.</p></div><div id="sec-7" class="subsection"><h3>Annotation and comparison of SRK sequences</h3><p id="p-17">To assess the utility of using SMRT sequencing of BAC clones to reconstruct complex loci, we extracted <em>SRK</em> exon 1 sequences from the <em>S</em>-haplotype assemblies by searching for BLAST hits to general <em>SRK</em> exon 1 forward (<em>SLGF</em>) and reverse (<em>SLGR</em>) primers (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Charlesworth <em>et al.</em> 2000</a>), extracting either the sequence between the two primer sites, or, if only one primer site was found, each 1kb sequence up- and downstream of the primer site. We then selected candidate sequences based on strong sequence similarity to known <em>SRK</em> exon 1 sequences or conversely rejected them based on stronger sequence homology to known <em>ARK3</em> (<em>Aly8</em>) sequences using BLAST (v2.5.0+). Exact parameters for sequence homology varied between candidate sequences due to high divergence in <em>SRK</em> alleles, but were always above 90%.</p><p id="p-18">In order to characterize the relationship between our <em>Capsella SRK</em>-like sequences, and known <em>SRK</em> and <em>ARK3</em> alleles, we bulk downloaded <em>722</em> publicly available Brassicaceae <em>SRK</em> and <em>ARK3</em> sequences of &gt;500 bp length from GenBank (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300467/-/DC1/TableS1.docx">Table S1</a>) and retained only those under 2000 bp of length. Duplicates were removed using dedupe.sh from BBMAP v34.56 (Joint Genome Institute), and we made an initial alignment between our <em>SRK</em> sequences and the publically available <em>SRK</em> and <em>ARK3</em> using MAFFT v7.245 with the E-INS-I algorithm (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Katoh <em>et al.</em> 2002</a>), which is suitable for sequences containing large unalignable gaps. Due to the sequence diversity present in <em>SRK</em> exon 1, it was necessary for us to manually edit the alignment in Seaview v4.6 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gouy <em>et al.</em> 2010</a>) to correct alignment errors. To visualize the phylogenetic relationship between our <em>SRK</em> sequences and those previously sequenced, we constructed a phylogenetic tree using RaXMl v8.2.3 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stamatakis 2014</a>), generating a neighbor-joining tree with the GTRGAMMA model, and 1000 bootstrap replicates. The tree was visualized using FigTree v1.4.2 (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>).</p><p id="p-19">To assess whether we had successfully sequenced the entire <em>S</em>-locus, we annotated our <em>S</em>-locus assemblies with Augustus v3.2.3 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stanke <em>et al.</em> 2004</a>) and RepeatMasker v4.0.7 (<a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) via Maker v2.31.9 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Holt and Yandell 2011</a>), with <em>Arabidopsis thaliana</em> as a model prediction species and protein homology data for <em>B120</em>, <em>ARK3</em>, <em>SRK</em>, <em>U-box</em>, <em>B70</em>, <em>DYT1</em>, <em>SBT3</em> and <em>AT4G21323</em> from <em>Arabidopsis lyrata</em> and <em>A. halleri</em>. Annotation of the highly variable <em>S</em>-locus gene <em>SCR</em> was unsuccessful with a homology search to existing <em>SCR</em> alleles. Using a sliding window approach in open reading frames, we searched for conserved patterns of 8 cysteine residues to find <em>SCR</em> exon 2. The resultant <em>gff</em> files were concatenated, and the annotation visualized using R v3.3.1 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">R Development Core Team 2008</a>).</p></div><div id="sec-8" class="subsection"><h3>Sequence conservation</h3><p id="p-20">To assess patterns of sequence conservation across the entire <em>S</em>-locus region between <em>ARK3</em> and <em>U-box</em>, we first extracted a larger region between B120 and <em>AT4G21323</em> as described above. <em>S</em>-locus sequences were then aligned using LASTZ v1.03.54 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harris 2007</a>) and the resultant “axt” files were converted to fasta format using axt2maf and maf2fasta, respectively. Pairwise sequence conservation, as the proportion of conserved bases per 250 bp sliding-window, was then calculated with a python script, and visualized using R v3.3.1 (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">R Development Core Team 2008</a>) (<a href="https://gitlab.com/slottelab/Sequence_conservation">https://gitlab.com/slottelab/Sequence_conservation</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-21">The sequences of CgrS-BAC1 and CgrS-BAC2 we generated in this study have been uploaded to ENA at EBI with project id: PRJEB24927. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300467/-/DC1/TableS1.docx">Table S1</a> contains Genbank accession numbers of <em>SRK</em> sequences used in this study.</p></div></div><div class="section results-discussion" id="sec-10"><h2 class="">Results and Discussion</h2><div id="sec-11" class="subsection"><h3>Sequencing and Assembly</h3><p id="p-22">SMRT sequencing of two BAC clones corresponding to two different <em>S</em>-haplotypes resulted in an N50 read length of 19,187 to 28,120 bp (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For additional short-read data for one of the BACs that was assembled based on long-read data, CgrS-BAC2, we obtained a total of 482.1 Mbp of Illumina MiSeq paired-end data (250 bp, &gt;Q30) corresponding to a coverage of 2938X.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489246/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489246/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Capsella S-locus sequencing summary</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">We obtained one large contig containing the <em>S</em>-locus sequence for each of our three <em>S</em>-locus assemblies, with a length between 164 kbp and 178 kbp, as well as several smaller contigs containing parts of the <em>E. coli</em> genome or only cloning vector. Circularisation and vector trimming resulted in polished and trimmed assemblies of sequences containing complete <em>S</em>-locus sequences plus flanking regions of a total length of 156636, 156640 (for the two assemblies of CgrS-BAC1), and 153563 bp (for CgrS-BAC2), see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><p id="p-24">Using SMRT sequencing allowed us to assemble the entire <em>S</em>-locus into one contig, in contrast to assemblies of the <em>S</em>-locus based on short-read data, which resulted in several contigs (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2011</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> 2012</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Durand <em>et al.</em> 2014</a>). For short-read assemblies, even additional PCR-based measures to bridge the gaps between separate contigs often do not resolve the physical distances and relative orientations of genes for all haplotypes (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2011</a>; <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> 2012</a>). Quantifying variation in length, gene orientation and repeat content can be important in answering the question on reduced recombination at the <em>S</em>-locus (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth 2016</a>), but the diversity can only be fully revealed, if the <em>S</em>-haplotypes are assembled as continuous sequences.</p></div><div id="sec-12" class="subsection"><h3>Assembly and sequencing errors</h3><p id="p-25">There were no structural rearrangements present between the two <em>S</em>-locus contigs resulting from independent sequencing and assembly of two separate assemblies of CgrS-BAC1 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), suggesting that the rate of structural errors is low and these assemblies are accurate.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/4/1327/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A S-locus sequence assemblies with two measures of indel errors indicated in black bars. Inference of indel errors are based on comparison of two independent SMRT-sequencing runs and assemblies of CgrS-BAC1 (upper) and alignment of Illumina short reads to assembly of CgrS-BAC2 (lower). Annotation of exons are shown as colored arrows, simple repeat sequences in red, and blue-boxes indicate positions of transposable elements. The genes flanking the S-locus are ARK3 (light blue) and U-box (light green). SCR was only annotated in CgrS-BAC2. B S-locus sequence conservation between the two Capsella S-locus BACs, created by aligning the S-locus regions with LASTZ and comparing sequence homology (in % between 0 and 100) using a fixed window size of 250 bp. Sequence similarity between CgrS-BAC1 and CgrS-BAC2 drops steeply at the borders of the S-locus, corresponding to the genes ARK3 and U-box, respectively, although some sequence similarity is also found at SRK. C ML phylogeny of all alignable SRK alleles (exon 1) above 500 bp from GenBank. Bootstrap support over 70% is represented with an asterisk (*). Our newly identified sequences, indicated with arrows, are found broadly distributed across the phylogeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1998768779" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A &lt;em&gt;S&lt;/em&gt;-locus sequence assemblies with two measures of indel errors indicated in black bars. Inference of indel errors are based on comparison of two independent SMRT-sequencing runs and assemblies of CgrS-BAC1 (upper) and alignment of Illumina short reads to assembly of CgrS-BAC2 (lower). Annotation of exons are shown as colored arrows, simple repeat sequences in red, and blue-boxes indicate positions of transposable elements. The genes flanking the &lt;em&gt;S&lt;/em&gt;-locus are &lt;em&gt;ARK3&lt;/em&gt; (light blue) and &lt;em&gt;U-box&lt;/em&gt; (light green). &lt;em&gt;SCR&lt;/em&gt; was only annotated in CgrS-BAC2. B &lt;em&gt;S&lt;/em&gt;-locus sequence conservation between the two &lt;em&gt;Capsella S&lt;/em&gt;-locus BACs, created by aligning the &lt;em&gt;S&lt;/em&gt;-locus regions with LASTZ and comparing sequence homology (in % between 0 and 100) using a fixed window size of 250 bp. Sequence similarity between CgrS-BAC1 and CgrS-BAC2 drops steeply at the borders of the &lt;em&gt;S&lt;/em&gt;-locus, corresponding to the genes &lt;em&gt;ARK3&lt;/em&gt; and &lt;em&gt;U-box&lt;/em&gt;, respectively, although some sequence similarity is also found at &lt;em&gt;SRK&lt;/em&gt;. C ML phylogeny of all alignable &lt;em&gt;SRK&lt;/em&gt; alleles (exon 1) above 500 bp from GenBank. Bootstrap support over 70% is represented with an asterisk (*). Our newly identified sequences, indicated with arrows, are found broadly distributed across the phylogeny.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/4/1327/F1.medium.gif" width="315" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/4/1327/F1.medium.gif" width="315" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/4/1327/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/4/1327/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489224" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">A <em>S</em>-locus sequence assemblies with two measures of indel errors indicated in black bars. Inference of indel errors are based on comparison of two independent SMRT-sequencing runs and assemblies of CgrS-BAC1 (upper) and alignment of Illumina short reads to assembly of CgrS-BAC2 (lower). Annotation of exons are shown as colored arrows, simple repeat sequences in red, and blue-boxes indicate positions of transposable elements. The genes flanking the <em>S</em>-locus are <em>ARK3</em> (light blue) and <em>U-box</em> (light green). <em>SCR</em> was only annotated in CgrS-BAC2. B <em>S</em>-locus sequence conservation between the two <em>Capsella S</em>-locus BACs, created by aligning the <em>S</em>-locus regions with LASTZ and comparing sequence homology (in % between 0 and 100) using a fixed window size of 250 bp. Sequence similarity between CgrS-BAC1 and CgrS-BAC2 drops steeply at the borders of the <em>S</em>-locus, corresponding to the genes <em>ARK3</em> and <em>U-box</em>, respectively, although some sequence similarity is also found at <em>SRK</em>. C ML phylogeny of all alignable <em>SRK</em> alleles (exon 1) above 500 bp from GenBank. Bootstrap support over 70% is represented with an asterisk (*). Our newly identified sequences, indicated with arrows, are found broadly distributed across the phylogeny.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We report two measures for indel error rate. For CgrS-BAC1, indel errors were inferred by counting differences between two separate SMRT assemblies of the same BAC (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>., sequencing ID pb126_1 &amp; pb192-4). For CgrS-BAC2, containing a different <em>S</em>-haplotype of <em>C. grandiflora</em>, indel errors were inferred by comparing MiSeq data to the SMRT assembly of the same BAC.</p><p id="p-28">There were no SNP differences and 9 indel differences over an alignment length of 156,644 bp of the two assemblies of CgrS-BAC1 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Thus, based on our technical replicates of library preparation and assembly, we estimate an indel error rate of 5.7×10<sup>−5</sup> indels per bp with a ratio of single to double bp indels of 2:1. Notably, these indels were not specifically found in homopolymer regions.</p><p id="p-29">Mapping of short-read Illumina MiSeq data to <em>S</em>-locus sequence CgrS-BAC2 resulted in an indel error rate estimate of 2.0×10<sup>−5</sup> indels per bp over a sequence length of 153,563 bp (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). Similarly, indels were the only errors and in this case all were single bp indels. Both methods for identifying indel error rates thus result in error rates on the order of 10<sup>−5</sup> indels per bp, whereas no SNP errors were detected using either approach.</p><p id="p-30">Current high throughput sequencing technologies show between 0.1 and ∼12% error rate of raw reads, with Illumina short read technologies generally below 1% and SMRT sequencing &gt;10%, reviewed in (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Reuter <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goodwin <em>et al.</em> 2016</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Mardis 2017</a>). The high error rate of SMRT sequencing raw reads is mitigated by a random distribution of these errors across individual reads and the ability to sequence circular fragments repeatedly, thus the consensus sequence is improved by multiple sequencing passes over the same continuous DNA molecule (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Rhoads and Au 2015</a>). With 15-fold coverage of single-molecule reads, the accuracy is raised to over 99% (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eid <em>et al.</em> 2009</a>), but using the so called circular consensus reduces the average read length, weakening the keystone of long read sequencing (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Travers <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hackl <em>et al.</em> 2014</a>).</p><p id="p-31">SMRT sequencing is useful for complete assembly of difficult loci (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bellec <em>et al.</em> 2016</a>) or even genomes, microbial or chloroplast genomes have been assembled into fewer contigs than short read technologies, or even single continuous sequences were produced by SMRT sequencing alone, reviewed in (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Rhoads and Au 2015</a>). If one aims to study genetic variation at large divergent loci, SMRT-assemblies reveal complete genic and intergenic regions, but for higher resolution at the base-pair level, additional validation is necessary, as in a direct comparison of short and long read sequencing technology, SMRT-sequencing performs worse at the single-nucleotide variant calling (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Quail <em>et al.</em> 2012</a>). Also, indel errors in SMRT assemblies can cause frame-shifts and create difficulties for annotation via homology search (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Du and Sun 2016</a>) or could lead to false-positives in detection of frame-shift mutations.</p><p id="p-32">At the order of 5.7 × 10<sup>−5</sup> indels per bp our SMRT assembly already shows a lower error rate than error rates previously recorded for HGAP assemblies of SMRT sequences at: 99.9995% concordance with Sanger Sequences of microorganism genomes at ∼80-100 × coverage (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2013</a>), though this study uses a higher coverage of 136 – 2690 x. Also, the assemblies performed better than error rates estimated for an <em>S</em>-locus study which found an average of 0.009 indel errors per bp (range 0–0.05), and an average of 0.02 substitutions errors per bp (range 0-0.1) based on 454 sequencing of <em>SRK</em> amplicons (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jørgensen <em>et al.</em> 2012</a>).</p><p id="p-33">The high accuracy even before error correction with short reads is likely owed to the fact that several Quiver polishing steps (see Materials and Methods) already work well at removing assembly errors if, as in our case, the coverage of long reads is high enough (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2013</a>).</p></div><div id="sec-13" class="subsection"><h3>Annotation of the S-locus</h3><p id="p-34">Annotation of our <em>S</em>-locus assemblies showed that this strategy resulted in full-length <em>S</em>-locus sequences (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>) containing both the <em>U-box</em> and <em>ARK3</em> flanking genes, as well as the key <em>S</em>-locus genes <em>SRK</em> and <em>SCR</em>. In CgrS-BAC1, <em>SCR</em> was not successfully annotated. The gene is known to be difficult to annotate due to its short nature and hyper variability. A phylogenetic tree of our <em>SRK</em> sequences and a set of publicly available <em>SRK</em> sequences confirms that our data falls within the range of sequence diversity observed at this locus in the Brassicaceae (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). The sequence similarity drops steeply at the genes bordering the <em>S</em>-locus, <em>ARK3</em> and <em>U-box</em> (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>), and the only large region showing sequence conservation within the <em>S</em>-locus correspond to the gene <em>SRK</em>, a genetic determinant of self-incompatibility, as has also been found previously (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2011</a>; <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Goubet <em>et al.</em> 2012</a>).</p></div><div id="sec-14" class="subsection"><h3>Cost and feasibility</h3><p id="p-35">Aligning short-read data to SMRT-assemblies for error correction eliminates the necessity of additional (PCR-based) validation, which enables a faster and simpler workflow, once the assembly and error correction is complete. SMRT sequencing is still relatively costly, adding to the costs of BAC library production (∼1700 € at time of publishing), but for certain studies long reads are indispensable, for instance to assemble regions of high repeat content and to accurately assemble intergenic regions (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Rhoads and Au 2015</a>). Using a double platform approach takes more financial resources, time and data processing, but can generate assemblies of higher accuracy than SMRT sequencing alone (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Rhoads and Au 2015</a>).</p><p id="p-36">High quality assemblies are necessary for many genetic studies, by the alignment of short read data directly to SMRT long reads, hybrid software are able to improve the accuracy of SMRT sequencing long reads (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Au <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Koren <em>et al.</em> 2012</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hackl <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Salmela and Rivals 2014</a>), <em>e.g.</em>, PBcR from ∼85% up to 99.9% (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Koren <em>et al.</em> 2012</a>), which can then be <em>de novo</em> assembled with higher confidence. Hybrid assemblies however are computationally intensive, especially early programs (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Au <em>et al.</em> 2012</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Koren <em>et al.</em> 2012</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Salmela and Rivals 2014</a>), as they must allow for more mismatches between short and long reads than other assembly methods. The approach of using short reads to error correct SMRT assemblies is a computationally simpler and efficient way to generate highly accurate assemblies.</p></div><div id="sec-15" class="subsection"><h3>Conclusions</h3><p id="p-37">We show that SMRT sequencing of BACs is an efficient way to obtain high-quality assemblies of the Brassicaceae <em>S</em>-locus, a locus that has been difficult to study due to its high content of repeats and high divergence among alleles. Independent SMRT sequencing runs of the same BAC clone allow us to estimate an error rate of 5.7 × 10<sup>−5</sup> indels per bp. These errors can efficiently be corrected using short reads, and such correction is important especially in the context of highly accurate studies of functional gene variants.</p><p id="p-38">This approach can be useful for studies of other genomic regions characterized by high divergence and repetitive content, such as other loci under long-term balancing selection (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fijarczyk and Babik 2015</a>), where reference based short-read sequencing technologies are not feasible.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">The authors thank Douglas Scofield and Christian Tellgren for bioinformatic assistance, Vincent Castric for discussion of <em>S</em>-locus evolution, Timothy Paape for useful comments on the manuscript, and Cindy Canton for assistance with plant work. The authors acknowledge the French Plant Genomic Resource Centre (INRA-CNRGV) for providing BAC library construction and screening. Sequencing was performed by the Uppsala Genome Centre and the SNP&amp;SEQ Technology Platform in Uppsala. These facilities are part of the National Genomics Infrastructure (NGI) Sweden and Science for Life Laboratory. The SNP&amp;SEQ Platform is also supported by the Swedish Research Council and the Knut and Alice Wallenberg Foundation. This study was supported by a grant from the Swedish Research Council to T.S.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-40">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300467/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300467/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-41"><em>Communicating editor: J. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-11-22"><span class="received-label">Received </span>November 22, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-02-20"><span class="accepted-label">Accepted </span>February 20, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Bachmann <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.1" data-doi="10.1371/journal.pone.0046679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Underwood</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving PacBio Long Read Accuracy by Short Read Alignment.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e46679</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAu%26rft.auinit1%253DK.%2BF.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46679%26rft.epage%253De46679%26rft.atitle%253DImproving%2BPacBio%2BLong%2BRead%2BAccuracy%2Bby%2BShort%2BRead%2BAlignment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046679%26rft_id%253Dinfo%253Apmid%252F23056399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056399&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.2" data-doi="10.1093/molbev/msl042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bechsgaard</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castric</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vekemans</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The transition to self-compatibility in <em>Arabidopsis thaliana</em> and evolution within S-haplotypes over 10 Myr.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1741</span>–<span class="cit-lpage">1750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msl042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl042%26rft_id%253Dinfo%253Apmid%252F16782760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16782760&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239905100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1327.3" data-doi="10.4172/2469-9853.1000128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtial</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cauet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodde</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vautrin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">Long Read Sequencing Technology to Solve Complex Genomic Regions Assembly in Plants.</span> <span class="cit-publ-name">Next Generat Sequenc &amp; Applic</span>. <span class="cit-vol">3</span>: <span class="cit-fpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4172/2469-9853.1000128</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLong%2BRead%2BSequencing%2BTechnology%2Bto%2BSolve%2BComplex%2BGenomic%2BRegions%2BAssembly%2Bin%2BPlants.%26rft.volume%253D3%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.4172%252F2469-9853.1000128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4172/2469-9853.1000128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina Sequence Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.5" data-doi="10.1093/molbev/msp224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castric</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechsgaard</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noureddine</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular Evolution within and between Self-Incompatibility Specificities.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msp224</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp224%26rft_id%253Dinfo%253Apmid%252F19773365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19773365&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272933900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.6" data-doi="10.1186/1471-2148-7-132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castric</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vekemans</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution under strong balancing selection: how many codons determine specificity at the female self-incompatibility gene <em>SRK</em> in Brassicaceae?</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-7-132</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DCastric%26rft.auinit1%253DV.%26rft.volume%253D7%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DEvolution%2Bunder%2Bstrong%2Bbalancing%2Bselection%253A%2Bhow%2Bmany%2Bcodons%2Bdetermine%2Bspecificity%2Bat%2Bthe%2Bfemale%2Bself-incompatibility%2Bgene%2BSRK%2Bin%2BBrassicaceae%253F%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-132%26rft_id%253Dinfo%253Apmid%252F17683611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17683611&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.7" data-doi="10.1111/eva.12291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The status of supergenes in the 21st century: Recombination suppression in Batesian mimicry and sex chromosomes and other complex adaptations.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">74</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12291</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D9%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12291%26rft_id%253Dinfo%253Apmid%252F27087840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27087840&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.8" data-doi="10.1006/anbo.1999.1015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Population-level studies of multiallelic self-incompatibility loci, with particular reference to Brassicaceae.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/anbo.1999.1015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D85%26rft.issue%253Dsuppl_1%26rft.spage%253D227%26rft.epage%253D239%26rft.atitle%253DPopulation-level%2BStudies%2Bof%2BMultiallelic%2BSelf-incompatibility%2BLoci%252C%2Bwith%2BParticular%2BReference%2Bto%2BBrassicaceae%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbo.1999.1015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/anbo.1999.1015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.9" data-doi="10.1093/molbev/msg170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartolomé</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">Haplotype Structure of the Stigmatic Self-Incompatibility Gene in Natural Populations of <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1741</span>–<span class="cit-lpage">1753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msg170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg170%26rft_id%253Dinfo%253Apmid%252F12832651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12832651&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186618200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartolomé</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">Diversity and linkage of genes in the self-incompatibility gene family in <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1519%26rft.epage%253D1535%26rft.atitle%253DDiversity%2Band%2BLinkage%2Bof%2BGenes%2Bin%2Bthe%2BSelf-Incompatibility%2BGene%2BFamily%2Bin%2BArabidopsis%2Blyrata%26rft_id%253Dinfo%253Apmid%252F12930757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.11" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klammer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.12" data-doi="10.1093/bioinformatics/btw458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improve homology search sensitivity of PacBio data by correcting frameshifts.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">i529</span>–<span class="cit-lpage">i537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw458</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw458%26rft_id%253Dinfo%253Apmid%252F27587671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27587671&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.13" data-doi="10.1126/science.1259442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meheust</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soucaze</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goubet</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallina</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dominance hierarchy arising from the evolution of a complex small RNA regulatory network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>(<span class="cit-issue">6214</span>): <span class="cit-fpage">1200</span>–<span class="cit-lpage">1205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1259442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDurand%26rft.auinit1%253DE.%26rft.volume%253D346%26rft.issue%253D6214%26rft.spage%253D1200%26rft.epage%253D1205%26rft.atitle%253DDominance%2Bhierarchy%2Barising%2Bfrom%2Bthe%2Bevolution%2Bof%2Ba%2Bcomplex%2Bsmall%2BRNA%2Bregulatory%2Bnetwork%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259442%26rft_id%253Dinfo%253Apmid%252F25477454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxNC8xMjAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.14" data-doi="10.1126/science.1162986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eid</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Real-time DNA sequencing from single polymerase molecules.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>(<span class="cit-issue">5910</span>): <span class="cit-fpage">133</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1162986</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162986%26rft_id%253Dinfo%253Apmid%252F19023044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMC8xMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.15" data-doi="10.1111/mec.13226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fijarczyk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babik</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Detecting balancing selection in genomes: Limits and prospects.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">3529</span>–<span class="cit-lpage">3545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D3529%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.16" data-doi="10.1038/nrg.2016.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Coming of age: Ten years of next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">333</span>–<span class="cit-lpage">351</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.49</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.49%26rft_id%253Dinfo%253Apmid%252F27184599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27184599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.17" data-doi="10.1371/journal.pgen.1002495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goubet</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergès</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmstetter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Contrasted patterns of molecular evolution in dominant and recessive self-incompatibility haplotypes in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1002495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002495</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGoubet%26rft.auinit1%253DP.%2BM.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002495%26rft.epage%253De1002495%26rft.atitle%253DContrasted%2Bpatterns%2Bof%2Bmolecular%2Bevolution%2Bin%2Bdominant%2Band%2Brecessive%2Bself-incompatibility%2Bhaplotypes%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002495%26rft_id%253Dinfo%253Apmid%252F22457631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22457631&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.18" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msp259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854763&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273704400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.19" data-doi="10.1073/pnas.0808012106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechsgaard</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuffer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recent speciation of <em>Capsella rubella</em> from <em>Capsella grandiflora</em>, associated with loss of self-incompatibility and an extreme bottleneck.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">5246</span>–<span class="cit-lpage">5251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0808012106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808012106%26rft_id%253Dinfo%253Apmid%252F19307580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzEzLzUyNDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.20" data-doi="10.1104/pp.111.174912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of the S-Locus Region in <em>Arabidopsis</em> Relatives.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">937</span>–<span class="cit-lpage">946</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.111.174912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.174912%26rft_id%253Dinfo%253Apmid%252F21810962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTcvMi85MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.21" data-doi="10.1093/bioinformatics/btu392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackl</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedrich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Förster</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Proovread: Large-scale high-accuracy PacBio correction through iterative short read consensus.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">3004</span>–<span class="cit-lpage">3011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu392%26rft_id%253Dinfo%253Apmid%252F25015988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25015988&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-thesis" id="cit-8.4.1327.22"><div class="cit-metadata unstructured">Harris, R. S., 2007 Improved Pairwise Alignment of Genomic DNA. PhD thesis, Penn. State Univ.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.23" data-doi="10.1016/j.ygeno.2015.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heather</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chain</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The sequence of sequencers : The history of sequencing DNA.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">107</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ygeno.2015.11.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D107%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2015.11.003%26rft_id%253Dinfo%253Apmid%252F26554401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2015.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26554401&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.24" data-doi="10.1023/A:1026494212540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Balancing selection and MHC.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1026494212540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D104%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1026494212540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1026494212540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000080245100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.25" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome- database management tool for second- generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.26" data-doi="10.1080/17550874.2012.748098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jørgensen</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagesen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brysting</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using high-throughput sequencing to investigate the evolution of self-incompatibility genes in the Brassicaceae: strategies and challenges.</span> <abbr class="cit-jnl-abbrev">Plant Ecol. Divers.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">473</span>–<span class="cit-lpage">484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/17550874.2012.748098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BEcol.%2BDivers.%26rft.volume%253D5%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1080%252F17550874.2012.748098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/17550874.2012.748098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.27" data-doi="10.1534/genetics.107.072231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Linkage disequilibrium and recombination rate estimates in the self-incompatibility region of <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2357</span>–<span class="cit-lpage">2369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.072231</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.072231%26rft_id%253Dinfo%253Apmid%252F17565949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzIzNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.28" data-doi="10.1093/nar/gkf436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">3059</span>–<span class="cit-lpage">3066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkf436</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DMAFFT%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Brapid%2Bmultiple%2Bsequence%2Balignment%2Bbased%2Bon%2Bfast%2BFourier%2Btransform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf436%26rft_id%253Dinfo%253Apmid%252F12136088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkf436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12136088&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177154300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.29" data-doi="10.1038/nbt.2280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid error correction and de novo assembly of single-molecule sequencing reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">693</span>–<span class="cit-lpage">700</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2280</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKoren%26rft.auinit1%253DS.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D693%26rft.epage%253D700%26rft.atitle%253DHybrid%2Berror%2Bcorrection%2Band%2Bde%2Bnovo%2Bassembly%2Bof%2Bsingle-molecule%2Bsequencing%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2280%26rft_id%253Dinfo%253Apmid%252F22750884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22750884&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.30" data-doi="10.1105/tpc.13.3.627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kusaba</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwyer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendershot</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrebalov</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasrallah</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Self-incompatibility in the genus <em>Arabidopsis</em>: characterization of the S locus in the outcrossing <em>A. lyrata</em> and its autogamous relative <em>A. thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">627</span>–<span class="cit-lpage">643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.13.3.627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DKusaba%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D627%26rft.epage%253D643%26rft.atitle%253DSelf-Incompatibility%2Bin%2Bthe%2BGenus%2BArabidopsis%253A%2BCharacterization%2Bof%2Bthe%2BS%2BLocus%2Bin%2Bthe%2BOutcrossing%2BA.%2Blyrata%2Band%2BIts%2BAutogamous%2BRelative%2BA.%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.3.627%26rft_id%253Dinfo%253Apmid%252F11251101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNjI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.31" data-doi="10.1534/g3.114.010843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leducq</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosset</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gries</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">É.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Self-Incompatibility in Brassicaceae: Identification and Characterization of SRK -Like Sequences Linked to the S -Locus in the Tribe Biscutelleae. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">983</span>–<span class="cit-lpage">992</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.010843</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D4%26rft.spage%253D983%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.010843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.32" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.33" data-doi="10.1111/mec.14051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Llaurens</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whibley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joron</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic architecture and balancing selection: the life and death of differentiated variants.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">2430</span>–<span class="cit-lpage">2448</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D2430%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.34" data-doi="10.1038/nprot.2016.182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">DNA sequencing technologies: 2006–2016.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">213</span>–<span class="cit-lpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2016.182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D12%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2016.182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2016.182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.35" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal. 17: 10–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCUTADAPT%2BREMOVES%2BADAPTER%2BSEQUENCES%2BFROM%2BHIGHTHROUGHPUT%2BSEQUENCING%2BREADS%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miege</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruffio-Châble</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrillac</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Intrahaplotype polymorphism at the <em>Brassica</em> S locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMiege%26rft.auinit1%253DC.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D811%26rft.epage%253D822%26rft.atitle%253DIntrahaplotype%2BPolymorphism%2Bat%2Bthe%2BBrassica%2BS%2BLocus%26rft_id%253Dinfo%253Apmid%252F11606555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvODExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.37" data-doi="10.1126/science.287.5461.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dew</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasulo</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A Whole-Genome Assembly of Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>(<span class="cit-issue">5461</span>): <span class="cit-fpage">2196</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.287.5461.2196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2196%26rft.epage%253D2204%26rft.atitle%253DA%2BWhole-Genome%2BAssembly%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2196%26rft_id%253Dinfo%253Apmid%252F10731133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.38" data-doi="10.1073/pnas.0406970101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nasrallah</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman-Broyles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boggs</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasrallah</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Natural variation in expression of self-incompatibility in <em>Arabidopsis thaliana</em>: implications for the evolution of selfing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>(<span class="cit-issue">45</span>): <span class="cit-fpage">16070</span>–<span class="cit-lpage">16074</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0406970101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0406970101%26rft_id%253Dinfo%253Apmid%252F15505209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ1LzE2MDcwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.39" data-doi="10.1093/molbev/msw299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novikova</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchimatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nizhynska</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voronin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome sequencing reveals the origin of the allotetraploid <em>Arabidopsis suecica</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">957</span>–<span class="cit-lpage">968</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D957%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.40" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion torrent, pacific biosciences and Illumina MiSeq sequencers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-web" id="cit-8.4.1327.41"><div class="cit-metadata unstructured">R Development Core Team, 2008 R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria. ISBN 3–900051–07–0, URL <a href="http://www.R-project.org">http://www.R-project.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.42" data-doi="10.1016/j.molcel.2015.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reuter</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spacek</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Throughput Sequencing Technologies.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">58</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">586</span>–<span class="cit-lpage">597</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2015.05.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D58%26rft.spage%253D586%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.05.004%26rft_id%253Dinfo%253Apmid%252F26000844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000844&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.43" data-doi="10.1016/j.gpb.2015.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhoads</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PacBio Sequencing and Its Applications.</span> <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">278</span>–<span class="cit-lpage">289</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gpb.2015.08.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.volume%253D13%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gpb.2015.08.002%26rft_id%253Dinfo%253Apmid%252F26542840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gpb.2015.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26542840&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.44" data-doi="10.1093/bioinformatics/btu538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivals</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LoRDEC: Accurate and efficient long read error correction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3506</span>–<span class="cit-lpage">3514</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu538%26rft_id%253Dinfo%253Apmid%252F25165095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25165095&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.45" data-doi="10.1073/pnas.74.12.5463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicklen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">DNA sequencing with chain-terminating.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">74</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">5463</span>–<span class="cit-lpage">5467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.74.12.5463</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSanger%26rft.auinit1%253DF.%26rft.volume%253D74%26rft.issue%253D12%26rft.spage%253D5463%26rft.epage%253D5467%26rft.atitle%253DDNA%2Bsequencing%2Bwith%2Bchain-terminating%2Binhibitors%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.12.5463%26rft_id%253Dinfo%253Apmid%252F271968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzQvMTIvNTQ2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification and characterization of a polymorphic receptor kinase gene linked to the self-incompatibility locus of <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchierup%26rft.auinit1%253DM.%2BH.%26rft.volume%253D158%26rft.issue%253D1%26rft.spage%253D387%26rft.epage%253D399%26rft.atitle%253DIdentification%2Band%2BCharacterization%2Bof%2Ba%2BPolymorphic%2BReceptor%2BKinase%2BGene%2BLinked%2Bto%2Bthe%2BSelf-Incompatibility%2BLocus%2Bof%2BArabidopsis%2Blyrata%26rft_id%253Dinfo%253Apmid%252F11333247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzEvMzg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC80LzEzMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.47" data-doi="10.1104/pp.125.4.2095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiba</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A Pollen Coat Protein, SP11/SCR, Determines the Pollen S-Specificity in the Self-Incompatibility of <em>Brassica</em> Species.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">125</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2095</span>–<span class="cit-lpage">2103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.125.4.2095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DShiba%26rft.auinit1%253DH.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D2095%26rft.epage%253D2103%26rft.atitle%253DA%2BPollen%2BCoat%2BProtein%252C%2BSP11%252FSCR%252C%2BDetermines%2Bthe%2BPollen%2BS-Specificity%2Bin%2Bthe%2BSelf-Incompatibility%2Bof%2BBrassica%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.125.4.2095%26rft_id%253Dinfo%253Apmid%252F11299389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTI1LzQvMjA5NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.48" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.49" data-doi="10.1093/nar/gkh379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinkamp</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">AUGUSTUS: A web server for gene finding in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>(<span class="cit-supplement">Web Server</span>): <span class="cit-fpage">W309</span>–<span class="cit-lpage">W312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D32%26rft.issue%253Dsuppl_2%26rft.spage%253DW309%26rft.epage%253DW312%26rft.atitle%253DAUGUSTUS%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bgene%2Bfinding%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh379%26rft_id%253Dinfo%253Apmid%252F15215400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15215400&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222273100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.50" data-doi="10.1093/nar/gkq543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rank</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eid</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A flexible and efficient template format for circular consensus sequencing and SNP detection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">e159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq543%26rft_id%253Dinfo%253Apmid%252F20571086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20571086&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-8.4.1327.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angly</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">Next Generation Sequence Assembly with AMOS</span>. <span class="cit-publ-name">Current Protocols in Bioinformatics</span>. 33:11.8:11.8.1–11.8.18.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.52" data-doi="10.1093/molbev/msx122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuchimatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goubet</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallina</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holl</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fobis-Loisy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Patterns of polymorphism at the self-incompatibility locus in 1,083 <em>Arabidopsis thaliana</em> genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1878</span>–<span class="cit-lpage">1889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.53" data-doi="10.1371/journal.pgen.1002838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuchimatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yew</span>  <span class="cit-name-given-names">C.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachelier</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent loss of self-incompatibility by degradation of the male component in allotetraploid <em>Arabidopsis kamchatica</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1002838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTsuchimatsu%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002838%26rft.epage%253De1002838%26rft.atitle%253DRecent%2Bloss%2Bof%2Bself-incompatibility%2Bby%2Bdegradation%2Bof%2Bthe%2Bmale%2Bcomponent%2Bin%2Ballotetraploid%2BArabidopsis%2Bkamchatica.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002838%26rft_id%253Dinfo%253Apmid%252F22844253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22844253&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F4%2F1327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vekemans</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Gene and allelic genealogies at a gametophytic self-incompatibility locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1157</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVekemans%26rft.auinit1%253DX.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1157%26rft.epage%253D1165%26rft.atitle%253DGene%2Band%2Ballelic%2Bgenealogies%2Bat%2Ba%2Bgametophytic%2Bself-incompatibility%2Blocus.%26rft_id%253Dinfo%253Apmid%252F7982569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzQvMTMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.4.1327.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1939</span> <span class="cit-article-title">The distribution of self-sterility alleles in populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D538%26rft.epage%253D552%26rft.atitle%253DTHE%2BDISTRIBUTION%2BOF%2BSELF-STERILITY%2BALLELES%2BIN%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F17246937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyNC80LzUzOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNC8xMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/4/1327.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/4/1315" title="Examination of Signatures of Recent Positive Selection on Genes Involved in Human Sialic Acid Biology" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1335" title="A Genomic Reference Panel for Drosophila serrata" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/4"><span>Volume 8 Issue 4, April 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/4.cover-source.jpg?itok=wC37V0ox" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/4/1327.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gd3tH9NEc87cEm3Z_uWPPn7qEYXAPqdY4b9D6wETq9o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/4/1327" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/4/1327" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/4/1327" class="active" data-icon-position="" data-hide-link-title="0">Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in Capsella</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489124" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ct-MYCEgkDqup-g9A2xhDmkfiOw9R06TOaBJQqm7tO4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1327.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/4/1327" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NUk4u556YIGy7amO0HCRW2Q21R7Sxt-p24wg80U55CA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/4/1327" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/4/1327.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489124" id="node489124--2" data-pisa="ggg;8/4/1327" data-pisa-master="ggg;g3.117.300467" data-apath="/ggg/8/4/1327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in <em>Capsella</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jörg A. Bachmann</span>, <span class="highwire-citation-author" data-delta="1">Andrew Tedder</span>, <span class="highwire-citation-author" data-delta="2">Benjamin Laenen</span>, <span class="highwire-citation-author" data-delta="3">Kim A. Steige</span> and <span class="highwire-citation-author" data-delta="4">Tanja Slotte</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1327-1333;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300467 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489124--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jörg A. Bachmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rg%2BA.%2BBachmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachmann%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rg%2BA.%2BBachmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jorg.bachmann@su.se" class="" data-icon-position="" data-hide-link-title="0">jorg.bachmann@su.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Tedder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BTedder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tedder%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BTedder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Benjamin Laenen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BLaenen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laenen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BLaenen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kim A. Steige </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kim%2BA.%2BSteige%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steige%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKim%2BA.%2BSteige%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tanja Slotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BSlotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Slotte%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanja%2BSlotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/4/1327" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/4/1327&amp;title=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/4/1327&amp;title=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/4/1327&amp;title=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/4/1327&amp;text=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/4/1327&amp;title=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/4/1327&amp;t=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/4/1327&amp;title=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/4/1327&amp;title=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489124/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489124" id="node489124--3" data-pisa="ggg;8/4/1327" data-pisa-master="ggg;g3.117.300467" data-apath="/ggg/8/4/1327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Targeted Long-Read Sequencing of a Locus Under Long-Term Balancing Selection in <em>Capsella</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jörg A. Bachmann</span>, <span class="highwire-citation-author" data-delta="1">Andrew Tedder</span>, <span class="highwire-citation-author" data-delta="2">Benjamin Laenen</span>, <span class="highwire-citation-author" data-delta="3">Kim A. Steige</span> and <span class="highwire-citation-author" data-delta="4">Tanja Slotte</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1327-1333;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300467 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489124--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jörg A. Bachmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rg%2BA.%2BBachmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachmann%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rg%2BA.%2BBachmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jorg.bachmann@su.se" class="" data-icon-position="" data-hide-link-title="0">jorg.bachmann@su.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Tedder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BTedder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tedder%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BTedder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Benjamin Laenen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BLaenen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laenen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BLaenen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kim A. Steige </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kim%2BA.%2BSteige%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steige%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKim%2BA.%2BSteige%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tanja Slotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Ecology, Environment and Plant Sciences, Science for Life Laboratory, Stockholm University, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BSlotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Slotte%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanja%2BSlotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1327&amp;rft.epage=1333&amp;rft.atitle=Targeted%20Long-Read%20Sequencing%20of%20a%20Locus%20Under%20Long-Term%20Balancing%20Selection%20in%20Capsella&amp;rft.volume=8&amp;rft.issue=4&amp;rft.date=2018-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bachmann%2C+J%C3%B6rg+A.&amp;rft.au=Tedder%2C+Andrew&amp;rft.au=Laenen%2C+Benjamin&amp;rft.au=Steige%2C+Kim+A.&amp;rft.au=Slotte%2C+Tanja"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489124/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489124/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489124/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489124/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489124/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489124/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489124/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489124/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489124/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489124/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489124/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489124/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/4/1327" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/4/1327.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/4/1327.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/4/1327.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/4/1327.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
