<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/7/1165" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/7/1165.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/7/1165.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/7/1165.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/7/1165/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/7/1165.atom" />
<meta name="HW.pisa" content="ggg;3/7/1165" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Generalized Admixture Mapping for Complex Traits" />
<meta name="DC.Identifier" content="10.1534/g3.113.006478" />
<meta name="DC.Date" content="2013-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Zhu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Admixture mapping is a popular tool to identify regions of the genome associated with traits in a recently admixed population. Existing methods have been developed primarily for identification of a single locus influencing a dichotomous trait within a case-control study design. We propose a generalized admixture mapping (GLEAM) approach, a flexible and powerful regression method for both quantitative and qualitative traits, which is able to test for association between the trait and local ancestries in multiple loci simultaneously and adjust for covariates. The new method is based on the generalized linear model and uses a quadratic normal moment prior to incorporate admixture prior information. Through simulation, we demonstrate that GLEAM achieves lower type I error rate and higher power than ANCESTRYMAP both for qualitative traits and more significantly for quantitative traits. We applied GLEAM to genome-wide SNP data from the Illumina African American panel derived from a cohort of black women participating in the Healthy Pregnancy, Healthy Baby study and identified a locus on chromosome 2 associated with the averaged maternal mean arterial pressure during 24 to 28 weeks of pregnancy." />
<meta name="DC.Contributor" content="Bin Zhu" />
<meta name="DC.Contributor" content="Allison E. Ashley-Koch" />
<meta name="DC.Contributor" content="David B. Dunson" />
<meta name="article:published_time" content="2013-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Generalized Admixture Mapping for Complex Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Admixture mapping is a popular tool to identify regions of the genome associated with traits in a recently admixed population. Existing methods have been developed primarily for identification of a single locus influencing a dichotomous trait within a case-control study design. We propose a generalized admixture mapping (GLEAM) approach, a flexible and powerful regression method for both quantitative and qualitative traits, which is able to test for association between the trait and local ancestries in multiple loci simultaneously and adjust for covariates. The new method is based on the generalized linear model and uses a quadratic normal moment prior to incorporate admixture prior information. Through simulation, we demonstrate that GLEAM achieves lower type I error rate and higher power than ANCESTRYMAP both for qualitative traits and more significantly for quantitative traits. We applied GLEAM to genome-wide SNP data from the Illumina African American panel derived from a cohort of black women participating in the Healthy Pregnancy, Healthy Baby study and identified a locus on chromosome 2 associated with the averaged maternal mean arterial pressure during 24 to 28 weeks of pregnancy.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/07/01" />
<meta name="citation_mjid" content="ggg;3/7/1165" />
<meta name="citation_id" content="3/7/1165" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/7/1165" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/7/1165.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/7/1165.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/7/1165.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006478" />
<meta name="citation_pmid" content="23665878" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1165" />
<meta name="citation_lastpage" content="1175" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bin Zhu" />
<meta name="citation_author_institution" content="Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, Maryland 20850" />
<meta name="citation_author_email" content="bin.zhu@nih.gov" />
<meta name="citation_author" content="Allison E. Ashley-Koch" />
<meta name="citation_author_institution" content="Center for Human Genetics, Duke University Medical Center, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author" content="David B. Dunson" />
<meta name="citation_author_institution" content="Department of Statistical Science, Duke University, Durham, North Carolina 27708" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Pregnancy Childbirth;citation_author=V. Allen;citation_author=K. S. Joseph;citation_author=K. Murphy;citation_author=L. Magee;citation_author=A. Ohlsson;citation_title=The effect of hypertensive disorders in pregnancy on small for gestational age and stillbirth: a population based study.;citation_pages=17-17;citation_volume=4;citation_year=2004;citation_issue=1;citation_pmid=15298717;citation_doi=10.1186/1471-2393-4-17" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=A. Basu;citation_author=H. Tang;citation_author=C. E. Lewis;citation_author=K. North;citation_author=J. D. Curb;citation_title=Admixture mapping of quantitative trait loci for blood lipids in African-Americans.;citation_volume=18;citation_year=2009;citation_pmid=19304782;citation_doi=10.1093/hmg/ddp122" />
<meta name="citation_reference" content="citation_title=Adaptive Control Processes: A Guided Tour;citation_author=E. R. Bellman;citation_year=1961" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Y. Cheng;citation_author=D. Reich;citation_author=T. Y. Wong;citation_author=R. Klein;citation_author=B. E. K. Klein;citation_title=Admixture mapping scans identify a locus affecting retinal vascular caliber in hypertensive African Americans: the Atherosclerosis Risk in Communities (ARIC) study.;citation_pages=e1000908-e1000908;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20419148;citation_doi=10.1371/journal.pgen.1000908" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. L. Freedman;citation_author=C. A. Haiman;citation_author=N. Patterson;citation_author=G. J. McDonald;citation_author=A. Tandon;citation_title=Admixture mapping identifies 8q24 as a prostate cancer risk locus in African-American men.;citation_volume=103;citation_year=2006;citation_pmid=16945910;citation_doi=10.1073/pnas.0605832103" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Practical issues in imputation-based association mapping.;citation_pages=e1000279-e1000279;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19057666;citation_doi=10.1371/journal.pgen.1000279" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Stat.;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Bayesian variable selection regression for genome-wide association studies and other large-scale problems.;citation_pages=1780-1815;citation_volume=5;citation_year=2011;citation_doi=10.1214/11-AOAS455" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. J. Hoggart;citation_author=E. J. Parra;citation_author=M. D. Shriver;citation_author=C. Bonilla;citation_author=R. A. Kittles;citation_title=Control of confounding of genetic associations in stratified populations.;citation_pages=1492-1504;citation_volume=72;citation_year=2003;citation_issue=6;citation_pmid=12817591;citation_doi=10.1086/375613" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=V. E. Johnson;citation_author=D. Rossell;citation_title=On the use of non-local prior densities in Bayesian hypothesis tests.;citation_pages=143-170;citation_volume=72;citation_year=2010;citation_doi=10.1111/j.1467-9868.2009.00730.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. H. L. Kao;citation_author=M. J. Klag;citation_author=L. A. Meoni;citation_author=D. Reich;citation_author=Y. Berthier-Schaad;citation_title=MYH9 is associated with nondiabetic end-stage renal disease in African Americans.;citation_pages=1185-1192;citation_volume=40;citation_year=2008;citation_issue=10;citation_pmid=18794854;citation_doi=10.1038/ng.232" />
<meta name="citation_reference" content="citation_title=Generalized Linear Models;citation_author=P. McCullagh;citation_author=J. A. Nelder;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. McKeigue;citation_title=Mapping genes that underlie ethnic differences in disease risk: methods for detecting linkage in admixed populations, by conditioning on parental admixture.;citation_pages=241-251;citation_volume=63;citation_year=1998;citation_issue=1;citation_pmid=9634509;citation_doi=10.1086/301908" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. McKeigue;citation_title=Prospects for admixture mapping of complex traits.;citation_pages=1-7;citation_volume=76;citation_year=2005;citation_issue=1;citation_pmid=15540159;citation_doi=10.1086/426949" />
<meta name="citation_reference" content="citation_journal_title=Epidemiol. Rev.;citation_author=M. L. Miranda;citation_author=P. Maxson;citation_author=S. Edwards;citation_title=Environmental contributions to disparities in pregnancy outcomes.;citation_volume=31;citation_year=2009;citation_pmid=19846592;citation_doi=10.1093/epirev/mxp011" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=G. Montana;citation_author=J. K. Pritchard;citation_title=Statistical tests for admixture mapping with case-control and cases-only data.;citation_pages=771-789;citation_volume=75;citation_year=2004;citation_issue=5;citation_pmid=15386213;citation_doi=10.1086/425281" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Myers;citation_author=L. Bottolo;citation_author=C. Freeman;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A Fine-Scale Map of Recombination Rates and Hotspots Across the Human Genome;citation_pages=321-324;citation_volume=310;citation_year=2005;citation_issue=5746;citation_pmid=16224025;citation_doi=10.1126/science.1117196" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Patterson;citation_author=N. Hattangadi;citation_author=B. Lane;citation_author=K. E. Lohmueller;citation_author=D. A. Hafler;citation_title=Methods for high-density admixture mapping of disease genes.;citation_pages=979-1000;citation_volume=74;citation_year=2004;citation_issue=5;citation_pmid=15088269;citation_doi=10.1086/420871" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. L. Price;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=K. C. Barnes;citation_author=N. Rafaels;citation_title=Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.;citation_pages=e1000519-e1000519;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543370;citation_doi=10.1371/journal.pgen.1000519" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Reich;citation_author=N. Patterson;citation_author=P. L. De Jager;citation_author=G. J. McDonald;citation_author=A. Waliszewska;citation_title=A whole-genome admixture scan finds a candidate locus for multiple sclerosis susceptibility.;citation_pages=1113-1118;citation_volume=37;citation_year=2005;citation_issue=10;citation_pmid=16186815;citation_doi=10.1038/ng1646" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. Reich;citation_author=N. Patterson;citation_author=V. Ramesh;citation_author=P. L. De Jager;citation_author=G. J. McDonald;citation_title=Admixture mapping of an allele affecting interleukin 6 soluble receptor and interleukin 6 levels.;citation_pages=716-726;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357077;citation_doi=10.1086/513206" />
<meta name="citation_reference" content="citation_journal_title=J. Med. Genet.;citation_author=M. L. Scherer;citation_author=M. A. Nalls;citation_author=L. Pawlikowska;citation_author=E. Ziv;citation_author=G. Mitchell;citation_title=Admixture mapping of ankle–arm index: identification of a candidate locus associated with peripheral arterial disease.;citation_volume=47;citation_year=2010;citation_pmid=19586928;citation_doi=10.1136/jmg.2008.064808" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=S. L. Scott;citation_title=Bayesian methods for hidden Markov models.;citation_pages=337-351;citation_volume=97;citation_year=2002;citation_doi=10.1198/016214502753479464" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. W. Smith;citation_author=S. J. O’Brien;citation_title=Mapping by admixture linkage disequilibrium: advances, limitations and guidelines.;citation_pages=623-632;citation_volume=6;citation_year=2005;citation_issue=8;citation_pmid=16012528;citation_doi=10.1038/nrg1657" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. W. Smith;citation_author=N. Patterson;citation_author=J. A. Lautenberger;citation_author=A. L. Truelove;citation_author=G. J. McDonald;citation_title=A high-density admixture map for disease gene discovery in african americans.;citation_pages=1001-1013;citation_volume=74;citation_year=2004;citation_issue=5;citation_pmid=15088270;citation_doi=10.1086/420856" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=D. Reich;citation_title=Ancestry informative marker panels for African Americans based on subsets of commercially available SNP arrays.;citation_pages=80-83;citation_volume=35;citation_year=2011;citation_issue=1;citation_pmid=21181899;citation_doi=10.1002/gepi.20550" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A haplotype map of the human genome.;citation_pages=1299-1320;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16255080;citation_doi=10.1038/nature04226" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the lasso.;citation_pages=273-282;citation_volume=73;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=C. A. Winkler;citation_author=G. W. Nelson;citation_author=M. W. Smith;citation_title=Admixture mapping comes of age.;citation_pages=65-89;citation_volume=11;citation_year=2010;citation_pmid=20594047;citation_doi=10.1146/annurev-genom-082509-141523" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. T. Wu;citation_author=Y. F. Chen;citation_author=T. Hastie;citation_author=E. Sobel;citation_author=K. Lange;citation_title=Genome-wide association analysis by lasso penalized logistic regression.;citation_volume=25;citation_year=2009;citation_pmid=19176549;citation_doi=10.1093/bioinformatics/btp041" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. J. Yang;citation_author=C. Cheng;citation_author=M. Devidas;citation_author=X. Cao;citation_author=Y. Fan;citation_title=Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia.;citation_pages=237-241;citation_volume=43;citation_year=2011;citation_issue=3;citation_pmid=21297632;citation_doi=10.1038/ng.763" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=X. Zhu;citation_author=R. S. Cooper;citation_author=R. C. Elston;citation_title=Linkage analysis of a complex disease through use of admixed populations.;citation_pages=1136-1153;citation_volume=74;citation_year=2004;citation_issue=6;citation_pmid=15131754;citation_doi=10.1086/421329" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhu;citation_author=A. Luke;citation_author=R. S. Cooper;citation_author=T. Quertermous;citation_author=C. Hanis;citation_title=Admixture mapping for hypertension loci with genome-scan markers.;citation_pages=177-181;citation_volume=37;citation_year=2005;citation_issue=2;citation_pmid=15665825;citation_doi=10.1038/ng1510" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=X. Zhu;citation_author=J. H. Young;citation_author=E. Fox;citation_author=B. J. Keating;citation_author=N. Franceschini;citation_title=Combined admixture mapping and association analysis identifies a novel blood pressure genetic locus on 5p13: contributions from the care consortium.;citation_volume=20;citation_year=2011;citation_pmid=21422096;citation_doi=10.1093/hmg/ddr113" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=H. Zou;citation_author=T. Hastie;citation_title=Regularization and variable selection via the elastic net.;citation_pages=301-320;citation_volume=67;citation_year=2005;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Generalized Admixture Mapping for Complex Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="twitter:description" content="Admixture mapping is a popular tool to identify regions of the genome associated with traits in a recently admixed population. Existing methods have been developed primarily for identification of a single locus influencing a dichotomous trait within a case-control study design. We propose a generalized admixture mapping (GLEAM) approach, a flexible and powerful regression method for both quantitative and qualitative traits, which is able to test for association between the trait and local ancestries in multiple loci simultaneously and adjust for covariates. The new method is based on the generalized linear model and uses a quadratic normal moment prior to incorporate admixture prior information. Through simulation, we demonstrate that GLEAM achieves lower type I error rate and higher power than ANCESTRYMAP both for qualitative traits and more significantly for quantitative traits. We applied GLEAM to genome-wide SNP data from the Illumina African American panel derived from a cohort of black women participating in the Healthy Pregnancy, Healthy Baby study and identified a locus on chromosome 2 associated with the averaged maternal mean arterial pressure during 24 to 28 weeks of pregnancy." />
<meta name="og-title" property="og:title" content="Generalized Admixture Mapping for Complex Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/7/1165" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Admixture mapping is a popular tool to identify regions of the genome associated with traits in a recently admixed population. Existing methods have been developed primarily for identification of a single locus influencing a dichotomous trait within a case-control study design. We propose a generalized admixture mapping (GLEAM) approach, a flexible and powerful regression method for both quantitative and qualitative traits, which is able to test for association between the trait and local ancestries in multiple loci simultaneously and adjust for covariates. The new method is based on the generalized linear model and uses a quadratic normal moment prior to incorporate admixture prior information. Through simulation, we demonstrate that GLEAM achieves lower type I error rate and higher power than ANCESTRYMAP both for qualitative traits and more significantly for quantitative traits. We applied GLEAM to genome-wide SNP data from the Illumina African American panel derived from a cohort of black women participating in the Healthy Pregnancy, Healthy Baby study and identified a locus on chromosome 2 associated with the averaged maternal mean arterial pressure during 24 to 28 weeks of pregnancy." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
    <title>Generalized Admixture Mapping for Complex Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"v__D7FN9cg7YtQQflZOLUSclDK-37TJ-eO7ukWmYFZ0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459491","apath":"\/ggg\/3\/7\/1165.atom","pisa":"ggg;3\/7\/1165","ac":{"\/ggg\/3\/7\/1165.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/7\/1165.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1165"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1165"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/7\/1165":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mm+wIke0us7q2xHUq3Xg6c8EnkkdBaaAZ6riOETeiMry8oINZjqTwGxSSFNpBpJ4v6J4hiXdWC4YH0nLzP2eNjrncuyrQucXiGWMg7Tpi2mFM3g3ryoGBhVXVS2wVGnqN5S5+rhs2G5e+ABbpVj8VRP0OTh1SFEjwuONivqnlklaH5\\\\\\\/XIkXTaJ2Mz1x6eUrSkmPJAGV5cGVyT7tJ4\\\\\\\/GHJA6pLRPgqdKAbO0oUKBs1sj218RiGidwtBWZO2cLbMIYa16E8+pTqje2v2sn9GqC4r4Jhag7LbonCQ1Jex\\\\\\\/9BFKMdzUwtt46KRthSUoID6O4C96Pk97FlW\\\\\\\/A9TxUs81K5rQkL0pxZOcsu9ATFbAb5ytycpma3p7qe9gjpgKmt4Lr41UsKvwHRcxZ9YAFx3y61aPkgX6ISYfiXf+65b1sUUvLAXCFxnrEMbyiE1VqnNeK7l6BWGzv1YtDBjAllUGHsuMv\\\\\\\/Nc6dr5c77lgojxigcthVWtcsGDewd55JttJ51+FeL0ztbxnoxxNe7FHHxzhYzA2PQdj9jLcfmvYuImkvhajLHYkS5pdOGb7HRePuuIcp7w6j3T\\\\\\\/tR8\\\\\\\/uw61ueIg2zTjdveJToucafqBTkVbAaHmTzEBJ0ucdDDNPV6eYaitZcZXDeqx2ughDITlhglPdYlDvpStC3Bn5upkTY8l8yLEXarSQKBMPqY7QZj9S1gA5IfDBMh3kylSgkdRC+jnGwnXPfenKPBq81Dvp4acQtWPY5TtRHW1F+b6JE9+yUxVnquN0b2Z9tVhX2iYEz7U6onZqjZYxo4X7AlFLNPqvTRiDEGsaXbGGfNOTZTypLcobtlvYDqF9YhhuozHo3WgxakizRBMeBvp0sJzLz5dkux0eJaevu8nnAnVWgSP\\\\\\\/gc4H3wrMENoocq2\\\\\\\/z+bEw5zou6IYzrw6k7fhuGTy6M1qPeq+ufrOdCr3lbFjh5DwwkRmz+KX72bPNyhYQ9IKY8LX2rbD6M9wb17iDp7EX5NvlvvMgeqchdBWV+3+HLlf0LGXTgiPZz06K1hLzgVCLt4Pt8eNC\\\\\\\/oP4I4nCvD7jOyNB6RaBzIYBq7CMHMoTCFDQ5Q+E1FvoGjeTDKGYjmPJZo7E44ABs29z8p7dzYlD5kpax+IwjWeRYKcRM\\\\\\\/P6yJ8E\\\\\\\/v1hzuqMAfbGvqmg+IqebaKTMJ2OTvcm7KeS0vTZCLIES691lVK92JCnXq69QRAvOc63ikhOg1C8yHTzUcoiTGAGMuBE7UKuELplTZgQ3u65Zxaz5TT2HCrnmwPymdRFyG\\\u0022,\\\u0022iv\\\u0022:\\\u0022VLrjEn0UlgalvH\\\\\\\/9PdNO2g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f76c5c7c1d9956959f7315fe772bcb8d39ae25a94a183446e6e0d88fa7ce1e7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228XhKYJPjPett6s7ntWrxiHl4+H13TSOJLkq7bH\\\\\\\/Ex6Oxi2toqTe+8YvhRb+6t2Vc1a\\\\\\\/SMxKxtOq3KJr8yh+dBayvdLUkE+RGHV0+S6P1mwHe4u8gdh6dJWWnJGQdEvjLJvNMgkjDFvCmu0mCi6wL2Op1ggxW3xgTULijQ\\\\\\\/MpzfxLNj3NS0a5QdpGfZEdeTR6pi0Rkz+br63vQhVAlSQC1ceTaDih+RtpXjCLj3Niq3oXvisedulF0gYcgqO8UO0\\\\\\\/vzM2WYvxpaZU9MahjJP0zFEslkpmqt6cqDehmmC6YhYN1p5b2oahYktLBOqYFUwDEkCZlJhTp92xUq2\\\\\\\/lum8P0X42A5Akj29jARJc7aXS4FPrOROsnASrPNSxqKu7GUD2O1d9nedMXhZ13vqgOscwH9jsyQAY8XtZGaft+179rMojvVGMKqb\\\\\\\/kzi0MhVYB3rcm0AQ484paf6kZfPhDtZ+3qmpmAHm54xoJ9\\\\\\\/VRPM22eJUCjvX5a+Q8bUV5YpBYmaK04sfSuKGaF0h42J58fUPqVE1Mfkuujm1euhKKShzCZzs1OzcGAi19TuoF6QPVEl\\\\\\\/1gYvsbqIr1qo0e0tkfKZVAWMAps1CYl33dOm137fwe6vRVboD6DPs6n5uYDt0x7LJzghzTo98Ird8k9SUwKmMyRo3eQ3OoJ2+7ilmj8YsVQxLPOlKlsRCY0CkB+3KOep91p7eYvDiDn9wmRfa8PBwRQ2NuEY4+4E31JgYFa\\\\\\\/1ioJ3+5gBV71IEt70YmB6VAbHnc+i1TPWUN3skmkzOKVqawCLDy5Ti3XZOpnvTStZ5M\\\\\\\/j0PP8O08Dt\\\\\\\/F5qURt3DOkPNnCYNFGTOkKZoYoQwUkqjae0JSjhpZOC1i3tFmi7EMDh2X3\\\\\\\/uPAtnf+q9Ez9uU+T1DxNviFn7tk4wOIIaDGiaYVlVgNA2\\\\\\\/P1izpTlmX4eqN4luTY4kJeZ5JXNIDk29ibKtAHEqY4GdZ0Wx\\\\\\\/ChP2MINQF9P0JGgwPhNCRjdgNN8vlU1UdHVX7nCOw0ygYbgBrxxNR5bOMwcqznjcHoUZwr+HS1SzC2sDQiFbEYnumaG4Svg93vYpZNio9\\\\\\\/O8Kd4Rh6D3Wtvw4iADYbja3JPYJlTCIkZEwXfDUKFeUQ+HWGRVkCM\\\\\\\/u8LqXVpX3nj8uoURcxCUueBfEEjDAqxa19+121LFN6ytVC6TrxJXRnLmcaulV\\\\\\\/7dJSuebpVk8CrSJSQ8iMrVMrIKUFhorFyrvcxypRp3ZsOMYa3Gd4kBIupX5ZiYqFuVqb4H6cvmuT4dCxy\\\\\\\/Wc1eV89rkub9iUeOBN31fVi4K+O2KhxoNww0nAtFnrUq6ITRYJtvvTqTiAwd2ac6BB4d4Hh2LOqy4zXN9h1qtQ8xhz1frXNgAaavJS7g2FjFHq49PlykEkLLv+V0Xb09DP2rmDXkCIDRe8ytnmyu\\\\\\\/W+uWC\\\\\\\/In+1WAlvwZ47hdWjoG+qyvXRzrosDY88Zi9ICmCI7qsauFXx6wRKEtfntZOD7RWjyQiJD38Uzr8J5Tf3VrkwYjWgnjNbjhbwIzV61mPINJTIadj9RTB6l7+pBeElvVREsTqMgE9e4jR8V2\\\\\\\/sNKsjF\\\\\\\/dk1XnNrB2pMvW0Qom9Ryu7oTi9GT9OS2D8Uvyc26Q+WP3aMm\\\\\\\/9OnVx4M0SvgDF3yVN0GYTUMBnBGebALlWcoFypSomXZPAgbXt8NMb1FlefcXR9Ps4ePyDxLuknDG7r3fEYylNjnZt54wmB3UZzLxldTrY5sRq98ibTu7RJNb7nCGRhqbJw0K\\\\\\\/9KvwdALWxFmQHj2GyfZKYp89RTg+M0lzQrCvzn\\\\\\\/QNb7CkpEfUnepBy0nIwQFkEvUeP8Uj424Lw02HEgQI8V9e0N4YjR2kAnJDpXwd5DFOEuRxmEK\\\\\\\/M9aso+Zsdb9qhk8FjtcXwhCfpw3PUCgJriKFMuIc53NuhH7YULL2FiiH7bcQ9YM7AhmKtXNdzYosHXHv3ipGCOOp3BgOtTfH5lTI996YADc26c3+ZEfHqxtyKanPNja3EvX\\\\\\\/g1ufxp8\\\\\\\/ubsMjRHDXg75smptTZkx1JCgnr4+nSG17RYVD3IAeG+XWHohkywtN5EpA9V6FEpSs4opD0rAqAYfwDebCZvlnzWNoVviVTZT+fFPXQLxaKBWF7x5jEyMn7HfSmkfe49GkeKR1vBb8VYANx\\\\\\\/qZ3RPUldsCAgMs4CgscIJwiulUzm3fWidJaKbw+AXsDk\\\\\\\/MlC3dTg5KY\\\\\\\/6PXSbUwQw9m04rJyD+jBpx5sLc8\\\\\\\/SgRKYVrbQwkQad9T5Albb\\\\\\\/pXeJ8P\\\\\\\/6WogGFzfZ6Wgam\\\\\\\/VBAKqZDkKTFyarDvHLk1uLbc7qusMLdLgAcK7DMnpWRfLpG+NY5O+39uhoIhbP7RVujhysOqj2uLMW\\\\\\\/IchgLUI2WqJKtNCmtAWFNY97whPiuHPjYf2IPkUf0tZrcH1FcbDcnP35\\\\\\\/AuaCH93Syu4cv7+lhLPCbwc\\\\\\\/VBNtK5uVSS8YqnMBi6CJz6cabHb\\\\\\\/YPgqTa0ctaBtC9n8ZTEz6TedIEcjf58k9F25qtD4\\\\\\\/GkAdoRNsAT7QG2Y43lFfMchEcSdVOk9dFX9BMvrumbhVUJ\\\\\\\/tpURx+tJep6npScgLjOMflZMzk0PaJi2PjMsJguE\\\\\\\/LtAs\\\\\\\/37wbQmXJlTYCNgvQi0nala0i+I66wpWEF2YNp\\\\\\\/OKlsJaMnGhEIEArofgz89jn3LHUaQnXXlEPUUA0qD4I6jRUuLffSRSb09lzg4UbKJKs4\\\\\\\/4vqjQdYLF7ULNAGnfii1q62DsKC+0i1s9zxL9ftXuCwgmeKtZHw3L7LbGk7Aw3sY22SfxH9oJeo1K7ldchuoBU57NRQZohoD6CfUeJADsjNXiwuOfLN0lnBdmQ3NVFfKAFIIuXhMKDFu2d4RLha5b7wq84RApAtEjjBXXyqQRH4HEM3GADIrl\\\\\\\/XuEZFAk3pqt0MrZsD0jDqIeKPf56W3GhExhUVecajR4WBNSEvii0\\\\\\\/7Hl4u72TyDAH2QCkyteCfRCK\\\\\\\/Medvgy9zQ2Zs7H3EUCH5g9eDT146zccn3YK4g\\\\\\\/OVV2geIcJGN21gK0fwJ7ka\\\\\\\/5wJKoFRA4m03Z87Qchbew6wM3MCrCUv+mAU0fsn5ssXo51sN+JKfXsQWYyLjmog+ycr2t2e4zKMvMUkM4YWWj5xAQqapufVCW77+595AuMMBPsjbtqWTDxo1VvI5M3i1MF6marXVtxuVE3GAsdtSeBC45ifzAZFJw8rSNi9V3Azjxh+b1su1\\\\\\\/evBi19XDwbEebqF5Ti6Hfoeqc94fHB92a\\\\\\\/OK9D9HyFZWRY5AOkdc0n5lxoGNED2bWJnQ4jclQDIq9w3LsArvaSi6QtvnEVtwqNSBZxgbgBfGJaBxCGgGrsc50tfaHRPqaDDnnbPRXBx2wmOVkG1y5YJDEtzEELRmJJUzuLrLUTPaMvGEKC5PMlNNnz4HjKwMet\\\\\\\/aMZvRaZ3dE5BnRLBf5oI+rIv81fK3GhUZzpznaXRJBpN42tDoSdndE3yDYCElHBLd\\\\\\\/38vZkJSfqrikcm8PE3T0MsUgUwM+oRXeW3bj\\\\\\\/wSsuAbmOGv+7D6H4gN4pYXsDChWkvO3URBsBPNnw6IijTm5RsUlABpKc27krfLbeclBUC4\\\\\\\/0meTtmCQN+Yjpxxb+e24+mvGjspRiL2bmJXOIhESc3XhYcqD6BB7GddGgMlfODgfNpGdboBdWe3sVBFOA09G9LGCpGICKwCCFSdcDDCOBdnFx9Aa5tUBvDZk5e7Vq5PrxLKPXoamSDwUxhj7wYhYr+gTHj18UM\\\\\\\/OIuXbhvD1oyqZheuWr45lbhXrIWq60JZk1cJMyS2mKOXIMUOZBMVaYgpMX6N3QYQTSy+kFjQVKUuSGuAs5S0OK1GK31ySbWHJL2S9E2HzEGtgPGIg3P+hcLkml1276GF\\\\\\\/BJO6UAO1z+zL6h+6U+QrbHosayDbuum9pSpBdaCfTKpG\\\\\\\/oLBSCz\\\\\\\/81jSYEXUZQVhnjAiLbhQP3WY\\\\\\\/AuCe6Ee794+PxHU0+cyMW1s04jER+4RMZr\\\\\\\/rW1AX90Y4QO6lJF5CcdGXD6Am\\\\\\\/2kAjw0RWc0aGROFz1apU7XfD2LfOGJ1SF36ES+Vv02zTpO\\\\\\\/DAMVsrfbkEmEW8G9IpTpmW32OLYFRdhLWCFe\\\\\\\/\\\\\\\/eWZXzjfqLMCfng57I2JBBvirmXEAdF3VP1Q0JYh1bYLtB99DD01QLMEOYmk6SI8kAZuwUL8C\\\\\\\/4N48tmv6BO6\\\\\\\/iZTDcLe5xe1ElRr7F8D+hCAypHRAPI+aGBrlC6Zzvo70yYo8TKMHHuEnaG2\\\\\\\/911L+pMfaxrHXdFcCUkS91\\\\\\\/OQMN1D4EQosbuyeO9mybzWdbYGnEfVJUxJRMBQ0PkJ+Nc+PCOBdetNs\\\\\\\/TgB7PJ8fR8hlXwCKfgYg2SOPp+c4FvYiGsHXK\\\\\\\/mFe7iIYhRqNXt\\\\\\\/heAaM9pyV+oUxU3iVygaC9N2SEJsoaOC8\\\\\\\/SM8CVx2KQVQzpBltr+udV1iwZlVarXl\\\\\\\/sTbGD4dOgEa3oOsIKEJ8AjF6gbFm3MmLlYzyzmsYatEtZKX\\\\\\\/fyHDKUBb+d3mFCtvr1vFUGOV4+MztE95CHbI12cQRWCjINHr2u1qM0q8gIDqAVURRxhJfzsOembckiwiXsVZNLoxCpZJ1eplBOtnjTIqLM0GElQpze484DxFKwdLYTGJGwaQIPHuRXgj2+VQKzNAmDQgC7beXo9zlP9FQkB+x\\\\\\\/EXmiOFpkr4S0lvRcxm5NoSPpwHOePxyuvnGFQMHqMVvf0yBKLZfZ6rUN9uV6nqhWfWbbaU0xWnGVkhMPgZXllElMNl5zdfCrYbtRY+a0CPzLgPud0Ew5+kJEYXIRA5UypND7BZvcTqbC5aUAHduGMJq1dkHPIUDwt3uyPTn8UsDKbagWiPReEJFGMLmiEyalpaioiq7YeSPAKFbfpUBKFNiJPeb2FJ2+oXBZIYuDBduG7ykCLe5Csb\\\\\\\/3I2JrdYpRb7ZVJ6s2NB7GYA0BVTjSmhcEB\\\\\\\/ueX7LUV\\\\\\\/v9YrsVE83rza31ZQ1y\\\\\\\/SkGtUYCzoi8NmeQyEqqnn6miRfhrbz5LKV7zS1dviMUyRZC0N6FGaNhR79AaKObvFy5Q26T5cpsXehTOiNpEZSUy6ly+t4cu9SiiKd7R5zZcH9fJHAWqFU2NObNGu4DUPIU56mnZvjSJEgkYNpMsrdV4Ypf9kFUEIY1xpuREMeTGr4rcGydLZp1I2pdT3YewNeJIS6oCUdwnJxollj6KdEzMwLIOFPu\\\\\\\/ijCEmAswBuB\\\\\\\/t\\\\\\\/b1cxhPTULw8yKmNDSPMdq2NpBAn5+7yIvgx53jtYgJZUnglP9saYYg7\\\\\\\/q5jo5pZNCH0SvA10zqVT2002vetGfd5E\\\\\\\/T6RYM\\\\\\\/nEJL5SF5Y4KHyCkZc0WQN97legmso8AweOV1llizUK07c83\\\\\\\/rkU2AJiOLLm6m5J32\\\\\\\/LnuYb7Raoyk\\\\\\\/WcF9t5maOxxWNjgB3z8So+DgDuxl6ZE6jItVg3JsYrf+H0S041ZhNO2GaBb\\\\\\\/Y3EM8hHUq82bqCTdFpnaJ7ESf54nN5d85fD86T6s4u+yxFxIXzngg6BET8PERYiex6yeCQflzNuEwn2Ywr1gwYi1LmmMxwoIIKnSs5MLfxtsGScCE6U2kH5plwBaAP2duOfYajxdth0ueiXuXOn\\\\\\\/6JzW3+ihVoa2yFFQf2eQdmY0FP6s9ZD8PxmV64+jWcG0WODmHpn2OOX\\\\\\\/vAhyzg1yC92DScz3wZUktEyowiLMnzkH7Vas4rEnn9ho5W5L\\\\\\\/aeTFDES9zQD1e0j9kVcdm8eiuSjBsuEb1+QfXkXrZfewGgXWk\\\\\\\/ep8em0vjai0\\\\\\\/uT6R9i0gJ1mY+LYtRRI3wNwctXcg4Qt0\\\\\\\/ZttiwKbQH\\\\\\\/mZwhEIwKgjUsA2DAnaWIY8iLsDPz4LGbO+dalXw6G6JEid0dPtweFxFuJdzQmGs7YaN6TV8av7VgKTAJbBjbzWHLHeksUUEalo9vLYewbW4QwdPYNP3McOvBFDG4lxyguciQx5yDT8C1hmcFDu40hzXSkQEoHGK4O7\\\\\\\/Y3JNBSPy\\\\\\\/G1Ru\\\\\\\/BKHGDdvF1rAYrqB0on+Ee8qICYSFpaQmStTA6dneOrPw\\\\\\\/vq6BARlOPjLDdGKw6KJK5ouiesL4oL2m0AaOootVea+U5ISEdl8t0K8iENuwsLt4yUP3jh9z8bDedzvh2oQAsSFWsphBusImnKJUQKv\\\\\\\/cjtWiZuSvuAPu8F79Su2+AKgdgVfA+yDo5AiTSoe8SzCzJ02P8MQGH0HiLjArsp5UQaHOunGJpaxGd4b7C7sv8\\\\\\\/EFpmiMN562PPWnABeWXlbGdRWV6DEx1TaWiiXyUCRr+51wxqKvGhzNfsyGAPzyvo2hJWuyBVgD4I8iyUdwodwG5J\\\\\\\/VZcL6tiGLgmwc5TugiHcSUcPykcPnWyziM9U0DpqThUwEbJGTDPbtIHqj1dW\\\\\\\/KbASux3fl5DmmukTiVImqw6M+tIk9Z1kZ2RzYHAD2xRc+i1C9Sj3PxW34tCW9rK5ybMC3Ew6t1phMEVdpMUwAZyNsJQT\\\\\\\/XgWJkNf3B4CU869TVO4Ev7Cz+Xrzdhu7iW4okGHVSN\\\\\\\/i7jOaX13+FXlPnN\\\\\\\/Oe3Yx7NIJv1XxUhpThjIkohwaPxZhPPeBzuK0q+NTsDzzLT7DuCiv9hey87D0PU9w9dHoNA+znuBSyWZk4vPU2YTJkQUbjG8O2uYrLwn+pOJNTczA4ZPUkLWige6+cGDZxmkr8xOgEp9BL0+m4GOXE3WCLgDxpSVwrcBoTRIWlb5fJaStutQ7nH2eORheEIPdxxfhkeaPeKuvOAFmuxDnIRLhs+3vbzvXhKxNiXwz4wLK1IAoDhqYFFFicRSm9YiS3KpJO7Cn3q7zzVKDHvbZP0rG60JFsSL6d7u7GRKzqza5DZw+MHgaxr2kaTNPUctSl0yWFkapNjQofwo7rGHg6z7NjMS4yYRenQ2dhacq82h\\\\\\\/7m\\\\\\\/MlFTU35VLIaIZKI\\\\\\\/IOtrSKg9hBuMh8XGkCsty619i8ORD3RD0mh7FGPbwZTnT72KB\\\\\\\/gDBZcG2sadVkxabKW3l+8qxbgpW4b\\\\\\\/AKW605FENlFBHCSHStSWZq9VXzbOVo3Y1G83F7J4RfSYc3Avg\\\\\\\/YjE\\\\\\\/R2mozaAf1wl6M4hmKHEUD\\\\\\\/RKkAga1BEUZQs8bmwpgI9ST7aY9gNivdeFEnkkOTowEKpLHwaveRxMe2dnyeAyd8EHl\\\\\\\/2dF+etBTvQue4wNeWRMULfClrZdZOWE\\\\\\\/yvT4DV1alNWmF0B5qTAvXQ8fCOwQybtABnITvbaQoK7WH2VRdHG35OKIYMXgM9wP6\\\\\\\/7hJcD4mY874accPL7B8zLLBDUbpTpqOHNMrUlZLLn3ALBNv0UhKdQe3KPBF86UAuXNf9\\\\\\\/wVY8U+d4GkjiT6ybRmfOiq2PiZwUY7zhlfku6+n0RGSaRP5Y9QngOCxF2MiCg\\\\\\\/8wqDFLU78AcnwzZFzS+eVGPZvTv5fV+JPr8MmgPOU7EqXu51otZjh2DLt1i5tV5Yad1my3\\\\\\\/Cc\\\\\\\/EdnqT0YEsCzl58PyBr261F8Y+PHuD0C4e92DrU1ghy6r7mpYOkp5q\\\\\\\/0eFVliqOYmQcTiPvx5Ww\\\\\\\/3OJF1tbtKPsk0C8kdRLcp+8ugg8N5fe4z5kEUTgD0xrWTZHi7FsTfkj7epbb9LXEBGEaVELGnC8nqx\\\\\\\/NjIEFrw+utLb3HNoeOrBSgiHMKpG1yXTtYsQ+l9UbA3u+bEwTs4T7mD0I8w59\\\\\\\/GYssHylWRYLDWGVYmRDFVSAhR+31JWxeasx14TLq5Gqe21g7MhID\\\\\\\/2rkxSQ93dm3Cj5weBSKrfsOw6XfmdL5dnKoDIE1XZc\\\\\\\/bfSVikfv0x9e4afGjDMUWDeQ\\\\\\\/QRPRhdr7O5tq44uKBi\\\\\\\/5s8z1UxBRdZdlPDzDQ4s+48JrfHyFHZSdevr44FZyuBDOCykfoZEvWEJw6wsRDmUgSJMRvxuE5o0yIPOpm4PY8YxEnMRJg+L\\\\\\\/gnguLhSRs8OwzHIZTlqv7N0wheu6hgoU0QxxztvrJ0kQquZZl5ENLRERt+7aWkdFPNHnAu4JYKpWgQ83ZslJbttIYg5gCDlrGvKjLUjnDyT8QPqmCgja\\\\\\\/04kVSxyJ00YozDnNiqV0TjLJS4Yf9r2K65kUvrMwnr\\\\\\\/8xY3SDT5iJVHcL3XnirSzLOlop0ZHdTqJghkKQIOT\\\\\\\/PbIxvYFMmBWNZ\\\\\\\/Wmu5FecK8A\\\\\\\/S7ZFe+T+zQ8AjMjFoMOUkTaFAUAUKue1sH+nkjYTcohsKeSGoDM0qIptkZlfyjrD87ob+XNh0Yk4AT6u5gZGzleooexw+j4lBgCGWrapNLYLFcVHU3uo1EdiOTh4QKsGsKYkpDIMN5oQqt+ryENfLzMtqt46TMVJOgQaApMbMAxeSvOndSeHJygxUINPlcJAH7fFZ7x+WVS8voOZxlYP7VrxWcquzcDeIBB1FQCB3PSL8VDD0Gdo473eU1nNJZB3gENlgOMWgnLJrre6\\\\\\\/ZFsQgVO2vP7x43SDtLUO97GADU7O9xDDHhSHsHu0KFpjxodQvualoeBzuRzI1QBeXKMmj6fK4WLKRZ+K\\\\\\\/QS5OYj2S7FRGypoxNO6Hr+L6yGPZhNgdNOVV5x+ms8dP6CwFCRkUZOR4h+AcEQ2yVH7iO9qnLRehrB6OJ3g0cqW\\\\\\\/LWnnqrBHqcoOXIKdDadyzAs9zSowG6wBSF7ECsBr2sLXyNj7hTzxJkqkbplLSYijAUmV+Q\\\\\\\/0AVQ26VXN9umB4jNO8SuJfWBJ9EuyTfJnLbBY7Xpc1GMZGGl0MrqwSmLVwVHl6TcFMVvFUiPRpmo7501j+jDY8Ib1SrI0ejcoQWiLAm0VpLS\\\\\\\/0nNYSo2uZZZaUDAzZPaz8VsrVN8a\\\\\\\/27TdtX0JAU6oBBsPe13lr8+CwOpXQyXgvpNUcVj+xm8lFu9YO6SIBYwwYm1nFyfsOffpQSlWzJC3oo6KYXuOibvagobnq4+Px3Us7qzmMrrEyR5vQZqh5PSWBe6uyGGt87J9YgHJbYMbSo8a4Assl98YQyKkrHW7mxrxeRzNTUpyhrvCmKOSd9WlXmZyf5Fqji\\\\\\\/e71ISL5pa75qp4JyHfK0eBqyXVcYMltvWtYxpgAIkICQIprW9oKUL\\\\\\\/JOWmRFlyaIOLhtz4WNhpSJMzkh8wWnsnU2R4lnVZopzLJg+tSK4r4g6Ibki1seNVGcKc38ZtADImUybsSAu1qKXksUEBWCWxwEco00jCU54jiFPY9Pn+lec3jqkze0YKneiJBvf9Q1JVNgrOvNKvVVscbTjJRpW1nvzHKWTqAmHNP7VeyBWMXrS+9yrVEALCDETEDBwsiNb2T74rNMScRQbyYRRzF96L62iZnAu3yuB4rQH2WObTTzCUybRIwvqr3dpJNgfHzsVCgRrxaYMbLliBoY1wIrBRNb4cQbQMaJql95qYoaXKfvgqlhjvLfeS+LW53SAjZknyrEaWFLrcEjZHi1WPUMA\\\\\\\/Shdhylh0xLSqKhI\\\\\\\/+Z5aDJGXVV3Nx8SeQzu4It0G8XpcfKf6EjY6RsjpQaFjsmYKe2aYtPqOOm26fBATYj3bjQk5yXnOH0zlU1pB0NQ7CJ96WzwfhLjGnwdFS2MEYkLGxZaLumTywU+Ix33y5UA13PbznpU8Khu\\\\\\\/LH2ByzFYrrOic6vUZEsMkqB6bWmw1BT7g8AQIytAm2iEOPI14WgvW\\\\\\\/qU\\\\\\\/haZaMDaonHMbEosDbqhZDDFeL+0KkJCaS0IitFOOxqpae\\\\\\\/XmGrmhh9Xn48E3fWwj4OF+sMSXjaA9+UYxasr3ZNWv4Z00hf6nbALneYDsTXKpfY9cVzU0g+uQm4Ervv9qefeLmbz2HJOnMvmnFhjR8tU3sc7PguzwiWnVEIZme2XgtiG7PpLcamI4DeWnTdmBJn+0mts4h83NaMFh70UcKGuGVW1Em3gizeuAzyRrU9u112cv+LlzCI4Nlk03zlFDz60v10QImY7WvM8I6pOfHtGPg8pRaw6m3qcRGGDhtvsktXWp8V9kseN1oD2H9\\\\\\\/+Fizx8WaLODwCHO+VKVGMd9KK4Sv+ZCX9seNz29H8qae+N7k5LDTaXBFxiS7aDpSHFJFXXvx2UuJCZlGYiemeX1zlDufLiwQa+zeBgvpg4oi2E4g\\\\\\\/E0JIIs0qzZMHvMB97tyrOLJbsIdDBsLzxGgN8vbzabaAhSiRr\\\\\\\/XJr+FDucQfP+7ifK\\\\\\\/7WFzJbG3ffYBrswU9lz5E6qyvy0RroODjZ64wca5MfhmPt7NjjN1+yTku+oSB8VHMEk+zT7aL6KPDIQUN2jmDbR+3D\\\\\\\/SpJS9CZEJVSStnTqmp95JmrQ7kdg0SwFoJTKbvFBR82ImyYGQFm\\\\\\\/pj6eSNYG\\\\\\\/NqkPcRwOSm0GcX9FVGOZcGa3rPpbWCX8gYGHAO3hwrVOL6MiccV4KokvHd5cOl8qqnwSD8alC8OuH0zNcN9zkG7Zz05dz3Tr5Os+2jfPKNyzjVpWi6Y7MppTRgvbUmFnXxCgNaax8BRcrWMgTIE+ams8Iq9W0FkIMz9IHeCv00\\\\\\\/yyOog1Aezq8BwKnqbL3gMlOPj\\\\\\\/oAEr78Amy6vgFyzooJN+\\\\\\\/JL86LYwatbLfZMrhkvfDR0AdmxLxAdV17\\\\\\\/xm34PiM21UIhFb32+2R00oBWi5FUN5Ofrk3pElBvpreJBVC6ITobSodxzJKDwfp3oOKxnZmfXDfSAxVj9PJwkzilY6XoezuJ0GlQZ8Nt1fKZ8na5SLAy9cKBCq5L+ASfI9zHnux7LCOiraa\\\\\\\/2zYPlXP0+0126u7xdjo+uZcTYg+j49to46bBN9efja3QPfYP\\\\\\\/ju2RtwVzQX\\\\\\\/fJZytOlt9FnhNxMQZYdxhjacBfOUYoGgE8NoOYr1393MT79gE+F1SIHRqwXUi8ACvmeUpSJJuJZhyns01YbS+4s+6p7KgDCt830IViRse9G06ledHd8IplQUb4HUQo+uOrONHMNmJvbbOe9kR5sB8mD0wT+KUttAuD9+gMWN4rHAumNathzOIS3POhYrORTQVvraNosiMJraED10VjrEPaoEGhCzH8w+VM1\\\\\\\/7PSvxEPfFYTPCEoa\\\\\\\/TFx5MQq7XXJKoZbPjcoSI4ttjPKVE39AyB7S8faStQ4Npbla2GbWtoPBTcIOj1G4gz4Zy0GcWj5Vo9cunYROfGii52bYWAVcRU8h\\\\\\\/ru0Y9DsnO\\\\\\\/KstTas13SnqWSUdyK0IY4aVKPKYvQ3+9shMSY3Avm24+BZN0DzhCt9IKvoN7mpRg3Zu6LqoaWekEHbomJ9LyRKN9KJ2cwiupRTzc6eMoGWrGyNdAv9KPFWuGT7gIsHR9eIwU+SZAFLuUTAVnFUshzt9kTtMbSTDnhERN0d6PVCD7uHoQbGLcJlCcI3EXT67C8aSxkzl\\\\\\\/RDX5ejSdFWgJpqwfQZMgSOkUQYjuOtsUzEjHtVg8lk7L1Ii1+cHaZ+Wmgq+z64B6eFv1rF27xO+DG0CeHfAuI53iDHE+X\\\\\\\/LH2UcLqUlzXgStx895hChVw61UC3KwOhrrSxJrKHV\\\\\\\/rTysVJlX0gBZePYx5sIwimvhC3\\\\\\\/qIT\\\\\\\/P\\\\\\\/p8Zhup8bahfdLHWb32HwOGBbsEH7BVei5nu+MjDQ8dTUOwF4ByQsx+0CCMPF8jd5zvndQhn3Fy8CS+TNjmaYc2MhVVjEE519sBWG3raC+NZnG5zgR9p9fHlBjoXgsRZN\\\\\\\/mg4iRmdUKJiqy2p0EwUXj3WrNV7+06GvBuLMOLwHNCwfSILaguVdTeSgSYJAw4WxRSLX2Ko4Yizk3\\\\\\\/rwS4pnlGONUBXP64DyhicgsRAEgvpGLLnyYiz\\\\\\\/bxxLuUWuFWcRHWKDJZeYXMNyLdaJ4VbtE4iJwJsui+KAbAsx4eFaiyvH99ZdMP4pAfTmbIl2LadrdRlUsUjnOlJgPuDqjwtIBjgwMrkJ8GgS57UBHEUCuNCZktUEZM75\\\\\\\/J3Tv+nd7pG4sC1IWiIa5mKaIrABx519sIyTcUrKAkB66HNOkrNCb2\\\\\\\/J9V5lpqxey3ifGMO8RdrJ+f2tmcFIT1grVR++ykZIdi9gha0c3aoINbjD\\\\\\\/1cuyzhQ8B\\\\\\\/E6LURSo4T2UX6xgpo+TsoJv3pcbGLBO9uBndI6NBrSBj5wM6BIsUP6aLlLNbtwMjFXsv3O80LYJ0I+tIXZzmWb8NUbQqh\\\\\\\/SGeyCdqzUdaxyEtECCU8Ma3h4BfNPhNY7kpHRdv4bI4y6SDsh1XzPgF+Jq8MbrGW19ds8R74W4rEGIfIOzcShIoSV6EYpT63yG+W\\\\\\\/hVoh1JkgolBQduzSv9yFbrKQFedW5wDZ3EQSaIjD7Yzh+zBgOaT4\\\\\\\/eFItjSfJ4uDBGgycRMyzlMpSZVehV8GI8vRD+5pkVTz7STYQZAj5vQRkHSVcMFPFueDAi2O6tguU4QfSAfRBUU2C37a0\\\\\\\/LYJ6G1j2\\\\\\\/tbYrbUId93wfSwduaXYtexTVp3Memc1ZpFA\\\\\\\/j7YS6oVch4gf2NPZCO2O1u4\\\\\\\/LS15Jft2EYO\\\\\\\/LTXKp5YOOT06w6YERSoAWLA2tqIB6s8brdwjOFHCLY2TJV1XCl2pdyn7reNAdV7uGOYZ6EJAvFwFTYac5elqUToZOjfQ6It0XzuqMBUPBPVJN6zLKBtWAYGcMq2xhlwYEpKoJ+DCYynZ9OIMyFcV+3QStLaZxXyRR2+RjUZlogrMtMmjfrCuEeOoUxGFfs4wIJufjqh9ClFKDGJ5vLEKf4alZ0QXnozyzP9R+7vVGCYCg6Q6liif0RlMTWUAi\\\\\\\/wf3176aSLAyY8NIeN2PEbBjb\\\\\\\/nmsGE9108qqJp8FKUEQ3ja25GdcIMFGklDjLoOc1jJD5S7pnRWSmQnjpmwAgVAU235xFNvtYcvna5lSUP\\\\\\\/IwaH\\\\\\\/TI5RKDeS+yF2uCj0Db5\\\\\\\/b6CCBquej0IVXxv1t4RDs\\\\\\\/jLrk9ha8qtu+cLryZncIE1MyvAgoRSJH2qstQBC02Ab2B+F\\\\\\\/MEHD\\\\\\\/+a6M+mgghqBQv26Vhox9Ht222TUvt1llvGOmvBFDQ2odKvPYacwqp1G5gioOf93OrUK0j0QGM6aOpy71agL7m8SOiy2Nv8p2r2X6qf3SUvsOp2+5WvH3kLsu04zuA1t9rgHkuXhz2OTp49MYlBJrD2Pjk1xFOImtsDNbc\\\\\\\/yB232NqGgApdKPwRWHgdbcprpkCVBT\\\\\\\/OSD\\\\\\\/l\\\\\\\/9Lv1S5Q8IWqfvCNyo\\\\\\\/Itsf+LC+NP\\\\\\\/oX7hgifPUzbMo8wsthb3AnIsNA3RJBXW3kzEQbPeYNacV80qOJyZigE2c7C4fqfUVynTqk+m+giTZHcbKTDrMIf4jRwNsrbWWNM9riK+WxbZwsNsvR\\\\\\\/r\\\\\\\/TY5X4oFn2b2+rKQUpevf1LyhvdevNgmRgZ+Fo\\\\\\\/61fm6juKfYA2tIKjs1nCeDTWWNzzZcQe+PM44qKTEJr0F1qvBObUESLBEjrJEYs03UbHj2+RvR7fauw0pslVLSPEh51YwvmCr1qNXG5xAaN5kCRGx\\\\\\\/k1oDnngnN6rKH9DIEgfUvmwce4kAY1khGQWqItyuZKOlPovWhHX8h1XZMI\\\\\\\/vIgGNpt6qv7M8c2mbOh6s1BlTQHtl0iYednRezsV9Dk0vSo7YqNG7IXk3swuDmOv7xAMGYTz4HWli+pnyJsoupcNHP1UNCH5qBtheQocrhmIzH7gJlpQmqrReFWwaQEoCst3JvwPRVjT0uvaCbzvpCmtjSz0+ab20uh3+iKSuS0HGNnbNOHb4VJAUF60Tt8oWqqP3E9C7gDZuoU6cBdeVObXJNwPrlFRCCCCZ4bRdn\\\\\\\/JaYpyW21bKaUan6TdOdHYR219eiTbbqNxA3cnIMV9BI2TCbo\\\\\\\/nIgen5JAbtr7mfdsx9TTA2itjxNKkr\\\\\\\/qpkQ5lzvXmS9UXDyWrj9Zh56gHY8izjA+L7I0mbZfSLbLEityr\\\\\\\/bMyD4N\\\\\\\/Y3xhYl1UJEOmcK\\\\\\\/D9gbHGcriQOnQfQfETTmBqyt0B5\\\\\\\/SoxkzAom\\\\\\\/joFeiPiKBODXjHkiGd\\\\\\\/OiPW6iMNNBvgxxRyPHI7Ox9Vw+Bk6IGWIZiOAV28uPGtbFMrK\\\\\\\/F4D5iFMAR2qgDnpoXkI5ZZKIca+ovwZLUOdAJ8QGX3hQo7EH1ZuB+T+95vxWXAQR3sfHLWaRHn894hDAgpSyC3v7moG2msSIUvEkJfBxu3NtL\\\\\\\/iWH3v\\\\\\\/XW1T4bQJHplZoTyc8bQFSmlV4VEjWL63pNaiJWGT72FQmuA09WLb2XeYprp3wJSJB3SAbmMLaPhCryX+VOgV6Zk2RW\\\\\\\/GNCv5bCJ1Fa6IKqwjxFd\\\\\\\/5Mdv4YUo\\\\\\\/FrbCq3HBuSqVEaA9nrewMmP5gnAG0omn6o2ER7GzU0HG7h84PTt4iEQFvWxyt0O0aUmO6iami7jDiFE\\\\\\\/9AGpgziTSw4GJq2K7VTfFzRPF9wIuTEoSkqczgOAySSNv2mvUE4axsbraPCHfl0vJGL5yXfFn6oMbLQ8v3mPe\\\\\\\/elrY557hgIdupeTh6KPFnUdlMI7pu8T4guVo\\\\\\\/SuOoTWg9VqOsJonu1PnPzcowmq0tSDoGxPBb0da127sNwfMIaNRQS7MYWsa0q7gfOo6kjE5VXnUpRzXVG8qIbgfvp5xw37cmLrPRdFC8Z2\\\\\\\/3Bi3wEm+wAZzOLVMxFmTY0f+Wkw4XX\\\\\\\/lsGPDIr2DyD2y+IBf2V0jfQ0CC2UysihFQ1LsselHtrkPuHQ2tOyl1woAX9jcymhC9KP4YsftgKbw4XmXn4OVr1ytZOJZ\\\\\\\/MYHcFVOEQOPjJ5guGCgTxtyVTp0RbIKEJhMDbxuYhb2dkZmPaFo3WQn4j0\\\\\\\/7w3MsPpfkJJIFqDgkNUhJgDUXb+EVMPcNDIXXyT0YWkZB6iNwGiGbJa27RU62edWTYkJj6Ylp6qYvR\\\\\\\/wiM5wQF2l8aAyv8ZXdqudnTUt+PjKcfxb+CSdImfSmWDTWrSs6fRkdJ6VRB26vlzvqqljLeMBUnZOIZGHNr\\\\\\\/uNyKoUEZEFnnjBJqy5kxlYfmnOURNPmEBZNl48n6pksiaeMjoozsqME09emoKENabjE843lJbC4JZMVoytOxQiTislTLfXrsVCF+bFnggQ64rU0TyuyC9X4zWXfpzG5zjFmJwRyz5Piadm0AkyhgSyUWDlc0IDkjBCreXtLKdzz4UVxTaE8\\\\\\\/MfESZoBd1H\\\\\\\/sy1EJn3SjSzTEGuFcIJ625RXcqK4pFLWndhOfnWx\\\\\\\/\\\\\\\/ZolT+Deti11Mt73wipmN8Xq\\\\\\\/IxeXXZ0ya23Ibq2i5Jp+7LS0L9CW9hibqaLx7ZpbbQscRH3sKq4zCx3BvQC47eE0GXtkqFy6EUupgikjqQ5DVGrHi493CXdGd40iTWKqoQhEZllGXlrOOe3rw9UJ8cJ4Q5XCTZ2CN8p5sXAdoo+3IlDqryFYglRZzCuNLNZTHvfc7AXbPBWkQH8ZKyEt\\\\\\\/9TFZALymZQSONhTunll72MjRBGfZkafMla4kfnWjA4BHpUl0wLPwrnyyAYZHM3YbI8\\\\\\\/IMdpk6TRaWUEeUpNVbdQiBUcSd1+0hh+LviriWpC3WzRBESENOscZZNJZD3ppHK4548mzfwUMrvhiupBXRC1JDwvrSwh8F5t+0cW8JOH8U0SHtNfUV2kmoQRZMdghZ+2auoczb1kHioqhAEqmBYDyqwNA42UP5E4wf7gzJCt\\\\\\\/Ohf0ThKZ98Rh6RiovsJYgpxp2v1VnC4gfVuDnzq5OcyIoPDwqf0bOPe6y+B+vl59SfGOsQ9JksSGM+8O\\\\\\\/6BEyUfXdPZsOxNAJsulnCED7y8v9mB2vdNKvEgBy\\\\\\\/nGiVymW48FBxDc\\\\\\\/cdHxMVNtfLubt3VRRZjHLIxssPbWe0YRB4TkL83RkMZf\\\\\\\/ybfGEgWT0NI+XUNrtE4wT8oRlQOnjpyY3UNZUrzkyDvrvMkQS\\\\\\\/4sBuYGHUzLcFN338dpZ0A8xM7bg4+AGj4rQUjhtc0MtYVzNgVREZ7mwQEdYyB66taNlrggeeWkeLQ4Qc4AaHonxx3vJOKiCuS3\\\\\\\/F6m1n+buWj29kSIluS2CWisUnOapddVZEeyjAksrVUlTDaL7\\\\\\\/Yv9FlGPUXgTmQb6Zh4SVY8b\\\\\\\/ansllo6IrviWwhZ4htog+QTBN7+zapq8e19S9PQJo3LzfOIm9ysfuHbXGdhQjbtZpiNPsVxqo7DjP5dhwvf5oyhUTG7fpJNuq1tPcEgrsB8D8cgO4QbvO7LYW8lI6SAsBwjweMurUtMn+ybgojiImugdY\\\\\\\/\\\\\\\/HrN21u2X7cc61ouSlipXimenLVkmyBLmYkaxcd1NdjBjnkgFU2VbBD\\\\\\\/HVYg4uX9\\\\\\\/EgJXOfrPfOFaPumbrRynRWPnMfDDsM\\\\\\\/pBwRqHzuXudS28Ic7jUlwgfGTzQlcsS31mkuil\\\\\\\/oPUxzLwxS91vss0wp9aLfrFG9jZitY7f3CYxqSMM6o987GZGqXEuqNVfjzbSQNkN8pxCon2byir2isrXfI2kKji3\\\\\\\/F7yxnNxMTWQBp4aEi6i+uzXmFlomKK9RDCntdo0XAuFuZNdXvdS6n2d5m3v+XORTyULZzIPM9RQzq3G3\\\\\\\/S69TKCqQLfzM1+rEH1zc+FJienFUguppA5MWqS+FDJFJ5FoLYUtMuWEM8nEbf0UG9ZwfopWIEK\\\\\\\/qSTE8vn9GraZolZ5RMS\\\\\\\/3pjQ8\\\\\\\/51UpyByYHVbwtskuPnTxiPISExf\\\\\\\/qo58UdapUGq8VllbKMEmKi0u+cy\\\\\\\/WWb+434SbKDP+zLfsLesvZRAkXgg3n8+gtlL4jcEpX8KZh5qVqxmrJnkph0qNqDCsYggTih8sBcayM7BoQMN\\\\\\\/dQivSeivBjDbUVzSOq9kJsQf7tgN4p81\\\\\\\/4dS4SvN0I0TSuK14hrlcNamLKzr6MF7qpl4MTozoBsrvtCNPyf5ySKJnyElUxRYVdZMJd7agILekNKaUvCVQl5tDEuoqCegUtzagfW7LQLqJatE8VvzPgwUx4B7yqrOYbO4qsRgDBzjvX7YwtzertJm5VVmC8ad6KWRBKUOSi445JK49D\\\\\\\/Fk7XVA4uZ8hVG5tAFsWvUb7\\\\\\\/OmH3zoydA3nJd\\\\\\\/qbwDqDUQ0SXNFRqTHRaUeqqoO\\\\\\\/DrlMDlIBY7y4K\\\\\\\/qBqnHVX8fFOsYXc95+3x7gdP5UkEQ5CDzPz57vfvpqPB2Bcth9GAU+UA0umEr7q8W0FQBk4drg20\\\\\\\/sVPw8JUKM7XSEuRgaIQGsRHYp2HoQz3w3oBT1yf3X2RJbdXYLH16UPHljMli7+yJxVSNDe2D0MIoyAwMsw++IHBmsyYKvonypYWyWMmF7meZV2ZMxLPiDUOZ6dUWG8hN2oji0QEPSmanHXj9StxTsxrnW9P0pes8j4huxVw4vlzvJB5v7M588nWYEoRcK+P4Jy6DlTnVTEKmVqdjJ6OeezcgTu9QaqkzI5fjsIREGY8nTFLvgkhZumeQ8m5NAuyIdM2\\\\\\\/DC6HWkRcYwzkPWuXZHr22Gvn4z3jo1lnNl5Ocvn012gNITVAQlh0raQt21AxZyUmz+gUI6IFKx5jTJcev5LvBX\\\\\\\/pDssajnVHzsSNxsh3WA06ejl9q28wp5vOJkGIqqLBVXNg9VtmwwX7ccfL0CW3OMHuAoMncx\\\\\\\/rVMIf7Dg9zphvM9XD1SWy1+O\\\\\\\/uf2El2YMYJ2iXlXbnGF1kNS+nXTI80V2xsMsYxmPhzHazT\\\\\\\/t8OuD3URHrdcUKpo4AzMufq18uhavx\\\\\\\/A29GnQQUAz6dIUMeWwT0PC3unusvnC+SPNc888ymLpbrpX1XJOwXZs5b44s63MMOv964cXNSMVWgpKPersBj9RQhEfu3NT5Fhn+OrAEKGJB321Ne65HZO7YsINGV\\\\\\\/rMrIHp+7RDJ5XVGJs1p7AtPH2k\\\\\\\/iCjh2G76X65LOtJUzEN8IogPVI2G4rg+gaouLEfXkB9iXSpuanbxiqM609RvRM01+eOofF9OCpJqy7VR91Q5uXQIAPdjoI3rJYHK+wGbmXlkjF+Q4xozc9PQ2J9wjZrcyjhGyIZF+TJYKY7+7Ve46W+nRjxXJ8PowoxKFewYoUHp2TkQSgwh11Z\\\\\\\/kpr61hLeqpvYOje0KpTxZ\\\\\\\/aWgjDSuWiPAeBCgjCHHoq8aju8dQKSVS6Q2ETNrCZnHhvaWHFhCeqhha4sPJUzXUM2PYAyAxyE\\\\\\\/2FBUpvhcTftzmprq\\\\\\\/jyoBYt4\\\\\\\/j9ZefwLFKEFIEOd6ghJAdwtWPIoHnxKhITipFq4uEgr7D6QXcHDLr5pPdOm9iDKRIhCpqgFcUJcL\\\\\\\/4g7Sdm2FQdBiC2srcXwNWFMMYmNjm+\\\\\\\/SDAmefR2HC86Z8zb6nw4X6\\\\\\\/xCSSrp2TewICvao\\\\\\\/7NZn7PmzQXar07NGVqFyliD7kvKLX2bEaTZ6dJ1oegLK2HF6+ECS2zXEqjZaFuPVKr+kJ6dt5Vc\\\\\\\/fvP+82MklwZoSONYC3iVGtycl38eKNSUTMb2r+kpdvduqz9vfuWbXCsdmahNa2ip0+7Xy73DzVQcoG1X2fcMAh5WmuPHakImPEqEn6f4jZ7m6wzYGb8VO78llEhGt1TQd6NOZndSakuCNUUHiQHzOoakuyQTMk+8kl+sSmklNVNmOInNrTeIE\\\\\\\/CYx3VxJ\\\\\\\/Ztvc4bYB65gZO8OxCyXeN1aOhohrtmM+0\\\\\\\/TlSUexQBV\\\\\\\/N0Z5NEFCUNgiRlF7f5bA6LfOlAjqVC1K71NBeuHgJcIr6jPpgE8OBrGqDc4gcr2QiG55oRqf0KBhVBQRUOZMMyCawLEOjMALWRCH\\\\\\\/Q+6\\\\\\\/\\\\\\\/k4hdt5iiNpqBPtNl3oOHTkmOBBNXjt5KXWhrLoMiIaCKnCbTc0P3qw0mBR9GV6NLzU7INeTaOuFlAroCG\\\\\\\/xyA4DWCvdhs7ahFD5ar7E5PTfJILmVtSNbQtrBj33dXX4O10v3vhUzDeP5GjV+Y3TMltJqRtkVYbJ2EFz7i8fz6GNWvsUPo1\\\\\\\/FBlYANJ4SuUjSktZdBGTPQHYru6yVswHALEoZxnzHNU1H3zyTs8u2ZXzLNL2XOxdBwp8uJIiquHtpzDZQ7p2aLAHgd5JVDQdB2enVjPMQquiynqKO2tcKPdXzGUjlyHCp0DpwIuHA4kOUnYBuY9PDtt8SkMWbfMWGwbOqwRddWX4\\\\\\\/CyOYdLOcPEqmzHNJi5zXt7\\\\\\\/GkIzxzePHv\\\\\\\/eqkzkGMRhP04gR1AqjOTIwDULaz4snwPIdC\\\\\\\/1Yc8bMZSlyLpg\\\\\\\/Fb3VropWEWZkRmprftgWSXNSEj5ciVyanNRu\\\\\\\/GW5Q21glXfGJIPaX0ZaxTtLUveHonP5JVYj28JVXi8\\\\\\\/cfVZw39uTHZ5Es3RqmByspjcAfccjZWtlDPQMFrBTX7b5jQIBxFUsyXeK9\\\\\\\/prsso+ouBCuaNYpZtBoimlvmUYaSPcSoK\\\\\\\/DBTNl7ivOTsUGLmEVVpKR1FTkcJAs03Rphg7JeJ4Ki2KOxEqZXIDMFe2dZSb80wCygmZTnpdeuggYynZtL1dBOfObPG0MYvKCTrveQMIzaAHpOJ2sEBwD+hpU32WKg1zSoD6M1a6msOGD0QTYYhELc8nZbZUfmoyMfSVlvJ38WWmiLP+zc0glaeNWcAJyPOV59UO9jPSQ1FA64sFlnupS2\\\\\\\/CRFLuJHjZFgxLnMhd7EnQgwnpJYSMNOovcwB+VQ8Q15CrFtLLZ4w+q0oVSipnU\\\\\\\/hzUYshjIIqEgojdE\\\\\\\/tJit2hO2JeRJmN7gzYPfb6rVej3zGfI4u4De383OL82OtfGlbWoD1gjBOfJvBlsUEnrriYpDnfcj7rjizZOuG8asg2MidoZGbtRatFhN8yQdbRJQNcHOkTorxcd8AARLO8XaKIvtQgW8mZReyZ3ogouQMXJsigJeC31L6L8AysXXT\\\\\\\/QHaH7QadbqLoyY+P3iJ8waeuyc6\\\\\\\/vqAs\\\\\\\/cTIdpS2EzZLAJFqH1VVlfpsK5w\\\\\\\/UqzCoC50rhg9amj8KDImJ+Fnp0FVtzJbM2DPpaOEhDTbDx1XI6PZvSgIPqEOGPhX\\\\\\\/SfbuIicLIBY3ePr9AEWzeuRijcY1r1HiLmSwqLEPR\\\\\\\/M4U3cYCBOM36sXCkuVYT8lZL9afux1p51nGdq\\\\\\\/4rpH7VoVBKnM\\\\\\\/4SoHr3f+5q1QuDfPg78KhzSSFh2yd\\\\\\\/QTz95HSKV5dV\\\\\\\/T8e6BL\\\\\\\/gthmDw6fe7oxsGLaBeFMkEUO4pzaE+977Z0LrfUT\\\\\\\/SbkeIi1FsRsGu7zaFWMujCILGhWk8mFcVRj4htShwOkfn+wP4WLsy8hxURELbvTDp59AP2hgEHuOKkSgJhF1pO6ounvmYeLMnRWyl0M+sjp+6bQJ6t8N+pN1qIt4rYyZVFoRpMTFdEQWGu3izR9ttx13O0sMfw+CLAxD0OxnqH5W+juXQsaeN\\\\\\\/upYxYppyyQ0efblINuJIPdAotwKH1i6HeP5eUlf+Hs6C7RBA883+TvhvXqu9pEup\\\\\\\/LXgMe30sWQZFj2NJswxIWWf\\\\\\\/xSrsARewADCGwCT1Cq5lgk5keXwD4R1z8801LJpL1KaQX5aNqZAkHYXhKRJHknq8I8vkqyXc5aWsV7ppGr0SG91VLXfx41uSj5BXaBYIuZ5k5VRX2IHgk\\\\\\\/gSND8yHUoDYa91kz7E9CCeLsJhiUlEGwVg1y+eNEkLdSIumIGrcGLzV9xhLdaSQS\\\\\\\/zpqky0NChzg\\\\\\\/6ggeV+id3H6W29Pc+53hGS4L6q1\\\\\\\/l\\\\\\\/kHlAMyez20SrgaVMvVIJAvKv2UFD2dkC93g77syCABvr7Cg8m6w7PKZdTMyL\\\\\\\/pSrmX0HCOtcPRE4Z0iMBvY1YQ22h2f0DI0\\\\\\\/xM4hd2QH1XlkeDTyA0LW4HGRPLrX+GOmfO7xWwBu\\\\\\\/E6JADxQ65fQ7ZYoEgBs3qSEONKJxlz7nI56rDLFBotrJGLqoD0mbhWAZFA\\\\\\\/Aua9P3HX5MQieLr0ySY\\\\\\\/2HZQhmI\\\\\\\/2p\\\\\\\/wNU6VjCvguXcQ0EVIUJB1bdRmFa6wZ5h1y4+cydOocbaRy9gGeiA6ymMGQ6MLxNr2pfVQDA6o5mR5wjNyzfN2P8D3GWO0HSEkwEg3wDZPsSEKPhE2gFLu3S++AEjD6EjR2RHnmovO0mbWuzMd9DElct7qqFviKOru\\\\\\\/P31dylU\\\\\\\/S0VhJS7\\\\\\\/UYefUxyQ4afObEh\\\\\\\/x8K7Rclv2CZgtWsCXsmvk7xgAodYRT1AwMG+\\\\\\\/8HtRomqNtq\\\\\\\/ZRJSJb8S+JhDZ6HPS7rgbZaBrxMrZ4OPh+9JeiEx5D7y2uwkI6u6bvyskSiJePQmOndLYcBD33QJCUe\\\\\\\/bqMlplpC530V2KYIvaKwf17lNTAZQhHFIaEyUgXQGxDvSSTO++EN+6Z7zmMnkOt+2d\\\\\\\/yShSeIomHlWMQh2PP1ZedDA3NV\\\\\\\/1aJEQx9FvOn73KK4TGTYgyn0DwqusUJmwc1iw+MwZFlQodhXfnwlUhRRKibQ94HNVNnF3U1WRGvNi16KfLpJG0iNKJF3+e\\\\\\\/f\\\\\\\/jzNam+NThc\\\\\\\/2Ga6uAQKOb9jrszeTHP3QZKO4vheERBKtPk9WMHl7OkFknYum2xpMEJpbogxU1Wz774luzMIOPPydstGdQFpkKU4Roof1fQXsql40SnfsDpLGKkVEGy455k0MNMB5Phe9df0LluR7dHwIsm\\\\\\\/rhQIWqKQf+98dU5wChhDV62y8IPFlMTRHq8qXYTEb8tSqN\\\\\\\/WHEhUVg\\\\\\\/9Qj0ugsVjfGldT9atw9q5zMCWrIjzCa7XcksBPeIQwnR11bYH4iOh1WMehdHdaDCL1OoUV8sSm3CLK7cqhZ\\\\\\\/KzTZ28VDa6bG60ZDMLuPz7OFaD2mTByajOGxGuzgPy5C72pDz+JuSiiRYEcTmSVYLSKZCbpD76MX5hDnkoLQVyffnnZYcbdzg\\\\\\\/HEb4Gbe4+xYJDZhyZ+dREXDhI208E9ufYJMzBkw4hkGn+5xOkPf6zBzhpXUPayEkZNZKTvOpgtHpQQi0\\\\\\\/gmpXZaJlMsywDLJRoc7VPAzpm+z23hOXNH4O14Ib\\\\\\\/J4ht\\\\\\\/PL5\\\\\\\/+3+3ciKG5gDRBlf+0fYQ8JnObYs\\\\\\\/2yTaEsunTpdDd9WD+t4oRbTBa0hWLMebRg2LC63MT3fQnsK5MtXFJqIfGhhG6MRQ1VYM3kQstuTzy57DIWlbh9oWNJkhG87h+ghe9rCSdeRHYq\\\\\\\/0Rkl9vFUC3X319QcOawDYQZObe2FxAz2WuJ3mF5pHPoFhoIMUXvwcfBkXEhWYuCNWyOKIT2EMZe6KooByS8jicEEFJHQxPI338dqA1dh7lLlgpLZujhasWIYUw7xGEfj4vQacErEWs5PT+Yk9yzgOUtVORGlQZLFVakYkdeyqUr\\\\\\\/7P69lErOxmX7+2Ne4vwWdQjCXS8P4fBz3+UlzaGIUDjgSQzVn\\\\\\\/CHIMPIPK+q1H5nQSUpbktM31trEVqB1PrhqDUjaUIHzkYlfw88z3GN0GNi1PozVc+DmkOkMnVmh1Ek2A5NPo6VwHkYoX2PKpXn0AvJIFqfP59ZCOvfF63Z6VqmrFnkAr8vwuDK\\\\\\\/Ux1pWcbqrIz19uaEAKBYP+9hozSgBjUUolF9Avj6lJoCZPNJJC\\\\\\\/D8wd7KyJ3S97ZFNNhpIoYObZxWllorIL5tgk98z\\\\\\\/a091rfFZKYzCZ31HRPhN9LST+TrA8jgrhrl\\\\\\\/xC1gU1ffSu0XXJlqCns+ekZOfiKH4ZY1QyyFNNFp+NUx1R94RqcVA9957gipbhC8wFwyc7Dpc\\\\\\\/AUN0yrtSZOTaYdcsq8nrtiag8g9UQ9i0EDL2mk9rFpeU5GrHwsLCQ+mq5Kic\\\\\\\/rP4T6NP\\\\\\\/PBT333IcuRH0B2VIuzf9zrfzxyz6mt2kT+zRCUzfOedsUdtrSqh1BkQThaJ2XIwfLzxNZwpTzuQbgfoLH0\\\\\\\/oGn6PYidlF3U3QK8njNxvLP6d8QQOs2nzDKn9OgRgDVGCwTyINMv7jYRIA\\\\\\\/pkzpEevfrK6459s6mXkxPWrNpA0vPHXE3scevkj0XNLvlP4BWnFTU1gPkh2SW8oSj3nbUV0J1DUE3kUH2L43YU4fdIpicebFTg7+rwkZ4+FEZ5s58zTwaQRQzW7vpzD9qNB16XO5NZ\\\\\\\/RPHoEfzxVCl9zVH\\\\\\\/abIK3LPJyzfMV7kj2Mla+JHUQ+Skzm3PuWhXL46zotZsoK9KCKCKFA53JKb4QV\\\\\\\/eQMVA2SwOUoJXxkLVNAL2VH\\\\\\\/Ip3TMTx8166XOQABjJoui6PG7A9c33BLSBX2\\\\\\\/mbS+cDuyRUg2qX2Ya0o9aayE8DSOeb0NuZKl+92kTO5MpFNzPw5fylekfK8QeAmecN1NI6zVVd0PAXIyW265CkhE4dOGTJBsFnN8EqIfOux9rljJY6bCfakG79stf9ygp+l\\\\\\\/rHGYN24TATSKSgvx1yRrBlfA5xnSYE9+FyODoDzUh5crJLRyBKAJxT43uS1eQC\\\\\\\/Rmd+eDqIqYFM7\\\\\\\/UsKkXRQwtQ8Pswr6H\\\\\\\/jn8tkGgJVdRWMsmkyQYmqiutaNYa1GuH6DER3vWKkducYyDa18n5tAmgeXrAFcv3Fg0slJHqvHXvlFWZs\\\\\\\/cxJhJuaPLgQHnem0c9inBetVaU7LmP4v+KrJWtnBq40dFqH83ZtKmXJnJ8DjxJXlBsyvO2FZBUNkVQLdenl3Ul+5L8Fx\\\\\\\/CBndD1rvuWVZHThR90yqPXxJHXq1KRwlQVKd\\\\\\\/g5c1I8ws0xS+caXKy6p7g8q57Rk6+0+vAnzw9eFaT0eYTnMh\\\\\\\/C6v8nQfAOJqb8ejfLcrwExJtuRt0Lg2WgPYAX+2tdCK6X3ltIP9f7xJnHZ+ATlVKUeHT+edSTZskt5tNirsDSS3FS1TXUKhE8k8G\\\\\\\/Kk+MJkPCpDFHQaAU0NGvfKZZL+Itjv8qEp1PPyUqO4EihfFTgrKVXQL9hXLkURCfPeY3tVVn27TMYuPDbcY4trDpaseTPHgAzD4tCpu+R2tb3JQlwrhxT0gRYCfbFNJvj0qMCyBa3rLahD0WXZhZE+wvlPji7oWVZkhzGqNwf0D57XvSOjhpifTkI71Knk4PUV7UBGqOcvKL1ZjZEecgc77QnFHR5EfGkoGpZzdaSNtsAOQbnjn2RgWRKAEJuM5yHNs93yFjg\\\\\\\/3EhNgBIZ3WjFHfwRiiGRgZSJIz\\\\\\\/t5f7ncSJpepSBGgh6vvH+SwZydBYEkT8va89xsiK7LowqtjS0vvm1a7BYl+ZevcwSpAwPDKyQv12SrAb2kssvnldTkRs67+qdDNpRGMfuJGhs2RHiQYkaK8lCgEDFfxtqxLuSZbAcYFlh7jOT1xUQrPSKvJqHqubiv4Z7V0VD9T9XCHWyuy+IMKo315fEdOUmE5OA1kmHLQhsC4s9ZonbpG\\\\\\\/uDQ2abJ\\\\\\\/jjnMMjvbTylOmsekmLzs2tEyPr21IGCSz35nkeDj9uZNW9Vl2oLXWQb54e0PEFZVNTdcsfcxD\\\\\\\/OFPZZrOZfeRo0vFGd4IjjJaL5hV5ccAbktwMEWtuXqpqjqZ7ehtPWwHfgfWjf6KUm9Sg+Lhb40W2gHn6IBFXLmsFBIUgLPs0FksVt1DCyqjt0s29S9JHHkgAOqZo1rNaIBYsKi8TCV7RU6mEQodGKIpn03M6l8O4wWj4mimZpDQ4U4HvnmQiOMvKXvd8gTHpPvn0sOPHa9tk8Qi5L3W+sDyD7Vcv8qQcKQTGPkGTYOsOLRDHEfr6f16Oe0WhAFSbW3QmIfIdAIZ3OWN6XVQu8ZMCRaUAxIAi\\\\\\\/VpJupMgymdilEE4LzkHzV43VBwc3XKPJ+2nv+4peHXU7lM2KKQHxomtLE6pFJSE6RSJDUTHLZsTaPWQb227tsvmtJ7j9RaIwsSQwIXC9LqEEIfcLUa0c5gdUIg3tJ49HKX9Kdltg2IkTXFq649C7vViuMERRXlAa9t09olivVeTMu8Fvi2SnkzYcFHN4XX4CEzFLtVTu7RetOcQfCdreUZLaOxDvh0qd0GfFbJnRKPdGzQUewzKYN8GwxPLFyNqi+IgE8n6gXcFW6MBGK8jvXGjvLURqoqqKhsDD1xx8EnL3PQWbtBB5RBYny0HYiJ\\\\\\\/tW7i4V399hyXQWdfK0IKH+\\\\\\\/DaoiTxQJb4OnNzCp7xiw5bkiNrQL7zA1URgULHEBeEV\\\\\\\/rDdvJx7nklhbxV0JW8no4NjYqoAYUKVKg4O2uw6VAe8bBuMynD\\\\\\\/cdsAOa1Ru2Nt5cJZR\\\\\\\/nshY\\\\\\\/ilihifsb4TraO30ttmegIg43pdnis7VRlZzWIfjxKryW57aTruB1g69ja7MB9hp7wg++wQMAALZuCIv8can+pcJx3VWVC9h29359JCLfhuNuGE6Vx6w6eJbNUnnjRTZBwWLN26+nK8WJdz5TGXW7bDGgqaSvoKmwqGnGCM5JVksNVvtzxQAj\\\\\\\/EnIIxqwAbS+M23hZTEWSESkXkfAX1wwcuRP2i6yEYKNO6BR8cMrH9NSk1aKYT34+GPUJFmmDW1EWwvn7nCVKfmdSKQVWUwbm2th7l113TGiJ3q+gcw7KsEFIl1YyloPb\\\\\\\/1H6cNuPEYnXdP3hth6QOaslbgatM5\\\\\\\/2q9T2\\\\\\\/svN4cySkBO+lk8\\\\\\\/yK+WkQk63h1oXTUcup7T37lsG3\\\\\\\/\\\\\\\/Jw9k4N3yhIFNlhHAvMjXDU9Adn0zXdMeGEn4bV5SQaYFpAJ0zbhuoAtDFC\\\\\\\/ppbk+rVXt3+N\\\\\\\/sr6JdTZStx7ayEB6Z9kaD9aWqoi0WTOCkOviQqhEMd8a58IIDEw+8dUSpX0sJfopUBiZohNhcL3bKYb+6vPCXqI2eeZP7MohBcNiy03VotQ9x2xyH9yJZflkiYRVpJWE5lcmjkPG3gRDNY0aOwaIh6qA7xQjRJVxKhyMqQTGYmRPeJS0zSWGeFADYLnLyWv87tEk6OSKRPiYLuyV5zOd2VM4EmIWK7BdMkJ5E+KIMzLp4FX4Xh77jgKZNXDXbidmmUvL4CNKAxmbFphDA6DAGB\\\\\\\/J6POkaZEPuyyMS\\\\\\\/HELdfGzztpc4L6kefw74oBO0sdtcLwO+shDzrEzJDjxgCiDxeGggGyQ3EMLv4eFPAy5F1YHHdYRCZ3gFgRXYur8XsSGlXsnp6rdNipWyFeP62FujBYwjMcUltpl4vlHNv5jXT1KkwUmXSLoWVQDnXm06ih+GK28QRKa2CmgUAhEHbJvYpyEBDOb6JwIKcu3eBFUqakkLatSiV41l97SmsdffqZqUFziQCd4mxj8cFbpyP6jiGDvMFh7TIPd76HsdoluFqOMcJ\\\\\\\/dG2Xkdt3gmhXkvanGJduyx4VuWofpzUNlLkuaGf8fjX2imPfa+x446iIMardvEvIP+GG0S9HZbfy8GBA9SI3OS4zi9vl46neuBrCQUsOvMEU1YJfQV7Dkmxno2hnX\\\\\\\/NEuN6knTr2IY7A6R4r3DMiNEl\\\\\\\/GQAoWjH1XCtBx+ED6FNYI9Tu1D3FQKg8bXEwa\\\\\\\/hk4j5vX5y8VIJ1yCS\\\\\\\/pcinnGNm\\\\\\\/9mkdHNGZVFWTYOFc+qKC946eeeYVYg72CI3IZSmXyyyU0mVsJGANTUJf37djNN3LeKTuz2ekBD3SZbo0mywIDvtcPfhOb+s9Akzan7VPD6nHa7XH4viyzCoOmi6YNlsNa57g2d6UZ2WrTrEmRCOJphmVAXDgAJD0ni9f+wNWx8A4GWTlFlVOH8dzm7fe8tUJXEXmCP7pT2OqgGOOUyspt2eNG3+oP5hr8S8P2ph3eXbLimGLmjbs3EWd2kg2LsovcG9H2cxcDqtb2jglSsPojtS8n7AxDeYG23CUCkLxrbrNWwa6w1jbeV2e35FVYUWPXB8hiyT3OKvsSaGcVhrnoVYJ+rGwpQjdvtSAcBBbQbEDI5JXnO\\\\\\\/awrYZNVG7eO\\\\\\\/4CSECZUpMnSKcDe0lHc\\\\\\\/v8LUrthmAJbOvWVXzqDmxF5VRvtdhNAy3ctghU1IDLWZ8X3z9+LkQLGl+tHxpEdkkuA0iDeuT\\\\\\\/cF3YqUmh3Z9\\\\\\\/yTRtgApT+HkhCYCgTsUedgkZ31gi5BhGW+Yr\\\\\\\/MTdwakrktf3Ul70L+Fn4H0Tbuuka7i\\\\\\\/8fWPJIlzjCVdoRBZ4W004MJBEp\\\\\\\/VXYwZCL6\\\\\\\/RaMGmiWuC9MJwCf\\\\\\\/anmafrun9KbK\\\\\\\/DgBulPFkZPlbfRlVCcDEsjFgk+QCgqQTw52Fc91YJdzZz16tZqTijmrOko4PQL+\\\\\\\/ytUPx7YCsxk2B0E\\\\\\\/3mN+IBwpbTkqOiLDD2HJTWIOziV6JC4\\\\\\\/Mfk43EL1W7PjOoX9DT+aIRBcP2dH0iz9VJoeuO757CErWfCr6MxbI\\\\\\\/ifB6TXFmuxnS\\\\\\\/qeVyez2owi81vZ7HCXDSKyNSWZrjszZ3Cw65LbiQR\\\\\\\/EvWUav4LwNz5TDu1zqPN1VvIdgEFdSLXLUiWQYWGVQpS\\\\\\\/cSEdtnKbk+AyVVJUPMGOj5QDcJEorsFh\\\\\\\/YWyoR7P2ciwjc5Hxrs26bu50bPGD7RI4QZskVz5T30sJtaFk1gU0kFb3HFWDWbaT68PuCKYZkAcNAeME4X8jpOH8\\\\\\\/AfWfAZQt4J9MxfI5N6b8y62swEesy4a0UBOwU2ZLJNhC\\\\\\\/snmHlKYEjBhGms22jQYJty6ApmUx5yw5E+GN0EI2KrdfNfVxrgo+OrkKRPSq03i8eFNlVzrG+u8R4sXYpbEb1fE07AVxfnamBfiQFSz+lAgIVQuhZPOY\\\\\\\/GbXGkAiZrDe\\\\\\\/rUKCdvh9bvDGfZtvzyEESi4MFhEZqkUY0jyet45WAaHUe6J9Zd+WtXg9B4iCTgozOw3wQH53CsF1tsdVB77MSyiDKFPKVGDHnw6niKAEYS+idghB9qYyLzTHaWwEPthfJcqtXBZHG8wYch18\\\\\\\/7ru1pEUuEKA6LGKGcPmPgdTirxgLSaxOAhxsjukouvPJuh46A2b\\\\\\\/uY6khbIrJASN8CUob10zFCByVVGJCu5bjTRJr\\\\\\\/aQHhoXkh+da15zgjMoaTZ9PmSxssRXLJKkh4fqE3TE1kOKvi5VPrT5T2QbblV\\\\\\\/0+DX\\\\\\\/JmnOV4j2cAPmOjqYnBHSxYxyjLAGmRT79k1FYBFSsEHuvIbLaPaABwH1EBkVsaoVRv+yV0KdPNk8nSGJMvVmy\\\\\\\/G9jU3Bei9fV+b+ZIrNt3v821SM5svWK3AU3c84mN0TGDEv8v9VaeIQ2bMVdGVALSch+6creMCRQBo795yT\\\\\\\/4rBaBs7KAnSrhwFnLj8woiQmC5T6r1MDVjcM0vF8DN\\\\\\\/R7TIeIvO3zt51Wf7jY3CzUbocYyp0PGKbTRYDg0sToHOpm3FXHOilrawKavYnUWTqwPgczssJwZler33+O96cGai\\\\\\\/k7mRBTgkt2V1S0TCFwq1NFy\\\\\\\/Xh3mcs8q03sKATIE0HES7JP\\\\\\\/bTyHzLUE6CwZR8yH94kyDWQe3ZF37t2QJPw2gWLNnxFAlc1NhfciBMV\\\\\\\/Kz0A0GbOp1zhP2VV4AVdUQC03kSldv5r9r+BIYXA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lyTpBgtbfnrlZkRoRp4GAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229e9e63f50b8be23ee296d9ddfaea7244027a5946d3808e2309e7b8729131f9ea\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gz0evLfPheAPGKPmgXFKenv0Dhe9fIZEGnufTYtqkF3qlTzNef6yhbxZKYChQf3KA4ZWtIL\\\\\\\/lWyIdIeaq+e8oGMHIVLbX9SWYariD2pDg4CQsjXyXOXu3sCcrJc7019N6KDyzA3ohH6rFWXPaEKP9FLVcLIt6CdLixTq2O7DQPH4FenDbcOV0qnqAQF41FO2+Nnlqn1tN+vfHH4k8D5QlG3B6Xo+wQnmSw9+5kCG2QaeD5wmhBaYxtY9Tfi0iFR+pTI+TJ5BbZYYM8efwFjtdGMFcTkC+DjfUEpodvToajQinVL\\\\\\\/PYNEj8j2US51W5xuReifGgwPnXxuVGz+hQIemh00Nef6i\\\\\\\/JWccGA\\\\\\\/M4IAtFM+1kGoL2gBYVCi3kk9hZpXJOWfXHML6EYGRxRAEbXktf\\\\\\\/+l+j4oBqkHxkmoFxahabgb8e7tEUyM\\\\\\\/Gq+DvNxJsrcefj8p58Gw1oKpZ3qtCNmUAMBXYznvYMt6L9jbpkG1IagxFP8+\\\\\\\/kCdOugSnsvirHCDB75MlZXAEICb8+im02cQXqOLp2\\\\\\\/NXj3NY5ywVRLpFypdgsM\\\\\\\/Qz8v6NPM4Mtt675fGYmBT4C1LK\\\\\\\/1qkLSs7NjrGpvG5iK5nHSg6zWVyK8nuvgV1RCHX13ekHK5pycQTQXGJxCJsu4GXOmxGaP0hwH3Y3UFcbcJor4pOiOzzYRdAsgS0cW1gtURdnROTzAJjmJYgFapLb5SmJtUaJPiDJuujaFuI7nMXjUZdEXSiFP9\\\\\\\/kwr+J0fkDVvmGeN5\\\\\\\/s+cjzJFIRPIScrk9xCZb5CtYfKiM0KQEodIK\\\\\\\/Hj7Qb3s5dYfIecvtl\\\\\\\/bfu9dkiDL6mVe01IpQeQcu\\\\\\\/Q1EbaJ+Bj8o5+K6Kch5+ibmuurtxJW66x\\\\\\\/99BMYBEdFH+gbJC+j2bMRPZPGLae3jSwPktzt3dLFU8IlRflm\\\\\\\/9LX+iJsRYHl4+W5uRp1taXNz+1N67DkNWKZvOg9bprd5h\\\\\\\/C7poIacjBgzsEo68GvhTjGwaelaIQZgTAFt7bKJr0pnCz8hDUysQBpwSB3cPLWWqCv20noH2d7eqArn8gq2vpciI2B\\\\\\\/1U3lAi+ix3MAw2oULVgQa+mXjDTIGyWga6JmvNSRikm5pFZeiETPwrBs0Iv++EmxHzWGMKZAOqZSR10ilvbnp4VY09p96dcfutfEd6r5OmXGmpnteC0O4YZ6AGhC7kJZTXsHYCOQJFSUrBMZSY9ZVSSWbDIQyIm6gNoArUXKDDhj+jawbp+wswUd7BppO4TZBQF8cn5RPpRrr842Md\\\\\\\/k6dN+lGGkx5YB6\\\\\\\/eMhcKDoTZcJDS8Kw6T20s2oJRCcI6H6FHdpUzDrAw9n8gcyqoVsAAFIpVY\\\\\\\/FLR+m+ucr9wKkrzIe+8ol5pzF\\\\\\\/IcfcFDgdyWCjZ7lTti4D2lsVDmuZ2uuhJ0WrKA1lt\\\\\\\/U+hyxVYxRNJXJ8TsgY4OtsVWgExLlVPUQlWKz57nzqBNJGf7KQrVpclUU7xvIml+UUPmHbfD\\\\\\\/l5D8XT0ImFP\\\\\\\/DE3SwGJ7R\\\\\\\/VBeUEKiVtAvdMUxziCTvkzNFu+7vFtv+rPOHem3b0N4uhxC51Ravr9W+0NjoV\\\\\\\/BuOLMNVZE74gUw9CZDLzZJNAAaNNkLgGE7kAa7gqoRAXWPDh5esKWwWG9UuCPF1\\\\\\\/inVzTL30O7vA\\\\\\\/RJmNkDU6tlUw8WYSZni3E5\\\\\\\/STzplu6vXJLXeD5xDU9byEhOOlXlFpZgxibMPsc+zJvyqSkMbjMfR+6WoXvZ0YtUeoh28tdhWRQmtO33Am4JY\\\\\\\/mE9kuboTq04X+x3QKEbQJ\\\\\\\/UYdY4ClOmFv6sAGkPHgPrswdkfGlWhsOXmm4bsd9NAdAYsXuAjXCSimeo61ntbM7lh8fBuGetDJOBEwcVepVfhHgUQmML6Kx0nhvg1OSGd\\\\\\\/XPPOutrc\\\\\\\/BrOr3MaaEC6J\\\\\\\/v+EYWZQ0q1GiOCbwz4nrPUgFm6gGrHQhaB1l7TEZqTchRzXwSHds+mehZrPT7PeRMiovhTJQx8PEAhn9bnooQ8oasc9v1A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GZ4Q5jTwIklR6XtvtqztsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022de4d04124bfbecd70747aa190a126a713653af3e7d7d3342a272cccfad9e4495\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459491 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1165" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uGjV-dIiJO47YYjjYdoaaM4vZZPUsdabMehXQb594YY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1165" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-quTFPaXNkVbkAGcsWHPo_AXQ-FuONF7ae_z3WZP9r4g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1151" title="Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1177" title="Spatial Profiling of Nuclear Receptor Transcription Patterns over the Course of Drosophila Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459491" id="node459491" data-pisa="ggg;3/7/1165" data-pisa-master="ggg;g3.113.006478" data-apath="/ggg/3/7/1165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Generalized Admixture Mapping for Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bin Zhu</span>, <span class="highwire-citation-author" data-delta="1">Allison E. Ashley-Koch</span> and <span class="highwire-citation-author" data-delta="2">David B. Dunson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1165-1175;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006478 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459491" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, Maryland 20850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bin.zhu@nih.gov" class="" data-icon-position="" data-hide-link-title="0">bin.zhu@nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Allison E. Ashley-Koch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Human Genetics, Duke University Medical Center, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allison%2BE.%2BAshley-Koch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashley-Koch%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllison%2BE.%2BAshley-Koch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David B. Dunson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistical Science, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BDunson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunson%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BDunson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/7/1165" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459491" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459491/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1165.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459491" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459491/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1165.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459491" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459491/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1165.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459491" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459491/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/7/1165.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459491" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459491/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Admixture mapping is a popular tool to identify regions of the genome associated with traits in a recently admixed population. Existing methods have been developed primarily for identification of a single locus influencing a dichotomous trait within a case-control study design. We propose a generalized admixture mapping (GLEAM) approach, a flexible and powerful regression method for both quantitative and qualitative traits, which is able to test for association between the trait and local ancestries in multiple loci simultaneously and adjust for covariates. The new method is based on the generalized linear model and uses a quadratic normal moment prior to incorporate admixture prior information. Through simulation, we demonstrate that GLEAM achieves lower type I error rate and higher power than ANCESTRYMAP both for qualitative traits and more significantly for quantitative traits. We applied GLEAM to genome-wide SNP data from the Illumina African American panel derived from a cohort of black women participating in the Healthy Pregnancy, Healthy Baby study and identified a locus on chromosome 2 associated with the averaged maternal mean arterial pressure during 24 to 28 weeks of pregnancy.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/generalized-linear-model" class="hw-term hw-article-keyword hw-article-keyword-generalized-linear-model" rel="nofollow">generalized linear model</a></li><li class="kwd"><a href="/keyword/local-ancestry" class="hw-term hw-article-keyword hw-article-keyword-local-ancestry" rel="nofollow">local ancestry</a></li><li class="kwd"><a href="/keyword/mapping-admixture-linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-mapping-by-admixture-linkage-disequilibrium" rel="nofollow">mapping by admixture linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/quadratic-normal-moment-prior" class="hw-term hw-article-keyword hw-article-keyword-quadratic-normal-moment-prior" rel="nofollow">quadratic normal moment prior</a></li><li class="kwd"><a href="/keyword/quantitative-traits" class="hw-term hw-article-keyword hw-article-keyword-quantitative-traits" rel="nofollow">quantitative traits</a></li></ul><p id="p-4">Admixture mapping, also known as mapping by admixture linkage disequilibrium, has become an important tool for localizing disease genes. A number of admixture mapping studies have successfully identified candidate loci associated with common complex traits and biomarkers (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Reich <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zhu <em>et al.</em> 2005</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Freedman <em>et al.</em> 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kao <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Yang <em>et al.</em> 2011</a>).</p><p id="p-5">As a genome-wide association approach, admixture mapping aims to identify susceptibility loci, which confer risk or are linked with other loci harboring risk variants, for complex-traits that have different prevalences between ancestral populations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">McKeigue 2005</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Winkler <em>et al.</em> 2010</a>). In recently admixed populations, such as African Americans or Hispanic Americans, the chromosome resembles a mosaic of ancestry blocks, with alleles inherited together from one ancestral population within each block. The ancestral populations have different risks for the trait, which is assumed to be due in part to frequency differences in risk variants. The block containing the risk variant is more likely to have originated from the high-risk ancestral population than the low-risk ancestral population. Hence, detecting the association between ancestry block and trait helps us to localize the susceptibility loci.</p><p id="p-6">The ancestral status of a block at a specific genomic region, or local ancestry, is unobserved and can be estimated based on ancestry informative markers (AIMs), such as single-nucleotide polymorphisms (SNPs), which vary in frequency across ancestral populations. AIMs tag the status of an ancestry block, similar to that of tagSNPs, which are used to characterize common haplotypes in a chromosomal region. In the African-American population, the linkage disequilibrium due to admixture extends for a much wider region than the linkage disequilibrium between haplotypes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Smith <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Patterson <em>et al.</em> 2004</a>). Hence, compared with the tagSNP-based genome-wide association study, admixture mapping requires many fewer markers to tag the whole genome and therefore increases the detection power at a reduced resolution, which is still greater than linkage analysis (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Patterson <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Smith and O’Brien 2005</a>). Moreover, admixture mapping is less vulnerable to allelic heterogeneity because it relies on local ancestry instead of alleles directly.</p><p id="p-7">Given the local ancestries of each individual, several hypothesis testing-based approaches have been proposed to test, one locus at a time, the null hypothesis that the AIM is unlinked to the complex-trait/disease for a dichotomous trait within a case-control study design. <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">McKeigue (1998)</a> proposed a test for gametic disequilibrium between an AIM locus and the trait locus, conditional on the parental admixture. <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Patterson <em>et al.</em> (2004)</a> suggested a Bayesian likelihood ratio test, comparing the likelihood under the alternative hypothesis (a given AIM locus is associated with the trait) <em>vs.</em> the one under the null hypothesis, for cases and controls respectively. <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Zhu <em>et al.</em> (2004)</a> described a Z-score statistic, similar to the one proposed by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Montana and Pritchard (2004)</a>, for testing the estimated local ancestry proportion is equal to one under the null hypothesis for case-control and case-only studies.</p><p id="p-8">In contrast, few methods are proposed for the quantitative traits and to consider multiple loci simultaneously while adjusting for other risk factors. To apply the aforementioned admixture methods primarily developed for a dichotomous trait, the common practice has been to dichotomize subjects with the least and greatest <em>q</em>% (<em>e.g.</em>, 20%) of the quantitative trait value as cases and controls; The remaining subjects with in-between quantitative trait values are then discarded (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Reich <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Scherer <em>et al.</em> 2010</a>). In addition, ADMIXMAP (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hoggart <em>et al.</em> 2003</a>) has been proposed for quantitative traits based on generalized linear model, which is also used by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Basu <em>et al.</em> (2009)</a> and <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zhu <em>et al.</em> (2011)</a> for one locus at a time. However, complex traits are commonly caused by joint effects of the multiple genes and other risk factors, such as age, sex, and smoking status. Investigating the association between AIM loci and a trait, one locus a time, without considering other loci or risk factors may capture a rather small proportion of joint effects and will possibly lead to inconsistent conclusions. Similar considerations have been addressed in association mapping using shrinkage priors (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wu <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Guan and Stephens 2011</a>).</p><p id="p-9">With these motivations, we propose regression-based generalized admixture mapping (GLEAM) for both quantitative and qualitative traits. The new approach is able to examine the association between the complex trait and single or multiple loci simultaneously while also adjusting for other risk factors. GLEAM is based on generalized linear models (GLMs) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">McCullagh and Nelder 1989</a>), with linear regression for continuous traits, logistic regression for binary (<em>e.g.</em>, case-control) traits and Poisson regression for count traits. The predictors in GLM include local ancestries at the given AIM loci and other risk factors. The local ancestry is defined as the number of alleles from the high-risk ancestral population, for example, 0, 1, or 2 alleles from African ancestry at a given AIM locus. The association examined in GLEAM can be adjusted by other risk factors. We assume for complex genetic traits that most loci have no association with the trait, a few loci may have small to modest association (<em>e.g.</em>, odds ratio &lt;2 for binary traits), and the loci with greater proportions of disease-causing alleles from the high-risk population would possibly have stronger association with the traits. This prior knowledge is incorporated into GLEAM by using a quadratic normal moment (QNM) prior (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Johnson and Rossell 2010</a>) for the coefficients in GLM (see <em>Material and Methods</em>) with the benefit of reducing the type I error while increasing the power, as demonstrated by the simulations in <em>Results</em>.</p><p id="p-10">The number of AIMs (1500~3000) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Smith <em>et al.</em> 2004</a>) is usually larger than the number of study subjects, and keeps increasing (&gt;4000) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tandon <em>et al.</em> 2011</a>) with advances due to the HapMap project (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">The International Hapmap Consortium 2005</a>) and commercially available genome-wide SNP arrays. It is not feasible to consider loci all together simultaneously due to the “curse of dimensionality” (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bellman 1961</a>). Rather, we propose a two-stage approach: in the first stage, we examine the association between local ancestries with the trait for one locus at a time and select a small subset of susceptibility loci; in the second stage, the associations between the various combinations of these selected loci and the trait are evaluated and the most significant ones are reported. The associations in both steps are assessed by the Bayes factor (BF), the ratio between the likelihood of observed traits under the alternative hypothesis (presence of association between single or multiple loci with traits) and that under the null hypothesis (lack of association).</p><p id="p-11">Different from the association mapping based on the SNPs that are directly measured, the local ancestries are unobserved and are inferred on the basis of the AIMs via use of the Hidden Markov Model (HMM) detailed in the Appendices. At each AIM locus, the number of alleles from the high-risk ancestral population is imputed multiple times for every subject, using an Markov chain Monte Carlo (MCMC) algorithm. By using the multiple imputed datasets of local ancestries, we are able to assess the association between the traits and local ancestries directly while taking imputation uncertainty into account through Bayesian averaging. Importantly, our multiple imputation approach preserves the admixture linkage disequilibrium between the AIM loci, which is crucial for multilocus admixture mapping in GLEAM. In addition, GLEAM can also use the local ancestries sampled by other local ancestry inferring methods, such as HAPMIX (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Price <em>et al.</em> 2009</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Generalized linear model with QNM prior</h3><p id="p-12">GLEAM is a regression method that extends the current approaches in various ways. The most obvious extension is to accommodate both quantitative and qualitative traits y<em><sub>i</sub></em> through a generalized linear model with the ability to adjust for covariates <strong><em>E</em></strong><em><sub>i</sub></em> = (<em>E<sub>i</sub></em><sub>1</sub>, <em>E<sub>i</sub></em><sub>2</sub>, …, <em>E<sub>iq</sub></em>)′. Specifically, we use the linear model for continuous traits,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>and the logistic model for dichotomous traits,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>p</em> local ancestries <strong><em>S</em></strong><em><sub>i</sub></em> = (<em>S<sub>i</sub></em><sub>1</sub>, <em>S<sub>i</sub></em><sub>2</sub>, …, <em>S<sub>ip</sub></em>)′ are considered and centered to have mean zero, <strong><em>β</em></strong> = (<em>β</em><sub>1</sub>, <em>β</em><sub>2</sub>, …, <em>β<sub>p</sub></em>)′ and <strong><em>α</em></strong> = (<em>α</em><sub>1</sub>, <em>α</em><sub>2</sub>, …, <em>α<sub>q</sub></em>)′ are the regression coefficients for <strong><em>S</em></strong><em><sub>i</sub></em> and <strong><em>E</em></strong><em><sub>i</sub></em> respectively, and <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-3.gif"/></noscript></span></span>. We use the Bayes factor to assess the admixture association between local ancestries and the trait of interest. The Bayes factor is the ratio of the marginal likelihoods under the alternative hypothesis, <strong>H</strong><sub>1</sub>: <em>β<sub>j</sub></em> ≠ 0 for <em>j</em> = 1, …, <em>p</em>, and null hypothesis, <strong>H</strong><sub>0</sub>: <em>β<sub>j</sub></em> = 0 for <em>j</em> = 1, …, <em>p</em>. Marginal likelihoods remove the parameters from the likelihood by integrating over the prior distribution. The larger the Bayes factor, the stronger the evidence in favor of <strong>H</strong><sub>1</sub>.</p><p id="p-13">As a prior distribution for <em>β</em> under <strong>H</strong><sub>1</sub>, we use the QNM prior having density<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-4.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-5.gif"/></noscript></span></span> is the <em>p</em>-dimensional multivariate normal distribution with the mean vector <strong><em>m</em></strong> and covariance matrix <strong><em>V</em></strong>, and <em>τ</em> is the dispersion parameter. The QNM prior is able to incorporate the case with a large number of loci of tiny effect. As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, the modes of the prior distribution will move toward zero when we reduce the value of <em>τ</em>. For illustration purposes, we only showed a particular value of <em>τ</em> = 0.01, but as we decrease this value, tiny effects are accommodated. For data containing a large number of loci of tiny effect, the empirical Bayes approach should estimate a very small value, and the QNM prior will concentrate on very small effect sizes. Usual priors face major problems in distinguishing the signal from the noise, and we argue that nonlocal priors such as the quadratic normal provide more accurate results for genetic effects on complex traits. Hence, The QNM prior increases the evidence in favor of both the true null and true alternative hypothesis, compared to other prior distributions (<em>e.g.</em>, intrinsic and Cauchy priors) (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Johnson and Rossell 2010</a>). Moreover, we specify <em>σ</em><sup>2</sup><strong>Σ</strong> as the covariance matrix of the (iterative weighted) least square estimation of <strong><em>β</em></strong> in the GLM. This choice not only leads to convenient computation but also easily incorporates the prior knowledge about the effect of local ancestry on the trait. For example, when <strong><em>S</em></strong><em><sub>i</sub></em> is orthogonal to <strong><em>E</em></strong><em><sub>i</sub></em>, <strong>Σ</strong> = (<strong><em>S</em>′<em>S</em></strong>)<sup>−1</sup> with <strong><em>S</em></strong> = [<strong><em>S</em></strong><sub>1</sub>, <strong><em>S</em></strong><sub>2</sub>, …, <strong><em>S</em></strong><em><sub>I</sub></em>]′ in the linear model for the continuous trait. As illustrated by the right panel of <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, the QNM prior with <strong>Σ</strong> = (<strong><em>S</em>′<em>S</em></strong>)<sup>−1</sup> suggests that for each locus, the greater the proportion of alleles from the high-risk population (<em>p<sub>a</sub></em>), on average the larger the risk effect of local ancestry. Such relationships frequently are observed in admixture mapping but not in association mapping based on SNPs in general. More importantly, when we investigate multiple loci simultaneously, it is crucial to take the correlation (linkage disequilibrium, LD) between the local ancestries into consideration. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> plots several volcano-shaped bivariate QNM densities for various correlations between two local ancestries. It is clear that for two loci with admixture linkage equilibrium (as shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), such as two loci on different chromosomes, their risk effects would be independent; and that for two loci with high admixture LD (as shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2D</a>), usually located in the same gene, they would have similar risk effects.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate quadratic normal moment prior (A) for &#x3C4; = 0.01 (&#x2014;), &#x3C4; = 0.05 (&#x22C5;&#x22C5;&#x22C5;), and &#x3C4; = 0.1 (&#x2212;&#x22C5;&#x2212;) when pa = 0.8; (B) for pa = 0.8 (&#x2014;), pa = 0.9 (&#x2026;), and pa = 0.99 (&#x2212;&#x22C5;&#x2212;) when &#x3C4; = 0.01. In both cases, &#x3C3;2 = 1 and  with Pr(Si = 0) = (1&#x2212;pa)2, Pr(Si = 1) = 2pa(1&#x2212;pa) and ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-292504072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Univariate quadratic normal moment prior (A) for &lt;em&gt;&#x3C4;&lt;/em&gt; = 0.01 (&#x2014;), &lt;em&gt;&#x3C4;&lt;/em&gt; = 0.05 (&#x22C5;&#x22C5;&#x22C5;), and &lt;em&gt;&#x3C4;&lt;/em&gt; = 0.1 (&#x2212;&#x22C5;&#x2212;) when &lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt; = 0.8; (B) for &lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt; = 0.8 (&#x2014;), &lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt; = 0.9 (&#x2026;), and &lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt; = 0.99 (&#x2212;&#x22C5;&#x2212;) when &lt;em&gt;&#x3C4;&lt;/em&gt; = 0.01. In both cases, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 1 and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-6&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with Pr(&lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt; = 0) = (1&#x2212;&lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt;)&lt;sup&gt;2&lt;/sup&gt;, Pr(&lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt; = 1) = 2&lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt;(1&#x2212;&lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt;) and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-7&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1165/F1.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/7/1165/F1.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Univariate quadratic normal moment prior (A) for <em>τ</em> = 0.01 (—), <em>τ</em> = 0.05 (⋅⋅⋅), and <em>τ</em> = 0.1 (−⋅−) when <em>p<sub>a</sub></em> = 0.8; (B) for <em>p<sub>a</sub></em> = 0.8 (—), <em>p<sub>a</sub></em> = 0.9 (…), and <em>p<sub>a</sub></em> = 0.99 (−⋅−) when <em>τ</em> = 0.01. In both cases, <em>σ</em><sup>2</sup> = 1 and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F1/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F1/embed/mml-math-6.gif"/></noscript></span></span> with Pr(<em>S<sub>i</sub></em> = 0) = (1−<em>p<sub>a</sub></em>)<sup>2</sup>, Pr(<em>S<sub>i</sub></em> = 1) = 2<em>p<sub>a</sub></em>(1−<em>p<sub>a</sub></em>) and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F1/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F1/embed/mml-math-7.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bivariate quadratic normal moment prior with &#x3C4;&#x3C3;2 = 0.1 and &#x3A3; = (S&#x2032;S)&#x2212;1, where S = [S1, S2]&#x2032;, S1 = (S1,1, S1,2, &#x2026;, S1000,1)&#x2032;, S2 = (S1,2, S2,2, &#x2026;, S1000,2)&#x2032; and Si1 &#x2208; {0, 1, 2} and Si2 &#x2208; {0, 1, 2}. We introduce correlation between Si1 and Si2 through the latent variables (Zi1, Zi2), where ,  and Cov(Zi1, Zi2) = &#x3C1;. let Si1 = 0 if Zi1 &#x2264; C0; Si1 = 2 if Zi1 &gt; C1; and Si1 = 0 otherwise with C0 = &#x3A6;&#x2212;1((1&#x2212;pa)2) and  where &#x3A6;&#x2212;1(&#x22C5;) denotes normal inverse cumulative distribution function. We consider four scenarios when pa = 0.8: (A) &#x3C1; = 0; (B) &#x3C1; = 0.25; (C) &#x3C1; = 0.5; and (D) &#x3C1; = 0.75 with contours drawn beneath the probability density function&#x2019;s surface." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-292504072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Bivariate quadratic normal moment prior with &lt;em&gt;&#x3C4;&#x3C3;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.1 and &lt;strong&gt;&#x3A3;&lt;/strong&gt; = (&lt;strong&gt;&lt;em&gt;S&lt;/em&gt;&#x2032;&lt;em&gt;S&lt;/em&gt;&lt;/strong&gt;)&lt;sup&gt;&#x2212;1&lt;/sup&gt;, where &lt;strong&gt;&lt;em&gt;S&lt;/em&gt;&lt;/strong&gt; = [&lt;strong&gt;&lt;em&gt;S&lt;/em&gt;&lt;/strong&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;strong&gt;&lt;em&gt;S&lt;/em&gt;&lt;/strong&gt;&lt;sub&gt;2&lt;/sub&gt;]&#x2032;, &lt;strong&gt;&lt;em&gt;S&lt;/em&gt;&lt;/strong&gt;&lt;sub&gt;1&lt;/sub&gt; = (&lt;em&gt;S&lt;/em&gt;&lt;sub&gt;1,1&lt;/sub&gt;, &lt;em&gt;S&lt;/em&gt;&lt;sub&gt;1,2&lt;/sub&gt;, &#x2026;, &lt;em&gt;S&lt;/em&gt;&lt;sub&gt;1000,1&lt;/sub&gt;)&#x2032;, &lt;strong&gt;&lt;em&gt;S&lt;/em&gt;&lt;/strong&gt;&lt;sub&gt;2&lt;/sub&gt; = (&lt;em&gt;S&lt;/em&gt;&lt;sub&gt;1,2&lt;/sub&gt;, &lt;em&gt;S&lt;/em&gt;&lt;sub&gt;2,2&lt;/sub&gt;, &#x2026;, &lt;em&gt;S&lt;/em&gt;&lt;sub&gt;1000,2&lt;/sub&gt;)&#x2032; and &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; &#x2208; {0, 1, 2} and &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; &#x2208; {0, 1, 2}. We introduce correlation between &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; and &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; through the latent variables (&lt;em&gt;Z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;Z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;), where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-8&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and Cov(&lt;em&gt;Z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;Z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = &lt;em&gt;&#x3C1;&lt;/em&gt;. let &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0 if &lt;em&gt;Z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; &#x2264; &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;; &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 2 if &lt;em&gt;Z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; &gt; &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;; and &lt;em&gt;S&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0 otherwise with &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt; = &#x3A6;&lt;sup&gt;&#x2212;1&lt;/sup&gt;((1&#x2212;&lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt;)&lt;sup&gt;2&lt;/sup&gt;) and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &#x3A6;&lt;sup&gt;&#x2212;1&lt;/sup&gt;(&#x22C5;) denotes normal inverse cumulative distribution function. We consider four scenarios when &lt;em&gt;p&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt; = 0.8: (A) &lt;em&gt;&#x3C1;&lt;/em&gt; = 0; (B) &lt;em&gt;&#x3C1;&lt;/em&gt; = 0.25; (C) &lt;em&gt;&#x3C1;&lt;/em&gt; = 0.5; and (D) &lt;em&gt;&#x3C1;&lt;/em&gt; = 0.75 with contours drawn beneath the probability density function&#x2019;s surface.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1165/F2.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/7/1165/F2.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462992" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">Bivariate quadratic normal moment prior with <em>τσ</em><sup>2</sup> = 0.1 and <strong>Σ</strong> = (<strong><em>S</em>′<em>S</em></strong>)<sup>−1</sup>, where <strong><em>S</em></strong> = [<strong><em>S</em></strong><sub>1</sub>, <strong><em>S</em></strong><sub>2</sub>]′, <strong><em>S</em></strong><sub>1</sub> = (<em>S</em><sub>1,1</sub>, <em>S</em><sub>1,2</sub>, …, <em>S</em><sub>1000,1</sub>)′, <strong><em>S</em></strong><sub>2</sub> = (<em>S</em><sub>1,2</sub>, <em>S</em><sub>2,2</sub>, …, <em>S</em><sub>1000,2</sub>)′ and <em>S<sub>i</sub></em><sub>1</sub> ∈ {0, 1, 2} and <em>S<sub>i</sub></em><sub>2</sub> ∈ {0, 1, 2}. We introduce correlation between <em>S<sub>i</sub></em><sub>1</sub> and <em>S<sub>i</sub></em><sub>2</sub> through the latent variables (<em>Z<sub>i</sub></em><sub>1</sub>, <em>Z<sub>i</sub></em><sub>2</sub>), where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F2/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F2/embed/mml-math-8.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F2/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F2/embed/mml-math-9.gif"/></noscript></span></span> and Cov(<em>Z<sub>i</sub></em><sub>1</sub>, <em>Z<sub>i</sub></em><sub>2</sub>) = <em>ρ</em>. let <em>S<sub>i</sub></em><sub>1</sub> = 0 if <em>Z<sub>i</sub></em><sub>1</sub> ≤ <em>C</em><sub>0</sub>; <em>S<sub>i</sub></em><sub>1</sub> = 2 if <em>Z<sub>i</sub></em><sub>1</sub> &gt; <em>C</em><sub>1</sub>; and <em>S<sub>i</sub></em><sub>1</sub> = 0 otherwise with <em>C</em><sub>0</sub> = Φ<sup>−1</sup>((1−<em>p<sub>a</sub></em>)<sup>2</sup>) and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F2/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/F2/embed/mml-math-10.gif"/></noscript></span></span> where Φ<sup>−1</sup>(⋅) denotes normal inverse cumulative distribution function. We consider four scenarios when <em>p<sub>a</sub></em> = 0.8: (A) <em>ρ</em> = 0; (B) <em>ρ</em> = 0.25; (C) <em>ρ</em> = 0.5; and (D) <em>ρ</em> = 0.75 with contours drawn beneath the probability density function’s surface.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Under the QNM prior for <em>β</em>, the Bayes factor is simply<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-11.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-12.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-13.gif"/></noscript></span></span> is the maximum likelihood estimate of <strong><em>β</em></strong>, adjusted by other risk covariates when necessary, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-14.gif"/></noscript></span></span> is the corresponding covariance matrix estimate and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-16.gif"/></noscript></span></span> are the empirical Bayes estimates. Bayes factor (3) is used to identify the loci associated with the traits, detailed as follows.</p></div><div id="sec-3" class="subsection"><h3>Generalized admixture mapping procedure</h3><p id="p-17">We propose a two-stage approach for GLEAM. In the first stage, we examine the marginal association between a single AIM locus and the trait, using the Bayes factors (3), one locus a time for <em>J</em> AIM loci. The loci at which <em>log</em><sub>10</sub><em>BF</em>(<strong>y</strong>) &gt; <em>δ</em> are considered susceptibility loci. Although the “one locus a time” approach explores the marginal association and is widely used, marginal association only reflects part of the relationship between the AIM loci and the trait. Several loci in different regions may show associations with the trait. Thus, it is desirable to quantify the evidence for joint association of multiple loci with the trait. For this reason, in the second stage, we list all possible combinations of susceptibility loci selected in the first stage. For each set of susceptibility loci, we can again calculate the Bayes factors for the joint association at those loci simultaneously. The most significant ones are reported. The local ancestries at the AIM loci are unobserved and imputed from the HMM. The imputation uncertainty could be properly accounted for by calculating weighted average of the Bayes factors for each imputed local ancestry dataset, which is similar to the strategy used by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Guan and Stephens (2008)</a> in imputation-based association mapping for testing untyped variants.</p></div><div id="sec-4" class="subsection"><h3>Simulation studies</h3><p id="p-18">We carried out simulation studies to assess the performance of GLEAM in terms of type I error rate and power under various scenarios and compared it with the method based on Bayesian likelihood ratio (BLR) by <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Patterson <em>et al.</em> (2004)</a>, which is implemented by the software ANCESTRYMAP (<a href="http://http://genepath.med.harvard.edu/~reich/Software.htm">http://genepath.med.harvard.edu/∼reich/Software.htm</a>) as well as regularized regression methods Lasso and elastic net (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tibshirani 1996</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zou and Hastie 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Friedman <em>et al.</em> 2010</a>). GLEAM and ANCESTRYMAP use slightly different HMMs to impute the local ancestries and regularized regression methods require given local ancestries. Because of these differences, we assumed the true local ancestries were given and focused on evaluating the ability of localizing susceptibility loci instead of estimating local ancestries. Our simulations were based on empirical data of local ancestries for 1001 African-American subjects from the HPHB Study (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Miranda <em>et al.</em> 2009</a>), with 1296 AIM loci measured across the genome. The MATLAB codes for simulating and analyzing the data are included in a <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006478/-/DC1/006478SI.zip">Supporting Information</a> folder online.</p><p id="p-19">We started by investigating the type I error rates for the local ancestries that were scattered around different regions of the genome and in linkage equilibrium. Under this scenario, the falsely localized AIM locus would be in the region remote from the true disease causing locus, which leads to a false positive finding. We first randomly sampled 1000 AIM loci with replacement from 1296 AIM loci for 1000 subjects. At each AIM locus, we simulated the local ancestries measured by the number of alleles from the African ancestral population from their maximum <em>a posteriori</em> (MAP) frequency estimates under the assumption of Hardy-Weinberg equilibrium. Ten sets of trait data were then generated such that we were able to assess the type I error rates under the genome-wide threshold level (<em>e.g.</em>, <em>α</em> = 10<sup>−4</sup>), by using the following null model for continuous traits: y<em><sub>i</sub></em> = <em>αE<sub>i</sub></em> + <em>ε<sub>i</sub></em> and for binary traits, logit{Prob(y<em><sub>i</sub></em> = 1)} = <em>αE<sub>i</sub></em>; where the continuous risk covariate <em>E<sub>i</sub></em> and the measurement error <em>ε<sub>i</sub></em> followed standard normal distributions. We considered two situations whereby <em>α</em> = 0 in the absence of a covariate effect and <em>α</em> = 1 in the presence of a covariate effect.</p><p id="p-20">We next examined power under the single locus alternative models. We simulated 100 sets of traits. Each set included 1000 subjects and one disease associated local ancestry whose location was randomly sampled from 259 AIM loci, where the proportion of African ancestral population ranged from 0.8321 to 0.8817 and was on the top 20% percentile among 1296 AIM loci. Given the local ancestry <em>S<sub>i</sub></em>, continuous covariates <strong><em>E</em></strong><em><sub>i</sub></em> and measurement error <em>ε<sub>i</sub></em> generated same as that for the null model, continuous traits were simulated from y<em><sub>i</sub></em> = <em>αE<sub>i</sub></em> + <em>βS<sub>i</sub></em> + <em>ε<sub>i</sub></em> and binary traits from logit{Prob(y<em><sub>i</sub></em> = 1)} = <em>αE<sub>i</sub></em> + <em>βS<sub>i</sub></em>. Under both models, the <em>β</em> was specified as <em>β</em> = <em>c</em> × proportion of African ancestral population which reflected the <em>a priori</em> observation that the locus with the larger proportion of the high-risk ancestral (here African American) population usually demonstrated stronger association with the traits. For continuous traits, we chose the values of effect size multiplier <em>c</em> as 0.2, 0.25, 0.3, 0.35, and 0.4 respectively, with the largest possible effect size equal to 0.3527. Similarly, we picked the <em>c</em> values as 0.4, 0.5, 0.6, 0.7, and 0.8 for binary traits with the largest possible odds ratio equal to 1.8537.</p><p id="p-21">We further considered a multilocus alternative model where two local ancestries were associated with the traits and there existed admixture linkage disequilibrium. To do so, we generated an artificial chromosome composed of two pieces from chromosome 1 and chromosome 4 with the length 139.50 Mb and 114.88 Mb, respectively, for 1000 subjects, based on empirical data on local ancestries from HPHB study. In the middle of each chromosome piece with 51 loci, there is one locus whose proportion of African ancestry population was among the highest in all 1296 AIM loci. In the simulations, those two loci are assumed to be associated with traits. We generated 100 sets of continuous and binary traits respectively, each of which was simulated similarly to the single locus alternative model except with two local ancestries involved and both effect size multiplier <em>c</em> values set at 0.7 for continuous traits and 0.35 for binary traits.</p><p id="p-22">The simulated datasets were analyzed by the GLEAM and the BLR method. Because the BLR method was primarily developed for binary traits, the BLR method required transformation of continuous traits into binary ones, such as defining the subjects with top 20% traits as the cases and the one with bottom 20% traits as controls.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Simulation studies</h3><p id="p-23"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> presents the empirical type I error rates for both the binary and continuous traits, with or without covariate effects. For the GLEAM and the BLR methods, we chose a threshold of 2 for <em>log</em><sub>10</sub><em>BF</em>(<strong>y</strong>) to control the genome-wide type I error rates. The regularization parameters of Lasso and elastic net are chosen with the minimal cross validation error. The loci with nonzero regression coefficients are regarded as the ones associated with the traits. As illustrated in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>, under the null model that all the local ancestries are in linkage equilibrium, the type I error rate is controlled at a low level with the median around 5 × 10<sup>−4</sup> for GLEAM and 4.2 × 10<sup>−3</sup> for the BLR method. In both cases, those type I error rates seem overly conservative. However, in the application to real data, slight admixture linkage disequilibrium between the AIM loci will significantly inflate the type I error rate close to the nominal levels (<em>i.e.</em>, <em>α</em> = 0.05 or 0.005), which is discussed in the later paragraphs. Comparing <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a> reveals that the type I error rates of GLEAM are consistently smaller than those of the method based on BLR and are little affected by the presence of covariate effects when properly adjusted. The covariates are not considered by the BLR method and have a mixed effect on type I error rates, where the median is slightly reduced with the maximal type I error rates increased. For the regularized regression methods Lasso and elastic net, the type I errors are significantly inflated, as shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a> and <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>. In addition, when a nonzero covariate presents, the type I errors will further increase.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The type I error rates under the null model (note the different scaling of the Y-axis for panels). The type I error rates are presented for both the binary and continuous traits respectively, with or without covariate effect (denoted by E1 and E0, respectively). For each simulated dataset, we calculate one type I error rate for each method. The results for 100 replications are summarized by the box plots, where the center bar is median, bottom and top of the box are the 25th and 75th percentile and the whiskers stretch out until the extreme values. (A) Generalized admixture mapping; (B) Method based on BLR; (C) Regularized regression with Lasso; (D) Regularized regression with elastic net." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-292504072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The type I error rates under the null model (note the different scaling of the Y-axis for panels). The type I error rates are presented for both the binary and continuous traits respectively, with or without covariate effect (denoted by E1 and E0, respectively). For each simulated dataset, we calculate one type I error rate for each method. The results for 100 replications are summarized by the box plots, where the center bar is median, bottom and top of the box are the 25th and 75th percentile and the whiskers stretch out until the extreme values. (A) Generalized admixture mapping; (B) Method based on BLR; (C) Regularized regression with Lasso; (D) Regularized regression with elastic net.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1165/F3.medium.gif" width="440" height="422"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/7/1165/F3.medium.gif" width="440" height="422"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462989" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">The type I error rates under the null model (note the different scaling of the Y-axis for panels). The type I error rates are presented for both the binary and continuous traits respectively, with or without covariate effect (denoted by E1 and E0, respectively). For each simulated dataset, we calculate one type I error rate for each method. The results for 100 replications are summarized by the box plots, where the center bar is median, bottom and top of the box are the 25th and 75th percentile and the whiskers stretch out until the extreme values. (A) Generalized admixture mapping; (B) Method based on BLR; (C) Regularized regression with Lasso; (D) Regularized regression with elastic net.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Power of the methods also was evaluated for binary and continuous traits under the single locus alternative model, with or without covariate effects. We considered various effect sizes of local ancestries with the results shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. For the binary trait, when the effect size is small, the BLR method performs better with larger power. With the increment of the effect sizes, GLEAM gradually outperforms the BLR method. For both methods, covariates have moderate effects on power, which is more obvious for the smaller effect sizes. For the continuous trait, the GLEAM performs significantly better at each effect size. These results were expected since the BLR method discards part of the dataset in order to transform the continuous trait into the binary one (case <em>vs.</em> control), which inevitably loses power. For all situations considered, the power of the GLEAM approach increases with the increment of the local ancestry effect size, most rapidly when the effect sizes are smaller and then levels off with larger effect sizes. In comparison, the power of the BLR method increases roughly linearly. Both GLEAM and BLR are less powerful than the regularized methods especially when the effect sizes are small. With the growth of the effect size, the power of GLEAM will quickly increase and be comparable to the ones of regularized regression.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Powers for single locus alternative models. Power is calculated for each dataset with 100 replications total for the binary or continuous traits simulated under the single locus alternative model with or without covariate effect. The &#xD7; indicates the median of powers by the GLEAM and&#x2022;denotes the median of powers by the method based on Bayesian likelihood ratio; &#x2218; denotes the median of regularized regression with lasso; &#x2218; denotes the median of regularized regression with elastic net. The whiskers on each bar represent the minimal and maximal powers respectively. The effect sizes of local ancestries are equal to the multiplication of effect size multiplier c and the proportion of African ancestry population. (A) Binary traits without covariate effect; (B) Binary traits with covariate effect; (C) Continuous traits without covariate effect; (D) Continuous traits with covariate effect." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-292504072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Powers for single locus alternative models. Power is calculated for each dataset with 100 replications total for the binary or continuous traits simulated under the single locus alternative model with or without covariate effect. The &#xD7; indicates the median of powers by the GLEAM and&#x2022;denotes the median of powers by the method based on Bayesian likelihood ratio; &#x2218; denotes the median of regularized regression with lasso; &#x2218; denotes the median of regularized regression with elastic net. The whiskers on each bar represent the minimal and maximal powers respectively. The effect sizes of local ancestries are equal to the multiplication of effect size multiplier &lt;em&gt;c&lt;/em&gt; and the proportion of African ancestry population. (A) Binary traits without covariate effect; (B) Binary traits with covariate effect; (C) Continuous traits without covariate effect; (D) Continuous traits with covariate effect.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1165/F4.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/7/1165/F4.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462935" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Powers for single locus alternative models. Power is calculated for each dataset with 100 replications total for the binary or continuous traits simulated under the single locus alternative model with or without covariate effect. The × indicates the median of powers by the GLEAM and•denotes the median of powers by the method based on Bayesian likelihood ratio; ∘ denotes the median of regularized regression with lasso; ∘ denotes the median of regularized regression with elastic net. The whiskers on each bar represent the minimal and maximal powers respectively. The effect sizes of local ancestries are equal to the multiplication of effect size multiplier <em>c</em> and the proportion of African ancestry population. (A) Binary traits without covariate effect; (B) Binary traits with covariate effect; (C) Continuous traits without covariate effect; (D) Continuous traits with covariate effect.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To understand the impact of admixture linkage disequilibrium on type I error rates and to evaluate the ability of localizing multiple loci simultaneously, we generated a set of artificial chromosomes as described previously, where two loci were associated with the traits, named as Locus 1 and Locus 2. Besides Locus 1 and Locus 2, we divided the remaining loci into three regions: region 1 (REG1) with 42 loci and region 2 (REG2) with 35 loci, where the admixture linkage disequilibrium measured by the correlation coefficient between a given locus at these regions and Locus 1 or Locus 2 was larger than 0.12 respectively; and region 3 (REG3), the unassociated loci which did not belong to region 1 and region 2. Strictly speaking, the identified loci except Locus 1 and Locus 2 were all false positives. However, in contrast to the loci found in region 3, which were completely false findings, the loci identified in Region 1 and Region 2 were partially correct and could be regarded as low-resolution findings instead, because the true associated locus did exist in the nearby region. Therefore, we evaluated the false positives in three regions separately. An ideal method under the prespecified genome-wide threshold would lead to few completely false positives in region 3 and to a small number of partially false positives in regions 1 and 2, while being able to identify the true associated loci with high frequency.</p><p id="p-28"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes the frequencies of identified loci for each locus or locus combination at different regions by GLEAM, BLR and regularized regression methods. For the GLEAM method, we applied the two-step approach outlined in the “Generalized admixture mapping procedure” subsection. The results by applying the first step only (GLEAM1) and by applying the two-step approach (GLEAM2) were both presented. For binary traits, both the BLR method and GLEAM1 could localize both Locus 1 and Locus 2 with high power. The type I error rates in region 1 were around the nominal level (0.025 and 0.003, respectively). The type I error rates in region 1 and region 2 were higher than the ones in region 3, which would decrease the resolution of the finding. Compared with GLEAM1, further applying the second step of generalized admixture mapping procedure (GLEAM2) could significantly improve the resolution by reducing the type I errors in region 1 (from 0.013 to 0.002) and region 2 (from 0.014 to 0.003). For continuous traits, GLEAM2 also performed best with much higher power and lower type I rate than the BLR method. Similar to the simulation results under null and single locus alternative model, regularized regressions show marginally higher power at the cost of inflated type I error rate, <em>e.g.</em>, power 1 for detecting both locus 1 and 2 with type I error rates 0.023 of Lasso and 0.029 of elastic net at region 3 for the continuous trait.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463024/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463024/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The frequency of identified loci for each locus or locus combination at different regions of the artificial chromosome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Application</h3><p id="p-33">We applied our approach to data from the Healthy Pregnancy, Healthy Baby (HPHB) study, which is a prospective cohort study of pregnant women aimed at identifying genetic, social, and environmental contributors to disparities in adverse birth outcomes in the Southern United States (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Miranda <em>et al.</em> 2009</a>). Consistent with previous studies, African-American women in HPHB have greater risk for maternal hypertension than white women during the pregnancy, which contributes to the poor birth outcomes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2004</a>). Even within the African-American subpopulation, some women have much greater blood pressure, and we hypothesize that one possible contributor may be the percentage of African ancestry. To explore this hypothesis, we applied GLEAM to investigate the association between the averaged maternal mean arterial pressure (MAP), defined as (1/3 × systolic blood pressure) + (2/3 × systolic blood pressure), during 24 to 28 weeks of pregnancy and local ancestries among these pregnant African-American women. Clinical and genetic data were available for 1004 non-Hispanic black women. A total of 1509 SNP AIMs were genotyped using the Illumina African-American admixture panel. After quality control measures described previously (A. E. Ashley-Koch, Me. E. Garrett, S. Edwards, K. S. Quinn, G. K. Swamy, and M. L. Miranda, unpublished results), the dataset consisted of 1001 non-Hispanic black women with 1296 AIMs.</p><p id="p-34">The proposed GLEAM approach was applied to this dataset to identify the local ancestry associated with the averaged maternal MAP, a continuous trait, while we adjusted for mother’s age. The local ancestries were multiply imputed based on the HMM. We first examined the marginal association between the trait and local ancestries, one locus a time. The results are summarized in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, where one local ancestry on the chromosome 2 was identified with its <em>log</em><sub>10</sub>(Bayes factor) = 2.05 exceeding the threshold 2. With only one local ancestry localized, the second step of the generalized admixture mapping procedure was unnecessary. The same data were analyzed by the BLR method, which treated the subjects with averaged maternal MAP more than 93.67 (top 20% quantile) as cases and the ones with averaged maternal MAP less than 79.33 (bottom 20% quantile) as control. No local ancestry was identified as being associated with the averaged maternal MAP with this approach, presumably due to its relatively low power compared with the GLEAM approach.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of log10(Bayes factor) for the association between the averaged maternal MAP during 24 to 28 weeks of pregnancy and genome-wide local ancestries among 1001 African-American subjects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-292504072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plot of &lt;em&gt;log&lt;/em&gt;&lt;sub&gt;10&lt;/sub&gt;(Bayes factor) for the association between the averaged maternal MAP during 24 to 28 weeks of pregnancy and genome-wide local ancestries among 1001 African-American subjects.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1165/F5.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/7/1165/F5.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1165/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462969" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Manhattan plot of <em>log</em><sub>10</sub>(Bayes factor) for the association between the averaged maternal MAP during 24 to 28 weeks of pregnancy and genome-wide local ancestries among 1001 African-American subjects.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-36">When the admixture linkage disequilibrium is used, admixture mapping is an indispensable tool to localize the alleles that are associated with the qualitative or quantitative traits and diseases that vary in prevalence across the ancestral populations. In this article, we propose a flexible and powerful generalized admixture mapping approach, which is based on the generalized linear model and is able to incorporate admixture prior information by using the quadratic normal moment prior and to adjust for covariates. The proposed method is applicable to both qualitative and quantitative traits with satisfactory power while controlling the type I error rates at a low level, and is able to be easily implemented as we demonstrated with our HPHB example.</p><p id="p-37">In addition to the flexibility to handle different types of traits, other attractive generalizations include consideration of multiple loci simultaneously. It is known that admixture linkage disequilibrium extends much further than haplotype linkage disequilibrium. Consequently, if we only examine one locus a time, the local ancestries which are highly correlated to the true disease associated local ancestry tend to be identified as significant ones as well. As demonstrated by the simulations, those false positives can be significantly reduced by considering multiple susceptible loci simultaneously, which reduce the type I error rates and improve the mapping resolution. In addition, GLEAM specifies a hidden Markov model treating the recombination rates varying across the genome, which allows us to infer the recombination “hotspots” in admixture population. Moreover, within the generalized linear model framework, it is straightforward to extend the current method to populations with more than to two ancestral populations, such as Hispanic populations, by adding extra ancestry population covariates. It is also easy to consider the interaction between the local ancestries and covariates with the properly specification of the priors on interaction coefficients.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">This work was supported by Award Number R01ES017436 from the National Institute of Environmental Health Sciences, by funding from the National Institutes of Health (5P2O-RR020782-O3), and the U.S. Environmental Protection Agency (RD-83329301-0) and by the Intramural Research Program of the National Cancer Institute, National Institutes of Health, Bethesda, Maryland. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of Environmental Health Sciences, the National Institutes of Health, or the U.S. Environmental Protection Agency.</p></div><div class="section app" id="app-1"><h2 class="">Appendices</h2><div class="section" id="sec-9"><h3>Hidden Markov Model</h3><p id="p-38">For a population-based design, suppose we have <em>I</em> unrelated subjects, each of which has the same set of <em>J</em> AIMs recorded. The local ancestry is measured by <em>S<sub>ij</sub></em> ∈ {0, 1, 2}, the number of alleles from the high-risk population A (<em>e.g.</em>, African) for the <em>i</em>th subject and the <em>j</em>th AIM. <em>S<sub>ij</sub></em> is unknown and will be imputed using the HMM. For African Americans with African and European ancestral populations, HMM assumes that given the <em>S<sub>ij</sub></em>, the distribution of <em>X<sub>ij</sub></em> ∈ {0, 1, 2}, the number of variant alleles, is independent of other <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-18.gif"/></noscript></span></span> with <em>j</em>′ ≠ <em>j</em> and is specified by the observation probability mass matrix <strong><em>P</em></strong><em><sub>j</sub></em> = {<em>p<sub>j</sub></em>(<em>m</em>, <em>n</em>)}<sub>3×3</sub> with <em>p<sub>j</sub></em>(<em>m</em>, <em>n</em>) = Prob(<em>X<sub>ij</sub></em> = <em>n</em>|<em>S<sub>ij</sub></em> = <em>m</em>) and<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-19.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-20.gif"/></noscript></span></span> is the minor allele probability at loci <em>j</em> in the high-risk population <em>A</em> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-21.gif"/></noscript></span></span> is the corresponding probability in the low-risk population <em>B</em>.</p><p id="p-39">The latent states <strong><em>S</em></strong><em><sub>i</sub></em> = {<em>S<sub>ij</sub></em>}<sub>1×</sub><em><sub>J</sub></em>, tagging the status of the ancestry blocks, are unobserved and modeled by a Markov chain which considers the genetic recombination events. Let <em>ρ<sub>i</sub></em> denote the genome-wide proportion of alleles from the high-risk population A for subject <em>i</em>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-22.gif"/></noscript></span></span> initial state vector, <em>R<sub>ij</sub></em> ∈ {0, 1, 2} the number of recombination events between AIM loci <em>j</em> − 1 and <em>j</em>, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-23.gif"/></noscript></span></span> the conditional state transition matrix given <em>r</em> recombination events between the neighboring AIM loci with <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-24.gif"/></noscript></span></span>. The Markov chain <strong><em>S</em></strong><em><sub>i</sub></em> is governed by the state transition matrix <strong><em>Q</em></strong><em><sub>ij</sub></em> = {<em>q<sub>ij</sub></em>(<em>m</em>, <em>n</em>)}<sub>3×3</sub> with <em>q<sub>ij</sub></em>(<em>m</em>, <em>n</em>) = Prob(<em>S<sub>ij</sub></em> = <em>n</em>|<em>S<sub>i</sub></em><sub>(</sub><em><sub>j</sub></em><sub>−1)</sub> = <em>m</em>). <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-25.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-26.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-28.gif"/></noscript></span></span> are specified as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-29.gif"/></noscript></span></span>and <em>R<sub>ij</sub></em> ~ Bin(2, <em>γ<sub>j</sub></em>) a binomial distribution with <em>γ<sub>j</sub></em> the probability that a recombination event occurs between the neighboring AIM loci in a single chromosome. Consequently, we can get,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-30.gif"/></noscript></span></span>We further specify informative prior distributions for the parameters <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-31.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-32.gif"/></noscript></span></span>, <em>γ<sub>j</sub></em> and <em>ρ<sub>i</sub></em> involved in the HMM. Although the <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-33.gif"/></noscript></span></span> of the high-risk population A is unknown, we have information on <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-34.gif"/></noscript></span></span>, the proportion of the variant allele <em>j</em> in a subpopulation of high-risk population A (<em>e.g.</em> YRI for African), from the HapMap or 1000 genome projects. Hence, we expect that <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-35.gif"/></noscript></span></span> would be close to <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-36.gif"/></noscript></span></span> and specify <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-37.gif"/></noscript></span></span> with the expectation <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-38.gif"/></noscript></span></span> and <em>τ<sup>A</sup></em> ~ ∪ [50, 1000] a uniform distribution to reflect the uncertainty in borrowing the subpopulation information. A similar specification is chosen for <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-39.gif"/></noscript></span></span> based on the proportion of the variant allele <em>j</em> in a subpopulation of low-risk population B (<em>e.g.</em>, CEU for European). As for <em>γ<sub>j</sub></em>, it is well known that the recombination probability is roughly proportional to <em>d<sub>j</sub></em> the genetic distance between (<em>j</em>−1)th and <em>j</em>th AIM loci. A common choice is <em>γ<sub>j</sub></em> = 1−exp(−<em>λd<sub>j</sub></em>) with <em>λ</em> = 6 the number of recombination events per Morgan since admixture (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Falush <em>et al.</em> 2003</a>; <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Patterson <em>et al.</em> 2004</a>). However, recombination “hotspots” can occur along the chromosomes where the recombination probabilities are much greater than the other regions (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Myers <em>et al.</em> 2005</a>). For this reason, we avoid the aforementioned parametric specification of <em>γ<sub>j</sub></em>. Instead, we let <em>γ<sub>j</sub></em> ~ Beta (<em>τ<sup>γ</sup>γ</em><sub>0</sub><em><sub>j</sub></em>, <em>τ<sup>γ</sup></em>(1 − <em>γ</em><sub>0</sub><em><sub>j</sub></em>)) with the expectation <em>E</em>(<em>γ<sub>j</sub></em>) = <em>γ</em><sub>0</sub><em><sub>j</sub></em> = 1−exp(−<em>λd<sub>j</sub></em>). Hence, on average the probability of recombination is proportional to the genetic distance while allowing significant deviation (<em>e.g.</em> ‘hotspots’) from the average. The deviation is measured by <em>τ<sup>γ</sup></em> with <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-40.gif"/></noscript></span></span>. In addition, for the admixed population, we often have knowledge about the proportions of ancestral populations at the population level. For example, the African American population in general consists of 80% African ancestral population and 20% European ancestral population (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Smith and O’Brien, 2005</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Winkler <em>et al.</em> 2010</a>). We borrow this population level information to specify <em>ρ<sub>i</sub></em>, the subject specific proportion of high-risk population A, by letting <em>ρ<sub>i</sub></em> ∼ Beta (<em>τ<sup>ρ</sup>ρ</em><sub>0</sub><em><sub>i</sub></em>, <em>τ<sup>ρ</sup></em>(1 − <em>ρ</em><sub>0</sub><em><sub>i</sub></em>)) with <em>ρ</em><sub>0</sub><em><sub>i</sub></em> (<em>e.g.</em> 0.8 for African American) and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-41.gif"/></noscript></span></span>.</p><p id="p-40">We use an MCMC algorithm to sample the local ancestries <strong><em>S</em></strong><em><sub>i</sub></em> for <em>i</em> = 1. 2, …, <em>I</em>, along with other parameters. The details of MCMC are given as follows.</p></div><div class="section" id="sec-10"><h3>MCMC algorithm for HMM</h3><p>We propose an MCMC algorithm for posterior computation of HMM as follows.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-42"><span class="list-label">(1) </span>Impute the missing AIM <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-42.gif"/></noscript></span></span>. Given the <strong><em>P</em></strong><em><sub>j</sub></em> and <em>S<sub>ij</sub></em>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-43.gif"/></noscript></span></span> can be easily sampled with probability mass <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-44.gif"/></noscript></span></span>.</p></li><li id="list-item-2"><p id="p-43"><span class="list-label">(2) </span>Update the latent states <strong><em>S</em></strong><em><sub>i</sub></em> for <em>i</em> = 1. 2, …, <em>I</em>. Given the <strong><em>Q</em></strong><em><sub>i</sub></em><sub>0</sub>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-45.gif"/></noscript></span></span> and <strong><em>R</em></strong><em><sub>i</sub></em> = {<em>R<sub>ij</sub></em>}<sub>1×</sub><em><sub>j</sub></em>, we will use the forward filtering backward sampling (FFBS) algorithm (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Scott 2002</a>) to sample the <strong><em>S</em></strong><em><sub>i</sub></em> in one block. The FFBS algorithm mixes more rapidly comparing to the direct Gibbs sampler which samples one <em>S<sub>ij</sub></em> a time conditional on the remains of <strong><em>S</em></strong><em><sub>i</sub></em>. Let <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-46.gif"/></noscript></span></span> and <strong><em>R</em></strong><em><sub>i</sub></em> = [<em>R<sub>i</sub></em><sub>1</sub>, <em>R<sub>i</sub></em><sub>2</sub>, …, <em>R<sub>iJ</sub></em>]′. We begin the FFBS algorithm by calculating <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-47.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-48.gif"/></noscript></span></span> recursively for <em>j</em> = 1. 2, …, <em>J</em> as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-49.gif"/></noscript></span></span></p></li></ul><p>where <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-50.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-51.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-52.gif"/></noscript></span></span>.</p><p id="p-44">We can then sample the <strong><em>S<sub>i</sub></em></strong> backward from <em>S<sub>iJ</sub></em> to <em>S<sub>i</sub></em><sub>1</sub> with<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-53.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-54.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-55.gif"/></noscript></span></span>The initial state <em>S<sub>i</sub></em><sub>0</sub> will be sampled with <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-56.gif"/></noscript></span></span>.</p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-45"><span class="list-label">(3) </span>Update the recombination count <strong><em>R</em></strong><em><sub>i</sub></em> = {<em>R<sub>ij</sub></em>}<sub>1×</sub><em><sub>J</sub></em> for <em>i</em> = 1, 2, …, <em>I</em>. <em>R<sub>ij</sub></em> is sampled with full conditional probability mass function<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-57.gif"/></noscript></span></span></p></li><li id="list-item-4"><p id="p-46"><span class="list-label">(4) </span>Update recombination probability <em>γ<sub>j</sub></em> from <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-58.gif"/></noscript></span></span> for <em>j</em> = 1. 2, …, <em>J</em>.</p></li><li id="list-item-5"><p id="p-47"><span class="list-label">(5) </span>Update the proportion ancestry from population A <em>ρ<sub>i</sub></em> from <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-59.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-60.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-61.gif"/></noscript></span></span>.</p></li><li id="list-item-6"><p id="p-48"><span class="list-label">(6) </span>Update <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-62.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-63.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-64.gif"/></noscript></span></span> and <strong><em>Q</em></strong><em><sub>i</sub></em><sub>0</sub> based on last <em>ρ<sub>i</sub></em> for <em>i</em> = 1, 2, …, <em>I</em>.</p></li><li id="list-item-7"><p id="p-49"><span class="list-label">(7) </span>Update <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-66.gif"/></noscript></span></span> for <em>j</em> = 1. 2, …, <em>J</em>. Let <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-68.gif"/></noscript></span></span> denotes the case that the allele from population A is variant allele when <em>S<sub>ij</sub></em> = 1 and <em>X<sub>ij</sub></em> = 1. <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-69.gif"/></noscript></span></span> is unobserved and can be imputed from <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-70.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-71.gif"/></noscript></span></span> is then sampled from <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-72.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-73.gif"/></noscript></span></span> is sampled from <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-74.gif"/></noscript></span></span></p></li><li id="list-item-8"><p id="p-50"><span class="list-label">(8) </span>Update <strong><em>P</em></strong><em><sub>j</sub></em> based on last <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-76.gif"/></noscript></span></span> for <em>j</em> = 1. 2, …, <em>J</em>.</p></li><li id="list-item-9"><p id="p-51"><span class="list-label">(9) </span>Update <em>τ<sup>A</sup></em> and <em>τ<sup>B</sup></em> using Random-Walk Metropolis-Hasting. For <em>τ<sup>A</sup></em>, we propose the new <em>τ<sup>A</sup></em>* = <em>τ<sup>A</sup></em> + <em>ϵ</em> where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-77.gif"/></noscript></span></span>. The posterior distribution of <em>τ<sup>A</sup></em>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-78.gif"/></noscript></span></span>. Then, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1165/embed/mml-math-79.gif"/></noscript></span></span>. We draw <em>μ<sup>A</sup></em> ∼ ∪ [0, 1]. If <em>μ<sup>A</sup></em> &lt; <em>α</em>(<em>τ<sup>A</sup></em>, <em>τ<sup>A</sup></em>*), then <em>τ<sup>A</sup></em> is replaced by <em>τ<sup>A</sup></em>*; otherwise, <em>τ<sup>A</sup></em> is unchanged. Similar update is conducted for <em>τ<sup>B</sup></em>.</p></li></ul></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006478/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006478/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-53"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-05-15"><span class="received-label">Received </span>May 15, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-05-01"><span class="accepted-label">Accepted </span>May 1, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Zhu <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.1" data-doi="10.1186/1471-2393-4-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohlsson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The effect of hypertensive disorders in pregnancy on small for gestational age and stillbirth: a population based study.</span> <abbr class="cit-jnl-abbrev">BMC Pregnancy Childbirth</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BPregnancy%2BChildbirth%26rft.aulast%253DAllen%26rft.auinit1%253DV.%2BM.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D17%26rft.atitle%253DThe%2Beffect%2Bof%2Bhypertensive%2Bdisorders%2Bin%2Bpregnancy%2Bon%2Bsmall%2Bfor%2Bgestational%2Bage%2Band%2Bstillbirth%253A%2Ba%2Bpopulation%2Bbased%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2393-4-17%26rft_id%253Dinfo%253Apmid%252F15298717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2393-4-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15298717&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.2" data-doi="10.1093/hmg/ddp122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">North</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curb</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Admixture mapping of quantitative trait loci for blood lipids in African-Americans.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2091</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp122%26rft_id%253Dinfo%253Apmid%252F19304782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8yMDkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-3.7.1165.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellman</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-source">Adaptive Control Processes: A Guided Tour</span>, <span class="cit-publ-name">Princeton University Press</span>, <span class="cit-publ-loc">Princeton, New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.4" data-doi="10.1371/journal.pgen.1000908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">B. E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Admixture mapping scans identify a locus affecting retinal vascular caliber in hypertensive African Americans: the atherosclerosis risk in communities (ARIC) study.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCheng%26rft.auinit1%253DC.%2BY.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000908%26rft.epage%253De1000908%26rft.atitle%253DAdmixture%2Bmapping%2Bscans%2Bidentify%2Ba%2Blocus%2Baffecting%2Bretinal%2Bvascular%2Bcaliber%2Bin%2Bhypertensive%2BAfrican%2BAmericans%253A%2Bthe%2BAtherosclerosis%2BRisk%2Bin%2BCommunities%2B%2528ARIC%2529%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000908%26rft_id%253Dinfo%253Apmid%252F20419148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20419148&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.6" data-doi="10.1073/pnas.0605832103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freedman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haiman</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Admixture mapping identifies 8q24 as a prostate cancer risk locus in African-American men.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">14068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605832103%26rft_id%253Dinfo%253Apmid%252F16945910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM4LzE0MDY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.7" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regularization paths for generalized linear models via coordinate descent.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.8" data-doi="10.1371/journal.pgen.1000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Practical issues in imputation-based association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000279%26rft.epage%253De1000279%26rft.atitle%253DPractical%2Bissues%2Bin%2Bimputation-based%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000279%26rft_id%253Dinfo%253Apmid%252F19057666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19057666&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.9" data-doi="10.1214/11-AOAS455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bayesian variable selection regression for genome-wide association studies and other large-scale problems.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1780</span>–<span class="cit-lpage">1815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat.%26rft.volume%253D5%26rft.spage%253D1780%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-AOAS455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/11-AOAS455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.10" data-doi="10.1086/375613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoggart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriver</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonilla</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittles</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Control of confounding of genetic associations in stratified populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHoggart%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D72%26rft.issue%253D6%26rft.spage%253D1492%26rft.epage%253D1504%26rft.atitle%253DControl%2Bof%2Bconfounding%2Bof%2Bgenetic%2Bassociations%2Bin%2Bstratified%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F375613%26rft_id%253Dinfo%253Apmid%252F12817591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/375613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12817591&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183275500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.11" data-doi="10.1111/j.1467-9868.2009.00730.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossell</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of non-local prior densities in Bayesian hypothesis tests.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D72%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2009.00730.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2009.00730.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.12" data-doi="10.1038/ng.232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">W. H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klag</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meoni</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berthier-Schaad</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MYH9 is associated with nondiabetic end-stage renal disease in African Americans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKao%26rft.auinit1%253DW.%2BH.%26rft.volume%253D40%26rft.issue%253D10%26rft.spage%253D1185%26rft.epage%253D1192%26rft.atitle%253DMYH9%2Bis%2Bassociated%2Bwith%2Bnondiabetic%2Bend-stage%2Brenal%2Bdisease%2Bin%2BAfrican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.232%26rft_id%253Dinfo%253Apmid%252F18794854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18794854&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259651000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-3.7.1165.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelder</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Generalized Linear Models</span>, <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.14" data-doi="10.1086/301908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKeigue</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mapping genes that underlie ethnic differences in disease risk: methods for detecting linkage in admixed populations, by conditioning on parental admixture.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMcKeigue%26rft.auinit1%253DP.%2BM.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D241%26rft.epage%253D251%26rft.atitle%253DMapping%2Bgenes%2Bthat%2Bunderlie%2Bethnic%2Bdifferences%2Bin%2Bdisease%2Brisk%253A%2Bmethods%2Bfor%2Bdetecting%2Blinkage%2Bin%2Badmixed%2Bpopulations%252C%2Bby%2Bconditioning%2Bon%2Bparental%2Badmixture.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F301908%26rft_id%253Dinfo%253Apmid%252F9634509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/301908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9634509&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074761000032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.15" data-doi="10.1086/426949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKeigue</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Prospects for admixture mapping of complex traits.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMcKeigue%26rft.auinit1%253DP.%2BM.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DProspects%2Bfor%2Badmixture%2Bmapping%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F426949%26rft_id%253Dinfo%253Apmid%252F15540159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/426949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15540159&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225643600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.16" data-doi="10.1093/epirev/mxp011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Environmental contributions to disparities in pregnancy outcomes.</span> <abbr class="cit-jnl-abbrev">Epidemiol. Rev.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpidemiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fepirev%252Fmxp011%26rft_id%253Dinfo%253Apmid%252F19846592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZXBpcmV2IjtzOjU6InJlc2lkIjtzOjc6IjMxLzEvNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.17" data-doi="10.1086/425281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Statistical tests for admixture mapping with case-control and cases-only data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMontana%26rft.auinit1%253DG.%26rft.volume%253D75%26rft.issue%253D5%26rft.spage%253D771%26rft.epage%253D789%26rft.atitle%253DStatistical%2Btests%2Bfor%2Badmixture%2Bmapping%2Bwith%2Bcase-control%2Band%2Bcases-only%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F425281%26rft_id%253Dinfo%253Apmid%252F15386213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/425281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15386213&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224303500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.18" data-doi="10.1126/science.1117196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A fine-scale map of recombination rates and hotspots across the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DS.%26rft.volume%253D310%26rft.issue%253D5746%26rft.spage%253D321%26rft.epage%253D324%26rft.atitle%253DA%2BFine-Scale%2BMap%2Bof%2BRecombination%2BRates%2Band%2BHotspots%2BAcross%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1117196%26rft_id%253Dinfo%253Apmid%252F16224025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0Ni8zMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.19" data-doi="10.1086/420871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattangadi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Methods for high-density admixture mapping of disease genes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D74%26rft.issue%253D5%26rft.spage%253D979%26rft.epage%253D1000%26rft.atitle%253DMethods%2Bfor%2Bhigh-density%2Badmixture%2Bmapping%2Bof%2Bdisease%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F420871%26rft_id%253Dinfo%253Apmid%252F15088269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/420871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15088269&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220926100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.20" data-doi="10.1371/journal.pgen.1000519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafaels</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000519%26rft.epage%253De1000519%26rft.atitle%253DSensitive%2Bdetection%2Bof%2Bchromosomal%2Bsegments%2Bof%2Bdistinct%2Bancestry%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000519%26rft_id%253Dinfo%253Apmid%252F19543370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543370&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.21" data-doi="10.1038/ng1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jager</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waliszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A whole-genome admixture scan finds a candidate locus for multiple sclerosis susceptibility.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DReich%26rft.auinit1%253DD.%26rft.volume%253D37%26rft.issue%253D10%26rft.spage%253D1113%26rft.epage%253D1118%26rft.atitle%253DA%2Bwhole-genome%2Badmixture%2Bscan%2Bfinds%2Ba%2Bcandidate%2Blocus%2Bfor%2Bmultiple%2Bsclerosis%2Bsusceptibility.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1646%26rft_id%253Dinfo%253Apmid%252F16186815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16186815&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232185600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.22" data-doi="10.1086/513206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramesh</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jager</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Admixture mapping of an allele affecting interleukin 6 soluble receptor and interleukin 6 levels.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DReich%26rft.auinit1%253DD.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D716%26rft.epage%253D726%26rft.atitle%253DAdmixture%2Bmapping%2Bof%2Ban%2Ballele%2Baffecting%2Binterleukin%2B6%2Bsoluble%2Breceptor%2Band%2Binterleukin%2B6%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F513206%26rft_id%253Dinfo%253Apmid%252F17357077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/513206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357077&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245465200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.23" data-doi="10.1136/jmg.2008.064808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nalls</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawlikowska</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Admixture mapping of ankle–arm index: identification of a candidate locus associated with peripheral arterial disease.</span> <abbr class="cit-jnl-abbrev">J. Med. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fjmg.2008.064808%26rft_id%253Dinfo%253Apmid%252F19586928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiam1lZGdlbmV0IjtzOjU6InJlc2lkIjtzOjY6IjQ3LzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.24" data-doi="10.1198/016214502753479464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bayesian methods for hidden Markov models.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D97%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214502753479464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214502753479464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173997500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.25" data-doi="10.1038/nrg1657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mapping by admixture linkage disequilibrium: advances, limitations and guidelines.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSmith%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D623%26rft.epage%253D632%26rft.atitle%253DMapping%2Bby%2Badmixture%2Blinkage%2Bdisequilibrium%253A%2Badvances%252C%2Blimitations%2Band%2Bguidelines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1657%26rft_id%253Dinfo%253Apmid%252F16012528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16012528&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230880700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.26" data-doi="10.1086/420856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lautenberger</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truelove</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A high-density admixture map for disease gene discovery in African Americans.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSmith%26rft.auinit1%253DM.%2BW.%26rft.volume%253D74%26rft.issue%253D5%26rft.spage%253D1001%26rft.epage%253D1013%26rft.atitle%253DA%2Bhigh-density%2Badmixture%2Bmap%2Bfor%2Bdisease%2Bgene%2Bdiscovery%2Bin%2Bafrican%2Bamericans.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F420856%26rft_id%253Dinfo%253Apmid%252F15088270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/420856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15088270&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220926100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.27" data-doi="10.1002/gepi.20550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ancestry informative marker panels for African Americans based on subsets of commercially available SNP arrays.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DTandon%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D80%26rft.epage%253D83%26rft.atitle%253DAncestry%2Binformative%2Bmarker%2Bpanels%2Bfor%2BAfrican%2BAmericans%2Bbased%2Bon%2Bsubsets%2Bof%2Bcommercially%2Bavailable%2BSNP%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20550%26rft_id%253Dinfo%253Apmid%252F21181899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21181899&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.28" data-doi="10.1038/nature04226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A haplotype map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1299%26rft.epage%253D1320%26rft.atitle%253DA%2Bhaplotype%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04226%26rft_id%253Dinfo%253Apmid%252F16255080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16255080&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D73%26rft.spage%253D273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.30" data-doi="10.1146/annurev-genom-082509-141523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Admixture mapping comes of age.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DWinkler%26rft.auinit1%253DC.%2BA.%26rft.volume%253D11%26rft.spage%253D65%26rft.epage%253D89%26rft.atitle%253DAdmixture%2Bmapping%2Bcomes%2Bof%2Bage.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082509-141523%26rft_id%253Dinfo%253Apmid%252F20594047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082509-141523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20594047&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282662200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.31" data-doi="10.1093/bioinformatics/btp041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide association analysis by lasso penalized logistic regression.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp041%26rft_id%253Dinfo%253Apmid%252F19176549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNS82LzcxNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.32" data-doi="10.1038/ng.763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devidas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D237%26rft.epage%253D241%26rft.atitle%253DAncestry%2Band%2Bpharmacogenomics%2Bof%2Brelapse%2Bin%2Bacute%2Blymphoblastic%2Bleukemia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.763%26rft_id%253Dinfo%253Apmid%252F21297632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21297632&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.33" data-doi="10.1086/421329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elston</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage analysis of a complex disease through use of admixed populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1136</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DX.%26rft.volume%253D74%26rft.issue%253D6%26rft.spage%253D1136%26rft.epage%253D1153%26rft.atitle%253DLinkage%2Banalysis%2Bof%2Ba%2Bcomplex%2Bdisease%2Bthrough%2Buse%2Bof%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F421329%26rft_id%253Dinfo%253Apmid%252F15131754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/421329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15131754&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221651900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.34" data-doi="10.1038/ng1510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quertermous</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Admixture mapping for hypertension loci with genome-scan markers.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DX.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D177%26rft.epage%253D181%26rft.atitle%253DAdmixture%2Bmapping%2Bfor%2Bhypertension%2Bloci%2Bwith%2Bgenome-scan%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1510%26rft_id%253Dinfo%253Apmid%252F15665825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15665825&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226690100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.35" data-doi="10.1093/hmg/ddr113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keating</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franceschini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Combined admixture mapping and association analysis identifies a novel blood pressure genetic locus on 5p13: contributions from the care consortium.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2285</span>–<span class="cit-lpage">2295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddr113%26rft_id%253Dinfo%253Apmid%252F21422096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMS8yMjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1165.36" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Regularization and variable selection via the elastic net.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D67%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/7/1165.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1151" title="Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1177" title="Spatial Profiling of Nuclear Receptor Transcription Patterns over the Course of Drosophila Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/7"><span>Volume 3 Issue 7, July 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/7.cover-source.jpg?itok=DKXH6P5c" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1165" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DHGyhu2stvRVR-H11ZitEJvEktFJ4cjarPNcziG1pVQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/7/1165" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/7/1165" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/7/1165" class="active" data-icon-position="" data-hide-link-title="0">Generalized Admixture Mapping for Complex Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459491" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-60luZ5dcfW9AB1O61E37kGrZEta8jsjDrALJIt94Xys" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1165.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/7/1165" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jlbWSx5vIKZCJj8kQg5KkDE_V1ARuvp0yYoPM-uuulY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/7/1165" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1165.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459491" id="node459491--2" data-pisa="ggg;3/7/1165" data-pisa-master="ggg;g3.113.006478" data-apath="/ggg/3/7/1165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Generalized Admixture Mapping for Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bin Zhu</span>, <span class="highwire-citation-author" data-delta="1">Allison E. Ashley-Koch</span> and <span class="highwire-citation-author" data-delta="2">David B. Dunson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1165-1175;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006478 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459491--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, Maryland 20850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bin.zhu@nih.gov" class="" data-icon-position="" data-hide-link-title="0">bin.zhu@nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Allison E. Ashley-Koch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Human Genetics, Duke University Medical Center, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allison%2BE.%2BAshley-Koch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashley-Koch%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllison%2BE.%2BAshley-Koch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David B. Dunson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistical Science, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BDunson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunson%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BDunson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/7/1165" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/7/1165&amp;title=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/7/1165&amp;title=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/7/1165&amp;title=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/7/1165&amp;text=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/7/1165&amp;title=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/7/1165&amp;t=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/7/1165&amp;title=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/7/1165&amp;title=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459491/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459491" id="node459491--3" data-pisa="ggg;3/7/1165" data-pisa-master="ggg;g3.113.006478" data-apath="/ggg/3/7/1165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Generalized Admixture Mapping for Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bin Zhu</span>, <span class="highwire-citation-author" data-delta="1">Allison E. Ashley-Koch</span> and <span class="highwire-citation-author" data-delta="2">David B. Dunson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1165-1175;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006478 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459491--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bin Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Rockville, Maryland 20850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bin%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABin%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bin.zhu@nih.gov" class="" data-icon-position="" data-hide-link-title="0">bin.zhu@nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Allison E. Ashley-Koch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Human Genetics, Duke University Medical Center, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allison%2BE.%2BAshley-Koch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashley-Koch%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllison%2BE.%2BAshley-Koch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David B. Dunson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistical Science, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BDunson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunson%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BDunson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1165&amp;rft.epage=1175&amp;rft.atitle=Generalized%20Admixture%20Mapping%20for%20Complex%20Traits&amp;rft.volume=3&amp;rft.issue=7&amp;rft.date=2013-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhu%2C+Bin&amp;rft.au=Ashley-Koch%2C+Allison+E.&amp;rft.au=Dunson%2C+David+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459491/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459491/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459491/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459491/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459491/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459491/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459491/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459491/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459491/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459491/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459491/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459491/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/7/1165" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendices</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/7/1165.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/7/1165.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/7/1165.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/7/1165.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
