<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/7/1151" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/7/1151.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/7/1151.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/7/1151.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/7/1151.atom" />
<meta name="HW.pisa" content="ggg;3/7/1151" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura" />
<meta name="DC.Identifier" content="10.1534/g3.113.006122" />
<meta name="DC.Date" content="2013-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 by the Genetics Society of America. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Paracentric inversions in populations can have a profound effect on the pattern and organization of nucleotide variability along a chromosome. Regions near inversion breakpoints are expected to have greater levels of differentiation because of reduced genetic exchange between different gene arrangements whereas central regions in the inverted segments are predicted to have lower levels of nucleotide differentiation due to greater levels of genetic flux among different karyotypes. We used the inversion polymorphism on the third chromosome of Drosophila pseudoobscura to test these predictions with an analysis of nucleotide diversity of 18 genetic markers near and away from inversion breakpoints. We tested hypotheses about how the presence of different chromosomal arrangements affects the pattern and organization of nucleotide variation. Overall, markers in the distal segment of the chromosome had greater levels of nucleotide heterozygosity than markers within the proximal segment of the chromosome. In addition, our results rejected the hypothesis that the breakpoints of derived inversions will have lower levels of nucleotide variability than breakpoints of ancestral inversions, even when strains with gene conversion events were removed. High levels of linkage disequilibrium were observed within all 11 breakpoint regions as well as between the ends of most proximal and distal breakpoints. The central region of the chromosome had the greatest levels of linkage disequilibrium compared with the proximal and distal regions because this is the region that experiences the highest level of recombination suppression. These data do not fully support the idea that genetic exchange is the sole force that influences genetic variation on inverted chromosomes." />
<meta name="DC.Contributor" content="Andre G. Wallace" />
<meta name="DC.Contributor" content="Don Detweiler" />
<meta name="DC.Contributor" content="Stephen W. Schaeffer" />
<meta name="article:published_time" content="2013-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Paracentric inversions in populations can have a profound effect on the pattern and organization of nucleotide variability along a chromosome. Regions near inversion breakpoints are expected to have greater levels of differentiation because of reduced genetic exchange between different gene arrangements whereas central regions in the inverted segments are predicted to have lower levels of nucleotide differentiation due to greater levels of genetic flux among different karyotypes. We used the inversion polymorphism on the third chromosome of &lt;i&gt;Drosophila pseudoobscura&lt;/i&gt; to test these predictions with an analysis of nucleotide diversity of 18 genetic markers near and away from inversion breakpoints. We tested hypotheses about how the presence of different chromosomal arrangements affects the pattern and organization of nucleotide variation. Overall, markers in the distal segment of the chromosome had greater levels of nucleotide heterozygosity than markers within the proximal segment of the chromosome. In addition, our results rejected the hypothesis that the breakpoints of derived inversions will have lower levels of nucleotide variability than breakpoints of ancestral inversions, even when strains with gene conversion events were removed. High levels of linkage disequilibrium were observed within all 11 breakpoint regions as well as between the ends of most proximal and distal breakpoints. The central region of the chromosome had the greatest levels of linkage disequilibrium compared with the proximal and distal regions because this is the region that experiences the highest level of recombination suppression. These data do not fully support the idea that genetic exchange is the sole force that influences genetic variation on inverted chromosomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/07/01" />
<meta name="citation_mjid" content="ggg;3/7/1151" />
<meta name="citation_id" content="3/7/1151" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/7/1151" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/7/1151.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/7/1151.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/7/1151.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006122" />
<meta name="citation_pmid" content="23665879" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1151" />
<meta name="citation_lastpage" content="1163" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andre G. Wallace" />
<meta name="citation_author_institution" content="Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301" />
<meta name="citation_author" content="Don Detweiler" />
<meta name="citation_author_institution" content="Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301" />
<meta name="citation_author" content="Stephen W. Schaeffer" />
<meta name="citation_author_institution" content="Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301" />
<meta name="citation_author_email" content="sws4@psu.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. S. Babcock;citation_author=W. W. Anderson;citation_title=Molecular evolution of the Sex-Ratio inversion complex in Drosophila pseudoobscura: analysis of the Esterase-5 gene region;citation_pages=297-308;citation_volume=13;citation_year=1996;citation_issue=2;citation_pmid=8587496;citation_doi=10.1093/oxfordjournals.molbev.a025589" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Begun;citation_author=C. F. Aquadro;citation_title=Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.;citation_pages=519-520;citation_volume=356;citation_year=1992;citation_issue=6369;citation_pmid=1560824;citation_doi=10.1038/356519a0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Betran;citation_author=J. Rozas;citation_author=A. Navarro;citation_author=A. Barbadilla;citation_title=The Estimation of the Number and the Length Distribution of Gene Conversion Tracts From Population DNA Sequence Data;citation_pages=89-99;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Chovnick;citation_title=GENE CONVERSION AND TRANSFER OF GENETIC INFORMATION WITHIN THE INVERTED REGION OF INVERSION HETEROZYGOTES;citation_pages=123-131;citation_volume=75;citation_year=1973;citation_issue=1;citation_pmid=4202769" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_title=GENETICS OF NATURAL POPULATIONS IX. TEMPORAL CHANGES IN THE COMPOSITION OF POPULATIONS OF DROSOPHILA PSEUDOOBSCURA;citation_pages=162-186;citation_volume=28;citation_year=1943;citation_issue=2;citation_pmid=17247077" />
<meta name="citation_reference" content="citation_journal_title=Carnegie Inst. Washington Publ.;citation_author=T. Dobzhansky;citation_title=Chromosomal races in Drosophila pseudoobscura and Drosophila persimilis.;citation_pages=47-144;citation_volume=554;citation_year=1944" />
<meta name="citation_reference" content="citation_title=Proceedings of the International Congress of Genetics;citation_title=Observations and experiments on natural selection in Drosophila;citation_author=T. Dobzhansky;citation_pages=210-224;citation_year=1949" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_title=GENETICS OF NATURAL POPULATIONS. XIX. ORIGIN OF HETEROSIS THROUGH NATURAL SELECTION IN POPULATIONS OF DROSOPHILA PSEUDOOBSCURA;citation_pages=288-302;citation_volume=35;citation_year=1950;citation_issue=3;citation_pmid=15414931" />
<meta name="citation_reference" content="citation_title=Contributions to the Genetics, Taxonomy, and Ecology of Drosophila pseudoobscura and its Relatives;citation_title=Taxonomy, geographic distribution, and ecology of Drosophila pseudoobscura and its relatives;citation_author=T. Dobzhansky;citation_author=C. Epling;citation_pages=1-46;citation_year=1944" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_author=A. H. Sturtevant;citation_title=INVERSIONS IN THE CHROMOSOMES OF DROSOPHILA PSEUDOOBSCURA;citation_pages=28-64;citation_volume=23;citation_year=1938;citation_issue=1;citation_pmid=17246876" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_author=M. L. Queal;citation_title=GENETICS OF NATURAL POPULATIONS. II. GENIC VARIATION IN POPULATIONS OF DROSOPHILA PSEUDOOBSCURA INHABITING ISOLATED MOUNTAIN RANGES;citation_pages=463-484;citation_volume=23;citation_year=1938;citation_issue=5;citation_pmid=17246895" />
<meta name="citation_reference" content="citation_journal_title=Drosoph. Inf. Serv.;citation_author=G. Gloor;citation_author=W. Engels;citation_title=Single-fly DNA preps for PCR.;citation_pages=148-149;citation_volume=71;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. T. Hamblin;citation_author=C. F. Aquadro;citation_title=DNA Sequence Variation and the Recombinational Landscape in Drosophila pseudoobscura: A Study of the Second Chromosome;citation_pages=859-869;citation_volume=153;citation_year=1999;citation_issue=2;citation_pmid=10511563" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. J. Hilliker;citation_author=G. Harauz;citation_author=A. G. Reaume;citation_author=M. Gray;citation_author=S. H. Clark;citation_title=Meiotic gene conversion tract length distribution within the rosy locus of Drosophila melanogaster.;citation_pages=1019-1026;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982556" />
<meta name="citation_reference" content="citation_journal_title=Oxford Surveys in Evolutionary Biology;citation_author=R. R. Hudson;citation_title=OXFORD SURVEYS IN EVOLUTIONARY BIOLOGY;citation_pages=1-44;citation_volume=7;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_title=ON THE DIVERGENCE OF ALLELES IN NESTED SUBSAMPLES FROM FINITE POPULATIONS;citation_pages=1057-1076;citation_volume=113;citation_year=1986;citation_issue=4;citation_pmid=3744026" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=M. Kreitman;citation_author=M. AguadÃ©;citation_title=A Test of Neutral Molecular Evolution Based on Nucleotide Data;citation_pages=153-159;citation_volume=116;citation_year=1987;citation_issue=1;citation_pmid=3110004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Innan;citation_author=W. Stephan;citation_title=The Coalescent in an Exponentially Growing Metapopulation and Its Application to Arabidopsis thaliana;citation_pages=2015-2019;citation_volume=155;citation_year=2000;citation_issue=4;citation_pmid=11032470" />
<meta name="citation_reference" content="citation_title=Mammalian Protein Metabolism;citation_title=Evolution of protein molecules;citation_author=T. H. Jukes;citation_author=C. R. Cantor;citation_pages=21-132;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=M. Kimura;citation_title=DNA and the Neutral Theory;citation_pages=343-354;citation_volume=312;citation_year=1986;citation_issue=1154;citation_pmid=2870526;citation_doi=10.1098/rstb.1986.0012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Kirkpatrick;citation_title=How and why chromosome inversions evolve.;citation_pages=e1000501;citation_volume=8;citation_year=2010;citation_pmid=20927412;citation_doi=10.1371/journal.pbio.1000501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_title=The detection of linkage disequilibrium in molecular sequence data.;citation_pages=377-388;citation_volume=140;citation_year=1995;citation_issue=1;citation_pmid=7635301" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_title=The New Systematics;citation_title=Bearings of the âDrosophilaâ work on systematics;citation_author=H. J. Muller;citation_pages=185-268;citation_year=1940" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Navarro;citation_author=A. Barbadilla;citation_author=A. Ruiz;citation_title=Effect of Inversion Polymorphism on the Neutral Nucleotide Variability of Linked Chromosomal Regions in Drosophila;citation_pages=685-698;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835391" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Navarro;citation_author=E. BetrÃ¡n;citation_author=A. Barbadilla;citation_author=A. Ruiz;citation_title=Recombination and Gene Flux Caused by Gene Conversion and Crossing Over in Inversion Heterokaryotypes;citation_pages=695-709;citation_volume=146;citation_year=1997;citation_issue=2;citation_pmid=9178017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Noor;citation_author=D. A. Garfield;citation_author=S. W. Schaeffer;citation_author=C. A. Machado;citation_title=Divergence Between the Drosophila pseudoobscura and D. persimilis Genome Sequences in Relation to Chromosomal Inversions;citation_pages=1417-1428;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=18039875;citation_doi=10.1534/genetics.107.070672" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. S. Painter;citation_title=A NEW METHOD FOR THE STUDY OF CHROMOSOME ABERRATIONS AND THE PLOTTING OF CHROMOSOME MAPS IN DROSOPHILA MELANOGASTER;citation_pages=175-188;citation_volume=19;citation_year=1934;citation_issue=3;citation_pmid=17246718" />
<meta name="citation_reference" content="citation_title=Drosophila Inversion Polymorphism;citation_title=Inversion polymorphisms in Drosophila pseudoobscura and Drosophila persimilis;citation_author=J. R. Powell;citation_pages=73-126;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Richards;citation_author=Y. Liu;citation_author=B. R. Bettencourt;citation_author=P. Hradecky;citation_author=S. Letovsky;citation_title=Comparative genome sequencing of Drosophila pseudoobscura: Chromosomal, gene, and cis-element evolution;citation_pages=1-18;citation_volume=15;citation_year=2005;citation_issue=1;citation_pmid=15632085;citation_doi=10.1101/gr.3059305" />
<meta name="citation_reference" content="citation_title=The Genetics and Biology of Drosophila;citation_author=P. A. Roberts;citation_title=THE GENETICS OF CHROMOSOME ABERRATION;citation_pages=67-184;citation_volume=1a;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Rozas;citation_author=M. AguadÃ©;citation_title=Evidence of extensive genetic exchange in the rp49 region among polymorphic chromosome inversions in Drosophila subobscura.;citation_pages=417-426;citation_volume=126;citation_year=1990;citation_issue=2;citation_pmid=2123162" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Rozas;citation_author=M. AguadÃ©;citation_title=Transfer of genetic information in the rp49 region of Drosophila subobscura between different chromosomal gene arrangements.;citation_pages=8083-8087;citation_volume=90;citation_year=1993;citation_issue=17;citation_pmid=8367467;citation_doi=10.1073/pnas.90.17.8083" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Rozas;citation_author=M. AguadÃ©;citation_title=Gene conversion is involved in the transfer of genetic information between naturally occurring inversions of Drosophila.;citation_pages=11517-11521;citation_volume=91;citation_year=1994;citation_issue=24;citation_pmid=7972094;citation_doi=10.1073/pnas.91.24.11517" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=S. W. Schaeffer;citation_title=Molecular population genetics of sequence length diversity in the Adh region of Drosophila pseudoobscura.;citation_pages=163-175;citation_volume=80;citation_year=2002;citation_issue=3;citation_pmid=12688655;citation_doi=10.1017/S0016672302005955" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. W. Schaeffer;citation_title=Selection in heterogeneous environments maintains the gene arrangement polymorphism of Drosophila pseudoobscura.;citation_pages=3082-3099;citation_volume=62;citation_year=2008;citation_issue=12;citation_pmid=18764919;citation_doi=10.1111/j.1558-5646.2008.00504.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. W. Schaeffer;citation_author=W. W. Anderson;citation_title=Mechanisms of genetic exchange within the chromosomal inversions of Drosophila pseudoobscura.;citation_volume=171;citation_year=2005;citation_pmid=16143625;citation_doi=10.1534/genetics.105.041947" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. W. Schaeffer;citation_author=P. Goetting-Minesky;citation_author=M. Kovacevic;citation_author=J. Peoples;citation_author=J. L. Graybill;citation_title=Evolutionary genomics of inversions in Drosophila pseudoobscura: Evidence for epistasis.;citation_volume=100;citation_year=2003;citation_pmid=12824467;citation_doi=10.1073/pnas.1432900100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. W. Schaeffer;citation_author=A. Bhutkar;citation_author=B. F. McAllister;citation_author=M. Matsuda;citation_author=L. M. Matzkin;citation_title=Polytene chromosomal maps of 11 Drosophila species: The order of genomic scaffolds inferred from genetic and physical maps.;citation_volume=179;citation_year=2008;citation_pmid=18622037;citation_doi=10.1534/genetics.107.086074" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Slatkin;citation_author=R. R. Hudson;citation_title=Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.;citation_pages=555-562;citation_volume=129;citation_year=1991;citation_issue=2;citation_pmid=1743491" />
<meta name="citation_reference" content="citation_title=Biometry;citation_author=R. R. Sokal;citation_author=F. J. Rohlf;citation_year=1981" />
<meta name="citation_reference" content="citation_title=The Genetics and Biology of Drosophila 3e;citation_title=Chromosomal polymorphism in natural and experimental populations;citation_author=D. Sperlich;citation_author=P. Pfriem;citation_pages=257-309;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=J. D. Storey;citation_title=A direct approach to false discovery rates.;citation_pages=479-498;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00346" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Sturtevant;citation_title=Genetic Factors Affecting the Strength of Linkage in Drosophila;citation_pages=555-558;citation_volume=3;citation_year=1917;citation_issue=9;citation_pmid=16586749;citation_doi=10.1073/pnas.3.9.555" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. H. Sturtevant;citation_author=G. W. Beadle;citation_title=The relations of inversions in the X chromosome of Drosophila melanogaster to crossing over and disjunction.;citation_pages=544-604;citation_volume=21;citation_year=1936" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Toomajian;citation_author=R. S. Ajioka;citation_author=L. B. Jorde;citation_author=J. P. Kushner;citation_author=M. Kreitman;citation_title=A Method for Detecting Recent Selection in the Human Genome From Allele Age Estimates;citation_pages=287-297;citation_volume=165;citation_year=2003;citation_issue=1;citation_pmid=14504236" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. G. Wallace;citation_author=D. Detweiler;citation_author=S. W. Schaeffer;citation_title=Evolutionary history of the third chromosome gene arrangements of Drosophila pseudoobscura inferred from inversion breakpoints.;citation_volume=28;citation_year=2011;citation_pmid=21339510;citation_doi=10.1093/molbev/msr039" />
<meta name="citation_reference" content="citation_title=The Genetics and Biology of Drosophila 3b;citation_title=Evolution of the repleta group;citation_author=M. Wasserman;citation_pages=61-139;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_author=T. Dobzhansky;citation_title=GENETICS OF NATURAL POPULATIONS. XII. EXPERIMENTAL REPRODUCTION OF SOME OF THE CHANGES CAUSED BY NATURAL SELECTION IN CERTAIN POPULATIONS OF DROSOPHILA PSEUDOOBSCURA;citation_pages=125-156;citation_volume=31;citation_year=1946;citation_issue=2;citation_pmid=17247189" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="twitter:description" content="Paracentric inversions in populations can have a profound effect on the pattern and organization of nucleotide variability along a chromosome. Regions near inversion breakpoints are expected to have greater levels of differentiation because of reduced genetic exchange between different gene arrangements whereas central regions in the inverted segments are predicted to have lower levels of nucleotide differentiation due to greater levels of genetic flux among different karyotypes. We used the inversion polymorphism on the third chromosome of Drosophila pseudoobscura to test these predictions with an analysis of nucleotide diversity of 18 genetic markers near and away from inversion breakpoints. We tested hypotheses about how the presence of different chromosomal arrangements affects the pattern and organization of nucleotide variation. Overall, markers in the distal segment of the chromosome had greater levels of nucleotide heterozygosity than markers within the proximal segment of the chromosome. In addition, our results rejected the hypothesis that the breakpoints of derived inversions will have lower levels of nucleotide variability than breakpoints of ancestral inversions, even when strains with gene conversion events were removed. High levels of linkage disequilibrium were observed within all 11 breakpoint regions as well as between the ends of most proximal and distal breakpoints. The central region of the chromosome had the greatest levels of linkage disequilibrium compared with the proximal and distal regions because this is the region that experiences the highest level of recombination suppression. These data do not fully support the idea that genetic exchange is the sole force that influences genetic variation on inverted chromosomes." />
<meta name="og-title" property="og:title" content="Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/7/1151" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Paracentric inversions in populations can have a profound effect on the pattern and organization of nucleotide variability along a chromosome. Regions near inversion breakpoints are expected to have greater levels of differentiation because of reduced genetic exchange between different gene arrangements whereas central regions in the inverted segments are predicted to have lower levels of nucleotide differentiation due to greater levels of genetic flux among different karyotypes. We used the inversion polymorphism on the third chromosome of Drosophila pseudoobscura to test these predictions with an analysis of nucleotide diversity of 18 genetic markers near and away from inversion breakpoints. We tested hypotheses about how the presence of different chromosomal arrangements affects the pattern and organization of nucleotide variation. Overall, markers in the distal segment of the chromosome had greater levels of nucleotide heterozygosity than markers within the proximal segment of the chromosome. In addition, our results rejected the hypothesis that the breakpoints of derived inversions will have lower levels of nucleotide variability than breakpoints of ancestral inversions, even when strains with gene conversion events were removed. High levels of linkage disequilibrium were observed within all 11 breakpoint regions as well as between the ends of most proximal and distal breakpoints. The central region of the chromosome had the greatest levels of linkage disequilibrium compared with the proximal and distal regions because this is the region that experiences the highest level of recombination suppression. These data do not fully support the idea that genetic exchange is the sole force that influences genetic variation on inverted chromosomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
    <title>Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_m6bpJN0zntXz2JOd9d7YFIIxulGveuOr-HnrBW2eU4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459492","apath":"\/ggg\/3\/7\/1151.atom","pisa":"ggg;3\/7\/1151","ac":{"\/ggg\/3\/7\/1151.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/7\/1151.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1151"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1151"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/7\/1151":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hbr7DPWUuekFGBWmGcSUYdY\\\\\\\/P62bbOlKyT8Jcqzbp4TDCjBdBQDFsS+jh1dvBzsgmgZOjaWnY5Ue91DFr\\\\\\\/MWcWZUbmD0aiaTcQCZKaBclOPw9SijhAX4VAuWzMVLRcWelWYmIvwTgEck2cg8eXQfCscqV1rUXcdLKc1PGT+o4u+7L6G2LwVLTvGoIE8D0AkuLvMH13vhQNKPoOAswT4Gn+S7m0r\\\\\\\/PxSTQQzkq6qbDhwOqSR9KY\\\\\\\/qNR\\\\\\\/OU4Z1dnpjzcUjIGBD3ORxsqKSXFr34SzB55\\\\\\\/rISLzaNZ1IEnvGCPxV8EjnAeLWjjuPsWp6Z65PMRq+lqWZQI4vLJ6anHkhBmdHzOUuDO\\\\\\\/r\\\\\\\/tOPZzTC9bWKzZJIm39wpKHTuePoo6hWPs4\\\\\\\/GAojMSlBW+4lx4GQDnL7Pz0zLSkGEzPaMRWkiatTD4vspO6Ckm\\\\\\\/nfTUJuXe9C9g2HlwAMIhUYX37W6F5nJqExZblymWRQP6hZ\\\\\\\/oI7zcszpISKr1PQ5+zK7bbXuIx7KDYf2B8fIO9VnmWs3h1jf\\\\\\\/eP9PAE\\\\\\\/uvZKTYIeAIDyNnopVefMBj5ZdXFUaPZrZ5\\\\\\\/rlb4A5k413D\\\\\\\/8GU4LZdH686NHkR052c332IfdrKz4ndau3kO\\\\\\\/oRuX\\\\\\\/7iM3vWEKBvDdY1\\\\\\\/xy+nUPyP8c2aieTz336+11bS5j4wT0bukhlDNcbdvtfo8w+6gq\\\\\\\/VCpLec+TAJuxk8emg9l5LJbXq69rUNi0N4LieYz0\\\\\\\/7VHvFmUDJEYVnGqCrwDnwpq9bACo6nIC0+xaSX\\\\\\\/FJZx0sRUgYVmQvB7uMp+X7eDirYFjlcVd1LEN2CoLegK5mEwYK9cPcz+IXxCpwKOVQkPc11UK\\\\\\\/ufq6HrGqZs+ujwWNmRCegfUKq65BzOPYaY\\\\\\\/1yE\\\\\\\/YI9YM+FmZjvqf5sPbsVHsBOoE\\\\\\\/qMJoXwxWUSX0Z7O4MDgF8TOvq9gNAgnwHdmYO1JoljS12heTyNN0lQDE1bE2amRvDC5kzE0dk6LoyfFF5QC0Zzc1bohY5pXo9DC\\\\\\\/XWGkQqroO\\\\\\\/imc+hg1zPVYXR73BxPMEcSlz\\\\\\\/qYPs6Gpszl21i6L6AL7KLjhvqzNUf+xKPGf2Q1q1P03NfXpyLyh2RdhSCrFGBIF0dX9bnrPHIDvv0zTe9SZSlWUKIOZ\\\\\\\/nDcKXoc5SrXbv8EZhzdOWA6aXD5N125o9smcVA51AUwDHDknQodqY4SotekbHXm8xnzao91amh\\\\\\\/bR5szdpsS05z3AGLULGog+RJ2yFFrrmzTC7ZQJ\\\\\\\/ZYVc1j4PVHVlAeQlHox2ih\\\u0022,\\\u0022iv\\\u0022:\\\u0022HgL4c0Qf01G6Jw6O1pzWMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223370a11ed79b9cdb602d534ed991578fe5615be5a615a556e310048369756732\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IVvkobR\\\\\\\/dp6IrAOHeutHRaUGK909BoVDX4tZ5I+ZErvEa9rwedffRa+4KZfYqrrpOtkgVtVUqqg0yIwPIK2uEn7WUhWDTruObMbdd\\\\\\\/Ryz9YMHYoNA90APB3fY\\\\\\\/y0jom9m6zF+0t+xj4smbYVI76bavmgfXgHO8\\\\\\\/bK9fz7TekujeXghMkaHAj\\\\\\\/2UbhqHImLNyZ4UkE35svb08yjTGsDj2itLwYD9JwVhI2IbuoPi8QTr7a3nEP6up7X7DZON0EdzYMOXWYUZpvKpWaSo\\\\\\\/57m34nm533wUdc5S1q+CteYnbgpmxSnkYkgECbRTQkYToeR\\\\\\\/QRx07SBW+3AKSaq8vl2dHUARy6dV8h60CWXy0YcazEnP3vEChZI4XJzkRhM\\\\\\\/ap25AzRq6tE0uoyXEi5XOuR8Npr7aJB1kNX3F\\\\\\\/LNlVzeG08QM0YuqSqbfPkLesfoPpwf\\\\\\\/0OioeUf\\\\\\\/E1TXVop6wMAI2fByvVEOu56RnrSz63We0w0QopcJJr10THdq7NA89yakl6Hgqy+xEz3ypO6p3\\\\\\\/Z6IekB6xwOxhVK0yAPizQQiEWcS+guG7Wg27FUGqA3lobVBHOfJXZ67bwYn6XhCmvIUy3GhWq5BIVWBl+6XK39f7E8\\\\\\\/NHZNV7L5lzRBWwQqaKrNFeTLvPCDiRBTiYKzKGZQVQ6BY9PceZID6NZoItauP7ftZKelgSKgqgnDIrjacFpDLX511kVMC5MUumNFV\\\\\\\/Cg9Aa7hwlfX88+TOCg4AB7ZzjAUz83XCuj4k4Z6yaIivDdhjY1oDOREUwMUipZd6tomr4ekgYAC+qtVioydETpU72yygF50Ozh\\\\\\\/FyhmPzCIsCCegUeZet85sShGz9MV5bq1PGU8rjREPh8UehooOUfPnx4o9e9I6d1rVQvEhtijoQu6M+D2rK5pSjwp8IvwuRmnC5UHIyHDKjF\\\\\\\/fTdsCJqVaXu1FtKp0FDJLh+iQBOJu5xoyvGB+Mjd8e4ZMYfRH4nfou\\\\\\\/2Cuq6J10YqZulZ7ZHKtqC6EjHifbTZcXbh7T9Qel6YIojBVimikJ3KI\\\\\\\/fh3hJxL2LuzpsojjFPTss4oyTESoL2RinuwVb\\\\\\\/fgi434wm\\\\\\\/wYhu8lWkyLpJmwdX2ofxLRC9TOFz\\\\\\\/4z+aeZmtcZAmXGZILM3z+ho7Sk3yRnNqtsD9jgnCCm1fFRopUkt\\\\\\\/gbf+f+ivFhYe8j2pEZbdwb+d0CvkE+5mkmDe+siN\\\\\\\/rrUL3xUKh7Jk1eByI8iAnapbuo6R6uzemTdEWLrPwzTCruub01FL479PuBl30YqDI8YzlcAA7lB+Dn8RgwjM20ufkrUg5pyjQ\\\\\\\/\\\\\\\/yAF9DVpuRWu5XIv1tywwKv1Itk6l4RJPwNRUkcQD9tev6Iyzwpo84pGz1Z3BZgbmQDZdgCweKh6Wm\\\\\\\/+uzD7WEvzL4Pe88013teG4oGLk01SQ5Qa7FeEKkFj1SKMyDFSYNHj8hxDglTd8uohdXfr2woHgiz6uFhvezHvBYmENH5raydVz2Ou0DQggL+i30lqF\\\\\\\/2k4XrdmOCOlSXyd\\\\\\\/Xg91Ipg3t+\\\\\\\/e9RQgbrEWl1zHSr8Cca0lT7urYP30HaCVEv4eFsLrpbu\\\\\\\/mMLkIa5B8eGz50mjnYMmtwPkcD4MYosDWQ6loOPj8cK0TU+lyzRFZGAB90jAaT6e2nnj0VULneT+TCZ4ngZmL9F9szoSM781LttXjfMd4ORJfRREUc40rMBMz9Fr3SFhUXC5upEk4+wkvq8m3FSDfKoCuRFlgj9EnuZx660Ig\\\\\\\/f8R5qKDyujnW7ZOH7pHvaPggYHSnOYrL6N6syWsx+ezuOPXAmbJ\\\\\\\/6gGbO+GmlFqRSeuBaNKFUJqg5hFE4R1mbJXjifR4tzH+iym+GbvMUK2Zb\\\\\\\/QtAbKvNvXwo\\\\\\\/dMGJ+Wrf\\\\\\\/w7hCJnaYj9PqgDCQZxx3Lbe0EFYDLQj0bRwev9Ai\\\\\\\/pPCD8enxbiWaHJjBYZGD4\\\\\\\/opsVXqf3ngKzwQ1lxmeXYmjt4D4Yy7tfaaJksbhViPQH\\\\\\\/yeJlNx5YQ3ISzr41EltX8cuC8fMMKetOpp7XN5s9geA1jA9Uw4R16SskScNFsoP+qVmcxhRc53eYYeJ9y+\\\\\\\/yzgZAqZT5eH9yfAD0pwnFUf9xpJUbGQC0bnv\\\\\\\/St7YBY\\\\\\\/5dGgM0jL2ufu0sgExM0pWFhRUiQARxU2j\\\\\\\/uLWlsVUvoIZYX\\\\\\\/VFjrESNytP+o29HQwmIB+oSQ9bAVgnJZuyXv9khcvk3J4lF6awsJEhhHZP4Ytwd\\\\\\\/n8NWxduVE2C5ozydhSNOrio26Uu+sOba6A9ONGSqWNQj9+n9tyfHUb1fXt1wbLFj92qRuskkl3jetT8ombgs0ZRcu9Bw49xtenQA2GiT6veHPOj4Qn6BSN4tOog4NJq9EWnooGCgZe72ZFuH1qiPQVvYGUDTqiDah0CM2Ye7HQp7asQi48XJue3mK6I+r\\\\\\\/C5VNqikXaxKH54nvvQJsDqW8Q9rCfjs+2fA4LMXr832DBqTwELcchx8my6lUfodtCRyEMXlAol3rNrZ9T+YNtjCzARJvmRNFttIsp2Tvi8AqwnsNR+BZn2Uecx0D1BME6ESm1\\\\\\\/NbaCwQwPjp7iWKVFy5T\\\\\\\/hv\\\\\\\/JIAEIaPELcAMU2ws3JufkexIlRkIZ5W0t5hAVoXkc4lD7dSWcnFtAxqNuXCcyepcOt91SD7oo62R\\\\\\\/C0TgAqr48t01OEn3oiSLKkMURiRKX0HdPV3WC4Gy7ZlzEU\\\\\\\/oDlCAJlCqNUNuXLk0aNkMNFjHmXcJWoJXYKMPfz4y9QSUq1Oc3\\\\\\\/M2CQdmgQNAuiAK3eY1NtH\\\\\\\/JM7kbZwfsC8Y5wcB9mXFxutekP96mG+fmbv9xHBuxhCcaJPXZzf4HE19haVqrYh8QS\\\\\\\/Zxx0X4uzEgmwVeDZu5wSq01+ZtSslfCC\\\\\\\/4uxD9myVgHaIqJd8CsXmurQ0jan+034JLFdd4\\\\\\\/2wxdJlgk3w+V5EKKk2inibXdGhwmm42QDRGdbIj6eiJFi368poXg7JyiGk8vKOtHw72s25y7xCtjPimMIHQK716Y7GNOImXbffuTavD52NmEybAic9jTYorUG1h38oyHCrzqC2v4FhEJDlI5J6gUj1kuNM9BNeMhCG0uko3Ba7E0kkw739rbF19D4iJmbvHibiECfeTj2j1OsmbiXdpRGatPNpCXVVOwFUHBN0c7w3Lb4PLwHMRJb\\\\\\\/JsTx+o2LxrzB4S3BU\\\\\\\/TS3bVhhRxv5rkwBuNm4UHalRDaCbN86cKdzMl8iStr9RevE+IkxyAHKkdt9L4iySPL90ZbNF44BoPtjRVH60ymUtBYdaia+FcUPtWRnRifHSqpZrxwokH\\\\\\\/6yg45yYAqMCn\\\\\\\/KPF\\\\\\\/hecvp5xQRuGvg\\\\\\\/P3nBh2htvai0X6ELyqidum8Vu382Ybu2Te2rf1gyYgKNURc\\\\\\\/5oVg6FluFTbOCJBtpcjFhPK6zr1v9Fwpz+6wvdfmmZhR2lnFq9IVFTGL6lRAuBiEB4cnEHvJ7Td8NyhhILCAT12V8NlA9TxznkhXkk7phfVrRN9HS4jxTbJLr7rMeaN5775Kg32fSY6zwiZhMr7y1jPX7uAQUgQz8BgutGI\\\\\\\/EZP5\\\\\\\/Lu4wtSbbuVcm4FrtAEUjgBZ4z6699ykxaEuPnOBDihdMKKlapPILPbP3eQNkQxZQHKTRv570gxQYjgUo8zAbXbdVOS4mAFNbBBjeXoKx62TnjJLr2MpVD9BMxSG2XHaH3r26m+8fZ0wHdkb3loQxvoL2nHAssy6uxTtq3NMmd5eAaA+zKPvWvVjlnhrxVRe3Wj8o2TzlTI76oXmozfNdTXZ\\\\\\\/wpkdyLWCk7WbQdYGDRgBsddZBOHtnsQ9eiJGEhg8jy\\\\\\\/fNz2\\\\\\\/BnyWD6rMPaUvkRpuj7y4c6aGCDsNcD\\\\\\\/0wKGJcElNf0riXsmlcR8Vl8CeA0g5DgPkOMewIY730qqVIsP9rurqOTP557ETMIRzby8P4Bp47mj+hAHEzM9Ixl7tjoP1w8yTQ7nXaxcxw7Xwd45prPpKJjT0q4Z\\\\\\\/hdMe7wIdgX0ERQ+DKxLZYaRT65lyaQRg8a\\\\\\\/+8vCxM1oBhoTijcMmoJdZ8+uSM9kCXD1zSY1gKIitSA7JagTbZneT954p5Afnmkq7S9DY0G7UwugnRRrZU2Gh3qHL+bKs6GmrLQfm7\\\\\\\/mH9EFyM0A8Bev7WLqgzFDhhsTA0ntltIvVkli4Ri4wkWJdarPPcb5\\\\\\\/tKXB2RDhUQFkyuypdj3FIAzoCrNy256NtJ8U3LlyniS+NHy3jthhBePqXawL4YefKZDqGN2wSLfusK12sTEWcca96k7\\\\\\\/kYjp1Y6bGF9\\\\\\\/jTcs0tFVRswlJqhPLkr4X5lx70dIE\\\\\\\/ZDdwqU7opTFbufOtagb6Fvd7AkphudlEHW2zKv+j4QrYuVkMrKnaN\\\\\\\/2YPFX0bBKvI555Cx0SovcZob1sqpofX+IgTnA8d0dRTqVmBtty+OoigeKgatjDXgio7r\\\\\\\/8te2YlCIySTHcNw\\\\\\\/kTltTn\\\\\\\/t\\\\\\\/1tAjp27+XgtymzqM25BW2DvWr9VkO9sIf5ZgshpmTSqVNQb\\\\\\\/64lddoXpccpRyNfi4aHfOCH51IDwsqFVMi5B8ucbEIbUzEZVasZ0zdNXXsm58sp6d+Yg8Hv+Mq8L4wvGR23cK0z7PNKd\\\\\\\/21wCoqzQkzoiHUjwdJPe6Irl5QQG1NWrpayijsWjPonFslwuSB9qZrDHsI67m3SUJYzjhNWbLhGT7pzE6+7NaQktUmrU\\\\\\\/FKXBPLC1+ZaIFRIPEQsoXuf9g82b8+hx5Ohk85f7cjlbhFwhQ3b\\\\\\\/HMq3gdGgmw4DYha+Zz2gFpv263O9N03BeyUZJ4iV35QOU\\\\\\\/3hX\\\\\\\/zcaxS2GiZyH5GJFR15pVebAzwBNFq41vq0I2txQanCPNHPMRlbXz0klESg15+sXtkpypIZYQZthlVFSa24+zw\\\\\\\/PBBiJiTQFuXMXFb0LQNny0zO5d6myFtWk7L38wIC5VI46y0pqZeC64Mzhs7h1705GubX47MIP88vaLvCpFgt5RTVs2acQnEX4am4VRciB0GJvkLadlKLnxX+cirC48c+lv7wLSJ0lN39WRAaEFQ\\\\\\\/6ISIUs+TTrTg6NyWn+Ge744K54HOi\\\\\\\/x6eT9+d4j2rVf2\\\\\\\/G5XKXxzoG5Sb9ScH3DXnAyaJE\\\\\\\/AuBAmwUjSRJcHuemUb7xhmX8SC4f1wY+hUJt1HFjzgQSMh73Urfo2617rNjJH+FYJnv2DWmpUYb4+IPF5co8buuCNw77vzhRBcJfrJ65vd5et5Dq7h+mS6r1sAEh94dXcRGyL+k1m683f1z34nIWhQo+veI4dHmjFltZIAPw590TjcgpP9cJajFkHj56eNLe69pUg+iR5cuKfsE+gqh9ayia5GqfKasoditqv9f1nzChLRjRjfJYjfEsDzGx08HDSPnYr8qQ8YGScQR4vTtyl4izCERMRHmN6\\\\\\\/zNK3bQR2ExRlyGnPKZ4uUVeJwVHGdr9RWgFGx2PbjdAG09QakKm+b9pF110nDdT8flOEQmLNF5xiOgpQc3SvrhMBmvOKwjqzDCXSFW4qMMzk8XcnsUqn20TElY7fbkMCcqWxXvy6XBaRx1f9KwynGeyxk3agdlgn\\\\\\\/qIC6fIGUc0e6RcZ6eeih4vus3d0Hzt+ZFOC8Oer86rG\\\\\\\/IBVqAvIK1sJNZO2\\\\\\\/wt3xgVKyobbJHPZK25JcIFMyUhJeXRZzioUD1z9HmVliTeQFonR+WIGTJAr5mMYT8b4D2Dp+t61D7BGU1uyfoAXbxC35y0CBZlPG3fuyLu8kUoYQwEWTRFhcKEEUHSD882CO5Zp2sEsqi0wM7DrSQhKqbQZ8j7K3GVx\\\\\\\/coHHKFnvAnhBYw9FhCG2f08evwegbNZGqJOBZmHu9Ok13YzlR+Q6avPsCVOL7NLjYnWSMsIg4fcNctt33ZoyZBY4idj2rgV\\\\\\\/Crk\\\\\\\/NHBai1pZ1UM\\\\\\\/4jmOZl4Hu8yK2h\\\\\\\/J87YdfQLBfpljp4SL34a7skgpnLAkOeK6oPcFam2PPE9ycu8c3hG05L79fp0MahnjnAx8QQ7z1F8PViEtBWGnItNWVEXq49YnrNls7r87LOmsX7qJYQ7zRItaQkHm9PKQEz+Nxk+BIm0oO9TEGhRw4v8JwfxINpNBSOFgeWOa8AgjwKvR+jA7b5ZBycZ80\\\\\\\/sZ7FadYjwKTZp3OP1td5WmOZRQNxRz9fGaKqlMsUgJRlCLPnKcdk+CP4qbzmGiPTdFtm+JRfgcHg9vfODbr3Mqx6A4WBomtT9q21W21e1mfY+gZV+Nx2xefTP7793\\\\\\\/NK9elW4efJPtDUxpoC2BAa4DXJic3Q8ZBmpcEjNB36+Yhzhsueob8TlPfsImXING6Lr3wsosZxktkRTJ9mglFIhYRToXRgnwCCgxqrcJQjCNbNX9fkcd8vk9730CBZj5f9G5YP94832fnAjIGEL8ZmC3SFv1IgFauFUF3klr+G2rOZc4GVVZVaiNDSvaHZIXISOr+wRvSHBORZIPtK9z8NwKLnfI9j1qJUyNwkI3nU0z\\\\\\\/I0goEwoZtAagH70Qx9bSTAx0USD01zqvJS9CJSzamaYYA4BTkQVNlsXXNx6DbYNHoVGjzxzdoP\\\\\\\/N7iX39NH44I5C6M\\\\\\\/QyFanyQ90oJMzy+Z2TKlY17g1DOo\\\\\\\/vhAYl52d8BUEREyXR8Ouca8lf9cyQdI2GwzaDhAeD5NRGHTC0q1+g+ISfnYZhDpNV0jUVjdjTrdrh8EZVuDhvsojj7+1EFAk\\\\\\\/1uvXK2LN1p0vzyqJfdqz6s+GfW3QNUqWUJEAvLykg+XZARrxEnhSc4jrYkvvW3iq0axt66ER15iKcBbUDE3H0jn194wDzg+EqsspSTSkvaZZ4g4twqquEv1h2i7Y2rjXIxukKZS7wSF1duKvgCATquMEozlfuDXb\\\\\\\/\\\\\\\/0qqnGE2BwLChMm0s4ej0+GVPX0\\\\\\\/rGdb8GQXxtgV8xWVoshUHFTi\\\\\\\/hbo6uJFgzj1xPr4WQvJtZ81dv0oVKZJ4o854Rb3JtZnI7rzVG0ZI7b1PaiaIYzdBhwTnnxDa5FcdssWfyeBDX3ETPksYLH8uB1gcfFfyaEIKV+pICJJjovELZA3rqPhKCiOL\\\\\\\/lK\\\\\\\/ohgMcw9\\\\\\\/FwdLymZOol6MfB5UAlDEyWA6eNQVArI1STqsPvHwf8fpiOX\\\\\\\/Z7SHCHnFSoDWOrkJpdu3w+0OY8Nf+8AKcspZpz4085DgpiIPkyNtG3sH\\\\\\\/NPP3ruugu3KHMBppFlxFJzD1bIMIi0M22mj+L5UEqfD1kh8P+5rEWyrDLD89s4NAxsJggJcO0B9TPEqN8AxBzJZxiXTjcdGAzl8aiwjn5dtcCMDpHsYd2z5GV+XvMpCfbecN5ysqKlWLjSMrLlVDB+yFmF\\\\\\\/9WzitBJQXoFInuJCtC8o0ad1nKs9iO7Dfx3NM4eeQn8qrl6zermUzCx0Bry+yeocvGIN0jBOcetclq2nTLQzb3JBwhuGB0mvdz4BWivEhFGi4M2XL5CTrlYh3F5C\\\\\\\/5L1SvOPf0sCrxyyNElou4LfSPXxKTxKqg6aGsmkN6YJTzGHv\\\\\\\/kuGePbLEY06pWFpQ99dbkcNa5q7FPWwSFT6KuEO7RapvaUX99VrxVmihrMl0uL6z+I0wxSrnW3+LGMu5KZYLYqpzBWf5\\\\\\\/DRJ1R2BN6FAqtNmDsSsds5\\\\\\\/dVnJIjcGZo0W3fWO6qMNUW\\\\\\\/e5IdDo9LVPYorl9f1LxFGLqN3Dp2w906pCICQlQHA6EolzBg+DtrH3czeE\\\\\\\/sFLQ0DJQA0QqgNS4OcMK894Xvfs0EpuahyAHlUSQ36GXltR+MtV7ofP7162kNrUalX7\\\\\\\/WV01VGXwo\\\\\\\/TIAAGHxKM2ToZabADpWtwLG+7BUxG0RBYwH+G4MDobTxIbvB5X+kcUFwWDbdsFgVEn5vfCu6Gny+jEFHtMIpZEoEhYqyBasfDusAhXVKVeTff5BmfkEewQPwTgY2Ir61KTIevjtEmsBSp59ppeDyUs8T6bPZK8onOd\\\\\\\/jSfjgh\\\\\\\/ZIby\\\\\\\/Jd55RWXcZSbqtHM\\\\\\\/DSZ3oO5zj+aamJ\\\\\\\/LMxj8y3ODhzPmJG86Wy45g7wHjCyAIzHfznrSCbTYa84G1Je7jiReED4eTUMk1JQuBZq1dQwyQM55JanFtdXMV2bkb2\\\\\\\/UWruNnDPnPwcgXS4fxtixBbZzqAejDhe0Yz0hSaJ+iOjHUJxbcWrw+Jh5AzDt3+mfdbB9sw+HCbdDIgmNenYxAc+hdyDSLoV0tHt83duczUjC\\\\\\\/vSNgVM51LMvDNs6y9BeNJi15IZrLIh0teU9vjDe3m\\\\\\\/VeM8BfGLb93taJwYueYvhsaGfmFZUd8ixZBKZvFc5xbZzXz8CU8w4myVXEOTiyEHgsfGM3Z58HXAgnRAN6+iPjpeH77brCyiUrOW7vSAmnA0emRqR2G5jAkWD07iL10NROywdhL52AfKKlsya1zcuBT+OmapQywIVlxgO+QeXeD6ynViWKQfbg7lRp0kfKx1b9M8Zx3yfnzs1Hpeme0Q5UH+EDW\\\\\\\/R+uT2fjmF37nyWgn4aER\\\\\\\/vXeqMxwBKNvrnL9J2DG3qF0qDP8pswhVLrwbtYfmYYmPyZizr2iofFGL7jbO60+awFPDesNKJ8ci8VuXVf4rEtzKBIj8tUEAblP+r+6BPyzx8DSp1hNy9RuAU02l+Lrb+Omj0mu7qxVzopHSIVYmgixnsYW+1Pefit2Uje0Y6yqZMNbi4baVZyhj1jTzrlTY5jtRXDtCj\\\\\\\/nAH1ydQX5lFF9zI8xzn9W4At5rLajnd3whZtJbZpwZC00Zbxys0OFQPyx6iJu6nMVdfmCC\\\\\\\/MVZ8wGjzjQ77c+oveqFvulYQaKNkfKLjhFVBghOnfNooH2t\\\\\\\/J2UPutEopDRygvQQC3FD0COY\\\\\\\/\\\\\\\/9Zas2gyF9uK+p4wVuFF6sVjXsP5gWzV8BXcTazEvmy0jizM\\\\\\\/fmm4vWfEi+YdM+gHrFTviODbFmibHsTMd0jTyly3+MUgGxF759cDDS8qd3xvHF6BhSS2xAvDXQ\\\\\\\/joiS8uHgn\\\\\\\/FgGeKOGFTxk6SpvTBpoiHXKXpZnUs5B9VNHkE\\\\\\\/ajaQMQFdKkZ+YjDXDQK6AKeatbjLllR8Qse4WjZ4+twr6\\\\\\\/HGsIeyCe1uUHPKNGPIbRYXrdMRS8dR+9WtNkK\\\\\\\/hQNl4KNETGnc9rdPGY5+ujOQ\\\\\\\/zDTC+4pk6Bcx\\\\\\\/oL12+d4exN57n\\\\\\\/7D\\\\\\\/sD0e0RzHqjRfNfehKNQtIMFObpB6NMgNczNBcT\\\\\\\/I7iy67JNt5v5yPsG\\\\\\\/SmNWIznFBcen6ZOdd4AH6GL5O94ZnzXsV2cYCaHC5XU\\\\\\\/\\\\\\\/vhNHC7yXDHBZbnJIHPFyolzBiUqsxAS1I6vBFUA3U7f9GYN+gqXi33UyX3pz1Yig5VBGKQp2zfAsO4M3d46fFoTPXug0sqkN8IVu9tN5aG4\\\\\\\/1Kex5WwhnCtLUxG8FIpl07JkVO\\\\\\\/8RTIQDG0C7GOWZJ27Pz7hgTHCWFAuVHrYPK8Hd2Uh3O2ZJ9arGBrR9FDrn9id0\\\\\\\/d8nmCBgJATsn+3qXhGpfI38cxjdQ8LblsumvfS\\\\\\\/diHJluI4ULWy7gEOebba9tpjvJ9HYd2Femsdm7cVOZfBz6NqceiemXWswEjVYxe3FgUG5XTQ8kYbw\\\\\\\/ysOoJ2E3VkMH9GhdHsduXKuRgoJgm8zmZFJs+0DhvUR7TvNb3CDUOYMQZeHnhBbQ2QgLY9ir7\\\\\\\/v6H8N\\\\\\\/1PK0JAAJ4xCEogH56IAVaVtcQbDRJiDJc9752H7\\\\\\\/iDx0fEi15rDIp7nV46SHvXUu2JypDuAvFOP1N2nlEreSGexwh0nmvk52xCoovWXRubwRD7eFUI5eWxy0ykEdDJ88RRbVaSvrCHcqrncctptI8UHjduuGROcvZqPkUyzMChgbXkMuh7tmh0o2aazL+tfcKSGJO0KR1LV5aIphHZB8MibMvJBkczuaSJpcGK2NDqZkmrMEh7yIWXlqk4gpwoaNc9Xbg1anpHMlktV3cQ\\\\\\\/ycJLZ0OwNn+W3ma8BxaOpkCvigiCQ\\\\\\\/NCSF0g\\\\\\\/dVaR7VnkHyJo+jigkSYGSuU0PfrpSqyRmgGPPIxnTdZyZzo1WPrSYFpbBiFjKJwvAg9rUkNGTH+pcdmu5fR6nWz54xY\\\\\\\/J+i\\\\\\\/HbT0JlsFcxR+\\\\\\\/pLStKdk+dlHLRoBI3khpZq90Qgimtf7OZwjXaHtT0geBblm8VuTbOqvqoGzk8G2ttXAGICKDSw3JfNxZXywdv7xmhDlZJWGddTv7R+pAwmwe\\\\\\\/3rDH98hs4AkLcpwvY3vuElua8jmen6tzvLmp3Ejsi3PSYgmKMLTzuUocxV0r\\\\\\\/PUjD8\\\\\\\/Y5\\\\\\\/iC9W6uGZ9jRbRXgdsHNz+NeYV\\\\\\\/o3TVjjokDnxsnzBfpB\\\\\\\/5yUJI47O3ZFlWc9Yw0PdRuxTzji4\\\\\\\/qNEVPXaqeSECMQh2hKERC1Ku81zXPp9DrVeTqhqWUMcmnJQL2q\\\\\\\/YVXxN1PndL87WTaTsqtXleoLikYo1BI7vUThWd0tzNwRqPf4JK\\\\\\\/2lwKwnnXGeXwVwYhaeEGHq1J6iu7yUEEsEcWlT6mVzx6TgNzC3rc9tDevh11romnNTP+3XhR3Mm+0TT\\\\\\\/V\\\\\\\/MA0HQEtRdspCmoE+\\\\\\\/DSvMSSjAyHJ9zfORA9zcbCiKMDenEo+Fvmd9NYN2ilWyZ5ExPpfS4eHEi+qJJKAVPWSxcyrKNagUxUk6ZWwMCPO3lcDRA7S+oYZwOH9TXP3lGFYK4WqAkhnhbk4xED8X1gBUE4Eqq4Lk17\\\\\\\/CkphwgA1cv7mJMN32XcHXmkppS7y2PlniQiSmX1n9eXo9ObA7osW4PBcgu2idnJu+b3RhX+tFJI4Ko1pZ38wwK5Ez4EeErraf2XaNQGd+A6eBbnqv99eWowRNgiv9En8U1uJj63PGRDFpKOgBUuxbtBk4cJJGGz1fvvHoDsUdsKvSiRk7vhtERUx+zZdcRWMkKfc5vQlO6iilk7UthFRQPKRo0crrMIAkh3m5yBsV\\\\\\\/a68zmW5FkSc6oTHPDHLSkpsNk6MppW4xsWX8tXt6HjQ5tp+S8VN4lOb1v9Qr\\\\\\\/LJLNhiDtZnVTEHRxzO+emLdT6epGKNOq9L7MH+3ZLCAmd4oKNIuaBScuQFBBffG7GVv1v3x6LDrfI8oycXiDBjjHntQxZf9liQnTk5RBVGhmrl2ymbU9vBgvewU8\\\\\\\/oBkEt9ICMAyzD44QS6AfRcPsQtHZLFFSDXSvi54T3GM4BGEkM8A64tX6Af4btSAKlr24JWBBr\\\\\\\/BrVYLEq0zrwVvNl3Z+2GSHURUnkT7Rbv418fSZg3RDp\\\\\\\/j29kDk2o+QGWmSMdK7OGpR6aaOrcD2\\\\\\\/WsRH3cJLVWq5PxXFVgB3tqojdzeaIgVDPsJ3o+oLFiQ1qX0vr4Fwt37EYBzhX6ItCpFZ5zDXEKohwpZbWNUexMuEKKoseUPn4+EoJHVmvv9\\\\\\\/qxYdNHdQNmMsvR9sgOEf6m3Bs2jna3uV5cxLsOCHaH4Tjw\\\\\\\/IDxS7xJPhIICxxIQJu4vZO6yoXrBm4pduw8RpCKH+MN7s9aqEMGnTnTMdFL2BXurZ8hmTljcUsVOj8fwLez4FWi68qYDg3jl4DNtrnX5j2xOFFj7M26jc97s\\\\\\\/2ZoHZnNFnuZvc3J+SSDPF0xlWCf5n+oCzp8YR5OLsb8KT0sOw9pY7DenM2iBJPA88Al6mAnxmtlWRHlBhQxhC1BAm0KaP5vXdtbI4T1HJV6gflAxtG1itPoASM6pucfrUl1QrRHVmkRyylFRnEmsEG6EPkQDPaqWVxpJSHvYI4SBejWCkVRXsfSHQHPmYRgdsWLAJ4s5zcSy7csGY1n0yCl7HS80XF8L2fRPuy9z5wEiq28uGXsJZ7vqrP7wmSleJv\\\\\\\/0k4WEFIHL120j6DaaPfXOKnBl8xLzESZk7tFxFAyv8CPAXtQdcEJ4iLqUKUohQMU9CUB5mhVCGQtfoNY3BcLgNUbWMbvyYpsVMKc7BKfQCY3TJyOQLGT5+rjodtFpTZw9E2BneVLGRwOCfC5bacFHPp+lVER9+1xalsurH25TipSYn4ZkBytUMkOe0S\\\\\\\/zZQlVfjtFkdPZO1dByn0kWHvdIEU2Ye\\\\\\\/L1B\\\\\\\/M4g8I+fVZgsjEQ8pGI+B3CaZLUyZEjwa0Chic\\\\\\\/HRxI+K7pQF3pAJsIvzTypCWoI4iGheVeNgc64AGnVr\\\\\\\/KtkCSmwY8VhpUpKoRF8V\\\\\\\/JWzKaspCM7MnoDo1OijhG21CGxzHeCf2Uw8xeIXVjPZEMqdHSmhSBh6iv3+NG3il11SoBpMT\\\\\\\/B\\\\\\\/DC5TIgwR3rdk6e00mNjNJ7+HMKRRmQj1ZzOe9Mhk7QnnqQEQK9X0PMuuoAJRCD2JIDsA5t0MiKENMdXvKGysVPejC5EDdZiBBfXE\\\\\\\/fOTiRL49ADqF0IeFTGGqsUIxCGI\\\\\\\/g40QR\\\\\\\/VQVzP0pF5NcVkeiIbe0hcZLnXd8RuciGLL1FelkAHTjo8w7t7ASQZsZ6ZY+\\\\\\\/HUuWg0TQAQW44qNX5SPfjIh4KF+A0OyExzqrC7E3DBGveQ5DYNcaU4GtwXGWweAIvP2V3oADNnBWAiTe0Cun++LwjJqMzBfcgewT659admeDA0EC1OiYiUp398q1Dgg3E5Zryv8WlqPustOOD1CR\\\\\\\/2HYfGY9sH9jU5+Ta9h9my9KruyZ\\\\\\\/w9l8sS\\\\\\\/Reqv\\\\\\\/CgPpTmTZzkr88txlfSAiBa4xkaEchSA+LvMrBzGiTfDF2zM3ZL5vz4LFfcQIY9aDs9q1FgPez+wqF\\\\\\\/KZ438NSsh4TYzZeWkbZjUUjw\\\\\\\/fO0w3AYvm4N87IeFowNNoRHye4cpRknQDwUimej+bQtza7E1QCTp8Ij7vQrR5sOfhLbaDdMvxqHX4aEBxs2kwXKzmMRPQwlY6ExiHKXAfU5bKHj7UCITQ\\\\\\\/hYVHZhccBfxu6F+w48qcOlENorBlyqrwIbSNOBXlPNkF\\\\\\\/z6ke8YcyQGXV2vIu0K6N5AbUSSGcuqyjvi2f8HFooR+5swQDXoQM9fJdn5zAkBK7Ebt3\\\\\\\/ZMwlRJX7Ha975AXkruIdZAcKWI6NQyBoaU4r94MG1vthq33juOuL0AUKnuHKdGdpUqJ43fuYw57MAXc4NuH0xrNzkXfxoiNfbdJwiUdU88pn37qJA2ZA2fQBpBQsNsx+UfHe\\\\\\\/NVIGiTpQ08ijm2RQjnEY345VE8t0bYfa4yuE87tcTZFP6TaXORn4NZceCCPxccQVLUZ0tFD+\\\\\\\/6En0veQGlOHefMBTqxpvR0aIUOAVEcaAGfBABmxCREtfHZiOb8MVIPb3fI+Zrnn98Pv2tlWxnt4FEI69MRZiOM6DC8yBTKW6vKVXEfCNRQB4zhK14uQMUTeoRjOIncWhL2KZ89io+I5WlT1YLPPco+mSNSJXLhJ\\\\\\\/mb1c3xajiHja8L3LIpivk3HkLRQy3XgIRFnIE\\\\\\\/07xSnIk9hgbVo9kGYEcDQKx7XmszDf2LAkMjCaFypMwRbbpvL8jxXOsCjlV3SwvXi8dEI+F3sLuMsjtpMVB+3Rc8Nt0ikUbPl0kWRdMe35DwhRUfOZvwtt0spWwgX+xzM5cbAxCVCTU3bzcZLyW83M2bgzyvixmtlnXmVcUChHHE4g8CIdrc67QXQqbbWsnK7nUY02+qRToFtxbj\\\\\\\/Q+2kiRjnlLY4Dv1UuAOMIHeRtVvt51n0tu22baKeJInQRf+vDHco\\\\\\\/vhosPTCxmMaYJuRVRENz\\\\\\\/Asm8DSpix7qY5EremxmwnqO76iTNf6fJITa+Gm0hpVt8y0jp0T1XhQ7F3RO05vbKGC5Lr+lph3LKEc+IVal2qc5szTABa2JWjVjt7JxZnUwVtNkTw3uOeZHcITd\\\\\\\/LgSDcWI08tndvvhzuBu1esBAVKrDaq8woBtY4Mq6HZqSEjC1iDTm1IDqL3VE\\\\\\\/PKJAcv\\\\\\\/Oncv\\\\\\\/HFK8kKlhz99JHvHxoTXatHrkfh\\\\\\\/qBNaB0Dx8+KtySd2q9b04xYeDoxBIf5UEjWjSRRMrCc1I\\\\\\\/TXU3TpiqDu3pNuklSbcHblvkdsouJ9VQNVLgXN9QAuIHpOaA1CWz9lWNHx7j6lvDsimbX2xQTLRqhR5ZgrGBsYl5UtxMgJDvF0leWe\\\\\\\/I8UKkMdowsD09pHGwWd6iqlMsnH18WyUBJsfjah19H+1o+uoLR7LQ63otAqZ26CjIxFTrMqkfh3iCMRKuWbTeIYebMXSx1KuvbhRagMjLI8xLBXiHMABVoaMXWNwjQiZgrs4vZ2QJUi\\\\\\\/KGB1EyIWyuueqVSCVPN07km2Gz7UKfUzIXSfhVeDTLrZPyC5NjVkUvQS9fljKrsduJR6yhpYbi3Nw1bH+SyH3PXIAOxaW8yUv2SCu\\\\\\\/FblMxafa+\\\\\\\/gJ1eSzpmfx0iDPWtnzwYUV+hX19KclY2MBMlLv3pChZT8EmfoHW8tL6m40sjQdoei5Niq2+nJHD1bBIu8Y5eP0L+yBJ4B7eHq5aj9LWkjJmbfj1m2yTM1BzZ7Mozg3tXUB+QmbRauiVLciP\\\\\\\/IEVLHFJivyocbFgdtLmii6x0iuGZSyhV2rNiI0TEGRYggPuMez3sz63qL2soQ+Xwg5h9OY1KzDM4RRQyZQwW5LiGjPOpLCEvHeg1LWA8q+JN+jnr3URZDhLg4lti8aCbQ2U59qegxUu8UM6ZpT76\\\\\\\/XB\\\\\\\/eJWXyPkefcVdN2NyhA+G8gv7rcJTky6BWfPu70n11DTQuP7+0pqBI1FRQA7RyOnCiU2FEHfOzkYYbEanbOZvPRAXNJKj2LUyBkwZgkOjaoYzOcNZG0HC2iNswlH3J645O27YinMd0TIXkGGWQKoaoAPcKGaxqcK5YKmBcWKmVeOUBtTPMDowBrknUUlaV3W2hjWdeS00PZprd9FrUZEXCmjnPRPLr0OKwjk3ew0Gfw8\\\\\\\/UY97mUV1FTyfe2keSJt2J7ralWvZK2KDyv4GkNAQR3sMK\\\\\\\/CTuyyshfHWkb\\\\\\\/vdnALGisI9dE6bnAybNOYPlFwfwhE2USdwx2MbaxAEci1z024hral+U+DIYHUZm0\\\\\\\/5jQqsgqzjLXw536xnLW6SrSHit7uUV1dIglkaLnYIdNBKf\\\\\\\/Du5nBNUgyo+svJj\\\\\\\/rjIEkVAlIzRDgQ82M8rC+QJw+3f0roBltMTyAg3uI94HbeVOdWWVRJaUSxFKXLqhE+0yIVLTizLGSxvPiHYkDP9dzq\\\\\\\/vehjOD5rfvSmYEmBhqKC3143X6yBuZpLSsTjbURNsnoQ6Uj3Pvct4YQy\\\\\\\/yB6OBws1BbvOWHGHCEV7V+fDDhWdrCEsC9zGrsJ9nlFXWS5TnhaEjmkzjFKblEgebyE5g7rBKjitNawwoaLvviisYm\\\\\\\/dYpZsn8kjgR51HSM9mbkDRj1UIMWZFpZKcq4HwFN1LHq\\\\\\\/V6H2nlDC051678zwhXiLBWq+qJVP36aJbD5HxihDW7y2A4vaeUERSlGHi1KD6MWvwRbxxsiX9kAq1eljhr5TmZ6CiieW6kssTvTB60Mh\\\\\\\/L+MgCNzgdyhE5fAFJNKyrIZHcPCuX\\\\\\\/HIQQ5\\\\\\\/xYiY6HF1N+t9KSK6F4XfHPNPC2WKWjCm\\\\\\\/5jF7+mK\\\\\\\/1XVjwoqUfAvZA5bRheZXjbfVU3ruIDZ7o7LsY7Cr7P8Mfxxo4gqYXC1VDH6bEmdSUYq8\\\\\\\/hsesaSXiq\\\\\\\/x018q2\\\\\\\/\\\\\\\/8bCvBUEi5DI7XIMXZ4FBsRB4X1E1Jgww2HQTOkkCc6YV9Edj9zFiyJQxZ9oruykVBz8x1u1qKwHdQNw2cIRTBJ03D2VFeNKLVcYBZY0egtnAH78\\\\\\\/dBA\\\\\\\/F5R9ggty2o0aIY934IHWUr2GluynSux0YsWGt0rH4SA4t5O+7UWWxOsyYJoTyvLW3S08cba36jIL4isisVcoVD3HdVkpY5OQehfI9t1ARFkajk1QxqOBtq2gKhZ2C4xtxCo23t8thFPeyxnV7ZdpEV25eXkbnuusVDFjiZJ+FVL18AeZvH01VGvqFHAJh37BJ2tJ+M4RsYddd81GosJtIZraZIWNXvDBi+VCs2buPD218c9g3amuxKfSe5RY\\\\\\\/n\\\\\\\/3hhn0dSRmo3uIlXSoIjWmlmvprRcAQIud\\\\\\\/aZ3A9k0kGZTEeiKXJKBnPzqYZ5s0kY02WwxICZmvwkqqopMPX+sblPcYHxRloyI3jUEo09obZ5kAQquSVUiBH6Q8gKAXSSmBvlvIayAsWtPBhHLII\\\\\\\/w1WDzVokTrFVPQRIR2KrgrdHbGaqWHqlkZvRmZmF8rRfO\\\\\\\/vl0uz9KV09fYSBX80AVEAjx1oCA4UHcb2I731lO29UVXxwnMKl08fU2tFHcJ6vrW+VGrUnPlaUGGOHplCTXfYCmuCByUJmZkIuwjR4neRu8yjfkADk5EMhJWYGStP8P0JotiYYMty5nvObLfIMAB43GE6Hyr\\\\\\\/v3fPn9040Q3aagOQ9vQ+fp8lny9p0HAkxP2xMg7sq0Nf4libBae+MASzhatHVwnPtF9AqcumGratk0OGXSjBL+1TeHsS8BJBG4aL7AMA27bKitZ19Z6bPuOel6qXNqBbuty87SSFsTwoJxy1H\\\\\\\/KvrR5igJEF1CZ9nGQBudvhyY0KrxiAWG32Nd+9Vtwv+HMYq\\\\\\\/442Y7gz2XZYrUuP5B5cswyihYARt3ef2oxhBtc\\\\\\\/M5tq85eB5c0bKlLg+6wT7iqKEUJZxwF6lPZqruZ3\\\\\\\/Ic48JTKOPXRavLBT9fgCx5GEe5vAkPC7ZyV6v6bWEoFBBveP9zkVsZ6TiRs+MPQVasl1pyvV863uSW9S\\\\\\\/Y7aWdOf5awlj2xEwgUQZFEulFYsS2gi1pxbkdxiGzwZb1G1CsMktJatmdzzKSOlpweVjxcT0Fej4xyXJeL6XQDYDkKNb4g0AVneVlrlXMkCbw9f5\\\\\\\/nHAo35nBAYShtEgKBQWb4a1hnVhMU0PYxftzBy3SbQjPwsvvukbST\\\\\\\/ChLlRwjpaeG\\\\\\\/HIocE0x+MBuHL\\\\\\\/G\\\\\\\/GjIaMTrbglX\\\\\\\/Lbk9nN\\\\\\\/CuhNjht9\\\\\\\/V4jhMRGNFyTf6A15NYDVyfHLw3yECLo5DtMungtlEK16tdyHlXHwS3PjfV4tLsIAZSOErpx5RQMypT2h\\\\\\\/WtWbcFzWxtZH893xQCIMdhjno1Yfd3hFOggo\\\\\\\/I8E2lbg3T8ZYz58aSeg0WvHUO8A4yqEJnb3A3oVnS6VfTC1KniuWt8eUaeNHUPJWY78oiDEcLNIh\\\\\\\/4ie8W9wFJfIcqVH945\\\\\\\/dBGRAZvC4MjiadJ505Ag7GzqFcaj9yt7gfPuaTa5WG7msxK5p4y8IbsVVHlrZOzl8zGETtLgVcdkLlB9+9OobkwMESQcvRLTczibLQniEC\\\\\\\/hRb3r6IdVy68meUUIPzUBnaoSEgQxaoEX9XgUDE+dWkRuhS3fAP55gpd9tTiwrTepA8Y3aCNLq1I13p2fXjhS7Ef1FRSIIa+7G2bBNDr0IT9\\\\\\\/knf79hBDZ1J\\\\\\\/8+EN6n0Ew4h+7ANr8pKyj+7P5d0Uplmk8kXjdDtkEEAgZ1eK+LyAMb3cyWnxRB4gcrj6bkVd4mb+0zSEZ2MTAz5UqZoBa2CFdVpzWyTqMEXsbJbpFdS9t35wIcKXPm38cyTzMQnSQZPCn4TsjXDol2K45DixXPDWyahivpsZndCS7VyBZPkLOQaNiUCuGHEWfb2xJVPBgCxOytcKvSgkX5y2bWl6J7HOj2WKwF1stPsoXr6B97mQxLpMgw6WpuKGDQJt94h9naFoj2ApOLWaDUeod0m+IxyibQKWuYnLP8IxW0AuQORzjxPmm46SMmDXCs4sxNeEsX0wOU38TbjtqtkTlRIqiwLw6GWo\\\\\\\/zBTP3\\\\\\\/uDPWRXEqvLs7sdGXpogJa3zJp\\\\\\\/Kd+lQRvmHLLd6XsDrJfI2M0npKSzwV5+eKdjVRtXZrG1YvaUPKwxyZlGhV+8cMutUqNjGgmDFJXUokriY8MoFbEPbw4kC7sSRZKyZqTt7m3ARw4Iq\\\\\\\/HT69q00\\\\\\\/W7iwZpdBjHbuMDIeKSfGnL4NNyQ65jjsfTY+aqDBZ8wPKYtOvEUxbVq3n4gNkcpGxSGYEPj8CdJ3IKBih2WTqFgQWNfBxUk9vWvtX6kJE7rYE5ozxxab9FXAtThIk\\\\\\\/A5Yz4gVdNyQyngdo7ydcnAUceDWwEGDkpylvaOlbU0fiP7CTq\\\\\\\/Q21AdLGQRI37FcmWorv9eiiNNQVnvKQtzfPi0KiIgo1OsZV1UWxg21enL7S2jztiRSEdYfZevxQAg197dadfBdN91y5t9Jd0oRpLONUJv92mZQMaGcAoJlkAaUTzeKIgrXfW8zDEilTjel\\\\\\\/oonKUkpfmOjLs4IR9fPQ8GtHxbkQhfnaqEqbsj5yL0rOYgqVCm8t\\\\\\\/09o7kxMYtGnQTVx\\\\\\\/35k3Bu40IoRdThjFvHjARfTJp7htEku18OQWlMUdu7QPYwsrixQOsCfY56IbKOyyBqVEVTnTuuOa27Sw4JOszza2SWRjL\\\\\\\/Gqfzg6nR6b\\\\\\\/\\\\\\\/MF9w3+006MzfpcHmC+PjAm+KdOLHAplByDxHxtkKuBojEeg8MMtLpi4Q7u1CpB5kGkmWlT5qBmifQcWLU23nS4ndB8RKa1\\\\\\\/sHfhgsxjwmsnrGH45LVJT5oP5AsqxYJyw9h9AC\\\\\\\/\\\\\\\/FNpaS3daAxlVprVyBdKRwt\\\\\\\/tcyPOQvdJGK\\\\\\\/d18Ee3Bu5XYZXa9qD631h8bj01jdqJZ+hoMkYjmIuc\\\\\\\/DQUzXk7\\\\\\\/EsRLt0KwWndJkAPS2Ax0rmt876lrVPVqrW+5\\\\\\\/b0vOTIzXjAq9eti8xBJe\\\\\\\/9pa1gRGO\\\\\\\/CA\\\\\\\/Gk7EHNJPIRqM1YOontQir7HNpwXPPR8EPWW\\\\\\\/2EK6kwQ\\\\\\\/hSUxB6ZKP3qADe1wtFf7WBiqAWoGPU3ZjOjjYqTuqYJYBC3f28P0h\\\\\\\/avwiH9\\\\\\\/2GlBhSfei7SvVD1sVwh\\\\\\\/7vnMBgyX9XBIoLO6161D8eSV+3t0Fec8NjxPYcFQvfSPhFoAeJaDC8LB7Grb0O+vqSg8QX9ae0CaM3K8KQSSZsBGnYq9Pg\\\\\\\/tSCOz9DOrK99kLX2Luds0IwX4nEJYYRp+9OISvkppESfbg\\\\\\\/Z8s\\\\\\\/4k2hbMmgDtShJ6MIsO+4e17zdskmWT8oT+0ouZopdC0clF8LXBjsUJZ2u6SpHZUVl8WT4C2DVAHCCajIRHV9641RuGtt2\\\\\\\/0SKu12hossUxmpBLzOBKMF11vRr5fiDNJFsL06\\\\\\\/Y1Ltx\\\\\\\/fvvKjKHrTQp236SmnzB27YyMfe6x7UhRfBeQJMUaMCXXOsjOF6sHX86KUbQ3CZjDdXuP4X8TbZ4ILaLgd\\\\\\\/tx95UD25CdXSlgC9hU2F9ouBl+R3jRea7sXCSITnrlOH0PlVjVetzC7rMW8qnUemAoMAPa4UEx0qs0wVW65kUYNE5HcY2+HHQOvjeS1JRUXMmaVqH6iOylO\\\\\\\/u5X5DcRgxhIODL6XbNHc+z2DaabBQt9L3txVB2mDwg9r8bWyW3j3fVzQe7ENUAna6Iq2pjTufTrBnGFXQrhnTgVEZMQcbTe6IwfZLaxsYHabIsytq3LWYZ5N5YOZzAwbyDiXIkQ\\\\\\\/Yaz7706N9rt6qNQg4NHogU5MMMz9Tqv4urMPe8XT0rM\\\\\\\/FpIJPhhRRuoMvzYBcMs4Zfq95SmyuUGbcEe9+7ko41yr+UUEB\\\\\\\/yPhkdhNn13lwMk2Iu0alsx2BU3upZbRWpLJQoSUgxSNI6VDC20gZtf2FKOcoAWtr+F1anbgioDpa6Uk81aP4Pkt9FExxkNW43SSloxKIT8kgD78z3UGBrEEaeVe7mZCzY38N9lDGnzLqzSqpfXDbvKAkbyTG3o+kRaL0jRNPW97ibdqb0TRPdtbCcEa\\\\\\\/eRQ2mEtym08NXVa0pvu4XNruDXRZkWURn\\\\\\\/oRJg7rOexPQCrafyThD4ZODob7wARqALecGdBWv\\\\\\\/1F554ueMqhu8N5Xxuvg97I6kRjrgLGs5zP1Xy3GzZP53illiNx3Ow54vt+UOkxgpLngJW+kT\\\\\\\/ANApSdj3B1CXb\\\\\\\/mAhHCZaAZas9Ogn2l5KuaQ5EhBd3edWlOr0UiYLWylkKTr6z9TKiGwBn6UivCMO9WUb8KzDC1iX+ClSjDmkrmTFOkROPK5B2eIJARahpPqCzIsq86o+JRNYyZSy0OJ5LWGjvsyxBoBNPDFZolrzURQSXtm1qzWEDJ+ZL51u3aFnOHonnbWPMjtCpJ4QO\\\\\\\/4EewVcmAURjpU36qLH5IhNLhlMorTTYdtLAmMX8nlAv\\\\\\\/woSA7BO2qcYv9+s217XYHMMHDao9V\\\\\\\/QPd7PCrCxIKHfGvmzYRlxHTS2l24ytlJhaUIGsoHR5NTDDs6y3Nw1epz8OEO5a1Uy+7X30hGs6Ded2GTzUnWnKnYD0ytousoMoFBHxQ3aR6Y4KDAXBsz1KI38AmWmiKopslPGqO8MmW62WvX3k6NDJkhBI1N32qz0wCnlQsa4wpVXGFGlBH5VBo4WTLMm7qeGqo5b4HDsL7OP9ust639FILuHC3M+y3yKx9MRbuxP6mtKceTuR+HBv82wftE7pZV2OPOGciXTYp5Nd2SphBV+YmmWa4mC2cTrqPtjWkwJ5u60pBCeHcgXJ+YGRgFor81pk5x8rfEzn+SiuKh+1nGqlm1y11vECPqiOEyRlaoyLLws9scm1DMf7nT2thEFOxfGBpd3QO0LZBiYxEhaTLwroyBHTB6RquLsOiY2+vfntOm8kX9d6ZHtaHdOTvZXC\\\\\\\/bZLVv+5+VdWjKk3Hh4iEJgVH6hlczsy9o2qDsy\\\\\\\/wGa9egGeJ57QXH0kO5UI4oJp\\\\\\\/NpAzPy6OwVQxy1dcdmThJuyF97OVxL8lIib1n6pLCLuPZwxRxr8Lkmo81gWi\\\\\\\/4ADsXM54aP5g7CqAkZdWVmSfclpxCX5d88rfYt5lZzMibm\\\\\\\/j8GpbPbZ6KbLkYCT1lJU6565U0bcmeNG7SVDGIWyWU4Ii2R04J8nhGTSkr0UHYh9lKezmNSN+PPIW24vSZk2VEJNfWns+CTztF9YRysFA7R7fYO28qpLkljbgzkhmk+R3nRFNoit6QEoQZYqHG2GbzWnbRyJIbHZfpiKJGUpx4BlK+5qtmEzIE6SakmifZSz1SrXY45MB0sKWxd8CjCWKUkKQavKnMkHdjMdOs2SOE\\\\\\\/If3pOuQmQhDA25Ot7bJoa6CSDgaqiLFB4I0WeXp4Yr2OCTdN6drzd3DMwcvyXaPOEt9jOmpgX\\\\\\\/IN\\\\\\\/blkPBNJ2S1YqTKAN\\\\\\\/ijoKNyWHHn1Tcc0aTO9GPRU7QNRZ2+ITOAH10lSLUc3pnFyc0prxsnwZkLc+Rj1IpTwrP\\\\\\\/TVvzCh9mf+5wTL+tCC8N37QskKwlQZv5riyc+ooP+rb6IIgCGAsAruyVWAmeIvJYScathej3z0aH1JASO4qLiVHKYYFeG6+A7sAQ+BaaWFhr8LqnNBZnPvqFXLEJ2aWjHJ5iBSqkrqwFQNn0e2mJJeznFGne2Ks7znhX1fthlLItQgwdX0Amr7VZhuoREoX\\\\\\\/TIm\\\\\\\/mJV8uk\\\\\\\/Cg0\\\\\\\/6mfZFGgLbDeYEZYkyB\\\\\\\/SpaqTiG5knuV2lhp\\\\\\\/nlgyBJjxSVKJL7HsfUp85kKfGXdTUCLwmG1p1xfiDqEVV0LHoNtKWejg7gaZ1AnwTmiPAsqFdSOiDMrrUKfTz+QeKrPgZWmOGjgnDRvaxiPNutRSgKVgoqWmUwVYGGLwPgidDZcN7RGG99J5jRiES8nWNrhHhBQ1dppUNUJGnydLlZ7kh2XqlpFC8I8fvLJ5BS6A5hGKHY3276Kuvn0K8ZGf05AO3rmZQDYfoOtsqQLq9njZt2CzxbTV+H\\\\\\\/F5QH9V251CbTVCrFg33sEqoSHlVV55royZwm4QMyVlwdBIuDFl0NUTtrdHB0odpHmWFdl0b4gJYr3Nl9t0p066NVk3u+NxSjoip8+Rt2lZ5TStPrAXWUqMGD09fyx+RKpfbukcp\\\\\\\/ZS9lzebW6bhdti164pvBvr4T9EEbACE2QMZWmjS3hQyBhWa4nMltMgDnn7yOOAGE4gwt0zaotPZydCDZnt5dLp9jYrOQ93rF7Skw2cS78wZUd+FKKHkn0y8KHhOFqWncFFtca6JcWvIXmO\\\\\\\/UxdcCQM3Q9hjF1qYpFP8\\\\\\\/IfNiax5eNAR9rjUiSImahrJ+eqLkUF8Q414d6i0wxp0fzmaoDW3HU1IGgAFoOPSFShsRKixijYeKoYiAgQdBTPgSGGc7YToV5CxjFplljYL8Xxww4HRXCsT05wQkj9S3sL1OYrZma+gsDalVoPUcda3akZmRFUBtGbDj\\\\\\\/a7NMRgdplo51wemg079IOJNZin6TY5pg6HCbe7FgcLpo9za13rHHy+FGs5b4JGal4hHVjzhuUW6l72ySKH63We9In7PF6UXmfv\\\\\\\/z+vFoLjjRbyNJ6dC+9oWlb4xbyA4Tv5V+WIWsqQXj2rSn\\\\\\\/CLtBYFj5cQvIr14iuUGPaZGxITPw1un2tBE8\\\\\\\/4eaBlQKuCbMz+WVT\\\\\\\/81mv9Cbqi+6rE2CEu7g37GHD0+ADsFW8UsujOZDd0ntF+yBUNp84PxdeFXgeWJwZGAZcf4Pj5aNIKBrfS2Za\\\\\\\/RARP1OmTHSMiSX65ZmO3VCRg2T5P\\\\\\\/yTF4zEJN3zYGfjhJpiiVmsLfHMf+TWgDEMannp9hA4eiYpDyLpewVjuSlpliHyfZCUbC6R50gSTOOjf3IDpDMREL8Jph7cVagXQiRwCJ0S4y6fIZ6m7XSYiwVMnBqrqi+bWeZ2+Z+eKArS6ofVb7CKiW2ZN+hkH6ovwUF1NrRd0oeOvPlbpqHzJgdnGylWK0cxE4NddVJ2FhR8qUo7Y\\\\\\\/1A+3uIfhFBSlLIFU24EvAD1ZYh3GFgaKUadSp0StHFHy6laTBIZ19R1Ds+iw5SMimO\\\\\\\/nAJIVAjLXO2ML+IIaWFVaLeu9iows\\\\\\\/ze+jMgPeL5wrdaTAqojEBJCQzV\\\\\\\/rLxeXoRuBMo25egK9JAvlvegb1Nc4Ur4LXKuZA4WNlN4GChHmt5a5yLz8Jy238cuqulytyQg8ynSY4qmUMOkg2xfczxIsCtR04UPkyF7r6pkEjRDwg91bvk9LqP7Os3GRmNu8wVaYgonVfbXbYFs3Y36QedD7kifiBck0Qye\\\\\\\/TvDlbQTri0dtjb0Wl8kn0QqT2\\\\\\\/RLAHCCht7DncWnZdJ0J5qt6f21ptR7mJ8yKtV9Qst7vFPvKbfi3fSR8xZ++g+AlRzN+pO13Tr5QZ\\\\\\\/fSfrAl4UM7Icku+zvGQwsq4OynU+voDx1FwdZO5pKmNc7do2mRTDlGQ4ZepoQ2pyKmiMkWrKQmqlhHBFMgaDZA3aQIa6qu6dZx75oBqEgudLzRDFUszhOkqYrKUW6eiRHAeWC4oYF3CxYzy2pDRkbofiyxyDeS+NOdlkzb37cutPndmzLXfsqdppU08Cu3m1hSrRDcvNTVydbAFr2ZrEK\\\\\\\/zKSmBhJMEtaC0bkkTulqqbqN4PwrgOsSx78Qlk\\\\\\\/+ipnSQBg7bDdWSxkZo2WqV2GxGwNPLR6Ob8BLBqbcgafjBGzwz3it3QNJWQj3TiUm0KPEAOPU6sadPKNSoIdCmZCfwif80JVC+aZuzu539pPAKX8Yj1xMFWockwaL3DP+dNOiSCjyRQVH+XNWoAhZ2VFvwXoxKSPeIDazJxdnk7\\\\\\\/EUsjOSbJUMTigo7DpgkP6xHjSPyr9BYFbQ8tzQAc8\\\\\\\/T5c3I276BJsXPfa+znYM1xTP4IBvZxsIyW1Tv4f0sUEbJTy7YImphsBK0uMmIYp3oXXJkshiq5Yz23SpxsiDoYJAAP6+NbF7NQoEdEKycoFuv7a1oRqB1P6dzVv0YTgMWidUzamcTSSnhGQ9jKgUHDXwIqW+2OYtqqtCfV334QuByHKkoSCs5fmXuJVNwNzk4bVKprBto03ZBtNfOn\\\\\\\/tFKr6XFUnQ5Q45InR06CX2RMKbRPtb58ZSTBaOtvqg8zxscZ2\\\\\\\/ensOck7EN0XctHL+L2IWsA39edu9oSB3YeAwCjgbtClp2RtqyOBzA2mEWb9i8zwq4mazYDefkK3wmnW5CwGPgO14eL1jcHk3nbJZooHmdzu4wbro5Dv7l\\\\\\\/7B690VnD7T71aSGdmv5daR8y0bRVltpqVFufBTFeWdZbhQ9ASIkLHFkARFuXwU8vGLviJYGliUxPA+K\\\\\\\/Ep7lBsXvIYg749hz+YL3LMznhXSG3mKZerIfbHZMoTXbdZWAdGmQQIu+VCPI+J8dS7dpk2G0vcRP4QA5e2KU7cJUgMi3LEx6\\\\\\\/MH0QKPl44GQu4vAY5qrI0v5ljpVKNvigNHb8ip2zOd0oFZz9BVaeSB5HL9NKTstJs077GG+WryGpYtBDOOeZB3k8tMeEb+PtzJJj529RLNrhVTpUGd42D5BRUsKlJlv0yKyw5bhZRCJpKHS0latnfPNEZ7jn5BAA0+jvloDv6m0z08vjhHgjUnj4X71Nfu2G9KWnZeGljPgO3q7QKx0Co3Z00yBI+60p8KYwoChOn9yqvHXO9WmJoeRcM31gAf4M02kyKOij6VUITAk7d42qklyUvlCU92TeWeGUtVoypjebaj9n793v0wwtLUpyHmaz47EjJjGQRH\\\\\\\/gYcAXOYDpptFelSwrhDB3yKPZN1zrKQqrGNUkdULhzdABOCpilVY2gZnOQw3WC5tigISDnYRl2MPk5RA6LwJXkJjMYhX+IWFO350V\\\\\\\/1dCYYmg2d2a4TKReSbOML0i9J2Ci0bdKhlxF1C5lp2jyd01nM6hyU4C7AxA5pBLlNlUsG04akMuXhTaG46pGpNlJqdT8u4ZSM4+yofavPZ6DYHkyBHObVyV0A2uyqaudGD958QhSU9sKT1WFsvZ4bTjxGkAbLMcBiRJEsYJDOtBRBnFEYw8IM1+HLs0gv1kJdzs8te3IBcfOSIUzkmXGvGQJvYcaa+dUfAtiuU2Y41BZ0vtbSudyzmBxJZ\\\\\\\/Y4FVEP7Di1UdfB5+pmqQDWzmq2AIMNH8mB8FjZot+6GRxbR\\\\\\\/0lrWoZrdvTd\\\\\\\/egVdzX6ZbGUy3yuWvBh+qHVlZ+T+Bqw3BwTngEoLz\\\\\\\/VYdgHFwBq0UBGjlV2xmMkTbJEz6cfYd9DkpACpPG46UZNl5UmVNDF34BxA5RZbu7o0xU0JP4LAPZyvFhyj1UeIdT236x9VrVRDRfCfyBF\\\\\\\/y4ZdcohA5F+jKFBb4F8iOEja7gNJgxiuEynF9nSQtWJjy3ATrYede\\\\\\\/7CearSSDWsTUuyVCc1fY5cRaZrrmES2lUhaCxkjv+xkR8lsLPN2FMG98cbi7cDdS+VCxok\\\\\\\/m5dx84Cy41VYf6O2nFmmcmPRVwKnzs12nyyqDYyh3PkXSKBx6Z\\\\\\\/2VUvhANMwEumI6Z\\\\\\\/nH1dkNdlCZ5Wrbyuwgg0\\\\\\\/ft8C0BKhYv4NOEUQxMFI0a10u5IUWR+BwmMhMUrSilZo8DiF6pcStTOPNUNS7Vb997InnCOyHFs92bIHqNf8Y8fXFOBFJ1ryduS38qDPd7BkH0+iIVDZhsctFWMR1PrYcn+EZ2ybLtCFNJwRKSAISzFZPU6AtDWNHD+P2DssUxc5TeRtJNx64XNEhUyUQBoqXh\\\\\\\/7vD44XezDYuP+VTFaS33J\\\\\\\/l5S4Ozt1EuV06Wa92Y0v578DpLUfQ7oVqDm7l1AVsqP0o1IHSs1LioBSZS53HCFrE30g+R+MGw4yrDuQ6yN72PO9HhPlt8r9Iw4gut1a3iR32X3boz7NCpgyyBfRDEQN4gtWB2LWOvKxu1Of3dxtwrT8LFjsPRrd4LXmA5Q7mO\\\\\\\/VcKp5c0uii7I6LbFU6oKzRGHTMsLG4ybxdAC5tQmI87gNTbfhme1y41EK0uY3JiAhW27uK0ZHxCrxAh6OiiR9SeSEepa7QX5ShANGVyOD\\\\\\\/OvZua9uZceg92JSmZDPB8Gmw2rqT9IpWkptPBACPtZgqVlqaEEmvuupdlcqVZwQ0mTyZ54GCp8obR31HGHhDEle2z8Fperp6B3r4\\\\\\\/i9+n\\\\\\\/BCPaL9E8wN\\\\\\\/UBFf7tzvjPtKilFhQR0fkfUvdLuPNzb4zAg5JpDD3GZmnyrHJEKwDo4UoUooR9Qs\\\\\\\/8SYFwv6xsxPXcU61z1yiVxeKWuUvneIk7d6BujhaIL0HeDBJimPHelbxMd3rBuAPW9MB2rVAMHJPCITgJxRWwh3C3sBxFDMZ2ZK0rPQ4EHZgzs4vHdoj28RedSE8KUsL3owoza7HZDx6g4lN5Np0WsMe39h\\\\\\\/yOdZgzqG0+hWXnyhieh9g772PsihcVntZl7mbYCXqPjsHxHyajy0OoEunGbxfCjp86WYBLtRYni511TjjpybNw02Q+sIu7G8PHKsLqeFkzu1y1M9pY0PQma2Iw5plwU2cAJ6B7fr9\\\\\\\/6xPtJ4bIzcwa9rm\\\\\\\/YxNfWP55Y\\\\\\\/PIY24NuvsSOsD2LZJ0EINlxX3Fin8+YeDHj2EcXRc42T5CTxe7ol7ngk348T7O4czw8vb5eieBTfYNTTIFFJT6CR+oXAQVOcSYc8h1gy973UntPO6xqZiBVVv7tG2QLDRecDOvu\\\\\\\/FLoXADaUBQ232j1ZSj4PKigybrqQkJbauaBh8iVxtYtuyDLoXOJtj5bfmsxPKmjUxyCTx62EZFGVDw743LrM04xbN\\\\\\\/tXgcuuZWa2x+El1+j\\\\\\\/fWxBUG7h8WWQ0YmY34XCjP\\\\\\\/\\\\\\\/1nCmnrXOkTgjtEHWWwWtT\\\\\\\/H1fo4eoaggLoW0bxP41vU4Xf60TkQ+yIWcj58IGLU3xNt4EW9AuQ0\\\\\\\/GInnge0lmCOlszqSkoEIyZjkermiuTwB3dQ1XdpYgvj+fe2KTVyWwbWqN2hHpdVADmYu9JIhiEzzra52JppE5fqpPziPNpvEbEtMyRwnkHcZatflkgT9NI2TJ7lUQJTd4DzVXDnzGipXBxvTmxB4rdIdG9xtF3E671vWuVmFp\\\\\\\/LsrTO7bUwTNPfuIOiG+YFvZfGIAYWk9apQiUneHPW+ayXUaHYf+ITGCJD6Yc1jRMw0hpQdp+mYCiptaugZJu1pUw+ROBJukz2WHTS2nhQbnr1wsac2THFPaNsvGg0n68EGIlKAc6uhQhobYWYl8DlvjjsOM4MQdBF\\\\\\\/4enGDhBcCJcGt67TG9A8MkOZJeMP61V7jha+prLOLeSNv7dehqgxB88nEpI4nN1fOrEvMhi6mdJa42QUKEQrqhimkuZsyltVj0Zx6t8Wy2K3cN8MCQWkOQTTM4dD6Eg6WWErdiksZzwp\\\\\\\/MGyZf6KKOxzWbRnLQaohQPVQt03uXfCJxB6IW3AihJ3kP1x85I\\\\\\\/iQuA3u+XuMu2shdus9MGhhOxCoXlth1Y+jPyYU0dCDl19OP4PF7hsK4SOuyhKfG8SChou+RXo+3OAviAQLVovJYAg+r4Cn7ffTnHZcJYCyPc\\\\\\\/6Sekaap5yH3bbv2JxYJliSXXGu0tR1teqFkk8VeACdhCpGoSYS7v3kZf+ruQC39fkoq4QM6l6JkdmU8jq2NOFhSw+s\\\\\\\/LQ1v0PdH49tOmhOUgXTWBGnuKLPvBTHPYBEtpLdeN+A3oKcLyuYjqSti+fPD\\\\\\\/KdwDJZZbFeKAN4St47G85EoqUmQ05hgACtGpFiCUjYBYKjRuWly+OgbwMigHXHuE3WgQ2nM9Ud5VSSzfZ04BDNZToCucEW9qf9S2IWaB0F9x4MRPAEEchJD4ES66oD4yIr1g38lafZLehM7+6eJ45Y+9KjY7Fvv2h3LPgBCwGP+9LjLco9od6KZ6y9PFjLA5FfYWQVz6QecQldvpPlFEPTuk\\\\\\\/XHM2SDTdeur9mm5g2oY0aNOq+HfQ5JiA7zPxHApNL3LJA8ujqQWiJuw08O3fJdfHcaR4IOjvmQ8KNCV8Jg9cxyP4VQ1EL1F99m2RbbMjJqw0M2765msK3PHMlLMpX74WpkAxHBwJQnYEG\\\\\\\/VOOCd+ClbYmpYjy0\\\\\\\/CmAQheU06xhTy49aUeUEEgoB\\\\\\\/H4nqfzXwP8kd\\\\\\\/jIHCB0etaGRPVVIafjTOJuOKgsqT+1kVQXYIf2QOHQUa6wrr\\\\\\\/kSWTs9sFtvIQbqXvM9FaPmZLl0H1FTcTguBEVPwcYEfTMIMxtFrT6HCpZiS2UeYoTKshtUt6ugr\\\\\\\/LfEEAMtdiHtH3lAfq6GbwBdNcy0FlWjqlso7fKhmQT1jYnpzKxNiYh4LfDiUu2azDGAvqCrQtzhemTZLNnEIKsKgrnD1hbJgHgE7t6oy6M9DhvYznA\\\\\\\/\\\\\\\/PMrxwyBinmY2ElOBoPjrWFa8+dTTHcdCEMGieZudM6jAUCnRmST4yENId61\\\\\\\/pwhhnXPo+KzupkX4DI73wqCDdZx26\\\\\\\/LZPM0koLN8BXqSGGcnUHGrMFXkkmFmh83Nx1lnCuu4hro9Vwi+wk5OOs+CZ\\\\\\\/P\\\\\\\/eo3fi7B20UqAWQ\\\\\\\/\\\\\\\/dtd\\\\\\\/2d+27mA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hn00x+QRvPotAKr5oDKGJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022767b05cbfe2372d6923cfdef431d53cd85309ab4297a10446caf1add7a9c9254\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TsAmViWWR5RHxMv0ew57ityhTbTwQYGWTHvIZwHqvTlyrMgYQdoQNKmwIjrAAaNgUas\\\\\\\/Y2VToW+DPmn\\\\\\\/KSHTaOgv8m8n7sZ+rKD8nf\\\\\\\/rBARQoK1t3tTKsj\\\\\\\/U5UIVn7Lv1ezjNgcBarVt\\\\\\\/JokXvSjb+UM8WBqPpQpWnYXATXSzued1fH9pFHlPOViuChrHAs1mc2LKUM4ZjGV3ZO3kOV7E6sZmdO+P1N5gxBNDItpAQhoABNCpnOQIf9YG41FN2C4IO1aQ3eOQaPu8MzWLfIIDve06doeFP6fR3xO38EJGu9FZDrqbo7AOXlovjmnUsG4qPFL0vbiKbjT+Xtu25mUqg8FSPd+LwFoQbhIUq6iFEP4FiswpSvAYQ4F2fyrB1qvVqRCdZKxMZm5widnLS1FfONPWukKanVZDdVXzh4ZiKcA4zRt5Skf2QqEcFZSmCalAn0CStwygqU\\\\\\\/G749murgQjioxcqdnMArdldb8+aEn7sxtuOuoKPadm290VE2ehcfhMBN5m9gzHDyrJ\\\\\\\/8nk2SpOyAZbWkPmABQEPM6mFhh6QlLFmzyCS4Wnld261Tc8TqksfSYTAhBVhqhrbT9f8cOE6gkFiHnF2PZ0KzmE3MEuwFr5AEUIwhHTqZAGl6XvIe1nME22EKXoZzB9Ff1xblPYzpr559Ru9a5zRKSHlFBun3AX8lwCE1rzJzjHEhYoMhqoQSEYCdM\\\\\\\/GRI9Gk\\\\\\\/Pijw1\\\\\\\/T\\\\\\\/W22i2Dfj\\\\\\\/DJ9ChWNLmHUdK6PuHWHnbS1BS6Uekgptlr7PpyFYOW5f9iocIK4AIXNVl7+TwDfn3oLxGnwSQPzWwZNtLGnjZRCUhYkRYt3Xhpc4A9iZtsIBAaZiMRMKT4XAXMbEVwyzmwOt4NUJVARv50UAmK2b4W2PU20JWUmI709trFOUY9Di51Vxt0WfFoJj1mXvcV0GCUT85XSMvfnmJ23KvHPTlhAbVQruNXKM+OO3gsjpZjiBVIi8vYVLB\\\\\\\/xf0dB9\\\\\\\/Jt903nnz23sW7pZRdqxwAEzQ0vVIZ+YB3hdDe5vgOjNSrGBd4+cJnNrfdUfGckUwRsKVFDnU04QHH1gsc0hCzlqasYk190pC1kjsXDR21fcY7q3OoVJfU\\\\\\\/Ui+nTrfMvIr\\\\\\\/TLQdYMn\\\\\\\/bEtHcMRoXs5cDF3Z9ecrfRb+2n6UgvpQL\\\\\\\/VX4PPRqtdI9SWBEnuBnGSgA72osdAzlR5gzcP4TlsxX14hErcBHn+pe4mnNHlumHSVcKt4tButnJFLvVyfcQ+\\\\\\\/rGYO4tBi\\\\\\\/XEVuH0Hp3Zhv1KbyDgn24CDaUOrTWCeSS1MEoiE0TTqM6CHy0klrjrUg1wZVgBtgOKbQPMq+tuVt\\\\\\\/3toZgYIAwMWphZQLaDdeFG01sr2QOoy1Mv4X9s7lhZ3Bt9pBiwv2GTngnomp0OPbDDH+0faHLv3wYY1o2xMe2aV5gpFf7Ckc+ljUewm12A3k\\\\\\\/D39bSG0Cyo+gQ5E+CYjv+2Nnf+0BUg8o6bQETZcbX4tgLx6wJ9xHwc8arTaid6B0H4bbERfayC8roIg+ehUdRmyXy9Tf2\\\\\\\/cLlbDLELVyDLVLA0PyNEyjUxn43AzDNE4Yi06n4yh5zzxFw47XDvNmTzWZ1XX\\\\\\\/XAdTq35\\\\\\\/dq7vxdg2ahKvLIvmsove9F8fiZS8KCMFqVNhSXUjnpBH57BY9EZzQ1kYbjim6NF8oXS9eUim9h+rvKSpIATn9WJBkhtLVjtpvDtpP8ng870JdrMaYr3vx+fVLkbPTheW1WdeldnAbNxHxkMWDXF+9CMu3KRcJbzytgcO2Jp5jxRpIsAVowjl+N7beHitU2lkqaeDP9huzSF91IUuy+7UUgUWGuhwawGcg+7DIMGLHJrlHrh8NZKuNh8LXbDgK\\\\\\\/1qAy03r6OZuGJY8QoAJ24l+dbzKjHLPO2ZZK59OkD8V8q0KMC\\\\\\\/P2+nPNKZfAgwjRWFx6iFkYoIb\\\\\\\/kiIEdJno74DsYdIX9JPOG49ltGeG47hzb0WB5GhFyl1EuIKmMq9tIkVrs4x5Xg2YTLjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022gi0rQFF9WeqU6pNYt3QJXQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227342bac2a988d371baee8873881bec636fec8aeddd57d8b369027bc16bc91bb9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459492 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1151" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0krLgoBe2jyT5KtG3-SWS5Q7Fi1qGuCjAisO2RdEEX4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1151" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EWvZWA6Q2mt4WhbTnBvfKz4i1lRBN6PPMTIUvpFevcA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1143" title="A Dynamic Database of Microarray-Characterized Cell Lines with Various Cytogenetic and Genomic Backgrounds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1165" title="Generalized Admixture Mapping for Complex Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459492" id="node459492" data-pisa="ggg;3/7/1151" data-pisa-master="ggg;g3.113.006122" data-apath="/ggg/3/7/1151.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Population Genetics of Inversion Breakpoint Regions in <em>Drosophila pseudoobscura</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andre G. Wallace</span>, <span class="highwire-citation-author" data-delta="1">Don Detweiler</span> and <span class="highwire-citation-author" data-delta="2">Stephen W. Schaeffer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1151-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006122 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459492" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andre G. Wallace </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andre%2BG.%2BWallace%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallace%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndre%2BG.%2BWallace%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Don Detweiler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Don%2BDetweiler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Detweiler%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADon%2BDetweiler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen W. Schaeffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BW.%2BSchaeffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaeffer%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BW.%2BSchaeffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sws4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">sws4@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/7/1151" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459492" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459492/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1151.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459492" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459492/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1151.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459492" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459492/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1151.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459492" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459492/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/7/1151.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459492" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459492/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Paracentric inversions in populations can have a profound effect on the pattern and organization of nucleotide variability along a chromosome. Regions near inversion breakpoints are expected to have greater levels of differentiation because of reduced genetic exchange between different gene arrangements whereas central regions in the inverted segments are predicted to have lower levels of nucleotide differentiation due to greater levels of genetic flux among different karyotypes. We used the inversion polymorphism on the third chromosome of <em>Drosophila pseudoobscura</em> to test these predictions with an analysis of nucleotide diversity of 18 genetic markers near and away from inversion breakpoints. We tested hypotheses about how the presence of different chromosomal arrangements affects the pattern and organization of nucleotide variation. Overall, markers in the distal segment of the chromosome had greater levels of nucleotide heterozygosity than markers within the proximal segment of the chromosome. In addition, our results rejected the hypothesis that the breakpoints of derived inversions will have lower levels of nucleotide variability than breakpoints of ancestral inversions, even when strains with gene conversion events were removed. High levels of linkage disequilibrium were observed within all 11 breakpoint regions as well as between the ends of most proximal and distal breakpoints. The central region of the chromosome had the greatest levels of linkage disequilibrium compared with the proximal and distal regions because this is the region that experiences the highest level of recombination suppression. These data do not fully support the idea that genetic exchange is the sole force that influences genetic variation on inverted chromosomes.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-pseudoobscura" class="hw-term hw-article-keyword hw-article-keyword-drosophila-pseudoobscura" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila pseudoobscura</em></a></li><li class="kwd"><a href="/keyword/chromosomal-inversion" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-inversion" rel="nofollow">chromosomal inversion</a></li><li class="kwd"><a href="/keyword/breakpoints" class="hw-term hw-article-keyword hw-article-keyword-breakpoints" rel="nofollow">breakpoints</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li></ul><p id="p-5">Drosophila chromosomal inversions that give rise to different arrangements of genes have a long history in population genetics and evolutionary genetics research. From their initial discovery as factors that suppress recombination (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sturtevant 1917</a>), inversion mutations that reordered genes in the genome became a powerful genetic marker to assay naturally occurring genetic variation. The ability to identify inversions with simple cytological analysis proved to be very important in these early studies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Painter 1934</a>). Collections of Drosophila species from natural populations revealed a wealth of gene arrangement polymorphisms and provided the first genetic based data to infer phylogenetic relationships (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dobzhansky and Sturtevant 1938</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wasserman 1982</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sperlich and Pfriem 1986</a>). The discovery of geographical gradients or clines of gene arrangement frequencies as well as seasonal cycling provided indirect evidence for selection acting on a genetic polymorphism. <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wright and Dobzhansky (1946)</a> used population cages of gene arrangement polymorphisms in <em>Drosophila pseudoobscura</em> to provide experimental evidence that selective effects could be replicated in the laboratory and that one could estimate the fitness differences among karyotypic genotypes. The ability of inverted chromosomes to suppress recombination (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sturtevant and Beadle 1936</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dobzhansky 1950</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kirkpatrick 2010</a>) has suggested that this source of variation may play a role in adaptation by maintaining particular combinations of genes together.</p><p id="p-6">Although chromosomal rearrangements have the potential to play a role in maintaining associations of alleles among multiple loci, inversion mutations can also shape nucleotide diversity within and outside of the inverted DNA segments in the absence of selection. Recombination plays an important role in homogenizing nucleotide variability between homologous chromosomes. New chromosomal inversion mutations are a potent isolating mechanism between homologous chromosomes that can lead to the differentiation of chromosomes with different gene orders. First, a newly inverted chromosome will capture a unique set of alleles based on the chromosome that was inverted initially. Second, new nucleotide mutations within arrangements are not likely to spread to different arrangements leading to differentiation of chromosomes. Recent theoretical studies have developed models to help understand how the presence of different gene arrangements can alter the pattern and organization of nucleotide variability along an inverted chromosome (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2000</a>). <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997)</a> examined how crossing over and gene conversion influence genetic exchange or flux among inverted chromosomes. Their analyses were performed using either a Poisson or a Counting model to determine the distribution of genetic exchange events along the chromosome. In the Poisson model, it is assumed that recombination events are independent while the Counting model assumes interference can occur among recombination events.</p><p id="p-7">The reduction of recombination in gene arrangement heterozygotes results from the production of unbalanced gametes (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Sturtevant and Beadle 1936</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Roberts 1976</a>). Not all crossing over events, however, produce unbalanced gametes either due to double crossovers or gene conversion. Gene-conversion events are cases in which small tracts of DNA, typically on the order of 300â400 nucleotides (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rozas and AguadÃ© 1990</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">1993</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">1994</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schaeffer and Anderson 2005</a>), are exchanged between chromosomes. This nonreciprocal transfer of genetic information can prove to be more influential as a mechanism of genetic exchange among inverted chromosomes than reciprocal exchange in the form of crossing over (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>). The <em>rosy</em> locus found within inverted segments of the third chromosome of <em>Drosophila melanogaster</em> was shown to have a higher rate of exchange among heterokarytypes initiated by gene conversion rather than crossing over (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chovnick 1973</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hilliker <em>et al.</em> 1994</a>). The Counting Model, which assumes crossovers interfere with one another, provided a better explanation for these observations by suggesting a one order of magnitude greater rate of recombination caused by gene conversion <em>vs.</em> crossing over within inverted regions (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>).</p><p id="p-8"><a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997)</a> showed that genetic exchange varies among the proximal, central, and distal regions of inverted chromosomes when a Counting model is assumed. Proximal regions (closer to the centromere) have lower levels of exchange than distal regions (nearer to the telomere). This predicts that proximal regions will diverge more among different gene arrangements than distal regions.</p><p id="p-9">The size of an inversion can also determine how effective genetic exchange can be in differentiation among gene arrangements. Gene conversion rates are expected to be constant across inverted regions, but viable cross over products can only occur if two exchanges happen within the inverted segment. As inverted segments increase in size, there is a greater probability for crossovers to reduce genetic differentiation with the central regions being the most likely region to be exchanged among arrangements (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Betran <em>et al.</em> 1997</a>; <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>). Gene conversion occurs at a constant rate across inversions of all sizes, but the short segment of DNA exchanged limits its power to reduce differentiation (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Betran <em>et al.</em> 1997</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Schaeffer and Anderson 2005</a>). This leads to the general prediction that regions closest to breakpoints are likely to be more diverged than central regions of inversions because genetic exchange is lowest near the breakpoints (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">2000</a>).</p><p id="p-10">The reduction of recombination is also expected to lead to high levels of linkage disequilibrium (LD) or nonrandom associations among nucleotide sites. Because new mutations occur on a single chromosome, they generate nonrandom associations with variation on the chromosome. Recombination will shuffle variation on the chromosome breaking up the newly formed nonrandom associations. Because inversions prevent recombination from shuffling nucleotide diversity, one would predict that there should be high levels of LD between the proximal and distal breakpoints. LD is proposed as a method to map human genetic disease genes. The presence of inversions can lead to high levels of LD without being associated with disease genes. This study examines the structure of LD in the presence of paracentric inversions.</p><p id="p-11"><a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a> explored the effects of chromosomal inversion polymorphisms on levels and patterns of nucleotide variability. Based on their models, they predicted that newer inversions will have lower nucleotide variability at inversion breakpoints than the central regions of the inverted segment while older inversions will have greater variability at the breakpoints than within the inverted segment.</p><p id="p-12"><em>Drosophila</em> species are ideal model systems to examine the predictions of <a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">2000</a>). The gene content is conserved among <em>Drosophila</em> species across the genus (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Muller 1940</a>), but the order of genes has been shuffled through the accumulation of fixed inversion mutations. As might be expected, populations of most <em>Drosophila</em> species are polymorphic for paracentric inversions providing the raw material for lineage specific gene orders (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Sperlich and Pfriem 1986</a>). For example, <em>D. pseudoobscura</em> populations have more than 30 different gene arrangements on its third chromosome that were created by a series of overlapping paracentric inversions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dobzhansky and Epling 1944</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Powell 1992</a>). Inversions in <em>D. pseudoobscura</em> provide an excellent system to determine how mechanisms of genetic exchange alter the pattern and organization of nucleotide variability and divergence within and among the regions of <em>Drosophila</em> gene arrangements. <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> (2003)</a> examined the pattern and organization of nucleotide diversity for eight genetic markers on the third chromosome. The data from the eight markers were used to reject part of the coadapted gene complex model for the maintenance of inversions originally proposed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dobzhansky (1949)</a>. Only one of the eight marker loci (vestigial) mapped near an inversion breakpoint in the <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> (2003)</a> study, while the other seven loci mapped well away from the breakpoints. To better test how genetic exchange on inverted chromosomes affects the pattern and organization of nucleotide diversity, we developed ten new genetic markers for the third chromosome that map near the proximal and distal inversion breakpoints of gene arrangements with high frequency and wide geographical distributions.</p><p id="p-13">The availability of the <em>Drosophila pseudoobscura</em> genomic sequence data has enabled us to examine patterns and levels of nucleotide variability associated with regions in and around inverted chromosomes (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Richards <em>et al.</em> 2005</a>). In this study, strains of <em>D. pseudoobscura</em> from several different inversion types (chromosomal arrangements) were sequenced. The recent study of <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wallace <em>et al.</em> (2011)</a> has inferred the ancestral relationships among these gene arrangements. Regions near inversion breakpoints along the third chromosome of <em>D. pseudoobscura</em> were sequenced in ancestral and derived gene arrangements to test predictions about breakpoint evolution (<a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>; <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> 2000</a>). First, estimates of nucleotide diversity were examined at each breakpoint region to see whether older inversions had higher levels of genetic variation at the breakpoints. Second, levels of variation in proximal regions, in the inverted regions, and in distal segments were estimated to test whether proximal regions are more divergent than distal regions as predicted by <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a>. Third, we derived two estimates of the neutral mutation parameter (4<em>N</em><sub>e</sub>Î¼), Ï and Î¸, among all regions to test for departures from the neutral theory of molecular evolution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kimura 1986</a>). Finally, we estimated intralocus and interlocus LD to determine whether proximal regions have greater levels of LD than distal regions because of the polarity of genetic exchanges between these two segments of the chromosome and to determine whether nucleotide variation in the proximal and distal breakpoints is in significant LD.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly strains and DNA extraction</h3><p id="p-14">Strains of <em>D. pseudoobscura</em> were collected in 1998 from four different populations in the southwestern United States by S. W. Schaeffer and W. W. Anderson (University of Georgia). The localities include: Davis Mountain, TX; James Reserve, CA; Mount Saint Helena, CA; and Kaibab National Forest, AZ (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> 2003</a>). The third chromosomes from the strains collected from these localities were extracted using either the Blade or Lobe Balancer strain (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dobzhansky and Queal 1938</a>). Genomic DNA was prepared from 141 isochromosomal lines of <em>Drosophila pseudoobscura</em> using a single-fly DNA extraction protocol (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gloor and Engels 1992</a>).</p></div><div id="sec-3" class="subsection"><h3>Polymerase chain reaction (PCR) primer design, amplification, and nucleotide sequencing</h3><p id="p-15">A total of 18 gene regions were sequenced in this study. We used eight gene regions that were previously sequenced by <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> (2003)</a>), engrailed (<em>en</em>), exuperantia 1 (<em>exu 1</em>), Myocyte enhancing factor 2 (<em>Mef 2</em>), even skipped (<em>eve</em>), amylase 1 (<em>Amy 1</em>), vestigial (<em>vg</em>), F6, and Ecdysone Receptor (<em>EcR</em>), and we used the primers and PCR conditions described in the study. These eight loci are evenly distributed across the <em>D. pseudoobscura</em> third chromosome. We designed PCR primers to amplify additional gene regions to be near the breakpoints of six polymorphic inversions using the third chromosome nucleotide sequence and cytogenetic map (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dobzhansky and Sturtevant 1938</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schaeffer <em>et al.</em> 2008</a>). The gene arrangement breakpoints examined in this study are Pikes Peak (PP), Santa Cruz (SC), Tree Line (TL), Standard (ST), Hypothetical (HY) and Chiricahua (CH) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dobzhansky 1944</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The vestigial locus (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> 2003</a>) was used as the marker for the distal ST to Arrowhead (AR) breakpoint because it is within 20 kb of the mapped AR distal breakpoint (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Richards <em>et al.</em> 2005</a>) requiring an additional 11 breakpoint markers to be developed. Because the SC and TL proximal breakpoint regions appear to be coincident on the cytological map (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dobzhansky 1944</a>), only one primer pair was designed to interrogate sequence evolution at both inversion breakpoints. Thus, a total of 10 new markers were developed to examine the evolution of breakpoints for the six gene arrangements of <em>D. pseudoobscura</em> that are frequent and widely distributed in this species (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Powell 1992</a>). We used the comparison of the <em>D. persimilis</em> and <em>D. pseudoobscura</em> genome sequences of the third chromosome as a guide for choosing breakpoint regions that were likely to be less constrained at the nucleotide sequence level (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Noor <em>et al.</em> 2007</a>). We named the regions with a letter to signify the proximal (p) or distal (d) breakpoint, a two-letter code for the ancestral gene arrangement, and a two-letter code for the derived gene arrangement (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/006122SIf.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS1.pdf">Table S1</a> with citation in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FileS2.pdf">File S2</a>). The PCR fragments amplified for the breakpoint regions were between 375 and 1413 bp in length. Five of the sequenced breakpoint regions, pHYST, dSTPP, dSTAR, dSCCH, and dHYST, included some coding sequences. Total PCR volume was 50 ÂµL which included 10 Ã polymerase buffer with MgCl<sub>2</sub>, both forward and reverse primers, deoxynucleotide triphosphate mix, Taq DNA polymerase, and the isolated target DNA. The 30-cycle PCR protocol included 2 minutes of initial denaturing at 95Âº, annealing at the temperatures specific to each primer pair (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) for 30 sec/kb, and extension at 72Âº for 1 min/kb. The amplification product was verified as the predicted length by running 5 ÂµL of the completed PCR on 2% agarose gel. Once the fragment size was verified, the completed PCR was treated with ExoSAP-IT (USB; Affymetrix Corporation) to remove contaminants such as unused deoxynucleotide triphosphates and primers remaining in the PCR mixture. Fragments with sequencing primer at a concentration of 1 Î¼M were then sequenced at the Penn State University Nucleic Acid Facility (University Park, PA) using an ABI Hitachi 3730XL DNA Analyzer. Each breakpoint region was sequenced in both forward and reverse directions. The forward and reverse sequence reads were assembled and conflicts between the two reads were resolved using the SEQMANII program (DNASTAR, Madison, WI).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of 18 genetic markers on the gene arrangements of Drosophila pseudoobscura. The inversion events that gave rise the seven gene arrangements (TL, SC, CH, HY, ST, PP, and AR) are indicated by the colored arrows. The thick colored lines trace the locations of the six pairs of inversion breakpoints on the seven D. pseudoobscura gene arrangement cytogenetic maps, where the breakpoints are colored according the events in the right hand figure. For example, thick red lines represent the PP arrangement. The thin black lines show the locations of the 18 genetic markers that were sequenced in this study. The homologous marker and breakpoint positions on each gene arrangement are connected by vertical/diagonal lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1831192352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Map of 18 genetic markers on the gene arrangements of &lt;em&gt;Drosophila pseudoobscura&lt;/em&gt;. The inversion events that gave rise the seven gene arrangements (TL, SC, CH, HY, ST, PP, and AR) are indicated by the colored arrows. The thick colored lines trace the locations of the six pairs of inversion breakpoints on the seven &lt;em&gt;D. pseudoobscura&lt;/em&gt; gene arrangement cytogenetic maps, where the breakpoints are colored according the events in the right hand figure. For example, thick red lines represent the PP arrangement. The thin black lines show the locations of the 18 genetic markers that were sequenced in this study. The homologous marker and breakpoint positions on each gene arrangement are connected by vertical/diagonal lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1151/F1.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/7/1151/F1.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462985" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Map of 18 genetic markers on the gene arrangements of <em>Drosophila pseudoobscura</em>. The inversion events that gave rise the seven gene arrangements (TL, SC, CH, HY, ST, PP, and AR) are indicated by the colored arrows. The thick colored lines trace the locations of the six pairs of inversion breakpoints on the seven <em>D. pseudoobscura</em> gene arrangement cytogenetic maps, where the breakpoints are colored according the events in the right hand figure. For example, thick red lines represent the PP arrangement. The thin black lines show the locations of the 18 genetic markers that were sequenced in this study. The homologous marker and breakpoint positions on each gene arrangement are connected by vertical/diagonal lines.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462997/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462997/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mean estimates of heterozygosity (Ï Â± SD) within- and between-gene arrangements in proximal, inverted, and distal regions of the third chromosome of <em>D. pseudoobscura</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-18">Seventy five strains had all 18 genetic markers sequenced, which enabled us to concatenate the data for LD analyses. Sixty-six strains were sequenced for at least one genetic marker. The sequences of 18 loci were extracted from Illumina next-generation sequencing reads for two additional strains, one inbred strain collected from Mather, CA that carried the TL arrangement and a second strain from the outgroup strain <em>D. miranda</em> SP138 (data kindly provided by M. A. F. Noor, Duke University). Thus, the number of strains sequenced for any of the 18 genetic markers varied from 86 to 144.</p></div><div id="sec-4" class="subsection"><h3>Nucleotide sequence alignment</h3><p id="p-19">Finished nucleotide sequences for each of the 18 genetic markers were aligned using the MEGALIGN program (DNASTAR, Madison, WI). Alignments were performed manually and inspected to insure that indels were scored consistently among the different sequences.</p></div><div id="sec-5" class="subsection"><h3>Nucleotide polymorphism and divergence at <em>D. pseudoobscura</em> breakpoint regions</h3><p id="p-20">We used the aligned sequences at the 18 markers to examine the predictions of the <a id="xref-ref-27-10" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">2000</a>) models by estimating levels of genetic variation within each gene arrangement and among arrangements. DnaSP was used for these population genomic analyses (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Librado and Rozas 2009</a>). The input aligned sequence files were updated within DnaSP to identify coding regions in the alignment based on the annotated <em>D. pseudoobscura</em> genome (version R2.27). In addition, strains were placed into a variety of subsets based on single gene arrangements or combinations of two gene arrangements. All nexus format data files used in these analyses are available on the ScholarSphere web site (<a href="http://ScholarSphere.psu.edu">http://ScholarSphere.psu.edu</a>). DnaSP was used to estimate nucleotide variability within all sequences at each locus. We tested the predictions of <a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a> by estimating how breakpoint position influences levels of nucleotide variability within and among five gene arrangements (AR, PP, CH, ST, and TL) with estimates of nucleotide heterozygosity based on the number of segregating sites (Îw) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Watterson 1975</a>) and the number of pairwise differences (Ï) in silent sites (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Tajima 1983</a>). We estimated the variance of Îw based on a model of no recombination (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Tajima 1983</a>). Nucleotide heterozygosity was estimated within the derived and ancestral arrangements as well as across the two arrangements.</p><p id="p-21">Estimates of nucleotide diversity used the <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jukes and Cantor (1969)</a> multiple hits correction. <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Wallace <em>et al.</em> (2011)</a> has shown that the HY chromosome is the ancestral gene arrangement (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Therefore, we polarized the inversion events on the <em>D. pseudoobscura</em> third chromosome phylogeny based on the HY chromosome being the common ancestor of all other arrangements. The estimated age (95% confidence limits) of the gene arrangements are: 2.01 (1.81â2.23) million years for HY, 1.38 (1.19â1.42) million years for SC and ST, 1.22 (1.09â1.34) million years for TL, 0.99 (0.88â1.10) million years for PP, 0.58 (0.51â0.65) million years for AR, and 0.51 (0.45â0.57) million years for CH. The confidence limits for the two oldest arrangements ST and TL overlap, but we assume that the TL is oldest extant arrangement based on migration-selection models (see page 3083 in <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schaeffer 2008</a>). The ages of the arrangements are critical for testing the models of <a id="xref-ref-27-11" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">2000</a>) where levels of variation in ancestral and derived inversions are key features of the model. The relative ages of the inversions are also necessary for coalescent models (see <em>Tests of an equilibrium neutral model</em> below). Nucleotide variability (Îw and Ï) was estimated across the gene markers of the third chromosome in ancestral and derived arrangements to determine whether diversity was greater at the breakpoints compared to central regions (<a id="xref-ref-27-12" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>) and to determine whether derived inversions had lower levels of diversity than ancestral arrangements but increased in the sample of ancestral and derived arrangements (<a id="xref-ref-26-10" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> 2000</a>). Using <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> as a guide, we ordered genes according to the derived arrangement in plots of heterozygosity along the chromosome.</p></div><div id="sec-6" class="subsection"><h3>Tests of an equilibrium neutral model</h3><p id="p-22">The Tajimaâs <em>D</em> statistic was used to test for departures from an equilibrium neutral theory of molecular evolution (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tajima 1989</a>). We estimated values for Tajimaâs <em>D</em> across all sequences at each locus and also within individual gene arrangements. In addition, the Hudson Kreitman, and Aguade (HKA) test (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hudson <em>et al.</em> 1987</a>) was used to test for departures from a neutral model. We used the software written by Jody Hey (Rutgers University (<a href="http://genfaculty.rutgers.edu/hey/software">http://genfaculty.rutgers.edu/hey/software</a>), which allows a multiple locus test with varying sample sizes.</p><p id="p-23">We used coalescent models with nested subsamples to test whether nucleotide diversity estimates within arrangements reject a neutral model (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hudson and Kaplan 1986</a>; <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> 2003</a>). Each genetic marker had different numbers of the six arrangements ranked from youngest to oldest (f<sub>CH</sub>, f<sub>AR</sub>, f<sub>PP</sub>, f<sub>ST</sub>, f<sub>TL</sub>, f<sub>SC</sub>), where f<sub>CH</sub>, f<sub>AR</sub>, f<sub>PP</sub>, f<sub>ST</sub>, f<sub>TL</sub>, and f<sub>SC</sub> are the numbers of CH, AR, PP, ST, TL, and SC chromosomes, respectively. These coalescent simulations generated topologies and coalescent times for 5000 individuals using standard approaches (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hudson 1990</a>); however, inversion and nucleotide mutations were generated on the branches of the tree rather than just nucleotide changes. First, inversion mutations were placed on the tree, then we counted the number of gene arrangements (<em>n</em><sub>i</sub>) and their rank-ordered frequencies in the terminal descendants (f<sub>1</sub>, f<sub>2</sub>, f<sub>3</sub>, f<sub>4</sub>, f<sub>5</sub>, f<sub>6</sub>), where f<sub>1</sub> is the least frequent and f<sub>6</sub> is the most frequent. In a neutral genealogy, the frequency of alleles will positively relate with age (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Toomajian <em>et al.</em> 2003</a>). Thus, a genealogy was accepted if the tree had an <em>n</em><sub>i</sub> = 6 and if the rank-ordered frequencies were greater than or equal to the observed age-ordered frequencies (f<sub>1</sub>â¥ f<sub>CH</sub>, f<sub>2</sub>â¥ f<sub>AR</sub>, f<sub>3</sub>â¥ f<sub>PP</sub>, f<sub>4</sub>â¥ f<sub>ST</sub>, f<sub>5</sub>â¥ f<sub>TL</sub>, f<sub>6</sub>â¥ f<sub>SC</sub>). Nucleotide mutations were added to an accepted genealogy according to standard coalescent methods (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hudson 1990</a>). Alleles were subsampled within each arrangement class up to the observed sample sizes of the different chromosomes (f<sub>CH</sub>, f<sub>AR</sub>, f<sub>PP</sub>, f<sub>ST</sub>, f<sub>TL</sub>, f<sub>SC</sub>). We estimated heterozygosity [Î¸<sub>w</sub> (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Watterson 1975</a>); Ï (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Tajima 1983</a>)] within each arrangement and among all arrangements for each replicate genealogy. A total of 1000 random genealogies were sampled and a 95% confidence interval was inferred for the heterozygosity estimates within each arrangement and among all arrangements. These confidence limits were compared with the observed estimates of Îw and Ï to determine whether the observed values had more extreme values.</p><p id="p-24">There is evidence that populations of <em>D. pseudoobscura</em> are exponentially growing at a modest rate (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schaeffer 2002</a>). Thus, we used a coalescent model that incorporates exponential growth with rate parameter Î± = 7 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Slatkin and Hudson 1991</a>). The methods for this coalescent were the same as above except that the coalescent times were determined for a model of exponential growth with rate parameter Î± = 7.</p></div><div id="sec-7" class="subsection"><h3>LD analysis</h3><p id="p-25">Fisherâs exact test (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sokal and Rohlf 1981</a>) was used to test pairs of variable sites within and between third chromosomal loci for significant LD. We concatenated the aligned sequences from the 11 breakpoint regions and seven gene regions of 76 strains to test for intra- and interlocus LD. Only comparisons were performed of sites capable of generating a significant result with the Fisherâs exact test (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lewontin 1995</a>). We used the Q-value approach with a false-discovery rate of 1% to overcome the multiple comparison problem (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Storey 2002</a>). We estimated the fraction of valid tests of LD that showed significant nonrandom association within and between each region. We plotted these values as a heat map to show regions with low <em>vs.</em> high levels of LD. The numbers of significant associations within and between loci were tested for departures from homogeneity using a Ï<sup>2</sup> test. We also tested the nucleotide sites for significant associations with the five major gene arrangements. A separate Q analysis with a false-discovery rate of 1% was performed for the gene arrangement specific association tests. We estimated the fraction of valid tests of LD that were significant between each region and the gene arrangement type. We plotted these values as a heat map tested for the numbers of significant tests for departures from homogeneity using a Ï<sup>2</sup> test.</p><p id="p-26">We also tested whether the LD increases as gene arrangements age. In the ST phylad, the PP arrangement is older than the AR arrangement. We asked whether an older arrangement accumulates more LD due to new mutations and reduced genetic exchange. We tested this hypothesis by creating two subsets of gene arrangement sequences. The first subset includes ST and AR arrangements representing LD for an ancestral and recent arrangement. The second subset includes ST and PP arrangements representing LD for an ancestral and an older arrangement. Each subset of data were tested for LD separately and the resulting probabilities from the two analyses were combined to determine an overall significance cutoff for the multiple tests using the Q-value approach and a common false discovery rate of 1%.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Tests of the impact of heterogeneous recombination rates on nucleotide polymorphism and divergence on inverted chromosomes in <em>D. pseudoobscura</em></h3><p id="p-27">The history of the gene arrangements of the third chromosome of <em>D. pseudoobscura</em> reflect a complex history that is written in the observed pattern of nucleotide variation. The evolutionary history that is reflected in the sequences at breakpoints differs from that seen at non-breakpoint sequences. <a id="xref-ref-27-13" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997)</a> used the Poisson and Counting models of recombination to predict levels of genetic flux across an inverted chromosome. The Poisson model assumes that crossing over and gene conversion events are independent, whereas the Counting model allows for interference of genetic exchange events. Their models showed that recombination rates will be lower in the proximal <em>vs.</em> distal chromosomal segment (<a id="xref-ref-27-14" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>), which predicts that proximal regions will have greater levels of nucleotide differentiation <em>vs.</em> distal segments of the inverted chromosome. A summary of all estimates of heterozygosity Îw and Ï in silent sites (synonymous sites in coding sequence plus noncoding sites) for the 18 genetic markers within and among all arrangements is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS2.pdf">Table S2</a> with citations in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FileS2.pdf">File S2</a>.</p><p id="p-28">We examined this prediction by estimating heterozygosity in the proximal, inverted, and distal regions within the five gene arrangements based on the position of the 18 markers relative to the derived arrangement breakpoints (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). Proximal markers were between the centromere and the proximal breakpoint, inverted markers were between the two breakpoints, and the distal markers were between the distal breakpoint and the telomere. In addition, we examined heterozygosity among two pairs of gene arrangements to determine whether heterozygosity increases in these three regions as expected. Mean estimates of heterozygosity in the three regions within- and between-gene arrangements are shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Mean estimates of Ï are greater in distal regions than in proximal regions in all but the ST arrangement. The differences in heterozygosity, however, are not significantly different from each other, although four of the five arrangements show the same trend, despite different sequenced regions used in the estimates. The older arrangement (PP) shows greater levels of variation when pooled with its ancestral arrangement than does the younger arrangement (AR) as is expected. The mean nucleotide differentiation is greatest within the inverted regions and lowest in the proximal segments.</p><p id="p-29">These data fail to support the theoretical predications of <a id="xref-ref-27-15" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997)</a> with the proximal region showing less heterozygosity than the inverted or distal regions. This observation may suggest that proximal regions are subject to partial selective sweeps (<a id="xref-ref-27-16" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> 1997</a>, <a id="xref-ref-26-11" class="xref-bibr" href="#ref-26">2000</a>). Regions that experience lower levels of recombination are expected to harbor less variation as a consequence of recurrent selective sweeps (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Begun and Aquadro 1992</a>). These results should be viewed with caution given the small number of loci sampled in each region. With the advent of next generation sequencing technologies it will now be possible to do more thorough examination of overall trends and increase the number of regions sampled across the chromosome.</p></div><div id="sec-10" class="subsection"><h3>Selection removes and elevates variation on the <em>D. pseudoobscura</em> third chromosome</h3><p id="p-30">Geographic and altitudinal clines, seasonal cycling, and population cage experiments have been used as circumstantial evidence that selection acts on the gene arrangements of <em>Drosophila pseudoobscura</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dobzhansky 1943</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wright and Dobzhansky 1946</a>), but the nature of the selective regime has not been clear.</p><p id="p-31">We tested the frequency spectra of each region within the five arrangements as well as all arrangements for departures from an equilibrium neutral model using Tajimaâs <em>D</em> (1989) test statistic. Tajimaâs <em>D</em> estimates the difference between two heterozygosity values, one based on the number of nucleotide differences between pairs of sequences and a second based on the number of segregating sites. The Tajimaâs <em>D</em> will have significantly positive values when a region has an abundance of intermediate frequency variants perhaps indicating the action of some form of balancing selection. Alternatively, significant negative values indicate that a region has an excess of rare variants either because it is recovering from a selective sweep or may reflect a recent population expansion (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Innan and Stephan 2000</a>). One can discriminate between the selective and demographic hypotheses based on whether a few loci or a majority of loci in the genome have a negative <em>D</em>, respectively.</p><p id="p-32">The results of the Tajimaâs tests show a consistent trend toward negative values within and among gene arrangements with only six of the 108 potential tests showing a significant negative <em>D</em> (pSTPP in TL, pHYSC in ST, pSCCH in ST, dSTAR in CH, <em>en</em> in AR, <em>Mef2</em> in ST; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS2.pdf">Table S2</a>). Two breakpoint regions showed a significant excess of intermediate frequency variants (<em>D</em> &gt; 0) (pSCCH in PP, dSCCH in TL), whereas none of the nonbreakpoint regions had a <em>D</em> significantly greater than zero.</p><p id="p-33">The overall trend of negative Tajimaâs <em>D</em> values at a majority of loci in <em>D. pseudoobscura</em> supports a previous conclusion that the populations of <em>D. pseudoobscura</em> are expanding (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hamblin and Aquadro 1999</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Schaeffer 2002</a>; <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> 2003</a>). There were a number of breakpoint regions that had a higher percentage of positive Tajimaâs <em>D</em> values. These results suggest that long-term maintenance of multiple gene arrangements has raised the frequency of allelic variants near some breakpoints.</p><p id="p-34">We used coalescent simulations with nested subsamples as a second approach to test whether levels of nucleotide variation within each arrangement depart from an equilibrium neutral model. These analyses ranked the arrangements according to their relative ages. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows that the estimates of heterozygosity (Ï, based on pairwise differences) across all arrangements are uniformly lower than the mean for the coalescent analysis assuming constant population sizes. In contrast, estimates of heterozygosity based on the number of segregating sites generally were greater for all loci and in some cases were greater than the 95% confidence interval (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FigureS1.pdf">Figure S1</a>). This result is expected given that the majority of loci in <em>D. pseudoobscura</em> have a negative Tajimaâs <em>D</em>. All estimates of Ï for all loci within the five arrangements are within the 95% confidence limits derived from the coalescent analysis.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed vs. expected estimates of nucleotide heterozygosity (&#x3C0;) at 18 D. pseudoobscura marker loci based on a coalescent analysis of a population of constant size. Nucleotide heterozygosity estimates based on the number of pairwise differences (&#x3C0;) within five gene arrangements (AR, PP, CH, ST, and TL) as well as among all arrangements are shown in the individual panels. The observed (Obs) estimates of &#x3C0; as well as the mean (P(0.500) and the 95% confidence interval (P(0.025) to P(0.975)) derived from 1000 coalescent simulations using nested subsamples for a constant population size model. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1831192352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Observed &lt;em&gt;vs.&lt;/em&gt; expected estimates of nucleotide heterozygosity (&#x3C0;) at 18 &lt;em&gt;D. pseudoobscura&lt;/em&gt; marker loci based on a coalescent analysis of a population of constant size. Nucleotide heterozygosity estimates based on the number of pairwise differences (&#x3C0;) within five gene arrangements (AR, PP, CH, ST, and TL) as well as among all arrangements are shown in the individual panels. The observed (Obs) estimates of &#x3C0; as well as the mean (P(0.500) and the 95% confidence interval (P(0.025) to P(0.975)) derived from 1000 coalescent simulations using nested subsamples for a constant population size model. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1151/F2.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/7/1151/F2.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Observed <em>vs.</em> expected estimates of nucleotide heterozygosity (Ï) at 18 <em>D. pseudoobscura</em> marker loci based on a coalescent analysis of a population of constant size. Nucleotide heterozygosity estimates based on the number of pairwise differences (Ï) within five gene arrangements (AR, PP, CH, ST, and TL) as well as among all arrangements are shown in the individual panels. The observed (Obs) estimates of Ï as well as the mean (P(0.500) and the 95% confidence interval (P(0.025) to P(0.975)) derived from 1000 coalescent simulations using nested subsamples for a constant population size model. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the results of coalescent simulations assuming an exponentially growing population with a growth rate parameter of Î± = 7 (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Schaeffer 2002</a>). The observed estimates of Ï across all arrangements matched the expected values (<em>P</em> = 0.5) for the 18 regions examined in the coalescent simulations assuming an exponentially growing population. The younger arrangements tended to show more observed heterozygosity (Ï) estimates outside of the upper confidence limit (<em>P</em> = 0.975) than older arrangements (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Estimates of heterozygosity based on the number of segregating sites showed similar trends, but showed more loci with values outside the 95% confidence interval in younger arrangements (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FigureS2.pdf">Figure S2</a>)</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed vs. expected estimates of nucleotide heterozygosity (&#x398;w) at 18 D. pseudoobscura marker loci based on a coalescent model of an exponentially growing population. Nucleotide heterozygosity estimates based on the number of segregating sites within five gene arrangements (AR, PP, CH, ST, and TL) as well as among all arrangements are shown in the individual panels. The observed (Obs) estimates of &#x398;w as well as the mean (P(0.500) and the 95% confidence interval (P(0.025) to P(0.975)) derived from 1000 coalescent simulations using nested subsamples for an exponentially growing population with growth rate parameter &#x3B1; = 7. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement. The proximal region is to the left." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1831192352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Observed &lt;em&gt;vs.&lt;/em&gt; expected estimates of nucleotide heterozygosity (&#x398;w) at 18 &lt;em&gt;D. pseudoobscura&lt;/em&gt; marker loci based on a coalescent model of an exponentially growing population. Nucleotide heterozygosity estimates based on the number of segregating sites within five gene arrangements (AR, PP, CH, ST, and TL) as well as among all arrangements are shown in the individual panels. The observed (Obs) estimates of &#x398;w as well as the mean (&lt;em&gt;P&lt;/em&gt;(0.500) and the 95% confidence interval (P(0.025) to P(0.975)) derived from 1000 coalescent simulations using nested subsamples for an exponentially growing population with growth rate parameter &#x3B1; = 7. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement. The proximal region is to the left.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1151/F3.medium.gif" width="395" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/7/1151/F3.medium.gif" width="395" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462933" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Observed <em>vs.</em> expected estimates of nucleotide heterozygosity (Îw) at 18 <em>D. pseudoobscura</em> marker loci based on a coalescent model of an exponentially growing population. Nucleotide heterozygosity estimates based on the number of segregating sites within five gene arrangements (AR, PP, CH, ST, and TL) as well as among all arrangements are shown in the individual panels. The observed (Obs) estimates of Îw as well as the mean (<em>P</em>(0.500) and the 95% confidence interval (P(0.025) to P(0.975)) derived from 1000 coalescent simulations using nested subsamples for an exponentially growing population with growth rate parameter Î± = 7. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement. The proximal region is to the left.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Finally, we used the HKA test (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hudson <em>et al.</em> 1987</a>) test to determine whether levels of polymorphism within the five arrangements are correlated with divergence between <em>D. pseudoobscura</em> and the outgroup <em>D. miranda</em>. We tested the 18 gene regions for deficiencies or excesses of variation. Only the TL chromosome of the five gene arrangements rejected an equilibrium neutral model with the HKA test (Ï<sup>2</sup> = 20.91, <em>df</em> = 17, <em>P</em> = 0.047; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS5.pdf">Table S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS6.pdf">Table S6</a> for the nonsignificant results for the other arrangements). The main reason for the rejection is the lack of polymorphic sites in the dHYST region despite high levels of divergence in this region. Three other regions had a deficiency of polymorphism (dSTPP, eve, and pSCCH) and were major contributors to the rejection of the neutral model with the HKA test.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462965/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462965/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">HKA test for the TL gene arrangement</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Given the suggested excesses of polymorphism, it is interesting to consider the distribution of these polymorphisms among the five different arrangements. A unique polymorphism is one that is segregating in one arrangement and not the others. Alternatively, a shared polymorphism is one that is variable in two or more arrangements. One can use the outgroup species <em>D. miranda</em> to infer which of the polymorphic nucleotides is the derived mutation. The frequency of derived mutations tends to be greater in breakpoint than in nonbreakpoint regions, and older arrangements have accumulated larger numbers of fixed derived mutations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FigureS3.pdf">Figure S3</a>). Of the 758 polymorphic sites in the 18 genetic markers, there was a significant excess of sites that were unique to one arrangement (466 observed <em>vs.</em> 258.6 expected) based on the assumption that polymorphisms occur independently on the different arrangements (Ï<sup>2</sup> = 512.7, df = 31, <em>P</em> = 3.7 Ã 10<sup>â91</sup>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FileS1.pdf">File S1</a> for a description of the analysis and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FigureS4.pdf">Figure S4</a>). The youngest arrangement AR has a significant excess of unique polymorphisms compared with the four other arrangements (Ï<sup>2</sup> = 16.5, <em>df</em> = 4, <em>P</em> = 0.002). TL had an excess of sites (14 observed <em>vs.</em> 4.2 expected) that were uniquely fixed within this arrangement (Ï<sup>2</sup> = 30.5, df = 2, <em>P</em> = 3.4x10<sup>â7</sup>). Eleven of these sites are fixed for nucleotide changes in the dHYSC gene region. Additional tests show that unique polymorphisms and fixed differences are not homogeneous among arrangements (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS9.pdf">Table S9</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS10.pdf">Table S10</a>).</p><p id="p-41">The results of the three tests for selection (Tajima, coalescent, HKA) reveal a complex evolutionary history for the third chromosome. As has been seen with the majority of loci in <em>D. pseudoobscura</em>, Tajimaâs <em>D</em> is negative for 76.9% of the loci on the third chromosome, which strongly supports a demographic rather than selective explanation for the excess of rare variants seen at most loci, but only four loci showed a significant excess of rare variants. The other 23.1% of the loci show a greater frequency of intermediate variants including two breakpoint loci with significant positive <em>D</em> values, which may indicate the selective maintenance of gene arrangements.</p><p id="p-42">The coalescent simulations show levels of heterozygosity fail to reject a neutral model of molecular evolution when a constant population is assumed (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). When the more realistic exponentially growing population model is used, we observe some loci with significantly more polymorphism within arrangement than expected (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). This signature suggests that the maintenance of the different arrangements through selection in heterogeneous environments (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schaeffer 2008</a>) has elevated the levels of polymorphism within particular arrangements. The analysis of derived mutations is consistent with this interpretation because there is an excess of unique derived mutations in each of the five arrangements (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS7.pdf">Table S7</a>). Unique mutations are expected in inversion genealogies because the suppression of recombination will limit the transmission of polymorphisms among different arrangements, but our data suggest that the selective maintenance of the different arrangements has allowed the accumulation of more unique polymorphisms than expected under a neutral model. Although all arrangements show an excess of unique polymorphisms, the AR has more unique polymorphisms than expected given the proportion of segregating sites in AR relative to the total number (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS7.pdf">Table S7</a>). This is quite striking because the AR is one of the youngest arrangements and has one of the widest distributions within the species. These data suggest that this chromosome has increased at a rate greater than the overall rate of population expansion.</p><p id="p-43">The pattern of nucleotide variation in the TL arrangement suggests a complex evolutionary history compared to the other four arrangements. Four loci within TL (dSTPP, eve, pSCCH, and dHYST) have no segregating sites. This low level of variation could indicate strong selective constraint or a recent selective sweep on a sequence near these four regions. The rejection of the neutral model with the HKA test suggests that these four regions are near a region that has recent selective sweep because the <em>D. miranda</em> outgroup sequence is not selectively constrained. The dHYSC region has seven derived variants with a frequency of 0.25, but the region also accounts for 11 of 14 fixed derived mutations in the TL arrangement. This suggests that this region may be recovering from a recent selective sweep. This is consistent with numerical models of selection in heterogeneous environments that find evidence for selective sweeps and balancing selection operating on the inversions (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Schaeffer 2008</a>).</p></div><div id="sec-11" class="subsection"><h3>Derived gene arrangements fail to show reduced DNA divergence compared with ancestral gene arrangements</h3><p id="p-44">The theoretical work of <a id="xref-ref-27-17" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-12" class="xref-bibr" href="#ref-26">2000</a>) developed predictions about how the past selective history of chromosomal inversions and genetic flux will affect the patterns of nucleotide heterozygosity and differentiation within and outside the inverted regions. We tested the <a id="xref-ref-27-18" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-13" class="xref-bibr" href="#ref-26">2000</a>) predictions by plotting nucleotide heterozygosity for the 11 inversion breakpoint and seven nonbreakpoint regions in the derived, ancestral, and both arrangements combined. We contrasted two sets of data. The first comparison examined the smaller and younger AR chromosome that was derived from ST (ST â AR, 5.9-Mb inversion accounting for ~23.8 cM on the genetic map) and the second comparison observed the larger and older PP arrangement derived from ST (ST â PP, 11.6-Mb inversion accounting for ~46.9 cM on the genetic map).</p><p id="p-45">The more recent event (ST to AR) shows that silent heterozygosity levels are elevated at the two breakpoint regions (red arrows) but are lower in the proximal, central inverted region, and distal regions. The derived AR arrangement has lower heterozygosity levels at its two breakpoint regions compared with the ancestral ST arrangement or randomly chosen AR or ST chromosomes, but in the majority of other regions AR has greater levels of heterozygosity than ST (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). These data support the predictions of the <a id="xref-ref-26-14" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a> model at breakpoints but not in the central regions where heterozygosity in the derived arrangement is greater than in the ancestral arrangement.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Silent nucleotide heterozygosity (&#x3C0;, Tajima 1983) along the chromosomes in ancestral and derived arrangements of D. pseudoobscura. The blue line shows levels of variation within a derived arrangement, the red line shows levels of variation in the ancestral arrangement, and the thick black line shows levels of variation among the ancestral and derived gene arrangements. The two red arrows in each panel show the locations of the proximal and distal breakpoints that converted the ancestral into the derived arrangement. The order of genes in each panel is given according to the derived arrangement. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement. The proximal region is to the left." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1831192352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Silent nucleotide heterozygosity (&#x3C0;, Tajima 1983) along the chromosomes in ancestral and derived arrangements of &lt;em&gt;D. pseudoobscura&lt;/em&gt;. The blue line shows levels of variation within a derived arrangement, the red line shows levels of variation in the ancestral arrangement, and the thick black line shows levels of variation among the ancestral and derived gene arrangements. The two red arrows in each panel show the locations of the proximal and distal breakpoints that converted the ancestral into the derived arrangement. The order of genes in each panel is given according to the derived arrangement. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement. The proximal region is to the left.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1151/F4.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/7/1151/F4.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462986" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Silent nucleotide heterozygosity (Ï, <a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Tajima 1983</a>) along the chromosomes in ancestral and derived arrangements of <em>D. pseudoobscura</em>. The blue line shows levels of variation within a derived arrangement, the red line shows levels of variation in the ancestral arrangement, and the thick black line shows levels of variation among the ancestral and derived gene arrangements. The two red arrows in each panel show the locations of the proximal and distal breakpoints that converted the ancestral into the derived arrangement. The order of genes in each panel is given according to the derived arrangement. The genetic markers labeled in red on the x-axis are genes within the inverted region of the derived arrangement. The proximal region is to the left.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The pattern for the ST to PP data provides less support for the <a id="xref-ref-26-15" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a> model. The ancestral ST arrangement has lower levels of heterozygosity estimates at 10 of 18 regions. The observed pattern differs from the expected pattern given a neutral coalescent model where the more derived arrangement has consistently lower heterozygosity (<a id="xref-ref-26-16" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> 2000</a>).</p><p id="p-48">According to the predictions of <a id="xref-ref-26-17" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a>, patterns of nucleotide variability along the chromosome of gene arrangements will depend on the age of the inversion. This prediction is based on theory that shows that as new inversions increase in frequency, they can eliminate variability in a large segment of the chromosome by means of a possible selective sweep. The resulting effect is low variability at the breakpoints of new inversions and higher variability within the inverted segment. However, older inversions that have reached equilibrium will show greater variability at the breakpoints than within the inverted segment. The ST/AR comparison is our best test of the <a id="xref-ref-26-18" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a> model because the breakpoints have been precisely mapped (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Richards <em>et al.</em> 2005</a>). The comparison of the ST/AR arrangements shows that nucleotide diversity is elevated at the ST to AR breakpoints with the ancestral ST arrangement having higher levels of diversity than the derived AR chromosome. The elevation of nucleotide heterozygosity at the breakpoints is expected given the age of the ST to AR inversion. This inversion mutation was estimated to have occurred 580,000 years ago (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Wallace <em>et al.</em> 2011</a>). If we assume that <em>D. pseudoobscura</em> has four generations per year (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Babcock and Anderson 1996</a>), then AR has existed for more than 1 million generations and would be expected to show elevated levels of nucleotide diversity at the breakpoints as these regions continued to accumulate nucleotide mutations. The proximal, distal, and inverted regions, however, failed to show the derived AR arrangement with consistently lower levels of variation compared to the ancestral ST chromosome (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). It should be noted, however, that the confidence intervals for heterozygosity in the ST and AR arrangements overlap (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1/TableS2.pdf">Table S2</a>).</p><p id="p-49">The pattern for the ST/PP comparison is quite different from the ST/AR data. The markers closest to the breakpoints do not show an elevation of nucleotide heterozygosity. At this time, we have not precisely mapped the positions of the ST to PP breakpoints so the lack of elevated heterozygosity may not be surprising. Despite this, the markers within the inverted region do not show reduced variability in the derived PP arrangement. In most cases, markers within the ST arrangement have lower heterozygosity than PP. The likely explanation is that ST to PP inversion is twice the size of the AR arrangement, which will likely result in more genetic exchange among chromosomes leading to more independence in the genetic information.</p><p id="p-50">The lack of concordance of the <em>D. pseudoobscura</em> data with the predictions from the <a id="xref-ref-27-19" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-19" class="xref-bibr" href="#ref-26">2000</a>) models may reflect that the inversion polymorphism in <em>D. pseudoobscura</em> is not a simple two arrangement polymorphism. The arrangements of <em>D. pseudoobscura</em> were generated through a set of overlapping inversions generated at different times and the opportunity for genetic exchange between the different inversions depends on the localities where each arrangement is found in nature. The fact that the different arrangements were generated at different times will determine when each chromosome begins to diverge. The pairing of chromosomes separated by two or more inversion mutations will serve to increase recombination suppression. The <a id="xref-ref-26-20" class="xref-bibr" href="#ref-26">Navarro <em>et al.</em> (2000)</a> model assumes that the derived arrangement has swept to intermediate frequency and the increase in frequency of derived arrangements in <em>D. pseudoobscura</em> may not resulted from a simple sweep. Finally, the different arrangements are distributed in a highly structured manner among populations potentially due to local adaptation (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Schaeffer 2008</a>) This level of structure will determine which arrangements have the opportunity to pair as heterozygotes (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Schaeffer and Anderson 2005</a>). This will not always be with chromosomes that are phylogenetically related. For instance, PP and AR co-occur in Texas, but these two arrangements are separated by two inversion steps. ST and CH co-occur in California, but these two arrangements are separated by three inversion steps.</p></div><div id="sec-12" class="subsection"><h3>How recombination suppression structures genetic variation in the presence of chromosomal rearrangements</h3><p id="p-51">Reduced recombination among the gene arrangements is expected to result in LD or nonrandom associations among nucleotide sites. Patterns of LD can also result from population structure and balancing selection. Pairs of segregating sites across all gene arrangements were tested for significant LD.</p><p id="p-52">We first examined LD among nucleotide sites across all arrangements. A total of 54,911 of the possible 2,249,006 pairwise comparisons of the 671 segregating sites were capable of rejecting the null hypothesis of no association with Fisherâs exact test. Intralocus and interlocus LD for the 18 genetic markers were estimated as the percentage of site pairs that had a significant LD value (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5D</a>). A general result is that the central region of the third chromosome tends to have highest levels of LD than centromeric and telomeric regions. There are hot and cold spots for LD across the chromosome with both random association among adjacent regions and nonrandom associations between distant sites (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5D</a>), a pattern previously observed by <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Schaeffer <em>et al.</em> (2003)</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise LD plots for within- and between-locus comparisons across the third chromosome of D. pseudoobscura. Each block in the matrix presents the fraction of pairwise comparisons of sites that were in significant LD. In all plots, levels of LD within or between loci are represented as percentage of valid comparisons that were less than the respective significance cutoff using a false-discovery rate of 1% (Storey 2002). For (A&#x2212;C), the loci within the inverted segment are indicated by the bracket. The key to the heat map is shown at the bottom left hand side of the figure where lower levels of association are marked in blue and higher levels of association are redder. White dots indicate within or between locus comparisons that were significantly lower than the average % of significant comparisons, whereas the black dots indicate comparisons that were significantly greater than the average % significant comparisons. (A) Heat map for LD using the ST and AR chromosomes. Significance cutoff was P = 0.00195. The loci (top to bottom) are given in the order of the Arrowhead arrangement. The average percentage of significant comparisons within locus was 27.4% (432 of 1574) and the average percentage of significant comparisons between loci was 2.0% (371 of 18,334). (B) Heat map for LD using the ST and PP chromosomes. Significance cutoff was P = 0.00195. The loci (top to bottom) are given in the order of the PP arrangement. The average percentage of significant comparisons within locus was 34.8% (706 of 2030) and the average percentage of significant comparisons between loci was 8.0% (1996 of 24,829). (C) Heat map for LD using the CH and TL chromosomes. Significance cutoff was P = 0.0029. The loci (top to bottom) are given in the order of the CH arrangement. The average percentage of significant comparisons within locus was 15.1% (316 of 2087) and the average percentage of significant comparisons between loci was 4.3% (943 of 21896). (D) Heat map for LD for all D. pseudoobscura chromosomes. Significance cutoff was P = 0.0047. The loci (top to bottom) are given in the order of the Arrowhead arrangement. The box below the triangular matrix shows significant associations of nucleotide sites within a gene locus with the corresponding gene arrangement shown in rows. The significance cutoff for these tests was P = 0.021. The interlocus boxes highlighted in white indicate comparisons nucleotides between the proximal and distal breakpoints. The average percentage of significant comparisons within locus was 35.3% (1385 of 3922), and the average percentage of significant comparisons between loci was 11.3% (5808 of 51,573)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1831192352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pairwise LD plots for within- and between-locus comparisons across the third chromosome of &lt;em&gt;D. pseudoobscura&lt;/em&gt;. Each block in the matrix presents the fraction of pairwise comparisons of sites that were in significant LD. In all plots, levels of LD within or between loci are represented as percentage of valid comparisons that were less than the respective significance cutoff using a false-discovery rate of 1% (Storey 2002). For (A&#x2212;C), the loci within the inverted segment are indicated by the bracket. The key to the heat map is shown at the bottom left hand side of the figure where lower levels of association are marked in blue and higher levels of association are redder. White dots indicate within or between locus comparisons that were significantly lower than the average % of significant comparisons, whereas the black dots indicate comparisons that were significantly greater than the average % significant comparisons. (A) Heat map for LD using the ST and AR chromosomes. Significance cutoff was &lt;em&gt;P&lt;/em&gt; = 0.00195. The loci (top to bottom) are given in the order of the Arrowhead arrangement. The average percentage of significant comparisons within locus was 27.4% (432 of 1574) and the average percentage of significant comparisons between loci was 2.0% (371 of 18,334). (B) Heat map for LD using the ST and PP chromosomes. Significance cutoff was &lt;em&gt;P&lt;/em&gt; = 0.00195. The loci (top to bottom) are given in the order of the PP arrangement. The average percentage of significant comparisons within locus was 34.8% (706 of 2030) and the average percentage of significant comparisons between loci was 8.0% (1996 of 24,829). (C) Heat map for LD using the CH and TL chromosomes. Significance cutoff was &lt;em&gt;P&lt;/em&gt; = 0.0029. The loci (top to bottom) are given in the order of the CH arrangement. The average percentage of significant comparisons within locus was 15.1% (316 of 2087) and the average percentage of significant comparisons between loci was 4.3% (943 of 21896). (D) Heat map for LD for all &lt;em&gt;D. pseudoobscura&lt;/em&gt; chromosomes. Significance cutoff was &lt;em&gt;P&lt;/em&gt; = 0.0047. The loci (top to bottom) are given in the order of the Arrowhead arrangement. The box below the triangular matrix shows significant associations of nucleotide sites within a gene locus with the corresponding gene arrangement shown in rows. The significance cutoff for these tests was &lt;em&gt;P&lt;/em&gt; = 0.021. The interlocus boxes highlighted in white indicate comparisons nucleotides between the proximal and distal breakpoints. The average percentage of significant comparisons within locus was 35.3% (1385 of 3922), and the average percentage of significant comparisons between loci was 11.3% (5808 of 51,573).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1151/F5.medium.gif" width="358" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/7/1151/F5.medium.gif" width="358" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1151/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-53" class="first-child">Pairwise LD plots for within- and between-locus comparisons across the third chromosome of <em>D. pseudoobscura</em>. Each block in the matrix presents the fraction of pairwise comparisons of sites that were in significant LD. In all plots, levels of LD within or between loci are represented as percentage of valid comparisons that were less than the respective significance cutoff using a false-discovery rate of 1% (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Storey 2002</a>). For (AâC), the loci within the inverted segment are indicated by the bracket. The key to the heat map is shown at the bottom left hand side of the figure where lower levels of association are marked in blue and higher levels of association are redder. White dots indicate within or between locus comparisons that were significantly lower than the average % of significant comparisons, whereas the black dots indicate comparisons that were significantly greater than the average % significant comparisons. (A) Heat map for LD using the ST and AR chromosomes. Significance cutoff was <em>P</em> = 0.00195. The loci (top to bottom) are given in the order of the Arrowhead arrangement. The average percentage of significant comparisons within locus was 27.4% (432 of 1574) and the average percentage of significant comparisons between loci was 2.0% (371 of 18,334). (B) Heat map for LD using the ST and PP chromosomes. Significance cutoff was <em>P</em> = 0.00195. The loci (top to bottom) are given in the order of the PP arrangement. The average percentage of significant comparisons within locus was 34.8% (706 of 2030) and the average percentage of significant comparisons between loci was 8.0% (1996 of 24,829). (C) Heat map for LD using the CH and TL chromosomes. Significance cutoff was <em>P</em> = 0.0029. The loci (top to bottom) are given in the order of the CH arrangement. The average percentage of significant comparisons within locus was 15.1% (316 of 2087) and the average percentage of significant comparisons between loci was 4.3% (943 of 21896). (D) Heat map for LD for all <em>D. pseudoobscura</em> chromosomes. Significance cutoff was <em>P</em> = 0.0047. The loci (top to bottom) are given in the order of the Arrowhead arrangement. The box below the triangular matrix shows significant associations of nucleotide sites within a gene locus with the corresponding gene arrangement shown in rows. The significance cutoff for these tests was <em>P</em> = 0.021. The interlocus boxes highlighted in white indicate comparisons nucleotides between the proximal and distal breakpoints. The average percentage of significant comparisons within locus was 35.3% (1385 of 3922), and the average percentage of significant comparisons between loci was 11.3% (5808 of 51,573).</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Breakpoint regions tended to have higher levels of LD than nonbreakpoint regions. Breakpoint regions had a higher fraction of site pairs that showed significant nonrandom associations within locus than non-breakpoint regions (39.8% <em>vs.</em> 22.8%, respectively). Interlocus comparisons of breakpoint regions also had greater fractions of site pairs in significant LD than interlocus comparisons of nonbreakpoint regions (14.5% <em>vs.</em> 8.1%, respectively).</p><p id="p-55">We asked whether the proximal and distal breakpoints for the five gene arrangements were in LD with each other. One might expect nucleotide variation at the two ends of an inversion to show significant association because allelic variation initially captured by the original inversion event will be associated along the entire inverted region and the strong suppression of recombination at the breakpoints will maintain these associations. Two pairs of inversion breakpoints support this hypothesis, the SC to TL and the SC to CH inversions, with 22.3 and 55.4% of the sites pairs in significant LD. The other four pairs of inversion breakpoints had few nonrandom associations despite these arrangements resulting from older inversion events.</p><p id="p-56">LD increases as a gene arrangementâs age increases. The AR and PP arrangements are each derived from the ST arrangement. We examined the pattern of significant LD estimated within and between loci for the younger comparison of ST and AR chromosomes and the older comparison of ST and PP arrangements. The ST and AR comparisons had lower levels of LD within and between loci than the ST and PP comparisons (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figures 5, A and B</a>). We can rule out differences in sample size as a potential explanation for why the AR/ST comparisons had fewer significant nonrandom associations than the PP/ST comparisons. The AR sample size was greater than that of PP, yet had fewer significant test results. In both cases, the greatest interlocus LD was observed among loci within the inversion breakpoints. We tested for significant LD for TL and CH chromosomes, which are both derived from the SC arrangement, and found a modest level of nonrandom associations among sites within and between loci (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). Interlocus LD appeared to be lower for the markers closest to the ends of the chromosome than for markers in the center of the chromosome. The dHYST and EcR markers are closest to the distal end of the chromosome and showed low levels of interlocus LD whereas the pSTPP and <em>en</em> markers closest to the proximal end of the chromosome showed low interlocus LD. All other markers positioned on other areas of the chromosome have higher levels of interlocus LD with the exception of the pSTAR region that showed no interlocus LD.</p><p id="p-57">Variation at the segregating sites showed significant associations with gene arrangement. LD was estimated between nucleotide sites and the five different gene arrangements (AR, PP, ST, CH and TL). A total of 944 of the possible 3355 pairwise comparisons of the 671 segregating sites were capable of rejecting the null hypothesis of no association between the segregating site and the inversion type with Fisherâs exact test. It was shown that the PP arrangement had the highest levels of LD and the AR arrangement had the lowest levels of LD. The EcR locus showed no evidence of LD in any of the gene arrangements. The pSTPP marker only showed evidence of LD with the PP arrangement. The markers at the extreme end of the chromosome showed no evidence of LD with the arrangements with the exception of the pSTPP breakpoint region in the PP arrangement. The PP arrangement had the longest continuous segment of LD along the chromosome.</p><p id="p-58">Over time, chromosomal rearrangements will differentiate from one another and accumulate unique polymorphisms because recombination suppression will isolate the different gene arrangements from each other. This predicts that LD will increase with gene arrangement age. This is what was observed. When the ST sequences were combined with the younger AR sequences, we saw modest levels of LD within locus and relatively low levels of LD among loci. When the ST sequences were combined with older PP sequences, we saw higher levels of LD within and between loci.</p><p id="p-59">An unusual feature of the LD analysis was that pairs of breakpoints for a single arrangement, say pSTAR and dSTAR, did not show high levels of LD. This was an unexpected result because one might expect that the initial inversion mutation would create LD among the polymorphic sites at the proximal and distal boundary regions and these associations would be maintained over time. For some breakpoints in our study, we are not confident that the genetic markers used were absolutely adjacent to the breakpoint regions. The ST to AR inversion breakpoints, however, have been precisely mapped and these genetic markers (pSTAR and dSTAR) are within 0.5 and 5.6 kb of their respective breakpoints. These markers show a deficiency of LD between the two breakpoints, despite high levels of intralocus LD (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>). This finding suggests that the regions near the inversion breakpoints did not capture segregating allelic sites at the time of the original inversion mutation. It appears that all LD associated with these two inversion breakpoints has accumulated since the inversion mutation happened. The distance that separates the two breakpoints is sufficient to allow these regions to evolve independently suggesting that new nucleotide mutations do not co-occur on the same chromosome.</p><p id="p-60">The results of the LD analysis within each arrangement provide a link between estimates of LD and the length of inversions. The PP arrangement had significant levels of LD distributed over a large portion of the chromosome (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). In fact, the pSTPP marker only showed evidence of LD in the PP arrangement, and it is close to the proximal breakpoint region of the PP chromosome. This finding is concordant with the model that newer inversions have low rates of recombination near the inversion breakpoints and low rates of recombination are indicated by significant LD. The breakpoint regions within the other gene arrangements also show significant LD, except for the AR arrangement. Overall, inversion breakpoint regions tend to provide more evidence of LD than non-breakpoint regions. In addition to estimating rates of nucleotide diversity, inversion breakpoints may provide information regarding rates of recombination by estimates of LD.</p><p id="p-61">The central region of the chromosome harbors the greatest levels of LD despite the expectations of the <a id="xref-ref-27-20" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997)</a> model, which suggests that genetic flux will be greatest in the central regions of inversions. Our data may differ from this expectation because we are dealing with a system of overlapping inversion mutations. The central region of the chromosome is expected to be inverted in most gene arrangement heterozygote combinations. Thus, the central regions would be expected to experience the strongest amount of recombination suppression compared to either the proximal or distal regions. The reduced recombination leads to strong non-random associations between sites. Thus, a system of overlapping inversions can increase the level of genetic structure and severely reduce genetic flux over a wider region of the chromosome.</p><p id="p-62">Models of <a id="xref-ref-27-21" class="xref-bibr" href="#ref-27">Navarro <em>et al.</em> (1997</a>, <a id="xref-ref-26-21" class="xref-bibr" href="#ref-26">2000</a>) made predictions about how genetic flux in the presence of paracentric inversions will impact genetic diversity along chromosomes. Our data did not conclusively support the predictions of these models. In some cases, we found that variation was elevated near some breakpoints, but not in others. Older inversions fail to show greater levels of diversity than derived mutations, even though breakpoint regions accumulate unique mutations, which should elevate levels of diversity and divergence. Breakpoint regions show elevated levels of LD, yet segregating sites within pairs of breakpoints fail to show nonrandom association with each other. The observed decoupling of genetic diversity patterns across the third chromosome of <em>D. pseudoobscura</em> reveals a complex history that reflects neutral and potentially selective forces acting in nature. Future analysis of the third chromosome with next generation re-sequencing will help to clarify issues left unanswered in this work.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">We thank Mary Poss, Hong Ma, and Avery August for thoughtful comments on early drafts of this manuscript. We also wish to thank an anonymous reviewer and Susan Celnicker for helping us to see a better way to organize our thoughts within the paper. Finally, we wish to thank the Department of Biology at The Pennsylvania State University for support on this work as well as support from the National Science Foundation (Grant Number OCI-0904166). Any opinions, findings, conclusions, or recommendations expressed in this article are those of the authors and do not necessarily reflect the views of the National Science Foundation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006122/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">GenBank Accession Numbers: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AF476112&amp;atom=%2Fggg%2F3%2F7%2F1151.atom">AF476112</a>â<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AF476921&amp;atom=%2Fggg%2F3%2F7%2F1151.atom">AF476921</a>, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JF285334&amp;atom=%2Fggg%2F3%2F7%2F1151.atom">JF285334</a>â<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JF286585&amp;atom=%2Fggg%2F3%2F7%2F1151.atom">JF286585</a>.</p></li><li class="fn" id="fn-5"><p id="p-64"><em>Communicating editor: S. Celniker</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-03-21"><span class="received-label">Received </span>March 21, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-05-01"><span class="accepted-label">Accepted </span>May 1, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 by the Genetics Society of America</li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.1" data-doi="10.1093/oxfordjournals.molbev.a025589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babcock</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Molecular evolution of the sex-ratio inversion complex in <em>Drosophila pseudoobscura</em>: Analysis of the <em>Esterase-5</em> gene region.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">297</span>â<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBabcock%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D297%26rft.epage%253D308%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bthe%2BSex-Ratio%2Binversion%2Bcomplex%2Bin%2BDrosophila%2Bpseudoobscura%253A%2Banalysis%2Bof%2Bthe%2BEsterase-5%2Bgene%2Bregion%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025589%26rft_id%253Dinfo%253Apmid%252F8587496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvMjk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.2" data-doi="10.1038/356519a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Levels of naturally occurring DNA polymorphism correlate with recombination rates in <em>D. melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">519</span>â<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D356%26rft.issue%253D6369%26rft.spage%253D519%26rft.epage%253D520%26rft.atitle%253DLevels%2Bof%2Bnaturally%2Boccurring%2BDNA%2Bpolymorphism%2Bcorrelate%2Bwith%2Brecombination%2Brates%2Bin%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F356519a0%26rft_id%253Dinfo%253Apmid%252F1560824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/356519a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1560824&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HN23100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betran</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">The estimation of the number and the length distribution of gene conversion tracts from population DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">89</span>â<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBetran%26rft.auinit1%253DE.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D99%26rft.atitle%253DThe%2BEstimation%2Bof%2Bthe%2BNumber%2Band%2Bthe%2BLength%2BDistribution%2Bof%2BGene%2BConversion%2BTracts%2BFrom%2BPopulation%2BDNA%2BSequence%2BData%26rft_id%253Dinfo%253Apmid%252F9136003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTQ2LzEvODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chovnick</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span>â<span class="cit-article-title">Gene conversion and transfer of genetic information within the inverted region of inversion heterozygotes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">123</span>â<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChovnick%26rft.auinit1%253DA.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D131%26rft.atitle%253DGENE%2BCONVERSION%2BAND%2BTRANSFER%2BOF%2BGENETIC%2BINFORMATION%2BWITHIN%2BTHE%2BINVERTED%2BREGION%2BOF%2BINVERSION%2BHETEROZYGOTES%26rft_id%253Dinfo%253Apmid%252F4202769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzUvMS8xMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span>â<span class="cit-article-title">Genetics of natural populations. IX. Temporal changes in the composition of populations of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">162</span>â<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DTh.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D162%26rft.epage%253D186%26rft.atitle%253DGENETICS%2BOF%2BNATURAL%2BPOPULATIONS%2BIX.%2BTEMPORAL%2BCHANGES%2BIN%2BTHE%2BCOMPOSITION%2BOF%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%26rft_id%253Dinfo%253Apmid%252F17247077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyOC8yLzE2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span>â<span class="cit-article-title">Chromosomal races in <em>Drosophila pseudoobscura</em> and <em>Drosophila persimilis</em>.</span> <abbr class="cit-jnl-abbrev">Carnegie Inst. Washington Publ.</abbr> <span class="cit-vol">554</span>: <span class="cit-fpage">47</span>â<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCarnegie%2BInst.%2BWashington%2BPubl.%26rft.volume%253D554%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-confproc" id="cit-3.7.1151.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span>â<span class="cit-article-title">Observations and experiments on natural selection in <em>Drosophila</em></span>, pp. <span class="cit-fpage">210</span>â<span class="cit-lpage">224</span> in <span class="cit-source">Proceedings of the International Congress of Genetics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bonnier</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">R.</span></span>. <span class="cit-publ-name">Berlingska Boktryckeriet</span>, <span class="cit-publ-loc">Lund, Sweden</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span>â<span class="cit-article-title">The genetics of natural populations. XIX. Origin of heterosis through natural selection in populations of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">288</span>â<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DT.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D288%26rft.epage%253D302%26rft.atitle%253DGENETICS%2BOF%2BNATURAL%2BPOPULATIONS.%2BXIX.%2BORIGIN%2BOF%2BHETEROSIS%2BTHROUGH%2BNATURAL%2BSELECTION%2BIN%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%26rft_id%253Dinfo%253Apmid%252F15414931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzNS8zLzI4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epling</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span>â<span class="cit-article-title">Taxonomy, geographic distribution, and ecology of <em>Drosophila pseudoobscura</em> and its relatives</span>, pp. <span class="cit-fpage">1</span>â<span class="cit-lpage">46</span> in <span class="cit-source">Contributions to the Genetics, Taxonomy, and Ecology of Drosophila pseudoobscura and its Relatives</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Epling</span>  <span class="cit-name-given-names">C.</span></span>. <span class="cit-publ-name">The Lord Baltimore Press</span>, <span class="cit-publ-loc">Baltimore, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span>â<span class="cit-article-title">Inversions in the chromosomes of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">28</span>â<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DTh.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D64%26rft.atitle%253DINVERSIONS%2BIN%2BTHE%2BCHROMOSOMES%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%26rft_id%253Dinfo%253Apmid%252F17246876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIyMy8xLzI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Queal</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span>â<span class="cit-article-title">Genetics of natural populations. II. Genic variation in populations of <em>Drosophila pseudoobscura</em> inhabiting isolated mountain ranges.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">463</span>â<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DTh.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D463%26rft.epage%253D484%26rft.atitle%253DGENETICS%2BOF%2BNATURAL%2BPOPULATIONS.%2BII.%2BGENIC%2BVARIATION%2BIN%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%2BINHABITING%2BISOLATED%2BMOUNTAIN%2BRANGES%26rft_id%253Dinfo%253Apmid%252F17246895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyMy81LzQ2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gloor</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engels</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Single-fly DNA preps for PCR.</span> <abbr class="cit-jnl-abbrev">Drosoph. Inf. Serv.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">148</span>â<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrosoph.%2BInf.%2BServ.%26rft.volume%253D71%26rft.spage%253D148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">DNA sequence variation and the recombinational landscape in <em>Drosophila pseudoobscura</em>: a study of the second chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">859</span>â<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D859%26rft.epage%253D869%26rft.atitle%253DDNA%2BSequence%2BVariation%2Band%2Bthe%2BRecombinational%2BLandscape%2Bin%2BDrosophila%2Bpseudoobscura%253A%2BA%2BStudy%2Bof%2Bthe%2BSecond%2BChromosome%26rft_id%253Dinfo%253Apmid%252F10511563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzIvODU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilliker</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harauz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reaume</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Meiotic gene conversion tract length distribution within the <em>rosy</em> locus of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1019</span>â<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHilliker%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1019%26rft.epage%253D1026%26rft.atitle%253DMeiotic%2Bgene%2Bconversion%2Btract%2Blength%2Bdistribution%2Bwithin%2Bthe%2Brosy%2Blocus%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F7982556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzEwMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Gene genealogies and the coalescent process.</span> <abbr class="cit-jnl-abbrev">Oxford Surveys in Evolutionary Biology</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOxford%2BSurveys%2Bin%2BEvolutionary%2BBiology%26rft.volume%253D7%26rft.spage%253D1%26rft.atitle%253DOXFORD%2BSURVEYS%2BIN%2BEVOLUTIONARY%2BBIOLOGY%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">On the divergence of alleles in nested subsamples from finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1057</span>â<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D113%26rft.issue%253D4%26rft.spage%253D1057%26rft.epage%253D1076%26rft.atitle%253DON%2BTHE%2BDIVERGENCE%2BOF%2BALLELES%2BIN%2BNESTED%2BSUBSAMPLES%2BFROM%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F3744026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjExMy80LzEwNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">AguadÃ©</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">A test of neutral molecular evolution based on nucleotide data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">153</span>â<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D116%26rft.issue%253D1%26rft.spage%253D153%26rft.epage%253D159%26rft.atitle%253DA%2BTest%2Bof%2BNeutral%2BMolecular%2BEvolution%2BBased%2Bon%2BNucleotide%2BData%26rft_id%253Dinfo%253Apmid%252F3110004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE2LzEvMTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">The coalescent in an exponentially growing metapopulation and its application to <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">2015</span>â<span class="cit-lpage">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D2015%26rft.epage%253D2019%26rft.atitle%253DThe%2BCoalescent%2Bin%2Ban%2BExponentially%2BGrowing%2BMetapopulation%2Band%2BIts%2BApplication%2Bto%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F11032470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzIwMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span>â<span class="cit-article-title">Evolution of protein molecules</span>, pp. <span class="cit-fpage">21</span>â<span class="cit-lpage">132</span> in <span class="cit-source">Mammalian Protein Metabolism</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Munro</span>  <span class="cit-name-given-names">H. N.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.20" data-doi="10.1098/rstb.1986.0012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">DNA and the neutral theory.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond., B</abbr> <span class="cit-vol">312</span>: <span class="cit-fpage">343</span>â<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D312%26rft.issue%253D1154%26rft.spage%253D343%26rft.epage%253D354%26rft.atitle%253DDNA%2Band%2Bthe%2BNeutral%2BTheory%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1986.0012%26rft_id%253Dinfo%253Apmid%252F2870526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvMTE1NC8zNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.21" data-doi="10.1371/journal.pbio.1000501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">How and why chromosome inversions evolve.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D8%26rft.spage%253De1000501%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000501%26rft_id%253Dinfo%253Apmid%252F20927412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927412&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.22" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>â<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">The detection of linkage disequilibrium in molecular sequence data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">377</span>â<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D140%26rft.issue%253D1%26rft.spage%253D377%26rft.epage%253D388%26rft.atitle%253DThe%2Bdetection%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bmolecular%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Apmid%252F7635301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzEvMzc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.24" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>â<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1940</span>â<span class="cit-article-title">Bearings of the âDrosophilaâ work on systematics</span>, pp. <span class="cit-fpage">185</span>â<span class="cit-lpage">268</span> in <span class="cit-source">The New Systematics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Huxley</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Effect of inversion polymorphism on the neutral nucleotide variability of linked chromosomal regions in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">685</span>â<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNavarro%26rft.auinit1%253DA.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D685%26rft.epage%253D698%26rft.atitle%253DEffect%2Bof%2BInversion%2BPolymorphism%2Bon%2Bthe%2BNeutral%2BNucleotide%2BVariability%2Bof%2BLinked%2BChromosomal%2BRegions%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F10835391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvNjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">BetrÃ¡n</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Recombination and gene flux caused by gene conversion and crossing over in inversion heterokaryotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">695</span>â<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNavarro%26rft.auinit1%253DA.%26rft.volume%253D146%26rft.issue%253D2%26rft.spage%253D695%26rft.epage%253D709%26rft.atitle%253DRecombination%2Band%2BGene%2BFlux%2BCaused%2Bby%2BGene%2BConversion%2Band%2BCrossing%2BOver%2Bin%2BInversion%2BHeterokaryotypes%26rft_id%253Dinfo%253Apmid%252F9178017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzIvNjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.28" data-doi="10.1534/genetics.107.070672"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfield</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Divergence between the <em>Drosophila pseudoobscura</em> and <em>D. persimilis</em> genome sequences in relation to chromosomal inversions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1417</span>â<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNoor%26rft.auinit1%253DM.%2BA.%2BF.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D1417%26rft.epage%253D1428%26rft.atitle%253DDivergence%2BBetween%2Bthe%2BDrosophila%2Bpseudoobscura%2Band%2BD.%2Bpersimilis%2BGenome%2BSequences%2Bin%2BRelation%2Bto%2BChromosomal%2BInversions%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070672%26rft_id%253Dinfo%253Apmid%252F18039875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE0MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">T. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1934</span>â<span class="cit-article-title">A new method for the study of chromosomal aberrations and the plotting of chromosomal maps in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">175</span>â<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPainter%26rft.auinit1%253DT.%2BS.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D188%26rft.atitle%253DA%2BNEW%2BMETHOD%2BFOR%2BTHE%2BSTUDY%2BOF%2BCHROMOSOME%2BABERRATIONS%2BAND%2BTHE%2BPLOTTING%2BOF%2BCHROMOSOME%2BMAPS%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F17246718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxOS8zLzE3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Inversion polymorphisms in <em>Drosophila pseudoobscura</em> and <em>Drosophila persimilis</em></span>, pp. <span class="cit-fpage">73</span>â<span class="cit-lpage">126</span> in <span class="cit-source">Drosophila Inversion Polymorphism</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Krimbas</span>  <span class="cit-name-given-names">C. B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Ann Arbor, MI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.31" data-doi="10.1101/gr.3059305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bettencourt</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hradecky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Letovsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Comparative genome sequencing of <em>Drosophila pseudoobscura</em>: Chromosomal, gene and <em>cis</em>-element evolution.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRichards%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DComparative%2Bgenome%2Bsequencing%2Bof%2BDrosophila%2Bpseudoobscura%253A%2BChromosomal%252C%2Bgene%252C%2Band%2Bcis-element%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3059305%26rft_id%253Dinfo%253Apmid%252F15632085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjY6IjE1LzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span>â<span class="cit-article-title">The genetics of chromosome aberration</span>, pp. <span class="cit-fpage">67</span>â<span class="cit-lpage">184</span> in <span class="cit-source">The Genetics and Biology of Drosophila</span> <span class="cit-vol">1a</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Novitski</span>  <span class="cit-name-given-names">E.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BGenetics%2Band%2BBiology%2Bof%2BDrosophila%26rft.volume%253D1a%26rft.spage%253D67%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCHROMOSOME%2BABERRATION%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">AguadÃ©</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Evidence of extensive genetic exchange in the <em>rp49</em> region among polymorphic chromosome inversions in <em>Drosophila subobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">417</span>â<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRozas%26rft.auinit1%253DJ.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D417%26rft.epage%253D426%26rft.atitle%253DEvidence%2Bof%2Bextensive%2Bgenetic%2Bexchange%2Bin%2Bthe%2Brp49%2Bregion%2Bamong%2Bpolymorphic%2Bchromosome%2Binversions%2Bin%2BDrosophila%2Bsubobscura.%26rft_id%253Dinfo%253Apmid%252F2123162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI2LzIvNDE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.34" data-doi="10.1073/pnas.90.17.8083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">AguadÃ©</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Transfer of genetic information in the <em>rp49</em> region of <em>Drosophila subobscura</em> between different gene arrangements.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">8083</span>â<span class="cit-lpage">8087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRozas%26rft.auinit1%253DJ.%26rft.volume%253D90%26rft.issue%253D17%26rft.spage%253D8083%26rft.epage%253D8087%26rft.atitle%253DTransfer%2Bof%2Bgenetic%2Binformation%2Bin%2Bthe%2Brp49%2Bregion%2Bof%2BDrosophila%2Bsubobscura%2Bbetween%2Bdifferent%2Bchromosomal%2Bgene%2Barrangements.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.17.8083%26rft_id%253Dinfo%253Apmid%252F8367467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTAvMTcvODA4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.35" data-doi="10.1073/pnas.91.24.11517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">AguadÃ©</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Gene conversion is involved in the transfer of genetic information between naturally occurring inversions of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">11517</span>â<span class="cit-lpage">11521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRozas%26rft.auinit1%253DJ.%26rft.volume%253D91%26rft.issue%253D24%26rft.spage%253D11517%26rft.epage%253D11521%26rft.atitle%253DGene%2Bconversion%2Bis%2Binvolved%2Bin%2Bthe%2Btransfer%2Bof%2Bgenetic%2Binformation%2Bbetween%2Bnaturally%2Boccurring%2Binversions%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.24.11517%26rft_id%253Dinfo%253Apmid%252F7972094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTEvMjQvMTE1MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.36" data-doi="10.1017/S0016672302005955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Molecular population genetics of sequence length diversity in the <em>Adh</em> region of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">163</span>â<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D163%26rft.epage%253D175%26rft.atitle%253DMolecular%2Bpopulation%2Bgenetics%2Bof%2Bsequence%2Blength%2Bdiversity%2Bin%2Bthe%2BAdh%2Bregion%2Bof%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672302005955%26rft_id%253Dinfo%253Apmid%252F12688655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672302005955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12688655&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181272900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.37" data-doi="10.1111/j.1558-5646.2008.00504.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Selection in heterogeneous environments maintains the gene arrangement polymorphism of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">3082</span>â<span class="cit-lpage">3099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D3082%26rft.epage%253D3099%26rft.atitle%253DSelection%2Bin%2Bheterogeneous%2Benvironments%2Bmaintains%2Bthe%2Bgene%2Barrangement%2Bpolymorphism%2Bof%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00504.x%26rft_id%253Dinfo%253Apmid%252F18764919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00504.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18764919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.38" data-doi="10.1534/genetics.105.041947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Mechanisms of genetic exchange within the chromosomal inversions of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1729</span>â<span class="cit-lpage">1739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.041947%26rft_id%253Dinfo%253Apmid%252F16143625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzE3MjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.39" data-doi="10.1073/pnas.1432900100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goetting-Minesky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovacevic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peoples</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graybill</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Evolutionary genomics of inversions in <em>Drosophila pseudoobscura</em>: Evidence for epistasis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">8319</span>â<span class="cit-lpage">8324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1432900100%26rft_id%253Dinfo%253Apmid%252F12824467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE0LzgzMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.40" data-doi="10.1534/genetics.107.086074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhutkar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAllister</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matzkin</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Polytene chromosomal maps of 11 Drosophila species: The order of genomic scaffolds inferred from genetic and physical maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1601</span>â<span class="cit-lpage">1655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.086074%26rft_id%253Dinfo%253Apmid%252F18622037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE2MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">555</span>â<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D2%26rft.spage%253D555%26rft.epage%253D562%26rft.atitle%253DPairwise%2Bcomparisons%2Bof%2Bmitochondrial%2BDNA%2Bsequences%2Bin%2Bstable%2Band%2Bexponentially%2Bgrowing%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F1743491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzIvNTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sokal</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohlf</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span>â<span class="cit-source">Biometry</span>. <span class="cit-publ-name">W. H. Freeman and Co.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sperlich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfriem</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">Chromosomal polymorphism in natural and experimental populations</span>, pp. <span class="cit-fpage">257</span>â<span class="cit-lpage">309</span> in <span class="cit-source">The Genetics and Biology of Drosophila 3e</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">H. L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. N.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.44" data-doi="10.1111/1467-9868.00346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">A direct approach to false discovery rates.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">479</span>â<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D64%26rft.spage%253D479%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.45" data-doi="10.1073/pnas.3.9.555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1917</span>â<span class="cit-article-title">Genetic factors affecting the strength of genetic linkage in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">555</span>â<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSturtevant%26rft.auinit1%253DA.%2BH.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D555%26rft.epage%253D558%26rft.atitle%253DGenetic%2BFactors%2BAffecting%2Bthe%2BStrength%2Bof%2BLinkage%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.3.9.555%26rft_id%253Dinfo%253Apmid%252F16586749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiIzLzkvNTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beadle</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span>â<span class="cit-article-title">The relations of inversions in the X chromosome of <em>Drosophila melanogaster</em> to crossing over and disjunction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">544</span>â<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D21%26rft.spage%253D544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-2" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span>â<span class="cit-article-title">Evolutionary relationship of DNA sequences in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>â<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ajioka</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kushner</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">A method for detecting recent selection in the human genome from allele age estimates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">287</span>â<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DToomajian%26rft.auinit1%253DC.%26rft.volume%253D165%26rft.issue%253D1%26rft.spage%253D287%26rft.epage%253D297%26rft.atitle%253DA%2BMethod%2Bfor%2BDetecting%2BRecent%2BSelection%2Bin%2Bthe%2BHuman%2BGenome%2BFrom%2BAllele%2BAge%2BEstimates%26rft_id%253Dinfo%253Apmid%252F14504236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzEvMjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzExNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.50" data-doi="10.1093/molbev/msr039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Detweiler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Evolutionary history of the third chromosome gene arrangements of <em>Drosophila pseudoobscura</em> inferred from inversion breakpoints.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2219</span>â<span class="cit-lpage">2229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr039%26rft_id%253Dinfo%253Apmid%252F21339510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzgvMjIxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-book" id="cit-3.7.1151.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span>â<span class="cit-article-title">Evolution of the <em>repleta</em> group</span>, pp. <span class="cit-fpage">61</span>â<span class="cit-lpage">139</span> in <span class="cit-source">The Genetics and Biology of Drosophila 3b</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">H. L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. N.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.52" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span>â<span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>â<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.7.1151.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1946</span>â<span class="cit-article-title">Genetics of natural populations. XII. Experimental reproduction of some of the changes caused by natural selection in certain populations of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">125</span>â<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D156%26rft.atitle%253DGENETICS%2BOF%2BNATURAL%2BPOPULATIONS.%2BXII.%2BEXPERIMENTAL%2BREPRODUCTION%2BOF%2BSOME%2BOF%2BTHE%2BCHANGES%2BCAUSED%2BBY%2BNATURAL%2BSELECTION%2BIN%2BCERTAIN%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%26rft_id%253Dinfo%253Apmid%252F17247189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzMS8yLzEyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/7/1151.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1143" title="A Dynamic Database of Microarray-Characterized Cell Lines with Various Cytogenetic and Genomic Backgrounds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1165" title="Generalized Admixture Mapping for Complex Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/7"><span>Volume 3 Issue 7, July 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/7.cover-source.jpg?itok=DKXH6P5c" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1151" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4zF2U4sPQtiZqMDI_XVbZ4b3rdDg5sey3RcXXw7eMTc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/7/1151" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/7/1151" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/7/1151" class="active" data-icon-position="" data-hide-link-title="0">Molecular Population Genetics of Inversion Breakpoint Regions in Drosophila pseudoobscura</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459492" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0ULvihn4gkuY89gGdRgAw_VmcqnLESuQwmlenorVvCo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1151.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/7/1151" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-8xHA3o4JtsUOO2vVTLAIBdf5Zq77_9OxVQPKp1oZUA0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/7/1151" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1151.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459492" id="node459492--2" data-pisa="ggg;3/7/1151" data-pisa-master="ggg;g3.113.006122" data-apath="/ggg/3/7/1151.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Population Genetics of Inversion Breakpoint Regions in <em>Drosophila pseudoobscura</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andre G. Wallace</span>, <span class="highwire-citation-author" data-delta="1">Don Detweiler</span> and <span class="highwire-citation-author" data-delta="2">Stephen W. Schaeffer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1151-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006122 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459492--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andre G. Wallace </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andre%2BG.%2BWallace%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallace%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndre%2BG.%2BWallace%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Don Detweiler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Don%2BDetweiler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Detweiler%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADon%2BDetweiler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen W. Schaeffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BW.%2BSchaeffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaeffer%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BW.%2BSchaeffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sws4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">sws4@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/7/1151" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/7/1151&amp;title=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/7/1151&amp;title=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/7/1151&amp;title=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/7/1151&amp;text=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/7/1151&amp;title=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/7/1151&amp;t=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/7/1151&amp;title=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/7/1151&amp;title=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459492/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459492" id="node459492--3" data-pisa="ggg;3/7/1151" data-pisa-master="ggg;g3.113.006122" data-apath="/ggg/3/7/1151.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Population Genetics of Inversion Breakpoint Regions in <em>Drosophila pseudoobscura</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andre G. Wallace</span>, <span class="highwire-citation-author" data-delta="1">Don Detweiler</span> and <span class="highwire-citation-author" data-delta="2">Stephen W. Schaeffer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1151-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006122 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459492--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andre G. Wallace </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andre%2BG.%2BWallace%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallace%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndre%2BG.%2BWallace%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Don Detweiler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Don%2BDetweiler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Detweiler%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADon%2BDetweiler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stephen W. Schaeffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BW.%2BSchaeffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaeffer%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BW.%2BSchaeffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sws4@psu.edu" class="" data-icon-position="" data-hide-link-title="0">sws4@psu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1151&amp;rft.epage=1163&amp;rft.atitle=Molecular%20Population%20Genetics%20of%20Inversion%20Breakpoint%20Regions%20in%20Drosophila%20pseudoobscura&amp;rft.volume=3&amp;rft.issue=7&amp;rft.date=2013-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wallace%2C+Andre+G.&amp;rft.au=Detweiler%2C+Don&amp;rft.au=Schaeffer%2C+Stephen+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459492/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459492/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459492/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459492/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459492/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459492/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459492/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459492/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459492/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459492/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459492/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459492/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/7/1151" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/7/1151.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/7/1151.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/7/1151.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/7/1151.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
