<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/7/1095" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/7/1095.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/7/1095.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/7/1095.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/7/1095.atom" />
<meta name="HW.pisa" content="ggg;3/7/1095" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?" />
<meta name="DC.Identifier" content="10.1534/g3.113.006510" />
<meta name="DC.Date" content="2013-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 White et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Empirical proof of human mitochondrial DNA (mtDNA) recombination in somatic tissues was obtained in 2004; however, a lack of irrefutable evidence exists for recombination in human mtDNA at the population level. Our inability to demonstrate convincingly a signal of recombination in population data sets of human mtDNA sequence may be due, in part, to the ineffectiveness of current indirect tests. Previously, we tested some well-established indirect tests of recombination (linkage disequilibrium vs. distance using D′ and r2, Homoplasy Test, Pairwise Homoplasy Index, Neighborhood Similarity Score, and Max χ2) on sequence data derived from the only empirically confirmed case of human mtDNA recombination thus far and demonstrated that some methods were unable to detect recombination. Here, we assess the performance of these six well-established tests and explore what characteristics specific to human mtDNA sequence may affect their efficacy by simulating sequence under various parameters with levels of recombination (ρ) that vary around an empirically derived estimate for human mtDNA (population parameter ρ = 5.492). No test performed infallibly under any of our scenarios, and error rates varied across tests, whereas detection rates increased substantially with ρ values &gt; 5.492. Under a model of evolution that incorporates parameters specific to human mtDNA, including rate heterogeneity, population expansion, and ρ = 5.492, successful detection rates are limited to a range of 7−70% across tests with an acceptable level of false-positive results: the neighborhood similarity score incompatibility test performed best overall under these parameters. Population growth seems to have the greatest impact on recombination detection probabilities across all models tested, likely due to its impact on sequence diversity. The implications of our findings on our current understanding of mtDNA recombination in humans are discussed." />
<meta name="DC.Contributor" content="Daniel James White" />
<meta name="DC.Contributor" content="David Bryant" />
<meta name="DC.Contributor" content="Neil John Gemmell" />
<meta name="article:published_time" content="2013-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Empirical proof of human mitochondrial DNA (mtDNA) recombination in somatic tissues was obtained in 2004; however, a lack of irrefutable evidence exists for recombination in human mtDNA at the population level. Our inability to demonstrate convincingly a signal of recombination in population data sets of human mtDNA sequence may be due, in part, to the ineffectiveness of current indirect tests. Previously, we tested some well-established indirect tests of recombination (linkage disequilibrium &lt;i&gt;vs.&lt;/i&gt; distance using D′ and r&lt;sup&gt;2&lt;/sup&gt;, Homoplasy Test, Pairwise Homoplasy Index, Neighborhood Similarity Score, and Max χ&lt;sup&gt;2&lt;/sup&gt;) on sequence data derived from the only empirically confirmed case of human mtDNA recombination thus far and demonstrated that some methods were unable to detect recombination. Here, we assess the performance of these six well-established tests and explore what characteristics specific to human mtDNA sequence may affect their efficacy by simulating sequence under various parameters with levels of recombination (ρ) that vary around an empirically derived estimate for human mtDNA (population parameter ρ = 5.492). No test performed infallibly under any of our scenarios, and error rates varied across tests, whereas detection rates increased substantially with ρ values &amp;gt; 5.492. Under a model of evolution that incorporates parameters specific to human mtDNA, including rate heterogeneity, population expansion, and ρ = 5.492, successful detection rates are limited to a range of 7−70% across tests with an acceptable level of false-positive results: the neighborhood similarity score incompatibility test performed best overall under these parameters. Population growth seems to have the greatest impact on recombination detection probabilities across all models tested, likely due to its impact on sequence diversity. The implications of our findings on our current understanding of mtDNA recombination in humans are discussed.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/07/01" />
<meta name="citation_mjid" content="ggg;3/7/1095" />
<meta name="citation_id" content="3/7/1095" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/7/1095" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/7/1095.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/7/1095.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/7/1095.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006510" />
<meta name="citation_pmid" content="23665874" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1095" />
<meta name="citation_lastpage" content="1104" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel James White" />
<meta name="citation_author_institution" content="Biodiversity and Informatics, Landcare Research New Zealand, Auckland, 1072" />
<meta name="citation_author_email" content="whited@landcareresearch.co.nz" />
<meta name="citation_author" content="David Bryant" />
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, University of Otago, Dunedin, 9054, New Zealand" />
<meta name="citation_author_institution" content="Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand" />
<meta name="citation_author" content="Neil John Gemmell" />
<meta name="citation_author_institution" content="Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand" />
<meta name="citation_author_institution" content="Centre for Reproduction and Genomics, Department of Anatomy, University of Otago, Dunedin, 9054, New Zealand" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Anisimova;citation_author=R. Nielsen;citation_author=Z. H. Yang;citation_title=Effect of Recombination on the Accuracy of the Likelihood Method for Detecting Positive Selection at Amino Acid Sites;citation_pages=1229-1236;citation_volume=164;citation_year=2003;citation_issue=3;citation_pmid=12871927" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Arenas;citation_author=D. Posada;citation_title=Recodon: coalescent simulation of coding DNA sequences with recombination, migration and demography.;citation_pages=458-458;citation_volume=8;citation_year=2007;citation_pmid=18028540;citation_doi=10.1186/1471-2105-8-458" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Arenas;citation_author=D. Posada;citation_title=Coalescent simulation of intracodon recombination.;citation_volume=184;citation_year=2010;citation_pmid=19933876;citation_doi=10.1534/genetics.109.109736" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Arenas;citation_author=D. Posada;citation_title=The effect of recombination on the reconstruction of ancestral sequences.;citation_volume=184;citation_year=2010;citation_pmid=20124027;citation_doi=10.1534/genetics.109.113423" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Awadalla;citation_author=A. Eyre-Walker;citation_author=J. M. Smith;citation_title=Linkage Disequilibrium and Recombination in Hominid Mitochondrial DNA;citation_pages=2524-2525;citation_volume=286;citation_year=1999;citation_issue=5449;citation_pmid=10617471;citation_doi=10.1126/science.286.5449.2524" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. J. Brown;citation_author=E. C. Garner;citation_author=A. K. Dunker;citation_author=P. Joyce;citation_title=The Power to Detect Recombination Using the Coalescent;citation_pages=1421-1424;citation_volume=18;citation_year=2001;citation_issue=7;citation_pmid=11420381" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. C. Bruen;citation_author=H. Philippe;citation_author=D. Bryant;citation_title=A simple and robust statistical test for detecting the presence of recombination.;citation_volume=172;citation_year=2006;citation_pmid=16489234;citation_doi=10.1534/genetics.105.048975" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=M. D’Aurelio;citation_author=C. D. Gajewski;citation_author=M. T. Lin;citation_author=W. M. Mauck;citation_author=L. Z. Shao;citation_title=Heterologous mitochondrial DNA recombination in human cells.;citation_volume=13;citation_year=2004;citation_pmid=15496432;citation_doi=10.1093/hmg/ddh326" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. L. Elson;citation_author=R. M. Andrews;citation_author=P. F. Chinnery;citation_author=R. N. Lightowlers;citation_author=D. M. Turnbull;citation_title=Analysis of European mtDNAs for recombination.;citation_pages=145-153;citation_volume=68;citation_year=2001;citation_issue=1;citation_pmid=11115380;citation_doi=10.1086/316938" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=A. Eyre-Walker;citation_author=N. H. Smith;citation_author=J. M. Smith;citation_title=How clonal are human mitochondria?;citation_pages=477-483;citation_volume=266;citation_year=1999;citation_issue=1418;citation_pmid=10189711;citation_doi=10.1098/rspb.1999.0662" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. M. Fontaine;citation_author=J. R. Cooley;citation_author=C. Simon;citation_title=Evidence for paternal leakage in hybrid periodical cicadas (Hemiptera: Magicicada spp.).;citation_pages=e892-e892;citation_volume=2;citation_year=2007;citation_issue=9;citation_pmid=17849021;citation_doi=10.1371/journal.pone.0000892" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=U. Gyllensten;citation_author=D. Wharton;citation_author=A. Josefsson;citation_author=A. C. Wilson;citation_title=Paternal inheritance of mitochondrial DNA in mice.;citation_pages=255-257;citation_volume=352;citation_year=1991;citation_issue=6332;citation_pmid=1857422;citation_doi=10.1038/352255a0" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=E. Hagelberg;citation_author=N. Goldman;citation_author=P. Lio;citation_author=S. Whelan;citation_author=W. Schiefenhovel;citation_title=Evidence for mitochondrial DNA recombination in a human population of island Melanesia;citation_pages=485-492;citation_volume=266;citation_year=1999;citation_issue=1418;citation_pmid=10189712;citation_doi=10.1098/rspb.1999.0663" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. Herrnstadt;citation_author=J. L. Elson;citation_author=E. Fahy;citation_author=G. Preston;citation_author=D. M. Turnbull;citation_title=Reduced-median-network analysis of complete mitochondrial DNA coding-region sequences for the major African, Asian, and European haplogroups.;citation_pages=1152-1171;citation_volume=70;citation_year=2002;citation_issue=5;citation_pmid=11938495;citation_doi=10.1086/339933" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Ingman;citation_author=H. Kaessmann;citation_author=S. Paabo;citation_author=U. Gyllensten;citation_title=Mitochondrial genome variation and the origin of modern humans.;citation_pages=708-713;citation_volume=408;citation_year=2000;citation_issue=6813;citation_pmid=11130070;citation_doi=10.1038/35047064" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=H. Innan;citation_author=M. Nordborg;citation_title=Recombination or Mutational Hot Spots in Human mtDNA?;citation_pages=1122-1127;citation_volume=19;citation_year=2002;citation_issue=7;citation_pmid=12082131;citation_doi=10.1093/oxfordjournals.molbev.a004170" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. L. Ivanov;citation_author=M. J. Wadhams;citation_author=R. K. Roby;citation_author=M. M. Holland;citation_author=V. W. Weedn;citation_title=Mitochondrial DNA sequence heteroplasmy in the Grand Duke of Russia Georgij Romanov establishes the authenticity of the remains of Tsar Nicholas II.;citation_pages=417-420;citation_volume=12;citation_year=1996;citation_issue=4;citation_pmid=8630496;citation_doi=10.1038/ng0496-417" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=I. B. Jakobsen;citation_author=S. Easteal;citation_title=A program for calculating and displaying compatibility matrices as an aid in determining reticulate evolution in molecular sequences;citation_pages=291-295;citation_volume=12;citation_year=1996;citation_issue=4;citation_pmid=8902355;citation_doi=10.1093/bioinformatics/12.4.291" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. B. Jorde;citation_author=M. Bamshad;citation_title=Questioning evidence for recombination in human mitochondrial DNA.;citation_pages=1931-1931;citation_volume=288;citation_year=2000;citation_issue=5473;citation_pmid=10877699;citation_doi=10.1126/science.288.5473.1931a" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Kivisild;citation_author=R. Villems;citation_title=Questioning Evidence for Recombination in Human Mitochondrial DNA;citation_pages=1931a-1931a;citation_volume=288;citation_year=2000;citation_issue=5473;citation_pmid=17835098;citation_doi=10.1126/science.288.5473.1931a" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Kraytsberg;citation_author=M. Schwartz;citation_author=T. A. Brown;citation_author=K. Ebralidse;citation_author=W. S. Kunz;citation_title=Recombination of Human Mitochondrial DNA;citation_pages=981-981;citation_volume=304;citation_year=2004;citation_issue=5673;citation_pmid=15143273;citation_doi=10.1126/science.1096342" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. K. Kuhner;citation_title=LAMARC 2.0: maximum likelihood and Bayesian estimation of population parameters.;citation_volume=22;citation_year=2006;citation_pmid=16410317;citation_doi=10.1093/bioinformatics/btk051" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kumar;citation_author=P. Hedrick;citation_author=T. Dowling;citation_title=Questioning evidence for recombination in human mitochondrial DNA.;citation_pages=1931-1931;citation_volume=288;citation_year=2000;citation_issue=5473;citation_pmid=10877701;citation_doi=10.1126/science.288.5473.1931a" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=U. Lakshmipathy;citation_author=C. Campbell;citation_title=The Human DNA Ligase III Gene Encodes Nuclear and Mitochondrial Proteins;citation_pages=3869-3876;citation_volume=19;citation_year=1999;citation_issue=5;citation_pmid=10207110;citation_doi=10.1128/MCB.19.5.3869" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. McVean;citation_author=P. Awadalla;citation_author=P. Fearnhead;citation_title=A Coalescent-Based Method for Detecting and Estimating Recombination From Gene Sequences;citation_pages=1231-1241;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901136" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Meunier;citation_author=A. Eyre-Walker;citation_title=The Correlation Between Linkage Disequilibrium and Distance: Implications for Recombination in Hominid Mitochondria;citation_pages=2132-2135;citation_volume=18;citation_year=2001;citation_issue=11;citation_pmid=11606711;citation_doi=10.1093/oxfordjournals.molbev.a003756" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Mishmar;citation_author=E. Ruiz-Pesini;citation_author=P. Golik;citation_author=V. Macaulay;citation_author=A. G. Clark;citation_title=Natural selection shaped regional mtDNA variation in humans.;citation_volume=100;citation_year=2003;citation_pmid=12509511;citation_doi=10.1073/pnas.0136972100" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=G. Piganeau;citation_author=A. Eyre-Walker;citation_title=A reanalysis of the indirect evidence for recombination in human mitochondrial DNA.;citation_pages=282-288;citation_volume=92;citation_year=2004;citation_issue=4;citation_pmid=14747833;citation_doi=10.1038/sj.hdy.6800413" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=G. Piganeau;citation_author=M. Gardner;citation_author=A. Eyre-Walker;citation_title=A broad survey of recombination in animal mitochondria.;citation_volume=21;citation_year=2004;citation_pmid=15342796;citation_doi=10.1093/molbev/msh244" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Pluzhnikov;citation_author=A. Di Rienzo;citation_author=R. R. Hudson;citation_title=Inferences About Human Demography Based on Multilocus Analyses of Noncoding Sequences;citation_pages=1209-1218;citation_volume=161;citation_year=2002;citation_issue=3;citation_pmid=12136023" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Posada;citation_author=K. A. Crandall;citation_title=Evaluation of methods for detecting recombination from DNA sequences: Computer simulations;citation_pages=13757-13762;citation_volume=98;citation_year=2001;citation_issue=24;citation_pmid=11717435;citation_doi=10.1073/pnas.241370698" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. H. Schierup;citation_author=J. Hein;citation_title=Consequences of Recombination on Traditional Phylogenetic Analysis;citation_pages=879-891;citation_volume=156;citation_year=2000;citation_issue=2;citation_pmid=11014833" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. H. Schierup;citation_author=J. Hein;citation_title=Recombination and the Molecular Clock;citation_pages=1578-1579;citation_volume=17;citation_year=2000;citation_issue=10;citation_pmid=11018163;citation_doi=10.1093/oxfordjournals.molbev.a026256" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=M. Schwartz;citation_author=J. Vissing;citation_title=Paternal Inheritance of Mitochondrial DNA;citation_pages=576-580;citation_volume=347;citation_year=2002;citation_issue=8;citation_pmid=12192017;citation_doi=10.1056/NEJMoa020350" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; genetic systems;citation_journal_abbrev=Genes Genet Syst;citation_author=W. Sherengul;citation_author=R. Kondo;citation_author=E. T. Matsuura;citation_title=Analysis of paternal transmission of mitochondrial DNA in Drosophila.;citation_pages=399-404;citation_volume=81;citation_year=2006;citation_issue=6;citation_pmid=17283385;citation_doi=10.1266/ggs.81.399" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. M. Smith;citation_title=Analyzing the mosaic structure of genes.;citation_pages=126-129;citation_volume=34;citation_year=1992;citation_issue=2;citation_pmid=1556748;citation_doi=10.1007/BF00182389" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. M. Smith;citation_author=N. H. Smith;citation_title=Detecting recombination from gene trees;citation_pages=590-599;citation_volume=15;citation_year=1998;citation_issue=5;citation_pmid=9580989;citation_doi=10.1093/oxfordjournals.molbev.a025960" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Sun;citation_author=B. J. Evans;citation_author=G. B. Golding;citation_title=“Patchy-Tachy” leads to false positives for recombination.;citation_volume=28;citation_year=2011;citation_pmid=21498600;citation_doi=10.1093/molbev/msr076" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=B. Thyagarajan;citation_author=R. A. Padua;citation_author=C. Campbell;citation_title=Mammalian Mitochondria Possess Homologous DNA Recombination Activity;citation_pages=27536-27543;citation_volume=271;citation_year=1996;citation_issue=44;citation_pmid=8910339;citation_doi=10.1074/jbc.271.44.27536" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Tsaousis;citation_author=D. P. Martin;citation_author=E. D. Ladoukakis;citation_author=D. Posada;citation_author=E. Zouros;citation_title=Widespread recombination in published animal mtDNA sequences.;citation_volume=22;citation_year=2005;citation_pmid=15647518;citation_doi=10.1093/molbev/msi084" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. J. White;citation_author=N. J. Gemmell;citation_title=Can indirect tests detect a known recombination event in human mtDNA?;citation_volume=26;citation_year=2009;citation_pmid=19369597;citation_doi=10.1093/molbev/msp073" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=D. J. White;citation_author=J. N. Wolff;citation_author=M. Pierson;citation_author=N. J. Gemmell;citation_title=Revealing the hidden complexities of mtDNA inheritance.;citation_pages=4925-4942;citation_volume=17;citation_year=2008;citation_issue=23;citation_pmid=19120984;citation_doi=10.1111/j.1365-294X.2008.03982.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. Wiuf;citation_author=T. Christensen;citation_author=J. Hein;citation_title=A Simulation Study of the Reliability of Recombination Detection Methods;citation_pages=1929-1939;citation_volume=18;citation_year=2001;citation_issue=10;citation_pmid=11557798;citation_doi=10.1093/oxfordjournals.molbev.a003733" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=Z. H. Yang;citation_title=Among-site rate variation and its impact on phylogenetic analyses.;citation_pages=367-372;citation_volume=11;citation_year=1996;citation_issue=9;citation_pmid=21237881;citation_doi=10.1016/0169-5347(96)10041-0" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=X. Zhao;citation_author=N. Li;citation_author=W. Guo;citation_author=X. Hu;citation_author=Z. Liu;citation_title=Further evidence for paternal inheritance of mitochondrial DNA in the sheep (Ovis aries).;citation_pages=399-403;citation_volume=93;citation_year=2004;citation_issue=4;citation_pmid=15266295;citation_doi=10.1038/sj.hdy.6800516" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=G. Zsurka;citation_author=K. G. Hampel;citation_author=T. Kudina;citation_author=C. Kornblum;citation_author=Y. Kraytsberg;citation_title=Inheritance of mitochondrial DNA recombinants in double-heteroplasmic families: potential implications for phylogenetic analysis.;citation_pages=298-305;citation_volume=80;citation_year=2007;citation_issue=2;citation_pmid=17236134;citation_doi=10.1086/511282" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. Zsurka;citation_author=Y. Kraytsberg;citation_author=T. Kudina;citation_author=C. Kornblum;citation_author=C. E. Elger;citation_title=Recombination of mitochondrial DNA in skeletal muscle of individuals with multiple mitochondrial DNA heteroplasmy.;citation_pages=873-877;citation_volume=37;citation_year=2005;citation_issue=8;citation_pmid=16025113;citation_doi=10.1038/ng1606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="twitter:description" content="Empirical proof of human mitochondrial DNA (mtDNA) recombination in somatic tissues was obtained in 2004; however, a lack of irrefutable evidence exists for recombination in human mtDNA at the population level. Our inability to demonstrate convincingly a signal of recombination in population data sets of human mtDNA sequence may be due, in part, to the ineffectiveness of current indirect tests. Previously, we tested some well-established indirect tests of recombination (linkage disequilibrium vs. distance using D′ and r2, Homoplasy Test, Pairwise Homoplasy Index, Neighborhood Similarity Score, and Max χ2) on sequence data derived from the only empirically confirmed case of human mtDNA recombination thus far and demonstrated that some methods were unable to detect recombination. Here, we assess the performance of these six well-established tests and explore what characteristics specific to human mtDNA sequence may affect their efficacy by simulating sequence under various parameters with levels of recombination (ρ) that vary around an empirically derived estimate for human mtDNA (population parameter ρ = 5.492). No test performed infallibly under any of our scenarios, and error rates varied across tests, whereas detection rates increased substantially with ρ values &gt; 5.492. Under a model of evolution that incorporates parameters specific to human mtDNA, including rate heterogeneity, population expansion, and ρ = 5.492, successful detection rates are limited to a range of 7−70% across tests with an acceptable level of false-positive results: the neighborhood similarity score incompatibility test performed best overall under these parameters. Population growth seems to have the greatest impact on recombination detection probabilities across all models tested, likely due to its impact on sequence diversity. The implications of our findings on our current understanding of mtDNA recombination in humans are discussed." />
<meta name="og-title" property="og:title" content="How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/7/1095" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Empirical proof of human mitochondrial DNA (mtDNA) recombination in somatic tissues was obtained in 2004; however, a lack of irrefutable evidence exists for recombination in human mtDNA at the population level. Our inability to demonstrate convincingly a signal of recombination in population data sets of human mtDNA sequence may be due, in part, to the ineffectiveness of current indirect tests. Previously, we tested some well-established indirect tests of recombination (linkage disequilibrium vs. distance using D′ and r2, Homoplasy Test, Pairwise Homoplasy Index, Neighborhood Similarity Score, and Max χ2) on sequence data derived from the only empirically confirmed case of human mtDNA recombination thus far and demonstrated that some methods were unable to detect recombination. Here, we assess the performance of these six well-established tests and explore what characteristics specific to human mtDNA sequence may affect their efficacy by simulating sequence under various parameters with levels of recombination (ρ) that vary around an empirically derived estimate for human mtDNA (population parameter ρ = 5.492). No test performed infallibly under any of our scenarios, and error rates varied across tests, whereas detection rates increased substantially with ρ values &gt; 5.492. Under a model of evolution that incorporates parameters specific to human mtDNA, including rate heterogeneity, population expansion, and ρ = 5.492, successful detection rates are limited to a range of 7−70% across tests with an acceptable level of false-positive results: the neighborhood similarity score incompatibility test performed best overall under these parameters. Population growth seems to have the greatest impact on recombination detection probabilities across all models tested, likely due to its impact on sequence diversity. The implications of our findings on our current understanding of mtDNA recombination in humans are discussed." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
    <title>How Good Are Indirect Tests at Detecting Recombination in Human mtDNA? | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ufG6Cvkabpcu5jR_wEr7zAAUpr9UvXVeXew9DX0kZeU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459493","apath":"\/ggg\/3\/7\/1095.atom","pisa":"ggg;3\/7\/1095","ac":{"\/ggg\/3\/7\/1095.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/7\/1095.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1095"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1095"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/7\/1095.full":true,"\/content\/3\/7\/1095":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lMsuTFa6F5RJ3jaTCUHOOkFKWWmJLlRThKqryQrrb3TB6DJknkbTA4hsvAQrZWynad1ORATVzZP7\\\\\\\/uwAjFtW7zgLMjQTftQV6NAkhdqUfWT9byanLdWhB8MkM7CsnJt5z1FUbwaxMK0wqF8wVHs8IgUtWqaA3ZzgfP+VtF32deAvcccnlRySH3nadBg7EyMWrQI1egu\\\\\\\/AKoiTP2tNEV5zTBNZsR6zUZLWw7rHaF3mSvIjfy3idNOvfCXrvdmaxfkj9VXPHesyflUq28SzDUAPPYZJ7XcDdl50nxoTe35l1e6otsQGWa5Ji9PDa9+n8g6CWSw8zL4jEIUlQfOQSRMr95xJ0wfCe1jBSIK9RZqAR6v\\\\\\\/c+rQNYYgjP9g0GjXj6RiDJsF9Y1XhN2jzP+6LvAyVvLdWpPodzJhMk9OL\\\\\\\/ourfyk9Tkhkzn1ArYUDibfBk69+PueVL8ZuuYYO1AjrGDpCrftuGYf3XXMU7ojVaS+uFrtfI\\\\\\\/qpBioJ0n3T3oEot4KfyjOUe22PMUPVJtgp4w9wJmMDB93IR\\\\\\\/oDzXqQpeswHCNsga5ejwOlkWauTRM2pJLHQqq\\\\\\\/QOo22QThmLWesxP\\\\\\\/sFsuj1zWS29+QoKWrpGnlrs2N1YPFDc1nP9SS0WD7tmuF\\\\\\\/CmVUK3PZsvnVgXQbzA4bTqRIR9oyhaT3X8\\\\\\\/XZSNWV9Har27k0Srt0x3TB7NCT2lYU4Oc7B3SQ6H8ZOenHykLI0Ka9nKXwQzagQ88Ypp2JxXDRq0TkXSL3+VYWA0O1yw2hQjmq45NfbGdsCq4U2MBsSSv54ZC8K0l8z5GTW8\\\\\\\/hgj\\\\\\\/B2vSomADvCiM8fvfyRGHV6OjKIYbSUeWYPUga+NaTutGFlL+mXpwkyZTtYsKXEheHoCSr\\\\\\\/ACLAsMdHF4I7gpHahoWuV8g+74rvLmf+jqIwCuVmc0Bsg+CBv9WmKJB8CMVmHbNsVzmpQ2kv5U9HM7xG2VmwiO39UceEijvcIUc4bBggwCEgUTHlnVfYvKkx6Eho1En9RzjN0BeZzLAwYMgd7sBkITLOG4l935F\\\\\\\/unRW8BwrT0iPwhOTrSPMPgZDCdMOEOmzU8agNT+lkjYW+0y5L8kVTE5QY6Liic1\\\\\\\/XhYFmKYqiRkG3lDKGS4FA+6DQ89iU5iEr+8spmdpvpgdBUden85HB7D9SPAB3ckco3N5XWsVzqd+JuTzN9MfOELQMiDkC93UDSZD44JtVCN\\\\\\\/JD8wYCNuI3VxBacOEvGTi+CaCQ9kPfSfw1kfgG7C3jyR+d12lOT1klQa+M3vCX\\\u0022,\\\u0022iv\\\u0022:\\\u0022uuu+c692g4euQKgPEDq1yg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a16649d751d6586977b20b766c013d62fccc5ce39ad13c473c9e464e1a86e50c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+M64bXQ2HHqPI5FfPR84SHYrFdYibFkesf+HRn+0Kw0UBcBpYz2W0caH8jX2RFkon14cc3coxka\\\\\\\/y3RJjGhXhq4+Bz9aVRDGSqDptB\\\\\\\/30kWAMVZ2PyeW2PX6zX0uYoXn6PMKzw3LPe+Syx7IGJGlc9LDyeaQRzNUoQJub5\\\\\\\/Na83vbvu51eHGX6DTC3jwpUMLokD2+nhQCLD+cW\\\\\\\/2XpoUJBL+y40A+m8EMOUN5Amgr34X+RqNb9b5kIxLsIkCT4MxCZWCThW8AfKMjydAtl2yWEK0Mfpy4B6F2\\\\\\\/F6kmpUnPJJQa+rD5ZuG1GIfO0LrJZUPZFPyC3WnSLkpQrFsIuKn+douu9cr6cr\\\\\\\/YrztH5\\\\\\\/gKRaNSB83TWtKkJDUM7UPWMeKH8X+BWsF9E8crfB5L717o\\\\\\\/T6\\\\\\\/cIAS\\\\\\\/+hSzGtRAGVxBEVB9rilnbTHOAZO\\\\\\\/ZK4b+fcA744BlAkjwCLsluFZZWgLU71d4AQg28vHJlPfYpFwGqvemKp+Dd75CKoXTErZVJfxyTvorxgtsI3mCWRuMGngctqiW+JL8ooAMXxDEL30laPHihbh6cm3\\\\\\\/xgYt92QZYXzYzosspQfoRBzyGbLTTRmqoWHHa4NrUmvRswDDhcipqyLSSxLc7\\\\\\\/AZ19Ef6xdOYXZpdhSLpx42WNNbvXLgAaagCc7OcS6SdEqrxHrn5C6HENf27rlRtw6DoYzT7AXatLNIl6l+dQBtT2KNdWwMW5aM0KIv1+gQUhS\\\\\\\/GOA3Wbe+bO7EWGHXnsbGtTxfoHDch4rT9GNNdOvjKRzz9jNHtkgFoUCEO7Z6ivyz\\\\\\\/oNKjT\\\\\\\/wZ9n3bmXxP0g+2wuZ8fz77bvVkmvQ27cCbUUMFxTiQ1nO8DFumwYvQj+dcMII14YQD5q0kHaepcNXqQjyzaoEdS2Soi6s8XfciYp\\\\\\\/RuOQoagE192aFnsDY8dwWcy9jXSC9lKG6G4nlOkLKE7fDBzaqInmadtu91\\\\\\\/Q5\\\\\\\/RU6vci3Yth2Ux5hQJKgLN0t7UZPAkr14KkVwo8BE5GGrb35sp2Xeycy+EBy5UK\\\\\\\/TZSHYwdqo4oG3BIoTc+qu1B7tf2qQQMLHi91gJWWwCXHpuw0OEb6yo90ZWnDDRjy1EImxjvITrB6ZECmGxSCxjfuYo3epMIsPW1xzVrzFM\\\\\\\/EP\\\\\\\/P+0R1xE8S8QQQn6HEBXNvsGRKvGrcOLqv5\\\\\\\/UqtvH92r0pvssKcFmotmknDonnOVXpvZUOjD4xjzgHJ7EGy3QPBubSsMc6aUkoQZg8qlG6vYakLOQ884h+UDVBzQj1AjBZWaN+ES6e8TRxH33I43S\\\\\\\/Lm1XUf\\\\\\\/NsEJMaMyJynBhA7e9\\\\\\\/q+OzM7RAGWrDEsJCTZ9FTe+bbXrcB1j8kEh64mp6bftAiu3j1evYsE72thzAENVBQFlPnaLAV54a90LjissUlWww10f4Q2c4d83k04JgKsdoKn\\\\\\\/SE1a1Cz\\\\\\\/trA6pFgWtoD53xKgNynSqx7GObtD7HwjrPhlGT+vs8chUro6zhxcUlucLWLQ72Xt5x8XvNVG4z9Hg\\\\\\\/O2ctpqpItzjTTZTSgJRpOkR7pVeBr0o2Dpbx9IE+eIIEL83unmgMrFLr9KiE8z5g+fE1CVaLCLevvOIFYUI5k7vAAbkqmX1jYb7HGCeFAiyGjk4QXNDVpwauTrHwtvn1KgrqTznEuvOrUCdIxy3msf6zx28WX2xiDgtn8DVeaqhXX+2Hgm3Nl5WiDMglHJ45CsPYLwOauaerHKxt4SgnE84UmucTuZuKk5LSFMyrBNWfuX46KuAtjj66S1epqYXFbO0jf9CwKnh7qiVk0xCAMDIRpColu6b4VuHUisF1VwFYcZyjplstnU1\\\\\\\/MbT3Mjyplczymf3msTaCFEMba+kCpXsxc8DjjEYNqyomvf+PfpzEacgZUodxrVn32YQSTP8wGsQb82FILdBIvU7hgXtOOkCMzZ9QP9CSxMSLtqzTSOOQGXk30aa8CZV7eCm2U8HY51uxJo3HLWyECramgkdDae2S5Fn8Y9FHQkbERRzSccYz1aXs9ceGyCo+j1F0I2b2HFYcR6KaCyDyB6PIU0sHGVRm8ea38eTsulLVudaHJ7foNsfdQikIYGT4NMZR0Pyz+0frta+daHxeUaI5tqivjdW6Y29HxYkR6f3u1eoisgIjhU2H2hjnM\\\\\\\/1nRqkLtN0ROcNYk99VvYeYyeep0PPflexyepRgYxPj7t1fFlxv\\\\\\\/bGkLFIFi93W9pYq2j3NR09UTkydxcAQuoD90DoBxISrphM+X8oMuUaejS3n5K+bcmQ+YaGaV+Fznk\\\\\\\/wX6oN\\\\\\\/\\\\\\\/AjrObHIgazIEpqz2ZnWFXDH5YI0p9alLYl1vT9Sze1p8HGdX+BA4cjw2C6sPtWGfAM\\\\\\\/B+euaNjSKyho7WMW5Ogm\\\\\\\/EPrdWXmTzCu\\\\\\\/xKfV0KUImOrRMLezttGihlT7xCI9B3Du6CYJ4DqmY6Tel2UDReYy954CPslpEZP6eotz1tmY4LSH4czba4RJipQH7SU8B49c5MSteAoES3anj6TdBgut6XQ9OoZtIpO+fn+1Vk2+RxDJ5TojHVeqgUNmz6NC9y5Gfm88HLjjegCDkxHUTUSfONRvEsdwgg4negxI2LWhMBOcc377TAlrQX5YIHab\\\\\\\/b1Em1o\\\\\\\/5IWrsFFQQmugujIgw7q69hJW2GSVNUESxIRt67o7tT7fbIh9\\\\\\\/0WSRlhIpU9XxR2hWy+j0sic0kJh9I7wX2+RHyfedUFowjiYgYfvUp00I\\\\\\\/tgnhN47EAmD2m\\\\\\\/FkmWX0Ls4kGBNDNrdp42sbjyfnPM4pT4kNN86mYbReLXc5hBazJqqNkGK352Ypza6UZeKxpE6wGNQ6oKNp0NQPgVLvYp4fMrrx9GtPnQmBFCJ+\\\\\\\/Xwhp+uZ1Wi+wcQ8XHU1TO3c2JWv911rgbQmvbTH9MO2JHkw+spNzK+yfkpk4IF1S7DoifUdxU20CTmNegfYdzB78cOGkynbYkKBecuj2Oa\\\\\\\/DSZnerMV3O7rW5RNqjC9sTBB1sjdjae5NO\\\\\\\/Y3Vj6TkXlt5IY+SCcS34cNkgtbfOXPaHA0QzakkZ2DT\\\\\\\/titjimvdyqEoQJ6x8Vla+wd8owD8M77FyCKiQWnh5O59\\\\\\\/AyMZ1M9gAY1YNrz7mGQAxk\\\\\\\/t5Zb\\\\\\\/Hf14RijpQVa5Ihey6A9lGGWI36oqGiXfFRurcbaawrye4rlvn5INZB+xHGQibexBtcBkl9mqRE\\\\\\\/eunCN\\\\\\\/L8tq8qcceqQyXlE8Wzf77yLHiV2\\\\\\\/Y+8\\\\\\\/7qT1N6fCp+c5urYEiF1yAMpTF55J3wXudbUhJk2yxh4oBl3Ps+NhHjmRH6+XWxiLwnsH0y1sNlvuV+IlO\\\\\\\/+ilRycquIxZwczpg7dkciHu\\\\\\\/xRcxCvc3ElEvsrL5LjP3uuU5\\\\\\\/wfCTEsNoS9Etl9JpJYY8iFQckMxv0HrRxCLD3tcEW9zC\\\\\\\/iKTNoWXkvxbYezilJaPFXegw408CezfJeTbFZibaxuG4MwpuQqCHcDH2ZOQJ972USC\\\\\\\/GIb37N\\\\\\\/+Um9r6j18C6RhJVAH3+18QwQONuUcqrFvqAbR84lExSZnk\\\\\\\/McZqjGJw70aCNuhcHdbgbAiFGLLjNFsVExUpk02OibYKcyfbSFup9dTG6ZzKXxLxEoF6h7uQZxoyxzcR4zlwR8zk4Jy40YIropTiFUUlFoHXXz0UWF68I7t0Yv6LsuHePHJj3VsX2as9qGkB\\\\\\\/\\\\\\\/a0paCzQmKE6wG99oIE9XVqCLe+CbwJMKbaqqxlZ5A4gWzx9vrqG7xWvLz5G44kxjiTCEKjeTWg8Uc1BnxcOMIJTuD4C70MqwBl8vATEuUzSG8EHTHwGBz4k9R8nHp4e1Eb3Oo\\\\\\\/SikUMldGuwFIEcpBkdDwK3nceP3MNEC1Matq9CAiMEFZ4t6pJiXnvw+Ih7pMhbQEESIwdkjxisA2KtWANZJxHIBJ4rPPz9SEuJwvjergJ68AAyZDhFclM1zZr7ttY891QGi8qIps6+LiUGAF+B3xqhzhORD6HxM1yS1k7D1A612UCLp\\\\\\\/MHtFCaXyBZQjubpFtTpiuu\\\\\\\/z\\\\\\\/M\\\\\\\/xaheOaCs6TUt4J2tVlU62LMexVbHg2dhIa\\\\\\\/t7Ra0k0VfArs7ofr3foivxVygOynr0RQzRDY9ITWH5hrkAj5R3J\\\\\\\/IWsuYAr2M95ts17Q6hrtRpW2ODV9ao\\\\\\\/tbej95nneCeQOxbE13txKvV+a7a8yLAOIRHOc+ABriutPk2fV66TSW0Igb0NFQUwHoAyo5VEbmMvTElF9G2\\\\\\\/3D4OGS07IlMeFiS9DrqS+Gm4rzid+giN5qKMySP8Q2oIr1BWO5LHx3VZ+Fmt17YndcggjmAFW2OzRBu0fHW82S4x4JYitdCm5tdnokxKwVKptJNyxwW4NWbbzCF4AWwpcwePSlx2V+ek0XKe27Vhh2+ilpzZ4tg00qOEdEpqLVb+T\\\\\\\/zo0YIuGEbJiZz5BHKY30PwSj0p1kPxT2w4dnbinGqp7rJ+mRMDJoKCD2KLDRbUDJkxV0w70L29OBJbnKrOz\\\\\\\/a422Bbexf+TQsI7npmGDmP+4otYoVaBhpv7vMxVm+qAhb7e2rVZbyUuJhBgEiTL7pbWWIYltQU5Bti6bzcB56WxoINe8aGc3kUmFajIHi62WP6BjE7r9\\\\\\\/rk9D\\\\\\\/kBIrbxZSwvXl+v0oT0Pk49dr8wsrEyB\\\\\\\/GBPi+spgb\\\\\\\/9TwF+3wfA4G0e0IhbElwwRUt6cLnlWK0z7QKMqBlD\\\\\\\/ufrHPeuseO8jVVUxCsXOab442S4YeNTvsz0HK9IK+LcIGWsIpwqnVLFnqOVSy53W66Htt8fAm5KDvns87Eyn6rRmXOpjBOpqjjBJtkOohRSNrZAJR0pLFZRd2rXguOaA8N9pVn6W+8pqUIFXrCXbOd55n5AzZgkw9ubw7B5YHX4Z\\\\\\\/0kapxhq\\\\\\\/MXwjaaw81Xts8tRc2UV7pdfgQKC1vPED008qsvXgFm\\\\\\\/Ys8oHtHhxsDNowsaexz75ywkNT3Stlf8NyUI+cxBTx6Q+V56gScw7xr12kBrC2vAZzgiJtlSEupH9PW\\\\\\\/91qbTKEcTENAs+BIlTvWeUuvpQZmemhiqCQa1pg+andG+\\\\\\\/MMltgKLTcehaGl3V0Jns48RwEtwP8Ht4dzEc7ruJim1L59f1+t7h32CNrT1Hlto6qhTPitrRH+y7ZrmgrC1Yx\\\\\\\/vA8Eb\\\\\\\/bYth3jRz9Y8Dw2A2eAN6WJ2HWdyDAivZ4NRvI4bBwTQU+\\\\\\\/4vq4XPfODu\\\\\\\/v2n70NElde1eu\\\\\\\/BpYzbhG2lsHqWZsFoj3KNkhWTRVSz2ZvM3h4yAuOtIBFlxj0ygbSskmX86z7FPwsr8zFI2+oJ8PkhxnBB30pR\\\\\\\/JrpL\\\\\\\/7FdI1n0TytfOkFY\\\\\\\/3ZPYrSyV5LSm6jged5hL9Wx9NUNc81Q365oQ1Nrnzfp33+MWK0wz7NpU4Jd8lkUTUpsFapvunu\\\\\\\/L0oMzciLKIM+9VJ4uoaTBxlRy3zWhBZ+W3IB7pSBaErcyNLiQ5M7x3xnFW72PlWjznnTeavT\\\\\\\/\\\\\\\/Uyn\\\\\\\/m5SHkDEy7fHNdIL57VhJaruR9nAWHo1iud2mGa0Ol9TimHu2uvK6k+OGHy5Zx9KS8\\\\\\\/ubBVY0oQb8zD4DOu\\\\\\\/zR9z9gdeRsXyakU+3BF7OhciPMlNO5QhYrjzEHpVMHPa7GI1Gu7IfSdJcPccnmDJlT4wIlS5rDnTJGiO6fFBMXSkDRDupQrR+0aJkA+QFsak6JFeVJFNU\\\\\\\/ts4r46rFHpyu2Nk2iH3VU5i8OUxBL4Wlb1zLqT4oGQDro4VKK6nLPFmJHKlNMzJRfJZENqO2ar4tHXJrs8F6Nw+XnWX4MvOY6naoQ64nCHfVAVJ37zwfGHPAwFEduTrUXZS24V1H6QAwmwqCU\\\\\\\/fbz\\\\\\\/9Ody+vOY9USeSVQLDvWPqPp\\\\\\\/gkmfKtdQLCeEcXsfG+mb58AbczA8AptBGKzzvRObQ+9DH2yK9OnBEKtNaP3Nvz7zpsKARM0UYIxFvL9ANkyNowWQkyd+8POe7Cju8v2OFUoLH1OaHz0\\\\\\\/f+1oD71hafaqqcjCAonQEauywFDOsXUbI0eyyUAFYj0E5CXNNcksv+1uHohbBW8nMUz9xyPggO5WeYg56RctLkxMV9xiGbd9730EO8URbtTqy36ddgqQ+FbcsPXXtg6OoYn4Y2Rk9w93erTejkhpVodizKjk0fO4c0AVObgt9BaOrS\\\\\\\/d4wOPhdGtomAflquc5CZhjPsWxGRdLVzKVWSaVH7X3ZHuqL8EMJnmObglCj6LBmRqxA9qHs5wpX5ciZNLhmkYLZDVjdEZjw7UccMGN5Jy4iotMOroIQ4e9kCejw\\\\\\\/HBHiGhLvghW83A2G55NatLugAoH4bDiA7I+OewF1HTBzXLV7MfNbtaNOvzXdgTTUvo9sFbo8uHoUgi11Cr+A5stqEcbb+7okTVBwSQaJ0Yr7RMAHeEwY+igj3v\\\\\\\/yG9tiBqtRvYfCVm\\\\\\\/VEK4m7U2iH+XjH5yMC4J2n00ZS7ez7uwIsU+y36pSHWuZiVVKVtAZOGxlrBDypNK6wbyOMeMN7J8LElasKVhMoGs3mhaT5XXV052XAVkRLRRuyxH1QFN99\\\\\\\/z+4pBVhuSOV1TdVnWRkwMrrso5UbxI3kAFPtEcn5eqrEGPYlG+\\\\\\\/9RJN3Un\\\\\\\/j4qfaaRhNDMtFu8Wr5LttpFv3ua1+NiMUhbyCy9BYxJksJynBCaQy3v4CFWZBXsSQNs\\\\\\\/5lzODABrqxnxN53JVOkKak1Ke2k\\\\\\\/esJ5nWnchP9xYUUGH6AmmCKD\\\\\\\/r8t2811\\\\\\\/k0IReVCU8bgzNp\\\\\\\/giTvmyoqgPsvOclcnoC8gfTl55Oa54XmQS6FJaCA\\\\\\\/JG3EM+4JiX2E7mvx5AhaetWATK84PGakefotZZ\\\\\\\/1dIk85QmkElt1PqMMnIfe\\\\\\\/mIBV1\\\\\\\/jkIWd3dOhEhTehCogRz7JQ+zDo08\\\\\\\/4HP3OQpFA1bijNgko1ZSlrVOZx7dNUiIaVs\\\\\\\/Dj9bBTFF+wdy5xUUhnqvG5sZ3exi8HqdgmRJe8zNa7gcZan7Tihiwf+j4tQ4E93atRKHarzMvHITkz+jf8AO+Pgj9nLG95bOlXI2f6CxXZuW50qhTjf6MXHXGp2aB3ULxeajTeXNPvqwh8IZaFGCZUKGbx8MFtlI68xZY70ubwWbC7jkfZ93lM1XReoXs1zsHSuz7Up2UPlQ0n+SzEi\\\\\\\/QPv0Cl0PgEYoLD4gqceLrmsgBEY9wbaqQYbN0L7MPwqgHFx4mzEZp+WxqDyjc0x2N1UF2sbivzxWoQ97PgF6YVEn0FjD+TtuxUkb1grr6LbrFT9\\\\\\\/p7\\\\\\\/I\\\\\\\/XWXEkF68ID2IXKvQn7PIwYXYGM+Z2iha898jMDXShx\\\\\\\/yR1M+66rlt3UEpexahNxh3S87tfjm01c4exC3SMV\\\\\\\/SYGp38YIHsW6XTP+mTEfm3\\\\\\\/BivIxSiWOum1NFj6mWR0Pg1e9Tv+mMwUOm77CwAcTgZ3dCXY+nVYhy51\\\\\\\/KEfyDz0BbDIOT9nwSARNKgQlNxPguzcq5hncnDodY\\\\\\\/JAcfm8ZJNdI9CPkdN6p4vZbtQijhATlZQGcv\\\\\\\/u9KZCHOLZwG4N7GxhWXzrxridomDYfitIKybdgMrRV9COaGt6eaJA8GKcxQQNHQs2HOqdb7belkXHlQOD7S8w24Vuo3PJuCkXTKhqmryJomhYZO38LvMGdL68CBNCgBWplTJ2W9hzV+7veh6YK7z1xVui2VwX8Ln2tYHVqVZ1NLaE21p084xxSPtS6kGfK2\\\\\\\/wgXuGLk+G3qKARH9FR1r62hFAfFCdFbhTNJjQDLRZUH616NMxs9I50iiOA7ZlsBIlYFhV1qn9AQzPOdiqn36vN1+wfLG2uho5TckF8XfiCByB7FpEBbukyK58i3vz2TkBAmFEZEnDX+4opO2uKnwVcwTO\\\\\\\/E0+TMUatYXvkGJGvInlkKXmkxIxf3sEhPizE0rUJFuh7rIPOgmQzvD1vt2BIicHwWUYSiMltWCLfPgVXdFJzcvyIQz6lXVeHOs8JfUVOo5ltRjkaWYha9eqvnEQW3SeEW4M+An57HxxdGF6ulH3IdxRxKtBheNl\\\\\\\/jMfChMMuVJa0pE3jiss6eMOX3OBGjT53SsQXBAPgR0GbNxuMlCGhq0xE8ShcERghwjw43bnlTz3\\\\\\\/Kl0jE3PBaSOJ\\\\\\\/5pxpjK9qdaqpHTqsDa7lgmAILGYQAvU5CpnEms+cH2H3FtI5s3U5UEpiD4j8sQqmo8nfq\\\\\\\/EJpsqONH6PsQtLfNHZx7BV9u\\\\\\\/nI31qulnHMSO\\\\\\\/tOBwJt798IkRoq6QsR3bNSqEIgRDBTM9ChLo29zaT2CkP82Sfn+SLRL3SBWvsKikFHRodv0PTY+KpR6tpiodolKwdv\\\\\\\/+6rTGNQBCMxlb4iUDNIIuXtVHHmQzqLVU5MrKm3\\\\\\\/vViPxHGmD9z9FtffcuKLjATC3cKAhqRUkX34Xcq98n7G\\\\\\\/EaUgKsOuLQUFu6KxqKh\\\\\\\/3k2xvfQPQGt07Kadg\\\\\\\/RCOaO55wbuf07DDNGlKl9CR92ljNdJt3LvbpF0D1QE5rm3W4yZhgcTK3WHedtUwns23OkwrKEYM0NAjJeYXdZ3eOjviMbAz6VyUpE5uy4dUgbAaQK\\\\\\\/mqBsY\\\\\\\/RdA+4UwAkPqOuGVQ+YcwLyvcqyce5e\\\\\\\/p+umin+eOpJ1rTS5h2YPDrFfbMFPf6RfWlEwzYh9WVp9CaSKXzyJm977j\\\\\\\/gwjBdG11viWdxDwHTCQ0bbx98SUyrm2i+oQrSGkGGledZwqqvLmrnBG1Nc3jSDOqskkNxt8mDMRI8poySO7iuZtKSceDY4UA5YlaIFUCC8vK0tfeNsjolJO71aMEcdBUdap9tqZBydxU0SQBSLtjpq\\\\\\\/c9kyy7Zo+4BKsP+9gYcFiLG3xC6\\\\\\\/LDaYhnpHWgV5JnM3Q3cRtge0SqBeSxqPpvwCsqfeAPjr0ljvMPmipTSSXr5urxf2S072qyL2gfrWLK90skp17Ct9xbzBsRpO\\\\\\\/QMm\\\\\\\/\\\\\\\/ScF1jOcKyvdk+raqt4qTAVMKCViMIFEk9uCPrZhwRDwnOwSAJVGK3A5KJG1oEXSLV1zGLpPrAnSYn5aP8947eInYQ4kkkoCHwg0kq31Q6e9DxgiK+eVFcrBf3y\\\\\\\/LsDBWG2p5WmJVoZ\\\\\\\/GeT6xkLfICjl3DmpkFa09meKlZe\\\\\\\/sCPz76t6UMcq+t1U+bckPS+pPazJBIUW\\\\\\\/Ah8QCJ425wZ7HPfDs8vkftJkp96guS+vcWhLHXoa6FiCpIfXfUPx4dnVntJJxSR3USAveaSpQUI6uQqipljh9OUtsJmpVJ2jHWuTLpZYQRSg888YPrf3+ZR1ATzlkaeb0YkBtg5UnIiuhbMjYdqZxa6XfFhBjP5WfJQzB2snS2QWrqJU7YNwUf0L8jvM7T2++rcvA8Ye+T4WCELg7XwVSApcvSgqh41sMS4Gpwjg6Iszrx7zbDK3GqVgEuQ9lTjElbv1Lnw7b8j02+lrSA\\\\\\\/Y4RRii8D4+2KvNeqqQSOB0pXxFTi08zPCXgeLb24LpDOgJXhhMkkReGbdU1KlDV1ssLhsODhHjcA2dhPiMj0UlFWHMFezIc\\\\\\\/TImgAPzWtrBwUfu5DwezcYpfseqfo+hmOanZThE7glF73mLcPKNP1E+Dv1GFqOt0bVdjCV5qadNRAXYhbi5Jqh3NMTGgbeGrZPTemE6aqrNXnNV2+5VduKof2i7R+0aQe+k47YS66YIXAl7\\\\\\\/gexcRKmyYkpzyotjaK7iYdo+B+UDZVXA+UR\\\\\\\/60cnu7sjnP8huqccK0GPgRv\\\\\\\/mkbBGAf6vpYMAG4k2XohDQE6LVlKjQl3dWcsgy6YaO7x0J2wYFFL7eUPrIRmYGyfUBN90nSG\\\\\\\/WKlcPv7OGecaRxyVQpPXGGTeAhpIrQbfoaw8r5o2RhRVg9QmQ7y7a76KkhjdAf\\\\\\\/Ob\\\\\\\/ZdyMgdif+dnGNOtJZW81Q7\\\\\\\/jwtby9fZ1dNSthajkwT23ybOk0r+GzvBNtWuE\\\\\\\/AoaLp8x6y3oePvE4P1b+qW4y4qpCFojEGNn+m\\\\\\\/pNs8hZvxSNnbwaksikhpCLx4uc1gTl+TD6uCK8n8P5i8BDkuDFcHY1RDl+216CBRILS460ToWcNvq1RLBRA3fYljGpIDiXNot8VurhH30zuIFhUZLUkPqkIT2xI\\\\\\\/Rl4iNLSKbp40hz0da6vSuDngxbPnjOyopJ\\\\\\\/V205fty8W0XoNVob7tdPkdjLIam+3s\\\\\\\/Z6My+MKWgK5lgnDHMoSRBoB5TI6yOBtXxf2HOUJ6xY+Ytk3\\\\\\\/VcP3jF0xd+d1kki\\\\\\\/\\\\\\\/mE8LR4hS55Mj8yfRV2eHrQ2rziY1uIc\\\\\\\/vD08VJ7OoKebBQb2p+Fe5rsRRE7XJ0YCHKElyPIx9R81ON4i6aapEvkgBX8uT2oPJSxkZcIicXFeQ24z5jt6Sn9CRnhr04HUpX\\\\\\\/BlUJkrEvrfHo5JXhwsn29r8qVeNzpsCBT3dzb06bnlDCeVfniVyL0DCVN8zzUi7fvVWFsTWhoLvQvCKAvOm4JKPBVEUhcfobOpuU7km2JVIVmRlPisGqEA5gSvs5\\\\\\\/j3neGH9qYy+PkJcKGjR9\\\\\\\/uhFLH+Nz+CPBdYCNEry+JPRVumNHBOAM1QAQbcDQrTVHZgdYcnWYJfxJOGh4WuMOml15eXAYCbQ5rOmw58kc0b4h5j\\\\\\\/qT6L4AUfydnDffXHxjuKdJLBNQcGpSnfgSIfosANWDjZ++goowHNGagbYghkrHufC92ebQGWJ05vkG3zCrwNl8ioGUF+EbQ8Xr2La1rKbEdLik8IfyHJnZuGpOuX1wlDnDQDvRBZ28sBLFB7mZvW6YvfhbCSaKfLZMyTCin\\\\\\\/M3tE\\\\\\\/60dhbxYNMxctftOE90YdmRaRs8G\\\\\\\/6Nf7SUd43UAEtO4vbHG\\\\\\\/IFM6i+o9u9N0qX3dhH1m9YqdR63PMj0m29MxhSjcLPuRIWclHu5qgOIPYFiZpFOcNScjC+2nDNC3NPfu02Y3rpLm7\\\\\\\/4dCkMcFEntYa5Ioom602UD39uUiARwH28BRcbilhUoUe+QeRszKQOkMjSIWcPUIMmFMPGnFduTYHGZfAaEF\\\\\\\/c+WoZ0edx8rKqwrgedfPFTA7K\\\\\\\/S+DKt3Qm16P54B9U\\\\\\\/zY6003eX1dcjrizw0jv4pshLoUVXXdmrJWIVWcZj+bba3B8\\\\\\\/LPsldThteeGLsRG90kgrNd4tVnG7z6g5XqfFeWpKZC1jqQz+F5kK0HEhuSthONPHb84RCujVwPg\\\\\\\/1sW\\\\\\\/y2VoLKoo5JSTZm6PBoMWCJyBIkWQyA9jENYlEMrW8dz+yoDQ549DHv7PFdIWPmkNaw542zEFXwj4XPKBsYwLMPIdHdNoo0H2fuILiZ6jyakDwRO1mp0BhBTeo1MAYzh3518BKHrHAX2+jTuoZHMLsNv2UFU6\\\\\\\/JeZs9kzLTedCj83yK+VVPjV7BCtLaqHEjRXeUHz6th4NYoE1\\\\\\\/2vfO+\\\\\\\/xGMz7LTRIX\\\\\\\/bhO\\\\\\\/Iv+CxVi7TESgoj9JZuwHGykVfak+9RIAx2tgerNBXQVGSfAh5mhYobRbmiKjhHDnn8h9QYc5TtNxe9eW\\\\\\\/nyp+QV6bwrzGXKSBGSK7DE3YSV\\\\\\\/vUKdIUaFA8rSYGY46XbnSAmGgTbUK68y+q87gOTxwXR5pHfWIXHWOX1OBqYoLo0xmw4zaDxpj9nWHMv15D8SWo5JwrF9qv1fLdVHxORyJXkszfVg1+4akaw34c5MsZzgfysp7jSm+8tNj8dpSnRvbOrfLb6cn\\\\\\\/dDmp3bIpUQXZuzlTVdWA2CoR3eEcLBh0ihycnBA8YqWYLoAVu498aicxITiI\\\\\\\/hXe2oZjp+S7B19XAXQxk+NG1FZuw2EqAfVr1Moas95cmT\\\\\\\/DHedJzjiQaMK3lBX+gPnYn1sXcO59Pjx7duZXuAvB01IllWuDhxwlpUW6golbeHPApHfR7TULh876lfydBkRPBE7JLaMY8KPd2wAfei96EkVWNRR86jFCFYzE0z\\\\\\\/klF0QCWnliSqg76JeP+3DzjnnNGeGSAJIKTrKLF6dWdOfmds9mKdfXp6lOOuwwwErt+w97n1VR\\\\\\\/KyOo\\\\\\\/8enKhRRZQqtcOXHtW3ZYyXIcY3K9ff\\\\\\\/OxlrA1dStlyjU+JxQyqMZ\\\\\\\/V4IbxI4EB\\\\\\\/vCrQw+cOBR\\\\\\\/0FWfBGGvbX\\\\\\\/rj0MZFqxfQgK+\\\\\\\/cLwGnzJSOHIF5IoXqQ\\\\\\\/1pDzY7Cqj+FNxIT0poOzit7G4AL8WUf1aYaf+9m8vLb9OASpdhDkxrjgF3S9RH7juTwpsCdckDlueLt3d6ltz3IN\\\\\\\/LUWYhvH0jdTDbXO8F+gbrXR+cHVGGObSLrZ5FTWClfwe2pWotiSorqJnGB9N7vtcbJ7kuOxVPFBi3qEaaQ6SKAp1mh0dqKZRNrACcCK9PJh28kl3L4Ue\\\\\\\/IhY+PxTTormS4\\\\\\\/Ou5LymYj2NdjdLo7Atu1eA6mGHj+TA005MfAQZRWy7Xd72yLzYRhA+M\\\\\\\/+auCgePAjrkeLJIGVDj7Q2xdgqvVr8Tc2II\\\\\\\/6e71OoFSSFpx2zLZHgomOjtAuRl7dMiR29vy8Sq15RPjhxq2OuplgO8iB47Rgx63b5GQ6NHhX\\\\\\\/n76zStbGAvWp5DocmEiISpiWPLGxspE3gowDphI\\\\\\\/QHvFK3h6uiaPxgyoijwk7fF0GdJxsWlX6DMnsxDZNOKkcQTvayV1QF3czI9o4tQLdOz7Z9XugH2eqV3JO8fKsTPo871he8WIckAfCH3IyOEmEFvy8QH8rcdX4jHux4HGiCuUcz\\\\\\\/\\\\\\\/9ZtvAx5jgaZ6RDiavjfWwtqLhqs38eYwILqpyE04pa64Mt9EBNPIEKUonB5bzPlSFFHO2lDVonOs8xs3mGAsvBhiSV8PNFGxw93bFR6ml+5cFs3xGCDy13aG2wqPf5xUtssZoP9iCR5sdld6l50oBGOMWr5oMZkFVeasxx3cvFl5R2yhjZ25Nrw4vjWS3ORnYCprd3MWGQbz\\\\\\\/TdPSrdaFDbIFxGzUx18aqzPhvzMQMf8H1Pw5Z6UENB+k8PKA5oEX1tzJXYwlyq9UKvA6Dwc1dVWtXu8x0lP3uAzor+bH9nW0Ykh0dfWudXrjtsNXpoTWlBDFLK4tEwuJH4a1F6sx7JoUuQAwZpLU4sxxYNQHwVt38z1s8JyasgRMKfrCWw2hVujyfhDZtmNXMc9IUO\\\\\\\/gSHoV3cDshScW0HA7beEH7nJyH8Z5u1DDBVt4GOjP6euGkz3QDjLeG9YrDNxJjSvlgM2y+o1u4PBRuzmPbqzRkUH74oC2YaUH86mv363MvdmG1SVvhyaQLGkrfpSDrlPPQAlbZkkuIpp6TKxa++LVZc26Pgg3\\\\\\\/mMg72lZues4e3QeMWYQzM8N541Ln7xFqn\\\\\\\/gHobnwU952oBKQ257J1BwESLV5SbPIhnBSU1w89WxZYE\\\\\\\/RDiO+NRpoddB+tIaZRTsS32LqZy8+TSNUlVI\\\\\\\/Z0WR6fiirS0F5iELI9c03wlXtryoNpXZf4vcOsK94GXyrh5Jmy9h\\\\\\\/b6+dHMuxrvVyqY+q4ek3z+RJdtmGOxMgNaShEz6IyY33FY5cpXKaJYnPOb0FUBg56PSrGiMzjE5jBEQpThiQZXK3xAu7QdM0Wn4MRhNN1aqNOHXUm5b4YDji7JkWIQcGgSrhivPksosm0z8ji399D8VJDL5AM8\\\\\\\/YQRRmoPgXses7gTp9yPUJnWCHlqJM\\\\\\\/FoimaogzxkndaB93ZKkRKy3sLMb2hlLv\\\\\\\/z8+NmkTUMlfwsbqdByWKdRw3Z0TJkq6Ym2Mlh7meFWUUbpgVvjVYWyjyK8ny0mqRSQJ23pgVvy5w+hdci81GBE\\\\\\\/Zo\\\\\\\/3Skcgqo1UUrt7603JNlW7m10L\\\\\\\/1qTD0Kxx+3u7bSotFprehJbGH\\\\\\\/HsZXGQNlm882B6miIWp0GJ3vtQ9OZyydGptN\\\\\\\/6OtUeJXGKY+2+clYk+ZDXDiApGDMPZPgCSKU7oBgUWDT31LKNuX4BKaK3Pcd4Yh2yGSwiCxrZzEl0NkdIOJSM\\\\\\\/Z9LaNMbjbC0xSk4Zp+hb9Msu3lzYCr+8n2ZdYvYMFPGNvCHm\\\\\\\/m3UYb4jBCN470wesFBOSZzPAzV+gM6AAjDRjqRw8hh6zWdnGlTBkZyUqL07ecBFJyvME1QnzI8dCl6FwXQvcLvI98fXDk78ki61vC+w1oXVFsDIZl\\\\\\\/\\\\\\\/ok8uiPJZvmwky5UVzY8R\\\\\\\/yVIZUN8C9C4qW3z3mrdvY7Tk\\\\\\\/lIAp6obmwsBDqsNvRnfCDWWMljEukueAw7m28gcGlEBtM45Hxv\\\\\\\/n+oEq6zp\\\\\\\/FORwEw4INYPep0SOIlcd\\\\\\\/B5\\\\\\\/hWwvAHp4FwtsLKFeE4dIwMnIHLCMTuVvfeZFDKqh8AH2yuG\\\\\\\/0J9pEvc3EV4olO3b4zh51sRXezXQfdjLquqGLNU8ijRSRF5Ty+KtkxTsuesOGWOrSEheOuLhvMRxxQ28Wn1OOFwnBkeRxJnUK7elU2NWhVV+vv+MoEZhmlO0pG2bPMqUJb7FWe7WgDvCm5W6lEb0PiGBTaOUD8DzH2E+GgZDTwOGcIiGl1xMmTHDIsj3PN4UM7JruNxtdzgql2An7LaV0Yor\\\\\\\/oJl6LyIV5iflY7QT8mh3rzA5rf3gwagObIXUPg9OUM5VAPG4D+pjiUDxAzaHtho4N8FvGULRn3xCnJPvcoB9Lyj4astK+4C7ssMq2iOox0MJrLVoH2ozIpNZGvVnqGzijOOM6lFCHA1uYSoSBq3yyAhy4CNEa7sixaY9Pt\\\\\\\/Z6c2Ygf4svIgsBKLwv4Ktqr9dRtFB+2tr8KX9e2VMbilj6hZLWX+5gucguoztQl60EU\\\\\\\/SQ49tkRWDL7Ok6w7to8PLTBbAjJtaKbfOTkXa2P1BhagqYwDKd+iaU0DlDCWXbmbO+E4D1rXqRYCf37WlhcfI7hmNYbHqwthS828lo4+mhHYcho1jh4o4QNOOFoIAFg6RDLLyfasJ7zsjK1kgzFABXI6D6fJsxDLIUuqiCidbGpfBt0J3TuR9BQEQPmbcTA9WW1s3X+yw71ojKjGr76+ODlPy1F93IKo5utpt0TrNZhYD2Ko7Z9U6NlcFf\\\\\\\/05iw72FMkH6oaTVck0NesPjW\\\\\\\/qbjCV\\\\\\\/5Bn08b9erKiBZM1e6ZZGivAkPmM1Xcf95EzGd7I842cKXBBDNOnBVEj1FM52OxkQyERDxSRh4bA8E9XKLs1B4VKXUUIboqhgMhIkXU++x4184d3m\\\\\\\/RlMIOl5K6uNkfiNyqoyotmNybKsGQX\\\\\\\/79CyvC1hRFrd3EOwXSuaqaABN7HjIA9QvE5+TFY00WR681iL7bGQ6PW5lZYUnpg5zaxQFr9YOvYS9tr8+Vb3dDxggZwDd26ArboxDxb6fLm5KxrZaGXhWdENNJwfH53ye0XLm0oGrYZr3MQzPo3zR+sfxeTVCHUR0Bwc2yeC22bwNuZB0ZTDAxOZgpb9Vw3VM164+XL4P0KPPbUbBDs\\\\\\\/yApALnlWz61JD+sxxTqb0lWibsBRs0\\\\\\\/Iv6mSEq+tgoftvnW\\\\\\\/SrNC6hulrX\\\\\\\/Dm7WzOdGsbBFdKrQLtzjVWkixTAqwuw7nP6drJbDy6KYZZp1kKFr3QjPQ\\\\\\\/88QXduLKBE7+uGTi6uLqEO6NCwtqdXGdp1vhEmqKGIuPjc5sj2l3M5M8F63m7UmNKIK9WHFcXsQglroIFVE8QBAV1kh4nHxgPafbujwh2mKlA1KVEeyx4qLa1BjegWV\\\\\\\/q0p5qmLWJDjg4ZqoYdTPKeq+k3mrhwUHjrXmAgQ\\\\\\\/iXnkLcL3uQa9OR63rSDZoXlgg5pWxTqlepB3fyuUOaYO\\\\\\\/KQy7DDYbxw3RJK4z13hpgpmqdqCpKyzYACqoufUzJCest4WC4toIgUuzt0so7qIRDx1D3qT3W7xL8mgdnykKxKc5ruJQLZMOx7qwvfltHprllSo7cY47fSZtcKD+RORt0WwtEtrACjZAu7z6UFDumPu4gHiwqomVkSCn9XmHIfyijpJqqyhR8ttIYezAQSGJOVAUOlwmQv9tgb\\\\\\\/I2JS5X5ps0US8y1277jjWbA6Nrl\\\\\\\/yehAjRqTFvozFiYnOCmxFqEJYIKK0evvOKrqKwzY6idY+2MH0zbnHVj3rMOHJi8o0OEXpXzxUjolNw2\\\\\\\/w0SnQY7fe9BpQDMDgLU6k8S30icQcAAH4u8MTljN3QqDTku+eLGts4Isocy3IuNhsfkGRiQa2x306M8FJO6rKDALZQIXIhrHoh4MM3cSlOEJDdk4nXIkHVBWl9ZKEBuIFuiCJGZK2gjMK6CcrJnnbACmqp4BiOieXphXCOai+bS\\\\\\\/dU5q56pI1JS3dd6nNlGtayzv6wREFbq6EMrJTJqgI+cV1Jg5MVtbmml7lCWJJc1A0KS6z8zSjs4\\\\\\\/c\\\\\\\/h223H2gi8EULWMS3w+qp3Aibx\\\\\\\/i1kMmZFkdAGO5gRrvk9rb+KrPSoHVOh5LYTrGTXXdw7NwSg1NXN6ZXvwmN3OnYxyMcTO+Mq+913eSau4Y2vnjRen02f5IU4mpCW\\\\\\\/8dDuSFjJL9rmjaRvGw5QQaZ2czeUlCpm2O3ZrcV9NFOMOOqXJBD7l91FJ8lrvTqgpCaq3EGBjIWHD6lC6L7Y1LxKjLyTvfdMacEh2vgy7pyjH+rMh7CSoUOG+tLiFsylUMKaUJ+zW8nU+GcRCSNtddmsHTlSHQRY9QXA6Nm8Hp4u2wuwznxBQHZkecmhmjbhPwCncdG0FALe2DZJ1tjaosFWAmnuDXUUunpm5ldlYwycx18csDuIQt+knJb4uWx4cpuDy8xmOkvkL7ZrSjLYl0HWGVZjoNXatK1eL1XhTqzUEf4PP2OTCjjkvM604ahMaiM5VR5RGesFGNxjIG3dNRJKYTj8+8jhsfwQIfxpco3mtITsNInrD8+l3LsNoIlRX8xzCjfIuP34lgCTTtrqfIGPJfiXPkKMQ\\\\\\\/pW5eHykkfQiW+mS91BieMpeY6wv1N12m24wIa0WNC9T3EcIC2a3yw4Vjm3lfCwgunNY+vcxhFuG2QQqtN51SzrfSAsJ86XMGo8Vi6ViByUlpxpS0IeioNIPIkfYep8WkR+PqsMz2pmBycelY41KQqZf6QEXNIiEZC2EX2nzCnp+FLnpg3ybGj6Q73EvT+mUuy0g5x6IFyCJknpIcoGyTwSWP2MXKyCFucxQtBh6\\\\\\\/M1IxEFCK7yTLu0iRMY4nRVDkC8sIqP2JzQjpeDRtBCBhFysjMeXdU3wImhBTnsUWJ6GdVBC98uIjDeideAdq3+YswYvi20J9\\\\\\\/2hrsZkTNQGhlVV0t3rGYxKoahaz1kmVQuJih2HA26D9F49SPW2ZbXk5qW7HxG5IRgv\\\\\\\/qn6xwwNL6ZdRpMEbxMwss3Qj0\\\\\\\/HRiNS+4Mw214oVMGguE4Edzu9tecai+3pGrOZeGoyQ+iTou3BVzI9rcKrgY6oQpCFUa0qaEspcJimRwD6g1UT7+hkOBq2VqjxU4dPhpdYzZgkwFO8JqrzcYpiaVvgPDtNbYFNEdn1YNmxg\\\\\\\/5mtrdEpBRmSITG9h++XsS7S\\\\\\\/6A7EA\\\\\\\/ydBwtYP748QagambQ9naXFdwppxVANtvXAohpd1U7puEyB1W4WtgCDI+G+xYH6cEdPd696W3oJATIUhIOFaxxZ5F8EnwOp21N+8AyzeQrlEjw7jMEyjJLQwqKfuAMIIYla9nJjtOlAWf+wJEdCiTUeX3uglriibmlQWYewFKSCi6Wx7vCiA2gaw\\\\\\\/c5gbSkdHy19VWI0OJ3ZomT19d2A42XQI\\\\\\\/qgRqJ0DvUu2\\\\\\\/RFaGkOOJddEE5pos8RD6orvoZOFsI1z2nGREetZe4wFECUPN+oc3kiJsZUflyUVxWNVZ4sDewaBsiEcK1jXqw\\\\\\\/GByxPOecW9F\\\\\\\/h5SExDo42riZcVIryoOp9Xu9cVexUpi2yWs1CRiWMnJHEWqzsYTNtB3gZxfM8rwQ4bERLzUo6DRGEpoi9ncj9VxCm6XdPGh1vh2sWu8PK6FFKfUZ13PRs6G9AXGhSu33SCG+6t6tOBHPJqpAEjIsCiNCo4ik0GUDoSM+yl7Hnx3MyjJJ0yH2J4WpIJ5+0c9slv9gFVXeQg3UgCRxN6gKvSSfHEtPcXH9PE9kcIm\\\\\\\/\\\\\\\/8eacfftVSNs90L\\\\\\\/duOiU4+AnO\\\\\\\/qJsUFVOyfbDdZdeQgdMwypZHN2Sv\\\\\\\/SFdeLC16hVylBaZeSuckGT2\\\\\\\/6\\\\\\\/+BVAIR0IeydPGLN14uWc7xPno6lB34CCHsSOfI9fqVxHPb6Cfd7enBd6fM0TR39tD6NlFSyrpihq5pczk\\\\\\\/BHr42\\\\\\\/wiVe5NOiIc4iOJdyU9KY4tu42oE6nHrUnHqbVwScE7OjxAO2r+58BZ0zeasRiHnsAHSBDz\\\\\\\/2IldLTSmLjP1OO28IQ75Azid480cHsEXftq8DsTPDmrq3ZkPdFEjP9frdxQQblKUMekn6L4Os47awb9it3xCIipKpJEMG2ByVH6mHuZEwi\\\\\\\/YG505CyeXRlntiyKp9Vl2vRB9v0B6D8lJ2IljsUyQVd\\\\\\\/h8tPCk47XfEpAmJGOZrcGGBV46CxocJhTMv+2x3a71W\\\\\\\/FAhxIYgCm\\\\\\\/oR7R+bqPqLB3yx9mOz+5yTJKpeM9WeXVKo\\\\\\\/Paa7\\\\\\\/ltiLaYreNdXnlfjuaNZyTca7G1jOxhNPhRfL7MAyamOpt41z4drbMf7jJJ0EN1RYXbrPBzv48Qkm66T9UQUXfoumG7aXradU0yfBvpHMw6po3YjwTOQoRRA1bo6rp2SEHNcA5qJG8X6hpRcVxdMK2QevUhGsLe7KTD7HIvgLeCdJM8z1hfTlqWtNmInqgONfGWE3Ije5wIkTIuMrDqxI1PkB8grCTNl9RKvovgjOK7EzSC38CDi4U3KipnFMZEWvLTmfkmNCigTsnQiz4K6t5OGOmfRFaHhm4Ukxv9oQ3AGBnYV6kXDszO4Hi5tCJyDDsSX40JFnUjR38hTmAh87DXNUPhSf1Oqz0MPpTXOrJa2NmFC4SJsmig7dwkxAe5jC47RYPfz7kq3Tw\\\\\\\/7prgi17B2r5UmNp+9CHMoQnk4ajwXJVszYO+zrMi78BZtGyIHnKdP8lLd+UzC7bPjNvqySE8BlpcFZj\\\\\\\/Fuvl7lxrA7Y7LUka3XejngviBqaBsrtIp+mBPG7x6Xbvd+He+CMZ9E3NwKw38H\\\\\\\/bz3I98vxANtxMxrXkZl5izbLMZaLOWlp9xBngxlFsMch5\\\\\\\/2jJuzSrAeVEWJTv9VXkUixWtOs+8fT+Ae2E6Bi7CWXbcgl1h7mC3mxTNrS2N4QVN3zA7kf4DIkdj+JGbc8aCBE9kgVA8IYCbvbBHApuTRI8KDp5e4NNelGNDJXd\\\\\\\/VEeRNSr178v32LMgpdb+tPd2br1yJjUWhXBq1BZkcbu5L\\\\\\\/Wr3u3OUSqfSrqtFKQte5f7kiwbZxvBZQcm28rVPqkaZKBSxRbYOhJ7sriLAwapsIzH6u5W\\\\\\\/wqYq+4pbCVTrH3KSnrMKt2Ku1BKViTspviadsTRC2rvrx5sXL\\\\\\\/Ny+mySTAQWI\\\\\\\/FeUt2daPPaxzjYi+ZxQ+ta+XirdohFvQdCTEZbsxm9yLOjb2JdqfEYTTlgxWc7j1Wn5TCUTosmGSS4CLe+ClkymXhsJ97ErtINvPRMgZ4zOj0of8Kx7r27MiWvVKVGjwmUpFpny+7HjtA8nfQ3uuzyhoXfXhbOe36w372ZKIxwTviC2pOYP5Zx0jtx\\\\\\\/XVfIP8Qv+6I+fTLIe2xn0lnijw3qHTBkBZucvO+AUEhslUmv+hiXl0eo1h4gvsvWbeS26+aI+W8Bg1IZgU9d9GjDq6bBnUCXEg0deUmgwTAnZEJHkFk5YZSB3zeb6Qbr3hWGE3K\\\\\\\/XtPzmI4aQSzQZpWELjr\\\\\\\/qUMdCCXLg6vRBYiJryzKd\\\\\\\/DnB2ANf7blkxOWbgj1v+Ntv6epNFQZKwIV08TyFdki+WvPJBh402mu5OusClL52YLxhlLY3UHt2KJ4nJIJSuxPxk+q73AGzy\\\\\\\/saXPeXR\\\\\\\/cJOZM5zh6wvY08Wnm4o3fm8Z91or8rBIS3SSPs93ef+5eWdDajnBcfQzVyweIzozB7KXWRidRWt7LjcpU0lLKV6IOEzXv9ipp+JE\\\\\\\/QiE+BVxEuxGKcrHrx7IoVRPKdtNePtNKFSZSX1xtKl6Sc4eGAJfxmmLn0bzjVISkDoQBm3uFBieiGCrcZbyRlS39He\\\\\\\/S4UekbAzQ+\\\\\\\/26qHU2OvtyRXdd7XdXRXSNxZ\\\\\\\/8em2IRSQ391rFLObifYZ0XE85jWlzB\\\\\\\/C3q\\\\\\\/W9vPGKH1ipPqyIud12FqEgfEg8JFS8Dk\\\\\\\/NroGzEaxbPF1d3BX1JL6daBOpiAsROUWVQ8rFeb\\\\\\\/HWk5vOUrLx+u1sfwybjR67zlBK9mFfSuIU0\\\\\\\/1PGQKO+G8idfj1CVoisGAXa+qOjL8SHDGxLf67Gz5MFX7KqzLI6tS2nYRdpYu2RTuC+j5kM9d1ChjARhsDx2omQtw6pYxNIq7kvBi1uFvWsqcUmuchQcvwFZynisuvcCFnd9WIald6BpgR+x2dH+QgP79g0bUQln4f5ZCnkaAABua8Mm2t4ilngsCeTi\\\\\\\/AGpzgYGT4PDu4o+3JMgsgbdz\\\\\\\/eeS\\\\\\\/q3VNJwlLt8VUGgwUitRtChRfJnLI8yy2k4bE9quxR8QThxhy32QxXsuiERCqxEam3KxZ5zi+5SFKci38+6erxWm2gKvhKYJDqYCPr5tkeS2Vt8l+Zgf7Uy+2473F2cqKM8sz5XE7ukW6Djv0h9AJ9rmgFAS\\\\\\\/eedhq8\\\\\\\/SnBSPIHhrhiEpTbPDOZobjXF7p\\\\\\\/debQlMDOSzebXlWX5yyViMGzClbb9VgsLb\\\\\\\/H4xbRHCvSbsOVomcyJQp6s1ezjb\\\\\\\/c6RAaOAftsj8QeKJDEiNYck91YZpI8XTLZIw24ARVzRNXIkr8OeqTvgq4Hkz00iwUUlqCoXKaceqummwxgMIJNrtgvOOUf0UfenZJVAd9qMtS6YmnA0tOjp9ECr8iSqDmpQq7iHzf2mfFd1gkq+le+\\\\\\\/x37kFrUy15bQasaW6OrrhCNHdvPdVd+pCylsExOVJvuO1NmFYKxLbB3DI565rxjCOrHGKhlQaPhjUaCt0BZmE\\\\\\\/yJpTujSM\\\\\\\/k9+kV9OoI44Ogn6nldqBQEoc1XD3+74tCHk7ypzdvFoUYSR3Igd4zG85\\\\\\\/JA3KQDFAN+OmeelQB0Apul5Y8O72kYQa1S0qy+p19hF4eKGswRMShJR7a1zcuixALoHI6m1ZtNLp+B65hgfs2PP4O7jI8C\\\\\\\/hzPY\\\\\\\/3aOQdhGr50JfJ1W82xEyHl4BG0sfD7+O+vOL9ktz1dnM2fh3OSVhZ+zZjVTpNoF8u8HtrqJcqm\\\\\\\/lvlnCgH8AWf3PehGz1MBC0eH60CSh6JdYqUH1zO7VqS1Cy08Hg8UFuntxLHP\\\\\\\/LYve2njHAP6thP72zUCdS70QfpT767ZEKT44FX3Kb2CeEEfPJS5srjoCbgtXKqrJoxY4JMSqUqCeCiB9sgT9haYY6EQcxPL2a4sqL\\\\\\\/Mo2KhmoLVF691L6ZjuW5LOppwQviaapSVDC\\\\\\\/N7ks+VmooWdPhov2vzWrUiMvPh\\\\\\\/iaEHI8zh\\\\\\\/iJbSpMDcBk0CIvxO1sir++pQvPvUlOFxfMcqWzuZPs7adsmAueh+NAdxATmNBuYgo+Yz6+oPvxK67HGOpwf++m6GZWCY6AkppbzJu2RmE6BXdUCYilzGQgBQoxaZa8TnDv8b\\\\\\\/xMjPGhsCMmU0o5ex1NoVeK2xjZS5vPr0H44AwEUy5UrnuzlL8SYHEok57CsT9JkCu8PzhjO8Gnufhy08hIYt5iYFDD8BjxgCCrQjvxJtSYg8J920VxT\\\\\\\/nxPUz2dPwt6sT2TtWT0bMcALfN4QMbw2B69cfwQDQ5RfaSvO5amPjM9N+ag4tbG4U4CoACY0DRBqFxd9p2kjdcSBizl4rk7vS+MSL0vDlmqLZ5T\\\\\\\/+E6V2py9uBiA4WL+\\\\\\\/6Y8SUMk2PaG6qQwa\\\\\\\/+8sOK8yvFRCpW1H38+u8HR+K9BCSlFuztEodfFCULQg62M68vCw+7Rqplnhq\\\\\\\/YDuCchBNNVCkKSkUFEzoPzdmBboPZih4pEzmnARUIb31ahjsTymNX9lLLSacniFmXaYkYFuhHLxAfFqlNV\\\\\\\/IXMgJA9zLmDNenwgGgbeOKsqMdk6Etghhc8tXuDvXVBphO7J8uM5N0Z9zKvV0m5RbHkGnFay8AGznROu1Rob7HPFpnQuyT2Et8IrMFR2oY55tMJzxNde2eNui2XIKjSnqmehsEgnLwDnQn1cR98blDwq7SCxYXdbtTCvLwZfiDPVMiw170THmHeOwWFRhaVhPikjKV5TMF4ZlV1plZqFEEnXcdAZoQeaPn65YB4\\\\\\\/zGf7F5WuVCK+RdBkuWJuWIeXLaxBiQ5ZzhBEts4x\\\\\\\/uZ7TVn36BVwIVV+ezK9JWsb2F\\\\\\\/1mDNQNoMGYGTBn1Ter20pno0PHmLvYqfskU22jusrcOeaU5ecHIAPctzrhLftbUFml2vsuMSNQMbk+K12fG7qnVGheFPD8p9PcFDItZgc2QUGQdyjAkVOSHoO9eCgtdiku6wNwkP8noTCdCJTbKPlpGj2fddie1tDW9UankOQ5pUPQIGbBHOyk9tMVHiMhFplFLslZLv13L\\\\\\\/22oXMiGvQ3WGmv6VS9A+WFpm53\\\\\\\/gCTvwI3JlHYomNI27ioUoeVqAID8krGJP6J2bVupLxoDOlACjhKa7i4uGxC08f5cxW1J9Lg5S19CSYva9\\\\\\\/ZdD4KKc4vu6nlQ3yIGy3rJjOBy2RbrGXvvzvomFnjuNz1b0LKwKz7sacJ89bYlReGgnIB99hjpkbQj0Ec3m9NtLsk4k\\\\\\\/+vEmh+K3SHvU+vX\\\\\\\/sp3TCMtFHL41TUTu3pCmaxBzV8oOEP7o\\\\\\\/ZM7p3fA5O7Xik1lJcqwhrMsjHdBCIFyGkwiWO5i6qBIGHTIJ+\\\\\\\/\\\\\\\/dSRjrVdVR\\\\\\\/PKPgk1qFng5oDHmVwfvZhR48Y\\\\\\\/6SshO3AKha7etbaTVKNEH+G9w6sx\\\\\\\/76hDJMPDGjvJqpwrOXNTjNnc0968I9tfvM13BqKV9H8Z5YwyJU9e2J2hAQ7Z0psoQI5pU0ym3uE\\\\\\\/5sIkrD\\\\\\\/\\\\\\\/M4ZEvXreQQoDTdp+6reJQAFV4QFwTCIQeNHF3B4fFuzhCnranBBR\\\\\\\/oYlz+WjMjGEcFgF4z0G3M\\\\\\\/G8nM4awJor3KfW44A11PCn9h0aj\\\\\\\/JcAmwH7HK7twzKQF5fQ2s15a0vvu4yHn6KmSUiYad1p\\\\\\\/eN3aOgkzffy4cdivEw8Yl0kD0WZyl4GpZrSYAwcRuujgFBIaPCfJ9kGdoDSZi2fLXI13NyLZLpoplrAoVywrVwCtMo4aBc17QU8jlQ8fXj7BI4VXHF4OLXGpNYlbLzSDKZ1zAXNj0MPbJWkaS81aj9+\\\\\\\/9GdJTwLu8UXc8BMrpIlMwVr9aj9zzrdCU7TbzVP3krpbi8v+\\\\\\\/qqXW59hue1iiG4DnAKzzpCFNpTxtVTgBuxMEsjmOUQXUpiFlg1\\\\\\\/fYFvjDyIdoMDyFy7rXmONN3F07Z34gwyMTSH+SPYShPeew9KGZVk90bNN+yQ1jCPB4LIjGC7Tm5RQer+tJ8vxVrraj343Ku3t98Scf\\\\\\\/38\\\\\\\/hEBMOwB621sTt+V3ud7990iZZ4PJbgmN3pW6HzR+tm3XaZ9ZvSLxgNBQ6bJQAJ\\\\\\\/361kxSRfNH0A1icw6d2EF2Y9aoeexiSQ3sgPeWV6HnU5mQvEdCUhq+u4SCev3Mzc1LekuMRgKCZEmpq3iqt5STBhbuceoLeQRiVbgOCiknm1Z1rQ3GGkTwcsd8O0gPFmKwkU8Ye2CtdLqYrwFf4auHqmal42xMZomVsuv0bR6PHttmI9bhQLM9zrlv533xqTBNMcp2zzowyvhg2dXsKRvyXsjwhXAkn4A6aZd4baq9Kd8RkM4gU9D5nxgfaudh1IOQADEYuGZp8pLsMzQXy2Dkdiv23Em2zObSuoi8A8lk4ze1A7mbqgHhpUtSZmAl5nhKJz8384hE4z2cF7pGnjGm1TWbtjJc7bFFdtp5zXnH8LySGMpE3FOuJ4hx2A9w\\\\\\\/kGk6y4rUdo8hCbA4ea1ZWLAmkTtFAzkDpvCSZeM96I0JFHsaPDKEmSZ5SjE0vJ0EWBbgkHEsiShLwssUHs+1WQvn9ClAPkMgwVZNIlNoKOudiEU+9dsJvDHg2jvlQV\\\\\\\/MzpqbJNIg2KsdwPRzEivcfjQ5WKbXUJx6jEItzx7TlO4Oo8G2uYbFchyGEwBouV9DTWbJsyw8adpEtlnNhCv7Uq+jdsAGHlZCKVLTIe2JBnoekkW25sWaKBj88sngDlMOD6jqO0uc2Mg4c13ITZ13piFiWovfaWC2PJrdRC+jb\\\\\\\/DP3RufROkuqv4kMdLbwL0kUIKE8ZnAQ6juASyPKek9+EJ9fi28R4Merlu6zbI5uI7KF3Jwrbn+RsYncJ0S4jfSmmMGIvnuMaJe21Du9Fci4MytFL6PV9PFoeoO9ENbnfyrRBFrZYwP3wVo0vEI\\\\\\\/O1Fsmrci63Apd3laO7wPhZn63taguSGEvV9d+G8qGjf88UFh+2l8WD7EzdjHHcMW6ZpvCw3rsMf+7GlwsOzlqxCmYwjyVlPY3FbK4EUp3KAPPm+tP+IHiqMuPlSYa5NS6uhpkV9NQFmj5JVsm341NxxBIao7tmR0yWPueoz8qqkd7G2FRznjTEQ3NqHNplJYAaEXoJy0rnlPKE\\\\\\\/lpABLikqFbMxm1Ioc1cagUEdhlHL7Bj+I82EZIEtujXlxhxfzuyc9Gq9VXAysRr+gMWO7wU2Iyk6MQqb2xWhhwmIuVYE7i3O\\\\\\\/gPVPwHcMLoGkK3khi3pSHFuWE\\\\\\\/XQQP96NwXnohTR4AY2usy55R8eTWy3iirzrX678k310l30qbKTztu57vb9kcY9yVkw5ELkOmbZdkacCnkChtiwY0Zdq5JhDhpOk4FvxyPI0kZiTg4zJA0Sm7q23GXvQIoXl62lL7ii41fSfD\\\\\\\/9NKY6ju1FYjL7gN31mHcMCU7\\\\\\\/WmdexxwqZoGbF4YM7Zw2h3+KqItjPoo3\\\\\\\/G7oDtkwqZfHXD1vcxDUIc+Zrcxs\\\\\\\/qeyUm6xMJbz710CnoFNQrne7gvtxxYu4F+Mc+4oM702Jef907EMmuxTZHAPpnNBc0MN+QX72Ym5yoRLjQ0a1qvC+KZAh+oO09XXrXSDDZ4wMrwkHcjKjq8fWiDpCk3HDkJ76rlE9wMVrLvnrQB0npw8cofvjiarcXvkQHwdZXP20IsxCfiTmN1zgrK39DbYJWiirXqPHuooDe9OtCpn1iS6HwUaRUk56roHNKCUbQfv7qg76TBgZab2IQ1\\\\\\\/G4Y3LElk6p0v4gIPV7ckYzVuZZUEHCIhoyDbdDZlJIEFOhXQ0QSqwfyH3K5jWXZ+6EvYvVxm4Nkoz\\\\\\\/kQfNSJlgN8Rbh4XEjw+xoF+f0hTg0h3Qp3FtKZ7hL0UDBvwiJUDf3ZeEFD78sj9TJjtcbOZyu9UDZi49SBvBZxRFX1wg5afHzRnk6O\\\\\\\/FsoP4L8ZK8FOsBA8Q2un2xMyJ1lQPRX4FFM2LO0HGeTv8htiNoFYEP+tmhtXxs9XP5oe4JEv5dUxGAJ\\\\\\\/KQCR0hUrv2PSO5YTYonQVt4GB\\\\\\\/UimNwrrOnH\\\\\\\/xV4l1QI8XzzO2ryXDaNApWE3OSzw0jjnsP+r0VuFesgRi\\\\\\\/jgxP2rk\\\\\\\/c9P70LNV1Er1kN1i4ylItzM7fHRGrW8t9FvPYGW+RjfV3LklXK500xylSytbr+BFJaxsALBONNDUHLKoNH4UgiR06oDmCQJx2Uq1d8iKMP\\\\\\\/9JR\\\\\\\/SfY+4NOkuV+fK3KsyvZND\\\\\\\/\\\\\\\/2nvpoMWCFju9DjLZafAqE6eNLppJ\\\\\\\/s5L\\\\\\\/zY+jLb+pOm\\\\\\\/G4FELzOjTeTAsjoiwikJFTRdbXPX0V40thHXoB2mcNmvh0B1jgeNkrWulbLuI4bxXzN3TO14VYSXKs2frcDqjvmFZKm8AYe0q8uwLLWxAnV6dhlO4CHw2ojVAIrZb9ODKqS3cjBCXnLvm4aHGQR94akIuBlAm4tFUk+NheINOzx+I\\\\\\\/7SerAX8VFxDMQhTO1Oy2vf\\\\\\\/9AVMV\\\\\\\/OW\\\\\\\/ZUXgZtJkboBYhdq8uC5vjZT5oO6XoQ0s29gHegn3FK\\\\\\\/ZGz+agTSHJari0xJRQtAEYtkqjZe1noEl7X4h5g\\\\\\\/\\\\\\\/8HleQdw2tQ71R2HF17EeCzmqycE9Gk5ZGhe5bd42pqs66FnnJlKlgmEnsCrmpCV4lAz4VFN4p7aEuSL8gWubQnPE8HBiaak84Vj9GWLURzD4szsk2XTJWYY0HZ\\\\\\\/ysXANNXkI1RWcn19YNdEUMUcaf3oS\\\\\\\/HQeVmkj3iE7nNuyelDTRLj4TnrEGTXo4xv9ZbI9ut1LZ7s14\\\\\\\/52ONUuXMUzaRrZCxuy71ccQokyJRkBb1SgOrRcZDHiyhooFftCbsdRpcfZfY2PTbHvf9+NS0JetEC8C6pZbt8NaEWZXts+YtM2y7moyP1jvBrb82lkOdPzDfPTHVe6swbe5k5rnI6RgazuWtcDcxcohPRzcDRLP8PZ\\\\\\\/Xe5C49QJqmtlwve04p9Mt1fJcqQ\\\\\\\/O7oaVSwq4C5Sqzdwfy0I2QDbo2\\\\\\\/NEGB3q80s1aWnDPUPq5a1sgDfLWfGyM9iS3TfeJQPilP4WFjhWkvU9DBX5PI+7i9j8Cser8fo6bpu7J1p6V0vZ+wZmLBC\\\\\\\/yyYh+ltgktl8Bl0scEg8dlgl90Aqj3jKBjadtbFL0Rt0d7aDhMDEzTfEO8N\\\\\\\/Huke+\\\\\\\/GUPYVhsg9HVJn4HvqtEV+e76864+NrKnJlx3t09sfqoyaFLvA6Zobe6JgB6gIxO0BEqqft3P2tG6nKmcjykefqnVb3YXwwJ4ejkdJVXHD+UZ44PqE8NOIacj+EzU0qHTbJpNK3xZoj0JgTUv4HieXB4xwTVijAVA6rZd8M+Gta+\\\\\\\/p0CkolZk0h2Cj53lJn+NnK+2r\\\\\\\/K9AJUDd7hoNYfKK3fkH+uOi18l+tVzcuy2yQfof33vioaHzjogCETNmWCJUM50XE1MO4iu8BAJ8JxvWR6Ihm5TMdzQrWS98KQkRglJslsPO4TGpxHDlx31O2j6povLIrRTk9aP3WmODPDVc8PMSN2kofMRIl\\\\\\\/IirqGVgQJix4AqtI1pCVU0SE63Pz6xZU8MGSTQh+H2AVBgp\\\\\\\/I8xUrzzyYdL+l8J9EuRpnUiJu\\\\\\\/JgkaH2J5ZglYIKLw3om5jjJnsZx4RGlulXp3PVt2DJKuuvQBfhPV7jlgx7frm7bKONyI2\\\\\\\/gHx1AVNUYNuP6SRElhs02XYRcifG9k0K96R5I1RrFXUcjbi0r0UJFnOthg++V3ESj0FlpNE4XsTbCcnOg32vLF9ZdZ5Q1vpuCBopkaTFcAoAh6nThH7YnY2ZqoJzFMNGCPC8pLZlKgv0SRMxvIdIEJQ1xHlKTU75MsQf6N7pdYs\\\\\\\/mukXtG7YzxONo5oEk0VjoSVHo8tg1y7BmywaUJuF1IPsLii11Up861mWGGWx24a+2\\\\\\\/5TfEQ2uIyF7Q3DOp06KTf2zXakGdQNVrr7as8NVLG58ne1azlpcFSjfv70uGztNFTbwhFzwQEl2+AhRCREeLoiRGEPsTZRuo42iahgAvmUqXsqJkcFl6pa32qI9W4y8USv\\\\\\\/mTM934viD5siMz4ofq6B09M92gbH4RoJHOeZ6wlM4puaUGaPy0iYLWorzOnHxOzphnp5ATzZLdX26Q+F6Ty+hVZ+nDzqE2RYaEXq82iWevmp2tP4rpY7slzQTgBzHPim+qeuLAwFzy2w4S9DwcvJUYe4OYj9jRnBCFZSfj5aphxhtYfnGZ3V6JlhIEmTdAfl4RXCxSQOsL4b9X3vfu1QZ1CuEu2AB3nuoH9SjIFk\\\\\\\/uZPx6554L\\\\\\\/AUESFt9YsOLARYQ1MV2yur7T0F6uK9V6orHtOrzC6ANLiqOS\\\\\\\/9PtLxxxUEeVpINPNNoB\\\\\\\/uVMK3bWN\\\\\\\/JuASoNRKqIGJBTSPYpIPVJcfbRqLilxYYaQ5MCZO1fAqAz2mTJGTwV0t6P3rRaO27Pujvr39TL7CzZnMaHlWanIROT7ZdkZR5wy7+a72Zt5y7\\\\\\\/1fPV+mlTKsHp8ATsrMj7H5UiCKZyxlHQCfTG7JW6JikaD9SLBhBuxF2J+GhcBRDZddyS95v+eLHYtJ7JqoKy9slQDTYsLrxyXjiFkYCWgulZXDPKV1k9v7w4CJsushk85bZMKgluSK5vr9xfRJMhDq1eEE+LBqvYUSI1FwcAhTCZRSLojc2yXspDu3zHG+u2q9U457QsNQfsuQpTRYeAxuxB6bqrgOufpF1ok+RibT5MJoI0aHDpMd2Xq+YttmvuRz2hyBvvJtuBaFL3784tn9\\\\\\\/EDgdCdP1lHRWmHtc6gOVZWLLNfiFJBdZR0AeCFb8HVO6MJXOEtNq7Z0QfH\\\u0022,\\\u0022iv\\\u0022:\\\u0022AvzIV7uBT7Q9ZRBmwjTiYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b13f4d2847e6d9eb8dd80fb5cc18c86620b08b6ae71692a5f6eb5f49af55af30\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OIsgMyJbOb6a8C47KrkTk6Iutq63viXKVcfwp2x7IH\\\\\\\/7DxZpLz\\\\\\\/ntJ40BzChq9IMpK55Yki52YVWg6N+lhMvRiCAUgWBo+OzC9HYBSB+f55011vrRcTNwnxDdqp7ULTz7vJQVpmkMikLhEQSD4pLOfD82YEtkiRlsixvBWvQHqKHRSEkGYLmeerpP5wO7yE7PlKrET5hjm66XRXlUih2DA82jVGDqsnBMca+LGyzDCCpR96BkJ6YwK0\\\\\\\/DRRAl88qZjbSmvEGK+iqVdvj2H8\\\\\\\/ik7dTx6es2zf+EiPb8M4cRZS\\\\\\\/0G6GVhiRSY5d2Tbupr1E7hk\\\\\\\/rxtb7huhUzk9rW9qS6ZcGlZXeR2iFDtvY5B+ZXACkg\\\\\\\/quEx8DHVQGKEBQZE7NMawFVVRGhuBGybQdbV9iV43d8KIpYxmussj02wPM3nUSOlzmVyV6KZNUaHtJppRczVgKzzICXcAWQB5wMdT6xCBqNEme3AoyOWd6kPxx+XCxQ2ffRQ5s+vWKuUtHz3DTXE4hVSSKGU3JjMAEzKIjHT4RIn36qlPHIX8jJm6Dz+\\\\\\\/oL82dBM5c7z1mMoUoa6BRTlxdlCOZcmSNWwZ\\\\\\\/xCr5EWgBS8Qn5avrohNdn5OJH7vFny9rHztLamUDFECGY4Zj+soRbvBR2\\\\\\\/SOYM3yoQG5Zk27\\\\\\\/3G47lJ6VmoTlAfk24\\\\\\\/iKvFWg+F06yFIq1KMJdX3pYt9Xymlj9nicAnfk1ivpIvpqxdBw7q9NCxE8wOazzRynRTgxW+avOd5saG8YPi2ZrZQkuh9oUyLqV3GSh\\\\\\\/6aX0lah3vnp8HrKSOXBAOSPB79J85cV9aK55BHI0IOjWvMoo4u57fYKq5EWZyD8tIoHuOrcLMZYCaDzLHXAiwebfmUxb+\\\\\\\/LNTpVXAkfaKo0v4vxK1VNWxW8okh0prGJx4XpY\\\\\\\/VTnrPVVL0uArIL6naPATStWbwzkGBRpj+31bs5GgTSFFfdBfPwmGIPnRO2uYF3k32wqZ36sukiTWnERrz6vS\\\\\\\/u6QUf2RuQNyrVVg+C1xuFNWqV8v5E4LwVb45r6Q4XKpifLi+C\\\\\\\/GdZr0x9lCOdeMP96dVQU6iA4jKaggeC0djk5j4JylbHjB3qznTAhhJJs2SxfMjRLQ1Ik8+xV+A1HIbUApIAAih\\\\\\\/mV2BPwY8zuihtYgsLrtXeJCjXOoKepzuFkVbEqQFcri4NuRx0\\\\\\\/X866HWU92d0w2PDY03BqNz0irOMtqyp4+5CieDvNx4G7+XezVHDBALk0klM4tRL1lSYQxTcQksvMUY5lVoMa5lBC7yRb3LBMuD3YjtP+1yMAq8xl856sZKiCznDKUU\\\\\\\/Tgm948IF05rX\\\\\\\/\\\\\\\/oumdbrPeD\\\\\\\/Dd5hv2rS68wkLQNHajtLbe80Jtf1tYhJJfQG4mRnPFhaYyRZH0gvcdAsfrFMh+kkbf6zO1VRGyyLgSLJLVsPuwnHRF3W119dxYH786ilCh5byIcF1Hol5R7dYkC2Z6lRqGA\\\\\\\/F8ku9NHOCE6yifx5WPoeLg160eRI5NAvnAM0dayMpl8qvIxF89\\\\\\\/gwIGR2tvfEUQ37KlRUgnkwzN4ay2O0u8RXojV++Gby5bAMIWfXY5i\\\\\\\/VxA3TXmUOVvUKGdKTEy5+dHPvtQQvCrdic2SNbqmTcX5cTkqoaEFW2DcJenyBtNSyctV7k7uyVCm+7Kphf9WZ\\\\\\\/7ZG+KRYPB3EK\\\\\\\/rbUi\\\\\\\/FgchRnnMJjEV9NfA8\\\\\\\/BrH+o5EukC8hfOo9m8A4h63ITeUSmDe5Ho6PNQj6pmK9FtHm\\\\\\\/Qi8DSrkXvAAcfLf1exWpOaUKrZsGTqS8BTd4Y92ciwibXMW50jYcr13WG3WwZ21kAXMXIDDss4Ptwdi3WLzGjY2+RHiMcsboAfovXhEhyrkSiMA4oP+bGbo4s9yhwHIS0SjrKG2bojqyeIY\\\\\\\/ZF6Aqd5\\\\\\\/6ZiTF05H+x8Tthot5Q+4vKevR65+BjORgnjTGlfpKg20qTl33miVAY8hZcyXXNIGagXn\\\\\\\/JH9UfpiShkCS8fuG6JcCCEpFWb5vG0J0IZzdztQLWvUL5nIcpmPDw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022YwK5k5c0U9p7MC7slG9bNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022167417f0d41a0cb61da86254151a654512dc89568c637084c560afd666c639df\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459493 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1095" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-abmrtTrswmn32tPhgfRMQCTJyiv-xGV9zgsURpdVHa0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1095" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-T3MWxXMrQYUUJJ4RBYeYoAYElFDktNsKUtkDhtW4nkM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1085" title="Using Genome-Wide Association Analysis to Characterize Environmental Sensitivity of Milk Traits in Dairy Cattle" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1105" title="Sequence-Based Mapping of the Polyploid Wheat Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459493" id="node459493" data-pisa="ggg;3/7/1095" data-pisa-master="ggg;g3.113.006510" data-apath="/ggg/3/7/1095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel James White</span>, <span class="highwire-citation-author" data-delta="1">David Bryant</span> and <span class="highwire-citation-author" data-delta="2">Neil John Gemmell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1095-1104;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006510 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459493" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel James White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodiversity and Informatics, Landcare Research New Zealand, Auckland, 1072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJames%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJames%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:whited@landcareresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">whited@landcareresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Bryant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics and Statistics, University of Otago, Dunedin, 9054, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BBryant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryant%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BBryant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Neil John Gemmell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Reproduction and Genomics, Department of Anatomy, University of Otago, Dunedin, 9054, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neil%2BJohn%2BGemmell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gemmell%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeil%2BJohn%2BGemmell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/7/1095" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459493" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1095.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459493" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1095.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459493" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1095.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459493" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/7/1095.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459493" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Empirical proof of human mitochondrial DNA (mtDNA) recombination in somatic tissues was obtained in 2004; however, a lack of irrefutable evidence exists for recombination in human mtDNA at the population level. Our inability to demonstrate convincingly a signal of recombination in population data sets of human mtDNA sequence may be due, in part, to the ineffectiveness of current indirect tests. Previously, we tested some well-established indirect tests of recombination (linkage disequilibrium <em>vs.</em> distance using D′ and r<sup>2</sup>, Homoplasy Test, Pairwise Homoplasy Index, Neighborhood Similarity Score, and Max χ<sup>2</sup>) on sequence data derived from the only empirically confirmed case of human mtDNA recombination thus far and demonstrated that some methods were unable to detect recombination. Here, we assess the performance of these six well-established tests and explore what characteristics specific to human mtDNA sequence may affect their efficacy by simulating sequence under various parameters with levels of recombination (ρ) that vary around an empirically derived estimate for human mtDNA (population parameter ρ = 5.492). No test performed infallibly under any of our scenarios, and error rates varied across tests, whereas detection rates increased substantially with ρ values &gt; 5.492. Under a model of evolution that incorporates parameters specific to human mtDNA, including rate heterogeneity, population expansion, and ρ = 5.492, successful detection rates are limited to a range of 7−70% across tests with an acceptable level of false-positive results: the neighborhood similarity score incompatibility test performed best overall under these parameters. Population growth seems to have the greatest impact on recombination detection probabilities across all models tested, likely due to its impact on sequence diversity. The implications of our findings on our current understanding of mtDNA recombination in humans are discussed.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/mtdna" class="hw-term hw-article-keyword hw-article-keyword-mtdna" rel="nofollow">mtDNA</a></li><li class="kwd"><a href="/keyword/indirect-tests" class="hw-term hw-article-keyword hw-article-keyword-indirect-tests" rel="nofollow">indirect tests</a></li><li class="kwd"><a href="/keyword/effectiveness" class="hw-term hw-article-keyword hw-article-keyword-effectiveness" rel="nofollow">effectiveness</a></li><li class="kwd"><a href="/keyword/human" class="hw-term hw-article-keyword hw-article-keyword-human" rel="nofollow">human</a></li></ul><p id="p-4">If recombination is occurring in human mitochondrial DNA (mtDNA) at a sufficiently high level, then many evolutionary analyses based on the molecule (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anisimova <em>et al.</em> 2003</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arenas and Posada 2010a</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">b</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schierup and Hein 2000a</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">b</a>), and indeed the models of evolution of the molecule itself, will need to undergo substantial revision. The question of whether recombination occurs at this level is, however, largely disregarded by the scientific community. This consensus has been reached based on a number of factors. It was long held that the mtDNA molecule evolves in a strictly clonal manner, with exclusively uniparental inheritance and no recombination. In fact, originally, it was not clear whether enzymes required for recombination were present in mitochondria at all. Furthermore, it was accepted that significant-enough mitochondrial heteroplasmy did not exist (more than one mitotype present in a cell or individual) for there to be any biological impact even if recombination were to occur. Early studies in which investigators reported apparent signals of recombination in human mtDNA (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Awadalla <em>et al.</em> 1999</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eyre-Walker <em>et al.</em> 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hagelberg <em>et al.</em> 1999</a>) were largely discounted by statistical arguments (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Innan and Nordborg 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jorde and Bamshad 2000</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2000</a>) and inaccuracies in sequence data (Hagelberg <em>et al.</em> 2000; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kivisild and Villems 2000</a>).</p><p id="p-5">However, the body of evidence supporting the potential for biologically important recombination in human mtDNA has increased over recent years. For example, in 1999 it was discovered that the enzymatic machinery necessary for recombination is present in mitochondria (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lakshmipathy and Campbell 1999</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Thyagarajan <em>et al.</em> 1996</a>). Important revelations came in 2004 and 2005, when widespread mtDNA recombination was discovered in up to 14.2% and 16.1% of animal populations, respectfully (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Piganeau <em>et al.</em> 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tsaousis <em>et al.</em> 2005</a>), although a recent and interesting study has shown that positive results for recombination in animal mtDNA could be an artifact of a specific form of mutation rate heterogeneity (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sun <em>et al.</em> 2011</a>). Experimental work has shown that recombination can occur between heterologous mitochondrial molecules, harboring pathogenic mutations, in human cytoplasmic hybrid cells (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">D’Aurelio <em>et al.</em> 2004</a>). Finally, studies on the patterns of tetraplasmy in doubly heteroplasmic individuals not only suggest mitochondrial recombination to be relatively common in human skeletal muscle tissue but also demonstrate the inheritability of recombinant mtDNA molecules (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zsurka <em>et al.</em> 2005</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2007</a>).</p><p id="p-6">For recombination to have a biological impact, it must occur between two different mitotypes; thus, a state of mitochondrial heteroplasmy is required. To this end, it is now accepted that mitochondrial heteroplasmy is a regular feature among higher eukaryotes (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">White <em>et al.</em> 2008</a>). As evidence, the paternal inheritance of mtDNA, a common source of heteroplasmy, is relatively widespread across the animal kingdom (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fontaine <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gyllensten <em>et al.</em> 1991</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sherengul <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhao <em>et al.</em> 2004</a>), and is also known to occur in humans (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schwartz and Vissing 2002</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zsurka <em>et al.</em> 2005</a>). Indeed, human mitochondrial heteroplasmy can persist in the germline over several generations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ivanov <em>et al.</em> 1996</a>).</p><p id="p-7">The most convincing <em>direct</em> evidence of a recombination event in human mtDNA to date remains that of a male sufferer of a mitochondrial myopathy, who had a 10:1 ratio of paternal to maternal mtDNA in his skeletal muscle. A deletion in his paternally derived mtDNA gave rise to the disorder, and it was found that 0.7% of mtDNA molecules were recombinant (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kraytsberg <em>et al.</em> 2004</a>). This research group subsequently went on to show that recombination in this tissue might be relatively common (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Zsurka <em>et al.</em> 2005</a>).</p><p id="p-8">To understand the appropriate role of recombination in evolutionary analyses centered on the mtDNA molecule, we require more than sporadic cases leading to dysfunction. Instead, we require evidence for mitochondrial recombination at a population level. Such evidence can be provided by the numerous statistical tests that estimate the probability that polymorphism distributions in DNA sequence are due to recombination (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>, see <a href="http://www.bioinf.manchester.ac.uk/recombination/programs.shtml">http://www.bioinf.manchester.ac.uk/recombination/programs.shtml</a> for an up to date list). The early evidence for mitochondrial recombination in human populations using these methods (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Awadalla <em>et al.</em> 1999</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Eyre-Walker <em>et al.</em> 1999</a>) is considered equivocal at best, however, and has largely been disregarded. The reasons for refuting the positive signals for recombination as revealed in these studies include the incorrect choice of statistical test (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jorde and Bamshad 2000</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2000</a>), the quality of sequence data (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kivisild and Villems 2000</a>), and inaccurate interpretation of results (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Innan and Nordborg 2002</a>). Subsequent attempts to repeat the findings of the first studies have been unsuccessful (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elson <em>et al.</em> 2001</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Piganeau and Eyre-Walker 2004</a>).</p><p id="p-9">The question we focus on here is why there is such a lack of compelling evidence for mtDNA recombination frequency in human populations, given that there is definitely potential for biologically relevant mitochondrial recombination to occur, and it is occurring at the population level in our animal counterparts. With a lack of compelling evidence, the majority of the scientific community is content to accept that it may simply just not be occurring at a significantly high enough level for due concern.</p><p id="p-10">One intriguing possibility is that recombination may well be occurring in human mtDNA but that we are not actually able to detect it with the current set of tools. In a previous study, we demonstrated that some of the most robust indirect statistical tests for recombination detection are not effective at detecting recombination in human mtDNA, even when it is known to have occurred (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a>).</p><p id="p-11">Here, we explore how some of the processes under which human mtDNA evolves may affect the effectiveness of indirect tests of recombination detection. Specifically, we evolve DNA sequence under parameters consistent with human mtDNA, including rate heterogeneity and population expansion. We perform these simulations by using variable levels of recombination to determine the limitations of six well-established indirect tests of recombination. The power of these recombination tests has been comprehensively assessed elsewhere (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>) and our objective here is not to repeat such work. Rather, our intent is to assess the effects, on the detection abilities of popular tests of recombination, of some of the key parameters known to impact the evolution of this molecule in humans. In particular, we ask the question, what level of recombination in human mtDNA might we need before it can be reliably detected with the currently available recombination detection tools? We find that current indirect tests of recombination perform very poorly even under our simple models of sequence evolution, and likely will perform even worse as model complexity increases toward a scenario closer to biological reality. We discuss how our findings could influence previous studies, and current opinions, of human mtDNA recombination.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sequence simulation</h3><p id="p-12">Sequence was simulated under a relatively simple scenario consistent with the evolution of human mtDNA, with varying levels of recombination. To estimate the population parameters to use for the simulation of sequences, recombinant mtDNA sequences from the only known case of empirical recombination detection in human mtDNA (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kraytsberg <em>et al.</em> 2004</a>) were run in LAMARC v2.1 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kuhner 2006</a>). Assuming a mutation rate, μ, of 5 × 10<sup>−7</sup> per site per generation for the mtDNA molecule (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mishmar <em>et al.</em> 2003</a>), the required number of individuals to generate the observed genetic diversity within this sample of recombined sequences, effective population size N, was approximated at 1370 (from N = θ/μ, where θ is the mutation rate parameter). The recombination factor, R, was estimated in LAMARC from R = r/μ, where r is the recombination rate per intersite link per generation and μ is the mutation rate per site per generation (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kuhner 2006</a>) and can be described as the relative frequency of recombination next to a given site compared with a mutation at that site. For our sequences, and assuming μ to be 5 × 10<sup>−7</sup>, R was estimated at 1.170 giving an r value of 5.8485 × 10<sup>−7</sup> (see <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a> for more details).</p><p id="p-13">These parameters were incorporated into RECODON v1.5.0, which generates sequence under a coalescent model (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arenas and Posada 2007</a>). Population parameters per locus were estimated from a product of twice the number of inheritable copies, the per-site factor and the length in basepairs of the locus, l. As such, the per-locus population recombination parameter, ρ, was approximately Nrl for mitochondrial sequence when we assume an equal number of females and males in the population (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arenas and Posada 2007</a>); we set l to 6854 bp, which gives a value for ρ of 5.492 for the empirical data set. To account for recombination occurring only in females for mtDNA, the per-site recombination rate was halved for the simulations; otherwise, the level of recombination is overestimated in the simulated sequences. Three models of sequence evolution were used: (1) constant population size and homogeneous mutation rate; (2) constant population size and heterogeneous mutation rate; and (3) population growth and heterogeneous mutation rate. For models that included rate heterogeneity a gamma distribution of mutation rates with alpha set at 0.2 was used (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yang 1996</a>), and for the model that included population growth an exponential growth rate of 1 × 10<sup>−3</sup> per generation was incorporated (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pluzhnikov <em>et al.</em> 2002</a>). For each evolutionary model, ρ was set to 0, 2, 5.492, 8, 10, and 12 to span the estimated value for mtDNA (5.492), and 100 samples of 33 sequences were generated for each level of recombination. Sequences were evolved under the Jukes-Cantor nucleotide model assuming equal frequencies for all nucleotides, with an equal transition-transversion ratio.</p></div><div id="sec-3" class="subsection"><h3>The tests</h3><p id="p-14">Six well-known indirect tests of recombination were chosen for assessment. These include three population-based tests: the correlation between linkage disequilibrium (LD) and physical distance using the D− measure of LD; the correlation between LD and physical distance using the r<sup>2</sup> measure of LD; and the Homoplasy Test (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Smith and Smith 1998</a>). Three general methods also were assessed, including Max χ<sup>2</sup> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Smith 1992</a>), the Neighborhood Similarity Score (NSS) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jakobsen and Easteal 1996</a>), and the more recently developed Pairwise Homoplasy Index (PHI) test (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>). We have described these indirect tests of recombination in some detail elsewhere (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a>) but briefly re-encapsulate the key points of difference among them here.</p><p id="p-15">The use of LD to predict recombination relies on a significant negative correlation between physical distance and strength of LD in the presence of recombination (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Awadalla <em>et al.</em> 1999</a>). For both measures of LD, Pearson’s correlation coefficient was used, and the statistical significance of the correlation was estimated after 1000 random permutations of the data using a Mantel test, all implemented in RecombiTEST (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Piganeau <em>et al.</em> 2004</a>). The Homoplasy Test is used to examine whether there are more homoplasies on a phylogenetic tree than expected under a model of clonal inheritance, where a homoplasy is the co-occurrence of a polymorphism on separate branches of the tree (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Smith and Smith 1998</a>). We implemented this test in the Linux operating system by using a C translation of the original QBasic version, an unsupported version kindly provided by David Posada (University of Vigo). To simulate the process of synonymous site selection, which is a recommended step to control for the compounding effects of selection on recombination detection, a second file was generated for analysis by using every third base pair of the 6854 bp-long simulated sequence. The C version of Homoplasy was validated by comparing its performance to the original QBasic program using five files with no recombination (ρ = 0) and five with an extreme level of recombination (ρ = 15). The results are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006510/-/DC1/006510SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006510/-/DC1/TableS1.pdf">Table S1</a>.</p><p id="p-16">The Max χ<sup>2</sup> test is used to compare the arrangement of segregating sites either side of a putative crossover break point. Both the NSS and PHI tests look at the distribution of incompatible sites, where sites are compatible only if their history includes no recurrent or convergent mutation or recombination. C versions of Max χ<sup>2</sup>, NSS and PHI (<a href="http://www.maths.otago.ac.nz/~dbryant/software.html">http://www.maths.otago.ac.nz/~dbryant/software.html</a>) were run in Linux, and the significance of results estimated by randomly positioning the informative sites 1000 times and determining the proportion of permuted scores that are: (1) below the observed test score, (2) at least as high as the observed test score, and (3) less than or equal to the observed score, respectfully. For all six tests, a significant signal for recombination was assumed for a dataset if a <em>P</em> value of 0.05 or less was achieved. For more detail on the tests see <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">White and Gemmell (2009)</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006510/-/DC1/FileS1.pdf">File S1</a>. Simulated sequence data is availabe for download from <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006510/-/DC1/FileS2.gz">File S2</a>.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><p id="p-17">For the Kraytsberg recombinant sequence data, recombination per site per generation was estimated by LAMARC as 5.849 × 10<sup>−7</sup>, which gives a most probable population recombination parameter per gene, ρ, of 5.492. The population mutation parameter, θ (Nµ), was estimated as 6.850 × 10<sup>−4</sup>, which equates to 4.695 per gene.</p><div id="sec-5" class="subsection"><h3>Effect of recombination rate</h3><p id="p-18">For sequence simulated under the simplest scenario of no population growth or mutation rate heterogeneity, recombination detection was at, or near, 100% for ρ = 12 for the majority of tests (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The clear exception to this is the Max χ<sup>2</sup> test, which reported recombination in only 87% of cases. For the lowest level of recombination with ρ set to 2, only one test (the Homoplasy Test) detected recombination more than 90% of the time. Overall, levels of detection were relatively low for ρ = 2, ranging from 58% to 99%, with a mean of 74.0% (compared with a mean of 98.0% for ρ = 12). Interestingly, for ρ = 5.492 (the value estimated for the empirical example of human mtDNA recombination) the rates of detection ranged from 81 to 100% across all tests, with a mean of 94.4% (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power of six indirect recombination tests at detecting recombination in simulated mtDNA sequence using (A) a simple mode of evolution and (B) a slightly more complex mode of evolution that incorporates mutation rate heterogeneity and population expansion, over a range of recombination values. D&#x2032;, LD vs. distance using D&#x2019;; r2, LD vs. distance using r2; HT, Homoplasy Test; HT 3rd, Homoplasy Test using only every third site; PHI p, Pairwise Homoplasy Index using permutations; PHI n, Pairwise Homoplasy Index using a normal approximation; NSS, Neighborhood Similarity Score; MS, Max &#x3C7;2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2047917001" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Power of six indirect recombination tests at detecting recombination in simulated mtDNA sequence using (A) a simple mode of evolution and (B) a slightly more complex mode of evolution that incorporates mutation rate heterogeneity and population expansion, over a range of recombination values. D&#x2032;, LD &lt;em&gt;vs.&lt;/em&gt; distance using D&#x2019;; r&lt;sup&gt;2&lt;/sup&gt;, LD &lt;em&gt;vs.&lt;/em&gt; distance using r&lt;sup&gt;2&lt;/sup&gt;; HT, Homoplasy Test; HT 3&lt;sup&gt;rd&lt;/sup&gt;, Homoplasy Test using only every third site; PHI p, Pairwise Homoplasy Index using permutations; PHI n, Pairwise Homoplasy Index using a normal approximation; NSS, Neighborhood Similarity Score; MS, Max &#x3C7;&lt;sup&gt;2&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1095/F1.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/7/1095/F1.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462878" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Power of six indirect recombination tests at detecting recombination in simulated mtDNA sequence using (A) a simple mode of evolution and (B) a slightly more complex mode of evolution that incorporates mutation rate heterogeneity and population expansion, over a range of recombination values. D′, LD <em>vs.</em> distance using D’; r<sup>2</sup>, LD <em>vs.</em> distance using r<sup>2</sup>; HT, Homoplasy Test; HT 3<sup>rd</sup>, Homoplasy Test using only every third site; PHI p, Pairwise Homoplasy Index using permutations; PHI n, Pairwise Homoplasy Index using a normal approximation; NSS, Neighborhood Similarity Score; MS, Max χ<sup>2</sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Although there was a slight reduction in efficiency of recombination detection when only third positions were considered in the Homoplasy Test, compared with when all sites are used, this difference was not significant.</p><p id="p-21">Five of the six tests reported some level of recombination when there was none. Only the PHI test did not. The percentage of false-positive results ranged from 1% for NSS to 30% with the Homoplasy Test when all sites were used.</p></div><div id="sec-6" class="subsection"><h3>Effect of mutation rate heterogeneity</h3><p id="p-22">When mutation rate heterogeneity, described by a gamma distribution with α equal to 0.2, is incorporated into sequence evolution, a very similar pattern to the simple scenario is observed in rates of detection across all tests of recombination. When means of detection rates across all tests are compared between the two scenarios, there is no significant difference (<em>P</em> &lt; 0.70, paired Student’s <em>t</em>-test). When results are compared between the two scenarios for each test separately, Max χ<sup>2</sup> showed a significant decrease in detection rates at all levels of recombination with mutation rate heterogeneity (<em>P</em> &lt; 0.009, paired Student’s <em>t</em>-test, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). LD <em>vs.</em> distance using r<sup>2</sup> also showed a smaller, yet significant, reduction in detection rates with mutation rate heterogeneity (<em>P</em> &lt; 0.042, paired Student’s <em>t</em>-test, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462995/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462995/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison of recombination detection rates for mtDNA between a simple model of evolution and a model that includes mutation rate heterogeneity but no population growth</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Effect of mutation rate heterogeneity and population growth</h3><p id="p-24">When a population growth rate of 1 × 10<sup>−3</sup>, and mutation rate heterogeneity described by a gamma distribution with α equal to 0.2, were incorporated into simulations, detection rates were significantly lower (<em>P</em> &lt; 8.18 × 10<sup>−3</sup>, paired Student’s <em>t</em>-test on means for all tests at each value of ρ, see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). All tests were less able to detect recombination at all levels of recombination. Although the Homoplasy Test maintained the greatest rate of detection, even at low levels of recombination (93% for all sites at ρ = 2), the rate of false-positive results was 61% for this test when all data were examined, and 37% when third positions were used. At the other end of the spectrum, Max χ<sup>2</sup> performed dismally with detection rates ranging from 4% to 11%. The detection rates of the four remaining tests were similar, ranging from 50% (NSS) to 74% (LD <em>vs.</em> distance using D′) at ρ =12. At ρ =5.492, the value estimated for the Kraytsberg recombinant sequence, rates of detection for these four tests ranged from 37% with LD <em>vs.</em> distance using r<sup>2</sup>, to 70% with LD <em>vs.</em> distance using D′.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="P value frequency distribution at &#x3C1; = 0 across the six indirect tests for the (A) simple model of sequence evolution (no rate heterogeneity or population growth) and (B) slightly more complex model of sequence evolution that incorporates mutation rate heterogeneity and population expansion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2047917001" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;P&lt;/em&gt; value frequency distribution at &lt;em&gt;&#x3C1;&lt;/em&gt; = 0 across the six indirect tests for the (A) simple model of sequence evolution (no rate heterogeneity or population growth) and (B) slightly more complex model of sequence evolution that incorporates mutation rate heterogeneity and population expansion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-2.medium.gif" width="361" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-2.medium.gif" width="361" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462864" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="P value frequency distribution at &#x3C1; = 0 across the six indirect tests for the (A) simple model of sequence evolution (no rate heterogeneity or population growth) and (B) slightly more complex model of sequence evolution that incorporates mutation rate heterogeneity and population expansion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2047917001" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;P&lt;/em&gt; value frequency distribution at &lt;em&gt;&#x3C1;&lt;/em&gt; = 0 across the six indirect tests for the (A) simple model of sequence evolution (no rate heterogeneity or population growth) and (B) slightly more complex model of sequence evolution that incorporates mutation rate heterogeneity and population expansion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-3.medium.gif" width="362" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-3.medium.gif" width="362" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1095/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462864" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child"><em>P</em> value frequency distribution at <em>ρ</em> = 0 across the six indirect tests for the (A) simple model of sequence evolution (no rate heterogeneity or population growth) and (B) slightly more complex model of sequence evolution that incorporates mutation rate heterogeneity and population expansion.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">A possible reason for the difference in results between simulations without rate heterogeneity and population growth, compared with those that did include these parameters, is the underlying sequence diversity. This has previously been shown to have a strong effect on the efficiency of indirect tests of recombination detection (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>). We calculated the mean nucleotide diversity, π, among sequences (1) in the simple scenario and (2) with mutation rate heterogeneity and population growth as 2.392 × 10<sup>−3</sup> and 1.257 × 10<sup>−3</sup>, respectively. The latter is comparable with the estimate of π for the empirical data set, 1.250 × 10<sup>−3</sup> (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a>). For simulated sequence with mutation rate heterogeneity and no population growth, π among sequences was estimated as 2.545 × 10<sup>−3</sup>.</p><p id="p-27">To assess the impact of the model of sequence evolution on false-positive error rates of the tests, histograms of <em>P</em> values obtained for ρ = 0 were plotted for both the most simple model of sequence evolution (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) and the model that included population growth and rate heterogeneity (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). A uniform distribution of <em>P</em> values indicates an unbiased test, a strongly right-skewed distribution indicates a conservative test, whereas a strongly left-skewed distribution indicates high false positive error rates. We observed great variability across tests, with the incompatibility tests (PHI and NSS) having distributions strongly skewed to the right whereas the homoplasy test shows a left skewed distribution.</p></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-28">In this work we have explored the possibility that the lack of evidence for recombination in human mtDNA is not necessarily due to its absence but rather might be a consequence of a lack of power to detect it among current tests. To test this thesis we simulated three sets of sequence with varying levels of recombination: (1) a sequence generated under a simple model using a reliable estimate for the rate of mtDNA mutation (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Mishmar <em>et al.</em> 2003</a>), but no population growth or mutation rate heterogeneity; (2) a sequence generated using a model that incorporates an unequal distribution of mutation rates across human mtDNA molecules (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Yang 1996</a>); and (3) a sequence generated using a model that incorporates a nonuniformly distributed mutation rate, as well as rates of recent human population growth (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Pluzhnikov <em>et al.</em> 2002</a>).</p><div id="sec-9" class="subsection"><h3>Performance of tests</h3><p id="p-29">Under the simple model of sequence evolution, the majority of tests were able to detect recombination at least 95% of the time when levels of recombination are relatively high, <em>i.e.</em>, with a population recombination parameter, ρ, of at least 8. The exception to this includes the Max χ<sup>2</sup> test for which a maximum rate of successful detection was estimated at 87%. When ρ is equal to 5.492, the estimated value for human mtDNA based on the Kraytsberg recombinant sequence data (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a>), detection rates were lower for all tests except the Homoplasy Test and decreased as low as 81% for Max χ<sup>2</sup> test.</p><p id="p-30">The performance of all tests changed dramatically when population growth and mutation rate heterogeneity were incorporated into the model of sequence evolution, but rate heterogeneity on its own had little effect (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). With the inclusion of an exponential growth factor into sequence simulation, there was a reduction in performance for all tests; most tests showed a marked reduction in their power of detection across all levels of recombination. This effect is most likely explained by the reduction in sequence diversity (<em>e.g.</em>, from π = 2.39 × 10<sup>−3</sup> (SD 0.60 × 10<sup>−3</sup>) in the simple scenario to 1.26 × 10<sup>−3</sup> (SD 0.41 × 10<sup>−3</sup>)), as the negative impact of low sequence diversity on the tests in this study has been demonstrated before (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>). Most severely affected was the Max χ<sup>2</sup> test, which is unsurprising given its inefficiency in scenarios of low sequence diversity, particularly under population growth (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>). Thus, this test (using the implementation of <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>) can be excluded as a useful test for detecting recombination under these conditions.</p><p id="p-31">An apparent exception to the overall trend of reduced power under population growth and mutation rate heterogeneity is the Homoplasy Test, which maintained levels of recombination detection greater than 90%, even for a ρ value of 2. The rate of false-positive results, however, more than doubled for this test to 61%. When only third positions are used, a step recommended in real data to prevent the compounding effects of selection on recombination detection (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Piganeau and Eyre-Walker 2004</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Smith and Smith 1998</a>), the rate of false-positive detection decreases to 37% with the homoplasy test, whereas simultaneously retaining relatively high power.</p><p id="p-32">Of course, a test might have high power not because it is accurate, but because it rejects the null model unnecessarily. We can check for this by simulating with no recombination, and then checking which tests falsely detect recombination. <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> clearly shows that the C version of the Homoplasy Test used here has an unacceptable false-positive rate with our data set, meaning that the apparent low false-negative rate (<em>i.e.</em>, high power) of this test is artifactual. In contrast, the incompatibility tests are overly conservative for our sequences, which have similar nucleotide diversity levels to empirical mtDNA data, suggesting inflated false-negative rates with a nominal <em>P</em> value cut-off of 0.05 for these tests.</p><p id="p-33">When these results are compared with previous studies (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2001</a>; <a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>) several comparisons can be made. Overall, the apparent power of tests to detect recombination, in a scenario of no population growth, is greater in this analysis than in previous studies (<a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>). For most tests, recombination is detected at greater percentages across all levels of recombination used here, despite the lower sequence divergence of our simulated sequences. It has previously been shown that the power of these tests increases with increasing sequence diversity (<a id="xref-ref-7-10" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>); however, we estimated an average pairwise sequence diversity, π, for the simple scenario sequences of 2.392 × 10<sup>−3</sup> (SD 0.60 × 10<sup>−3</sup>); approximately fivefold less than the least divergent sequences used in previous simulation studies (<a id="xref-ref-7-11" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>). Reasons for the greater rate of detection here, compared with previous studies, may therefore include a relatively large sample size in this study (n = 33), and a long sequence length (l = 6854 bp), both of which can increase the chance of recombination detection. However, the benefits of a long sequence length may be restricted by the incorporation of a sliding window into sequence analysis (<a id="xref-ref-7-12" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>).</p><p id="p-34">In their comprehensive review, <a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">Posada and Crandall (2001)</a> assessed 14 tests of recombination over a range of parameters and demonstrated that the Homoplasy Test was the most powerful at lower levels of divergence (π ~ 1%). As the level of diversity in our simulated sequences is lower again, and more biologically relevant for studies in humans, an observation of relatively high power with the Homoplasy Test for our data would not be completely unexpected. <a id="xref-ref-31-11" class="xref-bibr" href="#ref-31">Posada and Crandall (2001)</a> also showed that, with low mutation rate heterogeneity and low sequence divergence, false-positives should be low with the Homoplasy Test. However, we report a greater false-positive rate than previous studies at 25%, which is substantially larger than expectation (at <em>P</em> = 0.05), for the scenario that included no mutation rate heterogeneity.</p><p id="p-35">This observation suggests a discrepancy between the C version of the Homoplasy Test used in the Posada and Crandall’s 2001 study and that used here. To investigate this, we repeated the analyses done in the Posada and Crandall study, which used sequence simulated with lower diversity and shorter length, using our implemented C version. A similar false-positive rate was found as in Posada and Crandall’s study (4%; data not shown). The inflated probability of false-positive results here is therefore due to other factors, such as our larger sequence length and larger effective population size. Larger sequence length could give rise to increased false positives by the method in which the test accounts for homoplasies. With larger sequence length, and corresponding larger number of effective sites, the Homoplasy Test will consider the likelihood that any homoplasies present to have arisen by recurrent mutation to be very small, and instead will report recombination. The type 1 error rate increased to 61% when population growth and rate heterogeneity are included across all sites, and this increase is likely driven, at least in part, by the unequal distribution of mutation rates (<a id="xref-ref-31-12" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>). The apparent high power with the Homoplasy Test reported here should therefore be treated with caution.</p><p id="p-36">The PHI statistic of <a id="xref-ref-7-13" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> (2006)</a> has been reported as a powerful and robust test that can reliably detect recombination under a wide spectrum of parameters, including population growth. Moreover, it has been shown to report few false-positive results, even under strong mutation rate heterogeneity and autocorrelation of substitution rates. However, when there is little sequence diversity, it may be conservative in detecting recombination (<a id="xref-ref-7-14" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>). Our results agree with these initial findings in that the power of this test was good at high levels of recombination; however, its performance was reduced when recombination parameters were low. This reduced performance with low recombination is likely due to both reduced sequence diversity, and the incorporation of a sliding window that limits any benefits derived from the long sequence length (6854 bp). NSS, like PHI, is based on incompatibility, and incompatibility methods have been shown to be among the most powerful across a range of parameters (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2001</a>; <a id="xref-ref-31-13" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>; <a id="xref-ref-43-10" class="xref-bibr" href="#ref-43">Wiuf <em>et al.</em> 2001</a>). Here, it also appears to be successful for mtDNA, under no population growth and a uniform mutation rate. Our results agree with the results of earlier studies in that the incompatibility-based tests (PHI and NSS) are conservative when sequence diversity is low, especially at low levels of recombination (<a id="xref-ref-7-15" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-31-14" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>).</p><p id="p-37">The Max χ<sup>2</sup> method was originally developed for detecting recombination between distinct DNA sequences, showing substantial divergence (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Smith 1992</a>). Under such a scenario it has been shown to be a powerful test (<a id="xref-ref-31-15" class="xref-bibr" href="#ref-31">Posada and Crandall 2001</a>). Because of the low diversity of our simulated sequences, its poor performance in this study is unsurprising. However, the Max χ<sup>2</sup> test we used employs the entire sequence and all possible χ<sup>2</sup> values. If χ<sup>2</sup> tables with expected values of two or less are excluded, a step that prevents overtly large χ<sup>2</sup> values being produced when expected values are low, the Max χ<sup>2</sup> test may become more powerful without inflation of type 1 error (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Piganeau and Eyre-Walker 2004</a>).</p><p id="p-38">Historically, there has been some debate as to which measure of LD, r<sup>2</sup> or D′, is preferable for the detection of recombination. Some authors have questioned the use of LD decay as a reliable indication of recombination at all (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Innan and Nordborg 2002</a>). However, the LD association test that we performed compared favorably with four other tests under no population growth in a more recent simulation study in which the authors used both r<sup>2</sup> and D’ (<a id="xref-ref-7-16" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>). Although the first study to show recombination in human mtDNA with LD employed the r<sup>2</sup> measure (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Awadalla <em>et al.</em> 1999</a>), it has been argued that only D′, and not r<sup>2</sup>, is a suitable measure of LD for mtDNA based on certain statistical criteria, such as for maximum values of r<sup>2</sup> to be achieved all alleles must have equal frequency (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Jorde and Bamshad 2000</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2000</a>). In contrast, it has been suggested that the r<sup>2</sup> measure of LD is preferable over D′ for synonymous mutations in human mtDNA because synonymous sites in human mtDNA are described well by the finite sites model (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Piganeau and Eyre-Walker 2004</a>), a model for which r<sup>2</sup> is more suited (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McVean <em>et al.</em> 2002</a>). Simulation studies have generally not been able to differentiate the relative efficacies of using D′ or r<sup>2</sup> (<a id="xref-ref-7-17" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Meunier and Eyre-Walker 2001</a>). Like <a id="xref-ref-7-18" class="xref-bibr" href="#ref-7">Bruen <em>et al.</em> (2006)</a>, we found that r<sup>2</sup> is slightly less powerful under population growth, although in another study <a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">White and Gemmell (2009)</a> give an example in which a test based on r<sup>2</sup> detected recombination in human mtDNA when D′ did not—the only case in which we know recombination has occurred. It has been suggested that hypervariable mutation leading to recurrent mutation would cause a reduction in detection using D′ but not r<sup>2</sup> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Meunier and Eyre-Walker 2001</a>), and because this parameter is not included in our simulations here (whereas other evolutionary factors are), hypervariability could explain the discrepancy between the results presented here and those of our earlier analyses of empirical and simulated data. It would be interesting to explore further the effect of recurrent mutation on these tests.</p><p id="p-39">It is perhaps the scenario of mutation rate heterogeneity and population growth that is most relevant for human mtDNA, as these phenomena are believed to be major factors of human mtDNA evolution and recent demographic history (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Pluzhnikov <em>et al.</em> 2002</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Yang 1996</a>). As only minor differences are seen when mutation rate heterogeneity alone is included in models of sequence evolution, it is probable that population growth, and the subsequent reduction in sequence diversity, is the major contributor to the marked reduction in efficiency for tests under a slightly more complex scenario. Thus, it appears that the parameters under which human mtDNA is evolving have a substantial impact on the performance of these indirect tests of recombination.</p><p id="p-40">Ignoring the Homoplasy Test due to its inflated false-positive rate, and considering the discrepancies in the LD D′ test with previous studies (<a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a>), it is the incompatibility tests that are most robust to the parameters of sequence evolution implemented here, while maintaining low false-positive rates. Because the PHI test was unable to detect recombination in empirical data (<a id="xref-ref-41-9" class="xref-bibr" href="#ref-41">White and Gemmell 2009</a>), the NSS test may be the most suited of the tests included here to detect recombination in human mtDNA. The LD r<sup>2</sup> test may also be useful.</p><p id="p-41">The focus of this study, however, was not to provide estimates of the power of these tests to detect recombination, but rather to test how the parameters under which human mtDNA evolve might affect the efficacy of indirect tests of recombination. In general, the six approaches tested, all of which have been widely applied for the detection of recombination in sequence data, perform poorly. We demonstrate that under a realistic rate of recombination for human mtDNA, the performance of most indirect tests is reduced at least twofold when population growth and mutation rate heterogeneity are considered compared with a scenario when these factors are absent. Further, it is apparent that error rates vary dramatically across tests and that relying on a universal nominal <em>P</em> value of 0.05 to demonstrate statistical significance is simply not appropriate. As the evolution of human mtDNA is likely to be more complex than we simulated here, we strongly recommend further development of indirect tests of recombination for accurate detection in human mtDNA. At the very least, we suggest that simulations be run to explore the limitations of tests as part of empirical investigations.</p><p id="p-42">If we were to reinvestigate past efforts to detect recombination in human populations, considering the results of this study, conclusions may need to be reviewed. Studies thus far have used several indirect tests, including those that we have shown to be unsuitable for human mtDNA, and evidence has generally been negative or inconclusive (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Elson <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Herrnstadt <em>et al.</em> 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ingman <em>et al.</em> 2000</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Jorde and Bamshad 2000</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Piganeau and Eyre-Walker 2004</a>). By removing nonsuitable tests from studies, the landscape of evidence for no recombination in human populations changes. For example, in the most comprehensive screen for recombination in human mtDNA using indirect means to date, <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Piganeau and Eyre-Walker (2004)</a> used five tests on six population data sets and obtained twenty-eight <em>P</em> values. Of these 28, seven (25%) were statistically significant. However, if we remove <em>P</em> values obtained using LD <em>vs.</em> distance using D′ and the Max χ<sup>2</sup> test (the implementation of Max χ<sup>2</sup> in their study did not account for expected χ<sup>2</sup>values of less than 2), seven of 17 <em>P</em> values achieve statistical significance, or around 41%.</p><p id="p-43">Recombination in human mtDNA, if it is occurring, could not only have a dramatic impact on our understanding of disease etiology and transmission but also on numerous evolutionary inferences. Even small amounts could adversely compound evolutionary dating and bias phylogenetic analyses (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Schierup and Hein 2000a</a>,<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">b</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">White <em>et al.</em> 2008</a>). Recombination can lead to false discovery of positive selection using the dN/dS ratio, leading to erroneous interpretation of the impact of selection across genomes (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Anisimova <em>et al.</em> 2003</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Arenas and Posada 2010a</a>). Further, recombination has a direct and confounding effect on ancestral sequence reconstruction (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Arenas and Posada 2010b</a>). It is important, therefore, to obtain accurate estimates of mitochondrial recombination at the population level. Although attempts have been made, indirect evidence for mtDNA recombination in humans has been unconvincing thus far, despite compelling direct evidence. Here, we show that even if recombination in human mtDNA were to occur, its detection is not guaranteed using the currently available tests, and if we were to project from the trends presented here we would expect values of rho to need to be around three to four times that seen <em>in vivo</em> (<em>i.e.</em>, around 20), for rates of recombination detection to be in the 90% range for the tests reviewed here. This leads to the possibility that recombination could be occurring in human mtDNA at the population level, but that these levels are below current detection thresholds. This study is timely, as the advance of the new wave of sequencing technology will lead to many further opportunities to screen human populations for the presence of mitochondrial recombination. We foresee results from our work leading to improved accuracy of detection of human mitochondrial recombination frequency at the population level.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Konstantin Khrapko (Yale) for the original recombinant mtDNA dataset, Vladimir Mencl (Canterbury) for his programming expertise, Coen Willemse for support with the C translation of the homoplasy test, and David Posada (Vigo) for providing the C translation of the homoplasy test as well as making comments valuable to this manuscript. This work was further enriched after useful comments from an anonymous reviewer. This work was funded by the Royal Society of New Zealand, Marsden Grant UOC402, and by start-up funds provided by the University of Otago to N.J.G.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006510/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006510/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-12-18"><span class="received-label">Received </span>December 18, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-04-26"><span class="accepted-label">Accepted </span>April 26, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 White <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effect of recombination on the accuracy of the likelihood method for detecting positive selection at amino acid sites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnisimova%26rft.auinit1%253DM.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D1229%26rft.epage%253D1236%26rft.atitle%253DEffect%2Bof%2BRecombination%2Bon%2Bthe%2BAccuracy%2Bof%2Bthe%2BLikelihood%2BMethod%2Bfor%2BDetecting%2BPositive%2BSelection%2Bat%2BAmino%2BAcid%2BSites%26rft_id%253Dinfo%253Apmid%252F12871927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC8zLzEyMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.2" data-doi="10.1186/1471-2105-8-458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arenas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recodon: coalescent simulation of coding DNA sequences with recombination, migration and demography.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DArenas%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.spage%253D458%26rft.epage%253D458%26rft.atitle%253DRecodon%253A%2Bcoalescent%2Bsimulation%2Bof%2Bcoding%2BDNA%2Bsequences%2Bwith%2Brecombination%252C%2Bmigration%2Band%2Bdemography.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-458%26rft_id%253Dinfo%253Apmid%252F18028540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18028540&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.3" data-doi="10.1534/genetics.109.109736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arenas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Coalescent simulation of intracodon recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.109736%26rft_id%253Dinfo%253Apmid%252F19933876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzIvNDI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.4" data-doi="10.1534/genetics.109.113423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arenas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">The effect of recombination on the reconstruction of ancestral sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.113423%26rft_id%253Dinfo%253Apmid%252F20124027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NC80LzExMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.5" data-doi="10.1126/science.286.5449.2524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Linkage disequilibrium and recombination in hominid mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">2524</span>–<span class="cit-lpage">2525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAwadalla%26rft.auinit1%253DP.%26rft.volume%253D286%26rft.issue%253D5449%26rft.spage%253D2524%26rft.epage%253D2525%26rft.atitle%253DLinkage%2BDisequilibrium%2Band%2BRecombination%2Bin%2BHominid%2BMitochondrial%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.286.5449.2524%26rft_id%253Dinfo%253Apmid%252F10617471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODYvNTQ0OS8yNTI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The power to detect recombination using the coalescent.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D1421%26rft.epage%253D1424%26rft.atitle%253DThe%2BPower%2Bto%2BDetect%2BRecombination%2BUsing%2Bthe%2BCoalescent%26rft_id%253Dinfo%253Apmid%252F11420381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE4LzcvMTQyMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.7" data-doi="10.1534/genetics.105.048975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruen</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A simple and robust statistical test for detecting the presence of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2665</span>–<span class="cit-lpage">2681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048975%26rft_id%253Dinfo%253Apmid%252F16489234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI2NjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.8" data-doi="10.1093/hmg/ddh326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Aurelio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gajewski</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauck</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">L. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Heterologous mitochondrial DNA recombination in human cells.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3171</span>–<span class="cit-lpage">3179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddh326%26rft_id%253Dinfo%253Apmid%252F15496432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8yNC8zMTcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.9" data-doi="10.1086/316938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinnery</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lightowlers</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Analysis of European mtDNAs for recombination.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DElson%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D153%26rft.atitle%253DAnalysis%2Bof%2BEuropean%2BmtDNAs%2Bfor%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F316938%26rft_id%253Dinfo%253Apmid%252F11115380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/316938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11115380&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166399900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.10" data-doi="10.1098/rspb.1999.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">How clonal are human mitochondria?</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D266%26rft.issue%253D1418%26rft.spage%253D477%26rft.epage%253D483%26rft.atitle%253DHow%2Bclonal%2Bare%2Bhuman%2Bmitochondria%253F%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1999.0662%26rft_id%253Dinfo%253Apmid%252F10189711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjY2LzE0MTgvNDc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.11" data-doi="10.1371/journal.pone.0000892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fontaine</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooley</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence for paternal leakage in hybrid periodical cicadas (Hemiptera: Magicicada spp.).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFontaine%26rft.auinit1%253DK.%2BM.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De892%26rft.epage%253De892%26rft.atitle%253DEvidence%2Bfor%2Bpaternal%2Bleakage%2Bin%2Bhybrid%2Bperiodical%2Bcicadas%2B%2528Hemiptera%253A%2BMagicicada%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000892%26rft_id%253Dinfo%253Apmid%252F17849021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17849021&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.12" data-doi="10.1038/352255a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gyllensten</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wharton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Josefsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Paternal inheritance of mitochondrial-dna in mice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGyllensten%26rft.auinit1%253DU.%26rft.volume%253D352%26rft.issue%253D6332%26rft.spage%253D255%26rft.epage%253D257%26rft.atitle%253DPaternal%2Binheritance%2Bof%2Bmitochondrial%2BDNA%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F352255a0%26rft_id%253Dinfo%253Apmid%252F1857422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/352255a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1857422&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FX18500071&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.13" data-doi="10.1098/rspb.1999.0663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagelberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lio</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiefenhovel</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Evidence for mitochondrial DNA recombination in a human population of island Melanesia</span> [correction in Proc R Soc Lond B Biol Sci 2000;267:1595−1596.]. <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DHagelberg%26rft.auinit1%253DE.%26rft.volume%253D266%26rft.issue%253D1418%26rft.spage%253D485%26rft.epage%253D492%26rft.atitle%253DEvidence%2Bfor%2Bmitochondrial%2BDNA%2Brecombination%2Bin%2Ba%2Bhuman%2Bpopulation%2Bof%2Bisland%2BMelanesia%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1999.0663%26rft_id%253Dinfo%253Apmid%252F10189712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjY2LzE0MTgvNDg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.14" data-doi="10.1086/339933"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herrnstadt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Reduced-median-network analysis of complete mitochondrial DNA coding-region sequences for the major African, Asian, and European haplogroups.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">1152</span>–<span class="cit-lpage">1171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHerrnstadt%26rft.auinit1%253DC.%26rft.volume%253D70%26rft.issue%253D5%26rft.spage%253D1152%26rft.epage%253D1171%26rft.atitle%253DReduced-median-network%2Banalysis%2Bof%2Bcomplete%2Bmitochondrial%2BDNA%2Bcoding-region%2Bsequences%2Bfor%2Bthe%2Bmajor%2BAfrican%252C%2BAsian%252C%2Band%2BEuropean%2Bhaplogroups.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F339933%26rft_id%253Dinfo%253Apmid%252F11938495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/339933&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11938495&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175012400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.15" data-doi="10.1038/35047064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaessmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paabo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllensten</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mitochondrial genome variation and the origin of modern humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DIngman%26rft.auinit1%253DM.%26rft.volume%253D408%26rft.issue%253D6813%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMitochondrial%2Bgenome%2Bvariation%2Band%2Bthe%2Borigin%2Bof%2Bmodern%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35047064%26rft_id%253Dinfo%253Apmid%252F11130070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35047064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130070&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165815200046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.16" data-doi="10.1093/oxfordjournals.molbev.a004170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recombination or mutational hot spots in human mtDNA?</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1122</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D1122%26rft.epage%253D1127%26rft.atitle%253DRecombination%2Bor%2BMutational%2BHot%2BSpots%2Bin%2BHuman%2BmtDNA%253F%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004170%26rft_id%253Dinfo%253Apmid%252F12082131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTEyMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.17" data-doi="10.1038/ng0496-417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wadhams</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roby</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weedn</span>  <span class="cit-name-given-names">V. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mitochondrial DNA sequence heteroplasmy in the Grand Duke of Russia Georgij Romanov establishes the authenticity of the remains of TsarNicholas II.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DIvanov%26rft.auinit1%253DP.%2BL.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D417%26rft.epage%253D420%26rft.atitle%253DMitochondrial%2BDNA%2Bsequence%2Bheteroplasmy%2Bin%2Bthe%2BGrand%2BDuke%2Bof%2BRussia%2BGeorgij%2BRomanov%2Bestablishes%2Bthe%2Bauthenticity%2Bof%2Bthe%2Bremains%2Bof%2BTsar%2BNicholas%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0496-417%26rft_id%253Dinfo%253Apmid%252F8630496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0496-417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8630496&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UC55300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.18" data-doi="10.1093/bioinformatics/12.4.291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsen</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Easteal</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A program for calculating and displaying compatibility matrices as an aid in determining reticulate evolution in molecular sequences.</span> <abbr class="cit-jnl-abbrev">Comput. Appl. Biosci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DJakobsen%26rft.auinit1%253DI.%2BB.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D291%26rft.epage%253D295%26rft.atitle%253DA%2Bprogram%2Bfor%2Bcalculating%2Band%2Bdisplaying%2Bcompatibility%2Bmatrices%2Bas%2Ban%2Baid%2Bin%2Bdetermining%2Breticulate%2Bevolution%2Bin%2Bmolecular%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F12.4.291%26rft_id%253Dinfo%253Apmid%252F8902355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxMi80LzI5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.19" data-doi="10.1126/science.288.5473.1931a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bamshad</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Questioning evidence for recombination in human mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">1931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJorde%26rft.auinit1%253DL.%2BB.%26rft.volume%253D288%26rft.issue%253D5473%26rft.spage%253D1931%26rft.epage%253D1931%26rft.atitle%253DQuestioning%2Bevidence%2Bfor%2Brecombination%2Bin%2Bhuman%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5473.1931a%26rft_id%253Dinfo%253Apmid%252F10877699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.288.5473.1931a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10877699&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.20" data-doi="10.1126/science.288.5473.1931a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kivisild</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villems</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Questioning evidence for recombination in human mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">1931a</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKivisild%26rft.auinit1%253DT.%26rft.volume%253D288%26rft.issue%253D5473%26rft.spage%253D1931a%26rft.epage%253D1931a%26rft.atitle%253DQuestioning%2BEvidence%2Bfor%2BRecombination%2Bin%2BHuman%2BMitochondrial%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5473.1931a%26rft_id%253Dinfo%253Apmid%252F17835098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE0OiIyODgvNTQ3My8xOTMxYSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.21" data-doi="10.1126/science.1096342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraytsberg</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebralidse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Recombination of human mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKraytsberg%26rft.auinit1%253DY.%26rft.volume%253D304%26rft.issue%253D5673%26rft.spage%253D981%26rft.epage%253D981%26rft.atitle%253DRecombination%2Bof%2BHuman%2BMitochondrial%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1096342%26rft_id%253Dinfo%253Apmid%252F15143273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3My85ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.22" data-doi="10.1093/bioinformatics/btk051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">LAMARC 2.0: maximum likelihood and Bayesian estimation of population parameters.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtk051%26rft_id%253Dinfo%253Apmid%252F16410317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMi82Lzc2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.23" data-doi="10.1126/science.288.5473.1931a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowling</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Questioning evidence for recombination in human mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">1931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D288%26rft.issue%253D5473%26rft.spage%253D1931%26rft.epage%253D1931%26rft.atitle%253DQuestioning%2Bevidence%2Bfor%2Brecombination%2Bin%2Bhuman%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5473.1931a%26rft_id%253Dinfo%253Apmid%252F10877701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.288.5473.1931a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10877701&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.24" data-doi="10.1128/MCB.19.5.3869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lakshmipathy</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The human DNA ligase III gene encodes nuclear and mitochondrial proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3869</span>–<span class="cit-lpage">3876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLakshmipathy%26rft.auinit1%253DU.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D3869%26rft.epage%253D3876%26rft.atitle%253DThe%2BHuman%2BDNA%2BLigase%2BIII%2BGene%2BEncodes%2BNuclear%2Band%2BMitochondrial%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.5.3869%26rft_id%253Dinfo%253Apmid%252F10207110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE5LzUvMzg2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fearnhead</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A coalescent-based method for detecting and estimating recombination from gene sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1231%26rft.epage%253D1241%26rft.atitle%253DA%2BCoalescent-Based%2BMethod%2Bfor%2BDetecting%2Band%2BEstimating%2BRecombination%2BFrom%2BGene%2BSequences%26rft_id%253Dinfo%253Apmid%252F11901136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.26" data-doi="10.1093/oxfordjournals.molbev.a003756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meunier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The correlation between linkage disequilibrium and distance: Implications for recombination in hominid mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2132</span>–<span class="cit-lpage">2135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMeunier%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D2132%26rft.epage%253D2135%26rft.atitle%253DThe%2BCorrelation%2BBetween%2BLinkage%2BDisequilibrium%2Band%2BDistance%253A%2BImplications%2Bfor%2BRecombination%2Bin%2BHominid%2BMitochondria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003756%26rft_id%253Dinfo%253Apmid%252F11606711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8yMTMyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.27" data-doi="10.1073/pnas.0136972100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mishmar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Pesini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golik</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaulay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Natural selection shaped regional mtDNA variation in humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0136972100%26rft_id%253Dinfo%253Apmid%252F12509511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDAvMS8xNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.28" data-doi="10.1038/sj.hdy.6800413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piganeau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A reanalysis of the indirect evidence for recombination in human mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DPiganeau%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D282%26rft.epage%253D288%26rft.atitle%253DA%2Breanalysis%2Bof%2Bthe%2Bindirect%2Bevidence%2Bfor%2Brecombination%2Bin%2Bhuman%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800413%26rft_id%253Dinfo%253Apmid%252F14747833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14747833&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220403600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.29" data-doi="10.1093/molbev/msh244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piganeau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A broad survey of recombination in animal mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2319</span>–<span class="cit-lpage">2325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh244%26rft_id%253Dinfo%253Apmid%252F15342796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pluzhnikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Inferences about human demography based on multilocus analyses of noncoding sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPluzhnikov%26rft.auinit1%253DA.%26rft.volume%253D161%26rft.issue%253D3%26rft.spage%253D1209%26rft.epage%253D1218%26rft.atitle%253DInferences%2BAbout%2BHuman%2BDemography%2BBased%2Bon%2BMultilocus%2BAnalyses%2Bof%2BNoncoding%2BSequences%26rft_id%253Dinfo%253Apmid%252F12136023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS8zLzEyMDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.31" data-doi="10.1073/pnas.241370698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evaluation of methods for detecting recombination from DNA sequences: computer simulations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">13757</span>–<span class="cit-lpage">13762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPosada%26rft.auinit1%253DD.%26rft.volume%253D98%26rft.issue%253D24%26rft.spage%253D13757%26rft.epage%253D13762%26rft.atitle%253DEvaluation%2Bof%2Bmethods%2Bfor%2Bdetecting%2Brecombination%2Bfrom%2BDNA%2Bsequences%253A%2BComputer%2Bsimulations%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.241370698%26rft_id%253Dinfo%253Apmid%252F11717435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjQvMTM3NTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">a</span> <span class="cit-article-title">Consequences of recombination on traditional phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchierup%26rft.auinit1%253DM.%2BH.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D879%26rft.epage%253D891%26rft.atitle%253DConsequences%2Bof%2BRecombination%2Bon%2BTraditional%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Apmid%252F11014833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzIvODc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.33" data-doi="10.1093/oxfordjournals.molbev.a026256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">b</span> <span class="cit-article-title">Recombination and the molecular clock.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1578</span>–<span class="cit-lpage">1579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSchierup%26rft.auinit1%253DM.%2BH.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D1578%26rft.epage%253D1579%26rft.atitle%253DRecombination%2Band%2Bthe%2BMolecular%2BClock%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026256%26rft_id%253Dinfo%253Apmid%252F11018163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMC8xNTc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.34" data-doi="10.1056/NEJMoa020350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vissing</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Paternal inheritance of mitochondrial mtDNA.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">347</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DSchwartz%26rft.auinit1%253DM.%26rft.volume%253D347%26rft.issue%253D8%26rft.spage%253D576%26rft.epage%253D580%26rft.atitle%253DPaternal%2BInheritance%2Bof%2BMitochondrial%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa020350%26rft_id%253Dinfo%253Apmid%252F12192017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa020350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12192017&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177675400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.35" data-doi="10.1266/ggs.81.399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherengul</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuura</span>  <span class="cit-name-given-names">E. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of paternal transmission of mitochondrial DNA in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genes Genet. Syst.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2Bgenetic%2Bsystems%26rft.stitle%253DGenes%2BGenet%2BSyst%26rft.aulast%253DSherengul%26rft.auinit1%253DW.%26rft.volume%253D81%26rft.issue%253D6%26rft.spage%253D399%26rft.epage%253D404%26rft.atitle%253DAnalysis%2Bof%2Bpaternal%2Btransmission%2Bof%2Bmitochondrial%2BDNA%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fggs.81.399%26rft_id%253Dinfo%253Apmid%252F17283385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/ggs.81.399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17283385&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245138300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.36" data-doi="10.1007/BF00182389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Analyzing the mosaic structure of genes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D34%26rft.issue%253D2%26rft.spage%253D126%26rft.epage%253D129%26rft.atitle%253DAnalyzing%2Bthe%2Bmosaic%2Bstructure%2Bof%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00182389%26rft_id%253Dinfo%253Apmid%252F1556748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00182389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1556748&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HA89900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.37" data-doi="10.1093/oxfordjournals.molbev.a025960"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Detecting recombination from gene trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">590</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMaynard%2BSmith%26rft.auinit1%253DJ%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D590%26rft.epage%253D599%26rft.atitle%253DDetecting%2Brecombination%2Bfrom%2Bgene%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025960%26rft_id%253Dinfo%253Apmid%252F9580989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzUvNTkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.38" data-doi="10.1093/molbev/msr076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golding</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">“Patchy-Tachy” leads to false positives for recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2549</span>–<span class="cit-lpage">2559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr076%26rft_id%253Dinfo%253Apmid%252F21498600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzkvMjU0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.39" data-doi="10.1074/jbc.271.44.27536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thyagarajan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padua</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mammalian mitochondria possess homologous DNA recombination activity.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">27536</span>–<span class="cit-lpage">27543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DThyagarajan%26rft.auinit1%253DB.%26rft.volume%253D271%26rft.issue%253D44%26rft.spage%253D27536%26rft.epage%253D27543%26rft.atitle%253DMammalian%2BMitochondria%2BPossess%2BHomologous%2BDNA%2BRecombination%2BActivity%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.271.44.27536%26rft_id%253Dinfo%253Apmid%252F8910339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzEvNDQvMjc1MzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.40" data-doi="10.1093/molbev/msi084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsaousis</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ladoukakis</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zouros</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Widespread recombination in published animal mtDNA sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi084%26rft_id%253Dinfo%253Apmid%252F15647518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvOTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.41" data-doi="10.1093/molbev/msp073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemmell</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Can indirect tests detect a known recombination event in human mtDNA?</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1435</span>–<span class="cit-lpage">1439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp073%26rft_id%253Dinfo%253Apmid%252F19369597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTQzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.42" data-doi="10.1111/j.1365-294X.2008.03982.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolff</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemmell</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Revealing the hidden complexities of mtDNA inheritance.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4925</span>–<span class="cit-lpage">4942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWhite%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D4925%26rft.epage%253D4942%26rft.atitle%253DRevealing%2Bthe%2Bhidden%2Bcomplexities%2Bof%2BmtDNA%2Binheritance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03982.x%26rft_id%253Dinfo%253Apmid%252F19120984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03982.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19120984&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261104000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.43" data-doi="10.1093/oxfordjournals.molbev.a003733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiuf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A simulation study of the reliability of recombination detection methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1929</span>–<span class="cit-lpage">1939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DWiuf%26rft.auinit1%253DC.%26rft.volume%253D18%26rft.issue%253D10%26rft.spage%253D1929%26rft.epage%253D1939%26rft.atitle%253DA%2BSimulation%2BStudy%2Bof%2Bthe%2BReliability%2Bof%2BRecombination%2BDetection%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003733%26rft_id%253Dinfo%253Apmid%252F11557798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMC8xOTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.44" data-doi="10.1016/0169-5347(96)10041-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Among-site rate variation and its impact on phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D367%26rft.epage%253D372%26rft.atitle%253DAmong-site%2Brate%2Bvariation%2Band%2Bits%2Bimpact%2Bon%2Bphylogenetic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252896%252910041-0%26rft_id%253Dinfo%253Apmid%252F21237881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(96)10041-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21237881&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VC82300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.45" data-doi="10.1038/sj.hdy.6800516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Further evidence for paternal inheritance of mitochondrial DNA in the sheep (Ovis aries).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DZhao%26rft.auinit1%253DX.%26rft.volume%253D93%26rft.issue%253D4%26rft.spage%253D399%26rft.epage%253D403%26rft.atitle%253DFurther%2Bevidence%2Bfor%2Bpaternal%2Binheritance%2Bof%2Bmitochondrial%2BDNA%2Bin%2Bthe%2Bsheep%2B%2528Ovis%2Baries%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800516%26rft_id%253Dinfo%253Apmid%252F15266295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266295&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224092300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.46" data-doi="10.1086/511282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zsurka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampel</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornblum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraytsberg</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inheritance of mitochondrial DNA recombinants in double-heteroplasmic families: Potential implications for phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZsurka%26rft.auinit1%253DG.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D298%26rft.epage%253D305%26rft.atitle%253DInheritance%2Bof%2Bmitochondrial%2BDNA%2Brecombinants%2Bin%2Bdouble-heteroplasmic%2Bfamilies%253A%2Bpotential%2Bimplications%2Bfor%2Bphylogenetic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F511282%26rft_id%253Dinfo%253Apmid%252F17236134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/511282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17236134&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243729500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1095.47" data-doi="10.1038/ng1606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zsurka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraytsberg</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornblum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elger</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination of mitochondrial DNA in skeletal muscle of individuals with multiple mitochondrial DNA heteroplasmy.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZsurka%26rft.auinit1%253DG.%26rft.volume%253D37%26rft.issue%253D8%26rft.spage%253D873%26rft.epage%253D877%26rft.atitle%253DRecombination%2Bof%2Bmitochondrial%2BDNA%2Bin%2Bskeletal%2Bmuscle%2Bof%2Bindividuals%2Bwith%2Bmultiple%2Bmitochondrial%2BDNA%2Bheteroplasmy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1606%26rft_id%253Dinfo%253Apmid%252F16025113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16025113&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230880400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/7/1095.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1085" title="Using Genome-Wide Association Analysis to Characterize Environmental Sensitivity of Milk Traits in Dairy Cattle" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1105" title="Sequence-Based Mapping of the Polyploid Wheat Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/7"><span>Volume 3 Issue 7, July 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/7.cover-source.jpg?itok=DKXH6P5c" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1095.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9IFf8sSPpxv3ATNUuXUry67gYARntTY_rLJ0dfheX6A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/7/1095" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/7/1095" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/7/1095" class="active" data-icon-position="" data-hide-link-title="0">How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459493" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7IrtHJ-c0wwaFVOu3mHtL3I_c36gO9PnnkVi6O677ik" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1095.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/7/1095" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-dE-SEcax5xgarHsTf3gxWHul0HLMCH7ZzISIIcSHgkU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/7/1095" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1095.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459493" id="node459493--2" data-pisa="ggg;3/7/1095" data-pisa-master="ggg;g3.113.006510" data-apath="/ggg/3/7/1095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel James White</span>, <span class="highwire-citation-author" data-delta="1">David Bryant</span> and <span class="highwire-citation-author" data-delta="2">Neil John Gemmell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1095-1104;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006510 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459493--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel James White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodiversity and Informatics, Landcare Research New Zealand, Auckland, 1072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJames%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJames%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:whited@landcareresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">whited@landcareresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Bryant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics and Statistics, University of Otago, Dunedin, 9054, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BBryant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryant%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BBryant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Neil John Gemmell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Reproduction and Genomics, Department of Anatomy, University of Otago, Dunedin, 9054, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neil%2BJohn%2BGemmell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gemmell%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeil%2BJohn%2BGemmell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/7/1095" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/7/1095&amp;title=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/7/1095&amp;title=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/7/1095&amp;title=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/7/1095&amp;text=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/7/1095&amp;title=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/7/1095&amp;t=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/7/1095&amp;title=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/7/1095&amp;title=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459493/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459493" id="node459493--3" data-pisa="ggg;3/7/1095" data-pisa-master="ggg;g3.113.006510" data-apath="/ggg/3/7/1095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">How Good Are Indirect Tests at Detecting Recombination in Human mtDNA?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel James White</span>, <span class="highwire-citation-author" data-delta="1">David Bryant</span> and <span class="highwire-citation-author" data-delta="2">Neil John Gemmell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1095-1104;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006510 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459493--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel James White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biodiversity and Informatics, Landcare Research New Zealand, Auckland, 1072</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJames%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJames%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:whited@landcareresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">whited@landcareresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David Bryant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics and Statistics, University of Otago, Dunedin, 9054, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BBryant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryant%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BBryant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Neil John Gemmell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Allan Wilson Centre for Molecular Ecology and Evolution, University of Otago, Dunedin, 9054, New Zealand</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Reproduction and Genomics, Department of Anatomy, University of Otago, Dunedin, 9054, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neil%2BJohn%2BGemmell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gemmell%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeil%2BJohn%2BGemmell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1095&amp;rft.epage=1104&amp;rft.atitle=How%20Good%20Are%20Indirect%20Tests%20at%20Detecting%20Recombination%20in%20Human%20mtDNA%3F&amp;rft.volume=3&amp;rft.issue=7&amp;rft.date=2013-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=White%2C+Daniel+James&amp;rft.au=Bryant%2C+David&amp;rft.au=Gemmell%2C+Neil+John"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459493/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459493/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459493/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459493/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459493/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459493/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459493/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459493/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459493/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459493/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459493/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459493/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/7/1095" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/7/1095.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/7/1095.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/7/1095.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/7/1095.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
