<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/7/1069" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/7/1069.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/7/1069.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/7/1069.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/7/1069/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/7/1069.atom" />
<meta name="HW.pisa" content="ggg;3/7/1069" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, Strongylocentrotus purpuratus" />
<meta name="DC.Identifier" content="10.1534/g3.113.005769" />
<meta name="DC.Date" content="2013-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Kober, Pogson. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Codon usage bias has been documented in a wide diversity of species, but the relative contributions of mutational bias and various forms of natural selection remain unclear. Here, we describe for the first time genome-wide patterns of codon bias at 4623 genes in the purple sea urchin, Strongylocentrotus purpuratus . Preferred codons were identified at 18 amino acids that exclusively used G or C at third positions, which contrasted with the strong AT bias of the genome (overall GC content is 36.9%). The GC content of third positions and coding regions exhibited significant correlations with the magnitude of codon bias. In contrast, the GC content of introns and flanking regions was indistinguishable from the genome-wide background, which suggested a limited contribution of mutational bias to synonymous codon usage. Five distinct clusters of genes were identified that had significantly different synonymous codon usage patterns. A significant correlation was observed between codon bias and mRNA expression supporting translational selection, but this relationship was driven by only one highly biased cluster that represented only 8.6% of all genes. In all five clusters preferred codons were evolutionarily conserved to a similar degree despite differences in their synonymous codon usage distributions and magnitude of codon bias. The third positions of preferred codons in two codon usage groups also paired significantly more often in stems than in loops of mRNA secondary structure predictions, which suggested that codon bias might also affect mRNA stability. Our results suggest that mutational bias has played a minor role in determining codon bias in S. purpuratus and that preferred codon usage may be heterogeneous across different genes and subject to different forms of natural selection." />
<meta name="DC.Contributor" content="Kord M. Kober" />
<meta name="DC.Contributor" content="Grant H. Pogson" />
<meta name="article:published_time" content="2013-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, Strongylocentrotus purpuratus" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Codon usage bias has been documented in a wide diversity of species, but the relative contributions of mutational bias and various forms of natural selection remain unclear. Here, we describe for the first time genome-wide patterns of codon bias at 4623 genes in the purple sea urchin, &lt;i&gt;Strongylocentrotus purpuratus&lt;/i&gt;. Preferred codons were identified at 18 amino acids that exclusively used G or C at third positions, which contrasted with the strong AT bias of the genome (overall GC content is 36.9%). The GC content of third positions and coding regions exhibited significant correlations with the magnitude of codon bias. In contrast, the GC content of introns and flanking regions was indistinguishable from the genome-wide background, which suggested a limited contribution of mutational bias to synonymous codon usage. Five distinct clusters of genes were identified that had significantly different synonymous codon usage patterns. A significant correlation was observed between codon bias and mRNA expression supporting translational selection, but this relationship was driven by only one highly biased cluster that represented only 8.6% of all genes. In all five clusters preferred codons were evolutionarily conserved to a similar degree despite differences in their synonymous codon usage distributions and magnitude of codon bias. The third positions of preferred codons in two codon usage groups also paired significantly more often in stems than in loops of mRNA secondary structure predictions, which suggested that codon bias might also affect mRNA stability. Our results suggest that mutational bias has played a minor role in determining codon bias in &lt;i&gt;S. purpuratus&lt;/i&gt; and that preferred codon usage may be heterogeneous across different genes and subject to different forms of natural selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/07/01" />
<meta name="citation_mjid" content="ggg;3/7/1069" />
<meta name="citation_id" content="3/7/1069" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/7/1069" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/7/1069.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/7/1069.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/7/1069.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005769" />
<meta name="citation_pmid" content="23637123" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1069" />
<meta name="citation_lastpage" content="1083" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kord M. Kober" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064" />
<meta name="citation_author_institution" content="Department of Physiological Nursing, University of California, San Francisco, California 94143" />
<meta name="citation_author_email" content="kord.kober@nursing.ucsf.edu" />
<meta name="citation_author" content="Grant H. Pogson" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. A. Addison;citation_author=G. H. Pogson;citation_title=Multiple gene genealogies reveal asymmetrical hybridization and introgression among strongylocentrotid sea urchins.;citation_pages=1239-1251;citation_volume=18;citation_year=2009;citation_issue=6;citation_pmid=19222750;citation_doi=10.1111/j.1365-294X.2009.04094.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.;citation_pages=927-935;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005445" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Translational Selection and Yeast Proteome Evolution;citation_pages=1291-1303;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930740" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=H. Akashi;citation_author=A. Eyre-Walker;citation_title=Translational selection and molecular evolution.;citation_pages=688-693;citation_volume=8;citation_year=1998;citation_issue=6;citation_pmid=9914211;citation_doi=10.1016/S0959-437X(98)80038-5" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. G. Andersson;citation_author=C. G. Kurland;citation_title=Codon preferences in free-living microorganisms.;citation_pages=198-210;citation_volume=54;citation_year=1990;citation_issue=2;citation_pmid=2194095" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=M. Bailly-Bechet;citation_author=A. Danchin;citation_author=M. Iqbal;citation_author=M. Marsili;citation_author=M. Vergassola;citation_title=Codon usage domains over bacterial chromosomes.;citation_pages=e37-e37;citation_volume=2;citation_year=2006;citation_issue=4;citation_pmid=16683018;citation_doi=10.1371/journal.pcbi.0020037" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. J. Britten;citation_author=A. Cetta;citation_author=E. H. Davidson;citation_title=The single-copy DNA sequence polymorphism of the sea urchin Strongylocentrotus purpuratus.;citation_pages=1175-1186;citation_volume=15;citation_year=1978;citation_issue=4;citation_pmid=728997;citation_doi=10.1016/0092-8674(78)90044-2" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Bulmer;citation_title=Coevolution of codon usage and transfer RNA abundance.;citation_pages=728-730;citation_volume=325;citation_year=1987;citation_issue=6106;citation_pmid=2434856;citation_doi=10.1038/325728a0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Bulmer;citation_title=The selection-mutation-drift theory of synonymous codon usage.;citation_pages=897-907;citation_volume=129;citation_year=1991;citation_issue=3;citation_pmid=1752426" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=R. A. Cameron;citation_author=E. H. Davidson;citation_title=A basal deuterostome genome viewed as a natural experiment.;citation_pages=1-7;citation_volume=406;citation_year=2007;citation_issue=1-2;citation_pmid=17550788;citation_doi=10.1016/j.gene.2007.04.031" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. A. Cameron;citation_author=M. Samanta;citation_author=A. Yuan;citation_author=D. He;citation_author=E. Davidson;citation_title=SpBase: the sea urchin genome database and web site.;citation_volume=37;citation_year=2009;citation_pmid=19010966;citation_doi=10.1093/nar/gkn887" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Carlini;citation_author=Y. Chen;citation_author=W. Stephan;citation_title=The Relationship Between Third-Codon Position Nucleotide Content, Codon Bias, mRNA Secondary Structure and Gene Expression in the Drosophilid Alcohol Dehydrogenase Genes Adh and Adhr;citation_pages=623-633;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606539" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. I. Castillo-Davis;citation_author=D. L. Hartl;citation_title=Genome Evolution and Developmental Constraint in Caenorhabditis elegans;citation_pages=728-735;citation_volume=19;citation_year=2002;citation_issue=5;citation_pmid=11961106;citation_doi=10.1093/oxfordjournals.molbev.a004131" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. V. Chamary;citation_author=L. D. Hurst;citation_title=Evidence for selection on synonymous mutations affecting stability of mRNA secondary structure in mammals.;citation_pages=R75-R75;citation_volume=6;citation_year=2005;citation_issue=9;citation_pmid=16168082;citation_doi=10.1186/gb-2005-6-9-r75" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. V. Chamary;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals.;citation_pages=98-108;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418745;citation_doi=10.1038/nrg1770" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Chen;citation_author=W. Lee;citation_author=A. K. Hottes;citation_author=L. Shapiro;citation_author=H. H. McAdams;citation_title=Codon usage between genomes is constrained by genome-wide mutational processes.;citation_volume=101;citation_year=2004;citation_pmid=14990797;citation_doi=10.1073/pnas.0307827100" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Clarke;citation_title=Darwinian Evolution of Proteins;citation_pages=1009-1011;citation_volume=168;citation_year=1970;citation_issue=3934;citation_pmid=4909620;citation_doi=10.1126/science.168.3934.1009" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=W. G. Cochran;citation_title=Some methods for strengthening the common chi-2 Tests.;citation_pages=417-451;citation_volume=10;citation_year=1954;citation_doi=10.2307/3001616" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Cock;citation_author=T. Antao;citation_author=J. T. Chang;citation_author=B. A. Chapman;citation_author=C. J. Cox;citation_title=Biopython: freely available Python tools for computational molecular biology and bioinformatics.;citation_volume=25;citation_year=2009;citation_pmid=19304878;citation_doi=10.1093/bioinformatics/btp163" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_title=Population, Evolutionary and Genomic Consequences of Interference Selection;citation_pages=389-410;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019253" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_author=M. Aguade;citation_title=Natural Selection on Synonymous Sites Is Correlated With Gene Length and Recombination in Drosophila;citation_pages=239-249;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872963" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Drummond;citation_author=C. O. Wilke;citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.;citation_pages=341-352;citation_volume=134;citation_year=2008;citation_issue=2;citation_pmid=18662548;citation_doi=10.1016/j.cell.2008.05.042" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. B. DuMont;citation_author=J. C. Fay;citation_author=P. P. Calabrese;citation_author=C. F. Aquadro;citation_title=DNA Variability and Divergence at the Notch Locus in Drosophila melanogaster and D. simulans: A Case of Accelerated Synonymous Site Divergence;citation_pages=171-185;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166145;citation_doi=10.1534/genetics.167.1.171" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. Duret;citation_title=tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes.;citation_pages=287-289;citation_volume=16;citation_year=2000;citation_issue=7;citation_pmid=10858656;citation_doi=10.1016/S0168-9525(00)02041-2" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=L. Duret;citation_title=Evolution of synonymous codon usage in metazoans.;citation_pages=640-649;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433576;citation_doi=10.1016/S0959-437X(02)00353-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Duret;citation_author=D. Mouchiroud;citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis;citation_pages=4482-4487;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200288;citation_doi=10.1073/pnas.96.8.4482" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. B. Eisen;citation_author=P. T. Spellman;citation_author=P. O. Brown;citation_author=D. Botstein;citation_title=Cluster analysis and display of genome-wide expression patterns;citation_pages=14863-14868;citation_volume=95;citation_year=1998;citation_issue=25;citation_pmid=9843981;citation_doi=10.1073/pnas.95.25.14863" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=A. Eyre-Walker;citation_title=Synonymous codon bias is related to gene length in Escherichia coli: selection for translational accuracy?;citation_pages=864-872;citation_volume=13;citation_year=1996;citation_issue=6;citation_pmid=8754221;citation_doi=10.1093/oxfordjournals.molbev.a025646" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=W. M. Fitch;citation_title=The large extent of putative secondary nucleic acid structure in random nucleotide sequences or amino acid derived messenger-RNA.;citation_pages=279-291;citation_volume=3;citation_year=1974;citation_issue=4;citation_pmid=4416342;citation_doi=10.1007/BF01796043" />
<meta name="citation_reference" content="citation_title=The Causes of Molecular Evolution;citation_author=J. H. Gillespie;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Grantham;citation_author=C. Gautier;citation_author=M. Gouy;citation_author=R. Mercier;citation_author=A. Pavé;citation_title=Codon catalog usage and the genome hypothesis;citation_pages=r49-62;citation_volume=8;citation_year=1980;citation_issue=1;citation_pmid=6986610;citation_doi=10.1093/nar/8.1.197-c" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=R. M. Graze;citation_author=L. M. McIntyre;citation_author=B. J. Main;citation_author=M. L. Wayne;citation_author=S. V. Nuzhdin;citation_title=Regulatory divergence in Drosophila melanogaster and D. simulans, a genomewide analysis of allele-specific expression.;citation_volume=183;citation_year=2009;citation_pmid=19667135;citation_doi=10.1534/genetics.109.105957" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. A. Harris;citation_author=J. Clark;citation_author=A. Ireland;citation_author=J. Lomax;citation_author=M. Ashburner;citation_title=The Gene Ontology (GO) database and informatics resource;citation_pages=D258-261;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681407;citation_doi=10.1093/nar/gkh036" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Hasegawa;citation_author=T. Yasunaga;citation_author=T. Miyata;citation_title=Secondary structure of MS2 phage RNA and bias in code word usage;citation_pages=2073-2079;citation_volume=7;citation_year=1979;citation_issue=7;citation_pmid=537920;citation_doi=10.1093/nar/7.7.2073" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Graph. Statist.;citation_author=R. Ihaka;citation_author=R. Gentleman;citation_title=R: a language for data analysis and graphics.;citation_pages=299-314;citation_volume=5;citation_year=1996;citation_doi=10.2307/1390807" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. K. Ingvarsson;citation_title=Gene expression and protein length influence codon usage and rates of sequence evolution in Populus tremula.;citation_volume=24;citation_year=2007;citation_pmid=17204548;citation_doi=10.1093/molbev/msl212" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. L. King;citation_author=T. H. Jukes;citation_title=Non-Darwinian Evolution;citation_pages=788-798;citation_volume=164;citation_year=1969;citation_issue=3881;citation_pmid=5767777;citation_doi=10.1126/science.164.3881.788" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=D. Klämbt;citation_title=A model for messenger RNA sequences maximizing secondary structure due to code degeneracy.;citation_pages=57-65;citation_volume=52;citation_year=1975;citation_issue=1;citation_pmid=1152488;citation_doi=10.1016/0022-5193(75)90039-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kloster;citation_author=C. Tang;citation_title=SCUMBLE: a method for systematic and accurate detection of codon usage bias by maximum likelihood estimation.;citation_volume=36;citation_year=2008;citation_pmid=18495752;citation_doi=10.1093/nar/gkn288" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_title=Phylogenomics of Strongylocentrotid sea urchins.;citation_author=K. Kober;citation_author=G. Bernardi;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Evolutionary Genetics: From Molecules to Morphology;citation_title=The population and evolutionary genetics of codon bias;citation_author=M. Kreitman;citation_author=M. Antezana;citation_pages=82-101;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=255-258;citation_volume=324;citation_year=2009;citation_issue=5924;citation_pmid=19359587;citation_doi=10.1126/science.1170160" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Lowe;citation_author=S. Eddy;citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.;citation_volume=25;citation_year=1997;citation_pmid=9023104;citation_doi=10.1093/nar/25.5.0955" />
<meta name="citation_reference" content="citation_journal_title=J. Natl. Cancer Inst.;citation_author=N. Mantel;citation_author=W. Haenszel;citation_title=Statistical aspects of the analysis of data from retrospective studies of disease.;citation_volume=22;citation_year=1959;citation_pmid=13655060;citation_doi=10.1093/jnci/22.4.719" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=I. M. Meyer;citation_author=I. Miklós;citation_title=Statistical evidence for conserved, local secondary structure in the coding regions of eukaryotic mRNAs and pre-mRNAs;citation_pages=6338-6348;citation_volume=33;citation_year=2005;citation_issue=19;citation_pmid=16275783;citation_doi=10.1093/nar/gki923" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=K. Mita;citation_author=S. Ichimura;citation_author=M. Zama;citation_author=T. C. James;citation_title=Specific codon usage pattern and its implications on the secondary structure of silk fibroin mRNA.;citation_pages=917-925;citation_volume=203;citation_year=1988;citation_issue=4;citation_pmid=3210244;citation_doi=10.1016/0022-2836(88)90117-9" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Codon usage bias and tRNA abundance in Drosophila.;citation_pages=514-523;citation_volume=45;citation_year=1997;citation_issue=5;citation_pmid=9342399;citation_doi=10.1007/PL00006256" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Gene length and codon usage bias in Drosophila melanogaster, Saccharomyces cerevisiae and Escherichia coli.;citation_volume=26;citation_year=1998;citation_pmid=9628917;citation_doi=10.1093/nar/26.13.3188" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=H. Musto;citation_author=S. Cruveiller;citation_author=G. D’Onofrio;citation_author=H. Romero;citation_author=G. Bernardi;citation_title=Translational Selection on Codon Usage in Xenopus laevis;citation_pages=1703-1707;citation_volume=18;citation_year=2001;citation_issue=9;citation_pmid=11504850" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Navon;citation_author=Y. Pilpel;citation_title=The role of codon selection in regulation of translation efficiency deduced from synthetic libraries.;citation_pages=R12-R12;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21284851;citation_doi=10.1186/gb-2011-12-2-r12" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. A. Novembre;citation_title=Accounting for Background Nucleotide Composition When Measuring Codon Usage Bias;citation_pages=1390-1394;citation_volume=19;citation_year=2002;citation_issue=8;citation_pmid=12140252;citation_doi=10.1093/oxfordjournals.molbev.a004201" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=T. A. Oliver;citation_author=D. A. Garfield;citation_author=M. K. Manier;citation_author=R. Haygood;citation_author=G. A. Wray;citation_title=Whole-genome positive selection and habitat-driven evolution in a shallow and a deep-sea urchin.;citation_volume=2;citation_year=2010;citation_pmid=20935062;citation_doi=10.1093/gbe/evq063" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. R. Palumbi;citation_author=A. C. Wilson;citation_title=Mitochondrial DNA diversity in the sea urchins Strongylocentrotus purpuratus and S. droebachiensis.;citation_pages=403-415;citation_volume=44;citation_year=1990;citation_doi=10.2307/2409417" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=T. Pederson;citation_title=The sea urchin's siren.;citation_pages=9-14;citation_volume=300;citation_year=2006;citation_issue=1;citation_pmid=17113576;citation_doi=10.1016/j.ydbio.2006.10.006" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. Powell;citation_author=A. Caccone;citation_author=J. M. Gleason;citation_author=L. Nigro;citation_title=Rates of DNA evolution in Drosophila depend on function and developmental stage of expression.;citation_pages=291-298;citation_volume=133;citation_year=1993;citation_issue=2;citation_pmid=8094697" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. R. Powell;citation_author=E. N. Moriyama;citation_title=Evolution of codon usage bias in Drosophila;citation_pages=7784-7790;citation_volume=94;citation_year=1997;citation_issue=15;citation_pmid=9223264;citation_doi=10.1073/pnas.94.15.7784" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Qin;citation_author=W. B. Wu;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_author=W.-H. Li;citation_title=Intragenic Spatial Patterns of Codon Usage Bias in Prokaryotic and Eukaryotic Genomes;citation_pages=2245-2260;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611189;citation_doi=10.1534/genetics.104.030866" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Qiu;citation_author=R. Bergero;citation_author=K. Zeng;citation_author=D. Charlesworth;citation_title=Patterns of codon usage bias in Silene latifolia.;citation_volume=28;citation_year=2011;citation_pmid=20855431;citation_doi=10.1093/molbev/msq251" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=Y. Rao;citation_author=G. Wu;citation_author=Z. Wang;citation_author=X. Chai;citation_author=Q. Nie;citation_title=Mutation bias is the driving force of codon usage in the Gallus gallus genome.;citation_volume=18;citation_year=2011;citation_pmid=22039174;citation_doi=10.1093/dnares/dsr035" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. J. Saldanha;citation_title=Java Treeview—extensible visualization of microarray data.;citation_volume=20;citation_year=2004;citation_pmid=15180930;citation_doi=10.1093/bioinformatics/bth349" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. A. Shabalina;citation_author=A. Y. Ogurtsov;citation_author=N. A. Spiridonov;citation_title=A periodic pattern of mRNA secondary structure created by the genetic code.;citation_pages=2428-2437;citation_volume=34;citation_year=2006;citation_issue=8;citation_pmid=16682450;citation_doi=10.1093/nar/gkl287" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms.;citation_pages=28-38;citation_volume=24;citation_year=1986;citation_issue=1-2;citation_pmid=3104616;citation_doi=10.1007/BF02099948" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=T. M. Tuohy;citation_author=K. R. Mosurski;citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes;citation_pages=5125-5143;citation_volume=14;citation_year=1986;citation_issue=13;citation_pmid=3526280;citation_doi=10.1093/nar/14.13.5125" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. M. Sharp;citation_author=M. Averof;citation_author=A. T. Lloyd;citation_author=G. Matassi;citation_author=J. F. Peden;citation_title=DNA Sequence Evolution: The Sounds of Silence;citation_pages=241-247;citation_volume=349;citation_year=1995;citation_issue=1329;citation_pmid=8577834;citation_doi=10.1098/rstb.1995.0108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. C. Shields;citation_author=P. M. Sharp;citation_title=Synonymous codon usage in Bacillus subtilis reflects both translational selection and mutational biases;citation_pages=8023-8040;citation_volume=15;citation_year=1987;citation_issue=19;citation_pmid=3118331;citation_doi=10.1093/nar/15.19.8023" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=D. Shields;citation_author=P. Sharp;citation_author=D. Higgins;citation_author=F. Wright;citation_title=&quot;Silent&quot; sites in Drosophila genes are not neutral: evidence of selection among synonymous codons;citation_pages=704-716;citation_volume=5;citation_year=1988;citation_issue=6;citation_pmid=3146682" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. D. Singh;citation_author=V. L. Bauer DuMont;citation_author=M. J. Hubisz;citation_author=R. Nielsen;citation_author=C. F. Aquadro;citation_title=Patterns of mutation and selection at synonymous sites in Drosophila.;citation_volume=24;citation_year=2007;citation_pmid=18000010;citation_doi=10.1093/molbev/msm196" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Sodergren;citation_author=G. M. Weinstock;citation_author=E. H. Davidson;citation_author=R. A. Cameron;citation_author=R. A. Gibbs;citation_title=The Genome of the Sea Urchin Strongylocentrotus purpuratus;citation_pages=941-952;citation_volume=314;citation_year=2006;citation_issue=5801;citation_pmid=17095691;citation_doi=10.1126/science.1133609" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. E. Stajich;citation_author=D. Block;citation_author=K. Boulez;citation_author=S. E. Brenner;citation_author=S. A. Chervitz;citation_title=The Bioperl Toolkit: Perl Modules for the Life Sciences;citation_pages=1611-1618;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368254;citation_doi=10.1101/gr.361602" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stenico;citation_author=A. T. Lloyd;citation_author=P. M. Sharp;citation_title=Codon usage in Caenorhabditis elegans: delineation of translational selection and mutational biases;citation_pages=2437-2446;citation_volume=22;citation_year=1994;citation_issue=13;citation_pmid=8041603;citation_doi=10.1093/nar/22.13.2437" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=N. Stoletzki;citation_title=The surprising negative correlation of gene length and optimal codon use--disentangling translational selection from GC-biased gene conversion in yeast.;citation_pages=93-93;citation_volume=11;citation_year=2011;citation_pmid=21481245;citation_doi=10.1186/1471-2148-11-93" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy.;citation_volume=24;citation_year=2007;citation_pmid=17101719;citation_doi=10.1093/molbev/msl166" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=R. R. Strathmann;citation_title=The evolution and loss of feeding larval stages of marine invertebrates;citation_pages=894-906;citation_volume=32;citation_year=1978;citation_issue=4;citation_doi=10.2307/2407502;citation_georef=1979017285" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Tuller;citation_author=A. Carmi;citation_author=K. Vestsigian;citation_author=S. Navon;citation_author=Y. Dorfan;citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation.;citation_pages=344-354;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403328;citation_doi=10.1016/j.cell.2010.03.031" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Vicario;citation_author=C. E. Mason;citation_author=K. P. White;citation_author=J. R. Powell;citation_title=Developmental stage and level of codon usage bias in Drosophila.;citation_volume=25;citation_year=2008;citation_pmid=18755761;citation_doi=10.1093/molbev/msn189" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=Z. Wei;citation_author=R. C. Angerer;citation_author=L. M. Angerer;citation_title=A database of mRNA expression patterns for the sea urchin embryo.;citation_pages=476-484;citation_volume=300;citation_year=2006;citation_issue=1;citation_pmid=17007833;citation_doi=10.1016/j.ydbio.2006.08.034" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=B. Woolf;citation_title=On estimating the relation between blood group and disease.;citation_pages=251-253;citation_volume=19;citation_year=1955;citation_issue=4;citation_pmid=14388528;citation_doi=10.1111/j.1469-1809.1955.tb01348.x" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=M. Zama;citation_title=Codon usage pattern in alpha 2(I) chain domain of chicken type I collagen and its implications for the secondary structure of the mRNA and the synthesis pauses of the collagen.;citation_pages=772-776;citation_volume=167;citation_year=1990;citation_issue=2;citation_pmid=2322250;citation_doi=10.1016/0006-291X(90)92092-E" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=E. Zuckerkandl;citation_author=L. Pauling;citation_title=Molecules as documents of evolutionary history.;citation_pages=357-366;citation_volume=8;citation_year=1965;citation_issue=2;citation_pmid=5876245;citation_doi=10.1016/0022-5193(65)90083-4" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, Strongylocentrotus purpuratus" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
<meta name="twitter:description" content="Codon usage bias has been documented in a wide diversity of species, but the relative contributions of mutational bias and various forms of natural selection remain unclear. Here, we describe for the first time genome-wide patterns of codon bias at 4623 genes in the purple sea urchin, Strongylocentrotus purpuratus . Preferred codons were identified at 18 amino acids that exclusively used G or C at third positions, which contrasted with the strong AT bias of the genome (overall GC content is 36.9%). The GC content of third positions and coding regions exhibited significant correlations with the magnitude of codon bias. In contrast, the GC content of introns and flanking regions was indistinguishable from the genome-wide background, which suggested a limited contribution of mutational bias to synonymous codon usage. Five distinct clusters of genes were identified that had significantly different synonymous codon usage patterns. A significant correlation was observed between codon bias and mRNA expression supporting translational selection, but this relationship was driven by only one highly biased cluster that represented only 8.6% of all genes. In all five clusters preferred codons were evolutionarily conserved to a similar degree despite differences in their synonymous codon usage distributions and magnitude of codon bias. The third positions of preferred codons in two codon usage groups also paired significantly more often in stems than in loops of mRNA secondary structure predictions, which suggested that codon bias might also affect mRNA stability. Our results suggest that mutational bias has played a minor role in determining codon bias in S. purpuratus and that preferred codon usage may be heterogeneous across different genes and subject to different forms of natural selection." />
<meta name="og-title" property="og:title" content="Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, Strongylocentrotus purpuratus" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/7/1069" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Codon usage bias has been documented in a wide diversity of species, but the relative contributions of mutational bias and various forms of natural selection remain unclear. Here, we describe for the first time genome-wide patterns of codon bias at 4623 genes in the purple sea urchin, Strongylocentrotus purpuratus . Preferred codons were identified at 18 amino acids that exclusively used G or C at third positions, which contrasted with the strong AT bias of the genome (overall GC content is 36.9%). The GC content of third positions and coding regions exhibited significant correlations with the magnitude of codon bias. In contrast, the GC content of introns and flanking regions was indistinguishable from the genome-wide background, which suggested a limited contribution of mutational bias to synonymous codon usage. Five distinct clusters of genes were identified that had significantly different synonymous codon usage patterns. A significant correlation was observed between codon bias and mRNA expression supporting translational selection, but this relationship was driven by only one highly biased cluster that represented only 8.6% of all genes. In all five clusters preferred codons were evolutionarily conserved to a similar degree despite differences in their synonymous codon usage distributions and magnitude of codon bias. The third positions of preferred codons in two codon usage groups also paired significantly more often in stems than in loops of mRNA secondary structure predictions, which suggested that codon bias might also affect mRNA stability. Our results suggest that mutational bias has played a minor role in determining codon bias in S. purpuratus and that preferred codon usage may be heterogeneous across different genes and subject to different forms of natural selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7.cover-source.jpg" />
    <title>Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, Strongylocentrotus purpuratus | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"y88CZ03RfTZ9AnXuuTlz84z5tSCExbzRfsA6VGTN6Z4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459487","apath":"\/ggg\/3\/7\/1069.atom","pisa":"ggg;3\/7\/1069","ac":{"\/ggg\/3\/7\/1069.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/7\/1069.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1069"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/7\/1069"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/7\/1069":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226d5zvnuM6M3e5Uqmq2E0KpO7RAPURbYm4fsHQQIAS5ihjwvjcjSsm5fUrwPHyxksFMM9wWD+rw86F0ch9Qudakf4\\\\\\\/OLt+6F075L\\\\\\\/gvnyaPrd\\\\\\\/PKc4GyvY\\\\\\\/9sCDzS0rw3q5fxSMcUevxXgNHjgnyjGylVAUkKzwMkT0Qm7Qwf1n2XK8O3Kfl41DZSouFzyGvTrVIY3K5C+qdpByhdkkVhmIxzZ5+TyPMVHKqYvDJIc9AVuq0CfbsWnolUKB2r3f0givKGMZCzn28UY+a0zXep009kJua+GFDaqT+RDrPlP0Y1WmpD9qEhjdESTWIzEqYEp4NqWdSajWtT90VyyIS5gML02JiicXyfjNu5CT0wfbeltApg0Q8XjTWxm\\\\\\\/tYWqfI2HQt3QQLVoyks4\\\\\\\/cjEFETa0g3u5DsZAGsxIXIoA7q1ANAtNAi+JpiwFTVLj+k8aJmLhHtx2sq4sNJAeyl\\\\\\\/79BvPQIJQOGciEKgtcQR86UJ\\\\\\\/7A\\\\\\\/ABEnc1dPfFucz6VM5DJnEERCRqmsXYLkmuU1beFOh6yg9yfJkS+PN\\\\\\\/giHNlzBa4vnkxWu79UFgMRz0I3K7pLhXxkKd8qpuJA+HZZlhxfgxebSJHqCZZNYsJYDYun9DaGINrAEDin1PLx1A4fLz6AQ+DZJe9cKWVSKav\\\\\\\/SFzHEoINGNkBPxzeH35aujwYsViHz2RKXBuqb\\\\\\\/pBYHLW0sEWvPQUZVct5apSy4yWv3Zm8NNrbguEohEvZ\\\\\\\/LlCkza+i+4TM3Yxc93r6Mh6hURd5TmfmrVzsUpfZQzpPdTWu0b\\\\\\\/2ko\\\\\\\/qC8aK7M3xNriwVUx1uTcKiKFcqHckl3Re81xL79kM8Px0pvh+G6jTSQWbL1w2AphxNH96eXxm7pdu4guV6cMEhGtf8sRV9sZzu4pMPWSmTAZm7zlvQmiwejiGcriqH6Wumh4mCGPdegXjA9YltHjeMM5fT9oHUHAdsZ17CbYJCXqskoiOzzZgEzIFqoodra6TqI8PQFZFrTljBEkgJezoiK+xlbzSY7fxK1ykXrj1QKiP3pie9RJh\\\\\\\/mUB8lF\\\\\\\/IVPEEFb81MQIOUO7R\\\\\\\/2lD5\\\\\\\/IoUDdEZ3XcMIi3OOgFQ8viTIz+iB8\\\\\\\/qJK4EhNjPerHkUnbl\\\\\\\/axto1aGcjrfbv1jpb8R+UfMy1JLcRx+K2mXAwgAdp8kRkbzJuHDgXBrB3cBM\\\\\\\/2xjOeBekCf3X5pypMoF+XN4K6ntu3idcRxVlL+rDtt4OeyGXhZDjUhD7UUYo8kjAhAhBiRJmotj+whEj\\\\\\\/f9zuEcXbQeMi\\\\\\\/vMFoEg\\\u0022,\\\u0022iv\\\u0022:\\\u0022qaaxIOnOsVKercLlXlFYdQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fc3c250dd7e6d2c4b569e7dfde91ee48318d2159f27ebae5ca5ce4a64a6cf0a6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/qXuzOFQrszJRFKaE3a1QVYUN3zmgmcZZ73i2aKdU2yMHwJa5dIEHaBZOiVwI3QOi+ceQu7uEjkquzRhN8r5spPS\\\\\\\/Ufmw+LXjTVF1XzEtAPszKjZTcjBom5mSk+RRmdieE\\\\\\\/o6RBn7rOSc2S750\\\\\\\/MKHapsaCQkEzxmwSqH9Bl79VLhBBHScm72srtk5vpibRNm4czDwYU1oV4dpWvnzHKOTJU4ynSoQh\\\\\\\/sNZ20tYXnChdIXt2hSdQegtpRijzu5xi1TRDUrgMVUPYrNkubV5YNByg1aZ0QzqNba0GC8+Dwz5hjGeTbWEDdsWOTg1civxKB9HhO0PE+JsrV2479G1UOLOMaqRuFGGvq4l3kxZlM65n050KaK+7X\\\\\\\/bCWvejkxOFaJr0AttSWenO7yGLa5ECQwBadMgpJWoJWauZZDr\\\\\\\/IkyrLkkgg60F3pOwWhYBNZW0bwOkLnClUTU3g4xAALqhJKXPAClN4+3aTR4bVGQeiW2mr8pPZiRPPesgNfs8mZM6M8OcwZ92WAKTp86KFsI9iNfdthEr\\\\\\\/VtmpTzoDskal0mlfzJfhav1IWMT4JUj+WhDZAKo89BVSCFmyT0Wkv7AunDJA8\\\\\\\/I8xWG6SexoRMXZ47\\\\\\\/bcenaJDZkl0rN4z\\\\\\\/9sp2TTBy6ZarN3L\\\\\\\/wDwEcVAayfYaMTYbzqT33ZjnXJk0cs\\\\\\\/c9iEymA2HZpKGxmrREnSj6bqYvCkWuwE\\\\\\\/VN2ivORrFQJU\\\\\\\/JEA4E\\\\\\\/sJPNo0Q+5Yj0zQ\\\\\\\/YIMk+27TFwCZ1TQITNxDrSzs3vCcwku5bBI+HAB0rLlEtdhh\\\\\\\/WhwgUKVuSldxH8mKEQI6eJCvqftjvwmBxZ5gCD9CoNphICa73dJypywuuO6ac0buQecOBqBwgxBJ8TmPboEbGltvx9U9IT7BIKvCY6zzarMpCMWGaNEnUR24oauE4BYDEn4OScGmD0N4Xj0X40BufBDLhV4w1LJrq3FTGDU6eo2Lwv2gV4m\\\\\\\/Kh2teBP5TOUirjeBnK64bi2bSXlBqWA0LD0LS+xb9cdy5T9RaAbm4xR9ZHZw903E5QRsj83CDDCahOZMd2kTW50P3u8xaIDtGLOvJXjv9KyTm+mqcUBjV+yi50MpHVj+LIvoN4NwJ8TYVpK4tWT+XUAVQGSo7FuvskjWIe\\\\\\\/sw7Z5ppdx35vBNbC7alOrNgtL0D\\\\\\\/YxZ+yvvh2i3BkqYEUez7TzA6ZJJcx\\\\\\\/qjooFIY+Yeg0ZdMT94nVaGpn5A7HQ7A0tYpGwyZtzcAcdjYVhNcnJIQXVB6\\\\\\\/sT7gqK0FcIjSFScPRTcR2JE9YpKz\\\\\\\/5hXtRlrjWMQq227AB9E\\\\\\\/5xtiQnumW4D+NS4xm3tdb4ms260fofYEPdkKvM+2Uqs0MCp69VTboK6y1Yv7p4dQtyo9MzuF2Jrk\\\\\\\/a58SlXCfFt7fAjBswv\\\\\\\/BIX3LMUs2BdgFCeTFGIzLzRryHd4PHN0j9cdy9qjMdJC2sfW26VADnbCy72sAHwsf2XSpHrBcjBGhB0qFwJk\\\\\\\/LpHCPoErAIgKrqujDHQ0Qx4fLtyL0+vrKiv77Xx6scTpSlab0jCNEIJDPawXY9MG1TvbXkHcOloM+dqleGNe9Uf9oU0Svede2C6TEbyV47L8W5cFKnXaNaSbv1FLEs5X2seRZt+Ia8Me+BrJ68HbGi4W+t9ahZB1NVKU6eMftrK04mVAwxPFQeDo3gCeUtvUj0VTE7MsWZhPo0JrGzIIven9txSBm53eq\\\\\\\/+y1v96dFHgIyO0jHeE7wMVnRmx5XLqSDk+NHtg9SdEhg9koPJFoFqYJfsGj5w+UZPthEiWDYPEnrnZszgbHPYB+TSstqsAtzUnwID+fuwM\\\\\\\/xxK0SxDKxzaBD7tHJCm9S1TgF6ARvvQu6EwGhoGxW3411vGBnAH60IZwdfrLagtJxtzjQAVkJCdKqwV\\\\\\\/188MHqamX\\\\\\\/9f+S+owgFhb99fHrHSkZfzCZvMqr\\\\\\\/bY4lYKauhpxPfvzSnqJrUENBYpkE3dvBG1AsJdmgrVUV85WS9FF1JbqI5XqVONSf0omY3vlzGXeLhgamqcw87KdGGoP9WT97JE\\\\\\\/RKVAAWOXbP8tX7jg9fgflC0Ih5fGQ2fnGitzNijxg+zn3GQ0PPHZGWx9Dl+Nn6SaNOw55qwz7wOwZVaF0Sd6qLHDrG3NMFTXQBTKlPk6k4VmsgVbDbzRFR\\\\\\\/jYmh9\\\\\\\/Gj49+xP\\\\\\\/ofJNgyIGdTWlpOZ+MMcyjDgGl4sykZ7+U6PsZ4gB0bovlOsj3IhMv0tWXNDryJXWBvGtFqftykgzpXHJJ355NW8Ev+pM4d3mkbCZ9ba94z9FPNnIc9WoAzF4O6RlJ7SX9rtijnQCpyQxhUyC5aGVBXo9caXxtjYVw5T8aTWyhuwVkOnvOSVym53yJog3fP\\\\\\\/PsMGtHEKRo8ry7O8gbXKTcWgscY3BModwpdMzB4FbglOQh97jYoMIloE+9zpZrdiPnNWKAmVsgSO4emAJmUMcvo3ohzzar7aITh7aKAv\\\\\\\/tcy56KMbpefhc4hcxmNdsvrMs+dGddTRwM+FGTvVeoyYMVOtEPXPk8waoJvu1pciQdX\\\\\\\/rbNqt9jvFulIwM1mKnsQw4f8xoXgus3zAhHSAkPVwx0nnwjBH0zcPeVET23ykoqqCeCcU3\\\\\\\/GtsxrdkyBZo5ZFSb18oD2dabvJjNvodp5r4MpeDTGDuvY8ddF5MOyNCEQ69\\\\\\\/S0PDwYTWlikaAqvRWTD19pfHySoBgoug2m9z7MHDEiNUq4Vb0UtCmr7pYm4+uZcVC4I3FT7r\\\\\\\/WUQCeNijuraeSJJbzQ8Z7QJkyPBkUmgRZD67sdWT6SRTIFASgsQOmf6Ou68so4eLg3Z1eUKZq8wvCKSv30icZuM2Eav0Nvoa7E6xhNd9WCSuhMgt9+5nrysxRPrnQ1guZrshCC2JSRtruCw\\\\\\\/cQb4qLHsCyfL2Ss9\\\\\\\/xSYlGY2Sk3asX+YxLsEx8toLDNGumyLzPoHmPdQ37qaPz7lpq76qEV7NF7C1B4wXVw30rU1VmMnMQBp6msdRXPYw2iFnHl+AUmuHP8yFVmImB2OaZzyoaZvfPXOFF5rnL9div6tgtTuTpkqvHYzVPosHPuhzsGm6lwx6H5HFuGfOOj6q7F4LHIF5V3k7tHEKEIGYGqtBMnjAmBMTdV1kl8P59wjbtdVA7R1VTPUdHWSfAagCzaElis2AIoIZw4aVT6WYTj9xUhTYXoLbTa32PSDGRom4nbFQXzhQKbSaOkzNT01ZfeXaYd1okUMZrBm589tYpegqB6W2yAFGq52ncGQClkxv2tR586W1EHgTTLfWyNX+rTRFFAv8\\\\\\\/sWfaspIgzm7E\\\\\\\/BZE\\\\\\\/utE2H+8gVkHQf5ViwcvtXb1ktDsCJ40wC6zOFU5WSOHtyKsfqgC2WFQEz5VSvCyATG4LWq+0M5gGPqkJDlWxBTlhAoeymLGnU5gooeLxvcxXRaAz+ArxiUnugIvNmkOR2eJQw93MVm21vjUtC4hHiKgbVfg6g\\\\\\\/apt0iDH21BUHmevZr3anKOo+145ptYuLxHYse60To3UPnX+z8qXx98K1CsuBH9LvVk+RCLc2+T+rmsTtf3Syw1uBSCw6WyC69nte0meG5Ncsf2Eilt+PJCXaAVJNxwVpeq2G1rOX1\\\\\\\/ikCBbufzAyWD9OSTmZ7GQQi7dI05XgttKDqAsgvwrCMnEq6sauo6AMMDoOw6Yuj0o1DmoIfh3JdJQdAhd\\\\\\\/2bHwsqmEOQ7MsDDsyl\\\\\\\/y\\\\\\\/2Y56JWOHtq5\\\\\\\/tazuW1\\\\\\\/oTckEAvjSQWKeKP2pleWaSZGaThfQ3xHX\\\\\\\/RM27TqZFMn4F2Y7BfS4kH3Pyv0nLa1086etwDEkHu5lHNUuUOd6s5Vnwfjs2EwDD1GYfxTP2xiTlghGV+w638KvgmxgZjmyspp6UVttUGjlhHcQjrXBOWD7CMy7PZBvTPnLgMBRo\\\\\\\/q6eo0mKZ9BvGNjcpML6dViEUInAVsWeJuieDUmZC9TWAks7yaimsOcaz\\\\\\\/54N4lLj458\\\\\\\/EB57TCEvOSxIevz8+k8z7esyvhVxYHRwtb+CSUFc+BvzAaZGU3n1AaRpqga5ekUXu9Njj9+4q3+yUR\\\\\\\/qQasnYiBf+T+1waM8P1je1muOAUhAQ3dXz2AxXylkIiKUVfR+ZQZ1yhRRaDe0FFitP+BQzM+2Bx\\\\\\\/dPDv8S7BtKhhHkG2EyU5pnH04W\\\\\\\/pC6yQwOYDR4ZiSNlODkluBGKJeucXdgPnaE6imGI\\\\\\\/B0HEzJ7BU+uQFIjOdWhEwmCLLNp\\\\\\\/V+QcSalzHWyO30yTMXvpNKff589N5uOQh7WlhlfvA4y9bvQwopVnQFWNh91Dmelzg0u4QSv2qtXko9bHVw2hrG0du6gtmPPc5Rb3+VEX2nOIPrfl5uPCe4xWiBPaWaycvoBpNqHUU5B0W0YDD9WLxzCE5LeVnghknszibNya+J8Xz8079HjUg+OGsonIA+ze8vqN7uxXiS8FoNuZtCkQofaaOTScGOy+abjpZl9JI7Hba0a3qi6SpWmEp1\\\\\\\/zfXRtjWKDhxAnNOMg3AP8aTvObfHvyUiMx0RlUdYQpaTwdw7C1Wmh9NM+KsXS9oD9ypqEiJjqPiTswjKDmvmTM\\\\\\\/RJJ9aIPLB2+dKrRmCfO\\\\\\\/3U+jNvLz1bBt3v1O1MC1+6OxowqQs2gnwRYcMTQzXDfvwrD0hwwUAzLCzRCD3xu4wz8FKfoe7TFIojukqh5wJEorYCBFFZxoibYEBmgS0wOUD9zAbJ\\\\\\\/T5CyOUZQshJsidv9ne13NdZTSR\\\\\\\/DTpWr2jRaOj\\\\\\\/bZ1l9DuaL2TITSacJgZk6NMjYQ6bw54kYM0Foy07jM1aSdaUJMQ1WvGpYIU6sk0gjMt7rss7SjJYS7Ry0nubMB4YDGhAE84qB4wLgT8FcwuaFubPOMA3x49vWFJJJ6HpHqD6Tr7aZkEzq58nEb62rFPtiVMsbCGzhhPmSvEzL\\\\\\\/MiGku5jSIkj8dA553qzzLtWsF4faeQzxLmBJTQA8x+Dm5g5vdBzcHRFrV1UBeCC67jSn4sq6NuJNiWrC6emdlGayILDVQG672EuRVAjbTbzcCJF3aU+WNnRlOf8qN9kRb7M4FNVHgpttZ4EeDxhd8IOdp3SxVTD78GsiQ0FWulTJEJwN\\\\\\\/okbSBtF7hEqpAPBYOMdTqQnOWjCSVrKP6cZgFBetP36bKzFfSvUxrnC11M47lVeiK2ICKiqfE6rgJEYHsZ\\\\\\\/q3MtO2v1pN7EU1t37R1UPJ2NUBZTrm0ntfQ763inK4QiSfYhWZfRvWKugDIY5qCVURVE5dgGCHQ5DwQ1i1S\\\\\\\/LYhWoJLKEabTQ5czQria24UVmFIi+O65zJjgRzGljAPHRFLPiFOuS1VmqJfZaLvXUxG1JEEU2xWbbzwOG2os9nnSavlxRu+ngLdgfW5LSiGd4KXdVruMQ62EJIyFNmmzz7phAslcBh0RCePJJh7bZESz6bvNvyEaq0\\\\\\\/JmocnRFm0zusgrO4STxKlK\\\\\\\/JA9qlQbkdPv7ehk26lT8db6BMj6ofZ2eveSe8e378Xw7wfljybySue7GzWwGzhlHSWwavKx9SmOLKhSxegE\\\\\\\/RDC\\\\\\\/pjKNuO2YhuJPo1KlDpNlbuRYVgEF3qyewdVpXs1BmO2jofxgXoo6EmGtIKZoSGeCD5Xicuz\\\\\\\/CpF4h26gHgwlJ7LpIZM6VHlkqRe7KoLggalhZJodRCGyKnHJ9snrX5VI6Psp7Kglrdf4aFjsPmVL0emINnGE9IswCl3\\\\\\\/2wUg0nIszmjUVvEozmkUGu\\\\\\\/Xp6fyrTPecCbBHZOAlQGyKnsRON\\\\\\\/aXswfEDaXzW+YE\\\\\\\/R42tmqQH0bJEaxIQqtmCRMmrRtH\\\\\\\/L\\\\\\\/kGQ72ASyHMTJyHCPc6mbMr16A4DD\\\\\\\/WzUYYPJvh5GFQeHRHmt5kGNFfJ8kM2gotEMl0ZM6X1aO0iyVvNrdJeUttIv3XOvk7qi\\\\\\\/1LSU7oPMNNfA3BgI+Wk9aKb3ljIBbxWZ9kY8RNJPrM7cTi72KhCthf3Awy\\\\\\\/gmzGER9fwI1\\\\\\\/EW\\\\\\\/3dHtfYHTm4jDo2jW5YJUXper+Qloz\\\\\\\/WmssADlbd0h3iQIUh2IImajju32HhQ\\\\\\\/Haw+eiyhzRsd\\\\\\\/mU7r7efYVoWQTavHl3dESQgIOoD3cG\\\\\\\/THz6UY7g8iZ7RxDvPwWhbeyc+BSJCP32sir1ZIOKF8FQbn7zpPjRjl9b\\\\\\\/3EZn9HZxOOQWMn+SIfPEniEQpgVwXOVR3S0Eiv\\\\\\\/Nmg6WBn2BrZ8Zcrt2rWzxGeUBCblDkep3GApx2u93F9yRXt5D7GY6n81tPPoRaEzi+iXQpFmZcuCz6fvG2Dvu1W6XipmhV3r73ZSvt5kEgxgM\\\\\\\/1WUihebt9m1UbLHdzK4oBqHSUhRQ5pgHHvJIk9CCtl3wsSwskiOZx2cZ9HTGCixJ\\\\\\\/jFg+S+Unh\\\\\\\/ftF2Ke5GyrnxTK2IwyBX7MRFMRAy2Ds1gTNOs580ST5ZRIpuNmiC\\\\\\\/20Bya4od9K9wWD2QJHSOJU0Rnab1\\\\\\\/N2DlPzqcmMsfkf8mIN30DR6vFjDpxyDLIByKmiSGHAPdmX+\\\\\\\/047LjPfjhfuYg+yFtGVr1No2Egi1+JKzWFjgS1uO4e2AhtAWbb\\\\\\\/XUupvtQ9\\\\\\\/US\\\\\\\/mRJjPemYniAlkSyzZh95BVAaHhT5HhZiYITERuUByAij1VYI3tfjCneGGNhG\\\\\\\/\\\\\\\/P+zrFK4v2HzHajnzOMUPEY2Fw0ZTfRqbjv1\\\\\\\/PxZdBAtoIA+hng3A2NHmt8upIe5uiLYEqvluXY4cFlbr5NkVHOLbfueo3KEr+6uPmmuTvxyuKu+xD6NMLxasU\\\\\\\/FO0GlhgLs2klRX3BizZK9hIOswGTk\\\\\\\/nwAeATo85EaxfVwGzObGO6DFg4XAVzpifXnOjbVchb+YzBS2wkPIIVdvCgXMSgobnv8oeDjuI+kUTF2hq+2zom0bH7+2Rc+JwoF4Bcbg+BJBbB\\\\\\\/n+GwNg+xU9\\\\\\\/2F3jaY9dD2jr1QGgDJgvWMvk6z19F5kCi5rdZe98QxpdSeSgn4rYbgAP+OPcvY1RKcxNxdF6iw2DQMKi3a3r3znXiXujorN\\\\\\\/6NtQn+7\\\\\\\/nM9wZCi+2IDQjwSv3lZ2Gy5UhnOhuF\\\\\\\/mS6d4AqOmXGLxkUY8\\\\\\\/NapFjdCVSV5gvjB2t2TlL6K02aCxVIaYkgogOaG75VLkCnzz+wwh2FUu9Qrva82DyhJojhpKSGDYwjMz\\\\\\\/joR3oklkVTU72Mq+X0B\\\\\\\/kqyPleNqM+KNVdmAMUtQ4cmE95DOwhQuD1uzESA8m\\\\\\\/xoIZxKtEaRf9mQGhwXpNGzq1xJ48oBxyfuLhviM7DYtKU0sJEjDxIqd6O0BYDUC3NVw2x05sazx0p9YXTTDlswWC6gt5Tp2HhZBeU8OV3VuSOz+yknrRo9wDoS0B4HN9V+7wuv5NyWZJAdCx+\\\\\\\/ApwAFiv5xaJmnNeNxSKrjsJRo5kD3uIY7aEUPmYbVpYk+iCkASb1k0\\\\\\\/kqrWxPpXoy+rPh87Yk+70qnqrFNTabko7B0jQ+146TlV9RbH8sCCLvJ18A7qa\\\\\\\/V5tigy0T4rUk5XT6rOGlyzY5IWsTQb6VWnNI5diFO03nx34OHV8EQ0pC40OzSwNi775MtJ3njCEgaI1GNCy2xI9+STX0TOwJIKk6PEYcJEUFJKDunecjGA6FwqtJfp84CpSABT0cLm6Pe+h+nBuA3nEZGaSygFZUvrUpkwbs7fra5ggRxs0KaMn9YIsCWPZA1OU5r3hQMIbNgZexjGnlih6x5MdZGdlFhfdUd8gBm1fPmm1s+TImM+iLn4ToO1zwe2Y14pYCNznwdqo\\\\\\\/GkGPF34oo+xtJLwaQ5Dtgaa+QCGV\\\\\\\/ZRlFmkeKVnUWAMinsAbf6kBWAAKsqAApEJeSdUuhzyE5tWy+twtZedLg\\\\\\\/KYIRpEecthkqATDNvc2o\\\\\\\/w3mullRdAMntnsDs21DPkPvyRASjmhxkXYRSIO9nyMijxr+eLf+98eajRBNDPTGV9mHJ9kwT5w3uJlnnLJKg8Blyh5EZS9S8fPO0JgU+ZA76WSZEmUmQinC6LkNTZB1iHvUoSumZFcaF\\\\\\\/US8U+gPS7D5HSdR5Iw6yh44xU1tAZZ0AipIAgjdhQikond8831xkhLadHNYVd3jhjTJyMPcjLfPpkpPrviGuYYBsiK+bMcbpRowjV99MDGcAw9TnvPOSpIvgyVNLQSOo3kDQnGAkvGD+C5R11DdgspVWxWnPEn0WC4oeUJ1r8DPWKZp7nprvo+AyLPx9Uyj3yXE0igUKplhAc3DUgI1dPtsM26X6Qbiz5XuORTdjAsGQ56B+lhzXdRhTzQnnBEMpRc3bsB2PjKuWk5jcATyVR3isrhZoCLgGgSsUH2rP7OrFZANFDM83vIjpj4+LVwqVJ\\\\\\\/1Dj7zFiHx+fm7bEMqfr32NhCTtxigkCEy7QzvZ7nBh7R68apf6LCsvwPCeOgdi1sb3FbPOBUiJC8sbfmySteMNM6pdLYqo1F+IwcbF8nyIIbLPDC5o6NF911fdGuCW0vKUK\\\\\\\/m4d6bGxD1QpbfeAxJb2RwS0N7xZ910EwRXk\\\\\\\/1CeztosI4Urqsrdf8+hYgfcQRTPy5keGF+BWQo1flBdPlKCm\\\\\\\/0Gz6MjbIOYZGw0sq\\\\\\\/TUo3ThCwo3qaw+oOeY52gS30DtPeW395WzI448XiTGAxfPSUmVw4kE0iwHNyoRNQsd0tVd7BpsQeVL3i5sd6G616XtTMiUq\\\\\\\/Dq475mRENu0dsPmXQqUzxGEjYepmvbIa\\\\\\\/sEvHsw4ewZo1n4i106bz6PdVGCX4DD0u2Co9rhMEecHFi+b6uwIyA6x\\\\\\\/gZRPImUQzI9ksQcu+rXVhsQ3xVACLc1G5UYCgZHnSUMxlqYPrPMOXAbGnsFSbWFsqGqUFcUQitJo1Sw\\\\\\\/cSB2Vq\\\\\\\/GWBfalC2ZQxpDVsa0b+98DkzijWCKiqc5PzegDP+E7LP7pAAgJldkdrAlK4lLFNoiJDACKNNaj6TLfeC2\\\\\\\/\\\\\\\/\\\\\\\/cToopgv7R9QfwAmAR5JjjYzQRXfAy9YxcwmEmbx8HZCI6E6cJQ5t\\\\\\\/EE0MThmiFRSoAepWFt+PVDdnq\\\\\\\/Ins4JZ1BZQyHbPe4ODILRrtf3WMJmlVH+uWid77jCZNDB7Fze2q1PjcL\\\\\\\/uzdf4Ws4mj8rz18oZ\\\\\\\/zth7BDyh3WnERWkTNDtA7CBn6xjgbTJAAXrDgJho4xXtmgwJQKAgDH0HsIT3lZQkSxxsz3znDyE52b+sOaWe90UXDWZEgst6LtPR9Lq3eK7fJGWKz7UNg6zyVe5Lh4X4OW7kRWpl4txtUrrybhs9X\\\\\\\/T2R9NgVJhf0p7DcPn5ZbRBrneklTwPEkUpeJBxYG\\\\\\\/I50hyScqoauaJ99FJFkpW8du9tvya5AwBV7JLuamWkSKjq6jdESETO3nwwP+9k34krM+1MrIrsjhznQgqszWLIgsLUHhF2Sw1H6QY+0ZwfWU5SO1Cg\\\\\\\/nS+vDYhjt+VJe0bzOdHUAdyF8PVgDbvpFR82bnLBBsEJBUCQOwcuxM9RBEqasDY4dw8qjdKZNMOt58vnBNWUS3OU\\\\\\\/QA590P5ej4YIri2ou+8LDlTt9fWjql6APxjX9CGXtI4Z5R33nhaRKaqscCUwyVrxdHRx1+BOb7m7CKlIQuBf75XHk8NGDgVzM98LSiyi0sft7n8LGzt+ep2jlwzgQMVvfU6vYwXSC\\\\\\\/XxkR71qhpcvmb+pAaYsK8Up5Z6DaFtAxtO83b0w\\\\\\\/mXYVpXggVmnJiFYBiUkWNFdHptHoBuuUHB1LSvUZ968VHNl8TztDIGAYtRFe0l6zGEH0K++XzdjBQEIMKohaczj65O\\\\\\\/ezd2OOBN33RmMkqcLVPbSyF77eDgieSbmSGEfaImfUWWXEGhDU9Uyny+7GrEEi12zcPyPmka2hFaONuuFOyCtqX6gD0YbnVjCcpWaXvWQFtVisgFDPHqTPkPrRLaLGbBkQMEb8lFizc36X1EHOoyV6+7q5hvJ6pRNfB5HN9D5+YCklpvM8bduMXrWun3zu3Rr\\\\\\\/Y3\\\\\\\/MHPO4zQmAlP4p63T4IC9eTmN9ntDCerZJ4Hv4qKUvFHRvfSADC3dpZMQlhEpvnaV+bacdFoC4AFIr0Khb1ACvq7cp1D\\\\\\\/uZnfxIxoRtM8Z2HfFRawiqOdKk48VibTjDn2uYLgBiHCfjLJdGvch8AonUBKWZoWbBeRs6LmXJJ4HZgzIlrXP+KjKI2h+RXA5i2ZKayfTx54huJqOtm0cl\\\\\\\/dyl8hDCccgFgxx0+LFsOia3rZWaw\\\\\\\/WbUsJYfFRhXeDc7KCFGeN\\\\\\\/6iiMPQ0McnejCIZ3naIRSkM2XVBhW0Rk33GjRGGU9HSYBtmEfj7cLwCo4hM4CGdWz+woO6iE0Q8k4VKQ+HBcHHwDg1aMluw7tAWlNRnez0WE27uILcE1TsC0XdWrgiJUGLiaKLV9tgalkmPYYYewaEoTjNvYRqfxtD18AEWnPt2IS6v5eUy2U4n6cyysyKyCMt4A2QKUjyS+fPtTdksoqificFR9Xn0V9tzxW9TjtE9Ff3JlbpAQIQdd0+eZzItkIf9qEPAA3cB0AzvUzOnjiCtE5wf42D2ul7FKvFPqeDvgfy9CSvNN8pwNV\\\\\\\/RubvDnJINvYnqXB0wlvgCgU7jtdBFJPnA8l8ph3mxgNTDD2dQjlq8dgfqkdM7+3h\\\\\\\/1Aff6TV8NvKTgjFcsWkXvSbaSg+rizjtrjxioHVRvkIFMM+zuaLCnD4QpGkhm\\\\\\\/LcjXI7pf4inDCCVfepfQT\\\\\\\/HxyHf01zmbreqgzFMAmDzLQDaxMG0VCMmfet0exkmOK1vHsxyjMNDdoMw0J6xUaupQX\\\\\\\/4gFrWHSIJijWplANAXqEuZEQyydJ+0fbxibf1rRH\\\\\\\/X5hybZPVBwItlB68fhl+e+6irj9TnYCePs4MKrMFnLR5qILgG2ZwWr236vM4kGdbblO2Ms96EGb\\\\\\\/j41I\\\\\\\/giWzuLMvUQ1d4xj84ygwBzv0bu2uIQJxrd0nQ07mFgPPDFdj5SkLkrBtiMMFPmuCwbvXsUQANXaV63YDynhWus\\\\\\\/VA7vm5S76B1TwmlBlw0AU8RhJZOkyXMSGTaEQSNQh2h5e5F8JIF5WBC1NGJ32iacT4\\\\\\\/ZuuPfGqmL3vUq3ms0QS+cQuSp0WFaAwLEWWL8Ak9tGvCcPUynDpnhf\\\\\\\/kHNDDRQZFvK5hXiqya25VQOBErI9BWXemcQCUT\\\\\\\/eOyLkVGvT4zgxAC1rMCpI8LiF9XNxUyN5+A5zQ33Dewp6MInAuXvpeQ8qVnUxqz6MY3hbUDtjq9IdFD9IeoFOGhESR4FTe\\\\\\\/nUXj2nNZ82bg+4siBqEFOB17Mc705POJ1cnUWxFlcoU\\\\\\\/M27VAy\\\\\\\/d9ecZ0dHN4QVY70h8SVtBp1FtAnW53JhPl2vU0\\\\\\\/FF+t\\\\\\\/osVhzRxUsouLnKD1V\\\\\\\/+fS61s+y14xSld9wZWEZkvzK3iqISXn77GXVE1YUKTQPpOHjKkXtQYMfDuD7E+gMqOuZ5tILLQThyXQRzcGpsCawVJFE5slEEMYKWpYPwIHAW7ddbaAPMD93sTcPUoY943S8GG8FOR2AMAVdCqeZYBAPpcxqmrlYiLFujLu81e4nTwzFhOwJGI5gEW7ylxPE4q4d7vrGf9lQhFBjmIrz7uc9w3ipnADiF6VSrVqjsqYFbP9\\\\\\\/7wr7boZj5n9J6O4wfJllhd+6R3ruz+3+W9LmBd8ZsQgscuJnQNhBvxI\\\\\\\/mPVXWDvYlISr4\\\\\\\/KnKMt6iOHB2c\\\\\\\/JR9GkYbyEtnlaTfgZRJsyEXxuYlcJiqeTchxjntY977mRD7UI7EtzT3IRGBU1YM\\\\\\\/JCV592rkaYoMIBiJpVaDiXZ079bJPPYSfMqGxlNdFsaclWX+SBFcqQYX9vk9NorGdOfdO6z0lPgbOqFFSuHWOclgE6dxM7GhA30IefVZkKWiTfLrZHK8cKYzqW6d2XulIpnPrdwn6ioqel1abzxF0dIGfMpSG8a1s5rAzPB\\\\\\\/GzNgTSwnnkK+3L2kZdPAIqugVIycLN+MZaoJBPnmdsjjASivNczCYefiL6i397CEBL\\\\\\\/DeSghY0JGljgc8PQ\\\\\\\/bfEaQd+ejqbC93ZNSa0TjYu0pertg0w9GL232O8+85mpymIT60NaFIQ\\\\\\\/zUahH7qq8yjmzRwbUDLmZgCiP5gSeXJJQyt8J+0fD974bcmdPb\\\\\\\/ojfP7JIRpxJAKgQO8PztZeUJCm1NID6SlRkTjj5zURnwrKj4FjhhUz3xakBtTns9IER+8Af6cedoVV7GU6Fc1igFwMV9KxVy\\\\\\\/qBtUY2SbGQamTMPjLt8Z2U8FXHYFvXiuuypRoITbFSEQmYLWZX+tcs8SpEGzxxK8UvRNy05z2IsH8Iph+skkvfmCv8B5nniXE1PL3MH5nOQUIcB+hMUTo2PgeOvjP3OeCB23lmbEm\\\\\\\/\\\\\\\/0fRPd5xpyWbohOu6AGEhbKd4vBJwD2BxHUeHZeKKFsyKrFt1EZD+KrzhJ0\\\\\\\/n2sHrhSFm\\\\\\\/DQSM7lz4pPKTS7zw9f\\\\\\\/QMpBYq+eDJjtchXxg0JCuFHZbmNqDqm9LhmhE5J3q6pC3rXjh7wNHrsDuavNd+aDaAuuwfxtGtUxMaFpJt1YHmTRxrjYjqR8eGwVuI7sLrZNr6Gi4ufhYrGqMZgHERZ2NnjiMLnExyto\\\\\\\/x8diiznUmXbKks73JKp0mc\\\\\\\/fY\\\\\\\/KZ85SHjqQJW9ViSuAqkN582W8DKhFxnON\\\\\\\/UwBgevAQiygDOfIBz2sfX4pTNUazmx4noShIWb7PTmn37umtfKfu5mI0TztI5vRRZBpHfH6PVrMJ\\\\\\\/Sz85wFh1Hw8hP29xPkTxfgqLIrzyQ32UHgiC7qCp5UdrrPB9z6nptbJdZk\\\\\\\/wrNj6W9vG8cLx\\\\\\\/VV4GlFphCbExpaSs2oSO0iS8VoRQ44JnHyfQCFeIJZvRaipPp\\\\\\\/IVQqhBINL+\\\\\\\/0UcCy0jyRthtPDEVsjQqKSsT+zwX\\\\\\\/ZawicSiW3z2ifcU6ccjbR1u+2ZLebOowtScrqjJ0RaWOSwVEhe4g\\\\\\\/vpC1ZrGKLJ2oxl6UMFqwfeXqc3KK7fvAqdeII4Sx+bYauCaR0hIH4uk5e\\\\\\\/A4l5GN7knTCT4oGtEffEuWPGkmcg1fV5WKiNkJJpxrfGcH19VTA6PdZxHttCegYUu06RHyOxlHpkeoVFLHK4T0o7+1HhLBk8769T1uo5IAN+36o\\\\\\\/8OkHmIgSOVEZL59i3Mbulidt79FTbROnu88ufiKIUYe9nK5E1Zg7FDABbJhPasxV5dAtu9ZeZt3G2h6256Jpxg9\\\\\\\/nL5aNv+bRikXd6fYqKxaH8xsQvJdR5sbTkQnpAIfcN2XmifkU0EvKohdbM8hLi64LrsW6rKUfHFNcI7tacUvarnjuxRqCC6BzMV7R5OQXUdELZK4TlsXKb8n2pEJ\\\\\\\/wPfZKeFmsZxs\\\\\\\/Ry6Aoel7tK624aZcVFlIzR\\\\\\\/V9dzmA5TIVqYB9y7gy81curor5Quz46KefnfgfNmZnL2hdP1uTb23mCYtkENGIzJGD70hCmJC32x2ScZOMTfrVmrXXUFAjuUHCZazIwBqWUvVApCLbiumsnyOWfy8uGRAa+DUOLWwOJwDaC7Kc91HyAZB+GxxADZuAUmzylQey7GPrCsvySt8goch9OEiU9uWIZ4K5lUDvsp4nB\\\\\\\/20hXqZCFdMGzdu4rM0ndVWOmsED+QbE7brl4S3HrfWS6SXlS82aQgLvEMtoU8Eiw6m+Wi6GUBtDFEM8rI529M0jzeWfT7uDZbZkp1+DDTJqUHqgyqMhENEikW3N\\\\\\\/y3eQGFU7JkTYDycKO+L4qQgfR5KiiPrb6ytg\\\\\\\/g3Zx6ZCgFJITKSg1GuqUB09I5MBrJA090jkc2CKZwN0f+nQgiKiQv8fyazVcEWYEgrBc6ad7P1XpjHfVlsieXSq1wB96WV+XzyRmr0DYsvW1JRZAWtiKmEU00iUMmBRKGkJgBETPPzNJXYiJA0hIRjXtabW\\\\\\\/tIiqqQUOXehgYwQwxupiXvybTc5RKZZ5Z4Peqj0b4sevIHMApsVCoD9GGqgTzXG1pb\\\\\\\/iXjCto0A8AnaiKd3LkMHkv0I6QDyAZVdTCDtFikJoHIJKfRgHjSXyq2NDM89VT+t6KyaD4nn1bHSSh0c52lKy6Tg3ykkXBxzLBZDhKIHzgxI0hgrOs+AZLibJKydVIj6VjULJ6224DN1IPFW\\\\\\\/8haOQbKc++q\\\\\\\/y6Vd2LsgY2vBtyizbh2DiXbKSqIC4VUNrtY\\\\\\\/18rIkjLXqRH+O5GZCZDkPfNar\\\\\\\/XFcJKPzlGI4sxYwR9hBsJ+w9\\\\\\\/v2samhm\\\\\\\/t5RMFI6EQbSnQVblL\\\\\\\/KehPQLQ6cR2WFZhfYtsJU+akeRjzLULBPdcyHV6+c4u+Y2+u9il5hfcYaWhjCD1oT7UVnBoN0TTYiMYBD6FmMhgOSHHHoXFtezdP5c\\\\\\\/9CclvPQUwcgCkdun0j0MeW+uWVU54tdl6jqX4pdYDkXg2a2hTL0ZM47Q\\\\\\\/ibam+cPeQOhtSKU+hGN\\\\\\\/uTMALoI+vUf7zNDmq30LkV4w+pzkMkG3wCV5\\\\\\\/3JuCX1WcY3fZU5uJaH3Y1wjLIezzOXJvPvQe9d3NUKwsj7VIJYztjvUeJXn6xOk0TyD46xIWVKZtW6d583Fl5xYd9HlJaBlHCw1L2UxcURC16uAl91531AkAlytn8Mkm0rsgkvVA0XNgsd5ZAUc5Uldz4XS213EFBkvNjePaESX4\\\\\\\/BzyD+KXnjqLSeApnhfsnqOWXv4BHaGC0If9kbvmnCBWBeMYekXveeghfgIj74WsvHbQ6WUUrjqbO8Gy9eJOZu0EQLTdCBIc8scIg0oCcOH\\\\\\\/vfsvvQLsCZf3jstwApqylA\\\\\\\/gTzOXwSSHFmiZbxNgjdQGot+\\\\\\\/IS8zs0+gxEdf2ZT4IvbATvSUES\\\\\\\/ORpk0nSmmhPlzrFFKaux\\\\\\\/8TJMjxGUYQV30F6gje1qPuVULOw2\\\\\\\/qnaOVFagyw1N\\\\\\\/Cldi3RA5xGZLuaGb2nr\\\\\\\/YxC3K01iw4+0cKc9fC8n5v7AkjJQ7cjKUbW5oNHFQFnNL9aZBQjNRKTK7YWQg1cum76RzuaMkC6Ch1md9+F9uhJQFN8pMal02WJnkGJ4IBYDaJcAKNH7fFZvPcMQOAfc1uDvkSN3JHc0Bv3YdVLsr50GpQqO7OYICH5qD7kSj7pk3jQdwVFKhDj7BWPM2KQ3frm8AFTbWt8JbC5bkxilFYatXUaMBMG34t\\\\\\\/kgkwGCWuPy+YAHP9w6JVaHPN0kg5PwamByC3NiWgm9gShjH+ik6Pf97gzlnmUTvzNxwDvh5aa6jfNUsyKtviTY8+zkUFuU4Jl0i7Fci6oku61D58OS8XdF3JV8GDsrRedhxP+i3odohwjwwSQ\\\\\\\/xUme9OPBAGXAaZqJqeDmOYGFJDYzKwQyvmU9LK30XBlL7BRwZYFwBRocqFwWTjbvfxEWiQOOvDINWaawnAWOy12eVU4ZYdMGhR4dvsvLpKy6jIuQ\\\\\\\/kpXjMFJOsU3Pp5SaFGHN6GwIr+A1c1BsdgK\\\\\\\/ruLsRENZyb9L1pYCmavaDckF1hA0IFw39vWXxE5MoMEidA66GEoDnwWOtmYRiSci9vOfurZFI900t\\\\\\\/P9wgKWlRHFHil9I4n898nJQ5TDFRFNRSklPJl\\\\\\\/kXiRzmzMiQV4MAYBhjMlXMw0nusJU2uNu6ewkPpmGaCuCWCAHqBPaQM5wDdPW2HMVP+7NHie6JXWa1GE96F6qRnEOIF8NuiwuirPJxe4vog1KGAnAJcF7k8xn7DVJ7Y5XSSKj5g76dd8WY5YPPf9JFPqGRGuMUrlG224FsSmVxC6zz0\\\\\\\/9C2Ppv63yl3N7rUMsCkOH+1bzFGzL\\\\\\\/Cyta6HEcGbDvs2HtUp5Jfg1TUmgRIaftoep3mjKP96VPP9UBarzn\\\\\\\/Zrq8q7jYB84+G1g+WiAFltDDjHTItPOIW0SrQNdBHbJS5QT\\\\\\\/lSuu\\\\\\\/ld3YFy8yH0R3AEO48ZMtAOjyLOxieZ5QQJCcacu\\\\\\\/pXsvwvGcD1ZJXtvjsVZ7SEwosCtk44DYYRnvx9LKyVXWksPTZyTCxkx9fbnTZyBj0UTEhMwCNCiwOdABwersVj4\\\\\\\/3hQsHYyw4eYI4d5rVXCwZNHEhVk8kJ5NBLn2LhlN\\\\\\\/hEHhl18Sg0AxEXOBc6O5KYFJnMJdI2UwJk7R3OktwxOf0ASecS0awd8q+3BwEPOxbKKlF3V6sSkOcfhmMt1IIE444OH89LvoKwyvHSzSUQO0NOVv9YqceBwfrkTpWweBpSSx5AfgBLwltSgbDPDTlkAS\\\\\\\/w6GFLOjxkDk257gv50iCdScyfG957+k0Hsf55jWkfZLJw3inT+PJRry63OS92YZDtkpQjcEILM89HHtHI56nxIGWQgnqHatrHmHLAUAt9TR5B2d\\\\\\\/RVJpAwSXqH9Mw12PvDmWXyS8g8vlDgsIlEst\\\\\\\/AEw2VW2NPoRD8QsdE74I1N+A8SRxociat76Jx5OkHsctQwudQzfw7Ik2RE79ZbYdN0TytdlN8BLp07q48Lv53YvDPwaAUhv7ddt0FjvdyVt2PR92k+Qy0POO8JECrLgpH6UsZRvCrKT14hknArnkYRRhii5eJCmuDlOgwMsDy4uiKLmU4qRzFSYDb35WHn379yH4pSH9j3tiKfsOyVG8qrxwPpySeDSeKV1o+RPRCHRMwe1AJkjCiIyF3fpyYJ8HOrHeOL7XGsbp6DY+ovL+CHQXpbsDulKgvMXXsUOOjMOT0WZbH5uGTn\\\\\\\/eGR48lkxCPXKIF0Hjh2eyuAJwHXpjI7gP4ywxEMFiFyyQTYb8nVFG64JvrAIgZt7zcByaI+GqVD\\\\\\\/Ei9LRpfKbH9fPXBbqqFemLcHAimfzwLxhwzEynaEMVQCyhHmG6lIDBziLVHmtPCVvmIm4YBHRRaezJCfGiM18Ouf2b6Mb7KjRTV6cc19mnKvUl9IuwTSKmnWriynsItsfJyE5av72krWHsFGxFFnGUSHwm7t4OaKj+L5fD7EyGlfpwFD7VpnINOT8rxXsIK5ojXVMZj1brL8rtuGA6kisuu3JRK9mQ2q0VmjrSuZ+su4pMkTO5lJPYGazmPPBjY+N7rfgfenP4pOD4hWIvgvjCx5SFWaa1qulG1sF8TYr1RTr5KQuUadKOsPRjO0XdQXykP+Uyu8Ub\\\\\\\/Vp65mbaWlsQmyYkSH6IFT5ZVsLPl2eHVDnGkxStkA2gzhG\\\\\\\/370rSqRm814yZn\\\\\\\/v8O47CPtJqkqm0kNV9ffjO90msZD9CKG4KUHr4UWYrcONF9+JxivLuG9Jys7pr2RqlU2Nxrx8ISZfejfRESHa4Jjenqla3cPBIolfrwtHToJHkgStZgxTojtPJvq2\\\\\\\/cb2SY2oxFRd\\\\\\\/67+sx20Bc4SARFGku2oLS740lSe8ctodqjWj82HOggOc8VKiTj6JQpPGuRh8sUIUUpNuCotkvPDINH3GyNT8GwavPG\\\\\\\/XFbJ72wy7bvnzNy7pNUt+j3dQGHZhV9jONSTigpFF2W+MFEmptkHSl4PmWgRV2jjgpAGNMdjVnrL5ZST8EAvvzcSfpdppwZZ+Jdx4BcMj0Wh5hAu+u2I8NCnqsKTni4IpoRpmkwO\\\\\\\/deKtG2JzS6MpjnxLtVn9LoAMHdxVbtsAoaHzmKX6wnmynQ4DP5npEraXOw0Q74BjngMu6lVLxtg4SyNUpu80wvcknurjMjnEg+LuneTcgDaRTR19F5KF26oFT2LB\\\\\\\/RnVqISw+jH9e3oyUEArijCB7humf20e5zkOhDoeWwjxUvyIX8a8wfqwP7rNSTHzM+XXLpPZE\\\\\\\/rXFRVaZsxNmurhF\\\\\\\/mmsTYSUuSQLdSEZH1Eobh0xl2k6h2Gt97CzoRD1Jsx+WsRnzmKACZCu6Qwjfc\\\\\\\/KvOp5pHvrXuGLX5TNQBHNWD9a67FQzUnX1eWN4qE7oogrtn6uavYQ7L+Qd6yvgykJiMWlOj+O8nWIvRjcHcS47j5686KtlkE52s6g9ikNC6SBGGm3+vqSlG\\\\\\\/8Ek2zLyDQMdMPPROTgvVrbbLmQChHkJuzteCsUKCwzio4IwEamNZe9xGF0NFAZBSCqyg+snMLnWdAo80r2PVY7Cbw8pt6GU4HqMBS+L8Wgd8Ceu3UCXrnHWcgk0KesnsGRhuHq79VDDB\\\\\\\/6m5YsUaHap2LmjqSOulxDBt6JSCGMI\\\\\\\/zLn+gjQPhlxd0YCWN4Ydeq\\\\\\\/LFKrw+kzK6csS7DUn\\\\\\\/8TnQm0knhFVzoRMTjJuUP4ywQpx1Q7n31we7du+nlnqeoGjFmV9PZ55fWRPMA\\\\\\\/2jS2zJDWVUWOoDW48ZXsTdBLX5ap0tGFgadBkbXNbqD99da3rZmnmjBpwEIJkVXw638wqBFcFEp6aiCQTs1Jg3\\\\\\\/Nku7ZsF2iKDqo5Wq6uwOvDpKxQdbf2PHCnuHp3bN06ajnnwfWsSQ1MiYE35Uf9iX8Eml5dMD5sMttpdwBNnJyWxeHXo3RZCL7a\\\\\\\/vR0mVNFUbg5iIbkGajL6rK4sjqFaig3aRYujjPRfS3hjjdhMPH2yY+3aJm5ihz\\\\\\\/ChS\\\\\\\/\\\\\\\/AcvzQ1A\\\\\\\/xKQWw\\\\\\\/hCqpwEQDzaeum4hodTDXfCGkI2pLVWL1I5+wqG8\\\\\\\/gozu6cJPB+r1fLQXO+C\\\\\\\/wLSpwv4Z0DjaUxyE+KY0DYSu9KbXYRxXLZJzU7Y01RFMN+4qUdhOZlb2ykHxpvd7RjKk1pv9bWTEtWIzXDfS8fqsbrj5QdeW0\\\\\\\/KF59qBvPYnZMLxZ0ZNuUY8Ogqa7T6hW\\\\\\\/Ohh6aLlSvnNjGZsF3AV3ZN3tTd6s2UJ+dsEApgbcCxfCI9ehcuBfvjEETN0vBUueHMp2s1mf92KWDrOVQBrLFgzYQUG6urs05IYlSCJL\\\\\\\/Tcmb7Joaifny13jIE+RpvEprWZ9cHMGCDR9CgaIxmWbOBNuiGowwT3w\\\\\\\/J1sk5KkIfQ85367t4IUYSWMPKAH\\\\\\\/x\\\\\\\/Ap3F1oW2Qxk4TFaGY7ICEw7\\\\\\\/RZ5cxM16VbWmnBgtuP4uINjvZU8tpD2KQZPx3JRrnS0RrJk3Ik7YA+W4xADeaS6jCbLuHTkucCNm5zlPXlvDZ1tsuxIaoMHvqcVSA0iKVT41aaSp5lKukY+4pidbxaSf\\\\\\\/ICnJ7Is5F04QCybLqBudFO3WcmYNl9hv\\\\\\\/IEuLugcNZ1SW63XwMSkSCIpwnOx6dbP1MmUwfdysBO2aoPDzLKI3iBKqWt9Eg1rxhmKGPDs7qhntdLlW8GK8MHRS3Se9PaKl\\\\\\\/pr7Hdoor0sjgRJSIDJWgM8y1FUh0\\\\\\\/Avekxh88vpRLn7GfrWix\\\\\\\/83ZP2dcS49bySa1AgNujMeqoMqTAP6s2XZ3Db0Ommm4yzJiYDqMZCGs5B1b8sFsXYhivCZdNXX1HhjWffNc6SdpHx3pvhiL0CTQwFt+x9N7ICb\\\\\\\/BDJsmEip6Ki8tcM8KemKuUfSaKs8sZfb3EEIMuVUYw+lS7EhYKHCmNFY6ZEsUmpyFnC10v3qYvb\\\\\\\/UWkO6rKMdwkOJ0v79mt1ztzYAJPzi16D1LyTtdzdu3aRoD9Dru0dFBRn7H4dumLmcc4SfagW6eS4s+b0rvuGupOEcCgMmvujnCBpkQYg0S9d1z7xg5pcy9vgPHs8WBfBXTqY+bHS1qT9TXVQrXEQzJ5k8Kxeg3E75SmTDjr6FtkN2m0FYna0y9pwAf2l40xkm9Xs8FPcNhjRzoEAfnl0Oh4KDw5shj+a7qmMZxDhxoFBqwEEQb6nIp184FZgFotDZ3e1vmmPyEeBEqhVVSPYweasqj1zzUuhWk\\\\\\\/WSmCk2tMVkkeoJbfOP5jPcWJ5IFku5Gsq8WOSvJmQ85gT\\\\\\\/XWfCIgtcLo48Ev+\\\\\\\/w4+DgFd1Ldr15e02LBagDLNp6Ae4h3fKHDZ92lF5pYTQHjS\\\\\\\/PIGK4c5114K4zFm1NWUaJvlVM4hYEY9I\\\\\\\/FPsrHJPd2wccXeNl513kOt3VtFFbgjFfpuN3diedBiY6ijxt97nAzfo4OyBQo2RSDm7vsdsOnzrRz8\\\\\\\/F\\\\\\\/lZnguN+b6tL4exQR6z2muHHfknR9GLJAtQyKZuQRQSYA\\\\\\\/GPXvBWyjIt+wMEa3bt1+kr9yiozkxDBXoDbiXNZdiSuY4VYbhA08\\\\\\\/4JXiJD0iNkzh+aClMgyf2j7tPyQybM6SbkIqfByIJB2\\\\\\\/+hd6BTqZZsx+Bbkr\\\\\\\/HDEmhx0K8SNXwcYb7EOkVNb7KvJSy6aaQpJbpydF6SNOIG5dcgININHHAqNdQNbncbtXxSMF5hb+rAWCuXRR63B3LUNQxuhNyhZ\\\\\\\/kQSwedYv2zI0QYPJ\\\\\\\/qQyfRC+YNoKERv+rCR\\\\\\\/w3P6MNiziHqIRVUFimhR3d6SV9FD8GHRUjdGKD2G3BQKbz1XVJIxWSaPcZGk1P9LTPDD10AFB3OabMmn8029TnjOBMDCW+pw2Gv2j+t+OI3MMKkSrTyznpWFleTR77AjRDEBhMB+t9LgmP8GqMGCSYRtaVQO4\\\\\\\/Qc\\\\\\\/vz5pF8AiyyK6zRf\\\\\\\/k\\\\\\\/CSdnPpVMr7hYKiM83d4hduXjV1VkQSeZS84bbhorhtSpk1ofK1faV4YuGb9b6Pzhn0X\\\\\\\/huuDTOJTFIVd6qksNkfQ\\\\\\\/1KFCjAzRI6zN715bwot4k2Sx6yUfQSBW4UqImkU\\\\\\\/8bsnrbY5QNKVEGuC0NRdXN\\\\\\\/N40o0UgUCydRatpnpSYA2VzW8Yj7bR7kk8SUhAvIe2aeZX6ByQX4fBRhsc8uG6oVRI5BVke2KtL2j9ftB\\\\\\\/\\\\\\\/oCe6od4Mkij10MonRBVHiVvg9h1pYGmg6lI2DVsvYJzdYVyB8UcosH5qiHbRZ4C0i6WUuh0c7mHcLqQDV2zAoqJgsDP7nXLpSs2KNEoZVaqYGXVn+MwBFJvjlUeSfINsAsFj8Ijw\\\\\\\/vtT15nOedehmPXAIg9AgPojy1\\\\\\\/SedGkWraLvxg8fg0hQHd1raEp7OoBGyxzhar4k6B1xtdgjWzdCRrEqJ3advCSXOhgEg1KJpln8nkC2GSacD5h5K+GFPSr8TO8YKKBKGOr8L0+FHp3SxPMLwOFOlgOoNIH9QKGNv1Di0wVYNFK4LGFfoEU4p5FukGCBT7+KFbbuZFeWqWJTEdP\\\\\\\/kFaUcsgtoH4t+NaFs76xNSWh\\\\\\\/WUBy5Tvj6es\\\\\\\/oOpX+kE291huELCpWyuH\\\\\\\/iWs+HOqX\\\\\\\/OlByBcCckq5UM8wpQpD\\\\\\\/S5izUkGKgKQI8FexDlhvjVDRtqvRaIPQscKazAwQjT1\\\\\\\/vxKh3eC8AfJW1EXJVKsWDglGtfbi64SgfsjnC7aeeTw6sMRsAbXeHJGeJ2AtGbypglCiwTFsntLlgMD4TYZU8uG0I9q8S7yqptQXVpwXaPSuUlHmTcZ+nDoud9BiTA5LIG+t\\\\\\\/di0HT2M\\\\\\\/V3YlgYaStqByILU8E1IKIFygCEVqVJk1wDjH9wUiN8sRcI+qUuqUT8cFA01CQdfYT1uqAl8uktUlUDXctM0NJ1soI7bJrGE5n5mehNcjYTmVn\\\\\\\/bs2ay6H2n0qVX+6Teo8o0fh4rmfUoJEATxzGrL7fvX4FEJvP4LkTywGsnMS1S3e3iXE1TBg0vsAXGIKrfJZkWW334l+RNVtlVICiO7lGrM6w3PthgnNH0Y+iaYqzX8wssdFt+d\\\\\\\/pl\\\\\\\/fw8g24ouymupTI5rip7V2ojlelcJMMsvqwcPO4mxtGv6iu25ApJ7CBR1z5G1+kbqomnOOMLgwnnOkcLgaBam\\\\\\\/Hzt++d\\\\\\\/+Rfuu9+Z4JLXqDC4WlJI\\\\\\\/TdKDauzaaWC865hpX7g42ivdOV8I4Slk01w53oPsRWlaAHxmpupMUxTHzjx786UAeWjdC3nm3dP3XLJZLoLfxPijJmxUB\\\\\\\/PSSXwHC4+8Vif2lBwPBFEaMsY2X4r2mTaFJdX3cq697KMbzbfuOqExC8XtPiy1KFjGNYCyO+NrNIGTPmj6QsFmi6HZ3GYSf68eIYy1tAsIAc71sdZz2mF9g\\\\\\\/m3Oso0Pyskxuxy7ji7iEb4Z0TRfg1ywFVsiKgsisSQ65+MG2BUYkSK7fAebDDTYtPgLIdjxaVwyh5bRtK2Fc2WaoSf7qgHOiXyTG\\\\\\\/DpQ6Ob3a77kh8HY+cMpdVnxUM+SBZULOqrwfT2NXKcvYehXeAy98sxHX9wgJVqD3\\\\\\\/jGgEVA1eTok\\\\\\\/HIb2FOEywbdFcYizizVEc7siNqZx5e8EAatq7CTyagUAD12hqfmWLlPIghdHHMs2u8TC+aC8jCbYy7vUk+40glIMgqestEv6ULUE6\\\\\\\/sHH85BPoUgbx0L7zbgVEbtmBzoY8YvXg4K6uXBdk\\\\\\\/mPGge7m7CtKd2YMhwUBR5Ns++ieqYIZMcFA7XX74ZpEQzpbgTrQTB77FGdDYMlk0c6yaA0utBS86gRTrolfgZg5QfdxIqzik7SR1kYdUsd0TJDigHtsHfsyEYBk2vbvbfkqXJlWYCBKZ07PoapmaJJEN7eRlUPNkw6\\\\\\\/wO3agMLonEOLz8chnmD7egG3WI7Ux7KBW+BJKH2atIiYt6muzUKMSv\\\\\\\/0Bhh\\\\\\\/DLQii2r5YyuO\\\\\\\/x6COYr0np4yWYBqdVz+B15TEeHnxqe1YTPCJvaWTdy2cnnN62l4FKjLyucOqkgb+jc+TnOXKNbioThB2TDj9p43LmRsjyqpSEXILeSjjWtl1xD3sdI+uEwA352iTkkJnn2GROT2Jb9UqGSrMbHnyTaJXwLyVUdk32qGcGQGQ\\\\\\\/ftuPDsVs5RaedQ7a+\\\\\\\/qUvjF5PwIqWkVeBAA\\\\\\\/csPunLrvcCmd4JfAC++QI7P3wtse6kvl8fpci1Yhe7rBcX7vs\\\\\\\/gWzOzhd+igWDyYtDITyw9u0rLQaSg6eXwqyNXkQM24EEkPBS3UKPVf0fmjJLj8XLj2XMKzW3wNWfRTTlAg2\\\\\\\/DQOq2ed8s6y88Km5ZXMN6rcL1iMOFYd+tWsCLh3\\\\\\\/Dl53gmbSl\\\\\\\/jjZ3Ibl+96eem+HSCvniwA5j6sXpnr9fYirunHACF3JF88Oh\\\\\\\/wePeum+tbHYfhDlHIB1EFwEl2V8Rrw0DFDyeHPEgHh0yEdw+h5U8mD5v0kcx0YoqlvrSUBa+uT6EUWNYezEydlJ9BM6b2d8M7tByc0sXTa01\\\\\\\/6DSpzPCdeRZ30ApcaVM9cV7xhbARFeOxSCc\\\\\\\/iHwVsrHps434\\\\\\\/daPms\\\\\\\/lrPmGjOP0e+Xg2nhy8x\\\\\\\/Zi7jAJT8YpPqAfSu3l8NtNkp5J+IdYPShBVLAKxXT1jzrLTzAAUE2bAzBxHIxnFTOk2gQWq9tzHGtzjkVvCnwVwbFoME\\\\\\\/1V6UWndgPjubcnl4q6CqFtlXF6KGPp74M4f2yOyf5qETa45j++\\\\\\\/tblfZGU3nf1xs86zYmf0ejQ3OMcqICjAx4KjGOiShrt8BGCQ4a1evqql8FCz1P7EGoog2WDcMWDLXLmR6xttGCs3PyxGHt\\\\\\\/c17fyTMwALqnG4m4G7TLniMCO\\\\\\\/NC+hfNWzrxr2Fb7DxcyHSyC0Yoei97Ag+CzBVGHScAkYOqpJE2AHzXeh\\\\\\\/lDEDFX9NDkI4c3fU397vIJXOkrHT91Z1uLJQDHdgliEYOV0IzrCin658NVFLOp\\\\\\\/xnrqRSHV9Xf1Dybsz5tvSiVjBZ7RWVjXXoXuaKx3i4OaftE06OXV4h4UzpInFb9YIjTOLWRKqp2PqUcTMDy7+XGiLYWhTA1S5\\\\\\\/4IXxwtAc4EDFGhZuOzZnG1j7Gb+kyZJz0Fjnmhv7FBG\\\\\\\/Zwdmq0t85V+m8r3xG9jT75+yNQeciSZWEdzSiRttk+wJZbKXk34jOH6WaTHuEJpcOroPx2uv7zlhbxoF9EmKnMFNYPWvBf4Pr9cLgG7HjFIr5scgX9Vx\\\\\\\/eF39J\\\\\\\/RSo+O\\\\\\\/MeDPXOmaUDHccHNX6h3q6LyIbiGL7xgvFGCIt6TzhKhHh\\\\\\\/Ncs+WOZ3Y8+k8d3oYPOTaETm81LgHCljKlm0ILMbz4St9kX5IMI7u\\\\\\\/pnYkUlkopM97aaEzCuE6wgiIOaNSyqVsSn8CcPddkzgCyQoz3eIMI\\\\\\\/bpKBSuR3YvtA5zcQFwzJb+woEpON8t1NbqapbQDMrU1\\\\\\\/oEmpdfTjfXpKXrMoZrv6Xwn5KlRHOH5eUKvKHGg19cQOCZI9hKY3JZZp7q1rHfw6r1A6zy2a1pCbZy1FLo9cvtRD752jgg8eHPjfo\\\\\\\/OpHn8JGRr1bnfp2URJpyRSHm8P3pu6UJGUrSu5MpMvixvOITtq6q79CCvSyXciicd\\\\\\\/S8QF2BBcV\\\\\\\/isAB9VA2BHLkm+RimNn8dv0jHyGJJIBMbwIEgZ1MDRkLJt+9T4RNPg6T+B8klS4VNHD8fUP5GsuT7fkV3WqcGuVDbgD9oRYJccn16PQTxUiTrrvZL20kbT3TAbTDduWj9unLpPJDNfdIa9PeQhv5\\\\\\\/P42BZOuJhTZbuKoJct4xTvBH7tgVkl\\\\\\\/+xtU7vNe\\\\\\\/xIGtNV16yA2XvPrSaVIahx+cwOo4\\\\\\\/9DmoGsMlPYE6UJ0O07vtrqErdqv3UNYqzKu9JJ81wdPqtCwujrgi67iljmvFjsk3W\\\\\\\/+9ttkL0zIlN+guwqukwMbzH9TS5TI+uSbcSxhMrRgYWFd++jKxqXePHP4Z5iqTTx5bhJJd9MI2pkKXuxCHnTozguBa\\\\\\\/OG1ZFex3wkiFD2brQidy1nqvzui2mTpRou3wzEVX72qPeclnJKa7UZKC5v0JvH5F+CjxNy3rGg0mDTUqa+65KDXgb0LK8YJG0RGun9gtLT7+w4qQ78e47roLS8FEUWTMsZVIYfyVqPR2Bt\\\\\\\/ReMBoJsfhEAnCasWTozomAzIWEtPCQedLhIom6FlNWsY3yo5+rzt1Px+CUst\\\\\\\/gg+JUa0djdOxZYqYgrb\\\\\\\/6FWWb3C0xsHJpKBLdh+nZC9Hcw9HWszFRySZuFBeEk4toZ3+spsrh2ZohcHCcwPwoed9ISMQcicEmdIb+3FbMLG0dwsQRJRp4AYfTBly6hhcwx6L+5OD9C0948icyViX+3Ldh9BMPwkl1oVqya4v3ZL+RHq9qRLwLXucLNkLX2OrmWGC08hSkc\\\\\\\/Lmx\\\\\\\/ZKxEti7kCG0FzlEUBaQ1zjJpAAkAEb6wCi76sjAk8xqtqZsUvqm5a88vqDvAF1xuIbjnkHt\\\\\\\/1di\\\\\\\/MsSH7Jd0jCGqfFwD+wbRQtFOueEQPpwfaKM1dU5umGj+XGDRJJDzxlhs\\\\\\\/eXdxRuU\\\\\\\/nyedWC8YM39AT4PZm\\\\\\\/PN\\\\\\\/4Qf7koOjfHlfj7EBXgpH7E0Yfx\\\\\\\/EY92rTjYuFmhOZSrnEUzmxqnsmRKrOpvt7DeshnCECKI6JYNDtZuxEpHLJR1i7N1OXySvVv9Lgtk7xBsAy8UArTllcf\\\\\\\/04ZVFjJkFPqxmG6w3+valMrrpPRMDbLPszTvCzHzv7cNqeU+Rpr2d\\\\\\\/wwZpNNK\\\\\\\/RatzXfM\\\\\\\/JMkpIjnNetys7+HXLlk2Vq\\\\\\\/dfZbYD7JZ770ku6W\\\\\\\/FI\\\\\\\/t9WRRE7VDXIM6gR6kiY5g9WC13UXHG1sokXyvlBMdekI6E3zfFYNqUZReOJZCIzwvwjDFmfWrea+wTlThhu9ejmXO\\\\\\\/Go3SR53Zj1JYCkyEwBisaxvk8IyMBxLPjr21av1ZJ9GAYsYNt4CD17nLDy1A+XAaiKpLcLyas60Ae3QFrGC86JaC9QUxh5FRNy57EWoXIr6rYb7HYpMJeD4\\\\\\\/TBiOtNLGVbDJnRRGhiW3Yvh14RLgskM7S2jBrhKaHJQZICDyaD2U\\\\\\\/jZeC2E5NjplwMKtFFqIsF2LZHB4t7fEFDLmF2j6sKICQIY8Ad83ZmjEno6cpDn4T7M1fX8oXSHQKXjoLYUeiH0UpC3EtUfd1Tn9tpTwa2X0\\\\\\\/VsCzM0mm0ms0eQI8DL+yWUW+DH60hfHrs4ECoOWRTSueDnFlmi2AZgz6T6xMMCN1WJhSC7JRTMerkXiN1ZwJE5qG2SJt+L1dS\\\\\\\/XSM0AvQVL\\\\\\\/4q6F0chUU8wdVMYVBB8rrsH7YGwdGX2bzBcz3Lpq4fWheN4XOqASI2NzZYke\\\\\\\/T96PxgT62e4Km\\\\\\\/k5xLUdsq34aDZGyasfP8Q\\\\\\\/9d38ULx00VbbSllO7eV6HjMUpkCNj1b1hY0q\\\\\\\/wiO6IcmqAq2dfm1HeEJO4lLISdbNPgw0HvdexqOcaWmdveB\\\\\\\/9cFuT8+KyyPxDO4eo581Z7sk7Xg4euTHS+G85Q6s71qob8T1+Y4uT2Lub9ydbSqil6vsRZF2oiYjWoae+lUIFtea4CwhT8iX\\\\\\\/7\\\\\\\/IY6W0OEqgKl5hb45pPd75qaQkjDOPo1HsaDrA6nYtsgssHy6plB8MS6tiARhCSaChoZ3l9fCm8nuSMFY9M5E5ta06CV1NpZV6p3l69Zv0vDn1CclELt2iM5D7Z80UWYX94C\\\\\\\/BIADeWzEO0uU3qi692lzjRT8OZ3I0pqrEOxXYKLe6vQTbHgFuK3Asfdrmtx27y5O2pPluBE4H6oEh7mVSJ5Oh57sEZLG2aybPF8p5QZpzi58g9ZqnDWvCIKsHtT22o4cb\\\\\\\/LQi\\\\\\\/t0GUPu\\\\\\\/k1D54PZzpmAqB2sCpFnstBto5Y5L5kprVgbxwB8bmOA+DAvPH1V+pZfEZkh8UCcC0a7vmk0GjaJBKJ9TQ7fYtlOYJbZ91NIffRJYcpSmh2AjU8rBiE3pmK\\\\\\\/7VM\\\\\\\/0Oh+BY81fVYTyEIq+ohJKbpwatwfFuDZjf3Mm3\\\\\\\/HG16sJ2RJAD6izqKV8F4uwN7AtF4EGBG11ZyxjfV82xWyWg5fej26FVHgMTHmPZr7WiQTgP8PBbB1aRkqIIDLiNeL70PQaoFKLGf+mmK7kHvDuR9r0lw1z1yKtlzSIcc+unekH16cp1O83PvsSUzNFuhAm4aw3aZjIbhLxA7JPrsyl8SIsNV0ff2\\\\\\\/18X5crrS\\\\\\\/Dc9ShUn6CdX60gxVq\\\\\\\/7FKCgcQ+RRmJxTtHaKSTxQsvjvdrl+\\\\\\\/uoA0i3W+eNFVnf635vWK3jUkfCTNelf+7\\\\\\\/uwsc5JLCuzq72\\\\\\\/pQfwJr6ZC0aGMW10HF1I0nwOB86nSvFIp7cse3Yb6GW4PO2OSkRtWRQw0BQ9VtIz\\\\\\\/5X9kQF+4sjoWQJS39LYECHOTJM7HFAymXcmH4UytEAfpLOmhITXKGZMlmdlNReYCqpeZx78Ab7EVPAVxNYtE5nOrN3g54W\\\\\\\/o6c351Nv85xsIR6NbjkBqnnLJl25n0xhEtVKKxuNIgkf0+rn0kjpo3YYYraJNBDHR8zgDh\\\\\\\/gScK6SXDYAnXuTQrNW7UPjmcY\\\\\\\/TirEC2MxWRiYzPGu\\\\\\\/Ze3HAufZruR6eialvWfCJgqs4d7aXg\\\\\\\/5acvbrwxcO3cEWygQs\\\\\\\/0xHQIHUhFXpColXCWTaLi6RHEWjhL4wnjpd5uS5kJ\\\\\\\/jsbd+I4reZEhVllwtqJ74ID7AdXiOHrSj3UO5pYm8h8RSZcA5U8MJSlQHD8i6nRpKj9Vmv7bEUKGw6qpxRQvgnvzZ6rSG9i2FaPNVmkyin2e6dVpW\\\\\\\/i30Mc35mGyZFVwMUiKT4vUT4l5ltbSxsqjpPf\\\\\\\/Pu9pI+oLbuxZ1X2VS3VVoimDi1aMeHmxPsQI9VSiURox2+u3kikqcmeT9SMPMq0UxYv8Swkv2AINXFB7zepIULwz522zAlsbNmG0EsBJC8Ea4g0tv9CjkBv107RLGORKQ7jm\\\\\\\/F0G\\\\\\\/5AUcisAvExdBgzN0wLneer0hwK1+IIL+8b\\\\\\\/jU+3hyihW46Wpg7gmsmAKBYz9bvQtLp71mMiUIag54TbfsBZ2Z4JHkRGKVRIQgfRiosFYDqZCjxFIVaZTRMkyPXUB6+LsYpzRWVCfZUURzftonG\\\\\\\/6B8NEMqxnzTQrWpngA+1e3RKjUlWYWroYmnDeM9OUzEGiBUsSJW06KABryPiEI6c4kmMfpkXAYCtlTTQ0fSNC3WL0OGEsCE1tsuPhytBlMufjZyCXGdKTowJQJYY3Te7dBm1EWSLZ2YYbuYXTMNqizYPN\\\\\\\/CmATdTGyBHZXiTIWjdFZjWOJccpcbpovvrhmKBRF6Y4g05pQ9l5V3aIYHzqzWwt4qoKo2r2BkErMQm9R7ZFCC3rTzE9tl8PC+VWmrd+xXOQiMG\\\\\\\/Yos3AW0TJKQZS9+rX76dzDckGksrqsdZjTKMcWWBDtQgAcGdAwhQQamGJW7f3lZm6abjo7Ls6bo3x74AH41esokGz+zfhGwVUZhcoSsrvi6CUwL6IBwM\\\\\\\/vNG5CynsC4pBDuAmFd71A9QI6mVPByYkvqAiUHq+k9DR6ky00wAK68CuojhoAZw6i4gIOntY3G5vgyphTVJkqOlkFzF24uqcjqAwy0qljbzq673ts7NljKBOSnRdGPQ454DsOsxyGMKSmyx1ZoFgfqhvwarHHPUBYd\\\\\\\/lN4wT\\\\\\\/zTtPu3WgyvfV+urhNeESwTSn27GOlSTev3BP0ANjSMqQmNOdrmq1VEHPow8plK9lacW5y6AJarAnA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lXtdwKVXU5\\\\\\\/oFq5rXCLP5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221efecc9f385635df2924c42f2b4fd67ba7777c0c241cd8b20da6bab962da9d26\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002253KNPTK0yqxWS+nKcDmUPIbzRH5acDKjS217pYXf8KmRulldd6fyfO\\\\\\\/cl2c0r5+sxf12O2Bnydbkzz9ozZCy4w21Dgiv5\\\\\\\/kC15ybwzFKSOdApS6WTNr4gJABnudYwwhlNttIxRlR81Mowgx924BcGaDqzfIBJtw7RU3XMOMHM6lz4Mg9Af5UO7N8Fip8EZ2w8bUU2uo3TpzrvV\\\\\\\/YfuziHxrHeVhPKWozqdrhX\\\\\\\/qog+K9bMDv+tYjoJ9fcpHY2tE6I\\\\\\\/KXaBzxyQps0s+QY91zbTQjlLxtcbSPMAh0PyoFWMCTa34g41dbKKsbgyelBdhtseg9cnO4zVtTQernK9wd6H3MAeuKhyHwgiWejhst22w\\\\\\\/MX1MWIpEJEyvxb9v5dAR97wMW1YRZIjmgr0Qj3+4\\\\\\\/mzE10EmFAbUQQxSJ9oX5UhSa2dXA\\\\\\\/+tPJYAzcni9v9k3GiffKyU6DZ\\\\\\\/qH+gWiAO+WMKJuld2NDaTLJUUKAXaFQ84BwZATETLdDZ3hK6jsouLzOE6xs7ayFEYACCTvexG4uKGycGrcCDKo4ogJ0hA7fKiOuqpSLRNShs6l2v1rTyyGGoJojExq4Knq3W+IpawTsD5iq+JdVpHICDIx1QDpksUf61LiMNkVE8KkoZkjk3Svfg759tenpNBY1CI+eTkXbt8sgG0aS1paE6TBGnSkArckhreb47gtAjqGibuN1XNAM7aq7SuATe7iSf9YypaKvk4Tf5ZoXJFUvxamPCi+nous8WQk945i3Vyc5lrw55+mYpCojFwDhChRBbk3kazwJWim+0H3b6YacD2SKL1yANP2RfGQg1Q31n1WWM1wuIzU9s4sgpVmsBY5IiEVjwj80UEEtq4pmI\\\\\\\/hz4NcEOI5w2PJutTpMf30kjKBnn+avgcEm0aI5Hf7zHmTydvcJ0fTGtSkS6VpofT6HLHfxoeQYcQvln5sjEOHAmvJr5N\\\\\\\/q+kML9RLhj52f+kKXBX5gZ6es6WQf29JchbUlGba1F8FMyr0wSc7oQ1JH9mm3cf8A6eb4nN4by3U7D0I5wwJF0a5tV8\\\\\\\/KRH+635WKKqN\\\\\\\/CEzHPRk7oJvX1zAEPEkBm2uwKLmHk\\\\\\\/H5GafXiHkyUuhHsuXi1Sxd0Xx+CFbJPmMILd7dI5IrXoj9S6\\\\\\\/YY3YdJ0c5RRyGsbU42oo9MhcSwo+PXmDeimQ9XcM\\\\\\\/C30U9rCOfjqjWvDeEwdRdoz3Og3vvZlKnNMU6ZcummeyFLXE72a7m0E01bQPtTuWWm9pWvljgym6btVxsG0ikjFl3BjqCAhNPaFD+ShkcpdtBRz\\\\\\\/w6c4glSqRGj3ABMpYAnGxRoinOBlejDiqWQhdf5Ts0xegnfxm75FVoFhQP2NLnfbnhiZWsG7eSqNHLfajSVwj8IAUJuqiuMztu9qjEX9wrey1+4X+PIUzUQMUQCwIZc2u2sf2x4kvmLo39m\\\\\\\/gkdPDLOM5yG1iK9v655u5lTuTn7lgwFyYhnudd9kwxgFTILIdx9hnKeGpK0JPZ5iaIgkJNDnHGQgS8A9fvjio90Eab7GT3tPztLt2jUsTXGgwud+Xd9FkYJxzzrA\\\\\\\/3IXibFywurlIQfRM+8xUVkyFwv2Y6Ae2E0hTZ1c+gb3wAt3BgNX\\\\\\\/jikQ03dVmbzcinPkW0ZYiek+OEQJSAa+ANI0Av\\\\\\\/InJAI+OdYXXZprY5RYzEQDek1KSBXWU+Nq6ccLMyxg\\\\\\\/4hCMLwaPvfOA90NSlpYFaHLaMs0jVptFQypTTEDpe7GE0EyGQ6sfTi3A9Q9TzHqWQkgW7W+3+586X+riVBmwOKbmqjOGxt6M8isz\\\\\\\/1h61g2oZ\\\\\\\/h2Wf+gtDB8Maq1HtB7yQbVJAWFrPnviUSwmEKzvqslMj7yX5Kg40azIj00WraQ4RcPzy0vQcnIaVgMnMhkUnlaJjLYobvM8VVhIf5jw5K2EoC\\\\\\\/AKOP5S1nKYkVKEhqQBvZwnrOGaeZueff823YdzJyr6+MNnrx8BX1icn\\\\\\\/XuSEiGX5OfMrqK0Y7S51Z1rBhy9i3jMg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rQOly4CMLHnTrjrYpIsyJQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cb7117f6a15512a4567b78eddfe80679067257b0f3dcec9a71b858bcf69599b8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459487 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1069" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1X8UVELOk6H8LXkb-k9ofgcL60XyQdLegg9KdX3Ufr8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1069" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i-4hZDq3qZZiQE--KYrr2aJYm0AxUlqHVoCpeCDqiM8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1061" title="Genetic Variants That Confer Resistance to Malaria Are Associated with Red Blood Cell Traits in African-Americans: An Electronic Medical Record-based Genome-Wide Association Study" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1085" title="Using Genome-Wide Association Analysis to Characterize Environmental Sensitivity of Milk Traits in Dairy Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459487" id="node459487" data-pisa="ggg;3/7/1069" data-pisa-master="ggg;g3.113.005769" data-apath="/ggg/3/7/1069.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, <em>Strongylocentrotus purpuratus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kord M. Kober</span> and <span class="highwire-citation-author" data-delta="1">Grant H. Pogson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1069-1083;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005769 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459487" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kord M. Kober </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiological Nursing, University of California, San Francisco, California 94143</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kord%2BM.%2BKober%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kober%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKord%2BM.%2BKober%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kord.kober@nursing.ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">kord.kober@nursing.ucsf.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Grant H. Pogson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BH.%2BPogson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pogson%20GH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrant%2BH.%2BPogson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/7/1069" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459487" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459487/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1069.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459487" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459487/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1069.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459487" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459487/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/7/1069.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459487" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459487/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/7/1069.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459487" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459487/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Codon usage bias has been documented in a wide diversity of species, but the relative contributions of mutational bias and various forms of natural selection remain unclear. Here, we describe for the first time genome-wide patterns of codon bias at 4623 genes in the purple sea urchin, <em>Strongylocentrotus purpuratus</em>. Preferred codons were identified at 18 amino acids that exclusively used G or C at third positions, which contrasted with the strong AT bias of the genome (overall GC content is 36.9%). The GC content of third positions and coding regions exhibited significant correlations with the magnitude of codon bias. In contrast, the GC content of introns and flanking regions was indistinguishable from the genome-wide background, which suggested a limited contribution of mutational bias to synonymous codon usage. Five distinct clusters of genes were identified that had significantly different synonymous codon usage patterns. A significant correlation was observed between codon bias and mRNA expression supporting translational selection, but this relationship was driven by only one highly biased cluster that represented only 8.6% of all genes. In all five clusters preferred codons were evolutionarily conserved to a similar degree despite differences in their synonymous codon usage distributions and magnitude of codon bias. The third positions of preferred codons in two codon usage groups also paired significantly more often in stems than in loops of mRNA secondary structure predictions, which suggested that codon bias might also affect mRNA stability. Our results suggest that mutational bias has played a minor role in determining codon bias in <em>S. purpuratus</em> and that preferred codon usage may be heterogeneous across different genes and subject to different forms of natural selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/codon-bias" class="hw-term hw-article-keyword hw-article-keyword-codon-bias" rel="nofollow">codon bias</a></li><li class="kwd"><a href="/keyword/translational-selection" class="hw-term hw-article-keyword hw-article-keyword-translational-selection" rel="nofollow">translational selection</a></li><li class="kwd"><a href="/keyword/sea-urchin" class="hw-term hw-article-keyword hw-article-keyword-sea-urchin" rel="nofollow">sea urchin</a></li><li class="kwd"><a href="/keyword/antagonistic-pleiotropy" class="hw-term hw-article-keyword hw-article-keyword-antagonistic-pleiotropy" rel="nofollow">antagonistic pleiotropy</a></li><li class="kwd"><a href="/keyword/mrna-stability" class="hw-term hw-article-keyword hw-article-keyword-mrna-stability" rel="nofollow">mRNA stability</a></li><li class="kwd"><a href="/keyword/mutational-bias" class="hw-term hw-article-keyword hw-article-keyword-mutational-bias" rel="nofollow">mutational bias</a></li></ul><p id="p-4">The nonrandom usage of synonymous codons (codon bias) has been observed across all domains of life and varies in magnitude among closely related species and among genes within a genome (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Grantham <em>et al.</em> 1980</a>). The identity of the most frequently used synonymous codons within a genome may be similar or variable among loci. This cause of this variation has been attributed to mutational bias, random genetic drift, and natural selection, but the relative contributions of each remain unclear (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sharp and Li 1986</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bulmer 1991</a>). Because synonymous codon substitutions do not change protein sequences, it has long been suggested that such changes are neutral and thus reflect of nonadaptive processes (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">King and Jukes 1969</a>). Indeed, in broad taxonomic studies examining genome-wide patterns of codon bias researchers generally have found mutation processes may play a major role across broad domains of life (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chen <em>et al.</em> 2004</a>). However, the variation available in the degeneracy of the genetic code and the multistep nature of transcription through translation may provide subtle opportunities on which natural selection may act (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Clarke 1970</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sharp <em>et al.</em> 1995</a>), and many researchers have detected subtle forces of selection at play (reviewed by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Duret 2002</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Plotkin and Kudla 2011</a>).</p><p id="p-5">The possibility that natural selection could affect the rate of polypeptide synthesis through the matching of transfer RNA abundance and codon usage is not new (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zuckerkandl and Pauling 1965</a>). The <em>in vivo</em> rate and accuracy of translation are affected by the use or disuse of “major” codons in <em>Escherichia coli</em> (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Tuller <em>et al.</em> 2010</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Navon and Pilpel 2011</a>), and early observations of genome-wide synonymous codon usage in <em>D. melanogaster</em> and <em>C. elegans</em> showed a bias toward a subset of codons (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Shields <em>et al.</em> 1988</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Stenico <em>et al.</em> 1994</a>). This set of preferred codons corresponds to the most abundant tRNAs in the cell and, by proxy, the number of tRNA gene copies in the genome (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Moriyama and Powell 1997</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Duret 2000</a>). Subtle effects on fitness may be realized by a decrease in the cost of protein synthesis (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Sharp <em>et al.</em> 1995</a>) or by an increase in growth rate (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andersson and Kurland 1990</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kudla <em>et al.</em> 2009</a>). Accordingly, highly transcribed genes have been found to be the most biased loci in both <em>Drosophila</em> and yeast (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Duret and Mouchiroud 1999</a>). Furthermore, in <em>Drosophila</em> the magnitude of codon bias is accentuated at genes expressed during the high growth rate larval stage (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Powell <em>et al.</em> 1993</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Vicario <em>et al.</em> 2008</a>).</p><p id="p-6">Early studies on the nature of the synonymous codon usage in yeast identified two relatively homogeneous groups of genes with different patterns of synonymous codon preference (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sharp <em>et al.</em> 1986</a>). These different patterns proved to be biologically significant because they differentiated highly and lowly expressed genes. The selective advantages of translational efficiency and accuracy are clear and might explain the greater codon bias of highly expressed genes sharing one pattern of codon usage. However, it is unclear what selective processes, if any, might influence the persistence of the translationally suboptimal synonymous codons of the other group. It is possible that selection might influence the frequency of both major and minor codons in some taxa (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kreitman and Antezana 2000</a>). This antagonistic pleiotropy hypothesis posits that minor codon frequencies may be driven by conflicting adaptive forces acting in competition on synonymous codons (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Akashi and Eyre-Walker 1998</a>). According to this hypothesis, major codons reflect translationally optimal codons, whereas minor codons represent translationally suboptimal codons. Translationally suboptimal codons, however, are still subject to other forms of selection. Different forms of natural selection on synonymous codons may thus compete for the synonymous state of the same codon.</p><p id="p-7">Natural selection and mutation affect base composition and DNA sequence variation at the level of the genome, the gene, and at individual sites within genes. Genome-wide studies are capable of identifying major effectors of codon bias but the focus on major and minor codons may miss more subtle forms of natural selection. Identifying patterns of synonymous codon usage has proven useful in identifying group of genes likely under translational selection (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Sharp <em>et al.</em> 1986</a>). However, more than two clusters of synonymous codon usage patterns have been recently identified in both <em>E. coli</em> and <em>Bacillus subtilus</em>, and only one cluster in each species was associated with translational selection (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> 2006</a>). These results are intriguing because they suggest that different patterns of synonymous codon usage might exist among different groups of genes and that multiple forms of selection (or mutation) may be acting on the different groups. Before this study, it is unclear whether multiple clusters of codon usage groups occur outside of bacteria and yeast.</p><p id="p-8">Previous work on codon bias in metazoans has been performed primarily in the protostomes such as <em>Drosophila</em> and <em>C. elegans</em>. In deuterostomes, codon bias has been studied in vertebrates such as <em>Xenopus</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Musto <em>et al.</em> 2001</a>) and <em>Gallus</em> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rao <em>et al.</em> 2011</a>), as well as in mammals (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chamary <em>et al.</em> 2006</a>). However, there are a large number of nonchordate deuterostome species with large effective population sizes that could enable natural selection to act effectively on the subtle selection coefficients associated with synonymous codon usage (<em>i.e.</em>, <em>N<sub>e</sub>s</em> ≈ 1). Here, we investigate the patterns of synonymous codon usage in the purple sea urchin, <em>Strongylocentrotus purpuratus</em>. The purple sea urchin is an excellent candidate nonvertebrate deuterostome species in which to study codon bias. It is a widely distributed, broadcast spawner with a free swimming larval period of 2–3 months (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Strathmann 1978</a>) that results in a large effective population size, extensive levels of nucleotide polymorphism, minimal population structuring, and a well-supported phylogeny (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Britten <em>et al.</em> 1978</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Palumbi and Wilson 1990</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Addison and Pogson 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kober and Bernardi 2013</a>). <em>S. purpuratus</em> has been studied as a model organism for early development for more than a century and a half (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Pederson 2006</a>), has a well-annotated genome assembly and transcriptome (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Sodergren <em>et al.</em> 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cameron <em>et al.</em> 2009</a>), and is phylogenetically positioned to provide insight into vertebrate model systems (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cameron and Davidson 2007</a>). Furthermore, the majority of its predicted ~23,300 genes reside in regions with a narrow range of GC between 35 and 39% (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Sodergren <em>et al.</em> 2006</a>), thus minimizing the contribution of variable mutational bias across the genome to patterns of codon bias.</p><p id="p-9">The objectives of the present study were to (1) investigate the presence of codon bias and the identity of preferred codons in the <em>S. purpuratus</em> genome, (2) assess the presence of distinct clusters of codon usage, and (3) examine the relative contributions of translational selection and selection on mRNA stability on the observed patterns of codon bias. We describe for the first time the existence of codon bias a marine invertebrate and show how mutational bias is an unlikely explanation for the nonrandom usage of synonymous codons. We document the existence of five distinct clusters of genes with different codon usage patterns and show how translational selection and selection for mRNA secondary structure differentially affect subsets of the codon usage groups. Finally, we identify the presence of five codon usage clusters in <em>Drosophila melanogaster</em> and show how translational selection is acting upon one group (as in <em>S. purpuratus</em>), suggesting the wide generality of these patterns.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3><em>S. purpuratus</em> genomics</h3><p id="p-10"><em>S. purpuratus</em> genomic sequences were obtained from the Baylor College of Medicine (<a href="http://www.hgsc.bcm.tmc.edu/ftp-archive/Spurpuratus/fasta/Spur_v2.1/">http://www.hgsc.bcm.tmc.edu/ftp-archive/Spurpuratus/fasta/Spur_v2.1/</a>). The gene models (<em>i.e.</em>, genomic coordinates) of the Official Gene Set (OGS), GLEAN3_b-2.1-revised.gff and coding sequence annotations were obtained from the Sea Urchin Genome Database SpBase.org (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cameron <em>et al.</em> 2009</a>). Coding sequence was obtained from the genomic sequence using the OGS gene models. Estimates of mRNA expression levels for early developmental stages (2, 15, 30, 48, and 72 hr) for all <em>S. purpuratus</em> genes were downloaded from the National Institute of Dental and Craniofacial Research (<a href="http://urchin.nidcr.nih.gov/blast/exp.html">http://urchin.nidcr.nih.gov/blast/exp.html</a>). Following <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wei <em>et al.</em> (2006)</a>, we defined high expression as having an AU &gt;25,000 (Arbitrary Units) and used the maximum expression observed across all five stages for all analyses.</p><p id="p-11">Although the genome assembly is a high-quality draft and the OGS a well-annotated set of gene models, we applied a series of filters to minimize errors in our gene models. Each manually annotated gene prediction in the OGS underwent basic validation using CodonW (<a href="http://codonw.sourceforge.net/">http://codonw.sourceforge.net/</a>). Sequences identified by CodonW to have nonrecognized start codons, nontranslatable codons, partial last codons, and internal stop codons were discarded. Gene predictions without annotation data in SpBase (or annotated as “duplicates”), or as having excessive repeats or lacking sufficient information (<em>i.e.</em>, having insufficient degrees of freedom) as identified by SCUMBLE (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kloster and Tang 2008</a>) also were excluded. Of the remaining gene models, those with maximum expression scores less than 200 AU were removed. Finally, genes with identical expression scores for all time periods were treated as unhandled duplicates and those with the smallest codon bias scores (Wright’s Nc; see the section <em>Genome-wide patterns of codon bias</em>) were retained. These filters left 4623 unique gene models for the study. Gene ontology (GO) terms (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harris <em>et al.</em> 2004</a>) for 4232 of the 4623 genes were annotated using blastx interrogation of the SWISSPROT database (v. b2g.may10) with Blast2GO v. 2.4.9 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Conesa <em>et al.</em> 2005</a>). Fisher’s exact test with false discovery rate multiple testing corrections were used to assess enrichment of GO terms associated with genes in each group relative to entire set.</p><p id="p-12">Secondarily filtered counts of tRNA genes predicted by tRNAscan- SE (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lowe and Eddy 1997</a>) were obtained from the UCSC tRNA lab as a conservative set (<a href="http://gtrnadb.ucsc.edu/Spurp/Spurp-stats.html">http://gtrnadb.ucsc.edu/Spurp/Spurp-stats.html</a>). Amino acids identified by a single tRNA gene were excluded from the analysis. tRNA gene species counts were used as proxies for tRNA gene concentrations. General wobble rules were used to identify the synonymous codons recognized by isoaccepting tRNA genes. Following <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ikemura (1985)</a>, the frequency of use for tRNA genes with anticodons recognizing a single codon was estimated from the observed frequency of synonymous codon usage. For tRNA genes with anticodons identifying multiple codons, their frequency of use was the sum of the observed frequencies of each synonymous codon (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Ikemura 1985</a>). The frequencies of tRNA gene copy numbers and synonymous codons for each tRNA species were normalized to 1.0 using the greatest frequency observed. We tested for relationships between frequencies of the isoaccepting tRNA gene copy numbers and the usage of their corresponding synonymous codons. Correlations were determined using the cor() function in the R statistical package (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ihaka and Gentleman 1996</a>) for the entire dataset and separately for each codon usage group (see the section <em>Clustering genes by codon usage distributions</em> below).</p><p id="p-13">Data manipulations were performed using the Jim Kent’s source tools (<a href="http://hgwdev.cse.ucsc.edu/~kent/src">http://hgwdev.cse.ucsc.edu/~kent/src</a>), the UCSC Sea Urchin Genome Browser strPur2 (Karolchik <em>et al.</em> 2007), BioPerl (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Stajich <em>et al.</em> 2002</a>), and BioPython (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cock <em>et al.</em> 2009</a>). Comparative alignments and <em>dS</em> estimates for 2954 genes <em>Allocentrotus fragilis</em> and <em>S. franciscanus</em> genes were kindly provided by David Garfield (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Oliver <em>et al.</em> 2010</a>).</p></div><div id="sec-3" class="subsection"><h3>Codon bias measures</h3><p id="p-14">We used the effective number of codons, Nc (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Wright 1990</a>), to estimate codon bias. Nc (or ENC) is analogous to the effective number of alleles at a locus and ranges from 61 (random usage) to 20 (one codon per amino acid). To account for background nucleotide composition, Nc′ scores also were determined using ENCprime (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Novembre 2002</a>). Background composition was determined from the genomic nucleotide sequences of all nucleotides 1000 bp upstream and downstream of transcript initiation start and stop sites, respectively. Coding sequence GC levels (GC<sub>cds</sub>) and third position codon GC levels (GC3) were calculated using CodonW. The GC content of flanking regions (GC<sub>f</sub>) was estimated from nucleotide sequences 1000 bp upstream and downstream of the predicted start and stop codons, respectively. GC levels for introns (GC<sub>i</sub>) were calculated for all genes possessing introns (n = 4389). These data for S. <em>purpuratus</em> are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FileS1.txt">File S1</a>.</p><p id="p-15">To examine the relative contributions of individual amino acids to the overall codon bias of a gene, sENC-X scores were calculated for each amino acid (where X is a particular amino acid) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Moriyama and Powell 1997</a>). Because unscaled ENC-X scores depend on the degeneracy of the amino acid (<em>e.g.</em>, fourfold degenerate amino acids have ENC-X scores between 0 and 4), the sENC-X values are reset to fall between 0 (no bias) and 1 (maximum bias) for all amino acids. ENC-X scores were estimated using the ‘cusage’ program kindly provided by Etsuko Moriyama.</p><p id="p-16">To identify genome-wide preferred codons, we determined frequency of each codon’s usage within its codon family for each gene and tested the Spearman correlation between these frequencies with the overall codon bias of that gene (Nc′). The preferred codon for each amino acid was identified as that with the most significant negative correlation with Nc′ (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>). In other words, the genome-wide preferred codons are defined as the codons that are used more frequently as the bias of the gene increases (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/005769SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS1.pdf">Figure S1</a>). The significance threshold was defined by a <em>P</em>-value smaller than 0.05/<em>n</em>, where <em>n</em> is the number of codons in the codon family. This adjustment is to correct for performing more comparisons on more degenerate codon families (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>).</p></div><div id="sec-4" class="subsection"><h3>Clustering genes by codon usage distributions</h3><p id="p-17">We used the clustering method of <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> (2006)</a> to identify groups on the basis of their codon usage patterns. Each group was defined by a distribution of synonymous codon usage that differed from group to group. The number of clusters, <em>S</em>, was optimized in terms of the information content. <em>S</em> was determined by maximizing the difference in self-consistency, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-1.gif"/></noscript></span></span>, between the real data and a null model. <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-2.gif"/></noscript></span></span> provides a value reflecting the quality of the assignment of the gene <em>g</em> to cluster <em>s</em>, where the quality of the assignment relates to how close the value is to unity (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> 2006</a>). The geometric average, <em>B</em>(<em>S</em>), of all clusters provides the quality of corresponding assignments. A null distribution was generated using an artificial dataset generated from the real data and the quality of this set, <em>B<sub>random</sub></em>(<em>S</em>), also was determined. The maximum difference Δ(<em>S</em>) = <em>B</em>(<em>S</em>) − <em>B<sub>random</sub></em>(<em>S</em>) gives the number of clusters <em>S</em> retained. Clustering and assessments were performed using the software implementation of <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> (2006)</a>. For visualization in heatmaps, codon usage frequencies were centered on a random expectation using the degeneracy of the codon. The centered value, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-3.gif"/></noscript></span></span>, was calculated from the observed frequency <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-4.gif"/></noscript></span></span> of group <em>g</em> for codon <em>c</em> for amino acid <em>a</em> and the expected equal usage frequency based on the degeneracy <em>d</em> of the codon:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/7/1069/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span></p><p id="p-18">We also applied the probabilistic model of codon bias implemented by SCUMBLE v 1.0 (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Kloster and Tang 2008</a>) to identify clusters of genes with similar patterns of codon usage. The algorithm fits a codon usage model in which each gene <em>g</em> is assigned a given number of “offsets” <em>β<sub>i</sub></em>(<em>g</em>). An offset indicates to what extent gene <em>g</em> is affected by estimated bias (‘trend’) number <em>i</em>. A “preference function” <em>E<sub>i</sub></em>(<em>c</em>) for each trend indicates how much trend <em>i</em> favors/disfavors codon <em>c</em> (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Kloster and Tang 2008</a>). We explored models with 0 to 10 trends using SCUMBLE. The number of trends used for the analyses was based on the normalized variance <em>NV(g)</em> of the genes.</p></div><div id="sec-5" class="subsection"><h3>Conservation of preferred synonymous codons</h3><p id="p-19">To test for the conservation of preferred codons, we used Akashi’s test (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akashi 1994</a>). For all genes, each codon was identified as preferred or nonpreferred and as conserved (synonymous) or nonconserved (nonsynonymous) between <em>S. purpuratus</em> and either <em>A. fragilis</em> or <em>S. franciscanus</em> (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Stoletzki and Eyre-Walker 2007</a>) using alignments provided by David Garfield (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Oliver <em>et al.</em> 2010</a>). Preferred codons within groups delineated by the clustering methods described above were identified as having the highest frequency of usage within each separate group. Only genes with sufficient data for the contingency tables were used (n = 2349). The Woolf test for homogeneity across genes in each cluster (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Woolf 1955</a>) was implemented using woolf_test from the vcd library in the R statistical package (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ihaka and Gentleman 1996</a>). All tables for genes clustering into different groups were combined for analysis using the Cochran–Mantel–Haenszel procedure (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cochran 1954</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mantel and Haenszel 1959</a>) and tested using mantelhaen.test from the stats library in R. This allowed us to determine if site conservation (<em>i.e.</em>, synonymous substitutions) was independent of codon preference in any given codon usage cluster. To assess the strength of the association between conservation and codon preference, we followed <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Drummond and Wilke (2008)</a> and generated a reference distribution of the odds ratio for each codon usage group by performing Akashi’s test and computing the odds ratio for 1000 randomly generated sets of synonymous codons from the genes of that group.</p></div><div id="sec-6" class="subsection"><h3>Preference for mRNA folding stabilizing codons</h3><p id="p-20">To test if the third nucleotide positions of preferred codons were more likely to be stem-paired in RNA secondary structure, the data were stratified by gene. RNA secondary structure predictions were performed using RNAfold from the Vienna RNA Package v1.8.5 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hofacker 2004</a>). Nucleotides in the third position of each codon (N3) were identified as either preferred or non-preferred and as falling in a stem or loop in the folding prediction. Preferred codons for each grouping were determined as described above for the <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akashi (1994)</a> test. Then, 2 × 2 contingency tables were constructed for each gene (<em>e.g.</em>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS1.pdf">Table S1</a>) and Woolf tests for homogeneity and association tests using the Cochran–Mantel–Haenszel procedure were performed as described previously. This allowed us to test if folding role (<em>i.e.</em>, paired in a stem or unpaired in a loop) is independent of codon preference.</p></div><div id="sec-7" class="subsection"><h3><em>D. melanogaster</em> genomics</h3><p id="p-21">The genomic assembly for <em>D. melanogaster</em> was obtained from the UCSC Genome Browser (dm3, Apr. 2006, BDGP R5). A total of 11,776 gene models were obtained for the Release 5.4 Annotation Update from flyBase.org (Flybase 2007_03 Update). Each gene prediction underwent the same validation procedure described earlier for <em>S. purpuratus</em>, leaving 11,223 gene models for analyses. The coding sequence for <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> and 1000 bp of flanking sequence was obtained from flyBase.org. Estimates of mRNA expression were obtained from the LS mean of overall expression of <em>D. melanogaster (y</em>;<em>cnbwsp)</em> from the Affymetrix Gene Chip Drosophila Tiling 1.0R Arrays (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Graze <em>et al.</em> 2009</a>). Expression values for <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> were not available in these data. These gene data for <em>D. melanogaster</em> are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FileS2.txt">File S2</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genome-wide patterns of codon bias</h3><p id="p-22">The mean GC content of the <em>S. purpuratus</em> genome is 36.9% and the vast majority of the ~23,300 genes fall in regions of GC between 35 and 39% <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Sodergren <em>et al.</em> (2006)</a>. For the 4623 genes analyzed in the present study, the GC content of coding regions was greater than the genome-wide mean (48.4%) but the GC content of introns and flanking regions were not (35.1% and 36.3%, respectively). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> presents the overall extent of codon bias in our data. <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Wright’s (1990)</a> effective number of codons (Nc) ranged from 32 to 61, exhibiting a mean of 54.5 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Correcting for background GC yielded Nc′ scores ranging from 25.4 to 61 with mean of 51.0.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of codon bias scores of (A) Nc and (B) Nc&#x2032; in S. purpuratus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-624155640" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of codon bias scores of (A) Nc and (B) Nc&#x2032; in &lt;em&gt;S. purpuratus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1069/F1.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/7/1069/F1.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462919" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Distribution of codon bias scores of (A) Nc and (B) Nc′ in <em>S. purpuratus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Despite the weak overall signal of codon bias, we successfully identified preferred codons at all 18 amino acids having two or more synonymous codons (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS1.pdf">Figure S1</a> shows for isoleucine (<em>i.e.</em>, Ile) how the usage of the preferred codon (ATC) increases as the magnitude of codon bias increases. In contrast to the AT bias in the <em>S. purpuratus</em> genome, all preferred codons used a C or a G at the third position and the former were used far more frequently than the latter (15 <em>vs.</em> 3; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS2.pdf">Table S2</a>). We also observed significant genome-wide correlations between isoaccepting tRNA gene copy numbers and the usage of both synonymous codon and amino acids (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Across all groupings, synonymous codons identified by tRNA genes with a greater species counts were used more commonly. Many of the isoaccepting tRNA gene predictions were redundant for the synonymous codons they identified (not shown).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462905/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462905/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome-wide preferred synonymous codon usage in <em>S. purpuratus</em></span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462996/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462996/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Correlations between frequency of isoaccepting tRNA gene copy numbers and usage of synonymous codons and amino acids</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Identification of different codon usage groups</h3><p id="p-28">The clustering method of <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> (2006)</a> identified five significantly distinct clusters of genes having different codon usage patterns (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS3.pdf">Table S3</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The support for five groups was significantly better than six groups for both assignment probability [<em>B(S)</em> = 0.900 ± 0.002 <em>vs.</em> −0.880 ± 0.002, respectively] and maximum cluster stability [Δ<em>(S)</em> = 0.312 ± 0.003<em>vs</em>. 0.306 ± 0.002, respectively; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS2.pdf">Figure S2</a>A]. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows that although codon bias was evident across all five groups (<em>i.e.</em>, GC3 was elevated above both GC<em><sub>i</sub></em> and GC<em><sub>f</sub></em>), it was most highly skewed in groups 0 and 1. Genes in groups 0 and 1 showed the highest levels of GC3 and the most extreme degrees of codon bias. Although groups 0 and 1 were similar in their levels of codon bias and patterns of synonymous codon usage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS3.pdf">Table S3</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), group 0 genes exhibited dramatically greater levels of mRNA expression than the others (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap of codon usage frequencies in the five codon usage groups (CUG) in (A) Strongylocentrotus purpuratus and (B) Drosophila melanogaster. Codon usage frequencies were centered on the equal usage expectation (e.g., for fourfold degenerate codons, frequencies were centered on 1/4). Centered frequencies clustering was performed with hierarchical cluster analysis using pairwise complete-linkage by Euclidean distance for both columns (CUG) and rows (synonymous codons) using Cluster 3.0 (Eisen et al. 1998). Heatmap plots were generated with Java TreeView (Saldanha 2004). Synonymous codon labels are colored by the base composition of the third position (N3) with N3 of &#x2018;A&#x2019; and &#x2018;T&#x2019; nucleotides, colored pink, and &#x2018;G&#x2019; and &#x2018;C,&#x2019; colored green. The asterisk (*) denotes genome-wide preferred codons in S. purpuratus (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-624155640" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heatmap of codon usage frequencies in the five codon usage groups (CUG) in (A) &lt;em&gt;Strongylocentrotus purpuratus&lt;/em&gt; and (B) &lt;em&gt;Drosophila melanogaster&lt;/em&gt;. Codon usage frequencies were centered on the equal usage expectation (&lt;em&gt;e.g.&lt;/em&gt;, for fourfold degenerate codons, frequencies were centered on 1/4). Centered frequencies clustering was performed with hierarchical cluster analysis using pairwise complete-linkage by Euclidean distance for both columns (CUG) and rows (synonymous codons) using Cluster 3.0 (Eisen &lt;em&gt;et al.&lt;/em&gt; 1998). Heatmap plots were generated with Java TreeView (Saldanha 2004). Synonymous codon labels are colored by the base composition of the third position (N3) with N3 of &#x2018;A&#x2019; and &#x2018;T&#x2019; nucleotides, colored pink, and &#x2018;G&#x2019; and &#x2018;C,&#x2019; colored green. The asterisk (*) denotes genome-wide preferred codons in &lt;em&gt;S. purpuratus&lt;/em&gt; (see &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1069/F2.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/7/1069/F2.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Heatmap of codon usage frequencies in the five codon usage groups (CUG) in (A) <em>Strongylocentrotus purpuratus</em> and (B) <em>Drosophila melanogaster</em>. Codon usage frequencies were centered on the equal usage expectation (<em>e.g.</em>, for fourfold degenerate codons, frequencies were centered on 1/4). Centered frequencies clustering was performed with hierarchical cluster analysis using pairwise complete-linkage by Euclidean distance for both columns (CUG) and rows (synonymous codons) using Cluster 3.0 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Eisen <em>et al.</em> 1998</a>). Heatmap plots were generated with Java TreeView (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Saldanha 2004</a>). Synonymous codon labels are colored by the base composition of the third position (N3) with N3 of ‘A’ and ‘T’ nucleotides, colored pink, and ‘G’ and ‘C,’ colored green. The asterisk (*) denotes genome-wide preferred codons in <em>S. purpuratus</em> (see <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462990/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462990/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of codon bias, GC content, gene size, and mRNA expression levels in the five codon usage groups</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">The SCUMBLE model of <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Kloster and Tang (2008)</a> produced results that corroborated the existence of five distinct codon usage groups (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Histograms of normalized variance (<em>NV</em>) suggested that at least four trends were required to explain the data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS3.pdf">Figure S3</a>A) and models with more than four trends failed to show improvement (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS3.pdf">Figure S3</a>B). Offsets <em>β<sub>1</sub></em> and <em>β<sub>3</sub></em> correlated most strongly with GC3 and CT3 in all groups and in the pooled data, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS4.pdf">Table S4</a>). Groups were differentiated, however, only by offset <em>β</em><sub>2</sub> that was most strongly associated with mRNA expression levels at 72 hr for group 0, GT3 in group 2, and CT3 in groups 1, 3, and 4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS4.pdf">Table S4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of offsets &#x3B2;1 and &#x3B2;2 for a SCUMBLE model with four trends for S. purpuratus genes, colored by groups of genes clustered by codon usage distributions. Offset &#x3B2;1 is correlated most strongly to GC3 in all groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-624155640" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of offsets &#x3B2;&lt;sub&gt;1&lt;/sub&gt; and &#x3B2;&lt;sub&gt;2&lt;/sub&gt; for a SCUMBLE model with four trends for &lt;em&gt;S. purpuratus&lt;/em&gt; genes, colored by groups of genes clustered by codon usage distributions. Offset &#x3B2;&lt;sub&gt;1&lt;/sub&gt; is correlated most strongly to GC3 in all groups.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1069/F3.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/7/1069/F3.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463022" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Plot of offsets β<sub>1</sub> and β<sub>2</sub> for a SCUMBLE model with four trends for <em>S. purpuratus</em> genes, colored by groups of genes clustered by codon usage distributions. Offset β<sub>1</sub> is correlated most strongly to GC3 in all groups.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Patterns of codon usage bias</h3><p id="p-33">Codon bias in <em>S. purpuratus</em> was driven by synonymous codon usage in a subset of amino acids (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Four amino acids contributed significantly to codon bias in all five groups and in the pooled data [leucine (Leu), isoleucine (Ile), serine (Ser), and threonine (Thr)], whereas two amino acids did not contribute to codon bias in any group [histidine (His) and cysteine (Cys)]. Groups 0 and 1 exhibited the most strongly negative correlations (<em>i.e.</em>, <em>P</em> &lt; 10<sup>−10</sup>) between sENC-X and Nc′ (9 each) and group 3 had the fewest (3). Although there was considerable variability in the contributions of individual amino acids across groups, the strongest correlations in the entire set of 4623 genes tended to show significant patterns in four or five of the groups.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462925/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462925/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">The contributions of individual amino acids to codon bias in <em>S. purpuratus</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-37">The GC content of third positions (GC3) and coding regions (GC<sub>cds</sub>) exhibited significant correlations with the degree of codon bias (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). By contrast, there was a limited association between the GC content of introns and flanking regions and the degree of codon bias (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, C and D</a> and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). Our results show that GC content of introns or flanking regions for all genes displayed weak associations with Nc (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS5.pdf">Table S5</a>) and no significant correlations with Nc′ (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). GC3 was strongly correlated with GC<sub>cds</sub> in all groups (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Relationships between the GC content of introns and flanking regions with both GC3 and GC<sub>cds</sub> were much weaker and inconsistent among groups (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Codon bias and GC composition in protein coding gene regions for different codon usage groups. Codon bias is represented as Wright&#x2019;s Nc (Wright 1990). Regional GC composition is calculated from (A) GC3 content of exons, (B) GC content of exons, (C) GC content of introns, and (D) GC content of flanking regions. &#x201C;Rp&#x201D; denotes annotated ribosomal proteins. The genome-wide mean GC content is 36.9%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-624155640" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Codon bias and GC composition in protein coding gene regions for different codon usage groups. Codon bias is represented as Wright&#x2019;s Nc (Wright 1990). Regional GC composition is calculated from (A) GC3 content of exons, (B) GC content of exons, (C) GC content of introns, and (D) GC content of flanking regions. &#x201C;Rp&#x201D; denotes annotated ribosomal proteins. The genome-wide mean GC content is 36.9%.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1069/F4.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/7/1069/F4.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462993" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Codon bias and GC composition in protein coding gene regions for different codon usage groups. Codon bias is represented as Wright’s Nc (<a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">Wright 1990</a>). Regional GC composition is calculated from (A) GC3 content of exons, (B) GC content of exons, (C) GC content of introns, and (D) GC content of flanking regions. “Rp” denotes annotated ribosomal proteins. The genome-wide mean GC content is 36.9%.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462903/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462903/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Correlations between codon bias (Nc′), GC composition, and rates of protein evolution in <em>S. purpuratus</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-43">The magnitude of codon bias was significantly associated with gene expression levels, but only in group 0 (Nc’ ρ = −0.2578, <em>P</em> &lt; 0.0001; Nc ρ = −0.2695, <em>P</em> &lt; 0.0001). Highly biased, highly transcribed genes were primarily found in group 0 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). For example, 51 of the 54 annotated ribosomal proteins were assigned to group 0 (of the remaining three, one was in group 1 and two in group 4). However, as with <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> in <em>Drosophila</em>, we observed highly expressed, highly biased genes that did not share a synonymous codon usage pattern with ribosomal proteins and the other members of group 0. For example, Sp-Ets1/2 (SPU_002874) is a member of the ETS family of transcription factors. It acts as a transcriptional activator or repressor and is involved in cell senescence and death, stem cell development and tumorigenesis (RefSeq Release 49, July 2011). Sp-Ets1/2 is highly expressed (41,923 AU) and highly biased (Nc′ = 42.443) but is a member of group 1 rather than group 0.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synonymous codon usage bias and mRNA expression levels for genes in different codon usage groups in (A) S. purpuratus and (B) Drosophila melanogaster. Codon usage bias is Novembre&#x2019;s Nc&#x2032;. Red circles surround annotated ribosomal proteins (&#x2018;Rp&#x2019;). The black arrow labeled &#x201C;U&#x201D; points to &#x201C;ubiquitin-like/S30 ribosomal fusion protein&#x201D; (SPU_005280), an example of a highly expressed gene in S. purpuratus. The black arrow labeled &#x201C;E&#x201D; points to Sp-Ets1/2 (SPU_002874), an example of a highly expressed, highly biased gene that does not belong to codon usage group 0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-624155640" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Synonymous codon usage bias and mRNA expression levels for genes in different codon usage groups in (A) &lt;em&gt;S. purpuratus&lt;/em&gt; and (B) &lt;em&gt;Drosophila melanogaster&lt;/em&gt;. Codon usage bias is Novembre&#x2019;s Nc&#x2032;. Red circles surround annotated ribosomal proteins (&#x2018;Rp&#x2019;). The black arrow labeled &#x201C;U&#x201D; points to &#x201C;ubiquitin-like/S30 ribosomal fusion protein&#x201D; (SPU_005280), an example of a highly expressed gene in &lt;em&gt;S. purpuratus&lt;/em&gt;. The black arrow labeled &#x201C;E&#x201D; points to Sp-Ets1/2 (SPU_002874), an example of a highly expressed, highly biased gene that does not belong to codon usage group 0.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/7/1069/F5.medium.gif" width="315" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/7/1069/F5.medium.gif" width="315" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/7/1069/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463036" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Synonymous codon usage bias and mRNA expression levels for genes in different codon usage groups in (A) <em>S. purpuratus</em> and (B) <em>Drosophila melanogaster</em>. Codon usage bias is Novembre’s Nc′. Red circles surround annotated ribosomal proteins (‘Rp’). The black arrow labeled “U” points to “ubiquitin-like/S30 ribosomal fusion protein” (SPU_005280), an example of a highly expressed gene in <em>S. purpuratus</em>. The black arrow labeled “E” points to Sp-Ets1/2 (SPU_002874), an example of a highly expressed, highly biased gene that does not belong to codon usage group 0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Gene length was also negatively correlated with mRNA expression and Nc′ (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS4.pdf">Figure S4</a>). We did not observe relationships between codon bias and position along the protein using intragenic spatial codon usage estimates (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Qin <em>et al.</em> 2004</a>). The magnitude of codon bias did not differ between the middle sections of genes relative to amino- or carboxy-terminal regions, did not change over the length of the gene for different size classes and was not correlated with intron number or length (not shown). Thus, we were unable to find support for interference selection acting on synonymous codon usage in <em>S. purpuratus</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Comeron and Kreitman 2002</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462971/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462971/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Correlations between gene size and mRNA expression, codon bias (Nc′), and GC content in <em>S. purpuratus</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Conservation of preferred codons</h3><p id="p-49">We tested whether preferred codons were more evolutionarily constrained by applying <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Akashi’s (1994)</a> method. The conservation of codons was assessed using alignments containing <em>S. purpuratus</em>, <em>A. fragilis</em>, and <em>S. franciscanus</em>. We observed that preferred codons were more likely to be conserved than unpreferred codons in all five groups and for all genes combined (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). This is remarkable considering that each group had a different set of preferred codons defined for the test. The Woolf test on homogeneity of odds ratios (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Woolf 1955</a>) showed significant three-way associations for all groups. To further test this result, we randomly generated sets of preferred codons and found that our previously identified list exhibited a significantly stronger association between codon preference and conservation across each (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>). In all five codon usage groups, stronger correlations were detected between our identified preferred codons and the evolutionarily conserved codons than that for the all genes combined.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462880/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462880/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Akashi’s test for the conservation of preferred codons between species</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Role of preferred codons in stabilizing RNA structure</h3><p id="p-53">We found that the third positions of preferred codons were significantly more likely to be paired in stems within folded RNA secondary structures in groups 2 and 3 (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). Although all five groups had different distributions of preferred third positions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS2.pdf">Table S2</a>), each had two N3 preferred codons that were significantly overrepresented in stem pairs. In all five groups, G3 was significantly associated to be paired in stems in preferred codons. C3 of preferred codons were more likely to be stem-paired in all groups except group 2, where it was replaced by U3. The identity of the majority codon N3 base did not match the preferred stems for any group. The Woolf test on homogeneity of odds ratios (<a id="xref-ref-83-4" class="xref-bibr" href="#ref-83">Woolf 1955</a>) produced significant 3-way associations for all groups.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462963/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462963/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Tests for stem-pairing of preferred codon N3 in mRNA secondary structure predictions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Comparison to codon usage patterns in <em>Drosophila</em></h3><p id="p-56">To determine whether similar clusters of codon usage patterns occur in <em>Drosophila melanogaster</em>, we applied the <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> (2006)</a> clustering method to 11,223 gene models obtained from flybase.org. Similar to our results for <em>S. purpuratus</em>, we detected five significantly distinct groups (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/FigureS2.pdf">Figure S2</a>B). Although similar in overall number, the patterns of codon usage differed between the species. Overall, 2675 (23.8%) genes were assigned to group 0, 2919 (26.0%) to group 1, 607 (5.4%) to group 2, 4156 (37.0%) to group 3, and 867 (7.7%) to group 4. Highly expressed, highly biased genes were almost entirely represented by <em>Drosophila</em> group 4 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). As found in <em>S. purpuratus</em>, a majority of annotated small and large subunit ribosomal proteins were observed in this highly expressed group (54 of 60). Third positions in <em>Drosophila</em> groups 1, 2, 3, and 4 were more likely to be G or C (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Conversely, N3 for genes in <em>Drosophila</em> Group 2 were more likely to be A or T. Despite being highly biased (<em>Nc</em>′ = 43.31) and typically highly expressed, the synonymous codon usage in <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> was more similar to other members of <em>Drosophila</em> Group 3. Genome-wide, <em>S. purpuratus</em> genes were not as biased as <em>D. melanogaster</em> (Nc′ = 46.04 ± 6.83 and 51.08 ± 5.08, respectively). <em>S. purpuratus</em> group 0 genes were more biased than <em>D. melanogaster</em> genome-wide average (Nc′ = 42.50 ± 15.5) but less biased than <em>D. melanogaster</em> group 4 (Nc′ = 39.08 ± 7.51).</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-57">Synonymous codon usage is influenced by the competing actions of random genetic drift, natural selection, and mutation (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Sharp <em>et al.</em> 1995</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Duret 2002</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>). Despite decades of study on codon bias the relative contributions of mutational bias <em>vs.</em> selection favoring the speed and accuracy of protein translation, translational robustness, or mRNA stability remains unclear (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Duret 2002</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Plotkin and Kudla 2011</a>). Selection for translational efficiency has long been recognized as an important factor affecting synonymous codon usage bias in unicellular and multicellular organisms (<em>e.g.</em>, <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Powell and Moriyama 1997</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Akashi 1994</a>). However, other forms of selection can favor translationally suboptimal codons (<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Hershberg and Petrov 2008</a>) that in competition with translational selection can result in heterogeneous patterns of codon usage across the genome (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Akashi and Eyre-Walker 1998</a>). The present study has confirmed that multiple distinct patterns of codon usage bias exist in <em>S. purpuratus</em> that appear to result from different modes of selection (<em>i.e.</em>, translational efficiency and mRNA stability). Our results also suggest that mutational bias has played a negligible role in generating codon bias in <em>S. purpuratus</em>.</p><div id="sec-16" class="subsection"><h3>Distinct codon usage groups occur in <em>S. purpuratus</em></h3><p id="p-58">Synonymous codon usage varies between genes varies in both composition and magnitude (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Sharp <em>et al.</em> 1986</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Shields and Sharp 1987</a>). Genome-wide patterns of codon bias are driven primarily by the most highly biased genes, which, if they share preferred codons, will dominate the synonymous codon usage patterns. It is thus easy to overlook more subtle patterns of codon bias. This is evident in yeast where codon usage differs between highly and lowly expressed genes in both the level and nature of the bias (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Sharp <em>et al.</em> 1986</a>). A similar situation is seen in <em>Bacillus subtilis</em> where correspondence analysis identified an axis with ribosomal proteins at one end and genes with different codon usage patterns at the other end (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Shields and Sharp 1987</a>). <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> is a highly expressed and highly biased gene in <em>Drosophila</em> but exhibits a codon usage distribution different from other similar loci (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Dumont <em>et al.</em> 2004</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Singh <em>et al.</em> 2007</a>). These subtle patterns of codon usage appear to be biologically significant and need to be considered in studies examining codon bias.</p><p id="p-59">We identified five distinct codon usage groups in <em>S. purpuratus</em> using two independent methods (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figures 2A</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The magnitude of codon bias and the usage of genome-wide preferred codons varied dramatically between groups (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS3.pdf">Table S3</a>), as did the mean levels of maximum mRNA expression during early development (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). Previously, the detection of distinct codon usage clusters has been restricted to bacteria and yeast (<em>e.g.</em>, <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> 2006</a>; <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Kloster and Tang 2008</a>). In <em>E. coli</em> K12 and <em>B. subtilis</em>, <a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Bailly-Bechet <em>et al.</em> (2006)</a> identified four and five distinct codon usage clusters, respectively. In both species, genes belonging to different clusters were associated together in different chromosomal regions implicating some role for translational selection. The present study represents the first documentation of similar codon usage clusters existing in a metazoan. To assess the generality of our results in <em>S. purpuratus</em> we also identified five distinct codon usage groups in <em>D. melanogaster</em> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>), a well studied species where translational selection on rate and accuracy has been previously documented (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Powell and Moriyama 1997</a>). <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em>, being an interesting anomaly for synonymous codon usage in <em>Drosophila</em>, was expected to have a different codon usage pattern from other highly expressed and highly biased genes. As expected, <em><a href="http://flybase.org/reports/FBgn0004647.html">Notch</a></em> had a different codon usage distribution from the majority of ribosomal proteins in <em>Drosophila</em>. Our identification of distinct codon usage groups in <em>D. melanogaster</em> supports the reliability of the patterns we observed in <em>S. purpuratus</em> and suggests that similar clusters may be present in other species.</p></div><div id="sec-17" class="subsection"><h3>Mutational processes cannot explain codon bias in <em>S. purpuratus</em></h3><p id="p-60">If mutational bias is responsible for generating codon usage bias, synonymous sites are expected to have nucleotide compositions similar to introns and flanking regions. <em>S. purpuratus</em> is an excellent species to test for mutational bias because it possesses an AT-biased genome lacking isochores and the majority of genes reside in regions of GC content between 35 and 39% (<a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Sodergren <em>et al.</em> 2006</a>). Despite this overall AT bias, all 18 preferred codons used a C or a G at their third positions. In our data, the mean GC content of introns and flanking regions were similar to the genome-wide average and did not vary among the different codon usage clusters (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The magnitude of codon bias exhibited strong correlations with mean GC3 and GC<em><sub>cds</sub></em> in the most highly biased groups (0 and 1). However, the degree of codon bias in groups 0 and 1 showed no associations with either GC<sub>i</sub> or GC<sub>f</sub> (<a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS5.pdf">Table S5</a>, and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). These observations suggest that mutational bias has played a negligible role in influencing patterns of codon bias in <em>S. purpuratus</em>.</p></div><div id="sec-18" class="subsection"><h3>Selection on translation drives major codon usage patterns</h3><p id="p-61">Codon usage and tRNA abundances are thought to have coevolved to maximize the speed (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bulmer 1987</a>) or the accuracy of protein translation (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Akashi 1994</a>). Speed of translation is an adaptation for translation rate whereas translational accuracy acts to minimize the incorporation of incorrect amino acids into proteins. These two interrelated drivers of selection on translational efficiency (speed and accuracy) are difficult to disentangle and share similar predictions. For example, both hypotheses predict that the abundances of tRNA species should correlate with preferred codons in highly expressed genes (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Ikemura 1985</a>) and that high levels of gene expression should correlate with high levels of codon bias (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Duret and Mouchiroud 1999</a>). In <em>S. purpuratus</em> we obtained evidence for both of these predictions. We found that tRNA gene frequencies were significantly correlated with genome-wide preferred codons and tRNA gene copy number exhibited a significant positive relationship with amino acid counts. Furthermore, the magnitude of codon bias was strongly related to mRNA expression levels during early development (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>).</p><p id="p-62">Although we detected genome-wide evidence for translational selection, it was driven exclusively by genes in group 0 that represented only 8.6% of all loci. The ability of such a small number of genes to drive this overall pattern is attributable to their dramatically greater levels of mRNA expression (roughly four times the other groups; see <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). Strikingly similar results were obtained for <em>D. melanogaster</em>, in which highly expressed and highly biased genes were represented in only one of the five clusters identified (group 4; <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). In both species, the vast majority of ribosomal protein genes were present in the highly expressed cluster and <em>S. purpuratus</em> group 0 was overrepresented with GO terms related to transcription and cell growth (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS6.pdf">Table S6</a>). Translational selection thus apparently acts on only one of the five codon usage groups identified in both species and only a small proportion of genes (&lt;10%) are driving the genome-wide correlations between codon bias and mRNA expression.</p><p id="p-63">In <em>S. purpuratus</em>, codon usage group 1 shares similar patterns of codon preference to group 0 but fails to exhibit a correlation with gene expression levels. There are three amino acids [serine (Ser), proline [Pro], and glycine [Gly]) with different preferred codons between groups 0 and 1 (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Only Ser contributes to codon bias in both groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1/TableS3.pdf">Table S3</a>), whereas Pro contributes only to group 0. If group 1 genes have also been selected for translational efficiency then one explanation for this anomalous result is they are not expressed during early development but are highly expressed at later developmental stages. Recent studies in <em>Drosophila</em> have found that larval stages exhibit greater overall codon bias than adult stages, supporting the hypothesis that codon usage bias is influenced by selection for efficient protein production (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Vicario <em>et al.</em> 2008</a>). In contrast, genes expressed after embryogenesis in <em>C. elegans</em> have greater synonymous substitution rates than those expressed during embryogenesis (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Castillo-Davis and Hartl 2002</a>). The possibility of translational selection acting on group 1 genes in <em>S. purpuratus</em> at later developmental stages needs to be investigated further.</p></div><div id="sec-19" class="subsection"><h3>Preferred codons are evolutionarily conserved</h3><p id="p-64">Selection for the accuracy of translation is an additional explanation for the relationship between codon bias and rates of translation in <em>Drosophila</em> (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Akashi 1994</a>). The model assumes that the misincorporation of amino acids due to errors in translation adversely affects the functionality of proteins and reduces cell fitness. There are two unique predictions of the translational accuracy model: (1) synonymous codon usage at evolutionary conserved sites will be biased toward preferred codons, and (2) the degree of conservation will be stronger in highly expressed genes. As found for <em>Drosophila</em>, we observed that preferred codons were evolutionarily conserved across all genes in <em>S. purpuratus</em> (<a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>). In addition, we observed that preferred codons were conserved to a similar degree in all five codon usage groups with their respective sets of preferred codons despite differences in their synonymous codon usage patterns, levels of mRNA expression, and their magnitude of codon bias (<a id="xref-table-wrap-7-4" class="xref-table" href="#T7">Table 7</a>). This is particularly noteworthy for group 2, whose preferred codons end primarily in A and T. Selection for translational accuracy should also be strongest in the most highly expressed group 0 but instead appeared to be independent of mRNA expression. Therefore, selection for translational accuracy cannot account for the different patterns of codon usage observed between the clusters.</p></div><div id="sec-20" class="subsection"><h3>Preferred third codon positions are more likely to pair in mRNA stems</h3><p id="p-65">Selection to optimize heteroduplex base paring and maximize the stability of mRNA secondary structure has also been considered as a potential driver of codon usage bias for many years (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Fitch 1974</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Klämbt 1975</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Grantham <em>et al.</em> 1980</a>). A relationship between folding stability and codon preference has been established through the examination of mRNA secondary structure in mammals and other groups (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Fitch 1974</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hasegawa <em>et al.</em> 1979</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chamary and Hurst 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Meyer and Miklós 2005</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Chamary <em>et al.</em> 2006</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shabalina <em>et al.</em> 2006</a>). mRNA stability is expected to be influenced by base composition at third positions (N3) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Carlini <em>et al.</em> 2001</a>). A preference for G or C in N3 is seen in the most highly biased genes of <em>Drosophila</em>, which is almost entirely explained by an increased use of C over G (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Shields <em>et al.</em> 1988</a>). This preference for C in N3 of highly biased loci is not easily explained by local mutation effects but may result from its effect on mRNA stability (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gillespie 1991</a>).The stability of mRNA secondary structure can be increased through GC heteroduplex pairing in N3 or by an increase in the number of heteroduplex pairs. Recognizing there are other competing processes that determine preferred N3 composition (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Chamary and Hurst 2005</a>), and different definitions of preference not limited to GC (this study), we restricted our analysis to stems and loops and tested if the third positions of preferred codons were more likely to reside as paired members of stems than unpaired members of loops. More specifically, if natural selection acts on synonymous codons to increase mRNA stability then we expect N3 of preferred codons to more likely to reside as paired members of stems than unpaired members of loops.</p><p id="p-66">In <em>S. purpuratus</em>, we observed that any nucleotide in the third position of preferred codons (N3) were more likely to be in paired in stems than unpaired in loops only for genes in codon usage groups 2 and 3 (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>). Ignoring codon usage groups, the Mantel-Haenszel tests were significant for C3 and G3, which is consistent with the observation that all genome-wide preferred codons had a G or C in their 3<sup>rd</sup> positions. Considering the groups separately, group 2 differed sharply from the others in having G3 or U3 of preferred codons pairing in stems over loops. Group 0 exhibited the weakest relationships, which is not surprising because selection for mRNA stability may conflict with translational selection and other processes. No significant relationships were observed between mRNA stability and expression levels. Here again, mRNA stability can impact rates of translation and highly stable mRNA structures may be specifically avoided in rapidly transcribed regions of genes (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mita <em>et al.</em> 1988</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zama 1990</a>). Overall these results implicate a role for mRNA folding, perhaps acting in a compensatory fashion, in determining the patterns of synonymous codon usage in <em>S. purpuratus</em>.</p></div><div id="sec-21" class="subsection"><h3>Codon bias decreases with gene length</h3><p id="p-67">Codon bias decreases with increasing gene length in <em>S. purpuratus</em> similar to that observed in other species (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Comeron <em>et al.</em> 1999</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Duret and Mouchiroud 1999</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ingvarsson 2007</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Qiu <em>et al.</em> 2011</a>). This decrease in codon bias with increasing gene length may be a side-effect of the economy of translation where highly expressed, highly biased genes tend to be short (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akashi 2003</a>). Indeed, in group 0 gene length was correlated with both codon bias and mRNA expression levels, further supporting the action of translational selection on this group. However, in the other codon usage groups gene length was correlated with either codon bias (groups 1, 3, and 4) or expression (group 2), but not both. The negative correlation between gene length and codon usage may also be due to GC-biased gene conversion (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Stoletzki 2011</a>). Our results do not support an alternative prediction for translational accuracy where the cost of incorporation of erroneous amino acids during elongation will drive large genes to have higher codon bias than small genes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Eyre-Walker 1996</a>) as observed in yeast (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Eyre-Walker 1996</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Moriyama and Powell 1998</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Stoletzki and Eyre-Walker 2007</a>).</p><p id="p-68">In summary, we detected significant nonrandom patterns of synonymous codon usage in the purple sea urchin that were incompatible with the predicted effects of mutational bias. Five distinct clusters of genes were identified that used different sets of preferred codons. However, only one cluster exhibited a correlation between codon bias and mRNA expression consistent with translational selection. The third positions of preferred codons were more likely to be paired in stems in folded mRNA secondary structures in two additional codon usage groups, implicating selection for mRNA stability on synonymous codon usage. The discovery of five distinct codon usage groups in <em>D. melanogaster</em> was similar to that found for <em>S. purpuratus</em>, but the patterns of codon usage differed between species. However in both species only one cluster representing &lt;10% of all genes was associated with translational selection. Our results suggest that other forms of selection might be more important in determining the overall patterns of synonymous codon usage and that similar codon usage clusters might be present in other groups.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-69">We thank Giacomo Bernardi his support and extensive discussions. David Garfield, Heather Trumbower, Daryl Thomas, Eran Halperin, and Norah Saarman provided insightful comments on the manuscript. We thank Etsuko Moriyama for kindly supplying the ‘cusage’ software for the sENC-X calculations, Hong Qin for sharing his R code for codon length calculations, and Marc Bailly-Bechet for sharing his clustering code and guidance on its implementation. Todd Lowe kindly provided tRNA predictions and guidance. David Haussler and Rachel Harte generously provided access and support on UCSC CBSE computational clusters, and Andy Cameron provided annotations from SpBase.org. David Garfield and David Wray kindly provided early communications on codon usage bias in <em>S. purpuratus</em>. We also thank Josep Comeron and two anonymous reviewers for their helpful comments on improving the manuscript. This work was supported by the National Science Foundation (DEB-1011061), the STEPS Foundation, and the Earl and Ethyl Myer’s Trust. Funding for Open Access was provided by the UC Santa Cruz Open Access Fund.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005769/-/DC1</a>.</p></li><li class="fn" id="fn-21"><p id="p-70"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-01-29"><span class="received-label">Received </span>January 29, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-04-21"><span class="accepted-label">Accepted </span>April 21, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Kober, Pogson</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.1" data-doi="10.1111/j.1365-294X.2009.04094.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addison</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pogson</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Multiple gene genealogies reveal asymmetrical hybridization and introgression among strongylocentrotid sea urchins.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DAddison%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D1239%26rft.epage%253D1251%26rft.atitle%253DMultiple%2Bgene%2Bgenealogies%2Breveal%2Basymmetrical%2Bhybridization%2Band%2Bintrogression%2Bamong%2Bstrongylocentrotid%2Bsea%2Burchins.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04094.x%26rft_id%253Dinfo%253Apmid%252F19222750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04094.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19222750&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Synonymous codon usage in <em>Drosophila melanogaster</em>: natural selection and translational accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bnatural%2Bselection%2Band%2Btranslational%2Baccuracy.%26rft_id%253Dinfo%253Apmid%252F8005445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvOTI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Translational selection and yeast proteome evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1291%26rft.epage%253D1303%26rft.atitle%253DTranslational%2BSelection%2Band%2BYeast%2BProteome%2BEvolution%26rft_id%253Dinfo%253Apmid%252F12930740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzEyOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.4" data-doi="10.1016/S0959-437X(98)80038-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Translational selection and molecular evolution.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">688</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D688%26rft.epage%253D693%26rft.atitle%253DTranslational%2Bselection%2Band%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252898%252980038-5%26rft_id%253Dinfo%253Apmid%252F9914211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(98)80038-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9914211&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077614100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurland</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Codon preferences in free-living microorganisms.</span> <abbr class="cit-jnl-abbrev">Microbiol. Rev.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DAndersson%26rft.auinit1%253DS%2BG%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D210%26rft.atitle%253DCodon%2Bpreferences%2Bin%2Bfree-living%2Bmicroorganisms.%26rft_id%253Dinfo%253Apmid%252F2194095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1NC8yLzE5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.6" data-doi="10.1371/journal.pcbi.0020037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailly-Bechet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iqbal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsili</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vergassola</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Codon usage domains over bacterial chromosomes.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBailly-Bechet%26rft.auinit1%253DM.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253De37%26rft.epage%253De37%26rft.atitle%253DCodon%2Busage%2Bdomains%2Bover%2Bbacterial%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020037%26rft_id%253Dinfo%253Apmid%252F16683018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16683018&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.7" data-doi="10.1016/0092-8674(78)90044-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Britten</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cetta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">The single-copy DNA sequence polymorphism of the sea urchin <em>Strongylocentrotus purpuratus</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1175</span>–<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBritten%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D1175%26rft.epage%253D1186%26rft.atitle%253DThe%2Bsingle-copy%2BDNA%2Bsequence%2Bpolymorphism%2Bof%2Bthe%2Bsea%2Burchin%2BStrongylocentrotus%2Bpurpuratus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252878%252990044-2%26rft_id%253Dinfo%253Apmid%252F728997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(78)90044-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=728997&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978GC55400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.8" data-doi="10.1038/325728a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Coevolution of codon usage and transfer RNA abundance.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBulmer%26rft.auinit1%253DM.%26rft.volume%253D325%26rft.issue%253D6106%26rft.spage%253D728%26rft.epage%253D730%26rft.atitle%253DCoevolution%2Bof%2Bcodon%2Busage%2Band%2Btransfer%2BRNA%2Babundance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F325728a0%26rft_id%253Dinfo%253Apmid%252F2434856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/325728a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2434856&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G087100058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The selection-mutation-drift theory of synonymous codon usage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBulmer%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D897%26rft.epage%253D907%26rft.atitle%253DThe%2Bselection-mutation-drift%2Btheory%2Bof%2Bsynonymous%2Bcodon%2Busage.%26rft_id%253Dinfo%253Apmid%252F1752426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvODk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.10" data-doi="10.1016/j.gene.2007.04.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A basal deuterostome genome viewed as a natural experiment.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">406</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DAndrew%2BCameron%26rft.auinit1%253DR.%26rft.volume%253D406%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DA%2Bbasal%2Bdeuterostome%2Bgenome%2Bviewed%2Bas%2Ba%2Bnatural%2Bexperiment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2007.04.031%26rft_id%253Dinfo%253Apmid%252F17550788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2007.04.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17550788&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.11" data-doi="10.1093/nar/gkn887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samanta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SpBase: the sea urchin genome database and web site.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">D750</span>–<span class="cit-lpage">D754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn887%26rft_id%253Dinfo%253Apmid%252F19010966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8xL0Q3NTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The relationship between third-codon position nucleotide content, codon bias, mRNA secondary structure and gene expression in the drosophilid alcohol dehydrogenase genes <em>Adh</em> and <em>Adhr</em></span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCarlini%26rft.auinit1%253DD.%2BB.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D623%26rft.epage%253D633%26rft.atitle%253DThe%2BRelationship%2BBetween%2BThird-Codon%2BPosition%2BNucleotide%2BContent%252C%2BCodon%2BBias%252C%2BmRNA%2BSecondary%2BStructure%2Band%2BGene%2BExpression%2Bin%2Bthe%2BDrosophilid%2BAlcohol%2BDehydrogenase%2BGenes%2BAdh%2Band%2BAdhr%26rft_id%253Dinfo%253Apmid%252F11606539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.13" data-doi="10.1093/oxfordjournals.molbev.a004131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castillo-Davis</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genome evolution and developmental constraint in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCastillo-Davis%26rft.auinit1%253DC.%2BI.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D728%26rft.epage%253D735%26rft.atitle%253DGenome%2BEvolution%2Band%2BDevelopmental%2BConstraint%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004131%26rft_id%253Dinfo%253Apmid%252F11961106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvNzI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.14" data-doi="10.1186/gb-2005-6-9-r75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for selection on synonymous mutations affecting stability of mRNA secondary structure in mammals.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253DR75%26rft.epage%253DR75%26rft.atitle%253DEvidence%2Bfor%2Bselection%2Bon%2Bsynonymous%2Bmutations%2Baffecting%2Bstability%2Bof%2BmRNA%2Bsecondary%2Bstructure%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-9-r75%26rft_id%253Dinfo%253Apmid%252F16168082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-9-r75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16168082&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.15" data-doi="10.1038/nrg1770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DHearing%2Bsilence%253A%2Bnon-neutral%2Bevolution%2Bat%2Bsynonymous%2Bsites%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1770%26rft_id%253Dinfo%253Apmid%252F16418745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418745&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.16" data-doi="10.1073/pnas.0307827100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottes</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAdams</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage between genomes is constrained by genome-wide mutational processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">3480</span>–<span class="cit-lpage">3485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307827100%26rft_id%253Dinfo%253Apmid%252F14990797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0ODAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.17" data-doi="10.1126/science.168.3934.1009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Darwinian evolution of proteins.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DClarke%26rft.auinit1%253DB.%26rft.volume%253D168%26rft.issue%253D3934%26rft.spage%253D1009%26rft.epage%253D1011%26rft.atitle%253DDarwinian%2BEvolution%2Bof%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.168.3934.1009%26rft_id%253Dinfo%253Apmid%252F4909620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTM6IjE2OC8zOTM0LzEwMDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.18" data-doi="10.2307/3001616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Some methods for strengthening the common chi-2 Tests.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D10%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954UG27100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.19" data-doi="10.1093/bioinformatics/btp163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biopython: freely available Python tools for computational molecular biology and bioinformatics.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1422</span>–<span class="cit-lpage">1423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp163%26rft_id%253Dinfo%253Apmid%252F19304878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQyMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Population, evolutionary and genomic consequences of interference selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D410%26rft.atitle%253DPopulation%252C%2BEvolutionary%2Band%2BGenomic%2BConsequences%2Bof%2BInterference%2BSelection%26rft_id%253Dinfo%253Apmid%252F12019253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguade</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Natural selection on synonymous sites is correlated with gene length and recombination in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D249%26rft.atitle%253DNatural%2BSelection%2Bon%2BSynonymous%2BSites%2BIs%2BCorrelated%2BWith%2BGene%2BLength%2Band%2BRecombination%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9872963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMjM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.22" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.23" data-doi="10.1016/j.cell.2008.05.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDrummond%26rft.auinit1%253DD.%2BA.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D352%26rft.atitle%253DMistranslation-induced%2Bprotein%2Bmisfolding%2Bas%2Ba%2Bdominant%2Bconstraint%2Bon%2Bcoding-sequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.05.042%26rft_id%253Dinfo%253Apmid%252F18662548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.05.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662548&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257891700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.24" data-doi="10.1534/genetics.167.1.171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DuMont</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">DNA variability and divergence at the notch locus in <em>Drosophila melanogaster</em> and <em>D. simulans</em>: a case of accelerated synonymous site divergence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDuMont%26rft.auinit1%253DV.%2BB.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D171%26rft.epage%253D185%26rft.atitle%253DDNA%2BVariability%2Band%2BDivergence%2Bat%2Bthe%2BNotch%2BLocus%2Bin%2BDrosophila%2Bmelanogaster%2Band%2BD.%2Bsimulans%253A%2BA%2BCase%2Bof%2BAccelerated%2BSynonymous%2BSite%2BDivergence%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.171%26rft_id%253Dinfo%253Apmid%252F15166145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvMTcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.25" data-doi="10.1016/S0168-9525(00)02041-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D287%26rft.epage%253D289%26rft.atitle%253DtRNA%2Bgene%2Bnumber%2Band%2Bcodon%2Busage%2Bin%2Bthe%2BC.%2Belegans%2Bgenome%2Bare%2Bco-adapted%2Bfor%2Boptimal%2Btranslation%2Bof%2Bhighly%2Bexpressed%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902041-2%26rft_id%253Dinfo%253Apmid%252F10858656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02041-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10858656&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087898200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.26" data-doi="10.1016/S0959-437X(02)00353-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of synonymous codon usage in metazoans.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D640%26rft.epage%253D649%26rft.atitle%253DEvolution%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bmetazoans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900353-2%26rft_id%253Dinfo%253Apmid%252F12433576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00353-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433576&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.27" data-doi="10.1073/pnas.96.8.4482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4482</span>–<span class="cit-lpage">4487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4482%26rft.epage%253D4487%26rft.atitle%253DExpression%2Bpattern%2Band%252C%2Bsurprisingly%252C%2Bgene%2Blength%2Bshape%2Bcodon%2Busage%2Bin%2BCaenorhabditis%252C%2BDrosophila%252C%2Band%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4482%26rft_id%253Dinfo%253Apmid%252F10200288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ0ODIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.28" data-doi="10.1073/pnas.95.25.14863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cluster analysis and display of genome-wide expression patterns.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">14863</span>–<span class="cit-lpage">14868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEisen%26rft.auinit1%253DM.%2BB.%26rft.volume%253D95%26rft.issue%253D25%26rft.spage%253D14863%26rft.epage%253D14868%26rft.atitle%253DCluster%2Banalysis%2Band%2Bdisplay%2Bof%2Bgenome-wide%2Bexpression%2Bpatterns%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.25.14863%26rft_id%253Dinfo%253Apmid%252F9843981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMjUvMTQ4NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.29" data-doi="10.1093/oxfordjournals.molbev.a025646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Synonymous codon bias is related to gene length in Escherichia coli: selection for translational accuracy?</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">864</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D864%26rft.epage%253D872%26rft.atitle%253DSynonymous%2Bcodon%2Bbias%2Bis%2Brelated%2Bto%2Bgene%2Blength%2Bin%2BEscherichia%2Bcoli%253A%2Bselection%2Bfor%2Btranslational%2Baccuracy%253F%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025646%26rft_id%253Dinfo%253Apmid%252F8754221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzYvODY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.30" data-doi="10.1007/BF01796043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The large extent of putative secondary nucleic acid structure in random nucleotide sequences or amino acid derived messenger-RNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFitch%26rft.auinit1%253DW.%2BM.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D279%26rft.epage%253D291%26rft.atitle%253DThe%2Blarge%2Bextent%2Bof%2Bputative%2Bsecondary%2Bnucleic%2Bacid%2Bstructure%2Bin%2Brandom%2Bnucleotide%2Bsequences%2Bor%2Bamino%2Bacid%2Bderived%2Bmessenger-RNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01796043%26rft_id%253Dinfo%253Apmid%252F4416342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01796043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4416342&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974U165100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-3.7.1069.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source">The Causes of Molecular Evolution</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.32" data-doi="10.1093/nar/8.1.197-c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavé</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Codon catalog usage and the genome hypothesis.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">r49</span>–<span class="cit-lpage">r62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253Dr49%26rft.epage%253D62%26rft.atitle%253DCodon%2Bcatalog%2Busage%2Band%2Bthe%2Bgenome%2Bhypothesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.1.197-c%26rft_id%253Dinfo%253Apmid%252F6986610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/8.1.197-c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6986610&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JC46600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-3.7.1069.33" data-doi="10.1534/genetics.109.105957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Main</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulatory divergence in <em>Drosophila melanogaster</em> and <em>D. simulans</em>, a genomewide analysis of allele-specific expression.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">183</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">561</span>, <span class="cit-fpage">541SI</span>–<span class="cit-lpage">521SI</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105957%26rft_id%253Dinfo%253Apmid%252F19667135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzIvNTQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.34" data-doi="10.1093/nar/gkh036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ireland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomax</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Gene Ontology (GO) database and informatics resource.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D258</span>–<span class="cit-lpage">D261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGene%2BOntology%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD258%26rft.epage%253D261%26rft.atitle%253DThe%2BGene%2BOntology%2B%2528GO%2529%2Bdatabase%2Band%2Binformatics%2Bresource%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh036%26rft_id%253Dinfo%253Apmid%252F14681407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMi9zdXBwbF8xL0QyNTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.35" data-doi="10.1093/nar/7.7.2073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasunaga</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Secondary structure of MS2 phage RNA and bias in code word usage.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2073</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D2073%26rft.epage%253D2079%26rft.atitle%253DSecondary%2Bstructure%2Bof%2BMS2%2Bphage%2BRNA%2Band%2Bbias%2Bin%2Bcode%2Bword%2Busage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F7.7.2073%26rft_id%253Dinfo%253Apmid%252F537920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjcvNy8yMDczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.36" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-3.7.1069.37"><div class="cit-metadata unstructured">Hofacker, I. L., 2004 RNA secondary structure analysis using the Vienna RNA package. Curr Protoc Bioinformatics Chapter 12: Unit 12.12.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.38" data-doi="10.2307/1390807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ihaka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">R: a language for data analysis and graphics.</span> <abbr class="cit-jnl-abbrev">J. Comput. Graph. Statist.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BGraph.%2BStatist.%26rft.volume%253D5%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1390807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1390807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.40" data-doi="10.1093/molbev/msl212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene expression and protein length influence codon usage and rates of sequence evolution in <em>Populus tremula</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl212%26rft_id%253Dinfo%253Apmid%252F17204548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzMvODM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-3.7.1069.41"><div class="cit-metadata unstructured">Karolchik, D., R. M. Kuhn, R. Baertsch, G. P. Barber, H. Clawson <em>et al.</em>, 2008 The UCSC Genome Browser Database: 2008 update. Nulceic Acids Research<strong>: 36</strong>(Database issue): D773−D779.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.42" data-doi="10.1126/science.164.3881.788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Non-Darwinian evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">788</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKing%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D164%26rft.issue%253D3881%26rft.spage%253D788%26rft.epage%253D798%26rft.atitle%253DNon-Darwinian%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.164.3881.788%26rft_id%253Dinfo%253Apmid%252F5767777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjE2NC8zODgxLzc4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.43" data-doi="10.1016/0022-5193(75)90039-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klämbt</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">A model for messenger RNA sequences maximizing secondary structure due to code degeneracy.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DKl%25C3%25A4mbt%26rft.auinit1%253DD.%26rft.volume%253D52%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D65%26rft.atitle%253DA%2Bmodel%2Bfor%2Bmessenger%2BRNA%2Bsequences%2Bmaximizing%2Bsecondary%2Bstructure%2Bdue%2Bto%2Bcode%2Bdegeneracy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252875%252990039-9%26rft_id%253Dinfo%253Apmid%252F1152488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(75)90039-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1152488&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.44" data-doi="10.1093/nar/gkn288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloster</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">SCUMBLE: a method for systematic and accurate detection of codon usage bias by maximum likelihood estimation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">3819</span>–<span class="cit-lpage">3827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn288%26rft_id%253Dinfo%253Apmid%252F18495752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8xMS8zODE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kober</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylogenomics of Strongylocentrotid sea urchins.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-3.7.1069.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antezana</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The population and evolutionary genetics of codon bias</span>, pp. <span class="cit-fpage">82</span>–<span class="cit-lpage">101</span> in <span class="cit-source">Evolutionary Genetics: From Molecules to Morphology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Krimbas</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, U.K.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.47" data-doi="10.1126/science.1170160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.48" data-doi="10.1093/nar/25.5.0955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.5.0955%26rft_id%253Dinfo%253Apmid%252F9023104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvOTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.49" data-doi="10.1093/jnci/22.4.719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mantel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haenszel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Statistical aspects of the analysis of data from retrospective studies of disease.</span> <abbr class="cit-jnl-abbrev">J. Natl. Cancer Inst.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNatl.%2BCancer%2BInst.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjnci%252F22.4.719%26rft_id%253Dinfo%253Apmid%252F13655060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiam5jaSI7czo1OiJyZXNpZCI7czo4OiIyMi80LzcxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.50" data-doi="10.1093/nar/gki923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miklós</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Statistical evidence for conserved, local secondary structure in the coding regions of eukaryotic mRNAs and pre-mRNAs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">6338</span>–<span class="cit-lpage">6348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMeyer%26rft.auinit1%253DI.%2BM.%26rft.volume%253D33%26rft.issue%253D19%26rft.spage%253D6338%26rft.epage%253D6348%26rft.atitle%253DStatistical%2Bevidence%2Bfor%2Bconserved%252C%2Blocal%2Bsecondary%2Bstructure%2Bin%2Bthe%2Bcoding%2Bregions%2Bof%2Beukaryotic%2BmRNAs%2Band%2Bpre-mRNAs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki923%26rft_id%253Dinfo%253Apmid%252F16275783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8xOS82MzM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.51" data-doi="10.1016/0022-2836(88)90117-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mita</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Specific codon usage pattern and its implications on the secondary structure of silk fibroin mRNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">917</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMita%26rft.auinit1%253DK.%26rft.volume%253D203%26rft.issue%253D4%26rft.spage%253D917%26rft.epage%253D925%26rft.atitle%253DSpecific%2Bcodon%2Busage%2Bpattern%2Band%2Bits%2Bimplications%2Bon%2Bthe%2Bsecondary%2Bstructure%2Bof%2Bsilk%2Bfibroin%2BmRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990117-9%26rft_id%253Dinfo%253Apmid%252F3210244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90117-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3210244&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q766100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.52" data-doi="10.1007/PL00006256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Codon usage bias and tRNA abundance in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMoriyama%26rft.auinit1%253DE.%2BN.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D514%26rft.epage%253D523%26rft.atitle%253DCodon%2Busage%2Bbias%2Band%2BtRNA%2Babundance%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006256%26rft_id%253Dinfo%253Apmid%252F9342399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9342399&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YA94500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.53" data-doi="10.1093/nar/26.13.3188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Gene length and codon usage bias in <em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae</em> and <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3188</span>–<span class="cit-lpage">3193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.13.3188%26rft_id%253Dinfo%253Apmid%252F9628917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMy8zMTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruveiller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Onofrio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Translational selection on codon usage in <em>Xenopus laevis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1703</span>–<span class="cit-lpage">1707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMusto%26rft.auinit1%253DH.%26rft.volume%253D18%26rft.issue%253D9%26rft.spage%253D1703%26rft.epage%253D1707%26rft.atitle%253DTranslational%2BSelection%2Bon%2BCodon%2BUsage%2Bin%2BXenopus%2Blaevis%26rft_id%253Dinfo%253Apmid%252F11504850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMTcwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.55" data-doi="10.1186/gb-2011-12-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The role of codon selection in regulation of translation efficiency deduced from synthetic libraries.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DNavon%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DThe%2Brole%2Bof%2Bcodon%2Bselection%2Bin%2Bregulation%2Bof%2Btranslation%2Befficiency%2Bdeduced%2Bfrom%2Bsynthetic%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-2-r12%26rft_id%253Dinfo%253Apmid%252F21284851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21284851&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.56" data-doi="10.1093/oxfordjournals.molbev.a004201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Accounting for background nucleotide composition when measuring codon usage bias.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1390</span>–<span class="cit-lpage">1394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D1390%26rft.epage%253D1394%26rft.atitle%253DAccounting%2Bfor%2BBackground%2BNucleotide%2BComposition%2BWhen%2BMeasuring%2BCodon%2BUsage%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004201%26rft_id%253Dinfo%253Apmid%252F12140252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE5LzgvMTM5MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.57" data-doi="10.1093/gbe/evq063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfield</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manier</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haygood</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome positive selection and habitat-driven evolution in a shallow and a deep-sea urchin.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq063%26rft_id%253Dinfo%253Apmid%252F20935062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjIvMC84MDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.58" data-doi="10.2307/2409417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Mitochondrial DNA diversity in the sea urchins <em>Strongylocentrotus purpuratus</em> and <em>S. droebachiensis</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D44%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990CZ91500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.59" data-doi="10.1016/j.ydbio.2006.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pederson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The sea urchin’s siren.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DPederson%26rft.auinit1%253DT.%26rft.volume%253D300%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D14%26rft.atitle%253DThe%2Bsea%2Burchin%2527s%2Bsiren.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2006.10.006%26rft_id%253Dinfo%253Apmid%252F17113576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2006.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17113576&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.60" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caccone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigro</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Rates of DNA evolution in <em>Drosophila</em> depend on function and developmental stage of expression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D133%26rft.issue%253D2%26rft.spage%253D291%26rft.epage%253D298%26rft.atitle%253DRates%2Bof%2BDNA%2Bevolution%2Bin%2BDrosophila%2Bdepend%2Bon%2Bfunction%2Band%2Bdevelopmental%2Bstage%2Bof%2Bexpression.%26rft_id%253Dinfo%253Apmid%252F8094697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzIvMjkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.62" data-doi="10.1073/pnas.94.15.7784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evolution of codon usage bias in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">7784</span>–<span class="cit-lpage">7790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D94%26rft.issue%253D15%26rft.spage%253D7784%26rft.epage%253D7790%26rft.atitle%253DEvolution%2Bof%2Bcodon%2Busage%2Bbias%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.15.7784%26rft_id%253Dinfo%253Apmid%252F9223264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTUvNzc4NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.63" data-doi="10.1534/genetics.104.030866"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Intragenic spatial patterns of codon usage bias in prokaryotic and eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">2245</span>–<span class="cit-lpage">2260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DQin%26rft.auinit1%253DH.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D2245%26rft.epage%253D2260%26rft.atitle%253DIntragenic%2BSpatial%2BPatterns%2Bof%2BCodon%2BUsage%2BBias%2Bin%2BProkaryotic%2Band%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.030866%26rft_id%253Dinfo%253Apmid%252F15611189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzIyNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.64" data-doi="10.1093/molbev/msq251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergero</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patterns of codon usage bias in <em>Silene latifolia</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq251%26rft_id%253Dinfo%253Apmid%252F20855431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvNzcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.65" data-doi="10.1093/dnares/dsr035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation bias is the driving force of codon usage in the <em>Gallus gallus</em> genome.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsr035%26rft_id%253Dinfo%253Apmid%252F22039174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE4LzYvNDk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.66" data-doi="10.1093/bioinformatics/bth349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saldanha</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Java Treeview—extensible visualization of microarray data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3246</span>–<span class="cit-lpage">3248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth349%26rft_id%253Dinfo%253Apmid%252F15180930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTcvMzI0NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.67" data-doi="10.1093/nar/gkl287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shabalina</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogurtsov</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiridonov</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A periodic pattern of mRNA secondary structure created by the genetic code.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2428</span>–<span class="cit-lpage">2437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DShabalina%26rft.auinit1%253DS.%2BA.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D2428%26rft.epage%253D2437%26rft.atitle%253DA%2Bperiodic%2Bpattern%2Bof%2BmRNA%2Bsecondary%2Bstructure%2Bcreated%2Bby%2Bthe%2Bgenetic%2Bcode.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl287%26rft_id%253Dinfo%253Apmid%252F16682450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM0LzgvMjQyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvNy8xMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.68" data-doi="10.1007/BF02099948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D24%26rft.issue%253D1-2%26rft.spage%253D28%26rft.epage%253D38%26rft.atitle%253DAn%2Bevolutionary%2Bperspective%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bunicellular%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099948%26rft_id%253Dinfo%253Apmid%252F3104616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3104616&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F784400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.69" data-doi="10.1093/nar/14.13.5125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuohy</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosurski</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5125</span>–<span class="cit-lpage">5143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D14%26rft.issue%253D13%26rft.spage%253D5125%26rft.epage%253D5143%26rft.atitle%253DCodon%2Busage%2Bin%2Byeast%253A%2Bcluster%2Banalysis%2Bclearly%2Bdifferentiates%2Bhighly%2Band%2Blowly%2Bexpressed%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.13.5125%26rft_id%253Dinfo%253Apmid%252F3526280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMy81MTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.70" data-doi="10.1098/rstb.1995.0108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Averof</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matassi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peden</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">DNA sequence evolution: The sounds of silence.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D349%26rft.issue%253D1329%26rft.spage%253D241%26rft.epage%253D247%26rft.atitle%253DDNA%2BSequence%2BEvolution%253A%2BThe%2BSounds%2Bof%2BSilence%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1995.0108%26rft_id%253Dinfo%253Apmid%252F8577834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNDkvMTMyOS8yNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.71" data-doi="10.1093/nar/15.19.8023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Synonymous codon usage in <em>Bacillus subtilis</em> reflects both translational selection and mutational biases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">8023</span>–<span class="cit-lpage">8040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DShields%26rft.auinit1%253DD.%2BC.%26rft.volume%253D15%26rft.issue%253D19%26rft.spage%253D8023%26rft.epage%253D8040%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BBacillus%2Bsubtilis%2Breflects%2Bboth%2Btranslational%2Bselection%2Band%2Bmutational%2Bbiases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.19.8023%26rft_id%253Dinfo%253Apmid%252F3118331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xOS84MDIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">“Silent” sites in <em>Drosophila</em> genes are not neutral: evidence of selection among synonymous codons.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DShields%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D704%26rft.epage%253D716%26rft.atitle%253D%2522Silent%2522%2Bsites%2Bin%2BDrosophila%2Bgenes%2Bare%2Bnot%2Bneutral%253A%2Bevidence%2Bof%2Bselection%2Bamong%2Bsynonymous%2Bcodons%26rft_id%253Dinfo%253Apmid%252F3146682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjUvNi83MDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.73" data-doi="10.1093/molbev/msm196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer DuMont</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Patterns of mutation and selection at synonymous sites in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2687</span>–<span class="cit-lpage">2697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm196%26rft_id%253Dinfo%253Apmid%252F18000010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi8yNjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.74" data-doi="10.1126/science.1133609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sodergren</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genome of the sea urchin <em>Strongylocentrotus purpuratus</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">941</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSea%2BUrchin%2BGenome%2BSequencing%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D314%26rft.issue%253D5801%26rft.spage%253D941%26rft.epage%253D952%26rft.atitle%253DThe%2BGenome%2Bof%2Bthe%2BSea%2BUrchin%2BStrongylocentrotus%2Bpurpuratus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1133609%26rft_id%253Dinfo%253Apmid%252F17095691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTQvNTgwMS85NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzcvMTA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.75" data-doi="10.1101/gr.361602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulez</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chervitz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Bioperl toolkit: Perl modules for the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStajich%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1611%26rft.epage%253D1618%26rft.atitle%253DThe%2BBioperl%2BToolkit%253A%2BPerl%2BModules%2Bfor%2Bthe%2BLife%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.361602%26rft_id%253Dinfo%253Apmid%252F12368254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.76" data-doi="10.1093/nar/22.13.2437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stenico</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Codon usage in <em>Caenorhabditis elegans</em>: delineation of translational selection and mutational biases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2437</span>–<span class="cit-lpage">2446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStenico%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D13%26rft.spage%253D2437%26rft.epage%253D2446%26rft.atitle%253DCodon%2Busage%2Bin%2BCaenorhabditis%2Belegans%253A%2Bdelineation%2Bof%2Btranslational%2Bselection%2Band%2Bmutational%2Bbiases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.13.2437%26rft_id%253Dinfo%253Apmid%252F8041603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMy8yNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.77" data-doi="10.1186/1471-2148-11-93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The surprising negative correlation of gene length and optimal codon use–disentangling translational selection from GC-biased gene conversion in yeast.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DStoletzki%26rft.auinit1%253DN.%26rft.volume%253D11%26rft.spage%253D93%26rft.epage%253D93%26rft.atitle%253DThe%2Bsurprising%2Bnegative%2Bcorrelation%2Bof%2Bgene%2Blength%2Band%2Boptimal%2Bcodon%2Buse--disentangling%2Btranslational%2Bselection%2Bfrom%2BGC-biased%2Bgene%2Bconversion%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-93%26rft_id%253Dinfo%253Apmid%252F21481245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21481245&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.78" data-doi="10.1093/molbev/msl166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl166%26rft_id%253Dinfo%253Apmid%252F17101719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.79" data-doi="10.2307/2407502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strathmann</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">The evolution and loss of feeding larval stages of marine invertebrates.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">894</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D894%26rft.epage%253D906%26rft.atitle%253DThe%2Bevolution%2Band%2Bloss%2Bof%2Bfeeding%2Blarval%2Bstages%2Bof%2Bmarine%2Binvertebrates%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2407502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2407502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1979017285&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=A1978GF79200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.80" data-doi="10.1016/j.cell.2010.03.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vestsigian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorfan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTuller%26rft.auinit1%253DT.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D344%26rft.epage%253D354%26rft.atitle%253DAn%2Bevolutionarily%2Bconserved%2Bmechanism%2Bfor%2Bcontrolling%2Bthe%2Befficiency%2Bof%2Bprotein%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.031%26rft_id%253Dinfo%253Apmid%252F20403328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403328&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.81" data-doi="10.1093/molbev/msn189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicario</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Developmental stage and level of codon usage bias in Drosophila.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2269</span>–<span class="cit-lpage">2277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn189%26rft_id%253Dinfo%253Apmid%252F18755761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8yMjY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy83LzEwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.82" data-doi="10.1016/j.ydbio.2006.08.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angerer</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angerer</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A database of mRNA expression patterns for the sea urchin embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Bio.l</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DWei%26rft.auinit1%253DZ.%26rft.volume%253D300%26rft.issue%253D1%26rft.spage%253D476%26rft.epage%253D484%26rft.atitle%253DA%2Bdatabase%2Bof%2BmRNA%2Bexpression%2Bpatterns%2Bfor%2Bthe%2Bsea%2Burchin%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2006.08.034%26rft_id%253Dinfo%253Apmid%252F17007833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2006.08.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17007833&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242873300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.83" data-doi="10.1111/j.1469-1809.1955.tb01348.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woolf</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1955</span> <span class="cit-article-title">On estimating the relation between blood group and disease.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DWoolf%26rft.auinit1%253DB.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D251%26rft.epage%253D253%26rft.atitle%253DOn%2Bestimating%2Bthe%2Brelation%2Bbetween%2Bblood%2Bgroup%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1955.tb01348.x%26rft_id%253Dinfo%253Apmid%252F14388528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1955.tb01348.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14388528&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.84" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The ‘effective number of codons’ used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.85" data-doi="10.1016/0006-291X(90)92092-E"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zama</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Codon usage pattern in alpha 2(I) chain domain of chicken type I collagen and its implications for the secondary structure of the mRNA and the synthesis pauses of the collagen.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DZama%26rft.auinit1%253DM.%26rft.volume%253D167%26rft.issue%253D2%26rft.spage%253D772%26rft.epage%253D776%26rft.atitle%253DCodon%2Busage%2Bpattern%2Bin%2Balpha%2B2%2528I%2529%2Bchain%2Bdomain%2Bof%2Bchicken%2Btype%2BI%2Bcollagen%2Band%2Bits%2Bimplications%2Bfor%2Bthe%2Bsecondary%2Bstructure%2Bof%2Bthe%2BmRNA%2Band%2Bthe%2Bsynthesis%2Bpauses%2Bof%2Bthe%2Bcollagen.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0006-291X%252890%252992092-E%26rft_id%253Dinfo%253Apmid%252F2322250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0006-291X(90)92092-E&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2322250&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-3.7.1069.86" data-doi="10.1016/0022-5193(65)90083-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuckerkandl</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauling</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Molecules as documents of evolutionary history.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DZuckerkandl%26rft.auinit1%253DE.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D357%26rft.epage%253D366%26rft.atitle%253DMolecules%2Bas%2Bdocuments%2Bof%2Bevolutionary%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252865%252990083-4%26rft_id%253Dinfo%253Apmid%252F5876245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(65)90083-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5876245&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F7%2F1069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19656369100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/7/1069.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/7/1061" title="Genetic Variants That Confer Resistance to Malaria Are Associated with Red Blood Cell Traits in African-Americans: An Electronic Medical Record-based Genome-Wide Association Study" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/7/1085" title="Using Genome-Wide Association Analysis to Characterize Environmental Sensitivity of Milk Traits in Dairy Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/7"><span>Volume 3 Issue 7, July 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/7.cover-source.jpg?itok=DKXH6P5c" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/7/1069" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GECkeE4kgOkcA8Zv4FwrzAzk_nUV-Ar-RCobTbPV-C8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/7/1069" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/7/1069" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/7/1069" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, Strongylocentrotus purpuratus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459487" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qZ--5JvJaTVZaOx6t7C1qaeM-_eAY3CjaakVf4Ovfac" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1069.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/7/1069" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-n8x4pYjqkjeysA4e_UXByqXfbQ9kIxO2YWXggN7qQQo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/7/1069" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/7/1069.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459487" id="node459487--2" data-pisa="ggg;3/7/1069" data-pisa-master="ggg;g3.113.005769" data-apath="/ggg/3/7/1069.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, <em>Strongylocentrotus purpuratus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kord M. Kober</span> and <span class="highwire-citation-author" data-delta="1">Grant H. Pogson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1069-1083;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005769 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459487--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kord M. Kober </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiological Nursing, University of California, San Francisco, California 94143</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kord%2BM.%2BKober%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kober%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKord%2BM.%2BKober%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kord.kober@nursing.ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">kord.kober@nursing.ucsf.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Grant H. Pogson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BH.%2BPogson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pogson%20GH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrant%2BH.%2BPogson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/7/1069" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/7/1069&amp;title=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/7/1069&amp;title=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/7/1069&amp;title=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/7/1069&amp;text=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/7/1069&amp;title=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/7/1069&amp;t=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/7/1069&amp;title=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/7/1069&amp;title=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459487/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459487" id="node459487--3" data-pisa="ggg;3/7/1069" data-pisa-master="ggg;g3.113.005769" data-apath="/ggg/3/7/1069.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Patterns of Codon Bias Are Shaped by Natural Selection in the Purple Sea Urchin, <em>Strongylocentrotus purpuratus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kord M. Kober</span> and <span class="highwire-citation-author" data-delta="1">Grant H. Pogson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1069-1083;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005769 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459487--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kord M. Kober </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiological Nursing, University of California, San Francisco, California 94143</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kord%2BM.%2BKober%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kober%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKord%2BM.%2BKober%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kord.kober@nursing.ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">kord.kober@nursing.ucsf.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Grant H. Pogson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Santa Cruz, California 95064</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BH.%2BPogson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pogson%20GH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrant%2BH.%2BPogson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1069&amp;rft.epage=1083&amp;rft.atitle=Genome-Wide%20Patterns%20of%20Codon%20Bias%20Are%20Shaped%20by%20Natural%20Selection%20in%20the%20Purple%20Sea%20Urchin%2C%20Strongylocentrotus%20purpuratus&amp;rft.volume=3&amp;rft.issue=7&amp;rft.date=2013-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kober%2C+Kord+M.&amp;rft.au=Pogson%2C+Grant+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459487/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459487/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459487/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459487/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459487/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459487/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459487/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459487/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459487/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459487/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459487/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459487/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/7/1069" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/7/1069.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/7/1069.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/7/1069.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/7/1069.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
