<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/2/191" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/2/191.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/2/191.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/2/191.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/2/191/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/2/191.atom" />
<meta name="HW.pisa" content="ggg;2/2/191" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings" />
<meta name="DC.Identifier" content="10.1534/g3.111.001784" />
<meta name="DC.Date" content="2012-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2012 Welsh and McMillan. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Inbred model organisms are powerful tools for genetic studies because they provide reproducible genomes for use in mapping and genetic manipulation. Generating inbred lines via sibling matings, however, is a costly undertaking that requires many successive generations of breeding, during which time many lines fail. We evaluated several approaches for accelerating inbreeding, including the systematic use of back-crosses and marker-assisted breeder selection, which we contrasted with randomized sib-matings. Using simulations, we explored several alternative breeder-selection methods and monitored the gain and loss of genetic diversity, measured by the number of recombination-induced founder intervals, as a function of generation. For each approach we simulated 100,000 independent lines to estimate distributions of generations to achieve full-fixation as well as to achieve a mean heterozygosity level equal to 20 generations of randomized sib-mating. Our analyses suggest that the number of generations to fully inbred status can be substantially reduced with minimal impact on genetic diversity through combinations of parental backcrossing and marker-assisted inbreeding. Although simulations do not consider all confounding factors underlying the inbreeding process, such as a loss of fecundity, our models suggest many viable alternatives for accelerating the inbreeding process." />
<meta name="DC.Contributor" content="Catherine E. Welsh" />
<meta name="DC.Contributor" content="Leonard McMillan" />
<meta name="article:published_time" content="2012-02-01" />
<meta name="article:section" content="Mouse Genetic Resources" />
<meta name="citation_title" content="Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Inbred model organisms are powerful tools for genetic studies because they provide reproducible genomes for use in mapping and genetic manipulation. Generating inbred lines via sibling matings, however, is a costly undertaking that requires many successive generations of breeding, during which time many lines fail. We evaluated several approaches for accelerating inbreeding, including the systematic use of back-crosses and marker-assisted breeder selection, which we contrasted with randomized sib-matings. Using simulations, we explored several alternative breeder-selection methods and monitored the gain and loss of genetic diversity, measured by the number of recombination-induced founder intervals, as a function of generation. For each approach we simulated 100,000 independent lines to estimate distributions of generations to achieve full-fixation as well as to achieve a mean heterozygosity level equal to 20 generations of randomized sib-mating. Our analyses suggest that the number of generations to fully inbred status can be substantially reduced with minimal impact on genetic diversity through combinations of parental backcrossing and marker-assisted inbreeding. Although simulations do not consider all confounding factors underlying the inbreeding process, such as a loss of fecundity, our models suggest many viable alternatives for accelerating the inbreeding process.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/02/01" />
<meta name="citation_mjid" content="ggg;2/2/191" />
<meta name="citation_id" content="2/2/191" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/2/191" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/2/191.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/2/191.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/2/191.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001784" />
<meta name="citation_pmid" content="22384397" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mouse Genetic Resources" />
<meta name="citation_firstpage" content="191" />
<meta name="citation_lastpage" content="198" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Catherine E. Welsh" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Leonard McMillan" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="mcmillan@cs.unc.edu" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=N. Armstrong;citation_author=T. C. Brodnicki;citation_author=T. P. Speed;citation_title=Mind the gap: analysis of marker-assisted breeding strategies for inbred mouse strains.;citation_pages=273-287;citation_volume=17;citation_year=2006;citation_issue=4;citation_pmid=16596449;citation_doi=10.1007/s00335-005-0123-y" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging Collaborative Cross;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=Transplantation;citation_author=D. W. Bailey;citation_title=Recombinant-inbred strains. An aid to finding identity, linkage, and function of histocompatibility and other genes;citation_pages=325-327;citation_volume=11;citation_year=1971;citation_pmid=5558564;citation_doi=10.1097/00007890-197103000-00013" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. Boddhireddy;citation_author=J. L. Jannink;citation_author=J. C. Nelson;citation_title=Selective Advance for Accelerated Development of Recombinant Inbred QTL Mapping Populations;citation_pages=1294;citation_volume=49;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_title=The genomes of recombinant inbred lines;citation_pages=1133-1146;citation_volume=169;citation_year=2005;citation_pmid=15545647;citation_doi=10.1534/genetics.104.035212" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. W. Broman;citation_title=Genotype Probabilities at Intermediate Generations in the Construction of Recombinant Inbred Lines;citation_pages=403-412;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345609;citation_doi=10.1534/genetics.111.132647" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=E. J. Chesler;citation_author=D. R. Miller;citation_author=L. R. Branstetter;citation_author=L. D. Galloway;citation_author=B. L. Jackson;citation_title=The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.;citation_pages=382-389;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18716833;citation_doi=10.1007/s00335-008-9135-8" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_author=Collaborative Cross Consortium;citation_volume=190;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_title=The Mouse in Animal Genetics and Breeding Research;citation_author=E. J. Eisen;citation_year=2005" />
<meta name="citation_reference" content="citation_author=L. Flaherty;citation_author=V. Bolivar;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=X. Gan;citation_author=O. Stegle;citation_author=J. Behr;citation_author=J. G. Steffen;citation_author=P. Drewe;citation_title=Multiple reference genomes and transcriptomes for Arabidopsis thaliana.;citation_pages=419-423;citation_volume=477;citation_year=2011;citation_issue=7365;citation_pmid=21874022;citation_doi=10.1038/nature10414" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. Gibson;citation_author=T. F. C. Mackay;citation_title=Enabling population and quantitative genomics.;citation_pages=1-6;citation_volume=80;citation_year=2002;citation_issue=1;citation_pmid=12448852;citation_doi=10.1017/S0016672302005839" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Y. Liu;citation_author=Q. Zhang;citation_author=L. McMillan;citation_author=F. P. de Villena;citation_author=W. Wang;citation_title=Efficient genome ancestry inference in complex pedigrees with inbreeding;citation_volume=26;citation_year=2010;citation_pmid=20529906;citation_doi=10.1093/bioinformatics/btq187" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Markel;citation_author=P. Shu;citation_author=C. Ebeling;citation_author=G. A. Carlson;citation_author=D. L. Nagle;citation_title=Theoretical and empirical issues for marker-assisted breeding of congenic mouse strains.;citation_pages=280-284;citation_volume=17;citation_year=1997;citation_issue=3;citation_pmid=9354790;citation_doi=10.1038/ng1197-280" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mott;citation_author=C. J. Talbot;citation_author=M. G. Turri;citation_author=A. C. Collins;citation_author=J. Flint;citation_title=A new method for fine-mapping quantitative trait loci in outbred animal stocks;citation_volume=97;citation_year=2000;citation_pmid=11050180;citation_doi=10.1073/pnas.230304397" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. J. Paulo;citation_author=M. Boer;citation_author=X. Huang;citation_author=M. Koornneef;citation_author=F. van Eeuwijk;citation_title=A mixed model QTL analysis for a complex cross population consisting of a half diallel of two-way hybrids in Arabidopsis;citation_pages=107-114;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-008-9665-x" />
<meta name="citation_reference" content="citation_title=Mouse Genetics;citation_author=L. M. Silver;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. A. Taylor;citation_author=H. Meier;citation_author=D. D. Myers;citation_title=Host-Gene Control of C-Type RNA Tumor Virus: Inheritance of the Group-Specific Antigen of Murine Leukemia Virus;citation_pages=3190-3194;citation_volume=68;citation_year=1971;citation_issue=12;citation_pmid=4332013;citation_doi=10.1073/pnas.68.12.3190" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=D. W. Threadgill;citation_author=K. W. Hunter;citation_author=R. W. Williams;citation_title=Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort.;citation_pages=175-178;citation_volume=13;citation_year=2002;citation_issue=4;citation_pmid=11956758;citation_doi=10.1007/s00335-001-4001-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Valdar;citation_author=J. Flint;citation_author=R. Mott;citation_title=Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice;citation_pages=1783-1797;citation_volume=172;citation_year=2006;citation_pmid=16361245;citation_doi=10.1534/genetics.104.039313" />
<meta name="citation_reference" content="citation_journal_title=Pac. Symp. Biocomput.;citation_title=Inferring genome-wide mosaic structure;citation_author=Q. Zhang;citation_author=W. Wang;citation_author=L. McMillan;citation_author=F. Pardo-Manuel de Villena;citation_author=D. Threadgill;citation_pages=150-161;citation_year=2009" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="twitter:description" content="Inbred model organisms are powerful tools for genetic studies because they provide reproducible genomes for use in mapping and genetic manipulation. Generating inbred lines via sibling matings, however, is a costly undertaking that requires many successive generations of breeding, during which time many lines fail. We evaluated several approaches for accelerating inbreeding, including the systematic use of back-crosses and marker-assisted breeder selection, which we contrasted with randomized sib-matings. Using simulations, we explored several alternative breeder-selection methods and monitored the gain and loss of genetic diversity, measured by the number of recombination-induced founder intervals, as a function of generation. For each approach we simulated 100,000 independent lines to estimate distributions of generations to achieve full-fixation as well as to achieve a mean heterozygosity level equal to 20 generations of randomized sib-mating. Our analyses suggest that the number of generations to fully inbred status can be substantially reduced with minimal impact on genetic diversity through combinations of parental backcrossing and marker-assisted inbreeding. Although simulations do not consider all confounding factors underlying the inbreeding process, such as a loss of fecundity, our models suggest many viable alternatives for accelerating the inbreeding process." />
<meta name="og-title" property="og:title" content="Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/2/191" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Inbred model organisms are powerful tools for genetic studies because they provide reproducible genomes for use in mapping and genetic manipulation. Generating inbred lines via sibling matings, however, is a costly undertaking that requires many successive generations of breeding, during which time many lines fail. We evaluated several approaches for accelerating inbreeding, including the systematic use of back-crosses and marker-assisted breeder selection, which we contrasted with randomized sib-matings. Using simulations, we explored several alternative breeder-selection methods and monitored the gain and loss of genetic diversity, measured by the number of recombination-induced founder intervals, as a function of generation. For each approach we simulated 100,000 independent lines to estimate distributions of generations to achieve full-fixation as well as to achieve a mean heterozygosity level equal to 20 generations of randomized sib-mating. Our analyses suggest that the number of generations to fully inbred status can be substantially reduced with minimal impact on genetic diversity through combinations of parental backcrossing and marker-assisted inbreeding. Although simulations do not consider all confounding factors underlying the inbreeding process, such as a loss of fecundity, our models suggest many viable alternatives for accelerating the inbreeding process." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
    <title>Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yBm3wqxmum_j6mkjE1xzB6Nng1hP67LaulqabAqkKRM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459235","apath":"\/ggg\/2\/2\/191.atom","pisa":"ggg;2\/2\/191","ac":{"\/ggg\/2\/2\/191.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/2\/191.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/2\/191"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/2\/191"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/2\/191":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RrLqFnir5TLuAdTrGEUK8l1oDDQsXsZ\\\\\\\/xn1pxVVaC8Y8+KAggBRbxrXk29E2QEWBnrHDMt5ERGwgRCWB2EIrWf\\\\\\\/ljnG2456EPnFvQkzRGfF0dBUhZG9UnPPsgqzI\\\\\\\/2s7K97DzLFkebPS1Sb6tzIe1btXUGI\\\\\\\/gy9ZsuK38+QDOKbX5Pkm7XJZfX\\\\\\\/oaHXUh8YrRnnuUzQ7ugKHY7x3hc+PJ143z\\\\\\\/\\\\\\\/mkHWfhoDcpCvk7G+aXCEeMn75dNvvS+sSNvUoJ3EOj7J6gyNoO6FtPSTVSe\\\\\\\/E+ZAakXLbL34RzHKOylOHe13qX0r1+IbY4R69Wt2vgrM5d2dHKmYvu9mWT6uLJEJdUnasAct2pUvAGGt888WE3zCyOc+0+3S\\\\\\\/paRz2BSCd9VweCaIVSRvTho57\\\\\\\/XamCb6i5id04O98hrQYaMzzLxSNO2yrRV\\\\\\\/u+7hlJEyjJrcjSfnUi+jl0E3OnrNwh45wY7OGtMzv\\\\\\\/RWlxQ5LQcuy0O9rWhjxrrLJIFg3jKTv\\\\\\\/jNSpnxcNGkmRJMpgxTnqaTWEZMAkpT5XB1yVZgB5GjY7pF0vmVg1Ev5lOLHOkebADjmON7XWO5ecFAdPkgxwJSmGLXxM32dVH4kgS0OjJespIEsXaVxkmsHeQGqk6UqQ94tG35oObP5C8v0XXIfZ+0bSOdVNWKIaaslz4WJXrTv1xlxGxD5O0Xx3QWRer1GKUM0LKXJnyir8c2SWyZDQWMruKh4zPNE58y\\\\\\\/7\\\\\\\/2D6C4B6VX8x1ePZyawUwSB2jZtDI7f7jGBJgqouOkDjI0yVBQhf7XplbigXJJdphXgsRvsbn7HTwrwjCMFIKDyTM0b1xtCsNgBMASpJShYhUbTjh7togkdaGkrvezwYkJxUOSn\\\\\\\/C8\\\\\\\/n84puHA86FAiGuhq+Z+HwUesqU0Q63TtL57\\\\\\\/kQsCv9vOdg4f0l3oJ7DxElqW2RM6d7xj6GBCVPZx+i6mi\\\\\\\/A3B+5RC1vrBCpuxqi764uuSd+gYEN2e1jKpQIdr3bRY66Ig9oLKdR82Cyw2bDWk\\\\\\\/Hhpp2u05Z\\\\\\\/ljnQIp8IYpHVcXQDiHrTvJS2oqYxEX6vNncGzRCNeJ4a5N6cNUIWUAPLrr9O\\\\\\\/FVCUSla5IZh341OGvUbgn29zJG\\\\\\\/FieuN0Lq4Pl6e07IWU5Jp7OhtZxnGS4tf6jKkewTz5gitfIFYpGYDvcnotygoXp20XKesxwUs+rMhsFEUQSX2YNiTJCvoCYj\\\\\\\/QFT3\\\\\\\/3tYUYtNy7hQyHuAyeTbLEIOzQfp59r+0is4b\\\\\\\/KEjqPkzJUdJU84Drth+y\\\u0022,\\\u0022iv\\\u0022:\\\u0022L3SG0qvGIlhMXyzhPcmRkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022635c8493a24b9b4a439f8066621525b2eb5bb2ff974c12425a2e2cb1071272de\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nyI3iEc1KZx\\\\\\\/54O9da\\\\\\\/czSc9yvjzIsxKlLwtPVQrmDuiz93QCQ\\\\\\\/8VqLFkHV76yyaJCFfRuPcOdgob8dU2Rstt9ppogjcPW8Mt9KxIj7lTuW0aGUGod2TNmfzlCy4gFUaN\\\\\\\/ibpptfpDI0MdoUNwC74+9RdSpI3t80KPTl2iSp9ju6yQqC2wOs9L+SBkz10OCUS\\\\\\\/0UBtSS1bXYXjzLC\\\\\\\/JQsCxTf+s8g8sSyGdc+5D+gdKMgEkQwog\\\\\\\/LWZASQ8f4TCHGRFI5WMIGnawtzK5ZGpNvWsZeG9GxTnjhNjeBtIBV9hiTDRVRqyZxt3l7VAlI9EvONK+c0PIJegK7o1vjYq09pqgGROqzLXl4fZlTiuMekLOyvleIgQrT4dsHj2BrIRTVfSxQSau5MAH6QCxd36hbZCMxCk\\\\\\\/xeCdwO2J2i+YL4Ffo\\\\\\\/XKKnc0AR4fjnOdAv7vGuC\\\\\\\/8x25XNp\\\\\\\/R8b3lInXryYcDER\\\\\\\/RrTAEzSRbA5nRbcWbiR4auVJDI2LNBJeY9i5rfSjooKDxRgcY7pptpb76Qt2guqhgQXU4lPW2rC6m6orbwqyuBnIJE8tcZ1iaMcjE0F37UT77lkxrfYb\\\\\\\/OfdB6FAv7ze67nYg6z61imT4weyvxMJHHDl7xZaAJ2FcXqEO9xwN7xThLiR9KyQULKu94okuDqN9UIgZ0ibagPSPTlj3R8ttMs6VAJeEjhVmzxlxn0DeLG8pJ4X6oG9ms5MNibpL+mUl4K5nGHLHM\\\\\\\/c+9PCBJHZ14mdRTf+bZf1sRqOKhRD1DOIDgRTOp0LrGep6AXaTZcllIM0oMTRG10vsHJfCSrCx1Mj6vQvmbeJ6sSNMdI99u1pMX1y3cPbJRb+C\\\\\\\/lcSL0fc630d1aoS4vwxNirq\\\\\\\/2YLdNMhDMJ0xLyho0TCFPyXC\\\\\\\/k3dsXqSXw4GqNHWGyOk0LRl4jl61O0L9wneTDXPN3+I64ERGsXG+oSpYuH3Eht4zGci2fmfszBm3RNfQikmgYZGNjoS4y+pL0Gjiy9EU1hTCRdgS\\\\\\\/K6ZvZTc2x0IumDH5TwoPcsAdwTeHNFxjcMyNidvgzB8qy9c4+EnCTdFH6HwJK2nUqERrtHx9BZld5OHKpKmTffYgzRWhahcyjuF4FoBX9CxIj87SDtlzUf\\\\\\\/AQlIKmVqrO66XATn4U4x5u\\\\\\\/liCl3D90WBc9V0UQcraricV0wk65cHo1fK8e8b6wzm6Z+V3QUXTZVVLrFFNimmGdL7qf8Wqf8Fu1vdEQcDwp8HpQC1SV4n7+0AWWW9h6G\\\\\\\/CNzTKCLJoE7d\\\\\\\/\\\\\\\/VlNSrglbAeWKh+ywsejU0ZoeLHLVkyoq\\\\\\\/70c3SARYfiRFhXdmr+pxmMqgok6Lpf1jJRWBUvKcIjPIkPCJl8jTFTUbXd60s7svG5LvqDqoM6AWmySFaxRiguiRBWPDlc2jfawKlGlyyi8MMco2IFdG4PF2n\\\\\\\/FV50FEXQTdDhJUJmfho5L5+C7ItPthAz6nEyCqipdUzcfdtm7eU6nzb9kUnirdM3VpUH1FTlkKrwlw0bBtmBXHcrwlcVhG9nu1OnpBHpCbdilgERyGZxLGVAMAmu3smOTV+aISnyWuaCLOtYlDLXD4JXVY2+Kq8f0TCmJ+BSN+iAu4HZ2vt2685nK+ZcgFY2KYeId987uvrAHVm7vRm1UD2Y7QNMd8BOaMXf5kNmfSzGYqKn6Z8yUXQJxK0ayWHVFsusu+1GhHiBJ34qIVFcVTT7hoSiNtqTo2l\\\\\\\/Z4NB8Wl5EryVMKIEz8rynMF96g0G+M0OgA9EHdBjMQHu55g1in8iQYptkwezHACh8A3xqVL\\\\\\\/smJGFPU\\\\\\\/5Ev9aEHTbmFdmEw6Fi9qb6HzaK0XlaNIgfM7t\\\\\\\/cLC5ySSeYz6x2bZSIm7grgGg3uI5b9nLzJceh68L6BDcUpP5VveBp6bJpaY\\\\\\\/O5L9tnRN2EbMVYgmrAkjV0JFIdzwzUVeAtzMdgsZYn\\\\\\\/7qCz0WvH+okjStM2LivT44C+w6xuoHmMXL9OJWaFBUHRKdHHRsRyxls3F5KGJO7eapUCFT9gF8bwTF6TEwj67DWBJahhjfYts2rizYFknIFTfhn8s4GO1d9r06bKLc87f\\\\\\\/b7gwqFn1UBX882ynXjtNj2aPy4DnBS1EJUCG1BrKFfogLHNO1yrhbJpyZE8aUa\\\\\\\/KLFzExuQLikhdsy5SbxRwE2HWxOJaMH3EQyYZBD7kZaBlrS\\\\\\\/Fz+Ryr4BCtkqDfI9sNELgqvzBs8RI1g8l502kLJrS8S0Lsme28Xuujeft6Ir6GCIQ2Fh6fnCiHXWZsbCX8edo9CV1cqEHbx6EtsXZYZ8odTtVyVdkiRpoo8a9E4kfUvdgQng5YVaeQhwb4ayGX8VsgQIBhWPld0TeF6UbQTLvluyp0BWDgKh1Gms6Vpbk33C3ok4GaUsS55LtfPp1NnIpzfmF38TkmnbGPtVjgbqYonSIm1akzF5QlnejSAeX\\\\\\\/cIxURjaqB8Vb29zqjTjUV6bTSiZwpQ6BBSWAZBOX1oKq\\\\\\\/9VO281NFv02Of\\\\\\\/ukNHLy1uDUX8\\\\\\\/ZQa\\\\\\\/CZj8aESJxhZRiXSnzOX1O6w5MyPPlQJyrum4dPStkwMai+rdKrA7djOaHHFBzSwTbO9El2DkiH7t4fnpwEvip0DndRcV0r1tw0qWPiG4NVLNgZpgfEgrHujKgpXO0U8xjVNYHny+rQmsX3G2a7XT57yv31\\\\\\\/DEgY4weegD26CDtrCIQnZDmKj\\\\\\\/vKoiDA4wy5oFW9OrNG+b7esgVxn3nmXUv1tCV6s8fpNKASHOjLSSKrkR5d2UAUISUSv8\\\\\\\/5PEI3BBSjKCHc\\\\\\\/PGEgl66XiPbZx5QeKVCULXvxj3rb1yKiF0Y2nk6mQ+JPRknxb7JYCJ4oXA34NFByZ4Y3c887g7O4ulsYODpiylsB7G6x4sB+UwmQuHGpc\\\\\\\/8yj12SYtYBPR0EebgnQObV9yawyvsk+NnD5Du2b+C5y\\\\\\\/NQU4cHqYM6EB2+A3pEy2vAMR5py6pvd60WddIpgmQutd6rXN3htrZzLsSL8Cx0\\\\\\\/MOs9KqpfxXoC7e8KoDj3Dw7caqZQPjSXeBXsPicKRtPDaj2BDnlDQLKm0vqLyCmW6u1J7w7GsoO6uVwjGdD0\\\\\\\/nvmPK4qLKTmYwcTQJt7Kmr15yheLu5q9Mr2X7c3+inShsSpM89U676kGpNA6QvzgfEq+movRRJItOZeLhZ38HyqEnsF49mV4p5jFWbWHQO\\\\\\\/GCSVhUT5dwFX3PD1FHaXw+WPk5ZAL8kgFqeEdNz8pCvjSh+Es\\\\\\\/DtSMHt2Ph1LQmRXz1yM4wfe50cN4iQyRYZUQVkFAEwrauDEw3ykWaoLh8Lf4YAQXQ6Q2UqEJcvAh3K9by3A71adoR0XNubvqaNaBx+7R\\\\\\\/IWBa6hhyHDoU3xWWCFp7DVFhXuIrs9x3lh9M0+Tb1+s2MqU9Nw+i3l9qPKwnlWz37kTDr4PfZyp1pNu7h2h9RdH\\\\\\\/90\\\\\\\/y0g6eRwoJu8es+WioihdmUC7viOG5kfOagQUptlYDeYrE10sBAVSw7f5YOeJkdRkj6funYOavbsnbTDsZiFMh4LomSiXCMgcljUfRjEon+R+83X0Jo7+YXj\\\\\\\/m7MaBn3Isw7I+PPpWZNZt5avWU8SyYXnO2LH8JyZy9HFHMlZbyCze6KQxU7TG11QzVbi\\\\\\\/MOaD6pYUv96SAjzovKS37oqXeJFOGFVNKTMod4n+ohXeV0ibxXlSc1Uzbm9wLKOYh11x7iPGtpQeN2wiCGUiqHcHK8ohT0c7wOwTuEcm5+wQpYAnI+p4dU1rnimm\\\\\\\/MOZSIYw8eMl+5L6jeKV8s3vXKR\\\\\\\/KaHcOP2e\\\\\\\/HG1Wednk8BsTsbDvlsXcpAwKBIPwHl1+eiRvQnb0HP4PVJF\\\\\\\/wiwBh\\\\\\\/E8PLwtCN7KRt\\\\\\\/JRztO4mTB2fcPQ\\\\\\\/T98kYgSsvGyDYBp8FFLiRc3HQxaPlgF\\\\\\\/YPPqy+tyLaOmINIPqMBn9mqrcLjJW2RRhM+oduMQiB3yl30ZAVPRsQfSqSmQpQpWOgjxjALv\\\\\\\/ZApMHtzmwIcLC2LJ1tFvDdGCGX4Wk6qy96zCXv+54+1ej9sfI62ciCyG9tFsQ6rLtVtaHdf9oQNrbZH\\\\\\\/7OIiZOA5ecywEphfXShFl9xtaS52L7hWR2DPfMwVUG+qy1mH1VEsM0HBRA7uH3SDU9kY54837LnCor0OhIBAtR\\\\\\\/iawM9BDKC+HZBf9+injCyy8+TVRIKnvn6x5IPaCezCKSKPQTGQkb7KlAJXgyToGAws+FR7hwkr27b+J8pcO4pLelYacxNVKNeIXX5+nPuZGfH3ApvDsjrh4kNG32ATW3M8XpVVGsAJywHrV34qVrNTcx+4NVjkJk9ND9yelWSf4g6yYKBofOxUTRxgkh49eZnzidEAOjtfjxw7JUxPb0EY\\\\\\\/q3i6DlqG2PeT8P\\\\\\\/LHQXuql1WBGiKLWhfTh4Ld9mtvRwx3gG0XeVcxq8HEZEAJe05nctNhFwcwWH7qtJB1TEQRC41ea9v\\\\\\\/Qvk2r1gkICeUD5yiaqA1oNmiLkS8kgPoPaTf7\\\\\\\/XMYOVBFJiAP2i81SH4S\\\\\\\/8xGnGm\\\\\\\/FROz3AtLp8\\\\\\\/bt7gj\\\\\\\/m4xjJpdFnsy4oRoOjkVd8V7sk0eWK+q+0PUsInhJZmZdrG\\\\\\\/qa0lFMa4xPxs+f05+Ctn3c0MeDPRQcIoNPwjh74XuI1b7TJpdZLkFBs0iV7Q1SmhatN7KbaznTq11SJDGElOALlbu+XPkMj5gNuBRUeZGhYT7RVEykDuuig2l0OBI1dKbRBR5std4JFYBhzHOGg2aLv6Ga3TFnJlNlVtpdNVmYc1+IjPDOq\\\\\\\/e+QJdApWbcy5hHfloSacnYTlaeYzAsXwAfTy5vF+xCvxgRqpNPIwvHxvIji3geC7TzXZsRizOArVkM3O0gBwiLavjVhGISTmN\\\\\\\/SKq1MV7WsCgIRv6+utDd+VuB2FhHVBYyde0pqQk8JF+PwXaVqa7yeZXhNXdQMHl+fetQio82Oo29ulPTyYjLxe02Hl3pFHbMt0g0MGP9fwxOo58jwX2MDDGsyf5BZFB3HJOhvuUVYQj1tpSkis\\\\\\\/lj5CEunzmyrEozv4MLzHavdJC8kBoP3wUK17rT2aTDss7kGs51iTZlw3kqGn9UwhwFthjiA+dzusnaZF3slgtSO9ys9x1h\\\\\\\/vRr\\\\\\\/ezBTZuNXMx8V0kNCdRdt6GLw9JupVi3oicsmpOaO5cC31gXVABKw+Co4OJ\\\\\\\/G3LQdDO+xecbaQsp+Szm6GeIVuTxoPmKlUl9EGOq+bTqvQu6di7SuRkzZI8JHL069VRMokl704XGwGwAty6vWTcs6urNTWt3LV7a+GneIqBLlrUkNQvU3AXDWYpd4pVooKl4kI3ZBP\\\\\\\/no6Ka1pGZsVNBNETWooXfqzRZsRylo2P9yK4LwbWciEhxEJhhWBrLOUfFdbRuErCS2C5h5KWURT6P6nBm1QeHJkGOCVvygnh4SBblFa576MjUm0sPKBdNeb7UxJX3JEihk5TtFa6wmIJ6U5AdQ35w7O4iXO2JA5k9teNycGJw\\\\\\\/DdHpNKA4iVsEhnp0ffGOSUKSOn1gB8YcVFjVsWYQTQrwDGXyOo0IaIvwXI596G+OVSkBHYL20at3ovZR0l5esjWMKXFUKtSTpGk3dJiqD4Bj0PbSWhdmCLtmP1svQuvd4qnYDlkbXQFBmrdOR04pBx6ERAmoYHRZTn6BTzyd892YL+y1JeTWAXs38624AcjCcnkUEtEIPyiwhPZ8jRBFvOOWMyMQ5tYTfKVkxTsEqDWA1N+UD3oE9\\\\\\\/78lVJtw0gSpZAo+5YzARCz9jmwnY9SjFLJRZqSrJUXmGKnDECqm4aw4E7My2VfDD6aRb34INe\\\\\\\/4liRkUH00ulNaVW4hoPa4y9spX\\\\\\\/cKzwaZr6ibEmy6eRgb2cJVXt4OdUoA9IbPwAJeL5Ju8AMBZkI1lnYzHthRUZBzzcL2Ewdd3hxwjSE58stqcTtMWve3\\\\\\\/ID162ZUaumoJ+akr6U+6t3w\\\\\\\/QSfflGmmRB8CqOIcS5xjtUeq2JqMQD56zJbeCqZ1GAA+Q5uc9rysNPX+AF4RFirKg2VHr+rUv6HJw33kmwA\\\\\\\/5RLYwZUr\\\\\\\/Vm2NlsoxzDf3Yi7D2Ykox4nufEMMFGCEp4q5aeIqsojJ6uN7PamRJxKUeLrMJ99Lr6pKaThCWx3aE7qI3K1YLfYoYK98qFb3zRd+55v9hSnUljt9nd\\\\\\\/xyFe7eOikxgP\\\\\\\/Ewo+lomQzTj77hJ4cD+3TLFU6yhKbKwVAO+GZZWIj299XvDb6xgewaTRjPgZjEibr77AIvC0KpEJngmVmIsZTwjf9RPtEG17I7ww4kvbtTNqBJpNYvgrsruydnqIBTGDBDBwHsEPOpB\\\\\\\/LzDCeC1e6b8MHrQWNwHLyPFO4UwY6uCrLUK6R1KiEp14Z2N7oELvEqSF3ymLh2TR4dI1T9\\\\\\\/mGSmoGkZyoR+79cmfv7IQ9Yn+ErGEM0ZSF23\\\\\\\/kMq+35NSEGV3f7VgXCmU+HwjzmBxE7rh9cJhjumE6QhuGSB5dofFMMxF1VERjC20osNtZqbJj5MOtRIRwZlg1KKDGCo2hQwQU2fxOzOrX1YRQHYJq7HRKkjYXKnVdeieaCKHBcnGbaaBEXrPUwTEoBvFJuJI9\\\\\\\/2Owc1oTE90sjcj3cS5FglLdzKip3yGh1CFabcKGDS7Qh35XiVE4lLr79odIhkdnPnl0jFc9YOva0FMJGsBh9TvJ7mVgYNZysEF\\\\\\\/j0GyVK0YSStnovqq7CgQh8IXHSMKPAdN81xNa+RklUFPZwsFD2LxuYajdpfusqJWE1wAOcBnwJG0Yj2AIVnpVfmGgZcXabo3QHzcaMEc\\\\\\\/c3g9Bz46aqnh58WICw298yTOcBTUsyjgNv82ThEblkFE8lYG0V6yTOyMcnfUj1C+P0uz+c0keEgBzd+oJ0hlU06rWNf4cmfw76uByMm+k0kO+IZE67Nmp90YI34XUg+J3Zhv3QIjqzD7I8\\\\\\\/C4NZeB7gF2MQ0gNwiTbGqqHEOv+cegGOcW8s7GM3MOabjyJiAdzAwefGX+a4MCbqwQOllPtY1xL0HKbdC00i2CE96uQe+p2twPtwEtqIL5rUtJyeMN41J5vR7UpyUNFsrWdo5qSlren4yyKCwLTPioQ6I7pbyVCt28ru7Y+002KFUBV3G54jr+dK7bXZhj2UdOVYaboC0lAax2QPBv6oRyJm2jGwAVsVVWj3DMzBhc1uDV0eePj6WwvueXmKRjzygIBZvwSWsjFCr2mqGLQNWpC0Br23gKxEy7hcjVrkiCpdHsbK5b9frpvmVU1jJ56SB+2SfGyJ8ZHRChPqhEXgMz2oSIKfK\\\\\\\/r7QpWfjWHWTZIHn1xN\\\\\\\/K3sD0Ga1FGfiWr5Z\\\\\\\/XNvkk99nx7uhn8wplemxeiYvQuA+t8k6Sc30MHL5APaCnPu3ywkv1I9A3tTiHtRV13XjdJ+gGA8eUKe\\\\\\\/5MnSO4NUFispS\\\\\\\/1cO1XNQQkL83Nk5fwqlD2vDFsXdqrwjBwO4WkXrZfjS0gYPUyBesx3EkS8uwDRrUH3xikstfzLYymXHhryCH56YC0O6EV\\\\\\\/14c6gvuAZBfUjU0cheh4bMzEc4ACmQrPuno45PCDOC\\\\\\\/yXyX0WC+LdDUK4T30fgGR6HGgv\\\\\\\/BiIUkMr4RAyXDeTQFlWXrweVzaeZ8KvG6YaRW9SY6I4qYNwgLe+\\\\\\\/QduIwWEBQbhm9+hUW772LefHRn+MOm8vV1QG0lWAS1DyN1ccmbixe4j3v3B86\\\\\\\/p5LIy+KH5+wIiYrP14mhHMBRieiAJ\\\\\\\/2FvIdPUoygT5tkvqFI6b93brbxVokg1jWBToqn3U\\\\\\\/pBuuDXEf7RPM4Y4TV1M+fNn5Z0myxNgjHrlM+dPiTkRhHOBgAtYdjcc\\\\\\\/hHseVsgkR1n6U03Dxq+d0sWLukMV7ck3pfElUm+naPL5Nc3hx7ZfvjBPukXltMonKGHVttiiq+Ytnd5KuItb4uLdvs1d05k3eofaPGcnB7Zh\\\\\\\/aaBFZ0JKgc+bPHqGQo9LJXX\\\\\\\/njojX7EhsU\\\\\\\/Vpcqj2kPhSJoUA9keMfQWbiIN1TddDZVyOosxxxzPaCV5Y1Z0KNmiGG9JtZ8AV+55uTMOFRWCtr7v3wA3sCQcMCSiVYvPuOK3V4Fdy3O0dss2cpfocPd7EvUjBT8BkTNA+EeHYqm95k5dz6SzVmUUbh30XeYAGtJ1c4pMhqHHcrBJgAvywM56vodn2pImnGZXofRzcoIL6jxC1sL5+sz2xWlUykNljfE5oGfq1GnQktRktUFgK0PUZCHN6OXdNK0ppb\\\\\\\/ogr6h200W9IJT6avd34y88I9nfc5+8Nw1JrdHYB73oWLa1jkDaeWGWtKQIZ\\\\\\\/ihIHFBAbHJypBtBmnhCY8CNLw9bgyURdMaM6X8Unli6RVessrZltN1W\\\\\\\/I0ymfwuaKyz21ogZMl2kBGVScmJCPceiOOZnRJSM+UBKrpFJDkXdj7kheTJEYovKSY53f+djnbE1AKHBag2pWtBbRtmkV7Te9YI+vNnOz1Uk+X4w\\\\\\\/lp4sQIf1SPRzl4sxwBtNJEEjJFkKsvjVyGPG7+oiqejfC4IEvXDnZxHWz9i6g\\\\\\\/GY6+nqOGoxyx39sdLt8wOg7ccjQoeRFlHJnCrhRmXL9K1OSHU43ml5Ovw+OSGvW6C3yl3aMx5xpSf77Z+wt4kfI04ErSCTXUmUCkzPs3mqBrOsOXmuL8V4H5RX1WM4hn8qPfgYGsH7B64bMLYjQ2EaJUiHAwxNuj\\\\\\\/n672X+o9f0SfB4ZLncpSytGseakOzZkuxZoPjRvV5LNYXPx+twVGDYkbKDxSp2NR1TAHNrICBEgBfrQXjI\\\\\\\/qlHV\\\\\\\/xk2In7v0t8+w\\\\\\\/uYqlkc6bJVvA1uv4QuaZYm6qcJyTHMaV6WqJLXBpYmORmcHuKOc6HmBDXg8J1ocEwAS7HTEHLRIQrr8fLsbTR0a6JSthbb754ahjVevshFD\\\\\\\/6HpLN6HZreWhElosm\\\\\\\/iN8JVODWFv6yXeIVVsaL4bisQjH9O5t\\\\\\\/rISb05i5GOiREMkdMSwF72dc2tWXG8EocrN\\\\\\\/Crm9kOUNOc840II6gqgMeCZ0TxILJI2yHi+slqBbarDZIep0iZVjNfE6m43q0nIGH+AP3599kWnRrIhhpOu2ypbj35bPUNp\\\\\\\/s2SPDHbCfbiChrJaqAXiMuv2zbY1scFWhlHA\\\\\\\/wt5Fk+4vILNiEgrTAndT\\\\\\\/BTjnZO3LA30VD4YZfnfn8G8IQ4BUgwC5bWBJeZdZ0E5txLDWqGvUyCsTmcbWC66\\\\\\\/tw\\\\\\\/GX8Kl49nBxa2jBCBv93hc9QsZNPcMW6\\\\\\\/r7Rz5IKlI8QB436UA1YvDc\\\\\\\/OcNwxLxhA1NlV1nvmP3IFs37flEtN\\\\\\\/Y60MHfSYzzyc+8zKDDealTVPLMo9xfLoBbVmzNeeuIIWPuvyAaZPGpCvXd9zLkiuibxkyjG\\\\\\\/kkY9Z1wjaIeHMgasIGPMszcb8c277RvyaSVXxVrHNsTASv801HYpBkU80zAX7iCpaFWyo0HZJKppp1ZNrnqacy\\\\\\\/fpdkVQKa+0WDNtw9N21bv\\\\\\\/Qq9ItPWc2Yj53XvyWHERFWnENSuKmlOY2o2sg3JacQRHUFOuvUOGjytdbM4Ftr+EkHK81wiVcQ28d\\\\\\\/tQFv\\\\\\\/9N+xLb6X7UdUgV3ywcMe+cxP6WVRzSCegVfnkID5vaq\\\\\\\/hkyO392RgU1gSgirsTN1KxymCqtEQ6KB\\\\\\\/42bH5O7Lte1WSV\\\\\\\/vc\\\\\\\/hNR8aeINi4io+Dt+2p90B212cGa+ZKoDP1otPBL2Pc8t6JvygO9ozTtnXFsWQwRnrHy+d5ajqV+LRoY1Fg2FPbYjKgcZ52EksgBYqHE+eqOpcMe5stvbVoyoQCx\\\\\\\/v7ubL2NaXK+BiPAQ6TQZ2Kbf2p3Infb0DvKhjrev96g1K0ZumomRARQ6wxJUAAuOoXCSwAD165n\\\\\\\/Szjzg1Y8IV10toVlzQq1xqgCgSxSy1I2Eq5JuwQlDVPJQ0GM9NQBY2VkbunkhoiFYuVI13vk4\\\\\\\/z571QjsB4oS0qF1WR\\\\\\\/lITby3JAci30VVWB3Su0omztW5p6VcThQcHR9gMFnccUPrYojCK6QAErJk3VfiNmCjt+vLMcytTvWTuZhKMkyc6HxPBBxER6+jTTvvcH61U3f2ZyXyjdi3cAC9TtUnB25kf5jKbwOLbw69sOBV18WtGf5tLkt7R5pOQa5Bwx2\\\\\\\/XA7kOD43q4E0vGeq6TOfQJL4JzZx2SFezqm2tBMpbp8uWp1DqAU9yBlRB2ZUYJbPvqmx4Hkw3McUnqXZhigMsbchnkKbtD83h3Yp4ywiZLGhnFhdGXavNwVjJrQOFbwi4c42WL68ZplMmkZWYbsNnSAVCRMhstzS5w8Artb1hiUgn40rPHkJ7BOz1N15fVRd2gWaRqpRZrBAxnRxsOB2OD+0COYcg4uotYk6wZ+Xba5ymPjiYBAFiFWAJ+1fyxJEM30NzKZttL3cfP5ZNl03DD4gBhhLLnFLuDa+vIIsHGidHUpfV1pcqnvK0egMuXGD7w+E4sy7cY4AA2rFtbv65rRD1tl5VOIQ4LcXTuxAqIcBoWHyMpEza6WvWmJa2K92EyxMmcfOtoYkTQdlvICXn\\\\\\\/iJJbyEG\\\\\\\/mrc9iLE5Y0oxMBq7igex\\\\\\\/DnNivl24H7XDWi8ONiTagQawePlw+U0vbP69wqp4IkMAMonBWXuht2GLigpKHnaMf2dkGbs7f2s15v2NaVi8n5Ltocmg6kPFO7mm7Q2ECRy\\\\\\\/SBNKLiaToMe9K8CB1NdcYKPhoKkG8T6yKSVeTn5eVc+BQsPI\\\\\\\/ae6OviHUvFRShAGNkWjt92u+b8mF4WSfKDSJmy5KFgACV0aLY5er1mypdIUb9vAge1T0DF5YhaafYBPwlHZsntGmMk5LmQpi9Rj0Fu9uCGE4jrwyRzsQum6\\\\\\\/RWVdbeorNTFCKbrRsnTDejkQRYrcmz+keWfoFA7lSpQ0Gxb1+73YFwJkrIUOzriBjHYdxTwdhB2eqv2jTyZCN8Q8ipjR8dIRRpXv49Fw1qFpYIkOtFXV2XKv\\\\\\\/seh3tjV0Ady\\\\\\\/DiMpsGM3aEfTd3sg6Q8gAlJyrRgVL7MD1uO48BbCs4T1wLr2LblLbCAttC22BWDN+4D9GobP\\\\\\\/TjnPTE9bgCszbjKU2iyRdFhDJXJQzJ\\\\\\\/QeCNR9JOTPuLd4\\\\\\\/cZOMKa3MMaCVugacbYxmX780fbQ\\\\\\\/6ZXTrbeNGBb16G9Vs6BAMgI6pabDk5M1OkLmE2q\\\\\\\/Rc2VQEEk4a8JfK6k5lICVOvQTP+rhFtDRnwpQ18gIQJ0wsFoYdF0mwOt4hyQFSTNv7UIyALBJg6eUAQwcGrJOEWafeU8Z1WOJnG79ch3jDsvcvCTmJ5ONzKmFeNk9sjKrlm8MNULZvWaQi7VRaWNtHbBqPGdLTcE\\\\\\\/TA3WtSSo5nfG5tOro0JaZUu3qH\\\\\\\/irYXQ8150zr6WpyGK9yofKCau8r5AhtvJGT3F7DmgLfswrGo\\\\\\\/vxUqlOVXjh1sp2MHMeAEgJz+xi2DoDSL2XzYpFHUgfVj9M949In\\\\\\\/pCxu7A6TNoQ9IOUi4RjW+qrz3blkhohaS6r9NnaEZ+EH8jAjmaA0qC5gFk2Dm\\\\\\\/tWtRLk1lE5mZSYPm3OmptNLz4sm6mgFqm3WAj\\\\\\\/tezwTtrLIKLxwIa616OXCfE1oJbeRO\\\\\\\/GLlnISGEYsen9RAejjayoGPCqTo2nLvfDZJsxoMNqXXneT\\\\\\\/uz7oemVfX49P1rkz0J2jhyBB4OWMLXADWNWm6k+pLQMq0HGMp20Ih3JlnAKvnvg\\\\\\\/WggSUbfRMYBLMmCctGe2lqsrWKuHZwRbCLIaGUhUEAMTI33X2Quga3cwuh8RHxH2ktmZrCWBIkv1Ws4ZTXi3UufQZxQwZ9x\\\\\\\/yReNBIxCEqhyDDIgeYHV8bPNgJlwlOSVUucMwdFoEKt3OsO8lm0eWmbydIEeh\\\\\\\/MuBIunmBkvQjwfj\\\\\\\/ztaUQs0oCNOgPOhd8SvbWM3nkCz3tmS1DIv\\\\\\\/9uK3wciSiPHcHL8SxW2m+7cpHuYfPdO2CSKMlq5HtM4e4kI3AsLH+ozXHhgV1dgty9xTferyG7IhWapj1vDXu7brSgtRJnN9mZe\\\\\\\/hsu6s7Cc5nP90y+bzBrYiJgohnoSLgNSRrhruA6tyMqEbOjWhdSDhGh6\\\\\\\/jFjbsX8Q7dFTNfHB4YbBRXHb739Bk+mQGEEsiBf2v\\\\\\\/rCFqUdf0KmGmcYWbijmWDDs75Pw80aZCtyIvxqaeIExtZts+lnBOxr93WDmjn93lBG8LNLRwvvpD8fv4rV4eQZa7SqJ0\\\\\\\/08PAlC3ClFCtu7Cfndru0uOr9qGkkrV5\\\\\\\/JbzcHl+5TIRmnC7+ylQJanCmRdaQMhx8VNCBpaSe7lteLNAtnIfrvXr2UzcZxym4xCz0XIi8fwfQhFkkU2no0ld\\\\\\\/pMi06UZcd0Qz384ApSn3K3cACoso\\\\\\\/rUcmTP6TB9B+HFLKQXdWsqGB9o6fnlJc60gQynp2RX8ot7pMX0NjWBHvTMDvUYNxG557TTVW\\\\\\\/reEiZIieY9MuDfXa4jvgjSxHlIB5YfTY1Za\\\\\\\/bjck9QC28X4+rSCGV2dc214bKlZrg40+4IiLVfs68OBpLQGzyKPY5e3NJAnQ6OMpeP7uy9ABpMOc6SZtJydMh6\\\\\\\/eoOBRlpjgrQOAW34oLAjuy\\\\\\\/Fu2gZuZKRNGkZsdSsj02WIJDYTDxNlasLETHYvjo8YLHbzDADiO+RFJNfUOQ9cYtPDJGV8tXIe61XYKlJaHoLfzUfvT1MtjveasukxlYDcomehmigPa9ZGhDqpNcYFYECmi0Tz3prYYdT5ycdmL189VX1yHrT4pfkUBOxs+6rLPH0VB4doZ6RCwxWMDCP4IuTqMkM9J8vtP0cw9xrPf7CumdkZv03olB+Axkm8c+J5CiZ0EjDWuP1MtTcJEfCHo5xdm5X2D32ZnrSKwSe4LomYnkxWUQaEOPVKEWEHwjXUk2oX3CWejsiMIsUKmv2Qw6u2nWh1imRQ50ZXwNzdl8MTIPfN94m5M17ruKZ6gsxmRPFPRIaobqO2svZyh+cKmaVPtgfRHsfnlK1AbgN1I8PQqtBX\\\\\\\/EfjHq4BwsYz07rwfZXy4YTxA1sKWMF22XvPcgCPJnyM3OoKuukHIPR\\\\\\\/Eo+grxa\\\\\\\/0GH44PR7uaPnrJIFcewkLvEIu0IG8b2yOQRLsOgJsA5c8uZ9iDQhvp1kS3ctjS8Rlj7WNpqFshU8EpkSReUeGGGqI1yUIx0HmvO5e95O7kNQ7sWJngn1U50VfL4g0vwDr\\\\\\\/samdWi6FFdUD3H4b13x\\\\\\\/7GgW7DsX7NcLYyPdinPgcBh0jognRSYrYIooRgAXXbfVjaIa37XICqqbGR\\\\\\\/cVs6cZCKV18XwgogKlhHULeWXvZtbhGFTF6yyKwOX1Od3UAvRBy1efwBDWeDl\\\\\\\/sGK0F4\\\\\\\/r\\\\\\\/1nGB+6eRVQrny3+99UxhMGpaYV+PT2XdekC5e1kXjhDSqXybOftlXtYkSjDMQkWo5IG7IvEx\\\\\\\/fdwk6Y\\\\\\\/x4xBL9lXEia3U9ljohUTh9m0jWHWCpNX4u8+BzihWn7vyKoFxI5q1HSS\\\\\\\/yq1kvLfDwwAitQtK7t7wC18YqF51RXOt1R3u9VylO53a1m7Cg4VXBqTRWKa3XZCLlnHUcR\\\\\\\/35sY22wrUeJLIOq3+pwCwY\\\\\\\/EHwZsYY+dYls6ET5\\\\\\\/ZCEhk6eShQ57+Z8z0ZJsiQ4WhmBO4W3QZsDlrVP9A3ObkT68MtJMaglN4qdbsBUr6NOfyhm2zo8cPWc6\\\\\\\/uigL9UQs1ego+ZM+5sRJDA8u6e9ilU4LBjYlXi+Frdz7DOjDLeruXve3kK62hjMSbgkn8gMxoiL7eDHC3ar8GM8kWHc2fw+WBhVQxMvAYxc3PzecZ1JVK7nPhNO4Aye1sdjGVUIaAaA2gUQVCiwDRoBckL9HpoD1Tcgj3ZnidJQAqGQguPmCHGnRj2gJxKMlchAS9uQ\\\\\\\/J88EM5sNG2Ix\\\\\\\/ERsn3z5+\\\\\\\/XuOugRaIqoZNBNfhOjUczvExO287YFooRYBiP24lcJjSeMedSYICEOAxONovmIOZOvQknxHQAczHoZYHbldHuoyelZSDeNUencU5m5\\\\\\\/aEQ\\\\\\\/4dSiQ6D3XMQMS\\\\\\\/Jd5wOz4JjOcgALDl2QYKUWtGR6Z4VaL1IlLTqdaPxPb57syjaI8yavsAYHpXtDmQyry9UZ0aUkzqPHcxkyXhZHwRQHgEI9PUwcHg0tpAkGxwfGj8\\\\\\\/6vu36Avv3WHW6lv8p9635vK271fYzJTnURkMqd2QKHcg7weBPGXUjeaGbHgEgrho0GAErhyjt2Ms8528k5NfMPYblnE2XV\\\\\\\/HCLabB0EuTnAwShX+hRO3zg\\\\\\\/bcSN5Qocs5HQfwNwEHGx7hK0wwfPlWoEHsm2L86n2B\\\\\\\/j4B3yzpHOWuTYX+lJ736a8XlJ7YqzOXvUbyxly06lgxiUoqH6PdF8nyA1oVySTyRdmv46OGrD8jjWvVxCviB4sczGLN+eGqmcRbSmck6PiTcXk2gH\\\\\\\/UIwZkVRwLR8nEsiztMD0E5\\\\\\\/bWYdYPvIlgSrmsm4lVAZ7UXI18KQsKr9TOP0qY2S+EiuMQvcj4YaJRuQxo4bW0IcfqSLdtIoqkD8mt2SSvuqobbvu\\\\\\\/ScLnJfT6dbFjNZ1Wq038YDV0cgXC53pJ1kctCChLBvoSCyHB5cmPyWIUW2mGkuuKpnpmkCaRqisGNALG4ITFADXIYxEFnGxYJWMmPaCVtzLUxBaBgH51fPrdnWAUt4padMVY6baPPfSxM5aqNkdaiesiQSdjBbE7L\\\\\\\/dd2T2VqXlxjOvGW21qpOrfoSMU9SYySlOMVDAF3eZ19Hu6mR+\\\\\\\/h3MKFbVlIxyOrpNG\\\\\\\/ybi+mss3G6rootq7MvJP+SLLg6fdlcufehzBkEoMWw2wtuinZ9qx6BiYnkMXkuORcHpHa+A1cL9EZSn+z1pqq2sipA\\\\\\\/sLBrd6CdyQaN3LBlu3RKL+iuX536bcGRPjwo7FuLla9gUouM3+aEaElRq8ByoQlI2qq92K8c4qs68CE4NmzuWd4TDsij1H31mzjNW1ilHTalmAA5RPjzlkmBm7ODwOPI\\\\\\\/cyRbDq1CrzGq6Zk6dxXoUTT8zykbMgwCuMV97fSb7dKpLCedjBnnVwUL2CpmGVrkxQ1rv7LCA24zUtDm1HPyh8TP9jheVFDOgZW7a5ZR3Jy3VYRZXQLGvY5L8GZhtpBx5ERlrdmMmcHNulO2gFtUkBZ4em47wyvCP\\\\\\\/24WzH3ZhXIXoUEn3KqY++LrgeaCW2Byly+7qEo5QyyaAua450rb8d73rYPeiMOKxVfDw45NDcPa3lXHFAJAN4i+5M1cfnw5mi6VC7xEc1\\\\\\\/zELSmwaEyaKbcTHg3q5GxuDU34ARnNLETJ\\\\\\\/hyfDln6wjxNjmuhMKLUu7fwtxDECuPqj6d2eTOdaWoUU2LRdR6hXDgXerc9V\\\\\\\/LucolI5KahTNved9eke+V9Ou\\\\\\\/WebMwwNqwDZap9c3wsCxAIlvJI8P114jC\\\\\\\/hphM5bN128SWaTM4zKE\\\\\\\/zcozx8yKhgWM0UkqEJGvNjxi7t5ZRu6bUpvGunO0wKjWLucxo0wcjOvqJWV0DGI4qLdygAvz81RgPUe1SeSPZlMUiOXHde+XG2h4wJBRHKo2+uvHuLu6OzBkYTPOftVhrDQXQjgbrQGYWn4dPL81o2a9\\\\\\\/q0HnS3gSqWKV32tVDVxrOwp1drD1Hf01np48EnrUntolzHWqA+eFVo4tkg2AKZHjvq3J\\\\\\\/tZu5B0T6LpyzX3wLXtB60gC7UuIWwLxYk1rMD5vUDe\\\\\\\/qaVY640aJeosnHzRGqMO8uodBIkzJOwcvawc27humNqAD76UpNoHepr2R2Dmk55UpA1OTtMxG\\\\\\\/7\\\\\\\/RV5hxZt6jqR1IOgZAskb5wLVVx9fUVTJaEMoKHdzlIfknZJp\\\\\\\/0YuYqUP7r1PzAQ9G6qXMDxiAV7HYHSOlLK2It513lzCM4IpnPGk1tUkTBDzQoW60Yq1p\\\\\\\/a8DLcQrXA6s24H9XDxzQj6D5e+DnFOrPvP8Cyz93CxFrTRaKMHa9O9+kO4antMxMuc8J0wUVh6VUrYK4Di\\\\\\\/8CwdZKFtikVg5jNVEWImce8zixp81g0YHJuPtGThGunsVpnap7HdmwQP32uEApyBS+vJmdgAt0oT3q7GHZZ4TkmKkmFD+EJdlpkD1quvjVJy13uByN06a3YAfvJNpCoqETTrHNZgejy2VVWTrB9O\\\\\\\/IiBd5mBiA6Ec1kzfhPaTRWlhfthnh1MOPLr2LtxMTqEbTDgiButScIhJB3RkJRvrs0FJPqGTeW3e\\\\\\\/HH+yPBS43aAY5HnhuOfNMRYL58DZLhKZwJkcx1mlYnwrEvwbMLe8ZyAz6MSj9r53x1xXUv74F2fvkHLEKGXUbBis2h74KlZLdu8hXc03qQGeI1n4lv6wYgagoZ5Xxt5Top0BFMmPPv7cFi1a1dQT9K1TNFP3HtWe5N\\\\\\\/oeRx\\\\\\\/PQw2TRJYhM6NFXXRpGvxhjaloMg\\\\\\\/JLVQXHewIUo5L6r+hl2q3AceBw9HqF1PIJBGzS76IEVtPVUxbUS2Lrio9I\\\\\\\/TckAj7jjqpNDZXWqEvjiu3LwO0bup1YPgAbMKIJlD6NJV5\\\\\\\/0WAtqkMKF4HPCiC+AlZaJmhqkMnKBQIF61Ji8IhdBJDjUVRwFTI9RZ9EuO\\\\\\\/\\\\\\\/ngD7oBdmsIXxFfI0RyuiGz\\\\\\\/1HjjUY\\\\\\\/VNnQzYAYp1yHYej1Bmiz03HF4ZLcNUCGB0sRnI9XIwq8bVhTmahcyJn1DsrginAjO1oysV2ho5RWhuTR5Co\\\\\\\/IthtfS6CY8rq52LcAJC9jCpQtduZ+B6OWv9Czv8vMdt0y8lzksemq4vP5TTogjBx\\\\\\\/4ohQwrbvckVNzGO4PQLCDknu7yvteLhfp3S2Ine75A86lws04oRLGXTB8iiXPyp3i7It\\\\\\\/5A34ZknpL2eW0uGC9d3b5BkTwYYzB54yeONGSoM+PvLE9vwbZhja1aSI0qF5WV2pmfNVxMoY2XhmcVOoaC6uyYmLxt7V6zDKoqal87MN8hNWDXUhMFuP9uq+o9AhxuxKXPgHphDBAhqStqCMZki1ppd3Y9+Ez2XEPMlXRBKjser5m6bGnTaSxwSYXprBE4bBsXdthckBNLNk0uwK6CPG6Iyh5R9aGR98qwJpsJ62ykoF2nN6lZia772Xws5K275vRTG3Gx2aG7pdCCs4CdEiThR1SnFq3Vneeddcpklzf4DL1v+1a+Q6zwHDmKaSJM5F1\\\\\\\/dhovOzCBONGWdNyRZ8kTvnuL5gV55CdjKsjUbHJYQROgAsRaIT7H9PFe+2YhC3muAB13ga\\\\\\\/hi86Mh6EYFb7yDRCa2dINQcXCyP4+pM7kZN0WNaZOlTUGo8YIN7JdICin+QbaDBCUFYFI2XrVy5MUHnUSmFsbMc91dV4ZuB+yLJh9Q1GZvJdAFSlTKfAtPINY3YBUwddE18OCY\\\\\\\/9epQOSZ\\\\\\\/XB2qbygmmn9bo0QGvkfSYbP3EX7GCy0BYs+t9kf7bnf633toXj98yyv\\\\\\\/YIc74NbmF\\\\\\\/7\\\\\\\/EqYDBO1AuMuyB7nQaJt1r4R3SarD9MuL+3mWHY9Hv6V99gqEE2+7dRdsMNlF+s0Gfg+7tiqQe4+5sl0VG4KcqUbj8OAsBR0yKkqNmMkF+8OGHC5SPTqIH4G0hVrBM6zwMjnojUubMxeiYZe2TL4\\\\\\\/7RhfAWcCiuMsNbLPGgmstZ6tymIFyHKk9P9C0lz+7jih+MeNINvjk02\\\\\\\/m\\\\\\\/7UeEoEVqUuplN+Ep1DSr7FIDuLgUnXGgeU5RhNykeqe+GJwGeOAe55r1ixvA1A4h73APOF1E6GAdK6+skQIP\\\\\\\/MP\\\\\\\/wmC46T8FxkiNzgu4GdKclZbN08d21Dk6M+KLt+bHCWvVXe8c2+xNwXyV0JhAOqbMbwiQuZKV5QpgRPJFTTzw47Z3nNBtGRxjhxrLlRTAb2HVFQyK4TXP2QZEl3UAZuwNooJhTqgbevi+5zOKPXRnte5UgCRAlLdctkBm9fCjQbiDYOxN3famiFfHzowoV3kl+6YlVtH9FdZjCYPIEuMtSwk1mYhn8R8n1Vwv8yc4c3NtDWaLOaPLwz9sRK1YkwlVRMUOFlvVCJL1zZmrV6R41U9g+s96DX4L6MtTBJmJyqTSU8agKNzGLYudmAg6p6CI3kiQtqD32gvFZvj5Ixndi\\\\\\\/dLUXh2pSlbwU9ApZ+\\\\\\\/6FR\\\\\\\/uGI5Y1ZoYxLilgIJky1tulsPxvWMt+SNaThDpL03fXb0uwVX8CY51N+XO7bkW9Q3xXHH3j1vrY3lE+dW1XnkiipAvInrbJkDQyppktXESnTb4XRq61agdP8s9bukTVjGN5kwy34de1ENPp6xt1heWw6HDut\\\\\\\/iJ4uC+AtpnfS69Q1njTwo4mEcdBJRSSl1b7S7pttEek4aFQht\\\\\\\/4SSt7xUJmPfydgNUyricxN47SwBhPx6AQNp+BMM06RJHxKjW92kestI5YB39lsLD8lSEdZNiLOyMbGLWkWVSIzeIIkWp3VDrM3MezMxIPwzlhfNwO4I4pZMPulkR8\\\\\\\/qpD8TySq5KliS7Ilm6OyyAisv8GG2IBGYM9Ibrk2Oxj8DumCLZVC2YNWo+vgkdBVQesE4ca7W60nJMqylgGk9th9Ct3sUxxZebp2EEFefWp8TraqdBZyXrTploWPveKxfmGf\\\\\\\/ZG\\\\\\\/Y+SIQIQsVWTl54yA0gGMgg8DzcOGDm3ed6POtghCfM5Cz9tjMB4BOfyizbOeOoUjeJgY9HcX5WuDSPcILGueIZnWbfNvrnhOCzWd5EzZpPV0Uyxute3qSILo9MFh9vNziz8aiQt3AQzylgCKT8zL3DfXvhVEVTNBCzeDAe0qnm8N6oWWPdCqA1XfoGwoccpjE5uqZzXSJqgB5zsnRSSiGsRLsBllbzdRFEoEH6GbbbZ6uRRnjMpgMDxelMEZxE0zJ9YdOnHXg1w6QFXSfgVnU0BxfaIsKTAm0\\\\\\\/qOepD1zHJPl9WLWcj9XKXHvmAS5gDe09iTGFwJ6r1neYtXEymzi58uRnSfavsp\\\\\\\/m9WC\\\\\\\/uX85bUiTj3dVle33V+bKo7WNWA51nqyUHSLIK4LN4hCFpOC\\\\\\\/U35NFi95kxCYyYhJTZMoSk+aLw73RK2gDP5M7KyusyTWfo9zijW\\\\\\\/Dm+O6CTmcrMx7Jvdo+Vkpf45M1fFswyBBMKrzBXDMdJ2DbQ\\\\\\\/kMFuxMp4A1BQn4psnFtDbM9Qx0l8EB+jaAns+Q0v513JcgjPR2QgfuoVDuQAHQq+TvXy3pRmwd8TajY\\\\\\\/SUvXvjQh82WSbl3pI+XfnJYDRBj0uFGcoz0sLwbx0uKcFwj8bBp7Pbx+VgLolhvI9fWaNiDxPK0lzYCLKMBwvvACntXyRySaC548YPBAKTf2cmExhMPvwLbwhYs8HfhiJKs5ra004Ehz7EqUsgA4MR\\\\\\\/HGAljWP1YstZ5cD1CJiNetosjNr2j9r1s6LBg+XH0pN3RnnSptLffY6JgOfvD4I2R4qDp1Lj2z2KITwKIhrsYbgEr7Ho2jhcb95fofOaipSQAzXgkrtPFuLgURLi9sEcqFArPo0vfpLXC\\\\\\\/16bCSx+5nqE+C5ZDOuwSVGKQCqiikVMo8p8uVkVlsw6hbG8edn4JgnotFjoXpPJyD7yVQbqbeztV+KLeu0914HX7RQCJkUQNZXKkj3yk2\\\\\\\/SBsBinUlEhoq3ujvx5zHzZq3Wdh6CMyhtQy++PEui88k1Nv+4yzAWZXM7wm0AfFSBdRyMzLKtmWmOx9RY8AxrTVbEHMu5m0Mre+0wNo0S8+qXo0QZyddl9k1Q3ziC1HKgegJ1HklmoBR5T0We\\\\\\\/SYBb7uKwCNViqL8e\\\\\\\/4mKef1iQpcqR45S15dT2sOvQA2Dv96IYSAZ2goZVm4AOFGLw2Pij2raX4PcCnIBM35eIuudxUPFMYMw1NsvoQtyxivJhOZiYwDgwNAlgk\\\\\\\/pEV6EnhPNtWLmg33AYVEh\\\\\\\/cJSGNLqZ6mwpFIP3Gfv\\\\\\\/AkUwyHn1d1X4bofieVTk\\\\\\\/4O83QkuV2BfmL4\\\\\\\/E3Q\\\\\\\/z1XxdSj09cFFiYEn+moBDKu72aHabnTm1dEofhewLdK1uZLc3\\\\\\\/Jlcthagr2twfzlNC4WpCVLQh8hZRRTHfDhpLrCLWmTuLOCNJ0VdwQaJxULpgaYKIfGq+ycb3e6+wS9qo+vLWx\\\\\\\/ufyW7qAFpjWSW4zgp5pyxQ2CO5T2syo7nGJEVQXvuG9DuFG8Ied12cAImlhMEKCCP0KVefXgk+6qxuI2F3d3THdsX+vVnbKEPsP4h\\\\\\\/OUv55EC1IGb6yILY5Um7go47+z2BrIKEe\\\\\\\/7rfyVHqnHs6aHvAO85VXE8o7oe2KmaFR+sA0BC630eKdTjB9qZk48KC7l1IfKZaZuih8\\\\\\\/ayPpnrFzI85NZ\\\\\\\/g+N0SqFV0R4jlyqpzaJ0iB7qynMbShDA+qDdQTymNPxqMMIQ1YgPpSX1tOkgMHNILin6mx8NgOY46svrwF3GpOup0emnbyZFecRJeaclmZx26v9xAJFzb5I02IizOgbV8qqyl5mvAX1uNheqxGft\\\\\\\/ubjBV5H8cnvTH73aWs8frXR43JXULmx\\\\\\\/3Oz0jRGxMZzObloc3u2WFc2gZWHKDNLbYa01PmSrcNZKu+u5H84UNSpbuPoaoY6WB1hSidwG18J9\\\\\\\/r4px\\\\\\\/J+Y+7ddrkqxk7XnFyNw78betdKjv5WJdcI82OXY3pMep4tIrm1kQK3u6UfhbyRY\\\\\\\/3Z0TQPKHxwggQDPOQ+uoBew+B0uYXhWBmgeGjPzyHNbB2TJf6ngSe+cm7b2OmM+NeqHgHbDiHKEzEF4PSTLF\\\\\\\/FQYWTMepJ4NziWhl12TLLkqJh8C2Rs3adBPY9HI7aTm3X4RK2xq+OWV+\\\\\\\/iGl48Pt+GLcJm9u2TY0Di6U+THX98KGvIQk1+l6OEQS5iWyWSVNzJ1kajgWqlE7CM7ERoR2OQxs\\\\\\\/qwLquxrGE7xlJAfZ6u3FrILcv2dIU85zDwB1o\\\\\\\/a0EOFC0m2hJjcHdXXGO\\\\\\\/+Y338nqzvdvVo2stj+rwyt41vrx3pLXFnC8sjMsNJVeJ4Q\\\\\\\/lnatDZrWOGXruHdvPFOzSYi61wUmdV0Y4WZ4e8y9g+R50J4cl4hdJSHX1huTWpLBOYUrQAcY0BigcX9MMq8\\\\\\\/jbgt8SSJmAyRfxHn0H79xhOC71j1K1Jqx6RNfxrrxbAMLagFqqQh1nimSeJSukUBbELWOTa57AWmMuzJ7EFys8fF6Tuu504IOy22GLFQTbncZQNOEB+TSXaqvCIga2nZPzDMdv74q9wJs3z7qlUg15dPkecDbaAthBZoEeyqDMH74aUBuwJhmeFKXin8uHbJKS4Sibp6SgwkZogRsaJocipcabJe1v3SlFLAS1tHd0HDlLXppwmMIBB\\\\\\\/et7EcfUXBqtpTNxIUX6yTJ00VyoPAfXEsGsWyb9wPkE0Qj4n3O4EFSO4hL0XtfTSGt0NjZmq75MPJO8vRU8dh6ZFtri3Vn1DBD9PEcpVaGsFPgDCwVCsGz1KqGYaDcQQF2soORkqGknBGwuC5FWkNn3XRnJgHo6wkYtswjXVECphIgfsHUrsZKFmOhxpk2+6WLGgH5Yl+7aM1cSdf6zapuPM2\\\\\\\/hqH0ksA3i\\\\\\\/MLdSR10oMtcQOHttjgKDNCxAEXs1mqTIw4\\\\\\\/lSKfHXf5+fNxWHPqVrUq85drZYE2h3J1stQs6HTseCRkY4EzM3uzpagDXZC3xh9SyOBcDywCJBDLnPIU\\\\\\\/dCq85wfC8Z8BGjrYLIIKHDAAMxEFI5gFY+fjcprAwv0LEtMwJMsgvQCy7Xy8VWOkoxwRoRx83QYOXGP9e1IeO6wbm23HJzRwZI7++QQ+7ESyzdBgPBCu9M+kf0E50+CTkOYrlGZRRV4UVTD\\\\\\\/30AUEm7r1YFJz1oWN8S42BfGqeE43dAgI3\\\\\\\/qxsl1bfUJ5Ov7gijDYwq6rYOUsVECXBDk2UBBiskUOr\\\\\\\/HHWLT2wUoe+WL8g6\\\\\\\/gMSNOjkY7EirNclFZxXymKUvZ1mHjJg3n+gW6dYeqqMg8mj9DgTR6jrKkKrW+hOKx8eXZvEKjlQnCbtkwmnv\\\\\\\/2ouJHmFWuPrDv\\\\\\\/7Wd5fUBVZorxMC8Zk\\\\\\\/bIOLKB0jxUFOKP8Ro+bpmpMohSOErqV5wmmAPg6BgQOPHPMdbE+xo8tFZDIEKwuBrfn80sgJDVPhHJZRidRijZNijHrdUj2sXdNqVEdvF02wcvt2bN9oOvOm2rt\\\\\\\/AY\\\\\\\/UWAkhDDbr16E9QNhUiCmE67BBhmQsq1UVGEY6rNOenfeTuSKZUvaQTiioqhz3O33wDXlkZ\\\\\\\/v94uIe7Ijur7wQYcq+3fazUq7\\\\\\\/OXuF5f07O09\\\\\\\/YM6Pv+x4nLK\\\\\\\/Xurj6rxUXnJ9USf2glDrjb2m06GUPI76IIIH7pu\\\\\\\/9oH1U+yp+PLKyDwt4MMj5eoEd1ca4KbnrldfrtgTwTo49JgQ7caTcd9biyq+OQSX8ZNqjPUuX97JYh9S9UYA2XKGYdjjUVLuVt+KDp2CgQ0CZrcnZZ6q9XISgapaZx9y4B+H0SsiAR\\\\\\\/GTWwoJLTAURQWD\\\\\\\/v6F6\\\\\\\/lBhMPotUrZWWWOYZ77hxd0xDUYI4ENQxK7l5wOy9HHeaNKI9rnsY0+0sO\\\\\\\/IYJ+vB4JDRKrCLTceNlu2zdmpWdXkJvTatOhwKR6JjxLHLopW7frAaEFYgeEEy2NltbXAUR1A+cVHXtTdwZVqnatiouK3sIZ4CCWiMqjcsO0pczdro0KSPYH0IT+pLx\\\\\\\/4t8IjXPfHYpEGaazL5GdUybE+pZrf02htybnymRaANsXjD7oWRalLhkol0ZbQC67MAe9p5nkBv239dGIv1HaC6DRNL7NsNoX1UOP5R9\\\\\\\/lsaYlo9C44hq4Rmq8h8cZEKNsad9sEJRTT\\\\\\\/Z9jeStNWTsZ0DeyRLk7m0mhaedMuxHEo82Zs1vw8M0ya4tD+xY5dtpqME6\\\\\\\/fwoJsmEzHFuZ+6hiiNi\\\\\\\/RRIHWNO0GFYFXFkqVz\\\\\\\/gvMUnoS8tnSTl9RLQPIVak+gbTxPMckcC2+oQ4UspEhHSBVQywmfnZXq\\\\\\\/\\\\\\\/kuCnOVwLuOqtoDRJJ0yrufMFWbPVJd1hLMsJkRPuePOCxzGx\\\\\\\/X6SdG2r8PzAuYmaDCvROFXnAo7RfvDtj0RlwWIJaPgXcg+oCwJXsiB109O92PIEOFT2unzlc4lQfJsETGwOqJjSZPo9S3nQ8buY0KWLe\\\\\\\/e0A8qkt66odYrsDSqCVFF6YLXJ6PpGj8i4Wj2iBTAGHQJpl+YXxgmFtr6fRUWSlePMJaG4j0EhTJe1MJHM8d\\\\\\\/mfmKwuW+Xu8LUrXj5MnTu6dYuFLlXb4AXtiHonTI8K\\\\\\\/PILkFeYPhIjF1mOxMLBKcot6DCCLlTvwbK8YoHXc3Kn7pyODKFWapJNBNuhItvb16hcCfV7p41kwCkjQ0o0z6uXTxoz3OpY1LN8Y+1dBwrI8vGPYWw8e48vOs++QoRQCv523UOx9zWGBcmtowsF6bh7hkaQHz94RwVrvrp\\\\\\\/rHbRoQtJka+ft0YC0uvQ183X+enI6pszWmjaYsmzI3eiB3\\\\\\\/ELYgPAXMk1N1yKfT1RzZ0Bv0ylUKAmd9LuXg2QgDglBVT0Vw8JfOO0sQ9taeQDLLBQ\\\\\\\/JEQBZZEq05esLQivVvZYx4iKUJ8IZ8\\\\\\\/IVaSiLkxl0lGooKE+VbWaurR5Ly25GrwYo0J0uiGqF\\\\\\\/x+RxBfIaKkSG0Q9QJT+\\\\\\\/wA02JTSyBHPbqpBRAGszoGZ8XT5zRTRVM3\\\\\\\/WV+jVN0goMYE5M8rJQFDPSz+82FFQGIQNhpHsOg2EEEv911HZtm8o7LUApufVFKIwV0TWmElMM2O91XA+SZ8BSmtxONn2sE77sK1LUPRUpMInh+84C095mCfZHDS+skqexaMcDTUYqdv3JlBg1r4s8npwWx1lTRVno75MRa6DI\\\\\\\/rXrxERQPUFYvbIZiJoXgWBQrMf1N6PWDtLCXyizH\\\\\\\/dm29gKkOOBajbMG\\\\\\\/5XARCRvPDtwNxCaL6bsgKsAU0okOiwv6tMc\\\\\\\/oeEf4EtKcVuLxBIHHkjMJkIRxy88rrz7sb\\\\\\\/c0KNXwPn7hXHAYxxYYGUvExTm9XMGJomPHkYHBNGHF++WV3FQBcXqoZOK+B65W+0lpdZ5Dffd2nyxrEJHQegL7dYFR1pk4+j7w7jMcydUsSRZfMhM8Y8GtCHq4GU9D7D5tMnQmDugJ+ta\\\\\\\/zwU0oo6538WHsJBkgsBU7VvZyhK88N7DDtS947dZ6uEcfd1gi8hj2uTl0GU\\\\\\\/GVwu+F5xQl9pob4Ocim\\\\\\\/\\\\\\\/EGJxc5qLffbxzumq6tckNt\\\\\\\/ywxtF3s9p4nrlVUCeljxSLvXjPVvRVr0xP1i3T6Ncs3CZzvVmvlqwNmc21fC06K4mSYWsOgmJk7Ku28GyFBucKyA\\\\\\\/jm1rk283STwQ1sQxB7cTfp2e6BjhwQXxawFH21ziItNIViymNIt4uNWGA4YMhz6NbxxJwykkXSUe21jjzmPg9PfS+YA+u3aaWii2K61ekKN34oRrOyZMA7ZIK5rHAh18afL28JmXvBmp71+9cJkpLbyybc5F\\\\\\\/DxpTaY1Te0otGpVOX454yR5Y07Rp+RFtqhjuPy005mynCipiNlTt7co8i5FQwVe3dTTcgfyd3D0XZP5SdrdTVeM\\\\\\\/r8QC\\\\\\\/kqnTw562S1SnRwEmTbjQvqXtaU4v8XxX23Qlo+NHlsd9ZDrdPTXUdKPQjPtvoOoqrsRoo6dOXMXH3vdg9zPUWUTHNdkpskHCeW+P0BQ8nBXWZY0jvtBjqLYkNSi9x2fOd\\\\\\\/i4FSbEv6ilLZRpMDxBMo\\\\\\\/bBW9+jqT43twCrmroccV4+D1Ovd3nkloXGcCm53T\\\\\\\/7nYuXMKaEbSzOwkSOp2S66vo4uskFfvcpfcwmqIg3CZM7YGlm1QYJr5WaYYBHhtk2yCXfhvGquj5MBi4RKEWFdLRuuyq8z\\\\\\\/NWTByWrt8RW+660Lx4ALrJDTV+hApxw1g68EiJEQA4lcgGpFdHhaaMT9uFKemXAwF2XZrcZxqT76uBAPM2\\\\\\\/G0ihUpmi3LgdS2AuPzFSSbhUSG\\\\\\\/DjcoBdowA0psXDVu1qI+naPiEb04N29\\\\\\\/TFAk\\\\\\\/rN+dgCXR\\\\\\\/7qZQvQtr8OnXcV0MMfAnQSDSCr2n\\\\\\\/dqccCZO7WvnrJQdLDgey5FP8QA2J1VsZWR0cWKJxAahOOztvoPMf+vaCvG4ZZg5tvlAveqkdck7K3cYofbnPmCc+oF1GQA5A92FqkeI9Bv2xV\\\\\\\/u3aLOMuBtsv+OL5DCHjyebdoPx9n+\\\\\\\/AAvO33xTC19+5x36qot5lMrrP3s+0uJbI0KvNSF5cm3R9xKFwT7oT9UZ5atcm2JcbNYkdkD5dWGmZYoTtx5nuFHxZohAvVYiJg9ECwxfu7pQrqexBEVhIdrtJsRwpEqQVINhueWo7eu3saPrli3F77d1gDbtbpBnmItRuT4zrEHGaNBMJZvQGKWFeNrMFUY2aj\\\\\\\/+vHr2HaEBGfq5hmjE5bDXiOkGQ0buM7FgyaNZYAIV7lLxW\\\\\\\/Ul4YdHKm\\\\\\\/9ZzJ2sjR4J8VkT5DW3Z8YB55mxob\\\\\\\/iUokBESfxvoKLe3EntBoGFNS3zOg82wFxLkONNBqWGJ9CaLv5MNOXK4O8Jj8m+diMtpGy+B8AxWvw8RicEws6BhVDsEd2Ji1AyH2ZP2Begnpwx7yj1HxOPDZ3PewshCiA+jwYSOsq\\\\\\\/jFyomGsPbMvRvGP4rqw0RrVmdN5rfgm2gMpsXulNHlfOaZmRjqwgHfBOvwrSfhhaYy9mdy\\\\\\\/UjwxoDAkkfrD3XihSZXEQ\\\\\\\/1+8UTEFpQZRjCS5gXq+VkR7d+1N1AYlPCRYM2fQSMtZXTq0MAxra8+hIVN0cWGeEw+HM77EPf9R9QmhHo4EJPZZpbugRLz9w0XglEXYvj4eNnRE\\\\\\\/fJhhT9x6WJtHy1DcinJqTy6TgB87XhToEfyvJy1YK1p6hADPiL9qDI3NOf9XHOdFABT6gEX92ccacMPkii4VKtDGaHpW2+cS930DAZp00bDGAo7p0LtHqCUWq5AmuITUbPT7tvSTg5GuoCFgdDyIev2DBY8CTDByJVjhlmN59mRYOaNk+FaEA79bYlDTa+1Bipc1Fh7VJLTg2CFmMnRN1abyrNNqEMGQEA8rDas+MfjdOUzReHAOvd2s59j8ihj0mfFB\\\\\\\/Yx8SHrMDqz6W+dP281bG7VDuypiSf5jsiWe28U6MIx3r2YCLbCOUghoOs7r+L6bkFxhaFGBub9uUZjpc113PaIuLmsCrdzC6XiiqVvK+27tzI1tPzX876SWEC4kKWq9HWo0RcdRcZ0A0GyN2gVb1NOg6mNBt6oD73Vx3qxyw14dZLorBObLWiPwhyg12OG9PDzrbc22\\\\\\\/+C6w8tykc93G1RgeXVyxk5R8LUoNmCAc1lDKZjLpPDkGB1FwwD343R\\\\\\\/K8b\\\\\\\/9Ew0nrws8hghL7NRTHcHECkV9qdxa+dlTiO7X0SzO\\\\\\\/+P4ka0IAllB3XfYmSmq0MVYequRfZebfXT\\\\\\\/6DRGt4LtGDpEsOpGfi87tHdNkf0mvN630i4DxmdsvimUYeYcVzP1oj8vIkwuy+82EmqRZcFq2gdJEUxkooZTpY+3+qb9hWISEsg4ba+0uw1mgH4OvC9geEasqtbqxKd2aLnFpZF4j+x\\\\\\\/azGRsPJBHmtYhiI58b0Cpfvup9MKpYbwWD\\\\\\\/S3kydf9ONzs4MkeFDwyqYYdUxxgF2WQsPM8djx+A+ecn+YzzSCh6oRzvxRtI8N+WXNaq1s0fq1GfPB0EAeKMvePubYh\\\\\\\/HA0hJ88Pt71GGKYR2RiOXkruTEGArC9rT4X0Ewy2HX4MEDTJbXdq9QOFloaAeXsBOzXwXJlVGGJ15Kxv9YZocvf0p02DXzFF+jvJfueq6axg1xm4DPs3SFGVcB1PlLlDKF2wnRvsczfC7jHWaIZGzcwt7bHm\\\\\\\/aRRVi6ZKKFBvDfRlei+QB0g2vt2sluk0xclGrGXW+nCy\\\\\\\/JeWzMc\\\\\\\/hlt86qagrz2EoGchwJKC94zf5D\\\\\\\/vLpDN7Bb4UKSuKiTLDiVlYBAGuyvlh4UpEuWIop9i\\\\\\\/H4HIum8UfsFuSi8qjMVqk65FpAEN1gvynwtzU+crOTJ3Xlpc5LpebJ1dZFqcLuzdJ63oTNaHK6CfVRaSYYAvvQnOy28ZZFltqgQREdyYKAvA66WbYISRunrrhmwmm+t7B4wQtATR5M6F1EOXiHFG3udTxjC4pfxEJMZNjRaVKVKI9UP3ATbXVLtxx1IDM4IS14+px8UVtor8UxFch2bzUWZQv4G8FvtABGdCu8mDa73q6fhf3TOW3CiqnywTQcjp9\\\\\\\/NDwj\\\\\\\/as0Aq58O05Vb4jWhgATZ0owP9v40WtP3bLfBTyBZS2spaqhreEau\\\\\\\/agyZlafQn4JEIkKO1Wmcbsid78X19mU7hlScbzbKwwuCIO7x1P0vtI7jhtDjL0pgVd5Wrpm0ZW1Nve23NQyJN0Jk9jtw1RWbIw4zLqyg3Gd9cvO6Zmcd6e9KIciL2prjShhsdJKR8KoyGmjVR3suacel46G55PlvWlfc0TEXUD1j6SoY+Ck1hhUj4RTgrJk7U9Xva8+xsYJEpTSK+xAeec0n4ToLVz7muTIJuSFcvh1kwFFG7KAg1ZfUfBL1A2FxA4CknVORanOqJBYTf4wCtLrY5+WvPFEOOitW30ubqrRdKZxEq27+vS5hv6wgVSRypsM2OGtthyrdTgRpCTXXEIldWroAv9nruGpHnBTKSAXBRvG0BVFa5VlhOIlGxB2JT2ITqd51wP6MgTMUudYynunL92y1Da3qsJOxmnGEYnSuBSSo2tTnXA\\\\\\\/+6nW6c4SbW\\\\\\\/0Z3OALQKqAhl7rlD7K7GE90HnIcsJGPvFMKOhu0rW9eIu7sRTWkrU0qHAmaom8Rs5qJoS9nsmWZIgnJbZ0VqvD\\\\\\\/D7dLYdTgbYYp9zEa2z0yGhx\\\\\\\/eqfgm43EeJavyngpIQQ5O+aBskTmrug\\\\\\\/E+IWYDS9W9lThjVTB4Rnu9iI3XZ4mIlOP4F9\\\\\\\/xixmJ5WtMEcf6L+vKzYQxyQ0xjUw3fynOqs0Fp1TgGhg4V1DOvUwLwv7HMpAqwwewZC7btRjHiLrDInPNaztfP9EG6VQzjwSiKWBoX70cEbjzK+SAonF9pPXhhjjhdMCtTCxpAF0S3m0FVYBDUSSsWVaeSB5ADa+gb+Pevta4ybZsFQ1kBNMYuUzcK0tPrxnkRzqApVNrCoyZL4lGKhQQhe+8brBJEZLd4nikROJhYw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022b6Hf\\\\\\\/CzhQr6C4hQEz63egg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002292f02ba8a559161ca985f89886733418c8f8f18901e374396f23f5f56570f3ab\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002258hd5bodfcskfg\\\\\\\/rQ83BnnZ7C3n4GmzmpeRGqmX26BTIfmKGyzKmywN2hSWHtHqOmEUlAVNnedJMzugdSh6\\\\\\\/xn5kI1YANeLLINXZgB+BEMlnDVbThvHW+ebTNa7IU37S9N7LSfr7Vb8zA6p+PKuvJMSCq6iU5OedsSQE6fdEYxdYb5h5fa7KFC627C6yKHRilbHy15aZaIegyaNQeYvMoZGL0PesNXc4L\\\\\\\/ntUnf4LGQU7Azo6KdnOM\\\\\\\/efbkjrMxfC1gPqYUpDP\\\\\\\/5bTiPbAZ2Yo4+g9Grdde+kSF7kyLhYcGCYco3gycXupPhleQlgi\\\\\\\/MoweVLS80IndBcYo1Km2QNEZnUUBWLbZ\\\\\\\/kTqj71frhccGLH5RhNacycZ\\\\\\\/0oOnQN2Yv9ABQkyZPeh7muZI6IcxqxKLKytSWddqT9siQ+SxoP4bI9v2N7iOcfq8fIUhUjsn8+hPFckm80I8g+2jhmSoR3SqTMl4yEukneFgE2QeBgpRB5jjc4wr3Zjk6awFAuMZsZbCr2aygCM\\\\\\\/PSQTgR0kDkkgZ5KH3AoaqEHjO9PobdlnF9yaTqhYj6SFyp4IlintdjUOHVitFZLwBRCgzT+uLE5z\\\\\\\/t5u7jPjZviUF+RP5PsQZva880xsHh09LouN3R7rBD5qsl3TzWhWRFO4657NFuNj0maG4mQLk5vZ5kYVLiAfGwuxKt6LoFgrGvz\\\\\\\/eN8JiYu2C2FeZ+vXK0\\\\\\\/9hY6aKFHxGiBwF0eo+0aQI\\\\\\\/tSD4rEgyclZWdwEgVrXhgeYaTZz8dF4z0lZzIiIys5m7jKuNm7E1w11VMYofwA67RveOvUzdTq8kqDgOqlmhcJEELgFy1UKqJvBSeVO28BKMs6swDvMq7zkPgsOtX0csB0FysB8Rs0hzj4pb+YvlgsLctppyWVJnX0SN77QaSjR21XU3cSiQk8Xa+hzBkLD879eYCTpZrTt1tptprwXK5kuGupCbkrasak15+4TeeZoCXDkao28wobtF1VWvu\\\\\\\/5MzMopPJPCzR0GdgJykCK2jbYLJbDTXU\\\\\\\/Y5uka\\\\\\\/ponH\\\\\\\/nBlD4bbwXeAxOfZOUi8yGOF9EiTGAiSWkjjU\\\\\\\/RPcCq51opeHEjP2kmJDkrhbh298pOJYOgJy4\\\\\\\/diS0iC+T5gL+UNYG3p0Sypft7R0csd28sJQ2Fwqd5LuEwusuTgWDsPO4ZrzaYehAqsk8sNS3i9hezi3gtEr8pGzLIU7x90maADXnIh6iy7azNElUYB3utT3Jp3+6Og1PP2DSxOKzVTvA08tVNE3ByEdfY5ZNkL3AvzP3oVxIjrhTh+HKyrje3ifFt1Urhth\\\\\\\/dLWDYao4UF7ULJ8W76020HHTSsDzgzg2DG2LMTEkP6KlF2OjV5NUXF2kdATPKn\\\\\\\/UE7AThNYuQgZJ6gByCJ2H1ZEZ90u6HbbDp5oroQPdroj9anADoyv6jhDcPEdxVFwCZs6QtsMf+YJqJA8f\\\\\\\/kzOE6Y96CUJ47ieI53xUUXCB8Le1LFW1y3BhfPyoRwZ6KJvJpMS1RXsp9UU64kxFItfKxqrM8b4+Ek4yUWIdUV7L94XNCCtQCifbW1Ar5DdTt3o7fa+LburOLtz1oU1p2iw1QylwMBZ0+To+WGSg3xWmGTfwW6ZKAbhbJpvSAdBW9guvLagBvcrvuMdz5fWgAO1ecgslDQKkIl14K+S5QBJJ9qWPFxr4+DmmiQSxf0ouLSBPivhBs+nfI7QJR07WuNOWqDo+dG\\\\\\\/ypjt2kUdQXtS4SgTAsqxBrNn\\\\\\\/nnK3e3Q20EyEzQFF\\\\\\\/g\\\\\\\/iztle591DJJYB\\\\\\\/5gbtNIUJEJJQHSjuYpLhXVREhX23Wl7myRAh+\\\\\\\/t\\\\\\\/lNgCxagyjDbTDQ+t8x9iOCEYYqeAg0f7b4pC1m\\\\\\\/47JtIIIdm7DZfjpbskY32L8vMgFAf6dN8dol8emFrxP17PGQj3JDrKYWRsijnW+93wY\\\\\\\/hrG57zDotHoO2q3iH\\\\\\\/Hexqdyqnu15HjSpfxdV8uJfaEZ6wmA23W59riSGbZQefKbSsx4hi4N6onnFU7zo1Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xQKsuWjd1dxLEEh\\\\\\\/5Y8DDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002203c8f82fe49b10b30ecef92c5b6d5f11e9256839f77ebe62ceda86f2392698ce\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459235 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mouse-genetic-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/191" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0GSeQneJERg9ee-X3HCEvbSkq7N9_U3fnV8uH1cmHX4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/191" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SRolSq1GbW0toyuI4WEkaC_Y7kdG7h8pC1RA2-txSZk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/175" title="HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/199" title="Haplotype Probabilities in Advanced Intercross Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459235" id="node459235" data-pisa="ggg;2/2/191" data-pisa-master="ggg;2/2/191" data-apath="/ggg/2/2/191.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Catherine E. Welsh</span> and <span class="highwire-citation-author" data-delta="1">Leonard McMillan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">191-198;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459235" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Catherine E. Welsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BE.%2BWelsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welsh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BE.%2BWelsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mcmillan@cs.unc.edu" class="" data-icon-position="" data-hide-link-title="0">mcmillan@cs.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/2/191" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459235" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459235/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/191.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459235" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459235/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/191.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459235" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459235/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/191.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459235" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459235/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/2/191.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459235" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459235/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Inbred model organisms are powerful tools for genetic studies because they provide reproducible genomes for use in mapping and genetic manipulation. Generating inbred lines via sibling matings, however, is a costly undertaking that requires many successive generations of breeding, during which time many lines fail. We evaluated several approaches for accelerating inbreeding, including the systematic use of back-crosses and marker-assisted breeder selection, which we contrasted with randomized sib-matings. Using simulations, we explored several alternative breeder-selection methods and monitored the gain and loss of genetic diversity, measured by the number of recombination-induced founder intervals, as a function of generation. For each approach we simulated 100,000 independent lines to estimate distributions of generations to achieve full-fixation as well as to achieve a mean heterozygosity level equal to 20 generations of randomized sib-mating. Our analyses suggest that the number of generations to fully inbred status can be substantially reduced with minimal impact on genetic diversity through combinations of parental backcrossing and marker-assisted inbreeding. Although simulations do not consider all confounding factors underlying the inbreeding process, such as a loss of fecundity, our models suggest many viable alternatives for accelerating the inbreeding process.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/marker-assisted-breeding" class="hw-term hw-article-keyword hw-article-keyword-marker-assisted-breeding" rel="nofollow">marker-assisted breeding</a></li><li class="kwd"><a href="/keyword/mouse" class="hw-term hw-article-keyword hw-article-keyword-mouse" rel="nofollow">mouse</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li><li class="kwd"><a href="/keyword/recombinant-inbred-lines" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-lines" rel="nofollow">recombinant inbred lines</a></li><li class="kwd"><a href="/keyword/mouse-genetic-resource" class="hw-term hw-article-keyword hw-article-keyword-mouse-genetic-resource" rel="nofollow">Mouse Genetic Resource</a></li></ul><p id="p-4">Recombinant inbred lines (RILs), first developed in 1971 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Taylor <em>et al.</em> 1971</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bailey 1971</a>), have long been an important resource for genetics. Typically, RILs are derived by crossing two inbred strains followed by repeated generations of selfing or sibling mating to produce an inbred line whose genome is a mosaic of its parental lines. More recently, panels of multiway RILs have been developed that combine the genomes of multiple founder lines via an initial mixing stage followed by successive generations of inbreeding. Examples include mouse (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Threadgill <em>et al.</em> 2002</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chesler <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>), maize (Buckler <em>et al.</em> 2009), <em>Drosophila melanogaster</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gibson and Mackay 2002</a>), and <em>Arabidopsis thaliana</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Paulo <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gan <em>et al.</em> 2011</a>). For all species, inbreeding via either selfing or sibling mating is the primary process used for fixing the genetic background. RILs derived by sibling matings from two parental backgrounds require multiple generations to fix their genome as homozygous, and the number of generations depends on the diploid number. In mice, this requires at least 20 generations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eisen 2005</a>) and assuming an average of four generations per year, it takes a minimum of 5 years to create a new RIL. Moreover, a large fraction of the started RILs fail, presumably as the result of genetic incompatibilities affecting survival and reproduction (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Silver 1995</a>). Many recent efforts to generate RILs have focused on multiway crosses where more than two parental lines are initially mixed before inbreeding. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman (2005)</a> showed through simulation that eight-way RILs take on average 26.7 generations of sib-mating to reach 99% fixation, and 38.9 generations, on average, to reach complete fixation.</p><p id="p-5">Although a major source of genetic variation in a RIL is derived from the choice of founder strains, we focus on the additional genetic variations introduced by mixing of allele combinations via recombinations between founder genomes. This is the primary source of genetic variation between RILs. Therefore, the number of distinct founder segments, defined as the regions between recombination breakpoints on the RIL chromosomes, can be used as a measure of genetic diversity. From now on, we refer to these distinct founder segments simply as segments. Using simulations, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Broman (2005)</a> tracked the number of segments generated through recombination in inbred lines and used it as a comparison between the genetic diversity of two-way and eight-way sib-mating RILs. Recombinations in early generations increase diversity, but eventually diversity peaks and the process of inbreeding leads to a loss of segments. In an eight-way cross, the peak in diversity is reached at the seventh generation of inbreeding on average and before 10 generations of inbreeding for 75% of line starts (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Therefore, we will consider 10 generations of inbreeding as past the point of peak diversity. If inbreeding acceleration is started before this peak is reached, the resulting inbred lines are likely to see a reduction in the number of segments. Therefore, unless otherwise specified, we assume a randomized sib-pair mating scheme is used for the first 10 generations, after which we apply various nonmarker-assisted and marker-assisted acceleration techniques.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The average number of founder segments in eight-way RILs at various generations of inbreeding. This figure is based on 100,000 simulations, and the number of segments was tracked until they reached complete fixation. The average peak in the number of segments occurs at generation 7 and before generation 10 for 75% of all lines. Therefore, we consider generation 10 to be past the point of peak diversity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The average number of founder segments in eight-way RILs at various generations of inbreeding. This figure is based on 100,000 simulations, and the number of segments was tracked until they reached complete fixation. The average peak in the number of segments occurs at generation 7 and before generation 10 for 75% of all lines. Therefore, we consider generation 10 to be past the point of peak diversity.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F1.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/191/F1.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461196" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-6" class="first-child">The average number of founder segments in eight-way RILs at various generations of inbreeding. This figure is based on 100,000 simulations, and the number of segments was tracked until they reached complete fixation. The average peak in the number of segments occurs at generation 7 and before generation 10 for 75% of all lines. Therefore, we consider generation 10 to be past the point of peak diversity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Marker-assisted breeding techniques have been used to fix a selected haplotype interval against a fixed background in congenic strains (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Markel <em>et al.</em> 1997</a>). In mouse, marker-assisted âspeed congenicsâ have demonstrated a reduction in the number of generations of backcrossing from 10 generations to five. This reduction was achieved by selecting mice that retained the lowest proportion of heterozygous donor to recipient genome. These selection criteria have evolved overtime, as technology has allowed for more rapid genotyping (Flaherty and Bolivar 2007). Just as marker-assisted techniques have been used to improve mapping resolution in self-pollinated species (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boddhireddy <em>et al.</em> 2009</a>) and have been adapted for consomics (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Armstrong <em>et al.</em> 2006</a>), we adapt them for multiparental RILs. Rather than attempt to fix one specific genomic region or one complete chromosome, our goal is to achieve complete fixation of the genome in fewer generations than random sib-matings, without substantially impacting the overall genetic architecture of the inbred lines. In this article, we address accelerating the inbreeding process of outcrossing species by using a combination of alternative breeding strategies and marker-assisted inbreeding (MAI) techniques.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">We developed a simulator that represents a genome as a collection of intervals whose boundaries can be resolved at the resolution of a base pair rather than a string of alleles as is common in many breeding simulators (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Broman 2005</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Valdar <em>et al.</em> 2006</a>). The interval representation has the advantage of implicitly representing every base pair in the genome while explicitly tracking every recombination. This approach provides a conservative estimate of homozygosity because it treats every founder sequence as a separate genotype without taking into account regions of sequence identity among founders. Moreover, our interval model can be trivially converted to a string of alleles representation if given the founder sequences or markers from any platform.</p><p id="p-9">Despite the differences in the underlying representation, our simulator produces results nearly indistinguishable from those presented by <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Broman (2005)</a>. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the distribution of the number of generations to complete fixation and number of segments for both the two-way and eight-way sib-mating RILs based on the simulation of 100,000 RILs. For a randomized eight-way RIL our simulations show that it takes an average of 38.21 Â± 7.1 (SD) generations of sib-matings to reach complete fixation. The genomes of the resultant inbred lines have an average of 145.1 Â± 12.48 segments in their mosaic structure. Furthermore, 25.72 Â± 3.16 generations of sib-mating on average are needed to reach 99% fixation. These baseline metrics are used for comparison against our accelerated inbreeding simulations.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The number of generations to complete fixation (A) and the number of resulting founder segments (B) in two-way and eight-way RILs. On average, two-way RILs take 35.92 generations to reach complete fixation and have 91.95 segments. Eight-way RILs take 38.21 generations and have 145.12 segments on average. These figures are based on 100,000 simulations and are consistent with previous simulations (Broman 2005)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The number of generations to complete fixation (A) and the number of resulting founder segments (B) in two-way and eight-way RILs. On average, two-way RILs take 35.92 generations to reach complete fixation and have 91.95 segments. Eight-way RILs take 38.21 generations and have 145.12 segments on average. These figures are based on 100,000 simulations and are consistent with previous simulations (Broman 2005).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F2.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/2/191/F2.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461258" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-10" class="first-child">The number of generations to complete fixation (A) and the number of resulting founder segments (B) in two-way and eight-way RILs. On average, two-way RILs take 35.92 generations to reach complete fixation and have 91.95 segments. Eight-way RILs take 38.21 generations and have 145.12 segments on average. These figures are based on 100,000 simulations and are consistent with previous simulations (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Broman 2005</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Our analysis is based on an initial funnel-breeding scheme like that used in the eight-way Collaborative Cross (CC) (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 2004</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>), where the mixing of eight inbred lines occurs in two initial crossing stages (G1 and G2), followed by successive generations of sib-matings (G2:F1, G2:F2, etc) until the line becomes fully inbred. The eight founder strains are represented by letters A-H, and these labels are used to track the descent of genomic segments (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>).</p><p id="p-12">We introduce a notion of joint heterozygosity (JH) to express four possible states between the homologous alleles of a potential breeding pair. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows two homologous chromosomes from each parent of a potential breeding pair and depicts each of the JH states. The inbred state is achieved when both male and female samples are homozygous for the same founder state. We call this state same-same (SS). Another possible state involves a breeding pair that is heterozygous with alleles from two founders while the mate is homozygous. We call this different-same (DS). This state occurs in two forms, DS<sub>2</sub> when the heterozygous gene shares a founder allele with the homozygous allele of its mate, and DS<sub>3</sub>, when the heterozygous gene shares no founder alleles with its mate. The third state is opposite-same (Ss), where the male is homozygous for one founder and the female is homozygous for another. The final state is different-different (DD), where both male and female are heterozygous. This state comes in three variations, involving, two, three, and four founders, respectively. The two-founder state, called DD<sub>2</sub>, occurs when both male and female are heterozygous between the same founder alleles. DD<sub>3</sub> refers to when both male and female are heterozygous but share one common founder allele. DD<sub>4</sub> occurs when the male and female are heterozygous and do not share any founder alleles. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows a state diagram with these four states and their forms depicting all possible transitions between them in a single generation. The directed edge weights represent the probability of transitioning between JH states. A similar transition matrix, which uses 13 states instead of our seven, has also been derived by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman (2012)</a>. It is a simple matter to extend our JH model to two generations by finding every path of length two within the graph and inserting an edge with weight equal to the product of the two edges along its path. The weights of edges from a common source to a common destination, but passing through different intermediate states, can be added and combined into a single edge. This approach can be extended to <em>n</em> generations, and as <em>n</em> increases all of the heaviest edges eventually lead to the inbred (SS) state. For analytical expressions for extending our JH model for n generations, see (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Broman 2012</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="This image shows all possible JH states between a potential mating-pair and illustrates our notion of a genomic segment. DD stands for different-different and occurs in three variations. DD4 occurs when both breeders are heterozygous and do not share any founder alleles among them. DD3 occurs when both breeders are heterozygous and share one founder allele, whereas DD2 refers to both breeders being heterozygous for the same two founder alleles. DS stands for different-same and occurs in two variations. DS3 occurs when the heterozygous gene shares no founder alleles with the homozygous allele of its mate. DS2 refers to when the heterozygous gene shares one founder allele with its mate. Ss is opposite same, where the male is homozygous for one founder allele and the female is homozygous for another allele. The final state, SS (same-same), is achieved when both male and female are homozygous for the same founder allele. All JH segments are depicted with a chromosome fraction of 0.15, except for Ss, with 0.10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;This image shows all possible JH states between a potential mating-pair and illustrates our notion of a genomic segment. DD stands for different-different and occurs in three variations. DD&lt;sub&gt;4&lt;/sub&gt; occurs when both breeders are heterozygous and do not share any founder alleles among them. DD&lt;sub&gt;3&lt;/sub&gt; occurs when both breeders are heterozygous and share one founder allele, whereas DD&lt;sub&gt;2&lt;/sub&gt; refers to both breeders being heterozygous for the same two founder alleles. DS stands for different-same and occurs in two variations. DS&lt;sub&gt;3&lt;/sub&gt; occurs when the heterozygous gene shares no founder alleles with the homozygous allele of its mate. DS&lt;sub&gt;2&lt;/sub&gt; refers to when the heterozygous gene shares one founder allele with its mate. Ss is opposite same, where the male is homozygous for one founder allele and the female is homozygous for another allele. The final state, SS (same-same), is achieved when both male and female are homozygous for the same founder allele. All JH segments are depicted with a chromosome fraction of 0.15, except for Ss, with 0.10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F3.medium.gif" width="440" height="130"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/2/191/F3.medium.gif" width="440" height="130"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461329" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-13" class="first-child">This image shows all possible JH states between a potential mating-pair and illustrates our notion of a genomic segment. DD stands for different-different and occurs in three variations. DD<sub>4</sub> occurs when both breeders are heterozygous and do not share any founder alleles among them. DD<sub>3</sub> occurs when both breeders are heterozygous and share one founder allele, whereas DD<sub>2</sub> refers to both breeders being heterozygous for the same two founder alleles. DS stands for different-same and occurs in two variations. DS<sub>3</sub> occurs when the heterozygous gene shares no founder alleles with the homozygous allele of its mate. DS<sub>2</sub> refers to when the heterozygous gene shares one founder allele with its mate. Ss is opposite same, where the male is homozygous for one founder allele and the female is homozygous for another allele. The final state, SS (same-same), is achieved when both male and female are homozygous for the same founder allele. All JH segments are depicted with a chromosome fraction of 0.15, except for Ss, with 0.10.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A state diagram showing the transitions between all JH states in a single generation. The directed edges are labeled with the transition probability. The grayed-out nodes represent transient states; once a segment moves away from these three states, there are no returning edges. Transient states tend to go away after a few generations and are rarely seen past the point of peak diversity (as shown in Figure 5). CC lines begin inbreeding in one of the states, DD4, DD3,, and DD2. The desired inbred state for all intervals is SS. DS2 is the most likely to become SS. DD2 is the next most likely state to become fixed. It takes at least two generations to transit from Ss to SS, as there is no direct path between these two states." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A state diagram showing the transitions between all JH states in a single generation. The directed edges are labeled with the transition probability. The grayed-out nodes represent transient states; once a segment moves away from these three states, there are no returning edges. Transient states tend to go away after a few generations and are rarely seen past the point of peak diversity (as shown in Figure 5). CC lines begin inbreeding in one of the states, DD&lt;sub&gt;4&lt;/sub&gt;, DD&lt;sub&gt;3,&lt;/sub&gt;, and DD&lt;sub&gt;2&lt;/sub&gt;. The desired inbred state for all intervals is SS. DS&lt;sub&gt;2&lt;/sub&gt; is the most likely to become SS. DD&lt;sub&gt;2&lt;/sub&gt; is the next most likely state to become fixed. It takes at least two generations to transit from Ss to SS, as there is no direct path between these two states.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F4.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/2/191/F4.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461252" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-14" class="first-child">A state diagram showing the transitions between all JH states in a single generation. The directed edges are labeled with the transition probability. The grayed-out nodes represent transient states; once a segment moves away from these three states, there are no returning edges. Transient states tend to go away after a few generations and are rarely seen past the point of peak diversity (as shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). CC lines begin inbreeding in one of the states, DD<sub>4</sub>, DD<sub>3,</sub>, and DD<sub>2</sub>. The desired inbred state for all intervals is SS. DS<sub>2</sub> is the most likely to become SS. DD<sub>2</sub> is the next most likely state to become fixed. It takes at least two generations to transit from Ss to SS, as there is no direct path between these two states.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">In early generations the CC lines include genomic intervals in JH states involving three or more founders (DD<sub>3</sub>, DD<sub>4</sub>, DS<sub>3</sub>), but in later generations these intervals eventually transition to states with two or less founders (DD<sub>2</sub>, DS<sub>2</sub>, SS, and Ss; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). We can see in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> that DD<sub>4</sub>, DD<sub>3</sub>, and DS<sub>3</sub> are transient states, meaning that once this group of three states is left, there are no returning edges. In two-way RILs, the three transient states do not occur because there are at most two founders present. When selfing, the model further reduces to only two JH states, DD<sub>2</sub> and SS. The transition probabilities to reach the inbred state are incorporated into our metric for selecting the best mating pair at each generation, which is discussed later in this section.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A histogram of segments colored according to their JH state as a function of generation. In early generations, most segments have contributions from three or more founders, but by generation 10 (after the point of peak diversity), segments have contributions from two or fewer founders. This plot was created by tracking the JH states between breeder pairs and finding the average contribution of each state over 100,000 simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A histogram of segments colored according to their JH state as a function of generation. In early generations, most segments have contributions from three or more founders, but by generation 10 (after the point of peak diversity), segments have contributions from two or fewer founders. This plot was created by tracking the JH states between breeder pairs and finding the average contribution of each state over 100,000 simulations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F5.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/2/191/F5.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461194" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-16" class="first-child">A histogram of segments colored according to their JH state as a function of generation. In early generations, most segments have contributions from three or more founders, but by generation 10 (after the point of peak diversity), segments have contributions from two or fewer founders. This plot was created by tracking the JH states between breeder pairs and finding the average contribution of each state over 100,000 simulations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">Using the notion of JH state, we split the genome into intervals according to state and track the genomic fraction of each type. We combine these fractions to arrive at several useful measures. Adding the genomic fraction of all regions in the same-same state (SS) gives the fixed genomic fraction (FGF). We call the complement of this, or 1-FGF, the mating pair's combined heterozygous fraction (CHF). FGF and CHF can be used to assess how inbred a line is, such that FGF = 1 refers to fully inbred.</p><p id="p-18">In simulation, we tested a number of modified breeding schemes in an attempt to accelerate the inbreeding process. These nonmarker-assisted breeding schemes minimally impact the traditional RIL generation process. We considered several variations of backcrosses. The use of backcrosses was motivated by the analysis of <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Broman (2005)</a>, which identified a substantial advantage for selfing when compared to sib-mating. Selfing in two-way plant RILs takes on average 10.5 generations to reach complete fixation, which is a substantial reduction from the 35 generations needed when two-way sib-mating.</p><p id="p-19">The steadily decreasing cost of high-density genotyping combined with the advantages of considering each sample's individual full genetic makeup motivated us to also explore MAI techniques. The ability to compare potential breeding pairs based on their high density genotypes allows us to choose breeding pairs with the greatest likelihood of producing inbred offspring. The Ss (opposite same) is the least-preferred state in a breeding pair because it has no chance of becoming inbred in the next generation, as shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. In contrast, of the noninbred states, DS<sub>2</sub> has the greatest probably of becoming fixed in the next generation, and DD<sub>2</sub> is the next most likely.</p><p id="p-20">We choose the âbestâ breeding pair, by considering a weighted genomic mix of the JH types of all candidate mating pairs. The best pair is selected as the maximum of a weighted combination of transition probabilities for all JH segments of a given mating pair considering all chromosomes. For each distinct JH segment of a chromosome the probability that it will become inbred in the next generation (<em>i.e.</em>, the weight of the edge from the current JH state to the SS state) is multiplied by the chromosome fraction of the segment, and the sum is accumulated over all segments on the chromosome. This calculation results in a chromosome score ranging from 0, when the entire chromosome is Ss, DD<sub>3</sub>, DD<sub>4</sub>, or DS<sub>3</sub>, to 1 when the entire chromosome is SS. This approximation ignores the relative ordering of segments, and, therefore, does not consider linkage. The individual chromosome scores are then multiplied together, modeling their independent segregation, to arrive at the total pair score. Therefore, we assign a score for a given mating pair as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/191/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/191/embed/mml-math-1.gif"/></noscript></span></span></p><p id="p-21">This score is an approximation of the actual likelihood that the entire genome will become inbred in the next generation. We refer to this score as the weighted state metric (WSM). <em>JHSeg<sub>n,m</sub></em> represents a JH segment on the specified chromosome <em>i</em> induced by the pairing <em>n,m</em>, and the best pair is the maximum of this score over all possible pairs <em>n,m</em>. In self-pollinated species, our score simplifies to a scaled version of the FGF because the only relevant states are DD<sub>2</sub> and SS, which has been described previously (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Boddhireddy <em>et al.</em> 2009</a>).</p><p id="p-22">We explored two marker-assisted breeding schemes. The first of these is MAI, which modifies the breeding scheme only after the point of peak diversity is reached. Once the peak is reached, the WSM discussed previously is applied to choose the best breeding pairs. The second is a marker-assisted advanced intercross, which modifies the breeding scheme to choose sib-pairs to increase segments until either a specified generation or a desired number of segments is reached; it then reverts to choosing sib-pairs to accelerate inbreeding. Through simulations, we track the average number of generations to fully inbred and to 99% inbred as well as the average number of segments present in the inbred lines to compare the different breeding schemes.</p><p id="p-23">The simulator is written in Python and runs on a Dell Studio XPS with 8GB RAM, with dual-threaded quad-core processors. It takes approximately 5.5 hr to complete 100,000 simulations of eight-way RILs.</p><p id="p-24">For the purposes of this analysis, the eight-way CC funnel breeding scheme was used, but our simulator also supports the input of any breeding scheme using pedigree files. It has also been used to simulate two-way RILs, F2 crosses, and outbred populations.</p><p id="p-25">To test our MAI methods, we used the developing CC (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>) and a low-density genotyping platform we designed, referred to as the Mouse Universal Genotyping Array (MUGA). The SNPs on MUGA are evenly distributed with an average spacing of 325 Kb and a standard deviation of 191 Kb. In an eight-way cross, the genotypes at multiple markers (at a minimum three) are needed to distinguish among the founders. The founder assignments and recombination breakpoints are inferred from the genotypes using a hidden Markov model similar to the ones described by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mott <em>et al.</em> (2000)</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Zhang <em>et al.</em> (2009)</a>, and <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> (2010)</a>. Because multiple markers are needed to distinguish each founder, the effective founder-ascertainment resolution of MUGA is approximately 1 Mb.</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Nonmarker-assisted breeding schemes</h3><p id="p-26">The first breeding scheme examined was alternating back-crosses in successive generations, father-to-daughter in one generation followed by mother-to-son in the next (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001784/-/DC1/001784SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001784/-/DC1/FigureS1.pdf">Figure S1</a>). This scheme has many practical advantages in that it leverages known-fertile samples. Furthermore, this strategy also serves as a useful fallback for preserving lines. We simulated this approach starting after the point of peak diversity, with a backcross between a father and daughter followed by a backcross between a mother and son in the next generation (each breeder is used in two successive generations, alternating dam and sire). This process was repeated for each subsequent generation until complete fixation was achieved. Alternating backcrosses achieves a reduction in the number of generations to complete fixation with an average number of generations of 33.45 Â± 5.88 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). This represents a reduction of nearly five generations over randomized mating and a substantial reduction in variance. It decreases the number of segments in the resulting inbred lines to 141.21, a loss of about four segments on average. The alternating backcross also reduces the number of generations to 99% fixation to 23.45 Â± 3.11, a reduction of two generations.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A comparison of five breeder selection alternatives for generating an eight-way RIL, showing the number of generations to reach complete fixation (A) and the total number of segments (B) found in the final inbred lines. Random sib-pair mating is used as our baseline. The alternating backcross swaps between father&#x2212;daughter and mother&#x2212;son matings in successive generations. The father&#x2212;daughter scheme alternates between father&#x2212;daughter and random sibling matings in successive generations. MAI uses our weighted state metric to choose between 16 breeding pairs after the point of peak diversity. The selected advanced intercross modifies early stages of the breeding scheme to choose sib-pairs that maximize diversity, and then at a pre-established generation (10), it reverted to choosing sib-pairs to accelerate the inbreeding process." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A comparison of five breeder selection alternatives for generating an eight-way RIL, showing the number of generations to reach complete fixation (A) and the total number of segments (B) found in the final inbred lines. Random sib-pair mating is used as our baseline. The alternating backcross swaps between father&#x2212;daughter and mother&#x2212;son matings in successive generations. The father&#x2212;daughter scheme alternates between father&#x2212;daughter and random sibling matings in successive generations. MAI uses our weighted state metric to choose between 16 breeding pairs after the point of peak diversity. The selected advanced intercross modifies early stages of the breeding scheme to choose sib-pairs that maximize diversity, and then at a pre-established generation (10), it reverted to choosing sib-pairs to accelerate the inbreeding process.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F6.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/2/191/F6.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461202" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-27" class="first-child">A comparison of five breeder selection alternatives for generating an eight-way RIL, showing the number of generations to reach complete fixation (A) and the total number of segments (B) found in the final inbred lines. Random sib-pair mating is used as our baseline. The alternating backcross swaps between fatherâdaughter and motherâson matings in successive generations. The fatherâdaughter scheme alternates between fatherâdaughter and random sibling matings in successive generations. MAI uses our weighted state metric to choose between 16 breeding pairs after the point of peak diversity. The selected advanced intercross modifies early stages of the breeding scheme to choose sib-pairs that maximize diversity, and then at a pre-established generation (10), it reverted to choosing sib-pairs to accelerate the inbreeding process.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">There are several practical limitations to the alternating backcrossing approach. For instance, female fertility often spans a limited window that might not allow for mother-son backcrossing. Therefore, we also explored, through simulation, a modified breeding scheme involving only father-daughter backcrosses. Starting after the point of peak diversity, a father-daughter backcross is followed in the next generation by a random sib-mating. This breeding scheme (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001784/-/DC1/FigureS1.pdf">Figure S1</a>) is repeated for each subsequent generation until complete fixation is achieved. The father-daughter backcross takes 37.06 Â± 7.55 generations to reach complete fixation, and the inbred lines contain on average 142.39 Â± 12.24 segments (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). This breeding scheme also takes 24.70 Â± 3.54 generations to 99% fixation. Although the benefits of father-daughter mating appear modest relative to random sib-mating, in practice they are compensated for by a reduction in generation time resulting from a mature and known fertile sire.</p></div><div id="sec-4" class="subsection"><h3>Marker-assisted inbreeding</h3><p id="p-29">For all MAI techniques, random sib-matings were simulated until the point of peak diversity was passed. This was followed by subsequent generations of selecting the best breeding pair, until the line reached complete fixation.</p><p id="p-30">Using the WSM, we selected the best pair from sib-pairs, parentâchild backcrosses, or a combination of both. To see what other pair relationships were worth considering, we simulated 100,000 lines such that random sib-matings were used for 15 generations, at which time three mating pairs were generated, producing two male and two female offspring each. The best breeding pair was then chosen by comparing every female to every male (both parents and offspring). The pair with the lowest CHF was selected. Sib-pairs were selected 63% of the time, whereas backcrosses were chosen 23% of the time. Cousin-pairs (offspring from different mating pairs of the same generation) were the next most likely, being selected 6.9%. The remaining 7.1% included mating combinations such as auntânephew, uncleâniece, or grandparentâgrandchild. We concluded that non-sib, non-backcross matings should be used sparingly, except in the case of preserving a line.</p><p id="p-31">Because sib-pairs were most often the best option, we limited subsequent simulations to selecting the best sib-pair and report those statistics in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. For the MAI sib-pairs breeding scheme, random sib-matings were simulated until the point of peak diversity was reached. After this point, four female and four male offspring were simulated (4-4), all pairs were considered, and the best pair was chosen as the breeders. This process was continued until the line reached complete fixation. Our model is based on generation number and may require multiple litters to achieve the four females and four males assumed in simulation. A potential shortcoming of our model is that we report the time to inbred as a function of generations, not the number of litters or calendar time required to produce enough viable offspring. However, we did perform additional simulations assuming smaller litter sizes (two females, two males), and unbalanced sex-ratio (eight total offspring with one to seven females), and compared all three sets of assumptions (4-4, 2-2, 8) to the greedy approach of setting up breeders as soon as any sibling mating pairs are available. Any form of MAI was always able to considerably reduce the number of generations to achieve inbred status regardless of sex balance or litter size. Moreover, waiting for a sufficiently large breeder-candidate set always outperformed the greedy approach of setting up matings as soon as any pair was available. More details on this analysis appear in the supplementary documentation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001784/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461310/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461310/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1â</span> <p id="p-32" class="first-child">Number of generations to 100% fixation (fully inbred), 99% fixation, and number of segments for different breeding schemes</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Using this MAI breeding scheme, it was found that 99% fixation can be reached in an average of 16.44 Â± 1.00 generations, whereas complete fixation can be reached in 22.10 Â± 4.41 generations on average. These inbred lines have an average of 138.83 Â± 11.83 segments. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows that MAI reduces the CHF among mating pairs much faster than random sib-matings. As soon as the breeding scheme is altered at the point of peak diversity, the effect is apparent.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/191/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CHF as a function of number of generations. This plot shows that MAI reduces the CHF among breeding pairs much faster than random sib-matings. We can see the effect as soon as the breeding scheme is modified (at the point of peak diversity)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1082620370" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;CHF as a function of number of generations. This plot shows that MAI reduces the CHF among breeding pairs much faster than random sib-matings. We can see the effect as soon as the breeding scheme is modified (at the point of peak diversity).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/191/F7.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/2/2/191/F7.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/191/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/191/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461224" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child">CHF as a function of number of generations. This plot shows that MAI reduces the CHF among breeding pairs much faster than random sib-matings. We can see the effect as soon as the breeding scheme is modified (at the point of peak diversity).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Selected advanced intercrosses</h3><p id="p-36">Although MAI achieves a substantial reduction in the number of generations required to fix an RIL, it does so with an average loss of approximately seven segments per line. This result is unfortunate because the number of segments determines the resolution of a RIL panel for quantitative trait mapping (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). Therefore, we attempted to overcome this loss by using marker-assisted techniques in the first 10 generations of inbreeding to select mating pairs most apt to increase the number of recombination segments. We refer to these lines as selected advanced intercrosses (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darvasi and Soller 1995</a>) in that they attempt to increase the number of segments on every chromosome by maximizing diversity until a designated generation is reached. This is similar to work done in self-pollinating populations to maximize mapping resolution (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Boddhireddy <em>et al.</em> 2009</a>). After the designated generation, the same MAI techniques as discussed previously are used to select the breeding-pairs until the line is fixed. We found that it took on average 23.5 Â± 3.82 generations to become inbred. At the point of peak diversity, the lines had an average of 196.1 Â± 15.44 segments, compared with 167 segments in randomized sib-pair matings. The average number of segments in the final inbred animals was 155.6 Â± 12.53. On the basis of our analysis, if genotyping is done at every generation, the lines will become inbred in approximately the same number of generations as the MAI breeding strategy but will have approximately 17 more segments per animal. This could lead to increased mapping resolution in the final population.</p></div><div id="sec-6" class="subsection"><h3>Low-resolution sampling</h3><p id="p-37">In our MAI analysis, we assumed that one is able to accurately assign genomic regions to founders at single base-pair resolution. In reality, genotyping platforms have a limited resolution with which they can ascertain a founder's genomic sequence. This limited resolution creates two main obstacles to the use of MAI methods: the possibility that small recombination intervals might escape detection, and the imprecision with which the cross-over points of recombination can be detected. The impact of both of these limitations can, however, be modeled in a simulation.</p><p id="p-38">We modeled this reduced resolution by sampling the JH state at 1-Mb intervals. We simulated the breeding using the MAI breeding strategy discussed earlier, but modified the WSM to consider the JH state only at sample points. Furthermore, we declared lines inbred on the basis of the 1-Mb sampling (when all sample points were SS). We then inspected each declared âinbredâ mouse to see if, at a base-pair resolution, all intervals were truly fixed, and found them to be actually inbred only 38.3% of the time. On average we missed three nonfixed segments per line, and these segments were on average 327Â± 234 Kb. We also found that the lines were considered inbred approximately 2.5 generations earlier than MAI with complete observability. This finding implies that the inability to detect small recombinants might require additional inbreeding generations to attain the desired level of fixation.</p></div></div><div class="section discussion" id="sec-7"><h2 class="">Discussion</h2><p id="p-39">Through simulations, we have developed several alternatives to random sib-matings to dramatically accelerate the creation of RILs by as much as 16 generations. These include the judicious use of parental backcrossing and the selection of mating pairs based on genotypes from genome-wide SNPs. Both of these techniques, when applied after the point of peak diversity is reached, result in a negligible reduction in the number of segments. We also propose an advanced intercross variant in which MAI is applied during the early generations to increase the number of haplotype segments for better mapping resolution.</p><p id="p-40">In simulation we also have the luxury of assuming uniform litter sizes and equal sex ratios, but in reality the fecundity of a RIL and the sex-balance of litters are complicating issues. As lines become more inbred, fertility generally decreases (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Silver 1995</a>). One way to address this is to use backcrosses as discussed previously. However fertility issues might override the choice of âbest breeding pair.â To address this problem we calculate backups that, when used, may extend the number of generations required to achieve fixation.</p><p id="p-41">Taking fertility into account and prioritizing for the preservation of the lines, how do we select the final breeders? WSM optimizes for becoming inbred in one generation, but it might be more advantageous in the early MAI generations to select for animals whose probability to become inbred in two or more generations is maximized. However, in simulations, the two-generation metric generally chooses the same breeding pairs as the single-generation model, leading to the same number of generations to achieve fixation. Once lines reach small levels of residual heterozygosity, it might also be advantageous to maintain multiple breeding pairs selected to produce compatible offspring, which are more like sib-pairs than cousin-pairs. This provides more pair options, as well as a chance to compensate for uneven sex ratios or small litter sizes. Although it seems best to choose the optimal breeding pairs early on, finding good pairs near the end-game (fixing the last 1%â2% of the genome) is a harder problem. The last few heterozygous regions can take several generations to fix if compatible breeding pairs do not exist.</p><p id="p-42">The simulation software used in this analysis is available for download from <a href="http://sourceforge.net/p/breedingsim/">http://sourceforge.net/p/breedingsim/</a>. It has been adapted for many uses other than marker assisted inbreeding such as estimating the significance of measured statistics in the developing CC (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Fernando Pardo-Manuel de Villena and David Threadgill for their advice and comments on improving the simulator and the manuscript. In addition, we also want to thank the UNC Computational Genetics research group.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001784/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001784/-/DC1</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-07-11"><span class="received-label">Received </span>July 11, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-11-06"><span class="accepted-label">Accepted </span>November 6, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2012 Welsh and McMillan</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.1" data-doi="10.1007/s00335-005-0123-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodnicki</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Mind the gap: analysis of marker-assisted breeding strategies for inbred mouse strains</span>. <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">273</span>â<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DArmstrong%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D273%26rft.epage%253D287%26rft.atitle%253DMind%2Bthe%2Bgap%253A%2Banalysis%2Bof%2Bmarker-assisted%2Bbreeding%2Bstrategies%2Bfor%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-005-0123-y%26rft_id%253Dinfo%253Apmid%252F16596449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-005-0123-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16596449&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.2" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genetic analysis of complex traits in the emerging Collaborative Cross</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>â<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8xOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.3" data-doi="10.1097/00007890-197103000-00013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span>â<span class="cit-article-title">Recombinant-inbred strains. An aid to finding identity, linkage, and function of histocompatibility and other genes</span>. <abbr class="cit-jnl-abbrev">Transplantation</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">325</span>â<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransplantation%26rft.volume%253D11%26rft.spage%253D325%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00007890-197103000-00013%26rft_id%253Dinfo%253Apmid%252F5558564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00007890-197103000-00013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5558564&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971I836800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boddhireddy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Selective Advance for Accelerated Development of Recombinant Inbred QTL Mapping Populations</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D49%26rft.spage%253D1294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.5" data-doi="10.1534/genetics.104.035212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The genomes of recombinant inbred lines</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1133</span>â<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D169%26rft.spage%253D1133%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035212%26rft_id%253Dinfo%253Apmid%252F15545647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.104.035212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15545647&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227697200051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.6" data-doi="10.1534/genetics.111.132647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genotype probabilities at intermediate generations in the construction of recombinant inbred lines</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D403%26rft.epage%253D412%26rft.atitle%253DGenotype%2BProbabilities%2Bat%2BIntermediate%2BGenerations%2Bin%2Bthe%2BConstruction%2Bof%2BRecombinant%2BInbred%2BLines%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132647%26rft_id%253Dinfo%253Apmid%252F22345609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzE5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.2.191.7" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The genetic architecture of maize flowering time</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>â<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.8" data-doi="10.1007/s00335-008-9135-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branstetter</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics</span>. <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">382</span>â<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D382%26rft.epage%253D389%26rft.atitle%253DThe%2BCollaborative%2BCross%2Bat%2BOak%2BRidge%2BNational%2BLaboratory%253A%2Bdeveloping%2Ba%2Bpowerful%2Bresource%2Bfor%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9135-8%26rft_id%253Dinfo%253Apmid%252F18716833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9135-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716833&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.9" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">The Collaborative Cross, a community resource for the genetics analysis of complex traits</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>â<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.10"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Collaborative Cross Consortium</span>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The genome architecture of the Collaborative Cross Mouse Genetic Reference Population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-comment">389â401</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>â<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D141%26rft.spage%253D1199%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8582624&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TB62900035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-book" id="cit-2.2.191.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-source">The Mouse in Animal Genetics and Breeding Research</span>. <span class="cit-publ-name">Imperial College Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2.2.191.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flaherty</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolivar</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-comment">Congenic and consomic strains, chapter 7 in <em>Neurobehavioral Genetics: Methods and Applications</em>, edited by B. Jones and P. Mormede. CRC Press, Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.14" data-doi="10.1038/nature10414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drewe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multiple reference genomes and transcriptomes for <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">419</span>â<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGan%26rft.auinit1%253DX.%26rft.volume%253D477%26rft.issue%253D7365%26rft.spage%253D419%26rft.epage%253D423%26rft.atitle%253DMultiple%2Breference%2Bgenomes%2Band%2Btranscriptomes%2Bfor%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10414%26rft_id%253Dinfo%253Apmid%252F21874022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874022&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295080500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.15" data-doi="10.1017/S0016672302005839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Enabling population and quantitative genomics</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DGibson%26rft.auinit1%253DG.%26rft.volume%253D80%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D6%26rft.atitle%253DEnabling%2Bpopulation%2Band%2Bquantitative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672302005839%26rft_id%253Dinfo%253Apmid%252F12448852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672302005839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12448852&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178403300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.16" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.17" data-doi="10.1093/bioinformatics/btq187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Efficient genome ancestry inference in complex pedigrees with inbreeding</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i199</span>â<span class="cit-lpage">i207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq187%26rft_id%253Dinfo%253Apmid%252F20529906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTIvaTE5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8xOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.18" data-doi="10.1038/ng1197-280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Markel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebeling</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagle</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Theoretical and empirical issues for marker-assisted breeding of congenic mouse strains</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">280</span>â<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarkel%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D280%26rft.epage%253D284%26rft.atitle%253DTheoretical%2Band%2Bempirical%2Bissues%2Bfor%2Bmarker-assisted%2Bbreeding%2Bof%2Bcongenic%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1197-280%26rft_id%253Dinfo%253Apmid%252F9354790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1197-280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9354790&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YD66700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.19" data-doi="10.1073/pnas.230304397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turri</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">A new method for fine-mapping quantitative trait loci in outbred animal stocks</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12649</span>â<span class="cit-lpage">12654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.230304397%26rft_id%253Dinfo%253Apmid%252F11050180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjMvMTI2NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.20" data-doi="10.1007/s10681-008-9665-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A mixed model QTL analysis for a complex cross population consisting of a half diallel of two-way hybrids in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">107</span>â<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-008-9665-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-008-9665-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254877800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-book" id="cit-2.2.191.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-source">Mouse Genetics</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.22" data-doi="10.1073/pnas.68.12.3190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span>â<span class="cit-article-title">Host-gene control of C-type RNA tumor virus: inheritance of the group-specific antigen of murine leukemia virus</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">3190</span>â<span class="cit-lpage">3194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTaylor%26rft.auinit1%253DB.%2BA.%26rft.volume%253D68%26rft.issue%253D12%26rft.spage%253D3190%26rft.epage%253D3194%26rft.atitle%253DHost-Gene%2BControl%2Bof%2BC-Type%2BRNA%2BTumor%2BVirus%253A%2BInheritance%2Bof%2Bthe%2BGroup-Specific%2BAntigen%2Bof%2BMurine%2BLeukemia%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.68.12.3190%26rft_id%253Dinfo%253Apmid%252F4332013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNjgvMTIvMzE5MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8xOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.23" data-doi="10.1007/s00335-001-4001-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort</span>. <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">175</span>â<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DThreadgill%26rft.auinit1%253DD.%2BW.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D175%26rft.epage%253D178%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bcomplex%2Band%2Bquantitative%2Btraits%253A%2Bfrom%2Bfantasy%2Bto%2Breality%2Bvia%2Ba%2Bcommunity%2Beffort.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-001-4001-y%26rft_id%253Dinfo%253Apmid%252F11956758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-001-4001-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11956758&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175046300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.24" data-doi="10.1534/genetics.104.039313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Simulating the Collaborative Cross: power of quantitative trait loci detection and mapping resolution in large sets of recombinant inbred strains of mice</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1783</span>â<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D1783%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039313%26rft_id%253Dinfo%253Apmid%252F16361245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.104.039313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16361245&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F191.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.2.191.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Inferring genome-wide mosaic structure</span>. <abbr class="cit-jnl-abbrev">Pac. Symp. Biocomput.</abbr> <span class="cit-fpage">150</span>â<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/2/191.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/175" title="HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/199" title="Haplotype Probabilities in Advanced Intercross Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/2"><span>Volume 2 Issue 2, February 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/2.cover.gif?itok=i7gjzosG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/191" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-31ZQW56Z0HNr1TH1qir4Ya72b91XAY8_ZSkAH-mxEmA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mouse-genetic-resources" class="" data-icon-position="" data-hide-link-title="0">Mouse Genetic Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/2/191" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/2/191" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/2/191" class="active" data-icon-position="" data-hide-link-title="0">Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459235" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2j2vTgovdr0Ucv6KURgPY2Mmwy3vxcqCqpMWQAP5W0g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/191.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/2/191" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MyylRFa3IzzbCCaIEDrnlG8Tg_eIRtNxJxaoa6br1Nw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/2/191" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/191.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459235" id="node459235--2" data-pisa="ggg;2/2/191" data-pisa-master="ggg;2/2/191" data-apath="/ggg/2/2/191.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Catherine E. Welsh</span> and <span class="highwire-citation-author" data-delta="1">Leonard McMillan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">191-198;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459235--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Catherine E. Welsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BE.%2BWelsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welsh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BE.%2BWelsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mcmillan@cs.unc.edu" class="" data-icon-position="" data-hide-link-title="0">mcmillan@cs.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/2/191" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/2/191&amp;title=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/2/191&amp;title=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/2/191&amp;title=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/2/191&amp;text=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/2/191&amp;title=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/2/191&amp;t=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/2/191&amp;title=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/2/191&amp;title=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459235/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459235" id="node459235--3" data-pisa="ggg;2/2/191" data-pisa-master="ggg;2/2/191" data-apath="/ggg/2/2/191.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Catherine E. Welsh</span> and <span class="highwire-citation-author" data-delta="1">Leonard McMillan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">191-198;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459235--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Catherine E. Welsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BE.%2BWelsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welsh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BE.%2BWelsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mcmillan@cs.unc.edu" class="" data-icon-position="" data-hide-link-title="0">mcmillan@cs.unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=191&amp;rft.epage=198&amp;rft.atitle=Accelerating%20the%20Inbreeding%20of%20Multi-Parental%20Recombinant%20Inbred%20Lines%20Generated%20By%20Sibling%20Matings&amp;rft.volume=2&amp;rft.issue=2&amp;rft.date=2012-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Welsh%2C+Catherine+E.&amp;rft.au=McMillan%2C+Leonard"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459235/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459235/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459235/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459235/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459235/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459235/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459235/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459235/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459235/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459235/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459235/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459235/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459943" id="node459943" data-pisa="ggg;4/8/1429" data-pisa-master="ggg;g3.114.011759" data-apath="/ggg/4/8/1429.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/4/8/1429" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459232" id="node459232" data-pisa="ggg;2/2/199" data-pisa-master="ggg;2/2/199" data-apath="/ggg/2/2/199.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/2/2/199" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Haplotype Probabilities in Advanced Intercross Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mouse%20Genetic%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mouse Genetic Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/2/191" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/2/191.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/2/191.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/2/191.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/2/191.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
