<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/12/2711" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/12/2711.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/12/2711.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/12/2711.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/12/2711/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/12/2711.atom" />
<meta name="HW.pisa" content="ggg;5/12/2711" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments" />
<meta name="DC.Identifier" content="10.1534/g3.115.021105" />
<meta name="DC.Date" content="2015-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Kumar et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The nonadditive genetic effects may have an important contribution to total genetic variation of phenotypes, so estimates of both the additive and nonadditive effects are desirable for breeding and selection purposes. Our main objectives were to: estimate additive, dominance and epistatic variances of apple ( Malus × domestica Borkh.) phenotypes using relationship matrices constructed from genome-wide dense single nucleotide polymorphism (SNP) markers; and compare the accuracy of genomic predictions using genomic best linear unbiased prediction models with or without including nonadditive genetic effects. A set of 247 clonally replicated individuals was assessed for six fruit quality traits at two sites, and also genotyped using an Illumina 8K SNP array. Across several fruit quality traits, the additive, dominance, and epistatic effects contributed about 30%, 16%, and 19%, respectively, to the total phenotypic variance. Models ignoring nonadditive components yielded upwardly biased estimates of additive variance (heritability) for all traits in this study. The accuracy of genomic predicted genetic values (GEGV) varied from about 0.15 to 0.35 for various traits, and these were almost identical for models with or without including nonadditive effects. However, models including nonadditive genetic effects further reduced the bias of GEGV. Between-site genotypic correlations were high (&gt;0.85) for all traits, and genotype-site interaction accounted for &lt;10% of the phenotypic variability. The accuracy of prediction, when the validation set was present only at one site, was generally similar for both sites, and varied from about 0.50 to 0.85. The prediction accuracies were strongly influenced by trait heritability, and genetic relatedness between the training and validation families." />
<meta name="DC.Contributor" content="Satish Kumar" />
<meta name="DC.Contributor" content="Claire Molloy" />
<meta name="DC.Contributor" content="Patricio Muñoz" />
<meta name="DC.Contributor" content="Hans Daetwyler" />
<meta name="DC.Contributor" content="David Chagné" />
<meta name="DC.Contributor" content="Richard Volz" />
<meta name="article:published_time" content="2015-12-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The nonadditive genetic effects may have an important contribution to total genetic variation of phenotypes, so estimates of both the additive and nonadditive effects are desirable for breeding and selection purposes. Our main objectives were to: estimate additive, dominance and epistatic variances of apple (&lt;i&gt;Malus × domestica&lt;/i&gt; Borkh.) phenotypes using relationship matrices constructed from genome-wide dense single nucleotide polymorphism (SNP) markers; and compare the accuracy of genomic predictions using genomic best linear unbiased prediction models with or without including nonadditive genetic effects. A set of 247 clonally replicated individuals was assessed for six fruit quality traits at two sites, and also genotyped using an Illumina 8K SNP array. Across several fruit quality traits, the additive, dominance, and epistatic effects contributed about 30%, 16%, and 19%, respectively, to the total phenotypic variance. Models ignoring nonadditive components yielded upwardly biased estimates of additive variance (heritability) for all traits in this study. The accuracy of genomic predicted genetic values (GEGV) varied from about 0.15 to 0.35 for various traits, and these were almost identical for models with or without including nonadditive effects. However, models including nonadditive genetic effects further reduced the bias of GEGV. Between-site genotypic correlations were high (&amp;gt;0.85) for all traits, and genotype-site interaction accounted for &amp;lt;10% of the phenotypic variability. The accuracy of prediction, when the validation set was present only at one site, was generally similar for both sites, and varied from about 0.50 to 0.85. The prediction accuracies were strongly influenced by trait heritability, and genetic relatedness between the training and validation families.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/12/01" />
<meta name="citation_mjid" content="ggg;5/12/2711" />
<meta name="citation_id" content="5/12/2711" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/12/2711" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/12/2711.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/12/2711.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/12/2711.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021105" />
<meta name="citation_pmid" content="26497141" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2711" />
<meta name="citation_lastpage" content="2718" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Satish Kumar" />
<meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand" />
<meta name="citation_author_email" content="Satish.Kumar@plantandfood.co.nz" />
<meta name="citation_author" content="Claire Molloy" />
<meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand" />
<meta name="citation_author" content="Patricio Muñoz" />
<meta name="citation_author_institution" content="Agronomy Department, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author" content="Hans Daetwyler" />
<meta name="citation_author_institution" content="AgriBio Center, La Trobe University, Bundoora, Victoria 3083, Australia" />
<meta name="citation_author" content="David Chagné" />
<meta name="citation_author_institution" content="Plant &amp; Food Research, Palmerston North Research Center, Palmerston North, 4442, New Zealand" />
<meta name="citation_author" content="Richard Volz" />
<meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Crop Hortic. Sci.;citation_author=P.A. Alspach;citation_title=Estimation of genetic parameters of apple fruit quality from open pollinated families.;citation_pages=219-228;citation_volume=30;citation_year=2002;citation_doi=10.1080/01140671.2002.9514218" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. A. Araújo;citation_author=N. M. G. Borralho;citation_author=G. Dehon;citation_title=The importance and type of non additive genetic effects for growth in Eucalyptus globulus.;citation_pages=327-337;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-011-0443-x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. For. Sci.;citation_author=R. D. Burdon;citation_author=C. J. A. Shelbourne;citation_title=The use of vegetative propagules for obtaining genetic information.;citation_pages=418-425;citation_volume=4;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_title=Genomic prediction of breeding values when modeling genotype-environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Chagné;citation_author=R. N. Crowhurst;citation_author=M. Troggio;citation_author=M. W. Davey;citation_author=B. Gilmore;citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.;citation_pages=e31745-e31745;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363718;citation_doi=10.1371/journal.pone.0031745" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach.;citation_pages=e3395-e3395;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852893;citation_doi=10.1371/journal.pone.0003395" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=H. D. Daetwyler;citation_author=J. M. Hickey;citation_author=J. M. Henshall;citation_author=S. Dominik;citation_author=B. Gredler;citation_title=Accuracy of estimated genomic breeding values for wool and meat traits in a multi-breed sheep population.;citation_pages=1004-1010;citation_volume=50;citation_year=2010;citation_doi=10.1071/AN10096" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. D. Daetwyler;citation_author=U. K. Bansal;citation_author=H. S. Bariana;citation_author=M. J. Hayden;citation_author=B. J. Hayes;citation_title=Genomic prediction for rust resistance in diverse wheat landraces.;citation_pages=1795-1803;citation_volume=127;citation_year=2014;citation_pmid=24965887;citation_doi=10.1007/s00122-014-2341-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. E. Durel;citation_author=F. Laurens;citation_author=A. Fouillet;citation_author=Y. Lespinasse;citation_title=Utilization of pedigree information to estimate genetic parameters from large unbalanced data sets in apple.;citation_pages=1077-1085;citation_volume=96;citation_year=1998;citation_doi=10.1007/s001220050842" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. Hill;citation_author=M. Goddard;citation_author=P. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. J. King;citation_author=C. Maliepaard;citation_author=J. R. Lynn;citation_author=F. H. Alston;citation_author=C. E. Durel;citation_title=Quantitative genetic analysis and comparison of physical and sensory descriptors relating to fruit flesh firmness in apple (Malus pumila Mill.).;citation_pages=1074-1084;citation_volume=100;citation_year=2000;citation_doi=10.1007/s001220051389" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=A. B. Kouassi;citation_author=C.-E. Durel;citation_author=F. Costa;citation_author=S. Tartarini;citation_author=E. van de Weg;citation_title=Estimation of genetic parameters and prediction of breeding values for apple fruit-quality traits using pedigreed plant material in Europe.;citation_pages=659-672;citation_volume=5;citation_year=2009;citation_doi=10.1007/s11295-009-0217-x" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. Kumar;citation_author=R. K. Volz;citation_author=R. Weskett;citation_title=Genetic architecture of fruit quality traits in Malus x domestica (Borkh.) compared between own-rooted seedlings and vegetative propagules on ‘M. 9’ rootstock.;citation_pages=1079-1088;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-011-0396-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Kumar;citation_author=D. Chagné;citation_author=M. C. A. M. Bink;citation_author=R. K. Volz;citation_author=C. Whitworth;citation_title=Genomic selection for fruit quality traits in apple (MalusÃdomestica Borkh.).;citation_pages=e36674-e36674;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22574211;citation_doi=10.1371/journal.pone.0036674" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Kumar;citation_author=D. G. Garrick;citation_author=M. C. A. M. Bink;citation_author=C. Whitworth;citation_author=D. Chagné;citation_title=Novel genomic approaches unravel genetic architecture of complex traits in apple.;citation_pages=93;citation_volume=14;citation_year=2013;citation_pmid=23398910;citation_doi=10.1186/1471-2164-14-93" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=J. H. J. van der Werf;citation_title=Using the realized relationship matrix to disentangle confounding factors for the estimation of genetic variance components of complex traits.;citation_pages=22-22;citation_volume=42;citation_year=2010;citation_pmid=20546624;citation_doi=10.1186/1297-9686-42-22" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granie;citation_author=E. Manfredi;citation_author=J. M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=Z. Lin;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Genomic selection in crops, trees and forages: a review.;citation_pages=1177-1191;citation_volume=65;citation_year=2014;citation_doi=10.1071/CP13363" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=P. X. Lu;citation_author=D. A. Huber;citation_author=T. L. White;citation_title=Potential biases of incomplete linear models in heritability estimation and breeding value prediction.;citation_pages=724-736;citation_volume=29;citation_year=1999;citation_doi=10.1139/x99-047" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Mäki-Tanila;citation_author=W. G. Hill;citation_title=Influence of gene interaction on complex trait variation with multilocus models.;citation_volume=198;citation_year=2014;citation_pmid=24990992;citation_doi=10.1534/genetics.114.165282" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=S. J. McKay;citation_author=J. M. Bradeen;citation_author=J. J. Luby;citation_title=Prediction of genotypic values for apple fruit texture traits in a breeding population derived from ‘Honeycrisp’.;citation_pages=408-414;citation_volume=136;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. R. Muñoz;citation_author=M. F. Resende;citation_author=D. A. Huber;citation_author=T. Quezada;citation_author=M. D. Resende;citation_title=Genomic relationship matrix for correcting pedigree errors in breeding populations: impact on genetic parameters and genomic selection accuracy.;citation_pages=1115-1123;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2012.12.0673" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Muñoz;citation_author=M. F. Resende;citation_author=S. A. Gezan;citation_author=M. D. V. Resende;citation_author=G. de los Campos;citation_title=Unraveling additive from nonadditive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. R. Resende;citation_author=P. R. Muñoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Speed;citation_author=D. J. Balding;citation_title=Relatedness in the post-genomic era: is it still useful?;citation_pages=33-44;citation_volume=16;citation_year=2015;citation_pmid=25404112" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Stranden;citation_author=D. J. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=T. Ostersen;citation_author=M. Henryon;citation_author=M. S. Lund;citation_title=Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.;citation_pages=e45293-e45293;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23028912;citation_doi=10.1371/journal.pone.0045293" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Sun;citation_author=P. M. vanRaden;citation_author=J. B. Cole;citation_author=J. R. O’Connell;citation_title=Improvement of prediction ability for genomic selection of dairy cattle by including dominance effects.;citation_pages=e103934;citation_volume=9;citation_year=2014;citation_pmid=25084281;citation_doi=10.1371/journal.pone.0103934" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. Van Raden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Waldmann;citation_author=J. Hallander;citation_author=F. Hoti;citation_title=Efficient Markov Chain Monte Carlo Implementation of Bayesian Analysis of Additive and Dominance Genetic Variances in Noninbred Pedigrees;citation_pages=1101-1112;citation_volume=179;citation_year=2008;citation_issue=2;citation_pmid=18558655;citation_doi=10.1534/genetics.107.084160" />
<meta name="citation_reference" content="citation_title=Forest Genetics;citation_author=T. L. White;citation_author=W. T. Adams;citation_author=D. B. Neale;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Xu;citation_author=D. Zhu;citation_author=Q. Zhang;citation_title=Predicting hybrid performance in rice using genomic best linear unbiased prediction.;citation_volume=111;citation_year=2014;citation_pmid=25114224;citation_doi=10.1073/pnas.1413750111" />
<meta name="citation_reference" content="citation_journal_title=Jpn. J. Genet.;citation_author=Y. Yamada;citation_title=Genotype by environment interaction and genetic correlation of the same traits under different environments.;citation_pages=498-509;citation_volume=37;citation_year=1962;citation_doi=10.1266/jjg.37.498" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. Zapata-Valenzuela;citation_author=F. Isik;citation_author=C. Maltecca;citation_author=J. Wegrzyn;citation_author=D. Neale;citation_title=SNP markers trace familial linkages in a cloned population of Pinus taeda - prospects for genomic selection.;citation_pages=1307-1318;citation_volume=8;citation_year=2012;citation_doi=10.1007/s11295-012-0516-5" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=O. Zuk;citation_author=E. Hechter;citation_author=S. R. Sunyaev;citation_author=E. S. Lander;citation_title=The mystery of missing heritability: genetic interactions create phantom heritability.;citation_volume=109;citation_year=2012;citation_pmid=22223662;citation_doi=10.1073/pnas.1119675109" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="twitter:description" content="The nonadditive genetic effects may have an important contribution to total genetic variation of phenotypes, so estimates of both the additive and nonadditive effects are desirable for breeding and selection purposes. Our main objectives were to: estimate additive, dominance and epistatic variances of apple ( Malus × domestica Borkh.) phenotypes using relationship matrices constructed from genome-wide dense single nucleotide polymorphism (SNP) markers; and compare the accuracy of genomic predictions using genomic best linear unbiased prediction models with or without including nonadditive genetic effects. A set of 247 clonally replicated individuals was assessed for six fruit quality traits at two sites, and also genotyped using an Illumina 8K SNP array. Across several fruit quality traits, the additive, dominance, and epistatic effects contributed about 30%, 16%, and 19%, respectively, to the total phenotypic variance. Models ignoring nonadditive components yielded upwardly biased estimates of additive variance (heritability) for all traits in this study. The accuracy of genomic predicted genetic values (GEGV) varied from about 0.15 to 0.35 for various traits, and these were almost identical for models with or without including nonadditive effects. However, models including nonadditive genetic effects further reduced the bias of GEGV. Between-site genotypic correlations were high (&gt;0.85) for all traits, and genotype-site interaction accounted for &lt;10% of the phenotypic variability. The accuracy of prediction, when the validation set was present only at one site, was generally similar for both sites, and varied from about 0.50 to 0.85. The prediction accuracies were strongly influenced by trait heritability, and genetic relatedness between the training and validation families." />
<meta name="og-title" property="og:title" content="Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/12/2711" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The nonadditive genetic effects may have an important contribution to total genetic variation of phenotypes, so estimates of both the additive and nonadditive effects are desirable for breeding and selection purposes. Our main objectives were to: estimate additive, dominance and epistatic variances of apple ( Malus × domestica Borkh.) phenotypes using relationship matrices constructed from genome-wide dense single nucleotide polymorphism (SNP) markers; and compare the accuracy of genomic predictions using genomic best linear unbiased prediction models with or without including nonadditive genetic effects. A set of 247 clonally replicated individuals was assessed for six fruit quality traits at two sites, and also genotyped using an Illumina 8K SNP array. Across several fruit quality traits, the additive, dominance, and epistatic effects contributed about 30%, 16%, and 19%, respectively, to the total phenotypic variance. Models ignoring nonadditive components yielded upwardly biased estimates of additive variance (heritability) for all traits in this study. The accuracy of genomic predicted genetic values (GEGV) varied from about 0.15 to 0.35 for various traits, and these were almost identical for models with or without including nonadditive effects. However, models including nonadditive genetic effects further reduced the bias of GEGV. Between-site genotypic correlations were high (&gt;0.85) for all traits, and genotype-site interaction accounted for &lt;10% of the phenotypic variability. The accuracy of prediction, when the validation set was present only at one site, was generally similar for both sites, and varied from about 0.50 to 0.85. The prediction accuracies were strongly influenced by trait heritability, and genetic relatedness between the training and validation families." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
    <title>Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Xcw-BASjtb_8kQrvHM08uS0zzgbw-dPaj3scztJS-Z0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460225","apath":"\/ggg\/5\/12\/2711.atom","pisa":"ggg;5\/12\/2711","ac":{"\/ggg\/5\/12\/2711.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/12\/2711.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2711"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2711"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/12\/2711":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OarJp8fe83XhdUNgJTCLHgVyq+vUSJJSsSmW3Ent5UA2tcJDXj9bG2SXb8WZFCJGnV3kJTIf+L8RUX+CV+WZtMBtNb6KsQA6YsRLTcaEUIaKxHRuwmIr8\\\\\\\/6\\\\\\\/iq0J1y\\\\\\\/nohckTvZDwY3ekH3jlg+vuM\\\\\\\/WmX0ysf18M3CLLdJ+0BtPsfkPnQ1nUy2ExHhR4rKRBNB6k8HKgdL7b8UNfyVtcpnjfm96XIuXWIThCs9X8y189VfObGa5nimmqoDk9zuyYhCiR1Sz3znfW6rfsky6MYPCeFl5l8aZau6IH9W5RW\\\\\\\/VelzsfiOCzpcsrdDh1RBwdfADB7OBdLPi8oaWDPcwmkJVlI5K1IzrsBAD2rRI\\\\\\\/wRs1NmyKbaNEWtSlZSCx\\\\\\\/8TrOpXO6dWEE23QeMPlEdtroSdT\\\\\\\/tK4HUQDagJCW84Dul2KNh+niADix5gtz8+ZIxV3ftSqqSU\\\\\\\/nh+E1Cp4zTGudBE0wU1qF9m+Dvbu6gnnN6LsFdyuBja\\\\\\\/W7hwm+3H6tlW5yinAsX0A1tHKPuppKIQVozKzY8HJQ\\\\\\\/AxpUOiZQnMdWV8+atnJHbzZK1\\\\\\\/qu8jzEohicT7msGbeM49d\\\\\\\/HlqJK4MdMJJhzAvPgdIi4jRGQptwszQiG8dTg9Z0VlwKrQLKzx2fIce0aXItPdekjKOJgKHiikI59sDblM7hU0vXhGXLN+My2cFCkOOX2ob8CBGL\\\\\\\/vT8QQpv4WOUGepKfxRFADYcuDlrffoSAWHO1lToumYF5nrgHgMbi2QqHBiRbqa3gmoIn17cv0isId5wV8Mzt8khw93OU9AIaEPu9Ei8GmUJii18iktgigTJuT8\\\\\\\/P6JYNUkZc2QAxAWlSkrbWRkXKQlbk7sH31xP\\\\\\\/IBE+LhEZuZwHDvgB\\\\\\\/1sWFX8mDIoG4Gxy+6pJjqcyV\\\\\\\/odZBjS1oUpODhc7o7uUpfvU2+F1496mhUraCWio2tFD0ZUsTOprNLDrLnAotIEJzyuqLmlHw1u5OAbW4RoOabW+dkLy6dF3s\\\\\\\/jRlp6waRdVJn2FvmOJTLYMS9MeaLBMwmHurEQiNMa0NjVP6QuXRYAwgnsH6XGtfCn11eqFAyZP7JMGPBEMZ8hSk8ksgaDDb+0Err72y0hHKq+ULCnzOtt5\\\\\\\/fnhXnv0P+y1RYJti68fInnVSEEvf\\\\\\\/xsvTQe3XFhlM0BoFThR7TM+hycdYdcFodi\\\\\\\/Qk3UNn7jWeZw7inDsAtvJro2lAztktfRZ54hdSg+SJOodWOP8qZ6Ar+FsNSACcGDRxdvzjSYfWBnY8EwdebHu0AA\\\\\\\/uwUu\\\u0022,\\\u0022iv\\\u0022:\\\u0022o3asChj1WXXxAG\\\\\\\/GDUN\\\\\\\/4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fc05398d5bcf78a886de6483d739a14822ca352404ce369014d5b40459c11978\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uW4xIPj1mGxlm7InJ88zFMmJJdhvDmgx\\\\\\\/nTg01xzvY+91mP4QO+P\\\\\\\/vzMe\\\\\\\/GY2SPquYvtWfnMhv3beUabksBgwhE7UrgNsnJw2Zmy3N2wJ7fLKGigxIVcqFPlFcmMOVQhnxXhapB5BIurNV\\\\\\\/0QWoVKTC+fxQfdH+3UscHAD7RDi9MMM4BJ+MWgF+uvhfyipvqJT1BVC3LGoRjPzuvUljHqwvb1mQbiwxlLtQl99ygDoFThwVNNGVYn2Ec6V6OB0Bj0gB9h+GbeTFxdz8uqcQUJbVeGGL\\\\\\\/VlgRjAyNBBSVwuqyaHukzIMswpQqDXDlmWU1ynQQ7Mo+jNxfTrqBPMnUogLYdjAUr0uuO3xSI7h3ouNfwjA0oDBX9Q7bv6B8ukeDobL1AXOjyau1cE39nQQXIo9dam0Maos3fE\\\\\\\/p1cHdlbj7oFK5agxgIjtcQAPMz\\\\\\\/BGMyXOaJCtEywR6oxL0DWVjlsZ+LbnyEvkfaaGFgf5iHyn3vWjjql5SFejZ+63rt8qqEsVzSfVlNumpXBRK7BRjGvsknwh2d4rVP2YodoSAoNOmifH5wNW7iT1JEINOxqFPbKYN7RNnA9rjcjNPIRBWefBcWFc5PaC2o6g+2hQ7Xvog8YrIUpaZdNhevwGUmKrhCq33zxPzka4m9Tx0kApoIyH32DAnAjCrl\\\\\\\/74RUU0gdQos\\\\\\\/nsvdIczgYc+j7zykH5JGbdPsmrYXYy8ChM3GHfR61mk5jGGqdoiD+NiJty6Dz1Fc\\\\\\\/jPnPkBw+j2QSTWsTHSSjLcvqSVCTs7GsrlYNbNgmPKZNAG5Pkc1s8GT8qymJ0zPTp\\\\\\\/gNKv\\\\\\\/ez0dSWRcKh3KzGo1L7slXZUOy7BYaBBji6EUuX8SDWW5Aq6o9WgPyYsnYW3jRSA6mdspwXn+rkFeh11t5y4Hztghbr1\\\\\\\/lnzJCJT94+XuHq4ZjMt1NZvPyRiDE7kIVpWp5WcuHIm6lFdsJAJ3+M2S7fCTjbSWAfoN4gC6s06XhEe3KDTPqQcxL9KWrb+ISIDmzFbGYrU+uWKrIW8DcqQmfaTTDtJLZCCJJOb+6wqdcBGJBra\\\\\\\/e3wiV8V1p8DcL\\\\\\\/imfxU6G+Lyl9xHOJaZmUy71p9qKl5i8uJHxGT\\\\\\\/VdJkpN\\\\\\\/M\\\\\\\/H7r\\\\\\\/uEATdS4032kFLJBg\\\\\\\/v0KXIX3lccBR7BcTMn5EQYS+YGfrDcXVIgCufQb1WyOt\\\\\\\/r6oNzmuJkurcpNyzyePR3ap0+HG9GgRClvc9H01e0JsY\\\\\\\/Sua9V5asNOJ7m+8ouDi\\\\\\\/hHdqP6LIM316Vogb8S5MLsqH3d164t\\\\\\\/ldbU05PCFLfbA0DH1WZR9aIJEAojTfmuRfQ9wyik5b+J4RdROAcBiGnKheUMzD87FDGhfjg8+HQeRuEiV68dT5bDGnIFFVl2Hmhaw4So6k15cpoHLbyi3Pje27Jv0NR4aDOdWK8J6jZv3YVREGHc9VdFJXBPzUNAlkoVxIGP1XWE953t\\\\\\\/ylxx+3I9YIp5+FmZ7yYIIztEVcWIdHOJOKxaTnXFmEdsg1OsIz+98KDhxEwEG17iUCm\\\\\\\/+UwLPo1SBUEtdEG9QxnotBxa6VrLv9wubbrFuCSBAprzEc496\\\\\\\/UdFJBbTx\\\\\\\/t22OkCfDysKfnfj46tF5FkDaOwawR37+my8eiMuA3seGsrturW9pVyMysHWo63+zTK6ISSx1Xi8wWYN+RLx2G816D1yHEOeO+mCU6DbD\\\\\\\/LDp5or\\\\\\\/4xywmGthyH01EeFzO\\\\\\\/NjNLjK1LnM7EoSf48bmEpJntQ\\\\\\\/9MVeCG9wxr1YFYZ4327ZWHH1+zy86zJ7jrNXmTalmhcm28A1tEfye5D7CSoMUY9U3PdzHEMkQFt40Qkf12FwWi9kevf7I1UNQMaRhPMMJEcoZsp2uTaA3gAXyhqbwGnNcWoVHGJE7lGA3\\\\\\\/YUbPHPJpJ+XIenKv+o2CoWEtd\\\\\\\/d2EdKLqWr7SoEvXfIS6jxhIp3CaDzP\\\\\\\/6If5qkRuXlX\\\\\\\/dMyjhFKiMdPSQd+xoT+HB9amstiJvFrUrsqYQb+QUGM1vWiYGDeAnjCRyPr88sPSbUStdG39Vfl9J66uAhKFiAMsoC6oMyg5etlQgqO3G4oE8su4MnxR\\\\\\\/rMsO\\\\\\\/UP6Go+egKOCHYX6qb7GkHdpFv2SVzOcQTbIS2aLl0LLiljTsKCyYziCMv3QuddL98Pc3qrodTQ5FhGIeTnBLSzvKM8WuFlZqiukpK7CY6TTMkRs3tkHUojsEtzPpmGTB9\\\\\\\/y9pwuZANx0BN6m7W50ydXvadqRZctqsxyXKaFm7WsQ2ESzhQU8vhT03gjk30lsLpIECYG80+D6mNlm2TpFUpiaGBt8OObCiKFGgf02sxmAwaKNjNmysdAIpmFh2UcxzrEilnGTii9hBtUUMMWCJs3ojAaiHBQ1BFGY77EFASxunYQ1z7ZiQrXcGrJn70gYq9FtmxuKugxT8y7D\\\\\\\/3GmdHIoL00m+URnnak7RyeMQsO9GN3d64kugvPdPdTZKqSumLv8hCBsjm2jj+ZXP7iKuMzMDFx3X9ITYRL+4iyypV1fauNMwcI5viAz29RMCld8bA+rZ5e5SBGrEJKRVH13pwl0lIiLlUJqPllmCeHHe5boL5KPnhN2qx7iggszLHSsH2XssFUI5VEKBvjD7JsHlj95omJl4m\\\\\\\/8rnbIN8sqmUT5cAYVoyHxdwUqRjmNEaHCdI38Ye05AGhC+mukGcbXgMY9RuMVLZ0lKdvdjtLRyoGZ9f8t5+0PBO7zhsFlBOUofTjqGxXzmojCXKzK53POw43xqRaDzgRGw3wgDTNCZr3PN4+oKtIFWCKhuE0a1H+BZj9wPM32mP\\\\\\\/fJ73qBR6VHjhhqjchXMUFdxfsahkqoDBDFWpyItp1nvKe\\\\\\\/8VEpxnCTnvf7\\\\\\\/VPff5+YqIFm4q+moy1fD4xvzxvn7qov+dGQB3dn4o2Q8yNgFbDmJLxw7nBwxv5ImPRmLHj\\\\\\\/LWuZOWGzLuIIo+gExYpH35DvGtkQM92uFsFHwdC1zJduJCaOLKQ5P3K7z+pJFmVbZx1zhS3mDaOOXzto8NLZ0fv2BWKPOru444UOtaHindC\\\\\\\/K6oZfPldpTi0E9oncg2CG0apSkx8VESQsaDGYRaYL5BSOobWUJN3N4bnmwNBM14ZHhqxKEI9VxOfMpSLyaK6IBo97vpma6pghN0NHajV+NcAr8v9YQD0LfedN0QHgXvt66BFAg5Y46iFqifis0KQOlY6MI43Wght7qWBNX8qLZU9h7tDuDTnePcXNmjLxiiRdcy9KooNtNSflKA9DuD+xBszrnehguAdO7qcbJPC4Kp9iyW3Z8slIzi+3R1guF0qWdkv106FixPK39ivVEasBl8Vo9SKBk+CzRc8NX13vDLlGRwpOzeQKqagWVkMp6\\\\\\\/lrnONu8O5yHkRpyac2lBozgHONV7BcyGwxTSkxrE7KCy\\\\\\\/zN5n7gJQMnFIlP3m3UXX6tjhHPtRRCfcJ0TUAUtV9a8KwI3BalVM1cvn81K7Kb1FbyRS63zdWnRvqtgmamceep9KwLBQ3KZ+ZCbldV9dSsVZqAcWj94b59Ti3vXfsZsnGjyHNR\\\\\\\/oLtFfm16ssTc0PvPh9nc\\\\\\\/irn7tB1Sj+tNKz\\\\\\\/9uqmlAD3N9KiEWT6wxr+15esCSoovqHwuPp0SiKE0HZK9RPxfv7war5+w0bcs3eF+FeAigSExD7q9nxW8U4o0ZPNGmPCJF1DsjXga37fIQg\\\\\\\/FVYJO8NE4ADOzwZch5zLUZeSP2MuO+UsWlSegvQx3PBgAGF\\\\\\\/OTEPFBgu8nUtD2JbisjUIvC2J2jb+mEcaFG\\\\\\\/uMlwOqyW5ir3L1tGctVbMHJ7I6az2k0TTX2vICX1RbbBbju2Oaopkb+iK3NUMwd9OfIUa5rwi79RC6x3EJ9bNClgJnepLQrQoKRofn55JteOy5VOj\\\\\\\/B+FdpCpUWBqWOt07KL37Mt9gEx\\\\\\\/TQOywvAuzy6jz+76HVR\\\\\\\/olpT49f7j1wsMbTqpOuQ5XUGb5FzRMjCO+AzpeWfQaCTZgeVmc01XJ4zKHrqkV0BIHEoe5fD5sYymgloLFfJdOERtGKWt70BWRS0s60qLMr08QviYgHdqX\\\\\\\/xk6Z+AW7Sz+A1Hvb2NFVQDA9BIVYbqBmmQHx3tStHktBISyIuvsR0ogU3dHcbfD5EnyWJuq1aYJMDpg2cBbZPGwn1SUGqW8wQYt5AMwBFNM34UI8pJmw1jn+4rWyRW39Kd1mI9OtrrhuETmyvqlUncJKuj5hAzC\\\\\\\/GqrU4lDbsdPrGJL2g+F1tja2aTvXzSrAJAmDU62Iid\\\\\\\/or1wSoz92oX9d9Th5xZu68qtx2\\\\\\\/arLDWZmUIG6DMlx4zULOAzN\\\\\\\/f8y2UFAyCTZP3NiA5dd04w2gppNxjQPBEkZB+VSoObOrxGu\\\\\\\/FHX231R0jtgbuZpqp8Tl9slYkM+oMP2BDkjEw5WE4PTNJpjtl998j\\\\\\\/+tF5jCHrl1ergeVNkLN7xLfikg9+hw21SJHyooyY07bfcy1HnoaOiSP1\\\\\\\/BFVbX3JrMYERc7BAn0+\\\\\\\/nGjlFxvMpg\\\\\\\/uhPnHxs0\\\\\\\/CQQUdNf9jOXJcWzFJS9wiW9Ol1QnfBJCJfeGvGAdxjf9zYkjgx1qZ+uQdeQL\\\\\\\/Q2SJMCLipdXdwIhYfTkyikwOUG\\\\\\\/oiQED05GB5fjf5hcja1HLqtEMDf6VHLKYuATwLT7bBcfKUQwMqXVqWVgk8Ib81lDjlfRL5onNsXcQNrPUNBMugNCC+FpjUYkQ645N\\\\\\\/WmbXA+vcGjwKAhkyMkbR0ZEAQdoYfsZF4qlHV9kj68LAIFfpErR1\\\\\\\/3e\\\\\\\/RjO1eLRHLoucSAQfkQquUkCCYdGSOrIVykdUy\\\\\\\/z1tJ8xsA14bhYZgUR1jzDQzRXOPZV8v4gLSQfm7htJ3SUkENygvCHcHrqepqH8VIzub\\\\\\\/+KBKkrWgs2FdeHc8e8YikvIUauEPrih879NOTgu2cLiS7tT9+NccDD3PDJwvxPK9irNonqELTHiG1+pEMwCyhVnyK72mSk+4+ytRQtUnso2rjGdcKKcsqfSJ2phtnAXzAqZRqBgoEYvWlfwtLUvd2akToWk9dAnhXFTRASuw85CIx5z7i53aOhg7VE4XW+V5XvZ3iRukVsDpLpSq3LYcQS92P1kXOxWbLnzSUqc2gIj9c\\\\\\\/gh\\\\\\\/HTh20RC0M+WOyNaYShtFQaemZTqmeycULrJPhkiiUz46YUdxPUQ9jfIGF27lhWaMKRxqVBTDNt+6AK3qz88JiBT41ooOhCWQPcRN0OQqsraHoT70Gjw+K5eox8rFKJR7o7vaQ7up959tkSbOR63K12vyedgJtne+AyDOdxdNICkikhFF3n1\\\\\\\/aaFLKEHefa5xuuuDlfaxd9SkkenaZrQiDuda4amiwcInBoV5Td4UyKd5FJwhIdla+B\\\\\\\/cqNLnavY4kLGE0qRNblDP5EjT+BiDZ+6HWunBKQQGSFfCyxyRKTpb9ox+eVwGpbCiWokW85\\\\\\\/+YwLwmgNojIHIHFCeDK7kMtJ+d3\\\\\\\/EmUK\\\\\\\/ZVpMqMy1FckDRzZKA4PmapUIvvb+x69hPA2eRcKb96ux4Sc6PvmY52z6bRWaBcmsgUHOJ8BB38ynPUOnuNnDixPLboThawJQSmVB1QWDJUbGIB8AK2n1m8SVHkHlSAY\\\\\\\/U\\\\\\\/LThu91lFR61axw4m7yt6YWyp+hGQvbdrU2ryLtS8h4Y8O\\\\\\\/I+WbBaKSlmukBDcVN98e5nUxpdsIqOyTyf1B6PFcVhmivhnfJ3RAGyVNqgAHtK79CDkeeormy12IhFX4XnIT16Ke3gaL\\\\\\\/wXKkMUnceaaOUeU09QcXbR6bqrG4eUPeE0fDkKijSkDEp2jhRoiQhptyKqOMYwlPQ\\\\\\\/HI6EOTRmV5\\\\\\\/8EkycgJmltfsUZn58MkCM9IBfRTHFJwZ0gKCpPl8gTkQqaZHOijHcrkhXtKj4YkdbOJGc9YtQjhJbwiNCXKycc7M8C3UIK\\\\\\\/MUodCplm54064eqQn6MdYAe9VDeXPnHhTII3SVmKliKFtoUJ+DttOz3nYVVeyjdDpL0+fjEKMrVhb\\\\\\\/DHiQH49l8ZMswdvq4JDqB1qIhzNqiD7KP0EiTf3CKJK82FG+1xGqq3I3O2iCNHzL7qitOR4At4\\\\\\\/NG5HQp55aVl206FhjwSb3Czm9j\\\\\\\/H\\\\\\\/s7LN\\\\\\\/AWFiIoH6Tv60ICqzY1IAoqJRCxxXtyO\\\\\\\/5B+oTBkU7cbltuXkt\\\\\\\/imOeag3nREnsDdYlHYAkMnj+eO7XktKWc0+VjQhvdDxnZW7NhDgKbCB0e67zmcAb6sMl1UY3DXCvYNaSsccQRE5f24HJoHz1c5Oh3uYSth+DKNevkXMdRZCr1m3I9zVCqJiaHXP2ZeU3SM2SsxpdYlEmEm0y+qlLJdWVAAJpslzoAGIUuD1l8gGfjwMlB+VZbAWyG5bd179tQxgPB1bz0iWQ3sL\\\\\\\/9qX+jGejTTL1+1vuJY7NCb6DvlAKCzYXSlybBGVwKXK5SO3TG2ssdtJedj9W3eRCi+yUpgsABlfqrkSU4du2CztiZ71jXSMaXY2Pj0bfAi4akyFN\\\\\\\/Hhn5XI6EvQtcT+oLgCOMTCxqn9e+HuZS+fJ4iXi2rRIHoVFL5j3\\\\\\\/jSYvwy7Xt9VY61UKyo534v01bOev9AiVjIGT2yZSyUrckVYgR56wLcttGkK2L\\\\\\\/iMfqqKRC4ubFM5ST14MctaeyVz6Ouloo2pkyxrPtQZmaNQ0mRA6HrG0ZlcFG+nX0q0fHVWMBQ9ToFB33oU0GnsbYJIKvQ1AehFA9UvBZ+as9AzVLpr0yP4A5DA0i4Nlbku5D1UQ+8Hu7PjQgAb6T5rjLvsEdcCjieiljvBt3Ib7BjuGydKMyzCok8SxRgd49W6KokM9Ric0LizQFwIrhIaNpSjFAdNG7tZdAmLEs6SqsvLT3QZHyDxnR10IWsD0ywDXkkddXsPyTWjW9vmxM33+clhlsu+NhGsxohbsY8D3OhUxYP09Ty7ivnp2\\\\\\\/W7V3ANF4uE7Rlrl1hauD47A5q2SNby5NIoRv2AQrV1b5tdvucKDzSE2Kkcc2r6p79VFyfxCNSIccEI14GRO7nABXt1AaqkfHwc5wU2PokZF0cr7fo2SD56\\\\\\\/u6XDOjuTYzKG5VRdkLLLlqZ1DnzpMKFXuuaTzzLCQfzqo4dtF530Vab6CHdEx3fGEC5oz\\\\\\\/rrhLM4Msi9oSn9LARSQ4rrKayw2k8p4NeXvnof1gl7dXqymER2\\\\\\\/my9KqtcIIbCGoc3ge3f\\\\\\\/m61irxMnzbzzCSua6dNGibHqLSvsrdV5c3YJ9r\\\\\\\/ssHydbLgEh3bXWVLZgqNiTyMy+Z16Zv+zJiqYDv04i6zsgV0KO0J5IbU3cnjizywmLhRjkXaUHk3wI\\\\\\\/a\\\\\\\/ZNAbjCoPLFmYm44lakw9SBEjhuNL5qvfeA+xxrutgK0aE5xDV7xGySQ4PHkxKBYnjuHppTG\\\\\\\/G7QOov2nFmFsfV\\\\\\\/ZtmAdQZoCOnPwIxLndgOXVwdF8DshI28uH64AXkOCaRxG5g4VM3RCUscVS2QShSycBwJBoEq4o9rcbq7crKTT5n9wC3kabcBXN4zvdWccxIPXsyJODY6X+nGRA\\\\\\\/Q0QkxECCYpcOpk3ngOHB+K9dMM2Gn49xIz52ClzFj3JkrMUyHEinvqiyCiaU0S+J3Ge\\\\\\\/SsvxROK05\\\\\\\/bP837Ys+qm\\\\\\\/FtAIBf\\\\\\\/68vLLadQ+J7nprhdDy0CH2g9CjAIBwVr2j2QH9Pf+MtknxlNxfEEBj4OrLOSLJiPqzIylulKIsGMHvBUawTNf5vbzrDXL4ZJBC\\\\\\\/Wh1zeOQhoh8Ek51PW3ik1e81MddmsOkAnJ4v\\\\\\\/sYwwODtcbm4C\\\\\\\/VnAzq\\\\\\\/6NWFW5uueYsWM1jm\\\\\\\/GQVU1lA4NWDPLTHIbsZcGxtnOJMDPBEzkMNhMUgBeUfo0xEDCxvVHeAup4aHaS6TOq+MmhHYHDAi64QBVd8HmtJlTgrenZGwO5r952suMW10oCKAEYyQQp4s7pFDZs9bqESMnkjB3Z+tF4oXSTdPoU7tuEP+N7YRgQa8gIxYnnZIDemizYd2f04FLMr817tGFOhaFDDigp32WFH7C\\\\\\\/AM73\\\\\\\/viUMpAVxiw5BekrPhzrUiTx8JnJtl+Ha4pluf4Otb8707V5OVIE9lVdQgg1HY94iHEGw5gbaaSrm9+cNRSORNVLgEHdGKfhgeanyWeKqhJObXBVv2he2GE58XLX+sjPkK61DVjx1ewRtDcLx5ZIQCtqxSZSq4sukoAt49tWNxodr+csdr80OndXKjvciIVjA9em\\\\\\\/qWIKyJLKBSa13XSFDRiBvLpDGYIU\\\\\\\/kMwoYrEdmnOIyPSE5kMGFBkfZQSE7x7oY48F5ZF6Yt\\\\\\\/wIYQFkh38+\\\\\\\/0vL4ptnddZIncDzm5Nr1tvJqLPVA6dMCW2iOOU54\\\\\\\/arItbAv4LS98wN\\\\\\\/V9+fB5HI+g4Aii3iBU+lWou14dY0Ap0SvYmGhkpWCsHWLrraoOcNUt+Y8Z80u\\\\\\\/ncz7rr8CgcneAKynPxRhYcCEVZui3AJaG22X36dtY2vXiuSox+Ry9hONqQb1+6ZHDEw3lJPPf+S\\\\\\\/IeTirxQ3ic+OpTyh1NKLnD5hw50zuANbGArvaP6Oj53Vm672nA461V68skCAmJYtT4g4PYtwAZhSfgIEfzG7s3qVtPKmO6ON8tj7TnK+iTZMgl6DfyzzEC6hi7CCszK9vvnpLJA0v9qyPBa3SHAceQ3MD4GP+HvIAs\\\\\\\/p2XfF0vkOeskPBkONGqW1E97KPNbG\\\\\\\/WEb9txNa5okcF4XbC+CzUTITVo1D5QO0\\\\\\\/MTb38AS9irsLKA04SpLphbPrItQ975JbJ+pV3eAMnAKiYeLD4dnsTfne2GmB+g8Tq8lQs\\\\\\\/GZSuwwq9BKFP1UMBbZ1APcKJ\\\\\\\/8Tr53hVy3+Jfkz4JkHAzDGu5u0uS8ia4zDl5L5f5RCWrfTfHCGgFT8tp+AXC1DvDkXbMyho4yQCfvwzCqw\\\\\\\/KN2FFuEfm5bxKpLo7faQP3hX0WHLrT7FTzTfF7sI2Z8Y31zRrxc2sQdFYhgFQvgDe6p06gIm8HO5MLRue4YduZNWkA2k1N7qX4IfUAzIRJZbM3HmnW056+\\\\\\\/OysXXIEaPZ6Ee4Foal6pjgupbFI6alx187eTCppx0WWeSxeZ3OIBRiDdxXBzEf2bENZZ8B17nlg8E5vmeJ5wGC4ZGoDz5Ib9fslaoVc2b+PUaC\\\\\\\/+MGjt8Z8kopYBZUgAB9ejI0izHObD\\\\\\\/NcHH3mMo1ngi66K0hDK7VkhQVo2lS4MYnxokRfzZNJNLVwvY\\\\\\\/r6XAXMV7en4K94dalH0nKEIJ4kzvolIawI1gaqUPfK+wKzGMvu1af7MZi9GZMCQYLvbn\\\\\\\/OEC1L5cyOu4PAc\\\\\\\/kG4EKlRhrGfOCKIzYminKM\\\\\\\/0pDP2lI8qCAIdEWnh9epLvPl5ro7mDE1pQBBNJBaVRelImnyEL6Yr3j8F1+hpHbGL+oFwE0NSirXGr4YnRYDzOJHZwxzoF0LW\\\\\\\/qB0aFZF6h0FeqlG3QUo8oodOExRoabowX8U6c4dvf22EP5C24JNpfhHLGbGscajJLIvEq7NBiPVTuEuitKBWBL6I+phW6wFJFjQ5YU4\\\\\\\/RqBjMiRnvV5oyByx9vvHkfhs9rsPVlc9LVUbsnJIijnpUym+q8em3YojJoE9KnKymawqQ+mLjLJbZLUkDLEtoHUR2Iq9ev6+tAEJ2eTHoTgoOA8tAcYyXrwrWeKtGIV9X\\\\\\\/ewaOtx0vGsX9bvTbvocmcEijeoxZryzx7fYpEjfbaf20R9Ph6wLgBvI9KN7w1L13ljWAaSIp6vFBhLQZhorZpgjKX5yhAdXSsTHkSS1PYS0xpVoVGVcFyfivx7TxG019P0h5hCoZu+zEFtgaf0Nkzt1qn4rdp8pDKuawb491xFSHB9UFI4my2I0hyDyKdIrvvzughEjlQPIHEC0Gg47XPqXzbKkqrgnYevhqL55jO7jkutyVJ5vDdZLSHfl0Jk0iGSQkJ4c2J1S8OJY+waEVe44zC3jU50YwJtxKzyagiyUNvrEH6naBPIUQ1KIi0hPwM6zEwCJAJLlEA8IGTE2W+4KZdMGPapRnO1Y19Ht7h5tb67r1nWZTq4mP4ziMf4X2wJfzPrLoPRFTw4mXXaqo21GD+tW5cgo5YQISkxD19IQNe2fJKp3pco6xzOAy63SWzSWZ2voFAfKkw7fXeduzFmgmA3rCiugIqytureBg7LaqCsGdtq3flmBkwjUXvX8thgcu4oOV08uNt7FLz\\\\\\\/Zrt1mCdIq+fG75dF1GOs0ml3Uwytp9uiR4C+asl5Jh8TIWM5RVLXMFsBs+QviMOYWcLbRSdyu1ylj1sLQhLR04ktz9zO9XtsaGV5izkVIJ0ztW9hGJwtztM2abhhplt8z7y5cinlD1uHq3HSBEKxVZPKze1HoaJ2KeCgCfRPc9RN\\\\\\\/vqhq4fKKtgCogFrP3NB1Dk6hHW\\\\\\\/qcPL6Z8r\\\\\\\/QJ\\\\\\\/THc0TypSHZ+mLDt1g5RLnXI42Cdx7cLPgC\\\\\\\/DIm+i4CiU2sAEnB726dnW7YLb3QCv83ZTap67EvLIOS9j71t9iGFNKlFKgD7iCy+mn7YdTJ1L6h8Q7MKOmXjhFPwPthfsif7UvZnmfVRFNi1nHCQ\\\\\\\/ulpNBd6l0V0LgNlTVzz0rvfw4LbKajEDWeXkreD+2P+3QCi\\\\\\\/hh4lDOti1Y\\\\\\\/TqLD+oCtWD+B\\\\\\\/pNti1z3v6XKI7hdigVuk8jtrb2zsaK9z+9Ys1zeRjqG7xn4Ni+7Wn+snqBRHBMoJLPz8D6Hxe3T9pC97pWpRTw60ucSQdyNUvrKN8UPzt+7jluTgvfMxt+v1mzoMVHt\\\\\\\/5JmjyHD19Nho9FqpsEhgh7U7nqRYYwUBJLql04gBmynVxVZQfYjukhkF7cudmawBUBYBGbWXBwdbmsPNSH+3JoAa+RtS9zSDGCZ8o0KLfDENbd4oEU5hNEjCXobYcIzmpZtgww6srczMal2woOYlynw0Orq09uTT78POsc4l5EMtKM4W2lcozz2EEDGXd2RMHajz8zSlHL9T2AfE2xl6g93HdYiNB45xmO7mH2C0tie0RZOsbG0oh2mKAzGQhZCUtx4xZICk4Fb9gDKMbTPU\\\\\\\/Va1KcbzU82k4cX7M4H\\\\\\\/ecWhjcS7xnq9n2KQ6qLDpyir52CzgDIAKbdwNwAVXEXV4CCv11+\\\\\\\/AcDGVyUqfuCAhyXsnQcCCiU1A43OaOmFNCD9eoDi9ec4K3PlFHBrojaMALrM8IWWag9r0BQwqgN4gdOLzu+7Rnboochz93D8eWt2R5RKMxKtW61aeelh37+9P8dFH7DGRUWvdS5pWEne4FcT3aH29WjZQxK5A+3I1UDLrsAiZIxs9ZMVhPT0bOG7VAHnw3\\\\\\\/I8NLHa6eD7+CE8HYpnO0Uwf63YMifWHNLNKmQNPJcKKLX42U9aaH2mZodLXw0VrZSFqJIhtRtyqqhMxfDG098WiANcXeMXqeYk17kihYPm+q2GnTGweT9rM+uTZd3pEkQfIiun0T1Sclgoadmkz72W2PNBp3K1eV3WxGKGzvFFxsx3oauCr6uqKEnEtdxiZ4tTWfX2j4dCpWk8oAg1BDRxW6lclspKjJmX+tqCo7HX\\\\\\\/h2GY5a8c8KwpBU0VyBwMfSokUuEJAIXrhJm2aRxLux65vMBjbs85PFtNJ3qYjHLihh6BuXANcofj9S+ihqtRncnvo1SnDpUVgapdRrRk1SAlo2vMslNj0QO7an9lkzNcVxhl5PZOEXd7jSvMJ6Fpw8591mQbvU\\\\\\\/7X+oX23pdX+oYnVt3Vqe3S9TI5ipfEYsG3fOP2OdZlb2HB\\\\\\\/Y+aBX6l700Ev+9fM6BEWvSgCxC1wCenvDtPfPccGPQKteA\\\\\\\/8khwMdgpwWi+asmqzXARzJlHZqtM5rr24zbZbA40xGiAM8H0d+iIPyB7GhAcov+7IOGLFTiQDxLPmUwtzx7vBDQG9SWvHhggR4ErEDgqhK6M4Tf7aFOQhpPN1ytKKAr8th97M6JASOmzjUER89PEqTme3IeG7wOE2+k\\\\\\\/bkU3tzJNClMzuIu7GkuZ\\\\\\\/o5Y4Jmb0JQ4CwDfSR8mUouPW5Gi6UfNv\\\\\\\/9orAUI3ZcMeWgdEJz0W0F65MNGK1HnurnH3Xz7s9mj2TNQDbKsJdj3Ofg4AAO99nkefN3EWIG8nfZllyx9krcIf27XlmFxJrT7\\\\\\\/v7WA8CkRsn44sruJwa0ZB24pfheL5+zdrLNnNKEnPvhtrvC3M7QJJoP2iHb0j06\\\\\\\/\\\\\\\/iOxC7HguBKI\\\\\\\/f7moZtNx3zdmcZ55OHkwNC0m+jkdy8mYU5RvpGigxxYbeIEjKRelw7rcioGC8wDKn7uWiXb9w6AdDljh3M+avFofh+YMH59IQ6KKwRDYZQ7ppRYmtGixScuqSc3JG5j9pZoMTVAGU7KuzabbGDCXHmVFJkEt6I\\\\\\\/Mv6kQIpz54afnugOwNjIcD5Ic1b55p9TPDJatemtTCXiHLyMw6XUL+dJKbLO3gtGgCQINE5QB1gQKXWSj5sIOQVBaVSb4IcLvyUjm7v4efmlI9lZXmEdgvxp1OHwesE1wRhAr\\\\\\\/K7uNGkBf\\\\\\\/tcOg6iXde9W4cyK++MZUDeirDs7\\\\\\\/gusDbkEH02iypbHOfiE42zYdMbmbFq\\\\\\\/qP\\\\\\\/RPtFN2NzkDey4kuzyd+sp0GOBWvMRImDeTtdjqlYRRB9qaGDl8K05fDygG2YpMBFHywIgCuJ\\\\\\\/SFPzr803zVBrsmTi+d17WRueKqAQ0S95YFjQa2fVxSqVrwsWnoKrJO95arbej453LRxVHG2T7NKG8OPdjVoyGHU4JpHG7rYTWUMKT24NFiS47TG9fTyL8hJj9VQHNU38mRaEQ46lsaDKaPnx02nIMFVzD1y7p3IZFX9pBwf66pOx323vzarw3eG8H6FbWxvkc\\\\\\\/IfSMcc6WeOT7zCqklaYd4UE\\\\\\\/FYbfYFC5E1mNgVW8irjTTYVOApwHdIOwR16hTlP6VHcAVKoXzmkFoEobLEymOvL2QuBBiPbVUiRNKJSRG1jJ5MvmccJeESltFT4hecS4p5WXGq+Cw1nYoaukMKLofnT\\\\\\\/eVHrfNiWPa0xkUUcyxmgmK+wj99ZeGysU9t3INOQhQeRm3k4hd\\\\\\\/ojxazhnT15Jhbtm\\\\\\\/iY4s9jIV28Pj1WGLsQliqSuccAjy5afE+PdzWjzssOsoZHpC5EPTKqQJzaaYvkVY3e0AkoDbsL37+AJbKYLZ4qhMJtitk0vOe3RahWXS0yaPFyg7Ds0pPh4nGAOxUXOYyrx4GGyB9\\\\\\\/IvYIV6CN4NRSq5oJ\\\\\\\/pTc2DBd\\\\\\\/trOV838WJV0HRdfQNfnBO2sPli2RtEERy6JxpLas3p\\\\\\\/BPBwiIJ9MZgdgE2A93Ng1IAJ044h4oJjKcGiUO7whOqy5U71nf2vi0BNKpM46ufveN06rxNrtYTGnmTh90a+F6B0KxCu\\\\\\\/T+LmlCBWY\\\\\\\/x8gbnYblO514HRxlZBYD3yoYD3x+vvdJ0ChOH1p8kT5Wpva6GdcLK6PQhHeADmne1T\\\\\\\/uTwcETR7vuF6ptZzMWHebqWW0yRZSsw5uNSijyteqt9ilSkNpCr\\\\\\\/ZybnU4kzNSOSS1iVU8Ym4ITrYfYd8DpLz1sQz9ILdUA3aMhMvsS49VDSqSNp1hgzvKW+K9qVeinZUlGTFe2m7rbJhSJiLIm6+DNW0jjLvaNiuSpUlJO\\\\\\\/uU2RB0o10xvHENb4Gzst5RXFbEsadKHSpcNJWGi2+uKdGMl2mjEU26Rk2xpNcl5dexbzGLwW0fQqcOUBWnDe+wUbVZ9IMqKv1\\\\\\\/Z2M0hc\\\\\\\/lGmieix7YEgCgb7sbVNelSQLiUMTZbUEfTOWPZIPZPpWgJGdimgdntBeaFfS76rCjikRbaDERr5BRzBWdNqyb3dKYUurdkrFC6R1lKLncOAtqie6V6mUfPLOKzQzv2Eit3SwT1TUMOcJ7+g+91njdkJYpg2OfhlxY+sK1qrKkRJbtqPA8JWesVqAsPqKqZQFgmQE0Ah9DHbKcdSb5Rk6Ii9WRHmdxasxy52Qqe7RkOB5HAEfJEKOsfjAznRu4tDZI6k90hKi3mO5aPt\\\\\\\/5x0w35fLO+KyMOtuTt8WbJZvAuF6V0vW3t4DChR2gAE4vWCNGXIdrOxvxytnwYlRRaDMRMnIRTEGm5Vz5D4BWxSMDK9SQbNNfg0b+DwJPsVJtsl6\\\\\\\/O9NUwjA+dSulstsJBtm91u3aPfsnvd2NtJHG9Ph0SXqjBa\\\\\\\/ROxObGqiN2mPYI63gCWeOGUejOaL3kCnnqRQfyhTchwsQYs8tuj84gM9NtQ0HCH8\\\\\\\/ntEk2m6\\\\\\\/hscPC1y3DqrK3c5Hehj6rSg2rxsZYPV51DFgdnSeyW+17J763fpgaCNXbuUPOtlR2huUK7mrnG6toaaPNr1AGGYsm5v1wlNwc0FmzkcMM\\\\\\\/7mkPEyEwAipHqenynAfgzwN0beahcCKfos8EG9Hs80KixaDePemJAIDyg4r19nC6HVSvFlppwnRWjCNfX3Ej4zkZnyTfRxNOm8KYzNpPyxKfbyl8sGsC0+Bm\\\\\\\/U7DlZC+pWaoSh7Ap9YnuQ3gGEN2mffio3f\\\\\\\/JNMpO4TWX+a82QesKYM4b\\\\\\\/GayiVoH+zDF6xeU3vhfbWOflpxJ\\\\\\\/Ya36mqKShbCFIF+6Eq89WRNpNaSz8jOnKImZnl2cwh08RImMUsKqt8FJo7vhNhdftpdM5HiuPj\\\\\\\/x186+yGLyJXBTwlhUMRjxmwpkb7HpK+mcgQoSm\\\\\\\/nHTwR7mDqa++ibdl97UHAFsaOGNXh0L91qflNcTh45\\\\\\\/P337aWBhRbFXasPMO3OrrgrfVEUzJxuCu+xyxUqeJpvosVoHb91n6FPNIUVYffaYVvJNeYQMup5FLb+H2yDeODIRqR5RPZj4Sk2BI93yvKemhjMJkRhwVhCZzaQaZBRXwtNaub1gQ94EriqWl7noWq1o0OAAZjVqOtPuEdeoQMEsdRl95zNrqDz0wknUk9UAv9ZseYB8Jdemo9r\\\\\\\/HA\\\\\\\/IDDQaXhBLNyHlmjjyGV9ISeQR7cSEPcKdRqyF2AfJ2MVlSPTxbAm9FBmYP9aBL0FIgobTNZZZVFjE0BPUyBhGwBm5qKwd3sYnNCuZ3PDkKrAuOJE8OV5d830MWY84mCj5H2PAo0cunB56viPmEO0de1FCyNxsB98I1WTboL+D\\\\\\\/1ZuKBXwxJP37M6MoJ5jkNBJxfmsqrOQz53pNk\\\\\\\/6cRmTUdiSOs62FFr8O4hmYZHo6wpa\\\\\\\/62+V79OzhsY79tULrGChj29rEHq4GAZNCcioh3mQORsuzO1Cghd3DtmdYzTzwdEuaQFIVw4hVMpnz5vIdjhNHqQahtTKuuCHEcMHz4JyvavAPof0rEIbbVQ2gSjIWmcM39IPKUgD6hvqIL84bt4OfF+\\\\\\\/GD0Nf+bSGhcLeo+DZYnTCWRPdUFKAeDMv\\\\\\\/7y7\\\\\\\/6TWVzzWhtz2flq2uc\\\\\\\/HzydIXMVt7JQ7H91T+06dij6VtMJ5j71EpaUAmfw4qkwxd4kjwdmg2kmMdyVhks\\\\\\\/+483qrTsMnL1Vi5h7lblRujlmZ04QhOTt+MaKN8\\\\\\\/BS\\\\\\\/BBL1eztq5Sj5+AbWoq6DC\\\\\\\/jXpzckvgOcFWTGlXfg+\\\\\\\/5sPAP0tF2TjUCKNG07Wty\\\\\\\/\\\\\\\/69G37DQxs1fmCkYkXH2KT2mSKeSOXJHMrAaSkSaHJy0TFKm6dLsEO2U0RYd8tv5NHOn9KW6nY7msSP76G6ioSfktC+ymhN17Mymy9iZIGXPR8oV8neh3tMxGpFif66rXrHBXFPFTF8wTQhTCfhc9815ADyBxtnWnEXjqTR+eDFhjWHcLCZGuxQ7JI5FTSy7tiJ57yvdbPv6nlmY8Kq1iQ8FdPb0OVNDFP0dFBS2A2MHdgupCgRyoFTngN56J8MLVJt\\\\\\\/NHbsrKMwHOJAV7I7MULfFEw+lNXfdTlxECnGuDgkTpvtR0xY0u0w3UoTwscOUOhGYftK9iCthMqrKATIMlai50bXdhCr\\\\\\\/rMbod3Sw6oOr++w6cI6yTYd\\\\\\\/KjMP9x6IcrJMUNb2On25mGXKiQrfmG6PKe8i1ERTdZUes6YVvq9Y6QenZBL\\\\\\\/H2bGVuqxm52srpkV\\\\\\\/OWfWHA08XtPl40nMjeKWD3\\\\\\\/0K3IAOG+RY5PtvfB6qR9rRXkttBUY3IYDmlvgOUkWZiuPJjFKVxZhNHWm4vvCtViTPwOSZPqtRVfa5baWW1zBKrkY9+74ByZORseVfqUU6m1PwQbRrdCuisOLmD7CVHLqqE3QEvLejunfzeH\\\\\\\/ljIfh4NwTUD79mJ96CETyJZUMfpQdr1H3M2wE7ZjhnCD0sUXtx8wnuFDdhAI9LSjuiY53l9pevQjI11E6+yfED7Is6i76ogcVULOdp9QxrL6LWqb3kzUVz2YUJKjf7+dw0yGxBkYKD8J6814NREbLW92hqwrISNqZ6sn+i9PSk6OKNjOfnowE7hrTNpwdD6b5yFRg0wMdRa4iX1EnQZNTbNRwRHbi4DYdm+xRpEj+3ke+Ji62u13TN4BxP4mWSXM2bJTKIoDti60uU1IMa2rIwLrSdIkMtWCO6GHrLRowUvP5gyBiCDEytSH8EYUg1AJBguA0uO507I2RX51a5JLwFvzUz9aEi41P2oOEmApx0T5Hx395LFHnxctVB9fnJdcUpWCME3B\\\\\\\/5VyWg1BsezDHPZIca7ZCPVHLUxaXhfh18jV3BnbVSzUFnPAMfL+L3tWWp0gL2sBGsIKCiP5kPjlwWyZXxI+4W77ziCYoi15PuZuqJq2VSH1mKUVHmvrnIxRZrMaIo+yKiEfgj6hAxrAMvuHOY68QWHcRgv0+KUG5zllXBWPIpQx4HXECvhELbZvorZvYYGsMmmsVgOco4nXZkAlOudqNjOxDgQMUnrSFYY1qlmGWOsD4Mz+NobpLF9bQDdTmWu4clzHMpJ9jHDUleOVGqF7HjVSGnUQMgWO4\\\\\\\/AgWY5ISNWS1C\\\\\\\/7uKws42SFm7NdICuDZ3hsYVzr9NEpFK7fgh+kPiXBjmjg4e40eZ3lPEpD29qDpl7vp+LjAvjUdPYPb\\\\\\\/PGGugVaZj4\\\\\\\/AYIZZttIHKGsvL3nuORnA4H0LrwNHdtycAONF8guCdM4IZEpiK9\\\\\\\/yCCz0pTHxcwqJratTiIu0FgJJF6tf1KNSo5sWHgpWVBsh6cQ7w1\\\\\\\/Zf8SUSBcCbED5Y2krfuRJfe+dSj12V\\\\\\\/NQgK0e9\\\\\\\/e5oJSiL97m4tkPmKeqzxx8TDg3Rq31GD97n8ACl3VWnAg+LTLQz71xdYpATXiz5m9vlHIIen6jl9Ue\\\\\\\/1Glf4ovvh+K9P\\\\\\\/dbuSTon5WxGpVVEnO0ldsX0pQooBs0YI0ny0sKgaTzwVchY8e9C65q6+wgVwMGjkQk+FuJdZ0TumO\\\\\\\/\\\\\\\/2igji9q8tomHkti6lV77PVMnKqKpPyyNtBsC8AdeC4XNLQdBhRxur34CggWlxB\\\\\\\/xjhNyRNfuaB9Pe9k\\\\\\\/9C7HyobdX4D+U1uf47FYGhKGX42P8Mu19+ivKl82gJlnUVmwvAg+AMKVOrftjteE0m5vvanaYu4qTcjj2gpv4o9r7BhnT5Ea45xyF6C8U4PgKqasLErouVq6rq+o3mqR4XGBCPamAeo4YvJSYUau6j26+7mkRhytzW6wOBbAor7CvQ+Q+O6lmomN4Ts+HWyEbfUcuXqzd2Jpn6+zMdm43twVQGzNqhOdXcPYUBfqzu2+PADU2AqR2JhT\\\\\\\/3h\\\\\\\/83SwdYubhJ96obUXwAySZ290i9IJpaggyFZedM+6Sxs+gT0DsZJ6QHF2BEt0hKFuyv8JtJ5I8c3eiPPpVQFWnOVjEEIs2ZMijjcI4s1VeF1qwptIFiwRz80IO1IhTsMjSRzaGkOKexfBK\\\\\\\/CeRLGBIO0exH5W59NCF8hmTDxo1u8ZRL1lBtJeA03qmEG8xVhd8u8CeSHy1IOPPFbt9D8k7GmFdJCF01iTd1WVe7VlmER6rM92QXM8TOoaPtoDVsaVElsGYdXo6oI\\\\\\\/OaWsLv+EwSzBv4SI5YF22oXzq41KXlfduFV27ae8Q3mJBfuDa1p+in6RdjW5qsMcmZPS2u9Rns96m189UTGgG\\\\\\\/SsmKyHF0DAhdACOG\\\\\\\/UJyOy3Ju5nVDVuvg73tfHQmOf\\\\\\\/Gv4imii8TZ0eDsBefNGZjeVE2NSz8BBLnsaahYWWRTOUwlEI3RSpc1oCIVRY\\\\\\\/JLDdU9FlT\\\\\\\/Mo+iQZm1V4KAqqEnmAWyFQm8sDRlywVKZsU\\\\\\\/SKavIzqoluHKu71WPoaFqAWTQkzFTgL21tXb3GDFE4y73MNRPhvgU1IZCm3yFROkHdSZECuayjGvj\\\\\\\/F+lBbB9lanW4cRsQwg4fLAo9UYnDkcOHziM6uuhAsmKEKDsVxKzfLSVQaQrvwxWdqp675FlP25iUhVZvinOnUlNdg4x\\\\\\\/bhrHdT66chgS\\\\\\\/7QrJx4YhLuY05F4JNdx5kbgPVwNXxZuzrNQyijrMVtgSAKJ1wPWB9UCdb0VZSuz26ZWs+TYoITupdzWP5ET7ioEBLdXER99adg13sJaebRbxs9+wG0N1q5bXEt7\\\\\\\/BBVKSNx0DYvj37sWo4mdYoYvF2yffRWiLESs+ld9LLdq3NpTOwo1y29tf0UHNRn5dreo7ocAHdQv2EqmFR7aQibIIHflew\\\\\\\/nfmhKty7btCtqEPURkLK9Vk8A6fpBQp2QgG3lW6wuJDKSL\\\\\\\/JPL6BDzroNvGqtpKqzY0FAGdiD2yoDaBCzty6OJES\\\\\\\/GygE6RKTR6TUaTFmgO06EkiwWyB720XvYjVa7Y4BmO0gH8obZdcmiVzD8Dpp\\\\\\\/aNwWJ0wFqyA9uk4USinWGN\\\\\\\/\\\\\\\/4UGNrPIQoOiEgZtAynC0ljS3LlKYylVL5RP0SJhBIQ8YlUR6X1fOoYpVrUI\\\\\\\/8gkNpK0xUZwUODiT3frYRmr+ps4Y86jrkJCh8UVn0r3xrt7dIp2Ir+PI19xgYg7K7+AJ\\\\\\\/lORduUNGtkh+OGq8LiteZmS3pkqyvYKZNX0AkuoC2cQO1WdAFOQj7dVmQ5uaZIEAetJwfmb4oDYR3FSp4x+y7sEDAmMXy4ZFMneXphhMk5THzi625+hR1thp7qqsGxxYKd3UqdVH8cY5wqfdR1EUhB02VOycB+7ImacC1uzKIsLsiWuupkGwFfiosnOomwnMTAs81Nsj2SxDaNfUwJ1wqDRFM3FTSDeN06YOzQgvX+gd48mr\\\\\\\/WVJDH4DnpOKgTlPdbsQlEFOFO9ZubSutTDR5g\\\\\\\/kJl6HqTw+6nV34frUIMatZLwh7DhMMjxFVeJjFE4beQ1z76086DU0V+jdEuEO6OcT9fesJzWXAbTtcI4m5TfXmUP+ABVTw2hPBESYJKMn6Ld\\\\\\\/+gUH5WPjJ4jCWn4IKJmfhrW6JU0+fZJEVp+mfIG5VVQ+EJWM8SLxKwcyNxjHv8dVTb\\\\\\\/BfHY1RI\\\\\\\/ZLO5tBKNTCXVs0pkorCByYh+dVBu+ztDWF3nW1a3tTUrPjDNefkpZh1M400KKzRML1HctLuHssZiOfYYx81d4pcA1R2TuDdB4GUXhT\\\\\\\/KUrtdK1RLzHV7SoGDRCgiizveYnB\\\\\\\/90A5Za8yAbcqpfiMX2a+fuvTZ+gaVqYzfYfDLmxTv84SHBaN3hl8qkjD4LdleUSgc+h331XLNHqjrIXiIvfDfgwwdkv7Qm0jBnAoOni+I7OM6GZzHLVDk4apXzaT4snDGIAb4bjGOEGqTVwHXAFGwMgBOaXCQbwacCRgJlNIk\\\\\\\/fK9lejJPtqEudcN9ta+3cktuT7i5P7wTUDlkAEFlDxhK6VrGQqic0cvCLSmz\\\\\\\/wP9xkOA6E7aTRKn6AsjU9fMSV9WTbh155z3GswG9QvrRaogHmsyVLRk4FTA8taMWMYorPognWOr5hnZYDOkLHEvUelmeLw46RBsWEC6yEBdZdYtVEQI1+5fGFxFdVWWoW84uzHDVEPSLU\\\\\\\/w0s0dfEClbt4PfQMGNkUlCm+yGHWtQUzBtv1HLkyqdFkAG8NIPkLn4ZKZuLGnfy2qZ2E9rU9cFw+aBrL57ghPiJK1Omj1oxoX8bYEWp9Jw099hcpMmTe1mnYGe\\\\\\\/aWsVqxJ0CsSLnAotZ\\\\\\\/1FVy+hro77z0pjkAdemfaSiCgM9bWycxCIZiEJjTqPsZYPGI0AHsA2qAT+DkTfg2FQewZ6+nmasLBkpS69JelcWs6cT+Kt0seQ22ZCKpDkbfhVPEuRvUWlvxd\\\\\\\/4n\\\\\\\/eKCC6zX0pUxVujjA7OU5eZUaOYaL43Wv2VxSMtes5ill2BrjoNAXDFRuF0F5HOaYfjoztsK\\\\\\\/gHqiWFNmD5qG+NoCGBMkE0mPBwSchqQeU8DqgFo+ruIOG4bi4eFAVFDFPvN86vNR2GeN8N9VYoju\\\\\\\/+03h663UA5jmsGQ5Iuw1tvRPDaMSLqTpCRPW+SHOHJgJKhnO865hiCjCHHkco2zws2H8rHbwbJp3nOaDbNEnY9MIkzlwlDj1UOrFrVg\\\\\\\/g6Z+jaj3Zhm8n\\\\\\\/9q1lub7bRWYnUCAjvIqFDQkRYZI0Oqt+7zEC22xNXvPgr0sk10\\\\\\\/ZAFxb3h8e2eTPykwNfihfDunZEfpD9I\\\\\\\/asrjRF7kOcW6WHo178z5QxxhSIdzP9MsAeTk0gU9qxXf+flM2GatR5GceD7\\\\\\\/CHpvT+p7T7w5fHTwnzDXrTKIcC7U7R+89k+9RFNbAkvj71WdRoAomekx319I8IOpr\\\\\\\/i\\\\\\\/OwNRUgqXUqrZsz9hzaMy2rApl++luywtMWhfWX+JyBsmhBx9GJmUJBRYS0PWvBZmifphqlfHdYhzxEw0uPLAi9Ov4wryiKfR25Efh0rnc5Sb4OnF2gO8F79G1s1Y3xcznjcp3jUarNRjOUh+9sc0G+Mif0l2I+ir9nNxlz8ddMHmxmKn0BPCfG9JPA6yyzG9+wwE8LoBoQHAZjqn4CKtcplJu9AP2DTbJe9sJg86hObiw7D0uUxM4gj\\\\\\\/F6ciZEmoH5L7z1+bnHkM+\\\\\\\/ouqwJuDXAssTdScxFExmV9jWDUqQAa620v\\\\\\\/k+SeMANf8b+DbtyWqiw3ckQcniunEqMDtGj\\\\\\\/Wr+ryGNu1c7wWAd6Iwp1WFH1Us1V64diq+GEXNOrZe5FbOSu6vbvmMNKfLdzuRSS6xQDvMtjCxd0kry2Fuw3AkXS4xfEy2js3rjHenehBO4D+dngnWHckBnuPRtjwlduPdp\\\\\\\/XNALdI32F9xIq413hX09i89FwDIDJJ58+FO7MZ1t\\\\\\\/LB0\\\\\\\/m8R4dAiYJW4YoECIXKhJwPPLJH7pRzwujs1GhMyESSpIL09rddpHZuqzZsKBci3zKkidxIS87zG9Jsn+J4GIjfAdHxVPZ6Fs0DEsmPUAZV2k67AdV+y0eqpxb30d0q92vqDmS8PPIMz7WOL4wx+A\\\\\\\/sT8C7qfnGf8QTiDR+CS2qFDM7v2kXdjdicABUnHo3yvdY074VX+O3qzChg2mOtI43GylGBO+zF5x25yjuV2unNH\\\\\\\/Spnz45PlvCqxXNGLvz8sEBwXp07OlLGmL3ns\\\\\\\/9B9jGrVbigurvwrchkQ\\\\\\\/\\\\\\\/RHUFxUL5oN4q8iZIWrha35qWyNcAYhv8iiWg5yFVaT7++Yz7zi\\\\\\\/pxChZGHn1Cn3eDdU3O5oEbufpbO4ECyl65LunUfpIFAQhSMj1\\\\\\\/g+eESfgqaCOj590gBGA6MZFgHHWDp\\\\\\\/SevrvDT4lQ4gl7aY++WK1qjrX0WhuqP6rDEB7utdM4MnPYRXQRi4VYMzhQOcWAYMGEPgcAfjJ4Y6T+ef08M7U8HExhlt9BrxCg3Z51rVYQzA82Epsr7iDBxJ+541J1oLGXWzbrwyoYGsOp3GK+4VyDBkrbxdA+7JFQRj3P5CBOx2WvVW3HLuq4fgp34gZYMnUBkwSPRT0gymtDQaQ2B9yk1S6dtpUfpVnrqCRlO2g2r0aNzVrke5Z+e5VJh5DCUjb\\\\\\\/Rje1+IbL5wH7lzyL5r6rRfXaeUnc7E+7QOV\\\\\\\/OnsG+rm8L0uvZoI9hHRc5+iY28H0TuQZ8+hpK0F2j2tagEQdZ1e2Yd40pWRoe1mRfOY7sYaJUBQCY8tjXnUJmYOsCXKHie7+jzGqYUMbM1LWiD2eT2xZj3SUMjaYzOp0Bdai875sld\\\\\\\/KXVuhUZZra194sHTbVCGQehe6zODTS2goNxY1C6t2MGW9RQeMCM3phwdLMX23ofjf2BYVzdvpAD5BU8829pCt5czjbXFohXTbx+mnD4SYSjFeqhzbTg+rjzt+070fChfNVFmLQIZHTBSmVgYsFkXJ+YV3iSY16GjEpq6OheZ18pY+S3GVl0arfMOd\\\\\\\/vx5CTiPTwYYMjRSLmE5GBj\\\\\\\/I08IwxYq2xCbGYZIgsDbr1DNYYvbACk6iv2CeTrIjqKhhsr29bdX7G9Sxo9AfWRKRqnpk92zYQkFLEezQYPc3U4yN+UFwzQU0XjetaqgE2P7HnzjEgqmV6aeZXB3zwyH4v1PKZkAtRGbTqWSjy5MJEikVvp7Sn6y+mUUUnmROC+QggXpTEo1T3tNhDA\\\\\\\/IJSTcJYp+bjz0IqWDG2ty2QfeiG3rItIPrJWYnFcpFjYyOkL3zvXskDmUa2PqwmEcTTCyzzRRxCLlMmrq08FmRApIm8EvDyeGOJ+2JphRihaiiNB+WFqtgHZcuLpiA\\\\\\\/kyqT2x0ffqOFTG9iZjdYDO5Lc0585jKbhaxBXtaLqWMQKZbH23YxmMz96GxdN6WPjpIFpg1SFjOSYG\\\\\\\/M8TqqtPCgkbltUGMhL3oymJqOwobrdONbVufvsztFb78JBlnsQUQVEDN9zbARR7bXa3wsF2stCsLoqfDCAo74IUvlPazNS7c6YkPCJtS9n1eXfjLuvFs9dGP64NqeWDobU5zatn\\\\\\\/xD1oCeM8okzXbn4gx2XL\\\\\\\/4yj4fPJPW\\\\\\\/1T149UbdeaXA9S7Ff+ZTFvhIu7UrKOzRBJw7vfORtpv14cRnC7x0d2MXyW5GJQR5HZhdLRmFlQIum4JV3O+UTOM+XKEx8+mDHa6O0gaIVMbCVYC13NV75NgMNQoUgzNiRvYuzQL6OULnYWkV3YSY5hZCDSFkCNeO1x3bOAWdlUHQ7VAI8Qk88w8EbgmXF8HHrRidLxXzY8ttR78ING3hcNRgehaJQnnWmW2HIMMEiviZNm4DtGmJddFnRrf6hhIUKvWk8W8Xc4RODH\\\\\\\/\\\\\\\/l9Zaz8CK8n\\\\\\\/hGu5Pecf5cuxBpiWS1Q3UIKwEOpoy48DBHPguLYF5VJAEdX\\\\\\\/lCCXTagll8tF8wstnFNTDGBIfN4YNxYG4fUv2F3ts\\\\\\\/vYQg+YjHJ+EVe\\\\\\\/\\\\\\\/nAdGZBND6PaSdwpsZa3m0VTW+gnb0TC88e\\\\\\\/ovrv\\\\\\\/Q3mbnBHOxhInEwkKSFdG9lkESImu3tnzwjzK7q2GopCZ8rdn2Exzdo7KkIh3CWX8MwmQattqYBwhAwA86wHKh3oOOAWzj5jAe+0AuonKSgF0Q4A3eoU3sck\\\\\\\/uAiQMSRODTV5PiIoK3qWXaDvxao53pmJbElNtMuU\\\\\\\/7IGz\\\\\\\/wctI7CyPyR3bNMB6fZznKaQiM0Fxc1HnhZHMqgiJR4+HHUQpm\\\\\\\/irt1KzesNyHkOiEVHoE34fCdQuvzDRGi6C\\\\\\\/PNyst8j7022jmSF4O0L\\\\\\\/RFPJxxHoH6x3+YuNZWsa3deoVUXPz6IhGfaYJ1lO6bULDEdLp8KfzhLVp4gwM4pbRZeAB+DMVphAz+rjCyHinWJRW6JCqEtYLVGP4zifa4N7PpC+aOGLCQ+KMBFgUALAuhBVdPMoN3u5wEMiWHplgq9Xg9MEExffVg8khxdXbWq6IAq2YAtpdDy2KzTZ2Ad7pbM6qcu0ihvTR0in9eV2a3zgl9r6T8d990Fuiv3abq9wJNyTUMf7GVQS8\\\\\\\/wrfwzKDFKQNOq53BA6IhHj7vIdNnJ9fzunTFxKv2gLf4r6hccq5tl9k9UzYit\\\\\\\/DZybzeVc85wO9Q2TXxzVIDEUC+BKO\\\\\\\/DcQkBsxiFwLd+agU5bFRbNoHP6T+aYUZXXP6wmuHjRCAB\\\\\\\/E1dg5WI51MSAOFbLEJeHTzAP311OiMl8oXb+oNZ4S6MvvSQa2xM6sEIpxswRLKVeUf9BvbFr9G6o8wyYYN4lR1krnND9jNsyF2z3x4aKa3IV98Gw4F46neH+eam\\\\\\\/S9tUT\\\\\\\/Kthvn93iT5YTa\\\\\\\/QVTOiUHYuqJIlNucrUkWcnEQp3EFnArOeGfQnPnpYEpowxs0OysK44HTNom5c\\\\\\\/zGPcVRgOzP3NgHsi9OE9y9+S7iF\\\\\\\/eaJla9p428kqEVfkT3Blzjqc8yTZf0wHjbyuUlSzoTfNsDhBww5t5hN2ReRS8QFvHjGj8CbKTWcWreeEM1mYs71gA32x6VhJtadPv+dafHMuU1Qx0jI5NbRDFZS+9vzLLHlmOwcOwvsv7grD1uwmwUskNSAhwPW+sZKOMAVvnAHMuKwnwUJM93shVczri3nR5tkbieFPure7vAp1OJZjZii3z06c5DvXpEIvxkUZGJwn6uNjG5Ksbk+u+jf63X3RSUtDK3IP\\\\\\\/QPZXWE1Hf\\\\\\\/VdOiMz6CSP79MrV+V9w+3ldxUJqktQzAaDKJKp95TVAxPVRmNpOGRCk9RZqjQ8xzoZiiKGGA58KFWqN12VGvg67sFnAnDAC4B3GixSSeURtFPta6lm6C0kQCoFle6EgI\\\\\\\/eX3+ECTXj2OM8PgcIe6MYVm2Om111Exg060\\\\\\\/o72jnLOWULfbbXXXwBlPVly9GfyVDny0I02Ds2+YYA+gK2R3fEQkYxuEasXiBl0t5IGqJx5x7Br2U+Tle1lHCjhMUo\\\\\\\/R9XlWJMOPkLVHyuLdHkCkLKpjoMeuOnIQMFwwAqw3SpA25KSLYbFlo2w7MhCK549T8K\\\\\\\/ulZ7\\\\\\\/ehThCItbA3P+lP5pCYxBNN1jpUNvA7Kfks8IlBmW1I84Wi1v45ApfOJC5Wm7MTafpOe7DaMilN7AGvd6NTXh77fb1jjPz5IlhDGezJActn8R\\\\\\\/u3SmA9VYj2nYVN15nZczEXZ4cV9v2+kPMC0JEEzpasgud3L\\\\\\\/a8fA8Za9w9EjtMWC1h08vFnK+Hze41Oe\\\\\\\/6hQ7oTGHZSIoQPMRk3qW5HrUk3wFOIqSw2HioG4nGWwaV8yRodNqMrHAkI+gJNXAvNFwiwKCaFwjPR277HILBG1OBUsLx2LW8nskJGhU98hCmZ6CiGoLfvKD3qXiEOzM8aG3rzYRXOv0xVnnxrscAwsS9mZqgXQobARDXMUtVI8LQTMbhmtQ7hyEiRYT8lhDdogd1CUPRqv97alsJC4StOkAqGXDx7r+GJMNQtFDKO4YZ4cu4kxeD8crMoCZskg7iLW0RCBKVM41ZJljU6TUKj7Qy98Ku4J9x+W+qiI+XE0+cHs0LX31BZt9EQHRDeK8FSzlH8mEzgA87EW2SydocUVozkbFY8YQypRiRqgX0i6g0mQ4+dfQxFa8XHKT49gfdFiUdg9\\\\\\\/eZL+CpoRHbB0KiBXrIubYE5HTGAmtgGzcI6AFMKFwRE6JR0qC2da0D1MsUQdKd3Nn5oCjApWnrmC8PIS4lkchjzl7n15NnCAMMqv47RvNzlGo81Rj7nFKraezkqa1IUMD+4X++z1m6cnpTmd6nWeY2yFdni0H6UjfRo791\\\\\\\/fxSbw5R5wjCTub0KvorU22UegEIfHpeMXwCZ4yAVXrPKoP4JZPYkFSab68GRMKFiIJqOtj6o9uvA8yFz7pddb\\\\\\\/oXIfEczWZ5hXpCt9MvVpkn+ezCEQ\\\\\\\/q48bfQCY21yQQets6nhfaAVwm6lsQzWBWM00lHpBTCcUNbThbmtosIgLoD7IYXegy\\\\\\\/dE1XZYusWuNsXLx5NxX1kEWb039ri1NXACZqcwa1lU+7ogP48CYP47QaxoC8kjGeKCW+kc+eji7wtO7g5CXLUIXabTJxvOdLg3EadOHNTwVGccx0a8\\\\\\\/hNb6CW9B8RaKpZ+xU9jRhzn3wjWHT8\\\\\\\/FCaWGLqY\\\\\\\/uyS9Au2\\\\\\\/ceiEIXZ3yEIIojwf1tkL8UIidnv+FUdpM\\\\\\\/JGyOsWeYI3okwvfIyjPgY9BDLyVeUoO6uw66Ev5UOCrQq5OnLQHx7Nh2XmrydmgYc5hYDZXSrPXFETkwFwnN2GJpuWss9OwjvoJYape4biFToYkofQSRwGftscdWi4krI2tGSEex3Oqh4LiJgm9g\\\\\\\/cy8eFIwgOPolfTJvtRTTRGTXlZh\\\\\\\/\\\\\\\/sVt4Do7QFMONSSggl08u3HTQuseH+zFPEau90MQbWh6jKKZ0HzCTjb5+zFkxdRmfrXmp7HkfFR3l+B9mPu6\\\\\\\/QsLRB6yC\\\\\\\/OsFZor6QXgC4hEeOWzxgXEsF6fBoEHVSE75FEuPUderUf7FOxv2NWgw5n6ha67BGfZ33mSd9SRScrIkoR2FrjRaNbyd\\\\\\\/0fxNcX0+zvPQVrHjuuOmFlONXXeSwINheNOPT5ET9q0AFJJp\\\\\\\/KZrnogmYH42WF+ySMqWiInw0T6qEhRXN0PnxYp+8UIT6Bp5exvY9hiLxTti1d2rKxLWUEVte8iIQumKcUhcK638eTaLykJsNIclzWEY0g59dY6t0DoJRred\\\\\\\/5I9QRndsWdNh9vAAGaMhP9DAQLiqaZ0F9tiwqozrNc2AXw9VJk63sn7s82iGdtW3jKaww+XcDUJcQ3A0YgMliv1y+rTjt6nRO4S9TVMI9zB5FJCvBXjSWDrgOtv1T3QNHZXr5CFaTIcw6RaQOs3Hi12JIeTC0RAFCjghfijfKeeg6CBd\\\\\\\/0ns2CO4fYg\\\\\\\/WrlLyPxAvLlgfk0HM4T2Go87mmT8cOABXfuek6d5+G+12as43YBjtpb8ZwxWB0kQa56m9O\\\\\\\/T5XhWtk09iZJ07FwMM\\\\\\\/ntNm6XRl2FEiZCIJX4ptgSXweJS8DyLovzE4FKUF3ZiIfuSdBHFFcWjCk1xdCTF7LMdRoGmlMg2haE996qG8QTn6cvNByGZND4HY0ol+otRyRxv4XJOEIGa36OX6jHSNZnjTPtF7Sn1ippzMSeeNuYJiyAPJvBFG53+\\\\\\\/7b3gKuPduxwpQsR7qaVzjaw83hTdeAv4yclIIEgJsQmaQy9WliJXRpWz0GznBT0wXUBKRXNBrTTa5lo8VXhnGch5oo3k19SDFZlgFUdXWS6GjgUD6Of3GcHZVDzqppWJb2hKI9EzCDv001bMzAR9lfj34x\\\\\\\/4+HDiji7f19Ygyn9KfcIdRvQzmFA6YnUnOaMV0YEm1lgV+it+9E+ehmwMPWhYIPJ1YVl0+DFJ0a5s38BzGwTB7Ihqu1OdbFPOmL0xWlduLqi4AyUl5sPVYt9DYTyaNsjgshjdXNt1BmlIDmOZ3L8cmHTSZ84W0ZlRG2VNch3ISUEvGjIuFgu1p1SXDMCFNSQChwR3NFRgQ6nB3cA4\\\\\\\/Z1HLtPat8qGYMblGBymRGyFeOJf3QdfAqvDjrn+qJ\\\\\\\/DIzPaHxijCx3RvEiyQuTxu+\\\\\\\/gCbBtclU2vlq70yHwfoP7TEXHdSIIVmdp16gQJcsoLL4cNBbS3xG5YDmOmcH2cCVG0tMu69N66g7nPdyYiQMqfL42XezEDif1w6Pys7\\\\\\\/ziUpp9\\\\\\\/z0Y4pi25SaVM+HRHadcGzwWb6Zt5S63j8zrAh\\\\\\\/FIUIcPb+nwphctMQdLdlvPme\\\\\\\/gY4Uhd\\\\\\\/2+KvkyjdxzfjRs4ef6cYP1pGhMANu0Z9SxQnIx3v0RkTTygRwhLfXnnfolhW\\\\\\\/nepPVZNdsctcpxmVapcZQXrD5DPQBWHAQ1keHz5\\\\\\\/L7mIEl3g7mu3j4jRrKBiwO7jBoRWUJ\\\\\\\/6bdHwjd+639QTBWHMY7Kl0DaXBDumaqpmkHXLceJxHXe7hQJECJlAN8ekhOTr0Uc73\\\\\\\/618Skl99jAF+7IqFHy0rj0kb\\\\\\\/KgzvZcUZBu03keh3F4\\\\\\\/4UbMeTYsQW1aSoUyJe4RCcHdxdobk6mhvk6JIuIEheMezeO3pvMM9cP3gZbyDLHdhPVGHTc5AUKcL\\\\\\\/8RnzJyX7rb1ZeD8ltDIZxuuPvGy5JREmouxgJ2xa53vGAlv9VFXQRwJ1\\\\\\\/Yp4fF1PbH1Lcczd6HLLwsAXmLCxoNS4LZw46doboU\\\\\\\/kVscf\\\\\\\/jHX+RtnpA7B7iWkWUU\\\\\\\/ivTtKPtUO6umALrFb+5tEf1gVZvRwB7xzW8K49WPNv37QX6xCfpNmhm3V7AUfj6T0KlXI4lGvpp7SjE9d9Wzwl0nIOouCS9F3II76KDVpieP1mLsF26zXu1LvhSnTCxq5YbiIGuRlKeFf0b0QKKSdOvRdC0T8xzULqWi58hadZIaEJKXXWXd2N\\\\\\\/NM2jzAHgA6OnD3B1bXTqbEM\\\\\\\/28RfNDQ7ilHd5Md\\\\\\\/Eq0sJOKLaoNznpGARdBaWPW\\\\\\\/TldrjbBk9tWFlw1BtUHAEQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ep9SyFNwe+tzsW35E9kRiw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e113222d68358ee9bece3eedf0acbe4eefdd3406ec7bbef2f45ee7ccc6e446e7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WffWxhi\\\\\\\/cPNVaVZy0l+MR88wcfCM9F1r3349O5j7mRojGovZHvsIvSWF6YM2oRhQPpqF\\\\\\\/AByYsnV14s7iypPDAZBI9CavPBtyRYLiFqFasYXDY9ssjAuV7JQjID9QlxeGaLP0JkThNQjLubP9URjxeQ\\\\\\\/TGlkgS84QOIlQtDwWn6PgnoRyPA\\\\\\\/l6SF9gj85bY2swK1+Ew16CPpapM3PjMGDuELParMHbmtMlGVZ+tIfyIo0RrW5LK\\\\\\\/okXfKIoX8r91B3vWTk7tgQJJqnvzhDF2S\\\\\\\/A+9wlki9OROzQNAau2IsnU+R6\\\\\\\/xLZrCeLChsWEyzcUZ0A6gK32sYVV812wlJUU4clk\\\\\\\/5F+csZRMzXASH\\\\\\\/E\\\\\\\/g9cvJQQG5qI06dkmy4RUhwK5RGcFEI\\\\\\\/mr3HMIQnuUYbuxZtAn3dxm0ImA4NnGZMyGzPfxtZOnDORjKNTxABqclJtFVr5uu+9Q1eY2XVaiJZ4k+3tqGigBa9G7n2dZUt72QdeznA08b6ecbSRGJo51wVTbBlnsYE4sNOZSCSDG\\\\\\\/YhWCec7k8lelvEPSp5csmznqZ4t\\\\\\\/KyP0GpO0vuTtps0m4PQkfZNT2Harp8n0OWfLl6UeG3P3BiFvTCI+y9KrxZ+Vsopua57luj1z7PfzoH9ufPO4rtIOiBhldBBcBjCNbn1B2MV4LqfaIF1RzJpYHM1rvFBUDGQbDMuJoYfsIG72K\\\\\\\/wKxZ5OWdRE\\\\\\\/4GYy51d1w16vmCLqsnErTNicE0esh3HOHfJZi9YiTLfxN4caJqWm7msCo63uFYI9fxCRtiKfqgEdSbWtDiyewxP9WNsvvEamJLGc1USeiIL0R6Fc431k7QUd5PDndyG9pXCkziDdS7tQmUgfh84pEQXLFGJ2bVFgi0yRyBJD5L3ezk5re5uf7X0UBFD2R6b8RtJimatNSCQEOHv0JQbJNGpAIYAVsj\\\\\\\/ROxnX\\\\\\\/z38tHYezKju24wNbvKoKcaACSBgSeGdKI9nC\\\\\\\/kmm7tNSvzoo1wod38to4k7QLlYUcws98DzsvNP4a6GZXKZ33Kb\\\\\\\/GXF14vFd3KAe5M8v1MHm5\\\\\\\/h970tf+Tg\\\\\\\/PaISEfGys\\\\\\\/RvwDhawtt7\\\\\\\/CsVcFxXT\\\\\\\/kluD8QFonOxaeEGdEGd9Z1zHRzaxKJMxwzCY7SEz4BYJlMxECHzVfhxM6OQIcmlJUgq6UnZ4ZG6lQ7rnhxRm3ApV8lvVMUitaanbfGzo4TBA3DJxah2hi\\\\\\\/1QFe\\\\\\\/YWob3CZsC4o5tXORJ\\\\\\\/T05B223eO6KAgfr04RrsyXXIo8kSlWFfcSdjNSe5YhnCuo4nBcErt0t0Lddik3v43GvPaHGnFRoWHbk4baBCZu\\\\\\\/r8pfWL8w\\\\\\\/SGa6127mFNBb39iFeYgoDc2tBeAbR01IeHVkO\\\\\\\/5yNz\\\\\\\/PBpJjnSrxDb6w4vA5WVLDH0mETtEGTed9sRmE94c7dK5kVmFP4Vud7NfwNw\\\\\\\/FxvpAPXVKzSqtiImL9KGqbFe56CzZzwj6RboEnXBBU09jDnQ+U5t63DMBW22LCxdPim4sUcrj1uEz5TIACcSKxnOtiIceLNCAgHpfD4\\\\\\\/usJCSmyug3A9xrxvIbmnNQ2TSKBMKs3g8dP60f83qUIAZRbRa5C7\\\\\\\/Pjccybfg7lfVAIeTYNTfS7IEtQ0GD1s7m0u0WcTD2By3CtEXFwnkrk4DwvgCu3jyWMsQ4LqzHskxy7Cr0wNjQLui8+xSSRdI+qwurEw4t0IMxv2Sy8KhcJc0JvblgMlfiZ7fRbt+2c4gxH4sNwMmFPNRh0hsYoiEj6BoKx5Rbm5gAobr8GgggCu0tDQLxQwsKaUGD1oE\\\\\\\/yFeXS1i3HMrjgg3\\\\\\\/zbNKPG\\\\\\\/w1XZ78AuKoBEISS3qfqJEuoyS0w6PWhQxuB9ReEi5lu4rV4KR\\\\\\\/nHHp5QkXq8GfsV73i0AcPjiYJ062mxcLrIDl8hX8NG5txmFJ7LxYPKSB96kEPGmUy\\\\\\\/19+EfqB3jjFF+qakIenmkF3FLaemsoMhg2XSGDvPmtVIhuNv+sK9m8HtNdrKEugosOguJ7dQPw==\\\u0022,\\\u0022iv\\\u0022:\\\u00228z+UZoWDoU0\\\\\\\/33IQGZMqPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d7bd7f89e2b761ad8b66dae66bfd7c7ba1dd964e372d85c1bf623e2eb696acba\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460225 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2711" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GgkXVZ84MvJiTiVNz5NNQ3fKyaXz6VWwYk8bMpD8YIk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2711" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GDRDi2avbh-X5vF6Pi8f-BDVfFtCCCpDCYwwcmp6eOo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2693" title="Heterosis Increases Fertility, Fecundity, and Survival of Laboratory-Produced F1 Hybrid Males of the Malaria Mosquito Anopheles coluzzii" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2719" title="Caenorhabditis elegans glp-4 Encodes a Valyl Aminoacyl tRNA Synthetase" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460225" id="node460225" data-pisa="ggg;5/12/2711" data-pisa-master="ggg;g3.115.021105" data-apath="/ggg/5/12/2711.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Satish Kumar</span>, <span class="highwire-citation-author" data-delta="1">Claire Molloy</span>, <span class="highwire-citation-author" data-delta="2">Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="3">Hans Daetwyler</span>, <span class="highwire-citation-author" data-delta="4">David Chagné</span> and <span class="highwire-citation-author" data-delta="5">Richard Volz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2711-2718;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021105 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460225" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Satish Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satish%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASatish%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Satish.Kumar@plantandfood.co.nz" class="" data-icon-position="" data-hide-link-title="0">Satish.Kumar@plantandfood.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Molloy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BMolloy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Molloy%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BMolloy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Agronomy Department, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hans Daetwyler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>AgriBio Center, La Trobe University, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2BDaetwyler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daetwyler%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2BDaetwyler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Chagné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant & Food Research, Palmerston North Research Center, Palmerston North, 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BChagn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chagn%C3%A9%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BChagn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard Volz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BVolz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Volz%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BVolz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/12/2711" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460225" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2711.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460225" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2711.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460225" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2711.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460225" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/12/2711.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460225" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The nonadditive genetic effects may have an important contribution to total genetic variation of phenotypes, so estimates of both the additive and nonadditive effects are desirable for breeding and selection purposes. Our main objectives were to: estimate additive, dominance and epistatic variances of apple (<em>Malus × domestica</em> Borkh.) phenotypes using relationship matrices constructed from genome-wide dense single nucleotide polymorphism (SNP) markers; and compare the accuracy of genomic predictions using genomic best linear unbiased prediction models with or without including nonadditive genetic effects. A set of 247 clonally replicated individuals was assessed for six fruit quality traits at two sites, and also genotyped using an Illumina 8K SNP array. Across several fruit quality traits, the additive, dominance, and epistatic effects contributed about 30%, 16%, and 19%, respectively, to the total phenotypic variance. Models ignoring nonadditive components yielded upwardly biased estimates of additive variance (heritability) for all traits in this study. The accuracy of genomic predicted genetic values (GEGV) varied from about 0.15 to 0.35 for various traits, and these were almost identical for models with or without including nonadditive effects. However, models including nonadditive genetic effects further reduced the bias of GEGV. Between-site genotypic correlations were high (&gt;0.85) for all traits, and genotype-site interaction accounted for &lt;10% of the phenotypic variability. The accuracy of prediction, when the validation set was present only at one site, was generally similar for both sites, and varied from about 0.50 to 0.85. The prediction accuracies were strongly influenced by trait heritability, and genetic relatedness between the training and validation families.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/apple-breeding" class="hw-term hw-article-keyword hw-article-keyword-apple-breeding" rel="nofollow">apple breeding</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/malus-%C3%97-domestica" class="hw-term hw-article-keyword hw-article-keyword-malus-&#xD7;-domestica" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Malus × domestica</em></a></li></ul><p id="p-3">The main goal of most fruit breeding programs is to identify the best performing individuals (or genotypes) as potential commercial cultivars, and the secondary goal is to identify genotypes that can be used as parents in future crosses. The genetic value (GV), which includes both additive and nonadditive genetic effects, predicts a genotype’s performance for a trait when asexually propagated plants are used for production (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">White <em>et al.</em> 2007</a>). Nonadditive genetic variation results from interactions between alleles, whereby interactions between alleles at the same locus are called dominance, and interactions between alleles at different loci are called epistasis. When nonadditive genetic effects are an important source of variation, the use of additive genetic value or breeding value (BV) to determine potential commercial cultivars may lead to the selection of genotypes that do not have the highest GV.</p><p id="p-4">In best linear unbiased prediction (BLUP) of BVs, information from performance of relatives (<em>e.g.</em>, half-sibs, full-sibs) is incorporated through the use of the additive relationship matrix derived from pedigree records (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Henderson 1975</a>). This matrix represents the expected parental contributions (<em>i.e.</em>, genome sharing) between individuals in the pedigree. Nonadditive performance of relatives could also be included in BLUP analysis through nonadditive relationship matrices derived from the pedigree. However, partitioning of total genetic components into additive and nonadditive parts, using standard pedigree-based models, requires specific family structures involving mating designs (<em>e.g.</em>, factorial, diallel). Furthermore, clonally replicated progeny trials enable the estimation of individual-level BV and GV, as well as the magnitude of genotype-by-environment interaction (G × E) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burdon and Shelbourne 1974</a>). BLUP analysis of such trials can provide ranking for the selection of potential parents using additive effects (BV), and potential commercial cultivars from combining predicted additive and nonadditive effects (<em>i.e.</em>, GV).</p><p id="p-5">DNA marker-based relationship coefficients are more accurate than pedigree-based relationships because they account for deviations from expected parental contributions due to Mendelian sampling or segregation distortion. Furthermore, genomic relationship matrices (GRM) are particularly advantageous when pedigree relationships are not available for some individuals and/or when the recorded pedigree information may contain errors (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Muñoz <em>et al.</em> 2014a</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Speed and Balding 2015</a>). Studies on animal and plant species have demonstrated that the accuracy of artificial selection can be increased by using GRM rather than pedigree-based relationship matrices (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lee <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2013</a>). Numerous approaches have been developed to derive GRM based on genome-wide single nucleotide polymorphic (SNP) data (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Van Raden 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Legarra <em>et al.</em> 2009</a>). Similarly to the additive GRM, approaches to derive SNP-based estimates of nonadditive relationships (<em>e.g.</em>, dominance, epistasic) have also been developed (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sun <em>et al.</em> 2014</a>).</p><p id="p-6">An approach termed genomic BLUP (GBLUP), which directly estimates genomic breeding values (GEBV) using additive GRM, is now commonly used in genomic selection (GS) studies of animal and plant species (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lin <em>et al.</em> 2014</a>). GBLUP has been shown to be equivalent to ridge regression BLUP (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goddard 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stranden and Garrick 2009</a>), and one of the advantages of GBLUP is that the model can use the well-known mixed models framework similar to the traditional pedigree-based BLUP models. Also, GBLUP can be extended to include additive and nonadditive genetic effects to potentially increase the accuracy of estimates when nonadditive genetic effects make a substantial contribution to the genetic variation (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Muñoz <em>et al.</em> 2014b</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sun <em>et al.</em> 2014</a>).</p><p id="p-7">Although numerous studies on estimation of genetic parameters of apple fruit phenotypes have been conducted (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Durel <em>et al.</em> 1998</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alspach and Oraguzie 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kouassi <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> 2011</a>), there has not yet been a systematic dissection of genetic variance of apple phenotypes into additive and nonadditive variances. The objectives of this study were threefold. The first was to estimate additive, dominance, and epistatic variances using SNP-based relationship matrices. The second was to compare the accuracy of genomic predictions using GBLUP models with or without including nonadditive genetic effects. The third was to evaluate genotype-by-environment interaction (G × E) and its impact on prediction accuracy.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genetic material</h3><p id="p-8">A field trial consisting of 17 full-sib apple (<em>Malus × domestica</em> Borkh.) families, involving 25 advanced selections as parents, was used as the source of budwood for this study. Based on pedigree records, the average coefficient of relationship between the 25 parents was 0.15 (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS1.pdf">File S1</a>), resulting in varying degrees of relationships between the 17 full-sib families. Budwood from each of the 255 randomly selected genotypes (15 from each of the 17 families) was collected in the winter (June and July) of 2010. This budwood was held in cold storage before chip-budding eight lateral buds from each of the 255 genotypes onto nursery-grown 1-year-old ‘M9’ rootstocks in spring (September 2010), making eight clonal replicates of each of the 255 genotypes. In the following winter (July 2011), all successfully propagated seedlings were planted (3.0 × 0.5 m spacing) at two orchard sites, one in each of the key apple production regions in New Zealand, namely Hawkes Bay (HB, lat. 39°40´S, long. 176°53´E) and Motueka (MOT, lat. 41°6´S, long. 172°58´E). Generally, HB is warmer during winter, spring, and autumn than MOT. Rainfall and humidity are higher in MOT than in HB, particularly during spring. Comparing the soil types of the two sites, MOT has a more fertile soil than the HB orchard, thus trees are more vigorous at the former site. The initial goal was to plant four replicates of each genotype at each of the two sites; however, because of problems in the propagation survival, the number of plants available for each genotype varied. Ninety percent of the genotypes were represented by at least three clonal replicates at each site. There were only five individuals with only one replicate at each site. All trees received standard commercial management for nutrition, pesticide, fruit hand-thinning, and irrigation.</p></div><div id="sec-3" class="subsection"><h3>Phenotypes</h3><p id="p-9">Fruit harvesting, which was carried out in the fruiting season (February–May) during 2013 and 2014, began when fruit from each tree were judged to be mature, based on a change in skin background color from green to yellow, and when the starch pattern index was between 2 and 3 using a scale from 0 (full starch) to 7 (no starch). Phenotypes of each individual were collected in both years. One sample of six fruit from each clonal replicate of each genotype were stored for 70 days at 0.5°, followed by a further 7 days at 20° before evaluation. Fruit firmness (FF) was determined on opposite sides of each fruit after peel removal using a Fruit Texture Analyzer (GÜSS) fitted with an 11-mm diameter probe tip. A list of visual, sensory, and instrumental traits assessed following <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> (2011)</a> is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467109/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467109/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Apple (<em>Malus × domestica</em> Borkh.) fruit mean trait values at Hawke’s Bay and Motueka sites, New Zealand</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>SNP genotyping</h3><p id="p-12">A total of 247 (out of 255) individuals were genotyped using the IRSC apple 8K SNP array v1 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chagné <em>et al.</em> 2012</a>), based on the Infinium II technique. Genomic DNA (gDNA) was extracted from leaves using the NucleoSpin Plant II kit (Macherey-Nagel, Düren, Germany), and quantified using the Quant-iT PicoGreen Assay (Invitrogen); 200 ng of gDNA were used as a template for the reaction, following the manufacturer’s instructions. SNP genotypes were scored using the Genotyping Module (version 1.8.4) of the Illumina GenomeStudio software (Illumina Inc.). The reliability of each genotype call was measured using the GenCall score set at a minimum of 0.15, which is a lower bound for calling genotypes relative to its associated cluster. SNPs were subsequently discarded using the following sequence of criteria: GenCall score at the 50% rank (50% GC) &lt; 0.40; cluster separation (ClusterSep) &lt; 0.25; more than 5% missing calls. SNPs with minor allele frequency &lt; 0.05 were discarded, and then segregation discrepancy was checked within families. A total of 2828 SNPs were retained after various quality checks. BEAGLE 3.1 software (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Browning and Browning 2007</a>), with default settings, was then used for imputing missing SNP genotypes.</p></div><div id="sec-5" class="subsection"><h3>Genomic BLUP model</h3><p id="p-13">A two-step approach was used to estimate genetic parameters and make genomic predictions. The first step consisted of calculating best linear unbiased estimates (BLUEs) to account for fixed effects such as replicates and year, so that each individual had a single phenotypic value for each trait at each site. These estimates were then used as ‘phenotypes’ for estimation of variance components and BLUP of additive, dominance, and epistatic effects using the following linear mixed model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <strong><em>y</em></strong> is a vector (<em>ns</em> × 1) of adjusted phenotypes (BLUEs) on a trait; <em>n</em> is the number of individuals (247) and <em>s</em> is the number of sites (2); <strong><em>b</em></strong> is a vector of fixed effect (<em>i.e.</em>, the intercept, site); <strong><em>a</em></strong> ∼ <em>N</em>(0, <strong><em>G</em></strong><em><sub>a</sub><span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-2.gif"/></noscript></span></span></em>), <strong><em>d</em></strong> ∼ <em>N</em>(0, <strong><em>G</em></strong><em><sub>d</sub></em><span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-3.gif"/></noscript></span></span>), and <strong><em>i</em></strong> ∼ <em>N</em>(0, <strong><em>G</em></strong><em><sub>aa</sub><span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-4.gif"/></noscript></span></span></em>) are the vectors (<em>n</em> × 1) of random additive, dominance, and additive-by-additive epistatic effects respectively; <strong><em>G</em></strong><em><sub>a</sub></em>, <strong><em>G</em></strong><em><sub>d</sub></em>, and <strong><em>G</em></strong><em><sub>aa</sub></em> are the additive, dominance, and epistatic relationship matrices (<em>n</em> × <em>n</em>) respectively; <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-5.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-6.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-7.gif"/></noscript></span></span> represent additive, dominance, and epistatic variance respectively; <strong><em>e</em></strong> ∼ <em>N</em>(0, <strong><em>I<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-8.gif"/></noscript></span></span></em></strong>) is a vector of random residual terms and <strong><em>I</em></strong> is an identity matrix (<em>ns</em> × <em>ns</em>); <strong><em>t</em></strong><sub>1</sub> ∼ <em>N</em>(0, <strong><em>I</em></strong><sub>s</sub> ⊗ <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-9.gif"/></noscript></span></span>), <strong><em>t</em></strong><sub>2</sub> ∼ <em>N</em>(0, <strong><em>I</em></strong><sub>s</sub> ⊗ <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-10.gif"/></noscript></span></span>), and <strong><em>t</em></strong><sub>3</sub> ∼ <em>N</em>(0, <strong><em>I</em></strong><em><sub>s</sub></em> ⊗ <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-11.gif"/></noscript></span></span>) are the vectors (<em>ns</em> × 1) of random interactions of <strong><em>a</em></strong>, <strong><em>d</em></strong>, and <strong><em>i</em></strong> with site, respectively; <strong><em>I</em></strong><em><sub>s</sub></em> represents an identity matrix with order equal to the number of sites; ⊗ denotes the Kronecker product operation; <strong><em>Z</em></strong><sub>1</sub> – <strong><em>Z</em></strong><sub>3</sub> are incidence matrices (<em>ns</em> × <em>n</em>) for the main genetic effects, and <strong><em>Z</em></strong><sub>4</sub> – <strong><em>Z</em></strong><sub>6</sub> are incidence matrices (<em>ns</em> × <em>ns</em>) of interaction of main genetic effects with site, respectively.</p><p id="p-14">The genomic relationship matrices (<strong><em>G</em></strong><em><sub>a</sub></em>, <strong><em>G</em></strong><em><sub>d</sub></em>, and <strong><em>G</em></strong><em><sub>aa</sub></em>) were constructed using SNP marker information according to methods from previous studies (Vanraden 2008; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2012</a>). Briefly, <strong><em>G</em></strong><em><sub>a</sub></em> = <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-12.gif"/></noscript></span></span>, where <strong><em>M</em></strong> is an n × m matrix (n = number of individual, m = number of SNP loci) representing genotypes at each locus. The coefficient of <em>i</em><sup>th</sup> column of the <strong><em>M</em></strong> matrix are (0–2<em>p<sub>i</sub></em>), (1–2<em>p<sub>i</sub></em>) and (2–2<em>p<sub>i</sub></em>) for genotypes AA, AB and BB respectively; <em>p<sub>i</sub></em> and <em>q<sub>i</sub></em> are the frequencies of allele A and B at <em>i</em><sup>th</sup> SNP locus, respectively. Similarly, <strong><em>G</em></strong><em><sub>d</sub></em> = <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-13.gif"/></noscript></span></span>, where <em>H</em> is a n × m matrix of heterozygosity coefficients with elements (0–2<em>p<sub>i</sub>q<sub>i</sub></em>) and (1–2<em>p<sub>i</sub>q<sub>i</sub></em>) for homozygous (AA, BB) and heterozygous (AB) individuals at <em>i</em><sup>th</sup> SNP locus, respectively. The epistatic genomic relationship matrix (<strong><em>G</em></strong><em><sub>aa</sub></em>) was obtained as G#G, where # denotes the Hadamard product operation (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2012</a>). Pedigree-based relationship matrices, accounting for genetic relationships between the 25 parents, were also used for comparison purposes. <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> was implemented in software ASReml v3.0 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gilmour <em>et al.</em> 2009</a>).</p><p id="p-15">In the subsequent text, <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> will be termed as Model ADE, which includes three genetic components (<em>a</em>, <em>d</em>, and <em>i</em>). BLUP-GV (= <em>a</em> + <em>d</em> + <em>i</em>) of all 247 individuals were obtained from <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> using all available data. Estimates of variance components derived from <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> were used for calculating narrow-sense (<em>h</em><sup>2</sup>) and broad-sense (<em>H</em><sup>2</sup>) heritability, as the ratio of additive (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-14.gif"/></noscript></span></span>) to phenotypic variance (sum of all variance components in the model), and the ratio of total genetic variance (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-15.gif"/></noscript></span></span> = <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-16.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-17.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-18.gif"/></noscript></span></span>) to phenotypic variance, respectively. The method of <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yamada (1962)</a> was followed to calculate between-site genotypic correlations using estimates of variance components from <a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>. Reduced forms of <a id="xref-disp-formula-1-6" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>, by including only the additive component (Model A), and additive and dominance components (Model AD), were also tested. Goodness-of-fit for each model (<em>i.e.</em>, with and without nonadditive genetic effects) was evaluated by the log-likelihood value. The superiority of an alternative model over an additive model, obtained using all available data, was tested using a likelihood ratio test.</p></div><div id="sec-6" class="subsection"><h3>Model validation</h3><p id="p-16">To account for the family structure of our data, we applied a cross-validation scheme by using each full-sib family in-turn as a validation population (VP) and the remaining 16 families as a training population (TP), resulting in a 17-fold cross-validation. Prediction accuracy of a model was estimated as the correlation between genomic GV (GEGV) and the observed BLUP-GV of individuals in the VP. GEGV was defined as (<em>a</em> + <em>d</em> + <em>i</em>), (<em>a</em> + <em>d</em>), and <em>a</em> for the models ADE, AD, and A, respectively. The observed BLUP-GVs were linearly regressed on the GEGV, where the regression coefficient reflected the degree of bias of the GEGV, and a regression coefficient of one indicated no bias.</p><p id="p-17">Two different validation strategies were used to address practical situations. The first strategy mimics a scenario where a set of individuals (<em>i.e.</em>, VP) have not been field tested at any of the two sites (HB and MOT) of this study, <em>i.e.</em>, individuals in the VP were treated as missing at both sites. Thus, the prediction accuracy of a model in the first validation strategy was based solely on the performance of individuals in the TP. The second validation strategy considers a scenario where individuals in the VP are tested at one site (say, HB) but missing at the other site (say, MOT). The prediction accuracy in this strategy relies on the performance of TP in the target environment (MOT) and the performance of all individuals (VP and TP) from the test site (HB). This process was repeated so that all 17 families at both sites were predicted.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-18"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS1.pdf">File S1</a> contains pedigree relationships between parents, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS2.txt">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS3.txt">File S3</a> contain phenotypic and genotypic data respectively.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Basic features of the population</h3><p id="p-19">The average fruit weights (WT) at the HB and MOT sites were 183 g and 192 g, respectively, with a coefficient of variation (CV) of 23% at both sites (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Statistically significant differences between the site averages were observed for fruit weight (WT), greasiness (GRE), fruit firmness (FF), and crispness (CRI). The CVs were high and similar at both sites suggesting that heterogeneity of trait expression was consistent across the two key apple production sites. The average GRE score was higher but FF and CRI were significantly lower at MOT than at HB (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-20">A plot of the first two principal components of the SNP data matrix grouped seedlings largely according to their familial relationships (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). In almost every family there were some individuals that did not cluster within their pedigree-assigned full-sib family groupings. Such cases were more prevalent in families such as A536, A332, A334, and A535. A distribution of the SNP-based additive coefficient of relationship between all pairs of 247 individuals is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>. The SNP-based and pedigree-based average coefficients of relationship were 0.34 and 0.19, respectively. The SNP-based relatedness coefficients for pairs of supposedly half-sibs and full-sibs varied considerably (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). The average within-family relatedness coefficient varied from 0.35 (A535) to 0.66 (A537), and the between-family relatedness coefficient varied between 0.22 and 0.50 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS4.pdf">File S4</a>). Results in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, A and C</a> suggested some pollen contamination and/or mislabeling, which can be accounted for in analyses such as those in the present study by using SNP-based relationships.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal coordinates (PC) plot of all 247 individuals in the apple (Malus &#xD7; domestica Borkh.) families tested, derived from their marker genotypes (A). Pedigree-based grouping (i.e., full-sib families) is also depicted in different marker colors/shape (A). Distribution of the single nucleotide polymorphism (SNP)-based coefficients of relationships between all pairs of 247 individuals (B), and for pairs of half-sibs and full-sibs is also shown (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-44247246" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal coordinates (PC) plot of all 247 individuals in the apple (&lt;em&gt;Malus &#xD7; domestica&lt;/em&gt; Borkh.) families tested, derived from their marker genotypes (A). Pedigree-based grouping (&lt;em&gt;i.e.&lt;/em&gt;, full-sib families) is also depicted in different marker colors/shape (A). Distribution of the single nucleotide polymorphism (SNP)-based coefficients of relationships between all pairs of 247 individuals (B), and for pairs of half-sibs and full-sibs is also shown (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2711/F1.medium.gif" width="351" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/12/2711/F1.medium.gif" width="351" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467153" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Principal coordinates (PC) plot of all 247 individuals in the apple (<em>Malus × domestica</em> Borkh.) families tested, derived from their marker genotypes (A). Pedigree-based grouping (<em>i.e.</em>, full-sib families) is also depicted in different marker colors/shape (A). Distribution of the single nucleotide polymorphism (SNP)-based coefficients of relationships between all pairs of 247 individuals (B), and for pairs of half-sibs and full-sibs is also shown (C).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Estimates of variance components and heritability</h3><p id="p-22">The additive variance (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-19.gif"/></noscript></span></span>) was the major source of variability for WT (52%) and FF (50%), but the nonadditive sources were very prominent for other traits (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS5.pdf">File S5</a>). On average (across traits), the additive, dominance, and epistatic effects contributed about 30%, 16%, and 19% to the total phenotypic variance. The pedigree-based average estimates of additive and nonadditive variance were 22% and 25%, respectively (results not shown). For various traits, the magnitude of variance due to interaction of genetic effects with site varied between 0 and 3%, 0 and 4%, and 0 and 8% for additive, dominance, and epistatic effects, respectively. Estimates of between-site genotypic correlation varied between 0.87 (GRE) and 0.95 (WT), and the interaction of genetic effects with site collectively accounted for less than 10% of the total phenotypic variance (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467155/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467155/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimates of additive (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-20.gif"/></noscript></span></span>), dominance (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-21.gif"/></noscript></span></span>), and epistatic (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-22.gif"/></noscript></span></span>) genetic variance and their interaction variance in apple (<em>Malus × domestica</em> Borkh.) families with site (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-23.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-24.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/T2/embed/mml-math-25.gif"/></noscript></span></span>, respectively), expressed as the percentage of phenotypic variance (defined as the sum of variance components in the model), obtained using <a id="xref-disp-formula-1-7" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> (Model ADE)</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">The narrow-sense heritability (<em>h</em><sup>2</sup>) estimate was low (&lt; 0.20) for JUI and FIN, moderate (0.20 – 0.40) for GRE and CRI, and high (&gt;0.40) for WT and FF (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Estimated broad-sense heritability (<em>H</em><sup>2</sup>) was 0.34 for FIN and &gt;0.60 for other traits, reflecting a substantial amount of nonadditive genetic variance. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows that the estimated <em>h</em><sup>2</sup> from different models varied between 0.39 and 0.70 when the epistatic effect was dropped (<em>i.e.</em>, model AD), and between 0.47 and 0.82 when dominance and epistatic effects were excluded (<em>i.e.</em>, model A) from <a id="xref-disp-formula-1-8" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS6.pdf">File S6</a> for details). These are much higher than <em>h</em><sup>2</sup> estimates (0.11 – 0.52) of various traits obtained using the model ADE. The likelihood ratio test showed that model ADE had relatively low –2 log likelihood compared with model A (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS7.pdf">File S7</a>), but the goodness of fit was not improved significantly (<em>P</em> &gt; 0.05; degrees of freedom = 4).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Changes in estimated heritability of various apple (Malus &#xD7; domestica Borkh.) traits by fitting various genetic effects in the model. A, additive; AD, additive and dominance; ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; JUI: juiciness; WT: fruit weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-44247246" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Changes in estimated heritability of various apple (&lt;em&gt;Malus &#xD7; domestica&lt;/em&gt; Borkh.) traits by fitting various genetic effects in the model. A, additive; AD, additive and dominance; ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; JUI: juiciness; WT: fruit weight.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2711/F2.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/12/2711/F2.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467274" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Changes in estimated heritability of various apple (<em>Malus × domestica</em> Borkh.) traits by fitting various genetic effects in the model. A, additive; AD, additive and dominance; ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; JUI: juiciness; WT: fruit weight.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Accuracy of prediction</h3><p id="p-26">Prediction accuracies for the first validation strategy (no field presence of validation set) are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Correlation between the observed BLUP-GV and genomic predicted GV (<em>i.e.</em>, GEGV) was essentially the same for the additive model (model A) and for the model including additive and nonadditive effects (model ADE), and varied from about 0.15 (for JUI) to 0.35 (for FF) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The coefficient of regression of BLUP-GV on GEGV for various traits varied between 0.75 and 1.0 for model ADE, and between 0.55 and 0.75 for model A (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS8.pdf">File S8</a>). These results suggested that the degree of bias is higher for the additive model than for the model ADE. For the second validation strategy (validation set present only at one site), the accuracy of predicted GEGV was generally similar for both sites, and varied from about 0.50 (for FIN) to about 0.83 (for WT and FF) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Prediction accuracy of validation families was moderately correlated (<em>r</em> = 0.44) with the degree of genetic relatedness to the training families (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS9.pdf">File S9</a>). Prediction accuracy also showed strong relationship (<em>r</em> = 0.81) with trait heritability (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS10.pdf">File S10</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean (across families) prediction accuracy of various apple (Malus &#xD7; domestica Borkh.) traits using the additive component (Model A) and both additive and nonadditive components (Model ADE). Validation samples were assumed untested at both sites. The error bars represent standard error of the mean. A, additive; ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; JUI: juiciness; WT: fruit weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-44247246" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mean (across families) prediction accuracy of various apple (&lt;em&gt;Malus &#xD7; domestica&lt;/em&gt; Borkh.) traits using the additive component (Model A) and both additive and nonadditive components (Model ADE). Validation samples were assumed untested at both sites. The error bars represent standard error of the mean. A, additive; ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; JUI: juiciness; WT: fruit weight.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2711/F3.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/12/2711/F3.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467277" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Mean (across families) prediction accuracy of various apple (<em>Malus × domestica</em> Borkh.) traits using the additive component (Model A) and both additive and nonadditive components (Model ADE). Validation samples were assumed untested at both sites. The error bars represent standard error of the mean. A, additive; ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; JUI: juiciness; WT: fruit weight.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean (across families) prediction accuracy of various apple (Malus &#xD7; domestica Borkh.) traits for the cross-validation strategy where validation samples are assumed untested at a particular site. The prediction model included both additive and nonadditive components (i.e., model ADE). The error bars represent standard error of the mean. ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; HB, Hawke&#x2019;s Bay; JUI: juiciness; MOT, Motueka; WT: fruit weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-44247246" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mean (across families) prediction accuracy of various apple (&lt;em&gt;Malus &#xD7; domestica&lt;/em&gt; Borkh.) traits for the cross-validation strategy where validation samples are assumed untested at a particular site. The prediction model included both additive and nonadditive components (&lt;em&gt;i.e.&lt;/em&gt;, model ADE). The error bars represent standard error of the mean. ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; HB, Hawke&#x2019;s Bay; JUI: juiciness; MOT, Motueka; WT: fruit weight.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2711/F4.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/12/2711/F4.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2711/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467320" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Mean (across families) prediction accuracy of various apple (<em>Malus × domestica</em> Borkh.) traits for the cross-validation strategy where validation samples are assumed untested at a particular site. The prediction model included both additive and nonadditive components (<em>i.e.</em>, model ADE). The error bars represent standard error of the mean. ADE, additive, dominance, and epistatic; CRI: crispness; FF: fruit firmness; FIN: flavor intensity; GRE: greasiness; HB, Hawke’s Bay; JUI: juiciness; MOT, Motueka; WT: fruit weight.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Additive and nonadditive genetic variances</h3><p id="p-29">Genetic relationship matrices, constructed using genome-wide SNP markers, were fitted using a GBLUP approach to estimate nonadditive genetic variation. The difference between the pedigree-based and SNP-based additive coefficient of relationship (0.19 <em>vs.</em> 0.34) reflected that there were perhaps many more ancestral relationships not captured by the pedigree records. Different kinds of relationships (<em>e.g.</em>, half-sib, full-sib) along with permanent environmental or maternal effects shared by siblings make it challenging to partition nonadditive variance from the additive variance using traditional methods (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lee <em>et al.</em> 2010</a>). Nevertheless, ours is the first formal attempt to estimate additive, dominance, and epistatic variances of apple phenotypes. The genome-enabled estimate of nonadditive variance was similar to the additive variance (35% <em>vs.</em> 30%), suggesting that nonadditive effects are as important as additive effects for various apple phenotypes. <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mäki-Tanila and Hill (2014)</a> showed that when heterozygosity at individual loci is high, multilocus epistasis could lead to large increases in the nonadditive part of the genotypic variance. The use of F1 families, where heterozygosity is expected to be high, could have contributed to the large nonadditive variance observed in this study. Similarly to the results of a study on loblolly pine (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Muñoz <em>et al.</em> 2014b</a>), the genome-enabled estimates of genetic variance components in this study were somewhat larger than their pedigree-based counterparts. This could be due to better capture of relatedness using genome-wide markers.</p><p id="p-30">The additive variance decreased when dominance effects were included in the model, and it decreased further when epistatic effects were considered. The average <em>h</em><sup>2</sup> was 0.69, 0.52, and 0.30 for the models A, AD, and ADE, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). These results highlight the confounding nature of the additive and nonadditive effects. In other words, effects which the model ADE allocated to dominance and epistatic effects, were now being absorbed by the inferred additive genetic component in model A. This means that models ignoring nonadditive components would yield upwardly biased estimates of additive variance if the true variance components of ignored effects were not zero (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lu <em>et al.</em> 1999</a>). The covariance between full-sibs and half-sibs includes <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-33.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-34.gif"/></noscript></span></span> respectively, so there is a ‘hidden’ nonadditive component even if an additive model is fitted. Moreover, depending on the distribution of allele frequencies and linkage disequilibrium (LD) patterns, some of the variance due to interaction of alleles can manifest as additive variance (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zuk <em>et al.</em> 2012</a>). Similarly to our results, decrease in <em>h</em><sup>2</sup> by including nonadditive effects in the model have been reported by various studies on forest trees (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Waldmann <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Araújo <em>et al.</em> 2012</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Muñoz <em>et al.</em> 2014b</a>).</p><p id="p-31">The estimated <em>h</em><sup>2</sup> of WT (0.52) in this study was very similar to those in some other reports (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alspach and Oraguzie 2002</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kouassi <em>et al.</em> 2009</a>). Although the <em>h</em><sup>2</sup> of FIN (0.11) was similar to that reported by <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kouassi <em>et al.</em> (2009)</a>, it was lower than that found by <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Durel <em>et al.</em> (1998)</a> (0.39). Heritability (<em>h</em><sup>2</sup>) of CRI and JUI has been reported to vary between 0.15 and 0.40 (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Durel <em>et al.</em> 1998</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alspach and Oraguzie 2002</a>; <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Kouassi <em>et al.</em> 2009</a>), but higher heritabilities (about 0.60) have been reported (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> 2011</a>). The estimated <em>H</em><sup>2</sup> in our study varied between 0.34 (FIN) and 0.79 (FF), with an average of about 0.65 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">King <em>et al.</em> (2000)</a>, using clonally replicated seedlings from a single full-sib family, reported estimates of <em>H</em><sup>2</sup> of FF, CRI, and JUI to be 0.52, 0.57, and 0.46, respectively. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McKay <em>et al.</em> (2011)</a> reported estimated <em>H</em><sup>2</sup> values of JUI and CRI as 0.72 and 0.76, respectively, which were higher than those observed in our study (0.60 and 0.66, respectively). The ratio of nonadditive (= <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-35.gif"/></noscript></span></span>+<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-36.gif"/></noscript></span></span>) to additive (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2711/embed/mml-math-37.gif"/></noscript></span></span>) variance varied between 0.48 (WT) and 2.35 (JUI), with an average of 1.53, suggesting that nonadditive variance contributed significantly to the expression of apple fruit phenotypes.</p></div><div id="sec-14" class="subsection"><h3>Influence of explicit fitting of nonadditive components on prediction accuracy</h3><p id="p-32">BLUP-BV or deregressed EBVs have commonly been used as phenotypes in most applications of genomic selection in animal and plant species (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Garrick <em>et al.</em> 2009</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> 2012</a>). Such data allow only the estimation of allele substitution effects, so distinguishing between additive and nonadditive effects is not possible. The use of BLUEs as phenotypes in genomic predictions is desirable to distinguish between additive and nonadditive effects, and to reduce the computation time compared with using multiple records per individual (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> 2014</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Xu <em>et al.</em> 2014</a>). Using the phenotypes adjusted for nongenetic effects (replication, year), we obtained BLUP-GV (sum of additive, dominance, and epistatic effects) of all individuals for cross-validation of genomic predictions. There was a near-perfect correlation between BLUP-GV and BLUEs for all traits, reflecting that pedigree/family relationships had little effect on the performance ranking (<em>i.e.</em>, BLUP-GV), in contrast to unbalanced datasets with little or no clonal replication of individuals (<em>e.g.</em>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Garrick <em>et al.</em> 2009</a>). Similar observations were reported in studies on forest tress species (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zapata-Valenzuela <em>et al.</em> 2012</a>).</p><p id="p-33">The accuracy of predicting unobserved BLUP-GV varied from about 0.15 to 0.35, and these were almost identical between the models with or without including nonadditive effects. As shown by <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Xu <em>et al.</em> (2014)</a>, the lack of improvement in accuracy could be attributed to the high correlation between different estimated variance components. For all traits in our study, vectors of estimated additive effects were highly correlated with nonadditive effects for both the marker-based (about 0.80) and pedigree-based (about 0.60) models. If genes are not independently distributed (<em>i.e.</em>, LD) in the parents of a population, a covariance between the additive and nonadditive effects would be introduced (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Melchinger 1988</a>), and what otherwise would be epistatic variance becomes additive or dominance variance (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Hill <em>et al.</em> 2008</a>). Studies on animal species showed that prediction accuracy increased when both (additive and nonadditive) effects rather than just additive effects were included (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2012</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Sun <em>et al.</em> 2014</a>). Although the prediction accuracy of models A and ADE was similar for the apple phenotypes, the prediction bias was lower for the latter. The regression coefficients for the predictions using the model ADE were closer to 1, so including nonadditive effects did not improve the accuracy but helped to reduce the bias of predicting GV.</p><p id="p-34">Relatively low prediction accuracy (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) in this study compared with that in the study by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em> (2012)</a> was mainly due to the sample size (about 250 <em>cf</em>. 1000). Moreover, the large family sizes and a random cross-validation scheme in <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em> (2012)</a> meant that genomic predictions were essentially made within the same families that were part of the training population. For across family validation scheme as in this study, genetic relationship between the training and validation families, and population-level LD, are the key drivers of prediction accuracy (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2008</a>). Relationships at the level of parents and grandparents resulted in high relatedness among the 17 families in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS4.pdf">File S4</a>). Genetic relatedness between the training and validation families, and trait heritability were positively correlated with the prediction accuracy (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS9.pdf">File S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1/FileS10.pdf">File S10</a>), which is consistent with previous studies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Daetwyler <em>et al.</em> 2008</a>).</p></div><div id="sec-15" class="subsection"><h3>Genotype–environment interaction</h3><p id="p-35">It is common practice to include some common ‘checks’ or ‘controls’ across various test environments, to extrapolate performance of an accession at an untested site. Pedigree information could also be used for predicting performance if some known relatives were tested at the target site. Genome-wide SNPs, which provide precise estimates of genetic relationships, will provide better links between accessions at different sites for the understanding of G × E and predicting performance in different environments. The accuracy of predicting unobserved BLUP-GV at HB using performance from MOT, or vice versa, was about 0.75 for all traits except JUI (0.62) and FIN (0.52). Although the G × E variance components for JUI and FIN were similar to those for other traits (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), the lower heritability of these two traits might have contributed to these results.</p><p id="p-36">The prediction accuracy of the second validation strategy (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) was at least twice that of the first validation strategy (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). These results supported earlier findings that predicting the performance of untested individuals is more challenging than predicting the performance of individuals that have been evaluated at some sites (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burgueño <em>et al.</em> 2012</a>). When the training and validation samples are observed independently over different sites and ages/years, prediction accuracies can be affected, depending on the magnitude of genotype-by-site and genotype-by-age interactions (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> 2012</a>).</p><p id="p-37">This study showed that nonadditive genetic effects are important sources of genetic variation for apple phenotypes, so there is a good case to account for these effects when selecting individuals as potential cultivars. For selection of individuals as parents for new crosses the additive effects is probably more relevant as the nonadditive effects do not transfer very effectively to the next generation. The GBLUP approach is relatively simple to implement to estimate additive and nonadditive variances and to predict GEBV and GEGV. GBLUP models fitting only the additive effects can predict phenotype with similar accuracy to models fitting both the additive and nonadditive effects, but the degree of bias will be lower for the latter. Also, GBLUP models ignoring nonadditive effects will provide an overestimated additive variance that will exaggerate the expected genetic gains in breeding. The high correlation between the additive and nonadditive effects may be due to the genetic relatedness between samples, so including nonadditive genetic effects in GBLUP models may still improve prediction accuracy in populations of less-related individuals—this hypothesis needs to be tested.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">New Zealand Institute for Plant &amp; Food Research Limited (PFR) colleagues Luis Gea and Jo Stephens are thanked for their helpful comments. We thank reviewers and the editor for constructive feedback on the manuscript. The PFR Markers &amp; Mapping Team is thanked for help in collecting leaf samples. Slipstream Automation Ltd, New Zealand is thanked for providing high-throughput DNA extraction services. We also thank AgResearch Invermay, New Zealand, for providing Illumina SNP Array genotyping. This research was partly supported by Prevar (Contract no. 26015) and the New Zealand Ministry of Business, Innovation and Employment (MBIE) through a Core Funding program.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-39">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.021105/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-40"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-06"><span class="received-label">Received </span>August 6, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-10-05"><span class="accepted-label">Accepted </span>October 5, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Kumar <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.1" data-doi="10.1080/01140671.2002.9514218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alspach</span>  <span class="cit-name-given-names">P.A.</span></span></li></ol><cite>, and N.C. Oraguzie, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimation of genetic parameters of apple fruit quality from open pollinated families.</span> <abbr class="cit-jnl-abbrev">N. Z. J. Crop Hortic. Sci.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BCrop%2BHortic.%2BSci.%26rft.volume%253D30%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01140671.2002.9514218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01140671.2002.9514218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.2" data-doi="10.1007/s11295-011-0443-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araújo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borralho</span>  <span class="cit-name-given-names">N. M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehon</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The importance and type of non additive genetic effects for growth in <em>Eucalyptus globulus</em>.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0443-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0443-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.3" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing data inference for whole genome association studies using localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burdon</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelbourne</span>  <span class="cit-name-given-names">C. J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The use of vegetative propagules for obtaining genetic information.</span> <abbr class="cit-jnl-abbrev">N. Z. J. For. Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BFor.%2BSci.%26rft.volume%253D4%26rft.spage%253D418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.5" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, and J. Crossa, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype-environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.6" data-doi="10.1371/journal.pone.0031745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowhurst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-Wide SNP detection, validation, and development of an 8K SNP array for apple.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e31745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChagne%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De31745%26rft.epage%253De31745%26rft.atitle%253DGenome-wide%2BSNP%2Bdetection%252C%2Bvalidation%252C%2Band%2Bdevelopment%2Bof%2Ban%2B8K%2BSNP%2Barray%2Bfor%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0031745%26rft_id%253Dinfo%253Apmid%252F22363718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0031745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363718&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.7" data-doi="10.1371/journal.pone.0003395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of predicting the genetic risk of disease using a genome-wide approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e3395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3395%26rft.epage%253De3395%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bthe%2Bgenetic%2Brisk%2Bof%2Bdisease%2Busing%2Ba%2Bgenome-wide%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003395%26rft_id%253Dinfo%253Apmid%252F18852893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852893&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.8" data-doi="10.1071/AN10096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henshall</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gredler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accuracy of estimated genomic breeding values for wool and meat traits in a multi-breed sheep population.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D50%26rft.spage%253D1004%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN10096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN10096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.9" data-doi="10.1007/s00122-014-2341-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bariana</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction for rust resistance in diverse wheat landraces.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1795%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2341-8%26rft_id%253Dinfo%253Apmid%252F24965887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2341-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24965887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.10" data-doi="10.1007/s001220050842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durel</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurens</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouillet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lespinasse</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Utilization of pedigree information to estimate genetic parameters from large unbalanced data sets in apple.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D96%26rft.spage%253D1077%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.11" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-5.12.2711.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml User Guide Release 3.0</span>, <span class="cit-publ-name">VSN International Ltd</span>, <span class="cit-publ-loc">Hemel Hempstead, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.13" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.14" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI3MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.15" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.16" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.17" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.18" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1000008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.19" data-doi="10.1007/s001220051389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynn</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alston</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durel</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Quantitative genetic analysis and comparison of physical and sensory descriptors relating to fruit flesh firmness in apple (<em>Malus pumila</em> Mill.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">1074</span>–<span class="cit-lpage">1084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D100%26rft.spage%253D1074%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.20" data-doi="10.1007/s11295-009-0217-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kouassi</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durel</span>  <span class="cit-name-given-names">C.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tartarini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Weg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimation of genetic parameters and prediction of breeding values for apple fruit-quality traits using pedigreed plant material in Europe.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D5%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-009-0217-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-009-0217-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.21" data-doi="10.1007/s11295-011-0396-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weskett</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic architecture of fruit quality traits in <em>Malus</em> x <em>domestica</em> (Borkh.) compared between own-rooted seedlings and vegetative propagules on ‘M. 9’ rootstock.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D1079%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0396-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0396-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.22" data-doi="10.1371/journal.pone.0036674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection for fruit quality traits in apple (<em>Malus × domestica</em> Borkh.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36674%26rft.epage%253De36674%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bfruit%2Bquality%2Btraits%2Bin%2Bapple%2B%2528Malus%25C3%2597domestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036674%26rft_id%253Dinfo%253Apmid%252F22574211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574211&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.23" data-doi="10.1186/1471-2164-14-93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Novel genomic approaches unravel genetic architecture of complex traits in apple.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-93%26rft_id%253Dinfo%253Apmid%252F23398910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23398910&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.24" data-doi="10.1186/1297-9686-42-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using the realized relationship matrix to disentangle confounding factors for the estimation of genetic variance components of complex traits.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BH.%26rft.volume%253D42%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DUsing%2Bthe%2Brealized%2Brelationship%2Bmatrix%2Bto%2Bdisentangle%2Bconfounding%2Bfactors%2Bfor%2Bthe%2Bestimation%2Bof%2Bgenetic%2Bvariance%2Bcomponents%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-22%26rft_id%253Dinfo%253Apmid%252F20546624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20546624&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.25" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.26" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.27" data-doi="10.1071/CP13363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in crops, trees and forages: a review.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D65%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP13363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP13363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.28" data-doi="10.1139/x99-047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Potential biases of incomplete linear models in heritability estimation and breeding value prediction.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D29%26rft.spage%253D724%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx99-047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x99-047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.29" data-doi="10.1534/genetics.114.165282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mäki-Tanila</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Influence of gene interaction on complex trait variation with multilocus models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165282%26rft_id%253Dinfo%253Apmid%252F24990992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradeen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luby</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction of genotypic values for apple fruit texture traits in a breeding population derived from ‘Honeycrisp’.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">408</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D136%26rft.spage%253D408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.12.2711.31"><div class="cit-metadata unstructured">Melchinger, A. E., 1988 Means, variance, and covariances between relatives in hybrid populations with disequilibrium in the parent populations, pp. 400–415 in <em>Proceedings of the Second International Conference on Quantitative Genetics</em>, edited by B. S. Weir, E. J. Eisen, M. M. Goodman and G. Namkoong. Sinauer, Sunderland.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.32" data-doi="10.2135/cropsci2012.12.0673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quezada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic relationship matrix for correcting pedigree errors in breeding populations: impact on genetic parameters and genomic selection accuracy.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1115%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.12.0673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.12.0673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.33" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Unraveling additive from nonadditive effects using genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI3MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.34" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Relatedness in the post-genomic era: is it still useful?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D33%26rft_id%253Dinfo%253Apmid%252F25404112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25404112&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.36" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical note: derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.37" data-doi="10.1371/journal.pone.0045293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSu%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45293%26rft.epage%253De45293%26rft.atitle%253DEstimating%2Badditive%2Band%2Bnon-additive%2Bgenetic%2Bvariances%2Band%2Bpredicting%2Bgenetic%2Bmerits%2Busing%2Bgenome-wide%2Bdense%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045293%26rft_id%253Dinfo%253Apmid%252F23028912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028912&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.38" data-doi="10.1371/journal.pone.0103934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">vanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improvement of prediction ability for genomic selection of dairy cattle by including dominance effects.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e103934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103934%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103934%26rft_id%253Dinfo%253Apmid%252F25084281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25084281&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.39" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Raden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2711.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.40" data-doi="10.1534/genetics.107.084160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waldmann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoti</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, and M. J. Sillanpää, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient Markov chain Monte Carlo implementation of Bayesian analysis of additive and dominance genetic variances in noninbred pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWaldmann%26rft.auinit1%253DP.%26rft.volume%253D179%26rft.issue%253D2%26rft.spage%253D1101%26rft.epage%253D1112%26rft.atitle%253DEfficient%2BMarkov%2BChain%2BMonte%2BCarlo%2BImplementation%2Bof%2BBayesian%2BAnalysis%2Bof%2BAdditive%2Band%2BDominance%2BGenetic%2BVariances%2Bin%2BNoninbred%2BPedigrees%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084160%26rft_id%253Dinfo%253Apmid%252F18558655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzExMDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI3MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-5.12.2711.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Forest Genetics</span>, <span class="cit-publ-name">CABI</span>, <span class="cit-publ-loc">Wallingford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.42" data-doi="10.1073/pnas.1413750111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Predicting hybrid performance in rice using genomic best linear unbiased prediction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">12456</span>–<span class="cit-lpage">12461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413750111%26rft_id%253Dinfo%253Apmid%252F25114224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM0LzEyNDU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.43" data-doi="10.1266/jjg.37.498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-article-title">Genotype by environment interaction and genetic correlation of the same traits under different environments.</span> <abbr class="cit-jnl-abbrev">Jpn. J. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJpn.%2BJ.%2BGenet.%26rft.volume%253D37%26rft.spage%253D498%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fjjg.37.498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/jjg.37.498&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.44" data-doi="10.1007/s11295-012-0516-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zapata-Valenzuela</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isik</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maltecca</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP markers trace familial linkages in a cloned population of <em>Pinus taeda</em> - prospects for genomic selection.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D1307%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-012-0516-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-012-0516-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2711.45" data-doi="10.1073/pnas.1119675109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hechter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mystery of missing heritability: genetic interactions create phantom heritability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1119675109%26rft_id%253Dinfo%253Apmid%252F22223662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTE5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjcxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/12/2711.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2693" title="Heterosis Increases Fertility, Fecundity, and Survival of Laboratory-Produced F1 Hybrid Males of the Malaria Mosquito Anopheles coluzzii" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2719" title="Caenorhabditis elegans glp-4 Encodes a Valyl Aminoacyl tRNA Synthetase" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/12"><span>Volume 5 Issue 12, December 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/12.cover-source.jpg?itok=RngHqKQq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2711" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oLhtWKXrt_AENecxSqRLMHTZNRuhjhtGPVcH9QxsHu0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/12/2711" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/12/2711" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/12/2711" class="active" data-icon-position="" data-hide-link-title="0">Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460225" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NgrBoX_CcwZ_MDEJ0ioRqKlTnmjYC5ldHuceDEx6eEo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2711.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/12/2711" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sIiOztZmvhPwSgn345sqi38ruufKGksCSTq9JxtuM3E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/12/2711" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2711.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460225" id="node460225--2" data-pisa="ggg;5/12/2711" data-pisa-master="ggg;g3.115.021105" data-apath="/ggg/5/12/2711.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Satish Kumar</span>, <span class="highwire-citation-author" data-delta="1">Claire Molloy</span>, <span class="highwire-citation-author" data-delta="2">Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="3">Hans Daetwyler</span>, <span class="highwire-citation-author" data-delta="4">David Chagné</span> and <span class="highwire-citation-author" data-delta="5">Richard Volz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2711-2718;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021105 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460225--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Satish Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satish%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASatish%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Satish.Kumar@plantandfood.co.nz" class="" data-icon-position="" data-hide-link-title="0">Satish.Kumar@plantandfood.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Molloy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BMolloy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Molloy%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BMolloy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Agronomy Department, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hans Daetwyler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>AgriBio Center, La Trobe University, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2BDaetwyler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daetwyler%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2BDaetwyler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Chagné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant & Food Research, Palmerston North Research Center, Palmerston North, 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BChagn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chagn%C3%A9%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BChagn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard Volz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BVolz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Volz%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BVolz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/12/2711" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/12/2711&amp;title=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/12/2711&amp;title=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/12/2711&amp;title=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/12/2711&amp;text=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/12/2711&amp;title=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/12/2711&amp;t=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/12/2711&amp;title=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/12/2711&amp;title=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460225/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460225" id="node460225--3" data-pisa="ggg;5/12/2711" data-pisa-master="ggg;g3.115.021105" data-apath="/ggg/5/12/2711.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Enabled Estimates of Additive and Nonadditive Genetic Variances and Prediction of Apple Phenotypes Across Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Satish Kumar</span>, <span class="highwire-citation-author" data-delta="1">Claire Molloy</span>, <span class="highwire-citation-author" data-delta="2">Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="3">Hans Daetwyler</span>, <span class="highwire-citation-author" data-delta="4">David Chagné</span> and <span class="highwire-citation-author" data-delta="5">Richard Volz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2711-2718;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021105 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460225--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Satish Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satish%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASatish%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Satish.Kumar@plantandfood.co.nz" class="" data-icon-position="" data-hide-link-title="0">Satish.Kumar@plantandfood.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Molloy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BMolloy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Molloy%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BMolloy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Agronomy Department, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hans Daetwyler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>AgriBio Center, La Trobe University, Bundoora, Victoria 3083, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2BDaetwyler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daetwyler%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2BDaetwyler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Chagné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant & Food Research, Palmerston North Research Center, Palmerston North, 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BChagn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chagn%C3%A9%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BChagn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard Volz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Hawke’s Bay Research Center, Havelock North, 4130 New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BVolz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Volz%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BVolz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2711&amp;rft.epage=2718&amp;rft.atitle=Genome-Enabled%20Estimates%20of%20Additive%20and%20Nonadditive%20Genetic%20Variances%20and%20Prediction%20of%20Apple%20Phenotypes%20Across%20Environments&amp;rft.volume=5&amp;rft.issue=12&amp;rft.date=2015-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kumar%2C+Satish&amp;rft.au=Molloy%2C+Claire&amp;rft.au=Mu%C3%B1oz%2C+Patricio&amp;rft.au=Daetwyler%2C+Hans&amp;rft.au=Chagn%C3%A9%2C+David&amp;rft.au=Volz%2C+Richard"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460225/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460225/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460225/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460225/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460225/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460225/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460225/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460225/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460225/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460225/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460225/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460225/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/12/2711" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/12/2711.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/12/2711.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/12/2711.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/12/2711.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
