<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/12/2629" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/12/2629.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/12/2629.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/12/2629.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/12/2629/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/12/2629.atom" />
<meta name="HW.pisa" content="ggg;5/12/2629" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data" />
<meta name="DC.Identifier" content="10.1534/g3.115.019513" />
<meta name="DC.Date" content="2015-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Lopes et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Traditionally, exploration of genetic variance in humans, plants, and livestock species has been limited mostly to the use of additive effects estimated using pedigree data. However, with the development of dense panels of single-nucleotide polymorphisms (SNPs), the exploration of genetic variation of complex traits is moving from quantifying the resemblance between family members to the dissection of genetic variation at individual loci. With SNPs, we were able to quantify the contribution of additive, dominance, and imprinting variance to the total genetic variance by using a SNP regression method. The method was validated in simulated data and applied to three traits (number of teats, backfat, and lifetime daily gain) in three purebred pig populations. In simulated data, the estimates of additive, dominance, and imprinting variance were very close to the simulated values. In real data, dominance effects account for a substantial proportion of the total genetic variance (up to 44%) for these traits in these populations. The contribution of imprinting to the total phenotypic variance of the evaluated traits was relatively small (1–3%). Our results indicate a strong relationship between additive variance explained per chromosome and chromosome length, which has been described previously for other traits in other species. We also show that a similar linear relationship exists for dominance and imprinting variance. These novel results improve our understanding of the genetic architecture of the evaluated traits and shows promise to apply the SNP regression method to other traits and species, including human diseases." />
<meta name="DC.Contributor" content="Marcos S. Lopes" />
<meta name="DC.Contributor" content="John W. M. Bastiaansen" />
<meta name="DC.Contributor" content="Luc Janss" />
<meta name="DC.Contributor" content="Egbert F. Knol" />
<meta name="DC.Contributor" content="Henk Bovenhuis" />
<meta name="article:published_time" content="2015-12-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Traditionally, exploration of genetic variance in humans, plants, and livestock species has been limited mostly to the use of additive effects estimated using pedigree data. However, with the development of dense panels of single-nucleotide polymorphisms (SNPs), the exploration of genetic variation of complex traits is moving from quantifying the resemblance between family members to the dissection of genetic variation at individual loci. With SNPs, we were able to quantify the contribution of additive, dominance, and imprinting variance to the total genetic variance by using a SNP regression method. The method was validated in simulated data and applied to three traits (number of teats, backfat, and lifetime daily gain) in three purebred pig populations. In simulated data, the estimates of additive, dominance, and imprinting variance were very close to the simulated values. In real data, dominance effects account for a substantial proportion of the total genetic variance (up to 44%) for these traits in these populations. The contribution of imprinting to the total phenotypic variance of the evaluated traits was relatively small (1–3%). Our results indicate a strong relationship between additive variance explained per chromosome and chromosome length, which has been described previously for other traits in other species. We also show that a similar linear relationship exists for dominance and imprinting variance. These novel results improve our understanding of the genetic architecture of the evaluated traits and shows promise to apply the SNP regression method to other traits and species, including human diseases.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/12/01" />
<meta name="citation_mjid" content="ggg;5/12/2629" />
<meta name="citation_id" content="5/12/2629" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/12/2629" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/12/2629.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/12/2629.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/12/2629.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019513" />
<meta name="citation_pmid" content="26438289" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2629" />
<meta name="citation_lastpage" content="2637" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marcos S. Lopes" />
<meta name="citation_author_institution" content="Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands" />
<meta name="citation_author_email" content="Marcos.Lopes@topigsnorsvin.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4715-853X" />
<meta name="citation_author" content="John W. M. Bastiaansen" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands" />
<meta name="citation_author" content="Luc Janss" />
<meta name="citation_author_institution" content="Centre for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark" />
<meta name="citation_author" content="Egbert F. Knol" />
<meta name="citation_author_institution" content="Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands" />
<meta name="citation_author" content="Henk Bovenhuis" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=R. Abdollahi‐Arpanahi;citation_author=A. Pakdel;citation_author=A. Nejati‐Javaremi;citation_author=M. Moradi Shahrbabak;citation_author=G. Morota;citation_title=Dissection of additive genetic variability for quantitative traits in chickens using SNP markers.;citation_pages=183-193;citation_volume=131;citation_year=2014;citation_pmid=24460953;citation_doi=10.1111/jbg.12079" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Automat. Contr.;citation_author=H. Akaike;citation_title=A new look at the statistical model identification.;citation_pages=716-723;citation_volume=19;citation_year=1974;citation_doi=10.1109/TAC.1974.1100705" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. B. Bruce;citation_title=THE MENDELIAN THEORY OF HEREDITY AND THE AUGMENTATION OF VIGOR;citation_pages=627-628;citation_volume=32;citation_year=1910;citation_issue=827;citation_pmid=17816706;citation_doi=10.1126/science.32.827.627-a" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Charlesworth;citation_author=J. H. Willis;citation_title=The genetics of inbreeding depression.;citation_pages=783-796;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19834483;citation_doi=10.1038/nrg2664" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=Y. Cheng;citation_author=S. Rachagani;citation_author=A. Cánovas;citation_author=M. S. Mayes;citation_author=R. G. Tait;citation_title=Body composition and gene expression QTL mapping in mice reveals imprinting and interaction effects.;citation_pages=103;citation_volume=14;citation_year=2013;citation_pmid=24165562" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Da;citation_author=C. Wang;citation_author=S. Wang;citation_author=G. Hu;citation_title=Mixed model methods for genomic prediction and variance component estimation of additive and dominance effects using SNP markers.;citation_pages=e87666;citation_volume=9;citation_year=2014;citation_pmid=24498162;citation_doi=10.1371/journal.pone.0087666" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. B. Davenport;citation_title=DEGENERATION, ALBINISM AND INBREEDING;citation_pages=454-455;citation_volume=28;citation_year=1908;citation_issue=718;citation_pmid=17771943;citation_doi=10.1126/science.28.718.454-b" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D.-J. De Koning;citation_author=H. Bovenhuis;citation_author=J. A. van Arendonk;citation_title=On the Detection of Imprinted Quantitative Trait Loci in Experimental Crosses of Outbred Species;citation_pages=931-938;citation_volume=161;citation_year=2002;citation_issue=2;citation_pmid=12072486" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=D. B. Allison;citation_title=Predicting genetic predisposition in humans: the promise of whole-genome markers.;citation_pages=880-886;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21045869;citation_doi=10.1038/nrg2898" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. De Vries;citation_author=R. Kerr;citation_author=B. Tier;citation_author=T. Long;citation_title=Gametic imprinting effects on rate and composition of pig growth.;citation_pages=1037-1042;citation_volume=88;citation_year=1994;citation_pmid=24186259;citation_doi=10.1007/BF00220813" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=N. Duijvesteijn;citation_author=J. M. Veltmaat;citation_author=E. F. Knol;citation_author=B. Harlizius;citation_title=High-resolution association mapping of number of teats in pigs reveals regions controlling vertebral development.;citation_pages=542;citation_volume=15;citation_year=2014;citation_pmid=24981054;citation_doi=10.1186/1471-2164-15-542" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0;citation_author=A. R. Gilmour;citation_author=B. Gogel;citation_author=B. Cullis;citation_author=R. Thompson;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. A. Groenen;citation_author=A. L. Archibald;citation_author=H. Uenishi;citation_author=C. K. Tuggle;citation_author=Y. Takeuchi;citation_title=Analyses of pig genomes provide insight into porcine demography and evolution.;citation_pages=393-398;citation_volume=491;citation_year=2012;citation_issue=7424;citation_pmid=23151582;citation_doi=10.1038/nature11622" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. M. Hickey;citation_author=B. P. Kinghorn;citation_author=B. Tier;citation_author=J. F. Wilson;citation_author=N. Dunstan;citation_title=A combined long-range phasing and long haplotype imputation method to impute phase for SNP genotypes.;citation_pages=12-12;citation_volume=43;citation_year=2011;citation_pmid=21388557;citation_doi=10.1186/1297-9686-43-12" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=W. G. Hill;citation_title=Understanding and using quantitative genetic variation;citation_pages=73-85;citation_volume=365;citation_year=2010;citation_issue=1537;citation_pmid=20008387;citation_doi=10.1098/rstb.2009.0203" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=H. Hirooka;citation_author=D. De Koning;citation_author=B. Harlizius;citation_author=J. Van Arendonk;citation_author=A. Rattink;citation_title=A whole-genome scan for quantitative trait loci affecting teat number in pigs;citation_pages=2320-2326;citation_volume=79;citation_year=2001;citation_issue=9;citation_pmid=11583418" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J.-T. Jeon;citation_author=Ö. Carlborg;citation_author=A. Törnsten;citation_author=E. Giuffra;citation_author=V. Amarger;citation_title=A paternally expressed QTL affecting skeletal and cardiac muscle mass in pigs maps to the IGF2 locus.;citation_pages=157-158;citation_volume=21;citation_year=1999;citation_issue=2;citation_pmid=9988263;citation_doi=10.1038/5938" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=K. Krag;citation_author=L. Janss;citation_author=M. Shariati;citation_author=P. Berg;citation_author=A. J. Buitenhuis;citation_title=SNP-based heritability estimation using a Bayesian approach.;citation_pages=531-539;citation_volume=7;citation_year=2013;citation_pmid=23177174;citation_doi=10.1017/S1751731112002017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=H. A. Lawson;citation_author=J. M. Cheverud;citation_author=J. B. Wolf;citation_title=Genomic imprinting and parent-of-origin effects on complex traits.;citation_pages=609-617;citation_volume=14;citation_year=2013;citation_pmid=23917626;citation_doi=10.1038/nrg3543" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Le Rouzic;citation_author=J. M. Álvarez-Castro;citation_author=Ö. Carlborg;citation_title=Dissection of the genetic architecture of body weight in chicken reveals the impact of epistasis on domestication traits.;citation_volume=179;citation_year=2008;citation_pmid=18622035;citation_doi=10.1534/genetics.108.089300" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. H. Lee;citation_author=J. H. van der Werf;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Predicting unobserved phenotypes for complex traits from whole-genome SNP data.;citation_pages=e1000231-e1000231;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18949033;citation_doi=10.1371/journal.pgen.1000231" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Maher;citation_title=Personal genomes: The case of the missing heritability.;citation_pages=18-21;citation_volume=456;citation_year=2008;citation_issue=7218;citation_pmid=18987709" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. E. Melchinger;citation_author=H.-P. Piepho;citation_author=H. F. Utz;citation_author=J. Muminović;citation_author=T. Wegenast;citation_title=Genetic Basis of Heterosis for Growth-Related Traits in Arabidopsis Investigated by Testcross Progenies of Near-Isogenic Lines Reveals a Significant Role of Epistasis;citation_pages=1827-1837;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=18039884;citation_doi=10.1534/genetics.107.080564" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=P.R. Muñoz;citation_author=M.F. Resende;citation_author=S.A. Gezan;citation_author=M.D.V. Resende;citation_author=G. de los Campos;citation_title=Unraveling additive from non-additive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Nezer;citation_author=L. Moreau;citation_author=B. Brouwers;citation_author=W. Coppieters;citation_author=J. Detilleux;citation_title=An imprinted QTL with major effect on muscle mass and fat deposition maps to the IGF2 locus in pigs.;citation_pages=155-156;citation_volume=21;citation_year=1999;citation_issue=2;citation_pmid=9988262;citation_doi=10.1038/5935" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Nishio;citation_author=M. Satoh;citation_title=Including dominance effects in the genomic BLUP method for genomic evaluation.;citation_pages=e85792;citation_volume=9;citation_year=2014;citation_pmid=24416447;citation_doi=10.1371/journal.pone.0085792" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=M. J. R. Pante;citation_author=B. Gjerde;citation_author=I. McMillan;citation_author=I. Misztal;citation_title=Estimation of additive and dominance genetic variances for body weight at harvest in rainbow trout, Oncorhynchus mykiss.;citation_pages=383-392;citation_volume=204;citation_year=2002;citation_doi=10.1016/S0044-8486(01)00825-0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Ramos;citation_author=R. P. M. A. Crooijmans;citation_author=N. A. Affara;citation_author=A. J. Amaral;citation_author=A. L. Archibald;citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.;citation_pages=e6524-e6524;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19654876;citation_doi=10.1371/journal.pone.0006524" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=G. Shen;citation_author=W. Zhan;citation_author=H. Chen;citation_author=Y. Xing;citation_title=Dominance and epistasis are the main contributors to heterosis for plant height in rice.;citation_pages=11-18;citation_volume=215;citation_year=2014;citation_pmid=24388510" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=G. H. Shull;citation_title=The composition of a field of maize.;citation_pages=296-301;citation_volume=4;citation_year=1908" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. Sorensen;citation_author=R. Fernando;citation_author=D. Gianola;citation_title=Inferring the trajectory of genetic variance in the course of artificial selection.;citation_pages=83-94;citation_volume=77;citation_year=2001;citation_issue=1;citation_pmid=11279833;citation_doi=10.1017/S0016672300004845" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=D. J. Spiegelhalter;citation_author=N. G. Best;citation_author=B. P. Carlin;citation_author=A. Van Der Linde;citation_title=Bayesian measures of model complexity and fit.;citation_pages=583-639;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00353" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=T. Ostersen;citation_author=M. Henryon;citation_author=M. S. Lund;citation_title=Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.;citation_pages=e45293-e45293;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23028912;citation_doi=10.1371/journal.pone.0045293" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Sun;citation_author=P. M. VanRaden;citation_author=J. B. Cole;citation_author=J. R. O’Connell;citation_title=Improvement of prediction ability for genomic selection of dairy cattle by including dominance effects.;citation_pages=e103934;citation_volume=9;citation_year=2014;citation_pmid=25084281;citation_doi=10.1371/journal.pone.0103934" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=B. Tier;citation_author=K. Meyer;citation_title=Approximating prediction error covariances among additive genetic effects within animals in multiple‐trait and random regression models.;citation_pages=77-89;citation_volume=121;citation_year=2004;citation_doi=10.1111/j.1439-0388.2003.00444.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. A. Toro;citation_author=L. Varona;citation_title=A note on mate allocation for dominance handling in genomic selection.;citation_pages=33-33;citation_volume=42;citation_year=2010;citation_pmid=20699012;citation_doi=10.1186/1297-9686-42-33" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=A. A. Vinkhuyzen;citation_author=N. R. Wray;citation_author=J. Yang;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Estimation and partitioning of heritability in human populations using whole genome analysis methods.;citation_pages=75;citation_volume=47;citation_year=2013;citation_pmid=23988118;citation_doi=10.1146/annurev-genet-111212-133258" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=P.M. Visscher;citation_author=M.E. Goddard;citation_title=A general unified framework to assess the sampling variance of heritability estimates using pedigree or marker-based relationships.;citation_volume=199;citation_year=2015;citation_pmid=25361897;citation_doi=10.1534/genetics.114.171017" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=P. Visscher;citation_author=R. Pong-Wong;citation_author=C. Whittemore;citation_author=C. Haley;citation_title=Impact of biotechnology on (cross) breeding programmes in pigs.;citation_pages=57-70;citation_volume=65;citation_year=2000;citation_doi=10.1016/S0301-6226(99)00180-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=A. Legarra;citation_title=On the additive and dominant variance and covariance of individuals within the genomic selection scope.;citation_volume=195;citation_year=2013;citation_pmid=24121775;citation_doi=10.1534/genetics.113.155176" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wolf;citation_author=J. M. Cheverud;citation_title=Detecting maternal-effect loci by statistical cross-fostering.;citation_volume=191;citation_year=2012;citation_pmid=22377636;citation_doi=10.1534/genetics.111.136440" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. R. Wray;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Prediction of individual genetic risk to disease from genome-wide association studies.;citation_volume=17;citation_year=2007;citation_pmid=17785532;citation_doi=10.1101/gr.6665407" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Xiao;citation_author=J. Li;citation_author=L. Yuan;citation_author=S. D. Tanksley;citation_title=Dominance is the major genetic basis of heterosis in rice as revealed by QTL analysis using molecular markers.;citation_pages=745-754;citation_volume=140;citation_year=1995;citation_issue=2;citation_pmid=7498751" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=T. A. Manolio;citation_author=L. R. Pasquale;citation_author=E. Boerwinkle;citation_author=N. Caporaso;citation_title=Genome partitioning of genetic variation for complex traits using common SNPs.;citation_pages=519-525;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21552263;citation_doi=10.1038/ng.823" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Zeng;citation_author=A. Toosi;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_author=D. J. Garrick;citation_title=Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action.;citation_pages=11;citation_volume=45;citation_year=2013;citation_pmid=23621868;citation_doi=10.1186/1297-9686-45-11" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="twitter:description" content="Traditionally, exploration of genetic variance in humans, plants, and livestock species has been limited mostly to the use of additive effects estimated using pedigree data. However, with the development of dense panels of single-nucleotide polymorphisms (SNPs), the exploration of genetic variation of complex traits is moving from quantifying the resemblance between family members to the dissection of genetic variation at individual loci. With SNPs, we were able to quantify the contribution of additive, dominance, and imprinting variance to the total genetic variance by using a SNP regression method. The method was validated in simulated data and applied to three traits (number of teats, backfat, and lifetime daily gain) in three purebred pig populations. In simulated data, the estimates of additive, dominance, and imprinting variance were very close to the simulated values. In real data, dominance effects account for a substantial proportion of the total genetic variance (up to 44%) for these traits in these populations. The contribution of imprinting to the total phenotypic variance of the evaluated traits was relatively small (1–3%). Our results indicate a strong relationship between additive variance explained per chromosome and chromosome length, which has been described previously for other traits in other species. We also show that a similar linear relationship exists for dominance and imprinting variance. These novel results improve our understanding of the genetic architecture of the evaluated traits and shows promise to apply the SNP regression method to other traits and species, including human diseases." />
<meta name="og-title" property="og:title" content="Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/12/2629" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Traditionally, exploration of genetic variance in humans, plants, and livestock species has been limited mostly to the use of additive effects estimated using pedigree data. However, with the development of dense panels of single-nucleotide polymorphisms (SNPs), the exploration of genetic variation of complex traits is moving from quantifying the resemblance between family members to the dissection of genetic variation at individual loci. With SNPs, we were able to quantify the contribution of additive, dominance, and imprinting variance to the total genetic variance by using a SNP regression method. The method was validated in simulated data and applied to three traits (number of teats, backfat, and lifetime daily gain) in three purebred pig populations. In simulated data, the estimates of additive, dominance, and imprinting variance were very close to the simulated values. In real data, dominance effects account for a substantial proportion of the total genetic variance (up to 44%) for these traits in these populations. The contribution of imprinting to the total phenotypic variance of the evaluated traits was relatively small (1–3%). Our results indicate a strong relationship between additive variance explained per chromosome and chromosome length, which has been described previously for other traits in other species. We also show that a similar linear relationship exists for dominance and imprinting variance. These novel results improve our understanding of the genetic architecture of the evaluated traits and shows promise to apply the SNP regression method to other traits and species, including human diseases." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
    <title>Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KMUtP8kb6V3MCRLJquc9xMII5XDZh853-3isev-wvCg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460204","apath":"\/ggg\/5\/12\/2629.atom","pisa":"ggg;5\/12\/2629","ac":{"\/ggg\/5\/12\/2629.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/12\/2629.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2629"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2629"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/12\/2629":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/aIPvqI6CCz\\\\\\\/2JPvivquYjpUdHz\\\\\\\/mgETsBRn3owYQDz0rpElAu3u5dzBrj+mLfyQ6HjDSXKjVhTVj3mG0oZHyHU6ZxscU4ZTFbXCjsYSc99dn9FO7WZyIbJLJI7UjhrHLb7wcDQ2zlJXwbrBb++6ivA1C9qRfQQcTKy+YHKdZc18\\\\\\\/iaEWc9j0V1Di5oyKV8o86XQ1E8iWWxKys7rV+XvQpgl6Ekb7LrwRaz8L7lTY+0Fjlkb8ZYKlE+0U6XeQZD7GffeUoYtMdNPYbbj4Zgbwv6m53b+\\\\\\\/ZLbsC+uSLxdqVKN62qywZtFTdgHiKj9baia3fVWnTx1KbBhxV6pRUC7nTctn9xs8DiXQ72acSA5rYvvLAiqTR78jwcK9DXJg1Vx6Vvlxf\\\\\\\/2R54FIJGrIC0iBaoL0qreEGFJJXAA425LAnRC3pweSiVdyWcUTS5JbaQFydW2h9DlHdKyozP1iyybjjNvYJfehdmsIEEHex6QfVW0Vzsgs9dHfAR\\\\\\\/Ldnog55Yt6DlyUs6vlYwjxEOhPwRgs8zwK6UfP4yWHp7w+gDz45vTDluXM+uNF3A4Kq8KshWJ3q4lgHwVRWqAsWWCdJG1m4ViCVM5dakRb4LJ2u+\\\\\\\/8P+N8Hwgg4cJuU+Ui4y\\\\\\\/9wQWXMrbHW3CtLN4xYtbUNXvsllhm1bY4WA8zzcNy5mwk2zqoDXXjrDbEcx85Ar6HQPu8qKSleN1tDajvv+LB0P3cCROhhdL2aaWRqOYEsU22ytgq59WwzHVmgBgjzgyDEui5JhNukbJdgeGDmCM1b88+a1qrUej6y9s9cb8Y6BTRjNzGpIoinLkDmHT3+XbuLlPNNQ32PcvMpVfWkSlHStDs9Y5dChmeeY4Kx2VGxklSCQ9RL4dzN778WVw2cN3b\\\\\\\/Fp37Uq4S8Qj\\\\\\\/9dLaeMMoiwc2gk0CY700RE66we12JQz1Z+2nFN53t4jaEQzwsiQFWzlA9CmlAaeke+0yoj\\\\\\\/PUcP79L\\\\\\\/LdNDwZCKLISp3Qx8ahxxupKXsscJygjzOFZwbh5redNo6+0Zlry2v729cBApkxNuuil0DxkvydXQVq\\\\\\\/Ep9Vg+PrgGil13x7fe2yHstNGgyIhdthWiVn9OKwz3nyUuDlVrErGAEGnVtg3DlaXk6VB8jHT4C8gB\\\\\\\/7Hd2HRFSmazbZ35uq+KVItTm8HnCnIMGS4msGyf+3V0iiX1A67WoVbL7YpfpIfo0TvtegWWvguaarDuSEZ8OKwzlXU3RlcLLtfkR0qXcJ8yiHNZqcA0IYw6EmYWen33SFk8g5c+b\\\u0022,\\\u0022iv\\\u0022:\\\u0022m83AoOAfCYvAE6\\\\\\\/GBGqdwg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d6692b3d91326f4d75c6bb79a08917fafd5f509316e26fdaa9847fe69e21588\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R1oXAiooyRdlZyqlHy1kNtfONPWfnlh9wcbPEQj6K+tVXnYmmQAILW4CQdIxiy778iX6o3KpM52gjFXY3DppVS4t1wyEyXBAvCLge\\\\\\\/A\\\\\\\/X6WujlzNWqoTA4nWALhY2Y23LU1DhdVgOo7ZQm4X\\\\\\\/ohovCjgoVIUfgHzUBr4E7RlG\\\\\\\/KRF4z9H6bjawMg3qtHaswig5Hk1HWgQbzVgr+yAo+E4i6gJpj9K2mfYHgXODxxKuWKj6epVmIYa3r6m5lJMze7P8Ue37XREj7qdNlkv6kYfGZshTbWN7kgTvbGk9SaxZQXL3aSx8frjnFInPjPKKR34OTSwGKdfITDDVf14nBA+Utcew6BCrlE5w2FM+m6VR5ZOlfwlD3wJWk200ik8Jkn5t8k0TV6T04Kkq3GujJ61qdYV0UhkKPkJDO3g6z6LY6Qt2yAis+LmA3UoDuVYlnthbcx+WcPKd5zjTqKP4qcS1DRR26pIP0opIvt2NeXbj+lAtrun1l4vIwihw7fqnELcG2e5ftabT7BfDVzB5bmEXlrnQyDk6B1WnT4cV1wXp6o9CSJK32\\\\\\\/IKL5DpbFWWVMAnVQGH2vBIMQVdT9sJXJgJGnGW1IWpnnnLTDH3PAY\\\\\\\/AenABtlIIAT2oDIoOb92eXXQxVhIchd7L+77E3R8QsOdy\\\\\\\/RK2te1SVYBbrBkzDNNe0rIzCGfkzWbp4NkRP1iVeLw03s7Jb1ap41XHbv9GrvMd2ywG6HraWt6WFOaakXZGqsfiTDz8oFt7PHyNCaqBHR6LmGWQxooEIFcKBRKa4Kgt\\\\\\\/6Fj4s0mHskmOqVqSrD8p9Gyj41NpKc6OWcTrkQ82NZul2yDxWrzICPFf5OteI6NcJSAndjWttVX9t9B5DMwqhby7Gsrtz7Q8fwUB+SvizJiWE+WO0Q0Ka++mzP\\\\\\\/QtWdNf+bs\\\\\\\/J6ddabiW69RGDMqLzE8rx458mKVUqTIcBbK6huoKlUvgWy1k+UJyJGX\\\\\\\/k9iQlVGjwvoY26dMsdmvBpVPVTD7IoWtuQRDvPUT128Rd6OlwZ0ODH1az+zwk3bORk+2CYwtIQZdfXlIwAVZuVQISrMDPcO9Nfz11WCLRnyTKzsepBKkVaei+qBQ56FqXYfIUdw3S6Yfee4Zn2rH\\\\\\\/956sKya+ifhGqpe+rbEvi2aDmZac\\\\\\\/A5RkqVFKvnv0C3tgq0w05WdnGEWvtlX9fqY\\\\\\\/NSZX\\\\\\\/Y1cAff3CMf9wmnRGutTunrFR3N8a23j5\\\\\\\/Wb9OI7mRTRx+YjkDnucF5e8wp4HvljXzwjSh5BEU3RkWO7eUKSFnhny1tED6\\\\\\\/jr8gJwzkwWzDzFNLu1j6WszAm2YwO4pCQZTI6kblz9FW2moBL3VCi46G65lKyBGKp+8uDnvSNqjVbwqHB9pIy+36uJxkP2vFIf+pPUJUudJHMJkHnZQXGivWOQiIfHBopWefgXPYU1miIY1pQ5mCycRERfzeCB8AYOk3vYuUTL\\\\\\\/dbw0fm3tY2mpZFpV0lASyN0C+ROwvfYq2WputQc7VjNGXrJmjgp+YQudSMb8dpxYp8yfcJ5EzqcFz\\\\\\\/FB4ULXCaUu4tED4QFNFuM9BZKwvvyrUbZNv\\\\\\\/4QChTSFXE+Lxj3XwoKQ6Ax2gZdpfOJqRmhTRP5yTTzseH3UjmXfsRG1s1Ym7e75uo\\\\\\\/aru8VIZrpdIw\\\\\\\/dvH1JAFM3aZjvKpBsByFWNA4yYW+BYjP3t\\\\\\\/a0Gq1YeUvzqWlPBERgvOt5+nnok4X0OSVOPrLN8km2DlUSOcB1yCrP7mEOMs1fxqLRABlFJywwly7ZKM9s1npMcq7n9hfRPz8YBN4zYOnoDz9RqpxiYjRQO+ImzNtQFTZJroMiYxmGLwT0PPe0xlz+T9a6dm+JVqW2KkxlMlqcXBNjyBkJtIE+SBf8BHwsaT\\\\\\\/SassinLFnHYDS1HtKpHZNqo+9FxZXrM80Vd4nL5GzzAIYpfSqveRwxOIIu2efOW4ARqA8u\\\\\\\/L+FUtl\\\\\\\/MKka3u04Qnb0X7ypsrDLFkbqqgzvizm\\\\\\\/i62GNuPk28DMnqCnW21G2Wm6VFQJXcQmpkzOtdGsLBhJxa6gkhuYbgNwBcB2ihyaP2a2vT+gtydANoI5wcggq7\\\\\\\/Kngg4uRg8GnkoGOTEogiTEx2oLXyRKo8vemejmboEWE9DJutz87qRo4KTUBX0bM5VzTbKa2cyPou\\\\\\\/yiYg0RIpPDXlRRbhRveomZl9aQipMxThdr5BuxbuhiG3mNbq9ohSpSnGYEXrnjeem5XO5LFH05tWvufimBLvmeTlYEGSRPnxqPHiGzRQKR2BLGFK3YxoCAdY5TU4Ciyhr3NT876qsIsK5FKHKUA0vaK9Lzgpf1Tj3uIS1aHSCRLq8rZwS3QkNFeX7RpjckdE9qPw5vZbH3pNrzJwUlrC65uJjauhq6qLVZ+ILLQb2W71F5mmT0NcvDgZt58xWyTfyaZ2aOSaF9SgxKDHUMjtkejdsyiz9eAxr1E\\\\\\\/8fu514meE9GWgdp2CXh59vKzMRtTL6i1sn13LqCYwZaEKfRlhUsqSd+N7tEZ3BEMLFHj8tqr7SHrVoDW7hpfzX1t12VQD\\\\\\\/GrHA+ai8JCzqW4SEow3oTINhQ6L5rhmwrylYuHBm\\\\\\\/dItgvVVgLd3unOJtgG39I2gEu2Kiwcub2wchp8Tub1Y+cGHCd4Rw\\\\\\\/u6f7GnD6ZsM1nhsfLO4BXdPVhRwAmZONw1wnOyIPMBw5Yje8M\\\\\\\/5Gc0VK8JDUzrNXoYFBFR\\\\\\\/HpQfh\\\\\\\/jB2yJmFaOcsqTGccHih+mnvAzdxWMGgB1BL8wbZc4hq3yNtJebjVFKb46tvuzfqS4owKf8ENSSF8+\\\\\\\/D7vqdWRFpUApUwjQdc6QEUexf7jxzl7gHcQO\\\\\\\/5QUEcOye3tNhyXBz6mt0d5OFJ+CR6z1OXrOuLS\\\\\\\/S5d2zv6HzHB\\\\\\\/0govknMVXw1k0EunstgXONkOeNZ+ylgPg9aK0TTG9HyZKV1+KOav3fr9NGWNjT8L8m7X0j3dTNX1\\\\\\\/qnh+C8fz9WTGjj8hFTrF2usw130LdOwyFocSi6YVo30HLQ1mJoU9o1gmAqZPrQDzoJDHZHPbi5RdXgrco0iYT4HGYOBnaXb86fxzZBi4vobUbZFqsVPCQTgZHZ8yplmUzLiyngHenlOTyi2oKLhYkSPOx4KtkbhqZ6HA+UWnzKgpXHza6m+PxTUz1b0RMJTHG+8ZbuftIGSOlUzplJoXNrxyil5JNRGeiphcvFUYKqP+7Dh\\\\\\\/hU2XvQ9JkbSVj\\\\\\\/83vUVRhXClsPRZdVPkXoYgJ3Zd2YePoiotvdkoJ6\\\\\\\/aAiwmRCrDijJfSGiIyDiossqavPvoNftzbR1KTL4HA7nLwOlMOm19vl1kE8aNBkzTZwtmSc+bMxEWRAkRW4ZvAdjkHx3kw\\\\\\\/lPGpY1uB1Doch3tR33FPzZQ53EFa072OM6PxlZAj9QQCR2sVrt+nHcEyaZXQwlwAQNXgwYg2assF4x17O4CyKLfiEajHIkK2\\\\\\\/roKlDtjtax2XqNYaQzgaEVIHSWt9t7Pk83I49fFqAaY7QhaS0Jmk+ocAf5qHhMAaJW9qq8stjbuicNOMM0u\\\\\\\/GS6ZQsoYJwyiB\\\\\\\/h08ZWCM1h5uv175l4vC+O4FHP1ah1S0PqRHsRT12lhPEPydvg22kwAe0QiFmjbzel4QJriLJBllbc5QaDq3NdSOuhbv5dDKIrMENOVYm714b6qehhK7ms0pUH0L+M6eWvx5GFOMDCF3FeHmXneS53p\\\\\\\/JByEJ9KvlkObs3x\\\\\\\/Ko0IwN709UqBTajKfl\\\\\\\/fp+R2YdHh8WW8ieiHO7+TeCOhMZH3bX2LrUpz4erh0LKGE6AveY08Z5e+FvPtQm9Iy+4KCFOwTchz+sLa9trTgw\\\\\\\/rZh3cZ0Hs896g6OARGe1ROgtA9LGg4w3hBpXEo2NUShoCnLTxwvJZ\\\\\\\/hl+YF\\\\\\\/3vve4j4jM\\\\\\\/n59AwzoNa135yzYCVmwLfZXjCEg1VdlnuqAV8bRFeGzJazQPZAcEtnYShIPthned6Ke6\\\\\\\/tC7ePO0b8Kkk7VIG\\\\\\\/r3X7KGyq\\\\\\\/oyMHy26WfoF33vZX01UpxEXkyqlWNe3vZITv0VcwrxRc2oteNE3jGjIwDVbQV3yVcy3nycIpfZj38EQEtDsiYM3OT63mjGcuDzzPorkmgF0nYrV8QnCkYVGfHj6Y9e8GJM21i2Yz9g900TmYYemQVplytYmOq65uyNQsAyceDo+HrzmMBSIuErPpLDx2+eujRU0DmGICmXXL6gMRw479dspUrRykC4nMKdfE\\\\\\\/OvOVR+irnZqgjqZxXdJQIEILr\\\\\\\/I3dzpYi0MmN\\\\\\\/lrxM1prh0jwoynXU37Scz0lWQhnRxBM9+F+xN8hIQ5vDKPlBtyxQ2KJjYoA70oXtMea6gwLLGsawqvHHoGPNp2SeaPAto9SGs04QENNQ1J1NOFX4wI95EsUZmsSqF1Ekx7T7f3sQ55Bl7p4rBUf6MjMXV6HIcEiWZa2xwZ4Xjec3Llo24J73lML7viq3b\\\\\\\/3CQP2YNzM9tHc5PtVs+uxGe2C23pyrvEYXc3286sCQH7nT3jGS2lz6Vpce+lBFPCfrlfRuxaYGLjsWqipGbtc3jFWNjtRGSG\\\\\\\/flG3segQNGJ9d58O5y2M2kxyux6KD7vX2iMSKj4A4M6GWpaaIa8hezzdIO8SQBewg+IVJy7S9iQcjkhfqE1sdqg0fVW6Uma7uomuLwwL4LynCLpAug7vqQxB5JE\\\\\\\/wNBJw+g1PZgUTXZW3DLRykSIUpq81Kyw\\\\\\\/kFfnWRTiB+TVpIuKu1pzFiFeAchwFVVV1Cn0HbMl9ds1e6Vv29hMQSpud6rRZVTTvelEPFY0vScfhoxlaXaUvCqf1p+QY2lsX8wBr6emJRz25oMd3R320mG0vn+qPZXzNb79FaFpa6enxsdFslDkCmT5Y0jE3N4Ussfhv0ymHqJhtv4duzhw2uc1IZH7q2ozOiZIML1ZwNuyNSGDEoR+EGyrhS7x+jEURifhAONqD+DEcqrwjcjEEvSNmkfiIFDyCyw4t+B+jPyeZTyd+SXY7Rq+uYaXX9suTlv7\\\\\\\/7msYS5nnXomjMKURL3dPKF96\\\\\\\/ow25DcKGP5USb7DRV7snjD3H+SwSraOrG5ThcaLjeEYb8+cEPQt6lri3Q7mDpE+tr05opFptWQA+zhQxzIXX92ClUT+0JlK+3YhGjolTGMJGfJ0q7\\\\\\\/Ggv8CL1rlprM8vYDThA35YgdFTssoBJEHHFK9mAv0axwAMzBqhhrHfjid5jZjyKchb8WYDRkuqayYiFNdYFhxKrJ5L8NfTonSfodkgC\\\\\\\/4\\\\\\\/VMDBDdZP3NsTjvoZWbGTgKHpeY7u3QlyilcjXlKDZkQWrluA8zcoB36u6ydwe8YFgwflQSa0Ge1s0nyv2vSIpB3+AcNU+q9uckemrcu0gwfHOGsWfAMQ3+oQ69qp1TFXRiEjwXARLrVPcIWW4ZTFCKW88VjvXwpwijSscCVYDU0+vHYvscbF2hB45kNFQgX6+OYjaMOjgeGruI4Hobtnb\\\\\\\/sQmF4UxDI0bpWps1v\\\\\\\/hoyNETb9BBxQyEG\\\\\\\/nTU7QauFI5tGVcby9mY41KmmSK5iwPoCF5gy+Oxv337yrZKSMG5yhYmBm+YwC6qjYam8n0xNSUtbuaQY7VONw8aH3W5RbglUFfXbWBzM8KqMZLzCbeIrkktb49Kw0eytkH5RxYjaSGDleVdOaY\\\\\\\/hHSe0f3moqUfvxF\\\\\\\/FZhmLXQptf0NN2QyExUELGgAvpZOaJJmZzl2SoH5tITjMI2v+ybiJelehRDXCVSFPIhOmX2j8pQcMJuBCyYwfqMdepYtYqmzAy4jfMCabGZ8QP56UKnh1hNjvJA9AgYhdPXgdLU3x72tQMpBnTY+Ldcl63UwjoxSZrhWL9\\\\\\\/mX\\\\\\\/\\\\\\\/yAPXCtMM5tjrUQjqdYINg5lxw0W2jr1vwbPWGZxZzOGRHcpbGhqWV5Q98daTSnff42XjIEmgQWjnYvUatk3ol23Xo8zIAgE1qhLnXY0Sn1BhIWhySs\\\\\\\/LfGslXku8ScXRs16fV6EqrcdWV7l95z7S9MloIhQWL6nsGMl1uYqdj9\\\\\\\/ZFM9lrvtM5oLSeJuKiT273OCu2RwpOyR0LwGWfEgcKIOy8HjsbSxX5gNRH35ANr9QZltLH7NFxQ54+\\\\\\\/D2svCzQxozqCTgBQ\\\\\\\/XatUrpVN9rAwpijRz14pm\\\\\\\/zeyqwdVAkFXTG\\\\\\\/h+PqW6u+lDs2UfY1V3zwfqcTzsFREIB6dL2N+6iKO2J+2SjJEpa9W8cjT2X5pOV2o4nA9Q5ovMW\\\\\\\/y9cYdY2V7ZOIf1cgCkSZs0h8eUSs8iG1B9s748OLhZ3sbhA5C4rQBJ3sZbg0jZt8j0K6rgR4hafhar5cTBIipFR+2Tsg5RFc8fR3LE66UcTZIF5IB96YTdmxTkC31fOI9mSNnfzknbHBI7MF3fKHCThYBiCuvqaE1iybZX6CkBXXiH9wBP\\\\\\\/hOteNUGMzHVAS90jwUT0EjkWzNpjxhsa\\\\\\\/E44Fis7y8JsNmowxYn2jDh4zfAM4lbjE5w+QaCgeiExjriNMiHy6y4\\\\\\\/gtweg1hzSF94wqda9HnwX7gn1DETRCAlXCxMK7\\\\\\\/iJZlDE4b7ROncmm0aVRBuxQ+knfqSnRtxoeyri\\\\\\\/mzIHTudNWWrQBI8FjCXKjw\\\\\\\/wZjDQJOGsMFjy1a\\\\\\\/\\\\\\\/omLxvhsjudbhsklqQL1PVfSliUGYfx9gb\\\\\\\/1g7BANkoanRdJ3z6kf2NkLuy8U+fwkvWmebo3BEMzlKFe3gCyakiMgMVwQo0Jq1X+g9xNpA6j+XYIkmcMeUO2ueR5EZ05J8QDS4wZ2XOXssqaT\\\\\\\/uZt5EbigmimhaVRf8Od+MzNJBCzzQveHHadY73VN6hJ6igA4cXTAsKgaXFP58af\\\\\\\/zF69GMsgYzwrCxjTOy66nuC\\\\\\\/jfBchGDWnS1uZK8x8NoGmeIHArO6Z6yaqtW5Q+E5wZ5znKhqMdOO2OqQzpNDEJ2DaweWumC3sLc07Tsz8H5ppfSAbkcjTSsHhxmBedr\\\\\\\/5k4oztiWgLPmtGT34IHbxLxnfswIqF7tecYFKZLnNHbKEamJbWrDnfY4RFt3rKDzPcU592tIqg5u9cTzijSjMpe3xSn8oc41z7rY3qnLEOArHYibyspc\\\\\\\/b6Kn5p0iNtjGaGODjoLJ+Fyh2stIjeBoS6cXPXhfaheRjYPH6E3RdqL6az06F7POUNFHd7l0jPTpYHotftxQeRwGFmcNA8Ln3NVisIcioHc1zDUGyr1spmP8n8DK7kxA1js1DrTY9gnx35yL6fNQOyIHiwIFnhtj\\\\\\\/OHpESxaI7dWIBxT4gbh4PWCiKIEgzRVSNS9GjqotaVIOk+JkcmH27PA4TKdNi\\\\\\\/f3gqpqxmgJRXsfo9L0nx06V20zkNtAoeBUGS9zwlQ7xKLzSnlNgReeXO\\\\\\\/auTrFLV+GQTtflpUwuOzU07GA0QeuXKTZN0Gw3wuZfZ\\\\\\\/kjhgTnq4ehatYqvk5AR+hV5WH9jmZbP8VSFUPO3edBD1F1cgirN75rH7mecWrG6R69eabv8OlEW4hqES2ziR\\\\\\\/awtb1YGaPFm26anG5KYn\\\\\\\/u9cP5PCeRqRhI4Gx6xFwhLrxc+Z35LzN3mbipNKFEnOUCjfwn2mBYwSOXBMl9oZGvMxImqZqUqEO5Yx4EvuP4+DgJeyL\\\\\\\/G5vrQZtVr5dcYdLkgejpJRvgsB4EhYrwnJTYGNCPD1lrkKY6TMIqN7uSbMIyjj6T4v271efypukPT3cFek5+lo\\\\\\\/eMNl2VPgadn5EAJlUXLZj9SkgkR9QPpYThqgIkLAt+cOJXRFd8qjY+z5Yt9pyDMsm54ixtWLMReOd8xsUIkrTzmRfc0HdY6h+XLP6bcWrBprYsYDJuGXhkSxkbtVQORGCtR5qrXe8knKi4sp92sTj8\\\\\\\/vSUJzEvvlfy4SsNMBaSIf53QwuX9F9LBxCjyEv7ZNPTMiDGNsBdilWDUN8MOYuUAntZkX\\\\\\\/pKEzPQ2Nulg56Ww+0KgWHV2e301Kg0inhKl54pz8h6RAqy45z55dMW3iQeQjYPN6Yv\\\\\\\/wbSy+y16mYfdWwVFVgwTHiqKAZLrievl5Epcyipu8eS1bT901phgXdtbugYzJs\\\\\\\/2vsYS\\\\\\\/UsElGqI1nqqeJdTBO57mMTR\\\\\\\/xKJGdkEN1iEU8mChlfzWy1rue6F9ROJdLzJeEKIeoGhH+nHxypLrnEfvEfekdtIuZKrFLj2mtxQnwPrx4pHv5TT3HHphfu8r0SsHZkRk1uTOff97aDONpMJelxiy3NKvEBxp645V6QWfKX1z1TaeTO0wYwCeS77IGDUI03W7beWVchHOM+XHq\\\\\\\/1wEYqdFay45cFQmYgCeCuFP7GRupL5WONWwQBqDDWQIfP3HVgJUr077jIUTZ+9hEm086JirlR1lwulDATNjkd+ynbs4NQ3czarLCQALuUYJ2IIAU1aCZZBxVUFraUh2AalTANiylMu8yHeSMGIINWgh1EDX1z1VsF5sQ7cNkBpYEe0ncStioSER3DFhdmqHDIvzyRwM6HFkM\\\\\\\/aIfEmkzkzJemjv8Ge45Er+NfgxXeg489uTd9kFVZNZVYl+FMjhBGnu6fLMZgfSXJ8vnx9Fkd7Pj2U1q9O\\\\\\\/lPJ70hC9hHxXPLwGuIAob6xkbbfHaWg+wBwy3+cXXufCrMWnelg57qW6DCuiviMsIjAWta3JZifhGRtQXMEf0lnYu119OewDseasADxwrAegntOSfHI15NJPi+PFvn5NcCg3DxLuj6SzkibGKK1r2BS4U1xlbfvE\\\\\\\/WNZ+RoeuL1xjjKMYMGRmLM\\\\\\\/y8Q1lGtksE0CXAalrGoaiD8uzhBS9Eizg04LBIi8Qw8HDUBQMp5UwDEPt272XXgyM4fpNYWAa3PzBMtulA3CYxTRbcS4AyuG3uYfz6cp5y++ojmwkdYnjfHGuCbye\\\\\\\/\\\\\\\/\\\\\\\/DNqAnQ8aJdvqAL4crY1Zc+hy9wzYD5ea7Qd4zSkTC+aiuMJYexpALcgXiPKq9srhNF\\\\\\\/dA1VpdwTrhkuLkCOojj0dTXRbPHBIKBrStSjArplQQb4pYMwPubPZggWpc0EJszPWKO6P0QTqMnPtAL+7MSQt2EUTqBo8V37G+HP1KMOtQRtIfAzRden40Y9NblMlkdHFldFO1uPiwChCXlQJb4e+EW7oLiuZ4fGenRbFPkHqgrmT+cT1mAaITqg7y38Wj+gk8SWLxCFekpuOsdPlXkKWd436HGHnjphrHPCtmNnZtVtamn+MwKE4KFneKqDwKAXS7hdzyp1mvz5NrMphnQqtns\\\\\\\/XvR7iaPHkg1OBP5Uo8Wo5X5Q8VQUSYwGQq+VWjhp5HnS844DL+AnQAN1Dl6lkRxgaJIFJQctQgqalTU9ayiaAKcBwjITEO99XiQvrVtvRgwL+ocJKLIL7okms7BqtcOshTGZYGNYLTD\\\\\\\/BJH2Nm4PxWAE3pVFJfvuWqf73UDJZcav29uAjm70Jw5nKbCK0Z+9jTKI+T+xDQjxpjHb\\\\\\\/t0y+Dtbb75WxFcBbqjBVFnlcCB8dvr+8ALpg75kfR2e++ZoG2Ly7iSC5GeGMlVhACyjIcPMY4MgqR38mR1OttOuHMb4bxYQO7q7awhnJUdgeWtdTXZLwpsNP8gDLmjVTaY06VHvzfYR83e7qUXnoAMM6fs6x8VszeW2MaHosdTFIIrGdrOaO\\\\\\\/phMHsgjrlxEQBxMOtznB1kGhL2Sm66Fpec4t9QkdshOh2PkDnehFicJe0Hno3T6WOWEb1CEIkpqAc8FCxEZpPt5o\\\\\\\/40JO32BUGAGwv46ygjuFUPhU7RRyv1nL9jm0YZ9r3k7zUiimQRuVSJoTEuWxihtrYPfKF7Hj1Azrm2ypKd76FbiYZimZh2na7mm\\\\\\\/FO91VLm8WVXRM481rPlHJh55oWid\\\\\\\/QXuUPixxaHw\\\\\\\/uV5C92wYJJUk3NLw8\\\\\\\/LyJx\\\\\\\/omsa+ixzQLqTXdjSM7yBAPhOW2YkHo3rKM1auDIwZm9DQWmKglSiIZA78M3S6WZRSxgnYexRd6prslhUsJsdZkc4slzJvDnZD0L70iEJSBunsRZ2evLK\\\\\\\/yHH9KxwSeKy5awUQzJvgqZA\\\\\\\/HD5HfrZfpSiZU\\\\\\\/GsnVjZr6MKuQr9GkFTXFujtqYSkEMNv3LmpeQSHZXu77EwyFIo3hCfQwk++qkHR0xgDxVmcHxtReSlcIPm1T7D2zgKNCEokerXm4Pm8QV12nRYceXsI7UUcpygXZSkYEz3awJYXjIRvt99aefo9eH6JGfsafKuJIF6uIvuVsoaJGUbNJz8sBTcdFeEmtfSuuKEic+988UPWbTQZVCelRNvmFCwdqP0jLIi4rmDXOEP50G8P1fK8rLprZJfdYDryCGHVeOmgSh60Zz3cl812QG3dIumK3iMja5guwZb873\\\\\\\/89v4DDObcm573yesJmQOpQHcv4b3itlOU2VgzZQ62FW8B23wEG+r17Q9zGIxaSoV4AM5AkHXrxkb3f1d0T8M1V0KjMvw+RxyLM9Hs79ig7\\\\\\\/Hn6p2SXdj2wUj6fJ4w+3z0RihxRm0MC8C0CGxGttvZqEc91mfCdK5pa1iXpf6QHXjCHcSbdgsa4UVGYc1UJhn5mT51YlN0\\\\\\\/vcBZX9NnUzU4XRU238nwwN2CgmqzhCggiIa4w+QKm3oIXwZUfne8kJYSMgxFasrlow+sCiSd9lw3LZDucX0avD4qy8nwmo64kiC847MZQ71Jxpi7swoONJCzD6ClX6tmpTQLQJDoYfAjNmljIvgqGPD43160ni517RifI64snSggZpIg8tU8XYife1IyR+mn1iQ\\\\\\\/\\\\\\\/Qmlq1drhRxIxrNyRONTGFKzr0boAeb5bXJ4l6j0htSChiiE6tEvC5fU0PUosCxYxrM6jjYcdu69fia9adkErOhsDvcNlhIIp4VIX1+DCWalq9ATLpmCr3pJzHwlnMrcMyocGC8NYPHdg\\\\\\\/JlcY6wqo7sPNna1Phk+9MKO18WRR2tpzODqEOaPdOM8vx+dYDmvbzy2xLSUl5VReLAtSTRXt30uS0SiCdE3UF7XRju\\\\\\\/gqLcMmcP8VqcFSydAjXCbr4xmh9tepar2XZjzYIjDjGYimxw94a1TUHvds\\\\\\\/\\\\\\\/VthRzfKXXJvo\\\\\\\/l3Ju1wEziSBw3o10t8DaFyVXIoHRwUGE7n7CRxzw5C2YcY1nm8+E9Yx6zlA9TYYlIP9uekWbOxCJm2SAvuvzvsMP7oVIcw+c8\\\\\\\/zjwVUJnTiTD2S52tmLGsqYA031xRbqr0hM7CGx4sijqQkzEue8wy4yMuhz\\\\\\\/XUtqrWOfYzfcqrzRq8S7EXjQDwmGXxS2C8ZGfNxsYa12zmrppFR4dwDLDqNHVRAt4GAERrv8Fv7G\\\\\\\/LqxfAHwHhcMz8BJ+jdMWDikp+AwblT82hMRJ\\\\\\\/nI8eqE9besGQvpJzhZoNkMLKauDlb12DL38gwlcskFVKd3iSE8SThqFdhw67UcfJLAI0M9j\\\\\\\/q7585ff5uVNElVCQOeU+eOiRY9EXL8ltbe7lkosfv9WMUULU6D+l0U8d64\\\\\\\/02k0LhwExhnAYnT0G7zqtDK+oHTmqUMJDzTp9yjic9ZMLW6KFRo4uNpGqkpcQ1YrHr7k05g8BRHUVe2PheRlq2Jq9H1y+HdYoNtJ9E\\\\\\\/yEX1NjMa+YJYym+bXFaLiX2MBcXBj6TKhNssZkosYhNNW4lDqhvmXgavdZkuaPtOEfZuFNXEnH+Va6NgaY1dZClqQc5DfzvAaqTcdORyoVD+TFpxveNAPNuKsNwyO4FJ9r+d49OPqmRCG9Zzkbck0HuKkcsvQC4TXL\\\\\\\/Yh\\\\\\\/5R9SVinii3cdVKUNQk+g\\\\\\\/0doOUVoxx1ezSNyMCm3AN36c+rCN3YYz02CKDaPwzJyhhoiwSYOicUEFW0YlvS9xQupBnOfyEHcYSEUjzCRwvlvazwqfz1nzHf1HHGkVbfeKwGZTaYsMAUEY3KIjh005YQKe0+FgbadX6cZXk7fvY8q5FwQb6klUMk92kTVMPU7nVjknHnvtkY5hcey+3zC25PEVt75hwv0rpW7OMBiIw+rtM9LAFDvZbMnICJwpJOlTFfY0L4yVPk41AWnRxjaG7kpfGZwqV2YqPQG9YpSocYGn6uyE85cpIv01kRVMsii\\\\\\\/JUjDg2QiEc\\\\\\\/np95DGlGwTgwObY2Ymluo7dJajzuzJa8zrRbwHeqxzvaAV\\\\\\\/Ezqx9GMGadOSIdkxjYgVJ59P+PscoeUjpZKCER9s3WfcFVxBCyDnzRTKQAhRsQRuZTu0W7qYiNJGPEwOJZzrw0u\\\\\\\/ONWuPhRr+VCx2q81dN+bxzYY0NboM9dFkWnWu0yvSIP3JdvDT+eK2gU843gyJVElm2uodZ+uIVeO007JaS8Ev0UM+1AAU5PGntd0e1kcWG7dVcod73x\\\\\\\/hsxLNkr0oL1NNh27ilMrNShO17ROArmM3BM23jlmfGwojTrI9nwmqiN3fHtyBjLM3Z4vVHiQleyPBoJrH7vgMpJgpOMJFKzLsPA\\\\\\\/b1oLBxWeqV20uexMy+kJi2BETOMbPimCAWcZ1qIqxsaxGaz1ZmNp0Fz0VJQ5kn9uFjcg4z6ZCKmPEi\\\\\\\/7aXsYABWddcsRHmpkp01jeNrsjUXMWkma50XLfSzbJXu2XpldyK604xKGV1yanZ5QCWKnAHhIuKqZjUzvDsY+TaVREU3UDe4TkvAIHMGIz7gMgJq1fKeyIqBNvqCA9mXc6rWrbTQRB0iAUH6EBMSQUmn4CZJlE9TFC\\\\\\\/bdisS3QBnvi+kfc9T7tBvXi7O1bDu8\\\\\\\/slGnOdzLnJ5rGd+3h0o0jc7zYqgsMDyapZX7\\\\\\\/RZy89T0xaziv\\\\\\\/yy3Afau4PN8iUFEEBqSgRSAASfbyhmN1sn+F7p72r9XXHaCYyUvQVMJS2ULQcXH1L0JXReGSHIZdKH3UUVBvCOH0xYgeXZUnC9a35FHVMa43cx84vHlXoTlhsOBfpSFz0\\\\\\\/I6odH1EKmWk3aLrYgtArDFNnUOb2KSMSvGaO\\\\\\\/5bw81TximYxs3pJmXpKJhrwy7qwC8pE9XxzccCSnFRbW5p3N3ZuOuP+mJuqgajeuNECXMOPxyWTB4Z2Ej4q36cfZtycI0tZ8BX\\\\\\\/nBmRSyltaALOwkxcY3xzGl\\\\\\\/TDxtr1cGLn7mI1lMKtMJ2snoeg2ukjRixxyIb2d2kOrO2bNjpfjPEMbLEI7wuJfpBeIZB6GxOkvSoTMrTJcjbxQ6edNAWfc7DRn3Nvh6gO\\\\\\\/LE4hx2qRktGUJsComOr4g0AAGQSJUUvBfjVl1VUrRo8rw5+hWyJiM1h7FORtX8ZVj8wxwdD+8YZSstMx++318IIftbWGMBUnTEDFbhgdTXW1JCQcf\\\\\\\/KNdb6u+xdq6TIrhvtZ7sib1a9thFQUn8GsrBBO7\\\\\\\/QDZk7t+j7StHh6t+hulvApZJsJMES9a0S7G\\\\\\\/AW9SOACFK7dNPgW\\\\\\\/pALhrgDTAKnuoekamWHmMbjPs2OMz3EyJHXnJtyF2lX83aOcmJRvg92UOME0fTWkQp9JC5cgybcRM8cX5ItwFFzWY5ac5S\\\\\\\/cnc0TX7o3w6xw2Xs2Lzq7u9gZenPwduJvITLM2CL4aFqkAUUPC4wljcAWTvrUqY+SC5EZGcO4T0VKOVzQojMEJCQDf1dLqPfCQM4jQ85TFDThyrMCyeMgfVRyJHJf9bchpKhs85wpvlnni0Ru8LATWse0O46hzU\\\\\\\/rj6gJKb8d9GncW13G5G3yrVQnjob\\\\\\\/6rVFLmKA6K6+I6R8l1ZWLyNRcUcSS9pJLfOO9tiAoeLibGj\\\\\\\/YJwCAuXQXquBtzIr0EGdcNFc9EIdy5t71Db2Q\\\\\\\/1n58knMlOfRx7WGBX\\\\\\\/lxTPid+skOA7T+lbNlaQvQSeOgE+biPp5l\\\\\\\/IAfqsLNNK+XbAnf4n6xJ\\\\\\\/GGS1NhHK1mZlsZXx2UwaNVFeBnEP7n0RnsWCRnd6AmzVwvDdNCKGc\\\\\\\/La7HWJt\\\\\\\/\\\\\\\/f+RQ19jBrgcs2ZEWccz4EwhpCYIsdZq1+STjvylrfMZ2pxdaxyqWgSfDIAf69BOzNpA\\\\\\\/f32aJ7\\\\\\\/ICPLs073VUuN9HW\\\\\\\/HvUZS6JT+VG1ZcTUQ5qpz0TjympxedyBCdRxl8jYXZ6bev097sX3dZ78GiB50qRKf1LIKzGnpw2w1CDUIfFIblC8zVWlMCxQPAPUQ7so\\\\\\\/tdqsedHwwswIpOP0RbikRGBgJXBFoFxE+bUBGJO2z++nRJCCW\\\\\\\/JJNI+ZiUdnzq7yuEflfC801sEcWr6VtQc5Gu5WZ6SpCs9c\\\\\\\/b1Suqj4yBcz+U3+IyKY92yVsP2KYLxPnB65\\\\\\\/FKly\\\\\\\/CmtywPajwT8bwUXcoLa8tenMBHmSNAYKgtw9H\\\\\\\/R\\\\\\\/qgdwCLGHddcGKPvbWRo3HHykgitfizl0Jh9Zo8eh16iwyfmWbKJmS5X3pgNu2nRPda3X5dactWWPvAeuBEgXAM88XaO79vmek3WS2CD3M5mB+ugYdyxZG2LpRyMrXMJZuTLN+fJ9keV8Ppnq944f4uS+zQwvXmBBBwQL8uKHa3lOOklGWTp0BClCcYLKec10EAmeC8VJUOi8xTaBKk+HUslRoJ9VVuGwqbMxyvXnvS5WK\\\\\\\/yhMeP\\\\\\\/Xy\\\\\\\/HjUgSPp4xEQ2D5pAFPodqYFWbF5ahenNmfFFMRP79jc37nLyxhHGjSydCyUAX0YLoBF3liZpbqGWhJoRvdjonfOCQ2fLio\\\\\\\/U8sqHmu50uqHT7zP1qzik4rlqn2peK41SjTQSH7EB12xE4zef+tX2DGmWHDAS0ddNs9r00kYp8jvss6udg6SfBlOhU2LU5A7cod1Nz3bR08AOSdReiw4O4rqWeLI34yWnvTrMzDn89izqH1xsn707u7\\\\\\\/ceNnTAZdGSMsVokcpY45hj0juVZSoE76rydRNnvvlWOkTFW15TWbqjtsHYcPrygRrovAKGvKnxyYdwqbHZ5cVfNuGBTWppHq7ZoPSpfyaemPe3T4PwlqU0KLikUNWWKGqMMC2MU1lSGryg40NI\\\\\\\/o6vB3RRoWDiep\\\\\\\/pAGJT9FzPWXbi1Vx+dccPWefg4vM5+MrCPDEzKsg6WlEPYd1pkST7DnbO9sB3OlBWUFUBeazaowvSGpZiclMTYfbF7+4Hzz022j8UCYYhEDW6P\\\\\\\/JMN2ovzMUcb6LaveJhj7UcZ4m+dzATCK5m1kO8a9h\\\\\\\/+xXOm1D1RDeiMNMxbUNYteTmH78osfPYkY2kgtHLgFvgvqkerEFmWpQVVO4K0MKx\\\\\\\/JNDO2YTIXSErPGQyzWhTJxIjsacJf13VRBEQbgOERP751V2ZRrv8Xw6IJ05GhRe1Dtgm4ULwFfwH+fFdrJS7MAodjWQSGZT5Tinwp5hVGhyAF4zephE0EefLq3ENynVKFV0W7FsWXQ7qPsISirowHFXXpFE3OSiZ+gEE+Wnbnd96OiSoGRUr1SP27LH17ybs7nQoAxvjsiUxbGa5q8T1+7fAB8JzkTuxrDjF8eX7Q1s\\\\\\\/2t5KiVx8GNVeOaAOSqGnNCoPdbQqqCi18W8p9hdPWqJRsG8nIDL+GrESrDUv3ve5v0Re9ncvP1EISqoMdhATkLyloOH9yr0KLN+dC6\\\\\\\/TsJsp52VgDqUf25thEC2B6SK\\\\\\\/BW59HTH1W7lO7nc0ruwlT6WRQB+7ZxUem1wUhe5IpWKqJTXeyQjViXvwP9fR4cMWunnHV7eTmw24BxPkRpx7mAwHvIWzZ4KxrkrJt+NTcoXFgjyioebX1hfncx8bwPZoDKJywAeQ5IDH3eVikipXsTeh+GxQXhwa0Qn4PJ\\\\\\\/E4DhTUv1\\\\\\\/g3OUc5IzSHRQSmXcmvd\\\\\\\/+\\\\\\\/OU9rcPY111iKwxx9fhA8GCPOmngCdpBQcvOsn311dpo9dZA+qsG99ne\\\\\\\/1Zu8xk3fQ6rr0dh3gOVQnO5m2zLBKn6VR9Zg+Xa5z0TKUnyBekRwUtesodsSX1ZJwCJATE60g4IdDCM2\\\\\\\/wlQChLiQ1p7iO\\\\\\\/AvQVknDY6ESiljsDFt5m9zgOcDhNr\\\\\\\/it5i4V1Kh\\\\\\\/5Uo0a9UY3YqboHj+Wk1inaCyAHdIvIJN8NO5Ni4dNL6jcDWmhZsC1HS9CfmkV2JGiFNbz2qfhDFg\\\\\\\/kdam9FKPCq7uzKbmGWA6+VC9xjaXhCzglgrlrty+AHrtDEGKjhiheOvuy+kkgGGhxk1P59NEQC0M8FJRuxC2rtaNOUhqq4jKDRY+H3VKqlYxqb7t80rXaOdDMVJrH24CL2J2W+9JyiSt8UAHDmfuSRr3LPmNA5Qf8IImV6FyahFQsQF9YCqwLrmlVs3XsO+Nnp+tPhFfgsuMTgMM+XF\\\\\\\/K\\\\\\\/UKENL9+vlpwYN3fkjom\\\\\\\/yVEZ81nYakY8xIA3SdvRoBjIb80GWxUrItvOaImmPflQkgyl22FVdF2Rqtq55t\\\\\\\/E2bN78Q23f4iMKBsfGMbx6jbG4F8wq5kfrwoDT7SbFN8aA4b2XcuGIymmk8KT8xCeM3AWd06oYXD3xwltZAQnceMnv8YdY+jUB+nb+7QcDMUuGHIZLVk6frDQtfdqc6MfTv0tXJrvvLO049\\\\\\\/J5TvrMxwluKN4yRKf+pyKlknl+0dajzlsdWp+qY7INHKyKRi3fquq0fen6ijAETr0Yba4E\\\\\\\/wX+1\\\\\\\/q5iMeYW4sRkGhFbhMrm1xo2o9JY\\\\\\\/ZZYnbgx42Dgq\\\\\\\/KgVzqgyAF8SHtaf+88g4\\\\\\\/g6NZnNkhP80TW0Y+nW3fLq4BhzDW+jtUVYpiWlIUYWC49XevxKIkHpu2zSOqlfqBu54hRb5KpOcpuNQhNyOpHphAveMNFEMgPD07T14i6jGEYj4GvsA0MjhaqAXmNEj5Zxj7ZGAaYdQFP5kHCq+jBXSgtxH4tCP9AhOUK0Utw0+nUrBBYlZIizn7nI4HGbbTAwFSVmErwpSZxKGSEBMSA5vYmyqBlmHIQA8M2MPKbO\\\\\\\/\\\\\\\/o11qoQRnkBRNZyj05a3TL2AFwvwWyUP28cifKk7pHGBUxK4dDIyZSZrbF7agE51f2xiYXaE19ic9R\\\\\\\/35ZxJnCBgx8I8Cc3c4CYs6VOOwYOCFa1YynjaSIyfIuJU1lWvT3\\\\\\\/pRwKIlTwDvTq8g2K4RPBc8lBp+RTofHyS\\\\\\\/up8vcS3UDhnZ9dsQnHUKm4o4FukwW+2AYiF7aXSCoMWkehtRHJQznjksWRVfqVO1iSe9EccrFjprh6ub3fBkpwnNo2tI46RrahEu9\\\\\\\/WGcbnl4GJ7cxFT1A35emO11oKZrZpQ4d7uHZKBF7PjKNa8IF\\\\\\\/e8ATV39xHV\\\\\\\/+xLO9k2a1zY2mdPatrsx2bJB3WZvdxwFjjR+RPRhtBeqpDDap8M0261wqPkjTUqNv9URuOoJ4u7kFa0u4tpZa1q9\\\\\\\/PPL3Km+Gpmmuzq7N3G0DrnwH\\\\\\\/YTwSgtAKgnWb0MKRiH7oRcJD97Us52XFQVZYEmtfp3LrkHkz931BnUDzcZQrBKimEm9hRnZIQwIJSyG9US1rNsTOSt19v65UAvfIBOpcwAAOsdhvo0Py3XfqFzjDnFzn4hh9rDKSCMbtK6r2UCzQ9f4i72ilgKOjkjRFU+9BbR4RSO\\\\\\\/lISZfZjXMWePQheQVVYxhZKVDAgHAQpWS9AE0zptEPJ4TC6NOyldwbrlaWWSlJGVmyAPwv277BQvXPZ72FHZR7+JO9CvQ2xziWWgJh4BEHQgQjbSIipv8jdX50a8oIibkM\\\\\\\/L21W5FVxMkscMRj8Cn1rZcZ8rdF50XJxs1Znqe1232fr+1Fpv2sGiBvnIgOcX\\\\\\\/c8HA68kn+4LKEKhYhpkeH3FlNsSo423jCv\\\\\\\/TY3tSqVfe\\\\\\\/LzQi+k3YvjahdwP6UcONSuCKE\\\\\\\/WQGXqM32zG7Er5Bn\\\\\\\/sVv0qiEKxXq2AJBbWIi9tLFCJnLt6SGTEab8qN3tqv4Bm+XF\\\\\\\/b1cUp4NIG5B7sk0p+hKbE4T0ud2G1J5rpsAqzhy\\\\\\\/EvO2qwofj0LyMCLyv6uh93Z0HeJP+KtvozD99KhACvM+t45KYs5wt96gT91eqtzcezN850vJuX3vkeaSWguacEcypY09Q3P+C8ypVipypJA9om3dc38notrc2HffPZRDOVCN6BB+Ix+aDLe5oBacHdKBvk+yDEzwdzUfgZDNtcbDrcPUvBt0cOIQC60cM7EU5Ix6oUDoTVe6H7XmhO7KPert1FQo+EesBAo\\\\\\\/fvQUsSbbcVPj\\\\\\\/hR0ye8g\\\\\\\/jvP7DjdxQ+idQ\\\\\\\/D3l24X2zh9jV9sjMwrjQb\\\\\\\/hw1P+BIO3R9l5vy7NxvvO0l+cwJv0eGCKqJWF2EAQT8UBnMadLCBhYuXS+xNnk+Gd86VwlOUsEt7oDSn\\\\\\\/ryLRQamSwOVsVuFfOlTT3jt3snyrASzIkA4Yo1ZgL8h8yTuXX9Cul8mPUSnFDkrM64etlfvCwtfPi+aL5fpRHiFS0FbaDz5xtbEd3gaABcsmU2Y9rPnpkj4OFrK5I5WrdYM6pcti1W1rq\\\\\\\/8BL3K+GetnGNlUTjG29pPttzHyt1I9C6PP13cu8HJ5SVmvechyqJSphPT6i\\\\\\\/XqGohUyBtF5\\\\\\\/z6\\\\\\\/p3RwCI4B7FkzZwM2RT6Vw7RIbCeL+3tsTnwzavQMwU6YAbB3E\\\\\\\/4XoUs+cgHQZkFxKu9h622RBuZr1fcET8JLEZWkN1OTh09n5RFqMCndHejDcuFagwV27cMbZTltdMufI8Mn1giNd4ytLx2PbODiTizkaOwiQh51Wtj5flPTCUpd8TmqqXBuhD4z356IYv9ITqjSNQhbKmW1XvEJ0cFohHJCnswNFngcl106pX6TXte9DRK6vTz9MmRHlK61kSd3N8dObCHS2X\\\\\\\/8KRLf5PhV+0cdIUPGapneK2SRBDrn6xBE8S2m7qzug32LStR84swJfQjdDG+a5s31vGDMgfclbQD8NC00rKNTkKlEgSfm2aJ6n2D\\\\\\\/bRA5qUBVxYiWe9xyqxII143mm6NwnqLsUeKcC4mL7aBxDs0bv4CLU718L8OtyOQTs2STO31KzwfVilYxE5CdrJJh3w1nBrlbXVM3PUQ1quG5eWobJfdnCAvmccWEDa9xASbxQXvjrZSx733SZ6lW3WF77Wtz36TuKmszzAlsfNisu8W5s8APTkr3MKjktCdXf4CkhS8gPORSbeY5Yc38E+KAjlvSGmFp2wJIWkkquA7BHT8TvBqwJelcLQGeKAKwQiT3\\\\\\\/t2zwvRLp1Vedag9eAEW9td+\\\\\\\/ioiQcYzeTYkqWsIVg\\\\\\\/2M09Ja+6EnNDGDcKUqtk+dxsadhrUlgemBZ8guy5J6RhySmKbMjUjIW00qtTBJDoLhF5oNOKpk0F1E1XF2VRQ2V6bFBXchYLVfDJ\\\\\\\/fWiKFErLA\\\\\\\/BTxAy2o\\\\\\\/tXN6Ieod5iEZEQpH7pF6dEdU7ppDPXdnIKGwjIIQXJ5wYCkxXdqUd2svq4lylBsX3zolvUZ93ySOwMbRgI13E9klGC6aOtpFssrw6F6UET7ewZOxKTUGk782m8bafSkP3nGyQbS64QH0cq7lPIC3BG0Y+y6Wz2448rWKyzmJLn9cceSqdxS1CKUH2vUB+LwFGr8YOLFGo6ZLJjLC91+4Hdo92dgsnchNQs1j7ZvuVKT1MDhgdvB9eufguA9\\\\\\\/G9Zt95i\\\\\\\/U+056lhhteNVZwPpAHr8BQY+LhWWM1wT4X+SuzpZc0XbstDCMF4FWY4wdfp9e8BRonSwGJXO8zcnvnccYtg6sjd0MI4H1HSJiEB7sV8bCFMKRDdIKbmKFpjbt2ACC+PpVrQsr5QAp3rb8mhe7eRr\\\\\\\/Z7fSLFyYAB7ZRG+OR3PNvu7HClJk50tF+pi2FT6wZPiGmJOlLwUqLQFtrx1E1NKrui5wHTk4FTJtX7piS7roMcA40RqVaa76QV5oxA1Lzzxh7EvfHM8C\\\\\\\/sUGWpKGBnixVbDDFbTZRCG5v7rs8kcPG5j+6IKtnHyGd1Til6PlsSqF9TR609pmhBRBExUbR3rh7jqf8bbLo3JZRbFsBXD0Zpg4ck5La7+knQ7mSoEAwq1HRKrgTzON6gNCa4N7q5+u6zG5bPIdLsK+230QQ4rhtoaq0IUHmiLHz5rq5so7Ew8\\\\\\\/AVLMB9vQEH9KRe3gml9ehAhWjzozeM7tIB\\\\\\\/vyBOOv1nlN3EpwmnOI+jzPSkxw\\\\\\\/I+xiGtz4pi0IoK3x9ROnf+sZSZVflbvKQxYQd0VaSbkdpQw9HN+u8epC142yg7M61ARFqq2dfFeOGyyRwkx5ym7ivXYOYDFGzPJbFvfkbWMTYeXiu4C6QtSrvIOAawdO3QXZU91ZHKHtG1rLEGGhPKPYiQa7hBaCUt6kOHPUqDeUT\\\\\\\/3PBBNTgObHvYIdSneb7c6eq+GNX5DIeWGlTqiru0ODvk2t6s0DpCp2rWfhbK6\\\\\\\/GWRxqOAK3bvsqirrDD0Neh9lhZtNSTWNdjJ\\\\\\\/hFWMhoDwoWHVMTv2d2HIYilJagUWi3sLWHU7H4VtbsVvOhc+\\\\\\\/PzWafBzeen0FiGVTqyRs2QBgsFAUL5s6WdK3O6WxQBBReI\\\\\\\/brkLLeBickMz9Xh3MZA7gm1tm6RTE6zCOey4+qKoQQpEWS9aDcHc4e4QmJLO2IjJY2\\\\\\\/8WNGsT3PO1GjWIv+j4OJl3hW+nEgwUU9ZqfMquSkzCxLZCGJ79mn8ZVBiAtV+RxxLBlHV8w2DDjzIrrjYVcNsCJgvpoSlu1mJbI1J2Bii81veIVrFV1KOIGT13GzV85iqi7RcznemLPoqvKwx4XArn8aIfsb6CpDmuFyIGJ2\\\\\\\/7SvGppXIP2w\\\\\\\/lhY1u8Q0d6XhFNVOvwMF8G\\\\\\\/PMnR9jv+Z2PYBz7N2mQUM2PwFcOofltyn66kfvikuXETmip4HVc4GLlVEny0OMsGTQ+Hbu1LCEfUkUJFzNIm7GA3d3Uh+uLWMJ2fcXkDe2g8HkrvHZ+HJrL4ZR7oZ8VNyfXMeO5vrkKlygrpVIyX5RxoRf\\\\\\\/gHhM9adE3rE9D6dt+HYgajSjAiEX39EEFCE0dTT2WIzEUDAiqbQpUW\\\\\\\/+47flVhhxAaoI0dhfi2UhQiGW4m9WYGe\\\\\\\/7hDf0ueCJu2TdFA\\\\\\\/H\\\\\\\/hjOvcxGO7Qmm6nffUP1WYmFq6DfVf7vtEUpJIOXDflsDFAli37I62q3HrND3cYDcMeORQoW76m5zd\\\\\\\/98prfkhdJeja5aRCInNRaMTj4i5hUXCbNC5utL7inKEWyzT\\\\\\\/W4cVhM+JdHhjlS5WuSgxeAumRVJeuoPLs+aORky+DLtOnu+5z0tlF5xbTl1gmG5kOtnfpTZGMcNtWtYTQRrWZrhufZ8OWMmNI1kTBULBFlw\\\\\\\/SEb3\\\\\\\/IDkRFOdEUvnkY9vMMKH9LWDN\\\\\\\/XsvZdD3fj5VHSXuJePV3nJuSKMvvKC9dFer8cuM0RnFYGBxtp1OQwNv9KuD9AHmJLRzWLzp6slRdcNkIwrfPgE6S6Z+DSgPJFoL46d08\\\\\\\/O3kVKQiBfpiRWzMQLHPyGQnoXAVJze8+SWiKL4d\\\\\\\/pYtkSUkO2RAgT26pFO6p7K2htIOfSgYaLlluqz8HB8aThNT2wtlWqqVDPoHrCZ5wTxlyhp++A4W2QgcrbIFwO74rHDLM20xaR6r7Qy6PcggQH56lkIv7MKz\\\\\\\/4I+XDvnKeqhsIjmE7fx7Jtj1jE4qEe+MjGGQx2b4i3Bfs6PaJ13pjhLNrzchz9xPC4tR3hzenxyiIb6x9g6BaPyCVAPpjoqlt7dZ27rWiYAgpW1xZEk3DH+fl1\\\\\\\/pAHONnpYCT6GuBwA9ly4fqRy0eV+PcRJb2UG+a2PFl\\\\\\\/cb+eTRfoMfgpUSo0Ml\\\\\\\/zGnxz+r78wk2\\\\\\\/XbZxyZafQPI\\\\\\\/+SjtTcogSqlmEUcWklBXebAIA0eQzmxySfXuaoVOJ617hEz9VBHfDKxaPVBGif5eA6InCVUrwDPCwJlE55lqaTcjebQEr2Qr7JN2nP4D+UN8WzF\\\\\\\/PZuMN0+Fe+9QBMy1BraNinOrHmxMs8Ku6Sr1104CEsbo1I0yKwHShG6UAl30h6lt9305C5drRVBrp96b0Q1OAFEUivQ1dIClJB9iqAVlPcvmXqg2oRzeJ72rSKf5yT1ZXqMaPAFO6luprm3VeMV5V98AgRAvLZm3EmADOMDFlSPHZ4Us3LRe0hEkEVT\\\\\\\/irOc\\\\\\\/FPn6mHRPKBspNxKYHcNrunWyN1npDQNMTAxTHtLWpdIyQqNmeDecS7A+P1CAZ9ycVCb1AOKXA2I8S5FNFrR4yP6hnESgTtaHzQ0bk4E4EMNZDf2ffBn1LTdSRMJEB4hJQnIWbv+eXOrzqeGgPKPiMR4897tOVILMlrd79pviDl29p1zigWJqGm9WjpjRhpmcy\\\\\\\/oG\\\\\\\/jqS2onXiiy4lPpKj68TLJOhxFwBqs4+ZVkW9KtZ4lod32jf2Z77eaDBgIPiBqS6y5NpYjSPDVA1OmCQ8vzJMRMCx2MZgjSWAcbvjjtJqiJ5VmPH\\\\\\\/zUTeTutK0KehgWImeHTspzWOt08eVgB93RnrN9Wakpu0U8+RhyiCHtdR4FPPRX\\\\\\\/7s\\\\\\\/OVQnfzCtgF\\\\\\\/Ja2dLzIREFBo\\\\\\\/04KBLPJ1M8TnKpLmCotLI4ZEvoxpRiV1hK7pYd3L5OyfrSvqmmeFSyZscWEvUXfEAbWJcAoUydp1BnKq47n1Z9J3+Mw3Tjn7aI1PDVVa6DSE\\\\\\\/uouXB2OxyGrY2KeJ46qTvp0d\\\\\\\/\\\\\\\/rRyof2cK1suep3a+HbJfnndTJWkBrydyrL+Qpvgyy2iMelU7zlOfP3cMjOojY6p6OXjUxcymjIgxI6B5AFGP3uWyc78UrDXGaWCdBCzw\\\\\\\/gn1zeZnzrNJgoWFkri4YTkU5Npd7Ihh1zHCNS8f\\\\\\\/zqebXbzzSyOzPHu4\\\\\\\/q35etRiZBMiU6nLxPRkEd4k0rE+LRq69K33w+Jsn6o\\\\\\\/592u2rLFmbRc9DrQvfBpNQvklTjYIdhhwGLLbLi1yLT+ZidvU68TyKysh2pP63u93HiVsvgHVrDdssPMRHiva\\\\\\\/HQm4q+AzhS8EnK2pc7CHeJGAOBPp3nw4EsQ9UHqC\\\\\\\/NnHErk2Ax0WxKQSY8SdQf7ewbkWUZP7l8WNN53CirC7rgnwRENKH9KX6nLVhhPDaSVBqBb4UIs6LbjMyVttKoRMleY18JN7hOnfRa4KlY87CebcXtGPvhZOgWAglolugLdI2cBSmNMHdLzJcKXbG3PyenMkpaXham1SIq75owi6N39XW7WKk9bxPoQOtlvo8YIH3rIJqTuoAOJgxRs3oNlLT8xGpsfov4dZC+dfj+RogeZAMN8Mgx5PQc3WnYwxx8y9XutIc6EHBkPz1iXVzl9muP8AfcEg6PnbaE5tL9UsMFOVuwduHC42Z9wX1sM+wXHq1MiPkJkyzb391gcBxHKEnwIPHbCrfUnfEsZNEpGN3k+A2bQ5M2evz\\\\\\\/9nJuR3Ifzql4LYSVZPYM\\\\\\\/lz9AddC3+w7EcAmci3rYBcKnaJ3PuWQeVuSa5VtyMa0GqSSDlf13lFresB7aP4bb4GzrVXvxIBoOyFWOjkcvO6K+1hnzQB+L6E60THiuLxt8g+WteNexfEB2BQxp5iBQ7X\\\\\\\/XWA4DpqOLsiV2mBUcl\\\\\\\/zSSwaLNiYAVNgW8TL1vY9uztW4Cu2h2pRPTs3wMouS4Nyx9qafEsqdJE5y7okpYCkRgro6OVjuXYgSodLqD1Y4QdIwPTn4h8VhDWJ6ZZj34uUjZ0Nyh\\\\\\\/YY7fY0A\\\\\\\/qYYupeWIvk+8NRaG0Fd1NFB6Ft1xeNlPYxlZQc07RvpsnhF3wmsqLAVS7OdfSK65UV3SRTdK\\\\\\\/2UBkXDtaAyryTC6wmyeafA1afvTf\\\\\\\/Kmg\\\\\\\/VlTL4ibBqsJ0S8F7x4rTqQZFMxg720eL0M9\\\\\\\/IMlUjRuPb+htCBA3PnijRMq+iLQ1fYtp5rDYcaAfxgZaqwFR5i0Q+UvWe+IkNBv0aGRx+IDUSaOE8ew\\\\\\\/bi22JB\\\\\\\/ZloIRF7KrL6lz\\\\\\\/d3qK\\\\\\\/tJNQPK0ygawNnKlkI8ooAOQlK7+vbRFkIOvca861\\\\\\\/\\\\\\\/Tr6nDyD0sXJxeq0+Klm7UZiPobZj8\\\\\\\/JkhYD9OathLO9iwqoowYkZ3E01oVmVYItxhnTI2DvEZRD8MLL2yTU8N2B0b54Mxpa1y4e\\\\\\\/Iw\\\\\\\/UAqhU027nEMGJWgUz94GmwbUh3L7dBjXR74bo0PVXtrY4AQJvMaH5HVEIohqVfyH+ngoenbQkltxUnEe4dV7J0pvEMRtnOw1A8zLxDrZMnww6cYRZBtxqeajMzFuOMuBDub+nZPYrFvClHdXvhtnWTiAX7Zb\\\\\\\/zh0pSfaj5puLik+14CIDf9uKC0echIVXJfwnHUhVAwkJN1At7ZHYonHPrUsw7nwRZgH+CkhJ2nP8V8+R4paKYFos2zXDM1m8qUaZ6Dpv2Qa3SQ2J8MgB8im6\\\\\\\/u01OWghKCXgr\\\\\\\/2wKy\\\\\\\/\\\\\\\/y27F0emjjYa\\\\\\\/3BRRdtcq8yjCNQVmRcqlMzO0rwF3NfKUdFt3X3GSELDJRByZYAisHHRqJxWVm+zAnh3bSYWU1D2QMHHZ3F\\\\\\\/KziwXyARakgCqniMe8Evwz+cmCUA7D1TYMuwPuu1TWMs0mFLyHLvmfotSwAxsXrDWZf7fXDlGp5kC33NPKliWiou2hg6oliTD4SVddocSf71Oh3KcKZ1JQze0glLVPqeZ72Krd55R69CGm0GsYKvPYmDC1zPcoyfUqu0XG0uS82utMcZeJclSekL3TGNDl40ybJzURwrwMrx3GyKNiwhRPuMc7OK+aPeZOuL2IQeFzVhK36wDDKZy0FSr413L9uyGE347CP44HsVR7XI4Wi+4tryUgPhrKh9eVbCZf89cmTLWoIeXsJhpuopYThnm0z8w02Ghm1lbdvcsjC2GMIghvm4hDrekcLLERJerfo6+5VW4ea5J49\\\\\\\/wtrREYe2j1SSdZ1vw9unG7zrEUXbmh3SRoSN2ybeXxfl4hnA8P1hY+uJYOtHgnrCNOH0xK6LYIeBs34gUZZgUNA\\\\\\\/ruRuRM\\\\\\\/jC3NKPtvd44xuzWA72I4dGwVnDfEZSwrhNZJZwZHlzY\\\\\\\/LLT3\\\\\\\/zmrZ67O4cTSDXa6ablFfCknYnnR5u8Bq6ptWXLDCgFPkxzTLONbRz1T5dV4GEo\\\\\\\/JsnLgWaFatOt4AnBn9ZfXXE7QAdxX8xFCdeGddF29yaHKrIUY\\\\\\\/iMWihlMikljD79eqbSQTCUUAljP1ineesUNo9GBmN2mNOh0gi05RQgkh1VIEaUgcD4TwUHvkHgXeGPkJ3x1sDbW0J2g9FwZL3xIIzdSU9vXIhcX7DqiyR0Wk1nnrydc18b22uehgxqhhtELK+ngCF\\\\\\\/lb2NzoG38gRFG18EYPhoRXmfZXpTOC5Dx8hAxwFYSj5Kp+jNooZqX8to+1ra9gtIv7O8YK3t7ZfvZ8SQLimWughC9jrPqIg7IWkjwhI84PxIs+gDIRUSD352VMGV9eZLtZVH1b9Slbu2DxGQpRqUparVmb0\\\\\\\/prYTSQgs\\\\\\\/iKbh4WI+De9DXn79uXEV0btv91hHhSbijNZSCIyvWLDy8qcBAwSG9tC\\\\\\\/DTfvRRUuJ0Sv9\\\\\\\/gwp21QfGTUgGZ+7WNNzZdi390Tl3HyEmNclZF+A+MoBO7sYE0bIkG2s+FUJjZcOXKWUEvMD41a4OzLxVsE5tnMvoPYSxa35FH1zckX5\\\\\\\/ptUE6AZ4B0P2YPYzkNRsyeZCw+SA+RGzJEjA3\\\\\\\/QzEf56fTCtyjODpXVCcgSKNHJop4R\\\\\\\/HRmHnGNnkaaV2\\\\\\\/qtRE1cIK21EQatOGW2RbOB+l8WdOKr5lxITo36A86VM5SVwCHTmZfPZfgMG\\\\\\\/ysTKbs\\\\\\\/S6dUZ1OI6O3ziJQeH0T4z6KgOn\\\\\\\/KDTOPP9+JFWwLalHCIMbJn2KxeegTzurC\\\\\\\/cCfL4jbPyOej0QoaQnR+NCYV14oRYczG6wI83xA8BGGTL2CkMiHSqcibUJWNI81tL4086ksnbJrxKTaWNtafcDIDo68YXcHvYWwSRsOGPxqSl+XJsrSdh4PchnDbl1UEiMXp0oVaFIwoy3W\\\\\\\/Xn77G22DOYWVq8OAS2XmdwMsr5gw3YhvVTpj\\\\\\\/Ar+EAE\\\\\\\/MKeERjoFvdJuETW6jMss3dVVXLKQnR8B5VfeCL4nHyNmAg9qwAQMX6RIVSYHk+DWv0IED93MSj\\\\\\\/A1QghOcmHv435HoCgMFYB8A27ESRcBuvkBzHljp\\\\\\\/JboWf+PabQNL8dM\\\\\\\/GzqJ48PGe8szYaHPh5YhUwg3PjKETXn\\\\\\\/d3nh5vf7yuMXIDZYFyUoDjJ7glJy0z\\\\\\\/nuovyZLPBYHksJZ\\\\\\\/LgsKkQ9JKqYZ0JHSOurY45lFtzcRvkntDVx8c9JcUVuN0JJ\\\\\\\/36+pcTbWeD4GDyZCfKBfg\\\\\\\/LDv4bOzQoSWbK3EN92DUKma6szCnUvJv7FvbwcvObXYq8g7BpkA5hoPAM4KbhaeDPpwdEh7e8HvOUf0P8FiY\\\\\\\/LcvtfOIKbPCnWsxLC8cd9Q0FrNG0sCDxxz+GnskrLvv50tcYHjsGd7xg4\\\\\\\/UOpmt2LujS7wJLLGSpThCY5GbuM\\\\\\\/qdWbLIsV25e+uUbPFbxGxvcf1UWoMO\\\\\\\/mpUj4Kg1EvYLcldqSqKSkdYDgpNndOqIwLnA3Q3YOy8I92Aa1ydNAairUr6dyem59PYQ\\\\\\\/6EUHPrhZczUVGMRZFc\\\\\\\/QTAk9LeZ3lUDFD0wZjUQJV1WzyVEnQVgfDnE6A+R6ZkuvPD5yjcSg+O31bS5Ka9tba7j7wqdbQbXiSm27gWPLXoaJKs7eP5CfY3JzFgpjYz9sCwunANawdEqo+MxrsJrEcOAHPUoKn9iqncDmuWCeEkXaT40GaiYMpDmZhKz7cJbKWwDwUuRzIyE52UufJ0hVS7\\\\\\\/x6kuosXvXRG3BkZzY25\\\\\\\/NIDMntl7b6BKZjss3wH4RWUuBpYur\\\\\\\/L4OIOH\\\\\\\/PsSuq1y9INVui3HyeF9+\\\\\\\/lAqVeHYPL8SMHZwKuvq1cIUE+I2jN7641T1Kfy4g8fprqDmWpszwk19bwUtllUdP5hzgeS8fbIHgJwaFWxddajX9GR0zfGJrubIu\\\\\\\/Ykm0Iqdo3rm35UBjxscVSTtmTcFWJ0UBIpDLLBA7EEKrA70J1KdH15wO+cv5fM7PbXomBnFUt3wnq4u+L604a9mZftIanHi6tRZDivtqCG+Oj8H++6QplSxiyOBufjx74CE63CJYq\\\\\\\/Fdjn1L3nTh2pryfUQhmHDJkwCmhsit8V+ET3YsMlDV4BnsTZ11XBX5WNKSn67kkKbU18B5t9yCTrKTddgNbZQDvpfW0vGdGw2kNbWs6R0+HqeaNmzjjF0VojHGq0zpmf2k7bl5oXmr4MQ1Kj2I4QS3YWzVun9kqRmAQc67CHBn0+hZ1q0RBUhvvSBBnhG\\\\\\\/Ncga9x8w\\\\\\\/iiAoxa+Ih28G43CBcKgfuQsz1kSnTcZjgE\\\\\\\/8YoQ4WzSTG8FiQ721GS711dakQHzVzY0spJLb7EIJL2ZQqMU5TblHFzMFl0hwd8LPBu0xyOKiKLPDo5zQW5nMoJ4cVmT2HYRxbn8AFSE3GTH\\\\\\\/bJG4kBNHvXp1KIW7sC5y1+CR4175gxJcMeAaDt9SaKvodq+hwi4AHmfvfgliUCbgexOTtAaIyEYZjwjkuQNs6OrbohcIdMCLeAiouGDIWcKouKRkjpb53cqD9nQeOtkR+0gCLr7KEaFg2fN7BwIz1g0KkRYuSFZCDlZxtpbEEqXdtcGhOe7Q1YjaEcasv4LlSGbdWmn+2Q3C7+PHMPYzeEh\\\\\\\/46H9STiHgnk2J1WJyZZp+QJV7sWbC5JRNRPfu6sgzbJbbkgBGS3HulgusoKgISlLWPtH5HZFixZRQPHt6C2illsvUsy6T2Vvf4U2Hy5qedJBvDahWCnEcr6esgWb9DcYN6I9ynY+onm9yUewxusSfUc0Ude2KTY8kkR3lgwEYMmlnnZQNJGW3+98VTYWsZ43ApEzpfYrapGYfAr0NBr2JrGp0ObtmIExQOgf+aMpbgwnhW0Zc4YPyAziCS6dTzphe3wffoXi2lkGtfW6O3pAe1P36wQmPDgTQMUzAWb7QmWUa2q3DezPbhKI7hEe4myN+Aurzp+RKMjSaPM\\\\\\\/hSqn0rWxxfzJGiX+IQ6kWk0+\\\\\\\/sQWd2u5lylj4lZ8j4kbCWPXzh7jTgBjnWFm6+ao8khWEMxy5sSOQxXAlL74FENPxJLTmemdJMUrI1rcleSnUTExWo\\\\\\\/EyfuBn+YQ1IWRh7QePjxu9oFr0DuOQpaIqk+4k+Xaa7gm0JwltV0ql3Qt8LuID19sm3RAcjE8nkihnM9pC7xcxLTE+yBIcSBHCzeQolXQkOsNnSyvS66ZCR2+dOa91KoEexrxzFiYmGyo4SIU\\\\\\\/u1xAXx97Swh3DxMozB6ZTlW6P4vKNn2lV9d+bX0zATdRNTAAIt5xp3DbLWLRj1pPNvg6IW\\\\\\\/TMNF4wRFuA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FRRqTGv+BYMTGR1tn9JpZA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022666469d4019dd7700908a086f058bfdd2744046019dac863376ed4b44eee30cd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022E2R24aCBQD4j\\\\\\\/f+F2u5xbGNrsx56VhRqJw0oEyk0I5tS19eCOaRU\\\\\\\/q0n\\\\\\\/ALea155ndSDqgmHnVbpDy6IiMeWKIedLA5kP1D\\\\\\\/l94dpaB1zzML9KPZ0dw4Qsrfbwc7IAdKY92UviqKfRQbRjNnORRE6K8vx2e2rwOTFrtR6gC4oIJ+nY33BSnM6c1Cm41MZvHLAVUA8Xxbl5f4u2AG9NRI9SujhwR0EgVZHjhdZnzKBjxKP+Pn2eTa8YldluAba\\\\\\\/bXHqm9ogzol5e+d+D8UIBRn63iLL7LxzgSUuwXZDeG5dPq0AG9T9nk8X07r41ZXURMzgPzq2FHyP4ABQZq\\\\\\\/o8Qj19Y3LAwotLg1Mv6l8+bqVqJuiJLB31qWBrvrh2RINC\\\\\\\/avMpy+njnt3grxd\\\\\\\/HDcnwhnH9frN1ECMH2iRVXKmYuX\\\\\\\/q\\\\\\\/97wV4D2AESzr9Sf24C9QJ0ZGPfWWgK62EkNohEBEetJM2F\\\\\\\/IjBMJKgdO53HBsz8+pXTcuXjHtbGBpCOi4GhKjd0Tg3IC12cUt92ctPNye1Vz564GE8uU7aAcfwYSV3ksbvznaICuCfVJiR1dvTt\\\\\\\/tM1puTbLgz+XwT\\\\\\\/VOrZHR0pvsEiS47OZLW0WZJpOvxzCLhu+ILLn+6D8K1qmykqIslOoIjs\\\\\\\/c+hivm7tRnS+MxXIKSkWDGUIg56UQMuv4la3XynHPJtDXbIzhGKR\\\\\\\/MaiCdsuaTln4crJ9P4jEdujfjQHg0G16FLEgE4uvrH7KYoLJy0SZQ\\\\\\\/pIcBZDzLhaCEWmAsm19iB\\\\\\\/8YugoFDVRWoWyV3rVMYvHGJs+mLM19q5ykAP0j4oQ5fRFareEyhJ15Ckg163wQyUAITX7dT1pzgZVV5sJ4i1WI8LZ2P+QTZ8xvcbuh0pEfRD2J7eIsjT+xrIu\\\\\\\/YNbXXQ6BH+HDrj8NWuVByKfm74kAmzpCKdsTlaVuZbMho\\\\\\\/rbLKIbddaQGG1pVYPRi+Lo69U2SQjrKF1g3LAibIvMmHeq2hNx9j3TUbtQSq7G8WBpTt5Qoo3t+RSwC8bZpAWNZH0d5dzyx0g0gJgEdWJCEw4jixWKHdMh\\\\\\\/EjokHeYfvUb80\\\\\\\/RVNyBfJWmV0yHMrjhcLOiArQMtr04MhK0y6rNz89TfZTIg9rgLjujuJZEjjz+WDA7oZmubP7ZH1eH7CdqVcACCNTFcd7yQUyVq7YnCt10QtE7MObGmJKZoU2Hy3FgN0WE9\\\\\\\/6MwTxhgcLSmfgCS75xA+RD2k0DXl+n0BKXRZxdI\\\\\\\/GGlkWv4sQ9ZLbMsjhKdzyhcpuXXlc3e7IClSyT9sn2XKQGXL5aK01oJ00Wxg3Js7o0Gf95jWx4f+3ehCveu2QOlPjd1DVyT0Nz7TUrSWjdZ6EZj0Sv\\\\\\\/5dEi+KFGsodmHKyJWKS7FNKUlX8Q6WYSB1B1mEgxoAfsiQQpAxAqPXG\\\\\\\/sJs0Kw+WtD3NJ7W1iCIextrtC6CXOd0TEJer7clBnIhktO7C+e97WVEKsDOrTTX6K6HeWIPtu2KlRm1pWtYaCYZjB3\\\\\\\/7dEBdYx1ZMWYEcOOc\\\\\\\/ORg0Rkq1uBF+SKdu\\\\\\\/Pje0u45M7xCrj29\\\\\\\/rmA90EbbfU7Rd\\\\\\\/PHpnfC27\\\\\\\/UlgZ35NfEk2ELOmxoQdNjH+a\\\\\\\/WPiPcY9O3kpCZ2xM5ufOAXU03ha9aStf4aoc1UQxGKZrJh2DRqG+QIzXC4v1Qn\\\\\\\/Y4oGB7t7RyCX+1c9pmAd0Ugjo5DSzIL0YUJae5umrL1fVLE1vSaz6EyyzLGkDrOTvQR7yIkBHUl94Z8wOcsjVk7JguHkA1M2nvdC2vXwGGc7HRFClOqMNXG3Vm2e1CmO574FzX8OutPOJZYpjlvnmdcZ1Fp+tS+q8+ngoMh9sJ9KOj0aZgwtw7zSi2YOY47\\\\\\\/B9\\\\\\\/sGReKI2PEuIve5ykaKPhIOuv0\\\\\\\/I+jRHjq1+7tskKVaWm4lgFEkDAuKTnNBZEvoDyEbAph4t+Eg0yXZCWnePaW5dRWLi0nEptlCwkCRjOfw7\\\\\\\/lBQZ3OkXyPs0mBTlVa5qW2i89UJQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WvpDyZq1f8Mj+LKQVk\\\\\\\/zFQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002244125de40253aaeeaff8b493ce7ab4d83a569c7e9af5ebeb2bd0b405369bb404\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460204 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2629" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JOOj6qiiCydcHCdYpLLfqlq21YbPSTEMXmVLFQoFT1Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2629" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xS4BpMpdW8tzEtvfQZgt28sm4pKtolnSYcXT1dioCfw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2619" title="HES-Mediated Repression of Pten in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2639" title="Estimating Trait Heritability in Highly Fecund Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460204" id="node460204" data-pisa="ggg;5/12/2629" data-pisa-master="ggg;g3.115.019513" data-apath="/ggg/5/12/2629.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4715-853X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcos S. Lopes</span>, <span class="highwire-citation-author" data-delta="1">John W. M. Bastiaansen</span>, <span class="highwire-citation-author" data-delta="2">Luc Janss</span>, <span class="highwire-citation-author" data-delta="3">Egbert F. Knol</span> and <span class="highwire-citation-author" data-delta="4">Henk Bovenhuis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2629-2637;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019513 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460204" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcos S. Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BS.%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcos%2BS.%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4715-853X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcos S. Lopes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Marcos.Lopes@topigsnorsvin.com" class="" data-icon-position="" data-hide-link-title="0">Marcos.Lopes@topigsnorsvin.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John W. M. Bastiaansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BM.%2BBastiaansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastiaansen%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BW.%2BM.%2BBastiaansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luc Janss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%2BJanss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janss%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuc%2BJanss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Egbert F. Knol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Egbert%2BF.%2BKnol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knol%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEgbert%2BF.%2BKnol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Henk Bovenhuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henk%2BBovenhuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bovenhuis%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenk%2BBovenhuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/12/2629" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460204" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2629.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460204" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2629.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460204" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2629.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460204" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/12/2629.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460204" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Traditionally, exploration of genetic variance in humans, plants, and livestock species has been limited mostly to the use of additive effects estimated using pedigree data. However, with the development of dense panels of single-nucleotide polymorphisms (SNPs), the exploration of genetic variation of complex traits is moving from quantifying the resemblance between family members to the dissection of genetic variation at individual loci. With SNPs, we were able to quantify the contribution of additive, dominance, and imprinting variance to the total genetic variance by using a SNP regression method. The method was validated in simulated data and applied to three traits (number of teats, backfat, and lifetime daily gain) in three purebred pig populations. In simulated data, the estimates of additive, dominance, and imprinting variance were very close to the simulated values. In real data, dominance effects account for a substantial proportion of the total genetic variance (up to 44%) for these traits in these populations. The contribution of imprinting to the total phenotypic variance of the evaluated traits was relatively small (1–3%). Our results indicate a strong relationship between additive variance explained per chromosome and chromosome length, which has been described previously for other traits in other species. We also show that a similar linear relationship exists for dominance and imprinting variance. These novel results improve our understanding of the genetic architecture of the evaluated traits and shows promise to apply the SNP regression method to other traits and species, including human diseases.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/variance-component" class="hw-term hw-article-keyword hw-article-keyword-variance-component" rel="nofollow">variance component</a></li><li class="kwd"><a href="/keyword/phenotype-prediction" class="hw-term hw-article-keyword hw-article-keyword-phenotype-prediction" rel="nofollow">phenotype prediction</a></li><li class="kwd"><a href="/keyword/pigs" class="hw-term hw-article-keyword hw-article-keyword-pigs" rel="nofollow">pigs</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Traditionally, exploration of genetic variance in humans, plants, and livestock species has been limited mostly to the use of additive effects estimated using pedigree data. In this context, the role of genetics in complex traits has been quantified as heritability, <em>e.g.</em>, the proportion of the total phenotypic variance explained by additive genetic variance (Visscher and Goddard 2014). However, the estimation of heritability via the use of additive models does not only capture additive gene action but can potentially also capture part of the dominance effects and epistatic interactions (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falconer and Mackay 1996</a>). In addition, traditional additive models ignore imprinting effects, which also are expected to contribute to the genetic architecture and evolution of complex traits (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lawson <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2013</a>). Therefore, the proportion of phenotypic variation that is explained by all genetic effects and how much of the total genetic variation is actually due to additive effects is still unclear in modern genetics (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Vinkhuyzen <em>et al.</em> 2013</a>).</p><p id="p-4">One of the main limitations to better understanding the genetic architecture of complex traits is that typically the data structure does not allow simultaneous estimation of additive, dominance, and imprinting variance (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">De Vries <em>et al.</em> 1994</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> 2013</a>). Further, imprinting effects might be confounded with common litter or maternal effects (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wolf and Cheverud 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tier and Meyer 2004</a>). With the development of dense panels of single-nucleotide polymorphisms (SNPs), the exploration of genetic variation of complex traits is moving from the quantification of the resemblance between family members to the dissection of genetic variation at individual loci (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Vinkhuyzen <em>et al.</em> 2013</a>). Because these genetic effects can be estimated, simultaneously, we can now aim to quantify the contribution of additive, dominance, and imprinting variance to the total genetic variance.</p><p id="p-5">Dominance effects are of great interest for both plant and livestock breeding because dominance has been suggested as one of the genetic mechanisms explaining heterosis (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Visscher <em>et al.</em> 2000</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Xiao <em>et al.</em> 1995</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davenport 1908</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bruce 1910</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shull 1908</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Charlesworth and Willis 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shen <em>et al.</em> 2014</a>). Only recently, however, with the development of molecular genetics, attempts have been made to quantify and exploit the proportion of genetic variance due to dominance effects in plants (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>) and livestock (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Toro and Varona 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Su <em>et al.</em> 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Da <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nishio and Satoh 2014</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zeng <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sun <em>et al.</em> 2014</a>). Regarding imprinting, hundreds of imprinted genes (<em>e.g.</em>, <em>IGF</em>2, <em>DIO</em>3, and <em>NOEY</em>2) have been identified in mammals (<a href="http://geneimprint.com/site/genes-by-species">http://geneimprint.com/site/genes-by-species</a>); however, the fraction of the total genetic variance due to imprinting effects has not yet been investigated using genomic data.</p><p id="p-6">Moving beyond additive effects, <em>e.g.</em>, accounting for dominance and imprinting effects in addition to additive effects, may not only improve our understanding of the genetic architecture of complex traits, but also improve the prediction of phenotypes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lee <em>et al.</em> 2008</a>). This could be beneficial, for example, in predicting disease risk in humans (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wray <em>et al.</em> 2007</a>) or for establishing mating strategies in plant or animal breeding aimed at maximizing the phenotypic performance of the (crossbred) offspring (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Toro and Varona 2010</a>).</p><p id="p-7">The objective of this study was to estimate the contribution of additive, dominance, and imprinting effects to the total genetic variation with an SNP regression approach. The method was validated in simulated data and applied to three traits in three purebred pig populations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genomic data</h3><p id="p-8">A total of 2013 Landrace, 2402 Large White, and 1384 Pietrain animals were genotyped with the Illumina Porcine SNP60 Beadchip (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ramos <em>et al.</em> 2009</a>). SNPs with call rate &lt;0.95, minor allele frequency &lt;0.05, strong deviation from Hardy-Weinberg equilibrium (χ<sup>2</sup> &gt; 600), GenCall &lt; 0.15, unmapped SNPs, and SNPs located on sex chromosomes, according to the Sscrofa10.2 assembly of the reference genome (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Groenen <em>et al.</em> 2012</a>), were excluded from the data set. After quality control, 34,912 SNPs for Landrace, 36,578 SNPs for Large White, and 38,116 SNPs for Pietrain of the initial 64,232 SNPs were kept for phasing procedures. All animals had a frequency of missing genotypes &lt;0.05; therefore, no animals were excluded due to high frequency of missing genotypes.</p><p id="p-9">Phasing and imputation of missing genotypes were performed within each line via AlphaImpute (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2011</a>), which combines genomic and pedigree information to determine the parental origin of alleles. The pedigree depth used in this analysis was up to five generations (between genotyped animals).</p><p id="p-10">For each SNP of each individual, AlphaImpute (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hickey <em>et al.</em> 2011</a>) generates two probabilities, with P<sub>1</sub> being the probability that a specific allele was received from its father, say an allele G of a G/C SNP, and P<sub>2</sub> the probability that the same allele was received from its mother. Considering a heterozygous animal (GC) where the G allele was inherited, with certainty, from its father (and therefore a C allele from its mother), the probabilities would be P<sub>1</sub>= 1 and P<sub>2</sub>= 0. To obtain the regressors that allow the estimation of additive (<em>regA</em>), dominance (<em>regD</em>), and imprinting (<em>regI</em>) genetic variance, the following transformation of these probabilities was applied: <em>regA</em> = [(P<sub>1</sub> + P<sub>2</sub>) – 1], <em>regD</em> = (|P<sub>1</sub> – P<sub>2</sub>|), and <em>regI</em> = (P<sub>1</sub> – P<sub>2</sub>). Thus, the genotypes (GG, GC, CG, CC) were recoded as (−1, 0, 0, 1), (0, 1, 1, 0) and (0, −1, 1, 0) to evaluate additive, dominance and imprinting variances, respectively.</p><p id="p-11">To ensure accurate phasing, only animals that had both parents or at least one parent and one sib genotyped were used in further steps. As the result of these restrictions, 1538 Landrace, 1595 Large White, and 1272 Pietrain animals were available for the estimation of variance components.</p></div><div id="sec-3" class="subsection"><h3>Simulation</h3><p id="p-12">To verify whether our data structure and statistical model allow disentangling additive, dominance, and imprinting effects, we simulated for the Landrace population a trait with additive, dominance, and imprinting effects, with mean 0, total genetic variance equal to 0.30, and total phenotypic variance equal to 1. Real genotypes of this population were used in the simulation procedures. A total of 15 SNPs with minor allele frequency between 0.45 and 0.50 were selected randomly to have an effect on the trait [quantitative trait loci (QTL)]: five with only additive effects, five with only dominance effects, and five with only imprinting effects. These SNPs were located on different chromosomes that also were selected randomly. Each QTL accounted for 2% of the total phenotypic variance. Therefore, the additive, dominance, and imprinting heritabilities were 10% each. The genetic variance (V<sub>G</sub>) of a single QTL was defined as described by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Koning <em>et al.</em> (2002)</a>:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-1.gif"/></noscript></span></span>where V<sub>a</sub>, V<sub>d</sub>, and V<sub>i</sub> are, respectively, the additive, dominance, and imprinting variances and <em>a</em>, <em>d</em>, and <em>i</em> are, respectively, the additive, dominance, and imprinting effects of a given QTL with allele frequencies <em>p</em> and <em>q</em>. Because each QTL was simulated to either have an additive, dominance, or imprinting effect, <em>a</em> of each additive QTL, <em>d</em> of each dominant QTL and <em>i</em> of each imprinted QTL could be defined as: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-2.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-4.gif"/></noscript></span></span>. The simulated phenotype of the <em>j</em><sup>th</sup> animal then becomes:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-5.gif"/></noscript></span></span>where <em>n</em> is the number of QTL (SNPs) affecting the trait, <em>e</em> is a random environmental component sampled from a random distribution with variance equal to 0.70, and <em>regA</em>, <em>regD</em>, and <em>regI</em> are defined as described in the section <em>Genomic data</em>. We generated 10 replicates of this simulation, and SNPs that met the selection criteria were allowed to have an effect in only one of the replicates.</p></div><div id="sec-4" class="subsection"><h3>Phenotypes</h3><p id="p-13">The phenotypic data consisted of the traits number of teats, backfat, and lifetime daily gain, which corresponds to the average daily weight increase from birth to ∼120 kg. The response variables used to estimate the genetic variances were phenotypes preadjusted for fixed effects instead of the original observations. The preadjustment was based on a larger data set that included all contemporaneous animals of the genotyped animals, rather than just using the group of genotyped animals. Using this larger data set allowed us to account more accurately for contemporary group effects. The fixed effects estimates used for the preadjustment of the phenotypes were obtained by fitting a single trait pedigree-based linear model using ASReml v3.0 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gilmour <em>et al.</em> 2009</a>). The model for number of teats consisted of sex and herd-year-season as fixed effects and an additive genetic effect and a residual as random effects. The model for backfat consisted of sex, herd-year-week, and weight as fixed effects and an additive genetic effect, common litter effect, and a residual as random effects. For lifetime daily gain, the model consisted of sex and herd-year-week as fixed effects and an additive genetic effect, common litter effect and a residual as random effects. For the Landrace population, the final data set consisted of 141,248 animals for number of teats, 36,413 animals for backfat, and 37,071 animals for lifetime daily gain. For the Large White population, the final data set consisted of 156,065 animals for number of teats, 41,192 animals for backfat, and 41,740 animals for lifetime daily gain. For the Pietrain population, the final data set consisted of 33,964 animals for backfat and 31,184 animals for lifetime daily gain. The trait number of teats was not recorded in the Pietrain population. Descriptive statistics of the phenotypes are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467227/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467227/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Statistical analyses</h3><p id="p-15">Parameters were estimated using models with random regression on SNP genotypes. Single trait within-line analyses were performed with three different models implemented in the program BayZ (<a href="http://www.bayz.biz/">http://www.bayz.biz/</a>), the same for both real and simulated data:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-6.gif"/></noscript></span></span>where <strong>y</strong> is a vector of preadjusted phenotypic observations; <em>µ</em> is the mean of the populations and <strong>1</strong> a vector of ones; <strong>L</strong> is the design matrix for the common litter effects (only used for backfat and lifetime daily gain); <strong>b</strong> is a unknown vector of common litter effects; <strong>A</strong>, <strong>D</strong>, and <strong>I</strong> are design matrices with regressors for additive, dominance, and imprinting effects respectively; <strong>a</strong>, <strong>d,</strong> and <strong>i</strong> are unknown vectors of additive, dominance, and imprinting effects, respectively;, and <strong>e</strong> is a vector of residuals. The entries of the design matrices <strong>A</strong>, <strong>D,</strong> and <strong>I</strong> are regressors calculated from the observed phased probabilities of the marker genotypes (<em>regA</em>, <em>regD</em>, and <em>regI</em>), as described in the genomic data section above.</p><p id="p-16">Assumed distributions were: <strong>a</strong> ∼ <em>N</em>(<strong>0</strong>,<strong>I<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-7.gif"/></noscript></span></span></strong>), <strong>d</strong> ∼ <em>N</em>(<strong>0</strong>,<strong>I<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-8.gif"/></noscript></span></span></strong>), <strong>i</strong> ∼ <em>N</em>(<strong>0</strong>,<strong>I<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-9.gif"/></noscript></span></span></strong>), <strong>b</strong> ∼ <em>N</em>(<strong>0</strong>,<strong>I<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-10.gif"/></noscript></span></span></strong>), and <strong>e</strong> ∼ <em>N</em>(<strong>0</strong>,<strong>I<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-11.gif"/></noscript></span></span></strong>), with <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-12.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-13.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-14.gif"/></noscript></span></span> being the per-SNP variance for additive, dominance, and imprinting effects, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-16.gif"/></noscript></span></span> the common litter and residual variance, respectively. The model was fitted with a Bayesian approach in the Bayz software package (<a href="http://www.bayz.biz/">http://www.bayz.biz/</a>) as described by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Krag <em>et al.</em> (2013)</a> to estimate variance components and heritabilities in SNP-based models. The prior distributions for unknown variance parameters were set as unbounded uniform, which makes the Bayesian posterior distribution mathematically identical to the likelihood. The generated Monte Carlo chain starts with all regression parameters and other location parameters at zero, and all variance parameters at 1, and blocked Gibbs samplers are used to facilitate mixing. Each model was run as a single chain with a length of 500,000 (real data) and 100,000 (simulated data), which was sampled each 100 iterations. The first 50,000 iterations of each run were regarded as burn-in period.</p><p id="p-17">Because the evaluated models were SNP-based models, they do not readily provide estimates of total explained variance. One way to obtain the total explained variance from a model term like <strong>Aa</strong> is to write <em>var</em>(<strong>Aa</strong>) = <strong>AA</strong>′<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-17.gif"/></noscript></span></span> and compute or evaluate the expected average diagonal of <strong>AA</strong>′ to provide the constant to scale the per-SNP explained variance to total explained variance. In this way total variance in the models can be expressed as <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-18.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-19.gif"/></noscript></span></span> for MA, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-20.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-21.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-22.gif"/></noscript></span></span> for MAD and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-23.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-24.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-25.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-26.gif"/></noscript></span></span> for MADI, with <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-27.gif"/></noscript></span></span>= <strong>AA’<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-28.gif"/></noscript></span></span></strong> (total additive variance), <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-29.gif"/></noscript></span></span>= <strong>DD’<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-30.gif"/></noscript></span></span></strong> (total dominance variance), and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-31.gif"/></noscript></span></span>= <strong>II’<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-32.gif"/></noscript></span></span></strong> (total imprinting variance). Only for backfat and lifetime daily gain, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-33.gif"/></noscript></span></span> was included. Alternatively, the variance contributed by a random effect could be estimated by evaluating the sample variance of the entries of the vectors <strong>Aa, Dd</strong>, and <strong>Ii</strong> at each iteration of the Gibbs sampler (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sorensen <em>et al.</em> 2001</a>). This has the advantage that the posterior standard deviations also can be obtained for total explained variance, and explained variances can be split easily by chromosome. The latter is done by computing var(<strong>Aa</strong>) per Markov chain Monte Carlo cycle only for the part of the covariates in <strong>A</strong> and matching regression parameters in <strong>a</strong> that belong to a particular chromosome. The narrow-sense heritability was defined as <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-34.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-35.gif"/></noscript></span></span>, and the proportion of phenotypic variance explained by dominance and imprinting effects was defined as <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-36.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-37.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-38.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-39.gif"/></noscript></span></span>, respectively.</p><p id="p-18">The portioning of the genetic variance as described previously has been defined as the “genotypic model” (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> 2013</a>), which implies that <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-40.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-41.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/embed/mml-math-42.gif"/></noscript></span></span> are the variance of the genotypic additive, dominance, and imprinting values, respectively. The genotypic model and the breeding (or classical) model are statistically equivalent (<em>i.e.</em>, they lead to the same probability model). However, the parameters obtained with these models have different interpretations. In the genotypic model, the additive variance is the variance of additive effects (average difference between homozygotes), whereas in the breeding model, additive effects are functions of allele substitution effects. To make the variance estimates from the genotypic model comparable with the estimates of the breeding model, a transformation of these results was proposed by <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> (2013)</a>. We have applied the transformation proposed by <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> (2013)</a> to the estimates from the MAD model and included the results in the supporting material (supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019513/-/DC1/FileS1.docx">File S1</a>).</p><p id="p-19">Finally, we evaluated whether the proportion of variance explained by a single chromosome was related to its physical length. The length of a given chromosome was defined as the distance between the first and the last SNP on this chromosome according to the Sscrofa 10.2 assembly (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Groenen <em>et al.</em> 2012</a>). The relationship between variance explained and physical length of the chromosome was expressed as the coefficient of determination (r<sup>2</sup>) from the regression of variance explained on physical length. Variance explained by each chromosome individually was obtained based on the effects of SNPs on that chromosome. SNP effects were from the analyses where all SNPs from all chromosomes were evaluated simultaneously. Variance per chromosome was estimated for all three models evaluated (MA, MAD, and MADI).</p></div><div id="sec-6" class="subsection"><h3>Model comparison</h3><p id="p-20">Models were compared using the Deviance Information Criterion (DIC, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Spiegelhalter <em>et al.</em> 2002</a>). DIC is widely used for Bayesian model comparison and is analogous to the Akaike Information Criterion (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akaike 1974</a>). DIC combines a measure of model fit (the expected deviance) with a measure of model complexity (the effective number of parameters) over all iterations after burn-in. The model with lowest DIC is chosen as the best fitting model (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Spiegelhalter <em>et al.</em> 2002</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-21">Relevant data is available as supplemental information.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Simulation</h3><p id="p-22">Average narrow-sense heritability over the 10 replicates of the simulated trait was estimated at 0.116, 0.092, and 0.098 using the MA, MAD, and MADI models, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The average proportion of phenotypic variance explained by dominance effects was 0.111 using the MAD model and 0.097 using the MADI model. Using the MADI model, the average proportion of phenotypic variance explained by imprinting effects was 0.103.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467113/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467113/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimated variance components and proportion of phenotypic variance (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T2/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T2/embed/mml-math-43.gif"/></noscript></span></span>) explained by additive, dominance, and imprinting effects for the simulated data</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">The pairwise sampling correlation between additive, dominance, imprinting and error variance are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019513/-/DC1/TableS1.docx">Table S1</a>. The average correlation between the different variances of the 10 replicates of the simulated trait ranged from −0.586 to 0.018. The strongest correlations (−0.586 to −0.186) were observed between the residual variance and the variance of the three components of genetic variance evaluated. The lowest correlations (approximately zero) were observed between imprinting and additive variance and between imprinting and dominance variance.</p></div><div id="sec-10" class="subsection"><h3>Real data</h3><p id="p-25">For the trait number of teats, the narrow-sense heritability estimated using MA was 0.319 in the Landrace and 0.343 in the Large White population (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). When both MAD and MADI were used, the narrow-sense heritability was approximately the same in both populations (∼0.306). The estimates of the proportion of phenotypic variance explained by dominance effects, however, showed a large difference between populations. The proportion of phenotypic variance explained by dominance effects for number of teats in the Landrace population (0.039) was a little bit more than one third that of the Large White population (∼0.100). The proportion of phenotypic variance explained by imprinting effects for number of teats was low in both populations, 0.015 in the Landrace and 0.010 in the Large White.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467106/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467106/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Variance components and proportion of phenotypic variance (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T3/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T3/embed/mml-math-60.gif"/></noscript></span></span>) explained by additive, dominance, and imprinting effects for the real data in the Landrace population</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467252/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467252/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Variance components and proportion of phenotypic variance (<span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T4/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T4/embed/mml-math-79.gif"/></noscript></span></span>) explained by additive, dominance, and imprinting effects of the real data in the Large White population</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">For the trait backfat, the narrow-sense heritability estimated using MA was 0.520, 0.390, and 0.419 in the Landrace, Large White, and Pietrain populations, respectively. Additive heritabilities decreased to 0.469, 0.353, and 0.394 in the Landrace, Large White, and Pietrain populations, respectively, when the MA model was replaced by MADI (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). MAD and MADI resulted in almost the same estimates of narrow-sense heritability, and of the proportion of phenotypic variance explained by dominance effects for backfat in all populations and narrow-sense heritability was always lower than the estimate based on MA. Similar to number of teats, the proportion of phenotypic variance explained by dominance effects for backfat was variable between populations with estimates of 0.102 in the Landrace, 0.146 in the Large White, and 0.064 in the Pietrain population. The proportion of phenotypic variance explained by imprinting effects for backfat was 0.017 in the Landrace, 0.029 in the Large White, and 0.020 in the Pietrain population.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467314/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467314/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Variance components and proportion of phenotypic variance (<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T5/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12/2629/T5/embed/mml-math-98.gif"/></noscript></span></span>) explained by additive, dominance, and imprinting effects for the real data in the Pietrain population</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">Finally, for the trait lifetime daily gain, the narrow-sense heritability estimated using MA was 0.267 in the Landrace, 0.241 in the Large White, and 0.314 in the Pietrain population (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). Again, the narrow-sense heritability and the proportion of phenotypic variance explained by dominance effects were similar with the use of either MAD or MADI in all populations, and additive estimates were smaller than those from the MA model. The proportion of phenotypic variance explained by dominance effects estimated with MADI were greater for lifetime daily gain than for the other two traits in the Landrace (0.158) and Pietrain populations (0.199) and similar to dominance for backfat in the Large White population (0.130). The proportion of phenotypic variance explained by imprinting effects of lifetime daily gain was again low, as for the other 2 traits (&lt;0.020 in all populations).</p></div><div id="sec-11" class="subsection"><h3>Model comparison</h3><p id="p-31">Based on the estimated DIC (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>), MAD and MADI presented a better fit to the data than MA for all traits in all populations, except for number of teats in the Landrace population (which was the trait with the lowest proportion of dominance variance in this study). The MADI model was slightly superior to MAD for backfat in the Landrace population, for all traits in the Large White population, and for lifetime daily gain in the Pietrain population.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467030/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467030/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Deviance information criterion</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Variance explained by individual chromosomes</h3><p id="p-33">After estimation of SNP effects using all SNPs simultaneously, we estimated the additive, dominance, and imprinting variance by using the complete set of SNPs and also the variances per individual chromosomes. Estimates obtained from genome-wide SNPs were close to results from adding up the contributions of individual chromosomes. The largest difference was observed for number of teats in the Landrace population. The additive variance estimated using all SNPs was 0.33 for number of teats, while adding up the contributions of individual chromosomes resulted in an estimate of 0.39 for number of teats. The variance explained per chromosome for all traits in all populations using MADI is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019513/-/DC1/FileS2.xlsx">File S2</a>. We report results obtained using MADI. Very similar results were observed with all three models (MA, MAD, and MADI).</p><p id="p-34">For all traits, the proportion of additive, dominance, and imprinting variance explained per chromosome showed a strong linear relationship with chromosome length (r<sup>2</sup> ranging from 0.84 to 0.94).</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Simulation</h3><p id="p-35">Analysis of simulated data with the MADI model resulted in estimates of additive, dominance, and imprinting variance that were very close to the simulated values (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). However, the additive variance was overestimated when using the MA model and the dominance variance was overestimated with the MAD model. This is a sensible result as all models, except the MADI model, are under-parameterized. A model that allows a proper dissection of the variances should yield variance components that are uncorrelated (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hill 2010</a>). To test this we calculated the pairwise sampling correlations between the error, additive, dominance, and imprinting variance. The pairwise sampling correlations were moderate and mostly negative (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019513/-/DC1/TableS1.docx">Table S1</a>). On the basis of these simulation results, we therefore concluded that data structure and the methodology will allow us to disentangle additive, dominance and imprinting variance, although this simulated scenario may not be representative of the genetic architecture of a real complex trait.</p></div><div id="sec-15" class="subsection"><h3>Real data</h3><p id="p-36">In all populations, we observed a reduction in the narrow-sense heritability of all evaluated traits when dominance effects were accounted for (<em>e.g.</em>, using MAD instead of MA). The smallest decrease in narrow-sense heritability was observed for number of teats (4.2%) and the highest for lifetime daily gain (21.3%), both in the Landrace population (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The broad-sense heritability (sum of the heritabilities due to all genetic effects used in the model) of all evaluated traits increased in all three populations when dominance and imprinting effects were added to the model. The broad-sense heritability of lifetime daily gain was &gt;30% greater when using MADI compared to using MA (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). A reduction of additive genetic variance and an increase in the broad-sense heritability was previously reported (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Su <em>et al.</em> 2012</a>) when nonadditive genetic effects were included in the model to evaluate daily gain in pigs. For height in trees, the narrow-sense heritability was found to reduce by 26% with the inclusion of nonadditive genetic effects in the model (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>). According to <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> (2014)</a> and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pante <em>et al.</em> (2002)</a>, such a reduction in the additive genetic variance should be expected when dominance effects are present, since dominance effects also contribute to the additive genetic variance in the MA model. However, <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> (2013)</a> reported that such a reduction in the additive variance should be seen as an underestimation, as a consequence of overestimating the dominance variance. These authors described that when dominance is fitted in genotypic models (such as the one applied in the current study and by <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Su <em>et al.</em> (2012)</a> and <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> (2014)</a>, the part of the dominance effect that contributes to the allele substitution effect is shifted to the dominance variance. Because of this shift, the estimates from genotypic models are not directly comparable to pedigree-based estimates. Therefore, with methodology applied in the current study we could interpret the decrease in the narrow-sense heritability from the MA model compared to the MAD model as the contribution of dominance effects to the additive genetic variance. If the aim is to estimate breeding values and dominance deviations (<em>i.e.</em>, the traditional breeding model), the parameterization proposed by <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> (2013)</a> should be applied.</p><p id="p-37">The substantial greater values for the broad-sense heritability compared to the narrow-sense heritability indicate that dominance effects make an important contribution to the genetic variance of the evaluated traits and populations in this study, especially to the trait lifetime daily gain. For lifetime daily gain in the Pietrain population using MADI, we observed the greatest proportion of phenotypic variance explained by dominance effects (0.198), with a ratio between dominance and additive variance of 0.82. In other lines, the ratios were also high. Using the same model, we found that the ratio between dominance and additive variance was 0.71 in the Landrace and 0.66 in the Large White population. Moreover, our results also show that additive variance accounts for the largest fraction of the genetic variance. This is in agreement with a previous study that described that additive variance is expected to account for &gt;50% (often about 100%) of the total genetic variance (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hill <em>et al.</em> 2008</a>). However, the estimates of additive variance of the traits in the populations here evaluated might still become smaller if epistatic interactions exist and would be included as a separate variance component. Although the role of epistatic interactions in the genetic architecture of complex traits has been investigated in different species (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Su <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Le Rouzic <em>et al.</em> 2008</a>), we did not attempt to estimate epistatic variance because the power to identify these effects in segregating populations was expected to be low (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Melchinger <em>et al.</em> 2007</a>). To be able to detect epistatic interactions in outbred populations, loci with these effects should have a large effect and segregate with an intermediate frequency (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hill <em>et al.</em> 2008</a>).</p><p id="p-38">Individual loci that show the effect of imprinting have been identified for pigs, such as <em>IGF</em>2 (insulin-like growth factor 2 gene, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jeon <em>et al.</em> 1999</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nezer <em>et al.</em> 1999</a>). The contribution of imprinting to the total genetic variance is, however, still unknown. No reports were found in literature that attempt to quantify total imprinting variance using genomic data. In this study, the trait with the greatest proportion of phenotypic variance explained by imprinting effects was backfat (0.017 in the Landrace, 0.029 in the Large White, and 0.020 in the Pietrain population), although the estimates were still quite low, in comparison with the amount of narrow-sense heritability and the proportion of phenotypic variance explained by dominance effects. In mice, a gene expression QTL mapping study for body composition traits showed that imprinting QTL accounted for only a limited amount of the phenotypic variance (&lt;2.50%) for most traits (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cheng <em>et al.</em> 2013</a>). In a pedigree-based study in pigs, it was shown that 5–7% of the phenotypic variance of backfat and 1–4% of growth rate was explained by paternal imprinting (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">De Vries <em>et al.</em> 1994</a>). That study also showed maternal imprinting to account for 2–5% and 3–4% of the phenotypic variance of backfat and growth rate, respectively. Although our genomic estimates are lower than the pedigree-based estimates described by <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">De Vries <em>et al.</em> (1994)</a>, the two results agree that imprinting effects are more important for backfat than for growth traits. The amount of phenotypic variance of number of teats due to imprinting effects has not yet been described in the literature. However, two imprinted QTL have been reported on chromosomes 2 and 12 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hirooka <em>et al.</em> 2001</a>). These two QTL explained 1.3 and 2.2% of the phenotypic variance of number of teats, while in our study the proportion of phenotypic variance explained by imprinting effects of number of teats in both populations was ≤1.5%. The larger imprinting variances found by <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hirooka <em>et al.</em> (2001)</a> may in part be explained by the design, an experimental F2 population, analyzed in their QTL study.</p><p id="p-39">Due to the low proportion of phenotypic variance explained by imprinting, the relevance of estimating imprinting effects may be low when the aim is to predict the phenotypes number of teats, backfat, and lifetime daily gain in the evaluated populations. However, this study shows that, when present, dominance and imprinting variance can be detected and estimated with a SNP regression model. Using pedigree-based analysis this would typically not be feasible, for different reasons. First, the estimation of dominance variance using pedigree data requires data from large full-sib families (<a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Vitezica <em>et al.</em> 2013</a>), which is often not available in humans and livestock species. Second, pedigree-based methods have difficulties in disentangling imprinting from maternal and permanent environmental effects (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wolf and Cheverud 2012</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tier and Meyer 2004</a>). Third, pedigree-based analysis often overestimates additive variance (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Vinkhuyzen <em>et al.</em> 2013</a>) and underestimates dominance variance (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>). Although the use of genome-wide markers, compared to pedigree data, has been described as a more precise alternative to partition the genetic variance (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Vinkhuyzen <em>et al.</em> 2013</a>; <a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lee <em>et al.</em> 2008</a>), it also has its pitfalls. If the causal variants are not in linkage disequilibrium with the SNPs used for the estimation of the variance components, their contribution to the variance will not be captured. The proportion of the variance explained by the SNPs is therefore likely to be underestimated (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Vinkhuyzen <em>et al.</em> 2013</a>). This phenomenon has been described as “the case of the missing heritability” (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Maher 2008</a>). Our genomic estimates of the additive genetic variance (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>) were on average 28% lower than pedigree-based estimates that were obtained using the same data accounting only for additive effects. The pedigree-based heritability of number of teats was 0.340 in the Landrace and 0.420 in the Large White population; the pedigree-based heritability of backfat was 0.668 in the Landrace, 0.490 in the Large White and 0.513 in the Pietrain population; and the pedigree-based heritability of lifetime daily gain was 0.401 in the Landrace, 0.300 in the Large White and 0.474 in the Pietrain population (data not shown). Although these differences between the genomic and pedigree estimates are considerable, it is difficult to say if they are more likely due to an overestimation with pedigree or due to an underestimation with genomics. Nevertheless, using genomic data to estimate additive, dominance, and imprinting variances allows us to not only better understand the genetic architecture of the evaluated traits, but it might also improve the prediction of phenotypes compared to pedigree-based methods.</p><p id="p-40">In recent studies, the inclusion of dominance effects in genomic evaluations of livestock has been reported to increase the accuracy and decrease the bias of estimated breeding values (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Toro and Varona 2010</a>; <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Su <em>et al.</em> 2012</a>). In addition, using dominance in genomic evaluations is expected to result in greater cumulative response to selection of purebred animals for crossbred performance than additive models, especially in the presence of overdominance and when retraining is not performed at each generation (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zeng <em>et al.</em> 2013</a>). Even when purely additive effects were evaluated, the inclusion of dominance in the genomic evaluations did not decrease the accuracy of prediction (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Toro and Varona 2010</a>; <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Su <em>et al.</em> 2012</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Zeng <em>et al.</em> 2013</a>). In plants, simultaneously accounting for additive and non-additive effects was more stable and yielded higher predictive ability of the mean phenotype than models that only account for additive effects (<a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">Muñoz <em>et al.</em> 2014</a>). Also in mice, the prediction of phenotypes of complex traits using a model with additive and dominance effects has proven to be feasible and accurate (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lee <em>et al.</em> 2008</a>). Therefore, combining additive, dominance, and imprinting under a genomic prediction scope opens new perspectives for the optimization of animal and plant breeding programs aiming for an improved prediction of crossbred performance, and also for identification of individuals that are at a risk for a given disease.</p></div><div id="sec-16" class="subsection"><h3>Variance explained per individual chromosome</h3><p id="p-41">The strong linear relationship between chromosome length and proportion of variance explained per chromosome in our study was in line with the strong relationship between additive variance explained per chromosome and chromosome length previously described in humans (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2011</a>) and in chickens (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdollahi‐Arpanahi <em>et al.</em> 2014</a>). Here we also showed that the same applies for dominance and imprinting variance. This indicates that the additive, dominance, and imprinting variance of number of teats, backfat and lifetime daily gain in these populations is explained by many genes located throughout the genome, rather than by a few mutations with large effects.</p><p id="p-42">The relationship between variance explained and chromosome length for number of teats in the Large White population, backfat in the Landrace population, and lifetime daily gain in the Pietrain population using MADI is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Although our results show that the variance of all three genetic effects have a strong relationship with chromosome length, the r<sup>2</sup> for the additive variance was lower than the r<sup>2</sup> for dominance and imprinting variance, especially in the Landrace and Large White populations. In addition, in the Pietrain population, the r<sup>2</sup> for dominance and imprinting variance (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a>) was lower than the r<sup>2</sup> observed in the Landrace and Large White populations. This was observed because the proportion of variance explained by chromosome 8 in the Pietrain is clearly lower than in the Landrace and Large White populations. Having a closer look at the data of chromosome 8, we observed that the number of SNPs on this chromosome in the Pietrain population was on average 15% lower than in the Landrace and the Large White populations (n = 1632 in Pietrain, n = 1871 in Landrace, n = 1985 in Large White). Besides chromosome 8, the number of SNPs per chromosome was similar in all three populations. This difference in number of SNPs on chromosome 8 is observed because in the Pietrain population, compared to the Landrace and the Large White populations, more SNPs presented low minor allele frequency or were completely fixed and therefore were excluded from the estimation of the variance components. This large number of SNPs with low minor allele frequency (or completely fixed) could be due to an ascertainment bias due to the selection of SNPs for the SNP chip used in this study. However, this could be also an indication that genes that influence traits included in the selection index of Pietrain are located on this chromosome. The breeding objectives in Pietrain (sire line) are distinct from those in the Landrace and Large White (dam lines) which are more similar. Given this difference, some alleles could have moved to fixation in Pietrain but not in Landrace and Large White.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2629/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of additive, dominance, and imprinting variance explained per individual chromosome against physical length of the chromosome. (A) Number of teats in the Large White population; (B) backfat in the Landrace population; (C) lifetime daily gain in the Pietrain population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-157289930" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of additive, dominance, and imprinting variance explained per individual chromosome against physical length of the chromosome. (A) Number of teats in the Large White population; (B) backfat in the Landrace population; (C) lifetime daily gain in the Pietrain population.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2629/F1.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/12/2629/F1.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2629/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2629/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467301" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-43" class="first-child">Proportion of additive, dominance, and imprinting variance explained per individual chromosome against physical length of the chromosome. (A) Number of teats in the Large White population; (B) backfat in the Landrace population; (C) lifetime daily gain in the Pietrain population.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The proportion of additive variance explained by chromosome 7 for number of teats in the Large White population is relatively high (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). This chromosome explained 21% more additive variance than chromosome 13, which is 62% longer than chromosome 7. This large proportion of explained variance is in agreement with the presence of a QTL for number of teats on this chromosome. In a previous study on a subset of the current Large White population, it was shown that on chromosome 7 a QTL is located in the region of the <em>VRTN</em> gene, explaining 2.5% of genetic variance (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Duijvesteijn <em>et al.</em> 2014</a>). In the current study, we showed that chromosome 7 accounted for 7.75% of the additive variance (5.64% of the total genetic variance using MADI).</p><p id="p-45">Dominance effects account for a large proportion of the total genetic variance (up to 44%) for number of teats, backfat and lifetime daily gain in the pig populations evaluated. Although the contribution of imprinting effects to the total phenotypic variance of the evaluated traits was relatively small (1–3%), the SNP regression method allowed estimation of the additive, dominance and imprinting effects and resulting variances. Our results indicate a strong relationship between additive variance explained per chromosome and chromosome length, which has been previously described for other traits in other species. In addition, we also show that a similar linear relationship exists for dominance and imprinting variance. These novel results improve our understanding of the genetic architecture of the evaluated traits. The model can now be applied to other traits and species. Our results also open new perspectives for the inclusion of dominance and imprinting effects in prediction of phenotypes, especially regarding mate allocation techniques in animal and plant breeding, and for assessment of the risk of disease in humans.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We acknowledge input from anonymous reviewers, the associate editor Gustavo de los Campos, and Esinam Nancy Amuzu-Aweh, whose comments improved the manuscript. Financial support from Breed4Food Consortium is gratefully acknowledged.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-47">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019513/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019513/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-48"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-24"><span class="received-label">Received </span>June 24, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-28"><span class="accepted-label">Accepted </span>September 28, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Lopes <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.1" data-doi="10.1111/jbg.12079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdollahi‐Arpanahi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pakdel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nejati‐Javaremi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moradi Shahrbabak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dissection of additive genetic variability for quantitative traits in chickens using SNP markers.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12079%26rft_id%253Dinfo%253Apmid%252F24460953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24460953&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.2" data-doi="10.1109/TAC.1974.1100705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A new look at the statistical model identification.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Automat. Contr.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BAutomat.%2BContr.%26rft.volume%253D19%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTAC.1974.1100705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TAC.1974.1100705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.3" data-doi="10.1126/science.32.827.627-a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1910</span> <span class="cit-article-title">The Mendelian theory of heredity and the augmentation of vigor.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBRUCE%26rft.auinit1%253DA.%2BB.%26rft.volume%253D32%26rft.issue%253D827%26rft.spage%253D627%26rft.epage%253D628%26rft.atitle%253DTHE%2BMENDELIAN%2BTHEORY%2BOF%2BHEREDITY%2BAND%2BTHE%2BAUGMENTATION%2BOF%2BVIGOR%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.32.827.627-a%26rft_id%253Dinfo%253Apmid%252F17816706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjMyLzgyNy82MjctYSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.4" data-doi="10.1038/nrg2664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of inbreeding depression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bgenetics%2Bof%2Binbreeding%2Bdepression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2664%26rft_id%253Dinfo%253Apmid%252F19834483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834483&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270915400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rachagani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cánovas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tait</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Body composition and gene expression QTL mapping in mice reveals imprinting and interaction effects.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D103%26rft_id%253Dinfo%253Apmid%252F24165562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24165562&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.6" data-doi="10.1371/journal.pone.0087666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Da</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mixed model methods for genomic prediction and variance component estimation of additive and dominance effects using SNP markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e87666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De87666%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0087666%26rft_id%253Dinfo%253Apmid%252F24498162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0087666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498162&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.7" data-doi="10.1126/science.28.718.454-b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1908</span> <span class="cit-article-title">Degeneration, albinism and inbreeding.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDavenport%26rft.auinit1%253DChas.%2BB.%26rft.volume%253D28%26rft.issue%253D718%26rft.spage%253D454%26rft.epage%253D455%26rft.atitle%253DDEGENERATION%252C%2BALBINISM%2BAND%2BINBREEDING%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.28.718.454-b%26rft_id%253Dinfo%253Apmid%252F17771943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjI4LzcxOC80NTQtYiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">On the detection of imprinted quantitative trait loci in experimental crosses of outbred species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253Dde%2BKoning%26rft.auinit1%253DD.-J.%26rft.volume%253D161%26rft.issue%253D2%26rft.spage%253D931%26rft.epage%253D938%26rft.atitle%253DOn%2Bthe%2BDetection%2Bof%2BImprinted%2BQuantitative%2BTrait%2BLoci%2Bin%2BExperimental%2BCrosses%2Bof%2BOutbred%2BSpecies%26rft_id%253Dinfo%253Apmid%252F12072486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzIvOTMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.9" data-doi="10.1038/nrg2898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predicting genetic predisposition in humans: the promise of whole-genome markers.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">880</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D880%26rft.epage%253D886%26rft.atitle%253DPredicting%2Bgenetic%2Bpredisposition%2Bin%2Bhumans%253A%2Bthe%2Bpromise%2Bof%2Bwhole-genome%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2898%26rft_id%253Dinfo%253Apmid%252F21045869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2898&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21045869&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.10" data-doi="10.1007/BF00220813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Vries</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Gametic imprinting effects on rate and composition of pig growth.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D88%26rft.spage%253D1037%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220813%26rft_id%253Dinfo%253Apmid%252F24186259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24186259&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.11" data-doi="10.1186/1471-2164-15-542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duijvesteijn</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veltmaat</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knol</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlizius</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution association mapping of number of teats in pigs reveals regions controlling vertebral development.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D542%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-542%26rft_id%253Dinfo%253Apmid%252F24981054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24981054&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-5.12.2629.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-publ-name">Longmans Green</span>, <span class="cit-publ-loc">Harlow, Essex, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-5.12.2629.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml User Guide Release 3.0</span>. <span class="cit-publ-name">VSN International Ltd</span>, <span class="cit-publ-loc">Hemel Hempstead, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.14" data-doi="10.1038/nature11622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uenishi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuggle</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analyses of pig genomes provide insight into porcine demography and evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGroenen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D393%26rft.epage%253D398%26rft.atitle%253DAnalyses%2Bof%2Bpig%2Bgenomes%2Bprovide%2Binsight%2Binto%2Bporcine%2Bdemography%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11622%26rft_id%253Dinfo%253Apmid%252F23151582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23151582&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311031600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.15" data-doi="10.1186/1297-9686-43-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunstan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A combined long-range phasing and long haplotype imputation method to impute phase for SNP genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHickey%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D43%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DA%2Bcombined%2Blong-range%2Bphasing%2Band%2Blong%2Bhaplotype%2Bimputation%2Bmethod%2Bto%2Bimpute%2Bphase%2Bfor%2BSNP%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-12%26rft_id%253Dinfo%253Apmid%252F21388557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21388557&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.16" data-doi="10.1098/rstb.2009.0203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding and using quantitative genetic variation.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D365%26rft.issue%253D1537%26rft.spage%253D73%26rft.epage%253D85%26rft.atitle%253DUnderstanding%2Band%2Busing%2Bquantitative%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0203%26rft_id%253Dinfo%253Apmid%252F20008387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjExOiIzNjUvMTUzNy83MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.17" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirooka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harlizius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Arendonk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattink</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A whole-genome scan for quantitative trait loci affecting teat number in pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">2320</span>–<span class="cit-lpage">2326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHirooka%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.issue%253D9%26rft.spage%253D2320%26rft.epage%253D2326%26rft.atitle%253DA%2Bwhole-genome%2Bscan%2Bfor%2Bquantitative%2Btrait%2Bloci%2Baffecting%2Bteat%2Bnumber%2Bin%2Bpigs%26rft_id%253Dinfo%253Apmid%252F11583418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11583418&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171208900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.19" data-doi="10.1038/5938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeon</span>  <span class="cit-name-given-names">J.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Törnsten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giuffra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amarger</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A paternally expressed QTL affecting skeletal and cardiac muscle mass in pigs maps to the IGF2 locus.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJeon%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D158%26rft.atitle%253DA%2Bpaternally%2Bexpressed%2BQTL%2Baffecting%2Bskeletal%2Band%2Bcardiac%2Bmuscle%2Bmass%2Bin%2Bpigs%2Bmaps%2Bto%2Bthe%2BIGF2%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F5938%26rft_id%253Dinfo%253Apmid%252F9988263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/5938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9988263&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078399500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.20" data-doi="10.1017/S1751731112002017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krag</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shariati</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buitenhuis</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP-based heritability estimation using a Bayesian approach.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D7%26rft.spage%253D531%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731112002017%26rft_id%253Dinfo%253Apmid%252F23177174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731112002017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23177174&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.21" data-doi="10.1038/nrg3543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic imprinting and parent-of-origin effects on complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D609%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3543%26rft_id%253Dinfo%253Apmid%252F23917626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23917626&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.22" data-doi="10.1534/genetics.108.089300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Rouzic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Álvarez-Castro</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dissection of the genetic architecture of body weight in chicken reveals the impact of epistasis on domestication traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1591</span>–<span class="cit-lpage">1599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089300%26rft_id%253Dinfo%253Apmid%252F18622035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.23" data-doi="10.1371/journal.pgen.1000231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Predicting unobserved phenotypes for complex traits from whole-genome SNP data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BH.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000231%26rft.epage%253De1000231%26rft.atitle%253DPredicting%2Bunobserved%2Bphenotypes%2Bfor%2Bcomplex%2Btraits%2Bfrom%2Bwhole-genome%2BSNP%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000231%26rft_id%253Dinfo%253Apmid%252F18949033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18949033&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maher</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The case of the missing heritability.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMaher%26rft.auinit1%253DB.%26rft.volume%253D456%26rft.issue%253D7218%26rft.spage%253D18%26rft.epage%253D21%26rft.atitle%253DPersonal%2Bgenomes%253A%2BThe%2Bcase%2Bof%2Bthe%2Bmissing%2Bheritability.%26rft_id%253Dinfo%253Apmid%252F18987709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18987709&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260674000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.25" data-doi="10.1534/genetics.107.080564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muminović</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegenast</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic basis of heterosis for growth-related traits in Arabidopsis investigated by testcross progenies of near-isogenic lines reveals a significant role of epistasis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1827</span>–<span class="cit-lpage">1837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMelchinger%26rft.auinit1%253DA.%2BE.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D1827%26rft.epage%253D1837%26rft.atitle%253DGenetic%2BBasis%2Bof%2BHeterosis%2Bfor%2BGrowth-Related%2BTraits%2Bin%2BArabidopsis%2BInvestigated%2Bby%2BTestcross%2BProgenies%2Bof%2BNear-Isogenic%2BLines%2BReveals%2Ba%2BSignificant%2BRole%2Bof%2BEpistasis%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080564%26rft_id%253Dinfo%253Apmid%252F18039884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE4MjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2629.26" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M.F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M.D.V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unraveling additive from non-additive effects using genomic relationship matrices.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.27" data-doi="10.1038/5935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nezer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouwers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppieters</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Detilleux</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">An imprinted QTL with major effect on muscle mass and fat deposition maps to the IGF2 locus in pigs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNezer%26rft.auinit1%253DC.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D156%26rft.atitle%253DAn%2Bimprinted%2BQTL%2Bwith%2Bmajor%2Beffect%2Bon%2Bmuscle%2Bmass%2Band%2Bfat%2Bdeposition%2Bmaps%2Bto%2Bthe%2BIGF2%2Blocus%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F5935%26rft_id%253Dinfo%253Apmid%252F9988262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/5935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9988262&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078399500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.28" data-doi="10.1371/journal.pone.0085792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Including dominance effects in the genomic BLUP method for genomic evaluation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e85792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85792%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085792%26rft_id%253Dinfo%253Apmid%252F24416447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24416447&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.29" data-doi="10.1016/S0044-8486(01)00825-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pante</span>  <span class="cit-name-given-names">M. J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimation of additive and dominance genetic variances for body weight at harvest in rainbow trout, <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D204%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0044-8486%252801%252900825-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0044-8486(01)00825-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173785900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.30" data-doi="10.1371/journal.pone.0006524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affara</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRamos%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6524%26rft.epage%253De6524%26rft.atitle%253DDesign%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bin%2Bthe%2Bpig%2Busing%2BSNPs%2Bidentified%2Band%2Bcharacterized%2Bby%2Bnext%2Bgeneration%2Bsequencing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006524%26rft_id%253Dinfo%253Apmid%252F19654876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19654876&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dominance and epistasis are the main contributors to heterosis for plant height in rice.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D215%26rft.spage%253D11%26rft_id%253Dinfo%253Apmid%252F24388510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24388510&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shull</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1908</span> <span class="cit-article-title">The composition of a field of maize.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">296</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D4%26rft.spage%253D296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.33" data-doi="10.1017/S0016672300004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Inferring the trajectory of genetic variance in the course of artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSorensen%26rft.auinit1%253DD.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D94%26rft.atitle%253DInferring%2Bthe%2Btrajectory%2Bof%2Bgenetic%2Bvariance%2Bin%2Bthe%2Bcourse%2Bof%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004845%26rft_id%253Dinfo%253Apmid%252F11279833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279833&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.34" data-doi="10.1111/1467-9868.00353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Linde</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bayesian measures of model complexity and fit.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D64%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.35" data-doi="10.1371/journal.pone.0045293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSu%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45293%26rft.epage%253De45293%26rft.atitle%253DEstimating%2Badditive%2Band%2Bnon-additive%2Bgenetic%2Bvariances%2Band%2Bpredicting%2Bgenetic%2Bmerits%2Busing%2Bgenome-wide%2Bdense%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045293%26rft_id%253Dinfo%253Apmid%252F23028912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028912&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.36" data-doi="10.1371/journal.pone.0103934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improvement of prediction ability for genomic selection of dairy cattle by including dominance effects.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e103934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103934%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103934%26rft_id%253Dinfo%253Apmid%252F25084281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25084281&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.37" data-doi="10.1111/j.1439-0388.2003.00444.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Approximating prediction error covariances among additive genetic effects within animals in multiple‐trait and random regression models.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D121%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2003.00444.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2003.00444.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220353500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.38" data-doi="10.1186/1297-9686-42-33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A note on mate allocation for dominance handling in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DToro%26rft.auinit1%253DM.%2BA.%26rft.volume%253D42%26rft.spage%253D33%26rft.epage%253D33%26rft.atitle%253DA%2Bnote%2Bon%2Bmate%2Ballocation%2Bfor%2Bdominance%2Bhandling%2Bin%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-33%26rft_id%253Dinfo%253Apmid%252F20699012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20699012&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.39" data-doi="10.1146/annurev-genet-111212-133258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinkhuyzen</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimation and partitioning of heritability in human populations using whole genome analysis methods.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133258%26rft_id%253Dinfo%253Apmid%252F23988118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23988118&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-5.12.2629.40" data-doi="10.1534/genetics.114.171017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A general unified framework to assess the sampling variance of heritability estimates using pedigree or marker-based relationships.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">199</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171017%26rft_id%253Dinfo%253Apmid%252F25361897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzEvMjIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.41" data-doi="10.1016/S0301-6226(99)00180-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittemore</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Impact of biotechnology on (cross) breeding programmes in pigs.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D65%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252899%252900180-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(99)00180-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.42" data-doi="10.1534/genetics.113.155176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the additive and dominant variance and covariance of individuals within the genomic selection scope.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155176%26rft_id%253Dinfo%253Apmid%252F24121775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.43" data-doi="10.1534/genetics.111.136440"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting maternal-effect loci by statistical cross-fostering.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136440%26rft_id%253Dinfo%253Apmid%252F22377636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzEvMjYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.44" data-doi="10.1101/gr.6665407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of individual genetic risk to disease from genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1520</span>–<span class="cit-lpage">1528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6665407%26rft_id%253Dinfo%253Apmid%252F17785532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMC8xNTIwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Dominance is the major genetic basis of heterosis in rice as revealed by QTL analysis using molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXiao%26rft.auinit1%253DJ.%26rft.volume%253D140%26rft.issue%253D2%26rft.spage%253D745%26rft.epage%253D754%26rft.atitle%253DDominance%2Bis%2Bthe%2Bmajor%2Bgenetic%2Bbasis%2Bof%2Bheterosis%2Bin%2Brice%2Bas%2Brevealed%2Bby%2BQTL%2Banalysis%2Busing%2Bmolecular%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7498751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzIvNzQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.46" data-doi="10.1038/ng.823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasquale</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caporaso</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome partitioning of genetic variation for complex traits using common SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D519%26rft.epage%253D525%26rft.atitle%253DGenome%2Bpartitioning%2Bof%2Bgenetic%2Bvariation%2Bfor%2Bcomplex%2Btraits%2Busing%2Bcommon%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.823%26rft_id%253Dinfo%253Apmid%252F21552263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552263&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2629.47" data-doi="10.1186/1297-9686-45-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toosi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-11%26rft_id%253Dinfo%253Apmid%252F23621868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23621868&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/12/2629.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2619" title="HES-Mediated Repression of Pten in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2639" title="Estimating Trait Heritability in Highly Fecund Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/12"><span>Volume 5 Issue 12, December 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/12.cover-source.jpg?itok=RngHqKQq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2629" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3y_m8niXXqc8CThXWUwr2TgFnUFEbeJ6mCK_6Jx7Fg4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/12/2629" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/12/2629" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/12/2629" class="active" data-icon-position="" data-hide-link-title="0">Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460204" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-smo8aK_cPZwjuk56SHaTl56gCyCSYkUawz0N5OfzWls" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2629.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/12/2629" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OB8R7kq68AXIC6S-7R0wk8Auw6NZy5d4Z9ZFUEsR7xU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/12/2629" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2629.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460204" id="node460204--2" data-pisa="ggg;5/12/2629" data-pisa-master="ggg;g3.115.019513" data-apath="/ggg/5/12/2629.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4715-853X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcos S. Lopes</span>, <span class="highwire-citation-author" data-delta="1">John W. M. Bastiaansen</span>, <span class="highwire-citation-author" data-delta="2">Luc Janss</span>, <span class="highwire-citation-author" data-delta="3">Egbert F. Knol</span> and <span class="highwire-citation-author" data-delta="4">Henk Bovenhuis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2629-2637;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019513 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460204--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcos S. Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BS.%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcos%2BS.%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4715-853X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcos S. Lopes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Marcos.Lopes@topigsnorsvin.com" class="" data-icon-position="" data-hide-link-title="0">Marcos.Lopes@topigsnorsvin.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John W. M. Bastiaansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BM.%2BBastiaansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastiaansen%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BW.%2BM.%2BBastiaansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luc Janss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%2BJanss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janss%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuc%2BJanss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Egbert F. Knol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Egbert%2BF.%2BKnol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knol%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEgbert%2BF.%2BKnol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Henk Bovenhuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henk%2BBovenhuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bovenhuis%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenk%2BBovenhuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/12/2629" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/12/2629&amp;title=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/12/2629&amp;title=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/12/2629&amp;title=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/12/2629&amp;text=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/12/2629&amp;title=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/12/2629&amp;t=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/12/2629&amp;title=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/12/2629&amp;title=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460204/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460204" id="node460204--3" data-pisa="ggg;5/12/2629" data-pisa-master="ggg;g3.115.019513" data-apath="/ggg/5/12/2629.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Estimation of Additive, Dominance, and Imprinting Genetic Variance Using Genomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4715-853X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcos S. Lopes</span>, <span class="highwire-citation-author" data-delta="1">John W. M. Bastiaansen</span>, <span class="highwire-citation-author" data-delta="2">Luc Janss</span>, <span class="highwire-citation-author" data-delta="3">Egbert F. Knol</span> and <span class="highwire-citation-author" data-delta="4">Henk Bovenhuis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2629-2637;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019513 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460204--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marcos S. Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BS.%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcos%2BS.%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4715-853X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcos S. Lopes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Marcos.Lopes@topigsnorsvin.com" class="" data-icon-position="" data-hide-link-title="0">Marcos.Lopes@topigsnorsvin.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John W. M. Bastiaansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BM.%2BBastiaansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastiaansen%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BW.%2BM.%2BBastiaansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luc Janss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%2BJanss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janss%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuc%2BJanss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Egbert F. Knol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Topigs Norsvin Research Center, 6641SZ Beuningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Egbert%2BF.%2BKnol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knol%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEgbert%2BF.%2BKnol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Henk Bovenhuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University, 6708PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henk%2BBovenhuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bovenhuis%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenk%2BBovenhuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2629&amp;rft.epage=2637&amp;rft.atitle=Estimation%20of%20Additive%2C%20Dominance%2C%20and%20Imprinting%20Genetic%20Variance%20Using%20Genomic%20Data&amp;rft.volume=5&amp;rft.issue=12&amp;rft.date=2015-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lopes%2C+Marcos+S.&amp;rft.au=Bastiaansen%2C+John+W.+M.&amp;rft.au=Janss%2C+Luc&amp;rft.au=Knol%2C+Egbert+F.&amp;rft.au=Bovenhuis%2C+Henk"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460204/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460204/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460204/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460204/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460204/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460204/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460204/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460204/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460204/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460204/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460204/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460204/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/12/2629" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/12/2629.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/12/2629.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/12/2629.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/12/2629.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
