<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/12/2801" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/12/2801.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/12/2801.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/12/2801.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/12/2801.atom" />
<meta name="HW.pisa" content="ggg;5/12/2801" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing" />
<meta name="DC.Identifier" content="10.1534/g3.115.023317" />
<meta name="DC.Date" content="2015-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Guo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Single-molecule real-time (SMRT) sequencing generates much longer reads than other widely used next-generation (next-gen) sequencing methods, but its application to whole genome/exome analysis has been limited. Here, we describe the use of SMRT sequencing coupled with barcoding to simultaneously analyze one or a small number of genomic targets derived from multiple sources. In the budding yeast system, SMRT sequencing was used to analyze strand-exchange intermediates generated during mitotic recombination and to analyze genetic changes in a forward mutation assay. The general barcoding-SMRT approach was then extended to diffuse large B-cell lymphoma primary tumors and cell lines, where detected changes agreed with prior Illumina exome sequencing. A distinct advantage afforded by SMRT sequencing over other next-gen methods is that it immediately provides the linkage relationships between SNPs in the target segment sequenced. The strength of our approach for mutation/recombination studies (as well as linkage identification) derives from its inherent computational simplicity coupled with a lack of reliance on sophisticated statistical analyses." />
<meta name="DC.Contributor" content="Xiaoge Guo" />
<meta name="DC.Contributor" content="Kevin Lehner" />
<meta name="DC.Contributor" content="Karen O’Connell" />
<meta name="DC.Contributor" content="Jenny Zhang" />
<meta name="DC.Contributor" content="Sandeep S. Dave" />
<meta name="DC.Contributor" content="Sue Jinks-Robertson" />
<meta name="article:published_time" content="2015-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Single-molecule real-time (SMRT) sequencing generates much longer reads than other widely used next-generation (next-gen) sequencing methods, but its application to whole genome/exome analysis has been limited. Here, we describe the use of SMRT sequencing coupled with barcoding to simultaneously analyze one or a small number of genomic targets derived from multiple sources. In the budding yeast system, SMRT sequencing was used to analyze strand-exchange intermediates generated during mitotic recombination and to analyze genetic changes in a forward mutation assay. The general barcoding-SMRT approach was then extended to diffuse large B-cell lymphoma primary tumors and cell lines, where detected changes agreed with prior Illumina exome sequencing. A distinct advantage afforded by SMRT sequencing over other next-gen methods is that it immediately provides the linkage relationships between SNPs in the target segment sequenced. The strength of our approach for mutation/recombination studies (as well as linkage identification) derives from its inherent computational simplicity coupled with a lack of reliance on sophisticated statistical analyses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/12/01" />
<meta name="citation_mjid" content="ggg;5/12/2801" />
<meta name="citation_id" content="5/12/2801" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/12/2801" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/12/2801.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/12/2801.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/12/2801.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023317" />
<meta name="citation_pmid" content="26497143" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2801" />
<meta name="citation_lastpage" content="2808" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xiaoge Guo" />
<meta name="citation_author_institution" content="Graduate Program in Molecular Cancer Biology, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author" content="Kevin Lehner" />
<meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author" content="Karen O’Connell" />
<meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author" content="Jenny Zhang" />
<meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author" content="Sandeep S. Dave" />
<meta name="citation_author_institution" content="Duke Cancer Institute, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author_institution" content="Department of Medicine, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author" content="Sue Jinks-Robertson" />
<meta name="citation_author_institution" content="Duke Cancer Institute, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, Duke University, Durham, North Carolina 27710" />
<meta name="citation_author_email" content="sue.robertson@duke.edu" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. G. Gibson;citation_author=J. I. Glass;citation_author=C. Lartigue;citation_author=V. N. Noskov;citation_author=R. Y. Chuang;citation_title=Creation of a bacterial cell controlled by a chemically synthesized genome.;citation_volume=329;citation_year=2010;citation_pmid=20488990;citation_doi=10.1126/science.1190719" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Guillet;citation_author=P. A. Van Der Kemp;citation_author=S. Boiteux;citation_title=dUTPase activity is critical to maintain genetic stability in Saccharomyces cerevisiae.;citation_pages=2056-2066;citation_volume=34;citation_year=2006;citation_issue=7;citation_pmid=16617146;citation_doi=10.1093/nar/gkl139" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=X. Guo;citation_author=S. Jinks-Robertson;citation_title=Roles of exonucleases and translesion synthesis DNA polymerases during mitotic gap repair in yeast.;citation_pages=1024-1030;citation_volume=12;citation_year=2013;citation_pmid=24210827;citation_doi=10.1016/j.dnarep.2013.10.001" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=A. Hendel;citation_author=E. J. Kildebeck;citation_author=E. J. Fine;citation_author=J. T. Clark;citation_author=N. Punjya;citation_title=Quantifying genome-editing outcomes at endogenous loci with SMRT sequencing.;citation_pages=293-305;citation_volume=7;citation_year=2014;citation_pmid=24685129;citation_doi=10.1016/j.celrep.2014.02.040" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. I. Lang;citation_author=A. W. Murray;citation_title=Estimating the Per-Base-Pair Mutation Rate in the Yeast Saccharomyces cerevisiae;citation_pages=67-82;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202359;citation_doi=10.1534/genetics.107.071506" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Lippert;citation_author=N. Kim;citation_author=J. E. Cho;citation_author=R. P. Larson;citation_author=N. E. Schoenly;citation_title=Role for topoisomerase 1 in transcription-associated mutagenesis in yeast.;citation_volume=108;citation_year=2011;citation_pmid=21177427;citation_doi=10.1073/pnas.1012363108" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=K. Mitchel;citation_author=H. Zhang;citation_author=C. Welz-Voegele;citation_author=S. Jinks-Robertson;citation_title=Molecular structures of crossover and noncrossover intermediates during gap repair in yeast: implications for recombination.;citation_pages=211-222;citation_volume=38;citation_year=2010;citation_issue=2;citation_pmid=20417600;citation_doi=10.1016/j.molcel.2010.02.028" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Quail;citation_author=M. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. L. Schoch;citation_author=K. A. Seifert;citation_author=S. Huhndorf;citation_author=V. Robert;citation_author=J. L. Spouge;citation_title=Nuclear ribosomal internal transcribed spacer (ITS) region as a universal DNA barcode marker for fungi.;citation_volume=109;citation_year=2012;citation_pmid=22454494;citation_doi=10.1073/pnas.1117018109" />
<meta name="citation_reference" content="citation_journal_title=Vox sanguinis;citation_journal_abbrev=Vox Sang;citation_author=A. Shander;citation_author=M. D. Cappellini;citation_author=L. T. Goodnough;citation_title=Iron overload and toxicity: the hidden risk of multiple blood transfusions.;citation_pages=185-197;citation_volume=97;citation_year=2009;citation_issue=3;citation_pmid=19663936;citation_doi=10.1111/j.1423-0410.2009.01207.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=D. Sharon;citation_author=H. Tilgner;citation_author=F. Grubert;citation_author=M. Snyder;citation_title=A single-molecule long-read survey of the human transcriptome.;citation_pages=1009-1014;citation_volume=31;citation_year=2013;citation_pmid=24108091;citation_doi=10.1038/nbt.2705" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. W. Snyder;citation_author=A. Adey;citation_author=J. O. Kitzman;citation_author=J. Shendure;citation_title=Haplotype-resolved genome sequencing: experimental methods and applications.;citation_pages=344-358;citation_volume=16;citation_year=2015;citation_pmid=25948246;citation_doi=10.1038/nrg3903" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Takahashi;citation_author=G. Burguiere-Slezak;citation_author=P. A. Van Der Kemp;citation_author=S. Boiteux;citation_title=Topoisomerase 1 provokes the formation of short deletions in repeated sequences upon high transcription in Saccharomyces cerevisiae.;citation_volume=108;citation_year=2011;citation_pmid=21177431;citation_doi=10.1073/pnas.1012582108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. B. Thyme;citation_author=Y. Song;citation_author=T. J. Brunette;citation_author=M. D. Szeto;citation_author=L. Kusak;citation_title=Massively parallel determination and modeling of endonuclease substrate specificity.;citation_volume=42;citation_year=2014;citation_pmid=25389263;citation_doi=10.1093/nar/gku1096" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. J. Travers;citation_author=C. S. Chin;citation_author=D. R. Rank;citation_author=J. S. Eid;citation_author=S. W. Turner;citation_title=A flexible and efficient template format for circular consensus sequencing and SNP detection.;citation_volume=38;citation_year=2010;citation_pmid=20571086;citation_doi=10.1093/nar/gkq543" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Zhang;citation_author=V. Grubor;citation_author=C. L. Love;citation_author=A. Banerjee;citation_author=K. L. Richards;citation_title=Genetic heterogeneity of diffuse large B-cell lymphoma.;citation_volume=110;citation_year=2013;citation_pmid=23292937;citation_doi=10.1073/pnas.1205299110" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="twitter:description" content="Single-molecule real-time (SMRT) sequencing generates much longer reads than other widely used next-generation (next-gen) sequencing methods, but its application to whole genome/exome analysis has been limited. Here, we describe the use of SMRT sequencing coupled with barcoding to simultaneously analyze one or a small number of genomic targets derived from multiple sources. In the budding yeast system, SMRT sequencing was used to analyze strand-exchange intermediates generated during mitotic recombination and to analyze genetic changes in a forward mutation assay. The general barcoding-SMRT approach was then extended to diffuse large B-cell lymphoma primary tumors and cell lines, where detected changes agreed with prior Illumina exome sequencing. A distinct advantage afforded by SMRT sequencing over other next-gen methods is that it immediately provides the linkage relationships between SNPs in the target segment sequenced. The strength of our approach for mutation/recombination studies (as well as linkage identification) derives from its inherent computational simplicity coupled with a lack of reliance on sophisticated statistical analyses." />
<meta name="og-title" property="og:title" content="SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/12/2801" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Single-molecule real-time (SMRT) sequencing generates much longer reads than other widely used next-generation (next-gen) sequencing methods, but its application to whole genome/exome analysis has been limited. Here, we describe the use of SMRT sequencing coupled with barcoding to simultaneously analyze one or a small number of genomic targets derived from multiple sources. In the budding yeast system, SMRT sequencing was used to analyze strand-exchange intermediates generated during mitotic recombination and to analyze genetic changes in a forward mutation assay. The general barcoding-SMRT approach was then extended to diffuse large B-cell lymphoma primary tumors and cell lines, where detected changes agreed with prior Illumina exome sequencing. A distinct advantage afforded by SMRT sequencing over other next-gen methods is that it immediately provides the linkage relationships between SNPs in the target segment sequenced. The strength of our approach for mutation/recombination studies (as well as linkage identification) derives from its inherent computational simplicity coupled with a lack of reliance on sophisticated statistical analyses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
    <title>SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tRcXPkvjq71TdJXkvw7MTOZoGf14I_CAh-h_MXrGZjM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460193","apath":"\/ggg\/5\/12\/2801.atom","pisa":"ggg;5\/12\/2801","ac":{"\/ggg\/5\/12\/2801.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/12\/2801.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2801"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2801"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/12\/2801":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t27TCNMSbMv8l2\\\\\\\/HgtSfGLzsHUtw37iyxzIhARYquRW\\\\\\\/9PcxdzJm4AC5ZrpUpun4i7QLyiWWN0mxYmbdsISTweWO\\\\\\\/naM4ZxiZ7WQgR57kOUkqyL1m4ufqS601X1EjkKy7ce9mjKejJ5qTT7MsQLtMx4geiNpHwtfPHB17iOy7GRPxNLvQa9CzKRRCngPdfz0VqIuhcTEuwws2khB0Wn7vspLA3EkpZlgVb++MAFC582pmRP+rrVA1mL6N30GKliFEvyyR\\\\\\\/GXgPzUnG8Pv+4DCe\\\\\\\/oVjORnoh0pmN3tch2TLlBx6tUptyEewHJSTn7aDa\\\\\\\/6nG2rWAVk8OCJ12oHHKAT6aJy8I0Z99adbzz5p0XDF294phaLli4GtVyoieCFa3KE3Tim\\\\\\\/vnaWXZ3zpOBbOFtIXbf1Gi1BUMv9p0Jnzhp7yc9i0VslZl4MuVphwVkELoC9l9jPOftx\\\\\\\/WaJkkCJbobi4J7ImIKweqTvOTNe28vcR4QqrvH5DUkNnRq3aB4VRT2\\\\\\\/SELJPeb2wlKyqzqWLtqmn3iMhhxvD1WX2e3lGHZjRWoZDJCL2ijQAgVwf45nibksujgYgLP7oFWf9ZXaLuF45YFWePmLo9UN1ffWlzjxXbY7KsHLG9B05KNkselDgM+rCzYROO\\\\\\\/z3AKOkUE3yJWFuSTlE8OHzBqVrdxGPsxuxbAorvxlEuvNj6bc\\\\\\\/ZTnZtyo4WLo6bBenurg86P\\\\\\\/tQ7lNQKbakLEDmpZjPYi7otyDe6Ab\\\\\\\/S59Q9iYrjMrhevs60CEh85UiZKDY1XYpYWdir2HDJjmbqVsnVM4qG3iDyiojAUmv4VVYk6198EUGXqNU4d5IMTL8mejRVTvtIHNq+OlshiC7y0D+ahBYP9mOQsxx0fCUmAqzGteSFlMT604\\\\\\\/opwCQs0iUFMaWsH6DFLvirsXTvniTEIfL67\\\\\\\/1At0Zl9QijyxM05nfjDuyMorLhds\\\\\\\/aMF\\\\\\\/\\\\\\\/XOzk1hEOp\\\\\\\/P2FDOFQRDqeWLgufojHwkUwH9M8uT28GmUMHA+d5CuyQkrK0TN0BUd9DcYqMaHzQgu9uz7XIy\\\\\\\/gL5J3K6lbdfBEbL7WOIuB3Uvoiun99mUMTMZ1lpEMt8c9Ntbfeq\\\\\\\/24XDUHDc+WZ3ChvojQBQzOLpc1QQlAuXqWMTDzNMpRDMw4aHV4s2VpUnVGorrIqqH4cpM46hkXPLg7alRly64Lc5FZ4f9VxCGca5jo5QqDiOd54BZSpvfQx909wrYIM5P6Niv9+KU7RwVEK12+sGzeFNxJpXGKJiT+BXgfaeuyKaYmMYB6\\\u0022,\\\u0022iv\\\u0022:\\\u0022c4kXPpKM3HUmIpNges7PQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cd122831a1b168c9f8af01996ce8d0ee2c90a9dc4202492aee0a9b2e1a0a8349\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mixC0A5y6Nku7j88hOEzmHIJB6Tfmvdbacs\\\\\\\/NMajk7rLwaSz7p930TUAjdyJdVsaKNDwRSMibcG8JoG0sjqTD1opO3HFNsIPpZgk4bjyRzBNzA89+8ju13sCKcUpLp67q97KLNGxi+kvx7j5ANako\\\\\\\/+m3yPyQZQLrBuEbJO4NNSbsYGDVCkcBhVtQjQ5ejWJALVsl3n1J5PcD6Dr8a6RywEal9jnCAh3a4QL5CkpW+7uE89YL5s+3D+je30wFPHp7QG1xDvW4f\\\\\\\/JOcL5wsDY3n6O4RAZV\\\\\\\/acmENTRSOAj4ljM3xtOnewNkrC10jmIj6ihK9r5dDDEU95owhWheHQMZGp0iSzntwVu2g4Ot+cE9Cu6sdzbIWM05j3g0piEEdusqkA9esc1lVWiSXEwDLeQLVS9myIXXgZVQf3szNN8xqlLzG9VwsI2MJdbtQKTifPnIDh5\\\\\\\/3KgOYL2GAtyQjWZwfiuEh0jhPyoVmL1cOSb\\\\\\\/ovIZvhpq7Fmi7h9IZID3HZIONaD1ag8d0ddHkO7FU0Szd3I4odqrj7+BnNS5F0crCn+HWsZJYEWuVHaXa5CK1aUeSlsK4fRUWO+2Xs5kRv2\\\\\\\/bnqhTMbzbVsqgWbQwMq\\\\\\\/yzMdSnTeGsJsJxZtxaWBKDSztG3QH259Ia\\\\\\\/fL6S+GUTDzxu4ckUswVDpBnrDwKFyBYwFYQjQawtzel4\\\\\\\/XnekytPuFZd4a4NwuGGi1x5glavTuSn0KHjY5MvJBIxoIU1wY1X6qhwvRvsV7oDN8rFoDIUH3OrpFpU87tuaN33VhPUNFsxdSj15mHaAo7kloh\\\\\\\/yOd1Lzyk7w1wR5I3kMiFPnJx1Sy6smEq3ZphpU\\\\\\\/ODjh74XA6wEtmVVlLMV1Oyq23yfP2q7z95gBkuhiBYpEEAg00nuRwy+3G6IYkb7iwzbF5CNbYt8ijrYh5GCGTRdhGrC6m7KYtvBU+AAybsbeFb58CRIWv7OsDArrmUj2yxhj8eT1SXOukH21vG\\\\\\\/9DZqJhLInqTOA0TdYFPkO\\\\\\\/4rFDC5aL0kg9CI5npUoETk4pX6mN5gdmfdxXUAX1dypQ\\\\\\\/WyKcxzoy3OQh+jV3CzGW2DPmXDy8jydkENCWMRkEiKhpjK1eesjcp2A72IaQHwghmxAhDKGiri9awmYFaCWV+X\\\\\\\/gM1anT9EnLWrwkldpZTahFwl9YPn2lcyydMhXNmgxNtFQRkAXj0lWKW\\\\\\\/7C+2Oih\\\\\\\/BP7Q7zRiexyJIJCiM95LNg12BPgRVsL9JPGKJsOtyzcyRU6ICjKMx5y2unQOkJ2Wqs8oJ7VgULxYnd9Gwt1zrywIMAKOVj2ROeF0NuZlL6FBU4OpaaBsk192IHFK63KP\\\\\\\/fyt81onfPChxZ0WeEaYGc2gRrMf5d6UKFXz+gehgzaEmJnY+aT21E6EcYnv78hqHfpO7JoRPWwPtRfIA\\\\\\\/jHcelvP07NMHQvHbP6qG9EmUhcpE5Hyc39MO8eNx0Cjl8LAQ+a7yZ2uj4GyB0NvGgwIbZrHgPzDvElFCo0k3IHy3uKi35BzS0W\\\\\\\/0hS8ZF0lFZFGySkHS++zHCncBRcpxKl4srgpSz1rSxac2eEHvSyXLo1VCe83yIz+9pifDj0vUtXzDNEqUVCTGJMNehiXZhHPczBxFxQl6AhhH33YPnTHKamjTXyvLpcMxwr0XqrA6sv53pffDoVoiFqdttugGpXPkV\\\\\\\/JrETQJtT0zP6skkIT\\\\\\\/F1GotSuy0VwF7ZXMv04DVbYnG6C2P4MabTseNWbN9WoiwvdKqHVqlXO5dO+GG\\\\\\\/C3TiHn7kXnxGTCfPTY0Z+1e1ghVDD0puPBbn1T0L7a20PVsQNzmfx\\\\\\\/cKlS9Y0jBevIC7WtN\\\\\\\/HNbUdWnRbT7N1O0Xn2Rv2\\\\\\\/0MSIo84lhoZ5anPCv\\\\\\\/sQsHhClckTeDQbwfH4+c4dBtXAro3Xkva7FhqK83YURX\\\\\\\/w2MFs+BYA89+znH8gQvFrSZKCCmYZRMRYRrYheBO0nYd7f+1QfxTLI6f6UN3mzWYKRwBH1XI2vPy5J35\\\\\\\/Vm\\\\\\\/VccyVEJKmWhOD3BhdZZynZFMkuK\\\\\\\/ke+HU66k7h7AiiRT+\\\\\\\/4smhPJRtQDW56l\\\\\\\/TyEGZU0h7ojx1l++4YlGfIiz8YDfAZh+q8K5JIJUgJvp4PMVJzl9snkAw1o5YR0uwx06mKApR7f3vMDrR+D3L79XNPtLcq3FFTShEIG1tnXD9cLbjG\\\\\\\/1BF0k9tgVB1D0Ts8Fdh1V90HdihvfPQR7TaMQdxDYEWkGukVyPXo6vcm\\\\\\\/90PQonEYNz0NzPYIgjj+N2GL54nTG6af+q25St+hlKFIcdJ4YpUUzX+XYIfip31shTNJe\\\\\\\/0DdV7kYiM7rT8n1S32oILN2vk0Ee6KpJmlP6peOezXxMp7mux\\\\\\\/AVUwzpN0NnPXcpHoqQG9wKYA6z+DPIxwJ0BxQMVq9vGFvNd46KwzQ2UNy4lYIgI+hOiiBAj6rBW9w+uyPDUlQfunvwqpc11ySu6qFOU94mgfpJJSb1AKwsZzbGVJW6qgEpWWds10CdXZCRawvI6wXyl3Gc05+HEP5fw7u4o9sbv74cYkOI6Tp4vyQxrgoKdmedAdM7cQDzP47AdHjtt3Ix1\\\\\\\/Zoz2dv99tkPEsFlwrTfV9oPPqqOe7E05y4epyHPibx1fbCq3y\\\\\\\/Ox8dYPhGUnMgeRco0hMvMm6JxvNORhwVWgvDWhtxC5X1+iEzvqgwN\\\\\\\/jk44XAleHb4VPPyvHGsdsnxe7emk1TAfHoV7ox40R22ZrUmqJk\\\\\\\/E9I+K6+gW9wnZcoCT\\\\\\\/qx51isndggfdQgsDkJk4YVSdzmA7UJHoUEHnG\\\\\\\/DAhQXKM2L75TCvGQV9heCShcWdHvnph3kK1pUJgFGHyxUxjTH0ku9cS\\\\\\\/RF+GnEzuudHjcmb898ggkK39iuv6rD0If+dwqvKURmIgnLLUqsW40nlPxaHHG6b9Uu\\\\\\\/9pfFvDfffb7OFJzsdjI2zXuXCJx0yEAaD\\\\\\\/U\\\\\\\/Z2Tkh2lirH31wefI9owYP62qlTMhiy\\\\\\\/K9aJ3s+Kn2RWnsV+\\\\\\\/Jjg0rCuOHAKVEIcRp4mDQsNhYhHOGaDxgTusSe2qg1NQqfb3H7BMTwhrIA6sNvhLwO9Tx+WFr2ocEjdkT+D56p28VGAT0q7BiIE\\\\\\\/JCwihZyUUXpynrDK5fUqf2cjukXzQUmYw9trBmU5+v8T0zzTfLA0VFvB2CiNwNfgjlR37tqHxEtlmwIeqfcB19778YXrnzjYfjB49Knw9KZq84ICkYX+CyyMPKAhIWrej5vrFoX\\\\\\\/oRy0p3Qb4BLgmnPLaDWY76DdDOMptiHtqGCPL7xlYaCi4Ho3sDkvd6tlRFRGV5+\\\\\\\/zUkgw\\\\\\\/Nfp96XCCR6dASA6ClFOjBtr3eBoOHQDgT8B8Po1e0QLZNLB7EIQXbmbRbtpIMdlS\\\\\\\/okD\\\\\\\/A00Wqu4mnppQ5wEVpFxryaPmclEzm0s9Me+1xhLfkbXfnJ\\\\\\\/1INW2j3MjOTX\\\\\\\/GswSNawrdq1QViYdhC\\\\\\\/uOrM6P8LHnZ\\\\\\\/vRcdDYiEUKZFbaQXs0TSjksROvaB8ooPPe7Q72un2FtQNPCdzi9BFAmgjA7+oo7XhVFpuds3k3msK3m\\\\\\\/XVKnjutgYMnIUVIqrjtzzCX3IzAW5vtfrrM8kDifgQUdlLS\\\\\\\/vfN2jKMMDqPOapwhwZjuvoh\\\\\\\/sUAa65+tXJIA3ZWxKhSFDZCqbtMrq0nx+FDXcO4AIy02fDBQjTYpkL4TR32kn\\\\\\\/63BAj6gil+JLqGExLnVb9qeP52zF7ZFYhWskpk4UpNrXXyOfQmB58fn6JyKa36OSDsY7VN\\\\\\\/RTzR+ldzzXK6L7dXCit3jkVOkmFrSvfvzvFp1sdDaiyK9BfOhUySFPKNFD1zfOyvgX8P8MGPVusrCKJTG86l4c\\\\\\\/58xLiGtNrnQHAO6uz\\\\\\\/nleX\\\\\\\/iaj1XvgVose\\\\\\\/remO7TXq8birbBP+bFv5IINjIV4XtYY3fWgnsfImpFhVWULPcrdDQ4ZzUy5F9rCaB14jl31NaD48Cup1vTIJgzmb8TgWr1v10HI29M4puqvaXGCBavLMVNKAtpFkRcB76I3hI6ED766PZjvUO+o8S9m88sFf+XiEZW21aKZkNw6oYPtrkbGiUn9rTLiu3UMDV0d8vh5f7cZCQSK4ny8mQ+bbBgwXnCr\\\\\\\/br9VgzqUoOmCwSHD82Rt1R10Mhz3k+7aeljFZmprNwEXB+nOAMKN8TNYC4DdIXRFzFzoMDnvE6nbp4UQ3erupSE+4awqJLqcZecMtXu+fD\\\\\\\/+Kdm5WZb9MLQpn12tqIWFH53hjAPaKDhFHAB7sOpFN7CJ80Nx1Ao9WKCCLNgjG8N1JcROuNqaZe8FhdHMbQur05DFgs44XZWOQIaTJXkW0QLD0XA2czdmfa1fjuRCsQZaNphYzWWLCAmrFzYL\\\\\\\/TF9uoFluy1ED116tsLpLFHJnnl\\\\\\\/s+X0ij1ciVW+hMgF+HhMjKjvvw4xqHMAUxLnz6KXXTIS9aLq8G6C5L7kufO2XAFmsM4IOU6Q5stdOQFxX3Sl8IzBzORu0MCMaS0D2m+Y5ml\\\\\\\/mFB7CdpbVLWOJV4QE5UWql\\\\\\\/VPBq0+tjui7hG20ZipFzHCD1a9yxcbzYuuFaiWhsW4ACKSPIQlw5wKvEEJAluRTONzzfDrq4nY+MA32e0KWxllYNWSj6MNSvtgO8DV5Wa4HZxHmBlbHDlOnl0Suou2ILvsPoT4ZNBcR\\\\\\\/EuAxOJ5dB4tHFi93R9TzEwchprY\\\\\\\/3GLAu0K\\\\\\\/A5\\\\\\\/xw+LrWaBb1hXyJU76hgKw9A2Zs0IQGwVQnm9NG4FcN9CaWcFyWiRS+PxyxAKvBvNskWCE8QIH1k\\\\\\\/DzDvoyBeFzzszL3Ua3r4XUGZFsIA1Wb6ImMkCexVgdfa\\\\\\\/y3CexDjWl7tlGlpX0OftZRLZKBJya9GSmTdF3KB4UtLsStyypqyY4RuNNLrRYk1G138oebaT5j0eo+8Dq01H2h4efTq7aSqq28jM9chvx+ao+8SbAfg3yt49bagEihggjDBo\\\\\\\/KtzRlJGpi6Hxc1a3ShZNeT3p6yfFILvuR2Nx7mrnck8AK+bYFcOI5pNx5lcizlVazL5IKeIxlXT1X1qUA7uaWor8r2lzst5yxk1vVK483kgG4l8nyvcNgqzAYoGa+TA5cCj8IQyTpchfxJy\\\\\\\/Wd9SaGBmJS5VnqLz4Wc549vAIKzxMtiveVjzkHeoxNIrbFfN3Kudel0ltWTRDagVs9WdhQq5ZKz9ycHJufxHe+nwgFSZf0o8Tki1XyAJ4gE+3ji9JcZwLjweQTvWfv\\\\\\\/S+icvFwGYoI24oYdeAvF+b9u2jFAV21PYoMf2vl3V3iQBJpYE+mgvr1AAPv0B1CbaUqEfLmHGHgX0CmyQYkBnonb515qJthW2+tYEodt0ksM0rABVGrTTo5ikSF6hejkYT3vF\\\\\\\/GJ93lPUuauYmjqvBs\\\\\\\/08Q5zr3WgVv2+3YUP\\\\\\\/oeWTBwirJLgjkNx8P17KmoEGiQqMYgIWcrl5pkf3kousyfSkL71uwAmvknzqLjouqnWdNp6Xicp9GqpbzHH8fj5TNsKp8z7iNw5eaEvkOjxfzG1zCmz1BDk33qXei6aklLAsYAbv1odHKAFePxt2wHAeGLZl857VK7IoybT\\\\\\\/j1J7EBmVEvCYA6oiyp6yHLeX6TVBKdKXIt+LDvLofhVWtEN6o0n299DkpAuTqdpdhzxLQ2AzdfQf7KYQowjQzeLNkZJGSvoZUKjyobvaskV4OFVhOHuWpYNloHyniNaSyRojx4dPXWZnac1Hn8Dew1x6D5Xgjt1kWWjnkJC6PCQk1Sy4SbpTyNdXqPhpDZfiIpWpSNh5f5EX8oMwz11Wr\\\\\\\/7OylWGZAKF+bizWxYUqVXQ\\\\\\\/bXgC7XOo7hXNENJ6chXHedTYXlLPYFWRRC0F3Xoi+aCaBnIBjaMN7uaTxM7LlrPLEffNAQAphd9Cwx91klF2lUOwFdTVpTssGwjpo+zSCyaw+T0JdBN0h4n24WPENhw8SeilVW6wnZrG8sFPm8evMLgoJgP6Sl1\\\\\\\/D4u5AB23Q2fL6xBkIvC9nQcZEg9bF1v7LSibRRQgFunyX2xzitW7NJFQSQCmTiX54kQcw34T++JTgUoVggqbh2irSpoDqC8fzOj\\\\\\\/JnlgaSg344T6wMNAOPNok9gswMiQDA0VTOpslk\\\\\\\/axaZB4Kk2eacvmFU6SCPNuqWAfWa\\\\\\\/wnC1HZxFs4SvDljg6cV2V8w02I8IM4URlyQDZtiz6n3LDAmEyPsepXQqk1KECD1yJVTroqQN\\\\\\\/o4XD5yaCe5UF5wlZLeLhrVhZBXoXUVh+qFKwZY0BqeNyc+gF\\\\\\\/U0\\\\\\\/me+QfCMZh9Zn538xFADusjykLTiXIVOJwFEc2KLGPdVdzanGC8ve7b2juLItkxTaViHhEKq0hNk5a43Hdp2PqJFfVfRfbWd7BCMTzeDhUcDvbttnJ61INQRVP6ZY78+izmXNP5GOgRCIlMdhcKsALWRmbLYgspWmCyZ2Rjze6XcxZQKH4tyA0Ci\\\\\\\/vp2scNn8p+rQjUhOh+DJYHwQ3ph\\\\\\\/Y73\\\\\\\/bl8\\\\\\\/nn2fwr1cMoeYUw+lkukwlPa\\\\\\\/aTQ2UZdM2sgAfKBOX77UhldwFstLjqGQ6cRDkWht8z4pvzTuC\\\\\\\/B2cQZDCxLZVnOSQXbIAu1niy2vfXZUyrVzYzfc013XD2ABu5kytXex4Kx7sT3mn530hHisjd9oCcBUSFWkCltKA+qAdVBb2uw8BfrRGygziMWcs4pa1GK30FGQKajQgh7uFxIFhsr7KrsOoDOxFMXmyhdg8HBO4voP7EaAymdvap79r92KE22rHU8+eBfR\\\\\\\/y8Di\\\\\\\/YutAmp3lTJSS09ABraG0fyfjNlnFRQtknaAWHCs\\\\\\\/LHdoD3Gp+6LoHg75f5BaMEViLCTihYp5z\\\\\\\/1F0VNfQZ4HWjSHkxN53zPziH38Nb3ldwz\\\\\\\/qrpKStM279uNt0f3qmXPhDDQQVkr+oCPod85ufw2M78AkA0AleL+N8668mPWLDNz1WwG510Qwg6FQG4ckTt2xxJuOqBxWjKwGS1mCtp1clk2ZkIvk+qZrwkYJ4xZydw2jXlzxdIQqX5BhEajMdJdIQqAd5LYTn8DUlYJv3NLbyf42aP7FkUnTUWC\\\\\\\/5w7ANYYesiNs4ir18ewWTJVqc7cHfzdZWa69caD\\\\\\\/Mcl3s0+xksjRMHkaQ15uhpon\\\\\\\/e7wvfn7Vf2MqRCyQpIwH9uAT3nHmzS0lMHYzsE3nGqL5dsM0czUZ2VXzpSgUZATL8qbMqZEsnZ\\\\\\\/Xk1oZ2PlqLgJ2R7uKB8CQb1W8M5BfpyWqPguAO8qEceBeyjCVL9gWXoSDmDns8BUxxW93\\\\\\\/QQkpebn4W+8duDTy2JziABaCcBAUmU8ndTg6tuKVbMvSK3UuOBaUmdB0P3P2l8mxAHHZvRuneFpwDawknxBC929MABXDc4ukHLpXFf+tSQeM5CjCL5iiormob3l7XDLcO0Hsqp\\\\\\\/490Rd1NmcZLo8EGpKJvW3v+2pete79cgE5oKOt9e7CQOguigqW23OAUpRoUsXIUd6i4r5U22BCcDKgYfRcbwr776iQW41+ZeDu05YPdN\\\\\\\/qQNwtrEhLmOVYVVG6kTOK1f9fpllztKjr5TnhHuN99ekhABeTDbfD9yKkW6rJ03m9TcsM4vbB0q3TUZkP2Xf0YqDYdkfaNxqsW53540v\\\\\\\/R5AtSBnIDe9pjoJsaK8ma7l7fgiB6AMWQtE\\\\\\\/F3pgptXCmgNPQ3+D\\\\\\\/S5LX6KEZQlgdJg1bh418HcPCrF5txCtuBXIsfMHXVfrbywW3qhwnAI4Kgb1y34PpJ2F6hGzRwL65jDRZS75VDALT1jQGNtEFuAlcQnCBzX+N6KEIaqrWzFxbM1QhwSaXV3GiZePtO61ld4GvIi+N0KAMOqwwEn5hZjmOfT18R\\\\\\\/x5fqhct9kQ0IGuHiYvHETIXx0wVyyy95b2NJKshgDOrEWyvtuejVJNdi1y\\\\\\\/RUU\\\\\\\/W32zRTFhPOR5SizgeufAne9mzUFX0vXIOlgKEl0tQKlzFYn\\\\\\\/vXGTd41DahKpby+TYlMmb8CPLR1IpqMjfLtEzZ9ZwiN0lwSxoCmK\\\\\\\/oIbxRz9tat7nhJv77hDQUE3aDCIksBLtGhsaUKRwe3DYck7cbfkfdMq9e3rTCopgjtXAd4twhTsU9Nkmx1Nu9xaPxB8bvv6TPn4nPaZ4ds99A9VMEwzviKECHO12f\\\\\\\/uIi4wJUoqaE67d0h5n7\\\\\\\/VN6UqBuDaTquhd0WqTa\\\\\\\/jQ5K4AemweifAZLmQ2C0Z7dBgoXuB1ctIJIuC1fSnq43h5zyEyNuf4egnU1fqgjHtfylLuo56qNTx63Y\\\\\\\/TZkeguPN59qr9x\\\\\\\/uUCIaWZkOF8E8Yka\\\\\\\/6tH8ZEejWBNSIkpdiDvoOnnGV7JB7BfKKRg3gURrWKF+Ai2WYlaQA+yD2wPxmPQasUzGF3GmFtmoFOII8bBshsQfEhwPmY\\\\\\\/ly1CIzQzh\\\\\\\/mx\\\\\\\/WQjFO784jPmNILLFf\\\\\\\/XFWfHFZfiIiiUWjE77F9JY77V3672NABAQle57oMDulcsIBRe7uKYOO65kmG74WlgjAFs4wINkVgQv4LgMBbQRSVB5KFNO1F6FuqPvk7wg4CrJKshX\\\\\\\/aqv\\\\\\\/wBGy8kkUUVdElFnI7ya21GycIIwEBJAHKByofL6qfWVaviQ\\\\\\\/KJONpls6TaEeB+0xgKySvP65UHNN4B6a+TbntLPsGJcXo8iv2b2b\\\\\\\/ukcYuXD7aDmsakaXNfsqTFURxVWxKcbS+p8Rj6FT6sR8Blnm\\\\\\\/tRaBEdhsW13j2K5MvGE9S\\\\\\\/\\\\\\\/692NEaYYjd32HSJ+DRS0yYhcdfDygZN9NJPsT6XNmwMAIIt6uaX+1YA0g1\\\\\\\/IVahnGYqcr6Oq6NVTverXiCOtjbHXNsNPIrVlKzP0cvto2m3G3xCTSk14uX8unvDCWXfPGOxc0Auf9uXb0i+owvH\\\\\\\/cZpsdd58J1KoRIfog7d+hIRo92A75TwiRhqDIk0LaxW3XL9bG9BScfuIvQO20+bKG5A2cl0AsJfOwCx2oHOqwr2AvJAz2ENAj487N5K1UqqlsY+3T+IQvQ1jwMOeHFtGpO\\\\\\\/G67hzHNHdEaGFD6iJDP+KUfwCEslZ+SP5h7\\\\\\\/SYc0WcOcojeiClYwZOjdK2nIQekKB+VAjOrH+wue3OMn\\\\\\\/IytFkccmA7n9b6ros4GgCkpRLTX1Vp2X2w0XGJmcF9O04miOfcDcb6q4D43kGkZ95RV+2XQwtHm4q3SqNLAAdPhKVzbVvqpK3wT+Q71UxeEnfdbMo12njbVzAARnDMlRBYf7VhMYK88d\\\\\\\/UQEZYAshxzAtzq7PQw9cP4pj0cviSx0n9HLyEmCIHt3D+OKBvqA5Kc89w06oUnRfeSaLNU5L5OcB82V6TD\\\\\\\/zOB0YU2lc7jDWCvh0Gx6OHzuOdeh7xaxJZ3QJ7BsRghgk9QbcjIXRIPEGAA+9lg\\\\\\\/Y0ZjiZdaxSnQj5RWpunKlx0a6yAZU3QdbKPyBYOg2S+vf8ZEi38PCFgnySpDGyKoGJyyKZfN8KK47hwJMAJP0HZqdqwAn8J078\\\\\\\/Sbnnl\\\\\\\/cQgcdXL1jYjSMnlIvkpGhzI0O40SUTKUAf0b8t117ibVYd2BpcPnRZeySFAnu6FLR6CgGVItWV4AwQi2v7M1rcgH23xy6Jw2BKu3apxmaanlEiUPw0B0V1FlN6i9Da+5fzniMWfUcGUlA5YWhxvpqCWC5tg0nknh1zFjaM86xdKJs97LKuBbakgSau+7bL6g2kHGwvjMVK2UgusTEuJlUYuYKQohgPZmuESnV4uXE2O3k5MJK7\\\\\\\/bg+mLiSVUN62eVH5YAmykk6njONsqM6eX8RmUcZDFQiLASfP7XCE7l\\\\\\\/qt63iAKPNUyh+mqiC25S5JhxMQagSj8EsxW\\\\\\\/w7Cd93qGVYSY\\\\\\\/Lv3rVMiHhKx\\\\\\\/ginfBXBBeFfxHrUdNA9ODiFpKRRZ2e8DsYeKOis5+Gzp0kFU55ycuRoD1S1OXuzdurGXsKnEupIeKLbpN6NqSb3XZHL31x6qgT0xblVz3lpNwsxxBHgxElb8cVDtwZ9I0wVeBEUiqxy4uOV7cSwFGL4pUb0OXHvkWCkcGydN8nB3W\\\\\\\/UDDyyx90CFYNasP+9+ogVqf0w8Cp4qGgohmEUI0Y21iHT1rc51u3XbcWskeSxuOGG9vRDPcH0wB1vHsIvjjV3PrWwiTTGxuYJO1iOePONfGfmslAbOHjsOOa57QKKDXAsEFXLNHg5l2dD1vUQNUGzAgULSWAJc78Z+tsp4i6MF057BvCvsSXmZ\\\\\\\/a+2+4MOeplnp1vU3rnh8vzbPhX6N+4gG9vkztAN0G7TE3aJ\\\\\\\/wb5Bi3To2x79IIiW06pRxMqhDIaG\\\\\\\/xEyVE4TlT7E++XsgHBUEK16OK0PaAsO4P\\\\\\\/a8Ft+xYuxVgid3afsq+gb+EXoVFj++vyFXCI8TrSsbHiz8t6AIPn6NDdk14SmaG4Pu2ZSXKQCx6CdgGTUKjLbnOftW7qdG9grhp1cNCdLmITxYJPS1obMaRpD6Uk81nVGP98l6hdyrx1abNPQPmU25QJKmxx6p6U95p6DZ023SCS81XC70FnWX1d6zVc+w7F6Oq+cmz\\\\\\\/EHNCZexERCdBAjBSpq4C\\\\\\\/IHIsPAJpZWTk07LfUO0uRKXf41ikDOcSRcOGYVxkNQHN11b5ug5bJthlpBBhAlkdH9TePGvj0vi08VvmUfLsYSh1nQ8fR\\\\\\\/afkiVmcGrjQAVgQWcY\\\\\\\/vFw1\\\\\\\/9xCUvcvFQcMU9nyATZeY7AlyXgRh+hULOXKLb4gR+vC8DOHguqowOj\\\\\\\/0\\\\\\\/ywRQV\\\\\\\/jUDj1lV5UfmBAv40RN6+7YvN8n+ZlrjRjneIROvQiwJDM2ecXGX3m+jXUXSB+zaN0vC5R97DdgeL7647vkLSVjKsK3wkfegN2HSO7tExQSmemypTkUd\\\\\\\/Dp536+jKehiNfr4iIvZAi+NeKHJ+lnQlD4KmKVZNJh\\\\\\\/DnCdcKSphpqSZ6CKtuj2LJnkZksJQrx\\\\\\\/TGHreHxkTnlC2awLo7JAZX9UjANnNAE\\\\\\\/1xwGW6KDxr0JmW94RPFqxzCKImnFcC+P6u1lZpK4WvNfRzGLXsXtaLzn6l+QuKRWvMaEJwkI7NjeDgMxNYR2nmWhpHQQDzns0SHuBjPTPLZjAnBcwhRciCz5bRvw3e0zV7EinHXHHQIUmXzinEQ5JayLUDvgwOL9X7vGXW43GqTdo62ZDU9cYCJxqzjtYemcebOcOkXy2EjPUUcv8DEA64nrogD3T5eU60QaJ2waIbTRSf3OpSTbFXMjWdNKM7wkEEz+q8LFq5e1clzLCHZgfR2SmeWBuJNM4VRqEmbayBZ9YqCwJ0T9Nleu5pErFkbTMxDYnWuAiT74ZCmP7HCyLGSULRMyyRKwtYGaXCNg4jj8XQutb6kDsHdQdxAZWh\\\\\\\/gdgb9ZqdGi1KGWGQgbZLdJLjWOyc7yF+Y5LANbUXNwfZbnPyW1YzD1m56rubom\\\\\\\/FmQlFirWO8K0hHqF\\\\\\\/yWt975Iio93trhQ9IQUhUay92gTAjEpDcPdlDPHpW1BZlRUrIANmkMAS93n0TvNYdsT0kuFGVVNUNiG16yujGMNwttLm+CZ9iKvSIcTVm9tQc9iRHWnFZR1phkkuMHO6A7wHJs1t31HZ1WkjDyGBYWVPbTYi5D6dQ6eLFpvUV7qqt9LSXVBZJ1GLLpL0WvpcFDIeUqsVixEXJ78NUBVKBB6BOl4z26Es\\\\\\\/5IJkb6hcTdJMsqOwrM5VlZqopGapWBJuKyT9gIjtKqpWKYQ6eBpKznTJ8VO0ocafwAxajoW+x55LO5KsqhusR2NENeJ7e01LzQtY1Jvz+VYOYxyE+8g6sju6PmlFU9fTpaCEltYm2+zEa+MS+lS2bDPFKeafNXJR7M4y4e3X19+CamQFmyu1ngHghlXvKR4Z1YAOL8zuJWiUuymsVDGupf+FK0HcQz2bgzs+TU+lz10TGJPFfoHDo4Cgt5WX28ySrzjSCRb\\\\\\\/yT3Aup8dQ5UcYqNohglf3fw+CV0T8jebRoVu3hwrc3PojO91b8ScSpQzsaeK1zeivsJnPNS+p3XamupdUE32HBLfT9rA2KrqEyRQoddL3v4OAtXCA5wzYybXjDcykmHEoceZIF97\\\\\\\/2h4hvxY7Pb4Kq0gNpHWNEqICt91h\\\\\\\/myxuxuNyOcX\\\\\\\/23RIPcOPS0nSLzzJWeyFek\\\\\\\/QXX73D95As714xB4SrCDfSbRmohIR9BhpqeCB1DkFwDj80opeQ6zkuUHzRQgxE8SS7s7ioSGGHqIDXG5aNfM7Y7IUzVJgF47fE9Wggm8pBnkMAwfF3TlcvIqObjvliHwbIYDkChk6cDY1flNWRzmi8GMhRaOL0ipvzFLp3B+Go78Hh07joqzEtFBaOPkEIYjIYRZJMFkVR1KU28TgNx2z6ozdN4Bi8qREYcqm3obkyVxDm4awWgWyGiYgiC+5+DTH0zy5Xzpn4V5nrT\\\\\\\/C\\\\\\\/VAIyP7mt1TH\\\\\\\/3k1ToChbuvYKWLXctdQHSHixjxCZvu1PPg6hOEB\\\\\\\/Od4yr0LQTtSu4\\\\\\\/bS9YH8TRb1Lcr34p2qVvaDBtlDV40aMGqp718igX7uSmD1pLHCvm89zJKdWDn0S7U3ZHP8rSwObJRVEgTVgL0RxuWduB96AFSebjgbRSbnAJMaGxFYZTgxDnw6bVfzuL2nRroTMl5vfLLdN0dOzn2CodKrOwubaEyeMKiV+MuSLarxLc5w827nXyHWs0xap6ekhFGXfF\\\\\\\/DwqlzZTHv4e\\\\\\\/rso6n3LLAEUS53oo4x9FSy8C0Kymh2\\\\\\\/uM1WF6YIQbuDDcC+ELWjqNuD5YTxVqp4qo\\\\\\\/jNifmlmo3qzmooOqHTjtxi7\\\\\\\/DaI3zUikGHmCHPgef8WabESJZrGa3fbEORXG79RH6ybc1EOX1gZrJf1tkO+bS9qFSIDDCct8Sf2odXvWBefMVLusAMbYSurATnHcAojywgNYV\\\\\\\/EOFEpwe5qBg7OE8kqLwuc5LsK0apct1qFwoF9OoEQgts\\\\\\\/VgRtxUlJQ5IfWowGZ9Q1uA038iFsxLS6SSs4eGwBMaZA57FBB+3K3Po\\\\\\\/y1aTPAK84iHpTTVOMS2+uCaBwbo1JBtsvm6VZceWnEotMJK0J7ADBkAyYohbYdcG+Ha\\\\\\\/2HPpFvBg811J5imKrzDR1bnrRi1+0hSI1DePO9g+m3KT98lAodAfyaN9cEdzbe2C7EjbcY0dpuj0brGwyR9WFAHzv15+VXZM\\\\\\\/8FQQPStld+EfZy1pgyj5LcpZVUrBKPI8LZBdyRgTmGiJQ+QEMVM59lgjfVx0kN6pUJR8erRAzxMj5f1y3SUg+QqwZGtWJFcLaWpMDp7y5akmMGJgpRetnwgMHifOltTXKWn1jlQzRev0S5mxSEbDyuP0Eh6\\\\\\\/fAgGL0ARez+ZYW3RnwxZH6vRSVVOYCp+USYZk2f9HNnT3w2ih7xBL4WUheQKP\\\\\\\/nCAXU1bjA+\\\\\\\/xqP2\\\\\\\/yETd5TeEFzjfJJsyEOG2nmWVLzSkEqrW1azfm9QfO4YpF6oNDsd2HcIMg11cSBbsD+YBiWllHf8AJ9nYmi3CWHzMsBJMg07Z0qZy03HTGk+u5Tm7DtYiIkZUHpEkyDkFgsvZlh3WklGMAsgSbyImme+ufSxt5BOjYdzGeLhxbHgtwzR\\\\\\\/rlQKy\\\\\\\/0G+IVJSyRJ9NokDFzTqaUF2miQJzjF29GZh367dFs+jhEiAxRutNaKwabjiztjzI2d59IHtzUqmtnGUgwMA\\\\\\\/BN+yX\\\\\\\/DUwm\\\\\\\/4ue3hIumZREPx2JqhTsEwNaDOUkWYFosMpBIiRXMhvbB5uhmzdizpEtxU2zLIJs\\\\\\\/zegPNHpIAa26gA5ykaD23aIs7JV70UEwFfwEEw3XnImeXd8GRc\\\\\\\/mos+wJrQDiV\\\\\\\/1QidDMoGWpjH6FNZNo3UlbNlC+V6BlpFFw2h4uHOsPH+F7Pg0zg\\\\\\\/LezxFgH8VoIyWkdemcdZjR3keo87R2RjjV75HMCV1sDA0SajLP7GDeqdMdCyVITjrI3C4bH+vOtiVsS81MTjMHg6cUEKdaxvVQIlIg\\\\\\\/lsMmAOtPcyzhgw3wB4mOnBkra1Y6+3Y7o9gpAj9OZPp2ay2VXYpRjRCo6xtUojTMCTJL32o1DTuiRCZRk7UdXxhYa+KVFoNwhRJ\\\\\\\/Fd4U+sGJ2usUnfuGERX7zXY6ImHeeBxVwcjE7QnPuNYMMg5DOx43xmFvdqeTwogfGx5Gb+BxCoCjSIvqtGZuGFlOIQhUv3vEhW2jReao4+5odTk0Z5Y2Dzllzaupqekp9wjLCtsTOCvtDNVn\\\\\\\/5\\\\\\\/XyzIbhpmI4iBVHJQSVyBj4jlhHTIsRCvtkUWdptYEnfHClM0wkd2tx8aYUc0om5HfUR8YoGMLHXwKQsZ6UTBqJTxQqPs9P61zQa+yM5D2tMToe4ZqOwsuIjslFaF3EUgGU4N6o+3n4n\\\\\\\/6c45D1reNaCt1B3\\\\\\\/eR4gnbRuRzh6GiZ+YdV86SSYCk9lNAkGELlGq7mDtd9JP6PtwSKUDXZyNVaxnNH6HoikqXH8QgAGxo8JaWaxCuVB1VWYWB6g6Cdihce0o8SWewn0Z5EipqvrYZrw\\\\\\\/rOugk2nnchLW2xrx7VCrpyWw5Fj8v1tyknD09GqlRw0sSBA39op3cqAQLHPlpO3vadvfR3Y9IKcDcVFjCYHAm3syPcW\\\\\\\/HloyJ0+zegO9H3h\\\\\\\/FaSubw8MmYaQB5yJWZvfd8xDRveukNCg7iGQv5IcVlYco9ZvJVVbR+5VtJ7PVOVu6nLLFrlQbm\\\\\\\/msBR8l\\\\\\\/lk3QnVVH2euAIWuJJSnB9wUnQyDYt3Z4c0JcdGrARTjWv\\\\\\\/wfViVdA5qgvV0Yun10YCa7HOlDqDGC4mu9R3p6VogUfaNPmUcWNQHd7H+dslS3dsz4MJ7djbfz3wRA7647hn0ZVL1+5hU2mj3iU9L0fBpgmEUUXKHPxHyXXcjEGUm5Ir2ezQ6\\\\\\\/PRZSLlw3WpKdcuWgVg\\\\\\\/otwaclvZqZ3aOweXRAricyLeZJAPOVTlLUd7jckMF\\\\\\\/+tEtKLwpL+hE6M8h+lPL\\\\\\\/RMQwFPOcXjeFQPSJA+ve09vGSfpxOJ03JpAcMuxT50JyRIQFdCthXtEeGkbukF7GBlbi0Ip6yEefeJT5SxPsSSo+fYXMWx9Tw3U0F1izXQeLdViJve6CyY82gcqurAXNZQd60n\\\\\\\/LM6hbnSTuWJuX8904vqbvA0suVr2NYbmMAVhcN9GTwwDjvOcWCZkvpjuwsdSRrv1k58EhWqD5QaeW71lRWNxO8HMoUbE\\\\\\\/cx9AzSrP7db1TIOu61JO7dj6i9SFAFtek\\\\\\\/xyx4Le6Lf09yfA14SKF02YHZu5Y0BueeQ52+yEw809lGrF8jZIzuvllfijGbGFM+9xJS2jYMb1recO9XBgJWhRmbMPJdvKD8DQq0ZkZcjIjIq\\\\\\\/c3OQrays9esdsFL28fPwEjbTk9V19+ZsG0J3YFXaTuUzQBwj9KOACgWxzLabgNWL1k+XjH9Ce0Sc0Oh2\\\\\\\/2GHn3I5d+TxNcdo7tRwS42sVWpuhgrMJZE60+yde0YBasHtI7smgGj7b3ubjs6WcpK0R3dbFcsbl1g+6Lv\\\\\\\/HadLx\\\\\\\/RNr948tHXwL2vTPM+BHFnx++4xapZXeMmdis9bU\\\\\\\/vgoljhJA7WtMlbcbxGyzaf7R99cjArlS4IPCwAWd2OI6vXRkL\\\\\\\/\\\\\\\/Djng9ID7iznk0S0R4xZoUAS7UT24RwsBEW0gXrBRY2XqlhHzsmWO9KrSvlpGKXASwNZlXvkzvmbhicrqhL7extE1GfXoFbjRGGrbj1I6QJlu3PkFM8tXytrS5q04\\\\\\\/Pa\\\\\\\/l036DwW4+ntfAa40UavvhQ2kmb2Je48f5l0kbJQxLbY5eUCYEWj5jNttQU1J86Ow5vXjG8J8tQmYTDyltcof960QMuRFowGinkhMGuXiRoQN7rbZ8Y4SO4ntQIgkYXcMrpnMcA3+\\\\\\\/W+OdfoFm03NMBHEcipG7jOccDQvvV6kkkeWsDCWsXfMRU9KhJyqvqaAT12KNb4FkxJCN0Yscv+0GlAWBrhDybvBDXxoGVC5A4gCU\\\\\\\/wgeYUwOv8CAkH3O9hO4WoCoB1\\\\\\\/+Q7YJybzEo++8UmshuJhehjBF8cdzBWKzdEqimNv\\\\\\\/bowx7vKCeGAakvnqi1KmDPd6rV+60rT614uuX54hEVqEAply\\\\\\\/Lxlhgq+9yKqlWyit\\\\\\\/XZsVuhC\\\\\\\/U2YX1XRUWU+OB0sFk\\\\\\\/9XH7869HoYaIdES5LBgRg0kRsWotgBr+ML\\\\\\\/Zuxq\\\\\\\/Pu9y4+A3eVioqaQpx5h2VLrwLctAk+GILR0DkKe1HYUlI3402KH9Kqyq4Pwzon527do3jZi2q91DoDbQGCNEc46tUpAr3MtJ7YGN1osLspdTB8tvDRTBY2kC0A\\\\\\\/QHgm9OOt7j4CXT78wSVsXeux18SbVq+4\\\\\\\/3GjR1NDbU3bZhO5tzAhIFH5ypLZQP9u+u\\\\\\\/R\\\\\\\/aAmMu1GyCGfXUlysFQJjLlJdBry2XzI6xdQc9JtH6IL\\\\\\\/Tn2PDdnJFThILgdbCA8Pt3djByO7Pnm1XOEAlrhnXRyK5pcTQhxl5VZLR6cOQEh2d2kx47CXcsssAenOQlYuiYw4KGVOWU5QA0ATfXtbNPyXdmAzR2pIgiELvwrIc6PO\\\\\\\/4NL8oDn6x2oolF38CUB228o0O773COJxufmXn0KuqChL1o0dabWUukQgJA\\\\\\\/7RQcN5skDxip3cBLY+JXefMhsDAxVX2jyCr1r6yOkwMeOGN7R5dVAcq54uVuhLNLLSCP\\\\\\\/97WnkUXROMHeSi73Vjahi5Mpp4SC4ez7qD59H4CQZttJsj39hEfvg8BgI7mPJvIMhCp1YaPwwjAnknkV4GcdUuGhc0eLPb1vhSqEdbVlQGu\\\\\\\/JdHH91pn83KwJLfaZjAEMJL7X6fJFNG1mxcwi15LGhEGbAXd\\\\\\\/ckemBcbM8q+4mEj4UQccVXV5UXPf5Zvj47yzAdne4JH6aVJdbnt\\\\\\\/WrBTe2i94XYmdnqD\\\\\\\/5AR05ndEr2pLk+V71U3hHYLADXY5k8\\\\\\\/+j0iQkYEnqhNjI8pgiFbWDjC+SE8cmqUPAVukjuSBUWEkYAzHJb5iSYcCoB41KjvfYHFFiO90qfrdJIBhDYmUZRKXJi7m7nqJVy32d+Y8eVgM\\\\\\\/EafVwXoV2Yba7vedZFR\\\\\\\/\\\\\\\/7qrAVVnhPkRH3Jdb9+rD6JA8Ear8bzYmCjIgyitz5NcD6AyQxsoh+FYTc303w4XTsHzH9kgUH7SvGvERW6EBsA3ax3eQz+o3ODQy42cWWv5GOwQ3gJPq7rRto\\\\\\\/uKrI1VJFBs6EqaYyb\\\\\\\/bu\\\\\\\/L6gIZMVEAu6Df0wdvY0k3lCyVrpJgEPcQ1177UAg23SL53840Ldnd23UaQlsEO4zx\\\\\\\/qaNExOGq9u9sAzFeWiiKrqsb6CUSwM0cr5mBU1xdaqdySfHFTYm8N+ZR67KWdrpQmDKz\\\\\\\/PQrwgjhGGp1MPCuWuK3czVWc\\\\\\\/EgoDeQa7aUoD4PhdmjBPZFPndl+1+qUuVzIVy2M1UiOGvqzMi5VczYd5CbqWS3r4+ypR41WLj6dDpRou9oiYdjtsJt9MsQwXrx33cYx3LqgkzHFTmXcWuUg6M9PUzasIbfQcmIakVARBrOV22FLwHtMRotrawh2m7IJfW3c76vq\\\\\\\/gPkknDLixJEjpsBxJ2Mpi3yW0XmnzVSkTvhsCfk4Qy8\\\\\\\/2v6\\\\\\\/8ayTe\\\\\\\/dI4wqO7OFjmWLQdIf\\\\\\\/bDOHu50rhCJGkb616sL+NxNgL9eYkFCI7aEp8dCRJGvI3AOO\\\\\\\/9mnS9r36z99Wcd12u3Aus8zcDQ7tZcJEte+olTLBMLn8f5kC3HGWl8MR3YUZ60a+qFYiuSoVP3aRmV5l1\\\\\\\/nvEQ3UvZh4lnRNZFiHN+2LWWS71B1fSuo5BPiDtDuP8quhYin2v2MONYUbPuxUn7UCzCNtoe\\\\\\\/7csh0AYREYx12TcAq3FfRKrC\\\\\\\/9MC2U1IkSs5YXZ\\\\\\\/HGW\\\\\\\/KmAQbAm6FXrYHQlo8+3J5QhqCx7AAHEMCm4sr2nsttk6pL7huLmYthORWECyZbXtx9Wp9zDkgWfqFAKRdBgtX3n3rZpG+XYoRVNFXPdt3h+lQf5QeQQqiubNrwDr7KuSo939KG0PRD1VHgKL1s8wg1HllfnJd9+PKlpd2FVRw7SKcsdIqJ6pp70PVENEEmhaBVhHn2ecWnuV7Ze8amVEHvAdYLvzK1eJO1FVhgqGlp2WFTTNQoBGZtKzxEt25t8Ch4t3bM130nwnGvjqPANrAT65g+d5MV7i18wNyEc7UtEWYeBR9iXQiOd6Vfz7bkCgAdGt\\\\\\\/OZxYEsbGmr9pvOq7uROm+lu25LYn8vznQZ1Cq5ETOKjd1bUWGMgi\\\\\\\/0tSv1cAwnqwXMvY7BwEfDOeuI3PXW6AfZGv1tR5uKszwVARWmWd3HTRRSyjy6sWBnzr1tUIe9TJMiEwvphgdRKw+BGonj29WFtbtG3EE+YJ57oRTJHyZ\\\\\\\/JbBNtbEjeRlLjiTEvIi5uHZezCktr+GhzIInNQPb6NtPtk7+oznpi8tCrepw2gZvpcrZgCdsu3qoluJZO8fWs+HUWQeSH8ctSPb9IoWWW9S5SwO22D340d1Yhxmf28Ur3XynpQSRJQUgUm6XwOxY15wSL3U9PDALTGdz8HlD8o1l61M3+oxNLRGWHwAAsro3CRqLFi0c6nxPHZG9mwOBtpeSyqjzDhiw1nHQrxDmRcIbWSRnyiCgWmxh659TgfhNCqj2fuxpq2Jbczv6RYszSeyWu9ZFfPLFcDqLv\\\\\\\/N1TTT0zQ3dGjDP7xDYT0hws3D0OFpV8kj8a08WG+reFo4yC14BmWJ+c7TWi2Nc2999EP4iSG5azUD9gKKoRa4uvbjJ4b+H9i38Czdc8AgVqahnbUYje9i\\\\\\\/jZxDshx4RooAPjXFeJcj4UZJspaV4LGaDh9en4qQIJ\\\\\\\/L4xotVRzy3ky2\\\\\\\/pyTiqGW2Vv2pFQC7vujOka3VAzM5TIYX1UJaC7o2xmsBEcAjL0W8PaMLlCamTW1GTK\\\\\\\/d7gvm6e7tNEDsRAvszRfHrec+pJ4IRXYBWmDOPIXoBsqqz3RyOvvfUjqsKd\\\\\\\/u9QkV7PGd3X7eCCp5ojFsYMah8UsGVWmRGFo8hEaeKKSZDfOHOGdFSClMV2Irw5xxZc79wa11f3ixemlZ6GMX2Sc\\\\\\\/bHL3O9Aqh0HVwsc5ekOPpOI4eJO3UIyjoFNguRV78ISMgtXUrggKzN6Zi9th3pEwx\\\\\\\/eyYOErvhyY6dWUW84yK5u0RpM09VAnISAOfba9Ze03dqvp1HFshg4gSmBXSw8rKjuOLpZtgMJQGLTnlQDQF4EsENfRo2SnKFaQhsFD77nXvLnLlITe3HgF2W5BPAsn4ZXysmVoeF0fLfiU+yFWyW3y8T1lK8A6cLpxfzkmIyhIB0zXbVjjC1sffECPx+\\\\\\\/cBolYmFDYPfsKUiL3Kf05DmzAO\\\\\\\/dkLkMhu8eT+rQgWtprMlxgzO0opVDnqIQn9VEYYXx1MdnDq81YSDyRyMvckVjMbdkOoMMwY\\\\\\\/qY3XePcLZ58VJ9ua40g4fT6Aih8SodNs7M2\\\\\\\/jHW\\\\\\\/BiLhMOMx5FDg1wlYhLg2EyavokL2BrUzRXfi+pkRAk94Uj8xv1NqbLWxa+kKZVHJ7Rd8OsI8O4EOS+uT1Z0dXrWAt58WMzqj3VTN1q78otAiz0NouobhC5CCmavOD85WWQS5am352vSayn6NDMAsneYuE6VyFTnM\\\\\\\/EmrUb7cfoBV0Q1FEzjEl2joStnQCPDgBkkyF82lxzzMFAnuo4cnlBBHk6Iri0htJadqq9Y+Sym46RWDfbPRA6d8bc58Ewya4F0Ax6zyz2TUf32pCkSrF2\\\\\\\/bjfJeVlL3T+el+0hphs1ncXEMr2e5QPo5NktQCOIjD0cNCx8yBwSdWVMSNBWqYJfChbkVVBEFlGj8SpB1TsdcU9wwBte\\\\\\\/zr1XkJCY4LiB9fgcT0jKWX0VmOfRjmWTOIK6Z8tK0dSwp1382FT4gxYMuqbFaLXb9HEw37uMs4toMux0wVyWOo3o4T6OynG1ERYodEcO9B7j+u6RAUuVJbtNADIRERYLSCfx6GB\\\\\\\/1tW\\\\\\\/gw00HSBtvFS3v\\\\\\\/BwptdoLbbaH1A+FT3wutAvK5icQAbCiIV3gkil3Unp36ZMB+aiv7MF4a0zj4GN\\\\\\\/6TSKl0EDMKeS6Vg+5N6rfEzDVlsw410uAEh1+sD9QbbEV1ROsjjRRKuMlbZnfPFJ7XxW34yeO0lF1fLDNV\\\\\\\/JGtv1tLoNhYyx\\\\\\\/A3EoBxTr\\\\\\\/yYUVNXxIsvjk+E0r4atjfkVJIJ9sF+OjGvdgqDevdLew394JxxN0uIl878wQBLWDhaFgxA1+2fTPUOX8qxEV\\\\\\\/rJJMTFGjd\\\\\\\/y7C7AzsxyWHK7DJZsRV82S6TLe69R7UcXKEGCyzIJoSTg4cSUrGevAgd7OkN9PiC4o6hgEQeK3ouBk3nTbxqYIoJFZ7SeFwOg3GjA+g1M\\\\\\\/pHHY1qTvB7VHxPIaJSegX7lZeIMMxz14cOgS0NeMH5xJCVPeTb6xYApfm\\\\\\\/ha8U\\\\\\\/re7yCWPEXCq9G4XnahfovS\\\\\\\/mcTVwDLfzv+GD63ybNa5BwJS1GOHU0NY+BYjfqWNsBxBSyzIYgKqpMIcTz8tWFzLVRlomka7calUYvZniKDz3mMeecTgAuUuicdjAhjq4khPiZcHSDI2VLec+AapGR9sg1pQ2pMQUthqN+mbcXo\\\\\\\/8\\\\\\\/uMPNlMQ4nQxLfJtOHQGtZs20aO1AcJ+tp4BWDFARbQMsEZ8cETyJxLSirh+exiqmzBj467yvEpbgeMrVEKJGbySgy6YjKn4Xl3786jXKnfpyugaaq2r0cE57Ti6TPjH50LYhUNOSJvUSWnGYMirDj2vZFKq7en75Zen79LmN6Q3Rf5MWib0oW35KrS6xlTkySRMcWaruPnBZGIuYQYDyAO+j8GPplttZ0XoJSnbM2E7zg3o6oLe2a5w4n8TZsqX6WQt\\\\\\\/fecBRIObJo6AWosAlmzisAxmUCFQFYa0MdaLISpGl0sMhBIO70OHErbjq0kGG27URLuxXQNmTFA8UCROy6s4VGMTbU8\\\\\\\/jdqI7AKeTX4VONdXJ4MmlvXAQKRPCL20nJnkBf3ejriIji9bHu7B3vIFIv3K26PC0iy\\\\\\\/1+5+XTenFtKw09OhZw2BYniUH2QtsGXIre04r+QctIdmOdlfOpaVTHh2LpuC+UUCmYEPdalSN5PsfNtjZ6Uoy3QjTFKTouOD6yOg6t6MMlMnR2leyckytTlPIS3ODrg0856RL\\\\\\\/XUepaQpRPkzYuI6jfsHcVlKm8OCD9m\\\\\\\/o\\\\\\\/A5ZsCp\\\\\\\/Mi2w98F6\\\\\\\/rIEJmIsH9\\\\\\\/A0iINRDs9blWGn7mkEiMv6sCZsjQq+YbWkpmYtlC2gQkdktjCCcH6iXpuHEt03ciLbvn\\\\\\\/ZPV60nEnkL+J7HLoJolSBaKTzd1nImaz0mSQQPZKvEXAS6R\\\\\\\/EDzX\\\\\\\/TFncOdtaClhuTjkdEmgtshtIQCEUNeaoK6vZHXQM\\\\\\\/gIc5d9Pa2BLNRgETJCEp1WyeAl1LN\\\\\\\/RfOWojd+wSOi683sHUKlWul9QtK\\\\\\\/Ghtfj5O4TeA5yDIqx5d3BKUsPxnTX+XtxJA2rdzv4rXrGwy8LtBwhPA7CZ81NTylnw+8yVAk1XgustMovoEm6f4rZxMZn80KxXg+KzLSONTHfuMdQZrSB1547sRYhy\\\\\\\/XTsulZ1nwPNlTfSdwJ21Ic2WudTwV4H+GxBGN6Jmm7xzmqHX8kQGhUWXBhCCGai3f9aeiVeu0EjB6e4M+yVlUgN2g\\\\\\\/pHfYtRnt5QEyzO6IcUd\\\\\\\/RF7Y3m+WMp8oAKrlAn3\\\\\\\/LuNNvndrlvcyosctk6URMKWjTmGtmD0Fp5BhGtl3hZ00k2\\\\\\\/YBOWvxHHi9qgCKTIzKZSgdJ6euuaOyF3rF9fvvhksdkX2mDO73gN\\\\\\\/TYGJnCKMSZmR8tfhwVVr4cTV4oEGxZ1bmZkwpuQMwRH\\\\\\\/a1HtXQfrhnhcxsXUTUw5DX9cftkSw6e9TL+mYiMzNMxQ37qmH1P1qBDQIPqvbYOjFB1ylKxDTRoB429eLF\\\\\\\/S8sPZ6SkdOu8N4uUn\\\\\\\/8P9Im+klGXFAPILxlEurxXhkt3s6Fc51+XS3f7Da\\\\\\\/Vwzf+EII0\\\\\\\/\\\\\\\/57EYmCJNhFhtqCs6RENT0OmvAHl9P3y5p1kwmbn3Fy1g5a9WclnCVU6eF\\\\\\\/zeF7INsmCHPCXRXl8QTUVxeYRdRohVzU\\\\\\\/8\\\\\\\/QUBu91DKjKWORGI10xzPYUkIqn1ZwChWBKhb1QUHd5OECHsbcrFkZdPqi3vME9thnSu5+ecNDA3eM4sj5YzgdovA93cyEh6CUmaC9lNd3cBp6ezmKJu5J0oVw0i88FksBJk1fF8NM8aRa9cKiifiWhbUfuEQmeAHE3wgX1uOFvJ\\\\\\\/NXNMoGwGNY7n+ABnuJetRvO6TobqQ+duOTDB\\\\\\\/GKd1d3x6Pf64vOHzgwPHdetc6Q+Vfmng3q841FfQmE8m61rsHrOWBy5G3zpwDBqp9kvgxewcBrtJnocl7OaOBz9XqKyk69rhC75hpz\\\\\\\/iX\\\\\\\/sDHesy43xM3YHmrPs97\\\\\\\/xf1zdkUv9SNEmVeeZoYJol5LDwhl+5DSTRaos7mWIz\\\\\\\/f6RvDXQObELWg0XNreDr4cxyE84ayatlEg0RGc1jTMH+EA7bdLoKCfc45K+S8NLvFobV88bj41zkcQSmQrULEnTXwXftlzxph743AEsZFg6QDjis+5z5YOGLvlj4UaMHbplqTiisHbhD2Es9U0HnMraNz8Yt9H1SGXWotiPUg2o4nfjVnA+fTY7i3MGouw3aFj8Fdi1PMbZV\\\\\\\/rJFzSJErBVqGSWvNBQNNL2vzJ5uC3j\\\\\\\/8WjRCpN\\\\\\\/4ddItSkjrdfipTzI+C3n4qTO7p4M91tBY0bbzpzNriChNCnZMxq1RKWeDZxcZGqfy7g8RcVEvnaElLiDCqphOrSY82XKgeUaein5+uLZDMuSV4vt6LszxoHhvbIDkdtyGW8r3as8611gTJ7f5MCDXVqbGN4B9pezZSBBw+hI8DFnIoXNAQGrzPAYFr8fmjzEvxG6dJnpL6e2XnvTVm4ijbacSBHzJhLCtn633Q\\\\\\\/m4INx6eZV3e99cpw1jz2Jsa5BltD35rHhHFJNHACsnXKKH2QBV79Xasch4uxpsrp4hqnLq8+9H2+0GmqaQGr5MtH4e+SQj0yQ\\\\\\\/s9VWEN6CsjWH17SS3SxeXuTzbpJ8Clg\\\\\\\/5EfChIwLSGTeSzWnPmILvOjYp+fJCx3cxmslgQBx2W1bC6enB\\\\\\\/p4bFq2itrjYyUHdu2WdMC\\\\\\\/l+bThAr5VlrFdY1t0mYX6E7kH3HQxkmDRw9MjmW9wQtz2HE24AiZLVixSEO\\\\\\\/VjuLWNo8L9TLHRmvBWEgwsU29pnncHZrckQ0Cd6v5CgRaadXCnRy3TjmZFjPcuyWFrhlLdxw3\\\\\\\/0\\\\\\\/FuQoLgubEyRxZ0ZrXG4Ugp\\\\\\\/gXPGjh\\\\\\\/\\\\\\\/cNJCJxbZU7SganeGBSMCzcV9\\\\\\\/bGJTw3w89CxvthgCaq5xrQBu8rHqiTEpiTQ+ftmaQ49I8+iyWDCS3fK65NdPqsbeos3CRLoTZADX09CXHG\\\\\\\/dHyMC6ymtvvqWaSrUOrRHoZ\\\\\\\/+q9S7\\\\\\\/JfzpiMw3PX1g4f8T5Q042I8i44TcwIJQ5FO90I\\\\\\\/yjk2gAjycRh+7OI1Ly4JacUM6kvk9gVlW8FHX6quYBSpPzZ8CUDR+u7zdHbcKn5dB3Xby9xkqc0lNblFjTcTGjgBmq5q5ROENCqQWCvI3p1Tx4Iv0TejYrGMH0mxNhYYy1lDvow5u015zjtigF5+QP+FBMpMEzNxP2lsOaQLyhjrnBPajpQpl\\\\\\\/a9X+NbwMdl24hLIlK16W8xaMy2ekm7lyrU1yBbMoMuT9pqwGQdltRPLNFRShzVAty\\\\\\\/LjFDRhn4eEFCqiEUJWlEoZ60DkuneynyFXWr1f9YsA1z1xtKKqSLZoSFelauQ72jhtTm87ykmfQgPTTWAkQo1S2u7ebLDQaZrV0R5TmcqamwoyQHXRgIkMOukaNeOgkhVYilYqX60jAmUyppztwATawMW7inK+4kC6RWWk2boUCxYt82dT\\\\\\\/qcOcmunal1cw1efsXr3tJaSd6CLmxswPgOYjotGfaypjoyPP3EqOPww+Zc42GVJxiB+xhdGt\\\\\\\/tnnRJZFEfXkuqtv7xBDp3t3MM4uXiyKc+fZg\\\\\\\/B66B8D14k\\\\\\\/pETifn5fWMc2m6AoRkePORj+aPrZaMmnPmVxI+UD5+XCIVsFpgYl1yd\\\\\\\/CBfKqxiX\\\\\\\/k1hYyi1SQXNpMBHxLGsa2JiRXr0J4NaS0JWk6RWEEXlmbo1wV34wZFfUi1iOKBs4ph\\\\\\\/o707Rb9HdhadSSviCnm6PNGjm0nkcSJL0HMrF5gs8L5ojQ4fRvta113\\\\\\\/h4jmaCs1+hcK3wFJEfpvK0c13BX5WdXnwF2MLzfNqoRDuAFacMBpWr01E1uqur1M+hATL1gS9+hQxFiL+OXPEVlSiyr0kGXLqywWmsbRdgFA2ajlA0uPLJOr3GWAeqbPu7trBj9lIZmHLFph6ziAZCb6oqICSEDQ+wlrH9tVeKST2SDklW6m6GmNe8gAFWeIDXnNY+K81Z1hI9aigo4mVcyV84WIH6FdwPY0XfAoVjmaPjYE0lKfhOVsI8iDuqAcM+Rd75FKrzdeYJFJ07owE580kKP4NwSo6aEEbS7aNrJElYWNw2SStn9Jw8cKuGJZTKF\\\\\\\/9S\\\\\\\/rvQ\\\\\\\/fnoX7r7JnY8efE6TsQSFNe11dzXhvCilue3OonryB3mmmJxBviB2D9+i0BDnX0MuuzS2nw3FYEs8IkvR7rguADMDYPVqRLegNVpsPKpO2JaABbXPwJhwzCp0JSOCp9QxvRR8YBzxWP4H1QwaHU1lC8o1FVdlYn2MltEmt3C2tmA6CPL\\\\\\\/wZUY7P\\\\\\\/SUqkIkM+k+YqFVdeg\\\\\\\/8GTfp9wjDT24C9w3NxRHeuzsrG8bitfwFfAp3wlubgQj64NpygwYffIlrrni1z67I\\\\\\\/hYcLlSYN1FM9QKRKzWIRrQHs0dA5+YJ3cIhUt5Sn4tWyse2dfv6ktCiTR14vg\\\\\\\/eqwBznVW0svpYeYABWjt6Oz651WZLlj46mrt3Z1IsXWoe6yq3ftmjxQ7OVY2quhe1JiSchuer5d\\\\\\\/LX3byXWHHHNUV9LJ\\\\\\\/6d\\\\\\\/9lKdNt\\\\\\\/bVeHVHRJp\\\\\\\/RIC0AqDXBwWkBytyN\\\\\\\/VNfdEEvfvPokZ8S6h\\\\\\\/NGR5v6XqnzPgZaLDuG5nVZLXvH1PRoRJXVQfMFACEaHqs4xWa4wdcF0gkWcrMHZHJWSxAZVF5\\\\\\\/vPVUWVxf\\\\\\\/oI1kKawCNl7br48emHVatXUk0dhr9DfPg86oR96tCzNH7mbJtnLud2wKcNfActWk1BYGTua4+Hzu3suhG2MBOIT0Ag6c7f1AmaMUMGmfau1UeIvpZHqhRDk3eINGYw2bTWOBQ39\\\\\\\/Q85IaqJEzyr7SIyS3QbLj3tPEd5g2I90Uj62m\\\\\\\/kNsKbJOy5x0CvSHWso9sPgmBtNz+O0+0QvHk2hZcLBoFfezYh6IaJkc6O1KKAgVKgzAA9lol9E6wZ4w2roXK6VTvcoOCLhlxEwHiqAYLAr+buMrlwi9erogEz1hub40o9B+YkvRw7ss9MU8KtENTFQ8QZljWQWvp1yuxlxiAHchSBs+c3H6ifXQE4L8Tz5TA61iUJjeYEo27FUw4PTjZmQuc3dp8VXzuzwEIcfRDEPK6K9mgAm0xo+BoyRDYjc5lNzXlXni0x\\\\\\\/blkC9AyatqlwPGgSJte1CciLdNKPuTq61h8DxvVza1Yj3t7S7Ex7Zqfhop7YXXLIshkTDRurW1\\\\\\\/bqS54biKPEgJcHi7IWsYDCWkC8TdkC0ZNeq22v2LZ5pBM1Dck40t4vBKqzgIxH7a+LLu+T8Wa4ZKu57EIKr9cjZ3UtZvQZnX5+6WbVp3SeTEhd6pIB9LimzndBTmRhaZ5pB3ZXkforFJonTP1bPwFw+dFDi4yJmtTMpyBn5GSSUwpSmqJkFWhaFxBgeMLFb7y+l9bgF0mZNXRNCnD6uMGpAgr0OKkR+Cp+K1CrlKSTj4rH8Qg2xuKVmSs4iYV8ZPYc6GhEsiVbpjsScCpdLBe7pmnTvz31v0G87Dw5O\\\\\\\/eF6Qr\\\\\\\/xGx59\\\\\\\/hH6zl5CL0P\\\\\\\/yiYnFHU\\\\\\\/\\\\\\\/8qJnhBycPUqNoR15zKVDmb+Hv5Y8Lp1\\\\\\\/O3ePcTjbkqDftbw6k\\\\\\\/smtMbOC9gMY\\\\\\\/SLSSn+OVw7eEknIdHT3lNlhKCZr+ZDJLlhPvsdm3R8xoA9+ryyThKlYUJLucFsdY5U\\\\\\\/YUoLC7FNRB9D6rAE452ns2cfBLMzLdiisbEs1Qr5o1xBXO8e535B48wH1og3aov27R4DCuWnW+RUiFdrb+6HP\\\\\\\/j7iXb75VNHbwRbCTzvt8iHwkemD1FVE5SKkfptPJM6f4bsUUketYCj5avX9VV05ZLz36\\\\\\\/7VkqLPc6gXG8tvrP3oIpDG68DbKBWDl\\\\\\\/\\\\\\\/jPIdVPcw5ngJZXcjvy8iT1MWE38v73\\\\\\\/aK9oTyUWStA+9PNdmUL6XGuoHGVSc2KuD4TgowptRAYsY68oKoN6kNjqBhNNSlJ\\\\\\\/C7xRu0zJYk+2B2pClVzPhUZqXDHruE5O8+D\\\\\\\/ooPjL2kgQftha7rvIyxkOB3qeBC+50halbMo3pRXhsLWrbJz1tn9k4g+weiX2wEFT\\\\\\\/Fa0AgnhuNnqzjhcjC+u68anH7fkaDW4iNhPKArX3TlfOKLgRAlblFW3YSDfGToE3C9jXE705jDabwDZ5T6JCA3\\\\\\\/GeVBt2Uc9WjJ5F0nGf7Wg6sc60XxTNpYZ6b7EqipnIUtItqwPZjfjyndDg4K+zy\\\\\\\/oOZgkxp9ZJlwRPGGimKJG5ZqOxxFawk24ltYLePPrZ9rXr+BVLpFdmOnREWrY4ObP5ez5552COEk5XNWPh+mewTUhc6jnlN2BxCdeA4y28XidKyQMIh\\\\\\\/DYpVmXyRmwMln59296nmgJsimkBBiMgGCqdgDQGKbnLwgpB42TU6eIyluxS7k\\\\\\\/D53jspRKGYxw+K82B7z2KyT1xgznhrGwN9a97V9im7p4ErWRgTVSkEGecwv0YPbsccYajclTwAFkxnVBQG9nXyF7G3EV9oEa4riCy+1v8QFTOBKQZfJ4cgIuUnhMECihatKNbAYhaaVhnfYjp8dl86jfjvBBm3jA2vpRUFeaXcDIQAD2hLYtFSQbQYETYT1gtpW6vGzDj\\\\\\\/reqIJ0MhAjZ4GjNKn1zGyLu8lFNFET+xtQSUSC3dffCdHuToQPo8pTeb98zTWQqQG6ikP64B2fMixjvNOwthnDJf8BmC2juPcWkUrhJ2rRMRHjVlLWg3x2tGB\\\\\\\/yweDOOGq2GR7SksxSm37kvy47Tua6oELffSsrWYqSGFbA\\\\\\\/Plh55N\\\\\\\/pfTKnh2CrtdVNoVfanERLzxLoKREfROmfmwTyCa6gn1oqdxZp3ZrE3w7gxaOuDXL5OWh8xJMCyTk8kKlE2niiv+bmfq1YCIdMG3pSwfyF5ziSSFmJcAcaPXHi5DTt\\\\\\\/tnSl5syS\\\\\\\/JnKZVSzD67qqKpTFyqon1C3xvnBaWIqRbXKd9XfHgtzXqcfu0ivd76cLKrk8VGJM2DcnJvDDNvsv8M8uv53Ok2O63lgGB5bQdCnLO6J6cojXhOey5H0y8Qx5ouuLYW5ssSrqlv90FhxjkufgNyFEaopL0d+CoSoTu3b2+yvi6IkN\\\\\\\/diX6F18Qup6jd2sS6DRZ4dvD\\\\\\\/O7u89S45jYGcK72Ab9Dvxjey2gbTeUFUb9x\\\\\\\/wKqi66O3QkgwaMN2tW2aKdGUbTZaUmgi4M78Koxe9bt7KUxE9aQFKW4AQpMdxp6NubIBgoxC9952V46rr372o01OYk8FWpxJ4lXftzGoCw1kGVNfbSw8lGu8BbFEIi9UQ5tstkFywIXPVUeRth+xqtAYo65lQ6FTn57TT5rx5tiiLEzNEOFlKpLauOA4t0Imha8y0vd6FJhDzS\\\\\\\/45wxYmL5+MHIUvsmhmfJvDnv5+W0uIpHKX75wGfxoiwZfuBaUrJmdolTzRT\\\\\\\/t1f8no7kAVAalndD4Sy+08cIsZxK3LXvg1HSy4qp5YGiq7PBTy+Wh7uSr5xlJbXPa8eDcjtCkdxp5KUjSslQRAnFpq2YbZbxVc3V8NeWfZJOlKHGKiAukyl+EOEIyMlXP09VnmSEbOFzJXPRRqMo\\\\\\\/+QVmaRjHBcJwr3bQkVuTuG4cY\\\\\\\/2kfOoMVNMm4PmOkDnSHSshYJ1twI278AVHIWmrxmjCPZor+GiAfF4oiqDUhyW8ZXwvuzPILmTxQFuBCbyeI+yD5mS5vHux\\\\\\\/9xPABJ7VPscAuLlEPprQA281\\\\\\\/BveAekX+xaT6YYkIC5\\\u0022,\\\u0022iv\\\u0022:\\\u0022KFIGJzUkbjbG1J9wUci38w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229c69712011a7f839d8c942da1e63fb8eb137e5aabbde4f7c08cc2f5ed1b7fb19\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dpNFvTVtaUYM7thQtO\\\\\\\/oFSCbvV82ja2KoOSk1NdyC2NLSBC+nuAx5NhFDofZh8tsFJ8x9fGgsQNH2orJc7hEy\\\\\\\/Ydy0pvmoxK1b3RHm5CrbpMKYykVmfOp6eqzhdjrAHERkqPCAeD6p50JV0lw\\\\\\\/+pCeoum79a3kGEup\\\\\\\/IsqMq\\\\\\\/sYdgKSo468RA7nnyw9fMVIv075d0PigqQdA3avb72hTiOyU6\\\\\\\/XnGkqZTIh7k4thS\\\\\\\/PPKMTTzP\\\\\\\/SOjJX9OHtbSQT8HFHjYPu3ECdBFHyBXHesv0b8hvaNPFTZCBSP15uMrpiJFpZqCZhosQhqLFmD0x\\\\\\\/qn4NDY0oxsETfIKQBleNkpr3ShlCaA1G6137xsDgd\\\\\\\/KxYJ+uBb89rMdz3wdpkiXsLWpOdMdxouLcXubi+ShJLidMwiVBJFY3cF6tMyW0VU4bSMDWwoQKTeHxFmRadZb9+z5z7+v4eOH6Fl+1T2U7kO2FgwlvOeUc9\\\\\\\/Ndu8a9qHQ1yOX57Y06hOlz9bDj+oiZMSbsnR3NQC0GcNJ+uepGDPNWOhr+t1g2WRkYvATiCRnseX0HaYXnQn+ADplfMSF5FOqkIoxYFb6KSGSMZWQrIfiC17jeKDD8BART9UeDWrBRv4DQKHEPRWjG22syQLRfHyFh5R9Lw6cliUGGPw7wN7KcwNCQPHEVoIFLWp36z2iDMr9eoZliIGwRl4xSqD0XcEHfCo37kpTJ1zHnWRNgpm8JEBA8XQ6q0Grn8om6r2qg\\\\\\\/voC4kUHEuxOm8pnS3jmvTXwojbmTmUQQdMRNm5x7d7U5Tq93LAHhiyWs8YiXZfEJHKssuelWtE2fgKHK4zP42WttKUTcNgpk5BWBu3mFxJR9rEDh7GGOZohti9Aldwj\\\\\\\/ZRjQPlSFHJuhReRijezT7szBaD3+Mtm5BxutaAfySSB9Jeq3B2ogxg1UMWCkZDEVTZZj+8KuCzdBLnuSf4XF7OiDJS+7n0JUODnT7Pgph8OHV0ehusJs7lKH\\\\\\\/4axZXRNCmsh61VPlraNOOwGOqjThYiQmo5WH68x33MQdKj6okj18yq0AD9Vm48HccUeEtvKPqHZ4Ukns7bBM5j5+vzDKG6IwsBmMc\\\\\\\/X4QxtItel2tvAQ7k5j\\\\\\\/CP2dzcKbFXGI01LnmjDzshlBTzjdYEQQIYf2VNc0HkbdYcKD9bFosnbobwc5AwbD\\\\\\\/zs11G+AgeHcQWhXelYz+CCOrjiSRzuWdY\\\\\\\/c3bW4mIt32n5jKIRsnhz\\\\\\\/Tv0LbjLPOSEAmVDaLGI8e44NVAQQgrnUIL6t0rCqsuRgwqh9d5SbkgYhfjZbjLWBHmLCpfsgxHGYKO+KnNW61BKnHl1VRuyRpFv85SiCYxTljsrGJNRn5npFTClC5m3Kqd36C+1JPOL5N6uefjCAdTEOpoSBnuA8cNfoLmUkbuqcLfI\\\\\\\/92LQbGNLUcZjQ9ye0gQcZmgrFB3dJmKVSXh7tj8eKCVI5GnS\\\\\\\/frYLNDq3Ji53d\\\\\\\/oIBfne9wSTUneELXu8m8GdQ9YVUBqmPlVqwY\\\\\\\/G3\\\\\\\/1\\\\\\\/4tO3vSd207ETd82XxGmHmFQustDvu+l+LSd3LCd80mlbSN8udKl3+4Xy28iHAhzhwNfyUX1x7mxqC0bo2n3VZqmjz95Js1BzaSu4Qn7728nRU4qW9hlUdHZQ1FRFfTkYNnL7551iOF8nbgtX6xwFaWYezWSYXWPSpMLrhP+LqOubg02Uj97RdZhuiWrqAKKPKNi0jxilOdlUMOYUl3fv8Hmj3G8+RS7g\\\\\\\/MBpqDLfFruYQL8IzwyyF5Hrg\\\\\\\/a0BDhMGEGtrDHFKtgH9QRs8M12jUSWy1fCJOzsrGUN741sEhEzclDu2il+EQWGRKO+HHIK7HeaBjmJKcMyxIHR07l7FBDafehpe0f107werUBGBFToAKxShEIN4Blhwzzhqna2qlBkQCVZ7+BlffByBHZyts9c\\\\\\\/Cj\\\\\\\/WjAjP+sOdhbGstz3fA5wgD9TWXaZ2JUU0GPA7Y8WdkhnNwAMNOl7xK82Xlm3Tjyn69ATf4M59LhcYFzFjgfwRaYXvbsp30Q=\\\u0022,\\\u0022iv\\\u0022:\\\u0022i6ZfxEKgwOQ2V1MJLLe6sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022472de015563904bbd900392b8a0fd79ccb439d1fbd0e9ba5d5028d7ab94f8ed7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460193 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2801" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UXdxxr5sMuMbXyB_SDQXaHwfB9_u1kKALNfY17mZpdQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2801" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0laOj4h28EZHWPtfP72td-ci1z1s2baC6eZL4K4JX14" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2793" title="Development of INDEL Markers for Genetic Mapping Based on Whole Genome Resequencing in Soybean" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2809" title="Examination of Endogenous Rotund Expression and Function in Developing Drosophila Olfactory System Using CRISPR-Cas9–Mediated Protein Tagging" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460193" id="node460193" data-pisa="ggg;5/12/2801" data-pisa-master="ggg;g3.115.023317" data-apath="/ggg/5/12/2801.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xiaoge Guo</span>, <span class="highwire-citation-author" data-delta="1">Kevin Lehner</span>, <span class="highwire-citation-author" data-delta="2">Karen O’Connell</span>, <span class="highwire-citation-author" data-delta="3">Jenny Zhang</span>, <span class="highwire-citation-author" data-delta="4">Sandeep S. Dave</span> and <span class="highwire-citation-author" data-delta="5">Sue Jinks-Robertson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2801-2808;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023317 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460193" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaoge Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Molecular Cancer Biology, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoge%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoge%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kevin Lehner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BLehner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BLehner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen O’Connell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BO%E2%80%99Connell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Connell%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BO%25E2%2580%2599Connell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jenny Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenny%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJenny%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sandeep S. Dave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Duke Cancer Institute, Duke University, Durham, North Carolina 27710</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BS.%2BDave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dave%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandeep%2BS.%2BDave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sue Jinks-Robertson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Duke Cancer Institute, Duke University, Durham, North Carolina 27710</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular Genetics and Microbiology, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sue%2BJinks-Robertson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jinks-Robertson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASue%2BJinks-Robertson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sue.robertson@duke.edu" class="" data-icon-position="" data-hide-link-title="0">sue.robertson@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/12/2801" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460193" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460193/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2801.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460193" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460193/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2801.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460193" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460193/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2801.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460193" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460193/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/12/2801.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460193" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460193/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Single-molecule real-time (SMRT) sequencing generates much longer reads than other widely used next-generation (next-gen) sequencing methods, but its application to whole genome/exome analysis has been limited. Here, we describe the use of SMRT sequencing coupled with barcoding to simultaneously analyze one or a small number of genomic targets derived from multiple sources. In the budding yeast system, SMRT sequencing was used to analyze strand-exchange intermediates generated during mitotic recombination and to analyze genetic changes in a forward mutation assay. The general barcoding-SMRT approach was then extended to diffuse large B-cell lymphoma primary tumors and cell lines, where detected changes agreed with prior Illumina exome sequencing. A distinct advantage afforded by SMRT sequencing over other next-gen methods is that it immediately provides the linkage relationships between SNPs in the target segment sequenced. The strength of our approach for mutation/recombination studies (as well as linkage identification) derives from its inherent computational simplicity coupled with a lack of reliance on sophisticated statistical analyses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pacbio" class="hw-term hw-article-keyword hw-article-keyword-pacbio" rel="nofollow">PacBio</a></li><li class="kwd"><a href="/keyword/snp-linkage" class="hw-term hw-article-keyword hw-article-keyword-snp-linkage" rel="nofollow">SNP linkage</a></li><li class="kwd"><a href="/keyword/barcoded-sequencing" class="hw-term hw-article-keyword hw-article-keyword-barcoded-sequencing" rel="nofollow">barcoded sequencing</a></li><li class="kwd"><a href="/keyword/rare-variant" class="hw-term hw-article-keyword hw-article-keyword-rare-variant" rel="nofollow">rare variant</a></li><li class="kwd"><a href="/keyword/single-molecule-sequencing" class="hw-term hw-article-keyword hw-article-keyword-single-molecule-sequencing" rel="nofollow">single-molecule sequencing</a></li></ul><p id="p-3">Next-generation (next-gen) sequencing methods have revolutionized the characterization of entire genomes, epigenomes, and exomes. There remains a need, however, for high-throughput alternatives to traditional Sanger sequencing for repetitive analysis of a single target and for analysis of a small number of targets in multiple samples. In addition, the linkage relationships of single-nucleotide polymorphisms (SNPs) in complex genomes generally requires longer sequence reads than are obtained with common next-gen methods. In the current study, we describe the use of single-molecule real-time (SMRT) sequencing from Pacific Biosciences (PacBio) as a time-efficient, cost-effective, and computationally simple approach that meets these needs.</p><p id="p-4">SMRT technology utilizes a sequencing-by-synthesis approach in which a circular DNA molecule is used as a template for a single DNA polymerase. The first step in the sequencing protocol is conversion of linear amplicons to a circular form by ligation of a DNA polymerase-anchoring, stem-loop adapter (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Travers <em>et al.</em> 2010</a>) (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The resulting SMRT library is then sequenced using a SMRT cell containing ∼150,000 DNA polymerases anchored in individual nano-chambers. The template-directed incorporation of each nucleotide by a given DNA polymerase releases a unique fluorescent tag that is monitored in real time. The unique strength of SMRT technology relative to other next-gen sequencing methods is single-molecule reads that exceed 10 kb in length. To capitalize on the single-molecule feature and high data output afforded by SMRT sequencing, independent samples can be barcoded during PCR and pooled for SMRT library construction, thereby allowing massive parallel sequencing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SMRT sequencing pipeline. Each amplicon was barcoded with unique forward and reverse primer pairs in a 96-well format. Complementary amplicon strands are dark and light blue bars; black and red bars correspond to forward and reverse barcodes, respectively, conjugated to target-specific primers (arrows). Following amplicon pooling, hairpin adapters (orange) were attached during SMRT library construction, converting linear into circular molecules. A single DNA polymerase (purple) reads each circular template in real time during the sequencing reaction. Circular consensus sequence (CCS) reads obtained from a SMRT cell were sorted by barcodes and aligned to a reference sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1275147044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SMRT sequencing pipeline. Each amplicon was barcoded with unique forward and reverse primer pairs in a 96-well format. Complementary amplicon strands are dark and light blue bars; black and red bars correspond to forward and reverse barcodes, respectively, conjugated to target-specific primers (arrows). Following amplicon pooling, hairpin adapters (orange) were attached during SMRT library construction, converting linear into circular molecules. A single DNA polymerase (purple) reads each circular template in real time during the sequencing reaction. Circular consensus sequence (CCS) reads obtained from a SMRT cell were sorted by barcodes and aligned to a reference sequence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2801/F1.medium.gif" width="214" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/12/2801/F1.medium.gif" width="214" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467136" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">SMRT sequencing pipeline. Each amplicon was barcoded with unique forward and reverse primer pairs in a 96-well format. Complementary amplicon strands are dark and light blue bars; black and red bars correspond to forward and reverse barcodes, respectively, conjugated to target-specific primers (arrows). Following amplicon pooling, hairpin adapters (orange) were attached during SMRT library construction, converting linear into circular molecules. A single DNA polymerase (purple) reads each circular template in real time during the sequencing reaction. Circular consensus sequence (CCS) reads obtained from a SMRT cell were sorted by barcodes and aligned to a reference sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">In the current report, we describe development of a barcoding-SMRT pipeline for the repetitive analysis of a target sequence in budding yeast. Following the sorting of individual reads by barcode, sequences are aligned to the appropriate reference for analysis. The utility of this approach in the molecular analysis of recombination intermediates and in the rapid construction of mutation spectra is demonstrated. The high coverage obtained through repetitively sequencing a single mutation target additionally allowed determination of the error frequency at each position within a 2-kb amplicon. These error frequencies are relevant to identification of rare variants in more complex samples. Finally, the generality and robustness of the barcoding-SMRT approach for mutation identification and SNP phasing in a complex genome is demonstrated through analysis of target amplicons from primary lymphomas and derived cell lines.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and growth conditions</h3><p id="p-7">Low- and high-transcription strains (<em>pCAN-CAN1</em> and <em>pGAL-CAN1</em>, respectively) were previously described (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Lippert <em>et al.</em> 2011</a>). SJR3956 was used to identify recombinants after galactose induction of an I-<em>Sce</em>I generated double-strand break and was derived from W303 (<em>MATa RAD5 leu2-3,112 ura3 CAN1 his3-11,15 ade2-1</em>). Briefly, the strain contained a <em>lys2</em> allele cleaved by I-<em>Sce</em>I (chromosome II) and a 4-kb repair template with evenly spaced SNPs (chromosome V). Repair of the broken molecule via homologous recombination generated a selectable Lys<sup>+</sup> phenotype.</p><p id="p-8">Cells were grown nonselectively in YEP (1% Bacto-yeast extract, 2% Bacto peptone, 500 µg/ml adenine hemisulfate) supplemented with 2% glycerol and 2% ethanol for mutation analysis or with 2% raffinose for recombination experiments. Canavanine-resistant (Can-R) mutants were selected on synthetic complete glucose (SC) medium lacking arginine and supplemented with 60 μg/ml canavanine. For identification of recombinants, I-<em>Sce</em>I was induced by 0.1% galactose for 40 min and Lys<sup>+</sup> recombinants were selected by plating on SC-lysine medium. All growth was at 30°.</p></div><div id="sec-3" class="subsection"><h3>Yeast sample preparation and SMRT sequencing</h3><p id="p-9">Barcoded primers contained 20 nt of yeast-specific sequence conjugated to 16 nt of PacBio-designated forward and reverse barcodes F1–F24 and R1–R16, respectively (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023317/-/DC1/TableS1.docx">Table S1</a>). Specific barcode pairings used were F1–F12 with R1–R8, F1–F12 with R9–R16, F13–F24 with R1–R8, and F13–F24 with R9–R16. Custom oligonucleotides were synthesized by Eurofins MWG Operon and were provided in a 96-well format. To amplify <em>CAN1</em> when under control of its endogenous promoter, forward and reverse primers contained yeast-specific sequences 5′- CCCTCCGAAGGAAGACTCTC and 5′-GGGAGCAAGATTGTTGTGGT, respectively. To amplify <em>pGAL-CAN1</em>, the forward primer above was replaced with primer 5′-AATCTGTCGTCAATCGAAAG. For analysis of recombinants, the repaired allele was amplified using yeast-specific sequences 5′-GAGTGTATGGGCTGCATTGA and 5′-TTGGGAGTTGGGAATTGAAG conjugated to forward and reverse barcodes, respectively.</p><p id="p-10">Genomic DNA was isolated from unpurified yeast mutants/recombinants grown nonselectively for 3 d in 96-well microtiter plates. <em>CAN1</em> or <em>LYS2</em> sequences were then amplified using <em>Taq</em> polymerase (Bioline) and barcoded primers in the same 96-well format. Ten microliters of barcoded product from each mutant/recombinant were pooled, and the pooled sample was purified using the GeneJet PCR Purification Kit (Thermo Scientific). Six micrograms were sent to the Duke Center for Genomic and Computational Biology Core, where SMRTbell adapters were ligated to the barcoded amplicons (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Travers <em>et al.</em> 2010</a>). Sequencing of the resulting library was performed on a PacBio RSII instrument using P6-C4 chemistry and movies of 180 min.</p></div><div id="sec-4" class="subsection"><h3>Human DNA preparation and SMRT sequencing</h3><p id="p-11">DNA was extracted from diffuse large B-cell lymphomas (DLBCLs) or derived cell lines using the Qiagen Allprep RNA/DNA/Protein kit (Qiagen 80004). Briefly, 10<sup>6</sup> cells were centrifuged at 1100 rpm for 5 min, washed in PBS, and lysed in 600 μl RLT buffer. The lysate was passed through an AllPrep DNA spin column, which bound genomic DNA. The column was washed in AW1 buffer followed by a wash in AW2 buffer; DNA was eluted in 100 μl of TE buffer and quantified using a NanoDrop; 25 ng genomic DNA were used in individual PCR reactions and the following human sequences were contained within barcoded forward and reverse primers, respectively: 5′-GTAGAGGTGGGCACCCCCGCCTCCGCACTT and 5′-TGTGGCAACTTAGTTCAGCATATCCCCTGG for MYD88 exons 1-4, 5′-GGTAAGCTCAACCCTGCTCTGGCAAGAGAA and 5′-TGCATGGCAGCTAAATGCCTCAACAAGATC for MYD88 exon 5, and 5′-CCCCCCAAACAACTTACCATTGTAC and 5′-GTTCTCTTTCCCATTCGGTCTGCGC for EZH2 exon 16.</p></div><div id="sec-5" class="subsection"><h3>SMRT sequence analysis pipeline</h3><p id="p-12">Data were analyzed using an in-house pipeline (SmrtSeqTool) written in Python that uses Bowtie2 for read alignment and SAMtools/Bcftools for variant calling (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Shander <em>et al.</em> 2009</a>). The pipeline (<a href="https://sourceforge.net/projects/smrtseqtool/">https://sourceforge.net/projects/smrtseqtool/</a>) was run on a standard Mac with OS X Yosemite. Files needed to run the SmrtSeqTool analysis pipeline are: (1) a FASTQ file containing the PacBio-generated CCS reads (“.fastq” extension); (2) reference sequence (ref.txt) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Gibson <em>et al.</em> 2010</a>); (3) forward barcodes (ForBar.txt); and (4) reverse barcodes (RevBar.txt). Templates for files 2–4 are available in the “Sample” folder that can be downloaded from the website. The <em>CAN1</em> and <em>LYS2</em> reference sequences were from the <em>Saccharomyces</em> Genome Database (SGD); the MYD88 (Genbank, MYD88) and EZH2 (Genbank, EZH2) reference sequences were from GenBank.</p><p id="p-13">The first step of the pipeline uses the forward and reverse barcodes to generate different forward and reverse barcode pairs. Additionally, all 12 consecutive nucleotide windows for each barcode pair are generated. The CCS reads in FASTQ format are then sorted according to these barcode pairs. Each CCS read is classified according to a forward barcode and a reverse barcode. For each unique barcode pair, we demand that at least 12 of 16 nucleotides of the sequenced molecule match each barcode. CCS reads matching this criterion are written into the corresponding barcode files named F1R1, F1R2, etc. In our runs, none of the reads was assigned to more than one barcode combination. Once CCS reads are sorted, the pipeline uses Bowtie2’s default alignment (end-to-end read alignment) to align reads to the corresponding reference sequence and outputs a set of alignments in SAM format. The SAM alignment files are then processed and analyzed by SAMtools, followed by variant calling using its built-in Bcftools. For the purpose of our data analysis, SAMtools default parameters were used. Finally, each individual CCS read alignment was visualized using the Text Alignment View (tview) in SAMtools.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-14">A barcoding-SMRT sequencing pipeline was developed using budding yeast and applied to the analysis of strand-exchange intermediates generated during recombination and to the analysis of mutation spectra. To allow parallel sequencing of hundreds of independent mutants/recombinants, a 2-kb amplicon from each was tagged with a unique forward–reverse barcode combination and the tagged amplicons were pooled to construct a single SMRT library. Although SMRT sequencing has only ∼90% accuracy during a single read of a given template, a DNA polymerase can traverse a circular molecule multiple times before stochastically disengaging. This property allows construction of a circular consensus sequence (CCS) read that minimizes the contributions of random errors. We set a parameter of at least three complete passes of a given molecule for building a CCS read (<em>i.e.</em>, at least two reads of one strand and one read of the complementary strand) as the default setting of two produced poor-quality sequences. Following the sorting of CCS reads by forward–reverse barcode pairs, sorted reads were aligned back to the reference sequence using Bowtie2. The data reported here were generated using PacBio P6-C4 chemistry; the average read length was ∼15 kb, and ∼55,000 CCS reads of 2-kb amplicons were obtained per SMRT cell. The pipeline for our analysis is available at <a href="https://sourceforge.net/projects/smrtseqtool/">https://sourceforge.net/projects/smrtseqtool/</a>.</p><div id="sec-7" class="subsection"><h3>Barcoding-coupled SMRT sequencing for analysis of recombination intermediates</h3><p id="p-15">We have developed recombination assays that monitor the strand-exchange intermediates formed during repair of a defined double-strand break (DSB) in yeast (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Mitchel <em>et al.</em> 2010</a>). The key features of these systems are: (1) use of a diverged sequence with evenly spaced single-nucleotide polymorphisms (SNPs) as a repair template for a broken molecule and (2) use of mismatch-repair defective background, which prevents repair of mismatches formed during strand-exchange reactions. In early studies, substrates were small enough to be sequenced in a single Sanger reaction and mismatches created by strand exchange during recombination were detected manually as dual peaks on sequencing chromatograms (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Mitchel <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Guo and Jinks-Robertson 2013</a>). Given the labor-intensive nature of the analysis, especially with substrates that exceed ∼800 bp, a SMRT sequencing approach was developed. This was coupled with amplicon barcoding to allow the simultaneous sequencing of multiple recombinants.</p><p id="p-16"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> illustrates a mechanism of DSB repair that generates mismatches on only one side of the initiating break; blue and orange lines reflect the diverged recombination substrates. The 123 CCS reads obtained for a specific recombinant of this type, which was contained within a SMRT library of 96 recombinants, are presented in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. The number of single-molecule reads in each class is indicated, and the data are summarized in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>. Within the repaired orange molecule, there is a contiguous region where 51 of 123 single-molecule reads had blue SNPs (each square/circle is a SNP) instead of orange, revealing precisely the extent and position of strand exchange (boxed) during repair of the DSB. This specific application of SMRT sequencing demonstrated the general utility of the barcoding approach, the highly accurate phasing of SNPs within an amplicon, and a low error frequency (black squares in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>) associated with sorted CCS reads. Below, we extend the general framework to mutation analysis in yeast and to human tumor samples.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping strand exchange intermediates during recombination. (A) Model of double-strand break (DSB) repair that generates mismatched SNPs (blue paired with orange strand) on only one side of the initiating break. (B) SNP linkages in 123 CCS reads. SNPs are spaced at &#x223C;50-bp intervals and are indicated by orange or blue squares. Isolated blue or black squares within orange regions represent random errors that matched the blue SNP or did not match either SNP, respectively. (C) Sum of orange/blue SNPs (circles) at each position. In all panels, the region of strand exchange where orange and blue strands pair is boxed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1275147044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mapping strand exchange intermediates during recombination. (A) Model of double-strand break (DSB) repair that generates mismatched SNPs (blue paired with orange strand) on only one side of the initiating break. (B) SNP linkages in 123 CCS reads. SNPs are spaced at &#x223C;50-bp intervals and are indicated by orange or blue squares. Isolated blue or black squares within orange regions represent random errors that matched the blue SNP or did not match either SNP, respectively. (C) Sum of orange/blue SNPs (circles) at each position. In all panels, the region of strand exchange where orange and blue strands pair is boxed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2801/F2.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/12/2801/F2.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467188" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Mapping strand exchange intermediates during recombination. (A) Model of double-strand break (DSB) repair that generates mismatched SNPs (blue paired with orange strand) on only one side of the initiating break. (B) SNP linkages in 123 CCS reads. SNPs are spaced at ∼50-bp intervals and are indicated by orange or blue squares. Isolated blue or black squares within orange regions represent random errors that matched the blue SNP or did not match either SNP, respectively. (C) Sum of orange/blue SNPs (circles) at each position. In all panels, the region of strand exchange where orange and blue strands pair is boxed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Biological validation of the barcoding-SMRT approach for analysis of mutation spectra</h3><p id="p-18">Characterization of mutation signatures and patterns reveals underlying DNA sequence properties that promote instability, genetic consequences of endogenous and exogenous mutagens, and the substrates of enzymatic machineries that remove premutagenic DNA lesions. Forward mutation assays provide an unbiased view of all mutations types, but the target size often exceeds the limits of a Sanger-sequencing reaction. The yeast <em>CAN1</em> forward mutation assay was used to demonstrate feasibility of SMRT technology for accurate, parallel sequencing of hundreds of mutants. The <em>CAN1</em> locus has a 1.7-kb open reading frame and encodes arginine permease. The wild-type (WT) protein transports the toxic arginine analog canavanine into cells, resulting in death. Cells with a defective gene are resistant to canavanine (Can-R) and form colonies on canavanine-containing medium. Following the isolation of DNA from independent Can-R mutants, a 2-kb fragment from each was amplified using primers tagged with unique forward–reverse barcode combinations.</p><p id="p-19">Validation that SMRT sequencing captures known biological profiles was obtained by sequencing Can-R mutants isolated under normal transcription conditions or when transcription was highly elevated. We and others have demonstrated that, under high-transcription conditions, forward mutations in <em>CAN1</em> are dominated by short, 2- to 5-bp deletions rather than by base substitutions (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Lippert <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Takahashi <em>et al.</em> 2011</a>). Of 141 Can-R mutants isolated under low-transcription conditions, 92% contained a single base substitution, 7% had a 1-bp insertion/deletion (indel), and 1% had complex changes. This is similar to the mutation distribution reported by others using Sanger sequencing (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Lang and Murray 2008</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Lippert <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Guillet <em>et al.</em> 2006</a>). When Can-R mutants were isolated under high-transcription conditions, however, 2- to 5-bp deletions were the causative mutation in ∼61% (85/139) of the mutants sequenced. An example of aligned CCS reads derived from a mutant with a 2-bp deletion is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. Within the set of high-transcription mutants, we also detected a large, 40-bp internal deletion as well as complex sequence changes (examples are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023317/-/DC1/FigureS1.pdf">Figure S1</a>). In principle, there is no limit to the size of deletion or insertion that can be detected by SMRT sequencing, as long as the primer binding sites remain and amplicon production remains efficient. Although the average coverage of each mutant usually exceeded 100, the high accuracy of SMRT sequencing suggests that a causative mutation can be identified from only 5 to 10 CCS reads. To date, we have combined a maximum of ∼400 independent mutants in a single SMRT library, which was then sequenced in two SMRT cells. An average coverage per mutant of ∼100 CCS reads was obtained, indicating the feasibility of sequencing several thousand barcoded mutants in a single SMRT cell.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CAN1 sequencing data. (A) Alignment of CCS reads for a Can-R mutant harboring a 2-bp deletion. Dots and commas are matches to complementary strands; letters are deviations from the reference sequence and asterisks are deletions. (B) Base-substitution error frequency at each position in the CAN1 amplicon. The enlarged inset compares error frequencies derived from two independent SMRT libraries indicated by open and filled circles. See also Figure S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1275147044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;CAN1&lt;/em&gt; sequencing data. (A) Alignment of CCS reads for a Can-R mutant harboring a 2-bp deletion. Dots and commas are matches to complementary strands; letters are deviations from the reference sequence and asterisks are deletions. (B) Base-substitution error frequency at each position in the &lt;em&gt;CAN1&lt;/em&gt; amplicon. The enlarged inset compares error frequencies derived from two independent SMRT libraries indicated by open and filled circles. See also Figure S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2801/F3.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/12/2801/F3.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467241" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child"><em>CAN1</em> sequencing data. (A) Alignment of CCS reads for a Can-R mutant harboring a 2-bp deletion. Dots and commas are matches to complementary strands; letters are deviations from the reference sequence and asterisks are deletions. (B) Base-substitution error frequency at each position in the <em>CAN1</em> amplicon. The enlarged inset compares error frequencies derived from two independent SMRT libraries indicated by open and filled circles. See also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023317/-/DC1/FigureS1.pdf">Figure S1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Identification of rare variants in a complex sample</h3><p id="p-21">SMRT sequencing has a single-pass error rate of 10–15%, with &gt;90% of errors corresponding to single-nt indels and the remainder corresponding to base substitutions (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Quail <em>et al.</em> 2012</a>). In cases where there were several hundred CCS reads for a given Can-R mutant, the accompanying background noise was low, especially for base substitutions. To obtain a robust estimate of the base-substitution error frequency across the 2-kb interval sequenced, we combined the CCS reads from hundreds of independent Can-R mutants isolated under low-transcription conditions and determined the error frequency at each position. Two independent SMRT libraries were separately analyzed, and there was an average depth of coverage of ∼15,000 in each. The causative mutation in each independent mutant was excluded from the corresponding CCS reads to uncover background base substitution errors. Subsequently, all the CCS reads from both libraries were then aligned to determine a base-substitution error frequency at each position (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The average error frequency was ∼0.2% across the <em>CAN1</em> target sequence with some positions (8/2047) containing no errors; there were only two positions where the error frequency exceeded 1% (1.7% and 1.2% for positions 79 and 91, respectively), and these were adjacent to long homopolymer runs in the 5′ noncoding region. The inset in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> presents the error frequencies in each SMRT library within a 100-bp interval and reveals highly reproducible values. Although the relative contributions of PCR errors <em>vs.</em> SMRT-sequencing errors were not explored, it is possible that use of a higher-fidelity DNA polymerase during amplification would drive the error frequency even lower. With prior knowledge of the error-frequency landscape for a given amplicon, our data indicate that rare variants can be called with very high accuracy.</p></div><div id="sec-10" class="subsection"><h3>Comparing SMRT sequencing to exome sequencing in human lymphomas</h3><p id="p-22">To illustrate the broad utility and accuracy of the barcoding-based SMRT sequencing pipeline developed in yeast, we applied the same approach to diffuse large B-cell lymphoma (DLBCL) primary tumors and to DLBCL-derived cell lines. Eleven samples that had been previously characterized by Illumina exome sequencing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Zhang <em>et al.</em> 2013</a>) were analyzed by SMRT sequencing that focused on the MYD88 (myeloid differentiation primary response 88; involved in immune responses) and EZH2 (enhancer of zeste homolog 2; a histone methyl transferase) genes. For each sample, three 2-kb genomic segments were individually amplified using barcoded forward–reverse primer pairs: exons 1–4 of MYD88 (with associated introns), exon 5 of MYD88, and exon 16 of EZH2 with flanking intron sequences. The latter two genomic segments harbored mutations identified by exome sequencing and implicated in tumorigenesis. After PCR amplification of the MYD88 and EZH2 segments, the barcoded amplicons were pooled with yeast amplicons for construction of a SMRT library. From 33 barcoded segments subjected to SMRT sequencing, only one failed to return any CCS reads.</p><p id="p-23">A single polymorphism was detected in MYD88 exons 1–4 among the 11 samples analyzed by SMRT sequencing; this polymorphism was in each of the 43 CCS reads for the corresponding sample and hence was homozygous (data not shown). The genetic complexity of the other two genomic segments was high, likely reflecting inherent genetic instability coupled with an unknown tumor-cell ploidy and, for primary lymphoma samples, possible contamination with normal cells. The SNPs present in the CCS data for MYD88 exon 5 and EZH2 exon 16 are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023317/-/DC1/TableS2.docx">Table S2</a> and are compared to those previously detected in Illumina exome sequencing. Four and five SNPs were present in the MYD88 and EZH2 amplicons, respectively, used for SMRT sequencing; the reference base at each position is in lowercase and deviations from the reference are in uppercase bold. The number of occurrences of each SNP in the CCS and Illumina reads is given for each sample. The two sequencing methods were in excellent agreement with respect to the homozygous–heterozygous status of SNPs. Only one discrepancy was seen, and this was at position 19 of MYD88. Seventy-nine CCS reads had a C at this position; among Illumina reads, 10 had a C and two had a T.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467130/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467130/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison of SMRT sequences and Illumina exome sequences for MYD88 exon 5 (chromosome 3)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>SNP linkage/phasing in tumor samples</h3><p id="p-25">The distinctive feature of SMRT sequencing relative to other next-gen sequencing methods is read length, which routinely exceeds 10 kb. This greatly extends the length over which SNP linkages can be directly determined and does not require any computational manipulation of the data. Primary tumor DLBCL778, for example, was heterozygous for a single SNP at the first polymorphic position in the MYD88 exon 5 amplicon. Accordingly, 73 CCS reads had haplotype C-g-a-t for and 17 CCS reads had haplotype t-g-a-t; the aligned CCS reads are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> compiles the SMRT sequencing data for all DLBCL samples and groups the CCS reads by haplotype. The phasing ability of SMRT sequencing is particularly striking in cases in which a polymorphism was present at more than one position. Cell line Ly10 provides one such example, where there was heterozygosity at the first and third SNPs in the MYD88 amplicon, corresponding to four possible haplotypes: C-g-a-t, t-g-a-t, C-g-G-t, and t-g-G-t. SMRT sequencing revealed which two haplotypes were actually present; 28 and 16 CCS reads were C-g-G-t and t-g-a-t, respectively. The phasing revealed in the single-molecule reads was particularly interesting for EZH2 exon 16 in DBLCL799, where three of five SNP sites were heterozygous and there were eight possible haplotypes. There were four distinct haplotypes in the CCS reads, the most abundant of which were G-a-G-g-G and a-a-a-g-a with 41 and 57 CCS reads, respectively. There were two minority haplotypes that were also of similar frequencies: 11 and 12 CCS reads of a-a-a-g-G and G-a-G-g-a, respectively. As illustrated in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>, the latter two haplotypes can be related to the former two by a reciprocal crossover between the third and fifth polymorphic positions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SMRT sequence data from human lymphomas. (A) MYD88 exon 5 CCS reads from tumor DLBCL260. CCS reads with the t-g-a-t haplotype are shaded yellow. (B) Crossover event that results in four haplotypes for the EZH2 exon 16 sequence in tumor DLBCL799." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1275147044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SMRT sequence data from human lymphomas. (A) MYD88 exon 5 CCS reads from tumor DLBCL260. CCS reads with the t-g-a-t haplotype are shaded yellow. (B) Crossover event that results in four haplotypes for the EZH2 exon 16 sequence in tumor DLBCL799.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2801/F4.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/12/2801/F4.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2801/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467111" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">SMRT sequence data from human lymphomas. (A) MYD88 exon 5 CCS reads from tumor DLBCL260. CCS reads with the t-g-a-t haplotype are shaded yellow. (B) Crossover event that results in four haplotypes for the EZH2 exon 16 sequence in tumor DLBCL799.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467202/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467202/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of CCS reads from lymphoma cell lines</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-28">In the current study, we document the application of SMRT sequencing as a high-throughput alternative to Sanger sequencing for the analysis of a small number of targets in a large number of independent samples. The key elements in the approach are the unique barcoding of each target during PCR amplification and the pooling of all barcoded amplicons prior to construction of a single SMRT library. Library size is constrained only by available barcodes; with the most recent barcodes developed by PacBio (<a href="https://pacbio.secure.force.com/Share/Protocol?id=a1q70000000J4m5AAC">https://pacbio.secure.force.com/Share/Protocol?id=a1q70000000J4m5AAC</a>), the theoretical limit of unique barcode combinations exceeds 100,000. Following library sequencing, CCS reads are sorted by barcodes and aligned to the appropriate reference sequence. The final step can be done using an automated program that can be downloaded from <a href="https://sourceforge.net/projects/smrtseqtool/">https://sourceforge.net/projects/smrtseqtool/</a>.</p><p id="p-29">We initially developed the barcoding SMRT-sequencing framework to analyze mitotic recombination intermediates and mutation spectra in budding yeast, where it offers several advantages to Sanger sequencing. First, 2-kb amplicons were sequenced in a single reaction/run, eliminating the need for subsequent assembly of the sequence. With the current average read length of ∼15 kb, it should be possible to efficiently sequence amplicons in the 5-kb range. Second, the unique barcoding of each recombinant/mutant allowed hundreds to be sequenced in one or a few SMRT cells, which greatly reduces the overall sequencing cost. Depending on the specific application, the depth of coverage required will vary and will be the major determinant of how many samples can be sequenced in a single SMRT cell. For mutation identification in yeast, our data suggest that 5–10 CCS reads is sufficient; with ∼50,000 CCS reads for a 2-kb amplicon per SMRT cell, it should be possible to simultaneously sequence several thousand independent mutants. For yeast recombination studies or for analyses of more complex genomes, however, much deeper coverage is required. In our analyses of mutation spectra within a defined yeast gene, we combined ∼30,000 CCS reads obtained using two independent libraries to measure the base-substitution error frequency at each position across the 2-kb amplicon. The average error frequency was only 0.2%, but it varied from position to position. Between libraries, however, the error frequencies at independent positions were in excellent agreement. These data suggest that, with prior knowledge of the error profile, rare variants in a more complex sample can be identified with high accuracy. Moreover, as we demonstrate, a real mutation occurred in virtually every read. Thus, one needs to only look at the aligned reads provided to identify the causative mutation; no statistical analysis is needed.</p><p id="p-30">The barcoding-SMRT sequencing pipeline was developed in yeast, but its broader utility was demonstrated by applying the same approach to independent lymphoma samples. We analyzed 2-kb segments within the MYD88 and EZH2 genes and validated the specific changes detected using prior Illumina exome-sequencing data. Importantly, SMRT sequencing readily revealed the linkage relationships between polymorphic SNPs and the complexity of the haplotypes present in the samples analyzed. Although linkage relationships can be determined using other next-gen methods, it requires sophisticated computational methods (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Snyder <em>et al.</em> 2015</a>). As an alternative to whole genome or exome sequencing of tumor samples, it should be straightforward to barcode multiple independent targets of interest within a single genome or within multiple genomes. This may be particularly useful in screening tumors for associated defects in DNA repair pathways, which can inform subsequent chemotherapy, or for identifying mutations in genes whose products are potentially drugable. With a high depth of coverage and prior knowledge of the error landscape, it also should be possible to identify rare variants with high confidence.</p><p id="p-31">The current study focused primarily on high-throughput identification of mutations in specific targets, but the general framework can be readily adapted to other types of analyses. The combined barcoding, SMRT-sequencing approach, for example, is particularly well suited for characterization of splice variants in different tissues or genotypes. Different CCS reads will directly reflect splice variants, and their relative proportions should reflect the abundances of the corresponding mRNAs (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Sharon <em>et al.</em> 2013</a>). Other potential applications include examination of multiple markers for fungal species characterization (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Schoch <em>et al.</em> 2012</a>), validation of kilobase-sized oligonucleotides in synthetic systems (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Gibson <em>et al.</em> 2010</a>), rapid screening of various DNA-binding substrates for engineered proteins (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Thyme <em>et al.</em> 2014</a>), and examining genome alterations made by engineered endonucleases (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hendel <em>et al.</em> 2014</a>). As the technology improves, especially with regard to read length by single DNA polymerases and error reduction, applications for SMRT technology will continue to evolve.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">This work was supported by NIH grants to S.J.R. (GM038464, GM093197, and GM101690) and S.S.D. (CA136895). The Duke School of Medicine Core Facility Services Voucher Program provided support to S.J.R. for PacBio sequencing. All authors reviewed the manuscript. The authors declare no competing financial interests.</p><p id="p-33">Author contributions: X.G., K.L., and S.J.R. conceived and designed the yeast experiments. S.J.R. and S.S.D. conceived the experiments on primary tumor and tumor-derived cell lines. S.S.D. provided the primary tumor and tumor-derived cell line genomic DNA samples. X.G. performed all the experiments and collected data for SMRT analysis, and S.J.R. and K.O. provided suggestions and input. J.Z. provided the data analyses on Illumina. X.G. and S.J.R. interpreted and discussed the results. X.G. and S.J.R. wrote the paper.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-34">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023317/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023317/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-35"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-29"><span class="received-label">Received </span>July 29, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-10-12"><span class="accepted-label">Accepted </span>October 12, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Guo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-5.12.2801.1"><div class="cit-metadata unstructured">Genbank. EZH2. Available at: <a href="http://www.ncbi.nlm.nih.gov/nuccore/NG_032043.1">http://www.ncbi.nlm.nih.gov/nuccore/NG_032043.1</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-5.12.2801.2"><div class="cit-metadata unstructured">Genbank. MYD88. Available at: <a href="http://www.ncbi.nlm.nih.gov/nuccore/NG_016964.1">http://www.ncbi.nlm.nih.gov/nuccore/NG_016964.1</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.3" data-doi="10.1126/science.1190719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lartigue</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noskov</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuang</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Creation of a bacterial cell controlled by a chemically synthesized genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190719%26rft_id%253Dinfo%253Apmid%252F20488990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjkvNTk4Ny81MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjgwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.4" data-doi="10.1093/nar/gkl139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Kemp</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boiteux</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">dUTPase activity is critical to maintain genetic stability in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2056</span>–<span class="cit-lpage">2066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGuillet%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253D7%26rft.spage%253D2056%26rft.epage%253D2066%26rft.atitle%253DdUTPase%2Bactivity%2Bis%2Bcritical%2Bto%2Bmaintain%2Bgenetic%2Bstability%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl139%26rft_id%253Dinfo%253Apmid%252F16617146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM0LzcvMjA1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjgwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.5" data-doi="10.1016/j.dnarep.2013.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Roles of exonucleases and translesion synthesis DNA polymerases during mitotic gap repair in yeast.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1024</span>–<span class="cit-lpage">1030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D12%26rft.spage%253D1024%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2013.10.001%26rft_id%253Dinfo%253Apmid%252F24210827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2013.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24210827&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.6" data-doi="10.1016/j.celrep.2014.02.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kildebeck</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fine</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Punjya</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantifying genome-editing outcomes at endogenous loci with SMRT sequencing.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D7%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2014.02.040%26rft_id%253Dinfo%253Apmid%252F24685129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2014.02.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24685129&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.7" data-doi="10.1534/genetics.107.071506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating the per-base-pair mutation rate in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLang%26rft.auinit1%253DG.%2BI.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D82%26rft.atitle%253DEstimating%2Bthe%2BPer-Base-Pair%2BMutation%2BRate%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071506%26rft_id%253Dinfo%253Apmid%252F18202359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc4LzEvNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.8" data-doi="10.1073/pnas.1012363108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoenly</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Role for topoisomerase 1 in transcription-associated mutagenesis in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">698</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012363108%26rft_id%253Dinfo%253Apmid%252F21177427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDgvMi82OTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.9" data-doi="10.1016/j.molcel.2010.02.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welz-Voegele</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular structures of crossover and noncrossover intermediates during gap repair in yeast: implications for recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMitchel%26rft.auinit1%253DK.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D222%26rft.atitle%253DMolecular%2Bstructures%2Bof%2Bcrossover%2Band%2Bnoncrossover%2Bintermediates%2Bduring%2Bgap%2Brepair%2Bin%2Byeast%253A%2Bimplications%2Bfor%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.02.028%26rft_id%253Dinfo%253Apmid%252F20417600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.02.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20417600&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277598100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.10" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.11" data-doi="10.1073/pnas.1117018109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seifert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huhndorf</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spouge</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Nuclear ribosomal internal transcribed spacer (ITS) region as a universal DNA barcode marker for fungi.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">6241</span>–<span class="cit-lpage">6246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1117018109%26rft_id%253Dinfo%253Apmid%252F22454494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzE2LzYyNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-5.12.2801.12"><div class="cit-metadata unstructured">SGD. Available at: <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.13" data-doi="10.1111/j.1423-0410.2009.01207.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shander</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cappellini</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnough</span>  <span class="cit-name-given-names">L. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Iron overload and toxicity: the hidden risk of multiple blood transfusions.</span> <abbr class="cit-jnl-abbrev">Vox Sang.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVox%2Bsanguinis%26rft.stitle%253DVox%2BSang%26rft.aulast%253DShander%26rft.auinit1%253DA.%26rft.volume%253D97%26rft.issue%253D3%26rft.spage%253D185%26rft.epage%253D197%26rft.atitle%253DIron%2Boverload%2Band%2Btoxicity%253A%2Bthe%2Bhidden%2Brisk%2Bof%2Bmultiple%2Bblood%2Btransfusions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1423-0410.2009.01207.x%26rft_id%253Dinfo%253Apmid%252F19663936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1423-0410.2009.01207.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19663936&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270182800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.14" data-doi="10.1038/nbt.2705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilgner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A single-molecule long-read survey of the human transcriptome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D1009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2705%26rft_id%253Dinfo%253Apmid%252F24108091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24108091&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.15" data-doi="10.1038/nrg3903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Haplotype-resolved genome sequencing: experimental methods and applications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D344%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3903%26rft_id%253Dinfo%253Apmid%252F25948246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25948246&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.16" data-doi="10.1073/pnas.1012582108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burguiere-Slezak</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Kemp</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boiteux</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Topoisomerase 1 provokes the formation of short deletions in repeated sequences upon high transcription in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012582108%26rft_id%253Dinfo%253Apmid%252F21177431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDgvMi82OTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.17" data-doi="10.1093/nar/gku1096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thyme</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunette</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szeto</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Massively parallel determination and modeling of endonuclease substrate specificity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">13839</span>–<span class="cit-lpage">13852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1096%26rft_id%253Dinfo%253Apmid%252F25389263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi8yMi8xMzgzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjgwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.18" data-doi="10.1093/nar/gkq543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Travers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rank</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eid</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A flexible and efficient template format for circular consensus sequencing and SNP detection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq543%26rft_id%253Dinfo%253Apmid%252F20571086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNS9lMTU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2801.19" data-doi="10.1073/pnas.1205299110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubor</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic heterogeneity of diffuse large B-cell lymphoma.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1398</span>–<span class="cit-lpage">1403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1205299110%26rft_id%253Dinfo%253Apmid%252F23292937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzQvMTM5OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjgwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/12/2801.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2793" title="Development of INDEL Markers for Genetic Mapping Based on Whole Genome Resequencing in Soybean" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2809" title="Examination of Endogenous Rotund Expression and Function in Developing Drosophila Olfactory System Using CRISPR-Cas9–Mediated Protein Tagging" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/12"><span>Volume 5 Issue 12, December 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/12.cover-source.jpg?itok=RngHqKQq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2801" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-x8Z54BahRaZrhSLR6VDDv_NGwuJfatS6Phj91ehIEoU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/12/2801" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/12/2801" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/12/2801" class="active" data-icon-position="" data-hide-link-title="0">SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460193" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7pA6r20JnSn6wDd_L9Pel_mG7nZ1ixkszQmm7xHatxY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2801.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/12/2801" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-7us6GH18M_rAN_vkVv7_ov4d13AUUiI1zDrUW80bfBM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/12/2801" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2801.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460193" id="node460193--2" data-pisa="ggg;5/12/2801" data-pisa-master="ggg;g3.115.023317" data-apath="/ggg/5/12/2801.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xiaoge Guo</span>, <span class="highwire-citation-author" data-delta="1">Kevin Lehner</span>, <span class="highwire-citation-author" data-delta="2">Karen O’Connell</span>, <span class="highwire-citation-author" data-delta="3">Jenny Zhang</span>, <span class="highwire-citation-author" data-delta="4">Sandeep S. Dave</span> and <span class="highwire-citation-author" data-delta="5">Sue Jinks-Robertson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2801-2808;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023317 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460193--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaoge Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Molecular Cancer Biology, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoge%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoge%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kevin Lehner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BLehner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BLehner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen O’Connell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BO%E2%80%99Connell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Connell%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BO%25E2%2580%2599Connell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jenny Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenny%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJenny%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sandeep S. Dave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Duke Cancer Institute, Duke University, Durham, North Carolina 27710</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BS.%2BDave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dave%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandeep%2BS.%2BDave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sue Jinks-Robertson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Duke Cancer Institute, Duke University, Durham, North Carolina 27710</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular Genetics and Microbiology, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sue%2BJinks-Robertson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jinks-Robertson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASue%2BJinks-Robertson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sue.robertson@duke.edu" class="" data-icon-position="" data-hide-link-title="0">sue.robertson@duke.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/12/2801" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/12/2801&amp;title=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/12/2801&amp;title=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/12/2801&amp;title=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/12/2801&amp;text=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/12/2801&amp;title=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/12/2801&amp;t=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/12/2801&amp;title=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/12/2801&amp;title=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460193/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460193" id="node460193--3" data-pisa="ggg;5/12/2801" data-pisa-master="ggg;g3.115.023317" data-apath="/ggg/5/12/2801.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">SMRT Sequencing for Parallel Analysis of Multiple Targets and Accurate SNP Phasing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xiaoge Guo</span>, <span class="highwire-citation-author" data-delta="1">Kevin Lehner</span>, <span class="highwire-citation-author" data-delta="2">Karen O’Connell</span>, <span class="highwire-citation-author" data-delta="3">Jenny Zhang</span>, <span class="highwire-citation-author" data-delta="4">Sandeep S. Dave</span> and <span class="highwire-citation-author" data-delta="5">Sue Jinks-Robertson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2801-2808;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023317 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460193--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiaoge Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Graduate Program in Molecular Cancer Biology, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoge%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoge%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kevin Lehner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BLehner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BLehner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen O’Connell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BO%E2%80%99Connell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Connell%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BO%25E2%2580%2599Connell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jenny Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University Program in Genetics and Genomics, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenny%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJenny%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sandeep S. Dave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Duke Cancer Institute, Duke University, Durham, North Carolina 27710</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BS.%2BDave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dave%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandeep%2BS.%2BDave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sue Jinks-Robertson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Duke Cancer Institute, Duke University, Durham, North Carolina 27710</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular Genetics and Microbiology, Duke University, Durham, North Carolina 27710</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sue%2BJinks-Robertson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jinks-Robertson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASue%2BJinks-Robertson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sue.robertson@duke.edu" class="" data-icon-position="" data-hide-link-title="0">sue.robertson@duke.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2801&amp;rft.epage=2808&amp;rft.atitle=SMRT%20Sequencing%20for%20Parallel%20Analysis%20of%20Multiple%20Targets%20and%20Accurate%20SNP%20Phasing&amp;rft.volume=5&amp;rft.issue=12&amp;rft.date=2015-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Guo%2C+Xiaoge&amp;rft.au=Lehner%2C+Kevin&amp;rft.au=O%E2%80%99Connell%2C+Karen&amp;rft.au=Zhang%2C+Jenny&amp;rft.au=Dave%2C+Sandeep+S.&amp;rft.au=Jinks-Robertson%2C+Sue"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460193/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460193/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460193/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460193/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460193/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460193/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460193/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460193/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460193/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460193/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460193/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460193/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/12/2801" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/12/2801.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/12/2801.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/12/2801.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/12/2801.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
