<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/12/2547" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/12/2547.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/12/2547.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/12/2547.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/12/2547.atom" />
<meta name="HW.pisa" content="ggg;5/12/2547" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.115.020362" />
<meta name="DC.Date" content="2015-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Edae et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The advancement of next-generation sequencing technologies in conjunction with new bioinformatics tools enabled fine-tuning of sequence-based, high-resolution mapping strategies for complex genomes. Although genotyping-by-sequencing (GBS) provides a large number of markers, its application for association mapping and genomics-assisted breeding is limited by a large proportion of missing data per marker. For species with a reference genomic sequence, markers can be ordered on the physical map. However, in the absence of reference marker order, the use and imputation of GBS markers is challenging. Here, we demonstrate how the population sequencing (POPSEQ) approach can be used to provide marker context for GBS in wheat. The utility of a POPSEQ-based genetic map as a reference map to create genetically ordered markers on a chromosome for hexaploid wheat was validated by constructing an independent de novo linkage map of GBS markers from a Synthetic W7984 × Opata M85 recombinant inbred line (SynOpRIL) population. The results indicated that there is strong agreement between the independent de novo linkage map and the POPSEQ mapping approach in mapping and ordering GBS markers for hexaploid wheat. After ordering, a large number of GBS markers were imputed, thus providing a high-quality reference map that can be used for QTL mapping for different traits. The POPSEQ-based reference map and whole-genome sequence assemblies are valuable resources that can be used to order GBS markers and enable the application of highly accurate imputation methods to leverage the application GBS markers in wheat." />
<meta name="DC.Contributor" content="Erena A. Edae" />
<meta name="DC.Contributor" content="Robert L. Bowden" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="article:published_time" content="2015-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The advancement of next-generation sequencing technologies in conjunction with new bioinformatics tools enabled fine-tuning of sequence-based, high-resolution mapping strategies for complex genomes. Although genotyping-by-sequencing (GBS) provides a large number of markers, its application for association mapping and genomics-assisted breeding is limited by a large proportion of missing data per marker. For species with a reference genomic sequence, markers can be ordered on the physical map. However, in the absence of reference marker order, the use and imputation of GBS markers is challenging. Here, we demonstrate how the population sequencing (POPSEQ) approach can be used to provide marker context for GBS in wheat. The utility of a POPSEQ-based genetic map as a reference map to create genetically ordered markers on a chromosome for hexaploid wheat was validated by constructing an independent &lt;i&gt;de novo&lt;/i&gt; linkage map of GBS markers from a Synthetic W7984 × Opata M85 recombinant inbred line (SynOpRIL) population. The results indicated that there is strong agreement between the independent &lt;i&gt;de novo&lt;/i&gt; linkage map and the POPSEQ mapping approach in mapping and ordering GBS markers for hexaploid wheat. After ordering, a large number of GBS markers were imputed, thus providing a high-quality reference map that can be used for QTL mapping for different traits. The POPSEQ-based reference map and whole-genome sequence assemblies are valuable resources that can be used to order GBS markers and enable the application of highly accurate imputation methods to leverage the application GBS markers in wheat.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/12/01" />
<meta name="citation_mjid" content="ggg;5/12/2547" />
<meta name="citation_id" content="5/12/2547" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/12/2547" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/12/2547.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/12/2547.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/12/2547.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020362" />
<meta name="citation_pmid" content="26530417" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2547" />
<meta name="citation_lastpage" content="2553" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Erena A. Edae" />
<meta name="citation_author_institution" content="USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506" />
<meta name="citation_author_institution" content="Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_email" content="Erena.Edae@ars.usda.gov" />
<meta name="citation_author_email" content="jpoland@ksu.edu" />
<meta name="citation_author" content="Robert L. Bowden" />
<meta name="citation_author_institution" content="USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_email" content="Erena.Edae@ars.usda.gov" />
<meta name="citation_author_email" content="jpoland@ksu.edu" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=A. M. Allen;citation_author=G. L. A. Barker;citation_author=S. T. Berry;citation_author=J. A. Coghill;citation_author=R. Gwilliam;citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.).;citation_pages=1086-1099;citation_volume=9;citation_year=2011;citation_issue=9;citation_pmid=21627760;citation_doi=10.1111/j.1467-7652.2011.00628.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. M. Allen;citation_author=G. L. A. Barker;citation_author=P. Wilkinson;citation_author=A. Burridge;citation_author=M. Winfield;citation_title=Discovery and development of exome-based, codominant single nucleotide polymorphism markers in hexaploid wheat (Triticum aestivum L.).;citation_pages=279-295;citation_volume=11;citation_year=2013;citation_pmid=23279710;citation_doi=10.1111/pbi.12009" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. Ariyadasa;citation_author=M. Mascher;citation_author=T. Nussbaumer;citation_author=D. Schulte;citation_author=Z. Frenke;citation_title=A sequence-ready physical map of barley anchored genetically by two million single-nucleotide polymorphisms.;citation_volume=164;citation_year=2014;citation_pmid=24243933;citation_doi=10.1104/pp.113.228213" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Baird;citation_author=P. Etter;citation_author=T. Atwood;citation_author=M. Currey;citation_author=A. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=P. Berkman;citation_author=K. Lai;citation_author=M. Lorenc;citation_author=D. Edwards;citation_title=Next-generation sequencing application for wheat crop improvement.;citation_volume=99;citation_year=2012;citation_pmid=22268223;citation_doi=10.3732/ajb.1100309" />
<meta name="citation_reference" content="citation_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide Selection for Rapid Introgression of Exotic Germplasm in Maize;citation_pages=419-425;citation_volume=49;citation_year=2009;citation_issue=2;citation_doi=10.2135/cropsci2008.08.0452" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S.M. Chao;citation_author=J. Dubcovsky;citation_author=J. Dvorak;citation_author=M.C. Luo;citation_author=S.P. Baenziger;citation_title=Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (Triticum aestivum L.).;citation_pages=727-727;citation_volume=11;citation_year=2010;citation_pmid=21190581;citation_doi=10.1186/1471-2164-11-727" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. Chapman;citation_author=M. Mascher;citation_author=A. Buluç;citation_author=K. Barry;citation_author=E. Georganas;citation_title=A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.;citation_pages=26;citation_volume=16;citation_year=2015;citation_pmid=25637298;citation_doi=10.1186/s13059-015-0582-8" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez;citation_author=J.M. Hickey;citation_title=Genomic prediction in maize breeding population with genotyping-by-sequencing.;citation_pages=1903-1926;citation_volume=13;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Daetwyler;citation_author=U. Bansal;citation_author=H. Bariana;citation_author=M. Hayden;citation_author=B. Hayes;citation_title=Genomic prediction for rust resistance in diverse wheat landraces.;citation_pages=1795-1803;citation_volume=127;citation_year=2014;citation_pmid=24965887;citation_doi=10.1007/s00122-014-2341-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Edae;citation_author=P. Byrne;citation_author=S. Haley;citation_author=M. Lopes;citation_author=M. Reynolds;citation_title=Genome-wide association mapping of yield and yield components of spring wheat under contrasting moisture regimes.;citation_pages=791-807;citation_volume=127;citation_year=2014;citation_pmid=24408378;citation_doi=10.1007/s00122-013-2257-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=C. Feuillet;citation_author=J. E. Leach;citation_author=J. Rogers;citation_author=P. S. Schnable;citation_author=K. Eversole;citation_title=Crop genome sequencing: lessons and rationales.;citation_pages=77-88;citation_volume=16;citation_year=2011;citation_issue=2;citation_pmid=21081278;citation_doi=10.1016/j.tplants.2010.10.005" />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=C. Feuillet;citation_author=N. Stein;citation_author=L. Rossini;citation_author=S. Praud;citation_author=K. Mayer;citation_title=Integrating cereal genomics to support innovation in the Triticeae.;citation_pages=573-583;citation_volume=12;citation_year=2012;citation_issue=4;citation_pmid=23161406;citation_doi=10.1007/s10142-012-0300-5" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=M. Hahn;citation_author=S. Zhang;citation_author=L. Moyle;citation_title=Sequencing, assembling, and correcting draft genomes assembly using recombinant populations.;citation_volume=4;citation_year=2014;citation_pmid=24531727;citation_doi=10.1534/g3.114.010264" />
<meta name="citation_reference" content="citation_title=Crop Sci.;citation_author=E. Heffner;citation_author=A. Lorenz;citation_author=JL. Jannink;citation_author=M. Sorrells;citation_title=Plant breeding with genomic selection: Gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. C. M. Iehisa;citation_author=A. Shimizu;citation_author=K. Sato;citation_author=R. Nishijima;citation_author=K. Sakaguchi;citation_title=Genome-wide marker development for the wheat D genome based on single nucleotide polymorphisms identified from transcripts in the wild wheat progenitor Aegilops tauschii.;citation_pages=261-271;citation_volume=127;citation_year=2014;citation_pmid=24158251;citation_doi=10.1007/s00122-013-2215-5" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome.;citation_pages=286;citation_volume=345;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=H. Iwata;citation_author=J. L. Jannink;citation_title=Accuracy of genomic selection prediction in barley breeding programs: a simulation study based on the real single nucleotide polymorphism data of barley breeding lines.;citation_pages=1915-1927;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.12.0732" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=D. Jaganathan;citation_author=M. Thudi;citation_author=S. Kale;citation_author=S. Azam;citation_author=M. Roorkiwal;citation_title=Genotyping-by-sequencing based intra-specific genetic map refines a “QTL-hotspot” region for drought tolerance in chickpea.;citation_pages=559-571;citation_volume=290;citation_year=2015;citation_pmid=25344290;citation_doi=10.1007/s00438-014-0932-3" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Liu;citation_author=M. Bayer;citation_author=A. Druka;citation_author=J. Russell;citation_author=C. Hackett;citation_title=An evaluation of genotyping by sequencing (GBS) to map the Breviaristatum-e (ari-e) locus in cultivated barley.;citation_pages=104;citation_volume=15;citation_year=2014;citation_pmid=24498911;citation_doi=10.1186/1471-2164-15-104" />
<meta name="citation_reference" content="citation_title=Front. Genet.;citation_author=M. Mascher;citation_author=N. Stein;citation_title=Genetic anchoring of whole genome shotgun assemblies.;citation_pages=208;citation_volume=5;citation_year=2014;citation_pmid=25071835" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Mascher;citation_author=G. Muehlbauer;citation_author=D. Rokhsar;citation_author=J. Chapman;citation_author=J. Schmutz;citation_title=Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).;citation_pages=718-727;citation_volume=76;citation_year=2013;citation_pmid=23998490;citation_doi=10.1111/tpj.12319" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=T. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Poland;citation_author=P. Brown;citation_author=M. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Saintenac;citation_author=D. Jiang;citation_author=S. Wang;citation_author=E. Akhunov;citation_title=Sequencing-based mapping of the polyploid wheat genome.;citation_volume=3;citation_year=2013;citation_pmid=23665877;citation_doi=10.1534/g3.113.005819" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. Sorrells;citation_author=J. Gustafson;citation_author=D. Somers;citation_author=S. Chao;citation_author=D. Benscher;citation_title=Reconstruction of the synthetic W7984 x Opata M85 wheat reference population.;citation_pages=875-882;citation_volume=54;citation_year=2011;citation_issue=11;citation_pmid=21999208;citation_doi=10.1139/g11-054" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Spindel;citation_author=M. Wright;citation_author=C. Chen;citation_author=J. Cobb;citation_author=J. Gage;citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.;citation_pages=2699-2716;citation_volume=126;citation_year=2013;citation_pmid=23918062;citation_doi=10.1007/s00122-013-2166-x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_author=K. Swarts;citation_author=H. Li;citation_author=A. Navarr;citation_author=D. An;citation_author=M. Romay;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=S. Truong;citation_author=R. McCormick;citation_author=D. Morishige;citation_author=J. Mullet;citation_title=Resolution of genetic map expansion caused by excess heterozygosity in plant recombinant inbred populations.;citation_volume=4;citation_year=2014;citation_pmid=25128435;citation_doi=10.1534/g3.114.012468" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. Wang;citation_author=M. C. Luo;citation_author=Z. Chen;citation_author=F. You;citation_author=Y. Wei;citation_title=Aegilops tauschii single nucleotide polymorphisms shed light on the origins of wheat D-genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.;citation_pages=925-937;citation_volume=198;citation_year=2013;citation_pmid=23374069;citation_doi=10.1111/nph.12164" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Xu;citation_author=D. Zhu;citation_author=O. Zhang;citation_title=Predicting hybrid performance in rice using genomic best linear unbiased prediction.;citation_volume=111;citation_year=2014;citation_pmid=25114224;citation_doi=10.1073/pnas.1413750111" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=K. Semagn;citation_author=Y. Beyene;citation_author=R. Babu;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.;citation_pages=291-299;citation_volume=114;citation_year=2014;citation_pmid=25407079" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="twitter:description" content="The advancement of next-generation sequencing technologies in conjunction with new bioinformatics tools enabled fine-tuning of sequence-based, high-resolution mapping strategies for complex genomes. Although genotyping-by-sequencing (GBS) provides a large number of markers, its application for association mapping and genomics-assisted breeding is limited by a large proportion of missing data per marker. For species with a reference genomic sequence, markers can be ordered on the physical map. However, in the absence of reference marker order, the use and imputation of GBS markers is challenging. Here, we demonstrate how the population sequencing (POPSEQ) approach can be used to provide marker context for GBS in wheat. The utility of a POPSEQ-based genetic map as a reference map to create genetically ordered markers on a chromosome for hexaploid wheat was validated by constructing an independent de novo linkage map of GBS markers from a Synthetic W7984 × Opata M85 recombinant inbred line (SynOpRIL) population. The results indicated that there is strong agreement between the independent de novo linkage map and the POPSEQ mapping approach in mapping and ordering GBS markers for hexaploid wheat. After ordering, a large number of GBS markers were imputed, thus providing a high-quality reference map that can be used for QTL mapping for different traits. The POPSEQ-based reference map and whole-genome sequence assemblies are valuable resources that can be used to order GBS markers and enable the application of highly accurate imputation methods to leverage the application GBS markers in wheat." />
<meta name="og-title" property="og:title" content="Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/12/2547" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The advancement of next-generation sequencing technologies in conjunction with new bioinformatics tools enabled fine-tuning of sequence-based, high-resolution mapping strategies for complex genomes. Although genotyping-by-sequencing (GBS) provides a large number of markers, its application for association mapping and genomics-assisted breeding is limited by a large proportion of missing data per marker. For species with a reference genomic sequence, markers can be ordered on the physical map. However, in the absence of reference marker order, the use and imputation of GBS markers is challenging. Here, we demonstrate how the population sequencing (POPSEQ) approach can be used to provide marker context for GBS in wheat. The utility of a POPSEQ-based genetic map as a reference map to create genetically ordered markers on a chromosome for hexaploid wheat was validated by constructing an independent de novo linkage map of GBS markers from a Synthetic W7984 × Opata M85 recombinant inbred line (SynOpRIL) population. The results indicated that there is strong agreement between the independent de novo linkage map and the POPSEQ mapping approach in mapping and ordering GBS markers for hexaploid wheat. After ordering, a large number of GBS markers were imputed, thus providing a high-quality reference map that can be used for QTL mapping for different traits. The POPSEQ-based reference map and whole-genome sequence assemblies are valuable resources that can be used to order GBS markers and enable the application of highly accurate imputation methods to leverage the application GBS markers in wheat." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
    <title>Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"W1rWYfknm7pitIkO7Ek0Jh3W2Wbh0Sex9y5K0Akl_ow","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460197","apath":"\/ggg\/5\/12\/2547.atom","pisa":"ggg;5\/12\/2547","ac":{"\/ggg\/5\/12\/2547.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/12\/2547.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2547"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2547"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/12\/2547":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022B3a8+9H0vlJ77AbvlEFlYNGoHc5jLpE8yiEWl9XqHeD9dKfr\\\\\\\/U6pdm97cQuhlbbImRD9dhMZvY+ktslKT40RiX2gUrCcObB30WL34ULJJ8KlLiuSLBkw9uP5kDUaCDM35v+WTyxQocJb6PSBYZ8SkfNmAayYb+x6fa032Qz0fgYya\\\\\\\/a5gWepCoNK8OMyRq+qyEUFQad5q\\\\\\\/HOAS3XhbHPy870i35TeJ+lo3zs\\\\\\\/pGMMgTv+HivaPQCnE\\\\\\\/KgYhTE0ZbOFUs+n9aUAgwQTk1fLI6INdwUPgcL3sVFeRTXd5+4K2RCr5xIGB3bJJx3t6TQyPRlE+v3hCPnzaZvaxQaJGoJ6XdNVwgKHNYxMrAuNNchWTlMyMY8Id9xrIfw968Czgqq1WK7TRvRYKjMcFjN\\\\\\\/fVFRTfQGVjIaD58ZHsiVkU6cEUvTzByjTx0to1Tve+uAPhFh55zn2eFv8GAcn5JuLMitEezr36MFxU8u3OtwMwawzNOx4wKuRZ0tmEuIA3iD9oUH6+Pg6PUM9MBxDcF9dl2JqNMM49wXoZjVs8OcUmm4t+SLikyN6dXDuUftXABrocIhWtdOTQuONYCVagzeRWMA3TMC\\\\\\\/gSMLMwi72MUGCe6PdP9LbfaNSuQLNjFIpyM\\\\\\\/7yJ0Yd10DyI\\\\\\\/J8MljloPfDaHbyWhVL0ylRybE54yIfhRDQPPhoifescXqRGDKUS4MdPkFAlhyB0XDOC6uMkTkABeB3iBu8OQxSc\\\\\\\/qI04mYDdYeQ4HbJeNnIDNLckkuYUhwfDFcTdFg3FF1euWVEy4njX2lnTNxCY52pUriRRgeiOrtNPHcQmPrOcDpTJ5OnvZAQlO9suNk\\\\\\\/DoSmahv5xosOWjJR63X14aMMPkzUVncIS\\\\\\\/WIRR6wuerfoBrKZAjbVW7tfI6lJVu+KbTSWnrgj67pK5JSQ3QOroQI8fWdRBaZT9GI4wE4hbaBp7PTOgwFOZqaGQ9CzysY\\\\\\\/yrmIcg4yZ4YsWfm\\\\\\\/yPDZx\\\\\\\/FSd2BRgqSi2Cm5c\\\\\\\/QlU1VPhmFhqi5HWmx08cwOJw9UdUr6wehxL2e6uYciDRUVFbNgoL7uHIcgmnuWBzbLbGOHcRELVwAOvnvASZh6YtOkjptOv1+O5sNLxQ6WY1H8\\\\\\\/VwiKNh5A348nJTqqJbnOITP9G5x7K0WOHtfQWQPo5vzduhA2KVJppA9yCtNQvM5D3Ub30rrj2RL3AzaJ0DmhG9WoTxg01NULl7jVdfu5syguQIpEufJkGVGFFuQgDMEDdDCBoafICCf0lwcJ9N3GopWuCXxq3zzN\\\u0022,\\\u0022iv\\\u0022:\\\u0022SCxgO\\\\\\\/fDxCdhdrql3eVhVw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b1bf72617893d16f9b9f7bdfff39e01aff4b6482548182a69a9179ed1310b6af\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ahAIrp32287AYsiyBXcM+vXZpA60YCqd15lAARucRjjv31uOPamlnK1JLI6FJVYxKGiDnh+My4rF4nJodyAeb4iOrqY5\\\\\\\/JXafoPnTlLbqttkHv9Nl5OPJdXFibYQFIrp\\\\\\\/IjFqrstNUyRV\\\\\\\/Byr3ZNxjJMvPqqME2tu+l5MbwyS04ty1Y6ZaWTx0qBNW29JLd+eQU+Y2b2P7\\\\\\\/3MdBjrFp65cVit5mibNaHVSxEhIor74NEapZeBf7EO0WPWEV+lwcCE7kPj2GgDuebPLkT4t2sIMOEyEymzkDrGBqEdxgNCwqY1aBIT7ohtXxg8ayKNlXdDUv+kZXQTqMdBHQVYLF7zmUs04n9yaAlZYEGW9jhCS\\\\\\\/O6Dzf24fPsvP1Qy1CMklGCPqpsCVYLz9bNlHiysjwsxI16G5H29MibLo3DKDMkN21dgzKga14hrq7nNvnmUwe\\\\\\\/wOd8yx43lqN2U7UxkAs6S9hkkxTKriRksO+SEunRxkzoJ+uK9PkTOb\\\\\\\/qdpGlEmUnC7ZDfXXZwHHmXHku4j6axpOnSN\\\\\\\/hxwc0SR0SqfxCD58lLoXgUxaRewBj9upC527xPRBZWTq\\\\\\\/VBkvBU8S5vPx4K48\\\\\\\/eGH3642blUfOhwrjAJ0STS+\\\\\\\/Gw26nkNXFvrGLsqZVo7KiEKJoPL175sh0tTn8UEokBFBfQRbLfjCRt\\\\\\\/hNZ7AARLnKGrVaLDKs9t5BSz+tjI3c0T4fyCExMB0nXuOl2LIjMkd7nA6tp9HMQJ5gjbk6jc7z7q21UwEXNykGwvfM8HP5gRRswZbEicXtHF98ZJmqEqu7S2+utN2BMXcoaDrzDYybUJB6Ub375bV01C8lYDo\\\\\\\/s4KMMkDPcuJGLWE7p9to\\\\\\\/yyLtE9o6A67zll+mDkPKkbChxiwbyj80TSoYNQC9VZR+8lOkTNNUaDCs9QWSz43E\\\\\\\/OIToimDYbtVl4zcpz2fz\\\\\\\/jDYx6YcWFmNfDkePeOTXWUt0XXbKl4bUUbehKxKTn1p7grskQ4UqLSpWi9W390BAUXE0v5Sx6EVTbMiW3M7c9GawCcnv6e\\\\\\\/EbvOeLwJbt7oBIlvG4obX1qvB+TJTvyHW65hqtK6crlAbPYdDkW8X5pniKnpvJqmijavGYBUjDf5Hw7OC7I+2+mFob284\\\\\\\/Qo40s0eOokf\\\\\\\/F53UbmeeTVHpxGioVnt13ht1RLjqzELsDgqPw5BpF0YyrxUz\\\\\\\/qa0GdnHoMcWXOpaIrTCOPn0ZLer18OOll7yboefjzb8Rn935C8iruYuBcgU5hZooKjYvOZPTRZprzT9ioxnv\\\\\\\/HNID\\\\\\\/fGUao3Ms+NOkIX8q3hRt8uzFaFa5ujb4eCfm5K0JjA3beRZ18nkB6nFlpgdylFfVWsNvnbfBttrLqb239TvbO9DrMM4sErXJhWuNY\\\\\\\/sGBE5RxZuobhbYhVqxh+0A0YXF3DwZD\\\\\\\/r378EDqZ7bXB1N\\\\\\\/BxMRqZZ\\\\\\\/fjcZ5cBi\\\\\\\/p+wHQZX00nr4agFO1I4U+jUGc3pCIqhDJOd9AcTDgtU7PO3SHt6p17HU5d2s1JXguRMFb028GtEb1uC6f5+DTd+gYsTBRUSeOpFM4Pk0qN8roVb25gC0wD85fNyYojZc9DI4UiWoKZBRDgUaOQSRjcvqRf7uUfljpQofnfzpivcH2rEd+KM9cBSfeaAqIE\\\\\\\/txT\\\\\\\/La5B8eo6vjB4HEXiceOjcwxhe6jf\\\\\\\/PxhybxgFDgzrpTc1E5rf2H\\\\\\\/mtIt\\\\\\\/chHZgOnQoGthDI4hpB4g0i4+CeLAUzxgtEKfu3lLZXIEI4dQwWqhKp5m3rEwFoeftvFtl3mFxlVG99fZsG9NpJAXsJBPiT0hvbL+HssJ+dapfvfkddS5Yd3ZXncGpMG0MXwIvZpuHo8bwvb6ZXP19t9fhCAqEvbmgPKP2QWuplDU6XFhQNXOgpUTMmepYdJyv6W6k+ynrqQj3UO98Ct\\\\\\\/9eOYuhECET7XjLCT5Gwj7LTwkBWRpuFB5hLX0\\\\\\\/8GwX7F3kkeUUPsBFd8ImYuki\\\\\\\/n9C+QWwU6hIDHUHUrllaU7Payy8BwDpv2q2CcoO5MkfyRizaP6NJIcQvoaLv4z+pc8orKLaOX2oO6urJfd+HBJGaQmILpM5Z1rcuWt7g6t8zQbqyuRmYQlrI+FiZ6ToXWypa+2JKEl7yuyG6CTQvbPGi++z22l3XcI1MQ\\\\\\\/HSrRgeYJZ6sgDcnWNTuT1CTHipid0Ck44k+6luRBksC2cAU2ZmzkM\\\\\\\/1BIaIQvnwgwN7BCApPLzJd0yXJ2chRo5xcc0bfmH1Z4PMbncIeDlaQG\\\\\\\/Qzo9kRyJhuQ7zt5X9RafsvBGsSiQnxFDkURP6M7C437DFLUDSQeKrSWEiaiFKbJ3u4j2ANjcMNaGiDK5KGwnymZq5HzZZ8j2jhYuwuCYUSD\\\\\\\/sDZIqNSnfgL\\\\\\\/N3SJPLVxGx9G9bvDyWAD3pz3k495z0eETqFC6ztRPcI8EU6XjWkepeDWX0U9UEcGj8WdIc933FFKduNe7LxfsACHUqf7uusjLKv2+f2C\\\\\\\/78ovTLGRKTrvQuMRq\\\\\\\/PREWRcypovzFOytu7z4Ae\\\\\\\/WKbHZqXTBKUi83ZgqJ3U7L5OHgX5VJdDoSCsabstGA5ooVRCDtdLTcjf3lZOkcvvHM0OtlOdJoAT\\\\\\\/FMDwZOgI8nifc05GPSxFMinERnncBd2QSmYgxtkEodasMe6693LOvOsdQP4a9jolDnrZ3yy\\\\\\\/1wH7Vp4CQvcKSmqBu9mMvKQSGmsl5Ojw8YGo6TzroTVG\\\\\\\/MZMypkqjI150kk03Xr84WWENsQfoV6pVJm\\\\\\\/2+qbgsHT\\\\\\\/U47UiU7eZELg1I1C9Dz3XtzDNH9TlxAJ03UBidRGd5\\\\\\\/pbF97H3zGzyDS25s\\\\\\\/Bv0TBANnAKbU+b5zS9aQchOB4QT44\\\\\\\/B1cLfewahSFw1GI+mlOli2mtys8lbJo3itQpBuA0j6zucTqLNtjmhJXd4hxhdPE5DxE3tUz1G\\\\\\\/XQBWTn0eIGfu5uee5FEO6tHi+eVIZ\\\\\\\/rMKp1xJwimT7yRz61Q4UkVIO6T0UlVNLm0WtXZ9sF8Y1MZUcZCvzqp8yc6tCsT7DhROex0y3FIfCUg9t7ctPPGElGnQkCMjfNb4lz8pwiNso8AEw86VYJa+IkRgsbJU7VHDCyLl0p+d9ttEQ4QlADDpPZvhVwXFa\\\\\\\/I0u2C\\\\\\\/MivnRAs8gp4B0QW\\\\\\\/LXgItkJoST5uuAt2LULkDwyA\\\\\\\/pfeNgheuSr6hPyTuHiOBNHN1RcGubIh3U7M0ftvVNCZeZaayBXcoEAal1h9Yn6M\\\\\\\/k3CH\\\\\\\/8ynes8VSkViGGfZmAe5MYy79wSq7ZJXhDLFA5CLA\\\\\\\/eFgbQFjjXp5F4GJcML8Bs\\\\\\\/eizL41Fc7XmZFE+Jjjce9sid3TVj435+g1rNSVlRkyurAhUUJp04BvHVu8W2i42mhljh\\\\\\\/tLqUKtdQnPi297YgLT4CokiYxcZSJX3rIA\\\\\\\/pW9MgIDZOqkeJUxlvxlSRxCRFttuza+sxLaA9PcG8t80fdiOqwQDqhdk4Mn2BZmMgBxj7xHpcWTmg2cFkm0njqOVmh3ashRo6AcUevUNLxt0BBz8f+B8ZZMKMl\\\\\\\/a5YC7l3afES7zj2QKJFf98KWQJsYdWbqsu1tt3GXEmAB\\\\\\\/X\\\\\\\/H1X07bCtn74cyWaCH989oh8nUXerhRxrgdJ0\\\\\\\/qf5x8PP1ikPipfuRaLP72gzh3qYZ1+K1ZyDXD1J\\\\\\\/quc5Gz39lIEOYNF3\\\\\\\/Y5LZXGTWcf46tR\\\\\\\/9PtVpsdAgX8jWfvUUew71EZPdcCh6MFwxY4iCsAf1pJOeH3i\\\\\\\/GP2Qh46Uf8XpQe5Mh8foYVSR8qljpsvRioZTWp9DsV62ca7VwiGr3gV00OrZFBDKRDyFgeXcQwbQf4ZhfqYGkho57FU5acYXFEeZieqCfL0Xk6Al\\\\\\\/mSivch5MeliXFxZx7jCIzDNo98E6YkXOjXHrcT\\\\\\\/yLKmkca4fDbSXoVwuubHf1AgEDJLPrzNbJJnkfVdrv9dSaSlCnYDWLSNWQ+fwIQgdPrVWxNQ\\\\\\\/Hmqd9nGEwVAShB7Pz3mrey6G8AE0KtIlWRSE\\\\\\\/t54NJfcy2iTuD0ljZIkjEFLvwiRmS\\\\\\\/f\\\\\\\/DfJu9kKGq4fNeUf4+SRVxTx5fuO\\\\\\\/OI8q1YEW6FjuG5pSC0\\\\\\\/ThFGotfke4lWTb4xxQSdHCTp0DOhqx2UaQ53JviOaCHThf02EQtOh5Ex7DIovnX4lDUuJGnaiFt3UFSu4FBlwCeG3005mlMYOjJPThZf9d2N56XNxsGkD1SIaGN5UmEnhQ187DNoVDhA4e2uSM0kBg6VUIigZdB\\\\\\\/ESwDZIVjnJSbLJ6aIWuW21oh6EAaoSApxUeEQl8H48YjwKaFzouxV5UfymwvHAY2tk4bo9MbM+fcbGJ141+pDdb0Kx1FgP52k+YMaXjO1vbbusZJIrsXVNtx7BHUio+SPl8kBzOuSHPyN36+O1IykDzLGxlcw+fK+C4A1xugg8S6QKR8L6YHVacgNKK5hZT1mZLmMLMgv\\\\\\\/53gXqD1lFgcKNToZG5JlujuDMKIaOd\\\\\\\/EDtP5bi3A02BMElq6pdCzH3K7mhXM5xzElqYyU\\\\\\\/jY\\\\\\\/a\\\\\\\/2LURwF4WVch2G831vvy5ViZwRuAIxh8l4GI56SHXpJpcB+R4kQYlrajd9eKJAIGBUBlmssGmhyoCqFoBEhiW6IvZpE4eRVwUyzMBxzT\\\\\\\/lWPSWrrBVg1bKVPGnepP8SC5\\\\\\\/SKoDz4gbfPspZPQbm4zGQTjF0PsVnUAUnouCLT8p\\\\\\\/DkApZJocv7c6UZZjR6JBgHyfKrjcZExsT3yfoLSKPZh2eZJ79f8xt\\\\\\\/OQdGPkAA18uGjeSmcbmTdPYM6sRshx\\\\\\\/q5aUV9xVgrNxHpQwESMEYq3P8j07EmwSytFq1fr7uN+neHcoyhz\\\\\\\/\\\\\\\/MYlfdwfdWT3qBmXg1VVcxuyQ1Xnp1AXH8BinNUBhTGrZiZ19hW51RclJ\\\\\\\/S7DLizRTNPRr1DcXoWiUMx5sRzPLMRT0OgXLx5fUqZMVgkEdBYpjAXPt7H4zJ++TCnE7XHrOR5xMB7vjEBdZRnL4kHFrNII14QVm8t7N6ofQuOoFwQY\\\\\\\/aTzoab5mjxlpXutHPtikoDbM4cjESk5xJi16XQ3nq74qm5nMs3AHHrqU6AzyHaw52KEDOcMBsKETAjXRI1ueTUI7F99Q3anmahczImHEfTIiSppJ7L\\\\\\\/BnW8cO5p5NgN3WLEv0qyHMNwkutk4MrRQxvQoEEj9lWrzUlxYQe6bZUX+k18gmUVM826K1wunMv6uxUU7qlkpXbbN\\\\\\\/q1aPZNFgEQ2PxEwEOxywMzdAnY12nFsDawtrYi0ccoE6\\\\\\\/x5v5\\\\\\\/Fjqz7AHa1SCczuR5B1923E1+ZfPt7bEvI2J3x356S0KeAM30pBRwoZlIQxDb9XMBfvyga1YtqSeUUyFVR54UjDvm4Kp2ib1Blna5GPRZEiHU+L\\\\\\\/wWlOFFZoNYFUfsaU0oxGYnF6sDDpxaAyEyjBwNkIHa5OL06asBLGAovyEPvWFFG5lSIpt8hSj34U5K2g+8Goe8TtRbSbTAWH4cnPFGXqlarzdpjGC+qj3ao0cDAuj3ARUTxF+8E7Q486npjrvc7hblKrVoMsHbek\\\\\\\/FP4dLTh0KfbdBFI3I7CxajHEUZXxq3KXVJAFcBpeDpd1p144APzAlOpFQcnDvXcgfYKoTevh+JF8y5bDaoAWQue9Tfol3JHvi0z9WMOg5QfhRlfpW61UZn1UtjWnmkqXHkvSbV0Y2bqW4RAwi\\\\\\\/xmm4DtHR1g8ptlS9E0bFR\\\\\\\/6RstzZCg95k8mLswtU9HhYxHSuT9pYN1\\\\\\\/WyF9plV07GR6LxG6sOzzdReppMBWnlpKPzZy+Y2sd7TaB2m6ZeuZ940zOj8sLjGikx2L2ibCVfxNxqza6UJzAoYdmwAwDHF81SI0b8T9OFfOWtj2UO7LMIdyIchOWUOgoI+gXOby1OP44btVQ+snPqxEEWnFqIEtqV803XWqAAY\\\\\\\/dDJXrPStcgu0PlqvLQnDURVdowo+2Jo2+ACRGa8puwk5d1mppMXJ9\\\\\\\/STxIgb3fgQ5jndFt+PiDaToJw0AnOi\\\\\\\/RFL1b85OqAgson1NXKgGnFudkwdU1ag0Wm6ypMDEY3A1GWro6Vd2udhpuwYRctpHrRc9neMaiQ9oMRJus7mDR\\\\\\\/fhcCgW8VnQmIvYc0x1Dr2BSX3moya75YEO3Ap1KsCrfyGGQPiMlvL0cBFe1HqznB\\\\\\\/qQp+RnxaLYtwlY461SSqUhAeyWKG0anq4uXfZMSllnkvv2tYNp9U5ss6uwHfu4blh6qXY++ZYTBmKB1RhRCSXbbiq5kb5vnGctr+15jhd313aVxcPBXLoLQd0U1NTLloHs3I9SV0rk0A7TSyJgl65BUZEBaYgLDGQTe6VaSqSmZsA\\\\\\\/rW7Fi\\\\\\\/KUILkzowXNj+CVIMR4C8wTTP9LsYWZ6m6PFljqHKZLZ5BPHZM\\\\\\\/qe3oHnMgWq36eus3Vw7VDtIGLqYI0OuyIIangyAJy\\\\\\\/0+FDSuvpTtt+DQH0WjOUWij5pD\\\\\\\/DMLY13blYRY8PhO4XDLJpOZYdmwoARUXC7APK9anTmlWF7yiP9FsIoHHzthJJ3mU4ssCSLJahtKl3a0qX2+A+sSLn4YvIhjf4SKCXaGpL7eAru7NkMfzd8EPZkKnfSxnSaT59zDFhmqLpai9VT0BKj22gOxgcWed3ieZdZhM8GBzat\\\\\\\/tfDEYuYUSnGcsxgO266Q5siTESjKFkzH764xvCGhnR+3\\\\\\\/GsXOVE1T97W9Qo7YPvYwiO1aQDdxKhEErsDukCeUPPyXYTAKje9UFAcjP4oozwqlUhFMXt1q9d4O+SDlpSL+zQnZzopyrUv4EitodzPz8Pt8AXMV5pu1SKBJW1T8F3vxKUxEi7+H4OvIujBVFexrIYm4CMUu00EA10QBWJ\\\\\\\/RgsDkadMgBNqjAkIHXqFNr55BtAii\\\\\\\/xEINvIcEkpjEbhpAdRwLMEPZ\\\\\\\/XCHSLGbifviEMX5YA8tR6RaEiL4SoiGlZQLdzcXzksuiL1My2gkLObvaibQDtkqODGhy8+09qe7DknIWgh8exUts7yrvD85TnSlgbF8t16yoe90OqWQqXrc1EamFkrzXYeN58lUZFbXCbXDzxd344NjLs7+wCGFG6mUdl5mTdGAuHEsluSo9PpM6VnQSBXN8DEKBijHsxuZRE9u2UEB9CY+7fS0ILUk7LqUiHdSVF1Y11ZZ5pU7SmeJqWWx0JJW\\\\\\\/\\\\\\\/ojEEtAejQy\\\\\\\/8jGDFi3KJTiBj5zm9xpg2GLZxW3mjBFma58Hh7JE6PCPZPFdd7z5\\\\\\\/6gsvYB8\\\\\\\/lBU4yTSZyU7NyroLUPuZVFv3cGh0Hvghg2bu7MIWwr8bmpT4alKz60EZkMEBbPTWmdu47js+GK4jUQSFV21KOVBYv\\\\\\\/zJnTzBdZUJBqiGmaXlf52zl7rbpqlLig\\\\\\\/ALUHWfq2EC7LFLV3uYA4L06YTtl58\\\\\\\/cKW9\\\\\\\/3q7DHXu6IpTnjHJfVGc2e1+Sd\\\\\\\/jm2\\\\\\\/IHa0UQuGRxlJO6\\\\\\\/8kxguXU3GY+kF\\\\\\\/POp5SjVnppDcKzpUHoL0FmKdcEBL0b60fIXy6oPLKd9JHreM\\\\\\\/eptERJWa9T\\\\\\\/2QgRNH2q3gyuwBV24vJ9rcEYKOkQG8jrQr5\\\\\\\/rF+OXiSdE1IC+xUwza8wz1OdvSTy7hCKBfC\\\\\\\/SPQ0obJl3\\\\\\\/xWnMvZU01IZmkguY33X4xoCne\\\\\\\/cIRI0who02CpiBXbQk1Vl+bfQ71LGVYX2rTbuUnY9a34OHSZfPBBBMtXd9vR0tJIstEBhT4TLxkEuyHj1d8NRbhfi8uE+EFhzcwMfNnGNSZUMVhPOkFU1JQx7xiQ61YOgTM5klZsP6u3xzYyKEOI\\\\\\\/bzzlcC4fe7cr5kK36vDVEUHIVBnOD6BauxKFLxnir+\\\\\\\/byrJiRY3vrPLH6Jim8+R8zoeD1oSQxWqCdLykgS+EF\\\\\\\/1Bz\\\\\\\/V2hNZKSHeUNP2oykL7TSu4GKdz1nPNxRXQ29FhwZyeZ+wdedX8B5W2AjNP2LWhsiR\\\\\\\/G93\\\\\\\/YKXYz9n6sTxbpy5XXjEGSpf\\\\\\\/eHjxSDZBguhFrwA2Wv\\\\\\\/sm6LkmnpnA7JHjJH8Zk437lhKLivoIp7ixnq28LcrIFbRsx+lDs4+nuPJtdbiMbowa9CerxvE94FeLqPLQbH0ZL89G6uFmlF87k9lrCwX0U4P\\\\\\\/THZ01fic5vL3nRDOg0MWR8vgIXtqp4Nk+vOuF9LCTv2ucQKdauxrcmVKJpUwu14yOLxhyGtRGdqolre7E\\\\\\\/4XX2GymaZZT1tfJHWTbioykzZiEGgvKb7HK7NXB6zxBsBdrk5\\\\\\\/t1rMbgiShXWQOfcLjSdZyTjzJxE\\\\\\\/COTYhXwqj9SA2LWm9SgbPqXy25vXgaY4lGvXh2agii6PTy\\\\\\\/eDVBhlbT41h43CLk0YY8\\\\\\\/Et1WQGHZs2Pk9o2PoOKkdUxy0s36lfmmD135ChA2NSc9srDXFSVtTFEUgNWsClDcnGqIyjSU17DwreYyIRD2xrHB944CZb1BU2rVy29wr\\\\\\\/OFIJ\\\\\\\/Tb8Lt5fO5rxsBlj353L0OQx2Pi+A0MphFizqOdUUXhO9OJC5h13HwMWSLjq5H70BhIGlpmXokYMuakZ9OUvT6Zy6AXfd1ph\\\\\\\/ZdeWzFJ7RDosVvOid8O2lqJWr1uVFo4RrVonVZ4gjzp+BQwCB9H8ibVAlQiQK4MCTESG9nHdoE19iHZT1Bi0GBBO1WqXDAzbudecINfPVPSx29c7z9qCd5FsrGthIkSigReXAz89zNEfM\\\\\\\/dLhcRAw\\\\\\\/i52a5GNRfI9DJK+orPvTWOg\\\\\\\/0YKMOcqCuSy4o1fD0AzFpT\\\\\\\/gKHtUM1mDJuTZ9GI6qEtqzb2E3ykIPgH87IBOlh0GNgFtPHHQOxajDC1A2AL\\\\\\\/+ImRPjKA0+GgMdzwH3txsVHCsDAxDiC+CzqOAsXF91QJ95SodBSmL3nQ+x9YaumsBVeqYJGJr8POex1ogJz9ng0RC2b7A2P3douByLyT\\\\\\\/FMdPgDnTYcutv0oRKu5X5olLhDCsEX8odrnQrvJaLo\\\\\\\/SO6vXTLUziG+tssS53d1Mu1vVL7MR+vbuNMKLgMPsivIFFZGj\\\\\\\/SXPtl\\\\\\\/BNOG5mx5NCy4YMKBOZK2Y\\\\\\\/3mT4cIF27LogVgLxpndfmz1o45dB9VYcNA9cSa0gBHCvlt6p5FH5g6\\\\\\\/51cCy\\\\\\\/mYbivxcG3jyCawJ9qXg5ok67pJcqwFX0egE+Pn+nXQemjeaBtUvlDH6zmYrpiLzkW2MXNnLKEmHb2kSqFMt1maukyi51wK1FoxDHxc1+RJgqpY85P50DCzSlAZH7WupS+YsDN2k3xv8L2Zi139xuEkd8VauLpzq5Hn7tZstj5nHD4S9VXH4Pn5kfY5M+\\\\\\\/Z+OSRAHhdyeQ6W7MpvHWtz5HiotYdW0ZoMLgsa7JqQ9Q+T65X++KMLD+nMADR1RBFnmdWRLCdRk91mviA6EOHJ9U+EQHLUUlJhprXmwXakqjjee9BHXPyNNio0QYs8aTk3ZO41f5\\\\\\\/tetgB48S1\\\\\\\/\\\\\\\/hRGcL9djrkQYRrLRTiPSpwON\\\\\\\/p4wxwTuk3sJg\\\\\\\/N7nE8PeSrLwn97cKeFEVDA2yCE+0PE9r58vT4f0pXsEebrWxH0K8OOAqoIFkt0fsalUFZT28PmGzwlUzLf+baS\\\\\\\/HrhQogsv9LQpZDujcZb9AZLfCf\\\\\\\/+YVhik5UdKEKkjyiynyzjT96tqXPYSx7gsN+hsRwYrfGOZCABzt02CZFui7arWWx+KOh\\\\\\\/lXlOAjziTve\\\\\\\/rhMGn\\\\\\\/0pxUCfum92IzNISoabFI\\\\\\\/NgrKOLhaIK1QVGmlwJwS3SJiensp0TLQJKZi0H8x3EjwRDLQHXIEvFM0oo4HoxsGWB\\\\\\\/3GDfUBh82J+M8i6WvsDMY2\\\\\\\/a\\\\\\\/zLi6rGO+C1ibOHR9PbnTqkXvDc1cRGkn\\\\\\\/Ux3+RLOsy2CY0ssJEAQ6ijOuYdX7z9QMejmWRqT7nkmCqJ0P6ROEveqLKsF\\\\\\\/3oZWIcjuwBkmM3yrsHcA90LXicUkpIvOnKfytr3DOfF8MamtfVd5VGHbFohCkHS7hHz\\\\\\\/jSWXI6HwWyjuVUNEb0RI8Feg0rWYbL16VwfQPlAnDfbSIHOt0E\\\\\\\/cQQCPybmpQrCZhjeP8rvD\\\\\\\/m9lcR9ApoMp0Fq7B+7IY1iLzxle7fkGOSaHlE4KYI8kyDaMnHLZsFAapGaHo1LoPmN7FXZbCitkg5Itj+hxa+pzdRI62LzQi93dLwlzl5QT2\\\\\\\/rO4MVkANhUmmSwC3PeeZTvAb+DU37DVwYEWR7Ji9tvl5Nardcy1lGx5yAQHT9RT1n1ZHx4ARrk+ApI\\\\\\\/3vgiQ6uCftp\\\\\\\/uy85cSm92cvhTTUTHKNnZK16OvvxVRs8yDMHCyGxa9BJAIiAs6Vqp+ZnI1J6WAjvmsZbOuNEOdkjCYg2YHNGcuoVM0YoEEA7T8ZhBC4P5hMcg438+JeFD1JydLRgfy0kbPuczBKU00E3vbJ9RaM4Zm6+ZE8Bv+LEEo0eRFFcUdVzj+jH6n6T\\\\\\\/Y\\\\\\\/C13sqg10v1N\\\\\\\/UgOYNqhXXdbd0TfS\\\\\\\/mcyrzm9c6z91tS48kkLnQMTpXK+4zqFAgDYTuSt0+nxEaqNWQoKSfqMagrxPYSbbyDFT0bQ5Ds2yymEg2oMAEY1y+VeL+CrFEOQcWYHflrh0dYNPjvxHvcHPKHMiB5GH+hTuFVgAhwDSRHOi5kZ5CStQJIdS2kHe4HhxjQMlB3KJcHu2+NHPMposVmDsz\\\\\\\/AiwgkdelG1JH4I3gM9smqAmGIJXSDjXtSRepcORqzO9qvv0P6UIR\\\\\\\/uELJXp+4tZp\\\\\\\/c7oOavowmZsGGbNszbMeQBADzgaUbntt0SssayxgP1GUKSNj0FTtyMq4BztD9TfkGxu2WoWKAKdmH5WKjazwVhJ\\\\\\\/JU14V6gY5Dtk8n6U23J17pJuX3qcZaRD8PpYC6tb2K+7F5lbZ6JXOknyTGFkYaDInR6edbq1EFLBU1K2T8n8MhilKYAGCRzWRi8Nhkm3Lx4otVk5+ULBt+BOg+ACZ7GruU8H4GLwyHvivBG+07eZIOFDjdzc681qULXO0lU+Y\\\\\\\/pV\\\\\\\/GSkvSHgIgkvt+20wLQcn6W3bL1jtegMORS\\\\\\\/MnyusJSG3YRa8\\\\\\\/++n+QVXjD6RROhQEkYPYgpY9\\\\\\\/od2gyNc2G2gXStx8LnXxABxgUlysfJalKDSohdryGGyfQ0mQimb+m8me5L8xiEtpdfWayDYuI0VNdazGWKXBxTOXdBOjYQNKLCfSBFG+a69MHWoFG9UIjTen\\\\\\\/ZTF2crsXjqVvpmbmXKW4ACD+\\\\\\\/szw3MTI3xJyOTJDQ7HzcUWXz86ySomsSKAYMb4Dc7bs+57LwhMNgWseT4cRfxw6PjnVXNbUOCq236z8m0k3uGLpppNI61b\\\\\\\/g2OWJ\\\\\\\/rFYHAiDZNqLku8w\\\\\\\/F5cpQz4aqulg+FdmLSByWsSh8n7BDPo3r3IOZuF0bt5QXt3261mM4E9zGGVq9aQ5K3SVVE6d5g6rN5gyjGTymoJPOd9cMaIWFp9jpDTsn0FbOS6rKz6QgDjWA8MRWBinviHXZ3+sQgiOgXh4yoDw4C6dfMwgJwj2ZIvQ9t\\\\\\\/+7f0HzEa7D3iG7XJ12Mk+icVfutZIRYuUImrVQCxMuU+t91KkIHJa17csUx0QMssmwub7\\\\\\\/TZUsf6b9DG2g8yiST+Mbtp42mKdATbV5z3lwxPtVWPZ1MsKNzf4EjwO1xI5PYjREgoSaZlIZoqx\\\\\\\/g2eJqGwsjZeqR7mkaG9p3nx6y7RQCsi1AN6jgqG+i3US4z5THDIQ9MXIl0HogIeXsPOsFsXEwLSbBLtNk6pgI9IE\\\\\\\/nWTDhEKle6cqRu4xKdgmfSNFdrVoNxLu\\\\\\\/JWVi+b06GbVbSHmOtPODEG4isBh8TdDI6gMg1mz3CDgyB3EpFBwCpZtjNOO4J4WBeK0tjrgduG78sxVgEe1naPkSNPOkI4iLt+3rF1sr7C+5lmguquv5y\\\\\\\/msP0x1ihqwSxZHdQWOQOP\\\\\\\/tNzfP\\\\\\\/0hSZtK0YrZV5sDSaMrodsq7UDn1yo7qfKnFAM1qnCtqr6afabqwrx7zk\\\\\\\/T7d9oAcdwDbSgtOiyGkEdf+Pi8Z54t2OXiKJ4h80NGEnpQNT4D0dJ37O+gq2J9TdV\\\\\\\/i7rulICMaZs\\\\\\\/dLk0ZI\\\\\\\/Qt4svuSodrPcgBh\\\\\\\/XkH11OU9ZryF2qXOn7jKGfoxpyv6CxktEwml9y5CmGdK2A9SN0V2J9Yjg2lJlIVLGfFPiSxEUs28cboOZ+M1oFJfRm46NMyQ9frsAeguYgL28GLd05esdxe5sHc625wEdvZtyC97NGbjJawenxBenv3N5H6mElFaazoDYsBEQmAeMeHGg87GbkHCGWMMgtnBRikFrc0nPSC3cWZrdo8GaN5Yuuyd5YtuIcJ\\\\\\\/om5Udg4A9fodu8PSf80fvDAXs5e4QBddk0hULmCfLeX65I\\\\\\\/gUIwDr4TZrs1E52LlB9PkEgxKqbKn8hBrAJwqM3cVabYwEDh0zp3dBWTznXY7atRHrvfPVVCub+jTRGoIrz8d+xIkqBUQPLEYrSRSq8gn3v8plT17IezWcGtqKG\\\\\\\/1r+UZSzo+CWn6InC1OpRp9sXXni4qoTpATHcC5cWghBRba8hgJecOJWGBn+cqedKPrWv7SwQvEaSdbVB0WVRK7JX+ReF3elIt+5Q\\\\\\\/ddPz8YYpevd0k1oEHN8Kdf54wazDsqWeWkgdmyMXhIlnP5Q4OQAVwCBjFg4jyewv4Nn3iue+SkOac33WimPEQ5hluQanEyhGhBz951Y07mErNsWWc0jVJAZV\\\\\\\/ihu0+cJL4M2bioVoEvyvu1Gt8+FScZunH8zusQR2R704rBCQxMc4OPvvtjKTcK7gmG4Nz3bnj\\\\\\\/B++qyEbnAbt8SHk8uCIkHXjXu8KJa6YqC+D4wt62fBOfRvm\\\\\\\/NgSjwHbID3Y7Pr1F2yMxAzfjGkVrZPg5tCTVehvZBNxUBrbIiaIRH9sKxEAvBNi1ZwqSCST2a0nbWtdmr+KoezgcdOPQRKN4xHzW1MysIRJe6oJGt8bya\\\\\\\/y0rQ7c2e+lc237inf+oM\\\\\\\/1UUQaKiFc2mKrdVN6BIgsxArwtbRFnVtEk6R335QfcSk0VAYeC5+X5os8M6jXg6yqgo03Gw7nNui52uA4zQQZzia\\\\\\\/cEJgJmiFP2CGdTSqoDxICUxWgcC+SZU3oKU5LwEBI9d81sJBN0Mns+Xy+hGsyoQ4F3PsvnNWKQeUPPMpfmeapQao0zGjYVBu0xK65YufJ6wcJIDMMcVgClhSXI5\\\\\\\/6nCpqGZ7jSL520AmIvnxQXWe7w2JhZq2vDrfJmL9W6EyJD8fVXEnp6CG+yLOwEnzIS7bc5BaCYwPAXAjR\\\\\\\/0Q7HkQkOPJs1UvAM6L4g0PqiBT8t8dPZhU\\\\\\\/VFusmRJ8kRm0hhEhx22LYQPqY5bb8rOJds81Ov+KcN0WqSifeqxmRkwjYEOfBVO\\\\\\\/LpaKvG\\\\\\\/njPk803HAXJHKS0Q\\\\\\\/\\\\\\\/N5qW1he7ni8LsGAL0iRB0cDdnEhpMQxl\\\\\\\/MJJRqENA1b7SwZwCm1KPqD26Pkhc+Zq6ftNkS9qgcwXA1ol8JEw94sanCs+KQUdgE1+rx7qX+4R61BklZsbaAwhbmdozTEnfNWVLSWtUrK0o8JOaib9xs7FzFSPOo8ZsiycvZTHmdfLF77055qdftirj06o0m4lAwFnZKE86B68I+pSNvqYOVfRJWNGjlrUOhDoKvWQ5K2goQgwKzd3lFIJp23ZXbbqT7rywgB0Z9acTtbwzweNSCLCLF1B1fQAw7QvlK7A2boaVJOs8+MddQiWKoSyi7uq20aKbN5dbgHuIt4vJK\\\\\\\/fEU7LuYv5DJtyrn2p0HNu6PL\\\\\\\/RXtbyblTj0GiS2ir3sI2kpPfuVxEM28HYXhUE8\\\\\\\/G+77mZ9TOgGT44epwMKBn9IPnhAP+N42Z+Bpa++Dy1Qu5OKExmmbvS+g2h3310fvk72FoQUU8\\\\\\\/xRWLihXbwEj7zrw1o1M+V6UB4w+dIzh9cmE4hfq+vzQDhLz74R\\\\\\\/HFzPxWsa2TEO3ocl0i1\\\\\\\/Nm2G76v9JTC3fnAvLpdpBLTmkq9oYP2sz\\\\\\\/dJL8EnCJ6rtWOsoPL9vtcIhKskkAYshsfka9vNqS50WbiMcyWhGis8DreCkopIx0I1vJPM5t7tyIi0Swxc6YbMe7bazhFJ05BtG0tTeDu1NlYqWgXfA6dJgFpXQu6Wp5yMXqak\\\\\\\/r1Av0EaRmgQux9C6\\\\\\\/0lpw2tUdgR9mNtLU9RiSRmnwId+iKAE5TleTKP4j76KN66o6WqvqdbHdkcnrtew9imUjuER5H3Mgac5uANeZejRu+2nYvyStDntEHVFhLV8E2mcEBlvCZJO8cS7\\\\\\\/vmjS5YsnBykuyTMTIrmHMyXARLL3gLxukrhZ\\\\\\\/T26wj8uLzoRObzxd2uZtq\\\\\\\/r8FguuN\\\\\\\/gsIHCXQsxWvVg0KHCdRFc2u56kY9COelG4GtdnZQj4CIE93\\\\\\\/0DCcf+gx+XAdd0YhN4IRppL+rlfKgtLZn227hyWrIKbD+B9iDI5KOUEluN2F2yBxP7JfnEi7jQ+tteuewVpsmR8PMrqr\\\\\\\/WdGFzg1auC3O22lCiXYN4SlricwANGED\\\\\\\/9bs2ZRTI7n8E+A3jKyeeD8wsJFrD7WpZIpqQfWJchXQ+PA0n\\\\\\\/Z\\\\\\\/rZt1NI78essWtwMfkKGobDF2ucV5e44xXBMFCBZZrDIbeEl4KiOAQTMkcLR4qSa7fawiV566LjGynwWVYxwcJ9CTQWg4fiaYvyJSmN3gk5BK0kUxHn029DYdiT7mQBC17LKF6LT0OqkXoQMQLB8YTLC34770pt2ZqlWGAIjSaUGGC76Y01vT1h7rXlbD66zcd4i1LnUSeCMwUL\\\\\\\/0hlZNrBqK7h6+1ffVnokwhY\\\\\\\/vKWXDfOw6+YMQRLn0t5ibsgkxAntJpKkO2o4Ld0q38rEIP0eFrWU2Zs4Kophd+xXJ3v7QZ19Xnqq5N9z6RJ\\\\\\\/xywnUS4NE+\\\\\\\/1l+lgXCInTPBzL1ZX0Z3Slz2VMjxCsv29PIAg7scgZOBSZXIUbFpniIUwyNtrVzlIIdA6HEb1OiL0gEtJjTRe\\\\\\\/Q0BLTT\\\\\\\/GR1ZTs6HGbvOnsJAM\\\\\\\/YLrO5R55HWzdWNk9MOsE+vYJLdvBqBjWikAWfwzfZgfQqz+xlGqC9RZq1vDN2Law\\\\\\\/nswkkwo1wDIAmzemgYesX47bCVAUMpG7LHfas312vDfuKySX1E2e2CYTw2H4pKOHz6pq9NpTv+whVYZ8TTqPOSjWMpQgNdZVg3EfWE+gtB8M5pHqXyeIX0\\\\\\\/NN0snX3\\\\\\\/CkMG8K7XZplY4V5TDBq9c0qUX1oxWLyiChz3uK8UFJ01y2Qz4HeZ5ogNyEnEkftb3Btk183gPAcTtT2SvyldK3SAWDmMct3cxisnrS58nTC\\\\\\\/Hmdqis2gjmsJFcWh4nlRM9HlXvLDJANoRbk9gU6kV3gZIrhXyO9YpscnzpU51zYEl0cTf84mAIQ3Femwx8L7VPn3oNwasIbyio0x8xTUuFuuv+SK9dnYpPoChQRFbjip2Vn5ENAMjyOCJyjM\\\\\\\/XjZqkVPi2dWW8\\\\\\\/PIrX3i\\\\\\\/aq5IHdzHztQ9RGYCU2Ij1lZNp+\\\\\\\/JDwzdCl4KOrw2oLf9ostP6R\\\\\\\/gfVVuxUxTG0snfEl4AZ1y7llRlx+gF8IIhu1J6\\\\\\\/66Rjktp\\\\\\\/ea6+FUBVsbWtHDRE73HIL15J5mUW5kOXYdcGQ2Fsi1DuEnIUY\\\\\\\/VPAJfyqi2C524mCsia\\\\\\\/alljj1nw0j04u3t6GD8sUi33D1NXy2y3xMnRLUYH1yLrmoQpqxA15S3UsR+EBo28IwqPznfmP3tAewmu7Yh\\\\\\\/PPlAsQruAPnOgt0xj8BEdIU2F9IoiYI4W9A5u2e+gMjZP22VhoympHMI32rgH95gso5x7Fkg5ONsJHeX0k55vPaqnPxYAOtdCMY\\\\\\\/49TDxgb25tjknBHT3hw2meeRUoLTqmQKzoBhqBayKNvxN2b1F0dPhv1icMC9TWM4ympiF5BGIms8P+TvtVRTwdvr6gYY4HZN9RvxcAaPqHLGwLqll9NBnW0gwYcmKJSWvtWASuNEL+dlnRHaL9Tk9aCbFETsQOScRxBj6Jw7YsyDRVYeuY3mG8pTPaAiQrJcoeePKi6srAv7h0XaAuG70A4n+SiIPfI52owjeDppRGQ+TfXZ96Ncyi3vVQt8HNOhgMPA6XmBx77Ox1rqViDHVVPj8yvxs0jiEdVhrqIx+Eo5xoEOm1joAL27M7lL0bisUflrHePmaxV+vdDoo6wBQxdy686OxX5ZI06zkFDZtzqaMZ6wxfItrRKcbNbcbOO7k0\\\\\\\/q+Tp1DQ77WQbfSKdfcsRjiLY9c\\\\\\\/waIm7efexSZ+VP3a0eYEo2Th9pzB4cfG8vYi+e4M4\\\\\\\/lfQLYN1pLZdI3D2W2AFV1qk5Bk0DJDXP\\\\\\\/eVsn+BLhyxd9pZnCWEI3WKNIhT5N9tFka3t2xuQzjg3135kkTsgwmQCHMXRzp2X5jat+mxmKnXNrmFIuNpdKW6Gz6ycQjW+\\\\\\\/pYSfydv1n\\\\\\\/mnu06pnIHGDnoSI14jEKMGUcBvabDkDl9DPH+HZOH5mZb5yH\\\\\\\/YtWgj9pQO8tphSGHMjZT7OBril41mgahCNanm15AhfhtV\\\\\\\/R8\\\\\\\/jPYDNME75YClh+9GELUB3PfjnHnhx77HmAwL1fq9MtVz4iJqmUlOfL5HIQ0s08cj47rPk4xlQ81JB6QcQ\\\\\\\/b4GRf0LvRq5bfBzmUaPbxtsGgTemK19R0tszR5XfyNcEIfbhV6T7zgmXRv6hMV1ifm0lOHfrhiCuxJW9En3ltcOhIKEqPnE\\\\\\\/2HOQtaHFxwhLHrn+fhq2BpgPmbo\\\\\\\/xWXgLWdxdKA\\\\\\\/HIPelTK4Q9hUmy14Ec1BXHHMR84mxo+4VxnJDjaNKHtgb\\\\\\\/0cmSDij42dhGjYE54JUuMHCLT2qxGqFqedKEiM5KWABtieA6qDHV++3QyYJwfTNFcxikMYIhneJkv69XEhgSXFCI9NiHCzLBvdT78bgHLeB6KgSkJhNiIZdM\\\\\\\/4JJYjaaZ8JKzT6HFBMj6Va5s7dJgn8MeWS2WIWwkShWneh9k5zPhVf86VXk+ppAsdZdK4BuWAhj02WTPBt5ETfrtRlR9WzSbNxgSL9PHek5CKS9CTKplZcjhCYB\\\\\\\/1RbsXI79x9TvqZcwRblZvooFzSfaCfPW2ZZBm2C\\\\\\\/E7\\\\\\\/R\\\\\\\/01MfVCJcys6Z+axCqcWMMx8Osyk26X1trao78xwuYZ3HMVUUH\\\\\\\/EXvfC0C8LspZQGvcRQ6WOMfohsvH5\\\\\\\/fSOs+8G1VCjckGP16GG6GYHhM7TSuZGS9Dq+9cuZ0PxanbdxbGXnr6E0ZKvhhFzyPcYXVszEyAOw84m7ZPopGS\\\\\\\/jWZdc6J+qaZE5r\\\\\\\/zt00R0IooJ89WPz19eINBjGMoIoXrkXU+nYVHh3eYXqmIANgNhVMOXiSquWgxv2wpUsWaIsnmCZMqT7Ljh3\\\\\\\/BwE9b1YzI92LWYXfGlMIKECu4y+3Ib2yoRC2nM++uxh0o4rlb1nHu8IoXa5g9EYs95zxZUxJ94vvyCVKTyNOBW6P8GYBYw+ZUXxaqLwBhur4tVFrQLbOqt91zut3txkcWsbMz6Zc5NaMhXO8DS49sLrSf9bjn6ojS3wtTOFkXhHum67qkaNCFwu4XrV9KESxnTQ8r6tHG4FsJHafJ3HLvBeNE62jMRw8yTHHy6mt2RAy09ehkEXrxf1uGJbq1\\\\\\\/1YryjyGE8f8KfehbcaUs1gYF9NSwILjuWfKPKcQK7VIUi4ieBK1yeH3gXIE1HREfufVXstK17jd1DlEREVoNmiynctHGxs4RSYku52nM8ucFjwA+Cggp2q9E+2Z2BiocFkk2x2FFecgY2QIYxx6a9pVsUZZa4B093EkFMF17eRXzZV\\\\\\\/RbfR1DO7phucUuvda9piYKkukTknJl+LpFp8tImkL1BoE4evbV7IF+rntFacnft7pPWHmNHbdaTN1RCD4pXoyxDWqV9+zyIMfDhugwDRfhX0Q9YzPQPj95D7aeKMqeA\\\\\\\/5Cwqd4FiC\\\\\\\/uOlgpYB7tOHGSXaBcvAgJf\\\\\\\/jltXzh2jzI6ASoDFBTkNf+im49Gz35JS6+qfF6WdxH3XAlkVaDnCmxaIgALpdTo8zKxAnFA8r0BLGuS14cDXfJ5L3g93LoFEAwuYvZAwPjcPJXiUPIZfIqdhBRps9FlkQpZUT5u6WtKGMAA17q+m76RhVyc\\\\\\\/UTGdwNU6t74ZbGlrtFTf52OKZIs9Z79mXBe2aGLnkxvnMmmhWONRiWD1DzMn9IDYNHKNJHoiLw9oic+vtTAE4mqy769CnDExFwjs6ohl7QoIWpnErZf8GFzR+ST6YkLJSOWRqEYUYYGYQmDGV3h5h4bv7eGSdKhA8bCRfFsPmWBEEGYvMSR+bPZf2r+xk6qjOsZbkngTbotnRnniFsdVH6Vc+C0i8iOC8azVOTLzyh+Db4CRLhpRYjgMQIjU54Y4A5F\\\\\\\/APil2YQC1FCGzvM8KvFK30apCRd9+i\\\\\\\/X5dXVvHOCJnhjk8OdZvvLpIQ0Jo6JtrDXFbUEqJbT7NbDrBFgNL5AlsMdussF\\\\\\\/Y4M62q2RUG+6awlXYwmlEhHFyQKtHm2\\\\\\\/\\\\\\\/pPgufubPq7LNzHqdSRETNBJ8YIxyrI7CpU1jhPx9bBiMxTBJGUwdm82GyQ9vX4UhGfsP9HPtynXQTpEFsq+r+PEtZeXKX6Q6RYqbzYFrvDK4XW1VGyo+cShFCopwrIk1FTx+\\\\\\\/g\\\\\\\/PPQbbjINVwNQ7gcQuS4MrkuGH1Y7nkrfjZqIo74VHzAqudUFKtwFzo0LcYiv6iq7khlxe0pf3kFZPgcj2++sIaZTAqfWwP6oK4VxjX1AuOPOVady+d+9OPfsXZRj9E4FA9RgZJUEyeNqHHqXwbWw+8SClBXMrsU7uQdAUvHq1Fq3qh2AOEnSVjug7w0Viq\\\\\\\/\\\\\\\/x\\\\\\\/hAKNLCln1P4GIJgBrTsmp8YeLwbXwuiP1aQKPX\\\\\\\/0XKWlRsbdCXyj2wr5zatQcHb7ttBt9SqtmgTWK\\\\\\\/5YSPNvHXb5cBUL1MOLbL\\\\\\\/2+wttnc4sfp7qxsBqGH\\\\\\\/9EzUdPV3JpBO0lhww8zE4QUkr\\\\\\\/AH00b7wwHacmBg0jkOdTiayQt23Aezo9UYPj4H5Z7UVBCfhXspu3E8RshrZjrPIgZrky+nCnWVb+2ySHKA+xmn4WVwI0ImNTzi+TG0OOvhw9gFJaBK7Wp7NNTnxBOxrXA1biHazThCJVkclxnLOUJn5w2xVK\\\\\\\/laZypHhqDCbYiV95YsOLGAG0Ln2huEars\\\\\\\/kB9Wed1UfDo72Yhq35BH+S\\\\\\\/4N8\\\\\\\/Jc3FCOlA486m2+fAMUO8MYxmyLXYFx0Dg\\\\\\\/ULWCs+5J9kxi2UE+BhUm6ZpkIxChSUSV+yBvYBhHA6dv7XG+iI7C3nW6\\\\\\\/7DT3\\\\\\\/HHG5hWM0iyfzaEXMP91\\\\\\\/bMJZ9wufRdhcWQrsaFe1T\\\\\\\/oNM3W3Q72bEKczCHabqpLrGouySDHFflTE3ZCiX1fg1rFHFQNjwbqnylHf4MtQlWxGbhO7buAL0JgZpWh6+vsAppkW6d+0cTeEkHInjGCsNmZ1gw79CD0Tykwh03lLsMS9DRmPcZQlGXjSa1SSmbkMot2OcuHq2rFrCB+7owY04EoGKkoq4JoN9hph1SeyW0Ih4ZbOQxhYNR2dCuJDE91ZtUSk8VLosfQHGlwSF1B6+d2TwTgbAEBymZK8lD6xlLoUVUoZ4sPEncClNfmJT6g7MufLl8ANS3+3aGY9BVEKAe\\\\\\\/kVnvMjff8Mws31aleV+niWKRJqpbfLuY16q4C22IR2FC5t0xSQS+kGnDZCdtEf5IhzhHQAzyfrfVbDT5d7FMeTXSDk0ncXQhhF\\\\\\\/eaWccmvNvfENbMPY\\\\\\\/TQf0QK1aQayCa5qhWFMw+k4uh9W4jdUOByNZguDsTCXsR1i8OF6FsMqEZQKufqkYxKQfQ+0RCaiZNhoc9RuAxzYL79i2JslOCyIjzOSkmY0SOU1JaR1G6lnl8Sy1VX7D6qjU\\\\\\\/oResHGDYu6bxs1x048Yf\\\\\\\/0vyE9DAz+8TNvZtKFBlX5I\\\\\\\/5jh9fxCTOpSroG3NYMZZcNp+vo2r4HR+F\\\\\\\/V9UWRH725obtBXlXjg0oFkGdi5ah7NHfdasJDeL+MqFOGs9xG1h8QfVhzDHFOCSqyFW4Qvu824d+zvYVeTgv\\\\\\\/85vJvYzl3I+HksnKIUDAc9ZRnYP6GuOHympjuvFSfVab7RR+EGm63BlzH1vaf9w8H\\\\\\\/7E3bbHiCb1imMdeOcfELhk4H5ZkGAOjH+rZd8rlu7oB7MELEsn6ajnv5tNu7r36jUQ2vy+khT6ql+QjKTniciMRiap7U9K5yX61pmDTVdlLWifoE9YUzh2x7dmmNcHw5FzK0XmYwB2dMbO\\\\\\\/eqIczeXDohmxx894D8QoMFVme0YpA6b8k8lWN1AgK0r1uHxo5mafLLVGFz2Lxw3mNx99zfysFuSlAEaZTTZ+ib7ba90IDDiT\\\\\\\/SPAqdJCKXoJz9O3EVKJpOn9VNSTy1SCfNetm63hTjocdVHhxNdjFcJX\\\\\\\/KX1AlxxoFPL1PtH12WyeNZ8Iot3DV6pvY6vqjmt\\\\\\\/idcmW7VUuGogyU6Vov\\\\\\\/B0nsQ0HHp2nzUiHbdDS7u\\\\\\\/PjfgH315c48vBe\\\\\\\/mqFbszt5YL6ry3MZfeQLG77NGOQBPVnMzHMKuirY\\\\\\\/rABfgpbTldivrUe1LQwFBmkO6XiXDLC75w6WavouNHCInT1EnKUFlzgVfYQL4mgC6uyvnNONflemLynvgSobdumu9AtJeqIa21kplVJFX0zlcY3tLrpW\\\\\\\/OWeAPDuazclEGtWit5l7dalaSOe\\\\\\\/zbbee0XXsx0pq541HD2Wh+lhY9Eodl3y5jcnmwd3NyTQkWSmMcf8bH5j4DlRwRX3r57XqbzPpDNXlw1Up6Wn1BQ7wGQ2tVxMUddlLfj0lix6Jlrk\\\\\\\/bMiGHEQiOF5viYsJpHKSuWv6Ej5rJ2rgNNXrBWbJmuM\\\\\\\/WidZZLjy7w3UeVxEr\\\\\\\/3SVBmL7eeAs4YB5H3Nr70GgW0gISnfw6lWUjqRmpwiovnlFiPV4gC\\\\\\\/n+DHP2gyVsfo1+0de7G4OBEVmhRs9TeDnfdqjqMuERwHejDZ03zBZJ+OwrkC1wJuJGtR9jDopE81iAbfPv0gwb0GQNRFjAwS1zm1I\\\\\\\/SubsPsFB9qHqUF41iVyUcrdIj2wKmou0gBE9N++Z5KJnsIlxQu4IzNNRbc5C05aP5bkNvGgyUq3ckV8a6wvUSfZuw3GM9F2X+3snsAj4J0\\\\\\\/pNQp26phC+QcFck3H7u4r0+o4x0TtTReepa2PTtbUavosRFiK2v2FGKJKD2gbv8Awmu5DvV\\\\\\\/puES0swTSgyueiwmsJv9iXQUt+uFoWjYwzzGW49nPwlHyv7hA4ZssHevQnk8yaiT6f0kpv0JKuZXTC3Qu645U9v9iaQDiMhxHc7GUwKeqyL2Hz9hJ0OZWhItcxCmwlYMjOawyl6md8RR\\\\\\\/N6Fomxr6hni+S+bdJdjMuL3q47fCfkbBFa4yRdHU9PAgCkhFs3tkDqRaE8pffyAh3xgQhaNkqOJKzfd0mQXQYQTovlzhAj\\\\\\\/nURMNEoHgrkPKSNUzhRMcRUbJE9dXadalwRJWUnl3fz+4t1TWNcMl5kdluJs1gZ5jEb8kaZNOmOEA9gUJYLjHYzNGlus\\\\\\\/TvlitGKcoIHnqtL8Kp0VNZwi9n\\\\\\\/pHYCczOUdq7CziK+8OGv+r62PfaaXFhAv2WITCy+uWAXojvQou6KzQrQfrPXM4JpNeGZHfU+4\\\\\\\/h8YpIv2vipsWBTawvNf1LkSwE1QiocHvdRMKDGYHOIWZZyde\\\\\\\/ETP1wHFdPb301bfwdBJf8kRp\\\\\\\/jSJQtvovM13kMlXafcumWxZpBnMivA8hMSzGbIdV+\\\\\\\/kOXdCBuJaiTy6cmSrsZ\\\\\\\/j3faGds5PKbFYcbG5k1mC+EHapUDWyPluPVLl88YgDzZPYrR7dNigTpCMUEZPHp7szSygV24tLl9IbANRGlNtmN20P8nh+M3AxyG0rjsxQYxyuwurmnzRMdQuyz92xLdmQONAFIjt6MKzgnuHo6GNc5dYJwOFsujTUs0Tnu0ESkvy2hk08BbNLrBg98H5qAOE4ch\\\\\\\/RFZS0ufKFT6AnpMFFtGw73aY8FTg9lwbm+SVNuekMuNVrWhPVkGD9mGdI3Ghjpu9CPlcMJz9smEQxGASViWW8TLYJ9dFp3TsYAtVuGN3gOuW9JDyZuJ+O0tXW\\\\\\\/udKbjfc5oKHdOYZvp1M9jbsCfznCBvwqNaM04xY0l126jkbuxSsbjZzAFVa\\\\\\\/wI6QsSO2bkm7VpvCCnk3s+yDuLDwIZuR1cx39tOj0GxX+cVcWred6FCcJbyy3RVfmJsswjNV1W2i2MK8IOA8txZcsxiPWR3t7zB06eV7jT7NXVIv51UmUTyY7PFeLcs1flCCE6Pa+gFYbZRr779NPgPQYn47sICa9LWl50jQOavUZVPGMIfiqHapeWKiYaVNWju3A0DxRmaiQTjdYb8cshsUyxdmYAKUXtQsDi0U1\\\\\\\/B2Ubu8vAzuLQIR7l5d8On1exNrDoe5wfZSv4Gj9BWR4v6\\\\\\\/IKd630wGkJanuvGvUWdVYaTtXGHDdANXI3IR+tyhzrizVp0o71r8twNBDuKLse2nnLTW+GVf4yjkXz0wn2GlXHIIPYb9+H5r3mV+yWA8LKjzes7VS\\\\\\\/suAOo+9If8aTrebl9TZZmec9QrOIDLccpOSTLDQiuqdkfhUk310ieVTHtd9HYSDIbJft521aYjN0GhfSd4l1MNmNQU0+yjuIW5JvO4iBt3Ep4XGUYo0qtZynh48kU1BqF8+9kT8GcTsygJIFTkGBuQFfxjDc9sjw5CCsoS\\\\\\\/9qyjqSFUaGs750+srHf5e\\\\\\\/eGpuch36YUbMtBGjkH4X4qlRvbd22CSq91dHozCinMXyjVGHC5IzzI9wtjhS2ArgxgeVidTfiW\\\\\\\/VLmlT3cy521PygH8gxRfqh9+Mqkozqh\\\\\\\/CpQ+OQrVS4TpxDjGKL8jo4BbBeuyIU\\\\\\\/GgAv2KeqmoXnsRDUivLRGuLdzFVP00ZcwEJ5c+nZvhJh4oSD+h6oSffd0RiHsizJj\\\\\\\/e8561fvLWimSMzLoJ+MRFs9dfjhte9n2LP0fdUjC2\\\\\\\/\\\\\\\/z8qKqjhy4NSyucyy8khr+DF8tq2jeWW0j4igV0FB890h0EYTtOs7tJl\\\\\\\/l+RJk0E0PJBANO40+JnYDLT6tgmR3e+sjtAPi3crrxRincEaI5JM7kq3TtAdXafQJEnx6f6Ymk2v0sPfgAZ75K8IiYWbTMkmJdFh7QB\\\\\\\/dg5kHYw\\\\\\\/uOFV23pXmQPbhnwgKkQeT0CnYPrP5Pm9JUWLdKGWKWKqJsATk8jP8kC6mLppXg950\\\\\\\/j5RGZohsI4qVQpxQu11f9BxFxoQAHKRQCNAFzqtXA0NQoHvKKM2whC3+1l9nnyJe6x7hvR5YGpYw7Kq5b7wKd\\\\\\\/txVT+hC+Q+rvh\\\\\\\/aUg86GYJgYwPnzRybqfIlqq42zidFQBDXjYlhMTE+ZdpLaWX1OMEU3JGQmqJN8aIX6QrRUcsYvB8tYcUDpijFCaMsJdIfMLShEQLR8GSf\\\\\\\/RD9aNxMF2sOp9vQ52q\\\\\\\/1uMeOPdyxvXu\\\\\\\/\\\\\\\/3gnb0D5WwfM0omcjiC8PHUDE8Xq7T6YL2MJvwgLP1gfuMUWmZNvW1OAjiE2+xndtqOS5RivKn5sy9sgT+TrzynWSg3p4Qoucc6kRDpUF0gSImpsDZgTZqGd6L4Vtu58KgWcOYB3KB7AcKRm\\\\\\\/QJL4eFUqe5RTPr0HyfZoRDWoI4JIeK4DDiyGJS5E4R408qafKQ01iA7w8f+i7rJv7G+B5Di7LFXsIZ36QX5r0bd43trMryf5cdrBg01EasKTUMqABQ7MNE4FzvieuIR0QFZpk1w79T+NLl9kd2dVTgCUTDmg2plaWrO6yqWMnVwGqpWM8jTnpRuH4CSgbVXQ+4vqUCbvhFX+t7ZQH3era68cIUG9\\\\\\\/oQQsIW+JNQ+\\\\\\\/4FyIlBSX6s3RQE+t8J+\\\\\\\/fVxydH1A7de8pGgPEQOP93KvBnzBjy4Agp55uoGapNBMBKThr+bXZnM\\\\\\\/NN1l4DmAG3MNhV93rGrArYmeWp4lzw5ae563VnV0+3gySHorrnAmlCQt2PLwLGC+RddO8MjDeLCMHO4mwM0AvWIhsAto11NuHzLbV5DksXWn2nSHLO9etRSttlPZ2OKdWZSWoeLh0cmh1SoCPFS\\\\\\\/fiq7iumKaxJhP8ZxmggVtMp5P\\\\\\\/7JqCpXTqm5d7B\\\\\\\/XutDAVufPtoNQSEdzAtsyF3rqgf9x1XcdwDVj1sMp\\\\\\\/PL8mRKwJkslhlfW0+g2iGakk1MlXcanUfM3yG6j9uI\\\\\\\/Et1bvRvzIPJoujDeAAPpHJ3X5O3tDHD746EmFp6khgEEV7ivslC5OtoLoBkCqPJOM7gOXvCEPAk9xtD\\\\\\\/1nJkCFUCCByEyaQsSWMW2QK1zjiD1NHhW8ZlJVGiDPbHu9Z6yQWoU3VkXXM8Px+1vXNM\\\\\\\/SeqfJEBDSAS3IxoGGEPi7o8YQcAOlBgNobw+4UAOkFAY\\\\\\\/suHESFS0t5jQvp2QIsWEUVbEQdwS+Tkj3m01jE8y+VfQ10xST8V9kQOAAZxPVRwkGItf7sbBMCsHTcQXfYCsAwgvvLfaWeuOxppNqxsB5TAMYClC9RxQDUJgw0jvC2AAMzqLLYkVNMkYae44yBGiBNaFRT7QqYFbme+Rwo8BEnjyoUJEouJZ9rFx3ryeWRn\\\\\\\/HEkkv83rHh59V3v7TROAtYQKDEt1K3aqcgcAdgqh5xne55Gckpm8jgydL82r\\\\\\\/t\\\\\\\/SjK2cHStcp3BC\\\\\\\/lAw4htXpCdy0xAguiGNsXm3LFD8uKQ80m4IOQzsAylgNX2YvdGD9gzXvmHWFojSfH8q\\\\\\\/fCzrj+d9Tay\\\\\\\/s3xkWXlH74J9HRD2D\\\\\\\/n7s3m10VtpFP3VtGxb6Wp1FSRw2BNA\\\\\\\/BHs5UsrNXcU1V6PxnRVn78Rmxh1gK8Gx5G3G5thccTO7cTL7ifrs296zQf5zIsNakGTz3M2CF5+uHLhROkUeqxDPSVm+asWBqvT7wweKQ\\\\\\\/3NRJTF6YE0CUgSADDfOprIknj\\\\\\\/llbiNeiaBC\\\\\\\/dZ5JzcoGjZM1koXaOHMQxxUaueQTA33baqQVWSd\\\\\\\/v5J\\\\\\\/Vz2+jzAXR4Gjpx7KQJoOq7fZhb1cTpLaDR6Uzyh9G5gGdvi3R8qf2+AHUgiD9ZI15FdjncEL0o0GfYFA6kT1puYk9nS\\\\\\\/jEo8m\\\\\\\/OYoc\\\\\\\/pXZmkpj3upUtvIa7ht0UN03QFp7\\\\\\\/CE8uzN6Ttf4J28YCYFOehy9hNIYxV8Nm2L\\\\\\\/b+ZX46puEWpZdH\\\\\\\/qWHGww2OSkBXwS0tmkD5sVKRWK8MC3HqaREIj9Ckui1DAhlGviPW4bR7jY5VRG513MIB50B8gv6Llxp6hjqp6XsdlIhCFiylkO28ksEKP2L51\\\\\\\/GFX5444Eg9GSUUkv8ruZqpQfYQImmIIms+1xaezlYZuTQX9Dx06XuqxzVmKRJ36JO\\\\\\\/9FMjs0W9RAQPoMscndyYJrwsDWJknerBhaN+ViPgTBxAeeBuPUtnSFn\\\\\\\/p\\\\\\\/VMeQ6X61yFTULVQumQJWtE1lAF\\\\\\\/KbTERHD3L8ABMSxxtaC2Nh1HiH7uNfiDXymR1L5vZ9SBtHpHkknIErvA2Xde7oEEk0LIjCteWsjT0zAy2AwO9vM0RIvV+\\\\\\\/bnKl\\\\\\\/O9AvjHxp8QO+9NCE5Rjw+1+bHdy\\\\\\\/ZCb\\\\\\\/lVWZBxp1mRpPvEBuaRoPmBulpd5ZMamV++jJUYAJxITyVkyVHUto4IIGVktkdhXsJa6By\\\\\\\/QhJUPATArHrgbYo55EeDetQwh9APYpoVWbfYpbmPm0vMl7cG1iP1EHJhttg0fb2Hc2NigbcJ45OH2agL7M0fFkd2DAljVmFeNh9bViPapXY65ReQjWvX5rSC58UX3v9l4cuX93h3D52HAdmvP8z24B1+AOMGaDcaGCVKyJoV1XAY7UTWecAd2b3hhyuKx450cYgLEST3NJNg4p3jujXWu5B2+TVrzfq0QEg4gQ+vgysXPl5K2MjoFkhtjqfAxND+wa55Qp\\\\\\\/Bj1\\\\\\\/CGsxOQWSRVQOMbWFWUGG9d85EfLhvAdCquKZr12n2B34EVSI64V3I9y2AdsnrWycOL4g\\\\\\\/fVKem6teX6rIQvFUl5\\\\\\\/8cezm5xPzRGDEunWjCbnp2XvUApawHqeK\\\\\\\/EimtTP1+EScXL8xbMt67CM6xHEBPy3wohgPwM6YqkLO8giKg9fAgmLBuQ3+99g07CvBTMGT\\\\\\\/7UNuuv5p+\\\\\\\/tca9kCzCQrIpZUYSChDgA2andXsL4O1T+I+LJaljMesRPUX7DKYkrQxwxJhNmzp05DbZnFo1fZ5OLpgTuZXnv8Z1KsBGdJXTLNfFPr+5WVwi\\\\\\\/KeVpIN+2wYdz7aImxc9q1AcAY4RM54J0jvohxIh5dlu\\\\\\\/1egrvWS\\\\\\\/M96NeLTDpFTLJSa0Z3DbYhNwa7JEfMW1BaLvGr68xRtu8BCMQnSw1\\\\\\\/7wU0BCWS3DqBJ60DNeMdfeB+XsiIYUegGQSlPva7Up+iYs1o06pyt5KVNRN5MxYnkT3VSkx+Npk2eSqz8dWDIbGvr\\\\\\\/4sl2RYVlrZdZ6DQJNK7X6dsrGmxBr2VukTacCb3qa7LDZ15y\\\\\\\/8XeKmTwA9wyVG\\\\\\\/qph2oc6dH\\\\\\\/stpf57XFysIcBptkKIXdgz3PCIGh5Lg2iC4Ofsf8\\\\\\\/\\\\\\\/k+OrRdWfTL\\\\\\\/jKPLEtfrI+bXJeMxoLW0Qr9eHMgY1\\\\\\\/kHwvOryrlRdRsHWmTSUn\\\\\\\/z0ko19RwgyXCikiPNiNX\\\\\\\/x0+VeoNRymbvjk3xSr24qeq+c9cKNOOAIXTP\\\\\\\/ldRQSWAiXUSICmhweWj4gcA\\\\\\\/2fjsm9Y5CV7Injd2ICtd8GgHPMGEhVvQ\\\\\\\/3JAFUoNlZ6JMPD83Y5zyz1jo0u8\\\\\\\/5rk\\\\\\\/HuADBJ4LDnc+4yR1Nw2Gsws7X6KG1U3tFvrawMw35rABOwzKx0S0zRCJYapCNOlxpmbR4QoughJiJSTsHbrS8i9di2MWmBC2TqG3uGVq9qjp\\\\\\\/+9j645iGy6oa1uYcEdNlG49QbJpaA+1agfMPRaZOY8uj5Hq5SStfVdBxDkTYz93I\\\\\\\/gVQPvNPQEapDWujfOKb2fbfT9Y\\\\\\\/cJcGzeSjpAPru6swi1Jh73WkjRz4QUXaSfoJnoLXlHnkvSqE2l2kzN8wJ8v1nnCGGc1SLgUJWUxi31gUCpIlL9mhy9XJdH+a0nBKSgeARGNt+xWmtPEBM2gp2xEmqrND1JhwSoQ\\\\\\\/Yb0oPpIABu6GA5TWZNSkcekUz1soyQvBUGlZwvjpPQyzIMM0dmVM0eI6\\\\\\\/8ZWcoN9cfOhKwUveICHxbiTQs7dg0PxyFm9qZRhHd89ksLZKzDX+K+v04zpMsTRLxvIPSGkZDl4dh0TO1VcgXLOfhBjVOqqcOEhNAR3Eyq86Adao52mKIQI3c53MG+VztJeuziT3J+ax\\\\\\\/uPvUq\\\\\\\/voA1jXr++cbK29913VYPcC74W\\\\\\\/8D9+pdgAhA6Dy4o9IRUX0K6prAY3PXKoWFXeGLEPwB+KqVkzVwAVmA7z82CW5d+dqdp1XKyiueWtEva6cXC1DrPEiIP6y9O8vatcRFsJ6ArW6mNhc8NKlKIZkHWgiswtXEKXg5ADOnoBLxJjNDfCZX3nj35XjbHRRz61m1D9XwwWHhe3b1Lrd+ty0Q2KQVH80s4bZiE1Vj3IkWEFf1BT+AoPzrLT\\\\\\\/Lav8\\\\\\\/NvFPDHP+hnU2b89lMKyVeZxfmd5GgeMtmM+6y8ChyoGtsG\\\\\\\/J5OgfZxaKluFg++++vFALGUYYeW533PCM9I5j9gpTEl0Hmdk+fjjMUGG7TWrfjvCAWvGHwZwzct6UzUGA5vJYG5VyTHg1OAlR5z5sC+25j+ReIE0NqaOJmQLH3yDpqqX6xnTHHyRSbqLGlcX0Th\\\\\\\/UGKVdd8gwwedOSaHjDNMhRqUsiRiHu\\\\\\\/lV7XryKENrfs2nJJiAFYe7+27vzmfn1ilE3qlpGFqmtOQjy1M+WWoIH4muv3Vn+6\\\\\\\/QKb+JuIgq3bq7iwQ4WOqy+ydE5sm+S6BzlDfL688sCpGq9mKHonrxfnfU+0ZZzewTZPlej7fEI6ebgCe+UkgKmYeEKh5Mss7qX\\\\\\\/0NsqsaSGg7gptCTsSPsV+oaNHAniaCQoDipkWn\\\\\\\/yJ+ptATpYRWdTuEVWfLxK+Wz16Qk2yJ3r5DndDC2inCamCH4DfeE6lNDzzMTpBNrFS0p+vR9MtzLH26Arj1Vj6V21llMXDRrMrDWD+E8K4rsM+Xgwjb03bvu\\\\\\\/x8kZck8mdXw1BwNXzYiMOVME7uTXo6DoQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Dl9i\\\\\\\/8b4kuB0HANjW+4pJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c9c30896eb63a6e10d2e014354d2088cd42c2c87c48303ff7cd8c6f016a69dd1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022o1E0OpR2MSE+RYtwQAK5Hzqarornm6og3gmAg2s3jfBYrlqMxjnsrYWsx4SXoL23+iNaybpGLt7wA6YxiEBSQG6hV9i+O7oTPbU+IVqRNXm8MUKAN9YvjKi0haqbZeIPTVY2Q+22s9z+hdhKuaWs00PwCMkq+HxcJKPwJ1+8dIBt9AxoTB2IlIOAYfyxvUYmFVZHpZ0czkZWZnA7FUxHte7\\\\\\\/T36T7Ozx\\\\\\\/zag3Bkkegu67S9pcrxNhjS4NueP4AQRjvKdYLhkNeJJRBiIQsh8RB42mx+959l6rmNxk1WFWI30LUMxdCVyr7qVH6\\\\\\\/W3kBji6go2a5ric1EWgBNjLBmYqqvz4DUbrd2gjVR82DLGwMgMuFtmLKMoK5dZcaisljgI3UfCnWCYPbP6\\\\\\\/Olug1nka7uM85bM0KiGwYFP6J1+6ILoodJlDOamkLXluV9+aStrdLfykyCQ4nSrrk+lqeZB1xrK5HI02s\\\\\\\/zUo4LGp8FymfPK9QFIyPP3sj4o780gae1ioOjLXL7JUUXkqwg8wIZEvkKxm3fVJoef3kPwOolxUeb0ar48\\\\\\\/hWbzhaMR\\\\\\\/4UVF0UZd\\\\\\\/oO2F6Hwwwl1YK8SN06dTBLsvjSRoZIvqzNseTQfXfwXmr40ybvATr3B1g9\\\\\\\/\\\\\\\/+OL8DTiLmYwWnIEdWNaUNRiA\\\\\\\/UxYwgCjeVCUFN6OQIs7PGSEPrrrNw\\\\\\\/\\\\\\\/iiuWQgtOYhxBVAkv5ofVhtmqrSFOfPSgZf6paTXZmkT4mG\\\\\\\/9LH1P1GQ1nuXFEYUNuvba3u5k+BOZG5m4+3aw0HQBlgo1Rd35tUx1vaCN60VYKtQfjsvdv8f+SqwImifdsR6MB5zg6RVQd70BN5VpbzCsUiJ9srJafL8NEowd4DWev8xBJ4pSLs7kaJM7W\\\\\\\/yM1aPJr3WGqZv8nNQK+KuaB65NiNWhGRyB99G4BDxxNDsqGcFT5qloKaZphoMYDBewcgEcX\\\\\\\/7owczvSZC1JXIy8bEl4F0yGzr6ZmZzjucZkmOg4d\\\\\\\/wZrHYo0p0RjlqEOcBqukhxQvua08t+WfzNNYaG6edxQVQ8BoE64Oisbs1la5QLDkjD4CKzLlYcIZ9+yCCjPFoMn64AYyaiYqylrsgTHzFrsPuggDo1zJ2P7vm9OpkD1biXoehnpuKTPNfnOqxoaivAW+XDloHDUiTwICSemPM8dyD6cEvdrpc5kO8MIcFqkATSji4uUByqjxkYHGYWfB4fGqyULdd5N9upPx3rbKhFn7gz6A3HsPO6zAw7XQignHT+JVXWLcEJdQsGi6wjLhEWdIVWvsgDMtwWaL7iiQpiQdZbmzASnRZbeDXkkpapavYADnA8MUc8M5YpFgn4BmrwUsi0qfbiY8RhZXCIOnpXgr5WaQxyUb+zyUxvOtblBR4oNd6zPsa01a5rTwAEVBir02Wsiwwa1tOTB1UkywivosoamRutGZQd54LPElpJCoB\\\\\\\/qxfAZJWvxz+qsNFhnL+w\\\\\\\/SPqEVqTUCYprWDpric69M8cH55lmDW7RpG66Ttmx4hJJsrsrQe07r6gNn4DXR91veS1ouf4yXYJ7uxWv7W50tL65ycEWA6PHuXKHpbUN0y3dKNm\\\\\\\/l+WsZaGZCPIK\\\\\\\/5x\\\\\\\/d4+7RAD880\\\\\\\/SAvvPbB8mGbOk9dm24x01FhIb6YL2gMCKKIuREOfoBl\\\\\\\/9G2lLqg8\\\\\\\/gIuqaYquIlIuaigRLK3lrzJsQ3r51d7PEVCabEeT5DBsdITVABm8jtXGE3rXucfyQt6aQRbHE6EnkNregXXywodQG7SRa7yLdUPEkdfDsbdL+SmJe9BVRTbPh1Qb+CqmYs5Hoahc+KkZediKne0ZmyI5uIDSSSclc5jc7YxM44ZppF+sVPkhP1N8RDS4ld4QYnRxK5+DaGLoIhia+FxP5wCzXF3QSlXSF\\\\\\\/cBP985MQVPtVA+K1wKU\\\\\\\/8MsWmxx\\\\\\\/c3R8hpprduxxkkXlYjq7+hHhqdhQ1tCkff3lha9l\\\\\\\/ivZhb3l5Y77vKf++7muttFYMRJvvOBeRii49IJ94A2kUV4rw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Yq+td2Ecmg+7bdiKzpNrKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a7546012b29bcd57b482866aef98186ebadac8f5ee2605c55e896acc4266ca82\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460197 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2547" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kKg6qZF_blkq-MmqaUcamdYcKc8m4pAwq_hIgiL2rQs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2547" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bI0dX4BSupCuK5eF1BHLTwmPpsPHCDnZGo4xzU9QbHU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2539" title="Genome Sequences of Three Phytopathogenic Species of the Magnaporthaceae Family of Fungi" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2555" title="Markerless Escherichia coli rrn Deletion Strains for Genetic Determination of Ribosomal Binding Sites" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460197" id="node460197" data-pisa="ggg;5/12/2547" data-pisa-master="ggg;g3.115.020362" data-apath="/ggg/5/12/2547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erena A. Edae</span>, <span class="highwire-citation-author" data-delta="1">Robert L. Bowden</span> and <span class="highwire-citation-author" data-delta="2">Jesse Poland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2547-2553;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020362 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460197" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erena A. Edae </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erena%2BA.%2BEdae%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edae%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErena%2BA.%2BEdae%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Erena.Edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">Erena.Edae@ars.usda.gov</a>
<a href="mailto:jpoland@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">jpoland@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert L. Bowden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BBowden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowden%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BL.%2BBowden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Erena.Edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">Erena.Edae@ars.usda.gov</a>
<a href="mailto:jpoland@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">jpoland@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/12/2547" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460197" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2547.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460197" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2547.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460197" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2547.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460197" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/12/2547.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460197" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460197/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The advancement of next-generation sequencing technologies in conjunction with new bioinformatics tools enabled fine-tuning of sequence-based, high-resolution mapping strategies for complex genomes. Although genotyping-by-sequencing (GBS) provides a large number of markers, its application for association mapping and genomics-assisted breeding is limited by a large proportion of missing data per marker. For species with a reference genomic sequence, markers can be ordered on the physical map. However, in the absence of reference marker order, the use and imputation of GBS markers is challenging. Here, we demonstrate how the population sequencing (POPSEQ) approach can be used to provide marker context for GBS in wheat. The utility of a POPSEQ-based genetic map as a reference map to create genetically ordered markers on a chromosome for hexaploid wheat was validated by constructing an independent <em>de novo</em> linkage map of GBS markers from a Synthetic W7984 × Opata M85 recombinant inbred line (SynOpRIL) population. The results indicated that there is strong agreement between the independent <em>de novo</em> linkage map and the POPSEQ mapping approach in mapping and ordering GBS markers for hexaploid wheat. After ordering, a large number of GBS markers were imputed, thus providing a high-quality reference map that can be used for QTL mapping for different traits. The POPSEQ-based reference map and whole-genome sequence assemblies are valuable resources that can be used to order GBS markers and enable the application of highly accurate imputation methods to leverage the application GBS markers in wheat.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gbs" class="hw-term hw-article-keyword hw-article-keyword-gbs" rel="nofollow">GBS</a></li><li class="kwd"><a href="/keyword/wgs" class="hw-term hw-article-keyword hw-article-keyword-wgs" rel="nofollow">WGS</a></li><li class="kwd"><a href="/keyword/popseq" class="hw-term hw-article-keyword hw-article-keyword-popseq" rel="nofollow">POPSEQ</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li></ul><p id="p-3">Bread wheat (<em>Triticum aestivum</em> L.) is one of the world’s most important cereal crops, providing approximately 20% of all calories consumed by humans, and is a staple food crop for 30% of the human population. Increasing wheat production at the global scale is key to the efforts of filling the anticipated future food shortage gap attributable to global population increase and adverse effects of climate change on crop production. The application of advanced and precise molecular tools is needed to speed the development of new wheat varieties.</p><p id="p-4">The advent of next-generation sequencing technologies led to the emergence of high-throughput, sequence-based genotyping (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baird <em>et al.</em> 2008</a>) approaches such as genotyping-by-sequencing (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Poland and Rife 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mascher <em>et al.</em> 2013</a>). It also enabled the use of whole-genome shotgun sequencing (WGS) to generate genome assemblies of large and complex genomes. Although WGS assemblies of these large and complex genomes often contain many small, unordered contigs, assemblies can be made rapidly at low cost compared with approaches based on physical maps (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Mascher <em>et al.</em> 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mascher and Stein 2014</a>).</p><p id="p-5">The “gold-standard” method of genome sequencing is characterized by developing a physical map followed by sequencing of the minimum tiling path. Although the former needs coordinated efforts of several research laboratories and requires considerable investment in terms of time and money especially for large genome species like wheat, it is still considered as the indispensable method that leads toward a reference genome of a crop species without a reference genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Feuillet <em>et al.</em> 2011</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2012</a>). However, as a more tractable working assembly, the low-copy gene-space from the WGS approach is becoming an important genomic resource for gene discovery and functional genomics in species without a reference genome. This application has been proved on large genome crop species such as barley (<em>Hordeum vulgare</em>) and wheat by combining <em>de novo</em> WGS with classical genetic analysis (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Mascher <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hahn <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2015</a>).</p><p id="p-6">Population sequencing methodology, known as POPSEQ, was proposed as an integrated method to order and link contigs in WGS genome assemblies for gene isolation, genomics-assisted breeding, and genetic diversity assessment (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Mascher <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ariyadasa <em>et al.</em> 2014</a>). The method relies on the genetic segregation in biparental populations to create a linear order of contigs on an individual chromosome. The potential of POPSEQ to bring contigs into a linkage map was demonstrated on barley (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Mascher <em>et al.</em> 2013</a>) and then on hexaploid wheat (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2015</a>). A similar approach, which was referred to as recombinant population genome construction, was used to produce a high-quality genome assembly using a segregating population of <em>Caenorhabditis elegans</em> (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hahn <em>et al.</em> 2014</a>).</p><p id="p-7">Next-generation, sequencing-based genotyping approaches have been applied to understand the biological basis of agronomic traits in several plant species and for making selections via whole-genome prediction (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berkman <em>et al.</em> 2012</a>). Studies on major crop species such as <em>Zea mays</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernardo and Yu 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2013</a>), <em>T. aestivum</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Heffner <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012b</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> 2014</a>), <em>Oryza sativa</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Xu <em>et al.</em> 2014</a>), and <em>H. vulgare</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Iwata and Jannink 2011</a>) have indicated that genomic selection has the advantage of reducing the time needed to release new cultivars for production in plant breeding. Because genome-wide prediction needs a large number of evenly distributed markers per chromosome, genotyping-by-sequencing (GBS) markers are suitable for accurately predicting breeding values of candidate individuals in plant breeding programs. The GBS platform provides tens of thousands of markers with relatively low investment (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2012a</a>) and allows marker discovery and genotyping to be conducted simultaneously. GBS has been shown as an effective marker platform for whole-genome profiling and subsequent trait prediction (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012b</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2014</a>). Another promising area of application of GBS markers in plant breeding is for quantitative trait locus (QTL) mapping either by narrowing previously detected gene/QTL candidate regions or for the identification of novel gene/QTL regions that underlay economically important traits through saturating chromosomes with high-density GBS markers.</p><p id="p-8">The recent availability of a draft genome sequence and gene space assemblies for hexaploid and diploid wheat also makes the GBS platform an attractive approach for gene identification and precisely mapping QTL through anchoring trait-associated GBS single-nucleotide polymorphism (SNP) tags on draft genome sequence and/or gene space assemblies. From recent QTL studies with GBS markers, saturating chromosome regions with markers enabled the detection of previously detected and novel QTL for aluminum tolerance and leaf width in rice (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Spindel <em>et al.</em> 2013</a>), for drought tolerance in chickpea (Jaganathan <em>et al.</em> 2014), and for precise identification of the location of a dwarfing gene called <em>Breviaristatum-e</em> in barley (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2014</a>).</p><p id="p-9">A drawback of genotypic data from the GBS platform is often a large proportion of missing data points across samples as genomic DNA fragments are sequenced at low depth (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>). For GBS marker application in genomic selection, methods of imputing missing data points for unordered markers have been developed (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rutkoski <em>et al.</em> 2013</a>). However, many other imputation approaches that have been developed first require markers to be linearly ordered on a chromosome. Here we demonstrate the utility of the POPSEQ methodology for mapping, ordering and imputing marker data from GBS.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm and genotyping</h3><p id="p-10">We used recombinant inbred lines (RILs) from a cross between Synthetic W7984 and Opata M85 (“SynOpRIL”; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sorrells <em>et al.</em> 2011</a>) for this study. The total number of inbred lines used in this study was 183 lines, a subset of the larger population of ∼2000 lines. Genomic DNA was extracted from seedlings of each individual line grown in a greenhouse. The GBS libraries were constructed in 96-plex following the two-enzyme system GBS protocol with restriction enzymes <em>PstI</em> (CTGCAG) and <em>MspI</em> (CCGG) (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2012a</a>). Each library was sequenced on the Illumina HiSequation 2000 platform.</p></div><div id="sec-3" class="subsection"><h3>SNP calling and sequence data processing</h3><p id="p-11">Raw sequence data were processed with a custom java script in TASSEL 4, and a population-based SNP calling approach was used (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2012a</a>). Unique sequence tags of 64 bp were aligned internally allowing mismatches of up to 3 bp to identify SNPs within the tags. Fisher’s exact test was applied to determine independence of SNP alleles and then filter the SNPs. SNPs with up to 80% missing data points were retained for subsequent data analysis.</p></div><div id="sec-4" class="subsection"><h3>Linkage map construction and comparison with POPSEQ data</h3><p id="p-12">Construction of a linkage map for Synthetic W7984 × Opata M85 RIL population was performed with MSTMap software (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wu <em>et al.</em> 2008</a>) to group the markers into linkage groups. A total of 6362 polymorphic markers with up to 20% missing data, minor allele frequency of greater than 30%, and heterozygosity of less than 2% were considered for linkage map construction. Logarithm of odds score of 10 was used to cluster the markers into linkage groups. Linkage groups from the same chromosome were merged together and markers of the same chromosome were reordered with MSTmap.</p><p id="p-13">We used the synthetic wheat W7984 and Chinese Spring shotgun assemblies that were ordered with the POPSEQ approach using Synthetic W7984 × Opata M85 doubled haploid population (SynOpDH) to anchor SNP tags on genetically ordered WGS contigs (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">The International Wheat Genome Sequencing Consortium 2014</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2015</a>). To summarize in brief, the SynOpDH population, consisting of 90 individuals, was shotgun-sequenced, and SNPs were identified to construct a high-density linkage map that has all 21 wheat chromosomes. Then this highly dense genetic map was used to linearly order SNP-associated contigs in the WGS assembly of W7984 (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chapman <em>et al.</em> 2015</a>). The contigs of Chinese Spring assembly were also integrated into the same map of SynOpDH population (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">The International Wheat Genome Sequencing Consortium 2014</a>).</p><p id="p-14">Tags from which we detected SNPs were first aligned against both W7984 and Chinese Spring assemblies. To assemble the SNP tags, we used bwa software with “aln” method with default set up. The Samtools “view” method was used to further process sequence output. The SNP tags were filtered with a minimum alignment quality score of 37, and then markers that passed the quality requirements were merged with the high-density genetic map from SynOpDH population. Both W7984 and Chinese Spring assemblies and the high-density genetic map were linked by scaffold name in the former and by contig name in the latter, which facilitated navigation from assembly to genetic map or vice versa.</p><p id="p-15">Independently constructed <em>de novo</em> linkage maps were compared with the POPSEQ-based map for the number of markers correctly assigned to their respective linkage groups, and the linear relationship of marker order between <em>de novo</em> and POPSEQ maps. The step-by-step procedure of integrating GBS tags into the POPSEQ assembly is indicated in Supporting Information, <a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS1.tif">Figure S1</a>.</p></div><div id="sec-5" class="subsection"><h3>Missing data imputation</h3><p id="p-16">Genotypic data imputation was performed after anchoring SNP tags to the POPSEQ assembly. We aligned sequences of SNP tags for 33,664 SNPs that were obtained from the SynOpRIL population to W7984 and Chinese Spring assemblies to impute missing data points. For W7984 assembly, a total of 16,591 markers, and for Chinese Spring 9709 markers were imputed by use of the FSFHap (Full-Sib Haplotype imputation) method implemented in TASSEL 5 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Swarts <em>et al.</em> 2014</a>). The algorithm first detects two parental haplotypes and recombination break points via a Hidden Markov model and Viterbi algorithm. A missing data point is imputed to the matching genotype of flanking markers. If genotypes of markers flanking a missing data point do not match, the missing data point is left missing.</p><p id="p-17">Imputation accuracy of FSFHap method also was assessed by the use of squared correlation coefficient (R<sup>2</sup>). To calculate this parameter, 5% of the total genotypes were masked randomly for each marker in the dataset at genotypes where the SNP call was present. Squared correlation coefficient (R<sup>2</sup>) was calculated by comparing the original data where SNP calls were masked with corresponding imputed data for each marker. The R<sup>2</sup> calculation also was performed for the markers common between Chinese Spring and W7984 assemblies.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-18">Recombinant Inbred Lines used in this study are available on request from the Wheat Genetic Resource Center at KSU. Sequence data is available from NCBI SRA under SRP066497.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Validating POPSEQ with <em>de novo</em> linkage map</h3><p id="p-19">The use of an ultradense genetic linkage map constructed from the 90-individual doubled haploid population (SynOpDH) with the POPSEQ approach was validated through integrating SNP tags of GBS markers used to construct <em>de novo</em> linkage maps. We used an independent RIL mapping population made from the same synthetic W7984 and Opata M85 parents for this validation work. A total of 23 linkage groups with greater than two markers were obtained with a logarithm of odds score of 10. There was a total of 6360 markers in the linkage groups. Aligning sequences of SNP tags of these markers with sequences of linearly ordered gene space assembly of W7984 and Chinese Spring resulted in 3364 markers and 2049 markers that passed minimum quality alignment score of 37, respectively. A total of 3357 (99.8%) for W7984 and a total of 2037 markers (99.4%) for Chinese Spring were assigned correctly to their respective linkage group based on the sequence identity search in the assemblies. After removing incorrectly assigned markers and markers reported suspicious by the mapping algorithm in MSTmap, 21 linkage groups that correspond to the total number of hexaploid wheat chromosomes were obtained. Comparison of genetic map positions of the GBS markers in each of the 21 linkage groups based on the <em>de novo</em> map with that of positions of each marker based on the high density genetic map of POPSEQ also showed that there was a linear relationship between the two genetic maps (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker order relationship between de novo map and POPSEQ map for all 21 chromosomes of hexaploid wheat for markers anchored to W7984 assembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-579715151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker order relationship between &lt;em&gt;de novo&lt;/em&gt; map and POPSEQ map for all 21 chromosomes of hexaploid wheat for markers anchored to W7984 assembly.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2547/F1.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/12/2547/F1.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467205" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Marker order relationship between <em>de novo</em> map and POPSEQ map for all 21 chromosomes of hexaploid wheat for markers anchored to W7984 assembly.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker order relationship between de novo map and POPSEQ map for all 21 chromosomes of hexaploid wheat for markers anchored to Chinese Spring assembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-579715151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker order relationship between &lt;em&gt;de novo&lt;/em&gt; map and POPSEQ map for all 21 chromosomes of hexaploid wheat for markers anchored to Chinese Spring assembly.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2547/F2.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/12/2547/F2.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467288" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Marker order relationship between <em>de novo</em> map and POPSEQ map for all 21 chromosomes of hexaploid wheat for markers anchored to Chinese Spring assembly.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Originally, a total of 33,664 GBS markers with up to 80% missing data were obtained for the synthetic W7984 × Opata M85 RIL population. All marker tags, without consideration of the level of missing data, were integrated into POPSEQ data. After discarding markers with low alignment quality score (&lt;37), we found map positions for 16,591 markers (49.3%) for W7984 assembly and 9709 (28.8%) for Chinese Spring assembly. The number of markers per chromosome was greater for W7984 assembly for all chromosomes than that of Chinese Spring (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/TableS1.xlsx">Table S1</a>). Similarly, the average gap size per chromosome was low for all chromosomes in the case of W7984 assembly (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Maximum gap size for markers anchored to W7984 assembly was lower than 20 cM for all chromosomes. All chromosomes had less than 30 cM maximum gap size for markers anchored to the Chinese Spring assembly (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/TableS2.xlsx">Table S2</a>). The two assemblies had a total of 7040 markers in common, and there was a strong positive relationship between marker and map positions between the two maps (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS2.tif">Figure S2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide distribution of genotype-by-sequencing markers anchored to both W7984 and Chinese Spring (CS) assemblies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-579715151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide distribution of genotype-by-sequencing markers anchored to both W7984 and Chinese Spring (CS) assemblies.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2547/F3.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/12/2547/F3.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467266" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Genome-wide distribution of genotype-by-sequencing markers anchored to both W7984 and Chinese Spring (CS) assemblies.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average gap size (cM) for the markers anchored to both W7984 and Chinese Spring (CS) assemblies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-579715151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average gap size (cM) for the markers anchored to both W7984 and Chinese Spring (CS) assemblies.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2547/F4.medium.gif" width="440" height="371"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/12/2547/F4.medium.gif" width="440" height="371"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2547/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467276" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Average gap size (cM) for the markers anchored to both W7984 and Chinese Spring (CS) assemblies.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Data imputation</h3><p id="p-25">Two datasets comprised a total of 16,591 markers (for W7984 assembly) and 9709 markers (for Chinese Spring) were submitted to FSFHap for imputation. After imputation, 95% of the markers for W7984 assembly and 94% of the markers for Chinese Spring had less than 10% missing data points (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/TableS3.xlsx">Table S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/TableS4.xlsx">Table S4</a>). Furthermore, 92.4% and 89.8% of the markers had less than 5% of missing data for W7984 assembly and Chinese Spring assembly, respectively. Only 2.7% of the markers for W7984 assembly and 3.2% of the markers for Chinese Spring assembly had more than 20% remaining missing data points per marker. The greatest average missing data points per chromosome in the imputed data set was recorded for chromosome 2D (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS3.tif">Figure S3</a>), and there was no clear relationship between the amount of missing data per chromosome in the original and imputed data sets (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS4.tif">Figure S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS5.tif">Figure S5</a>). The number of un-imputed missing data points was greater for Chinese Spring than that of W7984 assembly in 15 chromosomes. The average proportion of heterozygous genotypes per marker was comparable between the two assemblies except for chromosomes 2A, 3D, and 4A, where differences between the two assemblies were large (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS6.tif">Figure S6</a>). The exceptionally high average proportion of heterozygous genotypes for chromosome 2A for Chinese Spring was due to high heterozygous genotypes per marker (28–37%) for markers within the interval 58.7−67.4 cM (around the centromeric region) (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/TableS4.xlsx">Table S4</a>). Markers on chromosome 4D had high heterozygote genotypes for both assemblies. However, except for these two chromosomes (2A and 4D), the average proportion of heterozygote genotypes per marker was less than 10% for all chromosomes (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS6.tif">Figure S6</a>). There was no relationship between the amount of heterozygote genotypes before imputation and the amount remaining after imputation for both assemblies (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS7.tif">Figure S7</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS8.tif">Figure S8</a>). The average proportion of heterozygote genotypes per markers in imputed data set was greater than that of before imputation for both assemblies for all chromosomes. The algorithm implemented in FSFHap is taking into account heterozygote undercalling for GBS platform SNP calling. However, taking into account the low proportion of heterozygote genotypes in the original data and the expected residual heterozygosity for RIL populations, FSFHap may be overestimating the amount of heterozygote genotypes per marker in the imputed data.</p><p id="p-26">Imputation accuracy of FSFHap was evaluated by the use of squared correlation coefficient (R<sup>2</sup>). The average R<sup>2</sup> for the markers anchored to Chinese Spring was 0.94, and 90% of the markers had R<sup>2</sup> greater than or equal to 0.8 (<a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/FigureS9.tif">Figure S9</a>). Similar average R<sup>2</sup> of 0.94 was obtained for the markers common between Chinese Spring and W7984 assemblies.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-27">We validated the utility of POPSEQ, a reference map-based marker ordering method, by using a new RIL population. We first constructed a <em>de novo</em> genetic linkage map by using filtered, high-quality GBS markers from the Synthetic W7984 × Opata M85 RIL population. For the POPSEQ map to be used as a reference map to order the GBS markers from different populations, first GBS markers within a <em>de novo</em> linkage group should be assigned to the same chromosome with the chromosome assignment of the markers based on aligning SNP tags sequences with the POPSEQ genome assembly. Second, marker order of the <em>de novo</em> linkage map should also agree with the order of the same markers on the POPSEQ-based reference genetic map. When a high alignment stringency level was used, both chromosome assignment and marker order indicated that there is a good agreement between the <em>de novo</em> linkage map and POPSEQ-based high-density genetic map. This supports that the POPSEQ approach can be used as an alternative method to assign and order GBS markers on wheat chromosomes.</p><p id="p-28">The advantage of using the POPSEQ approach for mapping and ordering GBS markers is twofold. In classical linkage mapping approach, chromosome assignment information is required for all constructed linkage groups. With the absence of this information, anchor markers are needed to guide linkage map construction and for downstream interpretation of QTL analysis. In the case of GBS markers, for species without a reference genome sequence, this information is presumably lacking and consequently construction of a <em>de novo</em> linkage map is a tedious job. Moreover, genetic map construction is limited by the large number of markers with a high proportion missing data. However, the POPSEQ anchored reference can give <em>a priori</em> marker positions as with any reference genome and avoids the need of both anchor markers and <em>de novo</em> map construction.</p><p id="p-29">By using the POPSEQ approach, we were able to genetically map and order 16,591 and 9711 GBS markers anchored to W7984 and Chinese Spring assemblies, respectively. The greater number of markers for W7984 assembly compared with Chinese Spring assembly may be attributable to W7984 being one of the parents of the biparental population from which SNP tags were detected. Chromosome 4D had the least number of markers whereas chromosome 3B the greatest number of markers for both assemblies (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/09/01/g3.115.020362.DC1/TableS1.xlsx">Table S1</a>). This agrees with the observation that the larger the chromosome size, the more number of markers it contains (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2012a</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Saintenac <em>et al.</em> 2013</a>). Similarly, as expected the total number of markers mapped to the D genome (25% for both assemblies) was lower than that of A (31% for W7984 assembly and 28% for Chinese Spring assemblies) and B (44% for W7984 assembly and 47% for Chinese Spring assembly) genomes as the D genome of hexaploid wheat is smaller and less diverse than A and B genomes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chao <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edae <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Iehisa <em>et al.</em> 2014</a>). However, the difference between the number of markers mapped to D and A genomes was lower than reported in the literature, which was about fivefold greater for both A and B genomes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2011</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cavanagh <em>et al.</em> 2013</a>). One of the probable explanations for this small difference between A and D genomes is attributable to reintroduction of D genome diversity through synthetic W7984 to the current hexaploid wheat mapping population.</p><p id="p-30">The maximum gap size per chromosome for markers anchored to W7984 assembly (&lt;20 cM) was lower than that of markers anchored to Chinese Spring assembly (&lt;30 cM). The original reference map from POPSEQ had good marker coverage (maximum gap size &lt;20 cM) for all chromosomes with the exception of 1A and 4D with maximum gap size of 26 cM and 28 cM, respectively (data not shown). Overall, the pattern of marker distribution on chromosomes observed for our population also has good agreement with that of reference map, indicating the potential of POPSEQ approach of marker ordering and imputing. However, our final high-quality SNP dataset represented only approximately 30 and 49% of the total original 33,664 SNP tags for Chinese Spring and W7984 assemblies, respectively. A majority of the markers did not pass the strict sequence alignment threshold we used to reduce the risk of assigning the tags to incorrect positions. Therefore, for full utilization of the POPSEQ-based gene space assemblies, consensus reference genetic maps of two or more populations are needed.</p><p id="p-31">Imputing missing data are a necessary step, particularly for the genotypic data sets with a large proportion of missing data per marker (up to 80% in our case). The high-density genetic map enabled us to use FSFHap as implemented in TASSEL 5 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Swarts <em>et al.</em> 2014</a>). FSFHap generally needs a large data set for best results. Imputation accuracy measurements (R<sup>2</sup>) found here indicated that FSFHap is still accurate in imputing a relatively small dataset with a large proportion of missing data. We found average imputation accuracy (R<sup>2</sup> = 0.94) roughly similar with that of average imputation accuracy (R<sup>2</sup> = 0.97) obtained for a large genotypic data set of maize nested association mapping panel (NAM RILs) used by <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Swarts <em>et al.</em> (2014)</a>. We observed an increase in the number of heterozygous genotypes called in the imputed data. Low-coverage sequencing with GBS is expected to undercall true heterozygous genotypes. Therefore, an increase in heterozygous calls is expected to some extent with the GBS imputation. Apart from the effect of algorithms implemented in FSFHap, the greater levels of heterozygosity for some markers near centromeric regions (<em>e.g.</em>, 2A) in the imputed dataset may be the result of deleterious alleles <em>in trans</em>-linkage combined with low levels of recombination in these regions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McMullen <em>et al.</em> 2009</a>).</p><p id="p-32">In conclusion, POPSEQ methodology can be used as an alternative method of mapping and ordering GBS markers. For crop species lacking a high-quality reference genome, like hexaploid wheat, genetically ordered markers allow the implementation of marker-order−dependent but highly accurate imputation algorithms (<em>e.g.</em>, FSFHap) to impute genotypic data with a large proportion of missing data points so that QTL/gene mapping can be done precisely. Because both the Chinese Spring and W7984 POPSEQ results were based on the same SynOpDH genetic mapping population, full integration of markers to the reference map needs POPSEQ-based, high-density genetic map and gene space assemblies that are developed from additional biparental populations.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">This project is funded by the United States Department of Agriculture (USDA) - Agricultural Research Service (Appropriation #3020-21000-010-00D) and National Research Initiative Competitive Grants CAP project 2011-68002-30029 from the USDA National Institute of Food and Agriculture, the US Agency for International Development (USAID Cooperative Agreement No. AID-OAA-A-13-0005), and the National Science Foundation (NSF) Plant Genome Research Program (IOS-1339389). This work was completed under the auspices of the Wheat Genetics Resource Center (WGRC) Industry/University Collaborative Research Center (I/UCRC) supported by NSF grant contract (IIP-1338897) and industry partners. Computational resources for this project were through Beocat HPC at Kansas State University, which is funded in part by NSF grants CNS-1006860, EPS-1006860, and EPS-0919443. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the USDA. The USDA is an equal opportunity provider and employer. This work represents contribution number 15-459-J from the Kansas Agricultural Experiment Station.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-34"><em>Communicating editor: J. D. Faris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-13"><span class="received-label">Received </span>July 13, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-15"><span class="accepted-label">Accepted </span>September 15, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Edae <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.1" data-doi="10.1111/j.1467-7652.2011.00628.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coghill</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gwilliam</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DAllen%26rft.auinit1%253DA.%2BM.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D1086%26rft.epage%253D1099%26rft.atitle%253DTranscript-specific%252C%2Bsingle-nucleotide%2Bpolymorphism%2Bdiscovery%2Band%2Blinkage%2Banalysis%2Bin%2Bhexaploid%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00628.x%26rft_id%253Dinfo%253Apmid%252F21627760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00628.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21627760&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297941700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.2" data-doi="10.1111/pbi.12009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burridge</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winfield</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Discovery and development of exome-based, codominant single nucleotide polymorphism markers in hexaploid wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12009%26rft_id%253Dinfo%253Apmid%252F23279710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23279710&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316825400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.3" data-doi="10.1104/pp.113.228213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ariyadasa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nussbaumer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frenke</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A sequence-ready physical map of barley anchored genetically by two million single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">412</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.228213%26rft_id%253Dinfo%253Apmid%252F24243933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMS80MTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI1NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.4" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping sing sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.5" data-doi="10.3732/ajb.1100309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berkman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Next-generation sequencing application for wheat crop improvement.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1100309%26rft_id%253Dinfo%253Apmid%252F22268223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMzY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.6" data-doi="10.2135/cropsci2008.08.0452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomewide selection for rapid introgression of exotic germplasm in maize.</span> <span class="cit-source">Crop Sci.</span> <span class="cit-vol">49</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D419%26rft.epage%253D425%26rft.atitle%253DGenomewide%2BSelection%2Bfor%2BRapid%2BIntrogression%2Bof%2BExotic%2BGermplasm%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264654000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.7" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.8" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI1NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.9" data-doi="10.1186/1471-2164-11-727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baenziger</span>  <span class="cit-name-given-names">S.P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population-and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (Triticum aestivum L.).</span> <span class="cit-source">BMC Genomics</span> <span class="cit-vol">11</span>: <span class="cit-fpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DChao%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D727%26rft.epage%253D727%26rft.atitle%253DPopulation-%2Band%2Bgenome-specific%2Bpatterns%2Bof%2Blinkage%2Bdisequilibrium%2Band%2BSNP%2Bvariation%2Bin%2Bspring%2Band%2Bwinter%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-727%26rft_id%253Dinfo%253Apmid%252F21190581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21190581&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.10" data-doi="10.1186/s13059-015-0582-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buluç</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georganas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0582-8%26rft_id%253Dinfo%253Apmid%252F25637298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0582-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25637298&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding population with genotyping-by-sequencing.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">13</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D13%26rft.spage%253D1903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.12" data-doi="10.1007/s00122-014-2341-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bariana</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction for rust resistance in diverse wheat landraces.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1795%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2341-8%26rft_id%253Dinfo%253Apmid%252F24965887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2341-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24965887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.13" data-doi="10.1007/s00122-013-2257-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edae</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association mapping of yield and yield components of spring wheat under contrasting moisture regimes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D791%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2257-8%26rft_id%253Dinfo%253Apmid%252F24408378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2257-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24408378&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotype-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.15" data-doi="10.1016/j.tplants.2010.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuillet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leach</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eversole</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Crop genome sequencing: lessons and rationales.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFeuillet%26rft.auinit1%253DC.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D77%26rft.epage%253D88%26rft.atitle%253DCrop%2Bgenome%2Bsequencing%253A%2Blessons%2Band%2Brationales.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.10.005%26rft_id%253Dinfo%253Apmid%252F21081278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21081278&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288342200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.16" data-doi="10.1007/s10142-012-0300-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuillet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Praud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrating cereal genomics to support innovation in the Triticeae.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DFeuillet%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D573%26rft.epage%253D583%26rft.atitle%253DIntegrating%2Bcereal%2Bgenomics%2Bto%2Bsupport%2Binnovation%2Bin%2Bthe%2BTriticeae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-012-0300-5%26rft_id%253Dinfo%253Apmid%252F23161406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-012-0300-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23161406&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311781700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.17" data-doi="10.1534/g3.114.010264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing, assembling, and correcting draft genomes assembly using recombinant populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010264%26rft_id%253Dinfo%253Apmid%252F24531727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNC82NjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI1NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.18" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">JL.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: Gain per unit time and cost.</span> <span class="cit-source">Crop Sci.</span> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.19" data-doi="10.1007/s00122-013-2215-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iehisa</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishijima</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide marker development for the wheat D genome based on single nucleotide polymorphisms identified from transcripts in the wild wheat progenitor <em>Aegilops tauschii</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D261%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2215-5%26rft_id%253Dinfo%253Apmid%252F24158251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2215-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24158251&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International Wheat Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D345%26rft.spage%253D286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.21" data-doi="10.2135/cropsci2010.12.0732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genomic selection prediction in barley breeding programs: a simulation study based on the real single nucleotide polymorphism data of barley breeding lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D1915%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.12.0732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.12.0732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000293473800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.12.2547.22" data-doi="10.1007/s00438-014-0932-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaganathan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kale</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotyping-by-sequencing based intra-specific genetic map refines a “QTL-hotspot” region for drought tolerance in chickpea.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">559</span>−<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D290%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-014-0932-3%26rft_id%253Dinfo%253Apmid%252F25344290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-014-0932-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25344290&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.23" data-doi="10.1186/1471-2164-15-104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An evaluation of genotyping by sequencing (GBS) to map the <em>Breviaristatum-e</em> (<em>ari-e</em>) locus in cultivated barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-104%26rft_id%253Dinfo%253Apmid%252F24498911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498911&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic anchoring of whole genome shotgun assemblies.</span> <span class="cit-source">Front. Genet.</span> <span class="cit-vol">5</span>: <span class="cit-fpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D208%26rft_id%253Dinfo%253Apmid%252F25071835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25071835&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.25" data-doi="10.1111/tpj.12319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokhsar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D76%26rft.spage%253D718%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12319%26rft_id%253Dinfo%253Apmid%252F23998490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23998490&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326138100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.26" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI1NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.27" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312663100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.28" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.29" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <span class="cit-source">Plant Genome</span> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>−<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2547.30"><div class="cit-metadata unstructured">Rutkoski, J., J. Poland, JL. Jannink, and M. Sorrells. 2013 Imputation of unordered markers and the impact on genomic selection accuracy. G3 (Bethesda) (3):427–429.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.31" data-doi="10.1534/g3.113.005819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequencing-based mapping of the polyploid wheat genome.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005819%26rft_id%253Dinfo%253Apmid%252F23665877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvNy8xMTA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.32" data-doi="10.1139/g11-054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustafson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benscher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reconstruction of the Synthetic W7984 × Opata M85 wheat reference population.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSorrells%26rft.auinit1%253DM.%2BE.%26rft.volume%253D54%26rft.issue%253D11%26rft.spage%253D875%26rft.epage%253D882%26rft.atitle%253DReconstruction%2Bof%2Bthe%2Bsynthetic%2BW7984%2Bx%2BOpata%2BM85%2Bwheat%2Breference%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg11-054%26rft_id%253Dinfo%253Apmid%252F21999208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g11-054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21999208&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.33" data-doi="10.1007/s00122-013-2166-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2699</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2699%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2166-x%26rft_id%253Dinfo%253Apmid%252F23918062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2166-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918062&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/articles/7/3/plantgenome2014.05.0023">https://dl.sciencesocieties.org/publications/tpg/articles/7/3/plantgenome2014.05.0023</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-5.12.2547.35" data-doi="10.1534/g3.114.012468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Resolution of genetic map expansion caused by excess heterozygosity in plant recombinant inbred populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1969.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012468%26rft_id%253Dinfo%253Apmid%252F25128435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTAvMTk2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjU0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.36" data-doi="10.1111/nph.12164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Aegilops tauschii</em> single nucleotide polymorphisms shed light on the origins of wheat D-genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D198%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12164%26rft_id%253Dinfo%253Apmid%252F23374069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.37" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.38" data-doi="10.1073/pnas.1413750111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Predicting hybrid performance in rice using genomic best linear unbiased prediction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">12456</span>–<span class="cit-lpage">12461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413750111%26rft_id%253Dinfo%253Apmid%252F25114224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM0LzEyNDU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2547.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D114%26rft.spage%253D291%26rft_id%253Dinfo%253Apmid%252F25407079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25407079&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/12/2547.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2539" title="Genome Sequences of Three Phytopathogenic Species of the Magnaporthaceae Family of Fungi" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2555" title="Markerless Escherichia coli rrn Deletion Strains for Genetic Determination of Ribosomal Binding Sites" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/12"><span>Volume 5 Issue 12, December 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/12.cover-source.jpg?itok=RngHqKQq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2547" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ydZDq_MF8LBmMmhkDRP2JXxD81xw_DKTzJkx8BeiYII" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/12/2547" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/12/2547" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/12/2547" class="active" data-icon-position="" data-hide-link-title="0">Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460197" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-feCrF7EysTfSEZikWSuOroKUyJoyiAKJfkqq4j1zkgA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2547.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/12/2547" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-G7fmp8arY0xPqvHTJqG5-UbMm8wgRQMPrH9Gkrb4WC4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/12/2547" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2547.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460197" id="node460197--2" data-pisa="ggg;5/12/2547" data-pisa-master="ggg;g3.115.020362" data-apath="/ggg/5/12/2547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erena A. Edae</span>, <span class="highwire-citation-author" data-delta="1">Robert L. Bowden</span> and <span class="highwire-citation-author" data-delta="2">Jesse Poland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2547-2553;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020362 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460197--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erena A. Edae </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erena%2BA.%2BEdae%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edae%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErena%2BA.%2BEdae%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Erena.Edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">Erena.Edae@ars.usda.gov</a>
<a href="mailto:jpoland@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">jpoland@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert L. Bowden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BBowden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowden%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BL.%2BBowden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Erena.Edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">Erena.Edae@ars.usda.gov</a>
<a href="mailto:jpoland@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">jpoland@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/12/2547" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/12/2547&amp;title=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/12/2547&amp;title=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/12/2547&amp;title=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/12/2547&amp;text=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/12/2547&amp;title=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/12/2547&amp;t=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/12/2547&amp;title=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/12/2547&amp;title=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460197/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460197" id="node460197--3" data-pisa="ggg;5/12/2547" data-pisa-master="ggg;g3.115.020362" data-apath="/ggg/5/12/2547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Application of Population Sequencing (POPSEQ) for Ordering and Imputing Genotyping-by-Sequencing Markers in Hexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erena A. Edae</span>, <span class="highwire-citation-author" data-delta="1">Robert L. Bowden</span> and <span class="highwire-citation-author" data-delta="2">Jesse Poland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2547-2553;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020362 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460197--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erena A. Edae </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erena%2BA.%2BEdae%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edae%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErena%2BA.%2BEdae%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Erena.Edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">Erena.Edae@ars.usda.gov</a>
<a href="mailto:jpoland@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">jpoland@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert L. Bowden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA-ARS, Hard Winter Wheat Genetics Research Unit, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BBowden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowden%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BL.%2BBowden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Erena.Edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">Erena.Edae@ars.usda.gov</a>
<a href="mailto:jpoland@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">jpoland@ksu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2547&amp;rft.epage=2553&amp;rft.atitle=Application%20of%20Population%20Sequencing%20%28POPSEQ%29%20for%20Ordering%20and%20Imputing%20Genotyping-by-Sequencing%20Markers%20in%20Hexaploid%20Wheat&amp;rft.volume=5&amp;rft.issue=12&amp;rft.date=2015-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Edae%2C+Erena+A.&amp;rft.au=Bowden%2C+Robert+L.&amp;rft.au=Poland%2C+Jesse"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460197/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460197/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460197/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460197/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460197/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460197/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460197/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460197/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460197/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460197/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460197/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460197/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/12/2547" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/12/2547.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/12/2547.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/12/2547.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/12/2547.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
