<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/12/2655" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/12/2655.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/12/2655.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/12/2655.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/12/2655.atom" />
<meta name="HW.pisa" content="ggg;5/12/2655" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Architecture of Disease Resistance Genes in Lettuce" />
<meta name="DC.Identifier" content="10.1534/g3.115.020818" />
<meta name="DC.Date" content="2015-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Christopoulou et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.116.027144" />
<meta name="DC.Description" content="Genome-wide motif searches identified 1134 genes in the lettuce reference genome of cv. Salinas that are potentially involved in pathogen recognition, of which 385 were predicted to encode nucleotide binding-leucine rich repeat receptor (NLR) proteins. Using a maximum-likelihood approach, we grouped the NLRs into 25 multigene families and 17 singletons. Forty-one percent of these NLR-encoding genes belong to three families, the largest being RGC16 with 62 genes in cv. Salinas. The majority of NLR-encoding genes are located in five major resistance clusters (MRCs) on chromosomes 1, 2, 3, 4, and 8 and cosegregate with multiple disease resistance phenotypes. Most MRCs contain primarily members of a single NLR gene family but a few are more complex. MRC2 spans 73 Mb and contains 61 NLRs of six different gene families that cosegregate with nine disease resistance phenotypes. MRC3, which is 25 Mb, contains 22 RGC21 genes and colocates with Dm13 . A library of 33 transgenic RNA interference tester stocks was generated for functional analysis of NLR-encoding genes that cosegregated with disease resistance phenotypes in each of the MRCs. Members of four NLR-encoding families, RGC1 , RGC2 , RGC21 , and RGC12 were shown to be required for 16 disease resistance phenotypes in lettuce. The general composition of MRCs is conserved across different genotypes; however, the specific repertoire of NLR-encoding genes varied particularly of the rapidly evolving Type I genes. These tester stocks are valuable resources for future analyses of additional resistance phenotypes." />
<meta name="DC.Contributor" content="Marilena Christopoulou" />
<meta name="DC.Contributor" content="Sebastian Reyes-Chin Wo" />
<meta name="DC.Contributor" content="Alex Kozik" />
<meta name="DC.Contributor" content="Leah K. McHale" />
<meta name="DC.Contributor" content="Maria-Jose Truco" />
<meta name="DC.Contributor" content="Tadeusz Wroblewski" />
<meta name="DC.Contributor" content="Richard W. Michelmore" />
<meta name="article:published_time" content="2015-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Architecture of Disease Resistance Genes in Lettuce" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide motif searches identified 1134 genes in the lettuce reference genome of cv. Salinas that are potentially involved in pathogen recognition, of which 385 were predicted to encode nucleotide binding-leucine rich repeat receptor (NLR) proteins. Using a maximum-likelihood approach, we grouped the NLRs into 25 multigene families and 17 singletons. Forty-one percent of these NLR-encoding genes belong to three families, the largest being &lt;i&gt;RGC16&lt;/i&gt; with 62 genes in cv. Salinas. The majority of NLR-encoding genes are located in five major resistance clusters (MRCs) on chromosomes 1, 2, 3, 4, and 8 and cosegregate with multiple disease resistance phenotypes. Most MRCs contain primarily members of a single NLR gene family but a few are more complex. MRC2 spans 73 Mb and contains 61 NLRs of six different gene families that cosegregate with nine disease resistance phenotypes. MRC3, which is 25 Mb, contains 22 &lt;i&gt;RGC21&lt;/i&gt; genes and colocates with &lt;i&gt;Dm13&lt;/i&gt;. A library of 33 transgenic RNA interference tester stocks was generated for functional analysis of NLR-encoding genes that cosegregated with disease resistance phenotypes in each of the MRCs. Members of four NLR-encoding families, &lt;i&gt;RGC1&lt;/i&gt;, &lt;i&gt;RGC2&lt;/i&gt;, &lt;i&gt;RGC21&lt;/i&gt;, and &lt;i&gt;RGC12&lt;/i&gt; were shown to be required for 16 disease resistance phenotypes in lettuce. The general composition of MRCs is conserved across different genotypes; however, the specific repertoire of NLR-encoding genes varied particularly of the rapidly evolving Type I genes. These tester stocks are valuable resources for future analyses of additional resistance phenotypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/12/01" />
<meta name="citation_mjid" content="ggg;5/12/2655" />
<meta name="citation_id" content="5/12/2655" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/12/2655" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/12/2655.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/12/2655.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/12/2655.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020818" />
<meta name="citation_pmid" content="26449254" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2655" />
<meta name="citation_lastpage" content="2669" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marilena Christopoulou" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Sebastian Reyes-Chin Wo" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Alex Kozik" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Leah K. McHale" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Maria-Jose Truco" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Tadeusz Wroblewski" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Richard W. Michelmore" />
<meta name="citation_author_institution" content="Genome Center and Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="rwmichelmore@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. Arya;citation_author=G. Kumar;citation_author=V. Acharya;citation_author=A. K. Singh;citation_title=Genome-wide identification and expression analysis of NBS-encoding genes in Malus domestica and expansion of NBS genes family in Rosaceae.;citation_pages=e107987;citation_volume=9;citation_year=2014;citation_pmid=25232838;citation_doi=10.1371/journal.pone.0107987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Bernatzky;citation_author=S. D. Tanksley;citation_title=TOWARD A SATURATED LINKAGE MAP IN TOMATO BASED ON ISOZYMES AND RANDOM cDNA SEQUENCES;citation_pages=887-898;citation_volume=112;citation_year=1986;citation_issue=4;citation_pmid=17246322" />
<meta name="citation_reference" content="citation_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Bernsel;citation_author=H. Viklund;citation_author=J. Falk;citation_author=E. Lindahl;citation_author=G. von Heijne;citation_title=Prediction of membrane-protein topology from first principles.;citation_volume=105;citation_year=2008;citation_pmid=18477697;citation_doi=10.1073/pnas.0711151105" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=T. O. Bozkurt;citation_author=S. Schornack;citation_author=M. J. Banfield;citation_author=S. Kamoun;citation_title=Oomycetes, effectors, and all that jazz.;citation_pages=483-492;citation_volume=15;citation_year=2012;citation_issue=4;citation_pmid=22483402;citation_doi=10.1016/j.pbi.2012.03.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Chin;citation_author=R. Arroyo-Garcia;citation_author=O. E. Ochoa;citation_author=R. V. Kesseli;citation_author=D. O. Lavelle;citation_title=Recombination and Spontaneous Mutation at the Major Cluster of Resistance Genes in Lettuce (Lactuca sativa);citation_pages=831-849;citation_volume=157;citation_year=2001;citation_issue=2;citation_pmid=11157000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Microbe Interact.;citation_author=M. Christopoulou;citation_author=L. McHale;citation_author=A. Kozik;citation_author=S. Reyes-Chin Wo;citation_author=T. Wroblewski;citation_title=Dissection of two complex clusters of resistance genes in Lettuce (Lactuca sativa).;citation_pages=751-765;citation_volume=28;citation_year=2015;citation_pmid=25650829;citation_doi=10.1094/MPMI-06-14-0175-R" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. M. Clark;citation_author=G. Schweikert;citation_author=C. Toomajian;citation_author=S. Ossowski;citation_author=G. Zeller;citation_title=Common Sequence Polymorphisms Shaping Genetic Diversity in Arabidopsis thaliana;citation_pages=338-342;citation_volume=317;citation_year=2007;citation_issue=5836;citation_pmid=17641193;citation_doi=10.1126/science.1138632" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Biol.;citation_author=I. Crute;citation_author=A. Johnson;citation_title=The genetic relationship between races of Bremia lactucae and cultivars of Lactuca sativa.;citation_pages=125-137;citation_volume=83;citation_year=1976;citation_doi=10.1111/j.1744-7348.1976.tb01701.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=C. Dardick;citation_author=P. Ronald;citation_title=Plant and animal pathogen recognition receptors signal through non-RD kinases.;citation_pages=14-28;citation_volume=2;citation_year=2006;citation_doi=10.1371/journal.ppat.0020014" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Dardick;citation_author=B. Schwessinger;citation_author=P. Ronald;citation_title=Non-arginine-aspartate (non-RD) kinases are associated with innate immune receptors that recognize conserved microbial signatures.;citation_pages=358-366;citation_volume=15;citation_year=2012;citation_issue=4;citation_pmid=22658367;citation_doi=10.1016/j.pbi.2012.05.002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Darriba;citation_author=G. L. Taboada;citation_author=R. Doallo;citation_author=D. Posada;citation_title=ProtTest 3: fast selection of best-fit models of protein evolution.;citation_volume=27;citation_year=2011;citation_pmid=21335321;citation_doi=10.1093/bioinformatics/btr088" />
<meta name="citation_reference" content="citation_title=Compendium of Lettuce Diseases.;citation_author=R. M. Davis;citation_author=K. V. Subbarao;citation_author=R. N. Raid;citation_author=E. A. Kurtz;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=W. J. Dower;citation_author=J. F. Miller;citation_author=C. W. Ragsdale;citation_title=High efficiency transformation of E.coli by high voltage electroporation;citation_pages=6127-6145;citation_volume=16;citation_year=1988;citation_issue=13;citation_pmid=3041370;citation_doi=10.1093/nar/16.13.6127" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=J. G. Ellis;citation_author=P. N. Dodds;citation_author=G. J. Lawrence;citation_title=Flax rust resistance gene specificity is based on direct resistance-avirulence protein interactions.;citation_pages=289-306;citation_volume=45;citation_year=2007;citation_pmid=17430087;citation_doi=10.1146/annurev.phyto.45.062806.094331" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=P. Ellis;citation_author=S. McClement;citation_author=P. Saw;citation_author=K. Phelps;citation_author=W. Vice;citation_title=Identification of sources of resistance in lettuce to the lettuce root aphid, Pemphigus bursarius.;citation_pages=305-315;citation_volume=125;citation_year=2002;citation_doi=10.1023/A:1016050706560" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=B. F. Farrara;citation_author=T. W. Ilott;citation_author=R. W. Michelmore;citation_title=Genetic analysis of factors for resistance to downy mildew (Bremia lactucae) in species of lettuce (Lactuca sativa and L. serriola).;citation_pages=499-514;citation_volume=36;citation_year=1987;citation_doi=10.1111/j.1365-3059.1987.tb02267.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. L. Guo;citation_author=J. Fitz;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=J. Cao;citation_title=Genome-wide comparison of nucleotide-binding site-leucine-rich repeat-encoding genes in Arabidopsis.;citation_volume=157;citation_year=2011;citation_pmid=21810963;citation_doi=10.1104/pp.111.181990" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. J. Hayes;citation_author=L. K. McHale;citation_author=G. E. Vallad;citation_author=M. J. Truco;citation_author=R. W. Michelmore;citation_title=The inheritance of resistance to Verticillium wilt caused by race 1 isolates of Verticillium dahliae in the lettuce cultivar La Brillante.;citation_pages=509-517;citation_volume=123;citation_year=2011;citation_issue=4;citation_pmid=21567237;citation_doi=10.1007/s00122-011-1603-y" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Hoekema;citation_author=P. R. Hirsch;citation_author=P. J. J. Hooykaas;citation_author=R. A. Schilperoort;citation_title=A binary plant vector strategy based on separation of vir-region and T-region of the Agrobacterium-tumefaciens Ti-plasmid;citation_pages=179-180;citation_volume=303;citation_year=1983;citation_doi=10.1038/303179a0" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=S. H. Hulbert;citation_author=C. A. Webb;citation_author=S. M. Smith;citation_author=Q. Sun;citation_title=RESISTANCE GENE COMPLEXES: Evolution and Utilization;citation_pages=285-312;citation_volume=39;citation_year=2001;citation_issue=1;citation_pmid=11701867;citation_doi=10.1146/annurev.phyto.39.1.285" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. H. Huson;citation_author=D. C. Richter;citation_author=C. Rausch;citation_author=T. Dezulian;citation_author=M. Franz;citation_title=Dendroscope: An interactive viewer for large phylogenetic trees.;citation_pages=460-460;citation_volume=8;citation_year=2007;citation_pmid=18034891;citation_doi=10.1186/1471-2105-8-460" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=T. Ilott;citation_author=S. Hulbert;citation_author=R. Michelmore;citation_title=Genetic analysis of the gene-for-gene interaction between lettuce (Lactuca sativa) and Bremia lactucae.;citation_pages=888-897;citation_volume=79;citation_year=1989;citation_doi=10.1094/Phyto-79-888" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. D. Jones;citation_author=J. L. Dangl;citation_title=The plant immune system.;citation_pages=323-329;citation_volume=444;citation_year=2006;citation_issue=7117;citation_pmid=17108957;citation_doi=10.1038/nature05286" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=T. L. Karasov;citation_author=M. W. Horton;citation_author=J. Bergelson;citation_title=Genomic variability as a driver of plant–pathogen coevolution?;citation_pages=24-30;citation_volume=18;citation_year=2014;citation_pmid=24491596;citation_doi=10.1016/j.pbi.2013.12.003" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=B. Kobe;citation_author=J. Deisenhofer;citation_title=The leucine-rich repeat: a versatile binding motif.;citation_pages=415-421;citation_volume=19;citation_year=1994;citation_issue=10;citation_pmid=7817399;citation_doi=10.1016/0968-0004(94)90090-6" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=A. Kohler;citation_author=C. Rinaldi;citation_author=S. Duplessis;citation_author=M. Baucher;citation_author=D. Geelen;citation_title=Genome-wide identification of NBS resistance genes in Populus trichocarpa.;citation_pages=619-636;citation_volume=66;citation_year=2008;citation_issue=6;citation_pmid=18247136;citation_doi=10.1007/s11103-008-9293-9" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. Krogh;citation_author=B. Larsson;citation_author=G. Von Heijne;citation_author=E. L. Sonnhammer;citation_title=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes.;citation_pages=567-580;citation_volume=305;citation_year=2001;citation_issue=3;citation_pmid=11152613;citation_doi=10.1006/jmbi.2000.4315" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=H. Kuang;citation_author=S.-S. Woo;citation_author=B. C. Meyers;citation_author=E. Nevo;citation_author=R. W. Michelmore;citation_title=Multiple genetic processes result in heterogeneous rates of evolution within the major cluster disease resistance genes in lettuce.;citation_volume=16;citation_year=2004;citation_pmid=15494555;citation_doi=10.1105/tpc.104.025502" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=H. Kuang;citation_author=O. E. Ochoa;citation_author=E. Nevo;citation_author=R. W. Michelmore;citation_title=The disease resistance gene Dm3 is infrequent in natural populations of Lactuca serriola due to deletions and frequent gene conversions at the RGC2 locus.;citation_pages=38-48;citation_volume=47;citation_year=2006;citation_issue=1;citation_pmid=16762035;citation_doi=10.1111/j.1365-313X.2006.02755.x" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=D. Leister;citation_title=Tandem and segmental gene duplication and recombination in the evolution of plant disease resistance gene.;citation_pages=116-122;citation_volume=20;citation_year=2004;citation_issue=3;citation_pmid=15049302;citation_doi=10.1016/j.tig.2004.01.007" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. Luo;citation_author=Y. Zhang;citation_author=Q. Hu;citation_author=J. Chen;citation_author=K. Li;citation_title=Dynamic nucleotide-binding site and leucine-rich repeat-encoding genes in the grass family.;citation_volume=159;citation_year=2012;citation_pmid=22422941;citation_doi=10.1104/pp.111.192062" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. V. McDonnell;citation_author=T. Jiang;citation_author=A. E. Keating;citation_author=B. Berger;citation_title=Paircoil2: improved prediction of coiled coils from sequence.;citation_volume=22;citation_year=2006;citation_pmid=16317077;citation_doi=10.1093/bioinformatics/bti797" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=L. McHale;citation_author=X. Tan;citation_author=P. Koehl;citation_author=R. Michelmore;citation_title=Plant NBS-LRR proteins: adaptable guards.;citation_pages=212-212;citation_volume=7;citation_year=2006;citation_issue=4;citation_pmid=16677430;citation_doi=10.1186/gb-2006-7-4-212" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. K. McHale;citation_author=M. J. Truco;citation_author=A. Kozik;citation_author=T. Wroblewski;citation_author=O. E. Ochoa;citation_title=The genomic architecture of disease resistance in lettuce.;citation_pages=565-580;citation_volume=118;citation_year=2009;citation_issue=3;citation_pmid=19005638;citation_doi=10.1007/s00122-008-0921-1" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=B. C. Meyers;citation_author=D. B. Chin;citation_author=K. A. Shen;citation_author=S. Sivaramakrishnan;citation_author=D. O. Lavelle;citation_title=The Major Resistance Gene Cluster in Lettuce Is Highly Duplicated and Spans Several Megabases;citation_pages=1817-1832;citation_volume=10;citation_year=1998;citation_issue=11;citation_pmid=9811791;citation_doi=10.1105/tpc.10.11.1817" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=B. C. Meyers;citation_author=K. A. Shen;citation_author=P. Rohani;citation_author=B. S. Gaut;citation_author=R. W. Michelmore;citation_title=Receptor-like Genes in the Major Resistance Locus of Lettuce Are Subject to Divergent Selection;citation_pages=1833-1846;citation_volume=10;citation_year=1998;citation_issue=11;citation_pmid=9811792;citation_doi=10.1105/tpc.10.11.1833" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=B. C. Meyers;citation_author=A. Kozik;citation_author=A. Griego;citation_author=H. Kuang;citation_author=R. W. Michelmore;citation_title=Genome-wide analysis of NBS-LRR–encoding genes in Arabidopsis.;citation_volume=15;citation_year=2003;citation_pmid=12671079;citation_doi=10.1105/tpc.009308" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=B. C. Meyers;citation_author=S. Kaushik;citation_author=R. S. Nandety;citation_title=Evolving disease resistance genes.;citation_pages=129-134;citation_volume=8;citation_year=2005;citation_issue=2;citation_pmid=15752991;citation_doi=10.1016/j.pbi.2005.01.002" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. W. Michelmore;citation_author=B. C. Meyers;citation_title=Clusters of Resistance Genes in Plants Evolve by Divergent Selection and a Birth-and-Death Process;citation_pages=1113-1130;citation_volume=8;citation_year=1998;citation_issue=11;citation_pmid=9847076" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Plant Pathol.;citation_author=R. Michelmore;citation_author=J. Wong;citation_title=Classical and molecular genetics of Bremia lactucae, cause of lettuce downy mildew.;citation_pages=19-30;citation_volume=122;citation_year=2008;citation_doi=10.1007/s10658-008-9305-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=R. Michelmore;citation_author=E. Marsh;citation_author=S. Seely;citation_author=B. Landry;citation_title=Transformation of lettuce (Lactuca sativa) mediated by Agrobacterium tumefaciens.;citation_pages=439-442;citation_volume=6;citation_year=1987;citation_pmid=24248927" />
<meta name="citation_reference" content="citation_title=Oomycete Genetics and Genomics: Diversity, Plant and Animal Interactions, and Toolbox;citation_title=Bremia lactucae and lettuce downy mildew;citation_author=R. Michelmore;citation_author=O. Ochoa;citation_author=J. Wong;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. Monaghan;citation_author=C. Zipfel;citation_title=Plant pattern recognition receptor complexes at the plasma membrane.;citation_pages=349-357;citation_volume=15;citation_year=2012;citation_issue=4;citation_pmid=22705024;citation_doi=10.1016/j.pbi.2012.05.006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Mondragon-Palomino;citation_author=B. S. Gaut;citation_title=Gene conversion and the evolution of three leucine-rich repeat gene families in Arabidopsis thaliana.;citation_volume=22;citation_year=2005;citation_pmid=16120808;citation_doi=10.1093/molbev/msi241" />
<meta name="citation_reference" content="citation_title=Embnew.news;citation_author=K. B. Nicholas;citation_author=H. Nicholas;citation_author=D. Deerfield;citation_title=GeneDoc: analysis and visualization of genetic variation.;citation_pages=14;citation_volume=4;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=B. Porter;citation_author=M. Paidi;citation_author=R. Ming;citation_author=M. Alam;citation_author=W. Nishijima;citation_title=Genome-wide analysis of Carica papaya reveals a small NBS resistance gene family.;citation_pages=609-626;citation_volume=281;citation_year=2009;citation_issue=6;citation_pmid=19263082;citation_doi=10.1007/s00438-009-0434-x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=E. Richly;citation_author=J. Kurth;citation_author=D. Leister;citation_title=Mode of Amplification and Reorganization of Resistance Genes During Recent Arabidopsis thaliana Evolution;citation_pages=76-84;citation_volume=19;citation_year=2002;citation_issue=1;citation_pmid=11752192;citation_doi=10.1093/oxfordjournals.molbev.a003984" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. C. Ronald;citation_author=B. Beutler;citation_title=Plant and Animal Sensors of Conserved Microbial Signatures;citation_pages=1061-1064;citation_volume=330;citation_year=2010;citation_issue=6007;citation_pmid=21097929;citation_doi=10.1126/science.1189468" />
<meta name="citation_reference" content="citation_journal_title=Molecular and General Genetics MGG;citation_author=H. Schöb;citation_author=C. Kunz;citation_author=F. Meins;citation_title=Silencing of transgenes introduced into leaves by agroinfiltration: a simple, rapid method for investigating sequence requirements for gene silencing.;citation_pages=581-585;citation_volume=256;citation_year=1997;citation_doi=10.1007/s004380050604" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Z.-Q. Shao;citation_author=Y.-M. Zhang;citation_author=Y.-Y. Hang;citation_author=J.-Y. Xue;citation_author=G.-C. Zhou;citation_title=Long-term evolution of nucleotide-binding site-leucine-rich repeat genes: understanding gained from and beyond the legume family.;citation_volume=166;citation_year=2014;citation_pmid=25052854;citation_doi=10.1104/pp.114.243626" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Shen;citation_author=H. Araki;citation_author=L. Chen;citation_author=J.-Q. Chen;citation_author=D. Tian;citation_title=Unique evolutionary mechanism in R-genes under the presence/absence polymorphism in Arabidopsis thaliana.;citation_volume=172;citation_year=2006;citation_pmid=16452149;citation_doi=10.1534/genetics.105.047290" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=K. A. Shen;citation_author=D. B. Chin;citation_author=R. Arroyo-Garcia;citation_author=O. E. Ochoa;citation_author=D. O. Lavelle;citation_title=Dm3 is one member of a large constitutively expressed family of nucleotide binding site-leucine-rich repeat encoding genes.;citation_pages=251-261;citation_volume=15;citation_year=2002;citation_issue=3;citation_pmid=11952128;citation_doi=10.1094/MPMI.2002.15.3.251" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=A. Stamatakis;citation_author=P. Hoover;citation_author=J. Rougemont;citation_title=A rapid bootstrap algorithm for the RAxML web servers.;citation_volume=57;citation_year=2008;citation_pmid=18853362;citation_doi=10.1080/10635150802429642" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=F. L. W. Takken;citation_author=A. Goverse;citation_title=How to build a pathogen detector: structural basis of NB-LRR function.;citation_pages=375-384;citation_volume=15;citation_year=2012;citation_issue=4;citation_pmid=22658703;citation_doi=10.1016/j.pbi.2012.05.001" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=M. J. Truco;citation_author=H. Ashrafi;citation_author=A. Kozik;citation_author=H. van Leeuwen;citation_author=J. Bowers;citation_title=An ultra high-density, transcript-based, genetic map of lettuce.;citation_volume=3;citation_year=2013;citation_pmid=23550116;citation_doi=10.1534/g3.112.004929" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=T. Wroblewski;citation_author=A. Tomczak;citation_author=R. Michelmore;citation_title=Optimization of Agrobacterium-mediated transient assays of gene expression in lettuce, tomato and Arabidopsis.;citation_pages=259-273;citation_volume=3;citation_year=2005;citation_issue=2;citation_pmid=17173625;citation_doi=10.1111/j.1467-7652.2005.00123.x" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=T. Wroblewski;citation_author=U. Piskurewicz;citation_author=A. Tomczak;citation_author=O. Ochoa;citation_author=R. W. Michelmore;citation_title=Silencing of the major family of NBS-LRR-encoding genes in lettuce results in the loss of multiple resistance specificities.;citation_pages=803-818;citation_volume=51;citation_year=2007;citation_issue=5;citation_pmid=17587302;citation_doi=10.1111/j.1365-313X.2007.03182.x" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=T. Wroblewski;citation_author=M. Matvienko;citation_author=U. Piskurewicz;citation_author=H. Xu;citation_author=B. Martineau;citation_title=Distinctive profiles of small RNA couple inverted repeat-induced post-transcriptional gene silencing with endogenous RNA silencing pathways in Arabidopsis.;citation_volume=20;citation_year=2014;citation_pmid=25344399;citation_doi=10.1261/rna.046532.114" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. X. Yue;citation_author=B. C. Meyers;citation_author=J. Q. Chen;citation_author=D. C. Tian;citation_author=S. H. Yang;citation_title=Tracing the origin and evolutionary history of plant nucleotide-binding site-leucine-rich repeat (NBS-LRR) genes.;citation_pages=1049-1063;citation_volume=193;citation_year=2012;citation_issue=4;citation_pmid=22212278;citation_doi=10.1111/j.1469-8137.2011.04006.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Z. Zhang;citation_author=F. Murat;citation_author=C. Pont;citation_author=T. Langin;citation_author=J. Salse;citation_title=Paleo-evolutionary plasticity of plant disease resistance genes.;citation_pages=187;citation_volume=15;citation_year=2014;citation_pmid=24617999;citation_doi=10.1186/1471-2164-15-187" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=T. Zhou;citation_author=Y. Wang;citation_author=J. Q. Chen;citation_author=H. Araki;citation_author=Z. Jing;citation_title=Genome-wide identification of NBS genes in japonica rice reveals significant expansion of divergent non-TIR NBS-LRR genes.;citation_pages=402-415;citation_volume=271;citation_year=2004;citation_issue=4;citation_pmid=15014983;citation_doi=10.1007/s00438-004-0990-z" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Architecture of Disease Resistance Genes in Lettuce" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide motif searches identified 1134 genes in the lettuce reference genome of cv. Salinas that are potentially involved in pathogen recognition, of which 385 were predicted to encode nucleotide binding-leucine rich repeat receptor (NLR) proteins. Using a maximum-likelihood approach, we grouped the NLRs into 25 multigene families and 17 singletons. Forty-one percent of these NLR-encoding genes belong to three families, the largest being RGC16 with 62 genes in cv. Salinas. The majority of NLR-encoding genes are located in five major resistance clusters (MRCs) on chromosomes 1, 2, 3, 4, and 8 and cosegregate with multiple disease resistance phenotypes. Most MRCs contain primarily members of a single NLR gene family but a few are more complex. MRC2 spans 73 Mb and contains 61 NLRs of six different gene families that cosegregate with nine disease resistance phenotypes. MRC3, which is 25 Mb, contains 22 RGC21 genes and colocates with Dm13 . A library of 33 transgenic RNA interference tester stocks was generated for functional analysis of NLR-encoding genes that cosegregated with disease resistance phenotypes in each of the MRCs. Members of four NLR-encoding families, RGC1 , RGC2 , RGC21 , and RGC12 were shown to be required for 16 disease resistance phenotypes in lettuce. The general composition of MRCs is conserved across different genotypes; however, the specific repertoire of NLR-encoding genes varied particularly of the rapidly evolving Type I genes. These tester stocks are valuable resources for future analyses of additional resistance phenotypes." />
<meta name="og-title" property="og:title" content="Genome-Wide Architecture of Disease Resistance Genes in Lettuce" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/12/2655" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide motif searches identified 1134 genes in the lettuce reference genome of cv. Salinas that are potentially involved in pathogen recognition, of which 385 were predicted to encode nucleotide binding-leucine rich repeat receptor (NLR) proteins. Using a maximum-likelihood approach, we grouped the NLRs into 25 multigene families and 17 singletons. Forty-one percent of these NLR-encoding genes belong to three families, the largest being RGC16 with 62 genes in cv. Salinas. The majority of NLR-encoding genes are located in five major resistance clusters (MRCs) on chromosomes 1, 2, 3, 4, and 8 and cosegregate with multiple disease resistance phenotypes. Most MRCs contain primarily members of a single NLR gene family but a few are more complex. MRC2 spans 73 Mb and contains 61 NLRs of six different gene families that cosegregate with nine disease resistance phenotypes. MRC3, which is 25 Mb, contains 22 RGC21 genes and colocates with Dm13 . A library of 33 transgenic RNA interference tester stocks was generated for functional analysis of NLR-encoding genes that cosegregated with disease resistance phenotypes in each of the MRCs. Members of four NLR-encoding families, RGC1 , RGC2 , RGC21 , and RGC12 were shown to be required for 16 disease resistance phenotypes in lettuce. The general composition of MRCs is conserved across different genotypes; however, the specific repertoire of NLR-encoding genes varied particularly of the rapidly evolving Type I genes. These tester stocks are valuable resources for future analyses of additional resistance phenotypes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/12.cover-source.jpg" />
    <title>Genome-Wide Architecture of Disease Resistance Genes in Lettuce | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XW0W5F3X6uPIzdgQb2BjprK10ARVASQgGggkmZW9RM8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460194","apath":"\/ggg\/5\/12\/2655.atom","pisa":"ggg;5\/12\/2655","ac":{"\/ggg\/5\/12\/2655.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/12\/2655.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2655"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/12\/2655"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/12\/2655":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Vp7h3gmdPfddvAU7NxttZBpWQqE6AdqUVHkt+sP3uPo+pn2PJuJsr6lsHVM+8JylaTLlxfND4S7FRQsdlhvhftUU3E6QhLhL4\\\\\\\/6PrgkujrlK7YZbwTlcm1V7LUrzWsW3W15C4MDEh\\\\\\\/tcx98SPJNAE3aS+4Feq\\\\\\\/ax6hMYgnA2B\\\\\\\/Jxkb7GzM9a9AN1gfQpU2\\\\\\\/FzDWS2qN8J\\\\\\\/diyUsWNGniw8syAkRSB0x8plLr9DJRJO6WLHq8jiU7LEFkZP+HpDJlvo80E6GVV4ofUmM+ombEx0rKjyncWpdPJIF6J89lBoc8xDt3+R\\\\\\\/yDX612X7tWVCo48VPtEZAe6B7MVhsRvlSVe9hpNxRDntEE5r54JLiRzV7FbEgNqnSeXOPsAn\\\\\\\/UzF1c8ge9DBDpshcAiKUvuyutZ4XIyRrtX24y2k4CDFd8M6r4H3EwKfRFDKhIlZNX+hBK4gmaapvelDC+sbTkzGYMejHWoBNprmhDmvhH7WC\\\\\\\/KPvMD9X11k1Ukx8FIo6RyKU7kOhFOb1La\\\\\\\/LH4dWv3Tb1qTFHBjD1u1lK0Nz01M8q3qhspieXBa1ddF26FSLHAWCJ869H9vYhAdpM4coZvlkwyEKwnPy1L2M10q0DMWiQnBqHg1Qrv3UPv8IU8dJye8lcs0DsLLMuM+RM+noo8qgjZP3XFpKqA2PwgWNdR3TAhwKkbWHHJ7cL3ujAaKdxHg45upyajNHzrfbhWoTo9b1CqeUN3xDDPFcQTEGbphRZ2UuHec5OTBkW+bgjgXP8iCrf\\\\\\\/fVsOlFocP190eFrqvoQMmbBJI7hF+21ONVyK\\\\\\\/kzvnhMySf17KadZ3ruZeSjpwnjTXuKhXk58tCoygBCEqu2etK7D3UjvOJxlGhIphU5BY4y0nZ3ZfVjPbsBMeP57rpWB8nml8HZNo6pf9eecXQKk\\\\\\\/T3QxNH\\\\\\\/wlBwxNLQ+q2AYC4vycmSlkSG8Y8eFKFuy7lgf0j04ZU\\\\\\\/hJG33\\\\\\\/ntwZ4q+w1sSoKKe04lgDydLfRaoPitoCJQIn1+iADih1ZMbHzV5UeE1BvmuOSUhoU4cJsRWFth47N5uEtEUkVitvJFCRbWsymUl1SmawWTqVtfOH76MTCFwSRUM5a5l1pDTGN9bN64TDjMAXm1SsOQVYTJa32b2NicMp9KMvxC6fEGVeLvCi3xcgrT1YKVtBELJt0SmBvFlhjzjI0M3vWGhiM7qkExoeL65uJcfnQJ5gkwwBr+fIGZapCM7OhT9H85Vldhmxtdl95FelgU7M+jXXjzfJkSrlwwJg3qwWObjpHSD\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u00226PusZM2HVjmTu0cG+YsVtQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222d97592577edbc475d74eaf4d0c0d0a3475d3141d789ee5967a2237bc568752d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GOY4dyxqDKjVrTagJhMt3o8JFGUIZaSEeVQa5ItrWiSrN8Vlyi6lLuisAommGyodZzj8wAuK5QbUOGdjWSopAi7o0NlcOBYo70dmpJyZuZXKsrW2lecbYwyI\\\\\\\/IRgDB59hABFTyBZEuqAdVLML6J1HELX7mBlympRIe+WrHwYkOeTPZPnz1zQu3RTOkjC\\\\\\\/Jaar\\\\\\\/ZkcVscRwziy0+eDqTl6n1E+nEK4Lu\\\\\\\/rmE6nVFCiUox94tTT1wFFtnsqRICGUr2mcIW8l6qUfmiNwlSFotIHEjXTUMH7InJ4YkC26GcDoCSDU24wlcxu1jfm9dd5SeeNeKb6Vf53LEg6JrxGQwOYWTsfGnvx1UK62wYt0GS6eNV+aXeiE4CUzD572Bw0yNebDN5cBbvUE733uBnVrFhyXhMIhrsLpNlcAtffr++m7zCORVPgAHEZynwmeQPkxPjbcRKNkbC+J+vKX7QgNn\\\\\\\/gYjOXT5Xu1o2Vd4H4BSWJk0ZLzPzyc86VDa6hODN8Yz9RW0meU7hkDe6DIxFVX7nPPRx5vLHYGgPf8xhJPuYoex2LbRRHNf02YxU2De6DoKeGiaZXshatG5837Nciu06oxp9CBDJYj02a+uOt2z\\\\\\\/gXbOYqNFrNGhEpXQHHrl7l5eLKI8mNeAFlwuO8f45wrf60SUk3VPGQ4deup6THrtRcF776KXcFYDU0b3Tq9AS9\\\\\\\/OlHPuNR7gbP+bswjc5UU81DdE\\\\\\\/gBs0yhczX7lIq0Of\\\\\\\/Yoj03NSBQj5OS2jvWLP2clA6FtTb5z2mIhEyGy5Ido78lbpu+ixMMs7h55uojHIMsVTMmPGfVtTjevk+Pr5YMCDZ14DBfjJHb1Z\\\\\\\/YSp7NufzAnLZwfBpZjy9PYOc3D\\\\\\\/lBRG0KDxjCSsS1nVJjoE699WN7+2EoQw47+IGIc+HwRLUMJaHAhAU5Babrh\\\\\\\/78Z76savtiLVkDOXcQSuzSIvYO8OiTRHOWBJax7v6n3Zd1sLfEIGiYxLIVkk4CYP4cWxupgaGtYW8DZ7D24lyB0iw\\\\\\\/G8Qf\\\\\\\/aWgNsmgUym9PKNfTbGeSEXTQ0dBZp10pHRXnpBvHP33+y4MxPRd8ZdkqM6WEHzHH\\\\\\\/VQh+1fbE\\\\\\\/T4BFe0J+Yb05MgFZInm\\\\\\\/JzYWaAHIA3fl+59Ni5Jc8lFnkx6R43ywhjGMu13wpCieH8RUYrqfpnccRRE1JhELUM2D9Ty+JjnxeMMWVv4t\\\\\\\/4H6rAqGNazv3OlgjeqoiM54nv0D6dSY4N4psHTcE84XnNDWf0hFsZOfgbqabWeKncnkf7BkaSBEvJpTbdDvLBmBc2m3BuvDzPaSokWpk\\\\\\\/+DQ7UD3efbcoZ4yK9mzp325sI1s+vgmqEwNYHQdVSSzwagjitJhNPDK4kWPa6gIZJJkilukPqq3YJVGMDFIi3pTDt2iNg1QBQwpb0Nm6oQj5mzqcP7G8hSdgNV7QupCXPJCp8r\\\\\\\/uCqVxVO9rkm8hNYt48AKPqBmdgmT4KcVvCnha9YiXzFXk4bgxUZJdxCDZwoPIO7RdQbrobN5fljAgJLxx5u3QYg\\\\\\\/v1+KsUAG+\\\\\\\/+\\\\\\\/62vToZIJu3G6uCQFB5u5bSaWGuyaQaEpK69LJj2f0731yNTl9Kv+dhiW0Bez5C8dZxyN3KqL07dloJswglq\\\\\\\/UHYuBbNqK3CdXg\\\\\\\/bL6eTkqm8kSfprGf+pPgwtFqo6CiJIdaTligGJgty2lI7kY1Gewoih+OFzg85oLvLMlgInTjoeafKV1sUV\\\\\\\/ly5SySfxQKC5PGfEOfaxoFBKsC5W8tQKVvrvzdbIbZYWhozSOxuhTaOhfqGeKruVEXNM8fCX7mOsvtOGGO70qHvvvG9eLnuE7lTnwlKe6Qd7PAPw\\\\\\\/qa1PMbyNbsZSUlXljIRoPOhe3cOmL5gCIWSNyVePvdFxdUtl6Bi5jZKxxGHv\\\\\\\/GOIJQtC35apkVdVwkRtM9geg6snCtyihvwVfOBzEBc1Sqkol\\\\\\\/awkIE0ru52jub33SN4DGUOMSeL0mpAiDHJNtc+FkU2c+UsxlPE\\\\\\\/FHlRAUwgqHxjj0E79ckv2iuw12Mcr\\\\\\\/43iTc3\\\\\\\/s65bK8rzwl5H\\\\\\\/NA3Q3cGvJxPfrP3DelgBUEYT4ofI3Y+weYz3abD7e0Dz1fFs\\\\\\\/KRjk17xT12BpDHnLg0YKVSHTnrrGRfEkHkXjiULCJ1jEwz9AMnAVCW0nia52f8t\\\\\\\/E69Fe35Q8A663eDFcb0e9JwBnVPxmp+jhJgHvUWGcP10sJJKga3WQgwFjZO\\\\\\\/6pO8\\\\\\\/FAryXggMvz4EWq2h2lAt007oiMRSuBQpQOq5YOlvgqmQG7NYcLHyzlhKZuUx+KlLZSgsBHqdUTcxdD98Wty6WGmL4COGqUO3e+7zFEcfmaJn9iupNnX\\\\\\\/Xgpi4Pe1u1bCD6cup7cuAyGW7ohKlaNr9ovTzH0wevGmbxR4PLURKf5X5Jm5oEpyJEzYdfVa1yXOk5SOFVt2WebmUPsxNckBsGE+AF4ASo3uO2Izv6fMuhhmfztWXEv0QlRvYunYt8lHdUdfDpi6t7sTdJHn3cKToC2QqjaWxhyM+U8f64He9mvIRKR9yEknlfobcMI4uWKV0Gu5cpo0loTdQ97f3Y1l7Fdpl0Wrrur0iaf53W8pAxNTqQtga0vh8XqzcFFmGPNOfQIdSKI3UzWGAQEsPC\\\\\\\/ynOFd1m4DKLVpRfNLUVIRbod8Pv62+ORS9bdgAPBTUrCN\\\\\\\/NnCeUdGqn\\\\\\\/cCK96HuQO7E+s0sORhtmfK19FHpqHVaGYbkUu+cJ7TFtq8g+b3VVOzP0EZTZc+5t26+iDu8MTOuTIwMyHm8uNaomzQlmTqYf4g5hIESxERp0v9lZN\\\\\\\/w4MxZIpyWmCQ58SfMZhZnlAOUR1cnH7pJ1DQfENs\\\\\\\/rG6WG15UJ4NzSUsTlhJbBYKPcqjjUweEEzvFHNK1vf3bI7stPAkjQ\\\\\\\/1EH01dejDCXgjPrGvtqNWyE6tT9ZJQjW3qjoXnzTJ7FsynlK2jyQbMZgrm1F6tC0isnTd8Y9B7OCj4lDLKht9M6Efd7QFSj6\\\\\\\/OROTNYtEbiR73BoUX\\\\\\\/1vcvHfqZkgoKR77mijV2tADLQpptZQzpm+4zxgZL0GVSN3bfVjtjJrwv4Rcv4mwcx9fCXdDh3qQ7Zhr1FOms67AVhocxPtadKVVv7o5+4IiOFQXaOg8qwPzc3tL6AyRbmMnCcABoiy6W98+s44xfpsxTdRFgU2sHuIANJRxdVf5UrS4ulJyWTWMhv3U6nVbv5ahRl7U8N0AtNWIPKCai9a3Odzz7Gn\\\\\\\/vz4wdVJ8Lsk\\\\\\\/mrRX5rfVNpF7nXfFBSYeh4HUEVYLFRcF1H4V+z+3ZPEOe9LMDM5PNh79dGCbVxTzpzQ13\\\\\\\/RW5nGNgOPXHNqooQIyAG3S9p+eM3RDMX77OMEuWYkHcYKF4kLPx4eNfN4gTA76Q8j1NIVVvwDiE6BExv\\\\\\\/RPtYHcMpnzoXEB1LTl6azFsjStt9QJbOzhREkb+7odaK\\\\\\\/Oo8KBy25atLnfKk2gGLthbhFHF5CRa8M17Tt7w+LMHJL\\\\\\\/utugF591XtPT2U\\\\\\\/riK93xGLa1Z9uXbHeOVj0pTomqSW7oHronpkF\\\\\\\/suPZcwWlO\\\\\\\/x59WgxsdTpbWkYG5dZU6SOQ0raiSfcCr+HXCJ\\\\\\\/CnyqU0yhvYGK0IFlM3t09gsdYMJIEJ3dm5hM+FmABaKn6bifrgPbKm0ajNZfUGJrPLlhaPUlFJBMN5VMK9AtEFpqSVyizWnHtlaRs2JsVTioKZE3tve5XPsadYAlIpRT1jvuqeSMbkMZsLguPoE+8FmSQL7Di1Kmw1Z5wXvBhNx8Gu2e\\\\\\\/WTpC0Ztoe\\\\\\\/lBqhBBU\\\\\\\/m40yO\\\\\\\/gqONPjH97mlZKX15AH4mmZ5\\\\\\\/OwPcurtitm\\\\\\\/xghG3cR9bcDKKxIg3fdT73M8gek1GWl1sqv3EP3\\\\\\\/\\\\\\\/t4JMhaPKdjK1MFOSncBy204RnWapx1w2lu7A0bXwe2uEbb2SA9AFGg9wdBMEi7BxmBdulT3ny3\\\\\\\/fUHiGgiJ3bKsBArEpC79UKfc6WfdwLc72LRgEk1RNu2UMsMsompePUduVo\\\\\\\/bqCnUoxiVQ9px2H1GhEkVZHQ5BO4+Z9NVVmpUDq1ZCSecSQX1cY8yanwM5fk7DcJ6twsMYXXrjNT55QUxqEa7cMMdJ3IKSJeWq6cJEuJ7E7nHcg07FWy9iTBtkkvptwtyNi80cqnIVI7cGXWkQu4vl+lAt3kT4L5OvJKPU\\\\\\\/2gPYoW4SFBXhyg2hco0lOm8A3vRnLhGefm4tqLdWW3ezlBHL6DKERqRzfNXklwIF1TeJONRBkJqlJm602ltrN9Afy+2t9T1EE806GJA8dXO58xE38RXcBH0Rl36A2dvY88CYrHecWarLt4Oozz2zsFhU9JLFnFnJB\\\\\\\/PZUlDLBui8pdmbKLkz8IH6KHY3ymVZg+6un30\\\\\\\/0vk73UwqJ2ocDatg8SL3oBLT9qqySnqVmUxq750U8o4Qt7aueB2GOGP8BJUKkU+3\\\\\\\/IJqc9tWXugLfe40C+SgMSkW2IgUNUEsz0K1NPHbb6i7QnOynFTk5UM3Ihh+33awRtYxXwacXKixCICb4SqZFbgv9RJoSJM7foXGdSc59wWpjl47zFBSK8ciJG94Lt5fNTJzYO17qEDdS75Q9ujoUW2m0xJYgjiZD49t6jTvQ\\\\\\\/q92NdvPFEEjYMX5CXeOYZPBH50NZ32xmP4luiOe18bBw75YIBF8PDyev1CiGxNlLpet70Ct7FTvx6L2SYh69M5cbZsuL3hEfwcBfQMDSZ6Rm5jnIXnBIaQO6AHrTnC0R8Gi+vu0wilAWmDXcBmaRYSYAMRF\\\\\\\/iQcCsMQje+47Tf903kiaZVIZMBj1yPtxYMC4\\\\\\\/xGjN4CEjabB6luRhB6cHAbTKqdT4F3c8G\\\\\\\/j\\\\\\\/oIFxqPLpWpF7uUx+APvHpIlv8pWNqPplQcGcK+fmlUmlHitL+coUDV9p4LKGU3Vw5Q7KRpqXcHO4bt+tsmNoFK5EOUMnZFqLqvIEelxbJ2ailaziv6Fk1D46Urh1hBljlNgD0E2TFzlaoZjietOXWFcJhNjmFn1iQtG+S8N6cua2r4SRu2EIhdicu5itjXZDbwDOOyYWLSCWwXGxBWHQuwkFEoz6AehenuihkMfLh5vu1plqsCq0uag9qTnsg2u1kf4Z\\\\\\\/Fxu+PTPRFG\\\\\\\/daDHY6AbBK+SMH3s8zEscieC9C+8jMdJknKpIJ\\\\\\\/lIc5rxbuZMBb9UzbtMVQqS1t3WMaOBsK4jFafnYPs6QP1Q9o9bwuD9G7zNX5Sg7pZ0qi8Vc2Y\\\\\\\/6LS8teRM1AQTyxeIxdbhmFjatpp6SFZRaTm6SIni8RGjNZbIAtS+x4GlA\\\\\\\/esVfq5876H7RMdH5Yu\\\\\\\/M7jiKRmbgswLkI29lJ2BRUucBP6pWO8EUYuFuzgkq2YhwKWDtjpN\\\\\\\/JmaTLcL81J1zb0Cpd1YEr1yp0II6WLeHc1evfzP3YUdPLbB188OgvStWGgVOmZz3n0RWhTfMcGkuEHOusA6\\\\\\\/QclVqBclnHjRMYri6zCpfMW1tSXI1oAt4clZzNigqeJtV3yEwCQgBHh8uu1v8GY2opy209NgGD1xFeMIKqEnPanwKgeN4MRr3nHLm3IYtNUuUXYCj+SXINX1uDyS0GDNR1Uva7n3NGjSc\\\\\\\/bPTXEbNm7yZ5c6IQAPxfJrUsSPCIYVx4V\\\\\\\/RSjKxSJ+\\\\\\\/OCoTNoO\\\\\\\/zJ5BYsGbOUcVVuileOJvJMwyinuyVEbpd8XOlEy\\\\\\\/K6x74\\\\\\\/fOX1mOb6DBQpDVM7lI7N1HTggKYstSSGiynu\\\\\\\/skfG9Z1eP0RgNfvcSXEs8ZDxFihz6vxrRVG+bUXs86tmD\\\\\\\/aBzfjv0q4ZoG2xTQ4vV5qso9Gri0rgHifgG0NLVIimdELJHJ4FtWzQjVCfmzSW6wU7+bi+KXH9DQHx0jYfyvNAAp7QW\\\\\\\/ovJB6Kxs9h+aeBGx2aZglsOh+3lmU8ZV4wgI1wijgzTDdRFuN8fAf63aVFri8CmXBfn7wEk5cUP2hmy0A8wkKKDd+OdZxqUfTCDlemkPZ5u9COHkYyAbg3yS+7cX3PypSvMe5VQ4v30va4beQslZmCKPF9FjFw7EpkCgS\\\\\\\/buygnbYgjwgnXUoqBno55U5LG87g+xevO4A7pZyae+Syf6MozOr9wM3W6d3Tcj2gHk39dkV+6DDDU2ENdvTJVKa+IZKewJ8kVxPFqBPto62HjMPo6+7QMuHul8Wf3EgydQIPKHlyZrcp6qSLzpxER2lN4qXYqcOuWzelOayEkbAE6vs3njKL\\\\\\\/7YdEjtUzGacF1578OnYD6iyuy0x3eaqCv2fxqToonf6joTpKyLWRRVmONRKufiCyOPjSXtnwcsV9\\\\\\\/RV3qXZWYl9Kva5zxqQXugD+UmP9ojj\\\\\\\/aQ0rZj+093A5uWJGKYiaL66eLfWu9phPxc1cJw\\\\\\\/B\\\\\\\/B5I69s1eo3X7wFC2zJZZOJbf0Rb\\\\\\\/JgGueLe1oJtA\\\\\\\/pSdZLaS4wLflnXeOxTaWCfTu0e2+vhkdngTlSlZo333WjYi9aOHzWsCJRAU1neg\\\\\\\/DoC7OCgRyZuhaYI12RT5aP67qiOgDqYd44kKM2gYPk3n42cMAPQ+d\\\\\\\/pvsMcAjUegQl0oDXLZG1mh1VukpagP1\\\\\\\/2uYsGr5HMtDx3UD9wE04AKp65pSZkgzhHox+xiEG5EmCpKfW2HHhRFY39s4pLokmjSxtitpo+rAX27DQ6RR+PVXMC4QP7JwootRQdG7rkTAEfxnhOGWphkVlqDpHTOyfmhOQ8Mn2Wiv23D8TSOia9G\\\\\\\/CJvk8jLWRsWlyKm3w1ph8\\\\\\\/esMGQM1C158aclgyULaY+DHsBW1ej7KBvhmA3ka3RjZDhBL4yHaXxRFSYFDw6JB334XEWKe5EWENxxsj+RV7YJgt99togvAggPCjEYrvk3v0TZbd1O6F4F5cdq2CK3i1o\\\\\\\/3B17Rozevp+6Fc8Alf0WCr7L5fi71RHuarNVjp++RZAdmSyLMwI\\\\\\\/a1F16XxTo0KoC7g35MRCEf7usOC4KSwVXm3hq8FOJEs9fQD3DRKBqGXwl8XW1\\\\\\\/nE3WZ6jvDG7mTZpztxA16LZViDLC\\\\\\\/4Q\\\\\\\/UDeY\\\\\\\/jgRD\\\\\\\/eCboP+TDrYqljE9tltPXJV5YsPyVT7eKTx2Vc7oFQDBFoSsO07KxELsh8v1qkAgRS7NFyT6lsVGMS+4dvHvg1AijjreZ0qgyv0LDcnQTq4O0uzgXm1K1MZPdNXnNC4jmllIeebHy8NjmWCBl+zsxtdbwvID8EWFjajEdTA1YyitgWxy+oSk9mKeWv5qCgflwnk9gQ63Y5\\\\\\\/sKkcHkrnIg+lENHF6C1PFaTAHYeXGOSMo2BSk8jbpcVLIj1aKvvMHY41aPpCGHc6Iz+rIpo9e4XZceV09Vfn5XNX4FS56UiWsCCu5QIFrUZxHwoOZ4Efz2SeI11Fn0R67PplesTxiIr6XFQvvt3Ubhoi+b+I0a\\\\\\\/mS5kZ1LEcYDvXSBHULf2jUKTaz2X5wvfpN8+UprYdnUWf27IkZzR3OBoGalhq530h8o24nHYYtu58rKdx\\\\\\\/fvZVibyw9ovsqQAeYCppTaWthqgUOhdaejJQqChvpqgXVLkUMdb5v7nJNJHhbjLVdLnUkJPgy+4+Hgl\\\\\\\/8Dke7k8POVqCCOCnPIIkvwmAcpgUdBEloF2FyfC+DYHPdmlBV8T+DfNQZn+gsJQ1FvZYkNIoZtsko53PRQGo+dfpua9D9faXvc0j2dNXbnbjOt8kwL5keAuGXD5bEazsKjzYsRkst5awQgx3Qyzk8L\\\\\\\/L6vBaehHQl6tD9Ar6lsURaUEQirQyH9RQlKk8UUyFs9kudrlo0uc6jmAOQjDyHs5v+gZuRP2ezuscosPf94B5MRNxpT7us25QvOEn7BeurImiIRjyK+YUsY9sS7\\\\\\\/lvPw38wObh8qrRu1ormdq4hdoghS73f5Erf+Z0CWs+I8UPckPe0oUDmfrvk2lzGI1lDbEpw9WBTkUKkbVMF0xeHRXk1yQvpsPb0qXESUTzJkK10eNUQUz6A6ypBHxv8iVUq4X0K1upsmxSp7xvryf8ZJbKm00RDV8prZoBCq5H\\\\\\\/pefiTaWJFSAwAUFSpmpe5Cgexmgt0QMSTeJqZxCA4QIcyjlJg\\\\\\\/I8mYVVDPh13gXFGIKZ3UpBmgLTcfRhIy0+tcw8EZ2\\\\\\\/1CvfO6m7h\\\\\\\/ptVthZJM\\\\\\\/EgJvQVBByWcymFIarVx3q79p9rf0TWouz9UsH0tkzF2sZ2LHzBXz55F4JrhkRDBVCy6sxN51Sn16quydoGWTdrT3+nXEvDQVzPJ2t+zipzdqiefCDggx4v\\\\\\\/sVRcBNcZFmIK4q8SlNkQhS\\\\\\\/CYntdq8YPmiES+\\\\\\\/6aqDpWSPPNeR+DDR+Ux8yw56EnMjVjNz77ntb9B1CAB\\\\\\\/W+blW5cK0IYNXPAYRJCNld5T9UKfebyHqClpujmsJFARrFMERhhQ8RGV4OFb8uDqFgdi7PZ0hDn6NcPlt7DdyxCt7w\\\\\\\/75i8PuwARu6EkhKayz7Bh6mb7shKhsYwHEhJfpLeKDbUfaw3H0zw7MZf5LpL55o8XMTDKoSlSc81r\\\\\\\/hHBSvNOsiVWOloIguECGnlyv5GgQI1QvYFIL6bevDol+L3TIWfXifWdwHi+3GBwTgFG0VFSwTE3Iw2U7H267U9G+dfkkpiB+GmN8gaxJXaEB8hk+L1LO6TVyemnbsUOFhA23G5Uk6DrRBXpl3XE53pahB6JyJLQ1Wu82aonOkZp\\\\\\\/Q1QlqbRjMHKmbP83xXvl29zJFe4UMiGcIsxMcmlJUoWI3A6qTcKTu7kmzloyAaWBO1I8IsVzDgz70iwDiwNimmJeUqH5jhrO8zl\\\\\\\/VD8\\\\\\\/z3bLj0SnR74NrgPkZB2gSQ\\\\\\\/KVUc19WMvtMIb2eatT6HSrb7qn5KzJdvlM39FQb\\\\\\\/nQM9\\\\\\\/jktbUf4Cyq9Y65JLcbf4RNR1wgQjtkz7ETMfHPWjP\\\\\\\/up5n5n1K3M8AJjV2v1NvEvv6KPihyalUBJI\\\\\\\/FjEtI0jk3kPaxTv8A9njg0Gtyx\\\\\\\/pxGgqEvA3eZq4w7PvON\\\\\\\/96fuG1yFnt7GaJm51yxB6Nv6SdPkj0O5GAhYguUXs5auRMaVK5wSuCAbHMuqPcKMjHVIeUh8pFf2Dev2K269ZzS30Kji6lsioL2Rk95mfeBUJikvVbRFtEhYozXoAvKfbEBnzhDaTatC8THF3A01dLYh+dm0GsP7nVhBah1PQvpWGkuZUEIkamBuLUyiafGmEuHia7eR8pSHEXlWhFvLF+IpdCnQ0Ex5hZrn3ShtEoaiB8izFf6qEgE+NuHai0NxMG7GN3sz81CpdxutY33bAXddh7XYpW+e7uF7\\\\\\\/Pdppf09NrQ5biqSb2XnpEssLAH+7sbCXnZW0MfDY219XhMIs8U8cUnpTXdfUZEq3TTnwFV8EoEkcPtmjcHN4AAuE8+1JYFxii3pMe8rOp9f1NfzwYs6FL3J+rh23fCvlW7VsM5xCcMtee69kkiYoMssL3g++MTNzsoNp5qIHiDnUN5SqOuTcktOirFx5QDb6LQkmUXeL\\\\\\\/wTnXpOJ1jjZoz2cgiDlO14nfLVhw3ei\\\\\\\/3\\\\\\\/2qBM8\\\\\\\/9xfFLOt9+KhLAF3Qvayp1UDzN5yhrTkR7CH9eCKOK0HebWe9CMocLePcgC6eZxXeV3tchoX5f\\\\\\\/ixSS1tViT5KCEqqHqC3h+yhXBbCD5W+qp63N8kPDVC6nfhSUhiNXYOqpMKLn7wgm0Ba4PvaMTN6sio3+7ttrdBlSFFzx9AIbO4xueEiGY+uf1AyOV2flpKG5L4v7hB3Jl8IV+u2R7lEUM6BrS32NE0AfeJPFq189azUJ0zwdfOjjOFuVWuWVNwHgBRrmQ8bbN4a1TVPGJL+L2kQw5Ch3R0aIX9OiXyvpGSR0fSxyZKTnMTl8iFQJ8ZMLMELpiZvkrbZfyBOpYw7Zuw0Gy+xTed8ClrlyNyUBXgtRt4Q6twiu+2tpOE0oVnbuANKtoH9XLS2Iefe5iHyk2QYIkg92a04xt+BjCcN0TVwWDDFJb8CLaCiocKxWNno\\\\\\\/vgDHnliosplkf5wT\\\\\\\/t5eZKo7Oklyv+OXUKNCPktp1AXsEoRqHibOP512gTsa5xa\\\\\\\/R4PNoi+4w\\\\\\\/3tyK60jt0+ewKd8QiAKiMNerU0pCEEtz8xnF8EIxgrSl7wq\\\\\\\/GgcrkwXeXZmN6pF7gpYYx83ybfG8tApocR1D03UnMuhzp6UtNPuXrY2+cc6hqZj\\\\\\\/Vh6okjuv4d+F2wV7Wkgrh11QnmI3kMHJgX5ILCwSWGqeNM\\\\\\\/pmvPdpzgpGaeJETIqDVbN4PLgJEwnapN8\\\\\\\/jCjBM9ksMp42jfziV75BEO+8RDmzZF1lCvgwgBfcN3TrEf9recn1I92JjSp7xg+oYxC6AsEqoIDDgMpHf\\\\\\\/HLQ8bH68tZNP97h8VxVoCLZCFQjAdwdJsPtymxcZuNNThL105\\\\\\\/FXeJMgM4cFXqZQMWQwzxSS\\\\\\\/AmFz2jf\\\\\\\/NAzn0kVhTyvljmsXXzHB2NhCoGtvWOH91m+NpjraLHBglcw\\\\\\\/zqcVDVZdjf3n0Q+LpgJeMXKGhLrDw2bACYJX8gXBRI2TikIbJ203t9C8n1xplRHhC7dZBmgmsvY9tZglZjT2X4dy4vFFENyGh0O17IpB7jTuG3bIymc04aQUUnMXklAOt4ESamPhFD0FCOVNyq3UX7xAON9GU8MOuV0SQ2cHDod7KcLnTYxD7ov8z9bTfOHzUnibBjVURlUjCy2vHsiyrQGP7y1qTHetp6im6Dgg\\\\\\\/oBfdsswt45OC\\\\\\\/c5XsbO9pvM7O1tDnGcjlQT1nkDj05OznhzJeujCoY38BDOW703eEER1Upwh8cF3QpRYDMkhMNxcnrWFcnKBydQI0ZDCAvDs8WKnZR1dPnze2SA8rmS0JTTNM5NjxB9xPWzK3kdafJbQ3zdBcQXqIEbHG0iSOyBx1\\\\\\\/9XmRD3XcRavp43acE8EdA4gtb7LCoSKSpTHKDjue5x3sCcygu+6RMhOQHffkGoeWED1cRPWDukMKaLyerpxoYi2pE1pqSrlX8aClwa7b\\\\\\\/3bokB6STsQaWi9PCu5B\\\\\\\/p5g0z8epYirn5gKr3IS2hGQ7nPkZ36yq7sT5c1mNiaIcDMjvs99pv7yMQbKdv4WWeIv1lEOl9Qa54NFvffa5Uy\\\\\\\/EQvZuIHOIH\\\\\\\/2pN0i4V27NiXolovABgkA88GepVpOK+Fb3pYNfqp4O52mMQg6EcLmWdVnnrr7aS+MZdEkjZeo91mB\\\\\\\/ZbfdboIsM8C8QVQHlwTpqq5VMtPdBDXrin0qGZ861Iie971V7jlN9C70ZF6L7UmNEuAF450kqMRiSvlpp1gl7Jof\\\\\\\/DQO9tIylhYqlOZvx9yl6sm++Fx2C+OoRh4Jg3Y4xFKEmmiAFp2fp26J1NNIOwdnRO2G9geh8ah5mCKJduHX7sCLtPI49IAm9a9pGTKNkkCdNteWgG1uT8qV4lR1VIR1GIbWJBuKt2d3ESlLc3pfS5tdnBr2QPheRzI74hGUZl9avb3M\\\\\\\/\\\\\\\/OdXv4aNI9Uj2i0iQi8IrWqAhZhivpwbd1gVaYWvKkvgI11vTwj2t+6Zj8YS72kyIOB6+QpxvUjGBjXFaqJtQFtaGPw4rc0Kih0GuO4\\\\\\\/FJLle4Fb9kjDWpc6yDWGVIvofmu5lV1tsifIHTl4LOwVJelrcbzA1bn7Zg+Gr92XhSu5brGPwbGddXX9J8vKvzFBvQEYkhdEtRxjS0MOf94Ashrpp8aZ\\\\\\\/v1tMLrS6fyymnPbKKVH3asGSR5j2QRGqM4YM5opsanBcY1L2UyKQ4gpKhIgVktCaRVjcJQqsAHBTuaaDm+vkGk+OfhnmNQ18vtCbogSG3GnaZkfdjyqMB+2UruYCBJer7SJ2evlOIHx3usHIKewGSP82WcHHLonu29\\\\\\\/A2NXud+cmmh9Ux8Yt0zb4\\\\\\\/ez0Tx5cR5GOGpCa4E316FYs7FBurSezZOEHCXtLRMcEcAYX06hNXvmuv5HWo4cjUfCEdNeNwQbyMPEWt5wnmjGg9M8lRniW5\\\\\\\/RQUfyH9WZWN7InFO+n1qftX0YqPvKuvgFofpeRFkhSgMvbmW097ijQv2ltugQ3PXjD+lHZBgopYNwRmvUlWez6Hnx6BM1kd8Gnxc5jcwuntTWdzypu2fmhirDz1JPDVQ6XF1NEvL5UptkrI19m6qCfrLmKtSVUGVLF3AWz1T\\\\\\\/I4XEUYes\\\\\\\/ulzaNN+YwvEmBfR1n8FSphi54o8o13v2gzhLeVw+pTgHd5JeIjfZUOgkIwXWPNsBJHDF8pL7tfVal4kYcGmQ047eT\\\\\\\/F79oShq2fZ7br8wwq6zZEqHLfdyFfFcPwfgWTS8zPWF6JsUiuP95q9HF16NtwGwYn8sfe2w5UmFPs7OyIUkXFozAiITDRNETzLbLY9itoGmBt7n+E48Mj0Jwf2QvLdsbfvA8VIZgCLhRsa1CGRPjOWSZyQO0Mz3dOYC1PnH1AXAGTzqx6JgrbCutKi+zVrQy1auUSrUXUnBqArtiDPEoV8K9nXuLEVjm1qiQSCFFEes+fediWzLc6dXtns3xDhMIde\\\\\\\/tDb5TH+e3XSSX5r7KBTVfMkYoRdNt7KHu1sNtM97b01F\\\\\\\/e\\\\\\\/ogwruulAzVgzGlabyr+Yrj8I+k67Z9WmtA+9zid6UC1JcV\\\\\\\/BHW7qkGLfZrLSHkDJnWb9qpUSF70uze3sd4nFt3ex4U3egWEY\\\\\\\/U9He4ej+ShGo6x8lHK9eouGXXl4+y3FYEO7iuAUeIQLtzgJAZUrOf4f7Dq7bK1c8vrGizMUYCKl8O\\\\\\\/iV0iKrlAQxPtqxaWYdWzVCBDZW\\\\\\\/VyCS1wlHH43Tqwhv\\\\\\\/YO9hpfjWaoK6cP1DbgV\\\\\\\/6d++zCoxBlyhxJGGQNboWSGR1cZXP9y\\\\\\\/Loljfm04q+bEt9DMrisROEL\\\\\\\/UP8n\\\\\\\/OWUqwfK\\\\\\\/9t\\\\\\\/FH9BDQ2BgXmOrrUoNYPHGF7JHiuH2etW4p21XKniVp6\\\\\\\/7qFLBcifdZnYbIbNPovrjLMuEvr1elzjOauusViKBl\\\\\\\/rCghQY2T4mfFnYFgjQPC2W+OpHUOD+WxebJs8Rk9mdqzWrXdduht2uWasKrd8ghTM5XDxCdiYPIqMflnP\\\\\\\/6vLACcL\\\\\\\/s5WUy7R0N3U9MvFkAV6Z5miuzOS\\\\\\\/Eyq9nieP8KakZYZ5NPrvH6ro7nesbPFwZL6kXWtJIZV58drIyaJewoqHz5+zJjeJ9CuQKZYY3XXpBzPRs8kWmkRNE+OQFN7hDPC70LVq7zin9D52+n8Mrgf77ZS0KYnhJ26kzrNP5R+HhLqWRZILVMxqiAL9R\\\\\\\/kMaaT2ym0\\\\\\\/\\\\\\\/IprSE6NXcV6rfRLiebWIdFUdszdyjS5z1P1Pc9UFXv2KeuOooofGArBfLCMcvylIIqVMa7o97gF07SUhd7U0XkzQigLy0mZjkiv2N9FMh78bG+9IeWxARnNtIvAQTEZhGYdX1KcXuRItsT0Z+RBCmT3YmcvgXC2+nx7PDaYmfKeH6lLVQu4gC3rh0mEOVtE17SVO5IYD2cmEQkUa2UTOiO5fq0SjPoKIjPH2twwZLD3igVRP0\\\\\\\/7dxGaCtS5l+1qsTvF4\\\\\\\/G1A\\\\\\\/0txGuwNz0KQ55lNEad5vrYP7ibZZWCsyXMboOHCBR5xdIZLV5GeWMEX4Q\\\\\\\/vuh27AKraJkzzjuidHufvPeDCr7Qo+lnvJeiWN3arL6c8912qxBJkGO2WRbjo9dsBpMBa79SFKRE2BGeT1qKgEKdnqcjhJ4taIbalyT+nkZ113hzb7WRYlQZ6Y5uABeh\\\\\\\/1JNofQFH3j4Lvym15WpsxohoeQsypstK2YS7BgKcMCuUEt1zr5zdDDkbm3oAh3OYNChbZwRIW2m1zdHUMYzjRwXnTgcsUBNdvDn0cjwCjmp4L7\\\\\\\/DZzX\\\\\\\/qb7hpdIQKbO5U23GMHLpTQAW5n2C1rooJgzSviQZvU7QWdjusjpPGv9K3hNTiT+3gcMPjsDZxbX1\\\\\\\/GKkqt9LhzGAkbZgA4PTTW8XmRictNwo\\\\\\\/yAm+zez5sPzlvsOg2ZB4cfpjycyRCSkUDswVyvbY5vCDJgm2HZ3ASyRYHQhpjcV08Sf70ozVpm2EYk0n3ldo4R36hz7wfO72W9qjmwFx2tAjQT1aw1t\\\\\\\/9oA2KpLadNXICs18iWth1W8MWg6EIrrR\\\\\\\/H8xvy4XtYlEiTmSTZEUjHaMZd+Jn8a62wkRDy8uy3YfXO4wU1FAHLVDWCXrPkVGGb4CdBfHJf8lOixNLoYgbdZY5hOFbgVuZ2Ktu+U31Z395KaRFVY\\\\\\\/rbwiyOPQs6v0B7e0ry1NlYkpbSQEmrk8qAoi7oOq2kSJ0tqK+ZFwKgX6pek1egqbNLZMHv+4URSBdMT6\\\\\\\/reLDzpG\\\\\\\/nZg47Ah1WWHL6sV2OHie+ws7HWmD1knny3PQlMy25qafgpg2LWmoayw+9MI1MaiVls6Y56uDZ+Q3ql2DLEOA2+dzZarMz+xCrKJ25USwE17l21z\\\\\\\/5xNydH8U1KC\\\\\\\/UgYMoVOuQPoeA9qjzNx8CeJ0uXCu3Wiqx4ampNNkulFW5ZqzPaTLnDDrps4ogPj+ZR7mZUneDQtvy5qI3jurRtypSPhnNeLz6Q7r8mj5k2WEDqHThjUqysxamqlU+6EbSDZ4wr7Ra7ASdMFrdVdo2xGoY1KRTkkswSiRVPlpgQ422czBMK2QWlgki5kqLhexJItNuwHDgqqR2zTDUtwy8UdiL\\\\\\\/Uwsy0CP1+iQ6MjoZlWnijsGvI2lDTBo3RF8E4fZ4pTboJ0y05KUWLYsXN+ZBqXmfbVq5BsPS5kMHbWh+uSv7BlfiJXmqv\\\\\\\/NvN70AAGI9EhlJFh1Cwx0t31Fzq2GQJ11vjmPf6VmQNIRHQa4gFWvnSt7Sew3OAWSBjVffwf8k3ua5HYg4trDgwg0W7ZP\\\\\\\/sdDPpH49SmM\\\\\\\/etFrdRVYNJ6KhEwgOpC4neYY8Ou8+lWTtgcnfY5tssQpmdozS5THpH0ySZ\\\\\\\/DKpMcf5n2TglUPaRaJiWFHlAVEwBfvx5lItNx4ykGcZGoiLvwy3P+5t9NjUCr6hXUT007TtbOQoHTEAi91BnMTUWVUiZaiPf6CfQwsMQjS08HCF61GroLLD6H4vvvHfOIcbgF7zK3gghvdl+6Y1ub3C9\\\\\\\/AUZHKnYrH9jfxIPYNrlL8L3vrCW2vfcf+D5S2sMdpMm98+3ygXnyCgs2s68Y0XWzX4+JwWPnI+pc2fLwTPzGJcEtIETtsGT8yTcAI\\\\\\\/CUQ2AEw757AoB\\\\\\\/NasnddMpWwCrvieby4Fm5YRuy7SzjL6Q6U+S6WOhmvwd\\\\\\\/rfyKfj3xpxGMRKLNVJtstlR7gebYWK3AemW84mYpt2\\\\\\\/Z9PP2F9Gq+KMEH552TssV37684KAkRq+OlpPBPDSS\\\\\\\/kwbwVNER\\\\\\\/\\\\\\\/6aKCIq8vNIsF54RcAdMzILh8TgCr4gK7EEmAEPD7F0d72ygbpl47POGiQQFL6I2iBJVAI7K6OwAXgBc8ycQBRkdD0po4Rix+mxutNZPICZBSO\\\\\\\/3W+4Xh0EsrWEcowcMbms0ia4ys9Xc9\\\\\\\/5ORA4mu4MlYnNKP6LORmFzS0ONRu+rvjyV5+U4fKyKVnBqGQvbiJ88cISSYqdF2enp8zfh+BYX4M1UzqVUedVnszaqy9Gg7pubW4Yfj9yfk2eqyisWwzuXw6gaqZPabfqbM5B2uv2KmoBW3f0Eox7Konxerp\\\\\\\/WTXRNwjMp46hRs0nP8\\\\\\\/\\\\\\\/ZuivIgqH\\\\\\\/yojqAWneBbcUiID4DSxHrzXq4rdTuRladSeKOJKKKzr7YqtrsSGL5ydjCNAg+BLrJ1jc8mWFU6nRDurGE0kVENeLgU0CFyxc6KOJHmzPytEeGTFsknrwM48\\\\\\\/baPEHC+gUtoF2EpWRA3zM9crm+G0we5GHHNsfLB3D8TefXhlmiVziHjB3vs0MFIO4B4PMzuAEkG+hdatB2tDngqeZmjZJvaQOZTQGtchv9+FUBoeP0CYxPNhcnT\\\\\\\/Xp0QlsxUJzi6t\\\\\\\/dkFhnH6JjpFaOUcivpeIDcG\\\\\\\/av4QXlTBm050bzc1A9dAvYBOeuSj7KDI2kCADkGtfLjjHz3TpgoVhYQY1pnbeUj846BUyIU6PFCD5k1kECzgzkceMeQ9qNbu8BLchj3X8ZV7KT30pO+Uy9wqEF86JMs7HyfJaWXWStgGz0C1QpaHdxGQmCWS4BP3npxSziJgTFQ9lY6hNTZMyb46rchJqROkgPQ2xNuvW0p9bBe6bnzMqXI7ExmJwnI6A7P0cyXPMELorSTYxpd3CAyiu5hQ9ux1W7p+Iit77dgd\\\\\\\/Nl6UYyfJMvCmBgufirT2yQzDW1ThCfSNUUR0cvBRWe8eSj5bu1jW81noMRbBi23iehtFhrTP5d4SrMyWQRfRZFYsdf7JwE\\\\\\\/3f1gkrbmzTBLVp21Uxss0BkCvrdJRQrwpZqndSErNYTmftRW9EfK65FQUVYbHgEvvfZZJX\\\\\\\/p8GORH2z+N3MVsB6PHlXraf19Xvjfh4xr1KSo66KwMthC6pKcpHKd\\\\\\\/QWAGX+BlQLv4gfYdibJd33reAvxtrs4jsLvEj+bPRT+ECFezuxF7Pi+aaWZ7\\\\\\\/i5f3A3sQmau1rIsZXQUGSPqDygIxb+qgrKtuCtEKoJwdP9EYMX6aPuYzrPtKXrTZR1k1IIn0aaRjJkjQzpqliXpKstocZDQGuPfWEuIAAAiDEVCjVZTrItqCCaoaBh5CNXo8Jc1SE7Ims+nCwj4iODX94g\\\\\\\/rEIC6h4Jx1AdapSTJe2i6uwmko\\\\\\\/vwYFB3korU9JpAMtY9LpZO1VNWzPVSlFoRiKWzHMZOH7WYY4ix8d7WGCba2uOG7G\\\\\\\/0eElgyfpfkWe\\\\\\\/Ub95aPQlrO5MZeLCeGk6hKw5ryNEROJObhey+mLSp6eV7DXRMlwit4PyqKfnZzysoHOXkCBkmH\\\\\\\/Ry4m9IlNb+Y2hq9nD2QMRBAKtIrPH8lRA5Xw5JFgd63r9SSF7MX00mcVD38VngwvNSlOkNyxfkWfwD57T\\\\\\\/1FrSd+eXSZg0hDr6aFRs2rOEBJ9UlbDProJyyEYeaVtcRQwblYstNAwtNd+7Ah49J7udiwGQZqEHCD8dDFOa+bpQAAwaas8yO\\\\\\\/q59xtzF87VI8lzTrVceyQZKLBwJCICqQG3O4SOLELXPugErXshfNr2GK8UIgIcVKGH1K\\\\\\\/KFsc4kkiFz2yzoejQZKStdUKEanUx8J5N2D0MQnR4ShRNXnd8EC+3zl\\\\\\\/dA\\\\\\\/JNwDBIUQB11hV0ompAp7Po+JbBqPhJAnewdh5s9G4SJMwupDhbUPK4bQoRp+b3FFf9x\\\\\\\/\\\\\\\/tkBs0ZIWS7kKniRZlRGrW+CzUV19EQrZQ7kusrshinks1zIPqBPcgQjGfKFhhe9HDo4TaQ9GuUqsOd7LPl8Iw4a9z6lBakt0O6R\\\\\\\/tibCATO4Tex0Rxmq2ePe6jge6RJFqafvEwefMBXvt+M0IsOe1nl\\\\\\\/a4TIEqPEgR\\\\\\\/ki2SjXYsLDG1P6WLNMB92YSD4XY4h3pdojJx9\\\\\\\/0+2jl0eWIUrQUddIKWXUbOwvrLNkOgfepHJ8IICYwad8H\\\\\\\/GMprEdzhiQmnWnty0HAsMxJEx59cQHB0\\\\\\\/Wt9ckDXhfmd5SUssTEXwCAbQ6djRdB7tMd2d1W3hMonEXm7BE812yANx5SdbmHiY6qENIvZOAg0nW0uPGn245HuZ\\\\\\\/27dhDq0HXZF+dIHfrijCKtLB1i1JRFvOP2JEs5Y0pbve3cA9Rc3N9gJvfOrgo2wYcmHPABAXjxAvJxKPoQKtwkwkPCWp32L2y5oJzCIT6G1HmTWTELpEp88ViJgX4qxJsQcjdgFsmQ6LIo3vqqvegfsTvOBeId1FKO58w3pAotjaMEFtVAMn1ewpQnKvup2F1Gkty4llBEHPb7Ta8S\\\\\\\/GAwse0iqOY5KmjKWUa9gVymRFHI0TNk7ZWcA4BtjbI1j0vMTF0sYo\\\\\\\/vgCoV3M13jS2Ym2103S2FI84VPwOmRO7Eaw2abZcodgGQBIoW+m0E6+UNdPt+v4oWyci6HANZsGCyS0Q8nbqzntpftK0XmZwJ0v2bjOLU6RTjMvcNd+ImOhOQ\\\\\\\/vTh0tVym4GJInNoWPZNEmi1W2x1WrLGME5hWYLTLUtvWjMCpOKTCkl6Kk3F3f8YCGFF67go5UJl52m72kv8NgpNFUObKPX3KDfN9vMokVI9NwZw81Ai95PrDarVPXM3HKtrBnS4b78dpyZVEELTDOcGHaf+UJ16\\\\\\\/90fA62DMhjyZDOe\\\\\\\/z42H0gc2zmH201ut\\\\\\\/9uqGvTourSY+brFmG5QghbgOOh0IzisHbnNc4bt3BDNLfYBYIJvsi6\\\\\\\/puYqA0rviWGC32Thg7XjTcREP0yVTqLh5qKkaGQQ+gEv0b2lSkmkIUdZy\\\\\\\/ctlWuLxzROnRa+BZez1RQP7r1Hc1t+IS7H4l9nUzPAPDjiOPgn7wBFoVjaoD\\\\\\\/Si56gvvtMIL60B56V9KeALvhiVOYa4bLJJG2lLivPw6s2kwwzJqBXQi6s1ydIZ\\\\\\\/ZzOFKgd76iaUJ9XiLdhYA4rE+l7Lza2CZkVhFu3SQjennvIejfeFP9+KfJmuydMz4bn9CPM+GDW6rk\\\\\\\/TOlJPuhGQSKj772kOeLx2hK3PCB942vhTkTA19JJGhQsAVqE9zqDp2DDPAYTDT\\\\\\\/F9I7H43VB6ush\\\\\\\/80V1lLciGxohGOkb8zBcJV5cgg2UWxdbzdSFR\\\\\\\/kNG6E1uwTbcjUuhvSvvVBsK88an9agHYRXcSwuTgNsyjFsWpQemMLv3QDFkmkG44xSj9cVdXKfDuVcFln0Zg\\\\\\\/2m9tcRZP45jJYUfkTeK3Uep9tK1FVG5WdcapHoGWrYpQxQgAMD\\\\\\\/h\\\\\\\/B0WvZpw3HFB00upPmWp2ZhpyC9UXNVaWQJ50WjwZX0GJVIG4cTGxJpf8hpSeFRac5ci0pHC7c91B5xj9g4ADck0JIwqQ0xxPoKNR9MXD0GwUfL9r2TbLNFU3yYiblcwtRlDBsoSMaZH1KQCqozsoaRxpkA8QXgl3dBVbFU8NXVuny\\\\\\\/0shqz9Uka2u764ZJQq5qzCuayFX97JrmVMmLDKZ1Q77qGAGem6HHpOUKifWYhkf018KCBZT+hZ4ZpfnEvMSbS6A6yMr2Lt7BDwqexEfjSOHYWcCNWCzvyLhnen6XAdyzi0jfYs3fch95pFBOjvCXvbVC2go9QrVWQqYwEJNYJ51s113\\\\\\\/VCt7Fl5HJOaxLL9myJb\\\\\\\/4y++nrfS+B5ZL9o4eG2uuHfiGkSrRt9AFmyYFm0WtYBVUvIS\\\\\\\/TNf\\\\\\\/+FrDA\\\\\\\/WVJrN+30vprxMYYtt6CQ2t\\\\\\\/jgMRRGOUXkqZa6n5hkTmznrvyO5KTnAQaMfnj5bNvcHMT4nAysU0n8qf3yLiGKdiaVWYzHNuX8Wt7NR6iQlZp0RYSwuA0HdcHMBCRz9BCjap0KvOKqsY1hJKTY79f5BzkOHl43hhztq6yhHCl0lYLO17+hG4VNpo0b5+nmpkgtBVnG0XWX7AWs2f0Uaich+JoO87Pa+LjHuN0ydZGotu7\\\\\\\/fB2hSEYzCZADXxakwNuHqXVWh+YyNfI9+\\\\\\\/VGCoxySbEY7jTMznogxyh0IGh0jfjvre7Pnz+6dsbcCaNbs9HydiQUg5ChmgsHKsUSR6IUr94nn9xxkAYmJqq67VhGloJtcwJ9YlXSoOmVH\\\\\\\/uyUlucbnm2jdHqBONGAQbZmSKrpYMBllzhAGDXQ088TTiUpRnZekc2JrSVIguTMGIXOxFiujFqbwh+oGl+2OLToylwW\\\\\\\/tgGbdzK56DcSG53fToO8wKkGHOfqrUo\\\\\\\/9ojps9KQN9MWu5E7rYUUNhYrdHaSU6MpASjKdeuIN6XQ13FUFoUvq\\\\\\\/gpfs2xBule7beAUFl4O6IioNGorkMwAk1Hi9frvF\\\\\\\/ryKjc3XytAB6jQpgyKXBnrSae\\\\\\\/nKgUlbWQBSqPGVzH91lc+bj9YIIE8rOULb5ZQAMUln8uG0SQBlKvI41rnwLQS\\\\\\\/T4\\\\\\\/\\\\\\\/CrM5oGpuvHWgfNXQPTLVTQ5+BQWMOIG+HZeDF5mZOzSA9SaqEUipJaD5J50mOfPdXCccNrQeswOzsM1SEmX2dQzvBSoWeFGuL+Ps+vttVNJ9weo4R\\\\\\\/mOj1aelgT5Cq15BXRa2RDPm1QWPvNHxACIphPnKJ3HJvma1zNYHglmvTX3ghYapPj6cXp3qTXISbfqUjk9x\\\\\\\/fb5EY7f2SLJ2M6\\\\\\\/Jb\\\\\\\/xW48lpZUz4Zo\\\\\\\/y75PZjnXZE40WYpeGUhUefXCKj0035xhQGL\\\\\\\/1tf7Jts+6hU49BVB5\\\\\\\/CUTGYF7AcAHu80JXRZ\\\\\\\/imrf3nOAd8uTeyvBCSH2xxQJTfEugaxpQEvPvq\\\\\\\/+v7kXvneFFmVPJe7AZbBWbDwwWurzp7Z\\\\\\\/k35OIZrHhlUH1sxwwNMK7JdOGffh5lnCq29Cu0YuUOa5R6ydpE4XBQihw\\\\\\\/ERCUDJSbwd7u6MsXYBzi9kKSZD4sBh\\\\\\\/63ep+Cvbn6OvFgBLbbETXT3tY2SKu7Vq0rDMn2rhdk5BTMxuq6z\\\\\\\/Xy\\\\\\\/EltwowlJ4+plFVLp6aRQ6rsC3035yOP8E3Dh0xvoqZ+L9MmhTml11QtEDkdmASBxbqzqhYRbWQnzU3WJD00idSOoKDjzJ5R\\\\\\\/gk42o4jpW2CQdfT1dA9vSt00z3ctDhrcZehXHtwurqSBjIaYotpquC\\\\\\\/Mhbpg1ZhJZB5W5KuxbED9nR4E6DtrTEpQTneyKI9Jr0l6tIKjQ2Tb2NNRiQovCsGSvnZA85nahiwtpc0S8IVXnRatZa+s2VLW9PN\\\\\\\/bjoeaQoX2f6UFXg1aXxgLl7muB22cWYCh7\\\\\\\/LFdaP0E1OwlkoNJ84kWhd0ZnEc6YCdb8ByYunxM6VgIaBz2v3UvqdtnQjERTL0DaWRKXLK3okknLhQ7lXQ23xAjx7FKdK9PPkGRnt1xuFAN\\\\\\\/bvR5NSJ4TDoHUgANOl8xuzbIkeUS\\\\\\\/OXGv\\\\\\\/E2YjtvfXBEsWazixTn1u8L4aBTgJshe+D7ALoV2IFqAnVcTlha3wbDIfSzK997ne7g+s3ZeEOIfXR6Ebl5qLHT9fDIYZ0+F15BTKZClq52NAb0uomLTcZYLpN14jusNEbirsfnGSzMzSKgmENEKDzvYF8jM6FHwYh1p3q5OnIfkqB4nBQ6a+RXVvHgaNRXnfLKMUoR1u3S44bAfUwJoIUnad7BjZ4cRieA0tpgm4zRvM8E88PuHDxJ1NNt2jjO0nbaEzQv5ConfUOjXo9\\\\\\\/g77ddKv+uCalOEqiiSqWVYd666W8V1KpDH8+z7mAkX0qhYChMgcA2lRDQhpuhsb0Of3U0rW9xWyMhERp4k+RS67eIcGPwszjlYFhJv+7I1qprrpgpO\\\\\\\/Pendne4exN59oMDEaQe16M6uodezzAHCVySm1957\\\\\\\/VIzWSLwKB+r3xDhq4voW060ttH73SwyUHZotIZCUBnb39Ju3h1cBXDDGIY2qDzbZsIoA5NOT2ydLSd4dnoBuC1pVk2jk7WJpXMD8xSLjFeM5tLQ\\\\\\\/I\\\\\\\/gzKMhUncF50giQJbaz1S6P1+9kbSUdcYlJQwjDiiUOUvqZ+1yB1wsVRNWmjBcWK4UWvk2a0xp5qjr5btSm+0M1UFWuPdBcegnd7graQJCR+ngQiUSqhH5kBFR2atviSgmMkVt0TfOC2AncvakC+jZAQsKlrBgfLKbrlkMHM1GjnaoyD\\\\\\\/p+VjukDq6z8fejLn8OTA2gr0bre7jHV5qkpZd34\\\\\\\/plNOct9M9uScRJosQ4FfBSLQXbofpqqs248na1KVAgWjcgpHuHaU93xhLMKrc3S9q8mHPFeuNxeB8+N3FjXEjc67n5Pz9hK0zNUYqLuOVC8mielWyN66ckUL1SOH\\\\\\\/+TYOEg6o\\\\\\\/BcgNIqZMbYZ5hBptCmh32TcXkr\\\\\\\/6g6vrPnQkHyrwXOYTYxOVnuugDB1ToYaTFU3wrSLjN6GFL6S68vOWHTBZCnQw0iDv8JJ\\\\\\\/Oq++zWJXT1SRNzxaP0Nn4Krg\\\\\\\/AjLn95qE2SfOX99Eo\\\\\\\/Ea4uX18HdZJIzq\\\\\\\/gQicXFc4AV1AnYTYCr1Q\\\\\\\/xNeCWFUpYEzA4FiocAxTVxe+9DPgP+KLrtFGSUzi20eGvLtnxecB3clTsgSZyV6dDVVArMtQJCjDWCIc+pPWF2JB7QzNLFIBoOvEicimnwhHj13tgfB+CXwPX3JvyvvY+zt1OrSHmvHTVuATYYw+OFWLA5GdR3vr\\\\\\\/IJ5vnzylVomOfyLGhpLwwsZKIUMKnNVllaK1F8mcCwjdzw2ssTow7+A8GoU3QZNRPm5MAJgH0z9LORHS7PYtofgQ3cR\\\\\\\/fYwHEk0blc2o9+O7Fs8sAqr7\\\\\\\/37U09\\\\\\\/Zg+35A5TfvtbqsN8gipHK9ztlTtvluKWF8V8M7uhP0zHc\\\\\\\/Wt1XV4Lm9O4lbtm\\\\\\\/drS4719oVpVtKCHsFhhe8j\\\\\\\/gZ67Uzq5q\\\\\\\/j0eAkQLaea3VrHtaoqxzd31uwkT\\\\\\\/TRg5YpK+t6G6VEey1Cg3RJvh9PWm9wuvmk4zmW4iRsSyeQSNRlc5c9QQxiNfNJaKtsXNiy0cQyhIoQyryodW9PANn6TzYk04yL8l8IzFvTEcQwCZxRWPQq2k7A75wEIDC5zikKsalmgAgMLwdM\\\\\\\/Fdmy3JXeRyv5U11L2WGOjO8v8F5+vygZ414cKalFThZPHeVojBen25oWjwiU+GIVjp7xmIXWfiZCO7ExUp6O5\\\\\\\/IEYCP+Cak8a5DYidGrZOwHzrbw6au8v+EXA5KdBIsMeuzzykCe2YX9hN3V6KrqCpnI2dWVhxXv1lxrINBTHsYySdVZ\\\\\\\/9XDsunydKchniK8c0f1GEzRz9eNp6GtVageEYRjNMHIWoW\\\\\\\/Qm32pB96yr4x\\\\\\\/RpHTbNdLRdBSQnZNuVn8V2Ixw+2+6Fst4SfB1HkjshLBxlcMxogfUcBjHvGIWhxuCq2psxsmJ8fu5F1TQGfgbNUxyY5gTw+TXA5jbH4vy0jmSsbiJDvxoK3kaAbacti4IMZgBu9nex5xy7stHjtlgz4tXJRDMlwp5C83xyHgq2H+gPEzfVVoRxdS5qhDIma2Byc1xl6ihCsXxfET1L64NHdxAbs\\\\\\\/eraF\\\\\\\/YXfXgQ6M7JaQ\\\\\\\/e+lYQXfZWGwyo7LKOX7u\\\\\\\/h\\\\\\\/72s3uJ9YZCnvNylllXtSdjFB9agPbRKn5Huq\\\\\\\/sQek83OmSg9ZJ3NnC9Yiqu6WYQRCx4ZRZTQOHEVlxHdt23JwxyTgVT9QiFfFI+1xo9+vaqcWD7ARs964+dWKdDJgiYkguYmejGL2+3vPZIFZnL\\\\\\\/wD6Uh+Fk66Ruyeqv4FFBOvORJS3zN+YQNP+e78RoheVNoWsgNLHGZxq5sX3XCTBxZcUHL\\\\\\\/EZbROZR5EfGRP9sF6z5RFrDVbaG6p6GcqPsFYsvJPdof7KgBGzrVCV6m54YUCNgrkqxf36eGZSK1\\\\\\\/p7P6tA5veQJPevePgLkXV1K7SAQxDfrfs9TsxmcULX++bRZCKgKQfB8UHLLjmcxdjT1pu5tNtNIdef2TZqIjxU6sLJPRmh0KTvdtcX0kkCfOaSB8TxMqwNXMO6GEW1hgoyvRl3Cf8wdgh8i1UZ1v8LGDezKtd5bhHBW3bW5ri0Rb\\\\\\\/wj9EAzWN4sTZYxpbNB+89nNycivr6oSyz3\\\\\\\/No3ezu7gZzAacbdkoAkHkIU3QLL1S5xSD14QIuXvUB5b7BBQ3pG25KJ4xP6AQeXruanhmLDOQgwus\\\\\\\/4hM33MXVyBPbUECI1hNo0hG0p67EWJyN2dseCQqi9Yg5zOD00uw8LQZcB4A\\\\\\\/fasG1Xm5slemHQcT02AogjRjGHllkhy+3v0KVUaNjEF30VVYWtLcWv7Ft3XvnlL15kJzQxf5Nud5t85c2XrTSDySspIen2OGq7IJ+ilE\\\\\\\/UweLcztaPjBHqPMRaOWum383Z8ZyAXko9Kskak7Ktfof86Mt5WP7HUYgtVcBwQXxic\\\\\\\/bZxX3UduxYNSE9UgWbjYcROgo6LbeJDzUC0otfTE2Dkn0kiC4XK8nsXmPPDvcLdh86PTsvyzg62KQFpPqapTALxccXuhSxwHOARD2a8Ka4h4jIjS5VbOTeevwB4bFl7Ka9kqjL9esKP+utwt0nEB2e15nk4E+qbcV2f+Ks8U5N7NVB39Q4nnIWUQPsIB\\\\\\\/QCmYKttas4SBg3CyMP8Il3IthEhezYgcltctC2NCkXbbysWuz95+P25pMpkUUedgdBWBR+CkaIc+75fIxJrI3pLufqiTNFbwrGEzjA3vMHLRmn1DFFI3EKxwg85STizEbpsX1HeI8OvOK9ULyUs+Ip29LVJWcBJcl1doyo8rh\\\\\\\/90SqXX5\\\\\\\/qq3xToXz\\\\\\\/7dZwfnfFI\\\\\\\/3EclA3z\\\\\\\/G4spnAbpYujxo0m0erIrNRLlepOati1RwdhStZUW41sXxIt\\\\\\\/1wVDA5v2V2F+t0KdID8\\\\\\\/DYP53OWFbqJGSUIuVQFDNnbSufCDPl\\\\\\\/rfHpJ\\\\\\\/shCiOT1SCP6oYo\\\\\\\/9FvQGld196Kisjs1Z4n2MTHWIAoBjg3HGv3OjZL2O0LhkeLT8jHBP5RjENUZodcn5\\\\\\\/+qk+C15u\\\\\\\/2RVMDg5ZwZnL7RNP7Da2cEtGE96MhHhLwQ2tW3yPxAkiW\\\\\\\/H5yG9HvSgtTVmKJXz7sTNoFQYLXznZrfY5Vr34jXA2cwDpW7dOaWGRekMKGrDmj43SjE9RNsr3+yWHE6fzv1dETxQi9jRnlzQJV\\\\\\\/b5HGSWoO4SqXtLNQuUmsbMcWUAYB3+dHB90ftbCGO\\\\\\\/dFcsUrth5wxnXSwxnvCTb5m6lgmmlqqxmMbrcS9qB49XLO89UwChxWftM2t807SHUZj\\\\\\\/dLSCQwuYW0t5xearLBRE1NWB+4\\\\\\\/wsMqatEz3y5KclIAN0lRYTOHSpjUa+nuy7ViAXyvu248aAUUpDm1NI+CZLzJXHAHuDqsWv6ka3aKnqSeDGpcn1p1\\\\\\\/mkDNwBQV6k\\\\\\\/fJ2H6s5MgEpcHDuxUScHgFJg\\\\\\\/uDzq3L0msRvManLlV6QWWHoDAXccn9GrPQayw2tSYsUl8OCd7rP4fWXJE9VhwV+Eb+ydasFurFi7vtP66wOAf548V7WZEWIBIKj4TyU6W0x+PGOE5DGBOLieMV39arV3llH6xhZyAGYCXpRcw7D1umKtBw50jfnjciO\\\\\\\/u5LMXvDV883SPylYnTh\\\\\\\/mxzXqIQLpjvBvd2o7\\\\\\\/3kn9C6k1zMLz4xftuL215XMGy2\\\\\\\/0ZgvnkCdm7QHesfnQotQHvZuVPS\\\\\\\/UijCzUy0z2N7ceTVphTBOFbcD\\\\\\\/qmZbxrS7GGyKwXdt7cwJRzR55lx19OmtXRIA7qneqmgGvU0XBw3UShKNSO0O+SpCTH+8VgIOJ9\\\\\\\/5i3uE8ITvrI6DQwOX2e5OMFN\\\\\\\/hJjCdrRSeNRNl4peNyv9nQOAOd1086dTzhqHBYF\\\\\\\/KsIO\\\\\\\/hY3dVTXtpc20WF+ZBJCo3jjNhO4GTLe4pcvtlAjFT4PdAOxgmccmFBkpk9vFK2Qx7F95F1PycMU+II50tqJcT79QZr\\\\\\\/hTMFUqRX9FOmvXW5HZX7ZUejiHh\\\\\\\/auB2mDxz9CzO3VrRJpWDe3CJPa88BfAQJKe1ZDGJRmjunpggA4q35FgKP8GXbxoPyeyOREdKavDAY9hA8DcaaSs9kSAaUh06VNPUnbDu\\\\\\\/9c4B+iSPWEYAWT8hMaoVuIgeEf3VM1prORnSE91snsIXDOhERk5M\\\\\\\/XCSHjshY85TQJP7zJdrG8WwfbffMhu7zoIVTuCPu5tZH40SaTU4FtPmbJGRPtIV8QDL522KqA1CxnQV2KvWjC\\\\\\\/pMs0ELoRTL\\\\\\\/lIlNMQKubS0Zp5AsOXRp2Btpgu8h8yGC2LBCcIT7t576iGD4ZAcRstqA1UZ65OziH0VD7euJAJoWJFEWAhdIydXgrrwaCS2E1FCEXuyeRPGHff+KJj5ueQTPkp+OJf2rMzwRKbV3WYdJaOnoPR5\\\\\\\/A23IYUDFh2WoOPyNxl3GJm1\\\\\\\/qJZR3GTsDVM93TGUOEoH4nnaEbAbgf3xER9iDgHrKxrDCp4W38jIXjEuDVf+7s9Jx5i8db2oV9Y+L\\\\\\\/OCyH8+W3a3LxwUi7483fUwepoTYATLxC9xEiH0koIuPPvMP09A7k61sLjy8ck9GA4CZW7Rqa6tzVgmWMoN1O2Vlkvpf62genUjNzEovkAD\\\\\\\/XwjijyHACGNl4Bdpxq6QGheRB0Gngp00tLc9vv9Qzjwd3IqLBlneo1LD4qMZLqbJuwTUPiW7f4pRR71Teb5rT\\\\\\\/gPINiTTTdu4MLXtgKR5Br9rm3XK\\\\\\\/D9N2JpdUOqm2o8iNAaeUHogSneUNzelio5fYYXL\\\\\\\/uHMj\\\\\\\/hUqtdQQxEKrMgmbIeFg3Ehv+5ZDY\\\\\\\/u367TVt\\\\\\\/YFzXUyD0vlAnUoEoxvgwZqLLb+W+WQJEv+h4XPF3Jxb0DK8UdkpNzKc+jy6X6sNBHRDeeuQui411Qy8iBNCD3\\\\\\\/oOqhzs8k3lzN0LWHJrgT6JohzcWJmUfm8+XWYbt54xETg18Wyiq2PCaAQoxuL6iCntAIFnv6WQh6HOQV3ut6jy5ispsb0BflGimgKTrgkiaMUCiwI3MzUd\\\\\\\/WWyW6Foj4sdQirjQxUAS3fajtXK7ezT2b4LcA2dHFO4xZg2LTXvmNhRHPl7tzB\\\\\\\/rXXZMINgOUlJ2qdh0NttA6kLlYnmubw62J0dCYYxgt3foaUAHnUqZP2\\\\\\\/thlAG4HcFcth597gvtelKOkuOMpd1EWvpg4CakTrQ2bsTn\\\\\\\/LZERVWlLRaaHZzIUa1bmsfLL\\\\\\\/V17Hgbh0GZXBTec4mOIVVzkjIVt2SqINSEKyD4Y4i472xWJKd8kNbzGadwhBoG4BMcQsMk\\\\\\\/OsnjT+QSTIGM3JY3XLRi7MTgzEiESIq72l1wqinRvmWTRi3pL2axBAB5ixx+6OjDrmFG89thaDs9621750ZCuDDmJcz46SitJ0ncy2GTxFrhP3m\\\\\\\/qL1NJzBG4T6f\\\\\\\/znsyScRWBKqx5p+X3eNRBYJxTlHbXaFJQy39127bbkhfNasMmChBgTsb7MsLOe22y4dINvL30sYGKibtBvXol1LI35uxkRPrsjz2ogAWzf\\\\\\\/FyR8p+4aJd4gujCISTli1dBJDtZ30h0zOYjtjm\\\\\\\/aeaLPwpopghsLvdWDSVUIoc4lv70y8QAG4mJLLYqncUwh6Lr8Xa+cxVJFd\\\\\\\/JYpBpuOIAq3SdXB5NscqWgdtQb10obe\\\\\\\/7fjiyP6H3oUxw534PMuEqv4wP1918Ua7N5lW1TBkjP+b+hRlYO+t9YTKU+XcihbAG+mxmB835iNrg4qEMB2qbQZ49i5uhrrso9RvwpDHGkN+bwOUonQ9DJzZwArVm2T5K40qDIS6c9Ya3awt0sp+BDr3jisYbrIDjapoHARtygBjdYYG2Q38Wf3MSX73AuYWdOxss4g2tEz1n791yUzqa0GDX4dYfT9INKC9Sq5R1sgI3AMn1Vz9B+21aZwk+VSqnARbWEiz61JgW175uBsjJX7ppiSt7OoFm8S2O4gj6ChSkDQ+XEGz2YKIA8fWBCdACp+H1+04DQ512f07cYs2PpllXpotLCpwAAMZVK67mJT0Uc8FEdcvHYSOKDsiwZ35VYCzU684TE\\\\\\\/yFmm6snzIZKixRz8ieLk7xjZHmt4qFV\\\\\\\/sYTREJ6X6lc6+9TtYJsGl7Nfe4M9zwuOlOLN0qQmt9PLMqActou78agGJqxWG+GavszajO7HfOYEdd2JcF5bxWiB82nfA9KURH8VV33A0\\\\\\\/T0dbgS1GXzLWZjnIITpGFUlynSx2g9CxnSaF98QMdnUxzxPUGE0kWcRM2JKZU7OzDfJb5BURlKCHyQNTADewsntUJoM2f8BQjxwY1hrz7+g\\\\\\\/Scp+Qk5PR0RL4b8OV3\\\\\\\/7Z3cLWgLvXoNl5yag2fVtxij7Kh02DyKxURyT7WHfajWhpFQ7QLAVHvCu2ywZYOGL6O8WOhbuJ30ORrMKkcCR629J8zD0siYewUu+V\\\\\\\/HdbO0+TIdwqsX5twRdC1ydQVnx502KKomEwT3RRPVm\\\\\\\/io8CcJpltZLRTg8x2Fk3EgOcXalD2AVM+ecV1cLv+12JoYS5X4Nr+psHj2DWa3dO3BVVQmINjaDESOd+eQAYi7RFRXhnoUvRpF1aXlUPpchdaOAbHmerIjUfnuLYu3hfEMatCRSmlIoEu614mUv5i+s4PkADvP2By\\\\\\\/M9MoQDsKsOML95BowIMUoAFgItWFm5DJX48bJKqGWsqherZyaGmbE+7cL6TRhHnJhMM6Ajz\\\\\\\/E1QIZYmZBuXD4C\\\\\\\/Nno6Jdx7y\\\\\\\/giwrn6BSbo8TaxWWbH9MlfU17YfiWBNEkyCnwWtLlRRZ5HtkGiWJFSr95AEtxnwvZIlcNm8hi3xF9GSA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Jx1p2IlHLiwuRNZEAVkr+A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225771af0f895984146eadd68427ef929571727cfe692045168a0173e0e3dc8bc6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LTvJL0nukNsKM2jo85q2O\\\\\\\/6kXj8v3irKZbMNBB4pojlaH47arSEdgU5ckrGCvtiMIDB9RuyzB8864xdvySE+wye+nq2kPbvxQy2wmWV9N+cEHXCDeOJa7IfqJ7\\\\\\\/SwdcLDuql0vjvJGfxHLxjq4dCmSfrC2tOtHODmCszLCwE3E5Y\\\\\\\/GFR6GGKXnaLPfmLuLBpTF1uIrWoJ\\\\\\\/tfzHYTqQj5bgzDus8R\\\\\\\/HT5NmiLZpyMSlkFuyBlzUqufnnewL0a++iGTSMqfca\\\\\\\/Xl0ECVpKow6g+XgEzIrrELw8cPXOnZ3IFrKvQ4nYJlu8OCas3TEwrKGVqxbv3JTw9r0kr4aPmB6u15re4kEmcumSxm3hHGmvcvmLItdNZ2vw8loJZfwX+MVR7r61oi4aLntKISyOmTToUVzmIVR1lMBHzlFXnpWx44pcqpPGzU39MfIf\\\\\\\/FWT3tCZzNCXo7aOLHtTTxaRbgvvPRqvOzKJ7\\\\\\\/OtMp4mFUhtIsn9iFnFw9lVDi298WzmL6FAK9Nt8Ytm7HBH1XX8JAYpLeVUfGLGKExt9mg+BAyAtHXcbyH29hH2p\\\\\\\/StI90AiTcpI1gzQvUd3HgtKDxGBGKv2K+1lZmvWclVxGN5tBmcZgRwq6avd2H3rIYQc6PAK8XQbgheiFEFS0f\\\\\\\/ZPR9z2oh0skuFzz+ovRIrFWdSJ51Er1XLzq5p1Y2NsZDOk+YsOsNsHCCJUWmn7P92OUqjUf+0jZE+0fZMR3fehPNsRInG82bPO4vYldDe7fFl+AXe5duUuld46sC0pbkVqIPvKjWyVJ9HynMYsI2+RxdBIAlSnfavlg0ifC8v42E4gI27HPLZMb9am7aU8VDo\\\\\\\/PN0EY0ANOHcvthkrl7pzi8Oa4Vdm5K3JfPmNFmhvTMSM7W2JjToTzF+lFCX40ESt0ogs6YcimLuqzoh4G3m5fTUtq3Qk2pX9sS5E0rD3jxNxGLWbjaONBNmf2AfbGq+H6a+Oqc4XdEXQcqfdMHlZHRHhSVC\\\\\\\/+ZcQg0oHkOVMr6wcKHb1e6Mm8678SXruIUEFOYy0gd9AVl0\\\\\\\/EltL1ldmjIf8JMNTRGXlZVjR1DeglW5beCVfMEoxbdnKCBqFs+KT5HUr4M8I1bQSsD+9mozAUOJ1tGQAwuLh1TgzVuxg8714vF2tTPI+WDQKwV7HK8q37hOlcugiGiEhxWHLcIpubWGoxl1Q8nTRdMqUVsDHBZoapPpyTCrJCveqqukISJiJIdlrJjhXZom5gyKHSx5OtwXbJi3aBSrO39lFGWp9kDdtNADOtSsfXnrybLDMMyDHDQJA\\\\\\\/jad\\\\\\\/tWcXAmHgfUHnw1qZst2EyckbzoUiPSc9+l2zmmLAWJIbmjUag47fqe0uAg0HIecQWNf9ryd4ZtrMAL1Pss264zwmSzsfOaPi3Kb3BPqIru+3WRXz+JMgviMqiwpK6AeAGhGaMDzIyANLzoVqKiasKdvuuMTt\\\\\\\/eU4quThWfWvwYAH1Qph1FvDaoI+jIsmatSzSkg3cdQH6s+VM8Y3lzxe7uCM66FKL3u6YvyYUgJBKWH+1KHD1jJQogfeo86uDpn20nXFgXr2mS9D4eYi+TF1pN4Ll1V02RLY1GPQPZi4ULdsR9kdvZz4whjtEA5uX6HwPEA9R0Ekuktvf5rw3b4NdYEB8cjQ3QV6jkcKINrxmos7hlQ4b5A4FnNL4FavgUYsNEzcrvuKw76WPIwDLQHtT42bskQ88BudHxmHNPpu\\\\\\\/3xgGpg0VD\\\\\\\/lu7WMRxwBDQcj93dfiSAVuksmuxC5ZGWngmU1exYByaItFDDhzKq2jR9DrVa9Y9WJuh2ChkTK7GAJ7qniBPxgSqON3\\\\\\\/lpZJ7UTaH55mUj7jbfd\\\\\\\/8iT+NDUFuWa9L25FgYTu5gwcim\\\\\\\/qRXRRIAfzMjj7CgazxBo07OZPcelsCSxMIy77N69KU+\\\\\\\/xuXxuWXFu7wCZG2n1k4pdgp4rGxrL8m7R4Wf0iRVyrAi5\\\\\\\/gGZylgmOvx8PbVg20Qn9EHD9TqCsKnUcwp2RP\\\\\\\/ow==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Apbeztzc9nevTMOxNbrKQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022367444fa84a4e88af17de6f1e5da74a7b0a8f39c990f15678e2b67e447762be9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460194 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2655" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N59wu6fWnpAwqfcDXP4ps_T9BF32vwGcfCi-bY1DWfE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-babalL0jia9-UaGKfGrlLx8lHX1jmN0DUqMU8RmjdnE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2647" title="Identification of Genes Uniquely Expressed in the Germ-Line Tissues of the Jewel Wasp Nasonia vitripennis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2671" title="The Founder Strains of the Collaborative Cross Express a Complex Combination of Advantageous and Deleterious Traits for Male Reproduction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460194" id="node460194" data-pisa="ggg;5/12/2655" data-pisa-master="ggg;g3.115.020818" data-apath="/ggg/5/12/2655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Architecture of Disease Resistance Genes in Lettuce</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marilena Christopoulou</span>, <span class="highwire-citation-author" data-delta="1">Sebastian Reyes-Chin Wo</span>, <span class="highwire-citation-author" data-delta="2">Alex Kozik</span>, <span class="highwire-citation-author" data-delta="3">Leah K. McHale</span>, <span class="highwire-citation-author" data-delta="4">Maria-Jose Truco</span>, <span class="highwire-citation-author" data-delta="5">Tadeusz Wroblewski</span> and <span class="highwire-citation-author" data-delta="6">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2655-2669;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020818 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460194" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marilena Christopoulou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marilena%2BChristopoulou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christopoulou%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarilena%2BChristopoulou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sebastian Reyes-Chin Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes-Chin%2BWo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes-Chin%2BWo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Leah K. McHale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leah%2BK.%2BMcHale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McHale%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeah%2BK.%2BMcHale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria-Jose Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria-Jose%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria-Jose%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tadeusz Wroblewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tadeusz%2BWroblewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wroblewski%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATadeusz%2BWroblewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/12/2655" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460194" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2655.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460194" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2655.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460194" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/12/2655.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460194" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/12/2655.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460194" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460194/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-68191 first last"><a href="/content/6/3/773" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - March 01, 2016</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genome-wide motif searches identified 1134 genes in the lettuce reference genome of cv. Salinas that are potentially involved in pathogen recognition, of which 385 were predicted to encode nucleotide binding-leucine rich repeat receptor (NLR) proteins. Using a maximum-likelihood approach, we grouped the NLRs into 25 multigene families and 17 singletons. Forty-one percent of these NLR-encoding genes belong to three families, the largest being <em>RGC16</em> with 62 genes in cv. Salinas. The majority of NLR-encoding genes are located in five major resistance clusters (MRCs) on chromosomes 1, 2, 3, 4, and 8 and cosegregate with multiple disease resistance phenotypes. Most MRCs contain primarily members of a single NLR gene family but a few are more complex. MRC2 spans 73 Mb and contains 61 NLRs of six different gene families that cosegregate with nine disease resistance phenotypes. MRC3, which is 25 Mb, contains 22 <em>RGC21</em> genes and colocates with <em>Dm13</em>. A library of 33 transgenic RNA interference tester stocks was generated for functional analysis of NLR-encoding genes that cosegregated with disease resistance phenotypes in each of the MRCs. Members of four NLR-encoding families, <em>RGC1</em>, <em>RGC2</em>, <em>RGC21</em>, and <em>RGC12</em> were shown to be required for 16 disease resistance phenotypes in lettuce. The general composition of MRCs is conserved across different genotypes; however, the specific repertoire of NLR-encoding genes varied particularly of the rapidly evolving Type I genes. These tester stocks are valuable resources for future analyses of additional resistance phenotypes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/reverse-genetics" class="hw-term hw-article-keyword hw-article-keyword-reverse-genetics" rel="nofollow">reverse genetics</a></li><li class="kwd"><a href="/keyword/nb-lrr" class="hw-term hw-article-keyword hw-article-keyword-nb-lrr" rel="nofollow">NB-LRR</a></li><li class="kwd"><a href="/keyword/lettuce-downy-mildew" class="hw-term hw-article-keyword hw-article-keyword-lettuce-downy-mildew" rel="nofollow">lettuce downy mildew</a></li><li class="kwd"><a href="/keyword/gene-silencing" class="hw-term hw-article-keyword hw-article-keyword-gene-silencing" rel="nofollow">gene silencing</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/bremia-lactucae" class="hw-term hw-article-keyword hw-article-keyword-bremia-lactucae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Bremia lactucae</em></a></li></ul><p id="p-4">Plants use multiple cell-surface and cytosolic receptors to perceive and react to pathogen attacks. These receptors are categorized into pattern recognition receptors that recognize conserved microbial signatures known as microbe-associated molecular patterns and nucleotide binding-leucine rich repeat receptor (NLR) proteins that recognize pathogen effector molecules (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jones and Dangl 2006</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ronald and Beutler 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Monaghan and Zipfel 2012</a>). Most pattern recognition receptors are receptor-like kinases (RLKs) or receptor-like proteins (RLPs), both containing an extracellular domain that can bind ligands and a transmembrane domain (TM) but only RLKs have a cytosolic kinase domain (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Monaghan and Zipfel 2012</a>). The majority of the RLKs known to play a role in disease resistance belong to the class of non-arginine-aspartate (non-RD) kinases (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dardick and Ronald 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dardick <em>et al.</em> 2012</a>).</p><p id="p-5">NLRs are cytoplasmic receptors that directly or indirectly recognize effectors and activate a strong defense reaction known as effector-triggered immunity (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jones and Dangl 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bozkurt <em>et al.</em> 2012</a>). They often contain additional domains but usually not a kinase. NLRs typically consist of a variable N terminus, a conserved internal nucleotide binding (NB)-ARC domain, and a highly polymorphic carboxy (C) terminus with multiple leucine-rich repeats (LRRs) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hulbert <em>et al.</em> 2001</a>). NLRs can be classified into two major groups, TNLs and non-TNLs, based on the presence or absence of a TOLL/interleukin-1 receptor (TIR) domain at the N-terminus (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Takken and Goverse 2012</a>). The sequence conservation of the NB domain within and between species is high and allows the classification into TNL <em>vs.</em> non-TNL proteins (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Meyers <em>et al.</em> 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McHale <em>et al.</em> 2006</a>). Some non-TNLs contain a coiled-coil (CC) domain and therefore collectively non-TNLs are referred to as CNLs.</p><p id="p-6">The NLR-encoding genes comprise one of the most polymorphic and abundant gene super-families in plants (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Karasov <em>et al.</em> 2014</a>). The numbers of NLR-encoding genes vary greatly among species, for example, there are only 54 in <em>Carica papaya</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Porter <em>et al.</em> 2009</a>), 150 in <em>Arabidopsis thaliana</em> (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Meyers <em>et al.</em> 2003</a>), 571 in <em>Medicago truncatula</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shao <em>et al.</em> 2014</a>), 465 in soybean (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Shao <em>et al.</em> 2014</a>), 402 in <em>Populus trichocarpa</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kohler <em>et al.</em> 2008</a>), 535 in <em>Oryza sativa</em> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhou <em>et al.</em> 2004</a>), and as many as 1015 in <em>Malus domestica</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arya <em>et al.</em> 2014</a>). The prevalence of the different classes also can vary considerably; TNL families have expanded in eudicots but are absent in grass genomes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meyers <em>et al.</em> 2005</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yue <em>et al.</em> 2012</a>). NLR-encoding genes are clustered in the genome as a result of lineage-specific segmental duplication events followed by local rearrangements (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> 2014</a>). Expansion of individual clusters occurs through tandem duplications resulting from unequal crossing over (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Richly <em>et al.</em> 2002</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Meyers <em>et al.</em> 2003</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Zhou <em>et al.</em> 2004</a>). The complexity of these clusters also is affected by gene loss through natural selection acting on a birth-and-death process (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Michelmore and Meyers 1998</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Leister 2004</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mondragon-Palomino and Gaut 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2011</a>). NLR-encoding genes also exhibit presence/absence variation polymorphisms within or among related species (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kuang <em>et al.</em> 2004</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shen <em>et al.</em> 2006</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Luo <em>et al.</em> 2012</a>). These genetic and evolutionary processes have resulted in complex, variable clusters of NLR genes in the genome that confer resistance to multiple diverse pathogens.</p><p id="p-7">Lettuce (<em>Lactuca sativa</em>) is one of the most valuable vegetable crops in the United States, with a value of approximately 2 billion dollars <em>per annum</em> and is host to a wide range of pathogens and pests. A wealth of genetic and genomic resources and tools are now available that enable genetic and functional studies of disease resistance in lettuce. More than 52 loci have been genetically defined that confer resistance phenotypes to 10 pathogens and one pest, nearly all of which cosegregate with clusters of NLR-encoding genes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hayes <em>et al.</em> 2011</a>; M. J. Truco and R. W. Michelmore, unpublished data). Lettuce downy mildew, caused by the obligate biotroph <em>Bremia lactucae</em> Regel, is the most economically important disease affecting the lettuce production worldwide (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Davis <em>et al.</em> 1997</a>) that has been studied extensively (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Michelmore and Wong 2008</a>). Lettuce downy mildew is one of the most characterized gene-for-gene plant−pathogen interactions; more than 25 genes for resistance to downy mildew (<em>Dm</em>) have been identified, most of which are dominant genes that confer resistance in a gene-for-gene manner (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crute and Johnson 1976</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Farrara <em>et al.</em> 1987</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ilott <em>et al.</em> 1989</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Michelmore <em>et al.</em> 2009</a>). Many of these resistance genes have been introgressed from wild <em>Lactuca</em> species. Studies of sequence diversity between lettuce cultivars showed elevated levels of polymorphism for five major resistance clusters (MRCs) on chromosomes 1, 2, 3, 4, and 8 (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Truco <em>et al.</em> 2013</a>). A draft 2.4-Gb reference sequence assembly is now available for the 2.7-Gb genome of <em>L. sativa</em> cv. Salinas along with an ultradense genetic map that assigns more than 96% of the assembled sequence to 1460 genetic bins ordered along the nine chromosomal linkage groups (<a href="http://lgr.genomecenter.ucdavis.edu">http://lgr.genomecenter.ucdavis.edu</a>; S. Reyes-Chin Wo, A. Kozik, D. Lavelle, and R. W. Michelmore, unpublished data).</p><p id="p-8">The MRC on chromosome 2 is the most studied MRC in lettuce. MRC2 includes at least eight <em>Dm</em> genes (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>), including <em>Dm3</em> the first resistance (<em>R</em>) gene cloned from lettuce (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Shen <em>et al.</em> 2002</a>), as well as resistance to root aphid (<em>Ra</em>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ellis <em>et al.</em> 2002</a>) and a quantitative trait locus (QTL) for resistance to anthracnose (<em>ANT1</em>) (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>). Analysis of deletion mutants during the map-based cloning efforts to clone <em>Dm3</em> identified 24 <em>Resistance Gene Candidate 2</em> (<em>RGC2</em>) family members, most of which mapped within MRC2, spanning a region of at least 3 Mb in cv. Diana (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>). Sequencing of <em>dm3</em> mutants and transgenic complementation established <em>RGC2B</em> as the gene conferring <em>Dm3</em> (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Shen <em>et al.</em> 2002</a>). Genetic analysis revealed that this region was not highly recombinogenic but subject to gene conversions and spontaneous mutations associated with chromosomal deletions (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2001</a>). Analysis of <em>RGC2</em> sequences from 47 accessions of <em>Lactuca</em> spp. indicated that gene conversions, mutation, and recombination events have been involved in the complex evolutionary history of this locus and distinguished two types of <em>RGC2</em> genes: fast-evolving Type I <em>RGC2</em> genes that are the product of frequent sequence exchange, are under diversifying selection, and exhibit low prevalence among different lettuce genotypes and slowly evolving Type II <em>RGC2</em> genes that have maintained allelic/orthologous relationships and are under balancing selection (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kuang <em>et al.</em> 2004</a>). A fragment of the LRR-encoding sequence of <em>RGC2B</em> used as the trigger sequence for RNA interference (RNAi) abrogated the resistance mediated by not only <em>Dm3</em> but also <em>Dm14</em>, <em>Dm16</em>, <em>Dm18</em>, and <em>Ra</em>, suggesting that these linked resistance phenotypes also are encoded by <em>RGC2</em> family members (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wroblewski <em>et al.</em> 2007</a>).</p><p id="p-9">In this paper we use a combination of genetic, whole-genome, and functional genomic approaches to determine the genome-wide distribution, sequence relationships, and phenotypes of NLR-encoding genes in lettuce. Candidate <em>R</em> genes were identified with motif searches and a maximum-likelihood approach was used to group NLR-encoding genes into 42 families. The lettuce reference genome assembly allowed the analysis of the five MRCs in lettuce. MRC2, MRC3, and MRC8 are the focus of this paper. MRC1 and MRC4 have been described in detail previously (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christopoulou <em>et al.</em> 2015</a>). A library of 33 RNAi tester stocks was generated and analyzed for members of 12 <em>RGC</em> families; it was used to identify four families involved in 16 resistance specificities against two pathogens. This set of tester stocks is a valuable resource for future analysis of additional resistance phenotypes.</p><div class="section methods-materials" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and DNA isolation</h3><p id="p-10">All lettuce accessions were obtained from our collection of germplasm at UC Davis. Plants were grown under greenhouse conditions. DNA isolations were performed on seedlings or leaf tissue with a CTAB protocol (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernatzky and Tanksley 1986</a>) with minor modifications.</p></div><div id="sec-3" class="subsection"><h3>Generation and evaluation of transgenic RNAi tester stocks</h3><p id="p-11">Candidate genes were selected for RNAi analysis based on their cosegregation with disease resistance phenotypes and representation of as many diverse gene sequences as possible. This study was initiated before the sequencing and assembly of the lettuce genome. Thus, the majority of candidate genes could not be assigned initially to a particular family due to limited sequence available from the lettuce EST collection; the average length of the cDNA clones was 0.8 kb and the maximum 2 kb. Motif searches of the lettuce ESTs and genomic amplification using degenerate primers were used to generate a library of candidate NLR encoding genes (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>). Only sequences that colocalized with mapped resistance specificities were considered as candidates for silencing. The most divergent genes, based on the available sequenced fragments, were selected for RNAi, including at least one candidate RNAi construct for every cluster of mapped resistance specificities.</p><p id="p-12">Selected EST clones were obtained from the Arizona Genomics Institute (<a href="http://www.genome.arizona.edu/">http://www.genome.arizona.edu/</a>) and used as a template for amplification of RNAi trigger sequences for each gene (supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS1.docx">Table S1</a>). iProof high-fidelity DNA polymerase was used for all DNA amplifications according to manufacturer’s instructions (Bio-Rad Laboratories, Hercules CA). Sequences were validated by Sanger sequencing. One 400- to 500-bp fragment of each candidate gene (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) was cloned as inverted repeats into either the pGollum or pSmeagol vectors. The first set of constructs used pGollum (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Wroblewski <em>et al.</em> 2007</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">2014</a>) and resulted in the trigger sequence between a 400-bp fragment of the <em>UidA</em>, a beta-glucuronidase (GUS) encoding reporter gene and the PDK intron in an inverted repeat structure (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS1.docx">Figure S1</a>). The later set of constructs was made with pSmeagol by the use of the same restriction sites as in pGollum (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS1.docx">Figure S1</a>). Initial RNAi constructs were generated with pGollum, and later constructs were generated with pSmeagol because data indicated that the arrangement in pSmeagol (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS1.docx">Figure S1</a>) provided greater levels of RNAi (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Wroblewski <em>et al.</em> 2014</a>); however, RNAi was successfully induced with both versions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467036/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467036/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">All RNAi constructs generated and tested targeting NLR-encoding genes, the gene model of the progenitor EST sequences were derived from, gene family they belong to, resistance specificities tested, vector used, predicted number of <em>RGCs</em> targeted for RNAi, and resistance phenotypes abrogated</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">Validated clones were transferred to <em>Agrobacterium tumefaciens</em> strain LBA4404 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hoekema <em>et al.</em> 1983</a>) by electroporation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dower <em>et al.</em> 1988</a>). Stable transgenics of cv. Cobham Green (that lacks any of the tested disease resistance phenotypes) expressing each RNAi construct were generated by cocultivation with <em>A. tumefaciens</em> by the U.C. Davis Ralph Pearson Transformation Facility (<a href="http://ucdptf.ucdavis.edu/">http://ucdptf.ucdavis.edu/</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Michelmore <em>et al.</em> 1987</a>)). Typically 10−15 primary (T<sub>1</sub>) transgenic lines were generated and evaluated for each construct.</p><p id="p-20">Efficacy of silencing was evaluated in transgenic T<sub>1</sub> plants and their progeny by the level of silencing of the <em>UidA</em> gene, which is simultaneously targeted for RNAi by these constructs as described previously (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wroblewski <em>et al.</em> 2005</a>, <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">2007</a>). GUS expression levels were determined using <em>Agrobacterium</em>-mediated transient expression assays (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schöb <em>et al.</em> 1997</a>). Two leaves were infiltrated for each GUS assay and the assays were repeated at least twice (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Christopoulou <em>et al.</em> 2015</a>). The presence of the RNAi transgene <em>in planta</em> was confirmed by polymerase chain reaction (PCR) using primers <em>TrangeneF</em> and <em>TransgeneR</em>, which amplify a ∼800-bp fragment spanning one half of the inverted repeat structure (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS1.docx">Table S1</a>). Primers <em>Le9005L</em> and <em>Le9005R</em> were used as a control to confirm successful PCR amplifications in progeny segregating for the RNAi transgene.</p></div><div id="sec-4" class="subsection"><h3>Phenotypic evaluation of RNAi transgenics</h3><p id="p-21">T<sub>1</sub> lines lacking <em>GUS</em> expression, and therefore presumed to be silenced for the targeted NLR-encoding gene, were selected for assays of resistance. Two or three independent silenced T<sub>1</sub> lines per construct were crossed to different genotypes carrying the resistance phenotypes to be tested. The F<sub>1</sub>s were tested for DM resistance and/or hypersensitive response (HR) elicited by bacterial effectors as well as for silencing of GUS expression (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The F<sub>1</sub>s were expected to segregate for the silencing phenotype because the T<sub>1</sub> transgenics were hemizygous. On average 10 to 15 F<sub>1</sub>s, were tested per cross for each resistance phenotype (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Resistance to DM was evaluated with <em>B. lactucae</em> isolates diagnostic for specific <em>Dm</em> genes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) on detached cotyledons and detached leaves as described previously (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Christopoulou <em>et al.</em> 2015</a>). To evaluate HR to bacterial effectors, F<sub>1</sub>s at the four- to six-leaf stage were pressure infiltrated with <em>A. tumefaciens</em> strain C58 expressing AvrB, AvrRpm1, AvrRpt2, AvrRps4, or AvrPpiC as described previously (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Christopoulou <em>et al.</em> 2015</a>). A C58 strain expressing AvrPto was used as a positive control and PTFS40 strain expressing <em>UidA</em> as the negative control.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flowchart of the experimental design for the evaluation of RNA interference (RNAi) constructs, in particular QGC7A16_LRR_RNAi. The methodology applies to all constructs described in this study (different genetic background and isolates used in each case, as described in Table 1 and Table 2). (A) Transient beta-glucuronidase (GUS) expression assays were used to evaluate silencing efficiency of primary transgenic lines (T1s). (B) The most efficiently silenced T1s were crossed to cv. Pennlake, carrying Dm13. (C) Fifteen and ten F1s derived from each cross were evaluated for silencing efficiency [GUS assays and (PCR) polymerase chain reaction detection] and resistance to B. lactucae isolate CS12. Pathogenicity assays were performed on detached cotyledons and repeated on leaf disks of 3- to 4-week-old plants. Sporulation was only observed in the presence of the RNAi construct (C2). The presence of the transgene was detected with GUS transient assays (C1) and further confirmed by polymerase chain reaction detection (C3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1950672315" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Flowchart of the experimental design for the evaluation of RNA interference (RNAi) constructs, in particular QGC7A16_LRR_RNAi. The methodology applies to all constructs described in this study (different genetic background and isolates used in each case, as described in Table 1 and Table 2). (A) Transient beta-glucuronidase (GUS) expression assays were used to evaluate silencing efficiency of primary transgenic lines (T&lt;sub&gt;1&lt;/sub&gt;s). (B) The most efficiently silenced T&lt;sub&gt;1&lt;/sub&gt;s were crossed to cv. Pennlake, carrying &lt;em&gt;Dm13&lt;/em&gt;. (C) Fifteen and ten F&lt;sub&gt;1&lt;/sub&gt;s derived from each cross were evaluated for silencing efficiency [GUS assays and (PCR) polymerase chain reaction detection] and resistance to &lt;em&gt;B. lactucae&lt;/em&gt; isolate CS12. Pathogenicity assays were performed on detached cotyledons and repeated on leaf disks of 3- to 4-week-old plants. Sporulation was only observed in the presence of the RNAi construct (C2). The presence of the transgene was detected with GUS transient assays (C1) and further confirmed by polymerase chain reaction detection (C3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2655/F1.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/12/2655/F1.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467177" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Flowchart of the experimental design for the evaluation of RNA interference (RNAi) constructs, in particular QGC7A16_LRR_RNAi. The methodology applies to all constructs described in this study (different genetic background and isolates used in each case, as described in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). (A) Transient beta-glucuronidase (GUS) expression assays were used to evaluate silencing efficiency of primary transgenic lines (T<sub>1</sub>s). (B) The most efficiently silenced T<sub>1</sub>s were crossed to cv. Pennlake, carrying <em>Dm13</em>. (C) Fifteen and ten F<sub>1</sub>s derived from each cross were evaluated for silencing efficiency [GUS assays and (PCR) polymerase chain reaction detection] and resistance to <em>B. lactucae</em> isolate CS12. Pathogenicity assays were performed on detached cotyledons and repeated on leaf disks of 3- to 4-week-old plants. Sporulation was only observed in the presence of the RNAi construct (C2). The presence of the transgene was detected with GUS transient assays (C1) and further confirmed by polymerase chain reaction detection (C3).</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467211/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467211/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Lettuce genotypes, their respective resistance specificities, and the diagnostic <em>B. lactucae</em> isolates used to test for LDM resistance</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3><em>In silico</em> identification of RNAi targets</h3><p id="p-24">BLAST searches (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) using the fragment of the gene that had been cloned as an inverted repeat were performed to identify potential targets of the trigger sequences in the lettuce genome. The minimum requirement for a target was one or more contiguous sequences of at least 21 nucleotides with 100% identity to the reference genome of cv. Salinas.</p></div><div id="sec-6" class="subsection"><h3>Identification and classification of genes involved in pathogen recognition</h3><p id="p-25">Predicted gene models of the lettuce reference genome <em>L. sativa</em> cv. Salinas (<a href="http://lgr.genomecenter.ucdavis.edu">http://lgr.genomecenter.ucdavis.edu</a>; R. W. Michelmore and S. Reyes-Chin Wo, unpublished data) were screened using Hidden Markov Models (HMMs), using <em>E</em>-value 1 e<sup>−10</sup> as the cutoff, to search for genes encoding domains characteristic of NLR, RLK, and ABC transporter proteins that are known to be involved in pathogen recognition in other plant species. The HMMs (acquired from <a href="http://pfam.xfam.org/">http://pfam.xfam.org/</a> unless indicated otherwise) were PF00931.17 and NBS_712.hmm (HMM profile available at <a href="http://niblrrs.ucdavis.edu/At_RGenes/">http://niblrrs.ucdavis.edu/At_RGenes/</a>) for the NB domain, PF01582.15 for TIR, 9 HMMS for the LRR (PF00560.28, PF07723.8, PF007725.7, PF12799.2, PF13306.1, PF13516.1, PF13504.1, PF13855.1 and PF14580.1), PF00069.20 for protein kinase, PF00005.22 for ABC transporters, and PF00400 for the WD40 domain. The CC motifs were predicted with Paircoil2 at <em>P</em> scores below 0.025 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">McDonnell <em>et al.</em> 2006</a>). TM domains were predicted using both TMHMM 2.0 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Krogh <em>et al.</em> 2001</a>) and SCAMPI (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernsel <em>et al.</em> 2008</a>). Gene models with protein kinase, TM and LRR domains were classified as putative RLK encoding genes. The presence of an LRR domain alone classified genes in a separate category since LRR motifs can be found in many functionally unrelated proteins that are not involved in disease resistance (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kobe and Deisenhofer 1994</a>).</p><p id="p-26">A few additional genes with domains characteristic of NLR- and RLP-encoding genes were identified, using InterproScan, when an improved genome annotation of the reference genome of cv. Salinas became available at the end of the study. Twenty-eight additional gene models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>) predicted to have a NB domain were combined with the results of the HMM searches to generate the final list of predicted gene models involved in pathogen recognition.</p></div><div id="sec-7" class="subsection"><h3>Cladistic analysis of NLR-encoding genes</h3><p id="p-27">For analysis of the sequences of putative NLR-encoding genes, the NB domain was extracted with HMMER3 (<a href="http://hmmer.org/">http://hmmer.org/</a>.) using default parameters with NBS_712.hmm. The amino acid sequences were trimmed to ∼30 aa before the Walker A and 10 aa after the MHD motifs. Multiple-sequence alignments were generated with MAFFT (version 7.1, einsi parameters, 1,00 maximum iterations) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katoh and Standley 2013</a>) and refined manually with the alignment editor GeneDoc (Nicholas <em>et al.</em> 1997). The sequence identity was estimated by pairwise comparisons by the use of ClustalX over the entire alignment. Maximum-likelihood analyses were conducted with the customizable version of RAxML 8.0 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stamatakis 2014</a>), hosted on the CIPRES portal (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Miller <em>et al.</em> 2010</a>). The Jones-Taylor-Thornton protein evolutionary model with a GAMMA model of heterogeneity and empirical amino acid frequencies was selected as the best-fitting substitution matrix based on ProtTest 3.3 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Darriba <em>et al.</em> 2011</a>). Branch support values were calculated by 1000 replicates of fast bootstrapping (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stamatakis <em>et al.</em> 2008</a>). Genes were assigned to families using a cutoff of 70% nucleotide sequence identity. We used Dendroscope (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huson <em>et al.</em> 2007</a>) to display the sequence relationships of the <em>RGC</em>s.</p></div><div id="sec-8" class="subsection"><h3>Identification of RGC2 sequences in cv. Salinas</h3><p id="p-28">Sequencing and assembly of the lettuce reference genome permitted a detailed comparison of the MRC2 structure in cv. Salinas with that previously deduced for cv. Diana (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>). The sequences of 22 <em>RGC2</em> members obtained from cv. Diana (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>) were searched by BLASTN against the cv. Salinas genome assembly. Presence/absence was determined by reciprocal best BLAST hits at a minimum of 97% identity and 97% of the query represented in the alignment. The lack of a unique best match, in terms of both sequence identity and length of alignment, was recorded as absence of that particular family member in the Salinas genome. To address possibility of miss-assembly, the raw genomic reads from cv. Salinas were mapped back to the query sequences using the CLC Genomics Workbench (<a href="http://www.clcbio.com">http://www.clcbio.com</a>), using parameters: mismatch cost = 2, insertion and deletion cost = 3, minimum length fraction = 0.9 and similarity 1. The genome assembly and the raw reads of cv. Diana were used as a control.</p></div><div id="sec-9" class="subsection"><h3>Identification of singleton NLR-encoding genes in other lettuce genotypes</h3><p id="p-29">The draft genome assemblies of lettuce cvs. Diana, Valmaine, Greenlake, La Brillante, and PI125246 of <em>L. sativa</em> and accession US96UC23 of the wild progenitor <em>L. serriola</em> (D. Lavelle and R. W. Michelmore, unpublished data) were mined for the presence of singleton NLR encoding genes. The identification of the best BLAST hits was performed as described in the section “Identification of <em>RGC2</em> sequences in cv. Salinas.”</p><div id="sec-10" class="subsection"><h4>Statement on data and reagent availability:</h4><p id="p-30">All vectors, silencing constructs, and <em>B. lactucae</em> isolates are available upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FileS1.docx">File S1</a> describes the cloning process for the RNAi vector. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS1.docx">Figure S1</a> displays the orientation of the inverted repeats in both RNAi vectors. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS2.docx">Figure S2</a> shows the distribution of NLR and non-NLR encoding genes as well as the position of the scaffolds and the genetic bins of MRC2 on chromosome 2. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS1.docx">Table S1</a> has all the primer sequences. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a> lists all the genes putatively involved in pathogen recognition ordered on chromosomal linkage groups along with all the mapped disease resistance phenotypes. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS3.docx">Table S3</a> shows all the NLR encoding genes predicted to be targeted by the RNAi constructs tested for the resistance phenotypes in MRC8A. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS4.docx">Table S4</a> has all the predicted RNAi targets for LEO266_TIR_RNAi. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS5.docx">Table S5</a> lists all the genes predicted to be silenced by constructs QGC7A16_LRR_RNAi and LserNBS02_NB_RNAi.</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-31">The lettuce genome sequence is available at <a href="https://lgr.genomecenter.ucdavis.edu/">https://lgr.genomecenter.ucdavis.edu/</a>. The genome sequences also can be found at GenBank: <a href="http://www.ncbi.nlm.nih.gov/bioproject/68025">http://www.ncbi.nlm.nih.gov/bioproject/68025</a> and <a href="http://www.ncbi.nlm.nih.gov/genome/352/">http://www.ncbi.nlm.nih.gov/genome/352/</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Genome-wide identification of candidate resistance genes</h3><p id="p-32">Motif searches and InterproScan identified 1134 gene models in the reference genome of lettuce cv. Salinas that encoded domains characteristic of proteins involved in pathogen recognition (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>). Although 408 gene models were designated initially as NLRs, 23 were classified as genes encoding only LRRs because the NB motifs could not be validated visually. A total of 385 genes were used to generate a multiple sequence alignment of the NB domain for cluster analysis. N- and C-terminal domains were identified for 328 of 385 genes; the remaining 57 lacked one or more characteristic domains, which could reflect the diversity of gene structures present and/or be an artifact of assembling multi-gene families from short reads (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Forty-five genes were predicted to have a TIR domain but not an NB or LRR. The average length of the coding region for genes containing at least a NB domain (385 in total) was 3064 kb.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467230/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467230/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Classification of the 1134 genes identified with domains characteristic of <em>R</em> genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">A total of 703 LRR-encoding gene models were identified that did not contain a NB, CC, or TIR domain; 425 of these also had a predicted TM domain (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). A protein kinase domain was detected in 298 of these 703 LRR-containing genes, 270 of which also had a TM domain and were therefore classified as putative RLKs. Clusters of RLK encoding genes are located on multiple chromosomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). These clusters are distinct from the clusters of NLR-encoding genes. Thirty-nine RLKs were non-RD kinases, 14 of which clustered within 22 Mb of each other at the opposite end of chromosome 2 to MRC2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>); no resistance phenotypes are known to map to this RLK cluster. Three RLPs with a predicted TM domain but no kinase domain and high sequence similarity to the tomato <em>Ve</em> gene are located within a 100 kb region of chromosome 9 that cosegregates with resistance to race 1 of <em>Verticillium dahliae</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hayes <em>et al.</em> 2011</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic distribution of candidate genes involved in disease resistance over the nine chromosomes of lettuce (LG1&#x2212;LG9; assembly version Lsat_1_v6_lg). Precise coordinates and references are provided in Table S2. Track A: Disease resistance phenotypes; bars reflect the resolution determined by mapping in populations other than the core reference mapping population. Track B: NLR-encoding genes; TNLs are colored green and CNLs blue. Each pixel represents a gene. MRCs are shown as red bars below the genes. The scale bar enumerates each chromosome in Mb. Track C: Genes encoding receptor-like kinases: non - arginine-aspartate (non-RD) RLKs are colored orange. Track D: Genes encoding cytochrome P450 proteins. Track E: Gene density. Track F: Repeat density." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1950672315" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic distribution of candidate genes involved in disease resistance over the nine chromosomes of lettuce (LG1&#x2212;LG9; assembly version Lsat_1_v6_lg). Precise coordinates and references are provided in Table S2. Track A: Disease resistance phenotypes; bars reflect the resolution determined by mapping in populations other than the core reference mapping population. Track B: NLR-encoding genes; TNLs are colored green and CNLs blue. Each pixel represents a gene. MRCs are shown as red bars below the genes. The scale bar enumerates each chromosome in Mb. Track C: Genes encoding receptor-like kinases: non - arginine-aspartate (non-RD) RLKs are colored orange. Track D: Genes encoding cytochrome P450 proteins. Track E: Gene density. Track F: Repeat density.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2655/F2.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/12/2655/F2.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467164" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Genomic distribution of candidate genes involved in disease resistance over the nine chromosomes of lettuce (LG1−LG9; assembly version Lsat_1_v6_lg). Precise coordinates and references are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>. Track A: Disease resistance phenotypes; bars reflect the resolution determined by mapping in populations other than the core reference mapping population. Track B: NLR-encoding genes; TNLs are colored green and CNLs blue. Each pixel represents a gene. MRCs are shown as red bars below the genes. The scale bar enumerates each chromosome in Mb. Track C: Genes encoding receptor-like kinases: non - arginine-aspartate (non-RD) RLKs are colored orange. Track D: Genes encoding cytochrome P450 proteins. Track E: Gene density. Track F: Repeat density.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Grouping of NLR-encoding genes into 42 multigene families and singletons based on sequence similarity within the NB domain</h3><p id="p-36">A maximum-likelihood approach was used to study the sequence relationships of the NLR-encoding genes. Three of the 385 sequences were considered as a single sequence for alignment purposes because they were identical at the amino acid level in the NB region although they differed in their other domains. Clades with at least 70% sequence identity in pairwise comparisons of the NB domain were designated as distinct multigene families. This grouped the 385 NLR-encoding genes into 25 multigene families and 17 singletons (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Nomenclature for each family was kept consistent with previous assignations as much as possible (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Meyers <em>et al.</em> 1998b</a>; <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>). Genes lacking a CC domain or TIR domain were designated either as CNL-like or TNL-like based on sequence similarity to the NB domains of CC-NB-LRR- and TIR-NB-LRR-encoding genes respectively.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dendrogram of lettuce nucleotide binding-leucine rich repeat receptors generated with RaxML. Bootstrap support values on major clades generated by 1000 repetitions. Nomenclature was kept consistent with previously described families (Meyers et al. 1998a; McHale et al. 2009)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1950672315" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dendrogram of lettuce nucleotide binding-leucine rich repeat receptors generated with RaxML. Bootstrap support values on major clades generated by 1000 repetitions. Nomenclature was kept consistent with previously described families (Meyers &lt;em&gt;et al.&lt;/em&gt; 1998a; McHale &lt;em&gt;et al.&lt;/em&gt; 2009).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2655/F3.medium.gif" width="390" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/12/2655/F3.medium.gif" width="390" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467077" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Dendrogram of lettuce nucleotide binding-leucine rich repeat receptors generated with RaxML. Bootstrap support values on major clades generated by 1000 repetitions. Nomenclature was kept consistent with previously described families (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>; <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">A larger number of CNL than TNL families and singletons were identified (28 and 14 respectively), although fewer CNL-encoding genes than TNL-encoding genes were identified in total (167 and 21, respectively). Seventeen of the 42 clades were singletons, whereas 12 families contained more than 10 members each and accounted for 84% of the 385 NLR-encoding genes analyzed (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Three TNL families, <em>RGC4</em>, <em>RGC12</em>, and <em>RGC16</em>, were greatly expanded with more than 40 members each, comprising 73% of all the TNLs in the lettuce reference genome. The largest family was <em>RGC16</em> with 62 members.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467260/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467260/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Number of NLR encoding genes per family and type</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Draft genome assemblies of lettuce cvs. Diana, Valmaine, Greenlake, La Brillante, and PI125246 of <em>L. sativa</em> and accession US96UC23 of the wild progenitor <em>L. serriola</em> were mined for the singletons identified in cv. Salinas. Single homologs with 99–100% sequence identity for the entire length of the coding sequence at the nucleotide level were identified for all singleton <em>RGC</em>s in all six additional genotypes. Therefore, the <em>RGC</em> singletons were highly conserved and did not exhibit presence or absence polymorphism across this diverse germplasm. The complex sequence relationships of the NLR-encoding multigene families make assembly and identification of orthologs difficult; this precluded their facile analysis across these draft assemblies.</p></div><div id="sec-15" class="subsection"><h3>Distribution of NLR-encoding genes and MRCs in the genome of lettuce</h3><p id="p-41">NLR-encoding genes are found in every chromosome. Chromosomes 5 and 6 have the lowest numbers, with only 13 and 2 such genes, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>). Chromosomes 2 and 3 predominantly contain CNLs (≥50% of all CNLs), whereas chromosomes 1, 4, 5, 7, 8, and 9 contain primarily TNLs (≥89% of all TNLs). Members of each multigene family are typically found in clusters on the same chromosome (at least 68% of the gene families) with the exception of <em>RGC12</em> that has a genome-wide distribution and <em>RGC14</em> that is present on four chromosomes. <em>RGC12</em> is the second largest family with 55 members present on six of the nine chromosomes; 20 members are present on chromosome 9, followed by 19 on chromosome 4, six on chromosomes 2 and 5, and one on chromosomes 6 and 8 (two are unmapped). Eight <em>RGC14</em> genes are on chromosome 7 with one, two, and three, on chromosomes 1, 5, and 9 respectively. The genome distribution of singletons was varied; <em>RGC37</em> was located with other NLR-encoding genes; however, the remainder was several Mb away from other NLRs (<em>RGC30</em> has not been mapped).</p><p id="p-42">NLR-encoding genes are concentrated in five MRCs as defined by multiple disease resistance phenotypes on chromosomes 1, 2, 3, 4, and 8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>; <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) (<a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Truco <em>et al.</em> 2013</a>). The MRCs contain both NLR-encoding genes as well as genes not related to disease resistance. They are in large gene-poor, transposon-rich regions of the cv. Salinas genome. The average gene density for the entire genome is 2.43 genes per 100 kb; however, the gene density within each MRC ranges from 1 to 2.08 and is even lower for the flanking region extending 50 Mb on either end of each MRC (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). MRC1 and MRC4 both span ∼5 Mb and are composed of multiple families of NLR-encoding genes as described earlier (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Christopoulou <em>et al.</em> 2015</a>). MRC2 extends over ∼73 Mb and cosegregates with the large cluster of <em>Dm</em> genes that map to chromosome 2 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). MRC3 spans approximately 25 Mb near the telomere of chromosome 3, contains almost exclusively <em>RGC21</em> genes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), and cosegregates with <em>Dm13</em>. Two clusters of NLRs on chromosome 8, MRC8A (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS3.docx">Figure S3</a>) and MRC8B (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS4.docx">Figure S4</a>), are defined by the disease resistance phenotypes that map to chromosome 8. MRC8A spans ∼52 Mb; MRC8B is smaller and spans only 7 Mb. A third cluster of <em>RGCs</em> spans almost 26 Mb at the end of chromosome 8 and cosegregates with two QTL for resistance to <em>Fusarium</em> and <em>Verticillium</em> wilts respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS5.docx">Figure S5</a>). Three distinct clusters of NLR encoding genes, MRC9A (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS6.docx">Figure S6</a>), MRC9B, and MRC9C, are present on chromosome 9; each spans approximately 72 Mb, 35 Mb, and 15 Mb. MRC9A cosegregates with two QTL for resistance to DM and the candidate RLK–encoding genes for resistance to <em>Verticillium dahliae</em>. MRC9B consists of a large cluster of 19 <em>RGC12</em>s; however, no known resistance phenotypes have so far been mapped to this locus.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467162/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467162/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Sizes and gene densities of the MRC</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the MRC2 locus in cvs. Diana (A) and Salinas (B and C). (A) Summary of the data previously published for the region as part of the map-based cloning of Dm3 (Meyers et al. 1998a). The order of 24 RGC2 members in cv. Diana, sequenced from 22 BACs, was based on the deletion breakpoints (shown as red bars) of nine fast neutron&#x2013;induced dm3 mutants. Genes depicted in green are Type I and in orange Type II genes (Kuang et al. 2004). RGC2DB is Dm3 and it is a Type I gene. The spacing shown is for illustrative clarity; the physical distance between the BACs is unknown. (B) The Dm3 mediated resistance was originally mapped to a region defined by markers SCV12 and SCI11 (Chin et al. 2001). This region spans approximately 17 Mb in cv. Salinas. The positions of the nucleotide binding-leucine rich repeat receptor (NLR) encoding genes are shown on the lower bar and they are color-coded with a different color for each gene family. NLR genes that do not belong to the RGC2 gene family are colored light pink (RGC18) and light blue (RGC41). The upper bar shows the distribution of scaffolds for this piece of chromosome. Black represents scaffolds without any predicted genes. The rest of the scaffolds are colored randomly. (C) The distribution of NLRs across the entire MRC2 locus, including the Dm3 region, is shown to scale on the lower bar. The scaffolds are shown on the upper bar." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1950672315" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the MRC2 locus in cvs. Diana (A) and Salinas (B and C). (A) Summary of the data previously published for the region as part of the map-based cloning of &lt;em&gt;Dm3&lt;/em&gt; (Meyers &lt;em&gt;et al.&lt;/em&gt; 1998a). The order of 24 &lt;em&gt;RGC2&lt;/em&gt; members in cv. Diana, sequenced from 22 BACs, was based on the deletion breakpoints (shown as red bars) of nine fast neutron&#x2013;induced &lt;em&gt;dm3&lt;/em&gt; mutants. Genes depicted in green are Type I and in orange Type II genes (Kuang &lt;em&gt;et al.&lt;/em&gt; 2004). &lt;em&gt;RGC2DB&lt;/em&gt; is &lt;em&gt;Dm3&lt;/em&gt; and it is a Type I gene. The spacing shown is for illustrative clarity; the physical distance between the BACs is unknown. (B) The &lt;em&gt;Dm3&lt;/em&gt; mediated resistance was originally mapped to a region defined by markers SCV12 and SCI11 (Chin &lt;em&gt;et al.&lt;/em&gt; 2001). This region spans approximately 17 Mb in cv. Salinas. The positions of the nucleotide binding-leucine rich repeat receptor (NLR) encoding genes are shown on the lower bar and they are color-coded with a different color for each gene family. NLR genes that do not belong to the &lt;em&gt;RGC2&lt;/em&gt; gene family are colored light pink (&lt;em&gt;RGC18&lt;/em&gt;) and light blue (&lt;em&gt;RGC41&lt;/em&gt;). The upper bar shows the distribution of scaffolds for this piece of chromosome. Black represents scaffolds without any predicted genes. The rest of the scaffolds are colored randomly. (C) The distribution of NLRs across the entire MRC2 locus, including the &lt;em&gt;Dm3&lt;/em&gt; region, is shown to scale on the lower bar. The scaffolds are shown on the upper bar.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2655/F4.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/12/2655/F4.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467125" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Comparison of the MRC2 locus in cvs. Diana (A) and Salinas (B and C). (A) Summary of the data previously published for the region as part of the map-based cloning of <em>Dm3</em> (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>). The order of 24 <em>RGC2</em> members in cv. Diana, sequenced from 22 BACs, was based on the deletion breakpoints (shown as red bars) of nine fast neutron–induced <em>dm3</em> mutants. Genes depicted in green are Type I and in orange Type II genes (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kuang <em>et al.</em> 2004</a>). <em>RGC2DB</em> is <em>Dm3</em> and it is a Type I gene. The spacing shown is for illustrative clarity; the physical distance between the BACs is unknown. (B) The <em>Dm3</em> mediated resistance was originally mapped to a region defined by markers SCV12 and SCI11 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2001</a>). This region spans approximately 17 Mb in cv. Salinas. The positions of the nucleotide binding-leucine rich repeat receptor (NLR) encoding genes are shown on the lower bar and they are color-coded with a different color for each gene family. NLR genes that do not belong to the <em>RGC2</em> gene family are colored light pink (<em>RGC18</em>) and light blue (<em>RGC41</em>). The upper bar shows the distribution of scaffolds for this piece of chromosome. Black represents scaffolds without any predicted genes. The rest of the scaffolds are colored randomly. (C) The distribution of NLRs across the entire MRC2 locus, including the <em>Dm3</em> region, is shown to scale on the lower bar. The scaffolds are shown on the upper bar.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of the major resistance cluster on chromosome 3 (MRC3) of the reference genome assembly of L. sativa cv. Salinas. The top panel shows the position of nucleotide binding-leucine rich repeat receptor (NLR) encoding genes relative to the genetic bins and the scaffolds, whereas the bottom panel shows the position of all other genes that map to this locus. Color-coding: TNLs are colored in shades of green and CNLs in shades of blue, defense response (DR) genes in red and the remainder of the genes in yellow. The NLRs are further subdivided based on the number of characteristic domains detected. Abbreviations used: N for NB, L for leucine-rich repeat, C for coiled-coil, T for TOLL/interleukin-1 receptor. Lowercase letters used whenever the domain itself was not detected but inferred based on the phylogeny of the NB domain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1950672315" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graphical representation of the major resistance cluster on chromosome 3 (MRC3) of the reference genome assembly of &lt;em&gt;L. sativa&lt;/em&gt; cv. Salinas. The top panel shows the position of nucleotide binding-leucine rich repeat receptor (NLR) encoding genes relative to the genetic bins and the scaffolds, whereas the bottom panel shows the position of all other genes that map to this locus. Color-coding: TNLs are colored in shades of green and CNLs in shades of blue, defense response (DR) genes in red and the remainder of the genes in yellow. The NLRs are further subdivided based on the number of characteristic domains detected. Abbreviations used: N for NB, L for leucine-rich repeat, C for coiled-coil, T for TOLL/interleukin-1 receptor. Lowercase letters used whenever the domain itself was not detected but inferred based on the phylogeny of the NB domain.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/12/2655/F5.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/12/2655/F5.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/12/2655/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467270" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-49" class="first-child">Graphical representation of the major resistance cluster on chromosome 3 (MRC3) of the reference genome assembly of <em>L. sativa</em> cv. Salinas. The top panel shows the position of nucleotide binding-leucine rich repeat receptor (NLR) encoding genes relative to the genetic bins and the scaffolds, whereas the bottom panel shows the position of all other genes that map to this locus. Color-coding: TNLs are colored in shades of green and CNLs in shades of blue, defense response (DR) genes in red and the remainder of the genes in yellow. The NLRs are further subdivided based on the number of characteristic domains detected. Abbreviations used: N for NB, L for leucine-rich repeat, C for coiled-coil, T for TOLL/interleukin-1 receptor. Lowercase letters used whenever the domain itself was not detected but inferred based on the phylogeny of the NB domain.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Partial conservation of MRC2 structures in cvs. Salinas and Diana</h3><p id="p-50">Eight <em>Dm</em> genes and resistance to root aphid (<em>Ra</em>) are clustered near one end of chromosome 2 that genetically define the borders of MRC2 (<a id="xref-ref-36-9" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>). The structure of MRC2 had been previously characterized extensively in cv. Diana as part of the map-based cloning of <em>Dm3</em> (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Shen <em>et al.</em> 2002</a>). A total of 23 BAC clones from cv. Diana were isolated containing 24 <em>RGC2</em> family members, 16 of which were ordered using the breakpoints in a series of deletion mutants (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Meyers <em>et al.</em> 1998a</a>). Also, seven additional <em>RGC2</em> members were partially sequenced from multiple genotypes to study the evolution of this cluster (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Kuang <em>et al.</em> 2004</a>).</p><p id="p-51">We revisited the structure of the MRC2 cluster taking into consideration the genome assembly of cv. Salinas. The total size of MRC2 in cv. Salinas is ∼73 Mb and contains 754 genes of which 61 are NLRs, 2 are RLKs, and 5 are putative RLPs encoding genes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). <em>Dm3</em> was originally mapped in cv. Diana to a region that spans approximately 17 Mb within MRC2 in cv. Salinas. Many rearrangements of the <em>RGC2</em> genes are evident between the two haplotypes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Fourteen of 21 <em>RGC2</em> genes present in Diana could be identified in the MRC2 assembly of cv. Salinas; 12 of these had been previously classified as conserved Type II genes in cv. Diana. Variable Type I genes, which are primarily found in the middle section of MRC2 in cv. Diana, are largely absent in cv. Salinas, possibly due to a large deletion. Only <em>RGC2SE</em> and <em>RGC2SN</em> Type I genes in cv. Diana had reciprocal best hits with single genes in cv. Salinas. There are two additional NLR genes in cv. Salinas that had high levels of sequence identity to multiple Type I genes from cv. Diana and therefore orthologous relationships could not be established. These results support the heterogeneous evolution of NLR genes at this locus with genes at the periphery evolving more slowly than genes in the middle of the cluster.</p></div><div id="sec-17" class="subsection"><h3>Differential silencing of NLRs encoded at MRC2 by NB and LRR trigger sequences</h3><p id="p-52">The role of <em>RGC2</em> gene family members in determining resistance specificities cosegregating with <em>Dm3</em> had been studied previously using RNAi (<a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Wroblewski <em>et al.</em> 2007</a>). Not only <em>Dm3</em>, but also <em>Dm14</em>, <em>Dm16</em> and <em>Dm18</em>, as well as <em>Ra</em> were abrogated when an RNAi construct targeting the LRR-encoding domain of <em>RGC2B</em> was tested in genotypes expressing each of these resistance phenotypes. Sequence identities typically differ between NB and LRR domains, with NB domains tending to be more conserved than LRR domains. Therefore, a fragment of the NB domain of <em>RGC2B</em> was used as the trigger sequence for RNAi to determine whether more, the same, or a different spectrum of <em>Dm</em> specificities were abrogated. Transgenics were generated in cv. Diana and T<sub>1</sub> lines were evaluated for RNAi efficacy. The Diana T<sub>2</sub>s tested for <em>Dm3</em>-mediated resistance segregated for the resistant phenotype as expected from hemizygous T<sub>1</sub>s. Presence of the transgene, confirmed by both PCR and GUS expression assays, was absolutely coincident with loss of resistance (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-53">The two cv. Diana transgenics with the greatest levels of silencing were crossed to different genotypes to test for <em>Dm2</em>, <em>Dm6</em>, <em>Dm15</em>, <em>Dm16</em>, and <em>Dm18</em> specificities. All the F<sub>1</sub>s tested segregated for the presence of the RNAi construct. <em>Dm3</em> and <em>Dm18</em> were abrogated by trigger sequences derived from both the NB and LRR domains of <em>Dm3</em>; however, <em>Dm14</em> and <em>Dm16</em> were abrogated by the LRR-derived trigger sequence but not the NB-derived trigger sequence (<em>Dm14</em> was not tested with the NB-RNAi construct). Conversely, <em>Dm6</em> was compromised by the NB-derived trigger sequences but not those derived from the LRR. This was surprising because NB domains tend to be more conserved than LRR domains; this suggests that <em>Dm16</em> is a chimeric Type I gene with a LRR domain that is more closely related to <em>Dm3</em> than <em>Dm6</em> but with a more distantly related NB domain.</p></div><div id="sec-18" class="subsection"><h3>Structure and functional analysis of NLRs encoded at MRC3 in cv. Salinas</h3><p id="p-54">MRC3 is a tight cluster consisting primarily of <em>RGC21</em> family members on chromosome 3 that cosegregates with <em>Dm13</em>. It spans ∼25 Mb and contains 322 genes of which 31 are predicted to be involved in pathogen recognition: 27 CNL (26 <em>RGC21</em>, 1 <em>RGC40</em>), two RLK, and two LRR encoding genes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>; <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>).</p><p id="p-55">Transgenics of cv. Cobham Green were generated for two constructs that targeted members of the <em>RGC21</em> family of NLR-encoding genes that mapped to chromosome 3. These trigger sequences were derived from ESTs that were subsequently shown to encode the NB and LRR domains of different <em>RGC21</em> family members (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Two T<sub>1</sub> transgenic lines per construct were silenced for <em>GUS</em> and therefore presumably silenced for the targeted NLR-encoding gene(s). F<sub>1</sub>s of crosses to introduce the transgenes into cv. Pennlake carrying <em>Dm13</em> segregated for the transgene as evidenced by both assays of transient expression of <em>GUS</em> and PCR amplification of the transgene. The F<sub>1</sub>s of Pennlake x QGC7A16_LRR_RNAi also segregated 1:1 for susceptibility:resistance to isolate CS12 of <em>B. lactucae</em>. The loss of resistance cosegregated perfectly with the presence of the transgene for F<sub>1</sub>s derived from two independent primary transgenics (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Therefore, <em>QGC7A16</em> or another <em>RGC21</em> family member(s) confers <em>Dm13</em>. However, LserNBS02_NB_RNAi that also targeted <em>RGC21</em> family members did not abrogate <em>Dm13</em>-mediated resistance, even though F<sub>1</sub>s carrying the transgene were effectively silenced for GUS expression.</p><p id="p-56">QGC7A16_LRR_RNAi was predicted as potentially targeting 16 of 28 <em>RGC21</em> family members and eight other genes encoding only an LRR domain, whereas LserNBS02_NB_RNAi was predicted to target 26 of 33 members of the <em>RGC21</em> family, 3 <em>RGC1</em> members and 3 unassigned CNLs. All of NLR-encoding genes predicted to be silenced by QGC7A16_LRR_RNAi also were predicted to be silenced by LserNBS02_NB_RNAi (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS5.docx">Table S5</a>). The lack of abrogation of resistance by LserNBS02_NB_RNAi despite it being predicted to target most known <em>RGC21</em> members could be explained by additional diverse <em>RGC21</em> genes unique to cv. Pennlake that are not present in the cv. Salinas assembly; however, both Salinas and Pennlake exhibit <em>Dm13</em> specificity. In addition to silencing NLR-encoding genes, QGC7A16_LRR_RNAi was predicted to silence eight LRR encoding genes that lack an NB domain. Seven out of the eight LRR encoding genes only targeted by QGC7A16_LRR_RNAi have high sequence similarity to other <em>RGC21</em> members. These could either be <em>RGC21</em> genes that are improperly assembled in the reference assembly of cv. Salinas or correctly assembled genes that lack an NB domain, one or more of which can confer <em>Dm13</em> specificity. Alternatively, <em>Dm13</em> may be encoded by a full length <em>RGC21</em> gene and inadequate levels of silencing occurred from LserNBS02_NB_RNAi.</p></div><div id="sec-19" class="subsection"><h3>Structure and functional analysis of NLRs encoded at MRC8</h3><p id="p-57">There are three MRCs on chromosome 8 that cosegregate with different resistance-related phenotypes. <em>Agrobacterium</em>-mediated transient expression of bacterial effectors AvrRps4 and AvrPpiC triggers a HR in several cultivars. MRC8A is a small cluster spread over a 54-Mb region on chromosome 8 in cv. Salinas that cosegregates with HR to AvrPpiC in cv. Valmaine. MRC8A has a total of 1132 genes, of which 23 genes are potentially involved in pathogen recognition: four TNLs (<em>RGC4</em>) and two CNLs (<em>RGC27</em>), eight RLKs, four RLPs, one TIR-, and four LRR-encoding genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS3.docx">Figure S3</a>; <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Three constructs were tested for abrogation of HR to AvrPpiC: Lsat11_NB_RNAi, Contig5632_TIR_RNAi, and QGD14014_NB_RNAi (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). These were predicted to potentially silence members of <em>RGC34</em>, <em>RGC4</em>, and <em>RGC5</em>, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS3.docx">Table S3</a>). Two RNAi transgenic lines exhibiting silencing per construct were crossed to cv. Valmaine and the hybrids were tested for silencing and the response to the AvrPpiC. All the tested F<sub>1</sub> progenies segregated for high levels of silencing of GUS coincident with the presence of the transgene. A strong HR occurred in both the presence and the absence of the transgene. Therefore, there is no evidence that any of these NLRs determine the HR to AvrPpiC.</p><p id="p-58">MRC8B is a dense cluster cosegregating with the HR to AvrRps4. It spans ∼7 Mb in cv. Salinas and contains 127 genes, 36 of which are <em>RGC4</em> members (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS4.docx">Figure S4</a>; <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>). Four constructs were tested: LEO266_TIR_RNAi (<em>RGC4</em>), LEO414_LRR_RNAi (<em>RGC9</em>), AY153833.1_LRR_RNAi (<em>RGC15</em>), and LEO_395_LRR_RNAi (<em>RGC29</em>) (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The last three constructs were tested because at the time of the experiments genetic data suggested that all of these RGCs co-segregated with HR to AvrRps4; however, they were subsequently shown not to be at this location in the whole genome assembly. Two independent transgenic lines for each construct were crossed to cv. Valmaine. The leaves of F<sub>1</sub>s were pressured infiltrated with <em>A. tumefaciens</em> cultures expressing <em>AvrRps4</em>. The F<sub>1</sub>s segregated for the transgene as shown by PCR and GUS transient assays; however, all of them retained a strong HR to AvrRps4 regardless of the silencing and the presence of the transgene (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS4.docx">Table S4</a>). Therefore, there is no evidence that members of <em>RGC4</em>, <em>RGC9</em>, or <em>RGC15</em>, <em>RGC29</em> targeted by these constructs have a role in the recognition of AvrRps4 in lettuce. <em>In silico</em> predictions of RNAi targets for these constructs showed that there are 3 and 30 NLR-encoding genes within MRC8A and MRC8B respectively that were unlikely to be silenced by these constructs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS3.docx">Table S3</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS4.docx">Table S4</a>). These are candidates for additional RNAi targets to test their involvement in HR to AvrPpiC and AvrRps4.</p><p id="p-59">MRC8C is a complex cluster that spans 25 Mb and contains 348 genes, 26 of which are NLR-encoding genes (<a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>). There are 9 <em>RGC15</em>, 11 <em>RGC9</em>, 4 <em>RGC29</em>, 1 <em>RGC7</em>, and 1 <em>RGC43</em> members (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/FigureS5.docx">Figure S5</a>). A QTL to <em>Fusarium oxysporum</em> and a minor QTL for resistance to <em>V. dahliae</em> cosegregate with MRC8C. These resistance phenotypes have to be assessed in the field and therefore no constructs were tested for these phenotypes.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-60">This paper provides a detailed description of the genome-wide architecture of disease resistance in lettuce. Thirty-six major phenotypic genes and 20 QTLs for resistance to 10 pathogens and one pest have been positioned relative to 1134 candidate genes in genomic scaffolds assigned to genetic bins ordered along the nine chromosomal linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>). Most but not all resistance phenotypes are located with arrays of NLR-encoding genes. Conversely, of the 42 families of NLR-encoding genes in <em>L. sativa</em> cv. Salinas, 26 were co-located with resistance phenotypes. We therefore defined five MRCs as regions of the genome enriched for determinants of resistance based on both phenotypic information from a variety of genotypes and the genomic distribution of NLR-encoding genes. This clustered distribution is similar to that observed in other species (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> 2014</a>). The total number of NLR-encoding genes observed in cv. Salinas (385) is in the middle of the range (54−1015) observed in other plant species (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Meyers <em>et al.</em> 2003</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Zhou <em>et al.</em> 2004</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kohler <em>et al.</em> 2008</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Porter <em>et al.</em> 2009</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arya <em>et al.</em> 2014</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Shao <em>et al.</em> 2014</a>). We detected more TNL than CNL genes in total but fewer TNL than CNL families; this is due the expansion of three TNL families, <em>RGC4</em>, <em>RGC12</em>, and <em>RGC16</em>, that make up 68% of all NLR-encoding genes. Seventeen families were only represented by singletons, of which 12 were CNLs and 5 were TNLs (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-61">Most families tend to be colocated in the same MRC, likely reflecting expansion by unequal crossing over (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2001</a>; <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Meyers <em>et al.</em> 2003</a>; <a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Zhou <em>et al.</em> 2004</a>). The largest family, <em>RGC16</em>, has 62 members, of which 60 are at MRC1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>). However, there are exceptions to the colocation of family members. <em>RGC14</em> is distributed in two clusters (on chromosomes 7 and 9), possibly as a consequence of a segmental duplication event. The second largest family, <em>RGC12</em>, with 55 members is widely distributed through the genome and present on six chromosomes. The multiple chromosomal locations of large families may reflect grouping of more slowly evolving genes into single families based on our 70% threshold for sequence identity. Several clusters are homogeneous, comprising of a single <em>RGC</em> family (MRC1, MRC4, MRC3, MRC8A, MRC8B). Some are complex comprising of multiple TNLs and CNLs; the most complex was MRC2 that had representatives of three TNL and three CNL families (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1/TableS2.xlsx">Table S2</a>). All MRCs contained genes other than NLR-encoding genes; however, they were not enriched for any classes of genes, other than NLR-encoding genes.</p><p id="p-62">The majority of the data presented are based on the reference genome of cv. Salinas. However, NLR-encoding gene repertoires can differ greatly between genotypes; the number of <em>RGC2</em> family members can vary from ∼12 to 42 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kuang <em>et al.</em> 2006</a>). The numbers detected in the genome assemblies of cvs. Salinas and Diana were in the middle of this range (20 and 29, respectively). Although all of the conserved Type II genes were present in both, cv. Diana had more of the variable Type I genes; however, the order of the Type II genes apparently differs between the haplotypes. Therefore, while the general architecture observed in cv. Salinas may be representative of <em>L. sativa</em>, the fine-scale details may be specific to individual genotypes; this needs to be taken into consideration when implementing functional studies and cloning strategies.</p><p id="p-63">Functional studies using RNAi successfully assigned 16 resistance phenotypes to four <em>RGC</em> families. The selection of RGCs targeted for RNAi predated the sequencing and assembly of the reference genome and was based on ESTs of NLR-encoding genes that had been mapped relative to disease resistance phenotypes (<a id="xref-ref-36-10" class="xref-bibr" href="#ref-36">McHale <em>et al.</em> 2009</a>). This provided RNAi transgenics targeting 12 of the 42 RGC families representing most of the major gene families and the majority of NLRs that cosegregate with resistance phenotypes. Members of <em>RGC1</em> are required for the <em>Dm5/8</em>- and <em>Dm45</em>-mediated resistance as well as for the HR to AvrB, AvrRpm1, and AvrRpt2, that map to MRC1. <em>RGC12</em> family members are required for <em>Dm4</em>, <em>Dm7</em>, <em>Dm11</em>, and <em>Dm44</em> specificities in MRC4 as described previously (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Christopoulou <em>et al.</em> 2015</a>). For MRC2 and MRC4, members of the largest <em>RGC</em> family present were shown to be required for the co-segregating resistance specificities. Interestingly, six constructs predicted to target all but four members of the largest family at MRC1, <em>RGC16</em>, failed to abrogate any of the nine tested phenotypes that map to this cluster.</p><p id="p-64">RNAi was an efficient method to assign resistance phenotypes to <em>RGC</em> families. Fragments of the NB, LRR and TIR domains were used as trigger sequences for RNAi for the 33 constructs used in this study. Abrogation of resistance was observed with all three types of trigger sequences. The different regions appeared to confer RNAi with similar efficacy; 1 of 4 TIR, 2 of 17 NB, and 4 of 12 LRR trigger sequences abrogated resistance. Although the NB domain is the most conserved NLR domain, targeting of the TIR and the LRR domains also resulted in silencing of multiple family members. When multiple trigger sequences targeted the same gene, as for <em>Dm3</em>, overlapping sets of specificities were abrogated. This provides the opportunity for designing trigger sequences that can abrogate specific subsets of family members.</p><p id="p-65">These 33 transgenic RNAi lines are resources for assigning additional resistance phenotypes to RGC families. Also, further lines can be generated to target more of the <em>RGC</em> families within MRCs with resistance phenotypes for which causal <em>RGCs</em> have not been identified, such as MRC1. After the sequence analysis and the ultradense map were refined it is evident that some existing RNAi constructs could also be tested for other specificities that cosegregate with RGCs of the targeted family. For example constructs Contig5632_TIR_RNAi and AF017754_NB_RNAi were tested for phenotypes that map to MRC8A and MRC3 respectively but target a large subset of <em>RGC4</em> members and would be good tester stocks for the HR to AvrRps4 (MRC8B). The RNAi strategy that we have implemented provides an efficient approach to attribute phenotypes to subsets of a multigene <em>RGC</em> family; however, achievement of single gene resolution is challenging by RNAi methods. Nonetheless, it does provide candidates for knockouts of individual genes using CRISPR/Cas9 technology and valuable information for the development of robust markers to facilitate breeding efforts.</p><p id="p-66">Overall, resistance phenotypes were shown to be encoded by both CNL and TNL multigene families. No resistance phenotype was encoded by a singleton such as the allelic series observed at the <em>L</em> locus in flax (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ellis <em>et al.</em> 2007</a>). Interestingly, when a <em>Dm</em> specificity was abrogated, other resistance phenotypes that mapped to the same locus were also compromised indicating that the same <em>RGC</em> family encoded multiple <em>Dm</em> genes. Conversely, in all cases in which silencing of a <em>Dm</em> specificity occurred, the causal <em>RGC</em> family contained more than 20 members. Together, these data suggest that the expansion of specific <em>RGC</em> families in lettuce is in response to evolutionary pressures exerted by variation in <em>B. lactucae</em>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-67">We thank Pauline Sanders, Dean Lavelle, Joana Nam, Oswaldo Ochoa, Brandon Seminatore, and Rebecca Walker for technical assistance. This research was supported by grants nos. 04-35300-14601 and 2005-35300-004447 from the National Research Initiative (NRI) Plant Genome Program of the USDA Cooperative State Research, Education and Extension Service (CSREES) and DBI0421630 and DBI0211923 from the Plant Genome Research Program of the National Science Foundation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-16"><p id="p-68">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020818/-/DC1</a></p></li><li class="fn" id="fn-17"><p id="p-69"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-07-27"><span class="received-label">Received </span>July 27, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-09-30"><span class="accepted-label">Accepted </span>September 30, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Christopoulou <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.2" data-doi="10.1371/journal.pone.0107987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arya</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide identification and expression analysis of NBS-encoding genes in <em>Malus domestica</em> and expansion of NBS genes family in Rosaceae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e107987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De107987%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0107987%26rft_id%253Dinfo%253Apmid%252F25232838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0107987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25232838&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernatzky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Toward a saturated linkage map in tomato based on isozymes and random cDNA sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBernatzky%26rft.auinit1%253DR.%26rft.volume%253D112%26rft.issue%253D4%26rft.spage%253D887%26rft.epage%253D898%26rft.atitle%253DTOWARD%2BA%2BSATURATED%2BLINKAGE%2BMAP%2BIN%2BTOMATO%2BBASED%2BON%2BISOZYMES%2BAND%2BRANDOM%2BcDNA%2BSEQUENCES%26rft_id%253Dinfo%253Apmid%252F17246322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTEyLzQvODg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.12.2655.4" data-doi="10.1073/pnas.0711151105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernsel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viklund</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindahl</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Prediction of membrane-protein topology from first principles.</span> <span class="cit-source">Proc. Natl. Acad. Sci. USA</span> <span class="cit-vol">105</span>: <span class="cit-fpage">7177</span>–<span class="cit-lpage">7181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0711151105%26rft_id%253Dinfo%253Apmid%252F18477697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzIwLzcxNzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.5" data-doi="10.1016/j.pbi.2012.03.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bozkurt</span>  <span class="cit-name-given-names">T. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schornack</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banfield</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamoun</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Oomycetes, effectors, and all that jazz.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBozkurt%26rft.auinit1%253DT.%2BO.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D483%26rft.epage%253D492%26rft.atitle%253DOomycetes%252C%2Beffectors%252C%2Band%2Ball%2Bthat%2Bjazz.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.03.008%26rft_id%253Dinfo%253Apmid%252F22483402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.03.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22483402&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arroyo-Garcia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavelle</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination and spontaneous mutation at the major cluster of resistance genes in Lettuce (<em>Lactuca sativa</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChin%26rft.auinit1%253DD.%2BB.%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D831%26rft.epage%253D849%26rft.atitle%253DRecombination%2Band%2BSpontaneous%2BMutation%2Bat%2Bthe%2BMajor%2BCluster%2Bof%2BResistance%2BGenes%2Bin%2BLettuce%2B%2528Lactuca%2Bsativa%2529%26rft_id%253Dinfo%253Apmid%252F11157000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzIvODMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.7" data-doi="10.1094/MPMI-06-14-0175-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christopoulou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHale</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dissection of two complex clusters of resistance genes in Lettuce (<em>Lactuca sativa</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">751</span>−<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BMicrobe%2BInteract.%26rft.volume%253D28%26rft.spage%253D751%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-06-14-0175-R%26rft_id%253Dinfo%253Apmid%252F25650829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-06-14-0175-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25650829&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.8" data-doi="10.1126/science.1138632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweikert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Common sequence polymorphisms shaping genetic diversity in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DClark%26rft.auinit1%253DR.%2BM.%26rft.volume%253D317%26rft.issue%253D5836%26rft.spage%253D338%26rft.epage%253D342%26rft.atitle%253DCommon%2BSequence%2BPolymorphisms%2BShaping%2BGenetic%2BDiversity%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1138632%26rft_id%253Dinfo%253Apmid%252F17641193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTgzNi8zMzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.9" data-doi="10.1111/j.1744-7348.1976.tb01701.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crute</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">The genetic relationship between races of <em>Bremia lactucae</em> and cultivars of <em>Lactuca sativa</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Biol.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BBiol.%26rft.volume%253D83%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7348.1976.tb01701.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7348.1976.tb01701.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.10" data-doi="10.1371/journal.ppat.0020014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dardick</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Plant and animal pathogen recognition receptors signal through non-RD kinases.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D2%26rft.spage%253D14%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0020014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0020014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000202894100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.11" data-doi="10.1016/j.pbi.2012.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dardick</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwessinger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Non-arginine-aspartate (non-RD) kinases are associated with innate immune receptors that recognize conserved microbial signatures.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DDardick%26rft.auinit1%253DC.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D358%26rft.epage%253D366%26rft.atitle%253DNon-arginine-aspartate%2B%2528non-RD%2529%2Bkinases%2Bare%2Bassociated%2Bwith%2Binnate%2Bimmune%2Breceptors%2Bthat%2Brecognize%2Bconserved%2Bmicrobial%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.05.002%26rft_id%253Dinfo%253Apmid%252F22658367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22658367&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.12" data-doi="10.1093/bioinformatics/btr088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ProtTest 3: fast selection of best-fit models of protein evolution.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr088%26rft_id%253Dinfo%253Apmid%252F21335321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNy84LzExNjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-5.12.2655.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subbarao</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raid</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source"><em>Compendium of Lettuce Diseases</em>.</span> <span class="cit-publ-name">APS Press</span>, <span class="cit-publ-loc">St. Paul, Minnesota</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.14" data-doi="10.1093/nar/16.13.6127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dower</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragsdale</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">High efficiency transformation of <em>E.coli</em> by high voltage electroporation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">6127</span>–<span class="cit-lpage">6145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DDower%26rft.auinit1%253DW.%2BJ.%26rft.volume%253D16%26rft.issue%253D13%26rft.spage%253D6127%26rft.epage%253D6145%26rft.atitle%253DHigh%2Befficiency%2Btransformation%2Bof%2BE.coli%2Bby%2Bhigh%2Bvoltage%2Belectroporation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F16.13.6127%26rft_id%253Dinfo%253Apmid%252F3041370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMy82MTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.15" data-doi="10.1146/annurev.phyto.45.062806.094331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Flax rust resistance gene specificity is based on direct resistance-avirulence protein interactions.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DEllis%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D45%26rft.spage%253D289%26rft.epage%253D306%26rft.atitle%253DFlax%2Brust%2Bresistance%2Bgene%2Bspecificity%2Bis%2Bbased%2Bon%2Bdirect%2Bresistance-avirulence%2Bprotein%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.45.062806.094331%26rft_id%253Dinfo%253Apmid%252F17430087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.45.062806.094331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17430087&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249481200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.16" data-doi="10.1023/A:1016050706560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClement</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phelps</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vice</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identification of sources of resistance in lettuce to the lettuce root aphid, <em>Pemphigus bursarius</em>.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D125%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1016050706560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1016050706560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.17" data-doi="10.1111/j.1365-3059.1987.tb02267.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farrara</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilott</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Genetic analysis of factors for resistance to downy mildew (<em>Bremia lactucae</em>) in species of lettuce (<em>Lactuca sativa</em> and <em>L. serriola</em>).</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D36%26rft.spage%253D499%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.1987.tb02267.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.1987.tb02267.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.18" data-doi="10.1104/pp.111.181990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide comparison of nucleotide-binding site-leucine-rich repeat-encoding genes in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.181990%26rft_id%253Dinfo%253Apmid%252F21810963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTcvMi83NTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.19" data-doi="10.1007/s00122-011-1603-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHale</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallad</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The inheritance of resistance to Verticillium wilt caused by race 1 isolates of <em>Verticillium dahliae</em> in the lettuce cultivar La Brillante.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D509%26rft.epage%253D517%26rft.atitle%253DThe%2Binheritance%2Bof%2Bresistance%2Bto%2BVerticillium%2Bwilt%2Bcaused%2Bby%2Brace%2B1%2Bisolates%2Bof%2BVerticillium%2Bdahliae%2Bin%2Bthe%2Blettuce%2Bcultivar%2BLa%2BBrillante.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1603-y%26rft_id%253Dinfo%253Apmid%252F21567237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1603-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21567237&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.20" data-doi="10.1038/303179a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoekema</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooykaas</span>  <span class="cit-name-given-names">P. J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schilperoort</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A binary plant vector strategy based on separation of vir-region and T-region of the <em>Agrobacterium-tumefaciens</em> Ti-plasmid</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D303%26rft.spage%253D179%26rft_id%253Dinfo%253Adoi%252F10.1038%252F303179a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/303179a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983QP69700072&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.21" data-doi="10.1146/annurev.phyto.39.1.285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulbert</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Resistance gene complexes: evolution and utilization.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DHulbert%26rft.auinit1%253DS.%2BH.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D285%26rft.epage%253D312%26rft.atitle%253DRESISTANCE%2BGENE%2BCOMPLEXES%253A%2BEvolution%2Band%2BUtilization%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.39.1.285%26rft_id%253Dinfo%253Apmid%252F11701867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.39.1.285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701867&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170927100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.22" data-doi="10.1186/1471-2105-8-460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dezulian</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dendroscope: An interactive viewer for large phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHuson%26rft.auinit1%253DD.%2BH.%26rft.volume%253D8%26rft.spage%253D460%26rft.epage%253D460%26rft.atitle%253DDendroscope%253A%2BAn%2Binteractive%2Bviewer%2Bfor%2Blarge%2Bphylogenetic%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-460%26rft_id%253Dinfo%253Apmid%252F18034891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18034891&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.23" data-doi="10.1094/Phyto-79-888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ilott</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulbert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Genetic analysis of the gene-for-gene interaction between lettuce (<em>Lactuca sativa</em>) and <em>Bremia lactucae</em>.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">888</span>–<span class="cit-lpage">897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D79%26rft.spage%253D888%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-79-888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-79-888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989AM24300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.24" data-doi="10.1038/nature05286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dangl</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The plant immune system.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D444%26rft.issue%253D7117%26rft.spage%253D323%26rft.epage%253D329%26rft.atitle%253DThe%2Bplant%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05286%26rft_id%253Dinfo%253Apmid%252F17108957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17108957&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242018300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.25" data-doi="10.1016/j.pbi.2013.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karasov</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic variability as a driver of plant–pathogen coevolution?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D18%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2013.12.003%26rft_id%253Dinfo%253Apmid%252F24491596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2013.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24491596&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.26" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvNzcyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.27" data-doi="10.1016/0968-0004(94)90090-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kobe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deisenhofer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The leucine-rich repeat: a versatile binding motif.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DKobe%26rft.auinit1%253DB.%26rft.volume%253D19%26rft.issue%253D10%26rft.spage%253D415%26rft.epage%253D421%26rft.atitle%253DThe%2Bleucine-rich%2Brepeat%253A%2Ba%2Bversatile%2Bbinding%2Bmotif.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0968-0004%252894%252990090-6%26rft_id%253Dinfo%253Apmid%252F7817399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0968-0004(94)90090-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7817399&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PP01900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.28" data-doi="10.1007/s11103-008-9293-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duplessis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baucher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geelen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide identification of NBS resistance genes in <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DKohler%26rft.auinit1%253DA.%26rft.volume%253D66%26rft.issue%253D6%26rft.spage%253D619%26rft.epage%253D636%26rft.atitle%253DGenome-wide%2Bidentification%2Bof%2BNBS%2Bresistance%2Bgenes%2Bin%2BPopulus%2Btrichocarpa.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-008-9293-9%26rft_id%253Dinfo%253Apmid%252F18247136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-008-9293-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18247136&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254114700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.29" data-doi="10.1006/jmbi.2000.4315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKrogh%26rft.auinit1%253DA.%26rft.volume%253D305%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D580%26rft.atitle%253DPredicting%2Btransmembrane%2Bprotein%2Btopology%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%253A%2Bapplication%2Bto%2Bcomplete%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4315%26rft_id%253Dinfo%253Apmid%252F11152613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11152613&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167760800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.30" data-doi="10.1105/tpc.104.025502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">S.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Multiple genetic processes result in heterogeneous rates of evolution within the major cluster disease resistance genes in lettuce.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2870</span>–<span class="cit-lpage">2894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.104.025502%26rft_id%253Dinfo%253Apmid%252F15494555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMS8yODcwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.31" data-doi="10.1111/j.1365-313X.2006.02755.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The disease resistance gene <em>Dm3</em> is infrequent in natural populations of <em>Lactuca serriola</em> due to deletions and frequent gene conversions at the <em>RGC2</em> locus.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKuang%26rft.auinit1%253DH.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D48%26rft.atitle%253DThe%2Bdisease%2Bresistance%2Bgene%2BDm3%2Bis%2Binfrequent%2Bin%2Bnatural%2Bpopulations%2Bof%2BLactuca%2Bserriola%2Bdue%2Bto%2Bdeletions%2Band%2Bfrequent%2Bgene%2Bconversions%2Bat%2Bthe%2BRGC2%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02755.x%26rft_id%253Dinfo%253Apmid%252F16762035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02755.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16762035&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238256700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.32" data-doi="10.1016/j.tig.2004.01.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leister</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Tandem and segmental gene duplication and recombination in the evolution of plant disease resistance genes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLeister%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D116%26rft.epage%253D122%26rft.atitle%253DTandem%2Band%2Bsegmental%2Bgene%2Bduplication%2Band%2Brecombination%2Bin%2Bthe%2Bevolution%2Bof%2Bplant%2Bdisease%2Bresistance%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.01.007%26rft_id%253Dinfo%253Apmid%252F15049302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15049302&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220327300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.33" data-doi="10.1104/pp.111.192062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dynamic nucleotide-binding site and leucine-rich repeat-encoding genes in the grass family.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.192062%26rft_id%253Dinfo%253Apmid%252F22422941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTkvMS8xOTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.34" data-doi="10.1093/bioinformatics/bti797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonnell</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keating</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Paircoil2: improved prediction of coiled coils from sequence.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti797%26rft_id%253Dinfo%253Apmid%252F16317077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMi8zLzM1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.35" data-doi="10.1186/gb-2006-7-4-212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McHale</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koehl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Plant NBS-LRR proteins: adaptable guards.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMcHale%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D212%26rft.epage%253D212%26rft.atitle%253DPlant%2BNBS-LRR%2Bproteins%253A%2Badaptable%2Bguards.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-4-212%26rft_id%253Dinfo%253Apmid%252F16677430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-4-212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16677430&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.36" data-doi="10.1007/s00122-008-0921-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McHale</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genomic architecture of disease resistance in lettuce.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMcHale%26rft.auinit1%253DL.%2BK.%26rft.volume%253D118%26rft.issue%253D3%26rft.spage%253D565%26rft.epage%253D580%26rft.atitle%253DThe%2Bgenomic%2Barchitecture%2Bof%2Bdisease%2Bresistance%2Bin%2Blettuce.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0921-1%26rft_id%253Dinfo%253Apmid%252F19005638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0921-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19005638&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262577300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.37" data-doi="10.1105/tpc.10.11.1817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivaramakrishnan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavelle</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">a</span> <span class="cit-article-title">The major resistance gene cluster in lettuce is highly duplicated and spans several megabases.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DMeyers%26rft.auinit1%253DB.%2BC.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D1817%26rft.epage%253D1832%26rft.atitle%253DThe%2BMajor%2BResistance%2BGene%2BCluster%2Bin%2BLettuce%2BIs%2BHighly%2BDuplicated%2Band%2BSpans%2BSeveral%2BMegabases%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.10.11.1817%26rft_id%253Dinfo%253Apmid%252F9811791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMS8xODE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.38" data-doi="10.1105/tpc.10.11.1833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span><span class="cit-comment">b</span> <span class="cit-article-title">Receptor-like genes in the major resistance locus of lettuce are subject to divergent selection.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1833</span>–<span class="cit-lpage">1846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DMeyers%26rft.auinit1%253DB.%2BC.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D1833%26rft.epage%253D1846%26rft.atitle%253DReceptor-like%2BGenes%2Bin%2Bthe%2BMajor%2BResistance%2BLocus%2Bof%2BLettuce%2BAre%2BSubject%2Bto%2BDivergent%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.10.11.1833%26rft_id%253Dinfo%253Apmid%252F9811792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMS8xODMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.39" data-doi="10.1105/tpc.009308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griego</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide analysis of NBS-LRR–encoding genes in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.009308%26rft_id%253Dinfo%253Apmid%252F12671079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE1LzQvODA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.40" data-doi="10.1016/j.pbi.2005.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaushik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nandety</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolving disease resistance genes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DMeyers%26rft.auinit1%253DB.%2BC.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D134%26rft.atitle%253DEvolving%2Bdisease%2Bresistance%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2005.01.002%26rft_id%253Dinfo%253Apmid%252F15752991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2005.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15752991&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227932600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Clusters of resistance genes in plants evolve by divergent selection and a birth-and-death process.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D1113%26rft.epage%253D1130%26rft.atitle%253DClusters%2Bof%2BResistance%2BGenes%2Bin%2BPlants%2BEvolve%2Bby%2BDivergent%2BSelection%2Band%2Ba%2BBirth-and-Death%2BProcess%26rft_id%253Dinfo%253Apmid%252F9847076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjgvMTEvMTExMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTIvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.42" data-doi="10.1007/s10658-008-9305-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Classical and molecular genetics of <em>Bremia lactucae</em>, cause of lettuce downy mildew.</span> <abbr class="cit-jnl-abbrev">Eur. J. Plant Pathol.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D122%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10658-008-9305-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10658-008-9305-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seely</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Transformation of lettuce (<em>Lactuca sativa</em>) mediated by <em>Agrobacterium tumefaciens</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D6%26rft.spage%253D439%26rft_id%253Dinfo%253Apmid%252F24248927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24248927&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L335900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-5.12.2655.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Bremia lactucae</em> and lettuce downy mildew</span>, <span class="cit-source"><em>Oomycete Genetics and Genomics: Diversity</em>, <em>Plant and Animal Interactions</em>, <em>and Toolbox</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kamoun</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lamour</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">John Wiley</span>, <span class="cit-publ-loc">Hoboken, New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.12.2655.45"><div class="cit-metadata unstructured">Miller, M. A., W. Pfeiffer, and T. Schwartz, 2010 Creating the CIPRES Science Gateway for inference of large phylogenetic trees, pp. 1–8 in <em>Gateway Computing Environments Workshop (GCE)</em>, 2010. New Orleans, LA, November 14, 2010 IEEE.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.46" data-doi="10.1016/j.pbi.2012.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monaghan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zipfel</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Plant pattern recognition receptor complexes at the plasma membrane.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DMonaghan%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D349%26rft.epage%253D357%26rft.atitle%253DPlant%2Bpattern%2Brecognition%2Breceptor%2Bcomplexes%2Bat%2Bthe%2Bplasma%2Bmembrane.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.05.006%26rft_id%253Dinfo%253Apmid%252F22705024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22705024&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.47" data-doi="10.1093/molbev/msi241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mondragon-Palomino</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene conversion and the evolution of three leucine-rich repeat gene families in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2444</span>–<span class="cit-lpage">2456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi241%26rft_id%253Dinfo%253Apmid%252F16120808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8yNDQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-5.12.2655.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicholas</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicholas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deerfield</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">GeneDoc: analysis and visualization of genetic variation.</span> <span class="cit-source">Embnew.news</span> <span class="cit-vol">4</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmbnew.news%26rft.volume%253D4%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.49" data-doi="10.1007/s00438-009-0434-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paidi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishijima</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis of <em>Carica papaya</em> reveals a small NBS resistance gene family.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DPorter%26rft.auinit1%253DB.%2BW.%26rft.volume%253D281%26rft.issue%253D6%26rft.spage%253D609%26rft.epage%253D626%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BCarica%2Bpapaya%2Breveals%2Ba%2Bsmall%2BNBS%2Bresistance%2Bgene%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-009-0434-x%26rft_id%253Dinfo%253Apmid%252F19263082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-009-0434-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19263082&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.50" data-doi="10.1093/oxfordjournals.molbev.a003984"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richly</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leister</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mode of amplification and reorganization of resistance genes during recent <em>Arabidopsis thaliana</em> Evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DRichly%26rft.auinit1%253DE.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D84%26rft.atitle%253DMode%2Bof%2BAmplification%2Band%2BReorganization%2Bof%2BResistance%2BGenes%2BDuring%2BRecent%2BArabidopsis%2Bthaliana%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003984%26rft_id%253Dinfo%253Apmid%252F11752192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjE5LzEvNzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.51" data-doi="10.1126/science.1189468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beutler</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant and animal sensors of conserved microbial signatures.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRonald%26rft.auinit1%253DP.%2BC.%26rft.volume%253D330%26rft.issue%253D6007%26rft.spage%253D1061%26rft.epage%253D1064%26rft.atitle%253DPlant%2Band%2BAnimal%2BSensors%2Bof%2BConserved%2BMicrobial%2BSignatures%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189468%26rft_id%253Dinfo%253Apmid%252F21097929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAwNy8xMDYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.52" data-doi="10.1007/s004380050604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schöb</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meins</span>  <span class="cit-name-given-names">F.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1997</span> <span class="cit-article-title">Silencing of transgenes introduced into leaves by agroinfiltration: a simple, rapid method for investigating sequence requirements for gene silencing.</span> <abbr class="cit-jnl-abbrev">Molecular and General Genetics MGG</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BGeneral%2BGenetics%2BMGG%26rft.volume%253D256%26rft.spage%253D581%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004380050604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004380050604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.53" data-doi="10.1104/pp.114.243626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Z.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hang</span>  <span class="cit-name-given-names">Y.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Long-term evolution of nucleotide-binding site-leucine-rich repeat genes: understanding gained from and beyond the legume family.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.114.243626%26rft_id%253Dinfo%253Apmid%252F25052854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjYvMS8yMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.54" data-doi="10.1534/genetics.105.047290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Unique evolutionary mechanism in R-genes under the presence/absence polymorphism in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047290%26rft_id%253Dinfo%253Apmid%252F16452149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzEyNDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.55" data-doi="10.1094/MPMI.2002.15.3.251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arroyo-Garcia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavelle</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>Dm3</em> is one member of a large constitutively expressed family of nucleotide binding site-leucine-rich repeat encoding genes.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DShen%26rft.auinit1%253DK.%2BA.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D251%26rft.epage%253D261%26rft.atitle%253DDm3%2Bis%2Bone%2Bmember%2Bof%2Ba%2Blarge%2Bconstitutively%2Bexpressed%2Bfamily%2Bof%2Bnucleotide%2Bbinding%2Bsite-leucine-rich%2Brepeat%2Bencoding%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI.2002.15.3.251%26rft_id%253Dinfo%253Apmid%252F11952128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI.2002.15.3.251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11952128&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174407800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.56" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.57" data-doi="10.1080/10635150802429642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoover</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rougemont</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A rapid bootstrap algorithm for the RAxML web servers.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150802429642%26rft_id%253Dinfo%253Apmid%252F18853362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjU3LzUvNzU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.58" data-doi="10.1016/j.pbi.2012.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takken</span>  <span class="cit-name-given-names">F. L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goverse</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">How to build a pathogen detector: structural basis of NB-LRR function.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DTakken%26rft.auinit1%253DF.%2BL.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D375%26rft.epage%253D384%26rft.atitle%253DHow%2Bto%2Bbuild%2Ba%2Bpathogen%2Bdetector%253A%2Bstructural%2Bbasis%2Bof%2BNB-LRR%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.05.001%26rft_id%253Dinfo%253Apmid%252F22658703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22658703&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-other" id="cit-5.12.2655.59" data-doi="10.1534/g3.112.004929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An ultra high-density, transcript-based, genetic map of lettuce.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004929%26rft_id%253Dinfo%253Apmid%252F23550116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNC82MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEyLzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.60" data-doi="10.1111/j.1467-7652.2005.00123.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomczak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Optimization of <em>Agrobacterium</em>-mediated transient assays of gene expression in lettuce, tomato and Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DWroblewski%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D259%26rft.epage%253D273%26rft.atitle%253DOptimization%2Bof%2BAgrobacterium-mediated%2Btransient%2Bassays%2Bof%2Bgene%2Bexpression%2Bin%2Blettuce%252C%2Btomato%2Band%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2005.00123.x%26rft_id%253Dinfo%253Apmid%252F17173625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2005.00123.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17173625&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227087400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.61" data-doi="10.1111/j.1365-313X.2007.03182.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskurewicz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomczak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Silencing of the major family of NBS-LRR-encoding genes in lettuce results in the loss of multiple resistance specificities.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWroblewski%26rft.auinit1%253DT.%26rft.volume%253D51%26rft.issue%253D5%26rft.spage%253D803%26rft.epage%253D818%26rft.atitle%253DSilencing%2Bof%2Bthe%2Bmajor%2Bfamily%2Bof%2BNBS-LRR-encoding%2Bgenes%2Bin%2Blettuce%2Bresults%2Bin%2Bthe%2Bloss%2Bof%2Bmultiple%2Bresistance%2Bspecificities.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03182.x%26rft_id%253Dinfo%253Apmid%252F17587302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03182.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17587302&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249424100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.62" data-doi="10.1261/rna.046532.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matvienko</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskurewicz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martineau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Distinctive profiles of small RNA couple inverted repeat-induced post-transcriptional gene silencing with endogenous RNA silencing pathways in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.046532.114%26rft_id%253Dinfo%253Apmid%252F25344399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8xOTg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMi8yNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.63" data-doi="10.1111/j.1469-8137.2011.04006.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tracing the origin and evolutionary history of plant nucleotide-binding site-leucine-rich repeat (NBS-LRR) genes.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DYue%26rft.auinit1%253DJ.%2BX.%26rft.volume%253D193%26rft.issue%253D4%26rft.spage%253D1049%26rft.epage%253D1063%26rft.atitle%253DTracing%2Bthe%2Borigin%2Band%2Bevolutionary%2Bhistory%2Bof%2Bplant%2Bnucleotide-binding%2Bsite-leucine-rich%2Brepeat%2B%2528NBS-LRR%2529%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.04006.x%26rft_id%253Dinfo%253Apmid%252F22212278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.04006.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22212278&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299778300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.64" data-doi="10.1186/1471-2164-15-187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pont</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salse</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Paleo-evolutionary plasticity of plant disease resistance genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-187%26rft_id%253Dinfo%253Apmid%252F24617999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24617999&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.12.2655.65" data-doi="10.1007/s00438-004-0990-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-wide identification of NBS genes in japonica rice reveals significant expansion of divergent non-TIR NBS-LRR genes.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DZhou%26rft.auinit1%253DT.%26rft.volume%253D271%26rft.issue%253D4%26rft.spage%253D402%26rft.epage%253D415%26rft.atitle%253DGenome-wide%2Bidentification%2Bof%2BNBS%2Bgenes%2Bin%2Bjaponica%2Brice%2Breveals%2Bsignificant%2Bexpansion%2Bof%2Bdivergent%2Bnon-TIR%2BNBS-LRR%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-0990-z%26rft_id%253Dinfo%253Apmid%252F15014983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-0990-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15014983&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F12%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221728600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/12/2655.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2647" title="Identification of Genes Uniquely Expressed in the Germ-Line Tissues of the Jewel Wasp Nasonia vitripennis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/12/2671" title="The Founder Strains of the Collaborative Cross Express a Complex Combination of Advantageous and Deleterious Traits for Male Reproduction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/12"><span>Volume 5 Issue 12, December 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/12.cover-source.jpg?itok=RngHqKQq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/12/2655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fB5lK9AvlwDpbyqpWNT8JySicu_h1ZR06xHPbTcdXeE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/12/2655" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/12/2655" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/12/2655" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Architecture of Disease Resistance Genes in Lettuce</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460194" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-vRJnazxH7xZS73HWRf_XnfMuRtA0KzUZX3IcbanDWx4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2655.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/12/2655" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IBEq2SuwOmdJe35soie2sSUMKzVKciOpKkSr0gsknCs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/12/2655" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/12/2655.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460194" id="node460194--2" data-pisa="ggg;5/12/2655" data-pisa-master="ggg;g3.115.020818" data-apath="/ggg/5/12/2655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Architecture of Disease Resistance Genes in Lettuce</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marilena Christopoulou</span>, <span class="highwire-citation-author" data-delta="1">Sebastian Reyes-Chin Wo</span>, <span class="highwire-citation-author" data-delta="2">Alex Kozik</span>, <span class="highwire-citation-author" data-delta="3">Leah K. McHale</span>, <span class="highwire-citation-author" data-delta="4">Maria-Jose Truco</span>, <span class="highwire-citation-author" data-delta="5">Tadeusz Wroblewski</span> and <span class="highwire-citation-author" data-delta="6">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2655-2669;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020818 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460194--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marilena Christopoulou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marilena%2BChristopoulou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christopoulou%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarilena%2BChristopoulou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sebastian Reyes-Chin Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes-Chin%2BWo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes-Chin%2BWo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Leah K. McHale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leah%2BK.%2BMcHale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McHale%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeah%2BK.%2BMcHale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria-Jose Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria-Jose%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria-Jose%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tadeusz Wroblewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tadeusz%2BWroblewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wroblewski%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATadeusz%2BWroblewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/12/2655" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/12/2655&amp;title=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/12/2655&amp;title=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/12/2655&amp;title=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/12/2655&amp;text=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/12/2655&amp;title=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/12/2655&amp;t=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/12/2655&amp;title=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/12/2655&amp;title=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460194/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460194" id="node460194--3" data-pisa="ggg;5/12/2655" data-pisa-master="ggg;g3.115.020818" data-apath="/ggg/5/12/2655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Architecture of Disease Resistance Genes in Lettuce</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marilena Christopoulou</span>, <span class="highwire-citation-author" data-delta="1">Sebastian Reyes-Chin Wo</span>, <span class="highwire-citation-author" data-delta="2">Alex Kozik</span>, <span class="highwire-citation-author" data-delta="3">Leah K. McHale</span>, <span class="highwire-citation-author" data-delta="4">Maria-Jose Truco</span>, <span class="highwire-citation-author" data-delta="5">Tadeusz Wroblewski</span> and <span class="highwire-citation-author" data-delta="6">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2655-2669;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020818 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460194--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marilena Christopoulou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marilena%2BChristopoulou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christopoulou%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarilena%2BChristopoulou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sebastian Reyes-Chin Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes-Chin%2BWo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes-Chin%2BWo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Leah K. McHale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leah%2BK.%2BMcHale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McHale%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeah%2BK.%2BMcHale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Maria-Jose Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria-Jose%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria-Jose%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tadeusz Wroblewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tadeusz%2BWroblewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wroblewski%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATadeusz%2BWroblewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genome Center and Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2655&amp;rft.epage=2669&amp;rft.atitle=Genome-Wide%20Architecture%20of%20Disease%20Resistance%20Genes%20in%20Lettuce&amp;rft.volume=5&amp;rft.issue=12&amp;rft.date=2015-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Christopoulou%2C+Marilena&amp;rft.au=Wo%2C+Sebastian+Reyes-Chin&amp;rft.au=Kozik%2C+Alex&amp;rft.au=McHale%2C+Leah+K.&amp;rft.au=Truco%2C+Maria-Jose&amp;rft.au=Wroblewski%2C+Tadeusz&amp;rft.au=Michelmore%2C+Richard+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460194/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460194/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460194/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460194/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460194/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460194/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460194/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460194/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460194/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460194/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460194/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460194/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/12/2655" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/12/2655.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/12/2655.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/12/2655.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/12/2655.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
