<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/12/3927" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/12/3927.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/12/3927.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/12/3927.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/12/3927.atom" />
<meta name="HW.pisa" content="ggg;6/12/3927" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data" />
<meta name="DC.Identifier" content="10.1534/g3.116.034744" />
<meta name="DC.Date" content="2016-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Shen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Understanding the phylogenetic relationships among the yeasts of the subphylum Saccharomycotina is a prerequisite for understanding the evolution of their metabolisms and ecological lifestyles. In the last two decades, the use of rDNA and multilocus data sets has greatly advanced our understanding of the yeast phylogeny, but many deep relationships remain unsupported. In contrast, phylogenomic analyses have involved relatively few taxa and lineages that were often selected with limited considerations for covering the breadth of yeast biodiversity. Here we used genome sequence data from 86 publicly available yeast genomes representing nine of the 11 known major lineages and 10 nonyeast fungal outgroups to generate a 1233-gene, 96-taxon data matrix. Species phylogenies reconstructed using two different methods (concatenation and coalescence) and two data matrices (amino acids or the first two codon positions) yielded identical and highly supported relationships between the nine major lineages. Aside from the lineage comprised by the family Pichiaceae, all other lineages were monophyletic. Most interrelationships among yeast species were robust across the two methods and data matrices. However, eight of the 93 internodes conflicted between analyses or data sets, including the placements of: the clade defined by species that have reassigned the CUG codon to encode serine, instead of leucine; the clade defined by a whole genome duplication; and the species Ascoidea rubescens . These phylogenomic analyses provide a robust roadmap for future comparative work across the yeast subphylum in the disciplines of taxonomy, molecular genetics, evolutionary biology, ecology, and biotechnology. To further this end, we have also provided a BLAST server to query the 86 Saccharomycotina genomes, which can be found at &lt;http://y1000plus.org/blast&gt;." />
<meta name="DC.Contributor" content="Xing-Xing Shen" />
<meta name="DC.Contributor" content="Xiaofan Zhou" />
<meta name="DC.Contributor" content="Jacek Kominek" />
<meta name="DC.Contributor" content="Cletus P. Kurtzman" />
<meta name="DC.Contributor" content="Chris Todd Hittinger" />
<meta name="DC.Contributor" content="Antonis Rokas" />
<meta name="article:published_time" content="2016-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Understanding the phylogenetic relationships among the yeasts of the subphylum Saccharomycotina is a prerequisite for understanding the evolution of their metabolisms and ecological lifestyles. In the last two decades, the use of rDNA and multilocus data sets has greatly advanced our understanding of the yeast phylogeny, but many deep relationships remain unsupported. In contrast, phylogenomic analyses have involved relatively few taxa and lineages that were often selected with limited considerations for covering the breadth of yeast biodiversity. Here we used genome sequence data from 86 publicly available yeast genomes representing nine of the 11 known major lineages and 10 nonyeast fungal outgroups to generate a 1233-gene, 96-taxon data matrix. Species phylogenies reconstructed using two different methods (concatenation and coalescence) and two data matrices (amino acids or the first two codon positions) yielded identical and highly supported relationships between the nine major lineages. Aside from the lineage comprised by the family Pichiaceae, all other lineages were monophyletic. Most interrelationships among yeast species were robust across the two methods and data matrices. However, eight of the 93 internodes conflicted between analyses or data sets, including the placements of: the clade defined by species that have reassigned the CUG codon to encode serine, instead of leucine; the clade defined by a whole genome duplication; and the species &lt;i&gt;Ascoidea rubescens&lt;/i&gt;. These phylogenomic analyses provide a robust roadmap for future comparative work across the yeast subphylum in the disciplines of taxonomy, molecular genetics, evolutionary biology, ecology, and biotechnology. To further this end, we have also provided a BLAST server to query the 86 Saccharomycotina genomes, which can be found at http://y1000plus.org/blast.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/12/01" />
<meta name="citation_mjid" content="ggg;6/12/3927" />
<meta name="citation_id" content="6/12/3927" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/12/3927" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/12/3927.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/12/3927.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/12/3927.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.034744" />
<meta name="citation_pmid" content="27672114" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3927" />
<meta name="citation_lastpage" content="3939" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xing-Xing Shen" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author" content="Xiaofan Zhou" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2879-6317" />
<meta name="citation_author" content="Jacek Kominek" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1916-0122" />
<meta name="citation_author" content="Cletus P. Kurtzman" />
<meta name="citation_author_institution" content="Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, U.S. Department of Agriculture, Peoria, Illinois 61604" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author" content="Chris Todd Hittinger" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5088-7461" />
<meta name="citation_author" content="Antonis Rokas" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7248-6551" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. R. Borneman;citation_author=B. A. Desany;citation_author=D. Riches;citation_author=J. P. Affourtit;citation_author=A. H. Forgan;citation_title=The genome sequence of the wine yeast VIN7 reveals an allotriploid hybrid genome with Saccharomyces cerevisiae and Saccharomyces kudriavzevii origins.;citation_volume=12;citation_year=2012;citation_pmid=22136070;citation_doi=10.1111/j.1567-1364.2011.00773.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Butler;citation_author=M. D. Rasmussen;citation_author=M. F. Lin;citation_author=M. A. S. Santos;citation_author=S. Sakthikumar;citation_title=Evolution of pathogenicity and sexual reproduction in eight Candida genomes.;citation_pages=657-662;citation_volume=459;citation_year=2009;citation_issue=7247;citation_pmid=19465905;citation_doi=10.1038/nature08064" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutierrez;citation_author=J. M. Silla-Martinez;citation_author=T. Gabaldon;citation_title=TrimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_volume=25;citation_year=2009;citation_pmid=19505945;citation_doi=10.1093/bioinformatics/btp348" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Darriba;citation_author=G. L. Taboada;citation_author=R. Doallo;citation_author=D. Posada;citation_title=ProtTest 3: fast selection of best-fit models of protein evolution.;citation_pages=164-165;citation_volume=27;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Dujon;citation_title=Yeast evolutionary genomics.;citation_pages=512-524;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20559329;citation_doi=10.1038/nrg2811" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. R. Eddy;citation_title=Accelerated Profile HMM Searches.;citation_pages=e1002195-e1002195;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22039361;citation_doi=10.1371/journal.pcbi.1002195" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. V. Edwards;citation_title=Is a new and general theory of molecular systematics emerging?;citation_pages=1-19;citation_volume=63;citation_year=2009;citation_issue=1;citation_pmid=19146594;citation_doi=10.1111/j.1558-5646.2008.00549.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. Felsenstein;citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach.;citation_pages=368-376;citation_volume=17;citation_year=1981;citation_issue=6;citation_pmid=7288891;citation_doi=10.1007/BF01734359" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. A. Fitzpatrick;citation_author=M. E. Logue;citation_author=J. E. Stajich;citation_author=G. Butler;citation_title=A fungal phylogeny based on 42 complete genomes derived from supertree and combined gene analysis.;citation_pages=99-99;citation_volume=6;citation_year=2006;citation_pmid=17121679;citation_doi=10.1186/1471-2148-6-99" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=B. Gibson;citation_author=G. Liti;citation_title=Saccharomyces pastorianus: genomic insights inspiring innovation for industry.;citation_pages=17-27;citation_volume=32;citation_year=2015;citation_pmid=25088523" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Hall;citation_author=F. S. Dietrich;citation_title=The Reacquisition of Biotin Prototrophy in Saccharomyces cerevisiae Involved Horizontal Gene Transfer, Gene Duplication and Gene Clustering;citation_pages=2293-2307;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073433;citation_doi=10.1534/genetics.107.074963" />
<meta name="citation_reference" content="citation_journal_title=J. Syst. Evol.;citation_author=T. A. Heath;citation_author=S. M. Hedtke;citation_author=D. M. Hillis;citation_title=Taxon sampling and the accuracy of phylogenetic analyses.;citation_pages=239-257;citation_volume=46;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. T. Hittinger;citation_title=Saccharomyces diversity and evolution: a budding model genus.;citation_pages=309-317;citation_volume=29;citation_year=2013;citation_pmid=23395329;citation_doi=10.1016/j.tig.2013.01.002" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. T. Hittinger;citation_author=A. Rokas;citation_author=S. B. Carroll;citation_title=Parallel inactivation of multiple GAL pathway genes and ecological diversification in yeasts.;citation_volume=101;citation_year=2004;citation_pmid=15381776;citation_doi=10.1073/pnas.0404319101" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=C. T. Hittinger;citation_author=A. Rokas;citation_author=F.-Y. Bai;citation_author=T. Boekhout;citation_author=P. Gonçalves;citation_title=Genomics and the making of yeast biodiversity.;citation_pages=100-109;citation_volume=35;citation_year=2015;citation_pmid=26649756;citation_doi=10.1016/j.gde.2015.10.008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. A. Hosner;citation_author=B. C. Faircloth;citation_author=T. C. Glenn;citation_author=E. L. Braun;citation_author=R. T. Kimball;citation_title=Avoiding missing data biases in phylogenomic inference: an empirical study in the Landfowl (Aves: Galliformes).;citation_volume=33;citation_year=2016;citation_pmid=26715628;citation_doi=10.1093/molbev/msv347" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=R. Hovmöller;citation_author=L. L. Knowles;citation_author=L. S. Kubatko;citation_title=Effects of missing data on species tree estimation under the coalescent.;citation_pages=1057-1062;citation_volume=69;citation_year=2013;citation_pmid=23769751;citation_doi=10.1016/j.ympev.2013.06.004" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. P. Huelsenbeck;citation_author=J. J. Bull;citation_author=C. W. Cunningham;citation_title=Combining data in phylogenetic analysis.;citation_pages=152-158;citation_volume=11;citation_year=1996;citation_issue=4;citation_pmid=21237790;citation_doi=10.1016/0169-5347(96)10006-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. H. Huson;citation_author=D. Bryant;citation_title=Application of phylogenetic networks in evolutionary studies.;citation_volume=23;citation_year=2006;citation_pmid=16221896;citation_doi=10.1093/molbev/msj030" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Y. James;citation_author=F. Kauff;citation_author=C. L. Schoch;citation_author=P. B. Matheny;citation_author=V. Hofstetter;citation_title=Reconstructing the early evolution of Fungi using a six-gene phylogeny.;citation_pages=818-822;citation_volume=443;citation_year=2006;citation_issue=7113;citation_pmid=17051209;citation_doi=10.1038/nature05110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Kobert;citation_author=L. Salichos;citation_author=A. Rokas;citation_author=A. Stamatakis;citation_title=Computing the internode certainty and related measures from partial gene trees.;citation_volume=33;citation_year=2016;citation_pmid=26915959;citation_doi=10.1093/molbev/msw040" />
<meta name="citation_reference" content="citation_title=Ascomycete Systematics: Problems and Perspectives in the Nineties;citation_title=Orders and families of ascosporogenous yeasts and yeast-like taxa compared from ribosomal RNA sequence similarities;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_pages=249-258;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Antonie van Leeuwenhoek;citation_journal_abbrev=Antonie van Leeuwenhoek;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_title=Identification and phylogeny of ascomycetous yeasts from analysis of nuclear large subunit (26S) ribosomal DNA partial sequences.;citation_pages=331-371;citation_volume=73;citation_year=1998;citation_issue=4;citation_pmid=9850420;citation_doi=10.1023/A:1001761008817" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_title=Phylogenetic relationships among yeasts of the “Saccharomyces complex” determined from multigene sequence analyses.;citation_volume=3;citation_year=2003;citation_pmid=12748053;citation_doi=10.1016/S1567-1356(03)00012-6" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_title=Multigene phylogenetic analysis of the Trichomonascus, Wickerhamiella and Zygoascus yeast clades, and the proposal of Sugiyamaella gen. nov. and 14 new species combinations.;citation_volume=7;citation_year=2007;citation_pmid=17311592;citation_doi=10.1111/j.1567-1364.2006.00157.x" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_title=Relationships among genera of the Saccharomycotina (Ascomycota) from multigene phylogenetic analysis of type species.;citation_volume=13;citation_year=2013;citation_pmid=22978764;citation_doi=10.1111/1567-1364.12006" />
<meta name="citation_reference" content="citation_journal_title=Mycoscience;citation_author=C. P. Kurtzman;citation_author=M. Suzuki;citation_title=Phylogenetic analysis of ascomycete yeasts that form coenzyme Q-9 and the proposal of the new genera Babjeviella, Meyerozyma, Millerozyma, Priceomyces, and Scheffersomyces.;citation_pages=2-14;citation_volume=51;citation_year=2010;citation_doi=10.1007/s10267-009-0011-5" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. P. Kurtzman;citation_author=C. J. Robnett;citation_author=E. Basehoar-Powers;citation_title=Phylogenetic relationships among species of Pichia, Issatchenkia and Williopsis determined from multigene sequence analysis, and the proposal of Barnettozyma gen. nov., Lindnera gen. nov. and Wickerhamomyces gen. nov.;citation_volume=8;citation_year=2008;citation_pmid=18671746;citation_doi=10.1111/j.1567-1364.2008.00419.x" />
<meta name="citation_reference" content="citation_title=The Yeasts: A Taxonomic Study;citation_author=C. P. Kurtzman;citation_author=J. W. Fell;citation_author=T. Boekhout;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Lartillot;citation_author=H. Philippe;citation_title=A Bayesian mixture model for across-site heterogeneities in the amino-acid replacement process.;citation_volume=21;citation_year=2004;citation_pmid=15014145;citation_doi=10.1093/molbev/msh112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Lartillot;citation_author=T. Lepage;citation_author=S. Blanquart;citation_title=PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating.;citation_volume=25;citation_year=2009;citation_pmid=19535536;citation_doi=10.1093/bioinformatics/btp368" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Q. Le;citation_author=O. Gascuel;citation_title=An improved general amino acid replacement matrix.;citation_volume=25;citation_year=2008;citation_pmid=18367465;citation_doi=10.1093/molbev/msn067" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Liang;citation_author=X. X. Shen;citation_author=P. Zhang;citation_title=One thousand two hundred ninety nuclear genes from a genome-wide survey support lungfishes as the sister group of tetrapods.;citation_volume=30;citation_year=2013;citation_pmid=23589454;citation_doi=10.1093/molbev/mst072" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Libkind;citation_author=C. T. Hittinger;citation_author=E. Valério;citation_author=C. Gonçalves;citation_author=J. Dover;citation_title=Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast.;citation_volume=108;citation_year=2011;citation_pmid=21873232;citation_doi=10.1073/pnas.1105430108" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Lin;citation_author=W. H. Li;citation_title=Expansion of hexose transporter genes was associated with the evolution of aerobic fermentation in yeasts.;citation_volume=28;citation_year=2011;citation_pmid=20660490;citation_doi=10.1093/molbev/msq184" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Liti;citation_author=A. N. N. Ba;citation_author=M. Blythe;citation_author=C. A. Müller;citation_author=A. Bergström;citation_title=High quality de novo sequencing and assembly of the Saccharomyces arboricolus genome.;citation_pages=69;citation_volume=14;citation_year=2013;citation_pmid=23368932;citation_doi=10.1186/1471-2164-14-69" />
<meta name="citation_reference" content="citation_journal_title=Ann. N. Y. Acad. Sci.;citation_author=L. Liu;citation_author=Z. Xi;citation_author=S. Wu;citation_author=C. C. Davis;citation_author=S. V. Edwards;citation_title=Estimating phylogenetic trees from genome-scale data.;citation_pages=36-53;citation_volume=1360;citation_year=2015;citation_pmid=25873435;citation_doi=10.1111/nyas.12747" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Liu;citation_author=J. W. Leigh;citation_author=H. Brinkmann;citation_author=M. T. Cushion;citation_author=N. Rodriguez-Ezpeleta;citation_title=Phylogenomic analyses support the monophyly of Taphrinomycotina, including Schizosaccharomyces fission yeasts.;citation_volume=26;citation_year=2009;citation_pmid=18922765;citation_doi=10.1093/molbev/msn221" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=V. L. Louis;citation_author=L. Despons;citation_author=A. Friedrich;citation_author=T. Martin;citation_author=P. Durrens;citation_title=Pichia sorbitophila, an Interspecies Yeast Hybrid, Reveals Early Steps of Genome Resolution After Polyploidization;citation_pages=299-311;citation_volume=2;citation_year=2012;citation_issue=2;citation_pmid=22384408;citation_doi=10.1534/g3.111.000745" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Maere;citation_author=K. Heymans;citation_author=M. Kuiper;citation_title=BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks.;citation_volume=21;citation_year=2005;citation_pmid=15972284;citation_doi=10.1093/bioinformatics/bti551" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. Marcet-Houben;citation_author=T. Gabaldón;citation_title=Beyond the whole-genome duplication: phylogenetic evidence for an ancient interspecies hybridization in the baker’s yeast lineage.;citation_pages=e1002220;citation_volume=13;citation_year=2015;citation_pmid=26252497;citation_doi=10.1371/journal.pbio.1002220" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. M. Medina;citation_author=G. W. Jones;citation_author=D. A. Fitzpatrick;citation_title=Reconstructing the fungal tree of life using phylogenomics and a preliminary investigation of the distribution of yeast prion-like proteins in the fungal kingdom.;citation_pages=116-133;citation_volume=73;citation_year=2011;citation_issue=3-4;citation_pmid=21938499;citation_doi=10.1007/s00239-011-9461-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Mirarab;citation_author=R. Reaz;citation_author=M. S. Bayzid;citation_author=T. Zimmermann;citation_author=M. S. Swenson;citation_title=ASTRAL: genome-scale coalescent-based species tree estimation.;citation_volume=30;citation_year=2014;citation_pmid=25161245;citation_doi=10.1093/bioinformatics/btu462" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Mirarab;citation_author=M. S. Bayzid;citation_author=B. Boussau;citation_author=T. Warnow;citation_title=Response to Comment on &quot;Statistical binning enables an accurate coalescent-based estimation of the avian tree&quot;;citation_pages=171-171;citation_volume=350;citation_year=2015;citation_issue=6257;citation_pmid=26450204;citation_doi=10.1126/science.aaa7719" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Morel;citation_author=L. Sterck;citation_author=D. Swennen;citation_author=M. Marcet-Houben;citation_author=D. Onesime;citation_title=Differential gene retention as an evolutionary mechanism to generate biodiversity and adaptation in yeasts.;citation_pages=11571;citation_volume=5;citation_year=2015;citation_pmid=26108467;citation_doi=10.1038/srep11571" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=S. Mühlhausen;citation_author=M. Kollmar;citation_title=Molecular phylogeny of sequenced Saccharomycetes reveals polyphyly of the alternative yeast codon usage.;citation_volume=6;citation_year=2014;citation_pmid=25053656;citation_doi=10.1093/gbe/evu152" />
<meta name="citation_reference" content="citation_journal_title=Mycological research;citation_journal_abbrev=Mycol Res;citation_author=N. H. Nguyen;citation_author=S.-O. Suh;citation_author=C. J. Marshall;citation_author=M. Blackwell;citation_title=Morphological and ecological similarities: wood-boring beetles associated with novel xylose-fermenting yeasts, Spathaspora passalidarum gen. sp. nov. and Candida jeffriesii sp. nov.;citation_pages=1232-1241;citation_volume=110;citation_year=2006;citation_issue=Pt 10;citation_pmid=17011177;citation_doi=10.1016/j.mycres.2006.07.002" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=H. Philippe;citation_author=F. Delsuc;citation_author=H. Brinkmann;citation_author=N. Lartillot;citation_title=Phylogenomics.;citation_pages=541-562;citation_volume=36;citation_year=2005;citation_doi=10.1146/annurev.ecolsys.35.112202.130205" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=D. E. Polev;citation_author=K. S. Bobrov;citation_author=E. V. Eneyskaya;citation_author=A. A. Kulminskaya;citation_title=Draft genome sequence of Geotrichum candidum strain 3C.;citation_pages=e00956-14;citation_volume=2;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Polzin;citation_author=A. Rokas;citation_title=Evaluating rare amino acid substitutions (RGC_CAMs) in a yeast model clade.;citation_pages=e92213;citation_volume=9;citation_year=2014;citation_pmid=24637883;citation_doi=10.1371/journal.pone.0092213" />
<meta name="citation_reference" content="citation_title=bioRxiv;citation_title=Sequenceserver: a modern graphical user interface for custom BLAST databases.;citation_author=A. Priyam;citation_author=B. J. Woodcroft;citation_author=V. Rai;citation_author=A. Munagala;citation_author=I. Moghul;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Riley;citation_author=S. Haridas;citation_author=K. H. Wolfe;citation_author=M. R. Lopes;citation_author=C. T. Hittinger;citation_title=Comparative genomics of biotechnologically important yeasts.;citation_volume=113;citation_year=2016;citation_pmid=27535936;citation_doi=10.1073/pnas.1603941113" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. Rokas;citation_author=S. B. Carroll;citation_title=Bushes in the tree of life.;citation_pages=e352-e352;citation_volume=4;citation_year=2006;citation_issue=11;citation_pmid=17105342;citation_doi=10.1371/journal.pbio.0040352" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=A. Rokas;citation_author=P. W. H. Holland;citation_title=Rare genomic changes as a tool for phylogenetics.;citation_pages=454-459;citation_volume=15;citation_year=2000;citation_issue=11;citation_pmid=11050348;citation_doi=10.1016/S0169-5347(00)01967-4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Rokas;citation_author=B. L. Williams;citation_author=N. King;citation_author=S. B. Carroll;citation_title=Genome-scale approaches to resolving incongruence in molecular phylogenies.;citation_pages=798-804;citation_volume=425;citation_year=2003;citation_issue=6960;citation_pmid=14574403;citation_doi=10.1038/nature02053" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. Salichos;citation_author=A. Rokas;citation_title=Inferring ancient divergences requires genes with strong phylogenetic signals.;citation_pages=327-331;citation_volume=497;citation_year=2013;citation_pmid=23657258;citation_doi=10.1038/nature12130" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. Salichos;citation_author=A. Stamatakis;citation_author=A. Rokas;citation_title=Novel information theory-based measures for quantifying incongruence among phylogenetic trees.;citation_volume=31;citation_year=2014;citation_pmid=24509691;citation_doi=10.1093/molbev/msu061" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. R. Scannell;citation_author=K. P. Byrne;citation_author=J. L. Gordon;citation_author=S. Wong;citation_author=K. H. Wolfe;citation_title=Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts.;citation_pages=341-345;citation_volume=440;citation_year=2006;citation_issue=7082;citation_pmid=16541074;citation_doi=10.1038/nature04562" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=D. R. Scannell;citation_author=O. A. Zill;citation_author=A. Rokas;citation_author=C. Payen;citation_author=M. J. Dunham;citation_title=The Awesome Power of Yeast Evolutionary Genetics: New Genome Sequences and Strain Resources for the Saccharomyces sensu stricto Genus;citation_pages=11-25;citation_volume=1;citation_year=2011;citation_issue=1;citation_pmid=22384314;citation_doi=10.1534/g3.111.000273" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the dimension of a model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T.-K. Seo;citation_title=Calculating bootstrap probabilities of phylogeny using multilocus sequence data.;citation_volume=25;citation_year=2008;citation_pmid=18281270;citation_doi=10.1093/molbev/msn043" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=X.-X. Shen;citation_author=L. Salichos;citation_author=A. Rokas;citation_title=A genome-scale investigation of how sequence, function, and tree-based gene properties influence phylogenetic inference.;citation_volume=8;citation_year=2016;citation_pmid=27492233;citation_doi=10.1093/gbe/evw179" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. C. Slot;citation_author=A. Rokas;citation_title=Multiple GAL pathway gene clusters evolved independently and by different mechanisms in fungi.;citation_volume=107;citation_year=2010;citation_pmid=20479238;citation_doi=10.1073/pnas.0914418107" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Song;citation_author=L. Liu;citation_author=S. V. Edwards;citation_author=S. Wu;citation_title=Resolving conflict in eutherian mammal phylogeny using phylogenomics and the multispecies coalescent model.;citation_volume=109;citation_year=2012;citation_pmid=22930817;citation_doi=10.1073/pnas.1211733109" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=M. S. Springer;citation_author=J. Gatesy;citation_title=The gene tree delusion.;citation_pages=1-33;citation_volume=94;citation_year=2016;citation_pmid=26238460;citation_doi=10.1016/j.ympev.2015.07.018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=A. Stamatakis;citation_author=P. Hoover;citation_author=J. Rougemont;citation_title=A rapid bootstrap algorithm for the RAxML web servers.;citation_volume=57;citation_year=2008;citation_pmid=18853362;citation_doi=10.1080/10635150802429642" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=M. Stanke;citation_author=S. Waack;citation_title=Gene prediction with a hidden Markov model and a new intron  submodel;citation_pages=ii215-225;citation_volume=19;citation_year=2003;citation_issue=90002;citation_pmid=14534192;citation_doi=10.1093/bioinformatics/btg1080" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_journal_abbrev=Mycologia;citation_author=J. Sugiyama;citation_author=K. Hosaka;citation_author=S.-O. Suh;citation_title=Early diverging Ascomycota: phylogenetic divergence and related evolutionary enigmas;citation_pages=996-1005;citation_volume=98;citation_year=2006;citation_issue=6;citation_pmid=17486975;citation_doi=10.3852/mycologia.98.6.996" />
<meta name="citation_reference" content="citation_title=Lectures on Mathematics in the Life Sciences;citation_title=Some probabilistic and statistical problems in the analysis of DNA sequences;citation_author=S. Tavaré;citation_pages=57-86;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_journal_abbrev=Mycologia;citation_author=J. W. Taylor;citation_author=M. L. Berbee;citation_title=Dating divergences in the Fungal Tree of Life: review and new analyses;citation_pages=838-849;citation_volume=98;citation_year=2006;citation_issue=6;citation_pmid=17486961;citation_doi=10.3852/mycologia.98.6.838" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. Wang;citation_author=X. Zhou;citation_author=D. Yang;citation_author=A. Rokas;citation_title=A genome-scale investigation of incongruence in culicidae mosquitoes.;citation_volume=7;citation_year=2015;citation_pmid=26608059;citation_doi=10.1093/gbe/evv235" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. M. Waterhouse;citation_author=F. Tegenfeldt;citation_author=J. Li;citation_author=E. M. Zdobnov;citation_author=E. V. Kriventseva;citation_title=OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs.;citation_volume=41;citation_year=2013;citation_pmid=23180791;citation_doi=10.1093/nar/gks1116" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. W. Wenger;citation_author=K. Schwartz;citation_author=G. Sherlock;citation_title=Bulk segregant analysis by high-throughput sequencing reveals a novel xylose utilization gene from Saccharomyces cerevisiae.;citation_pages=e1000942-e1000942;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20485559;citation_doi=10.1371/journal.pgen.1000942" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Whelan;citation_author=K. M. Kocot;citation_author=L. L. Moroz;citation_author=K. M. Halanych;citation_title=Error, signal, and the placement of Ctenophora sister to all other animals.;citation_volume=112;citation_year=2015;citation_pmid=25902535;citation_doi=10.1073/pnas.1503453112" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. J. Wickett;citation_author=S. Mirarab;citation_author=N. Nguyen;citation_author=T. Warnow;citation_author=E. Carpenter;citation_title=Phylotranscriptomic analysis of the origin and early diversification of land plants.;citation_volume=111;citation_year=2014;citation_pmid=25355905;citation_doi=10.1073/pnas.1323926111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. H. Wolfe;citation_author=D. C. Shields;citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.;citation_pages=708-713;citation_volume=387;citation_year=1997;citation_issue=6634;citation_pmid=9192896;citation_doi=10.1038/42711" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=K. H. Wolfe;citation_author=D. Armisén;citation_author=E. Proux-Wera;citation_author=S. S. ÓhÉigeartaigh;citation_author=H. Azam;citation_title=Clade- and species-specific features of genome evolution in the Saccharomycetaceae.;citation_volume=15;citation_year=2015;citation_pmid=26066552;citation_doi=10.1093/femsyr/fov035" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=Z. Xi;citation_author=L. Liu;citation_author=J. S. Rest;citation_author=C. C. Davis;citation_title=Coalescent vs. concatenation methods and the placement of Amborella as sister to water lilies.;citation_volume=63;citation_year=2014;citation_pmid=25077515;citation_doi=10.1093/sysbio/syu055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Xi;citation_author=L. Liu;citation_author=C. C. Davis;citation_title=The impact of missing data on species tree estimation.;citation_volume=33;citation_year=2016;citation_pmid=26589995;citation_doi=10.1093/molbev/msv266" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Yandell;citation_author=D. Ence;citation_title=A beginner's guide to eukaryotic genome annotation.;citation_pages=329-342;citation_volume=13;citation_year=2012;citation_issue=5;citation_pmid=22510764;citation_doi=10.1038/nrg3174" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=Z. Yang;citation_title=Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods.;citation_pages=306-314;citation_volume=39;citation_year=1994;citation_issue=3;citation_pmid=7932792;citation_doi=10.1007/BF00160154" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=Z. Yang;citation_title=Among-site rate variation and its impact on phylogenetic analyses.;citation_pages=367-372;citation_volume=11;citation_year=1996;citation_issue=9;citation_pmid=21237881;citation_doi=10.1016/0169-5347(96)10041-0" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=D. J. Zwickl;citation_author=D. M. Hillis;citation_title=Increased taxon sampling greatly reduces phylogenetic error.;citation_volume=51;citation_year=2002;citation_pmid=12228001;citation_doi=10.1080/10635150290102339" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="twitter:description" content="Understanding the phylogenetic relationships among the yeasts of the subphylum Saccharomycotina is a prerequisite for understanding the evolution of their metabolisms and ecological lifestyles. In the last two decades, the use of rDNA and multilocus data sets has greatly advanced our understanding of the yeast phylogeny, but many deep relationships remain unsupported. In contrast, phylogenomic analyses have involved relatively few taxa and lineages that were often selected with limited considerations for covering the breadth of yeast biodiversity. Here we used genome sequence data from 86 publicly available yeast genomes representing nine of the 11 known major lineages and 10 nonyeast fungal outgroups to generate a 1233-gene, 96-taxon data matrix. Species phylogenies reconstructed using two different methods (concatenation and coalescence) and two data matrices (amino acids or the first two codon positions) yielded identical and highly supported relationships between the nine major lineages. Aside from the lineage comprised by the family Pichiaceae, all other lineages were monophyletic. Most interrelationships among yeast species were robust across the two methods and data matrices. However, eight of the 93 internodes conflicted between analyses or data sets, including the placements of: the clade defined by species that have reassigned the CUG codon to encode serine, instead of leucine; the clade defined by a whole genome duplication; and the species Ascoidea rubescens . These phylogenomic analyses provide a robust roadmap for future comparative work across the yeast subphylum in the disciplines of taxonomy, molecular genetics, evolutionary biology, ecology, and biotechnology. To further this end, we have also provided a BLAST server to query the 86 Saccharomycotina genomes, which can be found at &lt;http://y1000plus.org/blast&gt;." />
<meta name="og-title" property="og:title" content="Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/12/3927" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Understanding the phylogenetic relationships among the yeasts of the subphylum Saccharomycotina is a prerequisite for understanding the evolution of their metabolisms and ecological lifestyles. In the last two decades, the use of rDNA and multilocus data sets has greatly advanced our understanding of the yeast phylogeny, but many deep relationships remain unsupported. In contrast, phylogenomic analyses have involved relatively few taxa and lineages that were often selected with limited considerations for covering the breadth of yeast biodiversity. Here we used genome sequence data from 86 publicly available yeast genomes representing nine of the 11 known major lineages and 10 nonyeast fungal outgroups to generate a 1233-gene, 96-taxon data matrix. Species phylogenies reconstructed using two different methods (concatenation and coalescence) and two data matrices (amino acids or the first two codon positions) yielded identical and highly supported relationships between the nine major lineages. Aside from the lineage comprised by the family Pichiaceae, all other lineages were monophyletic. Most interrelationships among yeast species were robust across the two methods and data matrices. However, eight of the 93 internodes conflicted between analyses or data sets, including the placements of: the clade defined by species that have reassigned the CUG codon to encode serine, instead of leucine; the clade defined by a whole genome duplication; and the species Ascoidea rubescens . These phylogenomic analyses provide a robust roadmap for future comparative work across the yeast subphylum in the disciplines of taxonomy, molecular genetics, evolutionary biology, ecology, and biotechnology. To further this end, we have also provided a BLAST server to query the 86 Saccharomycotina genomes, which can be found at &lt;http://y1000plus.org/blast&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
    <title>Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"isPtfoE4frJCeNosd2fAw1oP6JdK1oVNPOcQYa-eTHk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460506","apath":"\/ggg\/6\/12\/3927.atom","pisa":"ggg;6\/12\/3927","ac":{"\/ggg\/6\/12\/3927.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/12\/3927.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3927"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3927"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/12\/3927.long":true,"\/content\/6\/12\/3927":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XiCNxiCDfmx\\\\\\\/PiIvyMVPQT3qGNjlunNrNPdGB\\\\\\\/Uok+S33WegkAeVvjRK70+VWE2mOwCpwB1cXrPaqpjTReX+CiF+rkkvf4MxbIWzjLlJDjdOylPofZ8Eq2ecFpSKWIZ8FuPcmo+yBv0MmM3t0iGQZCCedFcCPIwmCOoQguJDshilRGNlkQyoY+VK4lgBEwtZRtc7C5bYkvk0QorBy0sGj+XQO\\\\\\\/uJAVcp5VnzbVuNOAW2hHvPm8JSYfowpXDsve3qpoZpuCV8ufrUNXwyjbSa\\\\\\\/16hLrVDrbNtpYU+KuhAF4nG7Jwqfq6j4lfQi9dVashpYxbmEEgtIebrnGr8IChsEFWOzvPbzENTXeSui8RcBa8n7JtdOBmIJ4cIeQmx6b5d9O7EW\\\\\\\/QkP4wqxLxl3CFyG72uGswrILdK23KwxzJj5pGfn\\\\\\\/UOhqyeU1wACAzWBm1OearWY2ZAcgyv5dEOvQn\\\\\\\/mgK3prSzcTI1wu+fBg\\\\\\\/EBHdtJlxY4yYYHVIycK1BWz20hRZwxdV7lA3n\\\\\\\/hezDkj+z9hqU444fuzY1tJOt9+yT\\\\\\\/ZIxQn6qnb71iBGgdBSwlCiNeHBkzD31KqR3FR4QSv3GhOy+OhDFXfky+q63zsayTtH7es+c+6bFSG3g3\\\\\\\/U8QGktKg0fp1JrYWg9ajoN1QALPINRaI5HLbktHNzRn2jcranP+tgACzgxnav0ONjFt3w0yOeR\\\\\\\/8R2k6CREdRaO4p\\\\\\\/y8tl1SYlu+fvH4UOxnZE+3PM6BE4sd9amvSOMB8G4m9Ns\\\\\\\/ikT7YmaosAep0p2tNHWh08IlfzJdcrUCbLQp5+qvAoAMmHZKUDlC2NDRUdL+bILYqQTRQIGs0t9GdJFrd2qkYVoGDzakXZLKo8gCn0lZZ6iBa0\\\\\\\/ji99pXSkDcWLN6TwiJLOiEUvHZnKJU6dTcskSr0JDk+1mttjHtuL5rB7NHVqTvN2ZjVqu+53quZKMI5zvrVFDpTERnwm8HWYF4IwjtBJ7cfwTdRaZfVlsqB9afmKOLdvnVMUXvv3eCgCTi3rzkMhjZzCKd55ZxQDf0NtQUGvPAbmU+UH7Q2nDPbN9zb1vIaPU4fbbWcgejVRwvqrvdmxnqWURWRoWpOdxYcvAr4bhZMp2xa6+7Kjfcd+UYtT7ZlwiZGr64cwZQFxH5NPGlXg0GvOlvLiYMqR\\\\\\\/t6uI42J0unJc3NeMgakUHuHVt9GfaUOHhdCwMNOg0lTMpH2fC7SeNDn+Po1gNbadMlRbReAlhCmhHxs+fOSua3WVrNclhlgmlJHMlKjwPgT1c\\\u0022,\\\u0022iv\\\u0022:\\\u0022ei4hb0HenVxh6ks8DuHsMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221d2777421cad7755c8153a1e1b37b220660f40c11fb8f7ab412f07745fe50277\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LI65kMaJgVu+Lri2Y5pQQr+6LuDBmpD\\\\\\\/ZAMejHb0Y2UVo2J4H63nKu4VtNSDRFyjrSepnIrUeHPwtBS3AMofKs389Wz8j4woNL\\\\\\\/Tqr9QZe0GlV2OgVxfuVCI700Uq3YRhRDayHYdYBVz\\\\\\\/O2z5Vw51jSW5pY9qaw09lfAceHjm5rLpV0UPw74f3CdML5YtS04wQBBjOb5E9skQNKQNqbBbaVx\\\\\\\/CVZGQy\\\\\\\/6X0eURfejJsZDG4eVwtv6sdsVA7u7Lb\\\\\\\/b4gpc0xhrlDoA3cXBX8eBxp\\\\\\\/w+IKxObZCBr2EZLuhCPiw++7uOBZbYY8NmHtxVK262QcG3vzx6uzCWNQtiXIXq05FKC3LPAU0FCi6uKVmuh5n5S1dlg6bDpFLdTemM5d0sDiw5eb+anKvxvJv2BczmzSR6pr8pALPjMEB3Jh8RjtgJOdS9iw9bYEgDoQMCgaKHaRBRulLEFn4jfzIwegceWX3OatGwommJ71OOblDy44oJVrqFKgm81LJzuxdHxeMUR310kY16T0jz1Vt\\\\\\\/EVSOptb2JWGNkVQT7B9B1XinrYCgIcDTIFVbQD32FWKBMc0CvpwxVqeNvTaL7lIiXyPr2UroD35UCDoD5zJhMQT6WYtMzBXT466cOIxPWg\\\\\\\/dYiRQXlmpHjS866n9ImTa5l8pgbE2hlh2sgctNwzsP1xEXykppWeku40KiRrqHS\\\\\\\/NiES4pTwECmV6HR5n2npdl9kt4eIHtEoDv1WyX9EuAZmIoW45LZ0dcWh649YB8QRuTYm5gHM90kZoc5eocpr8TM3Pyz5R2yuRASud\\\\\\\/st21sK\\\\\\\/4fdTt+0QbzhI10ukoykgTxQYXpYkLH+hATuKpuoyloXGUFzMt9VVnuxdaeAuQ0ofhNcXcmQdZWT7fkL0VyKYiGXKj6saJJucglHOHSXrQmGSxsctMXNSOcQyRLNz+owj7TFWfXFwomow4RV\\\\\\\/7aBzZ2fuZdIqJTmyBW2Mofr6DpwtHYiNPvryigyIus3dwvcaBPa25o8Tzn2zAPii1g0pTtowK3eT8sw1WqzpY01JD\\\\\\\/SH00BgRW3gTrFLOCWDbz9CJzqnGmmqxNyNkSNqJvVvBFNhAsyaVoHU1H1xN4wzwTHswtvzOaAa25eixsFL5UgGlhtNvRvXQ0BqJnAnX4uclyhtclK9+1COTDCzDs3NXXA5wfQS39l8P90sq4bjn3SETVsH75LSvX885QyQ7MocwCEqBi6RzW6fxWoBfRlH0ytQHv91cx5644EBj+1TN6Xbqo94zCVS7wD2ILuYXcKHDsa2mre9k3yT6f1Vg6dAy\\\\\\\/wgcpQb4e3ttIIMMuVz1Ni7svY7+TZSMa+fTy3efXrT2REX7F1nHEVAOTvtvuWkrdg+wmf8AxrJSapRJBIzvQw9ECWCY1245r8NKsIoGRsSsM3D7WVQn7z0Y170HpPw3DEWzJrwr69jBJkPoxdDPaByGu1ViN5uNNSvzIS1+XGMgV6lVYkrJ9Onk9HGMpAzehRvvegJRr6kisCw\\\\\\\/mDMja4F0exMMfrIsHA0DipL5lGfWtaq1Ro9oTCiwSiuq\\\\\\\/S8texb5xeu3I3VxvaWCfEvWtkOwm8qv0U0NPrRwIu9pf89OpZDv9NHEkZkf8Z6ObAR1jXvlsOismbksvHUSyhJ7FwjYGlI6BlTi+H7Fdo2Tpo7V9QcvfQaTqklD85Zaz2DLTjoxw\\\\\\\/hYopXf+IlRe5yQKolJ1zUd9VxGPUlDjZttQj6k17hTdPV+ePgyaQqT0GpHa6eARv8Z\\\\\\\/VqwKTRTx0ztFhMmggGU6FtPsfeSmDUJnt+Hx7juOXQmeYActcj6VLLJ8\\\\\\\/Y\\\\\\\/bl46IqM1kIAqO9JxnfZNI1VjZQQj8ptG73cVfbGg3hPXcuVE6Ez1Xur\\\\\\\/qtIL2GCMbp32G+Qf4pL2qlIpEuDjTQISdGT6ZqGTt+xWXCZDZLWjq+HVcSZDvC50FO79VfuyVH+ReuhYfGby19GATXxxr1UzIP3x7\\\\\\\/E4qePv3b7p\\\\\\\/mMIzylHrLKL9rRYI0OGf6\\\\\\\/OQWRqKqIDUCOCFyj9OIKwUZueRVVNhi44Bv2eWe4qJpurQaiRI5pyh7sZFLIYHIzKASzbT87reo\\\\\\\/mzw+4x9M5nQsgfT7nBQ097O7\\\\\\\/syMGMMWosYptompBETjF6BnUMFtAv13jU1qP\\\\\\\/BfvNNX3VUwxfZc79i4QIqmWmUy9WN34PSa61ehnyqnO7PP+5GhHlwxjsgWTX\\\\\\\/zSnMGnn\\\\\\\/2XN1GSDybChq7kIpUBJVh8e7Qfladi\\\\\\\/JWkAUVUH7SAgOGAcm7jkI7DcdlygTDqIG6zANM1EGh0x35Qg3mWtF9Qi4VipTAbhKcdxtvvKcMz1b4Ku5mrpYwvFKFWnXvqtkivovfYR8HRzqdvWdLOonpI+hXd1Jusu3z7c0Ehge3XAxb1hnFBclt0EK6Syq2z3u3tayG6Q9Rw5CeAjS0BZW3gIk+xR9GvBZfHsY1s0OtjCT6ps+s3hk+Qbbteo8ceNs0Y0JCnTbV39jR7bOGfwsrr\\\\\\\/cbpvNtAKLhmrVt1LLzQR\\\\\\\/eleGigusA0\\\\\\\/tLpC68FFy4RrKHA9DrpnhtFlXzH66zpYKOpmnGgQuIzp+ahI5bZZT1KC2Y1zDrwVx1A1Uzyh\\\\\\\/YBT5WW2GO32dQi7gYkCIAZlIUPA5GzSVyGlB7P\\\\\\\/VYW83n4lv1P7uJHs4UShJbYiz72ffiVFNmylDENekj+6clGvuS8rSLLvB0AOg5SeLy27KlF5hVD4TBocuP22j41igSzjuMRoBu0or8fSf\\\\\\\/qtuzZdPjyVDRd59me0qU9NFKZLwYT8HdqJfGLj0H95z7KUHZyLY13ND+gqHXI3XkP3XfK+Kf9d26P1TSpYtBhBHkG5bHpy\\\\\\\/QkhPPKN2wvJ01JLBWqnv0Rnomgp8+\\\\\\\/9BGGkATVWsBkYl0prn3Hq0IWSKZ7km3O3rGmdIaOgu4ggSdBGxwQPZrdaAj+UyFYrKOaNcba8nUmFoW\\\\\\\/nKDzk5UF70jFw1K0NIj0dNOtgwsp4klo+tBZh\\\\\\\/evyB2QPdo7bVC\\\\\\\/ItArOSzPMGydNyTC\\\\\\\/XUUYfhjdH5RPWjkUyZYhVwd42menI8zE7C9SM\\\\\\\/ObjI1x4M2oi\\\\\\\/KrMLlUmigIxp6HnIiuC7G0u8yH1js4XRif5p9G058WO56CJP+xdtRcrxPt8rAUuOMpx3gnTsGBhkl+jutCt5BnKo9PPQrMGJCbZqhIZa+aDlUfLpiAFJ1vKwAcpa8lb+NfdqH4\\\\\\\/Bqkk6swJi9jv22Kbw+Tq5Ow7X+uGwI2C3tYRv0tTjxy+4WY8\\\\\\\/mT\\\\\\\/z5eF6s9BQbLtiTSJa8UPZHk3qBBBviib2b6bEHizCwkbsgpyQob8Cj9Dd6HdFVODi+z\\\\\\\/dnTvIB3+dgi1hX5TvxZJ7WlQGuQyqGCRocCZXMEz0gYV4SxW2xS+yJdhFusAPSCS8YURV224K9imYYNpWvNGfJ\\\\\\\/fkr7sCG4UMh4gy8WKnDYp8xWr+9mQPE4ifbtM4HCcXCQofoLeZkbqBwGLEm6StCeGV27yxkgW5AFymg4qLpmoc5ticKvjXDzOEziARI9sO2SCs6ZYeIlphcC3JMf0BD2IEJ0RUajh1tHN\\\\\\\/CKqtciYuypq80fAkVZAzIoz5gdzCyrsCX9cWLgj\\\\\\\/PH+ramKbpYtred1H7jUpTD52vVYTLrJXTXQfYeV+6H+yNuqN+NoL0nex83E83JJwGXWQzh\\\\\\\/ce0f+bGz9nDfJkAtkNVuOOhsYjidqOM26nN5463ZqClBkHSOmZ4lItRzf0jd\\\\\\\/j9IQcYdaRgNp+ZthwcLJPUNPlgydHAixo49u+mYG0L5HkkXAkG8TBXFPiyis6pNqYqXZksk9RVW66Cak8ZtGGobL92VwCSBCEjXORNBsezYhRKNuaaDc+4I6ySJC8Lwa47EwGXBMy01kPRcTlVyKgGd1V16eSCsab1veG8UyRNr2PRML+XnbE3XYAgW3py0eCapuWhN3SyH\\\\\\\/+XNI8PV1BjoZvxNADn7pH5N9ekSzAtd0BxLXeMHhh5dcf4zjfw2hAUFNtzYmmIFPaC48q+eim3H75HWwD7Yy9H4g7kmB8t2OCBPrxRMtayO0Y4an8gvlj1dck1DJO2A0YSC1wt69abaiSqOHNvSs4PE\\\\\\\/u9DyS6xlRlMs+dFiy\\\\\\\/+hsXSjvx25gwGATiR6LUQn34zkXmqlFAwalzKrmsRRXCz6yzqPyc6KKd5VWJCl5BBC4TU091lhoUFdU8tL49GKyOl\\\\\\\/9NMfUI84aZITaAoT5BQ87pMEtssn\\\\\\\/hZZ7pZuBYp0DstLsKjokipwO6fSmS77S5E5Kk5ELVkYr4dmufklMc5+o\\\\\\\/sRwOSCS8e43RtYNoAzeNylBpm2IlBHJ4E5yL+Y9G4N5anmbxV6ZxdCyDtqAEx\\\\\\\/6nkBTLsHSVKtT1kjbJwvxH6qRI3eMzONmC9j8P2q4DIcCdZMf7b6kdZEJd8r2ssUtuYq6f+3B2s21IIM0lfIa613z\\\\\\\/Jmq58vT3rv74souzgU40rxnqnRGF8beagKMUYocvpJ9PIi937YJ3p8RCwlE5QlluBVwh+u1TshBU1ZR6X8W\\\\\\\/gpMzlqd0Xsi7tC\\\\\\\/Rop85Pas+ZSYdx9LRPs9w3F\\\\\\\/PRf7T2bqCQh2bERyL4gvGtJwvvG3Po6j284DFYZgwdXyyVXPopbcGOwReIKD5Ol5kalPMwjOeatquIB1ujYs5u32gibZHd\\\\\\\/gTB7VDPq8jNomuCH7WX1m4fdjkZu1B+7CbszXEgTHNnX7fTPriF35Js5ivQOFIj1UcXvHP9IKqRLrM5ZKFR8Q7jpxGq4ZONXAIpupu87y0Mf66sIbhiwBW6xnlvECBgynunlAGqtt8N8guP+bO3\\\\\\\/DSIQCr8xenuLTgIvbkeqfYahY8EULdQgbKk6nhh2DDt4Y5HH3H0eupLUJI9AUQJ5XH4IWy\\\\\\\/JW3m2VGv7CyirOIttvT0RYt7Ic2KySzBg8e2HeDYI5S6pUoDBNDA5QvNr+pTifxTChIuT1ADzufchxxcVQkzZtzT8lgvxzGVqMPYEm8D\\\\\\\/w46TZBEeTHHxyZXQfBKOh1ZBRMYMQC\\\\\\\/VKuCfL8HBmHB2aqT3VByDETTuUYZ7Ah+CHHXlaeIaPvXcWw1s31l6YfE2oY++pFmsruCuGOkPUpKLub6XlL1W2YJdf31aXplghFwUFSP3A+H1N53cjx0vlpHHIdD1OWvJYtRwHRawRwWTMseeyADQbItL77FwWskFWhhf380KOysnESRIqq7wWto6m\\\\\\\/jDmdrVwP41uE1lV3arSM5wKvk92sX6ihmNseUlllNtz\\\\\\\/1mZVCJ8wl\\\\\\\/AIhrrdWKtgmkXCMJnODwpVKvkbAU28oMUfeViBNfQ3uJXDRqwVZnkvUVN4fddbIU6i9VJqk87LUptGSP+CRmxs3MgRLKzH6VcvaBBFazd429uJOOKSzcNEtToz6VdnBDnPKEy2WQokdBywp6V9GUH9hQKAQTjXnR+KZO9vdKgizW9czQtIELZxGoej\\\\\\\/QDNHmu6jb6Z0CtWPOYZ9P3Zmj51ao7Wujvd0VPIoHUvX8LPIcvQFMniMOLxgiGJvIxBOJJb9KqabLbuXxwZfLLbzvHWxGit5RE4\\\\\\\/J91S+FGds7WHiRylppx4wSIPbsN453NwxmnotjUjf7M3c34OMsfHRYIX898WXxu9SqUCE66igySXuCNIW3TSoUu7fDhuQlzB0MzYuPGbXI5T0XKJli1GLUIOeOwFUio7fgwymCl1crFZpXY837EQ5ulNwhe3Gr8sTb1boKF79e7MSJUZr3UxJxy3jbaTQzC9YaZYXS9lMii6DHjtUYVzcow2Vd5b3W2\\\\\\\/bGYNNMv6gJVJeRAvQQblwPHAwl+6pRfDSS+zD+Qmn1tSmAGaCNF3kJAgCltYBi4\\\\\\\/6oy0MMAaM648vZgmVEZIcL5QPjojXe4FkYi4Hue1VRxPVfOUNhii1RbyS2G9rICGrGHK5s5rhSi7lf8lMZJx+4q8GiSoKov1VbmX4nBVK8zHKPLK2zCgVvK9SsZsdRXzR\\\\\\\/xRJd7uiuCZgtYIfRSzw4pmTiheNv6\\\\\\\/JFLYeNqMLBkr96c4sVKP7IjBl9J\\\\\\\/cvl9dZlHn88opkyGtafo+Mo3pZLsjT2RUYDiKOhVvYQNPsPa2SNhGUZKaw8++0ox5twvtMpdqFEuoo8HMh9AvN80rDzI1zRy0z7mlS\\\\\\\/CdC9tXZUB\\\\\\\/hQ\\\\\\\/CUIHeo2E1SK+VtI\\\\\\\/glU8ud3BdXZ11XFkaxYDSw+v5ZdGkC5SZzahUk3nIPnq4jCVka\\\\\\\/JPKxp9BxBj+9anIGt6bJwISSKhcfv8OW7qsuIH74\\\\\\\/RLtaNaxkCI42I9pOs5h7YAqE7ZuCF32S0Tuv13e4CapdPKJaz\\\\\\\/XTkWs8HWdEwQdDyLNYsk0A13uSVxl3\\\\\\\/87KNNSLf2y4bJuAPafz+L7bd8YG1VKgAdpjzks2ZDH1\\\\\\\/7XKh24h2zy7c9+LW8TpkGmlbz6kjkWglDlQdjZnpWzARU+jmzj3Rb\\\\\\\/UxyHzzlTmrMadn9mRpqh\\\\\\\/0TaT9k2NYQv0PYx6feDmzYVv1mEV+mw3B9\\\\\\\/6+Xa3OdonxhTTXPHUgDYsXUteEMZCfHkAFYvTNvD9gXDpKyCiaBNwCwAc6ONg\\\\\\\/MCi1PD5aU5TD\\\\\\\/L5ZP0\\\\\\\/WJO4R\\\\\\\/U9o4xxby\\\\\\\/+ZL\\\\\\\/PS\\\\\\\/AazwpncApe0WVIZyC9hu7Hg2VYwBl0fnuc1aDBUEl+RpaJyHp\\\\\\\/LSVJ0jfnE4PTgqcM5hfVJ+53jH\\\\\\\/WGJXyfMiAysHlXeWqckx\\\\\\\/+fmVLI3lA51UOL0iIMcyT4HNWMLmBQ+tc6CCBNdenjfeOyG7t7BpesmkKEsXDsoQ45PddRbaijcMJzSRf58P9+KaX7Djl3xqeXJW6MHo3ISI199Dy\\\\\\\/jb+ZBXh\\\\\\\/5cNel5ppyZJOUpVLddmOzepn7ywsP\\\\\\\/bH+MkV4Tk2j77X38rXVuVQeHyPvpumEfGIie5BQWtz75mjabKYuudC3Z2sVx1PBlc3nXe29Tw\\\\\\\/FiOJHFsyKt5A+T8Sg+OOp8aUpDYKlVzAHd+GQA8geXlUG2\\\\\\\/FNrgi1aMF5EryUjm5Fhh+QJzAjUxUajOhrRLJLcl\\\\\\\/H2hiQyATTlNi7QSHvGscZdQljCszI\\\\\\\/+PllIWLKfA0teozmYdwpDQFbStU09khY\\\\\\\/ngUIFoRObS7RDl6VmN6BWAIgQ5mvCwnbIcgEPPT9z9+dXgvOCDmvFp4SRGoAGZTS2AkpR+sWYeqCAcGaBI6rZoQOKwifp6I\\\\\\\/yHOjp0LgsZ8qu\\\\\\\/NmeVB7RlFrhCDFgU6P2BYybwlnwbk\\\\\\\/4KEtpX8OOVLA7afj\\\\\\\/7iANeMlIqAxBHTau1kyYN20xITe8Sg1EHzIQWcoS5OKGDujMZ0Ugx0209GmgVs\\\\\\\/FvUbeYp25DRxEaGPEGCgN3A\\\\\\\/Nl5QRgpr4\\\\\\\/Y1DbuNIAIEOEwMJrfxXDrhvDAtynIA2FzSNyVhsLt4S1SkKIUi0onKZJr2AMDIbUhwJs4fASopTxDx\\\\\\\/Hjo5rNAYU5YPf0jVers\\\\\\\/P4icyd\\\\\\\/izgViPA+myAKC9iHswiqn6AXG7jtsZFgwB9tBlvxzyb6f+rvjIcDwFIKIP+6u36HYeJIACD6+YGu1SKrXzsKjGZIAEoDTEjk8VVhi8MChNnE3jPn3ZL8G5EbXGjy4Z7HjFf3VxzVGkzpHuJ4om+yVwf8hhYtP4RbDYBTUK3WmYwgNesrEUNxprchfvj99Pjqz+EBXG0U9VfMpg9m+odU365fTt\\\\\\\/gqjVOC8q4ecENkD0t+D2AHV100uPJnF8YL9oN+Mhdmf46JpT8p4NhnxGeqFN3y6y4Kiklv2bw42w2D90MuY6AP6FlKucdw2ObN4XUOWnU2sI4xd5gCwmW2OSF0HbQP0Wu8Uvn3dJ7Ky6r\\\\\\\/Xbc9oOcJQebAsg2NaY4kdTQ\\\\\\\/1s+XxQpqSr+Vj6GGg8sIfXQ96tbIO0WdqCW9FWUfiQt1VQTfMk1IzJY\\\\\\\/dnhxVkXGwejBCpHEI2Xv0azTuBkxGDt8YDZKraqDVymePtaUryTgI32TMCrMw7tHRQZuTvi+NOgoZqm3AvkeYGtjh1M4Q2h2S7ZRoUxRFdMGjyKXG9\\\\\\\/RehjJvKfeIJzsMaTWk\\\\\\\/RIBWB2J0q1WeLjpl\\\\\\\/AKyH1HzpvSznKgMw12zgIMAgyAEM1aTLIq24Pwgma\\\\\\\/IXI9hzr+ICtFmBaXROfjN2+u0jkFRtHHQwK0eFnD4gPnvbXJGi8Gj5VqDqQIn82Li8DDi8bH9FJxJNrGEXmdKxJqfRbcWV\\\\\\\/WabvBhkpyiYWCtpsiQs\\\\\\\/219E6aIa9RLFGqjZF7\\\\\\\/e1MCttDQf0xaGM3MaR8XL8NQT1X08UzYpjoZuIdnzRfWNkmQD3huMw\\\\\\\/vFQ6oJK8aLAF4zFMRnuVlQCksIFjdcVeEjNl47ZRxG7mgroNhHgU8GI2yagMExkamccE40dMPjaYTZ09axTvvAkMn8pPeKHer1db2VwNXeNg7lwcSKc7f4FwlK23kOHSeHYKX0ru67h6PtXI2S9r77y53rx2zM8C2pFp9t5YhCe7BdHe9M5\\\\\\\/LSaKrcSEpW4ePNWRU6rJnYqNgu9y6qEKGJzyQkH0c1HM74k8WkvN5Dgmx1tH3M2Sq+\\\\\\\/lVnjxlvU5WseFeilPYyod9fMvYuKht705mlu7jDFFGa97PVC9JP5kY4FHyTGKywUMI7xN0ehCjr8+dEZuWhHKGsIgQoy\\\\\\\/mEUPFwXtIUdcTJHsS+bxFFNyksRSDLG1g4TBQth4DIrt4zUwx0L4daYnqig\\\\\\\/o37A8luHw0mS6Qg32BDCaJgsY9fceblSSoPcJCGqgjh6tTe7R0PjKMlPIkX0p7pvCjL5yE6ytZ43mHsX9iAnAUpIRe24UUmbpayzaN5b8CAVdsKixWQsDBZTcfuW3Vo0gdQGKNXd+3SizJxm9oK5GQEzW0vd7R+7NYW+akFGbK2XZJ5aGrxFkUcF+OX0pTFP9VpavAwrvPCGhURpjRAqKUUbZmtP6cdFzvYg7s7+bBHDeaQqbSbHGdQNypFiYNF7TFCJ2YqCYymP4eWkNu9k98MgaRRLtrXuIp7M3K3EcbYt\\\\\\\/5a\\\\\\\/f9K5v+79\\\\\\\/fSb9Qb81VUd95NuBBjkIYe9qSxCYsJ2c57D3+mg8FXVIglH5lu3EO4uT8lL6U6zivGo7IbGaYD7rog3nu5AC9pQCvbAyroLjKljHPt9xfws0+3IRptqL7JZj4b9kzzYM6CBV7QJ6TEh\\\\\\\/mQvgM79PNErCwgLu5J57E239wqoF6sMAt3H6xBZqOWhZjxNaZ46VBedrfdvcELsERusNxwi\\\\\\\/lJSuyevLry9OEi26dbr7cF53UouNvI21FO6g8xYmL3RmDo6jDv5KPwXoKywgO4PK1xCOMehWyXlkK\\\\\\\/x3fxU6btZSuUtNyb8iaWNG1sQK1DUUlYcfVW663GMfB2o491tl7E5hMY7l\\\\\\\/3b3lvdwZTeSnIxx2Rq27RdNk9fseWKg4MyayEzpC7yOPuAxQgHzsF8h6shFZsaqjOZ\\\\\\\/0h+CJBM3cia+3aVnreh9O8cAmBdIw4ENjy+k5U0h0YAQrgzmAiGOy8IeEgB5XIIoXD1tuWG2ieb5XQnN108fyPK1nfZDutqLl1h\\\\\\\/sjr+KKKmsg3lHR2FjriH6K1KU2Wk6hp\\\\\\\/1ZWdX9P3bGS4OgyE3BlqXmxc\\\\\\\/fjV2\\\\\\\/R7Qgs2z7pD9ni4Rf\\\\\\\/EyV0HR5RYeDPPxq8PkCVWbqh2jDmhqPGgbVzdTKBO3vVDzHie3CVBtwlxr1GU+H5E\\\\\\\/Zj5bsstEMVYCxv1i3BIB0V4VYXmnxmPwVhxUdR42qWCni64Q99P2RwqdIq99vZ4IjihDpqWI86YpL+2Mc3cChbbp3XiZ0IJ7ah761SvAk6X66SDih8YML6dRe5Q2wNVvT700T7+\\\\\\\/2OH96cPOi0Slm5nW8Br9mdiITyNQXPt8PogNBhmrxPqz8\\\\\\\/jnyGahNoVLioysAI8jS24ZCc5UP8v388KX5tteYC+I+iMLrwGgvZL2Kb0lD312SXfp9fKzHD5dfqJttpra5GfZaGCzWes6f6+bHOE84PYO3Pj0VQS5wc9JvxbLWuw176hzbKsOZIljcDRoHViM4Ygntq0Y\\\\\\\/b5E6Ot3ir4MI+7hr+jHPhkixqiJMm10vfLQ5T2+SfYlO\\\\\\\/JTreCR9GyL4fzhxV1ypfKErhuLVAj82VTfx+hIhar+OdXxku5II5dAJAU4mq4de\\\\\\\/TzCvRe6fNjIFyXUiLQmsp47YasPAW9W\\\\\\\/flUM+GbtHf\\\\\\\/n44lg2op2sl3gtUix71OODvBa5zXTsJ+f038aQuHoyyDd5tDN1NRNhyoFHBi2sve3rcI1VIhFKlmrnhESO95qcYku3xxdJU3625Pv5XuHVIrzhg\\\\\\\/zLHywh7NNCsMS80b4TxTdHak\\\\\\\/b3xRsbSf7x3KCfNdwIjpCGfP4iGxu2DTIHEWnmiVXE60V7YXKBYO8kIV+OlxBhwQg4Sgnnx8SjoH7pD6CKsH8yfSymGgPRptIP9D9FC+gKtTyvtQhFemTijUoxoR2nQmhGOwsz2ocrYu1rUfB8dLJtjGKq5pTsnXhLytHj1\\\\\\\/cUyVq+\\\\\\\/V0e61WauZIKIzwPRehKBXBRP8cqprdnB7CwNdkAP85qu38yykvmM3rYmMXj86i6b0W1UpHELRHJhVE01Rg+UlzxjyGs4F+75i5f95hdtsW\\\\\\\/95aBQbxACurjsN1YKAK7huQNAhtXkpyVk2i4uXUHJ53iionnaVTRBA8ndNNzM2KY9j56jSpHxx5hi0EUOHST67Lwj3ASjbO6p5DF18VPnDzsdUQt0+FS+lSal0MSTz16sew6EH1x+aSdq4dHPPKKfwFRm3DJ7jaYrzYUZiesK5VkuVKQv2OJ6bA6P5diK7qBKc3XaB+MZIqa3yLFcbXiO5C+RSxGPAwBuBML7zw8vx6EsyryaQWtnVuBTtHWHTCZGBy2tbkKjSEdUCKb+1H3B2g9rLyIkYB3fPUPtb2hK0N8bEWO\\\\\\\/CYm+zQTPYZ0yS5GmStt04+ogNVabEEaNuwwk1p2ql03llelVaPnyMasqkEpgARUqFO\\\\\\\/MTq+dp6YOTEcHPgaFOjPRpolGt9fJibZdV427y6L4a0Q1+XiPBQU+kSqhDcK1MI\\\\\\\/W9dWgMx1XynaxnDlLw3kSdcG4SaTf\\\\\\\/kpzab87VvWvxm9TZnAKSbpoc73XRouFNPPLGA8Kog6Mk25VL\\\\\\\/bDL9IFcguX2T7hFnxgDejxWaWPq0jtw20FzZa73Se11lNEkWH+GysiKDKySJe854nBCqCZv6c\\\\\\\/zImNVfauMBPmErOCE6+h\\\\\\\/Tkcm\\\\\\\/IW+h2Pj0LMuItcUxxjGnpsRDgVpdd4HDPjelz7eE4BmT4LogWIutQeonnWYhJ9DUDPdt\\\\\\\/9uMEyZwmITbhuflRZJmAKH7oK5f9ujm3zox48tDbcOuaPkQSbYD32rAiav4sXZR\\\\\\\/SuygJhimCSB4zT4J0YoO6hi1XLETL1WbdP50E2nwSM4qNjFemsFGe2CTrzFoR1j00G6tm\\\\\\\/H456IsgmMcwLGN5S5YscOXcCWp9ykIxgNxZ0gdDW+QKrV\\\\\\\/q\\\\\\\/r56Atqa7EQQbP61nvYP\\\\\\\/meITUTAmAgANY7RypYcTSWdC969s9Bg18y0yv13BdcLMNnyNWvWeq3yg\\\\\\\/\\\\\\\/wUyqexI6gzCPoUqOVJTF34O\\\\\\\/4KNMS+rFpYKnAtwUPBcOTbQNi1IrKho\\\\\\\/N\\\\\\\/GECuyo+AxOKjwrK+hATNodRic0RwsgrBSK0iA2UO+RKLpHXC4hSMZl4+5dm\\\\\\\/7euGUM\\\\\\\/g0S1fE1RbYXyoevCg3cZCIB38ZqxtWcf3o+wR6vSNFs8RZtWyju7Rj2U3bYoyIdFWdSqdCwZWuszzdhr+a9OuQSmT3OpsrVKZ7X+PklXzE6PALLGinP2fq4V+ef03jl9FGZ1+\\\\\\\/mAVjzLhYevOqb+Gz5evxE16DjMr32cDptWaa+YUEpjXfoo0KztTpFHdWo3ujsz8ixJhkJdm\\\\\\\/ncVgCdfj\\\\\\\/LHxyyJYUMDct5zWEufi8mY3WOg8L1GYMCI5vpYrq2cqUo5d0DR\\\\\\\/u6KBzA8h\\\\\\\/iEk3usKHVEof7kV2z7OCXdopLt0coi2wjhM9DQ0EY41kGWtra4vbONXu3Si9pfKyZKbpBzuHHaSZzPkn91uoN5ihjyAsBHUl+fPiXnuBW8gWVTtidU6bMMyTrXoMsQxedmEpUwiVp8OJKOBsASftRtuyQKpT5QAA38bqRqML01K10nTL5n9XxWbYw+SDX61vZ4D8qPXS6e6KYDvNS0omtpHM71EhSTCqHjd50NGsxH9PCvegopVPpq8B5LKgKu1h\\\\\\\/aq1ZW2F\\\\\\\/yWT14V5uFLckgcNxaLGqJWhvq7\\\\\\\/lttHI8zdB37t\\\\\\\/Pi4izEoNsLljC15vYhXXkZetSaoBwToadeXglVl1iHS42jFm360XqLXniWXagVdechQaBmFbXoeO8abbfS91qbZVSmz6Mf4sb6UdTfvDUgqdCF8d49dIFxXIhho50sqm1kANm+ZUz5TREFe\\\\\\\/OXjv8D7XUUj3O6+HKD7wUQwLR5YmROS3mnX1tU+f0a82FCv9GI1DaACcO0LCqqo41N1i0tZl0WA85HseDjSJKp\\\\\\\/ZUfAHCd2n5Sty3ga70ibmeqf4rPnFS7bYebxYUtXXn9XFgWaZ6HACHb6BETCGgXIIM5gTrkamzmU8JhEcGjpo2YG3FTNQ2fhd1herh\\\\\\\/DTMj06xBLUUm0FE36MeUfSU64YZaeGTt4bG0mq2fG3UnVch\\\\\\\/OLMhhOUqv6vJGtIIyc\\\\\\\/lWdSfVYuHcCmZGB9ezXnG7pApzSizsh898duuR\\\\\\\/SdNs1mEgJgUbrF1zjC6rXYzKs1GLZyqIn3LeLyhUD5zwq2UZkhBgQ7j\\\\\\\/detTE\\\\\\\/wtZBeL7mfC9\\\\\\\/SbDzG71F99KLf4xaWRBf0DIj2mnzkd9jaMXAtT7aw5fHXEx1l2kYfOajtJSgajIXtvGelRSc\\\\\\\/rB+MOiU8V4XlkUSpHlySAr6VqPmFyedZpmnSAmkmBt6vDN66EDcTW2WtiIaU4HA820WniLFzJ+sJduDNePAm386IxoY218fruZEqe2UEyBM64S4Gx4ZHtPF+OuySPg9pvMqwKa545VPX1pb8wr5ckRePsR0J8ZunBOUWiLziLfc9tv2sszhsNilPxFBMBvcgZEN4EoWgCdpPjt2BLkxX0SvksWvXAvWiXEJ42im9T0F12DQMcjtxjd9rqP5ZrRfkJJoxWiUbpv+mT9N45AHSar9TaRwiAfLTE5unWoPvfagMu2E1bBS5h4l8jk3Rh26rwquUW9EhG1YkxXdSMacSFI1v5aUhVY3zu+UnEbGJrTQ6qlk9OuUHdsdDaugHkHL433LtpICECZtUjUzVmkD7aC4SncmAM2V5dFGqRosyrQR6Pj1spLstsGwH3iQiszD3HgPJTS\\\\\\\/noBL3hRaLnxgDO\\\\\\\/WbS4zNUW1MtK1IonjvIvqJtIcGpubDWIoMC27lOUowEZ9WwXiCJH5kUoJ2JQGM6utqqwq2jIEFfdzBHHSw3\\\\\\\/5mlbmqf7V7n5phqH\\\\\\\/KFW+9Lpm8gSv\\\\\\\/yTXcabPzjEbQbkj4u3tI8BWly+kHkvcGhlcWvPHT9uInMZ20w\\\\\\\/te8g5axHPg80LhuVBP9VH4j4dW2YLBRhN+GEgDpUi18sRWhrb0GhgtVYDd9bPuEuQMj0+czgccIzVmCeExEoOsVHJqZ8\\\\\\\/1m2gdIza1YLDc76CcEwsv3f54PqpPrVudcfzF7Nx8rOlDHEnynaRENMsvh9dyPX\\\\\\\/MRQage3oOdGOMfVXhm5Lw7zRW7CaHjF3O2WmXqvghyUc6MbBH7IFnFwSqvaXDhbVFz0I2qDJ05\\\\\\\/2rFMgKILiucqE4lUb2Nqcj20KkpT2dfGgbpoabxF\\\\\\\/vov0jQDAUWorRmzBs1qk+urbZY9Wjuqy1gVsg9\\\\\\\/SC7Rm9dK1od4Jx98lSB0DP4Bczw3yPluURRx6gIMrNTyvd1fyuBrUNCudBsEKcCtHvF0oNyAHO5Dk2h0dXZsvMsabMADSLrPoZ58R56OOW8brD7wmyDXCWyoU3o0XSep9\\\\\\\/+MNQuRRCwB7pwYBETdgeMDpe6WMDSdsmrteLHgJ3L9dm\\\\\\\/SxC6Tjs7Xahd1gA28YfS5a0XM1MU+olmmqpN7Hl2+ojgKZXyArn96CRJlrbEBp0qDPlCYvMbqTnCy8blJHdtvPwIWrCQdSmsbJt4WjXT3bHGhkb3k5t1LhklifUAMiOQFeosT+PuWG2fqu\\\\\\\/m+4MFCSYNYzZ4oKn7aqA1vtmmWZeJRdHHQx8tyIAMkNCnvR1UGn9yN2kQ8G0H7euiYzwBtjFrscPKrvjpxrD7ZxC9nU8Qt+NeD5hls\\\\\\\/gkC5ysvmD0yRT1upqc8xBJUv6KTjqTvbuUaIbGy\\\\\\\/rvYDKLEo5vgYfZHTYbY\\\\\\\/\\\\\\\/uEaQ50Z70ve2VDNAbpFBmWVPYvRfWPuvZQOF17TnYGLGE+7EtzrhKFOHBad3mNNPqC7a+DmNmiBXJ3AZzTTAZrdu3+e3tsELkvezYZS9auDjXToRoWRZube5jezHnoJyV1IcBoT5fDkhgx17B92XrLTwGma0GswcBtv4SWi32iEgWWUyq8EHH3V7zTY4mUzGe4mEIwZdLgjJ8vNs0J7pQPlbM6ABpq8MANWNpJBR7JRhfBEr5aBcDdhEno0KOOItTbzWnzBkmSbV0YxX5ZJGpNIbWYoRnme7z0Gu\\\\\\\/sfzgx1hjEFq\\\\\\\/qRChYIYVif5NsECPK2U4UPqkZAogzT0KBhgJqSkg7CS75KTYHmGfxmIY6fkZiXOFaJa5LDC+7HSkvdnXVXz\\\\\\\/QZk9ntHE0GStAG\\\\\\\/06JjvGvaUPLfeXJnuUPye9Yzbr1+nFSv4z\\\\\\\/V2IBsAgkWTTfsx53plgLlNRyUesxcp85SMrM51s2jR3M40ZVkdy6T\\\\\\\/rGoxE8QeB5Y5Nh0PVrEKKKBcqvX\\\\\\\/+8bEV4wR9zdv4TgZMGnn3Xe5HYaa04QEXM39bFw45JeNJEKNF9srDVHyjeUc72vGb4jaRVTIPbRegUJEvtcL4a9UfsiCJHu5l0fy7lO2mKeZZjQIu1NTLe7rQrtzMoWZnl\\\\\\\/6Vq0pqAPzTvRo11rEZH6sbTHdFNpOabzhd2MHjQBTt9sF2MLn8FO3Fy0S1Td2UH1EcDT0hE7VYai5sWoGE7WYiHJeL+FMSQpR1QS7t\\\\\\\/bybVG\\\\\\\/me+2F\\\\\\\/pWz7Lf5TWbJ2DyJi2C8Hr43opintVbRd0nfYvgrSvhSw\\\\\\\/iy7T21x0C5AFYV9Jd9iYV9RLI6Lq0rkbelrjhHyOJSqPhk3NEWs30xuVSI7XP6v3XTy52xMppMDCwWp0v5\\\\\\\/LCUCxsJ+V\\\\\\\/86pN6G1zP5NwNlHBDmMRWeP5IbMfElUtT5P2ZhxVOIrHeGQdUZste184HJ3ut8wnDF+qGAzq1txaLAiNtFg+17tNJOod6rZLfivTwBCHG3STXvUq5JhbfYrvzv4C++BE5Dxvla1nGUF0sFNzQVMfF1ZRZbYvw2noIynaIyXg50i+3jvmd+CnH7cVGLAMAzLfSdT6CT4yqUgD9B5UbsZrIPlF6AZX1hc9vvz\\\\\\\/p5nIqTEXnjPCnbnjJYgFznBH8at\\\\\\\/QvCJ5Av0LuMJZy746zmuB9Ztcv7QbTc4hcZM2v6jMob9R9+iibDASfhvO1BTFjDTPxbi\\\\\\\/MCbrpuIlutn\\\\\\\/YDuTY5mn4wg35T7351RXWSQ1+qO1RH5+UgPxStOvxWSouw5aU\\\\\\\/9Tt2to0vizO7dx1EupfIcdosJCe2HHapoMyFXmg0Pxy2\\\\\\\/SbVr14tXsAiOSeru2zl+825f7EQaMGJdnAOQvWvELfAFo4pR+B+q7StJsbQjiI2hQ5Vyz4\\\\\\\/9lTWyMYDFG17O6OdVAarYq8trZFHg1ydK29zgDl8acFs8qpfQu8reBDk\\\\\\\/ntXyg8bFEUxAapKTG1a8\\\\\\\/ehKJytGCsUTStmHZm2ihgrjDECpXbGCzNcK5+Jp3iiMTTBuk70EVTvJkt99vX\\\\\\\/LKeSwgWyIJKvJzt7Hbeae3WsrmD8z5x2VPdGiRywK36POTmB9ZsMqynJyylD2u5JDwRoRd2Vxwey1GFbT49\\\\\\\/bjLweTt9V8vKERtnLLkFU5Sm6knWlplH8V7kUw08TTj2W+RjxIjyENM7\\\\\\\/bpAliB6TYZJ3dydEIkYzH\\\\\\\/p4zZlWE9zqixiUXbZiALusbQACfWTssiWuuVCNIHypo3v1BX7xAy5vHyTNALc9SeLzlYR62lIP90BunkAL7+2K77hwkPscTvX0wm3yHqv8HfxZm4xkdlsB+om8+\\\\\\\/2DppJJz5c7Tt7n493HHTmx3J7Oe\\\\\\\/qU5R+LaZ0010yc34hef0KoU4Q2b84AOQfOOjWvRXFmLZXMmRwSAqn0OjL2iGtcsdwxYBwyvWRfKzMyHknknTmC5ZjUaDUzi3X3l0xWLWGHFBqBV5e6qEVMjemjUH8GMswjbpDl7N5F8ZY49CMMeid5ln4dbgUqFUuTvURgcuk1ZD0UyLibpJfVzp1exRUdS+a5UCqyCcR\\\\\\\/65h0krlzkDQn+7aU9f2aj3rJ5Fd2Lk5t6axHyOkCRITD0zWTZc2KPRChE7dUej\\\\\\\/6s+ZCyLmWdeeIX9z+lEridjLBKWENJ4\\\\\\\/RNLykmw6yY4FbKdLhkfW2aJRffExD6\\\\\\\/m2PIVBsIETvstNFUIihXWQwlF68iXqBF3cZODXujvpsDS6vBP25Pu839tgvAI86HFRpigI4sJXp8Iy4nJGr9k+unD8zXOBHhcX4ELemLY+MRP22SSykldKoHeXtql2IUfGiZt36XK7wseM5o2Cn\\\\\\\/MBv0baHBjNDfVntiTP5hB6yQj04dqvQqQfH75uxVxeL2M7Mu8Sc\\\\\\\/Km\\\\\\\/HFLOVgaUTuFD8g4BZBUPwcw2LjKo6RMRdojhwSRD2C07gAYPEw3SJ7x4kdO4diuwH3LNyk2FIZyLcRs4GEQhs+RafUPTwT4HUYFobrwXOhjMrLjstKkVe2Ybq+P49vWLrqX1iXFfMHQdQxwz8FaMvNXClVH6xF\\\\\\\/WKYAlqBFICgbNDVWG1C8wuKbqdShxxeCUgDodlICehTYJw8SlCNhE2xOfjNRZiiMm5Vq9T+GBEL1H8A0eHUU05hYB7GjJ2mpkI9DZW3qEqIMQtgVTspfyntb1v4F5dm4JMsBOHC7qYcrOmZBYhZmnucMvv3BfWg3PZfM\\\\\\\/VPgP9pWywpPlfllhEQgzicHZoxDGJAihknxw9DDU3AH7XfWzwGwnW9gzKXPRM5BHukgSNEASWEmEhAr1eXFBHbrbhwoeWNuT+0OSijKgVV3avs8ybqKLpXKjUDa\\\\\\\/nbARMO\\\\\\\/YD0zGuvPv5GCnvO5R6jVxZpgTkRwm6Qn3mTxHE+LjFUSiT3ycSsqdkAScwr7odJQKm+WKiewJiKoMMy2WlUVQE\\\\\\\/Nwp1+VK2yDZXAWSU9f7Ls\\\\\\\/ku57qLqA2yh3FaXONP5oY\\\\\\\/JIKaOMJEzAyfYJWGRDzwYYn1pb1t3F4Fj3GvjFiyFF+\\\\\\\/6GjSM+RXjNskX2HTjoccK8AIEoKi7IDIk7CHNpt8z3ZhbD1bgKagzkN25KT7yuBWsRiHYTHJ8z9zUgvpf4HU+YbpRB7I0heaC2fGK4hXFlFOzaFgYqZ+wlA9bzt4ZI51wIGYNdxoAqebVRXRJf1pBoskxG6E6FlWAJ\\\\\\\/ZAwqKZrNgDQVdDXllFQNy3Z0BUG6MeZa\\\\\\\/2DeAD\\\\\\\/rmeanpaFCCQlI1P1T5euVPZTgrZlhNcV9bCNufEa9xyFVxcW5UxhqJaFFK\\\\\\\/5Yf6KEXeCegu86IBL2SFwQB6vHFaoKfezSTDPS4\\\\\\\/kU+qFnEM8RfNDTVOltmY3AhH5zgVaFzVKduj29mHdHS4p1e3+wnReA0W\\\\\\\/LZGv\\\\\\\/8dO9xXO0qHhwCQ+SnGp3PyHLS0MrGwSsdei01dzyzlzHIpbZQjz0yt\\\\\\\/yAiA70lsuZTrJNEsAqg5WW3aTsu4jpeazrA94PS6x27zVtsc6Yocu7XyRWRkOSZCdI6Wn84fJj3rXK43h7OFrSL7gJkdbBt0eGUPUNFlKXrPd1iWR\\\\\\\/fVexTf2iDoi5bWYKBPL7wda8iPNaufTFspRXc8nDONk1P6qpd2\\\\\\\/b\\\\\\\/u3076wZfk7DRueqVY4tGPBtvhvtLpUSzuiRMK\\\\\\\/V+j+CJVcH9mEzZz1eJaWNx\\\\\\\/RxOGL5K07C0NuQkqi8KkewdK2CRR3P5LG2TewLF7mQ8yVx7eNTG\\\\\\\/lG4XA\\\\\\\/uZ9OHntZo4QyKgnkgJH2AMGMiQqo4yo7RicaIgMtRS1+E97uhRPAwR2FbDAwpHXshu0xlTL58jcCzs5xgHxPgMK+ny9TaieOc+5re31l8U1CkTf81dCNfApyMEAYCz4GZpuTFjIa71s\\\\\\\/YZL3SRHZhq9FrOC2MMrH30d6rdz1FWFv6BGjllgxpBhjIfF2DhiD6kI9fH6MZ7VUBu6vYeXWynlXTYEfXolsAPwyU\\\\\\\/gFPClV6I0lRrM6R+HN8M4pX2Bft8C6ipRh3F6YDhRhxqJCsZry\\\\\\\/4+KSYaNb75TeQCNaQiG7Yf6bKcBQQGfp+SoUjqfmsYx67UU62KKNFnxaXO0kk5WOpInVNgiiKbvPni\\\\\\\/7wdnqPKZo2A66AMmgHbQt8FBPwPerXEfEcqpJx\\\\\\\/WchEKcxqtYQHRYX3I1swe5QdQ5fPJQWBT5qAgbeNw8X41uVwSsbj6RLSQqmQNw7p94z4gJ71Q8xcYrh1UIcXf8sCyWw\\\\\\\/JY1q3Xm8dzyOZ+owY3O\\\\\\\/kLYbggYNTA2CX32+oyNduTT+OiwY0TKpgz3LrrpvWaDFZ5pi+uibiJTtdPJMAE0NsGJvv8rleetMmzoVpfYoOD6kw\\\\\\\/+Z0HmEcFbwAk+npfQPrRVCgF1+OzpgFDM2IzRjdz7SVQGI9VhA5EKdANeRacRoW8DIMyI\\\\\\\/H5fPJiBQAUNH34eSpg1LylGEy92dIQ5kV9md2+sXfp+qQb88JQjikT06iDtfuvB\\\\\\\/Uzb13y5LwCThuZSAi0O8x5wkwT2wN92Rv89b3RwV76+LVLPCdVsOOe1JgJ2uXBZfh5yTM5nq+3HtvE2YOAucc+a4YS2B0oWES3\\\\\\\/6bgZungAmbiFCIItyqJoptsJQoaJzgAfFQ0XUjXJ4geoisIOr9JSCJ5J+V6jpvOQ89\\\\\\\/hWbLX\\\\\\\/01d4a4HiZLiINY0ssE2yuwsVuHN+yv6i5fs\\\\\\\/VAa1\\\\\\\/\\\\\\\/ModJxNFmigcgP5SwU0Lrnl18CnOUScf22v10SytcyBjKHG+caAqaoNfHhws+IXmt2ZWuBPDzGxsLhqIIjygV08oE5MeRp6ou0B15ariCBh2wIhWgyHst1zeBQWLsyX1rGb7vffu7Jxip+iE7RbN+y5c2j4KokiOhkE3xZqZI+UUDr1YQ986buaKELs451hARCC27j\\\\\\\/sBHGKV3LcM8IgPEBt28qimo3cvnYfsriOw8Sm+sxP3FYP72ZHABPxiH65yf04HNDUCS5E31\\\\\\\/gElHct5j\\\\\\\/BqG10llmB6p+h8Mqpk7l8MkX0xVhBzpwCsGg+mb9nJ2gcsayvlCHQzHTeyl\\\\\\\/giJCL4\\\\\\\/EXYA8UfRAOlB634l3aDT3CSy4w1QrCIN4PLAHEJPSmTmZolOzwR3KTc1TFnlu6yglg398PxkWgTCM1AOYtM7Zchw\\\\\\\/7co\\\\\\\/UObUe23QbMzX6egW7fxHL4Ni4q2v5T2u4baSE1I4Drpk2szvDCe8ebpqnQQ3shWgJUdDaLlrI2Q39U5Em08zcWQolU7iSgxNdfNlmDfCuNsOJRKCFTlfDAjZng5YN5xH5qcN8ViON5R+Z\\\\\\\/tmrXboLVKJSUW9AQAxcZ2T0iw1RLmlv4ukzlAL3ffGiFgTDgV7WEt6QOI9pXlnUyDreHZqefP4b0YkZFhcHIrKMG3LZJ0FiRMiAKvoyq5E3toXr6UWt\\\\\\\/AaQDmeeQglscr0Mq1N\\\\\\\/VxYuXjnOvTjqu0BACEkGxYrriC3j7LBBytt72jkRhBEOT0VvJGEqJ0kMe4vUJ0c6Y6mtM9HzvC5yzJftvv055HIREeTu64quStg9MnB54jWTJljqm\\\\\\\/v5gJRmw+s\\\\\\\/SoOtb4gZj0d+i6lPfsUyJQrl\\\\\\\/dQ8d3AaD9dydDt0BVZma9OSQ2NiEbD2P0UawchOqE4q6BT1ZE+KRT4+wn18SWhpbfFXQgQqi\\\\\\\/VHEs6xzpNVPXzPi52+F1hobAxXgRSuov029sZGUjczK1nwmHFXEXxfkw7kB3IV0l0IzupLo+SRGgYO1rfI39\\\\\\\/Zd1WnvEeMkvNGykv+ZpUXefpE8LRhrOz0pJpigYaPaMt6fQe7MIdy4bO7W1WP0hAF953+80a6v48xx7FVmgsyRXewSm4ZBclHlIKnscCDzkuoyKNIo5znXf5bnRdw1CWQWmUJ3ykkqkEQgQ8sVJk4z7fnZkjI\\\\\\\/sbcvGoz2p6isMiT3bT8pCKqH3QOigDrwNFCUwG2Fslz6z9Nt1zT2ylsTQrAcLCj+cXlDcaZrB7t01b1w3FJFyBbfbY6NkoPNFjDaofkb4NLkHi6UQcHDRMy8SvL3E\\\\\\\/Lcfgy+Y3vVEeWhSZovYfrm3mptevwgF+bXnsLz5HkKxf0oxCZx1YR2ctRcBkZzQgaSktnVhx8IfJDrkzsAhnw\\\\\\\/wRaegLsJdhhckGUgsOvakedYyWYvM0XkpGYwbII9CftNRWyqmXsq3pYTyhuUTsYYPbLrRe+f3Qz2QvKlsTNVzZkDCf9RgSoC+iOitMeQEPdipEnsmfzTdChkMzW6NYKpbr+ig7BDUECyYLFqNdh2BChIPKZHcUnussJD0GpDClSnKKt2g+U0wroAvIU\\\\\\\/L0jwqsQf8x2B7YF5lXyUN+nlf7LkRDOex9hi+9lxPxguIdIIiJDAI2NwT5qtqAwyKWU1pAwe\\\\\\\/ljo2RwMC+0uh7jEAHT4TjP4Et1fn13avha7boA+Eqlk\\\\\\\/UEcyhvKq3dfOGoxqHLmbjsJ3HOEynQSm8lO+1Bw9k5IIYj1+a11wETVG7bkPor596DkK7nGOlg7IORkOj3XzjWdWDhP0JyO2M0c\\\\\\\/Ue1Iu2xqO99vnIC1bZ7Gc+ooD3n3ZWt\\\\\\\/0phtG2YmZ6ZVwRDmtdzRy7hQ5CplUDB7cAA6FVV\\\\\\\/Xhyv0Dh9cSQk8cZbzLjk4vSIHxt0mVcC\\\\\\\/5CnxXDTaqgY+kDXEWs39hPaA8PSD8TFQcNsyxNdQjrMEWQ6j9ii34VUEq8FqIJf1FOMt8yHiKWkn12RQ6PsdG79lKqMco5op7U0H+swUSCNfnT\\\\\\\/rj57S9YfeyFq6d9xt62FNcqEwvBQmzWd1qxAVmw2L\\\\\\\/Ong\\\\\\\/ilsWRR4za3qCLLA08Cx8rfKOoGMkXhH+D9MR5ZB1y9r48T27xuEtSaEMAgtN8wGep9nNxBrut9iSX20eue8BCBVDyt2nFrz3J9MzolZsKA7023w5gZlVoq3ier7HxX1BdorE7C3etyrcSB3OPKb+EgcP1zz7uzhAGY7MxjuKdd5ywJMrUe7QPG9j4etasdDf6BEntDWkdA\\\\\\\/uND7hnDHEb2LnAp3zTEWcwbNFG2G249pBXit5DeJCmBJmKGOVdXl78X6h35UBZZs8GGJEGUuVAS9aKqFM9U16+19GU3mhGzDXw0KzeI6rC5Vvji\\\\\\\/kPy8JF77N+KXjMCNj+gEriPlhtgPfB+ubonfoSICQPUVYjPe9ku\\\\\\\/yFZpIZfQ+9wRAeOuMveskktauno+awjI1jImetQDNfh3pK5mZ9i4TomCYFKTF3zLGVxYoCLlYmKB89t3imFlsuRExS+tBv9h3OxxZFnHHPDb9+jQi5anjHGxvSkBIgAaI43xYTmkkff\\\\\\\/Bq+qDz1EUX7fZZfWx1atCxnfKO7q4iAHxb7xOK2GY63XwGVrWHcs8Df+fSsw80LbgsAp7be+34i7tm8IoDkruSVvRPC2BIyOjU7QNjvw3XrLdbpfefHViQni0BlQyHP70PBemZU5WeIWBfqzg\\\\\\\/9K3+MOz\\\\\\\/ay+A\\\\\\\/PgJ98OP0s1MKgelJpDT1aUTSs6EyBjfAsRzdtWRbpbF0y6t+hKK+TGGGCH6XoiFfA\\\\\\\/HYRaRmo2bXigrXfM7bLHQSjEp9ScXDZF3zrpHeEBnM4k6RA2Yn9SoO80EUioZTJu5qUmr3pPQ+zVtVp5ZDbQ136aH4J+2khQleSb0w2hgySwW5isbCNGgkrKozYtZHaXkVBOeNo5EaIDes+dkbWRm\\\\\\\/4oK19jDmPsawj4JGBHCX3oWtjEEZXLjCzgN8NrEHlptwyIzPSBh7bh1MzAmj860AkeZtNFaNJ+\\\\\\\/\\\\\\\/Kf3D8DHqJONHD9KB0ojZWUuLmZTGMiW\\\\\\\/4MT\\\\\\\/k2JjAzM7kgCJloufZdQ8d1d1C3OVkoPFxNL61GUk6T81BX2Hw8PE2Sx7sRHtIOriwzqtHBJ9P1k1+Y3vd+oz\\\\\\\/FnLiz0F\\\\\\\/TDkPxzhJMs\\\\\\\/WVw7vUMOz9t+KBxmYjojci+YHWMS1yPiYnHpW2JRtetSsC5DN\\\\\\\/rJYwSNqM6Ro2KguHgJrMvnwU41x3jh3Bxfai8SLRHpxoabAPlD1uupsDdTLPhIhtpYMvtwBalNttCY+VcUWCV+uZCZC5xI\\\\\\\/mGWEVb3w6xzVjqmKsJcLFVtgHDARQ6ZKdOFyqqpkWrsyyqw+gU\\\\\\\/FU0fZ9qJG+5hJhYmL7MgM4Oxm7hu3jWR\\\\\\\/7p1T4XvY4ObCYTO5YmqIQnylbNs2xmcDJYnUq9qSjnZpHifL57DUtYVoxQMwbwWfH9qhQkZ6OyWeXUS21rS81Jj+0JddAtEQTgGvPFw0FMVXDVqo6AMdw658RxTO5bm463BfUOR0q6A\\\\\\\/6H5HhS8Cd2CxespzTIcmSN5+FFnBzfVHsg5jhy5bh0KWZF+RqD6MAyPH+ptuD\\\\\\\/4zKNgf+N+I01Fh+tFE627ju+KLoKEwNlPSokgj+zd7Iie+pcsZ3bIAYT8Sxy2x\\\\\\\/RXOXL6r0ZWzC0VQRdfrUhIujZDwzu2ZRaJ7gsruTpX25JMm6vVX8rSKD5Tuu5JPrnqoFCX4G1G\\\\\\\/HgYuUe9c9q5kN4SDvUksyZKO7tFhcFkcdmmFrPl3LtXrUa5F948d8yQ4AuBOg5a+PE5fOKfa6QTq7fkCyR6KEDpntFXsjCHCe4gMWLwiM8tl3ha5YvPVS3kEil7gPNcs+TJUWxkMYCby4Kt1V+BUSde50Mnm\\\\\\\/dTdICAY9EtIeDzvuEZiMLo8vZEiPybOcbqxnes4coS\\\\\\\/yZJHGMKYj30fyH9G\\\\\\\/WDThwogsfM6QUbR9J0DzHL0uJWRBOBQ30xWq+FrTl8K7T9dB64B3kdCSYt0fsWCxOdzX12391e+vENe6Cj8Dbch\\\\\\\/BVtipHDUByRNRJSZOSdnYgm5vm+R6nZ76Sj0hcuBHg0XGi+FxNVcwiuKDIzQNFWU4xMMDk3vD9tTmNw35QKQIzuHSXM9tI4l59\\\\\\\/oEqz0xks5WEekAt\\\\\\\/6rxkjv7shkA7CAC2Ry4ANTuK7TAcU\\\\\\\/KoBC6BUKHT961iA6NtF53uFtHdNf9z6fbZPV05TMbiSoUCQIW4YuMhvGeSR\\\\\\\/sx9TD\\\\\\\/z0qRQZtkHsnjk\\\\\\\/VH\\\\\\\/uusOCQCtdTCN2D1oNZ14WG0uDWQiytGiLz7pl2XGlH1juHu70F1NWp3QIH\\\\\\\/x7WR7eRGuLqHymKy\\\\\\\/hBr6UkmN5W6TSJH2eqcSVcS\\\\\\\/lXl0vy7eQc7WxKI5q68XprRO4RCJ4jIug00DjbjBTW6N3iUUmEs0Zo6qMytZ4QMSs8s\\\\\\\/lyZ3OsL05kFHRg7CFKejim2QvD\\\\\\\/FUhf7Hc7\\\\\\\/hd2W4eq2Ej2ULHGgz5TzIP4zPv0emSb3vhFpuwVvvSqBUKFu+8rITkFpA\\\\\\\/5E5u2XU+uUagg36MXtqOsMJdKG+9\\\\\\\/cTeO4XazMS2PrkWUeMJwQNZCLZzy1+baXuL\\\\\\\/zpL7x\\\\\\\/qAbhIB8GYxzjwEuKJVZKKPQRyEybI\\\\\\\/xsAM5KaZQu1bMsRV0+Bar8KcYgtcqgiQKGLXu84Gs8hJ72+sn9n1OS1ByV1a9GHltTtKCu5hC6Od5n5YUEIkHztBXCB9r3QVJC02EZTrfeWBCOgfgQG5R8jVsMVucjRMZ33Sb9xexvEZMmA5X8itdUfk6qe9sq76QRTbXTjrNfgi6eXDGf4RdQklZziwwLUwBIwy9MZ8wjnMUMSdzI8Wc95NhP1kE+LL\\\\\\\/dC9e0Yz3LopsNtWut6\\\\\\\/a6d4J5IExyBITUEW3n9SXnmPIQX2sYhmkHcCXp2JJeUpBTSf6OPNyYx\\\\\\\/GEEGP6kJo+FeogiFQEJupWQ+v6zUGk\\\\\\\/T\\\\\\\/k\\\\\\\/zCra64WYCx1u6TijR18MFGCa6sYQHjYuKtj3KQMsHpC6PA\\\\\\\/IyByWYQsrfx1ClgJ9AXXi3sMn\\\\\\\/V8QuToCe2iEjsrVOlavn4md9ca9HFWR8b5qSc5bwoS1x+mg10ka3wdjjZMN+c+oUaQ3uiHnf9zJ2aiaQuxOLK6poyJx4wJ26+i++MT\\\\\\\/gjt59ZMlpWkoLqWjXdJdeqyJk+NUo5TEDBxLz3N7oTDqeIobjBOga233BVdDLVFSba+uAvkfYPaCaC78Azk5NUEZwnuE2f695qmylNxv5uVWf\\\\\\\/sVpxxCeZUnU1rCZFQ9aMs5AeQkq+mlOHPQeHRj0au7dUsMwzedal6km7tyNyCwQA\\\\\\\/q18VIzgQU76EDf6W35GcIWugsCyRMpIt\\\\\\\/E69A\\\\\\\/yYyc9dwaU7eQVJWxMweT6J2Dizc4+SmcOaLHMM1XPpT\\\\\\\/\\\\\\\/lPkQJrhiEJg4gxXZU5mLkf\\\\\\\/bj8xB\\\\\\\/kouJArDXdHF5i0nh\\\\\\\/Ldc\\\\\\\/BP0m0Xn088bTJAcZns6nLXtm5pj0fdyrwK3k8BJdffLRAoSr7fc87E1o7yfn\\\\\\\/DVygB9Q3jz54lJtVGyec7cwnRXOWmDbs8Nu8j9cH8f83H4BqiaK2HW9PavfOVA\\\\\\\/yHvAFybu+a74twkr+RTbRFqZ05K6Hh6dxUIUPKEPvEqIlmiZVHaORnyccRRk5l57jvmmFdb3FuR9bq3oXXXt1vdq\\\\\\\/o4QmD46AfadV7TSv4acj7uWhHQP\\\\\\\/IcXuDN\\\\\\\/selvJ2AZ9tOk3Cxvc83VGsNyLVJoF5pTUhYDxPo2KOD\\\\\\\/GbHMq2qnDx\\\\\\\/VCAhDjejxv0VfXIRWajLhsS8J1L2yvD2QvA20dW9xzWI1sQgOZNSlsQA4Txx3AQ4zb87aKC8sGOquoSKp0JoeuyiWhPWMd+AaH3i1i2m8Pvin1dbnGmH1GU9XY9vGkzPqAmphN\\\\\\\/CXg3oQHbIlShVDOzieTqulLlzaPBT0pjRIlblJ1o7RB6tiPDPq7rn0me+9axo+jjsmOSeai3ycudccAK0QChPR+qVqdhvpScKzKcQIrOh6Gm+fL+haSmasIaVEQku6plP4KZ6HFy7MP0ieMIiEEHGEec907FGe4OC3n118kUSeGoNSlppm0ETvh8eIiUYlfJZPxR9y\\\\\\\/WHeuBnBRKqIqtM4tylb45eUEB4H0QfKRms3h8QX2db43JGPzuvPNEpPZdwEJL6D23ooy5ZnJ3xj\\\\\\\/hzLQgYu5O6R7lpUsGqzTzuoUFHZx2KxPmyMa+8X+w6fmGyVGFxLUaJrqnLvnl3n+NZQ019FDMPUShvoqPowd5wsORt+onZYkH80g6HmmyHo\\\\\\\/ql5YKY97ByZsEUehZEkDKg0+Yk\\\\\\\/fMZQZ3aTdnAzxn+kgDq43TMvFpw4mof5NN23o9onCczJdkhtTH4elTVqMUo9IyyFl6KW\\\\\\\/g5LTlY82UcmHSA3aDSmCCkXC3DCFIz0l6qBPsGLUdp2NKTsl5cZGje2fp8HOQOQ5qn\\\\\\\/dbwQ82\\\\\\\/xmbFKEw8GMm5gZ\\\\\\\/GYQUnXFkadx0yyKLQSRnFliWJnB64aa3Hg280Blhe7LdFBH5zFx\\\\\\\/UK8WCJLS4oWwJJ+Q1xYR3KKYWVte0WU580w8jconJ9Oxv4qWsh23xRJCsSrag3\\\\\\\/vo8LVwPwBbo3Ljbi4qMkocYhwxIpeZooWa6Y9CAzvmUbXYqPix1a71jb5Aap1\\\\\\\/YkBy4SIwzoSF6bQ9W2H3jU+BRDDzxooB4\\\\\\\/UiNX4cf6CASoR8jkP77nTkInM\\\\\\\/knhFVXUlh9CIPhV4rry7TWx611lhI13Vv72JKezVuMSZbr8heLU9MhY3tBa4ahnpp\\\\\\\/tP3Yh4aIDYwNqDld9KMLHuYdVWlzSNnFeLBWxvA+PrxLbL31FTSvExW5KX7yq\\\\\\\/YAyR\\\\\\\/KDs1Pwe9cPXrcqMvBpY7mPGCInT+qXwkOJCYbxqurKB9Voghb\\\\\\\/Ol\\\\\\\/\\\\\\\/AFuqADjYtpZ3F7BRV2lNFrgPpjUSFte1GzlJYoXPwC9CKjvgNMtsJmcpIMqVZjHLJQdzfkeWlM\\\\\\\/roM0dpIG4qIbMh2wo4yKEkD\\\\\\\/44rOBTD1AiQSD+koXHzK9bTn\\\\\\\/491vTrpzDXmi4SJlj+qhuEgzrxGVRZlTOyv+wTvtTdKE+CM0yV\\\\\\\/uVSUZNirOQD+lZzbXVE++2qmvsRNiOlC9qk0lpW3xlxXt89897j+HFBr2TnR5s85usyaSV5xSjwGwpgfZR6O8rtA2tJrsfCgPmsM8xJRsKU6buHFaV2KHhXgX7TTwMGBF\\\\\\\/Ztyfh6rqacaeYTpwrNaco83l2VUbx9oGC+5648e10JmeYKpHEvNk8cE+VQR2RwmFHnKRGUaRB88Z2ByjrbIsSDCTc56G1zGQAPKhGAj+ETpNrOCtDErlvOC0bewok1sAuENhnnSxdtM9G8GMhnIpBgTHO+i+1KeTcY0wY0s+ytXVVTOIImGLfHrrdAVVzmJuDG+uOVsYEuAAiTxJogd\\\\\\\/guVaeh49bjlY5ezbR1AcH6OvtxQVik8TLnPUy04zlbCM5qqnutgA3Po1GZsH6ZMBqoO8YDc+pYBeoH7BsWX0OS9DroI6KtFCsNUPLHJSobVICAUXOeV0i2RN8JMUTOeojSuRx0YbGHMvJPAk2zIhkL3vEkAWsWvm8NGGwHeRUAdk+6STNlnQuAQqYGVjRzs3PPfhCcKpEselT8EnxuNNXkUNwAoZVKyEIj5HNta6Ooh1+7Tiu02uw08IE1yYgsc6gSb\\\\\\\/C42mgBptzg+h3MCNP70POmc2tOmVq0LbTuPU1YN9ad3knHpCWzxVKyC7eC9tsIoG1IMcAJQ0ZaO8QybZ7Kh4JYmkjxB6GQStXivDNpRlPZ9r099o02uwUsO5Wywjia+poxln3XCJyb0hQWFZ5OELcrG8jgBYn35P8\\\\\\\/ch61RsT7BDcTuNNVIutTj\\\\\\\/4pD8YxoTtN9pcBJ+OijZO9wMsuGLQuUOOpovxDRuv\\\\\\\/ziWu4bG4iA4dt0COoK\\\\\\\/XF7VJyipYbHCVLrz0uS06BXzL0OsN6ZL0tKAPwoc1988JNf\\\\\\\/QbrXLYc7G0uppgeK+f7Ia7WSoSJNCD4E1BJpimJ55iurM7gpiDYGC6b562\\\\\\\/vO2HpssRsp0m54TQNwL4Fwdk08KuV\\\\\\\/ifl0eww\\\\\\\/NnEMGc5vl2i4KiNUvuC5Z2bS18MlyRl1+HumzhJuEj843P+s+gXSl9c32xBlu15EUEtt8oKEe3DBnhgwJzdUX5FJPXlJTPEurms1GhvXLzQlPYGm7wBXaRUCqt9umvD2X4InoXsQ21QyQCgKuxgdPVUwgqFLXxq3LHn\\\\\\\/ZWC2baMerGtCitLif+3INV83MAMjseK6mwUb7\\\\\\\/5xDGdtioq3R\\\\\\\/\\\\\\\/HJigCxdI1pgGFtPupSgbiiCGN+GKfIWCmBH7rR+KaEnEAWJC0Mb91cdf7gouMw2rSfIeZvsUbP4J8b7cvtBFjaD35f6oIuMF\\\\\\\/BcKVJmPHvscvj\\\\\\\/omtzo7k\\\\\\\/iTlCjdOXPUrdVKl68vWQ4oC2ljqyvnlKELyxGhgPIXSfy+NIr0Rs0Zw1GUUGCl9aJV4BvpOaDrjZFJ7jIpJWLB1oTsRIgcg+J2g2BciFVbO1aWSJWUnIHXVbSEEEQmCLpxmX\\\\\\\/5dhI3PMbF09\\\\\\\/oOK9lNUyUmS\\\\\\\/\\\\\\\/oFpO+3ihFkMnI1hUliBkW1Na5m5hE0u7pp9R1uNB0a5+bNC9AIboaN\\\\\\\/47el4ryfrz0tI6x91jKt8ZmffjyQKgIa8vuccPsNiq3cXmWd1kEkA3crpUc66KKY3PkXKtS3H9Ol5HfwEM71V70E9V2wi07A3bFvAqNY799GC++61IVbKgZLra0qsr2S1SizsO7FW3Mx9lE5u0IyXTimGFkX\\\\\\\/t8+wiOUuAOKzLdOGLZ7Ox87tfjYv3fEL25hUtWzGoyr\\\\\\\/3bgSDX80XGZxin6z9tUZzHgTaoatrlQ5UvhJI02UoGOWNxGete1WygoDOCiT3e77hW4Qt26Mt9pSPPV4vr4C6C0ZcdbpJt6R7\\\\\\\/dD04FIfAk8Oaz1xSFaZC+qj3HHQmo9nKCAfEefJBz8rwNEMiHFxaAQmxJyFNgVm+5++2GoMKU47dRWPP1PsC2K2X5Iys3WNyhSenA305imhBXWSclc9bx4VlyaIyQzsJV5hTl4bRH00oJa8hixVHLDQP9EWp\\\\\\\/ufTmVkdM9pdOnk9CPd\\\u0022,\\\u0022iv\\\u0022:\\\u0022Jl2f\\\\\\\/P6iimmkVUJZU4ZiwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002272797732271b1f3dadb15974526aa131285789c5963a64ffd4cb78cf2008631f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jFyq3W4361IuZzW5hN1TIxVo8TBn+Th4NUvbiPYtuEVdMUv6Stb0zOeYDQSIWTiO\\\\\\\/SVat0XUbtmF4O+KRUlSMP7mCCto\\\\\\\/93JqmsB+eDhPXFWwA0fFl9dRXq9Nl9NaRsHqqvq99hdmWanUYwHjCZe+1ZNVslJGILU8u2aLNn5G91m495xpTjtetK5RB+s8oPseMaAEM\\\\\\\/cdchbrwT4h1MqBwMEqcyPBtsrAE1Me+BJ0kenv4vR80n9tGqn+Dgk8K397FBGtKRRaxcX\\\\\\\/J2nDmjab0lILsNfRoTq1T4rsjjOTZqJm3gwPcEeYm5jTaY5Rf3ZLsTr\\\\\\\/cQQL1+mrTqvRUr9rcTcbj26jNIvxYQjihw64T05MP7C96ueTXThr7Q+8yhgHK6ZyqCc3lmfdDfKgmLRKMVYXUTiM6NYpj2\\\\\\\/eO6Y2iAbkJtsXIFuyFx\\\\\\\/mR9+SkCtHYgfwYvF\\\\\\\/7t1ZxSf\\\\\\\/CQuyR04tPqQef7dAywiIbeGpK8Dg+te2VzE\\\\\\\/JOR4GE9bMXejEsmALFKCVSFTmJq4Ohb932dwmysj76i3DgPpcv4ns5dpFapr7D2G36WoMBAWX8uTitpKBtMN1e0TxVqrnBMOv1hy0a2\\\\\\\/6JdxQ3jmEtMLG8KpSi0w4No2TVmvNS\\\\\\\/wKjSHUSAiWyWc5qhziXTBHuesQ9Lr68CdUT9mNKrYJ45znDqN0L+5nFSWskBS+qYuNxgFJicRZyCQRhg9VkkrJNLr+q9RbaleLDl34gZ\\\\\\\/lLuWXvQI2sjTeJhyg+rrtYJwy5DnHyxNpdOLLM5DCFAg2lxeEQHYohkZVVOISFUre6EtlUHZURhZe6GZhlQ8S3T76mLRXZQ2YLQKIFFhy8SizgzFmrJyRcXd3moXgHImXqBZd9pFZB6aUPWgTFbMhoZ0sH\\\\\\\/uW5w4uCg7M0wzgVeHcNeMO2iqxzi2EGEB7aGT1tR9C+QetISUHRfZgpC0ORK2Hvba3DwovqvjIEw6eRut5uFAq\\\\\\\/P0OYOwYYa8bKBVmGAkQefSBvExFvmOn17te\\\\\\\/Ckjhul9d\\\\\\\/Ku+tOyHpjyTrlDAPckagq4A93VHkJvre0Uy3O6msIFuFtIp0QgehpwzP+z7ZizKa0LrkUNrSjvpS\\\\\\\/Y48PyAl9gcl8qhUhN0P7GZoB2PfIwPqmDrAzDUSBsQGXqJK9VmtDQqTASiCMW9ByWYEcbUMMS2ociTSUiSfVpqQndL+ECRS6hHNEMTwUKP7CbP18Vrus8P7QQsWZ9y\\\\\\\/upmayVQQ+gBlpWQ\\\\\\\/RwAUyHGhh6Q7ZpoPyvJ0CLxLS2EGRDugJZZq3cvtYng6KIjO9+no6YerJpbL7DdWd9yOLy84HNgIv5FroLFjhvHUovYWpUrGiNZEJKkexMlAdVknDjgUOmOqA3GNeTDrWP4\\\\\\\/jgTMFU06\\\\\\\/t+uwidfWe0MuGnV3Uv4scVxSFoAADRpuZrdyn6056l2DGRce271Y7UWs22A+GJoFI1tAQ9KpPYNpyibP6kSeU9XRDBWNsrW0h7z6NMxgdQKgs\\\\\\\/0L2j7nB6YJvv4ipceEpzzeF7kvpmBmqIzHzh1Nts0XKpDkww6\\\\\\\/Pua293e4wT8891a3y9lh0FG903GnpUAMfDhZxWi8FEdhMmemRCE6Kokr9DPCXAsuCN56eEzse+f9Jb2Mf5SNqWFCXmxC9o8u20KQehWPiwlDoNFa+B8sDH7CqDXeOqT7JEh2PBEoFZhyZ2c+z5Kd1adH4X9azey4DvV5oEOQla8MHp5ZQsV2I23FWaTM80wMl\\\\\\\/C3hYbXR7oH0fNZ3F7qRS8Sbra6BtZmYC3sJZBulEepD6DxzLqYVW6fa+IUSwaImLE3ptX4Vk\\\\\\\/Isiq01GddibhFja0QT2WnUYe1JnRVps59Lo8Vvz4Rq+MJgnS7x8ldmrtp04TeuzacngCEm9yPGcZxT3nl42s2qSDDPxTYWeCNEUL9\\\\\\\/nrG944ej+3n53mzB5RizbNqyUSJkHXU2B7ZoSgc34jhbCXJAmaBJvYtwVdVx7vc6C6Ffv4HHbRBNKGQI6AxGxZZkEjgFR5iKvou0PPr9YHbys=\\\u0022,\\\u0022iv\\\u0022:\\\u0022j8xlG6eo6QtNXlhxE0O0fg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e1e32f83b13a91b58ec390c9d501a3e78e2aa8f15fc8d036098cf11ae7267583\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460506 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3927" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bVSySxpZLbof7EqSNxrTvIAvo3gjQRQmRUhnDfRL2j0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3927" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9B0aI6-kWT0w5WU5yLkFLkFRPWCszQlH8cgPTc1NNWc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3913" title="X Chromosome Crossover Formation and Genome Stability in Caenorhabditis elegans Are Independently Regulated by xnd-1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3941" title="On Robust Association Testing for Quantitative Traits and Rare Variants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460506" id="node460506" data-pisa="ggg;6/12/3927" data-pisa-master="ggg;g3.116.034744" data-apath="/ggg/6/12/3927.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xing-Xing Shen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiaofan Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacek Kominek</span>, <span class="highwire-citation-author" data-delta="3">Cletus P. Kurtzman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chris Todd Hittinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3927-3939;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034744 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460506" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xing-Xing Shen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xing-Xing%2BShen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shen%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXing-Xing%2BShen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaofan Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaofan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaofan Zhou</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacek Kominek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacek%2BKominek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kominek%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacek%2BKominek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacek Kominek</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cletus P. Kurtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, U.S. Department of Agriculture, Peoria, Illinois 61604</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cletus%2BP.%2BKurtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurtzman%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACletus%2BP.%2BKurtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chris Todd Hittinger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/12/3927" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460506" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460506/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3927.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460506" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460506/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3927.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460506" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460506/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3927.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460506" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460506/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/12/3927.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460506" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460506/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Understanding the phylogenetic relationships among the yeasts of the subphylum Saccharomycotina is a prerequisite for understanding the evolution of their metabolisms and ecological lifestyles. In the last two decades, the use of rDNA and multilocus data sets has greatly advanced our understanding of the yeast phylogeny, but many deep relationships remain unsupported. In contrast, phylogenomic analyses have involved relatively few taxa and lineages that were often selected with limited considerations for covering the breadth of yeast biodiversity. Here we used genome sequence data from 86 publicly available yeast genomes representing nine of the 11 known major lineages and 10 nonyeast fungal outgroups to generate a 1233-gene, 96-taxon data matrix. Species phylogenies reconstructed using two different methods (concatenation and coalescence) and two data matrices (amino acids or the first two codon positions) yielded identical and highly supported relationships between the nine major lineages. Aside from the lineage comprised by the family Pichiaceae, all other lineages were monophyletic. Most interrelationships among yeast species were robust across the two methods and data matrices. However, eight of the 93 internodes conflicted between analyses or data sets, including the placements of: the clade defined by species that have reassigned the CUG codon to encode serine, instead of leucine; the clade defined by a whole genome duplication; and the species <em>Ascoidea rubescens</em>. These phylogenomic analyses provide a robust roadmap for future comparative work across the yeast subphylum in the disciplines of taxonomy, molecular genetics, evolutionary biology, ecology, and biotechnology. To further this end, we have also provided a BLAST server to query the 86 Saccharomycotina genomes, which can be found at <a href="http://y1000plus.org/blast">http://y1000plus.org/blast</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/phylogenomics" class="hw-term hw-article-keyword hw-article-keyword-phylogenomics" rel="nofollow">phylogenomics</a></li><li class="kwd"><a href="/keyword/maximum-likelihood-0" class="hw-term hw-article-keyword hw-article-keyword-maximum-likelihood" rel="nofollow">maximum likelihood</a></li><li class="kwd"><a href="/keyword/incongruence" class="hw-term hw-article-keyword hw-article-keyword-incongruence" rel="nofollow">incongruence</a></li><li class="kwd"><a href="/keyword/genome-completeness" class="hw-term hw-article-keyword hw-article-keyword-genome-completeness" rel="nofollow">genome completeness</a></li><li class="kwd"><a href="/keyword/nuclear-markers" class="hw-term hw-article-keyword hw-article-keyword-nuclear-markers" rel="nofollow">nuclear markers</a></li></ul><p id="p-3">Molecular phylogenetic analyses show that the fungal phylum Ascomycota is comprised of three monophyletic subphyla that share a common ancestor from ∼500 MYA (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kurtzman and Robnett 1994</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sugiyama <em>et al.</em> 2006</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Taylor and Berbee 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">James <em>et al.</em> 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>): the Saccharomycotina (syn. Hemiascomycota; <em>e.g.</em>, <em>Saccharomyces</em>, <em>Pichia</em>, <em>Candida</em>), the Pezizomycotina (syn. Euascomycota; <em>e.g.</em>, <em>Aspergillus</em>, <em>Neurospora</em>), and the Taphrinomycotina (syn. Archaeascomycota; <em>e.g.</em>, <em>Schizosaccharomyces</em>, <em>Pneumocystis</em>).</p><p id="p-4">Yeasts of the fungal subphylum Saccharomycotina exhibit remarkably diverse heterotrophic metabolisms, which have enabled them to successfully partition nutrients and ecosystems and inhabit every continent and every major aquatic and terrestrial biome (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>). While yeast species were historically identified by metabolic differences, recent studies have shown that many of these classic characters are subject to rampant homoplasy, convergence, and parallelism (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hittinger <em>et al.</em> 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hall and Dietrich 2007</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wenger <em>et al.</em> 2010</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Slot and Rokas 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lin and Li 2011</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Wolfe <em>et al.</em> 2015</a>). Despite the considerable progress in classifying yeasts using multilocus DNA sequence data, critical gaps remain (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kurtzman and Robnett 1998</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2003</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">2007</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nguyen <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kurtzman <em>et al.</em> 2008</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kurtzman and Suzuki 2010</a>); many genera are paraphyletic or polyphyletic, while circumscriptions at or above the family level are often poorly supported (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>).</p><p id="p-5">In recent years, phylogenomic analyses based on data matrices comprised of hundreds to thousands of genes from dozens of taxa have provided unprecedented resolution to several, diverse branches of the tree of life (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Song <em>et al.</em> 2012</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liang <em>et al.</em> 2013</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Xi <em>et al.</em> 2014</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wickett <em>et al.</em> 2014</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Whelan <em>et al.</em> 2015</a>). Although the genomes of several dozen yeast species are currently available (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>), published phylogenomic studies contain at most 25 yeast genomes (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rokas <em>et al.</em> 2003</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fitzpatrick <em>et al.</em> 2006</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Medina <em>et al.</em> 2011</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Marcet-Houben and Gabaldón 2015</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2016</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>).</p><p id="p-6">A robustly resolved backbone yeast phylogeny will be of great benefit, not only to the study of yeast biodiversity, but also to diagnosticians seeking to identify and treat yeast infections, to biotechnologists harnessing yeast metabolism to develop advanced biofuels, and to biologists designing computational and functional experiments. Toward that end, here we have used genome sequence data from 86 publicly available yeast genomes representing 9 of the 11 major lineages and 10 nonyeast fungal outgroups to reconstruct the backbone of the Saccharomycotina yeast phylogeny.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data acquisition</h3><p id="p-7">The workflow used to assemble the data sets for the inference of the backbone phylogeny of Saccharomycotina yeasts is described in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. To assemble a data set with the greatest possible taxonomic sampling as of January 11, 2016, we first collected all Saccharomycotina yeast species whose genomes were available (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>). We then excluded four publicly available genomes, namely, <em>Blastobotrys attinorum</em>, <em>B. petasosporus</em>, <em>Cephaloascus albidus</em>, and <em>C. fragrans</em>, which had been released under embargo and lacked a citable publication. In addition, we excluded the genomes of known hybrid species, such as <em>Pichia farinosa</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Louis <em>et al.</em> 2012</a>), <em>Saccharomyces cerevisiae</em> × <em>S. eubayanus</em> syn. <em>S. pastorianus</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Libkind <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gibson and Liti 2015</a>), and the wine yeast VIN7 (<em>S. cerevisiae</em> × <em>S. kudriavzevii</em>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Borneman <em>et al.</em> 2012</a>). For species with multiple isolates sequenced, we only included the genome of the isolate with the highest number of the “complete” genes (see below). These criteria resulted in the inclusion of genomes from 86 yeast species representing 9 of 11 major lineages of the subphylum Saccharomycotina (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>). Finally, we used the genomes of 10 nonyeast fungi that are representatives of the phylum Ascomycota as outgroups. Detailed information of the nomenclature, taxonomy, and source of the 96 genomes in our study is provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS1.xlsx">Table S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow illustrating the steps involved in the construction of the two phylogenomic data matrices used in this study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Workflow illustrating the steps involved in the construction of the two phylogenomic data matrices used in this study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F1.medium.gif" width="172" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/12/3927/F1.medium.gif" width="172" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Workflow illustrating the steps involved in the construction of the two phylogenomic data matrices used in this study.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>A custom BLAST database for the genomes of the 86 yeast species</h3><p id="p-9">To further facilitate the use of these 86 Saccharomycotina genomes by the broader research community, we set up a custom local BLAST database using Sequenceserver, version 1.0.8 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Priyam <em>et al.</em> 2015</a>). The database is free and publicly available through <a href="http://y1000plus.org/blast">http://y1000plus.org/blast</a>.</p></div><div id="sec-4" class="subsection"><h3>Assessment of genome assemblies and ortholog identification</h3><p id="p-10">Assessment of the 96 selected genome assemblies was performed using the BUSCO software, version 1.1b (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Simão <em>et al.</em> 2015</a>). Each individual genome was examined for the copy number of 1438 preselected genes (hereafter, referred to as BUSCO genes) that are single-copy in at least 90% of the 125 reference fungal genomes in the OrthoDB Version 7 database (<a href="http://www.orthodb.org">www.orthodb.org</a>) (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Waterhouse <em>et al.</em> 2013</a>). Briefly, for each BUSCO gene, a consensus protein sequence was generated from the hidden Markov model (HMM) alignment profile of the orthologous protein sequences among the 125 reference genomes using the HMMER software, version 3 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Eddy 2011</a>). This consensus protein sequence was then used as query in a tBLASTn (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Camacho <em>et al.</em> 2009</a>) search against each genome to identify up to three putative genomic regions, and the gene structure of each putative genomic region was predicted by AUGUSTUS (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Stanke and Waack 2003</a>). Next, the sequences of these predicted genes were aligned to the HMM alignment profile of the BUSCO gene, and the ones with alignment bit-score higher than a preset cutoff (90% of the lowest bit-score among the 125 reference genomes) were kept. If no predicted gene from a particular genome was retained after the last step, the gene was classified as “missing” from that genome. If one or more predicted genes from a genome were retained, these were further examined for their aligned sequence length in the HMM-profile alignment; predicted genes whose aligned sequence lengths were shorter than 95% of the aligned sequence lengths of genes in the 125 reference genomes were classified as “fragmented.” The remaining predicted genes were classified as “complete” if only one predicted gene was present in a genome, and “duplicated” if two or more “complete” predicted genes were present in a genome. Only the sequences of single-copy “complete” genes without any in-frame stop-codon(s) were used to construct ortholog groups across the 96 genomes. We excluded the orthologous group constructed from BUSCO gene “<em>BUSCOfEOG7MH16D</em>” from our subsequent analyses because sequences of this gene consistently failed to be predicted by AUGUSTUS across the 96 genomes.</p></div><div id="sec-5" class="subsection"><h3>Sequence alignment, alignment trimming, and removal of spurious sequences and low-quality genes</h3><p id="p-11">For each ortholog group, we first translated nucleotide sequences into amino acid sequences using a custom Perl script, taking into account the differential meaning of the CUG codon in the CUG-Ser clade of yeasts whose CUG codon encodes serine, instead of leucine (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dujon 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mühlhausen and Kollmar 2014</a>; <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>). Next, we aligned the amino acid sequences using the E-INS-i strategy as implemented by the program MAFFT, version 7.215 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Katoh and Standley 2013</a>), with the default gap opening penalty (–op = 1.53). We then used a custom Perl script to map the nucleotide sequences on the amino acid alignment and to generate the codon-based nucleotide alignment. Regions of ambiguous alignment in codon-based nucleotide alignments were trimmed using the trimAl software, version 1.4 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Capella-Gutierrez <em>et al.</em> 2009</a>) with the “gappyout” option on; otherwise, default settings were assumed. Finally, the trimmed codon-based alignments were translated into trimmed amino acid alignments.</p><p id="p-12">To minimize the inclusion of potentially spurious or paralogous sequences, the maximum likelihood (ML) phylogram for the trimmed codon sequence alignment of each ortholog group was inferred under an unpartitioned “GTR (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tavaré 1986</a>) + GAMMA (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Yang 1994</a>, <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">1996</a>)” model as implemented in RA×ML, version 8.2.3 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Stamatakis 2014</a>). Sequences whose terminal branch (leaf) lengths were at least 20 times longer than the median of all terminal branch lengths across the ML phylogram for a given orthologous group were excluded. In total, 49 sequences from 42 ortholog groups were removed. The resulting gene alignments were further filtered by length of trimmed gene alignment (alignments that were &lt;500 bp in length were removed from downstream analyses) and taxon number (alignments with &lt;50% gene occupancy, <em>i.e.</em>, that contained fewer than 48 taxa, were removed from downstream analyses).</p><p id="p-13">The remaining 1233 ortholog groups were used to generate two data matrices: (A) a C12 data matrix that included only the first and second codon positions of every gene (third codon positions were excluded because they showed much higher variation in GC content than the first and second codon positions; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS1.pdf">Figure S1</a>); and (B) an amino acid (AA) data matrix that included the amino acid sequences of every gene.</p></div><div id="sec-6" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-14">Phylogenetic analysis was performed separately for the AA and C12 data matrices using the ML optimality criterion (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Felsenstein 1981</a>), under two different approaches: concatenation (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huelsenbeck <em>et al.</em> 1996</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Rokas <em>et al.</em> 2003</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Philippe <em>et al.</em> 2005</a>) and coalescence (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Edwards 2009</a>). In the concatenation (<em>i.e.</em>, total evidence) approach, individual gene alignments are concatenated into a single data matrix and then analyzed jointly to infer the species phylogeny. Although this phylogenomic approach often yields a strongly supported phylogeny, it assumes that all individual genes have the same evolutionary history. In the coalescence approach, individual gene alignments are first used to estimate the individual gene trees, which are then used as input data to estimate the species phylogeny. Unlike the concatenation approach, the coalescence approach can efficiently account for differences in the evolutionary history among individual gene trees (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Liu <em>et al.</em> 2015</a>). However, the coalescence approach can be sensitive to errors and biases in estimating individual gene trees (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mirarab <em>et al.</em> 2015</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Springer and Gatesy 2016</a>), which in turn may mislead inference of the species phylogeny.</p><p id="p-15">To infer the concatenation phylogeny for the AA data matrix, we used an unpartitioned “PROTGAMMALG” model of amino acid substitution, as 681 out of 1233 genes favored “LG (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Le and Gascuel 2008</a>) + Gamma (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Yang 1994</a>, <a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">1996</a>)” for rate heterogeneity among sites as best-fitting model (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS2.pdf">Figure S2</a>). To infer the concatenation phylogeny for the C12 data matrix, we used an unpartitioned “GTR (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Tavaré 1986</a>) + GAMMA (<a id="xref-ref-88-3" class="xref-bibr" href="#ref-88">Yang 1994</a>, <a id="xref-ref-89-3" class="xref-bibr" href="#ref-89">1996</a>)” model of nucleotide substitution. In both cases, phylogenetic reconstruction was performed using five randomized maximum parsimony trees and five random trees as starting trees in RA×ML (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Stamatakis 2014</a>). Branch support for each internode was evaluated with 100 rapid bootstrapping replicates (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Stamatakis <em>et al.</em> 2008</a>). Finally, we also used a gene-based partition scheme (1233 partitions) to separately conduct ML tree search for the AA and C12 data matrices, in which parameters of evolutionary model (amino acid; see below; DNA, GTR+G) were separately estimated for each orthologous group (-q option) in RA×ML. As the ML trees produced by the gene-based partition scheme on both data matrices were topologically identical to the ML trees produced by the unpartitioned scheme (results are deposited on the figshare repository), and the computational resources required for partitioned analyses are much greater, bootstrap support and phylogenetic signal analyses were performed using only the unpartitioned scheme.</p><p id="p-16">For the coalescence-based analyses of the AA data matrix, the best-fitting model of amino acid evolution for each orthologous group was selected using the Bayesian Information Criterion (BIC) (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Schwarz 1978</a>), as implemented in ProtTest 3.4 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Darriba <em>et al.</em> 2011</a>). For the C12 data matrix, the GTR + GAMMA model was used to accommodate nucleotide substitution and rate heterogeneity among sites. In both cases, we inferred the best-scoring unrooted ML gene tree for every ortholog group by running 10 separate ML searches using RA×ML. Branch support for each internode was evaluated with 100 rapid bootstrapping replicates (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Stamatakis <em>et al.</em> 2008</a>). Individually estimated ML gene trees were used as input to estimate the coalescent-based phylogenies for the AA and C12 data matrices using the ASTRAL software, version 4.7.7 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mirarab <em>et al.</em> 2014</a>). The robustness of these phylogenies was evaluated by the multilocus bootstrap approach (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Seo 2008</a>) with 100 replicates, each of which consisted of individual gene trees each selected randomly from the set of 100 rapid bootstrapping trees available for each gene.</p><p id="p-17">Finally, we used internode certainty (IC) to quantify the incongruence by considering all most prevalent conflicting bipartitions for each individual internode among individual gene trees (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Salichos <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kobert <em>et al.</em> 2016</a>). The (partial) IC values were calculated from the set of the 1233 ML gene trees (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kobert <em>et al.</em> 2016</a>), as implemented in RA×ML, version 8.2.3. We found that the mean IC values of our 93 internal branches in both the concatenation- and coalescence-based phylogenies inferred from the AA data matrix were slightly higher than those of 93 internal branches in both the concatenation- and coalescence-based phylogenies inferred from the C12 data matrix (mean IC values in the concatenation- and coalescence-based phylogenies from the AA data matrix are 0.404 and 0.399, respectively, whereas mean IC values in the concatenation- and coalescence-based phylogenies from the C12 data matrix were 0.368 and 0.367, respectively). Thus, phylogenetic trees based on the AA data matrix showed lower levels of incongruence than those based on the C12 data matrix, in agreement with previous phylogenomic studies at similar evolutionary depths (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Rokas <em>et al.</em> 2003</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2016</a>). Finally, the incongruence among the 1233 individual (partial) gene trees generated by analysis of AA or C12 data was visualized in the form of a phylogenetic supernetwork [<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS8.pdf">Figure S8</a>; supernetwork by the Z-closure method with “tree size-weighted means” option implemented using the SplitsTree software, version 4.14.4 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huson and Bryant 2006</a>)].</p></div><div id="sec-7" class="subsection"><h3>Selecting subsets of genes with strong phylogenetic signal</h3><p id="p-18">Selecting strongly supported genes has been empirically shown to reduce incongruence among gene trees in phylogenetic analyses (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wang <em>et al.</em> 2015</a>). Since the AA data matrix showed lower levels of incongruence than the C12 data matrix (see above result), we examined the phylogenetic signal of individual genes using only the gene alignments of the AA data matrix. To quantify support for individual gene trees, we used two common phylogenetic measures on the AA data matrix: (1) the average bootstrap support (ABS), which was calculated using a custom Perl script, and corresponds to the average of bootstrap support values across the ML tree of a given gene; (2) the relative tree certainty (RTC), which was calculated in RA×ML, and corresponds to the average of all IC values across the ML tree of a given gene (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Salichos <em>et al.</em> 2014</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kobert <em>et al.</em> 2016</a>). IC values on each ML tree were calculated by examining the bipartitions present in the topologies generated by the 100 rapid bootstrap replicates.</p><p id="p-19">Four subsets of 1233 genes in the AA data matrix were constructed based on the ABS and RTC measures, respectively: the first two subsets included the 616 genes (top 50%) having the highest ABS or RTC values, respectively; the remaining two subsets included the 308 genes (top 25%) having the highest ABS or RTC values, respectively. Since the subsets constructed using the ABS and RTC phylogenetic measures showed no significant differences in the sets of genes included (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS3.pdf">Figure S3</a>), we used the two subsets (top 50 and 25%) based on ABS for subsequent analyses. For each subset, the concatenation phylogeny and the coalescent-based phylogeny, as well as their clade support values (bootstrap support and IC values), were separately estimated using RA×ML and ASTRAL by following the procedures described above.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-20">All data matrices and their resulting phylogenies have been deposited on the figshare repository at DOI: 10.6084/m9.figshare.3370675.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Genome completeness</h3><p id="p-21">Contig or scaffold N50 (<em>i.e.</em>, the contig or scaffold size above which 50% of the total length of the sequence assembly can be found) is by far the most widely used measure to assess the quality of a genome assembly (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Yandell and Ence 2012</a>). The higher N50 is, the better the assembly generally is. Nonetheless, this value does not assess assembly completeness in terms of gene content. Thus, we assessed completeness for each of 96 fungal genomes using the BUSCO set of 1438 single-copy, conserved genes among 125 fungi (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Simão <em>et al.</em> 2015</a>). We found that the percentage of “complete” BUSCO genes among the 96 genomes ranged from 65.2 to 98.5% of 1438 fungal BUSCO genes, with the average being 94.2% (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS2.xlsx">Table S2</a> for detailed values).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic quality assessment of the 86 yeast and 10 outgroup fungal genomes used in this study. The bar plot next to each species indicates the fractions of BUSCO genes that are present or missing in each genome. &#x201C;Complete&#x201D;: fraction of single-copy, full-length genes; &#x201C;Duplicated&#x201D;: fraction of multiple-copy, full-length genes; &#x201C;Fragmented&#x201D;: fraction of genes with a partial sequence; &#x201C;Missing&#x201D;: fraction of genes not found in the genome; &#x201C;Phylo&#x201D;: fraction of single-copy &#x201C;Complete&#x201D; genes used to construct the phylogenomic data matrices. Yeast species are arranged along the Y-axis in ascending order of the total number of &#x201C;Fragmented&#x201D; and &#x201C;Missing&#x201D; genes. The exact value of quality assessment of each species can be found in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic quality assessment of the 86 yeast and 10 outgroup fungal genomes used in this study. The bar plot next to each species indicates the fractions of BUSCO genes that are present or missing in each genome. &#x201C;Complete&#x201D;: fraction of single-copy, full-length genes; &#x201C;Duplicated&#x201D;: fraction of multiple-copy, full-length genes; &#x201C;Fragmented&#x201D;: fraction of genes with a partial sequence; &#x201C;Missing&#x201D;: fraction of genes not found in the genome; &#x201C;Phylo&#x201D;: fraction of single-copy &#x201C;Complete&#x201D; genes used to construct the phylogenomic data matrices. Yeast species are arranged along the Y-axis in ascending order of the total number of &#x201C;Fragmented&#x201D; and &#x201C;Missing&#x201D; genes. The exact value of quality assessment of each species can be found in Table S2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F2.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/12/3927/F2.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469779" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Genomic quality assessment of the 86 yeast and 10 outgroup fungal genomes used in this study. The bar plot next to each species indicates the fractions of BUSCO genes that are present or missing in each genome. “Complete”: fraction of single-copy, full-length genes; “Duplicated”: fraction of multiple-copy, full-length genes; “Fragmented”: fraction of genes with a partial sequence; “Missing”: fraction of genes not found in the genome; “Phylo”: fraction of single-copy “Complete” genes used to construct the phylogenomic data matrices. Yeast species are arranged along the Y-axis in ascending order of the total number of “Fragmented” and “Missing” genes. The exact value of quality assessment of each species can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS2.xlsx">Table S2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Only 10 of the 96 fungal genomes had &lt;90% of 1438 fungal BUSCO genes, with the assemblies of <em>Hanseniaspora valbyensis</em> (62.2%) and <em>H. uvarum</em> (64.5%) having the lowest coverage “complete” BUSCO genes, with ∼510 BUSCO genes either “missing,” “fragmented,” or “duplicated” in each of the two genomes. After performing GO term enrichment analysis of 365 genes missing in both <em>H. valbyensis</em> and <em>H. uvarum</em> with the <em>S. cerevisiae</em> GOSlim annotations using the Cytoscape plugin BinGO (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shannon <em>et al.</em> 2003</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Maere <em>et al.</em> 2005</a>), we found eight significantly overrepresented GO terms (Biological process: mitochondrion organization, mRNA processing, RNA splicing, peroxisome organization; Cellular component: endomembrane system, endoplasmic reticulum, peroxisome, Golgi apparatus; adjusted <em>P</em>-value ≤0.05; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS4.xlsx">Table S4A</a>). Five of the eight overrepresented GO terms are also found in analysis of all 1437 BUSCO genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS4.xlsx">Table S4B</a>). Since the N50 values for the assemblies of the two <em>Hanseniaspora</em> genomes are much higher (<em>H. valbyensis</em>, N50 = 332,595 bp; <em>H. uvarum</em>, N50 = 251,359 bp) than the N50 (83,174 bp) of one (<em>H. vineae</em>) of their closest relatives, their low coverage was unlikely due to lower quality of the genome assemblies. Rather, the <em>Hanseniaspora</em> genomes may be missing genes from specific functional categories or alternatively these genes were not detected due to the accelerated evolutionary rates of these genomes. This inference is consistent with the observation that the ancestral internode length leading to the two <em>Hanseniaspora</em> genomes was the longest in the yeast phylogeny (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), an observation also reported in the original genome study for <em>H. valbyensis</em> (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogenetic relationships of Saccharomycotina yeasts inferred from the concatenation-based analysis of a 1233 single-copy BUSCO gene amino acid (AA) data matrix. The ML phylogeny was reconstructed based on the concatenation amino acid data matrix (609,899 sites) under an unpartitioned LG + GAMMA substitution model using RA&#xD7;ML version 8.2.3 (Stamatakis 2014). Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values &#x2265;95%. Thicker branches show conflicts between concatenation-based phylogeny (Figure 3) and coalescence-based phylogeny (Figure 4). Note, branch lengths on the ML tree are given in the inset at the bottom left." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The phylogenetic relationships of Saccharomycotina yeasts inferred from the concatenation-based analysis of a 1233 single-copy BUSCO gene amino acid (AA) data matrix. The ML phylogeny was reconstructed based on the concatenation amino acid data matrix (609,899 sites) under an unpartitioned LG + GAMMA substitution model using RA&#xD7;ML version 8.2.3 (Stamatakis 2014). Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values &#x2265;95%. Thicker branches show conflicts between concatenation-based phylogeny (Figure 3) and coalescence-based phylogeny (Figure 4). Note, branch lengths on the ML tree are given in the &lt;em&gt;inset&lt;/em&gt; at the bottom left.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F3.medium.gif" width="333" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/12/3927/F3.medium.gif" width="333" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">The phylogenetic relationships of Saccharomycotina yeasts inferred from the concatenation-based analysis of a 1233 single-copy BUSCO gene amino acid (AA) data matrix. The ML phylogeny was reconstructed based on the concatenation amino acid data matrix (609,899 sites) under an unpartitioned LG + GAMMA substitution model using RA×ML version 8.2.3 (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Stamatakis 2014</a>). Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values ≥95%. Thicker branches show conflicts between concatenation-based phylogeny (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) and coalescence-based phylogeny (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Note, branch lengths on the ML tree are given in the <em>inset</em> at the bottom left.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Data matrix completeness</h3><p id="p-25">Following orthology identification, alignment trimming, and removal of spurious sequences and low-quality genes, we retained 1233 of the original 1438 orthologous groups from the 96 genomes (see <em>Materials and Methods</em>). None of the genomes of the 96 species had all 1233 orthologous groups, but 88 had &gt;1000 orthologous groups each (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS2.xlsx">Table S2</a>). The percentage of gene occupancy in the remaining eight species ranged from 60.5% (746/1233) to 78.6% (969/1233). Two recent studies showed that nonrandom bias in missing data might be potentially problematic for phylogenetic inference (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hosner <em>et al.</em> 2016</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Xi <em>et al.</em> 2016</a>), particularly for coalescence-based phylogenetic inference (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hovmöller <em>et al.</em> 2013</a>). Interestingly, the placements for the eight low gene-coverage species in our study were robust in both concatenation and coalescent analyses, as well as in the two data matrices and subsampling analyses (see results below), suggesting that the impact of missing data in this study is negligible.</p><p id="p-26">Among the 1233 orthologous groups, the percentage of sequence occupancy ranged from 50 to 100%, with an average value of 90%; 1084 orthologous groups displayed at least 80% sequence occupancy, and 24 contained sequences from all 96 species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS4.pdf">Figure S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS3.xlsx">Table S3</a>). In addition, gene alignment lengths ranged from 501 to 14,562 bp, with an average length of 1484 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS5.pdf">Figure S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/TableS3.xlsx">Table S3</a>). The AA and C12 data matrices contained a total of 609,899 and 1,219,798 sites, respectively.</p></div><div id="sec-12" class="subsection"><h3>A genome-wide yeast phylogeny</h3><p id="p-27">All phylogenetic analyses consistently separated the 10 nonyeast outgroup taxa (nine Pezizomycotina and one Taphrinomycotina species) from the Saccharomycotina yeasts (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>). Surprisingly, the genomes of two yeast isolates purported to be from the same species (<em>Geotrichum candidum</em> isolates CLIB 918 and 3C) were placed into two different clades; <em>G. candidum</em> CLIB 918 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Morel <em>et al.</em> 2015</a>) was nested within Saccharomycotina yeasts, whereas “<em>G. candidum</em>” 3C (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Polev <em>et al.</em> 2014</a>) was nested within the Pezizomycotina outgroup. Given its phylogenetic placement, we infer that the genome sequence of the isolate “<em>G. candidum</em>” 3C represents a misidentified Pezizomycotina species.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogenetic relationships of Saccharomycotina yeasts inferred from the coalescence-based analysis of a 1233 single-copy BUSCO gene amino acid (AA) data matrix. The coalescence-based phylogeny estimation was conducted using ASTRAL version 4.7.7 (Mirarab et al. 2014). Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values &#x2265;95%. Thicker branches show conflicts between coalescence-based phylogeny (Figure 4) and concatenation-based phylogeny (Figure 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The phylogenetic relationships of Saccharomycotina yeasts inferred from the coalescence-based analysis of a 1233 single-copy BUSCO gene amino acid (AA) data matrix. The coalescence-based phylogeny estimation was conducted using ASTRAL version 4.7.7 (Mirarab &lt;em&gt;et al.&lt;/em&gt; 2014). Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values &#x2265;95%. Thicker branches show conflicts between coalescence-based phylogeny (Figure 4) and concatenation-based phylogeny (Figure 3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F4.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/12/3927/F4.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469988" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">The phylogenetic relationships of Saccharomycotina yeasts inferred from the coalescence-based analysis of a 1233 single-copy BUSCO gene amino acid (AA) data matrix. The coalescence-based phylogeny estimation was conducted using ASTRAL version 4.7.7 (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Mirarab <em>et al.</em> 2014</a>). Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values ≥95%. Thicker branches show conflicts between coalescence-based phylogeny (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) and concatenation-based phylogeny (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Most internodes in the concatenation phylogenies inferred from the AA and C12 data matrices received high bootstrap support values that were ≥95% (AA, 91 out of 93 internodes; C12, 88 out of 93 internodes) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>). Similar to the results of the concatenation approach, the two coalescence-based phylogenies were also robustly supported, with 86/93 internodes (AA data matrix) and 86/93 internodes (C12 data matrix) showing 95% or greater bootstrap support (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>). There were five conflicting internodes between the AA and C12 concatenation-based phylogenies and two conflicting internodes between the AA and C12 coalescence-based phylogenies. In addition, comparison of the concatenation-based phylogeny to the coalescence-based phylogeny showed eight topological differences in the phylogenies inferred using the AA data matrix and five in the phylogenies inferred using the C12 data matrix. These topological differences are discussed below.</p></div><div id="sec-13" class="subsection"><h3>Stable and conflicted internodes</h3><p id="p-30">Of the nine major Saccharomycotina lineages, eight were monophyletic in all analyses; the only exception was the family Pichiaceae. This lineage was paraphyletic because <em>Komagataella pastoris</em>, which belongs to the <em>Komagataella</em> clade, groups within it (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Overall, the family Lipomycetaceae was resolved as the earliest-branching lineage of Saccharomycotina yeasts, followed by the family Trigonopsidaceae and the <em>Yarrowia</em> clades (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). A clade consisting of the family Pichiaceae, the CUG-Ser clade, and the <em>Komagataella</em> clade was well supported. The family represented by the most genome sequences, the Saccharomycetaceae, was recovered as the sister group to the family Saccharomycodaceae. This Saccharomycetaceae/Saccharomycodaceae clade was sister to the Phaffomycetaceae. These relationships were mostly recovered in two multigene studies (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kurtzman and Robnett 2013</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Mühlhausen and Kollmar 2014</a>) and one phylogenomic study based on 25 yeast genomes (<a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>), and they are broadly consistent with the most recent views of the yeast phylogeny (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>). Finally, eight of the nine major Saccharomycotina lineages were robustly and strongly supported in both concatenation- and coalescence-based analyses. The only exception was the family Ascoideaceae, whose support values under concatenation analyses were high (AA: BS = 95%; C12: BS = 97%) but were much lower under coalescence-based analyses (AA: BS = 53%; C12 = 45%). This finding is consistent with the instability in the placement of <em>Ascoidea rubescens</em> inferred from multiple analyses of different data matrices in the original genome study (<a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>). Thus, although this family was consistently recovered as the closest relative of the Saccharomycetaceae/Saccharomycodaceae/Phaffomycetaceae clade in our analyses (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>), its current placement in the yeast phylogeny should be considered tenuous and unresolved.</p><p id="p-31">To quantify incongruence between the 1233 orthologous groups across the 93 internodes of the yeast phylogeny, we used the 1233 individual ML gene trees to calculate IC values (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Salichos <em>et al.</em> 2014</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Kobert <em>et al.</em> 2016</a>). Our results showed that most of the internodes in concatenation- and coalescence-based phylogenies inferred from AA and C12 data matrices had IC values &gt;0 (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>), suggesting that those relationships were recovered by the majority of 1233 genes. For the aforementioned internodes that were incongruent between approaches (concatenation <em>vs.</em> coalescence) or data matrices (AA <em>vs.</em> C12), their IC values were often &lt;0 (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>). Examination of the phylogenetic supernetworks built from the AA and C12 data matrices also suggests that the degree of incongruence (visualized as the degree of reticulation in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS8.pdf">Figure S8</a>) among gene trees was negatively correlated with IC values. These conflicting internodes occurred within the WGD/allopolyploidization clade (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Wolfe and Shields 1997</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Marcet-Houben and Gabaldón 2015</a>) in the family Saccharomycetaceae (three topological differences), the CUG-Ser clade (four topological differences), and the <em>Yarrowia</em> clade (one topological difference).</p><p id="p-32">Within the WGD clade, the concatenation phylogenies identified the <em>Nakaseomyces</em> clade as the sister group to the genus <em>Saccharomyces</em> (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>), a relationship supported by several rare genomic changes (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Scannell <em>et al.</em> 2006</a>) and phylogenomic analysis (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2016</a>), whereas the phylogenies inferred from the coalescence-based approach strongly supported the <em>Kazachstania</em>/<em>Naumovozyma</em> clade as the sister group to the genus <em>Saccharomyces</em> (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>), a relationship supported by phylogenomic (<a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>) and multigene (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Mühlhausen and Kollmar 2014</a>) analyses. The monophyly of yeasts for the WGD clade was recovered by the concatenation- and coalescence-based phylogenies on the AA data matrix, as well as by the concatenation-based phylogeny on the C12 data matrix, consistent with recent studies (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dujon 2010</a>; <a id="xref-ref-60-8" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Wolfe <em>et al.</em> 2015</a>). In contrast, the coalescence-based phylogeny on the C12 data matrix weakly supported a paraphyly of the WGD clade, in which a ZT clade composed of the genus <em>Zygosaccharomyces</em> and the genus <em>Torulaspora</em> nested within the WGD clade in agreement with the results from multigene studies (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kurtzman and Robnett 2013</a>; <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Mühlhausen and Kollmar 2014</a>). Interestingly, a recent examination of 516 widespread orthologs from 25 yeast genomes inferred that the WGD event was the result of an allopolyploidization between the KLE clade (the genus <em>Kluyveromyces</em>, the genus <em>Lachancea</em>, and the genus <em>Eremothecium</em>) and the ZT clade (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Marcet-Houben and Gabaldón 2015</a>), providing a potential explanation for the observed instability of the WGD clade. A sister relationship between <em>S. arboricola</em> and <em>S. kudriavzevii</em> was only recovered by the coalescence-based phylogeny inferred from the AA data matrix (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>); this relationship contrasts with all other phylogenies inferred in this study, as well as those obtained in most published studies (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Scannell <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hittinger 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2013</a>; <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Mühlhausen and Kollmar 2014</a>; <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>).</p><p id="p-33">Within the CUG-Ser clade, <em>Suhomyces</em> (<em>Candida</em>) <em>tanzawaensis</em> was strongly recovered as sister to <em>Scheffersomyces stipitis</em> by both methods on the C12 data matrix, as well as by the coalescence-based phylogeny on the AA data matrix. In contrast, the concatenation-based phylogeny using the AA data matrix strongly supported a sister relationship between <em>Su. tanzawaensis</em> and a clade composed of <em>Sc. stipitis</em>, the genus <em>Spathaspora</em>, and some of the yeasts within the CUG-Ser clade, in agreement with the results of the original study describing the <em>Su. tanzawaensis</em> genome (<a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>). The closest relatives of the genus <em>Meyerozyma</em> were either <em>Debaryomyces hansenii</em>, <em>Candida tenuis</em>, or a clade containing <em>D. hansenii</em> and <em>Hyphopichia burtonii</em>, depending on the analysis and data matrix considered.</p><p id="p-34">Finally, within the <em>Yarrowia</em> clade, the concatenation phylogenies inferred from AA and C12 data matrices placed the <em>Candida apicola</em>/<em>Starmerella bombicola</em> clade as the sister group to <em>Blastobotrys (Arxula) adeninivorans</em> (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS6.pdf">Figure S6</a>), whereas the coalescence-based phylogenies inferred from AA and C12 data matrices supported <em>B. adeninivorans</em> as sister to a clade containing <em>Saprochaete clavata</em> and <em>G. candidum</em> isolate CLIB 918 (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS7.pdf">Figure S7</a>). Finally, the concatenation- and coalescence-based phylogenies inferred from AA and C12 data matrices consistently recovered a sister group of <em>Nadsonia fulvescens</em> and <em>Yarrowia lipolytica</em>, but its resolution in coalescence-based phylogenies inferred from C12 data matrices received weak median bootstrap support (BS = 65%). This group was not recovered in the previous multigene study of these taxa (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Kurtzman and Robnett 2013</a>).</p></div><div id="sec-14" class="subsection"><h3>Selecting genes with strong phylogenetic signal reduces incongruence</h3><p id="p-35">To examine whether the use of genes with strong phylogenetic signal could reduce incongruence among individual gene trees (<a id="xref-ref-60-9" class="xref-bibr" href="#ref-60">Salichos and Rokas 2013</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Wang <em>et al.</em> 2015</a>), we constructed two AA data matrices containing the 308 (top 25%) or 616 (top 50%) ortholog groups showing the highest average bootstrap values in their bootstrap consensus gene trees and reconstructed their phylogenies by concatenation and coalescence. The IC values of most internodes in both of the concatenation-based (ML) phylogeny (all orthologous groups, average IC value = 0.41; top 50%, average IC value = 0.55; top 25%, average IC value = 0.64) and the coalescence-based phylogeny (all orthologous groups, average IC value = 0.40; top 50%, average IC value = 0.54; top 25%, average IC value = 0.65) increased (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), suggesting that the use of genes with strong phylogenetic signal decreased the amount of incongruence in the yeast phylogeny.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the distributions of internode certainty (IC) values across three data matrices and two approaches. The three data matrices are: &#x201C;All genes,&#x201D; all 1233 genes in the AA data matrix; &#x201C;Top 50%,&#x201D; AA data matrix including the 616 genes with the strongest phylogenetic signal; &#x201C;Top 25%,&#x201D; AA data matrix including the 305 genes with the strongest phylogenetic signal (see Materials and Methods). For each data matrix, the set of individual ML gene trees is used to calculate (partial) internode certainty (IC) values for all internodes in the concatenation-based ML phylogeny (left panel) and the coalescence-based ASTRAL phylogeny (right panel), respectively. Rectangles in the boxplot denote 1st and 3rd quartiles. Horizontal thick bars represent mean IC values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the distributions of internode certainty (IC) values across three data matrices and two approaches. The three data matrices are: &#x201C;All genes,&#x201D; all 1233 genes in the AA data matrix; &#x201C;Top 50%,&#x201D; AA data matrix including the 616 genes with the strongest phylogenetic signal; &#x201C;Top 25%,&#x201D; AA data matrix including the 305 genes with the strongest phylogenetic signal (see &lt;em&gt;Materials and Methods&lt;/em&gt;). For each data matrix, the set of individual ML gene trees is used to calculate (partial) internode certainty (IC) values for all internodes in the concatenation-based ML phylogeny (left panel) and the coalescence-based ASTRAL phylogeny (right panel), respectively. Rectangles in the boxplot denote 1st and 3rd quartiles. Horizontal thick bars represent mean IC values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F5.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/12/3927/F5.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469765" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Comparison of the distributions of internode certainty (IC) values across three data matrices and two approaches. The three data matrices are: “All genes,” all 1233 genes in the AA data matrix; “Top 50%,” AA data matrix including the 616 genes with the strongest phylogenetic signal; “Top 25%,” AA data matrix including the 305 genes with the strongest phylogenetic signal (see <em>Materials and Methods</em>). For each data matrix, the set of individual ML gene trees is used to calculate (partial) internode certainty (IC) values for all internodes in the concatenation-based ML phylogeny (left panel) and the coalescence-based ASTRAL phylogeny (right panel), respectively. Rectangles in the boxplot denote 1st and 3rd quartiles. Horizontal thick bars represent mean IC values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">In agreement with the IC results, there were fewer topological differences between the concatenation- and coalescence-based phylogenies in the two reduced data matrices relative to the full data matrix (five topological differences instead of eight) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS9.pdf">Figure S9</a>). These five remaining conflicting internodes occurred within the CUG-Ser clade and the <em>Yarrowia</em> clade (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS9.pdf">Figure S9</a>). Specifically, the concatenation phylogeny inferred from the top 50% data matrix was topologically identical with that inferred from the complete data matrix, albeit more weakly supported. Furthermore, unlike the coalescence-based phylogeny recovered from the complete data matrix, the coalescence-based phylogeny from the top 50% data matrix recovered the <em>Nakaseomyces</em> clade as the sister group to the genus <em>Saccharomyces</em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). For the top 25% data matrix, both concatenation- and coalescence-based phylogenies supported this sister relationship. However, they consistently supported the paraphyly of the WGD clade (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1/FigureS9.pdf">Figure S9</a>), which was not recovered in concatenation- and coalescence-based phylogenies inferred from the top 50% data matrix but was observed in previous multigene studies (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Kurtzman and Robnett 2013</a>; <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Mühlhausen and Kollmar 2014</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Conflicts in the phylogenetic relationships of Saccharomycotina yeasts inferred from the concatenation-based (A) and coalescence-based (B) analysis of the 616 genes in the AA data matrix whose bootstrap consensus gene trees had the highest average bootstrap support (top 50%). Branch support values near internodes are indicated as bootstrap support values (internodes without designation have values &#x2265;95%). Thicker branches show conflicts between coalescence-based phylogeny and concatenation-based phylogeny. Note that outgroup taxa are not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Conflicts in the phylogenetic relationships of Saccharomycotina yeasts inferred from the concatenation-based (A) and coalescence-based (B) analysis of the 616 genes in the AA data matrix whose bootstrap consensus gene trees had the highest average bootstrap support (top 50%). Branch support values near internodes are indicated as bootstrap support values (internodes without designation have values &#x2265;95%). Thicker branches show conflicts between coalescence-based phylogeny and concatenation-based phylogeny. Note that outgroup taxa are not shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F6.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/12/3927/F6.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">Conflicts in the phylogenetic relationships of Saccharomycotina yeasts inferred from the concatenation-based (A) and coalescence-based (B) analysis of the 616 genes in the AA data matrix whose bootstrap consensus gene trees had the highest average bootstrap support (top 50%). Branch support values near internodes are indicated as bootstrap support values (internodes without designation have values ≥95%). Thicker branches show conflicts between coalescence-based phylogeny and concatenation-based phylogeny. Note that outgroup taxa are not shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">In summary, 75/83 internodes in this 86-taxon phylogeny of the Saccharomycotina yeasts are robust to different approaches (concatenation <em>vs.</em> coalescence) and phylogenomic data matrices (AA <em>vs.</em> C12), while 72 internodes are highly supported and the remaining 11 internodes are still unresolved or equivocal (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). After comparing our results to those of the most recent consensus view of the yeast phylogeny (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>) and to those of the most recent phylogenomic study (<a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>), we found that 14 of our strongly supported internodes are new to our study (<a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>; <a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>) (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Supported and unresolved internodes in phylogeny of Saccharomycotina yeasts. Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values &#x2265;95%. Solid lines indicate internodes that are robustly and highly supported by different approaches (concatenation, coalescence) and phylogenomic data matrices (AA, C12). Internodes reported as resolved in the most recent consensus view of yeast phylogeny (Hittinger et al. 2015) or the most recent yeast phylogenomic study (Riley et al. 2016) are labeled as black solid lines, whereas those that are new to this study are labeled as thicker green solid lines. Dashed red lines indicate internodes that show conflict or are weakly supported; we consider such internodes to be unresolved or equivocal. Note that the topology shown is the same as that shown in Figure 3 but with the outgroup taxa removed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2061257767" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Supported and unresolved internodes in phylogeny of Saccharomycotina yeasts. Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values &#x2265;95%. Solid lines indicate internodes that are robustly and highly supported by different approaches (concatenation, coalescence) and phylogenomic data matrices (AA, C12). Internodes reported as resolved in the most recent consensus view of yeast phylogeny (Hittinger &lt;em&gt;et al.&lt;/em&gt; 2015) or the most recent yeast phylogenomic study (Riley &lt;em&gt;et al.&lt;/em&gt; 2016) are labeled as black solid lines, whereas those that are new to this study are labeled as thicker green solid lines. Dashed red lines indicate internodes that show conflict or are weakly supported; we consider such internodes to be unresolved or equivocal. Note that the topology shown is the same as that shown in Figure 3 but with the outgroup taxa removed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3927/F7.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/12/3927/F7.medium.gif" width="357" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3927/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-40" class="first-child">Supported and unresolved internodes in phylogeny of Saccharomycotina yeasts. Branch support values near internodes are indicated as bootstrap support value (above) and internode certainty (below), respectively. * indicates bootstrap support values ≥95%. Solid lines indicate internodes that are robustly and highly supported by different approaches (concatenation, coalescence) and phylogenomic data matrices (AA, C12). Internodes reported as resolved in the most recent consensus view of yeast phylogeny (<a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">Hittinger <em>et al.</em> 2015</a>) or the most recent yeast phylogenomic study (<a id="xref-ref-56-10" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>) are labeled as black solid lines, whereas those that are new to this study are labeled as thicker green solid lines. Dashed red lines indicate internodes that show conflict or are weakly supported; we consider such internodes to be unresolved or equivocal. Note that the topology shown is the same as that shown in <a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a> but with the outgroup taxa removed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">The 11 remaining unresolved or equivocal internodes (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>) are placed at different evolutionary depths and in different lineages. Therefore, different strategies might be potentially helpful to resolve some of them, including but not limited to an increase in the density of taxonomic sampling (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Zwickl and Hillis 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heath <em>et al.</em> 2008</a>), the adoption of different types of data such as rare genomic changes (RGCs) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rokas and Holland 2000</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Scannell <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Butler <em>et al.</em> 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Polzin and Rokas 2014</a>), and the use of mixture substitution models that better account for across-site heterogeneity [<em>e.g.</em>, CAT model implemented in PhyloBayes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lartillot and Philippe 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lartillot <em>et al.</em> 2009</a>); currently, using this model in phylogenomic data matrices such as ours is computationally prohibitive]. Finally, we note that some of these internodes may represent genuine polytomies, in other words cases in which an ancestral yeast lineage split into more than two descendant lineages at approximately the same time (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rokas and Carroll 2006</a>).</p></div><div id="sec-15" class="subsection"><h3>Conclusions</h3><p id="p-42">Twenty years ago, the genome sequence of <em>S. cerevisiae</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goffeau <em>et al.</em> 1996</a>) ushered yeast biology into the age of genomics. Although we still lack genomic data from most of the known yeast biodiversity, the public availability of dozens of yeast genomes provided us with an opportunity to examine the quality of genomic data presently available for the lineage (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) and infer the backbone phylogeny of the Saccharomycotina (<a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4</a>). With several large-scale efforts to sample yeast biodiversity currently underway, such as the 1002 Yeast Genomes Project focusing on <em>S. cerevisiae</em> (<a href="http://1002genomes.u-strasbg.fr">http://1002genomes.u-strasbg.fr</a>), the iGenolevures Consortium (<a href="http://gryc.inra.fr">http://gryc.inra.fr</a>), and the Y1000+ Project focusing on sequencing the genomes of all known species of the subphylum Saccharomycotina (<a href="http://y1000plus.org">http://y1000plus.org</a>), the phylogenomic analyses reported in this study, including the classification of branches into resolved and unresolved (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>), provide a robust roadmap for future comparative research across the yeast subphylum, while highlighting clades in need of further scrutiny.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Thomas W. Jeffries, Meredith Blackwell, and the Department of Energy (DOE) Joint Genome Institute for releasing several genome sequences through MycoCosm prior to their formal publication (<a id="xref-ref-56-11" class="xref-bibr" href="#ref-56">Riley <em>et al.</em> 2016</a>) and Abigail Lind for help with the GO term enrichment analysis. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the United States Department of Agriculture (USDA). USDA is an equal opportunity provider and employer. This work was conducted in part using the resources of the Advanced Computing Center for Research and Education (ACCRE) at Vanderbilt University and of the UW-Madison Center for High Throughput Computing. This work was supported by the National Science Foundation (DEB-1442113 to A.R.; DEB-1442148 to C.T.H.), in part by the DOE Great Lakes Bioenergy Research Center (DOE Office of Science BER DE-FC02- 07ER64494), the USDA National Institute of Food and Agriculture (Hatch project 1003258 to C.T.H.), and the National Institutes of Health (NIAID AI105619 to A.R.). C.T.H. is an Alfred Toepfer Faculty Fellow, supported by the Alexander von Humboldt Foundation. C.T.H. is a Pew Scholar in the Biomedical Sciences, supported by the Pew Charitable Trusts.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-44">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034744/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-45"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-08-18"><span class="received-label">Received </span>August 18, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-21"><span class="accepted-label">Accepted </span>September 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Shen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.2" data-doi="10.1111/j.1567-1364.2011.00773.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riches</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome sequence of the wine yeast VIN7 reveals an allotriploid hybrid genome with <em>Saccharomyces cerevisiae</em> and <em>Saccharomyces kudriavzevii</em> origins.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2011.00773.x%26rft_id%253Dinfo%253Apmid%252F22136070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjEyLzEvODgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.3" data-doi="10.1038/nature08064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">M. A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakthikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of pathogenicity and sexual reproduction in eight Candida genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DButler%26rft.auinit1%253DG.%26rft.volume%253D459%26rft.issue%253D7247%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DEvolution%2Bof%2Bpathogenicity%2Band%2Bsexual%2Breproduction%2Bin%2Beight%2BCandida%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08064%26rft_id%253Dinfo%253Apmid%252F19465905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465905&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266608600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.4" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.5" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutierrez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silla-Martinez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TrimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk3MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ProtTest 3: fast selection of best-fit models of protein evolution.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D27%26rft.spage%253D164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.7" data-doi="10.1038/nrg2811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Yeast evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D512%26rft.epage%253D524%26rft.atitle%253DYeast%2Bevolutionary%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2811%26rft_id%253Dinfo%253Apmid%252F20559329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20559329&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.8" data-doi="10.1371/journal.pcbi.1002195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accelerated profile HMM searches.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002195%26rft.epage%253De1002195%26rft.atitle%253DAccelerated%2BProfile%2BHMM%2BSearches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002195%26rft_id%253Dinfo%253Apmid%252F22039361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039361&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.9" data-doi="10.1111/j.1558-5646.2008.00549.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Is a new and general theory of molecular systematics emerging?</span> <abbr class="cit-jnl-abbrev">Evolution.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DEdwards%26rft.auinit1%253DS.%2BV.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D19%26rft.atitle%253DIs%2Ba%2Bnew%2Band%2Bgeneral%2Btheory%2Bof%2Bmolecular%2Bsystematics%2Bemerging%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00549.x%26rft_id%253Dinfo%253Apmid%252F19146594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00549.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19146594&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262469600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.10" data-doi="10.1007/BF01734359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D368%26rft.epage%253D376%26rft.atitle%253DEvolutionary%2Btrees%2Bfrom%2BDNA%2Bsequences%253A%2Ba%2Bmaximum%2Blikelihood%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01734359%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01734359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MG91100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.11" data-doi="10.1186/1471-2148-6-99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Logue</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fungal phylogeny based on 42 complete genomes derived from supertree and combined gene analysis.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DFitzpatrick%26rft.auinit1%253DD.%2BA.%26rft.volume%253D6%26rft.spage%253D99%26rft.epage%253D99%26rft.atitle%253DA%2Bfungal%2Bphylogeny%2Bbased%2Bon%2B42%2Bcomplete%2Bgenomes%2Bderived%2Bfrom%2Bsupertree%2Band%2Bcombined%2Bgene%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-6-99%26rft_id%253Dinfo%253Apmid%252F17121679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-6-99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17121679&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Saccharomyces pastorianus</em>: genomic insights inspiring innovation for industry.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D32%26rft.spage%253D17%26rft_id%253Dinfo%253Apmid%252F25088523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25088523&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000348862300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.13" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 Genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.14" data-doi="10.1534/genetics.107.074963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The reacquisition of biotin prototrophy in <em>Saccharomyces cerevisiae</em> involved horizontal gene transfer, gene duplication and gene clustering.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2293</span>–<span class="cit-lpage">2307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHall%26rft.auinit1%253DC.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2293%26rft.epage%253D2307%26rft.atitle%253DThe%2BReacquisition%2Bof%2BBiotin%2BPrototrophy%2Bin%2BSaccharomyces%2Bcerevisiae%2BInvolved%2BHorizontal%2BGene%2BTransfer%252C%2BGene%2BDuplication%2Band%2BGene%2BClustering%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074963%26rft_id%253Dinfo%253Apmid%252F18073433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIyOTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedtke</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillis</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Taxon sampling and the accuracy of phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">J. Syst. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSyst.%2BEvol.%26rft.volume%253D46%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000256878000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.16" data-doi="10.1016/j.tig.2013.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Saccharomyces diversity and evolution: a budding model genus.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.01.002%26rft_id%253Dinfo%253Apmid%252F23395329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23395329&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319309100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.17" data-doi="10.1073/pnas.0404319101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Parallel inactivation of multiple GAL pathway genes and ecological diversification in yeasts.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">14144</span>–<span class="cit-lpage">14149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404319101%26rft_id%253Dinfo%253Apmid%252F15381776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM5LzE0MTQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.18" data-doi="10.1016/j.gde.2015.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boekhout</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomics and the making of yeast biodiversity.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D35%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2015.10.008%26rft_id%253Dinfo%253Apmid%252F26649756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2015.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26649756&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.19" data-doi="10.1093/molbev/msv347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosner</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glenn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimball</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Avoiding missing data biases in phylogenomic inference: an empirical study in the Landfowl (Aves: Galliformes).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1110</span>–<span class="cit-lpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv347%26rft_id%253Dinfo%253Apmid%252F26715628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMzLzQvMTExMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.20" data-doi="10.1016/j.ympev.2013.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hovmöller</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubatko</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Effects of missing data on species tree estimation under the coalescent.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D69%26rft.spage%253D1057%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2013.06.004%26rft_id%253Dinfo%253Apmid%252F23769751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2013.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23769751&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.21" data-doi="10.1016/0169-5347(96)10006-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Combining data in phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">152</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DHuelsenbeck%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D152%26rft.epage%253D158%26rft.atitle%253DCombining%2Bdata%2Bin%2Bphylogenetic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252896%252910006-9%26rft_id%253Dinfo%253Apmid%252F21237790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(96)10006-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21237790&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UB35100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.22" data-doi="10.1093/molbev/msj030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Application of phylogenetic networks in evolutionary studies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj030%26rft_id%253Dinfo%253Apmid%252F16221896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvMjU0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.23" data-doi="10.1038/nature05110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauff</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matheny</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofstetter</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reconstructing the early evolution of fungi using a six-gene phylogeny.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">818</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJames%26rft.auinit1%253DT.%2BY.%26rft.volume%253D443%26rft.issue%253D7113%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DReconstructing%2Bthe%2Bearly%2Bevolution%2Bof%2BFungi%2Busing%2Ba%2Bsix-gene%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05110%26rft_id%253Dinfo%253Apmid%252F17051209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17051209&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241362700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.24" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvNzcyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.25" data-doi="10.1093/molbev/msw040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kobert</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Computing the internode certainty and related measures from partial gene trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1606</span>–<span class="cit-lpage">1617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw040%26rft_id%253Dinfo%253Apmid%252F26915959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMzLzYvMTYwNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3927.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Orders and families of ascosporogenous yeasts and yeast-like taxa compared from ribosomal RNA sequence similarities</span>, pp. <span class="cit-fpage">249</span>–<span class="cit-lpage">258</span> in <span class="cit-source"><em>Ascomycete Systematics: Problems and Perspectives in the Nineties</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hawksworth</span>  <span class="cit-name-given-names">D. L.</span></span>. <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.27" data-doi="10.1023/A:1001761008817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Identification and phylogeny of ascomycetous yeasts from analysis of nuclear large subunit (26S) ribosomal DNA partial sequences.</span> <abbr class="cit-jnl-abbrev">Antonie van Leeuwenhoek</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.stitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.aulast%253DKurtzman%26rft.auinit1%253DC.%2BP.%26rft.volume%253D73%26rft.issue%253D4%26rft.spage%253D331%26rft.epage%253D371%26rft.atitle%253DIdentification%2Band%2Bphylogeny%2Bof%2Bascomycetous%2Byeasts%2Bfrom%2Banalysis%2Bof%2Bnuclear%2Blarge%2Bsubunit%2B%252826S%2529%2Bribosomal%2BDNA%2Bpartial%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1001761008817%26rft_id%253Dinfo%253Apmid%252F9850420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1001761008817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9850420&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077317100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.28" data-doi="10.1016/S1567-1356(03)00012-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Phylogenetic relationships among yeasts of the “<em>Saccharomyces</em> complex” determined from multigene sequence analyses.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1567-1356%252803%252900012-6%26rft_id%253Dinfo%253Apmid%252F12748053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjMvNC80MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.29" data-doi="10.1111/j.1567-1364.2006.00157.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multigene phylogenetic analysis of the <em>Trichomonascus</em>, <em>Wickerhamiella</em> and <em>Zygoascus</em> yeast clades, and the proposal of <em>Sugiyamaella</em> gen. nov. and 14 new species combinations.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2006.00157.x%26rft_id%253Dinfo%253Apmid%252F17311592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjcvMS8xNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.30" data-doi="10.1111/1567-1364.12006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Relationships among genera of the <em>Saccharomycotina</em> (<em>Ascomycota</em>) from multigene phylogenetic analysis of type species.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1567-1364.12006%26rft_id%253Dinfo%253Apmid%252F22978764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjEzLzEvMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.31" data-doi="10.1007/s10267-009-0011-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phylogenetic analysis of ascomycete yeasts that form coenzyme Q-9 and the proposal of the new genera <em>Babjeviella</em>, <em>Meyerozyma</em>, <em>Millerozyma</em>, <em>Priceomyces</em>, and <em>Scheffersomyces</em>.</span> <abbr class="cit-jnl-abbrev">Mycoscience</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycoscience%26rft.volume%253D51%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10267-009-0011-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10267-009-0011-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000274518400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.32" data-doi="10.1111/j.1567-1364.2008.00419.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robnett</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basehoar-Powers</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phylogenetic relationships among species of <em>Pichia</em>, <em>Issatchenkia</em> and <em>Williopsis</em> determined from multigene sequence analysis, and the proposal of <em>Barnettozyma</em> gen. nov., <em>Lindnera</em> gen. nov. and <em>Wickerhamomyces</em> gen. nov.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00419.x%26rft_id%253Dinfo%253Apmid%252F18671746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjgvNi85MzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3927.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fell</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boekhout</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>The Yeasts: A Taxonomic Study</em></span>. <span class="cit-publ-name">Elsevier Science</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.34" data-doi="10.1093/molbev/msh112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lartillot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A Bayesian mixture model for across-site heterogeneities in the amino-acid replacement process.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh112%26rft_id%253Dinfo%253Apmid%252F15014145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIxLzYvMTA5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.35" data-doi="10.1093/bioinformatics/btp368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lartillot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepage</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanquart</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2286</span>–<span class="cit-lpage">2288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp368%26rft_id%253Dinfo%253Apmid%252F19535536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTcvMjI4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.36" data-doi="10.1093/molbev/msn067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An improved general amino acid replacement matrix.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn067%26rft_id%253Dinfo%253Apmid%252F18367465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTMwNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.37" data-doi="10.1093/molbev/mst072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">One thousand two hundred ninety nuclear genes from a genome-wide survey support lungfishes as the sister group of tetrapods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1803</span>–<span class="cit-lpage">1807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst072%26rft_id%253Dinfo%253Apmid%252F23589454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzgvMTgwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.38" data-doi="10.1073/pnas.1105430108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Libkind</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valério</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dover</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">14539</span>–<span class="cit-lpage">14544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1105430108%26rft_id%253Dinfo%253Apmid%252F21873232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM1LzE0NTM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.39" data-doi="10.1093/molbev/msq184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Expansion of hexose transporter genes was associated with the evolution of aerobic fermentation in yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq184%26rft_id%253Dinfo%253Apmid%252F20660490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvMTMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.40" data-doi="10.1186/1471-2164-14-69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ba</span>  <span class="cit-name-given-names">A. N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blythe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High quality de novo sequencing and assembly of the <em>Saccharomyces arboricolus</em> genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-69%26rft_id%253Dinfo%253Apmid%252F23368932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-69&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23368932&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.41" data-doi="10.1111/nyas.12747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Estimating phylogenetic trees from genome-scale data.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1360</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D1360%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnyas.12747%26rft_id%253Dinfo%253Apmid%252F25873435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nyas.12747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873435&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.42" data-doi="10.1093/molbev/msn221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cushion</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Ezpeleta</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Phylogenomic analyses support the monophyly of Taphrinomycotina, including <em>Schizosaccharomyces</em> fission yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn221%26rft_id%253Dinfo%253Apmid%252F18922765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.43" data-doi="10.1534/g3.111.000745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Despons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Pichia sorbitophila</em>, an interspecies yeast hybrid, reveals early steps of genome resolution after polyploidization.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DLouis%26rft.auinit1%253DV.%2BL.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D299%26rft.epage%253D311%26rft.atitle%253DPichia%2Bsorbitophila%252C%2Ban%2BInterspecies%2BYeast%2BHybrid%252C%2BReveals%2BEarly%2BSteps%2Bof%2BGenome%2BResolution%2BAfter%2BPolyploidization%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000745%26rft_id%253Dinfo%253Apmid%252F22384408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvMi8yOTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.44" data-doi="10.1093/bioinformatics/bti551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heymans</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuiper</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3448</span>–<span class="cit-lpage">3449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti551%26rft_id%253Dinfo%253Apmid%252F15972284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTYvMzQ0OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.45" data-doi="10.1371/journal.pbio.1002220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Beyond the whole-genome duplication: phylogenetic evidence for an ancient interspecies hybridization in the baker’s yeast lineage.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1002220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002220%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002220%26rft_id%253Dinfo%253Apmid%252F26252497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26252497&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.46" data-doi="10.1007/s00239-011-9461-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reconstructing the fungal tree of life using phylogenomics and a preliminary investigation of the distribution of yeast prion-like proteins in the fungal kingdom.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMedina%26rft.auinit1%253DE.%2BM.%26rft.volume%253D73%26rft.issue%253D3-4%26rft.spage%253D116%26rft.epage%253D133%26rft.atitle%253DReconstructing%2Bthe%2Bfungal%2Btree%2Bof%2Blife%2Busing%2Bphylogenomics%2Band%2Ba%2Bpreliminary%2Binvestigation%2Bof%2Bthe%2Bdistribution%2Bof%2Byeast%2Bprion-like%2Bproteins%2Bin%2Bthe%2Bfungal%2Bkingdom.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-011-9461-4%26rft_id%253Dinfo%253Apmid%252F21938499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-011-9461-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21938499&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298056400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.47" data-doi="10.1093/bioinformatics/btu462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reaz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayzid</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swenson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ASTRAL: genome-scale coalescent-based species tree estimation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">i541</span>–<span class="cit-lpage">i548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu462%26rft_id%253Dinfo%253Apmid%252F25161245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTcvaTU0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.48" data-doi="10.1126/science.aaa7719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayzid</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boussau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warnow</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Response to Comment on “Statistical binning enables an accurate coalescent-based estimation of the avian tree.”</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMirarab%26rft.auinit1%253DS.%26rft.volume%253D350%26rft.issue%253D6257%26rft.spage%253D171%26rft.epage%253D171%26rft.atitle%253DResponse%2Bto%2BComment%2Bon%2B%2522Statistical%2Bbinning%2Benables%2Ban%2Baccurate%2Bcoalescent-based%2Bestimation%2Bof%2Bthe%2Bavian%2Btree%2522%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaa7719%26rft_id%253Dinfo%253Apmid%252F26450204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE0OiIzNTAvNjI1Ny8xNzEtYiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.49" data-doi="10.1038/srep11571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterck</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swennen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onesime</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Differential gene retention as an evolutionary mechanism to generate biodiversity and adaptation in yeasts.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">11571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D11571%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep11571%26rft_id%253Dinfo%253Apmid%252F26108467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep11571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26108467&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.50" data-doi="10.1093/gbe/evu152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mühlhausen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kollmar</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular phylogeny of sequenced <em>Saccharomycetes</em> reveals polyphyly of the alternative yeast codon usage.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3222</span>–<span class="cit-lpage">3237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu152%26rft_id%253Dinfo%253Apmid%252F25053656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjYvMTIvMzIyMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.51" data-doi="10.1016/j.mycres.2006.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">S.-O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackwell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Morphological and ecological similarities: wood-boring beetles associated with novel xylose-fermenting yeasts, <em>Spathaspora passalidarum</em> gen. sp. nov. and <em>Candida jeffriesii</em> sp. nov.</span> <abbr class="cit-jnl-abbrev">Mycol. Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1232</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycological%2Bresearch%26rft.stitle%253DMycol%2BRes%26rft.aulast%253DNguyen%26rft.auinit1%253DN.%2BH.%26rft.volume%253D110%26rft.issue%253DPt%2B10%26rft.spage%253D1232%26rft.epage%253D1241%26rft.atitle%253DMorphological%2Band%2Becological%2Bsimilarities%253A%2Bwood-boring%2Bbeetles%2Bassociated%2Bwith%2Bnovel%2Bxylose-fermenting%2Byeasts%252C%2BSpathaspora%2Bpassalidarum%2Bgen.%2Bsp.%2Bnov.%2Band%2BCandida%2Bjeffriesii%2Bsp.%2Bnov.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mycres.2006.07.002%26rft_id%253Dinfo%253Apmid%252F17011177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mycres.2006.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17011177&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242428700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.52" data-doi="10.1146/annurev.ecolsys.35.112202.130205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delsuc</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lartillot</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Phylogenomics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D36%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.35.112202.130205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.35.112202.130205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polev</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bobrov</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eneyskaya</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulminskaya</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Draft genome sequence of <em>Geotrichum candidum</em> strain 3C.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00956</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D2%26rft.spage%253De00956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.54" data-doi="10.1371/journal.pone.0092213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polzin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluating rare amino acid substitutions (RGC_CAMs) in a yeast model clade.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e92213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De92213%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0092213%26rft_id%253Dinfo%253Apmid%252F24637883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0092213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24637883&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-6.12.3927.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Priyam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodcroft</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munagala</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghul</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequenceserver: a modern graphical user interface for custom BLAST databases.</span> <span class="cit-source">bioRxiv</span> <a href="http://biorxiv.org/lookup/doi/10.1101/033142">http://biorxiv.org/lookup/doi/10.1101/033142</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.56" data-doi="10.1073/pnas.1603941113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haridas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative genomics of biotechnologically important yeasts.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">9882</span>–<span class="cit-lpage">9887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1603941113%26rft_id%253Dinfo%253Apmid%252F27535936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzM1Lzk4ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.57" data-doi="10.1371/journal.pbio.0040352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bushes in the tree of life.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De352%26rft.epage%253De352%26rft.atitle%253DBushes%2Bin%2Bthe%2Btree%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040352%26rft_id%253Dinfo%253Apmid%252F17105342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105342&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.58" data-doi="10.1016/S0169-5347(00)01967-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">P. W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Rare genomic changes as a tool for phylogenetics.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D454%26rft.epage%253D459%26rft.atitle%253DRare%2Bgenomic%2Bchanges%2Bas%2Ba%2Btool%2Bfor%2Bphylogenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252800%252901967-4%26rft_id%253Dinfo%253Apmid%252F11050348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(00)01967-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11050348&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165122400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.59" data-doi="10.1038/nature02053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-scale approaches to resolving incongruence in molecular phylogenies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">798</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D425%26rft.issue%253D6960%26rft.spage%253D798%26rft.epage%253D804%26rft.atitle%253DGenome-scale%2Bapproaches%2Bto%2Bresolving%2Bincongruence%2Bin%2Bmolecular%2Bphylogenies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02053%26rft_id%253Dinfo%253Apmid%252F14574403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574403&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186118500036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.60" data-doi="10.1038/nature12130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring ancient divergences requires genes with strong phylogenetic signals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12130%26rft_id%253Dinfo%253Apmid%252F23657258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23657258&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318952000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.61" data-doi="10.1093/molbev/msu061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel information theory-based measures for quantifying incongruence among phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu061%26rft_id%253Dinfo%253Apmid%252F24509691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzUvMTI2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.62" data-doi="10.1038/nature04562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DScannell%26rft.auinit1%253DD.%2BR.%26rft.volume%253D440%26rft.issue%253D7082%26rft.spage%253D341%26rft.epage%253D345%26rft.atitle%253DMultiple%2Brounds%2Bof%2Bspeciation%2Bassociated%2Bwith%2Breciprocal%2Bgene%2Bloss%2Bin%2Bpolyploid%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04562%26rft_id%253Dinfo%253Apmid%252F16541074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16541074&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235997600054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.63" data-doi="10.1534/g3.111.000273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zill</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The awesome power of yeast evolutionary genetics: new genome sequences and strain resources for the <em>Saccharomyces sensu stricto</em> genus.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DScannell%26rft.auinit1%253DD.%2BR.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D25%26rft.atitle%253DThe%2BAwesome%2BPower%2Bof%2BYeast%2BEvolutionary%2BGenetics%253A%2BNew%2BGenome%2BSequences%2Band%2BStrain%2BResources%2Bfor%2Bthe%2BSaccharomyces%2Bsensu%2Bstricto%2BGenus%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000273%26rft_id%253Dinfo%253Apmid%252F22384314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjEvMS8xMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.64" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating the dimension of a model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.65" data-doi="10.1093/molbev/msn043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">T.-K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Calculating bootstrap probabilities of phylogeny using multilocus sequence data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn043%26rft_id%253Dinfo%253Apmid%252F18281270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvOTYwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.66" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.67" data-doi="10.1093/gbe/evw179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genome-scale investigation of how sequence, function, and tree-based gene properties influence phylogenetic inference.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2565</span>–<span class="cit-lpage">2580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw179%26rft_id%253Dinfo%253Apmid%252F27492233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjgvOC8yNTY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.68" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTkvMzIxMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.69" data-doi="10.1073/pnas.0914418107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slot</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multiple GAL pathway gene clusters evolved independently and by different mechanisms in fungi.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10136</span>–<span class="cit-lpage">10141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0914418107%26rft_id%253Dinfo%253Apmid%252F20479238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIyLzEwMTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.70" data-doi="10.1073/pnas.1211733109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Resolving conflict in eutherian mammal phylogeny using phylogenomics and the multispecies coalescent model.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14942</span>–<span class="cit-lpage">14947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211733109%26rft_id%253Dinfo%253Apmid%252F22930817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM3LzE0OTQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.71" data-doi="10.1016/j.ympev.2015.07.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatesy</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The gene tree delusion.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D94%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.07.018%26rft_id%253Dinfo%253Apmid%252F26238460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.07.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26238460&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.72" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.73" data-doi="10.1080/10635150802429642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoover</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rougemont</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A rapid bootstrap algorithm for the RAxML web servers.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150802429642%26rft_id%253Dinfo%253Apmid%252F18853362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjU3LzUvNzU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.74" data-doi="10.1093/bioinformatics/btg1080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gene prediction with a hidden markov model and a new intron submodel.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-supplement">Suppl 2</span>): <span class="cit-fpage">ii215</span>–<span class="cit-lpage">ii225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D90002%26rft.spage%253Dii215%26rft.epage%253D225%26rft.atitle%253DGene%2Bprediction%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%2Band%2Ba%2Bnew%2Bintron%2B%2Bsubmodel%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg1080%26rft_id%253Dinfo%253Apmid%252F14534192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxNjoiMTkvc3VwcGxfMi9paTIxNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.75" data-doi="10.3852/mycologia.98.6.996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sugiyama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">S.-O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Early diverging Ascomycota: phylogenetic divergence and related evolutionary enigmas.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.stitle%253DMycologia%26rft.aulast%253DSugiyama%26rft.auinit1%253DJ.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D996%26rft.epage%253D1005%26rft.atitle%253DEarly%2Bdiverging%2BAscomycota%253A%2Bphylogenetic%2Bdivergence%2Band%2Brelated%2Bevolutionary%2Benigmas%26rft_id%253Dinfo%253Adoi%252F10.3852%252Fmycologia.98.6.996%26rft_id%253Dinfo%253Apmid%252F17486975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibXljb2xvZ2lhIjtzOjU6InJlc2lkIjtzOjg6Ijk4LzYvOTk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3927.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Some probabilistic and statistical problems in the analysis of DNA sequences</span>, pp. <span class="cit-fpage">57</span>–<span class="cit-lpage">86</span> in <span class="cit-source"><em>Lectures on Mathematics in the Life Sciences</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">R. M.</span></span>. <span class="cit-publ-name">American Mathematical Society</span>, <span class="cit-publ-loc">Providence, RI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.77" data-doi="10.3852/mycologia.98.6.838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berbee</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Dating divergences in the fungal tree of life: review and new analyses.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">838</span>–<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.stitle%253DMycologia%26rft.aulast%253DTaylor%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D838%26rft.epage%253D849%26rft.atitle%253DDating%2Bdivergences%2Bin%2Bthe%2BFungal%2BTree%2Bof%2BLife%253A%2Breview%2Band%2Bnew%2Banalyses%26rft_id%253Dinfo%253Adoi%252F10.3852%252Fmycologia.98.6.838%26rft_id%253Dinfo%253Apmid%252F17486961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibXljb2xvZ2lhIjtzOjU6InJlc2lkIjtzOjg6Ijk4LzYvODM4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.78" data-doi="10.1093/gbe/evv235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genome-scale investigation of incongruence in culicidae mosquitoes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3463</span>–<span class="cit-lpage">3471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv235%26rft_id%253Dinfo%253Apmid%252F26608059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjcvMTIvMzQ2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.79" data-doi="10.1093/nar/gks1116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tegenfeldt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D358</span>–<span class="cit-lpage">D365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1116%26rft_id%253Dinfo%253Apmid%252F23180791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9EMzU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.80" data-doi="10.1371/journal.pgen.1000942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bulk segregant analysis by high-throughput sequencing reveals a novel xylose utilization gene from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWenger%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000942%26rft.epage%253De1000942%26rft.atitle%253DBulk%2Bsegregant%2Banalysis%2Bby%2Bhigh-throughput%2Bsequencing%2Breveals%2Ba%2Bnovel%2Bxylose%2Butilization%2Bgene%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000942%26rft_id%253Dinfo%253Apmid%252F20485559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20485559&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.81" data-doi="10.1073/pnas.1503453112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocot</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moroz</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halanych</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Error, signal, and the placement of Ctenophora sister to all other animals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">5773</span>–<span class="cit-lpage">5778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503453112%26rft_id%253Dinfo%253Apmid%252F25902535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE4LzU3NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.82" data-doi="10.1073/pnas.1323926111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickett</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warnow</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phylotranscriptomic analysis of the origin and early diversification of land plants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E4859</span>–<span class="cit-lpage">E4868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323926111%26rft_id%253Dinfo%253Apmid%252F25355905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ1L0U0ODU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.83" data-doi="10.1038/42711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular evidence for an ancient duplication of the entire yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D387%26rft.issue%253D6634%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Ban%2Bancient%2Bduplication%2Bof%2Bthe%2Bentire%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42711%26rft_id%253Dinfo%253Apmid%252F9192896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192896&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD86900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.84" data-doi="10.1093/femsyr/fov035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armisén</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proux-Wera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ÓhÉigeartaigh</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Clade- and species-specific features of genome evolution in the Saccharomycetaceae.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">fov035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov035%26rft_id%253Dinfo%253Apmid%252F26066552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjExOiIxNS81L2ZvdjAzNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzkyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.85" data-doi="10.1093/sysbio/syu055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rest</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Coalescent <em>vs.</em> concatenation methods and the placement of <em>Amborella</em> as sister to water lilies.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu055%26rft_id%253Dinfo%253Apmid%252F25077515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjYzLzYvOTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.86" data-doi="10.1093/molbev/msv266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The impact of missing data on species tree estimation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">838</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv266%26rft_id%253Dinfo%253Apmid%252F26589995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMzLzMvODM4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.87" data-doi="10.1038/nrg3174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ence</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A beginner’s guide to eukaryotic genome annotation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DYandell%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D329%26rft.epage%253D342%26rft.atitle%253DA%2Bbeginner%2527s%2Bguide%2Bto%2Beukaryotic%2Bgenome%2Bannotation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3174%26rft_id%253Dinfo%253Apmid%252F22510764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22510764&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.88" data-doi="10.1007/BF00160154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D306%26rft.epage%253D314%26rft.atitle%253DMaximum%2Blikelihood%2Bphylogenetic%2Bestimation%2Bfrom%2BDNA%2Bsequences%2Bwith%2Bvariable%2Brates%2Bover%2Bsites%253A%2Bapproximate%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00160154%26rft_id%253Dinfo%253Apmid%252F7932792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00160154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7932792&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PB53900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.89" data-doi="10.1016/0169-5347(96)10041-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Among-site rate variation and its impact on phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D367%26rft.epage%253D372%26rft.atitle%253DAmong-site%2Brate%2Bvariation%2Band%2Bits%2Bimpact%2Bon%2Bphylogenetic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252896%252910041-0%26rft_id%253Dinfo%253Apmid%252F21237881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(96)10041-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21237881&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3927.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VC82300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3927.90" data-doi="10.1080/10635150290102339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zwickl</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillis</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Increased taxon sampling greatly reduces phylogenetic error.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150290102339%26rft_id%253Dinfo%253Apmid%252F12228001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUxLzQvNTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zOTI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/12/3927.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3913" title="X Chromosome Crossover Formation and Genome Stability in Caenorhabditis elegans Are Independently Regulated by xnd-1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3941" title="On Robust Association Testing for Quantitative Traits and Rare Variants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/12"><span>Volume 6 Issue 12, December 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/12.cover-source.jpg?itok=F-Hw09b0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3927.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j8kSwGLyKW2vSxZn4R3wiNQOylvZJmzvibtyLnkNwiM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/12/3927" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/12/3927" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/12/3927" class="active" data-icon-position="" data-hide-link-title="0">Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460506" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-itZQJHXhigC8bVfRY1NrhXEsfytoUrRYXJdnzxn9fXg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3927.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/12/3927" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1cYoJe_uHdhwJN18YWKFo3wn0r1QrKHx9QTCXnPV6RU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/12/3927" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3927.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460506" id="node460506--2" data-pisa="ggg;6/12/3927" data-pisa-master="ggg;g3.116.034744" data-apath="/ggg/6/12/3927.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xing-Xing Shen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiaofan Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacek Kominek</span>, <span class="highwire-citation-author" data-delta="3">Cletus P. Kurtzman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chris Todd Hittinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3927-3939;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034744 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460506--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xing-Xing Shen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xing-Xing%2BShen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shen%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXing-Xing%2BShen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaofan Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaofan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaofan Zhou</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacek Kominek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacek%2BKominek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kominek%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacek%2BKominek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacek Kominek</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cletus P. Kurtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, U.S. Department of Agriculture, Peoria, Illinois 61604</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cletus%2BP.%2BKurtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurtzman%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACletus%2BP.%2BKurtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chris Todd Hittinger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/12/3927" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/12/3927&amp;title=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/12/3927&amp;title=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/12/3927&amp;title=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/12/3927&amp;text=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/12/3927&amp;title=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/12/3927&amp;t=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/12/3927&amp;title=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/12/3927&amp;title=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460506/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460506" id="node460506--3" data-pisa="ggg;6/12/3927" data-pisa-master="ggg;g3.116.034744" data-apath="/ggg/6/12/3927.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xing-Xing Shen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiaofan Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jacek Kominek</span>, <span class="highwire-citation-author" data-delta="3">Cletus P. Kurtzman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chris Todd Hittinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3927-3939;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034744 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460506--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xing-Xing Shen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xing-Xing%2BShen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shen%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXing-Xing%2BShen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaofan Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaofan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiaofan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2879-6317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiaofan Zhou</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacek Kominek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacek%2BKominek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kominek%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJacek%2BKominek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1916-0122" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jacek Kominek</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cletus P. Kurtzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Mycotoxin Prevention and Applied Microbiology Research Unit, National Center for Agricultural Utilization Research, Agricultural Research Service, U.S. Department of Agriculture, Peoria, Illinois 61604</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cletus%2BP.%2BKurtzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kurtzman%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACletus%2BP.%2BKurtzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chris Todd Hittinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Genome Center of Wisconsin, DOE Great Lakes Bioenergy Research Center, Wisconsin Energy Institute, J. F. Crow Institute for the Study of Evolution, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BTodd%2BHittinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hittinger%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BTodd%2BHittinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5088-7461" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chris Todd Hittinger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3927&amp;rft.epage=3939&amp;rft.atitle=Reconstructing%20the%20Backbone%20of%20the%20Saccharomycotina%20Yeast%20Phylogeny%20Using%20Genome-Scale%20Data&amp;rft.volume=6&amp;rft.issue=12&amp;rft.date=2016-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shen%2C+Xing-Xing&amp;rft.au=Zhou%2C+Xiaofan&amp;rft.au=Kominek%2C+Jacek&amp;rft.au=Kurtzman%2C+Cletus+P.&amp;rft.au=Hittinger%2C+Chris+Todd&amp;rft.au=Rokas%2C+Antonis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460506/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460506/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460506/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460506/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460506/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460506/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460506/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460506/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460506/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460506/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460506/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460506/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/12/3927" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/12/3927.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/12/3927.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/12/3927.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/12/3927.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
