<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/12/3787" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/12/3787.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/12/3787.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/12/3787.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/12/3787.atom" />
<meta name="HW.pisa" content="ggg;6/12/3787" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)" />
<meta name="DC.Identifier" content="10.1534/g3.116.032805" />
<meta name="DC.Date" content="2016-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Gonzalez-Ibeas et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Sugar pine ( Pinus lambertiana Douglas) is within the subgenus Strobus with an estimated genome size of 31 Gbp. Transcriptomic resources are of particular interest in conifers due to the challenges presented in their megagenomes for gene identification. In this study, we present the first comprehensive survey of the P. lambertiana transcriptome through deep sequencing of a variety of tissue types to generate more than 2.5 billion short reads. Third generation, long reads generated through PacBio Iso-Seq have been included for the first time in conifers to combat the challenges associated with de novo transcriptome assembly. A technology comparison is provided here to contribute to the otherwise scarce comparisons of second and third generation transcriptome sequencing approaches in plant species. In addition, the transcriptome reference was essential for gene model identification and quality assessment in the parallel project responsible for sequencing and assembly of the entire genome. In this study, the transcriptomic data were also used to address questions surrounding lineage-specific Dicer-like proteins in conifers. These proteins play a role in the control of transposable element proliferation and the related genome expansion in conifers." />
<meta name="DC.Contributor" content="Daniel Gonzalez-Ibeas" />
<meta name="DC.Contributor" content="Pedro J. Martinez-Garcia" />
<meta name="DC.Contributor" content="Randi A. Famula" />
<meta name="DC.Contributor" content="Annette Delfino-Mix" />
<meta name="DC.Contributor" content="Kristian A. Stevens" />
<meta name="DC.Contributor" content="Carol A. Loopstra" />
<meta name="DC.Contributor" content="Charles H. Langley" />
<meta name="DC.Contributor" content="David B. Neale" />
<meta name="DC.Contributor" content="Jill L. Wegrzyn" />
<meta name="article:published_time" content="2016-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Sugar pine (&lt;i&gt;Pinus lambertiana&lt;/i&gt; Douglas) is within the subgenus Strobus with an estimated genome size of 31 Gbp. Transcriptomic resources are of particular interest in conifers due to the challenges presented in their megagenomes for gene identification. In this study, we present the first comprehensive survey of the &lt;i&gt;P. lambertiana&lt;/i&gt; transcriptome through deep sequencing of a variety of tissue types to generate more than 2.5 billion short reads. Third generation, long reads generated through PacBio Iso-Seq have been included for the first time in conifers to combat the challenges associated with &lt;i&gt;de novo&lt;/i&gt; transcriptome assembly. A technology comparison is provided here to contribute to the otherwise scarce comparisons of second and third generation transcriptome sequencing approaches in plant species. In addition, the transcriptome reference was essential for gene model identification and quality assessment in the parallel project responsible for sequencing and assembly of the entire genome. In this study, the transcriptomic data were also used to address questions surrounding lineage-specific Dicer-like proteins in conifers. These proteins play a role in the control of transposable element proliferation and the related genome expansion in conifers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/12/01" />
<meta name="citation_mjid" content="ggg;6/12/3787" />
<meta name="citation_id" content="6/12/3787" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/12/3787" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/12/3787.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/12/3787.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/12/3787.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032805" />
<meta name="citation_pmid" content="27799338" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3787" />
<meta name="citation_lastpage" content="3802" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel Gonzalez-Ibeas" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269" />
<meta name="citation_author" content="Pedro J. Martinez-Garcia" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Randi A. Famula" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Annette Delfino-Mix" />
<meta name="citation_author_institution" content="United States Department of Agriculture Forest Service, Institute of Forest Genetics, Placerville, California 95667" />
<meta name="citation_author" content="Kristian A. Stevens" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Carol A. Loopstra" />
<meta name="citation_author_institution" content="Department of Ecosystem Science and Management, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Charles H. Langley" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="David B. Neale" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="dbneale@ucdavis.edu" />
<meta name="citation_author_email" content="jill.wegrzyn@uconn.edu" />
<meta name="citation_author" content="Jill L. Wegrzyn" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269" />
<meta name="citation_author_email" content="dbneale@ucdavis.edu" />
<meta name="citation_author_email" content="jill.wegrzyn@uconn.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. F. Au;citation_author=V. Sebastiano;citation_author=P. T. Afshar;citation_author=J. D. Durruthy;citation_author=L. Lee;citation_title=Char-acterization of the human ESC transcriptome by hybrid sequencing.;citation_volume=110;citation_year=2013;citation_pmid=24282307;citation_doi=10.1073/pnas.1320101110" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. J. Axtell;citation_author=J. A. Snyder;citation_author=D. P. Bartel;citation_title=Common functions for diverse small RNAs of land plants.;citation_volume=19;citation_year=2007;citation_pmid=17601824;citation_doi=10.1105/tpc.107.051706" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. A. Banks;citation_author=T. Nishiyama;citation_author=M. Hasebe;citation_author=J. L. Bowman;citation_author=M. Gribskov;citation_title=The Selaginella genome identifies genetic changes associated with the evolution of vascular plants.;citation_volume=332;citation_year=2011;citation_pmid=21551031;citation_doi=10.1126/science.1203810" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. J. Cahill;citation_author=C. U. Köser;citation_author=N. E. Ross;citation_author=J. A. C. Archer;citation_title=Read length and repeat resolution: exploring prokaryote genomes using next-generation sequencing technologies.;citation_pages=e11518-e11518;citation_volume=5;citation_year=2010;citation_issue=7;citation_pmid=20634954;citation_doi=10.1371/journal.pone.0011518" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Conesa;citation_author=P. Madrigal;citation_author=S. Tarazona;citation_author=D. Gomez-Cabrero;citation_author=A. Cervera;citation_title=A survey of best practices for RNA-seq data analysis.;citation_pages=13;citation_volume=17;citation_year=2016;citation_pmid=26813401;citation_doi=10.1186/s13059-016-0881-8" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=C. Coruh;citation_author=S. Shahid;citation_author=M. J. Axtell;citation_title=Seeing the forest for the trees: annotating small RNA producing genes in plants.;citation_pages=87-95;citation_volume=18;citation_year=2014;citation_pmid=24632306;citation_doi=10.1016/j.pbi.2014.02.008" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. Coruh;citation_author=S. H. Cho;citation_author=S. Shahid;citation_author=Q. Liu;citation_author=A. Wierzbicki;citation_title=Comprehensive annotation of Physcomitrella patens small RNA loci reveals that the heterochromatic short interfering RNA pathway is largely conserved in land plants.;citation_volume=27;citation_year=2015;citation_pmid=26209555;citation_doi=10.1105/tpc.15.00228" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. T. Cuperus;citation_author=N. Fahlgren;citation_author=J. C. Carrington;citation_title=Evolution and functional diversification of MIRNA genes.;citation_volume=23;citation_year=2011;citation_pmid=21317375;citation_doi=10.1105/tpc.110.082784" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=E. V. Dolgosheina;citation_author=R. D. Morin;citation_author=G. Aksay;citation_author=S. C. Sahinalp;citation_author=V. Magrini;citation_title=Conifers have a unique small RNA silencing signature.;citation_volume=14;citation_year=2008;citation_pmid=18566193;citation_doi=10.1261/rna.1052008" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. L. Dong;citation_author=H. F. Liu;citation_author=J. C. Zhang;citation_author=S. J. Yang;citation_author=G. Y. Kong;citation_title=Single-molecule real-time transcript sequencing facilitates common wheat genome annotation and grain transcriptome research.;citation_pages=1039;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Bacterial Molecular Networks;citation_title=Using MCL to extract clusters from networks;citation_author=S. Dongen;citation_author=C. Abreu-Goodger;citation_pages=281-295;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. C. Edgar;citation_title=Search and clustering orders of magnitude faster than BLAST.;citation_volume=26;citation_year=2010;citation_pmid=20709691;citation_doi=10.1093/bioinformatics/btq461" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. J. Enright;citation_author=S. Van Dongen;citation_author=C. A. Ouzounis;citation_title=An efficient algorithm for large-scale detection of protein families;citation_pages=1575-1584;citation_volume=30;citation_year=2002;citation_issue=7;citation_pmid=11917018;citation_doi=10.1093/nar/30.7.1575" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. X. Feng;citation_author=C. A. Meyer;citation_author=Q. Wang;citation_author=J. S. Liu;citation_author=X. S. Liu;citation_title=GFOLD: a generalized fold change for ranking differentially expressed genes from RNA-seq data.;citation_volume=28;citation_year=2012;citation_pmid=22923299;citation_doi=10.1093/bioinformatics/bts515" />
<meta name="citation_reference" content="citation_title=Montane and Subalpine Vegetation of the Sierra Nevada and Cascade Ranges;citation_author=J. A. Fites-Kaufman;citation_author=P. Rundel;citation_author=N. Stephenson;citation_author=D. A. Weixelman;citation_pages=456-501;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. C. Glenn;citation_title=Field guide to next-generation DNA sequencers.;citation_pages=759-769;citation_volume=11;citation_year=2011;citation_pmid=21592312;citation_doi=10.1111/j.1755-0998.2011.03024.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. Huang;citation_author=T. Kendall;citation_author=E. S. Forsythe;citation_author=A. Dorantes-Acosta;citation_author=S. Li;citation_title=Ancient origin and recent innovations of RNA polymerase IV and V.;citation_volume=32;citation_year=2015;citation_pmid=25767205;citation_doi=10.1093/molbev/msv060" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Hunter;citation_author=P. Jones;citation_author=A. Mitchell;citation_author=R. Apweiler;citation_author=T. K. Attwood;citation_title=InterPro in 2011: new developments in the family and domain prediction database.;citation_volume=40;citation_year=2012;citation_pmid=22096229;citation_doi=10.1093/nar/gkr948" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Kanitz;citation_author=F. Gypas;citation_author=A. J. Gruber;citation_author=A. R. Gruber;citation_author=G. Martin;citation_title=Comparative assessment of methods for the computational inference of transcript isoform abundance from RNA-seq data.;citation_pages=150;citation_volume=16;citation_year=2015;citation_pmid=26201343;citation_doi=10.1186/s13059-015-0702-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_title=Silvics of North America;citation_title=Pinus lambertiana Dougl;citation_author=B. B. Kinloch;citation_author=W. H. Scheuner;citation_pages=370-379;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=T. Kleine;citation_title=Arabidopsis thaliana mTERF proteins: evolution and functional classification.;citation_pages=233-233;citation_volume=3;citation_year=2012;citation_pmid=23087700" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=I. Korf;citation_title=Genomics: the state of the art in RNA-seq analysis.;citation_pages=1165-1166;citation_volume=10;citation_year=2013;citation_pmid=24296473;citation_doi=10.1038/nmeth.2735" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Kozomara;citation_author=S. Griffiths-Jones;citation_title=miRBase: annotating high confidence microRNAs using deep sequencing data.;citation_volume=42;citation_year=2014;citation_pmid=24275495;citation_doi=10.1093/nar/gkt1181" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=I. J. Leitch;citation_author=L. Hanson;citation_author=M. Winfield;citation_author=J. Parker;citation_author=M. D. Bennett;citation_title=Nuclear DNA C-values Complete Familial Representation in Gymnosperms;citation_pages=843-849;citation_volume=88;citation_year=2001;citation_issue=5;citation_doi=10.1006/anbo.2001.1521" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. Li;citation_author=G. Fan;citation_author=K. Wang;citation_author=F. Sun;citation_author=Y. Yuan;citation_title=Genome sequence of the cultivated cotton Gossypium arboreum.;citation_pages=567-572;citation_volume=46;citation_year=2014;citation_issue=6;citation_pmid=24836287;citation_doi=10.1038/ng.2987" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=S. Li;citation_author=S. W. Tighe;citation_author=C. M. Nicolet;citation_author=D. Grove;citation_author=S. Levy;citation_title=Multi-platform assessment of transcriptome profiling using RNA-seq in the ABRF next-generation sequencing study.;citation_pages=1166;citation_volume=32;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=Z. Li;citation_author=A.E. Baniaga;citation_author=E.B. Sessa;citation_author=M. Scascitelli;citation_author=S. W. Graham;citation_title=Early genome duplications in conifers and other seed plants;citation_pages=e1501084-e1501084;citation_volume=1;citation_year=2015;citation_issue=10;citation_pmid=26702445;citation_doi=10.1126/sciadv.1501084" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biomed Biotechnol;citation_author=L. Liu;citation_author=Y. H. Li;citation_author=S. L. Li;citation_author=N. Hu;citation_author=Y. M. He;citation_title=Comparison of next-generation sequencing systems.;citation_pages=251364-251364;citation_volume=2012;citation_year=2012;citation_pmid=22829749;citation_doi=10.1155/2012/251364" />
<meta name="citation_reference" content="citation_journal_abbrev=Algorithms Mol Biol;citation_author=R. Lorenz;citation_author=S. H. Bernhart;citation_author=C. H. Z. Siederdissen;citation_author=H. Tafer;citation_author=C. Flamm;citation_title=ViennaRNA Package 2.0.;citation_pages=26-26;citation_volume=6;citation_year=2011;citation_pmid=22115189;citation_doi=10.1186/1748-7188-6-26" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. W. Lorenz;citation_author=R. Alba;citation_author=Y.-S. Yu;citation_author=J. M. Bordeaux;citation_author=M. Simões;citation_title=Microarray analysis and scale-free gene networks identify candidate regulators in drought-stressed roots of loblolly pine (P. taeda L.).;citation_pages=264-264;citation_volume=12;citation_year=2011;citation_pmid=21609476;citation_doi=10.1186/1471-2164-12-264" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=L. Ma;citation_author=A. Hatlen;citation_author=L. J. Kelly;citation_author=H. Becher;citation_author=W. C. Wang;citation_title=Angiosperms are unique among land plant lineages in the occurrence of key genes in the RNA-directed DNA methylation (RdDM) pathway.;citation_volume=7;citation_year=2015;citation_pmid=26338185;citation_doi=10.1093/gbe/evv171" />
<meta name="citation_reference" content="citation_title=Genetics, Genomics and Breeding of Conifers;citation_title=Transcriptomics;citation_author=J. Mackay;citation_author=J. F. D. Dean;citation_pages=323-357;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. D. Malone;citation_author=A. M. Anderson;citation_author=J. A. Motl;citation_author=C. H. Rexer;citation_author=D. L. Chalker;citation_title=Germ Line Transcripts Are Processed by a Dicer-Like Protein That Is Essential for Developmentally Programmed Genome Rearrangements of Tetrahymena thermophila;citation_pages=9151-9164;citation_volume=25;citation_year=2005;citation_issue=20;citation_pmid=16199890;citation_doi=10.1128/MCB.25.20.9151-9164.2005" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=P. E. Maloney;citation_author=D. R. Vogler;citation_author=A. J. Eckert;citation_author=C. E. Jensen;citation_author=D. B. Neale;citation_title=Population biology of sugar pine (Pinus lambertiana Dougl.) with reference to historical disturbances in the Lake Tahoe Basin: implications for restoration.;citation_pages=770-779;citation_volume=262;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=R. Margis;citation_author=A. F. Fusaro;citation_author=N. A. Smith;citation_author=S. J. Curtin;citation_author=J. M. Watson;citation_title=The evolution and diversification of Dicers in plants.;citation_pages=2442-2450;citation_volume=580;citation_year=2006;citation_issue=10;citation_pmid=16638569;citation_doi=10.1016/j.febslet.2006.03.072" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Martin;citation_author=Z. Wang;citation_title=Next-generation transcriptome assembly.;citation_pages=671-682;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21897427;citation_doi=10.1038/nrg3068" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Mathelier;citation_author=A. Carbone;citation_title=MIReNA: finding microRNAs with high accuracy and no learning at genome scale and from deep sequencing data.;citation_volume=26;citation_year=2010;citation_pmid=20591903;citation_doi=10.1093/bioinformatics/btq329" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. A. Matzke;citation_author=R. A. Mosher;citation_title=RNA-directed DNA methylation: an epigenetic pathway of increasing complexity.;citation_pages=394-408;citation_volume=15;citation_year=2014;citation_pmid=24805120;citation_doi=10.1038/nrg3683" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=B. C. Meyers;citation_author=M. J. Axtell;citation_author=B. Bartel;citation_author=D. P. Bartel;citation_author=D. Baulcombe;citation_title=Criteria for annotation of plant microRNAs.;citation_volume=20;citation_year=2008;citation_pmid=19074682;citation_doi=10.1105/tpc.108.064311" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=R. A. C. Montes;citation_author=F. F. Rosas-Cardenas;citation_author=E. De Paoli;citation_author=M. Accerbi;citation_author=L. A. Rymarquis;citation_title=Sample sequencing of vascular plants demonstrates widespread conservation and divergence of microRNAs.;citation_pages=3722;citation_volume=5;citation_year=2014;citation_pmid=24759728" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. Moreton;citation_author=A. Izquierdo;citation_author=R. D. Emes;citation_title=Assembly, assessment, and availability of de novo generated eukaryotic transcriptomes.;citation_pages=361;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Mol. Plant Pathol.;citation_author=H. Myburg;citation_author=A. M. Morse;citation_author=H. V. Amerson;citation_author=T. L. Kubisiak;citation_author=D. Huber;citation_title=Differential gene expression in loblolly pine (Pinus taeda L.) challenged with the fusiform rust fungus, Cronartium quercuum f.sp fusiforme.;citation_pages=79-91;citation_volume=68;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. B. Neale;citation_author=J. L. Wegrzyn;citation_author=K. A. Stevens;citation_author=A. V. Zimin;citation_author=D. Puiu;citation_title=Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.;citation_pages=R39;citation_volume=15;citation_year=2014;citation_pmid=24576357;citation_doi=10.1186/gb-2014-15-2-r39" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. H. Niu;citation_author=C. Liu;citation_author=H. W. Yuan;citation_author=P. Li;citation_author=W. Li;citation_title=Identification and expression profiles of sRNAs and their biogenesis and action-related genes in male and female cones of Pinus tabuliformis.;citation_pages=693;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y. C. Lin;citation_title=The Norway spruce genome sequence and conifer genome evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. N. Price;citation_author=P. S. Dehal;citation_author=A. P. Arkin;citation_title=FastTree 2--approximately maximum-likelihood trees for large alignments.;citation_pages=e9490-e9490;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20224823;citation_doi=10.1371/journal.pone.0009490" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Proost;citation_author=M. Van Bel;citation_author=D. Vaneechoutte;citation_author=Y. Van de Peer;citation_author=D. Inzé;citation_title=PLAZA 3.0: an access point for plant comparative genomics.;citation_volume=43;citation_year=2015;citation_issue=Database issue;citation_pmid=25324309;citation_doi=10.1093/nar/gku986" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Quail;citation_author=M. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Quast;citation_author=E. Pruesse;citation_author=P. Yilmaz;citation_author=J. Gerken;citation_author=T. Schweer;citation_title=The SILVA ribosomal RNA gene database project: improved data processing and web-based tools.;citation_volume=41;citation_year=2013;citation_pmid=23193283;citation_doi=10.1093/nar/gks1219" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=P. J. Rushton;citation_author=I. E. Somssich;citation_author=P. Ringler;citation_author=Q. J. Shen;citation_title=WRKY transcription factors.;citation_pages=247-258;citation_volume=15;citation_year=2010;citation_issue=5;citation_pmid=20304701;citation_doi=10.1016/j.tplants.2010.02.006" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=J. S. Sangha;citation_author=K. Gu;citation_author=J. Kaur;citation_author=Z. Yin;citation_title=An improved method for RNA isolation and cDNA library construction from immature seeds of Jatropha curcas L.;citation_pages=126-126;citation_volume=3;citation_year=2010;citation_pmid=20444276;citation_doi=10.1186/1756-0500-3-126" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=D. Sharon;citation_author=H. Tilgner;citation_author=F. Grubert;citation_author=M. Snyder;citation_title=A single-molecule long-read survey of the human transcriptome.;citation_pages=1009-1014;citation_volume=31;citation_year=2013;citation_pmid=24108091;citation_doi=10.1038/nbt.2705" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=C. Song;citation_author=M. Yu;citation_author=J. Han;citation_author=C. Wang;citation_author=H. Liu;citation_title=Validation and characterization of Citrus sinensis microRNAs and their target genes.;citation_pages=235-235;citation_volume=5;citation_year=2012;citation_pmid=22583737" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Song;citation_author=M.-H. Han;citation_author=J. Lesicka;citation_author=N. Fedoroff;citation_title=Arabidopsis primary microRNA processing proteins HYL1 and DCL1 define a nuclear body distinct from the Cajal body.;citation_volume=104;citation_year=2007;citation_pmid=17369351;citation_doi=10.1073/pnas.0701061104" />
<meta name="citation_reference" content="citation_title=Sequence of the Sugar Pine Megagenome Genetics.;citation_author=K. A. Stevens;citation_author=J. L. Wegrzyn;citation_author=A. Zimin;citation_author=D. Puiu;citation_author=M. Crepeau;citation_pages=1613-1626;citation_volume=204;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=D. Sundell;citation_author=C. Mannapperuma;citation_author=S. Netotea;citation_author=N. Delhomme;citation_author=Y. Lin;citation_title=The plant genome integrative explorer resource: PlantGenIE.org.;citation_pages=1149-1156;citation_volume=208;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Swarbreck;citation_author=C. Wilks;citation_author=P. Lamesch;citation_author=T. Z. Berardini;citation_author=M. Garcia-Hernandez;citation_title=The Arabidopsis information resource (TAIR): gene structure and function annotation.;citation_volume=36;citation_year=2008;citation_pmid=17986450;citation_doi=10.1093/nar/gkm965" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Thomas;citation_author=J. G. Underwood;citation_author=E. Tseng;citation_author=A. K. Holloway;citation_title=Long-read sequencing of chicken transcripts and identification of new transcript isoforms.;citation_pages=e94650;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Treutlein;citation_author=O. Gokce;citation_author=S. R. Quake;citation_author=T. C. Sudhof;citation_title=Cartography of neurexin alternative splicing mapped by single-molecule long-read mRNA sequencing.;citation_volume=111;citation_year=2014;citation_pmid=24639501;citation_doi=10.1073/pnas.1403244111" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=E. L. Van Dijk;citation_author=H. Auger;citation_author=Y. Jaszczyszyn;citation_author=C. Thermes;citation_title=Ten years of next-generation sequencing technology.;citation_pages=418-426;citation_volume=30;citation_year=2014;citation_pmid=25108476;citation_doi=10.1016/j.tig.2014.07.001" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. A. Visser;citation_author=J. L. Wegrzyn;citation_author=E. T. Steenkmap;citation_author=A. A. Myburg;citation_author=S. Naidoo;citation_title=Combined de novo and genome guided assembly and annotation of the Pinus patula juvenile shoot transcriptome.;citation_pages=1057;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Wegrzyn;citation_author=J. D. Liechty;citation_author=K. A. Stevens;citation_author=L. S. Wu;citation_author=C. A. Loopstra;citation_title=Unique Features of the Loblolly Pine (Pinus taeda L.) Megagenome Revealed Through Sequence Annotation;citation_pages=891-909;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653211;citation_doi=10.1534/genetics.113.159996" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Xia;citation_author=J. Xu;citation_author=S. Arikit;citation_author=B. C. Meyers;citation_title=Extensive families of miRNAs and PHAS loci in Norway spruce demonstrate the origins of complex phasiRNA networks in seed plants.;citation_volume=32;citation_year=2015;citation_pmid=26318183;citation_doi=10.1093/molbev/msv164" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=Z. C. Xu;citation_author=R. J. Peters;citation_author=J. Weirather;citation_author=H. M. Luo;citation_author=B. S. Liao;citation_title=Full-length transcriptome sequences and splice variants obtained by a combination of sequencing platforms applied to different root tissues of Salvia miltiorrhiza and tanshinone biosynthesis.;citation_pages=951-961;citation_volume=82;citation_year=2015;citation_pmid=25912611;citation_doi=10.1111/tpj.12865" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. Zhai;citation_author=D.-H. Jeong;citation_author=E. D. Paoli;citation_author=S. Park;citation_author=B. D. Rosen;citation_title=MicroRNAs as master regulators of the plant NB-LRR defense gene family via the production of phased, trans-acting siRNAs;citation_pages=2540-2553;citation_volume=25;citation_year=2011;citation_issue=23;citation_pmid=22156213;citation_doi=10.1101/gad.177527.111" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=B. H. Zhang;citation_author=X. P. Pan;citation_author=C. H. Cannon;citation_author=G. P. Cobb;citation_author=T. A. Anderson;citation_title=Conservation and divergence of plant microRNA genes.;citation_pages=243-259;citation_volume=46;citation_year=2006;citation_issue=2;citation_pmid=16623887;citation_doi=10.1111/j.1365-313X.2006.02697.x" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=B. H. Zhang;citation_author=X. P. Pan;citation_author=S. B. Cox;citation_author=G. P. Cobb;citation_author=T. A. Anderson;citation_title=Evidence that miRNAs are different from other RNAs.;citation_pages=246-254;citation_volume=63;citation_year=2006;citation_issue=2;citation_pmid=16395542;citation_doi=10.1007/s00018-005-5467-7" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. H. Zhang;citation_author=S. G. Zhang;citation_author=S. Y. Han;citation_author=X. M. Li;citation_author=Z. K. Tong;citation_title=Deciphering small noncoding RNAs during the transition from dormant embryo to germinated embryo in larches (Larix leptolepis).;citation_pages=e81452;citation_volume=8;citation_year=2013;citation_pmid=24339932;citation_doi=10.1371/journal.pone.0081452" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Zheng;citation_author=B. A. Schulman;citation_author=L. Z. Song;citation_author=J. J. Miller;citation_author=P. D. Jeffrey;citation_title=Structure of the Cul1-Rbx1-Skp1-F boxSkp2 SCF ubiquitin ligase complex.;citation_pages=703-709;citation_volume=416;citation_year=2002;citation_issue=6882;citation_pmid=11961546;citation_doi=10.1038/416703a" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="twitter:description" content="Sugar pine ( Pinus lambertiana Douglas) is within the subgenus Strobus with an estimated genome size of 31 Gbp. Transcriptomic resources are of particular interest in conifers due to the challenges presented in their megagenomes for gene identification. In this study, we present the first comprehensive survey of the P. lambertiana transcriptome through deep sequencing of a variety of tissue types to generate more than 2.5 billion short reads. Third generation, long reads generated through PacBio Iso-Seq have been included for the first time in conifers to combat the challenges associated with de novo transcriptome assembly. A technology comparison is provided here to contribute to the otherwise scarce comparisons of second and third generation transcriptome sequencing approaches in plant species. In addition, the transcriptome reference was essential for gene model identification and quality assessment in the parallel project responsible for sequencing and assembly of the entire genome. In this study, the transcriptomic data were also used to address questions surrounding lineage-specific Dicer-like proteins in conifers. These proteins play a role in the control of transposable element proliferation and the related genome expansion in conifers." />
<meta name="og-title" property="og:title" content="Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/12/3787" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Sugar pine ( Pinus lambertiana Douglas) is within the subgenus Strobus with an estimated genome size of 31 Gbp. Transcriptomic resources are of particular interest in conifers due to the challenges presented in their megagenomes for gene identification. In this study, we present the first comprehensive survey of the P. lambertiana transcriptome through deep sequencing of a variety of tissue types to generate more than 2.5 billion short reads. Third generation, long reads generated through PacBio Iso-Seq have been included for the first time in conifers to combat the challenges associated with de novo transcriptome assembly. A technology comparison is provided here to contribute to the otherwise scarce comparisons of second and third generation transcriptome sequencing approaches in plant species. In addition, the transcriptome reference was essential for gene model identification and quality assessment in the parallel project responsible for sequencing and assembly of the entire genome. In this study, the transcriptomic data were also used to address questions surrounding lineage-specific Dicer-like proteins in conifers. These proteins play a role in the control of transposable element proliferation and the related genome expansion in conifers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
    <title>Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UV1snZPAJjH9t3nV2NeCd2CwD0Ju8eDB8ZepgS-7wr0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460510","apath":"\/ggg\/6\/12\/3787.atom","pisa":"ggg;6\/12\/3787","ac":{"\/ggg\/6\/12\/3787.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/12\/3787.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3787"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3787"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/12\/3787":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OhCbRdzD6CKI8XaeksRX3HbDQYUgaT1eZhMgDrwHu3zcbIfpla4PR7ewPNyiT03031tcv59SCtLRVhefAQ3T\\\\\\\/MjSeg1dh8Gdl3fR1ux2wCQIaQAAyW9ltkaJCnmqweP11B7548kwcHFW2d\\\\\\\/FSXvRFg5XE7FtdiqFcC+oPNxPhjHm\\\\\\\/uW1bHYv6clejxwTDi\\\\\\\/ve+XXsB63ofZBt5k02eT0gW6so7FpIMabL2rxbON13J8cs450n9On\\\\\\\/usdAeOnHSfO4G5q5YanfTCm11cX8D4rb5h4g0w61PzjyzYL91grELyWEQRzF\\\\\\\/D9sByBphtYU1yfDCwVxwQ9So2xB89+wQ57wvpBbG9dJcItrXD4eHgb3sYlDhmlRO1zs\\\\\\\/C8Fht4UfZLK\\\\\\\/Ue+CyTKZ3TBYAdtjcqh9cxaxmXL9JJm+wk+k0k82Cbp2PHPMn7a3JqkFBNgS04lPD\\\\\\\/1eQFHP+oULoyg4gWv4NVWr+LRx5JDYwtMKVfcNFIPfJqrwRC+5rkvV1NUAUJ4kjt7GYMFlEd\\\\\\\/ou9owi2sFkACAVmilA6KBGm7ScjFhM\\\\\\\/LldhGnKttkpglAXyGwC5E7+3vLtdU2yh\\\\\\\/eL9kldXgmWZIivsDSHYes0lxBioRyN57niuo7EIF\\\\\\\/O6m5hThGHHKrJHF+NmFRbvj+Gu1kI2MATGZGgXPRm0gjWQj3fsceQkVX9ErcMn51kMAfA8GiBTbIrsTToUa7GRR0UeqafmAKtoao3rnA9YtdRVJ7\\\\\\\/t3\\\\\\\/rUV1bkyvsehvu0Frq43dVMkVhpPt97EJKeKTkCnDGDLjLyzhI+AlN46UhPs1wwMK0hdq1luEY\\\\\\\/5WbiiDgVaJmY9gzu1vJd8BPhl6Mij8TYGxnh9hrqUgYNMVD4wsFZfEszSEfu\\\\\\\/yKLSwCKSPg9s2TGTQNml\\\\\\\/pr6V0g+PhVTJ2KwdZgEp2ndHM18a6TdBBMy+pKogrKytVl5ht31H\\\\\\\/ho\\\\\\\/gjSaaZ8ukK9F+GtwK7jGXvueM2DE0+IbLUp5eKvKtowd6eVyzuYPkztMWJvbeUPNJU31KpnBKJ30+9D1AednetNqS3Mumtizzi1U2tP\\\\\\\/HAP9HjjVJb3ZUb53RGdsZ\\\\\\\/ZyG4aO1shAgSiFeuzrYB5v1e7d3eSIwmXZxElg50vddgrZxFBpTDBJorxD3NVYtMDmlEu7oE6Vuxcer1Gb5QmbXw\\\\\\\/G05NBoZtuU9FwPSnskU3N+t0O6YW9OebRm1\\\\\\\/JcjqH8CRDwH632Zr\\\\\\\/ZKWDzoOUs6M17rlRsHvPJVzbhEuWJDr8fkzure44HGUqSqNJazzjya\\\u0022,\\\u0022iv\\\u0022:\\\u00226uKJT3yNyQvBC5q8cm05NA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c74d30bade50453eb9c9bac72ad33f872ff63047bf0b90b01120f8c120cf494d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cBW1tOIGcsN\\\\\\\/wUgYqncJ7btpZr2R6vqyc8DLp28PGa+LLSXl3pWiy6jZqcIQ0N56Af\\\\\\\/tPzhFlSBF3FYAa78Fa\\\\\\\/wXbKv1raX4oXdA7c+RDqDxDJEqIE5tsCCRLrhatOQazPq6BmpszU34JxBDg4piD\\\\\\\/PogqJcCuHNRtfPXCitiLgzo+QQU6nEqhgNIgpiRtXG4HgRFXupnpsmrbJWWEKd7uUudHz5WKD1xbLVGZfK7TtTooJxIX0B1ATIUlNGQr7L5fRBo7hM0oVJW\\\\\\\/MylqXPjO6Yht7NwoUR2i7znhYY\\\\\\\/fERX42JPTX1e\\\\\\\/514WtFYaVOrE+Qw6ebZmCTw\\\\\\\/CPVuI8pQzUpg3BSQ7HUN1+kDFxBynk\\\\\\\/NwEOTYOwOxfgiOSLPJCSytzFGKO9waUC+HEYU7IogxFWOga3zmcQplWVUG6wXmmQMzBDUN869e\\\\\\\/S2JpSg6xtWOYATnuNDvkstts30DQDQjz2\\\\\\\/S2aJyCXEEXvLwa63FMx58VqfHLxHPppHE2uQANr10ATxxg2d4+yTDq6jGbhNm7mUHGsPCXtrPPS\\\\\\\/EjlqyQzeOXHrXVE6TFqkBGRNZG5wcNxcoVc2aeSvowJUU\\\\\\\/xuNqPOjAx1cLZckZK3e55LUGtlqCGALKJmeXXQz\\\\\\\/ajcHINMeqJ1chJ2kuhbpK6V+eBbgZadulUrMWS\\\\\\\/8aPS2mVOsWp8YWul43NM2mSSQuBWE\\\\\\\/3iA5gpzR7zm8A8vtdoxV9bWRXcSkxhWtaUqarGZu0zs7paC4Yu0hhYQXw0GOAFdxgiQITES\\\\\\\/gbQOa618t34fsGfhXxK8ETNgRWh6irA1Gzb+PbWxaZOjQWeLY7hhTlnF5pzeShAvMrQeVXie0sUylfHbizhBv5tf\\\\\\\/E7wXIuANhxk5QE4UvA21zyJW6JCtrHODTvXin6fLRg9sSwvXLbYL0Z1WkT6lEsaMlOkC9SeY4fu7GbnEXS7X\\\\\\\/Pe3u3TplGdMNT\\\\\\\/oXd4BDagIvDWdkBYz37IBSA3UFTRoFkWkymM13T16DvZi2nzEAsWmtmPLJMkaiOjt2YA8i2uszPDIPot\\\\\\\/ISsJ3qhbD8kzBhbVBFFGpQkdv2z8pFfqWloJEJdeESMqjuXd1UVhqe\\\\\\\/YKDEsL1UQ17wDrrRw6E9GlMMR08j81\\\\\\\/NBDH9G4vfTO1UEQNig1IKtXVeiOcprE5uxrQBMVGo3pw9e80QSdtfKCGKsBm0WUj6JYAuQznE9Jw8v5i+KyH5I4xH52ww\\\\\\\/s8shjJSLWSJtR54EPRyLbSEQWFAEjKfSn3PldD4phldSKtAeUzm3TIi5UhxVgkyMJnr2HKdLw0Zam+C9CVnA1et0KS+joYAUzDLWAF\\\\\\\/pWfSdK9bxBjbMoNowwDnZqtJQv2ruhnlh5WdsENEsQVB2147qeebXsnq2Q6c7yBdU\\\\\\\/xmtdzZcWb3aFEqeYdkoOCs32FURmX\\\\\\\/44DBlTXXezuCh3J0lLyyblp6yKE1CdcxDMTK0yJZ6Qa460oR2iH10l4PaFsPi+bj612WJWKleNk7yu7xOklOg8UF0D9pUN3P37gG+mFBU22MJqgIk3JTskv\\\\\\\/72l4FUCd8f2q56iXDzQVH2gEfiYYU8V2ZG8k9nUKHBWeRjtNx0b8V3Yh59TLdcaeLuXhhvNZr10s7T0SIxBzmbdSZxvO6yK3Q\\\\\\\/4EDA94w3JyeVW4wTb0XC7E5ndHjZFdDfRvRpQ\\\\\\\/0\\\\\\\/QJnHO+TbFTJ5B8EF66qG4w6NLKovK2K4Uu0FaspesOeWtGuMnKo5I5g9hGY1NXNpW51qziD1TRZjmtVpLaEVS4K47SX09Bb8b6w\\\\\\\/BgT6\\\\\\\/qcSpatyIAkycTI4XXFlDZYeqRq5IGWa+lpFMMckFSWOodPIeZfQ5\\\\\\\/IZ8gxVTm69v2QlcDNlXUKTIZ71QDA+nI501DHwvQ8vbzfZC6FQmea9NvqPfl8J7cFFbYRBlCUHopjUVRcIUDcoEv3kMWzfVzVTVnfawYZf219bc8+gPOww9dLzItJXnQRFK1vcb7unJ4d2InuMra7+g+ET5YqlNt32KMyKyCTPdSClgVSD1PjWpAL3ThxQm2wF1Oi7Y0wSygToAsaRg+919t5Kvo7fFCzLuA0knqoPI\\\\\\\/QTNlE5P+lTK3EwkTXllYziNIM6vzNn8gS0tUIBU1w33YkI+FQ9ZYw2TP3SVitdMUfMuE4bK07usHDRRV2qFhhop5GTjNa6x5tcMu8BXWH7\\\\\\\/BUP5d9fQBBgmcRDcHDTvJ3OKYzM9x2D+Rv5C3R997siQWPdhWNeHVLn\\\\\\\/iqnXbyKJJBNPtdJnOT7BdxIorG7YZznVnoJ\\\\\\\/rPn8lbELwURqpo4Pb3qTD\\\\\\\/wX6OSkdhXjWqoZ2xYgdCcRGJK2VJY7HLYzK8KTJ7cYOapblxJEL\\\\\\\/fG9QuZ0WVGUuge0mocOSlB5YY1J9NbVLKiKD5pvAxDLttypUGz54zQbEwuM+wIfJBGPBbEUJES5CHzdCBxIRJRw0v0EeYEY+S76HBktm6tF8ti48EQYATbevN3CMFwLW+uW4bStPN4lDLCIyH1ZzqrSV6YantQsFeZBnPs48vn5Dix17zpoqn1A5b6k1rz6YIcJP3F\\\\\\\/9m1rhO+yhVSpmkzBJJumfS1yphP0mcJOYJkY4675HySb9oKyrE4IIRVfYIdgsCbjnEX8Uq+zzXhvoUwAVGvyTDlyUwiTmTaL6zT9wmNh3Xp2do7CbuIMGGtNW+Ox\\\\\\\/AKW44ft4LnCiTMfqsHdJAMESbpdqMtPlc9dK6F3mUen7N7gMwoIjWUT2AwHqPWiqEzgV9kYMZs4UgiVEMoCyAsZgGKlrahNqDEyNiITFOIRfO7xuhhEtKiFsXqo8EdC1n1k6stEQgG2JoPCEYeZIJk1sUXw1EWs1F8UssKzGnVjFr0UKq\\\\\\\/4DM2\\\\\\\/3cXawGLx8A0OPgxpO9Lol6q9GZ+7YeDx\\\\\\\/OZsLfDL3+FLx7gs3L6YC6n2M9A86nUddfosdsUqTDSPrIueW0DVAdjhau6rNUKXpSh8WuuxxoTqib5roobdjYFoZZdVoxb3RJ1iowBnKqBAvVvKTOI2ZIU2ex6oPgC4ny1WsdRtyXUhlw1hkRFO2LUnOERRofZAvIxRnEWL1N4TxDDdX98aBGjhiG\\\\\\\/jUw+veN4rLv8NMl4O74lySpZUjMRBHGfE2B2tqO8jl6OiSNyoh\\\\\\\/ZVZmBISoPwZdU0oE9BIhF1997fjMYINUv+7oHIPD+VZsti2l+BYyW5lyi7zmms\\\\\\\/4CWOjV0918+qCRheX5LyrPHvEf02aDVv6FFWYEepKrrRiMSb1TQ5OWHQUpdaOgSAe5PJs4rfBtivI5XJQeMUQsyheZz1xpuq3z6gBMcGBJP5Y3YQBT+hB1rEBSQmnzv575nQrUHnL996i9wD7AfxtikhugSHtjCCAK6YZ6dunqDwcTOackitYHS\\\\\\\/HCUwxfAZRjRZ4WSnPBWj+bm+4ye9VBhWd6Z0Cbn\\\\\\\/zH3LpXjc36YdT+KGV+IJIEAHuHTcwnXfL3uRO5DLqei5OXuWFmRfn0tXwxUjdl\\\\\\\/JiRQ112OERdA3WcHNObbILw\\\\\\\/VT4EE6It8PZ9y8YGFzOQiZeiWQnNXWmgeZH6YuCNhpijXrCz7trEBrYFkJN1G+cEdxs7tOgtx7gBBtRqcIUABSxYD8h6ctXkKqThAzENEEYA2YgpfwMJqhRKPSt\\\\\\\/dvbUU9ipXpnuP\\\\\\\/I+fAJlO8dUOtQ+9\\\\\\\/h1Vr1001cKSh+lm7YlEKxQ44HNQUNt+fH2LKrLTpwgMPouQbCjTK\\\\\\\/mGQKDcG4qfdk27W7P+z5MWxxc26rqFfHjDTt842gA6ATkiyRpShjQWDoAkokJS0+AD4nWeMSBPI4ilJH6F40YkUBkouJYzRU9gnw2JqF8Ln9YF+U4lUiQyyFgpBHzcnVDjo2FgzYC0UG3o11resPcQfyrgVP0ZcVE7cBADTDDcJsVbdFxSdIYr9VAmeNVjqrmZWqqkhTAGVsEWV7ywYe4nVQ7Ja7Oz+wHvEOKog+KeJGSt4vEZOyMWiA4s0nEPTZOVQbeF7QEOrRFXqHFBEl5pOd9ipnrFhWjiYSHFKPqSmmb35aJGSfaZ+dRMHj\\\\\\\/z3rnTVPoGCM1jvpfufMuwS7T6IALdpm3s3JaKZMhiX2OZQ8+T3kwF4SLClv0GKMCXo1+w\\\\\\\/+PShzcWUIXysIBuY3EV\\\\\\\/HeEObwaL3zvrcJ8wl3i6iSFMUFoq+bb82vi32zNpMooejoILmha9PPKnW+vCnFR0Gr5yBZFGo+sVsiDpLChc0OTsnF+QpqSRO04BQb7Tt4xGIonB12MNpD2KBwQOXkqhUI6\\\\\\\/YY3vjJLmR7PVdbpVciOqysqgW6rEyc3651STkos9vqg0u+YIWRa5ImBPvw9jalx8inIFYDTnYQW2Dx6ghzf70UfoNWg03bn1cy0dEo6usLRC\\\\\\\/S3qL9CFHTmN8Aas1nCr0UkLPLWFulXheuU7URErIJqC1m2Uu\\\\\\\/EHn1pAAF4rmORFCcVgDoSPnKM5nHuUC+lk48niFsu6YPwE9qPVmnLax8m+ZK\\\\\\\/mDKKpKfo3hvKjRrLerULrwObuZ\\\\\\\/EJFCeWIw6c+5jXYVu8PnvPj2l8QWP5lA822Z9BO\\\\\\\/1BEorbUVmJALgHzZaTMyTFtacy550MzBfuUNnCN5JH9RJjZusjkri4yWP9YFVM9JFE9cQMynJUu8Wop+kxZN4ZOrOI2MtO4NzkLWAGrauZklVhttHUjuE8fjbMEaQthVrk4i+RIrXQfX6VHRR5Jde7M2EGV1mA86td1UkBqBWzqF\\\\\\\/z7RlvdTwkLPtcXzK\\\\\\\/hMIsIv5ktROjxNT358ZhUFZibwFbLEHCXPYHb9nogbceLDAU9cwN1L0oryCMFZXWpS4B8NFkjVrq2xVmI0E6W3K2C06qYDL3jbv77GSU\\\\\\\/KAtRu73iBJvRZX6XEYE0pzd+LM8YCtwpaEqyHhdMcfTQZLICkoUaExuxYJMltm25Lw2AqRSLXmX0p+O5ous3UMKEpJyJ3\\\\\\\/EWIt1shGS1C9U+O7ryzmXpvguqe4XuohoOv8VeE7ml2pOHNWvgr9ACAtbfbLCzKUMyFmWlyBhVQmcTaX2L5f1LKtOWR+vmIfzzGlJZbvA4\\\\\\\/wbu1AXS8HD4OnmXyA6eDtYe\\\\\\\/w1l8s42cMJT4PUD4ta9gdln3bOndZBbsjpZ07Fl65H5VrJkEHKXzMLIMrZmsjELQw\\\\\\\/Uo9ERuNrMur9rk\\\\\\\/dfvqAhVdOuysDoUZQ0cib0s7zV\\\\\\\/+yNEQ8SuLAOyWXImuP49txaF02RyxhD4hQ4LdOQ8rxaRyPveqcStxWkuky1NkC8ZhwuguLd4QJivInbMTuuiZ9bN44ABsXok+oOzhq+v\\\\\\\/UJtPfnzmUp1eYax5bGwSjwYojcCCJvIlwyQT5se\\\\\\\/MU96PmAs4pkUu6k6ru\\\\\\\/D6Fm5\\\\\\\/5a7Yj2u\\\\\\\/HnAUN7eJGrhAO0j3ZMB15oq1DOPLlJNo8kf9d93+NjPrGAiaNrrBQvAYxeKRih0vcj7\\\\\\\/nq4KvOkQn8aSIXKbbE5baudLAL0mW9j43hNbjsYMjgBvuqPcO+1zTV1ALoGLRAj+XF\\\\\\\/TY1nfrT1hRAs8UhsQN3\\\\\\\/U0cUAU6F3D6uzGFz+\\\\\\\/e4h90AcNhg\\\\\\\/SeFZSvgtRcb2RtonJAMH\\\\\\\/e5DlCrU\\\\\\\/jqBLdwjlq4lnvsAgyNs7t4JgybVII7ehMsxz980UssuPGpAbwi7GuRPgRk+zoLPbGDlyBWxVr8FIEeGtUQwzWFVhFui6XvnpAtDalAc43MeAeCyZmOPXJK\\\\\\\/gBfIYGLV2+Vdrtixs106fnbEYGnNYkogw\\\\\\\/G6i8wYVvUfAaK9M7VNfnead6KkXU\\\\\\\/CEMk2Lqy2xmxmc2sscgEM9EP3EjHigBW9BOb\\\\\\\/7tSIMKqYGk53CzNM+m\\\\\\\/cx8r3KR9PDUjur3+2l1bhSeHs5laSbxAVIaua2KfpdoD\\\\\\\/hf9eRT5atrAVQYZEbpG9k\\\\\\\/7QxWZb4g146ssL+id5UzvvH975M1cATsmW1mr3WOhp9EsLuUe2iZXOsYhJIDUfwTXYPrt6Vzy7Y62htNikwcNEkHiv+ZCMKzp2ErraqmQ1y0oajTe+Y+LNdZXpjNn90bbVt2jNYTL4KSS\\\\\\\/khyDGbi2Dzikn\\\\\\\/xr0mpsrifIFCjvRPD8zpR+6Ktl1cipSNTRzV\\\\\\\/JzkjHSaonHyRmlz\\\\\\\/q+9+h7kNMrXIqNe1AymNf2ZkVxvBRKXwIzAdlyNtpPPGCdpcGGRdcEPqOM\\\\\\\/W7AHu4B5S+S4V4g2S5Gh+Evz5Pts7Ny+Q7NV3QMHMp6BWxkYRBVj5d1M\\\\\\\/Tlfw\\\\\\\/IhIJG+MEzX+0nJldJVZD8v+qNP33ZF+qZxBtDHz9RzjBq2OJWpqWKfx8ne63t3YzwGPhaFJaVUiPfXeh8Ira7olVKuHpnVlry1CIuy6i2lEwA9w0VQS6HDto+s26cMty\\\\\\\/YDKlk6m8ESvg+skxSsCab26rrZRXQtnrynI19+VB6hzHlejUlYOvj\\\\\\\/aN8A3r03EtekgNewkkSg07S3Tz+m1flyPv7rPMggZ2uFZEc2xyB4hLnz8WMbwbnppKgsfuwpwIip7qZf3SU9M\\\\\\\/xy3RCF9nBNwdTh8hd2KZFuhgXCrq30DN7zK\\\\\\\/rpfIX1FgJp2yb6GJ6V0vpq3UpyhlCjqXH70Vscbd6f+l8L+M6gqaU99WvPZdW\\\\\\\/3FBAxOFRIthWXEwg9qxAFvH3YVCRoFi1Qwto32LJ20AHeg+l\\\\\\\/KewBUYsMu\\\\\\\/U2Z2MBXn2qG3+ttLBfjTqUwl+LXIj75AkCvhtNueCN8SABo\\\\\\\/JtdaGJqNzsauHJ9keRnNWtR3j2GUIdiZpFKdegFJPFp06eYiXqaZmn8LOWezWNuPSs4Y\\\\\\\/TCsE9naKGBLJ\\\\\\\/cCgCZ\\\\\\\/EvcZeVh9hYHsEVHXHV7QunPNo4U6zah2fFc0u5r3ZBvthQrbnRhB5\\\\\\\/DwlZXmR6O8hb\\\\\\\/YDBPrdA8FxmiNt3o8YG32eMW54v9IJdy6LK7nLiErNQA2cLzNc8kqW12OTsbYmr0b4jI\\\\\\\/yildatNR\\\\\\\/i0hbotj4kFrhHdWWsx9L0oaDGyscYtVjG1P5uYW8yMREcXG5Pr+j4XTDkbIk3+WGWAmz1scdW67ijMlcyaVn1FipavPzdH0vXZJYfNCqaVKbf8HxJ1io+fbditID+lz3PITFd1BG8H24E+SBkWpOY3kBZYuCZwvu6eRfWwx7clDNsC3SQkzBZUjH+F3PZ8Y7K\\\\\\\/kG+idBY0qwoVOUJnq3Af6aEsMAiXOfpEoYpDi9zaVbEyOzOd+CmdEmQlDRqYrjsYP\\\\\\\/Yndt73CKt17zqiQn0EU8tjtCywB2lf9LOoWPadC5OoZW60PtVkRXcacvl+kYsSJ93mujqLo5wxVsylzwN3HoOswP6sN2GASKVc9UYn7JR\\\\\\\/xJ6hLcG7b1Ocm7IBtp\\\\\\\/x9dXZKtKSEr3TPM8chJR9BQrcHBdt82sWA+fGYk14F8YmyW2eGTfnEF6CSVmBdJBRzLFEEPN+JGajhMJJDJl2GKpvqUt9K4MC3Eyybgj5IiG6yfDIw5d5pzIcAPcc8d0qJw5BJJSj2bOPV2y\\\\\\\/oAv\\\\\\\/Csxr5jqd89V8FVTHoe1MZ5R1pmKMPTRWq8GbdMZZ3uK3tylRk4pUtLg98Jcg3yFcfA+4HeZAXlDUYbhNZSz+N9pOzoXwbJ33tcutr6XtD\\\\\\\/linClIcOTi0MmAcqDhMVI1aDLd30EeoOVns4JOTyBP3rm3wsS60l84WmZZZmke1auLCqPHVEeEa4IAuFj8cYzEu1odhCwzCVRV1U3dObUMzGwFGepJwbbHXaw+BkPef8hHZ1vzsekrZcKdNfcmNqicRUsQXaxa0cd6gpVz2lDHifEGJ3ijmxNODy8Wo8E3GkpY\\\\\\\/cJMr\\\\\\\/LrxmXMqoriuKH3cvN6BOqkxMojMXgPv3ttnY76RYw5ChIrhjfByBZLGBNvdaj12NpAx2QZizHhcPUPPC8UqpAgLhIsRJv25K2\\\\\\\/0LdEkk8OivsqOiBdPAWuT5RfhfsF41kN\\\\\\\/sWYbZ+vt59FN4WC9vXfVS7JAnPq\\\\\\\/z8+iU6ZQIpl8KQ4pbCjj5xEz8jm30ZepRG9dTIp6EVKQnqp8uy1SMI8KUn41+4VDzyikX5zUbgMXFIHyVss4PPTWkexQhaWOzbrMmbpPHtixC6kcRtsD4o73DF10ldSRZrN5IZ7Pb\\\\\\\/nm5X5aUu4Bsi3J3BDqYVCJVo1dWIQ0a23uR5RLtzxFq7I\\\\\\\/WYqjdTHbAt5b00hnfWDuZT0CGIrVstmrwFX8LQZv9mu5HJD9SHrSPgYURTn9UkmH0zgXQDZzMoFpKcRhL094jiigEVH2BJ7\\\\\\\/TVt7hHYugmvBRramiQAxx\\\\\\\/aXLJBb1ewpdoHs81J84ZlpQoQZxp6kRX2L6zCrGIuwBDmquEL1qewsxhhwCyk5dDRS1SvywDrDzfKyhVV\\\\\\\/j\\\\\\\/06FsmvRyTHGcK5C0tm4RSfC1RYQnwEKNRke+xlBr0PHtv+dtb047VyW5vH63omQxE2F\\\\\\\/CuN6wzFE7rNAFPuH67XM9txqZPFWTE\\\\\\\/Hq1ZggKI+bLELr41NZXWCZXRTh915s67xweW7mYaVAxCM2UYrXduiTSl03ekQ87ciDRSl5Uy7+WMsz7gVKFc3TRQYUYAAwbMO9gnECB+rb27H8hqBS2mm+PrnMkam73FNYsyAXDZgubRHAqLEykkwPjxwQelTlelzKw6E09WehggyN3k3YIifCnGMXaGOzgmWQrKjP1R1nJ1Eh\\\\\\\/CID+n6RAP8yA8tpz71OxM7VBcCIvEm7LHvTkrWkIu7BzjLjbTFgRWFB67+UIGcJtvZjZLOcSRWvhDd6TQjcUZyliK8zPv55a+27uHG93Pz0Iv0+1OT1SVXpZDFv\\\\\\\/l5cMVqfH1ghnKd3N9klDP0By0x5pbpn9CjeavpRutxHT02S46CP4RpGaByRKYXUzBtf7u5slq4PDhovxz6Rw7EMOPP2LifkUOQCJ5AH8KX86LXI3KugXklbA\\\\\\\/jS0mIBvi1VcPvl\\\\\\\/rCzWEB9HpCHa5CwxQPcDL6WyKKH8O9FBzBWK4NmfVpP7e1kwhn6RTpGpn63l85RWGpXVKFBX1p2rP1VG0SYvDfUKQWUspMFrWPCXD\\\\\\\/jxxBG76Z7ITJiIGeBN9noNeWpCjfFVkQhRUrSPc\\\\\\\/HG\\\\\\\/lhS+km5EJ5WlIPCJ8p5Z+N7XJPkyTIuJ\\\\\\\/yLgA5Q6OT+9Qvj0WXQ391aRj5HDBuruTDEB+EmUEc79A9tgvPs+XT1MwZCdU5\\\\\\\/nVZUSSmjdLLUIQmMr+L5\\\\\\\/UeUAix8XT3fBZZWzx3jjMXd082iovRWeWAQLkk0hnQ5+AY2qJDB8mf5VlU+Kf0yi7Ztp5OtcJiPLxx+4LZLXRgtRWM5tSJ+D+V14iPGOKfsHnM\\\\\\\/IrCPw+6JIuaeSzrayc0RbXXAm5dkN7YCwAquNzsTpzskiDH1jfEH19rIpza\\\\\\\/HhL3n1ffYNebvnJWuuOugdqwL510E5LyqNJ2fSbnE8K+JWU\\\\\\\/KfO8DEkrIBm5RzY60Wl8uMeG+KOWeqRKM2K+jpCeFwo5BJetSyUfDUAagxem0xV3iQp9sM\\\\\\\/v3+c3sNpENtgzLawahTF\\\\\\\/ouIIk8XlrP+8tF5yWK3N83MKLdNjsVbOC+96PgT9Y\\\\\\\/\\\\\\\/91VXtZqs5wTMSgh1BNv9R75CTRXBxRl3e0P\\\\\\\/1+tWkP2mEpKPwH50iPIebHXOPcUyrPs\\\\\\\/7RLb9As6dm9+3UJS9ztLvCKHQRkcY8+CDpAeM+IoaK8+YIDJqeK8QjIh4y8dxm8VTgBIZdC8QH76s5ITJbTIAw0x0GV5UAj3c31NZ58456ONLGwzWPTPDAo35Qsbj1JLLpRz41goXwgWOtH52x0\\\\\\\/BbKmaheUDDY8wrPBfO3ISBh4WHoXdWVPbYo2PZdijc6iskndTo8Bxw15e9naiXOxY5FuoY9IxpmPa9mHbmFREqw6wZH69oHkA2H4gmW047AewcHM1jmyl3uEpwGunraAKLHh6ky9ulsK2YZjjWEZnrVyxusx+9zXForPwBBZpehTBm8J\\\\\\\/4FC\\\\\\\/0QxQHqCLz2nwX0EoBoVvxliszSZgaDb5aGeTer62oBU7I\\\\\\\/kIVtfErmrSixJOixjXbqjtnw85IEVj3EkS7G4Y7XfDEGfm2F+JWouLJs5G1CShJb168YvcEJb5XypYnlLgH8HoNJ6keLWwaJ7\\\\\\\/51q1BjNppckLjilSOywsYUBKehq318UdUobJB1IBI4WvdHHGsSZ\\\\\\\/u5DEq3ypUp7Nv1aFEjyacdnXa7F4H\\\\\\\/MBZhsfyIPkW6r5pmJkErohcPFagLDDeJ8VFvLuwTWscwber4TWJkei1e8SFadxJoZRB1psdTsCwk1LJ\\\\\\\/76f88VMUGvHTqx3GE6h78PuKEmMRwsIA4PrZ2qj57bxiC+RQIWooMJs6f9qq6JeFmTYp\\\\\\\/Y\\\\\\\/oFaZCuMfdvsBLbYQcptRu3\\\\\\\/VDQ2oWRzggoVvUSPefLXJxyPSWZ54WehZ0IwIQnj8gRzPGmYUG\\\\\\\/A0oj4wDUPcBc9Y5QYcXRLCxFS2HyOoKJAb\\\\\\\/RDS9b4lYgsysnE5M2F1OhHldLhGx3j9NvDceI9zasWVkV0PDgkoCbo+3kPvA\\\\\\\/j8L3Y1xWiKvmRZ\\\\\\\/F84ujS1O5PnzTHdnbNBdxyMeQ22Ek8Qm40q2kOFLzkhyLCnoBh7E722UJ322ffKKrgv4s8EHvK2d0DoWU0sN8mcag4NJ7ruiYIoMmzrNR9jihlqQxRFx7JkRE0vI1OtwUaQHFwaJQevMFh7Fze17ShWxREnXl9OTAyjpvCQUiuUkbCYSwv8PZ2vIEUQmTSQcMCcqNfu5wISPB2SM7iC5fDeQZdBNINAMmM2t2EmlevQL9Nh1yveWwCMtxuKuT8JC8crg2u41dt98zFQZEPF1ELD5yaeXeLolVYXCxg3XZ29KWFBnrlEi7mtCr9Ad5wgpseC3d3oJs37pXf2\\\\\\\/qtJzahwtPGeht0NTONdsvL1tH9CWYJC9e2HEAjdiumvSt9u939clwZOfZkojeKTQFCwEYP5J0cRQi4pJt3o81DyYNhNW0pHq2vW9pbsc1Wc9wBLmXbUtOQvNIqb+MRN71JryJ3IVm0\\\\\\\/5LARwqXiAFdSmPOvDsqND8PZeZ44\\\\\\\/35XzbcPJrPPY7yhXwTXMU\\\\\\\/MhrS88Ir4cSg00uCuQIrNrILMizMhmFdVXY5giCcuOiLl0qoOKSbe+\\\\\\\/CRaCYsrLDNWDCKTLWgriXaCNZU+EtZjkCzdVZF7Zy4rwuZvSfAbc90FXUE5HMQPUoHEKRuSGTaHLf5P0qJbQmKitQac2u5jFTqoxw0j0Nv8YQZ\\\\\\\/onpge1fQKUu8qPkk5fjBJ3V1npPz+G38WzR0jYI5JJAg5d+NAhoXWX54vHXL94csFgQgGiFOxZLb9bG9+TfDuTl7wBjxIyZsbTN7n4J0h+bpdECvbhy4J87K8m2al2+K1XKTfoDtki7B\\\\\\\/hx+st6rtVE9gi8fjTQxK8cqTUz96l\\\\\\\/ltH3xPAAeNKAf6cGzIcXkpP52E3NeBQ8HgcOcCZbpqA18qt94LVLqBjNRu+iDB1oeyoZsQYNsJF85moYmN+mADQxCvTbnbD\\\\\\\/nX2nTvSQn490w74PyRbtXxZkesF4r6QHeVGy926bEz7ZQfNgGbziJSvWH5HKaGKib24Qam\\\\\\\/6yIqUqSbQCGbK9l8MYo88Hb2yQQLMNOR7YWIdxFtUaMr\\\\\\\/1utxM5RFkalgIDU3lv2AbamaFP5VUSK9P0Zf+4T6QHopTAs88u6Gffpv7U5PeXpKgduO+CQVm6ZEOVLL\\\\\\\/02VEwRFJeyMCvvN14Y3hHv7eFtq3Sn84Q7eeuVpMnT22iZBXo2wxwh6hzjz2fw9eUof5dcUyI5kDbv01sA8nnc84ShuYaxqEmWy8+Waq1lyO4Eg8LLtqn4QJe7jUQhf1OHgRARnDKkA48YlNUPjlR9AXwyAarv0l76IXLkW0CAENQnTwCAb7i21c1+jewrLQdoRyuziauO51oCWDWxQsoKsvtHl\\\\\\\/3a6nt\\\\\\\/UIk5mfzfOpn1kP0HDq7Wr4aWn+flaJ57oPRfY3\\\\\\\/gexpmfZBNq05ahJzLKUQ9MOjdwSyFbqxYpSh6fmTgNu0c2odWN3XntdTdcSsPf9pIBnE3rZzUu5MKagoNDIqI17DHZBdQSVJW449QVGhC1iSGpzjcwzXjCAK\\\\\\\/u8R1dtz1VVQfCs2Lbd2HSl1knwS1DFL3gLXIkh3ZlraVTNuL8HLlj2lniLvhbg\\\\\\\/39ilCGkMqSvxtRpxTFZGISk3YD0fHMGGG69O3fsIX3AlI2e7wkDTeFBLD64tQnB5BIg4SEsdudBxlYdEh2pX+mYYUodfBGEk\\\\\\\/5ieIs8j9LG8ucwJbjICEcuW0uFaBVmE9aL2\\\\\\\/22Z0ppfLuoTOSUvI3UgJQHDg1YNcoOin1Mwlg57FQxH87WCCVWVHE5NsYFXpL5SlSLIRY2hE6iSRuAatrHvFWdNYAGJ2Cv+U7\\\\\\\/RfTXv2zOrfhVCryoGjX6N6W81+fFf2td+81s3fY8P0fpfvqiVR\\\\\\\/dpgJkB7\\\\\\\/HrAQxvfVFZdtooggh\\\\\\\/cUggO5QDk3D6C+d3dXbNHpftwhTFspI0xJAm7xBCmYcrto7rHDv59+b\\\\\\\/VgxvYhjhQS2iSm9KOnUIS4k9bYaGJdRgZeJLDGWYXgewR+LjIDl1wp9Q4T2hYzZ3pXvMm\\\\\\\/erLWuJYnJkO3PbgyzOYXPRT1gjtMNaiwbFH8coWweHsAWlZ0H5tKNUgarTXr7ZkFjJ+blUdGZFZ6\\\\\\\/zL\\\\\\\/\\\\\\\/azbkabfI9GVyw9Fz+Py9H6U66UrDy4q\\\\\\\/y+VLR8bXYzB874teJyjqSR+H75Wcvh0BRip+y4\\\\\\\/hnFBDL6AxPLY3bM5tji+QkrRI4IR9aifbIQdPWcmDkxVSuw3kGfe7tvTny7ThZg1eoePh8QYLWnkY48BfIeqcxldOgTgKsVNmT7GoxTa8\\\\\\\/ZFmUkkiop4jEJqlFOVpuDjb7Pbs3DAs1a0V898jQBV4dw8zX3BI4zpq\\\\\\\/MwXZV9Ta0TR1w9RfIsR2nuVpjhonfrPtF2g3COW\\\\\\\/UGasgeCRoY8F1guwSAx5gMFf47rzAAPNOkc6TPSTU3luKKVMapgB0DJb6qncUhWy7JxQ12K8bOCVRc1XaWKC6zn0nbr9TgTenZ7BYDIQNwBN9kQ2LrJEPgqe\\\\\\\/zovOz8FllPox0sFGq+7Wz5GQcPgMglrwIscXwzl2rDDa1u0T7k1e92i02+Q47pfHUEE6j2dX7+kJ92ZKE+5XhSVUZMHTRfMafYMiW8r\\\\\\\/zwjGkewj0uXL+gs7Ty9RtLgO7fvwjztYVMHSUzrIFonPIxaGYpOjkDXVHBK5H3O6lummFm+XJfvrcAUT6X8VlELNhLhHQyWq8zpa99YlA\\\\\\\/pfvQJEIR+TPDTARrr9wrKYqEcOAceYDUHIOcZVuY33DyNEwvYpxgwxcdpMVkwOKivsyF6zvov+8RwYv7y0ieWCoa\\\\\\\/XBO+RwDh0vqJvrUprIq\\\\\\\/WJdqkdFd2b85O7qSwLFUp98Am4yZGKF8\\\\\\\/OhAlAz0dYl5JvigIyCFXNqzewxY2II+\\\\\\\/r1vuIhvgvLOkjbrj5ErT\\\\\\\/J5hedZojwN7VcMrzY2RryD8BWT6LXB9bh\\\\\\\/V6WGGRXi0xuep+tJg2KqV3Q3NYxq8ZPVF0TdLEhmUVtTYzpPM8+Y\\\\\\\/pRAn0yxTCw44L6xDC3xGbmzmz3acYTQsp+KZMBOabQChU8HdfBDl+LXy7fHVJAi9sGh8tSRtCEcbU2VuIiydNax5MRetNlLPcCxdMzO0aatJa1\\\\\\\/qaG06xbessfjFG1qNzHWlH5nhY5lBfzhhiwf7ZcgEavpVfY+EO8BXQWtf9TQHB41SFPoGyS9l2MBQaZsiTH47r7fkrDxGuRNshHNUp8mvN6jg2dX0pZndPjPiTISw5IHcLFfBVUXC\\\\\\\/Gwit2AiW9LHcFHeOJJOoWIAg4kJWvrqBtGZ\\\\\\\/PGhI0R3hMkv\\\\\\\/IQBl2kagrN9PjP+uPJDYF5JSdeyhZw74D8tZ2NZCnQIEeh8NPR8pdHxp9k6QQJFCdUzL6XZYTxa59ocTILES7v7i4ptpMuQvWhEYFLNWETyZwk4G9Y93+Q7jKhBugkbD83paUUugkWrZa+yWZMczsBK\\\\\\\/A+R2unmSKwCRWdjQTWYL7yOaC8OV3VUB12RHXxHzbQeJuConWKqywPkJJcbvD5vefF++TIZIhDxfzAsXV72fGWmfGvSf7r0e89MD4p\\\\\\\/kGJiZq+tSQEe53k1CHbqF0wf+e9GjW6Mb+M8quKuIy0PV62UpUj2cB0ZdJ9Pn5oXYM0pXNCjfqWiy9VV33ehurI87YQeGTKFFLq6Kzlu22AcC02KXdjD+24XHaXr2NvE4JSzVwxwhOlgpUaDlDkodfF8pKWVdOvMYw8QW\\\\\\\/4S05IFPm5dkR7IZ7DmeVQSo6oNkfRvvAW8KAdgH4sKE13aCwuplFihquzOSoq914hghxMO3rhqAQyhnFEo4WzvOmQkS5OVnkbTla\\\\\\\/U4fXjboRoM3L5ZCZ2YRLjBosX1zEJX7L81viQe3fDXpkuH\\\\\\\/G87Rg1qjNlmZHonsT8Jm3MUY3nXgRSKiP5xIts67Zz1+i3yI0TArEulQGbBy5AGnr5qJpgnwFpq\\\\\\\/aO8o5rYDX5v\\\\\\\/b9zRGPd6jhZvgZN7eRz4YUr1BC6V4NTg0AS\\\\\\\/6k\\\\\\\/VZMqAj8k+jye4vPWkZvj3lXhk0BX53OP1xd9T\\\\\\\/1trCoLV8Y8FpiegmGUlnZOUuU4olMujDrRnRfnFlYyXaHfCcaoGT0F9+HtynsRExP+xf94H0rJbokk7+zO+37oaFYW5zWCs5MJNwGBhBdY+jYJTUaYqhPME9nONkDK61\\\\\\\/V7qdGH9tguVCqGLHxfyNee71GxXloMExwbK7fFq9SN5UqWcVf06UBpCGmhU09MB8eunHV6yKLYPsW8JeBwXzGCi3r1V5eEUKECVK3ZiUASedPfW5K7QcFDXzfMZlb+wwqKbp8Mt++nwggeaMu\\\\\\\/XXhTzBebUZdcoGfELV3NpgYtgYvb1Xd1rg7DMcw2t\\\\\\\/A7nQdu2TZK0yDR3BJH8eCCICWeO1415e2h0rc\\\\\\\/fpGXuQw0bKdwUx+cVWS0Vv1QtrFnbkMVrJZ0q9Vg1qlFQhL8V\\\\\\\/3l7hoFY4i3TS\\\\\\\/BSisxrcD6dKmjSxEOaFMlB\\\\\\\/L8C2DwyqHaYARqUxFu5K9LV+k5zVFgyZM+OfSbfLybxjFlSJGuIWeG\\\\\\\/BeGqgryo3IOVHA19hGWTPY51+YZsMNg16vlJIXOax9qcja3wJvqwC+moVNcyHqAP4OWB7iwewP2Rd07G2cUut7iH5oqiLHOXTZBiYdWCfN3uY\\\\\\\/c8sQXdK9WeY9GWvIvVLghvQmuC15T22n88azZbbrwRTMvpnYqEoScPl6Q\\\\\\\/j2\\\\\\\/5hO8EUXgjJPlVCDm6x92E2\\\\\\\/94KBVWMXEgSP88O9odhAnf\\\\\\\/E3D3R0j\\\\\\\/mZVJWxe5Y082mZt2SOGirL2On0uRUCc6F+5cjutFn+Gmpf4hncGWnk9qiEBbReWf1o+vbeEIwMRSaTPBv4l+BRjRDlp1KtGbJA6TiTMeMp99ydZA\\\\\\\/YnQDcMBpsVv+tZ4lIKuYM0jXvql1VjTi\\\\\\\/C4Z00tLCZ248E8tUpoxtHl50Q7h1ZgKOotbbcZr+KHzLJsfWqDf4RLnAAIJzMNTVXZsOTHuEN\\\\\\\/Q9zH6LKKeRdkFY8xVpmE557ZHt+xnNu+hrio\\\\\\\/qOJxURdFN9lFfQ5sN5iUCpauuU1vItCGtkehCZogcpTmYNnPiXw9rtCxYixpfuP1cV9iyGZqabxTeah0RKmMnLu8N2WH+Mx2zxGC9O7veQBfH8s4TeALLZGfSxAi2enjRdeTn2bAemmBGPJnM\\\\\\\/yBoYlBNmqqqrvEftGoBUlbr9ff0NoUh0PHqyvhIxxA5djuxvKJCjwBGzn6DWTMqhEYr+mIJwaG81HpfSYnxuNYQLsDhaU4viXhiEqqomfcx0c4KildqeaDsJbrSySZVoqITaWcrNdGjpKcCJ41HtMCHgfL5oFSo9\\\\\\\/RrJTLE2nP64T74JiWdjzPP5HQQeXhLccPhCMQNW3sF44wZBLeASjpnqUFgqz9aQMe9jV6ogg7aYI8jIhcfhOLWejLpaekFM4W8qPcufuNarDEKFcTUTBJoi\\\\\\\/33J71f0GOgF8GB5WKlyCxKQ3he4ty2MhwdGpiGOLGgZ0g9RTdaBRFFtNE2jy0\\\\\\\/zeLyMCqTALPHvolUg0dGhflAQaZbThGXzNNge5jSEEy\\\\\\\/qMMot0uVN\\\\\\\/5wdKxTBmKKRaCB6VYf6dF9G3oW8RXGFEXRABfpNi3rEZ2tyPJQravMdvMggq+3SgTQ5cg7l6mUfuYJptNW\\\\\\\/+uK+p5yyHb+dDBpv1rf7W3mchCCaMZcE5TTOZCLvZPrrkfotrT7hcEUG3KQx\\\\\\\/srFVbVzONwIdga+eCMaF7NV48slaXZxfm5x+gWckpUT7eOYbTeuN2pqBXfKNlR3Q522VgE+iSwKqyBra5gLQkltJMymB7gJt4nU5UnRjrjlHmxEQbrsVwjEcZDkXcu7J8GSMYj0bzB69BZrn5aUs1MNeQqYGaonn74AXxAlMB1wYhMH5wWUc+8nVnm3wUja5p5Iows91BQ\\\\\\\/55+YuRXSU\\\\\\\/SI2Ozurbu9Me8fHszSFoAMImcT3CSGANu4f8BrX4W+EJqFS6lJk7PDWc3i9tCiB4tuTnIQSo+ock9iRxRJZUWeZx4dqsQ5mwMMdrby+0A1gHVhh6Y4U5sRYcOGSJ\\\\\\\/+RMzMHUXXrX6\\\\\\\/HnOkuQGYuVspKW4SY+WCndncE+ZbtNI+Jl60jqy8W4rAL3ekB\\\\\\\/pbjJIE3tSEXK\\\\\\\/iQFnHxbtvnLIlxI43oUd\\\\\\\/10bDKT6JA6VOGNHp+JEnu7ggXfm8wIAvkn40Kf1frsvZvRxM2Q5Y0R9cr4WRLwiBzZPAldAEDuJtlXER88WgH1TC2YMx6HXnpIXW0mh3X9gaCkm2X0Eehgap1vNQEHT1u17IKXGCnYocca1rhTwKMfw8THKWOX0Vsy\\\\\\\/QKFr9XrpsmlE1e8+liXMR1eUZ3zBqlHpLNOBLxs2vLZvPeGTFGev6m80S7ZU3C9C+nHN8UPPDWlkjuVn5sFMSmH+yc4iXeq+tNMHOdPzEF6jXVyJOZzMUZUJzzBHsz9nBoniHnUxn1CTGnbNANnHBLCWEIoEMIpDYZPXO2tPVRU7rCrbKBs1qM3vc88eRhbpNKjaEEOieepwiYdo4w6Ll\\\\\\\/CmbAg7l7NUJM2ExkYlQdWu+3oaZfXsxDxxH84b9sFmYH8ay7oXQku9DCaaZByhTBJ31q6BhtmjwiEqh3Hrk2fjfbfxHKwR2GpYMBjQoZwlYe3KgydckK4G4rP7VSwrY6p33mxQqfFVCcyinKUeKQ6cB4\\\\\\\/TijORfPY5cql3JiI6zAy0PLtO20EUVp+P\\\\\\\/07c6Bth5IwcmvTQm1v0YO1nx9WrNQqVxca4R4ln2RUa13TmwHWVuEl0UAoQldlzc2OFbID9\\\\\\\/RA5vPrtxsuxByctvkDAQ+B6BP+U9y9RL4kBi1x\\\\\\\/mM53c2ti7KwCyFm9BuJCLZtGsTD5bXZXssqpWlQfwGH9lNr8KG4uKaLJm77YZW5ACCaTEYj76Swf3E+fpWRum\\\\\\\/onGoxykDjIfdmw9KdFaexioQJWenSSkijEic+aVMxlz0XXCNUr2DBaytLBThpKuOuSPjZTMatKhBkrr2RB82o7kU5OIxbNQZUwLrYTYqWLFa3D1mHlo\\\\\\\/FbRDXrI9xSNO2JQAZ4xhDAAsoWGAGiAbS25QIbnejaDe8H\\\\\\\/dnn4s512tyVGwGmb8wlTLPHLf1k\\\\\\\/MRV5WgH5LQ3KtIiHxn5Bzx26+VKmLcwsg\\\\\\\/X+AC75hfY6f\\\\\\\/PebImstrhmUQQG8CJjnaEDg7kkArqUXTmmBQDevV4L8Ur07NsKjNV8g4Zdx\\\\\\\/A17Hkdo5YukFXUsPf2Dls9owNtWmccQ5ZU4BEOiWftZoLacaxlkTQDY5z5cWZeb3nR+NlqRcTdpMAsXjdHPq1GNPnJwjrzvB7Xsua6CJOGaSF8eVqcjclB0Jo3W0586EucMOJTzSO27hrf80ANCdF60vyKdONysvt63cXgGKdlOUv1yLbRjK8K0zwBm1xx3cg0+M7CrzBFQO0lssgGTvFFyOKmrIJLaDQiQxMAJ898K+4EOTXdBEy+BZJ\\\\\\\/a0VHivdTR4sgzO5QRXKGjz5IjJ+n5ui5U82iYzfanHDKejJa+6non\\\\\\\/0PZvRXxCWSe6SILjsCLQli6wJTBpJUqXMJ4OncmQwFfn2cZtT6GnLJDo3mPW4zgOPZzboHS7I3SvXtzeJ2mwc4gE00eyQqDKhGxDdSnoYhF\\\\\\\/b9PdKujbcHt2ipeBkaxd83tHcYjbUCMHet7lWY9DPTXg6cz21egnSNTmMNl2Bh2XrgYp1kcRKta2T88FF0\\\\\\\/QMLnikSsCffhUi2XoQ\\\\\\\/\\\\\\\/Agnjm0IgvYcKoiBJI7wUG+k\\\\\\\/eLJG1Md3yZWezx4KMkQPAZfiX8UefqYK35ODviJFxyQX4TNh9VbbcSne+59O7yp4EbES6uAtQow+5WR6XkcxiEF9MdX9DMxFXI7FHrT43Y\\\\\\\/oHlmxqXw0HMgxpASRLsEWstm3OVZW83F0UqyyxtexKd7mg9kFCRnXqz9QwYfjiHbZI7RvHmkvEHlsYXzA\\\\\\\/T6T9z3kgAXpy9cTrJX24EAXpji0EXOiavMt5He8b5WRQnsEhjIukdo8cxe8uWL4Z9RTqOIpeMGlfDs+X1\\\\\\\/JDSPo4UVq\\\\\\\/LYif5f2BtgSCnuX\\\\\\\/pE2653iHUJ9fnt2Bf3YV0VpLpXV41aF+NTtfbploix9UY\\\\\\\/mBq8FE5qfiU\\\\\\\/BO1AiV27aTipeHN6NrN7cIwgFZCmi+SeUMVg0oRVpMz1Yw5SozPEXicJmvK7q4n4fhbFQIdmrtOf4hh2nHx2e9rs\\\\\\\/gVopasz9PjoyMmIgb6HfelK7D9KdjtFWvsRecMynPJh2B92d\\\\\\\/1IgSTd30uLuddjIsh19R2E9VOQuSLDIVAL\\\\\\\/FW\\\\\\\/R8lItcDg6hg2P6vPPBu+dYx4klQwh16Q4aTHArF2U6vTaLMeCMhdbQNX6IwEeQBqg1\\\\\\\/nO5aKXY5Q+CWyhYZPuDMQbiDchuJg0gdzEfyjDSHG\\\\\\\/NO9hEssP0UMLVOcG5uUw6UxjO8bIcb767EooMNMyUX0ZvA+62URzRQtdplkIqrWhvc2xuiSpF0c7KllQde84DrgXqMBmjrXDBXrlrDICm2eHnh1ohWv49msqwiwcrosHwp50ajLdxpQh3lRqhie\\\\\\\/9ZM5UzvTPxTQ\\\\\\\/oUUfiLvdufNHtjj33Bo3QdamKZExMn9yEgpAwR183SrgLOCEObFZB\\\\\\\/Y\\\\\\\/eECuxNFstCe8QpXuthPxTXtbNYrETENhTlmn0kIrE3NphuPMRTio0IUKSDaWqcpN72jMqOid2anJu7J3FGbCzASIsEkmlEQAj+Ra1dBXI6SAq9HLPZ3CJ+7My3rWH7uelKP+HsMpd29dxD6CwKWZGbNHjo1RM27sBgAWYOnYSciTcjaK6Rh3J4AlmEJqnBa3hFOF14S3WdnShN6jG0NvRsOPMJNdxLol\\\\\\\/IiICGr7shWAQtVzzoNclyvNQOyRX3ap7a82p7OfAnYp8bJUkPkitwmg0aK5DmI0BVoV4hv05QYPJmwiNj\\\\\\\/7kdqAANUOk4qetdw\\\\\\\/Sil4IFf4hzLPL+hOW88\\\\\\\/bmtiLcbETTbdarWLlvQiuqjvMjXFI8s3Wl8OmyWxGWpS+Vz5MBgJ7gLCyC9zg+6oh4VSMcuzKd3latJtq1fr\\\\\\\/Glp3qkIFd+7DxTwQz\\\\\\\/bwBtMY05UtdSOpfLKsA85KPLkXiM4Io9G5OIkf9ZPOjpPoVAFBRGSQOsXzZX4lQcnAh\\\\\\\/lW\\\\\\\/MwRXV3691+c1a\\\\\\\/PhF1LxjU4Xu0yQyaJ3CrkDxK2d4Y+AyMF2yMATGpcHnlZKpPzzdT4IB+2NJOr0l9qJGsKIrh0bvOZjw4nlFnGj9MKsJS+wJkyPhqhm2J7QTRWMD+ccixp8e5LhMDCsoQOs1UpCVgabP6IZdHuwCtgiVRzN5tzPiZIhrj11hwzj+\\\\\\\/Y6sSQy6YJrn+EB7gM9QbMc\\\\\\\/ny1SC5pCwblwSy6j0A57YLdtG45784p6Zi1sL8HqPqlsAiAfgnbTSse3mJj3P1+NJ7Pv6XATYLhdQUVrewFlu\\\\\\\/qxosY5WpPx9CEEZU4hT4XVxXaRf88FV5Bs2uFGTE7OFGfALWNPvyWdX\\\\\\\/yiMolmMeFG\\\\\\\/H1Yd3cK1ZOo1eqXA1kfvmaRuiR4CT8v48A4JbpuoTZn+p4osn14BdYMn2YH\\\\\\\/LFPPRpCT8FQHJqkRudfNNIxrWyZ6wqSNJmGMm3+cwSsRl5RA\\\\\\\/0l2Fj3I+j0EG2Y4A5WempgSpa78ol4+tDoXYYgAL1D6v1+ME5P8GKawAF+m5z358fLdJ6ne0jzKGbWiGdRNBt8c8zlqztQca3p5jt8\\\\\\\/6LLu6Iau\\\\\\\/UO+M1bTcOh0a6QLkuRorB1xrcJcp7QkhpJVddRBBVXygJrScIFg33\\\\\\\/ue8542bSqS0RG9zqJlzSmCm0Y9qWnX9PQSCdZwYN5Ua\\\\\\\/+nXpnHGpJoyKSFMlH1lOc+yM+IjA8S\\\\\\\/vjvKe+zBRQVoIldBNrBriASwrZa8u5\\\\\\\/RUf7b9lIkAFH3\\\\\\\/uGdY6KhUH81t0fIcx7jNxrW0PAaQeICt783JAvTqwMjiZeMruFYVVaRaMAw0XL5geD5LiSlclpvItumNGL3FcBHi8Ug2RiJKC4FzBYxnwpbAJVr8YaVj7tDxPmiQGo30p8\\\\\\\/QrZEAQNy7zRIgZYOjXeehyENAw4D07eMIAyzfBZF0nVbyICD7Qe3RrJ1WzLRsvrPMksK5lY\\\\\\\/Ur9BvPymVknWPW7s4p9vue\\\\\\\/Yhcsyr65qYy0Iqi6xc54ZiEQuHQhyZzPMTFQbQKL8Bh1UFdzxY3cFoHlxfiYLvsyBmRvmktlih1xLMYT3kTMb1I6eOAnRcb4q315Zcb4K\\\\\\\/Uy8y81Kr2C8KXfcgJwctpvbnXRGzSDYtUjck0xcntYIgT8rh6Sca8ZLJSeI1+RVO3nmnIFTZRvS8ZEXj3ZzWAwt8Klwt0naHlZfjMqh5EimcZN9xJvXUqta3zb4h2KiNByOBOgk39smz2TTYVqyv3rzRo1IEo0VRL0NfGB\\\\\\\/FZ\\\\\\\/rjursOq32el6xfjToe5ZvBm7+82Q\\\\\\\/eoPQyOVVH9MTw+IXKw3Tz\\\\\\\/\\\\\\\/2fxNMGVFfYI6DFmFj1D1fKu3nBJs1VBqa5r7RHAgiqFS+x54DeNtX9YpsHBO+AylYCEZFNtFojSuYLHWWhKWj6G\\\\\\\/jILOBscWbG4FJNzSYY3wVCEBWl\\\\\\\/EySrAS\\\\\\\/adXz4hsMMUH2imnfctl6KFEgI09BuQkCIqd0F10uMUQmM3cZP9q9BMWJ7jg+jwt1OZvtVccfh1sIygdQQVZ4hBtREuqERSDTfYX6CKnoQ4eBIG+YbPX2Eezj4KQyWp9UXb3RFI1LbI\\\\\\\/soVNfZccoTZ9+V09\\\\\\\/xcCcJSrBC2hw+6WKHJDCz64\\\\\\\/SRVQXl9ba83YQA2ficl7ki9QQICEtZxhH7EaNu3WnrzgHQd8kmgJi26eNmnaReDYJ6FzYyhHkoUIBj7MJSeMNXccCP\\\\\\\/FL\\\\\\\/e55n2OZUheci1hoW6eb+9KPn3Gd3wdRvAn0oVUilzvZffR6VQvmG3fIEF7+4ILckXv\\\\\\\/6T9ALFozMU7XlxwPIFmf+VElnP7c7P8q3LqLh5oXmbGFTkTSwiR\\\\\\\/QYK0BLaJ+l3awfT0rBsUOi0mi86yko784IkWo+yTN2JFmOvs5do305q1CA+9bv+56oqHCRPZvLxXQPOMusBz4sBHWevtd+\\\\\\\/7VkyEgpwuy5STD7Nujs73mWGqgQ\\\\\\\/A+f9UEgy1pQRjTen\\\\\\\/k2O\\\\\\\/9MwrJyiT7L\\\\\\\/+zYkI9y9S6zXrmbS0SNU2UpPQ55GWpcsDa72+tb5VSa+Ln32GxGQzpWNpWdWTwryi8GtZTvB+Ugg8ZCEat66ZI69vMeXRbwXYmRUAkzxYMBPkE3s1yNydALcGn9wWJT3hFrtULh7vzatselDDsRLRBwNAj\\\\\\\/TNSdYhcjUqyAYKwUJ2FvO2v7gkUufv4v\\\\\\\/m\\\\\\\/BKuIqeQxgUsawpEqhyFU+5zS3SI62OcRGE+YArM7AV+pf8ErkOE7Uqe93hkes4ud91f38zWOm\\\\\\\/cSZZayCK0dFMQ4OvgOLsSBwTfW1y9f8ja8xMIfFY7ZC95gCuPiA5tKFu3m32pot9Vt9nQ+kUHeq5fl8cm\\\\\\\/FPydYrVfBk\\\\\\\/31akVCa1\\\\\\\/5JAPDenzi2OF8aRU7ZIoUGWb+iBrWDB2gfpFTYHnnOEoDZdf1DX\\\\\\\/mJwfgKm+ew6lliL8mOi6bw7asWZmJWM4ezYGxRDWF09FeCLXIh4j55u6cy7HNeD6dRu+xMprW7iVERIwXRQSmzNCgoi9IXVsxKIA6FI4ZVqs1gEg\\\\\\\/lidgyrCD0YBmuiSQfSJ\\\\\\\/tD5Ln1up8cYuo7IULFUAuCT4CUIVbiNoQu5ovfaRGwYWQKfwUdxknlW0oL7PdRnxYa7Fwiq2c4azsb+8ockyWvq13J0sFULvNsL5Z1TRtwn9Ws6pWXA+RMuk15IM2pRNQsPD50ZC71BEEJIcFRjsTnqOHngIgg4QUGL8EQkC04SapVCYaM4UKckT6IgbhEpERwoguwl94mo2B\\\\\\\/pQPrx0SNaBxkjPVSScMJ642trSTwh0eC3YpE8uT18\\\\\\\/7LpEWfK+nYt\\\\\\\/1o5yLmQMhVzCR\\\\\\\/rtbdQkZ30pd6clHgkrmgAzJkfG9WcIpEJOVg\\\\\\\/JGNFa0QQbDKRI1+0njg2Xmlc0eVbsZCFZo\\\\\\\/p4s36De02SJo0eMYUJFeceAtbrktwjRfuNeG2kyEvJaJ1DzYPI4MyMmj1ZBFW6yGb8Jo+JPX4UYnpa+wDrXcBroyS2kspRSynZ+0hr7pgvcIi38ZGwSVZot6ZNMV0zNe1Ou+k8iL00Jf8ufvuCyub+90dS11Z04GBMgweOS8\\\\\\\/egKUIX0NLWT8VccoLtNeRRriXAxgjdoYw+ll+IF0IT7Oij4tkiOtQicBAIerbg+Nr18v7xiI2YlY\\\\\\\/3emn\\\\\\\/qn\\\\\\\/LNMwrNmTWbVANa8iZ5Nf1Yf4vHJQ7Gf1BsATLdRhatzLSbdhkWz1SBgl5FqPlMWHQHIEhsL85bFJP90q9\\\\\\\/nfQ8DMcFhlqj0IA8taj\\\\\\\/jIAnKiOqON3zMDKIWrb6l8C7w92J7PuO505+yM\\\\\\\/LmgnSp7KGiAICYhQXnidRgWFmYHrleIUja3n7f6kOCqW4rsUNHXgwCoNmf1RgZGYdhB6ktGpV6ISHOPUdPtJDDqtc+p00J1WtTPL98zVBcX6aRS5gNd1FH3Bp\\\\\\\/sFzD+7di05u\\\\\\\/beo9YajblJY84YR7bRFJmPn4SuSQdA0rr64pXbelkBEOjOwl02vw\\\\\\\/wM4cm1XxTz0z5qO+GjJ5K6ANnCn4LfmAkxqolUglLl2GYKQVaDG6Bc3gO0wwlu1muUnmGQ2r78Laqy0QGgCRZm106fqRoromOpjlpwORvvOCa6oF+OVdXdibHn17lJZSL95gReu1zbIL4WTjjy2yfO76S2p7z6ZFdjN0Ip0bEjgW5owOR0CGLkMuLY2KogVm1LL0SH0Zp8uz8wlSa890XNDWQYHWng0Zq7oMtU+qwgLExZsyCG1x7FQS7gBifWFMCVEbVJTud2Di0gz3ZKYM0qWmWiwWDnXnZd2misUH0F8TYn1SXK\\\\\\\/X3j3yo6cq87J+v8fZvTIrc\\\\\\\/ONYqOUAl0GcCiPJEFAhhXgUwaYCeOG50eeg91GqikxKvlnhiIFmmuSARkcijnQ9CGDRy1Y6tGog7L366dNnarcOJKqHkQk6EuU8hkUKTd9W\\\\\\\/kw6J0V35A9WEE4GKEbXJ\\\\\\\/l82l+ycqo5WEMmniHkoOjonJeTCfMVZy7QE8lyCwMVnvzzQQqIumytFYf3qwupD5iRYnKZ8e5Ju3Km8U3mChsFHk7hXS\\\\\\\/NruMOjD0sdOgqUI5XsQ+d2S76IdXOQxVwm14Jv9OOi1+ooTYgUQahGwnOiCiUGFhLndroN5oXhmIlEL9RulZndHa+iX9eirXIfNYGnGWpWj39cGPJQ1O4\\\\\\\/MhdhAFgzztMEDUe8fnto2iFsoIm0DCTmPXBWTslCamqu4BCr90oMH0VX4gfT2u2rxRgRi2+AiFkH53U1SQyOy2kbzmDMZyU9wmOI\\\\\\\/nA6Rr+n89ygJOKUyR1EVLSppWyG0HrdGn1Hgxl555UXj3GMvlU5NDM6TizjodnQUq4o\\\\\\\/I93hiOsqmviVnN2Fcl6rp9VMLtivU\\\\\\\/CLGAGsYIyssv2\\\\\\\/jCx5TgbIjWkGTkbnpXigN58Dzm07pqbV3IULQi6v\\\\\\\/TOnIdKoX1WSnfkQwkh8BRiZgwXQmsvLfoza5TdNYDSxg7FGUBd8CExNHsgr9WXy4vIu0Gp9GpieaDWuffR\\\\\\\/IBPiDzEufNpw9Vq4MFpBafxfqq5bMSEjqiVLEQo4kCJnHt74gHo2ksKcV95v4QHVHCYtw0WhuO9EPYRajrwm4HghQSrjkL2Ci\\\\\\\/+Vo5BRLtYFLbdZjvpwUObllM0+1qs1rv4maGqLC7\\\\\\\/+G+eU6mYY9vr33+poi9QGZ2JieBymeEUvz9duQa71AbVOi3GXH94VADrvlmGdS6VknY+op+K833UaWCnkfIPTBMZ22f4KA+1e35iauhfclXEbzcFtuNSxSZEU9aJ7P48WHf3SFNKtX2qvMFieGFWqHcnOv517sWXo2tzVkWjonnJFv6gQaZl2FWDCfWEj0mPiA\\\\\\\/xVeK7LZBorue60rCn23AG1GPGh5Q\\\\\\\/EfUNaupNzBnHic9qpSSQ2m7qwBTawbbpb1nJJuBDBBBIjvDy+QS43rjKDDxel6bbSM1eoUdqGOLvwNF7kwAM2R+Cu5ZyFgvepghamxcLzRSD18QakhZrELdMSg\\\\\\\/p7SRmq\\\\\\\/gfc2xNNGrTA0xjYoWXvm+FpgTmKghEUhc\\\\\\\/kKVo2QNnl8LBid9nMIEpCPnwj4XeDV3YASDtiUPAtt1+rJHanbfs5KPgQqgxJptGsxFFzTFNDfGSuJ+ga+YSbjy4ltjzoVwgV4tDXqfMqylCagGD3SNUQ6CXZDXPEeYg1eHP+dTec9WceMp9ta2IV\\\\\\\/gNw6FrARsGwSyauzT1\\\\\\\/l3aoMhtJBavU83wnAFe2y+lwywErbNjO2opvEUW+uu\\\\\\\/iiK56YffPlVyMXzomK2PX+TiANhQq1n0qyHJiZSA0W8qrTrZ9SMDs6fsPu3+OMdUERzS0YrDNVUHxYb2SIdeC6KpdADpHHVFrZgFQXTAfnCQeUAQinLmkq4ilHIJ4JgiNHt6GWVFML7fwwu0YOMKpL7gP\\\\\\\/+S+NkJ8CSBxGxTm\\\\\\\/CAK6Ddp8vc1szChIfaAr8vZGbWCuNIQlWEzLLj6V4Wtzr\\\\\\\/PgYuCZ5Ns8ztFRSVp4S\\\\\\\/JleRl6KvdZlaNPRRWHR6KAuf2sCPYQVfP3nax2GOwBVCL1nYIDaB6NPdNTag99LSG86w58lV+nC+cSDOTX+xJnYYmQoX4jhyjSmmijIk9RCTMHDtATIp6\\\\\\\/TtJpCJW0UM5Y8CRrxhPGn5a+4fwt1xWHdBxVmBsdqP3ekV\\\\\\\/Pkg1532XMfoSOkkOQOyvqodwJVgwVClFrfW2mQcoThdB+dsrDX5D6gC6ubHU17foS\\\\\\\/RFLD70t6KvpAHWTnX6VH5mMDxMMTNLheS+g0H0e1\\\\\\\/gQ5rz\\\\\\\/ndZTOohZOQicNg7STVfliJVcYewhpe0ZaG05jyeMl+U5dP3JYK2iI9v3nxyRiBun6C8PNkWEXam4WjExziLAV9J+oKbV5tsrJwciugMPcwjzbdivMQboGvPhFxvTtvnh1leCZs0ZZohbiWErs58Pu9Q5Qw79jGL3ZsxSzxQnS1yh7xYFvZr6K3CgekZJaMZqMqFksuHxM64S3tNG8ZMG2YeTbRnO1OdeR\\\\\\\/Lmau6Fg6safDmtQVGHvq2xUVs1rkoUR2dTi3y1ZJLpcxNGdh74ZKqg+FSl3Y+OgXfT6JD\\\\\\\/s\\\\\\\/XCp5kmgPuKs0YR2yFIqR7oFakrG9888yI+cuUzo\\\\\\\/\\\\\\\/bSKoZSc6OEJClMO6VfaudZAgRnAewXYA31ITUDTytcD0i+P64HVKmaRKagWr67xbCZ6DL9KKLthZm3Qqdl\\\\\\\/FUPQoe90OROXKH8g5hUI+vlDGYqhND3V0frXVo9Y5IOLpFyReC8sIOq4P93CNGb97hFHyTv35F+MZR1PlAF23FKDeyJRpBEvhLwKDBBps4nXbgepW80EUBzsHBVm+Vze4WtfSHXGBXGLvSjokxQhNBjvvagh90AK9Z+yZ9wzP\\\\\\\/J3RB8PLhTcKmjevttcz0M7hvg8Jb7lVTgH342aJyuQY1H+jrEP\\\\\\\/EDuXAbEhHYD04Nt7eTQaS0teFMRO6WDk9XG82zmd\\\\\\\/GDw+GnO1b+i2enOSWq\\\\\\\/y9Gp1cMgwZD08lM4yfZzSWSdtj2qcFiG3FCxl1w7qIDWHcgZ714DvgbxwUfBJLp7MYlSvZNTssFGJzVBGeQFk1D0RlxU4HxlRh7sbzRCdAEhjNdRUKZ8\\\\\\\/VsiiJpxqsnlUme\\\\\\\/jwbc74UaVS5VviFrqPoNEzf\\\\\\\/X7qYWytRrlT0raxPbONFBCg8TXQcsqSMluFgZMwc7o84zkweaKaV+HiA1pFRFF8RNOXoeW05LraANvnnJYHgDVogNjS9WLChEqBhc8EbSbv54NTlhtMEN6TLUKBj4QG4WjkahqB0gw2KBM+os4dtx1ABtuiTc6whJP65n3AxkdZtMTkOn9iApuJp\\\\\\\/6UzzWlGLL5KQ8R8sTQdDInnayfGr2X0sxs7ECo3dV3OqaDnq5Hwgue70s32iOln\\\\\\\/WhYrwX2UTlPv0BIcAEfNBIaXx7I4FMa8HWDQC6HllVtYLTmmnxh2xvfqicOsImsE56b8d\\\\\\\/XHRJNTkw2swU2t5hFQBi\\\\\\\/nsOgEbcVVy9azBLnBipoXnlNaaFoUkkgfxHefNeogT1oC4lnCY+qVY4yc17vERfa7wqYzPCGaeldt6jxh02IGOqJ1iSgIyMxPioOP5\\\\\\\/l+um3fxXtBJhzej8uDA5AfO7POQk50c83W6sF6NK1gaO4P7yzH199dpu5BWwSPGFCxC\\\\\\\/+1nEW5MiHOLAb9LBWXhf7hqVhwZ9Ty9aDYhHSWCqikLa4BkACEPYOLRPO9YpmXkd97N6S7u6nzFRlRl7nbecRMfL5QElbt1Bd8tM8NLcXVQl9mLt8+ZMyjT7UXmbaAvkIHKTmlwr1XEtf0v4HF9RYXYL1I\\\\\\\/EzGh55+zZ1SmTqVdsKJsjPs0xpYb9eFlGFTa1Ux1\\\\\\\/paY0IgxXie0t6Uv7+xG\\\\\\\/8aOQNmnXmeseKRn0a6C1s3JS72Jh68Ahdr\\\\\\\/i+GYlIgJATgqV4RQMtGfhLF5HIG1hYw0\\\\\\\/ZY31yeL9OEIbcp5wHC6MTKo59H\\\\\\\/nqubZNTXjrV9fiBy+gVk2rnI4CMSLJIdOLMoaIpMWPXBTeF+P6a6eQr4Qat7p8mM7+E6Mt\\\\\\\/9qspbOz2NG2uIthjO+QU3Fo2bOgkkoj+svRIFtL4Th3oXInyDwfPhdAtkNT1aP1o7+gbZ8M8BEMUv6UysQyvpvUPosrJoLTg7qZ6OiUjOFirz9\\\\\\\/aolYfzymQ3Dr6a0hsVNetG4DgVHJFGLl9UwulFFfDNWNrgqcz0W\\\\\\\/+nSGEhthPK\\\\\\\/K8RzxTaNh6mEeaOhB+yknvJ1OYH1wbZIwjPB+jLy\\\\\\\/v0Zq2aX7WfRSTPLFV3NvMZtYKawy+9ho9d4IKBl2vSYv0zhva7xpkk2fCGmIITA6v3pmH4MXnjvIB30PloW9d\\\\\\\/J0gAfClFcUIgAzwwxs6TiwjG7QrFmpa7eUG59ssQylISCAGH2I7SaT167ASeRYF5Xz3hMFjeRh+wxbJ0SYF8AHzi7clAx8mVtew57c3Ntel24hKJqeeJx74mPH1RB2jYRfDrQPdaMA8Mcx6NRteuXNrFaaDJfbfpt9I8N73hG4YloTPZoMFaQKLw1ey\\\\\\\/srBXrYfxUxz\\\\\\\/5Dbvz5ZMPB5Sz4o7coDwqg3rlxE1Y7zxZiouArIEEtV5JVfg7ukweoL5683QDiKYlPDpY4zQWxrRzma5mPPwav3N9HmEoq2gZgLWR95yEITjEpfPlz+c1\\\\\\\/W9xx5t9rDfVhragKO8Wp4wrv0oQkwbC07gVruQOhJ4RdEHfaQMgIc3yt1+sa9Plo82Q9rsLlhppcRKNBDaGWSIfJwMoVo8RitkrcH\\\\\\\/W3dQJte1eJXx1BtQjlUmM7\\\\\\\/rQpSnxLn4iHDZdcp3IhVqWMBUg9qpEBuccNcWYXqj+CBCNsDzKA8VzXrSEybeWtpNBi8rB0\\\\\\\/HnEkkV6Bc0mrFH\\\\\\\/C6Vr9tTa0eMZDtCg8jYSoEqQjn8OVKw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022leEDhmbqGtplld8+woVhCg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002226c4e2cbeb9362164c72bd972d8ed1f53822e410cd7407720584f154fb099b0e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rbx+e905AdnP6iH99wmyL4PBVp57YQQjVh9lggElXK+udpgcW53e7h38lb\\\\\\\/LlXXdsiaUZ8UEpxq4lHH476oiAUF0HfbtRcxc0Ob2RU258ptH5cu5Uy+Q1xJoERbasj+29roua3jWuNJUyuKJPVHgfR\\\\\\\/0InGuzj4hKZxTj2LAKEbTerCVIHYbhAWmC9O\\\\\\\/SiIRuFMuYDSocoI3lWxO3rlF3iLHt8zVrZYkuLDRHzYJ6mxRQ\\\\\\\/gYS6DDdnz8Ue9RQWPzpunCQyw00BXug0pPMqetxim5aWVff0MtvR0f2kRhQonwSev2Hr0q9ruLdiHVPZW5IpzXTuO\\\\\\\/piQM+Gbp9HtBTlWjrFk9Am7mjRtwn06N3yhhWKMkXay0rTivQPkMTFkFCxroDNDh1mauwjNZoXbcALZO+zUJ0jsKkYkFJ1c7aVMMC8YUS47osWsG1VdMFF1tup1IrhcLLKliScEgQwNFpBj1zj9YVj3U\\\\\\\/dDuf1WJxZzL53zxW3kyaOJrL74RnQRL0fuouoQ5SUL6JomgSucaJlkE7lNk779iowcjkMhtyx4rbkKOmkT19mVd+P0yolXuZ94f81oLN9C0Byo0Yx7yE1Ft6llQerIpi\\\\\\\/UfeD0KlpToeZPA81eE29daxVoTsJzLmG1cbfnZKyJSFaBp\\\\\\\/HP+TeWg1dX6bxqmOn50GfQljQEqfN5FkxU2zu35RgHtqHTZ0CKDqS3YmcTtjIyrnd5+WNC\\\\\\\/QgvQr4iDz6GH2dquo6F++Ph4a2ryzE\\\\\\\/jfb\\\\\\\/Q15TeeNco83cVA8lZC+S8tE9C707TNY9DRFu25LJkgGWWUDm\\\\\\\/hO3VxqyQCJYKioWSbVD9hblHM4QQZFPPZgABBgqhNQXc3hgsZWWi4Z3dJGaOYruiCeAw02VhaeidlxJPmWYrwP7XCB8Nly98e130hGEJveiHzaiKn7CLAlSurTTvxBoazf5U4ps80RFD10F32cFGfwSwplZg9biuvTG+XWkiRXYfDsmZQeZ+eumcYd4QM0K1VfTKtxdNwT6fakEmUgC7o8C7MCnKa6L7ZyzoIUzbvyw8cp3Q80gNKwgTG1vD\\\\\\\/zUCoJKgomzbgjZErm6QbhwyB+ktWumAc3akozmj7JgzOK2stGbf+y3yopQ1gJnJxI29sZa9oL2YE0TBznmiJ39bJa5qo6IgnsP4dCbsCB6tsIIaVpfXaICaedsDyaDCVKFwlOjL3gScALykBqwJ\\\\\\\/3nj9yIjjYYc4spcgxILw9eKp1B8FnaJRMWQPtjMJR42loFdkJsD6RYgGQZ2I1YjAiG5pf\\\\\\\/bpKf0S4Pt6jAtejaoIR809LgWgAl2yxhPah3JGfnoNamktBsSg6LKdnBOcGKt5QbYni7Wl6X+vt\\\\\\\/AvVCDcynK8p8ki4g8DmYbYl3GJ3P1BsOuaWhq2PQ7zXiLuNuQxgA7RJs129ZLFF1UKJ9LjpXSyn2hK0FE+TodlCiRGDglNJJXQs26HtknFxTXDOwpqrbkKbMPuQtaChIqsuhjsfO+fzuXIB8MjCe+kjl8gFTXMTd8AE8rm0lZdnIvnDu4WXQ\\\\\\\/WFx1Be05gu+txcuNaDyquw1WIivl4r3AwbVC5oJoR6JLHHXbLRWlxbLYuZEQz7kPpK2bVrdnxIyrTwYSkk0khgAuZDC2TM5uq5bAx+dBGpd16AhAY7Iyvc35t+YFubw\\\\\\\/uE\\\\\\\/ikwCa\\\\\\\/E8JyDF5TYqmHsnqE+vVlgx0Pqv+cKJmXM9LHR71pg4Q7qq8N2rugprmTF0s9VR1JqT9Hd6NQf6ObmZpv8D4nqtttDSKCXL5+bNelrfr8o94vipn\\\\\\\/cHMXB22x0CO3REjNVsgMVYvhSSSQXHBy+RqZUe4J0Mg0QuHEHpTE7q\\\\\\\/mgpDZf7p31siCmLbv6rSaOpWltE2j9qTWJYsGzjpq+ASZnh5F\\\\\\\/4M6KwBK6Cg1qXWuU34mXwB07WaWUyYtTtPxy8oLnUv0BfiOJEl5bMuLt+XyFMv6+WFpBnpb4sEwSfPjWbd5jfwy29Vt6GexhRFB90qz1TeW5KzHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022p2BXAC67RuM6+EDQZ5GgKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d822398c202bb328564062dfcc2832e87b39feb55be2fecf90d1a3af5dfc7662\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460510 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3787" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CVHtfCmpN2cX5MCOMWJ32u95ndRXXVd5en_tp5biuv8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3787" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lod1l5tXTeKoCs1Yr8dM7xKH7U2MbIneLWmr8IXrkCA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3765" title="Meeting Report: The Allied Genetics Conference 2016" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3803" title="Genome-Wide Association Study and QTL Mapping Reveal Genomic Loci Associated with Fusarium Ear Rot Resistance in Tropical Maize Germplasm" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460510" id="node460510" data-pisa="ggg;6/12/3787" data-pisa-master="ggg;g3.116.032805" data-apath="/ggg/6/12/3787.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessing the Gene Content of the Megagenome: Sugar Pine (<em>Pinus lambertiana</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Gonzalez-Ibeas</span>, <span class="highwire-citation-author" data-delta="1">Pedro J. Martinez-Garcia</span>, <span class="highwire-citation-author" data-delta="2">Randi A. Famula</span>, <span class="highwire-citation-author" data-delta="3">Annette Delfino-Mix</span>, <span class="highwire-citation-author" data-delta="4">Kristian A. Stevens</span>, <span class="highwire-citation-author" data-delta="5">Carol A. Loopstra</span>, <span class="highwire-citation-author" data-delta="6">Charles H. Langley</span>, <span class="highwire-citation-author" data-delta="7">David B. Neale</span> and <span class="highwire-citation-author" data-delta="8">Jill L. Wegrzyn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3787-3802;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032805 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460510" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Gonzalez-Ibeas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGonzalez-Ibeas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzalez-Ibeas%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGonzalez-Ibeas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pedro J. Martinez-Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BJ.%2BMartinez-Garcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez-Garcia%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BJ.%2BMartinez-Garcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Randi A. Famula </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randi%2BA.%2BFamula%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Famula%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandi%2BA.%2BFamula%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Annette Delfino-Mix </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture Forest Service, Institute of Forest Genetics, Placerville, California 95667</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annette%2BDelfino-Mix%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delfino-Mix%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnnette%2BDelfino-Mix%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kristian A. Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BA.%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BA.%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carol A. Loopstra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Ecosystem Science and Management, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carol%2BA.%2BLoopstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loopstra%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarol%2BA.%2BLoopstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David B. Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BB.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbneale@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dbneale@ucdavis.edu</a>
<a href="mailto:jill.wegrzyn@uconn.edu" class="" data-icon-position="" data-hide-link-title="0">jill.wegrzyn@uconn.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jill L. Wegrzyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jill%2BL.%2BWegrzyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wegrzyn%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJill%2BL.%2BWegrzyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbneale@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dbneale@ucdavis.edu</a>
<a href="mailto:jill.wegrzyn@uconn.edu" class="" data-icon-position="" data-hide-link-title="0">jill.wegrzyn@uconn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/12/3787" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460510" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460510/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3787.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460510" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460510/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3787.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460510" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460510/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3787.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460510" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460510/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/12/3787.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460510" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460510/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Sugar pine (<em>Pinus lambertiana</em> Douglas) is within the subgenus Strobus with an estimated genome size of 31 Gbp. Transcriptomic resources are of particular interest in conifers due to the challenges presented in their megagenomes for gene identification. In this study, we present the first comprehensive survey of the <em>P. lambertiana</em> transcriptome through deep sequencing of a variety of tissue types to generate more than 2.5 billion short reads. Third generation, long reads generated through PacBio Iso-Seq have been included for the first time in conifers to combat the challenges associated with <em>de novo</em> transcriptome assembly. A technology comparison is provided here to contribute to the otherwise scarce comparisons of second and third generation transcriptome sequencing approaches in plant species. In addition, the transcriptome reference was essential for gene model identification and quality assessment in the parallel project responsible for sequencing and assembly of the entire genome. In this study, the transcriptomic data were also used to address questions surrounding lineage-specific Dicer-like proteins in conifers. These proteins play a role in the control of transposable element proliferation and the related genome expansion in conifers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/conifer" class="hw-term hw-article-keyword hw-article-keyword-conifer" rel="nofollow">conifer</a></li><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/dicer" class="hw-term hw-article-keyword hw-article-keyword-dicer" rel="nofollow">dicer</a></li><li class="kwd"><a href="/keyword/gymnosperm" class="hw-term hw-article-keyword hw-article-keyword-gymnosperm" rel="nofollow">gymnosperm</a></li><li class="kwd"><a href="/keyword/dcl" class="hw-term hw-article-keyword hw-article-keyword-dcl" rel="nofollow">DCL</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-Seq</a></li></ul><p id="p-4">Gymnosperm genomes are among the largest sequenced to date. Their 14-fold variation between the minimum (<em>Gnetum ula</em>: 4.54 pg) and maximum (<em>Pinus gerardiana</em>: 57.35 pg) is much lower than the 1000-fold variation seen in angiosperms (1C = 0.05 ± 127.4 pg) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Leitch <em>et al.</em> 2001</a>). Interestingly, estimates of the total number of genes seems relatively constant across all land plants, ranging from 25,000 to 45,000, as observed recently in Norway spruce (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Nystedt <em>et al.</em> 2013</a>) as well as smaller genomes such as <em>Arabidopsis thaliana</em> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Swarbreck <em>et al.</em> 2008</a>) or <em>Gossypium arboreum</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2014a</a>). The cone bearing gymnosperms belonging to the Pinales order inhabit some of the largest ecosystems on earth, contributing significantly to global carbon assimilation. Within the Pinales, the Pinaceae are the largest extant conifer family with over 200 species. Their genomes have remarkable characteristics, including a constant number of chromosomes, enormous size, and a high proportion of repetitive elements (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Nystedt <em>et al.</em> 2013</a>). Despite challenges, inexpensive next-generation sequencing and custom assembly approaches produced two draft pine genomes (<em>P. taeda</em> and <em>P. lambertiana</em>) at 22 and 31 Gbp, respectively (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Stevens <em>et al.</em>, 2016</a>).</p><p id="p-5"><em>P. lambertiana</em> is a member of the genus <em>Pinus</em>, and is within the subgenus Strobus, which includes members known collectively as the white pines or five-needle pines. <em>P. lambertiana</em> occupies a variety of habitats throughout the Cascade range in Oregon to as far south as Baja California, Mexico. The majority of its range occurs in the mixed conifer forests of the Sierra Nevada (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kinloch and Scheuner 1990</a>). This tall and voluminous species shares habitat with several other tree species, and is rarely found in pure stands (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fites-Kaufman <em>et al.</em> 2007</a>). Disturbances such as historical logging, climate change, and introduction of the nonnative pathogen <em>Cronartium ribicola</em>, have sharply reduced <em>P. lambertiana</em> populations (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Maloney <em>et al.</em> 2011</a>).</p><p id="p-6">The conifer genomes have already contributed to advancements in conifer biology (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2015</a>); however, the fragmented nature of the final assemblies (each containing over 14 million scaffolds) supports the need for comprehensive transcriptomic resources (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Visser <em>et al.</em> 2015</a>). Recent advancements in transcriptome characterization, through techniques such as RNA-seq, have contributed to improved resolution of transcripts, and the subsequent ability to quantify gene expression in thousands of genes at a time (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Conesa <em>et al.</em> 2016</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kanitz <em>et al.</em> 2015</a>). Short read technologies, available through the numerous Illumina platforms, provide substantial depth at a low cost with reads that typically range from 50 to 300 nucleotides (nt) in length (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cahill <em>et al.</em> 2010</a>). In the absence of a contiguous genome assembly, researchers rely on <em>de novo</em> assembly techniques to organize those short reads into full-length transcripts (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Moreton <em>et al.</em> 2015</a>). Recently, the precision and sensitivity of RNA-seq have come into question, especially for transcriptome reconstruction (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Korf 2013</a>). A relatively new method known as “Isoform Sequencing” (Iso-Seq), developed by Pacific Biosciences (PacBio), is capable of identifying new isoforms up to 6 kb in length due to its long read, single molecule sequencing technology. This methodology has been used independently, as well as in combination with short read approaches to improve transcript identification. The Iso-Seq approach has been applied to human tumor cell lines and recently to select plant genomes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Xu <em>et al.</em> 2015</a>). To date, the effectiveness of long read transcriptome sequencing approaches has been evaluated shallowly in select angiosperms and never in conifers.</p><p id="p-7">Extensive transcriptome resources have been developed for several conifer species, particularly those of tremendous economic value. Early work has included cDNA microarrays to examine expression responses to biotic and abiotic stressors ranging from 1248 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Myburg <em>et al.</em> 2006</a>) to 26,496 ESTs (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lorenz <em>et al.</em> 2011b</a>). Following this, large-scale Sanger-based EST sequencing produced hundreds of thousands of sequences with the greatest contributions to <em>P. taeda</em> and <em>Picea glauca</em>, both having over 300,000 sequences in GenBank (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mackay and Dean 2011</a>). Among pines within the subgenus <em>Strobus</em>, very few resources have been developed. In this study, we have implemented PacBio Iso-Seq for the first time in conifers to improve the accuracy of transcript construction and evaluate its utility against traditional, short read, deep sequencing approaches. Novel sequencing approaches combined with comprehensive tissue sampling provides the greatest depth and most detailed analysis of a white pine transcriptome to date.</p><p id="p-8">The recent availability of a draft <em>P. lambertiana</em> genome sequence, coupled with transcriptomics, offers opportunities to study basic questions about the biology of conifers as it relates to genome evolution and gene expression. Genome sequencing in conifers has led to observations of genome expansion resulting primarily from transposable element (TE) proliferation rather than genome duplications (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wegrzyn <em>et al.</em> 2014</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Nystedt <em>et al.</em> 2013</a>). The peculiar profile of the small RNAs population in these plant species, and the previous identification of potential lineage-specific, Dicer-like (DCL) proteins (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dolgosheina <em>et al.</em> 2008</a>), raises questions about whether the mechanism for controlling genome size through epigenetic modifications works differently in gymnosperms. In this study, we take advantage of the characterized transcriptome to provide new insight on conventional and conifer-specific DCLs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-9">A comprehensive collection of tissues was made from 12 existing <em>P. lambertiana</em> trees (11-91 6000, 11-92 6000, 11-94 6000, 11-99 5701, JJ-86 11,101, JJ-101 11,105, GG 79 15,306, V-120 18,856, E-109 7392, B-109 BLM-8, JJ-105 11,200, 11-105 5503) in the clone bank at Badger Hill in the El Dorado National Forest in California (USDA Forest Service). This collection included: megagametophytes; embryos; cotyledon stage seedlings before development of primary needles, containing only cotyledons, stem, and root (labeled “basket” stage); primary needle stage seedlings; pollen; early female cones before pollination; female cones near pollination; 2 cm female cones after pollination; stems; and roots. From the same clone bank, open pollinated seeds were collected. Seeds were germinated and established seedlings were used for various treatments conducted at the Institute for Forest Genetics (Placerville, CA). Two grown seedlings were used to simulate a salt stress via a soil drench using large quantities of 200 mM NaCl, before harvesting all three tissues after 2 hr. To study effects of wounding, needle nose pliers were used to crush needles and stems while still on the tree. We harvested needles and stems after 4 hours. To simulate pathogen or insect attack, trees were treated with Jasmonic acid (JA) [100 μm JA plus 0.02% tween (a wetting agent)]. This solution was applied as a drench to the roots and sprayed on the foliage. Needles, stems, and roots were harvested after 4 hr of inoculation, but only the stem was used in our analysis. Tissues from samples were separately harvested in needles, roots, and stems and collected in 50 ml tubes. In order to preserve the integrity of the drought stress treatment, samples were frozen immediately, as water could initiate reversal.</p></div><div id="sec-3" class="subsection"><h3>Library construction and sequencing</h3><p id="p-10">Total RNA was isolated by adapting the method described by <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sangha <em>et al.</em> (2010)</a>, which combined a CTAB-based lysis solution with the silica column-based RNA binding, DNase, and washing steps from an RNeasy Plant Mini Kit (Qiagen, Germany). RNA quality was evaluated using the Agilent Bioanalyzer 2100 (Agilent Technologies, Folsom, CA). All Illumina libraries were constructed at the Vincent J. Coates Genome Sequencing Laboratory (University of California, Berkeley) on the IntegenX Apollo 324 robot (Wafergen, Fremont, CA). Illumina MiSeq libraries were constructed with an insert of 500 nt, and sequenced in individual lanes, 300 nt PE, 600 cycles, using Version 3 chemistry (Illumina, San Diego, CA). RNA samples for the Illumina HiSeq were treated prior to library construction with a Ribo-Zero rRNA Removal Kit (Plant) (Illumina). Nine HiSeq 2000 libraries were constructed with standard insert sizes, and sequenced as 100 nt PE in individual lanes (Illumina). PacBio Iso-Seq libraries were constructed following the PacBio modified protocol using the Clontech SMARTer PCR cDNA Synthesis Kit and Blue Pippin Size Selection System. Insert sizes were selected for the following inserts: 1, 2, and 3–6 kb (Sage Science, Beverly, MA). Libraries were then prepared using the SMRTbell library protocol (PacBio, Menlo Park, CA). Each library was sequenced across four SMRT cells on the PacBio RSII using P6-C4 chemistry, at the UC Davis Genome Center (University of California, Davis).</p></div><div id="sec-4" class="subsection"><h3>Quality control and transcriptome assembly</h3><p id="p-11">Short read technologies (Illumina MiSeq and HiSeq) and the PacBio Iso-Seq reads, which result from size-selected libraries ranging from 1000 to over 6000 nt, were included in both single and combined <em>de novo</em> assemblies. Seven MiSeq, 9 HiSeq, and 9 PacBio libraries were included. A total of 35 SMRT cells (1–4 SMRT cells per library) were sequenced and analyzed. The HiSeq and MiSeq Illumina reads were quality filtered and trimmed via Sickle (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Joshi and Fass 2011</a>) (v1.33, min. quality 35, min. sequence length 45 nt) and visually analyzed with FastQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Quality trimmed Illumina reads from each library were independently <em>de novo</em> assembled with Trinity (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Haas <em>et al.</em> 2013</a>) (v.trinityrnaseq-r20140413p1, min. contig length 300 nt PacBio data were quality filtered (min. length 300 nt, read quality ≥ 0.7) and analyzed with the SMRT pipeline (<a href="https://github.com/PacificBiosciences/SMRT-Analysis">https://github.com/PacificBiosciences/SMRT-Analysis</a>). Raw reads were processed to obtain the circular consensus reads (CCS) and, additionally, CCS were subjected to an isoform level clustering step with ICE/Quiver, also provided through the SMRT pipeline (default parameters). Chimeric reads were evaluated with the RS_IsoSeq classify tool as the difference between total full-length and total full-length nonchimeric reads. PacBio results are provided for transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). For analysis of the number of full-length transcripts, sequences were queried against a local database containing curated plant protein sequences by means of USEARCH-UBLAST (v7.0.1090, <em>E</em>-value threshold of 1e−9) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Edgar 2010</a>). Three types of hits were recorded: total hits (<sup>1</sup>H), hits covering 70% of the transcript (H2), and hits covering 70% of the transcript and 70% of the aligned protein (H3). These last two categories were used to estimate the proportion of potential full-length transcripts in the data. Rarefaction curves were generated by randomly selecting 1000 transcripts and analyzing mapped reads (see <em>Transcript abundance estimation</em> section) with the R (v3.3.0) package, Vegan (v2.3-4), to ascertain whether the sequencing depth and coverage were sufficient (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Oksanen <em>et al.</em> 2016</a>). Ribosomal RNA contamination among the assembled transcripts (before CDS identification) was assessed via BLAST (v2.2.29+, <em>E</em>-value 1e−9) against the SILVA database (release 04.04.2016) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Quast <em>et al.</em> 2013</a>).</p></div><div id="sec-5" class="subsection"><h3>Transcriptome annotation</h3><p id="p-12">Following assembly, coding DNA sequences (CDS) were identified with Transdecoder (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Haas <em>et al.</em> 2013</a>) (v.trinityrnaseq-r20140413p1) for both Illumina and PacBio CCS reads. Conifer protein sequences (<em>P. taeda</em> and <em>P. abies</em>), retrieved from PineRefSeq (<a href="http://treegenesdb.org/ftp/Genome_Data/genome/pinerefseq/Pita/">http://treegenesdb.org/ftp/Genome_Data/genome/pinerefseq/Pita/</a>) (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Wegrzyn <em>et al.</em> 2014</a>) and Congenie (<a href="ftp://plantgenie.org/Data/ConGenIE/Picea_abies/">ftp://plantgenie.org/Data/ConGenIE/Picea_abies/</a>) (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sundell <em>et al.</em> 2015</a>) projects, respectively, were used to train the machine learning component, and Pfam (v28.0) domain identification was used for CDS retention. High quality polished sequences from the ICE/Quiver clustering were also used for CDS identification with ANGEL (<a href="https://github.com/PacificBiosciences/ANGEL">https://github.com/PacificBiosciences/ANGEL</a>) using the same conifer sequences for training to complement the Transdecoder analysis. All CDS from Illumina and PacBio data were clustered at 95% sequence identity with USEARCH-UCLUST (v8.1.1861) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Edgar 2010</a>) to generate a nonredundant set of transcripts. For functional annotation, the longest complete CDS from each transcript was subject to USEARCH-UBLAST to identify local alignments (v7.0.1090, <em>E</em>-value threshold of 1e−9 and a weak <em>E</em>-value of 0.0001) (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Edgar 2010</a>). NCBI’s RefSeq Protein (Release 69) (accessed Dec 2015), and the <em>Arabidopsis</em> protein database (TAIR, v10) were queried. Selection and assignment of the best annotation based upon the alignments was performed with the Eukaryote Non-Model Transcriptome Annotation Pipeline (enTAP v1.0, <a href="https://github.com/SamGinzburg/WegrzynLab">https://github.com/SamGinzburg/WegrzynLab</a>, <em>E</em>-value 1e−5). Transcripts associated with bacterial, fungal, and insect contaminants were filtered as part of the annotation process. Gene Ontology (GO) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashburner <em>et al.</em> 2000</a>) terms were assigned for Molecular Function, Biological Process, and Cellular Component with Blast2GO (v3.2.7, default parameters) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> 2005</a>). MicroRNA (miRNA) annotation was analyzed with MIRENA (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mathelier and Carbone 2010</a>) via previously identified miRNAs available in MirBase (v21) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kozomara and Griffiths-Jones 2014</a>). Over 800,000 transcripts lacking a CDS were used as input. miRNA precursors were identified, allowing up to two mismatches and a minimum MFEI index of −0.85 as a cutoff (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zhang <em>et al.</em> 2006a</a>). Selection of high quality sequences was performed by manual inspection of RNA precursor secondary structures generated by ViennaRNA (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lorenz <em>et al.</em> 2011a</a>) on the set of conserved miRNAs across land plants (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Zhang <em>et al.</em> 2006b</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cuperus <em>et al.</em> 2011</a>). Precursors were considered high quality if they met previously described miRNA structural requirements (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Meyers <em>et al.</em> 2008</a>).</p></div><div id="sec-6" class="subsection"><h3>Evaluation of completeness</h3><p id="p-13">Completeness of the gene space was analyzed by following the single-copy orthologous approach deployed in the BUSCO pipeline (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Simão <em>et al.</em> 2015</a>) with default parameters and the plant reference set (950 orthologs). Assembled transcripts were also mapped against the <em>P. lambertiana</em> reference genome (v1.0) with Gmap (v2015-06-23) (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Stevens <em>et al.</em>, 2016</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wu and Watanabe 2005</a>). The gmapl version of the software was used due to the large assembled genome size. Mapping rate was calculated as the number of transcripts aligning at 98% of coverage and 98% of sequence identity, as well as 90% coverage/98% identity.</p></div><div id="sec-7" class="subsection"><h3>Gene family analysis</h3><p id="p-14">The Markov cluster (MCL) algorithm analysis (v.12-068) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Enright <em>et al.</em> 2002</a>), as implemented in TRIBE-MCL (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dongen and Abreu-Goodger 2012</a>), was used to cluster the 385,329 protein sequences from 13 species into orthologous groups. Species included: <em>Glycine max</em> (37,388), <em>Ricinus communis</em> (28,113), <em>Populus trichocarpa</em> (36,393), <em>A. thaliana</em> (27,160), <em>Theobroma cacao</em> (28,136), <em>Vitis vinifera</em> (25,663), <em>Oryza sativa</em> (41,186), <em>Zea mays</em> (37,805), <em>Physcomitrella patens</em> (36,393), <em>P. lambertiana</em> (33,113), <em>P. taeda</em> (21,346), <em>P. abies</em> (19,607), and <em>P. glauca</em> (13,026). Angiosperm sequences were retrieved from the PLAZA (v3.0) set (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Proost <em>et al.</em> 2015</a>), pine sequences from the PineRefSeq project (<a href="http://dendrome.ucdavis.edu/ftp/Genome_Data/genome/pinerefseq/">http://dendrome.ucdavis.edu/ftp/Genome_Data/genome/pinerefseq/</a>), and spruce sequences from the Congenie project (<a href="ftp://plantgenie.org/Data/ConGenIE/">ftp://plantgenie.org/Data/ConGenIE/</a>) (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Sundell <em>et al.</em> 2015</a>). All protein sequences were clustered at 95% identity with USEARCH-UCLUST. Subsequently, pairwise NCBI BLAST v2.2.29+ (<em>E</em>-value 1e−05) was run against the clustered set (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>). The negative log10 of the resulting blastp <em>E</em>-values was used to define the orthologous groups, and a moderate inflation value of 4.0 was selected. Following family assignments, Pfam domains were identified from the PLAZA (v3.0) annotations of the individual sequences. InterProScan (v.5.13-52.0; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hunter <em>et al.</em> 2012</a>) was applied to those sequences obtained outside of PLAZA. Pfam and GO assignments with <em>E</em>-values &lt; 1e−05 were retained. Families with protein domains classified as retroelements were removed. After functional assessment and filtering, custom scripts and Venn diagrams (<a href="http://bioinformatics.psb.ugent.be/webtools/Venn/">http://bioinformatics.psb.ugent.be/webtools/Venn/</a>) were applied to visualize gene family membership among species.</p></div><div id="sec-8" class="subsection"><h3>Transcript abundance estimation</h3><p id="p-15">Transcript abundance estimation between samples without replicates was calculated with Gfold (v.1.1.2) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feng <em>et al.</em> 2012</a>). Treated samples were compared against their respective control: NaCl-treated root samples (NACLR) <em>vs.</em> untreated root (DCR); stem of methyljasmonate-treated plants (JASS) <em>vs.</em> stem of untreated plants (DCS); and stem tissue after wounding (WS) <em>vs.</em> stem of untreated plants (DCS). Among tissue types, reproductive tissues were compared with the basket sample (blend of needle, stem, and root). Quality filtered Illumina reads were mapped against the set of 33,113 assembled <em>P. lambertiana</em> transcripts with Tophat2 (v2.1.0) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kim <em>et al.</em> 2013</a>) with default parameters. Alignment (SAM) files were used as input for Gfold. A minimum fold change of 2.0 (-sc 2.0) was required for genes to be identified as differentially expressed. The expression table provided by Gfold for the complete set of 33,113 transcripts was used as input for labdsv (v1.8-9) R (v3.3.0) package (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Roberts 2016</a>) to perform the Principal Component Analysis (PCA). Overrepresented GO terms in differentially expressed genes were analyzed with Blast2GO (v3.2.7, default parameters) (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> 2005</a>).</p></div><div id="sec-9" class="subsection"><h3>Analysis of Dicer gene family</h3><p id="p-16">DCL sequences were identified through functional annotations assigned via enTAP. Gene models corresponding to DCL <em>P. lambertiana</em> proteins were retrieved from the genome annotation (v1.0) (<a href="http://dendrome.ucdavis.edu/ftp/Genome_Data/genome/pinerefseq/Pila/v1.0/gene_models/">http://dendrome.ucdavis.edu/ftp/Genome_Data/genome/pinerefseq/Pila/v1.0/gene_models/</a>) and transcriptome. USEARCH similarity searches were performed against <em>Arabidopsis</em> DCLs from GenBank (NM_001197952.1, NM_001202869.1, NM_001161190.2, and NM_122039.4) as well as protein domains identified by InterProScan (helicase, Dicer, PAZ, RNAseIII, and ds-RNA binding). Protein sequence alignments were generated with MUSCLE (v3.8.31) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edgar 2004</a>). Phylogenetic trees were generated with Fastree (v2.1.8) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Price <em>et al.</em> 2010</a>) and visualized with FigTree (v1.4.1) (<a href="http://tree.bio.ed.ac.uk/">http://tree.bio.ed.ac.uk/</a>). The redundant set of transcripts (before sequence clustering) and not the unique set (33,113 transcripts) was used for DCL analysis, in order to identify sequence variants and to provide additional evidence that the same or similar transcripts were sequenced from different tissues and/or libraries.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-17">Assemblies are available in the TreeGenes database (<a href="http://treegenesdb.org/ftp/Genome_Data/genome/pinerefseq/Pila/v1.0/transcriptome/">http://treegenesdb.org/ftp/Genome_Data/genome/pinerefseq/Pila/v1.0/transcriptome/</a>) and in NCBI as a TSA submission (GEUZ00000000). Raw reads, including the current draft sugar pine genome assembly, are available in NCBI BioProject PRJNA174450 (samples SAMN05256544, SAMN05256552, SAMN05271999, SAMN05272013, SAMN05272041, SAMN05272042, SAMN05272043, SAMN05272242, SAMN05272243, SAMN05282317, SAMN05282318, SAMN05282319, SAMN05282324, SAMN05282872, and SAMN05282873; SRA accession numbers for sequencing data SRR3689473, SRR3696256, SRR3696257, SRR3710655, SRR3712438 to SRR3712442, SRR3723920 to SRR3723927, SRR3724538, SRR3825176 to SRR3825202, and SRR3882733 to SRR3882738). Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS1.xlsx">Table S1</a> contains the tissue sample description and sequencing statistics; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS2.xlsx">Table S2</a> lists the number of raw transcripts with similarity to ribosomal RNA; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS3.xlsx">Table S3</a> lists number and types of splice variants identified; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS4.xlsx">Table S4</a> contains all the statistically significant GO terms identified in differentially expressed transcript sets; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS5.xlsx">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS6.xlsx">Table S6</a> contain the number of proteins that compose the conifer-specific protein families and protein annotations, respectively; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS7.xlsx">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS8.xlsx">Table S8</a> contain the number of proteins that compose each <em>P. lambertiana</em>-specific protein family and protein annotation, respectively; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS9.xlsx">Table S9</a> lists miRNA precursors; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS1.pdf">Figure S1</a> shows rarefaction curves of sequenced libraries; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS2.pdf">Figure S2</a> shows transcript length distribution of assembled transcripts where no CDS was identified; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS3.pdf">Figure S3</a> shows transcript length distribution for samples covered by different sequencing technologies; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS4.pdf">Figure S4</a> shows the contribution of each technology to improve the transcript completeness (extension of <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>); <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS5.pdf">Figure S5</a> shows the number of splice variants provided by each technology in samples covered by several sequencing technologies (extension of <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>); <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS6.pdf">Figure S6</a> shows plant species with the most protein sequence similarity to <em>P. lambertiana</em> transcripts; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS7.pdf">Figure S7</a> shows a transcriptome characterization by tissue samples. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS8.pdf">Figure S8</a> shows a treatment-specific <em>P. lambertiana</em> transcript analysis; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS9.pdf">Figure S9</a> shows PCA of sugar pine samples used for gene expression estimation; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS10.pdf">Figure S10</a> number of differentially expressed genes shared among several samples; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS11.pdf">Figure S11</a> shows a phylogenetic analysis of DCL proteins from <em>P. lambertiana</em> and several plant species; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS12.pdf">Figure S12</a> shows a gene expression analysis inferred from sequencing data of <em>P. lambertiana</em> transcripts codifying for DCL proteins; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS13.pdf">Figure S13</a> shows secondary structure from three <em>P. lambertiana</em> miRNA precursors; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FileS1.pdf">File S1</a> provides an extended description of the analysis of 2 cm female cones and female cones at the time of pollination for transcriptome diversity analysis; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FileS2.pdf">File S2</a> provides an extended description of the gene expression analysis.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Transcriptome sequencing and assembly</h3><p id="p-18">A total of seven MiSeq, nine HiSeq libraries, and 35 PacBio SMRT cells corresponding to nine libraries from four samples were sequenced and analyzed, providing a set of 2.5 billion and 1.6 million Illumina and PacBio reads, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS1.xlsx">Table S1</a>). A variety of tissues were included: vegetative (stem, root, and needle), reproductive (male and female cones and embryos), and various biotic and abiotic treatments. Select samples were represented by several technologies [embryo by all three platforms, 2 cm female cones, and female cones at the time of pollination by both HiSeq and PacBio]. Deep sequencing was obtained for each Illumina library with read totals ranging from 116 to 134 million for HiSeq, and 14 to 19 million for MiSeq. Over 100,000 transcripts were obtained from each <em>de novo</em> library assembly with an average length of 906 nt. Total unique genes ranged from 50K to 100K for HiSeq and 49K to 116K for MiSeq. The size-selected PacBio libraries represented 11 unique libraries (four of 1 kb, four of 2 kb, and three of 3–6 kb) in order to capture the full range of transcript sizes, and generated 60,000 raw sequences per SMRT cell. This yielded between 14K and 125K full-length and nonchimeric transcripts per library. The percentage of chimeric reads discarded ranged from 0.15 to 0.43% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS1.xlsx">Table S1</a>). Transcripts had an average length of 1736, 1917, and 3570 nt for the 1, 2, and 3–6 kb size-selected libraries, respectively, revealing the effectiveness of size selection. Overall, quality was inversely proportional to read length, likely due to the fewer total passes to build the consensus sequence for the longest reads. SMRT tools provided an additional isoform level clustering step with ICE/Quiver producing three transcript sets: consensus isoforms (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). Clustering did not improve the length of the identified transcripts, but the high quality polished transcripts resulted in a set of sequences that performed well in terms of functional annotation and genome alignment.</p><p id="p-19">Following the detection of CDSs from 1,087,300 assembled transcripts, 278,812 were clustered at 95% identity to provide a set of 33,113 unique, high quality, full-length transcripts, which ranged from 300 to 13,000 nt in length (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In the case of PacBio transcripts, more than one open reading frame was identified in the opposite direction in 6.79% of the sequences. After transcript selection (CDS identification and clustering), the percentage was reduced to 0.05%. Sequencing saturation of the libraries was estimated for each technology (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS1.pdf">Figure S1</a>). HiSeq revealed the greatest saturation, MiSeq to a lesser extent, and PacBio reads did not reach complete saturation. Assembled transcripts with similarity to ribosomal RNA represented less than 0.1% for Illumina libraries and, at most, 3.8% for PacBio libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS2.xlsx">Table S2</a>). Sequences related to transposons and other retroelements accounted for 24% of sequences, which were discarded. This is not in agreement with estimations of transposon content in conifer genomes, since not all elements are transcribed.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The selection of P. lambertiana tissues for transcriptome sequencing and the sequencing technologies applied." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The selection of &lt;em&gt;P. lambertiana&lt;/em&gt; tissues for transcriptome sequencing and the sequencing technologies applied.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F1.medium.gif" width="360" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/12/3787/F1.medium.gif" width="360" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469909" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">The selection of <em>P. lambertiana</em> tissues for transcriptome sequencing and the sequencing technologies applied.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469825/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469825/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Transcriptome statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparison of sequencing technologies</h3><p id="p-21">DNA sequencing represents one of the most significant technological revolutions in the past decade (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">van Dijk <em>et al.</em> 2014</a>). Second generation technologies, as implemented by Illumina, have provided an increase in throughput at the cost of read length (25–300 nt) and quality compared to Sanger sequencing. Third generation technology, currently provided primarily through PacBio, provides lower throughput and lower quality reads at a higher price point but with significantly longer lengths (up to 20 kb) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Glenn 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Quail <em>et al.</em> 2012</a>). In this study, germinated <em>P. lambertiana</em> seed (embryo) libraries (PacBio, MiSeq, and HiSeq) were evaluated for their overall contribution to accurate and comprehensive <em>de novo</em> assembled transcripts. Reads from each library were assembled independently and subsequently combined. The assemblies were analyzed across several metrics to determine the ability of deep Iso-Seq to replace second generation strategies. The analysis focused on both the individual transcripts (length, completeness, and mapping rates), as well as the whole transcriptome (coverage and diversity) to provide the first in-depth sequencing technology comparison in conifers.</p><div id="sec-14" class="subsection"><h4>Comparison of Illumina and PacBio transcriptome assemblies:</h4><p id="p-22">Transcript length comparison of independently assembled reads demonstrated that PacBio overwhelmingly produces longer transcripts (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In comparing the selected CDSs (trimmed CDS sets as defined by Transdecoder), PacBio yielded a larger number of complete CDSs than Illumina [8940 <em>vs.</em> 7892 (MiSeq) and 8782 (HiSeq) transcripts], in spite of starting with fewer reads. However, the length of PacBio transcripts was significantly reduced after high quality, full-length protein sequences were selected. The resulting processed lengths were similar to the processed Illumina transcripts (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). It is difficult to assess, given the bias introduced by angiosperm-dominated databases, whether conifers have longer CDS sequences than PacBio is able to detect or this technology is producing unlikely transcripts with no biological meaning. GC content of the different sequence sets did not show strong differences, but a slight increase was noted after transcript selection (from 39 to 42%) relative to raw transcripts, and for PacBio transcripts relative to Illumina (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). In some studies, PacBio has shown bias toward GC-rich sequences in genome sequencing (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Quail <em>et al.</em> 2012</a>). Among transcripts without a CDS, 43.5% of Illumina and 55.6% of PacBio transcripts were not aligned, respectively. It is interesting that, in the case of Illumina, the transcript length difference between mapped and nonmapped transcripts was small, but was larger in the case of PacBio (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS2.pdf">Figure S2</a>). Less than 4% of transcripts were identified as full-length (70% of reciprocal coverage query:target) with either technology (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The same analysis after CDS selection yielded a significant improvement (as much as 21%) in the number of full-length transcripts (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), revealing transcript selection as efficient for selecting potential full-length protein-coding transcripts. To evaluate the transcripts against the draft genome, the percentage of transcripts aligning at various coverage and identity combinations was calculated. Less than 50% of PacBio transcripts mapped to the genome compared to the assembled Illumina transcripts (&gt; 70%) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Following transcript selection, close to 60% of PacBio transcripts aligned while nearly 90% of Illumina transcripts aligned. Some transcripts aligned to the end of two different scaffolds due to the fragmentation of the early draft genome assembly, contributing to a reduction of the mapping rates.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcript length distribution resulting from different assemblies of the embryo samples across the three technologies (HiSeq, MiSeq, and PacBio). Length of transcripts was used to build a box-plot distribution before and after transcript selection (CDS identification + clustering). PacBio results are provided for transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). Embryo transcriptome: combination of independent assemblies of Illumina and PacBio data and transcript selection (CDS identification + clustering). Average GC content of transcripts is shown in the bottom of the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Transcript length distribution resulting from different assemblies of the embryo samples across the three technologies (HiSeq, MiSeq, and PacBio). Length of transcripts was used to build a box-plot distribution before and after transcript selection (CDS identification + clustering). PacBio results are provided for transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). Embryo transcriptome: combination of independent assemblies of Illumina and PacBio data and transcript selection (CDS identification + clustering). Average GC content of transcripts is shown in the bottom of the figure.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F2.medium.gif" width="440" height="410"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/12/3787/F2.medium.gif" width="440" height="410"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470028" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Transcript length distribution resulting from different assemblies of the embryo samples across the three technologies (HiSeq, MiSeq, and PacBio). Length of transcripts was used to build a box-plot distribution before and after transcript selection (CDS identification + clustering). PacBio results are provided for transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). Embryo transcriptome: combination of independent assemblies of Illumina and PacBio data and transcript selection (CDS identification + clustering). Average GC content of transcripts is shown in the bottom of the figure.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcript completeness analysis of different assemblies for embryo samples across the three technologies (HiSeq, MiSeq, and PacBio). Total number of sequences (T) were queried against NCBI RefSeq plant proteins by means of USEARCH-UBLAST. Three types of hits were counted: total number of hits (1H), hits covering 70% of the transcript (H2), and hits covering 70% of the transcript and 70% of the aligned protein (H3). (A) Raw assembled transcripts. (B) Sequences after transcript selection (CDS identification + clustering). (C1) Raw transcripts obtained with SMRT analysis for library Embryo_3&#x2013;6 kb: transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). (C2) The same as C1, but expressed as percentage of sequences relative to the total number of transcripts (T). (D1) Sequences from (C1) after transcript selection (CDS identification + clustering). (D2) The same as (D1), but expressed as percentage of sequences relative to the total number of transcripts (T)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transcript completeness analysis of different assemblies for embryo samples across the three technologies (HiSeq, MiSeq, and PacBio). Total number of sequences (T) were queried against NCBI RefSeq plant proteins by means of USEARCH-UBLAST. Three types of hits were counted: total number of hits (&lt;sup&gt;1&lt;/sup&gt;H), hits covering 70% of the transcript (H2), and hits covering 70% of the transcript and 70% of the aligned protein (H3). (A) Raw assembled transcripts. (B) Sequences after transcript selection (CDS identification + clustering). (C1) Raw transcripts obtained with SMRT analysis for library Embryo_3&#x2013;6 kb: transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). (C2) The same as C1, but expressed as percentage of sequences relative to the total number of transcripts (T). (D1) Sequences from (C1) after transcript selection (CDS identification + clustering). (D2) The same as (D1), but expressed as percentage of sequences relative to the total number of transcripts (T).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F3.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/12/3787/F3.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469846" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Transcript completeness analysis of different assemblies for embryo samples across the three technologies (HiSeq, MiSeq, and PacBio). Total number of sequences (T) were queried against NCBI RefSeq plant proteins by means of USEARCH-UBLAST. Three types of hits were counted: total number of hits (<sup>1</sup>H), hits covering 70% of the transcript (H2), and hits covering 70% of the transcript and 70% of the aligned protein (H3). (A) Raw assembled transcripts. (B) Sequences after transcript selection (CDS identification + clustering). (C1) Raw transcripts obtained with SMRT analysis for library Embryo_3–6 kb: transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3). (C2) The same as C1, but expressed as percentage of sequences relative to the total number of transcripts (T). (D1) Sequences from (C1) after transcript selection (CDS identification + clustering). (D2) The same as (D1), but expressed as percentage of sequences relative to the total number of transcripts (T).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping rates of different transcript sets to the P. lambertiana genome (v1.0). Sequences were aligned to the P. lambertiana genome and the percentage of mapped transcripts was calculated at two combinations of coverage and sequence identity. (A) Transcripts obtained from the SMRT analysis for embryo (3-6 kb size-selected): transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3); before (A1) and after (A2) transcript selection (CDS identification + clustering). (B) Pool embryo: all size selected (PacBio), HiSeq, and MiSeq (Illumina); before (B1) and after (B2) transcript selection. (C) Complete transcriptome set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping rates of different transcript sets to the &lt;em&gt;P. lambertiana&lt;/em&gt; genome (v1.0). Sequences were aligned to the &lt;em&gt;P. lambertiana&lt;/em&gt; genome and the percentage of mapped transcripts was calculated at two combinations of coverage and sequence identity. (A) Transcripts obtained from the SMRT analysis for embryo (3-6 kb size-selected): transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3); before (A1) and after (A2) transcript selection (CDS identification + clustering). (B) Pool embryo: all size selected (PacBio), HiSeq, and MiSeq (Illumina); before (B1) and after (B2) transcript selection. (C) Complete transcriptome set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F4.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/12/3787/F4.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Mapping rates of different transcript sets to the <em>P. lambertiana</em> genome (v1.0). Sequences were aligned to the <em>P. lambertiana</em> genome and the percentage of mapped transcripts was calculated at two combinations of coverage and sequence identity. (A) Transcripts obtained from the SMRT analysis for embryo (3-6 kb size-selected): transcripts identified as full-length (Pa), and set of transcripts after ICE/Quiver for isoform level clustering: consensus sequences (Pb1), low quality polished sequences (Pb2), and high quality polished sequences (Pb3); before (A1) and after (A2) transcript selection (CDS identification + clustering). (B) Pool embryo: all size selected (PacBio), HiSeq, and MiSeq (Illumina); before (B1) and after (B2) transcript selection. (C) Complete transcriptome set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The single PacBio Iso-Seq embryo library (3–6 kb size selected) was selected to analyze each of the four outputs from the SMRT pipeline. After full-length transcripts (Pa in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) were assembled from raw reads, the software provides an optional isoform level clustering step to reduce isoform redundancy (Pb1 in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), and an additional sequence polishing step to improve quality (Pb2 and Pb3). The isoform level clustering step did not improve the length of the identified transcripts (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, lanes 4, 5, 6, and 7). Similar to the pool of all PacBio libraries (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, lanes 3 and 10), transcript selection of high quality proteins resulted in a reduction of transcript lengths (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, lanes 11, 12, 13, and 14). However, after transcript selection, longer lengths were achieved in the “polished” sets (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>, lanes 13 and 14). When evaluated against characterized proteins, CDS selection increased the number of full-length sequences for each category (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>). The total number of sequences decreased as the quality increased (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, C and D</a>). When aligned to the <em>P. lambertiana</em> draft genome reference, the four sets followed the same trend. The best results were obtained after transcript selection and for the polished sequences (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). In summary, ICE/Quiver polishing after isoform level clustering resulted in a drastic reduction in the number of final clustered and filtered sequences [<em>e.g.</em>, only 406 (2%) sequences were retrieved], but with significantly better performance in terms of quality (length, transcript completeness, and mapping rates).</p></div><div id="sec-15" class="subsection"><h4>Transcriptome coverage and diversity:</h4><p id="p-27">The 17,505 unique embryo transcripts generated from the combined HiSeq, MiSeq, and PacBio <em>de novo</em> assembled transcriptome were mapped against the <em>P. lambertiana</em> genome at 100% coverage and 90% identity. In total, 3846 transcripts did not map, 4410 mapped in more than two locations, and 9249 were single mapping units (SMUs) (one location in the genome). These SMUs were exclusively considered for downstream analysis. The complete set of transcripts for the embryo libraries (76,302 before clustering) were aligned to the genome with the same parameters, and those that overlapped with SMUs were selected. Of the 9249 SMUs, 4504 (49%), 3877 (42%), and 6883 (74%) were covered by HiSeq, MiSeq, or PacBio transcripts, respectively. These results revealed improved coverage by PacBio.</p><p id="p-28">A total of 1615 SMUs covered by all three technologies were evaluated on four different metrics. Examination of the longest splice variants revealed 1325 SMUs by HiSeq, 1191 by MiSeq, and 491 by PacBio (best result provided by one, two, or three technologies). Second, the number of SMUs where one single sequencing technology produced the longest splice variant was 251, 146, and 128 for HiSeq, MiSeq, and PacBio, respectively. Examination of transcript length distribution indicated that SMUs where PacBio was the best technology were shorter than their Illumina counterparts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS3.pdf">Figure S3</a>). Third, analysis of the contribution of each technology to the coverage of the SMU (where it was the longest transcript) was performed. For example, a single SMU with a HiSeq transcript of 1000 nt, a MiSeq of 600 nt, and a PacBio of 250 nt demonstrates that the HiSeq transcript improves the coverage by 400 nt relative to the MiSeq transcript, and 750 nt relative to the PacBio transcript. It is worth noting that the most significant improvements were observed for HiSeq and MiSeq transcripts relative to PacBio (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, lanes 2 and 4). Finally, the number of nonredundant splice variants was evaluated for each technology, for each SMU. Distribution across SMUs was improved in those transcripts originating from PacBio assemblies (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). For example, a set of 155 SMUs was covered by more than 30 variants as assembled with PacBio reads. On average (after the removal of outliers), the total number of splice variants per SMU was 1.6, 1.5, and 3.7 for HiSeq, MiSeq, and PacBio, respectively. A total of 92,300 splice variants were identified and characterized by type based on alignments to the reference genome. Overwhelmingly, length variants (alternative start or premature stop) were the most abundant, and intron retention was more abundant than exon skipping (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS3.xlsx">Table S3</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Contribution of each technology to improve the coverage of the single mapping units (SMU) when it performed best. (B) Example of splicing variants identified and mapped on the same SMU (P. lambertiana transcript annotated as &#x201C;embryo defective 2410 isoform protein&#x201D; via enTap) for each technology. In this example, HiSeq performed as the best technology providing the longest splicing variant. Longest splicing variant of the other two technologies was selected to calculate coverage improvement (as the sum of exon sequence lengths, blue dashed lines, Impr-1 = PacBio, Impr-2 = MiSeq) of HiSeq technology over them [lanes 1 and 2 from (A)]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Contribution of each technology to improve the coverage of the single mapping units (SMU) when it performed best. (B) Example of splicing variants identified and mapped on the same SMU (&lt;em&gt;P. lambertiana&lt;/em&gt; transcript annotated as &#x201C;embryo defective 2410 isoform protein&#x201D; via enTap) for each technology. In this example, HiSeq performed as the best technology providing the longest splicing variant. Longest splicing variant of the other two technologies was selected to calculate coverage improvement (as the sum of exon sequence lengths, blue dashed lines, Impr-1 = PacBio, Impr-2 = MiSeq) of HiSeq technology over them [lanes 1 and 2 from (A)].&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F5.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/12/3787/F5.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469747" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">(A) Contribution of each technology to improve the coverage of the single mapping units (SMU) when it performed best. (B) Example of splicing variants identified and mapped on the same SMU (<em>P. lambertiana</em> transcript annotated as “embryo defective 2410 isoform protein” via enTap) for each technology. In this example, HiSeq performed as the best technology providing the longest splicing variant. Longest splicing variant of the other two technologies was selected to calculate coverage improvement (as the sum of exon sequence lengths, blue dashed lines, Impr-1 = PacBio, Impr-2 = MiSeq) of HiSeq technology over them [lanes 1 and 2 from (A)].</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of splice variants generated across the three technologies (HiSeq, MiSeq and PacBio) in embryo." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of splice variants generated across the three technologies (HiSeq, MiSeq and PacBio) in embryo.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F6.medium.gif" width="393" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/12/3787/F6.medium.gif" width="393" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-30" class="first-child">Number of splice variants generated across the three technologies (HiSeq, MiSeq and PacBio) in embryo.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Libraries from female cone tissue, sequenced with both PacBio and HiSeq, were used to evaluate transcriptome diversity, similar to the embryo libraries, to assess if the 3–6 kb size selected libraries can improve transcript length results for PacBio (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS5.pdf">Figure S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FileS1.pdf">File S1</a>). This analysis consisted of evaluating all size selected libraries and just the longest 3–6 kb library. Similar conclusions were reached in this analysis. PacBio libraries performed better in terms of coverage and splice variant detection, while Illumina libraries were advantageous for transcript length, longest splice variant, and contribution to improve the length of the SMU.</p><p id="p-32">Transcriptome completeness was also analyzed with BUSCO for all three tissues used for sequencing technology review and evaluated in terms of technology. Lower completeness and higher variation (from 10 to 30%) between samples was achieved for PacBio libraries and better performance (up to 40%) for Illumina transcripts (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcriptome completeness analysis by BUSCO. Transcript sequences were compared to the plant set of single-copy conserved orthologs via BUSCO to estimate the percentage of completeness. Results are shown for sample embryo, 2 cm female cones, and female cones at time of pollination (lanes 1&#x2013;7), corresponding all of them to the samples used in the sequencing technology comparison. The final column reflects the complete P. lambertiana transcriptome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transcriptome completeness analysis by BUSCO. Transcript sequences were compared to the plant set of single-copy conserved orthologs via BUSCO to estimate the percentage of completeness. Results are shown for sample embryo, 2 cm female cones, and female cones at time of pollination (lanes 1&#x2013;7), corresponding all of them to the samples used in the sequencing technology comparison. The final column reflects the complete &lt;em&gt;P. lambertiana&lt;/em&gt; transcriptome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F7.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/12/3787/F7.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-33" class="first-child">Transcriptome completeness analysis by BUSCO. Transcript sequences were compared to the plant set of single-copy conserved orthologs via BUSCO to estimate the percentage of completeness. Results are shown for sample embryo, 2 cm female cones, and female cones at time of pollination (lanes 1–7), corresponding all of them to the samples used in the sequencing technology comparison. The final column reflects the complete <em>P. lambertiana</em> transcriptome.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Overall comparison:</h4><p id="p-34">The low cost per base and error rate of the Illumina platforms drives the continued market preference. Despite the lower throughput and high error rate, PacBio Iso-Seq libraries were highly productive in terms of number of high quality transcripts. For example, 7892, 8782, and 8940 complete high quality CDS were identified in embryo samples from 29 million MiSeq reads, 230 million HiSeq, and 362K PacBio reads. PacBio yielded shorter assembled transcripts and sequence length was much improved on the Illumina platforms. This is in contradiction to the work of <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Xu <em>et al.</em> (2015)</a> in <em>Salvia miltiorrhiza</em>, which reported longer PacBio transcripts compared to Illumina, although this comparison was performed prior to CDS selection. On the contrary, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Dong <em>et al.</em> (2015)</a> carried out a comparison in <em>Triticum aestivum</em> to determine length improvement of high quality (based on mapping rates) PacBio transcripts over previously annotated wheat gene models, and found minimal (45 nt on average) improvement. In our study, slightly better performance in sequence length was observed for HiSeq relative to MiSeq, and almost no difference in other metrics. Coverage of MiSeq libraries was lower than for HiSeq (only 7%), likely due to unusual HiSeq depth employed in this study (one lane per sample). MiSeq performed better than HiSeq in transcriptome completeness in the embryo sample as evaluated by BUSCO. This may suggest that the longer read length (300 nt PE) produced more representative sequences. PacBio produced the greatest number of splice variants, which is valuable given their role in regulating many biological processes in plant systems as well as the inability to accurately assess these in nonmodel systems. Recent studies in animal systems have benefited from long read technology for isoform detection (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Thomas <em>et al.</em> 2014</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Treutlein <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Au <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sharon <em>et al.</em> 2013</a>), while in plants (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Xu <em>et al.</em> 2015</a>), more efficient splice junction detection has been shown in technology comparisons (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2014b</a>). The promise of moving away from <em>de novo</em> transcriptome assembly of short reads and relying on third generation technologies has been proposed (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Martin and Wang 2011</a>). In our analysis, transcript yield of PacBio reached similar levels to Illumina, but transcript completeness was improved for the latter, suggesting the technology is not mature enough to replace the benefits of deep sequencing with short reads. A similar conclusion can be reached when comparing prices of the three technologies. MiSeq was 3 times more expensive than HiSeq (the least expensive), and PacBio 66 times. However, if we consider price per final transcript obtained instead of price per read, the difference is reduced and prices become much more similar. Accounting for all aspects, including price, technological, and biological concerns, a combination of both technologies is ideal for comprehensive and accurate transcriptome profiling.</p></div></div><div id="sec-17" class="subsection"><h3>Transcriptome characterization</h3><p id="p-35">Among the 33,113 unique high quality full-length transcripts, 30,809 were functionally annotated with a protein from publicly available sequence databases. A total of 26,568 had a descriptive functional annotation (informative), 3923 were uninformative (annotated as hypothetical, predicted, or otherwise noncharacterized proteins), and 1399 were strongly associated with fungal, insect, or bacterial sequences and removed from subsequent analysis. A total of 1243 remained unannotated, representing artifacts or potential novel conifer-specific proteins. In spite of not being annotated, at least one protein domain was identified in all (as required during selection of the CDS). Of these 1243, 351 contained a DUF-like domain (domain of unknown function), the most abundant occurrence labeled as DUF4283 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). A total of 189 transcripts contained a domain similar to cellulose synthases (PF03552). Proteins associated with cellulose metabolism were also identified in the gene family analysis as specific to <em>P. lambertiana</em>. Additionally, 94 X-box-related transcription factors were identified, which is expected due to the high specificity of these proteins for binding DNA (likely specific to <em>P. lambertiana</em>). When aligning the complete set of transcripts to characterized proteins, <em>A. thaliana</em> and <em>V. vinifera</em> dominated the annotations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS6.pdf">Figure S6</a>). The transcriptome was evaluated for completeness with BUSCO and over 78% of the 950 unique orthologs conserved across land plants were identified.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469834/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469834/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Most abundant protein domains identified in nonannotated <em>P. lambertiana</em> transcripts</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Expression profiles from several distinct tissue libraries were compared and unique transcripts were estimated. It is worth noting that the majority of unique sequences were expressed in female reproductive tissue (samples S, V, and M together, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS7.pdf">Figure S7A</a>). Also, few unique transcripts were identified in basket stage tissues (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS7.pdf">Figure S7A</a>) when compared to the other vegetative tissues, as expected, since this is a pool of cotyledons, stems, and roots. When the three vegetative tissues were compared to reproductive tissues, basket, and embryo, larger differences were observed for reproductive tissue (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS7.pdf">Figure S7B</a>). Since this deep sequencing represents a single individual, transcripts that clustered with sequences from the same library were considered to be library-specific gene products (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS8.pdf">Figure S8A</a>). This produced a range from 199 transcripts (basket) to 3482 transcripts (female cone at the time of pollination, sample V, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS1.xlsx">Table S1</a>). Interestingly, the female cones (2 wk before pollination) (sample M, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS1.xlsx">Table S1</a>), had a similar number of unique sequences to other vegetative tissues, when compared with other female cones samples (V and S, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS1.xlsx">Table S1</a>). The latter two were in a more developed stage of differentiated cone tissue. In total, 14,718 transcripts were shared by different libraries.</p><p id="p-38">The lack of replicates in this study hampers confident identification of differentially expressed genes. However, preliminary evaluation of this can contribute to tissue characterization and provide insights into the biological processes underlying the individuals sampled. Treated samples have been compared to their respective untreated control (see <em>Materials and Methods</em>), and reproductive tissue has been compared to the basket stage seedling sample, as a mix of vegetative (needle, root, and stem) tissue. Number of reads mapped on each transcript was used as an estimate of RNA accumulation. Expression profiles of all transcripts (in each library) were used for a PCA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS9.pdf">Figure S9</a>), where samples corresponding to reproductive tissue grouped on the left half of the plot, and vegetative tissue samples on the right, with the notable exception of Basket samples. This is likely a result of the combined tissues at the early “basket” stage of development. PCA results confirmed that Basket samples were the least informative considering both transcript uniqueness and transcript accumulation. Female cones at time of pollination (V samples) represented the greatest transcript richness (uniqueness, see above) and also distinctive RNA accumulation profiles. Stem (red circle) and root (green circle) samples clustered together, showing smaller transcriptomic changes after treatments (NaCl, wounding, or jasmonate) than those occurred consequence of developmental processes. Interestingly, there is a parallel PCA component of the same sense from healthy to treated tissue for both stem and root samples. On average, 5958 transcripts were identified as differentially expressed in each sample with a fold change &gt; 2.0, and shared genes among the samples compared. Following the same trend, jasmonate-treated and injured tissue shared more differentially expressed transcripts than NaCl-treated samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS10.pdf">Figure S10</a>). The role of jasmonate in both pathogen defense and wounding response might explain this observation. Embryonic tissue shared only a few differentially expressed genes with the three vegetative tissues, and more similarities were found between embryo and reproductive tissues as expected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS10.pdf">Figure S10</a>). Enriched GO terms identified in the differential expression comparisons, included: defense response (jasmonate-treated samples), response to stress and cell wall modification (tissue after wounding), ATPase and osmosensor activities (NaCl-treated samples), and regulation of developmental processes (reproductive tissue) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS4.xlsx">Table S4</a>). Despite experimental limitations, the identified differentially expressed genes were consistent with the underlying biology of the tissues and treatments (detailed analysis in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FileS2.pdf">File S2</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469856/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469856/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of GO terms overrepresented in differentially expressed <em>P. lambertiana</em> genes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Gene family analysis</h3><p id="p-40">A total of 51,475 families out of 13 species were retrieved from the gene family analysis implemented in TRIBE-MCL. Of these, 9844 contained at least five protein members after filtering for retroelements. A total of 731 were composed of proteins from a single species and 9113 from two or more species (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). Among conifers, the largest number of species-specific families was observed in <em>P. lambertiana</em> and the fewest in <em>P. glauca</em>, likely influenced by the varying transcript resources available for each species. A large number of proteins were shared by all species (11,349). Conservation among protein families was also compared across species grouped in four categories (bryophyte, gymnosperm, monocot, and dicot, <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). The largest number of shared families were those present in all four groups (4317). Both early land plants and gymnosperms shared more families with dicots than with monocots. Only 222 families were found unique in conifers: 4 unique to the genus <em>Picea</em>, 36 unique to <em>Pinus</em>, and 28 unique to <em>P. lambertiana</em>. Conifer and <em>P. lambertiana</em>-specific families and protein annotations are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS5.xlsx">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS6.xlsx">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS7.xlsx">Table S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS8.xlsx">Table S8</a>. The largest family (74 proteins with 12 from <em>P. lambertiana</em>) was composed of transferases and uncharacterized proteins, revealing potential novel proteins. An abundant family composed of <em>mTERF</em> transcription factors (3 families comprising 87 proteins, 25 from <em>P. lambertiana</em>) play important roles in plant growth, development, and abiotic stress tolerance, based on characterization in <em>Arabidopsis</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kleine 2012</a>). Little is known about the molecular mechanisms of <em>mTERF</em> that control transcription of the mitochondrial and chloroplastic genomes, but the high content and the presence in the conifer-specific set suggest specific roles in gymnosperms. WRKY transcription factors were also abundant (4 families, 68 proteins, 21 from <em>P. lambertiana</em>), known as key regulators of many processes, including responses to biotic and abiotic stresses, senescence, seed dormancy, seed germination, and plant responses to pathogens (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rushton <em>et al.</em> 2010</a>). F-box proteins known to be subunits of the E3 ubiquitin ligase aggregations named as the SCF quaternary complex (SKP1, Cullin1, F-box protein, and Rbx1, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Zheng <em>et al.</em> 2002</a>) were also identified as one of the most abundant families specific to conifers (8 families in total, 105 proteins, 35 from <em>P. lambertiana)</em>. In the <em>P. lambertiana</em>-specific set, two families containing proteins related to cellulose metabolism attracted attention, due to the potential connection to basal biology of a woody species. Among families shared by other species but potentially expanded in conifers, were two comprised of ATP binding proteins with large number of isoforms (787 members in <em>P. lambertiana</em>, and 390, 522, and 98 in <em>P. abies</em>, <em>P. taeda</em>, and <em>P. glauca</em>, respectively).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the TRIBE-MCL analysis to identify orthologous proteins and gene families. (A) Number of species-specific proteins and families (bold). (B) number of protein families shared by different species grouped in main classes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the TRIBE-MCL analysis to identify orthologous proteins and gene families. (A) Number of species-specific proteins and families (bold). (B) number of protein families shared by different species grouped in main classes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F8.medium.gif" width="440" height="165"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/12/3787/F8.medium.gif" width="440" height="165"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469789" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-41" class="first-child">Results of the TRIBE-MCL analysis to identify orthologous proteins and gene families. (A) Number of species-specific proteins and families (bold). (B) number of protein families shared by different species grouped in main classes.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-19" class="subsection"><h4>Characterization of the Dicer protein family:</h4><p id="p-42">Conifers have a distinguishing feature with regard to gene silencing and small RNA (sRNA) biogenesis in their unique 24-nt sRNA profile, which are associated with epigenetic processes and control of repetitive element proliferation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Matzke and Mosher 2014</a>). The peculiar sRNA profile and large genomes with TE content reaching 80% raises questions about the involvement of the sRNA machinery in conifer genome expansion. Key components of this pathway include specialized members of RNA-dependent DNA polymerase, RNA-dependent RNA polymerase, Argonaute, and DCL proteins (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2015</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Matzke and Mosher 2014</a>), the latter involved in the biogenesis of sRNAs. In addition to plant development and abiotic stress, a link between DCLs and plant pathogen response exists, at least for viruses and bacteria (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Matzke and Mosher 2014</a>). There are four different DCL proteins characterized in <em>Arabidopsis</em>. DCL3 is primarily responsible for the epigenetic pathway. This number varies in other plants such as poplar and rice (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Margis <em>et al.</em> 2006</a>). In spite of initial reports, it is generally accepted that the 24-nt-DCL3 pathway exists in conifers, but with spatial and/or temporal peculiarities. Transcriptomic studies in pine and larch have noted that 24-nt sRNAs are restricted mainly to reproductive tissues and are decreased or even absent in vegetative tissues (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Niu <em>et al.</em> 2015</a>). 21-nt sRNAs are associated with repetitive content in the Norway spruce genome (<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Nystedt <em>et al.</em> 2013</a>) and conifer-specific DCL1 variants have been described (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dolgosheina <em>et al.</em> 2008</a>).</p></div><div id="sec-20" class="subsection"><h4>Canonical plant DCLs shared by conifers:</h4><p id="p-43">In the <em>P. lambertiana</em> transcriptome, 12 transcripts were identified with sequence similarity and domain topology matching DCL features. Among these, six were supported by gene models in the draft genome sequence (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Stevens <em>et al.</em>, 2016</a>). These sequences were combined with plant DCL proteins to perform a phylogenetic analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS11.pdf">Figure S11</a>), including four conifers (<em>P. taeda</em>, <em>P. abies</em>, <em>P. glauca</em>, and <em>P. tabuliformis</em>), a monocot (<em>O. sativa</em>), a dicot (<em>A. thaliana</em>), <em>Amborella trichopoda</em> because of its phylogenetic position near the base of the flowering plants lineage, and <em>P. patens</em> (Bryophyta) and <em>Selaginella moellendorffii</em> (Lycopodiophyta) as model organisms of ancient land plants. The last two species have an additional interest because 24-nt small RNAs have been sequenced in <em>P. patens</em>, demonstrating the basal origin of the pathway, but they are weakly expressed compared to 21-nt sRNAs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Banks <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Coruh <em>et al.</em> 2014</a>). The proportion of 23–24-nt sRNAs relative to the 21-nt class is also reduced in the sporophyte of <em>S. moellendorffii</em>, where their expression is mostly limited to the gametophyte (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Banks <em>et al.</em> 2011</a>). It is worth noting that <em>S. moellendorffii</em>, in spite of a similar genome size and organization to <em>Arabidopsis</em>, has an increased repeat content and abundant LTR retrotransposons (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Banks <em>et al.</em> 2011</a>).</p><p id="p-44">In the phylogenetic analysis, all conifers and the selected plant sequences grouped according to the four main classes of DCLs described to date (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS11.pdf">Figure S11</a>). Two <em>P. lambertiana</em> sequences represented by two nonoverlapping gene models clustered with DCL3 proteins from other species, providing further evidence of its presence in gymnosperms. <em>P. patens</em> and <em>S. moellendorffii</em> have been reported to have no members for DCL2 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Axtell <em>et al.</em> 2007</a>). Accordingly, we did not identify this DCL in these species. No DCL2 counterpart for <em>P. lambertiana</em> was identified, but it was found in sequences from <em>P. abies</em>. DCL2 orthologs from <em>P. tabuliformis</em> have also been reported, indicating that all four DCLs are present in most conifers. The absence of DCL2 in <em>P. lambertiana</em> might be due to misrepresentation in the transcriptome, although other studies have failed to find DCL2 in specific species of the gymnosperm order Gnetales (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ma <em>et al.</em> 2015</a>). Investigating the needle transcriptomes of other white pines of which <em>P. lambertiana</em> is a member, <em>P. albicaulis</em> pine contained a high quality version of DCL2, while <em>P. flexilis</em> and <em>P. monticola</em> did not.</p></div><div id="sec-21" class="subsection"><h4>Conifer-specific set of DCL1 proteins:</h4><p id="p-45">The DCL1 sequences split into two independent clusters, one grouping contained the canonical DCL1 protein from <em>Arabidopsis</em> and other plants, while the other encompassed some <em>P. lambertiana</em> transcripts and the conifer members identified as potentially specific by <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Dolgosheina <em>et al.</em> (2008)</a>. This also included some new sequences originating from the <em>P. glauca</em> and <em>P. abies</em> genome projects. All DCL1 sequences were further explored for protein domain architecture (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). Most of the non-conifer specific sequences had a canonical DCL1 architecture (two helicase, one Dicer, one PAZ, two RNAseIII, and two ds-RNA binding domains, from N- to C-terminus). <em>P. lambertiana</em> and <em>P. taeda</em> DCL1s were complete, as well as those from <em>P. tabuliformis</em>, the remaining angiosperms, <em>P. patens</em>, and <em>S. moellendorffii</em>. The <em>Picea</em> DCL1s were not complete. For <em>P. abies</em>, one locus (MA_523069g0010, <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>) was located in a small scaffold. The two additional sequences (MA_10437243g0010 and MA_10437243g0020, <a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9</a>) corresponded to complementary DCL1 parts located in two consecutive gene models on the same scaffold, which is likely a fragmented gene model. For <em>P. glauca</em>, no additional models within the range of the one identified were found. Previously identified conifer-specific DCL1s, as well as the remaining conifer sequences used in this study, were represented by a portion of a complete DCL1 sequence (1–3 domains). They lacked the N-terminus and the PAZ domain, but had conserved RNAseIII and dsRNA-binding domains (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9</a>). This result may be due to incorrect gene models, or might represent a unique function. The prevalence of pseudogenes and the fragmented genome assemblies in conifers complicates the determination of whether these conifer-specific sequences are artifacts derived from functional DCLs. For example, the gene models corresponding to the three short DCL1 transcripts with genome representation were surrounded by abundant TEs, which can be indicative of pseudogenes. However, the high quality transcripts represented by high quality gene models (<em>e.g.</em>, DCL3) were flanked in a similar manner. It is worth noting that PacBio data were not particularly informative for the identification of these sequence variants.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein domain topology of DCL1 proteins from P. lambertiana and several plant species, including three conifers [P. taeda (Ptaeda), P. abies (Pabies), P. glauca (Pglauca), and P. tabuliformis (Ptabuliformis)], a monocot [O. sativa (Osativa)], a dicot [A. thaliana (Athaliana)], A. trichopoda (Atrichopoda), P. patens (Ppatens), and S. moellendorffii (Smoellendorffii). DCL1, Dicer-like 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Protein domain topology of DCL1 proteins from &lt;em&gt;P. lambertiana&lt;/em&gt; and several plant species, including three conifers [&lt;em&gt;P. taeda&lt;/em&gt; (Ptaeda), &lt;em&gt;P. abies&lt;/em&gt; (Pabies), &lt;em&gt;P. glauca&lt;/em&gt; (Pglauca), and &lt;em&gt;P. tabuliformis&lt;/em&gt; (Ptabuliformis)], a monocot [&lt;em&gt;O. sativa&lt;/em&gt; (Osativa)], a dicot [&lt;em&gt;A. thaliana&lt;/em&gt; (Athaliana)], &lt;em&gt;A. trichopoda&lt;/em&gt; (&lt;em&gt;Atrichopoda&lt;/em&gt;), &lt;em&gt;P. patens&lt;/em&gt; (Ppatens), and &lt;em&gt;S. moellendorffii&lt;/em&gt; (Smoellendorffii). DCL1, Dicer-like 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F9.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/6/12/3787/F9.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469817" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-46" class="first-child">Protein domain topology of DCL1 proteins from <em>P. lambertiana</em> and several plant species, including three conifers [<em>P. taeda</em> (Ptaeda), <em>P. abies</em> (Pabies), <em>P. glauca</em> (Pglauca), and <em>P. tabuliformis</em> (Ptabuliformis)], a monocot [<em>O. sativa</em> (Osativa)], a dicot [<em>A. thaliana</em> (Athaliana)], <em>A. trichopoda</em> (<em>Atrichopoda</em>), <em>P. patens</em> (Ppatens), and <em>S. moellendorffii</em> (Smoellendorffii). DCL1, Dicer-like 1.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>DCL1 protein variants in ancient plants:</h4><p id="p-47">Sequences from <em>S. moellendorffii</em> did not group with the DCL1 conifer-specific set, providing no evidence of shared genetic elements at this level with conifers. However, a protein sequence of similar features from <em>P. patens</em> was identified and clustered out of both DCL1 clades (conventional and conifer-specific, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS11.pdf">Figure S11</a> and <a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9</a>), suggesting a potential common origin for all species for these shortened DCL-like sequences. This sequence corresponds to a short variant of DCL1 recently characterized in <em>P. patens</em> and identified as MINIMAL DICER-LIKE (mDCL) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Coruh <em>et al.</em> 2015</a>). This gene lacks the N-terminal helicase domain of DCL proteins, and has only PAZ and RNAseIII domains. The mDCL is specifically required for 23-nt siRNA accumulation associated with genomic repetitive elements. Mutant analysis showed a dependence of this protein on DCL3 for generating the complete set of siRNAs (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Coruh <em>et al.</em> 2015</a>). Phylogenetic resolution of this protein remained unclear, although it clustered with DCL1 sequences in spite of its association with siRNAs. In this study, it also clustered along with DCL1-like sequences from other species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS11.pdf">Figure S11</a>). We were able to identify a truncated version of <em>S. moellendorffii</em> DCL with only a RNAseIII domain and with sequence similarity to DCL1s that clustered alongside mDCL1, both basal to the overall DCL1 lineage. It has been suggested that shortened versions of DCLs might arise frequently during evolution (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Coruh <em>et al.</em> 2015</a>). For example, truncated versions of DCLs, which lack the N-terminal helicases and the PAZ domain (similarly to those identified in conifers), also have been described as functional in other non-plant organisms (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Malone <em>et al.</em> 2005</a>). The link between the shortened proteins and the conifer-specific set remains elusive, but these data suggest that an ancient mDCL from <em>P. patens</em> could have evolved through lycophytes and gymnosperms and not through angiosperms. Cloning and experimental characterization of the truncated conifer-specific DCL1 proteins is needed to determine if they are functional, but experimental data reported on mDCL in <em>P. patens</em> and other species supports the idea that complete domain topology of canonical DCL1 is not a requirement.</p></div><div id="sec-23" class="subsection"><h4>Expression analysis of DCL transcripts:</h4><p id="p-48">Expression analysis indicated tissue specificity for both canonical and conifer-specific DCLs. The transcript potentially coding for conventional DCL1 was ubiquitously expressed across all samples analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS12.pdf">Figure S12A</a>). A similar profile was observed for one transcript coding for DCL4. The other two DCL4s were practically not expressed in any tissue, but were observed initially in cone samples. DCL3, which is involved in 24-nt sRNA biogenesis, was represented by three <em>P. lambertiana</em> transcripts, primarily expressed in reproductive tissues: one transcript slightly expressed in embryo, one in early female cones, and the third in pollen and highly overexpressed in embryo (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS12.pdf">Figure S12A</a>). Conifer-specific DCL1 transcripts had a mix of profiles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS12.pdf">Figure S12B</a>). One was virtually not expressed, the other ubiquitously expressed, and the last had a differential profile among reproductive tissues. The most interesting profile was transcript BRS/miseq/c28277_g1_i4|m.43092, which was highly overexpressed in embryo with a similar profile to Basket/c18190_g1_i2|m.24310 (conventional DCL3-like protein). Experimental validation of the DCL3 protein and this truncated variant of DCL1 is necessary to confirm functional association with a similar mechanism reported for <em>P. patens</em>.</p></div></div><div id="sec-24" class="subsection"><h3>miRNA precursor identification</h3><p id="p-49">In total, 185 potential miRNA precursors were identified. None of these had an exact match to sequences deposited in MirBase as all contained one or two mismatches. In examining the size distribution of the mature predicted mRNAs, only one 24-nt sequence (0.5%) was identified (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10C</a>). A low frequency of 24-nt small RNAs (involved in transposon control in angiosperms) has been reported in gymnosperms (<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Niu <em>et al.</em> 2015</a>). The huge genome of <em>P. lambertiana</em> is primarily composed of TEs and the observation here suggests additional support for the hypothesis. The lack of targeted small RNA sequencing data in this study hampers validation of identified mature miRNA sequences. To accommodate this, we considered only those that contained a mature miRNA with sequence similarity to those most conserved among plants (49 precursors). Of these, 19 aligned to the core conserved plant miRNAs (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10A</a>) and 30 specifically to other conifers (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10B</a>). In addition, the RNA secondary structures have been manually reviewed to select precursors satisfying miRNA structural requirements (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS13.pdf">Figure S13A</a>). Long precursors with strongly negative MFEI indexes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS13.pdf">Figure S13B</a>) were flagged as low quality as they resemble the structure of fold-back retrotransposons. Finally, multiple miRNA predictions on the same transcript corresponding to both strands of the miRNA duplex were collapsed, yielding a total of 37 and 9 high and low quality precursors, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS9.xlsx">Table S9</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Computational prediction of mature microRNA (miRNA) sequences from precursors identified in P. lambertiana transcripts corresponding to plant-conserved (A) and nonconserved conifer-related (B) miRNA families. (C) Length distribution of identified mature miRNAs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1556057278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Computational prediction of mature microRNA (miRNA) sequences from precursors identified in &lt;em&gt;P. lambertiana&lt;/em&gt; transcripts corresponding to plant-conserved (A) and nonconserved conifer-related (B) miRNA families. (C) Length distribution of identified mature miRNAs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3787/F10.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/6/12/3787/F10.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3787/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469918" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-50" class="first-child">Computational prediction of mature microRNA (miRNA) sequences from precursors identified in <em>P. lambertiana</em> transcripts corresponding to plant-conserved (A) and nonconserved conifer-related (B) miRNA families. (C) Length distribution of identified mature miRNAs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Precursor lengths ranged from 60 to 307 nt (125 nt in average), while source transcripts ranged from 246 to 2880 nt (1008 nt in average). Twenty-seven precursors successfully mapped to the <em>P. lambertiana</em> genome (Stevens <em>et al.</em>, unpublished results). Two sets of precursors (PILAmiRNA_026 and PILAmiRNA_007, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/TableS9.xlsx">Table S9</a>) were located in the same scaffold, which were further explored for potential miRNA clusters codified in polycistronic transcripts. Precursors contained on the same transcript provide information about coexpressed miRNAs in the same family or even different families. PILAmiRNA_007 corresponded to miR1313-like precursors, which were located 120 kb apart, so not further considered, but PILAmiRNA_026 corresponded to 2 miR1314 precursors placed only at 326 nt apart, suggesting a cluster (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1/FigureS13.pdf">Figure S13C</a>). However, the source transcript aligned only to the first precursor, questioning whether the transcript is complete, the second precursor is expressed independently, or the second <em>locus</em> represents a nonfunctional region. The mature miRNA contained a nucleotide variant at position 13 relative to the sequence predicted by Mirena on the second transcript-supported precursor, suggesting a mutation.</p><p id="p-52">The small number of precursors identified from the large transcriptome resource can be attributed to the short life span of primary miRNAs (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Song <em>et al.</em> 2007</a>). Sequencing data were used to estimate precursor accumulation, and differences in the level of expression were observed among miRNA families. The most abundant were miR156 and miR172 (seen in all samples and primarily in reproductive tissue). These miRNAs are conserved across nearly every plant species (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Montes <em>et al.</em> 2014</a>). One and two precursors were sequenced for miR156 and miR172, respectively. In contrast, the nonconserved miR950 showed moderate accumulation, mostly in stem samples, but 13 precursor variants were sequenced. The contrasting different ratios between level of expression and number of precursors detected in these three miRNA types serve as an example that different processing rates for different miRNA families might occur. miR950 has been characterized in <em>P abies</em>, <em>P taeda</em>, and <em>Pinus densata</em>, but is absent in the remaining plant species in miRBase, suggesting conifer-specificity. It has, however, been reported in flower buds and fruits in <em>Citrus sinensis</em> (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Song <em>et al.</em> 2012</a>). It has been suggested that its primary targets are NB-LRR genes, potentially as a source of phased secondary small interfering RNAs (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Xia <em>et al.</em> 2015</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zhai <em>et al.</em> 2011</a>). The lack of conservation across plants and the high number of precursor variants detected here may indicate an important role in conifers, and unique processing rates for this miRNA.</p></div><div id="sec-25" class="subsection"><h3>Conclusions</h3><p id="p-53">This study characterizes the transcriptome of <em>P. lambertiana</em>, expanding the scarce genomic resources available for the subgenus Strobus. Due to inherent technical challenges in conifer genome assemblies, these resources are essential to provide insight on the complete gene space. Among the prevalent pseudogenes and TEs, annotation of true gene models is hindered without transcriptomic evidence. With this resource, we also provide the first computational identification of miRNAs in <em>P. lambertiana</em>, and, related to gene silencing, undertake an exploration and comparative analysis of DCL and DCL-like proteins. This is an outstanding question in gymnosperm biology, since several conifer-specific DCL variants are under investigation. Expression analysis derived from sequencing data further supports a biological role of these variants. The results presented here highlight the peculiarities of this pathway in conifers and identifies similarities with ancient land plants. From a technical perspective, we have used PacBio’s Iso-Seq long read strategy for the first time in a conifer to improve the accuracy of transcript construction. The detailed short and long read technology comparison provides perspective and recommendations for those generating transcriptomic resources in nonmodel species.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We thank Pacific Biosciences for their support for library construction and Jeffrey Puryear (Texas A&amp;M University) for sample preparation and RNA extraction. This work was supported by United States Department of Agriculture/National Institute of Food and Agriculture (USDA/NIFA #2011-67009-30030) within the framework of the PineRefSeq project (<a href="http://pinegenome.org/pinerefseq/">http://pinegenome.org/pinerefseq/</a>).</p><p id="p-55"><em>Note added in proof:</em> See <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">K. A. Stevens <em>et al.</em> 2016</a> (pp. <a href="http://www.genetics.org/content/204/4/1613">1613–1626</a>) in <em>Genetics</em> for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032805/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-57"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-24"><span class="received-label">Received </span>June 24, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-07-13"><span class="accepted-label">Accepted </span>July 13, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Gonzalez-Ibeas <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-6.12.3787.2"><div class="cit-metadata unstructured">Andrews, S., and Q. C. Fast, A quality control tool for high throughput sequence data. Available at: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>. Accessed: July 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.3" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.4" data-doi="10.1073/pnas.1320101110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sebastiano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Afshar</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durruthy</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Char-acterization of the human ESC transcriptome by hybrid sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E4821</span>–<span class="cit-lpage">E4830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1320101110%26rft_id%253Dinfo%253Apmid%252F24282307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzUwL0U0ODIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.5" data-doi="10.1105/tpc.107.051706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Common functions for diverse small RNAs of land plants.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1750</span>–<span class="cit-lpage">1769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.051706%26rft_id%253Dinfo%253Apmid%252F17601824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTc1MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.6" data-doi="10.1126/science.1203810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishiyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasebe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gribskov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Selaginella genome identifies genetic changes associated with the evolution of vascular plants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203810%26rft_id%253Dinfo%253Apmid%252F21551031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAzMi85NjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.7" data-doi="10.1371/journal.pone.0011518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cahill</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Köser</span>  <span class="cit-name-given-names">C. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archer</span>  <span class="cit-name-given-names">J. A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Read length and repeat resolution: exploring prokaryote genomes using next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e11518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCahill%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De11518%26rft.epage%253De11518%26rft.atitle%253DRead%2Blength%2Band%2Brepeat%2Bresolution%253A%2Bexploring%2Bprokaryote%2Bgenomes%2Busing%2Bnext-generation%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011518%26rft_id%253Dinfo%253Apmid%252F20634954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20634954&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.8" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.9" data-doi="10.1186/s13059-016-0881-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madrigal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarazona</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Cabrero</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cervera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A survey of best practices for RNA-seq data analysis.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0881-8%26rft_id%253Dinfo%253Apmid%252F26813401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0881-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26813401&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.10" data-doi="10.1016/j.pbi.2014.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coruh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahid</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Seeing the forest for the trees: annotating small RNA producing genes in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D18%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2014.02.008%26rft_id%253Dinfo%253Apmid%252F24632306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2014.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24632306&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.11" data-doi="10.1105/tpc.15.00228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coruh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahid</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wierzbicki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comprehensive annotation of <em>Physcomitrella patens</em> small RNA loci reveals that the heterochromatic short interfering RNA pathway is largely conserved in land plants.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2148</span>–<span class="cit-lpage">2162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00228%26rft_id%253Dinfo%253Apmid%252F26209555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMjE0OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.12" data-doi="10.1105/tpc.110.082784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuperus</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahlgren</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution and functional diversification of MIRNA genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.082784%26rft_id%253Dinfo%253Apmid%252F21317375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvNDMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.13" data-doi="10.1261/rna.1052008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolgosheina</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aksay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahinalp</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magrini</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Conifers have a unique small RNA silencing signature.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1508</span>–<span class="cit-lpage">1515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1052008%26rft_id%253Dinfo%253Apmid%252F18566193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE0LzgvMTUwOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-molecule real-time transcript sequencing facilitates common wheat genome annotation and grain transcriptome research.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3787.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dongen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abreu-Goodger</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using MCL to extract clusters from networks</span>, pp. <span class="cit-fpage">281</span>–<span class="cit-lpage">295</span> in <span class="cit-source"><em>Bacterial Molecular Networks</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Helden</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Toussaint</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Thieffry</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.16" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.17" data-doi="10.1093/bioinformatics/btq461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Search and clustering orders of magnitude faster than BLAST.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2460</span>–<span class="cit-lpage">2461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTkvMjQ2MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.18" data-doi="10.1093/nar/30.7.1575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dongen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouzounis</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An efficient algorithm for large-scale detection of protein families.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1575</span>–<span class="cit-lpage">1584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEnright%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D1575%26rft.epage%253D1584%26rft.atitle%253DAn%2Befficient%2Balgorithm%2Bfor%2Blarge-scale%2Bdetection%2Bof%2Bprotein%2Bfamilies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.7.1575%26rft_id%253Dinfo%253Apmid%252F11917018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMwLzcvMTU3NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.19" data-doi="10.1093/bioinformatics/bts515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GFOLD: a generalized fold change for ranking differentially expressed genes from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2782</span>–<span class="cit-lpage">2788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts515%26rft_id%253Dinfo%253Apmid%252F22923299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjEvMjc4MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3787.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fites-Kaufman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rundel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephenson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weixelman</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source"><em>Montane and Subalpine Vegetation of the Sierra Nevada and Cascade Ranges</em></span>, pp. <span class="cit-fpage">456</span>–<span class="cit-lpage">501</span>. <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Terrestrial Vegetation of California</span>, <span class="cit-publ-loc">Berkeley, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.21" data-doi="10.1111/j.1755-0998.2011.03024.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glenn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Field guide to next-generation DNA sequencers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.03024.x%26rft_id%253Dinfo%253Apmid%252F21592312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.03024.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21592312&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.22" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>De novo</em> transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.23" data-doi="10.1093/molbev/msv060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsythe</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorantes-Acosta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ancient origin and recent innovations of RNA polymerase IV and V.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1788</span>–<span class="cit-lpage">1799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv060%26rft_id%253Dinfo%253Apmid%252F25767205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMyLzcvMTc4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.24" data-doi="10.1093/nar/gkr948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attwood</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">InterPro in 2011: new developments in the family and domain prediction database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D306</span>–<span class="cit-lpage">D312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr948%26rft_id%253Dinfo%253Apmid%252F22096229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMzA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-6.12.3787.25"><div class="cit-metadata unstructured">Joshi, N. A., and J. N. Fass, 2011 Sickle: a sliding-window, adaptive, quality-based trimming tool for FastQ files (Version 1.33) [Software]. Available at: <a href="https://github.com/najoshi/sickle.">https://github.com/najoshi/sickle.</a> Accessed: June 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.26" data-doi="10.1186/s13059-015-0702-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanitz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gypas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative assessment of methods for the computational inference of transcript isoform abundance from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0702-5%26rft_id%253Dinfo%253Apmid%252F26201343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0702-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26201343&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.27" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3787.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kinloch</span>  <span class="cit-name-given-names">B. B.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheuner</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title"><em>Pinus lambertiana</em> Dougl</span>, pp. <span class="cit-fpage">370</span>–<span class="cit-lpage">379</span> in <span class="cit-source"><em>Silvics of North America</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Honkala</span>  <span class="cit-name-given-names">B. H.</span></span>. <span class="cit-publ-name">USDA Forest Service. Agriculture Handbook</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kleine</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Arabidopsis thaliana</em> mTERF proteins: evolution and functional classification.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DKleine%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.spage%253D233%26rft.epage%253D233%26rft.atitle%253DArabidopsis%2Bthaliana%2BmTERF%2Bproteins%253A%2Bevolution%2Band%2Bfunctional%2Bclassification.%26rft_id%253Dinfo%253Apmid%252F23087700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23087700&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23087700&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.30" data-doi="10.1038/nmeth.2735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomics: the state of the art in RNA-seq analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1165%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2735%26rft_id%253Dinfo%253Apmid%252F24296473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296473&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327698100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.31" data-doi="10.1093/nar/gkt1181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozomara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">miRBase: annotating high confidence microRNAs using deep sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D68</span>–<span class="cit-lpage">D73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1181%26rft_id%253Dinfo%253Apmid%252F24275495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyL0QxL0Q2OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.32" data-doi="10.1006/anbo.2001.1521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winfield</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Nuclear DNA C-values complete familial representation in gymnosperms.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DLeitch%26rft.auinit1%253DI.%2BJ.%26rft.volume%253D88%26rft.issue%253D5%26rft.spage%253D843%26rft.epage%253D849%26rft.atitle%253DNuclear%2BDNA%2BC-values%2BComplete%2BFamilial%2BRepresentation%2Bin%2BGymnosperms%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbo.2001.1521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjg6Ijg4LzUvODQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.33" data-doi="10.1038/ng.2987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">567</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2987%26rft_id%253Dinfo%253Apmid%252F24836287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836287&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tighe</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolet</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grove</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Multi-platform assessment of transcriptome profiling using RNA-seq in the ABRF next-generation sequencing study.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1166</span> (erratum: Nat. Biotechnol. <span class="cit-vol">32</span>: 915).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D1166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.35" data-doi="10.1126/sciadv.1501084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baniaga</span>  <span class="cit-name-given-names">A.E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sessa</span>  <span class="cit-name-given-names">E.B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scascitelli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Early genome duplications in conifers and other seed plants.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e1501084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D1%26rft.issue%253D10%26rft.spage%253De1501084%26rft.epage%253De1501084%26rft.atitle%253DEarly%2Bgenome%2Bduplications%2Bin%2Bconifers%2Band%2Bother%2Bseed%2Bplants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1501084%26rft_id%253Dinfo%253Apmid%252F26702445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiMS8xMC9lMTUwMTA4NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.36" data-doi="10.1155/2012/251364"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison of next-generation sequencing systems.</span> <abbr class="cit-jnl-abbrev">J. Biomed. Biotechnol.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">251364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiomed%2BBiotechnol%26rft.aulast%253DLiu%26rft.auinit1%253DL.%26rft.volume%253D2012%26rft.spage%253D251364%26rft.epage%253D251364%26rft.atitle%253DComparison%2Bof%2Bnext-generation%2Bsequencing%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2012%252F251364%26rft_id%253Dinfo%253Apmid%252F22829749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2012/251364&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22829749&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.37" data-doi="10.1186/1748-7188-6-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhart</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siederdissen</span>  <span class="cit-name-given-names">C. H. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tafer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flamm</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">ViennaRNA Package 2.0.</span> <abbr class="cit-jnl-abbrev">Algorithms Mol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DLorenz%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DViennaRNA%2BPackage%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-26%26rft_id%253Dinfo%253Apmid%252F22115189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1748-7188-6-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22115189&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.38" data-doi="10.1186/1471-2164-12-264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alba</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bordeaux</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simões</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Mic-roarray analysis and scale-free gene networks identify candidate regulators in drought-stressed roots of loblolly pine (<em>P. taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLorenz%26rft.auinit1%253DW.%2BW.%26rft.volume%253D12%26rft.spage%253D264%26rft.epage%253D264%26rft.atitle%253DMicroarray%2Banalysis%2Band%2Bscale-free%2Bgene%2Bnetworks%2Bidentify%2Bcandidate%2Bregulators%2Bin%2Bdrought-stressed%2Broots%2Bof%2Bloblolly%2Bpine%2B%2528P.%2Btaeda%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-264%26rft_id%253Dinfo%253Apmid%252F21609476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21609476&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.39" data-doi="10.1093/gbe/evv171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatlen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becher</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Angiosperms are unique among land plant lineages in the occurrence of key genes in the RNA-directed DNA methylation (RdDM) pathway.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2648</span>–<span class="cit-lpage">2662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv171%26rft_id%253Dinfo%253Apmid%252F26338185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjcvOS8yNjQ4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3787.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">J. F. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptomics</span>, pp. <span class="cit-fpage">323</span>–<span class="cit-lpage">357</span> in <span class="cit-source"><em>Genetics</em>, <em>Genomics and Breeding of Conifers</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kole</span>  <span class="cit-name-given-names">C.</span></span>. <span class="cit-publ-name">Edenbridge Science Publishers &amp; CRC Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.41" data-doi="10.1128/MCB.25.20.9151-9164.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motl</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexer</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Germ line transcripts are processed by a Dicer-like protein that is essential for developmentally programmed genome rearrangements of <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">9151</span>–<span class="cit-lpage">9164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMalone%26rft.auinit1%253DC.%2BD.%26rft.volume%253D25%26rft.issue%253D20%26rft.spage%253D9151%26rft.epage%253D9164%26rft.atitle%253DGerm%2BLine%2BTranscripts%2BAre%2BProcessed%2Bby%2Ba%2BDicer-Like%2BProtein%2BThat%2BIs%2BEssential%2Bfor%2BDevelopmentally%2BProgrammed%2BGenome%2BRearrangements%2Bof%2BTetrahymena%2Bthermophila%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.20.9151-9164.2005%26rft_id%253Dinfo%253Apmid%252F16199890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8yMC85MTUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maloney</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogler</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population biology of sugar pine (<em>Pinus lambertiana</em> Dougl.) with reference to historical disturbances in the Lake Tahoe Basin: implications for restoration.</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">262</span>: <span class="cit-fpage">770</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D262%26rft.spage%253D770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.43" data-doi="10.1016/j.febslet.2006.03.072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fusaro</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtin</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The evolution and diversification of Dicers in plants.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">580</span>: <span class="cit-fpage">2442</span>–<span class="cit-lpage">2450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DMargis%26rft.auinit1%253DR.%26rft.volume%253D580%26rft.issue%253D10%26rft.spage%253D2442%26rft.epage%253D2450%26rft.atitle%253DThe%2Bevolution%2Band%2Bdiversification%2Bof%2BDicers%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2006.03.072%26rft_id%253Dinfo%253Apmid%252F16638569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2006.03.072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16638569&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237433500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.44" data-doi="10.1038/nrg3068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation transcriptome assembly.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMartin%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D671%26rft.epage%253D682%26rft.atitle%253DNext-generation%2Btranscriptome%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3068%26rft_id%253Dinfo%253Apmid%252F21897427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21897427&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.45" data-doi="10.1093/bioinformatics/btq329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathelier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MIReNA: finding microRNAs with high accuracy and no learning at genome scale and from deep sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2226</span>–<span class="cit-lpage">2234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq329%26rft_id%253Dinfo%253Apmid%252F20591903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTgvMjIyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.46" data-doi="10.1038/nrg3683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA-directed DNA methylation: an epigenetic pathway of increasing complexity.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">394</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3683%26rft_id%253Dinfo%253Apmid%252F24805120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24805120&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.47" data-doi="10.1105/tpc.108.064311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baulcombe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Criteria for annotation of plant microRNAs.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3186</span>–<span class="cit-lpage">3190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.064311%26rft_id%253Dinfo%253Apmid%252F19074682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8zMTg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montes</span>  <span class="cit-name-given-names">R. A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosas-Cardenas</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Paoli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Accerbi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rymarquis</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sample sequencing of vascular plants demonstrates widespread conservation and divergence of microRNAs.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3722%26rft_id%253Dinfo%253Apmid%252F24759728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24759728&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izquierdo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emes</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembly, assessment, and availability of <em>de novo</em> generated eukaryotic transcriptomes.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myburg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amerson</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubisiak</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Differential gene expression in loblolly pine (<em>Pinus taeda</em> L.) challenged with the fusiform rust fungus, <em>Cronartium quercuum</em> f.sp <em>fusiforme</em>.</span> <abbr class="cit-jnl-abbrev">Physiol. Mol. Plant Pathol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BMol.%2BPlant%2BPathol.%26rft.volume%253D68%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.51" data-doi="10.1186/gb-2014-15-2-r39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR39%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-2-r39%26rft_id%253Dinfo%253Apmid%252F24576357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-2-r39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24576357&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification and expression profiles of sRNAs and their biogenesis and action-related genes in male and female cones of <em>Pinus tabuliformis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.53" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wetterbom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuccolo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway spruce genome sequence and conifer genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-web" id="cit-6.12.3787.54"><div class="cit-metadata unstructured">Oksanen, J., F. G. Blanchet, R. Kindt, P. Legendre, P. R. Minchin <em>et al.</em>, 2016 Vegan: community ecology package. R package version 2.3–4. Available at: <a href="http://CRAN.R-project.org/package=vegan.">http://CRAN.R-project.org/package=vegan.</a> Accessed: August 2, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.55" data-doi="10.1371/journal.pone.0009490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FastTree 2 - approximately maximum-likelihood trees for large alignments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPrice%26rft.auinit1%253DM.%2BN.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9490%26rft.epage%253De9490%26rft.atitle%253DFastTree%2B2--approximately%2Bmaximum-likelihood%2Btrees%2Bfor%2Blarge%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009490%26rft_id%253Dinfo%253Apmid%252F20224823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20224823&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.56" data-doi="10.1093/nar/gku986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Proost</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Bel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaneechoutte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inzé</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PLAZA 3.0: an access point for plant comparative genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D974</span>–<span class="cit-lpage">D981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku986%26rft_id%253Dinfo%253Apmid%252F25324309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My9EMS9EOTc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.57" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.58" data-doi="10.1093/nar/gks1219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quast</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pruesse</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yilmaz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerken</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The SILVA ribosomal RNA gene database project: improved data processing and web-based tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>(<span class="cit-supplement">D1</span>): <span class="cit-fpage">D590</span>–<span class="cit-lpage">D596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1219%26rft_id%253Dinfo%253Apmid%252F23193283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9ENTkwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-web" id="cit-6.12.3787.59"><div class="cit-metadata unstructured">Roberts, D. W., 2016 labdsv: ordination and multivariate analysis for ecology. R package version 1.8–0. Available at: <a href="http://CRAN.R-project.org/package=labdsv.">http://CRAN.R-project.org/package=labdsv.</a> Accessed: August 1, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.60" data-doi="10.1016/j.tplants.2010.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rushton</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somssich</span>  <span class="cit-name-given-names">I. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringler</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Q. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">WRKY transcription factors.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DRushton%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D247%26rft.epage%253D258%26rft.atitle%253DWRKY%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.02.006%26rft_id%253Dinfo%253Apmid%252F20304701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20304701&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278197500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.61" data-doi="10.1186/1756-0500-3-126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sangha</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An improved method for RNA isolation and cDNA library construction from immature seeds of <em>Jatropha curcas</em> L.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DSangha%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D3%26rft.spage%253D126%26rft.epage%253D126%26rft.atitle%253DAn%2Bimproved%2Bmethod%2Bfor%2BRNA%2Bisolation%2Band%2BcDNA%2Blibrary%2Bconstruction%2Bfrom%2Bimmature%2Bseeds%2Bof%2BJatropha%2Bcurcas%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-3-126%26rft_id%253Dinfo%253Apmid%252F20444276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-3-126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20444276&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.62" data-doi="10.1038/nbt.2705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilgner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A single-molecule long-read survey of the human transcriptome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D1009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2705%26rft_id%253Dinfo%253Apmid%252F24108091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24108091&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.63" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTkvMzIxMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzc4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Validation and characterization of <em>Citrus sinensis</em> microRNAs and their target genes.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DSong%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.spage%253D235%26rft.epage%253D235%26rft.atitle%253DValidation%2Band%2Bcharacterization%2Bof%2BCitrus%2Bsinensis%2BmicroRNAs%2Band%2Btheir%2Btarget%2Bgenes.%26rft_id%253Dinfo%253Apmid%252F22583737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22583737&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.65" data-doi="10.1073/pnas.0701061104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesicka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedoroff</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Arabidopsis</em> primary microRNA processing proteins HYL1 and DCL1 define a nuclear body distinct from the Cajal body.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">5437</span>–<span class="cit-lpage">5442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701061104%26rft_id%253Dinfo%253Apmid%252F17369351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEzLzU0MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sequence of the Sugar Pine Megagenome Genetics.</span> <span class="cit-vol">204</span>: <span class="cit-fpage">1613</span>–<span class="cit-lpage">1626</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sundell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mannapperuma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Netotea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delhomme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The plant genome integrative explorer resource: PlantGenIE.org.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1149</span>–<span class="cit-lpage">1156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D1149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.68" data-doi="10.1093/nar/gkm965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarbreck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamesch</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Hernandez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Arabidopsis information resource (TAIR): gene structure and function annotation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D1009</span>–<span class="cit-lpage">D1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm965%26rft_id%253Dinfo%253Apmid%252F17986450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE2OiIzNi9zdXBwbF8xL0QxMDA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Underwood</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth cit-collab">Bench To Basinet CvDC Informatics Subcommittee</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Long-read sequencing of chicken transcripts and identification of new transcript isoforms.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e94650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.70" data-doi="10.1073/pnas.1403244111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treutlein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokce</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quake</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudhof</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cartography of neurexin alternative splicing mapped by single-molecule long-read mRNA sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E1291</span>–<span class="cit-lpage">E1299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1403244111%26rft_id%253Dinfo%253Apmid%252F24639501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzEzL0UxMjkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.71" data-doi="10.1016/j.tig.2014.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Dijk</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaszczyszyn</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thermes</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ten years of next-generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D418%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.07.001%26rft_id%253Dinfo%253Apmid%252F25108476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25108476&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342036000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steenkmap</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myburg</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naidoo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Combined <em>de novo</em> and genome guided assembly and annotation of the <em>Pinus patula</em> juvenile shoot transcriptome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.73" data-doi="10.1534/genetics.113.159996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loopstra</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unique features of the loblolly pine (<em>Pinus taeda</em> L.) megagenome revealed through sequence annotation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWegrzyn%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D891%26rft.epage%253D909%26rft.atitle%253DUnique%2BFeatures%2Bof%2Bthe%2BLoblolly%2BPine%2B%2528Pinus%2Btaeda%2BL.%2529%2BMegagenome%2BRevealed%2BThrough%2BSequence%2BAnnotation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159996%26rft_id%253Dinfo%253Apmid%252F24653211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.74" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.75" data-doi="10.1093/molbev/msv164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arikit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extensive families of miRNAs and PHAS loci in Norway spruce demonstrate the origins of complex phasiRNA networks in seed plants.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2905</span>–<span class="cit-lpage">2918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv164%26rft_id%253Dinfo%253Apmid%252F26318183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xMS8yOTA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.76" data-doi="10.1111/tpj.12865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirather</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Full-length transcriptome sequences and splice variants obtained by a combination of sequencing platforms applied to different root tissues of <em>Salvia miltiorrhiza</em> and tanshinone biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D82%26rft.spage%253D951%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12865%26rft_id%253Dinfo%253Apmid%252F25912611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25912611&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.77" data-doi="10.1101/gad.177527.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">D.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paoli</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MicroRNAs as master regulators of the plant NB-LRR defense gene family via the production of phased, trans-acting siRNAs.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2540</span>–<span class="cit-lpage">2553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZhai%26rft.auinit1%253DJ.%26rft.volume%253D25%26rft.issue%253D23%26rft.spage%253D2540%26rft.epage%253D2553%26rft.atitle%253DMicroRNAs%2Bas%2Bmaster%2Bregulators%2Bof%2Bthe%2Bplant%2BNB-LRR%2Bdefense%2Bgene%2Bfamily%2Bvia%2Bthe%2Bproduction%2Bof%2Bphased%252C%2Btrans-acting%2BsiRNAs%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.177527.111%26rft_id%253Dinfo%253Apmid%252F22156213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzIzLzI1NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.78" data-doi="10.1111/j.1365-313X.2006.02697.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Conservation and divergence of plant microRNA genes.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DZhang%26rft.auinit1%253DB.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D259%26rft.atitle%253DConservation%2Band%2Bdivergence%2Bof%2Bplant%2BmicroRNA%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02697.x%26rft_id%253Dinfo%253Apmid%252F16623887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02697.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16623887&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236396700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.79" data-doi="10.1007/s00018-005-5467-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Evidence that miRNAs are different from other RNAs.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DZhang%26rft.auinit1%253DB.%2BH.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D246%26rft.epage%253D254%26rft.atitle%253DEvidence%2Bthat%2BmiRNAs%2Bare%2Bdifferent%2Bfrom%2Bother%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-005-5467-7%26rft_id%253Dinfo%253Apmid%252F16395542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-005-5467-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16395542&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234689100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.80" data-doi="10.1371/journal.pone.0081452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deciphering small noncoding RNAs during the transition from dormant embryo to germinated embryo in larches (<em>Larix leptolepis</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e81452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De81452%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0081452%26rft_id%253Dinfo%253Apmid%252F24339932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0081452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24339932&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3787.81" data-doi="10.1038/416703a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffrey</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Structure of the Cul1-Rbx1-Skp1-F box(Skp2) SCF ubiquitin ligase complex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZheng%26rft.auinit1%253DN.%26rft.volume%253D416%26rft.issue%253D6882%26rft.spage%253D703%26rft.epage%253D709%26rft.atitle%253DStructure%2Bof%2Bthe%2BCul1-Rbx1-Skp1-F%2BboxSkp2%2BSCF%2Bubiquitin%2Bligase%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416703a%26rft_id%253Dinfo%253Apmid%252F11961546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416703a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11961546&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3787.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175033500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/12/3787.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3765" title="Meeting Report: The Allied Genetics Conference 2016" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3803" title="Genome-Wide Association Study and QTL Mapping Reveal Genomic Loci Associated with Fusarium Ear Rot Resistance in Tropical Maize Germplasm" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/12"><span>Volume 6 Issue 12, December 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/12.cover-source.jpg?itok=F-Hw09b0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3787" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZRFGJJHko7yBe1Xg3UhOmEPhTChvU7HmEU_xDYmoMmc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/12/3787" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/12/3787" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/12/3787" class="active" data-icon-position="" data-hide-link-title="0">Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460510" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XNDAIAwB0MCeeSp5NgODZcwHxPeiPuyx-Nifa_KbroA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3787.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/12/3787" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-B--UgkT4RzLyfW5L_BKpF1wOFOCAE-xgvRj8jNzWut8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/12/3787" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3787.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460510" id="node460510--2" data-pisa="ggg;6/12/3787" data-pisa-master="ggg;g3.116.032805" data-apath="/ggg/6/12/3787.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessing the Gene Content of the Megagenome: Sugar Pine (<em>Pinus lambertiana</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Gonzalez-Ibeas</span>, <span class="highwire-citation-author" data-delta="1">Pedro J. Martinez-Garcia</span>, <span class="highwire-citation-author" data-delta="2">Randi A. Famula</span>, <span class="highwire-citation-author" data-delta="3">Annette Delfino-Mix</span>, <span class="highwire-citation-author" data-delta="4">Kristian A. Stevens</span>, <span class="highwire-citation-author" data-delta="5">Carol A. Loopstra</span>, <span class="highwire-citation-author" data-delta="6">Charles H. Langley</span>, <span class="highwire-citation-author" data-delta="7">David B. Neale</span> and <span class="highwire-citation-author" data-delta="8">Jill L. Wegrzyn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3787-3802;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032805 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460510--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Gonzalez-Ibeas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGonzalez-Ibeas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzalez-Ibeas%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGonzalez-Ibeas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pedro J. Martinez-Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BJ.%2BMartinez-Garcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez-Garcia%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BJ.%2BMartinez-Garcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Randi A. Famula </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randi%2BA.%2BFamula%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Famula%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandi%2BA.%2BFamula%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Annette Delfino-Mix </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture Forest Service, Institute of Forest Genetics, Placerville, California 95667</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annette%2BDelfino-Mix%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delfino-Mix%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnnette%2BDelfino-Mix%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kristian A. Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BA.%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BA.%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carol A. Loopstra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Ecosystem Science and Management, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carol%2BA.%2BLoopstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loopstra%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarol%2BA.%2BLoopstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David B. Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BB.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbneale@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dbneale@ucdavis.edu</a>
<a href="mailto:jill.wegrzyn@uconn.edu" class="" data-icon-position="" data-hide-link-title="0">jill.wegrzyn@uconn.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jill L. Wegrzyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jill%2BL.%2BWegrzyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wegrzyn%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJill%2BL.%2BWegrzyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbneale@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dbneale@ucdavis.edu</a>
<a href="mailto:jill.wegrzyn@uconn.edu" class="" data-icon-position="" data-hide-link-title="0">jill.wegrzyn@uconn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/12/3787" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/12/3787&amp;title=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/12/3787&amp;title=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/12/3787&amp;title=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/12/3787&amp;text=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/12/3787&amp;title=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/12/3787&amp;t=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/12/3787&amp;title=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/12/3787&amp;title=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460510/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460510" id="node460510--3" data-pisa="ggg;6/12/3787" data-pisa-master="ggg;g3.116.032805" data-apath="/ggg/6/12/3787.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessing the Gene Content of the Megagenome: Sugar Pine (<em>Pinus lambertiana</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel Gonzalez-Ibeas</span>, <span class="highwire-citation-author" data-delta="1">Pedro J. Martinez-Garcia</span>, <span class="highwire-citation-author" data-delta="2">Randi A. Famula</span>, <span class="highwire-citation-author" data-delta="3">Annette Delfino-Mix</span>, <span class="highwire-citation-author" data-delta="4">Kristian A. Stevens</span>, <span class="highwire-citation-author" data-delta="5">Carol A. Loopstra</span>, <span class="highwire-citation-author" data-delta="6">Charles H. Langley</span>, <span class="highwire-citation-author" data-delta="7">David B. Neale</span> and <span class="highwire-citation-author" data-delta="8">Jill L. Wegrzyn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3787-3802;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032805 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460510--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Gonzalez-Ibeas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGonzalez-Ibeas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzalez-Ibeas%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGonzalez-Ibeas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pedro J. Martinez-Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BJ.%2BMartinez-Garcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez-Garcia%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BJ.%2BMartinez-Garcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Randi A. Famula </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randi%2BA.%2BFamula%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Famula%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandi%2BA.%2BFamula%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Annette Delfino-Mix </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>United States Department of Agriculture Forest Service, Institute of Forest Genetics, Placerville, California 95667</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annette%2BDelfino-Mix%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delfino-Mix%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnnette%2BDelfino-Mix%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kristian A. Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BA.%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristian%2BA.%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carol A. Loopstra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Ecosystem Science and Management, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carol%2BA.%2BLoopstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loopstra%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarol%2BA.%2BLoopstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David B. Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BB.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbneale@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dbneale@ucdavis.edu</a>
<a href="mailto:jill.wegrzyn@uconn.edu" class="" data-icon-position="" data-hide-link-title="0">jill.wegrzyn@uconn.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jill L. Wegrzyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut 06269</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jill%2BL.%2BWegrzyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wegrzyn%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJill%2BL.%2BWegrzyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbneale@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dbneale@ucdavis.edu</a>
<a href="mailto:jill.wegrzyn@uconn.edu" class="" data-icon-position="" data-hide-link-title="0">jill.wegrzyn@uconn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3787&amp;rft.epage=3802&amp;rft.atitle=Assessing%20the%20Gene%20Content%20of%20the%20Megagenome%3A%20Sugar%20Pine%20%28Pinus%20lambertiana%29&amp;rft.volume=6&amp;rft.issue=12&amp;rft.date=2016-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gonzalez-Ibeas%2C+Daniel&amp;rft.au=Martinez-Garcia%2C+Pedro+J.&amp;rft.au=Famula%2C+Randi+A.&amp;rft.au=Delfino-Mix%2C+Annette&amp;rft.au=Stevens%2C+Kristian+A.&amp;rft.au=Loopstra%2C+Carol+A.&amp;rft.au=Langley%2C+Charles+H.&amp;rft.au=Neale%2C+David+B.&amp;rft.au=Wegrzyn%2C+Jill+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460510/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460510/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460510/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460510/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460510/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460510/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460510/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460510/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460510/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460510/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460510/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460510/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/12/3787" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/12/3787.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/12/3787.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/12/3787.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/12/3787.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
