<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/12/3941" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/12/3941.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/12/3941.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/12/3941.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/12/3941/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/12/3941.atom" />
<meta name="HW.pisa" content="ggg;6/12/3941" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="On Robust Association Testing for Quantitative Traits and Rare Variants" />
<meta name="DC.Identifier" content="10.1534/g3.116.035485" />
<meta name="DC.Date" content="2016-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Wei et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="With the advance of sequencing technologies, it has become a routine practice to test for association between a quantitative trait and a set of rare variants (RVs). While a number of RV association tests have been proposed, there is a dearth of studies on the robustness of RV association testing for nonnormal distributed traits, e.g. , due to skewness, which is ubiquitous in cohort studies. By extensive simulations, we demonstrate that commonly used RV tests, including sequence kernel association test (SKAT) and optimal unified SKAT (SKAT-O), are not robust to heavy-tailed or right-skewed trait distributions with inflated type I error rates; in contrast, the adaptive sum of powered score (aSPU) test is much more robust. Here we further propose a robust version of the aSPU test, called aSPUr. We conduct extensive simulations to evaluate the power of the tests, finding that for a larger number of RVs, aSPU is often more powerful than SKAT and SKAT-O, owing to its high data-adaptivity. We also compare different tests by conducting association analysis of triglyceride levels using the NHLBI ESP whole-exome sequencing data. The QQ plots for SKAT and SKAT-O were severely inflated ( λ = 1.89 and 1.78, respectively), while those for aSPU and aSPUr behaved normally. Due to its relatively high robustness to outliers and high power of the aSPU test, we recommend its use complementary to SKAT and SKAT-O. If there is evidence of inflated type I error rate from the aSPU test, we would recommend the use of the more robust, but less powerful, aSPUr test." />
<meta name="DC.Contributor" content="Peng Wei" />
<meta name="DC.Contributor" content="Ying Cao" />
<meta name="DC.Contributor" content="Yiwei Zhang" />
<meta name="DC.Contributor" content="Zhiyuan Xu" />
<meta name="DC.Contributor" content="Il-Youp Kwak" />
<meta name="DC.Contributor" content="Eric Boerwinkle" />
<meta name="DC.Contributor" content="Wei Pan" />
<meta name="article:published_time" content="2016-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="On Robust Association Testing for Quantitative Traits and Rare Variants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;With the advance of sequencing technologies, it has become a routine practice to test for association between a quantitative trait and a set of rare variants (RVs). While a number of RV association tests have been proposed, there is a dearth of studies on the robustness of RV association testing for nonnormal distributed traits, &lt;i&gt;e.g.&lt;/i&gt;, due to skewness, which is ubiquitous in cohort studies. By extensive simulations, we demonstrate that commonly used RV tests, including sequence kernel association test (SKAT) and optimal unified SKAT (SKAT-O), are not robust to heavy-tailed or right-skewed trait distributions with inflated type I error rates; in contrast, the adaptive sum of powered score (aSPU) test is much more robust. Here we further propose a robust version of the aSPU test, called aSPUr. We conduct extensive simulations to evaluate the power of the tests, finding that for a larger number of RVs, aSPU is often more powerful than SKAT and SKAT-O, owing to its high data-adaptivity. We also compare different tests by conducting association analysis of triglyceride levels using the NHLBI ESP whole-exome sequencing data. The QQ plots for SKAT and SKAT-O were severely inflated (&lt;i&gt;λ&lt;/i&gt; = 1.89 and 1.78, respectively), while those for aSPU and aSPUr behaved normally. Due to its relatively high robustness to outliers and high power of the aSPU test, we recommend its use complementary to SKAT and SKAT-O. If there is evidence of inflated type I error rate from the aSPU test, we would recommend the use of the more robust, but less powerful, aSPUr test.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/12/01" />
<meta name="citation_mjid" content="ggg;6/12/3941" />
<meta name="citation_id" content="6/12/3941" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/12/3941" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/12/3941.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/12/3941.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/12/3941.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035485" />
<meta name="citation_pmid" content="27678522" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3941" />
<meta name="citation_lastpage" content="3950" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Peng Wei" />
<meta name="citation_author_institution" content="Department of Biostatistics, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030" />
<meta name="citation_author_institution" content="Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030" />
<meta name="citation_author_email" content="pwei2@mdanderson.org" />
<meta name="citation_author_email" content="weip@biostat.umn.edu" />
<meta name="citation_author" content="Ying Cao" />
<meta name="citation_author_institution" content="Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030" />
<meta name="citation_author" content="Yiwei Zhang" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455" />
<meta name="citation_author" content="Zhiyuan Xu" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455" />
<meta name="citation_author" content="Il-Youp Kwak" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455" />
<meta name="citation_author" content="Eric Boerwinkle" />
<meta name="citation_author_institution" content="Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Wei Pan" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455" />
<meta name="citation_author_email" content="pwei2@mdanderson.org" />
<meta name="citation_author_email" content="weip@biostat.umn.edu" />
<meta name="citation_reference" content="citation_author=G. R. Abecasis;citation_author=D. Applebaum-Bowden;citation_author=K. Barnes;citation_author=T. Blackwell;citation_author=J. Blangero;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=P. L. Auer;citation_author=A. P. Reiner;citation_author=S. M. Leal;citation_title=The effect of phenotypic outliers and non-normality on rare-variant association testing.;citation_pages=1188-1194;citation_volume=24;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=V. Bansal;citation_author=O. Libiger;citation_author=A. Torkamani;citation_author=N. J. Schork;citation_title=Statistical analysis strategies for association studies involving rare variants.;citation_pages=773-785;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20940738;citation_doi=10.1038/nrg2867" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=S. Basu;citation_author=W. Pan;citation_title=Comparison of statistical tests for disease association with rare variants.;citation_pages=606-619;citation_volume=35;citation_year=2011;citation_issue=7;citation_pmid=21769936;citation_doi=10.1002/gepi.20609" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=Y. Cao;citation_author=P. Wei;citation_author=M. Bailey;citation_author=J. S. Kauwe;citation_author=T. J. Maxwell;citation_title=A versatile omnibus test for detecting mean and variance heterogeneity.;citation_pages=51-59;citation_volume=38;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=J. Crosby;citation_author=G.M. Peloso;citation_author=P.L. Auer;citation_author=D.R. Crosslin;citation_author=N.O. Stitziel;citation_title=Loss-of-function mutations in APOC3, triglycerides, and coronary disease.;citation_pages=22-31;citation_volume=371;citation_year=2014;citation_pmid=24941081;citation_doi=10.1056/NEJMoa1307095" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=D. Devlin;citation_author=K. Roeder;citation_title=Genomic control for association studies.;citation_pages=997-1004;citation_volume=55;citation_year=1999;citation_issue=4;citation_pmid=11315092;citation_doi=10.1111/j.0006-341X.1999.00997.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Fan;citation_author=Y. Wang;citation_author=M. Boehnke;citation_author=W. Chen;citation_author=Y. Li;citation_title=Gene level meta-analysis of quantitative traits by functional linear models.;citation_volume=200;citation_year=2015;citation_pmid=26058849;citation_doi=10.1534/genetics.115.178343" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=P. J. Huber;citation_title=Robust estimation of a location parameter.;citation_pages=73-101;citation_volume=35;citation_year=1964;citation_doi=10.1214/aoms/1177703732" />
<meta name="citation_reference" content="citation_title=Robust Statistical Methods with R;citation_author=J. Jureckova;citation_author=J. Picek;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Lee;citation_author=M. J. Emond;citation_author=M. J. Bamshad;citation_author=K. C. Barnes;citation_author=M. J. Rieder;citation_title=Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies.;citation_pages=224-237;citation_volume=91;citation_year=2012;citation_issue=2;citation_pmid=22863193;citation_doi=10.1016/j.ajhg.2012.06.007" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=S. Lee;citation_author=G. Abecasis;citation_author=M. Boehnke;citation_author=X. Lin;citation_title=Rare-variant association analysis: study designs and statistical tests.;citation_pages=523;citation_volume=95;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. Li;citation_author=S. M. Leal;citation_title=Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data.;citation_pages=311-321;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18691683;citation_doi=10.1016/j.ajhg.2008.06.024" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. H. Li;citation_author=A. C. Morrison;citation_author=C. Kovar;citation_author=L. A. Cupples;citation_author=J. A. Brody;citation_title=Analysis of loss-of-function variants and 20 risk factor phenotypes in 8,554 individuals identifies loci influencing chronic disease.;citation_pages=640-642;citation_volume=47;citation_year=2015;citation_pmid=25915599;citation_doi=10.1038/ng.3270" />
<meta name="citation_reference" content="citation_author=S. Mukherjee;citation_author=C. O. Dushlaine;citation_author=C. V. Hout;citation_author=S. Bruse;citation_author=J. B. Leader;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=W. Pan;citation_title=Asymptotic tests of association with multiple SNPs in linkage disequilibrium.;citation_pages=497-507;citation_volume=33;citation_year=2009;citation_issue=6;citation_pmid=19170135;citation_doi=10.1002/gepi.20402" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=W. Pan;citation_title=Relationship between genomic distance-based regression and kernel machine regression for multi-marker association testing.;citation_pages=211-216;citation_volume=35;citation_year=2011;citation_pmid=21308765" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Pan;citation_author=J. Kim;citation_author=Y. Zhang;citation_author=X. Shen;citation_author=P. Wei;citation_title=A powerful and adaptive association test for rare variants.;citation_volume=197;citation_year=2014;citation_pmid=24831820;citation_doi=10.1534/genetics.114.165035" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=W. Pan;citation_author=Y. M. Chen;citation_author=P. Wei;citation_title=Testing for polygenic effects in genome-wide association studies.;citation_pages=306-316;citation_volume=39;citation_year=2015;citation_pmid=25847094;citation_doi=10.1002/gepi.21899" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=W. Pan;citation_author=I. Y. Kwak;citation_author=P. Wei;citation_title=A powerful pathway-based adaptive test for genetic association with common or rare variants.;citation_pages=86-98;citation_volume=97;citation_year=2015;citation_pmid=26119817;citation_doi=10.1016/j.ajhg.2015.05.018" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=D. J. Schaid;citation_author=S. K. McDonnell;citation_author=J. P. Sinnwell;citation_author=S. N. Thibodeau;citation_title=Multiple genetic variant association testing by collapsing and kernel methods with pedigree or population structured data.;citation_pages=409-418;citation_volume=37;citation_year=2013;citation_pmid=23650101;citation_doi=10.1002/gepi.21727" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=I. Tachmazidou;citation_author=G. Dedoussis;citation_author=L. Southam;citation_author=A. E. Farmaki;citation_author=G. R. Ritchie;citation_title=A rare functional cardioprotective APOC3 variant has risen in frequency in distinct population isolates.;citation_pages=2872;citation_volume=4;citation_year=2013;citation_pmid=24343240" />
<meta name="citation_reference" content="citation_title=Nature;citation_author=K. Walter;citation_author=J. L. Min;citation_author=J. Huang;citation_author=L. Crooks;citation_author=Y. Memari;citation_title=The UK10K project identifies rare variants in health and disease.;citation_pages=82-90;citation_volume=526;citation_year=2015;citation_pmid=26367797;citation_doi=10.1038/nature14962" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Wang;citation_author=R. C. Elston;citation_title=Improved power by use of a weighted score test for linkage disequilibrium mapping.;citation_pages=353-360;citation_volume=80;citation_year=2007;citation_issue=2;citation_pmid=17236140;citation_doi=10.1086/511312" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. C. Wu;citation_author=S. Lee;citation_author=T. Cai;citation_author=Y. Li;citation_author=M. Boehnke;citation_title=Rare-variant association testing for sequencing data with the sequence kernel association test.;citation_pages=82-93;citation_volume=89;citation_year=2011;citation_issue=1;citation_pmid=21737059;citation_doi=10.1016/j.ajhg.2011.05.029" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Yi;citation_author=N. Liu;citation_author=D. Zhi;citation_author=J. Li;citation_title=Hierarchical generalized linear models for multiple groups of rare and common variants: jointly estimating group and individual-variant effects.;citation_pages=e1002382-e1002382;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22144906;citation_doi=10.1371/journal.pgen.1002382" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="On Robust Association Testing for Quantitative Traits and Rare Variants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="twitter:description" content="With the advance of sequencing technologies, it has become a routine practice to test for association between a quantitative trait and a set of rare variants (RVs). While a number of RV association tests have been proposed, there is a dearth of studies on the robustness of RV association testing for nonnormal distributed traits, e.g. , due to skewness, which is ubiquitous in cohort studies. By extensive simulations, we demonstrate that commonly used RV tests, including sequence kernel association test (SKAT) and optimal unified SKAT (SKAT-O), are not robust to heavy-tailed or right-skewed trait distributions with inflated type I error rates; in contrast, the adaptive sum of powered score (aSPU) test is much more robust. Here we further propose a robust version of the aSPU test, called aSPUr. We conduct extensive simulations to evaluate the power of the tests, finding that for a larger number of RVs, aSPU is often more powerful than SKAT and SKAT-O, owing to its high data-adaptivity. We also compare different tests by conducting association analysis of triglyceride levels using the NHLBI ESP whole-exome sequencing data. The QQ plots for SKAT and SKAT-O were severely inflated ( λ = 1.89 and 1.78, respectively), while those for aSPU and aSPUr behaved normally. Due to its relatively high robustness to outliers and high power of the aSPU test, we recommend its use complementary to SKAT and SKAT-O. If there is evidence of inflated type I error rate from the aSPU test, we would recommend the use of the more robust, but less powerful, aSPUr test." />
<meta name="og-title" property="og:title" content="On Robust Association Testing for Quantitative Traits and Rare Variants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/12/3941" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="With the advance of sequencing technologies, it has become a routine practice to test for association between a quantitative trait and a set of rare variants (RVs). While a number of RV association tests have been proposed, there is a dearth of studies on the robustness of RV association testing for nonnormal distributed traits, e.g. , due to skewness, which is ubiquitous in cohort studies. By extensive simulations, we demonstrate that commonly used RV tests, including sequence kernel association test (SKAT) and optimal unified SKAT (SKAT-O), are not robust to heavy-tailed or right-skewed trait distributions with inflated type I error rates; in contrast, the adaptive sum of powered score (aSPU) test is much more robust. Here we further propose a robust version of the aSPU test, called aSPUr. We conduct extensive simulations to evaluate the power of the tests, finding that for a larger number of RVs, aSPU is often more powerful than SKAT and SKAT-O, owing to its high data-adaptivity. We also compare different tests by conducting association analysis of triglyceride levels using the NHLBI ESP whole-exome sequencing data. The QQ plots for SKAT and SKAT-O were severely inflated ( λ = 1.89 and 1.78, respectively), while those for aSPU and aSPUr behaved normally. Due to its relatively high robustness to outliers and high power of the aSPU test, we recommend its use complementary to SKAT and SKAT-O. If there is evidence of inflated type I error rate from the aSPU test, we would recommend the use of the more robust, but less powerful, aSPUr test." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
    <title>On Robust Association Testing for Quantitative Traits and Rare Variants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TsI8nkeoY4WX2QLQo_zQTJO3QgyHobA2NBIFLtb5vtU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460496","apath":"\/ggg\/6\/12\/3941.atom","pisa":"ggg;6\/12\/3941","ac":{"\/ggg\/6\/12\/3941.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/12\/3941.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3941"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3941"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/12\/3941":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VhXFoQns9\\\\\\\/E5sFHNnLDwlKoXW\\\\\\\/5l8epS7Vmc4h3SDIreSlsktqf85IcTdkPUz80XYVGkb1JdYJuGIVZg4q84AzmXa93DR6AE4jbfaTUZObt97um18QP5ABgr8ZqFK9Ss\\\\\\\/2EIOfUzNpUB5pJx\\\\\\\/Iz95PlI3j5KolTzhfWCKnp1111N+7vEBmatbUYVmigzGg0zNxzWgkIeMtTwYeEE9hcTXhfVrdDfXhoAEPSoIq8khbdeMuLADZOecuo9t98KeMgz1uV8qtsCO\\\\\\\/xoOklhLelzzCo+9tFlDixmB90ZY156JK7Zzqp2vC5hmdQd5JMypnPWBJ8RKpeu1Mg6k7SX5GTZNo7E\\\\\\\/GabNRqj68rNzZtmdkXOEFytdsgxato7K8W1PHBEI3JIBLF\\\\\\\/J\\\\\\\/dLJPhBbZxJjMMA5rPbW33eD6DHafmt+8Pyt4TRRhmdyzIH6k1NGzIzlE82VbsxefgnsnYrI9Sxyw1MXZ4rRNLXkimToLohILLJUMfnYItGJv22bxXhCHbxIeoVo6bKRffl1dovMON3RooKtSF8KSWcJoFPQnHL1pKrbauErHJWRr4+GzqHqPn1FlGqluV\\\\\\\/wlajBIDnjVVb\\\\\\\/M47xg25PYbv9tPD6zaboyeSuWtSXP2q1wsooNLkQoi2UBdpGodWspJ01CTOurTJ86\\\\\\\/nqHTNrRysr32VI0S8h2GATSVIMeRvsGBYIZ3JL++lM\\\\\\\/qn0U3gP7NDIdCWrKRvpwjdX10j2f6qVs96MlPgjmRIFAsQAeG9mJwvhd+dIUj9YDpcEo82xV16pVNLVEv0XMJrL4wWILyICFwKKBlaSLFUpFwQmkwkr93uyJz2j\\\\\\\/rcwKOqur7yr2wqiMbJAkiqhfk3pet+IjNmrhrJF5AUcvLBpCzOjtqEfTLkTeNY2KyJfa7dBaPUyRTgofCWNno7zRS2D1mS3dWlA26HAOhfEhV2Oy76utNJz15d6zgVeYUiAWTK9yIUAdsl5u\\\\\\\/eLWsiPz0XgDvY+eTAMfGjLlfWJtyrCrXCzSd49hbPwVGORzBjWHBB0hopQmMthLItTewJmXmQ6ZZMIt23ucZp2zCU0gPJk+OsygHyM+uu4mNOvCU3Qk2J3gw1j2949HVoo4Bk6d2S2a4VLgGQj6FEUbvp\\\\\\\/hjSckzGGsSTm3IRo8dskyufrCHeX6Flk8j8fxLOGw+Fapg98C1NIg8NfAzivc7ujGanFj2Z63zf8HIiyUmL2QgmM3P7Qk0s7C\\\\\\\/DQbruDHv3lmMFpErJJTo5l+q1DmeAiyJ24+Tx1mznpsS3SLF1lzZT\\\u0022,\\\u0022iv\\\u0022:\\\u0022nlqE7ldyCkGPyM98YFaLgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221b3fbd3f3a2ac58bc9d1c61c07f1dd630667428c9802304ee675c10e3d17e76e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Zf\\\\\\\/xO3SozR9TJ8EI4gyBhq3GkTjTw5QyP\\\\\\\/i6ME1IFGDlXFcojBcObcu4nAkgTUs2dimzVH3DD3FMcAj1mROGZrI5072Ixiyqj5bxYQbE7fAcsg\\\\\\\/qmriWlJLxZTlRTn6oq2isPVahH+Ad5Ia1F94OA+\\\\\\\/bEfDyxj2Tq8\\\\\\\/Hp00CBnjKUZiRw8Oe95ObS1PIPvs2\\\\\\\/ofyM88ZhwP8NL4EonSj0URgjJtREnN2sTctFtD+IgjmXf++6vtpbo4On0UYAbgAQIqEkQMGkjd1y5ndOTlJQSOxyENTC0sOw44ZQi1qA2WZUqLy8kQczxBDy3ShCyWMvqnPwGohsh0Tqt6UFG29UMKT+hBmPaJ3NHhVuUcYLW\\\\\\\/ri2AjR\\\\\\\/6Jj7QnNnMvkplEiFNUrZhgXZo9sFOvIbW0LEzKgNuf\\\\\\\/qwng13GOjOJ\\\\\\\/S6bp8M4cW9U\\\\\\\/WcoNi0PHJxL8qJWK9s6mIFDW+Hwcjog\\\\\\\/42TvA53fkdVDWP3JwmiZTFzhwLGZtYPlzE99CjeWEZckg0oMVBRVG67v2WPmVS1TwCqlzD\\\\\\\/gUE0VkqYlPvRRn7k16lnSp9wUqqLpP5M042FNKWTW4rPFuYDu9DQeDelNO2Qqlmb59P5CPrABC+E66SCG3A08+xf3j5+LAwH8+dFz97iq5gsCHBfUoZLSrn2Jc+JV+L0xA+\\\\\\\/BVfFihuAWi1jkv0tpC6hJsdASPFPyF+uiDRdqDwklxXyVp7AOCoK1CiL5hY8gsebwY7lm65BxcGRjuvKN29cgCc5yuGF\\\\\\\/z9gH7ebPTzeG6xe9KqIqEI2VD22eUtucOQaJXobcMNzYfNFnFGvVQg68D+Ondjr\\\\\\\/b4Ab1i3tCQDzMkRzpDV6nc0fb9tDK4NQxQih6nw8K0hP7QGv3lAMS2X3M\\\\\\\/\\\\\\\/QA69RlneT3iAbNr34vEEjMnxN2CqfODURJ+lzySubCS7uXVa8Zi6FTKy5yj7hnfv7RMPDh9cpQ+cuqCCcRRkpcu8xxLUOcpJsoAvSmXM\\\\\\\/a9g68RFpVAtwTbL0bE0v8FTuwq3NBT18aDb3od9E4rBRrddntEbUY\\\\\\\/8+oABxRaHJgHpBw6+T4vdGSzcs\\\\\\\/zXc4MtQoz2jrBSPMBUH4LIytWV57op6wi5rYWwxZX5iM1Z0NxQBhEPq\\\\\\\/lpfR2mtFawjQIYwtDFqpj7lCJ3DsN7QjcuZA3EFCea3gZnstQJtP+vCHbubc31UGnaP4bImlj6+wQ0W+BxGPkJ7GW3gtaah+XtQCS7b90yGyzSnz04zWio6VvMvVfoGzUG6Jjak4i9grVU66GqSs0Qj\\\\\\\/eYwA7Wx4JOd3HwKsDkI5HXcJHJBE6x6XpM7xOHURc36E4RhStEbXzQyqUvqFYQQX6acbeB8Py5yG7kYF5BDiXqQnNIQUD6pIxGgAuvkZ6tETc35BfmoArrVsAtI5Qfg3lhiXj9ub4h6kEVprREAe6c169yXBmcw0fnllxgwZ35Wha0yx9k53SPRU4LMRn4yuRDI7TLFAQWN4pkF3PcjmYwXMp0LzNE0HEzUQrytNDVmF0ydSVKVfzubcS\\\\\\\/gOZqznK3hdBO8GC6+ybMYWXoJZ1ZkGjluiLRhxBYfFFGeJY2e8rAZgD5TrmsJ0ET2Yje5bcnpNrw0423xPKPTQ7Tw8fFhMjFcvKoFsX4IwGTn2o5DaXt2eJK\\\\\\\/7K8z3AM7HiONsZYLlQPpxgIptWcspzr9sGO9m9un3\\\\\\\/48C5SElU+BAY6gClRkBVoXcvhubtrv1B6P1MCi6WBLn2MCiLivKWhDzCbiffZf3k3bQ2tFIav8XfDBcIFerb\\\\\\\/lpPii2UcsAGC3TNaYjBiMqG\\\\\\\/Sd0u80KeTeM9+4FSzVhNHb1FXH0yUlHQwOrTqsuPbmkB95OTsP2wz0Dr26Jo1\\\\\\\/v2xeSiaRWv9g0HvgUrIfwDl04hYG4E6uQcQIa2PYzFLpz3M0WpNifK6aH\\\\\\\/gOdiKO4DnKgaxvmG4diLpkpyt4CwOgkwzl3656OfzcbGAFsjvpj0CDHRznCOFKsUYs9RlbAN9V\\\\\\\/1uWkPMpXOj9A52\\\\\\\/1CC99Y4vu4gHXp8vhBMB9XOT68LcGZF7hHO9DoaHi7sjqRZM01woiqsyiC0+SlTGpfi0acFX5TeJrr+AEswB0a6bHC2MtR4YFCcWjsjUfYkOHKD\\\\\\\/jmta6PIuCqiiJ204NSETm9i3whS2+k0iyVMjOyW+BRGw+2xw4+pF7viWJKj69TAVJHHotOpz+YjqbFzCWT4+sPQRvXGw8ISIztP5HU9xxI\\\\\\\/8FUA5mTVkVH6a9yiRIt8i5JXMvQgzrlgcO4voCtE9htCQhHF6oiDiuY7QFJGFyCl1SmPXONy3bRg\\\\\\\/IqGx5Hf\\\\\\\/92rfMTVJne\\\\\\\/lR\\\\\\\/OfbvLTuX6DfLr\\\\\\\/XtL8AoQfpqaIp54o92egBVd\\\\\\\/c6OH\\\\\\\/SsbwxsldD2DZQdq5JrnXqNpORKs+\\\\\\\/JcP0Q6cTMetXDbpXI3E2ld648UCl73G4CmBNjMZZV5lWMtMHxScbhnfPOBnvrz56ArCzK0QvEx+A12spdwh1dr3xsQ\\\\\\\/6hab9GavS5efHATl2eNqWXZDLROsV0N54PvjH2L18oGb0mMtfNpdqBPIxC0LveQLyDRlrAWeZpnCYpnzzjG3eArgviiEuhwGt8N2W3OJqwfb+s8vwQmXBOV3K9FeuAxK0PWJqjBE9Cy5bgp83IrXEn9xwK8Dr3O6Wia+RqeHIIoW8rgBJKtI8mZGD+PBibD1hhcoCrxImIVXBSzFIP3FiJuyi+VqidAFDo\\\\\\\/rnH61bt9b1vj+YvhwM+wsnpHwzraMJb1GNZCpCWGX4oXCsuTROs\\\\\\\/y9VATvWN86eWTB7kJL\\\\\\\/l1lasc\\\\\\\/4YJNprB9GFRveheiOIwGgePxAlrSYjkvrp2dxwWu9pWu3hXwWROyeRj9lMNhDTX1Rkdjve2ZZJ0tPo8u9dHkEwJhJtMq5XfzpBWXWmDh0mFAIhcx9cF9hC2Cl4Ddu9OZkO4ytaiGVmH0uD7FtXCF5qf\\\\\\\/Oa3PxsuMcYZOKEHpvQMx6jKqwMJJz32MrRahWdMCOD6UQxxyb+mkjNuRG6Sz8+7R+X15HGQe1olhr5gc9Db93\\\\\\\/tAVH0NQvd\\\\\\\/JM0SCIMEC+Vd7HrwfRfj68h8Mp\\\\\\\/y+CArxd6YRgrrhw0aUgOV5u93R9LyRXnL5W1I378msvJEQoqkS66tHg+Kvc8ID7cEdmSarhUMKOJxTZyrkHn5sh5PC6eko8TetnrRfkg6Z8n51jNyYps39yybW1a5gHwv0yFLxAxgx9sLjNttyUvBDKmNPVWU\\\\\\\/vLB\\\\\\\/lhIPdqVV3lGYnCpO78pJoKHFacX3wPpKRXYEOzn3oQ0RmBzz5VcYJORGRsfLiuMy0idReKYRdVppIPyzCZq\\\\\\\/owbUZKpkOwRbq7z7jsL\\\\\\\/i+xOV5+yVvGmysgsY+ITVIHGJu\\\\\\\/b42wqfwdhTXcoquqbMu8d6\\\\\\\/6OdAN1uU5j9hnce522N480tsW9W\\\\\\\/ZqIwbnVZGgdajTogK6JFTb2wdupscZgLV2wdkdo0\\\\\\\/qKSWJRZ35oH862abuqKiL1IeEadGncIZd27yM\\\\\\\/XcdzWGcs7ErHsn1d96zQu7HqOAkPZn9Jvk8nYiRFI9S\\\\\\\/9UDBd3LD+WT6jRruoYkzuaKxCFZaPRnLRPySacB7H6W0n9ByHpsNRyvI2InuWMxM0slpUtH0sKEZQIQ2rl3oYDJd6+nuokECcuhSDM+iUFgsyEiGG5ge+gql1t36IgczecAdQfHlRTXjhNriSLuyWHFYrIZogyw4AxlyHZzLr0BN9oDngPD\\\\\\\/dEgNo\\\\\\\/A227zqtOSsWQ1yOpT7Qm43qUY4Hgn8xqw270NBtU4g957xXx42Ndi0UrylREffhqRoQqHc9FlTgeAGNQ398DgXH6DZ+2uv6\\\\\\\/nYUmsQoT73LBW7TzZclECpuOV9XuX69Ngomf0bIERBN1bAu4YM\\\\\\\/pMI67R\\\\\\\/K4PCofMnhlD9arihoqzKrXkYlZ7DRa7WT\\\\\\\/X2PXyPmEX+y436K7oqbmDZbeM7ZrlwoulwDs3tzDv6\\\\\\\/46s2RBY1Mac5mjICzbzBwOk0Mep0fzfq21wSeUNbBldhuigxItkxjqgpH8v5wI0Wa2rokD5jnNtSBierBDjdQq\\\\\\\/6a\\\\\\\/EqhZ753hHOpajVR5vh+2vtX2pxv\\\\\\\/4SDPYQMzUgyBfT+Kehl+lWZZ6CR3rkY57DeV0CJdp4lS\\\\\\\/JgV83bFdurJlphg9momMQkpOOhk45TWU2OwmiMEM2eFHd5RJWSXYH2\\\\\\\/VzAeQ7\\\\\\\/KbyggaMcznNqEJTbMtp+JfNGowS6zHs3bhvkBROMTSt9sZNUdMl5IqeGsGupl\\\\\\\/jctTOuJXAcyj5lzUWdnLVtp\\\\\\\/3ggHTMombBJ+HODYNi7bDlEImHb8TJssVbHjGv5MbvhYlhFoeGqLjTEsIFsasTusN2xUVUmsv0ApRCMCqP7pP5J\\\\\\\/5\\\\\\\/UuKy\\\\\\\/xb2xTK5B6c5H39aYoy4yNI9IAav7mvQ+9D13Rflp+wVJmXGnVBDaHAwhSjxglzTbfx4pFZ7FHe+teacFl1DcLgve4A88DSlcKyYu0D9tyQdOWsOcTfB3ZcpB4h2JUfqVgvNaRrXn9jpHMHvuZB1ad92qNOXaoKmX+PpGE+NSgx+sihNqqFjmuGJie1Nvgn23fV23v3Y+Uwefd8srvcIzNy6s0q\\\\\\\/\\\\\\\/RD+qS7YRc44D0xUeAFYcorIbCwbyIbzvsJYaRN2oMgZ7Naca1QwHBHFt\\\\\\\/R2St9kxiRZ4Jb3FCYlh\\\\\\\/sTfYwavGfNdLoRBv5vj7VLDjiUjuB+MCAO7p3WA3RVSdUpk8whkt+MUQeciZvkqLHrLVnsJgHCdgXhu27Q3UKHft9VSpujGqpwS1d3d0EjbxRjmSlkUhY5C8QHihTn895nwabd\\\\\\\/HB0tbu9tAzDXeDEYOTbE9fEM+vAh5\\\\\\\/YsWpRcvdRvRJB8xPd9RYUU393vqRUBkH4HY9\\\\\\\/ZXLHK33gqvrgqUy56ElrVLF5Vgpp1BwPvi+IZvYZPZyue2TlFm3rfmwDP1onBnv\\\\\\\/I5izWzcVEnacTCrQaWbH7HO5siJRmPkY\\\\\\\/xgxo5Nt4cksuUSR\\\\\\\/Nnl9LXLjkfnNagKo6QhSfJ\\\\\\\/TCCQQkLwc\\\\\\\/ZN3Pcx6PJPTL6UpadWs7OTCK1pKkidqmfA93w\\\\\\\/G96JQOsiGcgdY\\\\\\\/5Dh57YmD6bDAfbJ6sgV\\\\\\\/uAYucsY4BtF0tRHYFXOkBlpg354fAKq8lalCDf6oV00Suwnm0YZhOZTqIN08w5pNtEDx\\\\\\\/4KxtsahA\\\\\\\/j5HVCXFryW6SfOuk8yvkDi9PezPIleW9xhQMRZ2g5S+mxvgjb\\\\\\\/WrF90Em6R8n1qW9xw6QJCX7PXGyKG3AVSuxXfohmOSUT3VOSh9PAILWkNscGbSo9eF6G2T2n7bv0ElHbrZi3vMiHC9rn0atRwFJnJqfGYBFUIsNWscgXy0AY\\\\\\\/lkNm9wEFTVH1krAb3NmLRlmwPJcj7Nc9OV4ZuBG+2f+4uGniuia7JVw804dM3\\\\\\\/mekHFxnvlbHvmtoR1v\\\\\\\/+aB7z\\\\\\\/nHIKNNJxws2HPSEnzKhdM6mfUEbOzvmpl+GNW0\\\\\\\/ttlgY3FzPnhIncG0CksEYRftVf7Kcb2zqoX8qFHfifdi4jEs8vdMFl2E2x3mHbr9vaktYs7H9pWYDCd8LycAsXtVrpj4ovSAwGodEDPF6BMayUseSv7lJF+PQTOb0DryUDp59SAzXv4QXEEWLU9FkEMcDH\\\\\\\/2h\\\\\\\/jdkTvF0NxLRU2wiMGE7uL03XHx\\\\\\\/Bn94xCUhyUfow4NiH6Lnh3hmhSJlynyh8Bx+JO9tURYKBJ0B4f8tlMuIuIB64SJKQ3ic8M0ObSfwAKyqCd7qbaEmMeIpJSkBym4Jf4a+vX7R0utnvX1pTW\\\\\\\/jN0\\\\\\\/\\\\\\\/uBTQNq1hY1FTAFcJRRUs9qr3UM\\\\\\\/ruUkPRHhpAFCikFrwr2nWF31pNyBXezaA8opADs+ovQ46mgrFmaYHZqDT+gVovVaPQwuamqpr8nN3TOkLshhC9FYk\\\\\\\/mPGlwtY6okq6ANkaqnU1jFLIj8X5n4BPYPyQ9uNPFCXI\\\\\\\/Y5CxxASFG7HyDijL73wUTBet8zX3Z0+mUuFuvCE3XDA2JQy885yh5agS1Z4xxyEMPHPMyS63ramOP\\\\\\\/uKFcfsMucFYdwcvh+UuQYi1DXyNGSoA5Lj659ps2P7iGxwgXsFlUlqTJw89Cd5SBevjEvU33jPLQV0n2Q5ZzRma26sFMIycgPS9zO97wAgbFj0MQ+tR6wZ1PT83kpW9OpQ5by4CDLi\\\\\\\/SyXGYJEhfa2xGoL+zUUXGQ\\\\\\\/TQRO\\\\\\\/HpDOvo+R5IG4bT2jO9\\\\\\\/1SwBLq7Rq63sx7KgYCwTFcr7qQgadDWhoDhDUw7cLajoX08UCY0hWpe9fPBcXSqW\\\\\\\/cKqG\\\\\\\/dLac3Lhhkm5P0bsyBVfbDDyPFcTGNPoGqUaLSClrVyileWQ\\\\\\\/jmrtg3KxMil5YhNoKDbAqmz3gpFxNjKx04YIH9Gg9rZ8sHXoD9Oyb0S7QleLn+IyQx1wI3bDMstYSJMq7M7GZJ+r0DkZSQVjcFyxmW6Z4ni2hbrwBeKeLtIFYcWES40CFtgFIGmaLDQqCkx4xYXmhybakQ6Opep5OxTH9+bYDusFrqDq0pq7K0Yyz50VAIxl9auygDH\\\\\\\/om\\\\\\\/4YbLWflkC5nPE3tQrR97ZjhwJhGrhjtIxhxgNInIP1fJjrW\\\\\\\/Z67AQKyU710y6d1Uo+KtpLVLW\\\\\\\/uwQmfUoH5HTHQE74gOvUgUoJM46lPUR7QqRyDTDz5Gwco8JtCAk3vEdZ2dogqso9V2IBeKRgL7s++17lXuo4ueMSNmnvzbkaNOtlx1hqvCKRQAcfWCPizdvcq7NbDFiYaqtY1TFF+7JXHvwtP+KQt5ramLLvDFo10YXcyOXEtfz7rKVYautJelzD14b1wLjNGQQGff0EyVIHAIFw7sbJoZDyZ4HibLH3QwsWfSJx\\\\\\\/2lPAUO1GSyEXJxInOi8nXguBXUnRMRqPgNVKiWrpnRYVh1yoqg60RzRv5bE6pt8zwCelIZ92dnLmkBY63i\\\\\\\/qucPmyLXui516l5qwkOIJJ9pcAfuI2T+kQBoV5V42+5qHjiaK4BJK9RBntw7GkT4TkRqRQSATJZ5EvVBP6\\\\\\\/2txUiBHgj8QVnDgoE7Z2qMyVRqzbOnDm0wLVFVrIqHEQg8uTt8dLtwHyzy7B2Dk6\\\\\\\/14cgh3DZPhBNhG7sI7+l1+sDzR8yqoj1md7kTy1bjBhywq4hPaN7t1FU1KiuCKTcQ4L\\\\\\\/4A2bMcBxVe9FjUB5kt975d5H1wp\\\\\\\/gcm+U0knLa8FOqZ77SVCYOwIwWwIeAJCxRJ2nG+RyX5f81d2MpmBlxkaoc9ipNa353Sxv1Yi6k72K9b7l+0FgfTKNfRms+qLyaNLdPoDvuoSvqB1Y9t9UqDxo\\\\\\\/rckb1JdD\\\\\\\/9iXxYwmWI\\\\\\\/R9WRz8uHgaV+aY2Y1FcpFMw3XClMS83lbdedxS2IQwbXf6jQx8ZfB\\\\\\\/mvLEChC45RVMmMrtKsjs\\\\\\\/N+Xm2ElOHWnVUcG0CvVEhiuPCk5nIG8\\\\\\\/Ijrp0NFgoflt+h2hStJKx7BXAD0n9bJxqpEPVm8Ur417Im3liSJOFSlduqkvZXSikLfWjirn6cP6VFbOZW7fJu27vKuUjS7Q+4pbaqtgRnrQ1NLmNZGtN51MuIBprIoGB8xp2XWmu74BqyO6sDFqyR9P5uhMb1AkbA49sR2qUHiQBiB9G6S2msu0zEkHw\\\\\\\/smlLciBdn8ELedu2oXN+szXX9ysP\\\\\\\/iwC4Qm0Oez8Sk7O6DCMn34HHlwuPfI+F1qyNZjj3DWuMJ51gkGQrpb+m0VDVetZzBGJJV6J7BnQy\\\\\\\/oFvL\\\\\\\/E9xe73BRHzcVQINtRN2C9wT1y\\\\\\\/0jiJcy1EbbWDbm0lM8F1ZYOtr7Umgy4LGdFULf0HnEirQOhe5OLvEjInhx+2RRx7T8Tj\\\\\\\/KgsluS7loNCC58lQkx2i0nlOLCTUg7bGngFSNAzwMcBaFEdzemQ4Y6Dxy064Xs\\\\\\\/cQFUzMvWCcQLWxSDCICf246WLuyZCnBBdP9qZ5A\\\\\\\/hP3+K1wweWjW8YD3XeP\\\\\\\/\\\\\\\/3P6J40kNf4yFotzrWxkw7CPZQHLGm4zgRXnu2UF4SSe2rBeGOtvF5NNni0V2XHyeJiWKLWDtO2aZr4G2swk+0sylZXn3eoDgGW7dO3eC79GFbSt4cCTjbaTHrHbzQunFRW8LnCohsBePzIIiXVera9mC1MoIi6onTRqkuuMYrQtvju4D0ufMjIy7i4G9aL14L\\\\\\\/G4CqE66u\\\\\\\/jSZ06+bGQVeUAOW2lHJP3h9aGjUnBWxkIiV11cEWZNy8x5xSHJVYuR82Ai1yGLaS6oIIm8M+HtUUX7\\\\\\\/xMwNxwUjGjyejzD6FiRZqRAx\\\\\\\/Ip\\\\\\\/a1wIYrc88erW7QqlEDBvPyrH+tZpLqlHz4OlEfhVXGJLGkhAfyHcGvE3KzlUiuXvBPVv5M7BViZmbq4LUHe13UFFtaoNjwJdYSQnOfBLCuuesQ68Le3600lBgAwrNsaSK9u0Xk65C7CprB2ywFeZVTYs0Miq3rQmNXBnDQLcpNWZjUeEncvYVvIXZPfG2EkLQEsqAlop\\\\\\\/luVNTxHksAh357uR5rnieCRdnpDmatlHU8VFZEWRssHRJPV0hSm09UtvqwQFtuxXSOjeV\\\\\\\/SKOcRDnXLvkslSHTSNJbgNLIehM8GCuMgc6uUS\\\\\\\/uvVhnL1B\\\\\\\/EK8Q+YKQ110pyLxIi4K9DuB8vmIkSALhh1Ew9PT3eUY5kNtuwguTx+PiLGNGOYBPXjKRuDRyJw4F3l3xEp\\\\\\\/MqDU86u9U4zBv+XAT+d8r8b5qAc44hcFz9ouT+pTnihjuH3PgLaUZ5vHUQckuulLwJfIWDwhu+ohr501jRElY2lqo8rGOw6eyiSnPSeSK8NLd\\\\\\\/5tGTevj+QrBaVNnQXyTM4fd3qALZbMTwRGJKlNWhU20ef2ohha1koslFQEkmv2aZYZ9jlCxHkyeY7CT9uUgBvg+H9NEAzZiRc+lZn8JqkPKoj2sEb+3\\\\\\\/ZQZK252d6lPr6m0mOoWC7pzamoYr0fc02JP+J56SncECLR\\\\\\\/b75N5CYZKRxCIvXGTkdKO2Me35R3n7I3ydaW9lrQHmtm\\\\\\\/HC6gb16Zl7XnllTQDpb7m4pNHFDS8zAzsEmj2iUaY9Ya6rMry53qvscEVBksG5sv0BnnogRtXw5pWVNGzEhYaPCDuiMo7jVRR5DNWJrRg2qJv5q6H\\\\\\\/NgFWzSvxyzWJrfVpm+Hh0Qwm73cAOty67zLn0Ny\\\\\\\/yhsUrSZ9SFm0yDlfI1zPwtzXbaQwomQj0zTT6Wfo1bd34hS+ozDc88AtA5Kr0JFJIP6vCSNwjxajL746l89F0e44m3qxH9n0ZkzWjPNzRNPZ0WHbI\\\\\\\/2kIIwpphzRA5SNhpKhm3Bbcf91OqVNR5nubwXIHvjiM6aDhIYxeHmb0dH+p0dGHD3wQ4kgCm21zPAsWMk2jojh4OsKWi52CNi8MolLROKEaVkMEuttzqYoObMc3dSxApm6Rq+1ObofYVu0OkR1fyxosUwhXHoPy4Ss2mvn5ftJqWQXN2k18xsxLFVLvXk0xfRhvaSQXJqAJmllOOiBAjmrOlpsNCCr\\\\\\\/9jIw5JhsflIs4wIgZsI5Q8+BNdCK+qYjRS+kpJliyTv0xQxyfLHciy2swEBHoQ5nk62EBHx1Q57\\\\\\\/m85ciZiWsWjurK4HNfBSpuKHbebzds6j50zPWRk3WGmOnbQ6jAA7p1PnmC3TDZIqJ5C492Upqz6OOGQgoywHUp6HIrkw9Rela7PjFWDY91FGjRR+OLEpliMU29mCXFzrS4Q\\\\\\\/9a3+wEhxz0IwTjk5eZ+344bixdu656nixQgwrT2qcUUolXNraNlGvCRul76nWj5ozJmhSf6G4aJ4yQ2XEMsL6s4fecsU9bYgR1z9n0LPLp8XyVHjH+ipBbEiIGDdQec6AnZZeqhV2Vb4chPhUBidbdwJsVLzijUL6oHLMX6nfzLy8gKBpYO9wM9BX+DJBx3FTxrSilBcmUJjZ6IS6cP9NpyaIpizyhBA9AGg2iT2+uuOEaEcBi82UHDlGQS8yj54jUTWxKBQqMf7Lg3N\\\\\\\/vVGjUXT3PsaIv7AmdBiB7hyNjaW56Tna13IvmEgem8snWm3lPAzIu\\\\\\\/p7P9RxSWRjbHAQ5iO38YEkW38iOqW8+YhxIrErd7VnVM\\\\\\\/DNsLk5Q\\\\\\\/pv90n0VeRMIC+TDCz1cf4kM45hbaXgRQEP3SrkMFv3xjm6Xhnb+5T0i3UurKBqfN1icMPtymiLlS7HFmxOcQIQnD+\\\\\\\/ON5ZiS9arUQDKKF8YDwv1yMVVg0zYw\\\\\\\/1VLH8rVbEQaMEh7t8Z+ksyQiEyUi9M3Ifq48sX2x0jRRKLzUo239GG7eFWIlXhHwWOZRugjl9Q6VpfMHpmwvD74UF\\\\\\\/JSgTjfYrfbLvivq8cEVqOaK489C4+PjPEEZkEYnTJWxJ4NHqY\\\\\\\/LpdtzQTsBkb9icMdefLJkju1QcJMqcdWmSqbsqGfeAgHkjnL7B4cJls6gidVP2UmrRzAvMHKRxuB9aEGU5dgqOdU0q3zlTBQS4gPTW1BwOBDzDr3prF8KCjCavDCsi1IGOt9TzlkDqCXzLheV1zA0xTMq7Svcer\\\\\\\/oUUkjovWbu0kLOwYxqMiJ6RuNkv6BHkLsTSOhIWwYmex0+b0PTRXZpQOhnJwVxW0WLQWFT5FXEGbIuzFYbr7M\\\\\\\/O4vBK\\\\\\\/8dqzUthWCMQL06JkH3M9TApVdzCVntyOKlwSM1r8l9Ay57PRjxlgwhn31v6wVwTm++N\\\\\\\/miUWn\\\\\\\/EetyE1dth8c3lLLvEDVHkLbCD4DDtMePvnJhgDQxSTTVzBE8Ma7AlyiRDIMLnxk7LYgFqb64HnmEL5T2OpSXOOVHY0Ov5N8HPi9rcXhYwZy3UDlsMiY6Xgfr+kDFC56j+xiriPZixNZhuVPbSGeI6Cx1WDDWIMYEgAR7fmXdm3n7sE5SENp1s3mcAyMoMKAKOjOooK5NFdiH3TalxkAsU+Q0OZ3LUwtta\\\\\\\/C+L59zwesh+fOlWGT9m2reTXNDQyv3drdDRw9VMoWHhVuRquoHD3e4PdpcJYUSpuqA1pyG\\\\\\\/7VuWG+FuQyVIXyHi9fmhp8bf3TmYT5fw7kvG2CO+OLXkbIzgfnS9sBw4MOoLy1VRJ\\\\\\\/qHDWOFIAYrAOrfc2Rqf+8fazUnr0MrKUJuhsROR\\\\\\\/0ooEvH9kWl+olwHKlj3NFhzVQjJMdHEb994rkgO1kt6Q0Kfsgfv+fdNYzMaHqnnujscjKCk2Wzw+z8ktuA5YLfQ+md7tiaxJW5F68j\\\\\\\/mHSufwR97vThGzmvx5izlL1hFDUUiqIISiUXAfNl83Z9XDRq9NB52sLD2zLJxT7r9Wssu\\\\\\\/s8GoihOmCyC3MY2RR9nGxM6oEd7hjZYdzFyUQFRW6sUj1+Mz8fdHVvwu3RHsYZjjOaWYe6ueXOyb\\\\\\\/djHesOT6kcaiJoEfSaGTE8DcFJGV2dDIejHYOfVyyBN0jtwWKJyJWkyWS\\\\\\\/leSFiLtLmSBSB7Bf4U2t3gqZm4\\\\\\\/o0ojjnjUy8ME8x4EqVvdRl+ZBKJ0C7ykzpWjCsKcEeTPhhlyQKSpzrUmx7\\\\\\\/FbrpmRaqM4+1a9KNdkswQWNmxj53uXymEkvhrsvltXugw4uR1rXG0Ve4QwSpJwzyCQp\\\\\\\/jCUJzlfB2XVk5VxgRLe12ZEXqwAHexKpmHwbYr9GfJiUNocFFOVFYtCvSX8Bfw7WBbR5o+y3l4IoGoaJfAatM9lA4VGLKWLClkGv8DXSEwfZx568D1KxZLOiznB1PLD9Uo4C1uH6xv1mkTCoNH6ECPvSJsbhQcJAWFwduehGG77Bf1YScKaf1Ni5HVe15nEqx7wX1KsX+KC5n2LyTOYqymIoVJycnaCuhPDMljMArZ24pXDi6G9L4WJXGbDWXIo8rXeF05DyJgAQ3UI5b0W1iclTq5+3oCPtRMT0Y8B3VLaaM2d\\\\\\\/rqIm7Edo5R2JKhzxszuH1RtXb2q5P7KNErm3oZTl1xA2lCjl13BVIWdd\\\\\\\/lTHTOFSSvdj5skF8MF3f\\\\\\\/I0dPIkEYSiEyy48HawbUojvfx9D06vT5FeIhGpKBBeOzox+CVUqDMsGom3W64v35fc7B0YWtB4WyrU7c+kJ3FPuJoYPH+bUjtBmDICi65CsviVS92fuD3MW0oSC8tyini0lI6fYP1D+X7KDMuDdS1eOp\\\\\\\/GZDUq9KxUET5Fk9GYqRmKohIlO3KynK8Tk8AZYPW54HWUq83cmDl1jZ5Giirq8Z2DV+PxT03uc7D4He57MKj\\\\\\\/Ty0KyesPLEIDCfV1X029CMzV\\\\\\\/yGialPwAubF0EPsE+UUunQWeeMmL0HHDfjCStAEUBJdwFB4wHPDUHTiTZ\\\\\\\/Au04YWA1zqOIPAHt2CzfKFa8gbRXeCrNfdYBUuOB755fkwL1tgLEqMIqgANYcuSJyvn0dHU77pBLdHgiWc3DRebNTeIUfP0UHWzOivr+4YfMrLYY14li\\\\\\\/KUEg8QYwaID2seJVoun\\\\\\\/1QDMfXbpcFwI4Apolbopgt7EMmd3bVCMJvMcN4e1SRb3k2ejdaIjb0SWtYCYrjaPdxjQ67FGEwbo+LtADZ2pRov9DsJiGVc7ObijkXsmZYmOQXGdWBZuBOGnICePZeHrPJRHREcRVIj65HNiiaRZwc6jQ7eb0NYvTRylREIUPBLjPIfJdCGOf6Y58C5mFxBuOloxWzpNgMgom2EINMF3s5GJA3Do33CHyh4CxjiqKotSf4wDOwz1UwkWJD7a\\\\\\\/ztvF5cnLhEUJO88uBYNKXB8mGqBGeGUOT7u7JD5LOqjCrVRG\\\\\\\/rX3CfQ6EJ5ZLp9WvROWdOeoFBZZ0UeY3B01XCT5BPJzgctFs5BAiZkpiRYxJ+D3gKtYxZ1nuENx\\\\\\\/7Lp9BX0P4tdLWmRYjKfRU+7OcILdZBeso6bRafdayuKplwt7rC4eJ11oifyepZJJj5X6ZRusaIWVvCNKuR1qaMnD1u3xLF6QVEVGptbrTB\\\\\\\/pMLRlcMldqT2i6pDo8YMdzQqhpuS3rE\\\\\\\/O7cL9MWij008yKYcE6sYbCb8DzrxZnWNr6q\\\\\\\/odMadsKnPZDi2BNKfMyxaYte89Dy8Mngn5miBFyWZsZphRAOgwzroUNRF4ThbvTKsaUKVLoNo1qm6d3IjQ4EUtUyCXOZbwIVjFRUzX5liEgI4IK8ea46pi2LlRuvdixAP0U3mIxmgxOLUPO7ivtRJhUOdN8KhTseywFHNQ6PYJdFIY6ewFD5K83y5mhRVsQPwYne+7koxGIed55TmvfDaSv3MYPaZB3z++lt0pyFGG\\\\\\\/QNT7vbzTbrLTao0ibWo44b1hF36Ee+QPaB1KblDrqFJ8T59MbZ9Q1lcbM6Z6ICWyVbHFXoY2l3gvYduz6cS2t\\\\\\\/6XvsPdmA61xh04uWT8b7eZRQQt8co4TxKL89BJqgHNyMCP9hqjedsxGphBmuRal+g34LN1OOG7up6sbFrl5eCswb\\\\\\\/zF38XBYAJGbAkEPDcRJKtXgpOBD4\\\\\\\/G6CKJKk+70Lk3dFAl+seNWFEPKQsWhV4q0BNoV4WFanMd1kLD6cgEAFjDTWiOts8U7V\\\\\\\/ESYnUhceF1IFYOl\\\\\\\/cW45YIbhJAPyII4rWRVgZAf4W669qHjUk4Yw2CEPg2L889oihvrslAC96Kr024xSeARt0F16YP034DifPKf07csms8snpYeanS+8tsxfTefkl1D873sVcehgzZKbkdoiptIBDn79KUKLzf0MmvRmPJD6jt98EzaienbZpPR0vFan6+DhJQKLg+tZig54ctQSlyFbXgD5CfTN7KSl\\\\\\\/MRzjvdt8P80KPFErjKEA6OdaCFLfbytZiWF49wXicYBAcdmOt6A6716Vd0TwdM81ICtwUQWDqE\\\\\\\/QRGllxXGNoW4GmC1qxYAn7tykxQSh7VLv58ghE+PtTCO33MVDs5Z1\\\\\\\/NakgWE4iiws7Ao27gK0\\\\\\\/tPHtJ+jQfNd4s2c3GtRasbfEfuheK\\\\\\\/LC4tyd+J6KHrsWtv4Z+kg+VjncQAHZf5Poy4fVcWrRuI\\\\\\\/NiWO5xY7HDksSM3F7qYnkFM2ePhT9gugLHjd8FdRQ5bNw2vYogZpam1wh16xq3qr7hw43x9ncxiJ230gcjTxGkupsrIsrMMRyKiATblESYF2PZ6BxpZUKzF9mRh9QYkUErhh2RKv\\\\\\\/wWLoGOJTdUUmgAFGa+DL50xr8+ab3dDfQrPEVekndGfbmD2Ck1sjg6HFkW6jifYDvic5ccuz7\\\\\\\/X8KKmzJB\\\\\\\/moUGWdzllEmXClK6P38REWjMMBlf9p3D4TDmJtF7QuGjOysZGNZU2cpjcc8LeZXRyR4ewylhXSLUsPEgY8V+djbt9qKz3MYhlvGD2+IJLW9Ekg4hxzkVlH4DfR1EkCAvsJmtIMMHCP4PQcPjIv\\\\\\\/G2qWpNpEI05mjqqgZYFdBsG1v9SHeVV86wDtCu5u0zI9764hrVkRR5mKShs6QhCXGXfcCqPBi7s6tpzoC052+KTLrgjQNq6pdNykTEib+NikmFRWh1i4+gEmSFvtg7EG3Uvgsl26b\\\\\\\/ww3LGtQPdzfjbKrnSQ1lBb5ignUDK0C+kSQ8jMgth\\\\\\\/wSjiSMjLykYsUdRXoNEn67vAClASWpwXSbyusv+6zGjT0fjbvWV3oRWrJEpFzrSqdzgmmrpu\\\\\\\/dHh\\\\\\\/Dhmzr3hj+dJJrBe9PCP0znJIppn\\\\\\\/nEWrSNQNjbLNbRnDshVA6oMxpOK68PVvT8GY0qG61FLhU24UK2\\\\\\\/GmO23lUAkVCDrnckhM0vx+6Nw+CHr0POYC581g0JQw8bCRaFCe141TCU9lEh1ho4RftuwrB+BCECMg7o3j6zh970UYYKZDbNagUbhwo87lczqI92YZxPSNFWMHvnILowJdt8uhv+d+iGOVa4scduDr8lJ7LFPZmDcmfwylXX0761oUEAU0UmdcmpsDZhz78Z3+hySlPqi1lTzrL+qYlWhIXLBnKbhA47Jcnn0vMlwYqSI772WM7l2KoG5YL7pgB6WUaBz4eL97zPmmFLodCwLhaTr7sEKIA\\\\\\\/jLiujcBuiNteF2j6G5ThuldNeYYaQKSR9cnCcNtD0PI5+NdKlFhJJrT8kvhKtQQErsz79Sz1jPE3ABryZOkWIq\\\\\\\/4jEPSagJui+Elcq35shQL8Vrw8EPiY9eSdeXS4akeTFqJ+xTc\\\\\\\/br+hE6ru5mbf4J316y+0vtKKF0gdcdgljhX9c4zmzItxgNhy8o\\\\\\\/XWjUiJpcZ8KtqT1svX2NqXURMGWe7e97wI4viL3BmWy22s7vRVEzqEB4E4rBa4EB3XemIm7bYprKzQW5RdJzohMxFmmRI8qy6S+ZtSqJ70\\\\\\\/txjLq73vqfbKapr5KySkXTqSTVYEz4hRXKf7E5NS9W6GDPJ3AsdBP8UxicSRMFlnW\\\\\\\/y2DYT47OS75MvT7mQsnAOnkCXJAHB\\\\\\\/\\\\\\\/wk5ez3lNR0THPdCoZkIHv1YZzkr66o8GyDLV9AOA35ZZ9KJQUgCNCZ3oVvmD9WzKyYzjNS2Gqcp7eclJj12fdgYlK1OxD3hKptyEze+tWxPrBepJs81XJTWfKdarz0sUi5jGfyfom39mP2N7s8n4YJYolNipfnjQbfB0IGsIYR7L4\\\\\\\/2EqcLxHIL3Q5VQMxT1vowlAlU1\\\\\\\/tXwNn1\\\\\\\/ViTmeNLUKGOe\\\\\\\/CI\\\\\\\/xUFfAxMr9vWdoR9xEvYTksRiqdKjUZ1rZQ0kKX8bBbH4mFb\\\\\\\/+Nrh13GSJ4\\\\\\\/npCLtiMzeCEXtXA5ssZevBGEwVgxkPHxtT0O71SOqwt4Uyylg6nJVjFNgM8KZCKhSdw2lvp3LHjnFj3aFCGoUrMhQSiMHIsib7j0tGQG9fKj+QLKTGsbYryjgrP\\\\\\\/p6dPdpWbAH8V+0RlJzfNumcxP9Q77XFwyLiPZ6bSx8hKyv18TNk5HnuuFueFQNfH7yDy0eiyzEKYUZ+3DFtp8qXpG1l51ilr1lezoRF43QvldOQr5yn+NlTfWSYB4esqT\\\\\\\/mgPLobdpZ8CIo\\\\\\\/I0AG5JoxZ2bWWqonBf\\\\\\\/tGTWBz3rL9xrvrXwxLTKgdMBdkx159j4cRlpeSWm2Y7hOytg9TTH6w1R1qvpNX0OTG\\\\\\\/eUCVR8DEUT6kuB49i6s0Uh5h8HMk5gjQGZbv6Cx1wRyo3qI3ml7IRq8HfGfFarQmHKVBHrHLaH\\\\\\\/MVBuVXKYCtkz8\\\\\\\/1gMQGw9fm6iISmhevKcd+zklAzDHF9rhvo5nLc53fJVvM51TO7xr8tLWKbCk4eVOw4CEVFHzYtX5uHbcKN\\\\\\\/th\\\\\\\/WvItk1KLkSz7TFUr2ckKUy+C+Kjc+1VtoBqtwp0iYNRhkgeAprK84RWETFduvSmN+74\\\\\\\/eSMPtnYE5pPzlAe6H5DJnWvMG9GAKarHi8nZEfWhGXDTVyZGdXNFH24iH2TFOoppg321vu05pvjrEs2bMQHDmSpfYYWty0iGoOyFcz21ppyJB33+Sz+sT7bU5gzWev5lx20UIe+skycvyQDPQtJi0SP2+VQG9haf6I4xuDPauuPCqybOe8ZChKHxBUu89L+ppOeidWVSVE44MNB6b6vtwhMHhSjg3JeyrwHMlgSSP4ZIjFjHwZCvNQZeaAacKCY1Yyp85BvxqwHqXreIZOGv2FuQmxKV5EE1yCXGZWB7ZMBcnaD8u\\\\\\\/SV9gxRpT3ajHV+g6Q5vwFc0RNDzQSpKXCHfmkjQbpd19ooU+nB+QYKEYerl5kitPkHQrz8IPob27el+XffqxIhQKF3eocA9nG1++gDsG3ynXzIlVJGVrmwzM\\\\\\\/PSDLl9wlaFi66J2LkgW2URhjg\\\\\\\/wZ2xsE+eWOByDej60bwsIJuPC5OID1zp\\\\\\\/KAaGNjE+6RC9Bgnfu02TKsAuPhtgrZVqbiGb+qjP\\\\\\\/ei7MGxskFudqir3ILLdX3lX7rK1zVMr1wNO6VHiW4bl6hlM9n+27nakr5wX4ga6WWZ6K6do2iqiLv10ZzLZ44zrXToISlUO94MKaZNUVWfSsyh2h4qxBXDU5FZeXu+kobm7Amg6K89BMpL0ZZYC6G3ek\\\\\\\/R\\\\\\\/0HZauT9abU7H\\\\\\\/RGarzGh1\\\\\\\/gw1LEH\\\\\\\/KTctuUhmvDibRT38rkdu9rt+cvdqqjvL+\\\\\\\/bWnGlovxdF7sPfT6eY9HUnwpzUk0gcFJ0OqnZjyWmbTeyKqgx4FnAs0Qyy0d7L6t9zjMsYl0M4TUMcH\\\\\\\/wrbWUzgi3vBiGsJEFErmwnPB4NpoHlvoyGVMuNy72bOt43GuCX5pplIxFfemfHkZcA30Z2Z6JJU+mCkv0gzo6HbbsHaiakvydp8U+FOC+OtYTrTu\\\\\\\/tFr+SgTLgpLdKIHx5eJ12JLw3IQ8qKl+hqJisg1ZsVr4tIpJCDpP1cOS1ulVwou5YtuyVSYPmOV9IvfbLj\\\\\\\/gwBKSLTX\\\\\\\/7SpGt2wF1WPjbN+Dkr6athC2OQ7AIfriLOmqpV57zzNfrmX91YkuschMQ6adSr2u+i4pXhWlcOulC+XtmR0bgPekDiLoA2EURIvvaYkCfpNSbknsEEUm\\\\\\\/qNg4BxiqvAKnjBVoljRG4Rp9DijsloGjYbP1wGGz2lp70uoa\\\\\\\/2KOfwV8xlxSkDgi7\\\\\\\/V2AOw9TRCJEWfFmHTY3qXpfUk+ohZ1eSb2vYfwUACJnzCEx10nlKSPIeBHkluRz86Xv5\\\\\\\/3qNnBDc3kng7ftGxf\\\\\\\/9sNoyykxng5dfaNFzHijN5b1t8wguTpswU5Dkjm\\\\\\\/TZydakDXbdC5vTj1JQ8he+kRZsYqzOj88IzRx5YaBniYvyOpnv2kF86p2tIMFypEpPDlqnAuQ93zIOEn7FRek\\\\\\\/P4VG35MWxy0TLoq9LUS+o2xCyKGs4HTGtcoa6auCR+oIUPt9D2wXt05J99bsr03tEg\\\\\\\/6Sy0bcPuWikxEpAAXZkriM5JYfd0q1MxFdU4voMB53NeeUUM1hr4dPt+v1GfbjCvnd7s5ADwwQExsyZ1e1bKHN+MAnyxzIYzIklhULF02LtHN9tj12J7+fGRAN\\\\\\\/qcjse\\\\\\\/wKAtuuupTMkT1PhT1KW5ep\\\\\\\/kBRcrivWgHVWt+TB7MyaKwIW4B5B5LKjHdXBRYRPhDsnPtqSr0sldHAzsmmq1ZHSVCJcXTsfpiHJp283\\\\\\\/\\\\\\\/fUUaSrOosGaYNCWNRWwQGsy6f8TAofaZXI1HUX4LiLrqGHSv9vbCA\\\\\\\/L+RgskYH8YP266rQw5D0i0WIOk2oFqR57vPIxUIyoU6ms5a+6U8Ki6tDyXgMukxEzyiOJMrtpQxVqTcB0Vtzp9t1Fz1Xdk74J2RnGGle4KE1iGG2dS3MK2i4peOvDFrd6zzb+WJ+I6vBuLD2svdfLkiO1i5TGmJtM\\\\\\\/viA6pq0+m\\\\\\\/a09FCvuonGE3VchISwWesKQBdLE\\\\\\\/NgA1SIkilHDNfO8CCe+ECTUk5kFOh1JokSK+oMAwBOwK11IXzGCTHJqiIQBp3HgCc3hgp+WeHHLMDuT+U2IeTvoLlOPMXJw1+hZT9R8ZeI4T7xCbcX+gyhUTTo3qUTmzM\\\\\\\/M2d822rsXA+yBi0YclUr+oZ5jzoQrvesJsYDS4IRsgjSSLoW391\\\\\\\/X2Q+jINfy8jPlww30RuMO7rJC5G5pJayY86eVm2NivNMOJUOE5TMAuKfVai7Z35rRu02aLTa5+k8PCHvibA\\\\\\\/LUaP8ptGt0wQVpsbrA\\\\\\\/BAPR64CbQebJTegA\\\\\\\/3dccR03wARnnM5UBjLoArT5mddtoU6g8qIz87WvSceIvUglcrJu4vWef\\\\\\\/m+EozibJVkMQNGP59auPd4IMlXXLBHDQX4Bmdbt3thVe3g1ZMQJhxqrljEgnTGpK6SKbqNETtvE6oQw6SwBFJ2Ub4\\\\\\\/nI0odSaaqk5DCv9M25KiOJ26UaWP68rvFW0ID3yjZ9RyYzNFVJp2+6Q5Z7COdcmX\\\\\\\/UD9cg3SRFi9Dx3PveGcDZ0qZO6S62KBHAWV0HTVRvAEUdcFezNKJBmA1uBChxJ4cnUrhG8xfTAOa4srVoHbgzKUgNhpVErAMarm1xtISe7siva4VTZGJOvTred5HTprP8+LsuNJ2xpqMcYck8k0tdT72McRDXCBaqf6U7icX5L8J\\\\\\\/J0dtK1ibE9nVxowiGzBnbCxTTr7a+vYSI9fJ2RWtNUbjcZ29s+zuKAx6sWstbNSYNN\\\\\\\/b9ND\\\\\\\/mtJC+hoDD7QX3HvvdvxOZ4cEDaYwqAUGWznkytuqFnlVKNiNcK1j8zQfUmGFNb0z9fEXx\\\\\\\/oudLScf896FLGGFQmz3ZtlGIwEL1J81zdTADLAlyLCjcHtlMGqOQKZh1lALDaRBJAvlFE1B7sJJOVB4UWTFU8KTUkAoDo1cP83BNztpfT4IX1TgqjocbEjSYVyLE4vI3XMF5sTskjhoRTRQym72+I2qpnesoN\\\\\\\/Ugu2cdkZ1cqXu5jXlUnnK4DLc1KfmElSBotzZt+NDBRzDBg3j5+rQECZduqGL5dCg1ffe\\\\\\\/\\\\\\\/0xCeWwZ1S6gd+aBpRIlJdSks4Sw1NboMV+Y2h2Z68GXbc+9fkS3+E4bLRkDEaHtB2mO5fm1S1kP94MJEG2zj2NMru6HDtA4IhWbtVb038BdBdrbaoWl43zOCsJEuoM58XHc+YRagf6RcQoPmun+uV0VvvUXiVHem1hZv1N+b\\\\\\\/v7u9DDde2+Dwi6DCl7dzkNfhB\\\\\\\/AthDSjywmWaV2LifJ9kD8SPUpKKU5rEgOBSrADhZHTJErTxXMsbiygJHbC6dyEu5SKFl0TbxpfmEjDKX0Tv\\\\\\\/ItVuVSZ4l+r5uj9f5FwJjGCQoAAge0\\\\\\\/51lK\\\\\\\/Aag6kVW6BIDT0Ox\\\\\\\/hK+28pScf6ZoJv7GG8Wf+\\\\\\\/CL4DW5ogf9h6PVp81PvoRgQu1Q1M\\\\\\\/PgXKt1jrpzxgMNylRpzFO8J0H8u2ol0jgaYt3Lba06vZ5\\\\\\\/C1P0hNUtOGg0X02MvCWpqytojtUTzTq38hAdtOx2PLWykfJ6zeyz0CuiCY3aogZ3WcEx5e+A8G+hzMYlc8Lvw7rsN3AZs\\\\\\\/APwfagnwlFad4vebgVlB+WyamBtOfcWLxyLit50qWMZn9QN6T4RKJM2Tdsc8rRptQvGB3Ijgu7oJNsc9HX2bc1nImZF0IvdskIbGCRV9B0gNEIp4oxVZsDL2O9VsV8k5lTxf8cF13udjBOyrCvbc588N6dboSRu\\\\\\\/sKeXtV1YXZmTjulNxtmgVsr0xIfW9LLouHrZ6C5cKr33ZADc1iR38RrqlV+AcsYWA5DAt8MUrXZvsdTZE0C9xcxLX8sP6b6guKT0lVw12p\\\\\\\/B88WOtpQESMP5r5iGgIIQ7ZoTRdtnqJIpUVzIQgMzsft\\\\\\\/8IqNHoInD+LaD4nDwG4IW\\\\\\\/OeOWuZxUP9FsMNPFH9xDw3PLW7zqhVX3ZKNfprRI+GPfABZMVAYB0a7IsnGIspCN0O5bm\\\\\\\/PtCENS0kW5NzYVhOoiizpQSfuwzjmKf91RqZdApA06z7FvUwbUrbdCMhpU4EQZG7hkun9fmu+0hhWfdWzclXbe1CpvdMHnJxhpqbhY6i\\\\\\\/SENVCd2+1QU3jY2CHuRAb1jmYwYV+6g5xU2VCKbnna\\\\\\\/HHOEC52Kgx\\\\\\\/KBADROHR4wCeXr3PICZ10L8RgWV0cIQ6yXPd27p8997LPgJkC6p1je2RsDNDj4cEJhERqAoFmjKSeP7xD2TS6Wru9\\\\\\\/vVA6kxxMvzwLXAOagHCiobi9C27zN6r9xhW2oQNSfjs3vz6sYHGb23RYk4hDiaxO\\\\\\\/TKBqwNvwVhT\\\\\\\/s7w5fMpymDKFk1hQvthaOhQk4Tk1XPhPYEf8Vj+Uqo7IvGDbTmS6KMrlOx03\\\\\\\/V46QahKHNx+th2rVp1cgYFpHeyUO+Ahu5VtdF3BuxSzkit9XbdpAECFDN8fjHiOW2Cc4gS5fGrgYzwWZgxEQKSd+o1aWsumHoIWqXyCpOvGh5H1lWm2FMaQFNkjwzLspZuPGzstKBFfUg3VYaSgxfABLQWD3nvBdGoSYkxMkrYO5XKJXyv5KZqykboGdElnTeVSEqO1CA\\\\\\\/bRmOsbXTW9MYqoSwzc4Y6FeZ3Hchf3NJIysFgQ3VwJ12FqPRwJkCczjjoJn4yr8w0szkbqI0WWGiml0X1iDriGM6Vo\\\\\\\/ryx8uIcuWWcqI9j4qZF+hOvbEK6tsuDGQ9WhzGgFPMBHD+Li6qGi+31ooB+Bh1ouPSyA9o4NvCKA0AxVOcOSsiUjhLFeLwHC5Lx7E\\\\\\\/KElwU\\\\\\\/JHsl3xJPbdHdqS7iSCuwOLNKQt\\\\\\\/Xr6x1sg+v6QXVvx1stqMRvs0BJ+huHh\\\\\\\/KfXpzjNMxWgCQBLADASQpJS1o7AxFmTY96sirg1H\\\\\\\/gaYx2zxfJQNxBg2+UtllPO\\\\\\\/Pi6FQtCgP+lcW9rS\\\\\\\/odfBdZc36esfjpSznkZ6NIlBp+MA4DlRWypElcXIcmt6OdybjxlQRm+Q9YI91XJ5tuP9ZcqZd5urBwuVItVDqLp+pKa3eBIAbLeSlJDH\\\\\\\/YQPHI48Vxim\\\\\\\/dSdZZvuTC9HOF+3FDZlG1Q\\\\\\\/myc5S4Exz5NHG1lRmjaA+bFN9\\\\\\\/9lxLQuba7mgS+jch0dm5pgJ8gLPqbEj9E6L\\\\\\\/T41JN4BxJJL4CbYT\\\\\\\/38gC60rPe9OltxRB+GI4RfjDGlBCF\\\\\\\/7Q8JRJfaB+Yv0j\\\\\\\/tsYEFay76tkgHMrpibDwgZxE0ZcrGOtkE2DIOv0IhS3XRv\\\\\\\/iCmk\\\\\\\/0PQHLX9ZtIrH8tsxGu\\\\\\\/vGIyvLfuUNeR4GXDP+mYMOIWObQOUxhYYUN2\\\\\\\/S6YHbyhVsUt1\\\\\\\/M4XxFp3LOhQgZ1nCTsirR\\\\\\\/KQdTYBap2kr0tzFVs+W491hcOkdwRm198yDs6QrVFfwPQWJQ0ZquWbfkt8OAq3E2OBIruhzUeHKlEzaWb4eIP+krwEwOwKWL6CMg9Hhh7BTT\\\\\\\/Vyjc1xNBR2A++d0GydEvFEjb4JpnoH0Mx+pjlt90EMKvEU9f+KPx\\\\\\\/x3nUelFwJunfgMUj3fSRcT95Kez6DgpgkqPjuRPFWrKVYHLFG0D8\\\\\\\/K0fylcqlJnXqKYHTpMqw0Dt2c\\\\\\\/F6FVwSk1kJ6RH6zZvsDMNCLa8bUI\\\\\\\/kKBmYOGRHHhoikq00knSwfEHRBQd7G\\\\\\\/CCHUDf\\\\\\\/gzERRk9dPQVQQXR7B6a5dubhCuGCEd8RGfw2OFXWQ4kH\\\\\\\/tMNix6ydFyNc1oGncxF4V232sMUop6XQoo578qrUIjOw6J\\\\\\\/x9b6gaFef00y099CrVlcAMcrouMGxrVOEc\\\\\\\/Klh52JKdpM6cWIcI26Eq7iF4cIciI+Ip6QoH5rz\\\\\\\/3uUW8PIjSsiJHIjx80DYToTD1M0kTw\\\\\\\/fySRugpwkGSICRB3PndciC0bqDsEGNP89tuw+vfUxgCAtDehYAREAlqTPw1XUIi3AG1hzaoDkLzbkml+vDgF+JZOw+ZEZ+tRi9d7aSynAxKUHk67Folb2YG6TR29AaKbLF8jWCN77qf2sDOu+pYxMc40nHN6\\\\\\\/F3A4fYu+bpHQWkZmne1rulZT68k7lIc8r3N\\\\\\\/EYQ1YWbpsx9LoK3ottgaBLMPgCjNpl8GunGB7wOfFOewCfeNXotrkQHrOw20opO8X7ifKk3lXUKUfTIvzhF1iI5+alAE65VJe+JHaTuhP\\\\\\\/XE8HFE8\\\\\\\/PRehOi3hniIeTC67dSjWuu3djuujCGTLEXVTgQef24MoIXwRbmVs5AtQsmbko0yBMPidVWwZZDw0Mw\\\\\\\/ES7BO1SzW5lAos440P63vXn0+Z3k9Y2AicT1hl308X4N15NW9uMbguMQg3701QbcSO\\\\\\\/RVdq4tJsmfyEOeiyyTBKT+u8FqXNe6BnofUNWH0o89FsbH8FlJruCVenR3cEaxQMjjxSOEVhkEg7RNf8BFknmoqyUUC\\\\\\\/\\\\\\\/EoF0ELQ\\\\\\\/7naEJIzig7gTCfBO5POpgW6roscuVnmLfuieaAm4igXASPwBbT2Mttxri+dy8rjZURyD+eQgcRZfGJe0BewbYLyclx65bTS4e1M+NhBHE24mBEiQmvJm3nP9wDic94H6hmmuCxGZtrzdRevBEMtQX1v716Hl0Xcvlimmdrr8Ax\\\\\\\/a3WD5iTJFSLZTqM5bWTJrJlmx3MiOjETZnxWQKBjNSvGxS7I0Nxih3OqXQyrFcS2tWast4aeGL8HL3wqTaq\\\\\\\/iyFds3qjXwSB8I5veAziOm3V7zGLETH5azmX8O8cDbM0bFwsAj4wmgguv3+yc1iiw6tX+pPSj32WDKIhPjY+jg1VnjteSOpPbaRgk51xA5HcLroA\\\\\\\/QQAeYIwKCs\\\\\\\/9BHQ+ZukFSiCYylJ6aU01k8ZyihutUZ2E0eUWKJyEapiEdB2qLJleKOMjPSRBO6pXKbsxbQtqbP2s5c92SWxvZXatO7sm\\\\\\\/5HBmphYJWSjIWO8H2zoDxd2VPY+Pjij2u0WrVA3cXoNE6P42Yg6annDtNDpBZitd1gRQm7M14e\\\\\\\/mcZTAbi8QYh3hscBoIF+30r1nKXetuuCzSaWi\\\\\\\/LqzLuWBVebKv9mfHf+dNmol00gNKjQLDWYjuLoojLj5Mo0sXt767PBsmtsrtcsQCw1v\\\\\\\/ygiQFGO\\\\\\\/B1TOH0AsCsawr7JGPjbjEiMMMseQ7GiSW5t8j+5oLhx4LrEGz04qV\\\\\\\/DNFv9u9UlJ8yuhhhHM+4VMQt3XIsxJ39fVphCeb9dZuQNs74JcntE7azio0nss40poAT2Wsvyq8qmr7Ziaswnn4KLr9shh9njmLCj+8jC3lQjemZJQePMZrhhgdUnfvZxF2xIrZkE0rw\\\\\\\/iSayX0ZEONs7bXFFIdP2UKsoQw+1DES6HTADCN72FIgLD8yhnXatBhAib1oBzwBf0HdOElzDTTZNQV\\\\\\\/jeYLXcQ67B2EtvZYRCnPzzlw3oFo7shz2CnhrsvsF5xgpIIbRjDPfy2XhHkUOP4i5LpORZy7qzZ+Zr0YPOi61EI9NU0hUCbHi0LDIBrFmV8hh4Sprt\\\\\\\/1tJWr230rhZW6kQ+IThWFXKX3bkJQTU5LLKRUEm4igkukmliSAGQxGWFjQYzo3L\\\\\\\/f54b3+GeZJAaJUV\\\\\\\/1tApPmdM\\\\\\\/YuuBkZTvRH7jaxDzpmp9TkAgAWhPuwBVCofsGAjsRJDnPvEfDebISb6VEnyyo6aM1D0MiwU2t0\\\\\\\/v11aOmVOj4bMfY6O1HDQ\\\\\\\/vpfz5qAWFuy6oFcBwMebn8NqbzUJfDXSSqWrh782pstBkfhIyVtwBD12IdztpIWf8E+IPb1fMEjpYJiFIcC668i4X7e+3puruRDLOUuZXnkUBY36Ar+cP75guaZ4vG3G66JcHHO85TqBDRZFCtwlAF873RKUbkxZ3ugTE8KuCvn62IiFBc8jmHrY+fsRDNqZoZgRTwUuhbL8pKZQrQikaBwZcusjeM3RthJ6LXp7Cj+ox8ve2mclYXEu1yio7TENOBiD71B4MjNDa+nqFtH7PczB1UHcCsx+MlxpIaFrXMfMlZnTaBtqUu\\\\\\\/ZOAEEFk181Yjj+sss3nCnQnARWLaL1FlnanYpXiFbJYwAz0qv9dA2uNTSXs3WJBIvFQv3tzZ3H7aPo2RAOe87lf58h5PKRfLTMOcq3iLEF32EMge22l+qjo8qtL+mvfUhRca3OhUwmduG6V2g\\\\\\\/pFMntHfmzEQvA0xggJet4O+I9ABMM7vNqG5Ui5fDCHyUx5TdCIieASt5CCbvTFt5VtCMzOS13FAkzrYHUfcUrWpwwKaWB\\\\\\\/PnRPCbdZJsflg9BLGnIj5A7b7cXciQUNiwT8Ywww8\\\\\\\/9GBTzh\\\\\\\/5rhwboa1aiJZwhzmALX3N2VcCYb1cRV5J97IDvq4y1jV3as9axF3aCwFJUIV3uwgkZh7M2\\\\\\\/0ycJXthEA7oj3oxvv0riekqAjvY6bF8knLr08vkOsFfRJVaPgEQG8yqmYzDfPO\\\\\\\/MWrlstzdSYV\\\\\\\/6VhfDCoo9VSze6wwl\\\\\\\/aheuZDI3ZICWlLxppGq435lwvCnfqpOJR869OreAJh\\\\\\\/2HpoRcWY1degKh7pZKBWcct8sR154bixsIH\\\\\\\/xp9G\\\\\\\/9ZxoJpOKWgbHI6EIGqMRHkXR1w6UbYM3pGP9XNf0VRltD9eEOEf49+VAHOvKDIxp7lQbniEBQ9LCrNjBMqe7Yjl\\\\\\\/ieXE+yoDH+2PpfE4v\\\\\\\/NOuMrKhl9Igo3wmJHmf8p49\\\\\\\/ALF5TazpCFtwb0UIIxM4996Fr8fceUAIbcPYcUnuDuHbaA6\\\\\\\/RKRkF6g99k5W6rntowZNUqM9jL5UHYq8o3FsYnjRFdeB3Oi2CU58w\\\\\\\/KmPYFJRvqys7YrMRwoKCfXo8uWyMG0PSFWvqrRnBrA6ht\\\\\\\/7XKG\\\\\\\/Q7HjeQeNH6zNJmCv\\\\\\\/iYedThHRQHEj6y85u1t7ztlGQkeCJi3Kx3RO2bJb8gzhZjfNGDrXPiK\\\\\\\/N6KbeRSPE4NPfFRwF0I+0tLPouCLkqjuW9DzZSaMhv3J7gn222RoCrLXdQg1tJd99+di7gPd+JWHvC41lsICjJDQcyDCX96BWN6bDklhFAlRNQ2gXXD8yuGku2hFFLm7awrYJQSpvrndBODUGYgDJPf2oOg3N8CXpELwvm6lHIssWQEIbGz7Lk7N\\\\\\\/uNIJf+jwiUuR50G+bV9Ada8pdNcAS\\\\\\\/HOy\\\\\\\/5wTpSQOdc2zWRBXYlztJ0c+twewlDbSrA5o1LlxsmsZ0cThUbjppWDXzfXghnEz0AcpWjHsitg1FHQIufz2YxIpGTW4T2UM9+iQQDGgwOo8O6iGJGNQVTpysv5rohfviaeTrWBl4UCefQUdbB5a7QLZ3jdwr9UP3lxdUDKqPyZ8rmDr27QeF5gseAeBOPgimzYUxcVZ\\\\\\\/iNHvVgBHiYPPWGAriQJNCTJo+biCOFb+HxA0G6Yk0r+yU1iTAkwGkPzC2Quk\\\\\\\/ibME12qtztWCgwxvWNE8\\\\\\\/kk0A3gEgA4Op04vumvyjvVhJts+BT6NPuSEwRLMu2EYQZdrc9O2mWVfGK8p0Q6ngUeYbG75ZKVJFL0e\\\\\\\/4BGVqd5IpxFp1fV0PUonhGRIWs8ZZQ+oc7M4yA4Wd7nYBJtosbQWQeipsb0Sl6BMqV0cl+z0oRlQueVGV+t9kFlxmriFsBPZR7u0xLJ3fxjzUZtrQri\\\\\\\/25AZwq9FFrjtPptS5P\\\\\\\/sIKEnPKWaoYrzwquOCArpzwSTH92D+c6L7WQyA+gLQvtgt9CJvzw12vyjTt+Ht2u8xz\\\\\\\/9ZDS\\\\\\\/Rh\\\\\\\/fajzn+1tKvdM2OlhMaElmFhuFPx8RBtmaAlZshBA2Bf9t+OiL7SEAPAYyGdrd\\\\\\\/SRaxfWIhmFLho4bm5FLZ52Ptiqojy9AyBDumUoDAuv7we9+43bfrZ+qJIrjVhq2kkyEHvaB+IAtEYctxLV1LVoVEj2EEMhf85jKle3qCCSFOhPVazVtBNZt3BukovvhIj+nap9jiCQyNZY+8g9Wgbce5shkHaOhYeN40P7y1Gv+ouNO+R32J3KXh\\\\\\\/qnvGgqBsT04IcXr0v9wari\\\\\\\/njJV8JcSrRc\\\\\\\/isdfu7hag1TDeT5AKb1xL\\\\\\\/vS6X2uIX3DxFbR7OUoCY1gEA\\\\\\\/cJpWkGfAHA0\\\\\\\/JgdJT9zaBfjmhtOWEuuiOCWqghTEG6eRjT6x2dxfIWwnVatPP1ocnxno1zEDQTkaMsjD8e5SEp5nDWOuel1YzRc3h+MulJrBPn48zPn5LmALVkeL\\\\\\\/5mEW6SWd8tPN39VfVQ5ScSSVUvoLy5aaBCcVQflAGju+lcTzthOtsqT6q+CrGHQEJU6p00F8MToSgbHTjchnSvQftJkJ\\\\\\\/wzrphXu\\\\\\\/zoop5Ihy4ZrTqUGv8l0OY+lcFgn5Yk+rv95Fq876Xiv1X\\\\\\\/fshvtDiEEW326U7ZMassrxei4gGH6Lk5wBac3mn3SQRn5DsZh+J+KrUKiaGpOl8TgSe4s6dLex3vu8Ibue2zOSAzeORQf4pFR4xjtfU\\\\\\\/vIs2XvbB6Mnot0DzqVu30ZpfxNRPBnuD97TqWlOvtO8EXxGVqxJc+yfcrRQEdR5dkHsjmQMJZ7ZEkBKgKLI4MXgitAvRg\\\\\\\/xOdyk0NtAdo59NZGymlxdhpSzKIQXnYT4hMl6y4z5veJ06DIuK1OmejIGYhioK7FY7voK\\\\\\\/6EQMMwOyimCV22gGz2KGlD5MFG2O97CJA3+qeOmT0ROAomYVnPxpsy\\\\\\\/cAx9Pmw7zfA5+c69uZX3fj3qJU5acDkvcEgO3UiEkOKWKeGuCUbZywa1MckW6SF08pJkEylqXklsjJhbA6kWfGNoCJbPqRL\\\\\\\/osMLKTMmlOLJkcRKYrh3JC4+yt2hto7U5B668vw2N2s+tF+dA0aKavd5sugT\\\\\\\/9muhRWwVTOQZlVsjq6RDDnfJOYKaXON+BmSBltBZsExmyuZwUaUpoYf8YLO7+EgWdHuiCCn8a9QrMQho13efyPEgH9vKvX9zUsYQgvYRlVOVqjbV6CovBXCljzvsdAZVPkbAEshmCKSitzgpl0EOJ5PUaSfddm287B7DrwIpDfuKqTG3qEr+U5XfugmuAfZTf\\\\\\\/TAyPfHdiNUjPVydkmAihb12lJ4+CoQK2DHC\\\\\\\/nm2lcJj69OV4us4yo5HKKrNV\\\\\\\/cQYn7eDkLT\\\\\\\/KXnUFPZSNF1WzWESYIe9ixGHAPfjnbrsIZACxJpZkDZ4GapsBOMOHv\\\\\\\/7FrlDeQBfEjEkxsu\\\\\\\/SnmuGbeTTvZNplvEWpAGRGHM\\\\\\\/mrnSMxEzHM4CMsaYLRimpq8Ya\\\\\\\/VDsqs9cRTVPNxLkR+ahtIIf\\\\\\\/XLt73ZUhlSkZBz6FTOfP+YHgZZO37fLWalRI3lWsKoyMrmU7xAJD8HA88Mnfx7mYS\\\\\\\/u3e9ZHxvpWVmPHVV3axXbOsS921C5EsOfCoHcfwR5Rp9GKAv4QFutqWp9SHLR58\\\\\\\/F28AmZVmBZ5LJ2DTbuI7MHtXEklR8Mps\\\\\\\/gbw+Vw5Vf9BLOcSi0myfN96Ncj5WCEnIb1UkpVyvqbtmafswVwCO5xz4wHyc6+45KmUM2MIDwild+GWjXSLJaUUcYc6bCRS6xwAragXqW+GqXOeFXdpWJDyDhv5ovfA6ymFIwWth8VuYFVRS8URT6vMUtc6oIg7kdYTsy5N3ZfH8AxND40EAOdI5mqtugJbQYZ0YY+hyus7NQxXo2AdVCBAgB0kn1eeM7gr64524aZeoq54sOiOGQfeB\\\\\\\/z2zQ6KwYwF8aQ1XXjIXVvmMhcJ+Pctq6qHTlh0iuwnDTotepENTjYedEWjFP7fN2WOYXU08ElDco6JUrmEiQGa\\\\\\\/N+YkcL2m9p9krwZaXc4zP4rx38oiY3ZcW03lJ60c9EUNEC\\\\\\\/hb8xLxtWin\\\\\\\/3cTj1sbzm\\\\\\\/MfLlaWSC1mVVNTfNdn88IS4PEJF\\\\\\\/zARJlqiasNpVWhc6WwF\\\\\\\/MayQB+JafZzlrhbAOpasjimcefKJLgsD0rx9ayRwCUjTbimgKHTUJLav9qZqnFqDjD2VWcjVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xeoinCubjUiXOk8rxwIrlA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022814a0c6847af19944d2bf6a02831b07bdfc4e83f4dcba87d501ca1013e2c80f9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tpQ4rmcSpxeR2o+fH9yQYXtCjUYyLTutfhB4gIyWhvXhNBSxrb8e1zrrQ9YIYyv8PfLj17tXuaN68JdQoLne9p9Xl1IyGCihfmhXAmyXDhZ\\\\\\\/MiwH5UVkqro3ST0xKnb37Z2NXhAuPaYJKKuI3a+7j3XlPtsYojqae6b6wm76OUbYQve6kZAxgZNGS0w8ru3t7423qoSOoCp366ii7+VnLnbPLWiSXeUrhKckb51EDkPHVqgl5ssVLqDqGk2KM8bXC53b2ncQ4Y5e4qKb4pP4FF12QnUiK4P70ASH5tZ8gv3hcmlipMRiRXq1HUwWhs+pm8fp0bHMfj7gefYRVtHy6IySCbb2xIxYaijO\\\\\\\/ompNVbliszePwanYmWuXXCxXyy4oUoAk9KdU1FIAcK7x9HJ+hwhXBjRRc73iDF2xpFSeXRnVAX\\\\\\\/OodXnLrYA0JcyvU8p9ozXkCx235LGt\\\\\\\/7T9AeQzpWo8\\\\\\\/u6cNkhuwpdFVJiGZ1VdjGmkCM1britqsvJPGCPG4Hbv\\\\\\\/164uaJI+W3t35QBdR0b5xdtjYh3zBUZIeRHjso5ixdRfz\\\\\\\/kA3dytEHTNx2n2cjtbu7DDpsnvUDbK\\\\\\\/lNU1bz2hMGaTDYjh7S7jSVQdws9gA+1Fb7oP2u4ccYDGi0y7BibueK7J0xgbrm+HwGjiUx\\\\\\\/dDNzutAxxGTbo4HQldlWhDShuSXRX1HC8ZCb7aFWC\\\\\\\/WabzQgSCyS9KAUwrdplsOGd6RIptR6yE1wcZVUayRl8Cp7ih1e64E4hiqSCpSmVK9ebJo9sIxg8g+FEuL01T39rchzyksHf0\\\\\\\/+BPYwp0VG6pgMickecE+LHFU8FWG7bVKAbXrAUlG2Y5k8PcoFnHifXfcRfYO2WoGqHm+w+VvDzkEHd0reNfzgpJYWvK1N+rnVYyU\\\\\\\/zWAm3LegLrWTUAUYawPkkP6fW\\\\\\\/alRUFF5+RYXBzCYKaIUMsgRj2FyX23ZgM5xFuGspbj26kGziksR24LgPIeEhUiBuq4mOWO8sDxDiYjmAXH0rQLCjF8ZaMSN6V59fAoJJp2OBiQWd2xr0Yci8BPRpR0kkwVWBDlNyjr8JBDfT3n61IL5YpiLkbVqVa6siNpUC8y7CkT5kOHq4atb3AvIiwDogIo7rGWteVGEWc1sLiP2PdjBkqVzYXnUEsIt4+poo2FcEyuTpArvmUg4iANNgohujk6i6nke\\\\\\\/ycDG4+SXAfj8+bUVmLDcik3H0SeHLUN38FXKnwmVKb6vA9IPQ+4rP09DRBhPl8TBxmEqmEZ8HaobhNuP76Rai67pg0afoon4pBoEr5SI98kPu8b4D9BZgn9MzoxP0Gq\\\\\\\/aNgYe6Sn5K6aZxDn4T2qlsGEhXfjpOCtFMyLlzPmpS19AB6iXPQFmD\\\\\\\/9LIQEKO+VvsnguGT1qVqnlSbUK3I2rBF\\\\\\\/4yjujrKLIGi8AcKKT9k5A6OSITHoUwBGdIECu2qYwi3msEfVigCcRnZ\\\\\\\/HcMQpu6oFKUXwHejzKA6jpTsQSDUi+up5O\\\\\\\/aihB96RhglfladGN18O\\\\\\\/MW+Top7jkvf+3Am9MbnPZnMOhinNFTS8TxOoervyIuXconTejHdFjpOwHKK0AvQFv7Qg4a0zOWDgIvFJgVRLGuSD9aRe1Rlk\\\\\\\/FWKUxutfZR7jGV8WZ8vCn9bSnK7zIiabxM8xueCyfh3aHBUWnGq9v+i7GuDJ10WXOUySubSCVhEi7GPgNDXxLNk8ypRPT4eHj5LOHCRTzzhdIPGW4UfmIDlnqpRxPFL+9Py3f4oDPsee47J1B1P8ztJJMTOyhPI9TBsU\\\\\\\/Y0S4092Kw0G8PMCJ4B9Msp1317KXV6L6c0AZQas7njz3BWl1AtgQEQywWkJdeOUqUKx7Mn9UPltj0qmll4aTQYJYGs+EzuYEY\\\\\\\/XtvL4YQ\\\\\\\/WchnDkjZTDc\\\\\\\/7RFglRJEfFMMW8MTs3F2xYSVlFrAAKLtwW\\\\\\\/f7KvVJr+B+OOHgmVEThE\\\\\\\/LktEfhfSEeXlgQqXlLmP9R+vKUTwiTRHfrMDAsNiCPXS+g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aLPO7FOctxC9FBztilb+pg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022354e76c21e71dcdfba00359ed9858eb29bee8a2eaf28c20a2e46ac7b8e49c953\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460496 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3941" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-s7KnCYnbPhPzujERQjSEjL1iDNgek1wN-V072JRE0cU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3941" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VELbByZA8wIao4Czh1rlwPDirBDA9xQHDLVdO8_J61s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3927" title="Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3951" title="Characterization, Expression, and Functional Analysis of a Novel NAC Gene Associated with Resistance to Verticillium Wilt and Abiotic Stress in Cotton" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460496" id="node460496" data-pisa="ggg;6/12/3941" data-pisa-master="ggg;g3.116.035485" data-apath="/ggg/6/12/3941.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">On Robust Association Testing for Quantitative Traits and Rare Variants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Peng Wei</span>, <span class="highwire-citation-author" data-delta="1">Ying Cao</span>, <span class="highwire-citation-author" data-delta="2">Yiwei Zhang</span>, <span class="highwire-citation-author" data-delta="3">Zhiyuan Xu</span>, <span class="highwire-citation-author" data-delta="4">Il-Youp Kwak</span>, <span class="highwire-citation-author" data-delta="5">Eric Boerwinkle</span> and <span class="highwire-citation-author" data-delta="6">Wei Pan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3941-3950;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035485 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460496" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peng Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pwei2@mdanderson.org" class="" data-icon-position="" data-hide-link-title="0">pwei2@mdanderson.org</a>
<a href="mailto:weip@biostat.umn.edu" class="" data-icon-position="" data-hide-link-title="0">weip@biostat.umn.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ying Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yiwei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yiwei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYiwei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhiyuan Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyuan%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiyuan%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Il-Youp Kwak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Il-Youp%2BKwak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwak%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIl-Youp%2BKwak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Eric Boerwinkle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BBoerwinkle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boerwinkle%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BBoerwinkle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Wei Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pwei2@mdanderson.org" class="" data-icon-position="" data-hide-link-title="0">pwei2@mdanderson.org</a>
<a href="mailto:weip@biostat.umn.edu" class="" data-icon-position="" data-hide-link-title="0">weip@biostat.umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/12/3941" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460496" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460496/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3941.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460496" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460496/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3941.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460496" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460496/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3941.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460496" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460496/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/12/3941.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460496" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460496/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">With the advance of sequencing technologies, it has become a routine practice to test for association between a quantitative trait and a set of rare variants (RVs). While a number of RV association tests have been proposed, there is a dearth of studies on the robustness of RV association testing for nonnormal distributed traits, <em>e.g.</em>, due to skewness, which is ubiquitous in cohort studies. By extensive simulations, we demonstrate that commonly used RV tests, including sequence kernel association test (SKAT) and optimal unified SKAT (SKAT-O), are not robust to heavy-tailed or right-skewed trait distributions with inflated type I error rates; in contrast, the adaptive sum of powered score (aSPU) test is much more robust. Here we further propose a robust version of the aSPU test, called aSPUr. We conduct extensive simulations to evaluate the power of the tests, finding that for a larger number of RVs, aSPU is often more powerful than SKAT and SKAT-O, owing to its high data-adaptivity. We also compare different tests by conducting association analysis of triglyceride levels using the NHLBI ESP whole-exome sequencing data. The QQ plots for SKAT and SKAT-O were severely inflated (<em>λ</em> = 1.89 and 1.78, respectively), while those for aSPU and aSPUr behaved normally. Due to its relatively high robustness to outliers and high power of the aSPU test, we recommend its use complementary to SKAT and SKAT-O. If there is evidence of inflated type I error rate from the aSPU test, we would recommend the use of the more robust, but less powerful, aSPUr test.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/skat" class="hw-term hw-article-keyword hw-article-keyword-skat" rel="nofollow">SKAT</a></li><li class="kwd"><a href="/keyword/associate-testing" class="hw-term hw-article-keyword hw-article-keyword-associate-testing" rel="nofollow">associate testing</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li><li class="kwd"><a href="/keyword/rare-variants" class="hw-term hw-article-keyword hw-article-keyword-rare-variants" rel="nofollow">rare variants</a></li><li class="kwd"><a href="/keyword/robustness" class="hw-term hw-article-keyword hw-article-keyword-robustness" rel="nofollow">robustness</a></li></ul><p id="p-3">Thanks to the rapidly decreasing cost of the next-generation sequencing (NGS) technology, whole-exome sequencing (WES) and whole-genome sequencing (WGS) have been performed in many deeply phenotyped prospective cohort studies and electronic health record (EHR)-based cohorts of tens of thousands of individuals. Completed and ongoing large-scale WES and WGS sequencing efforts include the National Heart, Lung, and Blood Institute (NHLBI) Exome Sequencing Project (ESP) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> 2014</a>), Trans-Omics for Precision Medicine (TOPMed) Program (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2015</a>), the NHGRI Genome Sequencing Program (GSP), the UK10K project (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">UK10K Consortium 2015</a>), and the Geisinger MyCode project (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mukherjee <em>et al.</em> 2015</a>), to name a few. This big wave of sequencing data provides researchers with unprecedented opportunities to investigate low frequency [minor allele frequency (MAF) between 1 and 5%] and rare (MAF <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-1.gif"/></noscript></span></span>) single nucleotide variants (SNVs) in association with complex phenotypes and diseases (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Yi <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Schaid <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lee <em>et al.</em> 2014</a>). An example of the initial successes is the discovery of rare functional variants in <em>APOC3</em> associated with lower plasma triglyceride levels and a reduced risk of coronary heart disease (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> 2014</a>).</p><p id="p-4">Many phenotypes such as triglyceride and fasting glucose collected in population-based cohort studies are quantitative and may not follow a normal distribution, as explicitly or implicitly assumed in most existing statistical methods for rare variant (RV)-based association testing (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bansal <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fan <em>et al.</em> 2015</a>). However, there is a dearth of literature on the robustness of RV tests to the nonnormality of the observed traits, <em>e.g.</em>, due to skewness, which is expected to be ubiquitous in cohort studies. In particular, we find that commonly used RV tests, including the sequence kernel association test (SKAT) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wu <em>et al.</em> 2011</a>) and SKAT-O test (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lee <em>et al.</em> 2012</a>), are very sensitive to quantitative trait’s deviation from normality and can have severely inflated association p-values. For example, when applied to the ESP WES data in association with plasma triglyceride levels, as described in detail later on, SKAT and SKAT-O had globally inflated quantile–quantile (QQ) plots with genomic control (GC; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Devlin and Roeder 1999</a>) <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-2.gif"/></noscript></span></span> respectively. In addition to the case study of RV-triglyceride association testing, here we have conducted extensive simulation studies to investigate the performance of several commonly used RV tests, including the burden test (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li and Leal 2008</a>), SKAT, and SKAT-O, as well as our recently proposed adaptive sum of powered score (aSPU) test (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> 2014</a>), in the presence of nonnormal quantitative traits. We have also studied and compared some commonly used <em>ad hoc</em> strategies to deal with nonnormal traits, such as natural logarithm transformation, inverse normal transformation, Winsorizing, trimming, and minor allele count (MAC) thresholding. Although we find that the aSPU test is more robust than SKAT and SKAT-O, it can sometimes suffer from inflated type I error rates in the presence of a few contaminated observations. In response, we further propose a robust version of the aSPU test, called aSPUr. While the traditional variant-by-variant association test for common SNVs (MAF <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-3.gif"/></noscript></span></span>) has been shown to be robust to nonnormal distributed traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cao <em>et al.</em> 2014</a>), here we demonstrate that RV association testing can be very sensitive to quantitative trait’s subtle deviation from normality. Based on type I error control and statistical power considerations, we further provide practitioners with some general guidelines and a new robust test to deal with nonnormal quantitative traits.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Review of existing RV tests</h3><p id="p-5">We first review our recently proposed class of sum of powered score (SPU) tests and their adaptive version called aSPU test (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> 2014</a>). The former include the burden and SKAT tests as special cases. We then introduce a new robust version of the SPU and aSPU tests, denoted as SPUr and aSPUr. Consider a linear model for a quantitative trait,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-4.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-5.gif"/></noscript></span></span> is the trait for subject <em>i</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-6.gif"/></noscript></span></span> is the MAC (coded as 0, 1, or 2) of SNV <em>j</em> for subject <em>I</em>, and the error term <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-7.gif"/></noscript></span></span> is assumed to have a distribution with mean 0 and a constant variance <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-8.gif"/></noscript></span></span> The main interest is to test <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-9.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-10.gif"/></noscript></span></span> <em>i.e.</em>, none of the <em>k</em> variants in a set is associated with the phenotype. The score vector is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-11.gif"/></noscript></span></span>and its covariance matrix is <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-12.gif"/></noscript></span></span> which can be consistently estimated by <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-13.gif"/></noscript></span></span> In fact, for any generalized linear models (GLMs) with a canonical link function, the score vector <em>U</em> remains the same as the above.</p><p id="p-6"><a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> (2014)</a> proposed a class of SPU tests, for an integer <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-14.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-15.gif"/></noscript></span></span>Note that when <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-17.gif"/></noscript></span></span> the SPU test is equivalent to the burden test and the SKAT test under the linear kernel with equal RV weighting, respectively. Importantly, as <em>γ</em> increases, the SPU(<em>γ</em>) test puts more weights on the larger components of <em>U</em> while gradually ignoring the remaining components. In particular, we have<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-18.gif"/></noscript></span></span>As will be shown, since the SPU tests are based on resampling methods to calculate their p-values, they are invariant to monotone transformations, such as <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-19.gif"/></noscript></span></span> That is, we can define <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-20.gif"/></noscript></span></span> which uses only the largest component of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-21.gif"/></noscript></span></span> and does not aggregate information from other RVs. More generally, as we increase the value of <em>γ</em>, we put higher and higher weights on the larger components of <em>U</em>, effectively realizing RV selection. On the other hand, an even integer of <em>γ</em> automatically eliminates the effects of different signs of <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-22.gif"/></noscript></span></span>’s, avoiding power loss of the burden test in the presence of different association directions. However, an odd integer of <em>γ</em> might be more suitable, as in the SPU(1) or burden test, when the associations are all in the same direction.</p><p id="p-7">Without covariates, <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> (2014)</a> proposed using permutations to obtain p-values for the SPU tests. With covariates, the parametric bootstrap (or, alternatively, permuting residuals) can be performed. Briefly, we fit a null model under <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-23.gif"/></noscript></span></span> and obtain the residuals, then we randomly permute the residuals and add them to the estimated means of the traits from the null model, obtaining a new set of null traits <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-24.gif"/></noscript></span></span> We use the null traits <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-25.gif"/></noscript></span></span> to obtain a null statistic <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-26.gif"/></noscript></span></span> We repeat the above process <em>B</em> times, and calculate the p-value as <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-27.gif"/></noscript></span></span></p><p id="p-8">Since the power of an SPU(<em>γ</em>) test depends on the choice of <em>γ</em> while the optimal choice of <em>γ</em> depends on the unknown true association pattern of the RVs to be tested, it would be desirable to data-adaptively choose the value of <em>γ</em>. For this purpose, <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> (2014)</a> proposed an adaptive SPU (aSPU) test to combine information across multiple SPU tests with various values of <em>γ</em>. Suppose that we have some candidate values of <em>γ</em> in <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-28.gif"/></noscript></span></span> <em>e.g.</em>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-29.gif"/></noscript></span></span> as used in our later simulation experiments, and suppose that the p-value of the <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-30.gif"/></noscript></span></span> test is <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-31.gif"/></noscript></span></span> then our combining procedure is to take the minimum p-value:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-32.gif"/></noscript></span></span>Of course, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-33.gif"/></noscript></span></span> is no longer a genuine p-value; as for the SPU tests, we recourse to a resampling method to estimate its p-value. As before, first we simulate <em>B</em> independent copies of the null traits <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-34.gif"/></noscript></span></span> by the parametric bootstrap for <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-35.gif"/></noscript></span></span> We then calculate the corresponding SPU test statistics <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-36.gif"/></noscript></span></span> and their p-values <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-37.gif"/></noscript></span></span> Thus, we have <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-38.gif"/></noscript></span></span> and the final p-value of the aSPU test is <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-39.gif"/></noscript></span></span> We used <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-40.gif"/></noscript></span></span> in our simulation experiments. Note that, we can first use a smaller <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-41.gif"/></noscript></span></span> or so to scan a genome, then use a larger <em>B</em> to test on a few genes or regions that pass the significance criterion (<em>e.g.</em>, p-value <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-42.gif"/></noscript></span></span>) in the first step.</p></div><div id="sec-3" class="subsection"><h3>New tests: robust SPU and aSPU tests</h3><p id="p-9">A potential problem with the above Gaussian likelihood-based approach is its nonrobustness to outliers, which can be caused by non-Gaussian errors <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-43.gif"/></noscript></span></span> or contaminated traits <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-44.gif"/></noscript></span></span> Consider a situation where we observe a singleton for RV <em>j</em>; that is, say <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-45.gif"/></noscript></span></span> and all other <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-46.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-47.gif"/></noscript></span></span> Then the <em>j</em>th component of the score vector is <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-48.gif"/></noscript></span></span> which will be largely influenced by a single observation <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-49.gif"/></noscript></span></span> As to be shown later, in such a situation, if <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-50.gif"/></noscript></span></span> is contaminated or measured with error, then we may have inflated type I errors. To overcome the problem, we propose using a robust regression method. Rather than using the Gaussian-based likelihood, we propose using the Huber loss with the corresponding score vector <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-51.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-52.gif"/></noscript></span></span> and<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-53.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-54.gif"/></noscript></span></span> is chosen to maintain a high efficiency for a normal error (<em>i.e.</em>, trait) distribution, and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-55.gif"/></noscript></span></span> is an estimate of <em>σ</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Jureckova and Picek 2006</a>). Under <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-56.gif"/></noscript></span></span> we can use the median absolute deviation (MAD) as a robust estimate of <em>σ</em>. <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-57.gif"/></noscript></span></span> It is clear that the truncation of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-58.gif"/></noscript></span></span> at a constant <em>c</em> eliminates or alleviates the undue influence of outlying <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-59.gif"/></noscript></span></span>’s.</p><p id="p-10">We define a robust SPU (SPUr) test for a given <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-60.gif"/></noscript></span></span> as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-61.gif"/></noscript></span></span>With various values of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-62.gif"/></noscript></span></span> we obtain a class of the SPUr tests. Accordingly we define an adaptive robust SPU (aSPUr) test as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-63.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-64.gif"/></noscript></span></span> is the p-value of the SPUr(<em>γ</em>) test, and we use <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-65.gif"/></noscript></span></span> as before. The p-values of the SPUr and aSPUr tests are obtained in the same way as for the SPU and aSPU tests described earlier.</p><p id="p-11">Alternatively, based on some initial estimate <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-66.gif"/></noscript></span></span> (<em>e.g.</em>, the least squares or least absolute deviation estimate) of <em>β</em>, we define residuals <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-67.gif"/></noscript></span></span> and then use <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-68.gif"/></noscript></span></span> which might give higher power than using the other estimate of <em>σ</em> (which does not take account of possible effects of RVs). However, it is difficult to obtain reliable estimates of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-69.gif"/></noscript></span></span> for RVs, which in fact motivated the development of the burden tests and other methods. This is a topic to be explored in the future.</p></div><div id="sec-4" class="subsection"><h3>Comparison with Winsorizing and trimming</h3><p id="p-12">Two simple and straightforward ways to handle outliers are Winsorizing and trimming. For a specified small <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-70.gif"/></noscript></span></span> such as <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-71.gif"/></noscript></span></span> or 0.025, define the <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-72.gif"/></noscript></span></span>-percentile and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-73.gif"/></noscript></span></span>-percentile of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-74.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-76.gif"/></noscript></span></span> respectively. In Winsorizing, any <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-77.gif"/></noscript></span></span> satisfying <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-78.gif"/></noscript></span></span> is truncated at <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-79.gif"/></noscript></span></span> and any <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-80.gif"/></noscript></span></span> is truncated at <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-81.gif"/></noscript></span></span> In trimming, any observation <em>i</em> is removed from the dataset if <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-82.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-83.gif"/></noscript></span></span></p><p id="p-13">Winsorizing is to some degree like using the Huber loss function in truncating outlying trait values. However there are two important differences. First, the choice of the threshold <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-84.gif"/></noscript></span></span> is arbitrary, which may be too small or too large, depending on the unknown proportion of the outliers. Second, more importantly, in Winsorizing whether a trait value <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-85.gif"/></noscript></span></span> is judged to be an outlier or not is completely based on its absolute value <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-86.gif"/></noscript></span></span> without accounting for any covariates; if instead we Winsorize residuals <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-87.gif"/></noscript></span></span> it will be more similar to using the Huber loss. As will be shown, ignoring covariate effects may lead to severely inflated type I errors or power loss.</p><p id="p-14">In addition to the above two disadvantages shared with Winsorizing, trimming is too extreme in eliminating the observations judged to be, but in truth may or may not be, outliers, which often leads to severe loss of power.</p></div><div id="sec-5" class="subsection"><h3>Software and data availability</h3><p id="p-15">The aSPUr test has been implemented in <em>R</em> package “aSPU” available on the Comprehensive R Archive Network (CRAN): <a href="https://cran.r-project.org/web/packages/aSPU/">https://cran.r-project.org/web/packages/aSPU/</a>. The NHLBI ESP data are accessible from the National Center for Biotechnology Information (NCBI) dbGaP with accession numbers phs000398, phs000400, phs000401, and phs000281.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Simulation set-ups</h3><p id="p-16">To evaluate and compare the performance of various tests, we conducted extensive simulation studies under different trait distributions. The genotype data were simulated following <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Wang and Elston (2007)</a> and <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Basu and Pan (2011)</a>. Specifically, a latent variable <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-88.gif"/></noscript></span></span> was simulated from a <em>k</em>-dimensional multivariate normal distribution <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-89.gif"/></noscript></span></span> with <em>V</em> as an AR-1(<em>ρ</em>) correlation structure: <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-90.gif"/></noscript></span></span> for any <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-91.gif"/></noscript></span></span> Then we randomly drew from a uniform distribution <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-92.gif"/></noscript></span></span> <em>k</em> MAFs between 0.1 and 0.5%, and accordingly dichotomized <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-93.gif"/></noscript></span></span> to yield a haplotype. We similarly simulated another latent variable and the corresponding haplotype. We combined the two haplotypes to form the genotype <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-94.gif"/></noscript></span></span> for subject <em>i</em>. This process was repeated <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-95.gif"/></noscript></span></span> times to generate genotypes for <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-96.gif"/></noscript></span></span> subjects. We used <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-97.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-98.gif"/></noscript></span></span> to generate independent and correlated SNVs (in linkage equilibrium and in linkage disequilibrium) respectively. Note that we only used unphased genotypes, not haplotypes, in simulations.</p><p id="p-17">A trait <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-99.gif"/></noscript></span></span> was simulated from linear regression model <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-100.gif"/></noscript></span></span> with the following error distribution. First, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-101.gif"/></noscript></span></span> was independent and identically distributed (iid) from <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-102.gif"/></noscript></span></span> Second, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-103.gif"/></noscript></span></span> was iid from <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-104.gif"/></noscript></span></span> a Log-normal distribution with mean 0 and SD <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-105.gif"/></noscript></span></span> on the log scale. Third, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-106.gif"/></noscript></span></span> was iid from <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-107.gif"/></noscript></span></span> a <em>t</em>-distribution with degrees of freedom <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-108.gif"/></noscript></span></span> or 3. Fourth, <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-109.gif"/></noscript></span></span> was iid from a contaminated <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-110.gif"/></noscript></span></span> a single observation <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-111.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-112.gif"/></noscript></span></span> was randomly chosen and its trait had an additive error <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-113.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-114.gif"/></noscript></span></span> or 10.</p><p id="p-18">To evaluate the empirical type I error (null cases), we had <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-115.gif"/></noscript></span></span> For empirical power (nonnull cases), we randomly chose eight SNVs from <em>k</em> SNVs as causal ones with nonzero <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-116.gif"/></noscript></span></span>’s while other SNVs having their <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-117.gif"/></noscript></span></span> For causal SNVs, we used two sets of coefficients: <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-118.gif"/></noscript></span></span> (power set-up I) and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-119.gif"/></noscript></span></span> (power set-up II), which favors SKAT and the burden test, respectively. In the absence of other covariates, <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-120.gif"/></noscript></span></span> was just a constant 1 for the intercept term; otherwise, we randomly generated two independent covariates from <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-121.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-122.gif"/></noscript></span></span> To investigate the robustness of a method to the number of SNVs, we increased <em>k</em> from 8 to 256.</p></div><div id="sec-8" class="subsection"><h3>Simulation results</h3><p id="p-19"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the empirical type I error rates for various tests without any transformation. Under <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-123.gif"/></noscript></span></span> error distribution, all tests controlled the type I error rates satisfactorily at the nominal level <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-124.gif"/></noscript></span></span> Under heavy-tailed (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-125.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-126.gif"/></noscript></span></span>) and skewed (<span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-127.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-128.gif"/></noscript></span></span>) error distributions, SKAT and SKAT-O had severely inflated type I error rates, while aSPU and aSPUr controlled their type I error rates satisfactorily. With even just 1 (out of 400) quantitative trait contaminated, all the tests except aSPUr could not control the type I error rates well, though the aSPU test (along with the SPU tests) performed much better than SKAT and SKAT-O. The same conclusions held with correlated SNVs and with or without covariates (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS2.pdf">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS3.pdf">Table S3</a>). For nonnormal error distributions subject to Winsorizing or trimming, the results were dependent on the choice of the cut-off <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-129.gif"/></noscript></span></span> as shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS5.pdf">Table S5</a>. For example, when the error distribution was <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-130.gif"/></noscript></span></span> with no covariates, SKAT and SKAT-O after Winsorizing at <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-131.gif"/></noscript></span></span> could maintain a correct type I error rate, but not at <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-132.gif"/></noscript></span></span> For an error distribution of <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-133.gif"/></noscript></span></span> SKAT and SKAT-O could not control the type I error rates for either <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-134.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-135.gif"/></noscript></span></span> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS4.pdf">Table S4</a>). With covariates, the performance became worse, especially with trimming; neither Winsorizing nor trimming could control type I error rates at either <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-136.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-137.gif"/></noscript></span></span> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS5.pdf">Table S5</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469989/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469989/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Empirical type I error rates of various tests at the significance level of 0.05 for a quantitative trait with an error distribution (Distr), a number of independent SNVs (#SNVs), and with two covariates</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">For empirical power comparison, under <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-148.gif"/></noscript></span></span> error distribution the aSPU test performed similarly to SKAT or SKAT-O with a smaller number of SNVs; however, as the number of SNVs increased, the aSPU test became more powerful (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS6.pdf">Table S6</a>). As reported before (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2015a</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">b</a>), with increasing number of SNVs an SPU(<em>γ</em>) test with a larger <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-149.gif"/></noscript></span></span> value tended to be more powerful; in particular, SPU(4) could be much more powerful than SPU(2), <em>e.g.</em>, when there were 128 or more SNVs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS6.pdf">Table S6</a>). Of note, SPU(2) is equivalent to SKAT with a linear kernel which was optimal here and was used throughout (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pan 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Pan 2011</a>). On the other hand, the aSPUr was conservative, especially for causal SNVs with larger effect sizes (Cases III and IV in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS6.pdf">Table S6</a>), in which it was hard to distinguish a genuinely large effect size of a RV from a contaminated trait value. In robust statistics, one would like to use some initial estimator to estimate and thus take account of large effects, which however was almost impossible in the current context for RVs: with small MAFs, it is almost impossible to obtain reliable estimates for RVs. The same conclusions held with correlated SNVs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS7.pdf">Table S7</a>). For Winsorizing or trimming, there was always a dramatic loss of power with trimming, while Winsorizing performed well for a smaller number of SNVs. But its performance deteriorated as the number of SNVs increased; in particular, again its performance depended on the use of the <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-150.gif"/></noscript></span></span> level (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS8.pdf">Table S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/TableS9.pdf">Table S9</a>).</p><p id="p-21">We further investigated the effects of natural logarithm (Ln) and rank-based inverse normal (INV) transformations on the type I error rate and power. We considered the skewed <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-151.gif"/></noscript></span></span> and heavy-tailed <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-152.gif"/></noscript></span></span> error distributions. With the two covariates in the simulated data, we first regressed them out in a linear model under <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-153.gif"/></noscript></span></span> then used the residuals or their transformations to test their association with a set of SNVs.</p><p id="p-22"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the type I error rates and powers for a skewed error distribution <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-154.gif"/></noscript></span></span> First, without transformation both SKAT and SKAT-O gave severely inflated type I error rates, while aSPU and aSPUr controlled their type I error rates satisfactorily; with Ln transformation, all tests performed well, although the type I error rates of SKAT and SKAT-O might be slightly inflated; with INV transformation, all tests controlled the type I error rates satisfactorily. Second, under power set-up I which favored SKAT and SPU(2) because the association directions of the eight causal SNVs were different, without transformation although both aSPU and aSPUr could control the type I error rates, they lost power dramatically as compared to those with transformed traits; between the two, aSPUr was more powerful. On the other hand, with Ln transformation SKAT was most powerful, followed by SKAT-O, then aSPU, and finally aSPUr, though the power difference became smaller as the number of SNVs to be tested increased; with INV transformation, SKAT was most powerful for smaller numbers of SNVs while aSPU was more powerful for larger numbers of SNVs; for unknown reasons, aSPUr did not perform well. Third, under power set-up II which favored burden tests because the association directions of the eight causal SNVs were the same, without transformation although both aSPU and aSPUr could control the type I error rates, they lost power dramatically as compared to those with transformed traits; between the two, aSPU was more powerful. With Ln transformation, SKAT-O and aSPU were most powerful, followed by SKAT or aSPUr, though the power difference was not dramatic; with INV transformation, aSPUr was consistently best, followed by SKAT-O and aSPU (for which the former had an edge for a smaller number of SNVs while the latter had otherwise), finally by SKAT.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation results for a skewed error distribution  the first row is for type I errors, and the next two rows for power in set-up I with  and set-up II with " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-213886493" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation results for a skewed error distribution &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; the first row is for type I errors, and the next two rows for power in set-up I with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and set-up II with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3941/F1.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/12/3941/F1.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469724" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Simulation results for a skewed error distribution <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F1/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F1/embed/mml-math-155.gif"/></noscript></span></span> the first row is for type I errors, and the next two rows for power in set-up I with <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F1/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F1/embed/mml-math-156.gif"/></noscript></span></span> and set-up II with <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F1/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F1/embed/mml-math-157.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-24"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the type I error rates and powers for a heavy-tailed (and nonskewed) error distribution <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-158.gif"/></noscript></span></span> First, without transformation both SKAT and SKAT-O gave severely inflated type I error rates, while aSPU and aSPUr controlled their type I error rates satisfactorily. Although no reason to use a Ln transformation, to show possible effects of using an incorrect transformation, we also presented results based on the Ln transformation: again both aSPU and aSPUr were robust with well-controlled type I error rates, while SKAT and SKAT-O had severely inflated ones; with INV transformation, all tests were satisfactory. On the other hand, under power set-up I, with no or Ln transformation, although both aSPU and aSPUr could control the type I error rates, aSPU lost power dramatically while aSPUr did not as compared to those with transformed traits; between the two, aSPUr was much more powerful. We showed the results for SKAT and SKAT-O, even though they had severely inflated type I error rates. With INV transformation, SKAT and aSPUr were the winners, though SKAT was slightly more powerful for smaller numbers of SNVs while aSPUr was more powerful for larger numbers of SNVs, closely followed by aSPU, then SKAT-O. Finally, under power set-up II, with no or Ln transformation, although both aSPU and aSPUr could control the type I error rates, aSPU lost power dramatically while aSPUr did not as compared to those with transformed traits; between the two, aSPUr was more powerful; with INV transformation, aSPUr was best, closely followed by aSPU, then SKAT-O, and then SKAT.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation results for a heavy-tailed (and nonskewed) error distribution  the first row is for type I errors, and the next two rows for power in set-up I with  and set-up II with " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-213886493" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation results for a heavy-tailed (and nonskewed) error distribution &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-159&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; the first row is for type I errors, and the next two rows for power in set-up I with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-160&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and set-up II with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3941/F2.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/12/3941/F2.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469844" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Simulation results for a heavy-tailed (and nonskewed) error distribution <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F2/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F2/embed/mml-math-159.gif"/></noscript></span></span> the first row is for type I errors, and the next two rows for power in set-up I with <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F2/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F2/embed/mml-math-160.gif"/></noscript></span></span> and set-up II with <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F2/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F2/embed/mml-math-161.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Data example: application to the NHLBI ESP triglyceride phenotype</h3><p id="p-26">To further demonstrate the performance of various RV tests in a real data example, we analyzed the WES data in association with plasma triglyceride level in 1731 individuals of European ancestry who were sequenced in the NHLBI ESP project. The study subjects were selected from the following population-based cohorts: Atherosclerosis Risk in Communities, the Cardiovascular Heart Study, the Framingham Heart Study, and the Womens Health Initiative; see <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> (2014)</a> for details. We performed gene-based RV association tests, including SKAT, SKAT-O, T1 burden test, aSPU, and aSPUr, on untransformed, natural logarithm transformed, and rank-based inverse normal transformed triglyceride levels, denoted as TG, Ln(TG), and INV(TG), respectively. Following <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> (2014)</a>, we included nonsynonymous (nonsense and missense) and splice-site variants of MAF <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-162.gif"/></noscript></span></span> within each gene and excluded genes with cumulative MACs &lt;5, resulting in 13,978 genes. The genome-wide significance threshold was set at <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-163.gif"/></noscript></span></span> based on the Bonferroni procedure. As in <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> (2014)</a>, we performed natural logarithm transformation on the raw triglyceride level and adjusted for covariates including age, sex, two principal components capturing population substructure, and indicator variables for the ESP ascertainment scheme in all association testing. We used QQ plots and GC <em>λ</em> to detect possible inflation of the RV association test p-values. Because there were a large number of extremely rare variants, <em>e.g.</em>, singletons and doubletons, in the ESP WES data, we used the power set <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-164.gif"/></noscript></span></span> for both aSPU and aSPUr, as suggested by <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Pan <em>et al.</em> (2014)</a> for numerical stability. In addition, we used the following stage-wise bootstrap procedure for aSPU and aSPUr: we started with <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-165.gif"/></noscript></span></span> for all genes and then gradually increased <em>B</em>. If an estimated p-value was <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-166.gif"/></noscript></span></span> we increased <em>B</em> to <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-167.gif"/></noscript></span></span> to reestimate the p-value until <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-168.gif"/></noscript></span></span> for genome-wide significance. Moreover, we used <em>APOC3</em> as a positive control gene to compare the power of various tests. <em>APOC3</em> was identified as the top gene harboring putatively functional RVs associated with reduced level of Ln(TG), and was further replicated and confirmed in independent large samples (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> 2014</a>). In addition, it was identified in other RV association studies (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Tachmazidou <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li <em>et al.</em> 2015</a>).</p><p id="p-27"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3F</a> shows that TG was right-skewed with some individuals having extremely high TG levels. When applied to TG, the QQ plots for aSPU and aSPUr behaved normally as shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> (<span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-169.gif"/></noscript></span></span>). In contrast, the SKAT and SKAT-O tests had severely inflated QQ plots (<em>λ</em> = 1.89 and 1.78, respectively); the QQ plot for T1 was less inflated but had a discernable deviation from the null in the tail area (<em>λ</em> = 1.13). We investigated the effectiveness of some <em>ad hoc</em> strategies, including trimming, Winsorizing, and increasing the MAC threshold, in alleviating the p-value inflation. When trimming at <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-170.gif"/></noscript></span></span> <em>λ</em> for SKAT and SKAT-O was reduced to 1.09 and 1.10, respectively; when Winsorizing at <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-171.gif"/></noscript></span></span> <em>λ</em> was reduced to 1.12 and 1.11, respectively. Despite the improvement, the QQ plots for SKAT and SKAT-O remained inflated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/FigureS1.pdf">Figure S1</a>). When we excluded genes with a MAC <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-172.gif"/></noscript></span></span> the QQ plots for SKAT and SKAT-O were still inflated with <em>λ</em> = 1.36 and 1.30, respectively, whereas T1 had a much improved QQ plot (<em>λ</em> = 1.04) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/FigureS2.pdf">Figure S2</a>). However, increasing the MAC threshold to 30 would further exclude 8155 genes, including the positive control gene <em>APOC3</em> with 14 minor alleles. When applied to Ln(TG) and INV(TG), all tests had well-behaved QQ plots and <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-173.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-174.gif"/></noscript></span></span> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/FigureS4.pdf">Figure S4</a>). As shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3F</a>, TG approximately followed a Log-normal distribution, leading to similar results from the Ln and INV transformations; see the p-value comparison for <em>APOC3</em> in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. To investigate whether the RV association test p-value inflation was also applicable to common variants, we performed conventional variant-by-variant association testing of TG for 50,602 SNPs with an MAF <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-175.gif"/></noscript></span></span>. As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1/FigureS5.pdf">Figure S5</a>, the QQ plot was well behaved with <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-176.gif"/></noscript></span></span> suggesting that the p-value inflation was likely a unique problem for some RV association tests.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QQ plots for the analysis of triglyceride with 13,978 genes with MAC  (A) SKAT (genomic control ), (B) SKAT-O (), (C) T1 (), (D) aSPU (), and (E) aSPUr (). (F) Histogram of covariate-adjusted triglyceride residuals with variant carriers of APOC3 highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-213886493" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;QQ plots for the analysis of triglyceride with 13,978 genes with MAC &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-177&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A) SKAT (genomic control &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-178&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), (B) SKAT-O (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-179&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), (C) T1 (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-180&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), (D) aSPU (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-181&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and (E) aSPUr (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-182&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). (F) Histogram of covariate-adjusted triglyceride residuals with variant carriers of &lt;em&gt;APOC3&lt;/em&gt; highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3941/F3.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/12/3941/F3.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3941/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469851" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">QQ plots for the analysis of triglyceride with 13,978 genes with MAC <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-177.gif"/></noscript></span></span> (A) SKAT (genomic control <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-178.gif"/></noscript></span></span>), (B) SKAT-O (<span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-179.gif"/></noscript></span></span>), (C) T1 (<span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-180.gif"/></noscript></span></span>), (D) aSPU (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-181.gif"/></noscript></span></span>), and (E) aSPUr (<span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/F3/embed/mml-math-182.gif"/></noscript></span></span>). (F) Histogram of covariate-adjusted triglyceride residuals with variant carriers of <em>APOC3</em> highlighted.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469889/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469889/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">RV association testing results of positive control gene <em>APOC3</em> (among 13,978 genes with a MAC <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/T2/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/T2/embed/mml-math-183.gif"/></noscript></span></span>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-29"><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows the p-values and ranking of <em>APOC3</em> by various tests. In the analysis of TG, <em>APOC3</em> was ranked 62nd by aSPUr, but was not among the top 200 genes by all other methods; in the analyses of Ln(TG) and INV(TG), it was ranked among the top two by T1, SKAT-O, aSPU, and aSPUr, but not SKAT. This is consistent with the results reported in <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Crosby <em>et al.</em> (2014)</a> that <em>APOC3</em> was the top gene associated with Ln(TG) by the T1 test but its p-value was not genome-wide significant in the discovery samples from the ESP. We demonstrate here that the statistical significance of <em>APOC3</em> was dependent on the transformation of the phenotype TG. <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3F</a> shows that the carriers of the minor alleles for five out of six RVs in <em>APOC3</em> had reduced TG levels compared with the population average. In the presence of quite a few individuals with extremely high TG levels, <em>APOC3</em> was only nominally associated with TG and lowly ranked by all RV association tests except for aSPUr. By downweighting the extremely high TG observations, aSPUr increased the statistical significance and ranking of <em>APOC3</em> compared with aSPU and other tests, while avoiding global inflation of the p-values. On the other hand, since both Ln and INV transformations reduced the impact of extremely high TG observations, the association signal of <em>APOC3</em> was much amplified, resulting in its high ranking. In addition, as the majority of the variants in <em>APOC3</em> reduced the TG level, <em>i.e.</em>, the effects were roughly in the same direction, the T1 burden test and adaptive tests that incorporate the burden test, such as aSPU, aSPUr, and SKAT-O, yielded higher ranking for <em>APOC3</em> than did SKAT.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-30">In summary, we have demonstrated using extensive simulations and application to the ESP WES data that SKAT and SKAT-O are not robust to heavy-tailed or skewed error distributions of quantitative traits with inflated type I error rates. <em>Ad hoc</em> remediation procedures, such as trimming and Winsorizing, may not be effective in reducing the inflated type I error rates and may lead to severe power loss. Depending on the underlying trait distributions, Ln or INV transformation may help control the type I error rates for SKAT and SKAT-O, which, however, could lead to transformation-specific association results as illustrated in the <em>APOC3</em> example, as well as power loss as demonstrated in simulation set-up II in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. On the other hand, the aSPU test and the newly proposed aSPUr test are much more robust to quantitative traits’ deviation from normality.</p><p id="p-31">The nonrobustness of the SKAT test is mainly due to its poor asymptotic approximation of the null distribution in the presence of outliers. Note that the issue with SKAT remained with the use of its resampling method to calculate its p-values: we found that SKAT-Resampling implemented in the <em>R</em> package “SKAT” gave essentially equal p-values to those of SKAT in both simulations and real data application; the Pearson correlation between the two sets of the p-values was <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-194.gif"/></noscript></span></span> Moreover, the RV weighting scheme of SKAT makes its type I error inflation even worse. Since SKAT puts a higher weight on a more rare SNV <em>j</em>, if <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-195.gif"/></noscript></span></span> then it is a high-leverage point; in addition, if <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-196.gif"/></noscript></span></span> is outlying, then we know <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-197.gif"/></noscript></span></span> is an influential point. Hence, although SKAT’s weighting on rare SNVs might help it gain power to detect associated RVs, at the same time, the weighting also renders its nonrobustness to observations with outlying traits, which could happen when the trait has a heavy-tailed or right-skewed distribution, as shown in our simulations and supported by the real data application. For the latter, when applied to TG, SKAT with its default <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-198.gif"/></noscript></span></span> weighting and equal weighting gave a GC <em>λ</em> of 1.89 and 1.86, respectively.</p><p id="p-32">Recently <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Auer <em>et al.</em> (2016)</a> also reported that single SNV-based and SNV-set-based RV tests can be nonrobust to phenotypic outliers and nonnormality, which is in agreement with the main theme of this paper and highlights the importance of the topic studied here. They recommended the INV transformation for nonnormally distributed traits. Our work here is distinctive from Auer <em>et al.</em> in several important aspects. First, in addition to demonstrating the nonrobustness of existing RV tests, we have proposed a new SNV-set-based robust RV test, aSPUr. Second, while Auer <em>et al.</em> applied the Huber robust regression in the context of single SNV-based RV testing, we impose the Huber loss on the score vector <em>U</em> in the proposed aSPUr, which is generalizable to the broad class of score vector-based SNV-set RV tests, <em>e.g.</em>, SPU(1)/T1 and SPU(2)/SKAT. Third, in contrast to the finding of Auer <em>et al.</em> that the permutation test was the least powerful method when applied to single SNV-based RV test, we found the permutation-based aSPU and aSPUr to be robust in terms of both type I error control and maintaining high statistical power in the presence of true signals. Finally, we found that while the INV transformation could maintain the type I error rate, it could also lead to transformation-dependent ranking order of p-values as demonstrated in the <em>APOC3</em> example (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-33">We proposed the aSPUr test in the Huber loss framework. It is one of the first proposed and most thoroughly studied loss function in the robust statistics literature. For example, when the error distribution is normal, it has been shown that the Huber loss achieves <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12/3941/embed/mml-math-199.gif"/></noscript></span></span> asymptotic efficiency with the tuning parameter <em>c</em> being equal to 1.345 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Huber 1964</a>). We found that it performed satisfactorily in our extensive numerical experiments. Other loss functions are also possible, for example, Tukeys biweight function (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Jureckova and Picek 2006</a>), which warrants further investigation.</p><p id="p-34">In conclusion, we would recommend the use of the aSPU test for its robustness to heavily-tailed or skewed error distributions and its high power across many situations due to its adaptiveness. If there is evidence of inflated type I error or <em>λ</em>, <em>e.g.</em>, through QQ plots, then one may try the more robust aSPUr test or SKAT and SKAT-O with the INV transformation. Finally, neither Winsorizing nor trimming the data before applying another test, <em>e.g.</em>, SKAT or SKAT-O, outperformed the aSPUr test that was applied to the original data.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">This research was supported by National Institutes of Health (NIH) grant R01HL116720. P.W. was also supported by NIH grants R01CA169122 and R21HL126032, W.P. by R01GM113250, R01HL105397, and R01GM081535, and E.B. by RC2HL102419. The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-36">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035485/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-37"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-12"><span class="received-label">Received </span>July 12, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-21"><span class="accepted-label">Accepted </span>September 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Wei <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.12.3941.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Applebaum-Bowden</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackwell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blangero</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Sequence data processing and analysis of the first 20,000 human genomes in the NHLBI TopMed Program. Presented at the 65th Annual Meeting of The American Society of Human Genetics. Baltimore, MD.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auer</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The effect of phenotypic outliers and non-normality on rare-variant association testing.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1188</span>–<span class="cit-lpage">1194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D24%26rft.spage%253D1188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.3" data-doi="10.1038/nrg2867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libiger</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torkamani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Statistical analysis strategies for association studies involving rare variants.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBansal%26rft.auinit1%253DV.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D773%26rft.epage%253D785%26rft.atitle%253DStatistical%2Banalysis%2Bstrategies%2Bfor%2Bassociation%2Bstudies%2Binvolving%2Brare%2Bvariants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2867%26rft_id%253Dinfo%253Apmid%252F20940738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20940738&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283052800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.4" data-doi="10.1002/gepi.20609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparison of statistical tests for association with rare variants.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">606</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DBasu%26rft.auinit1%253DS.%26rft.volume%253D35%26rft.issue%253D7%26rft.spage%253D606%26rft.epage%253D619%26rft.atitle%253DComparison%2Bof%2Bstatistical%2Btests%2Bfor%2Bdisease%2Bassociation%2Bwith%2Brare%2Bvariants.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20609%26rft_id%253Dinfo%253Apmid%252F21769936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21769936&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauwe</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A versatile omnibus test for detecting mean and variance heterogeneity.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D38%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.6" data-doi="10.1056/NEJMoa1307095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peloso</span>  <span class="cit-name-given-names">G.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auer</span>  <span class="cit-name-given-names">P.L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosslin</span>  <span class="cit-name-given-names">D.R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitziel</span>  <span class="cit-name-given-names">N.O.</span></span>, </li><li><span class="cit-etal">et al.</span>, </li><li><span class="cit-auth cit-collab">The TG and HDL Working Group of the Exome Sequencing Project, National Heart, Lung, and Blood Institute</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Loss-of-function mutations in APOC3, triglycerides, and coronary disease.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">371</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D371%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1307095%26rft_id%253Dinfo%253Apmid%252F24941081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1307095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24941081&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338265700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.7" data-doi="10.1111/j.0006-341X.1999.00997.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic control for association studies.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDevlin%26rft.auinit1%253DB.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D997%26rft.epage%253D1004%26rft.atitle%253DGenomic%2Bcontrol%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.1999.00997.x%26rft_id%253Dinfo%253Apmid%252F11315092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.1999.00997.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11315092&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084218000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.8" data-doi="10.1534/genetics.115.178343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene level meta-analysis of quantitative traits by functional linear models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178343%26rft_id%253Dinfo%253Apmid%252F26058849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEwODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.9" data-doi="10.1214/aoms/1177703732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Robust estimation of a location parameter.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D35%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177703732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177703732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3941.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jureckova</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picek</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Robust Statistical Methods with R</span>. <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.11" data-doi="10.1016/j.ajhg.2012.06.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emond</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bamshad</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DS.%26rft.volume%253D91%26rft.issue%253D2%26rft.spage%253D224%26rft.epage%253D237%26rft.atitle%253DOptimal%2Bunified%2Bapproach%2Bfor%2Brare-variant%2Bassociation%2Btesting%2Bwith%2Bapplication%2Bto%2Bsmall-sample%2Bcase-control%2Bwhole-exome%2Bsequencing%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.06.007%26rft_id%253Dinfo%253Apmid%252F22863193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22863193&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rare-variant association analysis: study designs and statistical tests.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.13" data-doi="10.1016/j.ajhg.2008.06.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Methods for detecting associations with rare variants for common diseases: application to analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D321%26rft.atitle%253DMethods%2Bfor%2Bdetecting%2Bassociations%2Bwith%2Brare%2Bvariants%2Bfor%2Bcommon%2Bdiseases%253A%2Bapplication%2Bto%2Banalysis%2Bof%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.06.024%26rft_id%253Dinfo%253Apmid%252F18691683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.06.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691683&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.14" data-doi="10.1038/ng.3270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cupples</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brody</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analysis of loss-of-function variants and 20 risk factor phenotypes in 8,554 individuals identifies loci influencing chronic disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D640%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3270%26rft_id%253Dinfo%253Apmid%252F25915599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25915599&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.12.3941.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dushlaine</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hout</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruse</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leader</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Multivariate analysis of whole exome sequence data identifies rare variants with pleiotropic effects on obesity-related metabolic traits in 31,000 participants of the Regeneron Genetics Center Geisinger MyCode collaborative project DiscovEHR (PgmNr 39). Presented at the 65th Annual Meeting of The American Society of Human Genetics. Baltimore, MD.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.16" data-doi="10.1002/gepi.20402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Asymptotic tests of association with multiple SNPs in linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DPan%26rft.auinit1%253DW.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D497%26rft.epage%253D507%26rft.atitle%253DAsymptotic%2Btests%2Bof%2Bassociation%2Bwith%2Bmultiple%2BSNPs%2Bin%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20402%26rft_id%253Dinfo%253Apmid%252F19170135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19170135&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269432400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Relationship between genomic distance-based regression and kernel machine regression for multi-marker association testing.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D35%26rft.spage%253D211%26rft_id%253Dinfo%253Apmid%252F21308765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21308765&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.18" data-doi="10.1534/genetics.114.165035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A powerful and adaptive association test for rare variants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1095</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165035%26rft_id%253Dinfo%253Apmid%252F24831820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEwODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM5NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.19" data-doi="10.1002/gepi.21899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Testing for polygenic effects in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D39%26rft.spage%253D306%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21899%26rft_id%253Dinfo%253Apmid%252F25847094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25847094&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.20" data-doi="10.1016/j.ajhg.2015.05.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwak</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">A powerful pathway-based adaptive test for genetic association with common or rare variants.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D97%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.05.018%26rft_id%253Dinfo%253Apmid%252F26119817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.05.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26119817&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.21" data-doi="10.1002/gepi.21727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaid</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonnell</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinnwell</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibodeau</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple genetic variant association testing by collapsing and kernel methods with pedigree or population structured data.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D37%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21727%26rft_id%253Dinfo%253Apmid%252F23650101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23650101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tachmazidou</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dedoussis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southam</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmaki</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A rare functional cardioprotective APOC3 variant has risen in frequency in distinct population isolates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2872%26rft_id%253Dinfo%253Apmid%252F24343240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24343240&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-other" id="cit-6.12.3941.23" data-doi="10.1038/nature14962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">UK10K Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Min</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Memari</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The UK10K project identifies rare variants in health and disease.</span> <span class="cit-source">Nature</span> <span class="cit-vol">526</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14962%26rft_id%253Dinfo%253Apmid%252F26367797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26367797&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.24" data-doi="10.1086/511312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elston</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Improved power by use of a weighted score test for linkage disequilibrium mapping.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DT.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D353%26rft.epage%253D360%26rft.atitle%253DImproved%2Bpower%2Bby%2Buse%2Bof%2Ba%2Bweighted%2Bscore%2Btest%2Bfor%2Blinkage%2Bdisequilibrium%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F511312%26rft_id%253Dinfo%253Apmid%252F17236140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/511312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17236140&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243729500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.25" data-doi="10.1016/j.ajhg.2011.05.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rare-variant association testing for sequencing data with the sequence kernel association test.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DM.%2BC.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D93%26rft.atitle%253DRare-variant%2Bassociation%2Btesting%2Bfor%2Bsequencing%2Bdata%2Bwith%2Bthe%2Bsequence%2Bkernel%2Bassociation%2Btest.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.05.029%26rft_id%253Dinfo%253Apmid%252F21737059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.05.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21737059&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3941.26" data-doi="10.1371/journal.pgen.1002382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hierarchical generalized linear models for multiple groups of rare and common variants: jointly estimating group and individual-variants effects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DYi%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002382%26rft.epage%253De1002382%26rft.atitle%253DHierarchical%2Bgeneralized%2Blinear%2Bmodels%2Bfor%2Bmultiple%2Bgroups%2Bof%2Brare%2Band%2Bcommon%2Bvariants%253A%2Bjointly%2Bestimating%2Bgroup%2Band%2Bindividual-variant%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002382%26rft_id%253Dinfo%253Apmid%252F22144906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22144906&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3941.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/12/3941.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3927" title="Reconstructing the Backbone of the Saccharomycotina Yeast Phylogeny Using Genome-Scale Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3951" title="Characterization, Expression, and Functional Analysis of a Novel NAC Gene Associated with Resistance to Verticillium Wilt and Abiotic Stress in Cotton" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/12"><span>Volume 6 Issue 12, December 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/12.cover-source.jpg?itok=F-Hw09b0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3941" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OpPGIc1RThA8RLwAguVJY19tQPwPI2qmPKsF7daO1L0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/12/3941" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/12/3941" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/12/3941" class="active" data-icon-position="" data-hide-link-title="0">On Robust Association Testing for Quantitative Traits and Rare Variants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460496" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hSR3WT8rX97SKvwud0N5Z1u7cBp01LCerRlMZSx00Hc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3941.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/12/3941" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FuuR6fsa_nhUK6MYeBk9mFF3JkJvQvkzH1czaOIYKVc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/12/3941" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3941.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460496" id="node460496--2" data-pisa="ggg;6/12/3941" data-pisa-master="ggg;g3.116.035485" data-apath="/ggg/6/12/3941.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">On Robust Association Testing for Quantitative Traits and Rare Variants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Peng Wei</span>, <span class="highwire-citation-author" data-delta="1">Ying Cao</span>, <span class="highwire-citation-author" data-delta="2">Yiwei Zhang</span>, <span class="highwire-citation-author" data-delta="3">Zhiyuan Xu</span>, <span class="highwire-citation-author" data-delta="4">Il-Youp Kwak</span>, <span class="highwire-citation-author" data-delta="5">Eric Boerwinkle</span> and <span class="highwire-citation-author" data-delta="6">Wei Pan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3941-3950;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035485 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460496--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peng Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pwei2@mdanderson.org" class="" data-icon-position="" data-hide-link-title="0">pwei2@mdanderson.org</a>
<a href="mailto:weip@biostat.umn.edu" class="" data-icon-position="" data-hide-link-title="0">weip@biostat.umn.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ying Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yiwei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yiwei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYiwei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhiyuan Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyuan%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiyuan%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Il-Youp Kwak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Il-Youp%2BKwak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwak%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIl-Youp%2BKwak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Eric Boerwinkle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BBoerwinkle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boerwinkle%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BBoerwinkle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Wei Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pwei2@mdanderson.org" class="" data-icon-position="" data-hide-link-title="0">pwei2@mdanderson.org</a>
<a href="mailto:weip@biostat.umn.edu" class="" data-icon-position="" data-hide-link-title="0">weip@biostat.umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/12/3941" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/12/3941&amp;title=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/12/3941&amp;title=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/12/3941&amp;title=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/12/3941&amp;text=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/12/3941&amp;title=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/12/3941&amp;t=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/12/3941&amp;title=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/12/3941&amp;title=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460496/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460496" id="node460496--3" data-pisa="ggg;6/12/3941" data-pisa-master="ggg;g3.116.035485" data-apath="/ggg/6/12/3941.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">On Robust Association Testing for Quantitative Traits and Rare Variants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Peng Wei</span>, <span class="highwire-citation-author" data-delta="1">Ying Cao</span>, <span class="highwire-citation-author" data-delta="2">Yiwei Zhang</span>, <span class="highwire-citation-author" data-delta="3">Zhiyuan Xu</span>, <span class="highwire-citation-author" data-delta="4">Il-Youp Kwak</span>, <span class="highwire-citation-author" data-delta="5">Eric Boerwinkle</span> and <span class="highwire-citation-author" data-delta="6">Wei Pan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3941-3950;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035485 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460496--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peng Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peng%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeng%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pwei2@mdanderson.org" class="" data-icon-position="" data-hide-link-title="0">pwei2@mdanderson.org</a>
<a href="mailto:weip@biostat.umn.edu" class="" data-icon-position="" data-hide-link-title="0">weip@biostat.umn.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ying Cao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BCao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BCao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yiwei Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yiwei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYiwei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhiyuan Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyuan%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiyuan%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Il-Youp Kwak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Il-Youp%2BKwak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwak%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIl-Youp%2BKwak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Eric Boerwinkle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genetics Center, The University of Texas School of Public Health, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BBoerwinkle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boerwinkle%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BBoerwinkle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Wei Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Biostatistics, School of Public Health, University of Minnesota, Minneapolis, Minnesota 55455</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pwei2@mdanderson.org" class="" data-icon-position="" data-hide-link-title="0">pwei2@mdanderson.org</a>
<a href="mailto:weip@biostat.umn.edu" class="" data-icon-position="" data-hide-link-title="0">weip@biostat.umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3941&amp;rft.epage=3950&amp;rft.atitle=On%20Robust%20Association%20Testing%20for%20Quantitative%20Traits%20and%20Rare%20Variants&amp;rft.volume=6&amp;rft.issue=12&amp;rft.date=2016-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wei%2C+Peng&amp;rft.au=Cao%2C+Ying&amp;rft.au=Zhang%2C+Yiwei&amp;rft.au=Xu%2C+Zhiyuan&amp;rft.au=Kwak%2C+Il-Youp&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Pan%2C+Wei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460496/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460496/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460496/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460496/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460496/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460496/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460496/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460496/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460496/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460496/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460496/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460496/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/12/3941" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/12/3941.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/12/3941.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/12/3941.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/12/3941.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
