<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/12/3765" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/12/3765.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/12/3765.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/12/3765.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="meeting-report" />
<meta name="HW.identifier" content="/ggg/6/12/3765.atom" />
<meta name="HW.pisa" content="ggg;6/12/3765" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Meeting Report: The Allied Genetics Conference 2016" />
<meta name="DC.Identifier" content="10.1534/g3.116.036848" />
<meta name="DC.Date" content="2016-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Organizers of The Allied Genetics Conference 2016. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The mission of the Genetics Society of America (GSA) is to deepen our understanding of the living world by advancing the field of genetics. A cornerstone of this mission is promoting interaction among geneticists. To foster such connections and collaborations across the field, the GSA brought" />
<meta name="DC.Contributor" content="Organizers of The Allied Genetics Conference 2016" />
<meta name="article:published_time" content="2016-12-01" />
<meta name="article:section" content="Meeting Report" />
<meta name="citation_title" content="Meeting Report: The Allied Genetics Conference 2016" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/12/01" />
<meta name="citation_mjid" content="ggg;6/12/3765" />
<meta name="citation_id" content="6/12/3765" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/12/3765" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/12/3765.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/12/3765.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/12/3765.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036848" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="22" />
<meta name="citation_article_type" content="Meeting Report" />
<meta name="citation_section" content="Meeting Report" />
<meta name="citation_firstpage" content="3765" />
<meta name="citation_lastpage" content="3786" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Organizers of The Allied Genetics Conference 2016" />
<meta name="citation_author_institution" content="Genetics Society of America, Bethesda, Maryland, 20814" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. C. Andersen;citation_author=J. P. Gerke;citation_author=J. A. Shapiro;citation_author=J. R. Crissman;citation_author=R. Ghosh;citation_title=Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity.;citation_pages=285-290;citation_volume=44;citation_year=2012;citation_issue=3;citation_pmid=22286215;citation_doi=10.1038/ng.1050" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Auton;citation_author=A. Fledel-Alon;citation_author=S. Pfeifer;citation_author=O. Venn;citation_author=L. Ségurel;citation_title=A fine-scale chimpanzee genetic map from population sequencing.;citation_volume=336;citation_year=2012;citation_issue=6078;citation_pmid=22422862;citation_doi=10.1126/science.1216872" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. D. Boeke;citation_author=G. Church;citation_author=A. Hessel;citation_author=N. J. Kelley;citation_author=A. Arkin;citation_title=Genome engineering. The genome project-write.;citation_volume=353;citation_year=2016;citation_issue=6295;citation_pmid=27256881;citation_doi=10.1126/science.aaf6850" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=P. E. Bourne;citation_author=J. R. Lorsch;citation_author=E. D. Green;citation_title=Perspective: sustaining the big-data ecosystem.;citation_pages=S16-S17;citation_volume=527;citation_year=2015;citation_issue=7576;citation_pmid=26536219;citation_doi=10.1038/527S16a" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=D. J. Cavanaugh;citation_author=J. D. Geratowski;citation_author=J. R. A. Wooltorton;citation_author=J. M. Spaethling;citation_author=C. E. Hector;citation_title=Identification of a circadian output circuit for rest: activity rhythms in Drosophila.;citation_pages=689-701;citation_volume=157;citation_year=2014;citation_issue=3;citation_pmid=24766812;citation_doi=10.1016/j.cell.2014.02.024" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=Y. F. Chan;citation_author=M. E. Marks;citation_author=F. C. Jones;citation_author=G. Villarreal;citation_author=M. D. Shapiro;citation_title=Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a Pitx1 enhancer.;citation_volume=327;citation_year=2010;citation_issue=5963;citation_pmid=20007865;citation_doi=10.1126/science.1182213" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=X. Chen;citation_author=J. R. Bracht;citation_author=A. D. Goldman;citation_author=E. Dolzhenko;citation_author=D. M. Clay;citation_title=The architecture of a scrambled genome reveals massive levels of genomic rearrangement during development.;citation_pages=1187-1198;citation_volume=158;citation_year=2014;citation_issue=5;citation_pmid=25171416;citation_doi=10.1016/j.cell.2014.07.034" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=B. H. Cheung;citation_author=F. Arellano-Carbajal;citation_author=I. Rybicki;citation_author=M. de Bono;citation_title=Soluble guanylate cyclases act in neurons exposed to the body fluid to promote C. elegans aggregation behavior.;citation_pages=1105-1111;citation_volume=14;citation_year=2004;citation_issue=12;citation_pmid=15203005;citation_doi=10.1016/j.cub.2004.06.027" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. S. Collins;citation_author=J. M. Anderson;citation_author=C. P. Austin;citation_author=J. F. Battey;citation_author=L. S. Birnbaum;citation_title=Basic science: Bedrock of progress;citation_pages=1405-1405;citation_volume=351;citation_year=2016;citation_issue=6280;citation_pmid=27013720;citation_doi=10.1126/science.351.6280.1405-a" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Coop;citation_author=X. Wen;citation_author=C. Ober;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=High-resolution mapping of crossovers reveals extensive variation in fine-scale recombination patterns among humans.;citation_volume=319;citation_year=2008;citation_issue=5868;citation_pmid=18239090;citation_doi=10.1126/science.1151851" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. W. Crosland;citation_author=R. H. Crozier;citation_title=Myrmecia pilosula, an Ant with Only One Pair of Chromosomes;citation_pages=1278-1278;citation_volume=231;citation_year=1986;citation_issue=4743;citation_pmid=17839565;citation_doi=10.1126/science.231.4743.1278" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=C. Cutler;citation_author=P. Multani;citation_author=D. Robbins;citation_author=H. T. Kim;citation_author=T. Le;citation_title=Prostaglandin-modulated umbilical cord blood hematopoietic stem cell transplantation.;citation_volume=122;citation_year=2013;citation_issue=17;citation_pmid=23996087;citation_doi=10.1182/blood-2013-05-503177" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. de Bono;citation_author=C. I. Bargmann;citation_title=Natural variation in a neuropeptide Y receptor homolog modifies social behavior and food response in C. elegans.;citation_pages=679-689;citation_volume=94;citation_year=1998;citation_issue=5;citation_pmid=9741632;citation_doi=10.1016/S0092-8674(00)81609-8" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. G. Doak;citation_author=A. R. O. Cavalcanti;citation_author=N. A. Stover;citation_author=D. M. Dunn;citation_author=R. Weiss;citation_title=Sequencing the Oxytricha trifallax macronuclear genome: a pilot project.;citation_pages=603-607;citation_volume=19;citation_year=2003;citation_issue=11;citation_pmid=14585610;citation_doi=10.1016/j.tig.2003.09.013" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=W. Fang;citation_author=X. Wang;citation_author=J. R. Bracht;citation_author=M. Nowacki;citation_author=L. F. Landweber;citation_title=Piwi-interacting RNAs protect DNA against loss during Oxytricha genome rearrangement.;citation_pages=1243-1255;citation_volume=151;citation_year=2012;citation_issue=6;citation_pmid=23217708;citation_doi=10.1016/j.cell.2012.10.045" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. R. Gardner;citation_author=L. M. Kattenhorn;citation_author=H. R. Kondur;citation_author=M. von Schaewen;citation_author=T. Dorfman;citation_title=AAV-expressed eCD4-Ig provides durable protection from multiple SHIV challenges.;citation_pages=87-91;citation_volume=519;citation_year=2015;citation_issue=7541;citation_pmid=25707797;citation_doi=10.1038/nature14264" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. M. Gray;citation_author=D. S. Karow;citation_author=H. Lu;citation_author=A. J. Chang;citation_author=J. S. Chang;citation_title=Oxygen sensation and social feeding mediated by a C. elegans guanylate cyclase homologue.;citation_pages=317-322;citation_volume=430;citation_year=2004;citation_issue=6997;citation_pmid=15220933;citation_doi=10.1038/nature02714" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. T. Grimes;citation_author=C. W. Boswell;citation_author=N. F. Morante;citation_author=R. M. Henkelman;citation_author=R. D. Burdine;citation_title=Zebrafish models of idiopathic scoliosis link cerebrospinal fluid flow defects to spine curvature;citation_pages=1341-1344;citation_volume=352;citation_year=2016;citation_issue=6291;citation_pmid=27284198;citation_doi=10.1126/science.aaf6419" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. A. Guenther;citation_author=B. Tasic;citation_author=L. Luo;citation_author=M.A. Bedell;citation_author=D. M. Kingsley;citation_title=A molecular basis for classic blond hair color in Europeans.;citation_pages=748-752;citation_volume=46;citation_year=2014;citation_pmid=24880339;citation_doi=10.1038/ng.2991" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=S. Han;citation_author=S. Lee;citation_author=O. Bahar;citation_author=B. Schwessinger;citation_author=M. R. Robinson;citation_title=Tyrosine sulfation in a Gram-negative bacterium.;citation_pages=1153-1153;citation_volume=3;citation_year=2012;citation_pmid=23093190;citation_doi=10.1038/ncomms2157" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Jinek;citation_author=K. Chylinski;citation_author=I. Fonfara;citation_title=A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.;citation_volume=337;citation_year=2012;citation_issue=6096;citation_pmid=22745249;citation_doi=10.1126/science.1225829" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. S. Kayser;citation_author=Z. Yue;citation_author=A. Sehgal;citation_title=A Critical Period of Sleep for Development of Courtship Circuitry and Behavior in Drosophila;citation_pages=269-274;citation_volume=344;citation_year=2014;citation_issue=6181;citation_pmid=24744368;citation_doi=10.1126/science.1250553" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. C. Knight;citation_author=L. Xie;citation_author=W. Deng;citation_author=B. Guglielmi;citation_author=L. B. Witkowsky;citation_title=Dynamics of CRISPR-Cas9 genome interrogation in living cells;citation_pages=823-826;citation_volume=350;citation_year=2015;citation_issue=6262;citation_pmid=26564855;citation_doi=10.1126/science.aac6572" />
<meta name="citation_reference" content="citation_title=Nature;citation_author=P. Li;citation_author=J. L. Lahvic;citation_author=V. Binder;citation_author=E. K. Pugach;citation_author=E. B. Riley;citation_title=EPOXYEICOSATRIENOIC ACIDS ENHANCE EMBRYONIC HAEMATOPOIESIS AND ADULT MARROW ENGRAFTMENT;citation_pages=468-471;citation_volume=523;citation_year=2015;citation_pmid=26201599;citation_doi=10.1038/nature14569" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J.-L. Maître;citation_author=H. Turlier;citation_author=R. Illukkumbura;citation_author=B. Eismann;citation_author=R. Niwayama;citation_title=Asymmetric division of contractile domains couples cell positioning and fate specification.;citation_pages=344-348;citation_volume=536;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. T. Miller;citation_author=S. Beleza;citation_author=A. A. Pollen;citation_author=D. Schluter;citation_author=R. A. Kittles;citation_title=cis-Regulatory changes in Kit ligand expression and parallel evolution of pigmentation in sticklebacks and humans.;citation_pages=1179-1189;citation_volume=131;citation_year=2007;citation_issue=6;citation_pmid=18083106;citation_doi=10.1016/j.cell.2007.10.055" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. A. Mitchell;citation_author=J. Chuang;citation_author=N. Agmon;citation_author=C. Khunsriraksakul;citation_author=N. A. Phillips;citation_title=Versatile genetic assembly system (VEGAS) to assemble pathways for expression in S. cerevisiae.;citation_volume=43;citation_year=2015;citation_issue=13;citation_pmid=25956652;citation_doi=10.1093/nar/gkv466" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Möllenbeck;citation_author=Y. Zhou;citation_author=A. R. O. Cavalcanti;citation_author=F. Jönsson;citation_author=B. P. Higgins;citation_title=The pathway to detangle a scrambled gene.;citation_pages=e2330-e2330;citation_volume=3;citation_year=2008;citation_issue=6;citation_pmid=18523559;citation_doi=10.1371/journal.pone.0002330" />
<meta name="citation_reference" content="citation_title=Genetically Engineered Crops: Experiences and Prospects.;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Neurohr;citation_author=A. Naegeli;citation_author=I. Titos;citation_author=D. Theler;citation_author=B. Greber;citation_title=A midzone-based ruler adjusts chromosome compaction to anaphase spindle length.;citation_volume=332;citation_year=2011;citation_issue=6028;citation_pmid=21393511;citation_doi=10.1126/science.1201578" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_journal_abbrev=elife;citation_author=N. M. O’Brown;citation_author=B. R. Summers;citation_author=F. C. Jones;citation_author=S. D. Brady;citation_author=D. M. Kingsley;citation_title=A recurrent regulatory change underlying altered expression and Wnt response of the stickleback armor plates gene EDA;citation_pages=e05290-e05290;citation_volume=4;citation_year=2015;citation_issue=0;citation_pmid=25629660" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=R. N. Pruitt;citation_author=B. Schwessinger;citation_author=A. Joe;citation_author=N. Thomas;citation_author=F. Liu;citation_title=The rice immune receptor XA21 recognizes a tyrosine-sulfated protein from a Gram-negative bacterium;citation_pages=e1500245-e1500245;citation_volume=1;citation_year=2015;citation_issue=6;citation_doi=10.1126/sciadv.1500245" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. E. Ptak;citation_author=A. D. Roeder;citation_author=M. Stephens;citation_author=Y. Gilad;citation_author=S. Pääbo;citation_title=Absence of the TAP2 human recombination hotspot in chimpanzees.;citation_pages=e155-e155;citation_volume=2;citation_year=2004;citation_issue=6;citation_pmid=15208713;citation_doi=10.1371/journal.pbio.0020155" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. E. Ptak;citation_author=D. A. Hinds;citation_author=K. Koehler;citation_author=B. Nickel;citation_author=N. Patil;citation_title=Fine-scale recombination patterns differ between chimpanzees and humans.;citation_pages=429-434;citation_volume=37;citation_year=2005;citation_issue=4;citation_pmid=15723063;citation_doi=10.1038/ng1529" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. C. Reddy;citation_author=E. C. Andersen;citation_author=L. Kruglyak;citation_author=D. H. Kim;citation_title=A Polymorphism in npr-1 Is a Behavioral Determinant of Pathogen Susceptibility in C. elegans;citation_pages=382-384;citation_volume=323;citation_year=2009;citation_issue=5912;citation_pmid=19150845;citation_doi=10.1126/science.1166527" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Reischauer;citation_author=O. A. Stone;citation_author=A. Villasenor;citation_author=N. Chi;citation_author=S. W. Jin;citation_title=Cloche is a bHLH-PAS transcription factor that drives haemato-vascular specification.;citation_pages=294-298;citation_volume=535;citation_year=2016;citation_issue=7611;citation_pmid=27411634;citation_doi=10.1038/nature18614" />
<meta name="citation_reference" content="citation_journal_title=WormBook;citation_author=H. M. Robertson;citation_author=J. H. Thomas;citation_title=The putative chemoreceptor families of C. elegans.;citation_pages=1-12;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature neuroscience;citation_journal_abbrev=Nat Neurosci;citation_author=C. Rogers;citation_author=V. Reale;citation_author=K. Kim;citation_author=H. Chatwin;citation_author=C. Li;citation_title=Inhibition of Caenorhabditis elegans social feeding by FMRFamide-related peptide activation of NPR-1.;citation_pages=1178-1185;citation_volume=6;citation_year=2003;citation_issue=11;citation_pmid=14555955;citation_doi=10.1038/nn1140" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Shen;citation_author=G. Stracquadanio;citation_author=Y. Wang;citation_author=K. Yang;citation_author=L. A. Mitchell;citation_title=SCRaMbLE generates designed combinatorial stochastic diversity in synthetic chromosomes.;citation_volume=26;citation_year=2016;citation_issue=1;citation_pmid=26566658;citation_doi=10.1101/gr.193433.115" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Y. Song;citation_author=G. L. Wang;citation_author=L. L. Chen;citation_author=H. S. Kim;citation_author=L. Y. Pi;citation_title=A Receptor Kinase-Like Protein Encoded by the Rice Disease Resistance Gene, Xa21;citation_pages=1804-1806;citation_volume=270;citation_year=1995;citation_issue=5243;citation_pmid=8525370;citation_doi=10.1126/science.270.5243.1804" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Sulem;citation_author=D. F. Gudbjartsson;citation_author=S. N. Stacey;citation_author=A. Helgason;citation_author=T. Rafnar;citation_title=Genetic determinants of hair, eye and skin pigmentation in Europeans.;citation_pages=1443-1452;citation_volume=39;citation_year=2007;citation_issue=12;citation_pmid=17952075;citation_doi=10.1038/ng.2007.13" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=E. C. Swart;citation_author=J. R. Bracht;citation_author=V. Magrini;citation_author=P. Minx;citation_author=X. Chen;citation_title=The Oxytricha trifallax macronuclear genome: a complex eukaryotic genome with 16,000 tiny chromosomes.;citation_pages=e1001473;citation_volume=11;citation_year=2013;citation_issue=1;citation_pmid=23382650;citation_doi=10.1371/journal.pbio.1001473" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=O. J. Tamplin;citation_author=E. M. Durand;citation_author=L. A. Carr;citation_author=S. J. Childs;citation_author=E. J. Hagedorn;citation_title=Hematopoietic stem cell arrival triggers dynamic remodeling of the perivascular niche.;citation_pages=241-252;citation_volume=160;citation_year=2015;citation_issue=1–2;citation_pmid=25594182;citation_doi=10.1016/j.cell.2014.12.032" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. Wangler;citation_author=S. Yamamoto;citation_author=H. J. Bellen;citation_title=Fruit flies in biomedical research.;citation_volume=199;citation_year=2015;citation_issue=3;citation_pmid=25624315;citation_doi=10.1534/genetics.114.171785" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Meeting Report: The Allied Genetics Conference 2016" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="twitter:description" content="The mission of the Genetics Society of America (GSA) is to deepen our understanding of the living world by advancing the field of genetics. A cornerstone of this mission is promoting interaction among geneticists. To foster such connections and collaborations across the field, the GSA brought" />
<meta name="og-title" property="og:title" content="Meeting Report: The Allied Genetics Conference 2016" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/12/3765" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The mission of the Genetics Society of America (GSA) is to deepen our understanding of the living world by advancing the field of genetics. A cornerstone of this mission is promoting interaction among geneticists. To foster such connections and collaborations across the field, the GSA brought " />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
    <title>Meeting Report: The Allied Genetics Conference 2016 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"d6SRfT-A8HrRdFLuoG0MI4qVz5eDRYJLyVHKqojAcVc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460507","apath":"\/ggg\/6\/12\/3765.atom","pisa":"ggg;6\/12\/3765","ac":{"\/ggg\/6\/12\/3765.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/12\/3765.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3765"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3765"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/12\/3765":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jwjZHFNCQ0SqZJ7olWUzMThUP7h+3QD06PDjCTENWpSduF+XNp7dFt2ZDIhPPEZ\\\\\\\/vv6x685llqN4tvJXrN2Hv7hO5dqqUNr0S2mV8CMUjolidhmgy8u0\\\\\\\/znwiRzwX7x4Rv2SWAr5\\\\\\\/KiiRcrtQzS2WQdzGpAB3QZ+m2c0GA4oCncm2B1v+eogSgkz6FuWA+On\\\\\\\/OUbMP3b9Loj4kqKEhskGJiKi8kI65AX+gpC22ExMS+f6Ai3ayVqgde1ZOkZPIx6tXaTCNX43Gn5iGeqYbdI5RQ9uwAhqSgjZ7JVd7yzbfVyBdQ5G55lOxwpfs3zxg8+tb\\\\\\\/VR6GYK5OygO2th4g+SHvqP2FkC+irHPJSiQ\\\\\\\/Mc8Mb957zrzueYo3yiPQFuW1EK5+RaT2pBjD5vS9kTGQIPT3Anf3Tq+NZVuD7LHB17vRwZnoqQuSu\\\\\\\/gHOrleWeTpkaRhI7+W3JUo56Hjf0JESccWJl76b3I8z8pGthIj4\\\\\\\/i7iWiBCNzAIP6QcJ7bSvi5DrgS\\\\\\\/ItO6x7h+ZBn\\\\\\\/5ZeUzmMFUN7MxAzqFLuODMQvmvLJ6Sw\\\\\\\/MQPnU1GhFc8\\\\\\\/5oibkBRJ2pUlfm7sQm8FBtHU+BIEVTshEJxO5UkomnJ+Zu1aUSp34v74uzRQX8DmlSz7Gk6QsWr6kLHlXHXOohuuKrCC76d4YCeN3GQfCzU7UjSWFdAFfP5Uxb6BdyhhbtTvbmqFcv1DCei6HyXCb7fFT10c6nvs35IwR0CbmnumzhJHiwmH2V9auA6zObYxfHPO\\\\\\\/WOT5SeUTeNL9\\\\\\\/PQe8\\\\\\\/8TlN3AUn0uyE2hGIJyXlyV04goZdFBpwzBE1gqpKyV94v00kl2TFf2\\\\\\\/\\\\\\\/aH747IzrOuJULm2Shemj+n0wLYrbghUiJbsx36BIgImM1AZVym86rYPZLLWfbyykDmvpDVgmc\\\\\\\/7oxzEX4lxsfAr6wWalmTiAFFL16oUvX\\\\\\\/Tovjaewb+lE1f4fC74UfvQDwsZ8hQowkuVOMq08tWiBNaK7OqzSyHwBVD\\\\\\\/J0F9Heh5egyCgiHqyjtvIfsj00BHDV1fjMZi4yXHanBXUthNYwEN3SDgDdu+D1OaxT6qkLBB5VbEZ\\\\\\\/saTnNZqVV6f403\\\\\\\/zZHgz2YpPfPOqIeU398CWxqAF11baHjNu4F0NjCOFPCFKhrvCxGNAT7\\\\\\\/gRPUQudiXsXEkbOKVaj4tU3+4OSy5BhKWcBSeDt+geWuRqz7d+LjToTQSiGjhk+n9CKXN8RIHva\\\\\\\/701UQqYFxf28J0qA5AKvws8C7ReWoa4T6rXen8+8KC3D\\\\\\\/o7q\\\u0022,\\\u0022iv\\\u0022:\\\u0022RE8vZGIu0tgDoN\\\\\\\/4iCusaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b863a86b41eacbf186c00382508dfd44cb3df8339bd2b05fec35519b0baf18dd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022q6r7bPInkOOR6gq08haJ2bElk+Ti8sgFharUomEgF1G1wgwnYebP3eLqLhBKeoQFUsBTAu2a1fgpEUC+l1exedngB8Vv4r6kAgA1F1bRJ6hvfFOS4YvAUmZbm40RGp4VPG3GSTqks5GzWm5YNsKlugXclKzACXkCXbVkUg7Pe6J288BaPChcIwzCUaGjkffcdVGy+jWM8tVCe\\\\\\\/OFOJpObZjeCovA8BwJy5kkUs5M59Ji8HEFOXrdwEisR0NbsSqCzhqb7ZKZv7KxAqlqVk6ULhsPf4rT8CnLMoWigzRLNGI7STQV6La2C\\\\\\\/AOog\\\\\\\/dsl7J0uEs+BQTMrUtOs\\\\\\\/u627uqfoeC1lzffUNIpLfb88liQRF3fb6zav+4aDUkn0a6k2k0Kyoc2u8yRfC0ZTrTxYCuppa3yJgGW4Rnt40p6BGkhuPbFPWUm\\\\\\\/8nwzWRec3H2v++DsjPlU3LBo6xfNbkBnLtnk1KHLlUIrYphM1hHJtSfUt0ILJ7uwcq05\\\\\\\/wmpYEfzNQ5YQvrK9WhiRtpVv6Y6ILhP2M4nIVdgKRXmESR+7V5UV5BckxJw\\\\\\\/BHR6wwhJC2VyNCPCHm8ZR86nhueClJvb6Om97fEEemRpS0WYUiOxxBnnXutxNRz7ww0j3zxFNaaQcIDH6VPmPIgBwrZDT7H0b3XpeWZZ6prFRHEMzZSm5G5bRMM4FfVda4TVuhX7xYG86JdeMmFZ8EtnavBI4LnsPAjh3wAf0v6mZ4Tg+Tx4TTRb2sgqRsaqFzAbiNLQ9wIy2nJabmOhpC4sFZTex\\\\\\\/PvjQYl\\\\\\\/TzsK+hWN5RjCVKpEafxus3f+zD0hkddo6vWrn3hCkujP4jh2hZgVFHrJEEtt+XtXFmn3v1lgTM7dqU2OQ7PZ7umkgBAI+PbZLZB7cOp4OneaUQw44lC1QqUjIvBqQQTnv+QloZW0RM1mbedBGkcOEmpyQWVqaEIXoIqiC5TVBnHgRKlg6sKWaQP\\\\\\\/Ul43i2\\\\\\\/Hay\\\\\\\/+5V4INtHVE4t9z8dLHcu9HnEfy9j8rAQrgKi3NPc0ljUIsCrW7D1cL91zL7uxNSfBqgyRI4R5c3LdZ7XieJiAE+7yfknD+FIpIyyUzH4tKwbLekgAe0fTXBwtm5xMbpFw8WfczDH5D1gwI6vZHHxVkvgOe\\\\\\\/RGGDsPg+0e\\\\\\\/3c1CgdzlIjSZ7ZZKXda0NPZNJ7s3K69CWJ1nre+kx30SzBMnZfZpeDcNeb+8njF2711\\\\\\\/4rMScUDcWsoy04QURyhkEqc3qKD60n0Cl45bbZFmC\\\\\\\/qtazPlvEB5v7Fh3SjXG1w2kjFlyUpLZvXHdR0BLN19zFsZ5zGnqfxo9liyx2cSyW+NsUS62uxZI00F0vUmCxK92xHXkAoUiKiwbaRPX8r3VBSENQ7OT7uyzjhL72edYKMBiwpHtbJvy59kX31hCIeXNjvZjN+j1II3pkZbqcooa0z4\\\\\\\/kA+QkBdz0uBH7B9yECUtsPuiVphCf7fnPuoMXQKOLjCl+4Mn\\\\\\\/WJ2ruSZAVcuQABUlqxSHrwhYw6u7IRyG8\\\\\\\/WXubpDTEqJ3dZaPTwTupQAwOK2hulWSKpAt6oyimO\\\\\\\/CPbqazCEYkQgF8Rq8nL0TMGprOMjqidLY46LpP2dYrsPIy4IxiCK+jq8DurxfVbIIb6daibPph+hlR6KQJi6yksLm+6OAbbzhnREX0yg5kr\\\\\\\/cvQ\\\\\\\/wTEgaK9D11FLUDsDUizGUDbYBnAy\\\\\\\/wK+Ca6MLEYAZrceMY30x9ZaX0qxddsvnRA2KSRBb3p2+VBMrmT8CFEZBkg+fIHLwC6slkOb0CWSrWr+Ul7ibVUwyYI4B57VCJjHn1xr6orD\\\\\\\/nyFg1\\\\\\\/7lwrxiMVe7JjlauKqlxrA+o0jCwJO7U+c3ct5YRToULRXkE0MXY3Ufoy2UB5vHE1lK70TqMYw9AEZ5x0IgDrGLNGsBC\\\\\\\/hmHxG09jSQOqyux0u8B4nwLJ0StpOt6iHumKbQ5uBq41t7hbIzYor3h\\\\\\\/D30Ar6ngXNRYzvam+0Wh7XltCu0gEekAsQMHSffyG4ItLKQWdVpqP8oyCNEB5v5TIM5ioE2wVeN+9FCb8ompbCGZN2FnwBRHJCTZ938mOlPapsnHnXTsWvH05oNtT1QRlFzDoOL8pD37mKKLA6VVcLbc+tCbTF8JAXUnIFt3K07Cd7a9uo6mRphv\\\\\\\/iYkNBx98B10V0JUJ4I9L8SqlDNBLd5cttE5+VwsFkEizuw9ebbt0eoeKJTkGdLTPp3tOAZCvMuzj2GY1rD2d3rVbvolivnzefeJqCvJsytvh8gfrQXfcQTCjQ9kg4+uJ8\\\\\\\/DCrzIWIcqmNdRjwxtDwNhQ7t2InIfzOqnaLrAbNcEDYte+FQaMaCpsbsIHaawohnbA\\\\\\\/b2DwwUpHaFYiPlLNv8Y8erMce\\\\\\\/Knp7CKhBQYfBZwSGENTOWVtTcMHBsrXwRkJC1tRmxuR54DCuieEaiywdC+cao1Jx12op8gUJb3QvpSzkp5G1P7XvZ6CjV7Az1+CCz7tkErnvGWT8ew2NDsu72PnGkMJq8JBDIgPLHI4rvx\\\\\\\/remaoDbGEr7vQl6ZAld0vwFYz4vFmKTpRmZODzo6qRFZt4TIzFox3\\\\\\\/dBW3QbkopHQbqB11eYRJisiEtc1wXo8F56X1qsgtfCJ9rwOfqXzg8VWJy4tQru0L5oB5gFRoJX4EOwRlOYZ2uqN0UaQtedIuRD1ippG61lc2iDMbqell9d077wE6B5593GccwycWWkIuk\\\\\\\/xHVnYmvKafvMwa9\\\\\\\/n\\\\\\\/1zCFjIZ1ymqPED1Z89731BUyxVpKCQ4RiySXWtG+EUhO35lewgq22CfPqw6IM0Z3T3ohUrpHo5uVTHJP9t5vACqHh8H\\\\\\\/xzg7wdzCCEWt\\\\\\\/ebaDAqqUiMbVgrepq5p8m1kokoxS1SlW0Tyz64dqImoWZoC6xIqHSOL3TwEODcc\\\\\\\/zhHxYAslA66hdrwm\\\\\\\/VvINe+UGE\\\\\\\/xfvA3Y3RX84ut2k\\\\\\\/BPrjAcTac4Hrw08Knv8OK97TS6C6gN3Y8DTQvQ0wmFk5oZ3pXwBMGPjFCqIKLAiPC7y+4cVvnTl8mifMo5IG2argR5pVLcgSi4\\\\\\\/cL7BQLEM\\\\\\\/N+Gk\\\\\\\/TewY6N0iMPd7W7D56EMyEWX1G7vcCfGoSTdcW8buc+kF5DgFb3I+MVtU4Ogz\\\\\\\/AHsXAONHUc14cySvNtSXBavctdqOd9m2ruIwjhxqW0ISklNItCsk+8wDcTsn8EMtMTGGUayvg+dfazhZ4cKXkYFzyjbx23qHpy9otVlEx1W1AP+enj7jeJGrw5zfzcst2bv+gr9B\\\\\\\/GPBiz4R8iYNzqpi6X\\\\\\\/NaHLCoRzq1U4BPkl+giRybFd7IXmocf6KgAXiweF7DwlQZduV+uGKvbxbTlyVXkSKo2eWGB3vpJ8NRePX7b6w2aHzOdfbaOHq6Bxd8Agv4ENA4oTT5mqhbzlM\\\\\\\/Uhtjd4oXVUyFFNrJDiRBKUroZf71SRBWfLc7k6wEn4BNpx9SqLrTbuAZOfO1d4p8O5BIIPHBA09gqxIjZ68+8t4PT2DA4FR+30akAY5EdZ4aZ6+zyVMIzrjPHA6cevuH6QtgRfz1Kii2VlbP0DCnBCpyTXzSPJRF5WDQ7jPH0eCOd7MHGkJSM+pev3qAJjvdByMiF+H1A5YuczYQW1zjznrXmJEYPNm7O\\\\\\\/KhKDCMk2iXB3N6IhxAjU9GE3RSqc14B9RTGYsV1Cw1I+R8uED8\\\\\\\/SgELAQC1SHcel8T2YgVFTPa8qWn94pv2BwFPWkor4kuSbUBkPsX2cg7dYHGvQJS7U4L+Vk4MXdX1h5CYHgHDxnbIowh2vtXsAb\\\\\\\/dRbtjWIttAWZC6mO14QrKmKKfkZHHnJMvGneKxXht7gnwihfSmX0Ob5oKko2hivTHwGy+We3h3X4HKCnAx6eUv\\\\\\\/4uO7dFaHCKX4EiMkUMW\\\\\\\/Dcd9lMXUTWPRkNRBa9bpwDuu+YGHnXT1w7tsMepIt6C743CXyTcml3Yqa9QXLpbMCDRy4sSMXMz3+nbXtAfGBP3BaClAPRrTlYwodfDNL40AFUhBhe1QdTNE+Qa5p3htTHaJPzJcQIhDMRQ+tR0S9iGKDmnprK\\\\\\\/pf7aIaxhuNiEg3gLifR6pjOyjqypHqUNTdGGwMzhD8HGyLvbJ8FbGJXJB9tIw2SFDF9CQ4+0uTLfxsyxqYwhB1GPsz3JxSiDdwXL2YlRM1JSTVcXN4ywADKJfeXEDIbVsDU3gu7rHFWxjhCPR0eqF97YiwTkQPnVKomQI\\\\\\\/y5\\\\\\\/tmDJ3Uo8XIlqJ1YJnpmNPEA+Tkw1zz6MrvlTgYUuugxiyKuVdbG+HHOG8qqF4OYQB54Px1s0lqn1oLu1VaRNeTq9npfkm02Rkw8n\\\\\\\/dG+4n8ztTakNut0UlQMX2HuVE6\\\\\\\/i4LT\\\\\\\/FYOfdKewAsLrkYvQ0hUVgd3icvT8YKO0G8e7rocjI2Jyz6cuHVhNZN\\\\\\\/9MGhqeecd6DExkSd\\\\\\\/DNVSY\\\\\\\/zf98wpl660MFWdlMzbvmqINMGNBw7ouJWIzpwY1mo2DgesImqGXxYBgYGvEjGSQKwmg4QrFuWRWR8YGPltZqFx9MvCL3FAhAuEhqIR5+Y7BhMNJC0AUwiwVu7EAfO3lfFdxj6lnkKXN0vU6xnDGNAYgRGJomoBmPKtzjsxUvzWNfR0F5i+rLCLttpVDWNzys5bEz0j1gajWINXOj0fX0MVMv12xJn6kvuqAV+bUlQ4eL85t2qK6WuIvucSnyRY00a2BSUtIRcrnF0HdaN\\\\\\\/NYjQ+SZ4BztDlX8tEdPCHiTDCysNN15fkI9\\\\\\\/Gmf9lHtU8kzollViejstXsXGAoobg83NY6cyLyXkMTsanlxs1hg9Lk0E4bAVnWFuRpF5Ex9XbZ92CtuR4ezYNxgoTh\\\\\\\/9LdEP3w3kCen8905a7eco0iFetqgsfSG4f\\\\\\\/+ZnEyXrvb8toEKvbBBtLyEVhrco1avmkMjmvAOtkURHmNG2irAY23oVkFOeq4Jv2hUYWOliR51FnRx2CeADcn\\\\\\\/34FL6tYr42YoHnATeCqT54d7+HAxm9fC+vX+0QAvY4LjS2BIeJ0fd+3dnQ\\\\\\\/eK2Z4vE+uSBsMOLA2+H2tO3HNBXBg\\\\\\\/OUqFkrYJTzC3WZ1cRpPh9BygVQMJRYH2OOiFXhnSygWKHJJprrXnUZfl50aV2YGdSESeJsrfyedDpVXBSN+yTocdqKsYSYRom4JevatUTnpXpRixyfzIleqrcJ8B9QarYgX5\\\\\\\/q+aUfE0qeyiacwtcsPQhmuVQ3Lw5k1b6OgQT3Pn2ofstVMpDK96zXFrTYWRuZDrApLM18K\\\\\\\/ZMTH75EUYwmqxW\\\\\\\/2EG0RsqXk\\\\\\\/bznEyGCNo7clLYaAxDwdnVH+Sq9HMoom4g1YZ\\\\\\\/TMf1jGYUFbBfqIOy\\\\\\\/f\\\\\\\/BlS5WnLBSCzEmPWsiPHLKB9yomnsAd64n3dwaAHK0rqlfo+jxdK3CJKLZtP41ETbO\\\\\\\/ywV2XcVgBTFPumwH8bB0P9bjkdlQ53N0vlZvI1OUC5brOIniF6nXUZWsgsziy0w+WcRTbiqb8xx7HOWb5\\\\\\\/KPZONMBP57xE\\\\\\\/4\\\\\\\/QTBHwMHRNX69H+LTc64N61+EHmm\\\\\\\/JhTXnxukk4KlotQvmcDjsRxlVmZlMEOKkGUxcuisNTalGdLotCudaxQqlhbZ4ZR9H4M1RM2tBFRf88Hj1hmxePjbgXi4EkLe09Pgqa7G0pN3FntO\\\\\\\/lb6N1RBC87+YFIEwFkN9OLmpt3Bpe7xYGdjjNTt5pm6jbfylZqbOaGAJKbP9ngb6jjWXWjQlbiAu9OiMSWeHHQKQwZ4UH8GeHRhKyscGamvjTi9dN0NZ7KsqCpYTyIcev\\\\\\\/Dp0KE7iKrjIOCzQ1bcBUZUZBRwlyMJercmOfT1OmQQoAN1BkSZkhm2Uwz5q9kB9TdztsZUeNZbEs\\\\\\\/Qz6cYeRUCzqxmBh5wfdR5igIcKVbeDIJ+dqnih2MfagtwPesFsyVBsjiQ303iW+hCtkz\\\\\\\/DPxnhjvQ11jCC4Aijc\\\\\\\/Zb48L7tAykotFpYzx69HWylTWLI4bDP7a13+xx3FRLpFygQJS2fH\\\\\\\/yZc63z+CrFZPGNDRk9fjDzW6xqLIvh7r8W4HXXv+trbERYogaPF7CAec06fb6uo\\\\\\\/ugBBb\\\\\\\/MG\\\\\\\/uC0W2AZLQWTdAntNCPutzs6q\\\\\\\/gu1nTShQNbaKv9aze5ABFm0Hhp7Q6vhpMr4Q4Q2TXZ39Zw3jeRCl9ZTGZI9p9XHbBiGtTwneUeNrZJjV7QJ2LoU1KkaZbVZM2GzV1B0bcvVezAC+ybpySbsioZggsJqa4KB2VLVyt9FpjztCORKUweHCzV3cgTlOkxhIJeC\\\\\\\/giXUXY+lsau06VQdgHr9TivowI6jKuVUrAGgX0V3Sd24kyMvPHPhjmx9Ucl6S+Q1760SGE3UbyiSFMMiuwWV1FmCaps9xcaRVBP15VGwXjcSZHnC6bBMpIB6l\\\\\\\/PDCzAN2sMmrlAeCkcqEd6PnD0BuzWJ3F+GhZAqLxWKXMFau8uPKKCfvdk8JSZADSzrxUadR1INbrkwVJ9dry1Mo\\\\\\\/dyIbeX6AeVEvEBUIYQMuLbG6GTlACYDeRGtLV3DZWSp28EyO21zeNpnw6O6W76vmFEefU+7NPxOAZRqNYA3bhdO\\\\\\\/QmsOxMGyxjuzBFr43JNNenPTpj6L0vWRKeYVYjSEI600uOcQs\\\\\\\/703mjvBxmJj5WlvRJWoRKasL8E4Cv6zf7NCn\\\\\\\/rtqnOzW6IrIFP4udGeOfK0ZU+0C2ZC8+rYEczEhr2PN0vittsRfwOHUfufGNGX2jInI8+qhuEJFc1oh4P97ztl3JEXbbzrrNteFlh0\\\\\\\/eFo7nbT7xDF57RLCHKKshL7YU91aHppWOHMfkKinP6yd4FJnACs0b6XY7Q+V8j9\\\\\\\/WNzJkbpMXFJG4zmkF4HGohXXC\\\\\\\/+j6wVhzOTfJ\\\\\\\/zL0GRwDACjPqKT9H4OOVyS0CLlNXisHN4gDwLfRQYO4ovtKbo\\\\\\\/uhybfpeHCw39+FKhvlxLCDLk08LxujcSm\\\\\\\/YijG6y3nOWSPbF4gC8ixPSmMk\\\\\\\/JC2n0k9aQQTl\\\\\\\/aUz9xm7p53et5seIQDAJ26d8OmutELIQPsewNvRffMLS1juR4nIH+cM0uSWktN7V0MdEYQCuTEBG2iMzm87DKmkaMxMfS1DL1sfKqKidg6xPAwPULGJluBjwl0rV8Emfb9fKg883GfG1NPIX4GynB8saNWPSq+b2z5fHeirK1imx0cZCpVcz7e4xnLfD+EtpY5ScjD7P2H5hq83RV5bKcTSlV2ZLyQ6whUmlwBZTN9FCZt4jWluSnEPKflop3KnaTQyV7PhX2sjHJDUX\\\\\\\/PGY5USXQZw8yXvZVFIMXA5rEp\\\\\\\/3+d9M+qDmbLo03glQjBWhpwcz0TalL1aNVYveQ\\\\\\\/BcUn6s1PuCj7u6D6Q9mid\\\\\\\/I3emSutVj8M5NDDFDd\\\\\\\/C1+XqS2qghuzVM4jTm4x0tpB+SlFnNawnGg4B7s3aD6mrcHFrk2M67ipYSjRCUr0mAgJBgha2RTIfiTIpifHbM5U4EjkF5AhYovywfNpq85phG0GGQHSCdgKxoGnzC79vmxPfHducukQ84CpIi0P59\\\\\\\/QQNs\\\\\\\/6wyj11daMUfRbSl+BDLSBjsXqweBMh1yaIJs6TGQB79XqBIgrS+BtITJ4FqzQhotFb9I3BIkFNFGijhA1lBfBBvbzkJ9Uzt8lgx5J07wMAiizsgnzqfEcgmVPmujCfn4yGta3GXgOCjcKOwqawxFY4RLZWHKfGoLehrsJPHE1qgyn111BejWVMqVG0YI7dua0IWF9Cg4pe6C1eo7n2tggqp4GCUcvaPFda4k9ETfnkvbFTU8U3k1OGyDpUtlciWNJUBbmDr0kr68UehuAo\\\\\\\/d9tpNgVOx3gynFeZ2iT4zZGGSwHPG9uyTAZx6sFrTik0Jf1AeQaP8RlQOXHiFKMixjhL7XOhivXAF+MwsYH\\\\\\\/x7mwxMepw0vskNzWl7P64ffCnW1Jgakx9hg5EL8qpeIn6YfSJoZHJJ3PvMwVLThYR4nL4iOVYALH3oFLzQeJ0VCJkEW8yz8OgoxF074vsaiQ8EHo223uZBEJ59CXxNAxoJiGTkv3aclnPm\\\\\\\/xBS1GXjawfI0bwLrYE05oPoJhvxfO+SJXBEnr27TJc0bXuf6qelc8bqojCu+aDXaZjX0j3I0\\\\\\\/kEyw8YmXL7gLnDF9+i2MgB3ZPrCOwacKY7usZ0uKsvUQ98pPwSeWuDV8vCJjmwZ0pB2r\\\\\\\/Kk+jvcwWq2B2qQsQCS08\\\\\\\/Lqm\\\\\\\/idwVpwrWMwwwGqr7AIaBgfdoJ1Izw0V82BueigFpEV3sRhaGdx95QYMY++T\\\\\\\/b33PyPfGbHRj\\\\\\\/vq7pmUYDLxPvUlYkJQAzBl4SbZ9nVH81chp0HEvLp1TPcY1e8JMEaKWcyxE1mbCWfisSpErHmROWxYpGSqBKQSfcLq\\\\\\\/5F3s70l5eKtMOFUt2BYUaB02n5SBkEGwwrpD0Ejj3cyhE8zdMEjY6vqSce0i+sb3lO0PrDiLR6Eca0PFeUHjuLhmWeP3HXNH0ptGMQKDV3tnXeBVEhHO\\\\\\\/qidiR0kKo+WY4q0Dzw7baGwL3h+CxOprAr0eORBcqo78xh1nn\\\\\\\/I6qTX4p3tbnq+KjyYyQ65SnGTVTo3xPN6mn\\\\\\\/YRhJhumWm\\\\\\\/gmnizBtRWztFRkwwJOW1xDj2LibwqLFXVK9BCV+Segda07lWY6ucU0VR1iV+AaxB4F9FyTj9pu33GSxepYnYMfrlsYhT56FNznecsmClIW12dffdwGtibYDpy5tT6HFBaZgT28sZ+X8VqdrjDO5l7ASqey9mlX79SesgwXjw10cPRGiMXCeerQcB91PtyLEbMYTWu1FM6ndAbASBifT7psKzDhOrs3y\\\\\\\/TAMo\\\\\\\/JEjKfU+nI532mARN5kJcW09c01bs3cvSb59RtbajM\\\\\\\/wnWr4eJT5fHRkTNIGISJqitD143kkIz0FyH46FkKLG5XJibt7bcWr8DQA44XAL2GIW7CuW4zcUq1aP\\\\\\\/TK2DwqHGI6J1laRypRDcfmxAMiG2UKSvRz9SKHM17QnquWeSx54P0VE9oKvXmjQSl+RV\\\\\\\/MLP1TSWaFryBP+tJiLaL3BLbs7OLGjQPnHXjP3zQn5Sb4rdNEoJ+85wvyKDQarTbix5ZEhngzsS4ml3y1TArwK1HolcEZAskdYkZgYjEgHpGWoWvbHo+4yxustp6wH6+Pu98rsTbXXzpuz4SlWF12S0Es+unHzdbDg2TW2t8yGM0bJfYI7N2cImZiHnNNgoKYpBsukfinuuBgb+J3Hfay1neuD2DMDRBuRavpXwoR9sHKT2dWwoCd9bliVwA4fsPmbvSe\\\\\\\/58jZ2olyqrzH0aaKcDHU4wRJfQE5tmc5AqnFl4GVbPOKl\\\\\\\/CawP\\\\\\\/rI1S626ZSlHyKxp4pGlJ6k9gA3LaExqAnU5jtAgqbaNwI5ZaU+JrZy23Jt8GgywLBIxoXHh2reVB\\\\\\\/zEKIQ78LNu6+CFXio6lwBhiNQC+dk\\\\\\\/ldTEs1CxhSR3Rf4XWywb+R5AZ7ZMgBaCcp7e4o7Hempjs1eKBGC7U0uojxPb+hURDSKQxArCIaa06o8xlJtFjdzJoeolQYijDIxrW6m2E+CkOxIY8kwpAeG\\\\\\\/hOgoWFP0ePmTbHvpou2Tn1CPobMIVP\\\\\\\/wLA8ZbuR1YZJU3KCYqTapZN5axFKe4pf+e\\\\\\\/46B+nEyhgXOEfBceEQkM1MHia51T6q0jcbZeWx2bPgYUtqvFHJ6nIZ3KnZWRDOKyVss2CtUXdY\\\\\\\/8va9+9OLtBebSqB0+tqZNms3LXXvGjVyx1jrOUUlyGOxxToKhncclP9VQKRuCHq3LlEUWzDuzdbt0phB7AngfNo4hS+BvgejodBSwcxZBhi2UPP1VQ6XiR\\\\\\\/RFBmuRpJGf9lMDvuLcWhPz8lVbf\\\\\\\/FfuEVSwHcXqQD3LZrdYGvKFPZ3Dje8hYUPADazEW1HPeKo8bpoG3aYU9reRKQ0IWbVdN69MiY4mnFsktGN4nkmOIF8Ed+8SjmD1VcKAt1b9x1yq+bMnqacCldDHC9JifhrDYhL3wtj+2xNsRw16siotvcXkpqhe9\\\\\\\/j1mwF0MRxgyR5HxYj07p1\\\\\\\/Z7da6uVSG1ahF4ZvYlAuyKW8saSU8yNaxKOkpqhqI\\\\\\\/e2vdztUxttrY6cWsQsL5JmkAYSWjRMs3hhnFGmnn6qsCzgMkIZhhXvg8kDVj1vuLTaezTHhljns+2ENO4bKmmn6kHQ4uIGnae0Mif8fQzEJ+dfgvNP1Ttjvva\\\\\\\/TIjICqNDlEXDlfKnUsxc4MB8N7QNbOPp+IDosvLLDYBVAIBFSuABOtCAD3xrod\\\\\\\/97PIo0n+EEr42Vn00jf+hQPx5BIiJsQQ0fPFXllA8R3VQjteDDqM89LZh9hhktU0YHVgib1hn\\\\\\\/uRvvfLPzqRZVl60U2mwfr56vgovD7SFB0A0hX4CwGaRlNbccVASHzs90xrPa76JAtmHRyMAdGWjnm2sHO29petjFED7iw\\\\\\\/JBLaKQaShmdIwubuLvGEYoloRCtQF0K27+TM2kvSDbyfDgjjH3vsRJ57uwlSwcxRsGP3ik1l2HPR+y0QR4OJ3UeqyR7DGGhQ1uCijGxip8vGzj8sA4G9BeMuIALyx0MLzBVyRTu8whLbMUGATnrskszwGMw6YoxQEb5eibZZIm\\\\\\\/WnL\\\\\\\/\\\\\\\/og0jRWMbMVFA9ftvtHKwn+xn\\\\\\\/8ynnvzeW\\\\\\\/O\\\\\\\/9tQRiyO0XLjTxjBp6d20o7yRvkSDsYdPFO45znQ5lSBzzDSlGvvnM2HDB5wE1yHsFmKubcOY3pFwIU\\\\\\\/JZEGlWX8VWB0z+OYaR1moDPS4nmtUpGUFE1k3Bi5pD94JjADY2Too9LHlKU3MYKHEEhjFJPxUZ60fc9GbFBult1mbODYt+Y99VMr1mFfs\\\\\\\/CsDw38\\\\\\\/whE7VMMTycemWm6e446P\\\\\\\/\\\\\\\/BEb4wnXNLuU\\\\\\\/CEYXGwCEyK89yL1T4ubg8mHw4gZ8zmS095h4MOEUWUDaX8fL1v+4h00PRrLvanvchrc5ras6mR07ChJXEV1YBCWRCGU8ILxfbLOzb5Jvs3AWGguF+7Tn1\\\\\\\/C2Aw8PpGhgvARUd\\\\\\\/ZJX23JJbNzr\\\\\\\/eIoayidyHGyipmVYb0OBpEpWnOrm5z8l2PGAn4ccbErSiN8Ey3dwemqCqLFD5ttVVoU5q1uMKBS2ccNZUKQVnmbTPHq1SoloUXG0r81umRqrveDhlvexauSChInI0oeE51bLlGdTpLMB032jsMnXHaXRuaOwgUDrqVrL3FXFI8iykc6GYnSZKz9lRW7xMMPb1imMktmxW0B5U4ZofsHsr\\\\\\\/iXRO6kxsaB6sYXkqDPCkRlDs1tnnosPZyPa9uqk0GC5+iA6L8V1fAB7M+AfX7G0kAI6vdjWAtlb1Q79Vprkvg+RauVcWG9z4QqAri4UZF9R630kkWuS9eXX32eIk1bgE6wiZepo9QFPyh19NiyYvEbTjhO+4zyD3x0nVyh7ymq4auOP9JmPowuUCkDXYKpm9QroDF3QAvSPaTOJBO7x+DpIKq\\\\\\\/hAqFPAesz2RoATIZjBpR8a\\\\\\\/9KGshNoSwfxxlNa7EweD+SUYmhEJ8iriko4LOIjLHFHdN0SkTC0ws\\\\\\\/ODTVrfjFQQV9npYALQzayvqcZEj\\\\\\\/+dHRRFF0mMSZc0UxFYFP4W0hxSY3FwOzBrr5KTEzmoWH+M7X1+u6nxL7jg5jciPyiAw9LlSjnfxqODcpD5xXLYJFvw4SX2ln9bWeliSZ7mauq1xZxUhmO\\\\\\\/2MhS8b18CJi563\\\\\\\/Cv4TZuAXep4EHqCKOCwpMN3LxiPbOmQao35oPTJKIfyOTOTC12dSRB1FzUjc+dZTPfLmC2Puk30jsN+TymxTPTsuzwjo43kv1sC0Kjik3WBWM46xLU7rGi0aO7HsCd66Ht8zUPlBHmx\\\\\\\/lMoTl3Q3B1DirpdyTlqgq7z7s+hKKut0Kc2rCVOgpLhWFkfBJS+CqLgVCrfKCAU\\\\\\\/\\\\\\\/XjxeVD1rxDrxJhrPV3K1t2PlXbvK6ThDwqq5GRuazDEZ5PFzKbJXGoQG1MyBKD9N9CE\\\\\\\/dMapXCF25ycHjY3UviG+AGaEih7SKkY5CTVkf5eQ+++ZgUUqMwuEVBECjOfp5wVUlpOPoBs8CuWFZhRcIPIzOIqiH+utQpcemL5RJ2+FSA5oowkLEsVAowCxm\\\\\\\/nu5093Qc2JYQC0slUlUVqz0A2rJOGuW5h\\\\\\\/+SDAVSBPPIkSkfWEqxMY1lGarCKtz1EdcEXW4TpQCZqARDZ6Bt7hOuCWXzQrzKCuePPP0LAMZAJNCZLUAsVDNnlYr9GhALEQs84tcVzBjjT4O+5bY25GZhOJsl6AvD0fGMH1cEqN3sOWOFayuSxEYU\\\\\\\/97qBFz\\\\\\\/rfVzomBiaoypztNag82lIs9zQpmce\\\\\\\/V6grLVeiXTNLXNvb86m6DCN5mjO19ZJx7BB74ICKvS2zRG9v7r5xyislfJUdECfseGdYtzsy51vtmuOv\\\\\\\/HO0X0DE+3U8sKPN75Mvd0KRNgyOl4iLhgxf29AzRSJ+jvtwyjNhgxIhttLUAsrLFhUTQuuYAJImhhVPrBo3BEN8f7opt2ixAYrwM3y5c\\\\\\\/ZdOoKwzZe7WO2jkV3PPv+QAp3xmFOo36MjN5vGM\\\\\\\/sN+Z5YtEAeExnBBvnZL\\\\\\\/D4UMvOQ9tCI33nmX6cAn1K3ZW\\\\\\\/ml1j2Ie7FFM4ps6ipPLXQGcp1YX4tlEJFTpn+S12jcp7dcY+sDYjrTPkJnLbQ0lvkgpj9ltpiNt1yDfxC1hM3rfyQCnxijDewOyHlbH5GKVX9+bkn+Zq\\\\\\\/ilpzIQhc0gFptVgOg3PYeQxPk7lN9LxPG3wI8Jd9pavjR29Xoaiu6LKHuvbmFMg43XL8rF8X4iQ7AzRKq1d8Kc4U2yUJqoa0HIRtksQz72EOmGUz1AtU39CFxBqhEIVDjpdl8IxeuqaIubBJOJ4fWbrVvvT3qX4XPP4NokSTm10253O67F8FWn+Z1tijAb6ctRf2h4EQXDUg6tXRaQkE1puEI2SP4xAhXb7G0eg1QPAzFlpflOyCPaaO5D3rO7GNNwm\\\\\\\/\\\\\\\/aOXur1pmg1LFi6liGeGXNswP3deN\\\\\\\/5enB6xQmi0pblNTeoEj\\\\\\\/BuvKwSFR9kM1x76EMMEDjhbzmTfAgM4PDUeoSXK1zNd0WFOIAVZs0KVEA0gM58LG6QVrTRtnugBsfXUiPZYsf2JJG6gjpNqbtEm8wHldrHnwVV5PiE7+qIybKznuAXWNtQ60pfZMLkZ\\\\\\\/H2cst3EIgpkzaxMJpwLj5fLr\\\\\\\/VD34GX5m3TJam1a9KhxkMUc38meiUgGNpTArRNEIy4SQ8omf1Qgmnb1PcAba8Ety3jivyLkcAU9xsdRJxQekoMewIj31C+vmetc3IIVLoH3HDfXEyvO9ms5H55eZtB6NMeNfkw6nF3lvLHberEtD8OB8Xt3lZXmu85LwIcoLJyjozRuiJ2xS6M9ITkNNhPmCDW02Njfmn7GwSnJESTm7lxmbvV91pLeSdsCdoZ1CchGkcCmhx1aEBYoXdlZLndRhtN0+OuM8tzK7MRuY1McmqHDZ3L8ZGKCRRSnlYaYuWYJpY1JnsEtkZtuQqxRjkh8US\\\\\\\/Q77YHI80nvTRS16lZqplDxY6e0Eln2qGPft9UdS7VngBOpQ5SDvO5ggiRu1EMybbZ\\\\\\\/dItDZhLNgXMFQuUiWjof6ZAi0FUeK\\\\\\\/fl9eIN8aigJZfKmjjiUL\\\\\\\/9Z97BnwbK6ebpCB3ieo6p0dSAgBKIb9JKJV5PXfJ+hxNDGKKmVYrN0z8xF3DOqCyadis6tZ+m62je8NTDicT4SWzhPHKxZJnWVC9gSCKi6FHAAqi\\\\\\\/Wkp3gbO1TKZjdFHtkeEZjRY4hzL3PaNXJPqtxzj3qeF6Kw2AQXoJDia5tcKcAY2fl29XF5vKqzUE0z44Dmgy9l6Tbuj8b\\\\\\\/BiqtItNqVUEaykc\\\\\\\/hhLgrJcUVzLW2wLC5cWxevpF9Jn3zjIGr5mkmVEVB3p0u2j7DMZGJ0MfU+u4naoIZxpYC+mZs8Xl05iFSLzSJxWrXa49AHBSGP5HraZt7wMxGblAWkjiSbQS\\\\\\\/wwabvfqHUQDNypB\\\\\\\/\\\\\\\/RMd6nfXdLGK+RvufwJ1XteVN0U4JApFNRauMDnOcNgviUvnWwb7tTZjlSqYU2ETtmupeLUqO+WfQaqguOPt0vxrtiu3V2NmhtuVHEs9BY6FtnnCnWSxq2NtL7\\\\\\\/\\\\\\\/h1Q4XmOydaQ0SAtWbvIRmrCGJ5LW5R1VsvyEt9tmMXaDfMJR9pFSXC9v+Zzp57ACjxVaJeTp4h4gYVu6pF4Yc\\\\\\\/Rd1BKVs28rv5ksVySHfRXsGpg9GVNIwycIjowKzfzzTI8yT9ax1OiCzDDDKsGFXTjfHrlps7XPqkH\\\\\\\/VBk6WKVEPWIXM\\\\\\\/RyP9WFayEQ1zUUYz8r5uKeSBlMLW2RWQx2CSjfpKvFjk0axrJljJgJFu\\\\\\\/J7vCkYt7Day9PaTtJRvPNEnyUAS6of6p\\\\\\\/otdK1lHWh\\\\\\\/qJ5VrN+y1x4PgxT+SQaXRAzbOIxTIGzQ4vEPpgdDZOR5C4hKUXtIy602+beiAgZQHTo8B5xCJaGtLboiPiX3lxd7TWmoMTuFs+o2xfEz+tpIxLrO1rcSy89x\\\\\\\/Rtb4A9\\\\\\\/OEojN\\\\\\\/QkhPZjM6+nXYvStZPak+ZoHZ+Xke1XDz+P\\\\\\\/iyn+cSKdAa8EY+V89oUXekAPeh4DR2CL1fyNGlflsRb6U2bYvsxgy6uOuXHsYnHRBc1JRchMkqSwAfS03BSotRFSsSsXsSQBKeWhfqwiXolLfvxVDLcM8KJyH\\\\\\\/7T\\\\\\\/HQaNw20ngql3sWKZERs6F6E7np2emVLKU3V9+t4V3G16LdDkGQ3eT8A4znWT5OxkZsDuh9W0rZ9IQ2W9cqXpycSMLVLreLt9phCBsGp5UZb+boLqURyaG+1UOuIbcdLf8R5j7Wl07RLXWtzuVFzHH5oJgqIDY74ZPBzwB5PsB\\\\\\\/1f7uMWbcEpF4ryBfQG7hoBefniUztm5+PFwTYeWjsPPZW5p0\\\\\\\/XknbgF4zqBjNwwVAMLj1KZWV5peQ\\\\\\\/F9yWyW4R9yYV4l0rMZ0YVQJiJJOq17w4JrvBuLOI\\\\\\\/EzuXWeFJOlOYb41XGbTxV+zjLOaGogQvXV09a1bwnyTTcoHdKpWqctMmVCWXMD2gvZILoZND3O3L3wz0V4RlIwY8eolWcKXE5v9\\\\\\\/eJ8j3IfSL5sPHbQN0JLOMdwmPf5vLkMrB7+j6QUxrBUynw51GmNhJYRBgc9OtEMBELT5+O5rGqxiGDYJBYC4IWP+mnDUkGbG0J+WY+0euBqZqi2sLAky7EENXNLOTlBGID7k2huqZc7N1dmw7Wq9lA4pPI9K1wMb7Kcj7bWVqEcU8sahz+Z4XizsFtcoSfaD66qbR8o64vS\\\\\\\/\\\\\\\/pCsBEeWjJNRQdXghuY8mrTv2oZaRMqYBDFhmlnNDf\\\\\\\/097KrcOxqPiESNxwC1GyDb9Mq7sAnF0qmKbJaXgIItVPCKUsWJBG5j3S6Jr02CYwbkuOH9RCiAr9O1kpUfde0RLNP5aYLJktoh7pGCUrKhlRPGenJXRia82ba7PcdRsjksb8FjPGDF+Y7LXnVeAzuxNXJirWMiW1flF3OSjU7s7Z7XsWAqBIQNbhYWanojjrCc8+tPFQTtMNuorEbPVvX0DH33WcKEPzZqetDI8ZBTPAeUqelY5\\\\\\\/UTBV3SXnxIy\\\\\\\/EN1rkwNTDm9ld0LqLav9M\\\\\\\/4mt2zrr3gPquiDyZz4OoKVIHEfCyJ2gB\\\\\\\/qB\\\\\\\/LWRaqRPWI5NesvrGi5jcuSkYe77n9EQQ5rXWWtdx2fUcrnaANt3c02kpPqvjUN7YrzRqs38zI5+2aVc7EK2zwBGwq0OsyDGG2162ArQJ1n\\\\\\\/A5A1+5nAHiDy8YtevPm3pHoGjkDXfdQhF9d2ld4zrQ6vJg5AXi1ChHBCJeoBMHLSh4n2pfGKxXsVewRnFpMAzRt242viJN7SoPdZNfLLDzR2PsmyoL1u2eJE\\\\\\\/6crwklHUS4u0yNIAIH9SPgl7YTsCRN7Hj7u3sQ6XAcUcgaFATG07b7kxZstF8F7bXzMXJ7Xq0aB5PWhQt4HqS7dro+KIYhaJyMQ4GlhGLeLXx70RZhcN8J50z8Pz1ZQkOUOQAqM\\\\\\\/8qxudSgi77uhPQZ2lprR3TLEG6gciqGNA7+1SY7VX96ee187LjVbm9IUj\\\\\\\/L25L6jLUcctId+9Ci9TnePxbhFuaTVth\\\\\\\/AA\\\\\\\/YuVehuar1zHul6d9QrIHt0zj3AtnHijyMwrB3z+7pM9mx3m7YFaRMR14YNBPLbMaU7iCYkKNLQaNSOpUUFfWNE+2mVIxq8jP5kpLTqOFsdP0DMNIOx759aNUWoEKQMhVqyDd4Hx78e+CkDQcKl6NLKwEPhfwW+gthPgjlCJEO\\\\\\\/n9B4gIqWPflrbuz4obhWwghHgxGFZlzl4laGJw27Vueq+DHOPZf0hNj77b5F9yxjHMC5br\\\\\\\/EJRuf0SzGcJgrclT7z3I\\\\\\\/Pvo+8hZE0p9qinS\\\\\\\/9nTGckRaA5VS1gFX9ONZvsLdz0RJE7umWGY\\\\\\\/AWcOgbSsVt3CxHYgPyVBF6PBvr3r2tsX6n1RVhMDdXlPUhCrGNPAUOTVDw0lmbMnA340\\\\\\\/tJjQ+gxno88h0SK3naBKyTyb\\\\\\\/8SK45Cn8iAkpY+yFnSPSVDsIBJehD0u7pqVtdPZcSUOsne5ktEmpgI10qLaYLhnIB7R5lByQM9g6YLHbxmGz7N1gN\\\\\\\/TCZtwGmInalrS3tXbPw0wIsy8STXeDkyWOL5kCS3jZ\\\\\\\/1RqkqGXAOfshZSLhcVHNjUbAe72yojCsV8utNw88gdZxU4vlKKcQAfOXfjv05gO7xILgF5y+oWMTpVhxGejoF+finh5rLpluT\\\\\\\/NsaunaGOBUMUzY3LDnnZ\\\\\\\/Gj5BHPl9hCowXIv4oLELPfqy7YD8ySJgRiuI2bS\\\\\\\/jYkCvab2ypQSdXbuDA68S2uKk9AQcA0AQcrzCTNSNOiuYil6yMg+b9zxvcyEa6uyNmlq1QSwqt5HKKlaArij42Pl0bqyOTP4qAGHf7+3+WutwHAT3t5K39XJ6Gw+UoIXOqGYSUIQS6Z20F2PL\\\\\\\/9HeJXuA+MGFYAp3hLzZ6iQGdpkOMxnitr1iuQVPY7pzgYNrSYQw3FmWdi20zPTml1ngkXTdWNDqs4IfQ1KoCFZtGvOMRlehQKdVrusc1AuIushYtdXAeED4CCN6mUqZ2XelDl7eybjMscAcVlxsoasdDIC8nOfk\\\\\\\/A21nvAD9BIWYKFWO+wCB\\\\\\\/uLMt6ulTUzWFMdr5+vKUU+VrvRYAfZE+owTaEjObQp947cXVjU6rPYuahNinJLiP4sR5Cem+XxPQnHdgEIBxgR6h9F03wixQ0SM4k0KWW9SmaN+CbFn1HBrtqTR5oEW8m\\\\\\\/M1\\\\\\\/6S9ucg1QU9UwYx25HgNKWpbHeIuuLiFTo+ckraciLsTEwNOyAmcP8JfxTRCiK9JBbstdU+qoNvKGzN6EFCAq4enjFE3mR8JBmg3OUKboFdTf1M3o68QQx0\\\\\\\/+Cm4XP2QgyNcRT1JPqwRzJbGt05gq2RPhYRYKLKp9Llc\\\\\\\/kmEd1Rw4bRFLH3vPhnSwi5J6pvYWVS7mw5YL8DpnkQ0cLCz8a50E0A3P6u7eHEmE5yZQhqjklceZqV6vjo93MYgqXdeQIuWEAnqj1F2iM7psC72YoTN4YF+Nb\\\\\\\/y8Macuim4KiJm5rhXPHJSZQToKN6w2SiSaj1TlThlze+L0EFJbCiEjrVWWbtz155rdsPA6FsxVxw0yA8hnfpY6HwGe8nWwfmlO5H4GlFeVRaIRm5J+kWlSstLpPWDOBfimJSIQDsqqWsiD2\\\\\\\/keoM\\\\\\\/82WHxBbhMkg3u+n7KU1s1MTY7VyrpvBBel4vsMXmdHOYRI55MO+YfMFWJzqYRRoMw2zc5a9FNwSjxltd0wMeMVE4o4ynY9mAWhVg9H37ydyvSG33UuAO+7uSELh8\\\\\\\/sNlSFskOMhTSO5XSbD1xXAyWsrk3M+dMVvCMvqhh922oQNu0EYq\\\\\\\/TcSFvLpo6K2iEKVEXm6FXjGIzh\\\\\\\/5P6cj023LPrmdbYRib8GVck8SSHtb6qbKj559z1GLlwP\\\\\\\/QlaqUx8AHmICfuXk6\\\\\\\/jBfeiNFJYXXCh1tHIJTnKD3BbVIeEcF8vFgHjYTsXpoC94HjVFJ3mYr9P\\\\\\\/GkEW0EmZTAVhr0zA8R+jM1tLDEwUlQ4S7bqeD1Zb44SuTN9SmCoZDD8BABiO4oPajlc\\\\\\\/f0S8vlU84Trigb+j8d2G5j5adEf7RSL7wGytOSSXTTZB297\\\\\\\/JhYa\\\\\\\/u1rmyC4JE\\\\\\\/3xr86f48e84BbjUzNFo5NMxGps9xwadBEjWxfVjymakv8iEOal5r+yl4fq8y5IT8tSyc8TSGU6vb5MWHIw9e8QjzCQbvTL3JkjJ9weEew1E26J5jih6JUGrGIsd5Mxft6ZCR\\\\\\\/sDM3jddA5iFRU6aaSTsmOte\\\\\\\/Pluj\\\\\\\/+9I1biwLXTvRHzhZceeBo1loFOjDyIChtE6HM7e8rmPiyxDLWikz35IXpSWIUx6yxJjFdUIgRTMU5oj3SUIhmT7jc\\\\\\\/jFCDgn8z0JO2yyex\\\\\\\/qz5UAELC9JNN7YT\\\\\\\/DwxUkO9XaZHxm14CHnhtqJ+RGMwNwXJlZlkaBK4bRF\\\\\\\/qcvPbjBL\\\\\\\/foW8lXLiiFm8YYZOvtG6b1JXu9MKLlu0jMGtraoHZPVMj0u+4LjlGqqSicyGYekkKhwQji83xwUuL\\\\\\\/aszty3lP6SA1\\\\\\\/\\\\\\\/66TvemuVESss57S1DUqrFzI6WT4uHVYZshf9qZETmTdeRJ3bPGslZXnYH0Rk8cEkt1sPgpK9u+LkxqwfW4olPXFoQL4XsEu49NB6V9bSdqVtjadfpcZWpQiJpYr2LisgaGxvvr5EhkUnEXq3JwgKbVbJpvzfBpqYNNZcKcjhTsgX+zNFkPIRMsH4xBzEPlgVcJyfkolduNdlBcR9yHQnjToTJiZXhLXALEZqUlNrXv9SrTZY5uKasLGW3JEPbL8syC4a5C+IcR2BoCyjiV5qGEFr8WWiJS93XBobiG0RBxvZhrL7Egj9B\\\\\\\/9YyXkrMJMDNByheKuyXJwQWam66H5km5GmiDJdiQ+EQmT1IE7RKwUnQQnf5duOmQyj2YgFIXNd7iVuGdBQ73UTjb8q31tCZFyyiZPPGGk1M2yozOfuwcH4KqHLZUs+qdMkK0dgbqquN632OV7at4py\\\\\\\/7YVU9DG8CG2VRCeV6\\\\\\\/6hEWrBG2GT1J3iJHNpATFC6B+sm\\\\\\\/7p0EwzXGptnc5jrNq6GWn2POvopE59x+oZyNH1Pmirj+sOrSU+PugeQAHaGEsNtS\\\\\\\/hRfhn81Vliui5cwksygPI3HrIIaJwGV8tDjUYgfu\\\\\\\/X5JtTOHDcISEZM2loNwKaoMGhQHAjvYix+cMcmF+9NFOHavXNbGQ0jw6OsXWqvCdH97kIPvIvA7no1SynZmUJvHbB5kOnkbAZTQOCBf04j3Y7u5ntoC2MWrY4A6KpSzvDWDhWRiVJHGsP43n0dFi1UpIQiCS7KRYBHTB8nASFuBtVcRGVfvzpY+qmrT\\\\\\\/JThiBMJfmbm5zOQcmCUFh5SKT3ZQpzLP40lBialmXsNzP4L\\\\\\\/o5MEI1tfu5kEtiEzplVXu1Ye2\\\\\\\/fS0wxxEkieqmLTYavC4\\\\\\\/6QdHfS+zYgw\\\\\\\/Q0JHEYnOLLpX6ZRK0cQAQ2FNyQq\\\\\\\/DXllyxU0wmnoyZ183gkCFQUIuf4R81yyap1MPneiE7Ut69juDS1BhUXlSPOH4yS7UixYgUmaTRtMceRG7q5xGjxUzPuyyx\\\\\\\/WAhK5+iWhh2+B27q+BDQsr5m9Q7Y1nmtLxe\\\\\\\/t+h9TFskVCETecwmYv9+EW\\\\\\\/ThMlSfZstEGk+HlJhfRqqO1+dvp7dZ4lJ\\\\\\\/vArZEvCV9E2zEVUmBLYvbK9Z2Lq1j3w1KybrJU9saehd5okYjRK6m1tincjX\\\\\\\/b4pCR0TfB6UD7MTyusSYXfYWKhFYurhq+kzjiy5Ff0rXEoaj7g8561SM13TLZyTUytxx79YBbc7+LxRNxXGK4YJ\\\\\\\/+hPnH\\\\\\\/weFcrrb5HrywxSz6KnN6+nMI7hBm0oy+QyfqXJcYqKEgx22g7i0LAKaSv0bwvdzxNVyFFpVFJg0TjjDMbB1ske6tBW0SrqrbdX84AdAHLH2UhsL9FYzG\\\\\\\/WW5nVj4by6ePWRCbkg+nSJ93grrzheDfRZfB81xCVytX4iYcOGKUKHwMXxOn1GdmbW5ZPtrS8Nehf4SEp8HpWGuaCj2NeU4Gyq5oM\\\\\\\/NBhHY07ye1+QWRI6PPsnVwa4muM9u17Cy7NYDfiwHblpWCUFRk9\\\\\\\/2z0rpf6362+AfdtHmpkYE5PTBb2cYv6MszoOxIE4ty8GvtO5E9wFtOSCQYzDMBghnsP9fNhai82XutG+\\\\\\\/wuX1\\\\\\\/BRySP74bnNdA1tSnhN+KICgxhF1pnYnK\\\\\\\/MdvEzu9F8ZTer6NEsGXYtKbE63gMjx\\\\\\\/VOlkrHxD6MIpunIFQSI\\\\\\\/15WuXaweNlC1JnJOjCDQ7v2PIP0iAcMPvITW+t+AgCKusuyRuV5K\\\\\\\/r0vWiUOFAmLdlNHGi9QQbhAKjUj\\\\\\\/DJR0qE0LbSgM+a1yc1aJvWRS6GyDpxtDHvkCVXViVaKfMSh8STOg70hLb8Zgxc7FTY1NSFy6mMebon9E9DfafjyRgJlzGbcd+yVJUPM0SX0DafVtD4Ckd0fD3+hsqfCt47qaXBj8J4qHnUFkZzSeRcLZK4NCxTt8fNFSND4xtoJek+0WLomgzIuKPqF8S7kd1khrRMvWVxxP\\\\\\\/csj02iTtapLh8DxsPPDWgVjc0LwYwv0TuMUjV9V14vF5rripN0Ndzx+HbGmzMLAlJCEFpU0YihJKDIS6ZE2GbVIO3LqZtorTkKAwNdkkPKnaE45O6uuXJjlu01sooFW3SseO+w08SodpOT6dEvyg9eV1fVrop1tcwUaCCUncMVLNFGSSwvtpZZAYzVzv3eHbs5D1sAHBi3ugn49MXAY1cn3tSrfil3UkgUd92TO3ygadjOIejtabtX56fNaiNSBdRNOLuLyNoFhoPkdsoVGReDR6edR9jXdHZtyVIHwxT\\\\\\\/pUKuB2vv+SXqDqQzUZIf811t7oWzx\\\\\\\/bEC3sdqb3T2lji70FMOQRUGuMrYnWs0T3UTDQQ9WWXB0OjKW1SRzx9IP5LnadoCh9+rRcc3u10T32t49J4MJ+QrdSO\\\\\\\/ijC207ps7iq9l7JdY8ovGcnFgNwlmJz3NnGeIcSs0PaXu8Y7vtiiLmdRKBZ7rFBmmA\\\\\\\/w71Ivx514iYxTmAfMLKx8Y\\\\\\\/H+ukcyusU\\\\\\\/Sk0o4eH41TRJV1\\\\\\\/KBaJ7taIsauz6v05N1xGndzoG\\\\\\\/o+lkh4F4bynEhJqe\\\\\\\/QQ2UCqF20fhrvYZTsn2Iwru6F0Y9uDLZIL6T9Ocj9e4PWG2TZef0PAsFXrI\\\\\\\/Vq64Cn5KqwIwUkXvnyw\\\\\\\/PFGYQpZ5\\\\\\\/KeWFRfUxAkrU\\\\\\\/fdt7tvYn7CYOzXvOwjTFZeqEjr3sGtFRoB\\\\\\\/Y1PvQzIVWznZ1Spp+jrR+AKvF1soYbLktSpp1IV6BfuLUXbQ0jo1rF5hA720kYdzHaS4E\\\\\\\/T3\\\\\\\/UxXyz8z+oqmRzPQXsdlTjTmiF3UiQ8XV+QTmKePlDkuvM0x\\\\\\\/sTy+M860DE2+Kg1k0NCTPo6fL0iH8S4jx6nXdO7XzUUrrsXLugSzsmG5VGNEtq7Yl0+38WYc4MhG7kwwyw722lDeeSJF2KBolFZI8eQUK4fDWXB0ouRIh5eLMsHKkocdrsiItFVKpDKG1j43UF40JiFJ3\\\\\\\/yVlc98llo3l7uGnLunvyzTzgn5GoxMvowBaxMVfUcAaKjNqn8+Cv9Q0Ht\\\\\\\/Yd1xc7NakPi3EIQYRX+6XDtuHSUpiNcMwwqQVy5l83j1SuK9iKbfZNw7eCAC7h9DrGI2lpo8gQFvi1Mwy5bo9RvxHk3BHtjs1bFXgvkM7Q6RexseZBJZu4ndFFtNJ8koWg5E1lJ4bVRPn8ZaYNS4O8WYRZEekmv23sWzcv6jNNWntjlzReHt1h0G3OILizPQSls209QG+dyHvNFiMUspqc13WVxyxDJZdSdsQhxVKNpDkOKHQNNcmmuZ2rjSLnNfP+1lJ1xdmCsIj89WCAa4lL5ijLz05Lpz\\\\\\\/QSTezBO5eqPpnJS3cXjdhe00WIG7K4Xqrv7pio5nR1aX1UPc4EHL\\\\\\\/ELiWXxE6\\\\\\\/HJYF7p\\\\\\\/kSJcXkxmC4cwNsq74ow3bngIERlYkE1AdGKjHSy0ExvxcZ7qW+u6DH6PwL2bWBgJNLxNvBoJueE9p5O+BN2sJ3c6pOOqE\\\\\\\/d89nau\\\\\\\/JiYNm3fxc5bXFRfqV\\\\\\\/vYyqwXqicj6J0UVSqGUE6UHoMb7qy64aPGTZFZhrasMyuL6fZPyHb3\\\\\\\/WrzX8bCnrW+N9jqkD2f3WXP\\\\\\\/sBFKe0RQPfwbFSwmvysFXnIuJ2egnyNDRJMMWTI83H9iKiDlGzdonszWididlXRnflKZJs\\\\\\\/nUx3ZH+oknurXJXnyZzNCRygiTa0CB+2I6dCx5vfF9KnAURXv8w5+4oohc5JusyF3pMmRknzyRSHONgJ\\\\\\\/OqtcLXhZfpVVm6c6w6OhrOi9d8cPzXjdHIo9+t\\\\\\\/FhUK\\\\\\\/rgxLykrfBLP8VnGghrN2kcT8BkO0efDom3+CSFTZDXR5fhtK\\\\\\\/8oFqzgFFl7u7+zcpc+bZQLhUMrU5+KW+b3PCr\\\\\\\/yO8C1+TcK6mc2CoabsoABi\\\\\\\/OZESAWKWiwVJO8Wzt\\\\\\\/38L9HhYJLpJJc3hpqmag91Oj5U42V68f0XSZDJRhTxDw0Sf03I8xTNXwzV8ukuZYSbsJKaLe++Ui22cJgsIVdTXCQDJdbWkzqd2Xt8sknDr2aiNoeXPbGNcO7tow6zySSAIiijWU8hTrrWGSVk407khg9R+VJrA9LeM0vwEbar\\\\\\\/fmh0xHCcBaU6LLjJMUdwbl4Tg15\\\\\\\/3iJEU\\\\\\\/R695HJbE2mgpxAeqAFtf\\\\\\\/nv+7dk8ZUCJ6IHwXuG+7hXKCweHRzcOwvc+j28ZBkNvmPpcWX76py+sg0j7falH+MKIqtjTcHc2Z4tkjEe8ROTkb874pvZ4JHh3hoIR9mN2cPYidgT0ecNhgDO9RVxK6+FBF6ANU+ATk97WZEDLcdtOPY1eCppXubtSR3EC07OOyvCDWi7GSJh1TVxucti2w0c7JjQJx7wIeiS0Lye280QTIad+7RiRRoio2LDdfUR3JJpP5\\\\\\\/zgtiMcpBbzXk6A68AfdtdsL0zztqLr6o9KESR9uPAXRnezGdQ07uoXNo5GFOxj0x2f9dHp5dYblJyq7sdP5w5jnLP7TIyzoWzH\\\\\\\/Oh6DttqtXPZZnApqgcjkiUy5XHEFL2\\\\\\\/Gj6WyNvDD9iR2KE6WIY1m\\\\\\\/8YuuYw8kIdKk68In+rhZF1oTBP8s+wQdGDEQRAOreUTTZy7naeupn77gYXAwqOIjjhipbhdJtyLANheGk\\\\\\\/o7lpwl\\\\\\\/9Emy21oYRpIavWZIhKMaSoDn\\\\\\\/tknvoeNrC\\\\\\\/cr2GHMJnk2kWv7ff1UT8e7MvoVtI9v1GIghKY6yOfCmV2+1VolVqpckztu20Adm4wq+xHrQa1+RVlopZZPDPD3wTPY+6Fk\\\\\\\/gSR4yL16HQhI1u0OAdU2BT319x5kBjZz9UrFG2aDLl+hHZLXolCvOx+FD0HaYTTq2g2i9ucS9J18+xtKtat1M+DCzgOoQUorct77UYhOfkgq\\\\\\\/Tu4Ivhp+xPChPikHsCFCjhE0A9SAb4MqX9jSh+RP\\\\\\\/Mr1llYIQyYmQs0eEOJ76kz0QW\\\\\\\/rBainnsX5uu\\\\\\\/aahdWHlRf1uOS67CwlNUTSWWakyWCYZfa5HrC2oQgsI7VAkDqU\\\\\\\/EA\\\\\\\/qOXdlPh9\\\\\\\/c8v4+IdX6OAEHntNSjnAkyCg\\\\\\\/f6n\\\\\\\/eLcFio+d5s9FNDEB3VdGwKcmpIidUV230hIikPxrCn1bGLE+hnxym7Aia5qgZY4AJYBV6oas+lSVaydwUwRU3it8RRzqXWdW2zxdfbiaJrbcTM6LKUZDCTM69wrmaeYwe1dS2ahA9v77cBb4fyL2w9GSzISB\\\\\\\/VXTSZmP4IO7pfhtftCdoAEQ4oOvUS2EDVz6M412XDObOZRcQQqWph6pnexeYZtD6rYBF5s8FW66Bjooi\\\\\\\/LlPy\\\\\\\/nWAUkVM86VivyDyvJW1qci6TNio3yGTYugKSMo\\\\\\\/LV1zcaOqxt0TgAM7s92iUlADO5AhTUlQRy0Lr731E67w2SVdFo4XQMbty6HXLPgcLoPI4CPdscVPWvMUpcQxJwHU2gPcJcPjNZSjF5Rxv0F9lc6G0juVjeoWZl6UCgHCajx\\\\\\\/ls4qCzUKvtuaWMSv9s4LUa+Pu8P02DYhPiJfvA4rILXRRn4IkCR8oqIyoSNaT6YtAOrJBKkaEPakRlQU0TTwkmQUE9OHbBDWSOYWPRs84vn1\\\\\\\/eRsFY\\\\\\\/Zlr5iOCmbppF2sCcn7chKELmIA0sCyH+i5i4+\\\\\\\/1LQptEpfCAD\\\\\\\/pwSzC7f975RSdOMJcBuDMWo2Z8QCO118Bqe51USfQvizT3D3abzbR9qZjNR5U5qhgyDRxixKn7E8HIrTVlzYDm6psNRIkW8w5dI1cefLqCG5C81JmllTtut+Q8iYKpWH2nqaWaq7zU9zy2iodPeXrYAw\\\\\\\/pjw9Udhs8DW99CjlrEzS3ewngI8fJXSNeNucKNnfYjXcp3iIKOqMbTnHfmOp\\\\\\\/ZEyfIyyW0dcv9T1HDdCl2aOFRGho36oONrQa4P\\\\\\\/kWVSqgGmp8peybPDCJohdra\\\\\\\/FzFJVpK51Nva4LdWelywB+UdMQyUORPhezR2ghYkINOa1Ph+51D3HdJi+P0RIMcb+is9LeUgPbfNdTeZUtWAcs9WAisRNs7AxurU\\\\\\\/8vRErj8lNb8Ape8pygEVWVI6qV8hkyMhxTeKN19bGwcLmUSBbWDJ9kFLT9rd0YJgw853IZbfzQYqM9NklO1UICE+EVNPNAYll+jD3W\\\\\\\/PqnG+eCk5wp2U4QFZdm6Y9jVtm4YkSuo1oIzfv0qsUdHw4K9P2wCWVUKHOw7QhyI\\\\\\\/\\\\\\\/Cx\\\\\\\/DNmdbdf5PoAEUz54UsuVeTdjKHGrmLbqdOoGHM4Ub1TDbsPNSTxGX2+hgwlbeFg5zdLLCSj8WUDcy\\\\\\\/17iEM+XZP88J98VD0Qe8PVus3nNvLi4hDpjeFwkjtXIbmhWeDYdVHkkvnRR9ysLAdtLf0lYS1Wd1m8mZ6wVeYXmleO9zhxawjn7x1sQ27Q9OrQdUwyhAqINrPSXMmbEYC\\\\\\\/NyCccjTp2YXTtkemeNO2x4LFUOSbzWEVUT9Fpgpim2E\\\\\\\/18TwZysfrFZpMmQZEIqJK8QjosT77yHNqcC2BEpjmKWMhLCFFEWERitu3HCppSb4LukDKz0wBO3W0OKcDrA7YLAEGxyBSiIG9Nwd0Er\\\\\\\/fKDtJRrOHGrxgxjCecJ2F8l3+XO0HUZ9h9Tic51giCWIQTa1XgY7T4nWewTX8q8qLhenozdzWf+POtzarISqY6erGUSvs5v4roP5INyvrEXzwOUQiKQUQIdiLHNRk1vXwlDuxtbwiec4x03fVOfg7XdCf6u1NGhIC+OK2PCibW\\\\\\\/eqHt9TIL49H\\\\\\\/ebknK5FI7SXgb3r8dBiqObxWdvgtK7jEnIflB7m47l0Xem99na5XIP8PGqmPSSstefyFzGafar1FttKTCBEC7CEP7Mjv6vMfBzVx8n2FlRuR42LTfmY0bKRfCQke8vPdiHqh4OQ7qprwoxZarD2zd1+kBGY\\\\\\\/CMDxJ8ExtAm0o2tMRKQCzxEhXVf4jQTFM9jl1hm2XWpagvtXZ8UO+tdPR8KfdIvF22V4yVSkgCQ70RcOmGmNzeWkQfGy7NmqxTGXWH+BOMqDjTMPzSID8ELYHiXpjBXQZVuu3I6J+YQycw1SvK40vw9X9fYy8dAzhgcuFURoAjGN5Ovrly09gU0yaTWICatItYWz0Yn1LbHJsgY+BSef+sF7Xqea9MKfHvhSjK8hZ8fTEhcyrSCNgYp+jNzNtnTUohQmYBWNzUmyekATJRMr4nXvOhR60uxuHbQ\\\\\\\/SwCmb53GT2EhGynvjc6b71Qp8\\\\\\\/jroJ4XAUGHivqEgvpBWjO4qnQOgjKxJyQz3nHQcT8WeQ0Qp0CTQooYxcWpXTcEEOa7jK2CeCOD8l0Kk\\\\\\\/p+kLpqld2m7rrt47eb0kGSe3dh8Xtwwb83cuEPORpv0aREnPI4QA6pDbPVk\\\\\\\/CqtvR241ET1ZromLgrDTTpuoyx+gTW1CKgwWfnyJafuu3nOSGe7uvL0maODGtZf0Hi9P1wR1+R13D1VdUZU7b9TpIATmqqAZEZ3ej24Dg2eIeqprnN0UGpCtNOqMlwHI\\\\\\\/zBvSbhGwk1EgSOKTWWvNISupZXwYLcgo1qtDYnei\\\\\\\/xaEv3LgM0Abe90i\\\\\\\/quQNVQrVJoRqkGJamFt43zUbFntpI0gLmLwn2wGAuIIx8GvhXrDCQXRJJvmsFRbKgVIjqnc9ggbXdEoXIqBaC4NMUZkdlb\\\\\\\/F25nSEkmZUVhUWn7WVl1IH0WPM\\\\\\\/CDfijs5Qmsf8OALixf1gwtbe305fI4rL2h0+ryEfMk4QXpJ7d7P8xzv6k\\\\\\\/ZBIEHajw6weDo+Rnc7phcZLi3vRYfnLnX0fPg5Er8qkENpSxDjaIhc9cL0TXx067hPX5y7TiE2tg1MW0HPv8ARSl2AodfhWva2qdePYbe2Q8NyrDDSncZWYjH5zSLucXo+TOImSj8qUgvFdk8czJCthMP83I9dzejWloRFgtAmt5aljx9q\\\\\\\/kJDPWwGEezyrHEXafFKdQr2ABrnpb6ZBYFLOQEQIaT8ykZXFmJUSUq1cOvY80OIHF6wiG4dkNrqkCAQkNpizINSC4OvgXumR+MQ5zEZjPMaqTbJBTRMAu5d4MTr2Y2VZHSpC4OH4IhqjngCEfvOUTGknYXAbpY8LryR7a1Lzqd4RsmrqyvA7hcS281JbvAEVtFDwcdnxC4x0oQyVp6Cbw3I3ZsSHuRvMY756R+iHfDnkiwckuHwwAa4VdTrqR0cVRahZc0rWskaUCURK0ytZXXyBdYP7nwodS8xUD2Plmr1R31qD3EJLHHSUOH0blJpvppdvKEMZ1BjT7wlVO39mPxpa65OTlUSeGeUp4h4mnsLSUV6xifrJhouPuZgIpYw3bmoebniJeUHdjP092Naz8tsQmg11ohyfXCX2Sab0qwME7Se9ds3zN\\\\\\\/e+SLgiLQPJp6uzrdbQvjDBh99zDeamCWheYXkcDCC4nYVzg1UK0U60hQW7KIcZwMJwQgfTFN8glDy8AEra0GANX\\\\\\\/y+4T572OSGtHcqmACn2VXxctVPlY28jB7iLZYYfF5d3DxY7Cq\\\\\\\/bq4jqbcS4F0Mq0c6Xh42O0ZhiznIdWHx7czX6iRP1vM9Hjck9t9zbPUVWP4uXDx2SiRJlTY4EhD6L9acdbvXPfariOL91jEwBe+PSEQYJ4QIxSF+B\\\\\\\/6bEjNDM8guYD4LJIf7KLVNABxccf3lX\\\\\\\/i6OJt23TzAwE322fvYDFBPL81J9O2vH7\\\\\\\/RdAiXcssV200wl8ZPfgvOW8i53I\\\\\\\/h7HN5juqg4z\\\\\\\/KOrZ+rjhimuloB5c\\\\\\\/D4zY6xWZyQuuGJ4mQBhj6ratLCRMqSxbRPJ6PrTcuJ7ZCbshXXE3xm0jrQaRvTgD7sGgJbu8fEvkeosKA\\\\\\\/cNDnT2Dqa84DCliHtdEAmP7xuG5PCt8W94b+ZDwMwwXhule60TqU0BgOgLXlXk38dnccn\\\\\\\/BpuFb9PwvhyjxnVvB\\\\\\\/lOrS2goLUZPE6rrlLw9lF4Lnw5uiy2xpkO2O6rp\\\\\\\/NbOq3W6Y5gMAFKs\\\\\\\/+FFSaR4Omv47uNuiM\\\\\\\/EEh83uZ\\\\\\\/9qfGJmIAd1ZlBNkVoauoW1r1a\\\\\\\/GWykE1u7yTan0VulbpvnIEqHbhluBcS9W\\\\\\\/\\\\\\\/bU+j16mbMl50KOc6D7eraaEbEdf8EwuK1FRfaeaXCi+RgzubDZMyGIVywcYuPzCFJoUo8LxOfO28lEpP9WaFMOlE7MC748duGPmuq\\\\\\\/R4gCVxK0lOntGS4qWdojRS6ejGe0ovnGAbN4Pr9vFVhV4QdfmYuYvv7wPjMjIVMwXj+1AFWE9zUNOYq9ngMoU\\\\\\\/3mtSAHEfcMuKvnwHoid2Wk0qMGe1Bm18BvdcLkKCf+DJxX1tMmmEKPwpuocYfMq9j1Fr+Lyd6sva1KscT0Q3WJOoSPPM+iSEK1oAq0SR9o2+4VXWqjptFQukYDIAeQCK8HiLrIfkRVA8GSEv\\\\\\\/LRHSOrYs6E5GL\\\\\\\/wNc4s4K\\\\\\\/QQE9serVe5tqKPvvAkSX+jKq54L2hMKUpciA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022gO6YhAVaveSjFfaGpOfUGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225319a0c3180372629807ea551994230d6045295e52c05db49d606c0685757365\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022u7\\\\\\\/SULNWiXzenjJVXeI0B0JmFngZ6Zjt++HHwFCldPa1QnbbymPNNRSbUlGdae35gzACFC8mJnPifHUQltq9rT72HEI1zvqltf5VLn3y67pPom2PDChqidYBwhLscqgyXuFqaf3mod8NsEhw+3UaFtg53sQ8wRxkB8c7r3M4rmkO9Cha5eedeZG+1s0tdhgNeB0q4ME72EbggjgooxX6+NC8p3vt5CgiSItfln2AC+zot1LPSM5tvIO8xNoYJMaliM1fGHlHkpLEcmsDWD36crBb7AJ0jLYqXrd+pkFXnqaHUYWYEdpQqnYP86GJDwqREdo8VyI3VkTC3DPIiYAhMP5t3MhmTkp\\\\\\\/OI8WQDrdaCnLgNUIp8Vvde+YKwhwobv6UpEW\\\\\\\/OUhw0AxAfi+TSnMpwRLldDQNCWlURtra\\\\\\\/7b+jvQo3\\\\\\\/UPNXFxfDYzbGH\\\\\\\/e5tmGEKINml22pzBouGe423lTWDaT44bWzlPQ3ceBFRzrsxrfEWQ1eq3XEnwmj7g22TrkjpjBHuJ5HrXQ0T2adgtGS3S0Bs22PQ3uYtMpIBltABPO\\\\\\\/mpFjpTYV3HXcXnJrYY4z5lBGPKtfZ1dv1F7kyeDPbOGS1aTVR2oGouads3NrS5teo5nsEFPPxPonsbUEh48ov\\\\\\\/fwoy\\\\\\\/ZmFUTu1mXYtF0v+lpoJzwvQmFYPp1eFYMFiD5ZlOuCm6J+aw+YBzaTSjuu5UskzA5jPvP0qjvrx8kR14zC8HX6\\\\\\\/7CigjiX0lqItzwaEE\\\\\\\/4IQsqqRRqwDMSPgpg2wrkejBpEl\\\\\\\/LhK8m9+Xa2QZEFWV58hEsYrnLT0A5gl\\\\\\\/yQIB2rwnHGb9BZe8nyFacfovygD+otDAeJMaFSUzH2LQqKIjwH6EoJD++N0sEHrzmlb1bK0aM+iTuDQvvJIjw9o4\\\\\\\/9xI4Aq97sZFbXROdmUxuQkunm6p\\\\\\\/zn+3Z94OquoS9bZzBN4W7rvnsxAtzyaYxibj+SU+z\\\\\\\/OJjeKIHdxqQAAmo0UmALt9MqjcHmmVehvlsIj5EfytZ3l68VsnhFjkxApHbfcPvOJqPNr9WPXoj6gVYYoiTKWuPMCaqDyFgCEQQEzRi3iA\\\\\\\/xBcisl9Coynxd50BZiix9Uaghqi4XOkVceakeDFomMMQLMXjSmPwgZFXT9ibw+MnkdEB+6PLeaH1dUXlcSv0QL\\\\\\\/7TzKFv+wDiDo\\\\\\\/F9UwAfeXUN1GiaV20csW8h6wo2PE1eCJb7Qm7ttQsvlRO8mo+bVkWg6XVkunC05iE0EzFoNUwr8jlNi6Za3dYx3F8654ReUT+Ji+yl7wudyhWE\\\\\\\/OpKTzPHaF2YsKPhEdSRmx7gXJ5O+3m+56am3j6hw5lcBwvCDbnQvEGJXDUs\\\\\\\/vvz\\\\\\\/rI1KnrPtTWvMJyTqjy5pwthxKZ9zHXU9sO6hjRw2TNxvgCsOyIxxmKh6IMiVTQUIcNwFypntOnEtbRPhkPRArWZVxEDCyd2\\\\\\\/zjbNUMVEOknX4gUBVT7hnjl2tnus5Pwq9AQQyVsUIJvJozBbaxPTLt4VJrVhhlnvqWeKj8auCh2+5zMItvM7pBH+MvFwm70y1CJXMLS51TceA1dZuQWzJSVGdJXoyB32qykzcuEq31JnKQoo7T4rpR9QAk2bFntfyLKfgEwyvT5uK0AffhyF9lRGgqM9eNWFZ\\\\\\\/H3zxfx8+nikRaQUzbgJwmS6dBGYBOYnmNYo1lLO\\\\\\\/RpVT3WN88ZMb9WUnhETobaaTFARwh\\\\\\\/WErnu3IXf6Ztw18RwU7P\\\\\\\/ein2e7pOlYBoZ0G6TXAkZZ\\\\\\\/I+Gour80f\\\\\\\/J7mySX2W90U0GyxmE2+s1Kk3d6ccJywP\\\\\\\/xPkiiKytfPuNqxw\\\\\\\/f9I3poHJ7cVboqjSsCHxOL2BskHhYNgFa82xFD1EIsBRdVFtRgjpdR8e3XaGSSd1skj2TtfKCRSKWEMUhakwVKUp+MZv\\\\\\\/LHxi1LAZVnoAyW2ErWJEDms+kvwbDDZqhG+XrZv85QSbLF7InGPMBFNTPKI0dW4mk8R3MbBiC09l\\\\\\\/zp99DVcHDJjKJ9poA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DbqUIoTmg4WeGtHADGLvdg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d5dbfc700afa7563592ba112afdcc50795e7a910e514be46ae75977451e650fa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460507 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-meeting-report hw-article-category-meeting-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3765" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6gsRNI2QI3SS4Rs6_dTB-g-tOzqYLYN7SvtGK-5KuEQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3765" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bM7IB4f3W5k-P4vxZ_BYSL7EAPioSFwImzritj3KDV8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3757" title="CRISPR/Cas9-Mediated Genome Editing and Mutagenesis of EcChi4 in Exopalaemon carinicauda" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3787" title="Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460507" id="node460507" data-pisa="ggg;6/12/3765" data-pisa-master="ggg;6/12/3765" data-apath="/ggg/6/12/3765.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Meeting Report: The Allied Genetics Conference 2016</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">Organizers of The Allied Genetics Conference 2016</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3765-3786;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036848 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460507" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genetics Society of America, Bethesda, Maryland, 20814</div></span></div></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/12/3765" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460507" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460507/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3765.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460507" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460507/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3765.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460507" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460507/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3765.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460507" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460507/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/12/3765.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460507" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460507/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><ul class="kwd-group"><li class="kwd"><a href="/keyword/caenorhabditis" class="hw-term hw-article-keyword hw-article-keyword-caenorhabditis" rel="nofollow">Caenorhabditis</a></li><li class="kwd"><a href="/keyword/ciliates" class="hw-term hw-article-keyword hw-article-keyword-ciliates" rel="nofollow">ciliates</a></li><li class="kwd"><a href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow">Drosophila</a></li><li class="kwd"><a href="/keyword/mouse" class="hw-term hw-article-keyword hw-article-keyword-mouse" rel="nofollow">mouse</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/zebrafish" class="hw-term hw-article-keyword hw-article-keyword-zebrafish" rel="nofollow">zebrafish</a></li><li class="kwd"><a href="/keyword/population-genetics" class="hw-term hw-article-keyword hw-article-keyword-population-genetics" rel="nofollow">population genetics</a></li><li class="kwd"><a href="/keyword/evolutionary-genetics" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-genetics" rel="nofollow">evolutionary genetics</a></li><li class="kwd"><a href="/keyword/quantitative-genetics" class="hw-term hw-article-keyword hw-article-keyword-quantitative-genetics" rel="nofollow">quantitative genetics</a></li></ul><p id="p-2">The mission of the Genetics Society of America (GSA) is to deepen our understanding of the living world by advancing the field of genetics. A cornerstone of this mission is promoting interaction among geneticists. To foster such connections and collaborations across the field, the GSA brought together seven research communities for The Allied Genetics Conference (TAGC), July 13–17, 2016 in Orlando, Florida (<a href="http://genetics2016.org">http://genetics2016.org</a>). The individual meetings hosted at TAGC focused on the genetics of <em>Caenorhabditis elegans</em>, ciliates, <em>Drosophila</em>, mouse, yeast, and zebrafish, as well as population, evolutionary, and quantitative genetics. In addition to community-specific programming, the groups came together each day for an outstanding series of keynote presentations held in joint plenary sessions. Members of all communities also took part in shared education and professional development events.</p><p id="p-3">This report includes highlights contributed by the organizing committees of each community meeting, an update on model organism database news from TAGC, summaries of talks by Jef Boeke, Francis Collins, Jennifer Doudna, Anna-Katerina Hadjantonakis, David Kingsley, Laura Landweber/Richard Miller, Michael Miller, Molly Przeworski, Pamela Ronald, Amita Sehgal, and Leonard Zon, followed by a list of Honors and Awards conferred at the meeting.</p><div class="section other1" id="sec-1"><h2 class="">C. elegans Development, Cell Biology, and Gene Expression Meeting</h2><p id="p-4">The <em>C. elegans</em> Development, Cell Biology, and Gene Expression Meeting at TAGC featured a number of presentations that represented significant scientific and technical advances. Like its fellow model systems, <em>C. elegans</em> has experienced a revolution in genetics due to the development of powerful genome engineering techniques. A Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) workshop organized by Mike Boxem (Utrecht University) featured presentations by Alex Paix (Johns Hopkins University), who reported on the use of a highly efficient and versatile recombineering approach for making genome modifications, and a talk by Daniel Dickinson (The University of North Carolina) who described improvements to his streamlined method of using a self-excising drug selection cassette for deriving a transcriptional reporter, a knockout allele, and a translational fusion of any gene, all from a single initial CRISPR-mediated insertion. Meanwhile, Matthew Schwartz (HHMI/University of Utah) addressed one of the more time-consuming aspects of genome engineering: construction of repair templates. Using a Golden Gate Cloning strategy, he developed a system for rapidly building complex repair plasmids from multiple interchangeable subunits. One particularly important application of this approach was the use of the FLP recombinase system for tissue-specific expression of transgenes. Finally, Adam Norris (Harvard University) reported on a dual screenable/selectable marker strategy for the highly efficient production of gene knockouts up to an amazing 11.5 kb.</p><p id="p-5">In addition to refinements in genome engineering technology, the meeting ushered in several other technical achievements of note. Worms have lacked several powerful inducible gene expression technologies available in other model systems. Jonathan Liu (California Institute of Technology) developed a Gal4/UAS system derived from the yeast <em>Saccharomyces kudriavzevii</em>, and was able to demonstrate inducible tissue-specific expression of a transgene. Similarly, Peter Askjaer (Pablo de Olavide University) optimized FLP recombinase for use in <em>C. elegans</em>, and was able to attain spatiotemporal control of gene expression. Using either of these systems, the ability to control gene expression in a cell type- and developmental stage-dependent manner will undoubtedly open up new avenues of inquiry. Daniel Dickinson (The University of North Carolina) presented a single-molecule approach for quantifying protein–protein interactions in individual, staged <em>C. elegans</em> embryos, which was used to identify PAR protein complexes that are dynamically regulated during cell polarization.</p><p id="p-6">Among other highlights of the meeting were the three keynote addresses. Michael Miller (University of Alabama at Birmingham) described a role for chemoreceptors of gustatory sensory cilia in modulating sperm behavior. In response to certain pathogenic bacteria, these chemoreceptors elevate oxidative metabolism gene expression during spermatogenesis, resulting in the production of sperm with an enhanced ability to locate a mature oocyte (for more details see talk summary below). Oliver Hobert (Columbia University) presented a tour-de-force effort to provide a comprehensive neurotransmitter map of the <em>C. elegans</em> nervous system. His group has assigned neurotransmitter identity to an astounding 90% of the nervous system, and for almost three quarters of the neurons, they have determined how neurotransmitter identity is genetically specified. Barbara Conradt (Ludwig Maximilians University of Munich) described a multi-layered model of a cell death decision. The mother of the dying cell, in coordination with the engulfment pathways, activates the cell death machinery and predetermines which cell will undergo apoptosis after division. A microRNA-based mechanism prevents the mother cell from being killed by the activated cell death machinery.</p><p id="p-7">As the meeting showed, <em>C. elegans</em> continues to be a powerful genetic model system, and with an impressive set of new genetic tools, it is likely to remain so for the foreseeable future. Our community will meet again next year at the 21st International <em>C. elegans</em> Conference, June 21–25, 2017 at the University of California, Los Angeles.</p></div><div class="section other2" id="sec-2"><h2 class="">Ciliate Molecular Biology CONFERENCE</h2><p id="p-8">The Ciliate Molecular Biology (CMB) community has held biannual meetings since the early 1980s. The 2016 gathering, the first organized through GSA, met expectations for disseminating research findings and provided ample opportunity to discuss community-led initiatives and identify infrastructure needs. Importantly, the TAGC venue fostered networking with colleagues working in other model systems. The panel discussion “Speaking Up for Genetics and Model Organism Research” was particularly useful for articulating challenges that each research community faces, with common themes emerging. The “Why Ciliates” video provides a compelling approach to advocating for research to the general public, legislators, and administrators. The video can be viewed at <a href="https://vimeo.com/174609655">https://vimeo.com/174609655</a>.</p><p id="p-9">The meeting organizers invited three researchers from outside the CMB community to give plenary talks: Susan Dutcher (Washington University at St. Louis; cilia biogenesis), David Bilder (University of California, Berkeley; cell morphogenesis) and Ellen Pritham (University of Utah; mobile DNA). Susan Dutcher described her work in the model system <em>Chlamydomonas reinhardtii</em> on cytoplasmic assembly of dynein arms and formation of the transition zone at the base of the cilium. In a complementary talk, Anne-Marie Tassin (CNRS, Gif sur Yvette) described how Cep290 cooperates with two transition zone modules, MKS2 and NPHP, to generate the <em>Paramecium tetraurelia</em> ciliary gate. Chad Pearson (University of Colorado Health Science Center) described how Poc1p and Fob1p work together to stabilize radially symmetric triplet microtubules in <em>Tetrahymena thermophila</em>. Yu-Yang Jiang (University of Georgia) described a forward genetic approach to isolating suppressors of cilia length misregulation. This effort resulted in the cloning of the <em>Tetrahymena</em> ortholog of the <em>Chlamydomonas</em> long flagella four gene.</p><p id="p-10">Ciliated protozoa are a rich resource for studying genome rearrangements, as tens of thousands of breakage and rejoining reactions occur to transform a transcriptionally silent “germline” micronucleus into a transcriptionally active “somatic” macronucleus. Ellen Pritham discussed how different classes of mobile genetic elements from nonciliate eukaryotic species were found to carry host cellular genes. In these species, mobile DNA proliferation has created large gene families. Like bacterial contemporaries, eukaryotic mobile DNA serves as a vector for horizontal gene movement between species. Mireille Betermier (CNRS, University of Paris-Sud) provided mechanistic insights into a “domesticated” PiggyBac transposase in <em>Paramecium</em> that works in concert with the double-strand break repair complex KU70/80 to break and rejoin DNA during programmed DNA elimination. Aditi Singh (University of Bern) described the contribution of the chromatin remodeler SDGP (ISWI gene family) to the excision of short internally eliminated sequences (IESs) and larger TEs in the developing <em>Paramecium</em> macronucleus. Eric Meyer (Institut de Biologie de l’ENS, IBENS) discussed the role of scnRNAs in the removal of transposon-derived <em>Paramecium</em> sequences, and Andrea Frapporti (Institut Jaques Monod) provided evidence for transcriptional repression of TEs by the chromatin remodeler enhancer of zeste-like protein 1 (Ezl1). Doug Chalker (Washington University, St. Louis) talked about the role of the G-quadraplex binding protein Lia3, and the related Ltl1, in the removal of different classes of IESs in <em>Tetrahymena</em>.</p><p id="p-11">Whole genome sequencing (WGS) is becoming the preferred method to map mutations generated in forward genetic studies. Three WGS analyses described at the meeting identified genes involved in cell division (CDA1; Y. Jiang, University of Georgia), protein trafficking (VPS8; D. Sparvoli, University of Chicago), and cilia biogenesis (LF4; Y. Jiang, University of Georgia). WGS was also used to examine the evolutionary history (expansion and contraction) of <em>Paramecium</em> IESs (D. Sellis, Université Lyon 1), and study potential driving forces for the maintenance and loss of duplicated genes. RNA-seq also played a prominent role in many of the meeting talks. This included transcriptome analysis of the unconventional <em>T. thermophila</em> cell cycle, in which the micronucleus and macronucleus replicate their DNA and partition their chromosomes at different cell cycle stages (L. Zhang, Texas A&amp;M University). Other RNA-seq projects studied cold adaptation in the Antarctic ciliate <em>Euplote focardii</em> (C. Miceli, University of Camerino), sexual/asexual dimorphism in <em>Tetrahymena</em> (W. Miao, Institute of Hydrobiology, Chinese Academy of Sciences), and transcription in <em>P. aurelia</em> complex species that have diverged following an ancestral whole genome duplication (J. Gout, Indiana University). <em>Tetrahymena</em> Genome Database curator Naomi Stover (Bradley University) discussed a recent upgrade to community-based annotation by addition of Web Apollo. Data analysis can now be performed using the software TetraMine. This InterMine will enhance comparative genomics and transcriptomics analyses across species by providing a universal platform for data processing and analysis. On the more translational front, Janna Bednenko (Tetragenetics Inc.) reported on major improvements in protein overproduction in <em>Tetrahymena</em>, including the expression and purification of mammalian ion channels. Finally, while conventional reverse genetic approaches can be used to generate targeted gene disruptions and replacements by homologous recombination in <em>Tetrahymena</em>, Cas9/CRISPR technology could speed up the process, as well as provide opportunities for reverse genetics in less amenable ciliates. Rachel Howard-Till (University of Vienna) described the use of Cas9/CRISPR to study the role of a condensin involved in macronuclear development and DNA deletion.</p><p id="p-12">The next CMB Conference will be held in the summer of 2018 (date and US site to be determined). We are pleased that the meeting will be sponsored by the GSA and look forward to strengthening our affiliation. The 2018 meeting organizers are Chad Pearson (University of Colorado Health Science Center), Naomi Stover (Bradley University), and Martin Simon (Saarland University, Germany).</p></div><div class="section other3" id="sec-3"><h2 class="">57th Annual Drosophila Research Conference</h2><p id="p-13">This year’s meeting was chaired by David Bilder, Nancy Bonini, Ross Cagan, and Susan Celniker. The opening session featured a tribute to the <em>Drosophila</em> colleagues who passed away this year, with a special tribute by Thom Kaufman to William Gelbart, an influential contributor to the fly research community. The tribute was followed by presentation of the <em>Drosophila</em> Image Award, the Larry Sandler Award, and the “Discovery of the Homeobox” panel discussion. The Larry Sandler Award Memorial Lecture was given by Alejandra Figueroa-Clarevega (University of California, Berkeley), who spoke on tumor–host interactions, identifying a tumor-secreted factor that interrupts insulin signaling to induce host tissue wasting. The <em>Hox</em> panel consisted of three of the key researchers involved in discovering the homeobox in 1983: Matthew Scott (Carnegie Institution for Science), Michael Levine (Princeton University), and William McGinnis (University of California, San Diego). Cassandra Extavour served as an exciting moderator of the panel, having been trained by Antonio Garcia-Bellido, a postdoc of the father of developmental genetics, Edward Lewis. They reflected on the historical context of the independent discovery of the homeobox by two different teams, and the profound importance of the discovery in understanding animal development. New facets in this understanding were revealed in a plenary presentation given later in the week by Ingrid Lohmann (Ruprecht-Karls University). Lohmann spoke on the mechanisms that allow HOX factors to acquire exquisite spatial and temporal specificity, triggering diverse developmental programs with resolutions that can reach the single-cell level. Using larval feeding as a model system, Lohmann’s group proposes that HOX proteins may guide the recognition of interacting synaptic partners to regulate regional motor outputs.</p><p id="p-14">Among the many other highlights of the meeting, Amita Seghal (University of Pennsylvania) gave a keynote presentation on the molecular mechanisms of sleep (see summary in section below). Duojia Pan (Johns Hopkins University) spoke on Hippo signaling and expanding the physiological function of the pathway to many physiological processes beyond growth control. His group has linked the Hippo pathway to Toll signaling in the control of innate immunity. Alain Vincent (CNRS/University Toulouse 3) described a transcription factor code dynamically controlling muscle identity, revealing a cascade of regulatory interactions across position and developmental time. Jian Zhou (Princeton University) reported on efforts to systematically predict gene expression patterns in embryonic development. Zhou and colleagues used a computational approach to integrating cell lineage, genome-wide expression, and chromatin status data, providing comprehensive predictions for tissue-specific expression and a tool for exploratory analysis. Many new avenues of research were also explored in workshops, including well-attended sessions on the <em>Drosophila</em> Microbiota and the modMetabolome Model Organism Metabolomics Consortium.</p><p id="p-15">We look forward to meeting again next year at the 58th Annual <em>Drosophila</em> Research Conference, to be held March 29–April 2, 2017 in San Diego, CA.</p></div><div class="section other4" id="sec-4"><h2 class="">Mouse Genetics 2016</h2><p id="p-16">Mouse Genetics 2016 combined the 30th meeting of the International Mammalian Genome Society (IMGS) with the annual Mouse Molecular Genetics Meeting to provide a stimulating environment for sharing recent discoveries and developing new ideas and collaborations. Two named keynote lectures anchored the meeting: Joe Nadeau (Pacific Northwest Diabetes Research Institute) gave a lecture in honor of Verne Chapman, a founding member of the IMGS, in the “Translational and Systems Genetics” session. Nadeau told an intriguing story of epistasis, transgenerational epigenetics, and fertilization bias. Anna-Katerina Hadjantonakis (Memorial Sloan Kettering Cancer Center) gave a lecture in honor of Rosa Beddington, a leading mouse experimental embryologist. Using state-of-the-art imaging techniques, Hadjantonakis’ work showed that embryonic cells do not behave as classical experiments would suggest (for more details see summary below). In the “Stem Cells” session, Josh Brickman (Danish Stem Cell Institute) showed how embryonic stem cells can stall for time while making key decisions.</p><p id="p-17">The International Mouse Phenotyping Consortium provided updates in the “International Resources” session, revealing a high-throughput imaging platform for mouse embryos that facilitates detection of anomalies by nonspecialist groups. The “Technological Innovations” session continued this theme by promoting new embryo imaging techniques and CRISPR/Cas9 genome editing. In the “Development” session, Takashi Hiiragi (European Molecular Biology Laboratory, EMBL) showed that mechanical forces regulate asymmetric cell divisions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Maître <em>et al.</em> 2016</a>). “Comparative Genomics, Computational Methods, and Evolution” featured speaker Hopi Hoekstra (Harvard University), who described genes that regulate parenting behavior (“good dad, bad dad”) in deer mouse fathers. Freda Miller (SickKids Hospital Research Institute) showed how basic research in mouse models has led to clinical trials for brain injury cases in “Human Disease Models.” In “Epigenetics,” Jeannie Lee (Harvard University) demonstrated that the mouse X chromosome can still be partially dosage compensated (inactivated) when the noncoding RNA XIST was eliminated in early mouse development.</p><p id="p-18">Tyler Jacks (Massachusetts Institute of Technology) described how using <em>in situ</em> genome editing to engineer mouse cancer models has led to immune-targeted therapy in “Cancer and Immunology.”</p><p id="p-19">Keeping with the multi-organism theme of TAGC, workshops focused on cross-organism approaches and resources, such as genome editing, model organism databases (MODs), cell competition, and genetic reference populations. Mouse Genetics 2016 was preceded by a satellite symposium featuring trainee talks, five of which were selected for podium presentations in the main meeting. IMGS and GSA awards were given for top trainee talks and poster presentations. Jacob Moskowitz (University of Missouri) won the coveted “Verne Chapman Young Investigator” prize, and Krista Geister (Seattle Children’s Research Institute) received an award in honor of Mary Lyon for the top talk by a woman postdoctoral fellow.</p><p id="p-20">Mouse Genetics 2016 attendees left Orlando inspired and looking forward to the next meeting, an EMBL Conference: “Mammalian Genetics and Genomics: From Molecular Mechanisms to Translational Applications,” which will be held at the EMBL Advanced Training Centre, Heidelberg, Germany, October 24–27, 2017.</p></div><div class="section other5" id="sec-5"><h2 class="">Population, Evolutionary, and Quantitative Genetics Conference</h2><p id="p-21">The first Population, Evolutionary, and Quantitative (PEQ) Genetics Conference was a resounding success, serving as a hub that drew in many attendees from the six model organism-focused meetings, alongside those focused on nonmodel species. Many attendees appreciated having the program organized by genetic theme, and were excited to be introduced to new systems and unusual organisms. The meeting was launched with a keynote presentation by Patricia Wittkopp (University of Michigan) on using the yeast <em>S. cerevisiae</em> to investigate how mutation and selection affect the evolution of gene expression.</p><p id="p-22">A symposium the following day featured candidates for the James F. Crow Early Career Researcher Award, which honors the legacy of population genetics pioneer Jim Crow on the centennial of his birth. In a funny and moving tribute to this legacy, Daniel Hartl (Harvard University) talked of the valuable lessons he had learned from Crow, his PhD mentor, including treasuring your students. The symposium continued with six talented students and recent PhDs presenting on a diverse range of topics. Matthew Ackerman (Indiana University) spoke on a method for estimating pairwise relatedness coefficients from population genomic data. Emily Behrman (University of Pennsylvania) revealed striking seasonal oscillations in selection and adaptation in <em>Drosophila melanogaster</em> populations. Heath Blackmon (University of Minnesota) presented results from more than 1000 beetle species to explore the role of Y-aneuploidy in the evolution of sex chromosomes and genome architecture. Gili Greenbaum (Ben Gurion University) described a network theory-based approach to population structure analysis that does not require unrealistic assumptions. Sarah Sander (Cornell University), later announced as winner of the Crow Award, spoke on using fireflies as a model system for understanding animal signal and receptor evolution. Sandeep Venkataram (Stanford University) explored fitness pleiotropy and adaptive mutations in yeast using experimental evolution and high resolution lineage tracing.</p><p id="p-23">In a keynote session focused on complex traits, Dirk-Jan de Koning (Swedish University of Agricultural Sciences) highlighted an example of systems genetics in an applied context, using QTL mapping and GWAS to improve bone strength in egg-laying hens. Among the many other highlights of the meeting, Philipp Messer (Cornell University) described a population genetic model for CRISPR/Cas9-mediated gene drives. The results could be used to design more successful drive strategies or to incorporate mechanisms to limit the spread of a drive. Daniel Skelly (Duke University) presented a creative approach to mapping complex traits in yeast, combining a genetically diverse panel of cross-segregants and a method to deconvolve haplotype blocks from pooled libraries of unbarcoded strains. Beth Dumont (North Carolina State University) creatively used data from the Collaborative Cross to show that a locus on the X chromosome influences the genome-wide rate of meiotic recombination in female mice. Alexander Platt (Temple University) proposed a novel method for estimating the age of singleton or very rare alleles in a population using molecular marker information, drawing information from variants linked to the allele. Yun Ding (HHMI) described how quantifying variation in <em>Drosophila</em> courtship songs allowed genetic dissection of a behavioral difference. Richard Durbin (Wellcome Trust Sanger Institute) spoke on WGS of the cichlid fish species famous for their dramatic evolutionary radiation. Annalise Paaby (Georgia Institute of Technology) presented evidence that cryptic genetic variation is ubiquitous in wild <em>C. elegans</em>, in the form of alleles that have no effect unless the function of other genes is perturbed. The final keynote was given by John Willis (Duke University), on evidence for parallel genetic mechanisms in repeated adaptation of <em>Mimulus guttatus</em> to patches of serpentine soils and toxic mine tailings.</p><p id="p-24">The PEQ program ended with a stimulating session on epistasis, including a talk from Christian Landry (Université Laval), who explored the fate of duplicated yeast genes by examining compensatory changes in protein interaction networks. Gregory Lang (Lehigh University) investigated large-scale patterns of epistasis in yeast by measuring evolutionary trajectories in large pools of recombinant progeny containing random combinations of evolved mutations. David Rand (Brown University) spoke on epistasis between mitochondrial and nuclear genomes in <em>Drosophila</em>, finding that interactions were highly dependent on environmental background.</p><p id="p-25">The TAGC attendee survey revealed strong interest from the community in holding another PEQ meeting, although the details are still to be decided.</p></div><div class="section other6" id="sec-6"><h2 class="">Yeast Genetics Meeting</h2><p id="p-26">The Yeast Genetics Meeting was marked by 10 scientific sessions and two workshop sessions covering diverse aspects of transcriptional regulation, development, cell division, the stress response, intracellular trafficking, evolution, and new technologies. The conference opened with a session on genome dynamics punctuated by the Winge-Lindegren address of Rodney Rothstein (Columbia University Medical Center), which beautifully built on construction of the widely-used W303 strain to reveal the choreography of the DNA damage response and the mechanisms of homologous recombination. Day 2 saw an exciting session on post-transcriptional gene regulation, including Daniel Jarosz (Stanford University) describing the identification of ∼50 new yeast prions based on a transient overexpression of intrinsically disordered proteins. The “Epigenetics and Transcriptional Regulation” session featured new evidence presented by Ryan Janke (University of California, Berkeley) that the oncometabolite 2-hydroxyglutarate promotes conserved disruption of histone demethylation. The same work identified new metabolites with the potential to cause epigenetic instability. Thursday culminated with a session on human disease modeling, marked by the Lee Hartwell lecture delivered by Susan Gasser (Friedrich Miescher Institute for Biomedical Research) on the <em>in vivo</em> dynamics of chromatin, including striking new results on large-scale histone depletion after DNA damage. Day 3 began with “Division and Development,” including Michael McMurray (University of Colorado Anschutz Medical Campus), who described a first detailed analysis of the effects of the ascus on germination and mating behavior between sisters. Jessica Lao (University of California, San Francisco) described the first evidence of direct regulation of RNA degradation machinery by the DNA damage checkpoint. In the same session, James Broach (Penn State College of Medicine) received a lifetime achievement award and reviewed his long career using yeast to understand the biology of nutrient depletion stress, producing major insights into the RAS pathway, heterochromatin, and plasmids. Friday finished with several important talks on evolution. Aashiq Kachroo (The University of Texas at Austin) described efforts to understand the rules governing gene replaceability between yeast, humans, <em>Escherichia coli</em>, and plants by systematically rescuing essential gene deletions with orthologous proteins. Joseph Schacherer (University of Strasbourg) described the 1002 Yeast Genomes project, which sequenced &gt; 1000 diverse yeast isolates from natural environments, revealed an unprecedented view of genomic variation, and, coupled with phenotypic studies, permitted genotype–phenotype linkages to be suggested. As yeast usually pushes boundaries in this area, the “Revisiting Classical Genetics with New Technology” session on Saturday was extraordinary. Michael Costanzo (University of Toronto) presented the complete genetic interaction network of <em>S. cerevisiae</em>, moving toward the first long-sought wiring diagram of a eukaryotic cell. Zhimin Liu (Laufer Center) described new ultrahigh-throughput protein–protein interaction screening methods, which use deep sequencing of barcoded strains where bait- and prey-associated barcodes can be fused for sequencing of interacting proteins. Uri Weill (Weizmann Institute of Science) described a collection of N-terminal GFP fusion proteins that can be rapidly swapped to any other tag in an automated fashion. Agnes Michel (ETH Zürich) described coupling a saturating transposon–insertion screen with deep sequencing. Remarkably, this could map not only essential genes, but had sufficient resolution to reveal essential protein domains, genetic interacting partners, and drug sensitive or resistant alleles. Finally, Saturday night saw Lars Steinmetz (Stanford University and EMBL) receive the Ira Herskowitz Award, highlighting his important work on pervasive transcription and the technological advances that have led to a redefinition of the complexity of transcriptional isoforms and their potential role in cellular heterogeneity. The last day opened with a session on metabolic regulation (“The Fat and Sweet Sides of Life”), including Kobi Simpson-Lavy (Tel Aviv University), who shed light on the implications of protein aggregation in glucose repression. At the end of the session, poster awards were presented to three outstanding graduate students and one outstanding undergraduate student.</p><p id="p-27">The next Yeast Genetics Meeting will be held in 2018. Stay tuned for more details of the dates and location!</p></div><div class="section other7" id="sec-7"><h2 class="">12th International Conference on Zebrafish Development and Genetics</h2><p id="p-28">A highlight of the meeting was Didier Stainier (Max Planck Institute for Heart and Lung Research) bringing to a close the long, arduous struggle to clone one of the first zebrafish mutants to be described: <em>cloche</em>. The fascinating aspect of <em>cloche</em> is that its embryos are bloodless and also lack vasculature, meaning that it plays a very early role in the diversification of the early mesoderm into endothelial and hematopoietic tissue fates.</p><p id="p-29">It was recognized long ago that the <em>cloche</em> locus occupied a position very close to the distal tip of the telomere on chromosome 13. This region on the zebrafish genome is poorly assembled thereby greatly complicating efforts to identify the underlying gene by classical positional cloning. The pursuit of this locus became the holy grail for several groups.</p><p id="p-30">The approach taken by the Stainier laboratory provides further evidence of the power conferred by CRISPR/Cas9 technology. The researchers reasoned that <em>cloche</em> must be expressed prior to gastrulation and so carried out RNA-seq on pools of individually genotyped wild-type and <em>cloche</em> mutant embryos at 6–10 h postfertilization to identify significantly downregulated genes. Having identified 19 candidates, 18 gave no phenotype upon mutation using CRISPR. Knocking out the final candidate to be tested, <em>npas4-like (npas4l)</em>, phenocopied the mutant. This gene encodes a member of the bHLH-PAS family of transcription factors and directly regulates the <em>etv</em> and <em>tal1</em> genes, the earliest expressed endothelial and hematopoietic transcription factor genes identified so far. The identification of <em>cloche</em>/<em>npas4l</em> as a master regulator of endothelial and hematopoietic fate may lead to improved protocols for the generation of endothelia and hematopoietic cells <em>in vivo</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Reischauer <em>et al.</em> 2016</a>).</p><p id="p-31">In another impressive study, Daniel Grimes (Princeton University) utilized zebrafish deficient in the <em>ptk7</em> gene, which display a curved spine, reminiscent of human adolescent idiopathic scoliosis (IS). He revealed underlying defects in the formation and function of motile cilia in the central nervous system of this mutant, which perturbed the flow of cerebrospinal fluid (CSF), and caused abnormal spinal curvatures as the fish grew. Restoration of cilia motility after the onset of scoliosis blocked spinal curve progression. The results implicate irregularities in CSF flow as an underlying biological cause of IS, and suggest that noninvasive therapeutic intervention may prevent severe scoliosis (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grimes <em>et al.</em> 2016</a>).</p><p id="p-32">Many talks and posters highlighted the use of zebrafish as disease models. Craig Ceol (University of Massachusetts Medical School) showed that Gdf6 expression is upregulated in melanoma, and plays a role in inducing their neural crest cell identity, which is key to their malignant behavior. Marcel den Hoed (Uppsala University) presented a zebrafish model of atherosclerosis and demonstrated that Flt1 protects vasculature in the larvae from lipid deposits. Steve Farber (Carnegie Institution for Science) introduced new tools for studying high cholesterol levels. Tom Carney (IMCB A*STAR) presented work in which the liver in zebrafish is “humanized” by expressing human detoxification enzymes in this organ. This allows zebrafish to be used to more accurately assess toxicity of compounds on the human organ. Finally, in his plenary lecture, Len Zon (HHMI/Children’s Hospital Boston) showcased the value of the zebrafish as an animal model for human disease (for details see summary below).</p><p id="p-33">The Chi-Bien Chien Award was presented to Adam Miller (University of Oregon) for his work on electrical synapse formation. This award is given to a graduate student, postdoctoral associate, or newly appointed faculty member who has made a significant contribution to the zebrafish field. The George Streisinger Award recognizes a senior investigator who has made outstanding and continued contributions to the advancement of the zebrafish field. The inaugural award was given to Chuck Kimmel (University of Oregon) and will be presented at the upcoming Strategic Conference of Zebrafish Investigators in 2017.</p><div id="sec-8" class="subsection"><div id="sec-9" class="subsection"><h4>Other highlights included:</h4><p id="p-34">Jessica Nelson (University of Pennsylvania) revealed an unexpected role of huntingtin protein in learning.</p><p id="p-35">Shinsuke Seki (Tokyo University of Marine Science and Technology) used medaka spermatogonia to produce oocytes and sperm when transplanted; it may work for zebrafish too!</p><p id="p-36">Ashley Bruce (University of Toronto) showed that yolk syncytial layer (YSL) nuclei move through the microtubule network to drive epiboly (vegetal movement of the blastoderm and the YSL to enclose the yolk cell). It was previously thought to be driven by depolymerization of the network.</p><p id="p-37">Chase Bryan (University of Utah) produced some amazing movies of optic cup morphogenesis.</p><p id="p-38">Xuefei Yuan (The Hospital of Sick Children) gave a great talk on finding noncoding regions that drive expression in cardiac precursors.</p><p id="p-39">Yahui Lan (Weill Cornell Medical College) showed that zebrafish <em>tet2/3</em> mutants have a loss of epicardial migration and proliferation, and highlighted the role of DNA methylation in the process.</p><p id="p-40">Aaron Savage (University of Sheffield) used tissue-specific expression of NLS-Cas9 for tissue-specific knockouts by injecting sgRNA.</p><p id="p-41">Autumn Marsden (University of Iowa) showed that the EF-hand domain in Naked Cuticle (Nkd) is not required for canonical Wnt inhibition, but is required for Wnt-PCP.</p><p id="p-42">Tamara Stawicki (University of Washington) showed that intraflagellar transport genes are important in lateral line hair cell resistance to neomycin.</p><p id="p-43">Rob Cornell (University of Iowa) analyzed a tissue involved in oral-facial development (the periderm) to find genes involved in oral-facial clefting.</p><p id="p-44">The 13th Zebrafish Development and Genetics Meeting will be held in the summer of 2018 in Madison, WI.</p></div></div></div><div class="section other8" id="sec-10"><h2 class="">Model Organism Genome Resources</h2><p id="p-45">The gathering of so many model organism geneticists in one place made TAGC a natural forum for discussion of issues surrounding the funding and organization of MODs. Immediately before the meeting, several leaders in the community, with logistical support from GSA, wrote a statement of support for the MODs that was signed by over 11,000 model organism geneticists and presented at TAGC to Francis Collins and the National Institutes of Health (<a href="http://genestogenomes.org/action-alert-support-model-organism-database-funding/">http://genestogenomes.org/action-alert-support-model-organism-database-funding/</a>). The statement expressed enthusiastic support for the initiative to integrate elements of the existing MODs, but voiced concern about the possibility that overall support for MODs would not be sustained. Collins directly addressed these concerns in his joint plenary keynote presentation (see summary below).</p><p id="p-46">The meeting also coincided with the launch of the Alliance of Genome Resources (AGR), which was formed by the Gene Ontology Consortium and six MODs: <em>Saccharomyces</em> Genome Database, WormBase, FlyBase, Zebrafish Model Organism Database, Mouse Genome Database, and Rat Genome Database. The AGR was created to provide better support for the biological sciences via an integration of shared data, standardization of data models and interfaces, and unified outreach to researchers, educators, and the public (<a href="http://genestogenomes.org/model-organism-databases-join-forces-announcing-the-alliance-of-genome-resources/">http://genestogenomes.org/model-organism-databases-join-forces-announcing-the-alliance-of-genome-resources/</a>).</p></div><div class="section other9" id="sec-11"><h2 class="">Jef Boeke</h2><div id="sec-12" class="subsection"><h3>Yeast, the model eukaryote, leads the way in designer genomes</h3><div id="sec-13" class="subsection"><h4>Synthesizing and SCRaMbLEing a genome:</h4><p id="p-47">For the past 10 years, Jef Boeke (New York University Langone School of Medicine) has led an international group of researchers in an effort to synthesize the yeast genome. Known as <em>S. cerevisiae</em> 2.0 (Sc2.0), this “designer” genome will be fully synthesized from oligonucleotides. With support from the National Science Foundation along with additional funding secured by collaborating institutions around the world, this huge undertaking aims to build a yeast genome that will reveal fundamental insights into biology.</p><p id="p-48">To construct Sc2.0, the researchers are using the annotations in the <em>Saccharomyces</em> Genome Database (SDG) along with an assembly method called SWAP-IN. The investigators use SWAP-IN to gradually swap pieces of the yeast DNA for pieces of synthetic DNA until they have a designer synthetic chromosome derived completely from oligonucleotides. So far, the genome has been designed, and more than 60% of the yeast genome has been synthesized using this approach.</p><p id="p-49">Noting that designing and synthesizing a genome is not an interpretive exercise like reading a book or a DNA sequence, Boeke said planning an entire genome is more akin to a creative act involving a series of decisions. The researchers decided to modify the yeast genome in ways that would make it more stable, such as by removing repeat elements, introns, and transposons. They also removed all the transfer RNA (tRNA) genes, which are hotspots for genome instability, with a plan to relocate them to a special new chromosome they call a “neochromosome,” being designed by Patrick Cai and colleagues at the University of Edinburgh.</p><div id="sec-14" class="subsection"><h5>Insights on chromosome number:</h5><p id="p-50">When tackling chromosome 1, yeast’s smallest chromosome, the researchers were concerned that making it smaller by removing large subtelomeric repeat elements might make the chromosome unstable. This concern, combined with the fact that adding the new neochromosome would result in 17 chromosomes for Sc2.0, led to a decision to fuse chromosome 1 to another chromosome. This was successfully achieved using a CRISPR-based method, with little effect on global function; the yeast still grew well.</p><p id="p-51">After their success in fusing chromosome 1, the team considered other organisms with dramatic variations in chromosome number. For example, “Jack Jumper ant” females have a 2n = 2 karyotype, and since the males are haploid they operate with a single chromosome (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crosland and Crozier 1986</a>). At the other extreme is a fern with hundreds to &gt; 1000 chromosomes. Since they had successfully reduced the budding yeast chromosome number from 16 to 15, the researchers pondered how much further they could go. Thus far, they have fused several chromosomes, with no serious fitness defects resulting from these fusions.</p><p id="p-52">Following up on earlier work by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Neurohr <em>et al.</em> (2011)</a>, which enabled construction of a chromosome one-quarter the size of the yeast genome, the researchers are now working to string even more chromosomes together. They aim to first produce a yeast genome with four chromosomes of roughly equal size. If this can be accomplished, they will attempt to see whether the entire genome can reside on just one chromosome. If they are not able to reduce the number of chromosomes this much, it could help reveal factors underlying differences in chromosome number between species.</p></div><div id="sec-15" class="subsection"><h5>Ribosomal DNA and lessons on 3-D chromosome structure:</h5><p id="p-53">Ribosomal DNA (rDNA) resides on chromosome 12. Junbiao Dai and colleagues from Tsinghua University built a synthetic chromosome 12 that retained the yeast rDNA, and then removed the rDNA after supporting the cell’s growth with plasmid-borne rDNA. They were then able to “reprogram” the species identity by altering the internal transcribed spacer sequences of the rDNA. Subsequent experiments demonstrated that moving rDNA to another chromosome did bring about substantial changes in chromatin architecture, but was not associated with large fitness defects.</p><p id="p-54">Collaborators at the Pasteur Institut, led by Romain Koszul, also studied the trajectories and 3-D chromatin structure of the various synthetic chromosomes, finding that the way the synthetic chromosomes are positioned in the nucleus was surprisingly similar to the native counterparts. This suggests that transposable elements do not significantly impact the structure of chromosomes in the nucleus, in yeast at least.</p></div><div id="sec-16" class="subsection"><h5>Genome scrambling:</h5><p id="p-55">The researchers developed a genome scrambling system called SCRaMbLE (synthetic chromosome rearrangement and modification by <em>LoxP</em>-mediated evolution) that can be used to create many variants from genomes with one or more synthetic chromosomes. This approach involves inserting symmetrical <em>LoxP</em> sites in the 3′ untranslated region of each nonessential gene and at various landmarks, such as the site where a tRNA gene is removed. Once the synthetic chromosome is synthesized and validated structurally, the researchers add a chemically regulated Cre recombinase. When the recombinase is activated, it induces the formation of precisely defined deletions, inversions, translocations, and duplications.</p><p id="p-56">Boeke likened the technique to evolution on steroids. Activating the Cre recombinase in a synthetic or partially synthetic genome yields many millions of variants in one small tube of cells. These distinct “scrambleotypes” give rise to varied phenotypes that can be assessed for desired qualities.</p><p id="p-57">If the genes in the Sc2.0 genome are thought of as a deck of 5000 cards, then SCRaMBLE can be used to delete any combination of cards (genes), change card order, and duplicate or add back extra cards. Because of this capability, genome scrambling can be used to determine the minimal genome possible and to obtain gain-of-function mutations in which the altered gene product possesses a new function or a new pattern of gene expression. Since SCRaMBLE makes sequential changes, it can also be used to explore possible trajectories for evolutionary changes in gene content. SCRaMbLE might even be used to understand how much genome rearrangement it takes to define a new species in the laboratory. It may best be performed in heterozygous diploids, because when used in haploids, deleting genes is more likely to be lethal.</p><p id="p-58">SCRaMBLE was used on the right arm of synthetic chromosome 9, the smallest chromosome arm in the yeast genome, which was synthesized in a circle with 43 segments (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shen <em>et al.</em> 2016</a>). Using SCRaMBLE on yeast cells containing this synthetic chromosome arm allowed the researchers to choose random colonies with high fitness. Examining all the different scrambleotypes from the different colonies revealed massive combinatorial diversity. Even with deletions that reduce the length of the chromosome arm by a third, and massive duplications that make it almost four times larger than the original, high fitness was still obtained, and the rest of the genome was unaffected. More recently, the researchers have used genome scrambling to look for desirable phenotypes, such as yeast variants that grow at higher temperatures, which might be useful for brewing, winemaking, and clinical applications.</p></div><div id="sec-17" class="subsection"><h5>New possibilities with neochromosomes:</h5><p id="p-59">Neochromosomes are synthetic chromosomes not found in nature. Boeke’s team has used combinatorial neochromosome assembly to create various pigment combinations in yeast and is exploring the use of multicolor yeast for living artwork (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). With robotics, they expect to be able to make 100 kb of chromosome every 2 weeks. Boeke is a founder and director of Neochromosome, Inc., through which he intends to commercialize neochromosomes assembled in yeast.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="This &#x201C;living logo&#x201D; is made up of yeast growing on a rectangular Petri dish. The yeast strains are engineered to produce large amounts of violacein, a pigment from Chromobacterium violaceum, leading to a very dark purple that appears black here (Mitchell et al. 2015), and wild-type yeast that appear white here. The yeast cells are deposited on the plate using an acoustic droplet ejection robot, creating 25,000 &#x201C;biopixels.&#x201D; For more information, see www.yeastart.org." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-151034724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;This &#x201C;living logo&#x201D; is made up of yeast growing on a rectangular Petri dish. The yeast strains are engineered to produce large amounts of violacein, a pigment from &lt;em&gt;Chromobacterium violaceum&lt;/em&gt;, leading to a very dark purple that appears black here (Mitchell &lt;em&gt;et al.&lt;/em&gt; 2015), and wild-type yeast that appear white here. The yeast cells are deposited on the plate using an acoustic droplet ejection robot, creating 25,000 &#x201C;biopixels.&#x201D; For more information, see www.yeastart.org.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3765/F1.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/12/3765/F1.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470002" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-60" class="first-child">This “living logo” is made up of yeast growing on a rectangular Petri dish. The yeast strains are engineered to produce large amounts of violacein, a pigment from <em>Chromobacterium violaceum</em>, leading to a very dark purple that appears black here (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mitchell <em>et al.</em> 2015</a>), and wild-type yeast that appear white here. The yeast cells are deposited on the plate using an acoustic droplet ejection robot, creating 25,000 “biopixels.” For more information, see <a href="http://www.yeastart.org">www.yeastart.org</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h5>Moving to mammals:</h5><p id="p-61">Wrapping up, Boeke noted that it might be possible to use SWAP-In with mammalian cells. The Genome Project-write (GP-write) project is a proposal to develop technology that will reduce the costs of engineering and testing of large genomes more than 1000-fold within 10 years (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boeke <em>et al.</em> 2016</a>). While the Human Genome Project aimed to read the human genome, GP-write has the goal of building human and other genomes from scratch. This could enable the synthesis of many mammalian-sized genomes, perhaps thousands or more, which would allow testing of the impact of systematic alternations at a genome scale.</p><p id="p-62">GP-write is being envisioned as an international collaborative project, much like the Sc2.0, with a launch planned for late 2016. In addition to developing new technologies, the project also plans to develop an ethical framework for genome-scale engineering.</p></div></div></div></div><div class="section other10" id="sec-19"><h2 class="">Francis Collins</h2><div id="sec-20" class="subsection"><h3>Accelerating insights from basic genetics</h3><div id="sec-21" class="subsection"><h4>A perspective from NIH Director Francis Collins:</h4><p id="p-63">Francis Collins has served as Director of the NIH since 2009. A physician and geneticist, he previously led the International Human Genome Project and was the director of the NIH’s National Human Genome Research Institute (NHGRI) for 15 years. He offered a 30,000-foot view of how the NIH supports basic science, and specifically model organism research, highlighting what challenges NIH is facing, and how it is empowering research across different model organisms to further invigorate and inspire biomedical scientists.</p><p id="p-64">Collins dedicated his talk to the memory of his friend and colleague William Gelbart of Harvard University, a widely respected molecular and cellular biologist who encouraged and empowered others throughout his extraordinary life and career. Among his many accomplishments, Gelbart created FlyBase, the genetic database that serves as an indispensable resource for the <em>Drosophila</em> model organism community.</p><div id="sec-22" class="subsection"><h5>NIH support for basic research:</h5><p id="p-65">As the largest supporter of biomedical research in the world, the NIH is dedicated to furthering both basic and applied science. Although its stated mission clearly includes both fundamental knowledge and its potential applications, an overall decline in science funding over the past 15 years has led to a misperception that the NIH is prioritizing application-driven research over basic science. Emphasizing that the NIH administration strongly supports basic research, Collins outlined a recent analysis of the NIH’s grant-funding track record that demonstrates this commitment (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Collins <em>et al.</em> 2016</a>).</p><p id="p-66">Despite significant budgetary challenges faced by the agency as a whole, Collins said that the NIH remains committed to outstanding research being conducted in the model organism community. In response to a study published in <em>Genetics</em>, which reported a decline in NIH funding for <em>Drosophila</em> research (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wangler <em>et al.</em> 2015</a>), the NIH initiated a follow-up investigation. Using sophisticated search techniques to comb through verified grant records (as opposed to the simple keyword searches used by <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wangler <em>et al.</em> (2015</a>), an extensive analysis by the NIH’s Office of Portfolio Analysis (OPA) concluded that there had been, in fact, no reduction in <em>Drosophila</em> funding (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analyses conducted by the National Institute of Health&#x2019;s Office of Portfolio Analysis show that the award rates for Research Project Grants (R01s) involving key model organisms have remained steady and, in some cases, are increasing when compared to all R01s. ARRA stands for American Recovery and Reinvestment Act of 2009." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-151034724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analyses conducted by the National Institute of Health&#x2019;s Office of Portfolio Analysis show that the award rates for Research Project Grants (R01s) involving key model organisms have remained steady and, in some cases, are increasing when compared to all R01s. ARRA stands for American Recovery and Reinvestment Act of 2009.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3765/F2.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/12/3765/F2.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-67" class="first-child">Analyses conducted by the National Institute of Health’s Office of Portfolio Analysis show that the award rates for Research Project Grants (R01s) involving key model organisms have remained steady and, in some cases, are increasing when compared to all R01s. ARRA stands for American Recovery and Reinvestment Act of 2009.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">According to Collins, the OPA found similar results in its analyses of NIH support for research involving other model organisms. For example, <em>C. elegan</em>s funding has remained steady, and investigator-initiated research (R01) grants for <em>Danio rerio</em> (zebrafish) are actually increasing. The OPA also analyzed R01 success rates to determine whether there could be a bias related to the scientific peer-review process. In fact, the results suggest the opposite: grant success rates are substantially higher for model organism research involving <em>Drosophila</em>, <em>C. elegans</em>, and zebrafish.</p><p id="p-69">Research trends for mouse models are more difficult to parse, but Collins noted that almost half of all R01 projects utilize mice, suggesting that funding for research using this key model organism remains robust. Collins pointed attendees to the NIH Office of Extramural Research’s “Open Mike” blog (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lauer 2016</a>) for further discussion of NIH’s support of model organism research.</p></div><div id="sec-23" class="subsection"><h5>Dealing with big data:</h5><p id="p-70">Big data are both an enormous opportunity and an enormous challenge for scientists, and the NIH is devoting tremendous time and energy to wrestling with this issue. Collins cited an in-house estimate indicating that NIH science generated ∼650 petabytes of data in the past year alone, a number that continues to grow rapidly. For comparison, he pointed out that the entire content of the Library of Congress comprises only 3 petabytes of data.</p><p id="p-71">Big data are central to the practice of science today (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bourne <em>et al.</em> 2015</a>). An NIH Working Group on Data and Informatics, created in 2012 to examine this issue, recommended emphatically that the NIH confront big data challenges head-on (NIH 2013). To that end, the NIH is crafting a plan to capture and manage research data more effectively, tag it with appropriate metadata, and make it more easily findable and computable by the worldwide research community. One important NIH initiative in this area is the Big Data to Knowledge (BD2K) training and research program, created in 2013 with an annual budget that has been growing to reach $100 million. Philip Bourne was hired as Associate Director of Data Science (ADDS) for the NIH and also coleads the Scientific Data Council. Phil and his ADDS team are currently leading BD2K, and also seeking to assist management of data and informatics priorities across all 27 NIH institutes. Complementing those efforts, the NIH has also recently established a cross-NIH task force in charge of data infrastructure. The task force is seeking ways to pilot deposition of data sets and appropriate analytics in the cloud.</p></div><div id="sec-24" class="subsection"><h5>Data challenges in the model organism research community:</h5><p id="p-72">Like most other areas of biomedical science, model organism research is generating ever-larger data sets as genomic techniques, computational biology, and related research techniques continue to become more sophisticated. In addition, Collins noted an emerging interest in the community: analyzing data sets across various species of model organisms. Therefore, model organism researchers are uniquely positioned to become pioneers in the effort to bridge data management, access, and availability across large and often disparate databases, enabling scientists to follow innovative and unforeseen research paths.</p><p id="p-73">Model organism-specific genome databases came about in the 1990s, alongside the Human Genome Project. Collins said these databases, often funded by the NHGRI, are well designed, useful, and have the full support of the NIH. They are also extremely successful: tens of thousands of researchers use the data, which go well beyond genomics, on a monthly basis. However, each organism-specific database was created as a separate entity, and, as such, they lack shared standards that could further encourage cross-species analyses. Therefore, the NIH envisions a reorganization of the existing model organism databases (Mouse Genome Informatics, WormBase, ZFIN: The Zebrafish Model Organism Database, <em>Saccharomyces</em> Genome Database, FlyBase, and the Gene Ontology Consortium) to open them up to a wider community and facilitate more cross-organism investigations.</p><p id="p-74">According to Collins, such a change would offer many benefits for science. For example, a search for human orthologs in different model organisms currently requires a researcher to run individual organism searches, analyze results, and perform calculations on each database separately, and then attempt data integration. That is a very complex process to answer a fundamental and valuable research question. Collins said tackling the same question with a more unified and standardized genomic database that spans multiple model organisms would be much more efficient and could produce more reliable results.</p><p id="p-75">Another reason for integrating these databases is that the cost of hosting ever-growing stores of data on individual servers is increasingly straining the NIH’s budget, whereas cloud-based storage alternatives could enable a more sustainable funding model across genome databases. Cloud computing also can make data analysis simpler, removing the slow download times that large data sets require in order for analyses to be performed. Toward this end, BD2K is currently working to establish the NIH Commons, a cloud-based virtual ecosystem of indexed, accessible biomedical data. The NIH Commons will widen the audience for powerful data stores, while at the same time simplifying their use to increase the ease and efficiency of scientific inquiry.</p><p id="p-76">MODs would make excellent pilot databases for the Commons, Collins suggested, noting that the NHGRI has met with the MODs’ principal investigators to exchange ideas on the proposed integration. As an outgrowth of those discussions, the AGR has been formed and tasked with the challenging goal of charting a plan for integrating the MODs. To help achieve this bold vision, Collins said those running each MOD will need to update data management practices, enhance the interoperability of data and the integration of content, commit to reducing redundancies, and share best practices.</p><p id="p-77">While many scientists see value in such a reorganization, some remain worried that such an effort could degrade these critical resources. Collins pointed to a recent letter to the NIH signed by more than 10,000 model organism researchers that endorsed a general need for database integration, but also expressed concerns about the speed of the process and the need to retain certain essential functions of the individual databases. Stating that the NIH takes such concerns very seriously, Collins emphasized that the agency is taking great pains to preserve the value and services that each database currently provides, and does not intend to rush what is unquestionably going to be a complex process. Collins stressed, however, that maintaining the status quo is not an option: individual databases must be made interoperable to create a single, federated view of model organism data that will be useful, useable, and sustainable.</p></div><div id="sec-25" class="subsection"><h5>Bolstering public support for basic research:</h5><p id="p-78">Wrapping up, Collins reflected on the very tight fiscal climate within which the NIH—and the biomedical research community that it supports—has been operating since the early 2000s. These have been lean years, he said, and the constrained budget is reflected in today’s historically low success rates for grant applications. However, Collins said that he sees light at the end of the tunnel: The NIH budget for Fiscal Year (FY) 2016 included an increase of $2 billion, a 6.6% increment that was made possible by bipartisan Congressional support. Likewise, Collins noted that the FY2017 House and Senate budget proposals for the NIH also look promising.</p><p id="p-79">Collins urged attendees to consider their own roles in making the case to Congress and to the American taxpayer for the value of biomedical research. Basic science needs champions to tell the world how essential such work is, how remarkable the findings are, and how such discoveries can serve to advance medicine and stimulate the economy. Indeed, Collins said model organism researchers should seize this opportunity to become a symphony of voices sharing their excitement over their work and its potential to change the world.</p></div></div></div></div><div class="section other11" id="sec-26"><h2 class="">Jennifer Doudna</h2><div id="sec-27" class="subsection"><h3>CRISPR-Cas genome engineering</h3><div id="sec-28" class="subsection"><h4>The biology, technology, and ethics of a remarkable technique:</h4><p id="p-80">Ever since the discovery of the double helix structure of DNA, scientists have wondered if it would be possible to manipulate genomes, perhaps even at the level of individual base pairs. Jennifer Doudna (University of California, Berkeley) summarized her work in the development of CRISPR-Cas9, a technology that is ready to do just that.</p><p id="p-81">Although Doudna is today perhaps best known for her contributions to CRISPR genome editing, her career has focused not on DNA, but on RNA structures. Doudna recounted how the biochemistry of CRISPR systems and CRISPR-associated genes first came to be understood, how the gene editing technology was then developed, and what ethical implications are arising as the field moves forward.</p><div id="sec-29" class="subsection"><h5>Insights into bacteria–virus interactions:</h5><p id="p-82">The CRISPR story starts with bacteria. Bacteria live in constant battle with viruses, and so they have evolved many different ways to fight off viral infections. One of the ways is through “memorizing” sequences of viral DNA. In the context of studying these mechanisms, Doudna was introduced by Jill Banfield, a colleague at UC Berkeley, to the phenomenon of CRISPR, which are repetitive sequences of DNA in a bacterial genome. These repetitions flank short DNA sequences, 30–40 bp long, that originally come from invading viruses. CRISPR evolved as a method for bacterial cells to maintain a genetic record of viruses that have infected them, while also passing on that record to future generations of bacteria. In addition, specific “<em>cas</em>” (CRISPR-associated) genes are located near these sequences, which suggested involvement of a conserved pathway.</p><p id="p-83">In subsequent studies, Doudna and her colleagues further deciphered the biochemistry of this process. CRISPR-Cas proteins can detect foreign DNA in a cell (such as that from a virus) and integrate small sequences of that foreign DNA into the bacterial CRISPR array. These arrays become RNA copies that then combine with Cas proteins to become DNA “detectives,” searching for viruses by finding foreign DNA that matches the RNA transcripts. If found, the viral DNA is bound, cut, and degraded. This process illustrated an important way that bacteria cleverly adapt to viral invaders, and also provided valuable clues for a new approach to altering the genome.</p></div><div id="sec-30" class="subsection"><h5>Tracing the role of RNA:</h5><p id="p-84">Doudna’s work had long focused on how RNA controls gene expression in cells, and especially how it can recognize other nucleic acids. This pursuit led her to a collaboration with Emmanuelle Charpentier (Max Planck Institute for Infection Biology). Together, Doudna and Charpentier identified the biochemical activity of one Cas enzyme, named “Cas9” (CRISPR-associated protein 9).</p><p id="p-85">Further study revealed aspects of Cas9 that make it truly remarkable (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jinek <em>et al.</em> 2012</a>); it can bind to double-stranded DNA by matching a 20-nucleotide guide sequence in an RNA molecule. This reaction requires a second RNA molecule, known as the “tracr” (<em>trans</em>-activating CRISPR RNA), which interacts with the end of the CRISPR RNA enabling Cas9 to easily bind. This dual RNA-guided protein is what provides the immunity from viruses that previous studies had found but not completely explained.</p></div><div id="sec-31" class="subsection"><h5>Learning to harness the power of Cas9:</h5><p id="p-86">Understanding the mechanisms behind Cas9 suggested it might be harnessed to serve as a tool for genome editing. It could even be simplified; Doudna’s RNA expertise enabled her to connect the two RNA molecules and create one single RNA guide that would contain both the “handle” for Cas9 to bind and the DNA-targeting sequence. The combination is easily programmed to recognize a desired DNA sequence and make a targeted double-strand break.</p><p id="p-87">Doudna and her team developed a 3-D structural model to illustrate the details. Cas9 attaches to the guide RNA, and together they recognize the DNA sequence, unwinding the double helix to allow the RNA to bind and form an RNA/DNA hybrid that displaces the other DNA strand. Finally, Cas9 makes breaks in each DNA strand at the targeted sequence.</p><p id="p-88">Cas9 comes on the heels of a number of previous gene editing technologies, including zinc finger nucleases and transcription activator-like effector nucleases. These methods also introduce double-strand breaks at specific sequences, triggering DNA repair via either the nonhomologous end joining pathway or homology-directed repair. Although they are effective, engineering and testing these proteins is extremely expensive and time-intensive. By contrast, Cas9 can be easily and quickly programmed, a vast improvement over the previous methods.</p><p id="p-89">As their understanding of Cas9 grew more detailed, Doudna and her team remained stymied by one question: how does Cas9 unwind the DNA strands without an external energy source? Through comparisons of crystallographic images of Cas9 at work, they found that the protein radically changes its conformation as it forms a path with the guide RNA. It then binds to the DNA and changes again, pushing away one DNA strand as the other strand base-pairs, or “zips together” with the RNA guide. Follow-up studies using fluorescent markers confirmed these structural changes and pointed to their role in allowing the protein to only cut DNA that is a perfect, or near perfect, match to the guide RNA. The team also discovered that the search for DNA is three-dimensional: the protein does not bind once and slide along the strand until finds a match, but rapidly binds and releases as it searches for the target sequence (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Knight <em>et al.</em> 2015</a>).</p></div><div id="sec-32" class="subsection"><h5>The rapid rise of CRISPR-Cas9:</h5><p id="p-90">Though there are still more details to be fully uncovered, the CRISPR-Cas9 gene editing system has worked remarkably well in the many cell types in which it has been tested and, as a result, has been rapidly adopted in many fields.</p><p id="p-91">Doudna offered three reasons for the success and rapid rise of CRISPR-Cas9. First, the recognition mechanism relies on RNA/DNA base pairs, not a protein–DNA interaction, so it is much easier to program and change, akin to programming software rather than hardware. Second, CRISPR-Cas9 technology became available at the same time that researchers were learning much more about the role of genes in disease, and could therefore envision a range of enticing applications for the process in medicine and other fields. Finally, thanks to its natural evolution in bacteria, CRISPR-Cas9 is fast and accurate when detecting target DNA sequences.</p></div><div id="sec-33" class="subsection"><h5>Future directions and ethical considerations:</h5><p id="p-92">Despite its success so far, Doudna spoke of three key challenges to genome editing via CRISPR-Cas9: delivering it to the necessary cells or tissue, controlling the repairs, and considering the ethics of potential applications. This last challenge is especially important when considering the potential use of CRISPR-Cas9 to alter a human genome.</p><p id="p-93">Doudna’s lab so far has focused on the biochemistry involved, and specifically the question of how to use what we know so far about CRISPR-Cas9 in order to deliver it to different types of cells. To that end, the team is currently testing the delivery of CRISPR-Cas9 and guide RNA as a preassembled, purified protein/RNA combination. Results for human T cells and mouse brain cells so far have been promising and suggest that CRISPR-Cas9 could eventually be used to treat various genetic disorders.</p><p id="p-94">The ethical implications of this technology deserve careful scrutiny, and care must be taken to ensure that CRISPR-Cas9 is applied for biomedical, agricultural, and environmental advances in ways that are both safe and ethically sound. To this end, Doudna asserted that there is an urgent need for informed scientists to engage in broader discussions about the wider ethical and societal implications of this work.</p></div></div></div></div><div class="section other12" id="sec-34"><h2 class="">Anna-Katerina Hadjantonakis</h2><div id="sec-35" class="subsection"><h3>Single cells get together: cell lineage specification and tissue morphogenesis in the early mouse embryo</h3><div id="sec-36" class="subsection"><h4>Challenging the dogma about tissue segregation:</h4><p id="p-95">Anna-Katerina Hadjantonakis (Memorial Sloan Kettering Cancer Center) seeks to visualize the complexities of cell behavior in living mouse embryos. By doing so, her team want to understand how tissues are built. Their experiments have revealed a previously unknown role for extraembryonic visceral endoderm cells as the gut tissue develops in the mouse embryo.</p><p id="p-96">These findings challenge a long-held dogma in mammalian development that there is a strict segregation of embryonic and extraembryonic tissue. What Hadjantonakis and her team have seen instead is that some cells derived from the visceral endoderm, which begin in extraembryonic areas, persist in the embryo and contribute to the development of the future gut.</p><p id="p-97">The presentation honored renowned developmental biologist Rosa Beddington; much of Hadjantonakis’ work builds from Beddington’s legacy. It was Beddington who first alerted the field to the importance of the visceral endoderm.</p><div id="sec-37" class="subsection"><h5>Mammalian embryonic development:</h5><p id="p-98">For context, Hadjantonakis provided an overview of the currently accepted understanding of embryonic development in mammals, and specifically mouse embryos in the pregastrulation and gastrulation phases. The blastocyst, a universal stage of early mammalian development, has three cell types: the epiblast, trophoblast, and primitive endoderm. As the blastocyst grows, multiple interactions between these three cell types are required to determine the specification and position of their derivative tissue layers. This crosstalk is essential to successful mammalian development.</p><p id="p-99">As embryo development continues, the trophoblast cells form the placenta, the primitive endoderm cells form the amniotic sac, and the epiblast cells form somatic cells which comprise the three germ layers of the developing embryo (the endoderm, mesoderm, and ectoderm). Put another way, it is thought that the trophoblast and primitive endoderm form extraembryonic tissue but do not form any part of the mouse embryo. Rather, the epiblast forms all embryonic tissue, and thus gives rise to all of the body’s somatic cells as well as germ cells.</p><p id="p-100">In contrast, Hadjantonakis’ research suggests that descendants of the primitive endoderm, which later on in development are known as the visceral endoderm, may in fact give rise to some somatic cells, specifically the lining of the gastrointestinal tract.</p><p id="p-101">Breaking the symmetry of the blastocyst is a necessary process during development that leads to anterior-posterior (namely head-to-tail) polarity in the embryo. One aspect of mice that sets them apart from model organisms such as Drosophila or zebrafish is that anterior-posterior polarity is not encoded in the egg. Instead, specialized movements within the visceral endoderm break symmetry via a unique collective cell migration within the epithelium. Beddington’s work, and that of her many collaborators, set the stage for this realization.</p></div><div id="sec-38" class="subsection"><h5>Insights on cell movements and the role of extraembryonic tissue during gastrulation:</h5><p id="p-102">Hadjantonakis and her colleagues sought to visualize cells in living mouse embryos in order to reveal the intricate cell behaviors of gastrulation. To accomplish this goal, her team optimized techniques for culturing mouse embryos <em>ex utero</em>; for 3-D, and time-lapse imaging in the embryos; and for studying generational reporters. They also used GFP as a lineage tracer to identify cells as the embryo develops. In the course of this work, Hadjantonakis’ team learned that gut endoderm, though it becomes internal tissue, forms first on the surface on the embryo during gastrulation, which makes its behavior fairly easy to observe.</p><p id="p-103">After the visceral endoderm moves to the anterior of the cup-shaped mouse embryo, the posterior of the embryo begins to form. Then, cells from the epiblast move to the primitive streak (a transient structure which marks the posterior of the epiblast tissue layer), transition, and migrate away as they become mesoderm cells. As gastrulation proceeds, mesoderm cells circumnavigate the space between the two tissue layers, the outer visceral endoderm, and the inner epiblast.</p><p id="p-104">Previously, researchers believed that as epiblast cells came through the anterior region of the primitive streak they became definitive endoderm and in doing so displaced visceral endoderm cells to extraembryonic regions, so that ultimately, there were no visceral endoderm-derived cells in the vicinity of the gut endoderm. But Hadjantonakis’ results suggested otherwise: cells were not displaced. Instead, they stayed put but were dispersed, being incorporated into the gut endoderm as it formed.</p><p id="p-105">Hadjantonakis’ team repeated the experiment with different gene reporters and found this same pattern. An identical distribution of cells also occurred in a subsequent fate mapping experiment, and also when the team generated tetraploid chimeras in which tetraploid cells were labeled with a ubiquitous reporter.</p><p id="p-106">These findings reveal that nascent gut endoderm is made of two different kinds of cells: epiblast-derived definitive endoderm cells and visceral endoderm cells.</p></div><div id="sec-39" class="subsection"><h5>Investigating the lineage between extraembryonic and somatic tissue in adult mammals:</h5><p id="p-107">Hadjantonakis’ group next investigated a link between the extraembryonic tissue (specifically the visceral endoderm) and somatic tissue (specifically the endoderm), which goes on to form the lining of the digestive system in adult mammals.</p><p id="p-108">When the team observed the changes during gastrulation, they saw cells change shape, move from the primitive streak along with the mesoderm, and appear on the surface of the embryo. Eventually, they found that visceral endoderm cells were <em>not</em> dispersed in embryos where mutations affected gastrulation.</p><p id="p-109">To understand this process, the team zeroed in on <em>Sox17</em>, a gene encoding a transcription factor which is expressed in cells destined to become gut endoderm. <em>Sox17</em> is eventually expressed by all cells on the embryo's surface, irrespective of their origin. The researchers were able to conclude that visceral endoderm cells essentially change their identity, becoming the equivalent of epiblast-derived (definitive) endoderm cells, all of which express <em>Sox17</em>.</p></div><div id="sec-40" class="subsection"><h5>The role of Sox17 in cellular identity and endoderm formation:</h5><p id="p-110">Cells originating in the primitive streak that become definitive endoderm travel along the mesoderm and intercalate to the outer layer of an embryo. The group hypothesized that this must be how they acquire apicobasal polarity, and further, that visceral endoderm cells must transiently relax this polarity to allow cells into the epithelium.</p><p id="p-111">Given that gastrulation also propels the formation of two basement membranes from just one, Hadjantonakis’ team further hypothesized that perhaps the mesoderm layer cuts this basement membrane in two, or migrates and creates the second layer (a <em>de novo</em> formation). Preliminary data from their studies points to the latter.</p><p id="p-112">Interestingly, <em>Sox17</em> mutants have a only one basement membrane instead of two, as they have a complete absence of a basement membrane where the mesoderm and endoderm meet. <em>Sox17</em> mutants also show a failure of definitive endoderm specification. These findings suggest that <em>Sox17</em> must be regulating basement membrane gene expression but preliminary findings suggest that transcription regulation may not be the key mechanism involved.</p><p id="p-113">Hadjantonakis said the team’s results are starting to support a new model of gut endoderm construction: Cells exhibiting a propensity to form endoderm traveling through the mesodermal layer gain polarity, intercalate into the visceral endoderm layer, and make a new basement membrane as they do so. Other genes, which encode basement membrane components, are also critical, and the failure to make a basement membrane causes a cessation of gut endoderm development.</p></div><div id="sec-41" class="subsection"><h5>The fate of traveling visceral endoderm cells:</h5><p id="p-114">When Hadjantonakis and her team used time-lapse imaging of embryo growth, they saw that visceral endoderm derivative cells persist and proliferate after endoderm formation is complete. Although these cells are part of the epithelium, they are very active. One mystery is the exact fate of these visceral endoderm cells. Are they merely a scaffold to bring definitive endoderm cells to the surface of the embryo, or do they persist and contribute descendants to gut endoderm tissues such as the respiratory and digestive tracts and their associated organs? The team is currently investigating this question.</p><p id="p-115">Hadjantonakis’ team is attempting to determine whether there are any visceral endoderm derivative cells along the path in the body from the esophagus to the colon (known as the foregut, midgut, and hindgut) at later developmental and adult stages. Results so far suggest that visceral endoderm derivative cells make up ∼20% of gut tube midgestation cells, with most in the hindgut area (which will become the colon and the rectum).</p><p id="p-116">Hadjantonakis’ previous experimental methods reached their technical limits at a 8.75-d-old mouse embryo. To go further, the team has switched to a more technically challenging process called genetic inducible fate mapping. This work is ongoing, but preliminary data shows visceral endoderm derivative cells are present in the liver, pancreas, and intestine even in a 16.5-d-old embryo.</p></div></div></div></div><div class="section other13" id="sec-42"><h2 class="">David Kingsley</h2><div id="sec-43" class="subsection"><h3>Fishing for the secrets of stickleback and human evolution</h3><div id="sec-44" class="subsection"><h4>The regulatory changes behind evolutionary differences:</h4><p id="p-117">David Kingsley (HHMI/Stanford University), presented studies examining evolution in three-spine stickleback fish. Using genetic mapping, Kingsley and his colleagues have identified genes and chromosome regions responsible for a range of evolutionary traits in this species. This research provides strong evidence that regulatory changes in key developmental genes produce classic evolutionary differences in nature. The lessons learned about the genetic mechanisms that underlie the diversity and traits in sticklebacks likely apply to many animals, including humans.</p><p id="p-118">The researchers set out to attempt to answer several questions about evolution:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-119">How many genetic changes are required to produce differences seen in nature?</p></li><li id="list-item-2"><p id="p-120">What types of genes are involved?</p></li><li id="list-item-3"><p id="p-121">What types of mutations occur in those genes?</p></li><li id="list-item-4"><p id="p-122">If evolution has a problem to solve, are there lots of different ways to solve it, or does nature tend to use the same mechanisms over and over again?</p></li></ol><div id="sec-45" class="subsection"><h5>Why study sticklebacks?:</h5><p id="p-123">The three-spine stickleback offers an ideal species for studying evolution because it has undergone one of the most dramatic, recent, and repeated radiations in vertebrates. Marine fish established new freshwater populations at the end of the last ice age. New populations subsequently have had ∼10,000 years to adapt to new water conditions, food sources, predators, and parasites. This has produced huge differences in morphology, physiology, and behavior among many independent freshwater populations. Although many of these fish were originally classified as different species, the reproductive barriers between recently evolved forms can still be overcome using artificial fertilization in the laboratory. The genetic architecture of evolutionary differences can thus be mapped in this system, using large families to find the chromosome regions controlling different traits.</p><p id="p-124">When researchers first became interested in studying the molecular basis of evolution in these fish, most of the necessary genetic and genomic tools did not exist. There were almost no established gene sequences, clone libraries, genetic markers, linkage maps, or transgenic methods for sticklebacks. Over the past 15 years, the team has made tremendous progress in developing the necessary genetic, genomic, and transgenic methods. Today, there are complete physical and genetic maps for sticklebacks, a high-quality reference genome, resequencing of many wild populations, and efficient methods for making transgenic or knockout sticklebacks. By combining these molecular methods with traditional genetic crosses, it is now possible to map the key chromosome regions that control major evolutionary differences, and to identify the genes and mutations that influence a variety of traits.</p></div><div id="sec-46" class="subsection"><h5>A look at limb loss:</h5><p id="p-125">Kingsley first focused on studying limb development and pelvic loss, a classic trait that has evolved repeatedly in mammals, amphibian, reptiles, and fish. Marine sticklebacks have pelvic hind fins while some freshwater sticklebacks have lost these fins. By mapping this trait in the sticklebacks, the researchers found that the well-known developmental transcription factor PITX1 is the major locus controlling two-thirds of the variance in pelvic size. The freshwater fish have conserved the amino acid sequence found in the marine fish but have lost expression of it in the pelvis because they lack a tissue-specific enhancer sequence.</p><p id="p-126">The researchers successfully reversed this evolutionary change by reintroducing the marine pelvic enhancer and the <em>Pitx1</em> gene into eggs from a stickleback population without hind fins (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chan <em>et al.</em> 2010</a>). Introducing a single gene and regulatory sequence was sufficient to bring the pelvic hind fins back to pelvis-less fish. Having confirmed the sequence involved, the researchers compared its evolution in various stickleback populations and found evidence that this trait evolved via independent mutations that deleted this regulatory region.</p><p id="p-127">Taken together, these experiments revealed that a few chromosome regions can have large effects and that the major gene involved in pelvic hind fin loss is a key developmental control gene required for the formation of many different tissues. Even though the <em>Pitx1</em> gene plays essential roles in normal development, regulatory changes within the tissue-specific enhancers surrounding the gene can confine the major effects to a particular place in the body.</p></div><div id="sec-47" class="subsection"><h5>Mechanisms at play in other stickleback traits:</h5><p id="p-128">Using the same genetic mapping approach, the researchers identified chromosome regions and genes that control other stickleback traits. Most freshwater sticklebacks have only a few armor plates, while marine species are covered with armor from head to tail. The researchers found that the <em>EDA</em> gene—which encodes a signaling molecule in the tumor necrosis factor family—was the major locus controlling 75% of variance in plate number between marine and freshwater forms. In humans, mutations in <em>EDA</em> cause human ectodermal dysplasia. In sticklebacks, similar to <em>Pitx1</em>, there is a regulatory change in an enhancer that alters the expression of ectodysplasin along the sides of fish to bring about differences in the number of armor plates (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">O’Brown <em>et al.</em> 2015</a>).</p><p id="p-129">Using crosses between white and black sticklebacks, the researchers identified a single chromosome region that controls 50% of the variance in pigmentation score. The primary locus affecting this trait encodes a famous developmental signal in mammals known as stem cell factor, or KITLG. Mutations in this gene produce white mice that are sterile and have severe anemia. The sticklebacks have regulatory alterations that confine KITLG changes to the developing surface structures (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Miller <em>et al.</em> 2007</a>).</p><p id="p-130">When researchers similarly examined tooth number, armor plate size, and the number and length of spines, they found that in each case, the traits mapped to a few large-effect quantitative trait loci (QTL) or to QTL with small effects. The QTL controlling 20–75% of variance in these traits are all essential signals or transcription factors.</p><p id="p-131">All of the genes identified are also required for normal development. In each case, the researchers found that natural populations have gained or lost enhancers that produce large tissue-specific effects.</p></div><div id="sec-48" class="subsection"><h5>Broader implications for understanding evolution:</h5><p id="p-132">Since the same loci are repeatedly implicated in evolving the same trait in different stickleback populations, the researchers sought to determine whether these mechanisms might extend to other species, including humans. They examined manatees as a mammalian example of pelvic reduction, finding indirect evidence that the <em>Pitx1</em> gene might also be involved in pelvic reduction in these marine mammals. They are now conducting genetic studies of manatees, mice, humans, dolphins, cows, and elephants to better understand the mechanisms involved in pelvic reduction.</p><p id="p-133">Genome-wide association studies (GWAS) of classic human pigmentation traits conducted by other researchers showed that human pigmentation differences are associated with the same <em>KITLG</em> gene seen in stickleback pigmentation studies (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sulem <em>et al.</em> 2007</a>). The human differences also appeared to be regulatory, with the peak association signal mapping more than 300 kb upstream from the <em>KITLG</em> transcription unit. Kingsley’s lab used transgenic mice to functionally screen the blond hair association interval in northern Europeans, finding that the human genome contained a tissue-specific enhancer for hair follicle expression in this region. Blonds have a single base pair change in this enhancer that is not present in brunettes. Kingsley’s group introduced the human blond or brunette enhancer into two matched lines of mice and found that a single base pair change is sufficient to lighten hair color <em>in vivo</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guenther <em>et al.</em> 2014</a>) This work illustrates how eye and hair color can be controlled independently in different parts of the body. For example, altering an enhancer that affects <em>KITLG</em> expression in hair follicles but not in the eyes confines the pigmentation effects to just the hair.</p><p id="p-134">Kingsley noted that although pelvis and pigment traits are very different from each other, they hold important implications for understanding evolution since both traits are regulated by an essential developmental control gene surrounded by large regions of noncoding DNA containing many tissue-specific enhancers. In both cases, enhancers within the noncoding DNA control expression in specific places in the body. Changing enhancers in key developmental genes can produce large effects, but these effects are confined to particular anatomical locations, avoiding the deleterious consequences of protein-coding region mutations in the same genes.</p></div><div id="sec-49" class="subsection"><h5>Looking for patterns:</h5><p id="p-135">In their recent work, Kingsley’s group have moved from studying specific cases of trait evolution to using large-scale sequencing of many different stickleback species to find general patterns involved in evolution. Having seen that the same genes are reused when the same traits evolve in different lakes and streams, they analyzed WGSs of many independent marine and freshwater fish to look for other key genomic regions that are repeatedly selected during parallel evolution. The researchers made windows across the genome, and quantified the difference between marine and freshwater sticklebacks compared to variation within the two types (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jones <em>et al.</em> 2012</a>). Using this approach, they could see recurrent differentiation between the two stickleback forms for several of the loci they had already identified by forward genetic approaches, such as <em>EDA</em>. They also identified 84 regions involved in trait evolution, which they classified as coding for phenotypes, falling between genes, or only showing changes in noncoding DNA. From this, the researchers conclude that the majority of adaptive loci in these repeatedly evolving fish are based on regulatory changes.</p><p id="p-136">Kingsley noted that human geneticists have also identified many regions in the human genome that show signatures of positive selection during recent human history. These genomic regions show very similar patterns to those found underlying stickleback evolution. A fraction of adaptive alleles are based on protein-coding region changes. However, the vast majority of recent adaptive evolution appears to be regulatory in nature in both sticklebacks and humans. Given the similar results across these two very different organisms, similar trends will likely apply to many other complex animals as well.</p></div></div></div></div><div class="section other14" id="sec-50"><h2 class="">Laura Landweber and Richard Miller</h2><div id="sec-51" class="subsection"><h3>Genome rearrangement and organization in Oxytricha</h3><div id="sec-52" class="subsection"><h4>A complex epigenome:</h4><p id="p-137">Principal Investigator Laura Landweber and graduate student Richard Miller (Columbia University and Princeton University) presented research on the evolution of complex genomes, using the scrambled and fragmented genome of the ciliate <em>Oxytricha</em> as a focal point. The team seeks to understand how <em>Oxytricha</em>’<em>s</em> genes become reordered from many dispersed parts, how these genes became scrambled over evolutionary time, and the mechanisms orchestrating the events that lead to genome remodeling.</p><p id="p-138">Ciliates have two types of nuclei: a micronucleus, which is the germline nucleus, and a macronucleus, which is for general cell regulation (somatic nucleus). Even though the micronucleus is smaller, it has a complex genome that contains around a billion base pairs—a size and complexity similar to the human genome.</p><p id="p-139">Some ciliates, including <em>Oxytricha trifallax</em>, take the complex micronuclear genome and whittle it down by eliminating nearly all of the noncoding DNA. The macronuclear genome can be as small as 50,000–75,000 kb, representing a 90–95% DNA loss compared to the micronuclear genome. Coding DNA regions in the macronucleus can be ordered differently compared to the micronucleus, and must be unscrambled during development of the macronucleus.</p><div id="sec-53" class="subsection"><h5>The genomes of the micronucleus and macronucleus:</h5><p id="p-140">The team has gained insights into how genomes are assembled and the epigenetic influences involved in this process by comparing the micronucleus and macronucleus in <em>O. trifallax</em>. They discovered that the organism’s macronucleus includes over 16,000 nanochromosomes, most of which code for a single gene (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Swart <em>et al.</em> 2013</a>). These are flanked by short regulatory sequences and packaged with their own telomeres that mark and protect the sequence ends. There is an average of about 2000 copies of each chromosome and 64 million telomeres per somatic nucleus. All these chromosome copies and telomeres make the macronucleus larger, even though it has a less complex genome.</p><p id="p-141">When the researchers sequenced the germline micronuclear genome, they found more than 225,000 macronuclear-destined sequences (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2014</a>). In times of stress, the cells can mate and exchange haploid micronuclei. The intervening noncoding sequences are then removed and the macronuclear-destined sequences put into the correct order (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) through a template-directed rearrangement process to rebuild a new somatic macronucleus.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relationship between a schematic micronuclear (MIC) gene, scrambled during evolution, and its macronuclear (MAC) form, unscrambled during development. Large regions of nongenic sequence (light blue) are removed during development, such that gene segments (purple) are precisely reassembled. Reprinted from Doak et al. (2003), with permission from Elsevier." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-151034724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The relationship between a schematic micronuclear (MIC) gene, scrambled during evolution, and its macronuclear (MAC) form, unscrambled during development. Large regions of nongenic sequence (light blue) are removed during development, such that gene segments (purple) are precisely reassembled. Reprinted from Doak &lt;em&gt;et al.&lt;/em&gt; (2003), with permission from Elsevier.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3765/F3.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/12/3765/F3.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-142" class="first-child">The relationship between a schematic micronuclear (MIC) gene, scrambled during evolution, and its macronuclear (MAC) form, unscrambled during development. Large regions of nongenic sequence (light blue) are removed during development, such that gene segments (purple) are precisely reassembled. Reprinted from <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Doak <em>et al.</em> (2003</a>), with permission from Elsevier.</p><div class="sb-div caption-clear"></div></div></div><p id="p-143">Some genes’ precursor loci are combined and condensed in the micronucleus. For example, one germline chromosomal region can give rise to five different genes in the macronucleus while sharing the same first four segments in the micronucleus. All of these somatic chromosomes share the same 5′ subtelomeric DNA but have different gene regulation patterns (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2014</a>). There are also short repeat sequences present at the ends (Pointers in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). These microhomologies may provide guides to assist joining the segments and participate in a process that might be analogous to nonhomologous end joining. Since the short repeats are often just 2 bp, they do not contain all the necessary information for this process.</p></div><div id="sec-54" class="subsection"><h5>Rearranging scrambled genes:</h5><p id="p-144">To better understand the process of DNA deletion and reorganization during macronuclear development, the researchers previously examined the time course of programmed DNA rearrangement for a few scrambled genes. In one case, the precursor DNA has segments 3–8 in order, piece 8 is followed by segment 10, then −2 and −1, and finally segment 9. The first step fuses pieces 3–8 and eliminates the noncoding sequences between them. This first step was found to be one of the most error prone steps, leading to further investigations to identify whether there might be an RNA or DNA guided proofreading step involved. Also during the first step, −1 and −2 are fused, simplifying the locus into a four-segment scrambled gene. Then piece 9 can be merged either between piece 8 and 10, or pieces 9, −1, and −2 can be translated to the end of the gene. This puts −1 and −2 in the correct orientation but leaves 9 inverted. Piece 9 then translocates to produce the final, correctly rearranged product (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Möllenbeck <em>et al.</em> 2008</a>).</p></div><div id="sec-55" class="subsection"><h5>Retaining protein-coding sequences:</h5><p id="p-145">Landweber and colleagues also sequenced the genome of the micronucleus and made gene predictions using RNA-sequencing reads at different developmental time points. They found that not only are IESs (noncoding DNA) removed from the genome, but that protein-coding genes are also eliminated from the micronuclear genome on the way to becoming a macronucleus. Based on homology, these genes were predicted to have some conserved domains, including some predicted to be involved in chromatin structure and methyltransferase activities.</p><p id="p-146">The researchers found 810 genes that reside only in the germline and are expressed only during development, suggesting they may be involved with breaking up the germline and turning it into a soma (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2014</a>). The researchers posit that some of the 810 genes may provide unique functions, while others might enhance existing functions. There are interesting parallels in the <em>Ascaris suum</em> roundworm and the lamprey, which both have somatic cell lineages that delete large portions of DNA in their germline cells. This happens at an early stage of embryonic development, and both species have two different genomes within one multicellular organism at later stages.</p></div><div id="sec-56" class="subsection"><h5>Programming novel gene retentions:</h5><p id="p-147">The team developed a method to program novel retentions of germline-limited sequences using Piwi-interacting small RNAs (piRNAs) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fang <em>et al.</em> 2012</a>). This method, which has been useful for creating <em>Oxytricha</em> knockout mutants, involves injecting a small RNA during the early stages of mating to introduce an insertion mutation that leads to gene knockout. This allows testing for the retention of that IES in the next generation.</p><p id="p-148">Using this approach, the researchers investigated whether single-stranded RNA injection could program the retention of a germline-restricted gene. They first tested it with an uncharacterized gene that had no predicted domains but that was validated with mass spectrometry data. They showed at least partial retention of the IES containing the gene. Sequencing data also showed at least one complete retention of the IES plus the germline-restricted gene.</p><p id="p-149">The researchers then investigated whether somatically retained germline genes can be expressed during vegetative growth (misexpression mutation). They compared total RNA from the retention line to that of the wild-type line and found that DNA retention interferes with developmental control of a germline gene.</p><p id="p-150">Overall, these findings suggest that single-stranded RNA injection can be used to program the retention of germline-limited genes that reside on eliminated sequences; retention of these germline-limited genes is heritable to at least the F2 generation; and these retained genes lose their developmental control in subsequent generations and are transcribed outside of their normal developmental window.</p><p id="p-151">The researchers posit that the normal developmental control of this micronuclear (germline) gene is subverted by moving it to the macronucleus. This could be because the IES contains all of the regulatory sequences needed for that gene to turn on during late development, but since it is typically present only in the transcriptionally silent micronucleus it is not turned on at any other time. Evidence suggests that the retained uncharacterized protein is transcribed during vegetative growth, but there are other germline-restricted genes present in IES regions that could be tested with this method.</p></div></div></div></div><div class="section other15" id="sec-57"><h2 class="">Michael Miller</h2><div id="sec-58" class="subsection"><h3>Domestication of C. elegans sperm</h3><div id="sec-59" class="subsection"><h4>Insights on cell signaling:</h4><p id="p-152">Michael Miller (University of Alabama School of Medicine) studies the <em>C. elegans</em> adult gonad for insights into how animal cells coordinate their behavior. The transparent epidermis of <em>C. elegans</em> makes it easy to observe in live worms the events leading to fertilization, including sperm migration, oocyte growth, and meiotic progression, and gonadal muscle contraction. Miller and his team seek to delineate the signaling mechanisms coordinating these processes and discern their evolutionary origins.</p><p id="p-153">In particular, Miller presented research on <em>C. elegans</em> sperm, which he described as among the most successful sperm on the planet. In contrast to human sperm, where millions of sperm are present but very few even locate the oocyte, <em>C. elegans</em> sperm are thought to fertilize oocytes at a one-to-one ratio. These remarkable sperm have a pseudopod composed of protein fibers that allows them to crawl across the embryos, egg shells, and uterine walls to target the spermatheca where fertilization will occur. About 90% of the sperm in wild-type worms find the spermatheca within 1 hr. One contributor to their efficiency is prostaglandin signals that help guide them to the spermatheca. Previous research suggests that prostaglandins, which derive from the oocytes, provide sperm with positional information and stimulate sperm velocity (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sperm guidance in the C. elegans hermaphrodite reproductive tract. Sperm (red) migrate from the vulva (green pseudocolor) around developing embryos to the spermatheca (yellow pseudocolor). Oocytes (blue pseudocolor) are the source of prostaglandin attractants. Males produce sperm that are more efficient at responding to prostaglandins in specific microbial environments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-151034724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sperm guidance in the &lt;em&gt;C. elegans&lt;/em&gt; hermaphrodite reproductive tract. Sperm (red) migrate from the vulva (green pseudocolor) around developing embryos to the spermatheca (yellow pseudocolor). Oocytes (blue pseudocolor) are the source of prostaglandin attractants. Males produce sperm that are more efficient at responding to prostaglandins in specific microbial environments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3765/F4.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/12/3765/F4.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469820" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-154" class="first-child">Sperm guidance in the <em>C. elegans</em> hermaphrodite reproductive tract. Sperm (red) migrate from the vulva (green pseudocolor) around developing embryos to the spermatheca (yellow pseudocolor). Oocytes (blue pseudocolor) are the source of prostaglandin attractants. Males produce sperm that are more efficient at responding to prostaglandins in specific microbial environments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-155">Miller and his colleagues use reverse genetic screens to uncover how these sperm achieve such high efficiency and the role that prostaglandin signaling plays in this sperm guidance process. For the genetic screens, the investigators mate MitoTracker-labeled mutant males with wild-type unstained hermaphrodites and then examine how well the sperm move to the spermatheca. They also use time-lapse imaging to measure velocity and reversal frequency.</p><div id="sec-60" class="subsection"><h5>Prostaglandin signaling:</h5><p id="p-156">To test their hypothesis that <em>C. elegans</em> sperm use G-protein-coupled receptors (GPCRs) to bind to prostaglandins, Miller’s group conducted genetic screens to look for GPCR mutations that perturb the sperms’ ability to find the spermatheca. These experiments pointed to <em>srb</em> chemoreceptors as instrumental in helping sperm respond effectively to prostaglandins. These chemoreceptors are clustered in a 22 kb region of chromosome 2 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> 2006</a>). Single-gene and multiple-gene knockouts of the chemoreceptors showed that <em>srb</em> mutations affect sperm velocity and reversal frequency. They also found that sperm missing multiple <em>srbs</em> showed reduced competitiveness against wild-type sperm. However, mutations in the <em>srb</em> genes did not affect fertilization, spermatid size, or activation. In other words, if the sperm with these mutations found the spermatheca, they did not have problems fertilizing the egg. Overall, sperm with <em>srb</em> mutations behaved like wild-type sperm, except they did not respond as well to guidance cues.</p><p id="p-157">Contrary to their initial assumption that the srb chemoreceptors were expressed in sperm cells, fusing tdTomato to <em>srb</em> genomic loci revealed expression in the ciliated sensory neurons of the male’s nose. Srb mRNA or protein expression was not detected in the gonads. Some of the <em>srb</em> chemoreceptors are expressed only in the nose sensory neurons while others have a broader expression pattern, including in the male tail and in the hermaphrodite vulva. The team’s investigation showed that <em>srb</em> chemoreceptor signaling is necessary, and sufficient, in nose sensory neurons for sperm guidance. These results indicated that the SRB chemoreceptors are likely responding to cues from the external environment, not from prostaglandins in the hermaphrodite reproductive tract.</p></div><div id="sec-61" class="subsection"><h5>Oxygen levels and bacterial foes:</h5><p id="p-158">The team’s next series of experiments sought to reveal more about how the environment affects sperm motility. They began by examining <em>srb-13</em> transcriptome data, which showed a large enrichment for genes involved in immunity and pathogen response. Since this suggested that <em>srb-13</em> might be responding to pathogenic bacteria, the researchers tested a panel of 17 microbes known to coinhabit with <em>C. elegans</em>. They discovered five bacterial species that increase <em>srb-13</em> activity. When these bacteria were grown with the male worms, they produced sperm with poor navigational performance. Males infected with the bacteria that increased <em>srb-13</em> activity were smaller, appeared unhealthy, and mated poorly, indicating that the bacteria were likely pathogenic.</p><p id="p-159">The team investigated neuropeptide genes to better understand how SRB-13 transduces signals from sensory neurons to sperm. They found that mutations in <em>flp-18</em> and <em>flp-21</em> cause sperm guidance defects similar to the ones seen in the <em>srb</em> mutants. FLP-18 and FLP-21 are ligands for the neuropeptide Y receptor NPR-1 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rogers <em>et al.</em> 2003</a>), and <em>npr-1</em> mutants also showed sperm guidance defects. They also demonstrated that <em>flp-21</em> and <em>srb-13</em> act via the same genetic pathway.</p><p id="p-160">Since other researchers had shown that NPR-1 suppresses the hyperoxia sensor GCY-35 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gray <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheung <em>et al.</em> 2004</a>), Miller’s group tested the effects of <em>gcy-35</em> deletion. They found that <em>gcy-35</em> loss suppresses the sperm guidance defects, indicating that <em>srb-13</em> mutant guidance defects are dependent on this hyperoxia sensor. Others have shown that GCY-35/GCY-36 heteromers bind oxygen and that at ambient oxygen concentrations GCY-35 activity is stimulated, promoting behaviors important for pathogen avoidance (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gray <em>et al.</em> 2004</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cheung <em>et al.</em> 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Reddy <em>et al.</em> 2009</a>). Consistent with this data, growing <em>srb</em> mutant males in 10% oxygen suppresses the sperm guidance defects. Taken together, these findings suggest that SRB-13 represses GCY-35 activity.</p><p id="p-161">The researchers believe that wild pathogenic bacteria found in the hypoxic environment inside rotting plant stems trigger SRB-13 chemoreceptor signaling in <em>C. elegans</em> gustatory cilia. SRB-13 signaling represses the GCY-35 hyperoxia circuit. GCY-35, in turn, modulates oxidative metabolism gene expression in the testis, which responds by producing sperm with enhanced attractiveness to prostaglandins. SRB-13 seems to improve sperm motility performance in oxygen-rich environments such as the lab petri dish or on the surface of rotting plant stems. Based on these findings, Miller posits that the worms are trying to sense and avoid these pathogenic bacteria and increase their fertility in surface-like environments, away from the oxygen-depleted pathogen colony.</p></div><div id="sec-62" class="subsection"><h5>Downstream effects in sperm:</h5><p id="p-162">To find out what was happening further downstream in the sperm, the researchers used RNA sequencing data to identify potential sperm genes affected by <em>srb-13</em> mutations. <em>Srb-13</em> mutants exhibited a strong reduction in mRNAs that are involved in oxidative metabolism. Many of these RNAs were mitochondrial, and several of them encode electron transport chain subunits. Performing quantitative PCR on the mitochondrial genes revealed that several, but not all, of the mitochondrial transcripts were significantly reduced in sperm isolated from <em>srb-13</em> mutants. This finding suggests that the sperm guidance defects result in part from reduced sperm mitochondrial RNA stability. Looking closer at these mitochondrial genes, the researchers discovered that a mitochondrial genome deletion affecting mitochondrial RNA levels caused sperm guidance defects similar to those found in the <em>srb-13</em> mutants. Additional <em>srb-13</em> targets encoded in the nuclear genome, such as the mitochondrial fission mediator <em>drp-1</em>, also impact sperm guidance. Thus, SRB signaling appears to affect the sperm’s mitochondria, indirectly improving its ability to respond to guidance cues.</p></div><div id="sec-63" class="subsection"><h5>Domestication of C. elegans sperm:</h5><p id="p-163">Miller and colleagues posit that hermaphrodites are domesticating the <em>C. elegans</em> sperm. Through additional studies, they observed that <em>srb</em> expression is similar in hermaphrodites and males. They also found that <em>srb</em> mutations in hermaphrodites reduce brood size, which they attribute to sperm defects (either decreased sperm production, poor sperm guidance, or both), because the researchers were able to rescue the brood size defects by mating the hermaphrodites with wild-type males.</p><p id="p-164">Further work showed that the N2 Bristol laboratory strain of <em>C. elegans</em> has increased basal SRB signaling independent of pathogens. Others have shown that N2 Bristol has acquired mutations in multiple genes, including <em>npr-1</em>, that repress the hyperoxia-sensing circuitry (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de Bono <em>et al.</em> 1998</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Rogers <em>et al.</em> 2003</a>). Like N2 Bristol, several wild isolates collected from across the world have excellent sperm performance in the lab. On the other hand, wild isolates from Hawaii and California show poor sperm performance, suggesting that SRB activity is polymorphic among natural populations. Excellent sperm performance correlates with the presence of large chromosomal regions thought to have swept through the global population roughly 50–200 yr ago (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>). The researchers speculate that this sweep may have included mutations that increase basal SRB activity, thereby improving sperm performance in oxygen-rich environments. In this context, hermaphrodites may be shaping sperm performance characteristics important for fertility.</p></div></div></div></div><div class="section other16" id="sec-64"><h2 class="">Molly Przeworski</h2><div id="sec-65" class="subsection"><h3>The evolution of meiotic recombination</h3><div id="sec-66" class="subsection"><h4>Examining the role of PRDM9:</h4><p id="p-165">While evolution has produced clearly observable differences between species, there is equally complex diversity among individuals within a species. These heritable differences are influenced by drift, mutation, and recombination, in addition to natural selection. Understanding these factors provides a greater appreciation of and understanding for the processes underlying evolutionary changes.</p><p id="p-166">One such factor, known as meiotic recombination—the DNA shuffling that occurs during cell division—is the research focus of Molly Przeworski (Columbia University).</p><div id="sec-67" class="subsection"><h5>Key questions about meiotic recombination:</h5><p id="p-167">Meiotic recombination and factors that influence it have drawn attention from researchers across biological fields in recent years. These mechanisms have attracted long-standing interest from molecular biologists because they play fundamental roles in meiosis. They also have important biomedical implications. For instance, proper chromosomal segmentation is necessary to avoid pregnancy complications such as aneuploidy, which leads to spontaneous miscarriage or severe developmental disabilities.</p><p id="p-168">In another scientific community, evolutionary biologists study meiotic recombination for its role in creating new combinations of alleles on which natural selection can act. Some regions of the human genome have greater recombination rates, known as hotspots. Higher recombination rates lead to a more rapid generation of beneficial allele combinations, resulting in more efficient natural selection. Evolutionary biologists have sought to find out if there are differences in hotspots among individuals, and how these differences might affect evolutionary change in recombination profiles.</p><p id="p-169">Przeworski’s research integrates key underlying questions across molecular and evolutionary biology. Her lab has sought to understand how recombination works across different species, why it operates differently in different species, the evolutionary implications of how recombination is regulated, and how natural selection shapes recombination mechanisms to lead to diverse recombination patterns.</p></div><div id="sec-68" class="subsection"><h5>A method for identifying patterns of genetic variants:</h5><p id="p-170">Fine-scale meiotic recombination was originally studied through sperm typing, but, among other limitations, such methods cannot be scaled to the whole genome. Przeworski instead used a method for studying genetic recombination that looks at patterns of the genetic variants in a sample that result from population processes and recombination events from previous generations.</p><p id="p-171">Through this method, researchers can obtain time-averaged estimates of recombination for the ancestors of the sample, including both males and females. In effect, the approach allows researchers to determine what recombination rates in the past are likely to produce the specific patterns of genetic variation seen today. The method is also practical, facilitating a detailed look at the evolution of recombination without requiring cross-breeding of species in the laboratory; a set of individuals just needs to be resequenced.</p></div><div id="sec-69" class="subsection"><h5>Tracing the factors that influence recombination:</h5><p id="p-172">Applying this method to genetic variation data for the last decade, researchers have achieved a fine-scale resolution and identified the position of more than 30,000 hotspots along the human genome.</p><p id="p-173">In 2004, Przeworski and her colleagues used the method to compare humans to our closest evolutionary relatives, chimpanzees, by determining the locations of hotspots for each species (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ptak <em>et al.</em> 2004</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2005</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Auton <em>et al.</em> 2012</a>) and comparing the two sets. Contrary to an initial belief that there would be similarities in hotspot locations between such closely-linked species, Przeworski’s analysis showed the locations of hotspots were in fact extremely dissimilar between humans and chimps, essentially independently distributed along the genomes of the two species. These results suggested that some or many mechanisms of recombination differ between humans and chimpanzees.</p><p id="p-174">Given this foundational understanding that genetic recombination hotspots vary between species, Przeworski shifted her focus to potential individual differences in hotspots within one species. In a study of human genomes, Przeworski found that different individuals show different sets of recombination hotspots (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Coop <em>et al.</em> 2008</a>). Studying this phenomenon using a pedigree analysis, Przeworski concluded that this trait variation was heritable. Heritable variation for a trait allows for it to be mapped, which in this case revealed how hotspots are specified in humans and mammals generally.</p></div><div id="sec-70" class="subsection"><h5>The role of PRDM9:</h5><p id="p-175">In 2010, three independent studies were published that identified a specific gene, <em>Prdm9</em>, as responsible for specifying the location of recombination hotspots in primates and mice. PRDM9 has three domains, the last of which is a zinc finger that specifies DNA binding and another of which is a SET domain that trimethylates the histone H3K4. Researchers found that this gene is able to specify the location of recombination hotspots by binding DNA, trimethylating H3K4, and eventually recruiting SPO11, a protein that plays a role in double-strand breaks on the genome that, when repaired, result in recombinant products.</p><p id="p-176">When a break occurs in one of two homologs, the allele that breaks receives information from the other allele in the repair process. Certain alleles are more or less likely to recruit PRDM9 (termed “hot” and “cold,” respectively). The hot alleles will experience more frequent breaks on the genome; because the break in the hot allele will be repaired by the cold allele, the cold allele will be overtransmitted and the hot allele will cease to exist at the population level.</p><p id="p-177">This process behind recombination in which hotspots can be lost presents an interesting conundrum: If hotspots lead to their own extinction, then why do they exist in nature? One possible explanation exists in the binding domain of PRDM9. Specifically, if the zinc finger of PRDM9 mutates, it could restore a new set of hotspots.</p></div><div id="sec-71" class="subsection"><h5>Recombination with and without PRDM9:</h5><p id="p-178">Przeworski’s research takes advantage of naturally occurring populations that lack PRDM9 to further understand the influence it has on recombination locations. Specifically, she works with four types of birds: the double-barred finch, zebra finch, and two varieties of long-tailed finches. The extent of genetic variation in these PRDM9-lacking species is similar to the variation seen in humans, chimps, and gorillas, making these birds a useful comparison for hotspot evolution. In the bird species, Przeworski found recombination was concentrated at or near transcription start sites, a direct contrast to patterns seen in mice, where PRDM9 was found to direct recombination to take place away from transcription start sites.</p><p id="p-179">Przeworski’s team also compared hotspot locations between the species of birds. If the placements of hotspots were determined by chance, about three percent of hotspots would be shared across species. But in contrast to what she observed in chimpanzees and humans, which showed independent distributions of hotspots, the zebra and long-tailed finch species shared more than 70% of their hotspots.</p><p id="p-180">When recombination is directed by PRDM9, hotspots are lost through overtransmission of cold alleles, or are lost or changed by alterations to the binding specificity identified in the zinc finger of PRDM9. Research in birds lacking PRDM9 reveals that there is extensive conservation of hotspots in the absence of PRDM9. In these species, hot alleles cannot be easily lost because they have other responsibilities, including serving as transcription initiation sites.</p><p id="p-181">While PRDM9 is believed to direct recombination, little is known about how it acquired this role. Mammals benefit from PRDM9, yet the fact that there are species without the gene shows it is not essential to recombination. In ongoing work, Przeworski and her colleagues are examining potential explanations for this, including possible indirect benefits stemming from PRDM9’s role in moving recombination away from transcription start sites and the possibility that PRDM9 may serve as a defense against selfish genetic elements.</p></div></div></div></div><div class="section other17" id="sec-72"><h2 class="">Pamela Ronald</h2><div id="sec-73" class="subsection"><h3>Tomorrow’s table: organic farming, genetics, and the future of food</h3><div id="sec-74" class="subsection"><h4>Case studies in genetically-engineered crops:</h4><p id="p-182">Advancing food security for a growing world population and increasing the sustainability of agricultural practices are among the world’s most urgent goals. Pamela Ronald (University of California, Davis) shared her laboratory’s work on engineering rice for disease resistance and flood tolerance, and offered a perspective on plant genetics and engineered crops more broadly.</p><div id="sec-75" class="subsection"><h5>Disease resistance in rice:</h5><p id="p-183">Rice is a staple food for more than half of the world’s population, but every year ∼40% of the global crop is lost to the ravages of pests and disease. Conventional breeding practices have yielded disease-resistant strains of rice that are now commonly grown and eaten, but the exact mechanisms of disease resistance in these plants were not well understood until the 1990s.</p><p id="p-184">Ronald and her team investigated the genetics of both <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> (“Xoo”) and its host, rice, to isolate genes for resistance. Their experiments built on research by Gurdev Khush and colleagues at the International Rice Research Institute (IRRI), who for years collected wild rice species with agronomically useful phenotypes. One in particular, a strain from Mali, was found to be resistant to virtually all strains of <em>Xoo</em>. In 1995, Ronald and her team isolated the gene responsible for this resistance, <em>Xa21</em>, and discovered that it encodes a receptor kinase bearing a leucine rich repeat (LRR) motif (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Song <em>et al.</em> 1995</a>). Subsequent studies revealed a remarkably similar set of receptors across many organisms, including in flies, mice, and <em>Arabidopsis</em>.</p><p id="p-185">Having isolated the <em>Xa21</em> gene and the LRR-containing kinase, Ronald and her team next tackled identifying the microbial molecule that activates this rice receptor. This eventually led to the discovery of <em>Xanthomonas raxX</em>. Ronald's lab demonstrated that mutations in several genes in an <em>Xoo</em> operon—<em>raxST</em>, <em>raxA</em>, and <em>raxB</em>—disarmed the microbe's ability to activate the plant’s immune response. They then identified a small open reading frame, called RaxX, just upstream of the RaxSTAB operon. RaxX encodes a small protein with a conserved tyrosine, Y41, which is sulfated by raxST. RaxST is the first example of a bacterial tyrosine sulfotransferase (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Han et al. 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pruitt et al. 2015</a>), previously thought to be present only in eukaryotic species.</p><p id="p-186">Ronald noted that tyrosine sulfation is essential for modulating receptor-ligand interactions in many species. In an example well-known to plant biologists, sulfation of the <em>Sinorhizobium meliloti</em> Nod factor is required for nodulation of alfalfa. Sulfation is also required for the CCR5 coreceptor to recognize and resist gp120, a subunit of HIV (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gardner <em>et al.</em> 2015</a>). Ronald pointed out that a portion of the human population has a nonfunctional allele of CCR5, which makes these individuals immune to HIV.</p><p id="p-187">Ronald also showed that a synthetic sulfated RaxX peptide can activate XA21-mediated immunity, and that <em>Xoo</em> strains that can overcome <em>Xa21</em> have mutations in <em>RaxX</em>. Ronald's current focus is on determining if sulfated RaxX is secreted through the RaxA/RaxB type 1 secretion system.</p></div><div id="sec-76" class="subsection"><h5>Flood resistance in rice:</h5><p id="p-188">Floods are another significant danger for the world’s rice crop. Rice grows well in standing water, but if the plant is submerged for more than three days, almost all varieties will die. In places like Bangladesh, where two-thirds of the population’s average daily calories come from rice, sustained flooding could cause widespread famine. Scientists at IRRI had discovered an ancient rice strain capable of surviving up to two weeks of flooding, but conventional breeding methods were not able to introduce this trait into other varieties without undesirable side effects.</p><p id="p-189">Ronald and collaborators David Mackill and Kenong Xu sought to isolate the gene responsible for flood resistance. Using a map-based cloning strategy, Ronald’s team identified three ethylene response transcription factor (ERF) genes that had been linked, in other organisms, with tolerance to environmental stress. One gene, <em>sub1A</em>, was both rapidly upregulated under stress conditions and present only in the ancient flood-tolerant varieties. The team inserted <em>sub1A</em> into the rice genome using both genetic engineering and marker assisted breeding (MAS). When submerged, the Sub1 rice varieties yielded three times more grain than conventional varieties.</p><p id="p-190">The new Sub1 variety, developed at IRRI using MAS, was planted by an estimated 5 million farmers in 2015. As the climate changes, many parts of the world are experiencing greater precipitation and severe weather, contributing to increased flooding and posing a substantial danger to the world’s rice crop. Sub1 rice has helped ensure a reliable supply of this essential food under increasingly volatile conditions.</p></div><div id="sec-77" class="subsection"><h5>Social perceptions and the value of genetically engineered crops:</h5><p id="p-191">Ronald noted that her work, which involves introducing rice genes into rice plants, is generally more palatable for consumers than other forms of genetic engineering, such as introducing genes from viruses or bacteria into food crops. However, she noted that sometimes this type of genetic engineering is the most effective approach to enhancing food security and advancing sustainable agriculture.</p><p id="p-192">Ronald cited papaya as one example. More than 25 years ago, the papaya ringspot virus, which could not be controlled with any conventional farming methods, caused farmers to lose almost all of their papaya crop. As the virus spread and farmers grew more desperate, a research team led by local Hawaiian Dennis Gonsalves turned to then-new genetic engineering techniques to help.</p><p id="p-193">The researchers introduced a snippet from the genome of a mild virus strain into the papaya’s genome. Field trials of the new crop yielded 20 times more papaya compared to conventional plants. Two decades later, there is still no conventional or organic treatment that can control papaya ringspot virus. As a result, 80–90% of the papaya grown in Hawaii have been genetically engineered.</p><p id="p-194">Genetic engineering techniques are also saving the eggplant crop in Bangladesh, where caterpillar infestations can ruin this staple. Insecticides are effective but highly toxic, especially for farmers who don't have proper safety gear. Geneticists took advantage of an organic approach—the use of Bt, a bacterial protein that can attack the insect—and snipped the Bt gene from the bacteria and inserted it into the eggplant genome. Field tests in Bangladesh show that eggplant farmers are using far less insecticide, and sometimes none at all.</p><p id="p-195">Often consumers’ main concern is whether genetically engineered food is safe to eat. However, every major scientific society worldwide has concluded that existing genetically modified crops are safe to eat (National Academies of Sciences, Engineering, and Medicine 2016). In more than four decades of research in many different organisms, there has never been a single case of harm to humans or ecosystems caused by the process of genetic engineering.</p><p id="p-196">There is a clear and urgent need for sustainable agricultural practices and the technologies to support them. Ronald emphasized her view that cutting-edge techniques combined with ecological farming practices are central to addressing agriculture’s most pressing social, economic, and environmental questions.</p><p id="p-197">Beyond laboratory work, Ronald suggested that scientists have another important role to play: to explain these technologies and the opportunities they offer, so that society understands their value in helping to feed the world and reducing harm to the environment.</p></div></div></div></div><div class="section other18" id="sec-78"><h2 class="">Amita Sehgal</h2><div id="sec-79" class="subsection"><h3>Mechanisms, functions, and regulation of sleep cycles</h3><div id="sec-80" class="subsection"><h4>Findings from a Drosophila model:</h4><p id="p-198">Sleep is vital throughout the animal kingdom. Humans, for example, spend about one-third of our lives asleep, and insufficient or disordered sleep can have a huge impact on our health. Yet we lack a clear understanding of why sleep is necessary, the mechanisms behind it, and how it is regulated.</p><p id="p-199">Amita Sehgal (University of Pennsylvania), studies circadian rhythms and other factors that drive rhythmic behaviors such as sleep. About 15 years ago, Sehgal developed a model for sleep using the fruit fly (<em>D. melanogaster</em>). In the team’s behavioral assay, individual flies in a glass tube are monitored constantly using an infrared beam; computers record this data and identify 5 min or more of immobility as time spent asleep. Using this technique, Sehgal and her colleagues have focused on deciphering the molecular and cellular underpinnings of how sleep is restricted to a certain time of day, the function of sleep, and the regulation of sleep onset and maintenance.</p><div id="sec-81" class="subsection"><h5>Mechanisms supporting sleep cycles:</h5><p id="p-200">Establishing regular rhythms requires a complicated circuit of neurons and peptides. Fly brains have six groups of so-called “clock” cells, and of these, a group known as small lateral ventral neurons (s-LNv) is critical for driving the rhythm of sleep and wake in flies. These clock cells extend to the dorsal part of the fly brain and produce a peptide called Pigment Dispersing Factor, which is also important for maintaining rhythms.</p><p id="p-201">Building on this knowledge, Daniel Cavanaugh, a postdoctoral researcher working in Sehgal’s lab, was interested in examining which neurons in the fly brain other than the clock cells might play a role in determining rhythms. Cavanaugh’s results showed that the Pars Intercerebralis (PI), a group of neurons in the dorsal part of the fly brain, was required for maintaining rhythms (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavanaugh <em>et al.</em> 2014</a>). A polysynaptic circuit extends from central clock cells through other neurons in the brain to the PI, which enables daily timing.</p><p id="p-202">In flies, the PI contains insulin-producing cells and also other peptidergic neurons, making it a neuroendocrine structure similar to the mammalian hypothalamus. Profiling the PI to determine other peptides that may be integral to driving sleep–wake cycles through communicating time of day signals, the team found that one such peptide is DH44, the <em>Drosophila</em> equivalent of corticotrophin-releasing factor (CRF), which is involved in the human stress response.</p></div><div id="sec-82" class="subsection"><h5>Sleep functions:</h5><p id="p-203">Little research had been done on why sleep is required, particularly in early life. Matthew Kayser, also a postdoctoral fellow in the Sehgal laboratory, has used the fruit fly model to study age-related differences in sleep. Kayser’s results supported the idea that young flies sleep more and fall asleep faster than their older counterparts (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kayser <em>et al.</em> 2014</a>). The team also found that the mechanism behind this difference is dopamine. Dopamine is a critical chemical for many functions throughout the body, and one of its key effects is to produce arousal (in flies as well as mammals). Noting that older flies generally exhibit a higher level of dopamine in the brain, Kayser posits that this increased level of dopamine explains the greater wakefulness and reduced sleep seen in older <em>vs.</em> younger flies.</p><p id="p-204">In addition to having lower levels of dopamine generally, the activity patterns of certain areas of the brain seem to promote sleep in younger flies. For example, the dorsal fanshaped body (dFSB), a region of the fly brain important for promoting sleep, is inhibited by dopaminergic inputs. In the case of young flies, the dopamine-transmitting neurons projecting into the dFSB are less active and thus promote greater amounts of sleep in the younger population compared to the older flies.</p><p id="p-205">This age-related difference in sleep may be explained by the role sleep plays in development and behavior. In particular, sleep deprivation in young flies reduces mating when the flies are older. The research team found that experimentally stimulating dopamine neurons reduces dFSB activity and results in reduced sleep. This sleep loss in young flies reduces adult courtship, measured by the length of time male flies spend courting females, and decreases the percentage of males that follow through from courtship to copulation.</p><p id="p-206">In an effort to study the specific circuits involved in this altered behavior, Sehgal examined the olfactory system, or sense of smell. Flies use smells and pheromones to orient to each other, giving olfaction a large role in the mating ritual. The processing of smell starts with sensory input to the antenna. The signal transfers through synapses in olfactory glomeruli in the antenna lobe before being processed by other structures.</p><p id="p-207">After depriving both young and old flies of sleep by stimulating dopamine neurons, Sehgal’s group examined the size of the glomeruli. One particular glomerulus, VA1v, appeared to be sensitive to sleep loss in younger—but not older—flies. Unlike other areas, the VA1v glomerulus grows after the young flies have hatched. The strong effect of sleep deprivation on VA1v glomerulus size suggests that parts of the brain that grow posthatching are the areas that require the most sleep to properly develop, especially in the context of courtship circuitry.</p></div><div id="sec-83" class="subsection"><h5>Regulating sleep onset and maintenance:</h5><p id="p-208">Given sleep’s integral role in courtship behaviors, Sehgal sought to understand the mechanisms that regulate sleep. In this vein, she examined the neurotransmitters implicated in sleep. There is a great deal of overlap in the neurotransmitters that are involved in sleep in humans and flies, including dopamine, norepinephrine, serotonin, histamine, and γ-aminobutyric acid (GABA). In addition, sleep and insomnia-related neurological conditions in humans have been related to voltage-gated potassium channels and nicotinic acetylcholine receptor, both of which are also implicated in Sehgal’s <em>Drosophila</em> research.</p><p id="p-209">Throughout their work, Sehgal and her colleagues have found unbiased approaches to studying animal behaviors, including forward genetics, to be extremely useful methods. <em>Drosophila</em> is a particularly well-suited model for forward genetics. In fact, the first circadian genes identified in <em>Drosophila</em> were found through this type of experimentation. Forward genetics in <em>Drosophila</em> is particularly effective for investigating sleep because sleep phenotypes are easily modified and in flies there is less redundancy and compensation, which enables researchers to isolate more robust mutant phenotypes. Using such unbiased approaches, Sehgal’s team was able to isolate a “sleepless” mutant fly, which sleeps 80% less than flies without the mutation. The mutant fly has been extremely useful in studying the effects of low sleep.</p><p id="p-210">Sehgal’s team used these techniques to search for genes dedicated to sleep homeostasis. By conducting loss-of-function screens for mutants affecting daily sleep amount, Sehgal’s team found several genes that they determined were permissive for sleep, but not instructive, meaning they will allow but not induce sleep. Thus, they conducted an over-expression screen. Out of a screen of more than 12,000 genes, just one was implicated in inducing long sleep. Based on these findings, Sehgal believes single molecules that induce sleep will be rare.</p><p id="p-211">Sehgal’s research using a <em>Dropsophila</em> model has shed light on many of the mysteries of sleep. Venturing into the neural circuitry behind the daily rhythms supporting sleep, her team was able to trace the connections among central clock cells and the PI and identify peptides such as DH44 that are required for rhythmic behavior. Seeking to examine the functions of sleep, the lab revealed the important role of sleep in developing the neural foundation for fruit fly courtship behaviors. Finally, using unbiased approaches including forward genetics to examine regulation of sleep onset and maintenance, the team found a gene, <em>nemuri</em>, that appears to induce sleep. Such studies advance understanding of the role of sleep across the animal kingdom and could perhaps improve our ability to prevent and treat sleep-related health disorders.</p></div></div></div></div><div class="section other19" id="sec-84"><h2 class="">Leonard Zon</h2><div id="sec-85" class="subsection"><h3>Translating zebrafish development to the clinic</h3><div id="sec-86" class="subsection"><h4>Findings on blood stem cells and hematopoiesis:</h4><p id="p-212">Leonard Zon (HHMI/Harvard University/Boston Children’s Hospital) uses zebrafish as a model system to understand how new blood stem cells are formed and how they make new blood cells through hematopoiesis. Since hematopoiesis in zebrafish and humans is a very similar process, zebrafish are ideal for studying where new blood stem cells come from and how they create the billions of new blood cells a body needs daily. This work has led to some promising potential drug targets and treatments for leukemia, melanoma, and other diseases.</p><p id="p-213">Blood stem cells can either self-renew or differentiate into all of the lineages of the peripheral blood cells. They are the only stem cells for which a quantitative assay is available, and are the only stem cells, besides skin stem cells, that are currently used therapeutically. By mapping the fate of new cells in zebrafish, Zon’s group has identified many of the steps involved in the formation and migration of new blood stem cells. They are also trying to better understand the biochemistry and dynamics involved in each of these steps.</p><div id="sec-87" class="subsection"><h5>Watching stem cell colonization:</h5><p id="p-214">The researchers are particularly interested in blood stem cell colonization because understanding this key but poorly understood process could provide insight into how to improve the success rates for bone marrow transplants. To watch stem cell colonization taking place, the researchers created a transgenic zebrafish with fluorescently marked stem cells, revealing the birth of the stem cells in the aorta and movement of the stem cells through the blood circulation before their landing in the caudal hematopoietic tissue. There, endothelial cells surround the stem cells and stem cell division takes place. Finally, one stem cell leaves the stem cell niche while the other stays (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tamplin <em>et al.</em> 2015</a>).</p><p id="p-215">To better define the stem cell niche, the researchers conducted correlative electron microscopy on the same transgenic zebrafish. This work produced the highest resolution picture available for a stem cell niche in a vertebrate species. It showed the stem cell surrounded by a pocket of endothelial cells, which may allow a higher concentration of growth factors or offer a protective space for the stem cell. Surprisingly, the stem cell was physically attached to a stromal cell (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Using transgenic fish with labeled stromal and stem cells, Zon and colleagues found that the stromal cells help orient the plane of stem cell division, with most divisions occurring perpendicular to the stromal cell plane. Once the stem cell emerges through the endothelial wall into its niche, a macrophage begins interacting with the stem cell. They also saw that a second macrophage is responsible for escorting the stem cell to the stromal cell.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="High-Resolution Electron Microscopy of Endogenous Hematopoietic Stem and Progenitor Cell (HSPC) in the Perivascular Niche. Lodged HSPC (green), surrounding EC (electron capture) nuclei (blue, numbered), and stromal cells (pink and purple). Reprinted from Tamplin et al. (2015), with permission from Elsevier." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-151034724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;High-Resolution Electron Microscopy of Endogenous Hematopoietic Stem and Progenitor Cell (HSPC) in the Perivascular Niche. Lodged HSPC (green), surrounding EC (electron capture) nuclei (blue, numbered), and stromal cells (pink and purple). Reprinted from Tamplin &lt;em&gt;et al.&lt;/em&gt; (2015), with permission from Elsevier.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3765/F5.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/12/3765/F5.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3765/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469827" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-216" class="first-child">High-Resolution Electron Microscopy of Endogenous Hematopoietic Stem and Progenitor Cell (HSPC) in the Perivascular Niche. Lodged HSPC (green), surrounding EC (electron capture) nuclei (blue, numbered), and stromal cells (pink and purple). Reprinted from <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Tamplin <em>et al.</em> (2015</a>), with permission from Elsevier.</p><div class="sb-div caption-clear"></div></div></div><p id="p-217">To determine how many stem cells are born in the aorta, “brainbow” fish were mated to a blood-specific CreER<sup>t2</sup> line and Tamoxifen added at multiple times during development to produce multiply-labeled embryonic blood. By adulthood, all the embryonic blood is replaced with stem cell-derived blood. Although Zon’s group thought that counting the number of colors would let them back-calculate the number of stem cells born in the aorta, they realized they would need controls showing what each color looks like alone. Thus, they decided to use a ubiquitous Cre label to make every sperm cell a different color, yielding zebrafish embryos that are one of many colors. Fluorescence-activated cell sorting of the fish embryos revealed that 21 stem cells are born in the dorsal aorta, which suggests that a pool of stem cells is amplified in each stem cell niche.</p><p id="p-218">The researchers are also using these multi-color zebrafish to study a disorder known as clonal hematopoiesis of indeterminate potential (CHIP), which predisposes people to leukemia. With increasing age, the percentage of people with mutations in genes that cause clonal expansion increases, but scientists do not know why this causes a predisposition to leukemia. The researchers injected CHIP disease-causing genes or knocked out these genes to see if they could make the multiclonal (multicolor) fish monoclonal (one color). This work revealed that mutations in the <em>Asxl-1</em> gene are strongly tied to clonal expansion, enabling a search for molecules that might suppress the clonal expansion. They also found that different combinations of mutations result in different syndromes.</p></div><div id="sec-88" class="subsection"><h5>Improving leukemia treatment:</h5><p id="p-219">Bone marrow transplants are used today as a treatment for leukemia, but about 25% of patients die from the procedure. The patient first receives chemotherapy to erase the leukemia and then receives blood stem cells, typically from a relative, to rebuild the immune system. The transplanted stem cells make their way to the bone marrow where engraftment takes place.</p><p id="p-220">The researchers used zebrafish to find molecules that would increase the number of blood stem cells that would go on to produce blood cells after a transplant. One such molecule was also demonstrated to be successful in mice and is now in phase II clinical trials. The clinical trials have been promising so far. In addition to showing an increase in the number of stem cells, the patients receiving the treatment also experienced lower rates of graft <em>vs.</em> host disease, as well as less viral reactivation after the transplant (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cutler <em>et al.</em> 2013</a>). Meanwhile, Zon’s team is continuing to use zebrafish to study the nuances of how bone marrow engraftment occurs to further improve treatments. Some of the zebrafish experiments are revealing that inflammatory lipids are important in how blood stem cells traffic (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2015</a>).</p></div><div id="sec-89" class="subsection"><h5>Finding treatments for other diseases:</h5><p id="p-221">Zon and colleagues are also using zebrafish to look for new treatments for rare ribosomopathies, which underlie several rare genetic diseases. Ribosomopathies result from defects in ribosome biogenesis that activate P53 and lead to cell death. They used a zebrafish line with a <em>rps29</em> mutation, which causes a hemoglobin defect, to screen for potentially therapeutic molecules. Patients with this ribosome mutation have a disease called Diamond-Blackfan anemia, in which the bone marrow fails to make red blood cells. Very few therapy options exist for these patients except for transplantation. The zebrafish screen showed that the calmodulin inhibitor TFP, an antipsychotic in current use, completely reversed the effects of this mutation in zebrafish. Experiments with the drug in mouse and <em>in vitro</em> human models of Diamond-Blackfan anemia were also successful, and a clinical trial is slated to begin in 2017.</p><p id="p-222">Zon is also working to understanding melanoma and to find potential treatments for this skin cancer using zebrafish. By modeling melanoma in the zebrafish, they have found that a gene called <em>crestin</em> marks all the migrating neural crest cells in zebrafish embryos. In adult fish this gene is typically repressed, but when melanoma is present, <em>crestin</em> is expressed. GFP-tagged crestin labeled melanoma cells and allowed their detection before clinical melanoma was obvious in the fish. This is likely the first time that cancer has been observed in a vertebrate at the single-cell stage. <em>Crestin</em> activated in human melanoma was similar to that observed in the zebrafish preclinical patches. Crestin reactivation could provide a way to identify precancerous moles or offer a therapeutic target for repression of genes involved in melanoma formation and progression. Further studies showed that SATB2 overexpression leads to cranial neural crest development and cell migration, initiating cell migration programs and ultimately metastasis. Chemical screening revealed that the drug leflunomide can block neural crest development. This drug is now in clinical trials.</p><p id="p-223">To see if zebrafish could be used as a clinical translational model, Zon’s team developed an oral gavage procedure to direct drugs into the stomach of the zebrafish. This method offers an inexpensive way to test drugs, especially when compared to xenograft studies, which are valuable but expensive. Doing fish studies before moving to xenograft studies could be a useful and money-saving combination approach.</p></div></div></div></div><div class="section other20" id="sec-90"><h2 class="">Honors and Awards</h2><div id="sec-91" class="subsection"><h3>GSA Awards</h3><div id="sec-92" class="subsection"><h4>Thomas Hunt Morgan Medal</h4><p id="p-224">Nancy Kleckner (Harvard University)</p></div><div id="sec-93" class="subsection"><h4>Genetics Society of America Medal</h4><p id="p-225">Detlef Weigel (Max Plank Institute for Developmental Biology)</p></div><div id="sec-94" class="subsection"><h4>George W. Beadle Award</h4><p id="p-226">Susan E. Celniker (Lawrence Berkeley National Laboratory)</p></div><div id="sec-95" class="subsection"><h4>Elizabeth W. Jones Award for Excellence in Education</h4><p id="p-227">William Wood (University of Colorado)</p></div><div id="sec-96" class="subsection"><h4>The Novitski Prize</h4><p id="p-228">Leonid Kruglyak (HHMI and University of California, Los Angeles)</p></div><div id="sec-97" class="subsection"><h4>The Rosalind Franklin Young Investigator Award</h4><p id="p-229">Maria Barna (Stanford University)</p><p id="p-230">Carolyn McBride (Princeton University)</p></div></div><div id="sec-98" class="subsection"><h3>GSA Undergraduate Travel Awards</h3><p id="p-231">Cynthia Becker (Ithaca College).</p><p id="p-232">Diana Cardero (Florida International University).</p><p id="p-233">Brian Choi (Indiana University).</p><p id="p-234">Logan Condon (University of Washington).</p><p id="p-235">Max Haase (University of Wisconsin–Madison).</p><p id="p-236">Mathieu Henault (Laval University).</p><p id="p-237">Alex Lederer (University of Pittsburgh).</p><p id="p-238">Xiaoxue Lin (College of William and Mary).</p><p id="p-239">Maegan Neilson (College of the Holy Cross).</p><p id="p-240">Erin Ritchie (University of Utah).</p><p id="p-241">Carolyn Turcotte (Marist College).</p></div><div id="sec-99" class="subsection"><h3>C. elegans Development, Cell Biology, and Gene Expression Meeting</h3><div id="sec-100" class="subsection"><h4>Poster Awards:</h4><div id="sec-101" class="subsection"><h5>Graduate students:</h5><p id="p-242">1st Place: Kimberly Gauthier (MUHC Research Institute, McGill University).</p><p id="p-243">2nd Place: Cristina Matthewman (University of Miami Miller College of Medicine).</p><p id="p-244">3rd Place: Amel Alqadah (University of Illinois at Chicago).</p></div><div id="sec-102" class="subsection"><h5>Undergraduate students:</h5><p id="p-245">1st Place: James Brandt (Lewis &amp; Clark College).</p><p id="p-246">2nd Place<em>:</em> Maegan Neilson and Lauren Riley (College of the Holy Cross).</p></div></div></div><div id="sec-103" class="subsection"><h3>Ciliate Molecular Biology Conference</h3><div id="sec-104" class="subsection"><h4>Poster Awards:</h4><div id="sec-105" class="subsection"><h5>Graduate student:</h5><p id="p-247">Miguel Gonzales (Texas A&amp;M University).</p></div><div id="sec-106" class="subsection"><h5>Undergraduate student:</h5><p id="p-248">Evan Wilson (Missouri State University).</p></div></div></div><div id="sec-107" class="subsection"><h3>57th Annual Drosophila Research Conference</h3><div id="sec-108" class="subsection"><h4>Larry Sandler Memorial Lecture:</h4><p id="p-249">Alejandra Figueroa-Clarevega (University of California, Berkeley).</p></div><div id="sec-109" class="subsection"><h4>Drosophila Image Award:</h4><div id="sec-110" class="subsection"><h5>Winners:</h5><p id="p-250">Raghav Chhetri (Janelia Research Campus, HHMI).</p><p id="p-251">Tanya Wolff (Janelia Research Campus, HHMI).</p></div><div id="sec-111" class="subsection"><h5>Runners-up:</h5><p id="p-252">Jonathan Enriquez (Columbia University).</p><p id="p-253">Justin Bosch (Harvard University).</p></div></div><div id="sec-112" class="subsection"><h4>Poster Awards:</h4><div id="sec-113" class="subsection"><h5>Graduate students:</h5><p id="p-254">1st Place: Daniel Kelpsch (University of Iowa).</p><p id="p-255">2nd Place: Afsoon Saadin (University of Maryland Baltimore County).</p><p id="p-256">3rd Place: Sumaira Zamurrad (Albert Einstein College of Medicine).</p></div><div id="sec-114" class="subsection"><h5>Undergraduate students:</h5><p id="p-257">1st Place: Niahz Wince (Pennsylvania State University, Berks College).</p><p id="p-258">2nd Place: Phuong Nguyen (Drexel University).</p></div></div><div id="sec-115" class="subsection"><h4>Victoria Finney Undergraduate Travel Awards:</h4><p id="p-259">Andrew Blake (Delaware State University).</p><p id="p-260">Taylor Hinnant (East Carolina University).</p><p id="p-261">Diana Luong (Loyola University Chicago).</p><p id="p-262">Katherine Nichols (Muhlenberg College).</p><p id="p-263">Abigail O’Conner (University of Arizona).</p><p id="p-264">Nilang Shah (Emory University).</p><p id="p-265">Jarrod Shilts (Vanderbilt University).</p><p id="p-266">Samantha St. Clair (Indiana University).</p></div></div><div id="sec-116" class="subsection"><h3>2016 Mouse Genetics Conference</h3><div id="sec-117" class="subsection"><h4>IMGS Verne Chapman Young Investigator Prize:</h4><p id="p-267">Jacob Moskowitz (University of Missouri).</p></div><div id="sec-118" class="subsection"><h4>IMGS Mary Lyon Award:</h4><p id="p-268">Krista Geister (Seattle Children’s Research Institute).</p></div><div id="sec-119" class="subsection"><h4>IMGS Oral Presentation Awards:</h4><p id="p-269">Elijah Edmondson (Colorado State University)</p><p id="p-270">Nitha Kartha (Cornell University)</p></div><div id="sec-120" class="subsection"><h4>GSA Poster Awards:</h4><div id="sec-121" class="subsection"><h5>Graduate students:</h5><p id="p-271">1st Place: Meng Zhang (Yale University).</p><p id="p-272">2nd Place: Sarah Bay (Emory University).</p></div><div id="sec-122" class="subsection"><h5>Undergraduate students:</h5><p id="p-273">Andreea Radulescu (University of Surrey, UK).</p></div></div><div id="sec-123" class="subsection"><h4>IMGS Poster Awards:</h4><p id="p-274">Mark Webster (Western Michigan University)</p><p id="p-275">William Barrington (Texas A&amp;M University)</p><p id="p-276">Nirav Chhabra (HelmholtzZentrum Munchen, Germany)</p><p id="p-277">Sehoon Keum (Institute for Basic Science, Korea)</p><p id="p-278">Lucas Laudermilk (University of North Carolina, Chapell Hill)</p><p id="p-279">Sarah Lewis (University of Wisconsin, Madison)</p><p id="p-280">Stephen Wellard (Johns Hopkins Bloomberg School of Public Health)</p><p id="p-281">Sofia Ivanatsiv (University of Toronto)</p><p id="p-282">Shigeru Makino (RIKEN BioResource Center, Japan)</p><p id="p-283">Samantha Fletcher (Texas A&amp;M University)</p></div></div><div id="sec-124" class="subsection"><h3>Population, Evolutionary, and Quantitative Genetics</h3><div id="sec-125" class="subsection"><h4>James F. Crow Early Career Researcher Award:</h4><p id="p-284">Sarah E. Sander (Cornell University).</p></div><div id="sec-126" class="subsection"><h4>Poster Awards:</h4><div id="sec-127" class="subsection"><h5>Graduate students:</h5><p id="p-285">1st Place: Thom Nelson (University of Oregon).</p><p id="p-286">2nd Place: Michelle Parmenter (University of Wisconsin–Madison).</p><p id="p-287">3rd Place: April Peterson (University of Wisconsin–Madison).</p></div><div id="sec-128" class="subsection"><h5>Undergraduate student:</h5><p id="p-288">Mathieu Henault (IBIS Université Laval, Canada).</p></div></div></div><div id="sec-129" class="subsection"><h3>Yeast Genetics Meeting</h3><div id="sec-130" class="subsection"><h4>Yeast Genetics Meeting Lifetime Achievement Award:</h4><p id="p-289">James Broach (Penn State University).</p></div><div id="sec-131" class="subsection"><h4>Ira Herskowitz Award:</h4><p id="p-290">Lars Steinmetz (Stanford University/EMBL).</p></div><div id="sec-132" class="subsection"><h4>Winge-Lindegren Address:</h4><p id="p-291">Rodney Rothstein (Columbia University).</p></div><div id="sec-133" class="subsection"><h4>Lee Hartwell Lecture:</h4><p id="p-292">Susan Gasser, (Friedrich Miescher Institute).</p></div><div id="sec-134" class="subsection"><h4>Poster Awards:</h4><div id="sec-135" class="subsection"><h5>Graduate students:</h5><p id="p-293">1st Place: Dara Lo (University of Toronto, Canada).</p><p id="p-294">2nd Place (tied): Jon Laurent (University of Texas at Austin)and Yu-San Yang (University of Texas Southwestern Medical Center).</p></div><div id="sec-136" class="subsection"><h5>Undergraduate student:</h5><p id="p-295">Alex Lederer (University of Pittsburgh).</p></div></div></div><div id="sec-137" class="subsection"><h3>12th International Conference on Zebrafish Development and Genetics</h3><div id="sec-138" class="subsection"><h4>Chi-Bin Chien Award:</h4><p id="p-296">Adam Miller (University of Oregon).</p></div><div id="sec-139" class="subsection"><h4>George Streisinger Award:</h4><p id="p-297">Chuck Kimmel (University of Oregon).</p></div><div id="sec-140" class="subsection"><h4>International Zebrafish Society Poster Awards</h4><div id="sec-141" class="subsection"><h5>Postdoctoral researchers</h5><p id="p-298">Chi-Kuo Hu (Stanford University)</p><p id="p-299">Kazunori Okada (Okazaki Institute for Integrative Biology/National Institute for Basic Biology)</p><p id="p-300">Ivan Cruz (University of Washington)</p></div><div id="sec-142" class="subsection"><h5>Graduate students</h5><p id="p-301">Molly Matty (Duke University)</p><p id="p-302">Tyson Fuller (University of Iowa)</p><p id="p-303">Emilia Asante (University of Missouri)</p></div><div id="sec-143" class="subsection"><h5>Undergraduate students</h5><p id="p-304">Mike Waltman (NIH/NIDCD)</p><p id="p-305">Logan Condon (University of Washington)</p></div></div><div id="sec-144" class="subsection"><h4>EuFishBioMed Poster Award</h4><p id="p-306">Zhen Jiang (University of Sheffield)</p></div><div id="sec-145" class="subsection"><h4>Travel Awards</h4><p id="p-307">Mitchell D'Rozario (Washington University School of Medicine)</p><p id="p-308">Yaniv M Elkouby (University of Pennsylvania)</p><p id="p-309">Meagan Grant (Princeton University)</p><p id="p-310">Michael R.M. Harrison (Children's Hospital Los Angeles)</p><p id="p-311">Sundas Ijaz (Yale University)</p><p id="p-312">Junsu Kang (Duke University)</p><p id="p-313">Daniel A Lee (California Institute of Technology)</p><p id="p-314">Zairan Liu (University California, San Francisco)</p><p id="p-315">Braedan M McCluskey (University of Oregon)</p><p id="p-316">Jose L. Pelliccia (Princeton University)</p><p id="p-317">Tetiana Petrachkova (Western Michigan University)</p><p id="p-318">Dorien Schepers (University of Antwerp and Antwerp University Hospital)</p><p id="p-319">Jinelle Wint (Stony Brook University)</p></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-320">This report was produced with generous support from the Bill &amp; Melinda Gates Foundation. We thank Anne Frances Johnson, Nancy Lamontagne, Kathleen Pierce, and Sarah Banducci of Creative Science Writing for their assistance in preparing the report.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-321">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036848/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036848/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-322"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">1</span> See Supplemental <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036848/-/DC1/FileS1.pdf">File S1</a>.</p></li></ul></div><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Organizers of The Allied Genetics Conference 2016</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.1" data-doi="10.1038/ng.1050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crissman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.2" data-doi="10.1126/science.1216872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venn</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ségurel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A fine-scale chimpanzee genetic map from population sequencing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>(<span class="cit-issue">6078</span>): <span class="cit-fpage">193</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1216872%26rft_id%253Dinfo%253Apmid%252F22422862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzYvNjA3OC8xOTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.3" data-doi="10.1126/science.aaf6850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hessel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome engineering. The genome project-write.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>(<span class="cit-issue">6295</span>): <span class="cit-fpage">126</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf6850%26rft_id%253Dinfo%253Apmid%252F27256881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTMvNjI5NS8xMjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.4" data-doi="10.1038/527S16a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourne</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorsch</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspective: sustaining the big-data ecosystem.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">527</span>(<span class="cit-issue">7576</span>): <span class="cit-fpage">S16</span>–<span class="cit-lpage">S17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D527%26rft.spage%253DS16%26rft_id%253Dinfo%253Adoi%252F10.1038%252F527S16a%26rft_id%253Dinfo%253Apmid%252F26536219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/527S16a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26536219&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.5" data-doi="10.1016/j.cell.2014.02.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanaugh</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geratowski</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wooltorton</span>  <span class="cit-name-given-names">J. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spaethling</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hector</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of a circadian output circuit for rest: activity rhythms in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">689</span>–<span class="cit-lpage">701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D689%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.02.024%26rft_id%253Dinfo%253Apmid%252F24766812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.02.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24766812&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.6" data-doi="10.1126/science.1182213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villarreal</span>  <span class="cit-name-given-names">G.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a Pitx1 enhancer.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>(<span class="cit-issue">5963</span>): <span class="cit-fpage">302</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1182213%26rft_id%253Dinfo%253Apmid%252F20007865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2My8zMDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.7" data-doi="10.1016/j.cell.2014.07.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracht</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolzhenko</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clay</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The architecture of a scrambled genome reveals massive levels of genomic rearrangement during development.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1187</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D1187%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.07.034%26rft_id%253Dinfo%253Apmid%252F25171416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.07.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25171416&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.8" data-doi="10.1016/j.cub.2004.06.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arellano-Carbajal</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybicki</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bono</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Soluble guanylate cyclases act in neurons exposed to the body fluid to promote <em>C. elegans</em> aggregation behavior.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DB.%2BH.%26rft.volume%253D14%26rft.issue%253D12%26rft.spage%253D1105%26rft.epage%253D1111%26rft.atitle%253DSoluble%2Bguanylate%2Bcyclases%2Bact%2Bin%2Bneurons%2Bexposed%2Bto%2Bthe%2Bbody%2Bfluid%2Bto%2Bpromote%2BC.%2Belegans%2Baggregation%2Bbehavior.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2004.06.027%26rft_id%253Dinfo%253Apmid%252F15203005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2004.06.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15203005&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222305500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.9" data-doi="10.1126/science.351.6280.1405-a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battey</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birnbaum</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Basic science: bedrock of progress.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">351</span>(<span class="cit-issue">6280</span>): <span class="cit-fpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCollins%26rft.auinit1%253DF.%2BS.%26rft.volume%253D351%26rft.issue%253D6280%26rft.spage%253D1405%26rft.epage%253D1405%26rft.atitle%253DBasic%2Bscience%253A%2BBedrock%2Bof%2Bprogress%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.351.6280.1405-a%26rft_id%253Dinfo%253Apmid%252F27013720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE1OiIzNTEvNjI4MC8xNDA1LWEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.10" data-doi="10.1126/science.1151851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of crossovers reveals extensive variation in fine-scale recombination patterns among humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>(<span class="cit-issue">5868</span>): <span class="cit-fpage">1395</span>–<span class="cit-lpage">1398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151851%26rft_id%253Dinfo%253Apmid%252F18239090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2OC8xMzk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.11" data-doi="10.1126/science.231.4743.1278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crosland</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crozier</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title"><em>Myrmecia pilosula</em>, an ant with only one pair of chromosomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">231</span>(<span class="cit-issue">4743</span>): <span class="cit-fpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCROSLAND%26rft.auinit1%253DM.%2BW.%2BJ.%26rft.volume%253D231%26rft.issue%253D4743%26rft.spage%253D1278%26rft.epage%253D1278%26rft.atitle%253DMyrmecia%2Bpilosula%252C%2Ban%2BAnt%2Bwith%2BOnly%2BOne%2BPair%2Bof%2BChromosomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.231.4743.1278%26rft_id%253Dinfo%253Apmid%252F17839565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzEvNDc0My8xMjc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.12" data-doi="10.1182/blood-2013-05-503177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Multani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robbins</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prostaglandin-modulated umbilical cord blood hematopoietic stem cell transplantation.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">122</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">3074</span>–<span class="cit-lpage">3081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2013-05-503177%26rft_id%253Dinfo%253Apmid%252F23996087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTIyLzE3LzMwNzQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.13" data-doi="10.1016/S0092-8674(00)81609-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Bono</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Natural variation in a neuropeptide Y receptor homolog modifies social behavior and food response in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">679</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253Dde%2BBono%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D5%26rft.spage%253D679%26rft.epage%253D689%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bneuropeptide%2BY%2Breceptor%2Bhomolog%2Bmodifies%2Bsocial%2Bbehavior%2Band%2Bfood%2Bresponse%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981609-8%26rft_id%253Dinfo%253Apmid%252F9741632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81609-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9741632&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075851900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.14" data-doi="10.1016/j.tig.2003.09.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalcanti</span>  <span class="cit-name-given-names">A. R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stover</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequencing the <em>Oxytricha trifallax</em> macronuclear genome: a pilot project.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">603</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDoak%26rft.auinit1%253DT.%2BG.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D603%26rft.epage%253D607%26rft.atitle%253DSequencing%2Bthe%2BOxytricha%2Btrifallax%2Bmacronuclear%2Bgenome%253A%2Ba%2Bpilot%2Bproject.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2003.09.013%26rft_id%253Dinfo%253Apmid%252F14585610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2003.09.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14585610&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186530600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.15" data-doi="10.1016/j.cell.2012.10.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracht</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowacki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landweber</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Piwi-interacting RNAs protect DNA against loss during <em>Oxytricha</em> genome rearrangement.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1243</span>–<span class="cit-lpage">1255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D151%26rft.spage%253D1243%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.10.045%26rft_id%253Dinfo%253Apmid%252F23217708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.10.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23217708&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311999900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.16" data-doi="10.1038/nature14264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kattenhorn</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondur</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Schaewen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorfman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">AAV-expressed eCD4-Ig provides durable protection from multiple SHIV challenges.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">519</span>(<span class="cit-issue">7541</span>): <span class="cit-fpage">87</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D519%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14264%26rft_id%253Dinfo%253Apmid%252F25707797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25707797&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.17" data-doi="10.1038/nature02714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karow</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Oxygen sensation and social feeding mediated by a <em>C. elegans</em> guanylate cyclase homologue.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>(<span class="cit-issue">6997</span>): <span class="cit-fpage">317</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGray%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D430%26rft.issue%253D6997%26rft.spage%253D317%26rft.epage%253D322%26rft.atitle%253DOxygen%2Bsensation%2Band%2Bsocial%2Bfeeding%2Bmediated%2Bby%2Ba%2BC.%2Belegans%2Bguanylate%2Bcyclase%2Bhomologue.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02714%26rft_id%253Dinfo%253Apmid%252F15220933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15220933&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222631200032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.18" data-doi="10.1126/science.aaf6419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boswell</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morante</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henkelman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdine</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Zebrafish models of idiopathic scoliosis link cerebrospinal fluid flow defects to spine curvature.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">352</span>(<span class="cit-issue">6291</span>): <span class="cit-fpage">1341</span>–<span class="cit-lpage">1344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrimes%26rft.auinit1%253DD.%2BT.%26rft.volume%253D352%26rft.issue%253D6291%26rft.spage%253D1341%26rft.epage%253D1344%26rft.atitle%253DZebrafish%2Bmodels%2Bof%2Bidiopathic%2Bscoliosis%2Blink%2Bcerebrospinal%2Bfluid%2Bflow%2Bdefects%2Bto%2Bspine%2Bcurvature%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf6419%26rft_id%253Dinfo%253Apmid%252F27284198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTIvNjI5MS8xMzQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.19" data-doi="10.1038/ng.2991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guenther</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasic</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedell</span>  <span class="cit-name-given-names">M.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A molecular basis for classic blond hair color in Europeans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D748%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2991%26rft_id%253Dinfo%253Apmid%252F24880339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24880339&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.20" data-doi="10.1038/ncomms2157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahar</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwessinger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tyrosine sulfation in a Gram-negative bacterium.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DHan%26rft.auinit1%253DS.%2BW.%26rft.volume%253D3%26rft.spage%253D1153%26rft.epage%253D1153%26rft.atitle%253DTyrosine%2Bsulfation%2Bin%2Ba%2BGram-negative%2Bbacterium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2157%26rft_id%253Dinfo%253Apmid%252F23093190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23093190&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.21" data-doi="10.1126/science.1225829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chylinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>(<span class="cit-issue">6096</span>): <span class="cit-fpage">816</span>–<span class="cit-lpage">821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225829%26rft_id%253Dinfo%253Apmid%252F22745249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ni84MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.22" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>(<span class="cit-issue">7392</span>): <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.23" data-doi="10.1126/science.1250553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kayser</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sehgal</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A critical period of sleep for development of courtship circuitry and behavior in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>(<span class="cit-issue">6181</span>): <span class="cit-fpage">269</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKayser%26rft.auinit1%253DM.%2BS.%26rft.volume%253D344%26rft.issue%253D6181%26rft.spage%253D269%26rft.epage%253D274%26rft.atitle%253DA%2BCritical%2BPeriod%2Bof%2BSleep%2Bfor%2BDevelopment%2Bof%2BCourtship%2BCircuitry%2Band%2BBehavior%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1250553%26rft_id%253Dinfo%253Apmid%252F24744368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDQvNjE4MS8yNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.24" data-doi="10.1126/science.aac6572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guglielmi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witkowsky</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dynamics of CRISPR-Cas9 genome interrogation in living cells.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>(<span class="cit-issue">6262</span>): <span class="cit-fpage">823</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKnight%26rft.auinit1%253DS.%2BC.%26rft.volume%253D350%26rft.issue%253D6262%26rft.spage%253D823%26rft.epage%253D826%26rft.atitle%253DDynamics%2Bof%2BCRISPR-Cas9%2Bgenome%2Binterrogation%2Bin%2Bliving%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac6572%26rft_id%253Dinfo%253Apmid%252F26564855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTAvNjI2Mi84MjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-6.12.3765.25"><div class="cit-metadata unstructured">Lauer, M., 2016 A Look at Trends in NIH’s Model Organism Research Support. Available at: <a href="https://nexus.od.nih.gov/all/2016/07/14/a-look-at-trends-in-nihs-model-organism-research-support/">https://nexus.od.nih.gov/all/2016/07/14/a-look-at-trends-in-nihs-model-organism-research-support/</a><a href="http://.">.</a> Accessed: September 30, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-6.12.3765.26" data-doi="10.1038/nature14569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahvic</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binder</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugach</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epoxyeicosatrienoic acids enhance embryonic haematopoiesis and adult marrow engraftment.</span> <span class="cit-source">Nature</span> <span class="cit-vol">523</span>: <span class="cit-fpage">468</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D468%26rft.atitle%253DEPOXYEICOSATRIENOIC%2BACIDS%2BENHANCE%2BEMBRYONIC%2BHAEMATOPOIESIS%2BAND%2BADULT%2BMARROW%2BENGRAFTMENT%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14569%26rft_id%253Dinfo%253Apmid%252F26201599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26201599&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maître</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turlier</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Illukkumbura</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eismann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niwayama</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Asymmetric division of contractile domains couples cell positioning and fate specification.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.28" data-doi="10.1016/j.cell.2007.10.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beleza</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollen</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittles</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">cis-Regulatory changes in Kit ligand expression and parallel evolution of pigmentation in sticklebacks and humans.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1179</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMiller%26rft.auinit1%253DC.%2BT.%26rft.volume%253D131%26rft.issue%253D6%26rft.spage%253D1179%26rft.epage%253D1189%26rft.atitle%253Dcis-Regulatory%2Bchanges%2Bin%2BKit%2Bligand%2Bexpression%2Band%2Bparallel%2Bevolution%2Bof%2Bpigmentation%2Bin%2Bsticklebacks%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.055%26rft_id%253Dinfo%253Apmid%252F18083106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18083106&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.29" data-doi="10.1093/nar/gkv466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agmon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khunsriraksakul</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Versatile genetic assembly system (VEGAS) to assemble pathways for expression in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">6620</span>–<span class="cit-lpage">6630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv466%26rft_id%253Dinfo%253Apmid%252F25956652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My8xMy82NjIwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.30" data-doi="10.1371/journal.pone.0002330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Möllenbeck</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalcanti</span>  <span class="cit-name-given-names">A. R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jönsson</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The pathway to detangle a scrambled gene.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e2330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMollenbeck%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De2330%26rft.epage%253De2330%26rft.atitle%253DThe%2Bpathway%2Bto%2Bdetangle%2Ba%2Bscrambled%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002330%26rft_id%253Dinfo%253Apmid%252F18523559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0002330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18523559&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-6.12.3765.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">National Academies of Sciences, Engineering, and Medicine</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>Genetically Engineered Crops: Experiences and Prospects</em>.</span> <span class="cit-publ-name">The National Academies Press</span>, <span class="cit-publ-loc">Washington, D.C</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.32" data-doi="10.1126/science.1201578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neurohr</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naegeli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Titos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A midzone-based ruler adjusts chromosome compaction to anaphase spindle length.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>(<span class="cit-issue">6028</span>): <span class="cit-fpage">465</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1201578%26rft_id%253Dinfo%253Apmid%252F21393511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAyOC80NjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-6.12.3765.33"><div class="cit-metadata unstructured">NIH Working Group on Data and Informatics, 2013 NIH Request For Information: Management, Integration, And Analysis of Large Biomedical Datasets. Bethesda, MD: National Institutes of Health. Available at: <a href="http://acd.od.nih.gov/DIWG_RFI_FinalReport.pdf">http://acd.od.nih.gov/DIWG_RFI_FinalReport.pdf</a>. Accessed: September 30, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Brown</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A recurrent regulatory change underlying altered expression and Wnt response of the stickleback armor plates gene EDA.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e05290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D4%26rft.issue%253D0%26rft.spage%253De05290%26rft.epage%253De05290%26rft.atitle%253DA%2Brecurrent%2Bregulatory%2Bchange%2Bunderlying%2Baltered%2Bexpression%2Band%2BWnt%2Bresponse%2Bof%2Bthe%2Bstickleback%2Barmor%2Bplates%2Bgene%2BEDA%26rft_id%253Dinfo%253Apmid%252F25629660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25629660&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.35" data-doi="10.1126/sciadv.1500245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwessinger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The rice immune receptor XA21 recognizes a tyrosine-sulfated protein from a Gram-negative bacterium.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1500245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DPruitt%26rft.auinit1%253DR.%2BN.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De1500245%26rft.epage%253De1500245%26rft.atitle%253DThe%2Brice%2Bimmune%2Breceptor%2BXA21%2Brecognizes%2Ba%2Btyrosine-sulfated%2Bprotein%2Bfrom%2Ba%2BGram-negative%2Bbacterium%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1500245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS82L2UxNTAwMjQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.36" data-doi="10.1371/journal.pbio.0020155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptak</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilad</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pääbo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Absence of the TAP2 human recombination hotspot in chimpanzees.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DPtak%26rft.auinit1%253DS.%2BE.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253De155%26rft.epage%253De155%26rft.atitle%253DAbsence%2Bof%2Bthe%2BTAP2%2Bhuman%2Brecombination%2Bhotspot%2Bin%2Bchimpanzees.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020155%26rft_id%253Dinfo%253Apmid%252F15208713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15208713&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.37" data-doi="10.1038/ng1529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptak</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koehler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Fine-scale recombination patterns differ between chimpanzees and humans</span>. <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>. <span class="cit-vol">37</span>: <span class="cit-fpage">429</span>–<span class="cit-fpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPtak%26rft.auinit1%253DS.%2BE.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D434%26rft.atitle%253DFine-scale%2Brecombination%2Bpatterns%2Bdiffer%2Bbetween%2Bchimpanzees%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1529%26rft_id%253Dinfo%253Apmid%252F15723063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15723063&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228040000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.38" data-doi="10.1126/science.1166527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A polymorphism in npr-1 is a behavioral determinant of pathogen susceptibility in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>(<span class="cit-issue">5912</span>): <span class="cit-fpage">382</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReddy%26rft.auinit1%253DK.%2BC.%26rft.volume%253D323%26rft.issue%253D5912%26rft.spage%253D382%26rft.epage%253D384%26rft.atitle%253DA%2BPolymorphism%2Bin%2Bnpr-1%2BIs%2Ba%2BBehavioral%2BDeterminant%2Bof%2BPathogen%2BSusceptibility%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166527%26rft_id%253Dinfo%253Apmid%252F19150845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMi8zODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.39" data-doi="10.1038/nature18614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reischauer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villasenor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Cloche is a bHLH-PAS transcription factor that drives haemato-vascular specification.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">535</span>(<span class="cit-issue">7611</span>): <span class="cit-fpage">294</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D535%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18614%26rft_id%253Dinfo%253Apmid%252F27411634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27411634&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The putative chemoreceptor families of <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">WormBook</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWormBook%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.41" data-doi="10.1038/nn1140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reale</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatwin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inhibition of <em>Caenorhabditis elegans</em> social feeding by FMRFamide-related peptide activation of NPR-1.</span> <abbr class="cit-jnl-abbrev">Nat. Neurosci.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1178</span>–<span class="cit-lpage">1185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DRogers%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D1178%26rft.epage%253D1185%26rft.atitle%253DInhibition%2Bof%2BCaenorhabditis%2Belegans%2Bsocial%2Bfeeding%2Bby%2BFMRFamide-related%2Bpeptide%2Bactivation%2Bof%2BNPR-1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn1140%26rft_id%253Dinfo%253Apmid%252F14555955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn1140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14555955&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186229200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.42" data-doi="10.1101/gr.193433.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stracquadanio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SCRaMbLE generates designed combinatorial stochastic diversity in synthetic chromosomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">36</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.193433.115%26rft_id%253Dinfo%253Apmid%252F26566658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvMzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM3NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.43" data-doi="10.1126/science.270.5243.1804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pi</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A receptor kinase-like protein encoded by the rice disease resistance gene, Xa21.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">270</span>(<span class="cit-issue">5243</span>): <span class="cit-fpage">1804</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSong%26rft.auinit1%253DW.-Y.%26rft.volume%253D270%26rft.issue%253D5243%26rft.spage%253D1804%26rft.epage%253D1806%26rft.atitle%253DA%2BReceptor%2BKinase-Like%2BProtein%2BEncoded%2Bby%2Bthe%2BRice%2BDisease%2BResistance%2BGene%252C%2BXa21%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.270.5243.1804%26rft_id%253Dinfo%253Apmid%252F8525370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzAvNTI0My8xODA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.44" data-doi="10.1038/ng.2007.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulem</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stacey</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helgason</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafnar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic determinants of hair, eye and skin pigmentation in Europeans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1443</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSulem%26rft.auinit1%253DP.%26rft.volume%253D39%26rft.issue%253D12%26rft.spage%253D1443%26rft.epage%253D1452%26rft.atitle%253DGenetic%2Bdeterminants%2Bof%2Bhair%252C%2Beye%2Band%2Bskin%2Bpigmentation%2Bin%2BEuropeans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.13%26rft_id%253Dinfo%253Apmid%252F17952075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17952075&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251267400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.45" data-doi="10.1371/journal.pbio.1001473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swart</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracht</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magrini</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minx</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>Oxytricha trifallax</em> macronuclear genome: a complex eukaryotic genome with 16,000 tiny chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1001473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001473%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001473%26rft_id%253Dinfo%253Apmid%252F23382650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001473&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382650&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.46" data-doi="10.1016/j.cell.2014.12.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamplin</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagedorn</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Hematopoietic stem cell arrival triggers dynamic remodeling of the perivascular niche.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">160</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">241</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D160%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.12.032%26rft_id%253Dinfo%253Apmid%252F25594182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.12.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25594182&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3765.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3765.47" data-doi="10.1534/genetics.114.171785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wangler</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellen</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fruit flies in biomedical research.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">639</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171785%26rft_id%253Dinfo%253Apmid%252F25624315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvNjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zNzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3757" title="CRISPR/Cas9-Mediated Genome Editing and Mutagenesis of EcChi4 in Exopalaemon carinicauda" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3787" title="Assessing the Gene Content of the Megagenome: Sugar Pine (Pinus lambertiana)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/12"><span>Volume 6 Issue 12, December 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/12.cover-source.jpg?itok=F-Hw09b0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3765" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-J1nYr6AH4jKIP1FwfxhudZc3Yn0O65Y_KaP5_xLbnQ0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/meeting-report" class="" data-icon-position="" data-hide-link-title="0">Meeting Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/12/3765" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/12/3765" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/12/3765" class="active" data-icon-position="" data-hide-link-title="0">Meeting Report: The Allied Genetics Conference 2016</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460507" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S0vPpykyZgmR9_fO0uA-c32L1Hnr9He-irlUlZ0Raow" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3765.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/12/3765" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZhBgpIyskyF7ENwnztYe164G_lroDIYlXKhR3fVI7rQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/12/3765" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3765.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460507" id="node460507--2" data-pisa="ggg;6/12/3765" data-pisa-master="ggg;6/12/3765" data-apath="/ggg/6/12/3765.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Meeting Report: The Allied Genetics Conference 2016</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">Organizers of The Allied Genetics Conference 2016</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3765-3786;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036848 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460507--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genetics Society of America, Bethesda, Maryland, 20814</div></span></div></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/12/3765" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/12/3765&amp;title=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/12/3765&amp;title=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/12/3765&amp;title=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/12/3765&amp;text=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/12/3765&amp;title=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/12/3765&amp;t=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/12/3765&amp;title=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/12/3765&amp;title=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460507/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460507" id="node460507--3" data-pisa="ggg;6/12/3765" data-pisa-master="ggg;6/12/3765" data-apath="/ggg/6/12/3765.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Meeting Report: The Allied Genetics Conference 2016</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">Organizers of The Allied Genetics Conference 2016</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3765-3786;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036848 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460507--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Genetics Society of America, Bethesda, Maryland, 20814</div></span></div></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3765&amp;rft.epage=3786&amp;rft.atitle=Meeting%20Report%3A%20The%20Allied%20Genetics%20Conference%202016&amp;rft.volume=6&amp;rft.issue=12&amp;rft.date=2016-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=%2C+"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460507/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460507/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460507/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460507/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460507/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460507/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460507/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460507/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460507/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460507/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460507/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460507/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492491" id="node492491" data-pisa="ggg;9/3/635" data-pisa-master="ggg;g3.118.201003" data-apath="/ggg/9/3/635.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/3/635" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Meeting Report: Experimental and Evolutionary Approaches to Yeast and Other Organisms 2018</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486789" id="node486789" data-pisa="ggg;7/10/3237" data-pisa-master="ggg;g3.117.300124" data-apath="/ggg/7/10/3237.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/7/10/3237" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Meeting%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Meeting Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/12/3765" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#sec-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">C. elegans Development, Cell Biology, and Gene Expression Meeting</a></li><li class="even"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Ciliate Molecular Biology CONFERENCE</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">57th Annual Drosophila Research Conference</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Mouse Genetics 2016</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Population, Evolutionary, and Quantitative Genetics Conference</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Yeast Genetics Meeting</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">12th International Conference on Zebrafish Development and Genetics</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Model Organism Genome Resources</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Jef Boeke</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Francis Collins</a></li><li class="odd"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Jennifer Doudna</a></li><li class="even"><a href="#sec-34" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Anna-Katerina Hadjantonakis</a></li><li class="odd"><a href="#sec-42" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">David Kingsley</a></li><li class="even"><a href="#sec-50" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Laura Landweber and Richard Miller</a></li><li class="odd"><a href="#sec-57" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Michael Miller</a></li><li class="even"><a href="#sec-64" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Molly Przeworski</a></li><li class="odd"><a href="#sec-72" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Pamela Ronald</a></li><li class="even"><a href="#sec-78" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Amita Sehgal</a></li><li class="odd"><a href="#sec-84" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Leonard Zon</a></li><li class="even"><a href="#sec-90" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Honors and Awards</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/12/3765.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/12/3765.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/12/3765.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/12/3765.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
