<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/12/3825" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/12/3825.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/12/3825.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/12/3825.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/12/3825.atom" />
<meta name="HW.pisa" content="ggg;6/12/3825" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Transcriptome Sequencing of Diverse Peanut (Arachis) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability" />
<meta name="DC.Identifier" content="10.1534/g3.115.026898" />
<meta name="DC.Date" content="2016-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Chopra et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="To test the hypothesis that the cultivated peanut species possesses almost no molecular variability, we sequenced a diverse panel of 22 Arachis accessions representing Arachis hypogaea botanical classes, A-, B-, and K- genome diploids, a synthetic amphidiploid, and a tetraploid wild species. RNASeq was performed on pools of three tissues, and de novo assembly was performed. Realignment of individual accession reads to transcripts of the cultivar OLin identified 306,820 biallelic SNPs. Among 10 naturally occurring tetraploid accessions, 40,382 unique homozygous SNPs were identified in 14,719 contigs. In eight diploid accessions, 291,115 unique SNPs were identified in 26,320 contigs. The average SNP rate among the 10 cultivated tetraploids was 0.5, and among eight diploids was 9.2 per 1000 bp. Diversity analysis indicated grouping of diploids according to genome classification, and cultivated tetraploids by subspecies. Cluster analysis of variants indicated that sequences of B genome species were the most similar to the tetraploids, and the next closest diploid accession belonged to the A genome species. A subset of 66 SNPs selected from the dataset was validated; of 782 SNP calls, 636 (81.32%) were confirmed using an allele-specific discrimination assay. We conclude that substantial genetic variability exists among wild species. Additionally, significant but lesser variability at the molecular level occurs among accessions of the cultivated species. This survey is the first to report significant SNP level diversity among transcripts, and may explain some of the phenotypic differences observed in germplasm surveys. Understanding SNP variants in the Arachis accessions will benefit in developing markers for selection." />
<meta name="DC.Contributor" content="Ratan Chopra" />
<meta name="DC.Contributor" content="Gloria Burow" />
<meta name="DC.Contributor" content="Charles E. Simpson" />
<meta name="DC.Contributor" content="Jennifer Chagoya" />
<meta name="DC.Contributor" content="Joann Mudge" />
<meta name="DC.Contributor" content="Mark D. Burow" />
<meta name="article:published_time" content="2016-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Transcriptome Sequencing of Diverse Peanut (Arachis) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;To test the hypothesis that the cultivated peanut species possesses almost no molecular variability, we sequenced a diverse panel of 22 &lt;i&gt;Arachis&lt;/i&gt; accessions representing &lt;i&gt;Arachis hypogaea&lt;/i&gt; botanical classes, A-, B-, and K- genome diploids, a synthetic amphidiploid, and a tetraploid wild species. RNASeq was performed on pools of three tissues, and &lt;i&gt;de novo&lt;/i&gt; assembly was performed. Realignment of individual accession reads to transcripts of the cultivar OLin identified 306,820 biallelic SNPs. Among 10 naturally occurring tetraploid accessions, 40,382 unique homozygous SNPs were identified in 14,719 contigs. In eight diploid accessions, 291,115 unique SNPs were identified in 26,320 contigs. The average SNP rate among the 10 cultivated tetraploids was 0.5, and among eight diploids was 9.2 per 1000&amp;nbsp;bp. Diversity analysis indicated grouping of diploids according to genome classification, and cultivated tetraploids by subspecies. Cluster analysis of variants indicated that sequences of B genome species were the most similar to the tetraploids, and the next closest diploid accession belonged to the A genome species. A subset of 66 SNPs selected from the dataset was validated; of 782 SNP calls, 636 (81.32%) were confirmed using an allele-specific discrimination assay. We conclude that substantial genetic variability exists among wild species. Additionally, significant but lesser variability at the molecular level occurs among accessions of the cultivated species. This survey is the first to report significant SNP level diversity among transcripts, and may explain some of the phenotypic differences observed in germplasm surveys. Understanding SNP variants in the &lt;i&gt;Arachis&lt;/i&gt; accessions will benefit in developing markers for selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/12/01" />
<meta name="citation_mjid" content="ggg;6/12/3825" />
<meta name="citation_id" content="6/12/3825" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/12/3825" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/12/3825.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/12/3825.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/12/3825.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.026898" />
<meta name="citation_pmid" content="27729436" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3825" />
<meta name="citation_lastpage" content="3836" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ratan Chopra" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409" />
<meta name="citation_author" content="Gloria Burow" />
<meta name="citation_author_institution" content="United States Department of Agriculture, Agriculture Research Service, Cropping Systems Research Laboratory, Lubbock, Texas 79415" />
<meta name="citation_author" content="Charles E. Simpson" />
<meta name="citation_author_institution" content="Texas A&amp;M AgriLife Research, Stephenville, Texas 76401" />
<meta name="citation_author" content="Jennifer Chagoya" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409" />
<meta name="citation_author_institution" content="Texas A&amp;M AgriLife Research, Lubbock, Texas 79403" />
<meta name="citation_author" content="Joann Mudge" />
<meta name="citation_author_institution" content="National Center for Genome Resources, Santa Fe, New Mexico 87505" />
<meta name="citation_author" content="Mark D. Burow" />
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409" />
<meta name="citation_author_institution" content="Texas A&amp;M AgriLife Research, Lubbock, Texas 79403" />
<meta name="citation_author_email" content="mburow@tamu.edu" />
<meta name="citation_reference" content="citation_journal_title=Peanut Sci.;citation_author=W. F. Anderson;citation_author=C. C. Holbrook;citation_author=A. K. Culbreath;citation_title=Screening the peanut core collection for resistance to tomato spotted wilt virus.;citation_pages=57-61;citation_volume=23;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Bansal;citation_author=V. Bafna;citation_title=HapCUT: an efficient and accurate algorithm for the haplotype assembly problem.;citation_pages=153-159;citation_volume=24;citation_year=2008;citation_doi=10.1093/bioinformatics/btn298" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. R. Baring;citation_author=C. E. Simpson;citation_author=M. D. Burow;citation_author=M. C. Black;citation_author=J. M. Cason;citation_title=Registration of 'Tamrun OL07' Peanut;citation_pages=2721-2722;citation_volume=46;citation_year=2006;citation_issue=6;citation_doi=10.2135/cropsci2006.06.0413" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour.;citation_author=D. Bertioli;citation_author=G. Seijo;citation_author=F. Freitas;citation_author=J. Valls;citation_author=S. Leal-Bertioli;citation_title=An overview of peanut and its wild relatives.;citation_pages=134-149;citation_volume=9;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=D. J. Bertioli;citation_author=S. B. Cannon;citation_author=L. Froenicke;citation_author=G. Huang;citation_author=A. D. Farmer;citation_title=The genome sequences of Arachis duranensis and Arachis ipaënsis, the diploid ancestors of cultivated peanut.;citation_pages=438-446;citation_volume=48;citation_year=2016;citation_pmid=26901068;citation_doi=10.1038/ng.3517" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. Burow;citation_author=C. Simpson;citation_author=W. Faries;citation_author=J. Starr;citation_author=A. Paterson;citation_title=Molecular biogeographic study of recently described B- and A-genome Arachis species, also providing new insights into the origins of cultivated peanut.;citation_pages=107-119;citation_volume=52;citation_year=2009;citation_issue=2;citation_pmid=19234559;citation_doi=10.1139/G08-094" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. D. Burow;citation_author=J. L. Starr;citation_author=C.-H. Park;citation_author=C. E. Simpson;citation_author=A. H. Paterson;citation_title=Introgression of homoeologous quantitative trait loci (QTLs) for resistance to the root-knot nematode [Meloidogyne arenaria (Neal) Chitwood] in an advanced backcross-QTL population of peanut (Arachis hypogaea L.).;citation_pages=393-406;citation_volume=34;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=X. Chen;citation_author=W. Zhu;citation_author=S. Azam;citation_author=H. Li;citation_author=F. Zhu;citation_title=Deep sequencing analysis of the transcriptomes of peanut aerial and subterranean young pods identifies candidate genes related to early embryo abortion.;citation_pages=115-127;citation_volume=11;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Choi;citation_author=D. J. Kim;citation_author=T. Uhm;citation_author=E. Limpens;citation_author=H. Lim;citation_title=A Sequence-Based Genetic Map of Medicago truncatula and Comparison of Marker Colinearity with M. sativa;citation_pages=1463-1502;citation_volume=166;citation_year=2004;citation_issue=3;citation_pmid=15082563;citation_doi=10.1534/genetics.166.3.1463" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I.-Y. Choi;citation_author=D. L. Hyten;citation_author=L. K. Matukumalli;citation_author=Q. Song;citation_author=J. M. Chaky;citation_title=A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis.;citation_volume=176;citation_year=2007;citation_pmid=17339218;citation_doi=10.1534/genetics.107.070821" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Chopra;citation_author=G. B. Burow;citation_author=A. Farmer;citation_author=J. A. Mudge;citation_author=C. E. Simpson;citation_title=Comparisons of de novo transcriptome assemblers in diploid and polyploid species using peanut (Arachis spp.) RNA-Seq data.;citation_pages=e115055;citation_volume=9;citation_year=2014;citation_issue=12;citation_pmid=25551607;citation_doi=10.1371/journal.pone.0115055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=R. Chopra;citation_author=G. Burow;citation_author=A. Farmer;citation_author=J. Mudge;citation_author=C. E. Simpson;citation_title=Next-generation transcriptome sequencing, SNP discovery and validation in four market classes of peanut, Arachis hypogaea L.;citation_pages=1169-1180;citation_volume=290;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. A. Deokar;citation_author=L. Ramsay;citation_author=A. G. Sharpe;citation_author=M. Diapari;citation_author=A. Sindhu;citation_title=Genome wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly.;citation_pages=708;citation_volume=15;citation_year=2014;citation_pmid=25150411;citation_doi=10.1186/1471-2164-15-708" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Bonplandia;citation_author=A. Fernandez;citation_author=A. Krapovickas;citation_title=Cromosomas y evolución en Arachis (Leguminosae).;citation_pages=187-220;citation_volume=8;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=R. Garg;citation_author=R. K. Patel;citation_author=A. K. Tyagi;citation_author=M. Jain;citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification.;citation_volume=18;citation_year=2011;citation_pmid=21217129;citation_doi=10.1093/dnares/dsq028" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. Grabherr;citation_author=B. Haas;citation_author=M. Yassour;citation_author=J. Levin;citation_author=D. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_title=Evolution of Crop Plants;citation_title=Groundnut;citation_author=W. Gregory;citation_author=M. Gregory;citation_pages=151-154;citation_year=1976" />
<meta name="citation_reference" content="citation_title=Advances in Legume Science;citation_title=Structure, variation, evolution, and classification in Arachis;citation_author=W. Gregory;citation_author=A. Krapovickas;citation_author=M. Gregory;citation_pages=469-481;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=T. Halward;citation_author=H. T. Stalker;citation_author=E. Larue;citation_author=G. Kochert;citation_title=Genetic variation detectable with molecular markers among unadapted germplasm resources of cultivated peanut and related wild species.;citation_pages=1013-1020;citation_volume=34;citation_year=1991;citation_doi=10.1139/g91-156" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. C. Holbrook;citation_author=W. Dong;citation_title=Development and Evaluation of a Mini Core Collection for the U.S. Peanut Germplasm Collection;citation_pages=1540-1544;citation_volume=45;citation_year=2005;citation_issue=4;citation_doi=10.2135/cropsci2004.0368" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. C. Holbrook;citation_author=W. Anderson;citation_author=R. Pittman;citation_title=Selection of a Core Collection from the U.S. Germplasm Collection of Peanut;citation_pages=859-861;citation_volume=33;citation_year=1993;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=J. of Plant Regist.;citation_author=C. C. Holbrook;citation_author=P. Timper;citation_author=A. K. Culbreath;citation_author=C. K. Kvien;citation_title=Registration of ‘Tifguard’ peanut.;citation_pages=2;citation_volume=2;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. C. Hsi;citation_title=Registration of New Mexico Valencia C Peanut (Reg. No. 24);citation_pages=113-114;citation_volume=20;citation_year=1980;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Huang;citation_author=H. Jiang;citation_author=X. Ren;citation_author=Y. Chen;citation_author=Y. Xiao;citation_title=Abundant microsatellite diversity and oil content in wild arachis species.;citation_pages=e50002-e50002;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185514" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=Peanut Sci.;citation_author=T. E. Isleib;citation_author=H. E. Pattee;citation_author=F. G. Giesbrecht;citation_title=Ancestral Contributions to Roasted Peanut Attribute.;citation_pages=42-48;citation_volume=22;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Proc. Amer. Peanut Res. Educ. Soc.;citation_author=H. Jiang;citation_author=B. Liao;citation_author=N. Duan;citation_author=C. C. Holbrook;citation_author=B. Guo;citation_title=Development of a core collection of peanut germplasm in China.;citation_pages=33;citation_volume=36;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=H. Jiang;citation_author=H. Li;citation_author=R. Xiaoping;citation_author=C. Yuning;citation_author=Z. Xiaojing;citation_title=Diversity characterization and association analysis of agronomic traits in a Chinese peanut (Arachis hypogaea L.) mini-core collection.;citation_pages=159-169;citation_volume=56;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Khera;citation_author=H. D. Upadhyaya;citation_author=M. K. Pandey;citation_author=M. Roorkiwal;citation_author=M. Sriswathi;citation_title=Single nucleotide polymorphism-based genetic diversity in the reference set of peanut (spp.) by developing and applying cost-effective kompetitive allele specific polymerase chain reaction genotyping assays.;citation_pages=1-11;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Kochert;citation_author=T. Halward;citation_author=W. Branch;citation_author=C. Simpson;citation_title=RFLP variability in peanut (Arachis hypogaea L.) cultivars and wild species.;citation_pages=565-570;citation_volume=81;citation_year=1991;citation_pmid=24221368" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=G. Kochert;citation_author=H. Stalker;citation_author=M. Gimenes;citation_author=L. Galgaro;citation_author=C. Lopes;citation_title=RFLP and cytogenetic evidence on the origin and evolution of allotetraploid domesticated peanut Arachis hypogaea (Leguminosae).;citation_pages=1282-1291;citation_volume=83;citation_year=1996;citation_doi=10.2307/2446112" />
<meta name="citation_reference" content="citation_journal_title=Bonplandia;citation_author=A. Krapovickas;citation_author=W. Gregory;citation_title=Taxonomia del género Arachis (Leguminosae).;citation_pages=1-186;citation_volume=8;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=K. Krasileva;citation_author=V. Buffalo;citation_author=P. Bailey;citation_author=S. Pearce;citation_author=S. Ayling;citation_title=Separating homoeologs by phasing in the tetraploid wheat transcriptome.;citation_pages=R66;citation_volume=14;citation_year=2013;citation_pmid=23800085;citation_doi=10.1186/gb-2013-14-6-r66" />
<meta name="citation_reference" content="citation_journal_title=Cytologia (Tokyo);citation_author=G. Lavia;citation_title=Karyotypes of Arachis palustris and A. praecox (section Arachis), two species with basic chromosome number x = 9.;citation_pages=177-181;citation_volume=63;citation_year=1998;citation_doi=10.1508/cytologia.63.177" />
<meta name="citation_reference" content="citation_journal_title=Phanerogams-Angiosperm.;citation_author=G. Lavia;citation_author=A. Fernandez;citation_author=G. Seijo;citation_title=Cytogenetic and molecular evidences on the evolutionary relationships among Arachis species. plant genome: biodiversity and evolution;citation_pages=101-134;citation_volume=1E;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=A. Leonforte;citation_author=S. Sudheesh;citation_author=N. O. I. Cogan;citation_author=P. A. Salisbury;citation_author=M. E. Nicolas;citation_title=SNP marker discovery, linkage map construction and identification of QTLs for enhanced salinity tolerance in field pea (Pisum sativum L.);citation_pages=161;citation_volume=13;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=X. Li;citation_author=A. Acharya;citation_author=A. D. Farmer;citation_author=J. A. Crow;citation_author=A. K. Bharti;citation_title=Prevalence of single nucleotide polymorphism among 27 diverse alfalfa genotypes as assessed by transcriptome sequencing.;citation_pages=568-568;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23107476;citation_doi=10.1186/1471-2164-13-568" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=M. Lohse;citation_author=A. Nagel;citation_author=T. Herter;citation_author=P. May;citation_author=M. Schroda;citation_title=Mercator: a fast and simple web server for genome scale functional annotation of plant sequence data.;citation_pages=1250-1258;citation_volume=37;citation_year=2014;citation_doi=10.1111/pce.12231" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Lu;citation_author=B. Pickersgill;citation_title=Isozyme variation and species relationships in peanut and its wild relatives (Arachis L. - Leguminosae).;citation_pages=550-560;citation_volume=85;citation_year=1993;citation_pmid=24195929" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=N. Mallikarjuna;citation_author=S. Senthilvel;citation_author=D. Hoisington;citation_title=Development of new sources of tetraploid Arachis to broaden the genetic base of cultivated groundnut (Arachis hypogaea L.).;citation_pages=889-907;citation_volume=58;citation_year=2011;citation_doi=10.1007/s10722-010-9627-8" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R.F. McCormick;citation_author=S. K. Truong;citation_author=J. E. Mullet;citation_title=RIG: recalibration and interrelation of genomic sequence data with the GATK.;citation_volume=5;citation_year=2015;citation_pmid=25681258;citation_doi=10.1534/g3.115.017012" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=M. M. Moretzsohn;citation_author=S. Hopkins;citation_author=S. Mitchell;citation_author=J. Kresovich;citation_author=J. Valls;citation_title=Genetic diversity of peanut (Arachis hypogaea L.) and its wild relatives based on the analysis of hypervariable regions of the genome.;citation_pages=11-11;citation_volume=4;citation_year=2004;citation_pmid=15253775;citation_doi=10.1186/1471-2229-4-11" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E.D. Nagy;citation_author=Y. Guo;citation_author=S. Tang;citation_author=J. E. Bowers;citation_author=R. A. Okashah;citation_title=A high-density genetic map of Arachis duranensis, a diploid ancestor of cultivated peanut.;citation_pages=469-469;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22967170;citation_doi=10.1186/1471-2164-13-469" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_title=Peanuts—Cultures and Uses;citation_title=Breeding of the cultivated peanut (Arachis hypogaea L.);citation_author=A. J. Norden;citation_pages=175-208;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. J. Norden;citation_author=R. W. Lipscomb;citation_author=W. A. Carver;citation_title=Registration of Florunner Peanuts (Reg. No. 2);citation_pages=850-850;citation_volume=9;citation_year=1969;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. K. Pandey;citation_author=H. D. Upadhyaya;citation_author=A. Rathore;citation_author=V. Vadez;citation_author=M. S. Sheshshayee;citation_title=Genomewide association studies for 50 agronomic traits in peanut using the ‘reference set’ comprising 300 genotypes from 48 countries of the semi-arid tropics of the world.;citation_pages=e105228;citation_volume=9;citation_year=2014;citation_issue=8" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=S. Raina;citation_author=Y. Mukai;citation_title=Genomic in situ hybridization in Arachis (Fabaceae) identifies the diploid wild progenitors of cultivated (A. hypogaea) and related wild (A. monticola) peanut species.;citation_pages=251-262;citation_volume=214;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=S. Raina;citation_author=V. Rani;citation_author=T. Kojima;citation_author=Y. Ogihara;citation_author=K. Singh;citation_title=RAPD and ISSR fingerprints as useful genetic markers for analysis of genetic diversity, varietal identification, and phylogenetic relationships in peanut (Arachis hypogaea) cultivars and wild species.;citation_pages=763-772;citation_volume=44;citation_year=2001;citation_issue=5;citation_pmid=11681599" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. W. Schreiber;citation_author=M. J. Hayden;citation_author=K. L. Forrest;citation_author=S. L. Kong;citation_author=P. Langridge;citation_title=Transcriptome-scale homoeolog-specific transcript assemblies of bread wheat.;citation_pages=492-492;citation_volume=13;citation_year=2012;citation_pmid=22989011" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=G. Seijo;citation_author=G. I. Lavia;citation_author=A. Fernández;citation_author=A. Krapovickas;citation_author=D. A. Ducasse;citation_title=Genomic relationships between the cultivated peanut (Arachis hypogaea, Leguminosae) and its close relatives revealed by double GISH;citation_pages=1963-1971;citation_volume=94;citation_year=2007;citation_issue=12;citation_pmid=21636391;citation_doi=10.3732/ajb.94.12.1963" />
<meta name="citation_reference" content="citation_journal_title=Plant Growth Regul.;citation_author=M. G. Selvaraj;citation_author=G. Burow;citation_author=J. J. Burke;citation_author=V. Belamkar;citation_author=N. Puppala;citation_title=Heat stress screening of peanut (Arachis hypogaea L.) seedlings for acquired thermotolerance.;citation_pages=83-91;citation_volume=65;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Peanut Sci.;citation_author=C. Simpson;citation_title=Use of wild Arachis species/introgression of genes into A. hypogaea L.;citation_pages=114-116;citation_volume=28;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_title=Registration of 'COAN' Peanut;citation_pages=918-918;citation_volume=41;citation_year=2001;citation_issue=3;citation_doi=10.2135/cropsci2001.413918x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. E. Simpson;citation_author=S. C. Nelson;citation_author=J. L. Starr;citation_author=K. E. Woodard;citation_author=O. D. Smith;citation_title=Registration of TxAG-6 and TxAG-7 Peanut Germplasm Lines;citation_pages=1418-1418;citation_volume=33;citation_year=1993;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. E. Simpson;citation_author=M. R. Baring;citation_author=A. M. Schubert;citation_author=H. A. Melouk;citation_author=Y. López;citation_title=Registration of 'OLin' Peanut;citation_pages=1880-1881;citation_volume=43;citation_year=2003;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_author=G. T. Church;citation_author=M. D. Burow;citation_author=A. H. Paterson;citation_title=Registration of 'NemaTAM' Peanut;citation_pages=1561-1561;citation_volume=43;citation_year=2003;citation_issue=4;citation_doi=10.2135/cropsci2003.1561" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Regist.;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_author=M. R. Baring;citation_author=M. D. Burow;citation_author=J. M. Cason;citation_title=Registration of ‘Webb’ Peanut.;citation_pages=265-268;citation_volume=7;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Photosynthetica;citation_author=A. L. Singh;citation_author=R. N. Nakar;citation_author=K. Chakraborty;citation_author=K. A. Kalariya;citation_title=Physiological efficiencies in mini-core peanut germplasm accessions during summer season.;citation_pages=627-635;citation_volume=52;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_title=Polyploidy: recurrent formation and genome evolution.;citation_pages=348-352;citation_volume=14;citation_year=1999;citation_issue=9;citation_pmid=10441308;citation_doi=10.1016/S0169-5347(99)01638-9" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=H. T. Stalker;citation_author=J. P. Moss;citation_title=Speciation, cytogenetics, and utilization of Arachis species.;citation_pages=1-40;citation_volume=41;citation_year=1987;citation_doi=10.1016/S0065-2113(08)60801-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=UniProt: a hub for protein information.;citation_volume=43;citation_year=2015;citation_pmid=25348405;citation_doi=10.1093/nar/gku989" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=H. D. Upadhyaya;citation_title=Phenotypic diversity in groundnut (Arachis hypogaea L.) core collection assessed by morphological and agronomical evaluations.;citation_pages=539-550;citation_volume=50;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. D. Upadhyaya;citation_title=Variability for Drought Resistance Related Traits in the Mini Core Collection of Peanut;citation_pages=1432-1440;citation_volume=45;citation_year=2005;citation_issue=4;citation_doi=10.2135/cropsci2004.0389" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. D. Upadhyaya;citation_author=P. Bramel;citation_author=R. Ortiz;citation_author=S. Singh;citation_title=Developing a Mini Core of Peanut for Utilization of Genetic Resources;citation_pages=2150-2156;citation_volume=42;citation_year=2002;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=H. D. Upadhyaya;citation_author=L. J. Reddy;citation_author=C. L. L. Gowda;citation_author=S. Singh;citation_title=Identification of diverse groundnut germplasm: sources of early-maturity in a core collection.;citation_pages=261-267;citation_volume=97;citation_year=2006;citation_doi=10.1016/j.fcr.2005.10.010" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen. Res.;citation_author=H. D. Upadhyaya;citation_author=C. L. L. Gowda;citation_author=H. K. Buhariwalla;citation_author=J. H. Crouch;citation_title=Efficient use of crop germplasm resources: identifying useful germplasm for crop improvement through core and mini-core collections and molecular marker approaches.;citation_pages=25-35;citation_volume=4;citation_year=2006" />
<meta name="citation_reference" content="citation_title=Genetics, Genomics and Breeding of Peanuts;citation_title=Genetic resources, diversity and association mapping in peanut;citation_author=H. D. Upadhyaya;citation_author=S. Sharma;citation_author=S. L. Dwivedi;citation_pages=13-36;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bonplandia;citation_author=J. Valls;citation_author=C. Simpson;citation_title=New species of Arachis L. (Leguminosae) from Brazil, Paraguay and Bolivia.;citation_pages=35-64;citation_volume=14;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. L. Wang;citation_author=S. Sukumaran;citation_author=N. A. Barkley;citation_author=Z. Chen;citation_author=C. Y. Chen;citation_title=Population structure and marker-trait association analysis of the US peanut (Arachis hypogaea L.) mini-core collection.;citation_pages=1307-1317;citation_volume=123;citation_year=2011;citation_issue=8;citation_pmid=21822942" />
<meta name="citation_reference" content="citation_journal_title=Biochemical genetics;citation_journal_abbrev=Biochem Genet;citation_author=M. L. Wang;citation_author=N. Barkley;citation_author=Z. Chen;citation_author=R. Pittman;citation_title=FAD2 gene mutations significantly alter fatty acid profiles in cultivated peanuts (Arachis hypogaea).;citation_pages=748-759;citation_volume=49;citation_year=2011;citation_issue=11-12;citation_pmid=21681577" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Wu;citation_author=C. Ren;citation_author=T. Joshi;citation_author=T. Vuong;citation_author=D. Xu;citation_title=SNP discovery by high-throughput sequencing in soybean.;citation_pages=469;citation_volume=11;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Zhang;citation_author=S. Liang;citation_author=J. Duan;citation_author=J. Wang;citation_author=S. Chen;citation_title=De novo assembly and characterisation of the transcriptome during seed development, and generation of genic-SSR markers in peanut (Arachis hypogaea L.).;citation_pages=90-90;citation_volume=13;citation_year=2012;citation_pmid=22409576;citation_doi=10.1186/1471-2164-13-90" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Transcriptome Sequencing of Diverse Peanut (Arachis) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="twitter:description" content="To test the hypothesis that the cultivated peanut species possesses almost no molecular variability, we sequenced a diverse panel of 22 Arachis accessions representing Arachis hypogaea botanical classes, A-, B-, and K- genome diploids, a synthetic amphidiploid, and a tetraploid wild species. RNASeq was performed on pools of three tissues, and de novo assembly was performed. Realignment of individual accession reads to transcripts of the cultivar OLin identified 306,820 biallelic SNPs. Among 10 naturally occurring tetraploid accessions, 40,382 unique homozygous SNPs were identified in 14,719 contigs. In eight diploid accessions, 291,115 unique SNPs were identified in 26,320 contigs. The average SNP rate among the 10 cultivated tetraploids was 0.5, and among eight diploids was 9.2 per 1000 bp. Diversity analysis indicated grouping of diploids according to genome classification, and cultivated tetraploids by subspecies. Cluster analysis of variants indicated that sequences of B genome species were the most similar to the tetraploids, and the next closest diploid accession belonged to the A genome species. A subset of 66 SNPs selected from the dataset was validated; of 782 SNP calls, 636 (81.32%) were confirmed using an allele-specific discrimination assay. We conclude that substantial genetic variability exists among wild species. Additionally, significant but lesser variability at the molecular level occurs among accessions of the cultivated species. This survey is the first to report significant SNP level diversity among transcripts, and may explain some of the phenotypic differences observed in germplasm surveys. Understanding SNP variants in the Arachis accessions will benefit in developing markers for selection." />
<meta name="og-title" property="og:title" content="Transcriptome Sequencing of Diverse Peanut (Arachis) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/12/3825" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="To test the hypothesis that the cultivated peanut species possesses almost no molecular variability, we sequenced a diverse panel of 22 Arachis accessions representing Arachis hypogaea botanical classes, A-, B-, and K- genome diploids, a synthetic amphidiploid, and a tetraploid wild species. RNASeq was performed on pools of three tissues, and de novo assembly was performed. Realignment of individual accession reads to transcripts of the cultivar OLin identified 306,820 biallelic SNPs. Among 10 naturally occurring tetraploid accessions, 40,382 unique homozygous SNPs were identified in 14,719 contigs. In eight diploid accessions, 291,115 unique SNPs were identified in 26,320 contigs. The average SNP rate among the 10 cultivated tetraploids was 0.5, and among eight diploids was 9.2 per 1000 bp. Diversity analysis indicated grouping of diploids according to genome classification, and cultivated tetraploids by subspecies. Cluster analysis of variants indicated that sequences of B genome species were the most similar to the tetraploids, and the next closest diploid accession belonged to the A genome species. A subset of 66 SNPs selected from the dataset was validated; of 782 SNP calls, 636 (81.32%) were confirmed using an allele-specific discrimination assay. We conclude that substantial genetic variability exists among wild species. Additionally, significant but lesser variability at the molecular level occurs among accessions of the cultivated species. This survey is the first to report significant SNP level diversity among transcripts, and may explain some of the phenotypic differences observed in germplasm surveys. Understanding SNP variants in the Arachis accessions will benefit in developing markers for selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
    <title>Transcriptome Sequencing of Diverse Peanut (Arachis) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ZtPI6yltzkTaloU_9lwKK0o-wSRaDrvI72aIpAYv01U__yCSGhNVuCbGf6KLL54AUS64uP6kQfk1f_X5fJJp6-FY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__36Yiqb2U0AhDmqVjDsvW59h3D9k7d0px_jM-NJ9exZg__uqIT76aUscmJwriA5WgUcByCR2EI-ARNdlG2HGF7Oec__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kKHrGW00gjhIrT0tHX2vru9SfmvS5SD8Z2whU-3sz68","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460508","apath":"\/ggg\/6\/12\/3825.atom","pisa":"ggg;6\/12\/3825","ac":{"\/ggg\/6\/12\/3825.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/12\/3825.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3825"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3825"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/12\/3825":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PC1cQxBbJsw6imkxuywYbTY457qBL8E4WG9OCAjscj4C0Tf92MPpC8cxJJZtfYJJMV9HZWgyeaAC8JJIQ1K0OloFrSooaMfZ0AOua9v0iowGuI7EylzKJ5qGchNINY6TFP5NhyBRVOdX0vHsbRTw6KhxduZ8YYd00BF95oRhcNlda8htpn1Cf+YCFfDzPYMMeC8LCkwkw6TCHf9mlxg+dJFGoJGWTGIzPr459+lvMoZv54DlW+pTAEPkbNwbLsPTnVV1shhDPL2qr6GScf6V8Bh9xtR1tGUd\\\\\\\/Oc8vKW2+0HAJR1m1Fsqyocz8lOc+C0LAlZXosHE0eD29ZwKAxyPsL+1IGXPsYKlv6o8dhdMVInhWD\\\\\\\/weSlQ\\\\\\\/SQPaMgoxue4Pj4zbBUS93oa1EKR4VsI6az+62N6PMaFaMwNEXuN8U8pKCup8rpIHaydbmDxh\\\\\\\/P24fp5ft0K7z2egVBV80UlpWojAKnfcAi3kowrFm3Aj9TJ4QK39F\\\\\\\/+MJPuWBZ0hgAbHuvanv\\\\\\\/MApITMrdIJHFv4CPHXTFYZDZ58zSp83LvtKNGls52ksnIXEQH8xrApAhoAQFwSZx9Y1rWT63H6uyc2IyGs4hhpGXwxMFreNiueyTx+uk5aTfTmsWXq6BxYGmB5V+ILPAq0Z30ZRiXX3Pw\\\\\\\/J+ORyhgtf7RtcUVpRRHJNZ2j+5lbf\\\\\\\/U7dYIFhlvRe23m7k4AmMrL6OGpgXko05FVVheamjE7SAZr1w+27WSJEMhUPsUEnQnD+m1bfXWOhIfdvMEXcw49eQax99iF9htTpbs7Vaj+3KzAiYGXFjHvIDkUd9O8Czmz8UJkxO8TwozYBQolCUG8FleBGsQHicvTFk3LmjFCkDbVEICJnK5qY83albtN6R+g0dtIak0zgp\\\\\\\/NfMBCDWmZ1svlul37U71C\\\\\\\/GKpIp4R5YBll7+79iIuUtgWpNTjgBiltHtghDUmm4CgIPI56ya4Jri2dV+NjWKhMYLxNv64ClUSGuLjr3JnxUi5RjstqA0f+FVvV7VhBEaAcSY81NIF7WafbceqWeWFIVYt7KiWaxLjGp+BO6gHSg5J9VWPKjkm9GRDaqT9zJs0Rox2IleWHxWOrsoif1FkguGsNBrcq9LNCfR7q0ERvPEsIWwAAQxIatEo6JlMJ3KPOguCCPkf4lcFgpM+OUU1Gj0vzzW3CNclANPFXt1qV\\\\\\\/pfCSDzwhiQeOf99TwPbQS0siPVbgBnMHpcPa2gZ2OD9daiJEgzT5YqTscCab0F5Hp3PUOC4Wf8HG\\\\\\\/nJ4nkByH\\\u0022,\\\u0022iv\\\u0022:\\\u0022leNdei2y2I6xSUiGeqZL2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002257fa407680a51ff9eae96724c97fb61bc8669485ff6ac3620c0b65033914ba91\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dvvNgaKuAxoPutqPgjJkorwlY9ned69LNcWo8koYULOUqfpOE0GBwt99il4PgRtNWvKro8mLvbfSamPvha0X32M2+MWZY\\\\\\\/KYBVrg\\\\\\\/eYxx3GN48kWid6P1zDsEHX3TBj0TgPd6\\\\\\\/ugkWHXKj6OwTPbjOoM0PXmgK03pfwgJKpdkRRV4kcXA1Laah8vYrU9zrQW1X\\\\\\\/WLL2U4hD30yLaheC4VKdaRIinexgO7YvD2oTBikYHjqGgrwyxlTb+\\\\\\\/HwqU7mR6YstzERJjWP\\\\\\\/OmFh9zgOXZPJ5Sj0BXPkdsXFxBVS9CuF2Z04pDTNklQ+PlNr0xT0A3aiZwIq+7UpHtsTPMmU1u3KMUQWjvjFhBUuN8dJbhg5G94M4aprFaUck3Or7F4W4lQuhdzQI3MUa4dIMMQtxm5Ntjlfhd8Q3qTlXrG+g3kc7r6ZmxN7NrVPjBM5qRJo5sErKSHqzXPpiMlv470veCZlb6RV22sR0OvCn+sJ01tqvZe09H1q7E6f+Md2pVSbTFn6YngerRe7mnmc4BLkSF1R6ScSixhMAV3a4HXnGot9MBpWpaSyoCOJynHUs2Bomlw6NlgaLfsZIvTVFXgBp\\\\\\\/GFyAvFvgA2h0iMAVU1QKy\\\\\\\/+P4mlIgIUprRXSTuT6QvDTRy85l0B1fqLzVCvqNtjAxGXdMnhpnNvGcAcajVZ6ml5XFGYkSKvBUDJEa\\\\\\\/\\\\\\\/zv27qdCnnsC+rh0UXmRitRpjAhpmJwPvAZkZgnt4WsEwI+eoLwgzNPnQWFygHEF4hdDC+l793oyBCtFxRZI9imjb14rPfpgFeTMJyBV5UxVrvCdnBznT91JhRUeCEd10ajI+KF3U7BHf+N2yWNI1lkEY2cdyaFLNfegOYbbEF95jSZF+x\\\\\\\/ojM\\\\\\\/p22aWfeXWcVfH5q7eZbe8D3Ox82\\\\\\\/\\\\\\\/pTq0\\\\\\\/oeUPPBWNOXRgeLhKLpOK9QcpBXqlFM4yla3Znx1c7nTakEPamWtljA1sZ3CV69Sw73IDDzF0iOJd90yRSwNHC3KOzGFgFYeAEpVNYFl08SSyXMz0vczZQBi2FHrvbdiU4APnWR7n3wgWZ7M\\\\\\\/w0NvN8NGG4c5bbr\\\\\\\/llEXBzvZlxQDYkKgPKtsYEGWmPYf2N2pmSmV2z+4IyTdpM9N2vqIDJxqKv5ALaBmDyU\\\\\\\/EVXAp3hZduoPu4MV0ux5DDFkjsYF1olAPFhdlRlIbZPOV2ch50aS3ajOVVyTpry8TDkN8sc97c59tcr6jgpput0I\\\\\\\/N2XYX8I65CUYy4b9\\\\\\\/rXr8XWLQOiim5NKhGy\\\\\\\/ZiAQSixvAOowEiVYzUdz8T65cpLzER6YERmZ8QMC8N0Ow6qysQdHyneIE9IAviZpYQRR5\\\\\\\/kVS3MMNRnfP8aPIj3sknYKXWrITfk2x00x2d\\\\\\\/Vp9AG9jCVUz1AZkfizGkID1nPUEXUyIFtPUXWLKBjAZQNmWyMoK\\\\\\\/cjq4BhTRVfnAV2XjMms7cWIIOux3cRD2ahcfPwgCVvekloB+3sAOBQB5yaNd4bgm3OaVf+\\\\\\\/wjM0XHb6jf5iwnZnJwaWENRcgDiwfuDN9LCMLWoAZ19HWfbjlmUttP0y+u1WwcjdekFwYX6i8fzn4f2P79rIfG2Fzw7oM3IkqxXy9u9CVjkn6BYNfgTRt16XKRInsGOK4Jcl6MtbSQ\\\\\\\/d0+W9\\\\\\\/+pu9OxnOW49IxjkSvESsRg4z0kr4macikRbMQ6i2KYEGq70WtA3Gr2FjUtP15baSDRf0KV\\\\\\\/PtUH+dV6a2gwknbtA1ZD00saSS+a+kM70yAdaIYERebW3zajLUednoXqxB\\\\\\\/yQnl2QXNqYMbVJFUmppi\\\\\\\/1B7FpDd18RJXSiQOdHVOrvxu4l5d7nx\\\\\\\/VMQimz1nIK7xuZWUH8I6Pi9Po3NoTJq9lkXl2DbELXrnciA1uywNCOacHmKlmpmudQfIpPWlWtpVEbsxeR8phfFRjlRd8QDEYR0dnrp3jDeiP5SS7MZyCjiW1VXVVjXo4WMPy331czCpjIlQ+1d5w49Tp5ynM07oOYjVCFMEbqoRhGs8B+WpDiZ6sQMefSOkE6YqkdRW0JZZWZpLaYsVeQI8Ici\\\\\\\/qzX4outKtcXzGJ5lOFVlBAelwEq5kNkNMlmfR80Ao1aWx1YSeAFo\\\\\\\/d4YX272dRF507D2lnp\\\\\\\/WB8ATJ7zCAfsklv2AhYdw0auXSgRCBGX4fEnbjx\\\\\\\/2ghiUpTnogf3OaO58meS99Ew2nC7j+t7el714gPNyqe\\\\\\\/20KyEQpTJMFH29TUYBSKD5cMEL7jeaiZC3i4526UllZhW\\\\\\\/RBRwKBeO\\\\\\\/zHNcXbHgdGB0B+jsJAet0AWQKJ1eVNet1gkc7ZmKY7z\\\\\\\/YlTy3oQkks22rt1FKoxnT7YzbRCsni7LBjRM4RTNrl1x67qDK+UkhSnfnz6CODNUf6TLZWX8txgiMnxNAj1xlI1rcE3CGlp+IRcNRPHyK07y6sQkyWUo5AGef+VVVwYZ843vzSNh9lB+SHiiSfi4M0mmKFp2Pr5jpyZVtyD0d+bKoOMyuBRC03gDl\\\\\\\/40epmRCdUFLiyII7pDqrVcARr3GinZedSSWVjKOsY5vMaCl4tFbADn5Uk4gBMgDOnJJ5Oj4S2vwRq79NfklJZnEqGbM5ohke0bLsyrNiNihm8Z6dZfHlZGcRd1AG\\\\\\\/w7\\\\\\\/LD21digaUVZkvDTqH0hAUQ8yDoMWQIc7GAvhp9DEnOtvrHbk2dTqaTyq5yWyAL7juCBD6pE\\\\\\\/dovBCyDSSkx3Y7crFDt8peMa62vdvKgWkI9QjKCFLSx9mWGGMAXXX\\\\\\\/KSXPbBnjY0w1zOcVdMLwtCaWjqXsfIJOkO0pf3cb6+\\\\\\\/j\\\\\\\/azOYIm7DAEnDQsSN2LFTTsZnRhBcIr8QHKNX0xvE8AZyBNy0djIolYZFONU6aaWV668ASLPilwDlQDoQiDA9cFd1JnBaiRBL\\\\\\\/xu92DpCz73VhZL\\\\\\\/xaAgSnI4RRPbnLp50lC91EcXLOQGFA7kdnGS\\\\\\\/AQcfGKVLETzC51M7OQKnL7hgN\\\\\\\/ofvAy5oEs1XgEBD64MExvVsdPQVkIGAOR6cImYEyI6dTPELwoRFTzew7u7\\\\\\\/\\\\\\\/LVNj8u9Pn5mjxLUdJGXL5YG9XB7EUyg+OcNBieFdpn45D9ywjArfq0Cv7RvBNp+inRGnEQOuabTtDBD774IaiDDV2PDN5rZ4g0ks90YLS4I1\\\\\\\/wyPluXEqJWX9qkvowa94bSJVmTpvw+t9E9i7MMus26TkHKHO+nD4huGi2Mjn5WuYCC1DvfjtxNBXY2hk5FvTjw1N90Yz73rGXtxGUoytYh5vMBExMi\\\\\\\/e9JJr+RmJxI4vKQHwTqgrNJrcBK8OB5YFWvLphPURNarRF6OU1yiNSDMbjZQvYVX7jLAmAIG06oMODrQpl02XSmW7rBdyItxEFjC81iT52IZ0zoYYBOyjjreGfjEseetJd760KQe0xsyLptJ\\\\\\\/kRTGQt9wQpYbyWzpRH\\\\\\\/souqZFLdUT9yA9sA2zpIfWcQ6mR0tMqU5z3oRZb6xcSa3wO1sK70eoYS6CVjIXouJdz+NocHNQufVR+B2xSN8ANZuLSNbIMdM62p5Sf2n4q8QOoCREM6ZA\\\\\\\/0sZO96em0Qg3ciS3zp5nUzjypyLY1WOmoSSLlJujOgwbVAmRjZkC9ZlnfX5mZoL\\\\\\\/mdGftPJdTR6qZ\\\\\\\/0+ejZAmmFZwYYU5iW2GTgTMOizKjxyrbmialQvSBCP9YGi9K0DoFYp45gkpVZBKEwqst4e3m0cD+1hc9eEFCZOK4NII47uXToqCIgKM6ng+efjvCnbK2sdLW\\\\\\\/pXBG+aVR8Ij0ec8obI0Ymdri\\\\\\\/auYsUMMzeg5lqnFodb7fzNMw76XIN91ysDtNcolcI3jUTLdHRqV7\\\\\\\/jHUoNa\\\\\\\/E3vQx6NWDYYdzvsxA693Qr1thB1jEpKb56QWHjp9pSNs908750zJKA\\\\\\\/JAI6VgaE6E9aVyQDS5AvFqimKQ7r1Xnlpka7VYZJTmooeEn+CPMOopInbg7flPS2R2NfSY3HLmDJLiKH0Yxqj7smugZBKWtZyDDgTyFmreOoF2C3hvIzKf6hi5v7ouReZ4tcup6IMQySrMx9\\\\\\\/7IjdRtBm8McQwP3JqnfKNDm5ZIEMH\\\\\\\/omh313nqNcKStpEtpLTzZjM9iepKk+LvsQiz8dbaEOQXQgbUJlpiRvZQptmlzgxsOfBQHKL2x1GcHX\\\\\\\/HQRAKo0JYGwx50qkx\\\\\\\/sNpKpsjVl\\\\\\\/f481CkN14DR6hQpPakkXNHGhSbfWSdNq5584NLwejkq751X9wtjRdC0mo707alJzb0\\\\\\\/Fha7svW2VVWoA9m7aJshGbmqfaMAxxDP7QDqPCrSd7RdcUkxbRUjVUEMSPDep6hEZipLpchyJfGA3l5yUX7zjI\\\\\\\/SVH0hjvnNvcLRovrYLnfJ0p28zurgvwNf\\\\\\\/CjhJEweJUMjv8gengz\\\\\\\/S9+h5cHc9bZ+h\\\\\\\/01r86\\\\\\\/zAiVBcYQh11eBYuHdqQtcZQri7FQ19DBkUjd8LzP2V\\\\\\\/D1pjYiisB5CVxiL0bJpb8ogYOTX4utHKoSU5BG2TcIB3zUrVzjvYuiEWzsV6ZTK3ZM8h6dzdNEocLEGw9EGdtm8k2bIW\\\\\\\/tPPayuq2rNJ7TtJtPNI17nY0GoKH4vC8cLVPk1tatvY9+Z0uRts8Be9FslHwKr8lUvYbV2izVb\\\\\\\/V1olSxcC+Jb+VKbp2fb1kiRZ4+iP1JI2qvTPb8OObQ6vJLPQbw5OTBtpCZBUK415Ort6IR+2NSUGas9jcOzrp\\\\\\\/WUcGuXCHd+xVKx9GOjxJiQC4a1E6A4tqIQq\\\\\\\/liupm1Gv8O1YGN9d1ua2Z6f4JqL47Av4oLrgJ1b3xIwXmlQZifdCe17V1B0wqRs+lj33o9aczt4OhxtIbXi2aCzYtbOInYyX2ZHOQafy4LGi3QTld7SFrG410oU96bBvWE3VRyGa8UuCtalfRhzxh30bGIIwDcX\\\\\\\/sQcrhJXFZ8n5wFi01hv\\\\\\\/VeuNlLYuXQWA2vB8j4ZE913dpz9hcUYaX4nGXavyvwn2YQfeRIAFXS29kkB8k5vte+2daUZYGi8E70GCXcsUroWouUspaxT8Yh77XXLGynW\\\\\\\/TNlIh9NkoEQLH+C1ahFoIXu2gWMfWTfNKp65HY0nDJAvthKCCD6G0j7yap2sdmuXUYaynGOlX3tQQafhbhr40Y5bLPNCg5jqDqzt2MtfTw\\\\\\\/it7tIk4wzHNAHdUNpQOW10LyvCZqnQntMvvyDuMfdBN8l6FAj9OfEKi2yVF7PLw3cq5H8p7tmISEog5G2n7VuZke0wMkC3rf+RucN5r\\\\\\\/d2vCPFDnkrpZFrSe9Gwv0E7dQ6EUEs27DbmlGG2Jze8ZTZvjHWCFQ9hjhmunj0b3sVrkLNRi2HbZ2dcIuG9Y9ueQoXvKgpFnheDVKJ\\\\\\\/lOaTffsmn+t3P8Tt0EiKqWI2BfmkGtBBAtDTJO71NafOY9Y+dqPeZ4Lz1829irmRuhkWKYSB\\\\\\\/H6fyJGE6oCmaWAjxzDRTbAlw4l30tBwc3tZ5zPREk1bLnGXMc2HdEVU6k5E752KU6Sz4jSjLPmNu327sMym+sQZT5QBjx4BBsQlFpRr5gAHe1\\\\\\\/U9UDPysGYUocmsPWE+x2Qb8UXlPyo0r9qYTOeH1BusOiEmPjCB6L7UUwP5sNI\\\\\\\/fhVt4rHUtOpeRfLC7cxssyJENdHX+1AY1CvDTrvf+hpk2ofTdvKFmRnFc9wjRCsGtAtL4qty79B2yaRWrKaZa7LWUPERQzx3GmjwZBKY6dCKkW6wocT\\\\\\\/tgKnBHzYYPkOgktZR03H6adVomyX0VWAWH5DQ88jpiAKl2SbOH3pciM6nPlwZYeEPnxcaZfXbhxahyFhw2fz4D8uMTkP9A2BM1e4CSl5eAe8pUgDkgksTIfYHT0F+G8OFuYpBg9lQVgOeySoi\\\\\\\/BAtK9r\\\\\\\/X6nI4vF3x4QIVDWHe66+jk0HSaMgeV04+DUReNeKZuf65zZAphxZGMnIzV2Z8zDcFMs\\\\\\\/+Kt7UDKRFQUOtyc4frV+pgtAH5GNKorrvDVq\\\\\\\/yHza96jCs4l4Y6kiR7lb17Ak\\\\\\\/8jWOTLFHDB5Tn6kw39w3ixosh0XUw9NNZZ7gaANvGZTaylASzH8Qe7VtPUQpZ83rv7gLI82URQFVCYLVu+pyFvnJeIuCyUluB761ol4dAZJ+UaPvOKa1AqrpVgsLJu\\\\\\\/Wqnn10BGd3aF7F\\\\\\\/Xk0SjSVSGZixfkFd8HSvirLZVLvkDG6G48x3uIGmJgEywUatxnZcfK4sK7tWvYc76kk25pMBCcdn\\\\\\\/n53DG4PXOPFtCWDa1az06fGcT092\\\\\\\/Idm\\\\\\\/+CrqlTjd78V09VrBpDDpQagV8MK7d\\\\\\\/VlQJRIwkmBpPjPr2z+0Asmao4SFqqKh0S0ZEh3FcF8CVsIgoFE1PzaTmWh8A8zBFAzV7MQw2PDbLX02PaBqKtmjyceeDSKzt\\\\\\\/eQD5aCMsFdZX8pZLv2D\\\\\\\/bn3DCoj3jJMMkutvLD4eX6wSIdAocaY1cxEG6\\\\\\\/l5jHuCaK0hT95KPvhwwrguB7jWgxOQGGG\\\\\\\/0dSI50cgktdVGsg0w0lkotp5C8wvnPfZZKUfl1kaBdRfPxsWA\\\\\\\/DSb\\\\\\\/74UVDLEy9sCUcQ3f\\\\\\\/JZ8yxZYoSW+0p5l6q5jW4m9a0LHZ8IJfNFZOGOjtuyE7CGoy70NPNBK1PySltu1rbMeR8WCbvBgKOEHwDS9buHsfgArjUzBazzQjE0lZS\\\\\\\/0V3SbvetoWMFuLoDWxb\\\\\\\/U9El7048SXDgRdYVh66tGjTMzVfltYJRNPcLfWbsik8mO6VSHBHuYn1kqqzp6HuvtvhWPRbyZ14145U25y1ZFAHhRk97wIFs1f3tbPNgDr3NQG38n+VeAkbC8BWlBnWKVU25HcX36KTJuNXJofysw0BkVB11dA9bf7hkdu6Mn0f3LJJyW+a8xzw0ZOr6yR\\\\\\\/chmQ4Vk3LAinCbh0Fl7rfiDKo29PatYr+kx2i2vk8NnsfKrGjI\\\\\\\/cs6iwDWWqL\\\\\\\/7uKGtFRCh54LBBzGUMtGvXPPKWxAlMK047n5El0G1t0jzfxWvuspAfuFL\\\\\\\/VhXyuC9LOC6byhcfSMcwiiWYo0HWMwc61FeU5CDvSxkxiEG1jTtZhAZnH8Mj7fcFPflBjmXbHj9wGEUHohKwYhIKScQsKK+6C90weYbojWJs6e0tVj1Ja+n68ubZtVMiiNx7ptrADJ6rb2Gcv8TNgi1mUN8YLQSlDD0ez1AFEv995s9c4qZv8yi1c5C9Cd4gZM2LzNHdC1lptuARBVFNZKbebi24mgdTHZeYZYED5fYzXin9vEDkRyQc\\\\\\\/CaAy+sThIMSMnQ218u0MNTCU44ItWXRD95TGsq2+pv\\\\\\\/8\\\\\\\/3Um3xmR2ybLw3fMIBBU\\\\\\\/C9jWQiTJYW8ape9UmftxP+kt7i1atRuAg5S1SziW89eC5fs4Zfw+V26fmhQzyGu3CWS7NhV3yf5L0XUZULvE9waRvTn1uoebq3+LOqj3N9+Ldn9PAmh4wYtO37+R75HCntLx28kQasm\\\\\\\/lIGs9rfWc6hc3tuSapl3Y7SGSN0+c66zqdXASMEg93L6zpdOyhyC3LgG7w5Ko7p4fLUsUwXIQzHJJJKzplTMyVsYZIUyp8jJ0L\\\\\\\/5fWXcr3U7LtqjTe0VYObHBeROhfTTToxs03qEGmg3jgHNaeBixeS8kyVv4wE67wSHuZHAYsKU+CdH4zym31PCQPNfGjWx2bEfy+78o0\\\\\\\/D+\\\\\\\/dTtrz3Qea9vvYmViJo34WD3VX0JImlQeJ3f0SU1d+Ihhm+mbbY\\\\\\\/BZBGc5NyCmORYtSat0vYEE2jJWdrHcE4S4PG5xqbbZNb\\\\\\\/TOorEWHNmUTdfPKUBpD1PWqZD+Rcz5bGLJ6tgXfNXM6CXS\\\\\\\/dnEmPiB5tk3MumCuQyz8e1h189DZRHqioepU1bhork5QMYc8U1DrtWd8a\\\\\\\/8A9G4dm1a8JJRComrPPkC0mJ4IkAwiZ3hiXvB6hFXkMP2AK1pdx3xbkzgJzsXMq0VcxtGMtZS7Xrs1\\\\\\\/ijnxwc7dsQZtg2GdPpwGIA8LlfsUYUUuol36s224SlbU+2QEQYP8z10Ho\\\\\\\/UUjQnygbc0tam+qCzzFvQ1jrYV5PgOzVfQhbRuYzDkLxgueKgrQ5K\\\\\\\/cwLbPHu9NDlRqQPTAO\\\\\\\/DHEHQXKXwtCYHpsMmqucM1kZSvfeg7Z2NOW9B+P30jK+c2qAAHAPm+K6oWUW1W+SpxMtU2x+\\\\\\\/NHwGGuVdPXiE6tYDJPpAGUGu\\\\\\\/N6y0Ba5qtWB3N2sFXCJ094tdwC6qnWIzjkvvOa2cw5sZcK956L3bnhfI3lpXBRww4f\\\\\\\/OpmW2JtD8Ze8Any9zExWgKpS+exDU\\\\\\\/atXkr946IsDTLgwT0au6ojmUAsqsdKfbkj8yUdpDZoARd6UYmRLKO7LphT6+jHKHC\\\\\\\/M4md1pxHbB0pBCCC4KSxqkHhRc5ppvPYsl3Z9jy\\\\\\\/XvRHSbGJ0yxtbxgz3iGi1adM4Ka2SEjlAOEshWP4t2R0SbhLftuuhiytZ9WcBHmGNWiVXivhYvE9990lMn\\\\\\\/2sbolm9zWFjsyLYotpkpOwmASxQ2JHC+LtXwfRXUv8OAzdJBYf07LD0ESswhAQnWwrgnEt8vKFZuW1oz8ZY1C6\\\\\\\/UMKlfkSr8BUx7\\\\\\\/pOdy90BLufugrEkLDR7Yk8Wda0Mlys9JUizKFEdzCf\\\\\\\/+QM5c46uZZntoMgYGXPRjU9UvO5ERbsI0vIbvd6OsQ16zN\\\\\\\/VZ0\\\\\\\/94YlBfMq2QT4VPonOQ54+8CppK4RaCm3EBYy7h9P5FkD7dfcngcJgAXWhQUhBvhIZaEb7NAVLAJ4rUjv1DOksiuaW55X8opbu\\\\\\\/s7XZc3G+sASmWB6MbKwBcs9fPwc2adZ2WRWNHYeqcvzXQPmfQMdSenf0Yy3LGuED3rW73Xxp0We8Fkeml8WTMkG5G+gAtCQu\\\\\\\/bYM2qzM2FKVH8c8C7668NnNaTxeC5CoojI26O7st2BLVp+V9ByFadTSi8eqvN7iETxx2MaUYrlJlX5U+M4ADC+hCHrhpJwVLI9ZpiuHeYjkwZXgNNqLXT+GDQnsQMnuxgX\\\\\\\/dpyZNqXOJKTlPobCPhdSXtosjSs8UWWkJ6Yxdv7r7oGgilYHLhFuNMgGnevFMq+ILUD9hOjdiVi3agbD1n03mVNK9sKwG34Wi\\\\\\\/B3wCy+O5Zg2PjRt+AN\\\\\\\/s6Rk8o36ndw1I8WbMn\\\\\\\/UyE62xfxxdiny3BuM6e2FsAWWhgtX5mFt0bBzfCan9QGCOmshB0glDw0s\\\\\\\/aJoa4UCdBOralqYUER1fDdKBKAiVGBwLz0lP+J7w+IqAcldIx+eo511T3zEb7Zl23IFvabngDF65+xneAVI9K7XQjMB1J8uFpKFuEWQ8r8iQvbe2h3iejyICCaX+KlHokLBVlIgfBpcEMEK720dm7NwZMWUD5BZ8qv\\\\\\\/Ed2lGAnesKyxPmPphRdtSAH5YBoHH+9ukByxcyEoZemJL1Yf5DcKQ\\\\\\\/0RVsraFzwbV\\\\\\\/trCrTmukSCXja7QNc6GM6RdORlm9wbtwwzA\\\\\\\/\\\\\\\/fj47oLJP4EKukCi9LskLTaHggm1\\\\\\\/AoP1FyRVrCbzebvKTODc1wkG15AGl1JqSfg\\\\\\\/MankR8fBrRoSSzxJv0GaBK6WV1DIARoF71IfXRX9l\\\\\\\/0Tb8cxg1IULmRR8v9g26ZtxXhttEhzCIcBBraQqvnT2PJyijiOxKE6YIGwwHmUQuU55tMWwy2lcCjwe4DiAjcp2UiroWkRfbt9jdgazoeuH+zihsTwFyAjVJqziWNOFxIabEcL7SMpxctN0lbd42NEo4gR+LgLVPq9f5tGy1FyIZs7u3Ch3IYaeB7v3MyopM7lN6fDinjm8tP2yjya3jY6fVZMRCCWWkfSAIBZdcanKCOwjDBgmD7HeUIA\\\\\\\/j+dcOPCfD4l9s71T2Of9PBdL8Cmbhk1ppPCK3KzukhGMjqTPjKVITv2rJdUdA6lNtaRU2DPQJIfSoYZqKKyQ2StE6zUU8sO4fgX3M6jbVcfWeYZfukwmyxTm0mwiTmNGXl\\\\\\\/99rt\\\\\\\/HVTs6D+3hVoVoKnhnVe3TlvKbpjrFUFqgvjnSqSkWRqamFHuqeXtyThqcUAqqz4P0EmLz2O3Hje8cd5Iy54CG5F6oSP2oSwGHDpdhgHjEPKFoeK+skB\\\\\\\/aG\\\\\\\/VopT2O+AnLLYYfeX8uPR\\\\\\\/S0HyYKiGhGEHw23rjT7YG8NH1RhRxsqap10Vs7UO+5eLP0E6TAaHswGonuAUTN9U3joswpJMtAyC9rKUoa7wnoPndg4rrSMg5oy3YLSdDuGH0bBRRirW1qceO7uXx6EhsBxaIoXINPsjD\\\\\\\/Q5eGfA\\\\\\\/JBkp02PXCPsMdlE08esdFwDE\\\\\\\/R5un9dtbG1Uoz6IVZ2bQB\\\\\\\/YP38ttPms1dA+ZaFjbIu\\\\\\\/\\\\\\\/K+OKjuRGAkzxBwtCVa\\\\\\\/QymNOSm5G+QThYwxVnEw\\\\\\\/Elv9hncqLYH7zPhmjS9H7tJf0Js86v+gv1nihVqNr9UonZ5TbW6yG1pZ\\\\\\\/2WOsX6i+mb7ZEGG\\\\\\\/KEBCsnvzwA8nI4Emf8c9AmfstgQxFhmwaZNVKo+g2SD2Rcx57fsooXWzmbGTWy1viiOVXizIpiVzu3Ux+Bjalta9uKvkjStEMCiqCN41EuUntnivnK+IWskn8M1VizcibIlVdGQJ6Ip1NGJpiMubipfVSHRo20O8Ts\\\\\\\/gyEGoGp\\\\\\\/nZ2N3B\\\\\\\/O375G4fMy18PjQA7PoLszq1o+x09HLMbtjMGi6Oxt8uUJ9T+m2FH\\\\\\\/s6EH7E5G6kE\\\\\\\/Cr9pyAf+Kx9lKtc7biDBfgBEsHvN4C5D\\\\\\\/uj24x\\\\\\\/zWz88\\\\\\\/8zZjYsoKFMOby1TFDmwr526uNOvU0U61TBLY8RcVkJJS1rzNg3nIvIusEHgGXOx9vFtOnmpnwfP7dP+mkyl4XP48Kq5jZiMknb4p+JbAFzxalq8ABfvYLAE1K1ehTNrTgk1ab4PD\\\\\\\/eTr\\\\\\\/wSeMtl6qaKa+naYDIq0cqpdcdOibGJ58hodVuEhtvZitK04e1aL6GhVfUQbG7rPUU6tyHsrm3+R0PKTLE7Esm0fpwn2ghjQ8JxfW+vVh9xz\\\\\\\/nrtNG2PP0JMhniHnSs1mZJmEQLSOYVxC5IHWVuvloA80mCT610\\\\\\\/UcvSdzzs4++7gc59CnIi+Uw2GXJcZMVnvAciLn0K90t3NWQjwoTfLKcmYnkBarC60xh7YQHQQs5fndPc1dNdjveLbj2ds3BHNjIhbr5c\\\\\\\/4TTJrwwIZG7xZz0o\\\\\\\/DQ6Gi01fT9Rn42SOvNmWPeo8gmgFRRicB02ieZjT7lox7Ky9dOqnPxgO28P8TQMc7N4opmD8epcEcWFvXnpZnwtocIvMrD1lRdfdm+ymE23qsli4vIP\\\\\\\/P7PETEvP\\\\\\\/+My\\\\\\\/LadiGAHY+Ol9QN4V0rsBw6IEQGxB5CY9Fvti4qni7SAYL0adZEjr8vSsaHhqqgth6x2gmRgLAzjmXe3T9RvGlyxdu9tzZnJh1dfmu7cwtM\\\\\\\/kVSxxuoGPkHlQjZchVdG0ZLUv0scPY\\\\\\\/6jq609cRw6KCrD0ZdJ8svhA50WxpzZYrF77nS8WHBeGzam\\\\\\\/PTrkDsEBewapYOPkTkDDOaSvfIfrYrMEkxkPNolZCBU7iROwk0ybNoLFk3zh8T7EU8PzPqexxE+avyFQcQFlfR09uZV59qMEvDXNAo36rjHW5dWl+J3UcQKr2yoZXcavGFNVpWjIeCSpzQiQ6czsEU+nF11y4O56BrA5leP4ha7rqgyAaZFrSKYfFpS3sScMVFqdpx5\\\\\\\/aFTonJ1X6DB\\\\\\\/hSNHxwfl7mYGf9+nBIN3GYormy3ihY9gHJZ1+s7PIJ6JbqygrfCanB+VW4uhI+1h\\\\\\\/i9sMnTCiVj1gTFW3ug9QuQLm0X8R568yQ8VD7q9zVW8YMQfAg0ZE7oIvcqtvojkRsNf9jIAtb9Ml9DGPJzv3KODKFK4cPyuaNvW+BqTesghkNZpd6f1jPPl5vRgYl9WsD7FgydIERbc2fRSOCN5qw115cYPPpDLEZVmRboonrA0xDLqmpgqhnHj\\\\\\\/4VmAaBqboOw2TL70BTBcVrf3sB7UJVEC0jQr06R33Lz9AJvsvU016lK872cxwvuccya6Im1osT+qmlhFvOS5CPbRwnSsczE0LgsHkOhaZu9mlyQaMjGtu1UOvIydurYfeSPeTHR8WdlHg7nDUKC\\\\\\\/bBjgEMbrCqDja7ctPfQs0RvdUvSd\\\\\\\/wtH5eFbpJauU0smRugRXgFe8fZOmaYjo60YQ\\\\\\\/5GvIgeS+TmALoCXnbHg0eRblQAPgK+d5BD8elGElgVHYhAS4cklrwz29LmSrHfuSG1mlFYa8xbzleTJ9eCSBHxWKq1Ur98vlfwfYAusuFVkyORgAem2osOhjiFx6H0bRSZeudeq+DqqU6miUinm\\\\\\\/316jDIYGuRJdrTELgUvC5xkHhwuvt1wnKNBZNaZVyqWVto+Qwi4y3y80FA5KotM2dDzIEXTpDmFgC3bxE0MuEZhAqQ0ZAAdrOA+uO5mgJzJK4soxcmNiLQ\\\\\\\/S4MTxoY72ey919BVmaoSZTdip4TyYtbrEPkm44nf1qnIE+3GMBuynqJfr7SEJwk+zc6AUWiDHhkgKi2zn6NhGer1HPIj0p6nnlE4KriQ7lO0rRTDG8d6JEl5ma7+mKvTg11nSLg0Dn5ryPOfLiqKk21UuTbEewdqIuPQzDndoD5EPshE0iJM0UxCWjQsAQAkdhT9mozLoUxpAD+Yerb9ek8CVLMz\\\\\\\/y3uhiVC1HZdkX2Gb5ua\\\\\\\/U0f9\\\\\\\/KYlpXzJBh9X84gnVO795wBMWSd+J5VWcPAakNEAneW06IzlqjBHxB6nSx0Y9l2ixIH6cuqxMwWoojzt0i5EAZ0klKpfoiUplDDz+fJ5MB7fND9ekf0RxlvtGpOkFYMdjC9w96Kk5S51ziwVlDcmICMqik3Z46cdpmgk\\\\\\\/m+7jtoAdsE8\\\\\\\/x3Qo76yxA6L52vlaODnhpXnhKxJToCZu6cMbnL6B\\\\\\\/4YGjzRmpfUSAMS1geFQts74Hq6v68RWzW3jcqMIvmwX4Wnt\\\\\\\/3Mw2Qu7ToXDdJnIlMjN5qazfvMNKWYOQq1jqXH1cN452KnoqaYckikeSHQ4UFlqPOw\\\\\\\/qKcg0EoWkHO4Kx4qJPQ4WSZMfm2hLevk9Yk4Z0cEsed4PUFx6+tqJQMDzBEzPYDr7V+s1XEDkMjsGMG4qyXJYxTLLTkgdm4VNCCiG9gZcn7rNtQmT7y0A6NMqjL21arDOTEceS84+Zru4CjlrKdVojVEIv35Z9KbiKTkCiaIZ+83F+VaRexuog0uYmZwwMfk34LRc\\\\\\\/RgJJrolu5AVeUER2JC2UpLsMnjDsnXTmW6fVmrccu+k9+gsW4LII8veCrRbckrmh9PGN1FrsVzQIuuipQXlyx\\\\\\\/B+XnMXxMlv8nOtpN3H47kDzEfgkakYKbkDXgEMcNk37wA159l4q+a9rG8EbjPN1\\\\\\\/8wzLOaTo7AF+tOvx2zMHuZQ+9GGlzf+9R5wWwD99g6ZQC\\\\\\\/9bHxI1\\\\\\\/z\\\\\\\/jy3+2KJjxK7yL5C65BLBEqLe8QBry9l3b\\\\\\\/vRADKmIN7PHEvQ0ajIAHutGr6Vw5yixiHAL8dp503C\\\\\\\/tS3g1F6rgPYwL6uoyDl9SIpP\\\\\\\/HMtTJYp1xhiFlC7HQzgufmWAK3rtn7xaj2akyc2wI0Jx6xei7U7s9e9ikJr7Wi2EfXOPIwFO+i4Avsxg9aHb+iHJuwFia3sw+TYtZ1wCTwsNxVJ6sz8kkagdyMBPIH83ArPm6AWnw7SYdHYyWLWRC5GkytdMKxewA4tG8tnwqu528O0JjWLzlcCj2jsdqUlWM3VsQjO3UaywuLErzxF80NyLW693o5CoO2e9jMp8yKTLjzv3RDY6ikh+wfNXYfhbXPaGUKQSMk4OodKnRmh++dZPCBHcMnjg+MSaoU36JggdGsDZSF0v\\\\\\\/+3Wy0qNRlazRMs5rSMtt001L9Z1Ar4hWd8UyhU\\\\\\\/ecjwHThRc+RVA95WB5VmhrcPiJne\\\\\\\/+kqBhrLQMdqmyVMG6Qt9BeT0oaOQWtCT4Edhl73WZPF41cn4PTNA+eorAyvsELtsRY9uWDxCTGVHTAmhdiJ4F6uAdvUH73nABZ0NGWp6aGNVkdYZ1WXvak2q7q8yXn++OQrTNgk0JL9bmnY0IQS1pyBAg6Gemo8e1hyTBXl1+dKAL3m1\\\\\\\/V7wvAiEJvhpdeuhh6cOHL6bt\\\\\\\/iY36eCZRzCNT8XoRc1e9bkiIh10xTQntpRm5QUGBn7PX3ExvFSqPOkInZJ5OaAUgGQ1GVL5TOaymX3em44YxIx9nNYiKaPFVnWW\\\\\\\/oWHJ5Axj0QJ0JFQapKGN4qd+lYOrTC5YABD3hpv\\\\\\\/pIxiiteLaAouE2j8GfjC98Mu3ck4V\\\\\\\/s+bZiEfF6vJVu5VKIFhjygw6VNMS45tMtkeLmUOT1CBiITHDdoRQnAFUdukyaEcmq3hQZXpmHeRA6d+5LC0VnMC2VSXm\\\\\\\/WY2ajfHkOhgyhyC7ykuTDYHbXhPuTMQBGo3pcvUlrNEe\\\\\\\/CNYimnDmy18X6YNw\\\\\\\/JksGOMkJUt+HRLuEbIEGt8Z+yfTSqL5MqFI8JelLM7ju8iYfGywkQMcTOo7aEriLNtv2GDNrpKw21qYSn3ZkbmjcJki1z+SfxG0DElYlucqDpO2EuC2u3ZjWsLE7vPBz0Ssch1hNNAwP6pETrU5S\\\\\\\/YnBVcobmo4now0ZL8aL4NszuBd9tcx58QZBAGNFzMwc55Sp0OVCzISaBWPzXgY8829COqtaRbCIhacLTAwWtXWIzT\\\\\\\/tiMKK5LQy\\\\\\\/Xw8fCOpQCC4ZjfN4And1Cm\\\\\\\/AIWJ5M+WMehrvlcPTczgsJZSYGf9BuzMAH+SPG3ZOYBlkv7GnLnLZc\\\\\\\/OioYygGjUfNMjpZdnHWgYAt44vhg\\\\\\\/3TFlLVXPYL\\\\\\\/wl\\\\\\\/UeLtipw1Z\\\\\\\/KjTKslxdKvK5Pk4HUt9c+VB\\\\\\\/N3+i63YJc33BIW9TlES3xHIJNKuZqedK\\\\\\\/WbIxOBZspmbpRTdmYeu6zDBeZNwySfh2MM+3A3y7t7AqInhBkrk6JT9XZ+xCKqVsMzU6H3zXHI7q4rmQwejKCie1EfzzmKZs8tfVnohb0MOtxnjK7XA0lxOPaMIoLa50hZCIwI2L0AhYd\\\\\\\/Neq4AbqMb2HGNvQRKTZ1qAaAN1cyGMmGVkeiPcqUpGseJxqBnE9pXbsJ2RP7w0eqwftA64RWKKvsaOf2ofi5qo+Ig67HvpoMxWIpUPcvM5NsivhARCv5g815CKoLWDwzlxu0cJRqR0Cj5q5RIEBHA5w6X5Tru4pnsflUQyuZQk08p183YK1cDTauzGuy7PYtDiqCtUcm79zDBltg488qpPqUgoAettzngY0UjG5h8kuE4hWjfphuesNnHapQ3nCoekCSESWj12fGGFTL+cSaCZvRcJx+CKVpyfNizfV6nNfoBNHvqIFIVRpjjYRlho+CAT7FJDSxbx06Qo3pP5CuILKkJ2waI2RqZ6xd4y40SiV9ujDOG7sgCi5kEkPFrQAWdlgx+CCasymvGO4wwh0W6KSB1iFr4C2LnSHS9Q6JKexCYWDsPBg8jSIENibF5QMt5NaeV0hsm6mll2\\\\\\\/IszFun2s1UEr6WCZnhCIBwf7hNHC20yprL0v3IdprbEVPt99LFaarBuwhPilUA7uStfN\\\\\\\/ygXz9RjajjZ6\\\\\\\/4GBKonDkogWaKb9+W1XmO3MD7br4dHoF2q1LdujeKM5zkTWbl+IGU1KLM18mLt14SYYVYtteIM4f+l34tN7DaOBHroSzYHkieVJhXmtZkaVpA9j8GE7hBWkcUDq3Tww+2ENoD6+dLV5QSfX+AHo\\\\\\\/uR43HZoeCieqwNUVzQYXLXwpO5SN4WwSUoBCowX4Kv2\\\\\\\/nYxIhQ9Y4QAO7BoX4Q\\\\\\\/6jKpcwlip4aGLRuu4tLWRK\\\\\\\/UiMRgFhjPFHgVPsAB6oA3PtCIHL9G6XuDcNFeMMr9GwgptXU\\\\\\\/DbkhPKcAt8IVXMktcfRtK\\\\\\\/H\\\\\\\/DAoHzUDcXO8\\\\\\\/X\\\\\\\/uT5jHpQV9Tmpzmnl\\\\\\\/fHuekInPL3oWifJRtr9C0mGwEOoX2HEEY8B7hNNhyKyuzsh\\\\\\\/ufbjSnCtGQTUttEqcYaWLeYXYE\\\\\\\/3l0ldotFGdYu223CVnuuulwKPxr0nWmC2E7VwLX60WQ\\\\\\\/5VtZtEHsJjsLkRW9fT12Mu92AtKCEuZ5KSRbQEyL2Dp55FcIVZq\\\\\\\/bD\\\\\\\/Px1X\\\\\\\/VG69aJTyYvdcjv71kNmknSI57GQ3MkmVMeWockBLtEOWtOgPw7nhRiOMgMGKxggrtC8Qh2AHqAQ8ellCnD5mvwT0DaiXicj6hmHXnvwAwIlwCMoIwO3Mwse3sE4\\\\\\\/aZadYTT\\\\\\\/eSEe4BgllDHxJG7dAifccL6nSJhrJUd2U6DX5UXctP6hegMt1pIUCdcYeNik4Q9tE1RrtFwXA3FjgPxj\\\\\\\/Gx6I+T87xt1\\\\\\\/ypBn5HKmH+HPK+pj+jtU4TFY7i4k7AD5iNwfePP\\\\\\\/EFyDs4CQaf45K6Hxc+cHdKdghPqSTMI1fTggZ3ycFAyVrEBM+7WZPTpjDleErKy0KD6k8\\\\\\\/IhhZiNm1gNnlrilJLnczAtmSi\\\\\\\/iexTQRr4t4alDX602HfksGdUVqMV5LGs51c2ac4BPmS6OviJhKMjKHsEvdJPLyWdubO9R1zZTwuMZQLdd6f0dLE9C1AD59Zg+gYOCue1AkLDxzHXx2ZugAr2c+c8rv4wRwAMdEqF75SB0pw7exQ5sjWYH3YSjNMZwXnPos7vRQPuMQutcCK5uTYWJN1JVp5SRyMTT9YojQiTiw64axYGkINAJjp0qiuZoeGkU9I98QlNURMLipRqMVqBGjvr0K1mgfUPze6vM3lULkpcdpy4dmMUee85m5aqD6ogCPBxTan0uCjnpvRH5SnBnNtMBZhtnfO8v7omM8WPuBS50KiOumTzojgjDMJalEXWpFIuvS0vj8eGiT0Y7ZxWw+l\\\\\\\/3CBrmf1AIWU71yEogQc2s69i\\\\\\\/0EQ1\\\\\\\/SvTQ3GVJgSvVBtU5EyY+0bUnxj9qr+X+ZIqhWEmaezi5eIkX44HuTcIZY2oQULBAQIxxAe9VJw7uHdvV+BRY++rXI97jwcPN369iUzoaaiZwFy1fWxcpZ9I3kTwBnAKxRb28h6S90nnIS6WghSdtdD054hx\\\\\\\/0W5EkxamTYVNWAkvi9FKzCMwkRGFPGN\\\\\\\/ABtGVUUBPgrr5xK7Ed7MaCX14VEBpLThjJnSz5ZEEFctLFiyW2K3LWc6OwPoyzPn2yB8K+9VlrhIP3nSCJB7d0grTWEn08oQtIFeI9jvLjk2SnF9QF1ltudEgY7JipTbbWtdNXZ0ctO6iXoIljfRQoB2mbR6PupPiu4yeVmGZJPMQUaOL6ao3olEUshEYlfALgpnlg8Y+FkCwoIBtJimwHWWJHpPaqSvXuWT68NPrSfIxjnnUyoOl\\\\\\\/w0xs8PDwxZi3XtMt1U8jpCJSJfS3MLWvQrTiSLGxSk7VbJDE+Wl7sCRoS3yLYiwr8M7x46wjxDBc16n0lDXQt46MPWneF27OCIwxMIIU3MqaxK+dwopYFL\\\\\\\/Hc9QM8SCCJ4JYjSib1gqB6EFS4AsRGu3VJw87u05x7GgWoj2YozmZwz2AXDOTZn\\\\\\\/cE+oaZXQN2tGthxaNLjp08fgjGdXeeF9FoW\\\\\\\/c3TeUMRa5Wg77At\\\\\\\/oUkaD4MHRGLzRSE93oonUYp9vRxotDzLLkT+SM+MyFZBNJ\\\\\\\/zRniNypY3aEb+3TPc8acCXWEjfk0LT5ZdJphAIwCOXu\\\\\\\/IiyQPRIPbulQMvy9CYBNo3kv2FN9ppKtqXPIoCktUq+NWR\\\\\\\/ztQj2DGrA8SkELa9061T5i4OycHVBh\\\\\\\/8CGSPv77npS1vXt6pAuSQq\\\\\\\/Q2d6C8ph+oCS1Lt0sBhqxrdv6QEZB6e2N57uHaKqh0sYM3+k+7pCm9qIC6wz8T\\\\\\\/8DBKbRBqzp9ADn5WnRADDgIcelwDYY3JCwEAThG6YQqC86yeK\\\\\\\/rSVdbftjqIburPkv0+NWBp9OCFeN3Ex\\\\\\\/Qk0Y59qrrTko21wmwMdKlF6aemHnCdJKi79ojgLUG4oZFHQYzzBWVIf71DS\\\\\\\/c1vpoqnM17eARNGvdN4q71o9dH7Uf3BCjw2EMd4AWHv4rlNafPUGr2jVA3OslGDP+sA5oUrPNSHGm1uA42fpwkdjf768POcQG3XPkASuk6XNIU\\\\\\\/syFtfkH6CINLmRr4Xaykwpjtz2q+Ogd4jo96ccoQFpJKdVbSlqyXLewGonKGTBzOzOZyQY\\\\\\\/ZDP8eo1TAVb0Fu3qFF\\\\\\\/EeyAHOc8z9B3W198RxsNlo6W\\\\\\\/d2dXaf2NDZOZYkP+EJwT26Pfpt3dObLWOuOfFCjeZEMUbrIX5x29r6NnovmQl85Z862zRoB61hy5W0g692q\\\\\\\/H4vUEWHOLni34rK8rF+9Sg2i+WnAuCcubIOwFIdf058HG9Goj3qXgSaIlqEu6cBtbiYxwa0u7Vvr3R6mvXr03YO1oe4HZJ11IF76vUyE1tntTSxQrmPSX3h+g+YskB60KEFhdHktsSrwxoMB3TU7qOfmL1d+eCMtXvk+a6TtyOTwRC147qTvYFgAgd+HltT5\\\\\\\/sQVrZKIWTWpFcmHEPQXDqSWsP3hJc6AnjtPlDvMS6VlxVzA88xT96eQHs0cZuTdmagyfdiO7vWjztrlsf2\\\\\\\/BuPw0y83ZxKJDyXPTjrymUCUIUWrhmLi1gP7S8rjvBOUtzMHx8qoihlYmkkI111F8BxQnzJpGFoWd3L34lQzen36PR\\\\\\\/yjS\\\\\\\/Ct+eVSNnUnxIawWhSohGXKBYl71hY7i8qrHfiHc5ERpv3aQ0WdR1mzMCYVkS7ucgUYISpQtGGonENK4XUGQgz2AKvxwA2Y0vIAyn90W1yoXqdrjB2wmZk8s\\\\\\\/mqy\\\\\\\/80AL3e65ONnaD9vj3RMvmjnZONKd5+vulJsUse+feAFaB6tWzKkLAEXJ6D1jn5qDQeF\\\\\\\/yV7IqxjFk4KFUzLa8+e+DX4NxrBdb2r6E6nZEDpibd+olfB2wY96bAeW0zaZz4sAGLR7Hl4uo+\\\\\\\/SpYMKDFiON3i7QVZ4ZPgo\\\\\\\/ZC4cuyRRzLdkahVhzA5se4FsAxXGxHnmudmrPBH9ZSgeOyOF0PdgkPSDw3nOEsZ4KM8Mb7Lj\\\\\\\/wycq7E4UhhKGxbX4ZXjukENlyOMuBc03qzY2w8BZb0s8iWsLlDQG1z+BocpITtFc1rVP8eodioL1hOWezFZ5yhxTEEu8t8O9zuslMIQ5Y2eO3hj\\\\\\\/OrP5Iw8sPfXPEQZe6q4q\\\\\\\/2JkG7+9MHaDWUegL0p\\\\\\\/J21rFwH+kGVuOMm8z7krUr9vVyX0v\\\\\\\/BAL2LK4uuoMZi8auu4FAs\\\\\\\/0hXGfDH0kYe\\\\\\\/05y0yfFHva0ay0VUpRIZDoV65lk+Dmb8mW3PnJT2cOwni57DmK3WQwmhfZwdsmAxyv+VRVxTDccV0G5ufVvZdi\\\\\\\/oStf+rGcV01F4CLhmjYilsLMlq2G4czYoUeoV3ZMZm1R4eZbx4zFv6yjV7rYE19jSLx0mSQxpqM9i340SWp48v1K7Fw1VFxq+Np0HMX2+dBYDkZ9md7yfl0dqll3lYA2e4bXO9teB9l4LJOmrMHqp70ZWDly2sV0K1oIh5paq+mnLV5RgCiPQXGrlwD3\\\\\\\/3CpjVTeV+mGBlK7G8YXIeAsG0NNEFwWH8eK44Zl+rZrAjb\\\\\\\/NqpPqgn43VWHQddKG6k9ZCaP7x3W6u9acyVFmhNiB2Mh\\\\\\\/k7Apoe5\\\\\\\/pI1RH38Qqx3bCvVlrM9LrK+UuQfwRfxh16h0cgELV9Dw6T3jClhDSpplO\\\\\\\/Xdnp0E8T7qKn6ZtIjp94z160\\\\\\\/ZLxTURdwdBr2HQ3Lr0hCQQ1Sx365FtcCJ3iZ2mIAAjmBR26W4lbVQ8DzRIhR8XES9YE\\\\\\\/M+F4aGqMLCYldnVycDtmNG8uQG5nagLY5BeQvEk35J7JgLl3zRe3MRRDeZ0BRXvnjyvtBCB6c8bFR44BTWf1uqSwfKRiyJNN75LEqyJnnW5gSlUV4MVW9qj5ZTxq8e28TAS0wp6e2hopH6qtfB0T3jv0krgSe+GH86WY+f\\\\\\\/3D5UStrJhZ2ktlz\\\\\\\/Mb9zMzFJBhXVIP290LjDMdhenagAa23As9dVimwVRcacUrUxTb9pj1b+MimwGMDfpa4VTdKOkFjSgNcT0ixB8xNSjR9Cie5hbxXqa0oyk9V1s2thQD5lYBhaXDTg\\\\\\\/yyEoDOZtQmg0CxcGyUoFVEJSPt7a9iC\\\\\\\/HFG4uZqyJy6kUZ2Mj8PwN7e\\\\\\\/o5kDKygf+ned1N+wUZ1u7zWUCDp0EP6UdygrQ8TPHCWRzFtehvU6OTib+PL7QDSHkoNcJSDcZLd4KlPyn5u9UE\\\\\\\/zIDs0vM9EP9F+l+a+4zGFKRetxFGMHZA8VxwZ2S+USHHQBo2ux6Y1fXdgYS+R5+a0R\\\\\\\/Jgk5ntLpecde8xoGDFgzxx4BHJH\\\\\\\/sdoUg9wZdKGjuA7BMa4\\\\\\\/3xSLcBLisCZcbckVlSHD5mPDecJnE\\\\\\\/pY2IE4rJ4fuezzBY9PhVtNkSNTEGFD1eXe5ZlTutqyyQxfGWfdreW28bL5y24\\\\\\\/g4XKWaeq9bsdeCajN1ar1UlZXRI9zaQV31Lhn6NB8K9GRBIhnTDqA3iJXSVUJMJXGVa8Z8fZPDYVNrPVjhnuhauBGkkkt9Dgl1AaHUSr66sWemajdNnh8PMOWcrtCo+mTy8BpXxPQZtcwW3PIBDMyw7F9g48DPpHCPd2kkZ3cjGBotpREv05N4ytNMZNDavN\\\\\\\/QyC9vzhJYqNa0cJgS\\\\\\\/qiTJ9pJLKwpuZWhOnV5iFoBB24FzZxq3OE4jCT6GcCg4LwOOv43J\\\\\\\/HcD0UDfddqvBCA+rrcwHsGeeOezo0VDaGcj9\\\\\\\/ud0WBm23kX07e+GvpOpArJUv0X2CU0ZflLKfJTEQPkXK2BeMfUT\\\\\\\/vNGYv0glRPzuzlR85chIv0ytIJTS\\\\\\\/OPiClj9dOaVYxZ7ZY2SEZKxEaE+CXId9t+GytLgHOEMgQr2+2E4ZymMipXsZ1tO+mm\\\\\\\/dxbpa9KXsd3qwtJD6E4dndKe5b+m0ObxJJZsnUuKWiHj10HSrIRG2J5YQ2Gul9xqaWmSfayivPH\\\\\\\/jiPE6TSF9zJrliQeBLf0oXaMxM3PBkwwsmov9EpmHPDROn1Ek75+XHQbdmIgTnzOxDhqcuLWGv3N+pBJsnPU\\\\\\\/vYJWOThS8HHpCGqOdj\\\\\\\/CNmv5WBOPlbGlW8tdiFpx\\\\\\\/mH3Xluw7j0PPltcg2C9ZQGesE6x2xiWswsxGrxs0qjPqWN25n\\\\\\\/98ORBwsHlnyqhbXMnhwjUuXDBqqAaIEDULjkiBK7VwsHDkJDT0YjlcExPmCzAiVKuvoT+iyUD48eQ5bjVOu2\\\\\\\/GFvTK3TiWYSURsmY978N5e2vxotG1\\\\\\\/fnex6Jzg54JhZt+1dmj5zIokmQdhFenO0l75qWzoxUu4QUR8y+ZlcPYbXY6ckn\\\\\\\/aiJWwCVEz+gR\\\\\\\/prbRgznQEdPLaJEu6PF6NCA6WvPBdXNMERI1zqZYtPOGsdQFf0\\\\\\\/Uygb0ULGsyMo\\\\\\\/8PhhF89+jhlwDlSMA\\\\\\\/4asXkGHlSRQtA\\\\\\\/K6wf1HQDqVuuKPUu0Z1gWR6O2pK\\\\\\\/E3HyZjev15kHIBQ4IGzJX34Nuy22d\\\\\\\/xSxUZ1725y3QLX9zqlcyRsjUfcn2A0EF8PvsMBKveuMqh26sfUH8N4rETHCfPgHR7FTxVfxhOIvqQYqY4iSlOKMCT0KORWGP1sj5zqj+sBsTrta6NnGsgxdLSlVSLk7vH6PXoX2SpmJtnYsoKWJkFL99KHj0hUuHLMvdz9FT5kK4o+zJlhC2ZicN6NbQ\\\\\\\/utvmMNi6XOo\\\\\\\/v5d+VW1oOedx79Pt5RvmKiL3pBl1deFdd3J7p9xGZVR9jgvqSC06uSUAgsQucYDzZ56APLFVR1u+ZqdEqHWMIc2+WUbmyKVSOB+U\\\\\\\/OobxvN37WQKX9pdvyy0Y0Uxiro+pQtFfl3IvfLS0eBTWmruBio5mbMzgU1IgYv5wXcZ0NRNfzrygsEob0zQKhZZkDlHTRbiO3V7P89fap5Ibe+\\\\\\\/3G8YN0BlRXComKU3S+Qz1Z0LI0yBiFhB\\\\\\\/BuPpVLID7vbpoSMgPaeb0Ss\\\\\\\/rRvciQ90zWax1H27Bs457gbtzBcw\\\\\\\/32B8lVaSSt5hwPWdYOoB+WQ6MBchVtHrQkYuh6F\\\\\\\/e\\\\\\\/0WrgsMKD9y4+7zMKhL3EWfBhbMJAjj0a\\\\\\\/m+Np0hzCsINL9qu99ICe1SJ128DlZE15Y1wrPlcDPQrmJNbPZk32WlBGlal2KOHy1G8BEx4Xa4vJYF39jyuxt\\\\\\\/J88E8fcQ+doGDLYAAsQepePWpkrRf4DNibxcQVv+BwqTHGgjq7a4OLE1WSDmyr7vH6ME3C11Sx4hf60C5E1NYnCwHImoHQj3OtmeDg8r\\\\\\\/BQHJeJQmV9OCTS2\\\\\\\/Xe0q8MUVW3lMCSFYAMLVgIvKsjRFFsFN2Assv8UtREiXA90ynllUzGIaei7wiIK60ZlT+5dyW7s9573VJCCHjPvWG\\\\\\\/AfyCBRD3LCiiEBRkpxonXxm35KqtbLBzXnQV5lY\\\\\\\/dGcJkzryLeKPiCTkDtas2FIE\\\\\\\/GJPCavtSgmml+fVtlvju+Ec\\\\\\\/p9OoAXhZCLq278Ww7PWPzyDmDkJJAKHo8XrKq4uB6Xo+pUe+KqdWf9ezeTOmIip5z6yO9HaBZS4MtOClzwBqROOVl9J3J7DH7U5+1O2adGs+MSeGTzFGYfKKbM\\\\\\\/6zxbzU56bDGj7zdbXSg6w6r7UMpg3BOs3mtlU2OUfG53fOUkSllrq1Ptn0p0ByrKgA0c+vbh8EIYXIyLnvYYBCwX4kFISj1718HKOUYRiaHzAM20aCySWlYmiSAAtZrzKqJ6cz4jsLQd4pvxffOndrW9t+JYKELbYw8owNiNZeBv6GCxuYLB3cwi3SxYe3lelU6vQorosZaEj1rQTeQeF2AssjJPpswhg55i5tdBwoHwaY2U0avf7p+uZtynipQSNqU0JHjEX86GXCDAUxesARanXw94wmaDE3wUykRHGKHO07ZfpiVGYahDiQIT8ExVgMBOcOrgedjSc7VJK6Z1Gkx5Vc58rN1W+SbO\\\\\\\/+F5orICCyx\\\\\\\/OfBb0BznYLrn3O2+MovBa\\\\\\\/0aGFovkwbFo1naGfgCWGInG6zki7JIrr4\\\\\\\/pEB\\\\\\\/+rcVnU8JiRIQQcQSz1p9QQyo1Msa1r1qpx3UNvG\\\\\\\/XbQT3misT1E4tcPUihUP1CP9d3ZxFHS5bW+IizNve7IxWQao8q9e9sd+70lcma\\\\\\\/+oRXMIUqBorgp0h2Va8qBX5H8i+bkYGX9KGhA9Mhtp0ffS564KKIeicQlEXw0gjXOlS9c2Kv2+BFRKR6fIMmMcTF4\\\\\\\/YLaU6RS1Nsll4xTX2pfWFggbh8YGIfonJsrKlFIfWNSG0JTxam0iaVQZVtbUzlngd6LEVrLy2RsbR9tiJMg769mrYGjm1jxrbULQpWGEyduVxcCWsskCN09j65gyEawITUNsEZ8OAoKCQRG040989YJwUyoUu9YxAM0FqhIsm24eMezHMK6tgJiGyV\\\\\\\/9ScL\\\\\\\/q1A34iNboLUH4KfHwu1VsHY12qAU13DgaJ5wYUn5b\\\\\\\/OvBTfuXSDMqqh7HX7catHVczoYNPehJFd+mri8iCZ7nLBM2m8C50yOmDmsK7dF\\\\\\\/ixQE+G\\\\\\\/bkEjNYT2soPEGOtbBGvlYbSLqyYphQX5bNflMk4tanSaoFG47clcXwDzonHRnLAQOFP9ynlA3q9+ZV2L3LKqR956cmEFWUwxkKU\\\\\\\/xwI38qOGT1RF5C7rDDH0hCSvvWIl05wVbcNLsP5zmF0hdHZISnLnQWaKTk8VAERUGF8vfMVPRvRDLJCiBuvUlFzVkvQK+za8K5extABscn60QnDkv384tUrXZ8ksA3WX+VTID1Su7ZCJW0RCDz45gVqngZny8ImN6wU34DvCBpkSY9FpwcSfOFLa7MR\\\\\\\/UWgAZatYRKb4aNuqp82lHIEgx9q8QyEW4Mp5DuxCQCFuFtBUcPT7P5XUUT6b6K7yXEze\\\\\\\/DOVF2+Ka\\\\\\\/7K+79ZdjLy1nrTgadDVxVXoGCxGtHI+epCEOSO0OqTvov8TayD37UIXea8JgQGDT+Ci8rprI+1dEwNQcbj+eXE2AxvoRUQ+Rf8Kxn\\\\\\\/pzIU7nL8E+FKqZMZs8eBi3cgU8tu707pwm0bnCejE1DGhy3ipQ8i4e\\\\\\\/ZDi8L5KIOONDcx62PEnaX135fgOSFL\\\\\\\/yZSS3yhoZ5Km1puqzC10otN19F9fU2FyQaO35TZ8zi5fFnghgC5dacQt\\\\\\\/Z+9VPZJw8Ix2IZLrHFqA7YlbiqbmQf+5x2kieQ3vMfwdHqsMiWhM2gd9gVeeddGPFH+kD53eLtd5XBfEpMm1sGTipHHCSHiy1vhHYWoHtDxrZ2RhUXXrLDW6hP5YPmtXvqlGF9bA1RYckQY9e1Kysw\\\\\\\/gKsfNVnzjWwFk4bAqnZyeYORmyxLLHZQGbYxsPfLbkKGpQHw8kNI8J4sK6U03F0F5Uo16Uw9kb0EkpUVwUTSXrtuIQFwFU952VH1gG71m41IBKEnSwPPyZUTDt3a9JsvgiFNXtR1U1fx3Qn3zQ1F5RV5z3FeZ31B7HAogzHabxXNjlaHyVHKdJdI560tsvW9jZvnPEHN8XK7DTJX3j3iF9io6b04tlCZZyQQvgVorFZdu5\\\\\\\/\\\\\\\/CPXr498GAIM5G8ShzFfuUotpp+\\\\\\\/HriCLXt4JnpfQtoEtlN1jqjSPBMwjsu4s\\\\\\\/i3Hf\\\\\\\/qSfbE6TLekbUCYNk63\\\\\\\/kU05h6R1Kzs4xYW5S7uHUIOL1AhpZlyzUq8Verv7NB3TSTYFGbABCqi+5Fzuou3ztf+u9VFSjyuvAuJKtPcByhIDoTbwpjhNrXpybd7lvYxBnlOpni7aMPsY0bujPPl1tRBiYpCFYfxgyNOF1W1zEPA1sNE7mb8BNsktPPjFVmZbT3PsCpHTv0SG8MYJiVAVppHv24pJQTdvlq9oalfH74dFmufkgaz3WZA+xx2l0tdX5Z3DJT5\\\\\\\/TUj6ZEeUv4z9d\\\\\\\/RMIbpIMGNqzPjFsCyep1tYq5hca2FQdLR\\\\\\\/8LSxcLPVGo+4WGB\\\\\\\/n0QZqh7sJGdjcUgej0TQCiS0N0TFp7W22xJjKvHvtRcZObyxnjVzu0j6Pt4W2pQjY3shYP2ZHvtMTuVxHUjo1YNi5wrNPNAevgfVSuVME4I1eRogYX10LnQPscFoBCibNi56roltzRAE5ABRfk\\\\\\\/aEzFroo6H6yNWZXAeZmgI9HhI5Oh\\\\\\\/6b4L1AB1gs6gtr895Cc\\\\\\\/GawP\\\\\\\/Ivs\\\\\\\/VqUloa8Sn\\\\\\\/fzItK5LBzdc4p9weZkLndHYlRN9c4zUg5qXyH0SmNqu7rMQk5+XaZWtCw0Ly+i7aJimFEYBBJxdxwPwF\\\\\\\/eKUr7txEnt1WPahqBoNT2rw2EFdhCYlgsCQLqvcY1AVCdi+D9\\\\\\\/hT8n3dl8SZVKioQii7SEsFUElPIk2c78WTaUiNynZ2uFJW4giqKoRVt7KLV51DpWEV0nD4X+RGa0oc2FfDBry1lZf1DAIoip4wT+bAlHv5Gd\\\\\\\/lhDCUTB4mYPBkUHIED9SN+vXlJSvdL3zTaP2CtY9eF7kFNMtAtq9DKTFS0xS6gcjM4yWlku47rk91L4s\\\\\\\/QZKgHraJM+4h0mMvq8\\\\\\\/8YPMbduaq\\\\\\\/pj3\\\\\\\/Urlbchfgg6Wql2f760gBXOqn5k4qo0mBL16KgZnQYYuKMLN4YRo89oXygisOWKeuEBxhehBJApK45mpySwS5MRcJ4EuxiGv8QWCLjuz+BvWbQpLgXYtfZxaC+yI3ZvFGSlB5o39C7YHEnfmG5vCwnlYG4T6BOT4BdffHPQ559cHMLexMUUmhT76vzcrWhUInD4DvaQJqSBRO2S028lV9BVgqKgWVrHRJc2lb+UVXe7sh43nk9yqXgQ9J+qLVcDZ3ccHgPvIhAD5hdEi52ZqwhEpvRrV0iUbSkrXIGQIGfPcqvg1RenjgPU8JJv0SO8XdJ86iajPp+v52hfT5zheFWzize9z8+M7SwJp0tYCDvk0O5PeDJK8+NNFFf8V1X\\\\\\\/ARcpy81u5\\\\\\\/oWkymkTCIeLue5xFCVuH3EnaBlt93Yb3tY6x3U5Ocmp4BG9On7XoNXYOswRQovwXpWUn5WSLMRA9ogd45VkVE0e8neJ2NqwtYtj4ceratDucRyTI9ZqvedH7da8vjvn0BapVh+eolaMYD10WPJPdExmYj3ECIgUJwOGiNSrkQMXCSe1lhQDzlBRNGUQ5DcxHd2ZiwXX1iBKfH+NDBJJUAYJl+YRHPxI6xmg7trr962LJTSTzYlpsSXQ5oM51Nj+G8lU+ZyHll2Rt+rU1kJ7k2BzJE1TRcGqtP8V3jMWaXpSLZeIZF9hv+qufkn+de4pqDfgY9NUbloAKpsLY1upB1tWBRZTyZR20jo3hr94gyTzOCSwIjTyAA3of2xp8o1umgXfxNWGzXq1fxsd6VHET+ugt+TrdXQQcByfhS9Gsc9g+f4T6kd4rAwQCNtgwulMtYtbSwfXYNugXKhDLbw1MgnwmpjAvwkaUwiFxVRZZ9HYtuU\\\\\\\/bf+TdfFOk\\\\\\\/Mr5iIAGb3+I4w9rmtCWllmIVeQRsV5ODGbWc8V6KHRCcG\\\\\\\/4KU2\\\\\\\/p4Bngjef7esyo4o5SD5G00OZuzldt9cXa2c1v1\\\\\\\/towr5NbzIZ\\\\\\\/MpvI4Qp73ohxSqtxrTAipffG7Y56n68kc4VOLOMJI32nz0xsbQ\\\\\\\/PgM6EskqlXijGzDEODR4qq1AVu9aPgNfbMGGAA23nj5i64H87PbZraf2u8\\\\\\\/561RADSjh6\\\\\\\/j6G\\\\\\\/1WvStGXVwESIyTf7M3EmL6ep6okUTYpxp2Tmh9BH8xh06OkkkRyMBaWuB\\\\\\\/oQ1r97lDffvsgaEzhNNOdd1zMmBX5edTLaXmup4Bm7yJuuEegAWJG247g6QsIaFR9laj6vnYKZciZuds+k6T\\\\\\\/gKhFDQeYesmnVWqPQtNvBDSKhKojoIqny1+6OXkLscw4e1VufnKzVaP0yZTkwN6tbdrWz+\\\\\\\/K8a0kcPtqqeKo2m8cWjjiemGs6KSKe9HBTNJ6iY\\\\\\\/bIAj1KtY5DIJ7pwsM8A+UutLUDBZd51H1yB\\\\\\\/bPVRIpgDdy1nVv2bU6+782XeMQHXtKI2oOG\\\\\\\/Qwf+HpklyUpcqyBc8zx4LUyhrLBpm4DrdY3Vl50cyzjD4iHGHqK0Sru8kNNuNf6Ae6laMeJh3KCAAi0LDWYR1l1xE1nogaJybF0FguMRh3c+MEobRpjh5KtSL6nWxv+BysFxG6Abuib6cy1VHvPVQ7j4\\\\\\\/qvjT66rRraQIbQl3xG1fLZCYIg9KKQLKuHb6OJaV49QRJMrBIjL+ohozCfaouESHO+ezdAzJDTkRAnsqUtKzenfI8RVH5k0K1TPObF8ulVPohX3AZ8aPX4SMXpi9yEtQ\\\\\\\/1iMwTzUSU7IeGKcW5p8iBdYfb6AONOx6Evl9s53vh4Pd0yzuV6PbQGjB82L4LHc6b6+vQXqRIJSqqFtY6gEy6KImuI9pR2km22GdaMSnIS7QHuMoJHeCo4K+rFrrLbjcJue4GH+sU1r4OTA45FbVFyr33eX3wTsEAdg\\\\\\\/MzCe6rriDtNhZLipy2dwpAdsUHLHy4KO3qM9LINxr7oewNQ+bXLHOLjiSNTq1xRnTOVfGbH8rm58icupC8OBof+d4NygcLR0NsH\\\\\\\/co\\\\\\\/TlHMLpnqgj7etY0maVQYJ64xsvZq+usfqFqPYMis4yzC+Dd+ehZxVqGVlBvpzBO+JNORKjIG6r6FnMSVHzxTeKcQT5OibNyAdhL5u8CHxeVLpFmOtURu95BFDj2DPgtvoKgWSElNMYPBecdZLuVbEFi2pdBc4reTS1angkQqGeH2rydHyjiDQ5mykEcKVES\\\\\\\/B5NiG5omNL1rIObFYLyFsV+oyjGSe6fxZ\\\\\\\/tUzJvOAxg73C1Fu8HFwy2+nAzJPVkMINfqLi81QT\\\\\\\/jb4VfcPmpZeVw2M1h0Eu1u2iEaCQaT4s3oXan9kl2MFv2UnmTiyFopUbcpkFlLcDzG1qGAPQr2eYdpJFtHDW66bEWEgpgooaU7SHwzcsRLU6UdhvQohTNyJBMHB\\\\\\\/n5l6RIu31\\\\\\\/RX8sC0u9PfciGi98H+qxLEmi3jELd1KbLjzdrUNsbLAjFgrMoV5p+IzNo8E3nZOAIbS3lce6G\\\\\\\/zLhcxzqJBp6dGmJitE3771ASsAIWwO5gqnhEEir1f7qaiqPRn8JvsrmmdgsC8s16HE5cGwWc8EHP2E7PFGoiQtRILrH25iN+GeuCaSsGMY7C0M\\\\\\\/gwolRSauCc2FxL1Cs6O5btcdOfO8MWkRfoHpmmmzBYMqhd0i9kXnduplAb9N04VMdHBn1fIpUrEintA20jIn6uyPr0b\\\u0022,\\\u0022iv\\\u0022:\\\u0022X\\\\\\\/JjZ08q\\\\\\\/flZnYxFF\\\\\\\/ZQkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022deec10aa82a76ea22dd6261e9660f623e03b7842947b0607b6e16985dfe64320\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rIF5oXSsGlQgBzOdHh3TR+BhDXOFGt3dLuHUq1vqvUTb40CiOLykN5uEssUpDQlC9VlvZFUudlw+LWDW07PpTAIK63jdbYTGXO1SD8Vrlou9XzdTPUbcDosGdPGvg+KRu3ZgCoqXOM\\\\\\\/YaSuHxVxbsh06WSewfMhNrdsgTLbQh7Z66GZ02T\\\\\\\/e8VR4x7KgOPHLCFM8G4HNDdfsHfUZaIEtsFgz2KvDUQ2E5ioTEEQ0nu5PvsaXzlY6b3pZQNagHv80rTaZESx8ALY+4PVoAip9da2Qw2F\\\\\\\/m0R1YRnMdt2CkxcEzhqO7E8smu1ViIKIpul8mIiQCHQK+At9twG+NTtAwCnPyLMGoq\\\\\\\/Xc1Z7ckneWoftu3GW6CobTuaNwK+YSW1M5R0N2Mqcy0wCgfbuVwPDlVn6gEAdmepY+ZoU3r1r4BXNJAXwVznjt76RwfhUaqOn5CX6rIxk58EYnH5R7deDI4c5jITaFZZer80Oh0RsIzeoWf4V2mV0VlGyw9osHOhktG\\\\\\\/OGGioYRS\\\\\\\/rOUSO7iCAuuSM+oWdbZCVbrV+RA6vszOFuASyWsSnoYuoSTro6bYbqMIhWDUbttn+OtyxqS8A2t1UB9h+qNRDlj7V5N0d1pW\\\\\\\/6P4lmnfXYoiIcZyFZlf1+frtsMkJE1DzuEviGAJ4P2eUYmyP46TL+yfI1T\\\\\\\/eUI8mRX9h\\\\\\\/NyruBfQRiW\\\\\\\/seZvxvpSUpxaYndmFN5xzFT5RYaWcw5GHrLqLZdNBkk73t5zEqq9RQE1TZIy9B97CS67Bmy6Ma5RhQi77BL14Lb8TQ6aceDjIoeZVcP2l0gGfSmglSqVwr\\\\\\\/t0JcU4H+4dH0qVQjK04wkFSCZNvWMT5JygFztEZISK88SEbpLsJu6Nw4hKMtjh+b6mHxtdnt9M5zYlj05VOrqPRuwtjWHE1wTbR7P0Zs6sJ9vQVPWKgGTSMATTexUpeXvuqhlSUKNB\\\\\\\/zU0bQ14wNXbhvs8zL3iWu8k+vLy1LhqXNwobjboVDP\\\\\\\/Mocuj2I7b6qsmHH\\\\\\\/pcWc9KwRfl2wk63+RN46F4P1f0t+IRlx8gHc7+fmVQ9ya2Ll0T8bsuDWtN9VVf2nwp3a4YYzfGk6H7f4YQ7v6C\\\\\\\/Ng+JV5uUKj5cXRnCMwhhgS+m52X7Ji3orvVktyaa7\\\\\\\/7F53y85Sg3aSm+Lh6oZK9r0Hf6DP0\\\\\\\/YZwX6bj\\\\\\\/1BMPR0XdBRUyL1uKP2OvxRdkaRp2tk7EEn\\\\\\\/X\\\\\\\/SZYQ3XVaaeIrgSEs+eWpOUogGqxs\\\\\\\/fj0Bqn7EHqtMyerimyKg2MGGfF6clP17dP\\\\\\\/6SsOWKxBgm2fEJQ3NHRH2\\\\\\\/FhasTgn1J+W5dY1kUvG0yKqCw3SjujLshN5rNqOpjsFIz0vUolXUT0cn7WdoAsDjd3QYSTZm6f\\\\\\\/8O+JrHFkTZfNpPjNOk\\\\\\\/3BhXAl+gtVg\\\\\\\/bz6LD1uCWc+gwAF1FwBYv4WtMydJIviZwR8FU+UUV3TN36raCX2vdZPzKFLdQzdocQzUfwCQCx3\\\\\\\/ZsN2xNydV01HeokpPwm9lAyNIw2W26d1ER8b2FRJlSH8SsSVDK9mNLB6vvTZOlJiWC1MTu5dLRVK1979Lcbpdc6HO6XBhu31eYLfe9FwbQmOnNmSDDS6\\\\\\\/+ovTGrQvp8F5+J23HzP9Z61QBzFbSi8nMNl4zPDdabrenW+PfttzUv0WbrQ6duKEj3lajfR+uN18qBZLTQB3yJVSOIyOyo7TbZMujs6J53ZZ5U1wDmrVC\\\\\\\/EaGF+A1FYjoFMMt0TrrtaRCCo7ZicjoUIZMXTsTew2xgpM0s8h9Ki6UGpsBB4dov37NTyIdTbQUWWc0lA3tbLeNJkZKGAhMBcWoS+Fb8\\\\\\\/cB+WUlAok1TthfxrqxVe9x1foq9AQeGZgzebJR1qXHmMHBsrhVSmGHMxs49byFt5A5dldDeEyghR9wKnl3Fwq4ti6ujSItFL5PCnh\\\\\\\/xZLHOvA80fj9LT7IkDg21xDx1BCe0yDybfxnRhJGZmq+CexPdB0LLGdq6O3sKBMH876ciC7\\\\\\\/2qgUC1qbCjWN\\\\\\\/JGYAOJRU1A=\\\u0022,\\\u0022iv\\\u0022:\\\u0022VXe92ngtvQtXLv3EmSgM2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b59ea39afdf5570928f4ead93f6ec39e3cf764ca711027863501cbcda3c54ade\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460508 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3825" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aa59nRaZ3m9rBjS_eeVr5bD_YFWmkSLF4tTdyyBxNJA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3825" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-40R_ja2yTpvA6Fo73s12rYFghbNBbIelsY6tBBqmldc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3817" title="Tissue Culture as a Source of Replicates in Nonmodel Plants: Variation in Cold Response in Arabidopsis lyrata ssp. petraea" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3837" title="Recurrence of Chromosome Rearrangements and Reuse of DNA Breakpoints in the Evolution of the Triticeae Genomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460508" id="node460508" data-pisa="ggg;6/12/3825" data-pisa-master="ggg;g3.115.026898" data-apath="/ggg/6/12/3825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome Sequencing of Diverse Peanut (<em>Arachis</em>) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ratan Chopra</span>, <span class="highwire-citation-author" data-delta="1">Gloria Burow</span>, <span class="highwire-citation-author" data-delta="2">Charles E. Simpson</span>, <span class="highwire-citation-author" data-delta="3">Jennifer Chagoya</span>, <span class="highwire-citation-author" data-delta="4">Joann Mudge</span> and <span class="highwire-citation-author" data-delta="5">Mark D. Burow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3825-3836;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026898 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460508" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ratan Chopra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ratan%2BChopra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chopra%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARatan%2BChopra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gloria Burow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture, Agriculture Research Service, Cropping Systems Research Laboratory, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gloria%2BBurow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burow%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGloria%2BBurow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charles E. Simpson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Texas A&M AgriLife Research, Stephenville, Texas 76401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BE.%2BSimpson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simpson%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BE.%2BSimpson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer Chagoya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Texas A&M AgriLife Research, Lubbock, Texas 79403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BChagoya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chagoya%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BChagoya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joann Mudge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Center for Genome Resources, Santa Fe, New Mexico 87505</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joann%2BMudge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mudge%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoann%2BMudge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark D. Burow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Texas A&M AgriLife Research, Lubbock, Texas 79403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BBurow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burow%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BD.%2BBurow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mburow@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">mburow@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/12/3825" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460508" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460508/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3825.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460508" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460508/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3825.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460508" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460508/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3825.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460508" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460508/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/12/3825.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460508" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460508/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">To test the hypothesis that the cultivated peanut species possesses almost no molecular variability, we sequenced a diverse panel of 22 <em>Arachis</em> accessions representing <em>Arachis hypogaea</em> botanical classes, A-, B-, and K- genome diploids, a synthetic amphidiploid, and a tetraploid wild species. RNASeq was performed on pools of three tissues, and <em>de novo</em> assembly was performed. Realignment of individual accession reads to transcripts of the cultivar OLin identified 306,820 biallelic SNPs. Among 10 naturally occurring tetraploid accessions, 40,382 unique homozygous SNPs were identified in 14,719 contigs. In eight diploid accessions, 291,115 unique SNPs were identified in 26,320 contigs. The average SNP rate among the 10 cultivated tetraploids was 0.5, and among eight diploids was 9.2 per 1000 bp. Diversity analysis indicated grouping of diploids according to genome classification, and cultivated tetraploids by subspecies. Cluster analysis of variants indicated that sequences of B genome species were the most similar to the tetraploids, and the next closest diploid accession belonged to the A genome species. A subset of 66 SNPs selected from the dataset was validated; of 782 SNP calls, 636 (81.32%) were confirmed using an allele-specific discrimination assay. We conclude that substantial genetic variability exists among wild species. Additionally, significant but lesser variability at the molecular level occurs among accessions of the cultivated species. This survey is the first to report significant SNP level diversity among transcripts, and may explain some of the phenotypic differences observed in germplasm surveys. Understanding SNP variants in the <em>Arachis</em> accessions will benefit in developing markers for selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/groundnut" class="hw-term hw-article-keyword hw-article-keyword-groundnut" rel="nofollow">groundnut</a></li><li class="kwd"><a href="/keyword/alleles" class="hw-term hw-article-keyword hw-article-keyword-alleles" rel="nofollow">alleles</a></li><li class="kwd"><a href="/keyword/snps" class="hw-term hw-article-keyword hw-article-keyword-snps" rel="nofollow">SNPs</a></li><li class="kwd"><a href="/keyword/domestication" class="hw-term hw-article-keyword hw-article-keyword-domestication" rel="nofollow">domestication</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li></ul><p id="p-5">Domesticated peanut (<em>Arachis hypogaea</em> L.) is one of many polyploid species belonging to the genus <em>Arachis</em>, family Fabaceae, and is native to South America; many of these species come from a region including Brazil, Bolivia, and Paraguay (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gregory <em>et al.</em> 1980</a>). There are 80 species, including diploids and tetraploids, described in the genus, categorized into nine sections according to morphology and crossability (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Krapovickas and Gregory 1994</a>). <em>A. hypogaea L</em>. is classified into two subspecies, <em>hypogaea</em> and <em>fastigiata</em>, based on the presence or absence of flowers on the main axis, and spreading or erect growth habit. These two subspecies are further classified into six botanical varieties based on morphology (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Krapovickas and Gregory 1994</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Valls and Simpson 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lavia <em>et al.</em> 2008</a>).</p><p id="p-6">The origin of <em>A. hypogaea</em> L. and identity of progenitor species have been of interest to plant taxonomists, geneticists, and breeders. However, our knowledge of the origin of cultivated peanut is limited compared with other major crops. More than eight diploid species having either the A- or B- genome have been considered to be involved in the origin of peanut (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Norden 1973</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gregory and Gregory 1976</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kochert <em>et al.</em> 1991</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">1996</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fernandez and Krapovickas 1994</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Krapovickas and Gregory 1994</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lavia 1998</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Raina and Mukai 1999</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Raina <em>et al.</em> 2001</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Moretzsohn <em>et al.</em> 2004</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Seijo <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bertioli <em>et al.</em> 2011</a>). More recently, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Seijo <em>et al.</em> (2007)</a> and <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bertioli <em>et al.</em> (2011</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2016)</a> provided stronger evidence of <em>A. duranensis</em> and <em>A. ipaënsis</em> being the progenitor species of modern cultivars.</p><p id="p-7">All molecular studies, even using older types of molecular markers, of wild peanut species have identified significant molecular-level variability among these accessions (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Halward <em>et al.</em> 1991</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lu and Pickersgill 1993</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kochert <em>et al.</em> 1991</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">1996</a>). Wild <em>Arachis</em> species possess genetic variability in pest and disease resistance traits, which could be used to improve the cultivated peanut (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Stalker and Moss 1987</a>). Alleles that confer resistance to pests and disease in some wild <em>Arachis</em> species have been successfully transferred into cultivated peanut (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Simpson 2001</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mallikarjuna <em>et al.</em> 2011</a>).</p><p id="p-8">In contrast, many molecular studies have demonstrated no or little genetic variability in the cultivated species, <em>A. hypogaea</em>. This was first noticed with the use of allozyme and RFLP markers (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Halward <em>et al.</em> 1991</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Kochert <em>et al.</em> 1991</a>, <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">1996</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lu and Pickersgill 1993</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2009</a>), which demonstrated an almost complete lack of genetic diversity among the cultivated peanut accessions. It was concluded that a genetic bottleneck occurring as a result of the polyploidization event, coupled with a self-pollinating reproductive system, and the use of a few elite breeding lines with little exotic germplasm in breeding programs, has resulted in a narrow genetic base of peanut cultivars. Natural gene exchange between wild diploid species and cultivated peanut may have been limited due to genomic rearrangement as well as differences in ploidy levels (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Soltis and Soltis 1999</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2012</a>). Since then, &gt;10,000 SSR markers have been identified in peanut, many solely among wild species, but few SSR marker maps possess 200 or more SSR markers, again suggesting low genetic variability in the cultivated species.</p><p id="p-9">Despite the results of some molecular studies, phenotypic evaluation of germplasm collections, such as core collections of 1704 (ICRISAT), 831 (United States), and 582 (China) accessions (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Upadhyaya 2003</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Holbrook <em>et al.</em> 1993</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jiang <em>et al.</em> 2004</a>), and minicore collections (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Upadhyaya <em>et al.</em> 2002</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holbrook and Dong 2005</a>) point to a different conclusion. Evaluation has demonstrated significant phenotypic diversity for numerous traits, including resistance to leaf spots, tomato spotted wilt virus, other biotic stresses, for tolerance to drought or heat stress, and for early maturity (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Isleib <em>et al.</em> 1995</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson <em>et al.</em> 1996</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Upadhyaya 2003</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">2005</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Upadhyaya <em>et al.</em> 2006a</a>,<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">b</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Selvaraj <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wang <em>et al.</em> 2011a</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pandey <em>et al.</em> 2014</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Singh <em>et al.</em> 2014</a>). To date, these have not been accompanied by molecular characterization at SNP levels.</p><p id="p-10">Technology for DNA sequencing and SNP analysis has made great progress recently, both for high throughput and for low cost per sequence. Due to the ubiquity of SNPs, and the far greater power to identify polymorphisms than other types of marker analysis, sequencing is able to identify genetic diversity better than other marker types. RNASeq allows transcriptome profiling and SNP identification with high read depth at much lower cost than whole genome sequencing. Annotation of assembled RNASeq reads can help in understanding the function of a gene or a transcript. Annotations have been used in several crops for establishing genetic information where limited knowledge exists (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Garg <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2012</a>). SNPs in annotated transcripts associated with traits would benefit functional characterization of genes. For example, a mutation in the fatty acid desaturase FAD2B gene in peanut is being used for selection of the high-oleic trait (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wang <em>et al.</em> 2011b</a>). Use of RNASeq on elite peanut genotypes has provided evidence of large numbers of SNPs among accessions (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chopra <em>et al.</em> 2015</a>); such a large number of polymorphisms were not known using earlier molecular marker techniques (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Kochert <em>et al.</em> 1996</a>).</p><p id="p-11">Validation of SNPs identified by sequencing is required for use of these SNPs in generation of molecular maps or association with specific traits. Validation of SNPs in crops such as soybean (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wu <em>et al.</em> 2011</a>), peanut (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Nagy <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Khera <em>et al.</em> 2013</a>), or chickpea (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Deokar <em>et al.</em> 2014</a>) has benefited in producing high-density maps extended to QTL and GWAS analysis. Assembly, annotation, and validation of the information generated using RNASeq is an integrated process needed for application of datasets. Integrating these processes is becoming routine, as it is advantageous for a program to develop its own resources to address research goals of a particular crop.</p><p id="p-12">The hypothesis of the current work is that, commensurate with the phenotypic differences observed in the field, there is actually a substantial amount of unrealized genetic diversity in peanut at the nucleotide level (both among the diploid and tetraploid species accessions). Although recent studies at the genome levels were carried out in two progenitor wild species (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bertioli <em>et al.</em> 2016</a>), there is no survey of SNP level variability among the broader array of wild species, nor among cultivated accessions. Therefore, in this present study, a collection of eight cultivated tetraploid accessions, eight wild diploids, a natural amphidiploid, and a synthetic amphidiploid were selected for transcriptome analysis, adding in also data from four previously sequenced cultivated accessions (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chopra <em>et al.</em> 2015</a>). The objectives of the study were (a) to measure allelic and transcript diversity among the accessions of different species within the genus <em>Arachis</em>, (b) to estimate number of genes in peanut, (c) to compare to the expected phylogenic relationships of the <em>Arachis</em> accessions, (d) to highlight SNPs specific to one accession or to a group of market types or to a group of different genome affiliations, and (e) to validate the SNPs obtained from the bioinformatics calls that can be used in a breeding program.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant accessions</h3><p id="p-13">A total of 12 tetraploid accessions from <em>A. hypogaea</em> was used, 10 tetraploids representing the three botanical types cultivated in the United States, namely subsp. <em>hypogaea</em> var <em>hypogaea</em>, subsp. <em>fastigiata</em> var <em>fastigiata</em>, and subsp. <em>fastigiata</em> var <em>vulgaris</em>, and two tetraploids representing other botanical types namely, subsp. <em>hypogaea</em> var <em>hirsuta</em>, and subsp. <em>fastigiata</em> var <em>aequatoriana</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). <em>A. monticola</em> is the only known wild tetraploid relative in section <em>Arachis</em>, and was included for comparison to the cultivated peanut and potentially for understanding gene flow between <em>A. monticola</em> and <em>A. hypogaea</em>. Eight wild diploid accessions, including probable parental species <em>A. duranensis</em> and <em>A. ipaënsis</em> of <em>A. hypogaea</em> were also included. Use of these latter accessions provided a platform for assessing the extent of genetic variability among the wild species. A synthetic amphidiploid, TxAG-6 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Simpson <em>et al.</em> 1993</a>) was also included in the study (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>); parents of this were <em>A. cardenasii</em>, <em>A. diogoi</em>, and <em>A. batizocoi</em>. TxAG-6 has donated alleles that have resulted in release of four nematode-resistant cultivars, namely COAN (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Simpson and Starr 2001</a>), NemaTAM (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Simpson <em>et al.</em> 2003a</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">b</a>), Tifguard (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Holbrook <em>et al.</em> 2008</a>), and Webb (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Simpson <em>et al.</em> 2013</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469892/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469892/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of <em>Arachis</em> accessions utilized for the sequence analysis: their origin and ploidy levels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Sequences, de novo assembly and annotation</h3><p id="p-14">Plants were grown in the greenhouse at Texas A&amp;M AgriLife Research. Unopened leaves, roots, and pods from yellow, brown, and black pod maturity stages were selected for RNA isolation. RNA was isolated using the Trizol reagent (Invitrogen, Grand Island, NY). Accessions OLin, Tamrun OL07, Jupiter, and New Mexico Valencia C were sequenced on an Illumina GAIIx using 2 × 54 paired-end reads as reported previously (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chopra <em>et al.</em> 2015</a>). The remaining 18 genotypes were sequenced on an Illumina HiSeq 2000 instrument using 2 × 50 paired end reads at the National Center for Genome Resources, Santa Fe, NM. <em>De novo</em> assemblies were performed for all 22 genotypes using the Trinity (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grabherr <em>et al.</em> 2011</a>) assembler, with the parameters used in <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chopra <em>et al.</em> (2014)</a> for the peanut transcriptome. Assembly statistics from Trinity, which enumerate unique transcripts as an estimate of the number of genes in a transcriptome, and the measure of the total number of structural variants derived from genes, are reported.</p><p id="p-15">A consensus assembly was built by pooling transcripts of all 22 accessions, followed by elimination of duplicate transcripts using CAP3 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huang and Madan 1999</a>) at a minimum match of 95%. Annotations of the consensus assembly were performed against the <em>nr</em> database from NCBI, and from the Swissprot and Trembl databases from UniProt (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">UniProt Consortium 2015</a>). Sequences from the consensus assembly were assigned gene ontology (GO) categories using the Mercator web-based tool (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lohse <em>et al.</em> 2014</a>). Brief annotations of the transcripts in the consensus assembly assigned to different biological and molecular functions by Mercator are provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS1.xlsx">File S1</a>. Transcripts assigned to different transcription factor families are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS2.xlsx">File S2</a>.</p></div><div id="sec-4" class="subsection"><h3>Mapping and variant calling</h3><p id="p-16">Raw reads of all samples were aligned back to the OLin transcriptome sequence (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chopra <em>et al.</em> 2014</a>) using the bwa.0.7.5a (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li and Durbin 2010</a>) aligner, using the backtrack approach. These aligned reads were further processed using default values of GATK tools (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">DePristo <em>et al.</em> 2011</a>) to improve the qualities of the alignment process. Aligned files were sorted and indexed using Picard tools (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>). The GATK variant calling tool was used to identify SNPs and indels among these genotypes using default parameters for UnifiedGenotyper at a ploidy level of 2. Parameters of MQ (Mapping Quality) &gt; 40.0, AF (Allele Frequency) &gt; 0.10, and DP (Read Depth) &gt; 50 were used to filter the SNP calls using vcftools (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Danecek <em>et al.</em> 2011</a>). Variant statistics were calculated using the vcf-stats script, which reports variants among the samples used for comparisons.</p></div><div id="sec-5" class="subsection"><h3>Diversity and population structure</h3><p id="p-17">Using SNPs identified from sequences that had a read depth of at least 50, and variant allele frequency &gt;10% across 22 genotypes, principal components analysis (PCA) was performed using the <em>R</em> package SNPRelate (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zheng <em>et al.</em> 2012</a>). The SNP datasets obtained after using parameters to filter the SNPs were used to calculate the individual dissimilarities for each pair of genotypes using the command snpgdsHCluster[snpgdsDiss(genofile, num.thread = 2, autosome.only = FALSE)]. A dendrogram was generated for each of the paired accessions after 5000 permutations using the command snpgdsCutTree(“FileName,” n.prem = 5000).</p></div><div id="sec-6" class="subsection"><h3>SNP validation</h3><p id="p-18">Genomic DNA was prepared from seed tissue of the 12 sequenced accessions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS3.xlsx">File S3</a>), including treatment with RNaseA during homogenization, and purified using the Qiagen DNAeasy miniprep kit (Qiagen, Valencia, CA). Sixty-six pairs of primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS4.xlsx">File S4</a>) were designed to perform validation of allele calls in the sequenced genotypes. For each putative SNP, two allele-specific forward primers, and one common reverse primer were designed (LGC Genomics, Hoddesdon, UK).</p><p id="p-19">Genotyping reactions were performed on a LightCycler 480 (Roche, Branford, CT) in a final volume of 10 µl containing 1× KASP Reaction Mix (LGC Genomics, Hoddesdon, UK), 0.14 µl assay mix, and 10–20 ng of genomic DNA. The following cycling conditions were used: 15 min at 94°; 10 touchdown cycles of 20 sec at 94°, 60 sec at 65–57° (dropping 0.8° per cycle); and 26 cycles of 20 sec at 94°, 60 sec at 57°, and read at 37° for 5 sec. Fluorescence detection of the reaction was performed using a built-in scanner and the data were analyzed using the LightCycler 480 software (Roche, Branford, CT).</p></div><div id="sec-7" class="subsection"><h3>Phasing approach</h3><p id="p-20">To resolve homeologs in the peanut tetraploid reference sequence, we used the phasing approach. Genome-specific assemblies in tetraploid peanut using the phasing approach were generated by aligning OLin raw reads to the OLin reference transcriptome (60,798 contigs) using the BWA backtrack approach. Polymorphisms among the mapped reads were detected using the GATK software as it has been shown to perform well on RNAseq data (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McCormick <em>et al.</em> 2015</a>). Called SNPs and Multiple Nucleotide Polymorphisms (MNPs) were phased using the HapCUTv.0.5software (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bansal and Bafna 2008</a>) with default parameters. To generate homeolog-specific subassemblies, we used the strategy employed by <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Krasileva <em>et al.</em> (2013)</a>, sorting the reads within each phased SNP block based on the HapCUT output, and reassembling <em>de novo</em> the reads for each block and phase using parallelized runs. Haplotype information along with bam files were passed through readphaser to sort the reads within each block into phases based on HapCUT tables, and reassembled.</p><p id="p-21">To evaluate the resolution of the assemblies generated from each of these approaches, we mapped the raw reads back to each assembly. SNPs were called on each of these assemblies using GATK. Parameters used to evaluate assemblies were the number of heterozygous SNPs, homozygous SNPs, and number of SNPs per transcript.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-22">BAM files are deposited in SRA database under bioproject PRJNA248910: SRR1534396, SRR1535034, SRR1535035, SRR4039437, SRR4039359, SRR4039358, SRR4039357, SRR4039017, SRR4039016, SRR4039015, SRR4038966, SRR4038965, SRR4038285, SRR4037988, SRR4038281, SRR4038254, SRR4038252, SRR4038060, SRR4037987, SRR4037985, SRR4037986, and SRR4037959. <em>De novo</em> assemblies and annotations described in this manuscript have been deposited at Figshare at the following URLs: <a href="https://dx.doi.org/10.6084/m9.figshare.3580650.v2">https://dx.doi.org/10.6084/m9.figshare.3580650.v2</a>, <a href="https://dx.doi.org/10.6084/m9.figshare.3580656.v2">https://dx.doi.org/10.6084/m9.figshare.3580656.v2</a>, <a href="https://dx.doi.org/10.6084/m9.figshare.3580659.v1">https://dx.doi.org/10.6084/m9.figshare.3580659.v1</a>, and <a href="https://dx.doi.org/10.6084/m9.figshare.3580662.v1">https://dx.doi.org/10.6084/m9.figshare.3580662.v1</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Over 500 million reads provided a large dataset for de novo assembly and annotation</h3><p id="p-23">A total of 539 million filtered reads were obtained from 22 accessions including wild species plus cultivars and landraces of <em>A. hypogaea</em>, including different subspecies, and originating from different continents (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For diploids, the number of raw reads ranged from 13.8 to 24.5 million, and for tetraploids, the number of reads ranged from 18.4 to 43.4 million. <em>De novo</em> assemblies of filtered reads of diploid accessions using Trinity generated an average of 38,903 transcripts, and 27,045 unique transcripts (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) of size &gt;200 bp. Tetraploid <em>de novo</em> assemblies generated an average of 51,744 total transcripts and 29,234 unique transcripts. Overall the number of genes per diploid peanut genome averaged 28,321 based on the assemblies of 22 accessions.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469977/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469977/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><em>De novo</em> assembly characteristics of sequences from twenty-two transcriptomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">On merging the transcripts in diploids and tetraploids, we observed there was an increase in the total number of transcripts in each pool (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The resultant A-, B- and K- genome pooled assemblies had from 35,530 to 57,851 transcripts each. The A- genome pool had about 45% or more transcripts than the <em>de novo</em> assembly of a single A-genome species. The B- and K- genome pools had slightly lower ratios of pooled transcripts compared to individual species accessions (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). For the tetraploid genotypes of <em>A. hypogaea</em>, assembled transcripts were merged, and 100,328 transcripts were obtained.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469740/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469740/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Consensus assemblies generated from CAP3 for genome-specific pools</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469887/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469887/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Variants in each accession were identified in comparison to OLin as the reference</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">A consensus assembly resulted in 165,892 contigs built from 22 transcriptomes. About 79% of the contigs from the consensus assembly were annotated, and a description of hits from each database has been deposited in Figshare. Annotated transcripts were assigned to different GO categories and this consensus assembly of <em>Arachis</em> will be a valuable resource for capturing maximum number of genes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS1.xlsx">File S1</a>). Annotation of the sequences from the consensus assembly identified numerous transcription factors. Transcription factors accounted for 4.4% of the 165,892 transcripts in the consensus assembly, and these were categorized based on their DNA-binding domain (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS2.xlsx">File S2</a>). Transcription factors were assigned to different groups; b-ZIP, MADS, MYB, AP2/ERF, and C2H2, were among the top five categories (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representation of annotations of the transcripts in the consensus assembly assigned to different biological and molecular functions. (Note: Misc. signifies the transcripts that were not assigned to a category; Unknown/Predicted signifies transcripts assigned putative categories other than those listed)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-704161088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Representation of annotations of the transcripts in the consensus assembly assigned to different biological and molecular functions. (Note: Misc. signifies the transcripts that were not assigned to a category; Unknown/Predicted signifies transcripts assigned putative categories other than those listed).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3825/F1.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/12/3825/F1.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469836" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Representation of annotations of the transcripts in the consensus assembly assigned to different biological and molecular functions. (Note: Misc. signifies the transcripts that were not assigned to a category; Unknown/Predicted signifies transcripts assigned putative categories other than those listed).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of transcription factor categories in the Arachis consensus assembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-704161088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of transcription factor categories in the &lt;em&gt;Arachis&lt;/em&gt; consensus assembly.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3825/F2.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/12/3825/F2.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Overview of transcription factor categories in the <em>Arachis</em> consensus assembly.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Variant analysis identified &gt;300K SNPs</h3><p id="p-35">Across 22 accessions, we obtained 306,820 biallelic variants after filtering for quality parameters. SNPs were obtained by aligning raw reads of each accession to the OLin transcriptome (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chopra <em>et al.</em> 2014</a>). On mapping the reads to this reference, the number of raw reads aligned ranged from 84 to 89%. These aligned files were then processed further using the GATK pipeline to improve the quality of SNP calls.</p><p id="p-36">The lowest numbers of SNPs relative to OLin were found in the three cultivars Jupiter (3318), New Mexico Valencia C (3451), and Tamrun OL07 (3923). Landraces had more SNPs relative to OLin, ranging from 7081 to 10,832 SNPs (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). SNP diversity averaged about three times higher among landraces and United States peanut minicore accessions, demonstrating significant genic diversity in the cultivated species. For <em>hirsuta</em>, and <em>aequatoriana</em>, the number of SNPs ranged from 7081 to 8212 relative to OLin (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-37">For the wild species, the number of SNPs relative to OLin was far higher: for A- genome diploid accessions, from 106,882 to 117,812 per accession; for B-genome diploids, from 96,241 to 109,797; for the K-genome diploids, 157,444 to 168,289; and for natural and synthetic amphidiploids, 14,340 and 87,574, respectively (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). OLin reads were mapped back to the OLin reference for quality control of the variant calls.</p><p id="p-38">A total of 40,382 unique SNPs was present among the cultivated accessions in 14,719 transcripts, and 291,115 unique SNPs were present among the diploid accessions in 26,320 transcripts (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). The average number of SNPs among the tetraploids accessions was 0.5 per kilobase, and among the diploid accessions was 9.2 per kilobase (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-39">Many SNPs identified among accessions were unique to a genome or botanical class. A total of 191,723 SNPs was present when summed across four A- genome accessions compared to the OLin reference. We found that 43.2% of the 191,723 SNPs were shared by all four A- genome accessions, and the number of unique SNPs specific to each accession ranged from 3.4 to 6.9% (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Likewise, B- and K- genomes had 128,689 and 177,815 SNPs compared to OLin transcripts (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a>). The number of unique SNPs in <em>A. magna</em> GKSSc30097 compared to OLin was 32,430, more than the 18,092 unique SNPs between the tetraploid ancestor B- genome <em>A. ipaënsis</em> (GKPSSc30076) and OLin (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). The number of shared unique SNPs among the K- genome accessions was 147,918 relative to OLin, while 20,371 unique SNPs belonged to <em>A. batizocoi</em> K9484, and 9526 unique SNPs to <em>A. cruziana</em> KSSc36024 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams representing comparison of the number of SNPs among the diploid peanut accessions. (A) A- genome. (B) B- genome. (C) K- genome. Note: Four-way and two-way comparisons were made based on the variant sites present in the accession or a group of accessions relative to the OLin transcriptome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-704161088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagrams representing comparison of the number of SNPs among the diploid peanut accessions. (A) A- genome. (B) B- genome. (C) K- genome. Note: Four-way and two-way comparisons were made based on the variant sites present in the accession or a group of accessions relative to the OLin transcriptome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3825/F3.medium.gif" width="440" height="136"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/12/3825/F3.medium.gif" width="440" height="136"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469992" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Venn diagrams representing comparison of the number of SNPs among the diploid peanut accessions. (A) A- genome. (B) B- genome. (C) K- genome. Note: Four-way and two-way comparisons were made based on the variant sites present in the accession or a group of accessions relative to the OLin transcriptome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We found that 31,913 SNPs belonged to spanish, virginia, valencia and runners classes in comparison to the OLin transcripts (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). For two botanical types <em>hirsuta</em> and <em>aequatoriana</em>, the number of unique SNPs was 45.3 and 36.5% of 12,932 SNPs (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The wild tetraploid group included an accession derived from A- and B- genome <em>A. monticola</em> (GKBSPSc30062), and the synthetic amphidiploid A- and K- genome (TxAG-6), and the differences were reflected in the number of unique SNPs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams representing comparison of the number of SNPs among the tetraploid peanut accessions. (A) cultivated market types. (B) other botanical types. (C) tetraploid wild species and synthetic amphidiploids. Note: Four-way and two-way comparisons were made based on the variant sites present in the accession or a group of accessions relative to the OLin transcriptome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-704161088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagrams representing comparison of the number of SNPs among the tetraploid peanut accessions. (A) cultivated market types. (B) other botanical types. (C) tetraploid wild species and synthetic amphidiploids. Note: Four-way and two-way comparisons were made based on the variant sites present in the accession or a group of accessions relative to the OLin transcriptome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3825/F4.medium.gif" width="440" height="130"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/12/3825/F4.medium.gif" width="440" height="130"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Venn diagrams representing comparison of the number of SNPs among the tetraploid peanut accessions. (A) cultivated market types. (B) other botanical types. (C) tetraploid wild species and synthetic amphidiploids. Note: Four-way and two-way comparisons were made based on the variant sites present in the accession or a group of accessions relative to the OLin transcriptome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Accuracy of SNP calls was demonstrated by validation of selected SNPs by KASP marker analysis. A subset of 66 primer pairs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS4.xlsx">File S4</a>) was designed to validate the alleles called by the bioinformatic approach, and was tested on genomic DNA of 12 sequenced accessions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS3.xlsx">File S3</a>). A total of 782 SNP calls (12 genotypes and 66 primers) was selected, including the calls from diploid and tetraploid accessions for validation using KASP technology. Genotyping of two diploid accessions with 66 primers confirmed 88.52% of the 122 SNP calls, while 10 tetraploid accessions confirmed 80.00% of the 660 SNP calls (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1/FileS3.xlsx">File S3</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470022/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470022/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Summary of total and validated bioinformatic SNP calls among the sequenced accessions that included both diploids and tetraploids using allele-specific discrimination assays</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Diversity was associated with population structure</h3><p id="p-46">Analysis of SNP diversity demonstrated correlation with known botanical relationships or genome affinities. We assessed genetic diversity based on the above SNPs using PCA. Eigenvector 1 (<em>Y</em> axis) identified three groups: A-genome diploids as the first group; tetraploids, TxAG-6 (synthetic tetraploid) and K- genome as the second group; and B- genome diploids as the third group (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Eigenvector 2 (<em>X</em> axis) separated TxAG-6, K-genome diploids, and the tetraploids from each other. Clustering of genotypes for tetraploids using PCA failed to reveal evidence of subspecies of <em>hypogaea</em>, even after the third PCA including 22 accessions. To distinguish subspecies of tetraploids, we employed the PCA approach on the same set of SNPs from the tetraploids separately, and it was possible to distinguish var <em>hypogaea</em> from var <em>fastigiata</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). From the PCA (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>), we found that the B- or K- genome is closer to the tetraploids than the A- genome, but the number of variants for diploid accessions in <a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a> suggest that the B-genome is closer to the tetraploids than are the A- or K-genome diploids. From the dissimilarity matrix (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), the B-genome species were closer to the tetraploids, and the next closer parental species belonged to the A-genome. The <em>hirsuta</em> accession did not group with var <em>hypogaea</em>, contrary to expectation; however, validation of SNP calls was carried out using seed from the Plant Introduction station using KASP markers, and the phenotype was confirmed in the field. Surprisingly, the <em>A. monticola</em> accession grouped closer to the <em>fastigiata</em> accessions than to the <em>hypogaea</em> accessions, with the exception of the <em>hirsuta</em> accession, which was closer in the first principal component.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCA using SNP data. (A) Twenty-two genotypes analyzed, which included diploids and tetraploids. Dot colors mark genome affinities of accessions as follows: black dots, A-genome accessions; red dots, B-genome accessions; green dots, K-genome accessions; blue dot, synthetic amphidiploid; cyan dot, natural tetraploids. (B) Cultivated tetraploids along with Arachis monticola were analyzed with diploids omitted. Dot colors mark market types of accessions as follows: violet, valencias; blue, spanish; green, aequatoriana; red, runners; orange, virginias; pink, hirsutas; black A. monticola." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-704161088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PCA using SNP data. (A) Twenty-two genotypes analyzed, which included diploids and tetraploids. Dot colors mark genome affinities of accessions as follows: black dots, A-genome accessions; red dots, B-genome accessions; green dots, K-genome accessions; blue dot, synthetic amphidiploid; cyan dot, natural tetraploids. (B) Cultivated tetraploids along with &lt;em&gt;Arachis monticola&lt;/em&gt; were analyzed with diploids omitted. Dot colors mark market types of accessions as follows: violet, valencias; blue, spanish; green, aequatoriana; red, runners; orange, virginias; pink, hirsutas; black &lt;em&gt;A. monticola&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3825/F5.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/12/3825/F5.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470032" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">PCA using SNP data. (A) Twenty-two genotypes analyzed, which included diploids and tetraploids. Dot colors mark genome affinities of accessions as follows: black dots, A-genome accessions; red dots, B-genome accessions; green dots, K-genome accessions; blue dot, synthetic amphidiploid; cyan dot, natural tetraploids. (B) Cultivated tetraploids along with <em>Arachis monticola</em> were analyzed with diploids omitted. Dot colors mark market types of accessions as follows: violet, valencias; blue, spanish; green, aequatoriana; red, runners; orange, virginias; pink, hirsutas; black <em>A. monticola</em>.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dendrogram based on the distance matrix obtained from the polymorphism based on biallelic sites on different groups. (A) Natural tetraploids with their possible ancestors. (B) Cultivated tetraploids with their possible ancestors. (C) Synthetic tetraploid with parents from which the amphidiploid was derived." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-704161088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dendrogram based on the distance matrix obtained from the polymorphism based on biallelic sites on different groups. (A) Natural tetraploids with their possible ancestors. (B) Cultivated tetraploids with their possible ancestors. (C) Synthetic tetraploid with parents from which the amphidiploid was derived.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3825/F6.medium.gif" width="440" height="129"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/12/3825/F6.medium.gif" width="440" height="129"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3825/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469932" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Dendrogram based on the distance matrix obtained from the polymorphism based on biallelic sites on different groups. (A) Natural tetraploids with their possible ancestors. (B) Cultivated tetraploids with their possible ancestors. (C) Synthetic tetraploid with parents from which the amphidiploid was derived.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Phasing separated some of the merged homeologous sequences</h3><p id="p-49">There was evidence for significant collapse of homeologous sequences, and phasing separated a large number of merged homeologous sequences in this study. Our results among cultivars and landraces suggested that the frequency of SNPs in tetraploids was 0.5 SNP per kilobase or 0.09 per kilobase in pairwise comparisons, but we observed higher SNP rates on mapping the raw reads back to the OLin transcriptome, which could be the consequence of homeolog transcript collapse. To identify the variants in phased sequences, we aligned the sequences back to the OLin reference transcriptome. There were a total of 25,488 contigs (53.4%) that contained &gt;1 SNP, which were therefore good candidates for polymorphism phasing (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). Assuming that &gt;1 SNP per transcript could be a result of homeolog collapse, we selected transcripts with &gt;1 SNP to separate transcripts using the HapCUT program. We phased SNPs in 22,683 contigs containing &gt;1 SNP into 30,672 sub-transcripts, resulting in an average of 1.35 separated transcripts per contig. During the readphaser process, only 10,815 of 22,683 phased contigs were carried forward for assembly, as the remaining contigs had either evidence of a third haplotype or low read support. Phased assemblies consisted of a total of 81,244 contigs, which were further used to evaluate the separation of homeologs.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469938/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469938/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Summary of the assembly and SNP calls using <em>de novo</em> and phased approach on OLin transcriptome</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">The phasing strategy generated higher numbers of contigs than the <em>de novo</em> assembly approach, but had lesser representation of the transcriptome, as only 76–78% of the reads mapped back. Variant calls from phased assemblies indicated a sharp decrease in the number of heterozygous SNPs, and the number of contigs with multiple SNPs per transcript was lower compared to the <em>de novo</em> assembly (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). The number of contigs with &gt;2 SNPs was reduced by ∼33%, while the number of contigs with &gt;1 SNP was reduced by 20% in the phased assembly approach compared to the <em>de novo</em> approach.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>Significant genic diversity exists in peanut</h3><p id="p-52">SNP diversity analysis demonstrated the presence of significant genic diversity among cultivated accessions and wild species. The number of SNPs identified across all 22 genotypes suggested less genetic variation among <em>A. hypogaea</em> accessions compared to the wild diploids, which had substantially higher genetic variation. High phenotypic diversity observed among diploid accessions is consistent with genetic variation identified in comparison to the OLin reference. For the United States cultivars, the number of homozygous SNPs was in the range of 3000–3500, which agreed with the data previously reported by <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Chopra <em>et al.</em> (2015)</a>. Among the landraces of tetraploid peanut, homozygous SNPs ranged from 7000 to 10,000 SNPs, double to triple the number in United States cultivars. Such a high number of SNPs among the tetraploid accessions suggested high genic diversity exists compared to previous reports of low, or no, molecular variability (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Kochert <em>et al.</em> 1996</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2009</a>) using RFLP markers. Also, this genic variability among the landraces could reflect the level of phenotypic diversity observed among core and minicore collections of peanut. Indeed, four of the accessions screened in this study were chosen from the United States peanut minicore collection. Previous minicore screens have also shown that there is huge variability for traits such as water use efficiency (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Singh <em>et al.</em> 2014</a>), oil content (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Wang <em>et al.</em> 2011a</a>,<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">b</a>), bacterial wilt disease (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>), resistance to abiotic and biotic stresses (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Upadhyaya <em>et al.</em> 2013</a>), and agronomic traits (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Pandey <em>et al.</em> 2014</a>). Sufficient variation has been identified among tetraploid accessions through transcriptome sequencing for performing QTL analysis with high density marker linkage maps.</p><p id="p-53">To call SNPs on 22 accessions, we aligned raw reads to the reference (OLin). The raw reads aligning to the reference ranged from 84 to 89%, suggesting that the assembly used in this study had a good representation of the peanut transcriptomes. A total of 520 SNPs was identified by comparing the OLin transcriptome to the reference assembly; this may suggest the presence of gene copy number variation, call errors, or artifacts in the software used. The presence of SNPs when mapping reads back to the same reference has been cited previously (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nielsen <em>et al.</em> 2011</a>).</p><p id="p-54">The SNP rate in tetraploid peanuts identified was 0.5 per kilobase, which is lower than other related legume species such as soybean (2.7 SNPs per kilobase) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Choi <em>et al.</em> 2007</a>), field pea (2.7 SNPs per kilobase) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Leonforte <em>et al.</em> 2013</a>), and <em>Medicago truncatula</em> (1.96 SNPs per kilobase) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Choi <em>et al.</em> 2004</a>). However, in diploid species of peanut, including different genome affinities, SNP rates reported here were higher than in other legumes.</p></div><div id="sec-16" class="subsection"><h3>Phylogenetic analysis demonstrated that SNP diversity is associated with population structure</h3><p id="p-55">Previous genetic diversity analyses have indicated clear differentiation of diploid species and tetraploid subspecies (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2009</a>). Based on thousands of SNPs identified in this study, we also differentiated tetraploids from the diploids (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>), and individuals of the two subspecies of <em>A. hypogaea</em> (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). The number of SNPs differentiating accessions of <em>A. hypogaea</em> and <em>A. monticola</em> was also higher compared to differences among cultivars, providing evidence of additional untapped genetic diversity.</p><p id="p-56">In general, the <em>hypogaea</em> accessions grouped together; likewise the <em>fastigiata</em> accessions. Two unusual results were in the first PCA results, in which the <em>hirsuta</em> accession and <em>A. monticola</em> were closer to each other and to the <em>fastigiata</em> accessions. This is surprising, because the spreading type of these accessions matches the <em>hypogaea</em> accessions. However, there are other phenotypic data to consider—for example, the pronounced reticulation of pods of <em>hirsuta</em>, <em>peruviana</em>, and <em>aequatoriana</em> accessions resembles each other more than the typical <em>hypogaea</em> or <em>fastigiata</em> accessions. As with the case of the greater similarity of <em>A. monticola</em> to the <em>fastigiata</em> type, inclusion of data from other accessions in a separate study would be warranted before making broader conclusions, as the present data are based on only a single accession.</p></div><div id="sec-17" class="subsection"><h3>Validation of SNPs demonstrated high potential utility for marker-assisted breeding using KASP markers</h3><p id="p-57">Another goal of this research was to identify SNP markers that could be useful in breeding programs. Of the 782 selected SNP calls tested, 81% were confirmed on the LightCycler 480, which is significantly better than results of previous studies in peanut (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Khera <em>et al.</em> 2013</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Chopra <em>et al.</em> 2015</a>). The two diploid accessions used for validation were selected to demonstrate the approach used for identification of SNPs in peanut, and for development of a genetic map (R. Chopra <em>et al.</em>, unpublished results). Tetraploid accessions were used to confirm the SNP calls that could be eventually used in population studies. The overall validation success rate of 81.32% of 782 selected SNP sites was much higher than in a previous study by <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Chopra <em>et al.</em> (2015)</a>. Validation using KASP chemistry suggested that the variant calling approach is effective in diploid and tetraploid peanut. This provides a basis for selecting SNP markers between the parents for future breeding activities.</p><p id="p-58">Annotations of the transcripts in the consensus assembly and the OLin assembly by <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chopra <em>et al.</em> (2014)</a> provided evidence of SNPs in important trait-related transcripts. For example, SNPs were observed in oil biosynthesis genes, and in the genes of QTL regions affecting nematode resistance. Diagnostic KASP markers were designed and are being evaluated in the breeding program (R. Chopra <em>et al.</em>, unpublished results). Collectively, these datasets have the potential for enabling SNP identification throughout the genome for any polymorphic gene of interest, thus making the tagging of specific genes with molecular markers a possibility.</p></div><div id="sec-18" class="subsection"><h3>Unique genes per peanut genome are in the range of 25–32K</h3><p id="p-59">Results of <em>de novo</em> assembly suggest that there are about 25,000–31,500 unique transcripts in diploid and tetraploid <em>Arachis</em> roots, leaves, and developing pod tissues. This number could be a minimum value, as not all tissues were sampled. The number of transcripts in tetraploids could be twice that of diploids because of homeologous copies. Given the allo-tetraploid nature of the peanut, genes from both genomes are similar, and tools used in the study in general could not distinguish homeologous genes from each other. Consensus assemblies had more contigs than any other assembly by itself, which suggested that each genotype has a certain percentage of transcripts that could not be merged at a minimum sequence similarity threshold of 95%. This increase in the number of transcripts could be due to the larger differences among species/accessions used, or differences in expression of transcripts in samples. For tetraploids, the increase in number of transcripts in the consensus assembly can also be a result of the presence of homeologs. Consensus and individual assemblies from this study could benefit annotation of the genome for rare transcripts and gene expression studies.</p></div><div id="sec-19" class="subsection"><h3>Phasing of homeologous copies requires longer reads</h3><p id="p-60">In the case of diploids, the success rate of SNP validation was greater than for tetraploid calls. Complexity of the tetraploid genome resulting from homeologous copies may have affected the outcomes of validation. Based on previous studies in complex polyploids, such as wheat (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Schreiber <em>et al.</em> 2012</a>) and peanut (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Chopra <em>et al.</em> 2014</a>), it was observed that a significant number of homeologs would be merged even after optimizing the <em>de novo</em> assembly parameters. Tetraploid SNP calls consisted of many apparently heterozygous state genotypes, and to reduce these heterozygous calls, we evaluated the phasing approach. We utilized the postassembly approach reported by <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Krasileva <em>et al.</em> (2013)</a> to separate merged homeologous transcripts with assemblers. This approach gave a more complete overview of the tetraploid transcriptome with a smaller number of chimeric transcripts. Ideally, mapping the reads back to the reference derived from same accession should give fewer variant calls, but, due to collapse of some homeologs, the heterozygous variant calls were more in the <em>de novo</em> assembly. One way to estimate the reduction of homeolog collapse would be to observe the number of heterozygous variants upon mapping the reads back. Reduction in heterozygous SNP calls compared to the reference would be the first step for improving the assembly qualities.</p><p id="p-61">After evaluating the results of resolving homeologs, we suggest that, to get a complete separation of transcripts with no collapse would require: (a) longer sequence reads, because shorter reads can affect the extension of contigs with the poor quality on the ends of reads; (b) higher read coverage to support the haplotypes, because lower read depth haplotypes could be dropped by the phasing based approach; and (c) using the genome sequence of diploid progenitor species to avoid collapse, and to obtain a better representation of transcripts.</p></div><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-62">We report the first survey of significant SNP level and transcript level diversity in a wide array of peanut germplasm. Highlighted untapped genetic variability among accessions may explain some of the phenotypic differences observed in germplasm surveys. Transcriptome analysis among cultivated accessions, and comparison to wild species will open many avenues for future research. Among these are relationships of cultivated tetraploid species to their diploid ancestors, evolutionary development of the cultigen, distinguishing of homologous <em>vs.</em> homeologous SNPs for genetic mapping and QTL analysis, study of genome rearrangements after polyploidization, and expression bias among genes and genomes. Outcomes of this research will help increase of genomic resources in peanut, and benefit QTL mapping relevant for agronomic traits.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">We thank Halee Hughes, United States Department of Agriculture–Agriculture Research Service for technical support. This work was funded by grants from the Texas Peanut Producers Board award CY2008-Burow- TTU-Development to M.D.B. and C.E.S., and 2009-TTU-Burow-Genotyping to M.D.B., National Peanut Board grant #332/TX-99/1139 to M.D.B., and #332/TX-99/1213 to M.D.B. and C.E.S., Peanut Foundation grant 04-810-08 to M.D.B., Ogallala Aquifer Initiative award IPM12.06 to M.D.B., and United States Department of Agriculture/National Institute of Food and Agriculture Hatch Act award TEX08835 to M.D.B. The authors declare that they have no conflict of interest. All experiments were performed in accordance with current biomedical research ethical standards in the United States.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-13"><p id="p-64">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026898/-/DC1</a>.</p></li><li class="fn" id="fn-14"><p id="p-65"><a class="rev-xref" href="#xref-fn-14-1">↵</a><em>Communicating editor: J. Wendel</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-30"><span class="received-label">Received </span>July 30, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-03"><span class="accepted-label">Accepted </span>September 3, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Chopra <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culbreath</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Screening the peanut core collection for resistance to tomato spotted wilt virus.</span> <abbr class="cit-jnl-abbrev">Peanut Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeanut%2BSci.%26rft.volume%253D23%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.2" data-doi="10.1093/bioinformatics/btn298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bafna</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">HapCUT: an efficient and accurate algorithm for the haplotype assembly problem.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D24%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.12.3825.3" data-doi="10.2135/cropsci2006.06.0413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baring</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Registration of ‘Tamrun OL07’peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">2721</span>–<span class="cit-lpage">2722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBaring%26rft.auinit1%253DM.R.%26rft.volume%253D46%26rft.issue%253D6%26rft.spage%253D2721%26rft.epage%253D2722%26rft.atitle%253DRegistration%2Bof%2B%2527Tamrun%2BOL07%2527%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.06.0413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.06.0413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An overview of peanut and its wild relatives.</span> <abbr class="cit-jnl-abbrev">Plant Genet. Resour.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%26rft.volume%253D9%26rft.spage%253D134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.5" data-doi="10.1038/ng.3517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froenicke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genome sequences of <em>Arachis duranensis</em> and <em>Arachis ipaënsis</em>, the diploid ancestors of cultivated peanut.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D438%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3517%26rft_id%253Dinfo%253Apmid%252F26901068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26901068&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.6" data-doi="10.1139/G08-094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faries</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular biogeography study of recently described B- and A-genome <em>Arachis</em> species, also providing new insights into the origins of cultivated peanut.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBurow%26rft.auinit1%253DM.%2BD.%26rft.volume%253D52%26rft.issue%253D2%26rft.spage%253D107%26rft.epage%253D119%26rft.atitle%253DMolecular%2Bbiogeographic%2Bstudy%2Bof%2Brecently%2Bdescribed%2BB-%2Band%2BA-genome%2BArachis%2Bspecies%252C%2Balso%2Bproviding%2Bnew%2Binsights%2Binto%2Bthe%2Borigins%2Bof%2Bcultivated%2Bpeanut.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG08-094%26rft_id%253Dinfo%253Apmid%252F19234559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G08-094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19234559&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.12.3825.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Introgression of homoeologous quantitative trait loci (QTLs) for resistance to the root-knot nematode [<em>Meloidogyne arenaria</em> (Neal) Chitwood] in an advanced backcross-QTL population of peanut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deep sequencing analysis of the transcriptomes of peanut aerial and subterranean young pods identifies candidate genes related to early embryo abortion.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.9" data-doi="10.1534/genetics.166.3.1463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhm</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limpens</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A sequence based genetic map of <em>Medicago truncatula</em> and comparison of marker collinearity with <em>M. sativa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1463</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChoi%26rft.auinit1%253DH.-K.%26rft.volume%253D166%26rft.issue%253D3%26rft.spage%253D1463%26rft.epage%253D1502%26rft.atitle%253DA%2BSequence-Based%2BGenetic%2BMap%2Bof%2BMedicago%2Btruncatula%2Band%2BComparison%2Bof%2BMarker%2BColinearity%2Bwith%2BM.%2Bsativa%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.3.1463%26rft_id%253Dinfo%253Apmid%252F15082563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni8zLzE0NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.10" data-doi="10.1534/genetics.107.070821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaky</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070821%26rft_id%253Dinfo%253Apmid%252F17339218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNjg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.11" data-doi="10.1371/journal.pone.0115055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chopra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mudge</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparisons of <em>de novo</em> transcriptome assemblers in diploid and polyploid species using peanut (<em>Arachis</em> spp.) RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e115055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De115055%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0115055%26rft_id%253Dinfo%253Apmid%252F25551607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0115055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25551607&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chopra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mudge</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Next-generation transcriptome sequencing, SNP discovery and validation in four market classes of peanut, <em>Arachis hypogaea</em> L.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D290%26rft.spage%253D1169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.13" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzgyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.14" data-doi="10.1186/1471-2164-15-708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deokar</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diapari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sindhu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D708%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-708%26rft_id%253Dinfo%253Apmid%252F25150411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25150411&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.15" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Cromosomas y evolución en <em>Arachis</em> (Leguminosae).</span> <abbr class="cit-jnl-abbrev">Bonplandia</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBonplandia%26rft.volume%253D8%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.17" data-doi="10.1093/dnares/dsq028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>De novo</em> assembly of chickpea transcriptome using short reads for gene discovery and marker identification.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsq028%26rft_id%253Dinfo%253Apmid%252F21217129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjc6IjE4LzEvNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.18" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3825.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Groundnut</span>, pp. <span class="cit-fpage">151</span>–<span class="cit-lpage">154</span> in <span class="cit-source"><em>Evolution of Crop Plants</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">N. W.</span></span>. <span class="cit-publ-name">Longman Group, Ltd.</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3825.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Structure, variation, evolution, and classification in <em>Arachis</em></span>, pp. <span class="cit-fpage">469</span>–<span class="cit-lpage">481</span> in <span class="cit-source"><em>Advances in Legume Science</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Summerfield</span>  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bunting</span>  <span class="cit-name-given-names">A. H.</span></span>. <span class="cit-publ-name">Kew: Royal Botanic Gardens</span>, <span class="cit-publ-loc">Richmond, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.21" data-doi="10.1139/g91-156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halward</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larue</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kochert</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic variation detectable with molecular markers among unadapted germplasm resources of cultivated peanut and related wild species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D34%26rft.spage%253D1013%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg91-156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g91-156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.22" data-doi="10.2135/cropsci2004.0368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Development and evaluation of a minicore collection for the US peanut germplasm collection.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1540</span>–<span class="cit-lpage">1544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHolbrook%26rft.auinit1%253DC.%2BC.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D1540%26rft.epage%253D1544%26rft.atitle%253DDevelopment%2Band%2BEvaluation%2Bof%2Ba%2BMini%2BCore%2BCollection%2Bfor%2Bthe%2BU.S.%2BPeanut%2BGermplasm%2BCollection%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000230713700046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pittman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Selection of a core collection from the US germplasm collection of peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHolbrook%26rft.auinit1%253DC.%2BC.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D859%26rft.epage%253D861%26rft.atitle%253DSelection%2Bof%2Ba%2BCore%2BCollection%2Bfrom%2Bthe%2BU.S.%2BGermplasm%2BCollection%2Bof%2BPeanut%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1993LP37100043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timper</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culbreath</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvien</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Registration of ‘Tifguard’ peanut.</span> <abbr class="cit-jnl-abbrev">J. of Plant Regist.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2Bof%2BPlant%2BRegist.%26rft.volume%253D2%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.12.3825.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsi</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Registration of ‘New Mexico Valencia C’ peanut (Reg. No. 24).</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHsi%26rft.auinit1%253DD.%2BC.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D114%26rft.atitle%253DRegistration%2Bof%2BNew%2BMexico%2BValencia%2BC%2BPeanut%2B%2528Reg.%2BNo.%2B24%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Abundant microsatellite diversity and oil content in wild <em>Arachis</em> species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e50002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHuang%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De50002%26rft.epage%253De50002%26rft.atitle%253DAbundant%2Bmicrosatellite%2Bdiversity%2Band%2Boil%2Bcontent%2Bin%2Bwild%2Barachis%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F23185514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23185514&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.27" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: A DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isleib</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattee</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giesbrecht</span>  <span class="cit-name-given-names">F. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Ancestral Contributions to Roasted Peanut Attribute.</span> <abbr class="cit-jnl-abbrev">Peanut Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeanut%2BSci.%26rft.volume%253D22%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Development of a core collection of peanut germplasm in China.</span> <abbr class="cit-jnl-abbrev">Proc. Amer. Peanut Res. Educ. Soc.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BAmer.%2BPeanut%2BRes.%2BEduc.%2BSoc.%26rft.volume%253D36%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiaoping</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuning</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiaojing</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diversity characterization and association analysis of agronomic traits in a Chinese peanut (<em>Arachis hypogaea</em> L.) mini-core collection.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D56%26rft.spage%253D159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khera</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sriswathi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single nucleotide polymorphism-based genetic diversity in the reference set of peanut (spp.) by developing and applying cost-effective kompetitive allele specific polymerase chain reaction genotyping assays.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kochert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halward</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branch</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">RFLP variability in peanut (<em>Arachis hypogaea</em> L.) cultivars and wild species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D81%26rft.spage%253D565%26rft_id%253Dinfo%253Apmid%252F24221368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24221368&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FP48900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.33" data-doi="10.2307/2446112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kochert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gimenes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galgaro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">RFLP and cytogenetic evidence on the origin and evolution of allotetraploid domesticated peanut <em>Arachis hypogaea</em> (Leguminosae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1282</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D83%26rft.spage%253D1282%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2446112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2446112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VN75500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Taxonomia del género <em>Arachis</em> (Leguminosae).</span> <abbr class="cit-jnl-abbrev">Bonplandia</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBonplandia%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.35" data-doi="10.1186/gb-2013-14-6-r66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buffalo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearce</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayling</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Separating homoeologs by phasing in the tetraploid wheat transcriptome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR66%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r66%26rft_id%253Dinfo%253Apmid%252F23800085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23800085&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.36" data-doi="10.1508/cytologia.63.177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavia</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Karyotypes of <em>Arachis palustris</em> and <em>A. praecox</em> (section <em>Arachis</em>), two species with basic chromosome number x = 9.</span> <abbr class="cit-jnl-abbrev">Cytologia (Tokyo)</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytologia%2B%2528Tokyo%2529%26rft.volume%253D63%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1508%252Fcytologia.63.177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1508/cytologia.63.177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cytogenetic and molecular evidences on the evolutionary relationships among <em>Arachis</em> species. plant genome: biodiversity and evolution</span>. <abbr class="cit-jnl-abbrev">Phanerogams-Angiosperm.</abbr> <span class="cit-vol">1E</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhanerogams-Angiosperm.%26rft.volume%253D1%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leonforte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudheesh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salisbury</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP marker discovery, linkage map construction and identification of QTLs for enhanced salinity tolerance in field pea (<em>Pisum sativum</em> L.)</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D13%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.39" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzgyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.40" data-doi="10.1186/1471-2164-13-568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bharti</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prevalence of single nucleotide polymorphism among 27 diverse alfalfa genotypes as assessed by transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D568%26rft.epage%253D568%26rft.atitle%253DPrevalence%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bamong%2B27%2Bdiverse%2Balfalfa%2Bgenotypes%2Bas%2Bassessed%2Bby%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-568%26rft_id%253Dinfo%253Apmid%252F23107476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23107476&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.41" data-doi="10.1111/pce.12231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herter</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mercator: a fast and simple web server for genome scale functional annotation of plant sequence data.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1250</span>–<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D37%26rft.spage%253D1250%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000333705200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickersgill</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Isozyme variation and species relationships in peanut and its wild relatives (<em>Arachis</em> L. - Leguminosae).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D550%26rft_id%253Dinfo%253Apmid%252F24195929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24195929&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KH04900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.43" data-doi="10.1007/s10722-010-9627-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mallikarjuna</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senthilvel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoisington</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development of new sources of tetraploid <em>Arachis</em> to broaden the genetic base of cultivated groundnut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D58%26rft.spage%253D889%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-010-9627-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-010-9627-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.44" data-doi="10.1534/g3.115.017012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R.F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RIG: recalibration and interrelation of genomic sequence data with the GATK.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.017012%26rft_id%253Dinfo%253Apmid%252F25681258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC82NTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.45" data-doi="10.1186/1471-2229-4-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic diversity of peanut (<em>Arachis hypogaea</em> L.) and its wild relatives based on the analysis of hypervariable regions of the genome.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMoretzsohn%26rft.auinit1%253DM.%2Bd.%2Be.%2B%2B.%2BC.%26rft.volume%253D4%26rft.spage%253D11%26rft.epage%253D11%26rft.atitle%253DGenetic%2Bdiversity%2Bof%2Bpeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529%2Band%2Bits%2Bwild%2Brelatives%2Bbased%2Bon%2Bthe%2Banalysis%2Bof%2Bhypervariable%2Bregions%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-4-11%26rft_id%253Dinfo%253Apmid%252F15253775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-4-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15253775&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.46" data-doi="10.1186/1471-2164-13-469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">E.D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okashah</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-density genetic map of <em>Arachis duranensis</em>, a diploid ancestor of cultivated peanut.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNagy%26rft.auinit1%253DE.%2BD.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D469%26rft.epage%253D469%26rft.atitle%253DA%2Bhigh-density%2Bgenetic%2Bmap%2Bof%2BArachis%2Bduranensis%252C%2Ba%2Bdiploid%2Bancestor%2Bof%2Bcultivated%2Bpeanut.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-469%26rft_id%253Dinfo%253Apmid%252F22967170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22967170&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.47" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3825.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norden</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Breeding of the cultivated peanut (<em>Arachis hypogaea</em> L.)</span>, pp. <span class="cit-fpage">175</span>–<span class="cit-lpage">208</span> in <span class="cit-source"><em>Peanuts—Cultures and Uses</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C. T.</span></span>. <span class="cit-publ-name">Amer. Peanut Res. Educ. Soc.</span>, <span class="cit-publ-loc">Stillwater</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.12.3825.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norden</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipscomb</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Registration of ‘Florunner’ peanuts (Reg No. 2).</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DNorden%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D850%26rft.epage%253D850%26rft.atitle%253DRegistration%2Bof%2BFlorunner%2BPeanuts%2B%2528Reg.%2BNo.%2B2%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rathore</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vadez</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheshshayee</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide association studies for 50 agronomic traits in peanut using the ‘reference set’ comprising 300 genotypes from 48 countries of the semi-arid tropics of the world.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e105228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De105228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raina</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukai</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic in situ hybridization in <em>Arachis</em> (Fabaceae) identifies the diploid wild progenitors of cultivated (<em>A. hypogaea</em>) and related wild (<em>A. monticola</em>) peanut species.</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D214%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raina</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rani</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogihara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">RAPD and ISSR fingerprints as useful genetic markers for analysis of genetic diversity, varietal identification, and phylogenetic relationships in peanut (<em>Arachis hypogaea</em>) cultivars and wild species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DRaina%26rft.auinit1%253DS.%2BN.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D763%26rft.epage%253D772%26rft.atitle%253DRAPD%2Band%2BISSR%2Bfingerprints%2Bas%2Buseful%2Bgenetic%2Bmarkers%2Bfor%2Banalysis%2Bof%2Bgenetic%2Bdiversity%252C%2Bvarietal%2Bidentification%252C%2Band%2Bphylogenetic%2Brelationships%2Bin%2Bpeanut%2B%2528Arachis%2Bhypogaea%2529%2Bcultivars%2Band%2Bwild%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F11681599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11681599&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome-scale homoeolog-specific transcript assemblies of bread wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSchreiber%26rft.auinit1%253DA.%2BW.%26rft.volume%253D13%26rft.spage%253D492%26rft.epage%253D492%26rft.atitle%253DTranscriptome-scale%2Bhomoeolog-specific%2Btranscript%2Bassemblies%2Bof%2Bbread%2Bwheat.%26rft_id%253Dinfo%253Apmid%252F22989011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22989011&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.54" data-doi="10.3732/ajb.94.12.1963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavia</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducasse</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic relationships between the cultivated peanut (<em>Arachis hypogaea</em>, Leguminosae) and its close relatives revealed by double GISH.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DSeijo%26rft.auinit1%253DG.%26rft.volume%253D94%26rft.issue%253D12%26rft.spage%253D1963%26rft.epage%253D1971%26rft.atitle%253DGenomic%2Brelationships%2Bbetween%2Bthe%2Bcultivated%2Bpeanut%2B%2528Arachis%2Bhypogaea%252C%2BLeguminosae%2529%2Band%2Bits%2Bclose%2Brelatives%2Brevealed%2Bby%2Bdouble%2BGISH%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.94.12.1963%26rft_id%253Dinfo%253Apmid%252F21636391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI5NC8xMi8xOTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selvaraj</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belamkar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puppala</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heat stress screening of peanut (<em>Arachis hypogaea</em> L.) seedlings for acquired thermotolerance.</span> <abbr class="cit-jnl-abbrev">Plant Growth Regul.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGrowth%2BRegul.%26rft.volume%253D65%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Use of wild <em>Arachis</em> species/introgression of genes into <em>A. hypogaea</em> L.</span> <abbr class="cit-jnl-abbrev">Peanut Sci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeanut%2BSci.%26rft.volume%253D28%26rft.spage%253D114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.57" data-doi="10.2135/cropsci2001.413918x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Registration of ‘COAN’ peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSimpson%26rft.auinit1%253DC.E.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D918%26rft.epage%253D918%26rft.atitle%253DRegistration%2Bof%2B%2527COAN%2527%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2001.413918x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2001.413918x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170696200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodard</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Registration of TxAG-6 and TxAG-7 peanut germplasm lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSimpson%26rft.auinit1%253DC.%2BE.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D1418%26rft.epage%253D1418%26rft.atitle%253DRegistration%2Bof%2BTxAG-6%2Band%2BTxAG-7%2BPeanut%2BGermplasm%2BLines%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baring</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melouk</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">Registration of ‘OLin’ peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1880</span>–<span class="cit-lpage">1881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSimpson%26rft.auinit1%253DC.E.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D1880%26rft.epage%253D1881%26rft.atitle%253DRegistration%2Bof%2B%2527OLin%2527%2BPeanut%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000185174300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.60" data-doi="10.2135/cropsci2003.1561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">Registration of NemaTAM peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSimpson%26rft.auinit1%253DC.E.%26rft.volume%253D43%26rft.issue%253D4%26rft.spage%253D1561%26rft.epage%253D1561%26rft.atitle%253DRegistration%2Bof%2B%2527NemaTAM%2527%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183762800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baring</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Registration of ‘Webb’ Peanut.</span> <abbr class="cit-jnl-abbrev">J. Plant Regist.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegist.%26rft.volume%253D7%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakar</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalariya</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Physiological efficiencies in mini-core peanut germplasm accessions during summer season.</span> <abbr class="cit-jnl-abbrev">Photosynthetica</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhotosynthetica%26rft.volume%253D52%26rft.spage%253D627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.63" data-doi="10.1016/S0169-5347(99)01638-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Polyploidy: recurrent formation and genome evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DSoltis%26rft.auinit1%253DD.%2BE.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D348%26rft.epage%253D352%26rft.atitle%253DPolyploidy%253A%2Brecurrent%2Bformation%2Band%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252899%252901638-9%26rft_id%253Dinfo%253Apmid%252F10441308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(99)01638-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10441308&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081781400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.64" data-doi="10.1016/S0065-2113(08)60801-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moss</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Speciation, cytogenetics, and utilization of <em>Arachis</em> species.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D41%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2113%252808%252960801-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2113(08)60801-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.65" data-doi="10.1093/nar/gku989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">UniProt Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">UniProt: a hub for protein information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D204</span>–<span class="cit-lpage">D212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku989%26rft_id%253Dinfo%253Apmid%252F25348405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My9EMS9EMjA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Phenotypic diversity in groundnut (<em>Arachis hypogaea</em> L.) core collection assessed by morphological and agronomical evaluations.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D50%26rft.spage%253D539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.67" data-doi="10.2135/cropsci2004.0389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Variability for drought resistance related traits in the mini core collection of peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1432</span>–<span class="cit-lpage">1440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DUpadhyaya%26rft.auinit1%253DH.%2BD.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D1432%26rft.epage%253D1440%26rft.atitle%253DVariability%2Bfor%2BDrought%2BResistance%2BRelated%2BTraits%2Bin%2Bthe%2BMini%2BCore%2BCollection%2Bof%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000230713700031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bramel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Developing a mini core of peanut for utilization of genetic resources.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2150</span>–<span class="cit-lpage">2156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DUpadhyaya%26rft.auinit1%253DH.%2BD.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D2150%26rft.epage%253D2156%26rft.atitle%253DDeveloping%2Ba%2BMini%2BCore%2Bof%2BPeanut%2Bfor%2BUtilization%2Bof%2BGenetic%2BResources%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000181430200052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.69" data-doi="10.1016/j.fcr.2005.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">C. L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of diverse groundnut germplasm: sources of early-maturity in a core collection.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D97%26rft.spage%253D261%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2005.10.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2005.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">C. L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buhariwalla</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crouch</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Efficient use of crop germplasm resources: identifying useful germplasm for crop improvement through core and mini-core collections and molecular marker approaches.</span> <abbr class="cit-jnl-abbrev">Plant Gen. Res.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%2BRes.%26rft.volume%253D4%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3825.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwivedi</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic resources, diversity and association mapping in peanut</span>, pp. <span class="cit-fpage">13</span>–<span class="cit-lpage">36</span> in <span class="cit-source"><em>Genetics</em>, <em>Genomics and Breeding of Peanuts</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Mallikarjuna</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">New species of <em>Arachis</em> L. (Leguminosae) from Brazil, Paraguay and Bolivia.</span> <abbr class="cit-jnl-abbrev">Bonplandia</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBonplandia%26rft.volume%253D14%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkley</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Population structure and marker–trait association analysis of the US peanut (<em>Arachis hypogaea</em> L.) mini-core collection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DM.%2BL.%26rft.volume%253D123%26rft.issue%253D8%26rft.spage%253D1307%26rft.epage%253D1317%26rft.atitle%253DPopulation%2Bstructure%2Band%2Bmarker-trait%2Bassociation%2Banalysis%2Bof%2Bthe%2BUS%2Bpeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529%2Bmini-core%2Bcollection.%26rft_id%253Dinfo%253Apmid%252F21822942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21822942&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkley</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pittman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">FAD2 gene mutations significantly alter fatty acid profiles in cultivated peanuts (<em>Arachis hypogaea</em>).</span> <abbr class="cit-jnl-abbrev">Biochem. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Bgenetics%26rft.stitle%253DBiochem%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DM.%2BL.%26rft.volume%253D49%26rft.issue%253D11-12%26rft.spage%253D748%26rft.epage%253D759%26rft.atitle%253DFAD2%2Bgene%2Bmutations%2Bsignificantly%2Balter%2Bfatty%2Bacid%2Bprofiles%2Bin%2Bcultivated%2Bpeanuts%2B%2528Arachis%2Bhypogaea%2529.%26rft_id%253Dinfo%253Apmid%252F21681577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21681577&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuong</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNP discovery by high-throughput sequencing in soybean.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D11%26rft.spage%253D469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.76" data-doi="10.1186/1471-2164-13-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>De novo</em> assembly and characterisation of the transcriptome during seed development, and generation of genic-SSR markers in peanut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.spage%253D90%26rft.epage%253D90%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Band%2Bcharacterisation%2Bof%2Bthe%2Btranscriptome%2Bduring%2Bseed%2Bdevelopment%252C%2Band%2Bgeneration%2Bof%2Bgenic-SSR%2Bmarkers%2Bin%2Bpeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-90%26rft_id%253Dinfo%253Apmid%252F22409576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22409576&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3825.77" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjQvMzMyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzgyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/12/3825.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3817" title="Tissue Culture as a Source of Replicates in Nonmodel Plants: Variation in Cold Response in Arabidopsis lyrata ssp. petraea" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3837" title="Recurrence of Chromosome Rearrangements and Reuse of DNA Breakpoints in the Evolution of the Triticeae Genomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/12"><span>Volume 6 Issue 12, December 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/12.cover-source.jpg?itok=F-Hw09b0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3825" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-k3ZpBbG8otpKJpv2dv2nrU3VQkXwnalyYcAaULtnLZI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/12/3825" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/12/3825" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/12/3825" class="active" data-icon-position="" data-hide-link-title="0">Transcriptome Sequencing of Diverse Peanut (Arachis) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460508" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4cEldaUUy5CPIHeK-iW_-4zyYafFr2h4-RCF1nrggos" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3825.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/12/3825" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3hRkFDww7QyasZ70zYNWGQGy6uf-_3fEUQOA9nzKBaA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/12/3825" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3825.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460508" id="node460508--2" data-pisa="ggg;6/12/3825" data-pisa-master="ggg;g3.115.026898" data-apath="/ggg/6/12/3825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome Sequencing of Diverse Peanut (<em>Arachis</em>) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ratan Chopra</span>, <span class="highwire-citation-author" data-delta="1">Gloria Burow</span>, <span class="highwire-citation-author" data-delta="2">Charles E. Simpson</span>, <span class="highwire-citation-author" data-delta="3">Jennifer Chagoya</span>, <span class="highwire-citation-author" data-delta="4">Joann Mudge</span> and <span class="highwire-citation-author" data-delta="5">Mark D. Burow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3825-3836;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026898 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460508--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ratan Chopra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ratan%2BChopra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chopra%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARatan%2BChopra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gloria Burow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture, Agriculture Research Service, Cropping Systems Research Laboratory, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gloria%2BBurow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burow%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGloria%2BBurow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charles E. Simpson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Texas A&M AgriLife Research, Stephenville, Texas 76401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BE.%2BSimpson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simpson%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BE.%2BSimpson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer Chagoya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Texas A&M AgriLife Research, Lubbock, Texas 79403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BChagoya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chagoya%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BChagoya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joann Mudge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Center for Genome Resources, Santa Fe, New Mexico 87505</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joann%2BMudge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mudge%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoann%2BMudge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark D. Burow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Texas A&M AgriLife Research, Lubbock, Texas 79403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BBurow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burow%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BD.%2BBurow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mburow@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">mburow@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/12/3825" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/12/3825&amp;title=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/12/3825&amp;title=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/12/3825&amp;title=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/12/3825&amp;text=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/12/3825&amp;title=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/12/3825&amp;t=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/12/3825&amp;title=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/12/3825&amp;title=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460508/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460508" id="node460508--3" data-pisa="ggg;6/12/3825" data-pisa-master="ggg;g3.115.026898" data-apath="/ggg/6/12/3825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome Sequencing of Diverse Peanut (<em>Arachis</em>) Wild Species and the Cultivated Species Reveals a Wealth of Untapped Genetic Variability</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ratan Chopra</span>, <span class="highwire-citation-author" data-delta="1">Gloria Burow</span>, <span class="highwire-citation-author" data-delta="2">Charles E. Simpson</span>, <span class="highwire-citation-author" data-delta="3">Jennifer Chagoya</span>, <span class="highwire-citation-author" data-delta="4">Joann Mudge</span> and <span class="highwire-citation-author" data-delta="5">Mark D. Burow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3825-3836;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026898 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460508--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ratan Chopra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ratan%2BChopra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chopra%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARatan%2BChopra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gloria Burow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture, Agriculture Research Service, Cropping Systems Research Laboratory, Lubbock, Texas 79415</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gloria%2BBurow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burow%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGloria%2BBurow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charles E. Simpson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Texas A&M AgriLife Research, Stephenville, Texas 76401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BE.%2BSimpson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simpson%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BE.%2BSimpson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer Chagoya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Texas A&M AgriLife Research, Lubbock, Texas 79403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BChagoya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chagoya%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BChagoya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joann Mudge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Center for Genome Resources, Santa Fe, New Mexico 87505</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joann%2BMudge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mudge%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoann%2BMudge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark D. Burow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Soil Sciences, Texas Tech University, Lubbock, Texas 79409</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Texas A&M AgriLife Research, Lubbock, Texas 79403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BD.%2BBurow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burow%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BD.%2BBurow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mburow@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">mburow@tamu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3825&amp;rft.epage=3836&amp;rft.atitle=Transcriptome%20Sequencing%20of%20Diverse%20Peanut%20%28Arachis%29%20Wild%20Species%20and%20the%20Cultivated%20Species%20Reveals%20a%20Wealth%20of%20Untapped%20Genetic%20Variability&amp;rft.volume=6&amp;rft.issue=12&amp;rft.date=2016-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chopra%2C+Ratan&amp;rft.au=Burow%2C+Gloria&amp;rft.au=Simpson%2C+Charles+E.&amp;rft.au=Chagoya%2C+Jennifer&amp;rft.au=Mudge%2C+Joann&amp;rft.au=Burow%2C+Mark+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460508/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460508/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460508/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460508/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460508/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460508/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460508/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460508/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460508/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460508/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460508/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460508/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/12/3825" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/12/3825.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/12/3825.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/12/3825.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/12/3825.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
