<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/12/3893" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/12/3893.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/12/3893.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/12/3893.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/12/3893.atom" />
<meta name="HW.pisa" content="ggg;6/12/3893" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection" />
<meta name="DC.Identifier" content="10.1534/g3.116.035527" />
<meta name="DC.Date" content="2016-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 by the Genetics Society of America. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multi-parent populations (MPPs) capture and maintain the genetic diversity from multiple inbred founder strains to provide a resource for high-resolution genetic mapping through the accumulation of recombination events over many generations. Breeding designs that maintain a large effective population size with randomized assignment of breeders at each generation can minimize the impact of selection, inbreeding, and genetic drift on allele frequencies. Small deviations from expected allele frequencies will have little effect on the power and precision of genetic analysis, but a major distortion could result in reduced power and loss of important functional alleles. We detected strong transmission ratio distortion in the Diversity Outbred (DO) mouse population on chromosome 2, caused by meiotic drive favoring transmission of the WSB/EiJ allele at the R2d2 locus. The distorted region harbors thousands of polymorphisms derived from the seven non-WSB founder strains and many of these would be lost if the sweep was allowed to continue. To ensure the utility of the DO population to study genetic variation on chromosome 2, we performed an artificial selection against WSB/EiJ alleles at the R2d2 locus. Here, we report that we have purged the WSB/EiJ allele from the drive locus while preserving WSB/EiJ alleles in the flanking regions. We observed minimal disruption to allele frequencies across the rest of the autosomal genome. However, there was a shift in haplotype frequencies of the mitochondrial genome and an increase in the rate of an unusual sex chromosome aneuploidy. The DO population has been restored to genome-wide utility for genetic analysis, but our experience underscores that vigilant monitoring of similar genetic resource populations is needed to ensure their long-term utility." />
<meta name="DC.Contributor" content="Elissa J. Chesler" />
<meta name="DC.Contributor" content="Daniel M. Gatti" />
<meta name="DC.Contributor" content="Andrew P. Morgan" />
<meta name="DC.Contributor" content="Marge Strobel" />
<meta name="DC.Contributor" content="Laura Trepanier" />
<meta name="DC.Contributor" content="Denesa Oberbeck" />
<meta name="DC.Contributor" content="Shannon McWeeney" />
<meta name="DC.Contributor" content="Robert Hitzemann" />
<meta name="DC.Contributor" content="Martin Ferris" />
<meta name="DC.Contributor" content="Rachel McMullan" />
<meta name="DC.Contributor" content="Amelia Clayshultle" />
<meta name="DC.Contributor" content="Timothy A. Bell" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="article:published_time" content="2016-12-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multi-parent populations (MPPs) capture and maintain the genetic diversity from multiple inbred founder strains to provide a resource for high-resolution genetic mapping through the accumulation of recombination events over many generations. Breeding designs that maintain a large effective population size with randomized assignment of breeders at each generation can minimize the impact of selection, inbreeding, and genetic drift on allele frequencies. Small deviations from expected allele frequencies will have little effect on the power and precision of genetic analysis, but a major distortion could result in reduced power and loss of important functional alleles. We detected strong transmission ratio distortion in the Diversity Outbred (DO) mouse population on chromosome 2, caused by meiotic drive favoring transmission of the WSB/EiJ allele at the &lt;i&gt;R2d2&lt;/i&gt; locus. The distorted region harbors thousands of polymorphisms derived from the seven non-WSB founder strains and many of these would be lost if the sweep was allowed to continue. To ensure the utility of the DO population to study genetic variation on chromosome 2, we performed an artificial selection against WSB/EiJ alleles at the &lt;i&gt;R2d2&lt;/i&gt; locus. Here, we report that we have purged the WSB/EiJ allele from the drive locus while preserving WSB/EiJ alleles in the flanking regions. We observed minimal disruption to allele frequencies across the rest of the autosomal genome. However, there was a shift in haplotype frequencies of the mitochondrial genome and an increase in the rate of an unusual sex chromosome aneuploidy. The DO population has been restored to genome-wide utility for genetic analysis, but our experience underscores that vigilant monitoring of similar genetic resource populations is needed to ensure their long-term utility.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/12/01" />
<meta name="citation_mjid" content="ggg;6/12/3893" />
<meta name="citation_id" content="6/12/3893" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/12/3893" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/12/3893.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/12/3893.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/12/3893.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035527" />
<meta name="citation_pmid" content="27694113" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="3893" />
<meta name="citation_lastpage" content="3902" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elissa J. Chesler" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Daniel M. Gatti" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Andrew P. Morgan" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author" content="Marge Strobel" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Laura Trepanier" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Denesa Oberbeck" />
<meta name="citation_author_institution" content="Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health &amp; Science University, Oregon 97201" />
<meta name="citation_author" content="Shannon McWeeney" />
<meta name="citation_author_institution" content="Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health &amp; Science University, Oregon 97201" />
<meta name="citation_author" content="Robert Hitzemann" />
<meta name="citation_author_institution" content="Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health &amp; Science University, Oregon 97201" />
<meta name="citation_author" content="Martin Ferris" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author" content="Rachel McMullan" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author" content="Amelia Clayshultle" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author" content="Timothy A. Bell" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author_email" content="gary.churchill@jax.org" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging Collaborative Cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=E. J. Chesler;citation_author=D. R. Miller;citation_author=L. R. Branstetter;citation_author=L. D. Galloway;citation_author=B. L. Jackson;citation_title=The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.;citation_pages=382-389;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18716833;citation_doi=10.1007/s00335-008-9135-8" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. M. Chick;citation_author=S. C. Munger;citation_author=P. Simecek;citation_author=E. L. Huttlin;citation_author=K. Choi;citation_title=Defining the consequences of genetic variation on a proteome-wide scale.;citation_pages=500-505;citation_volume=534;citation_year=2016;citation_pmid=27309819;citation_doi=10.1038/nature18270" />
<meta name="citation_reference" content="citation_journal_title=Food Chem. Toxicol.;citation_author=R. J. Church;citation_author=D. M. Gatti;citation_author=T. J. Urban;citation_author=N. Long;citation_author=X. Yang;citation_title=Sensitivity to hepatotoxicity due to epigallocatechin gallate is affected by genetic background in diversity outbred mice.;citation_pages=19-26;citation_volume=76;citation_year=2015;citation_pmid=25446466;citation_doi=10.1016/j.fct.2014.11.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=. Collaborative Cross Consortium;citation_title=The genome architecture of the Collaborative Cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Dell’Acqua;citation_author=D. M. Gatti;citation_author=G. Pea;citation_author=F. Cattonaro;citation_author=F. Coppens;citation_title=Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays.;citation_pages=167;citation_volume=16;citation_year=2015;citation_pmid=26357913;citation_doi=10.1186/s13059-015-0716-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. P. Didion;citation_author=A. P. Morgan;citation_author=A. M. Clayshulte;citation_author=R. C. McMullan;citation_author=L. Yadgary;citation_title=A multi-megabase copy number gain causes maternal transmission ratio distortion on mouse chromosome 2.;citation_pages=e1004850;citation_volume=11;citation_year=2015;citation_pmid=25679959;citation_doi=10.1371/journal.pgen.1004850" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. P. Didion;citation_author=A. P. Morgan;citation_author=L. Yadgary;citation_author=T. A. Bell;citation_author=R. C. McMullan;citation_title=R2d2 drives selfish sweeps in the house mouse.;citation_volume=33;citation_year=2016;citation_pmid=26882987;citation_doi=10.1093/molbev/msw036" />
<meta name="citation_reference" content="citation_journal_title=Environ. Health Perspect.;citation_author=J. E. French;citation_author=D. M. Gatti;citation_author=D. L. Morgan;citation_author=G. E. Kissling;citation_author=K. R. Shockley;citation_title=Diversity outbred mice identify population-based exposure thresholds and genetic factors that influence benzene-induced genotoxicity.;citation_pages=237-245;citation_volume=123;citation_year=2015;citation_pmid=25376053;citation_doi=10.1289/ehp.1408202" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L. Y. Wu;citation_author=W. Valdar;citation_title=Quantitative trait locus mapping methods for diversity outbred mice.;citation_volume=4;citation_year=2014;citation_pmid=25237114;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Gnan;citation_author=A. Priest;citation_author=P. X. Kover;citation_title=The genetic basis of natural variation in seed size and seed number and their trade-off using Arabidopsis thaliana MAGIC lines.;citation_volume=198;citation_year=2014;citation_pmid=25313128;citation_doi=10.1534/genetics.114.170746" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Gu;citation_author=D. M. Gatti;citation_author=A. Srivastava;citation_author=E. M. Snyder;citation_author=N. Raghupathy;citation_title=Genetic architectures of quantitative variation in RNA editing pathways.;citation_volume=202;citation_year=2016;citation_pmid=26614740;citation_doi=10.1534/genetics.115.179481" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Ramia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=O. D. Iancu;citation_author=P. Darakjian;citation_author=N. A. Walter;citation_author=B. Malmanger;citation_author=D. Oberbeck;citation_title=Genetic diversity and striatal gene networks: focus on the heterogeneous stock-collaborative cross (HS-CC) mouse.;citation_pages=585-585;citation_volume=11;citation_year=2010;citation_pmid=20959017;citation_doi=10.1186/1471-2164-11-585" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Trends Endocrinol. Metab.;citation_author=S. A. Kelly;citation_author=F. P. Villena;citation_author=D. Pomp;citation_title=The ‘omics’ of voluntary exercise: systems approaches to a complex phenotype.;citation_pages=673-675;citation_volume=26;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Y. Liu;citation_author=A. P. Morgan;citation_author=E. J. Chesler;citation_author=W. Wang;citation_author=G. A. Churchill;citation_title=High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.;citation_volume=197;citation_year=2014;citation_pmid=24578350;citation_doi=10.1534/genetics.114.161653" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. P. Morgan;citation_title=Argyle: an R package for analysis of illumina genotyping arrays.;citation_volume=6;citation_year=2016;citation_pmid=26684930;citation_doi=10.1534/g3.115.023739" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. P. Morgan;citation_author=C. P. Fu;citation_author=C. Y. Kao;citation_author=C. E. Welsh;citation_author=J. P. Didion;citation_title=The mouse universal genotyping array: from substrains to subspecies.;citation_volume=6;citation_year=2016;citation_pmid=26684931;citation_doi=10.1534/g3.115.022087" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. Morgan;citation_author=J. M. Holt;citation_author=R. C. McMullan;citation_author=T. A. Bell;citation_author=A. M. Clayshulte;citation_title=The evolutionary fates of a large segmental duplication in mouse.;citation_volume=204;citation_year=2016;citation_pmid=27371833;citation_doi=10.1534/genetics.116.191007" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=J. M. Recla;citation_author=R. F. Robledo;citation_author=D. M. Gatti;citation_author=C. J. Bult;citation_author=G. A. Churchill;citation_title=Precise genetic mapping and integrative bioinformatics in diversity outbred mice reveals hydin as a novel pain gene.;citation_pages=211-222;citation_volume=25;citation_year=2014;citation_pmid=24700285;citation_doi=10.1007/s00335-014-9508-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Breeding designs for recombinant inbred advanced intercross lines.;citation_volume=179;citation_year=2008;citation_pmid=18505881;citation_doi=10.1534/genetics.107.083873" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=T. L. Smallwood;citation_author=D. M. Gatti;citation_author=P. Quizon;citation_author=G. M. Weinstock;citation_author=K. C. Jung;citation_title=High-resolution genetic mapping in the diversity outbred mouse population identifies Apobec1 as a candidate gene for atherosclerosis.;citation_volume=4;citation_year=2014;citation_pmid=25344410;citation_doi=10.1534/g3.114.014704" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the mouse diversity outbred population.;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. W. Tsaih;citation_author=K. Holl;citation_author=S. Jia;citation_author=M. Kaldunski;citation_author=M. Tschannen;citation_title=Identification of a Novel Gene for Diabetic Traits in Rats, Mice, and Humans;citation_pages=17-29;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236446;citation_doi=10.1534/genetics.114.162982" />
<meta name="citation_reference" content="citation_title=Modern Applied Statistics with S;citation_author=W. N. Venables;citation_author=B. D. Ripley;citation_year=2002" />
<meta name="citation_reference" content="citation_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. E. Welsh;citation_author=D. R. Miller;citation_author=K. F. Manly;citation_author=J. Wang;citation_author=L. McMillan;citation_title=Status and access to the Collaborative Cross population.;citation_pages=706-712;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22847377;citation_doi=10.1007/s00335-012-9410-6" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=M. A. White;citation_author=A. Ikeda;citation_author=B. A. Payseur;citation_title=A pronounced evolutionary shift of the pseudoautosomal region boundary in house mice.;citation_pages=454-466;citation_volume=23;citation_year=2012;citation_issue=7-8;citation_pmid=22763584;citation_doi=10.1007/s00335-012-9403-5" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
<meta name="twitter:description" content="Multi-parent populations (MPPs) capture and maintain the genetic diversity from multiple inbred founder strains to provide a resource for high-resolution genetic mapping through the accumulation of recombination events over many generations. Breeding designs that maintain a large effective population size with randomized assignment of breeders at each generation can minimize the impact of selection, inbreeding, and genetic drift on allele frequencies. Small deviations from expected allele frequencies will have little effect on the power and precision of genetic analysis, but a major distortion could result in reduced power and loss of important functional alleles. We detected strong transmission ratio distortion in the Diversity Outbred (DO) mouse population on chromosome 2, caused by meiotic drive favoring transmission of the WSB/EiJ allele at the R2d2 locus. The distorted region harbors thousands of polymorphisms derived from the seven non-WSB founder strains and many of these would be lost if the sweep was allowed to continue. To ensure the utility of the DO population to study genetic variation on chromosome 2, we performed an artificial selection against WSB/EiJ alleles at the R2d2 locus. Here, we report that we have purged the WSB/EiJ allele from the drive locus while preserving WSB/EiJ alleles in the flanking regions. We observed minimal disruption to allele frequencies across the rest of the autosomal genome. However, there was a shift in haplotype frequencies of the mitochondrial genome and an increase in the rate of an unusual sex chromosome aneuploidy. The DO population has been restored to genome-wide utility for genetic analysis, but our experience underscores that vigilant monitoring of similar genetic resource populations is needed to ensure their long-term utility." />
<meta name="og-title" property="og:title" content="Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/12/3893" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multi-parent populations (MPPs) capture and maintain the genetic diversity from multiple inbred founder strains to provide a resource for high-resolution genetic mapping through the accumulation of recombination events over many generations. Breeding designs that maintain a large effective population size with randomized assignment of breeders at each generation can minimize the impact of selection, inbreeding, and genetic drift on allele frequencies. Small deviations from expected allele frequencies will have little effect on the power and precision of genetic analysis, but a major distortion could result in reduced power and loss of important functional alleles. We detected strong transmission ratio distortion in the Diversity Outbred (DO) mouse population on chromosome 2, caused by meiotic drive favoring transmission of the WSB/EiJ allele at the R2d2 locus. The distorted region harbors thousands of polymorphisms derived from the seven non-WSB founder strains and many of these would be lost if the sweep was allowed to continue. To ensure the utility of the DO population to study genetic variation on chromosome 2, we performed an artificial selection against WSB/EiJ alleles at the R2d2 locus. Here, we report that we have purged the WSB/EiJ allele from the drive locus while preserving WSB/EiJ alleles in the flanking regions. We observed minimal disruption to allele frequencies across the rest of the autosomal genome. However, there was a shift in haplotype frequencies of the mitochondrial genome and an increase in the rate of an unusual sex chromosome aneuploidy. The DO population has been restored to genome-wide utility for genetic analysis, but our experience underscores that vigilant monitoring of similar genetic resource populations is needed to ensure their long-term utility." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/12.cover-source.jpg" />
    <title>Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"smNVu2F4ukfaYbWY4IyZdIyoIbxkygMMyF0N19C31b4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460500","apath":"\/ggg\/6\/12\/3893.atom","pisa":"ggg;6\/12\/3893","ac":{"\/ggg\/6\/12\/3893.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/12\/3893.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3893"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/12\/3893"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/12\/3893.supplemental":true,"\/content\/6\/12\/3893":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226lKfgBVysGFU8yq7+vWXGLqiQ1HKgCYamN\\\\\\\/CKHmf82+NGhVMRMoaPs1vwXNF6P6q9uEVnMiSRjD4IdaI\\\\\\\/OF5mgIWdTJATIvnszwgfrRiwlp3q0VGF6OIIinweuYIzTBX2S3b3czJijlXUyV8ka9OndUo2E6Pxt9nQGD7\\\\\\\/qyNsv3Wj42ivHb7\\\\\\\/3X\\\\\\\/u+kzonnXQE6yGdJ\\\\\\\/tGBe0tG84km0UfNsatXgFrSqNZ2B89\\\\\\\/fdKloSgh2YyiYcYJ+k9Tm\\\\\\\/MwZ44+dDmpn6tFNNr8B2MDo9yHymi4lpRy5RKIlLFjebYLA4PIlj2vfLctuZLMBxxnNBFRNaNYXkPqWJaKqZP92XV2yDrVB7\\\\\\\/NlDEVSrRmrX4vtrGKxtIBSgkdThKvSLyGGswLsdicf2VstvMz0RI\\\\\\\/IJuBsEWbxfWh4W2LhsaJr\\\\\\\/IxsElVZHXOCT0xCWOGAJheMoxkKLckleJhWCH9M1xraJhEipyoepxxhZK8cdwNKYubGloNySlo3RLdS58czWKFAFZEhKcms3ahRXrxHq01\\\\\\\/PE4\\\\\\\/gjoWZaGIzKMeHkIuYmZDcR7c\\\\\\\/LsvNbju2PYN8LVok1GliWDewm8oPBq3H1D2S8X1kNp1Mb9+l5IrzxlxYdZeCK7QNMM5tPHcqOaiH19YABKBikzT3N3hhGLDBBSLxo395vsQEhjLBZqJXfpG0oGCSioDpgV27VVTdRHcGOyJmMPWmvurbWuOs2nc\\\\\\\/NGsbsKF\\\\\\\/7CRuM9eJ5PAGXFzyZjZAl\\\\\\\/MNLV4611egkU8sLVX8wLUdxpH3Nt2nflVr5mLbAshmau5PPv73+YRdIScjdPusxpUmtWlmK3IBcNAspyHC6248xlWHZslRc\\\\\\\/AL2xYfVsXBmqS25t\\\\\\\/cmQbEcsBfxXWUdiYlMCiGeNUcRoIHeloMA+q0ktxIbw30OCM4HQ7kQvPh2zszybxUJ4RIF6X6\\\\\\\/1nuQ6AXaoCwDpPvPoS1LCiUAR7f2pC29+1WTiU0pdvx+wRw5Bx+8dj98aA3P1BI05MzkN4Rfs3gvFjQHwy0EZqxJEcxz+zFfjadnLd4LY0AG6iS8eEApjLcdDzkEXSTexepQbtS+t6iQ316v\\\\\\\/C3jdYd6sqVtCZKynku\\\\\\\/9yUQGnlTWOgM5dl6eBAF\\\\\\\/kvf6RWRg8QexzBG5EJoF8NPbKSWohHhMT8dFgcxvgMw2lHgEmLQrYTGeVEzI3Ub4bzPUcunYGKPMbx77gCFKSdqppN7yZTFErUB+t9ObEikaTmZd\\\\\\\/XAf2G9GNnqqEQis8fbBx\\\\\\\/aKqkvTLxND3CZ3kPP51eGg1\\\u0022,\\\u0022iv\\\u0022:\\\u0022A50rpDJ2iAg71mi1aaSJIQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b3968f6d478f2c34ca7cb0fafe70ddfb729a1e4af1b95d8af52510e3f03bc3d9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vWVZMPVL\\\\\\\/hrMcI4v3qBGXe5kfFPwbRn7x5jJkn7naLv05bR2PkrU\\\\\\\/u\\\\\\\/NDQwymEEX79dXNacxWRBpI++gqZmJY516Bj7i1RPdV4w\\\\\\\/ROrgIqTzYV14yMlH\\\\\\\/ErDywTFB4KP82NSf\\\\\\\/CIPr4HrXR9GZlukkKO+jE6Xh5ky0Y2xCB1fmd22wZu505AiOaRSc5BbI8jfdcS14hCnO+DyJXUfSmiEf0VjwTaOcRnuezvF62HvCYOvqws171NpITTpHJZxvP0mvZbG0RGQ6dh3yQGuVNB1HOcXCjt6DS3BNCYQpgQsnnFeGceGap7D\\\\\\\/+k\\\\\\\/55l9rM910ejKONBwNVs0daLbZV\\\\\\\/Y0dobpHwvErZTgoN7sPgUPGee39sZOiOOspnzONL6arMeM88CHZrdgnvjNKcBfeFSUTgt1DMoIqtnRD0gE9kIluAk9Im5ojGWAM1uRK4EcPiuqBoeof5kwXvui0QrT6i3Op5K6NCk51Qpi7vZnzQDt0RgrXWhMQrsuSnnbCKkm9HQesx6Bb06wEJo7aGtMQV+eTnxHf2oYths7tgGPyNt000TpAU86E1SS4WY3cI6CkVpRUOHU9Z1w3vEdlA5YtCZ8GCKoBhY2kgaU0FJNE5AUEfIRzwdaSpOowCFq5jaQfZ+4MxGlXHqNz8bXrmlHuM0pMCwOiMZD48OKVzgLEbt3Pcd6Vy\\\\\\\/S7RQq2ke0qdCzP\\\\\\\/uvhtWR2iyrgVKYUAeEoyy5swFJk0wzbxLYTL6MTHKZZ74bUq4JyMwyO\\\\\\\/+oyWRTatl1JIEuxUfj996IuGXve9uYGFdY0QgbjN41MOo4jvTOUSous45aopF2\\\\\\\/mYAP\\\\\\\/ZpjdvbSeGi3rMs3Pvtlu2QkeTcwDGLvGZqSYuach\\\\\\\/mxgDhwz33aiFSDId0yypcyGKMUVwBQ25I\\\\\\\/3KVT\\\\\\\/L046\\\\\\\/QhrAauce8MY7c7EaH6KmLBSoCWr25YiQdF30rXacfiDHitd6RBIf\\\\\\\/W+pGMAV04X7aOYOTvKbhkN8I45YMFt40JEmv9K9TUTWmmc+EIjSWZJrnHG7ub+SUG80c\\\\\\\/8E2HWL+gFWkBWIoqwUfZUzM0V6a\\\\\\\/xFpNgfUIT5RCuJtLFq6nPkEdXmPGUTkIZ1McK5hlSg\\\\\\\/rvRPYbh9oJ1Ghje86mA9zLDb6iTfIy2rdjouCFsSdGRjfdwmtj9EoKPU4YJZvawEtXZ+IIPlviyKAqYsCrozPIDOvc814JeQ3ljv0NaM1P2kFu9DZ\\\\\\\/wXeDE23WkLEVzqppdKo5BsURelJm8+kJr0fwNQt++iZJqd2WN0H2Te8WReMtSdEzjvfFsQGHxNN2Keyckt9D+JGhdCqFxIkmRyqz6k8B0fY22P5l92OyT7RXlUQEo4fHXjJ3nZGY8P0Hois0yxlTkPLliDEpMLwxYfZCYacJi\\\\\\\/dkOFxfsi0oPIEkh0K0xPshp3dSMCxeLL8vs3wS66i8qJRRG3L3N30yRE2wPMdrf6ayDkXYMfgHRmPRQlxg6rCFotqi8CpTaa96wW\\\\\\\/R0v+cAYY+aObENN6NbeHa6mhw\\\\\\\/3x\\\\\\\/ZHNqSPlyRrTo2zKtNUWYTQ2cL7aec6qTIL2\\\\\\\/IA5UVZJB8H7f6iNRUSEcqkgitqDILDNWlcTLQTcZWIOEUG7F\\\\\\\/361f9jZpqJgOocuvySQDIdUSW+jusMWS9dB0lhYGfwmvLqMWSsBmtqe4Uxm+HjzevzgSVuUfEQDkX9etiOKA0G8B9rWoLGmXyei1j3VCPQJUFHDCVtj9\\\\\\\/G4QlVkGn6zLKaCjGFXQ0h6jE7OFkme4oP1jTM7rbwsRUWHS\\\\\\\/VKrHE\\\\\\\/f7eJ9qQ\\\\\\\/WPBqMaj9khQ8LbOuQ6cSTiokvEcam5TjJVtG7sedObwTQn5yFT1lFM0p4sGmMieYZ4NbQ\\\\\\\/Nz+4KVKwaztngpAS+wDSbDREmN7aZVDiWJx3HE6BVMw9B+UzqkUxawFPBctW6wLM3S9wFTsx5Qw1nmvYBZdXh7Yvd\\\\\\\/E+OHDKVw+iuWN1TrMqRjHonXy6S6mI2z4408VC8hSthbrIcjomWlD28hLYcXu9QxEPci9wEMfKR8oZsjGZwpL7p51XjJMgdh70xeWFipvMP1877TeV924pgSrgsvtcbv7SFr\\\\\\\/UqaeCTjiGpEvL8oehRWHCic6I4YnSQBSqtX6VgFDGAZPpIU098MJE6BafYC34CN6qb+3e178FrJ0K9u+K3Cti5PvhGD9SafKIPrECgQIr1B46a9B1OSVh3R3C6zXHL++UIyMvTiaFyYBmH1k+pX9I25zoQ500\\\\\\\/WEE6+8j58owfuA3WHA93U19nx\\\\\\\/xxdvNxKlMV89SYsaJIF9ywG1\\\\\\\/mTo14dNP7\\\\\\\/56C1UIT+7B5nR5tZ\\\\\\\/6pX\\\\\\\/WvXtHXX\\\\\\\/q2doWOFcdk2aQfXIoQ4NEDW+uAf\\\\\\\/CA3bAXJDX8L9vF0tcoZh6\\\\\\\/Ll7V7MDAGlZOePREYW7S\\\\\\\/SlILeVu7fEFLz2MYT1ouPNZa63UjPMqQAXAjkR\\\\\\\/saV7I827RA8\\\\\\\/vsF8IxCr9WuSB6gtXkOOcwstc0f0DB7jdqodQ376c9SQJ9MtSnxLueB6RvvHjOya6YawHFRG3\\\\\\\/FhvSqzwFDA3e6Bav9WTnYNxmVWfmn\\\\\\\/DJARDiCNtwtnAkW\\\\\\\/9M\\\\\\\/qfrwUV8ESps6Nustqs7AlteA89JqbX34Msg1j5wDAdcOwEZUpFiX3Ep5UrFP\\\\\\\/BCYx72kaO0VQ38rIinHXf6RCN7cXlmeHwPM1jyx5dS8xjuNYI3yqSEjj1JID2k0cMM2yxKKrg6kl07SBijr3dJpZB0CgGKSOfWEodUVDa68+bKk+ZDEfdYS0x\\\\\\\/7cToZCJQdeib\\\\\\\/v8RXXVq3WJGpAOukUWoWXPNADvibzeATKmDl5LdoA5sixWqgdnjkNLwuEvrTHXk\\\\\\\/bT2ImjBro5VyXn7wh7CjNIkMqeF\\\\\\\/hX\\\\\\\/UMg3LmLZfAbRgijSEqM635d57fjee+84MikjoslXeT3K1cIgiri1SU+uabPnsS\\\\\\\/W5+LotPWhXbCgFdw3scqxypKd2v9E8O7lltuSXuCT2vNc8w8adNrrWbytm+FNGq0QIM5HretgE3K2C\\\\\\\/tD7OwjFYt+FrwoSUmc0uoKx\\\\\\\/a3tgTU9vT660ZXxp8yD8q1\\\\\\\/k7n5FoeHwW7td5V2Xv6RtRe\\\\\\\/JpXruyoD5Y7TTTUDnZoZIJlDwx9NCHEXOG1uZUMhVAUOPXcrJZsmYu8esqybblXwfvp\\\\\\\/e0B8aytE+IvkX8Q7V6ZLE2Jq99UV7n3GWhZtR4k1BKPf9I2FS25MaycAx33DvsIxkRmQ371BYFESWVSojZgWxQLsNd6DcLWzx9t89dUx5MwIL5F4VycQ0y7r7ybGAdVhCKicJC6HWdau\\\\\\\/k3UfuNA1POSyzWqJzyL\\\\\\\/NVRfiWF5RO+To9qFY+oNtDXpaLPvHAd\\\\\\\/F7yZ16OboyZqUfVjtUJQ\\\\\\\/ZSP2IhYml68Wns8Zg9MxXaCHUDycaTIGo7a40TlpVDKUW4B2m3XTLnd6WIRp0npK1CcyqYwqgPWipNkm62tvo6nzKO9MwKlgtwE4T3dbnBrkIfmoNrw5nL\\\\\\\/ev8wp5MbCViRdSUSvrsPRfg012xZcAQN\\\\\\\/BUWFGiUHk9D4QnQtH1GOzqqkwUtz0A6a3uGxGDMcwh6VfnNYBpTzGI7UckLaaoZYjDsukfL9IsEht0bEI3WVimUfXufGUWiM2tKjc\\\\\\\/\\\\\\\/jH6JzPomFQB\\\\\\\/sU57o23WFn6IiczxKYhJmJJCK6d939VtfFgo\\\\\\\/GtGFltxtz8VZWmpaOsxrs9h5r7cK5pAKK8E+\\\\\\\/jd5DLrVd\\\\\\\/6Z6YM1cP7EsLt+4J3S00OE\\\\\\\/nDJ5eq7lB34RrO8wJoAU5eivUPTrVE1dzO477TvE623lHHj+KaHdgLg\\\\\\\/XZtno8qDrDe8L4SnXu+9aUTM9a6WsQyTfRCKVYj3FBAohsN57zlnz7QjiPLgYlRm8JFXnyatCNYyVgWtiurebcAsKDZLG4v3d0ce\\\\\\\/1F5JRw9\\\\\\\/uvzFkMFBcwUTO4L4t8GCyFtiI0PPhk1TkX418wfN8G\\\\\\\/qEvIa6YedlrPaaCc5o1I3jAnQmep1KQ+WDpjlfUp71LggheU08TqMYsXV4g0K0F\\\\\\\/\\\\\\\/gjNHr5uoimx8yymkcEC60qf\\\\\\\/opOqdBwvHLrNLTsPmdjEgiscCFpSdSk1eBOwn8rnwoTUAEE2MKO7+713hO3r5EedS7yU4u0N90PU\\\\\\\/WEOwxckNTgO2bsmffWJsSIYz\\\\\\\/plmeZAQbsY4afGsMJAmN6pFTYI3em9I36rbhiKzFIW57oQq\\\\\\\/Vh9HLIUkwFEqEVNBBVYQSnG6GenpXs7iLzgZCgHXCFxbfTL4HAipDbSEgraTXBmWa+ax6EKEi6iMmTiw9Ylu5aewg+0aiP4XvEAR2bsgipZZKJzuJvz9ka6Q1uIKuek4bGFF+w4WsGv5zSkI6xUMzqr6tphEF+S3xPrPhbkpIA0qt5ROJFvQrAbfQbqMgeT\\\\\\\/oCn0nxpwN6p4cegMAPuYPCwq8BrRrehjCeQdfVNy5oUI31\\\\\\\/+kAKCisIeg9whLZ5QSVfslVwdP9jcD7WzFb\\\\\\\/ShU3lz5yVHR17sdHSe+1zMjvcg+r4Mx+Khjq0f4Be2xie2jdF1upHxZnimTpB2tlO4BzZzA\\\\\\\/rLWYXanCkgms40SJjxSngOPsB9X2M51B4fn0Q6NIAoPp9AaNbXGDSzrYdT+sQ5UjCQ89JmCBLALgtBBcihpXo65zafFjfsq5S\\\\\\\/ZDHvsFPy4+VkdrzlmcVyw\\\\\\\/9GcqhzO2MDxqKjKmXue3\\\\\\\/YRQUcOpJDk4UNG3JWBAsQj6z4qgzrq3EHl3EBekCig6JxELVwWfnMxELYjHvdt0MAbPGyJTzWhWlZFSr4V3vkA6URb433t0FteDVeu7eKsw9l70xOn4ziff+1c1Dt8BQLjNZ8rJvuTQVegyDp\\\\\\\/ADKC3Cya1Gc6NeXfMJOZ5tvZ2kbca51BZJL027TY8rtEW1bu0Nnk\\\\\\\/eNcvljr90qx5yc2Soy7uODFR5SeKiUMuI7SnAAycWslayUMGKBLKswFdwQ4THiudyvPBf9C4CdJ4UP6+pY38Sl0Z\\\\\\\/3WCatkzR1FC7PJ5Nf29isIYY9O94SCF9XbswboTGWN7C8\\\\\\\/E8JCMTnlMUX6OgOYCXsN3Rd5qW8i6oysP8pUTm+x8nE2sugk\\\\\\\/xOBWaUwF0089OQ67hgkmTj39aWCzOvO+2x6GOplTB61j0ip8Ize2C5SbvhtMI1WJ4Tn\\\\\\\/dUtT7zRplDgB8GVD6nJ5ZJkOYclB5ZbGIlhK85XkRJYYKwCS6\\\\\\\/Hs7DEC2\\\\\\\/XO64QLzORLAhJCHu1xfiVE5ayV6LduIRjr5UkhvKDAI3K69VaU7w3Zo0qByjm1IXdHV4cuJeg23cJmQsVHzVQITuYGogBHkdx8WyeJL5V4GNFSNrXHYtJk9j3mk7hAp4GJB1PiCIZACfjZoWur9VFWZZTCyRu6ldQ3bWtMnDOU02ugvgh2bdvvTX\\\\\\\/fvyIuBzRiONeDNB972heDZQFT\\\\\\\/wso+za\\\\\\\/81u06Q9r6amn2hUMKx3jpLH3LVV\\\\\\\/ee+NwcjeddevVEOwBP9OWR6k5\\\\\\\/OPPKJ+xeqpbd9tiqsiLftep2s3U++cmVl1GcSVMrbdTJsP+rlwlo9MFnVo0L4OVho99CSxi82vV\\\\\\\/x4u25YvBxhzj83IpBaRljLz6\\\\\\\/nLCSg4evt4EnQjzgfAIVYKF0odEeXiy42xvmvpQYKuxUbtpx6HJNK+rIfGPGfQtxDJOPhxS4GbmXJrbzwxONlUnjvG0tRs7jqaQ+9P6rfxUp9Pfiogm38aDQSVDWth0iUaWz2eEHfUEmMqKCD5U5BKaf9GiCgtmKjsrltTbUxKc3gDMpQWXHuEajundbpLaNEh8o3HqDUH2QZakJSs4zacXPpng78L3DxpEklOy4KYlJICEdn2t5XkrSiAGkeLVrLdsGZrKWq4M9JZxFyYqnuInvELnAzbI9I4Cy5cxLch3PCB8UZrYbRP9ZBvO4rsjjzKC+NLRNxT+s\\\\\\\/UGbnI0VOc0OKOJZLb5tki5\\\\\\\/1NpZ4Znc5H6H5TgOJDXYFX337UHQ8RhGZJJhGZnP5uwaEtp5q3fYYbR811lFjCEBZpcAIGVyh5wuDZMpKS00yqftS2fQUueI84cIwO6qnW1l9U92aLlPcDz5pooS35+JXtp0kn9a\\\\\\\/c2sdgUVGZR0Eb6VFkIs9rrJkYQDvbZjJIRMltH6fApEcGgkmAGt7DeLKfuDfoAm0S6pyR5qr4QwAHPKI4qOvNYKPGu56Xut\\\\\\\/v7SJCmFio1o5pLxinbZRfZSHE+snyFNKsaoSPYDBPnMahvJz+PP+38nXM+hQtAN6zmLXF5cSdwUOU3u4VlWpHGjKEU7zcbzTrekHCCuBTGgJiQA1c\\\\\\\/XQ\\\\\\\/9WyxenO1MzsTkBltmS4GiHCyV9Vg3ZqTFF7bmPmSzR28j9ADz1Ngd2IPUHiw07DoAvoTdIYPjX58NKVWgxwUL+muru8nN98bz7h39LCQst1ZQ\\\\\\\/LyJZmSwOoLwJbYDKi1b1kz09+ZhCYVEztCd1267ZyRIMaZTOzknhTPmVaJ61dWK0rj4Az7Y8\\\\\\\/en7KHg1icLF7qGvff02i3asIar8MmkEw84Uwe6326mbYb5wcqtdeF4XYzwuGVmLcwjqyLxyhCcl294EKMjZP\\\\\\\/YvyDBoN6bb07FJXWCAIDy1SyyC+WW6LhEFs2vGpExPOMH7psjnJqZ3lRjzsh3JmbmY8ZJeatsZe7cGcrfzXz1OdZP8w4cw96vZtQTm8Sgx\\\\\\\/gJq06aISDRQgGS8m1tJ5xk2OGrFlJ\\\\\\\/LbzFvp2f4Vuc1IM2beS\\\\\\\/xADFqRIWalyA9aWx7yh\\\\\\\/N9FbTQAY8lPjUwpIXv0EquiYQy2YG2wP2AtYb2xFzRwoMXtQFXyPlhjQaDOKfLMLxO\\\\\\\/QTaXIqW3PWDALs2yqHlxIJLjpezcnWLsbo7pSI88p22OHDuBY1ClpBz\\\\\\\/RENz9rEwQH0w6N71XKRgSUfKszMnwwdDoT3pmbUzcWWZtYN2KsrPMm0Jlvn+e+wuTQ7J6mIl9kgmlyRE+i+hkcvB8LgCJe1yCkTCcRJnt\\\\\\\/bUA4cu8Q0B6W1qoispZDOdItsAzrZQ5tOTWMOuiU3fsiT5\\\\\\\/Xx6WA0nLYk+pIUxVkbjdj9aqesVYd5bdGxy3jRquBb0Tf+eSzguAjFzR8KFMjxOXnwzv9MHHElcPBM8An3ECHp1tLpu30xjRyq23m8Dp1zalWmK+ypo\\\\\\\/9vmV59RirNQWjwbENHMo\\\\\\\/0SHsXT5vI9zDjZtG4WJ4ibZF\\\\\\\/DmGaQkyyNq6Ravkr\\\\\\\/TnkhxQVNwpghPB8mqh7j0PMAXUj0Nsed7OtniTB9fr0cAl2eP2EerK9WsfELcCuhTkS4N\\\\\\\/Bn7HxceVjFJmNXe07J7FAuqX1GW6KLWx9VHKx3XUM+nvFEsyJdSETPK13B5YuFb69DVKaNzVCPvDylqMBLvbkVhbcFd6Ze6W1FxikIL5325AE6vdvu1tK\\\\\\\/LNsbMBjU4dXt+e8gOy8XLSKhe\\\\\\\/E6DGPmh83pxElD1xaBJrDDdW02Y1ch+Fl2Iypah7Kzp2DuogqS1Tw6zTSjqsVjaYZt7k+0vNlTnNONi9X4P2S3QoE3Ya2kFsfLhrT8W+oYRue7ycY1O3vgRfUoX8tsWm6EKhCzneRn3PZ3PUzIDrDM3fFtH2NSAkWoo0V6rPDULArvjtWsBF4ccIp\\\\\\\/3IWEeO7+XHSOxZn6juUXz2InEGYJJOUnZuhuZ3b0bIkQhIujiRWHI5mvSkspJK2VzFOZ28uGZJaMwo9in\\\\\\\/K22ZBhNYi5mH5JHOirIPhMYRG5hTxsyqw\\\\\\\/DdyvXl\\\\\\\/5PJJDSsVKKOYPxzicsZ3a2wNQxGBT1WZVntIcQa0vh2EMYp2V+S+wavBh2n083QEH+qSjPt2Rw\\\\\\\/XM99xZEA\\\\\\\/yfCxQobP3+FXYStP6tkrbWlVTtlKuxycNukuEbIjOkIC7GRSmSWK0lUMxvOy3le2iyWfQ\\\\\\\/oUrSO+JqGr1+QE2ZV0bGDjG9lvP7lx3h4JqGEFq3CpddJgBY0PRfuuH1S7ITSQ\\\\\\\/VbYKy4glIG9wuHhNTsh+Mddmc2Vg5Q6c4o+QB03TOwiZLwcshJsANa8Q9KjF7kqu5zz1\\\\\\\/rzj8cHbPV570xXMZBMPM980w9vh74tzYmAv1+cDvwLbaoo6RtUAQ3INp\\\\\\\/gfyVWsd\\\\\\\/CjcbjOCwReZlLbIRYeYU2Je3o\\\\\\\/X1yvH6GGvkLE\\\\\\\/bIttsWw35z5iD3cM39P6JWtXa1fEdWC5UkvRhQpptw+7mC5LUiodyNV1hwS4jteyhnYAQiAR2D2oIKVU3X0E52kHxsFfgSyLayWy3gxnAiVGUcYs5c0MJnGpfNoaacnBncRQzG5NpDzYa2U4zXgLZYCKtQySD3CgivEYzUclKRxgtEDLQteoBYBKVqvM9sM3DkIvocc3veumge2HZqyQ8XB6i8E3WtBC0eipo4KqBJu4CHWZ8XtufSGHqUldj1kq0DQZMHdMBAEgMBoCVOONJeNsVLw21sT6ADrpm7Uq27KV9J034RT18mYnqrm2PuQOKFSkM2PhqNHHPHcep\\\\\\\/FHUEHdN+Eu+g8H4r0KxTT74I\\\\\\\/RrX4JagxmcSD5IkD4JwFsPZAPL8uCxeNWwD2uL7x8YU+83FzjLXvAQD1IebXbNR7YJz8DePxbYK43itjkeSbHX9hzzqf6VLWEVI76zpFzD3b21D4UC6v+vDpPaVzZVkcgUKjsTB3eRWJaUsUjJfIxN+V7ZTIlQjyZopMLnoWy7ASvZGyHVBYlg\\\\\\\/92cOnU9W8xsO6jMTYmYi0FB\\\\\\\/VZsnO85TFCP\\\\\\\/baKWLbNSIL1lZHvOGbL62WP5fZqvpFHRa9KTFG4mqoHp2kbpd5EHRhtzPnvbq5IyRhOOZx+mjlSLJFiM4F+pBBLnz93jO1dl7uNuX1KrVys94Tmp08ucv5cJysB9KGMtxLtwbDGc9D7mx767GWQRC9bDvJt4QIgBhsm8mV2F4mJBr9tpr++HaAbm3ISgCRWHPWAr4+\\\\\\\/gL5pjYtkVLUCnoBO+Px1++aCA2KLJcCuqIzy\\\\\\\/7p82wUYu7zvDVnXD8Xcfl\\\\\\\/BOI4IKk0\\\\\\\/X0lhhfMcMl0PFJesWuvoaCEHMxBQ8DbakJnibh2JnCnn5v\\\\\\\/LdBLZYwyKbOhl74owHpW3FoqYHusxXqqj1Se1hJIt2NrWlO2o8XIg+mgB+x0c4C7Mo7a98sQbo3eKXfGKMhYjqI5K8+0Y2a\\\\\\\/2Itdq4DlIsafqTEQgIlR\\\\\\\/5WFPc2oTiStEoE\\\\\\\/q\\\\\\\/lida9\\\\\\\/7a1CfP8dgM1IEXsGSd1mr7mG9QTamVGqU7Gre1O9i0LdNRpib87JnReeQLYqFvsLtqkbZTxpGOuay8njuHhEUh9qOhxlDHXb5eF623NMbVrSwGmbP4nsHdWOrzu\\\\\\\/4b454Pth28+08y9+rk+Y+LV6TQVXdzxdeCT1KiMupRZu9GViQkbwI1xMW59gi\\\\\\\/TlzzYTl71njstFSMzbhiRpVtIQMr8apnImH+On4\\\\\\\/Wv3D8oo1ziERqgwHCxdMi28ainxObVc2P0\\\\\\\/zqaHd\\\\\\\/QxHbrXSJvnaVbx6Bepa\\\\\\\/e+lHMsIaCQe\\\\\\\/RRCNIkriPH0Ds2HkkIiTdK9\\\\\\\/b6q7XOXTeXkm79wCovB1W7O1h5Mf1IklT\\\\\\\/6sr2gVmUtr2NUGTB48hgMswkvR5L87njC6kvPpo4vznbxdzVNUJE0w3ImizAfnlLPFcCwIFSmpmQFUsy1sMzq91w8osGZ3mXVAADrqI4UPNUb0srgYAuA4ZmcxxRLL95T50ofJwFYQwZ5SIopmXLg2VE0JgcMCLq380pyjvSQsk2OEVWwMBWoing5SQjuMAsrt0JjcSw2XEyAYj2MPvXzuO9wie2vXxDXmBIftvKJWwaSbTJAdaVHw3GSR53hcoVMejKtz8GW9mZx+\\\\\\\/PSNXjx4b5yyE0hLkDthtfzHtWJcTeQ1ZqxFQ4KwIizNgAdn05MvmIkxc8rMi9ET00SgFpojSj1cSqlcmkSrK4Xda\\\\\\\/1swjdmw00jlkZHDqLDgQXmgv\\\\\\\/Hafm+llwXBCkDRV2ASNvMsfYfZjd35WTLImh+44HTR5Qrmy8tTjmhbbASOj09P17h4UG1gqiNAQzt1Z8O0wE09v79ICiVpHtJ\\\\\\\/t0ySs8MF+7PlsrxY23kJUHgco7yx6eUdBCrXNs+Z5a9JtQhnDU+LTutzcFm19M+ssojLFFAIptJPx76GasjsJ5uJtWqgsTunSNPcNicIFrYLy5HXzBpp3ls\\\\\\\/aP7UbsS63uTuwBGfz+M91ZupPI9UAtOKvytb++zsUQYTDxwz0\\\\\\\/M65u+ONQDBUrr1U1K2du9t5\\\\\\\/y\\\\\\\/7Hr5zuEn5TydB8ah1mWAcUD7sLL\\\\\\\/1OW+1Y3yZZtIEAAeDoljWCKz\\\\\\\/YAv3LQp9M+vMXBXcfy7\\\\\\\/PBfZR6ra0TVABVBs7aqN2E97JG4R7+kFJ1vEdf+4GrZo+8kF2dCFV2kMJCCu9KdnVR3+jAgNzpi1sSE1raSNAUXi1+aVYDTsGD8qd4GOXJHEXLpDZAcsWKDEJiu9d+\\\\\\\/85W9q487O1EhBTMGx0DfifVaCzDf3EyR9hm8d9MtFRxEa0wyJE2oYtXVFyjeZPLZ13MokG0FN9AeHN9AiJJ9oVMCyOSfnqZweTxEseoM2DxbG8Xs4sV13xcsU9isj\\\\\\\/kXCRp25ZDveip+MOJdjncNRSG32maC9poHIYJ1W8vH+jwD3cl1Q67M6AFx1R9rQb8umzkST5ctFnFYXzoRcPOmW8Az2bhh+pGVZdvVFPcZb5p+8Lt4XstTaVYkiDKxZCgP25rapKurKkR3c2lkRCFv9+EhHE4YHHpPUH955FfacyMyteKZ3ruiN\\\\\\\/2f\\\\\\\/YCns9AWIjo6Yyi3Svy5oJd+Qeier3UqVeghzu69BIQPyIyb6m2VGIYC52m30UwDlHZtW8qVwNkaS+EXuOcnLaoRBpvSbUq1OoRcaena1I5+eBlmWe+6NZ9t7025VCCWzMbf\\\\\\\/HrTRr\\\\\\\/ni7RoxAwlvsor9h2CK+tkyDTBY199gvgNtWMn\\\\\\\/q4qYFMZrx8A7Lw5r2xKQu7yCjib\\\\\\\/k93w0ZOugKnekAkSH0HEs4VYacdeKx9VZE9zB9z01V6Vii4dWQaULRSkt5CmsvrgLVYl+EuWAfuZC5ZEYRj63UdImAgpMhshKIpRuWqFaR9ZufzOZv2sghFUnjIp+FfqSn78YQTqdRT2BMZPhhCnaqlvNVFy11bHMlNxEAvyCMP2rGeQlLl3L\\\\\\\/w8oEElkGAALvut3zRWxGK6Ac0awHPlUW+HOGYnmUUTIRDc6WVFJToyqP3rr7jvyZevHgU1JhgU4yYrQSjJx4+PijjtA9Ayf1jzkLs3YYj9Xc0XvvCCC2ahkSD6fhIMt+W\\\\\\\/5E29v8NyWPrDr4+R2cYSS+xgIOdgk4gu8UGT1k7g1t+k+9arn83QQTAYnxAIW87Mj31ERbGPIpv54++abpAGHC8gZDvbTUn2dXr+dy9ve2qAGa2tzBcg190\\\\\\\/mT\\\\\\\/REqGWjr\\\\\\\/LQr8hKmOhtApOjkyH+EzIiafzRCEeTXZ5etEFQ0fFWpYno0Ss7FBz4k\\\\\\\/0fJ1dZ60ECLmO8GZVUYpF1UtNBn7M8Yi7JGjZpTiWTE2BnY5Xsvy2jCYFYO3Rh9kWm+xZxWLEOBkzBQ0GtI+wodDHxu\\\\\\\/9dc2C1holhBou81bAc3c\\\\\\\/wR6no\\\\\\\/tSj6I5UhumVLcJbp63LzbV+OpBTxs2qWBxu+G8HH\\\\\\\/HbzOUnece22TMzqnI7k58P8BBAgyrirOI4DBjHJF0flKBJHyj5XkqKh8APirN4gbhY4oOIzjGKQ0RvR6XgLj8FMQshzvpxTvR\\\\\\\/\\\\\\\/uqFsEDcQDyw64hKNkhsOcwG9B9oZHNG1S4b0bZVtfBxkKGRTrWDKLX1TbOU3WOzcyLXSpqWyAO22pJwZQNTD2CnBHvkYU9Ys9jJvBrc9XGBwarFphJB5yNAyTv36JBsidF5qnoihhr48v926d9CogjgJl7h\\\\\\\/d7wqE6oIxlqehlqJ6hhc0PzuBkwEAn5W1km\\\\\\\/WGMcC2nJBWtrLllehaumeTPsgvvlab1J4PIgZZ\\\\\\\/io\\\\\\\/PxiEQZjPmDkk73wH+WP+CtmB+qCeCMVBJMtmMSA++b5IjHHhhp52wjMikRzVtpG8dAX5KwmkXz7+w1NNtaCHKsEkugAWBYBvmC5DhCTc4fg4IZR8oWm5ryKJaHQofuhaiAM0sbi5gXFF2VCBfIQR2RrVto419n6ShUtczndI65Pd6Vf+L5smGpOUa1OKhhwZ7zz32Yea+WrUYvKprfyBq2N7vXe7XTe\\\\\\\/vlIyQ4z7c+rwkyIbaYgqGCNCUpoxIIcMI6CIbAWCaXt1n7+lmOPydz6FxmIqyciy\\\\\\\/25bIJNzImpom\\\\\\\/sRMY05zFdMoeSuYa6\\\\\\\/hgnuwrDq1Z\\\\\\\/RLbJXTkxsE5yzZVeldQFpB+\\\\\\\/ZxmaXx5mI6Rii1slVSaXy+lGoW5muuJjQ+uXCjg3WT3TpYNuJ7e2e6IPj5SSduJopZbPWwUo+HSw7i8mAMz0qfAbf9oY8vbnRcpDmz\\\\\\\/iQDxg2LLPK+LTlBhAZPBmLpvaKHhiG08HWtzadxENBwFUgwsqNJXHPvtabNo6baFApa25CUU5G6I9sJqDhehrRs\\\\\\\/ssC9VIV2ybvAyejZZJueiBW5u3lW6dOex2rmlWbdK1uesBr57Quf1rPgYr5CUVSZD7u2na15KSW4c30SmD\\\\\\\/HRDhGXpP1dDq6fx0WtdCIrQIoBiiW3Xk6Y3p5pcx2wKlTEjXMtOQX9OCCrYxbf8lVYQQjUY54B81A+t47+uHlKV5SxRYEpZAXrnwjhcK\\\\\\\/6bG999jy0\\\\\\\/2abHbnKM3aE5enN1giCDZWveyvulVbgmQtH6HXx+vOEd8IrLNYANjepmW7L4eOvZzrIq+OAsFSLDy8tfXzBdsXRgRqmLuC0W\\\\\\\/tYfFMDZhMSDR0UxyK3hJqFbAz7s7MGAC9lXIXyO5m7UX4tQnYOyca\\\\\\\/YkXQyTIByKIxAEetr+k9WxANrsdtwzvyaUejjlinpvUoCbySmasIIdjc1GyCe+SNAFjljd+ImQOyp4WFtSB5tTyNv\\\\\\\/zt\\\\\\\/WkJULU\\\\\\\/XuHfJKNsE+Xy7pdZthwTrcj2VX63JQ5My+W6OOZfEu21KW5iiAwp+M8EJzeOSx009MGtWL8rth4n6Gve78PRLZ6RgqGmvyLcBoAXwG6GIuhzIIdHRtfgapuOcztSkVMMhMkUD1HyEsm8Q31VoYRDMHGKWc7qK2u3xztpw6MONMM9SKtp2Box+WWHOx\\\\\\\/K6aoJgpx05obk9Nu1112BylF8S1L8Sjf60aqeScjKak+GkCY\\\\\\\/nVHdmyzKr45wwP4ZmL0qe\\\\\\\/LwE+cwQYZj8Q4lsMIo0PSfyI0lWangyHvtDUIFTlfYEWzEbrxat0HM4Ky0sj0K57GPuXGPnR3jwGdiLFhqOXn6S9yv3hPwe+pmQiIdwg63M4SvbqukgOAu4yy8s0+PrChFgk3bGJ\\\\\\\/rwUETvBCITV31V9SDIKLtyGqKUiciyI1b0meZNuWdApwje6vuF3EXbhkenGJjm8uYkibWnHFMDun4Jd6sCVq\\\\\\\/iH2eHHQdRSUQglH7HYfaSQY5rPaIe4BLFl4lA\\\\\\\/h55QK5hrrD4J+TgnE7Vc7ciGNy4uI0FEzZUGEsAWw1vOZCPho426TmzOnE5HbehFMZGWzmvymebs9UHpJ0kVQr7QlzkUt0aESYSdL6t9uaEk10RD7+LKvU3AzfW2e6t5hKySTd1MuZwCrRve6r5W1Mm73S52nqEFEdIKjbABqZce+9WkeU0jlqpg4jcauoW76h0d8dlj4qGG\\\\\\\/80F6LGaZMNLz2gAaCj7ZSMwBDpiSi7\\\\\\\/Qpkg6Xy22bQPR6SzENvPM0GPsSa\\\\\\\/P0NmOtWCiH577B\\\\\\\/TH3pq+a9iGPKPcal+Qyt3yMbAqwfWqm4QlJEM7pwVVueWxGs\\\\\\\/UwKHK5ni+DmEvgmnDxXXOt9fZn\\\\\\\/65q8R7ZGcmNkr20U1BHNzEP4Os9QfTjm\\\\\\\/jTgde1aiS4hLIDl8AVJ24HsYTWAQhGo1Q0UUr1A2DVrsyMrqqbI5TfmCdoxRRGOd5I8aAaGB4p1eeVRzFpI5hxxgMVY4rCGYDaWyaJJc+S3s7zz+GpeKS38iFJBNkTKP9ptaiHFVt1IlizMOYI2Tl4+E8YfYIqbZohKsHsLejpt843WPTkKuvef73AkILVAs9dg7jdP7VJtCkf\\\\\\\/\\\\\\\/eYxj0HQkkaSwh97w3CIk4+u1\\\\\\\/WFeQsB+WLDSKZEVxrbFOjTc+yVptjmWCA1Zx5HA2YDtiN\\\\\\\/nDQLUiJejpkuu5qgH7ZssJmX9PP0gR83Pr+MmSquVpS+iXG7l\\\\\\\/15E9c9a65tb8+lxG7QctyygoFVloObIV7WXWcn1XqYblmwpY2R0GYalxz5vBv4mu7CLSlG\\\\\\\/WpO7pRCm\\\\\\\/NW\\\\\\\/wtezjN3g11+EyTjdKFa5ex1KY6+A\\\\\\\/QZXgctbszUIj1je1Ri4klxhE4Q6zu6cs3eFcg3WPIxnvXN2ZyJc63xEzvYTqOSSqrky6t\\\\\\\/Pxep5wKn01jR140ftVFyPVngIqlKXzy5sEo9OoypjgYM4zjoo3liRI8gD3y80iKHLIEVX92SCWjc8SX4kn514vbXjLMl+b738kHnDyrztZDtBEvfFumJYCZ7AkK2aVNP6xpfWZssMRWJ4vsRPxy4KYTDzJdMpAscqfqSlTjIlyibnAlkNtL0NvYtN5nBc48iAjjJyK0YRJSyh6c+xjzPttRxFGsUNd2IuZslsnQej1I1JvQxtXZZ+0Ze2ZL9w4Fsp2Dkxykg+qFBBfWMaaOPTmg4MosraOTcHW56OpXpEa\\\\\\\/1v4g+JAta21N+yKKXBrTcsHvH8Kgii+qwJzck\\\\\\\/Y72OcTGFUTfNkzFYsa\\\\\\\/ttRXPDCak09Q10NMph\\\\\\\/WBH8OztultRCVTMfJN+A2zq42\\\\\\\/U24EG0DVYFpgNrpGU2S7HCPcy682WXd5mbYYYx1qBU+wP0fOvxL9U+IC9uHxAB9rbmS\\\\\\\/qhDRf\\\\\\\/4ap0XkKPlnFqdOiHfUr4xqmaAO1vJWC4q4E5WJQHzQo6q0gST+EH7fcGMSIhr8eTxS7PMixjemwo5Z28aFy7Poto\\\\\\\/Tic4rKaawxYPXLNGigMYC2yMunc320+gTSE\\\\\\\/jdFG3pSnHn0zlF\\\\\\\/p9t1Pl2zKf0KpbFsnH+qc3m5rioaKrCZbAZFv2LO0uJFP30GjrgZm5U8BNsFKwThojQkS+nl9gmf\\\\\\\/DgnjAEn2X\\\\\\\/9TU2hXg1XF3a5PUhC2zmoAPc23PQQ7987min8T7GsXxBRi99gE\\\\\\\/KiRO2D14RbCPPbF+w6iC\\\\\\\/3TFJCzvODBCcui7Y+VTO2j3cAYVJkEIhWs4yQXMTL9tbWHskvfLy\\\\\\\/RMAmV5tqxyQleVjZ0nGXcqcAR4f6XRsQLFXns9c0\\\\\\\/z0mIBwGmB24GzIh0qKYZGBn7xbbKnvJU8ihnI153LSyKnrNBXyXyUSoT0xIlTdh\\\\\\\/z81jeJZ7+3qK7HfkRhPC159X0\\\\\\\/KjITuAWY4QWwSDDwoC7qe40D7jJdW2uQUyWtwE2AkyHnUKVcYzgW+Mayzv5t2n7z3qAPpjHRImbY6aIQ0\\\\\\\/1f62t+cPUQHPyk8\\\\\\\/dy4yeEhDezYBs3r6NjIGVs52ZLqhIT15R\\\\\\\/p\\\\\\\/Aar53g8H79L1mLIJ2h28T4OJkvP7vM1AhaVZpotdzZLTdIFIBborYP2MBZbcUVRIL1cSlQBW\\\\\\\/lOq6sVt89Dww0jHB0RLz3gelUBgVFyqjO2Cx\\\\\\\/BSgSS\\\\\\\/SlBAdwnsWsrMAp4K7k2ORXqeLcVx\\\\\\\/1i7W2Y3Sh+jMLqYhh0bPHArCMe0FfOOkI4vQckD8pbjviMvEmubKXOPMQz6FPvZjTlkFMETPW+uAKIP2vhVJmLQUh9Dtsnqf1Vxz+IQKR1+RLos0c7GCWigWa0aBeKtX1ISitAWqhlnujF1nTa726UmiBs0c+VKg+rUNYlm8pw8Wz8qsaeC5Mky9VMKvHi7SOucimxi1\\\\\\\/M3uS5kBlckCJ0\\\\\\\/hKzlLgrXYAYt0LfLJ4tDmTyuU3875RyqxuPKiaZTmev\\\\\\\/vnPO9hJR18jtqdIlBrxwAVrKXLHaFAR9WTbn8Mqz030GlHER0wAf8Ag2dgpknv5YGcxMfm34c4n58bWysw1Du4p25nRXo8X2ZXGMxFCEwxd9ovJDLx2Q0bE08c4Sxt3VFKQtYNA8EcHbTmUPkWbPzIOetY3mks7pJ8U\\\\\\\/Bc\\\\\\\/t9zs4KkGAAuPO34iGKQHRgVjWoSvNf\\\\\\\/51w4SI5VxZpE6MEFQGM\\\\\\\/7KsiLcndvyRREu4oPDphJSx0bCsiDGH7EM5czALxR59cRVSpw7jCAcGVGiXxAzplKbdQ7TlLlKs6EYQSCNOC+GjwE4Uqy36fIuoBQuGmgjWWWiB2tne394fVFEknvIxFJL4JOxLGeKZbY0cw9h4enRrbNsYBHVE7axLnfrYuBLwPmO6r4\\\\\\\/LJ\\\\\\\/8aYy\\\\\\\/FtvnpbrzUA2cWAYum1ZIVBiSXTUEmhh2Thno6CAw6qxOgkd3Dk4HKslEGuKPwpNXgRpm4aSktipj47amjQCVRQyEClt+u0bX7WeXC8\\\\\\\/pqcxtUyJjpR9llj8vXXHGodF+I\\\\\\\/LJLS7mwMQMAQ5pZXBU9ibX2+8U09H6Gv4nHKb39CPbmByLpFjTpe1LjyeIF0nC5XvjDYC1hi+MMATeGuK0WaxUAaUIhyLT2xXkAG3wBash\\\\\\\/oySA65Sh\\\\\\\/9o2KgtR87aYohd+WMhglfRAB8WGp2K4f9ExWvpYV384emIfkqlKMmv39WsTxQjyr3l84JRdjTq2vX4umrsnS0NkITCl6\\\\\\\/yJrYV+Je+p05m\\\\\\\/\\\\\\\/6aNL1xZi5dVERhtIaeLItfPqP1rv9CyQIiOxlK0X8DeU5qcGeiuy5lRhcutUUVBSWupMv\\\\\\\/H5X6ead8oxlK\\\\\\\/iAP0JlQyk\\\\\\\/nxl1f4tXUJc2NrPjYuR\\\\\\\/zpTXclsmzKeMVNeafk6NF81EgzcE8LFVdsEUej3UJMewUt9rcXaZRw0ziLgWRRp0jV9cf2fyRpGFP+B8OzquxqCs+Ji\\\\\\\/WClAKHEifLH2jXyJim3aqu\\\\\\\/r53f4SLS\\\\\\\/psLf9RjXkVctXgorJJj\\\\\\\/MZVYZAlXwEdq9LvbWYv3HFBcbvn10vwl2tZ2b77OPhmahKI8RX4H+ahTUyjP00QUQolkX\\\\\\\/qVZV4fJw\\\\\\\/36R1wxdulqI0Q4wpnBAQiRC9e0ZPUaFCsjsBvmdASexa4rJfVi\\\\\\\/8N2+GX73Y2JuUYzFH8BPMQt63M1GjD0DjjF0UwwswLSj957XHoetB4zYaRVcIJmYl6lNHIv37lg\\\\\\\/8k5vgSp1hcviEOLD2qSZ0R6cl+UuOclx3s+Gj9UhKkPcVDO4vYTv5n0JF\\\\\\\/O53nZEc\\\\\\\/LKRpoAWJUjk1Us9Uh6Gs9zDC97RhP+hq34hC5YTg6txTomZ97lSiLsccg+y3F4kMinrrNEJtoT4gLDZRzliEIB8ckJydAoAw4kqtTL8KH8mYnr+l19yCvxsbYlkpYP82V2Y2EvUBhSS5SWwyIiKwRwWo7MtO0YUB7U79eMm4dUQvqgUyFWmP6e\\\\\\\/MF7l1ihlAM6RwVc4QsGCPfDFiQ5M2YdEEU88FgZmhDN76zrOzgqMc1MhOlzc1mVXbnQZzE0VjrG+SV6omglQrWBgh7oJlwoCXmJ\\\\\\\/esjDfZopvKDLtq5b1A\\\\\\\/T6UidEuuxIMEat2tzCHF\\\\\\\/kzJcANVtXYCBYKHJg53FCP1doxyyp5aIx\\\\\\\/3sEIkSbF00m5PhDL5izuMAEgKkkOlbUMP4f5k4yauVeW8ym+sIgDC9gMPgED82m9oS8nMaCjXPxPjCcI4tAxU\\\\\\\/Awovj0pk2WGRuN194x1lqT2DviVuIFG6J8fnPM+pk3uX\\\\\\\/IYQMWA6xixopBCvGA2IaOhJriMluI+aSj0ASVxYiAnSXoR02vdy3sRR6\\\\\\\/yedapp1QDzV9o1FiUv0iPKaUZP9dOHi21kax+8iKnB1b41YNDEi9oBr6rdvJCM3kSu6PSu33mNIZS5F3MvQQMwyQxO1oNsBwXQGCArNta0lKnJu5ReypqsWrf+4+DminqQ27M4wnVMb00cDg9YivKsqd3IBRypWUu4kowhWJUynxZTW4PT89tu7w3NSsGdqfgVdyim9PEdtkl3wvRD++rvYZ45mk1bRwcZVIWA9uxjIFOLVhJCAQC94XkCd997uJh1DVqTLKROaQ+sHp5iUNSuQXGD\\\\\\\/KE7zP1gplQ9gZNNTA6sQZyGlUxlFVeIBNOgqf0ByRkR9hsrJZyKvPFB6qmHGNPN8JlMTZ9VMCsGB41v4M6SAV9aR4Hnf9j5oj7nR00\\\\\\\/K9Yyh4JHf5pyHZ80Ra1GJIIJ7vFi0wl9U8RaRwYrVqE3T+C+hZ7tTLABNcvr9C5+sDbpOZT3ryyUyCAD5C11o0m4Eprn5KnN4EqdoR44W7loCVM4i6rarnJEcac2meCAsz763FcqNBn2a5HppoVjhoYIQqjik6QZSi7fEoFKY9pROSiH8d5QDxomj8h9cL20uozNdkeY0i9NZrhqLe2yCOARiUgUFL\\\\\\\/QxT9Yku617UxZDUsHLewDqjXccKLws228B2ZZ458qSVsBWITCyOadKoZ6\\\\\\\/UhPslNaQUnKowb29k04nny2hGZlG8Kej1uqAfdQirU9e4UFa97H8RczEsCYDUZ2nA8Z7DR2wFxSn3V9NJTdsdy3FJv2dpKgdAlBQXMOxJqCIGbSFmaJFz03SXF0datufQuozCFzQ1sJVMdSM\\\\\\\/S\\\\\\\/6CS1BQPoPDEatSbc4kx+sPeFroor5MAcuwqvkYNjDSu0ZOJpX4PrtVS+22EDWmdxUhFUNgPH10qr69NY6UbFK7rub+O88HwBbcbe0sDwFARHaBbPRg53ITgAArjV21B1JcHoN3QiVxFf320o+tdAKgILqqtgqrmtqr7o5gsUbfORHPspodeOhgptMfnLH6HSwEGM4EMibGjD48FDyxfKdnAztLgzUXpOog85Xi+YKSNaoJDt49hKuptXWz\\\\\\\/uB6qteJMwzOSYxg+CYpK2l9T+VuA+zUb9CG3BIxnXyG0FNW9wqdsCJqR4UhSGBen2JPvEOQEbpIfsF\\\\\\\/pK7tguEw+YitL1GE0Ly8L7lBuGKYEnkAnfAmV5KjRvhFgPJN8Ubnuv2w5vV4Ja+j3ihzfOWdHGlN92BGTe7b+hTjkVA0IWKeavRrEL0\\\\\\\/87IAjV8VXjNkM6pe0qxPU7ggLtFTZ+vx6Etl51Cf+t9Wx6dFm248mA+gAjNfuJNsf83CGCqSESG7FnYDRdJj1tZ8RgxasvPnDhU+HHz\\\\\\\/nbRKb4tiS6ajbiB73W3lFTY4T8VK5ybSZdim5PHwUd6G2dA5HCA8OdB7v+912pV\\\\\\\/zcBFyrW0p7ZgniLGuY5yLby6lL1H4phryU\\\\\\\/eO3qchtsSmzPQGImrSsvk0\\\\\\\/J1kzBDYAWDsWAxRJKKMR+SNL2pngZVHLICm4JTytrPEP2E+oqIRpX1VjmnpLSgmIR6EZvas3gQYYITWyUZmbn1nrgp3LUVJD2nsrZFS9tLZy7azW1vru6h7AtcocMu3YzYYiAEuDLyirjDUWykqlWPXh0RoXanJvXFyjauMirx5XXwwl16DjEOEVzVlQ8N73EOAT9tZSx\\\\\\\/Ojb\\\\\\\/+rRV0jbzaashXxRJ6TgGghDfdPkmKLtElqPw8RefcTA67AhWDsRz8E\\\\\\\/9NW0MTMOr65Xy42\\\\\\\/GJR2nsnWislwHHIPATuR8JOiaM+W8SpPrV412kJZ3mHlYb0QGCEm8Nzrp1YDdQUYIRK9xjwZ22FD7I528dAkPEy+BD1osjXYV0qdNhDQhxtubiB6wC62f9PJVv153Gp8vY5x8xiJIl\\\\\\\/1EuwNfdOtlXSnGxo24\\\\\\\/vwGEI5eHYPsONAtNr962xzkUiUkX2ukT7\\\\\\\/pHeraghzvrI+6alPeAYIQLWaTcmfadFLpJXuD4j6I2u8PxjV4ZaKaRfjOhImYuiXaFO09SI3RHA2uzS38mDmXIR3JnPvcemE1LshD9i1RqzmqfYe2spSC8bgb6pkvHi2Fz\\\\\\\/qQtCv6CSemPx3\\\\\\\/OwIy+DqppsaVXhhU6jAnJPI\\\\\\\/UPE52JlAy8cjbeiC9ajS\\\\\\\/xMk3gVfnXdRWyfoLfWLk8AOVJVJ80yIGlxv39VY95hVeN4fkDY7nh0\\\\\\\/TSJu3GpvfPf4T7+Gmp8sHwkOVIo\\\\\\\/pGAmZ7k4I3hD8LOdiM9I5MC8MJuMvn5G88gTl2GhUBfJPdpV1GTd\\\\\\\/a2pVwiw5dBnUGkPwO5Xvmvx7SswEVLInqxYei71lRnM0QvK9iMHfwaMT3CIjqXCm\\\\\\\/70\\\\\\\/s\\\\\\\/1bVyCxDUdTVGBaQIwoNyoyRVVMhX32caIyWOSSHMo4u68mOz5zFfZGtPOS+doEYEb99fyrDBlzQIqxhFvdDsjOaBXZ0rjZ7dNKQKaGlxeRJMURX+nQax\\\\\\\/SSdKXn\\\\\\\/oJlPVLUow0IqMPgg5tVF+xM6DVeMw0zPpAJCeyHhxj593GbMNhbb7eOfYPr\\\\\\\/rEJrA4OBkbh40Pu7hRU2DwBGqs+pUqpEoX2MkL6STJPepii4Ar38jJeM2jixcaWSa+At8p8yI\\\\\\\/H3d1aCLVmQsHNtaGE5x6VAPg+jvwXAH5WZgOB+KQgt6MoJV3bvHMxmN9kb8Ju\\\\\\\/ro9mG14+tB8JaISQEJT+IT5km3K+OkrPFEk9XEMNeSEhjF0EGJTxeBVAxx\\\\\\\/czZxLRsTFEVolZqjGacm9N+tpEWJ3gbp8Ft5+fiGWSkF31SVHUkkrx7QnOrY4OLpiEBrP\\\\\\\/O3CPg6vvvml82MRU9oGdEqFcRMjOSsiaQjRo3q7TE1HoDSEfBZxeV6IK0apAZxeOFiFmkY\\\\\\\/\\\\\\\/YPphoLEz5SzCWeCX29qWb2ZTZXVxeyhIEDy\\\\\\\/QcBCfArf5hRmZon5r5a37JtKSoiEJEDQO7fvplLz0rmrqVuGRoiOjOFx+buPcrowI0DXAnpxbzLy\\\\\\\/BPDfanKIhUnLJ8BpPVvZtV1J6mQf4vlbvlbUYFQ3s4957cg3Dbs6ihn\\\\\\\/sFnCDKTmCPKHJHZa2a6LCkdBdUixGOjqbS65UpwdrKdoGo2CWvx1G6mCuo0Nr7m4Vc8+TfiqEYAWKQU7pYywajuKp1U7ihquyGs2jzjsS0Q1pVs++ao5GqF3FyI57Y0AWYae3qofpy\\\\\\\/STzkLX6GKzG+oF6eOQ0oxEmeoLYZW16cBaycN00THtd2TJM+RxceFOjdmeUByQeXtySvxqpOcHdUhJgQcd3wvBJ6c1AcjJUtajUR8y5Yi7\\\\\\\/X8a\\\\\\\/53qn3M0wkMMPhwHQFEtjhGygtrNufkUCJTgaEh7lLUkNWPxdQwT9ITp2ykzOi4ObA3vLMdIlNUkmXyVDiV618bZ2QGBx6QVkthb2hsWHx2vax2RibakbYyP6XMPgTU8k3Dgj8Dmi1S8H08v\\\\\\\/0wukOEwpahssc\\\\\\\/p8B0OJRhmLZqbZMYvwRgS7JZ7vbPubuBUzDM\\\\\\\/gmq7I9yzYOW3YCQQO7dID9beI128jLCWLVDj7nHavyhEpMdaKIEtZ7+xQLOr8hWwTzkzACZKAFhBlgxH1rszj3J2Z2BwpZO80gLJ6Kv+M0zJVlUDUy9nIAM2ndilGBtCCslHmoxAwQdP1iRG1i50zXcmXT60XzMqF\\\\\\\/COeCZ3tgLOi7ZlQIn7DSPmtnTuNDIPsR1dBBaF7Kjnoxw6nuKtG4rdXA3y8m2xSelQRiV2tq3kcYxRvbieA44Kaz6wwRW4qkz3X0wPgfcwt\\\\\\\/DUp8ea7hfVM8YQleZTUDEUesQdxyCDECaZtGfkEyOsEHYXMrDH4Rm0pPbVIDWuWlNAAowlZZ1ekBIjy1deR2yi1FciXyG+h7eRGULzd9Ar4mWYIpgVfxMPTpUyOK4dN2HEeBo6EAWAxOG6A2ev1D7YGRQrQvBbxsSiBNLQg6B94oO+kQWaVjdKDcXOWc+QlFxl6Ld3nmusJy\\\\\\\/7NSX7JW87cMUac8l5Xe6MioigL9jpeJSGQNwFKBLW63IVMyIe+0fcuYubaCVXJPevGfHl1+MCAg4T5VFCaCBAeprKg8o+M9IZM8C9wn6JCxhnXgM1H33JHtSGqbP5\\\\\\\/3WHzmFEXgVTW3uMt\\\\\\\/SFfcWQsKDK5ftyLWfSXeDk7fwyvbgf+lpyG6MAN1T6h+KDhjPBjJfyvhfiLOafdIXMekNnuivpzSkMBPqic0uMgYF\\\\\\\/hOs1W26xg4Y2mW8fmVt8JnsBWyKnMgmf6YhX3F54NFPfXd+9MC6NFn6VAuGWVXT+BHQ66J+LF\\\\\\\/EOlFIEgIVulkbwu2nd9NHRSHLUae+kBGTJYoBiWy6IjeZmaY4bEgJWihfmjGQOsB7eX22me3Rs6sOvPyl8GXxwMe5IBSEjAo8WcVM9jzACjrAYLqV+OX98ltXpNuxWhWAiOB2C84+B7hhKFDNrkjkoxF5CBGJhMHooCoNfaL3XlD7sfG\\\\\\\/kWSg7FQRfnPoZ5Z2iIhGnAb1f8BKbq8JzF+06Xj0LucJnkhmmUP20XPxSM59BtwJ3bNAAnqmZhWCgQXN5WzCH5OlzFqCKGMKFChZcg37XEk2X38BroPYhQ3JQUlGYDHwhBcyv\\\\\\\/fKnWmAVMQX46YJmc15NzndZHUGgFxGJB2M+\\\\\\\/m62totjusuwYF3J\\\\\\\/7shJgRCo2twjBNv\\\\\\\/Jx7YxwTs\\\\\\\/2\\\\\\\/vqHdPVJJkBI6Ng9whBbzxxrFeOiN12lTu947QY5SOA\\\\\\\/icyUJmpbpY3Ur4ZE9AHKGTkJGcrstf90elw19qPAh4+lzkFmnITcRupEY8d4Ks\\\\\\\/4wB+24ZoeGjbpOGF67NWi3k8DZzJRHkXcWLJK8qVL2IVEgfEulizcyLw3+a\\\\\\\/vhtS0iMwFpaomaAgtgItn3BLnz30Qp4y6d34bwBp+9s0ldwduAZ7IdNV7eRlRRbdftyIFCsAs0E1c03ACqvgDv5WUHapjKQjV83dM10V9GXJ3LqvB0gZEvHvv6rugPGbvEG5+wtbi9OhFEKqAloGvx16aKhkENYpEWWRwVcPdXY\\\\\\\/c6Mx8DDkwdlj\\\\\\\/MJQ3c\\\\\\\/CKPUbpxcJc7eClXwTvte8QJxLOVUkj\\\\\\\/6cH7iD5npWEH11Ddi+CMv6XxxXDVlYa7UW\\\\\\\/mYJYoW2mtAK\\\\\\\/uY9lb8o\\\\\\\/cS4qF2hHLPAtqhLtqiq6F73MHjmL\\\\\\\/PMPFeymLVKj9nUmvKmIEcjHUVvTcN6m4DYpX71qLvpfdO306kvBSAnsm4CFxaqs3WJ+iQbG+ZAKs9OAiS\\\\\\\/3qDLmtAQ55x6+6JFTvTh7cpSnwEQe2kUxc4ZKIvWi\\\\\\\/jNjEqfEvH7yQk6hoz170iTa78PFyPFMj0erBabvN1YaRoDBP7GB81sc4nodQ+TWCpJ0dtEjz\\\\\\\/0uoq6P2HgK40w32c9qCq7atIqfW05zyHMT+RfWd0Bwb6IcYpp5gTLOUysqP2og1LifXjvoURy8qXeaq2Kwd6weEndjA6id7vGNuvg6ACI8kvdaB597sAcFQdAX4S28K85zgjT1PHxzD0zt8TjzAGHswLgt6yH15Ps5bgsh83odmjdZcB1ceFrHwA+Hf9UBa9zDKlf7Jd5Mx9WtDoeFoYqsoQAOb0IIhEfBrq1atQFVlh7YInmmdNxpmN0JY7mKX8mF9EDouhOQShGh6G79QBNCzz+q8EPzAY+m8RgtSs\\\\\\\/0y7HeGNL7VKieH0exYWvK4mSrShgUwTR+\\\\\\\/cN3ytpVBDsMMiy\\\\\\\/dXs1CkV079c3h5zbEoeqV24S2CROTHxvrKiQl8ltHmc99p6vuKkl\\\\\\\/668ZkFCrb+CTeojo9K2gnyoiKB9lwkzRgzzJ0eiLFbzxX+o3+mYKNpLuJZ7I+I7V21ga3y9KK5PmMCQTwHX5asS+DlDKQHZ\\\\\\\/UYWepUkL\\\\\\\/GDcfBBG8DVnRDEisvU9QmMbSO5zL3J+BfRi93T5yF60nfz5q5AE13XRBuTbJaYhRTOCI1smns45M4EYYVgDNxgvturyx0iPh3\\\\\\\/1sW4tBisxn9UbUgS\\\\\\\/6IuO4jPu8myyi82WdgxTnMSRGUVlqI11xMkEMcJrQwIh+Ur7fJdyR3aeDQKU6rC5MHrgLaxtUVyW7hLV5e6BbnBho354EqystLgKO1UhF7r8wzlxVLMMKAkcEUxbWZ2yIbc+\\\\\\\/LpDkQ8lgZt8jqCVxXbRtvufsq2FmNHTbUX\\\\\\\/hrJpU6jT6VK6ydf5JeqarFHUvr5J7yBqPeJb\\\\\\\/S6Rfee\\\\\\\/ByKJ4ZoJKPPRgS+KU0\\\\\\\/j4uH52PTgeKlHh3L4oB2kndBM6ykyHmePMi+D\\\\\\\/5k5XgZzl\\\\\\\/j0lzP\\\\\\\/McLbTwaD28r7kWR\\\\\\\/M3EAjPlqcl\\\\\\\/oB4O5g4oScc\\\\\\\/IVEqep0MjD7uPVvh42X6hOQel18HtpgFzIXgv7LxGkDAewJ+W1qoDx77iW3IrI0vvutCfzKPmBFWEl6LOh95TrPqyVOfKyB7XGyueMmZW04LGl+pirlFFdxLeMg9h2gQDT0ZDxCfkmqvE8BFHqQ2MMbkU7BM8iIXN0Y3mk+yV6VMbivmKCM5pKoXD90oVuSCwdhVLkNWudf59bIKi6PopqlT8Yvdp9iRxLXeYPakte2j2h4WgkBmi6TmrrA\\\\\\\/DHZTiSD8J4rf8MKZrSps\\\\\\\/wZo5RK0KYSwbqfGquGsqM4BEyuPYERTAIMvfybkd9\\\\\\\/6b\\\\\\\/Qb\\\\\\\/5wjzqC1vVQuqzD6k4JlGANLqQ7c9qoEbmJWG4ksUS3E5jR2qNl+zFnjiNTuJojs\\\\\\\/pvwb4dWumh3tHbc4bkGQEJMdOs73rypAjm0netd3f07DeZemNdFAAhXl0RCtyFExbc1H0afe6ioUoRHzRJejr\\\\\\\/8gIukhZfOCWGNFuXMcjN09LrxZ\\\\\\\/9XjDYU3Aa\\\\\\\/vSiJK9M+id\\\\\\\/ZmA8RC8YsfR3K5R0vy69jdwtAnaNslGIY4K1nxL7KSgu2HniqzwFHJPQN0Xx04++EyWIytkYswbcRRpWQqAcSJXJREMG1S4QwmD\\\\\\\/TPp\\\\\\\/uJB6So+evUk8s1WGMUaAv\\\\\\\/K5jcUfmR7GpYrTsYVsz770C8ehiXDp\\\\\\\/2b2l0E5sVg1uDHWkErt8J1Y99X86fp\\\\\\\/nEvpcj6Hul9bRC7rp39KmB24yn9376nFn1HzFzpIxcgEXL8+9uM4m2uO7u3T79sxSvTRcURWTY+TNaJEdN698UUTcBf0PeeJ393yxjO\\\\\\\/Ewyxhy7TX8cIj6OJvCf\\\\\\\/y8Y74KgopeSXe9JMXbs0tmNhIlWJabTPvclYOAwsl1uEwIZFL+VPFrwtEI7ArgAmRIWPxMzi9\\\\\\\/H\\\\\\\/eZqS7+4+sowCMAbOTtgDtRUNl9KoUvtzq+OXaQAguXQyDcwsQu0vGsqI3pd5IhUzRNPP\\\\\\\/zDs9RjZ2i7B6\\\\\\\/yxw9YC9xfUMJZdYNoG6RmbVS7c\\\\\\\/H6G7pLgTGmcUWn3dpAGz\\\\\\\/ytTpd8FJYUQaaTbHzx0uTGdJ+J87AnCKj5lPyiJ6o4WFOId8RM5At5zrea8ApCGaR52RSVXq2QdQAeH3c1f4eVpWWP9Xx+eTMrMkw\\\\\\\/eBxT1jZc18WOYXN7znHN7GIGvtf5C2sNPe8EOW+6JyA19svApUUsE8yu4tOZGykGDxZF+NEzvnNMzXjpuvIOt2DL3eiZCHggXL1O6EKrYHXUrVPRX0IexEuRRi62JiZgndchlhqQAWEf0z4qSlBzhCFIi4sm\\\\\\\/ObqbZTniup7zx6hk3gv3Uym7GAEGz4X5u\\\\\\\/T+uT01bCBlUiHOMN3o6XgYaPjI8cH2WFcMe8TCUpEWuhBTy3iRA1L1Ftjbp1VWaAqjAfMi1JXP7nEdMLgnGZoWfSwr2abB6B0rIAldiGEta3eBW82O9Rp5Pbz8Ahn4teQEG2v0gbjToSxvCS2wbqh8aHUEHK1kyn6qPmQG9gqiO\\\\\\\/e4Me6TER4ProLeRtP1OOq8i4vE+w3ukgkrYdbPctmPdTYXdmgp4y346ybhNkCZyR4cAFw2\\\\\\\/OwMBYAU34jjwKng28tITsxFvGOs6KLwPvqYEBxZtDflFtaeFNXR5mTDIm0ox\\\\\\\/qSzCH9D6ZJdliKfiT9voa0Wc4Xw5czuIBkWsX+Q8ZIrWAeCH+FSHO7bnXyGqLSu8c0EAEUUaFtQOt+nUUbZqxB\\\\\\\/zAygtumbjgVzw\\\\\\\/wgrgN4OnQMgoOYD7HKTZv4Vc5YTz0W+qQ3aH9S3JivKLnpzAdsJ+vplZ1dii+uuGwE\\\\\\\/1WGmaNHN7FJEa\\\\\\\/Rju01wfa0QRLwyFnZz0VP7UxTYXhlyEt6iWQQkzFj3B8vFSyEjpls9gjDKEr5HXxzrErb6s7IXm+U2fazuR9szI1F73ZrNOvdQzpZxL9DL5NaVJ\\\\\\\/rigd7hxIAAmL30x3Pxewqcpa8r\\\\\\\/mxO209HmMxRHPI4AcgJ7MPJJ9ixtfWkGxt0UKxLIo7n4xahjjT5QE+iXbWCpPlsfQT6SFTdAby8heMTV2tV5Blj\\\\\\\/dcTa1KR8RlMl1ux97Ub19JDGPYQkgauMq\\\\\\\/a8O60EJogycrd45a7IW60DEMgPcV1oqDHwfCMusLZwkfd8WzxuZKHJvbi2wIKoeG2udzfTZFD6n3tN5o00o49p1ytqjqxZg6FzyOBG6HcM5TrFi8jIWHO7DVzxb6xJaJdHZ0DLzaOajIcYK+0js9I8ALun9uEP6Q1LWZVs7WLE6t8TVpqqXK+z\\\\\\\/G9ni\\\\\\\/PQmCy1s3shMwkBaFKLnkK8EhbcfWX1xgEY2VQ+gDWFltP3NsPyaqFX5fKUjDdHBwUaX7VLHLg7mANhmqPZih+yPkbQzkVLiPsOI\\\\\\\/LRqKE98XkzK0fh38o6puHHE\\\\\\\/fLgJiDm+qBzHnq4rHuXzhnp02njFzCU89jNT6W46hRnsmCtM7fKWxw9P+v7AT038gBputAymL0lDEWoL9+9AqXeEw3nuJ\\\\\\\/wDg5\\\\\\\/fSEabOpHgijVEiBO69CWfi6gCvPJJLZ9OR2LuuVe6N9cysp8uL7LtFNsjyUpyqZg5eq+rmS\\\\\\\/gelo9VXWP9+Ap3qm3Ts04gUjTDcpsDCPwXZinnBxr9U4yMV+a2gzkZClMCE7IjCo2G8bvqPgmNWgzA88dNpd8KpRnuwKqqjUZmhrz+QddFNIKBxJe3Jswa6J4+rR9CWqBGthZNaLWp9gYqT3fIZt9kksHD9CoD6\\\\\\\/FRGzZg3zADlZbI1Mg4jOGRmpPOoFBEj7E7VbA2RbE9W\\\\\\\/KyRLh+FnlQaetrrljOt7WK31xHi4FcueO4jycaE1Sri3neX6ik73YEEIyyHT2TrzPyaDX+z9AlaZ8zZmfb29NAOkg18a6WRKtOO9U\\\\\\\/0i\\\\\\\/88WJz3aHhRUkSmXKI44B+b46GL4x\\\\\\\/WeekvVPWwPOXXYyomcJDl8dRi5srRBfrDDe9C8IxDNCq\\\\\\\/1gyjHr604Uf4d2esQ2PZl9JdFz6YfLa2rIqYHkeae5PqJx1rcwBkWO30mL5IQ2RRAUerwVmBbMgByS4ZZDPNs7YyBZ4los3l2vvRmgT5Bx6EGspoY7ErVXJbNITAsV8PkLDOENtm7W4Kt7\\\\\\\/it5ZvvpBaYI\\\\\\\/wGhn8ejS+6fod3avK\\\\\\\/LRVEwkFOedlgkJsMmR3pwt2d\\\\\\\/psvCHtIQ51rnThWn7sy1lz6xF5cRedmHvZmt9xj2QJqRRwPi5fgAXfOCIUS7ZR65Jg4ugkZxv621aRoUhQuWYqafRrdGowZi1cEi1C3UqJsQWzo49Cju9rzP3kDjxcEonB\\\\\\\/+d13QZs3rWdlnEV9Pyxllkef33WRF2fBSYN8YrofAfDJKyDSuzB8qYpNu9w6kOTVYhubEqZEk0ZO4sFXdLtOQfeKi+mEYqoUT5Tki9UeyhI0zgTtSI+1dchytQESxHK\\\\\\\/O4IywgU5ovNFn\\\\\\\/mMmu01zsutv4\\\\\\\/hBRdwD8aX52caQSaIzq+bCWhUgMWh5fmQfAOTbBjEnkRMo48hPEeyCTbS2+5f3qM2sg4iAVyk8s8QYUvRLf96LIDaYod3fWp4LsewXu0rJuzOyADbqof\\\\\\\/vP38foO6Rej+\\\\\\\/1IQEAjg2iKD+46e4mn6+GxlNTrUK0\\\\\\\/uHuekAr3ydomshqjbVYvKVpHSIDnp3Gn1E3+9PWqQVIBKNT296d3GqDXylCnHEL1s5G84EOQQHQ+PMRcvX8DdCSMTIdrIo8ugGpi3isQutpvBcvu0UFbhn1zFpj+7kMlCoY4az54MTBn6b51eh+CTUiPPRGr1uoZPzqk8Mjm2uaGFyAt3GMIpMgA2ec2etBJaFhgTp1uWdwE+VXsz3TetXIjscvT5hMwYjJkEsAs8\\\\\\\/4EsKe84vQ7bbXt1TFc5bMbRGgLNhBmeiEAfd10T6EL8mfxGG4mx99fX\\\\\\\/ogV9xajLpCFXidYY22Xb5UgvBfU2zhvl+a+NpBoobu2tv4R\\\\\\\/wszMlVD\\\\\\\/65FDPkxNodVMq5aNuJ8\\\\\\\/H0Do93ElNGTiiHkMPGLa26XxMxZH714ifF0hM5cIEEdUK7w0N2TM6QlpLkpw2uRYdB+TRTGYHRCjtuhu8199AAmBi3DBlQ2R8t07I2QV1RiGbZm4OOXMWqO\\\\\\\/t+N4S5qInP9WmI1uizMxF3\\\\\\\/sVdhVDARMCnDYVgC21\\\\\\\/g+5oxFpg+h\\\\\\\/Ds9sKKN1SkcCSNW6HPKJd+slLZcPVTEUYskjOt01PY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/oLXKKe5PcdECMQUWbcR2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022976e656957596b782ae3e98c31395b702242c3e82966434949591d684543e242\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022re95cA7O4oqOmFomBZETQ3dnM9h\\\\\\\/KODaonwuAUEFPbVQ57k9L+I3F+xeSn6bhSbaFp\\\\\\\/+pmSH59MxH7gSJdWZrCeQBWHCHcwSVh0hQt91HdXfqKoZk5VnFzDnXUJHwhbw9Fe2dN4L\\\\\\\/em6Agur4nkyKfIUVLEu9116l7peiPEYOWX9xnSd2h1vNpTuxZ7LFTO9kai50PsFUH6Tgm1KHHnTzlaDLIk4q4FbLwO5535t33YltTOwme4K99QWjVV7MrV\\\\\\\/y0tjG06EwQMVY2XrD90lYNBeag4\\\\\\\/BMknLgQE5GJDvYQtfv6rKkrTOo3BPInF6m4gdz8BqKL2ioYDo16cSWHp\\\\\\\/JS0MV9EE8kmE+ZpfiG5xskEQPSnAp6H3eoovYkCDBwK8ikewK7ZpcmGFYtmSCoYHTQpgLzQLFKx2JR0zgLUwOFO3sXFjWgopEb3LyR\\\\\\\/OrrxrBxVrfCh\\\\\\\/gu+2PAuYsvuRuZZMMtr\\\\\\\/9CcZdsxwfTXm4LtC4q\\\\\\\/ahcKZS6vNebLiwsOvcaYCGG1GgeN6MzsR3Qfcx6J89tLr2WM7NrcN3Ew9dErWNjUj42LwJVdvn3If79fxvcSXUKJWRJQT6l2Zk+SrOilZgCfaP5T8uc6s0PgB59Rq4zRWni\\\\\\\/iifIOpp176cVjSW6Ow6dqFds1APk+Mqqru7PwIenBjpQ6eje9LfAMFBRoCMBnFofLsRDaJOBZMpgrC9kPTtGJRUbHkAPyab6Nv84oaQLqF4KFsKbCDFfleKdV9A5+jwbLHeNP2aero2uuTQ8qWsg5qn6kCWFZJXIyN3BGwucPwC1wmoJaQfdT7pBaYvrsVaezg4ZVYYYa6N4dcvSObFrLTo2+fnWz1vfL0l9lz\\\\\\\/cMSY7ov7ljrore5jF7oDzDVEvLMG1sAZUIH9OgO6pBQKHIQUQfEGcXcfI8a1jU93EsJjBK0SgUEfigvRyJC28U\\\\\\\/dK4vhdhI\\\\\\\/\\\\\\\/h89V2x2owFXBTVc9qV0F1RstugWMy6+ANtcRHc\\\\\\\/TlaKXUosGUsxUyeaC\\\\\\\/yQ\\\\\\\/e+FRlT5s1UUyNLp8Y735sTGZsd9iE0wvH2SRI4CeYgzMF0GwZXzpjEBC5kuAgR4ebn1b8V4nJ9BHJvIAICfJBXEBpjKfeIjQQZ5qgmTlonjeRNzo3+rEofUJqRRs5Qv4HpIT1YmKZ3bvAaPAuJ55Y1t5APMXZasqMgp8yfwR+22CbrosQFHHqiCMiKHMlpmmXSvSuyM8twUOoUDyVFGE7QDM9KOxEKfbhOkcIHK9Dguz1Q+VAFWDDd4N3jIV7HUeWo9EiXG56f0vkIAst2u1y9vPEr8Cxcy23TRZEiPsw2WBYSKE\\\\\\\/Nw08AnSwxCzbb8fX15qdhA2+QXLHBM+GxEO+UitG1FnKEUgAK\\\\\\\/VmCDfaRyacbtX6HcZa+ldEqG7uzIceuaV8tORAkYdmJPayYOnuDwQ62fg4CuivYUTQ5555WXEADr1xbHBNwDK5i8RQ22ZM8q\\\\\\\/nitE2oTzJUGnxFDqILupuNyDLAgxfUimrxvADoKgZBs1cwtmzvTiyhFmrlis8G3Q1ux5aRLMvo9NAAOYHxXtpT0ZKlUxS2KgaMunPdIDGY8rzOh7kx2LBDPblRv9ay7Gl6ltj8GOYDAXq8RvbDZyIA\\\\\\\/STxkQ9S8V3ZYWtdRUTrZZ\\\\\\\/YH8BWrSmlMFOMLPEWOjz2Ml+G9Zpnk4xJ0gnRl1fcgh9aAFsANDsU\\\\\\\/OZKvxH2HxzixJVKs7PFyTPh9q8ECRTcoEdgeijY\\\\\\\/4C2XMtWriqE0M4Z\\\\\\\/RR5Gr4orz1to7+VREwMpeBDwoW1S9FGTa+gjoRFTAqY\\\\\\\/8TR2yDtUJ9HvJnOumHh2RdhOz9bLOgyb7bdjgf0UQjXXC+4tqU6oWEvQEEgXKkM4dF4HqU8qrh1mNBMJPyP91rtZzMlH21UuMBD3wTD3fkDemOWwa59lu1GJKGu\\\\\\\/LZDgPNjPJqekbHliDXIUeTRZ9WymaGFB9s0kcsuOannOshx\\\\\\\/clGvs8Z5tsLF4kNWFGUsaYNjQZPebmbn9jNnRx4dhm4NpnnT1ICQ3908KkC3o9oimY\\\\\\\/YHoqpJu7AWRJHRKlL8WuhOKhvqfnzABvmjVcSpOz3NdAxDgtS75s6rBlaFv8l+Ec7WIyY\\\\\\\/Gl1Kwq6dE40IlYmEyrHX2zYSQGEFnw==\\\u0022,\\\u0022iv\\\u0022:\\\u002292mxhVfvTUCw5WecBKe5NQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a48a4e08d0a8169bce140d301b61bb3d172d3b6f11ac6ff85f41d8a5eed3b507\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460500 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3893" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mr61zlyX5Rare-yQP2wSGFy7OOlnlCKOBxpZDxjttVA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3893" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YkVsVNV833maBPn9LMOIeYt0E-VBMS7QU44QjdaK1ek" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3883" title="A Natural Mutation Involving both Pathogenicity and Perithecium Formation in the Fusarium graminearum Species Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3903" title="Autosomal and X-Linked Additive Genetic Variation for Lifespan and Aging: Comparisons Within and Between the Sexes in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460500" id="node460500" data-pisa="ggg;6/12/3893" data-pisa-master="ggg;g3.116.035527" data-apath="/ggg/6/12/3893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elissa J. Chesler</span>, <span class="highwire-citation-author" data-delta="1">Daniel M. Gatti</span>, <span class="highwire-citation-author" data-delta="2">Andrew P. Morgan</span>, <span class="highwire-citation-author" data-delta="3">Marge Strobel</span>, <span class="highwire-citation-author" data-delta="4">Laura Trepanier</span>, <span class="highwire-citation-author" data-delta="5">Denesa Oberbeck</span>, <span class="highwire-citation-author" data-delta="6">Shannon McWeeney</span>, <span class="highwire-citation-author" data-delta="7">Robert Hitzemann</span>, <span class="highwire-citation-author" data-delta="8">Martin Ferris</span>, <span class="highwire-citation-author" data-delta="9">Rachel McMullan</span>, <span class="highwire-citation-author" data-delta="10">Amelia Clayshultle</span>, <span class="highwire-citation-author" data-delta="11">Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="12">Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author" data-delta="13">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3893-3902;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035527 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460500" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marge Strobel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marge%2BStrobel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strobel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarge%2BStrobel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Laura Trepanier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BTrepanier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Trepanier%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BTrepanier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Denesa Oberbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denesa%2BOberbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oberbeck%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenesa%2BOberbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shannon McWeeney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BMcWeeney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McWeeney%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BMcWeeney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Robert Hitzemann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BHitzemann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hitzemann%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BHitzemann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Martin Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rachel McMullan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BMcMullan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMullan%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BMcMullan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Amelia Clayshultle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amelia%2BClayshultle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clayshultle%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmelia%2BClayshultle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Villena%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/12/3893" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460500" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3893.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460500" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3893.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460500" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/12/3893.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460500" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/12/3893.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460500" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multi-parent populations (MPPs) capture and maintain the genetic diversity from multiple inbred founder strains to provide a resource for high-resolution genetic mapping through the accumulation of recombination events over many generations. Breeding designs that maintain a large effective population size with randomized assignment of breeders at each generation can minimize the impact of selection, inbreeding, and genetic drift on allele frequencies. Small deviations from expected allele frequencies will have little effect on the power and precision of genetic analysis, but a major distortion could result in reduced power and loss of important functional alleles. We detected strong transmission ratio distortion in the Diversity Outbred (DO) mouse population on chromosome 2, caused by meiotic drive favoring transmission of the WSB/EiJ allele at the <em>R2d2</em> locus. The distorted region harbors thousands of polymorphisms derived from the seven non-WSB founder strains and many of these would be lost if the sweep was allowed to continue. To ensure the utility of the DO population to study genetic variation on chromosome 2, we performed an artificial selection against WSB/EiJ alleles at the <em>R2d2</em> locus. Here, we report that we have purged the WSB/EiJ allele from the drive locus while preserving WSB/EiJ alleles in the flanking regions. We observed minimal disruption to allele frequencies across the rest of the autosomal genome. However, there was a shift in haplotype frequencies of the mitochondrial genome and an increase in the rate of an unusual sex chromosome aneuploidy. The DO population has been restored to genome-wide utility for genetic analysis, but our experience underscores that vigilant monitoring of similar genetic resource populations is needed to ensure their long-term utility.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transmission-ratio-distortion" class="hw-term hw-article-keyword hw-article-keyword-transmission-ratio-distortion" rel="nofollow">transmission ratio distortion</a></li><li class="kwd"><a href="/keyword/meiotic-drive" class="hw-term hw-article-keyword hw-article-keyword-meiotic-drive" rel="nofollow">meiotic drive</a></li><li class="kwd"><a href="/keyword/aneuploidy" class="hw-term hw-article-keyword hw-article-keyword-aneuploidy" rel="nofollow">aneuploidy</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations-mpp" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations-mpp" rel="nofollow">multiparental populations MPP</a></li></ul><p id="p-3">The power of genetic mapping studies in model organism populations derives, in large part, from uniform and high allele frequencies at all variant loci across the genome. Multi-parent populations (MPPs) such as the Diversity Outbred (DO) mouse population provide high mapping precision due to the accumulation of recombination events across multiple breeding generations. It is important to maintain allelic balance during the breeding process and this can be achieved by maintaining a large effective population size with randomized matings (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Rockman and Kruglyak 2008</a>). The founding generation (G0) of the DO population consisted of randomly chosen mice from the incipient Collaborative Cross (CC) breeding lines, which were derived from eight inbred founder strains (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Svenson <em>et al.</em> 2012</a>). Software-assisted breeding has facilitated adherence to the randomized breeding design of the DO. However, natural phenomena such as allelic incompatibility between loci or meiotic drive have the potential to directionally disrupt allelic balance more rapidly than expected from stochastic genetic drift, which would require hundreds of generations to substantially alter allele frequencies in this population.</p><div class="section" id="sec-1"><div id="sec-2" class="subsection"><h3>Discovery of transmission ratio distortion (TRD) in the DO</h3><p id="p-4">Several thousand DO mice have been genotyped across multiple studies spanning from generations G4 through G21. Data sharing among the DO user community enabled us to monitor the autosomal and sex chromosomal haplotype distributions over time.</p><p id="p-5">In generations G8 and G9, we observed a substantial and growing distortion of allele frequencies on chromosome 2. An excess of WSB/EiJ alleles had been previously noted in the CC lines, where it appeared to have stabilized at a frequency of ∼0.20 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Collaborative Cross Consortium 2012</a>). However, the frequency of WSB/EiJ alleles in the DO continued to rise and, by G12, exceeded 0.60 (more than five times the expected value of 0.125). The cause for this rapid sweep was identified as a novel meiotic drive locus named <em>R2d2</em> (<em>responder to meiotic drive on chromosome 2</em>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Didion <em>et al.</em> 2015</a>). <em>R2d2</em> is a copy number variant; in a permissive genetic background, alleles with high copy number (including WSB/EiJ) are subject to TRD through the maternal germline due to meiotic drive. It appeared likely that, without some intervention, the WSB/EiJ allele would sweep to fixation and as a result genetic variation in the DO would be depleted across a large portion of chromosome 2. Further support for this conclusion was provided by a similar distortion in allele frequency observed in the region spanning the <em>R2d2</em> locus in a related outbred population known as the Heterogeneous Stock-Collaborative Cross (HS-CC), which has undergone many additional generations of outbreeding (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FigureS1.png">Figure S1</a>) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Iancu <em>et al.</em> 2010</a>).</p><p id="p-6">The fixation of a single haplotype across a large genomic region in a mapping population would create a blind spot with no detectable genetic variation. To date, over 145 quantitative trait loci (QTL) have been mapped to genetic variants on chromosome 2 [(URL: <a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>), 2/25/2016] and 730 features with phenotypic alleles [URL: <a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>), 2/25/2016] are localized to this chromosome. These include protein coding genes, chromosomal deletions, miRNA genes, antisense lncRNA, chromosomal duplications, chromosomal inversions, endogenous retroviral sequences, insertions, and lincRNA genes. Fixation of a single haplotype would eliminate our ability to detect the effects of variants in this region on complex traits, and would mask the region in expression QTL and other systems used in genetic analyses. Epistatic interactions involving loci on chromosome 2 would be limited to the WSB/EiJ haplotype, thereby either masking or exaggerating their effects in the DO population.</p></div><div id="sec-3" class="subsection"><h3>Remedial strategies</h3><p id="p-7">In order to maintain allelic variation on chromosome 2, we decided to intervene in the meiotic drive process. We sought an intervention that would deviate as little as possible from the original breeding strategy, in which two randomly selected progeny (one female and one male) from each of 175 DO lineages are assigned to new mating pairs at random with avoidance of paired siblings. Due to increasing demand for DO mice starting in generation G8, the randomized assignments were being made in duplicate such that two males and two females were drawn from each litter and assigned to mating groups “A” and “B.” Progeny from 175 mated pairs in the “A” group were selected to maintain the core DO population and progeny from mated pairs in the “B” group were used primarily for distribution. The availability of two mated pairs per lineage expanded the available pool of matings and allowed us to reduce the frequency of WSB/EiJ alleles, while avoiding a bottleneck in the core breeding population.</p><p id="p-8">Several remedial strategies could be employed to mitigate the impact of meiotic drive on genetic variation. One strategy was to fix the central region of chromosome 2 with the WSB/EiJ-derived haplotype leaving the flanking regions segregating for all eight founder haplotypes. However, this strategy would leave a large “blind spot” in the middle of the chromosome due to reduced recombination in that region (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Morgan <em>et al.</em> 2016b</a>). An alternative strategy was to purge the WSB/EiJ allele, preserving genetic variation along the entire chromosome but with seven, rather than eight, founder haplotypes represented in the central region. We concluded that retaining as much segregating variation as possible was the preferred solution. We speculated that the latter strategy could be augmented by the reintroduction of a WSB/EiJ haplotype carrying a spontaneous copy number reduction at <em>R2d2</em> that would be incapable of drive (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Didion <em>et al.</em> 2015</a>).</p><p id="p-9">The allelic distortion on chromosome 2 was first detected in genotypes from generations G8 and G9, but by the time these data were analyzed, the G12 matings had been established and WSB/EiJ allele frequency had exceeded 0.60 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Didion <em>et al.</em> 2016</a>). Based on the observed rate of change, we determined that it was still possible to purge the WSB/EiJ allele at <em>R2d2</em> while maintaining the essential characteristics of the DO population—random assortment and balanced allele frequencies—across the uninvolved regions of the genome. The need to monitor the progress of the purge and its potential impact on other regions of the genome lead us to the discovery of additional irregularities in the genetic makeup of the mitochondrial genome and sex chromosomes in the DO population, which we describe below.</p></div></div><div class="section materials-methods" id="sec-4"><h2 class="">Materials and Methods</h2><div id="sec-5" class="subsection"><h3>DO production colony maintenance</h3><p id="p-10">The DO mouse production colony is maintained at The Jackson Laboratory in a standard barrier, specific pathogen-free facility. Breeder pairs were housed in individual pens of duplex cages [pen dimensions 12″ × 6″ × 5″ (L × W × H)] on pressurized, individually ventilated racks with hardwood chip/shaving bedding. The mice were fed a Lab Diet 5K0Q (St. Louis, MO) <em>ad libitum</em> and were provided with filtered water in bottles acidified to pH 2.5–3.0. The room temperature was maintained at 70° (± 2°) and 50% humidity (± 20%), with a light cycle of 14 hr on and 10 hr off. The pups were weaned at 3 wk of age and housed in duplex cages with up to five sex-segregated animals per duplex pen. The core colony is maintained in 175 lineages. At each new generation, two progeny (one female, one male) are selected at random from each lineage and assigned to mating pairs at random with avoidance of sib-mating. These progeny are selected from first litters when possible and additional litters are used to provide mice for distribution. Due to an increase in demand at generation G8, we established a second mated pair within each lineage. Progeny from the “B” matings were primarily used for distribution. Breeding records are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS1.xlsx">File S1</a>.</p><p id="p-11">Many users of DO mice provided their genotype data for haplotype inference using the DOQTL software (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gatti <em>et al.</em> 2014</a>). These samples were genotyped on one of the MUGA, MegaMUGA, or GigaMUGA (Neogen, Lincoln, NE) array platforms (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Morgan <em>et al.</em> 2016a</a>). Genotype data from DO mice are being archived at <a href="http://do.jax.org">http://do.jax.org</a>, and will also be accessible from the Mouse Phenome Database (<a href="http://mpd.jax.org">http://mpd.jax.org</a>). We strongly encourage users of DO mice to contact E.J.C. or G.A.C. to coordinate submission of genotyping data.</p><p id="p-12">The HS-CC population was bred at Oregon Health &amp; Science University (OSHU) in the research colony of Robert Hitzemann. The HS-CC were formed from the eight CC founder strains using a pseudorandom breeding design (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Iancu <em>et al.</em> 2010</a>). The colony is maintained as 48 families using a rotational breeding strategy, <em>i.e.</em>, a male from family one is bred to a female from family two and so on. The colony is currently at G35. The HS-CC mice are housed at the Portland Veterans Affairs Medical Center (VAMC), in a nonbarrier facility under standard conditions. At G25, 88 individuals were genotyped using the MegaMUGA (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Morgan <em>et al.</em> 2016a</a>).</p></div><div id="sec-6" class="subsection"><h3>Marker-assisted purge of WSB/EiJ alleles at the R2d2 locus</h3><p id="p-13">To perform the breeding intervention in a cost-effective manner, all mating pairs were genotyped for the presence of WSB/EiJ alleles on chromosome 2 at three SNPs: rs27943666, rs28048346, and rs28030588. Primer sequences are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS2.xlsx">File S2</a>. Genotyping was performed by LGC Genomics (Beverly, MA). DNA was isolated using sodium hydroxide extraction followed by neutralization with Tris HCl. The genotype analysis enabled us to categorize the genotypes of each potential breeder as WSB/EiJ (W) carrier status, or all other alleles (a).</p><p id="p-14">Tracking of DO matings is matrilineal. Selected DO mice from generation <em>N</em>-1 are assigned to a mate pair with an identifier of the form GN0xxx, where <em>N</em> is the generation of the expected offspring and xxx is the lineage number of the dam. The sire is selected at random (with avoidance of siblings) and his maternal lineage is noted in the breeding records. Due to the nature of the production environment, there are some gaps and minor recording errors in the breeding records. Beginning with matings that produced the G8 generation, matings were set up in duplicate. The dam and sire of the A mating are full siblings of the dam and sire, respectively, in the B mating. Initially the A mating was designated for propagating the DO population and the B matings were used to expand production capacity. However, in the event that the A mating did not produce sufficient numbers of female and male offspring for the next generation, the B mating was available as a backup. In the event that neither the A nor the B mating produced a full set of offspring, another lineage was selected to provide either the dam, the sire, or both parents for the next generation matings. In the ideal, these replacements would not occur and drift in the population would achieve its theoretical minimum value.</p><p id="p-15">In order to execute the purge of WSB/EiJ alleles at <em>R2d2</em>, we deviated from this mating scheme in two ways.</p><p id="p-16">First, both the A and B mate pairs were genotyped. Four of the nine mating types (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) can produce at least some mice that do not carry a W allele, including progeny of the aa × aa matings, Wa × Wa matings, Wa × aa, and aa × Wa matings. Of these, only offspring of crosses with a Wa parent need to be genotyped to identify aa progeny. Progeny of the aa × WW and WW × aa matings are all Wa and do not need to be typed. Progeny of the WW × Wa and Wa × WW crosses are either WW or Wa and can be identified by genotyping.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469896/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469896/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Frequencies of the available mating types in the DO population classified by the presence (W) or absence (a) of the WSB/EiJ allele at the start of the intervention, generation G12</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">Second, to populate the next generation, offspring of all but the WW × WW mating types were retained, but progeny were typed as needed and those with either one or zero W alleles were selected for the next generation. To minimize the W allele frequency in the subsequent generation, only a fraction of the Wa × WW/WW × Wa progeny were retained. By prioritizing those mated pairs that reduced or retained W alleles, and by selecting specifically those that carried the minimum number of alleles, we expected to bring the allele frequency from 62 to &lt; 15% in one generation, and theoretically to purge the W allele completely in the next generation.</p><p id="p-18">The meiotic drive effect was originally reported to result in ∼66% transmission of W alleles from Wa matings based on population estimates. Further work revealed that selection at <em>R2d2</em> occurs only through the female germline and is background dependent, with transmission ratio varying from 50 to 100% in favor of the W allele depending on the mating (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Didion <em>et al.</em> 2015</a>). Therefore, marker-assisted breeding and selection required multiple generations and progressed more slowly than our original estimates.</p></div><div id="sec-7" class="subsection"><h3>Haplotype reconstruction</h3><p id="p-19">Two litters are consistently produced in both of the production and distribution colonies. Litter sizes and sex ratios were determined from breeding records of these colonies. Analyses are based on data separated by litter.</p><p id="p-20">We used the allele calls from the MegaMUGA and GigaMUGA platforms (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Morgan <em>et al.</em> 2016a</a>) as inputs to a hidden Markov model (HMM) and performed haplotype reconstruction in each DO mouse (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gatti <em>et al.</em> 2014</a>). Briefly, we estimated the posterior probability that each mouse was in one of 36 possible unphased diplotype states, given the allele call data. The HMM produces 36 diplotype probabilities (which sum to one) for each mouse at each marker. We estimated the frequency of each founder allele along the genome by condensing the 36 diplotype probabilities to eight founder haplotype probabilities and summing these across samples.</p><p id="p-21">Haplogroup assignment for the Y chromosome and mitochondrial genome was performed using linear discriminant analysis (LDA) applied to genotypes from the MegaMUGA and GigaMUGA arrays separately. Based on data from the Mouse Diversity Array (MDA) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Yang <em>et al.</em> 2011</a>), we determined that we could discriminate between six Y-chromosome haplogroups (“A” = A/J; “BCE” = C57BL/6J, 129S1/SvImJ, NZO/HlLtJ; “D” = NOD/ShiLtJ; “F” = CAST/EiJ; “G” = PWK/PhJ; and “H” = WSB/EiJ) and five mitochondrial haplogroups (“ABCD” = A/J, C57BL/6J, 129S1/SvImJ, NOD/ShiLtJ; “E” = NZO/HlLtJ; “F” = CAST/EiJ; “G” = PWK/PhJ; and “H” = WSB/EiJ) in the DO. For the Y chromosome, we trained a classifier on data from 86 (MegaMUGA) or 66 (GigaMUGA) inbred and F1 males with known Y chromosomes. We retained only genotypes at a set of seven high-confidence probes (JAX00725028, JAX00725070, JAX00725071, JAX00725036, JAX00725040, JAX00725087, and JAX00725083) carried over from the MDA and shared on both platforms. The classifier was applied to all male DO samples jointly using the predict.lda() function from the R package MASS (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Venables and Ripley 2002</a>). For the mitochondrial genome, we used a similar procedure, using the same samples for training but retaining all 46 probes (MegaMUGA) or 32 probes (GigaMUGA) mapping to the mitochondrial genome. Haplogroup assignment routines are implemented in the <em>R</em> package “argyle” (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Morgan 2016</a>).</p></div><div id="sec-8" class="subsection"><h3>Screen for R2d2 mutants and estimation of transmission ratio</h3><p id="p-22">The <em>R2d2</em> copy number was estimated in 71 DO G16 females carrying the WSB/EiJ allele at <em>R2d2</em> in heterozygosity using two copy number assays (Life Technologies, Carlsbad, CA, catalog numbers Mm00644079_cn and Mm00053048_cn). These assays target the <em>Cwc22</em> gene that is present at the <em>R2d2</em> locus (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Didion <em>et al.</em> 2015</a>). The female with the smallest copy number (DO-G16-107) was backcrossed for three generations to C57BL/6NJ males, selecting at each generation for heterozygous <em>R2d2</em> females. TRD was tested in the progeny of nine females from this pedigree (the original DO G16, one F<sub>1</sub>, three N<sub>2</sub>, and four N<sub>3</sub>) by crossing them to C57BL/6NJ males and genotyping 18–60 pups. DNA for PCR-based genotyping was performed on crude whole genomic DNA extracted by heating tissue in 100 μl of 25 mM NaOH/0.2 mM EDTA at 95° for 60 min followed by the addition of 100 μl of 40 mM Tris-HCl. The samples were then spun at 2000 rpm for 10 min and the supernatant collected for use as PCR template. The following primers were used in this study: Chr2-TRD-85.65-F, 5’-AGC CTA TAG GAT TGT GTT CTA ACC-3’, position in GRCm37 85657874 and Chr2-TRD-85.65-R, 5’-ACA GCC ATG TAC TAA TCT AAA CCT-3’, position in build GRCm37 85658306. PCR reactions contained 1.5–2 mM MgCl<sub>2</sub>, 0.2–0.25 mM dNTPs, 0.2–1.8 μM of each primer, and 0.5–1 units of GoTaq polymerase (Promega) in a final volume of 10–50 μl. Cycling conditions were 95°, 2 min; 35 cycles at 95, 55 and 72° for 30 sec each; with a final extension at 72°, 7 min. PCR products were loaded into a 2% agarose gel and run at 200 V for 40 min. Genotypes were scored, recorded, and we used a chi-square test to detect significant deviations from Mendelian expectations favoring the WSB/EiJ allele.</p></div><div id="sec-9" class="subsection"><h3>Sex chromosome abnormalities</h3><p id="p-23">DO mice were classified as males or females based on the mean hybridization intensity at markers located on the X and Y chromosomes; females have low signal from Y-linked probes and higher intensity at X-linked probes than males. XO females were identified by the lack of a Y chromosome, significantly reduced overall signal intensity for markers on the entire X chromosome, and by the complete lack of heterozygosity on chromosome X. Males with partial duplication of the distal X chromosome were identified by the presence of a Y chromosome and the presence of heterozygous calls on the distal portion of the X chromosome. To rule out the possibility that heterozygous calls on the X chromosome in males were technical artifacts, we used only markers with robust performance in females. We also confirmed that males with putative duplications have female-like hybridization intensity at heterozygous markers on the distal X, consistent with the presence of two X chromosomes.</p></div><div id="sec-10" class="subsection"><h3>Genome annotation</h3><p id="p-24">All genome coordinates are on GRCm38. SNPs in the eight DO founder strains were obtained from the Sanger Mouse Genomes Project release REL-1505 (version 5) files at <a href="ftp://ftp-mouse.sanger.ac.uk/REL-1505-SNPs_Indels/mgp.v5.merged.snps_all.dbSNP142.vcf.gz">ftp://ftp-mouse.sanger.ac.uk/REL-1505-SNPs_Indels/mgp.v5.merged.snps_all.dbSNP142.vcf.gz</a> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Keane <em>et al.</em> 2011</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-25">Breeding records are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS1.xlsx">File S1</a>. A complete sample list is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS5.xlsx">File S5</a>. Genotype data are available at <a href="http://churchill-lab.jax.org/website/Chesler_2016_DO">http://churchill-lab.jax.org/website/Chesler_2016_DO</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Discovery and remediation of the chromosome 2 selfish sweep</h3><p id="p-26">Monitoring of DO mouse genotypes from multiple experiments revealed an increasing frequency of WSB/EiJ alleles in a region of chromosome 2 centered at ∼90 Mb (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Although the effect was first noticed in data from generations G8 and G9, the G12 matings were already in place and the WSB/EiJ allele frequency had reached 60% before we could begin intervention. Animals that were distributed from the DO colony through G19 retained a high but steadily decreasing frequency of WSB/EiJ alleles (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Marker-assisted selection of progeny was used to establish matings in the core DO colony beginning at generation G13, and WSB/EiJ allele frequency declined rapidly for the next five generations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, C and D</a>). By G21, WSB/EiJ allele frequencies among breeder pairs dropped to zero and the WSB/EiJ allele had been completely purged from the DO at generation G22.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Progression of the selective sweep and purge of the WSB allele at the R2d2 locus. (A) The WSB allele frequency at the R2d2 locus in distributed DO mice increased through generation 12. (B) In generations 13 through 21, the WSB/EiJ allele frequency in distributed DO mice decreased as a result of intervention in the breeding design. (C) The WSB/EiJ allele frequency at selection markers flanking the R2d2 locus shows a steady decline among breeders from generations 12&#x2013;22. (D) WSB/EiJ allele frequency at the R2d2 locus in distributed DO mice is show in profile across all generations with genotype information. Colors changing to darker shades indicate forward progression through time. Chr, chromosome; DO, Diversity Outbred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1791450989" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Progression of the selective sweep and purge of the WSB allele at the &lt;em&gt;R2d2&lt;/em&gt; locus. (A) The WSB allele frequency at the &lt;em&gt;R2d2&lt;/em&gt; locus in distributed DO mice increased through generation 12. (B) In generations 13 through 21, the WSB/EiJ allele frequency in distributed DO mice decreased as a result of intervention in the breeding design. (C) The WSB/EiJ allele frequency at selection markers flanking the &lt;em&gt;R2d2&lt;/em&gt; locus shows a steady decline among breeders from generations 12&#x2013;22. (D) WSB/EiJ allele frequency at the &lt;em&gt;R2d2&lt;/em&gt; locus in distributed DO mice is show in profile across all generations with genotype information. Colors changing to darker shades indicate forward progression through time. Chr, chromosome; DO, Diversity Outbred.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3893/F1.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/12/3893/F1.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Progression of the selective sweep and purge of the WSB allele at the <em>R2d2</em> locus. (A) The WSB allele frequency at the <em>R2d2</em> locus in distributed DO mice increased through generation 12. (B) In generations 13 through 21, the WSB/EiJ allele frequency in distributed DO mice decreased as a result of intervention in the breeding design. (C) The WSB/EiJ allele frequency at selection markers flanking the <em>R2d2</em> locus shows a steady decline among breeders from generations 12–22. (D) WSB/EiJ allele frequency at the <em>R2d2</em> locus in distributed DO mice is show in profile across all generations with genotype information. Colors changing to darker shades indicate forward progression through time. Chr, chromosome; DO, Diversity Outbred.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The finding of a G16 DO mouse with a spontaneous mutation on the WSB/EiJ haplotype that reduced copy number at <em>R2d2</em> raised the possibility of reintroducing the WSB/EiJ haplotype to the selected region on chromosome 2. However, after comprehensive testing of maternal transmission ratio at <em>R2d2</em> in a pedigree segregating for the low copy-number allele, we concluded that the allele was still able to sustain meiotic drive (overall 159 progeny inherited the WSB/EiJ allele while 108 inherit the non-WSB/EiJ allele, <em>P</em> = 0.0018). Therefore, we suspended the plan to reintroduce the WSB/EiJ haplotype with the mutated <em>R2d2</em> allele.</p></div><div id="sec-14" class="subsection"><h3>Effect of selection on the genetic structure of the DO</h3><p id="p-29">In order to understand the impact of selection against the WSB/EiJ haplotype in the region spanning <em>R2d2</em>, we examined the frequency of all eight founder haplotypes across the genome, paying particular attention to chromosome 2 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We chose to contrast haplotype frequencies between distributed animals from generation G21 (which at the time of writing included genome-wide profiles of 504 mice) and generation G11 mice (one generation before the start of the purge for which we have access to genome-wide profiling for 879 mice). In the targeted region of chromosome 2, the decrease in WSB/EiJ allele frequency was offset by increases in most of the remaining founder haplotypes. The highest frequencies are associated with the 129S1/SvImJ (24.4%) and C57BL/6J (29.6%) haplotypes. The wild-derived haplotypes increased from 9.8 to 13.9% for CAST and 5.8–14.9% for PWK. There is an excess of the WSB haplotype throughout most of the nonselected portion of chromosome 2 due to linkage with <em>R2d2</em>, with a peak of 31.9% proximal to <em>R2d2</em>. The remaining seven founder haplotypes are not markedly changed and there were no regions with complete loss of any founder haplotype outside of the <em>R2d2</em> region. We expect that these imbalances will persist in the DO colony with some allowance for genetic drift.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Founder haplotype frequencies across chromosome 2. Each panel shows the founder haplotype frequency at generations G11 (dashed color) and G21 (solid color) for one of the eight founder strains. The dashed gray line indicates the expected allele frequency of 0.125." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1791450989" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Founder haplotype frequencies across chromosome 2. Each panel shows the founder haplotype frequency at generations G11 (dashed color) and G21 (solid color) for one of the eight founder strains. The dashed gray line indicates the expected allele frequency of 0.125.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3893/F2.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/12/3893/F2.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469890" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Founder haplotype frequencies across chromosome 2. Each panel shows the founder haplotype frequency at generations G11 (dashed color) and G21 (solid color) for one of the eight founder strains. The dashed gray line indicates the expected allele frequency of 0.125.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Genome-wide haplotype frequencies did not change substantially from generation G11 to G21 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS3.pdf">File S3</a>). The largest observed change in allele frequencies outside of chromosome 2 was an increase in the NOD haplotype from 13.6 to 33.5% over a region spanning from 90 to 100 Mb on chromosome 15.</p><p id="p-32">The eight founder strains of the DO differ substantially in their relative contribution of unique sequence variants, with the largest contribution of variants coming from CAST/EiJ and PWK/PhJ, followed by WSB/EiJ. Thus, haplotype frequencies alone give an incomplete picture of the changes in standing genetic variation following the purge of the WSB/EiJ haplotype (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The central region targeted by the marker-assisted purge contains 2772 SNP and 884 small indel variants that are private to WSB/EiJ. The flanking regions contain an additional 74,265 and 22,618 WSB private SNPs and indels, respectively. These alleles are lost or nearly lost in the current DO population. The distribution of minor allele frequencies for the majority of SNPs on chromosome 2 not private to WSB/EiJ remained stable throughout the selection process (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS4.pdf">File S4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Information content of biallelic SNPs on chromosome 2. The Shannon entropy (log2) of individual SNPs on chromosome 2 is plotted by chromosomal position. The maximum value (1.0) is achieved for SNPs with a minor allele frequency of 1/2. Shannon entropy is plotted for all SNPs at generations G11 (A) and G21 (C), and for SNPs that are private to WSB (B and D). A smoothing spline (red) indicates average variation in information content across chromosome 2. Chr, chromosome; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1791450989" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Information content of biallelic SNPs on chromosome 2. The Shannon entropy (log&lt;sub&gt;2&lt;/sub&gt;) of individual SNPs on chromosome 2 is plotted by chromosomal position. The maximum value (1.0) is achieved for SNPs with a minor allele frequency of 1/2. Shannon entropy is plotted for all SNPs at generations G11 (A) and G21 (C), and for SNPs that are private to WSB (B and D). A smoothing spline (red) indicates average variation in information content across chromosome 2. Chr, chromosome; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3893/F3.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/12/3893/F3.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469953" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Information content of biallelic SNPs on chromosome 2. The Shannon entropy (log<sub>2</sub>) of individual SNPs on chromosome 2 is plotted by chromosomal position. The maximum value (1.0) is achieved for SNPs with a minor allele frequency of 1/2. Shannon entropy is plotted for all SNPs at generations G11 (A) and G21 (C), and for SNPs that are private to WSB (B and D). A smoothing spline (red) indicates average variation in information content across chromosome 2. Chr, chromosome; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Chromosome Y and mitochondria</h3><p id="p-34">To determine whether there are significant changes in allele frequency in the mitochondria and the Y chromosome, and to determine whether putative changes are related to the <em>R2d2</em> purge, we compared the frequencies for G6 through G21. The initial description of the DO population (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Svenson <em>et al.</em> 2012</a>) did not report the haplotype frequencies in the mitochondria and the Y chromosome due to the lack of markers in the genotyping platform used at the time (MUGA). This shortcoming was addressed in newer genotyping arrays (MegaMUGA and GigaMUGA) (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Morgan <em>et al.</em> 2016a</a>). In addition to pre- and postpurge generations, we also calculated the input frequencies based on the genotypes of incipient CC mice used to establish the DO population (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chesler <em>et al.</em> 2008</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Svenson <em>et al.</em> 2012</a>). There is a reduction in the mitochondrial haplogroup ABCD, corresponding to founder strains A/J, C57BL/6J, 129S1/SvImJ, and NOD/ShiLtJ, concomitant with the purge and strongest between G13 and G14 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). For the Y chromosome, there is no evidence of directional changes either before or after the purge (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). We conclude that, despite the changes in mitochondrial haplotype frequency, the DO retains every type of Y chromosome and mitochondria. In fact, at G21, the frequencies of the four genetically distinct mitochondrial haplotypes are more evenly distributed than they were in the founding generations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequencies of mitochondrial (A) and Y-chromosome (B) haplogroups by generation. Size of points indicates the number of mice sampled per generation; ribbons indicate 95% binomial C.I. for the estimated frequencies. Strains A/J, C57BL/6J, 129S1/SvImJ, and NOD/ShiLtJ (&#x201C;ABCD&#x201D;) share the same mitochondrial haplogroup; strains C57BL/6J, 129S1/SvImJ, and NZO/HlLtJ (&#x201C;BCE&#x201D;) have Y-chromosome haplogroups indistinguishable with current genotyping arrays. Input frequencies are based on the breeding structure of the CC lines used to establish the DO population. DO, Diversity Outbred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1791450989" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequencies of mitochondrial (A) and Y-chromosome (B) haplogroups by generation. Size of points indicates the number of mice sampled per generation; ribbons indicate 95% binomial C.I. for the estimated frequencies. Strains A/J, C57BL/6J, 129S1/SvImJ, and NOD/ShiLtJ (&#x201C;ABCD&#x201D;) share the same mitochondrial haplogroup; strains C57BL/6J, 129S1/SvImJ, and NZO/HlLtJ (&#x201C;BCE&#x201D;) have Y-chromosome haplogroups indistinguishable with current genotyping arrays. Input frequencies are based on the breeding structure of the CC lines used to establish the DO population. DO, Diversity Outbred.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3893/F4.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/12/3893/F4.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469818" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Frequencies of mitochondrial (A) and Y-chromosome (B) haplogroups by generation. Size of points indicates the number of mice sampled per generation; ribbons indicate 95% binomial C.I. for the estimated frequencies. Strains A/J, C57BL/6J, 129S1/SvImJ, and NOD/ShiLtJ (“ABCD”) share the same mitochondrial haplogroup; strains C57BL/6J, 129S1/SvImJ, and NZO/HlLtJ (“BCE”) have Y-chromosome haplogroups indistinguishable with current genotyping arrays. Input frequencies are based on the breeding structure of the CC lines used to establish the DO population. DO, Diversity Outbred.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>X chromosome abnormalities</h3><p id="p-36">Examination of the genotypes on the sex chromosomes at G11 and G21 led us to the unexpected finding of DO mice with two types of sex chromosome abnormalities (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Seven XO females (out of 688 total females) are characterized by the absence of heterozygosity and reduced average hybridization intensity of the SNP probes on the X chromosome. In addition, we identified 65 DO males (out of 695 total males) with apparent duplication of the distal region of the X chromosome. The rate of XO females remains constant but there is a significant increase from 4.4% in generation G11 to 15% in G21 (<em>P</em> &lt; 0.0001) in the frequency of males with duplications of the distal X chromosome. We cannot exclude the possibility that some of these duplications may represent translocations from the X to the Y chromosome. The Y chromosome of the CAST/EiJ strain already has an expanded pseudoautosomal region (PAR) due to an X-to-Y translocation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">White <em>et al.</em> 2012</a>). Duplications of the distal X are challenging to identify in the DO due to the presence of this extended PAR in CAST/EiJ (see male 0568 in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) and the fact that the length of the duplicated region appears to vary (compare males M382 and 1172R in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sex-chromosome abnormalities in the DO. (A) LRR on chromosome X for (top to bottom): an XX female, an XO female, and four XY males. Black points indicate heterozygous genotype calls and gray points homozygous calls. Dotted horizontal line indicates mean LRR for each sample excluding the PAR, whose boundary is indicated by the dashed vertical lines. (B) Zoomed-in views of the distal end of chromosome X. Dashed vertical lines indicate the position of the PAR boundary in CAST/EiJ (left) and all other strains (right). Males with duplicated distal regions of the X chromosome, in the bottom two rows, have heterozygous genotypes and elevated hybridization intensity outside of the PAR. (C) Distribution of normalized hybridization intensity on chromosomes X (pink) and Y (blue) clearly differentiates karyotypes XX, XO, and XY. Panels are labeled with sample names and colored by sex-chromosome assignment: pink, XX; gray, XO; and blue, XY. chr, chromosome; DO, Diversity Outbred; LRR, Log-scale normalized hybridization intensity; PAR, pseudoautosomal region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1791450989" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sex-chromosome abnormalities in the DO. (A) LRR on chromosome X for (top to bottom): an XX female, an XO female, and four XY males. Black points indicate heterozygous genotype calls and gray points homozygous calls. Dotted horizontal line indicates mean LRR for each sample excluding the PAR, whose boundary is indicated by the dashed vertical lines. (B) Zoomed-in views of the distal end of chromosome X. Dashed vertical lines indicate the position of the PAR boundary in CAST/EiJ (left) and all other strains (right). Males with duplicated distal regions of the X chromosome, in the bottom two rows, have heterozygous genotypes and elevated hybridization intensity outside of the PAR. (C) Distribution of normalized hybridization intensity on chromosomes X (pink) and Y (blue) clearly differentiates karyotypes XX, XO, and XY. Panels are labeled with sample names and colored by sex-chromosome assignment: pink, XX; gray, XO; and blue, XY. chr, chromosome; DO, Diversity Outbred; LRR, Log-scale normalized hybridization intensity; PAR, pseudoautosomal region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3893/F5.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/12/3893/F5.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469761" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Sex-chromosome abnormalities in the DO. (A) LRR on chromosome X for (top to bottom): an XX female, an XO female, and four XY males. Black points indicate heterozygous genotype calls and gray points homozygous calls. Dotted horizontal line indicates mean LRR for each sample excluding the PAR, whose boundary is indicated by the dashed vertical lines. (B) Zoomed-in views of the distal end of chromosome X. Dashed vertical lines indicate the position of the PAR boundary in CAST/EiJ (left) and all other strains (right). Males with duplicated distal regions of the X chromosome, in the bottom two rows, have heterozygous genotypes and elevated hybridization intensity outside of the PAR. (C) Distribution of normalized hybridization intensity on chromosomes X (pink) and Y (blue) clearly differentiates karyotypes XX, XO, and XY. Panels are labeled with sample names and colored by sex-chromosome assignment: pink, XX; gray, XO; and blue, XY. chr, chromosome; DO, Diversity Outbred; LRR, Log-scale normalized hybridization intensity; PAR, pseudoautosomal region.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469808/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469808/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of DO mice with sex chromosome abnormalities (percent shown in parenthesis)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Litter size, sex ratio, and mating success in the DO</h3><p id="p-38">Analysis of breeding records for more than 5000 litters across 15 generations of DO breeders (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1/FileS1.xlsx">File S1</a>) show little directional change in litter size and sex ratio. However, an effect on litter size becomes apparent when matings are partitioned according to sex and <em>R2d2</em> genotype. As expected, litter size was not found to depend on sire genotype (Litter 1: <em>P</em> = 0.645 and Litter 2: <em>P</em> = 0.536). In contrast, there is a highly significant reduction in the litter size for <em>R2d2</em> heterozygous females (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The effect is present in both first (<em>P</em> = 1.22 × 10<sup>−21</sup>) and second (<em>P</em> = 2.7 × 10<sup>−10</sup>) litters. This result is consistent with previous reports that the level of meiotic drive observed in heterozygous females is positively correlated with a reduction in the average litter size. We conclude that fixation of the non-WSB/EiJ allele has mitigated a potentially deleterious effect of heterozygosity at the <em>R2d2</em> locus on DO maintenance and production.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Litter size distribution among the DO breeders by allele at the R2d2 locus. Wa alleles in dams leads to smaller litter sizes, but not in sires. The three alleles are &#x2018;aa&#x2019; for homozygous non-WSB/EiJ, &#x201C;Wa&#x201D; for heterozygous WSB/EiJ, and &#x201C;WW&#x201D; for homozygous WSB/EiJ. Red dots are the median of each group and black boxes are the interquartile range. P-values are from a one-way ANOVA of litter size vs. genotype. ANOVA, analysis of variance; DO, Diversity Outbred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1791450989" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Litter size distribution among the DO breeders by allele at the &lt;em&gt;R2d2&lt;/em&gt; locus. Wa alleles in dams leads to smaller litter sizes, but not in sires. The three alleles are &#x2018;aa&#x2019; for homozygous non-WSB/EiJ, &#x201C;Wa&#x201D; for heterozygous WSB/EiJ, and &#x201C;WW&#x201D; for homozygous WSB/EiJ. Red dots are the median of each group and black boxes are the interquartile range. &lt;em&gt;P&lt;/em&gt;-values are from a one-way ANOVA of litter size &lt;em&gt;vs.&lt;/em&gt; genotype. ANOVA, analysis of variance; DO, Diversity Outbred.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/12/3893/F6.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/12/3893/F6.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/12/3893/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-39" class="first-child">Litter size distribution among the DO breeders by allele at the <em>R2d2</em> locus. Wa alleles in dams leads to smaller litter sizes, but not in sires. The three alleles are ‘aa’ for homozygous non-WSB/EiJ, “Wa” for heterozygous WSB/EiJ, and “WW” for homozygous WSB/EiJ. Red dots are the median of each group and black boxes are the interquartile range. <em>P</em>-values are from a one-way ANOVA of litter size <em>vs.</em> genotype. ANOVA, analysis of variance; DO, Diversity Outbred.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The proportion of failed matings among the DO breeders varied from 3 to 7% prior and subsequent to the active intervention. However, higher rates of failure, reaching &gt; 10% in G14, were observed during the peak of the selection process. Failed matings result in a disruption of the ideal randomized mating scheme. They reduce the effective population size and represent the only route for selection to affect mitochondrial or Y chromosome allele frequencies.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-41">At 21 generations, the DO mouse population has proven to be a valuable resource for genetic mapping (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Recla <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Smallwood <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Church <em>et al.</em> 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">French <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gu <em>et al.</em> 2016</a>) and systems genetics (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kelly <em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chick <em>et al.</em> 2016</a>). However, an ongoing selective sweep driven by the <em>R2d2</em> locus occurred in the DO breeding colony and threatened to eliminate genetic variation across a large region of chromosome 2. We were able to rescue most of the standing allelic variation in this region using a marker-assisted breeding strategy to purge the WSB/EiJ allele that was responsible for the sweep. Our strategy had minimal impact on independent assortment of unlinked loci, and has maintained allelic variation throughout the genome. The allele purge reversed the naturally occurring process even though WSB/EiJ allele frequencies were quite high at the time marker-based selection was initiated. Five generations were required to completely remove the WSB/EiJ haplotype at <em>R2d2</em> from the breeding colony. As of the current generation (G22), it appears that we have eliminated the meiotic drive allele but we will monitor this locus for several more generations to ensure that the purge was complete.</p><p id="p-42">Although the decision to use a purge strategy retained much of the genetic variation on chromosome 2, loss of variation was inevitable. In addition to the complete loss of private WSB/EiJ alleles in the central region flanked by selection markers, there was a concomitant loss of WSB/EiJ in the nearby flanking regions of the genome. The other seven founder haplotypes, although retained in the region targeted by the purge, no longer occur in the expected ratios due to either drift or inadvertent selection. The most substantial distortion across this region appears to be a reduced frequency of private NOD/ShiLtJ alleles. Earlier detection and reversal of segregation distortion in the DO population might have reduced this impact.</p><p id="p-43">Deviations from idealized scenarios in MPPs are not insurmountable. For example, variation attributable to private WSB/EiJ variants on chromosome 2 is still amenable to study in the context of the CC and DO resources. The availability of a second population of outbred animals derived from the same founder strains, the HS-CC, provides an opportunity to query the impact of WSB/EiJ alleles in the <em>R2d2</em> region. It is interesting to note that the selective sweep in the HS-CC population was not complete by G25. This could reflect the impact of differences in the breeding scheme or reduction in frequency of genetic background effects that amplify drive at <em>R2d2</em>. WSB/EiJ alleles are also present in the CC inbred strains (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Welsh <em>et al.</em> 2012</a>). These resources will enable the identification of WSB/EiJ alleles with important phenotypic effects. Where questions of multi-locus interactions are concerned, strategies including CC × DO crosses, WSB/EiJ × DO crosses, and genome editing to reconstitute lost WSB/EiJ alleles in the DO population could be considered.</p><p id="p-44">We suspected that changes in the WSB/EiJ allele frequency at the <em>R2d2</em> locus could affect litter size or sex ratio. We observed a decrease in litter size among <em>R2d2</em> heterozygous dams and an increase in nonproductive matings during the purge. Given the breeding design of the DO, the haplogroup frequencies for the mitochondria and Y chromosome should not change due to drift. However, selection in favor of particular breeders during the purge as well as against unproductive breeding pairs could have an effect. For the Y chromosome, we observe little effect of the <em>R2d2</em> purge. In contrast, there is a substantial effect on the mitochondria. We speculate that the strong reduction of the ABCD haplogroup frequency, in particular between G13 and G14, was caused by the overrepresentation of this haplotype in females that were either homozygous for the WSB allele at <em>R2d2</em> (and thus excluded from the matings) or had the high TRD phenotypes (and thus contributed fewer breeders to the next generation). We believe that this association was due to chance, as we do not find any evidence for an association between mitochondria haplotype and presence or level of TRD in the DO (data not shown).</p><p id="p-45">We investigated other genomic features that could have been impacted by the purge, with particular focus on the sex chromosomes. It has been previously reported that the boundary of the pseudoautosomal region (PAR) of the sex chromosomes is located 430 kb proximal in the CAST/EiJ strain compared to other laboratory strains (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">White <em>et al.</em> 2012</a>). In other words, this 430 kb interval is now present in both the X and Y chromosomes in CAST/EiJ, so the presence of heterozygosity in that region should be diagnostic for the presence of the CAST/EiJ Y chromosome. The X chromosome duplications in males described here extend further into the X chromosome beyond this “extended PAR” and are not exclusively associated with any specific Y chromosome haplogroup. Whether the duplications are X-linked, Y-linked, or pseudoautosomal, and whether they are associated with XO aneuploidy, is unknown. While the increased frequency of the duplication may be explained in part by the improved sensitivity to detect this type of abnormality with GigaMUGA (G21) as compared to MegaMUGA (G11), it is possible that biology is driving the accumulation of these duplications. The difference in the sizes of the duplicated segments and their prevalence among distributed DO mice ten generations apart suggest that these duplications are not rare in the DO population.</p><p id="p-46">Multi-parent crosses have become increasingly popular for QTL studies (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gnan <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Tsaih <em>et al.</em> 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dell’Acqua <em>et al.</em> 2015</a>) due to the idiosyncratic histories of existing reference populations and the desire to meet ideal properties of uniform allele frequencies, randomized assortment, and recombination. Here, we demonstrate the need for proactive monitoring and potential corrective measures to maintain the utility of populations like the DO. Although we tried to eliminate biases that are typically introduced in conventional breeding programs, this was not entirely possible and an unexpected, naturally occurring selection drove the population structure away from these ideals. In a randomized breeding scheme where synchronization of matings is required, mild selection against late reproductive maturity will also occur. Biological factors including meiotic drive, recombination hot spots, and other phenomena will ultimately dictate the uniformity of allele frequencies and mapping precision of the population. Despite our best efforts to eliminate the most obvious sources of selection bias in breeding, such events are unavoidable.</p><p id="p-47">Our experience has provided several lessons for future efforts to construct multi-parent reference populations. Population monitoring is a crucial aspect of the development of genetic populations. It is essential for designers of MPPs to clearly prioritize resource development relative to the interesting research made possible by the study of undisrupted breeding history. Prior definition of quality metrics by stakeholders, <em>e.g.</em>, acceptable range of segregation distortion, long range linkage disequilibrium, and attrition due to reproductive variability, can facilitate rapid remedial measures to preserve essential population characteristics. In the case of the DO, remediation of segregation distortion to restore the population to its utility as a resource required a deviation from the original breeding plan and the noninterventionist ethos of its stakeholders. All multi-parent genetic reference populations deviate from the ideal in some sense. Selection and TRD are inevitable. However, with careful monitoring and selective interventions, the essential character of the population—balanced allele frequencies and low average kinship—can be retained. What is required for any population is a set of reliable lineage specific genotyping primers for each cross progenitor, routine cost-effective monitoring of cross-sectional genome-wide deviation from expected allele frequencies, and a sufficiently rapid genotyping and breeding decision process to enable marker-assisted interventions. It is possible that other distortions could emerge as DO production continues. We have instituted a routine check on allele frequencies across the population, and will be able to rapidly detect and evaluate the need for future interventions. By carefully monitoring allele frequencies we can detect and reverse unexpected changes and preserve the value of the DO as a premier platform for systems genetics in mammals, and recommend proactive monitoring and intervention in the construction of multi-parent resources.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank Alan Attie, Gillian Beamer, Carol Bult, Leah Rae Donahue, Alison Harrill, David Harrison, Alan Pack, Clarissa Parker, Daniel Pomp, Luanne Peters, and Karen Svenson for generously providing genotyping data. This work was supported in part by National Institutes of Health grants [GM076468 and GM070683 (G.A.C.) and DA037927 (E.J.C.)]; The Jackson Laboratory [U19AI100625-01 (M.F. and F.P.M.V.) and 5F30MH103925 (A.P.M.)], the University of North Carolina [AA13484, AA11034, AA10760, and MH51372 (R.H.)], and Oregon Health &amp; Science University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035527/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-50"><em>Communicating editor: E. Huang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-13"><span class="received-label">Received </span>July 13, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-21"><span class="accepted-label">Accepted </span>September 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 by the Genetics Society of America</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.1" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.2" data-doi="10.1007/s00335-008-9135-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branstetter</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D382%26rft.epage%253D389%26rft.atitle%253DThe%2BCollaborative%2BCross%2Bat%2BOak%2BRidge%2BNational%2BLaboratory%253A%2Bdeveloping%2Ba%2Bpowerful%2Bresource%2Bfor%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9135-8%26rft_id%253Dinfo%253Apmid%252F18716833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9135-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716833&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.3" data-doi="10.1038/nature18270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chick</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttlin</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Defining the consequences of genetic variation on a proteome-wide scale.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">534</span>: <span class="cit-fpage">500</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D500%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18270%26rft_id%253Dinfo%253Apmid%252F27309819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27309819&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.4" data-doi="10.1016/j.fct.2014.11.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sensitivity to hepatotoxicity due to epigallocatechin gallate is affected by genetic background in diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">Food Chem. Toxicol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BChem.%2BToxicol.%26rft.volume%253D76%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fct.2014.11.008%26rft_id%253Dinfo%253Apmid%252F25446466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fct.2014.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25446466&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.5" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collaborative Cross Consortium</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.6" data-doi="10.1186/s13059-015-0716-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dell’Acqua</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattonaro</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppens</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0716-z%26rft_id%253Dinfo%253Apmid%252F26357913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0716-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26357913&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.7" data-doi="10.1371/journal.pgen.1004850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayshulte</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadgary</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A multi-megabase copy number gain causes maternal transmission ratio distortion on mouse chromosome 2.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004850%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004850%26rft_id%253Dinfo%253Apmid%252F25679959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25679959&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.8" data-doi="10.1093/molbev/msw036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadgary</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">R2d2 drives selfish sweeps in the house mouse.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw036%26rft_id%253Dinfo%253Apmid%252F26882987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26882987&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.9" data-doi="10.1289/ehp.1408202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kissling</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity outbred mice identify population-based exposure thresholds and genetic factors that influence benzene-induced genotoxicity.</span> <abbr class="cit-jnl-abbrev">Environ. Health Perspect.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BHealth%2BPerspect.%26rft.volume%253D123%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1289%252Fehp.1408202%26rft_id%253Dinfo%253Apmid%252F25376053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1289/ehp.1408202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25376053&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.10" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft_id%253Dinfo%253Apmid%252F25237114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNjIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.11" data-doi="10.1534/genetics.114.170746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic basis of natural variation in seed size and seed number and their trade-off using Arabidopsis thaliana MAGIC lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1751</span>–<span class="cit-lpage">1758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.170746%26rft_id%253Dinfo%253Apmid%252F25313128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.12" data-doi="10.1534/genetics.115.179481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghupathy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic architectures of quantitative variation in RNA editing pathways.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179481%26rft_id%253Dinfo%253Apmid%252F26614740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNzg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.13" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.14" data-doi="10.1186/1471-2164-11-585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iancu</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darakjian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malmanger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oberbeck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic diversity and striatal gene networks: focus on the heterogeneous stock-collaborative cross (HS-CC) mouse.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DIancu%26rft.auinit1%253DO.%2BD.%26rft.volume%253D11%26rft.spage%253D585%26rft.epage%253D585%26rft.atitle%253DGenetic%2Bdiversity%2Band%2Bstriatal%2Bgene%2Bnetworks%253A%2Bfocus%2Bon%2Bthe%2Bheterogeneous%2Bstock-collaborative%2Bcross%2B%2528HS-CC%2529%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-585%26rft_id%253Dinfo%253Apmid%252F20959017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20959017&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.15" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villena</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomp</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The ‘omics’ of voluntary exercise: systems approaches to a complex phenotype.</span> <abbr class="cit-jnl-abbrev">Trends Endocrinol. Metab.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEndocrinol.%2BMetab.%26rft.volume%253D26%26rft.spage%253D673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.17" data-doi="10.1534/genetics.114.161653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161653%26rft_id%253Dinfo%253Apmid%252F24578350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk3LzEvOTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.18" data-doi="10.1534/g3.115.023739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Argyle: an R package for analysis of illumina genotyping arrays.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.023739%26rft_id%253Dinfo%253Apmid%252F26684930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMi8yODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.19" data-doi="10.1534/g3.115.022087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">The mouse universal genotyping array: from substrains to subspecies.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022087%26rft_id%253Dinfo%253Apmid%252F26684931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMi8yNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.20" data-doi="10.1534/genetics.116.191007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayshulte</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">The evolutionary fates of a large segmental duplication in mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.191007%26rft_id%253Dinfo%253Apmid%252F27371833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzEvMjY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.21" data-doi="10.1007/s00335-014-9508-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Recla</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bult</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Precise genetic mapping and integrative bioinformatics in diversity outbred mice reveals hydin as a novel pain gene.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-014-9508-0%26rft_id%253Dinfo%253Apmid%252F24700285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-014-9508-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24700285&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.22" data-doi="10.1534/genetics.107.083873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Breeding designs for recombinant inbred advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083873%26rft_id%253Dinfo%253Apmid%252F18505881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.23" data-doi="10.1534/g3.114.014704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smallwood</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quizon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution genetic mapping in the diversity outbred mouse population identifies Apobec1 as a candidate gene for atherosclerosis.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2353</span>–<span class="cit-lpage">2363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.014704%26rft_id%253Dinfo%253Apmid%252F25344410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTIvMjM1MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTIvMzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.24" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the mouse diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMi8zODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.25" data-doi="10.1534/genetics.114.162982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsaih</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holl</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaldunski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tschannen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of a novel gene for diabetic traits in rats, mice, and humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D29%26rft.atitle%253DIdentification%2Bof%2Ba%2BNovel%2BGene%2Bfor%2BDiabetic%2BTraits%2Bin%2BRats%252C%2BMice%252C%2Band%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.162982%26rft_id%253Dinfo%253Apmid%252F25236446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk4LzEvMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEyLzM4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-6.12.3893.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venables</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripley</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Modern Applied Statistics with S</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-6.12.3893.27" data-doi="10.1007/s00335-012-9410-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Status and access to the Collaborative Cross population.</span> <span class="cit-source">Mamm. Genome</span> <span class="cit-vol">23</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWelsh%26rft.auinit1%253DC.%2BE.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D706%26rft.epage%253D712%26rft.atitle%253DStatus%2Band%2Baccess%2Bto%2Bthe%2BCollaborative%2BCross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9410-6%26rft_id%253Dinfo%253Apmid%252F22847377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847377&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.28" data-doi="10.1007/s00335-012-9403-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A pronounced evolutionary shift of the pseudoautosomal region boundary in house mice.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWhite%26rft.auinit1%253DM.%2BA.%26rft.volume%253D23%26rft.issue%253D7-8%26rft.spage%253D454%26rft.epage%253D466%26rft.atitle%253DA%2Bpronounced%2Bevolutionary%2Bshift%2Bof%2Bthe%2Bpseudoautosomal%2Bregion%2Bboundary%2Bin%2Bhouse%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9403-5%26rft_id%253Dinfo%253Apmid%252F22763584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9403-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22763584&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306694700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.12.3893.29" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F12%2F3893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/12/3893.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/12/3883" title="A Natural Mutation Involving both Pathogenicity and Perithecium Formation in the Fusarium graminearum Species Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/12/3903" title="Autosomal and X-Linked Additive Genetic Variation for Lifespan and Aging: Comparisons Within and Between the Sexes in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/12"><span>Volume 6 Issue 12, December 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/12.cover-source.jpg?itok=F-Hw09b0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/12/3893.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Jt-huWWJwb8KA9eqy2rZs19vnhMHaBaLeIvgocIWEuk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/12/3893" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/12/3893" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/12/3893" class="active" data-icon-position="" data-hide-link-title="0">Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460500" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qmliRaJIS0o9aqbYiYnfPAhnQiESsmv2iPOG7xcYlNY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3893.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/12/3893" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-E-0UTEQ0tPNSJs82MblGLwJzmt9VB6IfzB-kArq0hoM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/12/3893" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/12/3893.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460500" id="node460500--2" data-pisa="ggg;6/12/3893" data-pisa-master="ggg;g3.116.035527" data-apath="/ggg/6/12/3893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elissa J. Chesler</span>, <span class="highwire-citation-author" data-delta="1">Daniel M. Gatti</span>, <span class="highwire-citation-author" data-delta="2">Andrew P. Morgan</span>, <span class="highwire-citation-author" data-delta="3">Marge Strobel</span>, <span class="highwire-citation-author" data-delta="4">Laura Trepanier</span>, <span class="highwire-citation-author" data-delta="5">Denesa Oberbeck</span>, <span class="highwire-citation-author" data-delta="6">Shannon McWeeney</span>, <span class="highwire-citation-author" data-delta="7">Robert Hitzemann</span>, <span class="highwire-citation-author" data-delta="8">Martin Ferris</span>, <span class="highwire-citation-author" data-delta="9">Rachel McMullan</span>, <span class="highwire-citation-author" data-delta="10">Amelia Clayshultle</span>, <span class="highwire-citation-author" data-delta="11">Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="12">Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author" data-delta="13">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3893-3902;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035527 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460500--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marge Strobel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marge%2BStrobel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strobel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarge%2BStrobel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Laura Trepanier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BTrepanier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Trepanier%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BTrepanier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Denesa Oberbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denesa%2BOberbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oberbeck%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenesa%2BOberbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shannon McWeeney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BMcWeeney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McWeeney%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BMcWeeney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Robert Hitzemann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BHitzemann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hitzemann%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BHitzemann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Martin Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rachel McMullan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BMcMullan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMullan%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BMcMullan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Amelia Clayshultle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amelia%2BClayshultle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clayshultle%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmelia%2BClayshultle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Villena%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/12/3893" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/12/3893&amp;title=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/12/3893&amp;title=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/12/3893&amp;title=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/12/3893&amp;text=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/12/3893&amp;title=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/12/3893&amp;t=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/12/3893&amp;title=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/12/3893&amp;title=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460500/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460500" id="node460500--3" data-pisa="ggg;6/12/3893" data-pisa-master="ggg;g3.116.035527" data-apath="/ggg/6/12/3893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Diversity Outbred Mice at 21: Maintaining Allelic Variation in the Face of Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elissa J. Chesler</span>, <span class="highwire-citation-author" data-delta="1">Daniel M. Gatti</span>, <span class="highwire-citation-author" data-delta="2">Andrew P. Morgan</span>, <span class="highwire-citation-author" data-delta="3">Marge Strobel</span>, <span class="highwire-citation-author" data-delta="4">Laura Trepanier</span>, <span class="highwire-citation-author" data-delta="5">Denesa Oberbeck</span>, <span class="highwire-citation-author" data-delta="6">Shannon McWeeney</span>, <span class="highwire-citation-author" data-delta="7">Robert Hitzemann</span>, <span class="highwire-citation-author" data-delta="8">Martin Ferris</span>, <span class="highwire-citation-author" data-delta="9">Rachel McMullan</span>, <span class="highwire-citation-author" data-delta="10">Amelia Clayshultle</span>, <span class="highwire-citation-author" data-delta="11">Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="12">Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author" data-delta="13">Gary A. Churchill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3893-3902;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035527 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460500--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elissa J. Chesler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elissa%2BJ.%2BChesler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chesler%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElissa%2BJ.%2BChesler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel M. Gatti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BGatti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gatti%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BM.%2BGatti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marge Strobel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marge%2BStrobel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strobel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarge%2BStrobel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Laura Trepanier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BTrepanier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Trepanier%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaura%2BTrepanier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Denesa Oberbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denesa%2BOberbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oberbeck%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenesa%2BOberbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shannon McWeeney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BMcWeeney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McWeeney%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BMcWeeney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Robert Hitzemann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Behavioral Neuroscience and Portland Alcohol Research Center, Oregon Health & Science University, Oregon 97201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BHitzemann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hitzemann%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BHitzemann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Martin Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rachel McMullan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BMcMullan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMullan%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BMcMullan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Amelia Clayshultle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amelia%2BClayshultle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clayshultle%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmelia%2BClayshultle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Villena%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gary.churchill@jax.org" class="" data-icon-position="" data-hide-link-title="0">gary.churchill@jax.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3893&amp;rft.epage=3902&amp;rft.atitle=Diversity%20Outbred%20Mice%20at%2021%3A%20Maintaining%20Allelic%20Variation%20in%20the%20Face%20of%20Selection&amp;rft.volume=6&amp;rft.issue=12&amp;rft.date=2016-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chesler%2C+Elissa+J.&amp;rft.au=Gatti%2C+Daniel+M.&amp;rft.au=Morgan%2C+Andrew+P.&amp;rft.au=Strobel%2C+Marge&amp;rft.au=Trepanier%2C+Laura&amp;rft.au=Oberbeck%2C+Denesa&amp;rft.au=McWeeney%2C+Shannon&amp;rft.au=Hitzemann%2C+Robert&amp;rft.au=Ferris%2C+Martin&amp;rft.au=McMullan%2C+Rachel&amp;rft.au=Clayshultle%2C+Amelia&amp;rft.au=Bell%2C+Timothy+A.&amp;rft.au=de+Villena%2C+Fernando+Pardo-Manuel&amp;rft.au=Churchill%2C+Gary+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460500/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460500/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460500/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460500/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460500/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460500/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460500/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460500/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460500/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460500/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460500/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460500/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/12/3893" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/12/3893.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/12/3893.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/12/3893.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/12/3893.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
